

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mathematical Identification of Critical Reactions in the Interlocked Feedback Model</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001103"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001103" />

    <meta name="citation_title" content="Mathematical Identification of Critical Reactions in the Interlocked Feedback Model"/>
    <meta itemprop="name" content="Mathematical Identification of Critical Reactions in the Interlocked Feedback Model"/>

      <meta name="citation_author" content="Hiroyuki Kurata"/>
            <meta name="citation_author_institution" content="Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Fukuoka, Japan"/>
      <meta name="citation_author" content="Takayuki Tanaka"/>
            <meta name="citation_author_institution" content="Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Fukuoka, Japan"/>
      <meta name="citation_author" content="Fumitaka Ohnishi"/>
            <meta name="citation_author_institution" content="Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Fukuoka, Japan"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001103.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1103"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Concepts relating the behavior of biochemical systems to their underlying molecular properties.; citation_author=MA Savageau; citation_journal_title=Arch Biochem Biophys; citation_volume=145; citation_number=1; citation_pages=612-621; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Parameter sensitivity as a criterion for evaluating and comparing the performance of biochemical systems.; citation_author=MA Savageau; citation_journal_title=Nature; citation_volume=229; citation_number=2; citation_pages=542-544; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Reverse engineering of biological complexity.; citation_author=ME Csete; citation_author=JC Doyle; citation_journal_title=Science; citation_volume=295; citation_number=3; citation_pages=1664-1669; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Surviving heat shock: control strategies for robustness and performance.; citation_author=H El-Samad; citation_author=H Kurata; citation_author=JC Doyle; citation_author=CA Gross; citation_author=M Khammash; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=4; citation_pages=2736-2741; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Module-based analysis of robustness tradeoffs in the heat shock response system.; citation_author=H Kurata; citation_author=H El-Samad; citation_author=R Iwasaki; citation_author=H Ohtake; citation_author=JC Doyle; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=5; citation_pages=e59; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Robustness of cellular functions.; citation_author=J Stelling; citation_author=U Sauer; citation_author=Z Szallasi; citation_author=FJ Doyle; citation_author=J Doyle; citation_journal_title=Cell; citation_volume=118; citation_number=6; citation_pages=675-685; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantifying robustness of biochemical network models.; citation_author=L Ma; citation_author=PA Iglesias; citation_journal_title=BMC Bioinformatics; citation_volume=3; citation_number=7; citation_pages=38; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Quantitative performance metrics for robustness in circadian rhythms.; citation_author=N Bagheri; citation_author=J Stelling; citation_author=FJ Doyle; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=8; citation_pages=358-364; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Extending the method of mathematically controlled comparison to include numerical comparisons.; citation_author=R Alves; citation_author=MA Savageau; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=9; citation_pages=786-798; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Robustness properties of circadian clock architectures.; citation_author=J Stelling; citation_author=ED Gilles; citation_author=FJ Doyle; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=10; citation_pages=13210-13215; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Modeling the molecular regulatory mechanism of circadian rhythms in Drosophila.; citation_author=JC Leloup; citation_author=A Goldbeter; citation_journal_title=Bioessays; citation_volume=22; citation_number=11; citation_pages=84-93; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Interlocked feedback loops within the Drosophila circadian oscillator.; citation_author=NR Glossop; citation_author=LC Lyons; citation_author=PE Hardin; citation_journal_title=Science; citation_volume=286; citation_number=12; citation_pages=766-768; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Interlocked feedback loops contribute to the robustness of the Neurospora circadian clock.; citation_author=P Cheng; citation_author=Y Yang; citation_author=Y Liu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=13; citation_pages=7408-7413; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Robust oscillations within the interlocked feedback model of Drosophila circadian rhythm.; citation_author=HR Ueda; citation_author=M Hagiwara; citation_author=H Kitano; citation_journal_title=J Theor Biol; citation_volume=210; citation_number=14; citation_pages=401-406; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A Real-coded Genetic Algorithm for Function Optimization Using Unimodal Normal Distribution Crossover.; citation_author=I Ono; citation_author=S Kobayashi; citation_journal_title=Proc of 7th Int Conf on Genetic Algorithms; citation_number=15; citation_pages=246-253; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Real-coded genetic algorithms and interval schemata.; citation_author=LJ Eshelman; citation_author=JD Schaffer; citation_number=16; citation_pages=187-202; citation_date=1993; citation_publisher=Morgan Kaufmann; " />
      <meta name="citation_reference" content="citation_title=CADLIVE Dynamic Simulator: Direct Link of Biochemical Networks to Dynamic Models.; citation_author=H Kurata; citation_author=K Masaki; citation_author=Y Sumida; citation_author=R Iwasaki; citation_journal_title=Genome Res; citation_volume=15; citation_number=17; citation_pages=590-600; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Modeling circadian oscillations with interlocking positive and negative feedback loops.; citation_author=P Smolen; citation_author=DA Baxter; citation_author=JH Byrne; citation_journal_title=J Neurosci; citation_volume=21; citation_number=18; citation_pages=6644-6656; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=CADLIVE for constructing a large-scale biochemical network based on a simulation-directed notation and its application to yeast cell cycle.; citation_author=H Kurata; citation_author=N Matoba; citation_author=N Shimizu; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=19; citation_pages=4071-4084; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of the circadian system in Drosophila melanogaster and mammals.; citation_author=R Stanewsky; citation_journal_title=J Neurobiol; citation_volume=54; citation_number=20; citation_pages=111-147; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Two novel doubletime mutants alter circadian properties and eliminate the delay between RNA and protein in Drosophila.; citation_author=V Suri; citation_author=JC Hall; citation_author=M Rosbash; citation_journal_title=J Neurosci; citation_volume=20; citation_number=21; citation_pages=7547-7555; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=OBIYagns: a grid-based biochemical simulator with a parameter estimator.; citation_author=S Kimura; citation_author=T Kawasaki; citation_author=M Hatakeyama; citation_author=T Naka; citation_author=F Konishi; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=22; citation_pages=1646-1648; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mathematical Identification of Critical Reactions in the Interlocked Feedback Model"/>
    <meta name="twitter:description" content="Dynamic simulations are necessary for understanding the mechanism of how biochemical networks generate robust properties to environmental stresses or genetic changes. Sensitivity analysis allows the linking of robustness to network structure. However, it yields only local properties regarding a particular choice of plausible parameter values, because it is hard to know the exact parameter values in vivo. Global and firm results are needed that do not depend on particular parameter values. We propose mathematical analysis for robustness (MAR) that consists of the novel evolutionary search that explores all possible solution vectors of kinetic parameters satisfying the target dynamics and robustness analysis. New criteria, parameter spectrum width and the variability of solution vectors for parameters, are introduced to determine whether the search is exhaustive. In robustness analysis, in addition to single parameter sensitivity analysis, robustness to multiple parameter perturbation is defined. Combining the sensitivity analysis and the robustness analysis to multiple parameter perturbation enables identifying critical reactions. Use of MAR clearly identified the critical reactions responsible for determining the circadian cycle in the Drosophila interlocked circadian clock model. In highly robust models, while the parameter vectors are greatly varied, the critical reactions with a high sensitivity are uniquely determined. Interestingly, not only the per-tim loop but also the dclk-cyc loop strongly affect the period of PER, although the dclk-cyc loop hardly changes its amplitude and it is not potentially influential. In conclusion, MAR is a powerful method to explore wide parameter space without human-biases and to link a robust property to network architectures without knowing the exact parameter values. MAR identifies the reactions critically responsible for determining the period and amplitude in the interlocked feedback model and suggests that the circadian clock intensively evolves or designs the kinetic parameters so that it creates a highly robust cycle."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001103.g009"/>

  <meta property="og:title" content="Mathematical Identification of Critical Reactions in the Interlocked Feedback Model" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7037'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6024'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6330&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001103">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001103" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001103&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mathematical Identification of Critical Reactions in the Interlocked Feedback Model
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiroyuki Kurata
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kurata@bio.kyutech.ac.jp">kurata@bio.kyutech.ac.jp</a></p>

                <p>Affiliation:
                  Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Fukuoka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takayuki Tanaka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Fukuoka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fumitaka Ohnishi
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Bioscience and Bioinformatics, Kyushu Institute of Technology, Fukuoka, Japan
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001103</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001103-g001" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g002" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-t001" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-t002" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-t003" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g003" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g004" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g005" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g006" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g007" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g008" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001103-g009" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001103">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001103" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Dynamic simulations are necessary for understanding the mechanism of how biochemical networks generate robust properties to environmental stresses or genetic changes. Sensitivity analysis allows the linking of robustness to network structure. However, it yields only local properties regarding a particular choice of plausible parameter values, because it is hard to know the exact parameter values <em>in vivo</em>. Global and firm results are needed that do not depend on particular parameter values. We propose mathematical analysis for robustness (MAR) that consists of the novel evolutionary search that explores all possible solution vectors of kinetic parameters satisfying the target dynamics and robustness analysis. New criteria, parameter spectrum width and the variability of solution vectors for parameters, are introduced to determine whether the search is exhaustive. In robustness analysis, in addition to single parameter sensitivity analysis, robustness to multiple parameter perturbation is defined. Combining the sensitivity analysis and the robustness analysis to multiple parameter perturbation enables identifying critical reactions. Use of MAR clearly identified the critical reactions responsible for determining the circadian cycle in the <em>Drosophila</em> interlocked circadian clock model. In highly robust models, while the parameter vectors are greatly varied, the critical reactions with a high sensitivity are uniquely determined. Interestingly, not only the <em>per-tim</em> loop but also the <em>dclk-cyc</em> loop strongly affect the period of PER, although the <em>dclk-cyc</em> loop hardly changes its amplitude and it is not potentially influential. In conclusion, MAR is a powerful method to explore wide parameter space without human-biases and to link a robust property to network architectures without knowing the exact parameter values. MAR identifies the reactions critically responsible for determining the period and amplitude in the interlocked feedback model and suggests that the circadian clock intensively evolves or designs the kinetic parameters so that it creates a highly robust cycle.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kurata H, Tanaka T, Ohnishi F (2007) Mathematical Identification of Critical Reactions in the Interlocked Feedback Model. PLoS ONE 2(10):
          e1103.
            doi:10.1371/journal.pone.0001103</p><p><strong>Academic Editor: </strong>Nick Monk, University of Nottingham, United Kingdom</p><p><strong>Received:</strong> July 30, 2007; <strong>Accepted:</strong> October 12, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Kurata et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The Ministry of Education, Science, Sports and Culture, Grant-in-Aid for Scientific Research (B), 2006, 18300098</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A goal of systems biology is the generation of a coherent understanding of the mechanisms of how the integration of various components generates functional and robust systems in living organisms <a href="#pone.0001103-Savageau1">[1]</a>–<a href="#pone.0001103-Csete1">[3]</a>. The importance for robustness is a functional criterion for performance of biochemical networks. Robustness is the ability to resume successful operation in the presence of signal and system uncertainties. Descriptions of the mechanisms by which biochemical networks are robust in the face of parameter uncertainty, environmental changes, and stochastic fluctuations have been published <a href="#pone.0001103-Csete1">[3]</a>–<a href="#pone.0001103-Stelling1">[6]</a>. Different types of feedback loops and pathway redundancies are involved in enhanced robustness. Sensitivity analysis of mathematical models describing complex networks could allow for linking robustness properties to network structure by measuring the degree to which parametric perturbations change various target dynamics <a href="#pone.0001103-Ma1">[7]</a>, <a href="#pone.0001103-Bagheri1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In principle, both molecular architecture and the values of kinetic parameters determine the robustness of dynamic systems. In biological systems, molecular structures are being built, but it is still hard to know the exact values of kinetic parameters <em>in vivo</em>. The values of kinetic parameters vary with time and environment and the measured values <em>in vitro</em> are often different from those <em>in vivo</em>. In most studies, a particular set of local kinetic parameters has been determined for convenience so that dynamic models reproduce target data. It is necessary to exclude the possibility that the calculated results are dependent on variations in the values assigned to kinetic parameters. The dependency of robustness on parameter values has been investigated by only a few reports. To compare some performances of alternative mathematical models, Alves <em>et al</em> <a href="#pone.0001103-Alves1">[9]</a> statistically searched their parameter values so that they make the other dynamic properties the same. Stelling <em>et al</em> <a href="#pone.0001103-Stelling2">[10]</a> studied dynamic properties linked to network structure in the <em>per-tim</em> feedback loop model by systematically investigating the 2D-parameter space and suggested some influential process determining the oscillator features. These previously presented random or systematic searches are a great step for approaching to global analysis, but they restricted the search space of parameters or the size of models due to calculation complexity and few criteria have been presented to determine whether the search is exhaustive.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The molecular mechanisms of how circadian clocks generate robust cycles have extensively been studied and negative feedback loops are found common structures for producing robust cycles <a href="#pone.0001103-Leloup1">[11]</a>–<a href="#pone.0001103-Cheng1">[13]</a>. An interlocked feedback model is typical architecture to provide robustness of the cycle and synclonization of two clocks to environmental perturbation or parameter uncertainty<a href="#pone.0001103-Ueda1">[14]</a>, while critical reactions responsible for determining cycle features such as amplitude and period remain to be seen due to experimental complexity. Generally both network architecture and its associated parameter values determine the system's function. For example, engineering design requires the tuning process of many parameters for producing an optimal and highly robust system. However, neither the optimal values of parameters nor the reactions critically responsible for a robust cycle are known in circadian clock systems.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here, we propose a new strategy, named mathematical analysis for robustness (MAR), that combines robustness analysis with an evolutionary search that explores many plausible solutions for kinetic parameters producing the same dynamic properties. To search large parameter spaces without any human bias, the two-step evolutionary method is proposed and novel and rigorous criteria are introduced that characterize whether the search approaches an exhaustive level. MAR leads to an understanding of the global mechanism by which the <em>Drosophila</em> interlocked feedback system generates the robustness of the circadian cycle without insisting on the exact values of kinetic parameters.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Mathematical Analysis for Robustness algorithm</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To link molecular architecture to robustness we propose MAR as shown in <a href="#pone-0001103-g001"><strong>Figure 1</strong></a>. It consists of two stages: exhaustive search and robustness analysis. In the former stage, (1) we determine the target dynamic features that a mathematical model should simulate, (2) explore all possible solution vectors of kinetic parameters that reproduce the target features using the two-step evolutionary search, and (3) test the validity of exhaustive search (<strong>Equations 3</strong>, <strong>7</strong>). In the latter stage, (4) we simulate the sensitivity of target features to perturbation of a single parameter (<strong>Equation 8</strong>), (5) calculate the robustness to perturbation of multiple parameters (<strong>Equation 13</strong>), and (6) identify the critical reactions responsible for determining target features by combining both the sensitivity and robustness analyses. Details of the algorithms are described in <a href="#s4">Materials and Methods</a>.</p>
<div class="figure" id="pone-0001103-g001"><div class="img"><a name="pone-0001103-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic diagrams of exhaustive search and robustness analysis in MAR.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A: A flowchart for MAR. B: The proposed two-step search method that consists of a random search (RS) and a search by genetic algorithms (GAs). C: How to make the initial populations for a search by GAs.</p>
<span>doi:10.1371/journal.pone.0001103.g001</span></div>

<h4>Validation of an exhaustive search method by using a theoretical model</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We propose the two-step evolutionary method that consists of the random search and the subsequent evolutionary search as a fast and non-biased method (<a href="#pone-0001103-g001"><strong>Figure 1BC</strong></a>). To demonstrate the advantage of the two-step method, we compared it with various evolutionary searches, such as Unimordal Normal Distribution Crossover (UNDX), UNDXm, Blend Crossover (BLX), and Simplex Crossover (SPX) <a href="#pone.0001103-Ono1">[15]</a>–<a href="#pone.0001103-Kurata2">[17]</a>, by using a theoretical model (<a href="#pone.0001103.s001"><strong>Text S1</strong></a> and <strong>Equations S1-S5</strong> in <a href="#pone.0001103.s001"><strong>Text S1</strong></a>). The two-step method was a best choice for a fast and non-biased search among available search method (<a href="#pone.0001103.s002"><strong>Table S1</strong></a>). To test whether the search by the two-step method becomes saturated or covers the entire solution space, we investigated the convergence of the parameters spectrum width and the variability of solutions (<em>VarS</em>) (<strong>Equations 3</strong>, <strong>7</strong>, <a href="#pone.0001103.s004"><strong>Figure S1</strong></a>). Finally the sensitivity distributions were numerically calculated. The range of the simulated sensitivity was consistent with the theoretical range (<strong>Equation S3</strong> in <a href="#pone.0001103.s001"><strong>Text S1</strong></a>) and the two-step method identified the potentially influential parameter (<strong>Equation 10</strong>, <a href="#pone.0001103.s005"><strong>Figure S2</strong></a>). By using a theoretical model, the two-step method is demonstrated to explore the entire solution space at a fast speed. The parameter spectrum width and the variability of solutions are useful measures for the search process of MAR.</p>


<h4>Mathematical model of a biochemical oscillator and exhaustive search</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>MAR is applied to the <em>Drosophila</em> circadian clock model as shown in <a href="#pone-0001103-g002"><strong>Figure 2</strong></a>. We have derived mathematical equations from the interlocked-feedback model developed by Ueda et al. <a href="#pone.0001103-Ueda1">[14]</a>, as shown in <a href="#pone-0001103-t001"><strong>Tables 1</strong></a><strong>–</strong><a href="#pone-0001103-t002"></a><a href="#pone-0001103-t003"><strong>3</strong></a>. The target feature of the circadian oscillator was set as the PER oscillation curves characterized by the period and the top and bottom of the curves (amplitude). There are few reliable experimental data that measure the absolute concentrations for the circadian proteins in <em>Drosophila</em>, while they have been provided by most literatures as the relative value to a constant level molecule. Thus, we produce a mathematical but qualitative model that reproduces the intrinsic features of the circadian cycle and adjust it so that the simulated time courses of the proteins are consistent with those presented by commonly employed simulation models, e.g., the PER concentration varies from a few nM to a few dozen nM <a href="#pone.0001103-Smolen1">[18]</a>. To explore the target PER oscillator, a fitness function was designed and twelve kinetic parameters were selected as the search parameters (<em>S[1]</em>, <em>A[1] = A[2]</em>, <em>R[1] = R[2]</em>,<em>V[1]</em>, <em>V[2]</em>, <em>D[1]</em>, <em>P[1]</em>, <em>T[1]</em>, <em>K[1]</em>, and <em>K[2]</em>). The two-step method was employed. Details of the fitness function and searches are described in Method (<strong>Equations 14</strong>, <strong>15</strong>) and the validity of these search parameters will be verified in the later section (Numerical sensitivity to single parameter perturbation). One thousand optimized PER oscillators were obtained by performing 2×10<sup>5</sup> random searches and the subsequent GAs. We investigated the dynamic features of PER, dCLK and TIM, where the circadian model was optimized with regard to the PER cycle (<a href="#pone.0001103.s006"><strong>Figures S3</strong></a>, <a href="#pone.0001103.s007"><strong>S4</strong></a>). The period for dCLK or TIM was consistent with that for PER. The amplitude and mean concentration of PER oscillators varied over a narrow range of values and the ratio of the amplitude to the mean concentration was approximately 1.8. The amplitude and mean concentration of dCLK and TIM varied over a wider range of values than those for PER, and dCLK and TIM oscillated with ratios of 1.8 and 1.4, respectively. These results indicate that they are good oscillators and reproduce the target features. According to experimental data regarding the circadian oscillator, the ratio of the amplitude to the mean concentration varies from 1.2 to 2, depending on experimental conditions. These values of the ration are reasonable within the range of experimental data.</p>
<div class="figure" id="pone-0001103-g002"><div class="img"><a name="pone-0001103-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>A schematic diagram of the interlocked feedback system in the <em>Drosophila</em> circadian clock.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>In the real biochemical reactions, the binding of PER-P:TIM to dCLK:CYC suppresses the transcription for PER and TIM and activates the transcription for dCLK. In this map, PER-P:TIM directly suppresses the transcription of PER and TIM and activates the transcription for dCLK. “:” and “-” indicate binding complexes and modification, respectively.</p>
<span>doi:10.1371/journal.pone.0001103.g002</span></div><div class="figure" id="pone-0001103-t001"><div class="img"><a name="pone-0001103-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Mathematical equations for the interlocked-feedback system in Drosophila</span></strong></p><span>doi:10.1371/journal.pone.0001103.t001</span></div><div class="figure" id="pone-0001103-t002"><div class="img"><a name="pone-0001103-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Components used in the mathematical model</span></strong></p><span>doi:10.1371/journal.pone.0001103.t002</span></div><div class="figure" id="pone-0001103-t003"><div class="img"><a name="pone-0001103-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Kinetic parameters used in the mathematical model</span></strong></p><span>doi:10.1371/journal.pone.0001103.t003</span></div>

<h4>Characterization of exhaustive search</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To characterize whether the search is sufficient or exhaustive we investigated the convergence for two measures: the parameter-spectrum width (<strong>Equation 3</strong>) and the variability in the feasible solution space of the parameter vectors (<strong>Equation 7</strong>). <a href="#pone-0001103-g003"><strong>Figure 3</strong></a> shows the changes in the parameter-spectrum width with respect to the number of solutions. The solutions were added by the progression of evolutionary searches. For all the search parameters, the parameter spectrum width increased with the number of solutions and then converged above a solution number of 400.</p>
<div class="figure" id="pone-0001103-g003"><div class="img"><a name="pone-0001103-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Convergence of the logarithmic width of the parameter spectrum for 12 search parameters.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>One thousand solutions for optimized kinetic parameters were obtained by using the repetition of the two-step search. The black solid line is <em>S[1]</em>, the black dotted line <em>A[1] = A[2]</em>, the black chain <em>R[1] = R[2]</em>, the blue solid line <em>V[1]</em>, the blue dotted line <em>V[2]</em>, the blue chain <em>D[1]</em>, the red solid line <em>P[1]</em>, the red dotted line <em>T[1]</em>, the yellow solid line <em>K[1]</em>, and the red chain <em>K[2]</em>.</p>
<span>doi:10.1371/journal.pone.0001103.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To reveal the variability of the 12-dimensional parameter vectors optimized to provide the similar time course for PER, we classified those solutions by using a hierarchical clustering with the average linkage method, where the parameter distances <em>D</em>(<strong>P(</strong><em>i</em><strong>)</strong>,<strong>P(</strong><em>j</em><strong>)</strong>) were calculated among the solutions (<strong>Equation 4</strong>). When the parameter spectra converged at a solution number of 400, we clustered the early 400 solutions into six super-balls, where the centroid for each cluster is employed as a representative vector.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Here we demonstrate whether the super-balls for those clusters are sufficient for containing the subsequently optimized solutions (from 401 to 1000). As shown in <a href="#pone-0001103-g004"><strong>Figure 4</strong></a>, we plot <em>VarS</em> (<strong>Equation 7</strong>) with respect to the solution number, while the number of the employed clusters is varied. This figure indicates the convergence of the variability in solution vectors with respect to the solution number. By using <em>VarS</em>, we investigate if all the solutions belong to one of the six clusters. <em>VarS</em> is less than one for the early 400 solutions when the six clusters are employed. When solutions were freshly added by the progression of evolutionary searches, the value of <em>VarS</em> increased, showing that some new solutions are distant from the centroid of the employed clusters. An increase in the number of employed clusters decreased the <em>VarS</em> value, which confirms that the increase in the employed clusters enlarges the solution space. When all the six clusters were employed, <em>VarS</em> was saturated close to one. This shows that all the solutions can be assigned to one of the employed clusters, suggesting that the search process is saturated at a solution number of 400.</p>
<div class="figure" id="pone-0001103-g004"><div class="img"><a name="pone-0001103-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Convergence for the variability in the solutions of kinetic parameters.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Six clusters were generated from using the early 400 solutions. The number of the employed clusters is changed from 1 to 6. <em>VarS</em> (Equation 7) is plotted with respect to the solution number. <em>VarS</em> of less than one shows the parameter solutions are included in the employed clusters. The brown line is one cluster employed, the blue line two clusters, the green line three clusters, the yellow four clusters, the red line five clusters, and the black line six clusters.</p>
<span>doi:10.1371/journal.pone.0001103.g004</span></div>

<h4>Numerical sensitivity to single parameter perturbation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The period and amplitude sensitivities for the PER oscillator were simulated for 400 optimized solutions. The numerical sensitivity of period (<em>S<sub>T</sub></em>) or amplitude (<em>S<sub>A</sub></em>) to variations in a single kinetic parameter was calculated (<strong>Equation 16</strong>). Broad distributions of the absolute sensitivities occur as shown in <a href="#pone-0001103-g005"><strong>Figure 5</strong></a>, where the minimum, maximum and mean values are plotted with respect to each kinetic parameter. The mean or minimum values indicate that this model provides a robust property of period and amplitude to parameter uncertainty. The period sensitivity was less than the amplitude sensitivity. Applying a threshold value of 10<sup>−8</sup> to the minimum sensitivities allowed the division of kinetic parameters into potentially influential and non-influential categories (<strong>Equation 10</strong>). Seventeen potentially influential parameters for period and amplitude were identified respectively, which were consistent between them. PER synthesis and degradation (<em>S[1], S[2], D[1], D[2], D[3], D[6], D[7]</em>), phosphorylation and transport for PER (<em>P[1], K[1], T[1], K[2]</em>), TIM synthesis and degradation (<em>S[3], S[4], D[4], D[5]</em>), and binding of PER-P and TIM (<em>V[1], V[2]</em>) are potentially influential. It indicates that the effect of every reaction involving the <em>per-time</em> loop cannot be cancelled in any possible models. It is reasonable because the <em>per-tim</em> loop is not redundant. By contrast, the kinetic parameters involving the <em>dclk-cyc</em> loop are not potentially influential. On the other hand, a maximum value of period sensitivity was more than 0.3 (7.2 h) with respect to some kinetic parameters, indicating that not only network structures but also parameter values are critical for generating a robust cycle. If an inappropriate set of kinetic parameters is provided, an oscillation can be damaged by even a small change in a single parameter. The interlocked feedback is able to provide robustness to the circadian rhythm when the kinetic parameters are appropriately tuned.</p>
<div class="figure" id="pone-0001103-g005"><div class="img"><a name="pone-0001103-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Distributions of period and amplitude sensitivities.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>For 400 solutions, we simulated the period and amplitude sensitivity by changing the value of each kinetic parameter by 1.5-fold. A change ratio of 1.5 was validated (data not shown). The absolute sensitivities are employed. The resulting distributions exclude the models whose oscillations are abolished by parameter perturbations (|sensitivity|&gt;10). The sensitivities with a value of less than 10<sup>−10</sup> are set to 10<sup>−10</sup>. The black circles indicate the minimum, the blue crosses the maximum, and the red triangles the mean, respectively. The kinetic parameters are sorted according to the minimum value of the absolute sensitivities in the descending order. Use of a threshold value of 10<sup>−8</sup> separates the critically influential parameters. (A) Period sensitivity, (B) Amplitude sensitivity.</p>
<span>doi:10.1371/journal.pone.0001103.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Numerical sensitivity analysis selected 17 potentially influential parameters within the provided 12-dimensional space. A question raises whether these parameters can be potentially influential in the entire 36 dimensional parameter space. Of course, such a complete search is impossible due to calculation explosion inherent of numerical analysis. Instead, an extended search is performed to investigate how an expansion of a search parameter space affects the choice of potentially influential parameters, where the number of search parameters is varied as 3, 6, 9, 12, 16, 25, and 36. (<a href="#pone.0001103.s003"><strong>Table S2</strong></a> and <a href="#pone.0001103.s008"><strong>Figure S5</strong></a>). The exploration with a search parameter number of ≤9, a low-dimensional subspace, could not obtain the consistent sets of potentially influential parameters. On the other hand, the analyses with a search parameter number of ≥16 selected the same parameter sets as the 12-dimensional search. Although the variability in the solutions does not necessarily converge less than one above a search parameter number of ≥16 due to an extremely large search space, this extended search suggests that the potentially influential parameters are consistent. In conclusion, twelve search parameters are necessary and sufficient for finding them.</p>


<h4>Robustness to multiple parameter perturbation</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>There are many possible models that generate circadian oscillators. It raises a question of which models are close to a real biochemical oscillator or a question of which reactions feature robust oscillators. Robustness to multiple parameter perturbation is a key criterion for model selection. Here we define the CVs of the period and amplitude distributions generated by randomly-perturbed parameters <em>CV</em>_<em>Period</em>(<em>i</em>) and <em>CV</em>_<em>Amplitude</em>(<em>i</em>) (<strong>Equation 13</strong>), which characterize the robustness of the period and amplitude to simultaneous changes in all kinetic parameters, assuming that the oscillators are always exposed to uncertainty or changes in parameters under strict environmental stresses. An oscillator with a small value of CV is a highly robust model, because the change in period or amplitude is suppressed to be small against random changes in all kinetic parameters.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>For all 400 solution vectors (oscillator models), the period and amplitude were simulated 10,000 times by randomly varying all kinetic parameters within a range from 1/1.5-fold to 1.5-fold, resulting in 400 distributions of the simulated period and amplitude. The means and CVs almost converged at 10,000 time simulations. The frequency distributions of the period and amplitude for a certain parameter vector are exemplified as shown in <a href="#pone-0001103-g006"><strong>Figure 6</strong></a>. We plotted the frequency distribution of the CVs of period and amplitude for 400 solution vectors as shown in <a href="#pone-0001103-g007"><strong>Figure 7</strong></a>. The CVs of period changed from 0.08 to 0.44. The mean periods were around 24 h (data not shown). A CV of 0.08 indicates 1.9 h, which significantly changes the circadian cycle. On the other hand, the CVs of amplitude changed above 0.4, which was larger than the CVs of period. The mean amplitudes were around 10 nM, which is reasonable because it is within the range of commonly simulated values <a href="#pone.0001103-Smolen1">[18]</a>. This figure indicates that an appropriate selection of kinetic parameters decreases the CVs, i.e., a tuning of kinetic parameters enhances robustness to random changes in all kinetic parameters. It is interesting if a biological system adjusts kinetic parameter values so that robustness is most enhanced.</p>
<div class="figure" id="pone-0001103-g006"><div class="img"><a name="pone-0001103-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Frequency distributions of the period and amplitude in a randomly perturbed oscillator model.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The period (A) and amplitude (B) were simulated 10,000 times while all kinetic parameters were randomly varied within a range from 1/1.5-fold to 1.5-fold. The oscillators with a long period (&gt;36 h) are excluded, resulting in the remove of the oscillators with small amplitude.</p>
<span>doi:10.1371/journal.pone.0001103.g006</span></div><div class="figure" id="pone-0001103-g007"><div class="img"><a name="pone-0001103-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Frequency of the CVs of period and amplitude distributions in randomly perturbed oscillator models.</span></strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>For 400 solution vectors (oscillator models), the period and amplitude were simulated 10,000 times by randomly varying all kinetic parameters within a range from 1/1.5-fold to 1.5-fold, resulting in 400 distributions of the simulated period and amplitude. The CVs were calculated for every period and amplitude distribution and the frequency distributions of the CVs for period (A) and amplitude (B) are plotted for 400 oscillator models.</p>
<span>doi:10.1371/journal.pone.0001103.g007</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>To investigate the mechanism of how the circadian clock models provide a robust property to the period and amplitude, the relationship between the CVs of period and amplitude was plotted (<a href="#pone.0001103.s009"><strong>Figure S6</strong></a>). Significant linear correlation was not observed between the CVs of period and amplitude. The mechanism that provides a robust property to the period is suggested to be different from that to the amplitude.</p>


<h4>Critical parameters in highly robust oscillators</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>MAR combines the numerical sensitivity analysis with the CV analysis to identify the critical reactions in the interlocked feedback model. To explore the parameters that show a high sensitivity of period or amplitude, we investigated how the robustness of the oscillator models, <em>CV</em> _<em>Period</em>(<em>i</em>) or <em>CV</em>_<em>Amplitude</em>(<em>i</em>) or (<strong>Equation 13</strong>), is correlated to the sensitivity distributions or to the kinetic parameter vectors. The period and amplitude sensitivity distributions were classified into 6 clusters by using hierarchical clustering with the average linkage (Matlab), respectively. They were named sensitivity clusters of 1 to 6. In the same way the parameter solution vectors were classified into 6 clusters, named parameter clusters of 1 to 6. As shown in <a href="#pone-0001103-g008"><strong>Figure 8</strong></a>, the cluster index that each model belongs to is plotted with respect to the model sorted in the ascending order of the CVs of period or amplitude.</p>
<div class="figure" id="pone-0001103-g008"><div class="img"><a name="pone-0001103-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Classification of the sensitivity distribution and the parameter solution vectors.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>The sensitivity distributions and their associated parameter solution vectors for all the oscillator models that reproduce the circadian cycle are classified into six clusters. The six clusters for the sensitivity distributions and their associated parameter vectors are named sensitivity cluster from 1 to 6 and parameter cluster from 1 to 6, respectively. The models are sorted in the ascending order of the CV value, where the left is highly robust models because their CVs are small. The sensitivity and parameter cluster indexes that each model belongs to are plotted with respect to the model index. The sensitivity cluster (A) and parameter cluster (B) are for the models sorted according to the CV period value. The sensitivity cluster (C) and parameter cluster (D) are for the models sorted according to the CV amplitude value. Note that the number of models is less than 400 because some sensitivity distributions with a very large value (|sensitivity|&gt;10) are excluded.</p>
<span>doi:10.1371/journal.pone.0001103.g008</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>For a high CV period (CV period order &gt;300) the period sensitivity distributions were varied because they belonged to all the clusters. By contrast, in highly robust models with a small CV value (CV period order &lt;50) their sensitivity distributions were consistent because most of the models belonged to the one cluster, while the parameter vectors were greatly varied (they belonged to six clusters). For a high CV amplitude (CV amplitude order &gt;300) the amplitude sensitivity distributions were varied. In highly robust models with a small CV value (CV amplitude order &lt;30), their sensitivity distributions were consistent because most of the models belonged to the one cluster, while the parameter vectors were different. The period and amplitude sensitivity distributions and their associated parameter vectors are illustrated (<a href="#pone.0001103.s010"><strong>Figures S7</strong></a> and <a href="#pone.0001103.s011"><strong>S8</strong></a>). These demonstrate that the period or amplitude sensitivity distributions are very consistent in highly robust models, while the parameter vectors are greatly varied. In other words the critical reactions are uniquely determined in highly robust models despite the variability of the kinetic parameters.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Next, the critical parameters were selected from the sensitivity distributions of both period and amplitude. For ten highly robust models, we calculated the means of period and amplitude absolute sensitivities and normalized them by their maximum values, as shown in <a href="#pone-0001103-g009"><strong>Figure 9</strong></a>. Twelve critical parameters showing a high sensitivity were selected that determine period and amplitude, respectively. The critical parameters responsible for period are <em>D[7], V[2], T[3], D[9], D[6], P[1], D[1], D[10], V[3], D[11], K[1], D[4];</em> those responsible for amplitude are <em>R[1], D[7], T[1], K[2], D[1], D[4], R[2], P[1], V[2], S[1], S[2], D[2].</em> (The parameters are sorted in the rank order of the absolute sensitivity value.) They feature highly robust oscillators (<a href="#pone.0001103.s012"><strong>Figures S9</strong></a>).</p>
<div class="figure" id="pone-0001103-g009"><div class="img"><a name="pone-0001103-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001103" data-uri="info:doi/10.1371/journal.pone.0001103.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001103.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001103.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001103.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>A distribution of the mean absolute sensitivity for period and amplitude in highly robust models.</span></strong></p><a id="article1.body1.sec2.sec7.fig2.caption1.p1" name="article1.body1.sec2.sec7.fig2.caption1.p1"></a><p>The blue and red bars are the period and amplitude sensitivities, respectively. The blue line is the threshold for selecting the critical parameters responsible for period and the red for amplitude. The absolute sensitivity values for ten highly robust models that show the smaller CV values are averaged. The resulting mean sensitivity is normalized by the maximum value. 1:<em>S[1]</em>, 2:<em>A[1]</em>, 3:<em>R[1]</em>, 4:<em>B[1]</em>, 5:<em>S[2]</em>, 6:<em>S[3]</em>, 7:<em>A[2]</em>, 8:<em>R[2]</em>, 9:<em>B[2]</em>, 10:<em>S[4]</em>, 11:<em>P[1]</em>, 12:<em>K[1]</em>, 13:<em>V[1]</em>, 14:<em>V[2]</em>, 15:<em>T[1]</em>, 16:<em>K[2]</em>, 17:<em>D[1]</em>, 18:<em>D[2]</em>, 19:<em>D[4]</em>, 20:<em>D[5]</em>, 21:<em>D[3]</em>, 22<em>:D[6]</em>, 23:<em>D[73]</em>, 24:<em>S[5]</em>, 25:<em>A[3]</em>, 26:<em>R[3]</em>, 27:<em>B[3]</em>, 28:<em>S[6]</em>, 29:<em>V[3]</em>, 30:<em>V[4]</em>, 31:<em>T[3]</em>, 32:<em>K[4]</em>, 33:<em>D[8]</em>, 34:<em>D[9]</em>, 35:<em>D[10]</em>, 36:<em>D[11]</em>.</p>
<span>doi:10.1371/journal.pone.0001103.g009</span></div><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>Degradation (<em>D[1],D[2],D[6],D[7],D[9],D[10],D[11]</em>) is the key reaction that determines both the period and amplitude of the PER oscillator. Since the CVs of period were hardly correlated with those of amplitude (<a href="#pone.0001103.s009"><strong>Figure S6</strong></a>) and the period sensitivity distributions were different from the amplitude ones, the mechanisms providing robust properties to period and amplitude are different. This is supported elsewhere <a href="#pone.0001103-Stelling2">[10]</a>. Here, the kinetic parameters are classified according to their biochemical processes. The amplitude of PER can be determined mainly by reactions in the <em>per-tim</em> loop (<em>S[1], S[2], R[1], R[2], T[1], K[2], D[1], K[1], D[4]</em>), especially the synthesis and transport of PER and TIM (<em>S[1], S[2], R[1], R[2],T[1], K[2]</em>). Both <em>T[1]</em> and <em>K[2]</em> are critical, indicating transport cannot be represented as a single linear reaction. It suggests that a nonlinear reaction plays a significant role in determining the amplitude. On the other hand, the period of PER is strongly affected not only by the <em>per-tim</em> loop but also by the <em>dclk-cyc</em> loop (<em>D[9],D[10],D[11],V[3],T[3]</em>), although the <em>dclk-cyc</em> loop is neither potentially influential nor does affect the amplitude of PER. It shows that the <em>dclk-cyc</em> loop plays a critical role in determining the PER period. The phosphorylation reaction (<em>P[1]</em>, <em>K[1]</em>) would play a major role in determining the period rather than the amplitude. Both <em>P[1]</em> and <em>K[1]</em> are critical, which suggests that a nonlinear reaction is necessary for determining the period.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The objective of MAR is to determine critical reactions without insisting on the exact values of kinetic parameters, thereby linking robust properties to specific molecular architectures. MAR seeks global and firm results that do not depend on particular parameter values. It is a powerful tool because it is still very hard to measure full kinetics <em>in vivo</em>. The key technique is to combine robustness analysis with a two-step evolutionary search that explores many solutions for kinetic parameters showing the similar dynamic features.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Compared with the previous method that systematically analyzed two-dimensional parameter space in the <em>per-time</em> feedback model <a href="#pone.0001103-Stelling2">[10]</a>, MAR enables more precise analysis of more complex models, the interlocked feedback model that contains the <em>per-tim</em> and <em>dclk-cyc</em> loops. MAR is advantageous because it enables one to explore a higher dimensional and much larger space of the kinetic parameters of a complex model. Searching a larger parameter space is demonstrated to make clearer separation of potentially influential parameters (<a href="#pone.0001103.s003"><strong>Table S2</strong></a> and <a href="#pone.0001103.s008"><strong>Figure S5</strong></a>). Notice that a low dimensional search may find a false set of potentially influential parameters. In the circadian model, a search parameter number of ≥12 should be used to obtain the true set of potentially influential reactions in a large parameter space. This shows the extensive search is really required to obtain global and firm results from mathematical analysis.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The potentially influential parameters, which become consistent with progression of search, means that the effect of them cannot be reduced very small or less than a certain threshold value in any possible models, but they do not necessarily indicate critical parameters responsible for target instances. The potentially influential parameters would be effective in testing whether the search is exhaustive rather than in exploring critical reactions.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>The central contribution here is that MAR provides the results compatible with biological knowledge or lead to new hypotheses on the function of cellular networks with realistic complexity. By searching wide parameter space, MAR gives global results that a local simulation cannot obtain. The identification of the relationship among the robustness to multiple parameter perturbation (<strong>Equation 13</strong>), the sensitivity distribution, and the parameter solution vectors could provide a basis to assume a potential mechanisms generating robustness. The robustness to multiple parameter perturbation greatly depends on the values of kinetic parameters. Actually, appropriate sets of kinetic parameters greatly enhance the robustness of period or amplitude, while their sensitivity values converge to a unique distribution. In highly robust models the critical reactions are uniquely determined despite the variability of the parameter values.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>While the phosphorylation of PER and the transport and synthesis of PER and TIM are critical for period and for amplitude, respectively, degradations are key reactions for both period and amplitude. This outcome agrees with previous simulation studies suggesting that circadian performance is greatly affected by changes in degradation <a href="#pone.0001103-Bagheri1">[8]</a>, <a href="#pone.0001103-Stelling2">[10]</a>. Interestingly, the <em>dclk-cyc</em> loop plays a major role in determining the PER period, although the <em>dclk-cyc</em> loop is neither potentially influential nor does affect the amplitude of PER. The mechanism generating robustness to period is found different from that to amplitude. These are supported by a general theory that the robustness of one function does not imply that of other functions due to robustness tradeoff<a href="#pone.0001103-Kurata1">[5]</a>, <a href="#pone.0001103-Stelling1">[6]</a>. Biochemical oscillators should be evolved to provide a robust property to different dynamic functions such as period and amplitude.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Engineering design conventionally consists of the investigation of functional specification and the subsequent basic and detail design. Specification is the act of understanding of a product to be modelled or of defining what a function has to achieve rather than how it has to do it. Basic design carries out the identification, classification and selection of constraints and then determines the design architecture of how the system obtains its specification. The process of detail design manipulates the system's parameters so as to satisfy the specification. In analogy to engineering design, evolution of a circadian clock is understandable in <em>Drosohpila</em>. The functional specification can be to provide robustness of the circadian cycle to various perturbations. In basic design under the constraint that the system must use genetic circuits within a cell, an interlocked feedback system is created as network architecture so that two clocks are consistently harmonized with environmental changes such as light pulse and they show robustness of the circadian period against various perturbations<a href="#pone.0001103-Ueda1">[14]</a>. Generally such robustness is provided by molecular architecture such as feedback <a href="#pone.0001103-Csete1">[3]</a>, <a href="#pone.0001103-Kurata1">[5]</a>, <a href="#pone.0001103-Stelling1">[6]</a>. Then the process of detail design further evolves kinetic parameters so as to optimize the specification in the given network structure. In terms of engineering design concept, the circadian clock is suggested to not only evolve an interlocked feedback system but also intensively design kinetic parameters for further enhanced robustness.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>General dynamic model</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Dynamic models for biochemical networks are formulated as ordinary differential equations:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>t</em> is the time, <strong>y</strong> is the vector whose elements are the differential variables for molecular concentrations, and <strong>p</strong> is the vector of kinetic parameters.</p>


<h4>Evolutionary search for all possible solutions</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The target features are determined that the dynamic model should reproduce. A fitness function must be designed so that the mathematical model reproduces the target dynamics and then all possible solutions are explored to satisfy the fitness function.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Excessive computational requirements make it impossible to explore the entire space of kinetic parameters in large-scale dynamic models. Techniques that address this problem should provide ways to reduce the number of search parameters combined with the method that allows searches within a large parameter space to proceed efficiently. In general, an evolutionary search seeks out a global minimum of certain fitness functions based on the heuristic assumption that best solutions will be found in the regions of the parameter space by using the genetic operations of selection, crossover, and mutation. However, the objective of MAR is not to find a global minimum, but to explore all possible solutions of kinetic parameter vectors that produce the target dynamics.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>MAR presents the two-step evolutionary search that combines a random search with a search by GAs <a href="#pone.0001103-Kurata2">[17]</a>, as shown in <a href="#pone-0001103-g001"><strong>Figure 1</strong></a>. First, the random search explores a large parameter space without any human biases and finds multiple coarse solutions showing a good fitness value. It is not necessary to find any solutions providing highest fitness values. The coarse solutions are employed to make initial populations for the subsequent GAs. Second, each initial population for GAs is created around one of the coarse solution vectors. The search by GAs focuses on the space surrounding the coarse solution and is applied to each initial population independently to find plausible solution vectors that show a high fitness value, i.e., provide the target features. The search is performed by various crossover methods, resulting in obtaining local solutions around the coarse solutions. The <em>i</em>-th resultant solution vector of kinetic parameters <strong>P(</strong><em>i</em><strong>)</strong> is given by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where <em>p(i, k)</em> is the value of the <em>k</em>-th parameter of the <em>i</em>-th solution vector and <em>N</em> is the number of search parameters.</p>


<h4>Characterization of exhaustive search</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The proposed two-step method explores multiple plausible solution vectors for kinetic parameters that generate target features or satisfy a fitness function. It is necessary to characterize whether the search is saturated or exhaustive. Here saturation means that the variability in all possible solutions is large enough to cover almost the entire solution space. Since numerical methods are not theoretically able to guarantee the exhaustive search, we present practically useful criteria: a parameter spectrum width and the variability in the solution vectors for parameters.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>First, a parameter spectrum is defined as the allowable range of each parameter. The range is from the minimum value to the maximum for the resultant parameter solutions. The logarithmic width of the parameter spectrum for each kinetic parameter is defined by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <em>La</em> is the number of all the solutions. The parameter spectrum width can be regarded as the indicator for a large-space search. Convergence of the width with respect to the number of generated solution vectors suggests that the search is approaching saturation. The number of the solutions at which the spectrum width begins to saturate is defined as <em>Ls</em>.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Second, to reveal the variability in the solution vectors that show the target dynamic behaviours, the distance between two <em>N-</em>dimensional solution vectors (<strong>P(</strong><em>i</em><strong>)</strong>,<strong>P(</strong><em>j</em><strong>)</strong>) is defined by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>We classify the solution vectors from <em>1</em> to <em>Ls</em> by using hierarchical clustering with the average linkage. The number of clusters (<em>M</em>) is determined by inspecting the dendrogram. Each cluster is named <em>C</em>(<em>j</em>), (<em>j</em> = <em>1,2,…M</em>). Assuming that the clusters form <em>N</em>-dimensional super-balls, we define the radius of each cluster as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where <strong>G(</strong><em>j</em><strong>)</strong> = (<em>g(j,1),g(j,2),g(j,3),…g(j,N)</em>) is the centroid for <em>C</em>(<em>j</em>) and <em>g</em>(<em>j, k</em>) is the <em>k</em>-th parameter of the centroid vector. The distances between the centroid and intra-cluster parameter vectors are calculated and the maximum distance is set to the radius of the cluster ball.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>Third, to investigate whether the solution vectors, {<strong>P(</strong><em>i</em><strong>)</strong>|<em>i = 1,2,…,La</em>}, can be assigned to one of the existing cluster balls, we define the relative minimum distance between <strong>P(</strong><em>i</em><strong>)</strong> and the centroid of each cluster as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>where the relative distance between <strong>P(</strong><em>i</em><strong>)</strong> and each <strong>G(</strong><em>j</em><strong>)</strong> is calculated and the relative minimum distance is selected as <em>MinD</em>(<strong>P</strong>(<em>i</em>)). <em>MinD</em>(<strong>P</strong>(<em>i</em>)) determines whether a solution vector <strong>P(</strong><em>i</em><strong>)</strong> belongs to one of the existing cluster balls or not. When <em>MinD</em>(<strong>P</strong>(<em>i</em>)) is less than one, <strong>P(</strong><em>i</em><strong>)</strong> belongs to one of the existing clusters. When <em>MinD</em>(<strong>P</strong>(<em>i</em>)) is greater than one, it does not any cluster.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>Finally, to display the convergence of the variability in the solution vectors, we define <em>VarS(L)</em> as the maximum distance of {<em>MinD</em>(<strong>P</strong><strong>(</strong><em>i</em><strong>)</strong>)|<em>i = 1,2,…L</em>}:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br><em>VarS(L)</em> is defined as an indicator of whether the <em>L</em> parameter solutions fall into one of the cluster-balls generated by the <em>Ls</em> solutions. When <em>VarS(L)</em> is less than one, the <em>L</em> solution vectors are assigned to one of the existing clusters, indicating that the cluster-balls contain all <em>L</em> solutions. In contrast, when <em>VarS(L)</em> is greater than one, these clusters are not able to contain all <em>L</em> solutions. Convergence of <em>VarS</em> to less than one indicates that the cluster-balls are able to cover the parameter space spanned by the <em>L</em> solution vectors. In other words, <em>Ls</em> is large enough to provide the variability in the solutions and the evolutionary search is suggested to approach saturation at a solution number of <em>Ls</em>.</p>


<h4>Numerical sensitivity to single parameter perturbation</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>It is important to note which structural characteristics of specific molecular networks are responsible for specific instances of robustness. The numerical sensitivity of a target instance to variations in a single kinetic parameter <em>p(i, k)</em> is defined as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>Sensitivity analysis contributes to finding critical reactions for determining the target feature, in which kinetic parameters showing high sensitivities are critical.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>In MAR the potentially influential parameters for determining a specific target feature are defined. The minimum sensitivity values provided by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>are compared for all kinetic parameters and the potentially influential parameters (<em>C</em>) are selected by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e021&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br>where <em>k</em> is the index of the selected parameters and σ is the threshold value that is determined from the distribution of <em>minS</em>(<em>k</em>). The threshold value depends on mathematical models or target instances. The numerical sensitivity for the potentially influential parameters cannot be reduced less than a threshold value of σ, while that for the other parameters can be reduced.</p>


<h4>Robustness to multiple parameter perturbation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>There are many possible models that produce the target instance. It raises a question of which models are close to a real biochemical model. Robustness to multiple parameter perturbation is a key criterion for model selection. The intracellular environment consistently varies with time and with changes in external stresses, which would provide uncertainty to kinetics for all intracellular molecules. To characterize the changes in a target instance with respect to simultaneous changes in all kinetic parameters, <em>Target</em>(<strong>p</strong><em><sub>r</sub></em>(<em>i</em>)) are simulated by randomly varying all kinetic parameters of <strong>p</strong>(<em>i</em>)(<em>i = 1,2,…,Ls</em>) within a specific range given by <em>ρ</em>, where <strong>p</strong><em><sub>r</sub></em>(<em>i</em>) = (<em>p<sub>r</sub>(i,1),p<sub>r</sub>(i,2),…p<sub>r</sub>(i,N)</em>) (<em>r = 1,2,…,Nr</em>) is the randomly-perturbed parameter vector and <em>Nr</em>indicates the repetition number of the random simulations. The component of <strong>p</strong><sub>r</sub>(<em>i</em>) is given by:<em>p<sub>r</sub></em>(<em>i,k</em>) = γ(<em>r,i,k</em>) · <em>p</em>(<em>i,k</em>), (<em>k</em> = <em>1,2,…,N</em>) where γ(<em>r,i,k</em>) is the uniform random value within a range of:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e022&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>By simulating <em>Target</em>(<strong>p</strong><sub>r</sub>(<em>i</em>)), the frequency distribution of <em>Target</em>(<strong>p</strong><sub>r</sub>(<em>i</em>)) occurs. To characterize its distribution, we define the mean and coefficient of variance (CV) by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e024&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>respectively. A small value of CV indicates that a distribution is narrow and the model provides a robust property to simultaneous changes in all kinetic parameters.</p>


<h4>Link of reactions to robustness</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Finally MAR combines the sensitivity distribution <em>S</em>(<em>i,k</em>) (<em>k = 1,2,…,N</em>) and <em>CV</em>_<em>Target</em>(<em>i</em>) with respect to <strong>p</strong>(<em>i</em>) to explore the critical reactions responsible for determining the target instance. For highly robust models with a small value of <em>CV</em>_<em>Target</em>(<em>i</em>), the parameters showing a high sensitivity are selected as critical parameters. Use of statistical methods investigates how the selected critical parameters feature highly robust models, thereby linking reactions to robustness.</p>


<h4>Biochemical model</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Circadian rhythms are defined as endogenously generated 24-h variations in behaviour and physiology that allow organisms to adapt to the varying environmental demands of the solar cycle. In <em>Drosophila</em>, oscillating levels of period (<em>per</em>) and timeless (<em>tim</em>) constitute the fly's molecular clock. The circadian feedbacks are composed of two interlocked negative feedback loops: a <em>per-tim</em> loop, which is activated by <em>Drosophila</em> clock (<em>dclk</em>) and repressed by PER:TIM, and a <em>dclk-cyc</em> loop, which is repressed by dCLK:CYC and derepressed by PER:TIM<a href="#pone.0001103-Glossop1">[12]</a>. A loop that connects <em>per-tim</em> to <em>dclk-cyc</em> represents a positive feedback loop. <a href="#pone-0001103-g002"><strong>Figure 2</strong></a> shows a schematic diagram of the circadian clock, with the notation of the reactions described elsewhere <a href="#pone.0001103-Kurata3">[19]</a>. PER and TIM form dimers and then inhibit transcription of their own genes. Expression of the <em>per</em> and <em>tim</em> genes is regulated by a pair of transcription factors, dCLK and CYC, whose activity is decreased by forming a complex with PER:TIM <a href="#pone.0001103-Stanewsky1">[20]</a>. Phosphorylation and proteolysis of PER make a time-delayed negative feedback, generating oscillations. PER is phosphorylated by DBT (encoded by the double-time (<em>dbt</em>) gene), which is present at roughly constant levels during the rhythm <a href="#pone.0001103-Suri1">[21]</a>. Phosphorylated PER (PER-P) is readily susceptible to degradation, while PER-P is stabilized by dimerization with TIM. The dimerized PER-P:TIM is transported to the nucleus. Transcription of <em>dclk</em> and <em>cyc</em> is regulated by PER-P:TIM-mediated release of dCLK:CYC-dependent repression.</p>


<h4>Mathematical model</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>We have derived mathematical equations from the interlocked-feedback model developed by Ueda et al. <a href="#pone.0001103-Ueda1">[14]</a>. In Ueda's model, while PER directly binds to TIM, it is not considered that phosphorylated PER binds to TIM. Thus, we present the modified mathematical model that includes the mechanism for DBT-phosphorylated PER binding to TIM (<a href="#pone-0001103-t001"><strong>Tables 1</strong></a><strong>–</strong><a href="#pone-0001103-t002"></a><a href="#pone-0001103-t003"><strong>3</strong></a>). DBT does not appear in the model as it is assumed that the DBT concentration is constant. The modified model is described by 11 ordinary differential equations. The CADLIVE system is employed for numerical simulation (<a href="http://www.cadlive.jp">http://www.cadlive.jp</a>) <a href="#pone.0001103-Kurata2">[17]</a>.</p>


<h4>Optimization for the mathematical model</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>The dynamic model of the circadian system has 36 kinetic parameters to search as shown in <a href="#pone-0001103-t003"><strong>Table 3</strong></a>, where <em>T[2], T[4], K[3],</em> and <em>K[5]</em> are set to zero to reduce the number of search parameters. Twelve parameters (<em>S[1], A[1], A[2], R[1], R[2], V[1], V[2], D[1], P[1], T[1], K[1], K[2]</em>) that are related to feedback architectures were explored using the evolutionary search in order to generate stable oscillators for PER. These parameters involve the transcription, phosphorylation, transport, and degradation of PER and the binding between PER-P and TIM. The values of the remaining parameters are estimated or provided from the previous model <a href="#pone.0001103-Ueda1">[14]</a>.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>Based on the two-step method we randomly vary the values of 12 parameters in logarithmic space to find parameter solutions that indicate some oscillations, where a basis parameter vector is determined that produces a circadian oscillator and the value of each basis parameter is 10<sup>−2</sup> to 10<sup>2</sup>-fold varied. Such solutions will be employed as the initial population for the subsequent search by GAs. In the random search, an oscillation curve of PER is sampled after a simulation time of 300 h and the fitness function is provided by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e025&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>where <em>X<sub>min</sub></em> is the minimum value of an oscillation curve of PER and <em>X<sub>max</sub></em> is its maximum. At this stage, it is important to find the coarse solutions showing some oscillations because GAs cannot evolve non-oscillatory solutions into stable oscillators. The initial population with 100 individuals is created around the coarse solution that shows some oscillatory behaviors.</p>
<a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>Next, to search for the solutions that show stable oscillations with a 24-h period and large amplitude, GAs are carried out in logarithmic space, where each coarse solution obtained by the random search is set to the basis parameters and the value of each basis parameter is 10<sup>−1</sup> to 10-fold varied. Twelve kinetic parameters are optimized where the generations and population are set to 100 and 100, respectively. The Unimordal Normal Distribution Crossover (UNDX) is employed as crossover <a href="#pone.0001103-Kurata2">[17]</a>, while mutation is not used. Twenty elites are selected for each generation. The cycle features provide the fitness function:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br>where <em>T</em> is the period. <strong>Equation 15</strong> is an experience formula that generates the 24-h cycles whose amplitude is large enough for the average of the oscillatory curves. A target PER oscillator is that the ratio of the amplitude to the mean concentration of PER (the first term in <strong>Equation 15</strong> is 1.8 (0.9×2), the amplitude (the second term) is 10 nM, and the period is between 23 and 25 h.</p>


<h4>Robustness analysis for the biochemical model</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>The numerical sensitivity of period (<em>S<sub>T</sub></em>) or amplitude (<em>S<sub>A</sub></em>) to variations in a single kinetic parameter is calculated by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001103.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br>where <em>Ls</em> and <em>N</em> are set to 400 and 36, respectively. Each kinetic parameter was changed by 1.5-fold. In robustness analysis to multiple parameter perturbation, characterized by the CVs of period and amplitude, the period and amplitude are simulated 10,000 times by randomly varying all kinetic parameters within a range from 1/1.5-fold to 1.5-fold (<em>ρ</em> = 0.5) for <em>Ls</em> models. The means and CVs: <em>mean_Period(i)</em>, <em>CV_Period(i)</em>, <em>mean_Amplituede(i)</em>, and <em>CV_Amplitude(i)</em>, are calculated for all the distributions(<em>i</em> = 1,2,…<em>Ls</em>).</p>


<h4>Computation</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>Since a search parameter space is very large, the obigrid system (RIKEN, Yokohama) is used to greatly increase a calculation speed, where dozens of computers are available simultaneously<a href="#pone.0001103-Kimura1">[22]</a>. Dynamic simulation programs are written in C. The Runge-Kutta method is employed.</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>A hierarchical clustering method with the average distance is used to classify parameter solution vectors. The Matlab functions, linkage and dendrogram, are employed to determine the number of clusters. Other statistical analyses are also carried out by Matlab.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001103"></div><a name="pone.0001103.s001" id="pone.0001103.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s001">TextS1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.08 MB PDF)</p>
<a name="pone.0001103.s002" id="pone.0001103.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s002">TableS1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.13 MB PDF)</p>
<a name="pone.0001103.s003" id="pone.0001103.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s003">TableS2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0001103.s004" id="pone.0001103.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s004">FigureS1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0001103.s005" id="pone.0001103.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s005">FigureS2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.09 MB PDF)</p>
<a name="pone.0001103.s006" id="pone.0001103.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s006">FigureS3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0001103.s007" id="pone.0001103.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s007">FigureS4. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(0.12 MB PDF)</p>
<a name="pone.0001103.s008" id="pone.0001103.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s008">FigureS5. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="postSiDOI">(0.14 MB PDF)</p>
<a name="pone.0001103.s009" id="pone.0001103.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s009">FigureS6. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0001103.s010" id="pone.0001103.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s010">FigureS7. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0001103.s011" id="pone.0001103.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s011">FigureS8. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="postSiDOI">(0.10 MB PDF)</p>
<a name="pone.0001103.s012" id="pone.0001103.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001103.s012">FigureS9. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001103.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="postSiDOI">(0.12 MB PDF)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: HK. Performed the experiments: HK TT FO. Analyzed the data: HK TT. Contributed reagents/materials/analysis tools: HK TT FO. Wrote the paper: HK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001103-Savageau1" id="pone.0001103-Savageau1"></a>Savageau MA (1971) Concepts relating the behavior of biochemical systems to their underlying molecular properties. Arch Biochem Biophys  145: 612–621.  <ul class="find" data-citedArticleID="1011356" data-doi="10.1016/s0003-9861(71)80021-8"><li><a href="http://dx.doi.org/10.1016/s0003-9861(71)80021-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concepts+relating+the+behavior+of+biochemical+systems+to+their+underlying+molecular+properties." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concepts+relating+the+behavior+of+biochemical+systems+to+their+underlying+molecular+properties.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001103-Savageau2" id="pone.0001103-Savageau2"></a>Savageau MA (1971) Parameter sensitivity as a criterion for evaluating and comparing the performance of biochemical systems. Nature  229: 542–544.  <ul class="find" data-citedArticleID="1011358" data-doi="10.1038/229542a0"><li><a href="http://dx.doi.org/10.1038/229542a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parameter+sensitivity+as+a+criterion+for+evaluating+and+comparing+the+performance+of+biochemical+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parameter+sensitivity+as+a+criterion+for+evaluating+and+comparing+the+performance+of+biochemical+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001103-Csete1" id="pone.0001103-Csete1"></a>Csete ME, Doyle JC (2002) Reverse engineering of biological complexity. Science  295: 1664–1669.  <ul class="find" data-citedArticleID="1011334" data-doi="10.1126/science.1069981"><li><a href="http://dx.doi.org/10.1126/science.1069981" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+engineering+of+biological+complexity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+engineering+of+biological+complexity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001103-ElSamad1" id="pone.0001103-ElSamad1"></a>El-Samad H, Kurata H, Doyle JC, Gross CA, Khammash M (2005) Surviving heat shock: control strategies for robustness and performance. Proc Natl Acad Sci U S A  102: 2736–2741.  <ul class="find" data-citedArticleID="1011336" data-doi="10.1073/pnas.0403510102"><li><a href="http://dx.doi.org/10.1073/pnas.0403510102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Surviving+heat+shock%3A+control+strategies+for+robustness+and+performance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Surviving+heat+shock%3A+control+strategies+for+robustness+and+performance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001103-Kurata1" id="pone.0001103-Kurata1"></a>Kurata H, El-Samad H, Iwasaki R, Ohtake H, Doyle JC, et al.  (2006) Module-based analysis of robustness tradeoffs in the heat shock response system. PLoS Comput Biol  2: e59.  <ul class="find" data-citedArticleID="1011344" data-doi="10.1371/journal.pcbi.0020059.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020059.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Module-based+analysis+of+robustness+tradeoffs+in+the+heat+shock+response+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Module-based+analysis+of+robustness+tradeoffs+in+the+heat+shock+response+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001103-Stelling1" id="pone.0001103-Stelling1"></a>Stelling J, Sauer U, Szallasi Z, Doyle FJ 3rd, Doyle J (2004) Robustness of cellular functions. Cell  118: 675–685.  <ul class="find" data-citedArticleID="1011364" data-doi="10.1016/j.cell.2004.09.008"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.09.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+of+cellular+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+of+cellular+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001103-Ma1" id="pone.0001103-Ma1"></a>Ma L, Iglesias PA (2002) Quantifying robustness of biochemical network models. BMC Bioinformatics  3: 38.  <ul class="find" data-citedArticleID="1011352"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quantifying+robustness+of+biochemical+network+models.&amp;auth=&amp;atitle=Quantifying+robustness+of+biochemical+network+models." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantifying+robustness+of+biochemical+network+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantifying+robustness+of+biochemical+network+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001103-Bagheri1" id="pone.0001103-Bagheri1"></a>Bagheri N, Stelling J, Doyle FJ 3rd (2007) Quantitative performance metrics for robustness in circadian rhythms. Bioinformatics  23: 358–364.  <ul class="find" data-citedArticleID="1011330" data-doi="10.1093/bioinformatics/btl627"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl627" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+performance+metrics+for+robustness+in+circadian+rhythms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+performance+metrics+for+robustness+in+circadian+rhythms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001103-Alves1" id="pone.0001103-Alves1"></a>Alves R, Savageau MA (2000) Extending the method of mathematically controlled comparison to include numerical comparisons. Bioinformatics  16: 786–798.  <ul class="find" data-citedArticleID="1011328" data-doi="10.1093/bioinformatics/16.9.786"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.9.786" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extending+the+method+of+mathematically+controlled+comparison+to+include+numerical+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extending+the+method+of+mathematically+controlled+comparison+to+include+numerical+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001103-Stelling2" id="pone.0001103-Stelling2"></a>Stelling J, Gilles ED, Doyle FJ 3rd (2004) Robustness properties of circadian clock architectures. Proc Natl Acad Sci U S A  101: 13210–13215.  <ul class="find" data-citedArticleID="1011366" data-doi="10.1073/pnas.0401463101"><li><a href="http://dx.doi.org/10.1073/pnas.0401463101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+properties+of+circadian+clock+architectures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+properties+of+circadian+clock+architectures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001103-Leloup1" id="pone.0001103-Leloup1"></a>Leloup JC, Goldbeter A (2000) Modeling the molecular regulatory mechanism of circadian rhythms in <em>Drosophila</em>. Bioessays  22: 84–93.  <ul class="find" data-citedArticleID="1011350" data-doi="10.1002/(sici)1521-1878(200001)22:1<84::aid-bies13&gt;3.0.co;2-i"><li><a href="http://dx.doi.org/10.1002/(sici)1521-1878(200001)22:1<84::aid-bies13&gt;3.0.co;2-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+the+molecular+regulatory+mechanism+of+circadian+rhythms+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+the+molecular+regulatory+mechanism+of+circadian+rhythms+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001103-Glossop1" id="pone.0001103-Glossop1"></a>Glossop NR, Lyons LC, Hardin PE (1999) Interlocked feedback loops within the Drosophila circadian oscillator. Science  286: 766–768.  <ul class="find" data-citedArticleID="1011340" data-doi="10.1126/science.286.5440.766"><li><a href="http://dx.doi.org/10.1126/science.286.5440.766" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interlocked+feedback+loops+within+the+Drosophila+circadian+oscillator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interlocked+feedback+loops+within+the+Drosophila+circadian+oscillator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001103-Cheng1" id="pone.0001103-Cheng1"></a>Cheng P, Yang Y, Liu Y (2001) Interlocked feedback loops contribute to the robustness of the <em>Neurospora</em> circadian clock. Proc Natl Acad Sci U S A  98: 7408–7413.  <ul class="find" data-citedArticleID="1011332" data-doi="10.1073/pnas.121170298"><li><a href="http://dx.doi.org/10.1073/pnas.121170298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interlocked+feedback+loops+contribute+to+the+robustness+of+the+Neurospora+circadian+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interlocked+feedback+loops+contribute+to+the+robustness+of+the+Neurospora+circadian+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001103-Ueda1" id="pone.0001103-Ueda1"></a>Ueda HR, Hagiwara M, Kitano H (2001) Robust oscillations within the interlocked feedback model of <em>Drosophila</em> circadian rhythm. J Theor Biol  210: 401–406.  <ul class="find" data-citedArticleID="1011370" data-doi="10.1006/jtbi.2000.2226"><li><a href="http://dx.doi.org/10.1006/jtbi.2000.2226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+oscillations+within+the+interlocked+feedback+model+of+Drosophila+circadian+rhythm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+oscillations+within+the+interlocked+feedback+model+of+Drosophila+circadian+rhythm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001103-Ono1" id="pone.0001103-Ono1"></a>Ono I, Kobayashi S (1997) A Real-coded Genetic Algorithm for Function Optimization Using Unimodal Normal Distribution Crossover. Proc of 7th Int Conf on Genetic Algorithms 246–253.  <ul class="find" data-citedArticleID="1011354" data-doi="10.1007/978-3-642-18965-4_8"><li><a href="http://dx.doi.org/10.1007/978-3-642-18965-4_8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Real-coded+Genetic+Algorithm+for+Function+Optimization+Using+Unimodal+Normal+Distribution+Crossover." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Real-coded+Genetic+Algorithm+for+Function+Optimization+Using+Unimodal+Normal+Distribution+Crossover.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001103-Eshelman1" id="pone.0001103-Eshelman1"></a>Eshelman LJ, Schaffer JD (1993)  Real-coded genetic algorithms and interval schemata. In: Whitley LD, editor. Foundations of Genetic Algorithms. San Fransisco: Morgan Kaufmann. pp. 187–202.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0001103-Kurata2" id="pone.0001103-Kurata2"></a>Kurata H, Masaki K, Sumida Y, Iwasaki R (2005) CADLIVE Dynamic Simulator: Direct Link of Biochemical Networks to Dynamic Models. Genome Res  15: 590–600.  <ul class="find" data-citedArticleID="1011346" data-doi="10.1101/gr.3463705"><li><a href="http://dx.doi.org/10.1101/gr.3463705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CADLIVE+Dynamic+Simulator%3A+Direct+Link+of+Biochemical+Networks+to+Dynamic+Models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CADLIVE+Dynamic+Simulator%3A+Direct+Link+of+Biochemical+Networks+to+Dynamic+Models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001103-Smolen1" id="pone.0001103-Smolen1"></a>Smolen P, Baxter DA, Byrne JH (2001) Modeling circadian oscillations with interlocking positive and negative feedback loops. J Neurosci  21: 6644–6656.  <ul class="find" data-citedArticleID="1011360"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modeling+circadian+oscillations+with+interlocking+positive+and+negative+feedback+loops.&amp;auth=&amp;atitle=Modeling+circadian+oscillations+with+interlocking+positive+and+negative+feedback+loops." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+circadian+oscillations+with+interlocking+positive+and+negative+feedback+loops." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+circadian+oscillations+with+interlocking+positive+and+negative+feedback+loops.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001103-Kurata3" id="pone.0001103-Kurata3"></a>Kurata H, Matoba N, Shimizu N (2003) CADLIVE for constructing a large-scale biochemical network based on a simulation-directed notation and its application to yeast cell cycle. Nucleic Acids Res  31: 4071–4084.  <ul class="find" data-citedArticleID="1011348" data-doi="10.1093/nar/gkg461"><li><a href="http://dx.doi.org/10.1093/nar/gkg461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CADLIVE+for+constructing+a+large-scale+biochemical+network+based+on+a+simulation-directed+notation+and+its+application+to+yeast+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CADLIVE+for+constructing+a+large-scale+biochemical+network+based+on+a+simulation-directed+notation+and+its+application+to+yeast+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001103-Stanewsky1" id="pone.0001103-Stanewsky1"></a>Stanewsky R (2003) Genetic analysis of the circadian system in <em>Drosophila melanogaster</em> and mammals. J Neurobiol  54: 111–147.  <ul class="find" data-citedArticleID="1011362" data-doi="10.1002/neu.10164"><li><a href="http://dx.doi.org/10.1002/neu.10164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analysis+of+the+circadian+system+in+Drosophila+melanogaster+and+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analysis+of+the+circadian+system+in+Drosophila+melanogaster+and+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001103-Suri1" id="pone.0001103-Suri1"></a>Suri V, Hall JC, Rosbash M (2000) Two novel doubletime mutants alter circadian properties and eliminate the delay between RNA and protein in <em>Drosophila</em>. J Neurosci  20: 7547–7555.  <ul class="find" data-citedArticleID="1011368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Two+novel+doubletime+mutants+alter+circadian+properties+and+eliminate+the+delay+between+RNA+and+protein+in+Drosophila.&amp;auth=&amp;atitle=Two+novel+doubletime+mutants+alter+circadian+properties+and+eliminate+the+delay+between+RNA+and+protein+in+Drosophila." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+novel+doubletime+mutants+alter+circadian+properties+and+eliminate+the+delay+between+RNA+and+protein+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+novel+doubletime+mutants+alter+circadian+properties+and+eliminate+the+delay+between+RNA+and+protein+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001103-Kimura1" id="pone.0001103-Kimura1"></a>Kimura S, Kawasaki T, Hatakeyama M, Naka T, Konishi F, et al.  (2004) OBIYagns: a grid-based biochemical simulator with a parameter estimator. Bioinformatics  20: 1646–1648.  <ul class="find" data-citedArticleID="1011342" data-doi="10.1093/bioinformatics/bth122"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OBIYagns%3A+a+grid-based+biochemical+simulator+with+a+parameter+estimator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OBIYagns%3A+a+grid-based+biochemical+simulator+with+a+parameter+estimator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.XML" value="110127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.PDF" value="691077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g001.PNG_L" value="72546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g001.PNG_M" value="126727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g001.PNG_S" value="11911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g001.TIF" value="337704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g001.PNG_I" value="46573"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g002.PNG_L" value="133765"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g002.PNG_M" value="146330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g002.PNG_S" value="11976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g002.TIF" value="594276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g002.PNG_I" value="72889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t001.PNG_L" value="240998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t001.PNG_M" value="134356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t001.PNG_S" value="12961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t001.TIF" value="1118290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t001.PNG_I" value="41889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e001.PNG" value="19219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e001.TIF" value="38352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e002.PNG" value="11685"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e002.TIF" value="28232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e003.PNG" value="12489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e003.TIF" value="29740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e004.PNG" value="15328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e004.TIF" value="31412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e005.PNG" value="14681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e005.TIF" value="33848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e006.PNG" value="20533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e006.TIF" value="37428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e007.PNG" value="20682"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e007.TIF" value="40716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e008.PNG" value="15492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e008.TIF" value="31372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e009.PNG" value="15458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e009.TIF" value="34276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e010.PNG" value="19808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e010.TIF" value="36644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e011.PNG" value="21358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e011.TIF" value="41360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t002.PNG_L" value="135244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t002.PNG_M" value="174114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t002.PNG_S" value="20455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t002.TIF" value="588770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t002.PNG_I" value="63035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t003.PNG_L" value="398399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t003.PNG_M" value="178728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t003.PNG_S" value="17032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t003.TIF" value="1833028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.t003.PNG_I" value="69484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g003.PNG_L" value="20580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g003.PNG_M" value="62627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g003.PNG_S" value="9301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g003.TIF" value="79082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g003.PNG_I" value="24800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g004.PNG_L" value="17888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g004.PNG_M" value="40947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g004.PNG_S" value="8597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g004.TIF" value="75876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g004.PNG_I" value="15805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g005.PNG_L" value="34809"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g005.PNG_M" value="86224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g005.PNG_S" value="9053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g005.TIF" value="114258"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g005.PNG_I" value="32430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g006.PNG_L" value="44758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g006.PNG_M" value="24222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g006.PNG_S" value="6637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g006.TIF" value="95710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g006.PNG_I" value="9009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g007.PNG_L" value="42779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g007.PNG_M" value="24103"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g007.PNG_S" value="6514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g007.TIF" value="99290"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g007.PNG_I" value="8732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g008.PNG_L" value="207014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g008.PNG_M" value="47618"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g008.PNG_S" value="9750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g008.TIF" value="511626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g008.PNG_I" value="30438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g009.PNG_L" value="41438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g009.PNG_M" value="79159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g009.PNG_S" value="10387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g009.TIF" value="240242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.g009.PNG_I" value="29918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e012.PNG" value="7314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e012.TIF" value="23268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e013.PNG" value="9636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e013.TIF" value="25804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e014.PNG" value="16059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e014.TIF" value="32600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e015.PNG" value="14485"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e015.TIF" value="30960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e016.PNG" value="10891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e016.TIF" value="27572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e017.PNG" value="19524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e017.TIF" value="35936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e018.PNG" value="9437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e018.TIF" value="25728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e019.PNG" value="10761"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e019.TIF" value="26700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e020.PNG" value="9236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e020.TIF" value="25540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e021.PNG" value="8306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e021.TIF" value="24268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e022.PNG" value="8657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e022.TIF" value="24628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e023.PNG" value="10818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e023.TIF" value="26680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e024.PNG" value="14456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e024.TIF" value="30284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e025.PNG" value="10857"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e025.TIF" value="26432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e026.PNG" value="16675"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e026.TIF" value="32804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e027.PNG" value="16484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.e027.TIF" value="32420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s001.PDF" value="84075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s002.PDF" value="130449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s003.PDF" value="105169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s004.PDF" value="108206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s005.PDF" value="87123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s006.PDF" value="101024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s007.PDF" value="117473"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s008.PDF" value="140296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s009.PDF" value="114087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s010.PDF" value="98333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s011.PDF" value="98310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001103.s012.PDF" value="122038"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001103&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001103" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001103&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001103&volume=&issue=&title=Mathematical Identification of Critical Reactions in the Interlocked Feedback Model&author_name=Hiroyuki%20Kurata%2C%20Takayuki%20Tanaka%2C%20Fumitaka%20Ohnishi&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001103" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103&amp;t=Mathematical%20Identification%20of%20Critical%20Reactions%20in%20the%20Interlocked%20Feedback%20Model" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103&title=Mathematical%20Identification%20of%20Critical%20Reactions%20in%20the%20Interlocked%20Feedback%20Model&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103&amp;title=Mathematical%20Identification%20of%20Critical%20Reactions%20in%20the%20Interlocked%20Feedback%20Model" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001103&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mathematical Identification of Critical Reactions in the Interlocked Feedback Model';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001103';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mathematical%20Identification%20of%20Critical%20Reactions%20in%20the%20Interlocked%20Feedback%20Model http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001103" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001103" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biochemical+simulations%22" title="Search for articles in the subject area:'Biochemical simulations'"><div class="flagText">Biochemical simulations</div></a>
              <div data-categoryid="17939" data-articleid="25040"
                   data-categoryname="Biochemical simulations"
                   class="flagImage" title="Flag 'Biochemical simulations' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biophysical+simulations%22" title="Search for articles in the subject area:'Biophysical simulations'"><div class="flagText">Biophysical simulations</div></a>
              <div data-categoryid="34601" data-articleid="25040"
                   data-categoryname="Biophysical simulations"
                   class="flagImage" title="Flag 'Biophysical simulations' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Circadian+oscillators%22" title="Search for articles in the subject area:'Circadian oscillators'"><div class="flagText">Circadian oscillators</div></a>
              <div data-categoryid="32325" data-articleid="25040"
                   data-categoryname="Circadian oscillators"
                   class="flagImage" title="Flag 'Circadian oscillators' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Circadian+rhythms%22" title="Search for articles in the subject area:'Circadian rhythms'"><div class="flagText">Circadian rhythms</div></a>
              <div data-categoryid="19705" data-articleid="25040"
                   data-categoryname="Circadian rhythms"
                   class="flagImage" title="Flag 'Circadian rhythms' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Convergent+evolution%22" title="Search for articles in the subject area:'Convergent evolution'"><div class="flagText">Convergent evolution</div></a>
              <div data-categoryid="20355" data-articleid="25040"
                   data-categoryname="Convergent evolution"
                   class="flagImage" title="Flag 'Convergent evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mathematical+models%22" title="Search for articles in the subject area:'Mathematical models'"><div class="flagText">Mathematical models</div></a>
              <div data-categoryid="18139" data-articleid="25040"
                   data-categoryname="Mathematical models"
                   class="flagImage" title="Flag 'Mathematical models' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphorylation%22" title="Search for articles in the subject area:'Phosphorylation'"><div class="flagText">Phosphorylation</div></a>
              <div data-categoryid="42723" data-articleid="25040"
                   data-categoryname="Phosphorylation"
                   class="flagImage" title="Flag 'Phosphorylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Solutions%22" title="Search for articles in the subject area:'Solutions'"><div class="flagText">Solutions</div></a>
              <div data-categoryid="20047" data-articleid="25040"
                   data-categoryname="Solutions"
                   class="flagImage" title="Flag 'Solutions' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3488'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4503'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4244&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
