

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001344"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001344" />

    <meta name="citation_title" content="Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders"/>
    <meta itemprop="name" content="Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders"/>

      <meta name="citation_author" content="David M. Mutch"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, UMR S 872, Paris, France"/>
      <meta name="citation_author" content="M. Ramzi Temanni"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Laboratoire d'Informatique Medicale and Bio-Informatique (LIM&BIO) EA3969, Paris Nord University, Bobigny, France"/>
      <meta name="citation_author" content="Corneliu Henegar"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, UMR S 872, Paris, France"/>
      <meta name="citation_author" content="Florence Combes"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, UMR S 872, Paris, France"/>
      <meta name="citation_author" content="Véronique Pelloux"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Assistance Publique-Hôpitaux de Paris (AP-HP), Pitié Salpêtrière Hospital, Department of Nutrition and Endocrinology, Centre de Recherche en Nutrition Humaine Ile de France (CRNH, Idf), Paris, France"/>
      <meta name="citation_author" content="Claus Holst"/>
            <meta name="citation_author_institution" content="Centre for Health and Society, Institute of Preventive Medicine, Copenhagen University Hospital, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Thorkild I. A. Sørensen"/>
            <meta name="citation_author_institution" content="Centre for Health and Society, Institute of Preventive Medicine, Copenhagen University Hospital, Copenhagen, Denmark"/>
      <meta name="citation_author" content="Arne Astrup"/>
            <meta name="citation_author_institution" content="Department of Human Nutrition, Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark"/>
      <meta name="citation_author" content="J. Alfredo Martinez"/>
            <meta name="citation_author_institution" content="Department of Physiology and Nutrition, University of Navarra, Pamplona, Spain"/>
      <meta name="citation_author" content="Wim H. M. Saris"/>
            <meta name="citation_author_institution" content="Department of Human Biology, NUTRIM, Maastricht University, Maastricht, The Netherlands"/>
      <meta name="citation_author" content="Nathalie Viguerie"/>
            <meta name="citation_author_institution" content="Inserm U858, Institut de Médecine Moléculaire de Rangueil, Laboratoire de recherches sur les obésités, Toulouse, France"/>
            <meta name="citation_author_institution" content="Institut Louis Bugnard, Université Paul Sabatier, IFR31, Toulouse, France"/>
      <meta name="citation_author" content="Dominique Langin"/>
            <meta name="citation_author_institution" content="Inserm U858, Institut de Médecine Moléculaire de Rangueil, Laboratoire de recherches sur les obésités, Toulouse, France"/>
            <meta name="citation_author_institution" content="Institut Louis Bugnard, Université Paul Sabatier, IFR31, Toulouse, France"/>
            <meta name="citation_author_institution" content="Centre Hospitalier Universitaire (CHU) de Toulouse, Laboratoire de biochimie, Institut Fédératif de Biologie de Purpan, Toulouse, France"/>
      <meta name="citation_author" content="Jean-Daniel Zucker"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, UMR S 872, Paris, France"/>
      <meta name="citation_author" content="Karine Clément"/>
            <meta name="citation_author_institution" content="INSERM, Nutriomique U872, Paris, France"/>
            <meta name="citation_author_institution" content="Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Université Paris Descartes, UMR S 872, Paris, France"/>
            <meta name="citation_author_institution" content="Assistance Publique-Hôpitaux de Paris (AP-HP), Pitié Salpêtrière Hospital, Department of Nutrition and Endocrinology, Centre de Recherche en Nutrition Humaine Ile de France (CRNH, Idf), Paris, France"/>

    <meta name="citation_date" content="2007/12/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001344.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1344"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Nutrigenomics and nutrigenetics: the emerging faces of nutrition.; citation_author=DM Mutch; citation_author=W Wahli; citation_author=G Williamson; citation_journal_title=FASEB J; citation_volume=19; citation_number=1; citation_pages=1602-1616; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The case for strategic international alliances to harness nutritional genomics for public and personal health.; citation_author=J Kaput; citation_author=JM Ordovas; citation_author=L Ferguson; citation_author=B van Ommen; citation_author=RL Rodriguez; citation_journal_title=Br J Nutr; citation_volume=94; citation_number=2; citation_pages=623-632; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Challenges of drug discovery for personalized medicine.; citation_author=KK Jain; citation_journal_title=Curr Opin Mol Ther; citation_volume=8; citation_number=3; citation_pages=487-492; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The use of transcriptomic biomarkers for personalized medicine.; citation_author=B Heidecker; citation_author=JM Hare; citation_journal_title=Heart Fail Rev; citation_volume=12; citation_number=4; citation_pages=1-11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.; citation_author=TR Golub; citation_author=DK Slonim; citation_author=P Tamayo; citation_author=C Huard; citation_author=M Gaasenbeek; citation_journal_title=Science; citation_volume=286; citation_number=5; citation_pages=531-537; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Diffuse large B-cell lymphoma outcome prediction by gene-expression profiling and supervised machine learning.; citation_author=MA Shipp; citation_author=KN Ross; citation_author=P Tamayo; citation_author=AP Weng; citation_author=JL Kutok; citation_journal_title=Nat Med; citation_volume=8; citation_number=6; citation_pages=68-74; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma.; citation_author=A Rosenwald; citation_author=G Wright; citation_author=WC Chan; citation_author=JM Connors; citation_author=E Campo; citation_journal_title=N Engl J Med; citation_volume=346; citation_number=7; citation_pages=1937-1947; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling identifies clinically relevant subtypes of prostate cancer.; citation_author=J Lapointe; citation_author=C Li; citation_author=JP Higgins; citation_author=RM van de; citation_author=E Bair; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=8; citation_pages=811-816; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prediction of central nervous system embryonal tumour outcome based on gene expression.; citation_author=SL Pomeroy; citation_author=P Tamayo; citation_author=M Gaasenbeek; citation_author=LM Sturla; citation_author=M Angelo; citation_journal_title=Nature; citation_volume=415; citation_number=9; citation_pages=436-442; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling predicts clinical outcome of breast cancer.; citation_author=L van't Veer; citation_author=H Dai; citation_author=V van d; citation_author=YD He; citation_author=AA Hart; citation_journal_title=Nature; citation_volume=415; citation_number=10; citation_pages=530-536; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identifying regulatory hubs in obesity with nutrigenomics.; citation_author=DM Mutch; citation_journal_title=Curr Opin Endocrinol Diabetes; citation_volume=13; citation_number=11; citation_pages=431-437; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptomics applied to obesity and caloric restriction.; citation_author=N Viguerie; citation_author=C Poitou; citation_author=R Cancello; citation_author=V Stich; citation_author=K Clement; citation_journal_title=Biochimie; citation_volume=87; citation_number=12; citation_pages=117-123; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Unraveling the genetics of human obesity.; citation_author=DM Mutch; citation_author=K Clement; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=13; citation_pages=e188; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene expression profiling of adipose tissue: individual, depot-dependent, and sex-dependent variabilities.; citation_author=S Klaus; citation_author=J Keijer; citation_journal_title=Nutrition; citation_volume=20; citation_number=14; citation_pages=115-120; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Does weight loss prognosis depend on genetic make-up?; citation_author=MJ Moreno-Aliaga; citation_author=JL Santos; citation_author=A Marti; citation_author=JA Martinez; citation_journal_title=Obes Rev; citation_volume=6; citation_number=15; citation_pages=155-168; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Improved prediction of treatment response using microarrays and existing biological knowledge.; citation_author=SM Lin; citation_author=J Devakumar; citation_author=WA Kibbe; citation_journal_title=Pharmacogenomics; citation_volume=7; citation_number=16; citation_pages=495-501; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Outcome signature genes in breast cancer: is there a unique set?; citation_author=L Ein-Dor; citation_author=I Kela; citation_author=G Getz; citation_author=D Givol; citation_author=E Domany; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=17; citation_pages=171-178; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarrays for cancer diagnosis and classification.; citation_author=A Perez-Diez; citation_author=A Morgun; citation_author=N Shulzhenko; citation_journal_title=Adv Exp Med Biol; citation_volume=593; citation_number=18; citation_pages=74-85; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Adipose tissue gene expression in obese subjects during low-fat and high-fat hypocaloric diets.; citation_author=N Viguerie; citation_author=H Vidal; citation_author=P Arner; citation_author=C Holst; citation_author=C Verdich; citation_journal_title=Diabetologia; citation_volume=48; citation_number=19; citation_pages=123-131; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic Polymorphisms and Weight Loss in Obesity: A Randomised Trial of Hypo-Energetic High- versus Low-Fat Diets.; citation_author=TIA Sorensen; citation_author=P Boutin; citation_author=MA Taylor; citation_author=LH Larsen; citation_author=C Verdich; citation_journal_title=PLoS Clinical Trials; citation_volume=1; citation_number=20; citation_pages=e12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Weight loss regulates inflammation-related genes in white adipose tissue of obese subjects.; citation_author=K Clement; citation_author=N Viguerie; citation_author=C Poitou; citation_author=C Carette; citation_author=V Pelloux; citation_journal_title=FASEB J; citation_volume=18; citation_number=21; citation_pages=1657-1669; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Introduction to Statistical Methods for Microarray Data Analysis.; citation_author=T Mary-Huard; citation_author=F Picard; citation_author=S Robin; citation_number=22; citation_date=2006; citation_publisher=Hermann; " />
      <meta name="citation_reference" content="citation_title=Classification in microarray experiments.; citation_author=S Dudoit; citation_author=J Fridlyand; citation_number=23; citation_pages=93-158; citation_date=2003; citation_publisher=Chapman & Hall; " />
      <meta name="citation_reference" content="citation_title=Gene selection and classification of microarray data using random forest.; citation_author=R Diaz-Uriarte; citation_author=S Alvarez de Andres; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=24; citation_pages=3; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Classification and regression by randomForest.; citation_author=A Liaw; citation_author=M Wiener; citation_journal_title=Rnews; citation_volume=2; citation_number=25; citation_pages=18-22; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Random Forests.; citation_author=L Breiman; citation_journal_title=Mach Learn; citation_volume=40; citation_number=26; citation_pages=5-32; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Systematic variation in gene expression patterns in human cancer cell lines.; citation_author=DT Ross; citation_author=U Scherf; citation_author=MB Eisen; citation_author=CM Perou; citation_author=C Rees; citation_journal_title=Nat Genet; citation_volume=24; citation_number=27; citation_pages=227-235; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Support vector machine classification and validation of cancer tissue samples using microarray expression data.; citation_author=TS Furey; citation_author=N Cristianini; citation_author=N Duffy; citation_author=DW Bednarski; citation_author=M Schummer; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=28; citation_pages=906-914; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Classification of multiple cancer types by multicategory support vector machines using gene expression data.; citation_author=Y Lee; citation_author=CK Lee; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=29; citation_pages=1132-1139; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Entire Regularization Path for the Support Vector Machine.; citation_author=T Hastie; citation_author=R Tibshirani; citation_author=J Zhu; citation_journal_title=Journal of Machine Learning Research; citation_volume=5; citation_number=30; citation_pages=1391-1415; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A study of cross-validation and bootstrap for accuracy estimation and model selection.; citation_author=R Kohavi; citation_journal_title=Proceedings of the 14th International Joint Conference on Artificial Intelligence; citation_number=31; citation_pages=1137-1143; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Randomized, multi-center trial of two hypo-energetic diets in obese subjects: high- versus low-fat content.; citation_author=M Petersen; citation_author=MA Taylor; citation_author=WH Saris; citation_author=C Verdich; citation_author=S Toubro; citation_journal_title=Int J Obes (Lond); citation_volume=30; citation_number=32; citation_pages=552-560; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Gene-diet interactions in obesity.; citation_author=L Perusse; citation_author=C Bouchard; citation_journal_title=Am J Clin Nutr; citation_volume=72; citation_number=33; citation_pages=1285S-1290S; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene-diet interactions on body weight changes.; citation_author=RJ Loos; citation_author=T Rankinen; citation_journal_title=J Am Diet Assoc; citation_volume=105; citation_number=34; citation_pages=S29-S34; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prediction of preadipocyte differentiation by gene expression reveals role of insulin receptor substrates and necdin.; citation_author=YH Tseng; citation_author=AJ Butte; citation_author=E Kokkotou; citation_author=VK Yechoor; citation_author=CM Taniguchi; citation_journal_title=Nat Cell Biol; citation_volume=7; citation_number=35; citation_pages=601-611; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Changes in gene expression foreshadow diet-induced obesity in genetically identical mice.; citation_author=RA Koza; citation_author=L Nikonova; citation_author=J Hogan; citation_author=JS Rim; citation_author=T Mendoza; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=36; citation_pages=e81; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Prediction of response to sibutramine therapy in obese non-diabetic and diabetic patients.; citation_author=N Finer; citation_author=DH Ryan; citation_author=CL Renz; citation_author=AC Hewkin; citation_journal_title=Diabetes Obes Metab; citation_volume=8; citation_number=37; citation_pages=206-213; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Predictive value of early weight loss in obesity management with orlistat: an evidence-based assessment of prescribing guidelines.; citation_author=A Rissanen; citation_author=M Lean; citation_author=S Rossner; citation_author=KR Segal; citation_author=L Sjostrom; citation_journal_title=Int J Obes Relat Metab Disord; citation_volume=27; citation_number=38; citation_pages=103-109; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders"/>
    <meta name="twitter:description" content="BackgroundThe ability to identify obese individuals who will successfully lose weight in response to dietary intervention will revolutionize disease management. Therefore, we asked whether it is possible to identify subjects who will lose weight during dietary intervention using only a single gene expression snapshot.Methodology/Principal FindingsThe present study involved 54 female subjects from the Nutrient-Gene Interactions in Human Obesity-Implications for Dietary Guidelines (NUGENOB) trial to determine whether subcutaneous adipose tissue gene expression could be used to predict weight loss prior to the 10-week consumption of a low-fat hypocaloric diet. Using several statistical tests revealed that the gene expression profiles of responders (8&ndash;12 kgs weight loss) could always be differentiated from non-responders (&lt;4 kgs weight loss). We also assessed whether this differentiation was sufficient for prediction. Using a bottom-up (i.e. black-box) approach, standard class prediction algorithms were able to predict dietary responders with up to 61.1%&plusmn;8.1% accuracy. Using a top-down approach (i.e. using differentially expressed genes to build a classifier) improved prediction accuracy to 80.9%&plusmn;2.2%.ConclusionAdipose gene expression profiling prior to the consumption of a low-fat diet is able to differentiate responders from non-responders as well as serve as a weak predictor of subjects destined to lose weight. While the degree of prediction accuracy currently achieved with a gene expression snapshot is perhaps insufficient for clinical use, this work reveals that the comprehensive molecular signature of adipose tissue paves the way for the future of personalized nutrition."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001344.g003"/>

  <meta property="og:title" content="Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5299'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5794'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4269&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001344">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001344" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001344&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David M. Mutch, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Université Paris Descartes, UMR S 872, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            M. Ramzi Temanni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Laboratoire d'Informatique Medicale and Bio-Informatique (LIM&BIO) EA3969, Paris Nord University, Bobigny, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Corneliu Henegar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Université Paris Descartes, UMR S 872, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Florence Combes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Université Paris Descartes, UMR S 872, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Véronique Pelloux, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Université Paris Descartes, UMR S 872, Paris, France, 
                  Assistance Publique-Hôpitaux de Paris (AP-HP), Pitié Salpêtrière Hospital, Department of Nutrition and Endocrinology, Centre de Recherche en Nutrition Humaine Ile de France (CRNH, Idf), Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Claus Holst, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Health and Society, Institute of Preventive Medicine, Copenhagen University Hospital, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thorkild I. A. Sørensen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Health and Society, Institute of Preventive Medicine, Copenhagen University Hospital, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Arne Astrup, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Nutrition, Faculty of Life Sciences, University of Copenhagen, Copenhagen, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Alfredo Martinez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physiology and Nutrition, University of Navarra, Pamplona, Spain
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wim H. M. Saris, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Biology, NUTRIM, Maastricht University, Maastricht, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nathalie Viguerie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Inserm U858, Institut de Médecine Moléculaire de Rangueil, Laboratoire de recherches sur les obésités, Toulouse, France, 
                  Institut Louis Bugnard, Université Paul Sabatier, IFR31, Toulouse, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dominique Langin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Inserm U858, Institut de Médecine Moléculaire de Rangueil, Laboratoire de recherches sur les obésités, Toulouse, France, 
                  Institut Louis Bugnard, Université Paul Sabatier, IFR31, Toulouse, France, 
                  Centre Hospitalier Universitaire (CHU) de Toulouse, Laboratoire de biochimie, Institut Fédératif de Biologie de Purpan, Toulouse, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Daniel Zucker, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Université Paris Descartes, UMR S 872, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Karine Clément
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:karine.clement@psl.aphp.fr">karine.clement@psl.aphp.fr</a></p>

                <p>Affiliations:
                  INSERM, Nutriomique U872, Paris, France, 
                  Centre de Recherche des Cordeliers, Pierre and Marie Curie University, UMR S 872, Paris, France, 
                  Université Paris Descartes, UMR S 872, Paris, France, 
                  Assistance Publique-Hôpitaux de Paris (AP-HP), Pitié Salpêtrière Hospital, Department of Nutrition and Endocrinology, Centre de Recherche en Nutrition Humaine Ile de France (CRNH, Idf), Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001344</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001344-g001" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001344-t001" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001344-g002" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001344-t002" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001344-g003" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001344-t003" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001344">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001344" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The ability to identify obese individuals who will successfully lose weight in response to dietary intervention will revolutionize disease management. Therefore, we asked whether it is possible to identify subjects who will lose weight during dietary intervention using only a single gene expression snapshot.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The present study involved 54 female subjects from the Nutrient-Gene Interactions in Human Obesity-Implications for Dietary Guidelines (NUGENOB) trial to determine whether subcutaneous adipose tissue gene expression could be used to predict weight loss prior to the 10-week consumption of a low-fat hypocaloric diet. Using several statistical tests revealed that the gene expression profiles of responders (8–12 kgs weight loss) could always be differentiated from non-responders (&lt;4 kgs weight loss). We also assessed whether this differentiation was sufficient for prediction. Using a bottom-up (i.e. black-box) approach, standard class prediction algorithms were able to predict dietary responders with up to 61.1%±8.1% accuracy. Using a top-down approach (i.e. using differentially expressed genes to build a classifier) improved prediction accuracy to 80.9%±2.2%.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Adipose gene expression profiling prior to the consumption of a low-fat diet is able to differentiate responders from non-responders as well as serve as a weak predictor of subjects destined to lose weight. While the degree of prediction accuracy currently achieved with a gene expression snapshot is perhaps insufficient for clinical use, this work reveals that the comprehensive molecular signature of adipose tissue paves the way for the future of personalized nutrition.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Mutch DM, Temanni MR, Henegar C, Combes F, Pelloux V, et al.  (2007) Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders. PLoS ONE 2(12):
          e1344.
            doi:10.1371/journal.pone.0001344</p><p><strong>Academic Editor: </strong>Dawn Albertson, Minnesota State University Mankato, United States of America</p><p><strong>Received:</strong> October 19, 2007; <strong>Accepted:</strong> November 28, 2007; <strong>Published:</strong> December 19, 2007</p><p><strong>Copyright:</strong> © 2007 Mutch et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the European Community NUGENOB-Nutrient Gene Interaction in Human Obesity contract QLK1-CT 2000-00618: <a href="http://www.nugenob.com/">www.nugenob.com/</a>, and DIOGENES-Diet, genes and obesity (Contract FP6-513946: <a href="http://www.diogenes-eu.org">http://www.diogenes-eu.org</a>), INSERM, the Programme National de Recherche en Nutrition Humaine (PNRH), and Paris 6 University. M Ramzi Temanni received thesis support from the Teaching and Research ministry (MRT grant).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Personalized nutritional and medical intervention is rapidly gaining more interest as general population-based recommendations continue to produce unclear and contradictory results <a href="#pone.0001344-Mutch1">[1]</a>–<a href="#pone.0001344-Heidecker1">[4]</a>. As such, the counsel and treatment of a complex disease like obesity requires a more fine-tuned and individual approach. Gene expression profiling has been positioned as a potential method for the identification of novel predictors of disease etiology, outcome, occurrence of co-morbidities and responsiveness to dietary intervention. While microarray analyses have been successfully used to classify and/or predict disease state or outcome in oncology <a href="#pone.0001344-Golub1">[5]</a>–<a href="#pone.0001344-vantVeer1">[10]</a>, no examples currently exist in which gene expression has been used to predict changes in weight after nutritional or clinical intervention in obese individuals. Rather, the many microarray studies performed using obesity models have repeatedly demonstrated that microarray analysis can successfully differentiate between preadipocytes and adipocytes, the various body fat depots in humans, and the cellular response of adipocytes to exogenous compounds <a href="#pone.0001344-Mutch2">[11]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The complex biology of obesity suggests that using microarrays to identify genes capable of foreshadowing adipose tissue metabolism is an ambitious endeavor. This is primarily because of the large inter-individual differences in response to a given dietary intervention for which the origin of this variation remains unknown <a href="#pone.0001344-Viguerie1">[12]</a>. Indeed, common obesity does not occur because of a single dysfunctional gene, but rather through gene-gene and gene-environment interactions that will vary from one individual to another <a href="#pone.0001344-Mutch3">[13]</a>–<a href="#pone.0001344-MorenoAliaga1">[15]</a>. While most microarray work in obesity has focused on class comparison (i.e. comparing obese versus lean, adipocyte versus preadipocyte, etc.), the line between class comparison and class prediction is often blurred and results are frequently positioned to create interest in novel genetic markers for the purpose of disease prognosis and management <a href="#pone.0001344-Lin1">[16]</a>, <a href="#pone.0001344-EinDor1">[17]</a>. Yet there is an inherent limitation with such an approach as inferred models tends to ‘over-fit’ data <a href="#pone.0001344-PerezDiez1">[18]</a>. Prior to claiming a gene or subset of genes as predictive, they must be confirmed in independent samples because often a predictor performs well only on the samples used to identify it. To circumvent this limitation, class prediction is best approached using supervised methods that extract information from a representative and randomly generated ‘training’ set under the guidance of a ‘teacher’, and then relate it to a ‘test’ set to confirm the validity of this extracted information. Only through such an approach can clinically relevant gene predictors be identified.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Selecting subjects from the Nutrient-Gene Interactions in Human Obesity-Implications for Dietary Guidelines (NUGENOB: <a href="http://www.nugenob.com">http://www.nugenob.com</a>) trial, we examined whether subcutaneous adipose tissue gene expression could be used to predict the amount of weight loss in an individual following the consumption of a low fat-hypocaloric diet. Subcutaneous tissue was used for prediction because of the relative ease and rapidity by which biopsies may be obtained. While little is known regarding the changes in adipose gene expression incurred following energy restriction with a low-fat, high-carbohydrate diet <a href="#pone.0001344-Viguerie2">[19]</a>, <a href="#pone.0001344-Sorensen1">[20]</a>, even less is known concerning the predictive capacity of human adipose gene expression profiles with regards to diet-induced weight loss. The aim of the present study was to assess whether a single gene expression snapshot prior to the 10 week consumption of a hypocaloric diet could be used to accurately predict whether an individual would lose weight or not. Through the use of a combination of statistical and supervised learning techniques, we demonstrate that the ability to distinguish between the expression profiles of dietary responders does not guarantee a high accuracy for class prediction. This is in contrast to the successful class comparison and prediction achieved with a publicly available cancer dataset <a href="#pone.0001344-Golub1">[5]</a>. These results are particularly relevant to the clinical field and applicable to those studying drug and nutritional responses in humans with nutritionally related diseases, such as obesity and its associated metabolic pathologies.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Subjects and study design</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Subjects were participants in the European multi-centre NUGENOB study (<a href="http://www.nugenob.org">www.nugenob.org</a>), which was supported by the European Community. Informed consent was obtained from all subjects. Clinical investigations were approved by the ethical committees of each participating centres and were performed according to the Declaration of Helsinki. Subjects (n = 771) were randomly assigned to one of two similarly energy-restricted diets: a low-fat, high-carbohydrate diet (LF) or a moderate-fat, low-carbohydrate diet (MF); however, the present study involved only females from the LF group. The LF diet was designed to provide 600 kcal/day less than the individually estimated energy requirement based on an initial resting metabolic rate measurement and multiplied by 1.3. The macronutrient composition of the LF diet was 20–25%:15%:60–65% for fat:protein:carbohydrate, respectively. Subjects completed a 3-day weighed food record for two weekdays and one weekend day before the start of the dietary intervention and at the end of the 10-week diet. This was done to assess the habitual diets of the subjects and to estimate their compliance, respectively. Subjects also completed 1-day weighed food records during the second, fifth and seventh weeks of the intervention. During the dietary intervention the subjects either visited or had telephone contact with the dietician every week.</p>


<h4>Subject Selection for Prediction Analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Subcutaneous adipose tissue biopsies were obtained for the majority of the 771 subjects participating in this dietary intervention study (both before and after the dietary intervention). An abdominal subcutaneous fat specimen (~1 g) was obtained by needle aspiration under local anaesthesia after an overnight fast. Biopsies were washed and stored in RNA later preservative solution (Qiagen, Courtaboeuf, France) at −80°C until analysis. Total RNA was extracted using the RNeasy total RNA Mini kit (Qiagen). When accounting for both drop outs during the intervention study and those biopsies that produced total RNA that did not meet quality and quantity controls, 319 subjects were assessed for weight loss after 10 weeks and subsequently divided into two groups: ‘responders’ (i.e. subjects losing between 8–12 kg) and ‘non-responders’ (subjects losing &lt;4 kg) (<a href="#pone-0001344-g001"><strong>Figure 1</strong></a>). Twenty-seven female subjects were randomly selected from each group after careful matching based on weight, height, body mass index (BMI), waist/hip ratio, energy intake, fat, carbohydrate, protein and alcohol energy intakes at baseline (<a href="#pone-0001344-t001"><strong>Table 1</strong></a>). Only total RNA from biopsies taken prior to the 10 week dietary intervention was used in the present study.</p>
<div class="figure" id="pone-0001344-g001"><div class="img"><a name="pone-0001344-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001344.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001344.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Weight loss curves during the 10 week hypocaloric diet.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The two groups were defined as responders (i.e. subjects losing between 8–12 kgs) and non-responders (i.e. subjects losing less than 4 kgs). Weight was measured in at least 43 subjects at each weekly time point. Error bars represent the 95% confidence intervals (equal to 1.96 * standard deviation).</p>
<span>doi:10.1371/journal.pone.0001344.g001</span></div><div class="figure" id="pone-0001344-t001"><div class="img"><a name="pone-0001344-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001344.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001344.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Baseline characteristics of responders (8–12 kgs weight loss) and non-responders (&lt;4 kgs weight loss) at T0.</span></strong></p><span>doi:10.1371/journal.pone.0001344.t001</span></div>

<h4>Sample preparation and microarray analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Total RNA concentration and quality was confirmed using the Agilent 2100 Bioanalyzer (Agilent Technologies, Massy, France). 200 ng of total RNA from each sample was amplified and transcribed into fluorescent cRNA using Agilent's Low RNA Input Linear Amplification kit (Agilent Technologies, Massy, France). Cyanine-5 dye was incorporated into all samples, while an in-house obese reference pool was labeled with cyanine-3 dye. In brief, the in-house reference pool was created by mixing equal amounts of total RNA extracted from adipose tissue samples of subjects undergoing plastic surgery, as previously described <a href="#pone.0001344-Clement1">[21]</a>. Samples were hybridized to Agilent 44K whole human genome microarrays, which are comprised of over 41,000 unique 60-mer oligonucleotide human sequences and transcripts. Sample preparation, hybridization, and microarray washing were performed according to manufacturer's recommendations (Agilent Technologies, Massy, France). Arrays were scanned using a GenePix 4000A Scanner (Axon Instruments-Molecular Devices, Sunnyvale, CA).</p>


<h4>Differential Gene Expression Analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Fifty-four microarrays, corresponding to 27 responders and 27 non-responders, were performed; however, one of the microarrays in the non-responders group was of poor quality and therefore not considered further. For the remaining 53 arrays, background signal was not subtracted prior to the Loess normalization of log-transformed microarray data <a href="#pone.0001344-MaryHuard1">[22]</a>. 14135 unique GeneIDs were present across 80% of the microarrays, representing 76.4% of the total genes on the 44K microarray platform. Differential gene expression, using a 5% false discovery rate (FDR), was assessed using the Significance Analysis of Microarrays (SAM) procedure (available at <a href="http://www-stat.stanford.edu/tibs/SAM/">http://www-stat.stanford.edu/tibs/SAM/</a>), a Fisher test and a paired Student's T-test. The three statistical approaches were used because of recent reports demonstrating that different algorithms can generate gene lists with differing degrees of predictive power <a href="#pone.0001344-Lin1">[16]</a>, <a href="#pone.0001344-EinDor1">[17]</a>. The Student's T-test score was generated using the mt.teststat function in the multtest package in ‘R’ (<a href="http://cran.r-project.org/src/contrib/Descriptions/multtest.html">http://cran.r-project.org/src/contrib/De​scriptions/multtest.html</a>). In all analyses the threshold for statistical significance was p&lt;0.05. Univariate analyses were performed using R software (available at <a href="http://www.r-project.org/">http://www.r-project.org/</a>). Multivariate analysis was performed using Umetrics SIMCA-P software (Umetrics AB, Umea, Sweden).</p>


<h4>Semi-quantitative real-time RT-PCR</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Reverse transcription was performed with 0.5 µg of total RNA and random hexamer primers, according to manufacturer's instructions (Promega, Charbonnieres-les-Bains, France). Sybr® green primers were designed and validated for target specificity and amplification efficiency. Primer sequences for <em>Ptgds, Fmod, Ifi27, Qprt, Fam69B, Tmem132A, Esam, Cldn5,</em> and <em>Loc374491</em> are listed in supplementary materials (<a href="#pone.0001344.s001"><strong>Table S1</strong></a>). RT-PCR amplification was performed using an ABI 7300 (Applied Biosystems, Foster City, CA, USA) with the following thermal cycling conditions: 2 min at 50°C, 10 min at 95°C, followed by 40 cycles of 95°C for 15 s and 60°C for 1 min for denaturation, annealing, and elongation. All samples were normalized to <em>18S</em> gene expression (18S rRNA Control kit, Eurogentec, Seraing, Belgium). Differences in gene expression were assesses using a two-tailed, homoscedastic Student's <em>t</em>-test.</p>


<h4>Prediction Analysis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Prediction analyses were performed using the following commonly used methods: Diagonal Linear Discriminant Analysis (DLDA) <a href="#pone.0001344-Dudoit1">[23]</a>, K-Nearest Neighbour (KNN), Random Forest (RF) <a href="#pone.0001344-DiazUriarte1">[24]</a>–<a href="#pone.0001344-Ross1">[27]</a>, and Support Vector Machines (SVM) <a href="#pone.0001344-Dudoit1">[23]</a>, <a href="#pone.0001344-DiazUriarte1">[24]</a>, <a href="#pone.0001344-Furey1">[28]</a>, <a href="#pone.0001344-Lee1">[29]</a>. All algorithms were implemented within the R environment: the DLDA model is available in the <em>sma</em> package, KNN is in <em>class</em> package, RF is in the <em>randomForest</em> package and SVM in <em>Kernlab.</em> Standard parameters were used for all methods except SVM, where the optimal slack parameter was tuned using the <em>svmpath</em> package <a href="#pone.0001344-Hastie1">[30]</a>. Only the 10592 GeneIDs for which no missing values were observed across all microarrays were used.</p>


<h4>Golub Leukemia Data</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To assess the performance of our statistical tests, a human Affymetrix microarray dataset corresponding to 47 subjects with acute lymphoblastic leukaemia (ALL) and 25 subjects with acute myeloid leukaemia (AML), as previously described by Golub and colleagues <a href="#pone.0001344-Golub1">[5]</a>, was downloaded from: <a href="http://wwwmaths.anu.edu.au/johnm/r/hddplot/">http://wwwmaths.anu.edu.au/johnm/r/hddpl​ot/</a>. The ‘Golub’ cancer dataset profiled the expression of 7129 genes in 47 subjects with acute lymphoblastic leukemia (ALL) and 25 subjects with acute myeloid leukemia (AML) to develop a class predictor <a href="#pone.0001344-Golub1">[5]</a>. GeneChips were normalized using the median signal across all 72 microarrays. While the original dataset was comprised of three groups (AML, ALL T-type and ALL B-type), the present analysis created only two classes: AML and ALL. This was done in order to perform a comparable analysis to the present two-class dietary response study and to ensure that all methodologies used were performing as expected.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Gene expression profiling immediately before the 10-week consumption of a low-fat, high-carbohydrate diet (LF-diet) was performed to assess whether subjects could, <em>a priori</em>, be classified as hypo-caloric diet responders and non-responders. To test this hypothesis, we determined whether A) responders could be differentiated from non-responders based on microarray data, and B) whether these gene expression profiles could be used for class prediction (i.e. can one predict whether an individual will lose weight in response to dietary intervention or not).</p>

<h4>Differentiating Responders from Non-Responders</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>As illustrated in <a href="#pone-0001344-g002"><strong>Figure 2</strong></a>, global gene expression was similar between dietary responders and non-responders. Nevertheless, using any of 3 different statistical approaches enabled us to identify gene sets differentiating the two populations. While a SAM analysis using a false discovery rate (FDR) of 5% did not identify any differences between the two populations, relaxing our selection criteria to a FDR of 8% identified 34 differentially expressed genes. The selection criteria for the two other analytical tests (Fisher test and Student's T-test) were arbitrarily set in such a manner as to select the top 100 genes that differentiated the two populations. In order to minimize spurious results that could be attributed to the varying sensitivity of the statistical tests and their ability to identify outliers (i.e. differentially expressed genes), we considered only those genes that were identified by all 3 statistical tests as candidate predictive genes (herein referred to as ‘predictors’). The following nine genes were identified as significantly increased in non-responders versus responders: prostaglandin D2 synthase (<em>Ptgds</em> : 1.6-fold), claudin 5 (<em>Cldn5</em> : 1.4-fold), fibromodulin (<em>Fmod</em> : 1.4-fold), interferon alpha-inducible protein 27 (<em>Ifi27</em> : 1.4-fold), quinolinate phosphoribosyltransferase (<em>Qprt</em> : 1.3-fold), family with sequence similarity 69 B (<em>Fam69b</em> : 1.3-fold), transmembrane protein 132A (<em>Tmem132A</em> : 1.2-fold), endothelial cell adhesion molecule (<em>Esam</em> : 1.2-fold), and TPTE/PTEN homologous inositol lipid phosphatase pseudogene (<em>LOC374491</em> : 1.1-fold). Statistical significance for <em>Ptgds</em>, <em>Cldn5</em>, <em>Qprt</em>, and <em>Tmem132A</em> was confirmed by real-time RT-PCR (p&lt;0.05; <a href="#pone-0001344-t002"><strong>Table 2</strong></a>); and while directional concordance was achieved for <em>Fmod</em>, <em>Ifi27</em>, <em>Fam69b</em> and <em>Esam</em>, changes in expression were not statistically significant. Suitable primers for <em>LOC374491</em> could not be designed; therefore this gene was not confirmed by real-time RT-PCR. Nevertheless, these 9 genes were deemed suitable candidates to assess their capacity to predict weight loss.</p>
<div class="figure" id="pone-0001344-g002"><div class="img"><a name="pone-0001344-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001344.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001344.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Distribution of the mean gene expression levels in responders and non-responders, computed from microarray measurements normalized with respect to the standard Gaussian distribution.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Each spot represents the mean expression for a single gene. Dotted lines indicate the 95% confidence interval of the means (equal to 1.96 * standard deviation).</p>
<span>doi:10.1371/journal.pone.0001344.g002</span></div><div class="figure" id="pone-0001344-t002"><div class="img"><a name="pone-0001344-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001344.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001344.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Validation of 8 predictors by real-time RT- PCR in comparison to microarray results.</span></strong></p><span>doi:10.1371/journal.pone.0001344.t002</span></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>In order to determine whether global gene expression profiles could permit responders and non-responders to be distinguished, partial least squares discriminant analysis (PLS-DA) was performed. <a href="#pone-0001344-g003"><strong>Figure 3A</strong></a> demonstrates that while a trend regarding the separation of the two groups using subcutaneous adipose tissue gene expression profiles exists, there is some overlap between the two populations (R<sup>2</sup> = 0.547 and Q<sup>2</sup> = −0.096, where R<sup>2</sup> explains the cumulative variation of the first two components and Q<sup>2</sup> indicates the variation explained by the model according to cross validation). In contrast to our adipose gene expression data, <a href="#pone-0001344-g003"><strong>Figure 3B</strong></a> clearly illustrates that the Golub cancer dataset can be differentiated by PLS-DA, apart from a single outlier, explained below (R<sup>2</sup> = 0.795 and Q<sup>2</sup> = 0.622). Although the first two principal components explain a significant amount of the variation in both datasets, only the Golub model performs well when cross-validated.</p>
<div class="figure" id="pone-0001344-g003"><div class="img"><a name="pone-0001344-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001344.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001344.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>A. Differentiating populations by PLS-DA.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Global gene expression analysis in sub-cutaneous tissue reveals a separation trend between dietary responders (black squares) and non-responders (red circles); however, there is a significant overlap between the two populations (R<sup>2</sup> = 0.547 and Q<sup>2</sup> = −0.096). B. ALL patients (black squares) can be clearly separated from AML patients (red circles), apart from a single patient (identified by the green circle) (R<sup>2</sup> = 0.795 and Q<sup>2</sup> = 0.622). R<sup>2</sup> explains the cumulative variation of the first two components and Q<sup>2</sup> indicates the variation explained by the model according to cross validation. Only a Q<sup>2</sup>&gt;0.5 indicates a good model.</p>
<span>doi:10.1371/journal.pone.0001344.g003</span></div>

<h4>Predicting Clinical Responders from Non-Responders</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Because the two obese groups could be differentiated as previously described, we employed several supervised machine learning algorithms to address whether this differentiation was sufficient to establish a robust classifier to identify responders and non-responders. Classification was assessed using support vector machine (SVM), random forest (RF), K-nearest neighbour (KNN), and diagonal linear discriminant analysis (DLDA) using a ten times 10-fold cross validation approach. Simultaneously, we examined the prediction accuracy of robust classifiers using a bottom-up analysis and a top-down analysis <a href="#pone.0001344-Lin1">[16]</a>. Whereas a bottom-up analysis uses all gene expression data in a ‘black-box’ approach, a top-down analysis uses existing biological knowledge to build a classifier (i.e. differentially expressed genes and/or molecular pathways).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>A training model for each classification algorithm was established using a cross-validation method, i.e. 9/10<sup>th</sup> of the cohort was used to establish a prediction model (training set) and the remaining 1/10<sup>th</sup> was used as a test set <a href="#pone.0001344-Kohavi1">[31]</a>. This process was repeated 10 times for each algorithm to assess variability in prediction accuracy. In all cases, prediction accuracy was compared to the ‘no information’ algorithm result that places all subjects in the majority class (i.e. for adipose tissue gene expression data = 51%; for Golub cancer data = 65%), where a value above the ‘no information’ result was considered an improvement in prediction accuracy. Using the entire microarray dataset (i.e. 10592 genes), a bottom-up analysis led to insignificant improvements in prediction accuracies versus the ‘no information’ algorithm (<a href="#pone-0001344-t003"><strong>Table 3</strong></a>). KNN performed best, giving a prediction accuracy of 61.1%±8.1%. SVM, RF, and DLDA were able to predict 45.9%±5.7%, 52.9%±11.3%, and 52.6%±8.2%, respectively. In contrast, using the entire Golub cancer dataset (processed identically to our complete adipose gene expression dataset), tumor prediction accuracy soared to 98.6%±0.0%, 96.9%±2.2%, 92.0%±2.9%, and 89.0%±4.1% using SVM, RF, KNN, and DLDA, respectively. The single outlier identified by PLS-DA (<a href="#pone-0001344-g003"><strong>Figure 3B</strong></a><strong>–green circle)</strong> was the only subject in the Golub dataset to be consistently misclassified by all supervised learning methods.</p>
<div class="figure" id="pone-0001344-t003"><div class="img"><a name="pone-0001344-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001344" data-uri="info:doi/10.1371/journal.pone.0001344.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001344.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001344.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001344.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001344.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001344.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Prediction accuracies using ten times 10-fold cross validation with different gene subsets.</span></strong></p><span>doi:10.1371/journal.pone.0001344.t003</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>In an attempt to improve our prediction accuracy, we examined gene sets obtained by Fisher and Student's T-tests. As the goal is to identify reliable predictors that can be used to screen whether new patients will lose weight with a hypocaloric diet or not, predictors need to be identified in a training set and not the entire dataset, i.e. valid predictors should consistently differentiate responders from non-responders in all possible comparisons, and not be dependent on a single comparative analysis. Therefore, the Fisher and Student's T-tests were performed using only gene expression data in the training set (corresponding to 9/10<sup>ths</sup> of dataset) to identify the top 100 differentially expressed genes. These 100 differentially expressed genes were then subsequently tested in the remaining 1/10<sup>th</sup> of the dataset. Neither of these tests led to improvements in prediction accuracies (<a href="#pone-0001344-t003"><strong>Table 3</strong></a>).</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Finally, using a purely top-down approach, the results obtained from the SAM, Fisher, and Student's T-test analyses were examined to determine whether these lists of differentially expressed genes could serve as reliable ‘predictors’. These gene lists can be found in the supplementary materials (<a href="#pone.0001344.s002"><strong>Table S2</strong></a>). In contrast to the previous Fisher and Student's T-test analyses, these different gene sets were obtained using all 53 microarrays and not a cross-validation method. The 34 differentially expressed genes identified by SAM led to prediction accuracies of 70.2%±5.7%, 75.0%±8.1%, 73.7%±4.5%, and 80.9%±2.2% for SVM, RF, KNN, and DLDA, respectively. While prediction accuracies improved overall with a top-down approach, neither the top 100 genes identified with a Fisher test, the top 100 genes identified with a Student's T-test, nor the 9 common genes performed better than the 34 genes identified by SAM (<a href="#pone-0001344-t003"><strong>Table 3</strong></a>).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The ability to predict whether an individual will respond successfully to dietary intervention with a significant weight loss clearly has important clinical ramifications. Indeed if a clinician could, <em>a priori</em>, know whether a patient's health status will improve in response to the consumption of a given diet then disease management will be profoundly modified in many aspects. The analysis of gene expression has been positioned, in large part because of its success in the field of oncology, as one means by which an individual's response to an intervention could be predicted. The present study revealed that while a single comprehensive snapshot of gene expression immediately prior to the 10-week consumption of a hypocaloric diet can differentiate responders from non-responders, at this stage it is insufficient for the accurate class prediction required for clinical use.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Studies in the oncology field suggest that classifying tumors by microarray analysis can be achieved, but despite these encouraging findings there is, up to now, no evidence that this approach can be useful in the study of nutrition related diseases such as obesity. When one considers that adipose tissue metabolism is regulated by not only an individual's genetic make-up, but also the obesogenic environmental factors an individual is exposed to (e.g. diet, physical activity, gut microbiota, viruses, etc.), then one must clearly consider the individual (or subset of individuals) to identify eventual responders from non-responders <a href="#pone.0001344-Mutch3">[13]</a>, <a href="#pone.0001344-Petersen1">[32]</a>. As previously reported, significant inter-individual differences in response to dietary interventions demonstrate that weight loss is governed by both genetic and lifestyle components <a href="#pone.0001344-Viguerie2">[19]</a>, <a href="#pone.0001344-Perusse1">[33]</a>, <a href="#pone.0001344-Loos1">[34]</a>. Indeed, class prediction would be simplified considerably if the aforementioned diseases arose purely because of gene dysfunction, rather than having both a genetic and environmental component.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Profiling subcutaneous adipose gene expression in 53 human subjects participating in the NUGENOB trial prior to the start of a 10-week hypo-caloric diet has revealed that responders (subjects losing between 8–12 kg) can be distinguished from non-responders (subjects losing less than 4 kg); however, the differences are minimal and do not cleanly separate, as illustrated by multivariate analysis. This was in contrast to the distinct separation of AML versus ALL subjects seen in the Golub cancer dataset. Thus, while differences in adipose gene expression profiles were identified (using SAM, Fisher and Student's T-test analyses), we hypothesize that the considerable overlap seen with PLS-DA confounds our ability to predict class (61.1% accurate at best using a bottom-up analysis). Furthermore, the low accuracies suggest that there are no dominant predictors within the gene expression dataset analyzed in this study. This is dramatically different from the Golub cancer dataset were we observed a nearly perfect class prediction (98.6% accuracy). Even though prediction accuracies were improved when we used a top-down analysis using differentially expressed gene sets, we still could not identify a classifier that performed as well as the classifier inferred from the Golub dataset. It is important to note that the differentially expressed gene sets used for the top-down analysis were generated using all 53 microarrays. Thus, even though we could achieve upwards to 80% prediction accuracy, the predictive power of these gene subsets requires validation in new subjects as they risk being specific to only the subjects used in the present study and therefore of little clinical relevance <a href="#pone.0001344-vantVeer1">[10]</a>, <a href="#pone.0001344-EinDor1">[17]</a>. This finding also highlights an important difference between our data and the Golub dataset–namely that we have attempted to predict an individual's response rather than classify an individual's disease state. As such, the present study implies that classifying tumor type might be less complex than predicting an individual's response to an intervention. Nevertheless the Golub dataset was perfectly suited to confirm that our prediction methodologies were functioning correctly. Therefore we believe that predicting a response to diet using a single snapshot of global gene expression prior to an intervention is not sufficient to identify responders from non-responders.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Examples exist in which an individual or subset of predictive genes have been proposed; however, these studies have performed class comparison rather than class prediction. For example, Tseng and colleagues performed microarray analysis of brown preadipocytes and postulated insulin receptor substrates and necdin as predictive genes for differentiation <a href="#pone.0001344-Tseng1">[35]</a>. And more recently, Koza <em>et al</em>. profiled adipose tissue gene expression and revealed that mice with low or high body fat gain could be distinguished prior to the consumption of a high fat diet <a href="#pone.0001344-Koza1">[36]</a>. While these studies may suggest that identifying predictive genes using a gene expression snapshot is possible, there are several critical differences between our work and these two aforementioned studies. Firstly, neither of these studies used humans. As such, using genetically identical mice eliminates a major confounder present in human studies. Secondly, neither study used supervised methods to identify predictors; rather, they both used statistical approaches to identify differentially regulated genes. As such, the actual predictive value of their candidates was not assessed. Thus, while these previous studies provide molecular insight into fat mass expansion, they have yet to definitely demonstrate that their molecular targets are clinically-reliable predictors.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>To the best of our knowledge, the use of gene expression to predict a response to dietary intervention in humans has not been previously performed; however, examples do exist in which the efficacy of anti-obesity drugs (sibutramine and orlistat) have been explored. Drug efficacy for an individual subject was determined by assessing weight loss in the first 3 months of treatment <a href="#pone.0001344-Finer1">[37]</a>, <a href="#pone.0001344-Rissanen1">[38]</a>. Subjects reaching a given threshold for weight loss in the first months of treatment were predicted to benefit from the long term administration of these anti-obesity drugs. Relating these findings to our microarray results suggests that predicting classes in obesity may be improved by studying changes in gene expression once dietary intervention has begun (rather than single time points). Such an approach would allow one to assess how the biological system responds once it has been challenged with an exogenous factor, and thereby possibly improve class prediction.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>In conclusion, microarrays can provide sound molecular insight into the biological mechanisms underlying adipose tissue metabolism; however, their potential in a clinical context to assist in the optimized nutritional counseling of obese individuals remains in its infancy. Nevertheless, alternate strategies to maximize the wealth of information on microarrays are being explored and suggest that predicting diet response for an individual will become more increasingly more accurate in the near future.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001344"></div><a name="pone.0001344.s001" id="pone.0001344.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001344.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Primer sequences for transcripts validated by real-time reverse-transcriptase PCR.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001344.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001344.s002" id="pone.0001344.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001344.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Gene lists used for the top-down analyses. The gene lists indicate the 34 genes identified with SAM and the top 100 genes identified by Fisher or Student's t-test analyses. SAM, significance analysis of microarrays; FDR, false discovery rate.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001344.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KC AA DM DL CH WS JM NV TS JZ MT. Performed the experiments: KC DM JZ MT VP. Analyzed the data: KC DM CH JZ MT FC. Contributed reagents/materials/analysis tools: KC AA DM DL CH WS JM NV TS JZ MT. Wrote the paper: KC DM MT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001344-Mutch1" id="pone.0001344-Mutch1"></a>Mutch DM, Wahli W, Williamson G (2005) Nutrigenomics and nutrigenetics: the emerging faces of nutrition. FASEB J  19: 1602–1616.  <ul class="find" data-citedArticleID="1032892" data-doi="10.1096/fj.05-3911rev"><li><a href="http://dx.doi.org/10.1096/fj.05-3911rev" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nutrigenomics+and+nutrigenetics%3A+the+emerging+faces+of+nutrition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nutrigenomics+and+nutrigenetics%3A+the+emerging+faces+of+nutrition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001344-Kaput1" id="pone.0001344-Kaput1"></a>Kaput J, Ordovas JM, Ferguson L, van Ommen B, Rodriguez RL, et al.  (2005) The case for strategic international alliances to harness nutritional genomics for public and personal health. Br J Nutr  94: 623–632.  <ul class="find" data-citedArticleID="1032870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+case+for+strategic+international+alliances+to+harness+nutritional+genomics+for+public+and+personal+health.&amp;auth=&amp;atitle=The+case+for+strategic+international+alliances+to+harness+nutritional+genomics+for+public+and+personal+health." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+case+for+strategic+international+alliances+to+harness+nutritional+genomics+for+public+and+personal+health." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+case+for+strategic+international+alliances+to+harness+nutritional+genomics+for+public+and+personal+health.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001344-Jain1" id="pone.0001344-Jain1"></a>Jain KK (2006) Challenges of drug discovery for personalized medicine. Curr Opin Mol Ther  8: 487–492.  <ul class="find" data-citedArticleID="1032868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Challenges+of+drug+discovery+for+personalized+medicine.&amp;auth=&amp;atitle=Challenges+of+drug+discovery+for+personalized+medicine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Challenges+of+drug+discovery+for+personalized+medicine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Challenges+of+drug+discovery+for+personalized+medicine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001344-Heidecker1" id="pone.0001344-Heidecker1"></a>Heidecker B, Hare JM (2007) The use of transcriptomic biomarkers for personalized medicine. Heart Fail Rev  12: 1–11.  <ul class="find" data-citedArticleID="1032866" data-doi="10.1007/s10741-007-9004-7"><li><a href="http://dx.doi.org/10.1007/s10741-007-9004-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+transcriptomic+biomarkers+for+personalized+medicine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+transcriptomic+biomarkers+for+personalized+medicine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001344-Golub1" id="pone.0001344-Golub1"></a>Golub TR, Slonim DK, Tamayo P, Huard C, Gaasenbeek M, et al.  (1999) Molecular classification of cancer: class discovery and class prediction by gene expression monitoring. Science  286: 531–537.  <ul class="find" data-citedArticleID="1032862" data-doi="10.1126/science.286.5439.531"><li><a href="http://dx.doi.org/10.1126/science.286.5439.531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+classification+of+cancer%3A+class+discovery+and+class+prediction+by+gene+expression+monitoring." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+classification+of+cancer%3A+class+discovery+and+class+prediction+by+gene+expression+monitoring.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001344-Shipp1" id="pone.0001344-Shipp1"></a>Shipp MA, Ross KN, Tamayo P, Weng AP, Kutok JL, et al.  (2002) Diffuse large B-cell lymphoma outcome prediction by gene-expression profiling and supervised machine learning. Nat Med  8: 68–74.  <ul class="find" data-citedArticleID="1032912" data-doi="10.1038/nm0102-68"><li><a href="http://dx.doi.org/10.1038/nm0102-68" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diffuse+large+B-cell+lymphoma+outcome+prediction+by+gene-expression+profiling+and+supervised+machine+learning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diffuse+large+B-cell+lymphoma+outcome+prediction+by+gene-expression+profiling+and+supervised+machine+learning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001344-Rosenwald1" id="pone.0001344-Rosenwald1"></a>Rosenwald A, Wright G, Chan WC, Connors JM, Campo E, et al.  (2002) The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma. N Engl J Med  346: 1937–1947.  <ul class="find" data-citedArticleID="1032908" data-doi="10.1056/nejmoa012914"><li><a href="http://dx.doi.org/10.1056/nejmoa012914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+molecular+profiling+to+predict+survival+after+chemotherapy+for+diffuse+large-B-cell+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+molecular+profiling+to+predict+survival+after+chemotherapy+for+diffuse+large-B-cell+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001344-Lapointe1" id="pone.0001344-Lapointe1"></a>Lapointe J, Li C, Higgins JP, van de RM, Bair E, et al.  (2004) Gene expression profiling identifies clinically relevant subtypes of prostate cancer. Proc Natl Acad Sci U S A  101: 811–816.  <ul class="find" data-citedArticleID="1032878" data-doi="10.1073/pnas.0304146101"><li><a href="http://dx.doi.org/10.1073/pnas.0304146101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+identifies+clinically+relevant+subtypes+of+prostate+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+identifies+clinically+relevant+subtypes+of+prostate+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001344-Pomeroy1" id="pone.0001344-Pomeroy1"></a>Pomeroy SL, Tamayo P, Gaasenbeek M, Sturla LM, Angelo M, et al.  (2002) Prediction of central nervous system embryonal tumour outcome based on gene expression. Nature  415: 436–442.  <ul class="find" data-citedArticleID="1032904" data-doi="10.1038/415436a"><li><a href="http://dx.doi.org/10.1038/415436a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+central+nervous+system+embryonal+tumour+outcome+based+on+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+central+nervous+system+embryonal+tumour+outcome+based+on+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001344-vantVeer1" id="pone.0001344-vantVeer1"></a>van't Veer L, Dai H, van d V, He YD, Hart AA, et al.  (2002) Gene expression profiling predicts clinical outcome of breast cancer. Nature  415: 530–536.  <ul class="find" data-citedArticleID="1032922" data-doi="10.1038/415530a"><li><a href="http://dx.doi.org/10.1038/415530a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001344-Mutch2" id="pone.0001344-Mutch2"></a>Mutch DM (2006) Identifying regulatory hubs in obesity with nutrigenomics. Curr Opin Endocrinol Diabetes  13: 431–437.  <ul class="find" data-citedArticleID="1032894" data-doi="10.1097/01.med.0000244224.76033.06"><li><a href="http://dx.doi.org/10.1097/01.med.0000244224.76033.06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+regulatory+hubs+in+obesity+with+nutrigenomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+regulatory+hubs+in+obesity+with+nutrigenomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001344-Viguerie1" id="pone.0001344-Viguerie1"></a>Viguerie N, Poitou C, Cancello R, Stich V, Clement K, et al.  (2005) Transcriptomics applied to obesity and caloric restriction. Biochimie  87: 117–123.  <ul class="find" data-citedArticleID="1032918" data-doi="10.1016/j.biochi.2004.12.011"><li><a href="http://dx.doi.org/10.1016/j.biochi.2004.12.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptomics+applied+to+obesity+and+caloric+restriction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptomics+applied+to+obesity+and+caloric+restriction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001344-Mutch3" id="pone.0001344-Mutch3"></a>Mutch DM, Clement K (2006) Unraveling the genetics of human obesity. PLoS Genet  2: e188.  <ul class="find" data-citedArticleID="1032896" data-doi="10.1371/journal.pgen.0020188"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unraveling+the+genetics+of+human+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unraveling+the+genetics+of+human+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001344-Klaus1" id="pone.0001344-Klaus1"></a>Klaus S, Keijer J (2004) Gene expression profiling of adipose tissue: individual, depot-dependent, and sex-dependent variabilities. Nutrition  20: 115–120.  <ul class="find" data-citedArticleID="1032872" data-doi="10.1016/j.nut.2003.09.020"><li><a href="http://dx.doi.org/10.1016/j.nut.2003.09.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+of+adipose+tissue%3A+individual%2C+depot-dependent%2C+and+sex-dependent+variabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+of+adipose+tissue%3A+individual%2C+depot-dependent%2C+and+sex-dependent+variabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001344-MorenoAliaga1" id="pone.0001344-MorenoAliaga1"></a>Moreno-Aliaga MJ, Santos JL, Marti A, Martinez JA (2005) Does weight loss prognosis depend on genetic make-up? Obes Rev  6: 155–168.  <ul class="find" data-citedArticleID="1032890" data-doi="10.1111/j.1467-789x.2005.00180.x"><li><a href="http://dx.doi.org/10.1111/j.1467-789x.2005.00180.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Does+weight+loss+prognosis+depend+on+genetic+make-up%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Does+weight+loss+prognosis+depend+on+genetic+make-up%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001344-Lin1" id="pone.0001344-Lin1"></a>Lin SM, Devakumar J, Kibbe WA (2006) Improved prediction of treatment response using microarrays and existing biological knowledge. Pharmacogenomics  7: 495–501.  <ul class="find" data-citedArticleID="1032884" data-doi="10.2217/14622416.7.3.495"><li><a href="http://dx.doi.org/10.2217/14622416.7.3.495" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+prediction+of+treatment+response+using+microarrays+and+existing+biological+knowledge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+prediction+of+treatment+response+using+microarrays+and+existing+biological+knowledge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001344-EinDor1" id="pone.0001344-EinDor1"></a>Ein-Dor L, Kela I, Getz G, Givol D, Domany E (2005) Outcome signature genes in breast cancer: is there a unique set? Bioinformatics  21: 171–178.  <ul class="find" data-citedArticleID="1032856" data-doi="10.1093/bioinformatics/bth469"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Outcome+signature+genes+in+breast+cancer%3A+is+there+a+unique+set%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Outcome+signature+genes+in+breast+cancer%3A+is+there+a+unique+set%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001344-PerezDiez1" id="pone.0001344-PerezDiez1"></a>Perez-Diez A, Morgun A, Shulzhenko N (2007) Microarrays for cancer diagnosis and classification. Adv Exp Med Biol  593: 74–85.  <ul class="find" data-citedArticleID="1032898" data-doi="10.1007/978-0-387-39978-2_8"><li><a href="http://dx.doi.org/10.1007/978-0-387-39978-2_8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarrays+for+cancer+diagnosis+and+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarrays+for+cancer+diagnosis+and+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001344-Viguerie2" id="pone.0001344-Viguerie2"></a>Viguerie N, Vidal H, Arner P, Holst C, Verdich C, et al.  (2005) Adipose tissue gene expression in obese subjects during low-fat and high-fat hypocaloric diets. Diabetologia  48: 123–131.  <ul class="find" data-citedArticleID="1032920" data-doi="10.1007/s00125-004-1618-x"><li><a href="http://dx.doi.org/10.1007/s00125-004-1618-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adipose+tissue+gene+expression+in+obese+subjects+during+low-fat+and+high-fat+hypocaloric+diets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adipose+tissue+gene+expression+in+obese+subjects+during+low-fat+and+high-fat+hypocaloric+diets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001344-Sorensen1" id="pone.0001344-Sorensen1"></a>Sorensen TIA, Boutin P, Taylor MA, Larsen LH, Verdich C, et al.  (2006) Genetic Polymorphisms and Weight Loss in Obesity: A Randomised Trial of Hypo-Energetic High- versus Low-Fat Diets. PLoS Clinical Trials  1: e12.  <ul class="find" data-citedArticleID="1032914" data-doi="10.1371/journal.pctr.0010012"><li><a href="http://dx.doi.org/10.1371/journal.pctr.0010012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Polymorphisms+and+Weight+Loss+in+Obesity%3A+A+Randomised+Trial+of+Hypo-Energetic+High-+versus+Low-Fat+Diets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Polymorphisms+and+Weight+Loss+in+Obesity%3A+A+Randomised+Trial+of+Hypo-Energetic+High-+versus+Low-Fat+Diets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001344-Clement1" id="pone.0001344-Clement1"></a>Clement K, Viguerie N, Poitou C, Carette C, Pelloux V, et al.  (2004) Weight loss regulates inflammation-related genes in white adipose tissue of obese subjects. FASEB J  18: 1657–1669.  <ul class="find" data-citedArticleID="1032850" data-doi="10.1096/fj.04-2204com"><li><a href="http://dx.doi.org/10.1096/fj.04-2204com" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Weight+loss+regulates+inflammation-related+genes+in+white+adipose+tissue+of+obese+subjects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Weight+loss+regulates+inflammation-related+genes+in+white+adipose+tissue+of+obese+subjects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001344-MaryHuard1" id="pone.0001344-MaryHuard1"></a>Mary-Huard T, Picard F, Robin S (2006) Introduction to Statistical Methods for Microarray Data Analysis. Mathematical and Computational Methods in Biology. Paris: Hermann.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001344-Dudoit1" id="pone.0001344-Dudoit1"></a>Dudoit S, Fridlyand J (2003)  Classification in microarray experiments. In: Speed TP, editor. Statistical Analysis of Gene Expression Microarray Data. London: Chapman &amp; Hall. pp. 93–158.  <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001344-DiazUriarte1" id="pone.0001344-DiazUriarte1"></a>Diaz-Uriarte R, Alvarez de Andres S (2006) Gene selection and classification of microarray data using random forest. BMC Bioinformatics  7: 3.  <ul class="find" data-citedArticleID="1032852" data-doi="10.1186/1471-2105-8-328"><li><a href="http://dx.doi.org/10.1186/1471-2105-8-328" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+selection+and+classification+of+microarray+data+using+random+forest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+selection+and+classification+of+microarray+data+using+random+forest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001344-Liaw1" id="pone.0001344-Liaw1"></a>Liaw A, Wiener M (2002) Classification and regression by randomForest. Rnews  2: 18–22.  <ul class="find" data-citedArticleID="1032882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Classification+and+regression+by+randomForest.&amp;auth=&amp;atitle=Classification+and+regression+by+randomForest." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+and+regression+by+randomForest." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+and+regression+by+randomForest.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001344-Breiman1" id="pone.0001344-Breiman1"></a>Breiman L (2001) Random Forests. Mach Learn  40: 5–32.  <ul class="find" data-citedArticleID="1032848"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Random+Forests.&amp;auth=&amp;atitle=Random+Forests." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Random+Forests." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Random+Forests.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001344-Ross1" id="pone.0001344-Ross1"></a>Ross DT, Scherf U, Eisen MB, Perou CM, Rees C, et al.  (2000) Systematic variation in gene expression patterns in human cancer cell lines. Nat Genet  24: 227–235.  <ul class="find" data-citedArticleID="1032910" data-doi="10.1038/73432"><li><a href="http://dx.doi.org/10.1038/73432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+variation+in+gene+expression+patterns+in+human+cancer+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+variation+in+gene+expression+patterns+in+human+cancer+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001344-Furey1" id="pone.0001344-Furey1"></a>Furey TS, Cristianini N, Duffy N, Bednarski DW, Schummer M, et al.  (2000) Support vector machine classification and validation of cancer tissue samples using microarray expression data. Bioinformatics  16: 906–914.  <ul class="find" data-citedArticleID="1032860" data-doi="10.1093/bioinformatics/16.10.906"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.10.906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Support+vector+machine+classification+and+validation+of+cancer+tissue+samples+using+microarray+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Support+vector+machine+classification+and+validation+of+cancer+tissue+samples+using+microarray+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001344-Lee1" id="pone.0001344-Lee1"></a>Lee Y, Lee CK (2003) Classification of multiple cancer types by multicategory support vector machines using gene expression data. Bioinformatics  19: 1132–1139.  <ul class="find" data-citedArticleID="1032880" data-doi="10.1093/bioinformatics/btg102"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+multiple+cancer+types+by+multicategory+support+vector+machines+using+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+multiple+cancer+types+by+multicategory+support+vector+machines+using+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001344-Hastie1" id="pone.0001344-Hastie1"></a>Hastie T, Tibshirani R, Zhu J (2004) The Entire Regularization Path for the Support Vector Machine. Journal of Machine Learning Research  5: 1391–1415.  <ul class="find" data-citedArticleID="1032864"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Entire+Regularization+Path+for+the+Support+Vector+Machine.&amp;auth=&amp;atitle=The+Entire+Regularization+Path+for+the+Support+Vector+Machine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Entire+Regularization+Path+for+the+Support+Vector+Machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Entire+Regularization+Path+for+the+Support+Vector+Machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001344-Kohavi1" id="pone.0001344-Kohavi1"></a>Kohavi R (1995) A study of cross-validation and bootstrap for accuracy estimation and model selection. Proceedings of the 14th International Joint Conference on Artificial Intelligence 1137–1143.  <ul class="find" data-citedArticleID="1032874"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+study+of+cross-validation+and+bootstrap+for+accuracy+estimation+and+model+selection.&amp;auth=&amp;atitle=A+study+of+cross-validation+and+bootstrap+for+accuracy+estimation+and+model+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+study+of+cross-validation+and+bootstrap+for+accuracy+estimation+and+model+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+study+of+cross-validation+and+bootstrap+for+accuracy+estimation+and+model+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001344-Petersen1" id="pone.0001344-Petersen1"></a>Petersen M, Taylor MA, Saris WH, Verdich C, Toubro S, et al.  (2006) Randomized, multi-center trial of two hypo-energetic diets in obese subjects: high- versus low-fat content. Int J Obes (Lond)  30: 552–560.  <ul class="find" data-citedArticleID="1032902" data-doi="10.1038/sj.ijo.0803186"><li><a href="http://dx.doi.org/10.1038/sj.ijo.0803186" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Randomized%2C+multi-center+trial+of+two+hypo-energetic+diets+in+obese+subjects%3A+high-+versus+low-fat+content." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Randomized%2C+multi-center+trial+of+two+hypo-energetic+diets+in+obese+subjects%3A+high-+versus+low-fat+content.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001344-Perusse1" id="pone.0001344-Perusse1"></a>Perusse L, Bouchard C (2000) Gene-diet interactions in obesity. Am J Clin Nutr  72: 1285S–1290S.  <ul class="find" data-citedArticleID="1032900"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene-diet+interactions+in+obesity.&amp;auth=&amp;atitle=Gene-diet+interactions+in+obesity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene-diet+interactions+in+obesity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene-diet+interactions+in+obesity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001344-Loos1" id="pone.0001344-Loos1"></a>Loos RJ, Rankinen T (2005) Gene-diet interactions on body weight changes. J Am Diet Assoc  105: S29–S34.  <ul class="find" data-citedArticleID="1032886" data-doi="10.1016/j.jada.2005.02.015"><li><a href="http://dx.doi.org/10.1016/j.jada.2005.02.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene-diet+interactions+on+body+weight+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene-diet+interactions+on+body+weight+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001344-Tseng1" id="pone.0001344-Tseng1"></a>Tseng YH, Butte AJ, Kokkotou E, Yechoor VK, Taniguchi CM, et al.  (2005) Prediction of preadipocyte differentiation by gene expression reveals role of insulin receptor substrates and necdin. Nat Cell Biol  7: 601–611.  <ul class="find" data-citedArticleID="1032916" data-doi="10.1038/ncb1259"><li><a href="http://dx.doi.org/10.1038/ncb1259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+preadipocyte+differentiation+by+gene+expression+reveals+role+of+insulin+receptor+substrates+and+necdin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+preadipocyte+differentiation+by+gene+expression+reveals+role+of+insulin+receptor+substrates+and+necdin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001344-Koza1" id="pone.0001344-Koza1"></a>Koza RA, Nikonova L, Hogan J, Rim JS, Mendoza T, et al.  (2006) Changes in gene expression foreshadow diet-induced obesity in genetically identical mice. PLoS Genet  2: e81.  <ul class="find" data-citedArticleID="1032876" data-doi="10.1371/journal.pgen.0020081"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020081" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+gene+expression+foreshadow+diet-induced+obesity+in+genetically+identical+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+gene+expression+foreshadow+diet-induced+obesity+in+genetically+identical+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001344-Finer1" id="pone.0001344-Finer1"></a>Finer N, Ryan DH, Renz CL, Hewkin AC (2006) Prediction of response to sibutramine therapy in obese non-diabetic and diabetic patients. Diabetes Obes Metab  8: 206–213.  <ul class="find" data-citedArticleID="1032858" data-doi="10.1111/j.1463-1326.2005.00481.x"><li><a href="http://dx.doi.org/10.1111/j.1463-1326.2005.00481.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+response+to+sibutramine+therapy+in+obese+non-diabetic+and+diabetic+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+response+to+sibutramine+therapy+in+obese+non-diabetic+and+diabetic+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001344-Rissanen1" id="pone.0001344-Rissanen1"></a>Rissanen A, Lean M, Rossner S, Segal KR, Sjostrom L (2003) Predictive value of early weight loss in obesity management with orlistat: an evidence-based assessment of prescribing guidelines. Int J Obes Relat Metab Disord  27: 103–109.  <ul class="find" data-citedArticleID="1032906" data-doi="10.1038/sj.ijo.0802165"><li><a href="http://dx.doi.org/10.1038/sj.ijo.0802165" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predictive+value+of+early+weight+loss+in+obesity+management+with+orlistat%3A+an+evidence-based+assessment+of+prescribing+guidelines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predictive+value+of+early+weight+loss+in+obesity+management+with+orlistat%3A+an+evidence-based+assessment+of+prescribing+guidelines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.XML" value="90705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.PDF" value="232064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g001.PNG_L" value="117149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g001.PNG_M" value="51786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g001.PNG_S" value="7124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g001.TIF" value="242538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g001.PNG_I" value="18397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t001.PNG_L" value="130495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t001.PNG_M" value="151958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t001.PNG_S" value="19042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t001.TIF" value="611542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t001.PNG_I" value="67357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g002.PNG_L" value="105058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g002.PNG_M" value="26184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g002.PNG_S" value="5459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g002.TIF" value="184652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g002.PNG_I" value="9165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t002.PNG_L" value="104843"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t002.PNG_M" value="137692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t002.PNG_S" value="17525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t002.TIF" value="444060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t002.PNG_I" value="49319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g003.PNG_L" value="131934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g003.PNG_M" value="76844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g003.PNG_S" value="10079"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g003.TIF" value="977024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.g003.PNG_I" value="38346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t003.PNG_L" value="161218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t003.PNG_M" value="90062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t003.PNG_S" value="10569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t003.TIF" value="618500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.t003.PNG_I" value="27086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.s001.XLS" value="18944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001344.s002.XLS" value="41984"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001344&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001344" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001344&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001344&volume=&issue=&title=Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders&author_name=David%20M.%20Mutch%2C%20M.%20Ramzi%20Temanni%2C%20Corneliu%20Henegar%2C%20Florence%20Combes%2C%20V%C3%A9ronique%20Pelloux%2C%20Claus%20Holst%2C%20Thorkild%20I.%20A.%20S%C3%B8rensen%2C%20Arne%20Astrup%2C%20J.%20Alfredo%20Martinez%2C%20Wim%20H.%20M.%20Saris%2C%20Nathalie%20Viguerie%2C%20Dominique%20Langin%2C%20Jean-Daniel%20Zucker%2C%20Karine%20Cl%C3%A9ment&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001344" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344&amp;t=Adipose%20Gene%20Expression%20Prior%20to%20Weight%20Loss%20Can%20Differentiate%20and%20Weakly%20Predict%20Dietary%20Responders" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344&title=Adipose%20Gene%20Expression%20Prior%20to%20Weight%20Loss%20Can%20Differentiate%20and%20Weakly%20Predict%20Dietary%20Responders&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344&amp;title=Adipose%20Gene%20Expression%20Prior%20to%20Weight%20Loss%20Can%20Differentiate%20and%20Weakly%20Predict%20Dietary%20Responders" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001344&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Adipose Gene Expression Prior to Weight Loss Can Differentiate and Weakly Predict Dietary Responders';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001344';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Adipose%20Gene%20Expression%20Prior%20to%20Weight%20Loss%20Can%20Differentiate%20and%20Weakly%20Predict%20Dietary%20Responders http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001344" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001344" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Adipose+tissue%22" title="Search for articles in the subject area:'Adipose tissue'"><div class="flagText">Adipose tissue</div></a>
              <div data-categoryid="48059" data-articleid="25518"
                   data-categoryname="Adipose tissue"
                   class="flagImage" title="Flag 'Adipose tissue' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Diet%22" title="Search for articles in the subject area:'Diet'"><div class="flagText">Diet</div></a>
              <div data-categoryid="18621" data-articleid="25518"
                   data-categoryname="Diet"
                   class="flagImage" title="Flag 'Diet' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25518"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25518"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Obesity%22" title="Search for articles in the subject area:'Obesity'"><div class="flagText">Obesity</div></a>
              <div data-categoryid="47275" data-articleid="25518"
                   data-categoryname="Obesity"
                   class="flagImage" title="Flag 'Obesity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="25518"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Support+vector+machines%22" title="Search for articles in the subject area:'Support vector machines'"><div class="flagText">Support vector machines</div></a>
              <div data-categoryid="32835" data-articleid="25518"
                   data-categoryname="Support vector machines"
                   class="flagImage" title="Flag 'Support vector machines' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Weight+loss%22" title="Search for articles in the subject area:'Weight loss'"><div class="flagText">Weight loss</div></a>
              <div data-categoryid="49823" data-articleid="25518"
                   data-categoryname="Weight loss"
                   class="flagImage" title="Flag 'Weight loss' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4623'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9744'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2558&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=765">Updated Figure 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
