

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001179"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001179" />

    <meta name="citation_title" content="Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA"/>
    <meta itemprop="name" content="Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA"/>

      <meta name="citation_author" content="Xiaoping An"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>
      <meta name="citation_author" content="Jun Lu"/>
            <meta name="citation_author_institution" content="Beijing YouAn Hospital, Capital Medical University, Beijing, China"/>
      <meta name="citation_author" content="Jian-dong Huang"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, The University of Hong Kong, Hong Kong Special Administrative Region (SAR), China"/>
      <meta name="citation_author" content="Baozhong Zhang"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>
      <meta name="citation_author" content="Dabin Liu"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>
      <meta name="citation_author" content="Xin Zhang"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>
      <meta name="citation_author" content="Jinhui Chen"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>
      <meta name="citation_author" content="Yusen Zhou"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>
      <meta name="citation_author" content="Yigang Tong"/>
            <meta name="citation_author_institution" content="State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China"/>

    <meta name="citation_date" content="2007/11/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001179.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1179"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Synthetic lac operator DNA is functional in vivo.; citation_author=HL Heyneker; citation_author=J Shine; citation_author=HM Goodman; citation_author=HW Boyer; citation_author=J Rosenberg; citation_journal_title=Nature; citation_volume=263; citation_number=1; citation_pages=748-752; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=Expression in Escherichia coli of a chemically synthesized gene for the hormone somatostatin.; citation_author=K Itakura; citation_author=T Hirose; citation_author=R Crea; citation_author=AD Riggs; citation_author=HL Heyneker; citation_journal_title=Science; citation_volume=198; citation_number=2; citation_pages=1056-1063; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Expression in Escherichia coli of chemically synthesized genes for human insulin.; citation_author=DV Goeddel; citation_author=DG Kleid; citation_author=F Bolivar; citation_author=HL Heyneker; citation_author=DG Yansura; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=76; citation_number=3; citation_pages=106-110; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=FokI method of gene synthesis.; citation_author=W Mandecki; citation_author=TJ Bolling; citation_journal_title=Gene; citation_volume=68; citation_number=4; citation_pages=101-107; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=A totally synthetic plasmid for general cloning, gene expression and mutagenesis in Escherichia coli.; citation_author=W Mandecki; citation_author=MA Hayden; citation_author=MA Shallcross; citation_author=E Stotland; citation_journal_title=Gene; citation_volume=94; citation_number=5; citation_pages=103-107; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A rapid method for the construction of synthetic genes using the polymerase chain reaction.; citation_author=PJ Dillon; citation_author=CA Rosen; citation_journal_title=Biotechniques; citation_volume=9; citation_number=6; citation_pages=298, 300; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Recursive PCR: a novel technique for total gene synthesis.; citation_author=C Prodromou; citation_author=LH Pearl; citation_journal_title=Protein Eng; citation_volume=5; citation_number=7; citation_pages=827-829; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Simultaneous mutagenesis of antibody CDR regions by overlap extension and PCR.; citation_author=N Hayashi; citation_author=M Welschof; citation_author=M Zewe; citation_author=M Braunagel; citation_author=S Dubel; citation_journal_title=Biotechniques; citation_volume=17; citation_number=8; citation_pages=310, 312, 314-315; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Dual asymmetric PCR: one-step construction of synthetic genes.; citation_author=GS Sandhu; citation_author=RA Aleff; citation_author=BC Kline; citation_journal_title=Biotechniques; citation_volume=12; citation_number=9; citation_pages=14-16; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Two-step total gene synthesis method.; citation_author=L Young; citation_author=Q Dong; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=10; citation_pages=e59; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides.; citation_author=WP Stemmer; citation_author=A Crameri; citation_author=KD Ha; citation_author=TM Brennan; citation_author=HL Heyneker; citation_journal_title=Gene; citation_volume=164; citation_number=11; citation_pages=49-53; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=DNAWorks: an automated method for designing oligonucleotides for PCR-based gene synthesis.; citation_author=DM Hoover; citation_author=J Lubkowski; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=12; citation_pages=e43; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PCR-based accurate synthesis of long DNA sequences.; citation_author=AS Xiong; citation_author=QH Yao; citation_author=RH Peng; citation_author=H Duan; citation_author=X Li; citation_journal_title=Nat Protoc; citation_volume=1; citation_number=13; citation_pages=791-797; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation, characterization, and molecular cloning of the cDNA encoding a novel phytase from Aspergillus niger 113 and high expression in Pichia pastoris.; citation_author=AS Xiong; citation_author=QH Yao; citation_author=RH Peng; citation_author=X Li; citation_author=HQ Fan; citation_journal_title=J Biochem Mol Biol; citation_volume=37; citation_number=14; citation_pages=282-291; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High level expression of a recombinant acid phytase gene in Pichia pastoris.; citation_author=AS Xiong; citation_author=QH Yao; citation_author=RH Peng; citation_author=PL Han; citation_author=ZM Cheng; citation_journal_title=J Appl Microbiol; citation_volume=98; citation_number=15; citation_pages=418-428; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A simple, rapid, high-fidelity and cost-effective PCR-based two-step DNA synthesis method for long gene sequences.; citation_author=AS Xiong; citation_author=QH Yao; citation_author=RH Peng; citation_author=X Li; citation_author=HQ Fan; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=16; citation_pages=e98; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=DNA shuffling by random fragmentation and reassembly: in vitro recombination for molecular evolution.; citation_author=WP Stemmer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=17; citation_pages=10747-10751; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Rapid evolution of a protein in vitro by DNA shuffling.; citation_author=WP Stemmer; citation_journal_title=Nature; citation_volume=370; citation_number=18; citation_pages=389-391; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=PCR-based gene synthesis as an efficient approach for expression of the A+T-rich malaria genome.; citation_author=C Withers-Martinez; citation_author=EP Carpenter; citation_author=F Hackett; citation_author=B Ely; citation_author=M Sajid; citation_journal_title=Protein Eng; citation_volume=12; citation_number=19; citation_pages=1113-1120; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=UpGene: Application of a web-based DNA codon optimization algorithm.; citation_author=W Gao; citation_author=A Rzewski; citation_author=H Sun; citation_author=PD Robbins; citation_author=A Gambotto; citation_journal_title=Biotechnol Prog; citation_volume=20; citation_number=20; citation_pages=443-448; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=EasyExonPrimer: automated primer design for exon sequences.; citation_author=X Wu; citation_author=DJ Munroe; citation_journal_title=Appl Bioinformatics; citation_volume=5; citation_number=21; citation_pages=119-120; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Correction of a mutation in a synthetic gene by DREAM technique, a site-directed mutagenesis.; citation_author=X Zhu; citation_author=J Chen; citation_author=X Zhang; citation_author=X An; citation_author=Y Zhou; citation_journal_title=China Biotechnology; citation_volume=27; citation_number=22; citation_pages=86-92; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=ELXR: a resource for rapid exon-directed sequence analysis.; citation_author=JJ Schageman; citation_author=CJ Horton; citation_author=S Niu; citation_author=HR Garner; citation_author=A Pertsemlidis; citation_journal_title=Genome Biol; citation_volume=5; citation_number=23; citation_pages=R36; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ExPrimer: to design primers from exon–exon junctions.; citation_author=KS Sandhu; citation_author=KK Acharya; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=24; citation_pages=2091-2092; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Method of artificial DNA splicing by directed ligation (SDL).; citation_author=EN Lebedenko; citation_author=KR Birikh; citation_author=OV Plutalov; citation_author=A Berlin Yu; citation_journal_title=Nucleic Acids Res; citation_volume=19; citation_number=25; citation_pages=6757-6761; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Engineering by PCR-based exon amplification of the genomic porcine interferon-gamma DNA for expression in Escherichia coli.; citation_author=K Vandenbroeck; citation_author=R Dijkmans; citation_author=A van Aerschot; citation_author=A Billiau; citation_journal_title=Biochem Biophys Res Commun; citation_volume=180; citation_number=26; citation_pages=1408-1415; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Elimination of introns from the interleukin 1 beta genome by DNA amplification and expression of interleukin 1 beta in Escherichia coli.; citation_author=DA Chistiakov; citation_author=SG Arsenian; citation_author=EA Braga; citation_author=AD Lapidus; citation_author=I Kozlov Iu; citation_journal_title=Mol Biol (Mosk); citation_volume=25; citation_number=27; citation_pages=1273-1284; citation_date=1991; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA"/>
    <meta name="twitter:description" content="BackgroundPolymerase chain reaction (PCR) is extensively applied in gene cloning. But due to the existence of introns, low copy number of particular genes and high complexity of the eukaryotic genome, it is usually impossible to amplify and clone a gene as a full-length sequence directly from the genome by ordinary PCR based techniques. Cloning of cDNA instead of genomic DNA involves multiple steps: harvest of tissues that express the gene of interest, RNA isolation, cDNA synthesis (reverse transcription), and PCR amplification. To simplify the cloning procedures and avoid the problems caused by ubiquitously distributed durable RNases, we have developed a novel strategy allowing the cloning of any cDNA or open reading frame (ORF) with wild type sequence in any spliced form from a single genomic DNA preparation.MethodologyOur &ldquo;Genomic DNA Splicing&rdquo; technique contains the following steps: first, all exons of the gene are amplified from a genomic DNA preparation, using software-optimized, highly efficient primers residing in flanking introns. Next, the tissue-specific exon sequences are assembled into one full-length sequence by overlapping PCR with deliberately designed primers located at the splicing sites. Finally, software-optimized outmost primers are exploited for efficient amplification of the assembled full-length products.ConclusionsThe &ldquo;Genomic DNA Splicing&rdquo; protocol avoids RNA preparation and reverse transcription steps, and the entire assembly process can be finished within hours. Since genomic DNA is more stable than RNA, it may be a more practical cloning strategy for many genes, especially the ones that are very large and difficult to generate a full length cDNA using oligo-dT primed reverse transcription. With this technique, we successfully cloned the full-length wild type coding sequence of human polymeric immunoglobulin receptor, which is 2295 bp in length and composed of 10 exons."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001179.g003"/>

  <meta property="og:title" content="Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4320'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9615'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6502&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001179">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001179" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001179&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiaoping An
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Xiaoping An, Jun Lu</p>

              
              

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jun Lu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Xiaoping An, Jun Lu</p>

              
              

                <p>Affiliation:
                  Beijing YouAn Hospital, Capital Medical University, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jian-dong Huang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jdhuang@hkucc.hku.hk">jdhuang@hkucc.hku.hk</a> (JH); <a href="mailto:tongyg@nic.bmi.ac.cn">tongyg@nic.bmi.ac.cn</a> (YT)</p>

                <p>Affiliation:
                  Department of Biochemistry, The University of Hong Kong, Hong Kong Special Administrative Region (SAR), China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Baozhong Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dabin Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xin Zhang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jinhui Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yusen Zhou, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yigang Tong
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:jdhuang@hkucc.hku.hk">jdhuang@hkucc.hku.hk</a> (JH); <a href="mailto:tongyg@nic.bmi.ac.cn">tongyg@nic.bmi.ac.cn</a> (YT)</p>

                <p>Affiliation:
                  State Key Laboratory of Pathogen and Biosecurity, Beijing Institute of Microbiology and Epidemiology, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001179</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001179-g001" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001179-g002" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001179-t001" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001179-g003" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001179">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001179" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Polymerase chain reaction (PCR) is extensively applied in gene cloning. But due to the existence of introns, low copy number of particular genes and high complexity of the eukaryotic genome, it is usually impossible to amplify and clone a gene as a full-length sequence directly from the genome by ordinary PCR based techniques. Cloning of cDNA instead of genomic DNA involves multiple steps: harvest of tissues that express the gene of interest, RNA isolation, cDNA synthesis (reverse transcription), and PCR amplification. To simplify the cloning procedures and avoid the problems caused by ubiquitously distributed durable RNases, we have developed a novel strategy allowing the cloning of any cDNA or open reading frame (ORF) with wild type sequence in any spliced form from a single genomic DNA preparation.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Our “Genomic DNA Splicing” technique contains the following steps: first, all exons of the gene are amplified from a genomic DNA preparation, using software-optimized, highly efficient primers residing in flanking introns. Next, the tissue-specific exon sequences are assembled into one full-length sequence by overlapping PCR with deliberately designed primers located at the splicing sites. Finally, software-optimized outmost primers are exploited for efficient amplification of the assembled full-length products.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The “Genomic DNA Splicing” protocol avoids RNA preparation and reverse transcription steps, and the entire assembly process can be finished within hours. Since genomic DNA is more stable than RNA, it may be a more practical cloning strategy for many genes, especially the ones that are very large and difficult to generate a full length cDNA using oligo-dT primed reverse transcription. With this technique, we successfully cloned the full-length wild type coding sequence of human polymeric immunoglobulin receptor, which is 2295 bp in length and composed of 10 exons.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>An X, Lu J, Huang J-d, Zhang B, Liu D, et al.  (2007) Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA. PLoS ONE 2(11):
          e1179.
            doi:10.1371/journal.pone.0001179</p><p><strong>Academic Editor: </strong>Xiaolin Wu, National Cancer Institute at Frederick, United States of America</p><p><strong>Received:</strong> August 22, 2007; <strong>Accepted:</strong> October 18, 2007; <strong>Published:</strong> November 14, 2007</p><p><strong>Copyright:</strong> © 2007 An et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported by National High Technology Research and Development Program of China (2001AA215361). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Gene cloning is one of the most frequently used technologies in a molecular biology laboratory. To study a particular gene, the first step is usually to clone and express it. However, most of the eukaryotic genes are interrupted by intervening sequences (introns), which make the gene of interest very large. Manipulation of the large genomic DNA is tedious and problematic due to size capacity of cloning vectors and multiple restriction endonucleases which make it difficult to find appropriate enzymes for subcloning. To circumvent these difficulties, the cDNA is often used instead of its large genomic counterpart. But cDNA cloning is usually troublesome, which involves mRNA preparation and reverse transcription, and thus requires RNA extraction kits and reverse transcription kits. For a particular cDNA cloning, a specific tissue with a relatively high level expression is often needed for the mRNA or total RNA extraction. Some of these tissues are quite rare and difficult to obtain, and some of the genes only have a very low level expression. Furthermore, RNases are ubiquitously expressed in tissues and RNase products are commonly used in molecular biology protocols (such as plasmid DNA preparation). Since RNases are very stable and difficult to remove completely, extra care must be taken when working with RNA. Lastly, RNAs are unstable polynucleotides and thus present their own problems in handling and storage.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>As the genomic DNA sequence and annotation databases skyrocketed in recent years, it has become easier to obtain information on virtually any gene and the corresponding protein sequence. With the advanced technology and low cost of oligonucleotide synthesis, <em>de novo</em> gene synthesis is becoming a common gene cloning option. Several gene synthesis strategies have been developed over the past three decades, including oligonucleotide ligation <a href="#pone.0001179-Heyneker1">[1]</a>–<a href="#pone.0001179-Goeddel1">[3]</a>, the <em>Fok</em>I method of gene synthesis<a href="#pone.0001179-Mandecki1">[4]</a>, <a href="#pone.0001179-Mandecki2">[5]</a>, self-priming PCR <a href="#pone.0001179-Dillon1">[6]</a>–<a href="#pone.0001179-Hayashi1">[8]</a>, dual asymmetric PCR<a href="#pone.0001179-Sandhu1">[9]</a>, combined dual asymmetric PCR and overlap extension PCR <a href="#pone.0001179-Young1">[10]</a>, assembly PCR <a href="#pone.0001179-Stemmer1">[11]</a>–<a href="#pone.0001179-Xiong1">[13]</a> and successive PCR <a href="#pone.0001179-Xiong2">[14]</a>–<a href="#pone.0001179-Xiong4">[16]</a>. Assembly PCR was originally adopted in DNA shuffling for <em>in vitro</em> evolution of DNA molecules <a href="#pone.0001179-Stemmer2">[17]</a>, <a href="#pone.0001179-Stemmer3">[18]</a>. With assembly PCR or successive PCR, any DNA sequence can be easily obtained <em>de novo</em> by assembling synthetic oligonucleotides. This technique makes DNA cloning very simple and is applied widely and extensively in molecular biology studies. To help design the assembly primers, as well as to optimize codon usage for various expression systems, some dedicated computer software <a href="#pone.0001179-WithersMartinez1">[19]</a> and internet online tools <a href="#pone.0001179-Hoover1">[12]</a>, <a href="#pone.0001179-Gao1">[20]</a> have been developed.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we report a novel PCR-based <em>in vitro</em> genomic DNA splicing strategy (designated as “PCR mediated <span class="underline">G</span>enomic <span class="underline">D</span>NA <span class="underline">S</span>plicing” or GDS strategy) for cloning of any eukaryotic cDNA or coding sequence from a genomic DNA preparation. This genomic DNA preparation serves as a universal PCR template and can be prepared from any tissue including peripheral blood cells. As for now, several human and mouse cDNAs have been cloned in our laboratories from a human genomic DNA template and a mouse genomic DNA template respectively, and here we take the multiple-exon human polymeric immunoglobulin receptor (<em>PIGR</em>) gene as an example to illustrate this novel cDNA cloning strategy.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Chemicals, enzymes and stains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><em>Taq</em> DNA polymerase, T4 DNA ligase, DL2000 DNA marker, Blood Genome DNA Extraction Kit, and restriction endonucleases were purchased from Takara Biotech Co., Ltd. (Dalian, China). Phusion high-fidelity DNA polymerase and T7 endonuclease I were from New England Biolabs, Inc. (Ipswich, MA). IPTG, X-Gal, and other chemicals were from Sigma-Aldrich, Inc. (St Louis, MO). Oligonucleotides were synthesized and purified by polyacrylamide gel electrophoresis by AuGCT Biotech Co., Ltd. (Beijing, China). TA cloning vector (pGEM®-T easy vector) was purchased from Promega Biotech Co., Ltd (Madison, WI). DH5α competent cells were from Tiangen Biotech Co., Ltd (Beijing, China). Glass milk DNA purification kits were from BioDev-Tech Co., Ltd (Beijing, China).</p>


<h4>Experimental design</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The principle of this strategy is composed of three steps (<a href="#pone-0001179-g001">Figure 1</a>): i) Amplification of all the exons of the desired gene with optimized primers within flanking introns. ii) Joining of exons by overlapping PCR. iii) Amplification of full-length products. The rationale of this strategy is as follows: All the exon sequences of a gene exist in the genome. But due to the low copy number (usually only one copy per genome) of every exon and the extremely high complexity of the genome, it is usually impossible to successfully amplify the full-length cDNA sequence by overlapping primers located at the precise splicing sites which are not possible for software optimization. However, it is always easy to obtain any single exon (usually together with some flanking intron sequences) from a genomic DNA preparation by PCR with software-optimized, highly efficient primers located in flanking introns. Then, by assembling the above exons with deliberately designed overlapping primers, it is easy to amplify the full-length cDNA or ORF of interest with software-optimized, highly efficient outmost primers.</p>
<div class="figure" id="pone-0001179-g001"><div class="img"><a name="pone-0001179-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001179.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001179.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schematic diagram of PCR mediated Genomic DNA Splicing (GDS) strategy.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Individual exons are amplified in the first PCR (exon PCR) following standard PCR protocol with software-optimized primers. A second PCR (overlapping PCR) is then carried out to join the exons and produce the full-length coding sequence. ATG and TAG indicate the start and stop codons, respectively.</p>
<span>doi:10.1371/journal.pone.0001179.g001</span></div>

<h4>Preparation of universal genomic DNA template</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A two hundred microliter whole blood sample was collected from a healthy donor and the genomic DNA was extracted with Blood Genome DNA Extraction Kit (Takara) according to the manufacturer's instructions. The purified DNA was dissolved in 50 µl TE buffer (10 mM Tris•Cl, pH 8.0, 1 mM EDTA) and its concentration was determined by an ultraviolet spectrophotometer (Milton Roy Spectronic 601, Bio-Rad Laboratories, Inc., Richmond, CA).</p>


<h4>Exon amplification and assembly</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Exon amplification was performed in individual tubes on a GeneAmp PCR system 2400 (Perkin Elmer, Foster City, CA). For each exon, a standard PCR was carried out with 25 µl total reaction mixture containing 50 mM KCl, 10 mM Tris-Cl, 1.5 mM MgCl<sub>2</sub>, 10 ng genomic DNA, 0.2 mM dNTP, 1 µM of each forward and reverse primer, and 0.5 unit Taq polymerase. The PCR parameters were as follows: pre-denaturation at 94°C for 3 minutes, followed by 35 cycles of denaturation at 94°C for 30 seconds, annealing at 55°C for 30 seconds, and polymerization at 72°C for 60 seconds, and a final incubation for 5 min at 72°C. The PCR products were pooled and used as a template in the subsequent overlapping PCR. In this second, overlapping PCR reaction, the overlapping PCR primer pairs of all the exons and the outmost primers (0.1 µM each) were pooled and used in a 25 µl reaction mixture containing 2 µl of the pooled template and the other components (see above). After 20 cycles of amplification (denaturation at 94°C for 30 seconds, annealing at 55°C for 30 seconds, and polymerization at 72°C for 120 seconds), 2 µl of the product was transferred into the final amplification tube with a total volume of 50 µl containing the other components as above and with the outmost primers to amplify the full-length ORF sequence following a standard PCR protocol. The PCR products were subjected to 1% agarose gel electrophoresis and the gel containing full-length DNA was excised and the DNA was purified with a glass milk kit (BioDev-Tech).</p>


<h4>PCR with the Phusion high-fidelity DNA polymerase</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>When the Phusion high-fidelity DNA polymerase was used in place of Taq polymerase, the thermocycle parameters were adjusted as follows: pre-denaturation at 98°C for 30 seconds, denaturation at 98°C for 15 seconds, and annealing at 60°C for 30 seconds, with other parameters unchanged. To add an A base at the 3′ terminus, the full-length PCR products were purified by extraction with equal volume phenol-chloroform-isoamyl alcohol (25:24:1) and precipitation with three volume ethanol. The DNA pellet was dissolved in 50 µl standard PCR reaction mixture (components see above, without primers) containing 1 unit Taq DNA polymerase. After incubation at 72°C for 20 minutes, the DNA was separated by 1% agarose gel electrophoresis and the full-length DNA was purified from the gel with a glass milk kit (BioDev-Tech).</p>


<h4>T7 endonuclease I treatment of PCR products</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The 3′ A tailed DNA products amplified by the Phusion high-fidelity DNA polymerase from two PCR reactions were dissolved in 15 µl distilled water and subjected to denaturation at 98°C for 1 minute, and then a stepwise annealing procedure was carried out as follows: 2 minutes at 85°C, 3 minutes at 75°C, 3 minutes at 65°C, 3 minutes at 37°C, followed by 3 minutes at room temperature. The T7 endonuclease I treatment was performed in a 20 µl solution containing the above DNA and 15 units of T7 endonuclease I as well as 1×T7 endonuclease I buffer. The mixture was incubated at 37°C for 1 hour to allow the mismatched molecules to be cleaved by the T7 endonuclease I.</p>


<h4>Cloning and sequencing</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The PCR products generated by Taq DNA polymerase or 3′ A tailed PCR products generated by the Phusion high-fidelity DNA polymerase were ligated with pGEM®-T easy vector (Promega). The ligation mixture was transformed into DH5α competent cells (Tiangen) and white colonies were picked and insertion was analyzed by PCR and restriction enzyme cleavage. Positive clones were subjected to DNA sequencing using an automated DNA sequencer ABI377 (Perkin Elmer).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Exon Primer Design</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>An online program EasyExonPrimer <a href="#pone.0001179-Wu1">[21]</a> (<a href="http://129.43.22.27/">http://129.43.22.27/</a>~primer/EasyExonPrimer.html), which is a free Internet service hosted by NCI, was used to design the exon amplification primers for the first round PCR. The human <em>PIGR</em> mRNA in NCBI GenBank database was retrieved, and the GenBank accession number as NM_002644 was obtained and submitted using this program. Default parameters were used for primer selection, i.e.: an optimal primer size of 22nt (18~25 nt); an optimal Tm of 60°C (45°C~65°C); an optimal primer GC percentage of 45 (30%~60%); an optimal product Tm of 60°C (45°C~65°C); a maximal self-complementarity of 8bp; and a maximal 3′ self-complementarity of 3bp. Exon 1 does not contain any coding sequence, and is excluded from amplification. Since exon 11 is longer than 1000 bp, which may make it difficult to be amplified, this exon sequence was broken into several smaller fragments for primer design. Only the first fragment of exon 11 contains coding sequence, thus only the primer pairs for this fragment were synthesized along with the primers of exon 2 though 10. The relative primer positions are represented in <a href="#pone-0001179-g002">Figure 2A</a> and the primer sequences are listed in <a href="#pone-0001179-t001">Table 1</a>.</p>
<div class="figure" id="pone-0001179-g002"><div class="img"><a name="pone-0001179-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001179.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001179.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Primer design for <em>PIGR</em> ORF cloning by GDS strategy.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>A. Exon primers for exon PCRs. All exons are represented by solid arrows and are flanked by optimized forward and reverse primers located in introns. The start codon resides in exon 2 and stop codon in exon 11. Exon 1 does not contain a coding sequence and was not amplified. For exon 11, only the region containing the coding sequence was amplified. B. Overlapping primers for overlapping PCR. C. Alignment of the overlapping primers with the cDNA sequence (only a portion is shown). Primers are boxed and their orientations are indicated by parallel arrows. Exon-exon junctions are marked by vertical lines.</p>
<span>doi:10.1371/journal.pone.0001179.g002</span></div><div class="figure" id="pone-0001179-t001"><div class="img"><a name="pone-0001179-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001179.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001179.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Primer sequences.</span></strong></p><span>doi:10.1371/journal.pone.0001179.t001</span></div>

<h4>Overlapping primer and outmost primer design</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Overlapping primers were manually designed at the exon-exon joining sites. To avoid a mismatch caused by an additional A base at the 3′ end produced by Taq polymerase, all the overlapping primers start after a T base. Other considerations for the overlapping primer design were no 3′ self-pairing and low 3′ G/C content. The length of the overlapping primers varied from 27 bp to 39 bp, with a minimal overlapping region of 15 bases. The outmost primers (OL2F and OL11R) for the full-length ORF amplification do not contain any overlapping sequence and were selected using computer software (Oligo version 6.71, Molecular Biology Insights, Inc., Cascade, CO). The positions of the overlapping primers and the outmost primers are illustrated in <a href="#pone-0001179-g002">Figure 2B</a> and their sequences are shown in <a href="#pone-0001179-t001">table 1</a>. <a href="#pone-0001179-g002">Figure 2C</a> shows alignment of the overlapping primers and the outmost primers with the <em>PIGR</em> coding sequence.</p>


<h4>Amplification of <em>PIGR</em> coding exons by standard PCR</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>A genomic DNA preparation was made from a blood sample of a healthy donor and was used as a universal exon PCR template. Since exon 1 and majority of exon 11 of <em>PIGR</em> gene do not contain any coding sequence, we only amplified exon 2 though 10 and the first fragment of exon 11, using a standard PCR protocol. In order to test the possibility of multiple exon assembly, we just used ordinary Taq DNA polymerase to perform the PCRs. Without any optimization of the PCR conditions, all of the exon PCRs were successful as demonstrated by 1% agarose gel electrophoresis (<a href="#pone-0001179-g003">Figure 3A</a>). The predicted length of exon 2 through exon 11 are 400 bp, 795 bp, 989 bp, 599 bp, 855 bp, 486 bp, 395 bp, 387 bp, 369 bp, 461 bp (fragment 1), respectively.</p>
<div class="figure" id="pone-0001179-g003"><div class="img"><a name="pone-0001179-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001179" data-uri="info:doi/10.1371/journal.pone.0001179.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001179.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001179.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001179.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001179.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001179.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Amplification, assembly and cloning of <em>PIGR</em> coding sequence by GDS strategy.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>A. Amplification of <em>PIGR</em> coding exons by standard PCR protocol with exon primers. The sizes of the amplified fragments of exon 2 through 11 are 400 bp, 795 bp, 989 bp, 599 bp, 855 bp, 486 bp, 395 bp, 387 bp, 369 bp, 461 bp, respectively. B. Amplification of full-length <em>PIGR</em> coding sequence. The size of the amplified full-length sequence is 2549 bp, including 2295 bp <em>PIGR</em> ORF and two short flanking sequences. C. Exon assembly of <em>PIGR</em> coding sequence. The start and stop codons are boxed. Exon-exon junctions are marked by vertical lines. The point mutations mentioned in the text are indicated by the mutated bases in parenthesis under the wild type ones. DL2000, DNA molecular weight marker (Takara).</p>
<span>doi:10.1371/journal.pone.0001179.g003</span></div>

<h4>Assembly of full-length <em>PIGR</em> ORF by overlapping PCR</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The above exon PCR products were pooled without purification and used as templates for a second round overlapping PCR to assemble the full-length <em>PIGR</em> coding sequence. To amplify the full-length PCR product, the outmost primers OL2F and OL11R were used to perform another round standard PCR. Agarose gel electrophoresis analysis of the final PCR product showed that dominant amplification product was a ~2500 bp band (2295 bp <em>PIGR</em> ORF plus two short flanking sequences) (<a href="#pone-0001179-g003">Figure 3B</a>). To simplify the full-length product amplification, we tried to combine the 20-cycle overlapping PCR and the final PCR by running only a single PCR step with normal amount of outmost primers and reduced amount of overlapping primers (each overlapping primer concentration equals one-fiftieth of the outmost primer concentration). We found this simplified protocol could amplify the full-length product, but with a reduced amount of final product.</p>


<h4>Sequence analysis of <em>PIGR</em> cDNA clones</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We cloned the full-length <em>PIGR</em> coding sequence into pGEM®-T Easy Vector. Five clones were picked up for DNA sequencing analysis and the results showed that all 5 clones contained correctly joined full-length <em>PIGR</em> coding sequence (<a href="#pone-0001179-g003">Figure 3C</a>), and since we used the error-prone Taq DNA polymerase in the above PCRs, there were an average of 2.5 point mutations per clone (the mutation rate is about 0.1%). Of all these mutations, nearly half located within the overlapping primers. Since the overlapping primers only compose a small fraction (much less than 50%) of the whole sequence, it suggests that synthetic oligonucleotides have a relatively high error rate<a href="#pone.0001179-Young1">[10]</a>, <a href="#pone.0001179-Stemmer1">[11]</a>. One of the clones had only two silent mutations and thus encoded wild type human polymeric immunoglobulin receptor (<a href="#pone-0001179-g003">Figure 3C</a>).</p>


<h4>Cloning of wild type cDNA with a high-fidelity thermostable DNA polymerase</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>In many cases wild type DNA sequences are required. In order to get wild type cDNA sequence, we then used the high-fidelity thermostable DNA polymerase Phusion DNA polymerase (New England Biolabs) to amplify the target DNA sequence in place of error-prone Taq polymerase. We sequenced three clones derived from Phusion DNA polymerase amplification and found two of them had the wild type sequence. But the other clone still contained a point mutation (G/A transition) at the location of the overlapping primer Ex3R (the mutation rate is about 0.01%) (<a href="#pone-0001179-g003">Figure 3C</a>). To further improve the method to get rid of possible point mutations, we applied T7 endonuclease I (New England Biolabs) to pre-treat the annealed full length PCR products before ligation into pGEM®-T Easy Vector. Since T7 endonuclease I hydrolyze the loop formed by mismatch between the mutant strand and the wide strand, it can prevent the error PCR products from ligation with T vectors. After transformation of the above ligation mixture, three positive clones were subjected to DNA sequencing. The results showed that all the three clones contained correctly assembled, full-length wild type <em>PIGR</em> coding sequence.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Similarly, we have cloned full-length coding sequences of human immunoglobulin J polypeptide gene (<em>IGJ</em>, NM_144646, 480 bp), human high-mobility group box 1 gene (<em>HMGB1</em>, NM_002128, 612 bp) and mouse urokinase-type plasminogen activator gene (<em>uPA,</em> NM_008873, 1302 bp). These successful cloning experiments suggest that our “Genomic DNA Splicing” technique is reliable and practicable.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>PCR-based <em>de novo</em> gene synthesis is now frequently used in molecular biology laboratories worldwide, with the free online tools to automate the primer design and, more importantly, to optimize the codon usage for efficient expression in intended organism, such as for <em>E.coli</em> or yeast expression <a href="#pone.0001179-Hoover1">[12]</a>, <a href="#pone.0001179-Gao1">[20]</a>. However, this gene synthesis strategy (known as “assembly PCR”) requires large amounts of chemically synthesized oligonucleotides, multiple rounds of PCRs, and often results in numerous unwanted point mutations due to the large number of synthesized bases and multiple rounds of PCRs<a href="#pone.0001179-Young1">[10]</a>, <a href="#pone.0001179-Stemmer1">[11]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Here we reported a new gene synthesis strategy, “PCR mediated Genomic DNA Splicing” (GDS), which resembles “assembly PCR” in procedures but takes advantage of genomic DNA to avoid synthesis of large amount of oligonucleotides, thus is cost-effective and can reduce the unwanted mutations introduced by chemical synthesis of large numbers of relatively long oligonucleotides<a href="#pone.0001179-Young1">[10]</a>. In the case of <em>PIGR</em> ORF (2549 bp) cloning, if the “assembly PCR” method had been used, we had to synthesize about 5000 bases, and all of them would contribute to the final sequence. When the GDS method was used, only 1058 bases were synthesized, and among them, only 549 bases (less than one-ninth of the “assembly PCR”) contributed to the final sequence. It has been reported that assembly PCR results in a point mutation rate of 0.1% even when the proof reading thermostable DNA polymerase Pfu was used <a href="#pone.0001179-Stemmer1">[11]</a>. In our previous assembly PCR experiment using Taq polymerase, which is error-prone, we found an average of 3 point mutations in a 720bp <em>de novo</em> synthesized sequence (the mutation rate was 0.4%) <a href="#pone.0001179-Zhu1">[22]</a>. However when we used the Taq polymerase to perform GDS cloning, the mutation rate was only about 0.1%.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>In order to enhance the fidelity of our method and get wild type sequences, we tried to apply a high-fidelity thermostable DNA polymerase (namely Phusion high-fidelity DNA polymerase) for the PCRs. Since this DNA polymerase possesses a 3′-5′ proof-reading endonuclease activity, its mutation rate is 50-fold lower than that of Taq DNA polymerase and 6-fold lower than that of <em>Pyrococcus furiosus</em> (Pfu) DNA polymerase (as claimed by the manufacturer). To further reduce the mutation rate, we employed the T7 endonuclease I to cleave the mismatched loop formed between the mutated sequence and the wild type sequence after the PCR products were denatured and then annealed. Our results showed that the mutation rate was reduced to about 0.01% when the Phusion high-fidelity DNA polymerase was used alone, and no point mutation was identified when the T7 endonuclease I was used as a combination with Phusion high-fidelity DNA polymerase. For the 2549 bp DNA sequence cloned in this report, wild type sequences were obtained in both experiments with the high-fidelity DNA polymerase.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>One advantage of the cDNA cloning strategy described in this report is that neither RNA extraction nor reverse transcription is needed, and thus it saves the costs of RNA extraction kits and cDNA synthesis kits. The second advantage is that it is not necessary to handle any specific tissues for a particular gene and the expression level of the gene is not a problem. The third advantage is that high fidelity DNA sequence is obtained with high efficiency while the process is simple and easy. The entire assembly process takes only a few hours. We found that even if some of the first round PCRs were not very successful (PCR product bands were not visible in ethidium bromide stained agarose gel), it usually did not affect the final full-length DNA amplification. So it is usually not necessary to optimize PCR conditions even if one or more exon primer pairs do not work very well in the first round PCR.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>In order to make the primer design easy, we searched the Internet for appropriate programs to automatically design exon amplification primers, and found a few useful applications, including Genomic Primers (<a href="http://www2.eur.nl/fgg/kgen/primer/Genomic_Primers.html">http://www2.eur.nl/fgg/kgen/primer/Genom​ic_Primers.html</a>), ELXR (<a href="http://ecom2.mwgdna.com/services/elxr/elxrdb_query.tcllocEBE">http://ecom2.mwgdna.com/services/elxr/el​xrdb_query.tcllocEBE</a>) <a href="#pone.0001179-Schageman1">[23]</a>, ampExon (<a href="http://bioinformatics.well.ox.ac.uk/mst-bin/ampexon.pl">http://bioinformatics.well.ox.ac.uk/mst-​bin/ampexon.pl</a>), ExonPrimer (<a href="http://ihg.gsf.de/ihg/ExonPrimer.html">http://ihg.gsf.de/ihg/ExonPrimer.html</a>), and EasyExonPrimer <a href="#pone.0001179-Wu1">[21]</a> (<a href="http://129.43.22.27/">http://129.43.22.27/</a>~primer/EasyExonPrimer.html). We chose to use EasyExonPrimer because it is highly automatic, very easy to use and free for use. This online tool is written in PERL and is based on a common PCR primer design program Primer3. We found that this online tool produced very efficient primers for exon amplification. All of our exon primers are designed by this software and almost all of them have successfully amplified the target sequences, including the exons of human <em>PIGR</em>, <em>IGJ</em>, <em>HMGB1</em> and mouse <em>uPA</em> genes, without PCR condition optimization. To design the exon-exon junction primers (overlapping primers, or splicing primers), we could not find any appropriate program. We tried to use an online application ExPrimer <a href="#pone.0001179-Sandhu2">[24]</a>, which is a web based computer program to design primers from exon-exon junctions of a specific gene transcript, but the output exon-exon junction primers need to be paired by other primers located in exons, and these primers are not suitable for overlapping PCR. So we just designed all of our overlapping primers manually.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>As for our knowledge, this is the first report of a rapid and efficient method to clone wild type multiple-exon cDNA sequences directly from a single genomic DNA preparation. Lebedenko <em>et al</em> reported an <em>in vitro</em> exon ligation method, but it involved complicated steps including two-step PCR to amplify the exons, cloning of all exons separately, introduction of specific restriction enzymes and ligation of multiple restriction fragments<a href="#pone.0001179-Lebedenko1">[25]</a>. A similar PCR-based exon ligation method was proposed by Vandenbroeck <em>et al</em>, which required multiple rounds of PCR and multiple ligation reactions of blunt-end PCR products and restriction fragments<a href="#pone.0001179-Vandenbroeck1">[26]</a>. Chistiakov <em>et al</em> used a PCR procedure to assemble exons of human interleukin 1 gene, but the exons were obtained from cloned DNA and only 3 exons were assembled by two rounds overlapping PCR, with each round of PCR to join only two exons (exon 5 and 6 were joined first, and then exon 5+6 was joined with exon 7)<a href="#pone.0001179-Chistiakov1">[27]</a>.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Thus we have developed a time- and cost-effective method for multiple-exon cDNA or ORF cloning directly from genomic DNA, which does not require tissue handling, RNA preparation and reverse transcription, and efficiently produces wild type full-length cDNA or ORF sequences.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr. Jan Melenhorst from the Hematology Branch, NHLBI, NIH for critically reading and editing the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YT JH. Performed the experiments: YT XA JL DL JC BZ XZ. Analyzed the data: YT XA JL. Contributed reagents/materials/analysis tools: YT YZ. Wrote the paper: YT JH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001179-Heyneker1" id="pone.0001179-Heyneker1"></a>Heyneker HL, Shine J, Goodman HM, Boyer HW, Rosenberg J, et al.  (1976) Synthetic lac operator DNA is functional in vivo. Nature  263: 748–752.  <ul class="find" data-citedArticleID="1017790" data-doi="10.1038/263748a0"><li><a href="http://dx.doi.org/10.1038/263748a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthetic+lac+operator+DNA+is+functional+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthetic+lac+operator+DNA+is+functional+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001179-Itakura1" id="pone.0001179-Itakura1"></a>Itakura K, Hirose T, Crea R, Riggs AD, Heyneker HL, et al.  (1977) Expression in Escherichia coli of a chemically synthesized gene for the hormone somatostatin. Science  198: 1056–1063.  <ul class="find" data-citedArticleID="1017794" data-doi="10.1126/science.412251"><li><a href="http://dx.doi.org/10.1126/science.412251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+in+Escherichia+coli+of+a+chemically+synthesized+gene+for+the+hormone+somatostatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+in+Escherichia+coli+of+a+chemically+synthesized+gene+for+the+hormone+somatostatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001179-Goeddel1" id="pone.0001179-Goeddel1"></a>Goeddel DV, Kleid DG, Bolivar F, Heyneker HL, Yansura DG, et al.  (1979) Expression in Escherichia coli of chemically synthesized genes for human insulin. Proc Natl Acad Sci U S A  76: 106–110.  <ul class="find" data-citedArticleID="1017786" data-doi="10.1073/pnas.76.1.106"><li><a href="http://dx.doi.org/10.1073/pnas.76.1.106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+in+Escherichia+coli+of+chemically+synthesized+genes+for+human+insulin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+in+Escherichia+coli+of+chemically+synthesized+genes+for+human+insulin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001179-Mandecki1" id="pone.0001179-Mandecki1"></a>Mandecki W, Bolling TJ (1988) FokI method of gene synthesis. Gene  68: 101–107.  <ul class="find" data-citedArticleID="1017798" data-doi="10.1016/0378-1119(88)90603-8"><li><a href="http://dx.doi.org/10.1016/0378-1119(88)90603-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FokI+method+of+gene+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FokI+method+of+gene+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001179-Mandecki2" id="pone.0001179-Mandecki2"></a>Mandecki W, Hayden MA, Shallcross MA, Stotland E (1990) A totally synthetic plasmid for general cloning, gene expression and mutagenesis in Escherichia coli. Gene  94: 103–107.  <ul class="find" data-citedArticleID="1017800" data-doi="10.1016/0378-1119(90)90474-6"><li><a href="http://dx.doi.org/10.1016/0378-1119(90)90474-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+totally+synthetic+plasmid+for+general+cloning%2C+gene+expression+and+mutagenesis+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+totally+synthetic+plasmid+for+general+cloning%2C+gene+expression+and+mutagenesis+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001179-Dillon1" id="pone.0001179-Dillon1"></a>Dillon PJ, Rosen CA (1990) A rapid method for the construction of synthetic genes using the polymerase chain reaction. Biotechniques  9: 298, 300.  <ul class="find" data-citedArticleID="1017782" data-doi="10.1385/0-89603-244-2:263"><li><a href="http://dx.doi.org/10.1385/0-89603-244-2:263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid+method+for+the+construction+of+synthetic+genes+using+the+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid+method+for+the+construction+of+synthetic+genes+using+the+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001179-Prodromou1" id="pone.0001179-Prodromou1"></a>Prodromou C, Pearl LH (1992) Recursive PCR: a novel technique for total gene synthesis. Protein Eng  5: 827–829.  <ul class="find" data-citedArticleID="1017802" data-doi="10.1093/protein/5.8.827"><li><a href="http://dx.doi.org/10.1093/protein/5.8.827" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recursive+PCR%3A+a+novel+technique+for+total+gene+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recursive+PCR%3A+a+novel+technique+for+total+gene+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001179-Hayashi1" id="pone.0001179-Hayashi1"></a>Hayashi N, Welschof M, Zewe M, Braunagel M, Dubel S, et al.  (1994) Simultaneous mutagenesis of antibody CDR regions by overlap extension and PCR. Biotechniques  17: 310, 312, 314–315.  <ul class="find" data-citedArticleID="1017788"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Simultaneous+mutagenesis+of+antibody+CDR+regions+by+overlap+extension+and+PCR.&amp;auth=&amp;atitle=Simultaneous+mutagenesis+of+antibody+CDR+regions+by+overlap+extension+and+PCR." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Simultaneous+mutagenesis+of+antibody+CDR+regions+by+overlap+extension+and+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Simultaneous+mutagenesis+of+antibody+CDR+regions+by+overlap+extension+and+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001179-Sandhu1" id="pone.0001179-Sandhu1"></a>Sandhu GS, Aleff RA, Kline BC (1992) Dual asymmetric PCR: one-step construction of synthetic genes. Biotechniques  12: 14–16.  <ul class="find" data-citedArticleID="1017804"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Dual+asymmetric+PCR%3A+one-step+construction+of+synthetic+genes.&amp;auth=&amp;atitle=Dual+asymmetric+PCR%3A+one-step+construction+of+synthetic+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dual+asymmetric+PCR%3A+one-step+construction+of+synthetic+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dual+asymmetric+PCR%3A+one-step+construction+of+synthetic+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001179-Young1" id="pone.0001179-Young1"></a>Young L, Dong Q (2004) Two-step total gene synthesis method. Nucleic Acids Res  32: e59.  <ul class="find" data-citedArticleID="1017830" data-doi="10.1093/nar/gnh058"><li><a href="http://dx.doi.org/10.1093/nar/gnh058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two-step+total+gene+synthesis+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two-step+total+gene+synthesis+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001179-Stemmer1" id="pone.0001179-Stemmer1"></a>Stemmer WP, Crameri A, Ha KD, Brennan TM, Heyneker HL (1995) Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides. Gene  164: 49–53.  <ul class="find" data-citedArticleID="1017810" data-doi="10.1016/0378-1119(95)00511-4"><li><a href="http://dx.doi.org/10.1016/0378-1119(95)00511-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-step+assembly+of+a+gene+and+entire+plasmid+from+large+numbers+of+oligodeoxyribonucleotides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-step+assembly+of+a+gene+and+entire+plasmid+from+large+numbers+of+oligodeoxyribonucleotides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001179-Hoover1" id="pone.0001179-Hoover1"></a>Hoover DM, Lubkowski J (2002) DNAWorks: an automated method for designing oligonucleotides for PCR-based gene synthesis. Nucleic Acids Res  30: e43.  <ul class="find" data-citedArticleID="1017792" data-doi="10.1093/nar/30.10.e43"><li><a href="http://dx.doi.org/10.1093/nar/30.10.e43" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNAWorks%3A+an+automated+method+for+designing+oligonucleotides+for+PCR-based+gene+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNAWorks%3A+an+automated+method+for+designing+oligonucleotides+for+PCR-based+gene+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001179-Xiong1" id="pone.0001179-Xiong1"></a>Xiong AS, Yao QH, Peng RH, Duan H, Li X, et al.  (2006) PCR-based accurate synthesis of long DNA sequences. Nat Protoc  1: 791–797.  <ul class="find" data-citedArticleID="1017822" data-doi="10.1038/nprot.2006.103"><li><a href="http://dx.doi.org/10.1038/nprot.2006.103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PCR-based+accurate+synthesis+of+long+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PCR-based+accurate+synthesis+of+long+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001179-Xiong2" id="pone.0001179-Xiong2"></a>Xiong AS, Yao QH, Peng RH, Li X, Fan HQ, et al.  (2004) Isolation, characterization, and molecular cloning of the cDNA encoding a novel phytase from Aspergillus niger 113 and high expression in Pichia pastoris. J Biochem Mol Biol  37: 282–291.  <ul class="find" data-citedArticleID="1017824" data-doi="10.5483/bmbrep.2004.37.3.282"><li><a href="http://dx.doi.org/10.5483/bmbrep.2004.37.3.282" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation%2C+characterization%2C+and+molecular+cloning+of+the+cDNA+encoding+a+novel+phytase+from+Aspergillus+niger+113+and+high+expression+in+Pichia+pastoris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation%2C+characterization%2C+and+molecular+cloning+of+the+cDNA+encoding+a+novel+phytase+from+Aspergillus+niger+113+and+high+expression+in+Pichia+pastoris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001179-Xiong3" id="pone.0001179-Xiong3"></a>Xiong AS, Yao QH, Peng RH, Han PL, Cheng ZM, et al.  (2005) High level expression of a recombinant acid phytase gene in Pichia pastoris. J Appl Microbiol  98: 418–428.  <ul class="find" data-citedArticleID="1017826" data-doi="10.1111/j.1365-2672.2004.02476.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2672.2004.02476.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+level+expression+of+a+recombinant+acid+phytase+gene+in+Pichia+pastoris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+level+expression+of+a+recombinant+acid+phytase+gene+in+Pichia+pastoris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001179-Xiong4" id="pone.0001179-Xiong4"></a>Xiong AS, Yao QH, Peng RH, Li X, Fan HQ, et al.  (2004) A simple, rapid, high-fidelity and cost-effective PCR-based two-step DNA synthesis method for long gene sequences. Nucleic Acids Res  32: e98.  <ul class="find" data-citedArticleID="1017828" data-doi="10.1093/nar/gnh094"><li><a href="http://dx.doi.org/10.1093/nar/gnh094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+rapid%2C+high-fidelity+and+cost-effective+PCR-based+two-step+DNA+synthesis+method+for+long+gene+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+rapid%2C+high-fidelity+and+cost-effective+PCR-based+two-step+DNA+synthesis+method+for+long+gene+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001179-Stemmer2" id="pone.0001179-Stemmer2"></a>Stemmer WP (1994) DNA shuffling by random fragmentation and reassembly: in vitro recombination for molecular evolution. Proc Natl Acad Sci U S A  91: 10747–10751.  <ul class="find" data-citedArticleID="1017812" data-doi="10.1073/pnas.91.22.10747"><li><a href="http://dx.doi.org/10.1073/pnas.91.22.10747" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+shuffling+by+random+fragmentation+and+reassembly%3A+in+vitro+recombination+for+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+shuffling+by+random+fragmentation+and+reassembly%3A+in+vitro+recombination+for+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001179-Stemmer3" id="pone.0001179-Stemmer3"></a>Stemmer WP (1994) Rapid evolution of a protein in vitro by DNA shuffling. Nature  370: 389–391.  <ul class="find" data-citedArticleID="1017814" data-doi="10.1038/370389a0"><li><a href="http://dx.doi.org/10.1038/370389a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+evolution+of+a+protein+in+vitro+by+DNA+shuffling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+evolution+of+a+protein+in+vitro+by+DNA+shuffling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001179-WithersMartinez1" id="pone.0001179-WithersMartinez1"></a>Withers-Martinez C, Carpenter EP, Hackett F, Ely B, Sajid M, et al.  (1999) PCR-based gene synthesis as an efficient approach for expression of the A+T-rich malaria genome. Protein Eng  12: 1113–1120.  <ul class="find" data-citedArticleID="1017818" data-doi="10.1093/protein/12.12.1113"><li><a href="http://dx.doi.org/10.1093/protein/12.12.1113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PCR-based+gene+synthesis+as+an+efficient+approach+for+expression+of+the+A%2BT-rich+malaria+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PCR-based+gene+synthesis+as+an+efficient+approach+for+expression+of+the+A%2BT-rich+malaria+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001179-Gao1" id="pone.0001179-Gao1"></a>Gao W, Rzewski A, Sun H, Robbins PD, Gambotto A (2004) UpGene: Application of a web-based DNA codon optimization algorithm. Biotechnol Prog  20: 443–448.  <ul class="find" data-citedArticleID="1017784" data-doi="10.1021/bp0300467"><li><a href="http://dx.doi.org/10.1021/bp0300467" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=UpGene%3A+Application+of+a+web-based+DNA+codon+optimization+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22UpGene%3A+Application+of+a+web-based+DNA+codon+optimization+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001179-Wu1" id="pone.0001179-Wu1"></a>Wu X, Munroe DJ (2006) EasyExonPrimer: automated primer design for exon sequences. Appl Bioinformatics  5: 119–120.  <ul class="find" data-citedArticleID="1017820" data-doi="10.2165/00822942-200605020-00007"><li><a href="http://dx.doi.org/10.2165/00822942-200605020-00007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EasyExonPrimer%3A+automated+primer+design+for+exon+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EasyExonPrimer%3A+automated+primer+design+for+exon+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001179-Zhu1" id="pone.0001179-Zhu1"></a>Zhu X, Chen J, Zhang X, An X, Zhou Y, et al.  (2007) Correction of a mutation in a synthetic gene by DREAM technique, a site-directed mutagenesis. China Biotechnology  27: 86–92.  <ul class="find" data-citedArticleID="1017832"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Correction+of+a+mutation+in+a+synthetic+gene+by+DREAM+technique%2C+a+site-directed+mutagenesis.&amp;auth=&amp;atitle=Correction+of+a+mutation+in+a+synthetic+gene+by+DREAM+technique%2C+a+site-directed+mutagenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correction+of+a+mutation+in+a+synthetic+gene+by+DREAM+technique%2C+a+site-directed+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correction+of+a+mutation+in+a+synthetic+gene+by+DREAM+technique%2C+a+site-directed+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001179-Schageman1" id="pone.0001179-Schageman1"></a>Schageman JJ, Horton CJ, Niu S, Garner HR, Pertsemlidis A (2004) ELXR: a resource for rapid exon-directed sequence analysis. Genome Biol  5: R36.  <ul class="find" data-citedArticleID="1017808" data-doi="10.1186/gb-2004-5-5-r36"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-5-r36" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ELXR%3A+a+resource+for+rapid+exon-directed+sequence+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ELXR%3A+a+resource+for+rapid+exon-directed+sequence+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001179-Sandhu2" id="pone.0001179-Sandhu2"></a>Sandhu KS, Acharya KK (2005) ExPrimer: to design primers from exon–exon junctions. Bioinformatics  21: 2091–2092.  <ul class="find" data-citedArticleID="1017806" data-doi="10.1093/bioinformatics/bti304"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ExPrimer%3A+to+design+primers+from+exon%E2%80%93exon+junctions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ExPrimer%3A+to+design+primers+from+exon%E2%80%93exon+junctions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001179-Lebedenko1" id="pone.0001179-Lebedenko1"></a>Lebedenko EN, Birikh KR, Plutalov OV, Berlin Yu A (1991) Method of artificial DNA splicing by directed ligation (SDL). Nucleic Acids Res  19: 6757–6761.  <ul class="find" data-citedArticleID="1017796" data-doi="10.1093/nar/19.24.6757"><li><a href="http://dx.doi.org/10.1093/nar/19.24.6757" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Method+of+artificial+DNA+splicing+by+directed+ligation+%28SDL%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Method+of+artificial+DNA+splicing+by+directed+ligation+%28SDL%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001179-Vandenbroeck1" id="pone.0001179-Vandenbroeck1"></a>Vandenbroeck K, Dijkmans R, van Aerschot A, Billiau A (1991) Engineering by PCR-based exon amplification of the genomic porcine interferon-gamma DNA for expression in Escherichia coli. Biochem Biophys Res Commun  180: 1408–1415.  <ul class="find" data-citedArticleID="1017816" data-doi="10.1016/s0006-291x(05)81353-1"><li><a href="http://dx.doi.org/10.1016/s0006-291x(05)81353-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Engineering+by+PCR-based+exon+amplification+of+the+genomic+porcine+interferon-gamma+DNA+for+expression+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Engineering+by+PCR-based+exon+amplification+of+the+genomic+porcine+interferon-gamma+DNA+for+expression+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001179-Chistiakov1" id="pone.0001179-Chistiakov1"></a>Chistiakov DA, Arsenian SG, Braga EA, Lapidus AD, Kozlov Iu I, et al.  (1991) Elimination of introns from the interleukin 1 beta genome by DNA amplification and expression of interleukin 1 beta in Escherichia coli. Mol Biol (Mosk)  25: 1273–1284.  <ul class="find" data-citedArticleID="1017780"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Elimination+of+introns+from+the+interleukin+1+beta+genome+by+DNA+amplification+and+expression+of+interleukin+1+beta+in+Escherichia+coli.&amp;auth=&amp;atitle=Elimination+of+introns+from+the+interleukin+1+beta+genome+by+DNA+amplification+and+expression+of+interleukin+1+beta+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elimination+of+introns+from+the+interleukin+1+beta+genome+by+DNA+amplification+and+expression+of+interleukin+1+beta+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elimination+of+introns+from+the+interleukin+1+beta+genome+by+DNA+amplification+and+expression+of+interleukin+1+beta+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.XML" value="66334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.PDF" value="391185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g001.PNG_L" value="294502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g001.PNG_M" value="46271"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g001.PNG_S" value="6488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g001.TIF" value="674516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g001.PNG_I" value="17316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g002.PNG_L" value="1241739"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g002.PNG_M" value="207663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g002.PNG_S" value="13719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g002.TIF" value="1998624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g002.PNG_I" value="87668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.t001.PNG_L" value="231022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.t001.PNG_M" value="166155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.t001.PNG_S" value="13745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.t001.TIF" value="1143612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.t001.PNG_I" value="51808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g003.PNG_L" value="1832594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g003.PNG_M" value="215477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g003.PNG_S" value="10095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g003.TIF" value="2965580"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001179.g003.PNG_I" value="69094"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001179&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001179" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001179&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001179&volume=&issue=&title=Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA&author_name=Xiaoping%20An%2C%20Jun%20Lu%2C%20Jian-dong%20Huang%2C%20Baozhong%20Zhang%2C%20Dabin%20Liu%2C%20Xin%20Zhang%2C%20Jinhui%20Chen%2C%20Yusen%20Zhou%2C%20Yigang%20Tong&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001179" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179&amp;t=Rapid%20Assembly%20of%20Multiple-Exon%20cDNA%20Directly%20from%20Genomic%20DNA" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179&title=Rapid%20Assembly%20of%20Multiple-Exon%20cDNA%20Directly%20from%20Genomic%20DNA&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179&amp;title=Rapid%20Assembly%20of%20Multiple-Exon%20cDNA%20Directly%20from%20Genomic%20DNA" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001179&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Rapid Assembly of Multiple-Exon cDNA Directly from Genomic DNA';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001179';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Rapid%20Assembly%20of%20Multiple-Exon%20cDNA%20Directly%20from%20Genomic%20DNA http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001179" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001179" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cloning%22" title="Search for articles in the subject area:'Cloning'"><div class="flagText">Cloning</div></a>
              <div data-categoryid="23551" data-articleid="25190"
                   data-categoryname="Cloning"
                   class="flagImage" title="Flag 'Cloning' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="25190"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="25190"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+cloning%22" title="Search for articles in the subject area:'DNA cloning'"><div class="flagText">DNA cloning</div></a>
              <div data-categoryid="50051" data-articleid="25190"
                   data-categoryname="DNA cloning"
                   class="flagImage" title="Flag 'DNA cloning' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+polymerase%22" title="Search for articles in the subject area:'DNA polymerase'"><div class="flagText">DNA polymerase</div></a>
              <div data-categoryid="40357" data-articleid="25190"
                   data-categoryname="DNA polymerase"
                   class="flagImage" title="Flag 'DNA polymerase' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25190"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25190"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+assembly+tools%22" title="Search for articles in the subject area:'Sequence assembly tools'"><div class="flagText">Sequence assembly tools</div></a>
              <div data-categoryid="40553" data-articleid="25190"
                   data-categoryname="Sequence assembly tools"
                   class="flagImage" title="Flag 'Sequence assembly tools' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9301'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5675'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=166&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
