

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: PathogenMip Assay: A Multiplex Pathogen Detection Assay</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000223"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000223" />

    <meta name="citation_title" content="PathogenMip Assay: A Multiplex Pathogen Detection Assay"/>
    <meta itemprop="name" content="PathogenMip Assay: A Multiplex Pathogen Detection Assay"/>

      <meta name="citation_author" content="Michael S. Akhras"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden"/>
      <meta name="citation_author" content="Sreedevi Thiyagarajan"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Andrea C. Villablanca"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Ronald W. Davis"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>
      <meta name="citation_author" content="Pål Nyrén"/>
            <meta name="citation_author_institution" content="Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden"/>
      <meta name="citation_author" content="Nader Pourmand"/>
            <meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America"/>

    <meta name="citation_date" content="2007/2/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000223.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e223"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Improved amplification of genital human papillomaviruses.; citation_author=PE Gravitt; citation_author=CL Peyton; citation_author=TQ Alessi; citation_author=CM Wheeler; citation_author=F Coutlee; citation_journal_title=J Clin Microbiol; citation_volume=38; citation_number=1; citation_pages=357-361; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Present and future of rapid and/or high-throughput methods for nucleic acid testing.; citation_author=G Csako; citation_journal_title=Clin Chim Acta; citation_volume=363; citation_number=2; citation_pages=6-31; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Real-time PCR in virology.; citation_author=IM Mackay; citation_author=KE Arden; citation_author=A Nitsche; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=3; citation_pages=1292-1305; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=DNA microarray format for detection and subtyping of human papillomavirus.; citation_author=CH Klaassen; citation_author=CF Prinsen; citation_author=HA de Valk; citation_author=AM Horrevorts; citation_author=MA Jeunink; citation_journal_title=J Clin Microbiol; citation_volume=42; citation_number=4; citation_pages=2152-2160; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Sentinel-base DNA genotyping using multiple sequencing primers for high-risk human papillomaviruses.; citation_author=B Gharizadeh; citation_author=B Zheng; citation_author=M Akhras; citation_author=M Ghaderi; citation_author=O Jejelowo; citation_journal_title=Mol Cell Probes; citation_number=5; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Deletion screening of the Duchenne muscular dystrophy locus via multiplex DNA amplification.; citation_author=JS Chamberlain; citation_author=RA Gibbs; citation_author=JE Ranier; citation_author=PN Nguyen; citation_author=CT Caskey; citation_journal_title=Nucleic Acids Res; citation_volume=16; citation_number=6; citation_pages=11141-11156; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Multiplex PCR: optimization and application in diagnostic virology.; citation_author=EM Elnifro; citation_author=AM Ashshi; citation_author=RJ Cooper; citation_author=PE Klapper; citation_journal_title=Clin Microbiol Rev; citation_volume=13; citation_number=7; citation_pages=559-570; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiplex polymerase chain reaction: a practical approach.; citation_author=P Markoulatos; citation_author=N Siafakas; citation_author=M Moncany; citation_journal_title=J Clin Lab Anal; citation_volume=16; citation_number=8; citation_pages=47-51; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Next revolution in the molecular theranostics of infectious diseases: microfabricated systems for personalized medicine.; citation_author=L Bissonnette; citation_author=MG Bergeron; citation_journal_title=Expert Rev Mol Diagn; citation_volume=6; citation_number=9; citation_pages=433-450; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular inversion probe analysis of gene copy alterations reveals distinct categories of colorectal carcinoma.; citation_author=H Ji; citation_author=J Kumm; citation_author=M Zhang; citation_author=K Farnam; citation_author=K Salari; citation_journal_title=Cancer Res; citation_volume=66; citation_number=10; citation_pages=7910-7919; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A ligase-mediated gene detection technique.; citation_author=U Landegren; citation_author=R Kaiser; citation_author=J Sanders; citation_author=L Hood; citation_journal_title=Science; citation_volume=241; citation_number=11; citation_pages=1077-1080; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=DNA diagnostics–molecular techniques and automation.; citation_author=U Landegren; citation_author=R Kaiser; citation_author=CT Caskey; citation_author=L Hood; citation_journal_title=Science; citation_volume=242; citation_number=12; citation_pages=229-237; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Padlock probes: circularizing oligonucleotides for localized DNA detection.; citation_author=M Nilsson; citation_author=H Malmgren; citation_author=M Samiotaki; citation_author=M Kwiatkowski; citation_author=BP Chowdhary; citation_journal_title=Science; citation_volume=265; citation_number=13; citation_pages=2085-2088; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Padlock probes reveal single-nucleotide differences, parent of origin and in situ distribution of centromeric sequences in human chromosomes 13 and 21.; citation_author=M Nilsson; citation_author=K Krejci; citation_author=J Koch; citation_author=M Kwiatkowski; citation_author=P Gustavsson; citation_journal_title=Nat Genet; citation_volume=16; citation_number=14; citation_pages=252-255; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection of individual microbial pathogens by proximity ligation.; citation_author=SM Gustafsdottir; citation_author=A Nordengrahn; citation_author=S Fredriksson; citation_author=P Wallgren; citation_author=E Rivera; citation_journal_title=Clin Chem; citation_volume=52; citation_number=15; citation_pages=1152-1160; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Diagnostic application of padlock probes–multiplex detection of plant pathogens using universal microarrays.; citation_author=M Szemes; citation_author=P Bonants; citation_author=M de Weerdt; citation_author=J Baner; citation_author=U Landegren; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=16; citation_pages=e70; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiplexed genotyping with sequence-tagged molecular inversion probes.; citation_author=P Hardenbol; citation_author=J Baner; citation_author=M Jain; citation_author=M Nilsson; citation_author=EA Namsaraev; citation_journal_title=Nat Biotechnol; citation_volume=21; citation_number=17; citation_pages=673-678; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Highly multiplexed molecular inversion probe genotyping: over 10,000 targeted SNPs genotyped in a single tube assay.; citation_author=P Hardenbol; citation_author=F Yu; citation_author=J Belmont; citation_author=J Mackenzie; citation_author=C Bruckner; citation_journal_title=Genome Res; citation_volume=15; citation_number=18; citation_pages=269-275; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Allele quantification using molecular inversion probes (MIP).; citation_author=Y Wang; citation_author=M Moorhead; citation_author=G Karlin-Neumann; citation_author=M Falkowski; citation_author=C Chen; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=19; citation_pages=e183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Classification of papillomaviruses.; citation_author=EM de Villiers; citation_author=C Fauquet; citation_author=TR Broker; citation_author=HU Bernard; citation_author=H zur Hausen; citation_journal_title=Virology; citation_volume=324; citation_number=20; citation_pages=17-27; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular biology of human papillomavirus infection and cervical cancer.; citation_author=J Doorbar; citation_journal_title=Clin Sci (Lond); citation_volume=110; citation_number=21; citation_pages=525-541; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PathogenMIPer : A tool for the design of molecular inversion probes to detect multiple pathogens.; citation_author=S Thiyagarajan; citation_author=M Karhanek; citation_author=M Akhras; citation_author=RW Davis; citation_author=N Pourmand; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=22; citation_pages=500; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Methodological improvements of pyrosequencing technology.; citation_author=B Gharizadeh; citation_author=M Akhras; citation_author=N Nourizad; citation_author=M Ghaderi; citation_author=K Yasuda; citation_journal_title=J Biotechnol; citation_number=23; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fabrication of high quality microarrays.; citation_author=M Dufva; citation_journal_title=Biomol Eng; citation_volume=22; citation_number=24; citation_pages=173-184; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Detection of gyrA mutations associated with ciprofloxacin resistance in Neisseria gonorrhoeae by rapid and reliable pre-programmed short DNA sequencing.; citation_author=B Gharizadeh; citation_author=M Akhras; citation_author=M Unemo; citation_author=B Wretlind; citation_author=P Nyren; citation_journal_title=Int J Antimicrob Agents; citation_volume=26; citation_number=25; citation_pages=486-490; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nested PCR with the PGMY09/11 and GP5(+)/6(+) primer sets improves detection of HPV DNA in cervical samples.; citation_author=AL Fuessel Haws; citation_author=Q He; citation_author=PL Rady; citation_author=L Zhang; citation_author=J Grady; citation_journal_title=J Virol Methods; citation_volume=122; citation_number=26; citation_pages=87-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of synthetic DNA bar codes in Saccharomyces cerevisiae gene-deletion strains.; citation_author=RG Eason; citation_author=N Pourmand; citation_author=W Tongprasit; citation_author=ZS Herman; citation_author=K Anthony; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=27; citation_pages=11046-11051; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ligation-based DNA diagnostics.; citation_author=U Landegren; citation_journal_title=Bioessays; citation_volume=15; citation_number=28; citation_pages=761-765; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The solution structure of a locked nucleic acid (LNA) hybridized to DNA.; citation_author=CB Nielsen; citation_author=SK Singh; citation_author=J Wengel; citation_author=JP Jacobsen; citation_journal_title=J Biomol Struct Dyn; citation_volume=17; citation_number=29; citation_pages=175-191; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A unique and universal molecular barcode array.; citation_author=SE Pierce; citation_author=EL Fung; citation_author=DF Jaramillo; citation_author=AM Chu; citation_author=RW Davis; citation_journal_title=Nat Methods; citation_volume=3; citation_number=30; citation_pages=601-603; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tri-nucleotide threading for parallel amplification of minute amounts of genomic DNA.; citation_author=E Pettersson; citation_author=M Lindskog; citation_author=J Lundeberg; citation_author=A Ahmadian; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=31; citation_pages=e49; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Rolling replication of short DNA circles.; citation_author=A Fire; citation_author=SQ Xu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=32; citation_pages=4641-4645; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The biotin-streptavidin interaction can be reversibly broken using water at elevated temperatures.; citation_author=A Holmberg; citation_author=A Blomstergren; citation_author=O Nord; citation_author=M Lukacs; citation_author=J Lundeberg; citation_journal_title=Electrophoresis; citation_volume=26; citation_number=33; citation_pages=501-510; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Design considerations and effects of LNA in PCR primers.; citation_author=D Latorra; citation_author=K Arar; citation_author=JM Hurley; citation_journal_title=Mol Cell Probes; citation_volume=17; citation_number=34; citation_pages=253-259; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=E6 and E7 oncoproteins induce distinct patterns of chromosomal aneuploidy in skin tumors from transgenic mice.; citation_author=AJ Schaeffer; citation_author=M Nguyen; citation_author=A Liem; citation_author=D Lee; citation_author=C Montagna; citation_journal_title=Cancer Res; citation_volume=64; citation_number=35; citation_pages=538-546; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="PathogenMip Assay: A Multiplex Pathogen Detection Assay"/>
    <meta name="twitter:description" content="The Molecular Inversion Probe (MIP) assay has been previously applied to a large-scale human SNP detection. Here we describe the PathogenMip Assay, a complete protocol for probe production and applied approaches to pathogen detection. We have demonstrated the utility of this assay with an initial set of 24 probes targeting the most clinically relevant HPV genotypes associated with cervical cancer progression. Probe construction was based on a novel, cost-effective, ligase-based protocol. The assay was validated by performing pyrosequencing and Microarray chip detection in parallel experiments. HPV plasmids were used to validate sensitivity and selectivity of the assay. In addition, 20 genomic DNA extracts from primary tumors were genotyped with the PathogenMip Assay results and were in 100% agreement with conventional sequencing using an L1-based HPV genotyping protocol. The PathogenMip Assay is a widely accessible protocol for producing and using highly discriminating probes, with experimentally validated results in pathogen genotyping, which could potentially be applied to the detection and characterization of any microbe."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000223.g006"/>

  <meta property="og:title" content="PathogenMip Assay: A Multiplex Pathogen Detection Assay" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1429'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8735'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=224&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000223">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000223" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000223&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    PathogenMip Assay: A Multiplex Pathogen Detection Assay
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael S. Akhras, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America, 
                  Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sreedevi Thiyagarajan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrea C. Villablanca, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ronald W. Davis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pål Nyrén, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biotechnology, Royal Institute of Technology, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nader Pourmand
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pourmand@stanford.edu">pourmand@stanford.edu</a></p>

                <p>Affiliation:
                  Stanford Genome Technology Center, Stanford University, Palo Alto, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000223</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000223-g001" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-g002" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-g003" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-g004" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-g005" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-g006" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-t001" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-t002" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000223-t003" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000223">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000223" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The Molecular Inversion Probe (MIP) assay has been previously applied to a large-scale human SNP detection. Here we describe the PathogenMip Assay, a complete protocol for probe production and applied approaches to pathogen detection. We have demonstrated the utility of this assay with an initial set of 24 probes targeting the most clinically relevant HPV genotypes associated with cervical cancer progression. Probe construction was based on a novel, cost-effective, ligase-based protocol. The assay was validated by performing pyrosequencing and Microarray chip detection in parallel experiments. HPV plasmids were used to validate sensitivity and selectivity of the assay. In addition, 20 genomic DNA extracts from primary tumors were genotyped with the PathogenMip Assay results and were in 100% agreement with conventional sequencing using an L1-based HPV genotyping protocol. The PathogenMip Assay is a widely accessible protocol for producing and using highly discriminating probes, with experimentally validated results in pathogen genotyping, which could potentially be applied to the detection and characterization of any microbe.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Akhras MS, Thiyagarajan S, Villablanca AC, Davis RW, Nyrén P, et al.  (2007) PathogenMip Assay: A Multiplex Pathogen Detection Assay. PLoS ONE 2(2):
          e223.
            doi:10.1371/journal.pone.0000223</p><p><strong>Academic Editor: </strong>Sheila Bowyer, National Institute for Communicable Diseases, South Africa</p><p><strong>Received:</strong> December 4, 2006; <strong>Accepted:</strong> January 26, 2007; <strong>Published:</strong> February 21, 2007</p><p><strong>Copyright:</strong> © 2007 Akhras et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by a grant from National Institutes of Health (PO1-HG000205). Michael S. Akhras was supported by: The Foundation Blanceflor Boncompagni-Ludovisi, née Bildt, The Fulbright Commission, Hans Werthén Foundation and Sixten Gemzéus Scholarship Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Multiplex assays to identify the presence of a wide variety of microbes within a specimen will have a profound impact on the efficient management of disease treatments and prevention, clinical follow-up studies, and the development of new therapies of prophylactic or therapeutic vaccines. Here we describe a complete protocol for a multiplex assay in single-tube reactions useful in pathogen diagnostics. We compare our results to those obtained from a conventional general-primer based amplification <a href="#pone.0000223-Gravitt1">[1]</a> approach when genotyping primary tumor genomic DNA extracts from cervical cancers.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Nucleic-acid based diagnostic techniques typically rely on unbiased PCR amplification <a href="#pone.0000223-Csako1">[2]</a> of the targets, which requires a set of general PCR amplification primers to hybridize upstream and downstream of the variable region, followed by either real-time PCR <a href="#pone.0000223-Mackay1">[3]</a>, hybridization assays <a href="#pone.0000223-Klaassen1">[4]</a> or template sequencing <a href="#pone.0000223-Gharizadeh1">[5]</a> for read-out interpretation. Multiplex PCR <a href="#pone.0000223-Chamberlain1">[6]</a>, has the potential to decrease cost, time and effort in pathogen diagnostics <a href="#pone.0000223-Elnifro1">[7]</a>. Engineering an efficient multiplex PCR requires laborious strategic planning in primer design, nucleotide concentrations, optimal salt and buffer conditions, and use of chemical adjuvants <a href="#pone.0000223-Markoulatos1">[8]</a> and is rarely capable of achieving multiplex degrees greater than a 20-plex.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The field of molecular diagnostics is experiencing a revolution in regards to theranostics, the integration of diagnostic technologies with therapeutic applications <a href="#pone.0000223-Bissonnette1">[9]</a>. As the number of biomarkers for microbial agents and disease markers <a href="#pone.0000223-Ji1">[10]</a> increases, simultaneous detection of multiple agents or risk factors implicated in particular clinical syndromes or diseases that share similar epidemiological features will become highly desirable. Groundbreaking research has been performed with the advent of a ligase-mediated gene detection technique <a href="#pone.0000223-Landegren1">[11]</a> and the many technologies based on the work of Landegren <em>et al</em> <a href="#pone.0000223-Landegren2">[12]</a>. Ligase-based technology has led to myriad promising techniques such as the padlock probe <a href="#pone.0000223-Nilsson1">[13]</a>, <a href="#pone.0000223-Nilsson2">[14]</a>, and the proximity ligation assay <a href="#pone.0000223-Gustafsdottir1">[15]</a>, which have both been investigated as potential pathogen diagnostic methods <a href="#pone.0000223-Gustafsdottir1">[15]</a>, <a href="#pone.0000223-Szemes1">[16]</a>. Molecular Inversion Probe (MIP) technology <a href="#pone.0000223-Hardenbol1">[17]</a>, has contributed new features to padlock probes, resulting in an ultra high-throughput method for SNP detection <a href="#pone.0000223-Hardenbol2">[18]</a> and further investigation as a potential quantitative method <a href="#pone.0000223-Wang1">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>As a model assay for MIP pathogen diagnostics, we chose to target the Human Papillomavirus (HPV), which is well known for its cancer-associated cervical infections and for the existence of multiple genotypes <a href="#pone.0000223-deVilliers1">[20]</a>. Based on the oncogenic potential, the subtypes are classified as “high-risk” or “low-risk” HPVs. We developed a 24-plex PathogenMip assay to target HPV genotypes commonly associated with cervical cancer progressions <a href="#pone.0000223-Doorbar1">[21]</a>. The MIP assay was performed in four sequential enzymatic driven reactions (<a href="#pone-0000223-g001">Figure 1</a>). Target recognition sites for the MIPs in this assay were designed using a previously described software tool, PathogenMIPer <a href="#pone.0000223-Thiyagarajan1">[22]</a>. We also attempted to introduce a novel ligase-dependent probe production at equal or increased assay effectiveness, which would be less expensive than direct manufacture of full-length probes. Pyrosequencing <a href="#pone.0000223-Gharizadeh2">[23]</a> and hybridization <a href="#pone.0000223-Dufva1">[24]</a> were applied to the HPV amplified L1-region to validate the results obtained from the PathogenMip Assay.</p>
<div class="figure" id="pone-0000223-g001"><div class="img"><a name="pone-0000223-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Schematic overviews of molecular inversion probe technology. <strong>A</strong>) Synthetic oligonucleotide containing following four regions; i) H1 and H2: homology regions comprised of unique continuous 40–50 base pair fragments for target recognition ii) BARCODE: molecular barcode comprised of a 20 base pair DNA tag for target identification iii) U1 and U2: universal primer regions for inverted probe amplification, and iv) R: restriction site for probe linearization. <strong>B</strong>) Upon target recognition, a DNA polymerase fills the missing gap in between the juxtaposition of the probes' flanking ends, and through the activity of a DNA ligase the probe is circularized. In all cases the missing nucleotide is a “G”. <strong>C</strong>) Circular DNA enrichment through DNA degradation by enzymes Exonuclease I and III. <strong>D</strong>) Probe linearization restriction site cleavage. <strong>E</strong>) All reacted and inverted probes are amplified with universal primers, of which one is biotinylated for subsequent amplicon validation.</p>
<span>doi:10.1371/journal.pone.0000223.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Ligation-based probe construction</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For cost-efficient production of MIPs, two shorter DNA constructs (denoted <strong>A</strong> and <strong>B</strong>) were produced in-house and ligated through a ligation-based production assay (<a href="#pone-0000223-g002">Figure 2</a>). Several different assay protocols were examined, in which: i) dCTP, dTTP and a polymerase (Stoffel fragment) were added to the ligation reaction, ii) bridge concentrations were varied from ratios of 2:1 down to 1:4 with respect to the concentrations of the <strong>A</strong> and <strong>B</strong> constructs, and iii) Ampligase was replaced with a T4 DNA ligase based kit (Fast-Link DNA-ligation kit, Epicenter Biotechnologies; Madison, WI). Gel-electrophoresis was used as read-out for all optimizations (data not shown).</p>
<div class="figure" id="pone-0000223-g002"><div class="img"><a name="pone-0000223-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Ligase-based probe construction for the PathogenMip Assay. Seen in the figure is the sequence for probe-16 prior to inversion. Two shorter fragments were synthesized and hybridized to a bridge complementary to the universal primer regions, common for all probes used in the assay. The coupling of the shorter fragments was mediated by polymerase and ligase activity to achieve maximum yield.</p>
<span>doi:10.1371/journal.pone.0000223.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Thus, best results with regards to yield and purity were achieved by inclusion of Stoffel fragment, dCTP and dTTP, and use of equal concentrations of bridge, <strong>A</strong>, and <strong>B</strong> constructs. No noticeable differences were seen with the use of different ligase-enzymes, and Ampligase was used in subsequent tests. Using this protocol, the ligation step produced clear DNA gel-bands at a 100 base pairs, which were isolated and purified. All 25 probes (including the reference probe, rMIP, which targets the β-globin region in human DNA) were pooled together into a mixture of roughly equal probe concentrations with a total concentration of 5 fmol/probe.</p>


<h4>PathogenMip Assay validation</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Our primary goal was to specifically genotype each target while using a complete 25-plex MIP mixture (24 HPV probes plus the control rMIP), without generation of false-positives or negatives. To enhance sensitivity and selectivity, we explored a variety of initial probe concentrations and thermoprofiles for the initial ligation and gap-filling step (<a href="#pone-0000223-g001">Figure 1B</a>). All optimizations of the PathogenMip Assay were based on UV-visualization of DNA band intensities after size separation by gel electrophoresis (data not shown). The initial probe concentrations yielded an appropriate signal when using 5 fmol of each contributing probe. Cycle times and annealing temperatures in the initial reaction step yielded an efficient protocol, comprising 5 cycles of denaturing and two annealing times at 50°C and 37°C.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Using the optimized conditions, all 21 HPV plasmids and 3 oligonucleotide targets (see <a href="#s4">Materials and Methods</a>, <em>DNA materials</em>) were detected with the PathogenMip Assay, and sequencing individually validated each result. The positive control, rMIP, was successfully validated through a reaction with a commercially available human genomic DNA, without positive matches to any of the HPV genotypes.</p>


<h4>PathogenMip Assay performance</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To further investigate assay sensitivity, we used a fixed concentration of the MIP mixture at 5 fmol/probe in the ligation step, while varying the amount of target DNA. This experiment was performed in two versions. The first set of reactions contained pure plasmids. The second set of reactions contained, in addition to the target plasmids, a constant amount (200 ng) of non-reactive, commercially available human genomic DNA. By spiking with human genomic DNA we hoped to approximate more realistic sample environments for which the assay was constructed. So that the experiments would be of significant value, the probe mixture did not include the rMIP, since rMIP would react with the human genomic DNA and give false quantification data. The plasmid concentrations ranged from 100 ng to 10 fg through a 10-fold dilution series. The original detection limit goal for the assays was 1 pg in the pure plasmid environment and 1 ng for plasmid in the presence of 200 ng human genomic DNA (data not shown), when cycling the PCR reaction at 35 cycles. By using an alternative PCR protocol, working at a higher annealing temperature (58°C) for 60 cycles, we were able to improve the sensitivity of the assay. No background was observed with either of the PCR conditions, but a 10-fold higher sensitivity was obtained with the more extensive 60 cycle-protocol. The new detection limit values were 100 fg for the pure plasmid and 100 pg for plasmid in the presence of 200 ng human genomic DNA (data not shown). Targets in all reactions were correctly typed as judged by L1-pyrosequencing validation.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In multiple HPV-infections, the viral load of each contributing genotype can differ by orders of magnitude <a href="#pone.0000223-Gharizadeh1">[5]</a>. To obtain a deeper understanding of the assay's capabilities when faced with such challenges, we set up a series of reactions to mimic this possibility. Here we chose a two-plasmid detection model in which plasmid HPV-45 was present at a fixed amount of 100 ng, while the second plasmid HPV-59 was present in a wide concentration range. In this scenario, we were able to obtain interpretable data for both genotypes for up to 2-logs difference in target DNA concentration, using the PathogenMip Assay on the different in parallel with pyrosequencing to confirmresults (<a href="#pone-0000223-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0000223-g003"><div class="img"><a name="pone-0000223-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Probe selectivity. The figure depicts superimposed pyrosequencing (dispensation time vs. light signal intensity) diagrams for two probes following the MIP reactions, targeting HPV-45 (black) and HPV-59 (red), mimicking a double HPV infection. <strong>A</strong>) The initial DNA amount of each contributing plasmid was 100 ng and equal levels of sequencing intensities are seen. <strong>B</strong>) The DNA amount of HPV-45 plasmid remained at 100 ng, while HPV-59 plasmid was set at 10 ng resulting in ~2-fold lower signal intensity. <strong>C</strong>) DNA amounts were set at 100 ng for HPV-45 plasmid and 1 nanogram for HPV-59 plasmid, which was observed as a ~3-fold decrease in signal intensity. <strong>D</strong>) DNA amounts of 100 ng of HPV-45 plasmid and 100 pg of HPV-59 plasmid resulted in signal from HPV-45-probe without a measurable signal from the probe for HPV-59.</p>
<span>doi:10.1371/journal.pone.0000223.g003</span></div>

<h4>Genotyping of primary tumor genomic DNA extracts</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The ultimate objective was to screen samples with unknown viral load. A total of 20 commercially available primary tumor genomic DNA extracts were collected from patients with cervical cancers. The DNA samples were screened through the HPV PathogenMip Assay and detected using both methods (pyrosequencing and the in-house barcode-chip assay). The results were validated by genotyping the samples in parallel using the conventional general primer based nested PGMY09/11 and GP5+/6+ amplification protocol. For all tumor-drived samples, the agreement among the methods (PathogenMip Assay and conventional genotyping) was 100%, with 10 samples positive for HPV-16, 5 for HPV-18, two for HPV-45, two for HPV-59 and one sample returning as HPV-negative. When using 200 ng of genomic DNA extract for the PathogenMip assay, equal levels of detection efficiency were obtained for both methods (<a href="#pone-0000223-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0000223-g004"><div class="img"><a name="pone-0000223-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Bar-histograms representing fluorescence intensities from the in-house barcode-chip for genotyping genomic DNA extracts from tumor samples derived from four patients with cervical cancers. Seen in the figure are four examples of single HPV infections, one from each genotype, observed in the sample set. HPV-16 was genotyped in sample OM-1751, HPV-18 in OM-1452, HPV-45 in OM-2258 and HPV-59 in OM-1569. The signal-intensities were normalized to the intensity of the peak for the reference probe targeting human β-globin gene (rMIP). The remaining bars constitute of the reaction background signal.</p>
<span>doi:10.1371/journal.pone.0000223.g004</span></div>

<h4>Multiplexing capabilities</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We also performed the PathogenMip assay screening a mixture of multiple HPV plasmids set at equal concentrations of a 100 ng. Many patients have multiple HPV infections and, though rare, such patients are known to carry as many as five subtypes <a href="#pone.0000223-Gharizadeh1">[5]</a>, so we set as our goal to prove that it is feasible to detect four simultaneous infections at equal viral loads using the PathogenMip Assay. HPV plasmids of four high-risk HPV genotypes (-16, -18, -34, and -59) were pooled in equal amounts at a 100 ng/plasmid, and screened simultaneously in triplicate using the complete 25-plex-probe mixture. The reactions were then validated by both pyrosequencing and hybridizations methods. All four HPV subtypes were detected at similar detection intensities differing by ±20% from the median value, when hybridized through the barcode to an in-house barcode-chip (<a href="#pone-0000223-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0000223-g005"><div class="img"><a name="pone-0000223-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Results from the in-house barcode-chip screening of a quadruple HPV-infection. A) In-house barcode-chip at wavelength 635 emitting the fluorophore signal for internal control Linker A, positive for all oligonucleotides in the chip. B) Wavelength 535 emitting the fluorophore signal for the biotinylated MIP amplicons and the positive control. The in-house barcode-chip was positive for all four HPV genotypes in the pool (HPV-16, -18, -34 and -59). C) Wavelength ratio 535/635 representing the signal ratio from the positive samples to that from linker A. D) Bar histograms representing mean value signal intensities for 2 arrays analyzed with 5 replicates per chip. The highest peak signal for HPV-59 was used for normalization for all the values in the graph for visual purposes. The variability in probes intensities can be explained by individual differences in probe composition and a batch-to-batch variation in the production of the probes.</p>
<span>doi:10.1371/journal.pone.0000223.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>A complete pathogen detection assay based on MIP technology is under construction. Here, we describe a successful model assay targeting 24 HPV genotypes most commonly associated with the progression to cervical cancer. Our pilot subset can now be extended to include all known sequenced HPV genotypes, as multiplexing up to 20,000 using the MIP method for human SNP genotyping, has been demonstrate to be feasible (<a href="http://www.affymetrix.com/technology/mip_technology.affx">www.affymetrix.com/technology/mip_techno​logy.affx</a>), and this would more than cover the existing variability in the HPV genome.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>In the case of a double HPV infection, the fact that the PathogenMip Assay detected and distinguished between two contributing viral loads that differed by 2-logs at an input level of 200ng genomic DNA is quite promising. While DNA yields from tumors vary widely, this level is in the mid-range of recovery in our experience. Exploration of a more full matrix of DNA double-loads and inputs is under way.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Primary tumor genomic DNA extracts were efficiently genotyped for a wide dynamic range of viral loads (<a href="#pone-0000223-g004">Figure 4</a>). The reference probe (rMIP) provided a positive reference control in each performed reaction (<a href="#pone-0000223-g004">Figure 4</a>) and, if used to normalize viral loads against total human genomic DNA content, may allow the researcher a tool to compare viral loads from genomic DNA extracts. The results of the PathogenMip Assay genotyping were in 100% agreement with the conventional genotyping methods.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our goal was to provide a complete, experimentally validated assay in a ready-to-use format, which could be applied generically to any pathogen based on genome target recognition. PathogenMiper <a href="#pone.0000223-Thiyagarajan1">[22]</a> serves as an <em>in silico</em> multiplex test system, whereby one can check the feasibility of designing a MIP assay for the specific targets one might want to include in a multiplex detection assay.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Microbial genomes commonly incur variations from insertions, deletions, or mutations that can generate false-negative results if the variation resides within a probes' targeted region. A more complete diagnostic approach would be to include multiple probes per genotype; this would also provide deeper epidemiological insights in functional genome patterns and variations, i.e. mutations coding for drug resistance <a href="#pone.0000223-Gharizadeh3">[25]</a>. Conventional PCR-based genotyping methods are restricted to reside within conserved genomic regions, which limits the choice of genotyping targets <a href="#pone.0000223-Gravitt1">[1]</a>, <a href="#pone.0000223-FuesselHaws1">[26]</a>. These methods are not robust to genomic changes, a problem that can be at least partially addressed by the multiplexing possible in the PathogenMip Assay.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In the current incarnation of the assay, we have incorporated a ligation step to facilitate probe generation. An oligonucleotide complementary to the universal primer region, which is present in each probe, can be used as a bridge so that one longer probe can be ligated together from two smaller constructs, and length requirements for probe synthesis are therefore cut in half. Longer oligonucleotides result in higher initial costs, and have been known to compromise sequence quality <a href="#pone.0000223-Eason1">[27]</a>. With our new ligation-based probe production, we introduce cost-efficient probe production with possibilities for innovation and potential increase in efficiency. Probe production using the ligation method can easily be performed in any academic or other research institution with budget constraints, using only standard labware and avoiding core facility based research. We are currently investigating parameters for increased ligation efficiency and probe production. The goal of such efforts would be a more homogeneous distribution of probe concentrations, thus enabling one to perform high-throughput production via one-tube reactions and use of fully automated workstations, e.g. Biomek FX (Beckman Coulter, Palo Alto, CA). Parameters that could be varied to this end include ligation at higher temperatures (preferred 65°C) to allow improved incubation profiles, and higher cycle numbers <a href="#pone.0000223-Landegren3">[28]</a>. Use of modified nucleotides, e.g. locked nucleic acids (LNA) <a href="#pone.0000223-Nielsen1">[29]</a> in the bridge would allow for higher melting temperatures (T<sub>m</sub>) for the universal primer regions. Also by replacing the conventional gel-purification protocol with one based on single-strand DNA separation, higher purity of ligated versus non-ligated DNA can be achieved, resulting in a more sensitive assay (data not shown). The introduction of a second universal primer region would require a ligation-based production assay based on three contributing constructs and two bridges complementary to the two universal primer regions. By designing the new probes so that the two primer regions flank the barcode region, a three-way ligation currently under investigation would result in new design features regarding assay possibilities and efficiency.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Although our preferred readout for the PathogenMip Assay uses sequencing or hybridization of the identifying barcode, we also used sequencing of the target-homologous regions of the PathogenMip inverted probes to double check target identity. In addition, we performed conventional HPV identification through the sequencing of the L1 regions. All three assays (PathogenMip/barcode, PathogenMip/target region, and HPV L1 sequencing) produced the same final result with respect to target identification, regardless of whether the readout was performed by pyrosequencing or hybridization methods. Pyrosequencing of the ID-tag was more time and cost-efficient than either hybridization to the in-house barcode-chip or the L1-hybridization-assay. Multiple-primer pyrosequencing, however, is limited to reactions of 20-plex or lower. A higher degree of multiplexing in a sequencing-based approach might be achieved through use of a high-throughput configuration of the pyrosequencing method; for example, the one offered by 454 Life Sciences (<a href="http://www.454.com/applications/index.asp">http://www.454.com/applications/index.as​p</a>). However, when high capacity is required, hybridization-based methods show more potential than sequencing methods, and here the PathogenMip Assay has the advantage of providing more discrimination through its targeted probes than would a standard hybridization to highly conserved regions. The use of ID-tag markers incorporated in the probes simplified sequence read-out in the current PathogenMip Assay, but a more forward looking approach involves refinement of the in-house barcode-chip hybridization piloted here. We anticipate that a high-throughput configuration of the current PathogenMip assay will be achievable through use of the commercially available Affymetrix TAG4-array <a href="#pone.0000223-Pierce1">[30]</a>, which is based on detection of barcodes through hybridization similar to the process described here. The TAG4-array covers a total of 80K features, which includes a total of 16,000 unique barcodes in five replicates.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Sensitivity levels obtained with the current assay were sufficient for a functional assay and were improved with the use of a more extensive amplification protocol. Previously described ligation-based genotyping methods <a href="#pone.0000223-Szemes1">[16]</a>, <a href="#pone.0000223-Pettersson1">[31]</a> have shown sensitivity ranging down to nanograms of genomic DNA for accurate genotyping. These methods have placed careful emphasis on the amplification step by either rolling circle amplification method <a href="#pone.0000223-Fire1">[32]</a> or stringent reaction cleanup with reversible magnetic biotin-streptavidin separation <a href="#pone.0000223-Holmberg1">[33]</a>, followed by design of a PCR run at high annealing temperatures and high cycle numbers, which also could be performed with the use of LNA universal primers <a href="#pone.0000223-Latorra1">[34]</a>. Further consideration of these parameters should allow improvement in sensitivity of a next-generation PathogenMip Assay.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>The potential uses for an assay that simultaneously detects multiple organisms based on differences in DNA sequences are almost unlimited. MIP-based diagnostics have already progressed in the field of human SNP-detection and cancer diagnostics <a href="#pone.0000223-Ji1">[10]</a> with ongoing clinical trials. Microarray techniques have gained ground in the pathogen diagnostic arena, with various genotyping chips such as the HPV-assay PapilloCheck (<a href="http://www.greinerbioone.com">www.greinerbioone.com</a>) upcoming. By combining the benefits of a high-throughput multiplex MIP-assay prior to PCR amplification and of a post-PCR hybridization based chip read-out, researchers can expand the scope of their surveillance to reach new insights into epidemiology and clinical management. We therefore encourage researchers with expertise in various pathogenic microbes to adapt the PathogenMip Assay to accomplish similar detection screenings as performed in this study. By combining the different models, we can reach a common goal toward a standardized virus-, bacteria-, or fungi- chip for routine diagnostics and epidemiological studies.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Experimental design</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Our goal was to design an assay to simultaneously distinguishing among over 100 HPV genotypes that differ only slightly in their genomic sequences (up to 90% homology). The critical regions of the probes that recognize and distinguish among the HPV subtypes were generated using PathogenMIPer, a previously described software tool <a href="#pone.0000223-Thiyagarajan1">[22]</a>. Each target sequence was designed through multiple, successive steps of evaluation of candidate sequences, based on user-defined criteria, followed by a BLAST search against non-target genomes potentially present in the sample that could otherwise cause background noise or interfere with signal. For the study, we included 24 HPV genotypes (<a href="#pone-0000223-t001">Table 1</a> and <a href="#pone-0000223-g006">Figure 6A</a>) most commonly associated with cervical cancer progression <a href="#pone.0000223-deVilliers1">[20]</a>. Our design strategy allowed the probes to target any region of the approximately 8000 bp viral genome. Probes that targeted the E6/E7 genes were preferred, since these genes are most commonly associated with cervical cancer progression <a href="#pone.0000223-Schaeffer1">[35]</a>. The HPV-PathogenMip Assay also included a reference probe (rMIP) targeting the β-globin region in human DNA (<a href="#pone-0000223-t001">Table 1</a>). Validation was performed by either ID-tag based multiple-primer DNA sequencing <a href="#pone.0000223-Gharizadeh1">[5]</a> or barcode hybridization, using our in-house microarray chip setup, the in-house barcode-chip. Each ID-tag was carefully designed so that it was easily distinguishable when up to four probes were sequenced in parallel during the validation reaction, without primer cross hybridization (<a href="#pone-0000223-t002">Table 2</a> and <a href="#pone-0000223-g006">Figure 6B</a>). A conventional genotyping assay was run in parallel, based on nested PCR amplification (<a href="#pone-0000223-g006">Figure 6C</a>). The PCR amplicons were followed by sequencing or hybridization interpretation (<a href="#pone-0000223-g006">Figure 6D–E</a>).</p>
<div class="figure" id="pone-0000223-g006"><div class="img"><a name="pone-0000223-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec4.sec1.fig1.caption1.p1" name="article1.body1.sec4.sec1.fig1.caption1.p1"></a><p>Schematic overview of the PathogenMip Assay. <strong>A</strong>) The 24 probes included in the assay are situated at their respective target sites on the approximately 8000 base pairs of double stranded HPV genomic DNA. Early genes (denoted E) code for virus integration and replication and late genes (denoted L) encode the viral capsule creation. The probes recognize ~40 base pair fragments unique for each targeted genotype. <strong>B</strong>) Following enzymatic inversion of reacted probes and universal amplification, the amplicons are used for subsequent appropriate HPV genotype screening. <strong>C</strong>) Conventional HPV genotyping takes a different approach, in which the nested primer pairs PGMY09/11 and GP5+/6+ amplify respectively ~450 base pair and ~150 base pair fragments that, through an appropriate readout process, will make up the basis for genotyping. These primers are restricted to the highly conserved genomic regions, most commonly found in the L1 gene. <strong>D</strong>) Multiple-primer DNA Pyrosequencing of an incorporated ID-tag. The diagrams depict the complementary sequence of the investigated probes -16 and -18. Marked in the figure is the ID-tag for each probe and the point of ligation where the probes circularized, incorporation of a dGTP, seen here as the complementary “C”. <strong>E</strong>) The in-house barcode chips here used to detect one HPV-16 positive, and one HPV-18 positive in human genomic DNA presence as seen with a positive rMIP in both chips.</p>
<span>doi:10.1371/journal.pone.0000223.g006</span></div><div class="figure" id="pone-0000223-t001"><div class="img"><a name="pone-0000223-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Probes used in this version of the PathogenMip Assay.</span></strong></p><span>doi:10.1371/journal.pone.0000223.t001</span></div><div class="figure" id="pone-0000223-t002"><div class="img"><a name="pone-0000223-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>ID-tags and multiple-primer mixes of complementary barcode sequencing primers (cBarcodeS) for effective DNA sequencing readout.</span></strong></p><span>doi:10.1371/journal.pone.0000223.t002</span></div>

<h4>DNA materials</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>All oligonucleotides used in the assay (<a href="#pone-0000223-t003">Table 3</a>) were produced in-house (Stanford Genome Technology Center). HPV plasmids were kindly provided by Dr. E. M. de Villiers (DKFZ; Heidelberg, Germany) (HPV-6, 11, 16, 18, 40, 45, 51 and 73); HPV-33, 34, 39, 42 and 66 by Dr. M. Favre (Institute Pasteur; Paris, France); HPV-31, 35, 43, 44, and 56 by Dr. A. Lorincz (Digene Corporation; Gaithersburg, MD); HPV-59 and 82 DNA by Dr. T. Matsukura (National Institute of Health; Tokyo, Japan) and HPV-52 by Dr. W. Lancaster (Wayne State University School of Medicine; Detroit, MI). The HPV plasmids were normalized at 100-ng/µl using a ND-100 Spectrophotometer (NanoDrop, Wilmington, DE). Because we lacked plasmids containing HPV subtypes 58, 68, and 69, we manufactured synthetic oligonucleotide targets, denoted as TEMP-58, -68 and -69 to complement the homologous regions of our probes for these genotypes (<a href="#pone-0000223-t003">Table 3B</a>). Twenty commercially available primary tumor genomic DNA extracts were obtained from Oncomatrix, Inc. (<a href="http://www.oncomatrix.com">www.oncomatrix.com</a>). Commercially available human genomic DNA G3041 (Promega, Madison, WI) was also used for assay validation.</p>
<div class="figure" id="pone-0000223-t003"><div class="img"><a name="pone-0000223-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000223" data-uri="info:doi/10.1371/journal.pone.0000223.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000223.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000223.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000223.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000223.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000223.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Oligonucleotides used in the HPV-PathogenMip Assay.</span></strong></p><span>doi:10.1371/journal.pone.0000223.t003</span></div>

<h4>Ligation-based probe construction</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The full-length probes were designed to be of ~110 base pairs. Each probe was synthetically manufactured in two shorter fragments, <strong>A</strong> and <strong>B</strong>, at about 50 base pair lengths, and with 5′-phosphorylated modifications (<a href="#pone-0000223-t003">Table 3A</a>). HPV-complementary portions of the probes were designed using PathogenMIPer, a previously described software tool for designing unique and specific probes, targeting any type of genome based on its sequence data in a FASTA file format <a href="#pone.0000223-Thiyagarajan1">[22]</a>. The universal primer region was chosen for the point of the <strong>A</strong> and <strong>B</strong> fragment separation, 10 base pairs upstream (5′-3′ directional viewpoint) of the first dUTP residue. A complementary bridge oligonucleotide was designed (<a href="#pone-0000223-t003">Table 3B</a>) to hybridize to the universal primer region. Ligation was performed in a one-step reaction. Equal concentrations (2.5 µM) of construct <strong>A</strong>, construct <strong>B</strong>, and the bridge oligonucleotide were pooled together with 2.5 units of Ampligase (Epicenter Biotechnologies, Madison, WI), 5 units of AmpliTaq DNA Polymerase Stoffel Fragment (Applied Biosystems, Foster City, CA) and 1.25 mM of each nucleotide dCTP and dTTP (Fermentas, Hanover, MD). A GeneAmp PCR system 9700 (Applied Biosystems, Foster City, CA) thermocycler was used for initial heating at 95°C for 10 min, followed by 5 cycles of denaturation and hybridization/ligation at 95°C for 2 min and 50°C for 2 min. The ligation products were then run out on a Sub-Cell GT Agarose Electrophoresis System (Bio-Rad, Hercules, CA) using a 2.5% Ultra pure L.M.P Agarose gel (Invitrogen, Carlsbad, CA) with 5 µg/ml Ethidium Bromide (Sigma-Aldrich, St. Louis MO) staining. Gel bands of fragments at about a 100 bases were carefully cut out and dissolved in 0.4 M NaCl at 70°C. DNA extraction of the gel bands was based on a phenol and chloroform (Sigma-Aldrich, St. Louis, MO) organic extraction protocol. The aqueous phase was collected after organic phase separation with equal volumes of: i) phenol, ii) 1:1 phenol/chloroform, and iii) chloroform. This procedure was followed by ethanol precipitation at −20°C over night. Glycogen (Sigma-Aldrich, St. Louis, MO) was added to facilitate DNA precipitation. The purified probes were normalized for probe concentration using a ND-100 Spectrophotometer (NanoDrop, Wilmington, DE).</p>


<h4>PathogenMip Assay</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Prior to the assay, genomic DNA was fragmented by digestion with EcoRI and Xba I according to manufacturer's instructions (New England BioLabs; Ipswich, MA). MIP reactions were performed as follows: <strong>i)</strong> 200 ng of previously digested genomic DNA (or 50 ng undigested HPV plasmid DNA), 5 fmol each of 25 probes, 0.25 units of (Epicenter Biotechnologies, Madison, WI), 0.25 mM dGTP (Fermentas, Hanover, MD), and 1 units of AmpliTaq DNA Polymerase Stoffel Fragment (Applied Biosystems, Foster City, CA) in 1× Ampligase buffer (Epicenter Biotechnologies, Madison, WI) run in a total volume of 10 µl/rxn. After pre-incubation at 95°C for 10 min, the reaction went through 5 cycles of 95°C for 2 min, 50°C for 30 min and 37°C for 10 min. <strong>ii)</strong> 10 µl of 130 mM Tris-HCl (pH 7.8), 85.75 mM KCl, 3.85 mM MgCl<sub>2</sub> and 0.1% BSA solution containing 5.6 units Exonuclease I and 112 units Exonuclease III (Epicenter Biotechnologies, Madison, WI) was added to each reaction. The mixture was incubated at 37°C for 60 min followed by enzyme deactivation at 80°C for 20 min. <strong>iii)</strong> 4 units of Uracil-DNA-Glycosylase (New England BioLabs, Ipswich, MA) was added to each reaction and incubated at 37°C for 20 min followed by enzyme deactivation at 80°C for 20 min. <strong>iv)</strong> PCR amplification was carried in a total reaction volume of 50 µl, containing 5 µl MIP reaction, GeneAmp 1X PCR Buffer II (Applied Biosystems, Foster City, CA), 2.5 mM MgCl<sub>2</sub> (Applied Biosystems, Foster City, CA), 0.12 mM dNTPs (Fermentas, Hanover, MD), 2.5 units AmpliTaq Gold DNA polymerase (Applied Biosystems, Foster City, CA) and 0.2 µM of each universal primer B-F-CHPV-U1 and R-CHPV-U2 (<a href="#pone-0000223-t003">Table 3B</a>). A 10-min incubation step at 95°C was followed by 35 cycles of amplification with a thermocycler GeneAmp PCR system 9700 (Applied Biosystems, Foster City, CA). Each cycle included a denaturation step at 95°C for 45 sec, and an annealing step at 52°C for 30 sec. A final extension was done at 72°C for 5 min. For three randomly chosen reactions, an alternative PCR protocol was run in parallel, which was comprised of 60 cycles with an increased annealing temperature at 58°C. The latter protocol was used to test whether increased sensitivity of the assay could be achieved with a more extensive PCR amplification.</p>


<h4>Multiple-primer DNA pyrosequencing of barcodes</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The B-F-CHPV-U1 primer used for universal probe amplification was biotinylated, and the leading strand could therefore be used as a sequencing template. Single strand template preparation was performed as previously described <a href="#pone.0000223-Gharizadeh2">[23]</a>, and oligonucleotides complementary to barcodes included in the probes were used as sequencing primers (<a href="#pone-0000223-t003">Table 3B</a>). Each of the 24 HPV targeting probes contained one out of four ID-tags, which lay the basis for a rapid multiple-primer based screening by sequencing <a href="#pone.0000223-Gharizadeh1">[5]</a>. The primers were pooled into six mixes of four primers with different ID-tags (<a href="#pone-0000223-t002">Table 2</a>). Pyrosequencing was performed with a cyclic de novo sequencing dispensation (ACGT) using a PSQ<sup>TM</sup>HS96A DNA sequencing system.</p>


<h4>Microarray barcode-chip template preparation</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>For more efficient microarray hybridizations, single stranded DNA was generated from the first PCR amplification. Amplification was carried out in a total reaction volume of 50 µl, containing 1 µl from the first PCR reaction, 0.125× BD Titanium Taq DNA polymerase (Clontech, Mountain View, CA), 1X BD Titanium Taq PCR buffer (Clontech, Mountain View, CA), 0.2 mM dNTPs (Fermentas, Hanover, MD) and 0.2 µM of each universal primer B-F-CHPV-U1. A 10 min incubation step at 95°C was followed by 35 cycles of amplification with a thermocycler GeneAmp PCR system 9700 (Applied Biosystems, Foster City, CA). Each cycle included a denaturation step at 95°C for 45 sec, and annealing at 54°C for 30 sec. A final extension was performed at 68°C for 5 min.</p>


<h4>Barcode microarray preparation</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Oligonucleotides were synthesized at the Stanford Genome Technology Center. Oligonucleotides used as probes on the array (<a href="#pone-0000223-t003">Table 3B</a>) consisted of (from 5′ to 3′): a 5′-amino group (for attachment to the array), a 5 bp poly-T sequence, Linker A, and a 20 bp complementary to barcode sequence. Probes were attached to the microarray essentially as described previously <a href="#pone.0000223-Dufva1">[24]</a>. Each probe was printed in quadruplicate or quintuple, and two complete arrays were present on each chip. The post-printing processing of the microarrays was performed as recommended by the slide manufacturer (Amersham Biosciences, Piscataway, NJ). Control oligonucleotides used to verify array quality included a poly-T (20 bp) with 5′-amino and internal-biotin modification as a labeling control (amino-B), a oligonucleotide complementary to Linker A with internal-Cy3 (amino-P) as an internal control for each spot's quality, and a 5′-amino modified poly-T (20 bp) as a DNA spacer.</p>


<h4>Array hybridizations</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>In the hybridization step, the biotinylated single-stranded target and amino-P were applied to the microarray. For the initial target hybridization step, we used 50 µl of target DNA in 1× hybridization buffer (100 mM MES, 1 M [Na+], 20 mM EDTA, 0.01 Tween20), 1.25× Denhardt's solution, and 5 nM amino-P. The hybridization was performed at 42°C for 12–16 hours. After hybridization, the microarray was washed 3 times in wash buffer and then labeled for 10 min at 50°C with a solution containing streptavidin-allophycocyanin (1mg/ml final concentration), 6× SSPE, 1× Denhardt's solution, and 0.01% Tween-20. The microarray was scanned for fluorescence intensity at 535 nm and 635 nm using a GenePix 4000 fluorescent scanner (Axon Instrument, Foster City, CA) set to scan at 450 PMT. GenePix Pro software was used to determine the total fluorescence signal from each spot on the array.</p>


<h4>Validation and genotyping of DNA samples</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>For validation of the PathogenMip Assay, the 20 genomic DNA extracts were characterized for HPV presence and subtypes with nested PCR amplification using general primers PGMY09/11 and GP5+/6+ <a href="#pone.0000223-Gravitt1">[1]</a> according to a previously described protocol <a href="#pone.0000223-FuesselHaws1">[26]</a> with an initial amount of 40 ng/ml genomic DNA, followed by sequencing as described previously <a href="#pone.0000223-Gharizadeh1">[5]</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Keith Anderson, Mike Jensen for technical support and Heather Halenbeck for critical reading on this work.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NP. Performed the experiments: MA AV. Analyzed the data: NP MA. Contributed reagents/materials/analysis tools: NP ST. Wrote the paper: NP MA. Other: Director, Stanford Genome Technology Center: RD. Michael S. Akhras' PhD advisor: PN.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000223-Gravitt1" id="pone.0000223-Gravitt1"></a>Gravitt PE, Peyton CL, Alessi TQ, Wheeler CM, Coutlee F, et al.  (2000) Improved amplification of genital human papillomaviruses. J Clin Microbiol  38: 357–361.  <ul class="find" data-citedArticleID="931144"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Improved+amplification+of+genital+human+papillomaviruses.&amp;auth=&amp;atitle=Improved+amplification+of+genital+human+papillomaviruses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+amplification+of+genital+human+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+amplification+of+genital+human+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000223-Csako1" id="pone.0000223-Csako1"></a>Csako G (2006) Present and future of rapid and/or high-throughput methods for nucleic acid testing. Clin Chim Acta  363: 6–31.  <ul class="find" data-citedArticleID="931124" data-doi="10.1016/j.cccn.2005.07.009"><li><a href="http://dx.doi.org/10.1016/j.cccn.2005.07.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Present+and+future+of+rapid+and%2For+high-throughput+methods+for+nucleic+acid+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Present+and+future+of+rapid+and%2For+high-throughput+methods+for+nucleic+acid+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000223-Mackay1" id="pone.0000223-Mackay1"></a>Mackay IM, Arden KE, Nitsche A (2002) Real-time PCR in virology. Nucleic Acids Res  30: 1292–1305.  <ul class="find" data-citedArticleID="931166" data-doi="10.1093/nar/30.6.1292"><li><a href="http://dx.doi.org/10.1093/nar/30.6.1292" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Real-time+PCR+in+virology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Real-time+PCR+in+virology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000223-Klaassen1" id="pone.0000223-Klaassen1"></a>Klaassen CH, Prinsen CF, de Valk HA, Horrevorts AM, Jeunink MA, et al.  (2004) DNA microarray format for detection and subtyping of human papillomavirus. J Clin Microbiol  42: 2152–2160.  <ul class="find" data-citedArticleID="931156" data-doi="10.1128/jcm.42.5.2152-2160.2004"><li><a href="http://dx.doi.org/10.1128/jcm.42.5.2152-2160.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarray+format+for+detection+and+subtyping+of+human+papillomavirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarray+format+for+detection+and+subtyping+of+human+papillomavirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000223-Gharizadeh1" id="pone.0000223-Gharizadeh1"></a>Gharizadeh B, Zheng B, Akhras M, Ghaderi M, Jejelowo O, et al.  (2006) Sentinel-base DNA genotyping using multiple sequencing primers for high-risk human papillomaviruses. Mol Cell Probes.   <ul class="find" data-citedArticleID="931138" data-doi="10.1016/j.mcp.2006.01.002"><li><a href="http://dx.doi.org/10.1016/j.mcp.2006.01.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sentinel-base+DNA+genotyping+using+multiple+sequencing+primers+for+high-risk+human+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sentinel-base+DNA+genotyping+using+multiple+sequencing+primers+for+high-risk+human+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000223-Chamberlain1" id="pone.0000223-Chamberlain1"></a>Chamberlain JS, Gibbs RA, Ranier JE, Nguyen PN, Caskey CT (1988) Deletion screening of the Duchenne muscular dystrophy locus via multiplex DNA amplification. Nucleic Acids Res  16: 11141–11156.  <ul class="find" data-citedArticleID="931122" data-doi="10.1093/nar/16.23.11141"><li><a href="http://dx.doi.org/10.1093/nar/16.23.11141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deletion+screening+of+the+Duchenne+muscular+dystrophy+locus+via+multiplex+DNA+amplification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deletion+screening+of+the+Duchenne+muscular+dystrophy+locus+via+multiplex+DNA+amplification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000223-Elnifro1" id="pone.0000223-Elnifro1"></a>Elnifro EM, Ashshi AM, Cooper RJ, Klapper PE (2000) Multiplex PCR: optimization and application in diagnostic virology. Clin Microbiol Rev  13: 559–570.  <ul class="find" data-citedArticleID="931132" data-doi="10.1128/cmr.13.4.559-570.2000"><li><a href="http://dx.doi.org/10.1128/cmr.13.4.559-570.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+PCR%3A+optimization+and+application+in+diagnostic+virology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+PCR%3A+optimization+and+application+in+diagnostic+virology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000223-Markoulatos1" id="pone.0000223-Markoulatos1"></a>Markoulatos P, Siafakas N, Moncany M (2002) Multiplex polymerase chain reaction: a practical approach. J Clin Lab Anal  16: 47–51.  <ul class="find" data-citedArticleID="931168" data-doi="10.1002/jcla.2058"><li><a href="http://dx.doi.org/10.1002/jcla.2058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+polymerase+chain+reaction%3A+a+practical+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+polymerase+chain+reaction%3A+a+practical+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000223-Bissonnette1" id="pone.0000223-Bissonnette1"></a>Bissonnette L, Bergeron MG (2006) Next revolution in the molecular theranostics of infectious diseases: microfabricated systems for personalized medicine. Expert Rev Mol Diagn  6: 433–450.  <ul class="find" data-citedArticleID="931120" data-doi="10.1586/14737159.6.3.433"><li><a href="http://dx.doi.org/10.1586/14737159.6.3.433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Next+revolution+in+the+molecular+theranostics+of+infectious+diseases%3A+microfabricated+systems+for+personalized+medicine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Next+revolution+in+the+molecular+theranostics+of+infectious+diseases%3A+microfabricated+systems+for+personalized+medicine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000223-Ji1" id="pone.0000223-Ji1"></a>Ji H, Kumm J, Zhang M, Farnam K, Salari K, et al.  (2006) Molecular inversion probe analysis of gene copy alterations reveals distinct categories of colorectal carcinoma. Cancer Res  66: 7910–7919.  <ul class="find" data-citedArticleID="931154" data-doi="10.1158/0008-5472.can-06-0595"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-0595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+inversion+probe+analysis+of+gene+copy+alterations+reveals+distinct+categories+of+colorectal+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+inversion+probe+analysis+of+gene+copy+alterations+reveals+distinct+categories+of+colorectal+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000223-Landegren1" id="pone.0000223-Landegren1"></a>Landegren U, Kaiser R, Sanders J, Hood L (1988) A ligase-mediated gene detection technique. Science  241: 1077–1080.  <ul class="find" data-citedArticleID="931158" data-doi="10.1126/science.3413476"><li><a href="http://dx.doi.org/10.1126/science.3413476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+ligase-mediated+gene+detection+technique." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+ligase-mediated+gene+detection+technique.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000223-Landegren2" id="pone.0000223-Landegren2"></a>Landegren U, Kaiser R, Caskey CT, Hood L (1988) DNA diagnostics–molecular techniques and automation. Science  242: 229–237.  <ul class="find" data-citedArticleID="931160" data-doi="10.1126/science.3051381"><li><a href="http://dx.doi.org/10.1126/science.3051381" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+diagnostics%26%23x2013%3Bmolecular+techniques+and+automation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+diagnostics%26%23x2013%3Bmolecular+techniques+and+automation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000223-Nilsson1" id="pone.0000223-Nilsson1"></a>Nilsson M, Malmgren H, Samiotaki M, Kwiatkowski M, Chowdhary BP, et al.  (1994) Padlock probes: circularizing oligonucleotides for localized DNA detection. Science  265: 2085–2088.  <ul class="find" data-citedArticleID="931172" data-doi="10.1126/science.7522346"><li><a href="http://dx.doi.org/10.1126/science.7522346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Padlock+probes%3A+circularizing+oligonucleotides+for+localized+DNA+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Padlock+probes%3A+circularizing+oligonucleotides+for+localized+DNA+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000223-Nilsson2" id="pone.0000223-Nilsson2"></a>Nilsson M, Krejci K, Koch J, Kwiatkowski M, Gustavsson P, et al.  (1997) Padlock probes reveal single-nucleotide differences, parent of origin and in situ distribution of centromeric sequences in human chromosomes 13 and 21. Nat Genet  16: 252–255.  <ul class="find" data-citedArticleID="931174" data-doi="10.1038/ng0797-252"><li><a href="http://dx.doi.org/10.1038/ng0797-252" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Padlock+probes+reveal+single-nucleotide+differences%2C+parent+of+origin+and+in+situ+distribution+of+centromeric+sequences+in+human+chromosomes+13+and+21." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Padlock+probes+reveal+single-nucleotide+differences%2C+parent+of+origin+and+in+situ+distribution+of+centromeric+sequences+in+human+chromosomes+13+and+21.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000223-Gustafsdottir1" id="pone.0000223-Gustafsdottir1"></a>Gustafsdottir SM, Nordengrahn A, Fredriksson S, Wallgren P, Rivera E, et al.  (2006) Detection of individual microbial pathogens by proximity ligation. Clin Chem  52: 1152–1160.  <ul class="find" data-citedArticleID="931146" data-doi="10.1373/clinchem.2005.065847"><li><a href="http://dx.doi.org/10.1373/clinchem.2005.065847" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+individual+microbial+pathogens+by+proximity+ligation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+individual+microbial+pathogens+by+proximity+ligation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000223-Szemes1" id="pone.0000223-Szemes1"></a>Szemes M, Bonants P, de Weerdt M, Baner J, Landegren U, et al.  (2005) Diagnostic application of padlock probes–multiplex detection of plant pathogens using universal microarrays. Nucleic Acids Res  33: e70.  <ul class="find" data-citedArticleID="931182" data-doi="10.1093/nar/gni069"><li><a href="http://dx.doi.org/10.1093/nar/gni069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diagnostic+application+of+padlock+probes%26%23x2013%3Bmultiplex+detection+of+plant+pathogens+using+universal+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diagnostic+application+of+padlock+probes%26%23x2013%3Bmultiplex+detection+of+plant+pathogens+using+universal+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000223-Hardenbol1" id="pone.0000223-Hardenbol1"></a>Hardenbol P, Baner J, Jain M, Nilsson M, Namsaraev EA, et al.  (2003) Multiplexed genotyping with sequence-tagged molecular inversion probes. Nat Biotechnol  21: 673–678.  <ul class="find" data-citedArticleID="931148" data-doi="10.1038/nbt821"><li><a href="http://dx.doi.org/10.1038/nbt821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplexed+genotyping+with+sequence-tagged+molecular+inversion+probes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplexed+genotyping+with+sequence-tagged+molecular+inversion+probes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000223-Hardenbol2" id="pone.0000223-Hardenbol2"></a>Hardenbol P, Yu F, Belmont J, Mackenzie J, Bruckner C, et al.  (2005) Highly multiplexed molecular inversion probe genotyping: over 10,000 targeted SNPs genotyped in a single tube assay. Genome Res  15: 269–275.  <ul class="find" data-citedArticleID="931150" data-doi="10.1101/gr.3185605"><li><a href="http://dx.doi.org/10.1101/gr.3185605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+multiplexed+molecular+inversion+probe+genotyping%3A+over+10%2C000+targeted+SNPs+genotyped+in+a+single+tube+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+multiplexed+molecular+inversion+probe+genotyping%3A+over+10%2C000+targeted+SNPs+genotyped+in+a+single+tube+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000223-Wang1" id="pone.0000223-Wang1"></a>Wang Y, Moorhead M, Karlin-Neumann G, Falkowski M, Chen C, et al.  (2005) Allele quantification using molecular inversion probes (MIP). Nucleic Acids Res  33: e183.  <ul class="find" data-citedArticleID="931186" data-doi="10.1093/nar/gni177"><li><a href="http://dx.doi.org/10.1093/nar/gni177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allele+quantification+using+molecular+inversion+probes+%28MIP%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allele+quantification+using+molecular+inversion+probes+%28MIP%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000223-deVilliers1" id="pone.0000223-deVilliers1"></a>de Villiers EM, Fauquet C, Broker TR, Bernard HU, zur Hausen H (2004) Classification of papillomaviruses. Virology  324: 17–27.  <ul class="find" data-citedArticleID="931188" data-doi="10.1016/j.virol.2004.03.033"><li><a href="http://dx.doi.org/10.1016/j.virol.2004.03.033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+papillomaviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+papillomaviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000223-Doorbar1" id="pone.0000223-Doorbar1"></a>Doorbar J (2006) Molecular biology of human papillomavirus infection and cervical cancer. Clin Sci (Lond)  110: 525–541.  <ul class="find" data-citedArticleID="931126" data-doi="10.1042/cs20050369"><li><a href="http://dx.doi.org/10.1042/cs20050369" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+biology+of+human+papillomavirus+infection+and+cervical+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+biology+of+human+papillomavirus+infection+and+cervical+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000223-Thiyagarajan1" id="pone.0000223-Thiyagarajan1"></a>Thiyagarajan S, Karhanek M, Akhras M, Davis RW, Pourmand N (2006) PathogenMIPer : A tool for the design of molecular inversion probes to detect multiple pathogens. BMC Bioinformatics  7: 500.  <ul class="find" data-citedArticleID="931184" data-doi="10.1186/1471-2105-7-500"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PathogenMIPer+%3A+A+tool+for+the+design+of+molecular+inversion+probes+to+detect+multiple+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PathogenMIPer+%3A+A+tool+for+the+design+of+molecular+inversion+probes+to+detect+multiple+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000223-Gharizadeh2" id="pone.0000223-Gharizadeh2"></a>Gharizadeh B, Akhras M, Nourizad N, Ghaderi M, Yasuda K, et al.  (2006) Methodological improvements of pyrosequencing technology. J Biotechnol.   <ul class="find" data-citedArticleID="931140" data-doi="10.1016/j.jbiotec.2006.01.025"><li><a href="http://dx.doi.org/10.1016/j.jbiotec.2006.01.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methodological+improvements+of+pyrosequencing+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methodological+improvements+of+pyrosequencing+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000223-Dufva1" id="pone.0000223-Dufva1"></a>Dufva M (2005) Fabrication of high quality microarrays. Biomol Eng  22: 173–184.  <ul class="find" data-citedArticleID="931128" data-doi="10.1016/j.bioeng.2005.09.003"><li><a href="http://dx.doi.org/10.1016/j.bioeng.2005.09.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fabrication+of+high+quality+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fabrication+of+high+quality+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000223-Gharizadeh3" id="pone.0000223-Gharizadeh3"></a>Gharizadeh B, Akhras M, Unemo M, Wretlind B, Nyren P, et al.  (2005) Detection of gyrA mutations associated with ciprofloxacin resistance in Neisseria gonorrhoeae by rapid and reliable pre-programmed short DNA sequencing. Int J Antimicrob Agents  26: 486–490.  <ul class="find" data-citedArticleID="931142" data-doi="10.1016/j.ijantimicag.2005.08.017"><li><a href="http://dx.doi.org/10.1016/j.ijantimicag.2005.08.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+gyrA+mutations+associated+with+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae+by+rapid+and+reliable+pre-programmed+short+DNA+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+gyrA+mutations+associated+with+ciprofloxacin+resistance+in+Neisseria+gonorrhoeae+by+rapid+and+reliable+pre-programmed+short+DNA+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000223-FuesselHaws1" id="pone.0000223-FuesselHaws1"></a>Fuessel Haws AL, He Q, Rady PL, Zhang L, Grady J, et al.  (2004) Nested PCR with the PGMY09/11 and GP5(+)/6(+) primer sets improves detection of HPV DNA in cervical samples. J Virol Methods  122: 87–93.  <ul class="find" data-citedArticleID="931136" data-doi="10.1016/j.jviromet.2004.08.007"><li><a href="http://dx.doi.org/10.1016/j.jviromet.2004.08.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nested+PCR+with+the+PGMY09%2F11+and+GP5%28%2B%29%2F6%28%2B%29+primer+sets+improves+detection+of+HPV+DNA+in+cervical+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nested+PCR+with+the+PGMY09%2F11+and+GP5%28%2B%29%2F6%28%2B%29+primer+sets+improves+detection+of+HPV+DNA+in+cervical+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000223-Eason1" id="pone.0000223-Eason1"></a>Eason RG, Pourmand N, Tongprasit W, Herman ZS, Anthony K, et al.  (2004) Characterization of synthetic DNA bar codes in Saccharomyces cerevisiae gene-deletion strains. Proc Natl Acad Sci U S A  101: 11046–11051.  <ul class="find" data-citedArticleID="931130" data-doi="10.1073/pnas.0403672101"><li><a href="http://dx.doi.org/10.1073/pnas.0403672101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+synthetic+DNA+bar+codes+in+Saccharomyces+cerevisiae+gene-deletion+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+synthetic+DNA+bar+codes+in+Saccharomyces+cerevisiae+gene-deletion+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000223-Landegren3" id="pone.0000223-Landegren3"></a>Landegren U (1993) Ligation-based DNA diagnostics. Bioessays  15: 761–765.  <ul class="find" data-citedArticleID="931162" data-doi="10.1002/bies.950151112"><li><a href="http://dx.doi.org/10.1002/bies.950151112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ligation-based+DNA+diagnostics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ligation-based+DNA+diagnostics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000223-Nielsen1" id="pone.0000223-Nielsen1"></a>Nielsen CB, Singh SK, Wengel J, Jacobsen JP (1999) The solution structure of a locked nucleic acid (LNA) hybridized to DNA. J Biomol Struct Dyn  17: 175–191.  <ul class="find" data-citedArticleID="931170" data-doi="10.1080/07391102.1999.10508352"><li><a href="http://dx.doi.org/10.1080/07391102.1999.10508352" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+solution+structure+of+a+locked+nucleic+acid+%28LNA%29+hybridized+to+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+solution+structure+of+a+locked+nucleic+acid+%28LNA%29+hybridized+to+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000223-Pierce1" id="pone.0000223-Pierce1"></a>Pierce SE, Fung EL, Jaramillo DF, Chu AM, Davis RW, et al.  (2006) A unique and universal molecular barcode array. Nat Methods  3: 601–603.  <ul class="find" data-citedArticleID="931178" data-doi="10.1038/nmeth905"><li><a href="http://dx.doi.org/10.1038/nmeth905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+unique+and+universal+molecular+barcode+array." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+unique+and+universal+molecular+barcode+array.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000223-Pettersson1" id="pone.0000223-Pettersson1"></a>Pettersson E, Lindskog M, Lundeberg J, Ahmadian A (2006) Tri-nucleotide threading for parallel amplification of minute amounts of genomic DNA. Nucleic Acids Res  34: e49.  <ul class="find" data-citedArticleID="931176" data-doi="10.1093/nar/gkl103"><li><a href="http://dx.doi.org/10.1093/nar/gkl103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tri-nucleotide+threading+for+parallel+amplification+of+minute+amounts+of+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tri-nucleotide+threading+for+parallel+amplification+of+minute+amounts+of+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000223-Fire1" id="pone.0000223-Fire1"></a>Fire A, Xu SQ (1995) Rolling replication of short DNA circles. Proc Natl Acad Sci U S A  92: 4641–4645.  <ul class="find" data-citedArticleID="931134" data-doi="10.1073/pnas.92.10.4641"><li><a href="http://dx.doi.org/10.1073/pnas.92.10.4641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rolling+replication+of+short+DNA+circles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rolling+replication+of+short+DNA+circles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000223-Holmberg1" id="pone.0000223-Holmberg1"></a>Holmberg A, Blomstergren A, Nord O, Lukacs M, Lundeberg J, et al.  (2005) The biotin-streptavidin interaction can be reversibly broken using water at elevated temperatures. Electrophoresis  26: 501–510.  <ul class="find" data-citedArticleID="931152" data-doi="10.1002/elps.200410070"><li><a href="http://dx.doi.org/10.1002/elps.200410070" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+biotin-streptavidin+interaction+can+be+reversibly+broken+using+water+at+elevated+temperatures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+biotin-streptavidin+interaction+can+be+reversibly+broken+using+water+at+elevated+temperatures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000223-Latorra1" id="pone.0000223-Latorra1"></a>Latorra D, Arar K, Hurley JM (2003) Design considerations and effects of LNA in PCR primers. Mol Cell Probes  17: 253–259.  <ul class="find" data-citedArticleID="931164" data-doi="10.1016/s0890-8508(03)00062-8"><li><a href="http://dx.doi.org/10.1016/s0890-8508(03)00062-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Design+considerations+and+effects+of+LNA+in+PCR+primers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Design+considerations+and+effects+of+LNA+in+PCR+primers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000223-Schaeffer1" id="pone.0000223-Schaeffer1"></a>Schaeffer AJ, Nguyen M, Liem A, Lee D, Montagna C, et al.  (2004) E6 and E7 oncoproteins induce distinct patterns of chromosomal aneuploidy in skin tumors from transgenic mice. Cancer Res  64: 538–546.  <ul class="find" data-citedArticleID="931180" data-doi="10.1158/0008-5472.can-03-0124"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-03-0124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=E6+and+E7+oncoproteins+induce+distinct+patterns+of+chromosomal+aneuploidy+in+skin+tumors+from+transgenic+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22E6+and+E7+oncoproteins+induce+distinct+patterns+of+chromosomal+aneuploidy+in+skin+tumors+from+transgenic+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.XML" value="102150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.PDF" value="521348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g001.PNG_L" value="71938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g001.PNG_M" value="42064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g001.PNG_S" value="2693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g001.TIF" value="261006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g001.PNG_I" value="19588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g002.PNG_L" value="20294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g002.PNG_M" value="13933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g002.PNG_S" value="1695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g002.TIF" value="51068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g002.PNG_I" value="6603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g003.PNG_L" value="442232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g003.PNG_M" value="60692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g003.PNG_S" value="2649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g003.TIF" value="937032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g003.PNG_I" value="25541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g004.PNG_L" value="253901"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g004.PNG_M" value="29265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g004.PNG_S" value="2131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g004.TIF" value="464192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g004.PNG_I" value="12852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g005.PNG_L" value="1613369"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g005.PNG_M" value="160372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g005.PNG_S" value="4773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g005.TIF" value="1995558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g005.PNG_I" value="58499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g006.PNG_L" value="590576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g006.PNG_M" value="75540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g006.PNG_S" value="3562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g006.TIF" value="932348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.g006.PNG_I" value="33315"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t001.PNG_L" value="157553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t001.PNG_M" value="88969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t001.PNG_S" value="7840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t001.TIF" value="815904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t001.PNG_I" value="82845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t002.PNG_L" value="80415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t002.PNG_M" value="94376"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t002.PNG_S" value="4435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t002.TIF" value="393876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t002.PNG_I" value="41695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t003.PNG_L" value="978592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t003.PNG_M" value="50858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t003.PNG_S" value="3050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t003.TIF" value="1557948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000223.t003.PNG_I" value="24424"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000223&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000223" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000223&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000223&volume=&issue=&title=PathogenMip Assay: A Multiplex Pathogen Detection Assay&author_name=Michael%20S.%20Akhras%2C%20Sreedevi%20Thiyagarajan%2C%20Andrea%20C.%20Villablanca%2C%20Ronald%20W.%20Davis%2C%20P%C3%A5l%20Nyr%C3%A9n%2C%20Nader%20Pourmand&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000223" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223&amp;t=PathogenMip%20Assay%3A%20A%20Multiplex%20Pathogen%20Detection%20Assay" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223&title=PathogenMip%20Assay%3A%20A%20Multiplex%20Pathogen%20Detection%20Assay&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223&amp;title=PathogenMip%20Assay%3A%20A%20Multiplex%20Pathogen%20Detection%20Assay" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000223&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'PathogenMip Assay: A Multiplex Pathogen Detection Assay';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000223';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20PathogenMip%20Assay%3A%20A%20Multiplex%20Pathogen%20Detection%20Assay http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000223" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000223" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+extraction%22" title="Search for articles in the subject area:'DNA extraction'"><div class="flagText">DNA extraction</div></a>
              <div data-categoryid="19965" data-articleid="23284"
                   data-categoryname="DNA extraction"
                   class="flagImage" title="Flag 'DNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genotyping%22" title="Search for articles in the subject area:'Genotyping'"><div class="flagText">Genotyping</div></a>
              <div data-categoryid="19731" data-articleid="23284"
                   data-categoryname="Genotyping"
                   class="flagImage" title="Flag 'Genotyping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="23284"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+papillomavirus+infection%22" title="Search for articles in the subject area:'Human papillomavirus infection'"><div class="flagText">Human papillomavirus infection</div></a>
              <div data-categoryid="48361" data-articleid="23284"
                   data-categoryname="Human papillomavirus infection"
                   class="flagImage" title="Flag 'Human papillomavirus infection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ligation+assay%22" title="Search for articles in the subject area:'Ligation assay'"><div class="flagText">Ligation assay</div></a>
              <div data-categoryid="50049" data-articleid="23284"
                   data-categoryname="Ligation assay"
                   class="flagImage" title="Flag 'Ligation assay' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligonucleotides%22" title="Search for articles in the subject area:'Oligonucleotides'"><div class="flagText">Oligonucleotides</div></a>
              <div data-categoryid="19655" data-articleid="23284"
                   data-categoryname="Oligonucleotides"
                   class="flagImage" title="Flag 'Oligonucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="23284"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probe+hybridization%22" title="Search for articles in the subject area:'Probe hybridization'"><div class="flagText">Probe hybridization</div></a>
              <div data-categoryid="48355" data-articleid="23284"
                   data-categoryname="Probe hybridization"
                   class="flagImage" title="Flag 'Probe hybridization' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3249'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1090'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1302&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=44841">Mega potential for multiplex screening</a><br>Posted by hbvinrsa</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
