

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Module-Based Outcome Prediction Using Breast Cancer Compendia</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001047"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001047" />

    <meta name="citation_title" content="Module-Based Outcome Prediction Using Breast Cancer Compendia"/>
    <meta itemprop="name" content="Module-Based Outcome Prediction Using Breast Cancer Compendia"/>

      <meta name="citation_author" content="Martin H. van Vliet"/>
            <meta name="citation_author_institution" content="Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands"/>
            <meta name="citation_author_institution" content="Bioinformatics and Statistics Group, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Christiaan N. Klijn"/>
            <meta name="citation_author_institution" content="Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands"/>
            <meta name="citation_author_institution" content="Bioinformatics and Statistics Group, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
            <meta name="citation_author_institution" content="Mouse Models for Breast Cancer, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Lodewyk F. A. Wessels"/>
            <meta name="citation_author_institution" content="Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands"/>
            <meta name="citation_author_institution" content="Bioinformatics and Statistics Group, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands"/>
      <meta name="citation_author" content="Marcel J. T. Reinders"/>
            <meta name="citation_author_institution" content="Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands"/>

    <meta name="citation_date" content="2007/10/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001047.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1047"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Gene expression profiling predicts clinical outcome of breast cancer.; citation_author=L Van 't Veer; citation_author=H Dai; citation_author=M van de Vijver; citation_author=Y He; citation_author=A Hart; citation_journal_title=Nature; citation_volume=415; citation_number=1; citation_pages=530-6; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Repeated observation of breast tumor subtypes in independent gene expression data sets.; citation_author=T Sorlie; citation_author=R Tibshirani; citation_author=J Parker; citation_author=T Hastie; citation_author=J Marron; citation_journal_title=PNAS; citation_volume=14; citation_number=2; citation_pages=8418-23; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer.; citation_author=Y Wang; citation_author=J Klein; citation_author=Y Zhang; citation_author=A Sieuwerts; citation_author=M Look; citation_journal_title=The Lancet; citation_volume=365; citation_number=3; citation_pages=671-9; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A gene-expression signature as a predictor of survival in breast cancer.; citation_author=M Van de Vijver; citation_author=Y He; citation_author=L van 't Veer; citation_author=H Dai; citation_author=A Hart; citation_journal_title=N. Engl. J. Med.; citation_volume=25; citation_number=4; citation_pages=1999-2009; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Oncomine: a cancer microarray database and integrated data-mining platform.; citation_author=D Rhodes; citation_author=J Yu; citation_author=K Shanker; citation_author=N Deshpande; citation_author=R Varambally; citation_journal_title=Neoplasia; citation_volume=1; citation_number=5; citation_pages=1-6; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prognostic meta-signature of breast cancer developed by two-stage mixture modelling of microarray data.; citation_author=R Shen; citation_author=D Ghosh; citation_author=A Chinnaiyan; citation_journal_title=BMC Genomics; citation_volume=5; citation_number=6; citation_pages=94; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Joint analysis of two microarray gene-expression data sets to select lung adenocarcinoma marker genes.; citation_author=H Jiang; citation_author=Y Deng; citation_author=HS Chen; citation_author=L Tao; citation_author=Q Sha; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=7; citation_pages=81; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A consensus prognostic gene expression classifier for er positive breast cancer.; citation_author=AE Teschendorff; citation_author=A Naderi; citation_author=NL Barbosa-Morais; citation_author=SE Pinder; citation_author=IO Ellis; citation_journal_title=Genome Biology; citation_volume=7; citation_number=8; citation_pages=R101; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Improving identification of differentially expressed genes in microarray studies using information from public databases.; citation_author=R Kim; citation_author=P Park; citation_journal_title=Genome Biology; citation_volume=5; citation_number=9; citation_pages=R70; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A module map showing conditional activity of expression modules in cancer.; citation_author=E Segal; citation_author=N Friedman; citation_author=D Koller; citation_author=D Regev; citation_journal_title=Nat. Genet.; citation_volume=36; citation_number=10; citation_pages=1090-8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integrative analysis of genome-wide experiments in the context of a large high-throughput data compendium.; citation_author=A Tanay; citation_author=I Steinfeld; citation_author=M Kupiec; citation_author=R Shamir; citation_journal_title=Molecular Systems Biology; citation_volume=1; citation_number=11; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Distinctive gene expression patterns in human mammary epithelial cells and breast cancers.; citation_author=C Perou; citation_author=S Jeffrey; citation_author=M van de Rijn; citation_author=C Rees; citation_author=M Eisen; citation_journal_title=PNAS; citation_volume=96; citation_number=12; citation_pages=9212-7; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular portraits of human breast tumours.; citation_author=C Perou; citation_author=T Sorlie; citation_author=M Eisen; citation_author=M van de Rijn; citation_author=S Jeffrey; citation_journal_title=Nature; citation_volume=(406); citation_number=13; citation_pages=747-52; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications.; citation_author=T Sorlie; citation_author=C Perou; citation_author=R Tibshirani; citation_author=T Aas; citation_author=S Geisler; citation_journal_title=PNAS; citation_volume=98; citation_number=14; citation_pages=10869-74; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Semi-supervised methods to predict patient survival from gene expression data.; citation_author=E Bair; citation_author=R Tibshirani; citation_journal_title=Plos Biology; citation_volume=2; citation_number=15; citation_pages=e108; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=Journal of the Royal Statistical Society, Series B; citation_volume=57; citation_number=16; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A protocol for building and evaluating predictors of disease state based on microarray data.; citation_author=L Wessels; citation_author=M Reinders; citation_author=A Hart; citation_author=C Veenman; citation_author=H Dai; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=17; citation_pages=3755-62; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Beyond independence: conditions for the optimality of the simple bayesian classifier.; citation_author=P Domingos; citation_author=M Pazzani; citation_number=18; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The hallmarks of cancer.; citation_author=D Hanahan; citation_author=RA Weinberg; citation_journal_title=Cell; citation_volume=(1); citation_number=19; citation_pages=57-70; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A mitochondria k channel axis is suppressed in cancer and its normalization promotes apoptosis and inhibits cancer growth.; citation_author=S Bonnet; citation_author=S Archer; citation_author=J AllalunisTurner; citation_author=A Haromy; citation_author=C Beaulieu; citation_journal_title=Cancer Cell; citation_volume=100; citation_number=20; citation_pages=37-51; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Examination of POU homeobox gene expression in human breast cancer cells.; citation_author=T Jin; citation_author=D Branch; citation_author=X Zhang; citation_author=S Qi; citation_author=B Youngson; citation_journal_title=International Journal of Cancer; citation_volume=81; citation_number=21; citation_pages=104-112; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA dependent conversion of Oct1 and Oct2 into transcriptional repressors by Groucho/TLE.; citation_author=S Malin; citation_author=Y Linderson; citation_author=J Almqvist; citation_author=I Ernberg; citation_author=T Tallone; citation_journal_title=Nucl. Acids Res.; citation_volume=33; citation_number=22; citation_pages=4618-4625; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=23; " />
      <meta name="citation_reference" content="citation_number=24; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Module-Based Outcome Prediction Using Breast Cancer Compendia"/>
    <meta name="twitter:description" content="BackgroundThe availability of large collections of microarray datasets (compendia), or knowledge about grouping of genes into pathways (gene sets), is typically not exploited when training predictors of disease outcome. These can be useful since a compendium increases the number of samples, while gene sets reduce the size of the feature space. This should be favorable from a machine learning perspective and result in more robust predictors.MethodologyWe extracted modules of regulated genes from gene sets, and compendia. Through supervised analysis, we constructed predictors which employ modules predictive of breast cancer outcome. To validate these predictors we applied them to independent data, from the same institution (intra-dataset), and other institutions (inter-dataset).ConclusionsWe show that modules derived from single breast cancer datasets achieve better performance on the validation data compared to gene-based predictors. We also show that there is a trend in compendium specificity and predictive performance: modules derived from a single breast cancer dataset, and a breast cancer specific compendium perform better compared to those derived from a human cancer compendium. Additionally, the module-based predictor provides a much richer insight into the underlying biology. Frequently selected gene sets are associated with processes such as cell cycle, E2F regulation, DNA damage response, proteasome and glycolysis. We analyzed two modules related to cell cycle, and the OCT1 transcription factor, respectively. On an individual basis, these modules provide a significant separation in survival subgroups on the training and independent validation data."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001047.g009"/>

  <meta property="og:title" content="Module-Based Outcome Prediction Using Breast Cancer Compendia" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4660'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6189'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7740&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001047">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001047" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001047&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Module-Based Outcome Prediction Using Breast Cancer Compendia
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martin H. van Vliet
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:M.H.vanVliet@TUDelft.nl">M.H.vanVliet@TUDelft.nl</a></p>

                <p>Affiliations:
                  Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands, 
                  Bioinformatics and Statistics Group, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christiaan N. Klijn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands, 
                  Bioinformatics and Statistics Group, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands, 
                  Mouse Models for Breast Cancer, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lodewyk F. A. Wessels, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands, 
                  Bioinformatics and Statistics Group, Department of Molecular Biology, Netherlands Cancer Institute, Amsterdam, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marcel J. T. Reinders
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Information and Communication Theory Group, Faculty of Electrical Engineering, Mathematics and Computer Science, Delft University of Technology, Delft, The Netherlands
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 17, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001047</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001047-g001" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g002" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-t001" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g003" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g004" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g005" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g006" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g007" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g008" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001047-g009" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001047">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001047" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The availability of large collections of microarray datasets (compendia), or knowledge about grouping of genes into pathways (gene sets), is typically not exploited when training predictors of disease outcome. These can be useful since a compendium increases the number of samples, while gene sets reduce the size of the feature space. This should be favorable from a machine learning perspective and result in more robust predictors.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We extracted modules of regulated genes from gene sets, and compendia. Through supervised analysis, we constructed predictors which employ modules predictive of breast cancer outcome. To validate these predictors we applied them to independent data, from the same institution (intra-dataset), and other institutions (inter-dataset).</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We show that modules derived from single breast cancer datasets achieve better performance on the validation data compared to gene-based predictors. We also show that there is a trend in compendium specificity and predictive performance: modules derived from a single breast cancer dataset, and a breast cancer specific compendium perform better compared to those derived from a human cancer compendium. Additionally, the module-based predictor provides a much richer insight into the underlying biology. Frequently selected gene sets are associated with processes such as cell cycle, E2F regulation, DNA damage response, proteasome and glycolysis. We analyzed two modules related to cell cycle, and the OCT1 transcription factor, respectively. On an individual basis, these modules provide a significant separation in survival subgroups on the training and independent validation data.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>van Vliet MH, Klijn CN, Wessels LFA, Reinders MJT (2007) Module-Based Outcome Prediction Using Breast Cancer Compendia. PLoS ONE 2(10):
          e1047.
            doi:10.1371/journal.pone.0001047</p><p><strong>Academic Editor: </strong>Xiaolin Wu, National Cancer Institute at Frederick, United States of America</p><p><strong>Received:</strong> July 14, 2007; <strong>Accepted:</strong> September 28, 2007; <strong>Published:</strong> October 17, 2007</p><p><strong>Copyright:</strong> © 2007 van Vliet et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was part of the BioRange programme of the Netherlands Bioinformatics Centre (NBIC), which is supported by a BSIK grant through the Netherlands Genomics Initiative (NGI).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Unraveling the structure of complex biological processes from genomic data sources has been a focal point in bioinformatics research. Thus far, supervised analysis of microarray data has been performed in a data-driven fashion <a href="#pone.0001047-VantVeer1">[1]</a>–<a href="#pone.0001047-VandeVijver1">[4]</a>. These studies have reported and tested prognostic markers, sets of genes, which are predictive of treatment response and outcome.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>One of the main issues in data-driven approaches is the small ratio of samples relative to the number of genes for a particular study, causing small sample size related problems. This problem can be addressed by reducing the number of features (input variables) or increasing the number of samples. The latter approach was pursued by combining two or even more datasets and then deriving prognostic markers from the resulting dataset <a href="#pone.0001047-Rhodes1">[5]</a>–<a href="#pone.0001047-Teschendorff1">[8]</a>. Employing more samples results in, for instance, better estimates of gene variances and improves estimates of the t-statistic <a href="#pone.0001047-Kim1">[9]</a>. This approach was also followed by Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a> and Tanay <em>et al.</em> <a href="#pone.0001047-Tanay1">[11]</a> who constructed microarray gene expression compendia (collections of microarray data sets spanning a diversity of phenotypes).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The supervised analyses performed on compendia are data-driven and currently still employing single genes as input features. As an alternative, knowledge of functional groupings of genes into, for example pathways, can be employed to define meta-features, called modules. Such meta-features have two important advantages. Firstly, a relevant module can be directly linked to the biological processes that underly the observed outcome. Secondly, moving from a gene-based to a module-based representation reduces the number of input variables, which alleviates the small sample size problem.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a> proposed a framework for the unsupervised knowledge-driven analysis of expression data. Within this framework, modules are extracted based on relevant gene sets from a compendium of microarray data. We follow that approach, and extend the framework to include a supervised classification analysis based on the extracted modules and the available clinical data. In addition, we introduce cancer-specific compendia, as an intermediate step between a single dataset and a complete human cancer compendium. Employing the supervised framework, we evaluate the predictive performance of classifiers derived from cancer-specific datasets, a cancer specific compendium, and a human cancer compendium. In addition, we wanted to investigate the capacity of these classifiers to generalize beyond the dataset on which they were trained. Therefore, we set up an experiment in which we validated our classifier on independent data from the same institution (intra-dataset validation), a combination of institutions (cross-dataset validation), and by validating on data from different institutions (inter-dataset validation). Finally, since we adopted the module extraction of Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a>, the optimized set of modules that is selected by the supervised analysis allows for a more transparent analysis of the obtained results. That is, the modules can be related to the original gene sets, and thus, to cellular processes, giving more insight into the mechanisms causing the outcome differences.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Our method extends the unsupervised knowledge-driven framework proposed by Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a> to the supervised classification domain. This extension allows the identification of module-based prognostic markers, rather than gene-based markers. The entire outline of our methodology is presented in <a href="#pone-0001047-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0001047-g001"><div class="img"><a name="pone-0001047-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Workflow of the approach.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>We extended the analysis of compendia <a href="#pone.0001047-Segal1">[10]</a> to the supervised classification domain. Several microarray datasets were collected to construct compendia at various levels of underlying phenotype diversity (1). Additionally, we gathered a collection of biologically meaningful gene sets from available databases (2). Using the module extraction framework proposed by <a href="#pone.0001047-Segal1">[10]</a>, we derived sets of modules (3) from these compendia and gene sets. Using these modules we construct a module activity matrix (4), allowing modules rather than single genes to be used as features. The predictive power of the different sets of modules is inspected within a classification context. Using a train/test protocol (5), we estimated the generalization error of all sets of modules <a href="#pone.0001047-Wessels1">[17]</a>. Succeedingly, we trained a final classifier (6), which was then validated on independent data (7), and its performance assessed (8). Furthermore, the approach allows the final set of modules that were selected in the classifier to be compared to the original gene sets (9), allowing the identification of biological processes underlying the development and progression of cancer.</p>
<span>doi:10.1371/journal.pone.0001047.g001</span></div>
<h4>Input: Compendium</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The usual approach is to analyse a single microarray dataset in isolation. To find cancer related modules, Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a> proposed to take multiple datasets into account that are all related to human cancer types (<a href="#pone-0001047-g001">Figure 1</a>, Step 1). With this global human cancer compendium (HCC), they formed modules of genes that regulate cancer in a general way. We focus on breast cancer, for which various datasets are available. We propose to construct cancer specific compendia, in our case breast cancer compendia (BCC), as an intermediate step between a single breast cancer dataset (BC) and a complete human cancer compendium (HCC), see <a href="#pone-0001047-g002">Figure 2</a> and <a href="#pone-0001047-t001">Table 1</a>. These cancer specific compendia will reduce the small sample size problem, but at the same time should ensure coherence in underlying phenotype compared to the more global human cancer compendium. <a href="#pone-0001047-g002">Figure 2</a> shows an example of how datasets from different institutions have been grouped into compendia.</p>
<div class="figure" id="pone-0001047-g002"><div class="img"><a name="pone-0001047-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Compendia of microarray data.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Microarray datasets can be grouped into compendia at various levels of underlying phenotypic diversity. The pie-chart indicates datasets from various origins, sizes, and cancer types, and the compendia are indicated by the outer rings. The ’Inter1’ training-validation configuration is depicted in this figure (<a href="#pone.0001047-VandeVijver1">[4]</a> as training, and <a href="#pone.0001047-Wang1">[3]</a> as validation). This is one of the six configurations employed (See <a href="#pone-0001047-t001">Table 1</a> for details).</p>
<span>doi:10.1371/journal.pone.0001047.g002</span></div><div class="figure" id="pone-0001047-t001"><div class="img"><a name="pone-0001047-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Experimental setup.</span></strong></p><span>doi:10.1371/journal.pone.0001047.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In our analyses we have also used the HCC that was published by Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a>. This compendium contains data from various cancer types and has a total of 1973 arrays, for which 14143 genes are present. The compendium already contained data from three previous breast cancer studies, in total 152 arrays: 26 arrays from the first study by Perou <em>et al.</em> <a href="#pone.0001047-Perou1">[12]</a>, 41 arrays from the second study by Perou <em>et al.</em> <a href="#pone.0001047-Perou2">[13]</a>, and 85 arrays from Sorlie <em>et al.</em> <a href="#pone.0001047-Sorlie2">[14]</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The additional breast cancer microarray datasets that we have used, originate from previously published research <a href="#pone.0001047-Sorlie1">[2]</a>–<a href="#pone.0001047-VandeVijver1">[4]</a>. The Vijver dataset consists of 295 breast cancer patients, the Wang dataset consists of 285 records, and the Sorlie data consists of 167 records. To be able to use these datasets in conjunction with the HCC, we mapped all the available probes to the same set of Entrez ids in the HCC. Furthermore, after mean-normalisation, we discretized each dataset separately into three levels: induced (1), basal (0), and repressed (−1), taking into account the skewing and variance in each of the datasets (Discretization was applied, because the module extraction procedure that Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a> proposed, requires a discrete input.).</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Outcome data (time to metastasis) was available for all patients in the Vijver and Wang datasets. For classification, the poor outcome group was defined as all patients with time to metastasis less than five years, and the good outcome group as those with time to metastasis greater than or equal to five years. Censored patients in the poor group were not taken into account when training and assessing a classifier. On the other hand, censored patients in the good group were included in both the training and validation <a href="#pone.0001047-Bair1">[15]</a>.</p>


<h4>Input: Gene Set Collection</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We collected 2682 gene sets from several biological databases and resources (<a href="#pone-0001047-g001">Figure 1</a>, Step 2), including some additional databases that were not included in the collection of gene sets used by Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a>, see <a href="#pone-0001047-g003">Figure 3</a>. In the original analysis presented by Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a>, approximately half of the gene sets were obtained by performing hierarchical clustering on the expression data. We chose to omit any hierarchical clusters in the collection of gene sets, as the inclusion of hierarchical clusters would introduce an additional data-driven bias. As a result, our analysis is more knowledge-driven when compared to the original study.</p>
<div class="figure" id="pone-0001047-g003"><div class="img"><a name="pone-0001047-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Pie chart indicating the origin of the gene sets.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A total of 2682 gene sets were collected. The GO, KEGG, GenMapp, and Tissue specific gene sets were taken from the study by Segal et al. <a href="#pone.0001047-Segal1">[10]</a>. The Reactome pathways were downloaded from the Reactome website <a href="#pone.0001047-MSigDB1">[23]</a>, and the MSDB gene sets were taken from the molecular signature database <a href="#pone.0001047-Reactome1">[24]</a>.</p>
<span>doi:10.1371/journal.pone.0001047.g003</span></div>

<h4>Unsupervised Analysis</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To extract modules from compendia of microarrays, we largely followed the knowledge-driven approach proposed by Segal <em>et al.</em> <a href="#pone.0001047-Segal1">[10]</a> (<a href="#pone-0001047-g001">Figure 1</a>, Step 3). In short, this unsupervised method finds modules in (compendia of) discretized microarray data. A module is defined as a subset of genes with correlated expression across a set of arrays, and is constructed by combining (parts of) gene sets based on discretized gene expression data. The module extraction process can be seeded by biologically relevant gene sets (extracted from e.g. GO and KEGG), thus incorporating a knowledge-driven component in the analysis. An in depth description of the procedure is given in the supplementary <a href="#pone.0001047.s001">Text S1</a>, and supplementary <a href="#pone.0001047.s002">Figure S1</a>.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Following the extraction of the modules, a module activity matrix is constructed for the training data as well as the validation data (<a href="#pone-0001047-g001">Figure 1</a>, Step 4). The module activity matrix represents the behavior of the group of genes in a module by a discrete variable.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The conversion from gene expression to module activity is done per array, per module. For the induced and repressed genes separately, we test whether the overlap of induced or repressed genes on the array with the module is significant, compared to a random draw. To this end, we use the hypergeometric distribution to calculate a p-value for the significance of the overlap. Following FDR correction <a href="#pone.0001047-Benjamini1">[16]</a> (significance threshold = 0.05) one of the following four situations occur:</p>


<ul class="bulletlist">

<li>Neither p-value is significant: the module activity is basal (i.e. 0)</li>

<li>Only the induction p-value is significant: the module is induced (i.e. 1)</li>

<li>Only the repression p-value is significant: the module is repressed (i.e. −1)</li>

<li>Both p-values are significant: the module activity is basal (i.e. 0)</li>

</ul><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p><a href="#pone-0001047-g004">Figure 4</a> presents an example of a set of microarrays and modules that are converted to a module activity matrix.</p>
<div class="figure" id="pone-0001047-g004"><div class="img"><a name="pone-0001047-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Converting gene expression data into module activity data.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>For a given gene expression dataset, and a set of modules we assessed the statistical significance of the overlap of induced/repressed genes with the modules using the hypergeometric distribution. This leads to two p-values for each array/module pair. These p-values are combined into a single discrete module activity score.</p>
<span>doi:10.1371/journal.pone.0001047.g004</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>For each of the validation schemes (<a href="#pone-0001047-t001">Table 1</a>) we constructed several module-based classifiers (BC, BCC, HCC, S456) based on modules extracted from the datasets indicated in the Features column of <a href="#pone-0001047-t001">Table 1</a>.</p>


<h4>Supervised Analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Supervised classification provides a means to identify modules with activities that are significantly associated with some relevant outcome variable, such as, metastasis-free survival in breast cancer (<a href="#pone-0001047-g001">Figure 1</a>, Step 5). To obtain an unbiased estimate of the generalization error of the different sets of modules, we used a double-loop cross validation procedure <a href="#pone.0001047-Wessels1">[17]</a>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In our experiment, we focused on differences between the sets of modules and we omitted an extensive evaluation of a range of different classifier types. Since all features are discrete, we used forward filtering as module selector, the mutual information as criterion to evaluate the individual modules (using maximally 200 modules), and a simple Bayes classifier <a href="#pone.0001047-Domingos1">[18]</a>. For the discretized gene-expression data we used the same setup as for the module-based approach.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Following the Train/Test procedure, we trained a final classifier (<a href="#pone-0001047-g001">Figure 1</a>, Step 6). This classifier was trained using the top ranked features, that were estimated in the train/test protocol. The final classifier was validated on an independent dataset (<a href="#pone-0001047-g001">Figure 1</a>, Step 7), which had not been employed in any of the training steps (<a href="#pone-0001047-g001">Figure 1</a>, Steps 2–6).</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>To assess the performance of the classifiers on the independent validation dataset, we constructed an ROC curve (<a href="#pone-0001047-g001">Figure 1</a>, Step 8). To compare the performance of various feature types we adopted the area under the curve (AUC) as a performance measure.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Finally, a feedback step relates the modules selected in the classifier to the original gene sets (<a href="#pone-0001047-g001">Figure 1</a>, Step 9). This allows direct insight into the underlying mechanisms, compared to the annotation lookup of single genes in terms of functional groups in data-driven approaches.</p>


<h4>Experimental Setup</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We wanted to investigate the capacity of our classifiers to generalize beyond the dataset that they were trained on. Therefore, we designed our experiments such that three different validation schemes were possible. In all cases the training and validation sets are non-overlapping (independent), i.e. no samples that were used during module extraction/training are employed in the validation. The following three scenarios were considered: training and validation on data from 1) the same institution (denoted as intra-lab validation); 2) a combination of the same and other institutions (cross-lab validation); and 3) separate institutions (inter-lab validation). Since we had equivalent outcome data for the Vijver and Wang datasets, we mirrored the role of both so that we ended up with a total of six experiments, as shown in <a href="#pone-0001047-t001">Table 1</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Extracting modules from the compendia</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>For each of the compendia we derived a set of modules using the discretized gene expression data as well as the 2682 gene sets as input. The number of modules that were found are listed in <a href="#pone-0001047-t001">Table 1</a>. The number of modules found ranged from 576 to 1163, which is a significant reduction in the number of features from the original 14143 genes. Additionally, we included the previously published 456 modules <a href="#pone.0001047-Segal1">[10]</a> in the current investigation (S456). These differ from the HCC modules, as they are constructed based on gene sets derived by hierarchical clustering.</p>


<h4>Classification performances</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The classification performances of the experiments listed in <a href="#pone-0001047-t001">Table 1</a>, are compared based on the AUCs obtained on the validation data. For each of the six experiments (Intra1, Cross1, Inter1, Intra2, Cross2, Inter2) the results obtained with each feature type (BC, BCC, HCC, S456, Genes) were ranked based on the AUC. <a href="#pone-0001047-g005">Figure 5</a> shows a boxplot of the ranks obtained for each of the feature types. A table containing all individual AUC values and ranks is presented in the supplementary information (<a href="#pone.0001047.s007">Table S1</a>). The median rank of the BC features is the lowest of all the feature sets, the BCC median rank is slightly worse, but still better than the HCC, S456, and Genes features which perform the worst.</p>
<div class="figure" id="pone-0001047-g005"><div class="img"><a name="pone-0001047-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Boxplot showing ranked AUC results.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Boxplot showing the median ranks of the performance of each of the five feature types across the six experiments (see <a href="#pone-0001047-t001">Table 1</a>). In each of the six experiments the features were ranked based on the AUC obtained on the independent validation set (1 best, 5 worst).</p>
<span>doi:10.1371/journal.pone.0001047.g005</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>To assess the statistical significance of the observed differences between the median ranks, we applied a one-sided Wilcoxon rank sum test to all pair-wise combinations of feature types. The obtained p-values are depicted in the left panel in <a href="#pone-0001047-g006">Figure 6</a>. We also employed the Wilcoxon rank sum test to perform pair-wise comparisons between the feature types derived from breast cancer compendia (BC+BCC), features types derived from human cancer compendia (HCC+S456) and Genes. The results are depicted in the right panel of <a href="#pone-0001047-g006">Figure 6</a>.</p>
<div class="figure" id="pone-0001047-g006"><div class="img"><a name="pone-0001047-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Pairwise comparison of the five feature types.</span></strong></p><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1"></a><p>Each cell (row = i, column = j) depicts the the p-value obtained by performing a one-sided Wilcoxon rank sum test with as alternative hypothesis that the median rank of type i is lower than type j, based on the AUCs achieved for each of the six experiments. The plot on the left shows individual comparisons, the plot on the right includes comparisons of groups of features. Cell-shading reflects the p-values.</p>
<span>doi:10.1371/journal.pone.0001047.g006</span></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>From the left panel in <a href="#pone-0001047-g006">Figure 6</a> we can conclude that, although the BCC modules have a lower median rank compared to the HCC modules, S456 modules, and Genes, there is not enough statistical evidence (at the 0.05 level) to support the claim that the BCC modules outperform the HCC modules, S456 modules, or the Genes. Since the BCC modules are derived from a larger collection of data than the BC modules, we would have expected a performance that is at least equal and possibly even better than the BC modules. Since breast cancer is known to be a heterogeneous disease, we hypothesize that differences in the subtype composition of the datasets cause the poorer performance of the BCC modules.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>From the right panel in <a href="#pone-0001047-g006">Figure 6</a>, we can, however, conclude that the BC and BCC modules jointly perform better than the HCC, and S456 modules (p = 0.006). This indicates that a human cancer compendium lacks specificity with respect to a breast cancer compendium. We can therefore conclude that for breast cancer specific prediction, a cancer specific compendium is more suitable compared to a more global human cancer compendium. As shown by Segal <em>et al.</em><a href="#pone.0001047-Segal1">[10]</a> the HCC and S456 modules may still be relevant for identifying differences between cancer types.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>The pairwise comparisons (left panel in <a href="#pone-0001047-g006">Figure 6</a>) indicate that the median rank of the BC modules is better than each of the other feature types (all p&lt;0.05). Moreover, gene-based classifiers show a very large variability in comparison to BC module-based classifiers (see <a href="#pone-0001047-g005">Figure 5</a>). One possible explanation for this observation is that the conversion of gene expression data to module activity data may, in fact, function as a form of regularization which removes noise. Typically, the amount of regularization needs to be optimized for a given classifier. We hypothesize that the fact that genes in a module are roughly associated with the same biological process, ensures an optimal degree of regularization.</p>
<a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>For a given classifier to serve as a prognostic index in clinical practice, a suitable operating point on its associated ROC curve needs to be selected. For outcome prediction in breast cancer, the True Positive Rate (TPR) is typically set at a desired threshold, and based on the ROC curve, the corresponding False Positive Rate (FPR) possible is determined. Therefore, we have re-evaluated the AUC scores by integrating over the TPR interval ranging from 0.5 to 1. This interval was chosen since it reflects a clinically more relevant range than the complete TPR range ([0,1]). All results are reported in the supplementary information (<a href="#pone.0001047.s008">Table S2</a>, <a href="#pone.0001047.s003">Figure S2</a> and <a href="#pone.0001047.s004">Figure S3</a>). Consistent with earlier results, the BC modules perform significantly better compared to all other feature types. In addition, the BCC modules now have a significantly lower median rank compared to the HCC and S456 modules (p = 0.05, and p = 0.01). This strengthens our conclusion that the BC modules perform better than the other feature types, and that a breast cancer specific compendium performs better compared to a human cancer compendium, especially when considering a clinically relevant setting.</p>


<h4>Interpretability of Gene and Module-based signatures</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>All classifiers output a signature of relevant features, that is predictive for survival. To better understand the biological processes associated with disease outcome in breast cancer, the signatures are further investigated. For gene-based features, the overlap with known pathways is employed to attach biological meaning to an obtained signature. Module-based classifiers, on the other hand, return a set of predictive modules rather than single genes. Each of these individual modules may provide links to relevant biological processes. Moreover, since modules are extracted from the data by combining (parts of) gene sets, these may link additional genes to known pathways, which are relevant to disease progression in breast cancer.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>To explore the association of the signatures to biological processes, we analyzed a gene-based and BC module-based signature. We chose to compare the gene-based signature and BC module-based signature from Inter1 (<a href="#pone-0001047-t001">Table 1</a>), since they were derived from the same dataset. These signatures consist of 21 genes, and 55 modules (<a href="#pone.0001047.s006">Dataset S1</a>), respectively.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>For every module in the module-based signature, as well as the 21 single-gene signature, we computed the enrichment for each of the 2682 gene sets employing the hypergeometric distribution. For the gene-based signature no gene sets were significantly enriched (<em>p</em>&lt;0.05 after Bonferroni correction), whereas 319 gene sets were significantly enriched in at least 1 of the 55 modules in the module-based signature (<em>p</em>&lt;0.05 after Bonferroni correction). The complete matrices of raw p-values are depicted in supplementary information (<a href="#pone.0001047.s001">Text S1</a>, and <a href="#pone.0001047.s005">Figure S4</a>).</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Many of the 319 gene sets are associated with similar biological processes within the context of the 55 module signature (i.e. they have similar enrichment profiles across the 55 modules). Therefore, we clustered the gene sets based on enrichment scores into seven distinct clusters employing complete linkage, hierarchical clustering with Euclidean distance as dissimilarity measure. The common biological themes associated with the gene sets in each of the resulting seven clusters are listed on the left in <a href="#pone-0001047-g007">Figure 7</a>.</p>
<div class="figure" id="pone-0001047-g007"><div class="img"><a name="pone-0001047-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Comparison of a module-based signature (A) and a gene-based signature (B).</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>The module-based signature from the Inter1 experiment contains 55 modules, and the gene-based signature contains 21 genes (<a href="#pone-0001047-t001">Table 1</a>). For both signatures an enrichment score for their overlap with the collection of 2682 gene sets was calculated based on the hypergeometric distribution. This resulted in a total of 319 gene sets that were enriched in at least one module or in the gene-based signature (<em>p</em>&lt;0.05 after Bonferroni correction), see supplemental <a href="#pone.0001047.s005">figure S4</a>. Several modules turned out to have a similar pattern of enrichment across the gene sets. Additionally, gene sets that relate to a common theme turned out to have a similar enrichment pattern across the modules. Therefore, we clustered the matrix of p-values in both dimensions (2-dimensional, hierarchical clustering, complete linkage, Euclidean distance). The dendrograms at the top, and to the left indicate the clustering, where we chose to group either dimension into seven distinct groups. The labels on the left indicate the most common biological theme, and the label on the bottom indicates the groups of modules formed along with the number of modules in each group in brackets. The main table shows the median p-value for the enrichment of each of the seven clusters of modules, across these seven groups of gene sets. Similarly, the table on the right shows the median p-values for the gene signature. Shading of the cells reflects the p-values.</p>
<span>doi:10.1371/journal.pone.0001047.g007</span></div><a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>Based on the complete table of enrichment p-values (Supplementary <a href="#pone.0001047.s005">Figure S4</a>), it is evident that there are clusters of modules that also show a similar enrichment pattern across the gene sets. Therefore, we clustered the modules into seven distinct groups, as depicted by the dendrogram at the top in supplementary <a href="#pone.0001047.s005">Figure S4</a>. These clusters were labeled Module groups 1 to 7, as indicated at the bottom in <a href="#pone-0001047-g007">Figure 7</a>.</p>
<a id="article1.body1.sec3.sec3.p6" name="article1.body1.sec3.sec3.p6"></a><p>The main table in <a href="#pone-0001047-g007">Figure 7</a> shows the aggregated enrichment p-values. More specifically, cell (<em>i</em>, <em>j</em>) depicts the median enrichment p-value for all modules in module group <em>j</em> with respect to all gene sets in gene set group <em>i</em>. The column vector on the right shows the median enrichment scores for the groups of gene sets with respect to the single gene signature.</p>
<a id="article1.body1.sec3.sec3.p7" name="article1.body1.sec3.sec3.p7"></a><p><a href="#pone-0001047-g007">Figure 7</a> shows several strong links of the module-based signature to biological processes. Five out of the seven groups of modules can be linked to biological processes that are known to be involved in cancer progression (Cell Cycle, DNA Damage, E2F transcription factors, and Proteasome). Most of these have been previously related to breast cancer <a href="#pone.0001047-Hanahan1">[19]</a>. It is interesting to note that glycolysis has only recently been identified as a key factor in tumor progression <a href="#pone.0001047-Bonnet1">[20]</a>.</p>


<h4>A detailed analysis of two modules</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Based on the module activity representation, the arrays in a dataset can be separated into 3 groups: arrays where the module is activated, repressed or showing basal activity. Using this separation, we present the Kaplan-Meier curves for two modules from the module-based signature, on the training data <a href="#pone.0001047-VandeVijver1">[4]</a>, as well as on the independent validation data <a href="#pone.0001047-Wang1">[3]</a>, see <a href="#pone-0001047-g008">Figures 8</a> and <a href="#pone-0001047-g009">9</a>.</p>
<div class="figure" id="pone-0001047-g008"><div class="img"><a name="pone-0001047-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>A cell cycle related module.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>A) Module activity data of a Cell Cycle related module (Module group 2 in <a href="#pone-0001047-g007">Figure 7</a>) that was extracted from the Vijver <a href="#pone.0001047-Jin1">[21]</a> data (Inter1, <a href="#pone-0001047-t001">Table 1</a>). The top heatmap shows the binary condition label, and the discrete module activity data (rows), for all the Vijver arrays (columns) <a href="#pone.0001047-VandeVijver1">[4]</a>. Arrays are ordered according to the metastasis free survival time. The heatmap in the middle shows the discrete gene expression data for the 55 genes (rows) in the module. On the left, a binary heatmap shows the 55 genes, along with the gene sets that show the most significant overlap with this module. The gene sets are ranked based on their p-value for the overlap with the module (hypergeometric distribution), we show the top 10 gene sets (p-values ranging from 10<sup>−51</sup> to 10<sup>−25</sup>, all significant at <em>p</em>&lt;0.05 after Bonferroni correction). On the right, two Kaplan-Meier curves indicate the predictive power of this module when arrays with the same module activity are grouped. B) The Kaplan-Meier curves for the three groups defined by the activity of this module on the Vijver <a href="#pone.0001047-Jin1">[21]</a> data (Inter1 training, <a href="#pone-0001047-t001">Table 1</a>). C) The Kaplan-Meier curves for the three groups defined by the activity of this module on the independent <a href="#pone.0001047-Wang1">[3]</a> data (Inter1 test data, <a href="#pone-0001047-t001">Table 1</a>). The legend indicates the three groups and lists the number of events and total number within the groups. P-values correspond to the logrank test.</p>
<span>doi:10.1371/journal.pone.0001047.g008</span></div><div class="figure" id="pone-0001047-g009"><div class="img"><a name="pone-0001047-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001047" data-uri="info:doi/10.1371/journal.pone.0001047.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001047.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001047.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001047.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001047.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>An Oct1 related module.</span></strong></p><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1"></a><p>A) Module activity data of an OCT1 transcription factor related module (Module group 4 in <a href="#pone-0001047-g007">Figure 7</a>) that was extracted from the Vijver <a href="#pone.0001047-Jin1">[21]</a> data (Inter1, <a href="#pone-0001047-t001">Table 1</a>). The top heatmap shows the binary condition label, and the discrete module activity data (rows), for all the Vijver arrays (columns) <a href="#pone.0001047-VandeVijver1">[4]</a>. Arrays are ordered according to the metastasis free survival time. The heatmap in the middle shows the discrete gene expression data for the 47 genes (rows) in the module. On the left, a binary heatmap shows the 47 genes, along with the gene sets that show the most significant overlap with this module. The gene sets are ranked based on their p-value for the overlap with the module (hypergeometric distribution), we show the top 10 gene sets (p-values ranging from 10<sup>−13</sup> to 10<sup>−7</sup>, all significant at <em>p</em>&lt;0.05 after Bonferroni correction). On the right, two Kaplan-Meier curves indicate the predictive power of this module when arrays with the same module activity are grouped. B) The Kaplan-Meier curves for the three groups defined by the activity of this module on the Vijver <a href="#pone.0001047-Jin1">[21]</a> data (Inter1 training, <a href="#pone-0001047-t001">Table 1</a>). C) The Kaplan-Meier curves for the three groups defined by the activity of this module on the independent <a href="#pone.0001047-Wang1">[3]</a> data (Inter1 test data, <a href="#pone-0001047-t001">Table 1</a>). The legend indicates the three groups and lists the number of events and total number within the groups. P-values correspond to the logrank test.</p>
<span>doi:10.1371/journal.pone.0001047.g009</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p><a href="#pone-0001047-g008">Figure 8</a> shows a module (from Module group 2) that has a highly significant overlap with Cell Cycle related gene sets (enrichment of the top gene set: <em>p</em>&lt;10<sup>−51</sup>), see <a href="#pone-0001047-g007">Figure 7</a>. This module is significantly associated with disease progression on both the datasets from Vijver <em>et al.</em> <a href="#pone.0001047-VandeVijver1">[4]</a> (<em>p&lt;</em>0.0001) and Wang <em>et al.</em> <a href="#pone.0001047-Wang1">[3]</a> (<em>p</em> = 0.0127). Deregulation of the cell cycle has been identified as one of the hallmarks of cancer <a href="#pone.0001047-Hanahan1">[19]</a>. More importantly, an increased activity of the cell cycle has been linked to more aggressive tumors. This is in accordance with our observation for this module, which shows that an induced module activity is linked to the subgroup with the worst outcome. Conversely, a repressed module activity shows the best outcome.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p><a href="#pone-0001047-g009">Figure 9</a> shows a module (from Module group 4) that has a significant enrichment for OCT1 transcription factor related gene sets (enrichment of the top gene set: <em>p</em>&lt;10<sup>−12</sup>). The Kaplan-Meier curves show a significant separation between the induced, basal, and repressed module activities on both the Vijver <em>et al.</em> <a href="#pone.0001047-VandeVijver1">[4]</a> data (<em>p</em>&lt;0.0001), and the Wang <em>et al.</em> <a href="#pone.0001047-Wang1">[3]</a> data (<em>p</em> = 0.0098). In breast cancer, the OCT1 transcription factor is known to be often overexpressed <a href="#pone.0001047-Jin1">[21]</a> relative to normal breast tissue, but its exact role in the tumorigenic process has remained unclear. Additionally, OCT1 has been identified as a transcriptional repressor <a href="#pone.0001047-Malin1">[22]</a>. We show that the concerted repression of downstream targets of the OCT1 transcription factor relates to a poor outcome group. On the other hand, an induced module activity relates to a subgroup with significantly better outcome. Thus, this module can be identified as a potential tumor suppressor module.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>By extending an existing unsupervised knowledge-driven framework to the supervised classification domain, we were able to investigate the effects of including knowledge from previous gene expression studies (through compendia) as well as known cellular processes (through gene sets) on the accuracy of outcome prediction in breast cancer. Our analysis included a validation of the classifiers on independent data, which allowed for an objective evaluation of the actual generalization behavior of the gene-based and module-based classifiers in a clinically relevant setting.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Classifiers based on genes had a very large variance, compared to the BC module-based classifier. We hypothesize that the conversion of gene expression data to module activity functions as a regularization step, where the extent of the regularization is controlled by the specificity of the modules, resulting in more stable classifiers.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>Overall, a trend emerges in the performance versus compendium specificity. Modules from the most specific single dataset showed the best performance-significantly outperforming all other classifiers. These were closely followed by modules extracted from a breast cancer specific compendium, which performed significantly better than modules from the human cancer compendium when evaluated across a clinically relevant TPR range. Finally, modules from the human cancer compendium showed the weakest performance. This indicates that it is preferable to employ a compendium specific to the cancer type under study. Moreover, the heterogeneity between different institutions tends to be more detrimental than the gain in sample size when a breast cancer specific compendium is constructed.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>A module-based approach to classification provides a signature of predictive modules, as opposed to a gene-based signature. Interpretation of a gene-based signature is usually limited to a mapping of the genes in the signature to functional categories. However, for the approach outlined here, it holds that the modules were constructed from biologically meaningful gene sets, and therefore these can be linked directly to the underlying biological processes. We illustrated this advantage by providing a meta-representation of the modules in one of the module-based classifiers, which reveals molecular processes, such as cell cycle, DNA damage, glycolysis, and proteasome, known to be involved in breast cancer. The gene-based signature provided no significant links at all. This gain in biological insight greatly favors the use of a module-based classifier.</p>
<a id="article1.body1.sec3.sec5.p5" name="article1.body1.sec3.sec5.p5"></a><p>Our research includes an in-depth analysis of two modules that were part of the module-based signature, which were related to cell cycle, and the OCT1 transciption factor. By themselves, these modules provide a significant separation in subgroups on the training and independent validation data. The cell cycle related module indicated that an induced module activity is linked to the worst outcome. This confirms the well known relationship between the cell cycle process and cancer in general. On the other hand, the OCT1 related module revealed a novel relationship to breast cancer outcome. Based on its module activity, this module could be designated as a tumor repressor module. Neither of these factors could be revealed from the gene-based signature. Therefore, we conclude that module-based signatures provide a much richer insight to the underlying biology compared to gene-based signatures.</p>
<a id="article1.body1.sec3.sec5.p6" name="article1.body1.sec3.sec5.p6"></a><p>Research on outcome prediction not only contributes to the development of reliable diagnostic tests, but also by improving our understanding of the processes involved in carcinogenesis, and specifically how these influence disease progression and therapy response. From a practical perspective, diagnostic tests based on small gene sets are preferred, and are also designed with this objective in mind. However, such sets often fail to provide significant biological insight into the disease. Our module-based classifiers were not designed to employ a minimal number of genes, and the large number of genes employed could be a limitation to the direct application of these classifiers in a clinical setting. However, in our study, the module-based classifier had a significantly lower variance in performance than the gene based classifier, a property which is clearly preferable in the clinical setting. We clearly demonstrate that the module-based gene sets provide a much richer feedback by revealing functional categories associated with disease outcome. These insights could speed up the development of anti-cancer drugs, since the identified processes will help focus the search for viable drug targets. In conclusion, while module-based classifiers are perhaps less practical for clinical use due to the large gene sets being employed, their robustness and the biological insights they provide will most likely result in both short and long term clinical benefit.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001047"></div><a name="pone.0001047.s001" id="pone.0001047.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s001">Text S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Module-based outcome prediction using breast cancer compendia.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001047.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0001047.s002" id="pone.0001047.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s002">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Methodology overview. Overview of the unsupervised module extraction procedure, followed by a supervised investigation of the relation between module expression and conditions. In this example no FDR correction was done, so as to retain a fair amount of significantly expressed gene sets/modules.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001047.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.16 MB EPS)</p>
<a name="pone.0001047.s003" id="pone.0001047.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s003">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Boxplot showing ranked AAC results. In each of the six experiments the features were ranked based on the AAC (TPR range from 0.5 to 1) obtained on the independent validation set (1 best, 5 worst). This boxplot shows the median rank along with the quartile ranges for each of the five features.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001047.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.59 MB EPS)</p>
<a name="pone.0001047.s004" id="pone.0001047.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s004">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Comparison of ranked AAC results. Two tables showing a pairwise comparison of the five feature types. Each cell (row = i, column = j) depicts the the p-value obtained by performing a one-sided Wilcoxon rank sum test with as null hypothesis that the median rank of type i is lower than type j, based on the AACs (TPR range from 0.5 to 1) achieved for each of the six experiments. The plot on the left shows individual comparisons, the plot on the right includes comparisons of groups of features. Cell-shading reflects the p-values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001047.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.81 MB EPS)</p>
<a name="pone.0001047.s005" id="pone.0001047.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s005">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Comparison of a module-based signature (A) and a gene-based signature (B). The module-based signature from the Inter1 experiment contains 55 modules, and the gene-based signature contains 21 genes (<a href="#pone-0001047-t001">Table 1</a>). For both signatures an enrichment score for their overlap with the collection of 2682 gene sets was calculated based on the hypergeometric distribution. This resulted in a total of 319 gene sets that were enriched in at least one module or in the gene-based signature (P&lt;0.05 after Bonferroni correction). Several modules turned out to have a similar pattern of enrichment across the gene sets. Additionally, gene sets that relate to a common theme turned out to have a similar enrichment pattern across the modules. Therefore, we clustered the matrix of p-values in both dimensions (2-dimensional, hierarchical clustering, complete linkage, Euclidean distance). The dendrograms at the top, and to the left indicate the clustering, where we chose to group either dimension into seven distinct groups. The labels on the right indicate the individual gene set labels, and the label on the bottom indicates the groups of modules formed along with the number of modules in each group in brackets. The main table shows the median p-value for the enrichment of each of the seven clusters of modules, across these seven groups of gene sets. Similarly, the table on the right shows the median p-values for the gene signature. Shading of the cells reflects the p-values.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001047.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(4.12 MB EPS)</p>
<a name="pone.0001047.s006" id="pone.0001047.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s006">Dataset S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001047.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(5.69 MB XLS)</p>
<a name="pone.0001047.s007" id="pone.0001047.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s007">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001047.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001047.s008" id="pone.0001047.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001047.s008">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001047.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank L.J. van 't Veer, and D.S.A Nuyten for their valuable discussions on the work presented.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LW CK Mv MR. Performed the experiments: Mv. Analyzed the data: Mv. Wrote the paper: LW Mv MR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001047-VantVeer1" id="pone.0001047-VantVeer1"></a>Van 't Veer L, Dai H, van de Vijver M, He Y, Hart A, et al.  (2002) Gene expression profiling predicts clinical outcome of breast cancer. Nature  415: 530–6.  <ul class="find" data-citedArticleID="1006228" data-doi="10.1038/415530a"><li><a href="http://dx.doi.org/10.1038/415530a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+profiling+predicts+clinical+outcome+of+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001047-Sorlie1" id="pone.0001047-Sorlie1"></a>Sorlie T, Tibshirani R, Parker J, Hastie T, Marron J, et al.  (2003) Repeated observation of breast tumor subtypes in independent gene expression data sets. PNAS  14: 8418–23.  <ul class="find" data-citedArticleID="1006218" data-doi="10.1073/pnas.0932692100"><li><a href="http://dx.doi.org/10.1073/pnas.0932692100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repeated+observation+of+breast+tumor+subtypes+in+independent+gene+expression+data+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repeated+observation+of+breast+tumor+subtypes+in+independent+gene+expression+data+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001047-Wang1" id="pone.0001047-Wang1"></a>Wang Y, Klein J, Zhang Y, Sieuwerts A, Look M, et al.  (2005) Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer. The Lancet  365: 671–9.  <ul class="find" data-citedArticleID="1006230" data-doi="10.1016/s0140-6736(05)70933-8"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)70933-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene-expression+profiles+to+predict+distant+metastasis+of+lymph-node-negative+primary+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene-expression+profiles+to+predict+distant+metastasis+of+lymph-node-negative+primary+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001047-VandeVijver1" id="pone.0001047-VandeVijver1"></a>Van de Vijver M, He Y, van 't Veer L, Dai H, Hart A, et al.  (2002) A gene-expression signature as a predictor of survival in breast cancer. N. Engl. J. Med.  25: 1999–2009.  <ul class="find" data-citedArticleID="1006226" data-doi="10.1056/nejmoa021967"><li><a href="http://dx.doi.org/10.1056/nejmoa021967" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene-expression+signature+as+a+predictor+of+survival+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene-expression+signature+as+a+predictor+of+survival+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001047-Rhodes1" id="pone.0001047-Rhodes1"></a>Rhodes D, Yu J, Shanker K, Deshpande N, Varambally R, et al.  (2004) Oncomine: a cancer microarray database and integrated data-mining platform. Neoplasia  1: 1–6.  <ul class="find" data-citedArticleID="1006212" data-doi="10.1016/s1476-5586(04)80047-2"><li><a href="http://dx.doi.org/10.1016/s1476-5586(04)80047-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oncomine%3A+a+cancer+microarray+database+and+integrated+data-mining+platform." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oncomine%3A+a+cancer+microarray+database+and+integrated+data-mining+platform.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001047-Shen1" id="pone.0001047-Shen1"></a>Shen R, Ghosh D, Chinnaiyan A (2004) Prognostic meta-signature of breast cancer developed by two-stage mixture modelling of microarray data. BMC Genomics  5: 94.  <ul class="find" data-citedArticleID="1006216"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Prognostic+meta-signature+of+breast+cancer+developed+by+two-stage+mixture+modelling+of+microarray+data.&amp;auth=&amp;atitle=Prognostic+meta-signature+of+breast+cancer+developed+by+two-stage+mixture+modelling+of+microarray+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prognostic+meta-signature+of+breast+cancer+developed+by+two-stage+mixture+modelling+of+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prognostic+meta-signature+of+breast+cancer+developed+by+two-stage+mixture+modelling+of+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001047-Jiang1" id="pone.0001047-Jiang1"></a>Jiang H, Deng Y, Chen HS, Tao L, Sha Q, et al.  (2004) Joint analysis of two microarray gene-expression data sets to select lung adenocarcinoma marker genes. BMC Bioinformatics  5: 81.  <ul class="find" data-citedArticleID="1006196"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Joint+analysis+of+two+microarray+gene-expression+data+sets+to+select+lung+adenocarcinoma+marker+genes.&amp;auth=&amp;atitle=Joint+analysis+of+two+microarray+gene-expression+data+sets+to+select+lung+adenocarcinoma+marker+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Joint+analysis+of+two+microarray+gene-expression+data+sets+to+select+lung+adenocarcinoma+marker+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Joint+analysis+of+two+microarray+gene-expression+data+sets+to+select+lung+adenocarcinoma+marker+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001047-Teschendorff1" id="pone.0001047-Teschendorff1"></a>Teschendorff AE, Naderi A, Barbosa-Morais NL, Pinder SE, Ellis IO, et al.  (2006) A consensus prognostic gene expression classifier for er positive breast cancer. Genome Biology  7: R101.  <ul class="find" data-citedArticleID="1006224" data-doi="10.1186/gb-2006-7-10-r101"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-10-r101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+consensus+prognostic+gene+expression+classifier+for+er+positive+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+consensus+prognostic+gene+expression+classifier+for+er+positive+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001047-Kim1" id="pone.0001047-Kim1"></a>Kim R, Park P (2004) Improving identification of differentially expressed genes in microarray studies using information from public databases. Genome Biology  5: R70.  <ul class="find" data-citedArticleID="1006200"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Improving+identification+of+differentially+expressed+genes+in+microarray+studies+using+information+from+public+databases.&amp;auth=&amp;atitle=Improving+identification+of+differentially+expressed+genes+in+microarray+studies+using+information+from+public+databases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+identification+of+differentially+expressed+genes+in+microarray+studies+using+information+from+public+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+identification+of+differentially+expressed+genes+in+microarray+studies+using+information+from+public+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001047-Segal1" id="pone.0001047-Segal1"></a>Segal E, Friedman N, Koller D, Regev D (2004) A module map showing conditional activity of expression modules in cancer. Nat. Genet.  36: 1090–8.  <ul class="find" data-citedArticleID="1006214" data-doi="10.1038/ng1434"><li><a href="http://dx.doi.org/10.1038/ng1434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+module+map+showing+conditional+activity+of+expression+modules+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+module+map+showing+conditional+activity+of+expression+modules+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001047-Tanay1" id="pone.0001047-Tanay1"></a>Tanay A, Steinfeld I, Kupiec M, Shamir R (2005) Integrative analysis of genome-wide experiments in the context of a large high-throughput data compendium. Molecular Systems Biology  1:   <ul class="find" data-citedArticleID="1006222" data-doi="10.1038/msb4100005"><li><a href="http://dx.doi.org/10.1038/msb4100005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrative+analysis+of+genome-wide+experiments+in+the+context+of+a+large+high-throughput+data+compendium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrative+analysis+of+genome-wide+experiments+in+the+context+of+a+large+high-throughput+data+compendium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001047-Perou1" id="pone.0001047-Perou1"></a>Perou C, Jeffrey S, van de Rijn M, Rees C, Eisen M, et al.  (1999) Distinctive gene expression patterns in human mammary epithelial cells and breast cancers. PNAS  96: 9212–7.  <ul class="find" data-citedArticleID="1006206" data-doi="10.1073/pnas.96.16.9212"><li><a href="http://dx.doi.org/10.1073/pnas.96.16.9212" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinctive+gene+expression+patterns+in+human+mammary+epithelial+cells+and+breast+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinctive+gene+expression+patterns+in+human+mammary+epithelial+cells+and+breast+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001047-Perou2" id="pone.0001047-Perou2"></a>Perou C, Sorlie T, Eisen M, van de Rijn M, Jeffrey S, et al.  (2000) Molecular portraits of human breast tumours. Nature  (406): 747–52.  <ul class="find" data-citedArticleID="1006208" data-doi="10.1038/35021093"><li><a href="http://dx.doi.org/10.1038/35021093" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+portraits+of+human+breast+tumours." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+portraits+of+human+breast+tumours.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001047-Sorlie2" id="pone.0001047-Sorlie2"></a>Sorlie T, Perou C, Tibshirani R, Aas T, Geisler S, et al.  (2001) Gene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. PNAS  98: 10869–74.  <ul class="find" data-citedArticleID="1006220" data-doi="10.1073/pnas.191367098"><li><a href="http://dx.doi.org/10.1073/pnas.191367098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+patterns+of+breast+carcinomas+distinguish+tumor+subclasses+with+clinical+implications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+patterns+of+breast+carcinomas+distinguish+tumor+subclasses+with+clinical+implications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001047-Bair1" id="pone.0001047-Bair1"></a>Bair E, Tibshirani R (2004) Semi-supervised methods to predict patient survival from gene expression data. Plos Biology  2: e108.  <ul class="find" data-citedArticleID="1006186" data-doi="10.1371/journal.pbio.0020108"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Semi-supervised+methods+to+predict+patient+survival+from+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Semi-supervised+methods+to+predict+patient+survival+from+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001047-Benjamini1" id="pone.0001047-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate: a practical and powerful approach to multiple testing. Journal of the Royal Statistical Society, Series B  57: 289–300.  <ul class="find" data-citedArticleID="1006188"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing.&amp;auth=&amp;atitle=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+false+discovery+rate%3A+a+practical+and+powerful+approach+to+multiple+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001047-Wessels1" id="pone.0001047-Wessels1"></a>Wessels L, Reinders M, Hart A, Veenman C, Dai H, et al.  (2005) A protocol for building and evaluating predictors of disease state based on microarray data. Bioinformatics  21: 3755–62.  <ul class="find" data-citedArticleID="1006232" data-doi="10.1093/bioinformatics/bti429"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti429" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protocol+for+building+and+evaluating+predictors+of+disease+state+based+on+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protocol+for+building+and+evaluating+predictors+of+disease+state+based+on+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001047-Domingos1" id="pone.0001047-Domingos1"></a>Domingos P, Pazzani M (1996) Beyond independence: conditions for the optimality of the simple bayesian classifier. ICML.   <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001047-Hanahan1" id="pone.0001047-Hanahan1"></a>Hanahan D, Weinberg RA (2000) The hallmarks of cancer. Cell  (1): 57–70.  <ul class="find" data-citedArticleID="1006194" data-doi="10.1016/s0092-8674(00)81683-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81683-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hallmarks+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hallmarks+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001047-Bonnet1" id="pone.0001047-Bonnet1"></a>Bonnet S, Archer S, AllalunisTurner J, Haromy A, Beaulieu C, et al.  (2007) A mitochondria k channel axis is suppressed in cancer and its normalization promotes apoptosis and inhibits cancer growth. Cancer Cell  100: 37–51.  <ul class="find" data-citedArticleID="1006190" data-doi="10.1016/j.ccr.2006.10.020"><li><a href="http://dx.doi.org/10.1016/j.ccr.2006.10.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+mitochondria+k+channel+axis+is+suppressed+in+cancer+and+its+normalization+promotes+apoptosis+and+inhibits+cancer+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+mitochondria+k+channel+axis+is+suppressed+in+cancer+and+its+normalization+promotes+apoptosis+and+inhibits+cancer+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001047-Jin1" id="pone.0001047-Jin1"></a>Jin T, Branch D, Zhang X, Qi S, Youngson B, et al.  (1999) Examination of POU homeobox gene expression in human breast cancer cells. International Journal of Cancer  81: 104–112.  <ul class="find" data-citedArticleID="1006198" data-doi="10.1002/(sici)1097-0215(19990331)81:1<104::aid-ijc18&gt;3.0.co;2-q"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0215(19990331)81:1<104::aid-ijc18&gt;3.0.co;2-q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Examination+of+POU+homeobox+gene+expression+in+human+breast+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Examination+of+POU+homeobox+gene+expression+in+human+breast+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001047-Malin1" id="pone.0001047-Malin1"></a>Malin S, Linderson Y, Almqvist J, Ernberg I, Tallone T, et al.  (2005) DNA dependent conversion of Oct1 and Oct2 into transcriptional repressors by Groucho/TLE. Nucl. Acids Res.  33: 4618–4625.  <ul class="find" data-citedArticleID="1006204" data-doi="10.1093/nar/gki744"><li><a href="http://dx.doi.org/10.1093/nar/gki744" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+dependent+conversion+of+Oct1+and+Oct2+into+transcriptional+repressors+by+Groucho%2FTLE." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+dependent+conversion+of+Oct1+and+Oct2+into+transcriptional+repressors+by+Groucho%2FTLE.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001047-MSigDB1" id="pone.0001047-MSigDB1"></a>  MSigDB (Accessed July 2005). v1.0 <a href="http://www.broad.mit.edu/gsea/">http://www.broad.mit.edu/gsea/</a>. <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001047-Reactome1" id="pone.0001047-Reactome1"></a>  Reactome (Accessed July 2005). Cold spring harbor laboratory, European bioinformatics institute, and GO consortium <a href="http://www.reactome.org">http://www.reactome.org</a>.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.XML" value="87808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.PDF" value="419041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g001.PNG_L" value="69591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g001.PNG_M" value="43237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g001.PNG_S" value="6468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g001.TIF" value="422692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g001.PNG_I" value="15595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g002.PNG_L" value="75182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g002.PNG_M" value="129518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g002.PNG_S" value="14081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g002.TIF" value="308364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g002.PNG_I" value="76950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.t001.PNG_L" value="265006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.t001.PNG_M" value="153753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.t001.PNG_S" value="22871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.t001.TIF" value="1182508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.t001.PNG_I" value="169293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g003.PNG_L" value="33775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g003.PNG_M" value="75114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g003.PNG_S" value="9062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g003.TIF" value="166938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g003.PNG_I" value="27387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g004.PNG_L" value="71147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g004.PNG_M" value="96603"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g004.PNG_S" value="7621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g004.TIF" value="357624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g004.PNG_I" value="33913"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g005.PNG_L" value="44652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g005.PNG_M" value="25207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g005.PNG_S" value="6233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g005.TIF" value="115026"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g005.PNG_I" value="9184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g006.PNG_L" value="84864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g006.PNG_M" value="53608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g006.PNG_S" value="7667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g006.TIF" value="297612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g006.PNG_I" value="18813"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g007.PNG_L" value="149998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g007.PNG_M" value="98709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g007.PNG_S" value="10709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g007.TIF" value="482328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g007.PNG_I" value="34842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g008.PNG_L" value="126345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g008.PNG_M" value="215422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g008.PNG_S" value="12131"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g008.TIF" value="812042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g008.PNG_I" value="75854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g009.PNG_L" value="227865"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g009.PNG_M" value="219298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g009.PNG_S" value="12313"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g009.TIF" value="1539834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.g009.PNG_I" value="76436"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s001.DOC" value="109056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s002.EPS" value="1158455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s003.EPS" value="587763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s004.EPS" value="806051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s005.EPS" value="4120459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s006.XLS" value="5687296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s007.DOC" value="31744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001047.s008.DOC" value="31744"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001047&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001047" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001047&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001047&volume=&issue=&title=Module-Based Outcome Prediction Using Breast Cancer Compendia&author_name=Martin%20H.%20van%20Vliet%2C%20Christiaan%20N.%20Klijn%2C%20Lodewyk%20F.%20A.%20Wessels%2C%20Marcel%20J.%20T.%20Reinders&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001047" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047&amp;t=Module-Based%20Outcome%20Prediction%20Using%20Breast%20Cancer%20Compendia" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047&title=Module-Based%20Outcome%20Prediction%20Using%20Breast%20Cancer%20Compendia&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047&amp;title=Module-Based%20Outcome%20Prediction%20Using%20Breast%20Cancer%20Compendia" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001047&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Module-Based Outcome Prediction Using Breast Cancer Compendia';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001047';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Module-Based%20Outcome%20Prediction%20Using%20Breast%20Cancer%20Compendia http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001047" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001047" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Breast+cancer%22" title="Search for articles in the subject area:'Breast cancer'"><div class="flagText">Breast cancer</div></a>
              <div data-categoryid="18763" data-articleid="24928"
                   data-categoryname="Breast cancer"
                   class="flagImage" title="Flag 'Breast cancer' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="24928"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="24928"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="24928"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+performance%22" title="Search for articles in the subject area:'Human performance'"><div class="flagText">Human performance</div></a>
              <div data-categoryid="18145" data-articleid="24928"
                   data-categoryname="Human performance"
                   class="flagImage" title="Flag 'Human performance' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Metastasis%22" title="Search for articles in the subject area:'Metastasis'"><div class="flagText">Metastasis</div></a>
              <div data-categoryid="20021" data-articleid="24928"
                   data-categoryname="Metastasis"
                   class="flagImage" title="Flag 'Metastasis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="24928"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="24928"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=968'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3364'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4511&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
