

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001838"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001838" />

    <meta name="citation_title" content="A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism"/>
    <meta itemprop="name" content="A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism"/>

      <meta name="citation_author" content="Daniel J. Kliebenstein"/>
            <meta name="citation_author_institution" content="Department of Plant Sciences, University of California Davis, Davis, California, United States of America"/>

    <meta name="citation_date" content="2008/3/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001838.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1838"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The origins of genomic duplications in Arabidopsis.; citation_author=TJ Vision; citation_author=DG Brown; citation_author=SD Tanksley; citation_journal_title=Science; citation_volume=290; citation_number=1; citation_pages=2114-2117; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A recent polyploidy superimposed on older large-scale duplications in the Arabidopsis genome.; citation_author=G Blanc; citation_author=K Hokamp; citation_author=KH Wolfe; citation_journal_title=Genome Research; citation_volume=13; citation_number=2; citation_pages=137-144; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.; citation_author=KH Wolfe; citation_author=DC Shields; citation_journal_title=Nature; citation_volume=387; citation_number=3; citation_pages=708-713; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes.; citation_author=G Blanc; citation_author=KH Wolfe; citation_journal_title=Plant Cell; citation_volume=16; citation_number=4; citation_pages=1667-1678; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Striking similarities in the genomic distribution of tandemly arrayed genes in Arabidopsis and rice.; citation_author=C Rizzon; citation_author=L Ponger; citation_author=BS Gaut; citation_journal_title=Plos Computational Biology; citation_volume=2; citation_number=5; citation_pages=989-1000; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=How Often Do Duplicated Genes Evolve New Functions.; citation_author=JB Walsh; citation_journal_title=Genetics; citation_volume=139; citation_number=6; citation_pages=421-428; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes.; citation_author=M Lynch; citation_author=JS Conery; citation_journal_title=Science; citation_volume=290; citation_number=7; citation_pages=1151-1155; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Divergence in expression between duplicated genes in Arabidopsis.; citation_author=EW Ganko; citation_author=BC Meyers; citation_author=TJ Vision; citation_journal_title=Molecular Biology and Evolution: doi:10.1093/molbev/msm1158; citation_number=8; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nonrandom divergence of gene expression following gene and genome duplications in the flowering plant Arabidopsis thaliana.; citation_author=T Casneuf; citation_author=S De Bodt; citation_author=J Raes; citation_author=S Maere; citation_author=Y Van de Peer; citation_journal_title=Genome Biology; citation_volume=7; citation_number=9; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=External factors accelerate expression divergence between duplicate genes.; citation_author=M Ha; citation_author=WH Li; citation_author=ZJ Chen; citation_journal_title=Trends in Genetics; citation_volume=23; citation_number=10; citation_pages=162-166; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Expression divergence between duplicate genes.; citation_author=WH Li; citation_author=J Yang; citation_author=X Gu; citation_journal_title=Trends in Genetics; citation_volume=21; citation_number=11; citation_pages=602-607; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression pattern shifts following duplication indicative of subfunctionalization and neofunctionalization in regulatory genes of Arabidopsis.; citation_author=JM Duarte; citation_author=LY Cui; citation_author=PK Wall; citation_author=Q Zhang; citation_author=XH Zhang; citation_journal_title=Molecular Biology and Evolution; citation_volume=23; citation_number=12; citation_pages=469-478; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Duplicate genes increase gene expression diversity within and between species.; citation_author=ZL Gu; citation_author=SA Rifkin; citation_author=KP White; citation_author=WH Li; citation_journal_title=Nature Genetics; citation_volume=36; citation_number=13; citation_pages=577-579; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of an Arabidopsis insect resistance quantitative trait locus.; citation_author=J Kroymann; citation_author=S Donnerhacke; citation_author=D Schnabelrauch; citation_author=T Mitchell-Olds; citation_journal_title=Proceedings Of The National Academy Of Sciences Of The United States Of America; citation_volume=100; citation_number=14; citation_pages=14587-14592; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Gene duplication and the diversification of secondary metabolism: side chain modification of glucosinolates in Arabidopsis thaliana.; citation_author=D Kliebenstein; citation_author=V Lambrix; citation_author=M Reichelt; citation_author=J Gershenzon; citation_author=T Mitchell-Olds; citation_journal_title=Plant Cell; citation_volume=13; citation_number=15; citation_pages=681-693; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Modeling gene and genome duplications in eukaryotes.; citation_author=S Maere; citation_author=S De Bodt; citation_author=J Raes; citation_author=T Casneuf; citation_author=M Van Montagu; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=16; citation_pages=5454-5459; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Variation and constraint in plant evolution and development.; citation_author=S Kalisz; citation_author=EM Kramer; citation_journal_title=Heredity; citation_number=17; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Constraint.; citation_author=K Schwenk; citation_author=GP Wagner; citation_number=18; citation_pages=52-60; citation_date=2003; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_title=Gene Ontology: tool for the unification of biology.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nature Genetics; citation_volume=25; citation_number=19; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=MetaCyc and AraCyc. Metabolic pathway databases for plant research.; citation_author=PF Zhang; citation_author=H Foerster; citation_author=CP Tissier; citation_author=L Mueller; citation_author=S Paley; citation_journal_title=Plant Physiology; citation_volume=138; citation_number=20; citation_pages=27-37; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=AraCyc: A biochemical pathway database for Arabidopsis.; citation_author=LA Mueller; citation_author=PF Zhang; citation_author=SY Rhee; citation_journal_title=Plant Physiology; citation_volume=132; citation_number=21; citation_pages=453-460; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Secondary metabolites and plant/environment interactions: a view through Arabidopsis thaliana tinged glasses.; citation_author=DJ Kliebenstein; citation_journal_title=Plant Cell Environ; citation_volume=27; citation_number=22; citation_pages=675-684; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The secondary metabolism of Arabidopsis thaliana: growing like a weed.; citation_author=JC D'Auria; citation_author=J Gershenzon; citation_journal_title=Current Opinion in Plant Biology; citation_volume=8; citation_number=23; citation_pages=308-316; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Secondary Metabolism in Arabidopsis.; citation_author=CCS Chapple; citation_author=BW Shirley; citation_author=M Zook; citation_author=R Hammerschmidt; citation_author=SC Somerville; citation_number=24; citation_pages=989-1030; citation_date=1994; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Seeing double: gene duplication and diversification in plant secondary metabolism.; citation_author=D Ober; citation_journal_title=Trends in Plant Science; citation_volume=10; citation_number=25; citation_pages=444-449; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of the indole alkaloid biosynthesis in the genus Hordeum: Distribution of gramine and DIBOA and isolation of the benzoxazinoid biosynthesis genes from Hordeum lechleri.; citation_author=S Grun; citation_author=M Frey; citation_author=A Gierl; citation_journal_title=Phytochemistry; citation_volume=66; citation_number=26; citation_pages=1264-1272; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Positive selection driving diversification in plant secondary metabolism.; citation_author=M Benderoth; citation_author=S Textor; citation_author=AJ Windsor; citation_author=T Mitchell-Olds; citation_author=J Gershenzon; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=103; citation_number=27; citation_pages=9118-9123; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biology and biochemistry of glucosinolates.; citation_author=BA Halkier; citation_author=J Gershenzon; citation_journal_title=Annual Review of Plant Biology; citation_volume=57; citation_number=28; citation_pages=303-333; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Specialist and generalist herbivores exert opposing selection on a chemical defense; citation_author=RA Lankau; citation_journal_title=New Phytologist; citation_volume=175; citation_number=29; citation_pages=176-184; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mutual Feedbacks Maintain Both Genetic and Species Diversity in a Plant Community; citation_author=RA Lankau; citation_author=SY Strauss; citation_journal_title=Science; citation_volume=317; citation_number=30; citation_pages=1561-1563; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic survey of gene expression diversity in Arabidopsis thaliana; citation_author=DJ Kliebenstein; citation_author=MAL West; citation_author=H Van Leeuwen; citation_author=K Kyunga; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=172; citation_number=31; citation_pages=1179-1189; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Diversity of flowering responses in wild Arabidopsis thaliana strains.; citation_author=J Lempe; citation_author=S Balasubramanian; citation_author=S Sureshkumar; citation_author=A Singh; citation_author=M Schmid; citation_journal_title=Plos Genetics; citation_volume=1; citation_number=32; citation_pages=109-118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A gene expression map of Arabidopsis thaliana development.; citation_author=M Schmid; citation_author=TS Davison; citation_author=SR Henz; citation_author=UJ Pape; citation_author=M Demar; citation_journal_title=Nature Genetics; citation_volume=37; citation_number=33; citation_pages=501-506; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The AtGenExpress global stress expression data set: protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses.; citation_author=J Kilian; citation_author=D Whitehead; citation_author=J Horak; citation_author=D Wanke; citation_author=S Weinl; citation_journal_title=Plant Journal; citation_volume=50; citation_number=34; citation_pages=347-363; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Control of stochasticity in eukaryotic gene expression.; citation_author=JM Raser; citation_author=EK O'Shea; citation_journal_title=Science; citation_volume=304; citation_number=35; citation_pages=1811-1814; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stochastic gene expression in a single cell.; citation_author=MB Elowitz; citation_author=AJ Levine; citation_author=ED Siggia; citation_author=PS Swain; citation_journal_title=Science; citation_volume=297; citation_number=36; citation_pages=1183-1186; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The probability of duplicate gene preservation by subfunctionalization. .; citation_author=M Lynch; citation_author=A Force; citation_journal_title=Genetics; citation_volume=154; citation_number=37; citation_pages=459-473; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Common sequence polymorphisms shaping genetic diversity in Arabidopsis thaliana.; citation_author=RM Clark; citation_author=G Schweikert; citation_author=C Toomajian; citation_author=S Ossowski; citation_author=G Zeller; citation_journal_title=Science; citation_volume=317; citation_number=38; citation_pages=338-342; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Global eQTL Mapping Reveals the Complex Genetic Architecture of Transcript Level Variation in Arabidopsis.; citation_author=MAL West; citation_author=K Kim; citation_author=DJ Kliebenstein; citation_author=H van Leeuwen; citation_author=RW Michelmore; citation_journal_title=Genetics; citation_volume=175; citation_number=39; citation_pages=1441-1450; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Natural Variation among Arabidopsis thaliana Accessions for Transcriptome Response to Exogenous Salicylic Acid.; citation_author=H Van Leeuwen; citation_author=DJ Kliebenstein; citation_author=MAL West; citation_author=KD kim; citation_author=R van Poecke; citation_journal_title=Plant Cell; citation_volume=19; citation_number=40; citation_pages=2099-2110; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Large-scale identification of single-feature polymorphisms in complex genomes.; citation_author=JO Borevitz; citation_author=D Liang; citation_author=D Plouffe; citation_author=HS Chang; citation_author=T Zhu; citation_journal_title=Genome Research; citation_volume=13; citation_number=41; citation_pages=513-523; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=COI1: An Arabidopsis gene required for jasmonate-regulated defense and fertility.; citation_author=DX Xie; citation_author=BF Feys; citation_author=S James; citation_author=M Nieto-Rostro; citation_author=JG Turner; citation_journal_title=Science; citation_volume=280; citation_number=42; citation_pages=1091-1094; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Coi1-Dependent Expression of an Arabidopsis Vegetative Storage Protein in Flowers and Siliques and in Response to Coronatine or Methyl Jasmonate.; citation_author=CE Benedetti; citation_author=DX Xie; citation_author=JG Turner; citation_journal_title=Plant Physiology; citation_volume=109; citation_number=43; citation_pages=567-572; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Recent advances in ABA signaling.; citation_author=SY Kim; citation_journal_title=Journal of Plant Biology; citation_volume=50; citation_number=44; citation_pages=117-121; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Drought and salt tolerance in plants.; citation_author=D Bartels; citation_author=R Sunkar; citation_journal_title=Critical Reviews in Plant Sciences; citation_volume=24; citation_number=45; citation_pages=23-58; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sugar-induced signal transduction in plants.; citation_author=S Smeekens; citation_journal_title=Annual Review of Plant Physiology and Plant Molecular Biology; citation_volume=51; citation_number=46; citation_pages=49-81; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genetic Architecture of Plastic Methyl Jasmonate Responses in Arabidopsis thaliana.; citation_author=DJ Kliebenstein; citation_author=A Figuth; citation_author=T Mitchell-Olds; citation_journal_title=Genetics; citation_volume=161; citation_number=47; citation_pages=1685-1696; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis PAD3, a gene required for camalexin biosynthesis, encodes a putative cytochrome P450 monooxygenase.; citation_author=N Zhou; citation_author=TL Tootle; citation_author=J Glazebrook; citation_journal_title=Plant Cell; citation_volume=11; citation_number=48; citation_pages=2419-2428; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Variation of glucosinolate accumulation among different organs and developmental stages of Arabidopsis thaliana.; citation_author=PD Brown; citation_author=JG Tokuhisa; citation_author=M Reichelt; citation_author=J Gershenzon; citation_journal_title=Phytochem; citation_volume=62; citation_number=49; citation_pages=471-781; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Modulation of CYP79 genes and glucosinolate profiles in Arabidopsis by defense signaling pathways.; citation_author=MD Mikkelsen; citation_author=BL Petersen; citation_author=E Glawischnig; citation_author=AB Jensen; citation_author=E Andreasson; citation_journal_title=Plant Physiology; citation_volume=131; citation_number=50; citation_pages=298-308; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Secondary metabolites influence Arabidopsis/Botrytis interactions: variation in host production and pathogen sensitivity.; citation_author=DJ Kliebenstein; citation_author=HC Rowe; citation_author=KJ Denby; citation_journal_title=Plant Journal; citation_volume=44; citation_number=51; citation_pages=25-36; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide patterns of single-feature polymorphism in Arabidopsis thaliana.; citation_author=JO Borevitz; citation_author=SP Hazen; citation_author=TP Michael; citation_author=GP Morris; citation_author=IR Baxter; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=104; citation_number=52; citation_pages=12057-12062; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis epithiospecifier protein promotes the hydrolysis of glucosinolates to nitriles and influences Trichoplusia ni herbivory.; citation_author=V Lambrix; citation_author=M Reichelt; citation_author=T Mitchell-Olds; citation_author=D Kliebenstein; citation_author=J Gershenzon; citation_journal_title=Plant Cell; citation_volume=13; citation_number=53; citation_pages=2793-2807; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Gene Controlling the Quantitative Trait Locus EPITHIOSPECIFIER MODIFIER1 Alters Glucosinolate Hydrolysis and Insect Resistance in Arabidopsis.; citation_author=Z-Y Zhang; citation_author=JA Ober; citation_author=DJ Kliebenstein; citation_journal_title=Plant Cell; citation_volume=18; citation_number=54; citation_pages=1524-1536; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Study of the role of antimicrobial glucosinolate-derived isothiocyanates in resistance of Arabidopsis to microbial pathogens.; citation_author=K-J Tierens; citation_author=B Thomma; citation_author=M Brower; citation_author=J Schmidt; citation_author=K Kistner; citation_journal_title=Plant Physiol; citation_volume=125; citation_number=55; citation_pages=1688-1699; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic control of natural variation in Arabidopsis thaliana glucosinolate accumulation.; citation_author=DJ Kliebenstein; citation_author=J Kroymann; citation_author=P Brown; citation_author=A Figuth; citation_author=D Pedersen; citation_journal_title=Plant Physiol; citation_volume=126; citation_number=56; citation_pages=811-825; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Comparative quantitative trait loci mapping of aliphatic, indolic and benzylic glucosinolate production in Arabidopsis thaliana leaves and seeds.; citation_author=DJ Kliebenstein; citation_author=J Gershenzon; citation_author=T Mitchell-Olds; citation_journal_title=Genetics; citation_volume=159; citation_number=57; citation_pages=359-370; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=MAM3 catalyzes the formation of all aliphatic glucosinolate chain lengths in Arabidopsis.; citation_author=S Textor; citation_author=JW de Kraker; citation_author=BGJ Hause; citation_author=JG Tokuhisa; citation_journal_title=Plant Physiol; citation_volume=144; citation_number=58; citation_pages=60-71; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Identification of a flavin-monooxygenase as the S-oxygenating enzyme in aliphatic glucosinolate biosynthesis in Arabidopsis.; citation_author=BG Hansen; citation_author=DJ Kliebenstein; citation_author=BA Halkier; citation_journal_title=The Plant Journal; citation_volume=50; citation_number=59; citation_pages=902-910; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=CYP79F1 and CYP79F2 have distinct functions in the biosynthesis of aliphatic glucosinolates in Arabidopsis.; citation_author=SX Chen; citation_author=E Glawischnig; citation_author=K Jorgensen; citation_author=P Naur; citation_author=B Jorgensen; citation_journal_title=Plant Journal; citation_volume=33; citation_number=60; citation_pages=923-937; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytochrome P450CYP79F1 from Arabidopsis catalyzes the conversion of dihomomethionine and trihomomethionine to the corresponding aldoximes in the biosynthesis of aliphatic glucosinolates.; citation_author=CH Hansen; citation_author=U Wittstock; citation_author=CE Olsen; citation_author=AJ Hick; citation_author=JA Pickett; citation_journal_title=Journal of Biological Chemistry; citation_volume=276; citation_number=61; citation_pages=11078-11085; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effect of Day-Length and Gibberellic Acid on the Flowering of Arabidopsis.; citation_author=J Langridge; citation_journal_title=Nature; citation_volume=180; citation_number=62; citation_pages=36-37; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=The gibberellic acid biosynthesis mutant ga1-3 of Arabidopsis thaliana is responsive to vernalization.; citation_author=SD Michaels; citation_author=RM Amasino; citation_journal_title=Developmental Genetics; citation_volume=25; citation_number=63; citation_pages=194-198; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Auxin promotes Arabidopsis root growth by modulating gibberellin response.; citation_author=XD Fu; citation_author=NP Harberd; citation_journal_title=Nature; citation_volume=421; citation_number=64; citation_pages=740-743; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Fitness Effects Associated with the Major Flowering Time Gene FRIGIDA in Arabidopsis thaliana in the Field American Naturalist; citation_author=TM Korves; citation_author=KJ Schmid; citation_author=AL Caicedo; citation_author=C Mays; citation_author=JR Stinchcombe; citation_volume=169; citation_number=65; citation_pages=E141-E157; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Antagonistic pleiotropic effects reduce the potential adaptive value of the FRIGIDA locus PNAS 10.1073/pnas.0708209104; citation_author=N Scarcelli; citation_author=JM Cheverud; citation_author=BA Schaal; citation_author=PX Kover; citation_number=66; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The molecular genetic basis of plant adaptation.; citation_author=IM Ehrenreich; citation_author=MD Purugganan; citation_journal_title=American Journal of Botany; citation_volume=93; citation_number=67; citation_pages=953-962; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events.; citation_author=JE Bowers; citation_author=BA Chapman; citation_author=JK Rong; citation_author=AH Paterson; citation_journal_title=Nature; citation_volume=422; citation_number=68; citation_pages=433-438; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A plant miRNA contributes to antibacterial resistance by repressing auxin signaling.; citation_author=L Navarro; citation_author=P Dunoyer; citation_author=F Jay; citation_author=B Arnold; citation_author=N Dharmasiri; citation_journal_title=Science; citation_volume=312; citation_number=69; citation_pages=436-439; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biosynthesis of methionine-derived glucosinolates in Arabidopsis thaliana: recombinant expression and characterization of methylthioalkylmalate synthase, the condensing enzyme of the chain-elongation cycle.; citation_author=S Textor; citation_author=S Bartram; citation_author=J Kroymann; citation_author=KL Falk; citation_author=A Hick; citation_journal_title=Planta; citation_volume=218; citation_number=70; citation_pages=1026-1035; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genetic variation of aliphatic glucosinolates in Arabidopsis thaliana and prospects for map based gene cloning.; citation_author=R Mithen; citation_author=H Campos; citation_journal_title=Entomologia Experimentalis et Applicata; citation_volume=80; citation_number=71; citation_pages=202-205; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Biochemical genetics of aliphatic glucosinolates in Brassica and Arabidopsis.; citation_author=R Mithen; citation_author=D Toroser; citation_journal_title=Seminar series; citation_volume=56 (Society for experimental biology); citation_number=72; citation_pages=261-275; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis glucosyltransferase UGT74B1 functions in glucosinolate biosynthesis and auxin homeostasis.; citation_author=CD Grubb; citation_author=BJ Zipp; citation_author=J Ludwig-Muller; citation_author=MN Masuno; citation_author=TF Molinski; citation_journal_title=Plant Journal; citation_volume=40; citation_number=73; citation_pages=893-908; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=BRANCHED-CHAIN AMINOTRANSFERASE4 is part of the chain elongation pathway in the biosynthesis of methionine-derived glucosinolates in Arabidopsis.; citation_author=J Schuster; citation_author=T Knill; citation_author=M Reichelt; citation_author=J Gershenzon; citation_author=S Binder; citation_journal_title=Plant Cell; citation_volume=18; citation_number=74; citation_pages=2664-2679; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Linking Metabolic QTL with Network and cis-eQTL Controlling Biosynthetic Pathways.; citation_author=AM Wentzell; citation_author=HC Rowe; citation_author=BG Hansen; citation_author=C Ticconi; citation_author=BA Halkier; citation_journal_title=PLOS Genetics; citation_volume=3; citation_number=75; citation_pages=e162; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Costs of resistance to natural enemies in field populations of the annual plant Arabidopsis thaliana.; citation_author=R Mauricio; citation_journal_title=American Naturalist; citation_volume=151; citation_number=76; citation_pages=20-28; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Experimental manipulation of putative selective agents provides evidence for the role of natural enemies in the evolution of plant defense.; citation_author=R Mauricio; citation_author=MD Rausher; citation_journal_title=Evolution; citation_volume=51; citation_number=77; citation_pages=1435-1444; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Venomous cone snails: molecular phylogeny and the generation of toxin diversity.; citation_author=DJD Espiritu; citation_author=M Watkins; citation_author=V Dia-Monje; citation_author=GE Cartier; citation_author=LJ Cruz; citation_journal_title=Toxicon; citation_volume=39; citation_number=78; citation_pages=1899-1916; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pronounced intraspecific haplotype divergence at the RPP5 complex disease resistance locus of Arabidopsis.; citation_author=L Noel; citation_author=TL Moores; citation_author=EA van der Biezen; citation_author=M Parniske; citation_author=MJ Daniels; citation_journal_title=Plant Cell; citation_volume=11; citation_number=79; citation_pages=2099-2111; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis Cytochrome P450 Monooxygenase 71A13 Catalyzes the Conversion of Indole-3-Acetaldoxime in Camalexin Synthesis.; citation_author=M Nafisi; citation_author=S Goregaoker; citation_author=CJ Botanga; citation_author=E Glawischnig; citation_author=CE Olsen; citation_journal_title=Plant Cell; citation_volume=19; citation_number=80; citation_pages=2039-2052; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis myrosinases TGG1 and TGG2 have redundant function in glucosinolate breakdown and insect defense.; citation_author=C Barth; citation_author=G Jander; citation_journal_title=Plant Journal; citation_volume=46; citation_number=81; citation_pages=549-562; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The third myrosinase gene TGG3 in Arabidopsis thaliana is a pseudogene specifically expressed in stamen and petal.; citation_author=J Zhang; citation_author=B Pontoppidan; citation_author=J Xue; citation_author=L Rask; citation_author=J Meijer; citation_journal_title=Physiologia Plantarum; citation_volume=115; citation_number=82; citation_pages=25-34; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative biochemical characterization of nitrile-forming proteins from plants and insects that alter myrosinase-catalysed hydrolysis of glucosinolates.; citation_author=M Burow; citation_author=J Markert; citation_author=J Gershenzon; citation_author=U Wittstock; citation_journal_title=Febs Journal; citation_volume=273; citation_number=83; citation_pages=2432-2446; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CYP83B1 is the oxime-metabolizing enzyme in the glucosinolate pathway in Arabidopsis.; citation_author=CH Hansen; citation_author=LC Du; citation_author=P Naur; citation_author=CE Olsen; citation_author=KB Axelsen; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=84; citation_pages=24790-24796; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The three desulfoglucosinolate sulfotransferase proteins in Arabidopsis have different substrate specificities and are differentially expressed.; citation_author=M Klein; citation_author=M Reichelt; citation_author=J Gershenzon; citation_author=J Papenbrock; citation_journal_title=Febs Journal; citation_volume=273; citation_number=85; citation_pages=122-136; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis ref2 mutant is defective in the gene encoding CYP83A1 and shows both phenylpropanoid and glucosinolate phenotypes.; citation_author=MR Hemm; citation_author=MO Ruegger; citation_author=C Chapple; citation_journal_title=Plant Cell; citation_volume=15; citation_number=86; citation_pages=179-194; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Arabidopsis mutants in the C-S lyase of glucosinolate biosynthesis establish a critical role for indole-3-acetaldoxime in auxin homeostasis.; citation_author=MD Mikkelsen; citation_author=P Naur; citation_author=BA Halkier; citation_journal_title=Plant Journal; citation_volume=37; citation_number=87; citation_pages=770-777; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism"/>
    <meta name="twitter:description" content="BackgroundMost eukaryotic genomes have undergone whole genome duplications during their evolutionary history. Recent studies have shown that the function of these duplicated genes can diverge from the ancestral gene via neo- or sub-functionalization within single genotypes. An additional possibility is that gene duplicates may also undergo partitioning of function among different genotypes of a species leading to genetic differentiation. Finally, the ability of gene duplicates to diverge may be limited by their biological function.Methodology/Principal FindingsTo test these hypotheses, I estimated the impact of gene duplication and metabolic function upon intraspecific gene expression variation of segmental and tandem duplicated genes within Arabidopsis thaliana. In all instances, the younger tandem duplicated genes showed higher intraspecific gene expression variation than the average Arabidopsis gene. Surprisingly, the older segmental duplicates also showed evidence of elevated intraspecific gene expression variation albeit typically lower than for the tandem duplicates. The specific biological function of the gene as defined by metabolic pathway also modulated the level of intraspecific gene expression variation. The major energy metabolism and biosynthetic pathways showed decreased variation, suggesting that they are constrained in their ability to accumulate gene expression variation. In contrast, a major herbivory defense pathway showed significantly elevated intraspecific variation suggesting that it may be under pressure to maintain and/or generate diversity in response to fluctuating insect herbivory pressures.ConclusionThese data show that intraspecific variation in gene expression is facilitated by an interaction of gene duplication and biological activity. Further, this plays a role in controlling diversity of plant metabolism."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001838.g007"/>

  <meta property="og:title" content="A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5714'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2743'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7830&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001838">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001838" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001838&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel J. Kliebenstein
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:Kliebenstein@ucdavis.edu">Kliebenstein@ucdavis.edu</a></p>

                <p>Affiliation:
                  Department of Plant Sciences, University of California Davis, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 19, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001838</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001838-t001" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-t002" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g001" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g002" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g003" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g004" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g005" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g006" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001838-g007" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001838">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001838" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Most eukaryotic genomes have undergone whole genome duplications during their evolutionary history. Recent studies have shown that the function of these duplicated genes can diverge from the ancestral gene via neo- or sub-functionalization within single genotypes. An additional possibility is that gene duplicates may also undergo partitioning of function among different genotypes of a species leading to genetic differentiation. Finally, the ability of gene duplicates to diverge may be limited by their biological function.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To test these hypotheses, I estimated the impact of gene duplication and metabolic function upon intraspecific gene expression variation of segmental and tandem duplicated genes within <em>Arabidopsis thaliana</em>. In all instances, the younger tandem duplicated genes showed higher intraspecific gene expression variation than the average Arabidopsis gene. Surprisingly, the older segmental duplicates also showed evidence of elevated intraspecific gene expression variation albeit typically lower than for the tandem duplicates. The specific biological function of the gene as defined by metabolic pathway also modulated the level of intraspecific gene expression variation. The major energy metabolism and biosynthetic pathways showed decreased variation, suggesting that they are constrained in their ability to accumulate gene expression variation. In contrast, a major herbivory defense pathway showed significantly elevated intraspecific variation suggesting that it may be under pressure to maintain and/or generate diversity in response to fluctuating insect herbivory pressures.</p>


<h3>Conclusion</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>These data show that intraspecific variation in gene expression is facilitated by an interaction of gene duplication and biological activity. Further, this plays a role in controlling diversity of plant metabolism.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kliebenstein DJ (2008) A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism. PLoS ONE 3(3):
          e1838.
            doi:10.1371/journal.pone.0001838</p><p><strong>Academic Editor: </strong>Geraldine Butler, University College Dublin, Ireland</p><p><strong>Received:</strong> December 21, 2007; <strong>Accepted:</strong> February 19, 2008; <strong>Published:</strong> March 19, 2008</p><p><strong>Copyright:</strong> © 2008 Daniel Kliebenstein. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding for this project was obtained by NSF grant DBI#0642481 to DJK.</p><p><strong>Competing interests:</strong> The author has declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Most eukaryotic genomes have undergone whole genome duplications during their evolutionary history with angiosperms having a particular enrichment in this process <a href="#pone.0001838-Vision1">[1]</a>–<a href="#pone.0001838-Blanc2">[4]</a>. In addition to whole genome events, small local events can generate tandem duplicated genes which are often considered younger than segmental duplicated genes obtained from a whole genome duplication <a href="#pone.0001838-Vision1">[1]</a>, <a href="#pone.0001838-Rizzon1">[5]</a>. Following either tandem or segmental duplication events, one of the duplicated genes can be rendered non-functional via the accumulation of deleterious mutations <a href="#pone.0001838-Walsh1">[6]</a>. Alternatively, both duplicated genes can be maintained if the presence of both copies is advantageous. Over time, the function of these duplicated genes can diverge from the ancestral gene or from each other. This divergence can occur via sub-functionalization such that the duplicate copies obtain differential expression patterns in terms of tissue specificity or stress response <a href="#pone.0001838-Lynch1">[7]</a>. Alternatively, one of the duplicates can obtain a novel function, a process known as neo-functionalization. In a number of species, recent work has associated gene duplication with divergent gene expression patterns in response to developmental or abiotic stress cues <a href="#pone.0001838-Ganko1">[8]</a>–<a href="#pone.0001838-Duarte1">[12]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Gene duplication may also allow partitioning of function among genotypes within a species, leading to genetic differentiation/intraspecific variation. This process is contingent upon genetic neo-functionalization and would suggest that duplicated genes should show higher levels of intraspecific variation in gene expression than “unique” genes. Preliminary evidence supporting the hypothesis that duplicated genes show higher levels of intraspecific variation was obtained in a study with yeast and fruit flies <a href="#pone.0001838-Gu1">[13]</a>. Given that segmental duplications are typically older than speciation events, it is likely that segmentally duplicated genes have fixed different functions and are in essence acting as unique genes. As such, it would be expected that increases in intraspecific variation may only be detected among younger tandem duplicated genes. In <em>Arabidopsis thaliana</em>, several cloned QTL associate intraspecific variation in gene expression with tandem duplication <a href="#pone.0001838-Kroymann1">[14]</a>, <a href="#pone.0001838-Kliebenstein1">[15]</a>. Yet to date there is little information on the genomic role of gene duplication in controlling intraspecific genetic variation in gene expression or other traits.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Another factor controlling the divergence and maintenance of gene duplicates is the biological function of the gene <a href="#pone.0001838-Rizzon1">[5]</a>, <a href="#pone.0001838-Casneuf1">[9]</a>, <a href="#pone.0001838-Maere1">[16]</a>. It is commonly held that some essential biological processes, such as primary energy metabolism, may be constrained in their ability to vary <a href="#pone.0001838-Kalisz1">[17]</a>, <a href="#pone.0001838-Schwenk1">[18]</a>. However, current studies typically use overarching functional assignments such as “nucleotide binding” or “hydrolase activity” provided by the Gene Ontology consortium. While providing a broad genomic perspective, these categories may obscure the influence of more specific biological function on genetic constraint <a href="#pone.0001838-Ashburner1">[19]</a> suggesting the need for a more specific biological definition.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A rich source for testing the relationship between biological function, gene duplication and intraspecific genetic variation are databases describing the metabolic pathways known and predicted within a host of organisms <a href="#pone.0001838-Zhang1">[20]</a>, <a href="#pone.0001838-Mueller1">[21]</a>. These databases provide specific metabolic roles for genes acting as a highly refined function prediction. For instance, the Arabidopsis metabolic networks includes genes involved in major energetic and carbon flux pathways that are essential for each cells survival, such as photosynthesis, the calvin cycle, and the TCA cycle (AraCyc; <a href="http://www.arabidopsis.org/biocyc/index.jsp">www.arabidopsis.org/biocyc/index.jsp</a>). The critical function of the genes in these primary metabolic pathways may be constrained in the range of gene expression values that the pathways transcripts can occupy and the more specific definition of their biological activity may enhance our ability to detect this.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In contrast to primary metabolism, genes involved in secondary metabolite pathways are marked by a high level of inter and intraspecific variation <a href="#pone.0001838-Kliebenstein2">[22]</a>–<a href="#pone.0001838-Chapple1">[24]</a>. The evolution of this diversity is predicted to be driven by gene duplication and consequent neo-functionalization of enzymes <a href="#pone.0001838-Kliebenstein1">[15]</a>, <a href="#pone.0001838-Ober1">[25]</a>–<a href="#pone.0001838-Benderoth1">[27]</a>. Glucosinolates are sulphur-rich, amino acid-derived compounds that form a major class of secondary metabolites in Arabidopsis. These compounds are hydrolyzed by an endogenous thioglucosidase, myrosinase, releasing toxic products upon disruption of plant cells by harvesting, processing, or mastication <a href="#pone.0001838-Halkier1">[28]</a>. Aliphatic glucosinolates are subject to diverse selection regimes in the wild <a href="#pone.0001838-Benderoth1">[27]</a>, <a href="#pone.0001838-Lankau1">[29]</a>, <a href="#pone.0001838-Lankau2">[30]</a>. We hypothesize that such secondary metabolite pathways may have higher levels of gene duplication and greater gene expression diversity than primary metabolite enzymes. Thus, Arabidopsis metabolism provides an excellent model to study how gene duplication and biological function interplay to control constraint, sub-functionalization and/or neo-functionalization following gene duplication.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In this study, I investigated the role that gene duplication and metabolic pathway organization play in controlling gene expression variation within <em>Arabidopsis thaliana</em> natural accessions. I also measured gene expression variation in developmental and abiotic stress datasets to allow comparison with other publications focused on gene duplication and divergence of response patterns. Recent tandem and older segmental duplicated genes showed a significant increase in intraspecific variation of gene expression in comparison the average gene, while unique genes showed lower levels of intraspecific variation. Gene expression variation of segmental and tandem duplicated genes was controlled by a greater number of genetic loci, and these loci had larger effects on gene expression variation for tandem duplicated genes. Gene participation in specific metabolic pathways predicted the level of intraspecific variation in gene expression, with major energy and amino acid pathways having relatively lower levels of gene expression variation. This suggests that these primary metabolism pathways are genetically constrained. In contrast, a major secondary metabolite pathway in Arabidopsis showed a significantly increased level of gene duplication and intraspecific gene expression variation, suggesting that this pathway may be structured to cope with fluctuating selection in the wild. As such, the influence of gene duplication on intraspecific gene expression variation partly depends upon the biological function of the gene involved.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Duplicated Genes have more variable transcript accumulation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To understand the role that gene duplication and gene expression diversity may play in controlling metabolic pathways, I analyzed transcript accumulation for the complete genome using four Affymetrix ATH1 microarray datasets. The first two datasets are two independent replicated experiments focused on natural genetic variation across multiple Arabidopsis accessions [Accessions I; <a href="#pone.0001838-Kliebenstein3">[31]</a> and Accessions II;<a href="#pone.0001838-Lempe1">[32]</a>]. These provide a measure of intraspecific gene expression variation. The additional datasets are replicated experiments measuring alteration of the transcriptome through response variation within a single genotype [Development; <a href="#pone.0001838-Schmid1">[33]</a>) and by abiotic stress (Abiotic; <a href="#pone.0001838-Kilian1">[34]</a>]. These provide a comparison with other studies focused on gene duplication and the evolution of divergent gene expression patterns in response to development or abiotic stress in a single genotype <a href="#pone.0001838-Rizzon1">[5]</a>, <a href="#pone.0001838-Casneuf1">[9]</a>, <a href="#pone.0001838-Maere1">[16]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>For each replicated experiment within a dataset, the mean transcript value per experiment was calculated. To compare transcriptomic responses between datasets, two related indicators of transcript variance were estimated per transcript per dataset, the variance mean ratio (VMR) and the coefficient-of-variance (CV) <a href="#pone.0001838-Raser1">[35]</a>, <a href="#pone.0001838-Elowitz1">[36]</a>. Given that the CV and VMR were obtained using the mean transcript level per experiment, they act as estimates of transcript variation in response to perturbations in development, abiotic stress or natural genetic variation rather then simply a measure of experimental variation. To control for errors introduced from cross-hybridization, only genes with unique probes sets were utilized for the analysis. All probe-sets annotated as recognizing multiple genes were removed. Both the VMR and CV showed that the average Arabidopsis transcript was most sensitive to developmental perturbation (<a href="#pone-0001838-t001">Tables 1</a> and <a href="#pone-0001838-t002">2</a>). Interestingly, both studies querying genetic variation showed similar levels of transcript variance, albeit lower levels than calculated for the response datasets (<a href="#pone-0001838-t001">Tables 1</a> and <a href="#pone-0001838-t002">2</a>).</p>
<div class="figure" id="pone-0001838-t001"><div class="img"><a name="pone-0001838-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span><strong>VMR for genes with different duplication states.</strong> Shown is the average VMR for the whole genome, unique genes, segmental duplicated genes and tandem duplicated genes.</span></strong></p><span>doi:10.1371/journal.pone.0001838.t001</span></div><div class="figure" id="pone-0001838-t002"><div class="img"><a name="pone-0001838-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span><strong>CV for genes with different duplication states.</strong> Shown is the average CV for the whole genome, unique genes, segmental duplicated genes and tandem duplicated genes.</span></strong></p><span>doi:10.1371/journal.pone.0001838.t002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Gene duplication has been associated with enabling neo- and sub-functionalization whereby duplicate genes may diverge <a href="#pone.0001838-Lynch1">[7]</a>, <a href="#pone.0001838-Ganko1">[8]</a>, <a href="#pone.0001838-Duarte1">[12]</a>, <a href="#pone.0001838-Lynch2">[37]</a>. To test if gene duplication is associated with increased transcript variation, I used a previous study that classified genes within the Arabidopsis genome as unique, segmental duplicate or tandem duplicate <a href="#pone.0001838-Rizzon1">[5]</a>. In this study, segmental duplicated genes contained both transpositional duplicates and polyploidy orthologs. In all four datasets, transcripts associated with tandem duplicated genes showed dramatically increased variance across perturbations and genotypes in comparison to the whole genome (<a href="#pone-0001838-g001">Figure 1</a>). Additionally, segmental duplicated genes showed elevated variation for three of the four datasets and unique genes had lower than expected variance across the response and natural genetic variation datasets (<a href="#pone-0001838-g001">Figure 1</a>). This shows that tandem duplications allow for increased variation in gene expression within Arabidopsis. At least one of two datasets suggests that the same is true for segmental duplications (<a href="#pone-0001838-g001">Figure 1</a>). As such, subdivision of gene function across duplicates may manifest as differential expression patterns across multiple genotypes or as differential expression patterns within a single genotype.</p>
<div class="figure" id="pone-0001838-g001"><div class="img"><a name="pone-0001838-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Gene duplication status alters transcript variance.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Genes within each of four independent microarray datasets were grouped as unique (pink), segmental duplicated (dark blue) and/or tandem duplicated (red) and the average of two measures of transcript variance was determined across all genes within these groups as well as for the whole genome (light blue). Within each dataset, letters distinguish gene groups that differ significantly at <em>P</em>&lt;0.001 as determined by bootstrapping. Within each dataset, variance estimates were standardized to the whole genomic average (set to 100%) indicated by the horizontal line. VMR: variance mean ratio. CV: the coefficient-of-variance. A. Average VMR for the transcripts. B. Average CV for the transcripts.</p>
<span>doi:10.1371/journal.pone.0001838.g001</span></div><a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Previous studies have shown that gene duplication or polymorphism occur disproportionately within specific gene families and Gene Ontology (GO) classes <a href="#pone.0001838-Rizzon1">[5]</a>, <a href="#pone.0001838-Ashburner1">[19]</a>, <a href="#pone.0001838-Clark1">[38]</a>. As expected, all four datasets showed increased transcript variation for specific GO terms associated with elevated gene duplication, such as ‘Response to abiotic and biotic stimulus’ <a href="#pone.0001838-Rizzon1">[5]</a>, as well as for gene families showing increased sequence polymorphism, e.g. Cytochromes P450, NBS-LRR and F-box genes <a href="#pone.0001838-Clark1">[38]</a> (Kliebenstein, unpublished data). However, neither gene family nor GO term is a precise measure of biological function.</p>


<h4>Comparison of Accession Variation with Abiotic Variation</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To directly compare the level of transcript variance due to natural genetic variation versus an abiotic treatment, I obtained a factorial dataset in which transcript accumulation between two Arabidopsis accessions, Bay-0 and Shahdara, was compared in the presence or absence of exogenous salicylic acid in two replicate experiments <a href="#pone.0001838-West1">[39]</a>. We calculated the total variance for each transcript across all microarrays and estimated the percent of per transcript variance that was due to the main experimental variables (accession, treatment and replicate) as well as their interaction terms. This showed that differences between the two accessions was a greater source of gene expression variance than treatment effect (<a href="#pone-0001838-g002">Figure 2</a>)<a href="#pone.0001838-VanLeeuwen1">[40]</a>. This was not due to polymorphisms that impact hybridization as there was no difference in genetic variance estimates for genes with or without a detected SFP as previously observed with these datasets <a href="#pone.0001838-Kliebenstein3">[31]</a>, <a href="#pone.0001838-West1">[39]</a>, <a href="#pone.0001838-Borevitz1">[41]</a>. In this analysis, both segmental and tandem duplicate genes had a significant enhancement in per transcript variance due to accession when compared to the average transcript (<a href="#pone-0001838-g002">Figure 2</a>). Conversely, unique genes had a diminished level of transcript variance. Interestingly, only segmental duplicate genes had an altered level of transcript variance in response to the treatment variation (<a href="#pone-0001838-g002">Figure 2</a>). Tandem and segmental duplicate genes did not have elevated transcript variance due to the replicate suggesting that tandem or segmental duplicated genes are not noisier than unique genes (<a href="#pone-0001838-g002">Figure 2</a>). This analysis further suggests that both segmental and tandem duplicate genes have more intraspecific gene expression variation than the average gene in the Arabidopsis genome.</p>
<div class="figure" id="pone-0001838-g002"><div class="img"><a name="pone-0001838-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Partitioning of transcript variance in a factorial analysis of genetic and abiotic variation.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Individual transcripts' variance components were partitioned via ANOVA within a replicated factorial experiment comparing natural genetic variation with an abiotic stress treatment. The average percent of total variation per individual transcript is shown for unique genes (pink), the whole genome average (light blue), segmental duplicated genes (dark blue) and tandem duplicated genes (red). Within a factor, letters show significantly different duplication classes.</p>
<span>doi:10.1371/journal.pone.0001838.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To provide another measure of intraspecific gene expression variation, I obtained a large replicated dataset investigating quantitative trait loci controlling gene expression (eQTL) between the Bay-0 and Sha accessions. If gene duplication allows for an increase in variation between accessions, then duplicated genes should have a higher frequency of genetic polymorphisms controlling their transcript abundance. In agreement with this prediction, duplicated genes had on average more total eQTL per transcript than the unique genes or the whole genome average (<a href="#pone-0001838-g003">Figure 3A</a>). This was partially due to duplicated genes, both tandem and segmental, more frequently having <em>cis-</em>eQTLs (eQTLs located at their physical positions) (<a href="#pone-0001838-g003">Figure 3B</a>). Interestingly, eQTLs detected for tandem duplicate genes have larger allelic effects than eQTLs for segmental duplicate genes (<a href="#pone-0001838-g003">Figure 3C and D</a>). This suggests that duplicated genes are less constrained in their ability to accumulate genetic polymorphisms influencing transcript accumulation than the average Arabidopsis gene.</p>
<div class="figure" id="pone-0001838-g003"><div class="img"><a name="pone-0001838-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Impact of gene duplication on eQTL analysis.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>An analysis of eQTL detected within 211 replicated Bay-0×Sha recombinant inbred lines was used to address the impact of gene duplication on genetic control of transcript variation. For all graphs, pink shows unique genes, light blue is the genome average, dark blue are the segmental duplicated genes and red is the tandem duplicated genes. A. Average number of <em>cis</em>-eQTL per transcript per duplication class. Letters show duplication classes with statistically significant differences. B. Average number of <em>cis</em>-eQTL per transcript per duplication class. Letters show duplication classes with statistically significant differences. C. QQ plot comparing distributions of the average eQTL R<sup>2</sup> for each transcript among duplication classes. The X-axis shows the average eQTL R<sup>2</sup> for transcripts every percentile within the whole genome. The Y-axis shows the average eQTL R<sup>2</sup> for transcripts every percentile within the comparison transcript set. D. QQ plot comparing distributions of the maximum (max) eQTL R<sup>2</sup> for each transcript among duplication classes. The X-axis shows the max eQTL R<sup>2</sup> for transcripts every percentile within the whole genome. The Y-axis shows the max eQTL R<sup>2</sup> for transcripts every percentile within the comparison transcript set.</p>
<span>doi:10.1371/journal.pone.0001838.g003</span></div>

<h4>Metabolic Consequences of Transcript Variation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Gene duplication is thought to provide much of the genetic material allowing plants to generate a vast diversity of metabolites. To test how gene duplication and associated gene expression variation may shape Arabidopsis metabolism, I focused on transcripts associated with specific metabolic pathways using the AraCyc databases predictions <a href="#pone.0001838-Zhang1">[20]</a>, <a href="#pone.0001838-Mueller1">[21]</a>. I then measured the average transcript variance (CV and VMR) across the transcripts within a metabolic pathway to develop a pathway estimate of variance and utilized bootstrapping to generate an empirical distribution of transcript variance. This empirical distribution was used to test for a difference between the specific metabolic pathways CV and the expected CV for a similarly-sized group of genes randomly drawn from the genome.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To test the utility of pathway CV estimates for interpreting metabolic pathway response to experimental variation, I investigated the response datasets. This identified a significant positive correlation between metabolic pathway transcript CVs in response to developmental and abiotic perturbations (P &lt;0.001, R<sup>2</sup> = 0.43 <a href="#pone-0001838-g004">Figure 4</a>). This is not unexpected given that both datasets utilize a single Arabidopsis genotype, Col-0, and similar signaling pathways are involved controlling gene expression in response to development and abiotic stress. The major energy conversion and amino acid biosynthetic pathways (e.g. TCA cycle, aerobic respiration, etc) showed significantly diminished pathway transcript CV in both datasets as would be expected for these essential metabolic pathways (<a href="#pone-0001838-g004">Figure 4</a>). Interestingly, pathways for two hormones responsible for controlling differential plant development and abiotic stress responses, abscisic acid and jasmonic acid, showed a significant increase in transcript CV within both datasets (<a href="#pone-0001838-g004">Figure 4</a>)<a href="#pone.0001838-Xie1">[42]</a>–<a href="#pone.0001838-Smeekens1">[46]</a>. The Arabidopsis secondary metabolite pathways further support the relationship between pathway transcript CV and biological function. A pathogen inducible compound, camalexin, only shows a significantly enhanced transcript CV in the Abiotic dataset. In contrast, the more developmentally controlled glucosinolate pathways show significantly enhanced transcript CV in only the Developmental dataset (<a href="#pone-0001838-g004">Figure 4</a>) <a href="#pone.0001838-Kliebenstein4">[47]</a>–<a href="#pone.0001838-Kliebenstein5">[51]</a>. Thus, the pathway level transcript CV estimate identifies predicted metabolic pathway responses within the response datasets.</p>
<div class="figure" id="pone-0001838-g004"><div class="img"><a name="pone-0001838-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of average metabolic pathway CVs between development and abiotic stress datasets.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The average CV per metabolic pathway for the Development and Abiotic microarray studies are plotted with the significance of each spot within the two studies represented by both color and shape: Pink squares = pathways with a significant deviation from the whole genome in both studies; red diamonds = deviation from the whole genome in Abiotic only; dark blue diamonds = deviation from the whole genome in Developmental only; light blue circles = no observed deviation from the whole genome average. The inset expands the lower left part of the graph. Significantly differing pathways are annotated to indicate biosynthesis (‘bio’) or degradation (‘deg’). GLS stands for glucosinolate. FAE stands for fatty acid elongation. CN stands for cyanate.</p>
<span>doi:10.1371/journal.pone.0001838.g004</span></div>

<h4>Metabolic Pathways and Natural Genetic Variation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>I next utilized the pathway level transcript CV to compare intraspecific variation measured in the two Accession datasets. A low pathway level transcript CV would suggest genetic constraints limiting gene expression diversity of pathway members. In contrast, elevated levels of pathway transcript CV may result from selection for increased gene expression diversity. The availability of two independent ATH1 microarray datasets investigating natural genetic variation in transcript accumulation within Arabidopsis allows a replicated analysis of metabolic pathways to detect biased transcript variance. Major energy and amino acid pathways showed significantly diminished pathway transcript variance in comparison to the average random gene set in both independent accession datasets (<a href="#pone-0001838-g005">Figure 5</a>) suggesting that transcript variance in these essential biochemical pathways is genetically constrained. The photosynthesis and calvin cycle pathways do show significant pathway variance within the development dataset displaying the potential of these pathways to vary (<a href="#pone-0001838-g004">Figures 4</a> and <a href="#pone-0001838-g005">5</a>). This increased genetic constraint in comparison to the average metabolic pathway may relate to a relative lack of gene duplicates in these pathways. For instance, the tRNA charging pathways have a paucity of gene duplicates in comparison to the random expectation (<a href="#pone-0001838-g006">Figure 6</a> and <a href="#pone.0001838.s004">Table S4</a>). Overall, the number of tandem duplicates within a pathway was positively correlated to the pathway's average transcript CV within the Accession datasets (P&lt;0.001, R<sup>2</sup> = 0.21, N  = 135; for this test, the Aliphatic Glucosinolate Biosynthetic pathway was removed given its high CV). This suggests that gene duplication status for the different metabolic pathways can predict the level of genetic variation for gene expression within a given metabolic pathway. Interestingly, none of the pathways with either elevated or diminished pathway level transcript CV showed significantly different estimates of sequence polymorphism in comparison to randomly generated pathways (<a href="#pone.0001838.s005">Table S5</a>)<a href="#pone.0001838-Borevitz2">[52]</a>.</p>
<div class="figure" id="pone-0001838-g005"><div class="img"><a name="pone-0001838-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Comparison of average metabolic pathway CV between two independent genetic variation studies.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The average CV per metabolic pathway for Accession I and Accession II are plotted with the significance of each spot within the two studies represented by both color and shape: black diamonds = pathways with a significant deviation from the whole genome in both datasets; red triangles = deviation from the whole genome in only Accession I; pink squares = deviation from the whole genome in only Accession II; blue circles = no observed deviation from the whole genome average. Significantly differing pathways are annotated to indicate biosynthesis (‘bio’) or degradation (‘deg’). Amino acid pathways are represented by their capitalized three letter code. GLS stands for glucosinolate.</p>
<span>doi:10.1371/journal.pone.0001838.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In contrast to the constrained major energy metabolism pathways, the aliphatic glucosinolate biosynthetic pathway showed significantly elevated pathway level transcript CV in both accession experiments suggesting that the transcripts in this pathway may be under selection for increased diversity. This secondary metabolite pathway is a major insect and pathogen defense pathway within Arabidopsis <a href="#pone.0001838-Lambrix1">[53]</a>–<a href="#pone.0001838-Tierens1">[55]</a>. In agreement with the increased transcript level variation, significant genetic variation for both content and structure of aliphatic glucosinolates is potentially influenced by diverse selective pressures <a href="#pone.0001838-Benderoth1">[27]</a>, <a href="#pone.0001838-Kliebenstein6">[56]</a>, <a href="#pone.0001838-Kliebenstein7">[57]</a>. While it is possible that this increased pathway level transcript CV could be caused by natural genetic variation in one or two transcription factors, accessions did not significantly covary for the transcripts in the aliphatic glucosinolate biosynthesis pathway (<a href="#pone-0001838-g007">Figure 7</a>). Interestingly, transcript CV increases along the pathway and is highest for transcripts determining structural variation in the final aliphatic glucosinolate, e.g. MAMs, CYP79Fs, FMOs and AOPs <a href="#pone.0001838-Kroymann1">[14]</a>, <a href="#pone.0001838-Kliebenstein1">[15]</a>, <a href="#pone.0001838-Textor1">[58]</a>–<a href="#pone.0001838-Hansen2">[61]</a>. The increased transcript CV for the aliphatic glucosinolate pathway is associated with significantly elevated levels of gene duplication for the genes within this pathway (<a href="#pone-0001838-g005">Figure 5</a> and <a href="#pone-0001838-g006">6</a>).</p>
<div class="figure" id="pone-0001838-g006"><div class="img"><a name="pone-0001838-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Biased frequency of gene duplication in metabolic pathways.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>The level of gene duplication within metabolic pathways that show biased gene duplication frequency is presented. χ<sup>2</sup> analysis was used to test duplication frequency within each metabolic pathway for deviation from the whole genome expectation. * and blue bars show pathways that have a statistically significant bias in gene duplication after adjusting for multiple comparisons at an FDR of 0.05. The bars are separated into biochemical groups (left to right: secondary metabolism, amino acid related, energy related, nucleotide related and hormone related). A. Total duplicated genes per metabolic pathway. B. Tandem duplicated genes per metabolic pathway.</p>
<span>doi:10.1371/journal.pone.0001838.g006</span></div><div class="figure" id="pone-0001838-g007"><div class="img"><a name="pone-0001838-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001838" data-uri="info:doi/10.1371/journal.pone.0001838.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001838.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001838.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001838.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001838.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Variation of transcripts within the aliphatic glucosinolate biosynthetic pathway.</span></strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>The variation of each aliphatic glucosinolate biosynthetic transcript across seven Arabidopsis accessions from the Accession I dataset is plotted. Accumulation of each transcript within each accession was standardized to the average transcript level across the accessions. Accession coding is shown in the legend. Genes for the different transcripts are ordered from first to last (left to right) biosynthetic step. Arrows group genes for different biosynthetic steps. ‘Duplication’ indicates whether each gene was classified as being a tandem (T), segmental (S) or unique (U) gene <a href="#pone.0001838-Rizzon1">[5]</a>. ‘Activity’ shows whether each gene has a unique (U), redundant (R), semi-redundant (SR) or unvalidated (?) activity. Activity was defined via analysis of published literature such that if a loss of the gene abolished the biosynthetic reaction it was classified as unique, if the loss partially abolished the reaction it was classified as semi-redundant, and if gene loss had no effect on the reaction it was deemed redundant. A question mark means that gene loss has not yet been evaluated. As there is only one probeset on the ATH1 microarray for <em>CYP79F1</em> and <em>CYP79F2</em>, they are jointly annotated as CYP79F in this study.</p>
<span>doi:10.1371/journal.pone.0001838.g007</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Several metabolic pathways showed differential levels of transcript variation between the two Accession datasets. This is most striking for the ent-kaurene biosynthetic pathway that has significantly elevated pathway transcript variance in Accession I but significantly diminished transcript variance in Accession II (<a href="#pone-0001838-g005">Figure 5</a>). Ent-kaurene is a precursor for gibberellin biosynthesis, whose pathways show a similar pattern of variation (<a href="#pone-0001838-g005">Figure 5</a>). Gibberellic acid is an important developmental regulator that is also believed to play a role in controlling flowering time under short day conditions <a href="#pone.0001838-Langridge1">[62]</a>–<a href="#pone.0001838-Fu1">[64]</a>. The plants for the Accession I dataset were grown under short-days and harvested just prior to flowering onset for some of the accessions. In contrast, the plants for the Accession II dataset were grown under long-days and harvested long before flowering. As such, the Accession I dataset may identify genetic variance in gibberellic acid associated with flowering time, a trait for which variation is likely under selection <a href="#pone.0001838-Korves1">[65]</a>–<a href="#pone.0001838-Ehrenreich1">[67]</a>. In contrast, the experimental conditions for Accession II may have accentuated the constrained developmental roles of gibberellic acid in seedlings, decreasing detection of genetic variation in transcript accumulation. Future work will be required to better understand the role of genotype × environment interactions in controlling the measurement of genetic variation in transcript accumulation.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Tandem Gene Duplication and Genetic Constraint</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>This study explores how gene duplication and metabolic function interact to control intraspecific variation in gene expression. Tandem duplicated genes in <em>Arabidopsis thaliana</em> show increased levels of intraspecific gene expression in all tested datasets (<a href="#pone-0001838-g001">Figures 1</a>–<a href="#pone-0001838-g002"></a><a href="#pone-0001838-g003">3</a> and <a href="#pone-0001838-t001">Tables 1</a> and <a href="#pone-0001838-t002">2</a>). This includes two different samples of Arabidopsis accessions, a detailed comparison of abiotic and genetic variation within two accessions, and the frequency of transcript accumulation polymorphisms that cause eQTL. Tandem duplications are likely younger than segmentally duplicated chromosome regions, which were probably generated during whole genome duplication events approximately 20–40 million years ago <a href="#pone.0001838-Vision1">[1]</a>, <a href="#pone.0001838-Blanc1">[2]</a>, <a href="#pone.0001838-Bowers1">[68]</a>. As such, the function of segmental duplicated genes is more likely to have been fixed than the tandem genes. In agreement with this hypothesis, the two Accession datasets showed that tandem duplications showed significantly higher intraspecific gene expression variation than the segmental duplicated genes (<a href="#pone-0001838-g001">Figure 1</a> and <a href="#pone-0001838-g002">2</a>). However, the segmental duplicates showed enhanced intraspecific gene expression variation in comparison to unique genes in both accession datasets as well as in the eQTL analysis. While the number of eQTLs did not differentiate between the duplication classes, the distribution of effect sizes showed that tandem duplicates typically had larger effect eQTLs (<a href="#pone-0001838-g003">Figure 3</a>). Interestingly, both duplication classes had more <em>cis</em> and <em>trans</em>-eQTLs than the average gene. The increased level of trans-eQTLs suggests that duplicated genes are involved in less constrained pathways than unique genes, but this remains to be further investigated. Overall, these data suggests that both tandem and segmental duplicate genes have greater potential to facilitate the generation of intraspecific variation than unique genes, but the tandem genes make a greater contribution.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>While we eliminated entire probe-set cross-hybridization by not analyzing these probe sets, any probe level cross-hybridization between duplicated genes may remain. However, if a probe hybridizes to two or more transcripts its effect will be limited in two ways. First, the other 10 probes in a probe set will still provide an accurate estimate. Secondly, any probe that binds two or more transcripts will be averaging across transcripts and likely lead to decreased estimates of variance. Previous work with the eQTL population or accession data suggested that any relationship between probe level variation and transcript level variation while significant was minimal in comparison to the probe sets variation <a href="#pone.0001838-Kliebenstein3">[31]</a>, <a href="#pone.0001838-West1">[39]</a>. Together, this suggests that individual probes are not likely a significant contribution to our variables and that our estimates are conservative.</p>


<h4>Metabolic Pathways and Gene Expression Constraint</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Essential metabolic pathways required for the production and movement of energy within the plant showed significantly lower levels of gene expression variation across development, abiotic stress and natural genetic variation (<a href="#pone-0001838-g004">Figures 4</a> and <a href="#pone-0001838-g005">5</a>). This suggests that gene expression for these pathways is constrained across a range of external, internal and genetic stimuli. This constraint agrees with the essentiality of these pathways, e.g. aerobic respiration and the TCA cycle, to an individual cells survival. Only one of these less variable pathways (tRNA charging) had a lower than average level of gene duplication, suggesting that the constraints on these pathways are not associated with altered duplication patterns. The two exceptions to the observation that major energy metabolic pathways are constrained are the calvin cycle and photosynthesis pathways. These pathways showed significant genetic constraint but no statistical support for developmental constraints most likely due to the difference in photosynthetic capacity between roots and shoots. The observed genetic constraints for these two pathways are not due to a technical issue limiting the ability to accurately estimate CV (<a href="#pone-0001838-g005">Figure 5</a>).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Development and abiotic stress are sometimes considered a comparison of internal versus external cues regulating gene expression variation. Interestingly, our analysis of metabolic pathways identified a significant positive correlation in pathway variation between the development and abiotic stress datasets (<a href="#pone-0001838-g004">Figure 4</a>). This positive relationship between development and abiotic stress suggests an association between gene expression responses to external and internal cues. This association is supported by a host of observations wherein plant responses to external factors are regulated via plant hormones that also control internal developmental cues <a href="#pone.0001838-Xie1">[42]</a>–<a href="#pone.0001838-Smeekens1">[46]</a>, <a href="#pone.0001838-Navarro1">[69]</a>. Exploring pleiotropic effects of similar signaling compounds being used for both responses to internal (development) and external (abiotic stress) cues upon gene expression divergence will be an interesting avenue for future research.</p>


<h4>The Aliphatic Glucosinolate Biosynthetic Pathway and Sub-functionalization versus Neo-functionalization</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Within the aliphatic glucosinolate biosynthetic pathway, increased transcript variation across naturally variable Arabidopsis accessions is associated with tandemly duplicated enzymes. This tempts the conclusion that these gene families are undergoing genetic sub-functionalization. However, biochemical analysis of these gene families shows that they have already undergone neo-functionalization, creating new biosynthetic activities in the duplicate enzymes. Examples include <em>MAM1</em> v <em>MAM3</em> <a href="#pone.0001838-Textor1">[58]</a>, <a href="#pone.0001838-Textor2">[70]</a>, <em>CYP79F1</em> v <em>CYP79F2</em> <a href="#pone.0001838-Chen1">[60]</a> and <em>AOP2</em> v <em>AOP3</em> <a href="#pone.0001838-Kliebenstein1">[15]</a>. As such, variance in gene expression leads to dramatic variation in glucosinolate structure and serves as evidence of genetic and biochemical neo-functionalization <a href="#pone.0001838-Kliebenstein6">[56]</a>, <a href="#pone.0001838-Mithen1">[71]</a>, <a href="#pone.0001838-Mithen2">[72]</a>. Even for those enzymatic steps where the tandem duplicates appear to have similar biochemical properties, manipulation of individual genes has a measurable consequence for the resulting glucosinolate output <a href="#pone.0001838-Hansen1">[59]</a>, <a href="#pone.0001838-Grubb1">[73]</a>, <a href="#pone.0001838-Schuster1">[74]</a>. Visible phenotypic changes associated with changes in expression of these loci indicate that these loci are not completely redundant. As such, the aliphatic glucosinolate biosynthetic pathway illustrates the difficulty in genomic/bioinformatic analysis of neo- versus sub-functionalization. Expression analysis would argue that sub-functionalization of the tandem duplicates has occurred, but biochemistry reveals neo-functionalization of the tandem duplicates.</p>


<h4>The Aliphatic Glucosinolate Biosynthetic Pathway and Intraspecific Variation</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Pathway level transcript variation for the aliphatic glucosinolate biosynthetic pathway was a dramatic outlier in both accession datasets (<a href="#pone-0001838-g005">Figure 5</a> and <a href="#pone.0001838.s003">Table S3</a>). Gene expression variation across the different accessions was distributed such that the enzymatic steps responsible for determining the final chemical structure had elevated gene expression variation, suggesting that variation in this pathway may be selected to generate structural diversity (<a href="#pone-0001838-g007">Figure 7</a>). However two of these enzymatic loci responsible for structural diversity, <em>MAM1/3</em> and <em>AOP2</em>, also pleiotropically regulate gene expression for the whole biosynthetic pathway, thus complicating our ability to separate the effects of content versus structure in this experiment <a href="#pone.0001838-Wentzell1">[75]</a>. A potential explanation for increased genetic diversity in the aliphatic glucosinolate biosynthetic pathway may be that gene expression diversity enables response to fluctuations in natural insect populations between different generations. Aliphatic glucosinolates provide a positive fitness benefit in the presence of generalist herbivores but can impart both a cost of production in the absence of herbivores and a fitness cost in the presence of specialist herbivores <a href="#pone.0001838-Lankau1">[29]</a>, <a href="#pone.0001838-Mauricio1">[76]</a>, <a href="#pone.0001838-Mauricio2">[77]</a>. As such, natural fluctuations in insect herbivore populations would lead to fluctuating selection pressures on aliphatic glucosinolates within Arabidopsis. Thus, Arabidopsis may respond to these unpredictable fluctuations via standing genetic variation in this pathway <a href="#pone.0001838-Kliebenstein6">[56]</a>. Similarly, evidence for the contribution of gene duplication and intraspecific variation to defense mechanisms has been found in studies of plant gene-for-gene resistance and cone snail toxin <a href="#pone.0001838-Espiritu1">[78]</a>, <a href="#pone.0001838-Noel1">[79]</a>.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>This work shows that the availability of gene duplications and the function of biochemical pathways interact to influence gene expression variation across a diverse array of biological and genetic factors. As such, I suggest a model in which gene duplication provides raw material for evolution but, at least in metabolic pathways, the biology of the pathway determines the likelihood that duplicated genes are maintained and evolve altered functionality. In secondary metabolic pathways where variation is required, duplicates will be maintained and evolve diverse expression patterns. In primary metabolic pathways where variation may be detrimental, this may actually select against the presence of duplicated genes and variable expression patterns. In the absence of functional biochemical evidence, it is difficult to classify gene expression changes between tandem duplicates as true sub-functionalization versus neo-functionalization coupled with intraspecific gene expression variation. Future tests may reveal whether the variation in metabolic pathways showing elevated transcript CV is an essential component of Arabidopsis fitness.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Microarray data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Four previously published datasets encompassing a large number of transcriptomic analysis utilizing Arabidopsis Affymetrix ATH1 microarrays were used for this experiment. One dataset, hereafter referred to as Development, contains replicated analysis of transcript variation across a variety of developmental stages and tissue types from <em>Arabidopsis thaliana</em> <a href="#pone.0001838-Schmid1">[33]</a>. A second dataset, hereafter annotated as Abiotic, contains replicated analysis of <em>Arabidopsis thaliana</em> transcriptional response to multiple abiotic stresses <a href="#pone.0001838-Kilian1">[34]</a>. Two independent datasets are from replicated experiments querying natural variation in gene expression across Arabidopsis accessions. These are annotated as Accession I, containing seven replicated accessions, Col-0, Cvi-1, Est, Kin-0, Mt-0, Tsu-1 and Van-0 <a href="#pone.0001838-Kliebenstein3">[31]</a>, and Accession II containing ten replicated accessions Bay-0, C24, Col-0, Cvi-1, Est, Kin-0, Ler, Nd-1, Shakdara and Van-0 <a href="#pone.0001838-Lempe1">[32]</a>. These accession experiments were independently conducted, with overlap of five accessions. Accession I microarrays measurements used five-week old plants grown under short day conditions (10:14 hours light:dark) <a href="#pone.0001838-Kliebenstein3">[31]</a>, while Accessions II analyzed RNA from 12-day old plants grown under long days (16:8 hours light:dark) <a href="#pone.0001838-Lempe1">[32]</a>. The short day conditions can be considered non-inductive for flowering, while long days would induce flowering <a href="#pone.0001838-Michaels1">[63]</a>. All microarrays were quantile normalized via GC-RMA and transcript levels converted to their log<sub>2</sub> values. The mean log<sub>2</sub> transcript accumulation per gene per experimental unit (accession, tissue, stress, or time point) was used for further analysis. Only biological replicates were utilized and technical replicates were discarded. When technical replicates were present, one array was chosen at random to represent that sample.</p>


<h4>Transcript variance measures</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>To compare variance across experiments and transcripts, I first estimated the per transcript variance (σ<sup>2</sup>) across the mean log<sub>2</sub> transcript accumulation per experimental unit for each transcribed locus measured within a given experimental dataset (<a href="#pone.0001838.s001">Table S1</a>). The mean (µ) for each transcript was determined across all microarrays for the given experimental dataset. Only biological replicates were utilized and technical replicates were discarded. These mean and variance values per dataset were used to generate two standardized measures of transcript accumulation variance comparison of variances for each transcript across datasets: the variance-to-mean ratio (VMR = σ<sup>2</sup>/µ) and the unit less coefficient of variance (CV = σ/µ)<a href="#pone.0001838-Raser1">[35]</a>, <a href="#pone.0001838-Elowitz1">[36]</a>( <a href="#pone.0001838.s001">Table S1</a>). These measures were independently estimated for each of the four microarray datasets. Only probe-sets that identified unique genes were utilized in this analysis and all probe-sets that identified multiple genes were discarded to control for potential cross-hybridization. Any cross hybridization at the individual probe level will likely lead to lower variance estimates for both duplicated genes.</p>


<h4>Gene Duplication</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A previous analysis of gene duplication within Arabidopsis was used to define genes as either unique (no duplicate copy), segmental duplicated (a duplicate somewhere in the genome but not tandem) and tandem duplicated (the duplicate gene is next to the tested gene) <a href="#pone.0001838-Rizzon1">[5]</a>. The high-stringency analysis from this previous work provided a base definition of duplication status for this analysis. Tandem duplicated genes are considered a sub-group of duplicated genes for this analysis (<a href="#pone.0001838.s001">Table S1</a>). This analysis includes 21,460 genes for which there are both microarray data and assigned duplication status. 12,676 genes are classified as unique, defined as having no duplicate at a stringency of &gt;50% identity and &gt;90% alignment length <a href="#pone.0001838-Rizzon1">[5]</a>. 7,272 genes are classified as being segmental duplicates and 1,512 genes are classified as being tandem duplicates with the duplicate copies immediately neighboring each other <a href="#pone.0001838-Rizzon1">[5]</a>.</p>


<h4>Metabolic pathway definition</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Aracyc version 3.1 was utilized to define metabolic pathways <a href="#pone.0001838-Zhang1">[20]</a>, <a href="#pone.0001838-Mueller1">[21]</a>. Specific secondary metabolite pathways were edited to include recently published enzymes and genetic loci <a href="#pone.0001838-Hansen1">[59]</a>, <a href="#pone.0001838-Nafisi1">[80]</a>. A previously un-annotated pathway, glucosinolate breakdown, was created using recently published data <a href="#pone.0001838-Lambrix1">[53]</a>, <a href="#pone.0001838-Zhang2">[54]</a>, <a href="#pone.0001838-Barth1">[81]</a>–<a href="#pone.0001838-Burow1">[83]</a>( <a href="#pone.0001838.s002">Table S2</a>). Only those metabolic pathways with at least five genes and five separate transcripts measured by the ATH1 microarray were further analyzed. Biochemical activity of genes involved in the aliphatic glucosinolate biosynthetic pathway was defined from the literature and unpublished data for three FMO genes <a href="#pone.0001838-Kliebenstein1">[15]</a>, <a href="#pone.0001838-Textor1">[58]</a>–<a href="#pone.0001838-Chen1">[60]</a>, <a href="#pone.0001838-Textor2">[70]</a>, <a href="#pone.0001838-Grubb1">[73]</a>, <a href="#pone.0001838-Schuster1">[74]</a>, <a href="#pone.0001838-Hansen3">[84]</a>–<a href="#pone.0001838-Mikkelsen2">[87]</a>.</p>


<h4>Duplicated Gene Variance</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>For the four independent datasets, VMR and CV were independently determined for all transcripts (<a href="#pone-0001838-g001">Figure 1</a>). The genes were then separated by duplication status and the mean VMR and CV were determined for unique, segmental duplicated and tandem duplicated genes for each dataset. To test if the mean VMR and CV for these duplication groupings were significantly different from the genomic mean, a bootstrap analysis was conducted. Random samples containing an identical number of genes to each of the three duplication groupings (12,767 genes for the unique, 7,272 genes for the segmental duplicate and 1,512 for the tandem duplicate group (<a href="#pone-0001838-g001">Figure 1</a>)) were drawn from the full genome and the mean VMR and CV of each of these samples was determined. Repeated 2,000 times, this generated a random distribution of mean VMR and CV values for each of the four microarray datasets. Independently for each microarray dataset, the mean VMR and CV for the unique, segmental duplicated and tandem duplicated genes were then compared to the appropriate random sampling distribution to test for a significant difference from the genomic mean. To test the significance of differences between the segmental and tandem duplicated gene groups for each dataset, a bootstrapping analysis was conducted such that 8,784 genes were randomly picked from the whole genome, with 1,512 of these genes being randomly assigned as tandem duplicates and the other 7,272 genes being assigned as segmental duplicates. The average CV and VMR of these two groups was determined, the difference in values obtained, and this process repeated 2,000 times. The difference between the observed values of CV and VMR for the duplicated and tandem duplicated genes for each dataset was compared to this random distribution to test if the observed difference was significantly different from a random expectation.</p>


<h4>Analysis of Accession Variance Components</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>A microarray dataset comparing the response of seven Arabidopsis accessions, Col-0, Cvi-1, Est, Kin-0, Mt-0, Tsu-1 and Van-0, to exogenous salicylic acid treatment was utilized to directly assess the impact of gene duplication on different sources of gene expression variation <a href="#pone.0001838-VanLeeuwen1">[40]</a>. I used a mixed linear model ANOVA in SAS to analyze the GCRMA normalized log<sub>2</sub> transcript level (gene expression) data from the factorial experiment to estimate the variance contributions of accession versus treatment. For each gene, the transcript level of Accession <em>g</em> under SA treatment <em>j</em> for the replication <em>r</em> is denoted as <em>y<sub>gnjkr</sub></em>. The ANOVA model for the log<sub>2</sub>-transformed expression levels is: <em>log<sub>2</sub>(y<sub>gjr</sub>) = µ+S<sub>j</sub></em>+<em>G<sub>g</sub></em>+<em>R<sub>r</sub></em>+<em>SG<sub>gj</sub>+SR<sub>jr</sub>+GR<sub>gr</sub>+ε<sub>gjr</sub></em> where <em>g</em> = 1, …,7; <em>j = </em>1,2; and <em>r</em> = 1, 2, 3. The main effects are denoted as <em>G</em>, <em>S</em>, and <em>R</em> and represent gene, treatment, and replicate respectively. Replicate was treated as a main effect to estimate its impact on transcript variance within this specific experiment. The error, <em>ε<sub>gikr</sub></em>, is assumed to be normally distributed with mean 0 and variance σ<sub>ε</sub><sup>2</sup>. The σ<sup>2</sup> for each main effect and interaction term was divided by the total σ<sup>2</sup> for each gene to obtain the percent of variance per term. This was repeated for all genes and for each term, producing an average percent variance for the four different gene groups, unique, whole genome, duplicate and tandem duplicate. The significance of deviation from the genome average for the unique, segmental duplicate and tandem duplicate groups was estimated via bootstrap analysis as described above.</p>


<h4>Analysis of eQTL Bias</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The expression QTL (eQTL) position and R<sup>2</sup> for each eQTL for each transcript was obtained from a previous eQTL mapping experiment comparing two Arabidopsis accessions, Bay-0 (Bayreuth) and Sha (Shahdara) <a href="#pone.0001838-West1">[39]</a>. For all transcripts, the average number of eQTL, the frequency of a <em>cis</em>-eQTL, the average R<sup>2</sup> per eQTL, and the maximum R<sup>2</sup> per eQTL was determined. The genome average for each variable was determined and then re-measured for each duplication group. The significance of deviation from the genome average for the unique, segmental duplicate and tandem duplicate groups was estimated via bootstrapping as described above.</p>


<h4>Metabolic Pathway Variance</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>For the four independent datasets, the mean VMR and CV were independently determined for all transcripts included in a given metabolic pathway (<a href="#pone.0001838.s003">Table S3</a>). Bootstrapping analyses were employed to test if the mean VMR and CV for transcript accumulation within each metabolic pathway were significantly different from a random genomic sample. For each round of the bootstrap, 5, 10, 15, 20, 25, 30, 35, 40, 45 and 50 genes were drawn at random from the given dataset and the mean VMR and CV determined across these genes. These gene numbers represent random metabolic pathways containing different numbers of genes. This was repeated 2,000 times to generate a random distribution of VMR and CV for each dataset. This generated 40 different random sampling distributions (four datasets×10 pathway sizes). For each metabolic pathway, the number of genes, N, was rounded to the nearest five and the random distribution from the appropriate dataset and pathway size utilized to test if that pathways VMR or CV differed significantly from a random “pathway” of similar size. <em>P</em> values were tested for significance under an FDR of 0.05 (<a href="#pone.0001838.s003">Table S3</a>).</p>


<h4>Metabolic Pathway Duplication</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>For each metabolic pathway, the number of segmental duplicated and tandem duplicated genes was determined from previous analysis (<a href="#pone.0001838.s001">Tables S1</a> and <a href="#pone.0001838.s004">S4</a>) <a href="#pone.0001838-Rizzon1">[5]</a>. The genomic frequency of segmental duplicated and tandem duplicated genes was then used to generate the expected numbers of segmental duplicated and tandem duplicated genes for each metabolic pathway (<a href="#pone.0001838.s004">Table S4</a>). Each pathway was tested for deviation from expected levels of gene duplication via χ<sup>2</sup> analysis with the <em>P</em> values tested for significance under an FDR of 0.05 (<a href="#pone.0001838.s004">Table S4</a>).</p>


<h4>Sequence Diversity</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>A previously published dataset measuring genomic sequence diversity in 27 accessions was used to test for differential sequence diversity between metabolic pathways <a href="#pone.0001838-Borevitz2">[52]</a>. Estimates of Θ, π, and Tajima's D per gene were obtained from published data and the average across the genes within a metabolic pathway was calculated. Bootstrap analysis as described above was conducted to compare each metabolic pathways average Θ, π, and Tajima's D to a random genomic sample with a similar sized pathway. No pathways were observed to have a statistically significant bias in any sequence diversity value.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001838"></div><a name="pone.0001838.s001" id="pone.0001838.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001838.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Per Gene Statistical Values. The per gene statistical values are presented for all genes measured on the Affymetrix ATH1 microarray for Arabidopsis for the four datasets utilized in this manuscript. The datasets derived from experiments querying gene expression in response to abiotic stress variation, developmental variation, and two independent analysis of variation between different natural Arabidopsis accessions. Probe Set represents the probe set on the ATH1 microarray, AGI is the Arabidopsis gene code, Duplication represents the predicted gene duplication status <a href="#pone.0001838-Rizzon1">[5]</a>. Mean = mean transcript accumulation (μ) in log<sub>2</sub> across the samples within the dataset, Var = per transcript variance (σ<sup>2</sup>) across the samples within the dataset, VMR = σ<sup>2</sup>/μ for each transcript, and CV = σ/μ for each transcript. For the dataset Accession I, R<sup>2</sup> is the pearson correlation coefficient for transcript accumulation between the different accessions using all of the genes within a pathway.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001838.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(8.64 MB XLS)</p>
<a name="pone.0001838.s002" id="pone.0001838.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001838.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Biosynthetic Pathways. List of biosynthetic pathways and genes per each pathway utilized within this manuscript.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001838.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.20 MB XLS)</p>
<a name="pone.0001838.s003" id="pone.0001838.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001838.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Estimates of transcript variance per biosynthetic pathway. The mean statistical values per pathway and significance of the deviation from the genomic mean are presented. Pathway lists the biosynthetic pathway, N is the number of genes per pathway, Mean is the mean transcript accumulation in log<sub>2</sub> across the genes within the pathway for the given dataset. VMR is the mean VMR across the genes within the pathway for the given dataset and P<sub>VMR</sub> is the likelihood that this is equivalent to a random collection of N genes from the whole genome as estimated by 2000 random permutations. CV is the mean CV across the genes within the pathway for the given dataset and P<sub>CV</sub> is the likelihood that this mean CV is equivalent to a random collection of N genes from the whole genome as estimated by 2000 random permutations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001838.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.06 MB XLS)</p>
<a name="pone.0001838.s004" id="pone.0001838.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001838.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Duplication bias within metabolic pathways. The distribution of gene duplications within the pathways is presented. Genes (N) lists the number of genes within a pathway while Observed gives the number of these genes that are classified as segmental duplicates or tandem duplicates. The genomic frequency of segmental duplicates and tandem duplicates were used to predict the expected frequency of duplications for a given N. The fit of goodness between the observed and expected values were tested via χ<sup>2</sup> and only P values significant at an FDR of 0.05 are presented as significant. This was done for both segmental duplicate and tandem duplicates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001838.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0001838.s005" id="pone.0001838.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001838.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Sequence diversity within metabolic pathways. The distribution of sequence diversity estimates for the metabolic pathways are presented, θ, π, and Tajima's D. The per gene values for these paramaters were obtained from previously published data <a href="#pone.0001838-Borevitz2">[52]</a>. P values for deviation from the genomic average were obtained via permutation as described.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001838.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>I would like to thank Dr. Barbara Halkier, Dr. Bjarne Gram Hansen, Heather Rowe, Ida Elken Sønderby and Adam Wentzell for reviewing this manuscript and Dr. Katherine Pollard for statistical insight.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DK. Performed the experiments: DK. Analyzed the data: DK. Contributed reagents/materials/analysis tools: DK. Wrote the paper: DK.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001838-Vision1" id="pone.0001838-Vision1"></a>Vision TJ, Brown DG, Tanksley SD (2000) The origins of genomic duplications in Arabidopsis. Science  290: 2114–2117.  <ul class="find" data-citedArticleID="1078176" data-doi="10.1126/science.290.5499.2114"><li><a href="http://dx.doi.org/10.1126/science.290.5499.2114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origins+of+genomic+duplications+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origins+of+genomic+duplications+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001838-Blanc1" id="pone.0001838-Blanc1"></a>Blanc G, Hokamp K, Wolfe KH (2003) A recent polyploidy superimposed on older large-scale duplications in the Arabidopsis genome. Genome Research  13: 137–144.  <ul class="find" data-citedArticleID="1078032" data-doi="10.1101/gr.751803"><li><a href="http://dx.doi.org/10.1101/gr.751803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+recent+polyploidy+superimposed+on+older+large-scale+duplications+in+the+Arabidopsis+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+recent+polyploidy+superimposed+on+older+large-scale+duplications+in+the+Arabidopsis+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001838-Wolfe1" id="pone.0001838-Wolfe1"></a>Wolfe KH, Shields DC (1997) Molecular evidence for an ancient duplication of the entire yeast genome. Nature  387: 708–713.  <ul class="find" data-citedArticleID="1078184"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+evidence+for+an+ancient+duplication+of+the+entire+yeast+genome.&amp;auth=&amp;atitle=Molecular+evidence+for+an+ancient+duplication+of+the+entire+yeast+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+an+ancient+duplication+of+the+entire+yeast+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+an+ancient+duplication+of+the+entire+yeast+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001838-Blanc2" id="pone.0001838-Blanc2"></a>Blanc G, Wolfe KH (2004) Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes. Plant Cell  16: 1667–1678.  <ul class="find" data-citedArticleID="1078034" data-doi="10.1105/tpc.021345"><li><a href="http://dx.doi.org/10.1105/tpc.021345" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+paleopolyploidy+in+model+plant+species+inferred+from+age+distributions+of+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+paleopolyploidy+in+model+plant+species+inferred+from+age+distributions+of+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001838-Rizzon1" id="pone.0001838-Rizzon1"></a>Rizzon C, Ponger L, Gaut BS (2006) Striking similarities in the genomic distribution of tandemly arrayed genes in Arabidopsis and rice. Plos Computational Biology  2: 989–1000.  <ul class="find" data-citedArticleID="1078156" data-doi="10.1371/journal.pcbi.0020115.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020115.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Striking+similarities+in+the+genomic+distribution+of+tandemly+arrayed+genes+in+Arabidopsis+and+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Striking+similarities+in+the+genomic+distribution+of+tandemly+arrayed+genes+in+Arabidopsis+and+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001838-Walsh1" id="pone.0001838-Walsh1"></a>Walsh JB (1995) How Often Do Duplicated Genes Evolve New Functions. Genetics  139: 421–428.  <ul class="find" data-citedArticleID="1078178"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=How+Often+Do+Duplicated+Genes+Evolve+New+Functions.&amp;auth=&amp;atitle=How+Often+Do+Duplicated+Genes+Evolve+New+Functions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+Often+Do+Duplicated+Genes+Evolve+New+Functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+Often+Do+Duplicated+Genes+Evolve+New+Functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001838-Lynch1" id="pone.0001838-Lynch1"></a>Lynch M, Conery JS (2000) The evolutionary fate and consequences of duplicate genes. Science  290: 1151–1155.  <ul class="find" data-citedArticleID="1078124" data-doi="10.1126/science.290.5494.1151"><li><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+fate+and+consequences+of+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+fate+and+consequences+of+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001838-Ganko1" id="pone.0001838-Ganko1"></a>Ganko EW, Meyers BC, Vision TJ (2007) Divergence in expression between duplicated genes in Arabidopsis. Molecular Biology and Evolution: doi:10.1093/molbev/msm1158.   <ul class="find" data-citedArticleID="1078066" data-doi="10.1093/molbev/msm158"><li><a href="http://dx.doi.org/10.1093/molbev/msm158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergence+in+expression+between+duplicated+genes+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergence+in+expression+between+duplicated+genes+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001838-Casneuf1" id="pone.0001838-Casneuf1"></a>Casneuf T, De Bodt S, Raes J, Maere S, Van de Peer Y (2006) Nonrandom divergence of gene expression following gene and genome duplications in the flowering plant <em>Arabidopsis thaliana</em>. Genome Biology  7:   <ul class="find" data-citedArticleID="1078046" data-doi="10.1186/gb-2006-7-2-r13"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-2-r13" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonrandom+divergence+of+gene+expression+following+gene+and+genome+duplications+in+the+flowering+plant+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonrandom+divergence+of+gene+expression+following+gene+and+genome+duplications+in+the+flowering+plant+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001838-Ha1" id="pone.0001838-Ha1"></a>Ha M, Li WH, Chen ZJ (2007) External factors accelerate expression divergence between duplicate genes. Trends in Genetics  23: 162–166.  <ul class="find" data-citedArticleID="1078074" data-doi="10.1016/j.tig.2007.02.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2007.02.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=External+factors+accelerate+expression+divergence+between+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22External+factors+accelerate+expression+divergence+between+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001838-Li1" id="pone.0001838-Li1"></a>Li WH, Yang J, Gu X (2005) Expression divergence between duplicate genes. Trends in Genetics  21: 602–607.  <ul class="find" data-citedArticleID="1078122" data-doi="10.1016/j.tig.2005.08.006"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.08.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+divergence+between+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+divergence+between+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001838-Duarte1" id="pone.0001838-Duarte1"></a>Duarte JM, Cui LY, Wall PK, Zhang Q, Zhang XH, et al.  (2006) Expression pattern shifts following duplication indicative of subfunctionalization and neofunctionalization in regulatory genes of Arabidopsis. Molecular Biology and Evolution  23: 469–478.  <ul class="find" data-citedArticleID="1078056" data-doi="10.1093/molbev/msj051"><li><a href="http://dx.doi.org/10.1093/molbev/msj051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+pattern+shifts+following+duplication+indicative+of+subfunctionalization+and+neofunctionalization+in+regulatory+genes+of+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+pattern+shifts+following+duplication+indicative+of+subfunctionalization+and+neofunctionalization+in+regulatory+genes+of+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001838-Gu1" id="pone.0001838-Gu1"></a>Gu ZL, Rifkin SA, White KP, Li WH (2004) Duplicate genes increase gene expression diversity within and between species. Nature Genetics  36: 577–579.  <ul class="find" data-citedArticleID="1078072" data-doi="10.1038/ng1355"><li><a href="http://dx.doi.org/10.1038/ng1355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplicate+genes+increase+gene+expression+diversity+within+and+between+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplicate+genes+increase+gene+expression+diversity+within+and+between+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001838-Kroymann1" id="pone.0001838-Kroymann1"></a>Kroymann J, Donnerhacke S, Schnabelrauch D, Mitchell-Olds T (2003) Evolutionary dynamics of an Arabidopsis insect resistance quantitative trait locus. Proceedings Of The National Academy Of Sciences Of The United States Of America  100: 14587–14592.  <ul class="find" data-citedArticleID="1078110" data-doi="10.1073/pnas.1734046100"><li><a href="http://dx.doi.org/10.1073/pnas.1734046100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+an+Arabidopsis+insect+resistance+quantitative+trait+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+an+Arabidopsis+insect+resistance+quantitative+trait+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001838-Kliebenstein1" id="pone.0001838-Kliebenstein1"></a>Kliebenstein D, Lambrix V, Reichelt M, Gershenzon J, Mitchell-Olds T (2001) Gene duplication and the diversification of secondary metabolism: side chain modification of glucosinolates in <em>Arabidopsis thaliana</em>. Plant Cell  13: 681–693.  <ul class="find" data-citedArticleID="1078094" data-doi="10.2307/3871415"><li><a href="http://dx.doi.org/10.2307/3871415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+duplication+and+the+diversification+of+secondary+metabolism%3A+side+chain+modification+of+glucosinolates+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+duplication+and+the+diversification+of+secondary+metabolism%3A+side+chain+modification+of+glucosinolates+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001838-Maere1" id="pone.0001838-Maere1"></a>Maere S, De Bodt S, Raes J, Casneuf T, Van Montagu M, et al.  (2005) Modeling gene and genome duplications in eukaryotes. Proceedings of the National Academy of Sciences of the United States of America  102: 5454–5459.  <ul class="find" data-citedArticleID="1078128" data-doi="10.1073/pnas.0501102102"><li><a href="http://dx.doi.org/10.1073/pnas.0501102102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+gene+and+genome+duplications+in+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+gene+and+genome+duplications+in+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001838-Kalisz1" id="pone.0001838-Kalisz1"></a>Kalisz S, Kramer EM (2007) Variation and constraint in plant evolution and development. Heredity.   <ul class="find" data-citedArticleID="1078086" data-doi="10.1038/sj.hdy.6800939"><li><a href="http://dx.doi.org/10.1038/sj.hdy.6800939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+and+constraint+in+plant+evolution+and+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+and+constraint+in+plant+evolution+and+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001838-Schwenk1" id="pone.0001838-Schwenk1"></a>Schwenk K, Wagner GP (2003)  Constraint. In: Hall BK, Olson WM, editors. Keywords and Concepts in Evolutionary Developmental Biology. Cambridge, MA: Harvard University Press. pp. 52–60.  <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001838-Ashburner1" id="pone.0001838-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene Ontology: tool for the unification of biology. Nature Genetics  25: 25–29.  <ul class="find" data-citedArticleID="1078022" data-doi="10.1038/75556"><li><a href="http://dx.doi.org/10.1038/75556" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Ontology%3A+tool+for+the+unification+of+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Ontology%3A+tool+for+the+unification+of+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001838-Zhang1" id="pone.0001838-Zhang1"></a>Zhang PF, Foerster H, Tissier CP, Mueller L, Paley S, et al.  (2005) MetaCyc and AraCyc. Metabolic pathway databases for plant research. Plant Physiology  138: 27–37.  <ul class="find" data-citedArticleID="1078188" data-doi="10.1104/pp.105.060376"><li><a href="http://dx.doi.org/10.1104/pp.105.060376" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MetaCyc+and+AraCyc.+Metabolic+pathway+databases+for+plant+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MetaCyc+and+AraCyc.+Metabolic+pathway+databases+for+plant+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001838-Mueller1" id="pone.0001838-Mueller1"></a>Mueller LA, Zhang PF, Rhee SY (2003) AraCyc: A biochemical pathway database for Arabidopsis. Plant Physiology  132: 453–460.  <ul class="find" data-citedArticleID="1078144" data-doi="10.1104/pp.102.017236"><li><a href="http://dx.doi.org/10.1104/pp.102.017236" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AraCyc%3A+A+biochemical+pathway+database+for+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AraCyc%3A+A+biochemical+pathway+database+for+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001838-Kliebenstein2" id="pone.0001838-Kliebenstein2"></a>Kliebenstein DJ (2004) Secondary metabolites and plant/environment interactions: a view through <em>Arabidopsis thaliana</em> tinged glasses. Plant Cell Environ  27: 675–684.  <ul class="find" data-citedArticleID="1078096" data-doi="10.1111/j.1365-3040.2004.01180.x"><li><a href="http://dx.doi.org/10.1111/j.1365-3040.2004.01180.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secondary+metabolites+and+plant%2Fenvironment+interactions%3A+a+view+through+Arabidopsis+thaliana+tinged+glasses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secondary+metabolites+and+plant%2Fenvironment+interactions%3A+a+view+through+Arabidopsis+thaliana+tinged+glasses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001838-DAuria1" id="pone.0001838-DAuria1"></a>D'Auria JC, Gershenzon J (2005) The secondary metabolism of <em>Arabidopsis thaliana</em>: growing like a weed. Current Opinion in Plant Biology  8: 308–316.  <ul class="find" data-citedArticleID="1078054" data-doi="10.1016/j.pbi.2005.03.012"><li><a href="http://dx.doi.org/10.1016/j.pbi.2005.03.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+secondary+metabolism+of+Arabidopsis+thaliana%3A+growing+like+a+weed." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+secondary+metabolism+of+Arabidopsis+thaliana%3A+growing+like+a+weed.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001838-Chapple1" id="pone.0001838-Chapple1"></a>Chapple CCS, Shirley BW, Zook M, Hammerschmidt R, Somerville SC (1994)  Secondary Metabolism in Arabidopsis. In: Meyerowitz EMaCSS, editor. Arabidopsis: Cold Spring Harbor Laboratory Press. pp. 989–1030.  <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0001838-Ober1" id="pone.0001838-Ober1"></a>Ober D (2005) Seeing double: gene duplication and diversification in plant secondary metabolism. Trends in Plant Science  10: 444–449.  <ul class="find" data-citedArticleID="1078152" data-doi="10.1016/j.tplants.2005.07.007"><li><a href="http://dx.doi.org/10.1016/j.tplants.2005.07.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seeing+double%3A+gene+duplication+and+diversification+in+plant+secondary+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seeing+double%3A+gene+duplication+and+diversification+in+plant+secondary+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001838-Grun1" id="pone.0001838-Grun1"></a>Grun S, Frey M, Gierl A (2005) Evolution of the indole alkaloid biosynthesis in the genus Hordeum: Distribution of gramine and DIBOA and isolation of the benzoxazinoid biosynthesis genes from <em>Hordeum lechleri</em>. Phytochemistry  66: 1264–1272.  <ul class="find" data-citedArticleID="1078070" data-doi="10.1016/j.phytochem.2005.01.024"><li><a href="http://dx.doi.org/10.1016/j.phytochem.2005.01.024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+indole+alkaloid+biosynthesis+in+the+genus+Hordeum%3A+Distribution+of+gramine+and+DIBOA+and+isolation+of+the+benzoxazinoid+biosynthesis+genes+from+Hordeum+lechleri." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+indole+alkaloid+biosynthesis+in+the+genus+Hordeum%3A+Distribution+of+gramine+and+DIBOA+and+isolation+of+the+benzoxazinoid+biosynthesis+genes+from+Hordeum+lechleri.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001838-Benderoth1" id="pone.0001838-Benderoth1"></a>Benderoth M, Textor S, Windsor AJ, Mitchell-Olds T, Gershenzon J, et al.  (2006) Positive selection driving diversification in plant secondary metabolism. Proceedings of the National Academy of Sciences of the United States of America  103: 9118–9123.  <ul class="find" data-citedArticleID="1078028" data-doi="10.1073/pnas.0601738103"><li><a href="http://dx.doi.org/10.1073/pnas.0601738103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+selection+driving+diversification+in+plant+secondary+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+selection+driving+diversification+in+plant+secondary+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001838-Halkier1" id="pone.0001838-Halkier1"></a>Halkier BA, Gershenzon J (2006) Biology and biochemistry of glucosinolates. Annual Review of Plant Biology  57: 303–333.  <ul class="find" data-citedArticleID="1078076" data-doi="10.1146/annurev.arplant.57.032905.105228"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biology+and+biochemistry+of+glucosinolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biology+and+biochemistry+of+glucosinolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001838-Lankau1" id="pone.0001838-Lankau1"></a>Lankau RA (2007) Specialist and generalist herbivores exert opposing selection on a chemical defense New Phytologist  175: 176–184.  <ul class="find" data-citedArticleID="1078116" data-doi="10.1111/j.1469-8137.2007.02090.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2007.02090.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specialist+and+generalist+herbivores+exert+opposing+selection+on+a+chemical+defense" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specialist+and+generalist+herbivores+exert+opposing+selection+on+a+chemical+defense%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001838-Lankau2" id="pone.0001838-Lankau2"></a>Lankau RA, Strauss SY (2007) Mutual Feedbacks Maintain Both Genetic and Species Diversity in a Plant Community Science  317: 1561–1563.  <ul class="find" data-citedArticleID="1078118" data-doi="10.1126/science.1147455"><li><a href="http://dx.doi.org/10.1126/science.1147455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutual+Feedbacks+Maintain+Both+Genetic+and+Species+Diversity+in+a+Plant+Community" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutual+Feedbacks+Maintain+Both+Genetic+and+Species+Diversity+in+a+Plant+Community%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001838-Kliebenstein3" id="pone.0001838-Kliebenstein3"></a>Kliebenstein DJ, West MAL, Van Leeuwen H, Kyunga K, Doerge RW, et al.  (2006) Genomic survey of gene expression diversity in <em>Arabidopsis thaliana</em> Genetics  172: 1179–1189.  <ul class="find" data-citedArticleID="1078098" data-doi="10.1534/genetics.105.049353"><li><a href="http://dx.doi.org/10.1534/genetics.105.049353" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+survey+of+gene+expression+diversity+in+Arabidopsis+thaliana" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+survey+of+gene+expression+diversity+in+Arabidopsis+thaliana%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001838-Lempe1" id="pone.0001838-Lempe1"></a>Lempe J, Balasubramanian S, Sureshkumar S, Singh A, Schmid M, et al.  (2005) Diversity of flowering responses in wild <em>Arabidopsis thaliana</em> strains. Plos Genetics  1: 109–118.  <ul class="find" data-citedArticleID="1078120" data-doi="10.1371/journal.pgen.0010006"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diversity+of+flowering+responses+in+wild+Arabidopsis+thaliana+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diversity+of+flowering+responses+in+wild+Arabidopsis+thaliana+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001838-Schmid1" id="pone.0001838-Schmid1"></a>Schmid M, Davison TS, Henz SR, Pape UJ, Demar M, et al.  (2005) A gene expression map of Arabidopsis thaliana development. Nature Genetics  37: 501–506.  <ul class="find" data-citedArticleID="1078160" data-doi="10.1038/ng1543"><li><a href="http://dx.doi.org/10.1038/ng1543" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+expression+map+of+Arabidopsis+thaliana+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+expression+map+of+Arabidopsis+thaliana+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001838-Kilian1" id="pone.0001838-Kilian1"></a>Kilian J, Whitehead D, Horak J, Wanke D, Weinl S, et al.  (2007) The AtGenExpress global stress expression data set: protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses. Plant Journal  50: 347–363.  <ul class="find" data-citedArticleID="1078088" data-doi="10.1111/j.1365-313x.2007.03052.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03052.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+AtGenExpress+global+stress+expression+data+set%3A+protocols%2C+evaluation+and+model+data+analysis+of+UV-B+light%2C+drought+and+cold+stress+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+AtGenExpress+global+stress+expression+data+set%3A+protocols%2C+evaluation+and+model+data+analysis+of+UV-B+light%2C+drought+and+cold+stress+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001838-Raser1" id="pone.0001838-Raser1"></a>Raser JM, O'Shea EK (2004) Control of stochasticity in eukaryotic gene expression. Science  304: 1811–1814.  <ul class="find" data-citedArticleID="1078154" data-doi="10.1126/science.1098641"><li><a href="http://dx.doi.org/10.1126/science.1098641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+stochasticity+in+eukaryotic+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+stochasticity+in+eukaryotic+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001838-Elowitz1" id="pone.0001838-Elowitz1"></a>Elowitz MB, Levine AJ, Siggia ED, Swain PS (2002) Stochastic gene expression in a single cell. Science  297: 1183–1186.  <ul class="find" data-citedArticleID="1078060" data-doi="10.1126/science.1070919"><li><a href="http://dx.doi.org/10.1126/science.1070919" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+gene+expression+in+a+single+cell." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+gene+expression+in+a+single+cell.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001838-Lynch2" id="pone.0001838-Lynch2"></a>Lynch M, Force A (2000) The probability of duplicate gene preservation by subfunctionalization. . Genetics  154: 459–473.  <ul class="find" data-citedArticleID="1078126"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+probability+of+duplicate+gene+preservation+by+subfunctionalization.+.&amp;auth=&amp;atitle=The+probability+of+duplicate+gene+preservation+by+subfunctionalization.+." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+probability+of+duplicate+gene+preservation+by+subfunctionalization.+." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+probability+of+duplicate+gene+preservation+by+subfunctionalization.+.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001838-Clark1" id="pone.0001838-Clark1"></a>Clark RM, Schweikert G, Toomajian C, Ossowski S, Zeller G, et al.  (2007) Common sequence polymorphisms shaping genetic diversity in <em>Arabidopsis thaliana</em>. Science  317: 338–342.  <ul class="find" data-citedArticleID="1078052" data-doi="10.1126/science.1138632"><li><a href="http://dx.doi.org/10.1126/science.1138632" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Common+sequence+polymorphisms+shaping+genetic+diversity+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Common+sequence+polymorphisms+shaping+genetic+diversity+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001838-West1" id="pone.0001838-West1"></a>West MAL, Kim K, Kliebenstein DJ, van Leeuwen H, Michelmore RW, et al.  (2007) Global eQTL Mapping Reveals the Complex Genetic Architecture of Transcript Level Variation in Arabidopsis. Genetics  175: 1441–1450.  <ul class="find" data-citedArticleID="1078182" data-doi="10.1534/genetics.106.064972"><li><a href="http://dx.doi.org/10.1534/genetics.106.064972" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+eQTL+Mapping+Reveals+the+Complex+Genetic+Architecture+of+Transcript+Level+Variation+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+eQTL+Mapping+Reveals+the+Complex+Genetic+Architecture+of+Transcript+Level+Variation+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001838-VanLeeuwen1" id="pone.0001838-VanLeeuwen1"></a>Van Leeuwen H, Kliebenstein DJ, West MAL, kim KD, van Poecke R, et al.  (2007) Natural Variation among <em>Arabidopsis thaliana</em> Accessions for Transcriptome Response to Exogenous Salicylic Acid. Plant Cell  19: 2099–2110.  <ul class="find" data-citedArticleID="1078174" data-doi="10.1105/tpc.107.050641"><li><a href="http://dx.doi.org/10.1105/tpc.107.050641" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+Variation+among+Arabidopsis+thaliana+Accessions+for+Transcriptome+Response+to+Exogenous+Salicylic+Acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+Variation+among+Arabidopsis+thaliana+Accessions+for+Transcriptome+Response+to+Exogenous+Salicylic+Acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001838-Borevitz1" id="pone.0001838-Borevitz1"></a>Borevitz JO, Liang D, Plouffe D, Chang HS, Zhu T, et al.  (2003) Large-scale identification of single-feature polymorphisms in complex genomes. Genome Research  13: 513–523.  <ul class="find" data-citedArticleID="1078036" data-doi="10.1101/gr.541303"><li><a href="http://dx.doi.org/10.1101/gr.541303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+identification+of+single-feature+polymorphisms+in+complex+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+identification+of+single-feature+polymorphisms+in+complex+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001838-Xie1" id="pone.0001838-Xie1"></a>Xie DX, Feys BF, James S, Nieto-Rostro M, Turner JG (1998) COI1: An Arabidopsis gene required for jasmonate-regulated defense and fertility. Science  280: 1091–1094.  <ul class="find" data-citedArticleID="1078186" data-doi="10.1126/science.280.5366.1091"><li><a href="http://dx.doi.org/10.1126/science.280.5366.1091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=COI1%3A+An+Arabidopsis+gene+required+for+jasmonate-regulated+defense+and+fertility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22COI1%3A+An+Arabidopsis+gene+required+for+jasmonate-regulated+defense+and+fertility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001838-Benedetti1" id="pone.0001838-Benedetti1"></a>Benedetti CE, Xie DX, Turner JG (1995) Coi1-Dependent Expression of an Arabidopsis Vegetative Storage Protein in Flowers and Siliques and in Response to Coronatine or Methyl Jasmonate. Plant Physiology  109: 567–572.  <ul class="find" data-citedArticleID="1078030" data-doi="10.1104/pp.109.2.567"><li><a href="http://dx.doi.org/10.1104/pp.109.2.567" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coi1-Dependent+Expression+of+an+Arabidopsis+Vegetative+Storage+Protein+in+Flowers+and+Siliques+and+in+Response+to+Coronatine+or+Methyl+Jasmonate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coi1-Dependent+Expression+of+an+Arabidopsis+Vegetative+Storage+Protein+in+Flowers+and+Siliques+and+in+Response+to+Coronatine+or+Methyl+Jasmonate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001838-Kim1" id="pone.0001838-Kim1"></a>Kim SY (2007) Recent advances in ABA signaling. Journal of Plant Biology  50: 117–121.  <ul class="find" data-citedArticleID="1078090" data-doi="10.1007/bf03030619"><li><a href="http://dx.doi.org/10.1007/bf03030619" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+advances+in+ABA+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+advances+in+ABA+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001838-Bartels1" id="pone.0001838-Bartels1"></a>Bartels D, Sunkar R (2005) Drought and salt tolerance in plants. Critical Reviews in Plant Sciences  24: 23–58.  <ul class="find" data-citedArticleID="1078024" data-doi="10.1080/07352680590910410"><li><a href="http://dx.doi.org/10.1080/07352680590910410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Drought+and+salt+tolerance+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Drought+and+salt+tolerance+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001838-Smeekens1" id="pone.0001838-Smeekens1"></a>Smeekens S (2000) Sugar-induced signal transduction in plants. Annual Review of Plant Physiology and Plant Molecular Biology  51: 49–81.  <ul class="find" data-citedArticleID="1078166" data-doi="10.1146/annurev.arplant.51.1.49"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.51.1.49" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sugar-induced+signal+transduction+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sugar-induced+signal+transduction+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001838-Kliebenstein4" id="pone.0001838-Kliebenstein4"></a>Kliebenstein DJ, Figuth A, Mitchell-Olds T (2002) Genetic Architecture of Plastic Methyl Jasmonate Responses in <em>Arabidopsis thaliana.</em> Genetics  161: 1685–1696.  <ul class="find" data-citedArticleID="1078100"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+Architecture+of+Plastic+Methyl+Jasmonate+Responses+in+Arabidopsis+thaliana.&amp;auth=&amp;atitle=Genetic+Architecture+of+Plastic+Methyl+Jasmonate+Responses+in+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+Architecture+of+Plastic+Methyl+Jasmonate+Responses+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+Architecture+of+Plastic+Methyl+Jasmonate+Responses+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001838-Zhou1" id="pone.0001838-Zhou1"></a>Zhou N, Tootle TL, Glazebrook J (1999) Arabidopsis PAD3, a gene required for camalexin biosynthesis, encodes a putative cytochrome P450 monooxygenase. Plant Cell  11: 2419–2428.  <ul class="find" data-citedArticleID="1078194" data-doi="10.2307/3870965"><li><a href="http://dx.doi.org/10.2307/3870965" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+PAD3%2C+a+gene+required+for+camalexin+biosynthesis%2C+encodes+a+putative+cytochrome+P450+monooxygenase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+PAD3%2C+a+gene+required+for+camalexin+biosynthesis%2C+encodes+a+putative+cytochrome+P450+monooxygenase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001838-Brown1" id="pone.0001838-Brown1"></a>Brown PD, Tokuhisa JG, Reichelt M, Gershenzon J (2003) Variation of glucosinolate accumulation among different organs and developmental stages of <em>Arabidopsis thaliana</em>. Phytochem  62: 471–781.  <ul class="find" data-citedArticleID="1078042" data-doi="10.1016/s0031-9422(02)00549-6"><li><a href="http://dx.doi.org/10.1016/s0031-9422(02)00549-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+of+glucosinolate+accumulation+among+different+organs+and+developmental+stages+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+of+glucosinolate+accumulation+among+different+organs+and+developmental+stages+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001838-Mikkelsen1" id="pone.0001838-Mikkelsen1"></a>Mikkelsen MD, Petersen BL, Glawischnig E, Jensen AB, Andreasson E, et al.  (2003) Modulation of CYP79 genes and glucosinolate profiles in Arabidopsis by defense signaling pathways. Plant Physiology  131: 298–308.  <ul class="find" data-citedArticleID="1078136" data-doi="10.1104/pp.011015"><li><a href="http://dx.doi.org/10.1104/pp.011015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+CYP79+genes+and+glucosinolate+profiles+in+Arabidopsis+by+defense+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+CYP79+genes+and+glucosinolate+profiles+in+Arabidopsis+by+defense+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001838-Kliebenstein5" id="pone.0001838-Kliebenstein5"></a>Kliebenstein DJ, Rowe HC, Denby KJ (2005) Secondary metabolites influence Arabidopsis/Botrytis interactions: variation in host production and pathogen sensitivity. Plant Journal  44: 25–36.  <ul class="find" data-citedArticleID="1078102" data-doi="10.1111/j.1365-313x.2005.02508.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02508.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Secondary+metabolites+influence+Arabidopsis%2FBotrytis+interactions%3A+variation+in+host+production+and+pathogen+sensitivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Secondary+metabolites+influence+Arabidopsis%2FBotrytis+interactions%3A+variation+in+host+production+and+pathogen+sensitivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001838-Borevitz2" id="pone.0001838-Borevitz2"></a>Borevitz JO, Hazen SP, Michael TP, Morris GP, Baxter IR, et al.  (2007) Genome-wide patterns of single-feature polymorphism in <em>Arabidopsis thaliana</em>. Proceedings of the National Academy of Sciences of the United States of America  104: 12057–12062.  <ul class="find" data-citedArticleID="1078038" data-doi="10.1073/pnas.0705323104"><li><a href="http://dx.doi.org/10.1073/pnas.0705323104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+patterns+of+single-feature+polymorphism+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+patterns+of+single-feature+polymorphism+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001838-Lambrix1" id="pone.0001838-Lambrix1"></a>Lambrix V, Reichelt M, Mitchell-Olds T, Kliebenstein D, Gershenzon J (2001) The Arabidopsis epithiospecifier protein promotes the hydrolysis of glucosinolates to nitriles and influences <em>Trichoplusia ni</em> herbivory. Plant Cell  13: 2793–2807.  <ul class="find" data-citedArticleID="1078112" data-doi="10.2307/3871535"><li><a href="http://dx.doi.org/10.2307/3871535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+epithiospecifier+protein+promotes+the+hydrolysis+of+glucosinolates+to+nitriles+and+influences+Trichoplusia+ni+herbivory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+epithiospecifier+protein+promotes+the+hydrolysis+of+glucosinolates+to+nitriles+and+influences+Trichoplusia+ni+herbivory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001838-Zhang2" id="pone.0001838-Zhang2"></a>Zhang Z-Y, Ober JA, Kliebenstein DJ (2006) The Gene Controlling the Quantitative Trait Locus <em>EPITHIOSPECIFIER MODIFIER1</em> Alters Glucosinolate Hydrolysis and Insect Resistance in Arabidopsis. Plant Cell  18: 1524–1536.  <ul class="find" data-citedArticleID="1078190" data-doi="10.1105/tpc.105.039602"><li><a href="http://dx.doi.org/10.1105/tpc.105.039602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Gene+Controlling+the+Quantitative+Trait+Locus+EPITHIOSPECIFIER+MODIFIER1+Alters+Glucosinolate+Hydrolysis+and+Insect+Resistance+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Gene+Controlling+the+Quantitative+Trait+Locus+EPITHIOSPECIFIER+MODIFIER1+Alters+Glucosinolate+Hydrolysis+and+Insect+Resistance+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001838-Tierens1" id="pone.0001838-Tierens1"></a>Tierens K-J, Thomma B, Brower M, Schmidt J, Kistner K, et al.  (2001) Study of the role of antimicrobial glucosinolate-derived isothiocyanates in resistance of Arabidopsis to microbial pathogens. Plant Physiol  125: 1688–1699.  <ul class="find" data-citedArticleID="1078172" data-doi="10.1104/pp.125.4.1688"><li><a href="http://dx.doi.org/10.1104/pp.125.4.1688" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+of+the+role+of+antimicrobial+glucosinolate-derived+isothiocyanates+in+resistance+of+Arabidopsis+to+microbial+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+of+the+role+of+antimicrobial+glucosinolate-derived+isothiocyanates+in+resistance+of+Arabidopsis+to+microbial+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001838-Kliebenstein6" id="pone.0001838-Kliebenstein6"></a>Kliebenstein DJ, Kroymann J, Brown P, Figuth A, Pedersen D, et al.  (2001) Genetic control of natural variation in <em>Arabidopsis thaliana</em> glucosinolate accumulation. Plant Physiol  126: 811–825.  <ul class="find" data-citedArticleID="1078104" data-doi="10.1104/pp.126.2.811"><li><a href="http://dx.doi.org/10.1104/pp.126.2.811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+control+of+natural+variation+in+Arabidopsis+thaliana+glucosinolate+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+control+of+natural+variation+in+Arabidopsis+thaliana+glucosinolate+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001838-Kliebenstein7" id="pone.0001838-Kliebenstein7"></a>Kliebenstein DJ, Gershenzon J, Mitchell-Olds T (2001) Comparative quantitative trait loci mapping of aliphatic, indolic and benzylic glucosinolate production in <em>Arabidopsis thaliana</em> leaves and seeds. Genetics  159: 359–370.  <ul class="find" data-citedArticleID="1078106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds.&amp;auth=&amp;atitle=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+quantitative+trait+loci+mapping+of+aliphatic%2C+indolic+and+benzylic+glucosinolate+production+in+Arabidopsis+thaliana+leaves+and+seeds.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001838-Textor1" id="pone.0001838-Textor1"></a>Textor S, de Kraker JW, Hause BGJ, Tokuhisa JG (2007) MAM3 catalyzes the formation of all aliphatic glucosinolate chain lengths in Arabidopsis. Plant Physiol  144: 60–71.  <ul class="find" data-citedArticleID="1078168" data-doi="10.1104/pp.106.091579"><li><a href="http://dx.doi.org/10.1104/pp.106.091579" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MAM3+catalyzes+the+formation+of+all+aliphatic+glucosinolate+chain+lengths+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MAM3+catalyzes+the+formation+of+all+aliphatic+glucosinolate+chain+lengths+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001838-Hansen1" id="pone.0001838-Hansen1"></a>Hansen BG, Kliebenstein DJ, Halkier BA (2007) Identification of a flavin-monooxygenase as the S-oxygenating enzyme in aliphatic glucosinolate biosynthesis in Arabidopsis. The Plant Journal  50: 902–910.  <ul class="find" data-citedArticleID="1078078" data-doi="10.1111/j.1365-313x.2007.03101.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2007.03101.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+flavin-monooxygenase+as+the+S-oxygenating+enzyme+in+aliphatic+glucosinolate+biosynthesis+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+flavin-monooxygenase+as+the+S-oxygenating+enzyme+in+aliphatic+glucosinolate+biosynthesis+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001838-Chen1" id="pone.0001838-Chen1"></a>Chen SX, Glawischnig E, Jorgensen K, Naur P, Jorgensen B, et al.  (2003) CYP79F1 and CYP79F2 have distinct functions in the biosynthesis of aliphatic glucosinolates in Arabidopsis. Plant Journal  33: 923–937.  <ul class="find" data-citedArticleID="1078050" data-doi="10.1046/j.1365-313x.2003.01679.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.01679.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CYP79F1+and+CYP79F2+have+distinct+functions+in+the+biosynthesis+of+aliphatic+glucosinolates+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CYP79F1+and+CYP79F2+have+distinct+functions+in+the+biosynthesis+of+aliphatic+glucosinolates+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001838-Hansen2" id="pone.0001838-Hansen2"></a>Hansen CH, Wittstock U, Olsen CE, Hick AJ, Pickett JA, et al.  (2001) Cytochrome P450CYP79F1 from Arabidopsis catalyzes the conversion of dihomomethionine and trihomomethionine to the corresponding aldoximes in the biosynthesis of aliphatic glucosinolates. Journal of Biological Chemistry  276: 11078–11085.  <ul class="find" data-citedArticleID="1078080" data-doi="10.1074/jbc.m010123200"><li><a href="http://dx.doi.org/10.1074/jbc.m010123200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytochrome+P450CYP79F1+from+Arabidopsis+catalyzes+the+conversion+of+dihomomethionine+and+trihomomethionine+to+the+corresponding+aldoximes+in+the+biosynthesis+of+aliphatic+glucosinolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytochrome+P450CYP79F1+from+Arabidopsis+catalyzes+the+conversion+of+dihomomethionine+and+trihomomethionine+to+the+corresponding+aldoximes+in+the+biosynthesis+of+aliphatic+glucosinolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001838-Langridge1" id="pone.0001838-Langridge1"></a>Langridge J (1957) Effect of Day-Length and Gibberellic Acid on the Flowering of Arabidopsis. Nature  180: 36–37.  <ul class="find" data-citedArticleID="1078114" data-doi="10.1038/180036a0"><li><a href="http://dx.doi.org/10.1038/180036a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+Day-Length+and+Gibberellic+Acid+on+the+Flowering+of+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+Day-Length+and+Gibberellic+Acid+on+the+Flowering+of+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001838-Michaels1" id="pone.0001838-Michaels1"></a>Michaels SD, Amasino RM (1999) The gibberellic acid biosynthesis mutant <em>ga1-3</em> of <em>Arabidopsis thaliana</em> is responsive to vernalization. Developmental Genetics  25: 194–198.  <ul class="find" data-citedArticleID="1078134" data-doi="10.1002/(sici)1520-6408(1999)25:3<194::aid-dvg2&gt;3.0.co;2-2"><li><a href="http://dx.doi.org/10.1002/(sici)1520-6408(1999)25:3<194::aid-dvg2&gt;3.0.co;2-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+gibberellic+acid+biosynthesis+mutant+ga1-3+of+Arabidopsis+thaliana+is+responsive+to+vernalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+gibberellic+acid+biosynthesis+mutant+ga1-3+of+Arabidopsis+thaliana+is+responsive+to+vernalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001838-Fu1" id="pone.0001838-Fu1"></a>Fu XD, Harberd NP (2003) Auxin promotes Arabidopsis root growth by modulating gibberellin response. Nature  421: 740–743.  <ul class="find" data-citedArticleID="1078064" data-doi="10.1038/nature01387"><li><a href="http://dx.doi.org/10.1038/nature01387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Auxin+promotes+Arabidopsis+root+growth+by+modulating+gibberellin+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Auxin+promotes+Arabidopsis+root+growth+by+modulating+gibberellin+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001838-Korves1" id="pone.0001838-Korves1"></a>Korves TM, Schmid KJ, Caicedo AL, Mays C, Stinchcombe JR, et al.  (2007) Fitness Effects Associated with the Major Flowering Time Gene FRIGIDA in <em>Arabidopsis thaliana</em> in the Field American Naturalist 169: E141–E157.  <ul class="find" data-citedArticleID="1078108" data-doi="10.1086/513111"><li><a href="http://dx.doi.org/10.1086/513111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+Effects+Associated+with+the+Major+Flowering+Time+Gene+FRIGIDA+in+Arabidopsis+thaliana+in+the+Field+American+Naturalist" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+Effects+Associated+with+the+Major+Flowering+Time+Gene+FRIGIDA+in+Arabidopsis+thaliana+in+the+Field+American+Naturalist%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001838-Scarcelli1" id="pone.0001838-Scarcelli1"></a>Scarcelli N, Cheverud JM, Schaal BA, Kover PX (2007) Antagonistic pleiotropic effects reduce the potential adaptive value of the FRIGIDA locus PNAS 10.1073/pnas.0708209104  <ul class="find" data-citedArticleID="1078158" data-doi="10.1073/pnas.0708209104"><li><a href="http://dx.doi.org/10.1073/pnas.0708209104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antagonistic+pleiotropic+effects+reduce+the+potential+adaptive+value+of+the+FRIGIDA+locus+PNAS+10.1073%2Fpnas.0708209104" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antagonistic+pleiotropic+effects+reduce+the+potential+adaptive+value+of+the+FRIGIDA+locus+PNAS+10.1073%2Fpnas.0708209104%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001838-Ehrenreich1" id="pone.0001838-Ehrenreich1"></a>Ehrenreich IM, Purugganan MD (2006) The molecular genetic basis of plant adaptation. American Journal of Botany  93: 953–962.  <ul class="find" data-citedArticleID="1078058" data-doi="10.3732/ajb.93.7.953"><li><a href="http://dx.doi.org/10.3732/ajb.93.7.953" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+genetic+basis+of+plant+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+genetic+basis+of+plant+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001838-Bowers1" id="pone.0001838-Bowers1"></a>Bowers JE, Chapman BA, Rong JK, Paterson AH (2003) Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events. Nature  422: 433–438.  <ul class="find" data-citedArticleID="1078040" data-doi="10.1038/nature01521"><li><a href="http://dx.doi.org/10.1038/nature01521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unravelling+angiosperm+genome+evolution+by+phylogenetic+analysis+of+chromosomal+duplication+events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unravelling+angiosperm+genome+evolution+by+phylogenetic+analysis+of+chromosomal+duplication+events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001838-Navarro1" id="pone.0001838-Navarro1"></a>Navarro L, Dunoyer P, Jay F, Arnold B, Dharmasiri N, et al.  (2006) A plant miRNA contributes to antibacterial resistance by repressing auxin signaling. Science  312: 436–439.  <ul class="find" data-citedArticleID="1078148" data-doi="10.1126/science.1126088"><li><a href="http://dx.doi.org/10.1126/science.1126088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+plant+miRNA+contributes+to+antibacterial+resistance+by+repressing+auxin+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+plant+miRNA+contributes+to+antibacterial+resistance+by+repressing+auxin+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001838-Textor2" id="pone.0001838-Textor2"></a>Textor S, Bartram S, Kroymann J, Falk KL, Hick A, et al.  (2004) Biosynthesis of methionine-derived glucosinolates in <em>Arabidopsis thaliana</em>: recombinant expression and characterization of methylthioalkylmalate synthase, the condensing enzyme of the chain-elongation cycle. Planta  218: 1026–1035.  <ul class="find" data-citedArticleID="1078170" data-doi="10.1007/s00425-003-1184-3"><li><a href="http://dx.doi.org/10.1007/s00425-003-1184-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+of+methionine-derived+glucosinolates+in+Arabidopsis+thaliana%3A+recombinant+expression+and+characterization+of+methylthioalkylmalate+synthase%2C+the+condensing+enzyme+of+the+chain-elongation+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+of+methionine-derived+glucosinolates+in+Arabidopsis+thaliana%3A+recombinant+expression+and+characterization+of+methylthioalkylmalate+synthase%2C+the+condensing+enzyme+of+the+chain-elongation+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001838-Mithen1" id="pone.0001838-Mithen1"></a>Mithen R, Campos H (1996) Genetic variation of aliphatic glucosinolates in <em>Arabidopsis thaliana</em> and prospects for map based gene cloning. Entomologia Experimentalis et Applicata  80: 202–205.  <ul class="find" data-citedArticleID="1078140" data-doi="10.1111/j.1570-7458.1996.tb00918.x"><li><a href="http://dx.doi.org/10.1111/j.1570-7458.1996.tb00918.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+of+aliphatic+glucosinolates+in+Arabidopsis+thaliana+and+prospects+for+map+based+gene+cloning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+of+aliphatic+glucosinolates+in+Arabidopsis+thaliana+and+prospects+for+map+based+gene+cloning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001838-Mithen2" id="pone.0001838-Mithen2"></a>Mithen R, Toroser D (1995) Biochemical genetics of aliphatic glucosinolates in Brassica and Arabidopsis. Seminar series  56 (Society for experimental biology): 261–275.  <ul class="find" data-citedArticleID="1078142" data-doi="10.1017/cbo9780511721809.017"><li><a href="http://dx.doi.org/10.1017/cbo9780511721809.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+genetics+of+aliphatic+glucosinolates+in+Brassica+and+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+genetics+of+aliphatic+glucosinolates+in+Brassica+and+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001838-Grubb1" id="pone.0001838-Grubb1"></a>Grubb CD, Zipp BJ, Ludwig-Muller J, Masuno MN, Molinski TF, et al.  (2004) Arabidopsis glucosyltransferase UGT74B1 functions in glucosinolate biosynthesis and auxin homeostasis. Plant Journal  40: 893–908.  <ul class="find" data-citedArticleID="1078068" data-doi="10.1111/j.1365-313x.2004.02261.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2004.02261.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+glucosyltransferase+UGT74B1+functions+in+glucosinolate+biosynthesis+and+auxin+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+glucosyltransferase+UGT74B1+functions+in+glucosinolate+biosynthesis+and+auxin+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001838-Schuster1" id="pone.0001838-Schuster1"></a>Schuster J, Knill T, Reichelt M, Gershenzon J, Binder S (2006) BRANCHED-CHAIN AMINOTRANSFERASE4 is part of the chain elongation pathway in the biosynthesis of methionine-derived glucosinolates in Arabidopsis. Plant Cell  18: 2664–2679.  <ul class="find" data-citedArticleID="1078162" data-doi="10.1105/tpc.105.039339"><li><a href="http://dx.doi.org/10.1105/tpc.105.039339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BRANCHED-CHAIN+AMINOTRANSFERASE4+is+part+of+the+chain+elongation+pathway+in+the+biosynthesis+of+methionine-derived+glucosinolates+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BRANCHED-CHAIN+AMINOTRANSFERASE4+is+part+of+the+chain+elongation+pathway+in+the+biosynthesis+of+methionine-derived+glucosinolates+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001838-Wentzell1" id="pone.0001838-Wentzell1"></a>Wentzell AM, Rowe HC, Hansen BG, Ticconi C, Halkier BA, et al.  (2007) Linking Metabolic QTL with Network and <em>cis</em>-eQTL Controlling Biosynthetic Pathways. PLOS Genetics  3: e162.  <ul class="find" data-citedArticleID="1078180" data-doi="10.1371/journal.pgen.0030162.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030162.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linking+Metabolic+QTL+with+Network+and+cis-eQTL+Controlling+Biosynthetic+Pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linking+Metabolic+QTL+with+Network+and+cis-eQTL+Controlling+Biosynthetic+Pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001838-Mauricio1" id="pone.0001838-Mauricio1"></a>Mauricio R (1998) Costs of resistance to natural enemies in field populations of the annual plant Arabidopsis thaliana. American Naturalist  151: 20–28.  <ul class="find" data-citedArticleID="1078130" data-doi="10.1086/286099"><li><a href="http://dx.doi.org/10.1086/286099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Costs+of+resistance+to+natural+enemies+in+field+populations+of+the+annual+plant+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Costs+of+resistance+to+natural+enemies+in+field+populations+of+the+annual+plant+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001838-Mauricio2" id="pone.0001838-Mauricio2"></a>Mauricio R, Rausher MD (1997) Experimental manipulation of putative selective agents provides evidence for the role of natural enemies in the evolution of plant defense. Evolution  51: 1435–1444.  <ul class="find" data-citedArticleID="1078132" data-doi="10.2307/2411196"><li><a href="http://dx.doi.org/10.2307/2411196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+manipulation+of+putative+selective+agents+provides+evidence+for+the+role+of+natural+enemies+in+the+evolution+of+plant+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+manipulation+of+putative+selective+agents+provides+evidence+for+the+role+of+natural+enemies+in+the+evolution+of+plant+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001838-Espiritu1" id="pone.0001838-Espiritu1"></a>Espiritu DJD, Watkins M, Dia-Monje V, Cartier GE, Cruz LJ, et al.  (2001) Venomous cone snails: molecular phylogeny and the generation of toxin diversity. Toxicon  39: 1899–1916.  <ul class="find" data-citedArticleID="1078062" data-doi="10.1016/s0041-0101(01)00175-1"><li><a href="http://dx.doi.org/10.1016/s0041-0101(01)00175-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Venomous+cone+snails%3A+molecular+phylogeny+and+the+generation+of+toxin+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Venomous+cone+snails%3A+molecular+phylogeny+and+the+generation+of+toxin+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001838-Noel1" id="pone.0001838-Noel1"></a>Noel L, Moores TL, van der Biezen EA, Parniske M, Daniels MJ, et al.  (1999) Pronounced intraspecific haplotype divergence at the RPP5 complex disease resistance locus of Arabidopsis. Plant Cell  11: 2099–2111.  <ul class="find" data-citedArticleID="1078150" data-doi="10.2307/3871012"><li><a href="http://dx.doi.org/10.2307/3871012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pronounced+intraspecific+haplotype+divergence+at+the+RPP5+complex+disease+resistance+locus+of+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pronounced+intraspecific+haplotype+divergence+at+the+RPP5+complex+disease+resistance+locus+of+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001838-Nafisi1" id="pone.0001838-Nafisi1"></a>Nafisi M, Goregaoker S, Botanga CJ, Glawischnig E, Olsen CE, et al.  (2007) Arabidopsis Cytochrome P450 Monooxygenase 71A13 Catalyzes the Conversion of Indole-3-Acetaldoxime in Camalexin Synthesis. Plant Cell  19: 2039–2052.  <ul class="find" data-citedArticleID="1078146" data-doi="10.1105/tpc.107.051383"><li><a href="http://dx.doi.org/10.1105/tpc.107.051383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+Cytochrome+P450+Monooxygenase+71A13+Catalyzes+the+Conversion+of+Indole-3-Acetaldoxime+in+Camalexin+Synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+Cytochrome+P450+Monooxygenase+71A13+Catalyzes+the+Conversion+of+Indole-3-Acetaldoxime+in+Camalexin+Synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001838-Barth1" id="pone.0001838-Barth1"></a>Barth C, Jander G (2006) Arabidopsis myrosinases TGG1 and TGG2 have redundant function in glucosinolate breakdown and insect defense. Plant Journal  46: 549–562.  <ul class="find" data-citedArticleID="1078026" data-doi="10.1111/j.1365-313x.2006.02716.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02716.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+myrosinases+TGG1+and+TGG2+have+redundant+function+in+glucosinolate+breakdown+and+insect+defense." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+myrosinases+TGG1+and+TGG2+have+redundant+function+in+glucosinolate+breakdown+and+insect+defense.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001838-Zhang3" id="pone.0001838-Zhang3"></a>Zhang J, Pontoppidan B, Xue J, Rask L, Meijer J (2002) The third myrosinase gene TGG3 in <em>Arabidopsis thaliana</em> is a pseudogene specifically expressed in stamen and petal. Physiologia Plantarum  115: 25–34.  <ul class="find" data-citedArticleID="1078192" data-doi="10.1034/j.1399-3054.2002.1150103.x"><li><a href="http://dx.doi.org/10.1034/j.1399-3054.2002.1150103.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+third+myrosinase+gene+TGG3+in+Arabidopsis+thaliana+is+a+pseudogene+specifically+expressed+in+stamen+and+petal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+third+myrosinase+gene+TGG3+in+Arabidopsis+thaliana+is+a+pseudogene+specifically+expressed+in+stamen+and+petal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001838-Burow1" id="pone.0001838-Burow1"></a>Burow M, Markert J, Gershenzon J, Wittstock U (2006) Comparative biochemical characterization of nitrile-forming proteins from plants and insects that alter myrosinase-catalysed hydrolysis of glucosinolates. Febs Journal  273: 2432–2446.  <ul class="find" data-citedArticleID="1078044" data-doi="10.1111/j.1742-4658.2006.05252.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2006.05252.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+biochemical+characterization+of+nitrile-forming+proteins+from+plants+and+insects+that+alter+myrosinase-catalysed+hydrolysis+of+glucosinolates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+biochemical+characterization+of+nitrile-forming+proteins+from+plants+and+insects+that+alter+myrosinase-catalysed+hydrolysis+of+glucosinolates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001838-Hansen3" id="pone.0001838-Hansen3"></a>Hansen CH, Du LC, Naur P, Olsen CE, Axelsen KB, et al.  (2001) CYP83B1 is the oxime-metabolizing enzyme in the glucosinolate pathway in Arabidopsis. J Biol Chem  276: 24790–24796.  <ul class="find" data-citedArticleID="1078082" data-doi="10.1074/jbc.m102637200"><li><a href="http://dx.doi.org/10.1074/jbc.m102637200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CYP83B1+is+the+oxime-metabolizing+enzyme+in+the+glucosinolate+pathway+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CYP83B1+is+the+oxime-metabolizing+enzyme+in+the+glucosinolate+pathway+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001838-Klein1" id="pone.0001838-Klein1"></a>Klein M, Reichelt M, Gershenzon J, Papenbrock J (2006) The three desulfoglucosinolate sulfotransferase proteins in Arabidopsis have different substrate specificities and are differentially expressed. Febs Journal  273: 122–136.  <ul class="find" data-citedArticleID="1078092" data-doi="10.1111/j.1742-4658.2005.05048.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2005.05048.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+three+desulfoglucosinolate+sulfotransferase+proteins+in+Arabidopsis+have+different+substrate+specificities+and+are+differentially+expressed." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+three+desulfoglucosinolate+sulfotransferase+proteins+in+Arabidopsis+have+different+substrate+specificities+and+are+differentially+expressed.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001838-Hemm1" id="pone.0001838-Hemm1"></a>Hemm MR, Ruegger MO, Chapple C (2003) The Arabidopsis ref2 mutant is defective in the gene encoding CYP83A1 and shows both phenylpropanoid and glucosinolate phenotypes. Plant Cell  15: 179–194.  <ul class="find" data-citedArticleID="1078084" data-doi="10.1105/tpc.006544"><li><a href="http://dx.doi.org/10.1105/tpc.006544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+ref2+mutant+is+defective+in+the+gene+encoding+CYP83A1+and+shows+both+phenylpropanoid+and+glucosinolate+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+ref2+mutant+is+defective+in+the+gene+encoding+CYP83A1+and+shows+both+phenylpropanoid+and+glucosinolate+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001838-Mikkelsen2" id="pone.0001838-Mikkelsen2"></a>Mikkelsen MD, Naur P, Halkier BA (2004) Arabidopsis mutants in the C-S lyase of glucosinolate biosynthesis establish a critical role for indole-3-acetaldoxime in auxin homeostasis. Plant Journal  37: 770–777.  <ul class="find" data-citedArticleID="1078138" data-doi="10.1111/j.1365-313x.2004.02002.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2004.02002.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arabidopsis+mutants+in+the+C-S+lyase+of+glucosinolate+biosynthesis+establish+a+critical+role+for+indole-3-acetaldoxime+in+auxin+homeostasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arabidopsis+mutants+in+the+C-S+lyase+of+glucosinolate+biosynthesis+establish+a+critical+role+for+indole-3-acetaldoxime+in+auxin+homeostasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.XML" value="151803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.PDF" value="712672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t001.PNG_L" value="47021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t001.PNG_M" value="28297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t001.PNG_S" value="7709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t001.TIF" value="258626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t001.PNG_I" value="8710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t002.PNG_L" value="48464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t002.PNG_M" value="28275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t002.PNG_S" value="7681"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t002.TIF" value="259560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.t002.PNG_I" value="8711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g001.PNG_L" value="118417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g001.PNG_M" value="71100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g001.PNG_S" value="8481"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g001.TIF" value="373180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g001.PNG_I" value="46807"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g002.PNG_L" value="94472"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g002.PNG_M" value="54557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g002.PNG_S" value="5316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g002.TIF" value="354608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g002.PNG_I" value="22335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g003.PNG_L" value="190819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g003.PNG_M" value="91868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g003.PNG_S" value="6478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g003.TIF" value="479724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g003.PNG_I" value="34785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g004.PNG_L" value="782370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g004.PNG_M" value="89186"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g004.PNG_S" value="11322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g004.TIF" value="1626926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g004.PNG_I" value="51297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g005.PNG_L" value="157788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g005.PNG_M" value="75321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g005.PNG_S" value="5149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g005.TIF" value="340110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g005.PNG_I" value="27759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g006.PNG_L" value="438021"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g006.PNG_M" value="53418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g006.PNG_S" value="4317"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g006.TIF" value="799752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g006.PNG_I" value="20549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g007.PNG_L" value="290546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g007.PNG_M" value="128871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g007.PNG_S" value="6804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g007.TIF" value="502134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.g007.PNG_I" value="48860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.s001.XLS" value="8644608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.s002.XLS" value="195584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.s003.XLS" value="64000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.s004.XLS" value="37888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001838.s005.XLS" value="46592"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001838&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001838" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001838&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001838&volume=&issue=&title=A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism&author_name=Daniel%20J.%20Kliebenstein&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001838" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838&amp;t=A%20Role%20for%20Gene%20Duplication%20and%20Natural%20Variation%20of%20Gene%20Expression%20in%20the%20Evolution%20of%20Metabolism" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838&title=A%20Role%20for%20Gene%20Duplication%20and%20Natural%20Variation%20of%20Gene%20Expression%20in%20the%20Evolution%20of%20Metabolism&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838&amp;title=A%20Role%20for%20Gene%20Duplication%20and%20Natural%20Variation%20of%20Gene%20Expression%20in%20the%20Evolution%20of%20Metabolism" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001838&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Role for Gene Duplication and Natural Variation of Gene Expression in the Evolution of Metabolism';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001838';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Role%20for%20Gene%20Duplication%20and%20Natural%20Variation%20of%20Gene%20Expression%20in%20the%20Evolution%20of%20Metabolism http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001838" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001838" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="26500"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Duplicated+genes%22" title="Search for articles in the subject area:'Duplicated genes'"><div class="flagText">Duplicated genes</div></a>
              <div data-categoryid="35271" data-articleid="26500"
                   data-categoryname="Duplicated genes"
                   class="flagImage" title="Flag 'Duplicated genes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+duplication%22" title="Search for articles in the subject area:'Gene duplication'"><div class="flagText">Gene duplication</div></a>
              <div data-categoryid="17929" data-articleid="26500"
                   data-categoryname="Gene duplication"
                   class="flagImage" title="Flag 'Gene duplication' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26500"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="26500"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Metabolic+pathways%22" title="Search for articles in the subject area:'Metabolic pathways'"><div class="flagText">Metabolic pathways</div></a>
              <div data-categoryid="20601" data-articleid="26500"
                   data-categoryname="Metabolic pathways"
                   class="flagImage" title="Flag 'Metabolic pathways' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26500"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="26500"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6779'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2349'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8700&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
