

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Gene-Centric Characteristics of Genome-Wide Association Studies</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001262"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001262" />

    <meta name="citation_title" content="Gene-Centric Characteristics of Genome-Wide Association Studies"/>
    <meta itemprop="name" content="Gene-Centric Characteristics of Genome-Wide Association Studies"/>

      <meta name="citation_author" content="Changzheng Dong"/>
            <meta name="citation_author_institution" content="Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
            <meta name="citation_author_institution" content="Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
            <meta name="citation_author_institution" content="Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Ziliang Qian"/>
            <meta name="citation_author_institution" content="Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
            <meta name="citation_author_institution" content="Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Peilin Jia"/>
            <meta name="citation_author_institution" content="Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
            <meta name="citation_author_institution" content="Graduate School of the Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Ying Wang"/>
            <meta name="citation_author_institution" content="Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
      <meta name="citation_author" content="Wei Huang"/>
            <meta name="citation_author_institution" content="Chinese National Human Genome Center at Shanghai, Shanghai, China"/>
            <meta name="citation_author_institution" content="Rui Jin Hospital, School of Medicine, Shanghai Jiaotong University, Shanghai, China"/>
      <meta name="citation_author" content="Yixue Li"/>
            <meta name="citation_author_institution" content="Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China"/>
            <meta name="citation_author_institution" content="Shanghai Center for Bioinformation Technology, Shanghai, China"/>

    <meta name="citation_date" content="2007/12/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001262.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1262"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome-Wide Association Analysis Identifies Loci for Type 2 Diabetes and Triglyceride Levels.; citation_author=R Saxena; citation_author=BF Voight; citation_author=V Lyssenko; citation_author=NP Burtt; citation_journal_title=Science; citation_volume=316; citation_number=1; citation_pages=1331-1336; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A genome-wide association study identifies novel risk loci for type 2 diabetes.; citation_author=R Sladek; citation_author=G Rocheleau; citation_author=J Rung; citation_author=C Dina; citation_author=L Shen; citation_journal_title=Nature; citation_volume=445; citation_number=2; citation_pages=881-885; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A variant in CDKAL1 influences insulin response and risk of type 2 diabetes.; citation_author=V Steinthorsdottir; citation_author=G Thorleifsson; citation_author=I Reynisdottir; citation_author=R Benediktsson; citation_author=T Jonsdottir; citation_journal_title=Nat Genet; citation_volume=39; citation_number=3; citation_pages=770-775; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Future of Genetic Studies of Complex Human Diseases.; citation_author=N Risch; citation_author=K Merikangas; citation_journal_title=Science; citation_volume=273; citation_number=4; citation_pages=1516-1517; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Variations on a Theme: Cataloging Human DNA Sequence Variation.; citation_author=FS Collins; citation_author=MS Guyer; citation_author=A Chakravarti; citation_journal_title=Science; citation_volume=278; citation_number=5; citation_pages=1580-1581; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association studies for common diseases and complex traits.; citation_author=J Hirschhorn; citation_author=M Daly; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=6; citation_pages=95-108; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide association studies: theoretical and practical concerns.; citation_author=WYS Wang; citation_author=BJ Barratt; citation_author=DG Clayton; citation_author=JA Todd; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=7; citation_pages=109-118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluating coverage of genome-wide association studies.; citation_author=JC Barrett; citation_author=LR Cardon; citation_journal_title=Nat Genet; citation_volume=38; citation_number=8; citation_pages=659-662; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evaluating and improving power in whole-genome association studies using fixed marker sets.; citation_author=I Pe'er; citation_author=PIW de Bakker; citation_author=J Maller; citation_author=R Yelensky; citation_author=D Altshuler; citation_journal_title=Nat Genet; citation_volume=38; citation_number=9; citation_pages=663-667; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Coverage and Characteristics of the Affymetrix GeneChip Human Mapping 100K SNP Set.; citation_author=DL Nicolae; citation_author=X Wen; citation_author=BF Voight; citation_author=NJ Cox; citation_journal_title=PLoS Genetics; citation_volume=2; citation_number=10; citation_pages=e67; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A gene-centric approach to genome-wide association studies.; citation_author=E Jorgenson; citation_author=JS Witte; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=11; citation_pages=885-891; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A genome-wide scalable SNP genotyping assay using microarray technology.; citation_author=KL Gunderson; citation_author=FJ Steemers; citation_author=G Lee; citation_author=LG Mendoza; citation_author=MS Chee; citation_journal_title=Nat Genet; citation_volume=37; citation_number=12; citation_pages=549-554; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays.; citation_author=H Matsuzaki; citation_author=S Dong; citation_author=H Loi; citation_author=X Di; citation_author=G Liu; citation_journal_title=Nat Meth; citation_volume=1; citation_number=13; citation_pages=109-111; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=14; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=15; " />
      <meta name="citation_reference" content="citation_title=Gene Ontology: tool for the unification of biology.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=16; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=From genomics to chemical genomics: new developments in KEGG.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=M Hattori; citation_author=KF Aoki-Kinoshita; citation_author=M Itoh; citation_journal_title=Nucl Acids Res; citation_volume=34; citation_number=17; citation_pages=D354-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The genetic association database.; citation_author=KG Becker; citation_author=KC Barnes; citation_author=TJ Bright; citation_author=SA Wang; citation_journal_title=Nat Genet; citation_volume=36; citation_number=18; citation_pages=431-432; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HGVbase: a human sequence variation database emphasizing data quality and a broad spectrum of data sources.; citation_author=D Fredman; citation_author=M Siegfried; citation_author=YP Yuan; citation_author=P Bork; citation_author=H Lehvaslaiho; citation_journal_title=Nucl Acids Res; citation_volume=30; citation_number=19; citation_pages=387-391; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Human Gene Mutation Database (HGMD): 2003 update.; citation_author=PD Stenson; citation_author=EV Ball; citation_author=M Mort; citation_author=AD Phillips; citation_author=JA Shiel; citation_journal_title=Hum Mutat; citation_volume=21; citation_number=20; citation_pages=577-581; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=21; " />
      <meta name="citation_reference" content="citation_title=Molecular Abnormality of an Inactive Aldehyde Dehydrogenase Variant Commonly Found in Orientals.; citation_author=A Yoshida; citation_author=IY Huang; citation_author=M Ikawa; citation_journal_title=PNAS; citation_volume=81; citation_number=22; citation_pages=258-261; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=In Vitro Splicing Deficiency Induced by a C to T Mutation at Position -3 in the Intron 10 Acceptor Site of the Phenylalanine Hydroxylase Gene in a Patient with Phenylketonuria.; citation_author=J Jaruzelska; citation_author=V Abadie; citation_author=Y d'Aubenton-Carafa; citation_author=E Brody; citation_author=A Munnich; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=23; citation_pages=20370-20375; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Synthesis of {beta} -hexosaminidase in Cell-Free Translation and in Intact Fibroblasts: An Insoluble Precursor alpha Chain in a Rare Form of Tay–Sachs Disease.; citation_author=RL Proia; citation_author=EF Neufeld; citation_journal_title=PNAS; citation_volume=79; citation_number=24; citation_pages=6360-6364; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=A Regulatory SNP Causes a Human Genetic Disease by Creating a New Transcriptional Promoter.; citation_author=M De Gobbi; citation_author=V Viprakasit; citation_author=JR Hughes; citation_author=C Fisher; citation_author=VJ Buckle; citation_journal_title=Science; citation_volume=312; citation_number=25; citation_pages=1215-1217; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Annotation of cis-regulatory elements by identification, subclassification, and functional assessment of multispecies conserved sequences.; citation_author=JR Hughes; citation_author=J-F Cheng; citation_author=N Ventress; citation_author=S Prabhakar; citation_author=K Clark; citation_journal_title=PNAS; citation_volume=102; citation_number=26; citation_pages=9830-9835; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Origin recognition complex binding to a metazoan replication origin.; citation_author=AK Bielinsky; citation_author=H Blitzblau; citation_author=EL Beall; citation_author=M Ezrokhi; citation_author=HS Smith; citation_journal_title=Curr Biol; citation_volume=11; citation_number=27; citation_pages=1427-1431; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Clustering and conservation patterns of human microRNAs.; citation_author=Y Altuvia; citation_author=P Landgraf; citation_author=G Lithwick; citation_author=N Elefant; citation_author=S Pfeffer; citation_journal_title=Nucl Acids Res; citation_volume=33; citation_number=28; citation_pages=2697-2706; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluation of the RET regulatory landscape reveals the biological relevance of a HSCR-implicated enhancer.; citation_author=EA Grice; citation_author=ES Rochelle; citation_author=ED Green; citation_author=A Chakravarti; citation_author=AS McCallion; citation_journal_title=Hum Mol Genet; citation_volume=14; citation_number=29; citation_pages=3837-3845; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evaluation of regulatory potential and conservation scores for detecting cis-regulatory modules in aligned mammalian genome sequences.; citation_author=DC King; citation_author=J Taylor; citation_author=L Elnitski; citation_author=F Chiaromonte; citation_author=W Miller; citation_journal_title=Genome Res; citation_volume=15; citation_number=30; citation_pages=1051-1060; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human-zebrafish non-coding conserved elements act in vivo to regulate transcription.; citation_author=JT Shin; citation_author=JR Priest; citation_author=I Ovcharenko; citation_author=A Ronco; citation_author=RK Moore; citation_journal_title=Nucl Acids Res; citation_volume=33; citation_number=31; citation_pages=5437-5445; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Highly Conserved Non-Coding Sequences Are Associated with Vertebrate Development.; citation_author=A Woolfe; citation_author=M Goodson; citation_author=DK Goode; citation_author=P Snell; citation_author=GK McEwen; citation_journal_title=PLoS Biology; citation_volume=3; citation_number=32; citation_pages=e7; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The value of gene-based selection of tag SNPs in genome-wide association studies.; citation_author=S Wiltshire; citation_author=PIW de Bakker; citation_author=MJ Daly; citation_journal_title=Eur J Hum Genet; citation_volume=14; citation_number=33; citation_pages=1209-1214; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=GenMAPP, a new tool for viewing and analyzing microarray data on biological pathways.; citation_author=KD Dahlquist; citation_author=N Salomonis; citation_author=K Vranizan; citation_author=SC Lawlor; citation_author=BR Conklin; citation_journal_title=Nat Genet; citation_volume=31; citation_number=34; citation_pages=19-20; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A Protein Interaction Map of Drosophila melanogaster.; citation_author=L Giot; citation_author=JS Bader; citation_author=C Brouwer; citation_author=A Chaudhuri; citation_author=B Kuang; citation_journal_title=Science; citation_volume=302; citation_number=35; citation_pages=1727-1736; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A Human Protein-Protein Interaction Network: A Resource for Annotating the Proteome.; citation_author=U Stelzl; citation_author=U Worm; citation_author=M Lalowski; citation_author=C Haenig; citation_author=FH Brembeck; citation_journal_title=Cell; citation_volume=122; citation_number=36; citation_pages=957-968; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Interaction between the melanocortin-1 receptor and P genes contributes to inter-individual variation in skin pigmentation phenotypes in a Tibetan population.; citation_author=J Akey; citation_author=H Wang; citation_author=M Xiong; citation_author=H Wu; citation_author=W Liu; citation_journal_title=Hum Genet; citation_volume=108; citation_number=37; citation_pages=516-520; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Combined Effects of Thrombosis Pathway Gene Variants Predict Cardiovascular Events.; citation_author=K Auro; citation_author=M Alanne; citation_author=K Kristiansson; citation_author=K Silander; citation_author=K Kuulasmaa; citation_journal_title=PLoS Genetics; citation_volume=3; citation_number=38; citation_pages=e120; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A Genomic Pathway Approach to a Complex Disease: Axon Guidance and Parkinson Disease.; citation_author=TG Lesnick; citation_author=S Papapetropoulos; citation_author=DC Mash; citation_author=J Ffrench-Mullen; citation_author=L Shehadeh; citation_journal_title=PLoS Genetics; citation_volume=3; citation_number=39; citation_pages=e98; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium sharing and haplotype-tagged SNP portability between populations.; citation_author=W Huang; citation_author=Y He; citation_author=H Wang; citation_author=Y Wang; citation_author=Y Liu; citation_journal_title=Proceedings of the National Academy of Sciences; citation_volume=103; citation_number=40; citation_pages=1418-1421; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The portability of tagSNPs across populations: A worldwide survey.; citation_author=A Gonzalez-Neira; citation_author=X Ke; citation_author=O Lao; citation_author=F Calafell; citation_author=A Navarro; citation_journal_title=Genome Res; citation_volume=16; citation_number=41; citation_pages=323-330; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tag SNP selection for Finnish individuals based on the CEPH Utah HapMap database.; citation_author=CJ Willer; citation_author=LJ Scott; citation_author=LL Bonnycastle; citation_author=AU Jackson; citation_author=P Chines; citation_journal_title=Genetic Epidemiology; citation_volume=30; citation_number=42; citation_pages=180-190; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=How well do HapMap SNPs capture the untyped SNPs?; citation_author=E Tantoso; citation_author=Y Yang; citation_author=K-B Li; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=43; citation_pages=238; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium.; citation_author=CS Carlson; citation_author=MA Eberle; citation_author=MJ Rieder; citation_author=Q Yi; citation_author=L Kruglyak; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=44; citation_pages=106-120; citation_date=2004; " />
      <meta name="citation_reference" content="citation_number=45; " />
      <meta name="citation_reference" content="citation_number=46; " />
      <meta name="citation_reference" content="citation_number=47; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=M Hou; citation_journal_title=Genome Res; citation_volume=15; citation_number=48; citation_pages=1034-1050; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Gene-Centric Characteristics of Genome-Wide Association Studies"/>
    <meta name="twitter:description" content="BackgroundThe high-throughput genotyping chips have contributed greatly to genome-wide association (GWA) studies to identify novel disease susceptibility single nucleotide polymorphisms (SNPs). The high-density chips are designed using two different SNP selection approaches, the direct gene-centric approach, and the indirect quasi-random SNPs or linkage disequilibrium (LD)-based tagSNPs approaches. Although all these approaches can provide high genome coverage and ascertain variants in genes, it is not clear to which extent these approaches could capture the common genic variants. It is also important to characterize and compare the differences between these approaches.Methodology/Principal FindingsIn our study, by using both the Phase II HapMap data and the disease variants extracted from OMIM, a gene-centric evaluation was first performed to evaluate the ability of the approaches in capturing the disease variants in Caucasian population. Then the distribution patterns of SNPs were also characterized in genic regions, evolutionarily conserved introns and nongenic regions, ontologies and pathways. The results show that, no mater which SNP selection approach is used, the current high-density SNP chips provide very high coverage in genic regions and can capture most of known common disease variants under HapMap frame. The results also show that the differences between the direct and the indirect approaches are relatively small. Both have similar SNP distribution patterns in these gene-centric characteristics.Conclusions/SignificanceThis study suggests that the indirect approaches not only have the advantage of high coverage but also are useful for studies focusing on various functional SNPs either in genes or in the conserved regions that the direct approach supports. The study and the annotation of characteristics will be helpful for designing and analyzing GWA studies that aim to identify genetic risk factors involved in common diseases, especially variants in genes and conserved regions."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001262.g005"/>

  <meta property="og:title" content="Gene-Centric Characteristics of Genome-Wide Association Studies" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9903'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=126'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=787&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001262">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001262" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001262&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Gene-Centric Characteristics of Genome-Wide Association Studies
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Changzheng Dong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 
                  Chinese National Human Genome Center at Shanghai, Shanghai, China, 
                  Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ziliang Qian, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 
                  Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peilin Jia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 
                  Graduate School of the Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ying Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Chinese National Human Genome Center at Shanghai, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wei Huang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:huangwei@chgc.sh.cn">huangwei@chgc.sh.cn</a> (WH); <a href="mailto:yxli@sibs.ac.cn">yxli@sibs.ac.cn</a> (YL)</p>

                <p>Affiliations:
                  Chinese National Human Genome Center at Shanghai, Shanghai, China, 
                  Rui Jin Hospital, School of Medicine, Shanghai Jiaotong University, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yixue Li
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:huangwei@chgc.sh.cn">huangwei@chgc.sh.cn</a> (WH); <a href="mailto:yxli@sibs.ac.cn">yxli@sibs.ac.cn</a> (YL)</p>

                <p>Affiliations:
                  Bioinformatics Center, Key Lab of Systems Biology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 
                  Shanghai Center for Bioinformation Technology, Shanghai, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001262</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001262-g001" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001262-g002" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001262-g003" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001262-g004" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001262-t001" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001262-g005" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001262">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001262" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The high-throughput genotyping chips have contributed greatly to genome-wide association (GWA) studies to identify novel disease susceptibility single nucleotide polymorphisms (SNPs). The high-density chips are designed using two different SNP selection approaches, the direct gene-centric approach, and the indirect quasi-random SNPs or linkage disequilibrium (LD)-based tagSNPs approaches. Although all these approaches can provide high genome coverage and ascertain variants in genes, it is not clear to which extent these approaches could capture the common genic variants. It is also important to characterize and compare the differences between these approaches.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In our study, by using both the Phase II HapMap data and the disease variants extracted from OMIM, a gene-centric evaluation was first performed to evaluate the ability of the approaches in capturing the disease variants in Caucasian population. Then the distribution patterns of SNPs were also characterized in genic regions, evolutionarily conserved introns and nongenic regions, ontologies and pathways. The results show that, no mater which SNP selection approach is used, the current high-density SNP chips provide very high coverage in genic regions and can capture most of known common disease variants under HapMap frame. The results also show that the differences between the direct and the indirect approaches are relatively small. Both have similar SNP distribution patterns in these gene-centric characteristics.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study suggests that the indirect approaches not only have the advantage of high coverage but also are useful for studies focusing on various functional SNPs either in genes or in the conserved regions that the direct approach supports. The study and the annotation of characteristics will be helpful for designing and analyzing GWA studies that aim to identify genetic risk factors involved in common diseases, especially variants in genes and conserved regions.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Dong C, Qian Z, Jia P, Wang Y, Huang W, et al.  (2007) Gene-Centric Characteristics of Genome-Wide Association Studies. PLoS ONE 2(12):
          e1262.
            doi:10.1371/journal.pone.0001262</p><p><strong>Academic Editor: </strong>Katrina Gwinn, National Institute of Neurological Disorders and Stroke, United States of America</p><p><strong>Received:</strong> September 2, 2007; <strong>Accepted:</strong> October 12, 2007; <strong>Published:</strong> December 5, 2007</p><p><strong>Copyright:</strong> © 2007 Dong et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>YL was supported by grants from National Basic Research Program of China (2006CB910700, 2004CB720103, 2004CB518606, 2003CB715901) and National High-Tech R&amp;D Program (863)(2006AA02Z334). WH was supported by National Science Fund for Distinguished Young Scholars (30625019), the National Basic Research Program (2004CB518605), Chinese High-Tech Program (2006AA020706), and Shanghai Science and Technology Committee (06XD14015). None of these institutions play any direct or indirect role in the design and conduct of the study, in the collection, analysis, and interpretation of the data, and in the preparation, review, or approval of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Genome-wide association (GWA) studies using high-throughput single nucleotide polymorphism (SNP) chips have shown the power to identify novel disease susceptibility loci <a href="#pone.0001262-Diabetes1">[1]</a>–<a href="#pone.0001262-Steinthorsdottir1">[3]</a>. Two SNP selection approaches are proposed to design high-density chips: the direct approach and the indirect approach <a href="#pone.0001262-Risch1">[4]</a>–<a href="#pone.0001262-Wang1">[7]</a>. The direct gene-centric approach, which focuses on genetic variants in genic regions <a href="#pone.0001262-Risch1">[4]</a>, <a href="#pone.0001262-Collins1">[5]</a> , can capture putative variants directly. The indirect approach using quasi-random SNPs or LD-based tagSNPs aims to capture most of the common variants in both genic and nongenic regions <a href="#pone.0001262-Hirschhorn1">[6]</a>, <a href="#pone.0001262-Wang1">[7]</a>. It provides higher coverage of genome and explores genic variants as well as potential variants in regions outside known genes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>It is clear that both approaches can cover the genome densely either directly or through linkage disequilibrium (LD) <a href="#pone.0001262-Barrett1">[8]</a>, <a href="#pone.0001262-Peer1">[9]</a> and be successful in identifying disease variants in genes <a href="#pone.0001262-Diabetes1">[1]</a>–<a href="#pone.0001262-Steinthorsdottir1">[3]</a>. It is not clear, however, the extent to which these approaches can capture the common genic variants. Moreover, it is also important to characterize and compare differences among the approaches used in GWA studies. Nicolae et al. <a href="#pone.0001262-Nicolae1">[10]</a> investigated Affymetrix GeneChip Human Mapping 100K and found that SNPs in the set were undersampled from coding regions and oversampled from regions outside genes. Jorgenson and Witte <a href="#pone.0001262-Jorgenson1">[11]</a> evaluated the coverage of both genic and nongenic SNPs, and estimated that random and tagSNP strategy for the indirect approaches could provide lower coverage of genic SNPs than nongenic SNPs. In this study, we perform a gene-centric evaluation to characterize the above approaches used in GWA studies.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Our evaluation is performed on three whole-genome commercial chips representing the above SNP-selection approaches: Illumina Human-1 Genotyping BeadChip (Human-1, gene-centric SNP panel) <a href="#pone.0001262-Gunderson1">[12]</a>, Affymetrix GeneChip Human Mapping 500K Array Set (GeneChip 500K, quasi-random SNP panel) <a href="#pone.0001262-Matsuzaki1">[13]</a> and Illumina Human Hap550 (Hap550, LD-based tagSNP panel) <a href="#pone.0001262-Gunderson1">[12]</a>. By using both the empirical Phase II HapMap CEU data (Utah residents with ancestry from northern and western Europe) <a href="#pone.0001262-The1">[14]</a> and the disease variants extracted from OMIM <a href="#pone.0001262-Online1">[15]</a>, we evaluated the coverage of the approaches in genic regions and the ability to capture disease variants in Caucasian population. Since SNPs in diseases associated pathways and functionally important sequences (for example, genes and evolutionarily conserved region) are more attractive, we further compared the distribution patterns of SNPs with these characteristics. Our study reveals the common and different characteristics between the approaches used in GWA studies.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Evaluating coverage and capturing disease variants in genic regions</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p><a href="#pone-0001262-g001"><strong>Figure 1</strong></a> shows coverage of three high-throughput SNP chips in genic regions (MAF≥0.05, CEU). Although three chips cover about 6~25% of Phase II HapMap SNPs directly (chip), the coverage increases quickly to 37~96% when counting tagged SNPs (r<sup>2</sup>≥0.8 and r<sup>2</sup>≥0.5). Since Hap550 uses the tagSNPs selected from Phase I+II HapMap data set , it gets the coverage of near 91% (r<sup>2</sup>≥0.8) as expected. If r<sup>2</sup> threshold set to 0.5, Human-1 and GeneChip 500K also gets the coverage of 53% and 84%, respectively. On average, Hap550 and GeneChip can get additional 3~4 folds nonredundant LD SNPs; while Human-1 can get 5~8 folds.</p>
<div class="figure" id="pone-0001262-g001"><div class="img"><a name="pone-0001262-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Coverage of three high-throughput SNP chips in genic regions.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Three chips cover Phase II HapMap SNPs either directly (chip) or through linkage disequilibrium (r<sup>2</sup>≥0.8 and r<sup>2</sup>≥0.5).</p>
<span>doi:10.1371/journal.pone.0001262.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>There are totally 1338 nonredundant SNPs in OMIM that are defined as disease variants and associated with either diseases or phenotypes. Among these 1338 nonredundant SNPs, 159 of which can be mapped to the Phase II HapMap CEU data. We then evaluate the ability of SNP chips to capture 100 of 159 disease variants with MAF≥0.5. Human-1 and Hap550 can directly capture 48% and 62% of variants, respectively (<a href="#pone-0001262-g002"><strong>Figure 2A</strong></a>), whereas GeneChip can only capture 11%. Via LD-tagging, all chips can capture more than 55% (r<sup>2</sup>≥0.8) and 75% (r<sup>2</sup>≥0.5) of SNPs. According to additional 59 disease variants with MAF&lt;0.05, 75% of them are uncaptured (<a href="#pone.0001262.s003"><strong>Table S1</strong></a>). <a href="#pone-0001262-g002"><strong>Figure 2B</strong></a> shows the overlap of captured variants (r<sup>2</sup>≥0.5). There are 65 of SNPs captured by all SNP chips, whereas 6 of them are not captured by any chips. It shows that the indirect approaches have the same ability as the direct approach to capture most of common disease variants in genes in HapMap.</p>
<div class="figure" id="pone-0001262-g002"><div class="img"><a name="pone-0001262-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Disease variants in HapMap captured by three high-throughput chips.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A. Shows the percentage of disease variants captured by three chips either directly (chip) or through linkage disequilibrium (r<sup>2</sup>≥0.8 and r<sup>2</sup>≥0.5). B. Displays the overlapped results captured by three chips (r<sup>2</sup>≥0.5).</p>
<span>doi:10.1371/journal.pone.0001262.g002</span></div>

<h4>Distribution patterns of SNPs in genic regions</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>SNPs are first classified into genic or nongenic regions with the annotation of dbSNP. In the analyses, less than 45% of SNPs in GeneChip 500K and Hap550 locate in genic regions, while 72% of Human-1 lie in genic regions (<a href="#pone.0001262.s001"><strong>Figure S1</strong></a>). When counting tagged SNPs, the difference is reduced to near ten percent. SNPs in genic regions can further be classified into one of the five functional classifications: flank, utr, synonymous, nonsynonymous and intron. The distribution patterns of GeneChip 500K and Hap550 are very similar (<a href="#pone-0001262-g003"><strong>Figure 3</strong></a>): similar high proportion of intron classification and similar distribution in other classifications. Almost 90% SNPs of GeneChip 500K and Hap550 locate in intron, meanwhile 7% SNPs lie in flank region, which makes it second-rich region. Each classification of utr, synonymous and nonsynonymous carries 1~2 percent SNPs. As expected, Human-1 carries relative small proportion of SNPs in intron region (70%) and high proportions in other regions (5~10%). It highlights the pertinent gene-centric design of the functional chip. Owing to the increasing proportion of intron region in Human-1, all three chips have the same distribution pattern when including tagged SNPs (r<sup>2</sup>≥0.8).</p>
<div class="figure" id="pone-0001262-g003"><div class="img"><a name="pone-0001262-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Distribution patterns of SNPs in genic regions.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>SNPs in genic regions are annotated with five functional classifications: flank (within 2 kb 5′ or 500 bp 3′ of a gene, originally named locus in dbSNP), utr (5′ and 3′ untranslated region), synonymous (synonymous coding SNP), nonsynonymous (nonsynonymous coding SNP) and intron (including splice-site SNPs).</p>
<span>doi:10.1371/journal.pone.0001262.g003</span></div>

<h4>Distribution patterns of SNPs in GO categories and KEGG pathways</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To examine whether SNPs of Human-1, GeneChip 500K and Hap550 distribute in the same ontologies and pathways via genes, we mapped SNPs to gene ontology (GO) annotation <a href="#pone.0001262-Ashburner1">[16]</a> and Kyoto Encyclopedia of Genes and Genomes (KEGG) <a href="#pone.0001262-Kanehisa1">[17]</a> pathways. The distribution of SNPs in all GO component, function and process categories (<a href="#pone-0001262-g004"><strong>Figure 4A</strong></a>) and KEGG pathways (<a href="#pone-0001262-g004"><strong>Figure 4B</strong></a>) remained largely the same between GeneChip 500K (middle rings) and Hap550 (outer rings), and slightly different with Human-1 (inner rings). We tested the hypothesis that equal percentages of SNPs on the chips distribute in KEGG human diseases pathways (<a href="#pone-0001262-t001"><strong>Table 1</strong></a>). GeneChip 500K and Hap550 show no significant differences in all pathways except for pathogenic Escherichia coli infection pathway. However, significant differences are found between the direct and the indirect approaches. Human-1 exhibits enrichment in several pathways such as Huntington's disease and pancreatic cancer pathways, whereas GeneChip 500K and Hap550 enrich in Parkinson's disease, Dentatorubropallidoluysian atrophy (DRPLA) and non-small cell lung cancer pathways. Furthermore, the difference enlarges (<a href="#pone.0001262.s004"><strong>Table S2</strong></a>) if LD-tagged SNPs are taken into consideration. More pathways exhibit significant difference between the direct and the indirect approaches.</p>
<div class="figure" id="pone-0001262-g004"><div class="img"><a name="pone-0001262-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Distribution patterns of SNPs in ontologies and pathways.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Outer rings: SNPs on Hap550. Middle rings: SNPs on GeneChip 500K. Inner rings: SNPs on Human-1. Each section represents the number of SNPs on the chips assigned to a given GO category or KEGG pathways. A. Distribution patterns in GO cellular component, molecular function and biological process categories. B. Distribution patterns in KEGG pathways.</p>
<span>doi:10.1371/journal.pone.0001262.g004</span></div><div class="figure" id="pone-0001262-t001"><div class="img"><a name="pone-0001262-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001262.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001262.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Distribution differences of SNPs (chip) in KEGG human diseases pathways.</span></strong></p><span>doi:10.1371/journal.pone.0001262.t001</span></div>

<h4>Distribution patterns of SNPs in evolutionary conserved introns and nongenic regions</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Highly evolutionarily conserved regions across species may contain unknown genes, for example, microRNA coding genes, or regulatory elements such as cis enhancers. It is important to survey the SNPs in conserved regions both outside genes and in introns. We plot SNP percentage against conservation score of sequence base in <a href="#pone-0001262-g005"><strong>Figure 5</strong></a>. If conservation threshold is set as 0.9, about two percent SNPs of GeneChip 500K and Hap550 locate in conserved intron and nongenic bases. Due to enrichment of SNPs in conserved sequence, Human-1 contains more fractions of SNPs lying in the evolutionarily conserved bases (17.5% for nongenic regions, 6.9% for intron). When considering tagged SNPs, the difference between chips is inconspicuous (<a href="#pone.0001262.s002"><strong>Figure S2</strong></a>) and about 3% of SNPs have scores exceed conserved threshold.</p>
<div class="figure" id="pone-0001262-g005"><div class="img"><a name="pone-0001262-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001262" data-uri="info:doi/10.1371/journal.pone.0001262.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001262.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001262.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001262.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001262.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Distribution patterns of SNPs in evolutionary conserved introns and nongenic regions.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The percentage of SNPs on three chips is plotted against conservation score. Human-1 contains more fractions of SNPs lying in evolutionarily conserved bases, according to its original design.</p>
<span>doi:10.1371/journal.pone.0001262.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Various GWA studies have been performed to examine the role of common genetic variants in complex diseases and traits, taking advantage of recent advances in high-throughput SNP genotyping technologies. It has been proved that both the direct and the indirect approaches are capable of identifying disease variants in genes. For example, an intron SNP (rs7903146) and nearby SNPs in LD with it in transcription factor 7-like 2 gene (TCF7L2) gene had been replicated in several researches with different approaches <a href="#pone.0001262-Diabetes1">[1]</a>–<a href="#pone.0001262-Steinthorsdottir1">[3]</a>. In our study, we show that the current high-density SNP chips provide very high coverage in genic regions and can capture most of known common disease variants under HapMap frame, no matter which SNP selection strategy is used. Acting as a hybrid of the indirect and the direct approaches to evaluate whole-genome association, Human-1 highlights SNPs enriched in genes and evolutionarily conserved regions. Therefore, we consider it as a typical chip of direct gene-centric approach and calculate tagged SNPs in HapMap. Although the indirect approaches using quasi-random SNPs or LD-based tagSNPs focus on common variants, irrespective of their genic location, they perform as good as the direct approach in genic regions via their high density and coverage. Our evaluation of coverage and characteristics is based on the Phase II HapMap data, which is the largest catalogue of common SNPs with genotyping information till now. Since most of SNPs on three chips show concordance with HapMap SNPs, it is reasonable though not very fair to use HapMap for characterizing the chips. To simplify the procedure, we only used the pairwise aggressive algorithm and two LD thresholds (r<sup>2</sup>≥0.8 and r<sup>2</sup>≥0.5) to capture tagged SNPs in CEU. On the whole, all three chips can cover more than one half of common SNPs from HapMap in genic regions.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>It is hard to estimate the ability of the approaches to capture disease variants. Then it is available to estimate whether the known susceptibility SNPs to common diseases are covered by ongoing GWA studies using chips. There are some public databases such as OMIM <a href="#pone.0001262-Online1">[15]</a>, GAD <a href="#pone.0001262-Becker1">[18]</a>, HGV <a href="#pone.0001262-Fredman1">[19]</a>, HGMD <a href="#pone.0001262-Stenson1">[20]</a> collecting variants that lead to human diseases and phenotypes variation. However, the number of susceptibility SNPs is limited in nowadays databases. Thus, we calculate and compare the ability of the chips to capture 1338 nonredundant SNPs in OMIM that affect susceptibility of human diseases, most of which are nonsynonymous mutations. With respect to common SNPs (MAF≥0.05) HapMap genotyped, the chips perform well and can capture most of them. According to rare SNPs, they are mostly ignored by the chips in current stage. It had been suggested that a genome-wide genotyping product could be coupled with a gene-centric SNP set such as SeattleSNPs Program for Genomic Application <a href="#pone.0001262-NHLBI1">[21]</a> to improve the ability of covering rare SNPs.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Several factors besides genomic coverage and map density can affect the power of gene-centric GWA studies. One is the proportion of SNPs in functionally important genic regions and conserved noncoding sequences. Nonsynonymous coding SNPs and SNPs in promoters are most traditionally attractive for their potential altering protein function <a href="#pone.0001262-Yoshida1">[22]</a>, altering transcript splicing <a href="#pone.0001262-Jaruzelska1">[23]</a> destabilizing protein 3D structure and reduce protein solubility <a href="#pone.0001262-Proia1">[24]</a>, and altering regulatory ability <a href="#pone.0001262-DeGobbi1">[25]</a>. Evolutionarily conserved regions across species may contain functionally important elements, for example, cis- regulatory elements <a href="#pone.0001262-Hughes1">[26]</a> and replication start points <a href="#pone.0001262-Bielinsky1">[27]</a>, or unknown genes such as microRNA coding genes <a href="#pone.0001262-Altuvia1">[28]</a>. Many computational approaches based on multi-species alignment have been developed to search regulatory elements in evolutionarily conserved regions <a href="#pone.0001262-Grice1">[29]</a>, <a href="#pone.0001262-King1">[30]</a>. Follow-up experiments also validated the potential function of transcriptional regulation and development association <a href="#pone.0001262-Shin1">[31]</a>, <a href="#pone.0001262-Woolfe1">[32]</a>. It is also important to characterize the conservation property of SNPs outside genes and in introns <a href="#pone.0001262-Wiltshire1">[33]</a>. Our results show that the indirect approaches have highly similar patterns in these important function sequences. Although difference exhibits between the indirect and the direct approaches for SNPs on the chip, it becomes inconspicuous after considering tagged SNPs.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Another important factor is the enrichment of SNPs in ontologies and pathways. Ontologies and pathways are essential and widely used for differential expression in pathway level <a href="#pone.0001262-Dahlquist1">[34]</a>, protein-protein interaction (PPI) analysis <a href="#pone.0001262-Giot1">[35]</a> and constructing PPI network <a href="#pone.0001262-Stelzl1">[36]</a>. Thus, it is reliable to analyze gene-gene interaction <a href="#pone.0001262-Akey1">[37]</a> and construct genetic interaction network via SNPs-enriched ontologies and pathways <a href="#pone.0001262-Auro1">[38]</a>, <a href="#pone.0001262-Lesnick1">[39]</a>. Lesnick et al. proposed a genomic pathway approach to construct models of axon-guidance pathway SNPs that can predict the susceptibility of Parkinson disease <a href="#pone.0001262-Lesnick1">[39]</a>. It hints the potential ability of mining disease associated ontologies and pathways using high-density SNP chips. We examined whether SNPs of Human-1, GeneChip 500K and Hap550 distribute in the same ontologies and pathways via genes and tested the hypothesis that the same fraction of SNPs on the chips distribute in the KEGG human disorder pathways. On most occasions, Human-1 has the similar fractions as GeneChip 500K and Hap550. Significant differences are observed in some pathways especially when considering tagged SNPs. Since Hap550 almost cover HapMap and Human-1 contains a limited subset (<a href="#pone-0001262-g001"><strong>Figure 1</strong></a>), it implies that SNPs of Human-1 are not evenly distributing in genes and pathways.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In this year, Affymetrix (<a href="http://www.affymetrix.com">http://www.affymetrix.com</a>) and Illumina (<a href="http://www.illumina.com">http://www.illumina.com</a>) released their one million commercial SNP chips, which are most high density SNP chips available till now. We can expect that the SNP chips will cover most of the human common SNPs and density will not be a common topic in the coming future (maybe in five years). One possible future direction of developing SNP chips is population-specific chips. The current chips aim at common SNPs of three representative populations (CEU, JPT+CHB, YRI), while CEU matches the SNP sets best and YRI worst. Although YRI-specific SNP chip has been designed, the population-specific (especially the populations other than three major populations) chips are needed since various researches have shown that the portability of tagSNPs across some populations is not satisfying <a href="#pone.0001262-Huang1">[40]</a>–<a href="#pone.0001262-Tantoso1">[43]</a>. Another possible direction is developing rare SNPs-based chips, however, potential large sample size still obstructs this way. A most realistic direction is developing disease/pathway specific chips for specific researches. This is quite different with previous candidate pathway/gene studies. The future disease/pathway specific chips will have the advantages of both high density and research specificity. That means all possible disease-oriented SNPs in pathways/genes are included, which is based on the knowledge and Bioinformatics annotations of the diseases. This will separate the “discovering susceptible SNPs” stage by WGA chips and “replicating associations and constructing genetic models” stage by specific chips similarly as we did with resequencing and genotyping. This will reduce the cost and increase the sample size greatly. Thus, WGA studies era will be realistic. For the above reason, Bioinformatics will be deeply involved in the designing of the chips, analyzing the data and constructing the models. Our analysis in this research will be an exploration in this future field.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data sets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Three genome-wide SNP chips (Human-1, GeneChip 500K and Hap550) were selected for evaluations, representing gene-centric, quasi-random SNPs and LD-based tagSNPs approaches, respectively. SNP lists were downloaded from their websites. Since most of SNPs on three chips show concordance with HapMap SNPs, we used Phase II HapMap CEU (release 21) data <a href="#pone.0001262-The1">[14]</a> to evaluate coverage of SNP chips in Caucasian population.</p>


<h4>LD-tagged SNPs and coverage calculation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>All SNPs of three chips were mapped to HapMap to ensure SNPs on the chip, and these SNPs were considered as tag SNPs to capture LD-tagged SNPs. Frequency and LD data of SNPs were downloaded from HapMap website. We simply used pairwise aggressive algorithm <a href="#pone.0001262-Carlson1">[44]</a> to ascertain SNPs that have pairwise r<sup>2</sup> larger than the specific thresholds (r<sup>2</sup>≥0.8 and r<sup>2</sup>≥0.5). Since Human-1 acts as a hybrid of the indirect and the direct approaches, we also calculate its tagged SNPs to get the maximum coverage. When calculating coverage, only common SNPs (MAF≥0.05) were considered. Coverage is estimated by SNPs (chip+tagged) divided by all SNPs in HapMap.</p>


<h4>Bioinformatics annotation for SNPs</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>All SNPs were annotated using National Center for Biotechnology Information (NCBI) dbSNP (build 126) <a href="#pone.0001262-dbSNP1">[45]</a>. Each SNP from various data sets was mapped to dbSNP via ref SNP (rs). SNPs without an rs number or not presented in current dbSNP would be ignored. A SNP was first annotated in gene or nongenic regions. Then genic SNP would further be annotated with five functional classifications: flank (within 2 kb 5′ or 500 bp 3′ of a gene, originally named locus in dbSNP), utr (5′ and 3′ untranslated region), synonymous (synonymous coding SNP), nonsynonymous (nonsynonymous coding SNP) and intron (including splice-site SNPs).</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>OMIM (Nov, 2006) <a href="#pone.0001262-Online1">[15]</a> deposited 1338 nonredundant SNPs that affect human diseases or phenotypes variation, most of which are nonsynonymous mutations. These SNPs were mapped to HapMap data and evaluated the capturing ability of the SNP chips. The procedure was same as coverage calculation.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>We examined whether SNPs of Human-1, GeneChip 500K and Hap550 distribute in the same ontologies and pathways via genes. SNPs were mapped to GO annotations and KEGG pathways via NCBI dbSNP <a href="#pone.0001262-dbSNP1">[45]</a> and Entrez Gene <a href="#pone.0001262-Entrez1">[46]</a>. We plotted the distribution of SNPs in categories with three concentric rings for the chips. In addition, we compared distribution difference between the paired chips in KEGG human disorders pathways. Chi-square tests between pairwise chips were performed to test whether two chips have same percentages of SNPs in the pathways. Bonferroni correction was proceeded to correct multiple testing.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>Base-by-base conservation scores for human bases were downloaded from UCSC Conservation Track <a href="#pone.0001262-UCSC1">[47]</a> which used phastCons <a href="#pone.0001262-Siepel1">[48]</a> to calculate conservation scores across 28 mammalian species. SNPs in nongenic and intron regions were mapped to the bases to attain conservation scores. Conservation scores can be considered as probabilities that each SNP lies in a conserved element <a href="#pone.0001262-Siepel1">[48]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001262"></div><a name="pone.0001262.s001" id="pone.0001262.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001262.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Percentage of SNPs in genic and nongenic regions. Shows the percentage of disease variants captured by three chips either directly (chip) or through linkage disequilibrium (r<sup>2</sup>≥0.8).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001262.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.18 MB DOC)</p>
<a name="pone.0001262.s002" id="pone.0001262.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001262.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Distribution patterns of SNPs in evolutionary conserved introns and nongenic regions. The percentage of SNPs (r<sup>2</sup>≥0.8) on three chips is plotted against conservation score.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001262.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.31 MB DOC)</p>
<a name="pone.0001262.s003" id="pone.0001262.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001262.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">MAF of 159 disease SNPs (r<sup>2</sup>≥0.5) in HapMap</p>
<p class="siDoi">doi:10.1371/journal.pone.0001262.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001262.s004" id="pone.0001262.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001262.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Distribution differences of SNPs (r<sup>2</sup>≥0.8) in KEGG human diseases pathways. 1. Chi-square tests between pairwise chips were performed to test whether two chips have same percentages of SNPs in the pathways. Bonferroni correction was proceeded to correct multiple testing. P-values smaller than significant level (P&lt;0.002) are in bold type.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001262.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YL WH CD. Analyzed the data: YW CD PJ ZQ. Wrote the paper: YL WH CD PJ ZQ.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001262-Diabetes1" id="pone.0001262-Diabetes1"></a>Saxena R, Voight BF, Lyssenko V, Burtt NP, et al. Diabetes Genetics Initiative of Broad Institute of Harvard and Mit LUaNIoBR (2007) Genome-Wide Association Analysis Identifies Loci for Type 2 Diabetes and Triglyceride Levels. Science  316: 1331–1336.  <ul class="find" data-citedArticleID="1025796" data-doi="10.1126/science.1142358"><li><a href="http://dx.doi.org/10.1126/science.1142358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-Wide+Association+Analysis+Identifies+Loci+for+Type+2+Diabetes+and+Triglyceride+Levels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-Wide+Association+Analysis+Identifies+Loci+for+Type+2+Diabetes+and+Triglyceride+Levels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001262-Sladek1" id="pone.0001262-Sladek1"></a>Sladek R, Rocheleau G, Rung J, Dina C, Shen L, et al.  (2007) A genome-wide association study identifies novel risk loci for type 2 diabetes. Nature  445: 881–885.  <ul class="find" data-citedArticleID="1025844" data-doi="10.1038/nature05616"><li><a href="http://dx.doi.org/10.1038/nature05616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001262-Steinthorsdottir1" id="pone.0001262-Steinthorsdottir1"></a>Steinthorsdottir V, Thorleifsson G, Reynisdottir I, Benediktsson R, Jonsdottir T, et al.  (2007) A variant in CDKAL1 influences insulin response and risk of type 2 diabetes. Nat Genet  39: 770–775.  <ul class="find" data-citedArticleID="1025846" data-doi="10.1038/ng2043"><li><a href="http://dx.doi.org/10.1038/ng2043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001262-Risch1" id="pone.0001262-Risch1"></a>Risch N, Merikangas K (1996) The Future of Genetic Studies of Complex Human Diseases. Science  273: 1516–1517.  <ul class="find" data-citedArticleID="1025838" data-doi="10.1126/science.273.5281.1516"><li><a href="http://dx.doi.org/10.1126/science.273.5281.1516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Future+of+Genetic+Studies+of+Complex+Human+Diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Future+of+Genetic+Studies+of+Complex+Human+Diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001262-Collins1" id="pone.0001262-Collins1"></a>Collins FS, Guyer MS, Chakravarti A (1997) Variations on a Theme: Cataloging Human DNA Sequence Variation. Science  278: 1580–1581.  <ul class="find" data-citedArticleID="1025790" data-doi="10.1126/science.278.5343.1580"><li><a href="http://dx.doi.org/10.1126/science.278.5343.1580" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variations+on+a+Theme%3A+Cataloging+Human+DNA+Sequence+Variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variations+on+a+Theme%3A+Cataloging+Human+DNA+Sequence+Variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001262-Hirschhorn1" id="pone.0001262-Hirschhorn1"></a>Hirschhorn J, Daly M (2005) Genome-wide association studies for common diseases and complex traits. Nat Rev Genet  6: 95–108.  <ul class="find" data-citedArticleID="1025810" data-doi="10.1038/nrg1521"><li><a href="http://dx.doi.org/10.1038/nrg1521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+studies+for+common+diseases+and+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+studies+for+common+diseases+and+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001262-Wang1" id="pone.0001262-Wang1"></a>Wang WYS, Barratt BJ, Clayton DG, Todd JA (2005) Genome-wide association studies: theoretical and practical concerns. Nat Rev Genet  6: 109–118.  <ul class="find" data-citedArticleID="1025858" data-doi="10.1038/nrg1522"><li><a href="http://dx.doi.org/10.1038/nrg1522" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+association+studies%3A+theoretical+and+practical+concerns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+association+studies%3A+theoretical+and+practical+concerns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001262-Barrett1" id="pone.0001262-Barrett1"></a>Barrett JC, Cardon LR (2006) Evaluating coverage of genome-wide association studies. Nat Genet  38: 659–662.  <ul class="find" data-citedArticleID="1025782" data-doi="10.1038/ng1801"><li><a href="http://dx.doi.org/10.1038/ng1801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+coverage+of+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+coverage+of+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001262-Peer1" id="pone.0001262-Peer1"></a>Pe'er I, de Bakker PIW, Maller J, Yelensky R, Altshuler D, et al.  (2006) Evaluating and improving power in whole-genome association studies using fixed marker sets. Nat Genet  38: 663–667.  <ul class="find" data-citedArticleID="1025834" data-doi="10.1038/ng1816"><li><a href="http://dx.doi.org/10.1038/ng1816" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+and+improving+power+in+whole-genome+association+studies+using+fixed+marker+sets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+and+improving+power+in+whole-genome+association+studies+using+fixed+marker+sets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001262-Nicolae1" id="pone.0001262-Nicolae1"></a>Nicolae DL, Wen X, Voight BF, Cox NJ (2006) Coverage and Characteristics of the Affymetrix GeneChip Human Mapping 100K SNP Set. PLoS Genetics  2: e67.  <ul class="find" data-citedArticleID="1025830" data-doi="10.1371/journal.pgen.0020067.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020067.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coverage+and+Characteristics+of+the+Affymetrix+GeneChip+Human+Mapping+100K+SNP+Set." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coverage+and+Characteristics+of+the+Affymetrix+GeneChip+Human+Mapping+100K+SNP+Set.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001262-Jorgenson1" id="pone.0001262-Jorgenson1"></a>Jorgenson E, Witte JS (2006) A gene-centric approach to genome-wide association studies. Nat Rev Genet  7: 885–891.  <ul class="find" data-citedArticleID="1025818" data-doi="10.1038/nrg1962"><li><a href="http://dx.doi.org/10.1038/nrg1962" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene-centric+approach+to+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene-centric+approach+to+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001262-Gunderson1" id="pone.0001262-Gunderson1"></a>Gunderson KL, Steemers FJ, Lee G, Mendoza LG, Chee MS (2005) A genome-wide scalable SNP genotyping assay using microarray technology. Nat Genet  37: 549–554.  <ul class="find" data-citedArticleID="1025808" data-doi="10.1038/ng1547"><li><a href="http://dx.doi.org/10.1038/ng1547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome-wide+scalable+SNP+genotyping+assay+using+microarray+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome-wide+scalable+SNP+genotyping+assay+using+microarray+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001262-Matsuzaki1" id="pone.0001262-Matsuzaki1"></a>Matsuzaki H, Dong S, Loi H, Di X, Liu G, et al.  (2004) Genotyping over 100,000 SNPs on a pair of oligonucleotide arrays. Nat Meth  1: 109–111.  <ul class="find" data-citedArticleID="1025826" data-doi="10.1038/nmeth718"><li><a href="http://dx.doi.org/10.1038/nmeth718" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotyping+over+100%2C000+SNPs+on+a+pair+of+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotyping+over+100%2C000+SNPs+on+a+pair+of+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001262-The1" id="pone.0001262-The1"></a>The International HapMap C (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="1025854" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001262-Online1" id="pone.0001262-Online1"></a>  Online Mendelian Inheritance in Man, OMIM (TM) <a href="http://www.ncbi.nlm.nih.gov/omim/">http://www.ncbi.nlm.nih.gov/omim/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0001262-Ashburner1" id="pone.0001262-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene Ontology: tool for the unification of biology. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="1025778" data-doi="10.1038/75556"><li><a href="http://dx.doi.org/10.1038/75556" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+Ontology%3A+tool+for+the+unification+of+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+Ontology%3A+tool+for+the+unification+of+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001262-Kanehisa1" id="pone.0001262-Kanehisa1"></a>Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, et al.  (2006) From genomics to chemical genomics: new developments in KEGG. Nucl Acids Res  34: D354–357.  <ul class="find" data-citedArticleID="1025820" data-doi="10.1093/nar/gkj102"><li><a href="http://dx.doi.org/10.1093/nar/gkj102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomics+to+chemical+genomics%3A+new+developments+in+KEGG.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001262-Becker1" id="pone.0001262-Becker1"></a>Becker KG, Barnes KC, Bright TJ, Wang SA (2004) The genetic association database. Nat Genet  36: 431–432.  <ul class="find" data-citedArticleID="1025784" data-doi="10.1038/ng0504-431"><li><a href="http://dx.doi.org/10.1038/ng0504-431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+association+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+association+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001262-Fredman1" id="pone.0001262-Fredman1"></a>Fredman D, Siegfried M, Yuan YP, Bork P, Lehvaslaiho H, et al.  (2002) HGVbase: a human sequence variation database emphasizing data quality and a broad spectrum of data sources. Nucl Acids Res  30: 387–391.  <ul class="find" data-citedArticleID="1025800" data-doi="10.1093/nar/30.1.387"><li><a href="http://dx.doi.org/10.1093/nar/30.1.387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HGVbase%3A+a+human+sequence+variation+database+emphasizing+data+quality+and+a+broad+spectrum+of+data+sources." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HGVbase%3A+a+human+sequence+variation+database+emphasizing+data+quality+and+a+broad+spectrum+of+data+sources.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001262-Stenson1" id="pone.0001262-Stenson1"></a>Stenson PD, Ball EV, Mort M, Phillips AD, Shiel JA, et al.  (2003) Human Gene Mutation Database (HGMD): 2003 update. Hum Mutat  21: 577–581.  <ul class="find" data-citedArticleID="1025850" data-doi="10.1002/humu.10212"><li><a href="http://dx.doi.org/10.1002/humu.10212" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Gene+Mutation+Database+%28HGMD%29%3A+2003+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Gene+Mutation+Database+%28HGMD%29%3A+2003+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001262-NHLBI1" id="pone.0001262-NHLBI1"></a>  NHLBI Program for Genomic Applications,SeattleSNPs <a href="http://pga.gs.washington.edu">http://pga.gs.washington.edu</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001262-Yoshida1" id="pone.0001262-Yoshida1"></a>Yoshida A, Huang IY, Ikawa M (1984) Molecular Abnormality of an Inactive Aldehyde Dehydrogenase Variant Commonly Found in Orientals. PNAS  81: 258–261.  <ul class="find" data-citedArticleID="1025866" data-doi="10.1073/pnas.81.1.258"><li><a href="http://dx.doi.org/10.1073/pnas.81.1.258" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+Abnormality+of+an+Inactive+Aldehyde+Dehydrogenase+Variant+Commonly+Found+in+Orientals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+Abnormality+of+an+Inactive+Aldehyde+Dehydrogenase+Variant+Commonly+Found+in+Orientals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001262-Jaruzelska1" id="pone.0001262-Jaruzelska1"></a>Jaruzelska J, Abadie V, d'Aubenton-Carafa Y, Brody E, Munnich A, et al.  (1995) In Vitro Splicing Deficiency Induced by a C to T Mutation at Position -3 in the Intron 10 Acceptor Site of the Phenylalanine Hydroxylase Gene in a Patient with Phenylketonuria. J Biol Chem  270: 20370–20375.  <ul class="find" data-citedArticleID="1025816" data-doi="10.1074/jbc.270.35.20370"><li><a href="http://dx.doi.org/10.1074/jbc.270.35.20370" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+Vitro+Splicing+Deficiency+Induced+by+a+C+to+T+Mutation+at+Position+-3+in+the+Intron+10+Acceptor+Site+of+the+Phenylalanine+Hydroxylase+Gene+in+a+Patient+with+Phenylketonuria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+Vitro+Splicing+Deficiency+Induced+by+a+C+to+T+Mutation+at+Position+-3+in+the+Intron+10+Acceptor+Site+of+the+Phenylalanine+Hydroxylase+Gene+in+a+Patient+with+Phenylketonuria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001262-Proia1" id="pone.0001262-Proia1"></a>Proia RL, Neufeld EF (1982) Synthesis of {beta} -hexosaminidase in Cell-Free Translation and in Intact Fibroblasts: An Insoluble Precursor alpha Chain in a Rare Form of Tay–Sachs Disease. PNAS  79: 6360–6364.  <ul class="find" data-citedArticleID="1025836" data-doi="10.1073/pnas.79.20.6360"><li><a href="http://dx.doi.org/10.1073/pnas.79.20.6360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthesis+of+%7Bbeta%7D+-hexosaminidase+in+Cell-Free+Translation+and+in+Intact+Fibroblasts%3A+An+Insoluble+Precursor+alpha+Chain+in+a+Rare+Form+of+Tay%E2%80%93Sachs+Disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthesis+of+%7Bbeta%7D+-hexosaminidase+in+Cell-Free+Translation+and+in+Intact+Fibroblasts%3A+An+Insoluble+Precursor+alpha+Chain+in+a+Rare+Form+of+Tay%E2%80%93Sachs+Disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001262-DeGobbi1" id="pone.0001262-DeGobbi1"></a>De Gobbi M, Viprakasit V, Hughes JR, Fisher C, Buckle VJ, et al.  (2006) A Regulatory SNP Causes a Human Genetic Disease by Creating a New Transcriptional Promoter. Science  312: 1215–1217.  <ul class="find" data-citedArticleID="1025794" data-doi="10.1126/science.1126431"><li><a href="http://dx.doi.org/10.1126/science.1126431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Regulatory+SNP+Causes+a+Human+Genetic+Disease+by+Creating+a+New+Transcriptional+Promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Regulatory+SNP+Causes+a+Human+Genetic+Disease+by+Creating+a+New+Transcriptional+Promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001262-Hughes1" id="pone.0001262-Hughes1"></a>Hughes JR, Cheng J-F, Ventress N, Prabhakar S, Clark K, et al.  (2005) Annotation of cis-regulatory elements by identification, subclassification, and functional assessment of multispecies conserved sequences. PNAS  102: 9830–9835.  <ul class="find" data-citedArticleID="1025814" data-doi="10.1073/pnas.0503401102"><li><a href="http://dx.doi.org/10.1073/pnas.0503401102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Annotation+of+cis-regulatory+elements+by+identification%2C+subclassification%2C+and+functional+assessment+of+multispecies+conserved+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Annotation+of+cis-regulatory+elements+by+identification%2C+subclassification%2C+and+functional+assessment+of+multispecies+conserved+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001262-Bielinsky1" id="pone.0001262-Bielinsky1"></a>Bielinsky AK, Blitzblau H, Beall EL, Ezrokhi M, Smith HS, et al.  (2001) Origin recognition complex binding to a metazoan replication origin. Curr Biol  11: 1427–1431.  <ul class="find" data-citedArticleID="1025786" data-doi="10.1016/s0960-9822(01)00444-4"><li><a href="http://dx.doi.org/10.1016/s0960-9822(01)00444-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+recognition+complex+binding+to+a+metazoan+replication+origin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+recognition+complex+binding+to+a+metazoan+replication+origin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001262-Altuvia1" id="pone.0001262-Altuvia1"></a>Altuvia Y, Landgraf P, Lithwick G, Elefant N, Pfeffer S, et al.  (2005) Clustering and conservation patterns of human microRNAs. Nucl Acids Res  33: 2697–2706.  <ul class="find" data-citedArticleID="1025776" data-doi="10.1093/nar/gki567"><li><a href="http://dx.doi.org/10.1093/nar/gki567" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+and+conservation+patterns+of+human+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+and+conservation+patterns+of+human+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001262-Grice1" id="pone.0001262-Grice1"></a>Grice EA, Rochelle ES, Green ED, Chakravarti A, McCallion AS (2005) Evaluation of the RET regulatory landscape reveals the biological relevance of a HSCR-implicated enhancer. Hum Mol Genet  14: 3837–3845.  <ul class="find" data-citedArticleID="1025806" data-doi="10.1093/hmg/ddi408"><li><a href="http://dx.doi.org/10.1093/hmg/ddi408" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+the+RET+regulatory+landscape+reveals+the+biological+relevance+of+a+HSCR-implicated+enhancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+the+RET+regulatory+landscape+reveals+the+biological+relevance+of+a+HSCR-implicated+enhancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001262-King1" id="pone.0001262-King1"></a>King DC, Taylor J, Elnitski L, Chiaromonte F, Miller W, et al.  (2005) Evaluation of regulatory potential and conservation scores for detecting cis-regulatory modules in aligned mammalian genome sequences. Genome Res  15: 1051–1060.  <ul class="find" data-citedArticleID="1025822" data-doi="10.1101/gr.3642605"><li><a href="http://dx.doi.org/10.1101/gr.3642605" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+regulatory+potential+and+conservation+scores+for+detecting+cis-regulatory+modules+in+aligned+mammalian+genome+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+regulatory+potential+and+conservation+scores+for+detecting+cis-regulatory+modules+in+aligned+mammalian+genome+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001262-Shin1" id="pone.0001262-Shin1"></a>Shin JT, Priest JR, Ovcharenko I, Ronco A, Moore RK, et al.  (2005) Human-zebrafish non-coding conserved elements act in vivo to regulate transcription. Nucl Acids Res  33: 5437–5445.  <ul class="find" data-citedArticleID="1025840" data-doi="10.1093/nar/gki853"><li><a href="http://dx.doi.org/10.1093/nar/gki853" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human-zebrafish+non-coding+conserved+elements+act+in+vivo+to+regulate+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human-zebrafish+non-coding+conserved+elements+act+in+vivo+to+regulate+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001262-Woolfe1" id="pone.0001262-Woolfe1"></a>Woolfe A, Goodson M, Goode DK, Snell P, McEwen GK, et al.  (2005) Highly Conserved Non-Coding Sequences Are Associated with Vertebrate Development. PLoS Biology  3: e7.  <ul class="find" data-citedArticleID="1025864" data-doi="10.1371/journal.pbio.0030007"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+Conserved+Non-Coding+Sequences+Are+Associated+with+Vertebrate+Development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+Conserved+Non-Coding+Sequences+Are+Associated+with+Vertebrate+Development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001262-Wiltshire1" id="pone.0001262-Wiltshire1"></a>Wiltshire S, de Bakker PIW, Daly MJ (2006) The value of gene-based selection of tag SNPs in genome-wide association studies. Eur J Hum Genet  14: 1209–1214.  <ul class="find" data-citedArticleID="1025862" data-doi="10.1038/sj.ejhg.5201678"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201678" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+value+of+gene-based+selection+of+tag+SNPs+in+genome-wide+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+value+of+gene-based+selection+of+tag+SNPs+in+genome-wide+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001262-Dahlquist1" id="pone.0001262-Dahlquist1"></a>Dahlquist KD, Salomonis N, Vranizan K, Lawlor SC, Conklin BR (2002) GenMAPP, a new tool for viewing and analyzing microarray data on biological pathways. Nat Genet  31: 19–20.  <ul class="find" data-citedArticleID="1025792" data-doi="10.1038/ng0502-19"><li><a href="http://dx.doi.org/10.1038/ng0502-19" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GenMAPP%2C+a+new+tool+for+viewing+and+analyzing+microarray+data+on+biological+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GenMAPP%2C+a+new+tool+for+viewing+and+analyzing+microarray+data+on+biological+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001262-Giot1" id="pone.0001262-Giot1"></a>Giot L, Bader JS, Brouwer C, Chaudhuri A, Kuang B, et al.  (2003) A Protein Interaction Map of Drosophila melanogaster. Science  302: 1727–1736.  <ul class="find" data-citedArticleID="1025802" data-doi="10.1126/science.1090289"><li><a href="http://dx.doi.org/10.1126/science.1090289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Protein+Interaction+Map+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Protein+Interaction+Map+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001262-Stelzl1" id="pone.0001262-Stelzl1"></a>Stelzl U, Worm U, Lalowski M, Haenig C, Brembeck FH, et al.  (2005) A Human Protein-Protein Interaction Network: A Resource for Annotating the Proteome. Cell  122: 957–968.  <ul class="find" data-citedArticleID="1025848" data-doi="10.1016/j.cell.2005.08.029"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.08.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Human+Protein-Protein+Interaction+Network%3A+A+Resource+for+Annotating+the+Proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Human+Protein-Protein+Interaction+Network%3A+A+Resource+for+Annotating+the+Proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001262-Akey1" id="pone.0001262-Akey1"></a>Akey J, Wang H, Xiong M, Wu H, Liu W, et al.  (2001) Interaction between the melanocortin-1 receptor and P genes contributes to inter-individual variation in skin pigmentation phenotypes in a Tibetan population. Hum Genet  108: 516–520.  <ul class="find" data-citedArticleID="1025774" data-doi="10.1007/s004390100524"><li><a href="http://dx.doi.org/10.1007/s004390100524" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+between+the+melanocortin-1+receptor+and+P+genes+contributes+to+inter-individual+variation+in+skin+pigmentation+phenotypes+in+a+Tibetan+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+between+the+melanocortin-1+receptor+and+P+genes+contributes+to+inter-individual+variation+in+skin+pigmentation+phenotypes+in+a+Tibetan+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001262-Auro1" id="pone.0001262-Auro1"></a>Auro K, Alanne M, Kristiansson K, Silander K, Kuulasmaa K, et al.  (2007) Combined Effects of Thrombosis Pathway Gene Variants Predict Cardiovascular Events. PLoS Genetics  3: e120.  <ul class="find" data-citedArticleID="1025780" data-doi="10.1371/journal.pgen.0030120.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030120.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combined+Effects+of+Thrombosis+Pathway+Gene+Variants+Predict+Cardiovascular+Events." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combined+Effects+of+Thrombosis+Pathway+Gene+Variants+Predict+Cardiovascular+Events.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001262-Lesnick1" id="pone.0001262-Lesnick1"></a>Lesnick TG, Papapetropoulos S, Mash DC, Ffrench-Mullen J, Shehadeh L, et al.  (2007) A Genomic Pathway Approach to a Complex Disease: Axon Guidance and Parkinson Disease. PLoS Genetics  3: e98.  <ul class="find" data-citedArticleID="1025824" data-doi="10.1371/journal.pgen.0030098"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Genomic+Pathway+Approach+to+a+Complex+Disease%3A+Axon+Guidance+and+Parkinson+Disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Genomic+Pathway+Approach+to+a+Complex+Disease%3A+Axon+Guidance+and+Parkinson+Disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001262-Huang1" id="pone.0001262-Huang1"></a>Huang W, He Y, Wang H, Wang Y, Liu Y, et al.  (2006) Linkage disequilibrium sharing and haplotype-tagged SNP portability between populations. Proceedings of the National Academy of Sciences  103: 1418–1421.  <ul class="find" data-citedArticleID="1025812" data-doi="10.1073/pnas.0510360103"><li><a href="http://dx.doi.org/10.1073/pnas.0510360103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+sharing+and+haplotype-tagged+SNP+portability+between+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+sharing+and+haplotype-tagged+SNP+portability+between+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001262-GonzalezNeira1" id="pone.0001262-GonzalezNeira1"></a>Gonzalez-Neira A, Ke X, Lao O, Calafell F, Navarro A, et al.  (2006) The portability of tagSNPs across populations: A worldwide survey. Genome Res  16: 323–330.  <ul class="find" data-citedArticleID="1025804" data-doi="10.1101/gr.4138406"><li><a href="http://dx.doi.org/10.1101/gr.4138406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+portability+of+tagSNPs+across+populations%3A+A+worldwide+survey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+portability+of+tagSNPs+across+populations%3A+A+worldwide+survey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001262-Willer1" id="pone.0001262-Willer1"></a>Willer CJ, Scott LJ, Bonnycastle LL, Jackson AU, Chines P, et al.  (2006) Tag SNP selection for Finnish individuals based on the CEPH Utah HapMap database. Genetic Epidemiology  30: 180–190.  <ul class="find" data-citedArticleID="1025860" data-doi="10.1002/gepi.20131"><li><a href="http://dx.doi.org/10.1002/gepi.20131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tag+SNP+selection+for+Finnish+individuals+based+on+the+CEPH+Utah+HapMap+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tag+SNP+selection+for+Finnish+individuals+based+on+the+CEPH+Utah+HapMap+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001262-Tantoso1" id="pone.0001262-Tantoso1"></a>Tantoso E, Yang Y, Li K-B (2006) How well do HapMap SNPs capture the untyped SNPs? BMC Genomics  7: 238.  <ul class="find" data-citedArticleID="1025852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=How+well+do+HapMap+SNPs+capture+the+untyped+SNPs%3F&amp;auth=&amp;atitle=How+well+do+HapMap+SNPs+capture+the+untyped+SNPs%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+well+do+HapMap+SNPs+capture+the+untyped+SNPs%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+well+do+HapMap+SNPs+capture+the+untyped+SNPs%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001262-Carlson1" id="pone.0001262-Carlson1"></a>Carlson CS, Eberle MA, Rieder MJ, Yi Q, Kruglyak L, et al.  (2004) Selecting a maximally informative set of single-nucleotide polymorphisms for association analyses using linkage disequilibrium. Am J Hum Genet  74: 106–120.  <ul class="find" data-citedArticleID="1025788" data-doi="10.1086/381000"><li><a href="http://dx.doi.org/10.1086/381000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selecting+a+maximally+informative+set+of+single-nucleotide+polymorphisms+for+association+analyses+using+linkage+disequilibrium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001262-dbSNP1" id="pone.0001262-dbSNP1"></a>  dbSNP <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP​/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">46.
              </span><a name="pone.0001262-Entrez1" id="pone.0001262-Entrez1"></a>  Entrez Gene <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP​/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0001262-UCSC1" id="pone.0001262-UCSC1"></a>  UCSC <a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0001262-Siepel1" id="pone.0001262-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res  15: 1034–1050.  <ul class="find" data-citedArticleID="1025842" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.XML" value="88946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.PDF" value="318414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g001.PNG_L" value="483396"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g001.PNG_M" value="25066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g001.PNG_S" value="4658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g001.TIF" value="713292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g001.PNG_I" value="8825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g002.PNG_L" value="1375141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g002.PNG_M" value="41047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g002.PNG_S" value="7557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g002.TIF" value="1782276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g002.PNG_I" value="24408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g003.PNG_L" value="458088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g003.PNG_M" value="34595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g003.PNG_S" value="6140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g003.TIF" value="720840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g003.PNG_I" value="14555"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g004.PNG_L" value="1638059"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g004.PNG_M" value="180403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g004.PNG_S" value="14153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g004.TIF" value="2421278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g004.PNG_I" value="86779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.t001.PNG_L" value="292288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.t001.PNG_M" value="165419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.t001.PNG_S" value="16593"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.t001.TIF" value="1297068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.t001.PNG_I" value="50556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g005.PNG_L" value="369397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g005.PNG_M" value="100330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g005.PNG_S" value="9525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g005.TIF" value="696122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.g005.PNG_I" value="36371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.s001.DOC" value="1176064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.s002.DOC" value="1307648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.s003.DOC" value="27136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001262.s004.DOC" value="72704"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001262&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001262" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001262&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001262&volume=&issue=&title=Gene-Centric Characteristics of Genome-Wide Association Studies&author_name=Changzheng%20Dong%2C%20Ziliang%20Qian%2C%20Peilin%20Jia%2C%20Ying%20Wang%2C%20Wei%20Huang%2C%20Yixue%20Li&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001262" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262&amp;t=Gene-Centric%20Characteristics%20of%20Genome-Wide%20Association%20Studies" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262&title=Gene-Centric%20Characteristics%20of%20Genome-Wide%20Association%20Studies&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262&amp;title=Gene-Centric%20Characteristics%20of%20Genome-Wide%20Association%20Studies" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001262&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Gene-Centric Characteristics of Genome-Wide Association Studies';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001262';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Gene-Centric%20Characteristics%20of%20Genome-Wide%20Association%20Studies http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001262" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001262" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40169" data-articleid="25356"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="25356"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+genetics%22" title="Search for articles in the subject area:'Evolutionary genetics'"><div class="flagText">Evolutionary genetics</div></a>
              <div data-categoryid="17209" data-articleid="25356"
                   data-categoryname="Evolutionary genetics"
                   class="flagImage" title="Flag 'Evolutionary genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetics+of+disease%22" title="Search for articles in the subject area:'Genetics of disease'"><div class="flagText">Genetics of disease</div></a>
              <div data-categoryid="17953" data-articleid="25356"
                   data-categoryname="Genetics of disease"
                   class="flagImage" title="Flag 'Genetics of disease' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25356"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genotyping%22" title="Search for articles in the subject area:'Genotyping'"><div class="flagText">Genotyping</div></a>
              <div data-categoryid="47691" data-articleid="25356"
                   data-categoryname="Genotyping"
                   class="flagImage" title="Flag 'Genotyping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="25356"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Linkage+disequilibrium%22" title="Search for articles in the subject area:'Linkage disequilibrium'"><div class="flagText">Linkage disequilibrium</div></a>
              <div data-categoryid="42897" data-articleid="25356"
                   data-categoryname="Linkage disequilibrium"
                   class="flagImage" title="Flag 'Linkage disequilibrium' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8226'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9281'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9046&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
