

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001781"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001781" />

    <meta name="citation_title" content="Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients"/>
    <meta itemprop="name" content="Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients"/>

      <meta name="citation_author" content="André F. A. Santos"/>
            <meta name="citation_author_institution" content="Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil"/>
      <meta name="citation_author" content="Renan B. Lengruber"/>
            <meta name="citation_author_institution" content="Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil"/>
      <meta name="citation_author" content="Esmeralda A. Soares"/>
            <meta name="citation_author_institution" content="Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil"/>
      <meta name="citation_author" content="Abhay Jere"/>
            <meta name="citation_author_institution" content="HIV Drug Resistance Program, National Cancer Insitute-Frederick, Frederick, Maryland, United States of America"/>
      <meta name="citation_author" content="Eduardo Sprinz"/>
            <meta name="citation_author_institution" content="Hospital de Clínicas de Porto Alegre, Porto Alegre, Brazil"/>
      <meta name="citation_author" content="Ana M. B. Martinez"/>
            <meta name="citation_author_institution" content="Fundação Universidade do Rio Grande, Rio Grande, Brazil"/>
      <meta name="citation_author" content="Jussara Silveira"/>
            <meta name="citation_author_institution" content="Fundação Universidade do Rio Grande, Rio Grande, Brazil"/>
      <meta name="citation_author" content="Fernando S. Sion"/>
            <meta name="citation_author_institution" content="Hospital Universitário Gaffrée e Guinle, Rio de Janeiro, Brazil"/>
      <meta name="citation_author" content="Vinay K. Pathak"/>
            <meta name="citation_author_institution" content="HIV Drug Resistance Program, National Cancer Insitute-Frederick, Frederick, Maryland, United States of America"/>
      <meta name="citation_author" content="Marcelo A. Soares"/>
            <meta name="citation_author_institution" content="Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil"/>
            <meta name="citation_author_institution" content="Unidade de Genética, Instituto Nacional de Câncer, Rio de Janeiro, Brazil"/>

    <meta name="citation_date" content="2008/3/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001781.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1781"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Structural characterization of reverse transcriptase and endonuclease polypeptides of the acquired immunodeficiency syndrome retrovirus.; citation_author=MM Lightfoote; citation_author=JE Coligan; citation_author=TM Folks; citation_author=AS Fauci; citation_author=MA Martin; citation_journal_title=J Virol; citation_volume=60; citation_number=1; citation_pages=771-775; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of human immunodeficiency virus type 1 reverse transcriptase complexed with double-stranded DNA at 3.0 A resolution shows bent DNA.; citation_author=A Jacobo-Molina; citation_author=J Ding; citation_author=RG Nanni; citation_author=AD Clark; citation_author=X Lu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=2; citation_pages=6320-6324; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Crystal structure at 3.5 A resolution of HIV-1 reverse transcriptase complexed with an inhibitor.; citation_author=LA Kohlstaedt; citation_author=J Wang; citation_author=JM Friedman; citation_author=PA Rice; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=256; citation_number=3; citation_pages=1783-1790; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=The structure of unliganded reverse transcriptase from the human immunodeficiency virus type 1.; citation_author=DW Rodgers; citation_author=SJ Gamblin; citation_author=BA Harris; citation_author=S Ray; citation_author=JS Culp; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=92; citation_number=4; citation_pages=1222-1226; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Retroviruses.; citation_author=JM Coffin; citation_author=SH Hughes; citation_author=HE Varmus; citation_number=5; citation_date=1997; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Specificity of human immunodeficiency virus-1 reverse transcriptase-associated ribonuclease H in removal of the minus-strand primer, tRNA(Lys3).; citation_author=JS Smith; citation_author=MJ Roth; citation_journal_title=J Biol Chem; citation_volume=267; citation_number=6; citation_pages=15071-15079; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Interaction of HIV-1 ribonuclease H with polypurine tract containing RNA-DNA hybrids.; citation_author=BM Wohrl; citation_author=K Moelling; citation_journal_title=Biochemistry; citation_volume=29; citation_number=7; citation_pages=10141-10147; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Update of the drug resistance mutations in HIV-1: Fall 2006.; citation_author=VA Johnson; citation_author=F Brun-Vezinet; citation_author=B Clotet; citation_author=DR Kuritzkes; citation_author=D Pillay; citation_journal_title=Top HIV Med; citation_volume=14; citation_number=8; citation_pages=125-130; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV-1 pol mutation frequency by subtype and treatment experience: extension of the HIVseq program to seven non-B subtypes.; citation_author=SY Rhee; citation_author=R Kantor; citation_author=DA Katzenstein; citation_author=R Camacho; citation_author=L Morris; citation_journal_title=AIDS; citation_volume=20; citation_number=9; citation_pages=643-651; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV-1 Protease and reverse-transcriptase mutations: correlations with antiretroviral therapy in subtype B isolates and implications for drug-resistance surveillance.; citation_author=SY Rhee; citation_author=WJ Fessel; citation_author=AR Zolopa; citation_author=L Hurley; citation_author=T Liu; citation_journal_title=J Infect Dis; citation_volume=192; citation_number=10; citation_pages=456-465; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mechanism for nucleoside analog-mediated abrogation of HIV-1 replication: balance between RNase H activity and nucleotide excision.; citation_author=GN Nikolenko; citation_author=S Palmer; citation_author=F Maldarelli; citation_author=JW Mellors; citation_author=JM Coffin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=11; citation_pages=2093-2098; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mutation of amino acids in the connection domain of human immunodeficiency virus type 1 reverse transcriptase that contact the template-primer affects RNase H activity.; citation_author=JG Julias; citation_author=MJ McWilliams; citation_author=SG Sarafianos; citation_author=WG Alvord; citation_author=E Arnold; citation_journal_title=J Virol; citation_volume=77; citation_number=12; citation_pages=8548-8554; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Line probe assay for rapid detection of drug-selected mutations in the human immunodeficiency virus type 1 reverse transcriptase gene.; citation_author=L Stuyver; citation_author=A Wyseur; citation_author=A Rombout; citation_author=J Louwagie; citation_author=T Scarcez; citation_journal_title=Antimicrob Agents Chemother; citation_volume=41; citation_number=13; citation_pages=284-291; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=HIV sequence databases.; citation_author=C Kuiken; citation_author=B Korber; citation_author=RW Shafer; citation_journal_title=AIDS Rev; citation_volume=5; citation_number=14; citation_pages=52-61; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MEGA2: molecular evolutionary genetics analysis software.; citation_author=S Kumar; citation_author=JK Tamura; citation_author=IB Jakobsen; citation_author=M Nei; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=15; citation_pages=1244-1245; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Full-length genome sequencing of HIV type 1 group O viruses isolated from a heterosexual transmission cluster in Senegal.; citation_author=CT Kane; citation_author=C Montavon; citation_author=MA Toure; citation_author=MA Faye; citation_author=AG Ndiaye; citation_journal_title=AIDS Res Hum Retroviruses; citation_volume=17; citation_number=16; citation_pages=1211-1216; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus reverse transcriptase and protease sequence database.; citation_author=SY Rhee; citation_author=MJ Gonzales; citation_author=R Kantor; citation_author=BJ Betts; citation_author=J Ravela; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=17; citation_pages=298-303; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=18; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Analysis for free: comparing programs for sequence analysis.; citation_author=HF Tippmann; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=19; citation_pages=82-87; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of HIV-1 reverse transcriptase in complex with a polypurine tract RNA:DNA.; citation_author=SG Sarafianos; citation_author=K Das; citation_author=C Tantillo; citation_author=AD Clark; citation_author=J Ding; citation_journal_title=EMBO J; citation_volume=20; citation_number=20; citation_pages=1449-1461; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of the minimal conserved structure of HIV-1 protease in the presence and absence of drug pressure.; citation_author=F Ceccherini-Silberstein; citation_author=F Erba; citation_author=F Gago; citation_author=A Bertoli; citation_author=F Forbici; citation_journal_title=AIDS; citation_volume=18; citation_number=21; citation_pages=F11-F19; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High sequence conservation of human immunodeficiency virus type 1 reverse transcriptase under drug pressure despite the continuous appearance of mutations.; citation_author=F Ceccherini-Silberstein; citation_author=F Gago; citation_author=M Santoro; citation_author=C Gori; citation_author=V Svicher; citation_journal_title=J Virol; citation_volume=79; citation_number=22; citation_pages=10718-10729; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mutations in the RNase H domain of HIV-1 reverse transcriptase affect the initiation of DNA synthesis and the specificity of RNase H cleavage in vivo.; citation_author=JG Julias; citation_author=MJ McWilliams; citation_author=SG Sarafianos; citation_author=E Arnold; citation_author=SH Hughes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=23; citation_pages=9515-9520; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mutational analysis of Tyr-501 of HIV-1 reverse transcriptase. Effects on ribonuclease H activity and inhibition of this activity by N-acylhydrazones.; citation_author=D Arion; citation_author=N Sluis-Cremer; citation_author=KL Mim; citation_author=ME Abram; citation_author=RS Fletcher; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=24; citation_pages=1370-1374; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Enzymatic analysis of two HIV-1 reverse transcriptase mutants with mutations in carboxyl-terminal amino acids residues conserved among retroviral ribonucleases H.; citation_author=S Volkmann; citation_author=BM Wohrl; citation_author=M Tisdale; citation_author=K Moelling; citation_journal_title=J Biol Chem; citation_volume=75; citation_number=25; citation_pages=71-78; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Virion instability of human immunodeficiency virus type 1 reverse transcriptase (RT) mutated in the protease cleavage site between RT p51 and the RT RNase H domain.; citation_author=ME Abram; citation_author=MA Parniak; citation_journal_title=J Virol; citation_volume=79; citation_number=26; citation_pages=11952-11961; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the ribonuclease H domain of HIV-1 reverse transcriptase.; citation_author=JF Davies; citation_author=Z Hostomska; citation_author=Z Hostomsky; citation_author=SR Jordan; citation_author=DA Matthews; citation_journal_title=Science; citation_volume=252; citation_number=27; citation_pages=88-95; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Interaction between human immunodeficiency virus type 1 reverse transcriptase and integrase proteins.; citation_author=EA Hehl; citation_author=P Joshi; citation_author=GV Kalpana; citation_author=VR Prasad; citation_journal_title=J Virol; citation_volume=78; citation_number=28; citation_pages=5056-5067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Disruption of a salt bridge between Asp 488 and Lys 465 in HIV-1 reverse transcriptase alters its proteolytic processing and polymerase activity.; citation_author=L Goobar-Larsoon; citation_author=K Backbro; citation_author=T Unge; citation_author=R Bhikhabhai; citation_author=L Vrang; citation_journal_title=Virology; citation_volume=196; citation_number=29; citation_pages=731-738; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Mutations in Human Immunodeficiency Virus Type-1 RNase H Primer Grip Enhance 3′-Azido-3′-Deoxythymidine Resistance.; citation_author=KA Delviks-Frankenberry; citation_author=GN Nikolenko; citation_author=R Barr; citation_author=VK Pathak; citation_journal_title=J. Virol.; citation_volume=81; citation_number=30; citation_pages=6837-6845; " />
      <meta name="citation_reference" content="citation_title=Identification of accessory mutations associated with high-level resistance in HIV-1 reverse transcriptase.; citation_author=PA Cane; citation_author=H Green; citation_author=E Fearnhill; citation_author=D Dunn; citation_journal_title=AIDS; citation_volume=21; citation_number=31; citation_pages=447-455; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mutations in the connection domain of HIV-1 reverse transcriptase increase 3′-azido-3′-deoxythymidine resistance.; citation_author=GN Nikolenko; citation_author=KA Delviks-Frankenberry; citation_author=S Palmer; citation_author=F Maldarelli; citation_author=MJ Fivash; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=104; citation_number=32; citation_pages=317-322; citation_date=2007; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients"/>
    <meta name="twitter:description" content="BackgroundAlthough extensive HIV drug resistance information is available for the first 400 amino acids of its reverse transcriptase, the impact of antiretroviral treatment in C-terminal domains of Pol (thumb, connection and RNase H) is poorly understood.Methods and FindingsWe wanted to characterize conserved regions in RT C-terminal domains among HIV-1 group M subtypes and CRF. Additionally, we wished to identify NRTI-related mutations in HIV-1 RT C-terminal domains. We sequenced 118 RNase H domains from clinical viral isolates in Brazil, and analyzed 510 thumb and connection domain and 450 RNase H domain sequences collected from public HIV sequence databases, together with their treatment status and histories. Drug-na&iuml;ve and NRTI-treated datasets were compared for intra- and inter-group conservation, and differences were determined using Fisher's exact tests. One third of RT C-terminal residues were found to be conserved among group M variants. Three mutations were found exclusively in NRTI-treated isolates. Nine mutations in the connection and 6 mutations in the RNase H were associated with NRTI treatment in subtype B. Some of them lay in or close to amino acid residues which contact nucleic acid or near the RNase H active site. Several of the residues pointed out herein have been recently associated to NRTI exposure or increase drug resistance to NRTI.ConclusionsThis is the first comprehensive genotypic analysis of a large sequence dataset that describes NRTI-related mutations in HIV-1 RT C-terminal domains in vivo. The findings into the conservation of RT C-terminal domains may pave the way to more rational drug design initiatives targeting those regions."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001781.g004"/>

  <meta property="og:title" content="Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=496'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4138'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8252&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001781">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001781" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001781&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            André F. A. Santos, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Renan B. Lengruber, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Esmeralda A. Soares, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Abhay Jere, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  HIV Drug Resistance Program, National Cancer Insitute-Frederick, Frederick, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eduardo Sprinz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Hospital de Clínicas de Porto Alegre, Porto Alegre, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ana M. B. Martinez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Fundação Universidade do Rio Grande, Rio Grande, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jussara Silveira, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Fundação Universidade do Rio Grande, Rio Grande, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fernando S. Sion, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Hospital Universitário Gaffrée e Guinle, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vinay K. Pathak, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  HIV Drug Resistance Program, National Cancer Insitute-Frederick, Frederick, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marcelo A. Soares
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:masoares@biologia.ufrj.br">masoares@biologia.ufrj.br</a></p>

                <p>Affiliations:
                  Laboratório de Virologia Humana, Departamento de Genética, Universidade Federal do Rio de Janeiro, Rio de Janeiro, Brazil, 
                  Unidade de Genética, Instituto Nacional de Câncer, Rio de Janeiro, Brazil
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 12, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001781</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001781-g001" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001781-g002" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001781-g003" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001781-t001" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001781-g004" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001781">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001781" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Although extensive HIV drug resistance information is available for the first 400 amino acids of its reverse transcriptase, the impact of antiretroviral treatment in C-terminal domains of Pol (thumb, connection and RNase H) is poorly understood.</p>


<h3>Methods and Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We wanted to characterize conserved regions in RT C-terminal domains among HIV-1 group M subtypes and CRF. Additionally, we wished to identify NRTI-related mutations in HIV-1 RT C-terminal domains. We sequenced 118 RNase H domains from clinical viral isolates in Brazil, and analyzed 510 thumb and connection domain and 450 RNase H domain sequences collected from public HIV sequence databases, together with their treatment status and histories. Drug-naïve and NRTI-treated datasets were compared for intra- and inter-group conservation, and differences were determined using Fisher's exact tests. One third of RT C-terminal residues were found to be conserved among group M variants. Three mutations were found exclusively in NRTI-treated isolates. Nine mutations in the connection and 6 mutations in the RNase H were associated with NRTI treatment in subtype B. Some of them lay in or close to amino acid residues which contact nucleic acid or near the RNase H active site. Several of the residues pointed out herein have been recently associated to NRTI exposure or increase drug resistance to NRTI.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This is the first comprehensive genotypic analysis of a large sequence dataset that describes NRTI-related mutations in HIV-1 RT C-terminal domains <em>in vivo</em>. The findings into the conservation of RT C-terminal domains may pave the way to more rational drug design initiatives targeting those regions.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Santos AFA, Lengruber RB, Soares EA, Jere A, Sprinz E, et al.  (2008) Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients. PLoS ONE 3(3):
          e1781.
            doi:10.1371/journal.pone.0001781</p><p><strong>Academic Editor: </strong>Beatriz Grinsztejn, Instituto de Pesquisa Clinica Evandro Chagas, FIOCRUZ, Brazil</p><p><strong>Received:</strong> July 9, 2007; <strong>Accepted:</strong> January 29, 2008; <strong>Published:</strong> March 12, 2008</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>Financial support for this study was from the Brazilian Ministry of Science and Technology (CNPq) grant no. 403589/2004-5, Rio de Janeiro State Science Foundation (FAPERJ) grant no. E-26/171.337/2002 and the Brazilian Ministry of Health (PN-DST/AIDS) grant no. 322/2005. The sponsors had no role in any step of the study design, data analysis or manuscript preparation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The HIV-1 RT is the enzyme responsible for synthesizing a double-stranded integrative cDNA from the single-stranded viral genomic RNA early in the virus life cycle. HIV-1 RT is a heterodimer composed of two subunits, p66 and p51; p51 is derived from the proteolytic processing and removal of C-terminus (codons 441 to 560) from p66 <a href="#pone.0001781-Lightfoote1">[1]</a>. The p66 subunit comprises two major portions, the polymerase (codons 1 to 440) and the RNase H domains. The polymerase domain encodes the RT fingers, palm, thumb and connection motifs <a href="#pone.0001781-JacoboMolina1">[2]</a>–<a href="#pone.0001781-Rodgers1">[4]</a>. The RNase H domain promotes RNA degradation from the DNA:RNA duplex during reverse transcription. Other RNase H functions include the removal of the tRNA<sub>3</sub><sup>Lys</sup> and the removal of the polypurine tract (PPT), primers for the DNA minus-strand and DNA plus-strand synthesis, respectively <a href="#pone.0001781-Coffin1">[5]</a>–<a href="#pone.0001781-Wohrl1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The polymerase RT portion is currently targeted by two classes of ARV drugs, the nucleoside reverse transcriptase inhibitors (NRTI) and the non-nucleoside RTI (NNRTI). HIV drug resistance mutations in this domain are extensively characterized for these classes of ARV drugs <a href="#pone.0001781-Johnson1">[8]</a>. To date, however, the influence of connection and RNase H RT domains in NRTI or NNRTI resistance is poorly understood. Currently the region of RT selected for sequencing in resistance studies ranges mostly from codons 1 to 250 <a href="#pone.0001781-Rhee1">[9]</a>, <a href="#pone.0001781-Rhee2">[10]</a>, and does not include these domains. Recently, Nikolenko <em>et al.</em> pointed out mutations in the RNase H domain that reduce its activity and concomitantly increase the resistance to thymidine analogues <em>in vitro</em> <a href="#pone.0001781-Nikolenko1">[11]</a>. They have also proposed a new drug resistance mechanism where the lowered RNase H activity would allow more time for NRTI excision. This mechanism is also influenced by the connection domain, as point mutations in the region that contact template-primer affect RNase H activity <a href="#pone.0001781-Julias1">[12]</a>. In the present study we analyzed the connection and RNase H domains from treatment-naïve and NRTI-experienced patients infected with different HIV-1 subtypes in the search for structural conservation patterns and for putative new NRTI-related mutations. This analysis will also enable the establishment of conserved regions in the connection and RNase H RT domains, which could be an essential tool for future rational drug design initiatives targeting those domains.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Sample collection</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Plasma samples from HIV-positive patients, both treatment-naïve and HAART-experienced, were collected from three Brazilian HIV/AIDS reference centers: two in Rio Grande do Sul (RS) state, Hospital de Clínicas de Porto Alegre and University Hospital of Rio Grande, and one located in Rio de Janeiro (RJ), Hospital Gaffrée e Guinle. Patients were recruited from 2002 to 2003 in RS and from 2004 to 2006 in RJ. All patients signed a written consent form for participating in the study, and the Internal Review Boards from all three Institutions have approved the study.</p>


<h4>RNA isolation, nested PCR and RNase H domain sequencing</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>HIV viral RNA was isolated from 50 µl of patient's plasma using Trizol, followed by cDNA synthesis with Moloney murine leukemia virus RT and random primers as previously described <a href="#pone.0001781-Stuyver1">[13]</a>. Viral RNase H domain was amplified by nested PCR using Platinum Taq polymerase (Invitrogen, CA, EUA). The first round was conducted using primers <span class="gene">5′tatacgtaagccacctggattc3′</span> (3762-3783 nucleotide positions relative to HXB2) and <span class="gene">5′cagtctacttgtccatgcatggcttc3′</span> (nt relative positions 4371-4396) with an annealing temperature of 55°C. The second round was done using primers <span class="gene">5′ggtaccagttagagaaagaaccca3′</span> (nucleotide positions 3826-3849) and <span class="gene">5′cattgcctctccaattactgtgatatttctcatg3′</span> (nt positions 4263-4295) with an annealing temperature of 56°C. A 469-bp fragment corresponding to RNase H codons was amplified, purified and sequenced in an automated ABI Prism 3100 Genetic Analyzer (Applied Biosystems, Foster City, CA, USA). Sequences obtained were assembled and manually edited using SeqMan (DNAStar, Madison, WI, USA).</p>


<h4>HIV-1 subtype characterization</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To eliminate sample contaminations or mix-ups and to determine the isolates' subtype, a FASTA file with all edited sequences and representative sequences of all HIV-1 subtypes obtained from Los Alamos HIV database (<a href="http://hiv-web.lanl.gov">http://hiv-web.lanl.gov</a>) <a href="#pone.0001781-Kuiken1">[14]</a> was constructed. The HIV-1 subtype of each sample was determined through phylogenetic inference using the neighbor-joining method and Kimura 2-parameter model of nucleotide substitution in the MEGA 2.1 package <a href="#pone.0001781-Kumar1">[15]</a>. One thousand replicates of bootstrap were performed to infer cluster robustness and the sequence of HIV-1 group O 99SE-MP1299 (GenBank acc. no. AJ302646) was used to outgroup the trees <a href="#pone.0001781-Kane1">[16]</a>.</p>


<h4>Global thumb, connection, and RNase H domain sequences</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>For the conservation analysis of thumb, connection and RNase H domains and potential drug resistance mutations, we searched all available HIV-1 group M sequences spanning all subtypes (A–D, F–H, J and K) and CRF01_AE and CRF02_AG from the Los Alamos HIV database (<a href="http://hiv-web.lanl.gov">http://hiv-web.lanl.gov</a>) <a href="#pone.0001781-Kuiken1">[14]</a> and the Stanford HIV Drug Resistance Database (<a href="http://hivdb.stanford.edu/">http://hivdb.stanford.edu/</a>) <a href="#pone.0001781-Rhee3">[17]</a>. Only isolates from patients for which treatment information was available, and which included the use of NRTI, were included in the analysis. The Stanford database already includes full treatment history for all sequences. For the Los Alamos database, ARV-experienced sequences had their GenBank accession number retrieved, which was used to track treatment history in the Stanford database. To prevent the use of more than one sample from the same patient, all sequences obtained were subjected to phylogenetic inference using the methodology described above, and for sequences shown to be phylogenetically close-related only one of them was kept.</p>


<h4>Determination of drug-related mutations</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In view of potential sequence biases created from comparing different HIV-1 polymorphisms present in different subtypes, only subtype B sequences were used for drug-related mutation comparisons. Sequences belonging to subtype B from treatment-naïve and experienced groups were separately aligned using ClustalX <a href="#pone.0001781-Thompson1">[18]</a> in the freely available program BioEdit v. 7.0.5.3 <a href="#pone.0001781-Tippmann1">[19]</a>. A subtype B consensus sequence from treatment-naive subjects was used as reference in the analysis. Nucleotide sequences were translated into amino acids and residue differences were compiled between treatment-naïve and NRTI-treated patients. All amino acid differences between both treatment groups were evaluated by statistical inference using two-tailed Fisher's exact test, and a <em>p</em>-value&lt;0.05 was considered significant.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>In an attempt to investigate the potential role of specific ARV drugs in selecting resistance mutations, we have analyzed 28 subtype B sequences from subjects that underwent monotherapy with AZT (the only drug for which there were cases of single drug exposure prior to sample collection). Those sequences were compared to the subtype B drug-naïve isolates described above for significance of amino acid residue frequencies.</p>


<h4>Structural visualization of HIV-1 RT domains</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To facilitate the visualization of conserved positions and the inferred drug-related mutations, the atomic coordinates of the crystal structure of HIV-1 BH10 (subtype B) RT in contact with the DNA:RNA hybrid (PDB entry 1HYS) were used <a href="#pone.0001781-Sarafianos1">[20]</a>. The software ViewerLite 5.0 was used to visualize the file and to color the conservation patterns and mutations which differ significantly between treatment-naïve and NRTI-treated groups.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Conservation of RT C-terminal domains</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Five hundred and ten sequences of thumb and connection domains (RT codons 298-440) were retrieved from the databases, of which 280 were treatment-naïve (HIV-1 subtype composition: 38 A, 72 B, 73 C, 34 D, 6 F, 4 G, 1 H, 1 K, 44 CRF01_AE and 7 CRF02_AG) and 230 were NRTI-treated (10 A, 188 B, 4 C, 4 D, 20 CRF01_AE and 4 CRF02_AG). For the RNase H domain (RT codons 441-560), we obtained 450 sequences, of which 276 were treatment-naïve (39 A, 66 B, 69 C, 35 D, 7 F, 5 G, 1 H, 1 J, 44 CRF01_AE and 9 CRF02_AG) and 174 were NRTI-treated (1 A, 153 B, 2 F and 18 CRF01_AE).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>We successfully amplified and sequenced the RNase H domain (codons 441-560) of 118 isolates from Brazilian HIV-positive patients. The phylogenetic inference showed that half of them belonged to subtype C (n = 59) and the other half was assigned to subtype B (n = 59) (data not shown). Regarding ARV treatment status, 58 patients were naive (B = 27 and C = 31) and 60 were on treatment (B = 32 and C = 28). Together with the sequences from public databases, we gathered 334 RNase H sequences of drug-naïve and 234 RNase H sequences of NRTI-treated patients. For the RT connection domain, we have obtained 510 isolates from the public databases for further analysis.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Sequences of RT C-terminal domains were aligned, subtyped by phylogenetic analysis (data not shown), and evaluated by their extent of codon polymorphisms (<a href="#pone-0001781-g001">Figure 1</a>). In drug-naive isolates, the number of non-variable sites (defined as sites exhibiting variation ≤1%) was 165 (63%), a similar number found among NRTI-treated isolates (163; 62%). In some cases, sites seemed more variable in the naïve isolates than in the NRTI-treated isolates. This is explained by the different subtype proportions among those two groups. While 26% of thumb and connection domain sequences from drug-naïve subjects were represented by subtype B, this proportion among treated subjects was of 82%. Similarly, subtype B accounted for only 28% of RNase H domain sequences from naïve subjects, while it accounted for 80% among treated subjects.</p>
<div class="figure" id="pone-0001781-g001"><div class="img"><a name="pone-0001781-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Analysis of conservation of HIV-1 RT connection and RNase H domains from HIV-1-infected drug-naïve and NRTI-treated subjects.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Amino acid residues are represented as the group M consensus. Degrees of conservation are color-coded as shown at the bottom. Conserved motifs (variability below 1%) are boxed.</p>
<span>doi:10.1371/journal.pone.0001781.g001</span></div>

<h4>NRTI-selected amino acid variations</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>We analyzed amino acid variations found <em>in vivo</em> when comparing NRTI-naïve and treated subjects at each one of 263 codons included in the study. Since a number of amino acid residues are polymorphic across all HIV-1 group M subtypes, we decided to search for NRTI-related mutations only among subtype B sequences available. Subtype B represented alone 51% (260/510) of thumb and connection domain strains and 51% (288/568) of RNase H domain strains in our analysis. The results can be seen in <a href="#pone-0001781-g002">Figures 2</a> and <a href="#pone-0001781-g003">3</a>. Fifteen amino acid substitutions had significantly different frequencies when comparing drug-naïve and treatment-experienced subtype B isolates. Of those, 12 had increased prevalence in treated sequences (R358K, G359S, A360T, A360V, K366R, A371V, K390R, A400T, I506L, K527N, K530R and Q547K). The mutations A360V, I506L and Q547K were only seen in subtype B patients under treatment. The largest difference found was for mutation K527N, with a frequency 16 times higher in treated patients (<a href="#pone-0001781-g003">Figure 3</a>). Mutation A371V was seen in 22.6% of treated versus only 2.3% of naïve subtype B isolates (10 times lower). Similarly, mutation K530R was found in 12.1% of treated isolates versus 1.1% of naïve isolates. Another mutation, G359S, was found in 15% of treated isolates, while it was seen only in 3.4% of the drug-naïve isolates (<a href="#pone-0001781-g002">Figure 2</a>). Three amino acid substitutions showed decreased proportions in treatment-experienced subtype B isolates when compared to drug-naïve isolates (I326V, T470N, and K512R). All 15 statistically-significant amino acid differences between drug-naïve and experienced sequences, together with their respective <em>p</em>-values, can also be seen in <a href="#pone-0001781-t001">Table 1</a>.</p>
<div class="figure" id="pone-0001781-g002"><div class="img"><a name="pone-0001781-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Comprehensive analysis of amino acid sequence variation in HIV-1 RT thumb and connection C-terminal domains showing polymorphic variations between subtype B viruses from drug-naïve and NRTI-treated subjects.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>Residues are displayed as the subtype B consensus; other residues found among sequences analyzed are shown below each consensus residue. Numbers shown in subscript for each residue represent total number of sequences analyzed (in the consensus sequence) and frequencies in viruses (other residues) from drug-naïve (left) and NRTI-treated (right) subjects. Residues which differed significantly (<em>p</em>&lt;0.05) in frequency between naïve and treated patients, as well as the corresponding changed residues, are boxed in gray.</p>
<span>doi:10.1371/journal.pone.0001781.g002</span></div><div class="figure" id="pone-0001781-g003"><div class="img"><a name="pone-0001781-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comprehensive analysis of amino acid sequence variation in HIV-1 RT RNase H C-terminal domains showing polymorphic variations between subtype B viruses from drug-naïve and NRTI-treated subjects.</span></strong></p><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1"></a><p>Residues are displayed as the subtype B consensus; other residues found among sequences analyzed are shown below each consensus residue. Numbers shown in subscript for each residue represent total number of sequences analyzed (in the consensus sequence) and frequencies in viruses (other residues) from drug-naïve (left) and NRTI-treated (right) subjects. Residues which differed significantly (<em>p</em>&lt;0.05) in frequency between naïve and treated patients, as well as the corresponding changed residues, are boxed in gray.</p>
<span>doi:10.1371/journal.pone.0001781.g003</span></div><div class="figure" id="pone-0001781-t001"><div class="img"><a name="pone-0001781-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001781.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001781.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Frequency of putative NRTI- and AZT-related mutations found among viruses from patients with NRTI or with AZT monotherapy exposure</span></strong></p><span>doi:10.1371/journal.pone.0001781.t001</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The role of specific ARV drugs in selecting the mutations depicted here is not easily determined, since the vast majority of treated patients whose viral samples were analyzed have been subjected to multiple, complex drug combinations prior to sample collection. In our dataset, none of the connection domain sequences analyzed was from patients in use of NNRTI, but a small fraction of the RNase H viral sequence (8.6%; 16/185) had previous exposure to that drug class. Therefore, we cannot completely rule out the possibility of a potential influence of NNRTI in the selection of mutations characterized here. However, a limited number of those samples (n = 28) are from patients older in the epidemic and exclusively subjected to AZT monotherapy. We have compared these sequences with those from the drug-naïve dataset in the positions pointed out above. It noteworthy that comparisons were possible only up to RT codon 500, as scarce sequence information is available after that codon for those viral sequences. The comparison can be seen in <a href="#pone-0001781-t001">Table 1</a>. Among the 10 mutations analyzed, 4 were significantly found more frequently in AZT-exposed subjects (A360V, A371V, K390R and A400T). These mutations are expected to be selected by AZT, and perhaps by other NRTI, particularly thymidine analogues.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Mutations subjected to selective pressure were highlighted onto the structural C-terminal domains of HIV-1 RT (<a href="#pone-0001781-g004">Figure 4</a>). In the connection domain, nine mutations were selected by treatment in subtype B. The RNase H domain harbored 6 mutations selected by treatment in this subtype. Interestingly, some of these residues lie in motifs known to contact the nucleic acid (G359S, A360T/V and K390R). As expected, the seven amino acid residues that comprise the RNase H catalytic site (D443, E478, D498, S499, H539, N545 and D549) were highly conserved and remained unchanged under NRTI treatment.</p>
<div class="figure" id="pone-0001781-g004"><div class="img"><a name="pone-0001781-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001781" data-uri="info:doi/10.1371/journal.pone.0001781.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001781.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001781.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001781.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001781.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Three-dimensional structure of HIV-1 RT connection (<em>A</em>) and RNase H (<em>B</em>) domains in contact with template viral DNA showing codons significantly associated with NRTI treatment.</span></strong></p><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1"></a><p>The atomic coordinates of HIV-1 RT isolate BH10 (entry 1HYS of the Brookhanven Protein Data Bank) was downloaded into the visualization software ViewerLite v. 5.0 and then manually colored. Subtype B-related substitutions with statistical significance are shown in red. Amino acid residues with asterisks denote those which interact with nucleic acid. Additional residues which interact with nucleic acid, but not found to be selected by treatment in our analysis, are shown in green. Amino acid residues depicted in orange in panel (<em>B</em>) represent the RNase H catalytic site.</p>
<span>doi:10.1371/journal.pone.0001781.g004</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The present study determined the conserved amino acid positions of thumb, connection and RNase H domains of HIV-1 group M RT. The comparison of polymorphisms between drug-naïve and NRTI-experienced patients has evidenced a total of 86 (33%) invariable sites common to both groups. Other studies recently published on the polymorphism of PR (codons 1-99) and RT N-terminal domains (1-320) observed highly conserved sites in approximately half of PR <a href="#pone.0001781-CeccheriniSilberstein1">[21]</a> and in over half of RT N-terminal domain <a href="#pone.0001781-CeccheriniSilberstein2">[22]</a>, independent of treatment. In both studies, the inclusion of non-B subtypes was negligible, with contributions of only 3.9% in the former study and 4.8% in the latter. The use of few non-B subtypes in those analyses may explain the high conservation of PR and RT enzymes as the standard consensus for resistance studies is the subtype B. In our study, we found that one third of RT C-terminal sites are highly conserved, independent of HIV-1 group M subtypes or treatment status.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Regions of C-terminal RT domains lying closer to the DNA:RNA hybrid were found conserved, including amino acid residues that interact with the primer DNA (G359, A360, H361, K395, E396, T473, N474, Q475, K476, Y501 and I505) and the template RNA strand (K390, R448, N474, Q475, Q500 and H539) <a href="#pone.0001781-Sarafianos1">[20]</a>. Site-directed mutagenesis of RNase H positions N474, Q475 and Y501 rendered the mutant enzymes inefficient in the removal of the PPT primer and modified the RNase H cleavage specificity <a href="#pone.0001781-Julias2">[23]</a>. In addition, changes at RNase H positions Q475 and Y501 produced defective viruses <a href="#pone.0001781-Arion1">[24]</a>, <a href="#pone.0001781-Volkmann1">[25]</a>. The essential role of those positions for RNase H is also reflected in our study, where they were found highly conserved. Positions T473, Q500 and I505 also had a high degree of conservation. Other two positions (R448 and K476) showed a low variability (2–5% variation).</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The motif ETFYVDGA spanning RT codons 438-445 was highly conserved (0–5% variation), which is consistent with the fact that it overlaps the protease cleavage site AETFYVD (codons 437-443) located between the connection and the RNase H domains. Mutagenesis experiments in this motif caused defects in virion RT and integrase processing <a href="#pone.0001781-Abram1">[26]</a>. Positions E438, T439, Y441 and V442 were highly invariable (variation ≤1%) (<a href="#pone-0001781-g001">Figure 1</a>). Positions F440 and D443 presented a low degree of variability (variation 2–5%). Position A437 was the most variable (14%). The impact of this polymorphism on the efficiency of cleavage remains to be experimentally determined.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Another conserved region comprises codons 492-511 and represents the RNase H β4 and αB secondary structures <a href="#pone.0001781-Davies1">[27]</a>. Amino acids D498, which interacts with divalent cations, and S499, which is also part of the RNase H catalytic site, lie in this motif. The motif 535-553 is also conserved, which harbors the invariable amino acids N545, H539 and D549, this last aspartic acid also interacting with divalent cations. The five residues D498, S499, H539, N545 and D549 comprise the RNase H catalytic site together with two other residues, D443 and E478 <a href="#pone.0001781-Davies1">[27]</a>. All positions were highly conserved with the exception of position D443, with 2% variation found naturally in drug-naïve patients, but highly conserved in treated isolates.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The function of the connection domain has not been fully characterized. The crystal structure of HIV-1 RT has shown five positions of the connection domain that interact with DNA (G359, A360, H361, K395 and E396) and a unique position (K390) that interacts with RNA <a href="#pone.0001781-Sarafianos1">[20]</a>. These positions are part of the RNase H primer grip motif, which helps positioning the template:primer at the RNase H active site. More recently, site-directed mutagenesis experiments in all those codons showed individual effects on reducing the RNase H activity either by altering the consensus 2-LTR circle junctions (H361 and E396) or by inserting tRNA primers at those junctions during reverse transcription (A360, H361, T362, K395 and E396) <a href="#pone.0001781-Julias1">[12]</a>. All mutations decreased the viral replicative capacity including mutations in residues G359 and K390. The authors suggested that these mutations weaken the binding of RT to nucleic acid, therefore affecting RNase H activity. In contrast to the high conservation seen in residues that contact nucleic acids in the RNase H domain, only two residues were highly conserved in the connection domain (H361 and T362; <a href="#pone-0001781-g001">Figure 1</a>). This can be explained by the proximity of these residues to the RNase H catalytic site <a href="#pone.0001781-Julias1">[12]</a>. The remaining residues presented natural variations within HIV-1 group M (10–70% variation). It is interesting to note that position 390 in the group M consensus is an arginine, where in subtype B it is a lysine. Moreover, the mutation K390R appears to be selected by NRTI usage in subtype B (<a href="#pone-0001781-g004">Figure 4</a>). The role of specific amino acid residues at this position and their impact on the primer grip function deserves further investigation.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>The most conserved region of the connection domain encodes amino acids 387-430, where 38 codons (86%) presented high conservation (0–4% variation). This is an interesting finding because this region contains one of two RT sites which interact with the viral integrase (codons 387-422) <a href="#pone.0001781-Hehl1">[28]</a>. Within this motif, 64% (23/36) of the codons had a high degree of conservation among HIV-1 group M strains. All conserved regions presented here are potential targets for new drugs aiming to inhibit HIV-1 group M replication.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>When we compared subtype B naïve and treatment-experienced sequence groups, differences between them were found. We discovered 15 mutations related to NRTI treatment. Until now, all studies published on HIV RTI drug resistance relied on the N-terminal half of RT, the target of both RTI drug classes. Such studies neglected the role of C-terminal RT domains in drug resistance. In 2005, Nikolenko <em>et al.</em> evidenced the importance of the RNase H domain to drug resistance. They showed that mutations which decreased <em>in vitro</em> RNase H activity greatly increased resistance to AZT and d4T <a href="#pone.0001781-Nikolenko1">[11]</a>. In the present study, we did not find any of the RNase H mutations pointed out in the literature that decreased RNase H activity <em>in vivo</em>. However, three of the mutations described here (A360V, I506L and Q547K) were present only in NRTI-experienced isolates. Mutation A360V lies in a residue that contacts nucleic acid, while mutation I506L is close to two residues of the RNase H primer grip <a href="#pone.0001781-Sarafianos1">[20]</a>. We think that these changes may weaken the RT-DNA:RNA contact, therefore decreasing RNase H activity, while still maintaining virus viability <em>in vivo</em>. Such hypothesis was corroborated by recent work of Delviks-Frankenberry and colleagues <a href="#pone.0001781-GoobarLarsoon1">[29]</a>, through alanine-scanning mutagenesis in RNase H primer grip residues. The authors showed concomitant decrease of RNase H activity and increase of AZT resistance by knocking out residues G359, A360, and K390. Each of these mutations did not confer resistance by itself, but in combination with thimydine analog mutations the fold change in resistance to AZT was from 2.5 to 6.5 times higher when compared to viruses carrying only TAMs <a href="#pone.0001781-GoobarLarsoon1">[29]</a>. All these residues are described herein as related to NRTI treatment.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>In view of potential biases created by differences in subtype distribution in drug-naïve and NRTI-experienced sequence groups, we conducted an exclusive analysis of subtype B, the one with the largest number of isolates sequenced, particularly those on NRTI treatment. A small number of NNRTI-experienced patients was present in the dataset, and as possible interactions between NRTI- and NNRTI-selected mutations is known to occur, we cannot completely discard a putative effect of NNRTI in the selection of mutations described here. Nine changes in the connection domain and 6 in the RNase H domain were found related to treatment (<a href="#pone-0001781-g002">Figures 2</a> and <a href="#pone-0001781-g003">3</a>). Some changes are within (G359S, A360V/T and K390R) or close (R358K and A400T) to the primer grip, while another is close to the RNase H catalytic site (Q547K; <a href="#pone-0001781-g004">Figure 4</a>). Experimental evidence for the role of those residues in drug resistance is just starting to emerge, but is rapidly accumulation. Nikolenko and colleagues have recently identified eight mutations in the connection domain (E312Q, G335C/D, N348I, A360I/V, V365I, A376S) that increase the resistance to AZT in phenotyping assay <a href="#pone.0001781-DelviksFrankenberry1">[30]</a>. In our analysis, mutation A360V was found significantly associated with NRTI treatment. Interestingly, the study by Nikolenko <em>et al</em>. also reported that the reversion of mutation I326V increased AZT resistance <a href="#pone.0001781-DelviksFrankenberry1">[30]</a>. Here, the decrease in frequency of this mutation was associated with treatment (<a href="#pone-0001781-g002">Figures 2</a> and <a href="#pone-0001781-g003">3</a>). Additionally, Cane <em>et al</em>. correlated some mutations in the RT connection domain with TAMs, G359S, A360V/T and A371V <a href="#pone.0001781-Cane1">[31]</a>. Once more, all those changes were evidenced in our analyses.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p>Among the NRTI-related mutations characterized herein, four of them have been associated with AZT in patients with exclusive exposure to that drug. Those included mutations A360V and A371T, found in patients with TAMs <a href="#pone.0001781-Nikolenko2">[32]</a>. We may speculate that, in addition to AZT, other thymidine analogues (such as d4T) may also select for those mutations, but additional experimental and analytical data is necessary to confirm this hypothesis.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p>This work was the first to comprehensively evaluate genotypic variation in the C-terminal domains of RT in the context of drug resistance <em>in vivo</em> and in most HIV-1 subtypes. The findings corroborate data by other studies <a href="#pone.0001781-Nikolenko1">[11]</a>, <a href="#pone.0001781-GoobarLarsoon1">[29]</a>–<a href="#pone.0001781-DelviksFrankenberry1">[30]</a> and expand our knowledge about the influence of C-terminal RT domains in antiretroviral drug resistance. We also elaborated a panel of conserved residues for HIV-1 group M independent of treatment which we believe is useful for the design of new drugs that target those domains. The role of RT mutations characterized herein is still undetermined, but they may become pivotal in developing optimized genotyping HIV drug resistance algorithms.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are in debt to all medical staff from all three HIV/AIDS clinical sites involved in the study. We also thank all patients who agreed to participate in this study. This study is part of A.F.A.S. PhD thesis at the Graduate Program in Genetics at UFRJ.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS VP. Performed the experiments: ES AS RL. Analyzed the data: MS AS VP RL AJ. Contributed reagents/materials/analysis tools: ES AM JS FS. Wrote the paper: MS AS VP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001781-Lightfoote1" id="pone.0001781-Lightfoote1"></a>Lightfoote MM, Coligan JE, Folks TM, Fauci AS, Martin MA, et al.  (1986) Structural characterization of reverse transcriptase and endonuclease polypeptides of the acquired immunodeficiency syndrome retrovirus. J Virol  60: 771–775.  <ul class="find" data-citedArticleID="1072926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Structural+characterization+of+reverse+transcriptase+and+endonuclease+polypeptides+of+the+acquired+immunodeficiency+syndrome+retrovirus.&amp;auth=&amp;atitle=Structural+characterization+of+reverse+transcriptase+and+endonuclease+polypeptides+of+the+acquired+immunodeficiency+syndrome+retrovirus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+characterization+of+reverse+transcriptase+and+endonuclease+polypeptides+of+the+acquired+immunodeficiency+syndrome+retrovirus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+characterization+of+reverse+transcriptase+and+endonuclease+polypeptides+of+the+acquired+immunodeficiency+syndrome+retrovirus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001781-JacoboMolina1" id="pone.0001781-JacoboMolina1"></a>Jacobo-Molina A, Ding J, Nanni RG, Clark AD Jr, Lu X, et al.  (1993) Crystal structure of human immunodeficiency virus type 1 reverse transcriptase complexed with double-stranded DNA at 3.0 A resolution shows bent DNA. Proc Natl Acad Sci U S A  90: 6320–6324.  <ul class="find" data-citedArticleID="1072910" data-doi="10.1073/pnas.90.13.6320"><li><a href="http://dx.doi.org/10.1073/pnas.90.13.6320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+complexed+with+double-stranded+DNA+at+3.0+A+resolution+shows+bent+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+complexed+with+double-stranded+DNA+at+3.0+A+resolution+shows+bent+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001781-Kohlstaedt1" id="pone.0001781-Kohlstaedt1"></a>Kohlstaedt LA, Wang J, Friedman JM, Rice PA, Steitz TA (1992) Crystal structure at 3.5 A resolution of HIV-1 reverse transcriptase complexed with an inhibitor. Science  256: 1783–1790.  <ul class="find" data-citedArticleID="1072920" data-doi="10.1126/science.1377403"><li><a href="http://dx.doi.org/10.1126/science.1377403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+at+3.5+A+resolution+of+HIV-1+reverse+transcriptase+complexed+with+an+inhibitor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+at+3.5+A+resolution+of+HIV-1+reverse+transcriptase+complexed+with+an+inhibitor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001781-Rodgers1" id="pone.0001781-Rodgers1"></a>Rodgers DW, Gamblin SJ, Harris BA, Ray S, Culp JS, et al.  (1995) The structure of unliganded reverse transcriptase from the human immunodeficiency virus type 1. Proc Natl Acad Sci U S A  92: 1222–1226.  <ul class="find" data-citedArticleID="1072938" data-doi="10.1073/pnas.92.4.1222"><li><a href="http://dx.doi.org/10.1073/pnas.92.4.1222" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+unliganded+reverse+transcriptase+from+the+human+immunodeficiency+virus+type+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+unliganded+reverse+transcriptase+from+the+human+immunodeficiency+virus+type+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001781-Coffin1" id="pone.0001781-Coffin1"></a>Coffin JM, Hughes SH, Varmus HE (1997) Retroviruses. New York: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0001781-Smith1" id="pone.0001781-Smith1"></a>Smith JS, Roth MJ (1992) Specificity of human immunodeficiency virus-1 reverse transcriptase-associated ribonuclease H in removal of the minus-strand primer, tRNA(Lys3). J Biol Chem  267: 15071–15079.  <ul class="find" data-citedArticleID="1072942"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Specificity+of+human+immunodeficiency+virus-1+reverse+transcriptase-associated+ribonuclease+H+in+removal+of+the+minus-strand+primer%2C+tRNA%28Lys3%29.&amp;auth=&amp;atitle=Specificity+of+human+immunodeficiency+virus-1+reverse+transcriptase-associated+ribonuclease+H+in+removal+of+the+minus-strand+primer%2C+tRNA%28Lys3%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specificity+of+human+immunodeficiency+virus-1+reverse+transcriptase-associated+ribonuclease+H+in+removal+of+the+minus-strand+primer%2C+tRNA%28Lys3%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specificity+of+human+immunodeficiency+virus-1+reverse+transcriptase-associated+ribonuclease+H+in+removal+of+the+minus-strand+primer%2C+tRNA%28Lys3%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001781-Wohrl1" id="pone.0001781-Wohrl1"></a>Wohrl BM, Moelling K (1990) Interaction of HIV-1 ribonuclease H with polypurine tract containing RNA-DNA hybrids. Biochemistry  29: 10141–10147.  <ul class="find" data-citedArticleID="1072952" data-doi="10.1021/bi00496a001"><li><a href="http://dx.doi.org/10.1021/bi00496a001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+of+HIV-1+ribonuclease+H+with+polypurine+tract+containing+RNA-DNA+hybrids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+of+HIV-1+ribonuclease+H+with+polypurine+tract+containing+RNA-DNA+hybrids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001781-Johnson1" id="pone.0001781-Johnson1"></a>Johnson VA, Brun-Vezinet F, Clotet B, Kuritzkes DR, Pillay D, et al.  (2006) Update of the drug resistance mutations in HIV-1: Fall 2006. Top HIV Med  14: 125–130.  <ul class="find" data-citedArticleID="1072912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006.&amp;auth=&amp;atitle=Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001781-Rhee1" id="pone.0001781-Rhee1"></a>Rhee SY, Kantor R, Katzenstein DA, Camacho R, Morris L, et al.  (2006) HIV-1 pol mutation frequency by subtype and treatment experience: extension of the HIVseq program to seven non-B subtypes. AIDS  20: 643–651.  <ul class="find" data-citedArticleID="1072932" data-doi="10.1097/01.aids.0000216363.36786.2b"><li><a href="http://dx.doi.org/10.1097/01.aids.0000216363.36786.2b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+pol+mutation+frequency+by+subtype+and+treatment+experience%3A+extension+of+the+HIVseq+program+to+seven+non-B+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+pol+mutation+frequency+by+subtype+and+treatment+experience%3A+extension+of+the+HIVseq+program+to+seven+non-B+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001781-Rhee2" id="pone.0001781-Rhee2"></a>Rhee SY, Fessel WJ, Zolopa AR, Hurley L, Liu T, et al.  (2005) HIV-1 Protease and reverse-transcriptase mutations: correlations with antiretroviral therapy in subtype B isolates and implications for drug-resistance surveillance. J Infect Dis  192: 456–465.  <ul class="find" data-citedArticleID="1072934" data-doi="10.1086/431601"><li><a href="http://dx.doi.org/10.1086/431601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+Protease+and+reverse-transcriptase+mutations%3A+correlations+with+antiretroviral+therapy+in+subtype+B+isolates+and+implications+for+drug-resistance+surveillance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+Protease+and+reverse-transcriptase+mutations%3A+correlations+with+antiretroviral+therapy+in+subtype+B+isolates+and+implications+for+drug-resistance+surveillance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001781-Nikolenko1" id="pone.0001781-Nikolenko1"></a>Nikolenko GN, Palmer S, Maldarelli F, Mellors JW, Coffin JM, et al.  (2005) Mechanism for nucleoside analog-mediated abrogation of HIV-1 replication: balance between RNase H activity and nucleotide excision. Proc Natl Acad Sci U S A  102: 2093–2098.  <ul class="find" data-citedArticleID="1072928" data-doi="10.1073/pnas.0409823102"><li><a href="http://dx.doi.org/10.1073/pnas.0409823102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+for+nucleoside+analog-mediated+abrogation+of+HIV-1+replication%3A+balance+between+RNase+H+activity+and+nucleotide+excision." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+for+nucleoside+analog-mediated+abrogation+of+HIV-1+replication%3A+balance+between+RNase+H+activity+and+nucleotide+excision.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001781-Julias1" id="pone.0001781-Julias1"></a>Julias JG, McWilliams MJ, Sarafianos SG, Alvord WG, Arnold E, et al.  (2003) Mutation of amino acids in the connection domain of human immunodeficiency virus type 1 reverse transcriptase that contact the template-primer affects RNase H activity. J Virol  77: 8548–8554.  <ul class="find" data-citedArticleID="1072914" data-doi="10.1128/jvi.77.15.8548-8554.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.15.8548-8554.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+amino+acids+in+the+connection+domain+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+that+contact+the+template-primer+affects+RNase+H+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+amino+acids+in+the+connection+domain+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+that+contact+the+template-primer+affects+RNase+H+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001781-Stuyver1" id="pone.0001781-Stuyver1"></a>Stuyver L, Wyseur A, Rombout A, Louwagie J, Scarcez T, et al.  (1997) Line probe assay for rapid detection of drug-selected mutations in the human immunodeficiency virus type 1 reverse transcriptase gene. Antimicrob Agents Chemother  41: 284–291.  <ul class="find" data-citedArticleID="1072944"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Line+probe+assay+for+rapid+detection+of+drug-selected+mutations+in+the+human+immunodeficiency+virus+type+1+reverse+transcriptase+gene.&amp;auth=&amp;atitle=Line+probe+assay+for+rapid+detection+of+drug-selected+mutations+in+the+human+immunodeficiency+virus+type+1+reverse+transcriptase+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Line+probe+assay+for+rapid+detection+of+drug-selected+mutations+in+the+human+immunodeficiency+virus+type+1+reverse+transcriptase+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Line+probe+assay+for+rapid+detection+of+drug-selected+mutations+in+the+human+immunodeficiency+virus+type+1+reverse+transcriptase+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001781-Kuiken1" id="pone.0001781-Kuiken1"></a>Kuiken C, Korber B, Shafer RW (2003) HIV sequence databases. AIDS Rev  5: 52–61.  <ul class="find" data-citedArticleID="1072922"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=HIV+sequence+databases.&amp;auth=&amp;atitle=HIV+sequence+databases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+sequence+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+sequence+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001781-Kumar1" id="pone.0001781-Kumar1"></a>Kumar S, Tamura JK, Jakobsen IB, Nei M (2001) MEGA2: molecular evolutionary genetics analysis software. Bioinformatics  17: 1244–1245.  <ul class="find" data-citedArticleID="1072924" data-doi="10.1093/bioinformatics/17.12.1244"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.12.1244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA2%3A+molecular+evolutionary+genetics+analysis+software." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA2%3A+molecular+evolutionary+genetics+analysis+software.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001781-Kane1" id="pone.0001781-Kane1"></a>Kane CT, Montavon C, Toure MA, Faye MA, Ndiaye AG, et al.  (2001) Full-length genome sequencing of HIV type 1 group O viruses isolated from a heterosexual transmission cluster in Senegal. AIDS Res Hum Retroviruses  17: 1211–1216.  <ul class="find" data-citedArticleID="1072918" data-doi="10.1089/088922201316912817"><li><a href="http://dx.doi.org/10.1089/088922201316912817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full-length+genome+sequencing+of+HIV+type+1+group+O+viruses+isolated+from+a+heterosexual+transmission+cluster+in+Senegal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full-length+genome+sequencing+of+HIV+type+1+group+O+viruses+isolated+from+a+heterosexual+transmission+cluster+in+Senegal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001781-Rhee3" id="pone.0001781-Rhee3"></a>Rhee SY, Gonzales MJ, Kantor R, Betts BJ, Ravela J, et al.  (2003) Human immunodeficiency virus reverse transcriptase and protease sequence database. Nucleic Acids Res  31: 298–303.  <ul class="find" data-citedArticleID="1072936" data-doi="10.1093/nar/gkg100"><li><a href="http://dx.doi.org/10.1093/nar/gkg100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+reverse+transcriptase+and+protease+sequence+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+reverse+transcriptase+and+protease+sequence+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001781-Thompson1" id="pone.0001781-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="1072946" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001781-Tippmann1" id="pone.0001781-Tippmann1"></a>Tippmann HF (2004) Analysis for free: comparing programs for sequence analysis. Brief Bioinform  5: 82–87.  <ul class="find" data-citedArticleID="1072948" data-doi="10.1093/bib/5.1.82"><li><a href="http://dx.doi.org/10.1093/bib/5.1.82" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+for+free%3A+comparing+programs+for+sequence+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+for+free%3A+comparing+programs+for+sequence+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001781-Sarafianos1" id="pone.0001781-Sarafianos1"></a>Sarafianos SG, Das K, Tantillo C, Clark AD Jr, Ding J, et al.  (2001) Crystal structure of HIV-1 reverse transcriptase in complex with a polypurine tract RNA:DNA. EMBO J  20: 1449–1461.  <ul class="find" data-citedArticleID="1072940" data-doi="10.1093/emboj/20.6.1449"><li><a href="http://dx.doi.org/10.1093/emboj/20.6.1449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+HIV-1+reverse+transcriptase+in+complex+with+a+polypurine+tract+RNA%3ADNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+HIV-1+reverse+transcriptase+in+complex+with+a+polypurine+tract+RNA%3ADNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001781-CeccheriniSilberstein1" id="pone.0001781-CeccheriniSilberstein1"></a>Ceccherini-Silberstein F, Erba F, Gago F, Bertoli A, Forbici F, et al.  (2004) Identification of the minimal conserved structure of HIV-1 protease in the presence and absence of drug pressure. AIDS  18: F11–F19.  <ul class="find" data-citedArticleID="1072896" data-doi="10.1097/01.aids.0000131394.76221.02"><li><a href="http://dx.doi.org/10.1097/01.aids.0000131394.76221.02" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+the+minimal+conserved+structure+of+HIV-1+protease+in+the+presence+and+absence+of+drug+pressure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+the+minimal+conserved+structure+of+HIV-1+protease+in+the+presence+and+absence+of+drug+pressure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001781-CeccheriniSilberstein2" id="pone.0001781-CeccheriniSilberstein2"></a>Ceccherini-Silberstein F, Gago F, Santoro M, Gori C, Svicher V, et al.  (2005) High sequence conservation of human immunodeficiency virus type 1 reverse transcriptase under drug pressure despite the continuous appearance of mutations. J Virol  79: 10718–10729.  <ul class="find" data-citedArticleID="1072898" data-doi="10.1128/jvi.79.16.10718-10729.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.16.10718-10729.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+sequence+conservation+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+under+drug+pressure+despite+the+continuous+appearance+of+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+sequence+conservation+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+under+drug+pressure+despite+the+continuous+appearance+of+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001781-Julias2" id="pone.0001781-Julias2"></a>Julias JG, McWilliams MJ, Sarafianos SG, Arnold E, Hughes SH (2002) Mutations in the RNase H domain of HIV-1 reverse transcriptase affect the initiation of DNA synthesis and the specificity of RNase H cleavage in vivo. Proc Natl Acad Sci U S A  99: 9515–9520.  <ul class="find" data-citedArticleID="1072916" data-doi="10.1073/pnas.142123199"><li><a href="http://dx.doi.org/10.1073/pnas.142123199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+RNase+H+domain+of+HIV-1+reverse+transcriptase+affect+the+initiation+of+DNA+synthesis+and+the+specificity+of+RNase+H+cleavage+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+RNase+H+domain+of+HIV-1+reverse+transcriptase+affect+the+initiation+of+DNA+synthesis+and+the+specificity+of+RNase+H+cleavage+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001781-Arion1" id="pone.0001781-Arion1"></a>Arion D, Sluis-Cremer N, Mim KL, Abram ME, Fletcher RS, et al.  (2002) Mutational analysis of Tyr-501 of HIV-1 reverse transcriptase. Effects on ribonuclease H activity and inhibition of this activity by N-acylhydrazones. J Biol Chem  277: 1370–1374.  <ul class="find" data-citedArticleID="1072892" data-doi="10.1074/jbc.m110254200"><li><a href="http://dx.doi.org/10.1074/jbc.m110254200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutational+analysis+of+Tyr-501+of+HIV-1+reverse+transcriptase.+Effects+on+ribonuclease+H+activity+and+inhibition+of+this+activity+by+N-acylhydrazones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutational+analysis+of+Tyr-501+of+HIV-1+reverse+transcriptase.+Effects+on+ribonuclease+H+activity+and+inhibition+of+this+activity+by+N-acylhydrazones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001781-Volkmann1" id="pone.0001781-Volkmann1"></a>Volkmann S, Wohrl BM, Tisdale M, Moelling K (1993) Enzymatic analysis of two HIV-1 reverse transcriptase mutants with mutations in carboxyl-terminal amino acids residues conserved among retroviral ribonucleases H. J Biol Chem  75: 71–78.  <ul class="find" data-citedArticleID="1072950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Enzymatic+analysis+of+two+HIV-1+reverse+transcriptase+mutants+with+mutations+in+carboxyl-terminal+amino+acids+residues+conserved+among+retroviral+ribonucleases+H.&amp;auth=&amp;atitle=Enzymatic+analysis+of+two+HIV-1+reverse+transcriptase+mutants+with+mutations+in+carboxyl-terminal+amino+acids+residues+conserved+among+retroviral+ribonucleases+H." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymatic+analysis+of+two+HIV-1+reverse+transcriptase+mutants+with+mutations+in+carboxyl-terminal+amino+acids+residues+conserved+among+retroviral+ribonucleases+H." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymatic+analysis+of+two+HIV-1+reverse+transcriptase+mutants+with+mutations+in+carboxyl-terminal+amino+acids+residues+conserved+among+retroviral+ribonucleases+H.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001781-Abram1" id="pone.0001781-Abram1"></a>Abram ME, Parniak MA (2005) Virion instability of human immunodeficiency virus type 1 reverse transcriptase (RT) mutated in the protease cleavage site between RT p51 and the RT RNase H domain. J Virol  79: 11952–11961.  <ul class="find" data-citedArticleID="1072890" data-doi="10.1128/jvi.79.18.11952-11961.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.18.11952-11961.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Virion+instability+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+%28RT%29+mutated+in+the+protease+cleavage+site+between+RT+p51+and+the+RT+RNase+H+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Virion+instability+of+human+immunodeficiency+virus+type+1+reverse+transcriptase+%28RT%29+mutated+in+the+protease+cleavage+site+between+RT+p51+and+the+RT+RNase+H+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001781-Davies1" id="pone.0001781-Davies1"></a>Davies JF 2nd, Hostomska Z, Hostomsky Z, Jordan SR, Matthews DA (1991) Crystal structure of the ribonuclease H domain of HIV-1 reverse transcriptase. Science  252: 88–95.  <ul class="find" data-citedArticleID="1072902" data-doi="10.1126/science.1707186"><li><a href="http://dx.doi.org/10.1126/science.1707186" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+ribonuclease+H+domain+of+HIV-1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+ribonuclease+H+domain+of+HIV-1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001781-Hehl1" id="pone.0001781-Hehl1"></a>Hehl EA, Joshi P, Kalpana GV, Prasad VR (2004) Interaction between human immunodeficiency virus type 1 reverse transcriptase and integrase proteins. J Virol  78: 5056–5067.  <ul class="find" data-citedArticleID="1072908" data-doi="10.1128/jvi.78.10.5056-5067.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.10.5056-5067.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+between+human+immunodeficiency+virus+type+1+reverse+transcriptase+and+integrase+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+between+human+immunodeficiency+virus+type+1+reverse+transcriptase+and+integrase+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001781-GoobarLarsoon1" id="pone.0001781-GoobarLarsoon1"></a>Goobar-Larsoon L, Backbro K, Unge T, Bhikhabhai R, Vrang L, et al.  (1993) Disruption of a salt bridge between Asp 488 and Lys 465 in HIV-1 reverse transcriptase alters its proteolytic processing and polymerase activity. Virology  196: 731–738.  <ul class="find" data-citedArticleID="1072906" data-doi="10.1006/viro.1993.1530"><li><a href="http://dx.doi.org/10.1006/viro.1993.1530" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disruption+of+a+salt+bridge+between+Asp+488+and+Lys+465+in+HIV-1+reverse+transcriptase+alters+its+proteolytic+processing+and+polymerase+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disruption+of+a+salt+bridge+between+Asp+488+and+Lys+465+in+HIV-1+reverse+transcriptase+alters+its+proteolytic+processing+and+polymerase+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001781-DelviksFrankenberry1" id="pone.0001781-DelviksFrankenberry1"></a>Delviks-Frankenberry KA, Nikolenko GN, Barr R, Pathak VKMutations in Human Immunodeficiency Virus Type-1 RNase H Primer Grip Enhance 3′-Azido-3′-Deoxythymidine Resistance. J. Virol.  81: 6837–6845.  <ul class="find" data-citedArticleID="1072904" data-doi="10.1128/jvi.02820-06"><li><a href="http://dx.doi.org/10.1128/jvi.02820-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+Human+Immunodeficiency+Virus+Type-1+RNase+H+Primer+Grip+Enhance+3%E2%80%B2-Azido-3%E2%80%B2-Deoxythymidine+Resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+Human+Immunodeficiency+Virus+Type-1+RNase+H+Primer+Grip+Enhance+3%E2%80%B2-Azido-3%E2%80%B2-Deoxythymidine+Resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001781-Cane1" id="pone.0001781-Cane1"></a>Cane PA, Green H, Fearnhill E, Dunn D, UK collaborative group on HIV Drug Resistance (2007) Identification of accessory mutations associated with high-level resistance in HIV-1 reverse transcriptase. AIDS  21: 447–455.  <ul class="find" data-citedArticleID="1072894" data-doi="10.1097/qad.0b013e3280129964"><li><a href="http://dx.doi.org/10.1097/qad.0b013e3280129964" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+accessory+mutations+associated+with+high-level+resistance+in+HIV-1+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+accessory+mutations+associated+with+high-level+resistance+in+HIV-1+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001781-Nikolenko2" id="pone.0001781-Nikolenko2"></a>Nikolenko GN, Delviks-Frankenberry KA, Palmer S, Maldarelli F, Fivash MJ JR, et al.  (2007) Mutations in the connection domain of HIV-1 reverse transcriptase increase 3′-azido-3′-deoxythymidine resistance. Proc Natl Acad Sci U S A  104: 317–322.  <ul class="find" data-citedArticleID="1072930" data-doi="10.1073/pnas.0609642104"><li><a href="http://dx.doi.org/10.1073/pnas.0609642104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+connection+domain+of+HIV-1+reverse+transcriptase+increase+3%E2%80%B2-azido-3%E2%80%B2-deoxythymidine+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+connection+domain+of+HIV-1+reverse+transcriptase+increase+3%E2%80%B2-azido-3%E2%80%B2-deoxythymidine+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.XML" value="77550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.PDF" value="422638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g001.PNG_L" value="2138764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g001.PNG_M" value="258384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g001.PNG_S" value="21609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g001.TIF" value="3100540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g001.PNG_I" value="226950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g002.PNG_L" value="1898193"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g002.PNG_M" value="125361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g002.PNG_S" value="7898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g002.TIF" value="3262282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g002.PNG_I" value="44649"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g003.PNG_L" value="1398046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g003.PNG_M" value="95762"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g003.PNG_S" value="6405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g003.TIF" value="2206234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g003.PNG_I" value="34335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.t001.PNG_L" value="188790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.t001.PNG_M" value="101383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.t001.PNG_S" value="11873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.t001.TIF" value="828150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.t001.PNG_I" value="31154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g004.PNG_L" value="2439894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g004.PNG_M" value="225240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g004.PNG_S" value="17565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g004.TIF" value="3452082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001781.g004.PNG_I" value="121317"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001781&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001781" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001781&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001781&volume=&issue=&title=Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients&author_name=Andr%C3%A9%20F.%20A.%20Santos%2C%20Renan%20B.%20Lengruber%2C%20Esmeralda%20A.%20Soares%2C%20Abhay%20Jere%2C%20Eduardo%20Sprinz%2C%20Ana%20M.%20B.%20Martinez%2C%20Jussara%20Silveira%2C%20Fernando%20S.%20Sion%2C%20Vinay%20K.%20Pathak%2C%20Marcelo%20A.%20Soares&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001781" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781&amp;t=Conservation%20Patterns%20of%20HIV-1%20RT%20Connection%20and%20RNase%20H%20Domains%3A%20Identification%20of%20New%20Mutations%20in%20NRTI-Treated%20Patients" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781&title=Conservation%20Patterns%20of%20HIV-1%20RT%20Connection%20and%20RNase%20H%20Domains%3A%20Identification%20of%20New%20Mutations%20in%20NRTI-Treated%20Patients&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781&amp;title=Conservation%20Patterns%20of%20HIV-1%20RT%20Connection%20and%20RNase%20H%20Domains%3A%20Identification%20of%20New%20Mutations%20in%20NRTI-Treated%20Patients" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001781&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Conservation Patterns of HIV-1 RT Connection and RNase H Domains: Identification of New Mutations in NRTI-Treated Patients';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001781';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Conservation%20Patterns%20of%20HIV-1%20RT%20Connection%20and%20RNase%20H%20Domains%3A%20Identification%20of%20New%20Mutations%20in%20NRTI-Treated%20Patients http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001781" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001781" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+sequence+analysis%22" title="Search for articles in the subject area:'Amino acid sequence analysis'"><div class="flagText">Amino acid sequence analysis</div></a>
              <div data-categoryid="20173" data-articleid="26386"
                   data-categoryname="Amino acid sequence analysis"
                   class="flagImage" title="Flag 'Amino acid sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="26386"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drug+therapy%22" title="Search for articles in the subject area:'Drug therapy'"><div class="flagText">Drug therapy</div></a>
              <div data-categoryid="18461" data-articleid="26386"
                   data-categoryname="Drug therapy"
                   class="flagImage" title="Flag 'Drug therapy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="26386"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribonucleases%22" title="Search for articles in the subject area:'Ribonucleases'"><div class="flagText">Ribonucleases</div></a>
              <div data-categoryid="46199" data-articleid="26386"
                   data-categoryname="Ribonucleases"
                   class="flagImage" title="Flag 'Ribonucleases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+analysis%22" title="Search for articles in the subject area:'Sequence analysis'"><div class="flagText">Sequence analysis</div></a>
              <div data-categoryid="47475" data-articleid="26386"
                   data-categoryname="Sequence analysis"
                   class="flagImage" title="Flag 'Sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26386"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26386"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1235'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3776'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5077&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7203">Re-analysis for multiple testing</a><br>Posted by Flandre</p>
      <p><a href="/annotation/listThread.action?root=8527">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
