

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001367"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001367" />

    <meta name="citation_title" content="Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness"/>
    <meta itemprop="name" content="Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness"/>

      <meta name="citation_author" content="Martha T. Hamblin"/>
            <meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Marilyn L. Warburton"/>
            <meta name="citation_author_institution" content="Centro Internacional de Mejoramiento de Maíz y Trigo, El Batan, Mexico, Mexico"/>
      <meta name="citation_author" content="Edward S. Buckler"/>
            <meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York, United States of America"/>
            <meta name="citation_author_institution" content="United States Department of Agriculture-Agricultural Research Service, Ithaca, New York, United States of America"/>

    <meta name="citation_date" content="2007/12/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001367.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1367"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=An evaluation of the utility of SSR loci as molecular markers in maize (Zea mays L.): comparisons with data from RFLPs and pedigree.; citation_author=JSC Smith; citation_author=ECL Chin; citation_author=H Shu; citation_author=OS Smith; citation_author=SJ Wall; citation_journal_title=Theor Appl Genet; citation_volume=95; citation_number=1; citation_pages=163-173; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Applications of single nucleotide polymorphisms in crop genetics.; citation_author=A Rafalski; citation_journal_title=Curr Opin Plant Biol; citation_volume=5; citation_number=2; citation_pages=94-100; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The evolution of molecular markers–just a matter of fashion?; citation_author=C Schlotterer; citation_journal_title=Nature Rev Genet; citation_volume=5; citation_number=3; citation_pages=63-69; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SNPs in ecology, evolution and conservation.; citation_author=PA Morin; citation_author=G Luikart; citation_author=RK Wayne; citation_journal_title=Trends Ecol Evol; citation_volume=19; citation_number=4; citation_pages=208-216; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Large scale genotyping of complex DNA.; citation_author=GC Kennedy; citation_author=H Matsuzaki; citation_author=S Dong; citation_author=WM Liu; citation_author=J Huang; citation_journal_title=Nature Biotechnol; citation_volume=21; citation_number=5; citation_pages=1233-1237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A review on SNP and other types of molecular markers and their use in animal genetics.; citation_author=A Vignal; citation_author=D Milan; citation_author=M San Cristobal; citation_author=A Eggen; citation_journal_title=Genet Sel Evol; citation_volume=34; citation_number=6; citation_pages=275-305; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Rate and pattern of mutation at microsatellite loci in maize.; citation_author=Y Vigouroux; citation_author=JS Jaqueth; citation_author=Y Matsuoko; citation_author=OS Smith; citation_author=WD Beavis; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=7; citation_pages=1251-60; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microsatellites: genomic distribution, putative functions, and mutational mechanisms: a review.; citation_author=YC Li; citation_author=AB Korol; citation_author=T Fahima; citation_author=A Beiles; citation_author=E Nevo; citation_journal_title=Mol Ecol; citation_volume=11; citation_number=8; citation_pages=2453-2465; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microsatellites: simple sequences with complex evolution.; citation_author=H Ellegren; citation_journal_title=Nature Rev Genet; citation_volume=5; citation_number=9; citation_pages=435-445; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Whole-genome scan, in a complex disease, using 11,245 single-nucleotide polymorphisms: comparison with microsatellites.; citation_author=S John; citation_author=N Shephard; citation_author=G Liu; citation_author=E Zeggini; citation_author=M Cao; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=10; citation_pages=54-64; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A method for the efficient management and utilization of large germplasm collections.; citation_author=DZ Skinner; citation_author=GR Bauchan; citation_author=G Auricht; citation_author=S Hughes; citation_journal_title=Crop Sci; citation_volume=39; citation_number=11; citation_pages=1237-1242; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Assessment of genetic diversity in three subsets constituted from the ICRISAT sorghum collection using random vs. non-random sampling procedures: A. Using morpho-agronomical and passport data.; citation_author=C Grenier; citation_author=PJ Bramel-Cox; citation_author=M Noirot; citation_author=KE Prasada-Rao; citation_author=P Hamon; citation_journal_title=Theor Appl Genet; citation_volume=101; citation_number=12; citation_pages=190-196; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Relationship between heterosis and heterozygosity at marker loci: a theoretical computation.; citation_author=A Charcosset; citation_author=M Lefort-Buson; citation_author=A Gallais; citation_journal_title=Theor Appl Genet; citation_volume=81; citation_number=13; citation_pages=571-575; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Overview of heterosis and heterotic groups in agronomic crops.; citation_author=AE Melchinger; citation_author=RK Gumber; citation_number=14; citation_pages=29-44; citation_date=1998; citation_publisher=CSSA; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity in tropical maize inbred lines: heterotic group assignment and hybrid performance determined by RFLP markers.; citation_author=LL Benchimol; citation_author=CL de Souza; citation_author=AAF Garcia; citation_author=PMS Kono; citation_author=CA Mangolin; citation_journal_title=Plant Breeding; citation_volume=119; citation_number=15; citation_pages=491-496; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data.; citation_author=JK Pritchard; citation_author=M Stephens; citation_author=P Donnelly; citation_journal_title=Genetics; citation_volume=155; citation_number=16; citation_pages=945-959; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Informativeness of genetic markers for inference of ancestry.; citation_author=NA Rosenberg; citation_author=LM Li; citation_author=R Ward; citation_author=JK Pritchard; citation_journal_title=Am J Hum Genet; citation_volume=73; citation_number=17; citation_pages=1402-1422; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparison of Single Nucleotide Polymorphisms and microsatellites in inference of population structure.; citation_author=N Liu; citation_author=L Chen; citation_author=S Wang; citation_author=C Oh; citation_author=H Zhao; citation_journal_title=BMC Genetics; citation_volume=6; citation_number=18; citation_pages=S26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A comparison of simple sequence repeat and single nucleotide polymorphism marker technologies for the genotypic analysis of maize (Zea mays L.); citation_author=ES Jones; citation_author=H Sullivan; citation_author=D Bhattramakki; citation_author=JSC Smith; citation_journal_title=Theor Appl Genet; citation_volume=115; citation_number=19; citation_pages=361-371; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites.; citation_author=K Liu; citation_author=M Goodman; citation_author=S Muse; citation_author=JS Smith; citation_author=E Buckler; citation_journal_title=Genetics; citation_volume=165; citation_number=20; citation_pages=2117-2128; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identifying adaptive genetic divergence among populations from genome scans.; citation_author=MA Beaumont; citation_author=DJ Balding; citation_journal_title=Mol Ecol; citation_volume=13; citation_number=21; citation_pages=969-980; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Measuring genetic distances between breeds: use of some distances in various short term evolution models.; citation_author=G Laval; citation_author=M San Cristobal; citation_author=C Chevalet; citation_journal_title=Genet Sel Evol; citation_volume=34; citation_number=22; citation_pages=481-507; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetic characterization of CIMMYT inbred maize lines and open pollinated populations using large scale fingerprinting methods.; citation_author=ML Warburton; citation_author=X Xianchun; citation_author=J Crossa; citation_author=J Franco; citation_author=AE Melchinger; citation_journal_title=Crop Sci; citation_volume=42; citation_number=23; citation_pages=1832-1840; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genetic characterization of 218 elite CIMMYT inbred maize lines using RFLP markers.; citation_author=ML Warburton; citation_author=JM Ribaut; citation_author=J Franco; citation_author=J Crossa; citation_author=P Dubreuil; citation_journal_title=Euphytica; citation_volume=142; citation_number=24; citation_pages=97-106; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity and heterosis. Chapter 10.; citation_author=AE Melchinger; citation_number=25; citation_date=1999; citation_publisher=CSSA; " />
      <meta name="citation_reference" content="citation_title=Relatedness estimation with different numbers of background markers and association mapping with different sample sizes.; citation_author=J Yu; citation_author=Z Zhang; citation_author=DA Tabanao; citation_author=G Pressoir; citation_author=MR Tuinstra; citation_journal_title=Theor Appl Genet. In press; citation_number=26; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Tropical maize germplasm: what can we say about its genetic diversity in the light of molecular markers?; citation_author=PR Laborda; citation_author=KM Oliveira; citation_author=AAF Garcia; citation_author=MEAGZ Paterniani; citation_author=AP de Souza; citation_journal_title=Theor Appl Genet; citation_volume=111; citation_number=27; citation_pages=1288-1299; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Probabilistic models for collecting genetic diversity: comparisons, caveats, and limitations.; citation_author=DR Lockwood; citation_author=CM Richards; citation_author=GM Volk; citation_journal_title=Crop Sci; citation_volume=47; citation_number=28; citation_pages=861-868; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Patterns of DNA sequence polymorphism along chromosome 1 of maize (Zea mays ssp. mays L.).; citation_author=MI Tenaillon; citation_author=MC Sawkins; citation_author=AD Long; citation_author=RL Gaut; citation_author=JF Doebley; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=29; citation_pages=9161-9166; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.; citation_author=F Tajima; citation_journal_title=Genetics; citation_volume=123; citation_number=30; citation_pages=585-595; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Integrating patterns of polymorphism at SNPs and STRs.; citation_author=BA Payseur; citation_author=AD Cutter; citation_journal_title=Trends Genet; citation_volume=22; citation_number=31; citation_pages=424-429; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The effects of artificial selection on the maize genome.; citation_author=SI Wright; citation_author=IV Bi; citation_author=SG Schroeder; citation_author=M Yamasaki; citation_author=JF Doebley; citation_journal_title=Science; citation_volume=308; citation_number=32; citation_pages=1310-1314; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.; citation_author=J Yu; citation_author=G Pressoir; citation_author=WH Briggs; citation_author=IV Bi; citation_author=M Yamasaki; citation_journal_title=Nature Genet; citation_volume=38; citation_number=33; citation_pages=203-208; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Anchoring 9,371 maize expressed sequence tagged unigenes to the bacterial artificial chromosome contig map by two-dimensional overgo hybridization.; citation_author=J Gardiner; citation_author=S Schroeder; citation_author=ML Polacco; citation_author=H Sanchez-Villeda; citation_author=Z Fang; citation_journal_title=Plant Physiol; citation_volume=134; citation_number=34; citation_pages=1317-26; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The use of MassARRAY technology for high throughput genotyping.; citation_author=C Jurinke; citation_author=D van den Boom; citation_author=CR Cantor; citation_author=H Koster; citation_journal_title=Adv Biochem Eng Biotechnol; citation_volume=77; citation_number=35; citation_pages=57-74; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PowerMarker: an integrated analysis environment for genetic marker analysis.; citation_author=K Liu; citation_author=SV Muse; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=36; citation_pages=2128-2129; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Generating samples under a Wright-Fisher neutral model of genetic variation.; citation_author=RR Hudson; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=37; citation_pages=337-338; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness"/>
    <meta name="twitter:description" content="While Simple Sequence Repeats (SSRs) are extremely useful genetic markers, recent advances in technology have produced a shift toward use of single nucleotide polymorphisms (SNPs). The different mutational properties of these two classes of markers result in differences in heterozygosities and allele frequencies that may have implications for their use in assessing relatedness and evaluation of genetic diversity. We compared analyses based on 89 SSRs (primarily dinucleotide repeats) to analyses based on 847 SNPs in individuals from the same 259 inbred maize lines, which had been chosen to represent the diversity available among current and historic lines used in breeding. The SSRs performed better at clustering germplasm into populations than did a set of 847 SNPs or 554 SNP haplotypes, and SSRs provided more resolution in measuring genetic distance based on allele-sharing. Except for closely related pairs of individuals, measures of distance based on SSRs were only weakly correlated with measures of distance based on SNPs. Our results suggest that 1) large numbers of SNP loci will be required to replace highly polymorphic SSRs in studies of diversity and relatedness and 2) relatedness among highly-diverged maize lines is difficult to measure accurately regardless of the marker system."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001367.g004"/>

  <meta property="og:title" content="Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4809'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7949'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1629&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001367">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001367" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001367&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Martha T. Hamblin
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mth3@cornell.edu">mth3@cornell.edu</a></p>

                <p>Affiliation:
                  Institute for Genomic Diversity, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marilyn L. Warburton, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centro Internacional de Mejoramiento de Maíz y Trigo, El Batan, Mexico, Mexico
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edward S. Buckler
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Institute for Genomic Diversity, Cornell University, Ithaca, New York, United States of America, 
                  United States Department of Agriculture-Agricultural Research Service, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001367</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001367-g001" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-t001" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-t002" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-g002" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-t003" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-g003" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-g004" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-t004" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-t005" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001367-t006" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001367">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001367" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>While Simple Sequence Repeats (SSRs) are extremely useful genetic markers, recent advances in technology have produced a shift toward use of single nucleotide polymorphisms (SNPs). The different mutational properties of these two classes of markers result in differences in heterozygosities and allele frequencies that may have implications for their use in assessing relatedness and evaluation of genetic diversity. We compared analyses based on 89 SSRs (primarily dinucleotide repeats) to analyses based on 847 SNPs in individuals from the same 259 inbred maize lines, which had been chosen to represent the diversity available among current and historic lines used in breeding. The SSRs performed better at clustering germplasm into populations than did a set of 847 SNPs or 554 SNP haplotypes, and SSRs provided more resolution in measuring genetic distance based on allele-sharing. Except for closely related pairs of individuals, measures of distance based on SSRs were only weakly correlated with measures of distance based on SNPs. Our results suggest that 1) large numbers of SNP loci will be required to replace highly polymorphic SSRs in studies of diversity and relatedness and 2) relatedness among highly-diverged maize lines is difficult to measure accurately regardless of the marker system.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hamblin MT, Warburton ML, Buckler ES (2007) Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness. PLoS ONE 2(12):
          e1367.
            doi:10.1371/journal.pone.0001367</p><p><strong>Academic Editor: </strong>Hans Ellegren, University of Uppsala, Sweden</p><p><strong>Received:</strong> October 5, 2007; <strong>Accepted:</strong> December 2, 2007; <strong>Published:</strong> December 26, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>NSF DBI 0321467, USDA-ARS, Institute for Genomic Diversity. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Until recently, Simple Sequence Repeats (SSRs), also called microsatellites, have been the genetic markers of choice, because they are economical to score, have high allelic diversity, and are usually selectively neutral <a href="#pone.0001367-Smith1">[1]</a>. Recent advances in technology, however, have produced a shift toward single nucleotide polymorphism (SNP) markers, particularly for model organisms with substantial genomic resources. Individual SNP markers, being biallelic, have lower information content than SSRs, but they occur at much higher density in the genome, are amenable to high-throughput methods such as genotyping arrays, and have lower genotyping error rates <a href="#pone.0001367-Rafalski1">[2]</a>–<a href="#pone.0001367-Kennedy1">[5]</a>. For an overview of many of the technical and statistical issues of SSR and SNP genotyping, see <a href="#pone.0001367-Vignal1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The different properties of SNP and SSR markers arise from inherent differences in their mutational processes as well as from biased sampling practices that intensify those differences. (In this paper, we use the term SNP to refer to a marker that is genotyped using high-throughput technology <a href="#pone.0001367-Schlotterer1">[3]</a>, not a polymorphism that has been scored in fully re-sequenced alleles.) Because nucleotide mutation rates are low (on the order of 10<sup>−8</sup>/bp/generation), the vast majority of SNPs are biallelic, and thus have a maximum heterozygosity of 0.5. Not only is the SSR mutational rate much higher (for dinucleotide repeats in maize, 5.2×10<sup>−4</sup> to 1.1×10<sup>−3</sup> /generation <a href="#pone.0001367-Vigouroux1">[7]</a>), but the slippage process can create a virtually unlimited number of new alleles <a href="#pone.0001367-Li1">[8]</a>, <a href="#pone.0001367-Ellegren1">[9]</a>. Maximal heterozygosity can thus approach 1.0, and frequencies are often skewed toward rare alleles. While re-sequencing studies can detect large numbers of singleton SNPs in many population samples (depending on population history), the selection of informative SNPs for genotyping studies (i.e., ascertainment) ensures that most SNP marker alleles will segregate at intermediate frequency, exaggerating the difference in frequency spectrum that already exists between SNPs and SSRs <a href="#pone.0001367-Schlotterer1">[3]</a>, <a href="#pone.0001367-Morin1">[4]</a>. This ascertainment bias must be corrected if SNP data are to be used in estimation of population genetic parameters.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The great interest in mapping genes affecting human health and the early use of SNP markers in this field have produced several comparisons of SSR and SNP markers in family-based linkage studies, e.g. <a href="#pone.0001367-John1">[10]</a>. In these studies, as well as in linkage studies of experimental populations, the mutational processes of the markers are not of great consequence because only a small number of meioses are captured by the data. In contrast, marker choice might have a greater effect on inferences about relatedness and genetic diversity among groups of individuals that are recently diverged on an evolutionary timescale, but are not considered relatives, such as populations of cases and controls, or diverse germplasm collections within a cultivated species. At this level of divergence, SNP-based distances will be due almost entirely to drift, while SSR-based distances will also be due in part to mutation.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Most crop plant germplasm collections have been assembled on the basis of geographic and phenotypic diversity, with the goal of capturing as much functional genetic diversity as possible. To allow more efficient exploitation of germplasm collections, core subsets are typically assembled by various criteria, e.g. <a href="#pone.0001367-Skinner1">[11]</a>, <a href="#pone.0001367-Grenier1">[12]</a>, and eventually characterized with presumably neutral molecular markers. Once the accessions have been scored with a common set of markers, estimation of genetic relationships among accessions provides information that is critical for choice of breeding material and for design of experimental crosses. In maize, for example, prediction of productive hybrid combinations and assignment of lines to heterotic groups has been based on the hypothesis that heterosis of hybrids increases monotonically with increasing genetic distance of the parents <a href="#pone.0001367-Charcosset1">[13]</a>–<a href="#pone.0001367-Benchimol1">[15]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Inference of ancestry among unrelated individuals is also important because of its influence on the accuracy of population-based methods for genetic mapping of complex traits: to prevent spurious results, ancestry must be assessed and accounted for in the analyses. The most widely-used approach is the model-based method of Pritchard <em>et al.</em> <a href="#pone.0001367-Pritchard1">[16]</a>, which assigns individuals membership in a population based on their alleles at a large number of genome-wide markers. Using this method, Rosenberg <em>et al.</em> <a href="#pone.0001367-Rosenberg1">[17]</a> found that dinucleotide repeat SSRs in humans are five to eight times as informative as random SNPs, a result that was confirmed by Liu <em>et al.</em> <a href="#pone.0001367-Liu1">[18]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Given the trend toward increased use of SNP markers, it is of interest to compare the performance of these two types of markers for characterization of the same set of individuals in a model crop plant, maize. In a recent study, Jones <em>et al.</em> <a href="#pone.0001367-Jones1">[19]</a> collected data for 80 SSRs and 187 SNPs in the same set of 58 inbred maize lines representing a subset of maize diversity derived from temperate Northern Flint and Southern Dent landraces. They demonstrated the technical advantages of SNPs over SSRs with respect to data quality, and found that measures of genetic distance based on SSRs, SNPs, and SNP haplotypes were not significantly correlated unless the inbreds were related by pedigree.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>We were interested in exploring further, with a larger data set, how evaluation of population structure and measures of genetic distance are affected by choice of marker type, and what properties of the markers are responsible for differences in their usefulness. In this paper, we compare analyses based on 89 SSRs (primarily dinucleotide repeats) to analyses based on 847 SNPs in individuals from the same 259 inbred maize lines. These lines represent the widest genetic diversity available among current and historic lines used in breeding <a href="#pone.0001367-Liu2">[20]</a>, including tropical and semi-tropical lines. We ask to what extent these two classes of markers provide concordant information about the structure of populations and the relationships among individuals. We also test whether any SNPs are more strongly differentiated than would be expected by chance, suggesting a history of positive selection.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Summary statistics</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>As expected, the allelic diversities and frequency spectra were very different for these data sets (<a href="#pone-0001367-t001">Table 1</a>, <a href="#pone-0001367-g001">Figure 1</a>). Due to the high rate of SSR mutation, this fairly modest number of SSR loci had a very large number of alleles, most at low frequencies in the population, and a large number of singletons. In contrast, the SNP loci showed large numbers of intermediate frequency alleles. Clearly, this was a consequence of ascertainment bias, since only SNPs that were discovered in a small set of lines were included in the study. Such bias is inherent to all SNP-based studies.</p>
<div class="figure" id="pone-0001367-g001"><div class="img"><a name="pone-0001367-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Allele frequency spectra for different classes of markers.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Note that the scale on the x axis is different for total SNPs, because only this class is biallelic.</p>
<span>doi:10.1371/journal.pone.0001367.g001</span></div><div class="figure" id="pone-0001367-t001"><div class="img"><a name="pone-0001367-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Properties of different marker sets.</span></strong></p><span>doi:10.1371/journal.pone.0001367.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To make greater use of the information content of the SNP data, we constructed haplotypes for those loci where more than one SNP had been scored from the same amplicon (see <a href="#s4">Methods</a>). The 554 loci in the “SNP Haplotypes” set had fewer total alleles than the 847 single SNPs because, at most loci, not all gametic types were observed (<a href="#pone-0001367-t002">Table 2</a>) and because more than half the loci were still single SNPs. However, expected heterozygosity was increased by about 20%, and the frequency spectrum shifted slightly toward rarer alleles, including substantially more singletons (<a href="#pone-0001367-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001367-t002"><div class="img"><a name="pone-0001367-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Number of haplotypes observed in multi-SNP amplicons.</span></strong></p><span>doi:10.1371/journal.pone.0001367.t002</span></div>

<h4>Assessment of population structure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Using SSR variation, Liu <em>et al.</em> <a href="#pone.0001367-Liu2">[20]</a> showed that maize diversity is best described as belonging to three population groups: Stiff Stalk (SS), Non-Stiff Stalk (NSS), and Tropical-Semitropical (TS). We used the same approach as Liu, namely the model-based method of Pritchard <em>et al.</em> <a href="#pone.0001367-Pritchard1">[16]</a>, to compare the SSR and SNP marker sets' ability to detect population structure and assign individuals to populations (see <a href="#s4">Methods</a>). <a href="#pone-0001367-g002">Figure 2</a> shows that, for all data sets, likelihood increased most when k (the number of populations in the model) was increased from two to three; results were very consistent across runs with k = 3 but became less consistent at higher values of k. Interestingly, the percent of individuals assigned to populations did not continue to increase with k, as might be expected: maximal assignment occurred at k = 2 , 3, or 4, depending on the markers used (<a href="#pone-0001367-t003">Table 3</a>), though many of the differences in percent assignment were small. The most dramatic difference was between k = 2 and k = 3 for the SNP+SSR data set: almost 20% more individuals were assigned under the model with k = 3. Strikingly, all assignment percentages for this data set were lower than for the SSR data set alone.</p>
<div class="figure" id="pone-0001367-g002"><div class="img"><a name="pone-0001367-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Estimated ln(probability of the data) and Var[ln(probability of the data)] for k from 2 to 5.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Values are from STRUCTURE run three times at each value of k, using A) 89 SSRs; B) 847 SNPs; C) 554 SNP haplotypes; D) 89 SSRs+847 SNPs. The blue diamonds are ln(probability of the data) and the pink squares are var[ln(probability of the data)].</p>
<span>doi:10.1371/journal.pone.0001367.g002</span></div><div class="figure" id="pone-0001367-t003"><div class="img"><a name="pone-0001367-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Percent population assignment based on different marker sets.</span></strong></p><span>doi:10.1371/journal.pone.0001367.t003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>For all values of k tested, it is clear that the SNP data did not contain sufficient information to resolve all the relationships that were detected by SSR variation, resulting in a lower percentage of individuals that could be assigned to a population. This was not due simply to the higher allele number, since a reduced data set of 80 SSRs with 1694 alleles (the same number as the SNP data set) resulted in the same percent assignment as the full data set (data not shown).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The assignment of individuals to populations was very consistent among marker types, for those individuals that were assigned. Assuming three populations as described by Liu <em>et al.</em> <a href="#pone.0001367-Liu2">[20]</a>, we plotted the relationship between membership in the NSS and TS populations based on SNPs and membership based on SSRs (<a href="#pone-0001367-g003">Figure 3</a>). The correlations were strong (R<sup>2</sup> = 0.88 and 0.93, respectively), but there was clearly much more spread along the y axes (SNPs) than along the x axes (SSRs): 116 lines had NSS membership between 0.2 and 0.8 based on SNPs, as opposed to only 58 lines that fell in that range based on SSRs. In no case was an individual assigned to a different population based on different marker information. In a small number of cases, SNP data resulted in assignments for individuals that were unassigned using SSR data. One such example is line F2834T, indicated by an arrow in each panel of <a href="#pone-0001367-g003">Figure 3</a>. Based on SNP data, F2834T had 82% of its ancestry in the TS cluster, while, based on SSR data, it had only 52% of its ancestry in that cluster and was thus classified as Mixed.</p>
<div class="figure" id="pone-0001367-g003"><div class="img"><a name="pone-0001367-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Comparison of membership in population clusters based on marker class.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Each point represents one individual's proportion of ancestry in the NSS (top panel) or TS (bottom panel) cluster, based on SSR (x axis) or SNP (y axis) data assuming three populations. The arrows indicate line F2834T (see text).</p>
<span>doi:10.1371/journal.pone.0001367.g003</span></div>

<h4>Distance measures</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Distance matrices based on allele sharing were constructed for all pairs of individuals using either SSR data or total SNP data, and the relationship between the distance matrices was plotted (<a href="#pone-0001367-g004">Figure 4</a>). For the small percentage of individual pairs that were closely related (SSR Distance &lt;0.65), there was a strong correlation between the distances for the two marker types (R<sup>2</sup> = 0.73, <a href="#pone-0001367-g004">Fig 4</a> top). However, the vast majority (97.7%) of SSR-based distances were large (≥0.65), and for those comparisons the correlation with SNP-based distance was modest (R<sup>2</sup> = 0.11). The lower panel of <a href="#pone-0001367-g004">Figure 4</a> shows that, while SSR-based distances varied from 0.65 to over 0.95, the SNPs provided poor resolution in this group, with distances that varied only from about 0.2 to 0.35. While comparisons at the individual level were not well-correlated across marker types, this was not the case for population-level comparisons: the correlation between allele-sharing distance among populations was &gt;0.99 (Mantel test).</p>
<div class="figure" id="pone-0001367-g004"><div class="img"><a name="pone-0001367-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Correlation between genetic distance based on SSRs and SNPs.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Each point represents the genetic distance between a pair of individuals, based on sharing of SSR (x axis) or SNP (y axis) alleles. The top panel shows the relationship for pairs of individuals whose SSR distance is &lt;0.65; the bottom panel shows the relationship for pairs of individuals whose SSR distance is ≥0.65.</p>
<span>doi:10.1371/journal.pone.0001367.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Given that the SSR data set had a much larger proportion of low frequency alleles, we tested whether this difference was responsible for the decrease in resolution provided by the SNP data set. To do this, we divided the SNP data into five frequency classes, calculated all pairwise distances based on each class, and calculated the correlation between those distances and the SSR-based distances. Although the results were slightly confounded by sample size, <a href="#pone-0001367-t004">Table 4</a> shows that the intermediate frequency SNPs were better able to resolve shorter distances, but that the low frequency SNPs contributed most of the resolution when distances were larger. Distances based on the SNPs at intermediate frequency are the best correlated with the distances based on SSRs, even though the SSR data set was dominated by rare alleles.</p>
<div class="figure" id="pone-0001367-t004"><div class="img"><a name="pone-0001367-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Correlation between genetic distance for SSRs and SNPs of different frequency classes.</span></strong></p><span>doi:10.1371/journal.pone.0001367.t004</span></div>

<h4>Core sets</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Liu <em>et al.</em> <a href="#pone.0001367-Liu2">[20]</a> found a total of 2039 SSR alleles in 260 inbred lines, and tested different sized “core sets” to see what fraction of the alleles could be captured. Because of the large number of rare alleles, 193 lines were needed to capture all 2039 alleles, and a set of 20 lines captured only 46% of the alleles. We repeated this analysis with our data sets, and found that a subset of 20 individuals captured almost all SNP alleles and over 90% of SNP haplotypes. <a href="#pone-0001367-t005">Table 5</a> shows the percent of alleles captured by subsets of 20, 30, 40, and 50 for the various marker data sets. When we removed nine loci from the SSR data set so that it had exactly 1694 alleles (like the SNP data set), the percent alleles captured by subsets of these sizes remained the same (data not shown).</p>
<div class="figure" id="pone-0001367-t005"><div class="img"><a name="pone-0001367-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Alleles captured by subsets of the data for different marker sets.</span></strong></p><span>doi:10.1371/journal.pone.0001367.t005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>This comparison is somewhat misleading, however, because of the ascertainment of the SNPs in our data set. If all 259 lines had been fully re-sequenced, there would be more rare alleles to be captured. We therefore simulated a set of 847 unlinked SNPs in 259 individuals under the standard neutral model (SNM), producing the full frequency spectrum of variation. As expected, the percent of alleles captured in a small core set dropped substantially (from 98% to 83% for a subset of 20), but was still much higher than the percent of SSR alleles captured in a core set of the same size (<a href="#pone-0001367-t005">Table 5</a>). Furthermore, the SNM likely overestimates the number of rare alleles that would be observed in a fully re-sequenced maize data set, due to the effect of the domestication bottleneck. Since the exact parameters appropriate for simulating such a data set are unknown (see <a href="#s3">Discussion</a>), we performed simulations under two reasonable bottleneck models for comparison to the SNM. As seen in <a href="#pone-0001367-t006">Table 6</a>, heterozygosity and the number of singletons in this data set were intermediate between the values observed in the ascertained SNP data set and under the SNM. The percent of alleles captured by these core sets was only slightly lower than was found for the ascertained data (<a href="#pone-0001367-t005">Table 5</a>), suggesting that we may not have missed a large number of rare SNP alleles.</p>
<div class="figure" id="pone-0001367-t006"><div class="img"><a name="pone-0001367-t006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001367" data-uri="info:doi/10.1371/journal.pone.0001367.t006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001367.t006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001367.t006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001367.t006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001367.t006.TIF"></span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Properties of data sets simulating full resequencing.</span></strong></p><span>doi:10.1371/journal.pone.0001367.t006</span></div>

<h4>Fst at individual loci</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Overall <em>F<sub>st</sub></em> was very similar regardless of marker set used (0.08–0.10), with <em>F<sub>st</sub></em> based on SSRs being in the lower end of the range, as expected because of their higher heterozygosity. <em>F<sub>st</sub></em> based on individual SNP loci, however, varied from 0 to 0.568; some of these differences in <em>F<sub>st</sub></em> may be due to the action of natural or artificial selection at particular loci. To test this hypothesis, we used the FDIST2 program <a href="#pone.0001367-Beaumont1">[21]</a>, which uses coalescent simulations to generate a neutral joint distribution of <em>F<sub>st</sub></em> and <em>H<sub>e</sub></em>. Loci that have an unusually large value of <em>F<sub>st</sub></em>, given their heterozygosity, are candidates for having experienced selection.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We found that 32 SNPs, about 3.8% of the 847 loci tested, had p-values &lt;0.05 (uncorrected for multiple tests), so there was little evidence for selection. P-values for two SNPs, both from the same locus (AY108077, in IBM2 Bin 6.04, with similarity to a calmodulin-binding heat-shock protein), were the only ones that were significant using a Bonferroni-corrected critical value of 0.05/847 = 0.000059. These two SNPs, whose alleles are in complete linkage disequilbrium (i.e., r<sup>2</sup> = 1), showed a pattern that was very common among the SNPs with higher <em>F<sub>st</sub></em>: the NSS and TS populations had the same allele at very high frequency, while the SS population had the alternate allele at a very high frequency. Usually the allele at high frequency in NSS and TS was also at high frequency in Mexican samples that included teosintes (these frequencies can be viewed at <a href="http://www.panzea.org">www.panzea.org</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Molecular markers serve a variety of purposes in genetic analysis. In this study we compared the ability of SSR and SNP markers to assess relatedness in a population where divergence times are long enough that most individuals are considered unrelated and the true relationships are unknown. We found that a set of 89 highly polymorphic SSRs performed better at clustering germplasm into populations than did a set of 847 SNPs or 554 SNP haplotypes, and that SSRs provided more resolution in measuring genetic distance based on allele-sharing. We observed only slightly higher resolution when we converted SNPs to haplotypes, perhaps because fewer than half our SNPs could be converted.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>These results are consistent with theoretical predictions <a href="#pone.0001367-Vignal1">[6]</a>, <a href="#pone.0001367-Laval1">[22]</a> and with other empirical studies, e.g. <a href="#pone.0001367-Rosenberg1">[17]</a>. According to Laval <em>et al.</em> <a href="#pone.0001367-Laval1">[22]</a>, (<em>k</em>-1) times more biallelic markers are needed to achieve the same genetic distance accuracy as a set of microsatellites with <em>k</em> alleles. In our case, the average number of alleles per SSR locus was about 20; thus we should need [(20–1) * 89] = 1691 SNP markers to achieve the same accuracy; we had almost exactly half that many SNP markers.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The difference in information content of SSRs and biallelic markers may also be due to differences in the frequency spectrum: SSR loci have many more low-frequency alleles than SNP loci, since at least half of all SNP alleles must be at high or intermediate frequency. This frequency difference is even greater when ascertained SNPs are genotyped.</p>

<h4>Population structure estimation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Population genetics theory predicts that “preferentially discovering SNPs with high heterozygosity leads to an underestimation of the magnitude of structure” <a href="#pone.0001367-Morin1">[4]</a>. Using SSR data and the model-based method of Pritchard <em>et al.</em> <a href="#pone.0001367-Pritchard1">[16]</a>, Liu <em>et al.</em> <a href="#pone.0001367-Liu2">[20]</a> found that the 259 maize inbred lines in this study were best described as belonging to three populations, with about 22% of the lines having mixed ancestry. We found that our SNP data could support a model of three or four populations and were largely consistent with the results of Liu <em>et al.</em>, although, regardless of the number of populations in the model, far more individuals (44.8–47.5%) were classified as mixed using SNP data. It was surprising that, as k was increased, the likelihood increased but the percent assignment declined.</p>


<h4>Measures of genetic distance in maize</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Our analysis showed that most maize lines in a diverse sample are separated by a large genetic distance and, consistent with the results of Jones <em>et al.</em> <a href="#pone.0001367-Jones1">[19]</a>, that measures of distance based on different markers were well-correlated only for the small subset of individuals that were closely related. Other studies of tropical maize, which is extremely diverse, have found that SSR variation does not provide evidence of population structure other than among individuals closely related by pedigree <a href="#pone.0001367-Warburton1">[23]</a>, <a href="#pone.0001367-Warburton2">[24]</a>. Collectively, these results suggest that relatedness among highly-diverged maize lines is difficult to measure accurately regardless of the marker system. This may explain the observation that only intragroup crosses show a correlation between parental genetic distance and midparent heterosis <a href="#pone.0001367-Melchinger2">[25]</a>. Individuals within groups have smaller genetic distances that can be more accurately assessed, while intergroup comparisons are in the range where resolution is poor (<a href="#pone-0001367-g003">Fig 3</a> top).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Yu <em>et al.</em> <a href="#pone.0001367-Yu1">[26]</a> used essentially the same data set as this one to test the sensitivity of relatedness estimation to marker number, as assessed by the power to detect QTL in mixed-model association mapping. Contrary to our results, they found that “the whole set of 89 SSR markers provided roughly the same amount of information as did the whole set of 912 SNP markers for relatedness construction.” This apparent discrepancy is probably due to the fact that, in the calculation of the kinship matrix, kinship estimates for individuals who were less related than average were all set to zero. Thus no attempt was made to estimate the larger genetic distances. Given this strategy, the difference in resolution was not of great practical importance. However, this may not be the optimal strategy. Accurate estimate of identity-by-state for thousands of markers may provide even greater power and control of type I errors.</p>


<h4>Implications for germplasm conservation</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The development of core sets is often described in terms of the fraction of SSR alleles retained in subsets of the collection. An implicit assumption is that the presence of unique SSR alleles in an individual or population is an indicator of the presence of unique functional variation. As stated in Laborda <em>et al.</em> <a href="#pone.0001367-Laborda1">[27]</a>, their goal was to “expose useful diversity for breeding purposes” by maximizing allelic diversity at markers. Similarly, Lockwood et al <a href="#pone.0001367-Lockwood1">[28]</a> state: “Sampling for allelic richness is important for conservation and in the development of genetic resource collections.” It is well known that most SSR variation is neutral, and that functional variation is much more likely to be associated with SNPs and indels in and around genes. Since a small sample of alleles captures most of the SNP variation segregating in a population, is it truly important to capture rare SSR alleles in germplasm collections?</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>We have shown that 98% of the (ascertained) SNP allelic variation in a sample of 259 diverse lines can be captured in a set of 20 lines, as opposed to only 48% of the SSR alleles. How much of this difference is due to the loss of rare SNP alleles due to ascertainment? Data sets simulating fully re-sequenced data showed that rare SNP alleles are indeed missed, however, because maize is not a population at equilibrium, it is not clear how many singletons would have been found by complete re-sequencing of these inbred lines. In a mix of landraces and inbreds, Tenaillon et al <a href="#pone.0001367-Tenaillon1">[29]</a> found an average D <a href="#pone.0001367-Tajima1">[30]</a> of about 0.1, and reported that average D was higher in the inbreds alone, indicating a loss of rare alleles. The bottleneck models reported in <a href="#pone-0001367-t005">Tables 5</a> and <a href="#pone-0001367-t006">6</a> are suggestive but do not reproduce the actual frequency spectrum in maize, so these results must be interpreted with caution. In any case, the number of lines required to capture all SNP alleles even in the non-bottlenecked population is far smaller than that required to capture all rare SSR alleles. Generalizing beyond maize, the extent to which rare alleles are lost in SNP genotyping studies will vary among species, since different population histories will produce different frequency spectra of variation. Eventually, the dramatically lower cost of new sequencing technologies may allow for full re-sequencing to replace SNP genotyping, eliminating the problem of ascertainment altogether.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>However, even if every SNP were observed and captured, SNP diversity at individual sites is unlikely to capture <em>all functional</em> variation, since different combinations of SNP alleles (both within and between loci) will give rise to haplotypes that may differ in their contributions to phenotypes of interest. SSR allelic diversity should be positively correlated with SNP haplotypic diversity for the simple reason that both attributes are functions of effective population size. (Due to their higher mutation rates, SSRs more accurately reflect effective population size in non-equilibrium populations.) As a proxy for larger effective population size, maximal SSR diversity ensures that novel combinations of functional SNP alleles are more likely to be captured in a sample. Note that this applies only at the genome-wide level: Payseur and Cutter <a href="#pone.0001367-Payseur1">[31]</a> showed that SSR heterozygosity and nucleotide heterozygosity at linked sites are only weakly correlated under a number of different scenarios (maximum correlation &lt;0.2), so we cannot expect that high SSR allelic diversity in a particular chromosomal region is indicative of high SNP or haplotype diversity as well. This suggests that capturing particular rare SSR alleles is not likely to capture rare functional variation at linked sites.</p>


<h4>General conclusions</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>While SNPs represent the latest technology, and have a number of technical advantages, there may be biological questions for which SSRs provide higher quality information. However, conclusions about the relative usefulness of SSRs and SNPs will vary depending on the specifics of the particular study. Because the SSR mutational process is very heterogeneous among loci <a href="#pone.0001367-Ellegren1">[9]</a>, results of these kinds of comparisons may depend on the particular SSR loci, or class of loci (e.g., repeat-type), surveyed. In addition, the choice of marker type, and the number of loci needed, is a function of the evolutionary history of the populations being investigated <a href="#pone.0001367-Morin1">[4]</a>, <a href="#pone.0001367-Ellegren1">[9]</a>. Extensive simulation studies are needed to understand how the interaction between mutational processes and population history affects population genetic inferences.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plant material</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Individuals from 259 maize inbred lines were used in this study: 33-16, 38-11, 4226, A188, A214N, A239, A272, A441-5, A554, A556, A6, A619, A632, A634, A635, A641, A654, A659, A661, A679, A680, A682, B10, B103, B104, B105, B109, B115, B14A, B164, B2, B37, B46, B52, B57, B64, B68, B73, B73Htrhm, B75, B76, B77, B79, B84, B97, C103, C123, C49A, CH701-30, CH9, CI187-2, CI21E, CI28A, CI31A, CI3A, CI64, CI66, CI.7, CI90C, CI91B, CM174, CM37, CM7, CML10, CML103, CML108, CML11, CML14, CML154Q, CML157Q, CML158Q, CML218, CML220, CML228, CML238, CML247, CML254, CML258, CML261, CML264, CML277, CML281, CML287, CML311, CML314, CML321, CML322, CML323, CML328, CML331, CML332, CML333, CML341, CML38, CML45, CML5, CML52, CML61, CML69, CML77, CML91, CML92, CMV3, CO106, CO125, CO255, DE1, DE_2, DE_3, DE811, E2558W, EP1, F2834T, F44, F6, F7, GA209, GT112, H105W, H49, H84, H91, H95, H99, Hi27, Hy, I137TN, I205, Ia5125, IDS91, K148, K4, K55, K64, Ki11, Ki14, Ki21, Ki3, Ki43, Ki44, Ky21, Ky226, Ky228, L317, L578, M14, M162W, M37W, MEF156-55-2, Mo17, Mo18W, Mo1W, Mo24W, Mo44, Mo45, Mo46, Mo47, MoG, Mp339, MS1334, MS153, MS71, Mt42, N192, N28Ht, N6, N7A, NC222, NC230, NC232, NC236, NC238, NC250, NC258, NC260, NC262, NC264, NC290A, NC294, NC296, NC296A, NC298, NC300, NC302, NC304, NC306, NC310, NC314, NC318, NC320, NC324, NC326, NC328, NC33, NC336, NC338, NC340, NC342, NC344, NC346, NC348, NC350, NC352, NC354, NC356, NC358, NC360, NC362, NC364, NC366, NC368, ND246, Oh40B, Oh43, Oh43E, Oh603, Oh7B, Os420, Pa762, Pa875, Pa880, Pa91, R109B, R168, R177, R229, R4, SC213R, SC357, SD44, T232, T234, T8, Tx303, Tx601, Tzi10, Tzi16, Tzi18, Tzi25, Tzi8, Tzi9, U267Y, Va102, Va14, Va17, Va22, Va26, Va35, Va59, Va85, Va99, VaW6, W117Ht, W153R, W182B, W22, W64A, WD, Wf9. These lines have been previously described <a href="#pone.0001367-Liu2">[20]</a>.</p>


<h4>SSRs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Of the 89 SSRs used in this study, 76 are dinucleotide repeats, 5 are trinucleotide repeats, 7 are tetranucleotide repeats, and one is a pentanucleotide repeat. Analysis of a largely overlapping data set has been published before <a href="#pone.0001367-Liu2">[20]</a>. All 89 of the markers and 210 of the 259 lines in our study are common to the two data sets. Markers scored by Liu <em>et al.</em> <a href="#pone.0001367-Liu2">[20]</a> not used in this study are: bnlg1014, bnlg1189, bnlg1288, bnlg1520, bnlg1839, bnlg1931, bnlg2238, phi031, phi096, and phi116; information on all SSRs can be found in the MaizeGDB database (<a href="http://www.maizegdb.org/">http://www.maizegdb.org/</a>).</p>


<h4>SNPs</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A collection of 913 SNPs that had been found to be polymorphic in a set of 14 maize inbred lines and 16 inbred teosintes <a href="#pone.0001367-Wright1">[32]</a> was scored in a set of 277 maize inbred lines <a href="#pone.0001367-Yu2">[33]</a>. They were designed from randomly selected genes out of the ~10,000 maize ESTs in the MMP-DuPont set <a href="#pone.0001367-Gardiner1">[34]</a>. SNP assay development and scoring was performed by Genaissance Pharmaceuticals using the Sequenom MassARRAY™ System <a href="#pone.0001367-Jurinke1">[35]</a>. Replicated assays indicate that the genotyping error rate is ~0.3%.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>After exclusion of lines not common to the two data sets and markers with &gt;20% missing data, the final data set consisted of 259 individuals scored for 847 SNPs and 89 SSRs. The SNPs were located in 563 PCR products throughout the genome, thus representing 563 loci. A list of the SNPs can be found in <a href="#pone.0001367.s001">Dataset S1</a>; information about each marker can be found at <a href="http://www.panzea.org/db/searches/webform/moldiversity_search">http://www.panzea.org/db/searches/webfor​m/moldiversity_search</a>.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Haplotypes were constructed when more than one SNP had been scored from a single amplicon. Because the individuals sampled came from inbred lines, the vast majority of SNP genotypes were homozygous, making haplotypes unambiguous. In cases where more than one SNP was heterozygous in an amplicon from one individual, haplotypes were inferred if only two haplotypes were observed in homozygous individuals (i.e., complete LD), otherwise they were treated as missing data.</p>


<h4>Data analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Summary statistics, genetic distances (allele sharing), core sets, and input files for STRUCTURE were obtained using PowerMarker <a href="#pone.0001367-Liu3">[36]</a>. To compare the assignment of individuals to NSS, SS, or TS populations using different marker sets, STRUCTURE <a href="#pone.0001367-Pritchard1">[16]</a> was run with k = 2, 3, 4, or 5 (i.e., assuming 2–5 populations) for each of four different data sets:</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>1) 89 SSRs.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>2) 847 SNPs.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>3) 554 SNPs or SNP haplotypes. SNPs from the same amplicon were grouped into haplotypes that were recoded as alleles. There were 251 2-SNP loci and 21 3-SNP loci, as well as 282 isolated SNPs. If the genotype of any SNP at a locus was missing in an individual, the entire locus was treated as missing data for that individual.</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>4) Data sets 1 and 2 combined.</p>
<a id="article1.body1.sec4.sec4.p6" name="article1.body1.sec4.sec4.p6"></a><p>For each value of k, STRUCTURE was run three times with burn-in and runs of 50,000 each. The admixture model was used, allele frequencies were assumed to be correlated, and alleles were assumed to be unlinked (an assumption that is violated by data sets 2 and 4). An individual was assigned to a population if it had ≥0.8 membership in that population.</p>
<a id="article1.body1.sec4.sec4.p7" name="article1.body1.sec4.sec4.p7"></a><p>For purposes of <em>F<sub>st</sub></em> calculation, individuals were designated as SS, NSS, TS, or mixed, according to the results of STRUCTURE using the SSR data set with k = 3. The three groups identified by STRUCTURE were classified as Stiff Stalk (SS), Non-Stiff Stalk (NSS), or Tropical-Semitropical (TS) based on the membership of individuals that typify those groups <a href="#pone.0001367-Liu2">[20]</a>.</p>


<h4>Tests for selection</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>We used the program FDIST2 <a href="#pone.0001367-Beaumont1">[21]</a> to test whether any of the values of <em>F<sub>st</sub></em> observed for individual SNPs were unusually large, given average <em>F<sub>st</sub></em> and locus-specific heterozygosity. Parameters for the simulations were: demes = 3, sampled populations = 3, sample size = 50 (following the recommendation of the FDIST2 documentation). Average <em>F<sub>st</sub></em> in the simulations = 0.1, the average observed for the SNP data set.</p>


<h4>Simulations</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Three data sets of 847 SNPs in 259 individuals were simulated using the program ms <a href="#pone.0001367-Hudson1">[37]</a>. One was generated under the standard neutral model, the others were generated under two different bottleneck models, both of which are modifications from Wright <em>et al.</em> <a href="#pone.0001367-Wright1">[32]</a>. Bottleneck 1 used “./ms 259 847 -s 1 -eN .0013 .0076 -eN .00208 1”, which produced average D <a href="#pone.0001367-Tajima1">[30]</a> of about 0.4, while Bottleneck 2 used “./ms 259 847 -s 1 -eN .0013 .05 -eN .00208 1”, which produced average D of about 0.2. In all cases, every SNP was sampled from a different locus, so that all SNPs were evolutionarily independent.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001367"></div><a name="pone.0001367.s001" id="pone.0001367.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001367.s001">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">SNP loci used in this study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001367.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Pat Brown, Mike Gore, Seth Murray, Zhiwu Zhang, Hans Ellegren, and two anonymous reviewers for discussion and/or comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EB MH MW. Analyzed the data: MH. Wrote the paper: MH. Other: Contributed the data: EB. Edited the paper: MW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001367-Smith1" id="pone.0001367-Smith1"></a>Smith JSC, Chin ECL, Shu H, Smith OS, Wall SJ, et al.  (1997) An evaluation of the utility of SSR loci as molecular markers in maize (<em>Zea mays</em> L.): comparisons with data from RFLPs and pedigree. Theor Appl Genet  95: 163–173.  <ul class="find" data-citedArticleID="1035336" data-doi="10.1007/s001220050544"><li><a href="http://dx.doi.org/10.1007/s001220050544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+evaluation+of+the+utility+of+SSR+loci+as+molecular+markers+in+maize+%28Zea+mays+L.%29%3A+comparisons+with+data+from+RFLPs+and+pedigree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+evaluation+of+the+utility+of+SSR+loci+as+molecular+markers+in+maize+%28Zea+mays+L.%29%3A+comparisons+with+data+from+RFLPs+and+pedigree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001367-Rafalski1" id="pone.0001367-Rafalski1"></a>Rafalski A (2002) Applications of single nucleotide polymorphisms in crop genetics. Curr Opin Plant Biol  5: 94–100.  <ul class="find" data-citedArticleID="1035328" data-doi="10.1016/s1369-5266(02)00240-6"><li><a href="http://dx.doi.org/10.1016/s1369-5266(02)00240-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applications+of+single+nucleotide+polymorphisms+in+crop+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applications+of+single+nucleotide+polymorphisms+in+crop+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001367-Schlotterer1" id="pone.0001367-Schlotterer1"></a>Schlotterer C (2004) The evolution of molecular markers–just a matter of fashion? Nature Rev Genet  5: 63–69.  <ul class="find" data-citedArticleID="1035332" data-doi="10.1038/nrg1249"><li><a href="http://dx.doi.org/10.1038/nrg1249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+molecular+markers%E2%80%93just+a+matter+of+fashion%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+molecular+markers%E2%80%93just+a+matter+of+fashion%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001367-Morin1" id="pone.0001367-Morin1"></a>Morin PA, Luikart G, Wayne RK, the SNP workshop group (2004) SNPs in ecology, evolution and conservation. Trends Ecol Evol  19: 208–216.  <ul class="find" data-citedArticleID="1035322" data-doi="10.1016/j.tree.2004.01.009"><li><a href="http://dx.doi.org/10.1016/j.tree.2004.01.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SNPs+in+ecology%2C+evolution+and+conservation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SNPs+in+ecology%2C+evolution+and+conservation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001367-Kennedy1" id="pone.0001367-Kennedy1"></a>Kennedy GC, Matsuzaki H, Dong S, Liu WM, Huang J, et al.  (2003) Large scale genotyping of complex DNA. Nature Biotechnol  21: 1233–1237.  <ul class="find" data-citedArticleID="1035302" data-doi="10.1038/nbt869"><li><a href="http://dx.doi.org/10.1038/nbt869" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large+scale+genotyping+of+complex+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large+scale+genotyping+of+complex+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001367-Vignal1" id="pone.0001367-Vignal1"></a>Vignal A, Milan D, San Cristobal M, Eggen A (2002) A review on SNP and other types of molecular markers and their use in animal genetics. Genet Sel Evol  34: 275–305.  <ul class="find" data-citedArticleID="1035342" data-doi="10.1186/1297-9686-34-3-275"><li><a href="http://dx.doi.org/10.1186/1297-9686-34-3-275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+review+on+SNP+and+other+types+of+molecular+markers+and+their+use+in+animal+genetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+review+on+SNP+and+other+types+of+molecular+markers+and+their+use+in+animal+genetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001367-Vigouroux1" id="pone.0001367-Vigouroux1"></a>Vigouroux Y, Jaqueth JS, Matsuoko Y, Smith OS, Beavis WD, et al.  (2002) Rate and pattern of mutation at microsatellite loci in maize. Mol Biol Evol  19: 1251–60.  <ul class="find" data-citedArticleID="1035344" data-doi="10.1093/oxfordjournals.molbev.a004186"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004186" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rate+and+pattern+of+mutation+at+microsatellite+loci+in+maize." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rate+and+pattern+of+mutation+at+microsatellite+loci+in+maize.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001367-Li1" id="pone.0001367-Li1"></a>Li YC, Korol AB, Fahima T, Beiles A, Nevo E (2002) Microsatellites: genomic distribution, putative functions, and mutational mechanisms: a review. Mol Ecol  11: 2453–2465.  <ul class="find" data-citedArticleID="1035308" data-doi="10.1046/j.1365-294x.2002.01643.x"><li><a href="http://dx.doi.org/10.1046/j.1365-294x.2002.01643.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellites%3A+genomic+distribution%2C+putative+functions%2C+and+mutational+mechanisms%3A+a+review." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellites%3A+genomic+distribution%2C+putative+functions%2C+and+mutational+mechanisms%3A+a+review.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001367-Ellegren1" id="pone.0001367-Ellegren1"></a>Ellegren H (2004) Microsatellites: simple sequences with complex evolution. Nature Rev Genet  5: 435–445.  <ul class="find" data-citedArticleID="1035288" data-doi="10.1038/nrg1348"><li><a href="http://dx.doi.org/10.1038/nrg1348" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microsatellites%3A+simple+sequences+with+complex+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microsatellites%3A+simple+sequences+with+complex+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001367-John1" id="pone.0001367-John1"></a>John S, Shephard N, Liu G, Zeggini E, Cao M, et al.  (2004) Whole-genome scan, in a complex disease, using 11,245 single-nucleotide polymorphisms: comparison with microsatellites. Am J Hum Genet  75: 54–64.  <ul class="find" data-citedArticleID="1035296" data-doi="10.1086/422195"><li><a href="http://dx.doi.org/10.1086/422195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+scan%2C+in+a+complex+disease%2C+using+11%2C245+single-nucleotide+polymorphisms%3A+comparison+with+microsatellites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+scan%2C+in+a+complex+disease%2C+using+11%2C245+single-nucleotide+polymorphisms%3A+comparison+with+microsatellites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001367-Skinner1" id="pone.0001367-Skinner1"></a>Skinner DZ, Bauchan GR, Auricht G, Hughes S (1999) A method for the efficient management and utilization of large germplasm collections. Crop Sci  39: 1237–1242.  <ul class="find" data-citedArticleID="1035334" data-doi="10.2135/cropsci1999.0011183x003900040046x"><li><a href="http://dx.doi.org/10.2135/cropsci1999.0011183x003900040046x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+method+for+the+efficient+management+and+utilization+of+large+germplasm+collections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+method+for+the+efficient+management+and+utilization+of+large+germplasm+collections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001367-Grenier1" id="pone.0001367-Grenier1"></a>Grenier C, Bramel-Cox PJ, Noirot M, Prasada-Rao KE, Hamon P (2000) Assessment of genetic diversity in three subsets constituted from the ICRISAT sorghum collection using random vs. non-random sampling procedures: A. Using morpho-agronomical and passport data. Theor Appl Genet  101: 190–196.  <ul class="find" data-citedArticleID="1035292" data-doi="10.1007/s001220051468"><li><a href="http://dx.doi.org/10.1007/s001220051468" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessment+of+genetic+diversity+in+three+subsets+constituted+from+the+ICRISAT+sorghum+collection+using+random+vs.+non-random+sampling+procedures%3A+A.+Using+morpho-agronomical+and+passport+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessment+of+genetic+diversity+in+three+subsets+constituted+from+the+ICRISAT+sorghum+collection+using+random+vs.+non-random+sampling+procedures%3A+A.+Using+morpho-agronomical+and+passport+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001367-Charcosset1" id="pone.0001367-Charcosset1"></a>Charcosset A, Lefort-Buson M, Gallais A (1991) Relationship between heterosis and heterozygosity at marker loci: a theoretical computation. Theor Appl Genet  81: 571–575.  <ul class="find" data-citedArticleID="1035286" data-doi="10.1007/bf00226720"><li><a href="http://dx.doi.org/10.1007/bf00226720" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+heterosis+and+heterozygosity+at+marker+loci%3A+a+theoretical+computation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+heterosis+and+heterozygosity+at+marker+loci%3A+a+theoretical+computation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001367-Melchinger1" id="pone.0001367-Melchinger1"></a>Melchinger AE, Gumber RK (1998)  Overview of heterosis and heterotic groups in agronomic crops. In: Lamkey KR, Staub JE, editors. Concepts and breeding of heterosis in crop plants. Madison, Wisconsin: CSSA. pp. 29–44.  <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0001367-Benchimol1" id="pone.0001367-Benchimol1"></a>Benchimol LL, de Souza CL Jr, Garcia AAF, Kono PMS, Mangolin CA, et al.  (2000) Genetic diversity in tropical maize inbred lines: heterotic group assignment and hybrid performance determined by RFLP markers. Plant Breeding  119: 491–496.  <ul class="find" data-citedArticleID="1035284" data-doi="10.1046/j.1439-0523.2000.00539.x"><li><a href="http://dx.doi.org/10.1046/j.1439-0523.2000.00539.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+in+tropical+maize+inbred+lines%3A+heterotic+group+assignment+and+hybrid+performance+determined+by+RFLP+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+in+tropical+maize+inbred+lines%3A+heterotic+group+assignment+and+hybrid+performance+determined+by+RFLP+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001367-Pritchard1" id="pone.0001367-Pritchard1"></a>Pritchard JK, Stephens M, Donnelly P (2000) Inference of population structure using multilocus genotype data. Genetics  155: 945–959.  <ul class="find" data-citedArticleID="1035326"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inference+of+population+structure+using+multilocus+genotype+data.&amp;auth=&amp;atitle=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+of+population+structure+using+multilocus+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+of+population+structure+using+multilocus+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001367-Rosenberg1" id="pone.0001367-Rosenberg1"></a>Rosenberg NA, Li LM, Ward R, Pritchard JK (2003) Informativeness of genetic markers for inference of ancestry. Am J Hum Genet  73: 1402–1422.  <ul class="find" data-citedArticleID="1035330" data-doi="10.1086/380416"><li><a href="http://dx.doi.org/10.1086/380416" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Informativeness+of+genetic+markers+for+inference+of+ancestry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Informativeness+of+genetic+markers+for+inference+of+ancestry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001367-Liu1" id="pone.0001367-Liu1"></a>Liu N, Chen L, Wang S, Oh C, Zhao H (2005) Comparison of Single Nucleotide Polymorphisms and microsatellites in inference of population structure. BMC Genetics  6: Suppl 1S26.  <ul class="find" data-citedArticleID="1035310" data-doi="10.1186/1471-2156-6-s1-s26"><li><a href="http://dx.doi.org/10.1186/1471-2156-6-s1-s26" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+Single+Nucleotide+Polymorphisms+and+microsatellites+in+inference+of+population+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+Single+Nucleotide+Polymorphisms+and+microsatellites+in+inference+of+population+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001367-Jones1" id="pone.0001367-Jones1"></a>Jones ES, Sullivan H, Bhattramakki D, Smith JSC (2007) A comparison of simple sequence repeat and single nucleotide polymorphism marker technologies for the genotypic analysis of maize (Zea mays L.) Theor Appl Genet  115: 361–371.  <ul class="find" data-citedArticleID="1035298" data-doi="10.1007/s00122-007-0570-9"><li><a href="http://dx.doi.org/10.1007/s00122-007-0570-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+simple+sequence+repeat+and+single+nucleotide+polymorphism+marker+technologies+for+the+genotypic+analysis+of+maize+%28Zea+mays+L.%29" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+simple+sequence+repeat+and+single+nucleotide+polymorphism+marker+technologies+for+the+genotypic+analysis+of+maize+%28Zea+mays+L.%29%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001367-Liu2" id="pone.0001367-Liu2"></a>Liu K, Goodman M, Muse S, Smith JS, Buckler E, et al.  (2003) Genetic structure and diversity among maize inbred lines as inferred from DNA microsatellites. Genetics  165: 2117–2128.  <ul class="find" data-citedArticleID="1035312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+structure+and+diversity+among+maize+inbred+lines+as+inferred+from+DNA+microsatellites.&amp;auth=&amp;atitle=Genetic+structure+and+diversity+among+maize+inbred+lines+as+inferred+from+DNA+microsatellites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+structure+and+diversity+among+maize+inbred+lines+as+inferred+from+DNA+microsatellites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+structure+and+diversity+among+maize+inbred+lines+as+inferred+from+DNA+microsatellites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001367-Beaumont1" id="pone.0001367-Beaumont1"></a>Beaumont MA, Balding DJ (2004) Identifying adaptive genetic divergence among populations from genome scans. Mol Ecol  13: 969–980.  <ul class="find" data-citedArticleID="1035282" data-doi="10.1111/j.1365-294x.2004.02125.x"><li><a href="http://dx.doi.org/10.1111/j.1365-294x.2004.02125.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+adaptive+genetic+divergence+among+populations+from+genome+scans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+adaptive+genetic+divergence+among+populations+from+genome+scans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001367-Laval1" id="pone.0001367-Laval1"></a>Laval G, San Cristobal M, Chevalet C (2002) Measuring genetic distances between breeds: use of some distances in various short term evolution models. Genet Sel Evol  34: 481–507.  <ul class="find" data-citedArticleID="1035306" data-doi="10.1186/1297-9686-34-4-481"><li><a href="http://dx.doi.org/10.1186/1297-9686-34-4-481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Measuring+genetic+distances+between+breeds%3A+use+of+some+distances+in+various+short+term+evolution+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Measuring+genetic+distances+between+breeds%3A+use+of+some+distances+in+various+short+term+evolution+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001367-Warburton1" id="pone.0001367-Warburton1"></a>Warburton ML, Xianchun X, Crossa J, Franco J, Melchinger AE, et al.  (2002) Genetic characterization of CIMMYT inbred maize lines and open pollinated populations using large scale fingerprinting methods. Crop Sci  42: 1832–1840.  <ul class="find" data-citedArticleID="1035346" data-doi="10.2135/cropsci2002.1832"><li><a href="http://dx.doi.org/10.2135/cropsci2002.1832" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+characterization+of+CIMMYT+inbred+maize+lines+and+open+pollinated+populations+using+large+scale+fingerprinting+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+characterization+of+CIMMYT+inbred+maize+lines+and+open+pollinated+populations+using+large+scale+fingerprinting+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001367-Warburton2" id="pone.0001367-Warburton2"></a>Warburton ML, Ribaut JM, Franco J, Crossa J, Dubreuil P, et al.  (2005) Genetic characterization of 218 elite CIMMYT inbred maize lines using RFLP markers. Euphytica  142: 97–106.  <ul class="find" data-citedArticleID="1035348" data-doi="10.1007/s10681-005-0817-y"><li><a href="http://dx.doi.org/10.1007/s10681-005-0817-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+characterization+of+218+elite+CIMMYT+inbred+maize+lines+using+RFLP+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+characterization+of+218+elite+CIMMYT+inbred+maize+lines+using+RFLP+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001367-Melchinger2" id="pone.0001367-Melchinger2"></a>Melchinger AE (1999)  Genetic diversity and heterosis. Chapter 10. In: Coors JG, Pandey S, editors. The genetics and exploitation of heterosis in crops. Madison, Wisconsin: CSSA.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001367-Yu1" id="pone.0001367-Yu1"></a>Yu J, Zhang Z, Tabanao DA, Pressoir G, Tuinstra MR, et al.  (2007) Relatedness estimation with different numbers of background markers and association mapping with different sample sizes. Theor Appl Genet. In press.   <ul class="find" data-citedArticleID="1035352"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Relatedness+estimation+with+different+numbers+of+background+markers+and+association+mapping+with+different+sample+sizes.&amp;auth=&amp;atitle=Relatedness+estimation+with+different+numbers+of+background+markers+and+association+mapping+with+different+sample+sizes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relatedness+estimation+with+different+numbers+of+background+markers+and+association+mapping+with+different+sample+sizes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relatedness+estimation+with+different+numbers+of+background+markers+and+association+mapping+with+different+sample+sizes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001367-Laborda1" id="pone.0001367-Laborda1"></a>Laborda PR, Oliveira KM, Garcia AAF, Paterniani MEAGZ, de Souza AP (2005) Tropical maize germplasm: what can we say about its genetic diversity in the light of molecular markers? Theor Appl Genet  111: 1288–1299.  <ul class="find" data-citedArticleID="1035304" data-doi="10.1007/s00122-005-0055-7"><li><a href="http://dx.doi.org/10.1007/s00122-005-0055-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tropical+maize+germplasm%3A+what+can+we+say+about+its+genetic+diversity+in+the+light+of+molecular+markers%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tropical+maize+germplasm%3A+what+can+we+say+about+its+genetic+diversity+in+the+light+of+molecular+markers%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001367-Lockwood1" id="pone.0001367-Lockwood1"></a>Lockwood DR, Richards CM, Volk GM (2007) Probabilistic models for collecting genetic diversity: comparisons, caveats, and limitations. Crop Sci  47: 861–868.  <ul class="find" data-citedArticleID="1035316" data-doi="10.2135/cropsci2006.04.0262"><li><a href="http://dx.doi.org/10.2135/cropsci2006.04.0262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probabilistic+models+for+collecting+genetic+diversity%3A+comparisons%2C+caveats%2C+and+limitations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probabilistic+models+for+collecting+genetic+diversity%3A+comparisons%2C+caveats%2C+and+limitations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001367-Tenaillon1" id="pone.0001367-Tenaillon1"></a>Tenaillon MI, Sawkins MC, Long AD, Gaut RL, Doebley JF, et al.  (2001) Patterns of DNA sequence polymorphism along chromosome 1 of maize (<em>Zea mays</em> ssp. <em>mays</em> L.). Proc Natl Acad Sci USA  98: 9161–9166.  <ul class="find" data-citedArticleID="1035340" data-doi="10.1073/pnas.151244298"><li><a href="http://dx.doi.org/10.1073/pnas.151244298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+DNA+sequence+polymorphism+along+chromosome+1+of+maize+%28Zea+mays+ssp.+mays+L.%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+DNA+sequence+polymorphism+along+chromosome+1+of+maize+%28Zea+mays+ssp.+mays+L.%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001367-Tajima1" id="pone.0001367-Tajima1"></a>Tajima F (1989) Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics  123: 585–595.  <ul class="find" data-citedArticleID="1035338"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism.&amp;auth=&amp;atitle=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001367-Payseur1" id="pone.0001367-Payseur1"></a>Payseur BA, Cutter AD (2006) Integrating patterns of polymorphism at SNPs and STRs. Trends Genet  22: 424–429.  <ul class="find" data-citedArticleID="1035324" data-doi="10.1016/j.tig.2006.06.009"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.06.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrating+patterns+of+polymorphism+at+SNPs+and+STRs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrating+patterns+of+polymorphism+at+SNPs+and+STRs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001367-Wright1" id="pone.0001367-Wright1"></a>Wright SI, Bi IV, Schroeder SG, Yamasaki M, Doebley JF, et al.  (2005) The effects of artificial selection on the maize genome. Science  308: 1310–1314.  <ul class="find" data-citedArticleID="1035350" data-doi="10.1126/science.1107891"><li><a href="http://dx.doi.org/10.1126/science.1107891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effects+of+artificial+selection+on+the+maize+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effects+of+artificial+selection+on+the+maize+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001367-Yu2" id="pone.0001367-Yu2"></a>Yu J, Pressoir G, Briggs WH, Bi IV, Yamasaki M, et al.  (2006) A unified mixed-model method for association mapping that accounts for multiple levels of relatedness. Nature Genet  38: 203–208.  <ul class="find" data-citedArticleID="1035354" data-doi="10.1038/ng1702"><li><a href="http://dx.doi.org/10.1038/ng1702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+unified+mixed-model+method+for+association+mapping+that+accounts+for+multiple+levels+of+relatedness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001367-Gardiner1" id="pone.0001367-Gardiner1"></a>Gardiner J, Schroeder S, Polacco ML, Sanchez-Villeda H, Fang Z, et al.  (2004) Anchoring 9,371 maize expressed sequence tagged unigenes to the bacterial artificial chromosome contig map by two-dimensional overgo hybridization. Plant Physiol  134: 1317–26.  <ul class="find" data-citedArticleID="1035290" data-doi="10.1104/pp.103.034538"><li><a href="http://dx.doi.org/10.1104/pp.103.034538" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anchoring+9%2C371+maize+expressed+sequence+tagged+unigenes+to+the+bacterial+artificial+chromosome+contig+map+by+two-dimensional+overgo+hybridization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anchoring+9%2C371+maize+expressed+sequence+tagged+unigenes+to+the+bacterial+artificial+chromosome+contig+map+by+two-dimensional+overgo+hybridization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001367-Jurinke1" id="pone.0001367-Jurinke1"></a>Jurinke C, van den Boom D, Cantor CR, Koster H (2002) The use of MassARRAY technology for high throughput genotyping. Adv Biochem Eng Biotechnol  77: 57–74.  <ul class="find" data-citedArticleID="1035300" data-doi="10.1007/3-540-45713-5_4"><li><a href="http://dx.doi.org/10.1007/3-540-45713-5_4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+MassARRAY+technology+for+high+throughput+genotyping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+MassARRAY+technology+for+high+throughput+genotyping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001367-Liu3" id="pone.0001367-Liu3"></a>Liu K, Muse SV (2005) PowerMarker: an integrated analysis environment for genetic marker analysis. Bioinformatics  21: 2128–2129.  <ul class="find" data-citedArticleID="1035314" data-doi="10.1093/bioinformatics/bti282"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti282" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PowerMarker%3A+an+integrated+analysis+environment+for+genetic+marker+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PowerMarker%3A+an+integrated+analysis+environment+for+genetic+marker+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001367-Hudson1" id="pone.0001367-Hudson1"></a>Hudson RR (2002) Generating samples under a Wright-Fisher neutral model of genetic variation. Bioinformatics  18: 337–338.  <ul class="find" data-citedArticleID="1035294" data-doi="10.1093/bioinformatics/18.2.337"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.2.337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+samples+under+a+Wright-Fisher+neutral+model+of+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+samples+under+a+Wright-Fisher+neutral+model+of+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.XML" value="95286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.PDF" value="363734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g001.PNG_L" value="94700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g001.PNG_M" value="54432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g001.PNG_S" value="12758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g001.TIF" value="1219910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g001.PNG_I" value="80959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t001.PNG_L" value="34212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t001.PNG_M" value="44087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t001.PNG_S" value="9311"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t001.TIF" value="149722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t001.PNG_I" value="14325"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t002.PNG_L" value="31779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t002.PNG_M" value="41354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t002.PNG_S" value="9452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t002.TIF" value="152988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t002.PNG_I" value="13480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g002.PNG_L" value="86880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g002.PNG_M" value="73402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g002.PNG_S" value="11231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g002.TIF" value="605636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g002.PNG_I" value="25341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t003.PNG_L" value="54435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t003.PNG_M" value="70971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t003.PNG_S" value="12095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t003.TIF" value="227450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t003.PNG_I" value="24142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g003.PNG_L" value="127715"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g003.PNG_M" value="96042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g003.PNG_S" value="12849"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g003.TIF" value="909740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g003.PNG_I" value="57769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g004.PNG_L" value="126022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g004.PNG_M" value="123735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g004.PNG_S" value="13176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g004.TIF" value="720056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.g004.PNG_I" value="70942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t004.PNG_L" value="113692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t004.PNG_M" value="65940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t004.PNG_S" value="9923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t004.TIF" value="537288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t004.PNG_I" value="20753"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t005.PNG_L" value="94712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t005.PNG_M" value="59174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t005.PNG_S" value="9588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t005.TIF" value="417616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t005.PNG_I" value="18228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t006.PNG_L" value="39895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t006.PNG_M" value="51931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t006.PNG_S" value="9874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t006.TIF" value="178552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.t006.PNG_I" value="16963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001367.s001.DOC" value="33280"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001367&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001367" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001367&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001367&volume=&issue=&title=Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness&author_name=Martha%20T.%20Hamblin%2C%20Marilyn%20L.%20Warburton%2C%20Edward%20S.%20Buckler&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001367" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367&amp;t=Empirical%20Comparison%20of%20Simple%20Sequence%20Repeats%20and%20Single%20Nucleotide%20Polymorphisms%20in%20Assessment%20of%20Maize%20Diversity%20and%20Relatedness" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367&title=Empirical%20Comparison%20of%20Simple%20Sequence%20Repeats%20and%20Single%20Nucleotide%20Polymorphisms%20in%20Assessment%20of%20Maize%20Diversity%20and%20Relatedness&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367&amp;title=Empirical%20Comparison%20of%20Simple%20Sequence%20Repeats%20and%20Single%20Nucleotide%20Polymorphisms%20in%20Assessment%20of%20Maize%20Diversity%20and%20Relatedness" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001367&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Empirical Comparison of Simple Sequence Repeats and Single Nucleotide Polymorphisms in Assessment of Maize Diversity and Relatedness';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001367';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Empirical%20Comparison%20of%20Simple%20Sequence%20Repeats%20and%20Single%20Nucleotide%20Polymorphisms%20in%20Assessment%20of%20Maize%20Diversity%20and%20Relatedness http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001367" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001367" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="25564"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Distance+measurement%22" title="Search for articles in the subject area:'Distance measurement'"><div class="flagText">Distance measurement</div></a>
              <div data-categoryid="21185" data-articleid="25564"
                   data-categoryname="Distance measurement"
                   class="flagImage" title="Flag 'Distance measurement' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25564"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="25564"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heterozygosity%22" title="Search for articles in the subject area:'Heterozygosity'"><div class="flagText">Heterozygosity</div></a>
              <div data-categoryid="40827" data-articleid="25564"
                   data-categoryname="Heterozygosity"
                   class="flagImage" title="Flag 'Heterozygosity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Maize%22" title="Search for articles in the subject area:'Maize'"><div class="flagText">Maize</div></a>
              <div data-categoryid="43001" data-articleid="25564"
                   data-categoryname="Maize"
                   class="flagImage" title="Flag 'Maize' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="25564"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25564"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3326'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6953'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8957&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
