

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000425"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000425" />

    <meta name="citation_title" content="Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis"/>
    <meta itemprop="name" content="Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis"/>

      <meta name="citation_author" content="Sol Efroni"/>
            <meta name="citation_author_institution" content="National Cancer Institute Center for Bioinformatics, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="Carl F. Schaefer"/>
            <meta name="citation_author_institution" content="National Cancer Institute Center for Bioinformatics, Rockville, Maryland, United States of America"/>
      <meta name="citation_author" content="Kenneth H. Buetow"/>
            <meta name="citation_author_institution" content="National Cancer Institute Center for Bioinformatics, Rockville, Maryland, United States of America"/>
            <meta name="citation_author_institution" content="Laboratory of Population Genetics, National Cancer Institute, Bethesda, Maryland, United States of America"/>

    <meta name="citation_date" content="2007/5/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000425.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e425"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Perturbations of the AKT signaling pathway in human cancer.; citation_author=DA Altomare; citation_author=JR Testa; citation_journal_title=Oncogene; citation_volume=24; citation_number=1; citation_pages=7455-7464; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The hallmarks of cancer.; citation_author=D Hanahan; citation_author=RA Weinberg; citation_journal_title=Cell; citation_volume=100; citation_number=2; citation_pages=57-70; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Colorectal cancer: mutations in a signalling pathway.; citation_author=DW Parsons; citation_author=TL Wang; citation_author=Y Samuels; citation_author=A Bardelli; citation_author=JM Cummins; citation_journal_title=Nature; citation_volume=436; citation_number=3; citation_pages=792; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=p53 regulates mitochondrial respiration.; citation_author=S Matoba; citation_author=JG Kang; citation_author=WD Patino; citation_author=A Wragg; citation_author=M Boehm; citation_journal_title=Science; citation_volume=312; citation_number=4; citation_pages=1650-1653; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Key cancer cell signal transduction pathways as therapeutic targets.; citation_author=R Bianco; citation_author=D Melisi; citation_author=F Ciardiello; citation_author=G Tortora; citation_journal_title=Eur J Cancer; citation_volume=42; citation_number=5; citation_pages=290-294; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A module map showing conditional activity of expression modules in cancer.; citation_author=E Segal; citation_author=N Friedman; citation_author=D Koller; citation_author=A Regev; citation_journal_title=Nat Genet; citation_volume=36; citation_number=6; citation_pages=1090-1098; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Integrative analysis of the cancer transcriptome.; citation_author=DR Rhodes; citation_author=AM Chinnaiyan; citation_journal_title=Nat Genet; citation_number=7; citation_pages=S31-37; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiclass cancer diagnosis using tumor gene expression signatures.; citation_author=S Ramaswamy; citation_author=P Tamayo; citation_author=R Rifkin; citation_author=S Mukherjee; citation_author=CH Yeang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=8; citation_pages=15149-15154; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer.; citation_author=Y Wang; citation_author=JG Klijn; citation_author=Y Zhang; citation_author=AM Sieuwerts; citation_author=MP Look; citation_journal_title=Lancet; citation_volume=365; citation_number=9; citation_pages=671-679; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Large-scale analysis of the human and mouse transcriptomes.; citation_author=AI Su; citation_author=MP Cooke; citation_author=KA Ching; citation_author=Y Hakak; citation_author=JR Walker; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=10; citation_pages=4465-4470; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gene expression analyses reveal molecular relationships among 20 regions of the human CNS.; citation_author=RB Roth; citation_author=P Hevezi; citation_author=J Lee; citation_author=D Willhite; citation_author=SM Lechner; citation_journal_title=Neurogenetics; citation_number=11; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.; citation_author=AI Su; citation_author=T Wiltshire; citation_author=S Batalov; citation_author=H Lapp; citation_author=KA Ching; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=12; citation_pages=6062-6067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=M Bittner; citation_number=13; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Oncogenic pathway signatures in human cancers as a guide to targeted therapies.; citation_author=AH Bild; citation_author=G Yao; citation_author=JT Chang; citation_author=Q Wang; citation_author=A Potti; citation_journal_title=Nature; citation_volume=439; citation_number=14; citation_pages=353-357; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An expression signature for p53 status in human breast cancer predicts mutation status, transcriptional effects, and patient survival.; citation_author=LD Miller; citation_author=J Smeds; citation_author=J George; citation_author=VB Vega; citation_author=L Vergara; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=15; citation_pages=13550-13555; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene expression alterations in prostate cancer predicting tumor aggression and preceding development of malignancy.; citation_author=YP Yu; citation_author=D Landsittel; citation_author=L Jing; citation_author=J Nelson; citation_author=B Ren; citation_journal_title=J Clin Oncol; citation_volume=22; citation_number=16; citation_pages=2790-2799; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Classification of human lung carcinomas by mRNA expression profiling reveals distinct adenocarcinoma subclasses.; citation_author=A Bhattacharjee; citation_author=WG Richards; citation_author=J Staunton; citation_author=C Li; citation_author=S Monti; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=17; citation_pages=13790-13795; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Summaries of Affymetrix GeneChip probe level data.; citation_author=RA Irizarry; citation_author=BM Bolstad; citation_author=F Collin; citation_author=LM Cope; citation_author=B Hobbs; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=18; citation_pages=e15; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cancer genes and the pathways they control.; citation_author=B Vogelstein; citation_author=KW Kinzler; citation_journal_title=Nat Med; citation_volume=10; citation_number=19; citation_pages=789-799; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Developing gene expression signatures of pathway deregulation in tumors.; citation_author=JW Watters; citation_author=CJ Roberts; citation_journal_title=Mol Cancer Ther; citation_volume=5; citation_number=20; citation_pages=2444-2449; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Probabilistic model of the human protein-protein interaction network.; citation_author=DR Rhodes; citation_author=SA Tomlins; citation_author=S Varambally; citation_author=V Mahavisno; citation_author=T Barrette; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=21; citation_pages=951-959; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis identifies a death-from-cancer signature predicting therapy failure in patients with multiple types of cancer.; citation_author=GV Glinsky; citation_author=O Berezovska; citation_author=AB Glinskii; citation_journal_title=J Clin Invest; citation_volume=115; citation_number=22; citation_pages=1503-1521; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.; citation_author=A Subramanian; citation_author=P Tamayo; citation_author=VK Mootha; citation_author=S Mukherjee; citation_author=BL Ebert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=23; citation_pages=15545-15550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pathway level analysis of gene expression using singular value decomposition.; citation_author=J Tomfohr; citation_author=J Lu; citation_author=TB Kepler; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=24; citation_pages=225; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=RO Duda; citation_author=PE Hart; citation_author=DG Stork; citation_volume=xx; citation_number=25; citation_pages=654 p; citation_date=2001; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_author=T Hastie; citation_author=R Tibshirani; citation_author=JH Friedman; citation_volume=xvi; citation_number=26; citation_pages=533 p; citation_date=2001; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_number=27; " />
      <meta name="citation_reference" content="citation_author=S Theodoridis; citation_author=K Koutroumbas; citation_volume=xiv; citation_number=28; citation_pages=689 p; citation_date=2003; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_author=AR Webb; citation_volume=xviii; citation_number=29; citation_pages=496 p; citation_date=2002; citation_publisher=Wiley; " />
      <meta name="citation_reference" content="citation_title=Cancer Molecular Analysis Project: weaving a rich cancer research tapestry.; citation_author=KH Buetow; citation_author=RD Klausner; citation_author=H Fine; citation_author=R Kaplan; citation_author=DS Singer; citation_journal_title=Cancer Cell; citation_volume=1; citation_number=30; citation_pages=315-318; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Pathway databases.; citation_author=CF Schaefer; citation_journal_title=Ann N Y Acad Sci; citation_volume=1020; citation_number=31; citation_pages=77-91; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nerve growth factor receptors and signaling in breast cancer.; citation_author=L Dolle; citation_author=E Adriaenssens; citation_author=I El Yazidi-Belkoura; citation_author=X Le Bourhis; citation_author=V Nurcombe; citation_journal_title=Curr Cancer Drug Targets; citation_volume=4; citation_number=32; citation_pages=463-470; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tamoxifen treatment for breast cancer enforces a distinct gene-expression profile on the human endometrium: an exploratory study.; citation_author=SC Gielen; citation_author=LC Kuhne; citation_author=PC Ewing; citation_author=LJ Blok; citation_author=CW Burger; citation_journal_title=Endocr Relat Cancer; citation_volume=12; citation_number=33; citation_pages=1037-1049; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Tamoxifen resistance in MCF7 cells promotes EMT-like behaviour and involves modulation of beta-catenin phosphorylation.; citation_author=S Hiscox; citation_author=WG Jiang; citation_author=K Obermeier; citation_author=K Taylor; citation_author=L Morgan; citation_journal_title=Int J Cancer; citation_volume=118; citation_number=34; citation_pages=290-301; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Interleukin-7 gene-modified dendritic cells reduce pulmonary tumor burden in spontaneous murine bronchoalveolar cell carcinoma.; citation_author=S Sharma; citation_author=RK Batra; citation_author=SC Yang; citation_author=S Hillinger; citation_author=L Zhu; citation_journal_title=Hum Gene Ther; citation_volume=14; citation_number=35; citation_pages=1511-1524; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=NMDA antagonist inhibits the extracellular signal-regulated kinase pathway and suppresses cancer growth.; citation_author=A Stepulak; citation_author=M Sifringer; citation_author=W Rzeski; citation_author=S Endesfelder; citation_author=A Gratopp; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=36; citation_pages=15605-15610; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Acid sphingomyelinase-deficient human lymphoblasts and mice are defective in radiation-induced apoptosis.; citation_author=P Santana; citation_author=LA Pena; citation_author=A Haimovitz-Friedman; citation_author=S Martin; citation_author=D Green; citation_journal_title=Cell; citation_volume=86; citation_number=37; citation_pages=189-199; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Endothelial apoptosis as the primary lesion initiating intestinal radiation damage in mice.; citation_author=F Paris; citation_author=Z Fuks; citation_author=A Kang; citation_author=P Capodieci; citation_author=G Juan; citation_journal_title=Science; citation_volume=293; citation_number=38; citation_pages=293-297; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Natural ceramide reverses Fas resistance of acid sphingomyelinase(−/−) hepatocytes.; citation_author=F Paris; citation_author=H Grassme; citation_author=A Cremesti; citation_author=J Zager; citation_author=Y Fong; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=39; citation_pages=8297-8305; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sphingosine-1-phosphate protects proliferating endothelial cells from ceramide-induced apoptosis but not from DNA damage-induced mitotic death.; citation_author=S Bonnaud; citation_author=C Niaudet; citation_author=G Pottier; citation_author=MH Gaugler; citation_author=J Millour; citation_journal_title=Cancer Res; citation_volume=67; citation_number=40; citation_pages=1803-1811; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Ceramide involvement in apoptosis and apoptotic diseases.; citation_author=K Thevissen; citation_author=IE Francois; citation_author=J Winderickx; citation_author=C Pannecouque; citation_author=BP Cammue; citation_journal_title=Mini Rev Med Chem; citation_volume=6; citation_number=41; citation_pages=699-709; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ceramide in apoptotic signaling and anticancer therapy.; citation_author=CF Lin; citation_author=CL Chen; citation_author=YS Lin; citation_journal_title=Curr Med Chem; citation_volume=13; citation_number=42; citation_pages=1609-1616; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Modulation of the ceramide level, a novel therapeutic concept?; citation_author=R Claus; citation_author=S Russwurm; citation_author=M Meisner; citation_author=R Kinscherf; citation_author=HP Deigner; citation_journal_title=Curr Drug Targets; citation_volume=1; citation_number=43; citation_pages=185-205; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Sphingolipids in anticancer therapy.; citation_author=JM Padron; citation_journal_title=Curr Med Chem; citation_volume=13; citation_number=44; citation_pages=755-770; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An emerging role for Ca2+/calcineurin/NFAT signaling in cancerogenesis.; citation_author=M Buchholz; citation_author=V Ellenrieder; citation_journal_title=Cell Cycle; citation_volume=6; citation_number=45; citation_pages=16-19; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Overexpression of c-myc in pancreatic cancer caused by ectopic activation of NFATc1 and the Ca2+/calcineurin signaling pathway.; citation_author=M Buchholz; citation_author=A Schatz; citation_author=M Wagner; citation_author=P Michl; citation_author=T Linhart; citation_journal_title=Embo J; citation_volume=25; citation_number=46; citation_pages=3714-3724; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cells as computation.; citation_author=A Regev; citation_author=E Shapiro; citation_journal_title=Nature; citation_volume=419; citation_number=47; citation_pages=343; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis"/>
    <meta name="twitter:description" content="Cancer is recognized to be a family of gene-based diseases whose causes are to be found in disruptions of basic biologic processes. An increasingly deep catalogue of canonical networks details the specific molecular interaction of genes and their products. However, mapping of disease phenotypes to alterations of these networks of interactions is accomplished indirectly and non-systematically. Here we objectively identify pathways associated with malignancy, staging, and outcome in cancer through application of an analytic approach that systematically evaluates differences in the activity and consistency of interactions within canonical biologic processes. Using large collections of publicly accessible genome-wide gene expression, we identify small, common sets of pathways &ndash; Trka Receptor, Apoptosis response to DNA Damage, Ceramide, Telomerase, CD40L and Calcineurin &ndash; whose differences robustly distinguish diverse tumor types from corresponding normal samples, predict tumor grade, and distinguish phenotypes such as estrogen receptor status and p53 mutation state. Pathways identified through this analysis perform as well or better than phenotypes used in the original studies in predicting cancer outcome. This approach provides a means to use genome-wide characterizations to map key biological processes to important clinical features in disease."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000425.g005"/>

  <meta property="og:title" content="Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4010'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=2261'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=252&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000425">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000425" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000425&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sol Efroni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Cancer Institute Center for Bioinformatics, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carl F. Schaefer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  National Cancer Institute Center for Bioinformatics, Rockville, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth H. Buetow
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:buetowk@mail.nih.gov">buetowk@mail.nih.gov</a></p>

                <p>Affiliations:
                  National Cancer Institute Center for Bioinformatics, Rockville, Maryland, United States of America, 
                  Laboratory of Population Genetics, National Cancer Institute, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 09, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000425</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000425-g001" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000425-g002" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000425-g003" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000425-t001" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000425-g004" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000425-g005" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000425">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000425" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Cancer is recognized to be a family of gene-based diseases whose causes are to be found in disruptions of basic biologic processes. An increasingly deep catalogue of canonical networks details the specific molecular interaction of genes and their products. However, mapping of disease phenotypes to alterations of these networks of interactions is accomplished indirectly and non-systematically. Here we objectively identify pathways associated with malignancy, staging, and outcome in cancer through application of an analytic approach that systematically evaluates differences in the activity and consistency of interactions within canonical biologic processes. Using large collections of publicly accessible genome-wide gene expression, we identify small, common sets of pathways – Trka Receptor, Apoptosis response to DNA Damage, Ceramide, Telomerase, CD40L and Calcineurin – whose differences robustly distinguish diverse tumor types from corresponding normal samples, predict tumor grade, and distinguish phenotypes such as estrogen receptor status and p53 mutation state. Pathways identified through this analysis perform as well or better than phenotypes used in the original studies in predicting cancer outcome. This approach provides a means to use genome-wide characterizations to map key biological processes to important clinical features in disease.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Efroni S, Schaefer CF, Buetow KH (2007) Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis. PLoS ONE 2(5):
          e425.
            doi:10.1371/journal.pone.0000425</p><p><strong>Academic Editor: </strong>Nick Monk, University of Sheffield, United Kingdom</p><p><strong>Received:</strong> January 5, 2007; <strong>Accepted:</strong> March 29, 2007; <strong>Published:</strong> May 9, 2007</p><p>This is an open-access article distributed under the terms of the Creative Commons Public Domain declaration which stipulates that, once placed in the public domain, this work may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose.</p><p><strong>Funding: </strong>This research was supported by the intramural research program of the NIH, National Cancer Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Biologic phenomena emerge as consequence of the action of genes and their products in pathways. Diseases arise through alteration of these complex networks <a href="#pone.0000425-Altomare1">[1]</a>–<a href="#pone.0000425-Bianco1">[5]</a>. In order to make mechanistic assertions that supplement current approaches to genome-wide analysis <a href="#pone.0000425-Segal1">[6]</a>–<a href="#pone.0000425-Wang1">[9]</a>, we map canonical biologic pathways to cancer phenotypes. A total of 2011 Affymetrix GeneChip array hybridizations obtained from 9 different publicly accessible data sources <a href="#pone.0000425-Su1">[10]</a>–<a href="#pone.0000425-Bhattacharjee1">[17]</a> were analyzed. The hybridizations represented 70 different tumor types (1348 samples). Additionally 83 different types of samples of normal histology were included (663 samples). Expression levels were adjusted using RMA<a href="#pone.0000425-Irizarry1">[18]</a>. The definition of normal used here excludes uninvolved and/or tumor adjacent samples obtained from individuals with cancer.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The use of pathways as a framework for analysis is not in itself novel. These include the projection of known cancer genes and gene expression data onto pathways <a href="#pone.0000425-Vogelstein1">[19]</a>, <a href="#pone.0000425-Watters1">[20]</a>. What distinguishes the work presented here is the systematic evaluation of the interaction structure across predefined canonical networks. In measuring the state of the interaction it combines information from gene state and network structure. Multiple gene states may result in a common pathway score. Conversely, pathway scores may show greater differences than gene signatures.</p>

<h4>Approaches to Pathway Analysis</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>This investigation complements other work utilizing pathway information.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>More specifically, Segal et. al. <a href="#pone.0000425-Segal1">[6]</a> defined biological modules and refined them to a set of statistically significant modules. They were able to use these modules to gain a better perspective on the different biological processes that are activated and de-activated in various clinical conditions. We note two main differences between what we present here and the work in Segal et. al. <a href="#pone.0000425-Segal1">[6]</a>: first, the biological modules used in the paper, although highly informative and useful, are internally defined within the paper. The determination of genes in these modules was derived from the same data to which they are later applied. The canonical pathways we use are externally defined independent from the data we analyze, represent current understanding in the field, and were not derived ad-hoc. Second, Segal et. al. do not make explicit use of the interconnections, or the network structure, that exists between genes that comprise biological modules. The scores for activity and consistency we present here depend on network structure and specific relations (such as inhibition and promotion) that are features of the network information.</p>
<a id="article1.body1.sec1.sec1.p3" name="article1.body1.sec1.sec1.p3"></a><p>Another important approach is that of Rhodes et. al. <a href="#pone.0000425-Rhodes2">[21]</a>, in which the human interactome network is used to identify subnetworks activated in cancer. The approach Rhodes el. al. use, in contrast to the one presented here, does not attempt to computationally and algorithmically highlight differences in phenotypes by building a classifier around measurable network features. Instead, it generates subnetworks by their association with sets of genes identified through the over (or under) expression in each biological phenotype. Rhodes et. al. approach does make use of the network structure to build the subnetwork, but does not make further use in observing the co-expression or co-silencing of sets of genes, as is the case in the work presented here.</p>
<a id="article1.body1.sec1.sec1.p4" name="article1.body1.sec1.sec1.p4"></a><p>Bild et. al. <a href="#pone.0000425-Bild1">[14]</a> and Glinski et. al. <a href="#pone.0000425-Glinsky1">[22]</a> demonstrate that gene signatures determined by small set of pre-selected canonical pathways can distinguish tumor characteristics. In their work, they start with a limited set of pathways, (e.g. Bild et. al. use 5 pathways) and show that they differ in different phenotypes. As this approach starts with a small set of pathways the authors chose to examine, it does not have the capacity to discover new pathway associations with phenotypes. Unlike the current work, it does not employ an objective method to identify set of pathways that can discriminate phenotypes.</p>
<a id="article1.body1.sec1.sec1.p5" name="article1.body1.sec1.sec1.p5"></a><p>Gene set Enrichment Analyses <a href="#pone.0000425-Subramanian1">[23]</a> allows the authors to choose a set of genes and to determine their relative statistical significance in a list of genes that separate phenotypes. Gene set enrichment starts with the premise of individual genes as classifiers. Pathway membership is measured to assess combined contributions. Again, the method does not make use of the structure of the network, nor does it provide a systemic account for the combined knowledge of pathways to reduce to an optimal set of classifying processes. Since the method starts with the discrimination of single genes, it can only build on this statistical inference, and does not account for any differences that come from the inter-dependency of multiple gene interactions. For example, if gene A seems to permutate randomly in the two phenotypes and gene B seems to permutate randomly in the two phenotypes then each of the genes will score poorly in a statistical significance test. However, the score defined by their combined dependence (e.g. (if A then B)) might provide much greater discrimination.</p>
<a id="article1.body1.sec1.sec1.p6" name="article1.body1.sec1.sec1.p6"></a><p>The method by Tomfohr, et. al. <a href="#pone.0000425-Tomfohr1">[24]</a> is perhaps the closest to the one presented here in that it looks at combined groups of genes and ranks them accordingly. However, Tomfohr, et. al. do not use the network structure knowledge to obtain scores, but instead perform Singular Value Decomposition (SVD) to choose a specific metagene, and define a pathway activity as the expression of that gene. As such, the result does not utilize the interdependence of the network as does the work presented above.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Evaluating a gene status:</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Gene status in evaluating the network interaction is calculated from the observed data as one of two alternative states: down and up. To be able to identify whether a gene is in a “down” state or an “up” state, we look at its (RMA adjusted <a href="#pone.0000425-Irizarry1">[18]</a>) expression value in a sample, compared to the expression values of the same gene in all other samples. To be able to accommodate a multitude of probability distributions, we use a gamma distribution as the template to both the “down” distribution form as well as the “up” distribution, and redefine the problem as a mixture of two gamma distribution. The suppressed form often follows an exponential distribution, which is one particular case of a gamma distribution. The promoted state often follows a form similar to a normal distribution, which may be approximated by a gamma distribution of a large mean. Per every probe set measured by the microarray, we look at the expression distribution and try to fit this distribution into a mixture of two gamma distributions. We do this by using an Expectation-Maximization (EM) algorithm, iterating over the data in a manner that guarantees the increase of likelihood of fitting the data by the modeled distributions. In the case of two gamma distributions, we first divide the data into two groups: “down” values and “up” values. The number of genes in the “up” group is <em>N<sub>U</sub></em> and the number of genes in the suppressed group is <em>N<sub>D</sub></em>. The prior probabilities are therefore:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e001&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We assume each group distributes according to a gamma distribution:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e002&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The objective of the EM algorithm is to provide us with maximum-likelihood estimates to the <em>a<sub>U</sub>, b<sub>U</sub></em> values for the promoted group and to the <em>a<sub>D</sub>, b<sub>D</sub></em> values for the suppressed group. Additionally, it computes the maximum-likelihood estimates of the mixture coefficients, η<sub>1</sub>, η<sub>2</sub>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>We assume that the expression distribution of each gene is either coming from a mixture of two distributions (one for the “up” case and one for the “down” case) or from a single distribution (for example, when the gene is “up” in all the samples we have). We determine the number of underlying distributions (one or two) using the EM algorithm in combination with a model selection method, see below.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>To find the maximum of the log likelihood, we need to find the maximum of the auxiliary function <em>Q</em> <a href="#pone.0000425-Duda1">[25]</a>:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e003&amp;representation=PNG" class="inline-graphic"></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e004&amp;representation=PNG" class="inline-graphic"></span><br>Here, <em>θ</em> is the collection of parameters that define the distribution, and the superscript 0 designates magnitudes that had been determined in the previous iteration.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>To find maxima, we differentiate <em>Q</em> with respect to the model parameters, and compare to zero.<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e005&amp;representation=PNG" class="inline-graphic"></span><br>And the coefficients<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e006&amp;representation=PNG" class="inline-graphic"></span><br>where Ψ(<em>x</em>) is the psi function <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e007&amp;representation=PNG" class="inline-graphic"></span>.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>Using a Lagrange multiplier to incorporate the constrain <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e008&amp;representation=PNG" class="inline-graphic"></span><br>we have to maximize the target function<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e009&amp;representation=PNG" class="inline-graphic"></span><br>with respect to the <em>η<sub>i</sub></em>, we derive<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e010&amp;representation=PNG" class="inline-graphic"></span><br>and obtain<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e011&amp;representation=PNG" class="inline-graphic"></span><br>We solve this numerically (using Matlab®) in every iterative step, until we reach some predefined convergence criterion.</p>


<h4>Choosing an optimal number of distributions:</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Obviously, the more distributions we take as our basis for the overall distributions, the better fit we have for the data and the better the likelihood will be. Consider, for example, as many distributions as there are data points. That would fit the data exactly and produce maximal likelihood. To overcome this, and to be able to choose an optimal number, we compare models with different number of distributions using the Bayesian Information Criterion (BIC)<a href="#pone.0000425-Hastie1">[26]</a>, computed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e012&amp;representation=PNG" class="inline-graphic"></span><br>This cost function compensates for the additional increase in complexity. The statistical model chosen is the one with the largest BIC.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p><strong>Calculate</strong> <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e013&amp;representation=PNG" class="inline-graphic"></span><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e014&amp;representation=PNG" class="inline-graphic"></span><br>And similarly:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e015&amp;representation=PNG" class="inline-graphic"></span><br>But we need the probability of being in the “promoted” state for a specific expression value:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e016&amp;representation=PNG" class="inline-graphic"></span><br>And since<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e017&amp;representation=PNG" class="inline-graphic"></span><br>we can obtain the needed values by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e018&amp;representation=PNG" class="inline-graphic"></span><br>For example, the expression of the gene CDKN1A in the dataset <a href="#pone.0000425-Bittner1">[13]</a> (a collection of 698 tumor samples) follows this distribution (see <a href="#pone-0000425-g001">Figure 1</a>):</p>
<div class="figure" id="pone-0000425-g001"><div class="img"><a name="pone-0000425-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>An example to the distribution of gene expression and its resemblance to a mixture of two Gamma distributions.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The Up/Down calls for gene states are based on an expression value classified as residing in one of the two distinct distributions.</p>
<span>doi:10.1371/journal.pone.0000425.g001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The two distinct distributions (Down and Up) are evident and the algorithm gives the parameters for the two gamma distributions.</p>


<h4>Pathway activity and pathway consistency</h4>


<ol class="order">

<li>Pathway consistency score: To determine the pathway consistency score of a given signaling pathway in a sample, we follow these steps:

<ol class="">

<li>

<ol class="alpha-lower">

<li>Every pathway is a collection of interactions. Input genes and output genes define each interaction. For each interaction in the pathway, we first look at the input genes and determine, for each such gene, the probability of being in a “down” or “up” state (see “gene state” above)</li>

<li>We then determine the probability of the materialization of the specific interaction as the joint probability of all needed components (genes)</li>

<li>Next, we look at the molecular output of the interaction. Usually, this output is a list of genes, for which we establish the probability of being in a “down” or “up” state (see “gene state” above)</li>

<li>Next, we calculate the likelihood of the output gene(s) being in one of the two states, under the given probability of the interaction (calculated in (b))</li>

<li>Lastly, to obtain the pathway consistency score, we calculate the consistency score for every interaction in the pathway and average the scores over all the interactions for which we were able to obtain a score. In <a href="#pone-0000425-g002">Figure 2</a> we show an example to calculating the consistency value of an interaction taken from the pathway “Signaling events mediated by Stem cell factor receptor (c-Kit)”, one of the NCI-Nature Curated pathways from the Pathway Interaction Database (PID)<a href="#pone.0000425-National1">[27]</a>. The specific steps to calculate consistency in this example are:

<ol class="">

<li>

<ol class="order">

<li>Establish probabilities to all genes involved in the interaction. This is done according to the steps described below (see “gene state” section). The values we obtain are: P(CREBBP) = 0.95; P(STAT5A) = 0.8; P(KIT) = 0.7</li>

<li>Calculate the joint probability of an active interaction. Since the input molecules to the interaction are not co-dependent, the joint probability of the interaction is P(CREBBP)×P(STAT5A) = 0.95×0.8 = 0.76</li>

<li>Calculate the likelihood that the output molecule is the result of the interaction. Since the molecule is solely dependent on the interaction the likelihood is straightforward:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.e019&amp;representation=PNG" class="inline-graphic"></span><br></li>

<li>Iterate this computation throughout all interactions in the pathway. The final score of a pathway is an average over all interactions.</li>

</ol></li>

</ol></li>

</ol></li>

</ol></li>

<li>A pathway activity score is the average over activity of interactions in a pathway. For example, in the previous example, the interaction activity is 0.76. The main advantage to calculating pathway activities on top of pathway consistencies is that activities can be calculated even when there is not enough data to work with the output, as is the case, for example, when the interaction is based on activating or modifying molecules without the generation of a novel molecule as output. In such cases, we can still calculate the activity, although consistency loses its meaning.</li>

</ol><div class="figure" id="pone-0000425-g002"><div class="img"><a name="pone-0000425-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>An example to calculating the consistency and activity of a single interaction.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>See <a href="#s2">Methods</a> for details.</p>
<span>doi:10.1371/journal.pone.0000425.g002</span></div>

<h4>Choosing a minimal set of pathways to classify phenotype</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>As we obtain pathway activity and consistency scores for each pathway, we are able to transform the representation of each bio-sample from a list of gene expression measurements into a novel representation, displaying each sample with the collection of pathway activity and consistency scores. As we use this representation to distinguish between phenotypes, we wish to find the minimal set of pathways scores that is able to make the distinction between phenotypic classes. We use feature selection to choose an optimal minimal set (see <a href="#s3">Results</a>). We used different methods of feature extraction and feature classification <a href="#pone.0000425-Theodoridis1">[28]</a>, <a href="#pone.0000425-Webb1">[29]</a>, including forward selection, backward selection, and floating search <a href="#pone.0000425-Webb1">[29]</a>. These methods help in eliminating pathway scores that do not contribute to making the distinction and highlighting specific pathways that together achieve an optimal classification rate.</p>


<h4>Pathway metric to predict outcome</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Representing each bio-sample using its pathway metrics allows us to look for patterns in the collection of pathways. By using clustering algorithms, we see that pathway metric values segregate samples into groups. If we look at the survival patterns of these groups, we see that in some cases and for some pathways, the groups correlate with distinct patterns of survival.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The analysis applied here treats a pathway as a network of genes whose interactions are logically evaluated in the pathway context to generate sets of scores. Biologic pathway structure information was obtained from public sources <a href="#pone.0000425-National1">[27]</a>, <a href="#pone.0000425-Buetow1">[30]</a>, <a href="#pone.0000425-Schaefer1">[31]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Each pathway is assessed for consistency and activity. A pathway consistency score is calculated as the average likelihood of the logical consistency of the collection of interactions given the calculated states of the genes (see <a href="#s2">Methods</a>). A pathway activity score is calculated as the average likelihood of the pathway's individual interactions being active given the calculated gene states. Using basic principles of machine supervised learning <a href="#pone.0000425-Theodoridis1">[28]</a>, <a href="#pone.0000425-Webb1">[29]</a> a classification algorithm that distinguished each oncogenic phenotype (e.g. cancer sample verse normal) was generated and validated. Based on simplicity and comparability of alternative approaches tested, a Bayesian linear discriminant classifier was used.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>First, a classification algorithm was derived to distinguish diverse cancer phenotypes from normal phenotype tissues. A classifier derived from an 1800 sample training set (10-fold validation) demonstrated 98% success in an independent validation test set of 211 samples (see <a href="#pone-0000425-g003">Figure 3</a> and <a href="#pone-0000425-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000425-g003"><div class="img"><a name="pone-0000425-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Classification results of the different classifiers tried.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>Each panel in the figure corresponds to a different phenotypic difference, according to panel captions. The horizontal axis in each panel corresponds to the one-dimensional projection calculated by the classification algorithm, that signifies distance between biological samples, according to the multi dimensional pathway metrics. The vertical axis is a jitter scatter of the samples to enable a clear view of the separation.</p>
<span>doi:10.1371/journal.pone.0000425.g003</span></div><div class="figure" id="pone-0000425-t001"><div class="img"><a name="pone-0000425-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000425.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000425.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Pathway names and classes.</span></strong></p><span>doi:10.1371/journal.pone.0000425.t001</span></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Since linear classifiers turn each of the pathways in the problem into a variable in the classifier, it is possible through feature analysis to identify subsets of classifier variables (pathways) that, as a group, distinguish the phenotypes with high accuracy. Feature selection was used to identify a set demonstrating the optimal 98% accuracy of the original classification in the validation sample analysis. It is composed of the activity scores of six pathways: Trka Pathway, DNA Damage pathway, Ceramide Pathway, Telomerase Pathway, CD40L Pathway and Calcineurin Pathway.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Cancer is a disease of great phenotypic and molecular heterogeneity. Even within a given organ site, phenotypic heterogeneity is associated with significant differences in cancer outcome. It is therefore of additional interest to identify molecular processes that underlie the phenotypic differences and that predict outcome. We therefore derived signatures for a variety of subtypes of breast cancer. These subtypes include: histologic grade (Elston grades 1 vs. 3, or grades 2 vs. 3); P53 status (mutated/wild type); estrogen receptor positive/negative status (ER+/−); and progesterone receptor positive/negative status (PgR+/−). The performance of the classifiers is displayed in <a href="#pone-0000425-g003">Figure 3</a>. In all cases, classifiers with a small number of pathways (three to six) achieved a high level of accuracy (83% to 95%). <a href="#pone-0000425-t001">Table 1</a> shows the different pathway groups that classify different phenotypes.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>We next evaluated the ability of the cancer subtype-specific signatures to stratify the 236 breast cancer samples by outcome. Following unsupervised clustering of the cancer samples using the pathways identified above, Kaplan Meier analyses was performed (<a href="#pone-0000425-g004">Figure 4</a>). In three cases, a single pathway from the sub-type signature significantly predicted outcome: the Circadian Rhythms pathway, from the grade 1/3 signature (P = 2.9E-11); the Sonic Hedgehog pathway, from the grade 2/3 signature (P = 4E-8); and Agrin in Postsynaptic Differentiation, from the P53 signature (P = 4.6E-7). The three pathways in the PgR+/− signature separated the samples into two groups with a P value .0001, with the Bone Remodelling pathway accounting for most of the effect. In addition, the five pathways in the ER+/− signature separated the samples into two groups with a P value of .004, with the SREBP pathway accounting for most of the effect.</p>
<div class="figure" id="pone-0000425-g004"><div class="img"><a name="pone-0000425-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Examples of the stratification of survival plots and their immediate connections to pathway activity/consistency.</span></strong></p><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1"></a><p>(A) (1) Kaplan-Meier survival plot of breast cancer patients from <a href="#pone.0000425-Miller1">[15]</a>, stratified according to clustering based on pathway activity. Panel (2) in (A) shows the activity score of the Sonic hedgehog pathway colored according to affiliation with either of the accordingly colored survival curves in (1); (B) The same analyses done with breast cancer patients from <a href="#pone.0000425-Miller1">[15]</a>, based on the pathway Bone Remodeling (see text for pathway choice). (C) Kaplan Meier survival plots of lung cancer patient data from <a href="#pone.0000425-Bhattacharjee1">[17]</a>, stratified according to activity of the Csk pathway and the (D) NFKβ pathway. In every panel, the (2) sub-panel shows the most influential pathway metric out of the group of stratifying pathways. This does not mean that the pathway represented is responsible for the entire separation into two groups.</p>
<span>doi:10.1371/journal.pone.0000425.g004</span></div><a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>It is important to note that a number of findings in the literature emerge independently from our pathway analysis of the breast cancer samples. As the importance of the ER+/− distinction in management of breast cancer is well established, we looked at each of these subgroups separately. It has been observed <a href="#pone.0000425-Dolle1">[32]</a> that the Trka Pathway (identified in both the generic oncogenic signature and the grade 2/3 signature) plays a significant role in ER- cases. Our analysis shows that the generic oncogenic signature separates the ER- samples into two groups (P = 4.6E-9) with the Trka pathway accounting for most of the effect, high activity of this pathway correlating with poor prognosis. Likewise, it has been observed <a href="#pone.0000425-Gielen1">[33]</a>, <a href="#pone.0000425-Hiscox1">[34]</a> that beta-catenin plays a significant role in the response to tamoxifen, a standard treatment for ER+ disease. To analyze the nature of the tamoxifen-induced response, we derived a classifier to distinguish the ER+ cases that had been treated with tamoxifen from those cases that had not been so treated and then used the pathways in the resulting signature to cluster the cases by outcome. The Beta-catenin pathway emerged as the most significant (P = 1E-13) pathway in predicting outcome.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>It has long been suggested that molecular classifications of cancer may have the capacity to transcend organ or tissue-specific definitions. More specifically, it has been suggested that molecular definitions that reflect the universal properties of cell type or ontology and that underpin a common molecular etiology may emerge across organ site definitions. To assess whether the signatures observed above in cancer of breast epithelium may generalize to other cancers, we examined their capacity to predict phenotypes in lung and colon cancer. We applied signatures derived from the breast cancer subtypes to cluster the lung cancer outcomes (<a href="#pone-0000425-g004">Figure 4</a>). Pathways predicting outcome included the IL-7 Pathway (P = .002) and the Csk Pathway (P = 3E-11). It has been previously noted that these pathways have been linked with outcome in lung cancer <a href="#pone.0000425-Sharma1">[35]</a>, <a href="#pone.0000425-Stepulak1">[36]</a></p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Lastly, we examined the general oncogenic signature's capacity to predict organ-site specific outcome. Interestingly, the signature pathways separated the 236 breast cancer samples into five different survival subgroups (P = 2E-8) and the 90 lung cancer samples into two different subgroups (P = 5E-17).</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The above results suggest that using the pathway as the unit of analysis can augment current individual gene based approaches to mapping phenotype to underlying molecular process. Objective identification of processes previously associated with phenotypes utilizing genome-wide datasets provides partial validation of the observed results. Newly observed process mappings to phenotypes, however, clearly require either verification from independent data sets or experimental confirmation.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The observations made through this analysis are provocative. Many of these pathways (e.g. apoptosis, telomere maintenance) have been previously described as universal components of oncogenesis<a href="#pone.0000425-Hanahan1">[2]</a>. Additionally, processes are identified that may underlie common cancer related phenotypes, such as inflammation. Interestingly, novel pathways are also identified as part of the general oncogenic signature as depicted in the six pathways collective (e.g. Ceramide and Calcineurin pathways). Recent interest in Ceramide supports this hypothesis. Ceramide has been long known to be involved in apoptosis <a href="#pone.0000425-Santana1">[37]</a>–<a href="#pone.0000425-Paris2">[39]</a> and recent work is looking at the relevancy of ceramide in cancer <a href="#pone.0000425-Bonnaud1">[40]</a>–<a href="#pone.0000425-Lin1">[42]</a> and in cancer therapy <a href="#pone.0000425-Claus1">[43]</a>, <a href="#pone.0000425-Padron1">[44]</a>. Similar interest has been developing in calcineurin. Whereas interest was previously confined to its activity in immune response, it is now becoming recognized as a predominant participant in oncogenesis <a href="#pone.0000425-Buchholz1">[45]</a>, <a href="#pone.0000425-Buchholz2">[46]</a>.&nbsp;The combination of this set of pathways may define key processes that are characteristic of a universal progenitor cell type.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Conversely, the pathway analysis of cancer sub-phenotypes may also provide novel mechanistic insights that reveal underlying biology. For example, tamoxifen is effective in treating some cases of ER+ breast cancer. In these cases, tamoxifen must be affecting the activity of interaction networks. It is therefore logical to hypothesize that there will be observable differences in network activity between those cases where tamoxifen is effective and those cases where the drug is not effective. Our approach uses pathway signatures to predict variance in outcome, which is taken as the measure of drug effectiveness. We speculate that our approach can reveal those networks that are both differentially activated in response to treatment with tamoxifen and important to tumor growth and sustainability.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The approach applied here has parallels to the use of gene maps for translating phenotypes into the molecular domain. First, pathway models represent a reproducible framework that can be tested across studies and extended as further knowledge becomes available. Also, the pathways and their structure provide a higher order construct for assessing the role of genes.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Each interaction within a pathway requires the contribution of multiple gene observations. Each single gene activity level contributes only in the context of other genes participating in an interaction within the process network. This is demonstrated by the observation that we were unable to derive effective classifiers, directly from the gene-state values alone (for the genes composing the main six pathways).</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>It is also interesting that five of the six pathways we use to classify normal and tumour samples form a single connected network (<a href="#pone-0000425-g005">Figure 5</a>, the telomerase pathway remains unconnected). This interconnection may provide novel opportunities for developing interventions. Knowledge of the connections may suggest alternative targets that would have multiple pathway effects. Minimally, it may permit the identification of complexities associated with target selection prior to intervention design.</p>
<div class="figure" id="pone-0000425-g005"><div class="img"><a name="pone-0000425-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000425" data-uri="info:doi/10.1371/journal.pone.0000425.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000425.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000425.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000425.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000425.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The sub network formed by the six pathways that together create the normal/tumor classifier.</span></strong></p><a id="article1.body1.sec4.fig1.caption1.p1" name="article1.body1.sec4.fig1.caption1.p1"></a><p>The joined pathways color shared nodes.</p>
<span>doi:10.1371/journal.pone.0000425.g005</span></div><a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>It is understood that the probabilistic classification of genes into alternative states of down and up is a simplification of much greater complexity patterns of gene behaviour and action. However, empiric evaluation of the observed data finds that gene expression patterns commonly can fit one of two alternative expression level distributions. Moreover, such simplification has proven valuable in other research domains. For example the simplification that abstracts digital logic from the underlying continuous flow of electrons in integrated circuits has enabled the design of devices of staggeringly complex functionality <a href="#pone.0000425-Regev1">[47]</a>.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>It is clear that current knowledge of biologic pathways is incomplete and imperfect. As such, processes identified are almost assuredly not the only factors influencing the phenotypes of interest. Nevertheless, where processes are identified, they serve as important targets for further investigation. Moreover, the process-oriented approach allows one to distinguish which components of the complex networks in which genes participate are differentially contributing to a phenotype of interest. The combined use of activity and consistency score permits the discrimination of processes activated because of the phenotype versus those whose logic differs between phenotypes. The latter (consistency), potentially is causally attributable to the phenotype and suggests candidates that have been altered. However, utilizing gene expression data, consistency scores can only be calculated for interactions involving transcription events, limiting their discriminatory power.</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: KB SE CS. Performed the experiments: SE. Analyzed the data: SE. Wrote the paper: KB SE CS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000425-Altomare1" id="pone.0000425-Altomare1"></a>Altomare DA, Testa JR (2005) Perturbations of the AKT signaling pathway in human cancer. Oncogene  24: 7455–7464.  <ul class="find" data-citedArticleID="949386" data-doi="10.1038/sj.onc.1209085"><li><a href="http://dx.doi.org/10.1038/sj.onc.1209085" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perturbations+of+the+AKT+signaling+pathway+in+human+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perturbations+of+the+AKT+signaling+pathway+in+human+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000425-Hanahan1" id="pone.0000425-Hanahan1"></a>Hanahan D, Weinberg RA (2000) The hallmarks of cancer. Cell  100: 57–70.  <ul class="find" data-citedArticleID="949414" data-doi="10.1016/s0092-8674(00)81683-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81683-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hallmarks+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hallmarks+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000425-Parsons1" id="pone.0000425-Parsons1"></a>Parsons DW, Wang TL, Samuels Y, Bardelli A, Cummins JM, et al.  (2005) Colorectal cancer: mutations in a signalling pathway. Nature  436: 792.  <ul class="find" data-citedArticleID="949436" data-doi="10.1038/436792a"><li><a href="http://dx.doi.org/10.1038/436792a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Colorectal+cancer%3A+mutations+in+a+signalling+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Colorectal+cancer%3A+mutations+in+a+signalling+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000425-Matoba1" id="pone.0000425-Matoba1"></a>Matoba S, Kang JG, Patino WD, Wragg A, Boehm M, et al.  (2006) p53 regulates mitochondrial respiration. Science  312: 1650–1653.  <ul class="find" data-citedArticleID="949424" data-doi="10.1126/science.1126863"><li><a href="http://dx.doi.org/10.1126/science.1126863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=p53+regulates+mitochondrial+respiration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22p53+regulates+mitochondrial+respiration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000425-Bianco1" id="pone.0000425-Bianco1"></a>Bianco R, Melisi D, Ciardiello F, Tortora G (2006) Key cancer cell signal transduction pathways as therapeutic targets. Eur J Cancer  42: 290–294.  <ul class="find" data-citedArticleID="949390" data-doi="10.1016/j.ejca.2005.07.034"><li><a href="http://dx.doi.org/10.1016/j.ejca.2005.07.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Key+cancer+cell+signal+transduction+pathways+as+therapeutic+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Key+cancer+cell+signal+transduction+pathways+as+therapeutic+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000425-Segal1" id="pone.0000425-Segal1"></a>Segal E, Friedman N, Koller D, Regev A (2004) A module map showing conditional activity of expression modules in cancer. Nat Genet  36: 1090–1098.  <ul class="find" data-citedArticleID="949452" data-doi="10.1038/ng1434"><li><a href="http://dx.doi.org/10.1038/ng1434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+module+map+showing+conditional+activity+of+expression+modules+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+module+map+showing+conditional+activity+of+expression+modules+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000425-Rhodes1" id="pone.0000425-Rhodes1"></a>Rhodes DR, Chinnaiyan AM (2005) Integrative analysis of the cancer transcriptome. Nat Genet 37 SupplS31–37.  <ul class="find" data-citedArticleID="949442" data-doi="10.1038/ng1570"><li><a href="http://dx.doi.org/10.1038/ng1570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrative+analysis+of+the+cancer+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrative+analysis+of+the+cancer+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000425-Ramaswamy1" id="pone.0000425-Ramaswamy1"></a>Ramaswamy S, Tamayo P, Rifkin R, Mukherjee S, Yeang CH, et al.  (2001) Multiclass cancer diagnosis using tumor gene expression signatures. Proc Natl Acad Sci U S A  98: 15149–15154.  <ul class="find" data-citedArticleID="949438" data-doi="10.1073/pnas.211566398"><li><a href="http://dx.doi.org/10.1073/pnas.211566398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiclass+cancer+diagnosis+using+tumor+gene+expression+signatures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiclass+cancer+diagnosis+using+tumor+gene+expression+signatures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000425-Wang1" id="pone.0000425-Wang1"></a>Wang Y, Klijn JG, Zhang Y, Sieuwerts AM, Look MP, et al.  (2005) Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer. Lancet  365: 671–679.  <ul class="find" data-citedArticleID="949472" data-doi="10.1016/s0140-6736(05)70933-8"><li><a href="http://dx.doi.org/10.1016/s0140-6736(05)70933-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene-expression+profiles+to+predict+distant+metastasis+of+lymph-node-negative+primary+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene-expression+profiles+to+predict+distant+metastasis+of+lymph-node-negative+primary+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000425-Su1" id="pone.0000425-Su1"></a>Su AI, Cooke MP, Ching KA, Hakak Y, Walker JR, et al.  (2002) Large-scale analysis of the human and mouse transcriptomes. Proc Natl Acad Sci U S A  99: 4465–4470.  <ul class="find" data-citedArticleID="949458" data-doi="10.1073/pnas.012025199"><li><a href="http://dx.doi.org/10.1073/pnas.012025199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+analysis+of+the+human+and+mouse+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+analysis+of+the+human+and+mouse+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000425-Roth1" id="pone.0000425-Roth1"></a>Roth RB, Hevezi P, Lee J, Willhite D, Lechner SM, et al.  (2006) Gene expression analyses reveal molecular relationships among 20 regions of the human CNS. Neurogenetics.   <ul class="find" data-citedArticleID="949446" data-doi="10.1007/s10048-006-0032-6"><li><a href="http://dx.doi.org/10.1007/s10048-006-0032-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+analyses+reveal+molecular+relationships+among+20+regions+of+the+human+CNS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+analyses+reveal+molecular+relationships+among+20+regions+of+the+human+CNS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000425-Su2" id="pone.0000425-Su2"></a>Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, et al.  (2004) A gene atlas of the mouse and human protein-encoding transcriptomes. Proc Natl Acad Sci U S A  101: 6062–6067.  <ul class="find" data-citedArticleID="949460" data-doi="10.1073/pnas.0400782101"><li><a href="http://dx.doi.org/10.1073/pnas.0400782101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000425-Bittner1" id="pone.0000425-Bittner1"></a>Bittner M (2006)   International Genomics Consortium (IGC) <a href="http://www.intgen.org/">http://www.intgen.org/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000425-Bild1" id="pone.0000425-Bild1"></a>Bild AH, Yao G, Chang JT, Wang Q, Potti A, et al.  (2006) Oncogenic pathway signatures in human cancers as a guide to targeted therapies. Nature  439: 353–357.  <ul class="find" data-citedArticleID="949392" data-doi="10.1038/nature04296"><li><a href="http://dx.doi.org/10.1038/nature04296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oncogenic+pathway+signatures+in+human+cancers+as+a+guide+to+targeted+therapies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oncogenic+pathway+signatures+in+human+cancers+as+a+guide+to+targeted+therapies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000425-Miller1" id="pone.0000425-Miller1"></a>Miller LD, Smeds J, George J, Vega VB, Vergara L, et al.  (2005) An expression signature for p53 status in human breast cancer predicts mutation status, transcriptional effects, and patient survival. Proc Natl Acad Sci U S A  102: 13550–13555.  <ul class="find" data-citedArticleID="949426" data-doi="10.1073/pnas.0506230102"><li><a href="http://dx.doi.org/10.1073/pnas.0506230102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+expression+signature+for+p53+status+in+human+breast+cancer+predicts+mutation+status%2C+transcriptional+effects%2C+and+patient+survival." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+expression+signature+for+p53+status+in+human+breast+cancer+predicts+mutation+status%2C+transcriptional+effects%2C+and+patient+survival.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000425-Yu1" id="pone.0000425-Yu1"></a>Yu YP, Landsittel D, Jing L, Nelson J, Ren B, et al.  (2004) Gene expression alterations in prostate cancer predicting tumor aggression and preceding development of malignancy. J Clin Oncol  22: 2790–2799.  <ul class="find" data-citedArticleID="949478" data-doi="10.1200/jco.2004.05.158"><li><a href="http://dx.doi.org/10.1200/jco.2004.05.158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+alterations+in+prostate+cancer+predicting+tumor+aggression+and+preceding+development+of+malignancy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+alterations+in+prostate+cancer+predicting+tumor+aggression+and+preceding+development+of+malignancy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000425-Bhattacharjee1" id="pone.0000425-Bhattacharjee1"></a>Bhattacharjee A, Richards WG, Staunton J, Li C, Monti S, et al.  (2001) Classification of human lung carcinomas by mRNA expression profiling reveals distinct adenocarcinoma subclasses. Proc Natl Acad Sci U S A  98: 13790–13795.  <ul class="find" data-citedArticleID="949388" data-doi="10.1073/pnas.191502998"><li><a href="http://dx.doi.org/10.1073/pnas.191502998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+human+lung+carcinomas+by+mRNA+expression+profiling+reveals+distinct+adenocarcinoma+subclasses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+human+lung+carcinomas+by+mRNA+expression+profiling+reveals+distinct+adenocarcinoma+subclasses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000425-Irizarry1" id="pone.0000425-Irizarry1"></a>Irizarry RA, Bolstad BM, Collin F, Cope LM, Hobbs B, et al.  (2003) Summaries of Affymetrix GeneChip probe level data. Nucleic Acids Res  31: e15.  <ul class="find" data-citedArticleID="949420" data-doi="10.1093/nar/gng015"><li><a href="http://dx.doi.org/10.1093/nar/gng015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Summaries+of+Affymetrix+GeneChip+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Summaries+of+Affymetrix+GeneChip+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000425-Vogelstein1" id="pone.0000425-Vogelstein1"></a>Vogelstein B, Kinzler KW (2004) Cancer genes and the pathways they control. Nat Med  10: 789–799.  <ul class="find" data-citedArticleID="949470" data-doi="10.1038/nm1087"><li><a href="http://dx.doi.org/10.1038/nm1087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cancer+genes+and+the+pathways+they+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cancer+genes+and+the+pathways+they+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000425-Watters1" id="pone.0000425-Watters1"></a>Watters JW, Roberts CJ (2006) Developing gene expression signatures of pathway deregulation in tumors. Mol Cancer Ther  5: 2444–2449.  <ul class="find" data-citedArticleID="949474" data-doi="10.1158/1535-7163.mct-06-0340"><li><a href="http://dx.doi.org/10.1158/1535-7163.mct-06-0340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developing+gene+expression+signatures+of+pathway+deregulation+in+tumors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developing+gene+expression+signatures+of+pathway+deregulation+in+tumors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000425-Rhodes2" id="pone.0000425-Rhodes2"></a>Rhodes DR, Tomlins SA, Varambally S, Mahavisno V, Barrette T, et al.  (2005) Probabilistic model of the human protein-protein interaction network. Nat Biotechnol  23: 951–959.  <ul class="find" data-citedArticleID="949444" data-doi="10.1038/nbt1103"><li><a href="http://dx.doi.org/10.1038/nbt1103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probabilistic+model+of+the+human+protein-protein+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probabilistic+model+of+the+human+protein-protein+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000425-Glinsky1" id="pone.0000425-Glinsky1"></a>Glinsky GV, Berezovska O, Glinskii AB (2005) Microarray analysis identifies a death-from-cancer signature predicting therapy failure in patients with multiple types of cancer. J Clin Invest  115: 1503–1521.  <ul class="find" data-citedArticleID="949412" data-doi="10.1172/jci23412"><li><a href="http://dx.doi.org/10.1172/jci23412" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+identifies+a+death-from-cancer+signature+predicting+therapy+failure+in+patients+with+multiple+types+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+identifies+a+death-from-cancer+signature+predicting+therapy+failure+in+patients+with+multiple+types+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000425-Subramanian1" id="pone.0000425-Subramanian1"></a>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, et al.  (2005) Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A  102: 15545–15550.  <ul class="find" data-citedArticleID="949462" data-doi="10.1073/pnas.0506580102"><li><a href="http://dx.doi.org/10.1073/pnas.0506580102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000425-Tomfohr1" id="pone.0000425-Tomfohr1"></a>Tomfohr J, Lu J, Kepler TB (2005) Pathway level analysis of gene expression using singular value decomposition. BMC Bioinformatics  6: 225.  <ul class="find" data-citedArticleID="949468" data-doi="10.1186/1471-2105-6-225"><li><a href="http://dx.doi.org/10.1186/1471-2105-6-225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathway+level+analysis+of+gene+expression+using+singular+value+decomposition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathway+level+analysis+of+gene+expression+using+singular+value+decomposition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000425-Duda1" id="pone.0000425-Duda1"></a>Duda RO, Hart PE, Stork DG (2001) Pattern classification. xx. New York: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000425-Hastie1" id="pone.0000425-Hastie1"></a>Hastie T, Tibshirani R, Friedman JH (2001) The elements of statistical learning : data mining, inference, and prediction : with 200 full-color illustrations. xvi. New York: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0000425-National1" id="pone.0000425-National1"></a>  National Cancer Institute Center for Bioinformatics 2005 Pathway Interaction Database <a href="http://pid.nci.nih.gov">http://pid.nci.nih.gov</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0000425-Theodoridis1" id="pone.0000425-Theodoridis1"></a>Theodoridis S, Koutroumbas K (2003) Pattern recognition. xiv. Amsterdam; Boston: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0000425-Webb1" id="pone.0000425-Webb1"></a>Webb AR (2002) Statistical pattern recognition. xviii. West Sussex, New Jersey, England: Wiley.   <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0000425-Buetow1" id="pone.0000425-Buetow1"></a>Buetow KH, Klausner RD, Fine H, Kaplan R, Singer DS, et al.  (2002) Cancer Molecular Analysis Project: weaving a rich cancer research tapestry. Cancer Cell  1: 315–318.  <ul class="find" data-citedArticleID="949402" data-doi="10.1016/s1535-6108(02)00065-x"><li><a href="http://dx.doi.org/10.1016/s1535-6108(02)00065-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cancer+Molecular+Analysis+Project%3A+weaving+a+rich+cancer+research+tapestry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cancer+Molecular+Analysis+Project%3A+weaving+a+rich+cancer+research+tapestry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000425-Schaefer1" id="pone.0000425-Schaefer1"></a>Schaefer CF (2004) Pathway databases. Ann N Y Acad Sci  1020: 77–91.  <ul class="find" data-citedArticleID="949450" data-doi="10.1196/annals.1310.009"><li><a href="http://dx.doi.org/10.1196/annals.1310.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pathway+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pathway+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000425-Dolle1" id="pone.0000425-Dolle1"></a>Dolle L, Adriaenssens E, El Yazidi-Belkoura I, Le Bourhis X, Nurcombe V, et al.  (2004) Nerve growth factor receptors and signaling in breast cancer. Curr Cancer Drug Targets  4: 463–470.  <ul class="find" data-citedArticleID="949406" data-doi="10.2174/1568009043332853"><li><a href="http://dx.doi.org/10.2174/1568009043332853" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nerve+growth+factor+receptors+and+signaling+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nerve+growth+factor+receptors+and+signaling+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000425-Gielen1" id="pone.0000425-Gielen1"></a>Gielen SC, Kuhne LC, Ewing PC, Blok LJ, Burger CW (2005) Tamoxifen treatment for breast cancer enforces a distinct gene-expression profile on the human endometrium: an exploratory study. Endocr Relat Cancer  12: 1037–1049.  <ul class="find" data-citedArticleID="949410" data-doi="10.1677/erc.1.01046"><li><a href="http://dx.doi.org/10.1677/erc.1.01046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tamoxifen+treatment+for+breast+cancer+enforces+a+distinct+gene-expression+profile+on+the+human+endometrium%3A+an+exploratory+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tamoxifen+treatment+for+breast+cancer+enforces+a+distinct+gene-expression+profile+on+the+human+endometrium%3A+an+exploratory+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000425-Hiscox1" id="pone.0000425-Hiscox1"></a>Hiscox S, Jiang WG, Obermeier K, Taylor K, Morgan L, et al.  (2006) Tamoxifen resistance in MCF7 cells promotes EMT-like behaviour and involves modulation of beta-catenin phosphorylation. Int J Cancer  118: 290–301.  <ul class="find" data-citedArticleID="949418" data-doi="10.1002/ijc.21355"><li><a href="http://dx.doi.org/10.1002/ijc.21355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tamoxifen+resistance+in+MCF7+cells+promotes+EMT-like+behaviour+and+involves+modulation+of+beta-catenin+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tamoxifen+resistance+in+MCF7+cells+promotes+EMT-like+behaviour+and+involves+modulation+of+beta-catenin+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000425-Sharma1" id="pone.0000425-Sharma1"></a>Sharma S, Batra RK, Yang SC, Hillinger S, Zhu L, et al.  (2003) Interleukin-7 gene-modified dendritic cells reduce pulmonary tumor burden in spontaneous murine bronchoalveolar cell carcinoma. Hum Gene Ther  14: 1511–1524.  <ul class="find" data-citedArticleID="949454" data-doi="10.1089/104303403322495025"><li><a href="http://dx.doi.org/10.1089/104303403322495025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interleukin-7+gene-modified+dendritic+cells+reduce+pulmonary+tumor+burden+in+spontaneous+murine+bronchoalveolar+cell+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interleukin-7+gene-modified+dendritic+cells+reduce+pulmonary+tumor+burden+in+spontaneous+murine+bronchoalveolar+cell+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000425-Stepulak1" id="pone.0000425-Stepulak1"></a>Stepulak A, Sifringer M, Rzeski W, Endesfelder S, Gratopp A, et al.  (2005) NMDA antagonist inhibits the extracellular signal-regulated kinase pathway and suppresses cancer growth. Proc Natl Acad Sci U S A  102: 15605–15610.  <ul class="find" data-citedArticleID="949456" data-doi="10.1073/pnas.0507679102"><li><a href="http://dx.doi.org/10.1073/pnas.0507679102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NMDA+antagonist+inhibits+the+extracellular+signal-regulated+kinase+pathway+and+suppresses+cancer+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NMDA+antagonist+inhibits+the+extracellular+signal-regulated+kinase+pathway+and+suppresses+cancer+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000425-Santana1" id="pone.0000425-Santana1"></a>Santana P, Pena LA, Haimovitz-Friedman A, Martin S, Green D, et al.  (1996) Acid sphingomyelinase-deficient human lymphoblasts and mice are defective in radiation-induced apoptosis. Cell  86: 189–199.  <ul class="find" data-citedArticleID="949448" data-doi="10.1016/s0092-8674(00)80091-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80091-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acid+sphingomyelinase-deficient+human+lymphoblasts+and+mice+are+defective+in+radiation-induced+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acid+sphingomyelinase-deficient+human+lymphoblasts+and+mice+are+defective+in+radiation-induced+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000425-Paris1" id="pone.0000425-Paris1"></a>Paris F, Fuks Z, Kang A, Capodieci P, Juan G, et al.  (2001) Endothelial apoptosis as the primary lesion initiating intestinal radiation damage in mice. Science  293: 293–297.  <ul class="find" data-citedArticleID="949432" data-doi="10.1126/science.1060191"><li><a href="http://dx.doi.org/10.1126/science.1060191" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endothelial+apoptosis+as+the+primary+lesion+initiating+intestinal+radiation+damage+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endothelial+apoptosis+as+the+primary+lesion+initiating+intestinal+radiation+damage+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000425-Paris2" id="pone.0000425-Paris2"></a>Paris F, Grassme H, Cremesti A, Zager J, Fong Y, et al.  (2001) Natural ceramide reverses Fas resistance of acid sphingomyelinase(−/−) hepatocytes. J Biol Chem  276: 8297–8305.  <ul class="find" data-citedArticleID="949434" data-doi="10.1074/jbc.m008732200"><li><a href="http://dx.doi.org/10.1074/jbc.m008732200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+ceramide+reverses+Fas+resistance+of+acid+sphingomyelinase%28%26%23x2212%3B%2F%26%23x2212%3B%29+hepatocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+ceramide+reverses+Fas+resistance+of+acid+sphingomyelinase%28%26%23x2212%3B%2F%26%23x2212%3B%29+hepatocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000425-Bonnaud1" id="pone.0000425-Bonnaud1"></a>Bonnaud S, Niaudet C, Pottier G, Gaugler MH, Millour J, et al.  (2007) Sphingosine-1-phosphate protects proliferating endothelial cells from ceramide-induced apoptosis but not from DNA damage-induced mitotic death. Cancer Res  67: 1803–1811.  <ul class="find" data-citedArticleID="949396" data-doi="10.1158/0008-5472.can-06-2802"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-2802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sphingosine-1-phosphate+protects+proliferating+endothelial+cells+from+ceramide-induced+apoptosis+but+not+from+DNA+damage-induced+mitotic+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sphingosine-1-phosphate+protects+proliferating+endothelial+cells+from+ceramide-induced+apoptosis+but+not+from+DNA+damage-induced+mitotic+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000425-Thevissen1" id="pone.0000425-Thevissen1"></a>Thevissen K, Francois IE, Winderickx J, Pannecouque C, Cammue BP (2006) Ceramide involvement in apoptosis and apoptotic diseases. Mini Rev Med Chem  6: 699–709.  <ul class="find" data-citedArticleID="949466" data-doi="10.2174/138955706777435643"><li><a href="http://dx.doi.org/10.2174/138955706777435643" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ceramide+involvement+in+apoptosis+and+apoptotic+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ceramide+involvement+in+apoptosis+and+apoptotic+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000425-Lin1" id="pone.0000425-Lin1"></a>Lin CF, Chen CL, Lin YS (2006) Ceramide in apoptotic signaling and anticancer therapy. Curr Med Chem  13: 1609–1616.  <ul class="find" data-citedArticleID="949422" data-doi="10.2174/092986706777441986"><li><a href="http://dx.doi.org/10.2174/092986706777441986" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ceramide+in+apoptotic+signaling+and+anticancer+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ceramide+in+apoptotic+signaling+and+anticancer+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000425-Claus1" id="pone.0000425-Claus1"></a>Claus R, Russwurm S, Meisner M, Kinscherf R, Deigner HP (2000) Modulation of the ceramide level, a novel therapeutic concept? Curr Drug Targets  1: 185–205.  <ul class="find" data-citedArticleID="949404" data-doi="10.2174/1389450003349272"><li><a href="http://dx.doi.org/10.2174/1389450003349272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+the+ceramide+level%2C+a+novel+therapeutic+concept%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+the+ceramide+level%2C+a+novel+therapeutic+concept%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000425-Padron1" id="pone.0000425-Padron1"></a>Padron JM (2006) Sphingolipids in anticancer therapy. Curr Med Chem  13: 755–770.  <ul class="find" data-citedArticleID="949430" data-doi="10.2174/092986706776055553"><li><a href="http://dx.doi.org/10.2174/092986706776055553" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sphingolipids+in+anticancer+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sphingolipids+in+anticancer+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000425-Buchholz1" id="pone.0000425-Buchholz1"></a>Buchholz M, Ellenrieder V (2007) An emerging role for Ca2+/calcineurin/NFAT signaling in cancerogenesis. Cell Cycle  6: 16–19.  <ul class="find" data-citedArticleID="949398" data-doi="10.4161/cc.6.1.3650"><li><a href="http://dx.doi.org/10.4161/cc.6.1.3650" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+emerging+role+for+Ca2%2B%2Fcalcineurin%2FNFAT+signaling+in+cancerogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+emerging+role+for+Ca2%2B%2Fcalcineurin%2FNFAT+signaling+in+cancerogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000425-Buchholz2" id="pone.0000425-Buchholz2"></a>Buchholz M, Schatz A, Wagner M, Michl P, Linhart T, et al.  (2006) Overexpression of c-myc in pancreatic cancer caused by ectopic activation of NFATc1 and the Ca2+/calcineurin signaling pathway. Embo J  25: 3714–3724.  <ul class="find" data-citedArticleID="949400" data-doi="10.1038/sj.emboj.7601246"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+c-myc+in+pancreatic+cancer+caused+by+ectopic+activation+of+NFATc1+and+the+Ca2%2B%2Fcalcineurin+signaling+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+c-myc+in+pancreatic+cancer+caused+by+ectopic+activation+of+NFATc1+and+the+Ca2%2B%2Fcalcineurin+signaling+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000425-Regev1" id="pone.0000425-Regev1"></a>Regev A, Shapiro E (2002) Cells as computation. Nature  419: 343.  <ul class="find" data-citedArticleID="949440" data-doi="10.1038/419343a"><li><a href="http://dx.doi.org/10.1038/419343a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cells+as+computation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cells+as+computation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.XML" value="89174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.PDF" value="608179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e001.PNG" value="6591"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e001.TIF" value="27556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e002.PNG" value="3348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e002.TIF" value="25280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e003.PNG" value="5540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e003.TIF" value="26992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e004.PNG" value="5043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e004.TIF" value="27664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e005.PNG" value="6903"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e005.TIF" value="29480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e006.PNG" value="6759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e006.TIF" value="27480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e007.PNG" value="1480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e007.TIF" value="22636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e008.PNG" value="1099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e008.TIF" value="22088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e009.PNG" value="2611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e009.TIF" value="24260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e010.PNG" value="5065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e010.TIF" value="26808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e011.PNG" value="2119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e011.TIF" value="23328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e012.PNG" value="8494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e012.TIF" value="30828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e013.PNG" value="1379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e013.TIF" value="22348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e014.PNG" value="6812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e014.TIF" value="27864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e015.PNG" value="7337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e015.TIF" value="28464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e016.PNG" value="3245"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e016.TIF" value="25176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e017.PNG" value="3190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e017.TIF" value="25076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e018.PNG" value="8873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e018.TIF" value="29692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g001.PNG_L" value="131834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g001.PNG_M" value="75081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g001.PNG_S" value="2791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g001.TIF" value="249966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g001.PNG_I" value="27838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e019.PNG" value="13803"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.e019.TIF" value="38028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g002.PNG_L" value="244180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g002.PNG_M" value="69487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g002.PNG_S" value="2218"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g002.TIF" value="377862"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g002.PNG_I" value="26930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g003.PNG_L" value="519423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g003.PNG_M" value="80657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g003.PNG_S" value="3139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g003.TIF" value="1065486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g003.PNG_I" value="49047"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.t001.PNG_L" value="366266"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.t001.PNG_M" value="127542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.t001.PNG_S" value="5305"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.t001.TIF" value="1693088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.t001.PNG_I" value="53297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g004.PNG_L" value="222398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g004.PNG_M" value="86416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g004.PNG_S" value="3304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g004.TIF" value="909640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g004.PNG_I" value="37099"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g005.PNG_L" value="254567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g005.PNG_M" value="52334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g005.PNG_S" value="2216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g005.TIF" value="682430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000425.g005.PNG_I" value="34075"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000425&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000425" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000425&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000425&volume=&issue=&title=Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis&author_name=Sol%20Efroni%2C%20Carl%20F.%20Schaefer%2C%20Kenneth%20H.%20Buetow&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000425" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425&amp;t=Identification%20of%20Key%20Processes%20Underlying%20Cancer%20Phenotypes%20Using%20Biologic%20Pathway%20Analysis" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425&title=Identification%20of%20Key%20Processes%20Underlying%20Cancer%20Phenotypes%20Using%20Biologic%20Pathway%20Analysis&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425&amp;title=Identification%20of%20Key%20Processes%20Underlying%20Cancer%20Phenotypes%20Using%20Biologic%20Pathway%20Analysis" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000425&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Identification of Key Processes Underlying Cancer Phenotypes Using Biologic Pathway Analysis';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000425';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Identification%20of%20Key%20Processes%20Underlying%20Cancer%20Phenotypes%20Using%20Biologic%20Pathway%20Analysis http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000425" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000425" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Breast+cancer%22" title="Search for articles in the subject area:'Breast cancer'"><div class="flagText">Breast cancer</div></a>
              <div data-categoryid="18763" data-articleid="23688"
                   data-categoryname="Breast cancer"
                   class="flagImage" title="Flag 'Breast cancer' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cellular+signal+transduction+analysis%22" title="Search for articles in the subject area:'Cellular signal transduction analysis'"><div class="flagText">Cellular signal transduction analysis</div></a>
              <div data-categoryid="27313" data-articleid="23688"
                   data-categoryname="Cellular signal transduction analysis"
                   class="flagImage" title="Flag 'Cellular signal transduction analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Developmental+signaling%22" title="Search for articles in the subject area:'Developmental signaling'"><div class="flagText">Developmental signaling</div></a>
              <div data-categoryid="18629" data-articleid="23688"
                   data-categoryname="Developmental signaling"
                   class="flagImage" title="Flag 'Developmental signaling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23688"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="40201" data-articleid="23688"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hedgehog+signaling%22" title="Search for articles in the subject area:'Hedgehog signaling'"><div class="flagText">Hedgehog signaling</div></a>
              <div data-categoryid="17323" data-articleid="23688"
                   data-categoryname="Hedgehog signaling"
                   class="flagImage" title="Flag 'Hedgehog signaling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23688"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probability+distribution%22" title="Search for articles in the subject area:'Probability distribution'"><div class="flagText">Probability distribution</div></a>
              <div data-categoryid="32037" data-articleid="23688"
                   data-categoryname="Probability distribution"
                   class="flagImage" title="Flag 'Probability distribution' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3677'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1950'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9926&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=14963">Sample data</a><br>Posted by einar</p>
      <p><a href="/annotation/listThread.action?root=9065">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
