

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001958"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001958" />

    <meta name="citation_title" content="HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model"/>
    <meta itemprop="name" content="HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model"/>

      <meta name="citation_author" content="Myriam Rai"/>
            <meta name="citation_author_institution" content="Laboratoire de Neurologie Expérimentale, Hôpital Erasme, Université Libre de Bruxelles (ULB), Brussels, Belgium"/>
      <meta name="citation_author" content="Elisabetta Soragni"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Kai Jenssen"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Ryan Burnett"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="David Herman"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Giovanni Coppola"/>
            <meta name="citation_author_institution" content="Program in Neurogenetics, Department of Neurology, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Daniel H. Geschwind"/>
            <meta name="citation_author_institution" content="Program in Neurogenetics, Department of Neurology, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Joel M. Gottesfeld"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Massimo Pandolfo"/>
            <meta name="citation_author_institution" content="Laboratoire de Neurologie Expérimentale, Hôpital Erasme, Université Libre de Bruxelles (ULB), Brussels, Belgium"/>

    <meta name="citation_date" content="2008/4/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001958.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1958"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Friedreich ataxia.; citation_author=M Pandolfo; citation_title=Genetic Instabilities and Neurological Diseases.; citation_number=1; citation_pages=277-298; citation_date=2006; citation_publisher=Elsevier-Academic Press; " />
      <meta name="citation_reference" content="citation_title=Frataxin deficiency and mitochondrial dysfunction.; citation_author=M Pandolfo; citation_journal_title=Mitochondrion; citation_volume=2; citation_number=2; citation_pages=87-93; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Friedreich ataxia and related RNA loss-of-function disorders.; citation_author=M Pandolfo; citation_title=Molecular Neurology.; citation_number=3; citation_date=2007; citation_publisher=Elsevier. Chapter 18; " />
      <meta name="citation_reference" content="citation_title=Supramolecular assemblies of human frataxin are formed via subunit-subunit interactions mediated by a non-conserved amino-terminal region.; citation_author=HA O'Neill; citation_author=O Gakh; citation_author=G Isaya; citation_journal_title=J Mol Biol; citation_volume=345; citation_number=4; citation_pages=433-439; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inhibition of Fe-S cluster biosynthesis decreases mitochondrial iron export: evidence that Yfh1p affects Fe-S cluster synthesis.; citation_author=OS Chen; citation_author=S Hemenway; citation_author=J Kaplan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=5; citation_pages=12321-12326; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Frataxin-mediated iron delivery to ferrochelatase in the final step of heme biosynthesis.; citation_author=T Yoon; citation_author=JA Cowan; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=6; citation_pages=25943-25946; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Assembly of human frataxin is a mechanism for detoxifying redox-active iron.; citation_author=HA O'Neill; citation_author=O Gakh; citation_author=S Park; citation_author=J Cui; citation_author=SM Mooney; citation_journal_title=Biochemistry; citation_volume=44; citation_number=7; citation_pages=537-545; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Aconitase and mitochondrial iron-sulphur protein deficiency in Friedreich ataxia.; citation_author=A Rotig; citation_author=P de Lonlay; citation_author=D Chretien; citation_author=F Foury; citation_author=M Koenig; citation_journal_title=Nat Genet; citation_volume=17; citation_number=8; citation_pages=215-217; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=RNAi-mediated suppression of the mitochondrial iron chaperone, frataxin, in Drosophila.; citation_author=PR Anderson; citation_author=K Kirby; citation_author=AJ Hilliker; citation_author=JP Phillips; citation_journal_title=Hum Mol Genet; citation_volume=14; citation_number=9; citation_pages=3397-3405; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inactivation of the Friedreich ataxia mouse gene leads to early embryonic lethality without iron accumulation.; citation_author=M Cossée; citation_author=H Puccio; citation_author=A Gansmuller; citation_author=H Koutnikova; citation_author=A Dierich; citation_journal_title=Hum Mol Genet; citation_volume=9; citation_number=10; citation_pages=1219-1226; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Knockout of frataxin gene causes embryo lethality in Arabidopsis.; citation_author=V Vazzola; citation_author=A Losa; citation_author=C Soave; citation_author=I Murgia; citation_journal_title=FEBS Lett; citation_volume=581; citation_number=11; citation_pages=667-672; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=C. elegans as a model for Friedreich Ataxia.; citation_author=N Ventura; citation_author=SL Rea; citation_author=ST Handerson; citation_author=I Condo; citation_author=R Testi; citation_author=TE Johnson; citation_journal_title=Faseb J; citation_volume=20; citation_number=12; citation_pages=1029-1030; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Friedreich ataxia mouse models with progressive cerebellar and sensory ataxia reveal autophagic neurodegeneration in dorsal root ganglia.; citation_author=D Simon; citation_author=H Seznec; citation_author=A Gansmuller; citation_author=N Carelle; citation_author=P Weber; citation_journal_title=J Neurosci; citation_volume=24; citation_number=13; citation_pages=1987-1995; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Friedreich's ataxia: autosomal recessive disease caused by an intronic GAA triplet repeat expansion.; citation_author=V Campuzano; citation_author=L Montermini; citation_author=MD Moltò; citation_author=L Pianese; citation_author=M Cossée; citation_journal_title=Science; citation_volume=271; citation_number=14; citation_pages=1423-1427; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Frataxin is reduced in Friedreich ataxia patients and is associated with mitochondrial membranes.; citation_author=V Campuzano; citation_author=L Montermini; citation_author=Y Lutz; citation_author=L Cova; citation_author=C Hindelang; citation_journal_title=Hum Mol Genet; citation_volume=6; citation_number=15; citation_pages=1771-1780; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Phenotypic variability in Friedreich ataxia: role of the associated GAA triplet repeat expansion.; citation_author=L Montermini; citation_author=A Richter; citation_author=K Morgan; citation_author=CM Justice; citation_author=D Julien; citation_journal_title=Ann Neurol; citation_volume=41; citation_number=16; citation_pages=675-682; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The GAA*TTC triplet repeat expanded in Friedreich's ataxia impedes transcription elongation by T7 RNA polymerase in a length and supercoil dependent manner.; citation_author=E Grabczyk; citation_author=K Usdin; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=17; citation_pages=2815-2822; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cloning, characterization, and properties of seven triplet repeat DNA sequences.; citation_author=K Ohshima; citation_author=S Kang; citation_author=JE Larson; citation_author=RD Wells; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=18; citation_pages=16773-16783; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Inhibitory effects of expanded GAA.TTC triplet repeats from intron I of the Friedreich ataxia gene on transcription and replication in vivo.; citation_author=K Ohshima; citation_author=L Montermini; citation_author=RD Wells; citation_author=M Pandolfo; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=19; citation_pages=14588-14595; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Sticky DNA: self-association properties of long GAA.TTC repeats in R.R.Y triplex structures from Friedreich's ataxia.; citation_author=N Sakamoto; citation_author=PD Chastain; citation_author=P Parniewski; citation_author=K Ohshima; citation_author=M Pandolfo; citation_journal_title=Mol Cell; citation_volume=3; citation_number=20; citation_pages=465-475; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=DNA triplet repeats mediate heterochromatin-protein-1-sensitive variegated gene silencing.; citation_author=A Saveliev; citation_author=C Everett; citation_author=T Sharpe; citation_author=Z Webster; citation_author=R Festenstein; citation_journal_title=Nature; citation_volume=422; citation_number=21; citation_pages=909-913; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Histone deacetylase inhibitors reverse gene silencing in Friedreich's ataxia.; citation_author=D Herman; citation_author=K Jenssen; citation_author=R Burnett; citation_author=E Soragni; citation_author=SL Perlman; citation_author=JM Gottesfeld; citation_journal_title=Nat Chem Biol; citation_volume=2; citation_number=22; citation_pages=551-558; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Therapeutics development for triplet repeat expansion diseases.; citation_author=NA Di Prospero; citation_author=KH Fischbeck; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=23; citation_pages=756-765; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Frataxin knockin mouse.; citation_author=CJ Miranda; citation_author=MM Santos; citation_author=K Ohshima; citation_author=J Smith; citation_author=L Li; citation_journal_title=FEBS Lett; citation_volume=512; citation_number=24; citation_pages=291-297; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Transcriptome analysis in Friedreich's ataxia identifies potential biomarkers related to treatment response.; citation_author=G Coppola; citation_author=R Burnett; citation_author=M Engelhardt; citation_author=MN Suberlak; citation_author=SL Perlman; citation_journal_title=Annals of Neurology; citation_volume=62; citation_number=25; citation_pages=S24-S24. (Abstr.); citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional genomic analysis of Friedreich's ataxia pathogenesis in vivo and in vitro.; citation_author=G Coppola; citation_author=M Engelhardt; citation_author=MN Suberlak; citation_author=EM Wexler; citation_author=MM Santos; citation_journal_title=Annals of Neurology; citation_volume=62; citation_number=26; citation_pages=S24-S24. (Abstr.); citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Limma: Linear models for microarray data.; citation_author=GK Smyth; citation_title=Bioinformatics and Computational Biology Solutions using R and Bioconductor; citation_number=27; citation_pages=397-420; citation_date=2005; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Translating the histone code.; citation_author=T Jenuwein; citation_author=CD Allis; citation_journal_title=Science; citation_volume=293; citation_number=28; citation_pages=1074-1080; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Relationship between histone H3 lysine 9 methylation, transcription repression, and heterochromatin protein 1 recruitment.; citation_author=MD Stewart; citation_author=J Li; citation_author=J Wong; citation_journal_title=Mol Cell Biol; citation_volume=25; citation_number=29; citation_pages=2525-2538; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rb interacts with histone deacetylase to repress transcription.; citation_author=RX Luo; citation_author=AA Postigo; citation_author=DC Dean; citation_journal_title=Cell; citation_volume=92; citation_number=30; citation_pages=463-473; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model"/>
    <meta name="twitter:description" content="BackgroundFriedreich ataxia, an autosomal recessive neurodegenerative and cardiac disease, is caused by abnormally low levels of frataxin, an essential mitochondrial protein. All Friedreich ataxia patients carry a GAA&sdot;TTC repeat expansion in the first intron of the frataxin gene, either in the homozygous state or in compound heterozygosity with other loss-of-function mutations. The GAA expansion inhibits frataxin expression through a heterochromatin-mediated repression mechanism. Histone modifications that are characteristic of silenced genes in heterochromatic regions occur at expanded alleles in cells from Friedreich ataxia patients, including increased trimethylation of histone H3 at lysine 9 and hypoacetylation of histones H3 and H4.Methodology/Principal FindingsBy chromatin immunoprecipitation, we detected the same heterochromatin marks in homozygous mice carrying a (GAA)230 repeat in the first intron of the mouse frataxin gene (KIKI mice). These animals have decreased frataxin levels and, by microarray analysis, show significant gene expression changes in several tissues. We treated KIKI mice with a novel histone deacetylase inhibitor, compound 106, which substantially increases frataxin mRNA levels in cells from Friedreich ataxia individuals. Treatment increased histone H3 and H4 acetylation in chromatin near the GAA repeat and restored wild-type frataxin levels in the nervous system and heart, as determined by quantitative RT-PCR and semiquantitative western blot analysis. No toxicity was observed. Furthermore, most of the differentially expressed genes in KIKI mice reverted towards wild-type levels.Conclusions/SignificanceLack of acute toxicity, normalization of frataxin levels and of the transcription profile changes resulting from frataxin deficiency provide strong support to a possible efficacy of this or related compounds in reverting the pathological process in Friedreich ataxia, a so far incurable neurodegenerative disease."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001958.g004"/>

  <meta property="og:title" content="HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4137'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=430'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=930&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001958">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001958" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001958&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Myriam Rai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Neurologie Expérimentale, Hôpital Erasme, Université Libre de Bruxelles (ULB), Brussels, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elisabetta Soragni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kai Jenssen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America
                </p>

                <p>
                    Current address: Merz Pharmaceutical GmbH, Frankfurt am Main, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ryan Burnett, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Herman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Giovanni Coppola, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Program in Neurogenetics, Department of Neurology, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel H. Geschwind, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Program in Neurogenetics, Department of Neurology, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joel M. Gottesfeld, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biology, The Scripps Research Institute, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Massimo Pandolfo
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:massimo.pandolfo@ulb.ac.be">massimo.pandolfo@ulb.ac.be</a></p>

                <p>Affiliation:
                  Laboratoire de Neurologie Expérimentale, Hôpital Erasme, Université Libre de Bruxelles (ULB), Brussels, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001958</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001958-g001" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001958-g002" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001958-g003" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001958-g004" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001958">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001958" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Friedreich ataxia, an autosomal recessive neurodegenerative and cardiac disease, is caused by abnormally low levels of frataxin, an essential mitochondrial protein. All Friedreich ataxia patients carry a GAA⋅TTC repeat expansion in the first intron of the frataxin gene, either in the homozygous state or in compound heterozygosity with other loss-of-function mutations. The GAA expansion inhibits frataxin expression through a heterochromatin-mediated repression mechanism. Histone modifications that are characteristic of silenced genes in heterochromatic regions occur at expanded alleles in cells from Friedreich ataxia patients, including increased trimethylation of histone H3 at lysine 9 and hypoacetylation of histones H3 and H4.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>By chromatin immunoprecipitation, we detected the same heterochromatin marks in homozygous mice carrying a (GAA)<sub>230</sub> repeat in the first intron of the mouse frataxin gene (KIKI mice). These animals have decreased frataxin levels and, by microarray analysis, show significant gene expression changes in several tissues. We treated KIKI mice with a novel histone deacetylase inhibitor, compound <strong>106</strong>, which substantially increases frataxin mRNA levels in cells from Friedreich ataxia individuals. Treatment increased histone H3 and H4 acetylation in chromatin near the GAA repeat and restored wild-type frataxin levels in the nervous system and heart, as determined by quantitative RT-PCR and semiquantitative western blot analysis. No toxicity was observed. Furthermore, most of the differentially expressed genes in KIKI mice reverted towards wild-type levels.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Lack of acute toxicity, normalization of frataxin levels and of the transcription profile changes resulting from frataxin deficiency provide strong support to a possible efficacy of this or related compounds in reverting the pathological process in Friedreich ataxia, a so far incurable neurodegenerative disease.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rai M, Soragni E, Jenssen K, Burnett R, Herman D, et al.  (2008) HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model. PLoS ONE 3(4):
          e1958.
            doi:10.1371/journal.pone.0001958</p><p><strong>Editor: </strong>Raphael Schiffmann, National Institutes of Health, United States of America</p><p><strong>Received:</strong> February 19, 2008; <strong>Accepted:</strong> March 2, 2008; <strong>Published:</strong> April 9, 2008</p><p><strong>Copyright:</strong> © 2008 Rai et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the GOFAR Foundation (Italy), the Friedreich Ataxia Research Alliance (USA), Ataxia UK, Ataxia Ireland, Fondazione CRT (Italy), the Belgian National Scientific Research Funds (FNRS), the Belgian Ministry of Scientific Policy (Program IAP6) to MP; from FARA, the National Ataxia Foundation (NAF), the National Institutes of Neurological Disorders and Stroke (NINDS) to JMG; and from NINDS to DG.</p><p><strong>Competing interests:</strong> MP and JMG are consultants for Repligen, Inc., that has licensed the described HDAC inhibitors for the Scripps Research Institute. The other authors have no competing interests.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Friedreich ataxia (FRDA, OMIM 229300) is an inherited recessive disorder characterized by progressive neurological disability and heart disease <a href="#pone.0001958-Pandolfo1">[1]</a>. Onset is usually in childhood, but it may vary from infancy to adulthood. Atrophy of sensory and cerebellar pathways causes ataxia, dysarthria, fixation instability, deep sensory loss and loss of tendon reflexes. Corticospinal degeneration leads to muscular weakness and extensor plantar responses. With progression, patients lose the ability to walk and become dependent for all activities. In some cases, visual loss and neurosensorial deafness further increase disability. A hypertrophic cardiomyopathy, present in most cases, may become symptomatic and even cause premature death. Other common problems include kyphoscoliosis, pes cavus, and, in 10 % of patients, diabetes mellitus <a href="#pone.0001958-Pandolfo1">[1]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>FRDA is caused by partial deficiency of the mitochondrial protein frataxin. Though the function of frataxin is still partly controversial, there is general agreement that it is involved in cellular iron homeostasis and that its deficiency results in multiple enzyme deficits, mitochondrial dysfunction and oxidative damage <a href="#pone.0001958-Pandolfo2">[2]</a>, <a href="#pone.0001958-Pandolfo3">[3]</a>. Frataxin binds ferrous iron through negatively charged amino acids on its surface <a href="#pone.0001958-ONeill1">[4]</a>, it promotes the mitochondrial synthesis of iron-containing molecules, in particular iron-sulfur clusters (ISCs) <a href="#pone.0001958-Chen1">[5]</a> and heme <a href="#pone.0001958-Yoon1">[6]</a>, and it controls the ability of iron to perform redox chemistry <a href="#pone.0001958-ONeill2">[7]</a>. Frataxin deficiency significantly affects ISC synthesis and results in reduced activities of several enzymes that require ISCs as prosthetic groups <a href="#pone.0001958-Rotig1">[8]</a>. Frataxin may also have a more general protective effect against oxidative stress and in determining antioxidant responses, even in the absence of excess iron.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Complete absence of frataxin is incompatible with life in higher organisms, as demonstrated by the embryonic lethality observed in systemic gene knock-out models <a href="#pone.0001958-Anderson1">[9]</a>–<a href="#pone.0001958-Ventura1">[12]</a> and by the eventual loss of cells targeted for frataxin gene deletion in conditional knock-out models <a href="#pone.0001958-Simon1">[13]</a>. The human disease is caused by the pathological hyperexpansion of a GAA⋅TTC repeat sequence, ranging from 60–1700 repeats, in the first intron of the frataxin (<em>FXN,</em> MIM: 606829, GeneID: 2395) gene that partially suppresses <em>FXN</em> gene expression <a href="#pone.0001958-Campuzano1">[14]</a>. This mutation is present at the homozygous state in most patients and in compound heterozygosity with a different loss-of-function mutation in a small minority of cases (about 5%). In FRDA patients, frataxin amounts vary between 5% and 30% of those of normal individuals, and are little more than 50% of normal in heterozygous FRDA carriers, who have no sign of disease <a href="#pone.0001958-Campuzano1">[14]</a>–<a href="#pone.0001958-Montermini1">[16]</a>. These findings suggest that restoring <em>FXN</em> gene expression in FRDA patients to heterozygote levels may substantially slow the course of the disease. In order to develop treatments to reduce or eliminate <em>FXN</em> transcriptional silencing, it is necessary to understand the underlying mechanisms. <em>In vitro</em> and in bacterial plasmids, pathological lengths of GAA repeats adopt a non-B, triple helical DNA structure that blocks and sequesters the advancing RNA polymerase <a href="#pone.0001958-Grabczyk1">[17]</a>–<a href="#pone.0001958-Sakamoto1">[20]</a>. The same repeats, when linked to a reporter gene in transgenic mice <a href="#pone.0001958-Saveliev1">[21]</a> and in cells from FRDA patients <a href="#pone.0001958-Herman1">[22]</a>, become associated with transcriptionally silent heterochromatin. Therefore, decondensing the chromatin structure at the GAA repeat expansion appears an appealing target for FRDA therapeutics. Since deacetylated histones are generally associated with silent heterochromatin, HDAC inhibitors (HDACI) have the potential to make heterochromatin revert to an open, active conformation that allows gene expression <a href="#pone.0001958-DiProspero1">[23]</a>. In a previous study, a substantial increase in frataxin in lymphoblastoid cells and in primary, non-replicating lymphocytes from FRDA patients was only obtained when using a specific class of HDACI, analogs of the compound BML-210 (<em>N<sup>1</sup></em>-(2-aminophenyl)-N<sup>8</sup>-phenyloctanediamide), including the highly active compound <strong>4b</strong> (<em>N</em><sup>1</sup>-(2-aminophenyl)-<em>N</em><sup>7</sup>-phenylheptanediamide) <a href="#pone.0001958-Herman1">[22]</a>. <strong>4b</strong> and related compounds raised frataxin levels to and above those of carriers, under conditions without apparent toxicity. The increase in <em>FXN</em> transcription is accompanied by increased acetylation of histone H3 at lysine14 (H3K14), as well as at H4K5 and H4K12 near the GAA repeat <a href="#pone.0001958-Herman1">[22]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>As a further step to evaluate these drugs as potential FRDA therapeutics, we have now investigated the efficacy and acute toxicity of a compound from this novel family of HDACI in a FRDA mouse model. Our data indicate that in a mouse model that carries expanded GAA repeats in the endogenous frataxin (<em>fxn</em>) gene, a member of this class of HDACI is able to restore frataxin levels and the gene expression profile to those of wild-type mice.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>fxn <sup>(GAA)230/(GAA)230</sup> (KIKI) mice recapitulate the genetic and epigenetic features of FRDA and have an altered gene expression profile as a consequence of mild frataxin deficiency</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We previously generated a knock-in mouse carrying a (GAA)<sub>230</sub> repeat in the first intron of the endogenous frataxin gene (<em>fxn</em>) <a href="#pone.0001958-Miranda1">[24]</a>, and we showed that homozyogous <em>fxn</em><sup>(GAA)230/(GAA)230</sup> (KIKI) mice have mildly but significantly lower frataxin levels than wild-type (WT) animals with the same strain background (C57Bl/6), as determined by densitometry of western blots and semiquantitave RT-PCR <a href="#pone.0001958-Miranda1">[24]</a>. In the present study, we confirmed by quantitative real-time RT-PCR the reduction in frataxin mRNA in KIKI brain, cerebellum and heart (<a href="#pone-0001958-g001">Figure 1A</a>). We then investigated the histone H3 and H4 post-translational modifications in chromatin near the GAA repeat by chromatin immunoprecipitation (ChIP). We found changes that correspond to those occurring in cells from FRDA patients <a href="#pone.0001958-Herman1">[22]</a>, though they are quantitatively less marked, as expected given the relatively short GAA repeats in KIKI mice compared to a typical FRDA patient (<a href="#pone-0001958-g001">Figure 1B</a>). Taken together, these findings validate the KIKI mouse as a model for the frataxin transcriptional silencing and corresponding epigenetic changes as observed in FRDA.</p>
<div class="figure" id="pone-0001958-g001"><div class="img"><a name="pone-0001958-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Frataxin mRNA levels and histone modifications on chromatin in the first intron of the frataxin gene in KIKI and WT mice.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) We determined frataxin mRNA levels by quantitative real-time RT-PCR in brain, heart and cerebellum of 8 weeks old WT and KIKI mice (n = 8). All values are normalized to RER1 and β2m mRNA levels and are relative to WT brain, set to 100. Error bars are SD. Data were generated in two independent experiments. (** <em>P</em>&lt;0.001). (B) Levels of H4K5, H4K8, H4K16, H3K14 acetylation and of H3K9 trimethylation in KIKI and WT mice brains (n = 8). We performed ChIP experiments with antibodies for murine histones H3 and H4 that carry each of the indicated modifications. We used primer pairs from the first intron of the mouse frataxin gene just upstream of the point of insertion of the GAA repeat in KIKI mice. Relative recovery, determined by quantitative real-time RT-PCR, is expressed in relation to GAPDH and the recovery in samples from WT animals is set to 100 for each antibody. Error bars are SEM. of four independent immunoprecipitations, each quantified in triplicate. (C) Histogram showing the number of genes differentially expressed in KIKI mouse brain (cerebral hemispheres), cerebellum and heart compared to WT mice. Genes with increased expression in KIKI mice are in red, genes with decresed expression in KIKI mice are in green. Only genes showing changes at P&lt;0.005 are counted.</p>
<span>doi:10.1371/journal.pone.0001958.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Though KIKI frataxin levels are not expected to cause an overt phenotype, as they even exceed the 50% expression found in asymptomatic gene knockout heterozygous mice <a href="#pone.0001958-Cosse1">[10]</a>, we investigated whether they cause changes in gene expression profiles in tissues that are affected in FRDA, such as the central nervous system (CNS) and heart. For this purpose, we performed high-density microarray studies to compare the global gene expression profile in the brain (cerebral hemispheres), cerebellum and heart of KIKI mice to wild-type mice. 615 unique genes were found to be differentially expressed in KIKI vs WT samples at <em>P</em>&lt;0.005 (<a href="#pone-0001958-g001">Figure 1C</a>). Fold changes were small in amplitude (within 2-fold changes). Complete gene lists are in <a href="#pone.0001958.s003">Table S1</a> and have been deposited in the GEO database (accession number GSE10745). KIKI mice therefore have a clear, reproducible molecular phenotype. The biological relevance of this phenotype is further indicated by the significant overlap with gene expression changes found in FRDA patients' lymphocytes and in more severely frataxin-deficient <em>fxn</em><sup>(GAA)230/-</sup> (KIKO) mice <a href="#pone.0001958-Coppola1">[25]</a>, <a href="#pone.0001958-Coppola2">[26]</a>. In the absence of any detectable pathology, and therefore of non-specific changes due to cell atrophy and degeneration, these gene expression modifications are likely to represent a direct compensatory or stress-related response to lower levels of frataxin that can be used as a biomarker to evaluate the effectiveness of attempts to restore normal frataxin levels.</p>


<h4>Effects of compound 106 on frataxin levels and histone modifications in KIKI mice</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Compound <strong>106</strong> (<em>N</em><sup>1</sup>-(2-aminophenyl)-<em>N</em><sup>7</sup>-<em>p</em>-tolylheptanediamide; see <a href="#pone-0001958-g002">Figure 2A</a> for structure and <a href="#pone.0001958.s002">Methods S1</a> for synthetic methods), a derivative of <strong>4b</strong>, was found to be highly active in increasing histone acetylation in cell culture and <em>FXN</em> mRNA levels in primary lymphocytes from FRDA patients (<a href="#pone-0001958-g002">Figure 2</a>, B and C). Moreover, the parent compound <strong>4b</strong> and other derivatives increased histone acetylation in the brains of WT mice when injected subcutaneously (data not shown). Based on these results, we next examined whether treatment of KIKI mice with <strong>106</strong> resulted in increased histone acetylation in the brain, and we estimated the duration of this effect. After three doses of 150 mg/Kg of <strong>106</strong> given subcutaneously once a day for three consecutive days, increased histone acetylation was evident in the brain at least for 24 hours after the last injection, but had completely disappeared after a week (<a href="#pone-0001958-g003">Figure 3A</a>). Next, a group of 12 KIKI mice received subcutaneous injections of 150 mg/kg of <strong>106</strong> every 24 hours for three consecutive days to evaluate the effects of the compound on <em>fxn</em> mRNA and frataxin protein levels and on histone modifications near the GAA repeat. No apparent animal toxicity was associated with <strong>106</strong> treatment. Tissues (brain, cerebellum, and heart) were processed for RNA and protein extraction and for chromatin immunoprecipitation (ChIP) experiments.</p>
<div class="figure" id="pone-0001958-g002"><div class="img"><a name="pone-0001958-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Effects of HDACI 106 on histone acetylation and <em>FXN</em> mRNA.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Structure of HDACI 106. (B) Histone H4 acetylation in normal fibroblast cells (GM08333 Coriell Cell Repositories, Camden, NJ) was measured by Western blot of total cell lysate after 24 hr incubation with indicated compounds. 9b (<em>N</em><sup>1</sup>-(2-methoxyphenyl)-<em>N</em><sup>7</sup>-phenylheptanediamide) has been described <a href="#pone.0001958-Herman1">[22]</a>. Compounds were dissolved in DMSO prior to addition to cell culture to give the final concentration indicated above at 0.1% DMSO. (C) <em>FXN</em> mRNA levels were measured in RNA from isolated PBMCs from carrier/patient pairs (1–3) after a 48 hour incubation with 106 or DMSO (at 0.1%). Quantitative real-time RT-PCR was used to determine relative <em>FXN</em> levels between each carrier/patient pair using GAPDH as a control housekeeping gene. Error is represented as the standard deviation of the mean from 3 determinations. Primary lymphocytes were obtained from donor blood from FRDA patients and their carrier relatives (under an approved Human Subjects Protocol, with appropriate informed consent) as described <a href="#pone.0001958-Herman1">[22]</a>.</p>
<span>doi:10.1371/journal.pone.0001958.g002</span></div><div class="figure" id="pone-0001958-g003"><div class="img"><a name="pone-0001958-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Effects of HDAC inhibitor 106 on frataxin mRNA and protein levels and on histone acetylation in KIKI mice.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>(A) The effect 106 on histone acetylation persists for at least 24 hours in KIKI mice. Mice were sacrificed, then extraction of brain acid-soluble nuclear proteins, SDS-PAGE, and western blotting with antibodies to total histone H3 and to acetylated histone H3 were performed. Sample in lane a is from the brain of a 106-treated mouse sacrificed after four hours, in lane b is from the brain of a vehicle-treated mouse, in lane c is from the brain of a 106-treated mouse sacrificed after 24 hours. (B) Frataxin mRNA levels in WT mice and in KIKI mice treated with either vehicle or HDAC inhibitor 106. Frataxin levels were determined by quantitative real-time RT-PCR relative to RER1 and β2m mRNAs, both unaffected by the HDACI. The WT frataxin mRNA level in each tissue was set to 100. Brain: n = 12. Cerebellum and Heart: n = 8. Data from 3 separate experiments were pooled and bars indicate SD. (** <em>P</em>&lt;0.001; * <em>P</em>&lt;0.05). (C) Western blot of frataxin in brain extracts of KIKI mice that had been treated with daily injections of 150 mg/Kg of 106 (lanes B5–B8) or vehicle (lanes B1–B4) for three days. Mice were sacrified 24 hours after the last injection. Actin was detected for normalization. (D) Densitometric analysis of the frataxin western blot shown in (C). (E) Levels of H4K5, H4K8, H4K16 and H3K14 acetylation of H3K9 trimethylation in KIKI mice treated with one subcutaneous injection per day of 150 mg/kg of 106 for 3 consecutive days compared to vehicle-treated KIKI littermates and WT (n = 6). We performed ChIP experiments with antibodies for murine histones H3 and H4 carrying each modification. Primer pairs corresponded to the first intron of the mouse frataxin gene just upstream of the point of insertion of the GAA repeat in KIKI mice. Relative recovery, determined by quantitative real-time PCR, is expressed in relation to GAPDH and the recovery in samples from WT animals is set to 100 for each antibody. Error bars are SEM of four independent immunoprecipitations, each quantified in triplicate.</p>
<span>doi:10.1371/journal.pone.0001958.g003</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p><em>Fxn</em> mRNA, quantified by quantitative real-time RT-PCR, was significantly lower in the brain, cerebellum and heart of vehicle-treated KIKI mice than in similarly treated WT animals. Treatment with <strong>106</strong> increased KIKI <em>fxn</em> mRNA to levels that did not significantly differ from WT (<a href="#pone-0001958-g003">Figure 3B</a>), thus demonstrating correction of <em>fxn</em> deciciency in these animals. We confirmed that increased <em>fxn</em> mRNA levels resulted in higher frataxin protein level by western blotting (<a href="#pone-0001958-g003">Figure 3</a>, C and D). Treatment with <strong>106</strong> did not result in increased <em>fxn</em> mRNA levels in WT animals (<a href="#pone.0001958.s001">Figure S1</a>), indicating that its effect was due to removal of the inhibition caused by the GAA expansion.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We then checked whether in KIKI mice, as observed in lymphocytes from FRDA patients, <strong>106</strong> treatment was accompanied by changes in histone modifications in chromatin near the GAA repeat that could lead to a more transcription-permissive state. After HDACI treatment, trimethylation of H3K9 remained high, as previously observed in FRDA cells <a href="#pone.0001958-Miranda1">[24]</a>, but acetylation levels increased at all other explored positions in histones H3 and H4 (<a href="#pone-0001958-g003">Figure 3E</a>).</p>


<h4>Effect of HDACI 106 on the global gene expression profile in the brain, cerebellum and heart of WT and KIKI mice</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p><a href="#pone-0001958-g004">Figure 4A</a> shows the effects of treatment with <strong>106</strong> on gene expression in the brain, cerebellum and heart of KIKI and WT mice evaluated by high-density microarray analysis <a href="#pone.0001958-Smyth1">[27]</a>. Complete gene lists are in <a href="#pone.0001958.s003">Table S1</a> and are available from the GEO database (accession number GSE10745). 670 probes (655 unique genes) were affected by <strong>106</strong> treatment in either tissue in either strain, showing small (less than two-fold) changes in expression. The relatively limited overall effect of <strong>106</strong> on expression profiles and the lack of major changes in apoptosis or tumorigenesis-related genes suggest that severe adverse effect from this compound are unlikely, at least at the utilized dosage.</p>
<div class="figure" id="pone-0001958-g004"><div class="img"><a name="pone-0001958-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001958" data-uri="info:doi/10.1371/journal.pone.0001958.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001958.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001958.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001958.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001958.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Effect of HDACI 106 on global gene expression profiles in the brain, cerebellum and heart of KIKI mice.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) 670 probes (655 unique genes) in total were affected by drug treatment in either tissue in either strain. A few drug-responsive genes were shared between the two strains. Quantification of the number of genes (among the 615 KIKI-related) that go in opposite direction after drug treatment show that 70–85% genes change but only in KIKI samples. In a subset of genes (mostly in cerebellum) this difference was statistically significant at <em>P</em>&lt;0.005. (B) Heatmaps representing the gene expression profiling before and after treatment with 106 in brain (left), cerebellum (center) and heart (right) of KIKI mice (four replicates per condition). Fold changes of the genes differentially expressed in KIKI vs. WT (brain = 61, cerebellum = 318, and heart = 182 genes at <em>P</em>&lt;0.005) are depicted before treatment (KIKI vs. WT, grey bars on the top) and after treatment (KIKI treated vs. KIKI untreated, blue bars on the top). Shades of red represent upregulation, shades of green downregulation. Genes and samples are clustered by similarity. (C) Most genes (67% in brain, 84% in cerebellum, 67% in heart) show coordinate changes towards normal levels after treatment. In a subset of these genes (34% in brain, 23% in cerebellum, 60% in heart) these changes reach statistical significance.</p>
<span>doi:10.1371/journal.pone.0001958.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Despite some overlap, likely to reflect the general effects of this HDACI on gene expression, <strong>106</strong>-induced modifications of expression profiles significantly differed between WT and KIKI mice. The most striking finding concerned the set of genes that were differentially expressed in untreated KIKI vs. WT mice. Most of these genes (67% in brain, 84% in cerebellum, 67% in heart) showed coordinate changes towards normal levels after <strong>106</strong> treatment (<a href="#pone-0001958-g004">Figure 4B</a>). In a subset of these genes (34% in brain, 23% in cerebellum, 60% in heart at <em>P</em>&lt;0.005; <a href="#pone.0001958.s003">Table S1</a> and <a href="#pone-0001958-g004">Figure 4C</a>), these changes reached statistical significance. In WT mice, the same genes showed no significant change or minor changes in either direction. Therefore, treatment with <strong>106</strong> modified the global gene expression profile in KIKI animals to resemble the WT profile.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In the present study we have demonstrated the <em>in vivo</em> feasibility of a therapeutic approach to activate the <em>FXN</em> gene in a mouse model that recapitulates the genetic and epigenetic features of FRDA. Previous work has shown that <em>FXN</em> silencing in FRDA is likely to be the consequence of chromatin changes induced by the expanded intronic GAA repeat. Post-translational modifications of histone tails are thought to form a code, called the histone code <a href="#pone.0001958-Jenuwein1">[28]</a>, that affect gene expression by providing binding sites for proteins involved in controlling chromatin condensation and transcription. Increased trimethylation at H3K9 and decreased acetylation at H3K14, H4K5, H4K8, H4K12 and H4K16 constitute hallmarks of silent heterochromatin and are found immediately upstream and downstream of the GAA repat expansion in cells from FRDA patients <a href="#pone.0001958-Herman1">[22]</a>. KIKI mice have similar changes, indicating that they are a suitable model for <em>in vivo</em> testing of treatments to alter histone modifications that may restore frataxin levels in FRDA. We chose a novel HDACI, compound <strong>106</strong>, for testing in the animal model. <strong>106</strong> has been developed as an analog of the compound BML-210, the first HDACI shown to be effective in increasing acetylation levels at critical histone residues near the GAA repeat and in restoring frataxin levels in cultured cells from FRDA patients <a href="#pone.0001958-Herman1">[22]</a>. In contrast, other common potent HDACIs, such as as suberoylanilide hydroxamic acid (SAHA), suberoyl bis-hydroxamic acid (SBHA), trichostatin A (TSA), and valproic acid (VPA) do not increase <em>FXN</em> gene expression in cells from FRDA patients. The molecular basis for why these compounds are ineffective, as compared to the pimelic diphenylamides, exemplified by <strong>106</strong>, is currently under investigation.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We have established that <strong>106</strong> penetrates the blood-brain barrier and increases histone acetylation in the brain at a dose that causes no apparent toxicity in WT C57Bl/6 or in KIKI mice. This compound was able to restore normal frataxin levels in the central nervous system and heart of KIKI mice, tissues that are relevant targets as they are involved in FRDA pathology. As no effect on frataxin levels was observed in similarly treated WT mice, we conclude that <strong>106</strong> directly interferes with the transcriptional repression mechanism triggered by the GAA repeat, which is thought to involve the induction of transcriptionally silent heterochromatin. Accordingly, the typical histone marks of heterochromatic regions that are present near the GAA repeat in KIKI mice were partially removed by treatment with <strong>106</strong>. In particular, acetylation increased with treatment at several lysine residues in histones H3 and H4, but no decrease in H3K9 trimethylation occurred. We propose that increased acetylation of H3K14 and of K5, K8 and K16 on H4, results in a more open, transcription permissive chromatin state despite persisting H3K9 trimethylation, because it interferes with binding of repressive proteins that recognize the trimethylated H3K9 mark, such as heterochromatin protein 1 (HP1) <a href="#pone.0001958-Stewart1">[29]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Restoring frataxin expression represents an important step toward a treatment for FRDA if it is followed by functional recovery of affected cells. KIKI mice do not show overt pathology or abnormal behavior, but we identified changes in the overall gene expression profiles in relevant tissues that constitutes an observable, reproducible and biologically relevant phenotype as well as a biomarker to monitor the effectiveness of treatments. Remarkably, after <strong>106</strong> treatment gene expression profiles showed a clear trend toward normalization. This phenomenon cannot be considered a non-specific consequence of HDACI treatment, because the involved genes were not significantly modified in treated WT mice, whose frataxin levels also remained stable. Normalization of the transcription profile changes induced by lowered frataxin provides strong support to a possible efficacy of this or related compounds in reverting the pathological process in FRDA, at least as long as major cell loss has not occurred. Based on our results, potential therapeutics may be developed for FRDA, a so far incurable neurodegenerative disease.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Animal procedures</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>GAA knock-in mice were generated and genotyped as described <a href="#pone.0001958-Miranda1">[24]</a>. Age and gender matched WT littermates were used as controls. Mice were treated by subcutaneous daily injections with 150 mg/kg of HDACI <strong>106</strong> or its equivalent of vehicle, for 3 consecutive days. Brain, heart and skeletal muscle were recovered 24 hours after last injection. All procedures respected regulations and guidelines of the Belgian state and European Union and were approved by the local ethical committee (CEBEA).</p>


<h4>Quantitative real-time RT-PCR</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Total RNA from brain stem and/or cerebellum was extracted by RNeasy Lipid Tissue Mini Kit (Qiagen) as recommended by the manufacturer. RNA from heart was extracted by RNeasy Fibrous Tissue Mini Kit (Qiagen). All RNA samples were treated with RNase-Free DNase Set (Qiagen) and quantified afterwards by measuring the optical density (NanoDrop ND-1000 Spectrophotometer, NanoDrop Technologies). Quality and purity of some samples were analyzed by gel electrophoresis on RNA assay chips (Experion System and StdSens analysis kit, BioRad Laboratories). We performed one step quantitative real-time PCR PCR using MultiScribe Reverse Transcriptase with Power SYBR Green (both from Applied Biosystems) on the ABI 7500 Fast Real Time PCR System (Applied Biosystems). Primers used for <em>Fxn</em> were <span class="gene">5′- CCTGGCCGAGTTCTTTGAAG-3′</span> and <span class="gene">5′- GCCAGATTTGCTTGTTTGG-3′</span>. RNA was standardized relative to <em>RER1</em> and ®<strong>2m</strong> (β2 microglobulin), mRNA using qBase 1.3.4 (Jan Hellemans &amp; Jo Vandesompele). Data are normalized to the <em>Fxn</em> mRNA level in WT mice ( = 100%).</p>


<h4>Microarray</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Three tissues were studied (brain, cerebellum, heart) in 4 biological replicates, two genotypes (KIKI and WT), and two treatments (<strong>106</strong> and vehicle), for a total of 48 microarrays. Total RNA (200 ng) was amplified, biotinylated and hybridized on Illumina Mouse v1.1 Expression arrays as per manufacturer's protocol. Raw data was log transformed and checked for outliers. Inter-array Pearson correlation and clustering based on variance were used as quality-control measures. Quantile normalization was used, and analysis of differential expression was performed using a linear model fitting <a href="#pone.0001958-Smyth1">[27]</a>, at a threshold of <em>P</em>&lt;0.005 for statistical significance. Briefly, a linear model was fitted across the dataset, contrasts of interest were extracted, and differentially expressed genes for each contrast were selected using an empirical Bayes test statistic<a href="#pone.0001958-Smyth1">[27]</a>. Differentially expressed genes were classified according to gene ontology, using Bioconductor packages and online tools (DAVID/EASE, <a href="http://david.niaid.nih.gov/david/ease.htm">http://david.niaid.nih.gov/david/ease.ht​m</a>). Pathway analysis was carried out using Ingenuity Pathway Analysis (Ingenuity Systems, <a href="http://www.ingenuity.com">www.ingenuity.com</a>).</p>


<h4>Western blot analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Tissues were homogenized in T-PER tissue protein extraction reagent (Pierce) for total proteins extraction. Histones were purified by acid extraction as described in the protocols from Upstate. Primary antibodies were diluted in Odyssey blocking buffer (Li-Cor) for frataxin (Chemicon) and actin (Sigma) or in PBS for total and acetylated histones antibodies (Upstate). Infrared dye conjugated secondary antibodies (anti-rabbit IRdye800Cw and anti-mouse IRdye680 from Li-Cor) were used to detect and quantify the signal of frataxin/actin using a Li-Cor Odyssey imaging system. Horseradish Peroxidase (HRP) - conjugated secondary antibodies (Santa Cruz Biotechnology) were used to detect the signal of total and acetylated histones by chemiluminescence.</p>


<h4>Chromatin immunoprecipitation</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>ChIP was performed as described previously <a href="#pone.0001958-Luo1">[30]</a>. We adapted the technique for use on fresh brain tissue and were able to test 3 different residues per one hemisphere. For each immunoprecipitation, lysate was incubated with one of the following antibodies (Anti-acetyl-Histone H3 (Lys14): Upstate Biotechnology 07-353; Anti-acetyl-Histone H4 (Lys5): Upstate 07-327; Anti-Histone H4 (acetyl K8): Abcam ab-1760; Anti-acetyl-Histone H4 (Lys16): Upstate 07-329; Anti-trimethyl-Histone H3 (Lys9)<strong>:</strong> Upstate 07-442; Normal Rabbit IgG<strong>:</strong> Upstate 12-370. Immunoprecipitated samples were quantified by real-time PCR following the standard curve method. Primers used for the region upstream intronic GAA repeats of <em>Fxn</em> were <span class="gene">5′- ACGACAAAGTCTCCCACAGG-3′</span> and <span class="gene">5′- GTCCAACAAGGCTTGATTCC-3′</span>. GAPDH primers were: <span class="gene">5′-TGGGTGGAGTGTCCTTTATCC-3′</span> and <span class="gene">5′-TATGCCCGAGGACAATAAGG-3′</span>.</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Data are presented as mean±s.d. or s.e.m., or as percentages. The significance of the difference between groups was evaluated with the Student's <em>t</em>-test. <em>P</em>&lt;0.05 was considered significant. (* <em>P</em>&lt;0.05 unless otherwise noted).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001958"></div><a name="pone.0001958.s001" id="pone.0001958.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001958.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Effect of HDACI 106 on frataxin mRNA levels in WT C56Bl6 mice. Mice were treated with either vehicle (n = 7) or HDAC inhibitor 106 (n = 7). Frataxin mRNA levels were determined in brain (hemispheres), cerebellum and heart by quantitative real-time RT-PCR relative to RER1 and β 2m mRNAs, both unaffected by the HDACI. In each tissue, the frataxin mRNA level in vehicle-treated animals was set to 100. No changes were observed after 106 treatment.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001958.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB TIF)</p>
<a name="pone.0001958.s002" id="pone.0001958.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001958.s002">Methods S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Supplemental synthetic methods for compound 106.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001958.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
<a name="pone.0001958.s003" id="pone.0001958.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001958.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Microarray data</p>
<p class="siDoi">doi:10.1371/journal.pone.0001958.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(15.57 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to thank Dr Susan Perlman for providing blood samples from patients.We thank Ana Lopes Da Cruz for animal technical assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MP MR DG JG. Performed the experiments: MR GC ES KJ RB DH. Analyzed the data: MP MR GC DG JG. Contributed reagents/materials/analysis tools: ES KJ RB DH. Wrote the paper: MP MR DG JG.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001958-Pandolfo1" id="pone.0001958-Pandolfo1"></a>Pandolfo M (2006)  Friedreich ataxia. In: Wells RD, Ashizawa T, editors. Genetic Instabilities and Neurological Diseases. Elsevier-Academic Press. pp. 277–298.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001958-Pandolfo2" id="pone.0001958-Pandolfo2"></a>Pandolfo M (2002) Frataxin deficiency and mitochondrial dysfunction. Mitochondrion  2: 87–93.  <ul class="find" data-citedArticleID="1088406" data-doi="10.1016/s1567-7249(02)00039-9"><li><a href="http://dx.doi.org/10.1016/s1567-7249(02)00039-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frataxin+deficiency+and+mitochondrial+dysfunction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frataxin+deficiency+and+mitochondrial+dysfunction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001958-Pandolfo3" id="pone.0001958-Pandolfo3"></a>Pandolfo M (2007)  Friedreich ataxia and related RNA loss-of-function disorders. In: Waxman S, editor. Molecular Neurology. Elsevier. Chapter 18.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001958-ONeill1" id="pone.0001958-ONeill1"></a>O'Neill HA, Gakh O, Isaya G (2005) Supramolecular assemblies of human frataxin are formed via subunit-subunit interactions mediated by a non-conserved amino-terminal region. J Mol Biol  345: 433–439.  <ul class="find" data-citedArticleID="1088396" data-doi="10.1016/j.jmb.2004.10.074"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.10.074" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Supramolecular+assemblies+of+human+frataxin+are+formed+via+subunit-subunit+interactions+mediated+by+a+non-conserved+amino-terminal+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Supramolecular+assemblies+of+human+frataxin+are+formed+via+subunit-subunit+interactions+mediated+by+a+non-conserved+amino-terminal+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001958-Chen1" id="pone.0001958-Chen1"></a>Chen OS, Hemenway S, Kaplan J (2002) Inhibition of Fe-S cluster biosynthesis decreases mitochondrial iron export: evidence that Yfh1p affects Fe-S cluster synthesis. Proc Natl Acad Sci U S A  99: 12321–12326.  <ul class="find" data-citedArticleID="1088374" data-doi="10.1073/pnas.192449599"><li><a href="http://dx.doi.org/10.1073/pnas.192449599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+Fe-S+cluster+biosynthesis+decreases+mitochondrial+iron+export%3A+evidence+that+Yfh1p+affects+Fe-S+cluster+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+Fe-S+cluster+biosynthesis+decreases+mitochondrial+iron+export%3A+evidence+that+Yfh1p+affects+Fe-S+cluster+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001958-Yoon1" id="pone.0001958-Yoon1"></a>Yoon T, Cowan JA (2004) Frataxin-mediated iron delivery to ferrochelatase in the final step of heme biosynthesis. J Biol Chem  279: 25943–25946.  <ul class="find" data-citedArticleID="1088426" data-doi="10.1074/jbc.c400107200"><li><a href="http://dx.doi.org/10.1074/jbc.c400107200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frataxin-mediated+iron+delivery+to+ferrochelatase+in+the+final+step+of+heme+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frataxin-mediated+iron+delivery+to+ferrochelatase+in+the+final+step+of+heme+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001958-ONeill2" id="pone.0001958-ONeill2"></a>O'Neill HA, Gakh O, Park S, Cui J, Mooney SM, et al.  (2005) Assembly of human frataxin is a mechanism for detoxifying redox-active iron. Biochemistry  44: 537–545.  <ul class="find" data-citedArticleID="1088398" data-doi="10.1021/bi048459j"><li><a href="http://dx.doi.org/10.1021/bi048459j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembly+of+human+frataxin+is+a+mechanism+for+detoxifying+redox-active+iron." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembly+of+human+frataxin+is+a+mechanism+for+detoxifying+redox-active+iron.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001958-Rotig1" id="pone.0001958-Rotig1"></a>Rotig A, de Lonlay P, Chretien D, Foury F, Koenig M, et al.  (1997) Aconitase and mitochondrial iron-sulphur protein deficiency in Friedreich ataxia. Nat Genet  17: 215–217.  <ul class="find" data-citedArticleID="1088410" data-doi="10.1038/ng1097-215"><li><a href="http://dx.doi.org/10.1038/ng1097-215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aconitase+and+mitochondrial+iron-sulphur+protein+deficiency+in+Friedreich+ataxia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aconitase+and+mitochondrial+iron-sulphur+protein+deficiency+in+Friedreich+ataxia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001958-Anderson1" id="pone.0001958-Anderson1"></a>Anderson PR, Kirby K, Hilliker AJ, Phillips JP (2005) RNAi-mediated suppression of the mitochondrial iron chaperone, frataxin, in Drosophila. Hum Mol Genet  14: 3397–3405.  <ul class="find" data-citedArticleID="1088368" data-doi="10.1093/hmg/ddi367"><li><a href="http://dx.doi.org/10.1093/hmg/ddi367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi-mediated+suppression+of+the+mitochondrial+iron+chaperone%2C+frataxin%2C+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi-mediated+suppression+of+the+mitochondrial+iron+chaperone%2C+frataxin%2C+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001958-Cosse1" id="pone.0001958-Cosse1"></a>Cossée M, Puccio H, Gansmuller A, Koutnikova H, Dierich A, et al.  (2000) Inactivation of the Friedreich ataxia mouse gene leads to early embryonic lethality without iron accumulation. Hum Mol Genet  9: 1219–1226.  <ul class="find" data-citedArticleID="1088380" data-doi="10.1093/hmg/9.8.1219"><li><a href="http://dx.doi.org/10.1093/hmg/9.8.1219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+the+Friedreich+ataxia+mouse+gene+leads+to+early+embryonic+lethality+without+iron+accumulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+the+Friedreich+ataxia+mouse+gene+leads+to+early+embryonic+lethality+without+iron+accumulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001958-Vazzola1" id="pone.0001958-Vazzola1"></a>Vazzola V, Losa A, Soave C, Murgia I (2007) Knockout of frataxin gene causes embryo lethality in Arabidopsis. FEBS Lett  581: 667–672.  <ul class="find" data-citedArticleID="1088422" data-doi="10.1016/j.febslet.2007.01.030"><li><a href="http://dx.doi.org/10.1016/j.febslet.2007.01.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Knockout+of+frataxin+gene+causes+embryo+lethality+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Knockout+of+frataxin+gene+causes+embryo+lethality+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001958-Ventura1" id="pone.0001958-Ventura1"></a>Ventura N, Rea SL, Handerson ST, Condo I, Testi R, Johnson TE (2006) C. elegans as a model for Friedreich Ataxia. Faseb J  20: 1029–1030.  <ul class="find" data-citedArticleID="1088424" data-doi="10.1096/fj.06-0505ufm"><li><a href="http://dx.doi.org/10.1096/fj.06-0505ufm" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=C.+elegans+as+a+model+for+Friedreich+Ataxia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22C.+elegans+as+a+model+for+Friedreich+Ataxia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001958-Simon1" id="pone.0001958-Simon1"></a>Simon D, Seznec H, Gansmuller A, Carelle N, Weber P, et al.  (2004) Friedreich ataxia mouse models with progressive cerebellar and sensory ataxia reveal autophagic neurodegeneration in dorsal root ganglia. J Neurosci  24: 1987–1995.  <ul class="find" data-citedArticleID="1088416" data-doi="10.1523/jneurosci.4549-03.2004"><li><a href="http://dx.doi.org/10.1523/jneurosci.4549-03.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Friedreich+ataxia+mouse+models+with+progressive+cerebellar+and+sensory+ataxia+reveal+autophagic+neurodegeneration+in+dorsal+root+ganglia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Friedreich+ataxia+mouse+models+with+progressive+cerebellar+and+sensory+ataxia+reveal+autophagic+neurodegeneration+in+dorsal+root+ganglia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001958-Campuzano1" id="pone.0001958-Campuzano1"></a>Campuzano V, Montermini L, Moltò MD, Pianese L, Cossée M, et al.  (1996) Friedreich's ataxia: autosomal recessive disease caused by an intronic GAA triplet repeat expansion. Science  271: 1423–1427.  <ul class="find" data-citedArticleID="1088370" data-doi="10.1126/science.271.5254.1423"><li><a href="http://dx.doi.org/10.1126/science.271.5254.1423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Friedreich%27s+ataxia%3A+autosomal+recessive+disease+caused+by+an+intronic+GAA+triplet+repeat+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Friedreich%27s+ataxia%3A+autosomal+recessive+disease+caused+by+an+intronic+GAA+triplet+repeat+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001958-Campuzano2" id="pone.0001958-Campuzano2"></a>Campuzano V, Montermini L, Lutz Y, Cova L, Hindelang C, et al.  (1997) Frataxin is reduced in Friedreich ataxia patients and is associated with mitochondrial membranes. Hum Mol Genet  6: 1771–1780.  <ul class="find" data-citedArticleID="1088372" data-doi="10.1093/hmg/6.11.1771"><li><a href="http://dx.doi.org/10.1093/hmg/6.11.1771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frataxin+is+reduced+in+Friedreich+ataxia+patients+and+is+associated+with+mitochondrial+membranes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frataxin+is+reduced+in+Friedreich+ataxia+patients+and+is+associated+with+mitochondrial+membranes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001958-Montermini1" id="pone.0001958-Montermini1"></a>Montermini L, Richter A, Morgan K, Justice CM, Julien D, et al.  (1997) Phenotypic variability in Friedreich ataxia: role of the associated GAA triplet repeat expansion. Ann Neurol  41: 675–682.  <ul class="find" data-citedArticleID="1088394" data-doi="10.1002/ana.410410518"><li><a href="http://dx.doi.org/10.1002/ana.410410518" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+variability+in+Friedreich+ataxia%3A+role+of+the+associated+GAA+triplet+repeat+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+variability+in+Friedreich+ataxia%3A+role+of+the+associated+GAA+triplet+repeat+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001958-Grabczyk1" id="pone.0001958-Grabczyk1"></a>Grabczyk E, Usdin K (2000) The GAA*TTC triplet repeat expanded in Friedreich's ataxia impedes transcription elongation by T7 RNA polymerase in a length and supercoil dependent manner. Nucleic Acids Res  28: 2815–2822.  <ul class="find" data-citedArticleID="1088384" data-doi="10.1093/nar/28.14.2815"><li><a href="http://dx.doi.org/10.1093/nar/28.14.2815" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+GAA*TTC+triplet+repeat+expanded+in+Friedreich%27s+ataxia+impedes+transcription+elongation+by+T7+RNA+polymerase+in+a+length+and+supercoil+dependent+manner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+GAA*TTC+triplet+repeat+expanded+in+Friedreich%27s+ataxia+impedes+transcription+elongation+by+T7+RNA+polymerase+in+a+length+and+supercoil+dependent+manner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001958-Ohshima1" id="pone.0001958-Ohshima1"></a>Ohshima K, Kang S, Larson JE, Wells RD (1996) Cloning, characterization, and properties of seven triplet repeat DNA sequences. J Biol Chem  271: 16773–16783.  <ul class="find" data-citedArticleID="1088400" data-doi="10.1074/jbc.271.28.16773"><li><a href="http://dx.doi.org/10.1074/jbc.271.28.16773" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning%2C+characterization%2C+and+properties+of+seven+triplet+repeat+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning%2C+characterization%2C+and+properties+of+seven+triplet+repeat+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001958-Ohshima2" id="pone.0001958-Ohshima2"></a>Ohshima K, Montermini L, Wells RD, Pandolfo M (1998) Inhibitory effects of expanded GAA.TTC triplet repeats from intron I of the Friedreich ataxia gene on transcription and replication in vivo. J Biol Chem  273: 14588–14595.  <ul class="find" data-citedArticleID="1088402" data-doi="10.1074/jbc.273.23.14588"><li><a href="http://dx.doi.org/10.1074/jbc.273.23.14588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibitory+effects+of+expanded+GAA.TTC+triplet+repeats+from+intron+I+of+the+Friedreich+ataxia+gene+on+transcription+and+replication+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibitory+effects+of+expanded+GAA.TTC+triplet+repeats+from+intron+I+of+the+Friedreich+ataxia+gene+on+transcription+and+replication+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001958-Sakamoto1" id="pone.0001958-Sakamoto1"></a>Sakamoto N, Chastain PD, Parniewski P, Ohshima K, Pandolfo M, et al.  (1999) Sticky DNA: self-association properties of long GAA.TTC repeats in R.R.Y triplex structures from Friedreich's ataxia. Mol Cell  3: 465–475.  <ul class="find" data-citedArticleID="1088412"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sticky+DNA%3A+self-association+properties+of+long+GAA.TTC+repeats+in+R.R.Y+triplex+structures+from+Friedreich%27s+ataxia.&amp;auth=&amp;atitle=Sticky+DNA%3A+self-association+properties+of+long+GAA.TTC+repeats+in+R.R.Y+triplex+structures+from+Friedreich%27s+ataxia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sticky+DNA%3A+self-association+properties+of+long+GAA.TTC+repeats+in+R.R.Y+triplex+structures+from+Friedreich%27s+ataxia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sticky+DNA%3A+self-association+properties+of+long+GAA.TTC+repeats+in+R.R.Y+triplex+structures+from+Friedreich%27s+ataxia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001958-Saveliev1" id="pone.0001958-Saveliev1"></a>Saveliev A, Everett C, Sharpe T, Webster Z, Festenstein R (2003) DNA triplet repeats mediate heterochromatin-protein-1-sensitive variegated gene silencing. Nature  422: 909–913.  <ul class="find" data-citedArticleID="1088414" data-doi="10.1038/nature01596"><li><a href="http://dx.doi.org/10.1038/nature01596" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+triplet+repeats+mediate+heterochromatin-protein-1-sensitive+variegated+gene+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+triplet+repeats+mediate+heterochromatin-protein-1-sensitive+variegated+gene+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001958-Herman1" id="pone.0001958-Herman1"></a>Herman D, Jenssen K, Burnett R, Soragni E, Perlman SL, Gottesfeld JM (2006) Histone deacetylase inhibitors reverse gene silencing in Friedreich's ataxia. Nat Chem Biol  2: 551–558.  <ul class="find" data-citedArticleID="1088386" data-doi="10.1038/nchembio815"><li><a href="http://dx.doi.org/10.1038/nchembio815" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Histone+deacetylase+inhibitors+reverse+gene+silencing+in+Friedreich%27s+ataxia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Histone+deacetylase+inhibitors+reverse+gene+silencing+in+Friedreich%27s+ataxia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001958-DiProspero1" id="pone.0001958-DiProspero1"></a>Di Prospero NA, Fischbeck KH (2005) Therapeutics development for triplet repeat expansion diseases. Nat Rev Genet  6: 756–765.  <ul class="find" data-citedArticleID="1088382" data-doi="10.1038/nrg1690"><li><a href="http://dx.doi.org/10.1038/nrg1690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Therapeutics+development+for+triplet+repeat+expansion+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Therapeutics+development+for+triplet+repeat+expansion+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001958-Miranda1" id="pone.0001958-Miranda1"></a>Miranda CJ, Santos MM, Ohshima K, Smith J, Li L, et al.  (2002) Frataxin knockin mouse. FEBS Lett  512: 291–297.  <ul class="find" data-citedArticleID="1088392" data-doi="10.1016/s0014-5793(02)02251-2"><li><a href="http://dx.doi.org/10.1016/s0014-5793(02)02251-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Frataxin+knockin+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Frataxin+knockin+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001958-Coppola1" id="pone.0001958-Coppola1"></a>Coppola G, Burnett R, Engelhardt M, Suberlak MN, Perlman SL, et al.  (2007) Transcriptome analysis in Friedreich's ataxia identifies potential biomarkers related to treatment response. Annals of Neurology  62: S24–S24. (Abstr.).  <ul class="find" data-citedArticleID="1088376"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transcriptome+analysis+in+Friedreich%27s+ataxia+identifies+potential+biomarkers+related+to+treatment+response.&amp;auth=&amp;atitle=Transcriptome+analysis+in+Friedreich%27s+ataxia+identifies+potential+biomarkers+related+to+treatment+response." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+analysis+in+Friedreich%27s+ataxia+identifies+potential+biomarkers+related+to+treatment+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+analysis+in+Friedreich%27s+ataxia+identifies+potential+biomarkers+related+to+treatment+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001958-Coppola2" id="pone.0001958-Coppola2"></a>Coppola G, Engelhardt M, Suberlak MN, Wexler EM, Santos MM, et al.  (2007) Functional genomic analysis of Friedreich's ataxia pathogenesis in vivo and in vitro. Annals of Neurology  62: S24–S24. (Abstr.).  <ul class="find" data-citedArticleID="1088378"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+genomic+analysis+of+Friedreich%27s+ataxia+pathogenesis+in+vivo+and+in+vitro.&amp;auth=&amp;atitle=Functional+genomic+analysis+of+Friedreich%27s+ataxia+pathogenesis+in+vivo+and+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomic+analysis+of+Friedreich%27s+ataxia+pathogenesis+in+vivo+and+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomic+analysis+of+Friedreich%27s+ataxia+pathogenesis+in+vivo+and+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001958-Smyth1" id="pone.0001958-Smyth1"></a>Smyth GK (2005)  Limma: Linear models for microarray data. In: Gentleman VCR, Dudoit S, Irizarry R, Huber W, editors. Bioinformatics and Computational Biology Solutions using R and Bioconductor. New York: Springer. pp. 397–420.  <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0001958-Jenuwein1" id="pone.0001958-Jenuwein1"></a>Jenuwein T, Allis CD (2001) Translating the histone code. Science  293: 1074–1080.  <ul class="find" data-citedArticleID="1088388" data-doi="10.1126/science.1063127"><li><a href="http://dx.doi.org/10.1126/science.1063127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translating+the+histone+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translating+the+histone+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001958-Stewart1" id="pone.0001958-Stewart1"></a>Stewart MD, Li J, Wong J (2005) Relationship between histone H3 lysine 9 methylation, transcription repression, and heterochromatin protein 1 recruitment. Mol Cell Biol  25: 2525–2538.  <ul class="find" data-citedArticleID="1088420" data-doi="10.1128/mcb.25.7.2525-2538.2005"><li><a href="http://dx.doi.org/10.1128/mcb.25.7.2525-2538.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+histone+H3+lysine+9+methylation%2C+transcription+repression%2C+and+heterochromatin+protein+1+recruitment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+histone+H3+lysine+9+methylation%2C+transcription+repression%2C+and+heterochromatin+protein+1+recruitment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001958-Luo1" id="pone.0001958-Luo1"></a>Luo RX, Postigo AA, Dean DC (1998) Rb interacts with histone deacetylase to repress transcription. Cell  92: 463–473.  <ul class="find" data-citedArticleID="1088390" data-doi="10.1016/s0092-8674(00)80940-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80940-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rb+interacts+with+histone+deacetylase+to+repress+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rb+interacts+with+histone+deacetylase+to+repress+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.XML" value="70505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.PDF" value="268853"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g001.PNG_L" value="272930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g001.PNG_M" value="53122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g001.PNG_S" value="12009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g001.TIF" value="738038"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g001.PNG_I" value="51977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g002.PNG_L" value="259109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g002.PNG_M" value="75184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g002.PNG_S" value="8405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g002.TIF" value="516798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g002.PNG_I" value="27551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g003.PNG_L" value="125608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g003.PNG_M" value="36668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g003.PNG_S" value="8200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g003.TIF" value="213676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g003.PNG_I" value="12931"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g004.PNG_L" value="381195"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g004.PNG_M" value="101238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g004.PNG_S" value="11525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g004.TIF" value="710070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.g004.PNG_I" value="36911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.s001.TIF" value="91184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.s002.DOC" value="22528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001958.s003.XLS" value="15574528"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001958&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001958" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001958&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001958&volume=&issue=&title=HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model&author_name=Myriam%20Rai%2C%20Elisabetta%20Soragni%2C%20Kai%20Jenssen%2C%20Ryan%20Burnett%2C%20David%20Herman%2C%20Giovanni%20Coppola%2C%20Daniel%20H.%20Geschwind%2C%20Joel%20M.%20Gottesfeld%2C%20Massimo%20Pandolfo&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001958" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958&amp;t=HDAC%20Inhibitors%20Correct%20Frataxin%20Deficiency%20in%20a%20Friedreich%20Ataxia%20Mouse%20Model" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958&title=HDAC%20Inhibitors%20Correct%20Frataxin%20Deficiency%20in%20a%20Friedreich%20Ataxia%20Mouse%20Model&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958&amp;title=HDAC%20Inhibitors%20Correct%20Frataxin%20Deficiency%20in%20a%20Friedreich%20Ataxia%20Mouse%20Model" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001958&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'HDAC Inhibitors Correct Frataxin Deficiency in a Friedreich Ataxia Mouse Model';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001958';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20HDAC%20Inhibitors%20Correct%20Frataxin%20Deficiency%20in%20a%20Friedreich%20Ataxia%20Mouse%20Model http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001958" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001958" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acetylation%22" title="Search for articles in the subject area:'Acetylation'"><div class="flagText">Acetylation</div></a>
              <div data-categoryid="21455" data-articleid="26740"
                   data-categoryname="Acetylation"
                   class="flagImage" title="Flag 'Acetylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cerebellum%22" title="Search for articles in the subject area:'Cerebellum'"><div class="flagText">Cerebellum</div></a>
              <div data-categoryid="47647" data-articleid="26740"
                   data-categoryname="Cerebellum"
                   class="flagImage" title="Flag 'Cerebellum' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin+immunoprecipitation%22" title="Search for articles in the subject area:'Chromatin immunoprecipitation'"><div class="flagText">Chromatin immunoprecipitation</div></a>
              <div data-categoryid="40167" data-articleid="26740"
                   data-categoryname="Chromatin immunoprecipitation"
                   class="flagImage" title="Flag 'Chromatin immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26740"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heart%22" title="Search for articles in the subject area:'Heart'"><div class="flagText">Heart</div></a>
              <div data-categoryid="47373" data-articleid="26740"
                   data-categoryname="Heart"
                   class="flagImage" title="Flag 'Heart' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="26740"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Immunoprecipitation%22" title="Search for articles in the subject area:'Immunoprecipitation'"><div class="flagText">Immunoprecipitation</div></a>
              <div data-categoryid="19963" data-articleid="26740"
                   data-categoryname="Immunoprecipitation"
                   class="flagImage" title="Flag 'Immunoprecipitation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="26740"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4165'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1605'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8814&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
