

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001133"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001133" />

    <meta name="citation_title" content="Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse"/>
    <meta itemprop="name" content="Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse"/>

      <meta name="citation_author" content="Igor V. Makunin"/>
            <meta name="citation_author_institution" content="Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Michael Pheasant"/>
            <meta name="citation_author_institution" content="Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="Cas Simons"/>
            <meta name="citation_author_institution" content="Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia"/>
      <meta name="citation_author" content="John S. Mattick"/>
            <meta name="citation_author_institution" content="Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia"/>

    <meta name="citation_date" content="2007/11/7"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001133.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1133"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=MicroRNAs: genomics, biogenesis, mechanism, and function.; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=116; citation_number=1; citation_pages=281-297; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Small regulatory RNAs in mammals.; citation_author=JS Mattick; citation_author=IV Makunin; citation_journal_title=Hum Mol Genet; citation_volume=14; citation_number=2; citation_pages=R121-132; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs as oncogenes and tumor suppressors.; citation_author=B Zhang; citation_author=X Pan; citation_author=GP Cobb; citation_author=TA Anderson; citation_journal_title=Dev Biol; citation_volume=302; citation_number=3; citation_pages=1-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Investigation of miRNA biology by bioinformatic tools and impact of miRNAs in colorectal cancer - regulatory relationship of c-Myc and p53 with miRNAs.; citation_author=Y Xi; citation_author=JR Edwards; citation_author=J Ju; citation_journal_title=Cancer Informatics; citation_volume=3; citation_number=4; citation_pages=245-253; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNA biogenesis and cancer.; citation_author=RI Gregory; citation_author=R Shiekhattar; citation_journal_title=Cancer Res; citation_volume=65; citation_number=5; citation_pages=3509-3512; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNA genes are frequently located near mouse cancer susceptibility loci.; citation_author=C Sevignani; citation_author=GA Calin; citation_author=SC Nnadi; citation_author=M Shimizu; citation_author=RV Davuluri; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=104; citation_number=6; citation_pages=8017-8022; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers.; citation_author=GA Calin; citation_author=C Sevignani; citation_author=CD Dumitru; citation_author=T Hyslop; citation_author=E Noch; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=7; citation_pages=2999-3004; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNA expression profiles classify human cancers.; citation_author=J Lu; citation_author=G Getz; citation_author=EA Miska; citation_author=E Alvarez-Saavedra; citation_author=J Lamb; citation_journal_title=Nature; citation_volume=435; citation_number=8; citation_pages=834-838; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A microRNA polycistron as a potential human oncogene.; citation_author=L He; citation_author=JM Thomson; citation_author=MT Hemann; citation_author=E Hernando-Monge; citation_author=D Mu; citation_journal_title=Nature; citation_volume=435; citation_number=9; citation_pages=828-833; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=High incidence of proviral integrations in the Hoxa locus in a new model of E2a-PBX1-induced B-cell leukemia.; citation_author=J Bijl; citation_author=M Sauvageau; citation_author=A Thompson; citation_author=G Sauvageau; citation_journal_title=Genes Dev; citation_volume=19; citation_number=10; citation_pages=224-233; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of oncogenes collaborating with p27Kip1 loss by insertional mutagenesis and high-throughput insertion site analysis.; citation_author=HC Hwang; citation_author=CP Martins; citation_author=Y Bronkhorst; citation_author=E Randel; citation_author=A Berns; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=11; citation_pages=11293-11298; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of candidate cancer-causing genes in mouse brain tumors by retroviral tagging.; citation_author=FK Johansson; citation_author=J Brodd; citation_author=C Eklof; citation_author=M Ferletta; citation_author=G Hesselager; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=12; citation_pages=11334-11337; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide retroviral insertional tagging of genes involved in cancer in Cdkn2a-deficient mice.; citation_author=AH Lund; citation_author=G Turner; citation_author=A Trubetskoy; citation_author=E Verhoeven; citation_author=E Wientjens; citation_journal_title=Nat Genet; citation_volume=32; citation_number=13; citation_pages=160-165; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome-based identification of cancer genes by proviral tagging in mouse retrovirus-induced T-cell lymphomas.; citation_author=R Kim; citation_author=A Trubetskoy; citation_author=T Suzuki; citation_author=NA Jenkins; citation_author=NG Copeland; citation_journal_title=J Virol; citation_volume=77; citation_number=14; citation_pages=2056-2062; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High-throughput retroviral tagging to identify components of specific signaling pathways in cancer.; citation_author=H Mikkers; citation_author=J Allen; citation_author=P Knipscheer; citation_author=L Romeijn; citation_author=A Hart; citation_journal_title=Nat Genet; citation_volume=32; citation_number=15; citation_pages=153-159; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Tumor suppressor gene identification using retroviral insertional mutagenesis in Blm-deficient mice.; citation_author=T Suzuki; citation_author=K Minehata; citation_author=K Akagi; citation_author=NA Jenkins; citation_author=NG Copeland; citation_journal_title=EMBO J; citation_volume=25; citation_number=16; citation_pages=3422-3431; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=New genes involved in cancer identified by retroviral tagging.; citation_author=T Suzuki; citation_author=H Shen; citation_author=K Akagi; citation_author=HC Morse; citation_author=JD Malley; citation_journal_title=Nat Genet; citation_volume=32; citation_number=17; citation_pages=166-174; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Haploinsufficiency of Runx1/AML1 promotes myeloid features and leukaemogenesis in BXH2 mice.; citation_author=N Yamashita; citation_author=M Osato; citation_author=L Huang; citation_author=M Yanagida; citation_author=SC Kogan; citation_journal_title=Br J Haematol; citation_volume=131; citation_number=18; citation_pages=495-507; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=RTCGD: retroviral tagged cancer gene database.; citation_author=K Akagi; citation_author=T Suzuki; citation_author=RM Stephens; citation_author=NA Jenkins; citation_author=NG Copeland; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=19; citation_pages=D523-527; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The human genome browser at UCSC.; citation_author=WJ Kent; citation_author=CW Sugnet; citation_author=TS Furey; citation_author=KM Roskin; citation_author=TH Pringle; citation_journal_title=Genome Res; citation_volume=12; citation_number=20; citation_pages=996-1006; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Activation of an oncogenic microRNA cistron by provirus integration.; citation_author=CL Wang; citation_author=BB Wang; citation_author=G Bartha; citation_author=L Li; citation_author=N Channa; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=21; citation_pages=18680-18684; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Retroviral insertional mutagenesis identifies genes that collaborate with NUP98-HOXD13 during leukemic transformation.; citation_author=C Slape; citation_author=H Hartung; citation_author=YW Lin; citation_author=J Bies; citation_author=L Wolff; citation_journal_title=Cancer Res; citation_volume=67; citation_number=22; citation_pages=5148-5155; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genome-wide analysis of retroviral DNA integration.; citation_author=F Bushman; citation_author=M Lewinski; citation_author=A Ciuffi; citation_author=S Barr; citation_author=J Leipzig; citation_journal_title=Nat Rev Microbiol; citation_volume=3; citation_number=23; citation_pages=848-858; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=M Hou; citation_journal_title=Genome Res; citation_volume=15; citation_number=24; citation_pages=1034-1050; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microarray profiling of microRNAs reveals frequent coexpression with neighboring miRNAs and host genes.; citation_author=S Baskerville; citation_author=DP Bartel; citation_journal_title=RNA; citation_volume=11; citation_number=25; citation_pages=241-247; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=c-Myc-regulated microRNAs modulate E2F1 expression.; citation_author=KA O'Donnell; citation_author=EA Wentzel; citation_author=KI Zeller; citation_author=CV Dang; citation_author=JT Mendell; citation_journal_title=Nature; citation_volume=435; citation_number=26; citation_pages=839-843; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Retroviral activation of the mir-106a microRNA cistron in T lymphoma.; citation_author=AM Lum; citation_author=BB Wang; citation_author=L Li; citation_author=N Channa; citation_author=G Bartha; citation_journal_title=Retrovirology; citation_volume=4; citation_number=27; citation_pages=5; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs and chromosomal abnormalities in cancer cells.; citation_author=GA Calin; citation_author=CM Croce; citation_journal_title=Oncogene; citation_volume=25; citation_number=28; citation_pages=6202-6210; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transposon-free regions in mammalian genomes.; citation_author=C Simons; citation_author=M Pheasant; citation_author=IV Makunin; citation_author=JS Mattick; citation_journal_title=Genome Res; citation_volume=16; citation_number=29; citation_pages=164-172; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse"/>
    <meta name="twitter:description" content="BackgroundMicroRNAs (miRNAs) are short non-coding RNAs that regulate differentiation and development in many organisms and play an important role in cancer.Methodology/Principal FindingsUsing a public database of mapped retroviral insertion sites from various mouse models of cancer we demonstrate that MLV-derived retroviral inserts are enriched in close proximity to mouse miRNA loci. Clustered inserts from cancer-associated regions (Common Integration Sites, CIS) have a higher association with miRNAs than non-clustered inserts. Ten CIS-associated miRNA loci containing 22 miRNAs are located within 10 kb of known CIS insertions. Only one CIS-associated miRNA locus overlaps a RefSeq protein-coding gene and six loci are located more than 10 kb from any RefSeq gene. CIS-associated miRNAs on average are more conserved in vertebrates than miRNAs associated with non-CIS inserts and their human homologs are also located in regions perturbed in cancer. In addition we show that miRNA genes are enriched around promoter and/or terminator regions of RefSeq genes in both mouse and human.Conclusions/SignificanceWe provide a list of ten miRNA loci potentially involved in the development of blood cancer or brain tumors. There is independent experimental support from other studies for the involvement of miRNAs from at least three CIS-associated miRNA loci in cancer development."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001133.g002"/>

  <meta property="og:title" content="Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4019'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8361'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3292&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001133">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001133" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001133&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Igor V. Makunin
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Igor V. Makunin, Michael Pheasant</p>

              
              

                <p>Affiliation:
                  Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Pheasant
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Igor V. Makunin, Michael Pheasant</p>

              
              

                <p>Affiliation:
                  Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cas Simons, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John S. Mattick
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:j.mattick@imb.uq.edu.au">j.mattick@imb.uq.edu.au</a></p>

                <p>Affiliation:
                  Australian Research Council (ARC) Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, Queensland, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 07, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001133</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001133-g001" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001133-g002" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001133-t001" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001133-t002" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001133-t003" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001133-t004" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001133">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001133" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>MicroRNAs (miRNAs) are short non-coding RNAs that regulate differentiation and development in many organisms and play an important role in cancer.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using a public database of mapped retroviral insertion sites from various mouse models of cancer we demonstrate that MLV-derived retroviral inserts are enriched in close proximity to mouse miRNA loci. Clustered inserts from cancer-associated regions (Common Integration Sites, CIS) have a higher association with miRNAs than non-clustered inserts. Ten CIS-associated miRNA loci containing 22 miRNAs are located within 10 kb of known CIS insertions. Only one CIS-associated miRNA locus overlaps a RefSeq protein-coding gene and six loci are located more than 10 kb from any RefSeq gene. CIS-associated miRNAs on average are more conserved in vertebrates than miRNAs associated with non-CIS inserts and their human homologs are also located in regions perturbed in cancer. In addition we show that miRNA genes are enriched around promoter and/or terminator regions of RefSeq genes in both mouse and human.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We provide a list of ten miRNA loci potentially involved in the development of blood cancer or brain tumors. There is independent experimental support from other studies for the involvement of miRNAs from at least three CIS-associated miRNA loci in cancer development.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Makunin IV, Pheasant M, Simons C, Mattick JS (2007) Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse. PLoS ONE 2(11):
          e1133.
            doi:10.1371/journal.pone.0001133</p><p><strong>Academic Editor: </strong>Christopher Arendt, Sanofi-Aventis, United States of America</p><p><strong>Received:</strong> July 26, 2007; <strong>Accepted:</strong> October 15, 2007; <strong>Published:</strong> November 7, 2007</p><p><strong>Copyright:</strong> © 2007 Makunin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Australian Research Council. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>MicroRNAs (miRNAs) are short RNA molecules, ~22 nucleotides long, capable of performing regulatory functions. In particular, miRNAs can suppress translation by non-perfect pairing to 3′ UTRs and/or cause degradation of mRNAs in the case of a perfect match between the miRNA and target mRNA <a href="#pone.0001133-Bartel1">[1]</a>. It seems that miRNAs do not have any catalytic activity but rather act as sequence-specific guides for associated protein complexes which are responsible for translation suppression or degradation of mRNA <a href="#pone.0001133-Mattick1">[2]</a>. The number of known miRNAs is growing rapidly, and hundreds of verified miRNAs are annotated in human, mouse, and other organisms (miRBase, <a href="http://microrna.sanger.ac.uk">http://microrna.sanger.ac.uk</a>).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A range of observations point to a link between miRNAs and cancer, which is not surprising given their central role in many cellular and developmental processes (for reviews see <a href="#pone.0001133-Zhang1">[3]</a>–<a href="#pone.0001133-Gregory1">[5]</a>). A large number of human and mouse miRNAs have also been shown to be located in regions associated with cancer <a href="#pone.0001133-Sevignani1">[6]</a>, <a href="#pone.0001133-Calin1">[7]</a>. The expression of various miRNAs is altered in cancer and miRNA profiling can be used for precise cancer classification <a href="#pone.0001133-Lu1">[8]</a>. Ectopic expression of the <em>mir-17-19b</em> cluster accelerates tumor formation in mice and has been accordingly classified as a potential oncogene <a href="#pone.0001133-He1">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Retroviruses, such as the murine leukemia virus (MLV), can cause tumor formation in mammals. Proviral insertions may activate proto-oncogenes or lead to inactivation of tumor suppressor genes in the vicinity of the insertion sites. Retroviral integration sites can be determined in animals with cancer using inverse PCR or similar techniques, and mapped to the genome sequence. Regions harboring multiple insertion sites in close proximity to each other are the most obvious candidates for cause of cancer development and are often named Common Integration Sites (CISs). In general, candidates for tumor-suppressor genes or proto-oncogenes are selected from protein-coding genes based on proximity to CISs.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Numerous genome-wide screens have been undertaken in mouse to identify genes involved in carcinogenesis, especially hematopoietic tumors <a href="#pone.0001133-Bijl1">[10]</a>–<a href="#pone.0001133-Yamashita1">[18]</a>. Data obtained from various screens on different cancer models (including studies with genetically modified mice) has been compiled into the Retroviral Tagged Cancer Gene Database (RTCGD, <a href="http://rtcgd.ncifcrf.gov">http://rtcgd.ncifcrf.gov</a>) <a href="#pone.0001133-Akagi1">[19]</a> which also provides annotations for the UCSC genome browser <a href="#pone.0001133-Kent1">[20]</a>. In the RTCGD, the CISs are defined as regions containing two inserts located within 20 kb, 3 inserts within 50 kb, and four or more inserts within 100 kb in the same cancer model (see FAQ section of RTCGD on <a href="http://rtcgd.ncifcrf.gov">http://rtcgd.ncifcrf.gov</a>) (for details see <a href="#pone.0001133-Suzuki2">[17]</a>).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>However, some CISs do not map near any known or annotated protein-coding sequence. It was shown that insertions of retroviruses in the vicinity of the <em>mir-17-92</em> miRNA polycistron cause tumor formation and increase miRNA expression, indicating that retroviral mutagenesis can be a potent tool for discovery of oncogenic miRNAs <a href="#pone.0001133-Wang1">[21]</a>, <a href="#pone.0001133-Slape1">[22]</a>. Considering the emerging regulatory role of microRNAs in cell differentiation and cancer we analyzed the association between publicly available retroviral integration sites and known miRNA loci in the mouse genome. We found that miRNA loci are significantly enriched in the vicinity of CISs which suggests that some of these miRNA loci may also be considered as candidate proto-oncogenes or tumor-suppressor genes.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Murine miRNA loci associate with retroviral common integration sites</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We analyzed co-localization between mouse miRNAs and retroviral integration sites determined from mice that developed cancer. For this analysis we used the 363 miRNAs from the miRNA registry (<a href="http://microrna.sanger.ac.uk">http://microrna.sanger.ac.uk</a>) that have been mapped to 381 locations within the well-assembled fraction of the mouse genome (four miRNAs mapped to more than one location). The locations correspond to the genomic positions of miRNA precursor sequences. We used the RTCGD database containing 2373 retroviral integration sites within Common Integration Sites (CIS inserts) and 3119 retroviral integration sites mapped outside of CISs (non-CIS inserts). We excluded from our analysis integration sites of Sleeping Beauty transposons because non-CIS insertions of Sleeping Beauty are non-randomly distributed among the chromosomes: chromosomes 1, 4, 6 and 15 harbor more than half of all Sleeping Beauty non-CIS integration sites.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Using a local mirror of the UCSC genome browser we found that CIS inserts are located within 5 kb of 17 murine miRNAs and within 10 kb of 22 miRNAs. Examples of co-localization between miRNAs and CIS inserts are shown in <a href="#pone-0001133-g001">Fig. 1</a> and a full list of CIS-associated miRNAs is given in <a href="#pone-0001133-t001">Table 1</a>. Further increasing the distance (past 10 kb) did not result in a significant increase of miRNA numbers associated with CIS inserts (<a href="#pone-0001133-g002">Fig. 2</a>) indicating that the association between miRNAs and CIS inserts is maximal at short distances.</p>
<div class="figure" id="pone-0001133-g001"><div class="img"><a name="pone-0001133-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001133.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001133.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Examples of co-localization between miRNAs and CIS retroviral integration sites in the mouse genome.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Each panel represents 20 kb of genomic DNA. Blue triangles indicate retroviral integration sites, red ticks represent miRNAs, black boxes and lines show the position of spliced transcripts. (A) chr11:87,562,001–87,582,000. (B) chrX:48,977,001–48,997,000. (C) chr14:113,914,001–113,934,000. Genome assembly mm8.</p>
<span>doi:10.1371/journal.pone.0001133.g001</span></div><div class="figure" id="pone-0001133-g002"><div class="img"><a name="pone-0001133-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001133.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001133.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The number of miRNAs co-localized with retroviral integration sites.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Blue diamonds represent the number of miRNAs located at the indicated distance or less from CIS inserts, and red diamonds correspond to miRNAs located at the indicated distance or less from non-CIS inserts. The trendline for miRNAs associated with non-CIS inserts is shown as red dotted line.</p>
<span>doi:10.1371/journal.pone.0001133.g002</span></div><div class="figure" id="pone-0001133-t001"><div class="img"><a name="pone-0001133-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Retroviral CIS insertions within 10 kb of murine miRNAs.</span></strong></p><span>doi:10.1371/journal.pone.0001133.t001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We used a bootstrap simulation to estimate the statistical significance of the co-localization between retroviral integration sites and miRNAs (see <a href="#s4">Materials and Methods</a>). Because some miRNAs are clustered in the genome and hence distributed non-uniformly, we grouped miRNAs into loci by adding 5, 10, 20 or 30 kb to each side of the miRNA location and combining the overlapping regions. This grouping is necessary to maintain clustered and tandemly repeated miRNAs as single units (loci) during the bootstrap procedure. Regions of the same sizes were randomly placed on the mouse genome and cases of overlap with retroviral integration sites were counted. The number of miRNA loci that are located 10 kb or less from CIS inserts is approximately 5.5 times higher than that observed for randomly placed loci, and the probability of obtaining such a number by chance is estimated as 1.7×10<sup>−5</sup>. The enrichment declines with the length of the miRNA loci, and the probability of obtaining a similar overlap between miRNA loci and CIS inserts by chance is higher for longer distances (<a href="#pone-0001133-t002">Table 2</a>). The bootstrap data indicate that the strongest association between miRNA loci and CIS inserts is at short distances, up to 10 kb. In agreement with this, the number of CIS retroviral inserts near to individual miRNAs is also highly enriched at short distances, and the enrichment declines with distance.</p>
<div class="figure" id="pone-0001133-t002"><div class="img"><a name="pone-0001133-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Bootstrap analysis for co-localization between miRNAs and CIS inserts.</span></strong></p><span>doi:10.1371/journal.pone.0001133.t002</span></div>

<h4>Non-CIS inserts are enriched in the vicinity of miRNA loci</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We analyzed the co-localization between miRNAs and retroviral inserts mapped outside of CISs. These non-CIS inserts are non-clustered retroviral integration sites obtained in cancer screens. Their role (if any) in tumorigenesis is unclear and so these inserts are generally omitted from analysis. Low saturation in some cancer screens suggests that some non-CIS inserts might be located in regions involved in tumorigenesis, but on the other hand it is possible to speculate that some are just by-products of the cancer screen.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The number of miRNA located at a given distance from non-CIS inserts increases proportionally to the distance between the miRNA and non-CIS insert (R<sup>2</sup> = 0.9396), whereas the number of miRNAs associated with CIS inserts does not display such a strong linear dependence (<a href="#pone-0001133-g002">Fig. 2</a>) (R<sup>2</sup> = 0.7831). The association of miRNA with CIS inserts is better described by a logarithmic trendline with R<sup>2</sup> = 0.945 (see <a href="#s4">Materials and Methods</a>). The bootstrap simulation showed a significant enrichment for miRNA loci associated with non-CIS inserts, especially for distances less than 5 kb (<a href="#pone-0001133-t003">Table 3</a>). The number of non-CIS retroviral integration sites in the vicinity of miRNAs has a slightly higher enrichment than the enrichment for miRNA loci. Close examination of miRNA loci associated with non-CIS inserts revealed several loci with two independent non-CIS inserts located very close to each other. For example, among 13 miRNA loci (16 miRNAs) with non-CIS inserts within 10 kb, ten loci have a single insert, and three loci have two inserts. These closely located inserts were isolated from different cancer models, which is why these inserts were classified as non-CIS. We analyzed the distribution of distances between non-CIS inserts in the genome. There is significant increase in the number of non-CIS inserts located within 3 kb of each other, whereas the number of non-CIS insertions at longer distances is more or less uniform when measured in 1 kb bins. A total of 332 out of 3119 inserts outside of CISs are located within 3 kb of each other.</p>
<div class="figure" id="pone-0001133-t003"><div class="img"><a name="pone-0001133-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Bootstrap analysis for co-localization between miRNAs and all non-CIS retroviral integration sites.</span></strong></p><span>doi:10.1371/journal.pone.0001133.t003</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>We removed all non-CIS inserts located within 3 kb of each other and repeated the bootstrap analysis. Nevertheless, even this dataset shows approximately two-fold enrichment for miRNA loci associated with non-CIS inserts separated by 3 kb or more (<a href="#pone-0001133-t004">Table 4</a>). The enrichment is similar for all distances analyzed but the association at longer distances is statistically more significant.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Based on this bootstrap analysis we conclude that miRNA loci show the strongest association with CIS inserts at short distances (less than 10 kb). At distances less than 10 kb the enrichment of miRNA loci overlapping with CIS inserts is two times higher than the enrichment of miRNA loci overlapping with non-CIS inserts. At longer distances, such as 30 kb, miRNA loci show a similar association both with CIS and non-CIS inserts.</p>
<div class="figure" id="pone-0001133-t004"><div class="img"><a name="pone-0001133-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001133" data-uri="info:doi/10.1371/journal.pone.0001133.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001133.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001133.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001133.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001133.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Bootstrap analysis for co-localization between non-CIS inserts separated by more than 3 kb and miRNAs.</span></strong></p><span>doi:10.1371/journal.pone.0001133.t004</span></div>

<h4>MicroRNA loci are enriched around starts and ends of protein-coding genes</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>It is known that integration of murine leukemia viruses and MLV-derived vectors preferentially occurs around promoter regions <a href="#pone.0001133-Bushman1">[23]</a>. Indeed, out of 3119 non-CIS retroviral sites from RTCGD database, 1190 (38%) are located within 5 kb of annotated transcription start sites of RefSeq genes (occupying ~6.8% of the genome). This represents more than 5-fold enrichment, higher than the enrichment of non-CIS insertions around miRNA loci (<a href="#pone-0001133-t003">Tables 3</a> and <a href="#pone-0001133-t004">4</a>). Out of 2373 CIS inserts, 989 (42%) are located within 5 kb from annotated transcription start sites of RefSeq genes.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We analyzed the distribution of miRNA loci in the mouse genome with respect to RefSeq genes. Out of 381 miRNA locations, 155 (41%) overlap RefSeq Genes which occupy 32% of the mouse genome. Among these, 22 (6%) miRNAs overlap exons (2% of the genome), and 133 (35%) are located in introns (30% of the genome). Somewhat surprisingly, we found that miRNAs are enriched close to the start or the end of genes: 69 (18%) and 72 (19%) miRNAs are located within 5 kb of RefSeq gene transcription start or end sites, respectively (~2.6 and ~2.8 fold enrichment). In total, 105 (51%) miRNA locations are located within 5 kb either from the start, end, or both (&gt;3-fold enrichment). Moreover, miRNAs show slightly higher enrichment in regions where gene start sites are separated from gene end sites by less than 10 kb (data not shown).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>MicroRNAs associated with non-CIS inserts tend to be close to promoters of RefSeq genes while CIS-associated miRNAs tend to be distant from promoters. For example, 13 miRNA loci (16 miRNAs) have non-CIS inserts mapped within 10 kb. Out of these, 9 loci (69%) are located within 10 kb from RefSeq gene starts, while out of 10 CIS-associated miRNA loci only 4 are less than 10 kb from RefSeq gene starts. Six CIS-associated miRNA loci containing 17 miRNAs are located more than 10 kb away from promoters of RefSeq genes, indicating that the observed association between miRNAs and CISs is not explained by co-localization of miRNAs near genes. A similar tendency is observed for miRNA 5k loci (miRNAs within 5 kb of each other): out of 10 miRNA 5k loci with non-CIS RIS within 5 kb (<a href="#pone-0001133-t003">Table 3</a>), 7 overlap RefSeq gene starts. Out of 7 miRNA 5k loci with CIS RIS within 5k (<a href="#pone-0001133-t002">Table 2</a>), 3 overlap RefSeq gene starts.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Interestingly, human miRNAs are also enriched around transcription start or end sites of RefSeq genes. There are 543 annotated miRNAs in the human genome mapped to 474 unique miRNA precursor locations. Out of these 474 miRNA locations 108 (23%) are located within 5 kb of RefSeq gene transcription start or/and end sites (2.2-fold enrichment). We merged these 474 locations into 311 miRNA 5k loci by adding 5 kb on each side of the precursor and then created the base-pair-wise union (OR) of locations. Out of 311 miRNA 5k loci 92 (30%) overlap either transcription start or end sites of RefSeq genes, or both. These 92 loci contain 110 (23%) miRNA precursors. It seems that miRNA loci in the vicinity of transcription start or end sites contain less miRNA precursors than miRNA loci located farther from genes (1.2 and 1.7 miRNA precursors per loci, respectively).</p>


<h4>CIS-associated miRNAs are conserved and their human orthologs are located in cancer-associated regions</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>CIS-associated miRNAs have some common features. Most (21 out of 22) CIS-associated miRNAs are located outside of RefSeq protein-coding genes. The exception, <em>mir-135a-1</em>, is located in the 3′ UTR of the gene 6230410P16Rik. In contrast, 5 out of 16 miRNAs having non-CIS inserts within 10 kb are located within RefSeq genes. On average, CIS-associated miRNA loci contain more miRNAs than non-CIS associated miRNA loci (2.2 and 1.2 miRNA per locus, respectively).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>CIS-associated miRNAs tend to be more conserved than miRNAs associated with non-CIS inserts, both in alignments of multiple vertebrate species and in human and mouse pairwise comparisons. First, we used pre-calculated phastCons <a href="#pone.0001133-Siepel1">[24]</a> conservation scores based on 17 vertebrates for whole miRNA precursor sequences. The average conservation score for 22 CIS-associated miRNA is 0.941 versus 0.739 for 16 non-CIS-associated miRNA (see <a href="#s4">Materials and Methods</a> for details). Second, we compared the sequence identity between mouse miRNA precursors and their orthologous human sequences. CIS-associated mouse miRNA precursors have 96% identity with human sequences whereas non-CIS-associated miRNAs display 91% identity, slightly lower than the average identity level for all mouse miRNA precursors (92%). In addition, CIS-associated miRNA precursors have significantly less indels between mouse and human sequences.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Considering the high conservation of CIS-associated miRNAs we looked for the involvement of human homologs in cancer development. Human homologs of eight CIS-associated miRNA loci are located in fragile regions and regions involved in cancer <a href="#pone.0001133-Calin1">[7]</a> (<a href="#pone-0001133-t001">Table 1</a>). It has been shown that miRNAs are generally down-regulated in cancer <a href="#pone.0001133-Lu1">[8]</a>. We therefore compared the available data on tissue-specific expression of human miRNAs <a href="#pone.0001133-Baskerville1">[25]</a> and the type of cancer associated with the CISs (blood or brain cancer) located in the vicinity of homologous murine miRNAs. Eight CIS-associated miRNA loci are co-localized with insertions determined from various types of blood cancer (<a href="#pone-0001133-t001">Table 1</a>). Human expression data in 24 different human organs and cell types was available for orthologs of members of 7 these loci <a href="#pone.0001133-Baskerville1">[25]</a>. There is a high correspondence between the miRNA expression pattern and the type of cancer induced by retroviral insertion near these miRNAs. Five loci have human miRNA orthologs whose expression is highest in tissues associated with blood development such as bone marrow or thymus. Another miRNA, <em>miR-22</em>, is enriched in thymus although it is most abundant in skeletal muscles <a href="#pone.0001133-Baskerville1">[25]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Here we demonstrate that murine miRNAs are associated with CISs. The enrichment of miRNAs loci in proximity to CIS inserts is higher than the enrichment of miRNAs around non-clustered retroviral insertions located outside of CISs. All but one CIS-associated miRNA are located outside RefSeq genes and some of them may be classified as proto-oncogenes or tumor-suppressor genes. Indeed, the well-characterized oncogenic miRNA cluster <em>mir-17-92</em> <a href="#pone.0001133-He1">[9]</a>, <a href="#pone.0001133-ODonnell1">[26]</a> is associated with CIS inserts (<a href="#pone-0001133-t001">Table 1</a>). Ectopic expression of the mir-17–92 cluster accelerates tumor development in a mouse B-cell lymphoma model <a href="#pone.0001133-He1">[9]</a>. Another example is the <em>mir-106a</em> miRNA cistron which shows numerous retroviral integrations in a lymphocyte tumor screen: tumors containing inserts close to the <em>mir-106a</em> miRNA cluster exhibit up to a 20-fold higher expression of these miRNAs <a href="#pone.0001133-Lum1">[27]</a>. There is also some evidence of the involvement of <em>mir-142</em> in cancer development <a href="#pone.0001133-Calin2">[28]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Formally, we cannot exclude the possibility that insertions affect (distant) regulatory elements situated in <em>cis</em> to protein-coding genes, especially when the CISs occur relatively close to genes, e.g., in <em>HOX</em> clusters (<a href="#pone-0001133-g001">Fig. 1c</a>). However, an equally if not more plausible explanation is that retroviral insertions are causing changes in the expression of miRNA genes, especially in those cases where the insertions occur in close proximity to miRNAs, rather than affecting more distant protein-coding genes. For example, all five CIS insertions in region XqA5 are less than 5 kb from the three miRNAs listed but more than 120 kb from the assigned RIS gene <em>Gpc3</em>, and four out of seven CIS insertions in region 11qC are closer to <em>mir-142</em> than to the nearest assigned gene <em>Supt4h2</em>. It is also possible that that (some) miRNA genes have chromatin structure open to retroviral integration similar to promoter regions of protein coding genes. The CIS-associated miRNAs represent prospective candidates for further experimental studies in the context of cancer association.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It appears that miRNAs are enriched around the transcription start and/or end sites of protein-coding genes both in human and mouse. Considering the strong bias of MLV integrations around promoter regions <a href="#pone.0001133-Bushman1">[23]</a> it is possible to speculate that the observed enrichment of non-CIS retroviral insertions near miRNAs may be at least partially due to the preferential location of miRNAs around promoter regions. Interestingly, miRNAs located close to transcription start or end sites tend to be non-clustered indicating a different type of organization of these miRNA genes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We used miRBase release 9.0 (October 2006) which contains 373 mouse miRNAs, 363 of which were mapped to 381 locations within the Mouse Feb 2006 (mm8) genome assembly (Build 36 “essentially complete” assembly by NCBI).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The July 2006 version of the Retroviral Tagged Cancer Gene Database (RTCGD, <a href="http://rtcgd.ncifcrf.gov">http://rtcgd.ncifcrf.gov</a>) <a href="#pone.0001133-Akagi1">[19]</a> contains 2373 retroviral integration sites classified as belonging to Common Integration Sites (CISs), and 3119 retroviral integration sites classified as located outside of CISs.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>All analysis was done on a local mirror of the UCSC Genome Browser <a href="#pone.0001133-Kent1">[20]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Bootstrap analysis was done similarly to <a href="#pone.0001133-Simons1">[29]</a>. Briefly, miRNAs were merged into miRNA loci by adding 5, 10, 20 or 30 kb on each side with subsequent base-pair-wise union (OR) on the UCSC Genome Browser. The resulting loci were randomly placed on the mouse genome with mapped CIS inserts or inserts outside of CISs and any cases of overlap were counted. Genome assembly gaps were removed from the analysis. Overlaps between randomly placed loci were prohibited. For each bootstrap test we performed 10<sup>7</sup> iterations.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The conservation of mouse miRNAs was estimated using phastCons conservation scores <a href="#pone.0001133-Siepel1">[24]</a> for whole miRNA precursors. The phastCons conservation scores were based on mouse-centric alignments of 17 species and were obtained from the UCSC genome browser <a href="#pone.0001133-Kent1">[20]</a>. The average phastCons score for bases within each known miRNA was calculated using the UCSC hgWiggle utility and the -doStats flag. Mouse-human base-pair identity scores were calculated using pairwise genome alignments obtained from the UCSC genome browser and the UCSC utilities axtAndBed and axtCalcMatrix.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>The enrichment of miRNAs around transcription start or end sites was calculated as follows: all RefSeq gene transcription start and end sites were extracted from the genome browser. Annotations were created by adding 5 or 10 kb to every transcription start or end site. The enrichment was calculated as the ratio between the fraction of miRNAs overlapping these annotations and the fraction of the genome occupied by the corresponding annotations.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>The linear and logarithmic trendlines and R2 values were calculated using Microsoft Excel 2003. The linear trendline for miRNAs associated with non-CIS inserts was described by y = 1.2286x+6.3333 with R<sup>2</sup> = 0.9396. The linear trendline for miRNAs associated with CIS inserts was described by y = 0.3371x+17.6 and R<sup>2</sup> = 0.7831. The logarithmic trendline for miRNAs associated with CIS inserts was described by y = 5.2282Ln(x)+9.3527 with R<sup>2</sup> = 0.945.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank the Retroviral Tagged Cancer Gene Database (RTCGD) for their assistance in providing access to the mapped retroviral insertion positions. We would like to thank anonymous reviewers for constructive comments and suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JM IM. Performed the experiments: MP IM CS. Analyzed the data: MP IM CS. Contributed reagents/materials/analysis tools: MP. Wrote the paper: JM MP IM. Other: Laboratory Head: JM. Provided support and facilities: JM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001133-Bartel1" id="pone.0001133-Bartel1"></a>Bartel DP (2004) MicroRNAs: genomics, biogenesis, mechanism, and function. Cell  116: 281–297.  <ul class="find" data-citedArticleID="1013910" data-doi="10.1016/s0092-8674(04)00045-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00045-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001133-Mattick1" id="pone.0001133-Mattick1"></a>Mattick JS, Makunin IV (2005) Small regulatory RNAs in mammals. Hum Mol Genet  14: R121–132.  <ul class="find" data-citedArticleID="1013940" data-doi="10.1093/hmg/ddi101"><li><a href="http://dx.doi.org/10.1093/hmg/ddi101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Small+regulatory+RNAs+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Small+regulatory+RNAs+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001133-Zhang1" id="pone.0001133-Zhang1"></a>Zhang B, Pan X, Cobb GP, Anderson TA (2007) MicroRNAs as oncogenes and tumor suppressors. Dev Biol  302: 1–12.  <ul class="find" data-citedArticleID="1013964" data-doi="10.1016/j.ydbio.2006.08.028"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2006.08.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+as+oncogenes+and+tumor+suppressors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+as+oncogenes+and+tumor+suppressors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001133-Xi1" id="pone.0001133-Xi1"></a>Xi Y, Edwards JR, Ju J (2007) Investigation of miRNA biology by bioinformatic tools and impact of miRNAs in colorectal cancer - regulatory relationship of c-Myc and p53 with miRNAs. Cancer Informatics  3: 245–253.  <ul class="find" data-citedArticleID="1013960"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Investigation+of+miRNA+biology+by+bioinformatic+tools+and+impact+of+miRNAs+in+colorectal+cancer+-+regulatory+relationship+of+c-Myc+and+p53+with+miRNAs.&amp;auth=&amp;atitle=Investigation+of+miRNA+biology+by+bioinformatic+tools+and+impact+of+miRNAs+in+colorectal+cancer+-+regulatory+relationship+of+c-Myc+and+p53+with+miRNAs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigation+of+miRNA+biology+by+bioinformatic+tools+and+impact+of+miRNAs+in+colorectal+cancer+-+regulatory+relationship+of+c-Myc+and+p53+with+miRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigation+of+miRNA+biology+by+bioinformatic+tools+and+impact+of+miRNAs+in+colorectal+cancer+-+regulatory+relationship+of+c-Myc+and+p53+with+miRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001133-Gregory1" id="pone.0001133-Gregory1"></a>Gregory RI, Shiekhattar R (2005) MicroRNA biogenesis and cancer. Cancer Res  65: 3509–3512.  <ul class="find" data-citedArticleID="1013922" data-doi="10.1158/0008-5472.can-05-0298"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-0298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+biogenesis+and+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+biogenesis+and+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001133-Sevignani1" id="pone.0001133-Sevignani1"></a>Sevignani C, Calin GA, Nnadi SC, Shimizu M, Davuluri RV, et al.  (2007) MicroRNA genes are frequently located near mouse cancer susceptibility loci. Proc Natl Acad Sci USA  104: 8017–8022.  <ul class="find" data-citedArticleID="1013946" data-doi="10.1073/pnas.0702177104"><li><a href="http://dx.doi.org/10.1073/pnas.0702177104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+genes+are+frequently+located+near+mouse+cancer+susceptibility+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+genes+are+frequently+located+near+mouse+cancer+susceptibility+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001133-Calin1" id="pone.0001133-Calin1"></a>Calin GA, Sevignani C, Dumitru CD, Hyslop T, Noch E, et al.  (2004) Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. Proc Natl Acad Sci USA  101: 2999–3004.  <ul class="find" data-citedArticleID="1013918" data-doi="10.1073/pnas.0307323101"><li><a href="http://dx.doi.org/10.1073/pnas.0307323101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+microRNA+genes+are+frequently+located+at+fragile+sites+and+genomic+regions+involved+in+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+microRNA+genes+are+frequently+located+at+fragile+sites+and+genomic+regions+involved+in+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001133-Lu1" id="pone.0001133-Lu1"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al.  (2005) MicroRNA expression profiles classify human cancers. Nature  435: 834–838.  <ul class="find" data-citedArticleID="1013934" data-doi="10.1038/nature03702"><li><a href="http://dx.doi.org/10.1038/nature03702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+expression+profiles+classify+human+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+expression+profiles+classify+human+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001133-He1" id="pone.0001133-He1"></a>He L, Thomson JM, Hemann MT, Hernando-Monge E, Mu D, et al.  (2005) A microRNA polycistron as a potential human oncogene. Nature  435: 828–833.  <ul class="find" data-citedArticleID="1013924" data-doi="10.1038/nature03552"><li><a href="http://dx.doi.org/10.1038/nature03552" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+microRNA+polycistron+as+a+potential+human+oncogene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+microRNA+polycistron+as+a+potential+human+oncogene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001133-Bijl1" id="pone.0001133-Bijl1"></a>Bijl J, Sauvageau M, Thompson A, Sauvageau G (2005) High incidence of proviral integrations in the Hoxa locus in a new model of E2a-PBX1-induced B-cell leukemia. Genes Dev  19: 224–233.  <ul class="find" data-citedArticleID="1013914" data-doi="10.1101/gad.1268505"><li><a href="http://dx.doi.org/10.1101/gad.1268505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+incidence+of+proviral+integrations+in+the+Hoxa+locus+in+a+new+model+of+E2a-PBX1-induced+B-cell+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+incidence+of+proviral+integrations+in+the+Hoxa+locus+in+a+new+model+of+E2a-PBX1-induced+B-cell+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001133-Hwang1" id="pone.0001133-Hwang1"></a>Hwang HC, Martins CP, Bronkhorst Y, Randel E, Berns A, et al.  (2002) Identification of oncogenes collaborating with p27Kip1 loss by insertional mutagenesis and high-throughput insertion site analysis. Proc Natl Acad Sci USA  99: 11293–11298.  <ul class="find" data-citedArticleID="1013926" data-doi="10.1073/pnas.162356099"><li><a href="http://dx.doi.org/10.1073/pnas.162356099" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+oncogenes+collaborating+with+p27Kip1+loss+by+insertional+mutagenesis+and+high-throughput+insertion+site+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+oncogenes+collaborating+with+p27Kip1+loss+by+insertional+mutagenesis+and+high-throughput+insertion+site+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001133-Johansson1" id="pone.0001133-Johansson1"></a>Johansson FK, Brodd J, Eklof C, Ferletta M, Hesselager G, et al.  (2004) Identification of candidate cancer-causing genes in mouse brain tumors by retroviral tagging. Proc Natl Acad Sci USA  101: 11334–11337.  <ul class="find" data-citedArticleID="1013928" data-doi="10.1073/pnas.0402716101"><li><a href="http://dx.doi.org/10.1073/pnas.0402716101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+candidate+cancer-causing+genes+in+mouse+brain+tumors+by+retroviral+tagging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+candidate+cancer-causing+genes+in+mouse+brain+tumors+by+retroviral+tagging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001133-Lund1" id="pone.0001133-Lund1"></a>Lund AH, Turner G, Trubetskoy A, Verhoeven E, Wientjens E, et al.  (2002) Genome-wide retroviral insertional tagging of genes involved in cancer in Cdkn2a-deficient mice. Nat Genet  32: 160–165.  <ul class="find" data-citedArticleID="1013938" data-doi="10.1038/ng956"><li><a href="http://dx.doi.org/10.1038/ng956" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+retroviral+insertional+tagging+of+genes+involved+in+cancer+in+Cdkn2a-deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+retroviral+insertional+tagging+of+genes+involved+in+cancer+in+Cdkn2a-deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001133-Kim1" id="pone.0001133-Kim1"></a>Kim R, Trubetskoy A, Suzuki T, Jenkins NA, Copeland NG, et al.  (2003) Genome-based identification of cancer genes by proviral tagging in mouse retrovirus-induced T-cell lymphomas. J Virol  77: 2056–2062.  <ul class="find" data-citedArticleID="1013932" data-doi="10.1128/jvi.77.3.2056-2062.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.3.2056-2062.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-based+identification+of+cancer+genes+by+proviral+tagging+in+mouse+retrovirus-induced+T-cell+lymphomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-based+identification+of+cancer+genes+by+proviral+tagging+in+mouse+retrovirus-induced+T-cell+lymphomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001133-Mikkers1" id="pone.0001133-Mikkers1"></a>Mikkers H, Allen J, Knipscheer P, Romeijn L, Hart A, et al.  (2002) High-throughput retroviral tagging to identify components of specific signaling pathways in cancer. Nat Genet  32: 153–159.  <ul class="find" data-citedArticleID="1013942" data-doi="10.1038/ng950"><li><a href="http://dx.doi.org/10.1038/ng950" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+retroviral+tagging+to+identify+components+of+specific+signaling+pathways+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+retroviral+tagging+to+identify+components+of+specific+signaling+pathways+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001133-Suzuki1" id="pone.0001133-Suzuki1"></a>Suzuki T, Minehata K, Akagi K, Jenkins NA, Copeland NG (2006) Tumor suppressor gene identification using retroviral insertional mutagenesis in Blm-deficient mice. EMBO J  25: 3422–3431.  <ul class="find" data-citedArticleID="1013954" data-doi="10.1038/sj.emboj.7601215"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tumor+suppressor+gene+identification+using+retroviral+insertional+mutagenesis+in+Blm-deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tumor+suppressor+gene+identification+using+retroviral+insertional+mutagenesis+in+Blm-deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001133-Suzuki2" id="pone.0001133-Suzuki2"></a>Suzuki T, Shen H, Akagi K, Morse HC, Malley JD, et al.  (2002) New genes involved in cancer identified by retroviral tagging. Nat Genet  32: 166–174.  <ul class="find" data-citedArticleID="1013956" data-doi="10.1038/ng949"><li><a href="http://dx.doi.org/10.1038/ng949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+genes+involved+in+cancer+identified+by+retroviral+tagging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+genes+involved+in+cancer+identified+by+retroviral+tagging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001133-Yamashita1" id="pone.0001133-Yamashita1"></a>Yamashita N, Osato M, Huang L, Yanagida M, Kogan SC, et al.  (2005) Haploinsufficiency of Runx1/AML1 promotes myeloid features and leukaemogenesis in BXH2 mice. Br J Haematol  131: 495–507.  <ul class="find" data-citedArticleID="1013962" data-doi="10.1111/j.1365-2141.2005.05793.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2141.2005.05793.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haploinsufficiency+of+Runx1%2FAML1+promotes+myeloid+features+and+leukaemogenesis+in+BXH2+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haploinsufficiency+of+Runx1%2FAML1+promotes+myeloid+features+and+leukaemogenesis+in+BXH2+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001133-Akagi1" id="pone.0001133-Akagi1"></a>Akagi K, Suzuki T, Stephens RM, Jenkins NA, Copeland NG (2004) RTCGD: retroviral tagged cancer gene database. Nucleic Acids Res  32: D523–527.  <ul class="find" data-citedArticleID="1013908" data-doi="10.1093/nar/gkh013"><li><a href="http://dx.doi.org/10.1093/nar/gkh013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RTCGD%3A+retroviral+tagged+cancer+gene+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RTCGD%3A+retroviral+tagged+cancer+gene+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001133-Kent1" id="pone.0001133-Kent1"></a>Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, et al.  (2002) The human genome browser at UCSC. Genome Res  12: 996–1006.  <ul class="find" data-citedArticleID="1013930" data-doi="10.1101/gr.229102"><li><a href="http://dx.doi.org/10.1101/gr.229102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+genome+browser+at+UCSC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+genome+browser+at+UCSC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001133-Wang1" id="pone.0001133-Wang1"></a>Wang CL, Wang BB, Bartha G, Li L, Channa N, et al.  (2006) Activation of an oncogenic microRNA cistron by provirus integration. Proc Natl Acad Sci USA  103: 18680–18684.  <ul class="find" data-citedArticleID="1013958" data-doi="10.1073/pnas.0609030103"><li><a href="http://dx.doi.org/10.1073/pnas.0609030103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+of+an+oncogenic+microRNA+cistron+by+provirus+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+of+an+oncogenic+microRNA+cistron+by+provirus+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001133-Slape1" id="pone.0001133-Slape1"></a>Slape C, Hartung H, Lin YW, Bies J, Wolff L, et al.  (2007) Retroviral insertional mutagenesis identifies genes that collaborate with NUP98-HOXD13 during leukemic transformation. Cancer Res  67: 5148–5155.  <ul class="find" data-citedArticleID="1013952" data-doi="10.1158/0008-5472.can-07-0075"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-07-0075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+insertional+mutagenesis+identifies+genes+that+collaborate+with+NUP98-HOXD13+during+leukemic+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+insertional+mutagenesis+identifies+genes+that+collaborate+with+NUP98-HOXD13+during+leukemic+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001133-Bushman1" id="pone.0001133-Bushman1"></a>Bushman F, Lewinski M, Ciuffi A, Barr S, Leipzig J, et al.  (2005) Genome-wide analysis of retroviral DNA integration. Nat Rev Microbiol  3: 848–858.  <ul class="find" data-citedArticleID="1013916" data-doi="10.1038/nrmicro1263"><li><a href="http://dx.doi.org/10.1038/nrmicro1263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+analysis+of+retroviral+DNA+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+analysis+of+retroviral+DNA+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001133-Siepel1" id="pone.0001133-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res  15: 1034–1050.  <ul class="find" data-citedArticleID="1013948" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001133-Baskerville1" id="pone.0001133-Baskerville1"></a>Baskerville S, Bartel DP (2005) Microarray profiling of microRNAs reveals frequent coexpression with neighboring miRNAs and host genes. RNA  11: 241–247.  <ul class="find" data-citedArticleID="1013912" data-doi="10.1261/rna.7240905"><li><a href="http://dx.doi.org/10.1261/rna.7240905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+profiling+of+microRNAs+reveals+frequent+coexpression+with+neighboring+miRNAs+and+host+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+profiling+of+microRNAs+reveals+frequent+coexpression+with+neighboring+miRNAs+and+host+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001133-ODonnell1" id="pone.0001133-ODonnell1"></a>O'Donnell KA, Wentzel EA, Zeller KI, Dang CV, Mendell JT (2005) c-Myc-regulated microRNAs modulate E2F1 expression. Nature  435: 839–843.  <ul class="find" data-citedArticleID="1013944" data-doi="10.1038/nature03677"><li><a href="http://dx.doi.org/10.1038/nature03677" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=c-Myc-regulated+microRNAs+modulate+E2F1+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22c-Myc-regulated+microRNAs+modulate+E2F1+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001133-Lum1" id="pone.0001133-Lum1"></a>Lum AM, Wang BB, Li L, Channa N, Bartha G, et al.  (2007) Retroviral activation of the mir-106a microRNA cistron in T lymphoma. Retrovirology  4: 5.  <ul class="find" data-citedArticleID="1013936"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Retroviral+activation+of+the+mir-106a+microRNA+cistron+in+T+lymphoma.&amp;auth=&amp;atitle=Retroviral+activation+of+the+mir-106a+microRNA+cistron+in+T+lymphoma." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+activation+of+the+mir-106a+microRNA+cistron+in+T+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+activation+of+the+mir-106a+microRNA+cistron+in+T+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001133-Calin2" id="pone.0001133-Calin2"></a>Calin GA, Croce CM (2006) MicroRNAs and chromosomal abnormalities in cancer cells. Oncogene  25: 6202–6210.  <ul class="find" data-citedArticleID="1013920" data-doi="10.1038/sj.onc.1209910"><li><a href="http://dx.doi.org/10.1038/sj.onc.1209910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+and+chromosomal+abnormalities+in+cancer+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+and+chromosomal+abnormalities+in+cancer+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001133-Simons1" id="pone.0001133-Simons1"></a>Simons C, Pheasant M, Makunin IV, Mattick JS (2006) Transposon-free regions in mammalian genomes. Genome Res  16: 164–172.  <ul class="find" data-citedArticleID="1013950" data-doi="10.1101/gr.4624306"><li><a href="http://dx.doi.org/10.1101/gr.4624306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposon-free+regions+in+mammalian+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposon-free+regions+in+mammalian+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.XML" value="86139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.PDF" value="170717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g001.PNG_L" value="445189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g001.PNG_M" value="132986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g001.PNG_S" value="12161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g001.TIF" value="797346"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g001.PNG_I" value="49337"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g002.PNG_L" value="255437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g002.PNG_M" value="78666"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g002.PNG_S" value="9910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g002.TIF" value="546972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.g002.PNG_I" value="29268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t001.PNG_L" value="294102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t001.PNG_M" value="114387"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t001.PNG_S" value="15444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t001.TIF" value="1646294"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t001.PNG_I" value="51982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t002.PNG_L" value="127544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t002.PNG_M" value="71206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t002.PNG_S" value="10316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t002.TIF" value="611058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t002.PNG_I" value="21947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t003.PNG_L" value="69118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t003.PNG_M" value="38161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t003.PNG_S" value="8711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t003.TIF" value="408182"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t003.PNG_I" value="11825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t004.PNG_L" value="53421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t004.PNG_M" value="29947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t004.PNG_S" value="8104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t004.TIF" value="278476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001133.t004.PNG_I" value="9049"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001133&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001133" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001133&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001133&volume=&issue=&title=Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse&author_name=Igor%20V.%20Makunin%2C%20Michael%20Pheasant%2C%20Cas%20Simons%2C%20John%20S.%20Mattick&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001133" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133&amp;t=Orthologous%20MicroRNA%20Genes%20Are%20Located%20in%20Cancer-Associated%20Genomic%20Regions%20in%20Human%20and%20Mouse" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133&title=Orthologous%20MicroRNA%20Genes%20Are%20Located%20in%20Cancer-Associated%20Genomic%20Regions%20in%20Human%20and%20Mouse&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133&amp;title=Orthologous%20MicroRNA%20Genes%20Are%20Located%20in%20Cancer-Associated%20Genomic%20Regions%20in%20Human%20and%20Mouse" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001133&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Orthologous MicroRNA Genes Are Located in Cancer-Associated Genomic Regions in Human and Mouse';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001133';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Orthologous%20MicroRNA%20Genes%20Are%20Located%20in%20Cancer-Associated%20Genomic%20Regions%20in%20Human%20and%20Mouse http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001133" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001133" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="25100"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="25100"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Functional+genomics%22" title="Search for articles in the subject area:'Functional genomics'"><div class="flagText">Functional genomics</div></a>
              <div data-categoryid="17853" data-articleid="25100"
                   data-categoryname="Functional genomics"
                   class="flagImage" title="Flag 'Functional genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25100"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40541" data-articleid="25100"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25100"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="25100"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MicroRNA%22" title="Search for articles in the subject area:'MicroRNA'"><div class="flagText">MicroRNA</div></a>
              <div data-categoryid="19821" data-articleid="25100"
                   data-categoryname="MicroRNA"
                   class="flagImage" title="Flag 'MicroRNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4230'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=455'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7839&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
