

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001430"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001430" />

    <meta name="citation_title" content="Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans"/>
    <meta itemprop="name" content="Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans"/>

      <meta name="citation_author" content="Daniela Contu"/>
            <meta name="citation_author_institution" content="Laboratorio di Immunogenetica, Ospedale Microcitemico, Cagliari, Italy"/>
      <meta name="citation_author" content="Laura Morelli"/>
            <meta name="citation_author_institution" content="Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy"/>
      <meta name="citation_author" content="Federico Santoni"/>
            <meta name="citation_author_institution" content="Center for Advanced Studies, Research and Development in Sardinia (CRS4), Pula, Italy"/>
      <meta name="citation_author" content="Jamie W. Foster"/>
            <meta name="citation_author_institution" content="Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy"/>
      <meta name="citation_author" content="Paolo Francalacci"/>
            <meta name="citation_author_institution" content="Dipartimento di Zoologia e Genetica Evoluzionistica, Università di Sassari, Sassari, Italy"/>
      <meta name="citation_author" content="Francesco Cucca"/>
            <meta name="citation_author_institution" content="Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy"/>

    <meta name="citation_date" content="2008/1/9"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001430.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1430"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Favism and Thalassæmia in Sardinia and their Relationship to Malaria.; citation_author=M Siniscalco; citation_author=L Bernini; citation_author=B Latte; citation_author=AG Motulsky; citation_journal_title=Nature; citation_volume=190; citation_number=1; citation_pages=1179-1180; citation_date=1961; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of beta-thalassemia in the Sardinian population.; citation_author=MC Rosatelli; citation_author=A Dozy; citation_author=V Faa; citation_author=A Meloni; citation_author=R Sardu; citation_journal_title=Am J Hum Genet; citation_volume=50; citation_number=2; citation_pages=422-426; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of wilson disease in the Sardinian population–evidence of a founder effect.; citation_author=G Loudianos; citation_author=V Dessi; citation_author=M Lovicu; citation_author=A Angius; citation_author=A Figus; citation_journal_title=Hum Mutat; citation_volume=14; citation_number=3; citation_pages=294-303; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A common mutation in Sardinian autoimmune polyendocrinopathy- candidiasis-ectodermal dystrophy patients.; citation_author=MC Rosatelli; citation_author=A Meloni; citation_author=M Devoto; citation_author=A Cao; citation_author=HS Scott; citation_journal_title=Hum Genet; citation_volume=103; citation_number=4; citation_pages=428-434; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A review of the recent epidemiological data on the worldwide incidence of type 1 (insulin-dependent) diabetes mellitus. World Health Organization DIAMOND Project Group.; citation_author=M Karvonen; citation_author=J Tuomilehto; citation_author=I Libman; citation_author=R La Porte; citation_journal_title=Diabetologia; citation_volume=36; citation_number=5; citation_pages=883-892; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The epidemiology of multiple sclerosis in Europe.; citation_author=M Pugliatti; citation_author=G Rosati; citation_author=H Carton; citation_author=T Riise; citation_author=J Drulovic; citation_journal_title=Eur J Neurol; citation_volume=13; citation_number=6; citation_pages=700-722; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=La civiltà nuragica.; citation_author=G Lilliu; citation_number=7; citation_date=1982; citation_publisher=Delfino; " />
      <meta name="citation_reference" content="citation_title=The inter-regional distribution of HLA class II haplotypes indicates the suitability of the Sardinian population for case-control association studies in complex diseases.; citation_author=R Lampis; citation_author=L Morelli; citation_author=M Congia; citation_author=MD Macis; citation_author=A Mulargia; citation_journal_title=Hum Mol Genet; citation_volume=9; citation_number=8; citation_pages=2959-2965; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=HLA structure of the Sardinian population: a haplotype study of 551 families.; citation_author=L Contu; citation_author=M Arras; citation_author=C Carcassi; citation_author=G La Nasa; citation_author=M Mulargia; citation_journal_title=Tissue Antigens; citation_volume=40; citation_number=9; citation_pages=165-174; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=From surnames to the history of Y chromosomes: the Sardinian population as a paradigm.; citation_author=G Zei; citation_author=A Lisa; citation_author=O Fiorani; citation_author=C Magri; citation_author=L Quintana-Murci; citation_journal_title=Eur J Hum Genet; citation_volume=11; citation_number=10; citation_pages=802-807; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Human Y-chromosome variation in the western Mediterranean area: implications for the peopling of the region.; citation_author=R Scozzari; citation_author=F Cruciani; citation_author=A Pangrazio; citation_author=P Santolamazza; citation_author=G Vona; citation_journal_title=Hum Immunol; citation_volume=62; citation_number=11; citation_pages=871-884; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis of Sardinia: I. data on 12 polymorphisms in 21 linguistic domains.; citation_author=N Cappello; citation_author=S Rendine; citation_author=R Griffo; citation_author=GE Mameli; citation_author=V Succa; citation_journal_title=Ann Hum Genet; citation_volume=60 ( Pt 2); citation_number=12; citation_pages=125-141; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Striking differentiation of sub-populations within a genetically homogeneous isolate (Ogliastra) in Sardinia as revealed by mtDNA analysis.; citation_author=C Fraumene; citation_author=E Petretto; citation_author=A Angius; citation_author=M Pirastu; citation_journal_title=Hum Genet; citation_volume=114; citation_number=13; citation_pages=1-10; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The genetic legacy of Paleolithic Homo sapiens sapiens in extant Europeans: a Y chromosome perspective.; citation_author=O Semino; citation_author=G Passarino; citation_author=PJ Oefner; citation_author=AA Lin; citation_author=S Arbuzova; citation_journal_title=Science; citation_volume=290; citation_number=14; citation_pages=1155-1159; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Y chromosome sequence variation and the history of human populations.; citation_author=PA Underhill; citation_author=P Shen; citation_author=AA Lin; citation_author=L Jin; citation_author=G Passarino; citation_journal_title=Nat Genet; citation_volume=26; citation_number=15; citation_pages=358-361; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=High-resolution analysis of human Y-chromosome variation shows a sharp discontinuity and limited gene flow between northwestern Africa and the Iberian Peninsula.; citation_author=E Bosch; citation_author=F Calafell; citation_author=D Comas; citation_author=PJ Oefner; citation_author=PA Underhill; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=16; citation_pages=1019-1029; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The Eurasian heartland: a continental perspective on Y-chromosome diversity.; citation_author=RS Wells; citation_author=N Yuldasheva; citation_author=R Ruzibakiev; citation_author=PA Underhill; citation_author=I Evseeva; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=17; citation_pages=10244-10249; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Sex-related bias and exclusion mapping of the nonrecombinant portion of chromosome Y in human type 1 diabetes in the isolated founder population of Sardinia.; citation_author=D Contu; citation_author=L Morelli; citation_author=P Zavattari; citation_author=R Lampis; citation_author=E Angius; citation_journal_title=Diabetes; citation_volume=51; citation_number=18; citation_pages=3573-3576; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Peopling of three Mediterranean islands (Corsica, Sardinia, and Sicily) inferred by Y-chromosome biallelic variability.; citation_author=P Francalacci; citation_author=L Morelli; citation_author=PA Underhill; citation_author=AS Lillie; citation_author=G Passarino; citation_journal_title=Am J Phys Anthropol; citation_volume=121; citation_number=19; citation_pages=270-279; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Excavating Y-chromosome haplotype strata in Anatolia.; citation_author=C Cinnioglu; citation_author=R King; citation_author=T Kivisild; citation_author=E Kalfoglu; citation_author=S Atasoy; citation_journal_title=Hum Genet; citation_volume=114; citation_number=20; citation_pages=127-148; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A Y chromosome census of the British Isles.; citation_author=C Capelli; citation_author=N Redhead; citation_author=JK Abernethy; citation_author=F Gratrix; citation_author=JF Wilson; citation_journal_title=Curr Biol; citation_volume=13; citation_number=21; citation_pages=979-984; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=High level of male-biased Scandinavian admixture in Greenlandic Inuit shown by Y-chromosomal analysis.; citation_author=E Bosch; citation_author=F Calafell; citation_author=ZH Rosser; citation_author=S Norby; citation_author=N Lynnerup; citation_journal_title=Hum Genet; citation_volume=112; citation_number=22; citation_pages=353-363; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phylogeography of Y-chromosome haplogroup I reveals distinct domains of prehistoric gene flow in europe.; citation_author=S Rootsi; citation_author=C Magri; citation_author=T Kivisild; citation_author=G Benuzzi; citation_author=H Help; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=23; citation_pages=128-137; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Paternal and maternal lineages in the Balkans show a homogeneous landscape over linguistic barriers, except for the isolated Aromuns.; citation_author=E Bosch; citation_author=F Calafell; citation_author=A Gonzalez-Neira; citation_author=C Flaiz; citation_author=E Mateu; citation_journal_title=Ann Hum Genet; citation_volume=70; citation_number=24; citation_pages=459-487; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The human colonization of Sardinia: a Late-Pleistocene human fossil from Corbeddu cave.; citation_author=PY Sondaar; citation_author=R Elburg; citation_author=G Klein-Hofmeijer; citation_author=F Martini; citation_author=M Sanges; citation_journal_title=Comptes Rendus de l'Académie des Sciences Paris; citation_volume=320; citation_number=25; citation_pages=145-150; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Palaeolithic Sardinians: paleontological evidence and methods.; citation_author=PY Sondaar; citation_number=26; citation_pages=45-51; citation_date=1998; citation_publisher=Oxbow Books; " />
      <meta name="citation_reference" content="citation_title=A signal, from human mtDNA, of postglacial recolonization in Europe.; citation_author=A Torroni; citation_author=HJ Bandelt; citation_author=V Macaulay; citation_author=M Richards; citation_author=F Cruciani; citation_journal_title=Am J Hum Genet; citation_volume=69; citation_number=27; citation_pages=844-852; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The molecular dissection of mtDNA haplogroup H confirms that the Franco-Cantabrian glacial refuge was a major source for the European gene pool.; citation_author=A Achilli; citation_author=C Rengo; citation_author=C Magri; citation_author=V Battaglia; citation_author=A Olivieri; citation_journal_title=Am J Hum Genet; citation_volume=75; citation_number=28; citation_pages=910-918; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regional differences among the Finns: a Y-chromosomal perspective.; citation_author=T Lappalainen; citation_author=S Koivumaki; citation_author=E Salmela; citation_author=K Huoponen; citation_author=P Sistonen; citation_journal_title=Gene; citation_volume=376; citation_number=29; citation_pages=207-215; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An Icelandic example of the impact of population structure on association studies.; citation_author=A Helgason; citation_author=B Yngvadottir; citation_author=B Hrafnkelsson; citation_author=J Gulcher; citation_author=K Stefansson; citation_journal_title=Nat Genet; citation_volume=37; citation_number=30; citation_pages=90-95; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Not all isolates are equal: linkage disequilibrium analysis on Xq13.3 reveals different patterns in Sardinian sub-populations.; citation_author=A Angius; citation_author=D Bebbere; citation_author=E Petretto; citation_author=M Falchi; citation_author=P Forabosco; citation_journal_title=Hum Genet; citation_volume=111; citation_number=31; citation_pages=9-15; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Major factors influencing linkage disequilibrium by analysis of different chromosome regions in distinct populations: demography, chromosome recombination frequency and selection.; citation_author=P Zavattari; citation_author=E Deidda; citation_author=M Whalen; citation_author=R Lampis; citation_author=A Mulargia; citation_journal_title=Hum Mol Genet; citation_volume=9; citation_number=32; citation_pages=2947-2957; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Reduced genetic structure of the Iberian peninsula revealed by Y-chromosome analysis: implications for population demography.; citation_author=C Flores; citation_author=N Maca-Meyer; citation_author=AM Gonzalez; citation_author=PJ Oefner; citation_author=P Shen; citation_journal_title=Eur J Hum Genet; citation_volume=12; citation_number=33; citation_pages=855-863; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The distribution of DR4 haplotypes in Sardinia suggests a primary association of insulin dependent diabetes mellitus with DRB1 and DQB1 loci.; citation_author=F Cucca; citation_author=R Lampis; citation_author=F Frau; citation_author=D Macis; citation_author=E Angius; citation_journal_title=Hum Immunol; citation_volume=43; citation_number=34; citation_pages=301-308; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Variation of the myelin oligodendrocyte glycoprotein gene is not primarily associated with multiple sclerosis in the Sardinian population.; citation_author=MG Marrosu; citation_author=R Murru; citation_author=G Costa; citation_author=MC Melis; citation_author=M Rolesu; citation_journal_title=BMC Genet; citation_volume=8; citation_number=35; citation_pages=25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Further evidence of a primary, causal association of the PTPN22 620W variant with type 1 diabetes.; citation_author=M Zoledziewska; citation_author=C Perra; citation_author=V Orru; citation_author=L Moi; citation_author=P Frongia; citation_journal_title=Diabetes.; citation_number=36; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Chelex 100 as a medium for simple extraction of DNA for PCR-based typing from forensic material.; citation_author=PS Walsh; citation_author=DA Metzger; citation_author=R Higuchi; citation_journal_title=Biotechniques; citation_volume=10; citation_number=37; citation_pages=506-513; citation_date=1991; " />
      <meta name="citation_reference" content="citation_number=38; " />
      <meta name="citation_reference" content="citation_title=Detection of numerous Y chromosome biallelic polymorphisms by denaturing high-performance liquid chromatography.; citation_author=PA Underhill; citation_author=L Jin; citation_author=AA Lin; citation_author=SQ Mehdi; citation_author=T Jenkins; citation_journal_title=Genome Res; citation_volume=7; citation_number=39; citation_pages=996-1005; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A nomenclature system for the tree of human Y-chromosomal binary haplogroups.; citation_author= Y-Chromosome-Consortium; citation_journal_title=Genome Res; citation_volume=12; citation_number=40; citation_pages=339-348; citation_date=2002; " />
      <meta name="citation_reference" content="citation_number=41; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood estimation from incomplete data via the EM algorithm.; citation_author=A Dempster; citation_author=N Laird; citation_author=D Rubin; citation_journal_title=Journal of the Royal Statistical Society; citation_volume=39; citation_number=42; citation_pages=1-38; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population.; citation_author=L Excoffier; citation_author=M Slatkin; citation_journal_title=Mol Biol Evol; citation_volume=12; citation_number=43; citation_pages=921-927; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Mathematical and statistical methods for genetic analysis.; citation_author=K Lange; citation_number=44; citation_date=1997; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=A measure of population subdivision based on microsatellite allele frequencies.; citation_author=M Slatkin; citation_journal_title=Genetics; citation_volume=139; citation_number=45; citation_pages=457-462; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Molecular evolutionary genetics.; citation_author=M Nei; citation_number=46; citation_pages=512; citation_date=1987; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_number=47; " />
      <meta name="citation_reference" content="citation_title=An integrated software package for population genetics data analysis.; citation_author=L Excoffier; citation_author=G Laval; citation_author=S Schneider; citation_journal_title=Evolutionary Bioinformatics Online; citation_volume=1; citation_number=48; citation_pages=47-50; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=49; " />
      <meta name="citation_reference" content="citation_title=PHYLIP-Phylogeny Inference Package (Version 3.2).; citation_author=J Felsestein; citation_journal_title=Cladistics; citation_volume=5; citation_number=50; citation_pages=164-166; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Inferences from DNA data: population histories, evolutionary processes and forensic match probabilities.; citation_author=IJW, M.E.; Wilson; citation_author=DJ Balding; citation_journal_title=J R Statist Soc A; citation_volume=166; citation_number=51; citation_pages=155-201; citation_date=2003; " />
      <meta name="citation_reference" content="citation_number=52; " />
      <meta name="citation_reference" content="citation_title=The effective mutation rate at Y chromosome short tandem repeats, with application to human population-divergence time.; citation_author=LA Zhivotovsky; citation_author=PA Underhill; citation_author=C Cinnioglu; citation_author=M Kayser; citation_author=B Morar; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=53; citation_pages=50-61; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Male demography in East Asia: a north-south contrast in human population expansion times.; citation_author=Y Xue; citation_author=T Zerjal; citation_author=W Bao; citation_author=S Zhu; citation_author=Q Shu; citation_journal_title=Genetics; citation_volume=172; citation_number=54; citation_pages=2431-2439; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Y chromosome evidence for Anglo-Saxon mass migration.; citation_author=ME Weale; citation_author=DA Weiss; citation_author=RF Jager; citation_author=N Bradman; citation_author=MG Thomas; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=55; citation_pages=1008-1021; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The History and Geography of Human Genes.; citation_author=LLM, P.; Cavalli-Sforza; citation_author=A Piazza; citation_number=56; citation_date=1994; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=Recent common ancestry of human Y chromosomes: evidence from DNA sequence data.; citation_author=R Thomson; citation_author=JK Pritchard; citation_author=P Shen; citation_author=PJ Oefner; citation_author=MW Feldman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=57; citation_pages=7360-7365; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A counter-clockwise northern route of the Y-chromosome haplogroup N from Southeast Asia towards Europe.; citation_author=S Rootsi; citation_author=LA Zhivotovsky; citation_author=M Baldovic; citation_author=M Kayser; citation_author=IA Kutuev; citation_journal_title=Eur J Hum Genet; citation_volume=15; citation_number=58; citation_pages=204-211; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Paleolithic Y-haplogroup heritage predominates in a Cretan highland plateau.; citation_author=L Martinez; citation_author=PA Underhill; citation_author=LA Zhivotovsky; citation_author=T Gayden; citation_author=NK Moschonas; citation_journal_title=Eur J Hum Genet; citation_volume=15; citation_number=59; citation_pages=485-493; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The human Y chromosome: an evolutionary marker comes of age.; citation_author=MA Jobling; citation_author=C Tyler-Smith; citation_journal_title=Nat Rev Genet; citation_volume=4; citation_number=60; citation_pages=598-612; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Comparison of mode estimation methods and application in molecular clock analysis.; citation_author=SB Hedges; citation_author=P Shah; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=61; citation_pages=31; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Robust estimators of the mode and skewness of continuous data.; citation_author=DR Bickel; citation_journal_title=Comp Stat Data Anal; citation_volume=39; citation_number=62; citation_pages=153-163; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans"/>
    <meta name="twitter:description" content="The island of Sardinia shows a unique high incidence of several autoimmune diseases with multifactorial inheritance, particularly type 1 diabetes and multiple sclerosis. The prior knowledge of the genetic structure of this population is fundamental to establish the optimal design for association studies in these diseases. Previous work suggested that the Sardinians are a relatively homogenous population, but some reports were contradictory and data were largely based on variants subject to selection. For an unbiased assessment of genetic structure, we studied a combination of neutral Y-chromosome variants, 21 biallelic and 8 short tandem repeats (STRs) in 930 Sardinian males. We found a high degree of interindividual variation but a homogenous distribution of the detected variability in samples from three separate regions of the island. One haplogroup, I-M26, is rare or absent outside Sardinia and is very common (0.37 frequency) throughout the island, consistent with a founder effect. A Bayesian full likelihood analysis (BATWING) indicated that the time from the most recent common ancestor (TMRCA) of I-M26, was 21.0 (16.0&ndash;25.5) thousand years ago (KYA) and that the population began to expand 14.0 (7.8&ndash;22.0) KYA. These results suggest a largely pre-Neolithic settlement of the island with little subsequent gene flow from outside populations. Consequently, Sardinia is an especially attractive venue for case-control genome wide association scans in common multifactorial diseases. Concomitantly, the high degree of interindividual variation in the current population facilitates fine mapping efforts to pinpoint the aetiologic polymorphisms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001430.g003"/>

  <meta property="og:title" content="Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=238'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5015'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6274&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001430">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001430" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001430&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniela Contu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Daniela Contu, Laura Morelli</p>

              
              

                <p>Affiliation:
                  Laboratorio di Immunogenetica, Ospedale Microcitemico, Cagliari, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura Morelli
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Daniela Contu, Laura Morelli</p>

              
              

                <p>Affiliation:
                  Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Federico Santoni, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Advanced Studies, Research and Development in Sardinia (CRS4), Pula, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jamie W. Foster, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy
                </p>

                <p>
                    Current address: Albert Park, Melbourne, Australia
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paolo Francalacci, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Dipartimento di Zoologia e Genetica Evoluzionistica, Università di Sassari, Sassari, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Francesco Cucca
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:fcucca@uniss.it">fcucca@uniss.it</a></p>

                <p>Affiliation:
                  Dipartimento di Scienze Biomediche, Università di Sassari, Sassari, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 09, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001430</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001430-g001" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001430-t001" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001430-t002" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001430-t003" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001430-g002" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001430-g003" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001430-t004" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001430">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001430" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The island of Sardinia shows a unique high incidence of several autoimmune diseases with multifactorial inheritance, particularly type 1 diabetes and multiple sclerosis. The prior knowledge of the genetic structure of this population is fundamental to establish the optimal design for association studies in these diseases. Previous work suggested that the Sardinians are a relatively homogenous population, but some reports were contradictory and data were largely based on variants subject to selection. For an unbiased assessment of genetic structure, we studied a combination of neutral Y-chromosome variants, 21 biallelic and 8 short tandem repeats (STRs) in 930 Sardinian males. We found a high degree of interindividual variation but a homogenous distribution of the detected variability in samples from three separate regions of the island. One haplogroup, I-M26, is rare or absent outside Sardinia and is very common (0.37 frequency) throughout the island, consistent with a founder effect. A Bayesian full likelihood analysis (BATWING<em>)</em> indicated that the time from the most recent common ancestor (TMRCA) of I-M26, was 21.0 (16.0–25.5) thousand years ago (KYA) and that the population began to expand 14.0 (7.8–22.0) KYA. These results suggest a largely pre-Neolithic settlement of the island with little subsequent gene flow from outside populations. Consequently, Sardinia is an especially attractive venue for case-control genome wide association scans in common multifactorial diseases. Concomitantly, the high degree of interindividual variation in the current population facilitates fine mapping efforts to pinpoint the aetiologic polymorphisms.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Contu D, Morelli L, Santoni F, Foster JW, Francalacci P, et al.  (2008) Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans. PLoS ONE 3(1):
          e1430.
            doi:10.1371/journal.pone.0001430</p><p><strong>Academic Editor: </strong>John Hawks, University of Wisconsin, United States of America</p><p><strong>Received:</strong> July 31, 2007; <strong>Accepted:</strong> December 3, 2007; <strong>Published:</strong> January 9, 2008</p><p><strong>Copyright:</strong> © 2008 Contu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by funds from Telethon-JDRF (GJT030477) to FC and Interreg III to PF.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Sardinia has long been of interest for human geneticists. Some demographic and genetic features of this population, related to its insularity, offered the opportunity to study the impact of natural selection, for instance in determining resistance against malaria <a href="#pone.0001430-Siniscalco1">[1]</a> and to clarify relevant aspects of the molecular bases of monogenic diseases common in Sardinia such as Thalassemia <a href="#pone.0001430-Rosatelli1">[2]</a>, Wilson disease <a href="#pone.0001430-Loudianos1">[3]</a>, and APECED <a href="#pone.0001430-Rosatelli2">[4]</a>. Now, increasingly, the interest has moved towards common multifactorial diseases. In particular, this island population has, together with Finland, the highest incidence of the autoimmune disease, type 1 diabetes, in the world <a href="#pone.0001430-Karvonen1">[5]</a> and represents the main exception to the north-south gradient in disease incidence observed in Europe. Another autoimmune disease, multiple sclerosis, also exhibits a considerably higher incidence in Sardinia compared with all the surrounding Southern-European populations <a href="#pone.0001430-Pugliatti1">[6]</a>. These data suggest that common type 1 diabetes and multiple sclerosis undetected susceptibility alleles are prevalent in Sardinia and that this population could be a suitable place to look for them using genome wide association scans. However, some key parameters of the genetic structure and past demographic history of this population are still only partially known. How old is this population? What is the explanation for the numerous founder effects observed for the genetic systems so far studied? What was the impact of the various invasions from outside populations on the genetic structure and substructure of the present time population? These issues need to be addressed.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Regarding the age of the population, there is some archaeological evidence of pre-Neolithic occupation after which this island population gradually increased in size during the Neolithic age. An advanced civilisation developed in the Bronze Age, characterised by the building of fortified towers, called <em>Nuraghe</em>, throughout the island. During this time (3.6–2.2 KYA) it is estimated that the island population reached approximately 300,000 inhabitants <a href="#pone.0001430-Lilliu1">[7]</a> and remained relatively constant during the subsequent Phoenician-Carthaginian (535 BC–238 BC) and Roman dominations (238 BC–476 AD); it did not significantly increase until around 300 years ago. For instance, in 1627 AD a fiscal survey estimated the population to be 297,000. Later, the population was 431,000 in 1781 AD, increased to 636,000 in 1871 AD, exceeded 1,000,000 in 1936 AD and currently totals 1,632,000 individuals.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>However, while these archaeological and historical findings provide some interesting clues, they do not give a dynamic portrait of the population history and of the actual impact of these events on the structure of the present-time population. Furthermore, they do not help clarify the early events of the foundation of this population. Genetics can address this in a more informative way.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Previous studies on autosomal variants indicate the absence of significant heterogeneity between large geographic subregions which have suffered, historically, repeated invasions, and the most internal and isolated part of the island which was substantially untouched by these occupations <a href="#pone.0001430-Lampis1">[8]</a>, <a href="#pone.0001430-Contu1">[9]</a>. However, given that this evidence was obtained from autosomal variants, mainly located in the HLA region, whose protein products are under strong natural selection, it is difficult to determine how representative they are for the genome as a whole. Furthermore, some studies have reported a degree of micro-heterogeneity, particularly when comparing small areas and isolated villages <a href="#pone.0001430-Zei1">[10]</a>–<a href="#pone.0001430-Fraumene1">[13]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Hence, interpretation of the overall genetic panorama is not straightforward. Prior knowledge of the genetic structure of a population is relevant not only to reconstruct its ancient demographic history but is also a fundamental prerequisite for association studies, and notably for genome wide association scans to dissect the genetic factors in multifactorial traits. For instance, using the cost effective case-control design in a regionally heterogeneous or substructured population, both false positive and false negative results will occur if patients and controls are not carefully sampled in equivalent proportions according to their origin.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>To obtain a more accurate portrait of genetic structure and descent, we determined the Y-chromosome haplotype structure of a large number of Sardinian males using a panel of informative neutral single nucleotide polymorphisms (SNPs) and short tandem repeats (STRs), also known as microsatellites, located in the male specific part of this chromosome. These markers are particularly well suited for these comparative analyses because they were not explicitly affected by evolutionary forces related to the immune response against pathogens. The non-recombining nature of this chromosome region as well as the availability of a reliable molecular clock, through the established mutation rate of STRs, allows a chronological reconstruction of haplotype genealogy, when these different types of markers are analysed jointly <a href="#pone.0001430-Semino1">[14]</a>–<a href="#pone.0001430-Wells1">[17]</a>. The aim of the study was thus to establish the intra-regional distribution of these Y-chromosome molecular markers and to infer past peopling and demographic events in Sardinia, evaluating the impact of these factors in association study design for multifactorial diseases.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>We established the distribution of the Y-chromosome haplogroup lineages in 376 newborn males from three large regions of Sardinia (<a href="#pone-0001430-g001">Figure 1</a> and <a href="#s4">Methods</a>). We found that the distribution of the various haplogroups was relatively homogeneous across these different areas of the island (<a href="#pone-0001430-t001">Table 1</a>). In fact, the pairwise F<sub>STs</sub> computed from the haplogroup frequencies were not significantly different for inter-population differentiation (<a href="#pone-0001430-t002">Table 2</a>). Similarly, pairwise R<sub>STs</sub> computed on the same sample using STR markers did not provide significant evidence of inter-population differentiation (<a href="#pone-0001430-t002">Table 2</a>).</p>
<div class="figure" id="pone-0001430-g001"><div class="img"><a name="pone-0001430-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Map of Mediterranean showing the location of Sardinia.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>Expanded section shows the island of Sardinia with the sample regions highlighted</p>
<span>doi:10.1371/journal.pone.0001430.g001</span></div><div class="figure" id="pone-0001430-t001"><div class="img"><a name="pone-0001430-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Haplotype frequencies and gene diversity of the Y-chromosome lineages in 376 newborns from three regions of Sardinia</span></strong></p><span>doi:10.1371/journal.pone.0001430.t001</span></div><div class="figure" id="pone-0001430-t002"><div class="img"><a name="pone-0001430-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Population pairwise F<sub>ST</sub> and R<sub>ST</sub> values between different Sardinian sub-regions</span></strong></p><span>doi:10.1371/journal.pone.0001430.t002</span></div><a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>To further evaluate the impact of population substructure in association studies, we compared the frequencies of biallelic variants from a collection of 399 type 1 diabetes patients, collected from the southern part of the island for a previous study <a href="#pone.0001430-Contu2">[18]</a> with those from the pool of 376 newborns collected from three geographically distant areas of the island (data referring to the STRs and SNPs fully and jointly typed in the Sardinian samples are reported in the supplementary <a href="#pone.0001430.s001">Table S1</a>). Even under this forced scenario – very different from the preferred collection of patients and controls in similar proportions from the same subregions- we did not obtain any significant evidence of heterogeneity (F<sub>ST</sub> = 0.001, P = 0.157).</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>With the information that the Sardinian population was relatively genetically homogeneous, we established the haplogroup frequencies in a joint sample set of the 376 newborns, 399 type 1 diabetes patients, and an additional 155 blood donors. In this enlarged data set of 930 males, we detected 19 haplogroup lineages (<a href="#pone-0001430-t003">Table 3</a>). Haplogroups defined by the M130 and M68 SNPs were tested for, but not detected. We then compared the haplogroup frequencies observed in these 930 Sardinian males with those reported in other Euro-Mediterranean populations. Distribution of the Y-chromosome haplogroups detected in the Sardinians set them apart from the rest of the European groups <a href="#pone.0001430-Semino1">[14]</a>, <a href="#pone.0001430-Bosch1">[16]</a>, <a href="#pone.0001430-Francalacci1">[19]</a>, <a href="#pone.0001430-Cinnioglu1">[20]</a>.</p>
<div class="figure" id="pone-0001430-t003"><div class="img"><a name="pone-0001430-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Haplotype frequencies of the Y-chromosome lineages and gene diversity in 930 Sardinian males</span></strong></p><span>doi:10.1371/journal.pone.0001430.t003</span></div><a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>The main discriminator of Sardinians from the other populations comes from the I-M26 haplogroup. This variant shows an overall frequency of 0.37 in Sardinia but is absent in most other populations, including the neighbouring island of Corsica <a href="#pone.0001430-Francalacci1">[19]</a>. It has been detected, though at much lower frequencies, only in the Basques <a href="#pone.0001430-Semino1">[14]</a>, <a href="#pone.0001430-Bosch1">[16]</a>, <a href="#pone.0001430-Francalacci1">[19]</a>, <a href="#pone.0001430-Cinnioglu1">[20]</a> and in a few other western European populations <a href="#pone.0001430-Capelli1">[21]</a>–<a href="#pone.0001430-Rootsi1">[23]</a>. In addition, the rare M18 SNP appears to be unique to Sardinia. Furthermore, some haplogroups present and relatively frequent on the island, such as G-M201 and E-M78, have only been detected in eastern countries while another haplogroup common in Sardinia, R-269, is more common in western populations <a href="#pone.0001430-Semino1">[14]</a>. As a result of this distribution of lineages in Sardinia, there is a high degree of gene diversity (0.801±0.010) (see <a href="#pone-0001430-t003">Table 3</a>) and significant evidence of differentiation from all the other groups for which matched genetic data are available (F<sub>ST</sub> values and their P values are shown in <a href="#pone.0001430.s002">Tables S2</a> and <a href="#pone.0001430.s003">S3</a>). An UPGMA (Unweighted Pair Group Method with Arithmetic mean) tree constructed using one thousand bootstrap FST distances, graphically illustrates the differentiation of Sardinians from the other populations (<a href="#pone-0001430-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0001430-g002"><div class="img"><a name="pone-0001430-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>UPGMA consensus tree correlating European populations constructed using 1000 F<sub>ST</sub> pairwise distances.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Numbers before forks indicate percentage of times that each group appeared in the input trees</p>
<span>doi:10.1371/journal.pone.0001430.g002</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>A possible reservation to this scenario is an SNP ascertainment bias, given that the samples originally screened for Y chromosome SNPs <a href="#pone.0001430-Underhill1">[15]</a> included Sardinian Y chromosomes, and in fact both M18 and M26 were discovered in Sardinian DNA samples. In principle, this kind of ascertainment bias would artifactually increase the differentiation of Sardinians from other populations. However, comparable results were also obtained computing pairwise R<sub>ST</sub> values (<a href="#pone.0001430.s004">Tables S4</a> and <a href="#pone.0001430.s005">S5</a>) based exclusively on STR data which are not prone to this bias (see <a href="#pone-0001430-g003">Figure 3</a> which shows a consensus tree from one thousand bootstraps). We next used BATWING analysis to establish the individual TMRCA for the most representative haplogroup lineages (I-M26, R-M269, G-M201, E-M78), collectively accounting for 71% of the Sardinian Y-chromosome haplogroups, analysed together with the 6 STR markers. The TMRCA values for these haplogroups are included in a span of 19.5–22.8 KYA (<a href="#pone-0001430-t004">Table 4</a>). We also assessed the TMRCA for the rare Sardinia-specific R-M18 haplogroup, and obtained a value of 8.1 (7.8–10.5) KYA.</p>
<div class="figure" id="pone-0001430-g003"><div class="img"><a name="pone-0001430-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>UPGMA consensus tree correlating European populations constructed using 1000 R<sub>ST</sub> pairwise distances.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>Numbers before forks indicate percentage of times that each group appeared in the input trees</p>
<span>doi:10.1371/journal.pone.0001430.g003</span></div><div class="figure" id="pone-0001430-t004"><div class="img"><a name="pone-0001430-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001430" data-uri="info:doi/10.1371/journal.pone.0001430.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001430.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001430.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001430.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001430.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>TMRCA values of the main Sardinian haplogroups provided by BATWING analysis</span></strong></p><span>doi:10.1371/journal.pone.0001430.t004</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>We also used BATWING to analyse the data from the 376 Sardinian newborns to evaluate when this island population began to expand, and obtained a value of 14.0 (7.8–22.0) KYA. To assess this data in the context of that observed in other European populations, we used matched published data <a href="#pone.0001430-Bosch1">[16]</a>, <a href="#pone.0001430-Cinnioglu1">[20]</a>, <a href="#pone.0001430-Bosch3">[24]</a> and computed the time of initial expansion of other populations of interest. We found a value of 9.5 (6.7–13.7) KYA for the Anatolian population, whereas two data sets of West-Balkan and Iberian populations provided a signal of initial expansion of 7.4 (6.5–16.1) and 9.8 (9.5–13.7) KYA respectively. BATWING analysis also suggested an effective population size at the beginning of the expansion of 789 (450–1264) for the Sardinians, 1741 (957–2857) for the Anatolians, 601 (218–867) for the Balkans and 339 (273–668) for the Iberians.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our findings indicate that three large, geographically distinct subregions of Sardinia show no significant evidence of inter-region genetic heterogeneity by any of the established measures of population differentiation. In particular, sub-populations from the Sardinian coastal regions (the Campidano and Gallura areas), which suffered cultural and political dominations over many years do not significantly differ from the most internal and isolated part of the island (Barbagia area), which was never under foreign control. This is in agreement with other studies that analysed different chromosomes and independent samples <a href="#pone.0001430-Lampis1">[8]</a>, <a href="#pone.0001430-Contu1">[9]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The homogenous distribution of variants in different macro-areas of the island is consistent with a common ancient origin and cultural background. For example, <em>Nuraghe</em>, the characteristic Sardinian prehistoric towers and castles, are widely and evenly distributed across the island and are substantially confined to Sardinia, thus supporting their local creation. At first glance their homogeneous intra-regional distribution could also be explained by mimicry by rival tribes, due to their strategic value. However, the full correspondence of their complex architectural features as well as the overlapping structure of the prehistoric villages surrounding them and the identity of the tools, sculptures, and artefacts of symbolic nature found with them is overwhelmingly suggestive of a common cultural background which may be consistent with internal gene flow and may have acted as a main homogenising force during the Bronze Age.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our data also provide some clues about the time when the Sardinian population was initially founded. We found that the common Y-chromosome haplogroups detected in Sardinia show rather even TMRCAs ranging from 19.5 to 22.8 KYA, which appear independent from their genealogy and thus indicative of earlier pre-Neolithic founder effects. In particular, the I-M26 haplogroup, showing a TMRCA of 21.0 (16.0–25.5) KYA, is especially informative because it is present in 37% of Sardinian males but is absent or very rare in all other human groups. Furthermore, I-M26 represents a terminal branch of the Y-chromosome tree and is thus a relatively recent haplogroup. Overall, it is therefore conceivable that little intra-haplogroup variability entered for I-M26 at the foundation of the Sardinian population and its TMRCA value is, in fact, roughly related to the initial occupation of the island.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>BATWING analysis also reveals that the Sardinian male population began to expand about 14 KYA, with an initial effective population size on the order of 1,000 individuals. The signal of expansion detected by BATWING probably does not predate the arrival of the Sardinian founders on the island, because we observed that other European sample sets provide more recent times of initial expansion.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Thus, despite the simplified demographic model and wide confidence intervals of the BATWING estimates, a coherent picture shows expansion starting in pre-Neolithic times. In fact, the inferred BATWING values are conservative, using a generation time of 25 years. Furthermore, to obtain more robust and reliable results we used mode-based estimates instead of the more commonly-used mean and median measures that tend to provide much older values for all the parameters assessed with BATWING (<a href="#s4">Methods</a> and unpublished results).</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>These genetic estimates are in agreement with archaeological evidence suggesting that Sardinia was already populated 18–10 KYA <a href="#pone.0001430-Sondaar1">[25]</a>, <a href="#pone.0001430-Sondaar2">[26]</a>. Furthermore, the genetic data are also consistent with geological data pointing to a land connection between Sardinia and the Italian peninsula (through Corsica) during the last glacial period, which might have allowed the passage of people for several thousand years until about 10 KYA. Evidence of an early pre-Neolithic population has also been derived from other genetic markers, including mitochondrial haplogroups V and H3 <a href="#pone.0001430-Torroni1">[27]</a>, <a href="#pone.0001430-Achilli1">[28]</a>. Sardinia may have attracted Palaeolithic hunter-gatherers, with its temperate climate; <em>Megaceros cazioti</em> deer and endemic species with a high reproductive rate; and other food sources such as fruits from the forests that covered the island.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The absence of the I-M26 lineage on the nearby island of Corsica <a href="#pone.0001430-Francalacci1">[19]</a> is somewhat surprising because Corsica should have been accessible to ancestral Sardinian settlers via a land bridge. However, Corsica may have been resettled from the Italian mainland during medieval times, a hypothesis supported by historical sources. Repopulation could have substituted for I-M26 and masked the putative Corsican ancestral genetic substratum.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>The data and conclusions presented in this study, and notably the absence of significant population sub-structure in Sardinia, have a significant impact on the design of genome wide association studies. Robust and statistically well-powered whole genome case-control association studies are thus promising in this population.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Other founder populations, such as the Finnish and the Icelandic, show some substructure in geographically distinct areas <a href="#pone.0001430-Lappalainen1">[29]</a>, <a href="#pone.0001430-Helgason1">[30]</a>. There are indications of micro-heterogeneity in subisolated in Sardinia as well, but they appear to be second-order effects. For example, one survey suggested the presence of heterogeneity among 21 subregions <a href="#pone.0001430-Cappello1">[12]</a>. However, an analysis of molecular variance (AMOVA) using the allele frequencies reported by the authors indicates that more than 99.8% of variation is within the tested subpopulations and less than 0.2% is between them (<a href="#pone.0001430.s006">Table S6</a>).</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Another study found some degree of heterogeneity in the distribution of Y chromosome markers of individuals with monophyletic surnames from different sub-regions <a href="#pone.0001430-Zei1">[10]</a> but, consistent with our results, found no evidence of significant heterogeneity when considering only the place of birth of the genotyped individuals. By selecting solely the monophyletic surnames, a relevant and shared part of the variability is removed from the analysis. Selection of monophyletic surnames is also more prone to the cumulative effect of mispaternity.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>Weak evidence of heterogeneity in the distribution of Y chromosome markers from different Sardinian sub-regions was also reported by Scozzari and colleagues <a href="#pone.0001430-Scozzari1">[11]</a>, but only for one of 17 markers tested, and comparing samples as small as 18 individuals (average of N = 33) and including small towns. Considerable evidence infers many demographic bottlenecks during the settlement of small and dispersed villages present on the island <a href="#pone.0001430-Fraumene1">[13]</a>, <a href="#pone.0001430-Angius1">[31]</a>, <a href="#pone.0001430-Zavattari1">[32]</a> which could account for the micro-heterogeneity and higher degree of background linkage disequilibrium observed in these small and isolated areas compared to the general Sardinian population <a href="#pone.0001430-Fraumene1">[13]</a>, <a href="#pone.0001430-Angius1">[31]</a>, <a href="#pone.0001430-Zavattari1">[32]</a>. Thus, the overall important feature that is confirmed by this study is the lack of significant differentiation between macro Sardinian areas. This finding is consistent with the fact that disorders such as type 1 diabetes and multiple sclerosis exhibit no significant differences in incidence in different Sardinian subregions.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12"></a><p>In addition, while we observed a relatively homogenous distribution of genetic markers across Sardinia, we found no restriction of inter-individual variability. For instance, the Sardinians exhibit a higher degree of inter-individual variation than the Spanish population <a href="#pone.0001430-Bosch1">[16]</a>, <a href="#pone.0001430-Flores1">[33]</a>. In general, Y chromosome variants common in either eastern or western European populations are detected at appreciable frequencies in this Mediterranean island. This ensures a high chance to detect, in this island, common ancient disease variants present in both western and eastern Europe that predate the foundation of Sardinia and its separation from the other European populations. Furthermore, these features, together with the fact that the population is ancient, account for highly informative haplotype splits observed in autosomal gene regions across the genome <a href="#pone.0001430-Cucca1">[34]</a>–<a href="#pone.0001430-Zoledziewska1">[36]</a>. This is especially useful during the post-detection fine mapping phase to highlight primarily associated variants in diseases common in this population when it is essential to exclude secondary hitchhiking effects.</p>
<a id="article1.body1.sec3.p13" name="article1.body1.sec3.p13"></a><p>The Y chromosome data also clearly underline the uniqueness of the Sardinian population and indicate that strong founder effects are not restricted to mutations involved in monogenic traits such as Beta-thalassemia <a href="#pone.0001430-Rosatelli1">[2]</a> or Wilson disease <a href="#pone.0001430-Loudianos1">[3]</a> but also include neutral, common SNPs, as illustrated by the I-M26 variant. This suggests that by virtue of these founder effects, some disease variants could be unique to the Sardinians or could be in the optimal range of frequency to allow their detection in this population.</p>
<a id="article1.body1.sec3.p14" name="article1.body1.sec3.p14"></a><p>Overall these data indicate that Sardinia provides a promising venue for effective genome wide association scans and for the subsequent fine mapping efforts to pinpoint aetiological polymorphisms in multifactorial trait research.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sample selection</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>DNA samples from newborns was extracted using the Chelex method <a href="#pone.0001430-Walsh1">[37]</a> from dried blood spots present on Guthrie Cards collected for neonatal screenings. In order to confine the genetic analyses to individuals of Sardinian origin, we took advantage of the patrilinear transmission of both chromosome Y markers and surnames and performed a survey of the surname distribution of each individual to be analysed using the website GENS <a href="#pone.0001430-httpgens.labo.net1">[38]</a>. More specifically, samples of individuals with surnames that were more common in continental Italy were not considered in this study. This selection was simplified by the fact that most Sardinians exhibit peculiar surnames, clearly distinct from those encountered in the Italian mainland. This peculiarity is due to the fact that the Sardinian language is differentiated within the romance language family, and hence the derived surnames are unequivocally distinguishable from the non-Sardinian ones. Using these surname based criteria, 0.9% of the chromosome Y samples were eliminated. Because of the sample selection strategy, this study is not addressed to evaluate the effect of very recent genetic flow from outside populations - essentially from the Italian mainland - that, according to demographic data, occurred mainly during the last sixty years. In addition, to ensure independence of the subjects analysed, only individuals born during an eight month sample selection period were included and, in cases of male twins, only one was considered. Using this approach we collected 187, 103 and 86 samples from newborns in delivery units located in Cagliari, Sorgono and Tempio respectively (<a href="#pone-0001430-g001">Figure 1</a>). The sample set from Cagliari includes newborns from the southern and most densely-populated area of Sardinia, the Campidano region. The sample set from Sorgono comprises newborns from the Barbagia area, representing the most internal and isolated area of the island. Until a few decades ago, the population of Barbagia lived in rather strict geographical and cultural isolation, distributed amongst small and dispersed villages. Finally, the sample set from Tempio refers to the Gallura area that is linguistically differentiated from the rest of the island, showing similarities with the dialect spoken on the nearby island of Corsica. Thus, these three regions were selected to maximise the possibility of finding population heterogeneity and to compare regions with different demographic history.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>We also considered an additional Sardinian sample set, for the most part from a case-control study, consisting of 155 healthy adult male blood donors from the transfusion centre of the Brotzu Hospital in Cagliari and 399 type 1 diabetes patient samples from a previous study <a href="#pone.0001430-Contu2">[18]</a>. The present study was approved by the institutional review board of the University of Cagliari. Each participant signed an informed consent form for the samples used. In the case of newborns, consent was obtained from the child's parents.</p>


<h4>Genotyping</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We assessed the samples for the SNPs M9, M13, M17, M18, M26, M35, M67, M68, M78, M89, M92, M102, M123, M130, M170, M172, M173, M201 M267 and M269. An Alu insertion (YAP) was used for M1, for a total of 21 Y-chromosome biallelic polymorphisms, all located in the male-specific portion of the Y-chromosome <a href="#pone.0001430-Semino1">[14]</a>, <a href="#pone.0001430-Underhill1">[15]</a>, <a href="#pone.0001430-Contu2">[18]</a>, <a href="#pone.0001430-Underhill2">[39]</a>. Lineages defined by the detected SNPs are indicated according to the cladistic name by mutation, following the conventional nomenclature of the Y-Chromosome Consortium (2002) <a href="#pone.0001430-YChromosomeConsortium1">[40]</a>. It should be noted that in the present study, nucleotide deletion and insertion characterising the M17 and M18 haplogroup lineages respectively were genotyped by direct capillary sequencing of all M173 positive samples, since conventional genotyping strategies do not appear to be appropriate in distinguishing between these variants (unpublished data).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The 9 STR loci considered in this study (<em>DYS19, DYS385a, DYS385b, DYS389-I, DYS389-II, DYS390, DYS391, DYS392, DYS393</em>) are all tetra-nucleotide repeats, except for the tri-nucleotide repeat <em>DYS392</em>, and were part of the Men-type Argus Y-HM PCR amplification kit, developed according to recommendations of the International Forensic Y-User Group <a href="#pone.0001430-httpwww.ystr.orgindex.html1">[41]</a>. Y-STR loci were genotyped by separating the fluorescent-tagged PCR products on a capillary-sequencer (ABI PRISM 3100 Genetic Analyzer) following the manufacturer's instructions. Note that 3 of the 9 STRs originally typed were excluded from the analyses since two of them (<em>DYS385a, DYS385b</em>) were duplicated and phylogenetically indistinguishable (and thus not valid for the tests performed in this study), and another (<em>DYS392</em>) provided genotypes of marginal quality below the threshold of acceptance (90%) for inclusion in this study.</p>


<h4>Statistical analysis of the data</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We used conventional pairwise F<sub>ST</sub> (Fixation Index-Statistics), applied on the data from biallelic markers, as a short-term genetic distance measure between different Sardinian and European populations <a href="#pone.0001430-Dempster1">[42]</a>–<a href="#pone.0001430-Lange1">[44]</a>, testing distribution under the null hypothesis of “no genetic differentiation between the populations”, by permuting haplotypes between populations. We also applied pairwise R<sub>ST</sub>, an <em>F</em>-statistic derived under a stepwise mutation model to the microsatellite data <a href="#pone.0001430-Slatkin1">[45]</a>. Gene diversity and its sampling variance were estimated according to Nei <a href="#pone.0001430-Nei1">[46]</a>, as the probability that two randomly-chosen haplotypes are different in the sample; an estimate equivalent to the expected heterozygosity for diploid data. We computed pairwise conventional F<sub>ST</sub>, R<sub>ST</sub> and gene diversity using the Arlequin 3.11 software <a href="#pone.0001430-httpcmpg.unibe.chsoftwarearlequin31">[47]</a>, <a href="#pone.0001430-Excoffier2">[48]</a>. Pre-processing of datasets for bootstrapping and format conversions was performed using ad hoc Python scripts on a Linux cluster machine. A thousand FST and RST distances were then clustered using PHYLIP (PHYlogeny Inference Package) v3.66 using the UPGMA tree drawing method and unrooted consensus trees were built with the PHYLIP executable “Consense”. Trees were then drawn with the PHYLIP executable “Drawtree” <a href="#pone.0001430-httpevolution.genetics.washington.eduphylip.html.1">[49]</a>, <a href="#pone.0001430-Felsestein1">[50]</a>. We carried out full likelihood Bayesian inference of genetic and demographic parameters under population growth using the BATWING programme (Bayesian analysis of trees with internal node generation) <a href="#pone.0001430-Wilson1">[51]</a>, <a href="#pone.0001430-httpwww.mas.ncl.ac.uknijw1">[52]</a>. BATWING uses a Markov chain Monte Carlo procedure to generate a sequence of genealogic and population trees, with associated model parameter values, consistent with the genetic data observed in a sample of individuals. At equilibrium, the sequence of trees samples from the posterior probability distribution of trees, given the observed data and the assumed underlying genetic and demographic model. The extended BATWING version used here assumes an unbounded single stepwise mutation model for the microsatellite loci and a coalescent process under an exponential model of population growth from an initially constant-size population. A unique prior for the microsatellite mutation rate was based on the Zhivotovsky et al. <a href="#pone.0001430-Zhivotovsky1">[53]</a> estimate as 6.9×10<sup>−4</sup> and applied to <em>DYS19, DYS389I, DYS389II, DYS390</em>, <em>DYS391</em>, and <em>DYS393</em>, setting gamma as (1.47, 2130) (mean = 0.00069, SD = 0.00057) <a href="#pone.0001430-Xue1">[54]</a>. In this analysis we avoided using samples containing the duplicated <em>DYS19</em> microsatellite. Weakly informative priors were also given other parameters to aid convergence of the MCMC process. The initial effective population size was given a gamma (1, 0.0001) (mean = 10,000, SD = 10,000), which covers the values commonly assumed for the global Y-chromosome effective population size as well as lower values to compensate for this being a regional sample and for representing the effective size before growth <a href="#pone.0001430-Weale1">[55]</a>. Population growth rate alpha per generation was given a gamma (2, 400) (mean = 0.005, SD = 0.0035). This very flat prior gives support to much lower alpha values and covers estimates of real (census) population growth in various parts of the world over the past few thousand years <a href="#pone.0001430-CavalliSforza1">[56]</a> as well as supporting lower values to allow for growth in effective population size being plausibly lower than real growth. The time in coalescent units when exponential growth began, or Beta, was given a gamma (2, 1) (mean = 2, SD = 1.41) <a href="#pone.0001430-Wilson1">[51]</a>. All other parameters were given flat, uninformative priors <a href="#pone.0001430-Weale1">[55]</a> Generation time was set at 25 years as used elsewhere in Y-chromosome studies <a href="#pone.0001430-Thomson1">[57]</a>–<a href="#pone.0001430-Martinez1">[59]</a>, adapting the estimated generation times for present day males to the presumably shorter life span in the past <a href="#pone.0001430-Jobling1">[60]</a>. Although a natural measure of the central tendency of a sample of continuous data is its mode (the most probable value), the mean and median are the most popular measures of location due to their simplicity and ease of estimation. The median is often used instead of the mean for asymmetric data because it is closer to the mode and is insensitive to extreme values in the sample.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>However, the distribution of our simulated data appears definitely skewed, non-normal and exposed to unpredictable contamination depending on the Markov Chain Monte Carlo sampling of the state space. In this condition, mean and median exhibit remarkable bias whereas the mode can be reliably estimated by several robust estimators <a href="#pone.0001430-Hedges1">[61]</a>. In this manuscript, after verifying that none of the markers examined showed bi-modal distribution, we employed the Half-Range Mode (HRM). This is based on subsequent subdivision of the data set in an iterative fashion and appears to be one of the best compromises between reliability, ease of implementation and computing time. Indeed HRM is robust for a wide variety of distribution and contamination levels <a href="#pone.0001430-Bickel1">[62]</a>. This mode estimate also seems less affected by the sample size effect observed with mean-based estimates since, in our data, good convergence for the mode could be obtained even with relatively small sampling (~10<sup>7</sup>), resulting in a good compromise between accuracy of calculation and computation time.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001430"></div><a name="pone.0001430.s001" id="pone.0001430.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001430.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">STRs typed in the three Sardinian subpopulations</p>
<p class="siDoi">doi:10.1371/journal.pone.0001430.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.58 MB DOC)</p>
<a name="pone.0001430.s002" id="pone.0001430.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001430.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Conventional Population Pairwise FST values between different European populations</p>
<p class="siDoi">doi:10.1371/journal.pone.0001430.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001430.s003" id="pone.0001430.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001430.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">FST P values. Number of permutations: 10100</p>
<p class="siDoi">doi:10.1371/journal.pone.0001430.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001430.s004" id="pone.0001430.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001430.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Pairwise squared size difference RST values between different European populations</p>
<p class="siDoi">doi:10.1371/journal.pone.0001430.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001430.s005" id="pone.0001430.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001430.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">RST P values. Number of permutations: 10100</p>
<p class="siDoi">doi:10.1371/journal.pone.0001430.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001430.s006" id="pone.0001430.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001430.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">AMOVA analysis of 12 pre-molecular markers on 21 linguistic domains of Sardinia</p>
<p class="siDoi">doi:10.1371/journal.pone.0001430.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>This article is dedicated to the memory of Giuseppe Pilia and Chiara Perra. We thank Michael Whalen, David Schlessinger, John Todd, Chris Tyler-Smith, Magdalena Zoledziewska and Ercole Contu for useful discussions and helpful comments, Patrizia Zavattari and Loredana Moi for technical support and Rebecca Lewis for editorial assistance in the preparation of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: FC LM DC FS PF. Performed the experiments: LM DC. Analyzed the data: FC LM DC FS. Contributed reagents/materials/analysis tools: FC. Wrote the paper: FC. Other: Designed and supervised the study: FC. Participated in the study design: PF LM DC. Wrote the manuscript: FC. Revised the manuscript: PF LM CD. Assisted in manuscript revision: JF. Developed software to implement data analysis: FS. Participated in analysis of the data: DC. Performed statistical analysis: FS LM. Proposed mode-based estimates in BATWING analysis: FS. Performed marker genotyping: DC. Participated in marker genotyping: LM. Designed figures and tables: LM. Participated in designing figures and tables: DC. Read and approved the final manuscript: DC LM FS JF PF FC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001430-Siniscalco1" id="pone.0001430-Siniscalco1"></a>Siniscalco M, Bernini L, Latte B, Motulsky AG (1961) Favism and Thalassæmia in Sardinia and their Relationship to Malaria. Nature  190: 1179–1180.  <ul class="find" data-citedArticleID="1041352" data-doi="10.1038/1901179a0"><li><a href="http://dx.doi.org/10.1038/1901179a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Favism+and+Thalass%C3%A6mia+in+Sardinia+and+their+Relationship+to+Malaria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Favism+and+Thalass%C3%A6mia+in+Sardinia+and+their+Relationship+to+Malaria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001430-Rosatelli1" id="pone.0001430-Rosatelli1"></a>Rosatelli MC, Dozy A, Faa V, Meloni A, Sardu R, et al.  (1992) Molecular characterization of beta-thalassemia in the Sardinian population. Am J Hum Genet  50: 422–426.  <ul class="find" data-citedArticleID="1041344"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+characterization+of+beta-thalassemia+in+the+Sardinian+population.&amp;auth=&amp;atitle=Molecular+characterization+of+beta-thalassemia+in+the+Sardinian+population." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+beta-thalassemia+in+the+Sardinian+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+beta-thalassemia+in+the+Sardinian+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001430-Loudianos1" id="pone.0001430-Loudianos1"></a>Loudianos G, Dessi V, Lovicu M, Angius A, Figus A, et al.  (1999) Molecular characterization of wilson disease in the Sardinian population–evidence of a founder effect. Hum Mutat  14: 294–303.  <ul class="find" data-citedArticleID="1041330" data-doi="10.1002/(sici)1098-1004(199910)14:4<294::aid-humu4&gt;3.0.co;2-9"><li><a href="http://dx.doi.org/10.1002/(sici)1098-1004(199910)14:4<294::aid-humu4&gt;3.0.co;2-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+wilson+disease+in+the+Sardinian+population%E2%80%93evidence+of+a+founder+effect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+wilson+disease+in+the+Sardinian+population%E2%80%93evidence+of+a+founder+effect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001430-Rosatelli2" id="pone.0001430-Rosatelli2"></a>Rosatelli MC, Meloni A, Devoto M, Cao A, Scott HS, et al.  (1998) A common mutation in Sardinian autoimmune polyendocrinopathy- candidiasis-ectodermal dystrophy patients. Hum Genet  103: 428–434.  <ul class="find" data-citedArticleID="1041346" data-doi="10.1007/s004390050846"><li><a href="http://dx.doi.org/10.1007/s004390050846" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+mutation+in+Sardinian+autoimmune+polyendocrinopathy-+candidiasis-ectodermal+dystrophy+patients." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+mutation+in+Sardinian+autoimmune+polyendocrinopathy-+candidiasis-ectodermal+dystrophy+patients.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001430-Karvonen1" id="pone.0001430-Karvonen1"></a>Karvonen M, Tuomilehto J, Libman I, La Porte R (1993) A review of the recent epidemiological data on the worldwide incidence of type 1 (insulin-dependent) diabetes mellitus. World Health Organization DIAMOND Project Group. Diabetologia  36: 883–892.  <ul class="find" data-citedArticleID="1041320" data-doi="10.1007/bf02374468"><li><a href="http://dx.doi.org/10.1007/bf02374468" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+review+of+the+recent+epidemiological+data+on+the+worldwide+incidence+of+type+1+%28insulin-dependent%29+diabetes+mellitus.+World+Health+Organization+DIAMOND+Project+Group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+review+of+the+recent+epidemiological+data+on+the+worldwide+incidence+of+type+1+%28insulin-dependent%29+diabetes+mellitus.+World+Health+Organization+DIAMOND+Project+Group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001430-Pugliatti1" id="pone.0001430-Pugliatti1"></a>Pugliatti M, Rosati G, Carton H, Riise T, Drulovic J, et al.  (2006) The epidemiology of multiple sclerosis in Europe. Eur J Neurol  13: 700–722.  <ul class="find" data-citedArticleID="1041338" data-doi="10.1111/j.1468-1331.2006.01342.x"><li><a href="http://dx.doi.org/10.1111/j.1468-1331.2006.01342.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+epidemiology+of+multiple+sclerosis+in+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+epidemiology+of+multiple+sclerosis+in+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001430-Lilliu1" id="pone.0001430-Lilliu1"></a>Lilliu G (1982) La civiltà nuragica. Sassari: Delfino.   <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0001430-Lampis1" id="pone.0001430-Lampis1"></a>Lampis R, Morelli L, Congia M, Macis MD, Mulargia A, et al.  (2000) The inter-regional distribution of HLA class II haplotypes indicates the suitability of the Sardinian population for case-control association studies in complex diseases. Hum Mol Genet  9: 2959–2965.  <ul class="find" data-citedArticleID="1041322" data-doi="10.1093/hmg/9.20.2959"><li><a href="http://dx.doi.org/10.1093/hmg/9.20.2959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+inter-regional+distribution+of+HLA+class+II+haplotypes+indicates+the+suitability+of+the+Sardinian+population+for+case-control+association+studies+in+complex+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+inter-regional+distribution+of+HLA+class+II+haplotypes+indicates+the+suitability+of+the+Sardinian+population+for+case-control+association+studies+in+complex+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001430-Contu1" id="pone.0001430-Contu1"></a>Contu L, Arras M, Carcassi C, La Nasa G, Mulargia M (1992) HLA structure of the Sardinian population: a haplotype study of 551 families. Tissue Antigens  40: 165–174.  <ul class="find" data-citedArticleID="1041294" data-doi="10.1111/j.1399-0039.1992.tb02041.x"><li><a href="http://dx.doi.org/10.1111/j.1399-0039.1992.tb02041.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HLA+structure+of+the+Sardinian+population%3A+a+haplotype+study+of+551+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HLA+structure+of+the+Sardinian+population%3A+a+haplotype+study+of+551+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001430-Zei1" id="pone.0001430-Zei1"></a>Zei G, Lisa A, Fiorani O, Magri C, Quintana-Murci L, et al.  (2003) From surnames to the history of Y chromosomes: the Sardinian population as a paradigm. Eur J Hum Genet  11: 802–807.  <ul class="find" data-citedArticleID="1041382" data-doi="10.1038/sj.ejhg.5201040"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+surnames+to+the+history+of+Y+chromosomes%3A+the+Sardinian+population+as+a+paradigm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+surnames+to+the+history+of+Y+chromosomes%3A+the+Sardinian+population+as+a+paradigm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001430-Scozzari1" id="pone.0001430-Scozzari1"></a>Scozzari R, Cruciani F, Pangrazio A, Santolamazza P, Vona G, et al.  (2001) Human Y-chromosome variation in the western Mediterranean area: implications for the peopling of the region. Hum Immunol  62: 871–884.  <ul class="find" data-citedArticleID="1041348" data-doi="10.1016/s0198-8859(01)00286-5"><li><a href="http://dx.doi.org/10.1016/s0198-8859(01)00286-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Y-chromosome+variation+in+the+western+Mediterranean+area%3A+implications+for+the+peopling+of+the+region." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Y-chromosome+variation+in+the+western+Mediterranean+area%3A+implications+for+the+peopling+of+the+region.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001430-Cappello1" id="pone.0001430-Cappello1"></a>Cappello N, Rendine S, Griffo R, Mameli GE, Succa V, et al.  (1996) Genetic analysis of Sardinia: I. data on 12 polymorphisms in 21 linguistic domains. Ann Hum Genet  60 ( Pt 2): 125–141.  <ul class="find" data-citedArticleID="1041288" data-doi="10.1111/j.1469-1809.1996.tb01183.x"><li><a href="http://dx.doi.org/10.1111/j.1469-1809.1996.tb01183.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analysis+of+Sardinia%3A+I.+data+on+12+polymorphisms+in+21+linguistic+domains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analysis+of+Sardinia%3A+I.+data+on+12+polymorphisms+in+21+linguistic+domains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001430-Fraumene1" id="pone.0001430-Fraumene1"></a>Fraumene C, Petretto E, Angius A, Pirastu M (2003) Striking differentiation of sub-populations within a genetically homogeneous isolate (Ogliastra) in Sardinia as revealed by mtDNA analysis. Hum Genet  114: 1–10.  <ul class="find" data-citedArticleID="1041312" data-doi="10.1007/s00439-003-1008-3"><li><a href="http://dx.doi.org/10.1007/s00439-003-1008-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Striking+differentiation+of+sub-populations+within+a+genetically+homogeneous+isolate+%28Ogliastra%29+in+Sardinia+as+revealed+by+mtDNA+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Striking+differentiation+of+sub-populations+within+a+genetically+homogeneous+isolate+%28Ogliastra%29+in+Sardinia+as+revealed+by+mtDNA+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001430-Semino1" id="pone.0001430-Semino1"></a>Semino O, Passarino G, Oefner PJ, Lin AA, Arbuzova S, et al.  (2000) The genetic legacy of Paleolithic Homo sapiens sapiens in extant Europeans: a Y chromosome perspective. Science  290: 1155–1159.  <ul class="find" data-citedArticleID="1041350" data-doi="10.1126/science.290.5494.1155"><li><a href="http://dx.doi.org/10.1126/science.290.5494.1155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+legacy+of+Paleolithic+Homo+sapiens+sapiens+in+extant+Europeans%3A+a+Y+chromosome+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+legacy+of+Paleolithic+Homo+sapiens+sapiens+in+extant+Europeans%3A+a+Y+chromosome+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001430-Underhill1" id="pone.0001430-Underhill1"></a>Underhill PA, Shen P, Lin AA, Jin L, Passarino G, et al.  (2000) Y chromosome sequence variation and the history of human populations. Nat Genet  26: 358–361.  <ul class="find" data-citedArticleID="1041364"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Y+chromosome+sequence+variation+and+the+history+of+human+populations.&amp;auth=&amp;atitle=Y+chromosome+sequence+variation+and+the+history+of+human+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y+chromosome+sequence+variation+and+the+history+of+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y+chromosome+sequence+variation+and+the+history+of+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001430-Bosch1" id="pone.0001430-Bosch1"></a>Bosch E, Calafell F, Comas D, Oefner PJ, Underhill PA, et al.  (2001) High-resolution analysis of human Y-chromosome variation shows a sharp discontinuity and limited gene flow between northwestern Africa and the Iberian Peninsula. Am J Hum Genet  68: 1019–1029.  <ul class="find" data-citedArticleID="1041280" data-doi="10.1086/319521"><li><a href="http://dx.doi.org/10.1086/319521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-resolution+analysis+of+human+Y-chromosome+variation+shows+a+sharp+discontinuity+and+limited+gene+flow+between+northwestern+Africa+and+the+Iberian+Peninsula." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-resolution+analysis+of+human+Y-chromosome+variation+shows+a+sharp+discontinuity+and+limited+gene+flow+between+northwestern+Africa+and+the+Iberian+Peninsula.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001430-Wells1" id="pone.0001430-Wells1"></a>Wells RS, Yuldasheva N, Ruzibakiev R, Underhill PA, Evseeva I, et al.  (2001) The Eurasian heartland: a continental perspective on Y-chromosome diversity. Proc Natl Acad Sci U S A  98: 10244–10249.  <ul class="find" data-citedArticleID="1041372" data-doi="10.1073/pnas.171305098"><li><a href="http://dx.doi.org/10.1073/pnas.171305098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Eurasian+heartland%3A+a+continental+perspective+on+Y-chromosome+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Eurasian+heartland%3A+a+continental+perspective+on+Y-chromosome+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001430-Contu2" id="pone.0001430-Contu2"></a>Contu D, Morelli L, Zavattari P, Lampis R, Angius E, et al.  (2002) Sex-related bias and exclusion mapping of the nonrecombinant portion of chromosome Y in human type 1 diabetes in the isolated founder population of Sardinia. Diabetes  51: 3573–3576.  <ul class="find" data-citedArticleID="1041296" data-doi="10.2337/diabetes.51.12.3573"><li><a href="http://dx.doi.org/10.2337/diabetes.51.12.3573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sex-related+bias+and+exclusion+mapping+of+the+nonrecombinant+portion+of+chromosome+Y+in+human+type+1+diabetes+in+the+isolated+founder+population+of+Sardinia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sex-related+bias+and+exclusion+mapping+of+the+nonrecombinant+portion+of+chromosome+Y+in+human+type+1+diabetes+in+the+isolated+founder+population+of+Sardinia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001430-Francalacci1" id="pone.0001430-Francalacci1"></a>Francalacci P, Morelli L, Underhill PA, Lillie AS, Passarino G, et al.  (2003) Peopling of three Mediterranean islands (Corsica, Sardinia, and Sicily) inferred by Y-chromosome biallelic variability. Am J Phys Anthropol  121: 270–279.  <ul class="find" data-citedArticleID="1041310" data-doi="10.1002/ajpa.10265"><li><a href="http://dx.doi.org/10.1002/ajpa.10265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peopling+of+three+Mediterranean+islands+%28Corsica%2C+Sardinia%2C+and+Sicily%29+inferred+by+Y-chromosome+biallelic+variability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peopling+of+three+Mediterranean+islands+%28Corsica%2C+Sardinia%2C+and+Sicily%29+inferred+by+Y-chromosome+biallelic+variability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001430-Cinnioglu1" id="pone.0001430-Cinnioglu1"></a>Cinnioglu C, King R, Kivisild T, Kalfoglu E, Atasoy S, et al.  (2004) Excavating Y-chromosome haplotype strata in Anatolia. Hum Genet  114: 127–148.  <ul class="find" data-citedArticleID="1041292" data-doi="10.1007/s00439-003-1031-4"><li><a href="http://dx.doi.org/10.1007/s00439-003-1031-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excavating+Y-chromosome+haplotype+strata+in+Anatolia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Excavating+Y-chromosome+haplotype+strata+in+Anatolia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001430-Capelli1" id="pone.0001430-Capelli1"></a>Capelli C, Redhead N, Abernethy JK, Gratrix F, Wilson JF, et al.  (2003) A Y chromosome census of the British Isles. Curr Biol  13: 979–984.  <ul class="find" data-citedArticleID="1041286" data-doi="10.1016/s0960-9822(03)00373-7"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00373-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Y+chromosome+census+of+the+British+Isles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Y+chromosome+census+of+the+British+Isles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001430-Bosch2" id="pone.0001430-Bosch2"></a>Bosch E, Calafell F, Rosser ZH, Norby S, Lynnerup N, et al.  (2003) High level of male-biased Scandinavian admixture in Greenlandic Inuit shown by Y-chromosomal analysis. Hum Genet  112: 353–363.  <ul class="find" data-citedArticleID="1041282"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High+level+of+male-biased+Scandinavian+admixture+in+Greenlandic+Inuit+shown+by+Y-chromosomal+analysis.&amp;auth=&amp;atitle=High+level+of+male-biased+Scandinavian+admixture+in+Greenlandic+Inuit+shown+by+Y-chromosomal+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+level+of+male-biased+Scandinavian+admixture+in+Greenlandic+Inuit+shown+by+Y-chromosomal+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+level+of+male-biased+Scandinavian+admixture+in+Greenlandic+Inuit+shown+by+Y-chromosomal+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001430-Rootsi1" id="pone.0001430-Rootsi1"></a>Rootsi S, Magri C, Kivisild T, Benuzzi G, Help H, et al.  (2004) Phylogeography of Y-chromosome haplogroup I reveals distinct domains of prehistoric gene flow in europe. Am J Hum Genet  75: 128–137.  <ul class="find" data-citedArticleID="1041340" data-doi="10.1086/422196"><li><a href="http://dx.doi.org/10.1086/422196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogeography+of+Y-chromosome+haplogroup+I+reveals+distinct+domains+of+prehistoric+gene+flow+in+europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogeography+of+Y-chromosome+haplogroup+I+reveals+distinct+domains+of+prehistoric+gene+flow+in+europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001430-Bosch3" id="pone.0001430-Bosch3"></a>Bosch E, Calafell F, Gonzalez-Neira A, Flaiz C, Mateu E, et al.  (2006) Paternal and maternal lineages in the Balkans show a homogeneous landscape over linguistic barriers, except for the isolated Aromuns. Ann Hum Genet  70: 459–487.  <ul class="find" data-citedArticleID="1041284" data-doi="10.1111/j.1469-1809.2005.00251.x"><li><a href="http://dx.doi.org/10.1111/j.1469-1809.2005.00251.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paternal+and+maternal+lineages+in+the+Balkans+show+a+homogeneous+landscape+over+linguistic+barriers%2C+except+for+the+isolated+Aromuns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paternal+and+maternal+lineages+in+the+Balkans+show+a+homogeneous+landscape+over+linguistic+barriers%2C+except+for+the+isolated+Aromuns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001430-Sondaar1" id="pone.0001430-Sondaar1"></a>Sondaar PY, Elburg R, Klein-Hofmeijer G, Martini F, Sanges M, et al.  (1995) The human colonization of Sardinia: a Late-Pleistocene human fossil from Corbeddu cave. Comptes Rendus de l'Académie des Sciences Paris  320: 145–150.  <ul class="find" data-citedArticleID="1041356"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+human+colonization+of+Sardinia%3A+a+Late-Pleistocene+human+fossil+from+Corbeddu+cave.&amp;auth=&amp;atitle=The+human+colonization+of+Sardinia%3A+a+Late-Pleistocene+human+fossil+from+Corbeddu+cave." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+colonization+of+Sardinia%3A+a+Late-Pleistocene+human+fossil+from+Corbeddu+cave." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+colonization+of+Sardinia%3A+a+Late-Pleistocene+human+fossil+from+Corbeddu+cave.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001430-Sondaar2" id="pone.0001430-Sondaar2"></a>Sondaar PY (1998)  Palaeolithic Sardinians: paleontological evidence and methods. In: Balmuth MS, Tykot RH, editors. Sardinian and Aegean chronology. Oxford: Oxbow Books. pp. 45–51.  <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0001430-Torroni1" id="pone.0001430-Torroni1"></a>Torroni A, Bandelt HJ, Macaulay V, Richards M, Cruciani F, et al.  (2001) A signal, from human mtDNA, of postglacial recolonization in Europe. Am J Hum Genet  69: 844–852.  <ul class="find" data-citedArticleID="1041362" data-doi="10.1086/323485"><li><a href="http://dx.doi.org/10.1086/323485" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+signal%2C+from+human+mtDNA%2C+of+postglacial+recolonization+in+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+signal%2C+from+human+mtDNA%2C+of+postglacial+recolonization+in+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001430-Achilli1" id="pone.0001430-Achilli1"></a>Achilli A, Rengo C, Magri C, Battaglia V, Olivieri A, et al.  (2004) The molecular dissection of mtDNA haplogroup H confirms that the Franco-Cantabrian glacial refuge was a major source for the European gene pool. Am J Hum Genet  75: 910–918.  <ul class="find" data-citedArticleID="1041274" data-doi="10.1086/425590"><li><a href="http://dx.doi.org/10.1086/425590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+molecular+dissection+of+mtDNA+haplogroup+H+confirms+that+the+Franco-Cantabrian+glacial+refuge+was+a+major+source+for+the+European+gene+pool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+molecular+dissection+of+mtDNA+haplogroup+H+confirms+that+the+Franco-Cantabrian+glacial+refuge+was+a+major+source+for+the+European+gene+pool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001430-Lappalainen1" id="pone.0001430-Lappalainen1"></a>Lappalainen T, Koivumaki S, Salmela E, Huoponen K, Sistonen P, et al.  (2006) Regional differences among the Finns: a Y-chromosomal perspective. Gene  376: 207–215.  <ul class="find" data-citedArticleID="1041326" data-doi="10.1016/j.gene.2006.03.004"><li><a href="http://dx.doi.org/10.1016/j.gene.2006.03.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regional+differences+among+the+Finns%3A+a+Y-chromosomal+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regional+differences+among+the+Finns%3A+a+Y-chromosomal+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001430-Helgason1" id="pone.0001430-Helgason1"></a>Helgason A, Yngvadottir B, Hrafnkelsson B, Gulcher J, Stefansson K (2005) An Icelandic example of the impact of population structure on association studies. Nat Genet  37: 90–95.  <ul class="find" data-citedArticleID="1041316" data-doi="10.1038/ng1492"><li><a href="http://dx.doi.org/10.1038/ng1492" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Icelandic+example+of+the+impact+of+population+structure+on+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Icelandic+example+of+the+impact+of+population+structure+on+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001430-Angius1" id="pone.0001430-Angius1"></a>Angius A, Bebbere D, Petretto E, Falchi M, Forabosco P, et al.  (2002) Not all isolates are equal: linkage disequilibrium analysis on Xq13.3 reveals different patterns in Sardinian sub-populations. Hum Genet  111: 9–15.  <ul class="find" data-citedArticleID="1041276" data-doi="10.1007/s00439-002-0753-z"><li><a href="http://dx.doi.org/10.1007/s00439-002-0753-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+all+isolates+are+equal%3A+linkage+disequilibrium+analysis+on+Xq13.3+reveals+different+patterns+in+Sardinian+sub-populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+all+isolates+are+equal%3A+linkage+disequilibrium+analysis+on+Xq13.3+reveals+different+patterns+in+Sardinian+sub-populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001430-Zavattari1" id="pone.0001430-Zavattari1"></a>Zavattari P, Deidda E, Whalen M, Lampis R, Mulargia A, et al.  (2000) Major factors influencing linkage disequilibrium by analysis of different chromosome regions in distinct populations: demography, chromosome recombination frequency and selection. Hum Mol Genet  9: 2947–2957.  <ul class="find" data-citedArticleID="1041380" data-doi="10.1093/hmg/9.20.2947"><li><a href="http://dx.doi.org/10.1093/hmg/9.20.2947" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+factors+influencing+linkage+disequilibrium+by+analysis+of+different+chromosome+regions+in+distinct+populations%3A+demography%2C+chromosome+recombination+frequency+and+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+factors+influencing+linkage+disequilibrium+by+analysis+of+different+chromosome+regions+in+distinct+populations%3A+demography%2C+chromosome+recombination+frequency+and+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001430-Flores1" id="pone.0001430-Flores1"></a>Flores C, Maca-Meyer N, Gonzalez AM, Oefner PJ, Shen P, et al.  (2004) Reduced genetic structure of the Iberian peninsula revealed by Y-chromosome analysis: implications for population demography. Eur J Hum Genet  12: 855–863.  <ul class="find" data-citedArticleID="1041308" data-doi="10.1038/sj.ejhg.5201225"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201225" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+genetic+structure+of+the+Iberian+peninsula+revealed+by+Y-chromosome+analysis%3A+implications+for+population+demography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+genetic+structure+of+the+Iberian+peninsula+revealed+by+Y-chromosome+analysis%3A+implications+for+population+demography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001430-Cucca1" id="pone.0001430-Cucca1"></a>Cucca F, Lampis R, Frau F, Macis D, Angius E, et al.  (1995) The distribution of DR4 haplotypes in Sardinia suggests a primary association of insulin dependent diabetes mellitus with DRB1 and DQB1 loci. Hum Immunol  43: 301–308.  <ul class="find" data-citedArticleID="1041298" data-doi="10.1016/0198-8859(95)00042-3"><li><a href="http://dx.doi.org/10.1016/0198-8859(95)00042-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+distribution+of+DR4+haplotypes+in+Sardinia+suggests+a+primary+association+of+insulin+dependent+diabetes+mellitus+with+DRB1+and+DQB1+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+distribution+of+DR4+haplotypes+in+Sardinia+suggests+a+primary+association+of+insulin+dependent+diabetes+mellitus+with+DRB1+and+DQB1+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001430-Marrosu1" id="pone.0001430-Marrosu1"></a>Marrosu MG, Murru R, Costa G, Melis MC, Rolesu M, et al.  (2007) Variation of the myelin oligodendrocyte glycoprotein gene is not primarily associated with multiple sclerosis in the Sardinian population. BMC Genet  8: 25.  <ul class="find" data-citedArticleID="1041332" data-doi="10.1186/1471-2156-8-25"><li><a href="http://dx.doi.org/10.1186/1471-2156-8-25" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+of+the+myelin+oligodendrocyte+glycoprotein+gene+is+not+primarily+associated+with+multiple+sclerosis+in+the+Sardinian+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+of+the+myelin+oligodendrocyte+glycoprotein+gene+is+not+primarily+associated+with+multiple+sclerosis+in+the+Sardinian+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001430-Zoledziewska1" id="pone.0001430-Zoledziewska1"></a>Zoledziewska M, Perra C, Orru V, Moi L, Frongia P, et al.  (2007) Further evidence of a primary, causal association of the PTPN22 620W variant with type 1 diabetes. Diabetes..   <em>In press</em>.  <ul class="find" data-citedArticleID="1041386" data-doi="10.2337/db07-0289"><li><a href="http://dx.doi.org/10.2337/db07-0289" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Further+evidence+of+a+primary%2C+causal+association+of+the+PTPN22+620W+variant+with+type+1+diabetes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Further+evidence+of+a+primary%2C+causal+association+of+the+PTPN22+620W+variant+with+type+1+diabetes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001430-Walsh1" id="pone.0001430-Walsh1"></a>Walsh PS, Metzger DA, Higuchi R (1991) Chelex 100 as a medium for simple extraction of DNA for PCR-based typing from forensic material. Biotechniques  10: 506–513.  <ul class="find" data-citedArticleID="1041368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Chelex+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material.&amp;auth=&amp;atitle=Chelex+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chelex+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chelex+100+as+a+medium+for+simple+extraction+of+DNA+for+PCR-based+typing+from+forensic+material.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001430-httpgens.labo.net1" id="pone.0001430-httpgens.labo.net1"></a>  <a href="http://gens.labo.net">http://gens.labo.net</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001430-Underhill2" id="pone.0001430-Underhill2"></a>Underhill PA, Jin L, Lin AA, Mehdi SQ, Jenkins T, et al.  (1997) Detection of numerous Y chromosome biallelic polymorphisms by denaturing high-performance liquid chromatography. Genome Res  7: 996–1005.  <ul class="find" data-citedArticleID="1041366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Detection+of+numerous+Y+chromosome+biallelic+polymorphisms+by+denaturing+high-performance+liquid+chromatography.&amp;auth=&amp;atitle=Detection+of+numerous+Y+chromosome+biallelic+polymorphisms+by+denaturing+high-performance+liquid+chromatography." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+numerous+Y+chromosome+biallelic+polymorphisms+by+denaturing+high-performance+liquid+chromatography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+numerous+Y+chromosome+biallelic+polymorphisms+by+denaturing+high-performance+liquid+chromatography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001430-YChromosomeConsortium1" id="pone.0001430-YChromosomeConsortium1"></a>Y-Chromosome-Consortium  (2002) A nomenclature system for the tree of human Y-chromosomal binary haplogroups. Genome Res  12: 339–348.  <ul class="find" data-citedArticleID="1041378" data-doi="10.1101/gr.217602"><li><a href="http://dx.doi.org/10.1101/gr.217602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+nomenclature+system+for+the+tree+of+human+Y-chromosomal+binary+haplogroups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+nomenclature+system+for+the+tree+of+human+Y-chromosomal+binary+haplogroups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001430-httpwww.ystr.orgindex.html1" id="pone.0001430-httpwww.ystr.orgindex.html1"></a>  <a href="http://www.ystr.org/index.html">http://www.ystr.org/index.html</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001430-Dempster1" id="pone.0001430-Dempster1"></a>Dempster A, Laird N, Rubin D (1977) Maximum likelihood estimation from incomplete data via the EM algorithm. Journal of the Royal Statistical Society  39: 1–38.  <ul class="find" data-citedArticleID="1041300"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum+likelihood+estimation+from+incomplete+data+via+the+EM+algorithm.&amp;auth=&amp;atitle=Maximum+likelihood+estimation+from+incomplete+data+via+the+EM+algorithm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+estimation+from+incomplete+data+via+the+EM+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+estimation+from+incomplete+data+via+the+EM+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001430-Excoffier1" id="pone.0001430-Excoffier1"></a>Excoffier L, Slatkin M (1995) Maximum-likelihood estimation of molecular haplotype frequencies in a diploid population. Mol Biol Evol  12: 921–927.  <ul class="find" data-citedArticleID="1041302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population.&amp;auth=&amp;atitle=Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum-likelihood+estimation+of+molecular+haplotype+frequencies+in+a+diploid+population.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001430-Lange1" id="pone.0001430-Lange1"></a>Lange K (1997) Mathematical and statistical methods for genetic analysis. New York, NY, USA: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0001430-Slatkin1" id="pone.0001430-Slatkin1"></a>Slatkin M (1995) A measure of population subdivision based on microsatellite allele frequencies. Genetics  139: 457–462.  <ul class="find" data-citedArticleID="1041354"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+measure+of+population+subdivision+based+on+microsatellite+allele+frequencies.&amp;auth=&amp;atitle=A+measure+of+population+subdivision+based+on+microsatellite+allele+frequencies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+measure+of+population+subdivision+based+on+microsatellite+allele+frequencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+measure+of+population+subdivision+based+on+microsatellite+allele+frequencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001430-Nei1" id="pone.0001430-Nei1"></a>Nei M (1987) Molecular evolutionary genetics. New York, NY, USA: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">47.
              </span><a name="pone.0001430-httpcmpg.unibe.chsoftwarearlequin31" id="pone.0001430-httpcmpg.unibe.chsoftwarearlequin31"></a>  <a href="http://cmpg.unibe.ch/software/arlequin3/">http://cmpg.unibe.ch/software/arlequin3/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0001430-Excoffier2" id="pone.0001430-Excoffier2"></a>Excoffier L, Laval G, Schneider S (2005) An integrated software package for population genetics data analysis. Evolutionary Bioinformatics Online  1: 47–50.  <ul class="find" data-citedArticleID="1041304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+integrated+software+package+for+population+genetics+data+analysis.&amp;auth=&amp;atitle=An+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrated+software+package+for+population+genetics+data+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrated+software+package+for+population+genetics+data+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001430-httpevolution.genetics.washington.eduphylip.html.1" id="pone.0001430-httpevolution.genetics.washington.eduphylip.html.1"></a>  <a href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu​/phylip.html</a>. <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0001430-Felsestein1" id="pone.0001430-Felsestein1"></a>Felsestein J (1989) PHYLIP-Phylogeny Inference Package (Version 3.2). Cladistics  5: 164–166.  <ul class="find" data-citedArticleID="1041306"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29.&amp;auth=&amp;atitle=PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PHYLIP-Phylogeny+Inference+Package+%28Version+3.2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001430-Wilson1" id="pone.0001430-Wilson1"></a>Wilson IJW, M.E.;, Balding DJ (2003) Inferences from DNA data: population histories, evolutionary processes and forensic match probabilities. J R Statist Soc A  166: 155–201.  <ul class="find" data-citedArticleID="1041374" data-doi="10.1111/1467-985x.00264"><li><a href="http://dx.doi.org/10.1111/1467-985x.00264" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferences+from+DNA+data%3A+population+histories%2C+evolutionary+processes+and+forensic+match+probabilities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferences+from+DNA+data%3A+population+histories%2C+evolutionary+processes+and+forensic+match+probabilities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001430-httpwww.mas.ncl.ac.uknijw1" id="pone.0001430-httpwww.mas.ncl.ac.uknijw1"></a>  <a href="http://www.mas.ncl.ac.uk/nijw/">http://www.mas.ncl.ac.uk/nijw/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">53.
              </span><a name="pone.0001430-Zhivotovsky1" id="pone.0001430-Zhivotovsky1"></a>Zhivotovsky LA, Underhill PA, Cinnioglu C, Kayser M, Morar B, et al.  (2004) The effective mutation rate at Y chromosome short tandem repeats, with application to human population-divergence time. Am J Hum Genet  74: 50–61.  <ul class="find" data-citedArticleID="1041384" data-doi="10.1086/380911"><li><a href="http://dx.doi.org/10.1086/380911" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effective+mutation+rate+at+Y+chromosome+short+tandem+repeats%2C+with+application+to+human+population-divergence+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effective+mutation+rate+at+Y+chromosome+short+tandem+repeats%2C+with+application+to+human+population-divergence+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001430-Xue1" id="pone.0001430-Xue1"></a>Xue Y, Zerjal T, Bao W, Zhu S, Shu Q, et al.  (2006) Male demography in East Asia: a north-south contrast in human population expansion times. Genetics  172: 2431–2439.  <ul class="find" data-citedArticleID="1041376" data-doi="10.1534/genetics.105.054270"><li><a href="http://dx.doi.org/10.1534/genetics.105.054270" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Male+demography+in+East+Asia%3A+a+north-south+contrast+in+human+population+expansion+times." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Male+demography+in+East+Asia%3A+a+north-south+contrast+in+human+population+expansion+times.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001430-Weale1" id="pone.0001430-Weale1"></a>Weale ME, Weiss DA, Jager RF, Bradman N, Thomas MG (2002) Y chromosome evidence for Anglo-Saxon mass migration. Mol Biol Evol  19: 1008–1021.  <ul class="find" data-citedArticleID="1041370" data-doi="10.1093/oxfordjournals.molbev.a004160"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Y+chromosome+evidence+for+Anglo-Saxon+mass+migration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Y+chromosome+evidence+for+Anglo-Saxon+mass+migration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001430-CavalliSforza1" id="pone.0001430-CavalliSforza1"></a>Cavalli-Sforza LLM, P.;, Piazza A (1994) The History and Geography of Human Genes. Princeton, New Jersey 08540 USA: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0001430-Thomson1" id="pone.0001430-Thomson1"></a>Thomson R, Pritchard JK, Shen P, Oefner PJ, Feldman MW (2000) Recent common ancestry of human Y chromosomes: evidence from DNA sequence data. Proc Natl Acad Sci U S A  97: 7360–7365.  <ul class="find" data-citedArticleID="1041360" data-doi="10.1073/pnas.97.13.7360"><li><a href="http://dx.doi.org/10.1073/pnas.97.13.7360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+common+ancestry+of+human+Y+chromosomes%3A+evidence+from+DNA+sequence+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+common+ancestry+of+human+Y+chromosomes%3A+evidence+from+DNA+sequence+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001430-Rootsi2" id="pone.0001430-Rootsi2"></a>Rootsi S, Zhivotovsky LA, Baldovic M, Kayser M, Kutuev IA, et al.  (2007) A counter-clockwise northern route of the Y-chromosome haplogroup N from Southeast Asia towards Europe. Eur J Hum Genet  15: 204–211.  <ul class="find" data-citedArticleID="1041342" data-doi="10.1038/sj.ejhg.5201748"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201748" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+counter-clockwise+northern+route+of+the+Y-chromosome+haplogroup+N+from+Southeast+Asia+towards+Europe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+counter-clockwise+northern+route+of+the+Y-chromosome+haplogroup+N+from+Southeast+Asia+towards+Europe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001430-Martinez1" id="pone.0001430-Martinez1"></a>Martinez L, Underhill PA, Zhivotovsky LA, Gayden T, Moschonas NK, et al.  (2007) Paleolithic Y-haplogroup heritage predominates in a Cretan highland plateau. Eur J Hum Genet  15: 485–493.  <ul class="find" data-citedArticleID="1041334" data-doi="10.1038/sj.ejhg.5201769"><li><a href="http://dx.doi.org/10.1038/sj.ejhg.5201769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paleolithic+Y-haplogroup+heritage+predominates+in+a+Cretan+highland+plateau." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paleolithic+Y-haplogroup+heritage+predominates+in+a+Cretan+highland+plateau.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001430-Jobling1" id="pone.0001430-Jobling1"></a>Jobling MA, Tyler-Smith C (2003) The human Y chromosome: an evolutionary marker comes of age. Nat Rev Genet  4: 598–612.  <ul class="find" data-citedArticleID="1041318" data-doi="10.1038/nrg1124"><li><a href="http://dx.doi.org/10.1038/nrg1124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+Y+chromosome%3A+an+evolutionary+marker+comes+of+age." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+Y+chromosome%3A+an+evolutionary+marker+comes+of+age.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001430-Hedges1" id="pone.0001430-Hedges1"></a>Hedges SB, Shah P (2003) Comparison of mode estimation methods and application in molecular clock analysis. BMC Bioinformatics  4: 31.  <ul class="find" data-citedArticleID="1041314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparison+of+mode+estimation+methods+and+application+in+molecular+clock+analysis.&amp;auth=&amp;atitle=Comparison+of+mode+estimation+methods+and+application+in+molecular+clock+analysis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+mode+estimation+methods+and+application+in+molecular+clock+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+mode+estimation+methods+and+application+in+molecular+clock+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001430-Bickel1" id="pone.0001430-Bickel1"></a>Bickel DR (2002) Robust estimators of the mode and skewness of continuous data. Comp Stat Data Anal  39: 153–163.  <ul class="find" data-citedArticleID="1041278" data-doi="10.1016/s0167-9473(01)00057-3"><li><a href="http://dx.doi.org/10.1016/s0167-9473(01)00057-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robust+estimators+of+the+mode+and+skewness+of+continuous+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robust+estimators+of+the+mode+and+skewness+of+continuous+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.XML" value="120798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.PDF" value="223695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g001.PNG_L" value="1170233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g001.PNG_M" value="99057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g001.PNG_S" value="8578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g001.TIF" value="1960012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g001.PNG_I" value="33686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t001.PNG_L" value="205751"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t001.PNG_M" value="114027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t001.PNG_S" value="14440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t001.TIF" value="1067520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t001.PNG_I" value="34952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t002.PNG_L" value="78417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t002.PNG_M" value="102599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t002.PNG_S" value="14557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t002.TIF" value="334032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t002.PNG_I" value="35683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t003.PNG_L" value="141604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t003.PNG_M" value="114254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t003.PNG_S" value="17045"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t003.TIF" value="734316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t003.PNG_I" value="75545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g002.PNG_L" value="27053"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g002.PNG_M" value="36166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g002.PNG_S" value="7112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g002.TIF" value="39438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g003.PNG_L" value="26749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g003.PNG_M" value="34382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g003.PNG_S" value="6780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g003.TIF" value="37540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.g003.PNG_I" value="11993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t004.PNG_L" value="49585"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t004.PNG_M" value="63773"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t004.PNG_S" value="10980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t004.TIF" value="231896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.t004.PNG_I" value="21397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.s001.DOC" value="642048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.s002.DOC" value="64000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.s003.DOC" value="61952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.s004.DOC" value="62976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.s005.DOC" value="59904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001430.s006.DOC" value="50688"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001430&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001430" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001430&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001430&volume=&issue=&title=Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans&author_name=Daniela%20Contu%2C%20Laura%20Morelli%2C%20Federico%20Santoni%2C%20Jamie%20W.%20Foster%2C%20Paolo%20Francalacci%2C%20Francesco%20Cucca&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001430" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430&amp;t=Y-Chromosome%20Based%20Evidence%20for%20Pre-Neolithic%20Origin%20of%20the%20Genetically%20Homogeneous%20but%20Diverse%20Sardinian%20Population%3A%20Inference%20for%20Association%20Scans" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430&title=Y-Chromosome%20Based%20Evidence%20for%20Pre-Neolithic%20Origin%20of%20the%20Genetically%20Homogeneous%20but%20Diverse%20Sardinian%20Population%3A%20Inference%20for%20Association%20Scans&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430&amp;title=Y-Chromosome%20Based%20Evidence%20for%20Pre-Neolithic%20Origin%20of%20the%20Genetically%20Homogeneous%20but%20Diverse%20Sardinian%20Population%3A%20Inference%20for%20Association%20Scans" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001430&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Y-Chromosome Based Evidence for Pre-Neolithic Origin of the Genetically Homogeneous but Diverse Sardinian Population: Inference for Association Scans';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001430';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Y-Chromosome%20Based%20Evidence%20for%20Pre-Neolithic%20Origin%20of%20the%20Genetically%20Homogeneous%20but%20Diverse%20Sardinian%20Population%3A%20Inference%20for%20Association%20Scans http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001430" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001430" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetics+of+disease%22" title="Search for articles in the subject area:'Genetics of disease'"><div class="flagText">Genetics of disease</div></a>
              <div data-categoryid="17953" data-articleid="25690"
                   data-categoryname="Genetics of disease"
                   class="flagImage" title="Flag 'Genetics of disease' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplogroups%22" title="Search for articles in the subject area:'Haplogroups'"><div class="flagText">Haplogroups</div></a>
              <div data-categoryid="43141" data-articleid="25690"
                   data-categoryname="Haplogroups"
                   class="flagImage" title="Flag 'Haplogroups' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="25690"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25690"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+genetics%22" title="Search for articles in the subject area:'Population genetics'"><div class="flagText">Population genetics</div></a>
              <div data-categoryid="32359" data-articleid="25690"
                   data-categoryname="Population genetics"
                   class="flagImage" title="Flag 'Population genetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+groupings%22" title="Search for articles in the subject area:'Population groupings'"><div class="flagText">Population groupings</div></a>
              <div data-categoryid="40421" data-articleid="25690"
                   data-categoryname="Population groupings"
                   class="flagImage" title="Flag 'Population groupings' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Type+1+diabetes%22" title="Search for articles in the subject area:'Type 1 diabetes'"><div class="flagText">Type 1 diabetes</div></a>
              <div data-categoryid="48163" data-articleid="25690"
                   data-categoryname="Type 1 diabetes"
                   class="flagImage" title="Flag 'Type 1 diabetes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Y+chromosomes%22" title="Search for articles in the subject area:'Y chromosomes'"><div class="flagText">Y chromosomes</div></a>
              <div data-categoryid="17981" data-articleid="25690"
                   data-categoryname="Y chromosomes"
                   class="flagImage" title="Flag 'Y chromosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2668'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3493'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4896&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6541">Probably Genotyping Error affects TMRCA calculation</a><br>Posted by vineviz</p>
      <p><a href="/annotation/listThread.action?root=17597">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=44233">Excellent Insight</a><br>Posted by vineviz</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
