

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Ancient Origin of the New Developmental Superfamily DANGER</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000204"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000204" />

    <meta name="citation_title" content="Ancient Origin of the New Developmental Superfamily DANGER"/>
    <meta itemprop="name" content="Ancient Origin of the New Developmental Superfamily DANGER"/>

      <meta name="citation_author" content="Nikolas Nikolaidis"/>
            <meta name="citation_author_institution" content="Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Dimitra Chalkia"/>
            <meta name="citation_author_institution" content="Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="D. Neil Watkins"/>
            <meta name="citation_author_institution" content="The Sidney Kimmel Cancer Institute, Johns Hopkins University, School of Medicine, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Roxanne K. Barrow"/>
            <meta name="citation_author_institution" content="The Solomon H. Snyder Department of Neuroscience, Johns Hopkins University, School of Medicine, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Solomon H. Snyder"/>
            <meta name="citation_author_institution" content="The Solomon H. Snyder Department of Neuroscience, Johns Hopkins University, School of Medicine, Baltimore, Maryland, United States of America"/>
            <meta name="citation_author_institution" content="Department of Pharmacology and Molecular Science, Johns Hopkins University, Baltimore, Maryland, United States of America"/>
            <meta name="citation_author_institution" content="Department of Psychiatry and Behavioral Sciences, Johns Hopkins University, Baltimore, Maryland, United States of America"/>
      <meta name="citation_author" content="Damian B. van Rossum"/>
            <meta name="citation_author_institution" content="Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America"/>
      <meta name="citation_author" content="Randen L. Patterson"/>
            <meta name="citation_author_institution" content="Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America"/>

    <meta name="citation_date" content="2007/2/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000204.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e204"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_author=SB Carroll; citation_author=JK Grenier; citation_author=SD Weatherbee; citation_number=1; citation_date=2004; citation_publisher=Blackwell Publishers; " />
      <meta name="citation_reference" content="citation_title=Gene regulatory networks and the evolution of animal body plans.; citation_author=EH Davidson; citation_author=DH Erwin; citation_journal_title=Science; citation_volume=311; citation_number=2; citation_pages=796-800; citation_date=2006; " />
      <meta name="citation_reference" content="citation_author=JC Avise; citation_number=3; citation_date=2000; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_title=DANGER, a novel regulatory protein of inositol 1,4,5-trisphosphate-receptor activity.; citation_author=DB van Rossum; citation_author=RL Patterson; citation_author=KH Cheung; citation_author=RK Barrow; citation_author=V Syrovatkina; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=4; citation_pages=37111-37116; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The mab-21 gene of Caenorhabditis elegans encodes a novel protein required for choice of alternate cell fates.; citation_author=KL Chow; citation_author=DH Hall; citation_author=SW Emmons; citation_journal_title=Development; citation_volume=121; citation_number=5; citation_pages=3615-3626; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Expression of zebrafish mab21 genes marks the differentiating eye, midbrain and neural tube.; citation_author=YM Wong; citation_author=KL Chow; citation_journal_title=Mech Dev; citation_volume=113; citation_number=6; citation_pages=149-152; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Two murine and human homologs of mab-21, a cell fate determination gene involved in Caenorhabditis elegans neural development.; citation_author=M Mariani; citation_author=D Baldessari; citation_author=S Francisconi; citation_author=L Viggiano; citation_author=M Rocchi; citation_journal_title=Hum Mol Genet; citation_volume=8; citation_number=7; citation_pages=2397-2406; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Expression profiling the developing mammalian enteric nervous system identifies marker and candidate Hirschsprung disease genes.; citation_author=TA Heanue; citation_author=V Pachnis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=8; citation_pages=6919-6924; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cell-autonomous involvement of Mab21l1 is essential for lens placode development.; citation_author=R Yamada; citation_author=Y Mizutani-Koseki; citation_author=T Hasegawa; citation_author=N Osumi; citation_author=H Koseki; citation_journal_title=Development; citation_volume=130; citation_number=9; citation_pages=1759-1770; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Requirement for Mab21l2 during development of murine retina and ventral body wall.; citation_author=R Yamada; citation_author=Y Mizutani-Koseki; citation_author=H Koseki; citation_author=N Takahashi; citation_journal_title=Dev Biol; citation_volume=274; citation_number=10; citation_pages=295-307; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phospholipase C-gamma: diverse roles in receptor-mediated calcium signaling.; citation_author=RL Patterson; citation_author=DB van Rossum; citation_author=N Nikolaidis; citation_author=DL Gill; citation_author=SH Snyder; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=11; citation_pages=688-697; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phospholipase Cgamma1 controls surface expression of TRPC3 through an intermolecular PH domain.; citation_author=DB van Rossum; citation_author=RL Patterson; citation_author=S Sharma; citation_author=RK Barrow; citation_author=M Kornberg; citation_journal_title=Nature; citation_volume=434; citation_number=12; citation_pages=99-104; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chance and necessity: the evolution of morphological complexity and diversity.; citation_author=SB Carroll; citation_journal_title=Nature; citation_volume=409; citation_number=13; citation_pages=1102-1109; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Maintenance of ancestral complexity and non-metazoan genes in two basal cnidarians.; citation_author=U Technau; citation_author=S Rudd; citation_author=P Maxwell; citation_author=PM Gordon; citation_author=M Saina; citation_journal_title=Trends Genet; citation_volume=21; citation_number=14; citation_pages=633-639; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Unexpected complexity of the Wnt gene family in a sea anemone.; citation_author=A Kusserow; citation_author=K Pang; citation_author=C Sturm; citation_author=M Hrouda; citation_author=J Lentfer; citation_journal_title=Nature; citation_volume=433; citation_number=15; citation_pages=156-160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic analysis of the Wnt gene family. Insights from lophotrochozoan members.; citation_author=B Prud'homme; citation_author=N Lartillot; citation_author=G Balavoine; citation_author=A Adoutte; citation_author=M Vervoort; citation_journal_title=Curr Biol; citation_volume=12; citation_number=16; citation_pages=1395; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=WH Li; citation_author=D Graur; citation_number=17; citation_date=1991; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=Transcriptome characterization elucidates signaling networks that control human ES cell growth and differentiation.; citation_author=R Brandenberger; citation_author=H Wei; citation_author=S Zhang; citation_author=S Lei; citation_author=J Murage; citation_journal_title=Nat Biotechnol; citation_volume=22; citation_number=18; citation_pages=707-716; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The secreted protein discovery initiative (SPDI), a large-scale effort to identify novel human secreted and transmembrane proteins: a bioinformatics assessment.; citation_author=HF Clark; citation_author=AL Gurney; citation_author=E Abaya; citation_author=K Baker; citation_author=D Baldwin; citation_journal_title=Genome Res; citation_volume=13; citation_number=19; citation_pages=2265-2270; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Proteomic, functional, and domain-based analysis of in vivo 14-3-3 binding proteins involved in cytoskeletal regulation and cellular organization.; citation_author=J Jin; citation_author=FD Smith; citation_author=C Stark; citation_author=CD Wells; citation_author=JP Fawcett; citation_journal_title=Curr Biol; citation_volume=14; citation_number=20; citation_pages=1436-1450; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Large-scale cDNA analysis reveals phased gene expression patterns during preimplantation mouse development.; citation_author=MS Ko; citation_author=JR Kitchen; citation_author=X Wang; citation_author=TA Threat; citation_author=X Wang; citation_journal_title=Development; citation_volume=127; citation_number=21; citation_pages=1737-1749; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A high throughput screen to identify secreted and transmembrane proteins involved in Drosophila embryogenesis.; citation_author=CC Kopczynski; citation_author=JN Noordermeer; citation_author=TL Serano; citation_author=WY Chen; citation_author=JD Pendleton; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=22; citation_pages=9973-9978; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Embryonic XMab21l2 expression is required for gastrulation and subsequent neural development.; citation_author=GT Lau; citation_author=OG Wong; citation_author=PM Chan; citation_author=KH Kok; citation_author=RL Wong; citation_journal_title=Biochem Biophys Res Commun; citation_volume=280; citation_number=23; citation_pages=1378-1384; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A protein-protein interaction network for human inherited ataxias and disorders of Purkinje cell degeneration.; citation_author=J Lim; citation_author=T Hao; citation_author=C Shaw; citation_author=AJ Patel; citation_author=G Szabo; citation_journal_title=Cell; citation_volume=125; citation_number=24; citation_pages=801-814; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Depletion of Mab21l1 and Mab21l2 messages in mouse embryo arrests axial turning, and impairs notochord and neural tube differentiation.; citation_author=RL Wong; citation_author=KL Chow; citation_journal_title=Teratology; citation_volume=65; citation_number=25; citation_pages=70-77; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Insertion-deletion biases and the evolution of genome size.; citation_author=TR Gregory; citation_journal_title=Gene; citation_volume=324; citation_number=26; citation_pages=15-34; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomes were forged by massive bombardments with retroelements and retrosequences.; citation_author=J Brosius; citation_journal_title=Genetica; citation_volume=107; citation_number=27; citation_pages=209-238; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Structural dynamics of eukaryotic chromosome evolution.; citation_author=EE Eichler; citation_author=D Sankoff; citation_journal_title=Science; citation_volume=301; citation_number=28; citation_pages=793-797; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mitotic recombination in Saccharomyces cerevisiae.; citation_author=F Prado; citation_author=F Cortes-Ledesma; citation_author=P Huertas; citation_author=A Aguilera; citation_journal_title=Curr Genet; citation_volume=42; citation_number=29; citation_pages=185-198; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genes in a refined Smith-Magenis syndrome critical deletion interval on chromosome 17p11.2 and the syntenic region of the mouse.; citation_author=W Bi; citation_author=J Yan; citation_author=P Stankiewicz; citation_author=SS Park; citation_author=K Walz; citation_journal_title=Genome Res; citation_volume=12; citation_number=30; citation_pages=713-728; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Non-recurrent 17p11.2 deletions are generated by homologous and non-homologous mechanisms.; citation_author=CJ Shaw; citation_author=JR Lupski; citation_journal_title=Hum Genet; citation_volume=116; citation_number=31; citation_pages=1-7; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Complex chromosome 9, 20, and 22 rearrangements in acute lymphoblastic leukemia with duplication of BCR and ABL sequences.; citation_author=MJ Stevens-Kroef; citation_author=R Dirckx; citation_author=LE Meers; citation_author=JC Albrechts; citation_author=HC Schouten; citation_journal_title=Cancer Genet Cytogenet; citation_volume=116; citation_number=32; citation_pages=119-123; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SH2-B and APS are multimeric adapters that augment TrkA signaling.; citation_author=X Qian; citation_author=DD Ginty; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=33; citation_pages=1613-1620; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Post-transcriptional silencing of RET occurs, but is not required, during raf-1 mediated differentiation of medullary thyroid carcinoma cells.; citation_author=EB Carson-Walter; citation_author=DP Smith; citation_author=BA Ponder; citation_author=SB Baylin; citation_author=BD Nelkin; citation_journal_title=Oncogene; citation_volume=17; citation_number=34; citation_pages=367-376; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Modulation of the Ras/Raf/MEK/ERK pathway by Ca(2+), and calmodulin.; citation_author=N Agell; citation_author=O Bachs; citation_author=N Rocamora; citation_author=P Villalonga; citation_journal_title=Cell Signal; citation_volume=14; citation_number=35; citation_pages=649-654; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolution of key cell signaling and adhesion protein families predates animal origins.; citation_author=N King; citation_author=CT Hittinger; citation_author=SB Carroll; citation_journal_title=Science; citation_volume=301; citation_number=36; citation_pages=361-363; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Concerted and birth-and-death evolution of multigene families.; citation_author=M Nei; citation_author=AP Rooney; citation_journal_title=Annu Rev Genet; citation_volume=39; citation_number=37; citation_pages=121-152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.; citation_author=SF Altschul; citation_author=TL Madden; citation_author=AA Schaffer; citation_author=J Zhang; citation_author=Z Zhang; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=38; citation_pages=3389-3402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=39; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Computational inference of homologous gene structures in the human genome.; citation_author=RF Yeh; citation_author=LP Lim; citation_author=CB Burge; citation_journal_title=Genome Res; citation_volume=11; citation_number=40; citation_pages=803-816; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=41; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees.; citation_author=N Saitou; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=4; citation_number=42; citation_pages=406-425; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform; citation_volume=5; citation_number=43; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing.; citation_author=HA Schmidt; citation_author=K Strimmer; citation_author=M Vingron; citation_author=HA von; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=44; citation_pages=502-504; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=45; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Quartet-based phylogenetic inference: improvements and limits.; citation_author=V Ranwez; citation_author=O Gascuel; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=46; citation_pages=1103-1116; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services.; citation_author=RD Finn; citation_author=J Mistry; citation_author=B Schuster-Bockler; citation_author=S Griffiths-Jones; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=47; citation_pages=D247-D251; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CDD: a Conserved Domain Database for protein classification.; citation_author=A Marchler-Bauer; citation_author=JB Anderson; citation_author=PF Cherukuri; citation_author=C Weese-Scott; citation_author=LY Geer; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=48; citation_pages=D192-D196; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=I Korf; citation_author=M Yandell; citation_author=J Bedell; citation_number=49; citation_date=2003; citation_publisher=O'Reilly; " />
      <meta name="citation_reference" content="citation_title=Improved prediction of signal peptides: SignalP 3.0.; citation_author=JD Bendtsen; citation_author=H Nielsen; citation_author=HG von; citation_author=S Brunak; citation_journal_title=J Mol Biol; citation_volume=340; citation_number=50; citation_pages=783-795; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes.; citation_author=A Krogh; citation_author=B Larsson; citation_author=HG von; citation_author=EL Sonnhammer; citation_journal_title=J Mol Biol; citation_volume=305; citation_number=51; citation_pages=567-580; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Human-mouse alignments with BLASTZ.; citation_author=S Schwartz; citation_author=WJ Kent; citation_author=A Smit; citation_author=Z Zhang; citation_author=R Baertsch; citation_journal_title=Genome Res; citation_volume=13; citation_number=52; citation_pages=103-107; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=zPicture: dynamic alignment and visualization tool for analyzing conservation profiles.; citation_author=I Ovcharenko; citation_author=GG Loots; citation_author=RC Hardison; citation_author=W Miller; citation_author=L Stubbs; citation_journal_title=Genome Res; citation_volume=14; citation_number=53; citation_pages=472-477; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PipMaker–a web server for aligning two genomic DNA sequences.; citation_author=S Schwartz; citation_author=Z Zhang; citation_author=KA Frazer; citation_author=A Smit; citation_author=C Riemer; citation_journal_title=Genome Res; citation_volume=10; citation_number=54; citation_pages=577-586; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=WAViS server for handling, visualization and presentation of multiple alignments of nucleotide or amino acids sequences.; citation_author=R Zika; citation_author=J Paces; citation_author=A Pavlicek; citation_author=V Paces; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=55; citation_pages=W48-W49; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator.; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Genome Res; citation_volume=14; citation_number=56; citation_pages=1188-1190; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Neurite outgrowth-regulating properties of GABA and the effect of serum on mouse spinal cord neurons in culture.; citation_author=M Bird; citation_author=A Owen; citation_journal_title=J Anat; citation_volume=193; citation_number=57; citation_pages=503-508; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Origin of the metazoan phyla: molecular clocks confirm paleontological estimates.; citation_author=FJ Ayala; citation_author=A Rzhetsky; citation_author=FJ Ayala; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=58; citation_pages=606-611; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Estimating metazoan divergence times with a molecular clock.; citation_author=KJ Peterson; citation_author=JB Lyons; citation_author=KS Nowak; citation_author=CM Takacs; citation_author=MJ Wargo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=59; citation_pages=6536-6541; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Ancient Origin of the New Developmental Superfamily DANGER"/>
    <meta name="twitter:description" content="Developmental proteins play a pivotal role in the origin of animal complexity and diversity. We report here the identification of a highly divergent developmental protein superfamily (DANGER), which originated before the emergence of animals (&sim;850 million years ago) and experienced major expansion-contraction events during metazoan evolution. Sequence analysis demonstrates that DANGER proteins diverged via multiple mechanisms, including amino acid substitution, intron gain and/or loss, and recombination. Divergence for DANGER proteins is substantially greater than for the prototypic member of the superfamily (Mab-21 family) and other developmental protein families (e.g., WNT proteins).&nbsp;DANGER proteins are widely expressed and display species-dependent tissue expression patterns, with many members having roles in development. DANGER1A, which regulates the inositol trisphosphate receptor, promotes the differentiation and outgrowth of neuronal processes. Regulation of development may be a universal function of DANGER family members. This family provides a model system to investigate how rapid protein divergence contributes to morphological complexity."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000204.g006"/>

  <meta property="og:title" content="Ancient Origin of the New Developmental Superfamily DANGER" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5203'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7394'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3972&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000204">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000204" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000204&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Ancient Origin of the New Developmental Superfamily DANGER
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nikolas Nikolaidis
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nikolas Nikolaidis, Dimitra Chalkia</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rlp25@psu.edu">rlp25@psu.edu</a> (RP); <a href="mailto:nxn7@psu.edu">nxn7@psu.edu</a> (NN)</p>

                <p>Affiliation:
                  Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dimitra Chalkia
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nikolas Nikolaidis, Dimitra Chalkia</p>

              
              

                <p>Affiliation:
                  Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            D. Neil Watkins, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Sidney Kimmel Cancer Institute, Johns Hopkins University, School of Medicine, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roxanne K. Barrow, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  The Solomon H. Snyder Department of Neuroscience, Johns Hopkins University, School of Medicine, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Solomon H. Snyder, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  The Solomon H. Snyder Department of Neuroscience, Johns Hopkins University, School of Medicine, Baltimore, Maryland, United States of America, 
                  Department of Pharmacology and Molecular Science, Johns Hopkins University, Baltimore, Maryland, United States of America, 
                  Department of Psychiatry and Behavioral Sciences, Johns Hopkins University, Baltimore, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Damian B. van Rossum, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Randen L. Patterson
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rlp25@psu.edu">rlp25@psu.edu</a> (RP); <a href="mailto:nxn7@psu.edu">nxn7@psu.edu</a> (NN)</p>

                <p>Affiliation:
                  Biology Department, Pennsylvania State University, University Park, Pennsylvania, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000204</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000204-g001" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000204-g002" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000204-g003" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000204-g004" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000204-g005" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000204-g006" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000204">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000204" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Developmental proteins play a pivotal role in the origin of animal complexity and diversity. We report here the identification of a highly divergent developmental protein superfamily (DANGER), which originated before the emergence of animals (~850 million years ago) and experienced major expansion-contraction events during metazoan evolution. Sequence analysis demonstrates that DANGER proteins diverged via multiple mechanisms, including amino acid substitution, intron gain and/or loss, and recombination. Divergence for DANGER proteins is substantially greater than for the prototypic member of the superfamily (Mab-21 family) and other developmental protein families (e.g., WNT proteins).&nbsp;DANGER proteins are widely expressed and display species-dependent tissue expression patterns, with many members having roles in development. DANGER1A, which regulates the inositol trisphosphate receptor, promotes the differentiation and outgrowth of neuronal processes. Regulation of development may be a universal function of DANGER family members. This family provides a model system to investigate how rapid protein divergence contributes to morphological complexity.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Nikolaidis N, Chalkia D, Watkins DN, Barrow RK, Snyder SH, et al.  (2007) Ancient Origin of the New Developmental Superfamily DANGER. PLoS ONE 2(2):
          e204.
            doi:10.1371/journal.pone.0000204</p><p><strong>Academic Editor: </strong>Emmanouil Dermitzakis, The Wellcome Trust Sanger Institute, United Kingdom</p><p><strong>Received:</strong> November 13, 2006; <strong>Accepted:</strong> January 14, 2007; <strong>Published:</strong> February 14, 2007</p><p><strong>Copyright:</strong> © 2007 Nikolaidis et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the SEARLE award and start-up monies from PSU (RLP), a Fellowship from the Eberly College of Sciences and the Huck Institute (DVR), USPHS grant Conte Center grant MH 68830 and MH 18501 (SHS) and Research Scientist Award DA 00074 (SHS).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A central question in evolutionary developmental biology is how genetic diversity and complexity have contributed to the origin and the remarkable diversification of the animal body plan. Development of the animal body plan is controlled by large gene regulatory networks, which consist of transcription factors, their target cis-regulatory modules, and signaling molecules <a href="#pone.0000204-Carroll1">[1]</a>, <a href="#pone.0000204-Davidson1">[2]</a>. As opposed to the early stages of body plan formation which involve specification of developing components and their spatial patterning, the later stages of development involve cellular differentiation <a href="#pone.0000204-Davidson1">[2]</a>. Genes that promote cellular differentiation display high levels of sequence divergence and have contributed to morphological complexity <a href="#pone.0000204-Davidson1">[2]</a>, <a href="#pone.0000204-Avise1">[3]</a>. Therefore, identifying new genes involved in cell differentiation and characterizing their evolutionary history is fundamentally important towards understanding the origin of animal complexity and diversity.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>We recently discovered the inositol 1,4,5-triphosphate receptor (IP<sub>3</sub>R) binding protein DANGER, which is an allosteric regulator of IP<sub>3</sub>R Ca<sup>2+</sup> dependence <a href="#pone.0000204-vanRossum1">[4]</a>. The human (<em>Homo sapiens</em>; Hs) DANGER protein consists of 547 amino acids and is predicted to contain a small portion of the Male abnormal 21 (Mab-21) domain <a href="#pone.0000204-vanRossum1">[4]</a>. The Mab-21 domain was first reported in the nematode <em>Caenorhabditis elegans</em> 
            <a href="#pone.0000204-Chow1">[5]</a> and has been defined as a domain of 360 amino acids. Proteins containing the Mab-21 domain are members of a small and highly conserved family with single genes in invertebrate and two genes in vertebrate species <a href="#pone.0000204-Wong1">[6]</a>, <a href="#pone.0000204-Mariani1">[7]</a>. Members of the MAB-21 family control cellular differentiation in <em>C. elegans</em>, <em>Danio rerio</em>, and <em>Mus musculus</em> 
            <a href="#pone.0000204-Chow1">[5]</a>–<a href="#pone.0000204-Yamada2">[10]</a>. When MAB-21 members are genetically deleted, gross morphological changes in developing animals ensue <a href="#pone.0000204-Chow1">[5]</a>–<a href="#pone.0000204-Yamada2">[10]</a>. The Gestalt Domain Detection Algorithm (GDDA) is a “seeding” algorithm which can identify highly divergent domains <a href="#pone.0000204-Patterson1">[11]</a>, <a href="#pone.0000204-vanRossum2">[12]</a>. GDDA analysis predicts that DANGER contains almost 90% of the Mab-21 domain sequence (<a href="#pone.0000204.s001">Figure S1</a>).</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Identification and phylogenetic analysis of the DANGER proteins</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We used the HsMAB21L1 and HsDANGER proteins as queries to search several species databases for similar protein sequences. This analysis revealed the presence of multiple metazoan proteins, denoted as DANGER (D) proteins (<a href="#pone.0000204.s016">Table S1</a>). Phylogenetic analyses of these sequences classify them into six families (D1–D6; <a href="#pone-0000204-g001">Figure 1A</a>). In our analysis the previously reported <a href="#pone.0000204-Wong1">[6]</a> highly conserved MAB-21 protein family is extended, and herein is denoted as D6. A unique DANGER sequence from the unicellular choanoflagellate species <em>Monosiga ovata</em> assumes an outgroup position (<a href="#pone-0000204-g001">Figure 1A</a> and <a href="#pone.0000204.s002">Figure S2</a>). The sea anemone <em>Nematostella vectensis</em> (Anthozoa, Cnidaria) is a phylogenetically basal animal which is composed of only 11 cell types <a href="#pone.0000204-Carroll2">[13]</a>. Notably, this species contains multiple DANGER sequences which cluster with different DANGER families, indicating that the DANGER superfamily was expanded and already diversified in the cnidarian-bilaterian ancestor (<a href="#pone-0000204-g001">Figure 1A</a> and <a href="#pone.0000204.s002">Figure S2</a>). <em>C. elegans</em> and <em>D. melanogaster</em> genomes likely lost many DANGER genes, suggesting a contraction of the DANGER superfamily in these protostomes; an evolutionary pattern also documented for the developmentally important WNT superfamily <a href="#pone.0000204-Technau1">[14]</a>. In deuterostomes (e.g. echinoderms and chordates), additional gene duplication events led to the emergence of new DANGER families.</p>
<div class="figure" id="pone-0000204-g001"><div class="img"><a name="pone-0000204-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>DANGER proteins originated early in metazoan evolution and encode a highly divergent MAB-21 domain.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Neighbor-joining (NJ) consensus tree of the DANGER superfamily defines orthologous relationships among DANGER sequences (D1–D6) from vertebrates (Dr, <em>Danio rerio</em>; Xl, <em>Xenopus laevis</em>; Xt, <em>Xenopus tropicalis</em>; Gg, <em>Gallus gallus</em>; Rn, <em>Rattus norvegicus</em>; Mm, <em>Mus musculus</em>; Hs, <em>Homo sapiens</em>), invertebrates (Ci, <em>Ciona intestinalis</em>; Sp, <em>Strongylocentrotus purpuratus</em>; Dm, <em>Drosophila melanogaster</em>; Ce, <em>Caenorhabditis elegans</em>; Cb, <em>Caenorhabditis briggsae</em>, Nv, <em>Nematostella vectensis</em>), and the choanoflagellate (Mo, <em>Monosiga ovata</em>). Numbers at branches are bootstrap values from the NJ analysis. Bootstrap and quartet puzzling support values from maximum parsimony (MP) and maximum likelihood (TREE-PUZZLE) analyses, respectively, are given in parentheses. (B) Domain prediction analysis using the Pfam Mab-21 profile, containing four sequences, identifies a partial Mab-21 domain (showed as red boxes) in the human DANGER (HsD1-5) proteins. Predicted signal peptides and transmembrane regions are depicted by green and blue boxes, respectively. (C) Domain prediction analysis, using a Mab-21 profile generated by psi-BLAST containing 63 animal sequences, results in Mab-21 domain extension in all human DANGER proteins. (D) Quantification of Mab-21 domain coverage as predicted by the Pfam and the psiBLAST-generated Mab-21 profiles in all human DANGER proteins. Similar Mab-21 domain extension is observed between orthologous DANGER sequences from other species.</p>
<span>doi:10.1371/journal.pone.0000204.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>With the exception of some anthozoan sequences, the orthologous DANGER groups are well defined. In contrast, relationships among paralogous groups remain ambiguous even when different protein regions, sequence datasets, and phylogenetic methods (Neighbor-Joining, Maximum Parsimony, and Maximum Likelihood) are used (<a href="#pone-0000204-g001">Figure 1A</a> and <a href="#pone.0000204.s002">Figure S2</a>). Genetic distances produced by several independent methods (data not shown) and the multiple sequence alignment (<a href="#pone.0000204.s003">Figure S3</a>) further demonstrate the divergence of paralogous DANGER sequences. This pattern of evolution resembles that of the WNT superfamily, in which the relationships among paralogous groups are also unclear <a href="#pone.0000204-Kusserow1">[15]</a>, <a href="#pone.0000204-Prudhomme1">[16]</a>.</p>


<h4>Domain analysis of the DANGER proteins</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Reverse position specific (rps)-BLAST similarity searches (<a href="#pone-0000204-g001">Figure 1B</a>) reveal that most DANGER sequences encode only a C-terminal portion of the Mab-21 domain. However, the Mab-21 domain consensus sequence (profile) in the Conserved Domain Database (CDD) is composed of only four sequences from the highly conserved D6 family. Therefore, we generated a new Mab-21 profile utilizing all D1–6 animal sequences (<a href="#pone.0000204.s016">table S1</a>), except the anthozoan sequences (see <a href="#s3">Materials and Methods</a> section). This analysis predicts that all DANGER proteins contain more than 55% of the Mab-21 domain and most of them score higher than 70% (<a href="#pone-0000204-g001">Figure 1C, 1D</a>, and <a href="#pone.0000204.s004">Figure S4</a>, <a href="#pone.0000204.s005">S5</a>).</p>


<h4>Evolution of the Mab-21 domain sequence</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To identify functionally important regions in the Mab-21 domain sequence, we calculated the <em>p</em>-distances across the multiple sequence alignment (MSA) of representative DANGER proteins (<a href="#pone-0000204-g002">Figure 2A</a>). This analysis shows the Mab-21 domain sequence is highly divergent among the paralogous DANGER sequences. In addition, the observed sharp changes in the <em>p</em>-distances along the MSA suggest that the Mab-21 domain can be roughly divided into two regions: the highly diverged N-terminus (positions in MSA 300–750) and the more conserved C-terminus (positions in MSA 751–900) (<a href="#pone-0000204-g002">Figure 2A</a>). Taking into account the presence of long insertions and/or deletions (indels) along the MSA we divided the Mab-21 domain into nine microdomains (<a href="#pone-0000204-g002">Figure 2B</a>) and calculated the <em>p</em>-distances for each microdomain (<a href="#pone-0000204-g002">Figure 2C</a>). The most conserved regions of the Mab-21 domain are microdomains VI–VIII (<a href="#pone-0000204-g002">Figure 2</a> and <a href="#pone.0000204.s006">Figure S6</a>, <a href="#pone.0000204.s007">S7</a>). We hypothesize that microdomains VI–VIII represent conserved functional units of the DANGER proteins, while microdomain I may provide temporal and/or spatial specificity.</p>
<div class="figure" id="pone-0000204-g002"><div class="img"><a name="pone-0000204-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Patterns of sequence evolution along the Mab-21 domain.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Graphical representation of the <em>p</em>-distances calculated along the multiple sequence alignment of representative DANGER proteins with a sliding window of 100 amino acids and a step of 50 amino acids. Bars represent standard errors of mean <em>p</em>-distance values. (B) Multiple sequence alignment of representative DANGER sequences (D1–D6) in block format. Microdomains II to IX are colored according to the mean value of the proportion of amino acid differences (<em>p</em>-distance) among all sequences. Intron positions are mapped as yellow arrows. (B) Graphical representation of the <em>p</em>-distances for each microdomain. Bars represent standard errors of mean <em>p</em>-distance values. (D) Pattern of sequence conservation (logo) along the three most conserved Mab-21 microdomains (VI–VIII).</p>
<span>doi:10.1371/journal.pone.0000204.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Microdomain I sequences are highly diverged and can be unambiguously aligned only among orthologous sequences (<a href="#pone.0000204.s003">Figure S3</a>). This microdomain in D1A–C, D2B, and D3A is predicted to encode signal peptides and/or transmembrane regions (see <a href="#s3">Materials and Methods</a> section) (<a href="#pone-0000204-g001">Figure 1B and 1C</a>). Our phylogenetic analysis suggests that the common ancestor of vertebrates contained all three DANGER gene-lineages encoding localization signals (<a href="#pone-0000204-g001">Figure 1A</a> and <a href="#pone.0000204.s002">Figure S2</a>). Coupling the aforementioned observations with the presence of introns at the C-terminus of microdomain I sequences (<a href="#pone-0000204-g002">Figure 2B</a>), we hypothesize that DANGER sequences have acquired microdomain I through exon-shuffling <a href="#pone.0000204-Li1">[17]</a> early in vertebrate evolution. This acquisition likely enabled some DANGER proteins to function in different cellular compartments or even to be secreted <a href="#pone.0000204-Brandenberger1">[18]</a>–<a href="#pone.0000204-Wong2">[25]</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To gain insight into the evolutionary processes that shaped the DANGER sequences, we examined the location of the indels within the Mab-21 domain sequence. For this purpose we utilized the pairwise alignments between the psi-BLAST-generated Mab-21 profile and each DANGER sequence. In order to compare these alignments, all DANGER sequences were mapped onto the Mab-21 profile sequence according to the pairwise alignment coordinates. This analysis reveals several conserved indels in both orthologous and paralogous groups (<a href="#pone-0000204-g003">Figure 3</a> and <a href="#pone.0000204.s008">Figure S8</a>, <a href="#pone.0000204.s009">S9</a>). Indels conserved among orthologous sequences suggest that these occurred in the ancestral sequence of a particular group. Conservation of indels between paralogous sequences, from anthozoa to mammals, suggests that these indels occurred in the common ancestor of the DANGER sequences very early in metazoan evolution (<a href="#pone.0000204.s009">Figure S9</a>, <a href="#pone.0000204.s010">S10</a>). Several conserved indels tend to coincide with intron positions (<a href="#pone-0000204-g003">Figure 3</a> and <a href="#pone.0000204.s008">Figure S8</a>, <a href="#pone.0000204.s009">S9</a>). Taking into account that frequent intron gain and/or loss occurred during DANGER evolution (<a href="#pone.0000204.s002">Figure S2</a>), we speculate that intron mobility could have been responsible for the fragmentation of the Mab-21 domain sequence. This speculation is supported by similar observations within the Wnt domain sequence (data not shown).</p>
<div class="figure" id="pone-0000204-g003"><div class="img"><a name="pone-0000204-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Evolutionary patterns of the Mab-21 domain sequence.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Rps-BLAST pairwise alignments between the psi-BLAST generated Mab-21 profile and DANGER sequences. Comparison of insertions (in profile sequence; black boxes) and deletions (in profile sequence; white boxes), and correspondence of indels (&gt;3 amino acids) with intron positions (yellow boxes). For comparison, all DANGER sequences are mapped onto the Mab-21 profile sequence according to pairwise alignment coordinates. The microdomains II–VIII are also shown. Species abbreviations are as in <a href="#pone-0000204-g001">Figure 1A</a>.</p>
<span>doi:10.1371/journal.pone.0000204.g003</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>Intron gain and/or loss cannot explain the presence of all observed indels. These additional indels could have resulted from DNA polymerase slippage <a href="#pone.0000204-Gregory1">[26]</a>. However, this is unlikely, as indels identified were usually longer than nine nucleotides (3 amino acids). Such long indels have been identified only in non-coding regions and have been attributed to recombination events followed by unequal cross-over <a href="#pone.0000204-Gregory1">[26]</a>. Thus, the indels in the Mab-21 domain sequence could have been the result of recombination. To test this hypothesis, we used comparative genomic analyses and found that in many cases DANGER genes are in conserved synteny among all species studied (<a href="#pone.0000204.s010">Figure S10</a>). In addition, the genomic regions flanking DANGER genes contain a high number of transposable elements (TEs), which are conserved among paralogous but not orthologous sequences (<a href="#pone.0000204.s011">Figure S11</a>, <a href="#pone.0000204.s012">S12</a>). The latter observation indicates that within <em>H. sapiens</em>, <em>M. musculus</em>, and <em>D. rerio</em> genomes the TEs flanking DANGER genes have been multiplied in each genome independently through different mechanisms including recombination (<a href="#pone.0000204.s011">Figure S11</a> and <a href="#pone.0000204.s012">S12</a>) <a href="#pone.0000204-Brosius1">[27]</a>–<a href="#pone.0000204-Prado1">[29]</a>. Thus, we hypothesize that recombination among TEs could provide an alternative explanation for the presence of indels along the Mab-21 domain sequence. Three findings support this notion. (A) The HsD3B is a candidate gene for the Smith-Magenis syndrome, which involves chromosome deletion through recombination among repeats <a href="#pone.0000204-Bi1">[30]</a>, <a href="#pone.0000204-Shaw1">[31]</a>. (B) The HsD4 is flanked by numerous repeats similar to these flanking the breakpoint cluster region (BCR) gene (<a href="#pone.0000204.s011">Figure S11</a>). The human BCR gene is located at the site of the translocation breakpoint found in chronic myeloid leukemia <a href="#pone.0000204-StevensKroef1">[32]</a>, which like Smith-Magenis region is a “hot-spot” for recombination. (C) The mRNA sequences of D1C, D3A, D4 and D5 genes in human and mouse contain TEs (<a href="#pone.0000204.s017">table S2</a>). These TEs within DANGER genes are likely the result of recent independent transpositions, since orthologous genetic regions contain different classes of repeats (<a href="#pone.0000204.s017">table S2</a>). Overall, these observations indicate that TEs have contributed to the evolution of DANGER gene sequences and support the notion that recombination events among TEs probably contributed to the genetic variation of DANGER sequences during vertebrate evolution.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In the morphologically simple animal <em>N. vectensis</em>, DANGER gene sequences encode either a complete or a partial Mab-21 domain (<a href="#pone.0000204.s005">Figure S5</a> and <a href="#pone.0000204.s008">S8</a>). In addition, the genetic distances among all <em>N. vectensis</em> DANGER sequences are rather high (<a href="#pone.0000204.s002">Figure S2</a>). Furthermore, these genes reside in genomic regions that contain many TEs (data not shown). The above observations suggest that both complete and partial Mab-21 domains likely existed before the emergence of bilateria and that recombination among TEs could have been responsible for the increased variation among anthozoan DANGER paralogs.</p>


<h4>Function of DANGER proteins in development</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>D6 proteins have been implicated in development and cellular differentiation. During <em>C. elegans</em> development, the Mab-21 protein is required for the choice of alternate cell fates and the formation of sensory organs in the male nematode tail <a href="#pone.0000204-Chow1">[5]</a>. Mutations of the <em>mab-21</em> gene affect movement, body shape, and fertility <a href="#pone.0000204-Chow1">[5]</a>. In <em>D. rerio</em> D6 genes were found to be highly expressed in the differentiating eye, the midbrain, and the neural tube <a href="#pone.0000204-Wong1">[6]</a>. In <em>Xenopus laevis</em> D6 proteins are required for the completion of gastrulation and neuronal development <a href="#pone.0000204-Lau1">[23]</a>. In <em>M. musculus</em> the expression of D6 genes is important for the development of the embryonic brain, eye, limbs, and neural crest derivatives. In particular, D6 deficient mice show defects in eye formation (absence of lens), notochord and neural tube development, organogenesis, and axial turning, resulting in death at mid-gestational stage <a href="#pone.0000204-Yamada1">[9]</a>, <a href="#pone.0000204-Yamada2">[10]</a>, <a href="#pone.0000204-Wong2">[25]</a>. We searched the PUBMED database to retrieve entries associated with the remaining DANGER proteins and found many members within high-throughput studies. In these reports, DANGER proteins from all six families are described within large developmental gene and protein networks <a href="#pone.0000204-Brandenberger1">[18]</a>, <a href="#pone.0000204-Jin1">[20]</a>–<a href="#pone.0000204-Kopczynski1">[22]</a>, <a href="#pone.0000204-Lim1">[24]</a>. Although, these studies do not provide a specific function for each DANGER protein, they suggest that DANGER proteins seem to be involved in a variety of developmental processes.</p>


<h4>Expression of DANGER genes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To identify the expression profile of the DANGER genes over different tissues and developmental stages, we utilized the expression (EST) data from the UniGene database for human and mouse paralogs (<a href="#pone.0000204.s013">Figure S13</a>). We found that DANGER sequences are differentially expressed in several tissues at various developmental stages (<a href="#pone.0000204.s013">Figure S13</a>) suggesting functional divergence between paralogous DANGER sequences. In addition, we compared the expression levels among orthologous DANGER pairs in comparable human and mouse tissues (<a href="#pone-0000204-g004">Figure 4</a>). In all cases the observed differences between the orthologous pairs are statistically significant (chi-square test, α = 0.01), suggesting that DANGER genes exhibit species specific expression levels.</p>
<div class="figure" id="pone-0000204-g004"><div class="img"><a name="pone-0000204-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Analysis of expressed sequence tags (ESTs) show that mouse and human DANGER sequences exhibit different transcriptional patterns.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Upper: Plot depicting the expression levels of the mouse DANGER sequences in different tissues. Lower: Plot depicting the expression levels of the human DANGER sequences in different tissues. (Additional information and the Unigene identification numbers for each gene used in this analysis can be found in <a href="#pone.0000204.s013">Figure S13</a>).</p>
<span>doi:10.1371/journal.pone.0000204.g004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We then tested the expression of the human D1A gene by northern analysis (<a href="#pone.0000204.s014">Figure S14</a>) and found it to be widely expressed agreeing with the EST analysis (<a href="#pone.0000204.s013">Figure S13</a>). We next performed immunohistochemistry on mouse embryos at embryonic age E13.5 (13.5 days post conception) and observed that D1A primarily expressed in the spinal cord. In contrast, mouse embryos at embryonic age E18 showed expression throughout the body (<a href="#pone.0000204.s014">Figure S14</a>). In addition, we examined adult mouse sections and determined that D1A is highly expressed in a number of terminally differentiated cells such as myotubes in skeletal muscle and crypt cells in the small intestine (<a href="#pone.0000204.s014">Figure S14</a>).</p>


<h4>DANGER1A is involved in cell differentiation</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>As D1A is expressed in terminally differentiated neurons (<a href="#pone.0000204.s014">Figure S14</a>), we used PC12 rat adrenal gland pheochromocytoma cells as a model system for cellular differentiation. This cell line can be induced to differentiate by application of nerve growth factor (NGF). Differentiated PC12 cells can be defined as possessing neurite extensions ≥2.5 times the length of the cell body <a href="#pone.0000204-Qian1">[33]</a>. PC12 cells overexpressing D1A display a ~3-fold increase in neurite length in response to low concentrations of NGF at early time points (<a href="#pone-0000204-g005">Figure 5A and 5B</a>). Inversely, PC12 cells depleted of D1A using siRNA display lower levels of differentiation at late time-points (~50% reduction at 7 days).</p>
<div class="figure" id="pone-0000204-g005"><div class="img"><a name="pone-0000204-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>DANGER1A increases neurite outgrowth in response to NGF signaling.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Top: 40× Phase picture and 485 nm emission of PC12 cells transfected with YFP-alone or YFP+MYC-tagged D1A stimulated with NGF (3 ng/ml 3 days). PC12 cells expressing D1A have increased neurite outgrowth. Bottom: Phase picture and 485 nm emission of PC12 cells transfected with YFP-alone or YFP+siRNA against rat D1A stimulated with NGF (3 ng/ml 7 days). PC12 cells depleted of endogenous D1A using siRNA have decreased neurite outgrowth. (B) Quantification of neurite outgrowth in PC12 and TT cells. (100 cells counted in 3 independent experiments) (C) Left: Immunofluorescent staining of mouse primary spinal cord neurons with Alexa488-conjugated DANGER1A antibody. Inset: Quantification of spinal cord neurite length in neurons overexpressing YFP alone or YFP+D1A. (D) Western analysis of D1A expression in the p1 (nuclear), p2 (heavy ER and mitochondria), p3(light ER and vesicles), or s3 (cytosol) cell fractions over a time course of 48 h stimulation with NFG (10 ng/ml). These individual blots were developed simultaneously on the same film. The numbers inside the blots quantify the change in D1A distribution over time course (Error: SEM; p&lt;0.01 n = 4). (E) Right: Western analysis of the MAP-kinase proteins ERK1/2, phospho-ERK1/2, Raf, phospho-Raf, and D1A levels in PC12 cells transfected with either YFP, YFP+D1A, or YFP+siRNA D1A. Cells were treated with vehicle (water), or 3 ng/ml NGF for 48 hours. Left: Quantification of changes in total and phospho-ERK1/2 and total D1A by scanning densitometry. All values are expressed as fold-change vs. control; ERK-(Error: SEM; p&lt;0.05 n = 3); D1A-(Error: SEM; p&lt;0.01 n = 3)</p>
<span>doi:10.1371/journal.pone.0000204.g005</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We have shown that the most conserved regions of the Mab-21 domain are microdomains VI–VIII (<a href="#pone-0000204-g002">Figure 2</a> and <a href="#pone.0000204.s006">Figure S6</a>, <a href="#pone.0000204.s007">S7</a>) and we have predicted that these domains may represent functional units within the Mab-21 domain. To test this prediction we engineered a deletion in D1A (amino acids 440–553; microdomains VIII–IX) and we assessed neurite outgrowth in PC12 cells. We found that <sup>Δ</sup>C-termD1A is unable to promote NGF-induced neurite outgrowth in PC12 cells (<sup>Δ</sup>C-termD1A; <a href="#pone-0000204-g005">Figure 5B</a>), supporting our original prediction.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>In addition to PC12 cells that respond to NGF, we used the human thyroid-derived neuroendocrine tumor (TT) cells which are defective in NGF-signaling, although they express cell-surface tyrosine kinase A (Trk-A) NGF receptors and can manifest neurites following transfection with constitutively active serine/threonine protein kinase Raf <a href="#pone.0000204-CarsonWalter1">[34]</a>. While TT cells express the full-length transcript of D1A as demonstrated by RT-PCR (data not shown), these cells do not express the full-length D1A protein (<a href="#pone.0000204.s015">Figure S15</a>) <a href="#pone.0000204-vanRossum1">[4]</a>. Therefore, we examined the influence of rescued-overexpression of full-length D1A on neurite extension in TT cells (<a href="#pone-0000204-g005">Figure 5B</a> and <a href="#pone.0000204.s015">Figure S15</a>). D1A overexpression induced neurite outgrowth in a small population of TT cells at very low levels of NGF (3 ng/ml), and provided a 10-fold increase in neurite extension when cells were treated with 50 ng/ml NGF. By contrast, NGF-stimulation (3 ng/ml 10 days) of TT cells overexpressing YFP-alone (control) did not cause neurite extension, while higher concentrations of NGF (50 ng/ml) induced neurite outgrowth in a small but reproducible population of control cells (~2%). As p75 has a low affinity for NGF, we presume that D1A is activated predominately by Trk-A receptors at the concentrations utilized.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>Finally, to obtain physiological relevance of the above results, we overexpressed D1A and assessed neurite length in primary spinal cord neurons. In 7 day cultures, we observed a ~2.5 fold increase in neurite outgrowth (<a href="#pone-0000204-g005">Figure 5C</a>), signifying the ability of D1A to participate in cell differentiation pathways.</p>


<h4>DANGER1A functions in the NGF pathway</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We next examined whether endogenous D1A expression and localization could be influenced by NGF treatment. In PC12 cells, we observed a biphasic response of D1A expression and localization, with increased expression 24 h post-treatment (<a href="#pone-0000204-g005">Figure 5D</a>). We observe a transient depletion of D1A at 4 hr post-NGF treatment with increasing expression after 8 hrs (n = 4). Following, we examined whether D1A expression could affect two NGF-linked kinases, Raf and the mitogen-activated protein kinase (ERK1/2) (<a href="#pone-0000204-g005">Figure 5E</a>). In differentiated PC12 cells, overexpression or siRNA-depletion of D1A did not affect Raf protein levels or phosphorylation at the time-points tested. Conversely, in PC12 cells overexpressing D1A the phosphorylation levels of ERK1/2 were increased, while protein levels were unaltered. Compared to NGF-stimulated PC12 control cells, siRNA-depletion of D1A did not affect the levels of ERK1/2 phosphorylation. However, full depletion of D1A via siRNA was not possible as NGF-treatment alone augments D1A expression. These data fit well with our recent evidence that D1A can influence IP<sub>3</sub>R mediated Ca<sup>2+</sup> activity, which is an intermediary of Trk-A receptor activation and ERK1/2 activity <a href="#pone.0000204-Agell1">[35]</a>. Our functional studies, together with previous reports <a href="#pone.0000204-Chow1">[5]</a>–<a href="#pone.0000204-Yamada2">[10]</a>, <a href="#pone.0000204-Brandenberger1">[18]</a>, <a href="#pone.0000204-Jin1">[20]</a>–<a href="#pone.0000204-Wong2">[25]</a> demonstrate that DANGER proteins have the capacity to function in developmental processes.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The phylogenetic grouping, similar domain content, expression profiles, and general activity in developmental pathways demonstrate that DANGER proteins comprise a new developmental superfamily. This superfamily likely remained undiscovered due to its high sequence divergence (mean <em>p</em>-distance among paralogs is 0.756±0.009), which far exceeds the divergence among the well studied WNT proteins (mean <em>p</em>-distance of WNT domains 0.561±0.016).</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Our findings demonstrate that the newly discovered superfamily of DANGER proteins is involved in developmental processes and is of ancient origin. This superfamily has continually been expanded and contracted during metazoan evolution (<a href="#pone-0000204-g006">Figure 6A</a>). The expression and function of DANGER proteins in cells which can terminally differentiate suggest that DANGERs may be part of the genetic toolkit responsible for the emergence of different cell types during animal evolution (<a href="#pone-0000204-g006">Figure 6A</a>). Likely, DANGER represents part of the eukaryotic genome that existed before the emergence of multicellular animals <a href="#pone.0000204-King1">[36]</a>. This is supported by the existence of at least one DANGER homolog in choanoflagellates and the apparent absence of DANGER sequences in other eukaryotic kingdoms. The <em>N. vectensis</em> genome encodes at least 14 proteins containing the Mab-21 domain, suggesting an ancient or a species-specific expansion of the DANGER superfamily. In contrast, in protostome, hemichordate, and urochordate genomes this family has experienced a remarkable contraction (<a href="#pone-0000204-g006">Figure 6A</a>).</p>
<div class="figure" id="pone-0000204-g006"><div class="img"><a name="pone-0000204-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000204" data-uri="info:doi/10.1371/journal.pone.0000204.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000204.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000204.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000204.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Distribution and evolution of DANGER genes in metazoa.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>(A) The tree on the left summarizes the phylogenetic relationships of the informative species used. Numbers at nodes represent cell types adopted from <a href="#pone.0000204-Carroll2">[13]</a>. Yellow boxes indicate the presence of a DANGER gene in a particular taxon. Numbers correspond to the number of genes found in each taxon. “X” indicates the absence of a DANGER gene in a particular taxon (blue boxes). Uncertain orthologous relationships are indicated by question marks (green boxes). “UN” indicates the unclassified DANGER proteins or the ones that could not be unconditionally assigned to a specific group. The choanoflagellate sequence from <em>M. ovata</em> assumes outgroup position in all phylogenetic analyses. Divergence times in million years ago (MYA) were taken from references <a href="#pone.0000204-Ayala1">[58]</a>, <a href="#pone.0000204-Peterson1">[59]</a>. (B–C) Two alternative evolutionary scenarios explain the evolution of DANGER families in metazoa. Both scenarios presume that the genome sequence of the vertebrate ancestor encoded at least six DANGER lineages, namely D1–D6. (B) According to the first scenario, anthozoa contain two (D2, D6) of the six vertebrate DANGER lineages, while the remaining DANGER lineages have evolved by repeated cycles of gene birth (+) and death (−). (C) In the second scenario anthozoa species contain sequences orthologous to all vertebrate DANGER groups. This scenario presupposes that in ecdysozoa, hemichordates, and urochordates four DANGER groups were lost. Dotted lines correspond to ancestral stages. The order of events is presented according to the species tree.</p>
<span>doi:10.1371/journal.pone.0000204.g006</span></div><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>From our data we cannot conclude how many DANGER gene-lineages existed in the bilaterian ancestor (<a href="#pone-0000204-g006">Figure 6B and 6C</a>). The most conservative evolutionary scenario suggests that at least two DANGER gene-lineages existed in the common ancestor of anthozoa and bilateria. According to this scenario the DANGER repertoire found in invertebrate and vertebrate species is the result of multiple gene birth and death events (<a href="#pone-0000204-g006">Figure 6B</a>) <a href="#pone.0000204-Nei1">[37]</a>. Alternatively, the anthozoan DANGER gene-lineages could have persisted during metazoan evolution resulting in the vertebrate DANGER repertoire (<a href="#pone-0000204-g006">Figure 6C</a>). The latter suggests the presence of ancient genetic complexity in anthozoa and loss of genes in invertebrates as it has been previously suggested <a href="#pone.0000204-Technau1">[14]</a>.</p>
<a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>To determine which microdomains have conserved functions or represent evolutionary innovations would require functional characterization of the DANGER proteins in both basal and crown animals. Elucidating the relationship between microdomain evolution and functional divergence would provide insights on how genetic and cellular complexity contributes to morphological diversity. In conclusion, this evolutionary study of DANGER genes offers a view into the workshop of a busy evolutionary tinker.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data collection</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Similarity searches were performed at the NCBI non redundant database (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>) using the HsMAB21L1 and HsD1A protein sequences as queries. psi-BLAST <a href="#pone.0000204-Altschul1">[38]</a>, and tBLASTn <a href="#pone.0000204-Altschul2">[39]</a> programs were used with default parameters, except that filter was used for the lookup table only. The NCBI fungal and protists protein databases were scanned for fruitfly MAB-21 related proteins using psi-BLAST program with default parameters, except that filter was used for the lookup table only. The genome and proteome of <em>Xenopus tropicalis</em> (<a href="http://genome.jgi-psf.org/Xentr4/Xentr4.home.html">http://genome.jgi-psf.org/Xentr4/Xentr4.​home.html</a>) (v. 4.1) were searched for DANGER sequences using tBLASTn and BLASTp programs respectively (default parameters). Complete genomic sequences, segments of unfinished high throughput genomic sequences, and scaffolds were used for gene prediction utilizing the GenomeScan web server (<a href="http://genes.mit.edu/genomescan.html">http://genes.mit.edu/genomescan.html</a>) <a href="#pone.0000204-Yeh1">[40]</a>. The proteomes of <em>Rattus norvegicus</em>, <em>Danio rerio</em>, and <em>Ciona intestinalis</em> at Ensembl database (<a href="http://www.ensembl.org/">http://www.ensembl.org/</a>) were searched for DANGER proteins using BLASTp program (default parameters). The proteome of <em>Nematostella vectensis</em> (<a href="http://genome.jgi-psf.org/Nemve1/Nemve1.home.html">http://genome.jgi-psf.org/Nemve1/Nemve1.​home.html</a>) (v. 1.0) was searched for DANGER sequences using BLASTp program (default parameters).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>BLAST searches were conducted by using high expected E-values (<em>E</em> = 10) to ensure that most sequences homologous to the hD1A and hMAB21L1 would be retrieved. The resulted BLAST hits were sorted according to the E-value and all pairwise alignments were manually inspected. An initial phylogenetic analysis was performed in order to determine which sequences would be retained. Referenced (supported by cDNA sequences) or predicted amino acid sequences were aligned using ClustalX program <a href="#pone.0000204-Thompson1">[41]</a>. Pairwise and multiple alignments were performed using BLOSUM protein weight series matrices, with default gap opening and gap extension penalties. Distance-based (<em>p</em>-distance) phylogenetic analysis was performed using the neighbor-joining (NJ) algorithm <a href="#pone.0000204-Saitou1">[42]</a> as implemented in MEGA3 program <a href="#pone.0000204-Kumar1">[43]</a>. Only sequences that unambiguously clustered within a particular DANGER family were retained. Accession numbers of all sequences used in this study are listed in <a href="#pone.0000204.s016">table S1</a>. Although, we cannot formally exclude the possibility that the structure of some genes may have been mispredicted, most human and mouse DANGER gene structures are supported by cDNA sequences.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>Annotated genomic regions for the human, mouse, zebrafish, fly and nematode DANGER sequences were collected from the NCBI and Ensembl databases.</p>


<h4>Multiple sequence alignments</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Referenced or predicted amino acid sequences were aligned using ClustalX program <a href="#pone.0000204-Thompson1">[41]</a>. Pairwise and multiple alignments were performed using BLOSUM protein weight series matrices, with default gap opening and gap extension penalties. The multiple sequence alignments were manually edited. The alignment used to generate <a href="#pone-0000204-g001">Figure 1</a> is shown in <a href="#pone.0000204.s003">Figure S3</a>. We need to note here that the accuracy of the alignments especially at the N-and C-termini could be affected by retained introns and missing exons.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Distance-based phylogenetic analyses were performed using the neighbor-joining (NJ) algorithm <a href="#pone.0000204-Saitou1">[42]</a> as implemented in MEGA3 program <a href="#pone.0000204-Kumar1">[43]</a>. <em>p</em>-distances (<a href="#pone.0000204.s002">Figure S2</a>) and poisson correction distances (<a href="#pone-0000204-g001">Figure 1A</a>) following the gamma distribution (α = 6.46) were used to construct the phylogenetic trees. The alpha parameter for rate variation among sites was estimated by the maximum-likelihood method as implemented in the TREE PUZZLE program <a href="#pone.0000204-Schmidt1">[44]</a>. The accuracy of the reconstructed trees was examined by the bootstrap test with 1,000 replications.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>Unweighted maximum-parsimony (MP) reconstructions were performed using the protpars program of PHYLIP v.3.65 package. The MP analysis was performed with a heuristic search of 100 bootstrap replicates (datasets) and randomization of sequence order. MP analysis provided a single most parsimonious tree per dataset requiring a total of 17,300–18,300 steps. A consensus tree was constructed with the program consense, included in PHYLIP, using the extended majority rule criterion.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>Maximum likelihood (ML) analyses were performed with TreePuzzle. The ML analysis was performed by using the quartet puzzling tree search procedure, with 25,000 puzzling steps. We used the Jones-Taylor-Thornton (JTT) model of substitution, with the frequencies of amino acids being estimated from the data set, and rate heterogeneity across sites modeled in eight rate categories. Quartet puzzling is only an approximation of the ML method <a href="#pone.0000204-Yang1">[45]</a> and is not immune to artifacts <a href="#pone.0000204-Ranwez1">[46]</a>. In addition, ML estimation with bootstrap application is computationally impracticable in the present case because of the size of the data set. For this reason we performed ML analysis as the method is implemented in PHYLIP v.3.65 by analyzing one dataset and using JTT model of substitution, gamma-distributed distances with five categories of HMM rates, global rearrangements of branching order, and randomization of sequence order. The resultant tree was essentially the same as the NJ tree, with respect to the major branching patterns (data not shown).</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>MP and ML analyses were performed on the LION-XM PC cluster run by the High Performance Computing Group of Penn State's Academic Services and Emerging Technologies (<a href="http://gears.aset.psu.edu/hpc/systems/lionxm/">http://gears.aset.psu.edu/hpc/systems/li​onxm/</a>).</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5"></a><p>The phylogenetic relationships among paralogous DANGER sequences and for some anthozoan sequences remain unclear (low bootstrap support and differences in topologies using different methods). Several approaches were used to resolve these relationships. First, different regions of the proteins were used for phylogenetic reconstruction (i.e. only the conserved C-terminus regions of the protein sequences). Second, pairwise deletion of gaps was used to maximize the number of sites used for phylogenetic reconstruction. And third, different datasets containing only representative sequences of the different DANGER groups were used (for example, only vertebrate sequences). These different approaches gave essentially identical results in the phylogenetic analyses.</p>


<h4>
               <em>p</em>-distance calculations</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Mean <em>p</em>-distances for sliding window and microdomain analyses were calculated using MEGA3 (complete deletion of gaps). Standard errors for <em>p</em>-distances were calculated using 500 bootstrap replications.</p>


<h4>GDDA analysis</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>GDDA analysis was conducted using our software as previously described <a href="#pone.0000204-Patterson1">[11]</a>, <a href="#pone.0000204-vanRossum2">[12]</a>. Briefly, GDDA modifies the original target sequence by inserting a proportion of the domain sequence (10–50%) (“seed”) at every amino acid position of the target sequence. The modified sequences are searched by rps-BLAST against the “seed” domain sequence and the percentages of coverage are plotted against each amino acid position. The modification of the original sequence increases the sensitivity of rps-BLAST search by providing a “constant” initiation sequence, the “seed”. This procedure allows BLAST to extent (i.e. ‘filling in the gaps’ hence gestalt) the alignment even between highly divergent sequences.</p>


<h4>Domain architecture analysis</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>The Mab-21 domain used in our domain architecture analysis has originally been defined in the Pfam database <a href="#pone.0000204-Finn1">[47]</a>, and was imported to the Conserved Domain Database (CDD) at NCBI (<a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml">http://www.ncbi.nlm.nih.gov/Structure/cd​d/cdd.shtml</a>) <a href="#pone.0000204-MarchlerBauer1">[48]</a>. Domain architecture analysis of all referenced or predicted DANGER protein sequences was performed by using the executable local version of the rps-BLAST program (BLAST suite version: 2.2.14) and its accompanied CDD database. All searches were performed on the LION-XM PC cluster. The CDD position specific scoring matrix (PSSM) for Mab-21 domain was formatted as database using the formatrpsdb program (BLAST utility in the BLAST suite) and parameters set as described in <a href="ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/README">ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/READ​ME</a> file. This formatted PSSM was used as one entry database and all collected proteins as queries. Small size databases are shown to allow more sensitive searches, which are computationally inexpensive <a href="#pone.0000204-Korf1">[49]</a>. Results were filtered by setting an <em>E</em>-value threshold at 0.001 and stored as both pairwise alignments (-m option set at 0), or tabular results (-m option set at 8) <a href="#pone.0000204-Korf1">[49]</a>. Quantification of the Mab-21 domain coverage was performed by using the tabular results and applying the following formula:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000204.e001&amp;representation=PNG" class="inline-graphic"></span><br>Subject start and end correspond to the Mab-21 domain coordinates on the pairwise alignments. The amino acid length of the Mab-21 domain PSSM consensus sequence equals to 360.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Mab-21 domain PSSM was built in CDD by using only four highly conserved MAB-21 protein sequences (<em>C. elegans</em> MAB-21, <em>D. melanogaster</em> MAB21a and MAB21b, and <em>M. musculus</em> MAB21L1; data from CDD FASTA files as provided at <a href="ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/">ftp://ftp.ncbi.nih.gov/pub/mmdb/cdd/</a>). We built a new Mab-21 domain PSSM by utilizing the stand alone version of psi-BLAST. All D1–6 animal sequences (<a href="#pone.0000204.s016">table S1</a>), except the anthozoan sequences, were formatted as database using the formatdb BLAST utility with default parameters. HsMAB21L1 was used as query in an iterated psi-BLAST search using all D1–6 animal sequences (<a href="#pone.0000204.s016">table S1</a>), except the anthozoan sequences, as database. A new Mab-21 PSSM was built during the psi-BLAST searches utilizing the –C option of the stand alone psi-BLAST version. This psi-BLAST-generated Mab-21 PSSM was formatted for rps-BLAST searches and used as database for Mab-21 domain predictions. rps-BLAST searches were performed as in the case of original CDD Mab-21 PSSM. Quantification of the Mab-21 domain coverage was performed using formula (1) by modifying the size of the consensus psi-BLAST-generated Mab-21 domain sequence to 359.</p>
<a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3"></a><p>Signal peptide sequences were predicted using the SignalP v. 3.0 server (<a href="http://www.cbs.dtu.dk/services/SignalP/">http://www.cbs.dtu.dk/services/SignalP/</a>) <a href="#pone.0000204-Bendtsen1">[50]</a>. Transmembrane regions were predicted using the TMHMM v. 2.0 server (<a href="http://www.cbs.dtu.dk/services/TMHMM-2.0/">http://www.cbs.dtu.dk/services/TMHMM-2.0​/</a>) <a href="#pone.0000204-Krogh1">[51]</a>.</p>


<h4>Conserved synteny</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>The flanking genes of each DANGER gene were identified from the already annotated genome sequences (NCBI and Ensembl databases). In cases in which the flanking genes were not fully annotated similarity searches were performed using BLASTP (with <em>E</em> value threshold equal to 10<sup>−10</sup>) to identify homologs in other species.</p>


<h4>Genomic alignments</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>Approximately 25 Kbp upstream and downstream of each DANGER gene were retrieved from NCBI and Ensembl databases. Alignments among DANGER genomic sequences were performed using the BLASTZ <a href="#pone.0000204-Schwartz1">[52]</a> tool as implemented in the zPicture web tool (<a href="http://zpicture.dcode.org">http://zpicture.dcode.org</a>) <a href="#pone.0000204-Ovcharenko1">[53]</a>. BLASTZ generates sequence alignments between a reference sequence (first) and one or more other sequences. BLASTZ is a local alignment tool and identifies matches independent of their linear organization in the input sequences. Thus, zPicture identifies aligned regions independent of their location and orientation in the second sequence and maps these regions onto the reference sequence. In zPicture, BLASTZ alignments are visualized as standard percent identity plots (pip) <a href="#pone.0000204-Schwartz2">[54]</a>.</p>


<h4>Transposable elements identification and annotation</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>To identify and annotate the transposable elements in the genomic regions of DANGER sequences the repeatmasker tool (default parameters) was employed (<a href="http://www.repeatmasker.org">www.repeatmasker.org</a>).</p>


<h4>Other computational tools</h4>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1"></a><p>Graphs were generated with Microsoft Excel program. Statical tests were performed using MINITAB student 12 program. Domain architecture visualization was performed using the structure drawing tool (<a href="http://warta.bio.psu.edu/cgi-bin/Tools/StrDraw.pl">http://warta.bio.psu.edu/cgi-bin/Tools/S​trDraw.pl</a>). MSAs in block format were generated using WAVIS server (<a href="http://wavis.img.cas.cz">http://wavis.img.cas.cz</a>) <a href="#pone.0000204-Zika1">[55]</a>. Sequence conservation patterns (logos) were generated using the WebLogo server (<a href="http://weblogo.berkeley.edu/">http://weblogo.berkeley.edu/</a>) <a href="#pone.0000204-Crooks1">[56]</a>.</p>


<h4>Cell lines</h4>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1"></a><p>PC12 cells were cultured in Dulbecco's Minimal Essential Medium supplemented with 10% horse serum, 5% fetal bovine serum, 2 mM L-glutamine and 1% penicillin-streptomycin. TT cells were obtained from ATCC and were cultured in RPMI 1640 supplemented with 15% fetal bovine serum, 1 mM glutamine and 1% penicillin-streptomycin. Embryonic spinal cord neurons were cultured and plated as previously described <a href="#pone.0000204-Bird1">[57]</a>.</p>


<h4>RNA Sequences for DANGER1A</h4>
<a id="article1.body1.sec3.sec12.p1" name="article1.body1.sec3.sec12.p1"></a><p>Two siRNA sequences were used for DANGER1A deletion, each with similar efficacy.</p>
<a id="article1.body1.sec3.sec12.p2" name="article1.body1.sec3.sec12.p2"></a><p>5′ aagaatgccccagcgctcatt 3′ Human</p>
<a id="article1.body1.sec3.sec12.p3" name="article1.body1.sec3.sec12.p3"></a><p>5′ aatacgagtttgaccttgctt 3′ Rat</p>


<h4>Expression protocols</h4>
<a id="article1.body1.sec3.sec13.p1" name="article1.body1.sec3.sec13.p1"></a><p>Transfection of PC12, primary mouse spinal cord neurons, and TT cells with either 100 nM siRNA duplex and 1 µg YFP, or 1 µg DANGER1A and 1 µg YFP, was performed using Lipofectamine 2000 (Invitrogen), according to the manufacturer's specifications.</p>


<h4>Antibodies and reagents</h4>
<a id="article1.body1.sec3.sec14.p1" name="article1.body1.sec3.sec14.p1"></a><p>Plasmids were from the following sources: EYFP, Matchmaker© and Myc-CMV vector cDNA from CLONETECH; and human M5 muscarinic receptor cDNA from L.&nbsp;Birnbaumer (NIH). Human Northern Blot from CLONTECH (Palo Alto, CA). Carbachol, Protein A agarose, GST agarose, Sigma (St. Louis, MO). siRNA duplex was from Qiagen (Valencia, CA). TrkA, p75, phosphor-TrkA, and phospho-p75 antibodies from Upstate Biotech (Charlottesville, VA). Antibodies were used as to the manufacturers instructions.</p>


<h4>Subcellular fractionation</h4>
<a id="article1.body1.sec3.sec15.p1" name="article1.body1.sec3.sec15.p1"></a><p>After inducing differentiation by NGF, cells were harvested by gently scraping plates with a cell scraper, and were washed once with cold PBS. The washed pellet was subjected to one freeze-thaw cycle in liquid nitrogen. Pellets were resuspended in 1 ml Buffer A (250 mM sucrose, 10 mM Tris-HCl [pH 7.5], 1 mM EGTA, 1mM PMSF, one protease inhibitor pill). Cells were then homogenized on ice using a 1 ml glass Dounce homogenizer with a tight fitting pestle until ~95% of cells were disrupted as judged by Trypan blue staining. Crude lysates were centrifuged at 1,000× g for 15 min at 4°C to remove nuclei and unbroken cells. The supernatant was collected and the pellet (P1) discarded. The low speed supernatant was then subjected to 10,000× g centrifugation for 15 min, which yielded the 10,000× g pellet (P2). The supernatant from the P2 pellet was centrifuged at 15,000× g to completely rid the supernatant of any remaining mitochondria. Finally the 15,000× g supernatant was separated into cytosol (S3) and light membrane (P3) fractions by centrifugation at 100,000× g for 1 h. The 100,000× g supernatant was collected as the S3 fraction and the pellet was resuspended in 40–70 µl of Buffer A. P2 was washed twice by resuspending cells in 100 µl Buffer A and pelleting (10,000× g for 15 min). After the final wash, P2 was resuspended in 50–100 µl Buffer A. All fractionations were repeated a minimum of four times with essentially identical results. Subcellular fractions were characterized using nucleoporin (P1- nuclear), cytochrome c oxidase (P2- mitochondria, heavy ER), heme oxygenase 2 (P3- light ER), and lactate dehydrogenase (S3- cytosol).</p>


<h4>Immunofluorescence</h4>
<a id="article1.body1.sec3.sec16.p1" name="article1.body1.sec3.sec16.p1"></a><p>Primary spinal cord neurons were cultured on coverslips for 10 days then fixed with 0.4% paraformaldehyde in 3× PBS for 30 min and washed three times in 3× PBS for 10 min. Coverslips were quenched in 50 mM NH<sub>4</sub>Cl for 15 min and washed three times in 2× PBS for 10 min. Cells were permeabilized in saponin solution (3× PBS, 1% bovine serum albumin, 1% goat serum, and 0.075% w/v saponin) for 1 hr. Coverslips were washed three times in 3× PBS for 10 min and subsequently inverted on 250 µl anti-DANGER1a conjugated to Alexa 488© fluorophores from Molecular Probes in a wet chamber overnight at 4°C. Coverslips were washed three times in 3× PBS for 10 and mounted onto slides with Slo-fade©. Fluorescence signals were detected with a Zeiss LSM410 confocal laser scanning microscope, using a 63x/NA 1.4 objective.</p>


<h4>Immunohistochemistry</h4>
<a id="article1.body1.sec3.sec17.p1" name="article1.body1.sec3.sec17.p1"></a><p>Formalin fixed, paraffin embedded 5 µM sections were dewaxed, rehydrated and subjected to antigen retrieval by incubation in 10mM sodium citrate pH 6.0 at 98°C for 20 mins. Sections were then immunostained using the Vectastain<sup>TM</sup> ABC system according to the manufacturer instructions using affinity purified anti-DANGER antibody described above at a dilution of 1/50.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000204"></div><a name="pone.0000204.s001" id="pone.0000204.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">GDDA analysis of the mouse DANGER1A protein reveals the presence of an almost complete Mab-21 domain. (A) rps-BLAST similarity search of the MmD1A protein with E-value threshold equal to 0.01 and no filter for low complexity regions does not predict any domain on the query (MmD1A) sequence. (B) GDDA analysis with 10–50% “seeds” of the N-terminus of the Mab-21 domain consensus sequence, as defined in the Conserved Domain Database, identifies almost 90% of the Mab-21 domain sequence in MmD1A.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000204.s002" id="pone.0000204.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Phylogenetic relationships of the DANGER superfamily including all identified anthozoan (<em>Nv, Nematostella vectensis</em>) sequences. The tree was constructed with the NJ method using <em>p</em>-distances for 206 amino acid sites after elimination of alignment gaps. The <em>p</em>-distances are known to give a higher resolution of branching pattern because of the smaller standard errors. Numbers at branches represent bootstrap values. Species in red fonts denote the presence of introns in their corresponding DANGER coding sequence, while species in black fonts denote the absence of introns in their corresponding DANGER coding sequence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000204.s003" id="pone.0000204.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Multiple sequence alignment of the DANGER superfamily used to generate the phylogenetic tree of <a href="#pone-0000204-g001">Fig. 1A</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.69 MB PDF)</p>
<a name="pone.0000204.s004" id="pone.0000204.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Plots of the informative parameters for the rps-BLAST pairwise alignments between the Mab-21 domain profile (Pfam or psi-BLAST-generated) and the human DANGER proteins. (A) Mab-21 domain coverage, (B) sequence identity, (C) sequence similarity, (D) proportion of gaps, and (E) alignment bit score. Use of the psi-BLAST-generated Mab-21 profile results in increased domain coverage and alignment bit score; sequence identity and sequence similarity are increased for DANGER 5 and 6 groups, while for the remaining DANGER groups are decreased. The proportion of gaps in groups DANGER 1–3, and 5 is also increased.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.06 MB PDF)</p>
<a name="pone.0000204.s005" id="pone.0000204.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Plots of the informative parameters for the rps-BLAST pairwise alignments between Mab-21 domain profiles (Pfam or psi-BLAST-generated) and choanoflagellate, anthozoan, arthropod, echinoderm, and urochordate DANGER proteins. (A) Mab-21 domain coverage, (B) sequence identity, (C) sequence similarity, (D) proportion of gaps, and (E) alignment bit score.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000204.s006" id="pone.0000204.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Graphical representation of mean <em>p</em>-distances in the DANGER superfamily. (A) The graph depicts the mean <em>p</em>-distances per DANGER family within each Mab-21 micro-domain (II–IX). (B) The graph shows the mean <em>p</em>-distances per Mab-21 micro-domain for each DANGER family. Bars represent standard errors.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000204.s007" id="pone.0000204.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s007">Figure S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Pattern of sequence conservation (logo) along the less conserved Mab-21 micro-domains. (A) micro-domain II. (B) micro-domain III. (C) micro-domain IV. (D) micro-domain V. (E) micro-domain IX. Y axis represents the amount of information present at every amino acid position in the sequence, measured in bits.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.78 MB PDF)</p>
<a name="pone.0000204.s008" id="pone.0000204.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s008">Figure S8. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Comparison of pairwise alignments between the psi-BLAST-generated Mab-21 profile and the anthozoan DANGER sequences reveals conservation of insertions (black boxes) and deletions (white boxes), and correspondence of indels (&gt;3 amino acids) with intron positions (yellow boxes). For clarity, all sequences are mapped onto the Mab-21 profile sequence according to the pairwise alignment coordinates. Also, the pairwise alignments for <em>D. melanogaster</em> and <em>S. purpuratus</em> proteins are shown. Nv, <em>N. vectensis</em>; DmD4, CG7194; Dm4, CG15865; Sp2, XP_794693.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000204.s009" id="pone.0000204.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s009">Figure S9. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">The region 230–258 of the block formatted pairwise alignments shown in <a href="#pone-0000204-g003">Figure 3</a> is magnified to demonstrate the positional conservation of insertions (magenta fonts) among the DANGER proteins. The amino acids in yellow bold fonts correspond to exon boundaries at the nucleotide level. Hs, <em>H. sapiens</em>; Dm, <em>D. melanogaster</em>; Ce, <em>C. elegans</em>; Nv, <em>N. vectensis</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.13 MB PDF)</p>
<a name="pone.0000204.s010" id="pone.0000204.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s010">Figure S10. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">DANGER genes are in conserved synteny among vertebrates (D1–D5) or metazoa (D6). Genes are depicted as pentagon arrows to show transcription orientation. (A) D6 family members are in conserved synteny among all metazoan taxa used in this study, except for <em>D. melanogaster</em>. In particular, the MAB21L1 (L1) genes are located in an intron of the neurobeachin (NBEA) gene and the MAB21L2 (L2) genes are located in an intron of the LPS-responsive vesicle trafficking, beach and anchor containing (LRBA) gene. In data not shown, phylogenetic analysis of the NBEA and LRBA genes from vertebrates, and invertebrates suggests that these genes are homologous and have been duplicated in vertebrates like the L1 and L2 genes. These data suggest that the L1 and L2 genes are products of <em>en block</em> duplication in vertebrates. The coding sequence of D6 family members from <em>N. vectensis</em>, and vertebrates is not interrupted by introns, while the orthologous sequences from ecdysozoa (<em>D. melanogaster</em> and <em>C. elegans</em>), <em>C. intestinalis</em> and <em>S. purpuratus</em> contain introns (see <a href="#pone-0000204-g003">Fig. 3A</a>, <a href="#pone-0000204-g004">4A</a>, and <a href="#pone.0000204.s002">Figure S2</a>). In the <em>D. melanogaster</em> genome the D6 gene is duplicated and both copies are located on chromosome X, region 5D1-D2, approximately 800 kb downstream of the fly neurobeachin homolog (chromosome X, region 4F3-4F3). This duplication seems to have occurred in all insects, since <em>Anopheles gambiae</em>, <em>Apis mellifera</em>, <em>Tribolium castaneum</em>, and <em>Bombyx mori</em> all contain two copies of the D6 gene (data not shown). (B) D1A genes are in conserved synteny among vertebrates. <em>D. rerio</em> genome contains two copies of the D1A gene. (C) D1B genes are in conserved synteny among mammals. Our exhaustive similarity searches did not reveal D1B orthologs from other vertebrate species; here we show that the <em>X. tropicalis</em> syntenic region bears NUP133 gene in the syntenic position of the mammalian D1B gene. (D) D1C genes are in conserved synteny among warm-blooded animals. (E) D4 family members are in conserved synteny among vertebrates. (F) D5 family members are in conserved synteny among vertebrates. (G) D2A genes are in conserved synteny among vertebrates. D. rerio genome contains two copies of the D2A gene. (H) D3A genes are in conserved synteny among tetrapods. (I) D3B genes are in conserved synteny among tetrapods. (J) D2B genes are in conserved synteny among mammals. Our exhaustive similarity searches did not reveal D2B orthologs in non-mammalian vertebrates. Gene abbreviations: GSTO1, glutathione S-transferase omega 1; GSTO2, glutathione S-transferase omega 2; TMEM127, transmembrane protein 127; WDR39, WD repeat domain 39; ASCC3L1, activating signal cointegrator 1 complex subunit 3-like 1; NCAPH, non-SMC condensin I complex, subunit H; TMC7, transmembrane channel-like gene 7; COQ7, coenzyme Q7 homolog, ubiquinone (yeast); SYT17, synaptotagmin XVII; DDX43, DEAD (Asp-Glu-Ala-Asp) box polypeptide 43; MTO1, mitochondrial translation optimization 1 homolog (<em>S. cerevisiae</em>); EEF1A1, eukaryotic translation elongation factor 1 alpha 1; ATP1A1, ATPase type 1A1; FGF12, fibroblast growth factor 12; HRASLS, HRAS-like suppressor; ATP13A5, ATPase type 13A5; ATP13A4, ATPase type 13A4; ATF4, activating transcription factor 4 (tax-responsive enhancer element B67); LLGL1, lethal giant larvae homolog 1 (<em>Drosophila</em>); FLII, flightless I homolog (<em>Drosophila</em>); TOP3A, topoisomerase (DNA) III alpha; SMCR8, Smith-Magenis syndrome chromosome region, candidate 8; FGF11, fibroblast growth factor 11.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000204.s011" id="pone.0000204.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s011">Figure S11. </a></strong></p><a id="article1.body1.sec4.supplementary-material11.caption1.p1" name="article1.body1.sec4.supplementary-material11.caption1.p1"></a><p class="preSiDOI">zPicture visualization of representative alignments among DANGER genomic sequences. zPicture uses the local alignments tool BLASTZ to generate sequence alignments between a reference sequence (the first one) and one or more sequences. BLASTZ identifies matches independent of their linear organization in the input sequences and zPicture maps these alignments onto the reference (first) sequence. Local alignments are visualized as standard percent identity plots. (A) Visualizations of pairwise alignments among <em>H. sapiens</em> (h) DANGER paralogs. Note: When all sequences are masked for transposable elements (TEs) no similarity is found, except between HsD1A–HsD1B coding sequences. (B) Visualizations of pairwise alignments among <em>M. musculus</em> (m) DANGER paralogs. Note: When all sequences are masked for TEs no similarity is found, except between MmD1A–MmD1B coding sequences. (C) Visualizations of pairwise alignments among D. rerio (z) DANGER paralogs. Note: When all sequences are masked for TEs no similarity is found, except between DrD1A1–DrD1A2 coding sequences. (D) Upper: visualizations of pairwise alignments among D1A orthologous sequences from <em>H. sapiens</em>, <em>M. musculus</em>, and <em>D. rerio</em>. Lower: visualizations of pairwise alignments among D1A orthologous sequences from <em>H. sapiens</em>, <em>M. musculus</em>, and <em>D. rerio</em> after masking of TEs. (E) Upper: visualizations of pairwise alignments among MAB21L1 (L1) orthologous sequences from <em>H. sapiens</em>, <em>M. musculus</em>, and <em>D. rerio</em>. Lower: visualizations of pairwise alignments among L1 orthologous sequences from <em>H. sapiens</em>, <em>M. musculus</em>, and <em>D. rerio</em> after masking of TEs. (F) Upper: visualization of pairwise alignments between L1 and L2 paralogous sequences. Lower: Visualizations of pairwise alignments between L1 and L2 paralogous sequences, after masking of TEs. (G) Visualizations of pairwise alignments between HsD2B and HsD2A genomic regions. (H) Visualizations of pairwise alignments between HsD3A and HsBCR genomic regions. (I) Visualizations of pairwise alignments between HsD3B and HsBCR genomic regions. (J) Visualization of pairwise alignments between the genomic sequences of HsD1C and HsD1B using the zPicture web tool. (K) Upper: Visualization of pairwise alignments between the genomic sequences of HsD3A and HsD3B. Lower: Visualization of pairwise alignments between the genomic sequences of HsD3A and HsD3B after masking the transposable elements (TEs). (L) Visualization of pairwise alignments between the genomic sequences of HsD4 and HsBCR. DANGER genes are shown in blue boxes (exons) and blue lines (introns). Arrows show transcriptional orientation. Green boxes denote TEs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s011</p><a id="article1.body1.sec4.supplementary-material11.caption1.p2" name="article1.body1.sec4.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.18 MB PDF)</p>
<a name="pone.0000204.s012" id="pone.0000204.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s012">Figure S12. </a></strong></p><a id="article1.body1.sec4.supplementary-material12.caption1.p1" name="article1.body1.sec4.supplementary-material12.caption1.p1"></a><p class="preSiDOI">Repeat content of the genomic regions flanking the DANGER genes (approximately 25 Kbp upstream and downstream of each DANGER gene were analyzed). (A) Average number of different repetitive element families in zebrafish, mouse, and human DANGER genomic regions. (B) Average number of different repetitive element families in each DANGER family. (C) Repeat content in the <em>D. rerio</em> DANGER genomic regions. (D) Repeat content in the mouse DANGER genomic regions. (E) Repeat content in the human DANGER genomic regions. DNA, DNA trasnposons; LINE, long-interspersed nucleotide elements; LC, low complexity regions; LTR, retrovirus-like elements with Long Terminal Repeats; SR, simple repeats; SINE, short-interspersed nucleotide elements; UN, unclassified repeats.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s012</p><a id="article1.body1.sec4.supplementary-material12.caption1.p2" name="article1.body1.sec4.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.09 MB PDF)</p>
<a name="pone.0000204.s013" id="pone.0000204.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s013">Figure S13. </a></strong></p><a id="article1.body1.sec4.supplementary-material13.caption1.p1" name="article1.body1.sec4.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Analysis of expressed sequence tags (ESTs) show that mouse and human DANGER sequences exhibit different transcriptional patterns. (A) Total expression levels (sum of different tissues) for all <em>M. musculus</em> DANGER genes. (B) Expression of <em>M. musculus</em> DANGER genes in different tissues showed as transcripts per million. (C) Expression of <em>M. musculus</em> DANGER genes in different developmental stages. (D) Total expression levels (sum of different tissues) for <em>H. sapiens</em> DANGER genes. (E) Expression of <em>H. sapiens</em> DANGER genes in different tissues. (F) Expression of <em>H. sapiens</em> DANGER genes in different health states. EST data were collected from the UniGene database of NCBI (<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?">http://www.ncbi.nlm.nih.gov/entrez/query​.fcgi?</a>db = unigene). The UniGene identification number for each sequence is given below. For mouse: MmL1, Mm.384353; MmL2, Mm.389466; MmD5, Mm.32900; MmD4, Mm.101559; MmD3A, Mm.307163; MmD3B, Mm.339760; MmD2A, Mm.155887; MmD2B, Mm.280165; MmD1A, Mm.29457; MmD1B, Mm.57559; MmD1C, Mm.323386; and for human: HsD1A, Hs.523252; HsD1B, Hs.65009; HsD1C, Hs.530899; HsD4, Hs.14577; HsD3A, Hs.148677; HsD3B, Hs.551967; HsD2A, Hs.151443; HsL1, Hs.584776; HsL2, Hs.584852. Tissues and or stages are plotted if they contained more than 80,000 EST sequences. The presence of many ESTs demonstrates that DANGER sequences are expressed and thus DANGERs do not probably represent pseudogenes. The different expression patterns suggest putative functional divergence among paralogous DANGER genes supporting our phylogenetic analyses, which predicts that the different DANGER groups evolve following the model of divergent evolution.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s013</p><a id="article1.body1.sec4.supplementary-material13.caption1.p2" name="article1.body1.sec4.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0000204.s014" id="pone.0000204.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s014">Figure S14. </a></strong></p><a id="article1.body1.sec4.supplementary-material14.caption1.p1" name="article1.body1.sec4.supplementary-material14.caption1.p1"></a><p class="preSiDOI">DANGER1A is expressed in the brain and a variety of terminally differentiated tissues. (A) Random primers of HsD1A probed against a multiple tissue Northern Blot. (B–D) Immunohistochemistry on sagittal sections of E13.5 whole mouse, P0 brain and adult brain stained with polyclonal antibody against D1A, respectively. (E) Top: Horizontal brain section depicting the strong D1A staining in the CA1 of the hippocampus (arrow), and the lack of D1A staining in the dente gyrus (arrowhead). Bottom: Horizontal brain section depicting the strong D1A staining in the cerebellum. (F) Clockwise from left: anti-D1A staining in mouse pancreas, peripheral nerve, small intestine, cerebral cortex, skeletal muscle, and adrenal gland. The antibody used is specific to DANGER1a and is described in van Rossum et al. 2006 (Reference number 4 in main text). Antibody reactivity is indicated by the brown staining, while cells absent in DANGER expression are violet in color.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s014</p><a id="article1.body1.sec4.supplementary-material14.caption1.p2" name="article1.body1.sec4.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(1.22 MB PDF)</p>
<a name="pone.0000204.s015" id="pone.0000204.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s015">Figure S15. </a></strong></p><a id="article1.body1.sec4.supplementary-material15.caption1.p1" name="article1.body1.sec4.supplementary-material15.caption1.p1"></a><p class="preSiDOI">(A) Left: Western analysis of (20 µg) thyroid tumor (TT) cell or Brain lysates blotted with mouse monoclonal antibody against p75. Middle Left: Western analysis of (20 µg) PC12 or TT cell lysates blotted with polyclonal antibody against TrkA. Middle Right: Western analysis of (20 µg) TT cell lysate +/−50 ng/ml NGF for 48 h, blotted with monoclonal antibody against anti phospho-TrkA. GAPDH was used as a loading control. Right: Western analysis of (20 µg) PC12 cell or TT cell lysates blotted with polyclonal antibody against DANGER1A. PC12 cell express a major band (59 kDa) corresponding to the full-length protein and a second band (41 kDa), which is presumed to be a cleavage/breakdown product of D1A protein, since both bands are siRNA-sensitive. TT cells express only the 41 kDa band. Actin was used as a loading control. (B) YFP or YFP+DANGER1A transfected TT cells visualized by epifluorescent microscopy. Arrow denotes neurite growth.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s015</p><a id="article1.body1.sec4.supplementary-material15.caption1.p2" name="article1.body1.sec4.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.18 MB PDF)</p>
<a name="pone.0000204.s016" id="pone.0000204.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s016">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material16.caption1.p1" name="article1.body1.sec4.supplementary-material16.caption1.p1"></a><p class="preSiDOI">List of sequences used in this study</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s016</p><a id="article1.body1.sec4.supplementary-material16.caption1.p2" name="article1.body1.sec4.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
<a name="pone.0000204.s017" id="pone.0000204.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000204.s017">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material17.caption1.p1" name="article1.body1.sec4.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Repetitive elements found in DANGER mRNA sequences. Numbers correspond to the mRNA coordinates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000204.s017</p><a id="article1.body1.sec4.supplementary-material17.caption1.p2" name="article1.body1.sec4.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.01 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Drs. Jan Klein, Hong Ma, Graham Thomas, Jaynath Banavar, and Kirill Kiselyov for critically reviewing the manuscript. RLP and DVR would like to thank Robert E. Rothe, Frank Herbert, and Neil Young for useful discussion.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NN RP DC SS Dv. Performed the experiments: DW NN RP DC RB Dv. Analyzed the data: DW NN RP DC Dv. Contributed reagents/materials/analysis tools: NN RP DC SS Dv. Wrote the paper: NN RP DC SS Dv.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000204-Carroll1" id="pone.0000204-Carroll1"></a>Carroll SB, Grenier JK, Weatherbee SD (2004) From DNA to Diversity: Molecular Genetics and the Evolution of Animal Design. Malden, MA: Blackwell Publishers.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000204-Davidson1" id="pone.0000204-Davidson1"></a>Davidson EH, Erwin DH (2006) Gene regulatory networks and the evolution of animal body plans. Science  311: 796–800.  <ul class="find" data-citedArticleID="929724" data-doi="10.1126/science.1113832"><li><a href="http://dx.doi.org/10.1126/science.1113832" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+regulatory+networks+and+the+evolution+of+animal+body+plans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+regulatory+networks+and+the+evolution+of+animal+body+plans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000204-Avise1" id="pone.0000204-Avise1"></a>Avise JC (2000) Phylogeography: The History and Formation of Species. Cambridge, MA: Harvard University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0000204-vanRossum1" id="pone.0000204-vanRossum1"></a>van Rossum DB, Patterson RL, Cheung KH, Barrow RK, Syrovatkina V, et al.  (2006) DANGER, a novel regulatory protein of inositol 1,4,5-trisphosphate-receptor activity. J Biol Chem  281: 37111–37116.  <ul class="find" data-citedArticleID="929806" data-doi="10.1074/jbc.m608760200"><li><a href="http://dx.doi.org/10.1074/jbc.m608760200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DANGER%2C+a+novel+regulatory+protein+of+inositol+1%2C4%2C5-trisphosphate-receptor+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DANGER%2C+a+novel+regulatory+protein+of+inositol+1%2C4%2C5-trisphosphate-receptor+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000204-Chow1" id="pone.0000204-Chow1"></a>Chow KL, Hall DH, Emmons SW (1995) The mab-21 gene of Caenorhabditis elegans encodes a novel protein required for choice of alternate cell fates. Development  121: 3615–3626.  <ul class="find" data-citedArticleID="929718"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+mab-21+gene+of+Caenorhabditis+elegans+encodes+a+novel+protein+required+for+choice+of+alternate+cell+fates.&amp;auth=&amp;atitle=The+mab-21+gene+of+Caenorhabditis+elegans+encodes+a+novel+protein+required+for+choice+of+alternate+cell+fates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mab-21+gene+of+Caenorhabditis+elegans+encodes+a+novel+protein+required+for+choice+of+alternate+cell+fates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mab-21+gene+of+Caenorhabditis+elegans+encodes+a+novel+protein+required+for+choice+of+alternate+cell+fates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000204-Wong1" id="pone.0000204-Wong1"></a>Wong YM, Chow KL (2002) Expression of zebrafish mab21 genes marks the differentiating eye, midbrain and neural tube. Mech Dev  113: 149–152.  <ul class="find" data-citedArticleID="929792" data-doi="10.1016/s0925-4773(02)00012-6"><li><a href="http://dx.doi.org/10.1016/s0925-4773(02)00012-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+zebrafish+mab21+genes+marks+the+differentiating+eye%2C+midbrain+and+neural+tube." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+zebrafish+mab21+genes+marks+the+differentiating+eye%2C+midbrain+and+neural+tube.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000204-Mariani1" id="pone.0000204-Mariani1"></a>Mariani M, Baldessari D, Francisconi S, Viggiano L, Rocchi M, et al.  (1999) Two murine and human homologs of mab-21, a cell fate determination gene involved in Caenorhabditis elegans neural development. Hum Mol Genet  8: 2397–2406.  <ul class="find" data-citedArticleID="929758" data-doi="10.1093/hmg/8.13.2397"><li><a href="http://dx.doi.org/10.1093/hmg/8.13.2397" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+murine+and+human+homologs+of+mab-21%2C+a+cell+fate+determination+gene+involved+in+Caenorhabditis+elegans+neural+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+murine+and+human+homologs+of+mab-21%2C+a+cell+fate+determination+gene+involved+in+Caenorhabditis+elegans+neural+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000204-Heanue1" id="pone.0000204-Heanue1"></a>Heanue TA, Pachnis V (2006) Expression profiling the developing mammalian enteric nervous system identifies marker and candidate Hirschsprung disease genes. Proc Natl Acad Sci U S A  103: 6919–6924.  <ul class="find" data-citedArticleID="929732" data-doi="10.1073/pnas.0602152103"><li><a href="http://dx.doi.org/10.1073/pnas.0602152103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+the+developing+mammalian+enteric+nervous+system+identifies+marker+and+candidate+Hirschsprung+disease+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+the+developing+mammalian+enteric+nervous+system+identifies+marker+and+candidate+Hirschsprung+disease+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000204-Yamada1" id="pone.0000204-Yamada1"></a>Yamada R, Mizutani-Koseki Y, Hasegawa T, Osumi N, Koseki H, et al.  (2003) Cell-autonomous involvement of Mab21l1 is essential for lens placode development. Development  130: 1759–1770.  <ul class="find" data-citedArticleID="929796" data-doi="10.1242/dev.00399"><li><a href="http://dx.doi.org/10.1242/dev.00399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-autonomous+involvement+of+Mab21l1+is+essential+for+lens+placode+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-autonomous+involvement+of+Mab21l1+is+essential+for+lens+placode+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000204-Yamada2" id="pone.0000204-Yamada2"></a>Yamada R, Mizutani-Koseki Y, Koseki H, Takahashi N (2004) Requirement for Mab21l2 during development of murine retina and ventral body wall. Dev Biol  274: 295–307.  <ul class="find" data-citedArticleID="929798" data-doi="10.1016/j.ydbio.2004.07.016"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2004.07.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+for+Mab21l2+during+development+of+murine+retina+and+ventral+body+wall." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+for+Mab21l2+during+development+of+murine+retina+and+ventral+body+wall.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000204-Patterson1" id="pone.0000204-Patterson1"></a>Patterson RL, van Rossum DB, Nikolaidis N, Gill DL, Snyder SH (2005) Phospholipase C-gamma: diverse roles in receptor-mediated calcium signaling. Trends Biochem Sci  30: 688–697.  <ul class="find" data-citedArticleID="929764" data-doi="10.1016/j.tibs.2005.10.005"><li><a href="http://dx.doi.org/10.1016/j.tibs.2005.10.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phospholipase+C-gamma%3A+diverse+roles+in+receptor-mediated+calcium+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phospholipase+C-gamma%3A+diverse+roles+in+receptor-mediated+calcium+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000204-vanRossum2" id="pone.0000204-vanRossum2"></a>van Rossum DB, Patterson RL, Sharma S, Barrow RK, Kornberg M, et al.  (2005) Phospholipase Cgamma1 controls surface expression of TRPC3 through an intermolecular PH domain. Nature  434: 99–104.  <ul class="find" data-citedArticleID="929808" data-doi="10.1038/nature03340"><li><a href="http://dx.doi.org/10.1038/nature03340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phospholipase+Cgamma1+controls+surface+expression+of+TRPC3+through+an+intermolecular+PH+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phospholipase+Cgamma1+controls+surface+expression+of+TRPC3+through+an+intermolecular+PH+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000204-Carroll2" id="pone.0000204-Carroll2"></a>Carroll SB (2001) Chance and necessity: the evolution of morphological complexity and diversity. Nature  409: 1102–1109.  <ul class="find" data-citedArticleID="929714" data-doi="10.1038/35059227"><li><a href="http://dx.doi.org/10.1038/35059227" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chance+and+necessity%3A+the+evolution+of+morphological+complexity+and+diversity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chance+and+necessity%3A+the+evolution+of+morphological+complexity+and+diversity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000204-Technau1" id="pone.0000204-Technau1"></a>Technau U, Rudd S, Maxwell P, Gordon PM, Saina M, et al.  (2005) Maintenance of ancestral complexity and non-metazoan genes in two basal cnidarians. Trends Genet  21: 633–639.  <ul class="find" data-citedArticleID="929788" data-doi="10.1016/j.tig.2005.09.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.09.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maintenance+of+ancestral+complexity+and+non-metazoan+genes+in+two+basal+cnidarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maintenance+of+ancestral+complexity+and+non-metazoan+genes+in+two+basal+cnidarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000204-Kusserow1" id="pone.0000204-Kusserow1"></a>Kusserow A, Pang K, Sturm C, Hrouda M, Lentfer J, et al.  (2005) Unexpected complexity of the Wnt gene family in a sea anemone. Nature  433: 156–160.  <ul class="find" data-citedArticleID="929748" data-doi="10.1038/nature03158"><li><a href="http://dx.doi.org/10.1038/nature03158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unexpected+complexity+of+the+Wnt+gene+family+in+a+sea+anemone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unexpected+complexity+of+the+Wnt+gene+family+in+a+sea+anemone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000204-Prudhomme1" id="pone.0000204-Prudhomme1"></a>Prud'homme B, Lartillot N, Balavoine G, Adoutte A, Vervoort M (2002) Phylogenetic analysis of the Wnt gene family. Insights from lophotrochozoan members. Curr Biol  12: 1395.  <ul class="find" data-citedArticleID="929770" data-doi="10.1016/s0960-9822(02)01068-0"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)01068-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+analysis+of+the+Wnt+gene+family.+Insights+from+lophotrochozoan+members." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+analysis+of+the+Wnt+gene+family.+Insights+from+lophotrochozoan+members.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000204-Li1" id="pone.0000204-Li1"></a>Li WH, Graur D (1991) Fundamentals of Molecular Evolution. Sunderland, MA: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0000204-Brandenberger1" id="pone.0000204-Brandenberger1"></a>Brandenberger R, Wei H, Zhang S, Lei S, Murage J, et al.  (2004) Transcriptome characterization elucidates signaling networks that control human ES cell growth and differentiation. Nat Biotechnol  22: 707–716.  <ul class="find" data-citedArticleID="929708" data-doi="10.1038/nbt971"><li><a href="http://dx.doi.org/10.1038/nbt971" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+characterization+elucidates+signaling+networks+that+control+human+ES+cell+growth+and+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+characterization+elucidates+signaling+networks+that+control+human+ES+cell+growth+and+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000204-Clark1" id="pone.0000204-Clark1"></a>Clark HF, Gurney AL, Abaya E, Baker K, Baldwin D, et al.  (2003) The secreted protein discovery initiative (SPDI), a large-scale effort to identify novel human secreted and transmembrane proteins: a bioinformatics assessment. Genome Res  13: 2265–2270.  <ul class="find" data-citedArticleID="929720" data-doi="10.1101/gr.1293003"><li><a href="http://dx.doi.org/10.1101/gr.1293003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+secreted+protein+discovery+initiative+%28SPDI%29%2C+a+large-scale+effort+to+identify+novel+human+secreted+and+transmembrane+proteins%3A+a+bioinformatics+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+secreted+protein+discovery+initiative+%28SPDI%29%2C+a+large-scale+effort+to+identify+novel+human+secreted+and+transmembrane+proteins%3A+a+bioinformatics+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000204-Jin1" id="pone.0000204-Jin1"></a>Jin J, Smith FD, Stark C, Wells CD, Fawcett JP, et al.  (2004) Proteomic, functional, and domain-based analysis of in vivo 14-3-3 binding proteins involved in cytoskeletal regulation and cellular organization. Curr Biol  14: 1436–1450.  <ul class="find" data-citedArticleID="929734" data-doi="10.1016/j.cub.2004.07.051"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.07.051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic%2C+functional%2C+and+domain-based+analysis+of+in+vivo+14-3-3+binding+proteins+involved+in+cytoskeletal+regulation+and+cellular+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic%2C+functional%2C+and+domain-based+analysis+of+in+vivo+14-3-3+binding+proteins+involved+in+cytoskeletal+regulation+and+cellular+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000204-Ko1" id="pone.0000204-Ko1"></a>Ko MS, Kitchen JR, Wang X, Threat TA, Wang X, et al.  (2000) Large-scale cDNA analysis reveals phased gene expression patterns during preimplantation mouse development. Development  127: 1737–1749.  <ul class="find" data-citedArticleID="929738"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Large-scale+cDNA+analysis+reveals+phased+gene+expression+patterns+during+preimplantation+mouse+development.&amp;auth=&amp;atitle=Large-scale+cDNA+analysis+reveals+phased+gene+expression+patterns+during+preimplantation+mouse+development." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+cDNA+analysis+reveals+phased+gene+expression+patterns+during+preimplantation+mouse+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+cDNA+analysis+reveals+phased+gene+expression+patterns+during+preimplantation+mouse+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000204-Kopczynski1" id="pone.0000204-Kopczynski1"></a>Kopczynski CC, Noordermeer JN, Serano TL, Chen WY, Pendleton JD, et al.  (1998) A high throughput screen to identify secreted and transmembrane proteins involved in Drosophila embryogenesis. Proc Natl Acad Sci U S A  95: 9973–9978.  <ul class="find" data-citedArticleID="929740" data-doi="10.1073/pnas.95.17.9973"><li><a href="http://dx.doi.org/10.1073/pnas.95.17.9973" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+high+throughput+screen+to+identify+secreted+and+transmembrane+proteins+involved+in+Drosophila+embryogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+high+throughput+screen+to+identify+secreted+and+transmembrane+proteins+involved+in+Drosophila+embryogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000204-Lau1" id="pone.0000204-Lau1"></a>Lau GT, Wong OG, Chan PM, Kok KH, Wong RL (2001) Embryonic XMab21l2 expression is required for gastrulation and subsequent neural development. Biochem Biophys Res Commun  280: 1378–1384.  <ul class="find" data-citedArticleID="929750" data-doi="10.1006/bbrc.2001.4290"><li><a href="http://dx.doi.org/10.1006/bbrc.2001.4290" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Embryonic+XMab21l2+expression+is+required+for+gastrulation+and+subsequent+neural+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Embryonic+XMab21l2+expression+is+required+for+gastrulation+and+subsequent+neural+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000204-Lim1" id="pone.0000204-Lim1"></a>Lim J, Hao T, Shaw C, Patel AJ, Szabo G, et al.  (2006) A protein-protein interaction network for human inherited ataxias and disorders of Purkinje cell degeneration. Cell  125: 801–814.  <ul class="find" data-citedArticleID="929754" data-doi="10.1016/j.cell.2006.03.032"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.03.032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein-protein+interaction+network+for+human+inherited+ataxias+and+disorders+of+Purkinje+cell+degeneration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein-protein+interaction+network+for+human+inherited+ataxias+and+disorders+of+Purkinje+cell+degeneration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000204-Wong2" id="pone.0000204-Wong2"></a>Wong RL, Chow KL (2002) Depletion of Mab21l1 and Mab21l2 messages in mouse embryo arrests axial turning, and impairs notochord and neural tube differentiation. Teratology  65: 70–77.  <ul class="find" data-citedArticleID="929794" data-doi="10.1002/tera.10018"><li><a href="http://dx.doi.org/10.1002/tera.10018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Depletion+of+Mab21l1+and+Mab21l2+messages+in+mouse+embryo+arrests+axial+turning%2C+and+impairs+notochord+and+neural+tube+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Depletion+of+Mab21l1+and+Mab21l2+messages+in+mouse+embryo+arrests+axial+turning%2C+and+impairs+notochord+and+neural+tube+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000204-Gregory1" id="pone.0000204-Gregory1"></a>Gregory TR (2004) Insertion-deletion biases and the evolution of genome size. Gene  324: 15–34.  <ul class="find" data-citedArticleID="929730" data-doi="10.1016/j.gene.2003.09.030"><li><a href="http://dx.doi.org/10.1016/j.gene.2003.09.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insertion-deletion+biases+and+the+evolution+of+genome+size." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insertion-deletion+biases+and+the+evolution+of+genome+size.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000204-Brosius1" id="pone.0000204-Brosius1"></a>Brosius J (1999) Genomes were forged by massive bombardments with retroelements and retrosequences. Genetica  107: 209–238.  <ul class="find" data-citedArticleID="929710" data-doi="10.1007/978-94-011-4156-7_23"><li><a href="http://dx.doi.org/10.1007/978-94-011-4156-7_23" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomes+were+forged+by+massive+bombardments+with+retroelements+and+retrosequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomes+were+forged+by+massive+bombardments+with+retroelements+and+retrosequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000204-Eichler1" id="pone.0000204-Eichler1"></a>Eichler EE, Sankoff D (2003) Structural dynamics of eukaryotic chromosome evolution. Science  301: 793–797.  <ul class="find" data-citedArticleID="929726" data-doi="10.1126/science.1086132"><li><a href="http://dx.doi.org/10.1126/science.1086132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+dynamics+of+eukaryotic+chromosome+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+dynamics+of+eukaryotic+chromosome+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000204-Prado1" id="pone.0000204-Prado1"></a>Prado F, Cortes-Ledesma F, Huertas P, Aguilera A (2003) Mitotic recombination in Saccharomyces cerevisiae. Curr Genet  42: 185–198.  <ul class="find" data-citedArticleID="929768"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mitotic+recombination+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Mitotic+recombination+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitotic+recombination+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitotic+recombination+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000204-Bi1" id="pone.0000204-Bi1"></a>Bi W, Yan J, Stankiewicz P, Park SS, Walz K, et al.  (2002) Genes in a refined Smith-Magenis syndrome critical deletion interval on chromosome 17p11.2 and the syntenic region of the mouse. Genome Res  12: 713–728.  <ul class="find" data-citedArticleID="929704" data-doi="10.1101/gr.73702"><li><a href="http://dx.doi.org/10.1101/gr.73702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+in+a+refined+Smith-Magenis+syndrome+critical+deletion+interval+on+chromosome+17p11.2+and+the+syntenic+region+of+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+in+a+refined+Smith-Magenis+syndrome+critical+deletion+interval+on+chromosome+17p11.2+and+the+syntenic+region+of+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000204-Shaw1" id="pone.0000204-Shaw1"></a>Shaw CJ, Lupski JR (2005) Non-recurrent 17p11.2 deletions are generated by homologous and non-homologous mechanisms. Hum Genet  116: 1–7.  <ul class="find" data-citedArticleID="929784" data-doi="10.1007/s00439-004-1204-9"><li><a href="http://dx.doi.org/10.1007/s00439-004-1204-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Non-recurrent+17p11.2+deletions+are+generated+by+homologous+and+non-homologous+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Non-recurrent+17p11.2+deletions+are+generated+by+homologous+and+non-homologous+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000204-StevensKroef1" id="pone.0000204-StevensKroef1"></a>Stevens-Kroef MJ, Dirckx R, Meers LE, Albrechts JC, Schouten HC, et al.  (2000) Complex chromosome 9, 20, and 22 rearrangements in acute lymphoblastic leukemia with duplication of BCR and ABL sequences. Cancer Genet Cytogenet  116: 119–123.  <ul class="find" data-citedArticleID="929786" data-doi="10.1016/s0165-4608(99)00119-3"><li><a href="http://dx.doi.org/10.1016/s0165-4608(99)00119-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+chromosome+9%2C+20%2C+and+22+rearrangements+in+acute+lymphoblastic+leukemia+with+duplication+of+BCR+and+ABL+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+chromosome+9%2C+20%2C+and+22+rearrangements+in+acute+lymphoblastic+leukemia+with+duplication+of+BCR+and+ABL+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000204-Qian1" id="pone.0000204-Qian1"></a>Qian X, Ginty DD (2001) SH2-B and APS are multimeric adapters that augment TrkA signaling. Mol Cell Biol  21: 1613–1620.  <ul class="find" data-citedArticleID="929772" data-doi="10.1128/mcb.21.5.1613-1620.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.5.1613-1620.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SH2-B+and+APS+are+multimeric+adapters+that+augment+TrkA+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SH2-B+and+APS+are+multimeric+adapters+that+augment+TrkA+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000204-CarsonWalter1" id="pone.0000204-CarsonWalter1"></a>Carson-Walter EB, Smith DP, Ponder BA, Baylin SB, Nelkin BD (1998) Post-transcriptional silencing of RET occurs, but is not required, during raf-1 mediated differentiation of medullary thyroid carcinoma cells. Oncogene  17: 367–376.  <ul class="find" data-citedArticleID="929716" data-doi="10.1038/sj.onc.1201938"><li><a href="http://dx.doi.org/10.1038/sj.onc.1201938" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Post-transcriptional+silencing+of+RET+occurs%2C+but+is+not+required%2C+during+raf-1+mediated+differentiation+of+medullary+thyroid+carcinoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Post-transcriptional+silencing+of+RET+occurs%2C+but+is+not+required%2C+during+raf-1+mediated+differentiation+of+medullary+thyroid+carcinoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000204-Agell1" id="pone.0000204-Agell1"></a>Agell N, Bachs O, Rocamora N, Villalonga P (2002) Modulation of the Ras/Raf/MEK/ERK pathway by Ca(2+), and calmodulin. Cell Signal  14: 649–654.  <ul class="find" data-citedArticleID="929692" data-doi="10.1016/s0898-6568(02)00007-4"><li><a href="http://dx.doi.org/10.1016/s0898-6568(02)00007-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+the+Ras%2FRaf%2FMEK%2FERK+pathway+by+Ca%282%2B%29%2C+and+calmodulin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+the+Ras%2FRaf%2FMEK%2FERK+pathway+by+Ca%282%2B%29%2C+and+calmodulin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000204-King1" id="pone.0000204-King1"></a>King N, Hittinger CT, Carroll SB (2003) Evolution of key cell signaling and adhesion protein families predates animal origins. Science  301: 361–363.  <ul class="find" data-citedArticleID="929736" data-doi="10.1126/science.1083853"><li><a href="http://dx.doi.org/10.1126/science.1083853" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+key+cell+signaling+and+adhesion+protein+families+predates+animal+origins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+key+cell+signaling+and+adhesion+protein+families+predates+animal+origins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000204-Nei1" id="pone.0000204-Nei1"></a>Nei M, Rooney AP (2005) Concerted and birth-and-death evolution of multigene families. Annu Rev Genet  39: 121–152.  <ul class="find" data-citedArticleID="929760" data-doi="10.1146/annurev.genet.39.073003.112240"><li><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concerted+and+birth-and-death+evolution+of+multigene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concerted+and+birth-and-death+evolution+of+multigene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000204-Altschul1" id="pone.0000204-Altschul1"></a>Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, et al.  (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res  25: 3389–3402.  <ul class="find" data-citedArticleID="929694" data-doi="10.1093/nar/25.17.3389"><li><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gapped+BLAST+and+PSI-BLAST%3A+a+new+generation+of+protein+database+search+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000204-Altschul2" id="pone.0000204-Altschul2"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="929696" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000204-Yeh1" id="pone.0000204-Yeh1"></a>Yeh RF, Lim LP, Burge CB (2001) Computational inference of homologous gene structures in the human genome. Genome Res  11: 803–816.  <ul class="find" data-citedArticleID="929802" data-doi="10.1101/gr.175701"><li><a href="http://dx.doi.org/10.1101/gr.175701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+inference+of+homologous+gene+structures+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+inference+of+homologous+gene+structures+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000204-Thompson1" id="pone.0000204-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="929790" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000204-Saitou1" id="pone.0000204-Saitou1"></a>Saitou N, Nei M (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. Mol Biol Evol  4: 406–425.  <ul class="find" data-citedArticleID="929776"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.&amp;auth=&amp;atitle=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+neighbor-joining+method%3A+a+new+method+for+reconstructing+phylogenetic+trees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000204-Kumar1" id="pone.0000204-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform  5: 150–163.  <ul class="find" data-citedArticleID="929746" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000204-Schmidt1" id="pone.0000204-Schmidt1"></a>Schmidt HA, Strimmer K, Vingron M, von HA (2002) TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics  18: 502–504.  <ul class="find" data-citedArticleID="929778" data-doi="10.1093/bioinformatics/18.3.502"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREE-PUZZLE%3A+maximum+likelihood+phylogenetic+analysis+using+quartets+and+parallel+computing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000204-Yang1" id="pone.0000204-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="929800" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000204-Ranwez1" id="pone.0000204-Ranwez1"></a>Ranwez V, Gascuel O (2001) Quartet-based phylogenetic inference: improvements and limits. Mol Biol Evol  18: 1103–1116.  <ul class="find" data-citedArticleID="929774" data-doi="10.1093/oxfordjournals.molbev.a003881"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quartet-based+phylogenetic+inference%3A+improvements+and+limits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quartet-based+phylogenetic+inference%3A+improvements+and+limits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000204-Finn1" id="pone.0000204-Finn1"></a>Finn RD, Mistry J, Schuster-Bockler B, Griffiths-Jones S, Hollich V, et al.  (2006) Pfam: clans, web tools and services. Nucleic Acids Res  34: D247–D251.  <ul class="find" data-citedArticleID="929728" data-doi="10.1093/nar/gkj149"><li><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pfam%3A+clans%2C+web+tools+and+services." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pfam%3A+clans%2C+web+tools+and+services.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000204-MarchlerBauer1" id="pone.0000204-MarchlerBauer1"></a>Marchler-Bauer A, Anderson JB, Cherukuri PF, Weese-Scott C, Geer LY, et al.  (2005) CDD: a Conserved Domain Database for protein classification. Nucleic Acids Res  33: D192–D196.  <ul class="find" data-citedArticleID="929756" data-doi="10.1093/nar/gki069"><li><a href="http://dx.doi.org/10.1093/nar/gki069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CDD%3A+a+Conserved+Domain+Database+for+protein+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CDD%3A+a+Conserved+Domain+Database+for+protein+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000204-Korf1" id="pone.0000204-Korf1"></a>Korf I, Yandell M, Bedell J (2003) BLAST. Sebastopol, CA: O'Reilly.   <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0000204-Bendtsen1" id="pone.0000204-Bendtsen1"></a>Bendtsen JD, Nielsen H, von HG, Brunak S (2004) Improved prediction of signal peptides: SignalP 3.0. J Mol Biol  340: 783–795.  <ul class="find" data-citedArticleID="929702" data-doi="10.1016/j.jmb.2004.05.028"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.05.028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+prediction+of+signal+peptides%3A+SignalP+3.0." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+prediction+of+signal+peptides%3A+SignalP+3.0.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000204-Krogh1" id="pone.0000204-Krogh1"></a>Krogh A, Larsson B, von HG, Sonnhammer EL (2001) Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. J Mol Biol  305: 567–580.  <ul class="find" data-citedArticleID="929744" data-doi="10.1006/jmbi.2000.4315"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model%3A+application+to+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model%3A+application+to+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000204-Schwartz1" id="pone.0000204-Schwartz1"></a>Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, et al.  (2003) Human-mouse alignments with BLASTZ. Genome Res  13: 103–107.  <ul class="find" data-citedArticleID="929780" data-doi="10.1101/gr.809403"><li><a href="http://dx.doi.org/10.1101/gr.809403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human-mouse+alignments+with+BLASTZ." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human-mouse+alignments+with+BLASTZ.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000204-Ovcharenko1" id="pone.0000204-Ovcharenko1"></a>Ovcharenko I, Loots GG, Hardison RC, Miller W, Stubbs L (2004) zPicture: dynamic alignment and visualization tool for analyzing conservation profiles. Genome Res  14: 472–477.  <ul class="find" data-citedArticleID="929762" data-doi="10.1101/gr.2129504"><li><a href="http://dx.doi.org/10.1101/gr.2129504" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=zPicture%3A+dynamic+alignment+and+visualization+tool+for+analyzing+conservation+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22zPicture%3A+dynamic+alignment+and+visualization+tool+for+analyzing+conservation+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000204-Schwartz2" id="pone.0000204-Schwartz2"></a>Schwartz S, Zhang Z, Frazer KA, Smit A, Riemer C, et al.  (2000) PipMaker–a web server for aligning two genomic DNA sequences. Genome Res  10: 577–586.  <ul class="find" data-citedArticleID="929782" data-doi="10.1101/gr.10.4.577"><li><a href="http://dx.doi.org/10.1101/gr.10.4.577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PipMaker%26%23x2013%3Ba+web+server+for+aligning+two+genomic+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PipMaker%26%23x2013%3Ba+web+server+for+aligning+two+genomic+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000204-Zika1" id="pone.0000204-Zika1"></a>Zika R, Paces J, Pavlicek A, Paces V (2004) WAViS server for handling, visualization and presentation of multiple alignments of nucleotide or amino acids sequences. Nucleic Acids Res  32: W48–W49.  <ul class="find" data-citedArticleID="929804" data-doi="10.1093/nar/gkh358"><li><a href="http://dx.doi.org/10.1093/nar/gkh358" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WAViS+server+for+handling%2C+visualization+and+presentation+of+multiple+alignments+of+nucleotide+or+amino+acids+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WAViS+server+for+handling%2C+visualization+and+presentation+of+multiple+alignments+of+nucleotide+or+amino+acids+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000204-Crooks1" id="pone.0000204-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: a sequence logo generator. Genome Res  14: 1188–1190.  <ul class="find" data-citedArticleID="929722" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+a+sequence+logo+generator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+a+sequence+logo+generator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000204-Bird1" id="pone.0000204-Bird1"></a>Bird M, Owen A (1998) Neurite outgrowth-regulating properties of GABA and the effect of serum on mouse spinal cord neurons in culture. J Anat  193(Pt 4): 503–508.  <ul class="find" data-citedArticleID="929706" data-doi="10.1046/j.1469-7580.1998.19340503.x"><li><a href="http://dx.doi.org/10.1046/j.1469-7580.1998.19340503.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neurite+outgrowth-regulating+properties+of+GABA+and+the+effect+of+serum+on+mouse+spinal+cord+neurons+in+culture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neurite+outgrowth-regulating+properties+of+GABA+and+the+effect+of+serum+on+mouse+spinal+cord+neurons+in+culture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000204-Ayala1" id="pone.0000204-Ayala1"></a>Ayala FJ, Rzhetsky A, Ayala FJ (1998) Origin of the metazoan phyla: molecular clocks confirm paleontological estimates. Proc Natl Acad Sci U S A  95: 606–611.  <ul class="find" data-citedArticleID="929700" data-doi="10.1073/pnas.95.2.606"><li><a href="http://dx.doi.org/10.1073/pnas.95.2.606" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+the+metazoan+phyla%3A+molecular+clocks+confirm+paleontological+estimates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+the+metazoan+phyla%3A+molecular+clocks+confirm+paleontological+estimates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000204-Peterson1" id="pone.0000204-Peterson1"></a>Peterson KJ, Lyons JB, Nowak KS, Takacs CM, Wargo MJ, et al.  (2004) Estimating metazoan divergence times with a molecular clock. Proc Natl Acad Sci U S A  101: 6536–6541.  <ul class="find" data-citedArticleID="929766" data-doi="10.1073/pnas.0401670101"><li><a href="http://dx.doi.org/10.1073/pnas.0401670101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+metazoan+divergence+times+with+a+molecular+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+metazoan+divergence+times+with+a+molecular+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.XML" value="172424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.PDF" value="968738"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g001.PNG_L" value="990757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g001.PNG_M" value="81114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g001.PNG_S" value="4017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g001.TIF" value="1649892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g001.PNG_I" value="35584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g002.PNG_L" value="2949452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g002.PNG_M" value="158390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g002.PNG_S" value="9188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g002.TIF" value="4116612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g002.PNG_I" value="128256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g003.PNG_L" value="3768835"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g003.PNG_M" value="202701"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g003.PNG_S" value="5383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g003.TIF" value="4067342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g003.PNG_I" value="64408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g004.PNG_L" value="557462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g004.PNG_M" value="52385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g004.PNG_S" value="2495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g004.TIF" value="1298192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g004.PNG_I" value="23962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g005.PNG_L" value="2270287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g005.PNG_M" value="265808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g005.PNG_S" value="10361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g005.TIF" value="2935776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g005.PNG_I" value="131419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g006.PNG_L" value="2276331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g006.PNG_M" value="123146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g006.PNG_S" value="5310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g006.TIF" value="3067560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.g006.PNG_I" value="58667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.e001.PNG" value="10029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.e001.TIF" value="19188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s001.PDF" value="66416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s002.PDF" value="16338"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s003.PDF" value="685194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s004.PDF" value="61703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s005.PDF" value="66037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s006.PDF" value="65895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s007.PDF" value="1776558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s008.PDF" value="12386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s009.PDF" value="130439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s010.PDF" value="66487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s011.PDF" value="179870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s012.PDF" value="93606"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s013.PDF" value="73789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s014.PDF" value="1215165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s015.PDF" value="177644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s016.PDF" value="14719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000204.s017.PDF" value="9459"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000204&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000204" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000204&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000204&volume=&issue=&title=Ancient Origin of the New Developmental Superfamily DANGER&author_name=Nikolas%20Nikolaidis%2C%20Dimitra%20Chalkia%2C%20D.%20Neil%20Watkins%2C%20Roxanne%20K.%20Barrow%2C%20Solomon%20H.%20Snyder%2C%20Damian%20B.%20van%20Rossum%2C%20Randen%20L.%20Patterson&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000204" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204&amp;t=Ancient%20Origin%20of%20the%20New%20Developmental%20Superfamily%20DANGER" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204&title=Ancient%20Origin%20of%20the%20New%20Developmental%20Superfamily%20DANGER&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204&amp;title=Ancient%20Origin%20of%20the%20New%20Developmental%20Superfamily%20DANGER" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000204&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Ancient Origin of the New Developmental Superfamily DANGER';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000204';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Ancient%20Origin%20of%20the%20New%20Developmental%20Superfamily%20DANGER http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000204" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000204" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Database+searching%22" title="Search for articles in the subject area:'Database searching'"><div class="flagText">Database searching</div></a>
              <div data-categoryid="32021" data-articleid="23246"
                   data-categoryname="Database searching"
                   class="flagImage" title="Flag 'Database searching' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23246"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23246"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23246"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="23246"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="23246"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23246"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Vertebrates%22" title="Search for articles in the subject area:'Vertebrates'"><div class="flagText">Vertebrates</div></a>
              <div data-categoryid="43095" data-articleid="23246"
                   data-categoryname="Vertebrates"
                   class="flagImage" title="Flag 'Vertebrates' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9615'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1347'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9238&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
