

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000946"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000946" />

    <meta name="citation_title" content="Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure"/>
    <meta itemprop="name" content="Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure"/>

      <meta name="citation_author" content="Ying Sheng"/>
            <meta name="citation_author_institution" content="Computational Biology Unit, Bergen Center for Computational Science, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Programme for Genomics and Bioinformatics, Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm, Sweden"/>
      <meta name="citation_author" content="Pär G. Engström"/>
            <meta name="citation_author_institution" content="Computational Biology Unit, Bergen Center for Computational Science, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Programme for Genomics and Bioinformatics, Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm, Sweden"/>
      <meta name="citation_author" content="Boris Lenhard"/>
            <meta name="citation_author_institution" content="Computational Biology Unit, Bergen Center for Computational Science, University of Bergen, Bergen, Norway"/>
            <meta name="citation_author_institution" content="Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway"/>

    <meta name="citation_date" content="2007/9/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000946.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e946"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=MicroRNAs: genomics, biogenesis, mechanism, and function.; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=116; citation_number=1; citation_pages=281-297; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=miRBase: microRNA sequences, targets and gene nomenclature.; citation_author=S Griffiths-Jones; citation_author=RJ Grocock; citation_author=S van Dongen; citation_author=A Bateman; citation_author=AJ Enright; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=2; citation_pages=D140-144; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The microRNA Registry.; citation_author=S Griffiths-Jones; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=3; citation_pages=D109-111; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs in mammalian development.; citation_author=CT Lee; citation_author=T Risom; citation_author=WM Strauss; citation_journal_title=Birth Defects Res C Embryo Today; citation_volume=78; citation_number=4; citation_pages=129-139; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Human MicroRNA targets.; citation_author=B John; citation_author=AJ Enright; citation_author=A Aravin; citation_author=T Tuschl; citation_author=C Sander; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=5; citation_pages=e363; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets.; citation_author=BP Lewis; citation_author=CB Burge; citation_author=DP Bartel; citation_journal_title=Cell; citation_volume=120; citation_number=6; citation_pages=15-20; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of hundreds of conserved and nonconserved human microRNAs.; citation_author=I Bentwich; citation_author=A Avniel; citation_author=Y Karov; citation_author=R Aharonov; citation_author=S Gilad; citation_journal_title=Nat Genet; citation_volume=37; citation_number=7; citation_pages=766-770; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic shadowing and computational identification of human microRNA genes.; citation_author=E Berezikov; citation_author=V Guryev; citation_author=J van de Belt; citation_author=E Wienholds; citation_author=RH Plasterk; citation_journal_title=Cell; citation_volume=120; citation_number=8; citation_pages=21-24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of tissue-specific microRNAs from mouse.; citation_author=M Lagos-Quintana; citation_author=R Rauhut; citation_author=A Yalcin; citation_author=J Meyer; citation_author=W Lendeckel; citation_journal_title=Curr Biol; citation_volume=12; citation_number=9; citation_pages=735-739; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Microarray analysis of microRNA expression in the developing mammalian brain.; citation_author=EA Miska; citation_author=E Alvarez-Saavedra; citation_author=M Townsend; citation_author=A Yoshii; citation_author=N Sestan; citation_journal_title=Genome Biol; citation_volume=5; citation_number=10; citation_pages=R68; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A pancreatic islet-specific microRNA regulates insulin secretion.; citation_author=MN Poy; citation_author=L Eliasson; citation_author=J Krutzfeldt; citation_author=S Kuwajima; citation_author=X Ma; citation_journal_title=Nature; citation_volume=432; citation_number=11; citation_pages=226-230; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stage-specific expression of microRNAs during Xenopus development.; citation_author=T Watanabe; citation_author=A Takeda; citation_author=K Mise; citation_author=T Okuno; citation_author=T Suzuki; citation_journal_title=FEBS Lett; citation_volume=579; citation_number=12; citation_pages=318-324; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification of messenger RNAs and microRNAs associated with fragile X mental retardation protein.; citation_author=R Duan; citation_author=P Jin; citation_journal_title=Methods Mol Biol; citation_volume=342; citation_number=13; citation_pages=267-276; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The human DiGeorge syndrome critical region gene 8 and Its D. melanogaster homolog are required for miRNA biogenesis.; citation_author=M Landthaler; citation_author=A Yalcin; citation_author=T Tuschl; citation_journal_title=Curr Biol; citation_volume=14; citation_number=14; citation_pages=2162-2167; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MicroRNAs in mammalian development and tumorigenesis.; citation_author=JR O'Rourke; citation_author=MS Swanson; citation_author=BD Harfe; citation_journal_title=Birth Defects Res C Embryo Today; citation_volume=78; citation_number=15; citation_pages=172-179; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA expression profiles classify human cancers.; citation_author=J Lu; citation_author=G Getz; citation_author=EA Miska; citation_author=E Alvarez-Saavedra; citation_author=J Lamb; citation_journal_title=Nature; citation_volume=435; citation_number=16; citation_pages=834-838; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of two novel classes of small RNAs in the mouse germline: retrotransposon-derived siRNAs in oocytes and germline small RNAs in testes.; citation_author=T Watanabe; citation_author=A Takeda; citation_author=T Tsukiyama; citation_author=K Mise; citation_author=T Okuno; citation_journal_title=Genes Dev; citation_volume=20; citation_number=17; citation_pages=1732-1743; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Embryonic stem cell-specific MicroRNAs.; citation_author=HB Houbaviy; citation_author=MF Murray; citation_author=PA Sharp; citation_journal_title=Dev Cell; citation_volume=5; citation_number=18; citation_pages=351-358; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Numerous microRNPs in neuronal cells containing novel microRNAs.; citation_author=J Dostie; citation_author=Z Mourelatos; citation_author=M Yang; citation_author=A Sharma; citation_author=G Dreyfuss; citation_journal_title=Rna; citation_volume=9; citation_number=19; citation_pages=180-186; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=RNAi-mediated allelic trans-interaction at the imprinted Rtl1/Peg11 locus.; citation_author=E Davis; citation_author=F Caiment; citation_author=X Tordoir; citation_author=J Cavaille; citation_author=A Ferguson-Smith; citation_journal_title=Curr Biol; citation_volume=15; citation_number=20; citation_pages=743-749; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The expression profile of microRNAs in mouse embryos.; citation_author=J Mineno; citation_author=S Okamoto; citation_author=T Ando; citation_author=M Sato; citation_author=H Chono; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=21; citation_pages=1765-1771; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A large imprinted microRNA gene cluster at the mouse Dlk1-Gtl2 domain.; citation_author=H Seitz; citation_author=H Royo; citation_author=ML Bortolin; citation_author=SP Lin; citation_author=AC Ferguson-Smith; citation_journal_title=Genome Res; citation_volume=14; citation_number=22; citation_pages=1741-1748; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=New microRNAs from mouse and human.; citation_author=M Lagos-Quintana; citation_author=R Rauhut; citation_author=J Meyer; citation_author=A Borkhardt; citation_author=T Tuschl; citation_journal_title=Rna; citation_volume=9; citation_number=23; citation_pages=175-179; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Reduced accumulation of specific microRNAs in colorectal neoplasia.; citation_author=MZ Michael; citation_author=OC SM; citation_author=NG van Holst Pellekaan; citation_author=GP Young; citation_author=RJ James; citation_journal_title=Mol Cancer Res; citation_volume=1; citation_number=24; citation_pages=882-891; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-directed cleavage of HOXB8 mRNA.; citation_author=S Yekta; citation_author=IH Shih; citation_author=DP Bartel; citation_journal_title=Science; citation_volume=304; citation_number=25; citation_pages=594-596; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=miRNPs: a novel class of ribonucleoproteins containing numerous microRNAs.; citation_author=Z Mourelatos; citation_author=J Dostie; citation_author=S Paushkin; citation_author=A Sharma; citation_author=B Charroux; citation_journal_title=Genes Dev; citation_volume=16; citation_number=26; citation_pages=720-728; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of many microRNAs that copurify with polyribosomes in mammalian neurons.; citation_author=J Kim; citation_author=A Krichevsky; citation_author=Y Grad; citation_author=GD Hayes; citation_author=KS Kosik; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=27; citation_pages=360-365; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of novel genes coding for small expressed RNAs.; citation_author=M Lagos-Quintana; citation_author=R Rauhut; citation_author=W Lendeckel; citation_author=T Tuschl; citation_journal_title=Science; citation_volume=294; citation_number=28; citation_pages=853-858; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of new central nervous system specific mouse microRNAs.; citation_author=G Wheeler; citation_author=S Ntounia-Fousara; citation_author=B Granda; citation_author=T Rathjen; citation_author=T Dalmay; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=29; citation_pages=2195-2200; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA Mirn122a reduces expression of the posttranscriptionally regulated germ cell transition protein 2 (Tnp2) messenger RNA (mRNA) by mRNA cleavage.; citation_author=Z Yu; citation_author=T Raabe; citation_author=NB Hecht; citation_journal_title=Biol Reprod; citation_volume=73; citation_number=30; citation_pages=427-433; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mouse microRNA profiles determined with a new and sensitive cloning method.; citation_author=S Takada; citation_author=E Berezikov; citation_author=Y Yamashita; citation_author=M Lagos-Quintana; citation_author=WP Kloosterman; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=31; citation_pages=e115; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Many novel mammalian microRNA candidates identified by extensive cloning and RAKE analysis.; citation_author=E Berezikov; citation_author=G van Tetering; citation_author=M Verheul; citation_author=J van de Belt; citation_author=L van Laake; citation_journal_title=Genome Res; citation_volume=16; citation_number=32; citation_pages=1289-1298; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Approaches to microRNA discovery.; citation_author=E Berezikov; citation_author=E Cuppen; citation_author=RH Plasterk; citation_journal_title=Nat Genet 38 Suppl; citation_number=33; citation_pages=S2-7; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Kernel-based machine learning protocol for predicting DNA-binding proteins.; citation_author=N Bhardwaj; citation_author=RE Langlois; citation_author=G Zhao; citation_author=H Lu; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=34; citation_pages=6486-6493; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hierarchical multi-label prediction of gene function.; citation_author=Z Barutcuoglu; citation_author=RE Schapire; citation_author=OG Troyanskaya; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=35; citation_pages=830-836; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MultiLoc: prediction of protein subcellular localization using N-terminal targeting sequences, sequence motifs and amino acid composition.; citation_author=A Hoglund; citation_author=P Donnes; citation_author=T Blum; citation_author=HW Adolph; citation_author=O Kohlbacher; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=36; citation_pages=1158-1165; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of clustered microRNAs using an ab initio prediction method.; citation_author=A Sewer; citation_author=N Paul; citation_author=P Landgraf; citation_author=A Aravin; citation_author=S Pfeffer; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=37; citation_pages=267; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Classification of real and pseudo microRNA precursors using local structure-sequence features and support vector machine.; citation_author=C Xue; citation_author=F Li; citation_author=T He; citation_author=GP Liu; citation_author=Y Li; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=38; citation_pages=310; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reliable prediction of Drosha processing sites improves microRNA gene prediction.; citation_author=SA Helvik; citation_author=O Snove; citation_author=P Saetrom; citation_journal_title=Bioinformatics; citation_volume=23; citation_number=39; citation_pages=142-149; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=De Novo SVM Classification of Precursor MicroRNAs from Genomic Pseudo Hairpins Using Global and Intrinsic Folding Measures.; citation_author=SN Kwang Loong; citation_author=SK Mishra; citation_journal_title=Bioinformatics; citation_number=40; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Hairpins in a Haystack: recognizing microRNA precursors in comparative genomics data.; citation_author=J Hertel; citation_author=PF Stadler; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=41; citation_pages=e197-202; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Combining SVMs with Various Feature Selection Strategies.; citation_author=Y-W Chen; citation_author=C-J Lin; citation_number=42; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evidence that microRNA precursors, unlike other non-coding RNAs, have lower folding free energies than random sequences.; citation_author=E Bonnet; citation_author=J Wuyts; citation_author=P Rouze; citation_author=Y Van de Peer; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=43; citation_pages=2911-2917; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The microRNAs of Caenorhabditis elegans.; citation_author=LP Lim; citation_author=NC Lau; citation_author=EG Weinstein; citation_author=A Abdelhakim; citation_author=S Yekta; citation_journal_title=Genes Dev; citation_volume=17; citation_number=44; citation_pages=991-1008; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes.; citation_author=A Siepel; citation_author=G Bejerano; citation_author=JS Pedersen; citation_author=AS Hinrichs; citation_author=M Hou; citation_journal_title=Genome Res; citation_volume=15; citation_number=45; citation_pages=1034-1050; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide identification of human functional DNA using a neutral indel model.; citation_author=G Lunter; citation_author=CP Ponting; citation_author=J Hein; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=46; citation_pages=e5; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Ultraconserved elements in the human genome.; citation_author=G Bejerano; citation_author=M Pheasant; citation_author=I Makunin; citation_author=S Stephen; citation_author=WJ Kent; citation_journal_title=Science; citation_volume=304; citation_number=47; citation_pages=1321-1325; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Arrays of ultraconserved non-coding regions span the loci of key developmental genes in vertebrate genomes.; citation_author=A Sandelin; citation_author=P Bailey; citation_author=S Bruce; citation_author=PG Engstrom; citation_author=JM Klos; citation_journal_title=BMC Genomics; citation_volume=5; citation_number=48; citation_pages=99; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Highly conserved non-coding sequences are associated with vertebrate development.; citation_author=A Woolfe; citation_author=M Goodson; citation_author=DK Goode; citation_author=P Snell; citation_author=GK McEwen; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=49; citation_pages=e7; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=New technologies, new findings, and new concepts in the study of vertebrate cis-regulatory sequences.; citation_author=JL Gomez-Skarmeta; citation_author=B Lenhard; citation_author=TS Becker; citation_journal_title=Dev Dyn; citation_volume=235; citation_number=50; citation_pages=870-885; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A global genomic transcriptional code associated with CNS-expressed genes.; citation_author=PJ Bailey; citation_author=JM Klos; citation_author=E Andersson; citation_author=M Karlen; citation_author=M Kallstrom; citation_journal_title=Exp Cell Res; citation_volume=312; citation_number=51; citation_pages=3108-3119; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MicroRNA genes are transcribed by RNA polymerase II.; citation_author=Y Lee; citation_author=M Kim; citation_author=J Han; citation_author=KH Yeom; citation_author=S Lee; citation_journal_title=Embo J; citation_volume=23; citation_number=52; citation_pages=4051-4060; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human microRNAs are processed from capped, polyadenylated transcripts that can also function as mRNAs.; citation_author=X Cai; citation_author=CH Hagedorn; citation_author=BR Cullen; citation_journal_title=RNA; citation_volume=10; citation_number=53; citation_pages=1957-1966; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Primary transcripts and expressions of mammal intergenic microRNAs detected by mapping ESTs to their flanking sequences.; citation_author=J Gu; citation_author=T He; citation_author=Y Pei; citation_author=F Li; citation_author=X Wang; citation_journal_title=Mamm Genome; citation_volume=17; citation_number=54; citation_pages=1033-1041; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=CAGE: cap analysis of gene expression.; citation_author=R Kodzius; citation_author=M Kojima; citation_author=H Nishiyori; citation_author=M Nakamura; citation_author=S Fukuda; citation_journal_title=Nat Methods; citation_volume=3; citation_number=55; citation_pages=211-222; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome.; citation_author=P Carninci; citation_author=T Kasukawa; citation_author=S Katayama; citation_author=J Gough; citation_author=MC Frith; citation_journal_title=Science; citation_volume=309; citation_number=56; citation_pages=1559-1563; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Combining multi-species genomic data for microRNA identification using a Naive Bayes classifier.; citation_author=M Yousef; citation_author=M Nebozhyn; citation_author=H Shatkay; citation_author=S Kanterakis; citation_author=LC Showe; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=57; citation_pages=1325-1334; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification and classification of conserved RNA secondary structures in the human genome.; citation_author=JS Pedersen; citation_author=G Bejerano; citation_author=A Siepel; citation_author=K Rosenbloom; citation_author=K Lindblad-Toh; citation_journal_title=PLoS Comput Biol; citation_volume=2; citation_number=58; citation_pages=e33; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome.; citation_author=S Washietl; citation_author=IL Hofacker; citation_author=M Lukasser; citation_author=A Huttenhofer; citation_author=PF Stadler; citation_journal_title=Nat Biotechnol; citation_volume=23; citation_number=59; citation_pages=1383-1390; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Large-scale sequencing reveals 21U-RNAs and additional microRNAs and endogenous siRNAs in C. elegans.; citation_author=JG Ruby; citation_author=C Jan; citation_author=C Player; citation_author=MJ Axtell; citation_author=W Lee; citation_journal_title=Cell; citation_volume=127; citation_number=60; citation_pages=1193-1207; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The UCSC Genome Browser Database.; citation_author=D Karolchik; citation_author=R Baertsch; citation_author=M Diekhans; citation_author=TS Furey; citation_author=A Hinrichs; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=61; citation_pages=51-54; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes.; citation_author=WJ Kent; citation_author=R Baertsch; citation_author=A Hinrichs; citation_author=W Miller; citation_author=D Haussler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=62; citation_pages=11484-11489; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure.; citation_author=DH Mathews; citation_author=J Sabina; citation_author=M Zuker; citation_author=DH Turner; citation_journal_title=J Mol Biol; citation_volume=288; citation_number=63; citation_pages=911-940; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Aligning multiple genomic sequences with the threaded blockset aligner.; citation_author=M Blanchette; citation_author=WJ Kent; citation_author=C Riemer; citation_author=L Elnitski; citation_author=AF Smit; citation_journal_title=Genome Res; citation_volume=14; citation_number=64; citation_pages=708-715; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=From sequences to shapes and back: a case study in RNA secondary structures.; citation_author=P Schuster; citation_author=W Fontana; citation_author=PF Stadler; citation_author=IL Hofacker; citation_journal_title=Proc Biol Sci; citation_volume=255; citation_number=65; citation_pages=279-284; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=LAGAN and Multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA.; citation_author=M Brudno; citation_author=CB Do; citation_author=GM Cooper; citation_author=MF Kim; citation_author=E Davydov; citation_journal_title=Genome Res; citation_volume=13; citation_number=66; citation_pages=721-731; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence.; citation_author=TM Lowe; citation_author=SR Eddy; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=67; citation_pages=955-964; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The Vertebrate Genome Annotation (Vega) database.; citation_author=JL Ashurst; citation_author=CK Chen; citation_author=JG Gilbert; citation_author=K Jekosch; citation_author=S Keenan; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=68; citation_pages=D459-465; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The UCSC Known Genes.; citation_author=F Hsu; citation_author=WJ Kent; citation_author=H Clawson; citation_author=RM Kuhn; citation_author=M Diekhans; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=69; citation_pages=1036-1046; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=TFBS: Computational framework for transcription factor binding site analysis.; citation_author=B Lenhard; citation_author=WW Wasserman; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=70; citation_pages=1135-1136; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Complex Loci in human and mouse genomes.; citation_author=PG Engstrom; citation_author=H Suzuki; citation_author=N Ninomiya; citation_author=A Akalin; citation_author=L Sessa; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=71; citation_pages=e47; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A new generation of JASPAR, the open-access repository for transcription factor binding site profiles.; citation_author=D Vlieghe; citation_author=A Sandelin; citation_author=PJ De Bleser; citation_author=K Vleminckx; citation_author=WW Wasserman; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=72; citation_pages=D95-97; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure"/>
    <meta name="twitter:description" content="BackgroundMicroRNAs (miRNAs) are endogenous small noncoding RNA gene products, on average 22 nt long, found in a wide variety of organisms. They play important regulatory roles by targeting mRNAs for degradation or translational repression. There are 377 known mouse miRNAs and 475 known human miRNAs in the May 2007 release of the miRBase database, the majority of which are conserved between the two species. A number of recent reports imply that it is likely that many mammalian miRNAs remain to be discovered. The possibility that there are more of them expressed at lower levels or in more specialized expression contexts calls for the exploitation of genome sequence information to accelerate their discovery.Methodology/Principal FindingsIn this article, we describe a computational method-mirCoS-that uses three support vector machine models sequentially to discover new miRNA candidates in mammalian genomes based on sequence, secondary structure, and conservation. mirCoS can efficiently detect the majority of known miRNAs and predicts an extensive set of hairpin structures based on human-mouse comparisons. In total, 3476 mouse candidates and 3441 human candidates were found. These hairpins are more similar to known miRNAs than to negative controls in several aspects not considered by the prediction algorithm. A significant fraction of predictions is supported by existing expression evidence.Conclusions/SignificanceUsing a novel approach, mirCoS performs comparably to or better than existing miRNA prediction methods, and contributes a significant number of new candidate miRNAs for experimental verification."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000946.g008"/>

  <meta property="og:title" content="Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6642'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6132'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2985&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000946">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000946" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000946&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ying Sheng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Computational Biology Unit, Bergen Center for Computational Science, University of Bergen, Bergen, Norway, 
                  Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway, 
                  Programme for Genomics and Bioinformatics, Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pär G. Engström, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Computational Biology Unit, Bergen Center for Computational Science, University of Bergen, Bergen, Norway, 
                  Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway, 
                  Programme for Genomics and Bioinformatics, Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm, Sweden
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Boris Lenhard
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:Boris.Lenhard@bccs.uib.no">Boris.Lenhard@bccs.uib.no</a></p>

                <p>Affiliations:
                  Computational Biology Unit, Bergen Center for Computational Science, University of Bergen, Bergen, Norway, 
                  Sars International Centre for Marine Molecular Biology, University of Bergen, Bergen, Norway
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000946</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000946-g001" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-t001" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g002" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-t002" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g003" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g004" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-t003" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g005" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g006" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-t004" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-t005" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g007" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000946-g008" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000946">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000946" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>MicroRNAs (miRNAs) are endogenous small noncoding RNA gene products, on average 22 nt long, found in a wide variety of organisms. They play important regulatory roles by targeting mRNAs for degradation or translational repression. There are 377 known mouse miRNAs and 475 known human miRNAs in the May 2007 release of the miRBase database, the majority of which are conserved between the two species. A number of recent reports imply that it is likely that many mammalian miRNAs remain to be discovered. The possibility that there are more of them expressed at lower levels or in more specialized expression contexts calls for the exploitation of genome sequence information to accelerate their discovery.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In this article, we describe a computational method-mirCoS-that uses three support vector machine models sequentially to discover new miRNA candidates in mammalian genomes based on sequence, secondary structure, and conservation. mirCoS can efficiently detect the majority of known miRNAs and predicts an extensive set of hairpin structures based on human-mouse comparisons. In total, 3476 mouse candidates and 3441 human candidates were found. These hairpins are more similar to known miRNAs than to negative controls in several aspects not considered by the prediction algorithm. A significant fraction of predictions is supported by existing expression evidence.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Using a novel approach, mirCoS performs comparably to or better than existing miRNA prediction methods, and contributes a significant number of new candidate miRNAs for experimental verification.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Sheng Y, Engström PG, Lenhard B (2007) Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure. PLoS ONE 2(9):
          e946.
            doi:10.1371/journal.pone.0000946</p><p><strong>Academic Editor: </strong>Xiaolin Wu, National Cancer Institute at Frederick, United States of America</p><p><strong>Received:</strong> July 5, 2007; <strong>Accepted:</strong> September 7, 2007; <strong>Published:</strong> September 26, 2007</p><p><strong>Copyright:</strong> © 2007 Sheng et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Functional Genomics Programme (FUGE) of the Research Council of Norway and a core grant from the Sars Centre for Marine Molecular Biology.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>MicroRNAs (miRNAs) are an abundant class of ~22 nt long endogenous non-protein-coding RNAs that function by binding to target sites on 3′-UTRs of messenger RNAs (mRNAs) to repress translation or mediate mRNA degradation (reviewed in <a href="#pone.0000946-Bartel1">[1]</a>). Mature miRNAs are synthesized from longer 70–100 nt precursors (pre-miRNAs), each of which forms a hairpin structure that contains one or two mature miRNAs in either or both of its arms. Thus far, a total of 4584 miRNAs have been identified and reported to miRBase (release 9.2, May 2007), a database that stores experimentally validated miRNAs and their homologs <a href="#pone.0000946-GriffithsJones1">[2]</a>, <a href="#pone.0000946-GriffithsJones2">[3]</a>. Several recent observations indicate that miRNAs, in general, may be essential for organisms to differentiate into multiple cell- and tissue types and/or to keep cells in a particular differentiation state <a href="#pone.0000946-Lee1">[4]</a>. miRNA target prediction in mammals indicates that ~10–30% of protein-coding genes may be under control of currently known miRNAs <a href="#pone.0000946-John1">[5]</a>, <a href="#pone.0000946-Lewis1">[6]</a>. This number may still increase because many additional miRNAs have been predicted. There are currently 475 human miRNAs and 377 mouse miRNAs in miRBase, but recent studies have suggested that the number of miRNAs in a vertebrate genome can be as many as 800–1000 <a href="#pone.0000946-Bentwich1">[7]</a>, <a href="#pone.0000946-Berezikov1">[8]</a>. The high number of miRNA genes, their diverse expression patterns <a href="#pone.0000946-LagosQuintana1">[9]</a>–<a href="#pone.0000946-Watanabe1">[12]</a> and the abundance of potential miRNA targets suggest that miRNAs are likely to be involved in a broad spectrum of human diseases. Indeed, components required for miRNA procession and/or function have been implicated in fragile X mental retardation <a href="#pone.0000946-Duan1">[13]</a>, DiGeorge syndrome <a href="#pone.0000946-Landthaler1">[14]</a>, and cancer <a href="#pone.0000946-ORourke1">[15]</a>. Lu et al. <a href="#pone.0000946-Lu1">[16]</a> demonstrated recently that miRNAs can indeed be developed into potent cancer markers.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>As miRNAs are likely to play a central role in development and also in disease, it is important to understand their function. An important step towards this would be to assemble a complete catalogue of miRNA genes. Experimental cloning efforts have successfully identified highly expressed miRNAs from various tissues <a href="#pone.0000946-LagosQuintana1">[9]</a>, <a href="#pone.0000946-Poy1">[11]</a>, <a href="#pone.0000946-Watanabe2">[17]</a>–<a href="#pone.0000946-Berezikov2">[32]</a>. However, cloning methods are highly biased towards miRNAs that are abundantly and/or ubiquitously expressed. On the other hand, computational prediction of miRNAs could become a powerful aid for finding tissue-specific or lowly expressed miRNAs. A number of computational methods for miRNA prediction have been described and appear to complement each other because they take different approaches to miRNA prediction (reviewed in <a href="#pone.0000946-Berezikov3">[33]</a>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Support vector machines (SVMs) are machine learning algorithms widely used to solve classification problems. A SVM assigns an object to one of several classes based on a set of input features associated with the object. In bioinformatics, superiority of SVMs over other classification methods has been shown for prediction of DNA-binding proteins <a href="#pone.0000946-Bhardwaj1">[34]</a>, gene function <a href="#pone.0000946-Barutcuoglu1">[35]</a> and protein subcellular localization <a href="#pone.0000946-Hoglund1">[36]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here we describe a method-mirCoS-to predict conserved miRNAs in mammalian genomes. Being fundamental functional RNA elements, many pre-miRNAs have maintained sequence and secondary structure conservation across large evolutionary distances. Conservation can be described by several features which can serve as inputs for a SVM model. Currently, five other SVM-based miRNA prediction algorithms are available. Four of them are aimed at non-conserved miRNA prediction <a href="#pone.0000946-Sewer1">[37]</a>–<a href="#pone.0000946-KwangLoong1">[40]</a>, and therefore they can not achieve sufficient specificity when applied to entire large genomes. The fifth is aimed at annotating the results from non-coding RNA prediction <a href="#pone.0000946-Hertel1">[41]</a>, and has better specificity, predicting about 5000 miRNAs in the human genome. However, this number is still about five times higher than recent estimates <a href="#pone.0000946-Bentwich1">[7]</a>, <a href="#pone.0000946-Berezikov1">[8]</a> and may include many false positives. To improve on this, we have added a number of previously unused features, and built a composite SVM model consisting of three sequentially applied SVMs and applied it to the human and mouse genomes. We predict about 3400 human-mouse conserved candidate miRNA genes, many of which show evidence of sequence and secondary structure conservation across all vertebrates (including fish) and are supported by independent evidence not used by our prediction method. Finally, we show that mirCoS performs better than or comparably to other recent methods. Because the great majority of our predictions are novel, the method described here constitutes a worthy addition to the arsenal of computational methods aimed at completing the mammalian miRNA collection.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Design and validation of mirCoS-a SVM-based method for miRNA prediction</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>SVMs classify objects based on a set of features for each object. With the goal of predicting mature miRNAs, we selected features describing three aspects of precursor and mature miRNAs: (1) sequence conservation of pre-miRNA, (2) secondary structure of pre-miRNA and its conservation, and (3) placement and secondary structure of mature miRNA within its pre-miRNA (<a href="#pone-0000946-t001">Table 1</a>). We chose to train one SVM (SVM1) for the first of these feature sets, another (SVM2) for the second, and two SVMs (one for human and one for mouse data, collectively termed SVM3) for the third feature set. We applied the SVMs sequentially, so that only candidates that were classified as positive by SVM1 were passed on to SVM2, and similarly for SVM2 and SVM3 (<a href="#pone-0000946-g001">Figure 1</a>). The main reason for dividing the prediction task over three SVMs was to reduce running time: the second and third feature sets require secondary structure predictions that are expensive to compute for whole genomes, and the number of objects to classify increases almost 26-fold for the third feature set, because there are many putative miRNA positions within each pre-miRNA hairpin. Additionally, the sieve effect of sequential application of three SVMs aided in increasing the specificity of predictions, which we consider more important than sensitivity at this stage of the search for unknown candidate miRNAs and their selection for experimental validation.</p>
<div class="figure" id="pone-0000946-g001"><div class="img"><a name="pone-0000946-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Outline of the mirCoS method.</span></strong></p><span>doi:10.1371/journal.pone.0000946.g001</span></div><div class="figure" id="pone-0000946-t001"><div class="img"><a name="pone-0000946-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Features used in mirCoS.</span></strong></p><span>doi:10.1371/journal.pone.0000946.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We applied mirCoS to 976,746 regions from the mouse genome that are conserved in other vertebrates (conserved region set, CRS; see <a href="#s3">Methods</a>). The CRS regions have a median size of 90 bp and cover 121,685,671 bp in total. Positive and negative examples are required to train SVMs and evaluate their performance. Our positive training examples for SVM1 consisted of all 310 regions in the CRS that overlapped known mouse miRNAs from miRBase release 9.1, and our positive training examples for SVM2 and SVM3 were derived from this set. Our negative training examples consisted of regions selected randomly from the rest of the CRS such that, for each training set and chromosome, the number of negative examples was the same as the number of positive examples. Because only a very low fraction of the mouse genome sequence is likely to encode miRNA, it is safe to assume that most negative training examples are not miRNAs. This can also be verified from the final result: only 0.3% of the regions in the CRS were classified as pre-miRNA. See <a href="#s3">Methods</a> for further details about the construction of training sets.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>For the selection of features to include in SVMs we used the F-score, which measures the discriminatory power of individual features. The F-score is related to the F-statistic used in analysis of variance, and has been shown to perform well in selecting features for SVMs <a href="#pone.0000946-Chen1">[42]</a>. <a href="#pone-0000946-t001">Table 1</a> lists all selected features and their F-scores. A description of our rationale for choosing to evaluate these particular features follows.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>To find appropriate features for describing the sequence conservation of pre-miRNAs, we inspected the vertebrate conservation track in the University of California Santa Cruz (UCSC) Genome Browser (<a href="http://www.genome.ucsc.edu/">http://www.genome.ucsc.edu/</a>) (<a href="#pone-0000946-g002">Figure 2A</a>). Pre-miRNA genes are often highly conserved, but conservation drops off rapidly at their edges. Within pre-miRNAs, variations are more likely to occur in the central part of the conservation block, which corresponds to the loop part when the sequence is folded into a stem-loop structure. The generality of these observations is reflected in a cumulative human-mouse conservation profile based on all known mouse miRNAs that can be aligned to the human genome (<a href="#pone-0000946-g002">Figure 2B</a>). Similar, but less steep, miRNA conservation profiles have been distinguished in alignments of multiple primate species <a href="#pone.0000946-Berezikov1">[8]</a>. Consistent with this observation that most known pre-miRNA genes tend to stand out as islands of very high conservation in the genome, the best feature we could find for SVM1 was a measure of the ratio of high to intermediate conservation within a classified region (<a href="#pone-0000946-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000946-g002"><div class="img"><a name="pone-0000946-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>pre-miRNAs display a characteristic conservation profile.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Typically, pre-miRNAs are highly conserved, but the conservation drops off rapidly at their borders and is often lower in the middle region, which corresponds to the loop. (A) Conservation profile of known pre-miRNA <em>hsa-mir-1-1</em> in the UCSC Genome browser (<a href="http://www.genome.ucsc.edu/">http://www.genome.ucsc.edu/</a>). (B) Cumulative conservation profile of known mouse pre-miRNAs (from miRBase 8.2) conserved in human. Pre-miRNA regions were extended by 50 bp on each end and length-normalized to the range [-50,50]. The y-axis shows the fraction of analyzed sequences that are conserved at the position indicated on the x-axis.</p>
<span>doi:10.1371/journal.pone.0000946.g002</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>For a candidate region to be passed to SVM2, we required its predicted secondary structure to contain a pre-miRNA-like hairpin. All features in SVM2 relate to pre-miRNA hairpin structures. It has been shown that pre-miRNA hairpins, in contrast to other noncoding RNAs, have lower free energy of folding than randomized sequences with the same nucleotide content <a href="#pone.0000946-Bonnet1">[43]</a>. Accordingly, the best feature we found for SVM2 was normalized free energy of the predicted hairpin (<a href="#pone-0000946-t001">Table 1</a>). If a candidate region is a true pre-miRNA, its secondary structure should be conserved in all species where there is significant sequence conservation. As one of the features for SVM2, we therefore used predicted hairpin secondary structure conservation between each candidate mouse region and the most evolutionary distant genome that it could be aligned to, considering eleven vertebrate genomes at distances ranging from dog and cow to fish (see <a href="#s3">Methods</a>).</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>For SVM3, the most discriminatory features–which were inspired by criteria successfully used for mature miRNA prediction in <em>C. elegans</em> <a href="#pone.0000946-Lim1">[44]</a>-measured the amount and conservation of base-pairing within the part of the predicted secondary structure corresponding to the miRNA. This is readily explained by the fact that mature miRNAs are always on the stems of hairpin structures, and the part of a stem that corresponds to a miRNA tends to have a high level of base pairing.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7"></a><p>We tested the performance of SVM1 and SVM2 by jackknife cross-validation and obtained sensitivity estimates of 92% and 94%, respectively. For SVM3, since the number of positive examples was large, we used a different repeated holdout scheme to estimate its performance (see <a href="#s3">Methods</a>) and obtained an average sensitivity of 85%.</p>


<h4>Prediction of 3400 miRNA genes conserved in sequence and structure</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Application of mirCoS caused the number of candidate regions to decrease dramatically from 976,746 to 3476, while 68% of the known conserved miRNAs (from miRBase 9.1) that were used to train the model were retained. <a href="#pone-0000946-t002">Table 2</a> shows the number of candidates and known miRNAs retained at each step of the prediction pipeline. Our final result set contained 3476 candidate pre-miRNAs from mouse and 3441 from human (<a href="#pone-0000946-t002">Table 2</a>; detail genome coordinates are in <a href="#pone.0000946.s003">Dataset S1</a> and <a href="#pone.0000946.s004">Dataset S2</a>). Six percent of these candidates represent known pre-miRNAs from miRBase 9.1, while the remaining ones are putative novel pre-miRNAs.</p>
<div class="figure" id="pone-0000946-t002"><div class="img"><a name="pone-0000946-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Number of candidates and known miRNAs maintained at each step of genome-wide screening.</span></strong></p><span>doi:10.1371/journal.pone.0000946.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Five predictions of novel pre-miRNAs are illustrated in <a href="#pone-0000946-g003">Figure 3</a>. All these predictions have a very high level of conservation. To verify that the conservation constraints in the model worked well overall, we examined over what evolutionary distance our predictions were conserved in both sequence and structure, considering alignments to eleven vertebrate genomes as we did when computing input features for SVM2. Of our predictions for the mouse genome, 68% were aligned to the genome of an organism at an evolutionary distance ranging from dog and cow to (any of the) fish and showed predicted secondary structure conservation with that organism (<a href="#pone-0000946-g004">Figure 4</a>). Although this is lower than the result for known mouse pre-miRNAs conserved in human (80%), it is more than twofold the result of 27% for the initial set of candidate regions (the CRS) that were classified by SVM1. We obtained a very large enrichment for the farthest conservation examined, that with fish: 12% of predictions had fish conservation (either to zebrafish, <em>Tetraodon</em> or fugu), compared to only 3% of the CRS.</p>
<div class="figure" id="pone-0000946-g003"><div class="img"><a name="pone-0000946-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Predicted secondary structures and conservation profiles of five candidate pre-miRNA genes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The figure shows five examples from our predictions. Black bars indicate which regions of conservation profiles that correspond to predicted hairpins. Secondary structures of candidate pre-miRNAs were predicted by MFOLD v3.1 <a href="#pone.0000946-Mathews1">[63]</a>. Conservation profiles were obtained from the UCSC Genome Browser (<a href="http://www.genome.ucsc.edu/">http://www.genome.ucsc.edu/</a>). The candidates show canonical secondary structures and conservation profiles.</p>
<span>doi:10.1371/journal.pone.0000946.g003</span></div><div class="figure" id="pone-0000946-g004"><div class="img"><a name="pone-0000946-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Candidate pre-miRNAs are conserved in sequence and predicted structure over large evolutionary distances.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>We used alignments between mouse and eleven other organisms to assess over what evolutionary distance each mouse region was conserved in both sequence and structure. Bars indicate what fraction of a particular set of regions that are conserved at a given distance. For each region, we only noted the most evolutionary distant species/clade at which we found it to be conserved. E.g. the leftmost gray bar spans 4%, indicating that 4% of known mouse miRNAs were found to be conserved in opossum, but not in chicken, frog or fish. The requirement for conservation was that regions should align over at least 37 nt and their predicted secondary structures have an RNAdistance score ≤48 (see <a href="#s3">Methods</a>).</p>
<span>doi:10.1371/journal.pone.0000946.g004</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The sequence conservation features used in the model are based on phastCons scores, which primarily reflect patterns of base substitutions <a href="#pone.0000946-Siepel1">[45]</a>. Lunter et al. <a href="#pone.0000946-Lunter1">[46]</a> described a method to identify conserved regions based on rates of base insertions and deletions, and found that most known miRNA genes were within the identified conserved regions. Consistently, 80% of our candidate human miRNA genes overlap with conserved regions identified by Lunter et al. at 10% false discovery rate, compared to only 45% of the regions in the CRS and 3% of randomly selected regions.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Some of our novel candidates are homologs of known pre-miRNAs from other organisms. We BLASTed all our novel mouse candidates against all miRBase 9.1 pre-miRNAs using default blastn settings, and filtering the results to retain alignments (BLAST high-scoring segment pairs; HSPs) of length ≥50 bases and identity ≥75%. There were hits for five candidates: four hit known pre-miRNAs from human and the remaining candidate only hit two pre-miRNAs from chicken (gga-mir-147-1 and gga-mir-147-2). All these hits are likely true homologs, because sequence identity was high (mean: 91.3%, range: 87.1%–97.4%), as was the proportion of known pre-miRNA sequence aligned (mean: 87.0%, range: 58.3%–98.5%), strongly supporting that the five novel mouse candidates are true pre-miRNAs. Further analysis of the candidate that only had BLAST hits in chicken revealed that it was highly similar to mouse miRNA miR-147 cloned by Lagos-Quintana et al. <a href="#pone.0000946-LagosQuintana1">[9]</a>: 19/21 bases from the clone match. Importantly, the part of our candidate hairpin that aligned with miR-147 was within a 26 nt region predicted to contain a mature miRNA by our method. Mouse miR-147 had not been deposited in miRBase because it did not have a sufficiently good match to the mouse genome assembly. In the current mouse genome assembly (NCBI Build 36), the clone has five different matches with 19 identities each, but still no better match. However, Lagos-Quintana et al. found that their clone aligned with 20 identities to a region of the human genome predicted to form a hairpin structure. This region was subsequently deposited in miRBase as hsa-mir-147 based on the evidence from the mouse clone. The chicken miRNAs gga-mir-147-1 and gga-mir-147-2, which have not been experimentally validated, were annotated based on similarity (76% identity over 72 bases) to hsa-mir-147. Our novel human and mouse candidates are also similar to hsa-mir-147 (81% identity over 72 bases, although BLAST only found a 26-base HSP), but more similar to gga-mir-147-1/2 (87–89% identity over 70 bases). Experimental validation is required to show whether the mir-147 entries in miRBase, as well as our novel human and mouse candidates, represent true pre-miRNAs.</p>


<h4>Independent evidence supports the validity of miRNA predictions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To further assess the validity of our predictions, we examined several features that were not used in the prediction pipeline.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>1. <em>Intronic vs. intergenic predictions -</em> The proportion of intronic miRNA genes is very similar between our predictions and known miRNAs: Of the 310 mouse miRNAs in miRBase 9.1 that are conserved (<a href="#pone-0000946-t002">Table 2</a>), we found 30% to be located within introns of protein-coding genes, compared to 35% for our candidates. For conserved human miRNAs, we found 35% of known and 37% of candidate miRNAs to be intronic (<a href="#pone-0000946-t003">Table 3</a>).</p>
<div class="figure" id="pone-0000946-t003"><div class="img"><a name="pone-0000946-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Results from genome-wide screening for human and mouse miRNAs.</span></strong></p><span>doi:10.1371/journal.pone.0000946.t003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p><em>2. Genomic clustering of predictions</em>-Many known miRNA genes occur in clusters along chromosomes. We identified spatial clusters among our predictions as described in <a href="#s3">Methods</a>. Of our mouse candidates, 513 were clustered with one or more other candidates. The mouse candidates formed 209 clusters, 44 of which contained one or more known miRNA genes. Nineteen novel mouse candidates were clustered with a known miRNA gene, lending strong support to the validity of those predictions. The results for human were similar: 494 candidates formed a total of 210 clusters, 47 of which contained one or more known miRNA genes. Thirteen novel human candidates were clustered with a known miRNA.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p><em>3. Pattern composition</em>-Based on their genomic distribution, it was postulated that many highly conserved noncoding elements (HCNEs) may function as developmental enhancers <a href="#pone.0000946-Bejerano1">[47]</a>–<a href="#pone.0000946-Woolfe1">[49]</a>. Several have indeed been demonstrated to possess enhancer function (reviewed in <a href="#pone.0000946-GomezSkarmeta1">[50]</a>) and an enrichment of sequence patterns characteristic of binding sites for certain developmental transcription factors has been found in a large subset of HCNEs <a href="#pone.0000946-Bailey1">[51]</a>. Since sequence patterns were not explicitly considered in mirCoS, we used pattern occurrence as an independent means to assess whether our model had discriminated between likely developmental enhancers and miRNA genes. For this analysis, we used a published HCNE set produced by scanning the human genome for regions with at least 95% sequence identity in mouse, as well as evidence of conservation in <em>Fugu</em> <a href="#pone.0000946-Sandelin1">[48]</a>. <a href="#pone-0000946-g005">Figure 5</a> shows that HCNEs have a much greater incidence of putative binding sites for homeobox transcription factors than do known (p&lt;10<sup>−15</sup>, Wilcoxon test) and candidate (p&lt;10<sup>−15</sup>) miRNA genes. Figures for other transcription factors are in <a href="#pone.0000946.s001">Figure S1</a>. This difference in pattern composition can only be partially accounted for by differences in dinucleotide composition, suggesting that many of the binding sites predicted in HCNEs are functional (<a href="#pone-0000946-g005">Figure 5C</a>). Of our candidate pre-miRNAs, 221 (6%) overlapped with a HCNE. The occurrence of putative homeobox binding sites in the sequences for these 221 candidates was lower than for remaining HCNEs (p = 3×10<sup>−7</sup>, Wilcoxon test), indicating that mirCoS to some extent distinguishes between HCNEs that are miRNA genes and HCNEs that are developmental enhancers (<a href="#pone-0000946-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0000946-g005"><div class="img"><a name="pone-0000946-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>mirCoS can distinguish pre-miRNAs from highly conserved developmental enhancer regions.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>We compared differences in pattern composition among known pre-miRNAs, candidate pre-miRNAs and HCNEs. Each sequence was searched for putative transcription factor binding sites using the familial binding profile for homeobox transcription factors from the JASPAR database <a href="#pone.0000946-Vlieghe1">[72]</a> at a score threshold of 80%. (A) Sequences were partitioned into four non-overlapping sets (I-IV) as indicated in the Venn diagram. (B) Cumulative distributions of number of predicted binding sites per 100 bp for sequence sets I-IV. The distributions for candidate pre-miRNAs (blue, green) are more similar to the distribution for known pre-miRNAs (red) than to the distribution for HCNEs not predicted to be pre-miRNAs (gray). (C) Solid bars show the average number of predicted sites per 100 bp over each of sequence sets I-IV. Shaded bars show results for corresponding control sets: controls for dinucleotide composition generated by, for each sequence, constructing a first-order Markov chain and using it to generate a new sequence (diagonal shading lines), and controls for single nucleotide composition generated by randomly shuffling the bases in each sequence (vertical shading lines). Error bars indicate 95% confidence intervals.</p>
<span>doi:10.1371/journal.pone.0000946.g005</span></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p><em>4. Evidence for transcription</em>-Some miRNA genes are known to be transcribed by RNA polymerase II as large primary miRNA transcripts which have cap structures and poly(A) tails <a href="#pone.0000946-Lee2">[52]</a>, <a href="#pone.0000946-Cai1">[53]</a>. Additional support for the idea that intergenic miRNAs are transcribed as large primary miRNA transcripts comes from a recent study where the flanking genomic sequences (2.5 kb upstream and 4 kb downstream) of many intergenic mammalian miRNAs were found to align with expressed sequence tags (ESTs) <a href="#pone.0000946-Gu1">[54]</a>. In agreement with these results, we found EST and/or cDNA support for 59% of mouse miRNAs in miRBase 8.0 (the version used in <a href="#pone.0000946-Gu1">[54]</a>) and for 50% of mouse miRNAs miRBase 9.1 (<a href="#pone-0000946-t004">Table 4</a>). For our intergenic mouse miRNA predictions not represented in miRBase 9.1, we found EST/cDNA support for transcription of 345 (21%) of 1640. Corresponding rates for intergenic subsets of the CRS and randomly selected genomic regions are significantly lower (13% and 10%, respectively; p =  4×10<sup>−9</sup> and p&lt;10<sup>−15</sup>, respectively, compared to the rate for our predictions with chi-square tests) (<a href="#pone-0000946-t004">Table 4</a>) indicating that our set of miRNA predictions is enriched for transcribed sequences. The lower rate for our candidates compared to known miRNAs is expected, because most known miRNAs have been found by cloning methods, and miRNAs which have not yet been cloned are likely to be expressed at lower levels or in more restricted contexts. To more directly assess whether transcription start sites are present at or closely upstream of our miRNA predictions, we turned to cap analysis of gene expression (CAGE) data. CAGE is a technique to obtain sequence tags (CAGE tags) of about 20 nucleotides from 5′-ends of capped transcripts <a href="#pone.0000946-Kodzius1">[55]</a>. More than seven million CAGE tags have been sequenced from 145 mouse cDNA libraries and mapped to the mouse genome <a href="#pone.0000946-Carninci1">[56]</a>. If primary miRNA transcripts have cap structures, their 5′-ends should be detectable by CAGE. We found 19% of known intergenic mouse pre-miRNAs to have more than one CAGE tag on the pre-miRNA or within 500 bp upstream, compared to 8% of our intergenic candidates (excluding known miRNAs, as for the cDNA/EST comparison above; genome coordinates of mouse and human candidates with CAGE data support are in <a href="#pone.0000946.s005">Dataset S3</a> and <a href="#pone.0000946.s006">Dataset S4</a>). Corresponding rates for the CRS and randomly selected genomic regions are only 4% and 1%, respectively (<a href="#pone-0000946-g006">Figure 6</a>; p = 3×10<sup>−8</sup> and p&lt;10<sup>−15</sup>, respectively, compared to the rate for our predictions with chi-square tests). The difference in transcriptional support between miRBase 9.1 miRNAs and our novel predictions is similar between the comparison with cDNA/EST data and the comparison with CAGE data. However, the negative control sets (CRS and random genomic regions) have less support from CAGE than cDNA/EST data compared to known miRNAs and our predictions. The explanation may be that cDNAs and ESTs represent a variety of overlapping transcribed regions, while CAGE tags near genomic locations of pre-miRNA 5′-ends more specifically pinpoints transcriptional start sites for primary miRNA transcripts.</p>
<div class="figure" id="pone-0000946-g006"><div class="img"><a name="pone-0000946-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>CAGE expression data supports miRNA predictions.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Cumulative distribution of number of CAGE tags mapping to known intergenic pre-miRNA genes or within 500 bp upstream (red), and corresponding distributions for predicted intergenic pre-miRNAs (blue), randomly selected intergenic genomic regions of the same size (green) and intergenic regions from the CRS (black). Known and predicted pre-miRNAs tend to have more overlapping or upstream CAGE tags than either of the control sets. The inset shows a magnification for tags counts of 0–40.</p>
<span>doi:10.1371/journal.pone.0000946.g006</span></div><div class="figure" id="pone-0000946-t004"><div class="img"><a name="pone-0000946-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>cDNA and EST support for transcription of intergenic mouse miRNAs.</span></strong></p><span>doi:10.1371/journal.pone.0000946.t004</span></div>

<h4>Comparison to other methods</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We compared our results to three other studies where SVMs were used to predict human or mouse miRNAs <a href="#pone.0000946-Sewer1">[37]</a>, <a href="#pone.0000946-Xue1">[38]</a>, <a href="#pone.0000946-Hertel1">[41]</a>, as well as to two other recent studies where different techniques were used to predict miRNAs in the human or mouse genome <a href="#pone.0000946-Berezikov1">[8]</a>, <a href="#pone.0000946-Yousef1">[57]</a> (<a href="#pone-0000946-t005">Table 5</a>).</p>
<div class="figure" id="pone-0000946-t005"><div class="img"><a name="pone-0000946-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Comparison between mirCoS and other methods.</span></strong></p><span>doi:10.1371/journal.pone.0000946.t005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Xue et al. <a href="#pone.0000946-Xue1">[38]</a> presented a method called triplet-SVM that recognizes pre-miRNAs based on their composition of small (3 nt) structure features. The method was trained on known human pre-miRNAs and achieved a high sensitivity (~90%) when applied to known pre-miRNAs from human and several other organisms. Unlike mirCoS, triplet-SVM does not make use of cross-species information, and can therefore detect organism-specific miRNAs. However, triplet-SVM was not designed to be applied to whole genomes, even when combined with a conservation filter. As a specificity test, the authors applied it to human-mouse conserved segments from 1 Mb of human chromosome 19. The method classified 270 sequences from the 1 Mb region as putative pre-miRNAs, suggesting that it may predict several hundred thousand pre-miRNAs if applied to the entire human genome. Although triplet-SVM has a high sensitivity, its specificity in a whole-genome scan is therefore not comparable to ours.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Sewer et al. <a href="#pone.0000946-Sewer1">[37]</a> also presented an SVM-based method (miR-abela) that does not make use of cross-species information. It appears that, with current knowledge of miRNA biology, such methods are unable to achieve the specificity required for whole-genome scans. Sewer et al. did not attempt a whole-genome scan, but applied miR-abela to detect clustered miRNA genes in human, mouse and rat genomic regions harboring known miRNAs. They detected 73 human and 51 mouse candidates that were not found by mirCoS, mainly because of our conservation constraints. Of these candidates, 12 and 10 represent known miRNAs (from miRBase 9.1), respectively. Conversely, we found 31 mouse and 26 human conserved miRNA candidates that miR-abela did not detect in the explored regions. Of these candidates, 11 and 10 represent known miRNAs in mouse or human, respectively. Thus, the two methods appear to complement each other well.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>Recently, two methods have been described that detect conserved RNA secondary structures in alignments of multiple genomes <a href="#pone.0000946-Pedersen1">[58]</a>, <a href="#pone.0000946-Washietl1">[59]</a>. Hertel and Stadler <a href="#pone.0000946-Hertel1">[41]</a> described an SVM-based method called RNAmicro, designed to detect pre-miRNAs in the output from such surveys. Like mirCoS, RNAmicro uses twelve different features for SVM classification. Although the exact choice of features differs, they relate to the same aspects of pre-miRNA: sequence, secondary structure and conservation. To compare the performance of mirCoS to RNAmicro, we obtained RNAmicro predictions for the human genome (J. Hertel, personal communication). Starting from conserved secondary structures detected by RNAz <a href="#pone.0000946-Washietl1">[59]</a> and filtering the results at a score cutoff of 0.5, RNAmicro achieved sensitivity similar to our method: out of the 474 known human miRNAs in miRBase 9.1, RNAmicro detected 202 and we detected 208. However, at the same score cutoff of 0.5, RNAmicro predicted 58% more miRNA candidates in the human genome than mirCoS (5440 compared to 3441). From these counts, it appears that mirCoS has a higher specificity than RNAmicro, although RNAmicro specificity has been estimated to be high by comparison with a dataset of non-miRNA noncoding RNAs (J. Hertel, personal communication). The better performance of mirCoS is likely due to more stringent requirements in the model. RNAmicro asks for pre-miRNA secondary structures with stems of at least 10 bp, while mirCoS asks for secondary structures with a stems of at least 17 bp (the length of the shortest known mature miRNA). In mature miRNA prediction, RNAmicro mainly considers conservation of mature miRNAs, while mirCoS also considers base pairing (which is the most important criterion in MIRscan <a href="#pone.0000946-Lim1">[44]</a>) and minimum free energy of mature miRNAs. It is also interesting that the methods predict different candidates. Of the known miRNAs found by RNAmicro, we missed 34, and of those found by our method, RNAmicro missed 40. Comparing the entire candidate sets from the two methods, only 897 candidates (including 168 known miRNAs) are found in both sets. Predicted miRNAs are more likely to be true if clustered with other predicted or known miRNAs. In the human genome, 341 clusters were either created or expanded by adding our predictions to those from RNAmicro and known miRNAs.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Yousef et al. <a href="#pone.0000946-Yousef1">[57]</a> used a different machine learning method, naïve Bayes classifier, to predict miRNAs conserved between human and mouse. They applied their method (BayesMiRNAfind) to the forward strand of the mouse genome sequence and presented results for different score cutoffs. At a cutoff that produced a similar number of miRNA gene predictions (1697) as mirCoS did on the forward strand of the mouse genome (1731), the Bayesian method detected 59 out of 135 known miRNA genes in miRBase 6.0. mirCoS found 111 of the same 135 miRNA genes, thus achieving a much higher sensitivity.</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>Berezikov et al. <a href="#pone.0000946-Berezikov1">[8]</a> presented a method (here called RegEx) to detect miRNA genes by using a set of rules (implemented as regular expressions) to first scan human-rodent conservation profiles and subsequently predicted RNA secondary structures. The property of pre-miRNAs to have lower folding free energy than random sequences was also taken into account by filtering the results with the program Randfold <a href="#pone.0000946-Bonnet1">[43]</a>. We compared mirCoS with RegEx, because the features considered are similar and because, to our knowledge, RegEx represents one of the best miRNA prediction algorithms to date. On the current miRBase release (9.1), mirCoS has a somewhat higher sensitivity, recovering 208 known human miRNAs, among them 53 not found by RegEx, while RegEx recovered 191 known human miRNAs, among them 36 not found by us (<a href="#pone-0000946-g007">Figure 7</a>). Our higher sensitivity may be at the expense of specificity, because in total we predicted about three times more human miRNA genes than RegEx (3441 compared to 976). However, in addition to the 53 known conserved miRNAs missed by RegEx, our results contain many novel candidates that are highly likely to be true miRNAs and were missed by RegEx. Examples include two candidates clustered with known human miRNAs at chr14:100,411,114-100,411,204 and chr14:100,566,140-100,566,220 (coordinates refer to NCBI build 35). In total in the human genome, 196 clusters were either created or expanded by adding our predictions to those from RegEx and known miRNAs (from miRBase 9.1). Moreover, comparison with human CAGE data <a href="#pone.0000946-Carninci1">[56]</a> indicates that mirCoS and RegEx have comparable specificity. We found 6.0% of our intergenic human candidates not represented in miRBase 9.1 to have more than one CAGE tag on the predicted pre-miRNA or within 500 bp upstream. The corresponding rate for intergenic RegEx predictions not represented in miRBase 9.1 was 6.1%, and rates for intergenic subsets of miRBase 9.1, the CRS and randomly selected regions were 9.4%, 2.0% and 0.6% respectively (these rates are lower than those given above for mouse, because less CAGE tags have been sequenced for human). To gain more insight into the differences between mirCoS and RegEx, we applied Randfold as an extra filter to our results, in the same way done in RegEx, and filtered out coding sequence and repeats from the results of RegEx, in the same way done in mirCoS. After this filtering, the trends in sensitivity and specificity remained, but were less pronounced: mirCoS found 181 known human miRNAs, compared to 176 for RegEx, and the total number of predictions was reduced to 1667 and 694, respectively, with 337 predictions shared between the two methods.</p>
<div class="figure" id="pone-0000946-g007"><div class="img"><a name="pone-0000946-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>The overlap between our predictions and those from Berezikov et al. is small.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Venn diagram showing the intersections between human miRNAs predicted by Berezikov et al.<a href="#pone.0000946-Berezikov1">[8]</a> (gray rectangle), our human predictions (large open rectangle) and known human miRNAs (horizontal rectangle).</p>
<span>doi:10.1371/journal.pone.0000946.g007</span></div>

<h4>Opportunities for improvements in miRNA prediction</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Completion of the mammalian miRNA collection is likely to require an ensemble of approaches, including high-throughput sequencing, prediction of conserved miRNAs from multiple-species comparisons, and prediction of miRNAs that are not deeply conserved by analysis of single sequences. Each of these strategies has its drawbacks. Although high-throughput sequencing of small RNAs can identify numerous miRNAs with high specificity <a href="#pone.0000946-Ruby1">[60]</a>, miRNAs with highly restricted expression patterns may be difficult to detect even with massive sequencing capacity, especially in complex organisms. As demonstrated in this and earlier work <a href="#pone.0000946-Berezikov1">[8]</a>, <a href="#pone.0000946-Hertel1">[41]</a>, cross-species conservation is highly informative for miRNA prediction. However, many recently discovered human miRNAs are not deeply conserved <a href="#pone.0000946-Bentwich1">[7]</a>. As discussed in the comparison with different methods above, current methods geared towards prediction of non-conserved miRNAs do not have sufficient specificity for searching large genomes. Very recently, two SVM-based methods for detecting miRNAs without use of cross-species comparisons were published <a href="#pone.0000946-Helvik1">[39]</a>, <a href="#pone.0000946-KwangLoong1">[40]</a>. Despite innovative feature choices and significant improvements over previous methods, these new methods do not achieve sufficient specificity for application to the entire human genome, where they would predict on the order of ten or hundred thousand candidate miRNAs while recovering known miRNAs at a sensitvity of 85-90%.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>All prediction methods are likely to benefit from the fact that new miRNAs are now continuously discovered and validated-with additional positive examples, prediction methods can be better trained. The integration of new genome sequences, miRNA target predictions and expression data into prediction pipelines will lead to further improvements. Moreover, the fact that there is little overlap among predictions from different top-performing miRNA prediction methods <a href="#pone.0000946-Hertel1">[41]</a> suggests that much can be gained from combining strategies used in different methods. Finally, our knowledge of miRNA biology and biochemistry is advancing at a rapid rate: additional knowledge about miRNA biogenesis and target recognition is likely to aid the development of improved prediction strategies <a href="#pone.0000946-Helvik1">[39]</a>.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We have developed a computational method–mirCoS-to predict the location of pre-miRNAs in genomic sequences. Unlike some other recent methods <a href="#pone.0000946-Sewer1">[37]</a>, <a href="#pone.0000946-Xue1">[38]</a>, mirCoS is applicable to whole large genomes. Here, we have applied it to discover new miRNAs in the human and mouse genomes. The resulting predictions can be used to guide experiments that aim to clone and characterize new miRNAs. Importantly, we demonstrated that the performance of the method is very good. The predicted pre-miRNAs resemble known miRNAs in several aspects that were not considered by the prediction algorithm. The method outperforms another recently published method <a href="#pone.0000946-Yousef1">[57]</a> in detecting human-mouse conserved miRNAs, and measures up to and complements other top-performing methods <a href="#pone.0000946-Berezikov1">[8]</a>, <a href="#pone.0000946-Hertel1">[41]</a> because there is little overlap among sets of miRNA candidates predicted by the different methods. Many of the predictions that we report, and that were not found by the methods we have compared with, are likely to represent real miRNAs, because many of them are located close to known miRNAs or miRNA predictions from other methods.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Methods"></a><h3>Methods</h3>
<h4>Sequence, expression and annotation data</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Human genome assembly hg17 (NCBI Build 35), mouse genome assembly mm5 (NCBI Build 33) and annotations [cDNA-to-genome alignments, UCSC Known Gene coordinates, pseudogene and repeat locations, phastCons scores (calculated from multiple alignments of mm5, rn3, hg17, canFam1 and galGal2), pairwise net alignments between different genomes and multiple alignments of 16 vertebrate genomes with mouse (mm7, rn3, oryCun1, hg17, panTro1, rheMac1, canFam2, bosTau2, dasNov1, loxAfr1, echTel1, monDom2, galGal2, xenTro1, danRer3, tetNig1 and fr1)] were downloaded from the UCSC Genome Browser Database <a href="#pone.0000946-Karolchik1">[61]</a>. We obtained genomic coordinates of known miRNAs from miRBase <a href="#pone.0000946-GriffithsJones1">[2]</a>, <a href="#pone.0000946-GriffithsJones2">[3]</a>. The CAGE data was produced in the FANTOM3 project <a href="#pone.0000946-Carninci1">[56]</a> and can be downloaded at the FANTOM3 homepage (<a href="http://fantom.gsc.riken.go.jp">http://fantom.gsc.riken.go.jp</a>). For the final version of the paper, the coordinates of predicted miRNAs have been lifted to NCBI Build 36 coordinates (hg18 and mm8).</p>


<h4>Construction of the conserved region set</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>By inspecting phastCons scores <a href="#pone.0000946-Siepel1">[45]</a> for known mouse miRNAs from miRBase 7.1 in the UCSC Genome Browser, we observed that most mouse miRNAs either coincide with a distinct plateau of high conservation (<a href="#pone-0000946-g002">Figure 2</a>) or have very limited conservation. To construct the input to SVM1, we designed a set of rules for extracting conserved regions, so that the resulting set of conserved regions contained all known miRNAs that we by manual inspection found to have distinct conservation plateaus. We obtained two sets of conserved regions (0.14-regions and 0.5-regions) by scanning the mouse genome for maximal regions with phastCons scores above 0.14 and 0.5, respectively. Short (&lt;37 bp) 0.14-regions were merged with any neighboring 0.14-regions less than 20 bp away. If exactly two 0.5-regions were within the same 0.14-region, we merged them. These merging steps were carried out because some miRNAs have a conservation drop in the part corresponding to the hairpin (<a href="#pone-0000946-g002">Figure 2</a>). If more than two 0.5-regions were within the same 0.14-region, we discarded them. Remaining (single or merged) 0.5-regions that spanned at least 37 bp constituted our set of candidate regions for SVM1. The threshold of 37 bp was chosen because it corresponded to the combination of the shortest known mouse mature miRNA (17 bp) and the shortest loop (3 bp). The conservation cutoffs were set to 0.14 and 0.5, because these were the maximal cutoffs at which all known mouse miRNAs with distinct conservation plateaus were included when requiring that the retained regions span at least 37 bp.</p>


<h4>Composite SVM model</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>We used LIBSVM tools (<a href="http://www.csie.ntu.edu.tw/%E2%88%BCcjlin/libsvm">http://www.csie.ntu.edu.tw/~cjlin/libsvm</a>) to build the SVM models, with a radial basis function as kernel. Kernel parameters were selected by a grid search, using the scripts distributed with LIBSVM. The three SVMs were applied sequentially, as described in Results and Discussion. For each region that was classified as positive by SVM1, we extracted its best match in the human genome according to a net alignment between the two genomes <a href="#pone.0000946-Kent1">[62]</a>. We extracted hairpins that had a single loop and a stem of at least 17 bp from MFOLD v3.1 <a href="#pone.0000946-Mathews1">[63]</a> secondary structure predictions for the human and mouse regions. We discarded all hairpins that in either genome overlapped with repeat annotation on the genome or CDS annotation in cDNA sequence (using all Genbank and RefSeq cDNAs mapped to the genome by UCSC as of May, 2004). Further, we only retained hairpins that were predicted in both organisms, i.e. on corresponding strands of aligned segments. To compute SVM2 and SVM3 features involving energy values, we obtained minimum free energy (MFE) values from MFOLD output, which contains the energy of each structure unit. We first computed an energy value for each nucleotide by dividing the energy of each structure unit (e.g. a base pair) equally among the nucleotides involved, and then computed feature values as the average energy per nucleotide considered for the feature (e.g. the first feature for SVM3 was computed as the average energy over the nucleotides in the predicted hairpin). To measure secondary structure conservation beyond rodents and hominids, we obtained a multiple alignment between mouse genome assembly mm7 (NCBI Build 35) and fifteen other vertebrate genomes from the UCSC Genome Browser Database <a href="#pone.0000946-Blanchette1">[64]</a>. We considered assemblies for eleven vertebrates in the following order: canFam2, bosTau2, dasNov1, loxAfr1, echTel1, monDom2, galGal2, xenTro1, danRer3, tetNig1 and fr1 (<a href="#pone-0000946-g004">Figure 4</a>). We used RNAfold to re-fold mouse hairpin sequences and their aligned sequences from other genomes, and computed conservation with RNAdistance <a href="#pone.0000946-Schuster1">[65]</a>. To generate the features for SVM3, each hairpin classified as positive by SVM2 was scanned with a sliding window of length of 17nt (the size of the shortest known miRNA in mouse). To assess conservation of base pairs in the stems, we aligned human and mouse hairpins from corresponding strands and regions with LAGAN <a href="#pone.0000946-Brudno1">[66]</a>. We trained one instance of SVM3 for human and one for mouse, and only retained candidates that were predicted as positive in both organisms. Following SVM3, we used the tRNAscan-SE Search Server (<a href="http://selab.janelia.org/tRNAscan-SE">http://selab.janelia.org/tRNAscan-SE</a>)<a href="#pone.0000946-Lowe1">[67]</a> to eliminate putative transfer RNAs from the results. We also eliminated any predictions that overlapped with Vega <a href="#pone.0000946-Ashurst1">[68]</a> or Yale (<a href="http://www.pseudogene.org">http://www.pseudogene.org</a>) pseudogene annotations in the human genome. Finally, because new protein-coding genes are still being discovered, we discarded all predictions that overlapped with CDS annotation from human and mouse UCSC Known Genes as of March, 2006 <a href="#pone.0000946-Hsu1">[69]</a>.</p>


<h4>SVM training and testing</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Our set of positive examples for training SVM1 consisted of all regions in the candidate set that overlapped a known mouse miRNA (from miRBase 9.1), so that the overlap accounted for at least half of the length of the pre-miRNA. Our set of positive examples for training SVM2 consisted of all known mouse pre-miRNA hairpins retained after applying SVM1, and folding and filtering the sequences as described above. We obtained positive examples for training SVM3 by sliding a 17nt window across all known miRNAs in the hairpins retained after SVM2. For each SVM, negative examples were obtained by randomly sampling from the entire set of candidate regions that were retained after applying previous SVMs but did not represent known miRNAs. The sampling was done so that, for each training set and chromosome, the number of negative examples was the same as the number of positive examples.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>To determine the optimal proportions of positive and negative examples for training SVM1, we randomly separated the positive examples into two sets of equal size. One of these was used to construct nine different training sets, by adding negative examples at a proportion ranging from 10% to 90%. We trained one SVM for each of these training sets, and used the remaining positive examples and the entire CRS as negative examples to evaluate SVM performance. Based on sensitivity and number regions classified as miRNA, we chose to use equal numbers of positive and negative examples in the final training set (<a href="#pone-0000946-g008">Figure 8</a>). Although we only performed this test on SVM1, we chose to use equal numbers of positive and negative examples for all three SVMs.</p>
<div class="figure" id="pone-0000946-g008"><div class="img"><a name="pone-0000946-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000946" data-uri="info:doi/10.1371/journal.pone.0000946.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000946.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000946.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000946.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Effects of varying the proportion of positive entries in the training set for SVM1.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>Proportions of positive examples that were correctly classified (sensitivity, red line) and negative examples that were incorrectly classified (1-specificity, black line), as functions of the proportion of positive entries in the training set. Ideally, sensitivity should be maximized while 1-specificity should be minimized. From the figure, we can see that 1-specificity increases in three stages. At the beginning of the second increasing stage, sensitivity has already entered into a slowly increasing stage. We therefore chose this point (x = 50%, vertical line) as the proportion of positive entries to use in the training set.</p>
<span>doi:10.1371/journal.pone.0000946.g008</span></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Because the number of known miRNAs is quite small, we did not separate them into distinct training and test sets. For SVM1 and SVM2, we instead validated performance by jackknife (a.k.a. leave-one-out) cross-validation. For SVM3, the number of positive examples was larger because of the sliding window approach used, and training SVM3 on all positive examples would have been too time-consuming. We therefore randomly divided them into 6 groups (for mouse) and 6 groups (for human), and added the same number of negative examples to each group. We trained one model with each of these groups, and tested each model on a different group. The cross-validation rate we report is the average over all groups. The training group with best sensitivity and smallest number of regions classified as positive was chosen as the training set for the final procedure.</p>


<h4>Feature selection</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We used F-scores to measure the discriminatory power of each feature. The F-score is a simple measure of the discrimination of two sets of real numbers. Given training vectors <em>x<sub>k</sub></em>, if the number of positive and negative instances are <em>n</em><sub>+</sub> and <em>n</em><sub>−</sub>, respectively, then the F-score of the <em>i</em>th feature is defined as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000946.e001&amp;representation=PNG" class="inline-graphic"></span><br>where <em>x̅</em><em><sub>i</sub></em>, <em>x̅</em><em><sub>i</sub></em><sup>(+)</sup>, <em>x̅</em><em><sub>i</sub></em><sup>(−)</sup>are averages of the <em>i</em>th feature over the whole, positive, and negative data sets, respectively; <em>x<sub>k,i</sub></em><sup>(+)</sup> is the <em>i</em>th feature of the <em>k</em>th positive instance, and <em>x<sub>k,i</sub></em><sup>(−)</sup> is the <em>i</em>th feature of the <em>k</em>th negative instance. Larger F-scores indicate better discrimination <a href="#pone.0000946-Chen1">[42]</a>. We considered many features for each SVM. To determine which features to use in the final models, we ranked the features by F-score. Given <em>m</em> ranked features, we trained <em>m</em>-1 SVMs using features 1 and 2 for the first SVM, features 1, 2 and 3 for the second SVM etc. For feature selection, we judged SVM performance by training on the entire training sets, and studying the sensitivity and number of predicted miRNAs when the SVMs were applied to the entire set of candidates (see <a href="#pone.0000946.s002">table S1</a>). <a href="#pone-0000946-t001">Table 1</a> lists all features used in the final SVMs.</p>


<h4>Analysis of pre-miRNA candidates</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>We classified miRNA gene candidates as intronic or intergenic according to their location relative to UCSC Known Genes <a href="#pone.0000946-Hsu1">[69]</a>. We identified spatial miRNA gene clusters by clustering candidates predicted to be transcribed from the same strand. Intergenic candidates were clustered if they were separated by up to 6500 bp (the largest observed distance between two known clustered miRNAs), and intronic candidates if they were in the same intron. For transcription factor binding site searches, we used the TFBS Perl library <a href="#pone.0000946-Lenhard1">[70]</a>. To find support for miRNA expression, we downloaded all ESTs and and cDNAs mapped to mouse genome assembly NCBI 36 from the UCSC Genome Browser Database <a href="#pone.0000946-Karolchik1">[61]</a> in September 2006 and processed the mappings as described in <a href="#pone.0000946-Engstrom1">[71]</a> to remove low-quality sequences, reliably assign sequences to a genomic strand, and merge sequences obtained from the same cDNA clone. The selection of random regions for comparison with expression (cDNA, EST and CAGE) data was done so that these regions had similar chromosome and size distributions as the mouse miRNA predictions. Because comparisons of miRNA predictions and control sets to expression data were strand-specific, for the purpose of these comparisons, we randomly assigned strands to the randomly selected regions and the regions sampled from the CRS.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000946"></div><a name="pone.0000946.s001" id="pone.0000946.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000946.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Pattern composition differences among known pre-miRNAs, candidate pre-miRNAs and HCNEs. Each sequence was searched for putative transcription factor binding sites using the familial binding profiles for HMG, ETS, Forkhead, MADS, REL, TRP cluster (MYB), bHLH(zip) and bZIP cEBP-like subclass transcription factors and binding profiles for pax6, nkx2.2, nkx6.1, gsh2 and oct from the JASPAR database [78] at a score threshold of 80%. All definitions and analysis are the same as what is described in the legend for <a href="#pone-0000946-g004">Figure 4</a>. In most cases, the distributions for candidate pre-miRNAs (blue, green) are more similar to the distribution for known pre-miRNAs (red) than to the distribution for HCNEs not predicted to be pre-miRNAs (gray).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000946.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.27 MB PDF)</p>
<a name="pone.0000946.s002" id="pone.0000946.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000946.s002">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Process of feature selection for each SVM. Two tables are given for each SVM. The first table shows all tested features, ordered by F-score. The second table shows, for different feature sets, the number of input regions classified as positive by the model, and the number of known miRNAs (miRBase 7.1) classified as positive. Feature sets chosen for the final models are indicated in bold typeface and shaded in gray.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000946.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0000946.s003" id="pone.0000946.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000946.s003">Dataset S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Mouse candidate pre-miRNAs. File format: BED (see <a href="http://genome.ucsc.edu/FAQ/FAQformat">http://genome.ucsc.edu/FAQ/FAQformat</a>) Description: Genome coordinates of all mouse candidate pre-miRNAs(Candidates 801 was deleted because of lifting to NCBI mouse genome assembly Build 36). The coordinates refer to the NCBI mouse genome assembly Build 36 (mm8).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000946.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.14 MB TXT)</p>
<a name="pone.0000946.s004" id="pone.0000946.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000946.s004">Dataset S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Human candidate pre-miRNAs. File format: BED. Description: Genome coordinates of all human candidate pre-miRNAs. The coordinates refer to the NCBI human genome assembly Build 36.1 (hg18).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000946.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.14 MB TXT)</p>
<a name="pone.0000946.s005" id="pone.0000946.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000946.s005">Dataset S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Mouse candidate pre-miRNAs with CAGE data support. File format: BED. Description: Genome coordinates of all mouse candidate pre-miRNAs with CAGE support. The coordinates refer to the NCBI mouse genome assembly Build 36 (mm8).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000946.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.01 MB TXT)</p>
<a name="pone.0000946.s006" id="pone.0000946.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000946.s006">Dataset S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Human candidate pre-miRNAs with CAGE data support. File format: BED. Description: Genome coordinates of all human candidate pre-miRNAs with CAGE support. The coordinates refer to the NCBI human genome assembly Build 36.1 (hg18).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000946.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank David Fredman for comments on the manuscript, and Altuna Akalin and Albin Sandelin for useful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BL. Performed the experiments: PE YS. Analyzed the data: PE YS. Wrote the paper: PE BL YS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000946-Bartel1" id="pone.0000946-Bartel1"></a>Bartel DP (2004) MicroRNAs: genomics, biogenesis, mechanism, and function. Cell  116: 281–297.  <ul class="find" data-citedArticleID="996824" data-doi="10.1016/s0092-8674(04)00045-5"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00045-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000946-GriffithsJones1" id="pone.0000946-GriffithsJones1"></a>Griffiths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ (2006) miRBase: microRNA sequences, targets and gene nomenclature. Nucleic Acids Res  34: D140–144.  <ul class="find" data-citedArticleID="996862" data-doi="10.1093/nar/gkj112"><li><a href="http://dx.doi.org/10.1093/nar/gkj112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRBase%3A+microRNA+sequences%2C+targets+and+gene+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRBase%3A+microRNA+sequences%2C+targets+and+gene+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000946-GriffithsJones2" id="pone.0000946-GriffithsJones2"></a>Griffiths-Jones S (2004) The microRNA Registry. Nucleic Acids Res  32: D109–111.  <ul class="find" data-citedArticleID="996864" data-doi="10.1093/nar/gkh023"><li><a href="http://dx.doi.org/10.1093/nar/gkh023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microRNA+Registry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microRNA+Registry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000946-Lee1" id="pone.0000946-Lee1"></a>Lee CT, Risom T, Strauss WM (2006) MicroRNAs in mammalian development. Birth Defects Res C Embryo Today  78: 129–139.  <ul class="find" data-citedArticleID="996898" data-doi="10.1002/bdrc.20072"><li><a href="http://dx.doi.org/10.1002/bdrc.20072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+in+mammalian+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+in+mammalian+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000946-John1" id="pone.0000946-John1"></a>John B, Enright AJ, Aravin A, Tuschl T, Sander C, et al.  (2004) Human MicroRNA targets. PLoS Biol  2: e363.  <ul class="find" data-citedArticleID="996878" data-doi="10.1371/journal.pbio.0020363"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020363" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+MicroRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+MicroRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000946-Lewis1" id="pone.0000946-Lewis1"></a>Lewis BP, Burge CB, Bartel DP (2005) Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell  120: 15–20.  <ul class="find" data-citedArticleID="996904" data-doi="10.1016/j.cell.2004.12.035"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.12.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+microRNA+targets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+microRNA+targets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000946-Bentwich1" id="pone.0000946-Bentwich1"></a>Bentwich I, Avniel A, Karov Y, Aharonov R, Gilad S, et al.  (2005) Identification of hundreds of conserved and nonconserved human microRNAs. Nat Genet  37: 766–770.  <ul class="find" data-citedArticleID="996830" data-doi="10.1038/ng1590"><li><a href="http://dx.doi.org/10.1038/ng1590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+hundreds+of+conserved+and+nonconserved+human+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+hundreds+of+conserved+and+nonconserved+human+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000946-Berezikov1" id="pone.0000946-Berezikov1"></a>Berezikov E, Guryev V, van de Belt J, Wienholds E, Plasterk RH, et al.  (2005) Phylogenetic shadowing and computational identification of human microRNA genes. Cell  120: 21–24.  <ul class="find" data-citedArticleID="996832" data-doi="10.1016/j.cell.2004.12.031"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.12.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+shadowing+and+computational+identification+of+human+microRNA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+shadowing+and+computational+identification+of+human+microRNA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000946-LagosQuintana1" id="pone.0000946-LagosQuintana1"></a>Lagos-Quintana M, Rauhut R, Yalcin A, Meyer J, Lendeckel W, et al.  (2002) Identification of tissue-specific microRNAs from mouse. Curr Biol  12: 735–739.  <ul class="find" data-citedArticleID="996890" data-doi="10.1016/s0960-9822(02)00809-6"><li><a href="http://dx.doi.org/10.1016/s0960-9822(02)00809-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+tissue-specific+microRNAs+from+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+tissue-specific+microRNAs+from+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000946-Miska1" id="pone.0000946-Miska1"></a>Miska EA, Alvarez-Saavedra E, Townsend M, Yoshii A, Sestan N, et al.  (2004) Microarray analysis of microRNA expression in the developing mammalian brain. Genome Biol  5: R68.  <ul class="find" data-citedArticleID="996920" data-doi="10.1186/gb-2004-5-9-r68"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-9-r68" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarray+analysis+of+microRNA+expression+in+the+developing+mammalian+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarray+analysis+of+microRNA+expression+in+the+developing+mammalian+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000946-Poy1" id="pone.0000946-Poy1"></a>Poy MN, Eliasson L, Krutzfeldt J, Kuwajima S, Ma X, et al.  (2004) A pancreatic islet-specific microRNA regulates insulin secretion. Nature  432: 226–230.  <ul class="find" data-citedArticleID="996928" data-doi="10.1038/nature03076"><li><a href="http://dx.doi.org/10.1038/nature03076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+pancreatic+islet-specific+microRNA+regulates+insulin+secretion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+pancreatic+islet-specific+microRNA+regulates+insulin+secretion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000946-Watanabe1" id="pone.0000946-Watanabe1"></a>Watanabe T, Takeda A, Mise K, Okuno T, Suzuki T, et al.  (2005) Stage-specific expression of microRNAs during Xenopus development. FEBS Lett  579: 318–324.  <ul class="find" data-citedArticleID="996948" data-doi="10.1016/j.febslet.2004.11.067"><li><a href="http://dx.doi.org/10.1016/j.febslet.2004.11.067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stage-specific+expression+of+microRNAs+during+Xenopus+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stage-specific+expression+of+microRNAs+during+Xenopus+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000946-Duan1" id="pone.0000946-Duan1"></a>Duan R, Jin P (2006) Identification of messenger RNAs and microRNAs associated with fragile X mental retardation protein. Methods Mol Biol  342: 267–276.  <ul class="find" data-citedArticleID="996856" data-doi="10.1385/1-59745-123-1:267"><li><a href="http://dx.doi.org/10.1385/1-59745-123-1:267" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+messenger+RNAs+and+microRNAs+associated+with+fragile+X+mental+retardation+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+messenger+RNAs+and+microRNAs+associated+with+fragile+X+mental+retardation+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000946-Landthaler1" id="pone.0000946-Landthaler1"></a>Landthaler M, Yalcin A, Tuschl T (2004) The human DiGeorge syndrome critical region gene 8 and Its D. melanogaster homolog are required for miRNA biogenesis. Curr Biol  14: 2162–2167.  <ul class="find" data-citedArticleID="996896" data-doi="10.1016/j.cub.2004.11.001"><li><a href="http://dx.doi.org/10.1016/j.cub.2004.11.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+DiGeorge+syndrome+critical+region+gene+8+and+Its+D.+melanogaster+homolog+are+required+for+miRNA+biogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+DiGeorge+syndrome+critical+region+gene+8+and+Its+D.+melanogaster+homolog+are+required+for+miRNA+biogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000946-ORourke1" id="pone.0000946-ORourke1"></a>O'Rourke JR, Swanson MS, Harfe BD (2006) MicroRNAs in mammalian development and tumorigenesis. Birth Defects Res C Embryo Today  78: 172–179.  <ul class="find" data-citedArticleID="996924" data-doi="10.1002/bdrc.20071"><li><a href="http://dx.doi.org/10.1002/bdrc.20071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNAs+in+mammalian+development+and+tumorigenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNAs+in+mammalian+development+and+tumorigenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000946-Lu1" id="pone.0000946-Lu1"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al.  (2005) MicroRNA expression profiles classify human cancers. Nature  435: 834–838.  <ul class="find" data-citedArticleID="996910" data-doi="10.1038/nature03702"><li><a href="http://dx.doi.org/10.1038/nature03702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+expression+profiles+classify+human+cancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+expression+profiles+classify+human+cancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000946-Watanabe2" id="pone.0000946-Watanabe2"></a>Watanabe T, Takeda A, Tsukiyama T, Mise K, Okuno T, et al.  (2006) Identification and characterization of two novel classes of small RNAs in the mouse germline: retrotransposon-derived siRNAs in oocytes and germline small RNAs in testes. Genes Dev  20: 1732–1743.  <ul class="find" data-citedArticleID="996950" data-doi="10.1101/gad.1425706"><li><a href="http://dx.doi.org/10.1101/gad.1425706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+two+novel+classes+of+small+RNAs+in+the+mouse+germline%3A+retrotransposon-derived+siRNAs+in+oocytes+and+germline+small+RNAs+in+testes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+two+novel+classes+of+small+RNAs+in+the+mouse+germline%3A+retrotransposon-derived+siRNAs+in+oocytes+and+germline+small+RNAs+in+testes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000946-Houbaviy1" id="pone.0000946-Houbaviy1"></a>Houbaviy HB, Murray MF, Sharp PA (2003) Embryonic stem cell-specific MicroRNAs. Dev Cell  5: 351–358.  <ul class="find" data-citedArticleID="996874" data-doi="10.1016/s1534-5807(03)00227-2"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00227-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Embryonic+stem+cell-specific+MicroRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Embryonic+stem+cell-specific+MicroRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000946-Dostie1" id="pone.0000946-Dostie1"></a>Dostie J, Mourelatos Z, Yang M, Sharma A, Dreyfuss G (2003) Numerous microRNPs in neuronal cells containing novel microRNAs. Rna  9: 180–186.  <ul class="find" data-citedArticleID="996854" data-doi="10.1261/rna.2141503"><li><a href="http://dx.doi.org/10.1261/rna.2141503" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Numerous+microRNPs+in+neuronal+cells+containing+novel+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Numerous+microRNPs+in+neuronal+cells+containing+novel+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000946-Davis1" id="pone.0000946-Davis1"></a>Davis E, Caiment F, Tordoir X, Cavaille J, Ferguson-Smith A, et al.  (2005) RNAi-mediated allelic trans-interaction at the imprinted Rtl1/Peg11 locus. Curr Biol  15: 743–749.  <ul class="find" data-citedArticleID="996852" data-doi="10.1016/j.cub.2005.02.060"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.02.060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNAi-mediated+allelic+trans-interaction+at+the+imprinted+Rtl1%2FPeg11+locus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNAi-mediated+allelic+trans-interaction+at+the+imprinted+Rtl1%2FPeg11+locus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000946-Mineno1" id="pone.0000946-Mineno1"></a>Mineno J, Okamoto S, Ando T, Sato M, Chono H, et al.  (2006) The expression profile of microRNAs in mouse embryos. Nucleic Acids Res  34: 1765–1771.  <ul class="find" data-citedArticleID="996918" data-doi="10.1093/nar/gkl096"><li><a href="http://dx.doi.org/10.1093/nar/gkl096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+expression+profile+of+microRNAs+in+mouse+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+expression+profile+of+microRNAs+in+mouse+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000946-Seitz1" id="pone.0000946-Seitz1"></a>Seitz H, Royo H, Bortolin ML, Lin SP, Ferguson-Smith AC, et al.  (2004) A large imprinted microRNA gene cluster at the mouse Dlk1-Gtl2 domain. Genome Res  14: 1741–1748.  <ul class="find" data-citedArticleID="996936" data-doi="10.1101/gr.2743304"><li><a href="http://dx.doi.org/10.1101/gr.2743304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+large+imprinted+microRNA+gene+cluster+at+the+mouse+Dlk1-Gtl2+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+large+imprinted+microRNA+gene+cluster+at+the+mouse+Dlk1-Gtl2+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000946-LagosQuintana2" id="pone.0000946-LagosQuintana2"></a>Lagos-Quintana M, Rauhut R, Meyer J, Borkhardt A, Tuschl T (2003) New microRNAs from mouse and human. Rna  9: 175–179.  <ul class="find" data-citedArticleID="996892" data-doi="10.1261/rna.2146903"><li><a href="http://dx.doi.org/10.1261/rna.2146903" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+microRNAs+from+mouse+and+human." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+microRNAs+from+mouse+and+human.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000946-Michael1" id="pone.0000946-Michael1"></a>Michael MZ, SM OC, van Holst Pellekaan NG, Young GP, James RJ (2003) Reduced accumulation of specific microRNAs in colorectal neoplasia. Mol Cancer Res  1: 882–891.  <ul class="find" data-citedArticleID="996916"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reduced+accumulation+of+specific+microRNAs+in+colorectal+neoplasia.&amp;auth=&amp;atitle=Reduced+accumulation+of+specific+microRNAs+in+colorectal+neoplasia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+accumulation+of+specific+microRNAs+in+colorectal+neoplasia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+accumulation+of+specific+microRNAs+in+colorectal+neoplasia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000946-Yekta1" id="pone.0000946-Yekta1"></a>Yekta S, Shih IH, Bartel DP (2004) MicroRNA-directed cleavage of HOXB8 mRNA. Science  304: 594–596.  <ul class="find" data-citedArticleID="996958" data-doi="10.1126/science.1097434"><li><a href="http://dx.doi.org/10.1126/science.1097434" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-directed+cleavage+of+HOXB8+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-directed+cleavage+of+HOXB8+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000946-Mourelatos1" id="pone.0000946-Mourelatos1"></a>Mourelatos Z, Dostie J, Paushkin S, Sharma A, Charroux B, et al.  (2002) miRNPs: a novel class of ribonucleoproteins containing numerous microRNAs. Genes Dev  16: 720–728.  <ul class="find" data-citedArticleID="996922" data-doi="10.1101/gad.974702"><li><a href="http://dx.doi.org/10.1101/gad.974702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=miRNPs%3A+a+novel+class+of+ribonucleoproteins+containing+numerous+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22miRNPs%3A+a+novel+class+of+ribonucleoproteins+containing+numerous+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000946-Kim1" id="pone.0000946-Kim1"></a>Kim J, Krichevsky A, Grad Y, Hayes GD, Kosik KS, et al.  (2004) Identification of many microRNAs that copurify with polyribosomes in mammalian neurons. Proc Natl Acad Sci U S A  101: 360–365.  <ul class="find" data-citedArticleID="996884" data-doi="10.1073/pnas.2333854100"><li><a href="http://dx.doi.org/10.1073/pnas.2333854100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+many+microRNAs+that+copurify+with+polyribosomes+in+mammalian+neurons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+many+microRNAs+that+copurify+with+polyribosomes+in+mammalian+neurons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000946-LagosQuintana3" id="pone.0000946-LagosQuintana3"></a>Lagos-Quintana M, Rauhut R, Lendeckel W, Tuschl T (2001) Identification of novel genes coding for small expressed RNAs. Science  294: 853–858.  <ul class="find" data-citedArticleID="996894" data-doi="10.1126/science.1064921"><li><a href="http://dx.doi.org/10.1126/science.1064921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+novel+genes+coding+for+small+expressed+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+novel+genes+coding+for+small+expressed+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000946-Wheeler1" id="pone.0000946-Wheeler1"></a>Wheeler G, Ntounia-Fousara S, Granda B, Rathjen T, Dalmay T (2006) Identification of new central nervous system specific mouse microRNAs. FEBS Lett  580: 2195–2200.  <ul class="find" data-citedArticleID="996952" data-doi="10.1016/j.febslet.2006.03.019"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.03.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+new+central+nervous+system+specific+mouse+microRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+new+central+nervous+system+specific+mouse+microRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000946-Yu1" id="pone.0000946-Yu1"></a>Yu Z, Raabe T, Hecht NB (2005) MicroRNA Mirn122a reduces expression of the posttranscriptionally regulated germ cell transition protein 2 (Tnp2) messenger RNA (mRNA) by mRNA cleavage. Biol Reprod  73: 427–433.  <ul class="find" data-citedArticleID="996962" data-doi="10.1095/biolreprod.105.040998"><li><a href="http://dx.doi.org/10.1095/biolreprod.105.040998" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+Mirn122a+reduces+expression+of+the+posttranscriptionally+regulated+germ+cell+transition+protein+2+%28Tnp2%29+messenger+RNA+%28mRNA%29+by+mRNA+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+Mirn122a+reduces+expression+of+the+posttranscriptionally+regulated+germ+cell+transition+protein+2+%28Tnp2%29+messenger+RNA+%28mRNA%29+by+mRNA+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000946-Takada1" id="pone.0000946-Takada1"></a>Takada S, Berezikov E, Yamashita Y, Lagos-Quintana M, Kloosterman WP, et al.  (2006) Mouse microRNA profiles determined with a new and sensitive cloning method. Nucleic Acids Res  34: e115.  <ul class="find" data-citedArticleID="996942" data-doi="10.1093/nar/gkl653"><li><a href="http://dx.doi.org/10.1093/nar/gkl653" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mouse+microRNA+profiles+determined+with+a+new+and+sensitive+cloning+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mouse+microRNA+profiles+determined+with+a+new+and+sensitive+cloning+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000946-Berezikov2" id="pone.0000946-Berezikov2"></a>Berezikov E, van Tetering G, Verheul M, van de Belt J, van Laake L, et al.  (2006) Many novel mammalian microRNA candidates identified by extensive cloning and RAKE analysis. Genome Res  16: 1289–1298.  <ul class="find" data-citedArticleID="996834" data-doi="10.1101/gr.5159906"><li><a href="http://dx.doi.org/10.1101/gr.5159906" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Many+novel+mammalian+microRNA+candidates+identified+by+extensive+cloning+and+RAKE+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Many+novel+mammalian+microRNA+candidates+identified+by+extensive+cloning+and+RAKE+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000946-Berezikov3" id="pone.0000946-Berezikov3"></a>Berezikov E, Cuppen E, Plasterk RH (2006) Approaches to microRNA discovery. Nat Genet 38 Suppl S2–7.  <ul class="find" data-citedArticleID="996836" data-doi="10.1038/ng1794"><li><a href="http://dx.doi.org/10.1038/ng1794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approaches+to+microRNA+discovery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approaches+to+microRNA+discovery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000946-Bhardwaj1" id="pone.0000946-Bhardwaj1"></a>Bhardwaj N, Langlois RE, Zhao G, Lu H (2005) Kernel-based machine learning protocol for predicting DNA-binding proteins. Nucleic Acids Res  33: 6486–6493.  <ul class="find" data-citedArticleID="996838" data-doi="10.1093/nar/gki949"><li><a href="http://dx.doi.org/10.1093/nar/gki949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kernel-based+machine+learning+protocol+for+predicting+DNA-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kernel-based+machine+learning+protocol+for+predicting+DNA-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000946-Barutcuoglu1" id="pone.0000946-Barutcuoglu1"></a>Barutcuoglu Z, Schapire RE, Troyanskaya OG (2006) Hierarchical multi-label prediction of gene function. Bioinformatics  22: 830–836.  <ul class="find" data-citedArticleID="996826" data-doi="10.1093/bioinformatics/btk048"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btk048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hierarchical+multi-label+prediction+of+gene+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hierarchical+multi-label+prediction+of+gene+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000946-Hoglund1" id="pone.0000946-Hoglund1"></a>Hoglund A, Donnes P, Blum T, Adolph HW, Kohlbacher O (2006) MultiLoc: prediction of protein subcellular localization using N-terminal targeting sequences, sequence motifs and amino acid composition. Bioinformatics  22: 1158–1165.  <ul class="find" data-citedArticleID="996872" data-doi="10.1093/bioinformatics/btl002"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MultiLoc%3A+prediction+of+protein+subcellular+localization+using+N-terminal+targeting+sequences%2C+sequence+motifs+and+amino+acid+composition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MultiLoc%3A+prediction+of+protein+subcellular+localization+using+N-terminal+targeting+sequences%2C+sequence+motifs+and+amino+acid+composition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000946-Sewer1" id="pone.0000946-Sewer1"></a>Sewer A, Paul N, Landgraf P, Aravin A, Pfeffer S, et al.  (2005) Identification of clustered microRNAs using an ab initio prediction method. BMC Bioinformatics  6: 267.  <ul class="find" data-citedArticleID="996938" data-doi="10.1186/1471-2105-6-267"><li><a href="http://dx.doi.org/10.1186/1471-2105-6-267" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+clustered+microRNAs+using+an+ab+initio+prediction+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+clustered+microRNAs+using+an+ab+initio+prediction+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000946-Xue1" id="pone.0000946-Xue1"></a>Xue C, Li F, He T, Liu GP, Li Y, et al.  (2005) Classification of real and pseudo microRNA precursors using local structure-sequence features and support vector machine. BMC Bioinformatics  6: 310.  <ul class="find" data-citedArticleID="996956"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Classification+of+real+and+pseudo+microRNA+precursors+using+local+structure-sequence+features+and+support+vector+machine.&amp;auth=&amp;atitle=Classification+of+real+and+pseudo+microRNA+precursors+using+local+structure-sequence+features+and+support+vector+machine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Classification+of+real+and+pseudo+microRNA+precursors+using+local+structure-sequence+features+and+support+vector+machine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Classification+of+real+and+pseudo+microRNA+precursors+using+local+structure-sequence+features+and+support+vector+machine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000946-Helvik1" id="pone.0000946-Helvik1"></a>Helvik SA, Snove O Jr, Saetrom P (2007) Reliable prediction of Drosha processing sites improves microRNA gene prediction. Bioinformatics  23: 142–149.  <ul class="find" data-citedArticleID="996868" data-doi="10.1093/bioinformatics/btl570"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl570" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reliable+prediction+of+Drosha+processing+sites+improves+microRNA+gene+prediction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reliable+prediction+of+Drosha+processing+sites+improves+microRNA+gene+prediction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000946-KwangLoong1" id="pone.0000946-KwangLoong1"></a>Kwang Loong SN, Mishra SK (2007) De Novo SVM Classification of Precursor MicroRNAs from Genomic Pseudo Hairpins Using Global and Intrinsic Folding Measures. Bioinformatics.   <ul class="find" data-citedArticleID="996888" data-doi="10.1093/bioinformatics/btm026"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btm026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=De+Novo+SVM+Classification+of+Precursor+MicroRNAs+from+Genomic+Pseudo+Hairpins+Using+Global+and+Intrinsic+Folding+Measures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22De+Novo+SVM+Classification+of+Precursor+MicroRNAs+from+Genomic+Pseudo+Hairpins+Using+Global+and+Intrinsic+Folding+Measures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000946-Hertel1" id="pone.0000946-Hertel1"></a>Hertel J, Stadler PF (2006) Hairpins in a Haystack: recognizing microRNA precursors in comparative genomics data. Bioinformatics  22: e197–202.  <ul class="find" data-citedArticleID="996870" data-doi="10.1093/bioinformatics/btl257"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl257" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hairpins+in+a+Haystack%3A+recognizing+microRNA+precursors+in+comparative+genomics+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hairpins+in+a+Haystack%3A+recognizing+microRNA+precursors+in+comparative+genomics+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000946-Chen1" id="pone.0000946-Chen1"></a>Chen Y-W, Lin C-J (2003) Combining SVMs with Various Feature Selection Strategies.   URL <a href="http://www.csie.ntu.edu.tw/%E2%88%BCcjlin/papers/features.pdf">http://www.csie.ntu.edu.tw/~cjlin/papers​/features.pdf</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0000946-Bonnet1" id="pone.0000946-Bonnet1"></a>Bonnet E, Wuyts J, Rouze P, Van de Peer Y (2004) Evidence that microRNA precursors, unlike other non-coding RNAs, have lower folding free energies than random sequences. Bioinformatics  20: 2911–2917.  <ul class="find" data-citedArticleID="996842" data-doi="10.1093/bioinformatics/bth374"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth374" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+that+microRNA+precursors%2C+unlike+other+non-coding+RNAs%2C+have+lower+folding+free+energies+than+random+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+that+microRNA+precursors%2C+unlike+other+non-coding+RNAs%2C+have+lower+folding+free+energies+than+random+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000946-Lim1" id="pone.0000946-Lim1"></a>Lim LP, Lau NC, Weinstein EG, Abdelhakim A, Yekta S, et al.  (2003) The microRNAs of Caenorhabditis elegans. Genes Dev  17: 991–1008.  <ul class="find" data-citedArticleID="996906" data-doi="10.1101/gad.1074403"><li><a href="http://dx.doi.org/10.1101/gad.1074403" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microRNAs+of+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microRNAs+of+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000946-Siepel1" id="pone.0000946-Siepel1"></a>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, et al.  (2005) Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res  15: 1034–1050.  <ul class="find" data-citedArticleID="996940" data-doi="10.1101/gr.3715005"><li><a href="http://dx.doi.org/10.1101/gr.3715005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+elements+in+vertebrate%2C+insect%2C+worm%2C+and+yeast+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000946-Lunter1" id="pone.0000946-Lunter1"></a>Lunter G, Ponting CP, Hein J (2006) Genome-wide identification of human functional DNA using a neutral indel model. PLoS Comput Biol  2: e5.  <ul class="find" data-citedArticleID="996912" data-doi="10.1371/journal.pcbi.0020005"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+identification+of+human+functional+DNA+using+a+neutral+indel+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+identification+of+human+functional+DNA+using+a+neutral+indel+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000946-Bejerano1" id="pone.0000946-Bejerano1"></a>Bejerano G, Pheasant M, Makunin I, Stephen S, Kent WJ, et al.  (2004) Ultraconserved elements in the human genome. Science  304: 1321–1325.  <ul class="find" data-citedArticleID="996828" data-doi="10.1126/science.1098119"><li><a href="http://dx.doi.org/10.1126/science.1098119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ultraconserved+elements+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ultraconserved+elements+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000946-Sandelin1" id="pone.0000946-Sandelin1"></a>Sandelin A, Bailey P, Bruce S, Engstrom PG, Klos JM, et al.  (2004) Arrays of ultraconserved non-coding regions span the loci of key developmental genes in vertebrate genomes. BMC Genomics  5: 99.  <ul class="find" data-citedArticleID="996932"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Arrays+of+ultraconserved+non-coding+regions+span+the+loci+of+key+developmental+genes+in+vertebrate+genomes.&amp;auth=&amp;atitle=Arrays+of+ultraconserved+non-coding+regions+span+the+loci+of+key+developmental+genes+in+vertebrate+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arrays+of+ultraconserved+non-coding+regions+span+the+loci+of+key+developmental+genes+in+vertebrate+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arrays+of+ultraconserved+non-coding+regions+span+the+loci+of+key+developmental+genes+in+vertebrate+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000946-Woolfe1" id="pone.0000946-Woolfe1"></a>Woolfe A, Goodson M, Goode DK, Snell P, McEwen GK, et al.  (2005) Highly conserved non-coding sequences are associated with vertebrate development. PLoS Biol  3: e7.  <ul class="find" data-citedArticleID="996954" data-doi="10.1371/journal.pbio.0030007"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Highly+conserved+non-coding+sequences+are+associated+with+vertebrate+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Highly+conserved+non-coding+sequences+are+associated+with+vertebrate+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000946-GomezSkarmeta1" id="pone.0000946-GomezSkarmeta1"></a>Gomez-Skarmeta JL, Lenhard B, Becker TS (2006) New technologies, new findings, and new concepts in the study of vertebrate cis-regulatory sequences. Dev Dyn  235: 870–885.  <ul class="find" data-citedArticleID="996860" data-doi="10.1002/dvdy.20659"><li><a href="http://dx.doi.org/10.1002/dvdy.20659" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+technologies%2C+new+findings%2C+and+new+concepts+in+the+study+of+vertebrate+cis-regulatory+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+technologies%2C+new+findings%2C+and+new+concepts+in+the+study+of+vertebrate+cis-regulatory+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000946-Bailey1" id="pone.0000946-Bailey1"></a>Bailey PJ, Klos JM, Andersson E, Karlen M, Kallstrom M, et al.  (2006) A global genomic transcriptional code associated with CNS-expressed genes. Exp Cell Res  312: 3108–3119.  <ul class="find" data-citedArticleID="996822" data-doi="10.1016/j.yexcr.2006.06.017"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2006.06.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+genomic+transcriptional+code+associated+with+CNS-expressed+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+genomic+transcriptional+code+associated+with+CNS-expressed+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000946-Lee2" id="pone.0000946-Lee2"></a>Lee Y, Kim M, Han J, Yeom KH, Lee S, et al.  (2004) MicroRNA genes are transcribed by RNA polymerase II. Embo J  23: 4051–4060.  <ul class="find" data-citedArticleID="996900" data-doi="10.1038/sj.emboj.7600385"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7600385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA+genes+are+transcribed+by+RNA+polymerase+II." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA+genes+are+transcribed+by+RNA+polymerase+II.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000946-Cai1" id="pone.0000946-Cai1"></a>Cai X, Hagedorn CH, Cullen BR (2004) Human microRNAs are processed from capped, polyadenylated transcripts that can also function as mRNAs. RNA  10: 1957–1966.  <ul class="find" data-citedArticleID="996846" data-doi="10.1261/rna.7135204"><li><a href="http://dx.doi.org/10.1261/rna.7135204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+microRNAs+are+processed+from+capped%2C+polyadenylated+transcripts+that+can+also+function+as+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+microRNAs+are+processed+from+capped%2C+polyadenylated+transcripts+that+can+also+function+as+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000946-Gu1" id="pone.0000946-Gu1"></a>Gu J, He T, Pei Y, Li F, Wang X, et al.  (2006) Primary transcripts and expressions of mammal intergenic microRNAs detected by mapping ESTs to their flanking sequences. Mamm Genome  17: 1033–1041.  <ul class="find" data-citedArticleID="996866" data-doi="10.1007/s00335-006-0007-9"><li><a href="http://dx.doi.org/10.1007/s00335-006-0007-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Primary+transcripts+and+expressions+of+mammal+intergenic+microRNAs+detected+by+mapping+ESTs+to+their+flanking+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Primary+transcripts+and+expressions+of+mammal+intergenic+microRNAs+detected+by+mapping+ESTs+to+their+flanking+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000946-Kodzius1" id="pone.0000946-Kodzius1"></a>Kodzius R, Kojima M, Nishiyori H, Nakamura M, Fukuda S, et al.  (2006) CAGE: cap analysis of gene expression. Nat Methods  3: 211–222.  <ul class="find" data-citedArticleID="996886" data-doi="10.1038/nmeth0306-211"><li><a href="http://dx.doi.org/10.1038/nmeth0306-211" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CAGE%3A+cap+analysis+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CAGE%3A+cap+analysis+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000946-Carninci1" id="pone.0000946-Carninci1"></a>Carninci P, Kasukawa T, Katayama S, Gough J, Frith MC, et al.  (2005) The transcriptional landscape of the mammalian genome. Science  309: 1559–1563.  <ul class="find" data-citedArticleID="996848" data-doi="10.1126/science.1112014"><li><a href="http://dx.doi.org/10.1126/science.1112014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptional+landscape+of+the+mammalian+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptional+landscape+of+the+mammalian+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000946-Yousef1" id="pone.0000946-Yousef1"></a>Yousef M, Nebozhyn M, Shatkay H, Kanterakis S, Showe LC, et al.  (2006) Combining multi-species genomic data for microRNA identification using a Naive Bayes classifier. Bioinformatics  22: 1325–1334.  <ul class="find" data-citedArticleID="996960" data-doi="10.1093/bioinformatics/btl094"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combining+multi-species+genomic+data+for+microRNA+identification+using+a+Naive+Bayes+classifier." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combining+multi-species+genomic+data+for+microRNA+identification+using+a+Naive+Bayes+classifier.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000946-Pedersen1" id="pone.0000946-Pedersen1"></a>Pedersen JS, Bejerano G, Siepel A, Rosenbloom K, Lindblad-Toh K, et al.  (2006) Identification and classification of conserved RNA secondary structures in the human genome. PLoS Comput Biol  2: e33.  <ul class="find" data-citedArticleID="996926" data-doi="10.1371/journal.pcbi.0020033.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020033.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+classification+of+conserved+RNA+secondary+structures+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000946-Washietl1" id="pone.0000946-Washietl1"></a>Washietl S, Hofacker IL, Lukasser M, Huttenhofer A, Stadler PF (2005) Mapping of conserved RNA secondary structures predicts thousands of functional noncoding RNAs in the human genome. Nat Biotechnol  23: 1383–1390.  <ul class="find" data-citedArticleID="996946" data-doi="10.1038/nbt1144"><li><a href="http://dx.doi.org/10.1038/nbt1144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+of+conserved+RNA+secondary+structures+predicts+thousands+of+functional+noncoding+RNAs+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000946-Ruby1" id="pone.0000946-Ruby1"></a>Ruby JG, Jan C, Player C, Axtell MJ, Lee W, et al.  (2006) Large-scale sequencing reveals 21U-RNAs and additional microRNAs and endogenous siRNAs in C. elegans. Cell  127: 1193–1207.  <ul class="find" data-citedArticleID="996930" data-doi="10.1016/j.cell.2006.10.040"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.10.040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+sequencing+reveals+21U-RNAs+and+additional+microRNAs+and+endogenous+siRNAs+in+C.+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+sequencing+reveals+21U-RNAs+and+additional+microRNAs+and+endogenous+siRNAs+in+C.+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000946-Karolchik1" id="pone.0000946-Karolchik1"></a>Karolchik D, Baertsch R, Diekhans M, Furey TS, Hinrichs A, et al.  (2003) The UCSC Genome Browser Database. Nucleic Acids Res  31: 51–54.  <ul class="find" data-citedArticleID="996880" data-doi="10.1093/nar/gkg129"><li><a href="http://dx.doi.org/10.1093/nar/gkg129" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Genome+Browser+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Genome+Browser+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000946-Kent1" id="pone.0000946-Kent1"></a>Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D (2003) Evolution's cauldron: duplication, deletion, and rearrangement in the mouse and human genomes. Proc Natl Acad Sci U S A  100: 11484–11489.  <ul class="find" data-citedArticleID="996882" data-doi="10.1073/pnas.1932072100"><li><a href="http://dx.doi.org/10.1073/pnas.1932072100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution%27s+cauldron%3A+duplication%2C+deletion%2C+and+rearrangement+in+the+mouse+and+human+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution%27s+cauldron%3A+duplication%2C+deletion%2C+and+rearrangement+in+the+mouse+and+human+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000946-Mathews1" id="pone.0000946-Mathews1"></a>Mathews DH, Sabina J, Zuker M, Turner DH (1999) Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure. J Mol Biol  288: 911–940.  <ul class="find" data-citedArticleID="996914" data-doi="10.1006/jmbi.1999.2700"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.2700" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expanded+sequence+dependence+of+thermodynamic+parameters+improves+prediction+of+RNA+secondary+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expanded+sequence+dependence+of+thermodynamic+parameters+improves+prediction+of+RNA+secondary+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000946-Blanchette1" id="pone.0000946-Blanchette1"></a>Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, et al.  (2004) Aligning multiple genomic sequences with the threaded blockset aligner. Genome Res  14: 708–715.  <ul class="find" data-citedArticleID="996840" data-doi="10.1101/gr.1933104"><li><a href="http://dx.doi.org/10.1101/gr.1933104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aligning+multiple+genomic+sequences+with+the+threaded+blockset+aligner." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aligning+multiple+genomic+sequences+with+the+threaded+blockset+aligner.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000946-Schuster1" id="pone.0000946-Schuster1"></a>Schuster P, Fontana W, Stadler PF, Hofacker IL (1994) From sequences to shapes and back: a case study in RNA secondary structures. Proc Biol Sci  255: 279–284.  <ul class="find" data-citedArticleID="996934" data-doi="10.1098/rspb.1994.0040"><li><a href="http://dx.doi.org/10.1098/rspb.1994.0040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+sequences+to+shapes+and+back%3A+a+case+study+in+RNA+secondary+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+sequences+to+shapes+and+back%3A+a+case+study+in+RNA+secondary+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000946-Brudno1" id="pone.0000946-Brudno1"></a>Brudno M, Do CB, Cooper GM, Kim MF, Davydov E, et al.  (2003) LAGAN and Multi-LAGAN: efficient tools for large-scale multiple alignment of genomic DNA. Genome Res  13: 721–731.  <ul class="find" data-citedArticleID="996844" data-doi="10.1101/gr.926603"><li><a href="http://dx.doi.org/10.1101/gr.926603" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LAGAN+and+Multi-LAGAN%3A+efficient+tools+for+large-scale+multiple+alignment+of+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LAGAN+and+Multi-LAGAN%3A+efficient+tools+for+large-scale+multiple+alignment+of+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000946-Lowe1" id="pone.0000946-Lowe1"></a>Lowe TM, Eddy SR (1997) tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res  25: 955–964.  <ul class="find" data-citedArticleID="996908" data-doi="10.1093/nar/25.5.955"><li><a href="http://dx.doi.org/10.1093/nar/25.5.955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNAscan-SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000946-Ashurst1" id="pone.0000946-Ashurst1"></a>Ashurst JL, Chen CK, Gilbert JG, Jekosch K, Keenan S, et al.  (2005) The Vertebrate Genome Annotation (Vega) database. Nucleic Acids Res  33: D459–465.  <ul class="find" data-citedArticleID="996820" data-doi="10.1093/nar/gki135"><li><a href="http://dx.doi.org/10.1093/nar/gki135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Vertebrate+Genome+Annotation+%28Vega%29+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Vertebrate+Genome+Annotation+%28Vega%29+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000946-Hsu1" id="pone.0000946-Hsu1"></a>Hsu F, Kent WJ, Clawson H, Kuhn RM, Diekhans M, et al.  (2006) The UCSC Known Genes. Bioinformatics  22: 1036–1046.  <ul class="find" data-citedArticleID="996876" data-doi="10.1093/bioinformatics/btl048"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+UCSC+Known+Genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+UCSC+Known+Genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000946-Lenhard1" id="pone.0000946-Lenhard1"></a>Lenhard B, Wasserman WW (2002) TFBS: Computational framework for transcription factor binding site analysis. Bioinformatics  18: 1135–1136.  <ul class="find" data-citedArticleID="996902" data-doi="10.1093/bioinformatics/18.8.1135"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.8.1135" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TFBS%3A+Computational+framework+for+transcription+factor+binding+site+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TFBS%3A+Computational+framework+for+transcription+factor+binding+site+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0000946-Engstrom1" id="pone.0000946-Engstrom1"></a>Engstrom PG, Suzuki H, Ninomiya N, Akalin A, Sessa L, et al.  (2006) Complex Loci in human and mouse genomes. PLoS Genet  2: e47.  <ul class="find" data-citedArticleID="996858" data-doi="10.1371/journal.pgen.0020047"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+Loci+in+human+and+mouse+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+Loci+in+human+and+mouse+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0000946-Vlieghe1" id="pone.0000946-Vlieghe1"></a>Vlieghe D, Sandelin A, De Bleser PJ, Vleminckx K, Wasserman WW, et al.  (2006) A new generation of JASPAR, the open-access repository for transcription factor binding site profiles. Nucleic Acids Res  34: D95–97.  <ul class="find" data-citedArticleID="996944" data-doi="10.1093/nar/gkj115"><li><a href="http://dx.doi.org/10.1093/nar/gkj115" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+generation+of+JASPAR%2C+the+open-access+repository+for+transcription+factor+binding+site+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+generation+of+JASPAR%2C+the+open-access+repository+for+transcription+factor+binding+site+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.XML" value="149971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.PDF" value="434465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g001.PNG_L" value="53451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g001.PNG_M" value="43712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g001.PNG_S" value="15556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g001.TIF" value="298090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g001.PNG_I" value="101902"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t001.PNG_L" value="160130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t001.PNG_M" value="96034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t001.PNG_S" value="10887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t001.TIF" value="722150"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t001.PNG_I" value="28755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g002.PNG_L" value="204896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g002.PNG_M" value="119451"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g002.PNG_S" value="12651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g002.TIF" value="801246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g002.PNG_I" value="47359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t002.PNG_L" value="69711"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t002.PNG_M" value="38201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t002.PNG_S" value="8429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t002.TIF" value="335278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t002.PNG_I" value="11532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g003.PNG_L" value="57395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g003.PNG_M" value="55991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g003.PNG_S" value="12660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g003.TIF" value="447456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g003.PNG_I" value="40336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g004.PNG_L" value="62246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g004.PNG_M" value="69744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g004.PNG_S" value="11043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g004.TIF" value="334126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g004.PNG_I" value="24720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t003.PNG_L" value="51782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t003.PNG_M" value="69760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t003.PNG_S" value="11659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t003.TIF" value="238674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t003.PNG_I" value="22558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g005.PNG_L" value="209613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g005.PNG_M" value="84788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g005.PNG_S" value="14871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g005.TIF" value="747130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g005.PNG_I" value="69934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g006.PNG_L" value="53871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g006.PNG_M" value="73303"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g006.PNG_S" value="11078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g006.TIF" value="333106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g006.PNG_I" value="26754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t004.PNG_L" value="202936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t004.PNG_M" value="115655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t004.PNG_S" value="11185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t004.TIF" value="785398"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t004.PNG_I" value="35495"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t005.PNG_L" value="116248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t005.PNG_M" value="71967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t005.PNG_S" value="9607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t005.TIF" value="515460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.t005.PNG_I" value="21663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g007.PNG_L" value="22889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g007.PNG_M" value="13671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g007.PNG_S" value="7109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g007.TIF" value="65946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g007.PNG_I" value="3887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g008.PNG_L" value="54011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g008.PNG_M" value="55108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g008.PNG_S" value="9133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g008.TIF" value="311356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.g008.PNG_I" value="19145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.e001.PNG" value="14825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.e001.TIF" value="30512"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.s001.PDF" value="1267613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.s002.DOC" value="49664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.s003.TXT" value="141159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.s004.TXT" value="139960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.s005.TXT" value="5569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000946.s006.TXT" value="3770"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000946&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000946" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000946&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000946&volume=&issue=&title=Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure&author_name=Ying%20Sheng%2C%20P%C3%A4r%20G.%20Engstr%C3%B6m%2C%20Boris%20Lenhard&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000946" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946&amp;t=Mammalian%20MicroRNA%20Prediction%20through%20a%20Support%20Vector%20Machine%20Model%20of%20Sequence%20and%20Structure" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946&title=Mammalian%20MicroRNA%20Prediction%20through%20a%20Support%20Vector%20Machine%20Model%20of%20Sequence%20and%20Structure&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946&amp;title=Mammalian%20MicroRNA%20Prediction%20through%20a%20Support%20Vector%20Machine%20Model%20of%20Sequence%20and%20Structure" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000946&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Mammalian MicroRNA Prediction through a Support Vector Machine Model of Sequence and Structure';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000946';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Mammalian%20MicroRNA%20Prediction%20through%20a%20Support%20Vector%20Machine%20Model%20of%20Sequence%20and%20Structure http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000946" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000946" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Comparative+genomics%22" title="Search for articles in the subject area:'Comparative genomics'"><div class="flagText">Comparative genomics</div></a>
              <div data-categoryid="32937" data-articleid="24730"
                   data-categoryname="Comparative genomics"
                   class="flagImage" title="Flag 'Comparative genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24730"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24730"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="24730"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="24730"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22MicroRNA%22" title="Search for articles in the subject area:'MicroRNA'"><div class="flagText">MicroRNA</div></a>
              <div data-categoryid="19821" data-articleid="24730"
                   data-categoryname="MicroRNA"
                   class="flagImage" title="Flag 'MicroRNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+stem-loop+structure%22" title="Search for articles in the subject area:'RNA stem-loop structure'"><div class="flagText">RNA stem-loop structure</div></a>
              <div data-categoryid="46413" data-articleid="24730"
                   data-categoryname="RNA stem-loop structure"
                   class="flagImage" title="Flag 'RNA stem-loop structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24730"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8370'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7452'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1203&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=21577">How about RNA22 ?</a><br>Posted by mathijsv</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
