

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000877"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000877" />

    <meta name="citation_title" content="An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length"/>
    <meta itemprop="name" content="An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length"/>

      <meta name="citation_author" content="Hua Wong"/>
            <meta name="citation_author_institution" content="Laboratoire de Physique Théorique de la Matière Condensée, Université Pierre et Marie Curie, Paris, France"/>
      <meta name="citation_author" content="Jean-Marc Victor"/>
            <meta name="citation_author_institution" content="Laboratoire de Physique Théorique de la Matière Condensée, Université Pierre et Marie Curie, Paris, France"/>
      <meta name="citation_author" content="Julien Mozziconacci"/>
            <meta name="citation_author_institution" content="Laboratoire de Physique Théorique de la Matière Condensée, Université Pierre et Marie Curie, Paris, France"/>
            <meta name="citation_author_institution" content="Cell Biology and Biophysics, European Molecular Biology Laboratory, Heidelberg, Germany"/>

    <meta name="citation_date" content="2007/9/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000877.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e877"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Van Holde KE Chromatin. Springer series in molecular biology; citation_number=1; " />
      <meta name="citation_reference" content="citation_title=Higher-order structure of chromatin and chromosomes.; citation_author=CL Woodcock; citation_author=S Dimitrov; citation_journal_title=Curr Opin Genet Dev; citation_volume=11; citation_number=2; citation_pages=130-135; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The location of the linker histone on the nucleosome.; citation_author=A Travers; citation_journal_title=Trends Biochem Sci; citation_volume=24; citation_number=3; citation_pages=4-7; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Chromatin conformation and salt-induced compaction: three-dimensional structural information from cryoelectron microscopy.; citation_author=J Bednar; citation_author=RA Horowitz; citation_author=J Dubochet; citation_author=CL Woodcock; citation_journal_title=J Cell Biol; citation_volume=131; citation_number=4; citation_pages=1365-1376; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Nucleosomes, linker DNA, and linker histone form a unique structural motif that directs the higher-order folding and compaction of chromatin.; citation_author=J Bednar; citation_author=RA Horowitz; citation_author=SA Grigoryev; citation_author=LM Carruthers; citation_author=JC Hansen; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=5; citation_pages=14173-14178; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Physical constraints in the condensation of eukaryotic chromosomes. Local concentration of DNA versus linear packing ratio in higher order chromatin structures.; citation_author=JR Daban; citation_journal_title=Biochemistry; citation_volume=39; citation_number=6; citation_pages=3861-3866; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=EM measurements define the dimensions of the “30-nm” chromatin fiber: evidence for a compact, interdigitated structure.; citation_author=PJ Robinson; citation_author=L Fairall; citation_author=VA Huynh; citation_author=D Rhodes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=7; citation_pages=6506-6511; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nucleosome arrays reveal the two-start organization of the chromatin fiber.; citation_author=B Dorigo; citation_author=T Schalch; citation_author=A Kulangara; citation_author=S Duda; citation_author=RR Schroeder; citation_journal_title=Science; citation_volume=306; citation_number=8; citation_pages=1571-1573; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structure of the 300A chromatin filament: X-ray diffraction from oriented samples.; citation_author=J Widom; citation_author=A Klug; citation_journal_title=Cell; citation_volume=43; citation_number=9; citation_pages=207-213; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Structure of chromatin and the linking number of DNA.; citation_author=A Worcel; citation_author=S Strogatz; citation_author=D Riley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=10; citation_pages=1461-1465; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Chromatin fibers are left-handed double helices with diameter and mass per unit length that depend on linker length.; citation_author=SP Williams; citation_author=BD Athey; citation_author=LJ Muglia; citation_author=RS Schappe; citation_author=AH Gough; citation_journal_title=Biophys J; citation_volume=49; citation_number=11; citation_pages=233-248; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Interdigitated solenoid model for compact chromatin fibers.; citation_author=JR Daban; citation_author=A Bermudez; citation_journal_title=Biochemistry; citation_volume=37; citation_number=12; citation_pages=4299-4304; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=X-ray structure of a tetranucleosome and its implications for the chromatin fibre.; citation_author=T Schalch; citation_author=S Duda; citation_author=DF Sargent; citation_author=TJ Richmond; citation_journal_title=Nature; citation_volume=436; citation_number=13; citation_pages=138-141; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A triple helix model for the structure of chromatin fiber.; citation_author=V Makarov; citation_author=S Dimitrov; citation_author=V Smirnov; citation_author=I Pashev; citation_journal_title=FEBS Lett; citation_volume=181; citation_number=14; citation_pages=357-361; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Chromatin higher order structure: chasing a mirage?; citation_author=K van Holde; citation_author=J Zlatanova; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=15; citation_pages=8373-8376; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Chromatin: a tunable spring at work inside chromosomes.; citation_author=E Ben-Haim; citation_author=A Lesne; citation_author=JM Victor; citation_journal_title=Phys Rev E Stat Nonlin Soft Matter Phys; citation_volume=64; citation_number=16; citation_pages=051921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=How the chromatin fiber deals with topological constraints.; citation_author=M Barbi; citation_author=J Mozziconacci; citation_author=JM Victor; citation_journal_title=Phys Rev E Stat Nonlin Soft Matter Phys; citation_volume=71; citation_number=17; citation_pages=031910; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Compact oligomers and nucleosome phasing.; citation_author=K Tatchell; citation_author=KE Van Holde; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=75; citation_number=18; citation_pages=3583-3587; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Nucleosome arcs and helices.; citation_author=J Dubochet; citation_author=M Noll; citation_journal_title=Science; citation_volume=202; citation_number=19; citation_pages=280-286; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Phase diagram of nucleosome core particles.; citation_author=S Mangenot; citation_author=A Leforestier; citation_author=D Durand; citation_author=F Livolant; citation_journal_title=J Mol Biol; citation_volume=333; citation_number=20; citation_pages=907-916; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Internal structure of the 30 nm chromatin fiber.; citation_author=S Bartolome; citation_author=A Bermudez; citation_author=JR Daban; citation_journal_title=J Cell Sci; citation_volume=107 ( Pt 11); citation_number=21; citation_pages=2983-2992; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Solvent mediated interactions in the structure of the nucleosome core particle at 1.9 a resolution.; citation_author=CA Davey; citation_author=DF Sargent; citation_author=K Luger; citation_author=AW Maeder; citation_author=TJ Richmond; citation_journal_title=J Mol Biol; citation_volume=319; citation_number=22; citation_pages=1097-1113; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A relationship between the helical twist of DNA and the ordered positioning of nucleosomes in all eukaryotic cells.; citation_author=J Widom; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=23; citation_pages=1095-1099; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Twist constraints on linker DNA in the 30-nm chromatin fiber: implications for nucleosome phasing.; citation_author=J Yao; citation_author=PT Lowary; citation_author=J Widom; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=24; citation_pages=9364-9368; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Two DNA-binding sites on the globular domain of histone H5 are required for binding to both bulk and 5 S reconstituted nucleosomes.; citation_author=MM Duggan; citation_author=JO Thomas; citation_journal_title=J Mol Biol; citation_volume=304; citation_number=25; citation_pages=21-33; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Chromatin superstructure-dependent crosslinking with DNA of the histone H5 residues Thr1, His25 and His62.; citation_author=AD Mirzabekov; citation_author=DV Pruss; citation_author=KK Ebralidse; citation_journal_title=J Mol Biol; citation_volume=211; citation_number=26; citation_pages=479-491; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Mapping the interaction surface of linker histone H1(0) with the nucleosome of native chromatin in vivo.; citation_author=DT Brown; citation_author=T Izard; citation_author=T Misteli; citation_journal_title=Nat Struct Mol Biol; citation_volume=13; citation_number=27; citation_pages=250-255; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Complex of linker histone H5 with the nucleosome and its implications for chromatin packing.; citation_author=L Fan; citation_author=VA Roberts; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=28; citation_pages=8384-8389; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of globular domain of histone H5 and its implications for nucleosome binding.; citation_author=V Ramakrishnan; citation_author=JT Finch; citation_author=V Graziano; citation_author=PL Lee; citation_author=RM Sweet; citation_journal_title=Nature; citation_volume=362; citation_number=29; citation_pages=219-223; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The structure of histone H1 and its location in chromatin.; citation_author=J Allan; citation_author=PG Hartman; citation_author=C Crane-Robinson; citation_author=FX Aviles; citation_journal_title=Nature; citation_volume=288; citation_number=30; citation_pages=675-679; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Linker histone-dependent DNA structure in linear mononucleosomes.; citation_author=A Hamiche; citation_author=P Schultz; citation_author=V Ramakrishnan; citation_author=P Oudet; citation_author=A Prunell; citation_journal_title=J Mol Biol; citation_volume=257; citation_number=31; citation_pages=30-42; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Linker histone-dependent organization and dynamics of nucleosome entry/exit DNAs.; citation_author=A Sivolob; citation_author=A Prunell; citation_journal_title=J Mol Biol; citation_volume=331; citation_number=32; citation_pages=1025-1040; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Supramolecular ordering of DNA in the cholesteric liquid crystalline phase: an ultrastructural study.; citation_author=A Leforestier; citation_author=F Livolant; citation_journal_title=Biophys J; citation_volume=65; citation_number=33; citation_pages=56-72; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Cryoelectron microscopy of lambda phage DNA condensates in vitreous ice: the fine structure of DNA toroids.; citation_author=NV Hud; citation_author=KH Downing; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=34; citation_pages=14925-14930; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Effect of mono- and multivalent salts on angle-dependent attractions between charged rods.; citation_author=KC Lee; citation_author=I Borukhov; citation_author=WM Gelbart; citation_author=AJ Liu; citation_author=MJ Stevens; citation_journal_title=Phys Rev Lett; citation_volume=93; citation_number=35; citation_pages=128101; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nucleosome gaping supports a functional structure for the 30 nm chromatin fiber.; citation_author=J Mozziconacci; citation_author=JM Victor; citation_journal_title=J Struct Biol; citation_volume=143; citation_number=36; citation_pages=72-76; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Role of histone tails in chromatin folding revealed by a mesoscopic oligonucleosome model.; citation_author=G Arya; citation_author=T Schlick; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=37; citation_pages=16236-16241; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=JUMNA (junction minimisation of nucleic acids).; citation_author=R Lavery; citation_author=K Zakrzewska; citation_author=H Sklenar; citation_journal_title=Computer Physics Communications; citation_volume=91; citation_number=38; citation_pages=135-158; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Full cyclic coordinate descent: solving the protein loop closure problem in Calpha space.; citation_author=W Boomsma; citation_author=T Hamelryck; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=39; citation_pages=159; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Selectively Damped Least Squares for Inverse Kinematics.; citation_author=SR Buss; citation_author=J-S Kim; citation_journal_title=J Graph Tools; citation_volume=10; citation_number=40; citation_pages=37-49; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GROMACS: a message-passing parallel molecular dynamics implementation.; citation_author=HJC Berendsen; citation_author=D van der Spoel; citation_author=R van Drunen; citation_journal_title=Computer Physics Communications; citation_volume=91; citation_number=41; citation_pages=43-56; citation_date=1995; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length"/>
    <meta name="twitter:description" content="In the nucleus of eukaryotic cells, histone proteins organize the linear genome into a functional and hierarchical architecture. In this paper, we use the crystal structures of the nucleosome core particle, B-DNA and the globular domain of H5 linker histone to build the first all-atom model of compact chromatin fibers. In this 3D jigsaw puzzle, DNA bending is achieved by solving an inverse kinematics problem. Our model is based on recent electron microscopy measurements of reconstituted fiber dimensions. Strikingly, we find that the chromatin fiber containing linker histones is a polymorphic structure. We show that different fiber conformations are obtained by tuning the linker histone orientation at the nucleosomes entry/exit according to the nucleosomal repeat length. We propose that the observed in vivo quantization of nucleosomal repeat length could reflect nature's ability to use the DNA molecule's helical geometry in order to give chromatin versatile topological and mechanical properties."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000877.g006"/>

  <meta property="og:title" content="An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1280'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1890'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2831&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000877">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000877" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000877&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hua Wong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Physique Théorique de la Matière Condensée, Université Pierre et Marie Curie, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-Marc Victor, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire de Physique Théorique de la Matière Condensée, Université Pierre et Marie Curie, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julien Mozziconacci
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:pipo@lptmc.jussieu.fr">pipo@lptmc.jussieu.fr</a></p>

                <p>Affiliations:
                  Laboratoire de Physique Théorique de la Matière Condensée, Université Pierre et Marie Curie, Paris, France, 
                  Cell Biology and Biophysics, European Molecular Biology Laboratory, Heidelberg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000877</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000877-g001" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000877-g002" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000877-t001" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000877-g003" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000877-g004" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000877-g005" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000877-g006" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000877">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000877" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>In the nucleus of eukaryotic cells, histone proteins organize the linear genome into a functional and hierarchical architecture. In this paper, we use the crystal structures of the nucleosome core particle, B-DNA and the globular domain of H5 linker histone to build the first all-atom model of compact chromatin fibers. In this 3D jigsaw puzzle, DNA bending is achieved by solving an inverse kinematics problem. Our model is based on recent electron microscopy measurements of reconstituted fiber dimensions. Strikingly, we find that the chromatin fiber containing linker histones is a polymorphic structure. We show that different fiber conformations are obtained by tuning the linker histone orientation at the nucleosomes entry/exit according to the nucleosomal repeat length. We propose that the observed <em>in vivo</em> quantization of nucleosomal repeat length could reflect nature's ability to use the DNA molecule's helical geometry in order to give chromatin versatile topological and mechanical properties.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wong H, Victor J-M, Mozziconacci J (2007) An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length. PLoS ONE 2(9):
          e877.
            doi:10.1371/journal.pone.0000877</p><p><strong>Academic Editor: </strong>Pu Chen, University of Waterloo, Canada</p><p><strong>Received:</strong> June 20, 2007; <strong>Accepted:</strong> August 18, 2007; <strong>Published:</strong> September 12, 2007</p><p><strong>Copyright:</strong> © 2007 Wong et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Work supported by the CNRS and UPMC.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The nucleus of all eukaryotic cells contains meters of DNA organized by histone proteins into a hierarchical architecture: the chromatin <a href="#pone.0000877-1">[1]</a>, <a href="#pone.0000877-Woodcock1">[2]</a>. The chromatin fundamental unit, the nucleosome core particle (NCP), organizes 147 base pairs (bp) of DNA in a 1.7 left-handed superhelical turns around an octamer of four core histones (H2A, H2B, H3, H4)<sub>2</sub>. Nucleosomes are regularly spaced along DNA forming a beads-on-a-string structure, which can in turn fold into a compact 30 nm diameter fiber. A fifth histone type (H1/H5) helps this folding by interacting with DNA at the entry-exit of each nucleosome <a href="#pone.0000877-Travers1">[3]</a>. This interaction has been observed at the molecular level <em>in vitro</em> <a href="#pone.0000877-Bednar1">[4]</a> and induces the wrapping of 20 additional DNA bp to complete the second full turn around each nucleosome. Semi-compact fibers can then be obtained by the apposition of entry and exit DNA linkers <a href="#pone.0000877-Bednar2">[5]</a>. However, this zigzag folding pattern is not compact enough to explain the high density of chromatin as measured <em>in vivo</em> in metaphasic chromosomes <a href="#pone.0000877-Daban1">[6]</a> (6 nucleosomes/11 nm compared to 11 nucleosomes/11 nm).</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recently, highly compact fibers have been reconstructed <em>in vitro</em>, using a strong nucleosome positioning DNA sequence, a high divalent cation concentration, and linker histones <a href="#pone.0000877-Robinson1">[7]</a>. These compact fibers have been imaged by electron microscopy (EM), but the obtained images do not give precise information on the positions of nucleosomes and DNA linkers. In order to provide a model of the internal structure of the compact fiber, Robinson <em>et al.</em> measured the dimensions (diameter and linear compaction) of fibers reconstructed with different DNA linker lengths, ranging from 10 to 70 bp. The three main models of fiber internal structure, reviewed in <a href="#pone.0000877-Dorigo1">[8]</a> predict discriminant variations of those two dimensions with respect to a change of nucleosomal repeat length (NRL):</p>


<ol class="roman-lower">

<li>The solenoidal model <a href="#pone.0000877-Widom1">[9]</a> predicts no variation of the two dimensional parameters with the NRL.</li>

<li>The two-start super-coiled model, or twisted ribbon <a href="#pone.0000877-Worcel1">[10]</a> predicts a linear variation of the compaction with the NRL.</li>

<li>The two-start twisted model, or cross-linker model <a href="#pone.0000877-Williams1">[11]</a> predicts a linear variation of the diameter with the NRL.</li>

</ol><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The results of Robinson <em>et al.</em> are reported on <a href="#pone-0000877-g001">Fig. 1</a>. They find that there are two different classes of fibers. Arrays with NRL ranging from 177 up to 207 DNA bp fold into a fiber that is roughly 35 nm in diameter and 11 nucleosomes/11 nm in compaction, while arrays with linker length ranging from 50 up to 70 bp produce fibers of 45 nm in diameter and 15 nucleosomes/11 nm in compaction. Those measures led them to propose an interdigitated solenoidal structure closely related to the one proposed by Daban and Bermudez <a href="#pone.0000877-Daban2">[12]</a>. This structure combines two interesting features: it is compact enough to account for the EM measurements and its diameter is fixed by nucleosome face-to-face contacts and therefore does not depend on linker length. Nucleosomes are radially positioned around the fiber axis, the n+5<sup>th</sup> being stacked on the n<sup>th</sup>. Linkers are bent, making a loop in between two consecutive nucleosomes. The size of this loop is then increasing with the length of the linker. However, in their model, Robinson <em>et al.</em> do not provide any explicit path of the DNA linker within the fiber.</p>
<div class="figure" id="pone-0000877-g001"><div class="img"><a name="pone-0000877-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>EM-measurements of the diameter (A) and the compaction (B) of reconstituted chromatin fibers containing linker histone for 7 different NRL.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Adapted from <a href="#pone.0000877-Robinson1">[7]</a>.</p>
<span>doi:10.1371/journal.pone.0000877.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study, we propose the first all-atom structural model of the chromatin fiber containing H5 linker histone. In the first part of our work, we provide a classification of the different possibilities for nucleosome core particle (NCP) positions and orientations in the context of a homogeneous fiber. We then connect those NCP with DNA linkers of given lengths using skeletal animation together with inverse kinematics (see <a href="#s4">Material and Methods</a>). For each NRL used in the experiment by Robinson <em>et al.</em> we screen all possible nucleosomes positions and orientations compatible with the fiber dimensions. We selected the ones in which DNA linkers were able to connect consecutive nucleosomes without steric clashes in-between atoms or chemical bonds significant distortions. H5 was then placed into the structures that satisfy both energetic and steric constraints according to previous experimental and computational results.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Our findings suggest that, as previously proposed <a href="#pone.0000877-Travers1">[3]</a>, <a href="#pone.0000877-Bednar2">[5]</a>, H5 leads to the compaction of the fiber by closing two turns of DNA around the nucleosome. The unexpected outcome of our study is that the chromatin fiber containing linker histones is a highly polymorph structure, tuned by the NRL. Most of the structures we describe here have already been proposed based on numerous experimental and theoretical studies on the 30 nm chromatin fiber: solenoids <a href="#pone.0000877-Widom1">[9]</a>, 2-start helix <a href="#pone.0000877-Dorigo1">[8]</a>, <a href="#pone.0000877-Williams1">[11]</a>, <a href="#pone.0000877-Schalch1">[13]</a>, 3-start helix <a href="#pone.0000877-Makarov1">[14]</a> and 5-start helix <a href="#pone.0000877-Daban2">[12]</a>. In the past decades, so many different models were proposed for compact fibers that the question whether there is really such a structure ultimately arose <a href="#pone.0000877-vanHolde1">[15]</a>. We interpret this multiplicity of models as a property of the fiber itself: there is a multiplicity of fiber structures! We propose that these different structures, which differ both in mechanical <a href="#pone.0000877-BenHaim1">[16]</a> and topological <a href="#pone.0000877-Barbi1">[17]</a> properties, are selected by changing the NRL. Those different physical properties in turn can control the accessibility to the DNA sequence.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Inventory of NCP positions within a regular fiber</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The input data used in our modeling process are the dimensions (i.e. diameter and compaction) of the chromatin fiber. Very regular and homogeneous fibers have been reconstructed and carefully measured, <em>in vitro</em>, using a strong positioning sequence, linker histones and a high magnesium concentration <a href="#pone.0000877-Robinson1">[7]</a>. Using cryo-EM results, a technique that is more likely to keep the structures intact, it has been found that the diameter of the fiber was ~35 nm. This diameter was conserved for NRL comprised between 177 and 207 bp. For longer NRL, the diameter was shown to jump to ~45 nm. In a similar manner, the compaction of such a fiber was shown to increase from 11 nucleosomes/11 nm for 35 nm diameter fibers to 16 nucleosomes/11 nm for 45 nm diameter fibers (<a href="#pone-0000877-g001">Fig. 1</a>). We therefore based our <em>in silico</em> fiber reconstruction on those two values of the diameter and compaction.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The first step of our modeling process is to place 30 NCP all-atom structures in 3D space in order to obtain a regular fiber of given diameter and compaction (see <a href="#s4">Material and Methods</a>). Previous structural studies on isolated NCP showed that they have a strong tendency to interact through their faces <a href="#pone.0000877-Tatchell1">[18]</a>, forming arcs, helices <a href="#pone.0000877-Dubochet1">[19]</a> or liquid crystal phases <a href="#pone.0000877-Mangenot1">[20]</a>. There is now increasing evidence that this stacking is also favored within dense chromatin fibers <a href="#pone.0000877-Dorigo1">[8]</a>, <a href="#pone.0000877-Schalch1">[13]</a>. Additionally, EM <a href="#pone.0000877-Bartolome1">[21]</a> and theoretical <a href="#pone.0000877-BenHaim1">[16]</a> studies pointed out that in the context of a chromatin fiber, nucleosomes are radially distributed, with their dyad axis orthogonal to the fiber axis <a href="#pone.0000877-Bartolome1">[21]</a>. Based on these experimental pieces of evidence, a simple space-filling model suggests that regular fiber structures are multiple left or right-handed, helices of stacked nucleosomes <a href="#pone.0000877-Daban2">[12]</a>. To label these different structures, we used one number and two letters. The number refers to the number of starts of the helix while the first letter, capital R or L, refers to the handedness of the helix and the second letter, lower case r or l, refers to the handedness of the DNA path between nucleosomes (<a href="#pone-0000877-g002">Fig. 2</a>). We would like to stress here that all fiber models with a DNA linker placed inside the fiber and stacked nucleosomes fall into this classification. For example, the two-start helix proposed by Richmond and colleagues from the crystal structure of a tetra-nucleosome <a href="#pone.0000877-Schalch1">[13]</a> is the 2Lr fiber whereas the one proposed by Robinson and co-workers is the 5Rl <a href="#pone.0000877-Robinson1">[7]</a>. Importantly, the authors of the latter paper refer to their structure as a one-start structure (according to the DNA path around the fiber axis), whereas in our notations, it is a five-start structure, the n+5<sup>th</sup> nucleosome being stacked on the n<sup>th </sup>one.</p>
<div class="figure" id="pone-0000877-g002"><div class="img"><a name="pone-0000877-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Scheme of the NCP positions within the chromatin fiber.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Top: the spheres represent 3D positions of the nucleosomes' centers of mass. The joints in between them show the DNA connectivity. The j angle is the azimuthal rotation angle in between consecutive nucleosomes. Bottom: schematic representation of the cylindrical surface containing all nucleosomes' centers of mass. The angle q is the nucleosomes tilt. A scaled all-atom structure of nucleosomal DNA has been included in one of the box representing nucleosomes. The lower case l corresponds to the left-handed helicity of the DNA path connecting nucleosomes. The capital R corresponds to the right-handedness of the helices made by stacked nucleosomes. This fiber is therefore denominated 5Rl in our classification.</p>
<span>doi:10.1371/journal.pone.0000877.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Based on this classification, we built a library of 16 template fiber structures ranging from one to five start helices, by placing the 30 NCP crystal structures (1KX5) <a href="#pone.0000877-Davey1">[22]</a> according to the measured fibers dimensions. We then tried to connect those NCP with DNA linkers of different lengths, corresponding to the different NRL used in the experiment and selected the fibers in which consecutive NCP can be connected without steric clashes in-between atoms or chemical bonds significant distortions.</p>


<h4>Placement of the DNA linkers connecting consecutive NCP</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We found that the phasing angle between DNA at the exit of one nucleosome and the entry of the consecutive nucleosome is a very strong constraint to select a structure, due to the DNA natural helical twist <a href="#pone.0000877-BenHaim1">[16]</a>, <a href="#pone.0000877-Widom2">[23]</a>, <a href="#pone.0000877-Yao1">[24]</a>. In fact, for each NRL, only 1 to 3 NCP positions and orientations were compatible with both the length and the relaxed twist of the linkers (see <a href="#pone-0000877-t001">Table 1</a>). In some cases, phasing and distance between consecutive NCP were compatible with a given NRL, but linkers were intersecting with each other. Using forward kinematics on the first 20 bp at the entry/exit of the nucleosomes, i.e. by moving them by hand, we found that wrapping 20 bp of the linker around the histone octamer was a very efficient way to avoid this steric hindrance.</p>
<div class="figure" id="pone-0000877-t001"><div class="img"><a name="pone-0000877-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Possible fiber structures for different NRL.</span></strong></p><span>doi:10.1371/journal.pone.0000877.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>For the 177 bp NRL, the only structure that satisfied all the geometric constraints had nucleosomes stacked in a 5-start right-handed helix and DNA following a left-handed path around the fiber axis: according to our classification, it is a 5Rl. This structure is very close to the structure proposed by Robinson <em>et al.</em> based on simple geometric constraints. In their paper, they suggested that longer linkers (for NRL 187, 197, 207 bp) were bent to fit in the interior of the fiber. Our modeling procedure allowed us to test this prediction. We found that this structure was still compatible with a 187 NRL. However, for longer NRL, 5Rl type fibers could not be obtained anymore because of persistent steric clashes in between linkers. For 197 and 207 bp NRL, only 3 and 2-start structures could match physical and geometrical constraints. For the 217 bp NRL, we found no structure compatible with a 35 nm diameter fiber (<a href="#pone-0000877-t001">Table 1</a>; 35 nm diameter, 217 bp NRL). However, if the diameter was increased by 10 nm, as suggested by the EM measurements <a href="#pone.0000877-Robinson1">[7]</a>, a new class of fibers could be obtained. The 217 bp NRL was compatible with both 4 and 5-start right handed helices whereas 227 and 237 bp NRL were compatible with 3 and 2-start left handed helices. For each NRL, a 3D representation of the DNA path in the compact fiber of lower energy is presented in <a href="#pone-0000877-g003">Fig. 3A</a>. As can be seen from the top views, DNA follows a path shaped as: a pentagon for 177, a square for 187 and 217, a triangle for 197 and 227 and a line along the diameter for 207 and 237. This simple geometrical fact can explain how linkers of different lengths can fit inside an internal cavity of constant size.</p>
<div class="figure" id="pone-0000877-g003"><div class="img"><a name="pone-0000877-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Structures of the chromatin fiber for different NRL.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) All-atom models of the 7 chromatin fibers obtained by H5-induced folding of a regular chromatin array <a href="#pone.0000877-Robinson1">[7]</a>. For clarity, DNA is shown as a tube. The yellow color corresponds to linkers whereas nucleosomal DNA is presented in pink. Molecular surfaces of H5 globular domains are shown in white. Each structure corresponds to a different NRL. From left to right: 177 bp, 5Rl; 187 bp, 4Rl; 197 bp, 3Rr; 207 bp, 2Rl; 217 bp, 4Rl; 227 bp, 3Ll; 227 bp, 2Lr. For 177 to 207 bp NRL, one DNA repeat length is shown in color (blue, purple, green and orange for 177, 187, 197, and 207 bp NRL respectively). B) Close-up of the four chromatosomes presented in color in A. C- and N-terminal tail extensions are presented in light pink and yellow respectively. The H5 globular domain residues that interact with entry and exit DNA linkers are colored in blue and red. The residues that are interacting with nucleosomal DNA at the dyad axis are colored in green. C) Molecular details of the GH5 positioned on a nucleosome (for 177 and 187 bp NRL). The residues involved in DNA binding <a href="#pone.0000877-Duggan1">[25]</a>–<a href="#pone.0000877-Brown1">[27]</a> are shown in licorice. In blue: residues interacting with one DNA linker; in red: residues interacting with the other DNA linker; in green: residues inserted into the DNA major groove next to the dyad axis. R47, shown in cyan is never found in contact with DNA.</p>
<span>doi:10.1371/journal.pone.0000877.g003</span></div>

<h4>Position, orientation and role of linker histone within the chromatin fiber</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The last step of our modeling was to place the linker histone globular domain inside our “DNA-and-core-histones” all-atom fiber. Several models have been proposed for the placement and orientation of H1/H5 <a href="#pone.0000877-Travers1">[3]</a>. The common feature of those models is that H1/H5 binds at the entry/exit of the nucleosome and closes two turns of DNA around the core histones by interacting with one or two DNA linkers. Biochemical studies have identified the positively charged surface residues involved in DNA-H1/H5 contacts <a href="#pone.0000877-Duggan1">[25]</a>–<a href="#pone.0000877-Brown1">[27]</a>. Recently, using a computational approach, plausible positions and orientations of the H5 linker histone on a nucleosome were proposed <a href="#pone.0000877-Fan1">[28]</a>. The residues involved in this positioning, i.e. those interacting with DNA, were matching the residues pointed out by former biochemical studies. Here, we placed the atomic structure of the globular domain of H5 (1HST) <a href="#pone.0000877-Ramakrishnan1">[29]</a> according to these published structural data.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In <a href="#pone-0000877-g003">Fig. 3B</a> we present a detailed view of chromatosomes (i.e. nucleosome+linker histone) found in fibers constructed for NRL equal to 177, 187, 197 and 207 bp. In these four structures, the globular domain of H5 could bind three DNA double strands. Our model showed different conformations of the chromatosome for different fiber structures, suggesting that H1/H5 helps adjusting the entry/exit angle of DNA on a nucleosome for all the different possible fiber structures.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>First, two different positions of the globular part of the linker histone GH5 within the chromatosome could be identified:</p>


<ul class="bulletlist">

<li>For 177 bp repeat length, the entry and exit linkers were fully wrapped on the nucleosome and GH5 could be placed on the dyad axis, just in between the two linkers, in a similar way as in the former model of Allan <em>et al</em> <a href="#pone.0000877-Allan1">[30]</a> (<a href="#pone-0000877-g003">Fig. 3B</a>, dark blue).</li>

<li>For longer NRL, the entry/exit linker are more opened and form a pocket in which GH5 can be placed, bridging both DNA linkers together with the nucleosomal DNA at the dyad axis (<a href="#pone-0000877-g003">Fig. 3B</a>, purple, green and orange).</li>

</ul><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>In <a href="#pone-0000877-g003">Fig. 3C</a>, we present a detailed view of the contacts made by GH5 with DNA in the chromatosome for those two conformations. Importantly, 80% of the contacts in between GH5 and DNA were the same in both conformations. By comparing those two conformations, we propose that a first tuning of the entry/exit linker angle can be achieved via a ball-bearing-like rotation of GH5 on itself around the dyad axis.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>An examination of the DNA linker path at the exit of each chromatosome pointed out that a second way to tune the entry/exit angle was to make a stem in between linkers at the chromatosome exit (see <a href="#pone-0000877-g003">Fig. 3B</a>). Similar structures were indeed observed on mono-nucleosomes assembled on linear DNA fragments together with linker histones <a href="#pone.0000877-Hamiche1">[31]</a> and on regular fibers <a href="#pone.0000877-Bednar2">[5]</a>. Different stem lengths could account for the differences in DNA linker orientations in 4, 3, and 2-starts structures. The H5-dependent organization and dynamics of the nucleosome entry/exit DNA on mini-circles <a href="#pone.0000877-Sivolob1">[32]</a> pointed out the role of the linker histone C-terminal tail in stabilizing such structures. Accordingly, in our model, both the C-terminal and N-terminal tails of the linker histone were found to be in close contact with the DNA linkers at the entry/exit of the chromatosome. We propose that this interaction depends on linker length:</p>


<ul class="bulletlist">

<li>For the 177 bp NRL, the DNA joining consecutive chromatosomes is only 10 bp long. There is no possible binding site for H5 tails and the linker is essentially straight.</li>

<li>For the 187, 197 and 207 bp NRL, H5 tails are able to make respectively, one, two, and three contacts with DNA linkers. These interactions induce DNA kinks, and a consequent change in DNA linkers' relative direction at the chromatosome exit.</li>

</ul>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Linker-linker and nucleosome-nucleosome interactions in a compact chromatin fiber: energetic considerations</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>In the fibers presented on <a href="#pone-0000877-g003">Fig. 3A</a>, 60 to more than 80% of the fiber volume is occupied by proteins or DNA. Linkers are aligned next to each other allowing this very high fiber compaction. This arrangement is reminiscent of DNA cholesteric liquid crystals (especially the structures obtained for the 197, 207, 227 and 237 bp NRL). The pitch of the cholesteric phase depends on the number of starts of the chromatin fiber helix and ranges from 35 to 100 nm. These values are in good agreement with <em>in vivo</em> measurements in sperm nuclei <a href="#pone.0000877-Leforestier1">[33]</a>; however, whereas DNA cholesteric crystals are only left-handed, our model shows both left and right-handed helicity. The distance between two DNA double strands can be found to be as small as 2.5 nm, as observed in hexagonal packing of bacteriophage DNA <a href="#pone.0000877-Hud1">[34]</a>. It should be noticed that this high compaction is favored by the strong positive divalent ion concentration. Mg<sup>2+</sup> ions not only permit negatively charged DNA strands to come into vicinity, but moreover participate in the formation of electrostatic bridges which in turn insure the stability of the structure <a href="#pone.0000877-Lee1">[35]</a>. When the NRL increases from 177 to 207 bp, the amount of DNA material inside the fiber also increases. We propose that the concentration of divalent cation is locally rising and participates in the electrostatic stability of the structure.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The stacking of nucleosomes bring into close proximity residues H2A–(A53, E56, E64), H2B–(Q44, V45, E110) of one nucleosome with H3–(E73, Q76, D77), H4–(K20, L22, R23) of the neighboring nucleosome so that specific interactions between them can occur upon nucleosome gaping <a href="#pone.0000877-Mozziconacci1">[36]</a> and potentially lock the fiber in a compact state <a href="#pone.0000877-Dorigo1">[8]</a>, <a href="#pone.0000877-Davey1">[22]</a> (see <a href="#pone-0000877-g004">Fig. 4</a>). Our model does not include explicit histone tail rearrangements. However, since we provide a detailed model of the dense fiber, the role of each histone tails in nucleosome/nucleosome and linker/nucleosome interactions can be addressed. On <a href="#pone-0000877-g004">Fig. 4</a>, we present the extension of each N-terminal tail and H2A C-terminal tail based on results obtained using a mesoscopic oligonucleosome model <a href="#pone.0000877-Arya1">[37]</a>. H3 tails interact with DNA linkers; H4 and H2A tails participate in nucleosome stacking interactions whereas H2B tails are likely to bridge adjacent columns of stacked nucleosomes. We hope that this model will help to gain meaningful insights into the influence of histone tails modifications on chromatin structure.</p>
<div class="figure" id="pone-0000877-g004"><div class="img"><a name="pone-0000877-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>All-Atom structures of the chromatin fiber for 4 different NRL.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>Top: one nucleosome is presented at higher resolution to show the color codes used for histone, histone tails and specific residues. Histone tail extensions are presented as colored circles. Middle: top and side view of the all atom fibers for the 177, 187, 197 and 207 bp NRL. Bottom: close-up on each chromatosome highlighting the gaping of each nucleosome.</p>
<span>doi:10.1371/journal.pone.0000877.g004</span></div>

<h4><em>In vivo</em> relevance of our results</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Our approach intends to model fibers obtained <em>in vitro</em> in very specific and controlled conditions. The question of the <em>in vivo</em> relevance of our modeling therefore arises.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>In our models, the nucleosome tilt angle is ranging from 12° for 2-start helices to 40° for 5-start helices. This value is in good agreement with the range measured for chromatin fibers isolated from different species (13° to 38°) <a href="#pone.0000877-Robinson1">[7]</a>. We suggest that the wide range of tilt angle values observed <em>in vivo</em> corresponds to different structures rather than to an error around a mean value corresponding to a unique structure.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>While comparing the radius of isolated chromatin fibers with the NRL in different organisms, Langmore and co-workers, concluded that there is a linear relationship between those two quantities <a href="#pone.0000877-Williams1">[11]</a>, in apparent contradiction with the <em>in vitro</em> results found by Robinson <em>et al.</em> <a href="#pone.0000877-Robinson1">[7]</a>. However, a careful re-examination of their results leads to an opposite conclusion. In fact, in Fig.7 from reference <a href="#pone.0000877-Williams1">[11]</a>, one can clearly distinguish two classes of fibers, one of them, for NRL shorter than 210 bp, corresponds to fiber diameters around 31 nm and the other, for longer linkers, corresponds to fiber diameters around 39 nm. The fact that two statistically different clusters exist suggests that the two different classes of diameter reported in <a href="#pone.0000877-Robinson1">[7]</a> can be relevant <em>in vivo</em> as well. However, note that difference in values (31 nm vs. 35 nm and 39 nm vs. 45 nm) for the compaction and diameter of the fiber found in different studies can be explained by the different biological sample preparation and imaging.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>It is still a matter of debate whether nucleosomes are strongly and regularly positioned <em>in vivo</em>. To see how our model can account for non-uniform NRL, we plotted the twist energy of the linkers for all NRL ranging from 186 to 200 for all possible structures (<a href="#pone-0000877-g005">Fig.5</a>). As expected, for right-handed helices, minimal energy is found for 187 and 197 bp NRL. This minimum is rather flat allowing 1 to 2 bp local changes in the NRL without energy increase.</p>
<div class="figure" id="pone-0000877-g005"><div class="img"><a name="pone-0000877-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Plots of DNA linker twist for different structures and NRL.</span></strong></p><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1"></a><p>A) For right-handed helices. B) For left-handed helices. Non-displayed points correspond to NRL for which the linker does not join the consecutive nucleosomes.</p>
<span>doi:10.1371/journal.pone.0000877.g005</span></div><a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p><a href="#pone-0000877-g005">Fig. 5</a> also shows that for left-handed helices the minimum energy is found for 192 bp NRL. Interestingly, it has been measured that 192 bp NRL favors the formation of left-handed helices <a href="#pone.0000877-Williams1">[11]</a> and that chromatin fiber isolated from chicken erythrocyte is a mix of both left and right handed conformations <a href="#pone.0000877-Bartolome1">[21]</a>. This further suggests that the different structures reported here might be relevant <em>in vivo.</em></p>
<a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>The NRL used by Robinson <em>et al</em>. in their experiment (177, 187, 197, 207, 217, 227 and 237 bp) <a href="#pone.0000877-Robinson1">[7]</a> were chosen to match the NRL most often found in nature (159, 171, 178, 188, 192, 196, 206–207 and 217–218 bp) <a href="#pone.0000877-Widom2">[23]</a>. As the period of this NRL quantization is close to the helical repeat of DNA, it has been proposed to be required for selecting a precise phasing angle between consecutive nucleosomes, necessary for higher order folding of the beads-on-a-string nucleosomal array. This point was illustrated in <a href="#pone.0000877-Sivolob1">[32]</a> in the case of the solenoidal model and could be extended to other structures. Our results support a similar, however, more precise explanation: we propose here that different NRL correspond to different structures, ranging from the classical solenoid for 167 bp (not shown) to n-start cross-linker models. This prediction could be tested by inter-nucleosome cross-linking followed by linker digestion experiments on regular fibers as illustrated in <a href="#pone.0000877-Dorigo1">[8]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Calculation of NCP positions within a fiber</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The first step of our modeling is to place the NCP, in a way that prevents steric clashes (<a href="#pone-0000877-g002">Fig. 2</a>). Knowing the dimensions of the fiber and the number of starts considered, the tilt <em>θ</em> of the NCP with respect to the fiber axis can be simply calculated:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>In this expression, <em>D</em> is the diameter of the fiber, <em>a</em> is the diameter of a NCP, and <em>N</em> is the number of starts of the helix.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>The position of the n+1<sup>th</sup> NCP can be derived from the position of the n<sup>th</sup> one by rotation around the fiber axis of an angle <em>ϕ</em> and a translation of length <em>d</em> along the fiber axis<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <em>C</em> is the compaction expressed in nucleosomes/11nm. The sign±in the expression of <em>ϕ</em> corresponds to right or left handed helices (R or L), while it corresponds to right or left-handed DNA paths (r or l) in the expression of <em>d</em>.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Knowing the diameter and the compaction of a fiber, we can place the atomic structure of each nucleosome (1KX5) <a href="#pone.0000877-Davey1">[22]</a> according to those geometric constraints (<a href="#pone-0000877-g002">Fig. 2</a>).</p>


<h4>Molecular modeling of linker DNA</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We first remove 10 DNA bp at the entry-exit of each NCP. We then add a straight, B-form, random sequence, DNA double helix of (NRL–127) bp length at the exit of the first nucleosome. This DNA is created with a rise of 3.4 Å and a helical repeat of 10.5 bp using JUMNA <a href="#pone.0000877-Lavery1">[38]</a>. At the end of this linker, we add one extra base pair (in red on <a href="#pone-0000877-g006">Fig. 6</a>), which we try to align with the entry of the consecutive nucleosome (in green on <a href="#pone-0000877-g006">Fig. 6</a>) effectively superimposing their coordinate frames. To achieve this goal, we place a linear skeleton along the DNA axis divided in bones of 10 bp length (in blue on <a href="#pone-0000877-g006">Fig. 6</a>) and use inverse kinematics (IK) in order to deform this skeleton. IK is common in robotics and has already been used in molecular modeling studies of protein loops <a href="#pone.0000877-Boomsma1">[39]</a>. Solving the IK problem tells you how much you have to rotate each joint in an articulated chain so that the end of the chain (effector on <a href="#pone-0000877-g006">Fig. 6</a>) reaches a required position and orientation (target on <a href="#pone-0000877-g006">Fig. 6</a>). Once the skeleton has been deformed, it is possible to reconstruct the full DNA double helix around it using deformation matrices. The all-atom structure is then refined using a short energy minimization run. The novelty of this approach is that it offers the possibility to build all-atom models of bent DNA linkers. We made use of the free 3D package Blender (<a href="http://blender3d.org/cms/Home.2.0.html">http://blender3d.org/cms/Home.2.0.html</a>) because it offers a robust IK algorithm based on a damped Jacobian transpose method <a href="#pone.0000877-Buss1">[40]</a> completed with a constraint system. The constraint system allows us to limit the rotational freedom of each link in order to mimic the shape a real DNA would adopt. For each linker, we restrict the torsion/bending of DNA to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.e004&amp;representation=PNG" class="inline-graphic"></span> where L is the length of the linker and C the respective twist/bending persistence length of DNA (50 nm/75 nm). This allows us to select structures in which the DNA linker joins consecutive nucleosomes with bending and twisting energies smaller than kT. Energies have been calculated as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.e005&amp;representation=PNG" class="inline-graphic"></span><br>where <strong><em>C</em></strong> is the twist persistent length of DNA, <strong><em>Tw</em></strong> the twist of the linker and <strong><em>L</em></strong> the linker length. Note that we increased the bending flexibility of the first 20 DNA bp at the entry/exit of each nucleosome, to mimic its interaction with H5 or with histone core tails. When we find a linker fitting inside the structure, we simply replicate it in order to form a continuous DNA double strand connecting all nucleosomes in the fiber.</p>
<div class="figure" id="pone-0000877-g006"><div class="img"><a name="pone-0000877-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000877" data-uri="info:doi/10.1371/journal.pone.0000877.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000877.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000877.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000877.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000877.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Illustration of the IK chain used in order to place linkers in between consecutive nucleosomes.</span></strong></p><a id="article1.body1.sec4.sec2.fig1.caption1.p1" name="article1.body1.sec4.sec2.fig1.caption1.p1"></a><p>In pink we show the nucleosomal DNA and in yellow the DNA linker. The bones supporting the linker DNA are shown in blue. We start from a straight conformation and then try to see whether or not it is possible to superimpose the last bp of the DNA linker (in red) with the first DNA bp of the consecutive nucleosome (in green) when relocating the effector on the target.</p>
<span>doi:10.1371/journal.pone.0000877.g006</span></div>

<h4>Docking of linker histone on the nucleosome</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We first minimize the atomic structure (1HST) and run a 800 ps molecular dynamics simulation in explicit solvent using GROMACS <a href="#pone.0000877-Berendsen1">[41]</a> in order to relax the protein side chains. Then we insert lysine K85 next to the major groove of DNA at the dyad axis, as it was found to be in 27 out of 30 top-ranked docking positions <a href="#pone.0000877-Fan1">[28]</a>, and oriented the globular domain so that the residues involved in DNA binding (R42, R47, R73, R74, R94, K40, K69, K85, K97, H 25 and H62) are making close contacts with the entry and exit linkers.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to acknowledge M. Barbi, A. Lesne, A. Prunell, A. Bancaud and N. Desprat for fruitful discussions. We are grateful to M. Quaggetto for technical support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JM. Performed the experiments: HW. Analyzed the data: JM HW. Contributed reagents/materials/analysis tools: HW. Wrote the paper: JM. Other: Initiated the work: JV. Provided critical comments: JV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000877-1" id="pone.0000877-1"></a>Van Holde KE Chromatin. Springer series in molecular biology  <ul class="find" data-citedArticleID="990642" data-doi="10.1007/978-1-4612-3490-6"><li><a href="http://dx.doi.org/10.1007/978-1-4612-3490-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Van+Holde+KE+Chromatin.+Springer+series+in+molecular+biology" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Van+Holde+KE+Chromatin.+Springer+series+in+molecular+biology%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000877-Woodcock1" id="pone.0000877-Woodcock1"></a>Woodcock CL, Dimitrov S (2001) Higher-order structure of chromatin and chromosomes. Curr Opin Genet Dev  11: 130–135.  <ul class="find" data-citedArticleID="990716" data-doi="10.1016/s0959-437x(00)00169-6"><li><a href="http://dx.doi.org/10.1016/s0959-437x(00)00169-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-order+structure+of+chromatin+and+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-order+structure+of+chromatin+and+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000877-Travers1" id="pone.0000877-Travers1"></a>Travers A (1999) The location of the linker histone on the nucleosome. Trends Biochem Sci  24: 4–7.  <ul class="find" data-citedArticleID="990708" data-doi="10.1016/s0968-0004(98)01339-5"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01339-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+location+of+the+linker+histone+on+the+nucleosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+location+of+the+linker+histone+on+the+nucleosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000877-Bednar1" id="pone.0000877-Bednar1"></a>Bednar J, Horowitz RA, Dubochet J, Woodcock CL (1995) Chromatin conformation and salt-induced compaction: three-dimensional structural information from cryoelectron microscopy. J Cell Biol  131: 1365–1376.  <ul class="find" data-citedArticleID="990652" data-doi="10.1083/jcb.131.6.1365"><li><a href="http://dx.doi.org/10.1083/jcb.131.6.1365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+conformation+and+salt-induced+compaction%3A+three-dimensional+structural+information+from+cryoelectron+microscopy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+conformation+and+salt-induced+compaction%3A+three-dimensional+structural+information+from+cryoelectron+microscopy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000877-Bednar2" id="pone.0000877-Bednar2"></a>Bednar J, Horowitz RA, Grigoryev SA, Carruthers LM, Hansen JC, et al.  (1998) Nucleosomes, linker DNA, and linker histone form a unique structural motif that directs the higher-order folding and compaction of chromatin. Proc Natl Acad Sci U S A  95: 14173–14178.  <ul class="find" data-citedArticleID="990654" data-doi="10.1073/pnas.95.24.14173"><li><a href="http://dx.doi.org/10.1073/pnas.95.24.14173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleosomes%2C+linker+DNA%2C+and+linker+histone+form+a+unique+structural+motif+that+directs+the+higher-order+folding+and+compaction+of+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleosomes%2C+linker+DNA%2C+and+linker+histone+form+a+unique+structural+motif+that+directs+the+higher-order+folding+and+compaction+of+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000877-Daban1" id="pone.0000877-Daban1"></a>Daban JR (2000) Physical constraints in the condensation of eukaryotic chromosomes. Local concentration of DNA versus linear packing ratio in higher order chromatin structures. Biochemistry  39: 3861–3866.  <ul class="find" data-citedArticleID="990666" data-doi="10.1021/bi992628w"><li><a href="http://dx.doi.org/10.1021/bi992628w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical+constraints+in+the+condensation+of+eukaryotic+chromosomes.+Local+concentration+of+DNA+versus+linear+packing+ratio+in+higher+order+chromatin+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical+constraints+in+the+condensation+of+eukaryotic+chromosomes.+Local+concentration+of+DNA+versus+linear+packing+ratio+in+higher+order+chromatin+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000877-Robinson1" id="pone.0000877-Robinson1"></a>Robinson PJ, Fairall L, Huynh VA, Rhodes D (2006) EM measurements define the dimensions of the “30-nm” chromatin fiber: evidence for a compact, interdigitated structure. Proc Natl Acad Sci U S A  103: 6506–6511.  <ul class="find" data-citedArticleID="990700" data-doi="10.1073/pnas.0601212103"><li><a href="http://dx.doi.org/10.1073/pnas.0601212103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EM+measurements+define+the+dimensions+of+the+%26%23x201c%3B30-nm%26%23x201d%3B+chromatin+fiber%3A+evidence+for+a+compact%2C+interdigitated+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EM+measurements+define+the+dimensions+of+the+%26%23x201c%3B30-nm%26%23x201d%3B+chromatin+fiber%3A+evidence+for+a+compact%2C+interdigitated+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000877-Dorigo1" id="pone.0000877-Dorigo1"></a>Dorigo B, Schalch T, Kulangara A, Duda S, Schroeder RR, et al.  (2004) Nucleosome arrays reveal the two-start organization of the chromatin fiber. Science  306: 1571–1573.  <ul class="find" data-citedArticleID="990672" data-doi="10.1126/science.1103124"><li><a href="http://dx.doi.org/10.1126/science.1103124" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleosome+arrays+reveal+the+two-start+organization+of+the+chromatin+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleosome+arrays+reveal+the+two-start+organization+of+the+chromatin+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000877-Widom1" id="pone.0000877-Widom1"></a>Widom J, Klug A (1985) Structure of the 300A chromatin filament: X-ray diffraction from oriented samples. Cell  43: 207–213.  <ul class="find" data-citedArticleID="990710" data-doi="10.1016/0092-8674(85)90025-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(85)90025-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+the+300A+chromatin+filament%3A+X-ray+diffraction+from+oriented+samples." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+the+300A+chromatin+filament%3A+X-ray+diffraction+from+oriented+samples.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000877-Worcel1" id="pone.0000877-Worcel1"></a>Worcel A, Strogatz S, Riley D (1981) Structure of chromatin and the linking number of DNA. Proc Natl Acad Sci U S A  78: 1461–1465.  <ul class="find" data-citedArticleID="990718" data-doi="10.1073/pnas.78.3.1461"><li><a href="http://dx.doi.org/10.1073/pnas.78.3.1461" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+chromatin+and+the+linking+number+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+chromatin+and+the+linking+number+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000877-Williams1" id="pone.0000877-Williams1"></a>Williams SP, Athey BD, Muglia LJ, Schappe RS, Gough AH, et al.  (1986) Chromatin fibers are left-handed double helices with diameter and mass per unit length that depend on linker length. Biophys J  49: 233–248.  <ul class="find" data-citedArticleID="990714" data-doi="10.1016/s0006-3495(86)83637-2"><li><a href="http://dx.doi.org/10.1016/s0006-3495(86)83637-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+fibers+are+left-handed+double+helices+with+diameter+and+mass+per+unit+length+that+depend+on+linker+length." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+fibers+are+left-handed+double+helices+with+diameter+and+mass+per+unit+length+that+depend+on+linker+length.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000877-Daban2" id="pone.0000877-Daban2"></a>Daban JR, Bermudez A (1998) Interdigitated solenoid model for compact chromatin fibers. Biochemistry  37: 4299–4304.  <ul class="find" data-citedArticleID="990668" data-doi="10.1021/bi973117h"><li><a href="http://dx.doi.org/10.1021/bi973117h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interdigitated+solenoid+model+for+compact+chromatin+fibers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interdigitated+solenoid+model+for+compact+chromatin+fibers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000877-Schalch1" id="pone.0000877-Schalch1"></a>Schalch T, Duda S, Sargent DF, Richmond TJ (2005) X-ray structure of a tetranucleosome and its implications for the chromatin fibre. Nature  436: 138–141.  <ul class="find" data-citedArticleID="990702" data-doi="10.1038/nature03686"><li><a href="http://dx.doi.org/10.1038/nature03686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X-ray+structure+of+a+tetranucleosome+and+its+implications+for+the+chromatin+fibre." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X-ray+structure+of+a+tetranucleosome+and+its+implications+for+the+chromatin+fibre.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000877-Makarov1" id="pone.0000877-Makarov1"></a>Makarov V, Dimitrov S, Smirnov V, Pashev I (1985) A triple helix model for the structure of chromatin fiber. FEBS Lett  181: 357–361.  <ul class="find" data-citedArticleID="990690" data-doi="10.1016/0014-5793(85)80292-1"><li><a href="http://dx.doi.org/10.1016/0014-5793(85)80292-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+triple+helix+model+for+the+structure+of+chromatin+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+triple+helix+model+for+the+structure+of+chromatin+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000877-vanHolde1" id="pone.0000877-vanHolde1"></a>van Holde K, Zlatanova J (1995) Chromatin higher order structure: chasing a mirage? J Biol Chem  270: 8373–8376.  <ul class="find" data-citedArticleID="990722" data-doi="10.1074/jbc.270.15.8373"><li><a href="http://dx.doi.org/10.1074/jbc.270.15.8373" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+higher+order+structure%3A+chasing+a+mirage%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+higher+order+structure%3A+chasing+a+mirage%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000877-BenHaim1" id="pone.0000877-BenHaim1"></a>Ben-Haim E, Lesne A, Victor JM (2001) Chromatin: a tunable spring at work inside chromosomes. Phys Rev E Stat Nonlin Soft Matter Phys  64: 051921.  <ul class="find" data-citedArticleID="990656" data-doi="10.1103/physreve.64.051921"><li><a href="http://dx.doi.org/10.1103/physreve.64.051921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin%3A+a+tunable+spring+at+work+inside+chromosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin%3A+a+tunable+spring+at+work+inside+chromosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000877-Barbi1" id="pone.0000877-Barbi1"></a>Barbi M, Mozziconacci J, Victor JM (2005) How the chromatin fiber deals with topological constraints. Phys Rev E Stat Nonlin Soft Matter Phys  71: 031910.  <ul class="find" data-citedArticleID="990648" data-doi="10.1103/physreve.71.031910"><li><a href="http://dx.doi.org/10.1103/physreve.71.031910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+the+chromatin+fiber+deals+with+topological+constraints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+the+chromatin+fiber+deals+with+topological+constraints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000877-Tatchell1" id="pone.0000877-Tatchell1"></a>Tatchell K, Van Holde KE (1978) Compact oligomers and nucleosome phasing. Proc Natl Acad Sci U S A  75: 3583–3587.  <ul class="find" data-citedArticleID="990706" data-doi="10.1073/pnas.75.8.3583"><li><a href="http://dx.doi.org/10.1073/pnas.75.8.3583" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compact+oligomers+and+nucleosome+phasing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compact+oligomers+and+nucleosome+phasing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000877-Dubochet1" id="pone.0000877-Dubochet1"></a>Dubochet J, Noll M (1978) Nucleosome arcs and helices. Science  202: 280–286.  <ul class="find" data-citedArticleID="990674" data-doi="10.1126/science.694532"><li><a href="http://dx.doi.org/10.1126/science.694532" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleosome+arcs+and+helices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleosome+arcs+and+helices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000877-Mangenot1" id="pone.0000877-Mangenot1"></a>Mangenot S, Leforestier A, Durand D, Livolant F (2003) Phase diagram of nucleosome core particles. J Mol Biol  333: 907–916.  <ul class="find" data-citedArticleID="990692" data-doi="10.1016/j.jmb.2003.09.015"><li><a href="http://dx.doi.org/10.1016/j.jmb.2003.09.015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phase+diagram+of+nucleosome+core+particles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phase+diagram+of+nucleosome+core+particles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000877-Bartolome1" id="pone.0000877-Bartolome1"></a>Bartolome S, Bermudez A, Daban JR (1994) Internal structure of the 30 nm chromatin fiber. J Cell Sci  107 ( Pt 11): 2983–2992.  <ul class="find" data-citedArticleID="990650"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Internal+structure+of+the+30+nm+chromatin+fiber.&amp;auth=&amp;atitle=Internal+structure+of+the+30+nm+chromatin+fiber." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Internal+structure+of+the+30+nm+chromatin+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Internal+structure+of+the+30+nm+chromatin+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000877-Davey1" id="pone.0000877-Davey1"></a>Davey CA, Sargent DF, Luger K, Maeder AW, Richmond TJ (2002) Solvent mediated interactions in the structure of the nucleosome core particle at 1.9 a resolution. J Mol Biol  319: 1097–1113.  <ul class="find" data-citedArticleID="990670" data-doi="10.1016/s0022-2836(02)00386-8"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00386-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solvent+mediated+interactions+in+the+structure+of+the+nucleosome+core+particle+at+1.9+a+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solvent+mediated+interactions+in+the+structure+of+the+nucleosome+core+particle+at+1.9+a+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000877-Widom2" id="pone.0000877-Widom2"></a>Widom J (1992) A relationship between the helical twist of DNA and the ordered positioning of nucleosomes in all eukaryotic cells. Proc Natl Acad Sci U S A  89: 1095–1099.  <ul class="find" data-citedArticleID="990712" data-doi="10.1073/pnas.89.3.1095"><li><a href="http://dx.doi.org/10.1073/pnas.89.3.1095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+relationship+between+the+helical+twist+of+DNA+and+the+ordered+positioning+of+nucleosomes+in+all+eukaryotic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+relationship+between+the+helical+twist+of+DNA+and+the+ordered+positioning+of+nucleosomes+in+all+eukaryotic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000877-Yao1" id="pone.0000877-Yao1"></a>Yao J, Lowary PT, Widom J (1993) Twist constraints on linker DNA in the 30-nm chromatin fiber: implications for nucleosome phasing. Proc Natl Acad Sci U S A  90: 9364–9368.  <ul class="find" data-citedArticleID="990720" data-doi="10.1073/pnas.90.20.9364"><li><a href="http://dx.doi.org/10.1073/pnas.90.20.9364" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Twist+constraints+on+linker+DNA+in+the+30-nm+chromatin+fiber%3A+implications+for+nucleosome+phasing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Twist+constraints+on+linker+DNA+in+the+30-nm+chromatin+fiber%3A+implications+for+nucleosome+phasing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000877-Duggan1" id="pone.0000877-Duggan1"></a>Duggan MM, Thomas JO (2000) Two DNA-binding sites on the globular domain of histone H5 are required for binding to both bulk and 5 S reconstituted nucleosomes. J Mol Biol  304: 21–33.  <ul class="find" data-citedArticleID="990676" data-doi="10.1006/jmbi.2000.4205"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+DNA-binding+sites+on+the+globular+domain+of+histone+H5+are+required+for+binding+to+both+bulk+and+5+S+reconstituted+nucleosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+DNA-binding+sites+on+the+globular+domain+of+histone+H5+are+required+for+binding+to+both+bulk+and+5+S+reconstituted+nucleosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000877-Mirzabekov1" id="pone.0000877-Mirzabekov1"></a>Mirzabekov AD, Pruss DV, Ebralidse KK (1990) Chromatin superstructure-dependent crosslinking with DNA of the histone H5 residues Thr1, His25 and His62. J Mol Biol  211: 479–491.  <ul class="find" data-citedArticleID="990694" data-doi="10.1016/0022-2836(90)90366-t"><li><a href="http://dx.doi.org/10.1016/0022-2836(90)90366-t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+superstructure-dependent+crosslinking+with+DNA+of+the+histone+H5+residues+Thr1%2C+His25+and+His62." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+superstructure-dependent+crosslinking+with+DNA+of+the+histone+H5+residues+Thr1%2C+His25+and+His62.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000877-Brown1" id="pone.0000877-Brown1"></a>Brown DT, Izard T, Misteli T (2006) Mapping the interaction surface of linker histone H1(0) with the nucleosome of native chromatin in vivo. Nat Struct Mol Biol  13: 250–255.  <ul class="find" data-citedArticleID="990662" data-doi="10.1038/nsmb1050"><li><a href="http://dx.doi.org/10.1038/nsmb1050" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+the+interaction+surface+of+linker+histone+H1%280%29+with+the+nucleosome+of+native+chromatin+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+the+interaction+surface+of+linker+histone+H1%280%29+with+the+nucleosome+of+native+chromatin+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000877-Fan1" id="pone.0000877-Fan1"></a>Fan L, Roberts VA (2006) Complex of linker histone H5 with the nucleosome and its implications for chromatin packing. Proc Natl Acad Sci U S A  103: 8384–8389.  <ul class="find" data-citedArticleID="990678" data-doi="10.1073/pnas.0508951103"><li><a href="http://dx.doi.org/10.1073/pnas.0508951103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complex+of+linker+histone+H5+with+the+nucleosome+and+its+implications+for+chromatin+packing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complex+of+linker+histone+H5+with+the+nucleosome+and+its+implications+for+chromatin+packing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000877-Ramakrishnan1" id="pone.0000877-Ramakrishnan1"></a>Ramakrishnan V, Finch JT, Graziano V, Lee PL, Sweet RM (1993) Crystal structure of globular domain of histone H5 and its implications for nucleosome binding. Nature  362: 219–223.  <ul class="find" data-citedArticleID="990698" data-doi="10.1038/362219a0"><li><a href="http://dx.doi.org/10.1038/362219a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+globular+domain+of+histone+H5+and+its+implications+for+nucleosome+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+globular+domain+of+histone+H5+and+its+implications+for+nucleosome+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000877-Allan1" id="pone.0000877-Allan1"></a>Allan J, Hartman PG, Crane-Robinson C, Aviles FX (1980) The structure of histone H1 and its location in chromatin. Nature  288: 675–679.  <ul class="find" data-citedArticleID="990644" data-doi="10.1038/288675a0"><li><a href="http://dx.doi.org/10.1038/288675a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+of+histone+H1+and+its+location+in+chromatin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+of+histone+H1+and+its+location+in+chromatin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000877-Hamiche1" id="pone.0000877-Hamiche1"></a>Hamiche A, Schultz P, Ramakrishnan V, Oudet P, Prunell A (1996) Linker histone-dependent DNA structure in linear mononucleosomes. J Mol Biol  257: 30–42.  <ul class="find" data-citedArticleID="990680" data-doi="10.1006/jmbi.1996.0144"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linker+histone-dependent+DNA+structure+in+linear+mononucleosomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linker+histone-dependent+DNA+structure+in+linear+mononucleosomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000877-Sivolob1" id="pone.0000877-Sivolob1"></a>Sivolob A, Prunell A (2003) Linker histone-dependent organization and dynamics of nucleosome entry/exit DNAs. J Mol Biol  331: 1025–1040.  <ul class="find" data-citedArticleID="990704" data-doi="10.1016/s0022-2836(03)00831-3"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00831-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linker+histone-dependent+organization+and+dynamics+of+nucleosome+entry%2Fexit+DNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linker+histone-dependent+organization+and+dynamics+of+nucleosome+entry%2Fexit+DNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000877-Leforestier1" id="pone.0000877-Leforestier1"></a>Leforestier A, Livolant F (1993) Supramolecular ordering of DNA in the cholesteric liquid crystalline phase: an ultrastructural study. Biophys J  65: 56–72.  <ul class="find" data-citedArticleID="990688" data-doi="10.1016/s0006-3495(93)81063-4"><li><a href="http://dx.doi.org/10.1016/s0006-3495(93)81063-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Supramolecular+ordering+of+DNA+in+the+cholesteric+liquid+crystalline+phase%3A+an+ultrastructural+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Supramolecular+ordering+of+DNA+in+the+cholesteric+liquid+crystalline+phase%3A+an+ultrastructural+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000877-Hud1" id="pone.0000877-Hud1"></a>Hud NV, Downing KH (2001) Cryoelectron microscopy of lambda phage DNA condensates in vitreous ice: the fine structure of DNA toroids. Proc Natl Acad Sci U S A  98: 14925–14930.  <ul class="find" data-citedArticleID="990682" data-doi="10.1073/pnas.261560398"><li><a href="http://dx.doi.org/10.1073/pnas.261560398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cryoelectron+microscopy+of+lambda+phage+DNA+condensates+in+vitreous+ice%3A+the+fine+structure+of+DNA+toroids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cryoelectron+microscopy+of+lambda+phage+DNA+condensates+in+vitreous+ice%3A+the+fine+structure+of+DNA+toroids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000877-Lee1" id="pone.0000877-Lee1"></a>Lee KC, Borukhov I, Gelbart WM, Liu AJ, Stevens MJ (2004) Effect of mono- and multivalent salts on angle-dependent attractions between charged rods. Phys Rev Lett  93: 128101.  <ul class="find" data-citedArticleID="990686" data-doi="10.1103/physrevlett.93.128101"><li><a href="http://dx.doi.org/10.1103/physrevlett.93.128101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+mono-+and+multivalent+salts+on+angle-dependent+attractions+between+charged+rods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+mono-+and+multivalent+salts+on+angle-dependent+attractions+between+charged+rods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000877-Mozziconacci1" id="pone.0000877-Mozziconacci1"></a>Mozziconacci J, Victor JM (2003) Nucleosome gaping supports a functional structure for the 30 nm chromatin fiber. J Struct Biol  143: 72–76.  <ul class="find" data-citedArticleID="990696" data-doi="10.1016/s1047-8477(03)00102-3"><li><a href="http://dx.doi.org/10.1016/s1047-8477(03)00102-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleosome+gaping+supports+a+functional+structure+for+the+30+nm+chromatin+fiber." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleosome+gaping+supports+a+functional+structure+for+the+30+nm+chromatin+fiber.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000877-Arya1" id="pone.0000877-Arya1"></a>Arya G, Schlick T (2006) Role of histone tails in chromatin folding revealed by a mesoscopic oligonucleosome model. Proc Natl Acad Sci U S A  103: 16236–16241.  <ul class="find" data-citedArticleID="990646" data-doi="10.1073/pnas.0604817103"><li><a href="http://dx.doi.org/10.1073/pnas.0604817103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+histone+tails+in+chromatin+folding+revealed+by+a+mesoscopic+oligonucleosome+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+histone+tails+in+chromatin+folding+revealed+by+a+mesoscopic+oligonucleosome+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000877-Lavery1" id="pone.0000877-Lavery1"></a>Lavery R, Zakrzewska K, Sklenar H (1995) JUMNA (junction minimisation of nucleic acids). Computer Physics Communications  91: 135–158.  <ul class="find" data-citedArticleID="990684" data-doi="10.1016/0010-4655(95)00046-i"><li><a href="http://dx.doi.org/10.1016/0010-4655(95)00046-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=JUMNA+%28junction+minimisation+of+nucleic+acids%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22JUMNA+%28junction+minimisation+of+nucleic+acids%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000877-Boomsma1" id="pone.0000877-Boomsma1"></a>Boomsma W, Hamelryck T (2005) Full cyclic coordinate descent: solving the protein loop closure problem in Calpha space. BMC Bioinformatics  6: 159.  <ul class="find" data-citedArticleID="990660"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Full+cyclic+coordinate+descent%3A+solving+the+protein+loop+closure+problem+in+Calpha+space.&amp;auth=&amp;atitle=Full+cyclic+coordinate+descent%3A+solving+the+protein+loop+closure+problem+in+Calpha+space." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Full+cyclic+coordinate+descent%3A+solving+the+protein+loop+closure+problem+in+Calpha+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Full+cyclic+coordinate+descent%3A+solving+the+protein+loop+closure+problem+in+Calpha+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000877-Buss1" id="pone.0000877-Buss1"></a>Buss SR, Kim J-S (2005) Selectively Damped Least Squares for Inverse Kinematics. J Graph Tools  10: 37–49.  <ul class="find" data-citedArticleID="990664" data-doi="10.1080/2151237x.2005.10129202"><li><a href="http://dx.doi.org/10.1080/2151237x.2005.10129202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selectively+Damped+Least+Squares+for+Inverse+Kinematics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selectively+Damped+Least+Squares+for+Inverse+Kinematics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000877-Berendsen1" id="pone.0000877-Berendsen1"></a>Berendsen HJC, van der Spoel D, van Drunen R (1995) GROMACS: a message-passing parallel molecular dynamics implementation. Computer Physics Communications  91: 43–56.  <ul class="find" data-citedArticleID="990658" data-doi="10.1016/0010-4655(95)00042-e"><li><a href="http://dx.doi.org/10.1016/0010-4655(95)00042-e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GROMACS%3A+a+message-passing+parallel+molecular+dynamics+implementation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GROMACS%3A+a+message-passing+parallel+molecular+dynamics+implementation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.XML" value="87737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.PDF" value="481205"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g001.PNG_L" value="216532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g001.PNG_M" value="43068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g001.PNG_S" value="8407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g001.TIF" value="338044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g001.PNG_I" value="19814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g002.PNG_L" value="447069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g002.PNG_M" value="111161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g002.PNG_S" value="10113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g002.TIF" value="721724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g002.PNG_I" value="42510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.t001.PNG_L" value="80405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.t001.PNG_M" value="66550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.t001.PNG_S" value="13830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.t001.TIF" value="575604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.t001.PNG_I" value="44699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g003.PNG_L" value="2791600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g003.PNG_M" value="442025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g003.PNG_S" value="20177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g003.TIF" value="3692286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g003.PNG_I" value="147316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g004.PNG_L" value="3778790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g004.PNG_M" value="431515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g004.PNG_S" value="16288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g004.TIF" value="5001658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g004.PNG_I" value="130560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g005.PNG_L" value="405080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g005.PNG_M" value="103821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g005.PNG_S" value="9380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g005.TIF" value="673210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g005.PNG_I" value="35814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e001.PNG" value="9873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e001.TIF" value="25572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e002.PNG" value="11184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e002.TIF" value="26932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e003.PNG" value="6213"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e003.TIF" value="22204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e004.PNG" value="6191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e004.TIF" value="22128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e005.PNG" value="7662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.e005.TIF" value="23548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g006.PNG_L" value="997750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g006.PNG_M" value="152198"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g006.PNG_S" value="9538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g006.TIF" value="1308648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000877.g006.PNG_I" value="51925"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000877&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000877" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000877&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000877&volume=&issue=&title=An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length&author_name=Hua%20Wong%2C%20Jean-Marc%20Victor%2C%20Julien%20Mozziconacci&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000877" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877&amp;t=An%20All-Atom%20Model%20of%20the%20Chromatin%20Fiber%20Containing%20Linker%20Histones%20Reveals%20a%20Versatile%20Structure%20Tuned%20by%20the%20Nucleosomal%20Repeat%20Length" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877&title=An%20All-Atom%20Model%20of%20the%20Chromatin%20Fiber%20Containing%20Linker%20Histones%20Reveals%20a%20Versatile%20Structure%20Tuned%20by%20the%20Nucleosomal%20Repeat%20Length&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877&amp;title=An%20All-Atom%20Model%20of%20the%20Chromatin%20Fiber%20Containing%20Linker%20Histones%20Reveals%20a%20Versatile%20Structure%20Tuned%20by%20the%20Nucleosomal%20Repeat%20Length" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000877&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'An All-Atom Model of the Chromatin Fiber Containing Linker Histones Reveals a Versatile Structure Tuned by the Nucleosomal Repeat Length';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000877';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20An%20All-Atom%20Model%20of%20the%20Chromatin%20Fiber%20Containing%20Linker%20Histones%20Reveals%20a%20Versatile%20Structure%20Tuned%20by%20the%20Nucleosomal%20Repeat%20Length http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000877" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000877" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bending%22" title="Search for articles in the subject area:'Bending'"><div class="flagText">Bending</div></a>
              <div data-categoryid="22873" data-articleid="24592"
                   data-categoryname="Bending"
                   class="flagImage" title="Flag 'Bending' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromatin%22" title="Search for articles in the subject area:'Chromatin'"><div class="flagText">Chromatin</div></a>
              <div data-categoryid="31933" data-articleid="24592"
                   data-categoryname="Chromatin"
                   class="flagImage" title="Flag 'Chromatin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24592"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24592"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+structure%22" title="Search for articles in the subject area:'DNA structure'"><div class="flagText">DNA structure</div></a>
              <div data-categoryid="34399" data-articleid="24592"
                   data-categoryname="DNA structure"
                   class="flagImage" title="Flag 'DNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="24592"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Kinematics%22" title="Search for articles in the subject area:'Kinematics'"><div class="flagText">Kinematics</div></a>
              <div data-categoryid="20035" data-articleid="24592"
                   data-categoryname="Kinematics"
                   class="flagImage" title="Flag 'Kinematics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleosomes%22" title="Search for articles in the subject area:'Nucleosomes'"><div class="flagText">Nucleosomes</div></a>
              <div data-categoryid="34075" data-articleid="24592"
                   data-categoryname="Nucleosomes"
                   class="flagImage" title="Flag 'Nucleosomes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=420'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5268'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7693&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=17337">Graphical modeling of the Beta globin gene cluster</a><br>Posted by Pipo</p>
      <p><a href="/annotation/listThread.action?root=9385">Comparison between our structure and Robinson et al. structure </a><br>Posted by Pipo</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
