

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001882"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001882" />

    <meta name="citation_title" content="An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation"/>
    <meta itemprop="name" content="An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation"/>

      <meta name="citation_author" content="Maria E. Figueroa"/>
            <meta name="citation_author_institution" content="Department of Developmental and Molecular Biology, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Mark Reimers"/>
            <meta name="citation_author_institution" content="Department of Biostatistics, Virginia Commonwealth University, Richmond, Virginia, United States of America"/>
      <meta name="citation_author" content="Reid F. Thompson"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Kenny Ye"/>
            <meta name="citation_author_institution" content="Department of Epidemiology and Population Health, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Yushan Li"/>
            <meta name="citation_author_institution" content="Department of Developmental and Molecular Biology, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Rebecca R. Selzer"/>
            <meta name="citation_author_institution" content="Roche NimbleGen, Inc. Madison, Wisconsin, United States of America"/>
      <meta name="citation_author" content="Jakob Fridriksson"/>
            <meta name="citation_author_institution" content="Roche NimbleGen, Inc. Madison, Wisconsin, United States of America"/>
      <meta name="citation_author" content="Elisabeth Paietta"/>
            <meta name="citation_author_institution" content="Our Lady of Mercy Comprehensive Cancer Center, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Peter Wiernik"/>
            <meta name="citation_author_institution" content="Our Lady of Mercy Comprehensive Cancer Center, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Roland D. Green"/>
            <meta name="citation_author_institution" content="Roche NimbleGen, Inc. Madison, Wisconsin, United States of America"/>
      <meta name="citation_author" content="John M. Greally"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
            <meta name="citation_author_institution" content="Department of Medicine, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>
      <meta name="citation_author" content="Ari Melnick"/>
            <meta name="citation_author_institution" content="Department of Developmental and Molecular Biology, Albert Einstein College of Medicine, Bronx, New York, United States of America"/>

    <meta name="citation_date" content="2008/3/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001882.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1882"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Translating the histone code.; citation_author=T Jenuwein; citation_author=CD Allis; citation_journal_title=Science; citation_volume=293; citation_number=1; citation_pages=1074-1080; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA methylation patterns and epigenetic memory.; citation_author=A Bird; citation_journal_title=Genes Dev; citation_volume=16; citation_number=2; citation_pages=6-21; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mapping global histone acetylation patterns to gene expression.; citation_author=SK Kurdistani; citation_author=S Tavazoie; citation_author=M Grunstein; citation_journal_title=Cell; citation_volume=117; citation_number=3; citation_pages=721-733; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic maps and comparative analysis of histone modifications in human and mouse.; citation_author=BE Bernstein; citation_author=M Kamal; citation_author=K Lindblad-Toh; citation_author=S Bekiranov; citation_author=DK Bailey; citation_journal_title=Cell; citation_volume=120; citation_number=4; citation_pages=169-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene silencing in cancer in association with promoter hypermethylation.; citation_author=JG Herman; citation_author=SB Baylin; citation_journal_title=N Engl J Med; citation_volume=349; citation_number=5; citation_pages=2042-2054; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=DNA methylation immediately adjacent to active histone marking does not silence transcription.; citation_author=AB Brinkman; citation_author=SW Pennings; citation_author=GG Braliou; citation_author=LE Rietveld; citation_author=HG Stunnenberg; citation_journal_title=Nucleic Acids Res; citation_number=6; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=hTERT is expressed in cancer cell lines despite promoter DNA methylation by preservation of unmethylated DNA and active chromatin around the transcription start site.; citation_author=RL Zinn; citation_author=K Pruitt; citation_author=S Eguchi; citation_author=SB Baylin; citation_author=JG Herman; citation_journal_title=Cancer Res; citation_volume=67; citation_number=7; citation_pages=194-201; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Methylation of histone H3 Lys 4 in coding regions of active genes.; citation_author=BE Bernstein; citation_author=EL Humphrey; citation_author=RL Erlich; citation_author=R Schneider; citation_author=P Bouman; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=8; citation_pages=8695-8700; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The fundamental role of epigenetic events in cancer.; citation_author=PA Jones; citation_author=SB Baylin; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=9; citation_pages=415-428; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Epigenetic gene silencing in cancer - a mechanism for early oncogenic pathway addiction?; citation_author=SB Baylin; citation_author=JE Ohm; citation_journal_title=Nat Rev Cancer; citation_volume=6; citation_number=10; citation_pages=107-116; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Concurrent DNA hypermethylation of multiple genes in acute myeloid leukemia.; citation_author=JR Melki; citation_author=PC Vincent; citation_author=SJ Clark; citation_journal_title=Cancer Res; citation_volume=59; citation_number=11; citation_pages=3730-3740; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Aberrant methylation and impaired expression of the p15(INK4b) cell cycle regulatory gene in chronic myelomonocytic leukemia (CMML).; citation_author=M Tessema; citation_author=F Langer; citation_author=J Dingemann; citation_author=A Ganser; citation_author=H Kreipe; citation_journal_title=Leukemia; citation_volume=17; citation_number=12; citation_pages=910-918; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inactivation of the DNA repair gene O6-methylguanine-DNA methyltransferase by promoter hypermethylation is a common event in primary human neoplasia.; citation_author=M Esteller; citation_author=SR Hamilton; citation_author=PC Burger; citation_author=SB Baylin; citation_author=JG Herman; citation_journal_title=Cancer Res; citation_volume=59; citation_number=13; citation_pages=793-797; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Incidence and functional consequences of hMLH1 promoter hypermethylation in colorectal carcinoma.; citation_author=JG Herman; citation_author=A Umar; citation_author=K Polyak; citation_author=JR Graff; citation_author=N Ahuja; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=14; citation_pages=6870-6875; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer.; citation_author=EE Cameron; citation_author=KE Bachman; citation_author=S Myohanen; citation_author=JG Herman; citation_author=SB Baylin; citation_journal_title=Nat Genet; citation_volume=21; citation_number=15; citation_pages=103-107; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Silenced tumor suppressor genes reactivated by DNA demethylation do not return to a fully euchromatic chromatin state.; citation_author=KM McGarvey; citation_author=JA Fahrner; citation_author=E Greene; citation_author=J Martens; citation_author=T Jenuwein; citation_journal_title=Cancer Res; citation_volume=66; citation_number=16; citation_pages=3541-3549; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular classification of cancer: class discovery and class prediction by gene expression monitoring.; citation_author=TR Golub; citation_author=DK Slonim; citation_author=P Tamayo; citation_author=C Huard; citation_author=M Gaasenbeek; citation_journal_title=Science; citation_volume=286; citation_number=17; citation_pages=531-537; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=MLL translocations specify a distinct gene expression profile that distinguishes a unique leukemia.; citation_author=SA Armstrong; citation_author=JE Staunton; citation_author=LB Silverman; citation_author=R Pieters; citation_author=ML den Boer; citation_journal_title=Nat Genet; citation_volume=30; citation_number=18; citation_pages=41-47; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Prognostically useful gene-expression profiles in acute myeloid leukemia.; citation_author=PJ Valk; citation_author=RG Verhaak; citation_author=MA Beijen; citation_author=CA Erpelinck; citation_author=S Barjesteh van Waalwijk van Doorn-Khosrovani; citation_journal_title=N Engl J Med; citation_volume=350; citation_number=19; citation_pages=1617-1628; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma.; citation_author=A Rosenwald; citation_author=G Wright; citation_author=WC Chan; citation_author=JM Connors; citation_author=E Campo; citation_journal_title=N Engl J Med; citation_volume=346; citation_number=20; citation_pages=1937-1947; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Prognostic significance of O6-methylguanine-DNA methyltransferase determined by promoter hypermethylation and immunohistochemical expression in anaplastic gliomas.; citation_author=M Berell; citation_author=A Tortosa; citation_author=E Verger; citation_author=J Gil; citation_author=N Viñolas; citation_journal_title=Clin Cancer Res; citation_volume=11; citation_number=21; citation_pages=5167-5174; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=CpG island hypermethylation of the DNA repair enzyme methyltransferase predicts response to temozolomide in primary gliomas.; citation_author=MF Paz; citation_author=R Yaya-Tur; citation_author=I Rojas-Marcos; citation_author=G Reynes; citation_author=M Pollan; citation_journal_title=Clin Cancer Res; citation_volume=10; citation_number=22; citation_pages=4933-4938; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Comparative isoschizomer profiling of cytosine methylation: the HELP assay.; citation_author=B Khulan; citation_author=R Thompson; citation_author=K Ye; citation_author=Mj Fazzari; citation_author=M Suzuki; citation_journal_title=Genome Research; citation_volume=16; citation_number=23; citation_pages=1046-1055; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=DNA methylation changes in leukaemia.; citation_author=JR Melki; citation_author=SJ Clark; citation_journal_title=Semin Cancer Biol; citation_volume=12; citation_number=24; citation_pages=347-357; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genome-wide profiling of CpG methylation identifies novel targets of aberrant hypermethylation in myeloid leukemia.; citation_author=C Gebhard; citation_author=L Schwarzfischer; citation_author=TH Pham; citation_author=E Schilling; citation_author=M Klug; citation_journal_title=Cancer Res; citation_volume=66; citation_number=25; citation_pages=6118-6128; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Large-scale CpG methylation analysis identifies novel candidate genes and reveals methylation hotspots in acute lymphoblastic leukemia.; citation_author=KH Taylor; citation_author=KE Pena-Hernandez; citation_author=JW Davis; citation_author=GL Arthur; citation_author=DJ Duff; citation_journal_title=Cancer Res; citation_volume=67; citation_number=26; citation_pages=2617-2625; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Triple analysis of the cancer epigenome: an integrated microarray system for assessing gene expression, DNA methylation, and histone acetylation.; citation_author=H Shi; citation_author=SH Wei; citation_author=YW Leu; citation_author=F Rahmatpanah; citation_author=JC Liu; citation_journal_title=Cancer Res; citation_volume=63; citation_number=27; citation_pages=2164-2171; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Diverse histone modifications on histone 3 lysine 9 and their relation to DNA methylation in specifying gene silencing.; citation_author=J Wu; citation_author=SH Wang; citation_author=D Potter; citation_author=JC Liu; citation_author=LT Smith; citation_journal_title=BMC Genomics; citation_volume=8; citation_number=28; citation_pages=131; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative assessment of gene targeting in vitro and in vivo by the pancreatic transcription factor, Pdx1. Importance of chromatin structure in directing promoter binding.; citation_author=SK Chakrabarti; citation_author=JC James; citation_author=RG Mirmira; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=29; citation_pages=13286-13293; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High-throughput screening of chromatin immunoprecipitates using CpG-island microarrays.; citation_author=MJ Oberley; citation_author=J Tsao; citation_author=P Yau; citation_author=PJ Farnham; citation_journal_title=Methods Enzymol; citation_volume=376; citation_number=30; citation_pages=315-334; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative high-throughput analysis of DNA methylation patterns by base-specific cleavage and mass spectrometry.; citation_author=M Ehrich; citation_author=MR Nelson; citation_author=P Stanssens; citation_author=M Zabeau; citation_author=T Liloglou; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=31; citation_pages=15785-15790; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method.; citation_author=KJ Livak; citation_author=TD Schmittgen; citation_journal_title=Methods; citation_volume=25; citation_number=32; citation_pages=402-408; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Analysis of chromosome breakpoints in neuroblastoma at sub-kilobase resolution using fine-tiling oligonucleotide array CGH.; citation_author=RR Selzer; citation_author=TA Richmond; citation_author=NJ Pofahl; citation_author=RD Green; citation_author=PS Eis; citation_journal_title=Genes Chromosomes Cancer; citation_volume=44; citation_number=33; citation_pages=305-319; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Quality assessment of microarrays: visualization of spatial artifacts and quantitation of regional biases.; citation_author=M Reimers; citation_author=JN Weinstein; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=34; citation_pages=166; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns.; citation_author=MB Eisen; citation_author=PT Spellman; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=35; citation_pages=14863-14868; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Correspondence analysis applied to microarray data.; citation_author=K Fellenberg; citation_author=NC Hauser; citation_author=B Brors; citation_author=A Neutzner; citation_author=JD Hoheisel; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=36; citation_pages=10781-10786; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Interactive exploration of microarray gene expression patterns in a reduced dimensional space.; citation_author=J Misra; citation_author=W Schmitt; citation_author=D Hwang; citation_author=LL Hsiao; citation_author=S Gullans; citation_journal_title=Genome Res; citation_volume=12; citation_number=37; citation_pages=1112-1120; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing. 2.5.1 ed.; citation_author=RDC Team; citation_number=38; citation_date=2007; citation_publisher=R Foundation for Statistical Computing; " />
      <meta name="citation_reference" content="citation_title=MADE4: an R package for multivariate analysis of gene expression data.; citation_author=AC Culhane; citation_author=J Thioulouse; citation_author=G Perriere; citation_author=DG Higgins; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=39; citation_pages=2789-2790; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The human genome browser at UCSC.; citation_author=WJ Kent; citation_author=CW Sugnet; citation_author=TS Furey; citation_author=KM Roskin; citation_author=TH Pringle; citation_journal_title=Genome Res; citation_volume=12; citation_number=40; citation_pages=996-1006; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation"/>
    <meta name="twitter:description" content="The molecular heterogeneity of acute leukemias and other tumors constitutes a major obstacle towards understanding disease pathogenesis and developing new targeted-therapies. Aberrant gene regulation is a hallmark of cancer and plays a central role in determining tumor phenotype. We predicted that integration of different genome-wide epigenetic regulatory marks along with gene expression levels would provide greater power in capturing biological differences between leukemia subtypes. Gene expression, cytosine methylation and histone H3 lysine 9 (H3K9) acetylation were measured using high-density oligonucleotide microarrays in primary human acute myeloid leukemia (AML) and acute lymphocytic leukemia (ALL) specimens. We found that DNA methylation and H3K9 acetylation distinguished these leukemias of distinct cell lineage, as expected, but that an integrative analysis combining the information from each platform revealed hundreds of additional differentially expressed genes that were missed by gene expression arrays alone. This integrated analysis also enhanced the detection and statistical significance of biological pathways dysregulated in AML and ALL. Integrative epigenomic studies are thus feasible using clinical samples and provide superior detection of aberrant transcriptional programming than single-platform microarray studies."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001882.g007"/>

  <meta property="og:title" content="An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7341'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8027'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1079&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001882">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001882" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001882&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria E. Figueroa
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Maria E. Figueroa, Mark Reimers</p>

              
              

                <p>Affiliation:
                  Department of Developmental and Molecular Biology, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Reimers
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Maria E. Figueroa, Mark Reimers</p>

              
              

                <p>Affiliation:
                  Department of Biostatistics, Virginia Commonwealth University, Richmond, Virginia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Reid F. Thompson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Genetics, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenny Ye, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Epidemiology and Population Health, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yushan Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Developmental and Molecular Biology, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rebecca R. Selzer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Roche NimbleGen, Inc. Madison, Wisconsin, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jakob Fridriksson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Roche NimbleGen, Inc. Madison, Wisconsin, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Elisabeth Paietta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Our Lady of Mercy Comprehensive Cancer Center, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Wiernik, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Our Lady of Mercy Comprehensive Cancer Center, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Roland D. Green, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Roche NimbleGen, Inc. Madison, Wisconsin, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John M. Greally
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:jgreally@aecom.yu.edu">jgreally@aecom.yu.edu</a> (JG); <a href="mailto:amelnick@aecom.yu.edu">amelnick@aecom.yu.edu</a> (AM)</p>

                <p>Affiliations:
                  Department of Molecular Genetics, Albert Einstein College of Medicine, Bronx, New York, United States of America, 
                  Department of Medicine, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ari Melnick
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:jgreally@aecom.yu.edu">jgreally@aecom.yu.edu</a> (JG); <a href="mailto:amelnick@aecom.yu.edu">amelnick@aecom.yu.edu</a> (AM)</p>

                <p>Affiliation:
                  Department of Developmental and Molecular Biology, Albert Einstein College of Medicine, Bronx, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 26, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001882</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001882-t001" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g001" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g002" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g003" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g004" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g005" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g006" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-g007" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-t002" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001882-t003" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001882">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001882" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The molecular heterogeneity of acute leukemias and other tumors constitutes a major obstacle towards understanding disease pathogenesis and developing new targeted-therapies. Aberrant gene regulation is a hallmark of cancer and plays a central role in determining tumor phenotype. We predicted that integration of different genome-wide epigenetic regulatory marks along with gene expression levels would provide greater power in capturing biological differences between leukemia subtypes. Gene expression, cytosine methylation and histone H3 lysine 9 (H3K9) acetylation were measured using high-density oligonucleotide microarrays in primary human acute myeloid leukemia (AML) and acute lymphocytic leukemia (ALL) specimens. We found that DNA methylation and H3K9 acetylation distinguished these leukemias of distinct cell lineage, as expected, but that an integrative analysis combining the information from each platform revealed hundreds of additional differentially expressed genes that were missed by gene expression arrays alone. This integrated analysis also enhanced the detection and statistical significance of biological pathways dysregulated in AML and ALL. Integrative epigenomic studies are thus feasible using clinical samples and provide superior detection of aberrant transcriptional programming than single-platform microarray studies.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Figueroa ME, Reimers M, Thompson RF, Ye K, Li Y, et al.  (2008) An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation. PLoS ONE 3(3):
          e1882.
            doi:10.1371/journal.pone.0001882</p><p><strong>Editor: </strong>Eshel Ben-Jacob, Tel Aviv University, Israel</p><p><strong>Received:</strong> January 24, 2008; <strong>Accepted:</strong> February 19, 2008; <strong>Published:</strong> March 26, 2008</p><p><strong>Copyright:</strong> © 2008 Figueroa et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Maria E. Figueroa is supported by an ASH Fellow Scholar Award. Ari M. Melnick is supported by NCI R01 CA104348, the Chemotherapy Foundation, the Sam Waxman Cancer Research Foundation, and the G&amp;P Foundation and is a Leukemia and Lymphoma Society Scholar. John M. Greally is supported by a grant from the National Institutes of Health (NIH) (R01 HD044078). Reid F. Thompson is supported by NIH MSTP Training Grant GM007288.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Regulation of gene expression involves multi-layered mechanisms in which epigenetic modifications such as DNA methylation and histone tail modifications play a major role<a href="#pone.0001882-Jenuwein1">[1]</a>, <a href="#pone.0001882-Bird1">[2]</a>. Post-translational modifications of histones at specific residues help to determine chromatin structure and therefore accessibility to gene promoters and regulatory regions. Amongst these marks, acetylation of lysine 9 on histone H3 (H3K9 acetylation) has been linked to gene activation and active transcription<a href="#pone.0001882-Kurdistani1">[3]</a>, <a href="#pone.0001882-Bernstein1">[4]</a>. Cytosine methylation at promoter regions, on the other hand, is associated with gene silencing<a href="#pone.0001882-Herman1">[5]</a>. Epigenetic regulation of gene expression has additional complexities; not only is the presence of specific epigenetic marks important but their localization and density also seem to play a crucial role<a href="#pone.0001882-Brinkman1">[6]</a>–<a href="#pone.0001882-Bernstein2">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Disruption of epigenetic regulation during malignant transformation can profoundly alter a cellular phenotype, resulting in aberrant cellular proliferation and survival. Epigenetic dysregulation is currently recognized as one of the hallmarks of cancer <a href="#pone.0001882-Jones1">[9]</a>, <a href="#pone.0001882-Baylin1">[10]</a>. DNA methylation at promoter regions of key negative cell cycle regulators and DNA repair genes leads to their abnormal epigenetic silencing in many neoplasms<a href="#pone.0001882-Herman1">[5]</a>, <a href="#pone.0001882-Melki1">[11]</a>–<a href="#pone.0001882-Herman2">[14]</a>. However, it is not clear whether this aberrant DNA methylation pattern is sufficient to determine gene silencing, or whether it is in fact part of a more complex process involving chromatin remodeling factors and changes in histone modifications<a href="#pone.0001882-Cameron1">[15]</a>, <a href="#pone.0001882-McGarvey1">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Gene expression profiling studies have been performed with the aim of dissecting the molecular subtypes of several neoplasms, in an effort to predict accurately tumor behavior and to identify important oncogenic genes and biological pathways. These studies have revealed the presence of unique gene expression signatures distinguishing specific subgroups of cancers and have served to improve our understanding of the biology of these diseases (e.g. <a href="#pone.0001882-Golub1">[17]</a>–<a href="#pone.0001882-Rosenwald1">[20]</a>). However, only part of the cellular information is contained at the messenger RNA level, and transcriptional activity is dependent on multiple factors. Among these factors are epigenetic marks, such as cytosine methylation and histone tail modifications, which help to determine and regulate chromatin structure and function including gene expression.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Therefore, while gene expression studies using DNA microarrays have had a great impact in the study of cancer, it is important to recognize that there are limitations associated with this technique. Firstly, gene expression microarrays capture a snapshot of the cell's transcriptome, detecting genes being actively transcribed at the time of RNA extraction, but they do not capture any information concerning the genes' regulatory states and consequently their potential for transcriptional changes in response to stimuli. For example, a locus such as the <em>O</em><sup>6</sup>-methylguanine DNA methyltransferase <em>(MGMT)</em> gene is not prognostically useful in terms of its basal expression state <a href="#pone.0001882-Berell1">[21]</a> but the cytosine methylation status of its promoter provides an excellent indicator of how well gliomas will respond when treated by alkylating agents <a href="#pone.0001882-Paz1">[22]</a>. We hypothesize that biologically significant changes in expression can be missed by expression arrays due to technical limitations, but might be captured by epigenomic studies by identifying genes at which promoter cytosine methylation or H3K9 acetylation differ and testing them with highly-quantitative techniques.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In order to test these hypotheses, we carried out genome-wide studies for DNA methylation and H3K9 acetylation as well as gene expression microarrays in patients with acute myeloid and lymphoblastic leukemia (AML and ALL, respectively). These cell types were chosen so that we could test whether the technical approach we were exploring was feasible in typical clinical samples, using cell types that should be markedly distinctive. We show here that the integration of the information captured by these different platforms results in a more comprehensive detection of differentially regulated genes and an enhancement of the apparent biological relevance of the findings.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Multiplatform epigenomic microarray analysis can be performed on routine leukemia clinical samples</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Bone marrow aspirates from three adult patients with AML and two patients with ALL were enriched for mononuclear cells by Ficoll gradient separation to yield &gt;90% leukemia blast cells (<strong>see </strong><a href="#pone-0001882-t001"><strong>Table 1</strong></a><strong> for patient characteristics</strong>). Frozen aliquots of these samples were thawed for analysis using three different microarray platforms. Unsheared high quality genomic DNA was extracted from 5–10 million cells for genome-wide cytosine methylation analysis using the HELP (<strong><span class="underline">H</span></strong>paII tiny fragment <strong><span class="underline">e</span></strong>nrichment by <strong><span class="underline">l</span></strong>igation mediated <strong><span class="underline">P</span></strong>CR) assay<a href="#pone.0001882-Khulan1">[23]</a>. Ten million cells were cross-linked by exposure to formaldehyde for ChIP-chip with a specific antibody for histone H3 lysine 9 (H3K9) acetylation, a specific antibody for total histone H3 and non-specific rabbit IgG. Total RNA was extracted from 20 million cells using Qiagen's RNeasy mini kit for expression array analysis.</p>
<div class="figure" id="pone-0001882-t001"><div class="img"><a name="pone-0001882-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Patients' characteristics</span></strong></p><span>doi:10.1371/journal.pone.0001882.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The HELP cytosine methylation analysis was performed in triplicate for each sample. Methylation status could be assigned in &gt;86% of probe sets (i.e. 1–14% of probe sets failed to provide a clear signal). The correlation between cytosine methylation profiles in individual replicates from the same sample was r = 0.93–0.99 and correlation amongst different patients was r = 0.78–0.91 (<a href="#pone.0001882.s002"><strong>Figure S1A</strong></a>). After initial quality control one replicate from ALL-2 was excluded from further analysis. ChIP-chip was performed in duplicates and the correlation between replicates was r = 0.92–0.93, while correlation between different patients ranged between r = 0.88–0.92 (<a href="#pone.0001882.s002"><strong>Figure S1B</strong></a>). After initial quality control one replicate from ALL-2 and one from AML-3 were excluded from further analysis. Gene expression analysis was performed in quadruplicate and the best two replicates for each patient were selected based on our initial quality control (see <a href="#s4">methods</a> section). Correlation between replicates was r = 0.95–0.99. Correlation between the five patients in gene expression ranged from r = 0.84–0.96. (<a href="#pone.0001882.s002"><strong>Figure S1C</strong></a>). For each platform, single locus gene validation was performed on a subset of genes respectively by MassArray for methylation (9 genes), by qChIP for H3K9 acetylation (10 genes), or qRT-PCR for gene expression (15 genes), demonstrating that each microarray platform was highly accurate in predicting the actual abundance of mRNA, DNA methylation and histone acetylation of each gene (<a href="#pone.0001882.s003"><strong>Figures S2A, S2B and S2C</strong></a>). Taken together, these data indicate that multi-array analysis is robust and feasible in frozen leukemia patient samples.</p>


<h4>ALL and AML have distinct DNA methylation profiles</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Although aberrant DNA methylation is known to occur in human leukemias,<a href="#pone.0001882-Melki2">[24]</a> it is not known whether (as has been shown for gene expression profiling) different types of leukemia have specific and distinct profiles of promoter DNA methylation. All five leukemia samples presented a bimodal distribution of DNA methylation, with approximately two-thirds of HpaII fragments detected as methylated and one-third as hypomethylated. In order to determine whether AML and ALL samples display distinct DNA methylation signatures, we performed unsupervised clustering of HELP array data. The DNA methylation profiles from leukemia samples were submitted to three different unsupervised clustering algorithms including hierarchical clustering (HC), principal component analysis (PCA) and correspondence analysis (COA). Each of these methods detects global differences between samples as distances and represents them in different ways. HC of HELP profiles separated the cases into two separate nodes containing either the ALLs or the AMLs. (<a href="#pone-0001882-g001"><strong>Figure 1A</strong></a>). Similar to HC, PCA yielded a clear difference between AML and ALL (<a href="#pone.0001882.s004"><strong>Figure S3A</strong></a>). COA revealed an identical node structure as HC and allowed us to generate a list of genes differentially methylated between AML and ALL (<a href="#pone-0001882-g001"><strong>Figure 1B</strong></a>). All three of these clustering methods demonstrated that promoter methylation signatures could accurately segregate the samples into the AML and ALL categories. Since there are 10 distinct ways in which 5 samples can be divided into two groups of 2 and 3 samples each, it was calculated that there was a 1 in 10 chance that this separation of the samples by lineage was due to chance.</p>
<div class="figure" id="pone-0001882-g001"><div class="img"><a name="pone-0001882-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Epigenomic platforms readily classify leukemia samples according to lineage.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Unsupervised clustering of DNA methylation by HELP and H3K9 acetylation ChIP-chip data succeeded in accurately segregating the samples according to their lineage. Panel A: Dendrogram representing the result of hierarchical clustering of leukemia samples using DNA methylation data. The scale on the left represents the correlation distance metric. Panel B: Heatmap of top 150 genes from the first principal component of correspondence analysis, which separated ALL samples from AML. Genes are shown on the rows and samples on the columns, and data were row-centered. Low values corresponding to greater methylation are represented in blue and high values corresponding to less methylation are in red. Panel C: Hierarchical clustering of leukemia samples using H3K9 acetylation ChIP-chip. Panel D: Heatmap of top 100 genes from the first principal component of correspondence analysis, which separated ALL samples from AML. Low values corresponding to less H3K9 acetylation are represented in blue and high values corresponding to greater H3K9 acetylation are in red.</p>
<span>doi:10.1371/journal.pone.0001882.g001</span></div>

<h4>Histone 3 lysine 9 acetylation profiles distinguish ALL from AML cell types</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>In contrast to DNA methylation, acetylation of histone H3 on lysine 9 has been associated with genes that are transcriptionally active<a href="#pone.0001882-Kurdistani1">[3]</a>, <a href="#pone.0001882-Bernstein1">[4]</a>. We predicted that the epigenetic information contained in the H3K9 acetylation status of genes would be informative regarding the biological phenotype of AML and ALL. H3K9 acetyl ChIP-chip performed robustly using patient samples and revealed a bimodal distribution in which approximately one-third of all gene promoters represented displayed H3K9 acetylation. In order to determine whether H3K9 acetylation status could distinguish AML from ALL cells, unsupervised clustering (HC, PCA and COA) was again performed (<a href="#pone-0001882-g001"><strong>Figures 1C &amp; 1D</strong></a><a href="#pone.0001882.s004"><strong> and Figure S3B</strong></a>). All three methods yielded a similar segregation of the cases into two classes consistent with their biological lineages. Taken together, these data indicate that AML and ALL are epigenetically distinct cell types that can be classified through genome-wide analysis of DNA methylation and H3K9 acetylation.</p>


<h4>DNA methylation, histone acetylation and gene expression profiling identify distinct cohorts of genes in AML and ALL cells</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>We next asked whether genes detected as robustly differentially methylated, H3K9 acetylated and expressed by microarray analysis would be overlapping or complementary. In order to identify the genes that most significantly distinguish AML from ALL cells in terms of gene expression, H3K9 acetylation and DNA methylation, we performed a supervised analysis using a T-test, with lineage as the dichotomous variable. For DNA methylation data, a second condition requiring a minimum difference of 1.5 between the two sample means was implemented in order to maximize the capture of biologically significant changes in methylation (<a href="#pone-0001882-g002"><strong>Figure 2A</strong></a>). Analysis was restricted to the 17,180 genes that are represented on all three microarrays platforms used. Using this common denominator, 1,359 genes were identified as differentially expressed between the two types of leukemia at a significance level of p&lt;0.02. Histone H3 lysine 9 acetylation, on the other hand, identified 374 genes at the same level of significance, while 190 genes were classified as differentially methylated (<a href="#pone-0001882-g002"><strong>Figures 2B, 2C and 2D</strong></a>). Overlap between gene expression and H3K9 acetylation signatures included 45 genes, overlap between DNA methylation and gene expression included 13 genes, while DNA methylation and H3K9 acetylation overlapped on 3 genes. All three platforms overlapped on only 1 gene: the metallopeptidase <em>LMLN</em>. These partially overlapping signals probably result from both the distinct biological nature of the parameters being measured by each different platform as well as from unique technical limitations that affect each one. Thus, gene signatures obtained using epigenomic and gene expression profiling can provide complementary sets of genes (<a href="#pone-0001882-g002"><strong>Figure 2E</strong></a>), resulting in a greater set of unique biological features being captured, which would otherwise be missed by any single platform.</p>
<div class="figure" id="pone-0001882-g002"><div class="img"><a name="pone-0001882-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Gene expression, DNA methylation and H3K9 acetylation identify partially overlapping genes signatures.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Supervised analysis of the three datasets using a T-test and leukemic lineage as the dichotomous variable identified partially overlapping gene signatures distinguishing between ALL and AML. Panel A: Plot of differences between sample means (x axis) vs. −log<sub>10</sub> p values (y axis), illustrating the two criteria required for significance in HELP data. Blue dots represent genes with p&lt;0.02 and a difference in sample means&gt;1.5. Panel B: Heatmap of gene signature identified using gene expression arrays. Genes are shown on the rows and samples on the columns, and data were row-centered. Low values of gene expression are represented in blue and high values are in red. Panel C: Heatmap of gene signature identified using H3K9 acetylation arrays, constructed as described for Panel A. Low values corresponding to less H3K9 acetylation are represented in blue and high values corresponding to greater H3K9 acetylation are in red. Panel D: Heatmap of gene signature identified using DNA methylation (HELP) arrays, constructed as described for Panel A. Low values corresponding to greater methylation are represented in blue and high values corresponding to less methylation are in red. Panel E: Venn diagram illustrating overlap between the three gene signatures.</p>
<span>doi:10.1371/journal.pone.0001882.g002</span></div>

<h4>H3K9 acetylation widely correlates with gene expression</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Acetylation of lysine 9 on histone H3 has been associated with actively transcribed genes <a href="#pone.0001882-Kurdistani1">[3]</a>, <a href="#pone.0001882-Bernstein1">[4]</a>, but the degree of association is unknown at the genome-wide level in leukemia patients. In order to define with greater precision the quantitative relation between variation in histone acetylation and gene expression, we studied the correlation between promoter H3K9 acetylation and gene transcript abundance. Since correlations between variables are degraded rapidly by increasing noise levels, we selected genes with a SNR&gt;2.5. The median correlation between H3K9 acetylation and gene expression over 393 genes with a SNR&gt;2.5 in both measures was 0.86 (<a href="#pone-0001882-g003"><strong>Figure 3A</strong></a>). In order to test whether these findings could be simply due to an artifact of the small sample size, we developed a statistical model in which we selected genes that were as much as possible statistically independent and then re-calculated the correlations after several sample-label permutations. This analysis showed that the probability of obtaining the observed correlations by chance was astronomically small (p&lt;1 in 108) <strong>(see supplementary methods).</strong> Therefore, the degrees of H3K9 acetylation and gene transcript level are quantitatively tightly linked in these leukemic cells.</p>
<div class="figure" id="pone-0001882-g003"><div class="img"><a name="pone-0001882-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>H3K9 acetylation correlates with active gene expression and can help rescue genes from within the noise level of gene expression arrays.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Panel A: Smoothed histogram of gene-by-gene correlation between gene expression and H3K9acetyl ChIP-chip for genes displaying a high SNR on both platforms. Median correlation was r = 0.84. Panel B: Comparative fold difference in mRNA levels (y axis) as detected by gene expression arrays and qRT-PCR for five genes that displayed differential H3K9 acetylation but had not been detected as differentially expressed on the microarrays. Panels C and D: Correlation for those same 5 genes between H3K9 acetylation levels (y axis) and gene expression levels (x axis) measured by either gene expression arrays (panel C) or qRT-PCR (panel D). Correlation between H3K9 acetylation and gene expression was significantly higher when the latter was measured by qRT-PCR, indicating that H3K9 acetyl ChIP-chip could predict genuine differences in gene expression levels that were missed by gene expression arrays, probably due to relative compression of the signal on the gene expression arrays as demonstrated in panel B.</p>
<span>doi:10.1371/journal.pone.0001882.g003</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Since microarray measures are inherently corrupted by noise, we hypothesized that the true underlying biological correlations between gene expression and H3K9 acetylation were much greater than captured through a simple correlation of high SNR genes. In other words, that this epigenetic modification could detect differences in gene expression missed by expression arrays. In order to test this hypothesis, we randomly selected a group of five genes (<em>FER, HOXA1, DOCK5, LT4R1</em> and <em>PAX5</em>) among those that presented differential H3K9 acetylation with an SNR&gt;2.5, but that were not detected as differentially expressed by microarray analysis. A more precise measurement of transcript abundance of these genes was performed by qRT-PCR, which revealed that differential expression was significantly greater than had been detected by microarray: the range of fold difference between ALL and AML by gene expression array was 1.0–2.8 and by qRT-PCR 2.3–45.6 (<a href="#pone-0001882-g003"><strong>Figure 3B</strong></a>). Furthermore, the correlation between H3K9 acetylation and gene expression measured by qRT-PCR was considerably higher than between H3K9 acetylation and array gene expression (r = 0.74 vs. 0.42, respectively) (<a href="#pone-0001882-g003"><strong>Figures 3C and 3D</strong></a>). The data suggest that significant differential gene expression can be missed by expression microarray analysis but recaptured by integrating functionally relevant epigenetic modifications such as H3K9 acetylation. These data also confirm the close association between transcriptional activation and histone acetylation at the genome-wide level.</p>


<h4>Genome-wide promoter methylation status shows an overall inverse correlation with gene expression and H3K9 levels</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Promoter DNA methylation is generally believed to be associated with gene silencing, although it is not clear whether this can be generalized in a whole genome analysis. Therefore, we determined the relationship between gene expression and DNA methylation in our set of leukemia patient samples. As before, high SNR genes (&gt;2.5) were selected for this analysis. Correlation of expression and DNA methylation among these genes revealed a bimodal distribution where two-thirds of genes displayed a strong positive correlation between expression and the log HpaII/MspI ratio (which translates biologically into an inverse correlation between gene expression and DNA methylation levels) where the peak value was r = 1.0. The remaining one-third of genes showed a weak negative correlation (peak at r = −0.5) (<a href="#pone-0001882-g004"><strong>Figure 4A</strong></a>). The presence of a strong correlation between gene expression and log HpaII/MspI ratio was still clearly detected even at SNR cutoffs of 1.3. These data indicate that for a majority of promoters, DNA methylation is strongly associated with gene silencing. A similar bimodal distribution was detected for the correlations between DNA methylation and H3K9 acetylation. The largest peak of the correlations was found at r = 1.0. A second and less defined peak that represented a smaller population of genes was found with a slightly negative correlation (around r = −0.5) (<a href="#pone-0001882-g004"><strong>Figure 4B</strong></a>). While the first peak is to be expected, representing silenced genes with methylated promoters lacking the H3K9ac mark associated with active chromatin, the second peak may be explained by a number of different factors (see <a href="#s3">discussion</a>). Overall, the results demonstrate that for most genes, the presence of high levels of methylation corresponds with repression, while high levels of acetylation correspond to activation, as exemplified in <a href="#pone-0001882-g004"><strong>Figure 4C</strong></a>.</p>
<div class="figure" id="pone-0001882-g004"><div class="img"><a name="pone-0001882-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Promoter DNA methylation shows genome-wide inverse correlation with gene expression and H3K9 acetylation: Panel A:</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Smoothed histogram of gene-by-gene correlations between log(HpaII/MspI) values and gene expression, showing a positive correlation between the two measures for the majority of genes, which translates into a negative biological correlation (<em>i.e.</em> higher promoter methylation correlates with lower gene expression). Panel B: Smoothed histogram of gene-by-gene correlations between log(HpaII/MspI) values and H3K9 acetylation, showing a positive correlation between the two measures for many of the genes, which translates into a negative biological correlation (<em>i.e.</em> higher promoter methylation correlates with lower promoter H3K9 acetylation). Panel C: Graphical representation of the data from all three platforms for one of the cases (AML.2) as custom tracks in the UCSC genome browser<a href="#pone.0001882-Kent1">[40]</a>. Four representative genes are shown here to illustrate the correlation between the three platforms. H3K9 acetylation data (in blue) is represented as the ratio of the signal between the H3K9 acetyl channel and the input channel; DNA methylation (in red) is represented as log(HpaII/MspI), so that a negative deflection corresponds to a methylated HpaII fragment while a positive one corresponds to a hypomethylated fragment; finally, gene expression data (in green) is represented as median-centered log<sub>2</sub> of RMA-normalized intensities.</p>
<span>doi:10.1371/journal.pone.0001882.g004</span></div>

<h4>Integration of histone acetylation and gene expression profiling synergistically enhances the detection of differentially regulated genes between AML and ALL samples</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Taken together with the fact that H3K9 acetylation and gene expression capture different cohorts of genes, the above findings led us to predict that integrating the measurement of individual gene variation captured by H3K9 acetylation and gene expression profiles would recover biologically significant differences missed by both platforms when considered independently of each other. In order to determine if this was the case, we tested whether integration of gene expression and H3K9 acetylation could enhance the discovery of genes that discriminate between AML and ALL cells. As mentioned above, a T-test analysis (p&lt;0.02 (absolute t score&gt;4.5)) identified 1,359 differentially expressed genes and 374 differentially acetylated, with an overlap of 45 genes (<a href="#pone-0001882-g002"><strong>Figure 2D</strong></a>). We next looked for genes that did not meet these criteria for significance on either platform, but that were marginally significant for both of them (p&gt;0.02 and &lt;0.14; absolute t score&lt;4.5 and &gt;2 with both t scores displaying the same direction). Applying these criteria allowed identification of an additional 382 genes that are likely to be discriminative between ALL and AML (<a href="#pone-0001882-g005"><strong>Figures 5A and 5B</strong></a>). Since the noise in each platform is expected to be random, genes that in both platforms manage to reach a borderline level of significance are unlikely to be due to random chance. Based on the integration of their individual t-scores, the probability that such genes are differentially regulated is increased to p&lt;0.01 (p&lt;(0.14<sup>2</sup>)/2). Nine genes (6 test and 3 negative controls) were chosen for validation by single locus quantitative ChIP (qChIP) and qRT-PCR. All six test genes were found to be both differentially H3K9 acetylated (<a href="#pone-0001882-g005"><strong>Figure 5C</strong></a>) and differentially expressed (<a href="#pone-0001882-g005"><strong>Figure 5D</strong></a>), while none of the negative controls were found to show any significant difference by either qChIP or qRT-PCR. Due to the limiting amount of sample available we were unable to perform replicate PCR runs, but each qPCR experiment was itself ran in triplicate. The data suggest that integrated analysis captures biological differences between tumors otherwise missed by independent analysis of gene expression or histone acetylation.</p>
<div class="figure" id="pone-0001882-g005"><div class="img"><a name="pone-0001882-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Gene expression and H3K9 acetylation synergize to increase the number of differentially expressed genes identified.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Panel A: Dot plot of t scores from T-test for gene expression (x axis) and H3K9 acetylation (y axis). 1,359 genes were identified as differentially expressed and 374 as differentially H3K9 acetylated, with absolute t scores&gt;4.5 (p&lt;0.02). 44 genes crossed the threshold for both assays (upper right). 83 genes that passed H3K9 acetylation threshold displayed marginal t scores (&gt;2 and &lt;4.5) for expression, while 154 genes that passed expression threshold displayed marginal t scores (&gt;2 and &lt;4.5) for H3K9 acetylation. Differential expression of the former genes was validated as in <a href="#pone-0001882-g004">Fig 4</a>. Combining genes with marginal t scores (&gt;2 and &lt;4.5) for both platforms simultaneously yielded an additional 382 genes with positive correlation that would have been missed by both platforms (central square, red). For illustrating purposes, points corresponding to gene expression t scores&gt;8 or H3K9 acetylation t scores&gt;7 were excluded from the figure. Panel B: Bar graph illustrating synergism between the gene expression and H3K9 acetylation analyses. 1314 genes were identified only by gene expression, 329 by H3K9 acetylation alone, and 44 genes were captured by both platforms. After the integrated analysis, an additional 382 genes were identified. Panels C and D: Validation of fold enrichment for H3K9 acetylation by qChIP (panel C) and fold difference in expression by qRT-PCR (panel D) of 9 genes (6 test and 3 negative controls) identified by the integrated analysis that had been previously missed by both.</p>
<span>doi:10.1371/journal.pone.0001882.g005</span></div>

<h4>Integration of DNA methylation and gene expression profiling has an additive effect on the detection of differentially regulated genes between AML and ALL samples</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We next wished to determine if DNA methylation profiles, like H3K9 acetylation, could enhance the capture of differentially-regulated genes. The same analytical approach to the integration of gene expression and DNA methylation was used to quantify to what extent these two platforms reinforced each other in the discrimination of leukemic lineage of our samples. An initial significance level of p&lt;0.02 (absolute t score&gt;4.5) was once again chosen, but an additional requirement of a difference between the two means greater than 1.5 was applied to the methylation data as described above. In this way, 1,359 genes were differentially expressed and 190 were differentially methylated between ALL and AML cells with an overlap of 13 genes. In order to test whether the two platforms reinforced each other, we examined how many genes were consistently marginally significant on both platforms (p&gt;0.02 and &lt;0.14; absolute t score&lt;4.5 and &gt;2). However, in order to ensure a potentially biologically relevant difference in methylation, we once more had an additional criterion of a minimum difference between the sample means&gt;1.5 for the methylation data. When these criteria were applied to the integration of gene expression and DNA methylation we did not find a significant number of additional genes. However, when qRT-PCR was performed on a subset of genes (eight genes) that were identified as differentially methylated but that were not predicted as differentially expressed on the arrays, significant actual differences in mRNA levels between ALL and AML cells were found for all of these genes. This level of differential gene expression by qRT-PCR was comparable to that measured for genes that had been successfully identified as differentially expressed by the expression arrays (<a href="#pone-0001882-g006"><strong>Figure 6A and 6B</strong></a>). These results suggest that the HELP platform is capable of rescuing differentially expressed genes. However, it does so in an additive fashion with gene expression arrays as opposed to the synergistic results obtained with the integration of gene expression and H3K9 acetylation.</p>
<div class="figure" id="pone-0001882-g006"><div class="img"><a name="pone-0001882-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>DNA methylation identifies differentially expressed genes missed by gene expression arrays.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>Panel A: Fold difference in gene expression by qRT-PCR between ALL and AML in a subset of genes identified as differentially methylated but not as differentially expressed (left), a group of genes that had been identified as differentially expressed but not differentially methylated (center), and in a group of genes that had not been identified as either differentially expressed nor differentially methylated. Panel B: Table showing the median, maximum and minimum values in fold difference for the three groups, reflecting how a subgroup of genes that is missed by gene expression arrays but identified by DNA methylation display comparable differences in gene expression levels when using a more sensitive technique such as qRT-PCR.</p>
<span>doi:10.1371/journal.pone.0001882.g006</span></div>

<h4>Integration of platforms improves identification of biological pathways that are differentially regulated in AML versus ALL cells</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>Our next goal was to determine whether over-representation of particular gene networks and canonical pathways could be found among the genes identified by the different platforms. For this purpose we used the Ingenuity Pathway Analysis software (Redwood City, CA). We compared and contrasted the biological gene networks captured as being differentially regulated in AML versus ALL cells by the three different platforms compared with those captured by integrated analysis. We found that the top networks captured by each of the three individual platforms yielded smaller and less directly affected networks centered in part around HoxA9 and APP in H3K9 acetylation, TNF and NFkB in cytosine methylation, and TERT and NFkB in gene expression (<a href="#pone.0001882.s005"><strong>Figure S4A</strong></a>). In contrast, the integrated analysis captured gene networks which were clearly centered around TNF and TP53, including multiple directly involved genes associated with the central node (<a href="#pone.0001882.s005"><strong>Figure S4B</strong></a>). In addition, networks of likely pathological importance (such as the one centering around MYC) were completely missed by the single platforms but were rescued when the data for the three platforms were integrated (<a href="#pone.0001882.s005"><strong>Figure S4C</strong></a>). This result demonstrates that the integration of data from gene expression and epigenetic platforms not only rescues genes that are missed by either platform alone, but may also enhance their power to more accurately define the most highly affected biological pathways.</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>Likewise, the integrated analysis provided a more statistically significant capture of canonical pathways (<em>e.g.</em> p = 9.5×e<sup>−10</sup> for the TP53 signaling pathway, p = 2.33×e<sup>−9</sup> for the protein ubiquitylation pathway and p = 4.03×e<sup>−9</sup> for the WNT-β catenin signaling pathway) than any single platform (<a href="#pone-0001882-g007"><strong>Figure 7A</strong></a><strong> and</strong> <a href="#pone-0001882-t002"><strong>Table 2</strong></a>), and rescued to the analysis another group of pathways that would otherwise have been missed (<em>e.g.</em> the PI3K/AKT signaling pathway (p = 3.27×e<sup>−7</sup>) and the chemokine signaling pathway (p = 1.06×e<sup>−3</sup>)) (<a href="#pone-0001882-g007"><strong>Figure 7B</strong></a>, <a href="#pone-0001882-t003"><strong>Table 3</strong></a><strong> and <a href="#pone.0001882.s006">Table S1</a></strong>). These data suggest that integrative analysis provides a superior platform for hypothesis-generating experiments and potentially greater accuracy in detecting the most relevant biological processes that distinguish different tumors from each other.</p>
<div class="figure" id="pone-0001882-g007"><div class="img"><a name="pone-0001882-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Integration of data from all three platforms enhances the biological information captured.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>Panel A: Bar graph comparing the significance level of enrichment (y axis: −log(p-value)) for genes within specific canonical pathways for the integrated analysis and the individual platforms. Panel B: Bar graph showing a subset of canonical pathways that were detected as significant by the integrated analysis, but that had been missed by the individual platforms.</p>
<span>doi:10.1371/journal.pone.0001882.g007</span></div><div class="figure" id="pone-0001882-t002"><div class="img"><a name="pone-0001882-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Enrichment of specific canonical pathways by the integrated analysis and the individual platforms</span></strong></p><span>doi:10.1371/journal.pone.0001882.t002</span></div><div class="figure" id="pone-0001882-t003"><div class="img"><a name="pone-0001882-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001882" data-uri="info:doi/10.1371/journal.pone.0001882.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001882.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001882.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001882.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001882.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001882.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Canonical pathways rescued by the integrated analysis</span></strong></p><span>doi:10.1371/journal.pone.0001882.t003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Histone modifications and DNA methylation play a critical role in regulating gene expression by modifying the chromatin structure of genes and recruiting additional regulatory factors. Our work is based on the hypothesis that a truly integrated epigenomic analysis could yield superior insight into the transcriptional programming of cancer cells beyond that obtained by simply measuring abundance of mRNAs through expression microarrays, since epigenetic modifications will capture information not only from genes being actively transcribed, but they will also reflect the <em>availability</em> for transcription by informing on the chromatin structure at specific loci. As proof of principle, we selected two functionally validated epigenetic marks–cytosine methylation and histone 3 lysine 9 acetylation–in addition to standard gene expression arrays and tested their ability to identify gene regulatory differences between the AML and ALL cell types. First, we demonstrate that such multiplatform epigenomic studies can be readily performed in enriched leukemia cells from standard clinical trial patient specimens. Second, using a novel approach for integrated analysis, we demonstrate that not only is there a functional relationship between gene expression and epigenetic marks but, more importantly, that these platforms synergize to provide a more complete and comprehensive analysis of transcriptional programming in human cells. Some of the strengths of this study include the rigorous quality control steps, the use of a powerful DNA methylation platform on specially designed high density oligonucleotide microarrays, the use of primary patient materials, the performance of the three different assays using the same type of 50-mer high density oligonucleotide arrays in multiple replicates and the extensive single locus validation.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Recently DNA methylation microarrays have been used to study acute leukemias as well as other malignancies. Groups of hypermethylated genes have been identified by such studies in AML cell lines <a href="#pone.0001882-Gebhard1">[25]</a> as well as in ALL patient samples <a href="#pone.0001882-Taylor1">[26]</a>. Thus, the integration of genetic and epigenetic platforms seems only natural, since individually both types of platforms have proven to capture biologically relevant information. Along these lines, some groups have begun to investigate the potential to be found in the combination of information from different microarray platforms. Shi et. al. used a CpG island microarray containing 1507 expressed CpG island sequence tags to carry out a triple analysis of histone acetylation, DNA methylation and gene expression in an ovarian cancer cell line treated with trichostatin A and 5′-deoxyazacytidine. While they were able to detect a functional interaction between histone acetylation and DNA methylation, they could not demonstrate an overall correlation between changes in epigenetic modifications and changes in expression levels <a href="#pone.0001882-Shi1">[27]</a>. Wu et. al. used a combination of ChIP-chip for H3K9 modifications and Differential Methylation Hybridization (DMH) on a 9.2 K mouse promoter array and showed an inverse correlation between H3K9 acetylation and DNA methylation, while no significant correlation could be found between DNA methylation and H3 dimethyl-K9 at the promoter level<a href="#pone.0001882-Wu1">[28]</a>. In our study, however, we propose the use of a combination of three different high-density genomic and epigenomic platforms for the in-depth analysis of their relationship in the context of human cancer specimens.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Posing a simple biological question–that is the differentiation between cell types in a sample set–we determined first by carrying out a systematic unsupervised clustering analysis that epigenomic platforms can be readily used for profiling and classification of leukemia clinical samples. Moreover, the combination of DNA methylation and H3K9 acetylation to gene expression data resulted in a significantly larger number of genes being identified that distinguished ALL from AML samples. Since each one of these platforms is affected by unique technical limitations, it is not surprising that they would result in the detection of only partially overlapping cohorts of genes. The existence of such limitations was confirmed by the fact that restricting the analysis to the subset of genes that displayed high signal to noise ratios on any two platforms (i.e. those genes that we can be certain were accurately measured by both platforms) resulted in a high degree of correlation between the different measures.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Furthermore, we hypothesized that this technical limitation due to the presence of noise in gene expression arrays was significantly affecting our ability to detect genuine differences in mRNA levels. By looking at a group of genes that displayed a significant difference between ALL and AML in either H3K9 acetylation or DNA methylation levels but did not display significant differences on gene expression arrays we found that when the mRNA levels of these genes were measured by qRT-PCR, an underlying difference in gene expression could be readily detected. Thus, we were able to confirm that there is an important degree of loss of information when carrying out genome-wide studies by gene expression microarrays alone, and that this information can be recovered by the integration of epigenetic data, reflecting the additive benefit obtained from such an integrated approach.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>However, our main goal was to investigate whether gene expression and epigenomic microarrays were capable of reinforcing each other. Our data show that it is possible to further harness the power of integrated epigenomics to identify differentially regulated genes, since genes missed by both gene expression and epigenetic profiling could be recovered for analysis by taking advantage of the tendency of gene expression profiling to correlate positively or negatively with epigenetic marks. For this we looked for genes that were marginally below the significance threshold on gene expression <em>and</em> H3K9 acetylation and for which both measures behaved <em>concordantly</em>. Using these new criteria, an additional 382 genes were identified that had been missed by both platforms. Careful single locus validation of randomly selected genes from this cohort confirmed such genes to be genuinely differentially acetylated and expressed, thus demonstrating the <em>synergistic</em> power of this integrative analysis.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>We propose that the additive and/or synergistic ability of integrative genomics and epigenomics to capture differentially-regulated genes in human clinical samples will enhance understanding of disease pathogenesis when carried out in an adequately designed study. The current study used the extreme comparison of ALL with AML clinical samples to demonstrate proof of principle of the approach. However, even with limited numbers of samples, the integrated analysis captures gene networks missed by single platforms, improves the level of confidence in gene networks which were only partially recognized by single platforms, and may center networks more completely around critical mediators of tumorigenesis so that subsequent functional studies could focus on gene products most likely to occupy central roles in the biology of the specific tumors.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In summary, our simple approach for integrated analysis shows a functional relationship between gene expression and epigenetic marks and, more importantly, demonstrates that these platforms synergize to provide a more complete and comprehensive analysis of transcriptional programming. We predict that when applied to large cohorts of patients enrolled in clinical trials, this integrated epigenomics approach will provide more accurate disease classification and more powerful prognostic information, which could be then used to design improved risk adapted and targeted therapy clinical trials.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Leukemia samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Specimens were obtained from 5 patients diagnosed with ALL (2 cases) or AML (3 cases). All patients had signed informed consent. Use of the ALL samples was approved through the Institutional Review Board (IRB) of Our Lady Of Mercy Cancer Center and use of the AML samples was approved through the Eastern Cooperative Oncology Group IRB for the E1900 clinical trial. Furthermore, the study was also approved by the Albert Einstein College of Medicine's Institutional Review Board. Mononuclear cells were isolated from heparinized bone marrow or peripheral blood specimens by ficoll-hypaque density centrifugation. The cells were washed twice in phosphate-buffered saline and subsequently frozen at a concentration of 40 million cells/mL in 90% fetal bovine serum and 10% dimethylsulfoxide. Characterization of antigen profiles by multiparameter flow cytometry was performed on mononuclear cells prior to freezing. Antibody binding was evaluated on a FACSCalibur using the CellQuest software program (Becton-Dickinson, Mountainview, CA). The patient characteristics are summarized in <a href="#pone-0001882-t001"><strong>Table 1</strong></a>.</p>


<h4>Genomic DNA extraction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Genomic DNA was extracted from 5–10×10<sup>6</sup> cells following a standard phenol-chloroform protocol followed by an ethanol precipitation and resuspension in 10 mM Tris-HCl pH 8.0.</p>


<h4>Total RNA extraction</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Total RNA was extracted from 20×10<sup>6</sup> using an RNeasy Mini kit from Qiagen (Valencia, CA), eluting twice in 30 µL of RNAse-free water.</p>


<h4>Chromatin Immunoprecipitation Assay (ChIP)</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>10×10<sup>6</sup> bone marrow cells were resuspended in 10 mL of DMEM and fixed with 1% formaldehyde at room temperature for 10 minutes. Reactions were quenched with 0.125 M glycine for 5 min, and the cells washed twice in cold PBS. Cells were lysed in 1.3 mL of lysis buffer (1% SDS+50 mM Tris-HCl pH 8.1+10 mM EDTA pH 8.0+protease inhibitors) on ice for 10 minutes and sonicated using a Branson microtip sonicator at 45% amplitude (Fisher scientific, Pittsburgh, PA) in order to achieve chromatin fragmentation to an average of 500 bp. Chromatin was pre-cleared with 100 µL of Protein A agarose beads (Roche, Indianapolis, IN) for 2 hours at 4°C, and then diluted 10-fold in dilution buffer (1.1% Triton X-100+0.01% SDS+167 mM NaCl+16.7 mM Tris-HCl pH 8.1+1.2 mM EDTA+protease inhibitors). One hundred microliters were set apart for the input sample. Immunoprecipitation (IP) reactions in chromatin from 1×10<sup>6</sup> cells were carried overnight at 4°C with either 2.5 µg of anti-acetylated histone H3 lysine 9 antibody (Upstate, Catalog #: 07-352, lot: 31388) or an equal amount of IgG isotype control (Jackson ImmunoResearch). On the following day immune complexes were recovered by adding 25 µL of protein A agarose beads to each reaction, incubating at 4°C for 45 minutes and then spinning down at 3000 rpm. Each sample was washed 5 times, 10 minutes each time, as follows: 1× low salt wash (0.1% SDS+1% Triton X-100+2 mM EDTA+20 mM Tris-HCl pH 8.1+150 mM NaCl), 1× high-salt wash (0.1% SDS+1% Triton X-100+2 mM EDTA+20 mM Tris-HCl pH 8.1+500 mM NaCl), 1× LiCl wash (0.25 M LiCl+1% NP-40+1% deoxycholate+1 mM EDTA+10 mM Tris-HCl pH 8.1) and 2× TE pH 8.0 washes (10 mM Tris-HCl+1 mM EDTA). DNA from each of the IPs and the input sample was eluted twice for 20 minutes at room temperature in 100 µL of elution buffer (1% SDS+0.1 M NaHCO<sub>3</sub>) followed by an overnight incubation at 65°C to reverse the cross-linking. DNA samples were cleaned using a QIAquick PCR purification kit (QIAGEN, Valencia, CA) using the manufacturer's protocol, but using 30 µL of elution buffer for the final elution from the column. Enrichment was calculated by quantitative real-time PCR in a DNA Engine Opticon 2 real-time thermocycler from Biorad (Hercules, CA) using primers directed against the promoter regions of the <em>CD10</em>, <em>CD20</em>, <em>CD33</em> or <em>MPO</em> genes (for primer sequences see <a href="#pone.0001882.s007"><strong>Table S2</strong></a>). Specific enrichment for H3K9ac was determined using the ΔΔC(t) method as previously described<a href="#pone.0001882-Chakrabarti1">[29]</a>. Samples were amplified using a ligation-mediated PCR protocol as previously described<a href="#pone.0001882-Oberley1">[30]</a> and maintenance of enrichment was verified by quantitative PCR prior to submission for labeling and hybridization onto the NimbleGen human HG17 promoter tiling oligonucleotide microarray (2005-04-18_HGS17_min_promoter_set; design ID: 1871) containing 385,000 probes covering 24,134 promoters (1 kb and +500 bp around the transcription start-site, on average). ChIP-chip microarray data has been submitted to the GEO database for public access (accession number pending)</p>


<h4>DNA methylation analysis by HELP</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The HELP assay was carried out as previously published<a href="#pone.0001882-Khulan1">[23]</a> with slight modifications. One microgram of genomic DNA was digested overnight with either HpaII or MspI (NEB, Ipswich, MA). On the following day the reactions were extracted once with phenol-chloroform and resuspended in 11 µL of 10 mM Tris-HCl pH 8.0 and the digested DNA was used to set up an overnight ligation of the JHpaII adapter using T4 DNA ligase. The adapter-ligated DNA was used to carry out the PCR amplification of the HpaII and MspI-digested DNA as previously described<a href="#pone.0001882-Khulan1">[23]</a>. Both amplified fractions were submitted to Roche NimbleGen, Inc. (Madison, WI) for labeling and hybridization onto a human HG17 custom-designed oligonucleotide array (50-mers) covering 25,626 HpaII amplifiable fragments (HAF) located at gene promoters and imprinted regions. HpaII amplifiable fragments are defined as genomic sequences contained between two flanking HpaII sites found within 200–2,000 bp from each other. Each HAF on the array is represent by 15 individual probes, randomly distributed across the microarray slide. HELP microarray data has been submitted to the GEO database for public access (accession number pending)</p>


<h4>Quantitative DNA methylation analysis by MassArray Epityping</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Validation of HELP microarray findings was carried out by MALDI-TOF mass spectrometry using EpiTyper by MassArray (Sequenom, CA) on bisulfite-converted DNA as previously described<a href="#pone.0001882-Ehrich1">[31]</a>. MassArray primers were designed to cover the flanking HpaII sites for a given HAF, as well as any other HpaII sites found up to 2,000 bp upstream of the downstream site and up to 2,000 bp downstream of the upstream site, in order to cover all possible alternative sites of digestion (primer sequences available on <a href="#pone.0001882.s007"><strong>Table S2</strong></a>).</p>


<h4>cDNA synthesis for Gene expression assays</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Ten micrograms of total RNA were used to synthesize cDNA using Superscript III (Invitrogen, Carlsbad, California) using the manufacturer's protocol for qRT-PCR. Double-stranded cDNA synthesis for gene expression arrays was likewise carried out using Invitrogen's Superscript III, but a modified oligo-d(T) primer (<span class="gene">5′GGCCAGTGAATTGTAATACGACTCACTATAGGGAGGCG​GTTTTTTTTTTTTTTTTTTTTTTTT-3′</span>) was used in lieu of the kit's oligo-d(T) primer. The second strand was synthesized for 2 hours at 16°C in a reaction containing 30 µL of 5×second strand buffer (Invitrogen), 90 µL of RNAse-free water, 3 µL dNTP 10 mM, 1 µL of BSA 1 µg/µL, 1 ul of E. coli DNA ligase 10 U/µL (Invitrogen), 4 µL E. coli DNA polymerase I 10 U/µL (Invitrogen) and 1 µL of RNase H 2 U/µL (Invitrogen). Finally, 2 µL of T4 DNA Polymerase 5 U/µL (Invitrogen) were added and the reaction incubated at 16°C for another 10 minutes. Reactions were cleaned with Qiagen's Qiaquick PCR purification kit and submitted for labeling and hybridization onto the NimbleGen standard human HG17 60mer 385,000 probe gene expression array (2005-04-20_Human_60mer_1in2; design ID: 1877). Gene expression microarray data has been submitted to the GEO database for public access (accession number pending)</p>


<h4>Quantitative real time PCR</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The expression values of fifteen genes were validated by quantitative RT-PCR (qRT-PCR). cDNA was synthesized using the Superscript III First Strand kit from Invitrogen (Invitrogen's Superscript III) as per manufacturer's protocol. The ΔΔC(t) method was used to determine relative gene expression levels <a href="#pone.0001882-Livak1">[32]</a> using Power SYBRGreen from Applied Biosystems (Foster City, CA) and a DNA Engine Opticon 2 real-time thermocycler from Biorad (Hercules, CA). All primer sequences are available as supplementary data <a href="#pone.0001882.s007"><strong>(Table S2)</strong></a>.</p>


<h4>Sample labeling and Microarray hybridizations</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>All samples for microarray hybridization were processed at the Roche NimbleGen Service Laboratory. Samples were labeled using Cy-labeled random primers (9mers) and then hybridized onto the corresponding microarray platform and scanned using a GenePix 4000B scanner (Axon Instruments) as previously described<a href="#pone.0001882-Selzer1">[33]</a>.</p>


<h4>Microarray quality control</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>All microarray hybridizations were subjected to extensive quality control using the following strategies. First, uniformity of hybridization was evaluated using a modified version of a previously published algorithm<a href="#pone.0001882-Reimers1">[34]</a> adapted for the NimbleGen platform, and any hybridization with strong regional artifacts was repeated. Second, normalized signal intensities from each array were compared against a 20% trimmed mean of signal intensities across all arrays in that experiment, and any arrays displaying a significant intensity bias that could not be explained by the biology of the sample were excluded. Finally, replicate reproducibility was estimated by using scatter plots and a Pearson correlation matrix, and any clear outliers were excluded from further analysis <strong>(<a href="#pone.0001882.s002">Figures S1A, S1B and S1C</a>).</strong></p>


<h4>Acetyl histone H3 lysine 9 ChIP-chip analysis</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>The H3K9 acetylation status at each promoter was computed by taking the log-ratio between the probe intensities of the ChIP product and input chromatin, which were co-hybridized on the same assay. For each of the 24,134 promoter regions, we computed the moving average of the log-ratio along its 15 probes with a window size of 3 probes, and found the maximum value of the moving averages of over each region covered by 15 probes. The maximum values served as the summary of histone H3 lysine 9 acetylation of the promoter regions and were used to correlate with gene expression profiles. All chips that passed primary quality control showed a bimodal distribution with a distinct lower mode. We took these lower modes to indicate non-enriched fragments, and normalized across arrays by lining up these modes.</p>


<h4>HELP data processing and analysis</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>Signal intensities at each HpaII amplifiable fragment were calculated as a robust (25% trimmed) mean of their component probe-level signal intensities. Any fragments found within the level of background MspI signal intensity, measured as 2.5 mean-absolute-differences (MAD) above the median of random probe signals, were categorized as “failed.” These “failed” loci therefore represent the population of fragments that did not amplify by PCR, whatever the biological (e.g. genomic deletions and other sequence errors) or experimental cause. On the other hand, “Methylated” loci were so designated when the level of HpaII signal intensity was similarly indistinguishable from background. PCR-amplifying fragments (those not flagged as either “methylated” or “failed”) were normalized using an intra-array quantile approach wherein HpaII/MspI ratios are aligned across density-dependent sliding windows of fragment size-sorted data. Analysis of normalized data revealed the presence of a bimodal distribution. For each sample a cutoff was selected at the point that more clearly separated these two populations and the data were centered around this point. Each fragment was then categorized as either methylated, if the centered log HpaII/MspI ratio was less than zero, or hypomethylated if on the other hand the log ratio was greater than zero.</p>


<h4>Gene expression data processing and analysis</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>Raw data (.pair files) from the scanner were processed with Roche NimbleGen's version of the RMA algorithm (without background correction) in NimbleScan 2.3 software. Gene expression microarray assays were performed in quadruplicate for each patient and 16 out of the 20 chips were selected on the basis of standard quality metrics and uniformity of hybridization. Next we selected 10 chips with the least evidence of technical artifact compared with the median across all chips, and constructed expression profiles for each of the five samples by averaging each pair of replicate chips among these 10.</p>


<h4>Descriptive statistics</h4>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1"></a><p>Noise in gene expression and other microarray measures was estimated as the standard error of the averages of the measures across technical replicates. For the ChIP-chip arrays the average standard error was 0.58. Gene expression values were processed by NimbleScan 2.3 software using a quantile normalization, which depresses estimates of variance at either end of the intensity distribution relative to the middle. Most of our measures of interest came from the middle range of intensities and their average estimated standard error was 0.30. For HELP, the average standard error of the quantile-normalized ratio estimates was 0.21. For between-measures correlation analysis we selected genes whose measured standard deviation was at least 2.5 times the standard error of the averages of replicates. Signal to noise ratio (SNR) for all the platforms was defined as the ratio of variation across samples compared to the variation between replicates of the same sample</p>


<h4>Clustering analysis</h4>
<a id="article1.body1.sec4.sec15.p1" name="article1.body1.sec4.sec15.p1"></a><p>Unsupervised clustering of HELP and ChIP-chip data using hierarchical clustering (correlation distance and complete linkage) <a href="#pone.0001882-Eisen1">[35]</a>, correspondence analysis <a href="#pone.0001882-Fellenberg1">[36]</a> and principal component analysis <a href="#pone.0001882-Misra1">[37]</a> were performed using the open-source statistical software R <a href="#pone.0001882-Team1">[38]</a> and the BioConductor package MADE4 <a href="#pone.0001882-Culhane1">[39]</a>. Since there are 10 distinct ways in which 5 samples can be divided into two groups of 2 and 3 samples each, it was calculated that there was a probability of 1 in 10 that this analysis would achieve accurate separation of samples by lineage due to chance.</p>
<a id="article1.body1.sec4.sec15.p2" name="article1.body1.sec4.sec15.p2"></a><p>Supervised analysis was done using a standard T-test with leukemic lineage (AML or ALL) as the dichotomous variable. A significance level of p&lt;0.02 was chosen for all three platforms. In the case of the methylation data, a second criterion of a difference&gt;1.5 between the mean of the two populations was added in order to increase the likelihood of detection of biologically significant changes in methylation levels.</p>


<h4>Pathway analysis</h4>
<a id="article1.body1.sec4.sec16.p1" name="article1.body1.sec4.sec16.p1"></a><p>Using the Ingenuity Pathway Analysis software (IPA) (Redwood City, CA) we carried out an analysis of the biological information retrieved by each of the individual platforms alone, and compared it to the information obtained by the integrated analysis of all three platforms. Enrichment of genes associated with specific canonical pathways was determined relative to the Ingenuity knowledge database for each of the individual platforms and the integrated analysis at a significance level of p&lt;0.01. Biological networks captured by the different microarray platforms were generated using IPA and scored based on the relationship between the total number of genes in the specific network and the total number of genes identified by the microarray analysis.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001882"></div><a name="pone.0001882.s001" id="pone.0001882.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Supplementary methods: Model for testing likelihood of correlations between measures</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001882.s002" id="pone.0001882.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Correlation matrix between replicate arrays and different biological samples for each platform. Panel A: Correlation matrix for all normalized HELP microarrays that passed hybridization quality control. Panel B: Correlation matrix for H3K9-acetyl ChIP-chip arrays that passed initial quality control. Panel C: Correlation matrix for the ten gene expression arrays selected for analysis after quality control assessment of all hybridizations.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.52 MB TIF)</p>
<a name="pone.0001882.s003" id="pone.0001882.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s003">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Technical validations by single locus studies for each microarray platform. Panel A: Correlation between qRT-PCR findings (x axis: Delta C(t) to GAPDH) and gene expression arrays (y axis: log2 intensity). The overall negative correlation (r = −0.72) confirms the good quality of the gene expression array data. Panel B: Correlation between H3K9-acetyl ChIP-chip (x axis: log H3K9ac/Input) and fold enrichment over non-specific IgG by qChIP (y axis). Since H3K9ac ChIP-chip data display a bimodal distribution, the lowest point between the two peaks (log H3K9acetyl = 2.5) was selected as a cutoff for comparison with qChIP. While most points below this cutoff point show minimal enrichment by qChIP, points to the right of this cutoff clearly show increasing amounts of enrichment with increasing log H3k9ac/input values. Panel C: Relationship between DNA methylation by HELP (x axis: log HpaII/MspI) and percent cytosine methylation measured by MALDI-TOF mass spectrometry.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.86 MB TIF)</p>
<a name="pone.0001882.s004" id="pone.0001882.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s004">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Unsupervised clustering using of epigenomic data principal component analysis (PCA). Unsupervised clustering of DNA methylation and H3K9 acetylation data using PCA separated the leukemia samples according to lineage along the first principal component. Panel A: Two-dimensional representation of PCA of DNA methylation data. ALL samples (in red) readily cluster apart from AML samples (in blue) along the first principal component (x axis) (left); and heatmap of top 100 genes from the first principal component (right). Genes are shown on the rows and samples on the columns, and data were row-centered. Low values represented in blue and high values in red. Panel B: Two-dimensional representation of PCA of H3K9-acetyl ChIP-chip data. ALL samples (in red) and AML samples (in blue) readily segregated along the first principal component (x axis) (left); and heatmap of top 100 genes from the first principal component (right). Genes are shown on the rows and samples on the columns, and data were row-centered. Low values represented in blue and high values in red.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.57 MB TIF)</p>
<a name="pone.0001882.s005" id="pone.0001882.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s005">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Highest scoring networks from Ingenuity Pathway Analysis comparing the individual platforms and the integrated analysis. Biological gene networks identified as differentially regulated in ALL versus AML. Networks were generated using the genes identified through the analysis of the individual platforms or from the genes identified when information from all three platforms was integrated. Panel A: The top two scoring networks identified using H3K9 acetylation data centered around HoxA9 and APP. Genes that were present in the analysis gene list appear colored in grey while genes identified indirectly appear in white. Panel B: The top two scoring networks identified using DNA methylation data centered around TNF and NFkB. Panel C: The top two scoring networks identified using gene expression data centered around TERT and NFkB. Panel D: The top two scoring networks identified using data from the integration of all three platforms centered around TNF and TP53. Panel E: The biological network centering around MYC was among the highest scoring networks (network #6) when using the integrated data, while none of the individual platforms succeeded in identifying the MYC network directly.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(2.90 MB TIF)</p>
<a name="pone.0001882.s006" id="pone.0001882.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s006">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Canonical pathway analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001882.s007" id="pone.0001882.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001882.s007">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Primer sequences</p>
<p class="siDoi">doi:10.1371/journal.pone.0001882.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are grateful to Jeff Olson, Eunice Flores, Paul Oeth, and Min Seob Lee from Sequenom for their invaluable help with the initial setup and optimization of the MassArray EpiTyper.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JG AM MF. Performed the experiments: RS MF YL JF. Analyzed the data: JG KY AM MF MR RT. Contributed reagents/materials/analysis tools: JG RS KY RG MR RT JF EP PW. Wrote the paper: JG RS KY RG AM MF MR RT YL JF EP PW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001882-Jenuwein1" id="pone.0001882-Jenuwein1"></a>Jenuwein T, Allis CD (2001) Translating the histone code. Science  293: 1074–1080.  <ul class="find" data-citedArticleID="1082098" data-doi="10.1126/science.1063127"><li><a href="http://dx.doi.org/10.1126/science.1063127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Translating+the+histone+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Translating+the+histone+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001882-Bird1" id="pone.0001882-Bird1"></a>Bird A (2002) DNA methylation patterns and epigenetic memory. Genes Dev  16: 6–21.  <ul class="find" data-citedArticleID="1082072" data-doi="10.1101/gad.947102"><li><a href="http://dx.doi.org/10.1101/gad.947102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+patterns+and+epigenetic+memory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+patterns+and+epigenetic+memory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001882-Kurdistani1" id="pone.0001882-Kurdistani1"></a>Kurdistani SK, Tavazoie S, Grunstein M (2004) Mapping global histone acetylation patterns to gene expression. Cell  117: 721–733.  <ul class="find" data-citedArticleID="1082106" data-doi="10.1016/j.cell.2004.05.023"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.05.023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+global+histone+acetylation+patterns+to+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+global+histone+acetylation+patterns+to+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001882-Bernstein1" id="pone.0001882-Bernstein1"></a>Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, et al.  (2005) Genomic maps and comparative analysis of histone modifications in human and mouse. Cell  120: 169–181.  <ul class="find" data-citedArticleID="1082068" data-doi="10.1016/j.cell.2005.01.001"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.01.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+maps+and+comparative+analysis+of+histone+modifications+in+human+and+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001882-Herman1" id="pone.0001882-Herman1"></a>Herman JG, Baylin SB (2003) Gene silencing in cancer in association with promoter hypermethylation. N Engl J Med  349: 2042–2054.  <ul class="find" data-citedArticleID="1082094" data-doi="10.1056/nejmra023075"><li><a href="http://dx.doi.org/10.1056/nejmra023075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+silencing+in+cancer+in+association+with+promoter+hypermethylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+silencing+in+cancer+in+association+with+promoter+hypermethylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001882-Brinkman1" id="pone.0001882-Brinkman1"></a>Brinkman AB, Pennings SW, Braliou GG, Rietveld LE, Stunnenberg HG (2007) DNA methylation immediately adjacent to active histone marking does not silence transcription. Nucleic Acids Res.   <ul class="find" data-citedArticleID="1082074" data-doi="10.1093/nar/gkl1014"><li><a href="http://dx.doi.org/10.1093/nar/gkl1014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+immediately+adjacent+to+active+histone+marking+does+not+silence+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+immediately+adjacent+to+active+histone+marking+does+not+silence+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001882-Zinn1" id="pone.0001882-Zinn1"></a>Zinn RL, Pruitt K, Eguchi S, Baylin SB, Herman JG (2007) hTERT is expressed in cancer cell lines despite promoter DNA methylation by preservation of unmethylated DNA and active chromatin around the transcription start site. Cancer Res  67: 194–201.  <ul class="find" data-citedArticleID="1082140" data-doi="10.1158/0008-5472.can-06-3396"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-3396" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=hTERT+is+expressed+in+cancer+cell+lines+despite+promoter+DNA+methylation+by+preservation+of+unmethylated+DNA+and+active+chromatin+around+the+transcription+start+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22hTERT+is+expressed+in+cancer+cell+lines+despite+promoter+DNA+methylation+by+preservation+of+unmethylated+DNA+and+active+chromatin+around+the+transcription+start+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001882-Bernstein2" id="pone.0001882-Bernstein2"></a>Bernstein BE, Humphrey EL, Erlich RL, Schneider R, Bouman P, et al.  (2002) Methylation of histone H3 Lys 4 in coding regions of active genes. Proc Natl Acad Sci U S A  99: 8695–8700.  <ul class="find" data-citedArticleID="1082070" data-doi="10.1073/pnas.082249499"><li><a href="http://dx.doi.org/10.1073/pnas.082249499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Methylation+of+histone+H3+Lys+4+in+coding+regions+of+active+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Methylation+of+histone+H3+Lys+4+in+coding+regions+of+active+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001882-Jones1" id="pone.0001882-Jones1"></a>Jones PA, Baylin SB (2002) The fundamental role of epigenetic events in cancer. Nat Rev Genet  3: 415–428.  <ul class="find" data-citedArticleID="1082100"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+fundamental+role+of+epigenetic+events+in+cancer.&amp;auth=&amp;atitle=The+fundamental+role+of+epigenetic+events+in+cancer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fundamental+role+of+epigenetic+events+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fundamental+role+of+epigenetic+events+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001882-Baylin1" id="pone.0001882-Baylin1"></a>Baylin SB, Ohm JE (2006) Epigenetic gene silencing in cancer - a mechanism for early oncogenic pathway addiction? Nat Rev Cancer  6: 107–116.  <ul class="find" data-citedArticleID="1082064" data-doi="10.1038/nrc1799"><li><a href="http://dx.doi.org/10.1038/nrc1799" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epigenetic+gene+silencing+in+cancer+-+a+mechanism+for+early+oncogenic+pathway+addiction%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epigenetic+gene+silencing+in+cancer+-+a+mechanism+for+early+oncogenic+pathway+addiction%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001882-Melki1" id="pone.0001882-Melki1"></a>Melki JR, Vincent PC, Clark SJ (1999) Concurrent DNA hypermethylation of multiple genes in acute myeloid leukemia. Cancer Res  59: 3730–3740.  <ul class="find" data-citedArticleID="1082112"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Concurrent+DNA+hypermethylation+of+multiple+genes+in+acute+myeloid+leukemia.&amp;auth=&amp;atitle=Concurrent+DNA+hypermethylation+of+multiple+genes+in+acute+myeloid+leukemia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Concurrent+DNA+hypermethylation+of+multiple+genes+in+acute+myeloid+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Concurrent+DNA+hypermethylation+of+multiple+genes+in+acute+myeloid+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001882-Tessema1" id="pone.0001882-Tessema1"></a>Tessema M, Langer F, Dingemann J, Ganser A, Kreipe H, et al.  (2003) Aberrant methylation and impaired expression of the p15(INK4b) cell cycle regulatory gene in chronic myelomonocytic leukemia (CMML). Leukemia  17: 910–918.  <ul class="find" data-citedArticleID="1082134" data-doi="10.1038/sj.leu.2402891"><li><a href="http://dx.doi.org/10.1038/sj.leu.2402891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aberrant+methylation+and+impaired+expression+of+the+p15%28INK4b%29+cell+cycle+regulatory+gene+in+chronic+myelomonocytic+leukemia+%28CMML%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aberrant+methylation+and+impaired+expression+of+the+p15%28INK4b%29+cell+cycle+regulatory+gene+in+chronic+myelomonocytic+leukemia+%28CMML%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001882-Esteller1" id="pone.0001882-Esteller1"></a>Esteller M, Hamilton SR, Burger PC, Baylin SB, Herman JG (1999) Inactivation of the DNA repair gene O6-methylguanine-DNA methyltransferase by promoter hypermethylation is a common event in primary human neoplasia. Cancer Res  59: 793–797.  <ul class="find" data-citedArticleID="1082086"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inactivation+of+the+DNA+repair+gene+O6-methylguanine-DNA+methyltransferase+by+promoter+hypermethylation+is+a+common+event+in+primary+human+neoplasia.&amp;auth=&amp;atitle=Inactivation+of+the+DNA+repair+gene+O6-methylguanine-DNA+methyltransferase+by+promoter+hypermethylation+is+a+common+event+in+primary+human+neoplasia." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+the+DNA+repair+gene+O6-methylguanine-DNA+methyltransferase+by+promoter+hypermethylation+is+a+common+event+in+primary+human+neoplasia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+the+DNA+repair+gene+O6-methylguanine-DNA+methyltransferase+by+promoter+hypermethylation+is+a+common+event+in+primary+human+neoplasia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001882-Herman2" id="pone.0001882-Herman2"></a>Herman JG, Umar A, Polyak K, Graff JR, Ahuja N, et al.  (1998) Incidence and functional consequences of hMLH1 promoter hypermethylation in colorectal carcinoma. Proc Natl Acad Sci U S A  95: 6870–6875.  <ul class="find" data-citedArticleID="1082096" data-doi="10.1073/pnas.95.12.6870"><li><a href="http://dx.doi.org/10.1073/pnas.95.12.6870" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incidence+and+functional+consequences+of+hMLH1+promoter+hypermethylation+in+colorectal+carcinoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incidence+and+functional+consequences+of+hMLH1+promoter+hypermethylation+in+colorectal+carcinoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001882-Cameron1" id="pone.0001882-Cameron1"></a>Cameron EE, Bachman KE, Myohanen S, Herman JG, Baylin SB (1999) Synergy of demethylation and histone deacetylase inhibition in the re-expression of genes silenced in cancer. Nat Genet  21: 103–107.  <ul class="find" data-citedArticleID="1082076" data-doi="10.1038/5047"><li><a href="http://dx.doi.org/10.1038/5047" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synergy+of+demethylation+and+histone+deacetylase+inhibition+in+the+re-expression+of+genes+silenced+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synergy+of+demethylation+and+histone+deacetylase+inhibition+in+the+re-expression+of+genes+silenced+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001882-McGarvey1" id="pone.0001882-McGarvey1"></a>McGarvey KM, Fahrner JA, Greene E, Martens J, Jenuwein T, et al.  (2006) Silenced tumor suppressor genes reactivated by DNA demethylation do not return to a fully euchromatic chromatin state. Cancer Res  66: 3541–3549.  <ul class="find" data-citedArticleID="1082110" data-doi="10.1158/0008-5472.can-05-2481"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-2481" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Silenced+tumor+suppressor+genes+reactivated+by+DNA+demethylation+do+not+return+to+a+fully+euchromatic+chromatin+state." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Silenced+tumor+suppressor+genes+reactivated+by+DNA+demethylation+do+not+return+to+a+fully+euchromatic+chromatin+state.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001882-Golub1" id="pone.0001882-Golub1"></a>Golub TR, Slonim DK, Tamayo P, Huard C, Gaasenbeek M, et al.  (1999) Molecular classification of cancer: class discovery and class prediction by gene expression monitoring. Science  286: 531–537.  <ul class="find" data-citedArticleID="1082092" data-doi="10.1126/science.286.5439.531"><li><a href="http://dx.doi.org/10.1126/science.286.5439.531" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+classification+of+cancer%3A+class+discovery+and+class+prediction+by+gene+expression+monitoring." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+classification+of+cancer%3A+class+discovery+and+class+prediction+by+gene+expression+monitoring.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001882-Armstrong1" id="pone.0001882-Armstrong1"></a>Armstrong SA, Staunton JE, Silverman LB, Pieters R, den Boer ML, et al.  (2002) MLL translocations specify a distinct gene expression profile that distinguishes a unique leukemia. Nat Genet  30: 41–47.  <ul class="find" data-citedArticleID="1082062" data-doi="10.1038/ng765"><li><a href="http://dx.doi.org/10.1038/ng765" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MLL+translocations+specify+a+distinct+gene+expression+profile+that+distinguishes+a+unique+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MLL+translocations+specify+a+distinct+gene+expression+profile+that+distinguishes+a+unique+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001882-Valk1" id="pone.0001882-Valk1"></a>Valk PJ, Verhaak RG, Beijen MA, Erpelinck CA, Barjesteh van Waalwijk van Doorn-Khosrovani S, et al.  (2004) Prognostically useful gene-expression profiles in acute myeloid leukemia. N Engl J Med  350: 1617–1628.  <ul class="find" data-citedArticleID="1082136" data-doi="10.1056/nejmoa040465"><li><a href="http://dx.doi.org/10.1056/nejmoa040465" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prognostically+useful+gene-expression+profiles+in+acute+myeloid+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prognostically+useful+gene-expression+profiles+in+acute+myeloid+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001882-Rosenwald1" id="pone.0001882-Rosenwald1"></a>Rosenwald A, Wright G, Chan WC, Connors JM, Campo E, et al.  (2002) The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma. N Engl J Med  346: 1937–1947.  <ul class="find" data-citedArticleID="1082124" data-doi="10.1056/nejmoa012914"><li><a href="http://dx.doi.org/10.1056/nejmoa012914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+molecular+profiling+to+predict+survival+after+chemotherapy+for+diffuse+large-B-cell+lymphoma." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+molecular+profiling+to+predict+survival+after+chemotherapy+for+diffuse+large-B-cell+lymphoma.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001882-Berell1" id="pone.0001882-Berell1"></a>Berell M, Tortosa A, Verger E, Gil J, Viñolas N, et al.  (2005) Prognostic significance of O6-methylguanine-DNA methyltransferase determined by promoter hypermethylation and immunohistochemical expression in anaplastic gliomas. Clin Cancer Res  11: 5167–5174.  <ul class="find" data-citedArticleID="1082066" data-doi="10.1158/1078-0432.ccr-05-0230"><li><a href="http://dx.doi.org/10.1158/1078-0432.ccr-05-0230" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prognostic+significance+of+O6-methylguanine-DNA+methyltransferase+determined+by+promoter+hypermethylation+and+immunohistochemical+expression+in+anaplastic+gliomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prognostic+significance+of+O6-methylguanine-DNA+methyltransferase+determined+by+promoter+hypermethylation+and+immunohistochemical+expression+in+anaplastic+gliomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001882-Paz1" id="pone.0001882-Paz1"></a>Paz MF, Yaya-Tur R, Rojas-Marcos I, Reynes G, Pollan M, et al.  (2004) CpG island hypermethylation of the DNA repair enzyme methyltransferase predicts response to temozolomide in primary gliomas. Clin Cancer Res  10: 4933–4938.  <ul class="find" data-citedArticleID="1082120" data-doi="10.1016/s0028-3770(05)83427-7"><li><a href="http://dx.doi.org/10.1016/s0028-3770(05)83427-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CpG+island+hypermethylation+of+the+DNA+repair+enzyme+methyltransferase+predicts+response+to+temozolomide+in+primary+gliomas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CpG+island+hypermethylation+of+the+DNA+repair+enzyme+methyltransferase+predicts+response+to+temozolomide+in+primary+gliomas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001882-Khulan1" id="pone.0001882-Khulan1"></a>Khulan B, Thompson R, Ye K, Fazzari Mj, Suzuki M, et al.  (2006) Comparative isoschizomer profiling of cytosine methylation: the HELP assay. Genome Research  16: 1046–1055.  <ul class="find" data-citedArticleID="1082104" data-doi="10.1101/gr.5273806"><li><a href="http://dx.doi.org/10.1101/gr.5273806" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+isoschizomer+profiling+of+cytosine+methylation%3A+the+HELP+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+isoschizomer+profiling+of+cytosine+methylation%3A+the+HELP+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001882-Melki2" id="pone.0001882-Melki2"></a>Melki JR, Clark SJ (2002) DNA methylation changes in leukaemia. Semin Cancer Biol  12: 347–357.  <ul class="find" data-citedArticleID="1082114" data-doi="10.1016/s1044-579x(02)00055-x"><li><a href="http://dx.doi.org/10.1016/s1044-579x(02)00055-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+changes+in+leukaemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+changes+in+leukaemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001882-Gebhard1" id="pone.0001882-Gebhard1"></a>Gebhard C, Schwarzfischer L, Pham TH, Schilling E, Klug M, et al.  (2006) Genome-wide profiling of CpG methylation identifies novel targets of aberrant hypermethylation in myeloid leukemia. Cancer Res  66: 6118–6128.  <ul class="find" data-citedArticleID="1082090" data-doi="10.1158/0008-5472.can-06-0376"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-0376" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+profiling+of+CpG+methylation+identifies+novel+targets+of+aberrant+hypermethylation+in+myeloid+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+profiling+of+CpG+methylation+identifies+novel+targets+of+aberrant+hypermethylation+in+myeloid+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001882-Taylor1" id="pone.0001882-Taylor1"></a>Taylor KH, Pena-Hernandez KE, Davis JW, Arthur GL, Duff DJ, et al.  (2007) Large-scale CpG methylation analysis identifies novel candidate genes and reveals methylation hotspots in acute lymphoblastic leukemia. Cancer Res  67: 2617–2625.  <ul class="find" data-citedArticleID="1082130" data-doi="10.1158/0008-5472.can-06-3993"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-06-3993" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+CpG+methylation+analysis+identifies+novel+candidate+genes+and+reveals+methylation+hotspots+in+acute+lymphoblastic+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+CpG+methylation+analysis+identifies+novel+candidate+genes+and+reveals+methylation+hotspots+in+acute+lymphoblastic+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001882-Shi1" id="pone.0001882-Shi1"></a>Shi H, Wei SH, Leu YW, Rahmatpanah F, Liu JC, et al.  (2003) Triple analysis of the cancer epigenome: an integrated microarray system for assessing gene expression, DNA methylation, and histone acetylation. Cancer Res  63: 2164–2171.  <ul class="find" data-citedArticleID="1082128"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Triple+analysis+of+the+cancer+epigenome%3A+an+integrated+microarray+system+for+assessing+gene+expression%2C+DNA+methylation%2C+and+histone+acetylation.&amp;auth=&amp;atitle=Triple+analysis+of+the+cancer+epigenome%3A+an+integrated+microarray+system+for+assessing+gene+expression%2C+DNA+methylation%2C+and+histone+acetylation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Triple+analysis+of+the+cancer+epigenome%3A+an+integrated+microarray+system+for+assessing+gene+expression%2C+DNA+methylation%2C+and+histone+acetylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Triple+analysis+of+the+cancer+epigenome%3A+an+integrated+microarray+system+for+assessing+gene+expression%2C+DNA+methylation%2C+and+histone+acetylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001882-Wu1" id="pone.0001882-Wu1"></a>Wu J, Wang SH, Potter D, Liu JC, Smith LT, et al.  (2007) Diverse histone modifications on histone 3 lysine 9 and their relation to DNA methylation in specifying gene silencing. BMC Genomics  8: 131.  <ul class="find" data-citedArticleID="1082138" data-doi="10.1186/1471-2164-8-131"><li><a href="http://dx.doi.org/10.1186/1471-2164-8-131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Diverse+histone+modifications+on+histone+3+lysine+9+and+their+relation+to+DNA+methylation+in+specifying+gene+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Diverse+histone+modifications+on+histone+3+lysine+9+and+their+relation+to+DNA+methylation+in+specifying+gene+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001882-Chakrabarti1" id="pone.0001882-Chakrabarti1"></a>Chakrabarti SK, James JC, Mirmira RG (2002) Quantitative assessment of gene targeting in vitro and in vivo by the pancreatic transcription factor, Pdx1. Importance of chromatin structure in directing promoter binding. J Biol Chem  277: 13286–13293.  <ul class="find" data-citedArticleID="1082078" data-doi="10.1074/jbc.m111857200"><li><a href="http://dx.doi.org/10.1074/jbc.m111857200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+assessment+of+gene+targeting+in+vitro+and+in+vivo+by+the+pancreatic+transcription+factor%2C+Pdx1.+Importance+of+chromatin+structure+in+directing+promoter+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+assessment+of+gene+targeting+in+vitro+and+in+vivo+by+the+pancreatic+transcription+factor%2C+Pdx1.+Importance+of+chromatin+structure+in+directing+promoter+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001882-Oberley1" id="pone.0001882-Oberley1"></a>Oberley MJ, Tsao J, Yau P, Farnham PJ (2004) High-throughput screening of chromatin immunoprecipitates using CpG-island microarrays. Methods Enzymol  376: 315–334.  <ul class="find" data-citedArticleID="1082118" data-doi="10.1016/s0076-6879(03)76021-2"><li><a href="http://dx.doi.org/10.1016/s0076-6879(03)76021-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+screening+of+chromatin+immunoprecipitates+using+CpG-island+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+screening+of+chromatin+immunoprecipitates+using+CpG-island+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001882-Ehrich1" id="pone.0001882-Ehrich1"></a>Ehrich M, Nelson MR, Stanssens P, Zabeau M, Liloglou T, et al.  (2005) Quantitative high-throughput analysis of DNA methylation patterns by base-specific cleavage and mass spectrometry. Proc Natl Acad Sci U S A  102: 15785–15790.  <ul class="find" data-citedArticleID="1082082" data-doi="10.1073/pnas.0507816102"><li><a href="http://dx.doi.org/10.1073/pnas.0507816102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+high-throughput+analysis+of+DNA+methylation+patterns+by+base-specific+cleavage+and+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+high-throughput+analysis+of+DNA+methylation+patterns+by+base-specific+cleavage+and+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001882-Livak1" id="pone.0001882-Livak1"></a>Livak KJ, Schmittgen TD (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods  25: 402–408.  <ul class="find" data-citedArticleID="1082108" data-doi="10.1006/meth.2001.1262"><li><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+relative+gene+expression+data+using+real-time+quantitative+PCR+and+the+2%28-Delta+Delta+C%28T%29%29+Method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001882-Selzer1" id="pone.0001882-Selzer1"></a>Selzer RR, Richmond TA, Pofahl NJ, Green RD, Eis PS, et al.  (2005) Analysis of chromosome breakpoints in neuroblastoma at sub-kilobase resolution using fine-tiling oligonucleotide array CGH. Genes Chromosomes Cancer  44: 305–319.  <ul class="find" data-citedArticleID="1082126" data-doi="10.1002/gcc.20243"><li><a href="http://dx.doi.org/10.1002/gcc.20243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+chromosome+breakpoints+in+neuroblastoma+at+sub-kilobase+resolution+using+fine-tiling+oligonucleotide+array+CGH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+chromosome+breakpoints+in+neuroblastoma+at+sub-kilobase+resolution+using+fine-tiling+oligonucleotide+array+CGH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001882-Reimers1" id="pone.0001882-Reimers1"></a>Reimers M, Weinstein JN (2005) Quality assessment of microarrays: visualization of spatial artifacts and quantitation of regional biases. BMC Bioinformatics  6: 166.  <ul class="find" data-citedArticleID="1082122"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Quality+assessment+of+microarrays%3A+visualization+of+spatial+artifacts+and+quantitation+of+regional+biases.&amp;auth=&amp;atitle=Quality+assessment+of+microarrays%3A+visualization+of+spatial+artifacts+and+quantitation+of+regional+biases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quality+assessment+of+microarrays%3A+visualization+of+spatial+artifacts+and+quantitation+of+regional+biases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quality+assessment+of+microarrays%3A+visualization+of+spatial+artifacts+and+quantitation+of+regional+biases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001882-Eisen1" id="pone.0001882-Eisen1"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863–14868.  <ul class="find" data-citedArticleID="1082084" data-doi="10.1073/pnas.95.25.14863"><li><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+and+display+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+and+display+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001882-Fellenberg1" id="pone.0001882-Fellenberg1"></a>Fellenberg K, Hauser NC, Brors B, Neutzner A, Hoheisel JD, et al.  (2001) Correspondence analysis applied to microarray data. Proc Natl Acad Sci U S A  98: 10781–10786.  <ul class="find" data-citedArticleID="1082088" data-doi="10.1073/pnas.181597298"><li><a href="http://dx.doi.org/10.1073/pnas.181597298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correspondence+analysis+applied+to+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correspondence+analysis+applied+to+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001882-Misra1" id="pone.0001882-Misra1"></a>Misra J, Schmitt W, Hwang D, Hsiao LL, Gullans S, et al.  (2002) Interactive exploration of microarray gene expression patterns in a reduced dimensional space. Genome Res  12: 1112–1120.  <ul class="find" data-citedArticleID="1082116" data-doi="10.1101/gr.225302"><li><a href="http://dx.doi.org/10.1101/gr.225302" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interactive+exploration+of+microarray+gene+expression+patterns+in+a+reduced+dimensional+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interactive+exploration+of+microarray+gene+expression+patterns+in+a+reduced+dimensional+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001882-Team1" id="pone.0001882-Team1"></a>Team RDC (2007) R: A language and environment for statistical computing. 2.5.1 ed. Vienna: R Foundation for Statistical Computing.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001882-Culhane1" id="pone.0001882-Culhane1"></a>Culhane AC, Thioulouse J, Perriere G, Higgins DG (2005) MADE4: an R package for multivariate analysis of gene expression data. Bioinformatics  21: 2789–2790.  <ul class="find" data-citedArticleID="1082080" data-doi="10.1093/bioinformatics/bti394"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti394" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MADE4%3A+an+R+package+for+multivariate+analysis+of+gene+expression+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MADE4%3A+an+R+package+for+multivariate+analysis+of+gene+expression+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001882-Kent1" id="pone.0001882-Kent1"></a>Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, et al.  (2002) The human genome browser at UCSC. Genome Res  12: 996–1006.  <ul class="find" data-citedArticleID="1082102" data-doi="10.1101/gr.229102"><li><a href="http://dx.doi.org/10.1101/gr.229102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+genome+browser+at+UCSC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+genome+browser+at+UCSC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.XML" value="121856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.PDF" value="520500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t001.PNG_L" value="129181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t001.PNG_M" value="73583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t001.PNG_S" value="10420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t001.TIF" value="578184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t001.PNG_I" value="23207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g001.PNG_L" value="411070"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g001.PNG_M" value="57246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g001.PNG_S" value="8999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g001.TIF" value="1352050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g001.PNG_I" value="20909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g002.PNG_L" value="776264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g002.PNG_M" value="60179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g002.PNG_S" value="6769"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g002.TIF" value="1230796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g002.PNG_I" value="21467"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g003.PNG_L" value="530587"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g003.PNG_M" value="56660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g003.PNG_S" value="6356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g003.TIF" value="1161116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g003.PNG_I" value="20532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g004.PNG_L" value="644088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g004.PNG_M" value="53571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g004.PNG_S" value="10822"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g004.TIF" value="1602216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g004.PNG_I" value="49106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g005.PNG_L" value="1107819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g005.PNG_M" value="109319"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g005.PNG_S" value="9600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g005.TIF" value="2019510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g005.PNG_I" value="36806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g006.PNG_L" value="383643"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g006.PNG_M" value="75009"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g006.PNG_S" value="9207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g006.TIF" value="951702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g006.PNG_I" value="39964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g007.PNG_L" value="1477378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g007.PNG_M" value="111201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g007.PNG_S" value="10440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g007.TIF" value="3645814"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.g007.PNG_I" value="38752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t002.PNG_L" value="72781"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t002.PNG_M" value="43057"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t002.PNG_S" value="8537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t002.TIF" value="349520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t002.PNG_I" value="13119"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t003.PNG_L" value="44700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t003.PNG_M" value="58662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t003.PNG_S" value="10544"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t003.TIF" value="225018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.t003.PNG_I" value="18900"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s001.DOC" value="60416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s002.TIF" value="3773908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s003.TIF" value="1654994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s004.TIF" value="2666478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s005.TIF" value="3395116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s006.XLS" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001882.s007.XLS" value="47104"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001882&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001882" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001882&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001882&volume=&issue=&title=An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation&author_name=Maria%20E.%20Figueroa%2C%20Mark%20Reimers%2C%20Reid%20F.%20Thompson%2C%20Kenny%20Ye%2C%20Yushan%20Li%2C%20Rebecca%20R.%20Selzer%2C%20Jakob%20Fridriksson%2C%20Elisabeth%20Paietta%2C%20Peter%20Wiernik%2C%20Roland%20D.%20Green%2C%20John%20M.%20Greally%2C%20Ari%20Melnick&start_page=1&end_page=14" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001882" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882&amp;t=An%20Integrative%20Genomic%20and%20Epigenomic%20Approach%20for%20the%20Study%20of%20Transcriptional%20Regulation" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882&title=An%20Integrative%20Genomic%20and%20Epigenomic%20Approach%20for%20the%20Study%20of%20Transcriptional%20Regulation&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882&amp;title=An%20Integrative%20Genomic%20and%20Epigenomic%20Approach%20for%20the%20Study%20of%20Transcriptional%20Regulation" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001882&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'An Integrative Genomic and Epigenomic Approach for the Study of Transcriptional Regulation';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001882';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20An%20Integrative%20Genomic%20and%20Epigenomic%20Approach%20for%20the%20Study%20of%20Transcriptional%20Regulation http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001882" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001882" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Acetylation%22" title="Search for articles in the subject area:'Acetylation'"><div class="flagText">Acetylation</div></a>
              <div data-categoryid="21455" data-articleid="26588"
                   data-categoryname="Acetylation"
                   class="flagImage" title="Flag 'Acetylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="26588"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Epigenetics%22" title="Search for articles in the subject area:'Epigenetics'"><div class="flagText">Epigenetics</div></a>
              <div data-categoryid="18227" data-articleid="26588"
                   data-categoryname="Epigenetics"
                   class="flagImage" title="Flag 'Epigenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Epigenomics%22" title="Search for articles in the subject area:'Epigenomics'"><div class="flagText">Epigenomics</div></a>
              <div data-categoryid="37019" data-articleid="26588"
                   data-categoryname="Epigenomics"
                   class="flagImage" title="Flag 'Epigenomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="26588"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Histones%22" title="Search for articles in the subject area:'Histones'"><div class="flagText">Histones</div></a>
              <div data-categoryid="18573" data-articleid="26588"
                   data-categoryname="Histones"
                   class="flagImage" title="Flag 'Histones' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leukemias%22" title="Search for articles in the subject area:'Leukemias'"><div class="flagText">Leukemias</div></a>
              <div data-categoryid="20901" data-articleid="26588"
                   data-categoryname="Leukemias"
                   class="flagImage" title="Flag 'Leukemias' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="26588"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2737'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8847'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9526&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=13269">GEO accession number</a><br>Posted by kenfigueroa</p>
      <p><a href="/annotation/listThread.action?root=18517">GEO accession number</a><br>Posted by kenfigueroa</p>
      <p><a href="/annotation/listThread.action?root=12081">GEO accession number</a><br>Posted by kenfigueroa</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
