

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000322"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000322" />

    <meta name="citation_title" content="Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains"/>
    <meta itemprop="name" content="Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains"/>

      <meta name="citation_author" content="Joseph Schacherer"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Douglas M. Ruderfer"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="David Gresham"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Kara Dolinski"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="David Botstein"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America"/>
      <meta name="citation_author" content="Leonid Kruglyak"/>
            <meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America"/>

    <meta name="citation_date" content="2007/3/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000322.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e322"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genealogy of principal strains of the yeast genetic stock center.; citation_author=RK Mortimer; citation_author=JR Johnston; citation_journal_title=Genetics; citation_volume=113; citation_number=1; citation_pages=35-43; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Life with 6000 genes.; citation_author=A Goffeau; citation_author=BG Barrell; citation_author=H Bussey; citation_author=RW Davis; citation_author=B Dujon; citation_journal_title=Science; citation_volume=274; citation_number=2; citation_pages=546, 563-7; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Exploring the metabolic and genetic control of gene expression on a genomic scale.; citation_author=JL DeRisi; citation_author=VR Iyer; citation_author=PO Brown; citation_journal_title=Science; citation_volume=278; citation_number=3; citation_pages=680-686; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.; citation_author=EA Winzeler; citation_author=DD Shoemaker; citation_author=A Astromoff; citation_author=H Liang; citation_author=K Anderson; citation_journal_title=Science; citation_volume=285; citation_number=4; citation_pages=901-906; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.; citation_author=P Uetz; citation_author=L Giot; citation_author=G Cagney; citation_author=TA Mansfield; citation_author=RS Judson; citation_journal_title=Nature; citation_volume=403; citation_number=5; citation_pages=623-627; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Global mapping of the yeast genetic interaction network.; citation_author=AH Tong; citation_author=G Lesage; citation_author=GD Bader; citation_author=H Ding; citation_author=H Xu; citation_journal_title=Science; citation_volume=303; citation_number=6; citation_pages=808-813; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative trait loci mapped to single-nucleotide resolution in yeast.; citation_author=AM Deutschbauer; citation_author=RW Davis; citation_journal_title=Nat Genet; citation_volume=37; citation_number=7; citation_pages=1333-1340; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Structural and functional analysis of the MAL1 locus of Saccharomyces cerevisiae.; citation_author=MJ Charron; citation_author=RA Dubin; citation_author=CA Michels; citation_journal_title=Mol Cell Biol; citation_volume=6; citation_number=8; citation_pages=3891-3899; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=The sensing of nutritional status and the relationship to filamentous growth in Saccharomyces cerevisiae.; citation_author=M Gagiano; citation_author=FF Bauer; citation_author=IS Pretorius; citation_journal_title=FEMS Yeast Res; citation_volume=2; citation_number=9; citation_pages=433-470; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An interlaboratory comparison of physiological and genetic properties of four Saccharomyces cerevisiae strains.; citation_author=JP van Dijken; citation_author=J Bauer; citation_author=L Brambilla; citation_author=P Duboc; citation_author=JM Francois; citation_journal_title=Enzyme Microb Technol; citation_volume=26; citation_number=10; citation_pages=706-714; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The core meiotic transcriptome in budding yeasts.; citation_author=M Primig; citation_author=RM Williams; citation_author=EA Winzeler; citation_author=GG Tevzadze; citation_author=AR Conway; citation_journal_title=Nat Genet; citation_volume=26; citation_number=11; citation_pages=415-423; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mrc1 is required for sister chromatid cohesion to aid in recombination repair of spontaneous damage.; citation_author=H Xu; citation_author=C Boone; citation_author=HL Klein; citation_journal_title=Mol Cell Biol; citation_volume=24; citation_number=12; citation_pages=7082-7090; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rtt106p is a histone chaperone involved in heterochromatin-mediated silencing.; citation_author=S Huang; citation_author=H Zhou; citation_author=D Katzmann; citation_author=M Hochstrasser; citation_author=E Atanasova; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=13; citation_pages=13410-13415; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity in yeast assessed with whole-genome oligonucleotide arrays.; citation_author=EA Winzeler; citation_author=CI Castillo-Davis; citation_author=G Oshiro; citation_author=D Liang; citation_author=DR Richards; citation_journal_title=Genetics; citation_volume=163; citation_number=14; citation_pages=79-89; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray.; citation_author=D Gresham; citation_author=DM Ruderfer; citation_author=SC Pratt; citation_author=J Schacherer; citation_author=MJ Dunham; citation_journal_title=Science; citation_volume=311; citation_number=15; citation_pages=1932-1936; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Four linked genes participate in controlling sporulation efficiency in budding yeast.; citation_author=G Ben-Ari; citation_author=D Zenvirth; citation_author=A Sherman; citation_author=L David; citation_author=M Klutstein; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=16; citation_pages=e195; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Elements of the yeast pheromone response pathway required for filamentous growth of diploids.; citation_author=H Liu; citation_author=CA Styles; citation_author=GR Fink; citation_journal_title=Science; citation_volume=262; citation_number=17; citation_pages=1741-1744; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Macromolecule synthesis in temperature-sensitive mutants of yeast.; citation_author=LH Hartwell; citation_journal_title=J Bacteriol; citation_volume=93; citation_number=18; citation_pages=1662-1670; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.; citation_author=CB Brachmann; citation_author=A Davies; citation_author=GJ Cost; citation_author=E Caputo; citation_author=J Li; citation_journal_title=Yeast; citation_volume=14(2); citation_number=19; citation_pages=115-32; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A yeast centromere acts in cis to inhibit meiotic gene conversion of adjacent sequences.; citation_author=EJ Lambie; citation_author=GS Roeder; citation_journal_title=Cell; citation_volume=52; citation_number=20; citation_pages=863-873; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Clustering of meiotic double-strand breaks on yeast chromosome III.; citation_author=F Baudat; citation_author=A Nicolas; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=21; citation_pages=5213-5218; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inaugural article: global mapping of meiotic recombination hotspots and coldspots in the yeast Saccharomyces cerevisiae.; citation_author=JL Gerton; citation_author=J DeRisi; citation_author=R Shroff; citation_author=M Lichten; citation_author=PO Brown; citation_journal_title=Proc Natl Acad Sci U S A.; citation_volume=97; citation_number=22; citation_pages=11383-11390; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Principles of Population Genetics.; citation_author=DL Hartl; citation_author=AG Clark; citation_number=23; citation_date=1997; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=The subtelomeric Y' repeat family in Saccharomyces cerevisiae: an experimental system for repeated sequence evolution.; citation_author=EJ Louis; citation_author=JE Haber; citation_journal_title=Genetics; citation_volume=124; citation_number=24; citation_pages=533-545; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Chromosome ends: all the same under their caps.; citation_author=FE Pryde; citation_author=HC Gorham; citation_author=EJ Louis; citation_journal_title=Curr Opin Genet Dev; citation_volume=7; citation_number=25; citation_pages=822-828; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inferences of evolutionary relationships from a population survey of LTR-retrotransposons and telomeric-associated sequences in the Saccharomyces sensu stricto complex.; citation_author=G Liti; citation_author=A Peruffo; citation_author=SA James; citation_author=IN Roberts; citation_author=EJ Louis; citation_journal_title=Yeast; citation_volume=22; citation_number=26; citation_pages=177-192; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=27; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SGD: Saccharomyces Genome Database.; citation_author=JM Cherry; citation_author=C Adler; citation_author=C Ball; citation_author=SA Chervitz; citation_author=SS Dwight; citation_journal_title=Nucleic Acids Res; citation_volume=26; citation_number=28; citation_pages=73-79; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The generic genome browser: a building block for a model organism system database.; citation_author=LD Stein; citation_author=C Mungall; citation_author=S Shu; citation_author=M Caudy; citation_author=M Mangone; citation_journal_title=Genome Res.; citation_volume=12; citation_number=29; citation_pages=1599-1610; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genetic control of direct-repeat recombination in Saccharomyces: the effect of rad52 and rad1 on mitotic recombination at GAL10, a transcriptionally regulated gene.; citation_author=BJ Thomas; citation_author=R Rothstein; citation_journal_title=Genetics; citation_volume=123; citation_number=30; citation_pages=725-738; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Regulation of pyrimidine biosynthesis in Saccharomyces cerevisiae.; citation_author=F Lacroute; citation_journal_title=J Bacteriol; citation_volume=95; citation_number=31; citation_pages=824-832; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Multiplicity of the amino acid permeases in Saccharomyces cerevisiae. I. Evidence for a specific arginine-transporting system.; citation_author=M Grenson; citation_author=M Mousset; citation_author=JM Wiame; citation_author=J Bechet; citation_journal_title=Biochim Biophys Acta; citation_volume=127; citation_number=32; citation_pages=325-338; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=Carbohydrate metabolism during ascospore development in yeast.; citation_author=SM Kane; citation_author=R Roth; citation_journal_title=J Bacteriol; citation_volume=118; citation_number=33; citation_pages=8-14; citation_date=1974; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains"/>
    <meta name="twitter:description" content="Ten years have passed since the genome of Saccharomyces cerevisiae&ndash;more precisely, the S288c strain&ndash;was completely sequenced. However, experimental work in yeast is commonly performed using strains that are of unknown genetic relationship to S288c. Here, we characterized the nucleotide-level similarity between S288c and seven commonly used lab strains (A364A, W303, FL100, CEN.PK, &sum;1278b, SK1 and BY4716) using 25mer oligonucleotide microarrays that provide complete and redundant coverage of the &sim;12 Mb Saccharomyces cerevisiae genome. Using these data, we assessed the frequency and distribution of nucleotide variation in comparison to the sequenced reference genome. These data allow us to infer the relationships between experimentally important strains of yeast and provide insight for experimental designs that are sensitive to sequence variation. We propose a rational approach for near complete sequencing of strains related to the reference using these data and directed re-sequencing. These data and new visualization tools are accessible online in a new resource: the Yeast SNPs Browser (YSB; http://gbrowse.princeton.edu/cgi-bin/gbrowse/yeast_strains_snps) that is available to all researchers."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000322.g005"/>

  <meta property="og:title" content="Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5579'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6963'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9221&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000322">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000322" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000322&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used <i>Saccharomyces cerevisiae</i> Strains
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph Schacherer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America, 
                  Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Douglas M. Ruderfer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America, 
                  Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Gresham, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America, 
                  Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kara Dolinski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America, 
                  Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Botstein, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America, 
                  Department of Molecular Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leonid Kruglyak
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:leonid@genomics.princeton.edu">leonid@genomics.princeton.edu</a></p>

                <p>Affiliations:
                  Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, New Jersey, United States of America, 
                  Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000322</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000322-t001" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000322-g001" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000322-g002" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000322-g003" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000322-g004" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000322-t002" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000322-g005" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000322">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000322" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Ten years have passed since the genome of <em>Saccharomyces cerevisiae</em>–more precisely, the S288c strain–was completely sequenced. However, experimental work in yeast is commonly performed using strains that are of unknown genetic relationship to S288c. Here, we characterized the nucleotide-level similarity between S288c and seven commonly used lab strains (A364A, W303, FL100, CEN.PK, ∑1278b, SK1 and BY4716) using 25mer oligonucleotide microarrays that provide complete and redundant coverage of the ~12 Mb <em>Saccharomyces cerevisiae</em> genome. Using these data, we assessed the frequency and distribution of nucleotide variation in comparison to the sequenced reference genome. These data allow us to infer the relationships between experimentally important strains of yeast and provide insight for experimental designs that are sensitive to sequence variation. We propose a rational approach for near complete sequencing of strains related to the reference using these data and directed re-sequencing. These data and new visualization tools are accessible online in a new resource: the Yeast SNPs Browser (YSB; <a href="http://gbrowse.princeton.edu/cgi-bin/gbrowse/yeast_strains_snps">http://gbrowse.princeton.edu/cgi-bin/gbr​owse/yeast_strains_snps</a>) that is available to all researchers.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Schacherer J, Ruderfer DM, Gresham D, Dolinski K, Botstein D, et al.  (2007) Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used <em>Saccharomyces cerevisiae</em> Strains. PLoS ONE 2(3):
          e322.
            doi:10.1371/journal.pone.0000322</p><p><strong>Academic Editor: </strong>Justin Fay, Washington University in St. Louis School of Medicine, United States of America</p><p><strong>Received:</strong> January 8, 2007; <strong>Accepted:</strong> February 28, 2007; <strong>Published:</strong> March 28, 2007</p><p><strong>Copyright:</strong> © 2007 Schacherer et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by National Institute of General Medical Sciences grant R01 GM046406 and center grant P50 GM071508 (D.B.), NIH grant R37 MH059520 and James S. McDonnell Foundation grant 99-11T (L.K.).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>For decades, the budding yeast, <em>Saccharomyces cerevisiae</em>, has been used as a model organism for studying eukaryotic molecular and cell biology. Its primacy as a model system and its small and compact genome made it the best eukaryotic candidate for the early sequencing efforts. In 1996, the strain S288c, first isolated in the 1950's through genetic crosses by Robert Mortimer <a href="#pone.0000322-Mortimer1">[1]</a>, became the first eukaryote to be completely sequenced <a href="#pone.0000322-Goffeau1">[2]</a>. The determination of this reference sequence facilitated the construction of the first whole genome microarrays <a href="#pone.0000322-DeRisi1">[3]</a> and the systematic deletions of all genes <a href="#pone.0000322-Winzeler1">[4]</a>, as well as the first whole genome protein and genetic interaction maps <a href="#pone.0000322-Uetz1">[5]</a>, <a href="#pone.0000322-Tong1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Despite the fact that most of the genomic information has been obtained from S288c, this strain is often not the ideal genetic background for studying particular aspects of biology. In fact, the S288c background has a number of drawbacks, such as low sporulation efficiency <a href="#pone.0000322-Deutschbauer1">[7]</a>, an inability to grow on maltose <a href="#pone.0000322-Charron1">[8]</a> and failure to initiate filamentous growth upon nitrogen starvation <a href="#pone.0000322-Gagiano1">[9]</a>. For these reasons, other genetic backgrounds are used for physiological <a href="#pone.0000322-vanDijken1">[10]</a>, genetic and genomic analyses <a href="#pone.0000322-Primig1">[11]</a>–<a href="#pone.0000322-Huang1">[13]</a> in many laboratories. These other genetic backgrounds are sometimes of known close (e.g. A363A and W303) or distant (SK1) genetic relatedness to S288c. However, for some experimentally important strains the relationship is unclear and largely undocumented.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The use of these different strains may contribute to the inconsistencies in some biological results because it is often assumed that the genomic sequence information of S288c can be extrapolated to other strains. In fact, even for strains closely related to S288c, the small number of sequence differences may still have important consequences for different biological pathways and phenotypes. Therefore, understanding genetic differences between strains has become extremely important. Mortimer and Johnson <a href="#pone.0000322-Mortimer1">[1]</a> traced a genealogy of the commonly used <em>S. cerevisiae</em> strains from knowledge of their history. More recently, Winzeler <em>et al.</em> <a href="#pone.0000322-Winzeler2">[14]</a> determined a subset of DNA sequence variation among a set of 14 <em>S. cerevisiae</em> strains using low-coverage oligonucleotide arrays and discovered 11,115 sites of variation among them. Although this study provided some insight into allelic differences between yeast strains, it was limited by the proportion of the genome covered by the array (~16%) and the 25 bp resolution for localizing these variants.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We recently developed a method for characterizing nucleotide variation in the entire genome using 25mer oligonucleotide microarrays (Affymetrix yeast tiling arrays) that provide complete and redundant coverage of the ~12 Mb <em>S. cerevisiae</em> genome <a href="#pone.0000322-Gresham1">[15]</a>. This design provides for multiple measurements of each nucleotide's contribution to hybridization efficiency and therefore has the ability to detect the presence and location of single nucleotide polymorphisms (SNPs) and deletion events throughout the entire yeast genome with near nucleotide precision. We have employed this approach to characterize the nucleotide-level similarity and divergence between S288c and 7 commonly used lab strains (A364A, W303, FL100, CEN.PK, ∑1278b, SK1 and BY4716). The analyses revealed which genomic regions of each strain are derived from the reference strain and which regions are highly diverged, indicating a different ancestry. The data are presented in an online database, the Yeast SNPs Browser (YSB; <a href="http://gbrowse.princeton.edu/cgi-bin/gbrowse/yeast_strains_snps">http://gbrowse.princeton.edu/cgi-bin/gbr​owse/yeast_strains_snps</a>). YSB represents a valuable tool for the yeast community, enabling the development of genomic resources for other yeast strains and informing conventional molecular biology methods such as PCR primer and Southern blot probe design.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Frequency of SNPs between strains</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For this study, 7 unsequenced laboratory strains, A364A, W303, FL100, CEN.PK, ∑1278b, SK1 and BY4716, were chosen because of their importance and frequent use in research (<a href="#pone-0000322-t001">Table 1</a>). For example, the SK1 background is used for studying sporulation and meiosis <a href="#pone.0000322-Deutschbauer1">[7]</a>, <a href="#pone.0000322-BenAri1">[16]</a>, ∑1278b is used in pseudohyphal growth studies <a href="#pone.0000322-Liu1">[17]</a> and A364A is widely used in studies of the cell cycle <a href="#pone.0000322-Hartwell1">[18]</a>. As a control, we also included the strain BY4716 that is known to be nearly identical to S288c <a href="#pone.0000322-Brachmann1">[19]</a>.</p>
<div class="figure" id="pone-0000322-t001"><div class="img"><a name="pone-0000322-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Strains of <em>S. cerevisiae</em> used in this study</span></strong></p><span>doi:10.1371/journal.pone.0000322.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>To characterize the genomes of these unsequenced laboratory strains at the nucleotide level, we hybridized genomic DNA from each strain to a high-density Affymetrix yeast tiling microarray and detected potential SNPs using the software package <em>SNPscanner</em> as described <a href="#pone.0000322-Gresham1">[15]</a>. Where possible, we analyzed a prototrophic wildtype clone that was representative of the strain. The provenance of the strains is indicated in <a href="#pone-0000322-t001">Table 1</a>. <em>SNPscanner</em> detects SNPs based on up to 7 measurements of a nucleotide's effect on hybridization efficiency and calculates the log of the likelihood ratio (referred as the prediction signal) for the presence of a SNP at each nucleotide in the genome. The ability to detect a large number of SNPs in an entire diverged genome was tested in our previous study using the two completely sequenced strains, RM11_1a and YJM789. Analysis of a single hybridization of YJM789 with <em>SNPscanner</em> correctly detected 86.9% of 30,303 known SNPs at a prediction signal of 5, with only 7 false positives calls (i.e. a rate of 1 false positive/2 Mb) across the nonrepetitive portion of the genome.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We therefore extended this approach to obtain a nucleotide-resolution comparison of the commonly used laboratory strains with the S288c reference strain by analyzing data from a single hybridization of each strain. Using the same threshold settings (a prediction signal of 5 and heuristic filters) as reported previously <a href="#pone.0000322-Gresham1">[15]</a>, we found a moderately low number of predicted SNPs in the two strains closely related to S288c (4,894 calls in A364A and 7,955 calls in W303), whereas a higher number of SNPs was detected in the more distant strains (15,352 calls in FL100, 25,298 calls in ∑1278b and 37,424 calls in SK1) (<a href="#pone-0000322-t001">Table 1</a>). The divergence between the different strains and the reference varies between 0.05% to 0.36% of the genome. As expected, the number of detected SNPs for the BY4716 strain was very low (39 calls in total).</p>


<h4>Genome-wide distribution of SNPs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To identify strains that share a common recent ancestry with the reference strain, we examined the genome-wide distribution of variation for each strain. For strains derived from S288c, we expect a small number of SNPs nonrandomly distributed across the genome. By contrast, the strains unrelated to S288c should have a high number of SNPs distributed throughout the genome.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The distribution of SNPs in A364A, W303 (both known to be the result of crosses between S288c and unrelated strains) and CEN.PK is indeed nonrandom. The genomes are mosaics of large regions identical to S288c interspersed with small regions of high sequence divergence (<a href="#pone-0000322-g001">Fig. 1</a>). In the strains closely related to S288c, variation was found in a small fraction of the genome, as previously described <a href="#pone.0000322-Winzeler2">[14]</a> (<a href="#pone-0000322-g002">Fig. 2</a>). For example, 80% of the SNPs in A364A, W303 and CEN.PK are found in 10%, 15% and 18% of the genome, respectively. By contrast, variation in SK1 is distributed relatively evenly throughout the genome.</p>
<div class="figure" id="pone-0000322-g001"><div class="img"><a name="pone-0000322-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Prediction signal across chromosome 5 for the laboratory strains A364A, W303, Σ1278b and SK1.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A364A and W303 genomes are mosaics of regions diverged from S288C, as indicated by extended regions of positive prediction signal for polymorphisms, interspersed with extended regions of sequence identity or low polymorphism, indicating common ancestry or recent divergence. The genomes of Σ1278b and SK1 show extensive divergence for most or all regions.</p>
<span>doi:10.1371/journal.pone.0000322.g001</span></div><div class="figure" id="pone-0000322-g002"><div class="img"><a name="pone-0000322-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Genome-wide distribution of variation.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Curves represent the fraction of the genome (X-axis) that contains a given fraction of all polymorphism calls (Y-axis) for the strains A364A, W303, CEN.PK, FL100, Σ1278b and SK1, respectively (from left to right).</p>
<span>doi:10.1371/journal.pone.0000322.g002</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>For the strains that are clearly not close relatives of S288c, we asked if SNPs are unevenly distributed across the chromosomes. In all strains, variation is distributed across the genome, but decreases with proximity to the centromere (<a href="#pone-0000322-g003">Fig. 3</a>). The centromeric regions (within 20 kb of the centromeres) show a lower-than-average variability. This observation is possibly linked to the lack of DNA double-strand breaks (i.e the presence of meiotic recombination coldspots) near the centromeres <a href="#pone.0000322-Lambie1">[20]</a>–<a href="#pone.0000322-Gerton1">[22]</a>. Because of a reduced number of cross-overs and gene conversion events, these regions are less polymorphic, a result that is consistent with population genetic theory and observations in other organisms (e.g. Drosophila) <a href="#pone.0000322-Hartl1">[23]</a>.</p>
<div class="figure" id="pone-0000322-g003"><div class="img"><a name="pone-0000322-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Distribution of variation by distance from centromere.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>The height of each bar represents the number of SNPs detected in the 3 strains that are not close relatives of S288c (FL100, ∑1278b and SK1) in a 5 kb bin, starting at the centromere and moving up to 55 kb away in either direction, pooled across all chromosomes (that is, the first bin contains all SNPs within 0–5 kb of a centromere, the next bin contains all SNPs 5–10 kb from the centromere, and so on out to the last bin which contains all SNPs 55–60 kb from the centromere). The polymorphism rate is lower in the 25–30 kb closest to the centromere.</p>
<span>doi:10.1371/journal.pone.0000322.g003</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>By contrast, it is well known that subtelomeric regions that undergo frequent recombination events display increased variability at the sequence level <a href="#pone.0000322-Louis1">[24]</a>–<a href="#pone.0000322-Liti1">[26]</a>. However, these regions are rich in redundant sequences that are responsible for high levels of false positives using our strategy. In an effort to reduce false positives, <em>SNPscanner</em> removes all probes that are repeated on the array, thereby removing many of these regions from the initial analysis. With these regions removed, we fail to detect a higher proportion of genetic variability at the regions located near the telomere. When we then include these repeated regions, we see a marked increase in SNP calls near the telomere. However, we are currently unable to determine the percentage of these that are real versus false positives.</p>


<h4>Deletion variants in the surveyed strains</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As shown in our previous study <a href="#pone.0000322-Gresham1">[15]</a>, hybridization of genomic DNA to the yeast tiling arrays can be used to detect deletions and predict the breakpoints with a resolution of a few basepairs. Therefore, these data provide a precise and global view of the deletion events in the laboratory strains examined. To distinguish regions that are deleted from those that are present but highly polymorphic-situations that result in similar <em>SNPscanner</em> profiles-we restricted our analysis to deleted regions that are longer than 500 bp in size (<a href="#pone-0000322-t001">Table 1</a> and <a href="#pone.0000322.s001">S1</a>).</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>A total of 113 deletions met this criterion in a comparison between the reference strain and the other strains of this study. The number of deletions varied from 1 in the BY4716 strain to 36 in the SK1 strain. As expected, only the known engineered deletion of the <em>LYS2</em> gene was detected in the BY4716 strain. The mapped deletions range from 0.5 to 9.1 kb, with the majority falling between 0.5 and 1 kb (<a href="#pone-0000322-g004">Fig. 4</a>).</p>
<div class="figure" id="pone-0000322-g004"><div class="img"><a name="pone-0000322-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Size distribution of the deletion variants identified.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>To identify a confident set of the deletions, we limited analysis to those greater than 500 bp in size.</p>
<span>doi:10.1371/journal.pone.0000322.g004</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We identified all the genes that are involved in these deletion events. In total, 87 genes carry a whole or partial deletion in at least one strain (S1). Twenty-five genes have partial deletions, whereas 62 genes are deleted in full. We compared the function of the deleted genes on the basis of their Gene Ontology (GO) annotations <a href="#pone.0000322-Ashburner1">[27]</a> by using the GO Term Finder from the <em>Saccharomyces</em> genome database <a href="#pone.0000322-Cherry1">[28]</a>. We found no significant enrichment with respect to biological function. A large proportion of the deleted genes (~26%) have not yet been assigned a biological function. However, some of them play an important role in metabolism and are involved in transporter activity (<em>SEO1</em>, <em>ARR3</em>, <em>AGP3</em>, <em>MAL11</em>, <em>QCR8</em>, <em>HXT17</em>, <em>OPT2</em> and <em>SGE1</em>genes). All of these genes except for <em>QCR8</em> are located within 25 kb of the chromosome ends, consistent with a relationship between adaptation and subtelomeric regions, as previously reported <a href="#pone.0000322-Winzeler2">[14]</a>.</p>


<h4>Sequencing strategy for the related S288c strains</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Random shotgun whole genome sequencing is of unquestioned utility for de novo determination of an organism's genome. However, once a reference genome sequence has been determined for a species, applying this approach to re-sequence other closely related members of the population represents a redundant and inefficient approach. Although our method for comparing genomes at nucleotide resolution does not provide the actual DNA sequence of a genome, it does offer one approach to a more efficient resequencing of closely related strains. In this scheme, we suggest that initial determination of regions that are diverged from the reference genome using <em>SNPscanner</em> can then be followed by targeted re-sequencing of these regions.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>We propose that this approach is applicable for strains A364A, W303 and CEN.PK, where the majority of variation was found in a small fraction (&lt;20%) of the genome. By selecting and sequencing only the 1 kb regions of high diversity as identified by <em>SNPscanner</em>, we can obtain sequence information for 90% of a yeast strain's genome by sequencing only 1–3 megabases, substantially minimizing cost and effort (<a href="#pone-0000322-t002">Table 2</a>). Although these strains are experimentally important, it is unlikely that they will be of high priority for high coverage sequencing projects. Therefore, we propose that this approach will be informative until whole genome sequencing is greatly reduced in cost. This approach is limited to strains that are clear relatives of S288c, with much of their genomes being identical and the differences clustered into small regions. In addition, this approach will not identify regions of the genome (insertions) that are present in the sample strain but unrepresented in the reference strain.</p>
<div class="figure" id="pone-0000322-t002"><div class="img"><a name="pone-0000322-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Sequencing strategy for the S288c related strains</span></strong></p><span>doi:10.1371/journal.pone.0000322.t002</span></div>

<h4>The Yeast SNPs Browser</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To allow researchers to easily visualize and explore these data we have created a database, the Yeast SNPs Browser, which can be used to view all the SNPs and deletions distinguishing these strains, along with the prediction signal across every base in the genome. We used the Generic Genome Browser (GBrowse), a web-based application for displaying chromosomal features <a href="#pone.0000322-Stein1">[29]</a>, to provide user-friendly access to the data via the web. The Yeast SNPs Browser provides a way to view the putative SNPs in all the strains assayed in the context of the S288C genome. One can search by gene, feature name or oligonucleotide sequence, browse chromosomal regions, and zoom from an overview of an entire chromosome to the individual nucleotide level (<a href="#pone-0000322-g005">Fig. 5</a>). By zooming in to about 100 bp resolution, one can see the nucleotide sequence of the S288C reference strain in the region of the SNP. In addition, the prediction score is available as a graph across every base in the genome (<a href="#pone-0000322-g005">Fig. 5</a>).</p>
<div class="figure" id="pone-0000322-g005"><div class="img"><a name="pone-0000322-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000322" data-uri="info:doi/10.1371/journal.pone.0000322.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000322.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000322.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000322.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000322.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Yeast SNPs Browser screenshots.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Overview track of the entire chromosome IX for the FL100 and SK1 strains, showing the prediction signal across the chromosome. (B) Zoom-in of the region that includes the <em>MUC1</em> gene, showing the SNPs called and the associated prediction scores.</p>
<span>doi:10.1371/journal.pone.0000322.g005</span></div>

<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>There are several <em>S. cerevisiae</em> strains that are commonly used in laboratory experiments, some of which are distantly related to the sequenced reference strain S288c. Understanding this variation is important for interpreting biological results from experiments performed with these strains. Using yeast tiling arrays, we have defined the genetic variation between the common laboratory strains of yeast at the nucleotide level in a reliable, cost-effective manner. We have also created a simple and visually pleasing resource for researchers. These data provide a valuable tool for the yeast community. They should inform experimental work that is reliant on knowledge of sequence such as the design of primers for PCR reactions and probes for Southern analysis. For studies of gene expression in the different laboratory strains, these data can also be useful as a source of information to design a “universal” microarray by selecting probes in regions of complete or high sequence similarity, in order to avoid any potential confounding effects on hybridization of strain-specific sequence variation. Finally, these data constitute the first step toward discovering the underlying genetic differences that contribute to the phenotypic differences among these strains.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Yeast strains</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Strains used in this study are listed in <a href="#pone-0000322-t001">Table 1</a>.</p>


<h4>Sample preparation and hybridization on arrays</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Yeast strains were grown in yeast extract, peptone, and dextrose (YPD) medium. Total genomic DNA was purified from 30 ml YPD culture using Qiagen Genomic-Tips 100/G and Genomic DNA Buffers as per the manufacturer's instructions. Genomic DNA was digested with DNaseI, labeled and hybrizided to Affymetrix Yeast Tiling Arrays as described in Gresham <em>et al.</em> <a href="#pone.0000322-Gresham1">[15]</a>.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000322"></div><a name="pone.0000322.s001" id="pone.0000322.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000322.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Genes carrying a whole or partial deletion in the surveyed strains</p>
<p class="siDoi">doi:10.1371/journal.pone.0000322.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors gratefully acknowledge helpful discussions with members of the Botstein, Kruglyak and Dunham laboratories. We thank J. Matese and D. Storton for technical support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: DB LK DG JS. Performed the experiments: DG JS. Analyzed the data: KD DR JS. Wrote the paper: LK DR DG JS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000322-Mortimer1" id="pone.0000322-Mortimer1"></a>Mortimer RK, Johnston JR (1986) Genealogy of principal strains of the yeast genetic stock center. Genetics  113: 35–43.  <ul class="find" data-citedArticleID="940108"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genealogy+of+principal+strains+of+the+yeast+genetic+stock+center.&amp;auth=&amp;atitle=Genealogy+of+principal+strains+of+the+yeast+genetic+stock+center." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genealogy+of+principal+strains+of+the+yeast+genetic+stock+center." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genealogy+of+principal+strains+of+the+yeast+genetic+stock+center.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000322-Goffeau1" id="pone.0000322-Goffeau1"></a>Goffeau A, Barrell BG, Bussey H, Davis RW, Dujon B, et al.  (1996) Life with 6000 genes. Science  274: 546, 563–7.  <ul class="find" data-citedArticleID="940084" data-doi="10.1126/science.274.5287.546"><li><a href="http://dx.doi.org/10.1126/science.274.5287.546" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life+with+6000+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life+with+6000+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000322-DeRisi1" id="pone.0000322-DeRisi1"></a>DeRisi JL, Iyer VR, Brown PO (1997) Exploring the metabolic and genetic control of gene expression on a genomic scale. Science  278: 680–686.  <ul class="find" data-citedArticleID="940076" data-doi="10.1126/science.278.5338.680"><li><a href="http://dx.doi.org/10.1126/science.278.5338.680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+metabolic+and+genetic+control+of+gene+expression+on+a+genomic+scale." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+metabolic+and+genetic+control+of+gene+expression+on+a+genomic+scale.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000322-Winzeler1" id="pone.0000322-Winzeler1"></a>Winzeler EA, Shoemaker DD, Astromoff A, Liang H, Anderson K, et al.  (1999) Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis. Science  285: 901–906.  <ul class="find" data-citedArticleID="940122" data-doi="10.1126/science.285.5429.901"><li><a href="http://dx.doi.org/10.1126/science.285.5429.901" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+characterization+of+the+S.+cerevisiae+genome+by+gene+deletion+and+parallel+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+characterization+of+the+S.+cerevisiae+genome+by+gene+deletion+and+parallel+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000322-Uetz1" id="pone.0000322-Uetz1"></a>Uetz P, Giot L, Cagney G, Mansfield TA, Judson RS, et al.  (2000) A comprehensive analysis of protein-protein interactions in <em>Saccharomyces cerevisiae</em>. Nature  403: 623–627.  <ul class="find" data-citedArticleID="940120" data-doi="10.1038/35001009"><li><a href="http://dx.doi.org/10.1038/35001009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+analysis+of+protein-protein+interactions+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000322-Tong1" id="pone.0000322-Tong1"></a>Tong AH, Lesage G, Bader GD, Ding H, Xu H, et al.  (2004) Global mapping of the yeast genetic interaction network. Science  303: 808–813.  <ul class="find" data-citedArticleID="940118" data-doi="10.1126/science.1091317"><li><a href="http://dx.doi.org/10.1126/science.1091317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+mapping+of+the+yeast+genetic+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+mapping+of+the+yeast+genetic+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000322-Deutschbauer1" id="pone.0000322-Deutschbauer1"></a>Deutschbauer AM, Davis RW (2005) Quantitative trait loci mapped to single-nucleotide resolution in yeast. Nat Genet  37: 1333–1340.  <ul class="find" data-citedArticleID="940078" data-doi="10.1038/ng1674"><li><a href="http://dx.doi.org/10.1038/ng1674" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+trait+loci+mapped+to+single-nucleotide+resolution+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+trait+loci+mapped+to+single-nucleotide+resolution+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000322-Charron1" id="pone.0000322-Charron1"></a>Charron MJ, Dubin RA, Michels CA (1986) Structural and functional analysis of the MAL1 locus of Saccharomyces cerevisiae. Mol Cell Biol  6: 3891–3899.  <ul class="find" data-citedArticleID="940072"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Structural+and+functional+analysis+of+the+MAL1+locus+of+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Structural+and+functional+analysis+of+the+MAL1+locus+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+functional+analysis+of+the+MAL1+locus+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+functional+analysis+of+the+MAL1+locus+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000322-Gagiano1" id="pone.0000322-Gagiano1"></a>Gagiano M, Bauer FF, Pretorius IS (2002) The sensing of nutritional status and the relationship to filamentous growth in <em>Saccharomyces cerevisiae</em>. FEMS Yeast Res  2: 433–470.  <ul class="find" data-citedArticleID="940080" data-doi="10.1111/j.1567-1364.2002.tb00114.x"><li><a href="http://dx.doi.org/10.1111/j.1567-1364.2002.tb00114.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sensing+of+nutritional+status+and+the+relationship+to+filamentous+growth+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sensing+of+nutritional+status+and+the+relationship+to+filamentous+growth+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000322-vanDijken1" id="pone.0000322-vanDijken1"></a>van Dijken JP, Bauer J, Brambilla L, Duboc P, Francois JM, et al.  (2000) An interlaboratory comparison of physiological and genetic properties of four <em>Saccharomyces cerevisiae</em> strains. Enzyme Microb Technol  26: 706–714.  <ul class="find" data-citedArticleID="940128" data-doi="10.1016/s0141-0229(00)00162-9"><li><a href="http://dx.doi.org/10.1016/s0141-0229(00)00162-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+interlaboratory+comparison+of+physiological+and+genetic+properties+of+four+Saccharomyces+cerevisiae+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+interlaboratory+comparison+of+physiological+and+genetic+properties+of+four+Saccharomyces+cerevisiae+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000322-Primig1" id="pone.0000322-Primig1"></a>Primig M, Williams RM, Winzeler EA, Tevzadze GG, Conway AR, et al.  (2000) The core meiotic transcriptome in budding yeasts. Nat Genet  26: 415–423.  <ul class="find" data-citedArticleID="940110" data-doi="10.1038/82539"><li><a href="http://dx.doi.org/10.1038/82539" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+core+meiotic+transcriptome+in+budding+yeasts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+core+meiotic+transcriptome+in+budding+yeasts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000322-Xu1" id="pone.0000322-Xu1"></a>Xu H, Boone C, Klein HL (2004) Mrc1 is required for sister chromatid cohesion to aid in recombination repair of spontaneous damage. Mol Cell Biol  24: 7082–7090.  <ul class="find" data-citedArticleID="940126" data-doi="10.1128/mcb.24.16.7082-7090.2004"><li><a href="http://dx.doi.org/10.1128/mcb.24.16.7082-7090.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mrc1+is+required+for+sister+chromatid+cohesion+to+aid+in+recombination+repair+of+spontaneous+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mrc1+is+required+for+sister+chromatid+cohesion+to+aid+in+recombination+repair+of+spontaneous+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000322-Huang1" id="pone.0000322-Huang1"></a>Huang S, Zhou H, Katzmann D, Hochstrasser M, Atanasova E, et al.  (2005) Rtt106p is a histone chaperone involved in heterochromatin-mediated silencing. Proc Natl Acad Sci U S A  102: 13410–13415.  <ul class="find" data-citedArticleID="940094" data-doi="10.1073/pnas.0506176102"><li><a href="http://dx.doi.org/10.1073/pnas.0506176102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rtt106p+is+a+histone+chaperone+involved+in+heterochromatin-mediated+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rtt106p+is+a+histone+chaperone+involved+in+heterochromatin-mediated+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000322-Winzeler2" id="pone.0000322-Winzeler2"></a>Winzeler EA, Castillo-Davis CI, Oshiro G, Liang D, Richards DR, et al.  (2003) Genetic diversity in yeast assessed with whole-genome oligonucleotide arrays. Genetics  163: 79–89.  <ul class="find" data-citedArticleID="940124"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays.&amp;auth=&amp;atitle=Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+in+yeast+assessed+with+whole-genome+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000322-Gresham1" id="pone.0000322-Gresham1"></a>Gresham D, Ruderfer DM, Pratt SC, Schacherer J, Dunham MJ, et al.  (2006) Genome-wide detection of polymorphisms at nucleotide resolution with a single DNA microarray. Science  311: 1932–1936.  <ul class="find" data-citedArticleID="940088" data-doi="10.1126/science.1123726"><li><a href="http://dx.doi.org/10.1126/science.1123726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+detection+of+polymorphisms+at+nucleotide+resolution+with+a+single+DNA+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+detection+of+polymorphisms+at+nucleotide+resolution+with+a+single+DNA+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000322-BenAri1" id="pone.0000322-BenAri1"></a>Ben-Ari G, Zenvirth D, Sherman A, David L, Klutstein M, et al.  (2006) Four linked genes participate in controlling sporulation efficiency in budding yeast. PLoS Genet  2: e195.  <ul class="find" data-citedArticleID="940068" data-doi="10.1371/journal.pgen.0020195.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020195.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Four+linked+genes+participate+in+controlling+sporulation+efficiency+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Four+linked+genes+participate+in+controlling+sporulation+efficiency+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000322-Liu1" id="pone.0000322-Liu1"></a>Liu H, Styles CA, Fink GR (1993) Elements of the yeast pheromone response pathway required for filamentous growth of diploids. Science  262: 1741–1744.  <ul class="find" data-citedArticleID="940104" data-doi="10.1126/science.8259520"><li><a href="http://dx.doi.org/10.1126/science.8259520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elements+of+the+yeast+pheromone+response+pathway+required+for+filamentous+growth+of+diploids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elements+of+the+yeast+pheromone+response+pathway+required+for+filamentous+growth+of+diploids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000322-Hartwell1" id="pone.0000322-Hartwell1"></a>Hartwell LH (1967) Macromolecule synthesis in temperature-sensitive mutants of yeast. J Bacteriol  93: 1662–1670.  <ul class="find" data-citedArticleID="940092"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Macromolecule+synthesis+in+temperature-sensitive+mutants+of+yeast.&amp;auth=&amp;atitle=Macromolecule+synthesis+in+temperature-sensitive+mutants+of+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Macromolecule+synthesis+in+temperature-sensitive+mutants+of+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Macromolecule+synthesis+in+temperature-sensitive+mutants+of+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000322-Brachmann1" id="pone.0000322-Brachmann1"></a>Brachmann CB, Davies A, Cost GJ, Caputo E, Li J, et al.  (1998) Designer deletion strains derived from <em>Saccharomyces cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications. Yeast  14(2): 115–32.  <ul class="find" data-citedArticleID="940070"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Designer+deletion+strains+derived+from+Saccharomyces+cerevisiae+S288C%3A+a+useful+set+of+strains+and+plasmids+for+PCR-mediated+gene+disruption+and+other+applications.&amp;auth=&amp;atitle=Designer+deletion+strains+derived+from+Saccharomyces+cerevisiae+S288C%3A+a+useful+set+of+strains+and+plasmids+for+PCR-mediated+gene+disruption+and+other+applications." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Designer+deletion+strains+derived+from+Saccharomyces+cerevisiae+S288C%3A+a+useful+set+of+strains+and+plasmids+for+PCR-mediated+gene+disruption+and+other+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Designer+deletion+strains+derived+from+Saccharomyces+cerevisiae+S288C%3A+a+useful+set+of+strains+and+plasmids+for+PCR-mediated+gene+disruption+and+other+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000322-Lambie1" id="pone.0000322-Lambie1"></a>Lambie EJ, Roeder GS (1988) A yeast centromere acts in cis to inhibit meiotic gene conversion of adjacent sequences. Cell  52: 863–873.  <ul class="find" data-citedArticleID="940100" data-doi="10.1016/0092-8674(88)90428-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(88)90428-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+yeast+centromere+acts+in+cis+to+inhibit+meiotic+gene+conversion+of+adjacent+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+yeast+centromere+acts+in+cis+to+inhibit+meiotic+gene+conversion+of+adjacent+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000322-Baudat1" id="pone.0000322-Baudat1"></a>Baudat F, Nicolas A (1997) Clustering of meiotic double-strand breaks on yeast chromosome III. Proc Natl Acad Sci U S A  94: 5213–5218.  <ul class="find" data-citedArticleID="940066" data-doi="10.1073/pnas.94.10.5213"><li><a href="http://dx.doi.org/10.1073/pnas.94.10.5213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+of+meiotic+double-strand+breaks+on+yeast+chromosome+III." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+of+meiotic+double-strand+breaks+on+yeast+chromosome+III.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000322-Gerton1" id="pone.0000322-Gerton1"></a>Gerton JL, DeRisi J, Shroff R, Lichten M, Brown PO, et al.  (2000) Inaugural article: global mapping of meiotic recombination hotspots and coldspots in the yeast <em>Saccharomyces cerevisiae</em>. Proc Natl Acad Sci U S A.  97: 11383–11390.  <ul class="find" data-citedArticleID="940082" data-doi="10.1073/pnas.97.21.11383"><li><a href="http://dx.doi.org/10.1073/pnas.97.21.11383" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inaugural+article%3A+global+mapping+of+meiotic+recombination+hotspots+and+coldspots+in+the+yeast+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inaugural+article%3A+global+mapping+of+meiotic+recombination+hotspots+and+coldspots+in+the+yeast+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000322-Hartl1" id="pone.0000322-Hartl1"></a>Hartl DL, Clark AG (1997) Principles of Population Genetics. Sunderland, Massachusetts: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0000322-Louis1" id="pone.0000322-Louis1"></a>Louis EJ, Haber JE (1990) The subtelomeric Y' repeat family in <em>Saccharomyces cerevisiae</em>: an experimental system for repeated sequence evolution. Genetics  124: 533–545.  <ul class="find" data-citedArticleID="940106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+subtelomeric+Y%27+repeat+family+in+Saccharomyces+cerevisiae%3A+an+experimental+system+for+repeated+sequence+evolution.&amp;auth=&amp;atitle=The+subtelomeric+Y%27+repeat+family+in+Saccharomyces+cerevisiae%3A+an+experimental+system+for+repeated+sequence+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+subtelomeric+Y%27+repeat+family+in+Saccharomyces+cerevisiae%3A+an+experimental+system+for+repeated+sequence+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+subtelomeric+Y%27+repeat+family+in+Saccharomyces+cerevisiae%3A+an+experimental+system+for+repeated+sequence+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000322-Pryde1" id="pone.0000322-Pryde1"></a>Pryde FE, Gorham HC, Louis EJ (1997) Chromosome ends: all the same under their caps. Curr Opin Genet Dev  7: 822–828.  <ul class="find" data-citedArticleID="940112" data-doi="10.1016/s0959-437x(97)80046-9"><li><a href="http://dx.doi.org/10.1016/s0959-437x(97)80046-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+ends%3A+all+the+same+under+their+caps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+ends%3A+all+the+same+under+their+caps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000322-Liti1" id="pone.0000322-Liti1"></a>Liti G, Peruffo A, James SA, Roberts IN, Louis EJ (2005) Inferences of evolutionary relationships from a population survey of LTR-retrotransposons and telomeric-associated sequences in the <em>Saccharomyces</em> sensu stricto complex. Yeast  22: 177–192.  <ul class="find" data-citedArticleID="940102"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inferences+of+evolutionary+relationships+from+a+population+survey+of+LTR-retrotransposons+and+telomeric-associated+sequences+in+the+Saccharomyces+sensu+stricto+complex.&amp;auth=&amp;atitle=Inferences+of+evolutionary+relationships+from+a+population+survey+of+LTR-retrotransposons+and+telomeric-associated+sequences+in+the+Saccharomyces+sensu+stricto+complex." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferences+of+evolutionary+relationships+from+a+population+survey+of+LTR-retrotransposons+and+telomeric-associated+sequences+in+the+Saccharomyces+sensu+stricto+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferences+of+evolutionary+relationships+from+a+population+survey+of+LTR-retrotransposons+and+telomeric-associated+sequences+in+the+Saccharomyces+sensu+stricto+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000322-Ashburner1" id="pone.0000322-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="940064"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000322-Cherry1" id="pone.0000322-Cherry1"></a>Cherry JM, Adler C, Ball C, Chervitz SA, Dwight SS, et al.  (1998) SGD: <em>Saccharomyces</em> Genome Database. Nucleic Acids Res  26: 73–79.  <ul class="find" data-citedArticleID="940074" data-doi="10.1093/nar/26.1.73"><li><a href="http://dx.doi.org/10.1093/nar/26.1.73" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SGD%3A+Saccharomyces+Genome+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SGD%3A+Saccharomyces+Genome+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000322-Stein1" id="pone.0000322-Stein1"></a>Stein LD, Mungall C, Shu S, Caudy M, Mangone M, et al.  (2002) The generic genome browser: a building block for a model organism system database. Genome Res.  12: 1599–1610.  <ul class="find" data-citedArticleID="940114" data-doi="10.1101/gr.403602"><li><a href="http://dx.doi.org/10.1101/gr.403602" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+generic+genome+browser%3A+a+building+block+for+a+model+organism+system+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+generic+genome+browser%3A+a+building+block+for+a+model+organism+system+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000322-Thomas1" id="pone.0000322-Thomas1"></a>Thomas BJ, Rothstein R (1989) The genetic control of direct-repeat recombination in <em>Saccharomyces</em>: the effect of rad52 and rad1 on mitotic recombination at GAL10, a transcriptionally regulated gene. Genetics  123: 725–738.  <ul class="find" data-citedArticleID="940116"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+genetic+control+of+direct-repeat+recombination+in+Saccharomyces%3A+the+effect+of+rad52+and+rad1+on+mitotic+recombination+at+GAL10%2C+a+transcriptionally+regulated+gene.&amp;auth=&amp;atitle=The+genetic+control+of+direct-repeat+recombination+in+Saccharomyces%3A+the+effect+of+rad52+and+rad1+on+mitotic+recombination+at+GAL10%2C+a+transcriptionally+regulated+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+control+of+direct-repeat+recombination+in+Saccharomyces%3A+the+effect+of+rad52+and+rad1+on+mitotic+recombination+at+GAL10%2C+a+transcriptionally+regulated+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+control+of+direct-repeat+recombination+in+Saccharomyces%3A+the+effect+of+rad52+and+rad1+on+mitotic+recombination+at+GAL10%2C+a+transcriptionally+regulated+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000322-Lacroute1" id="pone.0000322-Lacroute1"></a>Lacroute F (1968) Regulation of pyrimidine biosynthesis in <em>Saccharomyces</em> <em>cerevisiae</em>. J Bacteriol  95: 824–832.  <ul class="find" data-citedArticleID="940098"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+pyrimidine+biosynthesis+in+Saccharomycescerevisiae.&amp;auth=&amp;atitle=Regulation+of+pyrimidine+biosynthesis+in+Saccharomycescerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+pyrimidine+biosynthesis+in+Saccharomycescerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+pyrimidine+biosynthesis+in+Saccharomycescerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000322-Grenson1" id="pone.0000322-Grenson1"></a>Grenson M, Mousset M, Wiame JM, Bechet J (1966) Multiplicity of the amino acid permeases in <em>Saccharomyces cerevisiae</em>. I. Evidence for a specific arginine-transporting system. Biochim Biophys Acta  127: 325–338.  <ul class="find" data-citedArticleID="940086" data-doi="10.1016/0304-4165(66)90387-4"><li><a href="http://dx.doi.org/10.1016/0304-4165(66)90387-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplicity+of+the+amino+acid+permeases+in+Saccharomyces+cerevisiae.+I.+Evidence+for+a+specific+arginine-transporting+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplicity+of+the+amino+acid+permeases+in+Saccharomyces+cerevisiae.+I.+Evidence+for+a+specific+arginine-transporting+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000322-Kane1" id="pone.0000322-Kane1"></a>Kane SM, Roth R (1974) Carbohydrate metabolism during ascospore development in yeast. J Bacteriol  118: 8–14.  <ul class="find" data-citedArticleID="940096"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Carbohydrate+metabolism+during+ascospore+development+in+yeast.&amp;auth=&amp;atitle=Carbohydrate+metabolism+during+ascospore+development+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Carbohydrate+metabolism+during+ascospore+development+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Carbohydrate+metabolism+during+ascospore+development+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.XML" value="63812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.PDF" value="423695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t001.PNG_L" value="108274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t001.PNG_M" value="39268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t001.PNG_S" value="2372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t001.TIF" value="504306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t001.PNG_I" value="17107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g001.PNG_L" value="548892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g001.PNG_M" value="14364"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g001.PNG_S" value="1794"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g001.TIF" value="1242256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g001.PNG_I" value="8878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g002.PNG_L" value="62097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g002.PNG_M" value="41366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g002.PNG_S" value="2181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g002.TIF" value="207836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g002.PNG_I" value="18324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g003.PNG_L" value="22563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g003.PNG_M" value="11818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g003.PNG_S" value="1859"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g003.TIF" value="85652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g003.PNG_I" value="5370"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g004.PNG_L" value="12133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g004.PNG_M" value="18949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g004.PNG_S" value="1694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g004.TIF" value="125960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g004.PNG_I" value="8951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t002.PNG_L" value="31594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t002.PNG_M" value="30620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t002.PNG_S" value="2220"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t002.TIF" value="167286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.t002.PNG_I" value="14604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g005.PNG_L" value="94465"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g005.PNG_M" value="76187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g005.PNG_S" value="3854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g005.TIF" value="1067654"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.g005.PNG_I" value="37462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000322.s001.DOC" value="24522"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000322&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000322" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000322&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000322&volume=&issue=&title=Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains&author_name=Joseph%20Schacherer%2C%20Douglas%20M.%20Ruderfer%2C%20David%20Gresham%2C%20Kara%20Dolinski%2C%20David%20Botstein%2C%20Leonid%20Kruglyak&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000322" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322&amp;t=Genome-Wide%20Analysis%20of%20Nucleotide-Level%20Variation%20in%20Commonly%20Used%20Saccharomyces%20cerevisiae%20Strains" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322&title=Genome-Wide%20Analysis%20of%20Nucleotide-Level%20Variation%20in%20Commonly%20Used%20Saccharomyces%20cerevisiae%20Strains&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322&amp;title=Genome-Wide%20Analysis%20of%20Nucleotide-Level%20Variation%20in%20Commonly%20Used%20Saccharomyces%20cerevisiae%20Strains" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000322&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genome-Wide Analysis of Nucleotide-Level Variation in Commonly Used Saccharomyces cerevisiae Strains';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000322';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genome-Wide%20Analysis%20of%20Nucleotide-Level%20Variation%20in%20Commonly%20Used%20Saccharomyces%20cerevisiae%20Strains http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000322" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000322" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23482"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+analysis%22" title="Search for articles in the subject area:'Genome analysis'"><div class="flagText">Genome analysis</div></a>
              <div data-categoryid="40539" data-articleid="23482"
                   data-categoryname="Genome analysis"
                   class="flagImage" title="Flag 'Genome analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23482"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="23482"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotide+sequencing%22" title="Search for articles in the subject area:'Nucleotide sequencing'"><div class="flagText">Nucleotide sequencing</div></a>
              <div data-categoryid="19663" data-articleid="23482"
                   data-categoryname="Nucleotide sequencing"
                   class="flagImage" title="Flag 'Nucleotide sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nucleotides%22" title="Search for articles in the subject area:'Nucleotides'"><div class="flagText">Nucleotides</div></a>
              <div data-categoryid="17443" data-articleid="23482"
                   data-categoryname="Nucleotides"
                   class="flagImage" title="Flag 'Nucleotides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="32473" data-articleid="23482"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="23482"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4594'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=731'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7446&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
