

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001245"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001245" />

    <meta name="citation_title" content="A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry"/>
    <meta itemprop="name" content="A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry"/>

      <meta name="citation_author" content="Hongying Li"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Zhongwen Huang"/>
            <meta name="citation_author_institution" content="National Center for Soybean Improvement, Nanjing Agricultural University, Nanjing, Jiangsu, People’s Republic of China"/>
            <meta name="citation_author_institution" content="Department of Agronomy, Henan Institute of Science and Technology, Xinxiang, Henan, People’s Republic of China"/>
      <meta name="citation_author" content="Junyi Gai"/>
            <meta name="citation_author_institution" content="National Center for Soybean Improvement, Nanjing Agricultural University, Nanjing, Jiangsu, People’s Republic of China"/>
      <meta name="citation_author" content="Song Wu"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Yanru Zeng"/>
            <meta name="citation_author_institution" content="School of Forestry and Biotechnology, Zhejiang Forestry University, Lin’an, Zhejiang, People’s Republic of China"/>
      <meta name="citation_author" content="Qin Li"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Rongling Wu"/>
            <meta name="citation_author_institution" content="Department of Statistics, University of Florida, Gainesville, Florida, United States of America"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001245.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1245"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Body size and metabolism.; citation_author=M Kleiber; citation_journal_title=Hilgardia; citation_volume=6; citation_number=1; citation_pages=315-353; citation_date=1932; " />
      <meta name="citation_reference" content="citation_author=WA Calder; citation_number=2; citation_date=1984; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_title=A general model for the structure, and allometry of plant vascular systems.; citation_author=GB West; citation_author=JH Brown; citation_author=BJ Enquist; citation_journal_title=Nature; citation_volume=400; citation_number=3; citation_pages=664-667; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Invariant scaling relations across tree-dominated communities.; citation_author=BJ Enquist; citation_author=KJ Niklas; citation_journal_title=Nature; citation_volume=410; citation_number=4; citation_pages=655-660; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Global allocation rules for patterns of biomass partitioning across seed plants.; citation_author=BJ Enquist; citation_author=KJ Niklas; citation_journal_title=Science; citation_volume=295; citation_number=5; citation_pages=1517-1520; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The origin of allometric scaling laws in biology from genomes to ecosystems: towards a quantitative unifying theory of biological structure and organization.; citation_author=GB West; citation_author=JH Brown; citation_journal_title=J Exp Biol; citation_volume=208; citation_number=6; citation_pages=1575-1592; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A phyletic perspective on the allometry of plant biomass-partitioning patterns and functionally equivalent organ-categories.; citation_author=KJ Niklas; citation_journal_title=New Phytologist; citation_volume=171; citation_number=7; citation_pages=27-40; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Natural selection and developmental constraints in the evolution of allometries.; citation_author=WA Frankino; citation_author=BJ Zwaan; citation_author=DL Stern; citation_author=PM Brakefield; citation_journal_title=Science; citation_volume=307; citation_number=8; citation_pages=718-720; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=RH Peter; citation_number=9; citation_date=1983; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_author=KJ Niklas; citation_number=10; citation_date=1994; citation_publisher=University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=The predominance of quarter-power scaling in biology.; citation_author=VM Savage; citation_author=JF Gillooly; citation_author=WH Woodruff; citation_author=GB West; citation_author=AP Allen; citation_author=BJ Enquist; citation_author=JH Brown; citation_journal_title=Fun Eco; citation_volume=l18; citation_number=11; citation_pages=257-282; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=No foundation of a “3/4 power scaling law” for respiration in biology.; citation_author=HU Riisgard; citation_journal_title=Ecol Let; citation_volume=1; citation_number=12; citation_pages=71-73; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Re-examination of the 3/4-law of metabolism.; citation_author=PS Dodds; citation_author=DH Rothman; citation_author=JS Weitz; citation_journal_title=J Theor Biol; citation_volume=209; citation_number=13; citation_pages=9-27; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mammalian basal metabolic rate is proportional to body mass.; citation_author=CR White; citation_author=RS Seymour; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=14; citation_pages=4046-4049; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A general model for the origin of allometric scaling laws in biology.; citation_author=GB West; citation_author=JH Brown; citation_author=BJ Enquist; citation_journal_title=Science; citation_volume=276; citation_number=15; citation_pages=122-126; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The fourth dimension of life: Fractal geometry and allometric scaling of organisms.; citation_author=GB West; citation_author=JH Brown; citation_author=BJ Enquist; citation_journal_title=Science; citation_volume=284; citation_number=16; citation_pages=1677-1679; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Size and form in efficient transportation networks.; citation_author=JR Banavar; citation_author=A Maritan; citation_author=A Rinaldo; citation_journal_title=Nature; citation_volume=399; citation_number=17; citation_pages=130-132; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Supply-demand balance and metabolic scaling.; citation_author=JR Banavar; citation_author=J Damuth; citation_author=A Maritan; citation_author=A Rinaldo; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=99; citation_number=18; citation_pages=10506-10509; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The developmental basis for allometry in insects.; citation_author=DL Stern; citation_author=DJ Emlen; citation_journal_title=Development; citation_volume=126; citation_number=19; citation_pages=1091-1101; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Development rate-Modelling developmental time and temperature.; citation_author=P Rombough; citation_journal_title=Nature; citation_volume=424; citation_number=20; citation_pages=268-269; citation_date=2003; " />
      <meta name="citation_reference" content="citation_author=FL Bookstein; citation_number=21; citation_date=1991; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=On applications of geometric morphometrics to studies of ontogeny and phylogeny.; citation_author=FJ Rohlf; citation_journal_title=Syst Biol; citation_volume=47; citation_number=22; citation_pages=147-158; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=On applications of geometric morphometrics to studies of ontogeny and phylogeny: a reply to Rohlf.; citation_author=ML Zelditch; citation_author=DL Swiderski; citation_author=BL Lundrigan; citation_journal_title=Syst Biol; citation_volume=47; citation_number=23; citation_pages=159-167; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Allometry and developmental integration of body growth in a piranha, Pygocentrus nattereri (Teleostei: Ostariophysi).; citation_author=ML Zelditch; citation_author=WL Fink; citation_journal_title=J Morphol; citation_volume=223; citation_number=24; citation_pages=341-355; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Ontogenetic allometry of body shape in the Neotropical catfish Callichthys (Teleostei: Siluriformes).; citation_author=RE Reis; citation_author=ML Zelditch; citation_author=WL Fink; citation_journal_title=Copeia; citation_volume=1998; citation_number=25; citation_pages=177-182; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Functional mapping - How to map and study the genetic architecture of dynamic complex traits.; citation_author=RL Wu; citation_author=M Lin; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=26; citation_pages=229-237; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The International HapMap Project.; citation_journal_title=Nature; citation_volume=426; citation_number=27; citation_pages=789-794; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps.; citation_author=ES Lander; citation_author=D Botstein; citation_journal_title=Genetics; citation_volume=121; citation_number=28; citation_pages=185-199; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci.; citation_author=ZB Zeng; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=90; citation_number=29; citation_pages=10972-10976; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Precision mapping of quantitative trait loci.; citation_author=ZB Zeng; citation_journal_title=Genetics; citation_volume=136; citation_number=30; citation_pages=1457-1468; citation_date=1994; " />
      <meta name="citation_reference" content="citation_author=M Lynch; citation_author=B Walsh; citation_number=31; citation_date=1998; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Multiple interval mapping for quantitative trait loci.; citation_author=C-H Kao; citation_author=Z-B Zeng; citation_author=RD Teasdale; citation_journal_title=Genetics; citation_volume=152; citation_number=32; citation_pages=1203-1216; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A model selection approach for the identification of quantitative trait loci in experimental crosses (with discussion).; citation_author=KW Broman; citation_author=TP Speed; citation_journal_title=J Roy Stat Soc B; citation_volume=64; citation_number=33; citation_pages=641-656; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bayesian model selection for genome-wide epistatic analysis.; citation_author=NJ Yi; citation_author=B Yandell; citation_author=G Churchill; citation_author=D Allison; citation_author=EJ Eisen; citation_author=D Pomp; citation_journal_title=Genetics; citation_volume=170; citation_number=34; citation_pages=1333-1344; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An efficient Bayesian model selection approach for interacting quantitative trait loci models with many effects.; citation_author=NJ Yi; citation_author=D Shriner; citation_author=S Banerjee; citation_author=T Mehta; citation_author=D Pomp; citation_author=BS Yandell; citation_journal_title=Genetics; citation_volume=176; citation_number=35; citation_pages=1865-1877; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Statistical models for studying the genetic architecture of dynamic complex traits.; citation_author=RL Wu; citation_author=M Lin; citation_author=W Zhao; citation_author=W Hou; citation_author=B Zhang; citation_author=Q Zhuge; citation_author=MR Huang; citation_author=LA Xu; citation_journal_title=J Nanjing Forestry Univ; citation_volume=30; citation_number=36; citation_pages=1-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A unified semiparametric framework for quantitative trait loci analyses, with application to spike phenotypes.; citation_author=CF Jin; citation_author=JP Fine; citation_author=BS Yandell; citation_journal_title=J Am Stat Assoc; citation_volume=102; citation_number=37; citation_pages=56-67; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Multiple trait analysis of genetic mapping for quantitative trait loci.; citation_author=C Jiang; citation_author=Z-B Zeng; citation_journal_title=Genetics; citation_volume=140; citation_number=38; citation_pages=1111-1127; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Multitrait least squares for quantitative trait loci detection.; citation_author=SA Knott; citation_author=CS Haley; citation_journal_title=Genetics; citation_volume=156; citation_number=39; citation_pages=899-911; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genotype environment interaction for quantitative trait loci affecting life span in Drosophila melanogaster.; citation_author=C Vieira; citation_author=EG Pasyukova; citation_author=ZB Zeng; citation_author=JB Hackett; citation_author=RF Lyman; citation_author=TFC Mackay; citation_journal_title=Genetics; citation_volume=154; citation_number=40; citation_pages=213-227; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Enhanced efficiency of quantitative trait loci mapping analysis based on multivariate complexes of quantitative traits.; citation_author=AB Korol; citation_author=YI Ronin; citation_author=AM Itskovich; citation_author=J Peng; citation_author=E Nevo; citation_journal_title=Genetics; citation_volume=157; citation_number=41; citation_pages=1789-1803; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Maximum-likelihood approaches applied to quantitative genetics of natural populations.; citation_author=RG Shaw; citation_journal_title=Evolution; citation_volume=45; citation_number=42; citation_pages=143-151; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=A statistical model for the genetic origin of allometric scaling laws in biology.; citation_author=RL Wu; citation_author=CX Ma; citation_author=RC Littell; citation_author=G Casella; citation_journal_title=J Theor Biol; citation_volume=217; citation_number=43; citation_pages=275-287; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Exponential mapping of quantitative traits governing allometric relationships in organisms.; citation_author=CX Ma; citation_author=G Casella; citation_author=RC Littell; citation_author=AI Khuri; citation_author=RL Wu; citation_journal_title=J Math Biol; citation_volume=47; citation_number=44; citation_pages=313-324; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A hyperspace model to decipher the genetic architecture of developmental processes: Allometry meets ontogeny.; citation_author=RL Wu; citation_author=W Hou; citation_journal_title=Genetics; citation_volume=172; citation_number=45; citation_pages=627-637; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genetic mapping of allometric scaling laws.; citation_author=F Long; citation_author=YQ Chen; citation_author=JM Cheverud; citation_author=RL Wu; citation_journal_title=Genet Res; citation_volume=87; citation_number=46; citation_pages=207-216; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Functional mapping of quantitative trait loci undelying the character process: A theoretical framework.; citation_author=CX Ma; citation_author=G Casella; citation_author=RL Wu; citation_journal_title=Genetics; citation_volume=161; citation_number=47; citation_pages=1751-1762; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines.; citation_author=C Jiang; citation_author=Z-B Zeng; citation_journal_title=Genetica; citation_volume=101; citation_number=48; citation_pages=47-58; citation_date=1997; " />
      <meta name="citation_reference" content="citation_author=PJ Diggle; citation_author=P Heagerty; citation_author=KY Liang; citation_author=SL Zeger; citation_number=49; citation_date=2002; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Modeling nonstationary longitudinal data.; citation_author=V Núñez-Antón; citation_author=DL Zimmerman; citation_journal_title=Biometrics; citation_volume=56; citation_number=50; citation_pages=699-705; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Parametric modeling of growth curve data: An overview (with discussion).; citation_author=DL Zimmerman; citation_author=V Núñez-Antón; citation_journal_title=Test; citation_volume=10; citation_number=51; citation_pages=1-73; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Finding the observed information matrix when using the EM algorithm.; citation_author=TA Louis; citation_journal_title=J Roy Stat Soc B; citation_volume=44; citation_number=52; citation_pages=226-233; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=A general framework for analyzing the genetic architecture of developmental characteristics.; citation_author=RL Wu; citation_author=CX Ma; citation_author=M Lin; citation_author=G Casella; citation_journal_title=Genetics; citation_volume=166; citation_number=53; citation_pages=1541-1551; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Empirical threshold values for quantitative trait mapping.; citation_author=GA Churchill; citation_author=RW Doerge; citation_journal_title=Genetics; citation_volume=138; citation_number=54; citation_pages=963-971; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Invariant scaling relationships for interspecific plant biomass production rates and body size.; citation_author=KJ Niklas; citation_author=BJ Enquist; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=98; citation_number=55; citation_pages=2922-2927; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=QTL mapping of ten agronomic traits on the soybean (Glycine).; citation_author=WK Zhang; citation_author=YJ Wang; citation_author=GZ Luo; citation_author=JS Zhang; citation_author=CY He; citation_author=XL Wu; citation_author=JY Gai; citation_author=SY Chen; citation_journal_title=Theor Appl Genet; citation_volume=108; citation_number=56; citation_pages=1131-1139; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=RL Wu; citation_author=C-X Ma; citation_author=G Casella; citation_number=57; citation_date=2007; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Static, ontogenetic, and evolutionary allometry: A multivariate comparison in nine species of water striders.; citation_author=CP Klingenberg; citation_author=M Zimmermann; citation_journal_title=Am Nat; citation_volume=140; citation_number=58; citation_pages=601-620; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Functional evo-devo.; citation_author=C Breuker; citation_author=V Debat; citation_author=CP Klingenberg; citation_journal_title=Trends Ecol Syst; citation_volume=21; citation_number=59; citation_pages=488-492; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A computational approach for functional mapping of quantitative trait loci that regulate thermal performance curves.; citation_author=JS Yap; citation_author=C Wang; citation_author=R Wu; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=60; citation_pages=e554; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Quantitative laws for metabolism and growth.; citation_author=L von Bertalanffy; citation_journal_title=Quart Rev Biol; citation_volume=32; citation_number=61; citation_pages=217-231; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=A general model for ontogenetic growth.; citation_author=GB West; citation_author=JH Brown; citation_author=BJ Enquist; citation_journal_title=Nature; citation_volume=413; citation_number=62; citation_pages=628-631; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A generalized model for detecting genetic determinants underlying longitudinal traits with unequally spaced measurements and time-dependent correlated errors.; citation_author=W Hou; citation_author=CW Garvan; citation_author=W Zhao; citation_author=M Behnke; citation_author=FD Eyler; citation_author=RL Wu; citation_journal_title=Biostatistics; citation_volume=6; citation_number=63; citation_pages=420-433; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A unified statistical model for functional mapping of genotype×environment interactions for ontogenetic development.; citation_author=W Zhao; citation_author=J Zhu; citation_author=M Gallo-Meagher; citation_author=RL Wu; citation_journal_title=Genetics; citation_volume=168; citation_number=64; citation_pages=1751-1762; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry"/>
    <meta name="twitter:description" content="Although ontogenetic changes in body shape and its associated allometry has been studied for over a century, essentially nothing is known about their underlying genetic and developmental mechanisms. One of the reasons for this ignorance is the unavailability of a conceptual framework to formulate the experimental design for data collection and statistical models for data analyses. We developed a framework model for unraveling the genetic machinery for ontogenetic changes of allometry. The model incorporates the mathematical aspects of ontogenetic growth and allometry into a maximum likelihood framework for quantitative trait locus (QTL) mapping. As a quantitative platform, the model allows for the testing of a number of biologically meaningful hypotheses to explore the pleiotropic basis of the QTL that regulate ontogeny and allometry. Simulation studies and real data analysis of a live example in soybean have been performed to investigate the statistical behavior of the model and validate its practical utilization. The statistical model proposed will help to study the genetic architecture of complex phenotypes and, therefore, gain better insights into the mechanistic regulation for developmental patterns and processes in organisms."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001245.g004"/>

  <meta property="og:title" content="A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1804'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4376'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9449&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001245">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001245" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001245&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hongying Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhongwen Huang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  National Center for Soybean Improvement, Nanjing Agricultural University, Nanjing, Jiangsu, People’s Republic of China, 
                  Department of Agronomy, Henan Institute of Science and Technology, Xinxiang, Henan, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Junyi Gai
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rwu@stat.ufl.edu">rwu@stat.ufl.edu</a> (RW); <a href="mailto:sri@njau.edu.cn">sri@njau.edu.cn</a> (JG)</p>

                <p>Affiliation:
                  National Center for Soybean Improvement, Nanjing Agricultural University, Nanjing, Jiangsu, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Song Wu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yanru Zeng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  School of Forestry and Biotechnology, Zhejiang Forestry University, Lin’an, Zhejiang, People’s Republic of China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qin Li, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rongling Wu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:rwu@stat.ufl.edu">rwu@stat.ufl.edu</a> (RW); <a href="mailto:sri@njau.edu.cn">sri@njau.edu.cn</a> (JG)</p>

                <p>Affiliation:
                  Department of Statistics, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001245</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001245-g001" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001245-g002" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001245-g003" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001245-t001" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001245-g004" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001245-t002" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001245">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001245" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Although ontogenetic changes in body shape and its associated allometry has been studied for over a century, essentially nothing is known about their underlying genetic and developmental mechanisms. One of the reasons for this ignorance is the unavailability of a conceptual framework to formulate the experimental design for data collection and statistical models for data analyses. We developed a framework model for unraveling the genetic machinery for ontogenetic changes of allometry. The model incorporates the mathematical aspects of ontogenetic growth and allometry into a maximum likelihood framework for quantitative trait locus (QTL) mapping. As a quantitative platform, the model allows for the testing of a number of biologically meaningful hypotheses to explore the pleiotropic basis of the QTL that regulate ontogeny and allometry. Simulation studies and real data analysis of a live example in soybean have been performed to investigate the statistical behavior of the model and validate its practical utilization. The statistical model proposed will help to study the genetic architecture of complex phenotypes and, therefore, gain better insights into the mechanistic regulation for developmental patterns and processes in organisms.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Li H, Huang Z, Gai J, Wu S, Zeng Y, et al.  (2007) A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry. PLoS ONE 2(11):
          e1245.
            doi:10.1371/journal.pone.0001245</p><p><strong>Academic Editor: </strong>Shin-Han Shiu, Michigan State University, United States of America</p><p><strong>Received:</strong> November 7, 2006; <strong>Accepted:</strong> October 17, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Li et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Natural Science Foundation of China (No. 30490250), the National 973 Program (2002CB111304, 2004CB7206 and 2006CB101708) and NSF grant (No. 0540745)</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>An incredible diversity has been observed in the scaling relationships among different body parts or traits, and between these and overall body size <a href="#pone.0001245-Kleiber1">[1]</a>–<a href="#pone.0001245-Niklas1">[7]</a>. The differentiation in such allometries among traits has been thought to be a driving force by which morphology evolves <a href="#pone.0001245-Frankino1">[8]</a>. Perhaps the most fundamental allometric relationship is the one that relates physiological, morphological and anatomical attributes with body size <a href="#pone.0001245-Kleiber1">[1]</a>, <a href="#pone.0001245-Calder1">[2]</a>, <a href="#pone.0001245-Peter1">[9]</a>, <a href="#pone.0001245-Niklas2">[10]</a>. Interestingly, the preponderance of data suggests that many metabolism-related structural traits scale as multiples of one quarter of body size <a href="#pone.0001245-Savage1">[11]</a>, rather than one third as expected from Euclidean geometric scaling. Despite some vigorous debate <a href="#pone.0001245-Riisgard1">[12]</a>–<a href="#pone.0001245-White1">[14]</a>, the quarter-power allometric scaling has been regarded as a universal phenomenon in biology, explained from fundamental principles of biology and biophysics <a href="#pone.0001245-West3">[15]</a>–<a href="#pone.0001245-Banavar2">[18]</a>. However, even with over a century of interest in the evolution of allometry, essentially nothing is known about the genetic and developmental mechanisms of differentiation in allometric scaling relationships, although developmental processes must have played a central role in maintaining the functional scaling relationships among traits as well as in their evolution <a href="#pone.0001245-West2">[6]</a>, <a href="#pone.0001245-Stern1">[19]</a>, <a href="#pone.0001245-Rombough1">[20]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The past two decades have witnessed a surge of interest in applying geometric morphometric approaches to understand how body shape changes and how such a change is associated with allometry during ontogeny <a href="#pone.0001245-Bookstein1">[21]</a>–<a href="#pone.0001245-Zelditch1">[23]</a>. For instance, these approaches have been used to study the ontogeny of body shape change for a few number of fishes <a href="#pone.0001245-Zelditch2">[24]</a>, <a href="#pone.0001245-Reis1">[25]</a>, showing that body shape changes during ontogeny are not simply the result of uniform large-scale events but that localized small-scale shape changes contribute to its ontogeny. However, none of these studies have attempted to detect the genetic machinery for ontogenetic allometry from a developmental perspective.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>One promising approach is to characterize specific genetic variants that regulate the ontogeny of allometry and compare them with those genetic variants that determine body size <a href="#pone.0001245-Wu1">[26]</a>. The feasibility of this approach results from two recent significant developments. First, the progress of whole genome sequence projects in microbes, plants, animals and human beings provides fundamental information about the organization and structure of genomes and proteins <a href="#pone.0001245-The1">[27]</a>. Second, the availability of powerful statistical methods allows direct association studies between genetic variants and complex metabolic processes <a href="#pone.0001245-Lander1">[28]</a>–<a href="#pone.0001245-Jin1">[37]</a>. Among these methods, a full-dimensional analysis of multiple traits can map and estimate quantitative trait loci (QTLs) for trait correlations <a href="#pone.0001245-Jiang1">[38]</a>–<a href="#pone.0001245-Korol1">[41]</a>. However, these multi-trait mapping approaches cannot take advantages of ontogenetic allometry, and will thus be less powerful than an approach that specifically incorporates allometry. Further, these methods do not extend easily to many time points or to missing data because of computational burden and estimation instability <a href="#pone.0001245-Shaw1">[42]</a>. By incorporating the mathematical aspects of allometric scaling into the mixture model-based framework, Wu and group developed a series of conceptual models and computational algorithms for detecting QTLs that govern allometry and testing the hypotheses about the genetic control of allometry <a href="#pone.0001245-Wu3">[43]</a>–<a href="#pone.0001245-Long1">[46]</a>. However, there is a serious lack of sophisticated models that have power to detect genetic variants responsible for the combined effects of size and development on allometric attributes at different organizational levels.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this article, we will frame a general genetic model for explaining universal allometric scaling laws and derive a statistical algorithm for detecting particular QTLs that contribute to these laws. The model embeds the allometric power equation into the framework for functional mapping constructed to map a dynamic trait <a href="#pone.0001245-Wu1">[26]</a>, allowing the identification of QTLs that determine the degree and pattern of the response of a body part to body size in development. The new model can be readily extended to predict how the structure and functioning of a biological system are affected by genetic interactions derived from different regions of the genome. The utilization of the model has been tested and validated by analyzing real data from the soybean genome project, in which several significant QTLs were detected on different soybean chromosomes to affect the allometric scaling between stem and whole-plant biomass during development. The empirical power of the model and the precision of its parameter estimation for a practical data set have been investigated through computer simulation studies. The model will provide a quantitative framework for analyzing the genetic architecture of ontogenetic changes in shape and allometry.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Allometry</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Consider a simple backcross or recombinant inbred line (RIL) design in which <em>n</em> progeny are segregating in a 1:1 ratio at each locus. A genetic linkage map, aimed to identify segregating quantitative trait loci (QTL), is constructed with polymorphic markers genotyped through the genome. All the progeny are measured for two developmentally related traits, Z and Y, at <em>T</em> time points, (<em>t</em><sub>1</sub>,…<em>t<sub>T</sub></em>). If two traits of a similar developmental origin can be modeled by an allometric equation, this can be expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>t</em> denotes a particular time, Z<sub>0</sub> is a normalization constant and β is a scaling exponent. Taking log-transformation at both sides of Equation 1, the allometric equation is linearized as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e003&amp;representation=PNG" class="inline-graphic"></span><br>We assume that the log-transformed traits at time <em>t</em> have a normal distribution. The observations for the two traits at time <em>t</em> can be expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e004&amp;representation=PNG" class="inline-graphic"></span><br>where <em>e</em>(<em>t</em>) is the measurement error at time <em>t</em> following <em>N</em>(0, σ<sup>2</sup>(<em>t</em>)).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p><a href="#pone-0001245-g001">Figure 1</a> plots two allometrically related traits, stem biomass and whole-plant biomass, during development for soybean plants randomly sampled from an RIL population, in which the original power relationship (<a href="#pone-0001245-g001">Fig. 1A</a>) is straightened out after log-log transformation (<a href="#pone-0001245-g001">Fig. 1B</a>). Such a log-linear allometric relationship has been justified from fundamental biological principles <a href="#pone.0001245-West3">[15]</a>–<a href="#pone.0001245-Banavar2">[18]</a>.</p>
<div class="figure" id="pone-0001245-g001"><div class="img"><a name="pone-0001245-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The allometric scaling relationship between stem biomass and whole-plant biomass for the RILs from a soybean mapping population.</span></strong></p><span>doi:10.1371/journal.pone.0001245.g001</span></div>

<h4>Likelihood</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>For each of the two dynamic traits, <em>z</em>(<em>t</em>) and <em>y</em>(<em>t</em>), functional mapping has established a general statistical framework for mapping its underlying QTLs with molecular markers <a href="#pone.0001245-Ma2">[47]</a>. In this study, we will incorporate the allometric scaling law (Equation 1) into functional mapping. But different from the previous treatment for bivariate functional mapping by Wu and Hou <a href="#pone.0001245-Wu4">[45]</a>, we will found functional mapping on the dependent trait, connected with the independent trait by the allometric equation. To simplify the description of our model, we assume that one single QTL is involved in the allometric control. The derivation of a more realistic multiple-QTL model is conceptually straightforward with the idea of the one QTL model, but this extension raises many statistical issues, such as model selection for the optimal number of QTL involved (see ref. <a href="#pone.0001245-Broman1">[33]</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Similar to Ma et al. <a href="#pone.0001245-Ma2">[47]</a>, we formulate the likelihood function for one dynamic trait, y, controlled by a QTL bracketed by a pair of marker (M), as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where y<em><sub>i</sub></em> = (<em>y<sub>i</sub></em>(<em>t</em><sub>1</sub>),…,<em>y<sub>i</sub></em>(<em>t<sub>T</sub></em>))′ is the observation vector, (<em>t</em><sub>1</sub>,…,<em>t<sub>T</sub></em>)′ denote the time points when the observations are measured, <em>f</em>(y<em><sub>i</sub></em>;u<em><sub>j</sub></em>, Σ) is the multivariate density function for different QTL genotypes (subscripted by <em>j</em> = 1 for <em>QQ</em> or 2 for <em>qq</em>) with mean vector u<em><sub>j</sub></em> = (<em>u<sub>j</sub></em>(<em>t</em><sub>1</sub>),…<em>u<sub>j</sub></em>(<em>t<sub>T</sub></em>))′ and time-dependent covariance matrix Σ, and ω<sub>1|<em>i</em></sub> and ω<sub>2|<em>i</em></sub> are the conditional probabilities of a QTL genotype, 1 or 2, given the genotype of progeny <em>i</em> for two flanking markers. The conditional probabilities are expressed in terms of the recombination fractions (for the backcross design) or the proportions of recombinant homozygotes (for the RIL design) <a href="#pone.0001245-Jiang2">[48]</a> between the left marker and QTL (<em>r</em><sub>1</sub>) and between the QTL and right marker (<em>r</em><sub>2</sub>).</p>


<h4>Modelling the Mean Vector</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>If the allometric relationship between two biological traits is controlled by a QTL, the linearized power equation (2) can be used to model the genotypic mean vector in the likelihood (3) with genotype-specific parameter sets (α<sub>1</sub>, β<sub>1</sub>) or (α<sub>2</sub>, β<sub>2</sub>). Thus, by testing the difference between these two parameter sets, we can conclude whether there is a specific QTL for allometric scaling and how the QTL controls the scaling relationship. Wu and Hou <a href="#pone.0001245-Wu4">[45]</a> modeled the allometric scaling relationship by incorporating the genotypic vectors of the two traits, <em>y</em> and <em>z</em>, i.e.,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <em>u<sub>j</sub></em>(<em>t</em>) and <em>v<sub>j</sub></em>(<em>t</em>) are the genotypic values of traits <em>y</em> and <em>z</em> for QTL genotype <em>j</em> at time point <em>t</em>. This treatment needs to simultaneously estimate the genotypic vector of traits <em>y</em> and <em>z</em>, expressed as (<em>u<sub>j</sub></em>(<em>t</em><sub>1</sub>),…, <em>u<sub>j</sub></em>(<em>t<sub>T</sub></em>), <em>v<sub>j</sub></em>(<em>t</em><sub>1</sub>),…, <em>v<sub>j</sub></em>(<em>t<sub>T</sub></em>)). More importantly, the time-dependent covariance matrix for each trait and the time-dependent covariance matrix between the two traits need to be specified at a time, leading to a double-sized covariance matrix of dimension 2T×2T. All these will largely increase the number of parameters to be estimated, making the computation quickly prohibitive and the parameter estimation imprecise.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>To overcome this problem, we will formulate a different model for the allometric relationship. Given that the allometric change of one trait is not only regulated by the underlying genes, but also by physiology-and metabolism-related characteristics that contain the influences of both genes and environments <a href="#pone.0001245-Niklas1">[7]</a>, we model the genotypic vector of a trait with the phenotypic value of a second allometrically related trait. Thus, Equation 4 is changed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where <em>u<sub>j</sub></em><sub>|<em>i</em></sub>(<em>t</em>) = E[<em>y<sub>i</sub></em>(<em>t</em>)|<em>Z<sub>i</sub></em>(<em>t</em>), QTL genotype <em>j</em>]. This can be explained from genetic and statistical perspectives. In genetics, Equation 5 states how the genotypic value of trait <em>y</em> for QTL genotype <em>j</em> scales as, or responds to, the phenotypic change of trait <em>z</em> during ontogeny. If parameter set (α<em><sub>j</sub></em>, β<em><sub>j</sub></em>) is not different between QTL genotypes, this means that this QTL does not determine the allometric scaling between traits <em>y</em> and <em>z</em>. Equation 5 indicates that given the QTL genotype we can regress <em>y</em> on the covariate <em>z</em> using a linear regression model.</p>


<h4>Modeling the Time-Dependent Covariance</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The residual covariance matrix, Σ, generally follows an autocorrelation structure, which can be mathematically modeled. A number of statistical models, such as autoregressive <a href="#pone.0001245-Diggle1">[49]</a> and antedependent <a href="#pone.0001245-NezAntn1">[50]</a> models, have been formulated to model such a structure. In Zimmerman and Núñez-Antón <a href="#pone.0001245-Zimmerman1">[51]</a>, the advantages of structured antedependent (SAD) model have been extensively discussed, which include (1) the assumptions of variance and correlation stationary are not needed, and (2) closed forms exist for the inverse and determinant of the SAD matrix.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>For first order SAD models with an antedependence parameter ρ, if we assume the innovation variances σ<sup>2</sup>(<em>t</em>) to be a constant σ<sup>2</sup> over time, explicit forms of variance and correlation functions can be obtained as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e008&amp;representation=PNG" class="inline-graphic"></span><br>for <em>t<sub>k</sub></em>≥<em>t<sub>j</sub></em>. For this simplest SAD(1) model, the variance and correlation functions are non-stationary. They change as time and time interval change.</p>


<h4>Computational Algorithm</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>When Equation 5 is substituted into the likelihood of Equation 3, the likelihood function is now expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>where u<sub><em>j</em>|<em>i</em></sub> = (<em>u<sub>j</sub></em><sub>|<em>i</em></sub>(<em>t</em><sub>1</sub>),…, <em>u<sub>j</sub></em><sub>|<em>i</em></sub>(<em>t<sub>T</sub></em>))′(<em>j</em> = 1, 2) are the QTL genotype-specific mean vectors which also depend on the trait <em>z</em> (see Equation 5).</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The underlying unknown parameters are composed of (<em>r</em><sub>1</sub> or <em>r</em><sub>2</sub>, α<em><sub>j</sub></em>, β<em><sub>j</sub></em>, ρ, σ<sup>2</sup>). In the <a href="#pone.0001245.s001">Appendix S1</a>, we derive the EM algorithm to obtain the maximum likelihood estimates (MLEs) of these unknown parameters. The estimates of the sampling errors of the MLEs can be obtained from Louis’ <a href="#pone.0001245-Louis1">[52]</a> approach derived within the context of a mixture model.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The heritability of trait <em>y</em> at time <em>t</em> is calculated as follows:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e010&amp;representation=PNG" class="inline-graphic"></span><br>where<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e011&amp;representation=PNG" class="inline-graphic"></span><br>and<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e012&amp;representation=PNG" class="inline-graphic"></span><br>with<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e013&amp;representation=PNG" class="inline-graphic"></span><br>Here <em>u</em><sub>z</sub>(<em>t</em>) and σ<sup>2</sup>(<em>t</em>) are the mean and variance for trait <em>z</em> at time <em>t</em>, respectively, which can be estimated from observations.</p>


<h4>Hypothesis Testing</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>As shown in our previous publications <a href="#pone.0001245-Wu1">[26]</a>, <a href="#pone.0001245-Wu3">[43]</a>, <a href="#pone.0001245-Ma2">[47]</a>, <a href="#pone.0001245-Wu5">[53]</a>, functional mapping is advantageous for the tests of biologically meaningful hypotheses regarding genetic actions and organ development. Here, we outline several important hypotheses for the genetic control of allometric scaling. The first hypothesis is about the existence of QTL, which can be tested by formulating the null hypothesis,</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>H<sub>0</sub>: α<sub>1</sub> = α<sub>2</sub> and β<sub>1</sub> = β<sub>2</sub></p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>H<sub>1</sub>: At least one of the equalities in H<sub>0</sub> does not hold.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>The log-likelihood ratio statistic is calculated as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>where the tildes and hats are the MLEs of parameters under the null and alternative hypotheses, respectively. The LR value is then compared with the critical threshold determined from permutation tests, as advocated by Churchill and Doerge <a href="#pone.0001245-Churchill1">[54]</a>, to test the significance of the QTL hypothesized.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>We are also interested in the genetic cause for the differentiation in ontogenetic allometric scaling. This can be investigated by testing the normalization (α) and exponent constant (β) individually. Some study suggests that α is a characteristic of species or populations <a href="#pone.0001245-Niklas2">[10]</a>, whereas a recent survey by Niklas and Enquist <a href="#pone.0001245-Niklas3">[55]</a> shows that all plants have a similar normalization constant and, therefore, comply with a single allometric formula. This debate can be solved by testing whether α equals to a specific constant for different plant species.</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>In practice, the exponent coefficient β can be considered as a constant if the allometric relationship of the two traits studied is known. For example, body length scales as the 1/4-power of body mass <a href="#pone.0001245-West3">[15]</a>, <a href="#pone.0001245-West4">[16]</a>. In this case, β = 1/4 can be directly substituted into Equation 5 to obtain estimates for the remainder of the unknown parameters. Owing to the reduced number of the unknowns to be estimated, such a substitution can potentially increase the precision and power of parameter estimation.</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>All the tests for α and β can be performed by calculating a likelihood ratio statistic which asymptotically follows a chi-square distribution with the corresponding degrees of freedom. In actual data analyses, an empirical approach based on simulation studies can be used to determine the threshold for these tests.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>A Worked Example</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We used a real example from the soybean genome project to validate the model proposed for mapping ontogenetic allometry. Two original inbred lines of soybean, Kefeng No. 1 and Nannong 1138-2, as parents were crossed to generate an F<sub>1</sub> population which was selfed for 7 generations to produce an RIL population composed of two groups of homozygous genotypes each containing two identical alleles from a different parental line. Let 1 and 2 denote the homozygotes derived from the Kefeng No. 1 alleles and Nannong 1138-2 alleles, respectively. A total of 184 RILs were genotyped for 488 molecular markers (restricted fragment length polymorphisms, simple sequence repeats and amplified fragment length polymorphisms) that construct a linkage map with 25 linkage groups covering 4,151.2 cM of the soybean genome <a href="#pone.0001245-Zhang1">[56]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The RILs were planted in a simple lattice design with multiple replicates in a plot at Jiangpu Station, Nanjing Agricultural University, Nanjing, China. The plants were harvested to measure their above- and under-ground biomass for eight times with the first time at the 28th day after emergence and successive seven times every 10 days thereafter. For the same RIL, the phenotypic values measured for different times correspond to successive measurements on a time scale. In this study, we will analyze the genetic control of the ontogenetic allometric scaling relationship between stem and whole-plant biomass.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>As shown by a subset of RILs from the mapping population in <a href="#pone-0001245-g001">Figure 1</a>, stem biomass scales as a power function of whole-plant biomass. The Pearson correlation coefficients between the two log-transformed traits from the samples for all subjects are all close to 1, ranging from 0.9381 to 0.9992. The sample mean of the coefficients for all individuals is 0.9885 with a sample standard error 0.0092. Thus, a high linear trend between the log-transformed stem biomass and whole plant biomass is appropriate for our allometric mapping model.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>This relationship was incorporated into functional mapping to characterize specific QTL that control the allometric change of stem biomass relative to whole-plant biomass. Different from the backcross population, the conditional probabilities of a QTL genotype given marker genotypes are expressed in terms of the proportion of recombinant homozygotes <a href="#pone.0001245-Wu6">[57]</a>. We detected five significant allometry QTLs, two located between markers GMKF082c and GMKF168b and at marker A520T on chromosome 3, one located between markers GMKF059a and satt319 on chromosome 6, one located between markers Satt372 and Satt154 on chromosome 10, and one between markers GMKF082b and satt331 on chromosome 24, as indicated by peaks of the LR profile beyond the 5% critical threshold obtained from 1000 permutation tests (<a href="#pone-0001245-g002">Fig. 2</a>). We tend to claim two QTL on chromosome 3 in this particular example (although they were not tested simultaneously) because their detected positions are about 120 cM apart, suggesting an unlinked relationship. The permutation tests <a href="#pone.0001245-Churchill1">[54]</a> were performed by repeatedly reshuffling stem biomass and whole-plant biomass among different RIL progeny but leaving marker genotypes unchanged. It is interesting to point out that the QTL on chromosome 6 is located between two closely spaced markers (5 cM apart), with 4 cM to the left marker and 1 cM to the right one. In conjunction with a narrow LR peak, this suggests that the detection of this QTL has a high resolution. Two QTLs were counted on chromosome 3 because they are distant enough from each other to infer the existence of two unlinked QTLs. On chromosome 24, there are two well-separated peaks, but they are two close to claim the existence of two different QTL. Thus, we only counted one QTL with a higher peak. In the Discussion, we will provide a possible solution into the test of two linked QTLs under the framework of allometry QTL mapping by implementing the idea of composite interval mapping <a href="#pone.0001245-Zeng1">[29]</a>, <a href="#pone.0001245-Zeng2">[30]</a>.</p>
<div class="figure" id="pone-0001245-g002"><div class="img"><a name="pone-0001245-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The LR profile of the likelihoods under the null (there is no QTL) and alternative hypothesis (there is a QTL) across the lengths of 25 chromosomes for the allometric scaling relationship between stem and whole-plant biomass growth trajectories in a soybean RIL population.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The 5% significance critical threshold (10.98) determined from 1000 permutation tests is indicated by the broken horizontal line. The arrowed broken vertical line indicates the MLE of the QTL location.</p>
<span>doi:10.1371/journal.pone.0001245.g002</span></div><a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>The model provided the MLEs of genotype-specific curve parameters and covariance-structuring SAD parameters when each of the significant QTL was detected (<a href="#pone-0001245-t001">Table 1</a>). These estimates display great precision, as reflected by their small sampling errors estimated by Louis’ approach <a href="#pone.0001245-Louis1">[52]</a>. The estimated genotypic power curve parameters are used to calculate additive genetic effects, <em>a</em>(<em>t</em>), at each QTL that vary with time-dependent whole-plant biomass by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e015&amp;representation=PNG" class="inline-graphic"></span><br>for an RIL design. The positive value of <em>a</em>(<em>t</em>) implies that parent Kefeng No. 1 contributes favorable alleles to increased stem biomass, whereas the negative value corresponds to the favorable contribution made by parent Nannong 1138-2. As shown by <a href="#pone-0001245-g003">Fig. 3</a>, the additive effects of each QTL on stem biomass change with whole-plant biomass. Based on their signs, it is suggested that at the two QTLs on chromosomes 10 and 24 favorable alleles for increased stem biomass are contributed by parent Kefeng No. 1, whereas the inverse pattern is true for the three QTLs on chromosomes 3 and 6. We estimated the heritability of each QTL for stem biomass at the fifth time point, which ranges from 0.0198 to 0.0381 for the five QTL detected at different chromosomes.</p>
<div class="figure" id="pone-0001245-g003"><div class="img"><a name="pone-0001245-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Body size-dependent additive genetic effects calculated from ontogenetic allometry curves for two different genotypes at each of the five QTL detected on chromosomes 3, 6,10 and 24.</span></strong></p><span>doi:10.1371/journal.pone.0001245.g003</span></div><div class="figure" id="pone-0001245-t001"><div class="img"><a name="pone-0001245-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001245.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001245.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Maximum likelihood estimates (MLEs) of genotype-specific power parameters (<em>α</em> and <em>β</em>) for each QTL detected and SAD(1) parameters (ρ and σ<sup>2 </sup>) that model the covariance structure.</span></strong></p><span>doi:10.1371/journal.pone.0001245.t001</span></div>

<h4>Computer Simulation</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Monte Carlo simulation studies were performed to examine the reliability of the parameters estimates in the soybean example above by mimicking the data structure of the mapping population. Also, additional simulation analyses were used to investigate the statistical properties of the model in terms of estimation precision and power under different sample sizes and heritability levels. An RIL population was simulated for 11 equally spaced markers that construct a linkage group of length 200 cM. A QTL that affects the ontogenetic allometric scaling relationship between traits <em>y</em> and <em>z</em> is assumed at 85 cM from the first marker. The data for marker genotypes were simulated in terms of the recombinant homozygote proportion (R). The genetic distances between markers are calculated from the recombination fractions (r) with the Haldane map function. The recombination fractions were calculated from the recombinant homozygote proportions using<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e016&amp;representation=PNG" class="inline-graphic"></span><br>Dynamic trait <em>y</em> is assumed for each RIL plant to follow a multivariate normal distribution with mean vector specified by Equation 5 and covariance matrix specified by the SAD model. Dynamic trait <em>z</em> is also assumed to follow a multivariate normal distribution with time-increasing means and SAD-structured covariance matrix. And observations of traits <em>y</em> and <em>z</em> are obtained at 8 time points (1,…, 8). The innovative variance for trait <em>y</em> is determined by assuming different heritability levels (<em>H</em><sup>2</sup> = 0.1 and 0.4) at the nearly middle period (time point 5) of time course. The size of heritability reflects the contributions of other unobserved genes to phenotypic variation as well as the influences of measurement errors that cause observations to deviate from allometry. Thus, a small heritability is partially associated with a large degree of deviation from allometry. Two different sample sizes (n = 100 and 400) are considered for the RIL population.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The ontogenetic allometric functional mapping was used to map QTL for the simulated data. <a href="#pone-0001245-g004">Figure 4</a> illustrates the LR profiles for the data simulated under different heritability levels and sample sizes. In general, the location of the QTL can be well estimated even for a small sample size (100) and heritability (0.1). The estimation accuracy of the QTL location can be increased when the sample size increase to 400 and or the heritability increases to 0.4. The MLEs of the curve parameters and covariance-structuring parameters are tabulated in <a href="#pone-0001245-t002">Table 2</a>. All the parameters can be reasonably estimated as indicated by small standard errors, with increased estimation precision associated with increased sample sizes and heritabilities. When the sample size and heritability are small (100 and 0.1), the power to detect a significant allometry QTL is reasonably high, and can increase dramatically when <em>n</em> increases to 400 and/or when <em>H</em><sup>2</sup> increases to 0.4.</p>
<div class="figure" id="pone-0001245-g004"><div class="img"><a name="pone-0001245-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001245.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The LR plots for the simulated data under different sample sizes (n = 100 and 400) and heritabilities (<em>H</em><sup>2</sup> = 0.1 and 0.4).</span></strong></p><span>doi:10.1371/journal.pone.0001245.g004</span></div><div class="figure" id="pone-0001245-t002"><div class="img"><a name="pone-0001245-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001245" data-uri="info:doi/10.1371/journal.pone.0001245.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001245.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001245.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001245.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001245.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Averaged MLEs of the parameters (with standard errors given in the parentheses) based on 400 simulation replicates under different simulation schemes combining different heritabilities (<em>H</em><sup>2</sup>) and sample size (<em>n</em>).</span></strong></p><span>doi:10.1371/journal.pone.0001245.t002</span></div>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>The term allometry that describes scaling relationships between different organ parts can be understood from three different perspectives: static, ontogenetic and evolutionary <a href="#pone.0001245-Stern1">[19]</a>, <a href="#pone.0001245-Klingenberg1">[58]</a>. Static allometry refers to the scaling among individuals between two different traits after growth has ceased or at a particular developmental stage. Ontogenetic allometry is the growth trajectory of one trait relative to the other (i.e., shape) during an individual’s lifetime. Evolutionary (or phylogenetic) allometry is the size relationship between traits across species. Much earlier work has focused on the developmental processes and constraints that shape static allometry <a href="#pone.0001245-Enquist2">[5]</a> as well as on the evolution of allometries <a href="#pone.0001245-Frankino1">[8]</a>. With the recognition of development as an evolutionary factor, evolutionary developmental biology (evo-devo) has revived an interest in understanding the process of evolution <a href="#pone.0001245-Breuker1">[59]</a>. It is anticipated that ontogenetic allometry that determine the direction and pattern of development will be a component of primary importance to construct the evo-devo framework.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Although allometry has been an important subject of biological research for over a century, little is known about the mechanism of its genetic control. Recent genomic technologies have opened a new avenue to generate genome-wide marker data and, therefore, characterize the specific loci or DNA sequence variants that are associated with the phenotypic variation in static <a href="#pone.0001245-Lander1">[28]</a>–<a href="#pone.0001245-Zeng2">[30]</a>, <a href="#pone.0001245-Jin1">[37]</a> or dynamic traits <a href="#pone.0001245-Wu1">[26]</a>. In this article, we have developed a statistical model for deploying such technologies to map quantitative trait loci (QTLs) that are responsible for ontogenetic allometry. This model allows for the characterization of genetic loci that cause ontogenetic shape change and transformations during growth and development.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The model for mapping ontogenetic allometry is built on the foundation of functional mapping <a href="#pone.0001245-Ma2">[47]</a>, aimed to map QTL that control growth trajectories of a trait. Yet, the new model is different from conventional functional mapping, in which a different but allometrically related trait is embedded through the power equation within the mean vector as a covariate in terms of statistical definition. The function of such embedment is to directly characterize specific QTL that determine ontogenetic changes of allometry and push the hypothesis tests at the interface between genetic actions and shape development. The approach for treating ontogenetic allometric scaling in this article is different from that published in Wu and Hou <a href="#pone.0001245-Wu4">[45]</a> who jointly modeled two different growth trajectories. Because the current approach only needs to model the relationship of the growth trajectory of two traits, it is more efficient and precise in parameter estimation and computation than Wu and Hou’s approach.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The proposed model has been tested through simulation studies. It is possible that this model can provide the reasonable estimation of the underlying parameters when a trait trajectory has a modest heritability, i.e., with non-genetic variation outweighing genetic variation. In practice, when a trait has a relatively low heritability (e.g., 0.10), a sample size of 400 is recommended to provide satisfactory precision for parameter estimation and power for QTL detection. Our simulation did not test the influence of deviation from allometry on parameter estimation and power. In a similar dynamic genetic study, Yap et al. <a href="#pone.0001245-Yap1">[60]</a> found that such an influence can be significant but can well be compensated by using a large sample size. This model was used to analyze a real example from the soybean genome project <a href="#pone.0001245-Zhang1">[56]</a> in which there exists a strong linear trend between stem and whole-plant biomass, leading to the detection of five significant QTLs that control ontogenetic allometric scaling between these two traits.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The biological relevance of our model can be enhanced by incorporating the growth equation into the mean vector. Empirical studies on the basis of the goodness-of-fit of observational data suggest that growth can be described by a logistic curve <a href="#pone.0001245-vonBertalanffy1">[61]</a>, which has been justified by fundamental biological principles <a href="#pone.0001245-West5">[62]</a>. If a logistic equation is used to describe the growth trajectory of trait <em>z</em>, we can estimate the curve parameters for each individual, using non-linear least-squares approach, on the basis of<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e017&amp;representation=PNG" class="inline-graphic"></span><br>where parameter set (<em>a<sub>i</sub></em>, <em>b<sub>i</sub></em>, <em>r<sub>i</sub></em>) define the curve shape of an individual <em>i</em>. These estimates are then substituted into Equation 5 for genotype-specific scaling relationships, which is expressed as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001245.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>The advantage of Equation 8 lies in its capacity to test the relationship between ontogenetic allometry and growth trajectory through the implementation of growth equation.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>The model presented in this article illustrates the idea of mapping the ontogenetic allometry of different biological traits by assuming one underlying QTL. A more sophisticated model that involves multiple QTL and their interactions in a genetic network can be derived with this idea (see <a href="#pone.0001245-Wu5">[53]</a>) although it needs more extensive computation and model selection <a href="#pone.0001245-Broman1">[33]</a>. The model described here assumes a full marker data set, but it can be readily modified to consider missing marker data based on a hidden Markov model as advocated by Jiang and Zeng <a href="#pone.0001245-Jiang2">[48]</a>. Also, when phenotypic data are missing at arbitrary time points, the measurement schedule will become unequally spaced. Hou et al. <a href="#pone.0001245-Hou1">[63]</a> derived an approach for modeling the structure of a longitudinal covariance matrix containing unequal spaced time points, which can be used in this allometric mapping model.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>In our example, more than one LR peak was detected on the same chromosome 6. We claimed the existence of two different QTL because they seem to be far enough from each other. However, more precise determination of multiple QTL should be based on multiple interval mapping as proposed by <a href="#pone.0001245-Kao1">[32]</a>. In particular, Yandell and colleagues constructed a series of Bayesian models that are shown to be powerful for the determination of multiple QTL at the same time <a href="#pone.0001245-Yi1">[34]</a>, <a href="#pone.0001245-Yi2">[35]</a>. These advanced genetic mapping approaches will stimulate our incorporation to build up a practically more useful allometry mapping framework. Our model can be extended to incorporate the effects of other factors on ontogenetic allometry. For example, in animals, a significant relationship occurs between reproductive status and ontogenetic shape change <a href="#pone.0001245-Bookstein1">[21]</a>–<a href="#pone.0001245-Zelditch1">[23]</a>. Within both males and females, reproductive classes had significantly different body shapes and in females the trajectories of shape change among reproductive classes were significantly different. By incorporating sexes into the model, sex-dependent QTL for ontogenetic shape changes can be estimated and tested.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p>Allometric shape changes in development may reflect functional changes and possible relationships between morphology and environment. It is straightforward to incorporate environmental factors into the allometry model to test the genetic effects of QTL on such relationships. The inclusion of multiple environments, as reported in <a href="#pone.0001245-Zhao1">[64]</a>, will allow investigating the environment-dependent expression of this allometry QTL.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001245"></div><a name="pone.0001245.s001" id="pone.0001245.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001245.s001">Appendix S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001245.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.09 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: RW JG. Performed the experiments: ZH. Analyzed the data: HL SW QL. Contributed reagents/materials/analysis tools: HL ZH YZ. Wrote the paper: RW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001245-Kleiber1" id="pone.0001245-Kleiber1"></a>Kleiber M (1932) Body size and metabolism. Hilgardia  6: 315–353.  <ul class="find" data-citedArticleID="1024270"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Body+size+and+metabolism.&amp;auth=&amp;atitle=Body+size+and+metabolism." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Body+size+and+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Body+size+and+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001245-Calder1" id="pone.0001245-Calder1"></a>Calder WA (1984) Size, Function, and Life History. Cambridge, MA: Harvard University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0001245-West1" id="pone.0001245-West1"></a>West GB, Brown JH, Enquist BJ (1999) A general model for the structure, and allometry of plant vascular systems. Nature  400: 664–667.  <ul class="find" data-citedArticleID="1024318" data-doi="10.1038/23251"><li><a href="http://dx.doi.org/10.1038/23251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+model+for+the+structure%2C+and+allometry+of+plant+vascular+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+model+for+the+structure%2C+and+allometry+of+plant+vascular+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001245-Enquist1" id="pone.0001245-Enquist1"></a>Enquist BJ, Niklas KJ (2001) Invariant scaling relations across tree-dominated communities. Nature  410: 655–660.  <ul class="find" data-citedArticleID="1024254" data-doi="10.1038/nature02023"><li><a href="http://dx.doi.org/10.1038/nature02023" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invariant+scaling+relations+across+tree-dominated+communities." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invariant+scaling+relations+across+tree-dominated+communities.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001245-Enquist2" id="pone.0001245-Enquist2"></a>Enquist BJ, Niklas KJ (2002) Global allocation rules for patterns of biomass partitioning across seed plants. Science  295: 1517–1520.  <ul class="find" data-citedArticleID="1024256" data-doi="10.1126/science.1066360"><li><a href="http://dx.doi.org/10.1126/science.1066360" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+allocation+rules+for+patterns+of+biomass+partitioning+across+seed+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+allocation+rules+for+patterns+of+biomass+partitioning+across+seed+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001245-West2" id="pone.0001245-West2"></a>West GB, Brown JH (2005) The origin of allometric scaling laws in biology from genomes to ecosystems: towards a quantitative unifying theory of biological structure and organization. J Exp Biol  208: 1575–1592.  <ul class="find" data-citedArticleID="1024320" data-doi="10.1242/jeb.01589"><li><a href="http://dx.doi.org/10.1242/jeb.01589" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+origin+of+allometric+scaling+laws+in+biology+from+genomes+to+ecosystems%3A+towards+a+quantitative+unifying+theory+of+biological+structure+and+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+origin+of+allometric+scaling+laws+in+biology+from+genomes+to+ecosystems%3A+towards+a+quantitative+unifying+theory+of+biological+structure+and+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001245-Niklas1" id="pone.0001245-Niklas1"></a>Niklas KJ (2006) A phyletic perspective on the allometry of plant biomass-partitioning patterns and functionally equivalent organ-categories. New Phytologist  171: 27–40.  <ul class="find" data-citedArticleID="1024292" data-doi="10.1111/j.1469-8137.2006.01760.x"><li><a href="http://dx.doi.org/10.1111/j.1469-8137.2006.01760.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phyletic+perspective+on+the+allometry+of+plant+biomass-partitioning+patterns+and+functionally+equivalent+organ-categories." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phyletic+perspective+on+the+allometry+of+plant+biomass-partitioning+patterns+and+functionally+equivalent+organ-categories.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001245-Frankino1" id="pone.0001245-Frankino1"></a>Frankino WA, Zwaan BJ, Stern DL, Brakefield PM (2005) Natural selection and developmental constraints in the evolution of allometries. Science  307: 718–720.  <ul class="find" data-citedArticleID="1024258" data-doi="10.1126/science.1105409"><li><a href="http://dx.doi.org/10.1126/science.1105409" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+and+developmental+constraints+in+the+evolution+of+allometries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+and+developmental+constraints+in+the+evolution+of+allometries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001245-Peter1" id="pone.0001245-Peter1"></a>Peter RH (1983) The Ecological Implications of Body Size. Cambridge, MA: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0001245-Niklas2" id="pone.0001245-Niklas2"></a>Niklas KJ (1994) Plant Allometry: The Scaling of Form and Process. Chicago, IL: University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001245-Savage1" id="pone.0001245-Savage1"></a>Savage VM, Gillooly JF, Woodruff WH, West GB, Allen AP, Enquist BJ, Brown JH (2004) The predominance of quarter-power scaling in biology. Fun Eco  l18: 257–282.  <ul class="find" data-citedArticleID="1024308" data-doi="10.1111/j.0269-8463.2004.00856.x"><li><a href="http://dx.doi.org/10.1111/j.0269-8463.2004.00856.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+predominance+of+quarter-power+scaling+in+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+predominance+of+quarter-power+scaling+in+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001245-Riisgard1" id="pone.0001245-Riisgard1"></a>Riisgard HU (1998) No foundation of a “3/4 power scaling law” for respiration in biology. Ecol Let  1: 71–73.  <ul class="find" data-citedArticleID="1024302" data-doi="10.1046/j.1461-0248.1998.00020.x"><li><a href="http://dx.doi.org/10.1046/j.1461-0248.1998.00020.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=No+foundation+of+a+%E2%80%9C3%2F4+power+scaling+law%E2%80%9D+for+respiration+in+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22No+foundation+of+a+%E2%80%9C3%2F4+power+scaling+law%E2%80%9D+for+respiration+in+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001245-Dodds1" id="pone.0001245-Dodds1"></a>Dodds PS, Rothman DH, Weitz JS (2001) Re-examination of the 3/4-law of metabolism. J Theor Biol  209: 9–27.  <ul class="find" data-citedArticleID="1024252" data-doi="10.1006/jtbi.2000.2238"><li><a href="http://dx.doi.org/10.1006/jtbi.2000.2238" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Re-examination+of+the+3%2F4-law+of+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Re-examination+of+the+3%2F4-law+of+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001245-White1" id="pone.0001245-White1"></a>White CR, Seymour RS (2003) Mammalian basal metabolic rate is proportional to body mass. Proc Natl Acad Sci USA  100: 4046–4049.  <ul class="find" data-citedArticleID="1024328" data-doi="10.1073/pnas.0436428100"><li><a href="http://dx.doi.org/10.1073/pnas.0436428100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+basal+metabolic+rate+is+proportional+to+body+mass." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+basal+metabolic+rate+is+proportional+to+body+mass.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001245-West3" id="pone.0001245-West3"></a>West GB, Brown JH, Enquist BJ (1997) A general model for the origin of allometric scaling laws in biology. Science  276: 122–126.  <ul class="find" data-citedArticleID="1024322" data-doi="10.1126/science.276.5309.122"><li><a href="http://dx.doi.org/10.1126/science.276.5309.122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+model+for+the+origin+of+allometric+scaling+laws+in+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+model+for+the+origin+of+allometric+scaling+laws+in+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001245-West4" id="pone.0001245-West4"></a>West GB, Brown JH, Enquist BJ (1999) The fourth dimension of life: Fractal geometry and allometric scaling of organisms. Science  284: 1677–1679.  <ul class="find" data-citedArticleID="1024324" data-doi="10.1126/science.284.5420.1677"><li><a href="http://dx.doi.org/10.1126/science.284.5420.1677" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+fourth+dimension+of+life%3A+Fractal+geometry+and+allometric+scaling+of+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+fourth+dimension+of+life%3A+Fractal+geometry+and+allometric+scaling+of+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001245-Banavar1" id="pone.0001245-Banavar1"></a>Banavar JR, Maritan A, Rinaldo A (1999) Size and form in efficient transportation networks. Nature  399: 130–132.  <ul class="find" data-citedArticleID="1024236" data-doi="10.1038/20144"><li><a href="http://dx.doi.org/10.1038/20144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Size+and+form+in+efficient+transportation+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Size+and+form+in+efficient+transportation+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001245-Banavar2" id="pone.0001245-Banavar2"></a>Banavar JR, Damuth J, Maritan A, Rinaldo A (2002) Supply-demand balance and metabolic scaling. Proc Natl Acad Sci USA  99: 10506–10509.  <ul class="find" data-citedArticleID="1024238" data-doi="10.1073/pnas.162216899"><li><a href="http://dx.doi.org/10.1073/pnas.162216899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Supply-demand+balance+and+metabolic+scaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Supply-demand+balance+and+metabolic+scaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001245-Stern1" id="pone.0001245-Stern1"></a>Stern DL, Emlen DJ (1999) The developmental basis for allometry in insects. Development  126: 1091–1101.  <ul class="find" data-citedArticleID="1024312"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+developmental+basis+for+allometry+in+insects.&amp;auth=&amp;atitle=The+developmental+basis+for+allometry+in+insects." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+developmental+basis+for+allometry+in+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+developmental+basis+for+allometry+in+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001245-Rombough1" id="pone.0001245-Rombough1"></a>Rombough P (2003) Development rate-Modelling developmental time and temperature. Nature  424: 268–269.  <ul class="find" data-citedArticleID="1024306" data-doi="10.1038/424268a"><li><a href="http://dx.doi.org/10.1038/424268a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+rate-Modelling+developmental+time+and+temperature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+rate-Modelling+developmental+time+and+temperature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001245-Bookstein1" id="pone.0001245-Bookstein1"></a>Bookstein FL (1991) Morphometric Tools for Landmark Data: Geometry and Biology. NY: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001245-Rohlf1" id="pone.0001245-Rohlf1"></a>Rohlf FJ (1998) On applications of geometric morphometrics to studies of ontogeny and phylogeny. Syst Biol  47: 147–158.  <ul class="find" data-citedArticleID="1024304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny.&amp;auth=&amp;atitle=On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001245-Zelditch1" id="pone.0001245-Zelditch1"></a>Zelditch ML, Swiderski DL, Lundrigan BL (1998) On applications of geometric morphometrics to studies of ontogeny and phylogeny: a reply to Rohlf. Syst Biol  47: 159–167.  <ul class="find" data-citedArticleID="1024348"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny%3A+a+reply+to+Rohlf.&amp;auth=&amp;atitle=On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny%3A+a+reply+to+Rohlf." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny%3A+a+reply+to+Rohlf." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+applications+of+geometric+morphometrics+to+studies+of+ontogeny+and+phylogeny%3A+a+reply+to+Rohlf.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001245-Zelditch2" id="pone.0001245-Zelditch2"></a>Zelditch ML, Fink WL (1995) Allometry and developmental integration of body growth in a piranha, <em>Pygocentrus nattereri</em> (Teleostei: Ostariophysi). J Morphol  223: 341–355.  <ul class="find" data-citedArticleID="1024350" data-doi="10.1002/jmor.1052230309"><li><a href="http://dx.doi.org/10.1002/jmor.1052230309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allometry+and+developmental+integration+of+body+growth+in+a+piranha%2C+Pygocentrus+nattereri+%28Teleostei%3A+Ostariophysi%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allometry+and+developmental+integration+of+body+growth+in+a+piranha%2C+Pygocentrus+nattereri+%28Teleostei%3A+Ostariophysi%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001245-Reis1" id="pone.0001245-Reis1"></a>Reis RE, Zelditch ML, Fink WL (1998) Ontogenetic allometry of body shape in the Neotropical catfish Callichthys (Teleostei: Siluriformes). Copeia  1998: 177–182.  <ul class="find" data-citedArticleID="1024300" data-doi="10.2307/1447715"><li><a href="http://dx.doi.org/10.2307/1447715" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ontogenetic+allometry+of+body+shape+in+the+Neotropical+catfish+Callichthys+%28Teleostei%3A+Siluriformes%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ontogenetic+allometry+of+body+shape+in+the+Neotropical+catfish+Callichthys+%28Teleostei%3A+Siluriformes%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001245-Wu1" id="pone.0001245-Wu1"></a>Wu RL, Lin M (2006) Functional mapping - How to map and study the genetic architecture of dynamic complex traits. Nat Rev Genet  7: 229–237.  <ul class="find" data-citedArticleID="1024330" data-doi="10.1038/nrg1804"><li><a href="http://dx.doi.org/10.1038/nrg1804" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+mapping+-+How+to+map+and+study+the+genetic+architecture+of+dynamic+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+mapping+-+How+to+map+and+study+the+genetic+architecture+of+dynamic+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001245-The1" id="pone.0001245-The1"></a>The International HapMap Consortium (2003) The International HapMap Project. Nature  426: 789–794.  <ul class="find" data-citedArticleID="1024314" data-doi="10.1038/nrg1351"><li><a href="http://dx.doi.org/10.1038/nrg1351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+International+HapMap+Project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+International+HapMap+Project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001245-Lander1" id="pone.0001245-Lander1"></a>Lander ES, Botstein D (1989) Mapping Mendelian factors underlying quantitative traits using RFLP linkage maps. Genetics  121: 185–199.  <ul class="find" data-citedArticleID="1024278"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps.&amp;auth=&amp;atitle=Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+Mendelian+factors+underlying+quantitative+traits+using+RFLP+linkage+maps.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001245-Zeng1" id="pone.0001245-Zeng1"></a>Zeng ZB (1993) Theoretical basis for separation of multiple linked gene effects in mapping quantitative trait loci. Proc Natl Acad Sci USA  90: 10972–10976.  <ul class="find" data-citedArticleID="1024352" data-doi="10.1073/pnas.90.23.10972"><li><a href="http://dx.doi.org/10.1073/pnas.90.23.10972" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Theoretical+basis+for+separation+of+multiple+linked+gene+effects+in+mapping+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Theoretical+basis+for+separation+of+multiple+linked+gene+effects+in+mapping+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001245-Zeng2" id="pone.0001245-Zeng2"></a>Zeng ZB (1994) Precision mapping of quantitative trait loci. Genetics  136: 1457–1468.  <ul class="find" data-citedArticleID="1024354"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Precision+mapping+of+quantitative+trait+loci.&amp;auth=&amp;atitle=Precision+mapping+of+quantitative+trait+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Precision+mapping+of+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Precision+mapping+of+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001245-Lynch1" id="pone.0001245-Lynch1"></a>Lynch M, Walsh B (1998) Genetics and Analysis of Quantitative Traits. Sunderland, MA: Sinauer.   <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0001245-Kao1" id="pone.0001245-Kao1"></a>Kao C-H, Zeng Z-B, Teasdale RD (1999) Multiple interval mapping for quantitative trait loci. Genetics  152: 1203–1216.  <ul class="find" data-citedArticleID="1024268" data-doi="10.1534/genetics.103.021642"><li><a href="http://dx.doi.org/10.1534/genetics.103.021642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+interval+mapping+for+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+interval+mapping+for+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001245-Broman1" id="pone.0001245-Broman1"></a>Broman KW, Speed TP (2002) A model selection approach for the identification of quantitative trait loci in experimental crosses (with discussion). J Roy Stat Soc B  64: 641–656.  <ul class="find" data-citedArticleID="1024244" data-doi="10.1111/1467-9868.00354"><li><a href="http://dx.doi.org/10.1111/1467-9868.00354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+model+selection+approach+for+the+identification+of+quantitative+trait+loci+in+experimental+crosses+%28with+discussion%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+model+selection+approach+for+the+identification+of+quantitative+trait+loci+in+experimental+crosses+%28with+discussion%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001245-Yi1" id="pone.0001245-Yi1"></a>Yi NJ, Yandell B, Churchill G, Allison D, Eisen EJ, Pomp D (2005) Bayesian model selection for genome-wide epistatic analysis. Genetics  170: 1333–1344.  <ul class="find" data-citedArticleID="1024344" data-doi="10.1534/genetics.104.040386"><li><a href="http://dx.doi.org/10.1534/genetics.104.040386" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+model+selection+for+genome-wide+epistatic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+model+selection+for+genome-wide+epistatic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001245-Yi2" id="pone.0001245-Yi2"></a>Yi NJ, Shriner D, Banerjee S, Mehta T, Pomp D, Yandell BS (2007) An efficient Bayesian model selection approach for interacting quantitative trait loci models with many effects. Genetics  176: 1865–1877.  <ul class="find" data-citedArticleID="1024346" data-doi="10.1534/genetics.107.071365"><li><a href="http://dx.doi.org/10.1534/genetics.107.071365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+efficient+Bayesian+model+selection+approach+for+interacting+quantitative+trait+loci+models+with+many+effects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+efficient+Bayesian+model+selection+approach+for+interacting+quantitative+trait+loci+models+with+many+effects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001245-Wu2" id="pone.0001245-Wu2"></a>Wu RL, Lin M, Zhao W, Hou W, Zhang B, Zhuge Q, Huang MR, Xu LA (2006) Statistical models for studying the genetic architecture of dynamic complex traits. J Nanjing Forestry Univ  30: 1–12.  <ul class="find" data-citedArticleID="1024332"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+models+for+studying+the+genetic+architecture+of+dynamic+complex+traits.&amp;auth=&amp;atitle=Statistical+models+for+studying+the+genetic+architecture+of+dynamic+complex+traits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+models+for+studying+the+genetic+architecture+of+dynamic+complex+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+models+for+studying+the+genetic+architecture+of+dynamic+complex+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001245-Jin1" id="pone.0001245-Jin1"></a>Jin CF, Fine JP, Yandell BS (2007) A unified semiparametric framework for quantitative trait loci analyses, with application to spike phenotypes. J Am Stat Assoc  102: 56–67.  <ul class="find" data-citedArticleID="1024266" data-doi="10.1198/016214506000000834"><li><a href="http://dx.doi.org/10.1198/016214506000000834" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+unified+semiparametric+framework+for+quantitative+trait+loci+analyses%2C+with+application+to+spike+phenotypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+unified+semiparametric+framework+for+quantitative+trait+loci+analyses%2C+with+application+to+spike+phenotypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001245-Jiang1" id="pone.0001245-Jiang1"></a>Jiang C, Zeng Z-B (1995) Multiple trait analysis of genetic mapping for quantitative trait loci. Genetics  140: 1111–1127.  <ul class="find" data-citedArticleID="1024262"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+trait+analysis+of+genetic+mapping+for+quantitative+trait+loci.&amp;auth=&amp;atitle=Multiple+trait+analysis+of+genetic+mapping+for+quantitative+trait+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+trait+analysis+of+genetic+mapping+for+quantitative+trait+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+trait+analysis+of+genetic+mapping+for+quantitative+trait+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001245-Knott1" id="pone.0001245-Knott1"></a>Knott SA, Haley CS (2000) Multitrait least squares for quantitative trait loci detection. Genetics  156: 899–911.  <ul class="find" data-citedArticleID="1024274"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multitrait+least+squares+for+quantitative+trait+loci+detection.&amp;auth=&amp;atitle=Multitrait+least+squares+for+quantitative+trait+loci+detection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multitrait+least+squares+for+quantitative+trait+loci+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multitrait+least+squares+for+quantitative+trait+loci+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001245-Vieira1" id="pone.0001245-Vieira1"></a>Vieira C, Pasyukova EG, Zeng ZB, Hackett JB, Lyman RF, Mackay TFC (2000) Genotype environment interaction for quantitative trait loci affecting life span in <em>Drosophila melanogaster</em>. Genetics  154: 213–227.  <ul class="find" data-citedArticleID="1024316"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genotype+environment+interaction+for+quantitative+trait+loci+affecting+life+span+in+Drosophila+melanogaster.&amp;auth=&amp;atitle=Genotype+environment+interaction+for+quantitative+trait+loci+affecting+life+span+in+Drosophila+melanogaster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotype+environment+interaction+for+quantitative+trait+loci+affecting+life+span+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotype+environment+interaction+for+quantitative+trait+loci+affecting+life+span+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001245-Korol1" id="pone.0001245-Korol1"></a>Korol AB, Ronin YI, Itskovich AM, Peng J, Nevo E (2001) Enhanced efficiency of quantitative trait loci mapping analysis based on multivariate complexes of quantitative traits. Genetics  157: 1789–1803.  <ul class="find" data-citedArticleID="1024276"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Enhanced+efficiency+of+quantitative+trait+loci+mapping+analysis+based+on+multivariate+complexes+of+quantitative+traits.&amp;auth=&amp;atitle=Enhanced+efficiency+of+quantitative+trait+loci+mapping+analysis+based+on+multivariate+complexes+of+quantitative+traits." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhanced+efficiency+of+quantitative+trait+loci+mapping+analysis+based+on+multivariate+complexes+of+quantitative+traits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhanced+efficiency+of+quantitative+trait+loci+mapping+analysis+based+on+multivariate+complexes+of+quantitative+traits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001245-Shaw1" id="pone.0001245-Shaw1"></a>Shaw RG (1987) Maximum-likelihood approaches applied to quantitative genetics of natural populations. Evolution  45: 143–151.  <ul class="find" data-citedArticleID="1024310" data-doi="10.2307/2408890"><li><a href="http://dx.doi.org/10.2307/2408890" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum-likelihood+approaches+applied+to+quantitative+genetics+of+natural+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum-likelihood+approaches+applied+to+quantitative+genetics+of+natural+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001245-Wu3" id="pone.0001245-Wu3"></a>Wu RL, Ma CX, Littell RC, Casella G (2002) A statistical model for the genetic origin of allometric scaling laws in biology. J Theor Biol  217: 275–287.  <ul class="find" data-citedArticleID="1024334" data-doi="10.1006/jtbi.2002.3114"><li><a href="http://dx.doi.org/10.1006/jtbi.2002.3114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+statistical+model+for+the+genetic+origin+of+allometric+scaling+laws+in+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+statistical+model+for+the+genetic+origin+of+allometric+scaling+laws+in+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001245-Ma1" id="pone.0001245-Ma1"></a>Ma CX, Casella G, Littell RC, Khuri AI, Wu RL (2003) Exponential mapping of quantitative traits governing allometric relationships in organisms. J Math Biol  47: 313–324.  <ul class="find" data-citedArticleID="1024286" data-doi="10.1007/s00285-003-0212-z"><li><a href="http://dx.doi.org/10.1007/s00285-003-0212-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exponential+mapping+of+quantitative+traits+governing+allometric+relationships+in+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exponential+mapping+of+quantitative+traits+governing+allometric+relationships+in+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001245-Wu4" id="pone.0001245-Wu4"></a>Wu RL, Hou W (2006) A hyperspace model to decipher the genetic architecture of developmental processes: Allometry meets ontogeny. Genetics  172: 627–637.  <ul class="find" data-citedArticleID="1024336" data-doi="10.1534/genetics.105.045310"><li><a href="http://dx.doi.org/10.1534/genetics.105.045310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+hyperspace+model+to+decipher+the+genetic+architecture+of+developmental+processes%3A+Allometry+meets+ontogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+hyperspace+model+to+decipher+the+genetic+architecture+of+developmental+processes%3A+Allometry+meets+ontogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001245-Long1" id="pone.0001245-Long1"></a>Long F, Chen YQ, Cheverud JM, Wu RL (2006) Genetic mapping of allometric scaling laws. Genet Res  87: 207–216.  <ul class="find" data-citedArticleID="1024280" data-doi="10.1017/s0016672306008172"><li><a href="http://dx.doi.org/10.1017/s0016672306008172" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+mapping+of+allometric+scaling+laws." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+mapping+of+allometric+scaling+laws.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001245-Ma2" id="pone.0001245-Ma2"></a>Ma CX, Casella G, Wu RL (2002) Functional mapping of quantitative trait loci undelying the character process: A theoretical framework. Genetics  161: 1751–1762.  <ul class="find" data-citedArticleID="1024288"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+mapping+of+quantitative+trait+loci+undelying+the+character+process%3A+A+theoretical+framework.&amp;auth=&amp;atitle=Functional+mapping+of+quantitative+trait+loci+undelying+the+character+process%3A+A+theoretical+framework." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+mapping+of+quantitative+trait+loci+undelying+the+character+process%3A+A+theoretical+framework." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+mapping+of+quantitative+trait+loci+undelying+the+character+process%3A+A+theoretical+framework.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001245-Jiang2" id="pone.0001245-Jiang2"></a>Jiang C, Zeng Z-B (1997) Mapping quantitative trait loci with dominant and missing markers in various crosses from two inbred lines. Genetica  101: 47–58.  <ul class="find" data-citedArticleID="1024264"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+quantitative+trait+loci+with+dominant+and+missing+markers+in+various+crosses+from+two+inbred+lines.&amp;auth=&amp;atitle=Mapping+quantitative+trait+loci+with+dominant+and+missing+markers+in+various+crosses+from+two+inbred+lines." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+quantitative+trait+loci+with+dominant+and+missing+markers+in+various+crosses+from+two+inbred+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+quantitative+trait+loci+with+dominant+and+missing+markers+in+various+crosses+from+two+inbred+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001245-Diggle1" id="pone.0001245-Diggle1"></a>Diggle PJ, Heagerty P, Liang KY, Zeger SL (2002) Analysis of Longitudinal Data. Oxford, UK: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">50.
              </span><a name="pone.0001245-NezAntn1" id="pone.0001245-NezAntn1"></a>Núñez-Antón V, Zimmerman DL (2000) Modeling nonstationary longitudinal data. Biometrics  56: 699–705.  <ul class="find" data-citedArticleID="1024290" data-doi="10.1111/j.0006-341x.2000.00699.x"><li><a href="http://dx.doi.org/10.1111/j.0006-341x.2000.00699.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+nonstationary+longitudinal+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+nonstationary+longitudinal+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001245-Zimmerman1" id="pone.0001245-Zimmerman1"></a>Zimmerman DL, Núñez-Antón V (2001) Parametric modeling of growth curve data: An overview (with discussion). Test  10: 1–73.  <ul class="find" data-citedArticleID="1024360" data-doi="10.1007/bf02595823"><li><a href="http://dx.doi.org/10.1007/bf02595823" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parametric+modeling+of+growth+curve+data%3A+An+overview+%28with+discussion%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parametric+modeling+of+growth+curve+data%3A+An+overview+%28with+discussion%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001245-Louis1" id="pone.0001245-Louis1"></a>Louis TA (1982) Finding the observed information matrix when using the EM algorithm. J Roy Stat Soc B  44: 226–233.  <ul class="find" data-citedArticleID="1024282"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Finding+the+observed+information+matrix+when+using+the+EM+algorithm.&amp;auth=&amp;atitle=Finding+the+observed+information+matrix+when+using+the+EM+algorithm." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+the+observed+information+matrix+when+using+the+EM+algorithm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+the+observed+information+matrix+when+using+the+EM+algorithm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001245-Wu5" id="pone.0001245-Wu5"></a>Wu RL, Ma CX, Lin M, Casella G (2004) A general framework for analyzing the genetic architecture of developmental characteristics. Genetics  166: 1541–1551.  <ul class="find" data-citedArticleID="1024338" data-doi="10.1534/genetics.166.3.1541"><li><a href="http://dx.doi.org/10.1534/genetics.166.3.1541" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+framework+for+analyzing+the+genetic+architecture+of+developmental+characteristics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+framework+for+analyzing+the+genetic+architecture+of+developmental+characteristics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001245-Churchill1" id="pone.0001245-Churchill1"></a>Churchill GA, Doerge RW (1994) Empirical threshold values for quantitative trait mapping. Genetics  138: 963–971.  <ul class="find" data-citedArticleID="1024248"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Empirical+threshold+values+for+quantitative+trait+mapping.&amp;auth=&amp;atitle=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+threshold+values+for+quantitative+trait+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+threshold+values+for+quantitative+trait+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001245-Niklas3" id="pone.0001245-Niklas3"></a>Niklas KJ, Enquist BJ (2001) Invariant scaling relationships for interspecific plant biomass production rates and body size. Proc Natl Acad Sci USA  98: 2922–2927.  <ul class="find" data-citedArticleID="1024296" data-doi="10.1073/pnas.041590298"><li><a href="http://dx.doi.org/10.1073/pnas.041590298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invariant+scaling+relationships+for+interspecific+plant+biomass+production+rates+and+body+size." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invariant+scaling+relationships+for+interspecific+plant+biomass+production+rates+and+body+size.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001245-Zhang1" id="pone.0001245-Zhang1"></a>Zhang WK, Wang YJ, Luo GZ, Zhang JS, He CY, Wu XL, Gai JY, Chen SY (2004) QTL mapping of ten agronomic traits on the soybean (<em>Glycine</em>). Theor Appl Genet  108: 1131–1139.  <ul class="find" data-citedArticleID="1024356" data-doi="10.1007/s00122-003-1527-2"><li><a href="http://dx.doi.org/10.1007/s00122-003-1527-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=QTL+mapping+of+ten+agronomic+traits+on+the+soybean+%28Glycine%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22QTL+mapping+of+ten+agronomic+traits+on+the+soybean+%28Glycine%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001245-Wu6" id="pone.0001245-Wu6"></a>Wu RL, Ma C-X, Casella G (2007) Statistical Genetics of Quantitative Traits: Linkage, Maps, and QTL. NY: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">58.
              </span><a name="pone.0001245-Klingenberg1" id="pone.0001245-Klingenberg1"></a>Klingenberg CP, Zimmermann M (1992) Static, ontogenetic, and evolutionary allometry: A multivariate comparison in nine species of water striders. Am Nat  140: 601–620.  <ul class="find" data-citedArticleID="1024272" data-doi="10.1086/285430"><li><a href="http://dx.doi.org/10.1086/285430" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Static%2C+ontogenetic%2C+and+evolutionary+allometry%3A+A+multivariate+comparison+in+nine+species+of+water+striders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Static%2C+ontogenetic%2C+and+evolutionary+allometry%3A+A+multivariate+comparison+in+nine+species+of+water+striders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001245-Breuker1" id="pone.0001245-Breuker1"></a>Breuker C, Debat V, Klingenberg CP (2006) Functional evo-devo. Trends Ecol Syst  21: 488–492.  <ul class="find" data-citedArticleID="1024242" data-doi="10.1016/j.tree.2006.06.003"><li><a href="http://dx.doi.org/10.1016/j.tree.2006.06.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+evo-devo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+evo-devo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001245-Yap1" id="pone.0001245-Yap1"></a>Yap JS, Wang C, Wu R (2007) A computational approach for functional mapping of quantitative trait loci that regulate thermal performance curves. PLoS ONE  2(6): e554.  doi:10.1371/journal.pone.0000554.  <ul class="find" data-citedArticleID="1024342" data-doi="10.1371/journal.pone.0000554"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000554" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computational+approach+for+functional+mapping+of+quantitative+trait+loci+that+regulate+thermal+performance+curves." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computational+approach+for+functional+mapping+of+quantitative+trait+loci+that+regulate+thermal+performance+curves.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001245-vonBertalanffy1" id="pone.0001245-vonBertalanffy1"></a>von Bertalanffy L (1957) Quantitative laws for metabolism and growth. Quart Rev Biol  32: 217–231.  <ul class="find" data-citedArticleID="1024362" data-doi="10.1086/401873"><li><a href="http://dx.doi.org/10.1086/401873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+laws+for+metabolism+and+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+laws+for+metabolism+and+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001245-West5" id="pone.0001245-West5"></a>West GB, Brown JH, Enquist BJ (2001) A general model for ontogenetic growth. Nature  413: 628–631.  <ul class="find" data-citedArticleID="1024326" data-doi="10.1038/35098076"><li><a href="http://dx.doi.org/10.1038/35098076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+model+for+ontogenetic+growth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+model+for+ontogenetic+growth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001245-Hou1" id="pone.0001245-Hou1"></a>Hou W, Garvan CW, Zhao W, Behnke M, Eyler FD, Wu RL (2005) A generalized model for detecting genetic determinants underlying longitudinal traits with unequally spaced measurements and time-dependent correlated errors. Biostatistics  6: 420–433.  <ul class="find" data-citedArticleID="1024260" data-doi="10.1093/biostatistics/kxi019"><li><a href="http://dx.doi.org/10.1093/biostatistics/kxi019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+generalized+model+for+detecting+genetic+determinants+underlying+longitudinal+traits+with+unequally+spaced+measurements+and+time-dependent+correlated+errors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+generalized+model+for+detecting+genetic+determinants+underlying+longitudinal+traits+with+unequally+spaced+measurements+and+time-dependent+correlated+errors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001245-Zhao1" id="pone.0001245-Zhao1"></a>Zhao W, Zhu J, Gallo-Meagher M, Wu RL (2004) A unified statistical model for functional mapping of genotype×environment interactions for ontogenetic development. Genetics  168: 1751–1762.  <ul class="find" data-citedArticleID="1024358" data-doi="10.1534/genetics.104.031484"><li><a href="http://dx.doi.org/10.1534/genetics.104.031484" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+unified+statistical+model+for+functional+mapping+of+genotype%C3%97environment+interactions+for+ontogenetic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+unified+statistical+model+for+functional+mapping+of+genotype%C3%97environment+interactions+for+ontogenetic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.XML" value="108972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.PDF" value="199264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e001.PNG" value="7484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e001.TIF" value="23480"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e002.PNG" value="9854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e002.TIF" value="26136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e003.PNG" value="8754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e003.TIF" value="24764"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e004.PNG" value="8153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e004.TIF" value="24204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g001.PNG_L" value="206533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g001.PNG_M" value="88064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g001.PNG_S" value="5067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g001.TIF" value="40556447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g001.PNG_I" value="31421"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e005.PNG" value="10791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e005.TIF" value="27096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e006.PNG" value="8497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e006.TIF" value="24500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e007.PNG" value="8532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e007.TIF" value="24548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e008.PNG" value="20393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e008.TIF" value="35832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e009.PNG" value="17253"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e009.TIF" value="34384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e010.PNG" value="8578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e010.TIF" value="24468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e011.PNG" value="12454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e011.TIF" value="29032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e012.PNG" value="10677"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e012.TIF" value="26488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e013.PNG" value="8068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e013.TIF" value="23948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e014.PNG" value="11503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e014.TIF" value="28212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g002.PNG_L" value="175529"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g002.PNG_M" value="89415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g002.PNG_S" value="9361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g002.TIF" value="979694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g002.PNG_I" value="31955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e015.PNG" value="10327"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e015.TIF" value="26632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g003.PNG_L" value="69627"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g003.PNG_M" value="53153"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g003.PNG_S" value="5788"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g003.TIF" value="774452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g003.PNG_I" value="20353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t001.PNG_L" value="130175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t001.PNG_M" value="80508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t001.PNG_S" value="10641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t001.TIF" value="603838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t001.PNG_I" value="24363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e016.PNG" value="6967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e016.TIF" value="22964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g004.PNG_L" value="123042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g004.PNG_M" value="82263"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g004.PNG_S" value="7894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g004.TIF" value="876996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.g004.PNG_I" value="31273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t002.PNG_L" value="105975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t002.PNG_M" value="66939"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t002.PNG_S" value="9830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t002.TIF" value="479288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.t002.PNG_I" value="20771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e017.PNG" value="7789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e017.TIF" value="23736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e018.PNG" value="9896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.e018.TIF" value="25800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001245.s001.DOC" value="90624"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001245&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001245" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001245&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001245&volume=&issue=&title=A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry&author_name=Hongying%20Li%2C%20Zhongwen%20Huang%2C%20Junyi%20Gai%2C%20Song%20Wu%2C%20Yanru%20Zeng%2C%20Qin%20Li%2C%20Rongling%20Wu&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001245" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245&amp;t=A%20Conceptual%20Framework%20for%20Mapping%20Quantitative%20Trait%20Loci%20Regulating%20Ontogenetic%20Allometry" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245&title=A%20Conceptual%20Framework%20for%20Mapping%20Quantitative%20Trait%20Loci%20Regulating%20Ontogenetic%20Allometry&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245&amp;title=A%20Conceptual%20Framework%20for%20Mapping%20Quantitative%20Trait%20Loci%20Regulating%20Ontogenetic%20Allometry" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001245&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Conceptual Framework for Mapping Quantitative Trait Loci Regulating Ontogenetic Allometry';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001245';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Conceptual%20Framework%20for%20Mapping%20Quantitative%20Trait%20Loci%20Regulating%20Ontogenetic%20Allometry http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001245" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001245" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Covariance%22" title="Search for articles in the subject area:'Covariance'"><div class="flagText">Covariance</div></a>
              <div data-categoryid="33497" data-articleid="25322"
                   data-categoryname="Covariance"
                   class="flagImage" title="Flag 'Covariance' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Evolutionary+developmental+biology%22" title="Search for articles in the subject area:'Evolutionary developmental biology'"><div class="flagText">Evolutionary developmental biology</div></a>
              <div data-categoryid="34445" data-articleid="25322"
                   data-categoryname="Evolutionary developmental biology"
                   class="flagImage" title="Flag 'Evolutionary developmental biology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+mapping%22" title="Search for articles in the subject area:'Gene mapping'"><div class="flagText">Gene mapping</div></a>
              <div data-categoryid="47615" data-articleid="25322"
                   data-categoryname="Gene mapping"
                   class="flagImage" title="Flag 'Gene mapping' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25322"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="25322"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Morphogenesis%22" title="Search for articles in the subject area:'Morphogenesis'"><div class="flagText">Morphogenesis</div></a>
              <div data-categoryid="17355" data-articleid="25322"
                   data-categoryname="Morphogenesis"
                   class="flagImage" title="Flag 'Morphogenesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Soybean%22" title="Search for articles in the subject area:'Soybean'"><div class="flagText">Soybean</div></a>
              <div data-categoryid="18107" data-articleid="25322"
                   data-categoryname="Soybean"
                   class="flagImage" title="Flag 'Soybean' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Variant+genotypes%22" title="Search for articles in the subject area:'Variant genotypes'"><div class="flagText">Variant genotypes</div></a>
              <div data-categoryid="40547" data-articleid="25322"
                   data-categoryname="Variant genotypes"
                   class="flagImage" title="Flag 'Variant genotypes' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3357'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1546'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7198&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
