

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000501"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000501" />

    <meta name="citation_title" content="The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network"/>
    <meta itemprop="name" content="The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network"/>

      <meta name="citation_author" content="Duygu Balcan"/>
            <meta name="citation_author_institution" content="Department of Physics, Faculty of Sciences and Letters, Istanbul Technical University, Istanbul, Turkey"/>
      <meta name="citation_author" content="Alkan Kabakçıoğlu"/>
            <meta name="citation_author_institution" content="Department of Physics, Faculty of Arts and Sciences, Koç University, Istanbul, Turkey"/>
      <meta name="citation_author" content="Muhittin Mungan"/>
            <meta name="citation_author_institution" content="Department of Physics, Faculty of Arts and Sciences, Boğaziçi University, Istanbul, Turkey"/>
            <meta name="citation_author_institution" content="Gürsey Institute, Istanbul, Turkey"/>
      <meta name="citation_author" content="Ayşe Erzan"/>
            <meta name="citation_author_institution" content="Department of Physics, Faculty of Sciences and Letters, Istanbul Technical University, Istanbul, Turkey"/>
            <meta name="citation_author_institution" content="Gürsey Institute, Istanbul, Turkey"/>

    <meta name="citation_date" content="2007/6/6"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000501.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e501"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genomics, gene expression and DNA arrays.; citation_author=DJ Lockhart; citation_author=EA Winzeler; citation_journal_title=Nature; citation_volume=405; citation_number=1; citation_pages=827-836; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular Biology of the Cell.; citation_author=B Alberts; citation_author=A Johnson; citation_author=J Lewis; citation_author=M Raff; citation_author=K Roberts; citation_number=2; citation_date=2002; citation_publisher=Garland Science; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory networks in Saccharomyces cerevisiae.; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=F Robert; citation_author=DT Odom; citation_author=Z Bar-Joseph; citation_journal_title=Science; citation_volume=298; citation_number=3; citation_pages=799-804; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Genomic analyis of regulatory network dynamics reveals large topological changes.; citation_author=NM Luscombe; citation_author=MM Babu; citation_author=H Yu; citation_author=M Snyder; citation_author=SA Teichmann; citation_journal_title=Nature; citation_volume=431; citation_number=4; citation_pages=308-312; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The YEASTRACT database: a tool for the analysis of transcription regulatory associations in Saccharomyces cerevisiae.; citation_author=MC Teixeira; citation_author=P Monteiro; citation_author=P Jain; citation_author=S Tenreiro; citation_author=AR Fernandes; citation_journal_title=Nucl Acids Res; citation_volume=34; citation_number=5; citation_pages=D446-451; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identifying perturbation responsive key transcription factors in Saccharomyces cerevisiae.; citation_author=B Kınıkoğlu; citation_journal_title=Submitted to Yeast; citation_number=6; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Topological and causal structure of the yeast transcriptional regulatory network.; citation_author=N Guelzim; citation_author=S Bottani; citation_author=P Bourgine; citation_author=F Kepes; citation_journal_title=Nature Genetics; citation_volume=31; citation_number=7; citation_pages=60-63; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Global mapping of the yeast genetic interaction network.; citation_author=AHY Tong; citation_author=G Lesage; citation_author=GD Bader; citation_author=H Ding; citation_author=H Xu; citation_journal_title=Science; citation_volume=303; citation_number=8; citation_pages=808-813; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptional regulatory code of a eukaryotic genome.; citation_author=CT Harbison; citation_author=DB Gordon; citation_author=TI Lee; citation_author=NJ Rinaldi; citation_author=KD Macisaac; citation_journal_title=Nature; citation_volume=431; citation_number=9; citation_pages=99-104; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Metabolic stability and epigenesis in randomly constructed genetic nets.; citation_author=SA Kauffman; citation_journal_title=J Theor Biol; citation_volume=22; citation_number=10; citation_pages=437-467; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Emergence of scaling in random networks.; citation_author=AL Barabasi; citation_author=R Albert; citation_journal_title=Science; citation_volume=286; citation_number=11; citation_pages=509-512; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Network biology: understanding the cell’s functional organization.; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=Nature Reviews-Genetics; citation_volume=5; citation_number=12; citation_pages=101-113; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Exploring complex networks.; citation_author=SH Strogatz; citation_journal_title=Nature; citation_volume=410; citation_number=13; citation_pages=268-276; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Principles of transcription control in the metabolic network of S. cerevisiae.; citation_author=J Ihmels; citation_author=R Levy; citation_author=N Barkai; citation_journal_title=Nature Biotechnology; citation_volume=22; citation_number=14; citation_pages=86-92; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Back to the biology in systems biology: What can we learn from biomolecular networks?; citation_author=S Huang; citation_journal_title=Brief Funct Gen Prot; citation_volume=2; citation_number=15; citation_pages=279-297; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Random model for RNA interference yields scale free network.; citation_author=D Balcan; citation_author=A Erzan; citation_journal_title=Eur Phys J B; citation_volume=38; citation_number=16; citation_pages=253-260; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analytical solution of a stochastic content-based network model.; citation_author=M Mungan; citation_author=A Kabakçıoğlu; citation_author=D Balcan; citation_author=A Erzan; citation_journal_title=J Phys A 38; citation_volume=(44); citation_number=17; citation_pages=9599-9620; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Scaling properties of coding and non-coding DNA sequences.; citation_author=Y Almirantis; citation_author=A Provata; citation_journal_title=J Stat Phys; citation_volume=97; citation_number=18; citation_pages=233-262; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Modern Graph Theory.; citation_author=B Bollobas; citation_number=19; citation_date=1998; citation_publisher=Springer Verlag; " />
      <meta name="citation_reference" content="citation_title=Statistical mechanics of complex networks.; citation_author=R Albert; citation_author= Barabasi; citation_author= AL; citation_journal_title=Rev Mod Phys; citation_volume=74; citation_number=20; citation_pages=47-97; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Collective dynamics of ‘small world’ networks.; citation_author=DJ Watts; citation_author=SH Strogatz; citation_journal_title=Nature; citation_volume=393; citation_number=21; citation_pages=440-442; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evolution of networks.; citation_author=SN Dorogovstsev; citation_author=JFF Mendes; citation_journal_title=Adv Phys; citation_volume=51; citation_number=22; citation_pages=1079-1187; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Dynamical and correlation properties of the internet.; citation_author=R Pastor-Satorras; citation_author=A Vazquez; citation_author=A Vespignani; citation_journal_title=Phys Rev Lett; citation_volume=87; citation_number=23; citation_pages=258701; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Characterization and modeling of protein-protein interaction networks.; citation_author=V Colizza; citation_author=A Flammini; citation_author=A Maritan; citation_author=A Vespignani; citation_journal_title=Physica A; citation_volume=352; citation_number=24; citation_pages=1-27; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The rich-club phenomenon in the internet topology.; citation_author=S Zhou; citation_author=RJ Mondragon; citation_journal_title=IEEE Commun Lett; citation_volume=8; citation_number=25; citation_pages=180-182; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detecting rich-club ordering in complex networks.; citation_author=V Colizza; citation_author=A Flammini; citation_author=MA Serrano; citation_author=A Vespignani; citation_journal_title=Nature Physics; citation_volume=2; citation_number=26; citation_pages=110-115; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Aggregation of topological motifs in the Escherichia coli transcriptional regulatory network.; citation_author=R Dobrin; citation_author=QK Beg; citation_author=AL Barabasi; citation_author=ZN Oltvai; citation_journal_title=BMC Bioinformatics; citation_volume=5; citation_number=27; citation_pages=10; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Network motifs: simple building blocks of complex networks.; citation_author=R Milo; citation_author=S Shen-Orr; citation_author=S Itzkovitz; citation_author=N Kashtan; citation_author=D Chklovskii; citation_journal_title=Science; citation_volume=298; citation_number=28; citation_pages=824-827; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Revealing modular organization in the yeast transcriptional network.; citation_author=J Ihmels; citation_author=G Friedlander; citation_author=S Bergmann; citation_author=O Sarig; citation_author=Y Ziv; citation_journal_title=Nature Genetics; citation_volume=31; citation_number=29; citation_pages=370-377; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sequenceing and comparison of yeast species to identify genes and regulatory elements.; citation_author=M Kellis; citation_author=N Patterson; citation_author=M Endrizzi; citation_author=B Birren; citation_author=ES Lander; citation_journal_title=Nature; citation_volume=423; citation_number=30; citation_pages=241-254; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The topological relationship between the large-scale attributes and local interaction patterns of complex networks.; citation_author=A Vazques; citation_author=R Dobrin; citation_author=D Sergi; citation_author=J-P Eckmann; citation_author=ZN Oltvai; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=31; citation_pages=17940-17945; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Similarities and differences in genome-wide expression data of six organisms.; citation_author=S Bergmann; citation_author=J Ihmels; citation_author=N Barkai; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=32; citation_pages=85-93; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immunology for physicists.; citation_author=AS Perelson; citation_author=G Weisbuch; citation_journal_title=Rev Mod Phys; citation_volume=69; citation_number=33; citation_pages=1219-1267; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Dynamics of Gene Expression in an Artificial Genome.; citation_author=T Reil; citation_number=34; citation_pages=457-466; citation_date=1999; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Structure and dynamics of a gene network model incorporating small RNAs.; citation_author=N Geard; citation_author=J Wiles; citation_number=35; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Towards more biological mutation operators in gene regulation studies.; citation_author=J Watson; citation_author=N Geard; citation_author=J Wiles; citation_journal_title=BioSystems; citation_volume=76; citation_number=36; citation_pages=239-248; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The yeast coexpression network has a small-world, scale-free architecture and can be explained by a simple model.; citation_author=V van Noort; citation_author=B Snel; citation_author=MA Huynen; citation_journal_title=EMBO Rep; citation_volume=5; citation_number=37; citation_pages=280-284; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Network motifs in natural and artificial transcriptional regulatory networks.; citation_author=W Banzhaf; citation_author=PD Kuo; citation_journal_title=J Biol Phys Chem; citation_volume=4; citation_number=38; citation_pages=85-92; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolution of gene networks by gene duplications: a mathematical model and its implications on genome organization.; citation_author=A Wagner; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=91; citation_number=39; citation_pages=4387-4391; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The yeast protein interaction network evolves rapidly and contains few redundant duplicate genes.; citation_author=A Wagner; citation_journal_title=Mol Biol Evol; citation_volume=18(7); citation_number=40; citation_pages=1283-1292; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Complex Networks in Genomics and Proteomics.; citation_author=RV Sole; citation_author=R Pastor-Satorras; citation_number=41; citation_date=2002; citation_publisher=Wiley-VCH Verlag; " />
      <meta name="citation_reference" content="citation_title=1st and 2nd moment of counts of words in random texts generated by Markov-chains.; citation_author=J Kleffe; citation_author=M Borodovsky; citation_journal_title=Comp Appl Biosci; citation_volume=8; citation_number=42; citation_pages=433-441; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Communication in the Presence of Noise.; citation_author=CE Shannon; citation_journal_title=Proc IRE; citation_volume=37; citation_number=43; citation_pages=10-21; citation_date=1949; " />
      <meta name="citation_reference" content="citation_title=A critical point for random graphs with a given degree sequence.; citation_author=M Molloy; citation_author=B Reed; citation_journal_title=Random Structures and Algorithms; citation_volume=6; citation_number=44; citation_pages=161-180; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Scale-free networks from varying vertex intrinsic fitness.; citation_author=G Caldarelli; citation_author=A Capocci; citation_author=P De Los Rios; citation_author=MA Munoz; citation_journal_title=Phys Rev Lett; citation_volume=89; citation_number=45; citation_pages=258702; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A proposal for using the ensemble approach to understand genetic regulatory networks.; citation_author=S Kauffman; citation_journal_title=J Theor Biol; citation_volume=230; citation_number=46; citation_pages=581-590; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=On information and sufficiency.; citation_author=S Kullback; citation_author=RA Leibler; citation_journal_title=Annals of Mathematical Statistics; citation_volume=22; citation_number=47; citation_pages=79-86; citation_date=1951; " />
      <meta name="citation_reference" content="citation_title=MEDUSA-New Model of Internet Topology Using k-shell Decomposition. cond-mat/0601240. arXiv preprint.; citation_author=S Carmi; citation_author=S Havlin; citation_author=S Kirkpatrick; citation_author=Y Shavitt; citation_author=E Shir; citation_number=48; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network"/>
    <meta name="twitter:description" content="The regulation of gene expression in a cell relies to a major extent on transcription factors, proteins which recognize and bind the DNA at specific binding sites (response elements) within promoter regions associated with each gene. We present an information theoretic approach to modeling transcriptional regulatory networks, in terms of a simple &ldquo;sequence-matching&rdquo; rule and the statistics of the occurrence of binding sequences of given specificity in random promoter regions. The crucial biological input is the distribution of the amount of information coded in these cognate response elements and the length distribution of the promoter regions. We provide an analysis of the transcriptional regulatory network of yeast Saccharomyces cerevisiae, which we extract from the available databases, with respect to the degree distributions, clustering coefficient, degree correlations, rich-club coefficient and the k-core structure. We find that these topological features are in remarkable agreement with those predicted by our model, on the basis of the amount of information coded in the interaction between the transcription factors and response elements."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000501.g003"/>

  <meta property="og:title" content="The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2965'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7803'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6462&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000501">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000501" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000501&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Duygu Balcan
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:balcand@indiana.edu">balcand@indiana.edu</a></p>

                <p>Affiliation:
                  Department of Physics, Faculty of Sciences and Letters, Istanbul Technical University, Istanbul, Turkey
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alkan Kabakçıoğlu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics, Faculty of Arts and Sciences, Koç University, Istanbul, Turkey
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Muhittin Mungan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Physics, Faculty of Arts and Sciences, Boğaziçi University, Istanbul, Turkey, 
                  Gürsey Institute, Istanbul, Turkey
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ayşe Erzan
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Physics, Faculty of Sciences and Letters, Istanbul Technical University, Istanbul, Turkey, 
                  Gürsey Institute, Istanbul, Turkey
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 06, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000501</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000501-t001" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000501-g001" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000501-g002" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000501-g003" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000501">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000501" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The regulation of gene expression in a cell relies to a major extent on transcription factors, proteins which recognize and bind the DNA at specific binding sites (response elements) within promoter regions associated with each gene. We present an information theoretic approach to modeling transcriptional regulatory networks, in terms of a simple “sequence-matching” rule and the statistics of the occurrence of binding sequences of given specificity in random promoter regions. The crucial biological input is the distribution of the amount of information coded in these cognate response elements and the length distribution of the promoter regions. We provide an analysis of the transcriptional regulatory network of yeast <em>Saccharomyces cerevisiae</em>, which we extract from the available databases, with respect to the degree distributions, clustering coefficient, degree correlations, rich-club coefficient and the <em>k</em>-core structure. We find that these topological features are in remarkable agreement with those predicted by our model, on the basis of the amount of information coded in the interaction between the transcription factors and response elements.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Balcan D, Kabakçıoğlu A, Mungan M, Erzan A (2007) The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network. PLoS ONE 2(6):
          e501.
            doi:10.1371/journal.pone.0000501</p><p><strong>Academic Editor: </strong>Gustavo Stolovitzky, IBM Thomas J. Watson Research Center, United States of America</p><p><strong>Received:</strong> December 7, 2006; <strong>Accepted:</strong> May 9, 2007; <strong>Published:</strong> June 6, 2007</p><p><strong>Copyright:</strong> © 2007 Balcan et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>AE is grateful for partial support from the Turkish Academy of Sciences.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>With the development of high throughput experimental techniques <a href="#pone.0000501-Lockhart1">[1]</a> a large amount of data on gene interactions <a href="#pone.0000501-Alberts1">[2]</a> is now available <a href="#pone.0000501-Lee1">[3]</a>–<a href="#pone.0000501-Knkolu1">[6]</a>, revealing a complex network. The organizational principles underlying these genetic regulatory networks are of great experimental <a href="#pone.0000501-Lee1">[3]</a>,  and theoretical <a href="#pone.0000501-Kauffman1">[10]</a>–<a href="#pone.0000501-Huang1">[15]</a> interest.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In this paper we would like to present an information theoretic approach to modeling genetic interaction networks. We believe that this approach provides an understanding of how interactions based on shared information might arise spontaneously between subsequences of any sufficiently long linear code, even when this code is completely random, and how a complex network emerges as a result.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We construct a null model of a transcriptional regulatory network (TRN) by adapting the “sequence-matching” rule which we have introduced earlier <a href="#pone.0000501-Balcan1">[16]</a>, <a href="#pone.0000501-Mungan1">[17]</a> as a condition for the establishment of edges between nodes of a network. A sequence S is said to match another sequence S′ if it is contained in S′ as an uninterrupted subsequence. In the case of the TRN, the nodes consist of genes, with their associated promoter regions (PRs) and the transcription factors (TFs) which they code, if any. To model this network, we label TFs by the binding sequences they recognize, and represent the PRs by another set of (typically longer) sequences. The conditions that need to be satisfied for a TF to recognize and bind a specific DNA sequence within a promoter region are mimicked by the sequence-matching rule between the sequences associated with the respective TFs and PRs.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The biological input to the model consists of the effective length distribution of the binding sequences recognized by the transcription factors of yeast <a href="#pone.0000501-Lee1">[3]</a>, <a href="#pone.0000501-Harbison1">[9]</a> and the form of the length distribution of the intergenic regions <a href="#pone.0000501-Almirantis1">[18]</a>, in the absence of more specific data regarding the lengths of the promoter regions. By effective length we mean the bit-wise information content (site specificity) of a binding motif together with its variations and experimental uncertainties in its determination. The empirical binding sequences of the TFs are reported <a href="#pone.0000501-Lee1">[3]</a>, <a href="#pone.0000501-Harbison1">[9]</a> in an extended alphabet (e.g. “rACGCGt” for the transcription factor MBP1) specifying the base preferences or the binding affinities as a set of letters with variable case. We have converted the information contents coded via this extended alphabet into to a binary code by assigning zero, one and two bits representing low, medium and high information contents (see <a href="#s5">the Results and the Methods</a> sections).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Our model is a null model in the sense that no further knowledge specific to the TRN of yeast is provided apart from the distribution of the amount of information coded in the cognate response elements; no assumptions are made regarding the actual amino acid code of a TF, nor its possible three dimensional folding pattern; all possible refinements involving steric constraints, chromatin folding, trans-vs. cis-regulation, etc. have been neglected. A TF is simply labeled by a binary string whose length is drawn from the effective bit-length distribution of the cognate response elements, and this string is then queried in the randomly composed strings representing the PRs, to establish an interaction. Since the specific content of the strings does not enter into the assumptions of the model, the strings are composed randomly, with uniform probabilities. Thus our null model is designed to test the null hypothesis that the topology of the TRN is essentially determined by the exchange of information between the cognate/cognate response elements, of given length distributions.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In order to test the predictions of this model against real data, we make a detailed analysis of the topological features of the directed network corresponding to the TRN of yeast using the available data <a href="#pone.0000501-Lee1">[3]</a>–<a href="#pone.0000501-Knkolu1">[6]</a> (see <a href="#pone-0000501-t001">Table 1</a> for the databases used). We have also investigated (supporting <a href="#pone.0000501.s001">Text S1</a>) the frequency of “3-motifs,” namely the frequency of occurrence of different directed edge configurations which may be found within connected sub-graphs containing 3 nodes.</p>
<div class="figure" id="pone-0000501-t001"><div class="img"><a name="pone-0000501-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000501.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000501.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Yeast databases.</span></strong></p><span>doi:10.1371/journal.pone.0000501.t001</span></div><a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>We demonstrate that our model is able to capture with convincing precision all the global topological features of this directed network such as the distribution of the in-, out- and total degree, i.e., number of neighbors per node <a href="#pone.0000501-Bollobas1">[19]</a>–<a href="#pone.0000501-Dorogovstsev1">[22]</a>, the clustering coefficient <a href="#pone.0000501-Bollobas1">[19]</a>–<a href="#pone.0000501-Dorogovstsev1">[22]</a> measuring the probability that the neighbors of a node are connected to each other, the degree-degree correlation <a href="#pone.0000501-PastorSatorras1">[23]</a>, <a href="#pone.0000501-Colizza1">[24]</a>, namely the correlation between the respective number of neighbors of neighboring nodes, the rich-club coefficient <a href="#pone.0000501-Zhou1">[25]</a>, <a href="#pone.0000501-Colizza2">[26]</a> indicating the extent of clustering among highly connected nodes, and the <em>k</em>-core structure <a href="#pone.0000501-Bollobas1">[19]</a>, displaying the hierarchical organization of the links. This thorough topological characterization allows us to discriminate between our model and pared-down versions thereof, which capture some but not all of the above features of the yeast network. In this sense the model we present here is a minimal null model.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>The focuses of both empirical and theoretical network-theory approaches to gene regulatory networks have been studies of the degree distribution <a href="#pone.0000501-Guelzim1">[7]</a>, <a href="#pone.0000501-Tong1">[8]</a> and network motifs <a href="#pone.0000501-Dobrin1">[27]</a>–<a href="#pone.0000501-Kellis1">[30]</a>. Barabasi and co-workers <a href="#pone.0000501-Vazques1">[31]</a> have claimed that the global properties of gene regulatory networks of yeast (<em>Saccharomyces cerevisiae</em>) and <em>Escherichia coli</em>, as well as protein-protein interaction and metabolic networks, can be modeled by the preferential attachment <a href="#pone.0000501-Barabasi1">[11]</a>, <a href="#pone.0000501-Albert1">[20]</a> rule and that these networks are scale-free, with the degree distribution having a scaling exponent <a href="#pone.0000501-Dobrin1">[27]</a><em>γ</em>~2.5. Smaller values for this exponent can also be found in the literature <a href="#pone.0000501-Tong1">[8]</a>, <a href="#pone.0000501-Bergmann1">[32]</a>. Bergmann et al. <a href="#pone.0000501-Bergmann1">[32]</a> have suggested that the degree distribution might have a universal scale-free behavior independent of any particular organism. The claims of scale-invariance are typically based on linear fits to the log-log plots of the degree distribution data over narrow intervals of about two decades or less, and with imperfect agreement between the data and the fit. The careful analysis of Guelzim et al. <a href="#pone.0000501-Guelzim1">[7]</a> has revealed that the in- and out-degree distributions are rather different, with the former having an exponential-like decay and being confined to a much narrower range. Regarding the local organization of such networks, statistics of the various <em>n</em>-node motifs have been reported to be significantly different from randomized versions of the same network <a href="#pone.0000501-Dobrin1">[27]</a>–<a href="#pone.0000501-Kellis1">[30]</a>, and therefore assumed to be of functional or evolutionary significance.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>It should be mentioned that the idea of using the matching of linear codes, as embodied in our sequence-matching rule, to model the satisfaction of a broad set of requirements for the binding of proteins to other molecules, is not entirely new. Complementarity of binary sequences (“bit-strings”) of fixed uniform length representing anticores and the antigens which “recognize” them have been employed in modeling immune networks in the early 1990’s <a href="#pone.0000501-Perelson1">[33]</a>, although the emphasis at this stage was more on the dynamics of small networks constructed in this way, than on their topological features. There have also been several earlier studies of models of gene regulatory networks on rather elaborate “Artificial Genomes” <a href="#pone.0000501-Reil1">[34]</a> based on various alphabets and matching rules <a href="#pone.0000501-Geard1">[35]</a>–<a href="#pone.0000501-Banzhaf1">[38]</a>, some of them coupled with the duplication and divergence model introduced by Wagner <a href="#pone.0000501-Wagner1">[39]</a>–<a href="#pone.0000501-Sole1">[41]</a>. The results are not uniform and depend on the detailed assumptions made in the models.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>The organization of the paper is as follows. In the Analyses section, we elaborate in some detail the idea of an information-theoretic approach to interaction networks. We explain the sequence-matching rule and discuss the biological input that goes into our model. In the Results section, our model simulations are compared with a thorough characterization of the gene regulatory network of yeast using the most comprehensive available data (see <a href="#pone-0000501-t001">Table 1</a>). Here we also present the results of comparisons with several, more restricted models, and with randomized networks. Our conclusions are presented under Discussion. The <a href="#s5">Methods</a> section is reserved for more technical details.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Analyses"></a><h3>Analyses</h3>
<h4>Information theoretic approach to interaction networks</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The information-theoretic approach we would like to present in this paper is quite generic and promises to be widely applicable to systems which can be described in terms of networks of interacting nodes. In this approach, an interaction, represented as an edge connecting a pair of nodes, is established if and only if a number of more or less stringent constraints are fulfilled. The number and strictness of the constraints may be quantified as a certain amount of information, or code, that has to be shared between the two nodes. The topology of the interaction network is then determined by the distribution function of the required amount of shared information between the interacting nodes.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The way in which we model the shared information, corresponding to a set of constraints, is via a string-matching condition we have introduced earlier <a href="#pone.0000501-Balcan1">[16]</a>, <a href="#pone.0000501-Mungan1">[17]</a>. In this “content-based model,” the condition for establishing a connection is that a code, represented by a string associated with one node, match, letter for letter, a sub-string of the code associated with another node. In this model, matching each successive letter will correspond to satisfying an additional constraint. The number of constraints can thus be mapped to the length of a string to be matched. The chance satisfaction of the constraints is smaller, the longer the strings, or the larger the alphabet from which they are constructed.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>In fact, it can be shown that to a first approximation the probability that a random string of length <em>l</em><sub>1</sub> is included inside another random string of length <em>l</em><sub>2</sub> (<em>l</em><sub>1</sub>≤<em>l</em><sub>2</sub>) is given by <a href="#pone.0000501-Mungan1">[17]</a>,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>r</em> is the number of letters in the alphabet from which the strings are chosen independently and with equal probability for each letter (1/4 for four nucleic acids, 1/2 for a binary code etc.). This approximation is valid for lengths <em>l</em><sub>2</sub>−<em>l</em><sub>1</sub> smaller than or comparable to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e002&amp;representation=PNG" class="inline-graphic"></span>, which amounts to the condition that correlations arising from multiple occurrences of the shorter string inside the longer one can be ignored. In fact, when <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e003&amp;representation=PNG" class="inline-graphic"></span> the above expression reduces to<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>which is the naive result obtained by treating the probabilities of a match of the shorter string along the <em>l</em><sub>2</sub>−<em>l</em><sub>1</sub>+1 positions on the longer string as independent. Also note that the average number of occurrences, 〈<em>n</em>〉, of the shorter randomly drawn string inside the longer one is given exactly <a href="#pone.0000501-Kleffe1">[42]</a> by the right hand side of Eq. (2). We see that Markov′s Inequality for integral valued random variables implies that Prob(<em>n</em>≥1)≤〈<em>n</em>〉 and re-expresses the fact that for <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e005&amp;representation=PNG" class="inline-graphic"></span> the probability of having more than one occurrence can be neglected.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>To use a “lock and key” analogy to illustrate the idea of simultaneously satisfying a number of constraints, the first string may be regarded as the “key” combination that opens the “lock,” which in this case may be opened by more than one key (at most <em>l</em><sub>2</sub>−<em>l</em><sub>1</sub>+1 keys). The probability of a chance hit on one of the right combinations decreases exponentially with <em>l</em><sub>1</sub>, as exp(−<em>l</em><sub>1</sub> ln <em>r</em>). Note that −<em>l</em><sub>1</sub> ln <em>r</em> is in fact the so called “Shannon information” <a href="#pone.0000501-Shannon1">[43]</a> of a random “key,” selected from an alphabet of <em>r</em> letters.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>We may easily compute the information <a href="#pone.0000501-Shannon1">[43]</a> coded in a string of a given length, whose loci may have different sets of probabilities for encountering different letters. This quantity can then be used to define an effective length for a corresponding string of random letters. This is described in the <a href="#s5">Methods</a> section.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>This approach seems to be particularly well-suited to the description of gene regulatory networks, which operate on a cognate/cognate response element basis. TFs are the cognates, which bind the cognate response elements, i.e., the binding sequences (regulatory sequences) within the PRs of different genes. The “key” to the promoter region, so to speak, is the binding motif.</p>


<h4>Sequence matching model for the transcription regulatory network</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The nodes of our model network correspond to genes, only a small percentage of which code for transcription factors. With each node we associate a sequence, which represents the PR through which the corresponding gene may be regulated. With those nodes/genes coding TFs, we also associate a second sequence, uncorrelated with the first, representing the binding motif recognized by the TF. For simplicity we assume that there is only one transcription factor that is coded by each regulatory gene. (see <a href="#pone-0000501-g001">Fig. 1a</a>)</p>
<div class="figure" id="pone-0000501-g001"><div class="img"><a name="pone-0000501-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000501.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000501.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The model.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(a) The mechanism of interaction between the genes as envisaged in our model. The genes are indicated by ellipses (green if they code transcription factors (TFs), blue otherwise), the TFs by triangles with the associated binding motif (regulatory sequence) in the box underneath. Non-TF proteins are symbolized by the “P” shape, and the promoter regions (PRs) upstream of each gene are shown as red boxes. Binding occurs if the binding motif exactly matches a subsequence in the PR, as is the case here at PR4. PRs in the model are typically much longer than depicted here. (b) Distribution of the amount of bit-wise information coded by each regulatory sequence recognized and bound by the 102 TFs in the yeast genome, compiled from the recently published data by Harbison et al. <a href="#pone.0000501-Harbison1">[9]</a>. This distribution is adopted as the length distribution of the random regulatory sequences (“binding motifs”) in our model.</p>
<span>doi:10.1371/journal.pone.0000501.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In our model the binding motifs and the PRs are represented as random binary sequences (thus, with <em>r</em> = 2), whose lengths obey different probability distributions. The TF binding motifs are typically short sequences with a narrow length distribution <a href="#pone.0000501-Lee1">[3]</a>, <a href="#pone.0000501-Harbison1">[9]</a>, since a TF selectively binds 5–10 bases and not much more. A single TF can bind a number of similar sequences, and we have used the information content of the binding motifs representing these sequences in order to obtain a distribution of effective lengths (see <a href="#pone-0000501-g001">Fig. 1b</a>) for the randomly generated binary sequences representing our TF binding motifs. This is described in the <a href="#s5">Methods</a> section.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>For the PRs we make the assumption that the lengths are distributed in the same way as the lengths of the intergenic regions, obeying long tailed power-law distributions <a href="#pone.0000501-Almirantis1">[18]</a> whose exponent is the only free parameter in the model, and will be determined from a comparison of the topological features of the model and the experimental regulatory networks, as described in the next section. We have tested whether a simpler assumption regarding the distribution of the PR lengths could also suffice. Taking a constant PR length for all genes leads to similar qualitative behavior, although it performs worse when compared with actual yeast data (see supporting <a href="#pone.0000501.s002">Text S2</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The amount of information coded in these randomly generated binding motifs and promoter regions thus constitutes the essential biological ingredient of our model and dictates the overall topology of the resultant networks.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>The mechanism for establishing connections between nodes of the gene regulatory network is given by the string matching condition <a href="#pone.0000501-Balcan1">[16]</a>, <a href="#pone.0000501-Mungan1">[17]</a> described above, between the binding motifs of the TFs and all possible uninterrupted subsequences of the PRs. The (directed) network of regulatory gene interactions is obtained by drawing a directed link from each TF-producing node A to all those nodes B, B′, B″,… whose PRs contain the binding motif associated with the TF coded by node A (see <a href="#pone-0000501-g001">Fig. 1a</a> and supporting <a href="#pone.0000501.s003">Text S3</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>The yeast transcriptional regulatory network</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To make a quantitative comparison with the yeast transcriptional regulatory network possible, we choose the total number of genes and the proportion of those genes coding for transcription factors (4.8%, see <a href="#pone-0000501-t001">Table 1</a>) in conformity with the Yeastract data set <a href="#pone.0000501-Teixeira1">[5]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The length distribution of the binding motifs in the model genome was derived from the yeast data provided by Harbison et al. <a href="#pone.0000501-Harbison1">[9]</a>, where the motifs were reported as letter sequences comprising the symbols for the four bases {ATGC}, or the symbols {YMKRSW} signifying a preference for any two out of four bases, etc., with the corresponding lower case letters indicating a lower confidence level. In order to account for such variations in the information content of the motifs, we assigned two bits to each of the letters {ATGC} appearing in the motif, signifying a high information content at that position, one bit to each of the letters {atgcYMKRSWymkrsw} and zero bits otherwise. The length of the bit sequence obtained in this way roughly corresponds to the amount of shared information, measured by the Shannon entropy <a href="#pone.0000501-Shannon1">[43]</a>, required for the binding of the TF. Performing this calculation (see the <a href="#s5">Methods</a> section) for each TF <a href="#pone.0000501-Harbison1">[9]</a>, we obtain the length distribution shown in <a href="#pone-0000501-g001">Fig. 1b</a>.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>We assume that the lengths of the PRs follow a power law distribution similar to that of the intergenic regions <a href="#pone.0000501-Almirantis1">[18]</a>, with <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where 0≤<em>μ</em>≤2. We also stipulate that <em>l</em> is restricted to the interval <em>l</em><sub>min</sub>≤<em>l</em>≤<em>l</em><sub>max</sub>, where <em>l</em><sub>min</sub> coincides with the peak of the motif-length distribution shown in <a href="#pone-0000501-g001">Fig. 1b</a>, while <em>l</em><sub>max</sub>−<em>l</em><sub>min</sub>+1 = 250. In this choice we are guided by the finding <a href="#pone.0000501-Harbison1">[9]</a> that most of the probability for encountering a TF binding site is contained within a window of 250 base pairs (bps) located approximately 100 bps upstream of a gene. Note that the 250 bps window does not double as we move from the four-letter alphabet to a binary one, because the matching probabilities and the total number of positions at which the TFs may bind are required to remain invariant under this transformation. This amounts to preserving the average number of matches of a four-letter RS of length <em>l</em><sub>1</sub> in a four-letter PR of length <em>l</em><sub>2</sub>, which is given by the right hand side of Eq. (2) (see the Analyses section). The denominator in Eq. (2), namely <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e007&amp;representation=PNG" class="inline-graphic"></span>, remains invariant under the transformation from a four- to a two-letter alphabet, as can be seen from the <a href="#s5">Methods</a> section. For the numerator of Eq. (2) to also remain unaltered, <em>l</em><sub>2</sub> should be changed such that <em>l</em><sub>2</sub>−<em>l</em><sub>1</sub> remains approximately unchanged; in particular, for <em>l</em><sub>1</sub>≪<em>l</em><sub>2</sub>, it means that <em>l</em><sub>2</sub> should be left unchanged.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>Once the shape of the length distribution of the binding sequences and the functional form, as well as the support, of the length distribution of the PRs have been fixed through the available biological data, the only remaining adjustable parameter in our model is the exponent <em>μ</em> of the power law distribution of PR lengths, <em>p</em><sub>PR</sub>(<em>l</em>).</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Clearly, the length distribution for the PRs must be tested against null assumptions, and this we do in <a href="#pone.0000501.s002">Text S2</a>. We find that, once the form of the distribution has been chosen as in Eq. (3), any value of <em>μ</em> within the interval [0,2] performs reasonably well, while, say, fixing all the PR lengths to be identical gives markedly different results.</p>
<a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>In order to optimize the value of <em>μ</em>, we could compare all the available topological characterizations of randomly generated model networks obeying the constraints on the number of nodes, the length distribution of the binding sequences, for different values of 0≤<em>μ</em>≤2, with those of the yeast TRN. It is obviously desirable, however, to find one number to compare with experiment, rather than, say, the whole degree distribution or the degree-degree correlation function <em>k</em><sub>nm</sub>(<em>k</em>). In fact, once <em>p</em><sub>PR</sub>(<em>l</em>) is chosen to be of the power-law form given in Eq. (3), <em>then</em> choosing <em>μ</em> so that <em>k</em><sub>max</sub>, the maximum number of <em>k</em>-cores <a href="#pone.0000501-Bollobas1">[19]</a> of the model, coincides with that of the network obtained from one of the yeast data sources, is sufficient for the rest of the topological features of the respective networks to fall right on top of each other, as shown in <a href="#pone-0000501-g002">Fig. 2</a> and <a href="#pone-0000501-g003">Fig. 3</a>.</p>
<div class="figure" id="pone-0000501-g002"><div class="img"><a name="pone-0000501-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000501.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000501.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The hierarchical structure of the TRN as revealed by the k-core decomposition.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The <em>k</em>-core visualization of a single realization of our model network (Left) obtained with the visualization tool lanet-vi (<a href="http://arxiv.org/abs/cs.NI/0504107">http://arxiv.org/abs/cs.NI/0504107</a>) The length distribution exponent of the PR sequences has been adjusted to <em>μ</em> = 0.1 to match the number of <em>k</em>-cores to that obtained from Yeastract data (Right). Dots represent the nodes of the network, while edges between nodes depict connections. Nodes belonging to different <em>k</em>-shells are indicated by different colors (on the right hand side) and are arranged around concentric circles, whose average radius decreases with <em>k</em>. In particular, a node of a given shell is placed just inside (outside) the corresponding circle, if it is preferentially connected to lower (higher) <em>k</em>-shells. The sizes of the dots indicate the degree of the respective nodes; see legends to the left of the figures for representative sizes.</p>
<span>doi:10.1371/journal.pone.0000501.g002</span></div><div class="figure" id="pone-0000501-g003"><div class="img"><a name="pone-0000501-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000501" data-uri="info:doi/10.1371/journal.pone.0000501.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000501.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000501.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000501.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000501.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Topological features.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>Above: Degree distributions extracted from the Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> data (red circles), superposed on the corresponding degree distributions of one hundred realizations of the model network (black dots). From left to right, (a) The total degree distribution with an inset showing a log-linear plot for <em>k</em>/<em>k</em><sub>av</sub>≤10, where one may observe that both the model and the data points almost fall on a straight line. (b) The in-degree distribution plotted on a semi-logarithmic scale. (c) The out-degree distribution plotted on a log-log scale. The axes are scaled by the appropriate average total degree in order to factor out sample-to-sample fluctuations in the network size. Below: Comparison of (d) the clustering coefficient <em>c</em>(<em>k</em>), (e) the typical degree-degree correlations between neighboring nodes <em>k</em><sub>nm</sub>(<em>k</em>), and (f) the rich-club coefficient <em>r</em>(<em>k</em>), from left to right, for 100 realizations of the model (black dots) and the Yeastract data (red circles).</p>
<span>doi:10.1371/journal.pone.0000501.g003</span></div><a id="article1.body1.sec3.sec1.p7" name="article1.body1.sec3.sec1.p7"></a><p>We build our ensemble of model networks starting from 6000 nodes. Out of these, almost all the nodes with nonzero degree belong to the largest connected component (see <a href="#pone-0000501-t001">Table 1</a>), whose size depends on the value of <em>μ</em>. The analyses for the degree distributions, the rich-club coefficient and the <em>k</em>-core structure have been performed for all the nodes with nonzero degree, while the clustering coefficient and the degree-degree correlation have been calculated on the largest connected component.</p>
<a id="article1.body1.sec3.sec1.p8" name="article1.body1.sec3.sec1.p8"></a><p>In <a href="#pone-0000501-g002">Fig. 2</a>, we show the <em>k</em>-core visualization (obtained by means of the LaNet-Vi tool, <a href="http://arxiv.org/abs/cs.NI/0504107">http://arxiv.org/abs/cs.NI/0504107</a>) of one realization of the model network (left) and the Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> data (right). Here <em>μ</em> has been fixed to 0.1, making the mean and the mode of <em>k</em><sub>max</sub> for the model ensemble to coincide with the value we compute from the Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> database, at <em>k</em><sub>max</sub> = 9 (see <a href="#pone.0000501.s002">Text S2</a> for details). Both the model and the experimental network exhibit a highly hierarchical structure with a nested sequence of <em>k</em>-shells and an almost exclusively radial arrangement of the edges. The distinct hierarchical organization of the edges is not very sensitive to the precise value of <em>μ</em>, while the total number of shells decreases as <em>μ</em> increases. (see <a href="#pone.0000501.s002">Text S2</a>)</p>
<a id="article1.body1.sec3.sec1.p9" name="article1.body1.sec3.sec1.p9"></a><p>In <a href="#pone-0000501-g003">Fig. 3</a>, we report our results for the in-, out- and total degree distribution <a href="#pone.0000501-Bollobas1">[19]</a>–<a href="#pone.0000501-Dorogovstsev1">[22]</a>, the clustering coefficient <a href="#pone.0000501-Bollobas1">[19]</a>–<a href="#pone.0000501-Dorogovstsev1">[22]</a>, the degree-degree correlation <a href="#pone.0000501-PastorSatorras1">[23]</a>, <a href="#pone.0000501-Colizza1">[24]</a> and the rich-club coefficient <a href="#pone.0000501-Zhou1">[25]</a>, <a href="#pone.0000501-Colizza2">[26]</a> (the precise definitions of which are given in the <a href="#s5">Methods</a> section), with the choice of <em>μ</em> = 0.1, i.e., the topological features displayed in <a href="#pone-0000501-g003">Fig. 3</a> are obtained without any further adjustment of <em>μ</em>. Results for the yeast TRN, which we have extracted from the Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> data have been superposed on the scatter plots of one hundred independent realizations of randomly generated model networks with identical parameters.</p>
<a id="article1.body1.sec3.sec1.p10" name="article1.body1.sec3.sec1.p10"></a><p>The total degree distribution is obtained by ignoring the directionality of the interactions and is generally different from the superposition of in- and out-degree distributions. In <a href="#pone-0000501-g003">Fig. 3a</a>, Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> data for the degree distribution is shown on top of a scatter plot obtained by superposing the results of the ensemble of model networks. The average total degree of the yeast TRN extracted from Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> is <em>k</em><sub>Yeastract</sub> = 5.9, while that found from our model is <em>k</em><sub>Model</sub> = 6.9 with the standard deviation <em>σ<sub>k</sub></em> = 0.7. In <a href="#pone-0000501-g003">Fig. 3b</a>, we exhibit the in-degree distribution obtained from the Yeastract <a href="#pone.0000501-Teixeira1">[5]</a> data, and the corresponding scatter plot.</p>
<a id="article1.body1.sec3.sec1.p11" name="article1.body1.sec3.sec1.p11"></a><p>The out-degree distribution of the yeast and model networks has a rather large scatter of points due to the relatively small number of TFs. Comparing with the scatter plot obtained from one hundred realizations, we find again that the actual yeast data falls within the boundaries set by the model ensemble (<a href="#pone-0000501-g003">Fig. 3c</a>).</p>
<a id="article1.body1.sec3.sec1.p12" name="article1.body1.sec3.sec1.p12"></a><p>In <a href="#pone-0000501-g003">Fig. 3(d,e,f)</a>, we report the three topological coefficients, namely, the clustering coefficient, the degree-degree correlation and the “rich-club” coefficient, that go beyond degree-distributions in characterizing the network. The agreement is extremely good; in particular, the shoulder observed in the “rich-club” coefficient in <a href="#pone-0000501-g003">Fig. 3f, a</a> feature common to both gene regulation and protein-protein interaction networks <a href="#pone.0000501-Colizza2">[26]</a>, is captured accurately in our model. The average clustering coefficient for yeast is <em>C</em><sub>Yeastract</sub> = 0.08, while <em>C</em><sub>Model</sub> = 0.06 with the standard deviation <em>σ<sub>c</sub></em> = 0.01.</p>
<a id="article1.body1.sec3.sec1.p13" name="article1.body1.sec3.sec1.p13"></a><p>We have compared the topological properties of our model networks with those of the TRN of yeast obtained from the different sources listed in <a href="#pone-0000501-t001">Table 1</a>. The number of interacting nodes in these data sets vary between 2884 and 4252, while the number of interacting pairs vary between 6441 and 12530. All the data sets give rise to statistically similar topological features as can be seen by superposing the data points from all different sources on the scatter plots obtained from our model (see <a href="#pone-0000501-g001">Fig. 1</a> of <a href="#pone.0000501.s004">Text S4</a>). The topological features computed from all the different data sets are reproduced faithfully with a single <em>μ</em> optimized with respect to Yeastract only.</p>


<h4>Comparison with randomized results and null-null models</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In this section we briefly discuss comparing the topological features of our model with randomized versions thereof, and with pared-down models which incorporate only a few elements of our model, selected to mimic only certain phenomenological properties of the target network, but not incorporating either the full biological information in the form of realistic length distributions, or the full sequence matching rule. These pared-down versions of our model are designed to act as null-hypotheses with respect to the null-model we have constructed, and therefore we will call them “null-null” models. The details of the computations and figures are presented in <a href="#pone.0000501.s005">Text S5</a>.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>To check the significance of our results, we have subjected both the real and the model networks to a rewiring of the edges while <em>i)</em> keeping the degree distributions fixed while ignoring the directionality and <em>ii)</em> conserving the directionality so that the in- and out- degree distributions are kept fixed separately.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Randomly exchanging the endpoints of pairs of edges while keeping only the total degree of each node fixed, destroys the hierarchical structure, and the topological features computed here are radically altered. Rewiring the model network and yeast TRN while keeping the in- and out-degree of each node separately invariant by conserving the directionality of the edges, leaves all of the topological features displayed in <a href="#pone-0000501-g002">Figs. 2</a> and <a href="#pone-0000501-g003">3</a> practically unchanged.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>It should be noted that, in our model the lengths (and the contents) of the binding sequence with which we label the TF and the PR associated with the same node are assigned independently of each other. Thus there is no correlation between the in- and out-degrees of a given node. Our model is, therefore, a null model in this respect. Invariance of the topological features under a random rewiring which conserves the in- and out- degree distributions, suggests that the in- and out-degree distributions together are able to determine <em>all</em> of the global topological features of the network in question. The achievement of our model is that it does not have to import these degree distributions from empirical data; it is able to capture them by means of the string-matching rule and the length distributions of the TF and PR sequences appropriate to the organism under study.</p>
<a id="article1.body1.sec3.sec2.p5" name="article1.body1.sec3.sec2.p5"></a><p>To double check our randomization procedure, we have simulated an ensemble of configuration model networks <a href="#pone.0000501-Molloy1">[44]</a>, whose in- and out-degree sequences are extracted from different realizations of our content-based model. We connect the nodes randomly, while respecting the in- and out-degree assignments (see <a href="#pone.0000501.s005">Text S5</a> for details). The topological features of the resulting networks are indistinguishable from the model ensemble.</p>
<a id="article1.body1.sec3.sec2.p6" name="article1.body1.sec3.sec2.p6"></a><p>To see whether we could reproduce certain features of the yeast TRN using only the fact that there are two types of nodes in this network, those coding for TFs, and others that do not, we constructed an Erdös-Rényi type of null-null model, where out-edges connect the TF-coding nodes to randomly picked nodes in the network with a probability <em>p</em> given by the density of edges on the yeast TRN. The resulting network is a superposition of two Erdös-Rényi random graphs. The results are quantitatively different from the yeast TRN in all respects, as shown in <a href="#pone.0000501.s005">Text S5</a>; however, qualitatively, they are somewhat reminiscent of the yeast network.</p>
<a id="article1.body1.sec3.sec2.p7" name="article1.body1.sec3.sec2.p7"></a><p>A coarse-grained, or mean-field, version of our model is obtained if, instead of the fluctuations coming from the chance coincidence of individual strings, one takes the ensemble averaged probability for a matching to occur between strings of given lengths, as in Eq. (1). This can be thought of as a hidden-variable model <a href="#pone.0000501-Caldarelli1">[45]</a>, where, instead of just the two types of nodes considered above, one has a superposition of a whole spectrum of Erdös-Rényi networks, with the connection probabilities <em>p</em>(<em>l<sub>i</sub>,k<sub>j</sub></em>), between pairs of nodes <em>i</em> and <em>j</em>, with binding motifs and PRs of length <em>l<sub>i</sub></em> and <em>k<sub>j</sub></em>. Simulation results of this null-model are presented in <a href="#pone.0000501.s005">Text S5</a>.</p>
<a id="article1.body1.sec3.sec2.p8" name="article1.body1.sec3.sec2.p8"></a><p>The fidelity of the mean-field version to the yeast TRN is indistinguishable from that of the full content-based model. This gives us confidence that analytical calculations of ensemble averaged properties are quite meaningful. It should be remembered that <em>i)</em> the length distributions of the PR or binding strings have been extracted from empirical data using our information-theoretical approach to the binding specificities of the binding motifs, and <em>ii)</em> that the connection probabilities <em>p</em>(<em>l,k</em>) were derived from the string-matching condition.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Our results support our hypothesis that the topology of the TRN is predominantly determined by the interactions between the TFs and the response elements. In our model these interactions are schematized via the sequence-matching rule for the sharing of information between the cognates/cognate response elements. The close structural similarity between the model and the real yeast transcriptional regulatory network, with respect to a diverse set of criteria shows that they are part of the same statistical ensemble of networks <a href="#pone.0000501-Kauffman2">[46]</a>. This observation is further supported by a comparison of the frequency of various triangular network motifs (not to be confused with the “binding motifs” in the text), as provided in <a href="#pone.0000501.s001">Text S1</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>It should be noted that the present approach could also be adapted to model the topology of the gene regulatory network of <em>E. coli</em>, by taking into account the fact that in the prokaryotic genome the genes are organized into operons, each operon being regulated by a single promoter region. Work on this problem is in progress.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The sequence-matching rule should be viewed as an information-theoretical constraint, where the interaction between two genes requires the fulfillment of a set of conditions which we symbolically represent as the matching of two random sequences. The more stringent the prerequisites of the interaction, the longer is the random binding sequence that is to be matched. The length of the PR establishes the size of the phase space in which the motif is to be sought. The properties of the network are then determined by the distributions obeyed by the lengths of the binding motifs as well as the promoter regions.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Interpreted within an information-theoretical framework, our model has sufficient generality to accommodate other interactions based on constraint satisfaction mechanisms, such as protein networks, where the interactions are dictated by certain steric and chemical conditions.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>The topological features of the networks investigated here, and shown to be shared by the yeast transcriptional regulatory network, strongly point to the possibility that these networks did not have to be assembled from scratch, but rather emerged spontaneously, given any sufficiently long, complex linear code, and a mechanism for the transcription of some of its subsequences into molecules (proteins) that in their turn have an affinity for parts of this code and bind it. That the length distribution of these sequences (and not their contents) is sufficient to reproduce the topological features investigated here, indicates a certain level of robustness to point mutations.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Our model provides a means to roughly predict the parameters of the length distribution of the regulatory sequences in other organisms (assuming, e.g., a Gaussian form), given their regulatory network topology. Even more significantly, it may help identify the biologically relevant topological features that may have been acquired in the course of evolution, which require more specific information than the length distributions to be pinpointed, and hence are not captured by our model.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Methods"></a><h3>Methods</h3>
<h4>Bit-wise information content</h4>
<a id="article1.body1.sec5.sec1.p1" name="article1.body1.sec5.sec1.p1"></a><p>In this model we have assumed that the information content of a sequence can be computed as the sum of the information content of each letter in the sequence, i.e., that the letters are not correlated, although their relative frequencies of occurrence may depend on their position. We have moreover, assumed that positions within the sequence have equal significance, i.e., the maximum amount of information which can be contained in any position within the sequence is uniform.</p>
<a id="article1.body1.sec5.sec1.p2" name="article1.body1.sec5.sec1.p2"></a><p>In a given sequence of length <em>L</em>, with letters chosen from an alphabet of length <em>r</em>, the information content, which is the negative of the Shannon entropy <a href="#pone.0000501-Shannon1">[43]</a>, is given by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e008&amp;representation=PNG" class="inline-graphic"></span> with<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <em>f<sub>ij</sub></em>, <em>i</em> = 1,…,<em>r</em> are the relative frequencies of the different letters at each position <em>j</em> in the sequence. Note that <em>I<sub>j</sub></em> = 0 if we know for sure that a certain (e.g., <em>i</em>th) letter and no other, will appear (in which case the relative frequency <em>f<sub>ij</sub></em> = 1 and <em>f<sub>i</sub></em><sub>′<em>j</em></sub> = 0, for <em>i</em>′≠<em>i</em>). Thus, Shannon information is the amount of information which we receive from a signal over and above what we already knew about the system. Let us define a relative Shannon information, <em>R</em> = Σ<em><sub>j</sub>R<sub>j</sub></em>≡Σ<em><sub>j</sub>I<sub>j</sub></em>+<em>L</em> ln <em>r</em>, which is the difference between <em>I</em> and the Shannon information communicated by a signal composed from an alphabet with equi-probable letters. This is the definition of information content which we will use. (Note that the so called log-odds matrix, where one takes ln(<em>f<sub>ij</sub></em>.<em>p<sub>ij</sub></em>), where <em>p<sub>ij</sub></em> is the so called “background” probability, here taken to be uniformly equal to 1/<em>r</em>, is related but not quite the same, since the natural logarithms are not multiplied in this case by the respective frequencies in computing the information content. The Kullback-Leibler (information) divergence <a href="#pone.0000501-Kullback1">[47]</a> is another closely related measure, which, for a uniformly random background distribution, is identical to the relative information defined here.)</p>
<a id="article1.body1.sec5.sec1.p3" name="article1.body1.sec5.sec1.p3"></a><p>For a four-letter alphabet, the length increment which the <em>j</em>th member of the sequence will contribute, is, <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br></p>
<a id="article1.body1.sec5.sec1.p4" name="article1.body1.sec5.sec1.p4"></a><p>The bit-wise information content of a sequence is the number of binary digits 0,1, needed to code the same amount of information. Thus the bit-wise length increment of the same character will be <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br></p>
<a id="article1.body1.sec5.sec1.p5" name="article1.body1.sec5.sec1.p5"></a><p>However, <em>δl<sub>j</sub></em> is not, in general, an integer (neither is <em>δL<sub>j</sub></em>). Therefore a coarse graining, which entails a certain amount of arbitrariness, is called for. In the context of transcriptional gene regulation, the binding sequences are reported in Harbison et al. <a href="#pone.0000501-Harbison1">[9]</a> in an enriched alphabet, with upper case letters indicating a high preference, lowercase letters a weaker preference, with the letters {ATCG} for the nucleic acids, the “ambiguity codes” {SWRYKM} for pairs of letters out of the four, i.e., S = C or G, W = A or T, R = A or G, Y = C or T, K = G or T, M = A or C, the codes {HBVD} for different triplets out of the four letters, and finally, N indicating “no preference.” Plotting <em>δl<sub>j</sub></em> on the real line for all the empirically encountered binding motif elements, we find that these values fall into distinct clusters over the interval [0,2], grouped according to their codes within this enriched alphabet. Thus, for example the interval (1.04,2] corresponds to {ATGC}, (0.3,1.04] to {actgswrykm}, and [0,0.3] to the letter {n}. We accordingly make the following choice for a coarse grained, integer valued Δ<em>l<sub>j</sub></em>, <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>with the bit-wise length of a sequence being finally given by Σ<em><sub>j</sub></em>Δ<em>l<sub>j</sub></em>.</p>
<a id="article1.body1.sec5.sec1.p6" name="article1.body1.sec5.sec1.p6"></a><p>Note that, in computing the relative information content, instead of assuming equal probabilities for the different letters of the alphabet, we could have taken the approximate proportions of 0.3, 0.3, 0.2 and 0.2 of the letters {ATCG}. Then, the Shannon information for a random sequence composed of these letters with the aforesaid probabilities would have been −1.366 rather than −1.386 = −ln4. This introduces a uniform shift of the relative information content by a small increment, 0.02. The length increment per character thus gets uniformly shifted by 0.02/ln2 = 0.029. Since the shift is uniform, taking the same criteria for the coarse graining procedure yields an identical spectrum of bit-wise length increments as the one obtained with equi-probable random letters for the nucleic acids.</p>


<h4>Topological quantifiers of complex networks</h4>
<a id="article1.body1.sec5.sec2.p1" name="article1.body1.sec5.sec2.p1"></a><p>The degree <em>k</em> of a node is the number of nodes connected to it. When the graph is directed, one distinguishes in-, out-, and total degrees of a node, with their corresponding distributions. In the measures below we have ignored the directionality of the network.</p>
<a id="article1.body1.sec5.sec2.p2" name="article1.body1.sec5.sec2.p2"></a><p>The clustering coefficient is given <a href="#pone.0000501-Bollobas1">[19]</a>–<a href="#pone.0000501-Dorogovstsev1">[22]</a> by the formula: <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br>where Δ<em><sub>i</sub></em> is the number of triangles that contain node <em>i</em>. The quantity <em>C</em>(<em>k</em>) plotted in <a href="#pone-0000501-g003">Fig. 3d</a> is the average of <em>C<sub>i</sub></em> over the nodes with degree <em>k</em>.</p>
<a id="article1.body1.sec5.sec2.p3" name="article1.body1.sec5.sec2.p3"></a><p>The degree-degree correlation function <em>k</em><sub>nm</sub>(<em>k</em>) is <a href="#pone.0000501-Colizza1">[24]</a>, <a href="#pone.0000501-PastorSatorras1">[23]</a><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>where <em>p</em>(<em>k</em>′|<em>k</em>) is the conditional probability that a node with degree <em>k</em> is connected to a node with degree <em>k</em>′.</p>
<a id="article1.body1.sec5.sec2.p4" name="article1.body1.sec5.sec2.p4"></a><p>The “rich-club” coefficient <a href="#pone.0000501-Zhou1">[25]</a>, <a href="#pone.0000501-Colizza2">[26]</a><em>r</em>(<em>k</em>) is the total number e<em><sub>&gt;K</sub></em> of edges connecting nodes with degree greater than <em>k</em>, normalized by the maximum possible number of such connections, <a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000501.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br>where <em>N<sub>&gt;K</sub></em> is the total number of nodes with degree greater than <em>k</em>.</p>
<a id="article1.body1.sec5.sec2.p5" name="article1.body1.sec5.sec2.p5"></a><p>The hierarchical organization of a network is revealed by the <em>k</em>-core decomposition, which performs a successive pruning on the least connected vertices of a network <a href="#pone.0000501-Bollobas1">[19]</a>. At each step one removes all nodes with a degree less than <em>k</em> along with their edges and continues in this manner until all nodes have at least degree <em>k</em>. The remaining nodes constitute the <em>k</em>-core. Next, <em>k</em> is incremented by one, and the process is repeated until no nodes are left. The <em>k</em>-shell is defined as the set of nodes that belong to the <em>k</em>-core, but not the (<em>k</em>+1)-core. The <em>k</em>-core decomposition provides a highly detailed topological characterization of the network, if, besides the total number of shells, the distribution of the nodes over the shells and inter- and intra-shell connectivity <a href="#pone.0000501-Carmi1">[48]</a> are also taken into account. (see <a href="#pone.0000501.s002">Text S2</a>)</p>

</div>

<div id="section6" class="section"><a id="s6" name="s6" toc="s6" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000501"></div><a name="pone.0000501.s001" id="pone.0000501.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000501.s001">Text S1. </a></strong></p><a id="article1.body1.sec6.supplementary-material1.caption1.p1" name="article1.body1.sec6.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Comparison of 3-Motifs for the Model and the various Yeast Data Sources</p>
<p class="siDoi">doi:10.1371/journal.pone.0000501.s001</p><a id="article1.body1.sec6.supplementary-material1.caption1.p2" name="article1.body1.sec6.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.05 MB PDF)</p>
<a name="pone.0000501.s002" id="pone.0000501.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000501.s002">Text S2. </a></strong></p><a id="article1.body1.sec6.supplementary-material2.caption1.p1" name="article1.body1.sec6.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Qualitative and Quantitative Aspects of the k-core Structure: Choosing the Length Distribution of the Promoter Regions</p>
<p class="siDoi">doi:10.1371/journal.pone.0000501.s002</p><a id="article1.body1.sec6.supplementary-material2.caption1.p2" name="article1.body1.sec6.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.40 MB PDF)</p>
<a name="pone.0000501.s003" id="pone.0000501.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000501.s003">Text S3. </a></strong></p><a id="article1.body1.sec6.supplementary-material3.caption1.p1" name="article1.body1.sec6.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Ranking of Overlapping Sets of Regulated Genes and Sequence Inclusion</p>
<p class="siDoi">doi:10.1371/journal.pone.0000501.s003</p><a id="article1.body1.sec6.supplementary-material3.caption1.p2" name="article1.body1.sec6.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0000501.s004" id="pone.0000501.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000501.s004">Text S4. </a></strong></p><a id="article1.body1.sec6.supplementary-material4.caption1.p1" name="article1.body1.sec6.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Comparison of the Model with the Yeast Data from Different Databases</p>
<p class="siDoi">doi:10.1371/journal.pone.0000501.s004</p><a id="article1.body1.sec6.supplementary-material4.caption1.p2" name="article1.body1.sec6.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB PDF)</p>
<a name="pone.0000501.s005" id="pone.0000501.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000501.s005">Text S5. </a></strong></p><a id="article1.body1.sec6.supplementary-material5.caption1.p1" name="article1.body1.sec6.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Randomization Procedures and Null-Null-Models</p>
<p class="siDoi">doi:10.1371/journal.pone.0000501.s005</p><a id="article1.body1.sec6.supplementary-material5.caption1.p2" name="article1.body1.sec6.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(2.19 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We would like to thank Betül Kırdar and Beste Kınıkoğlu for the use of their data and useful discussions. It is a pleasure to thank Alessandro Vespignani and Ignacio Alvarez-Hamelin for bringing <em>k</em>-core decomposition to our attention, and for the use of their web-based <em>k</em>-core visualization tool. AE would like to thank Tamás Vicsek and András Czirók for a useful discussion, and partial support from the Turkish Academy of Sciences.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>All of the authors contributed equally to all aspects of this paper.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000501-Lockhart1" id="pone.0000501-Lockhart1"></a>Lockhart DJ, Winzeler EA (2000) Genomics, gene expression and DNA arrays. Nature  405: 827–836.  <ul class="find" data-citedArticleID="956402" data-doi="10.1038/35015701"><li><a href="http://dx.doi.org/10.1038/35015701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomics%2C+gene+expression+and+DNA+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomics%2C+gene+expression+and+DNA+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000501-Alberts1" id="pone.0000501-Alberts1"></a>Alberts B, Johnson A, Lewis J, Raff M, Roberts K, et al.  (2002) Molecular Biology of the Cell. NY: Garland Science.   Chapter 9.  <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0000501-Lee1" id="pone.0000501-Lee1"></a>Lee TI, Rinaldi NJ, Robert F, Odom DT, Bar-Joseph Z, et al.  (2002) Transcriptional regulatory networks in <em>Saccharomyces cerevisiae</em>. Science  298: 799–804.  <ul class="find" data-citedArticleID="956400" data-doi="10.1126/science.1075090"><li><a href="http://dx.doi.org/10.1126/science.1075090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+networks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000501-Luscombe1" id="pone.0000501-Luscombe1"></a>Luscombe NM, Babu MM, Yu H, Snyder M, Teichmann SA, et al.  (2004) Genomic analyis of regulatory network dynamics reveals large topological changes. Nature  431: 308–312.  <ul class="find" data-citedArticleID="956404" data-doi="10.1038/nature02782"><li><a href="http://dx.doi.org/10.1038/nature02782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analyis+of+regulatory+network+dynamics+reveals+large+topological+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analyis+of+regulatory+network+dynamics+reveals+large+topological+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000501-Teixeira1" id="pone.0000501-Teixeira1"></a>Teixeira MC, Monteiro P, Jain P, Tenreiro S, Fernandes AR, et al.  (2006) The YEASTRACT database: a tool for the analysis of transcription regulatory associations in <em>Saccharomyces cerevisiae</em>. Nucl Acids Res  34: D446–451.  <ul class="find" data-citedArticleID="956424" data-doi="10.1093/nar/gkj013"><li><a href="http://dx.doi.org/10.1093/nar/gkj013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+YEASTRACT+database%3A+a+tool+for+the+analysis+of+transcription+regulatory+associations+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+YEASTRACT+database%3A+a+tool+for+the+analysis+of+transcription+regulatory+associations+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000501-Knkolu1" id="pone.0000501-Knkolu1"></a>Kınıkoğlu B, et al.  (2006) Identifying perturbation responsive key transcription factors in <em>Saccharomyces cerevisiae</em>. Submitted to Yeast.   <ul class="find" data-citedArticleID="956396"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identifying+perturbation+responsive+key+transcription+factors+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Identifying+perturbation+responsive+key+transcription+factors+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+perturbation+responsive+key+transcription+factors+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+perturbation+responsive+key+transcription+factors+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000501-Guelzim1" id="pone.0000501-Guelzim1"></a>Guelzim N, Bottani S, Bourgine P, Kepes F (2002) Topological and causal structure of the yeast transcriptional regulatory network. Nature Genetics  31: 60–63.  <ul class="find" data-citedArticleID="956378" data-doi="10.1038/ng873"><li><a href="http://dx.doi.org/10.1038/ng873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Topological+and+causal+structure+of+the+yeast+transcriptional+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Topological+and+causal+structure+of+the+yeast+transcriptional+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000501-Tong1" id="pone.0000501-Tong1"></a>Tong AHY, Lesage G, Bader GD, Ding H, Xu H, et al.  (2004) Global mapping of the yeast genetic interaction network. Science  303: 808–813.  <ul class="find" data-citedArticleID="956426" data-doi="10.1126/science.1091317"><li><a href="http://dx.doi.org/10.1126/science.1091317" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+mapping+of+the+yeast+genetic+interaction+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+mapping+of+the+yeast+genetic+interaction+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000501-Harbison1" id="pone.0000501-Harbison1"></a>Harbison CT, Gordon DB, Lee TI, Rinaldi NJ, Macisaac KD, et al.  (2004) Transcriptional regulatory code of a eukaryotic genome. Nature  431: 99–104.  <ul class="find" data-citedArticleID="956380" data-doi="10.1038/nature02800"><li><a href="http://dx.doi.org/10.1038/nature02800" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+regulatory+code+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+regulatory+code+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000501-Kauffman1" id="pone.0000501-Kauffman1"></a>Kauffman SA (1969) Metabolic stability and epigenesis in randomly constructed genetic nets. J Theor Biol  22: 437–467.  <ul class="find" data-citedArticleID="956388" data-doi="10.1016/0022-5193(69)90015-0"><li><a href="http://dx.doi.org/10.1016/0022-5193(69)90015-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+stability+and+epigenesis+in+randomly+constructed+genetic+nets." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+stability+and+epigenesis+in+randomly+constructed+genetic+nets.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000501-Barabasi1" id="pone.0000501-Barabasi1"></a>Barabasi AL, Albert R (1999) Emergence of scaling in random networks. Science  286: 509–512.  <ul class="find" data-citedArticleID="956356" data-doi="10.1126/science.286.5439.509"><li><a href="http://dx.doi.org/10.1126/science.286.5439.509" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Emergence+of+scaling+in+random+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Emergence+of+scaling+in+random+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000501-Barabasi2" id="pone.0000501-Barabasi2"></a>Barabasi AL, Oltvai ZN (2004) Network biology: understanding the cell’s functional organization. Nature Reviews-Genetics  5: 101–113.  <ul class="find" data-citedArticleID="956358" data-doi="10.1038/nrg1272"><li><a href="http://dx.doi.org/10.1038/nrg1272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+biology%3A+understanding+the+cell%26%23x2019%3Bs+functional+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+biology%3A+understanding+the+cell%26%23x2019%3Bs+functional+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000501-Strogatz1" id="pone.0000501-Strogatz1"></a>Strogatz SH (2001) Exploring complex networks. Nature  410: 268–276.  <ul class="find" data-citedArticleID="956422" data-doi="10.1038/35065725"><li><a href="http://dx.doi.org/10.1038/35065725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000501-Ihmels1" id="pone.0000501-Ihmels1"></a>Ihmels J, Levy R, Barkai N (2004) Principles of transcription control in the metabolic network of <em>S. cerevisiae</em>. Nature Biotechnology  22: 86–92.  <ul class="find" data-citedArticleID="956384" data-doi="10.1038/nbt918"><li><a href="http://dx.doi.org/10.1038/nbt918" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Principles+of+transcription+control+in+the+metabolic+network+of+S.+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Principles+of+transcription+control+in+the+metabolic+network+of+S.+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000501-Huang1" id="pone.0000501-Huang1"></a>Huang S (2004) Back to the biology in systems biology: What can we learn from biomolecular networks? Brief Funct Gen Prot  2: 279–297.  <ul class="find" data-citedArticleID="956382" data-doi="10.1093/bfgp/2.4.279"><li><a href="http://dx.doi.org/10.1093/bfgp/2.4.279" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Back+to+the+biology+in+systems+biology%3A+What+can+we+learn+from+biomolecular+networks%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Back+to+the+biology+in+systems+biology%3A+What+can+we+learn+from+biomolecular+networks%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000501-Balcan1" id="pone.0000501-Balcan1"></a>Balcan D, Erzan A (2004) Random model for RNA interference yields scale free network. Eur Phys J B  38: 253–260.  <ul class="find" data-citedArticleID="956352" data-doi="10.1140/epjb/e2004-00055-7"><li><a href="http://dx.doi.org/10.1140/epjb/e2004-00055-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Random+model+for+RNA+interference+yields+scale+free+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Random+model+for+RNA+interference+yields+scale+free+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000501-Mungan1" id="pone.0000501-Mungan1"></a>Mungan M, Kabakçıoğlu A, Balcan D, Erzan A (2005) Analytical solution of a stochastic content-based network model. J Phys A 38  (44): 9599–9620.  <ul class="find" data-citedArticleID="956410" data-doi="10.1088/0305-4470/38/44/001"><li><a href="http://dx.doi.org/10.1088/0305-4470/38/44/001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analytical+solution+of+a+stochastic+content-based+network+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analytical+solution+of+a+stochastic+content-based+network+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000501-Almirantis1" id="pone.0000501-Almirantis1"></a>Almirantis Y, Provata A (1999) Scaling properties of coding and non-coding DNA sequences. J Stat Phys  97: 233–262.  <ul class="find" data-citedArticleID="956350" data-doi="10.1016/s0378-4371(97)00424-x"><li><a href="http://dx.doi.org/10.1016/s0378-4371(97)00424-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scaling+properties+of+coding+and+non-coding+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scaling+properties+of+coding+and+non-coding+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000501-Bollobas1" id="pone.0000501-Bollobas1"></a>Bollobas B (1998) Modern Graph Theory. New York: Springer Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0000501-Albert1" id="pone.0000501-Albert1"></a>Albert R, Barabasi , AL  (2001) Statistical mechanics of complex networks. Rev Mod Phys  74: 47–97.  <ul class="find" data-citedArticleID="956346" data-doi="10.1103/revmodphys.74.47"><li><a href="http://dx.doi.org/10.1103/revmodphys.74.47" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+mechanics+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+mechanics+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000501-Watts1" id="pone.0000501-Watts1"></a>Watts DJ, Strogatz SH (1998) Collective dynamics of ‘small world’ networks. Nature  393: 440–442.  <ul class="find" data-citedArticleID="956436" data-doi="10.1038/30918"><li><a href="http://dx.doi.org/10.1038/30918" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collective+dynamics+of+%26%23x2018%3Bsmall+world%26%23x2019%3B+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collective+dynamics+of+%26%23x2018%3Bsmall+world%26%23x2019%3B+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000501-Dorogovstsev1" id="pone.0000501-Dorogovstsev1"></a>Dorogovstsev SN, Mendes JFF (2002) Evolution of networks. Adv Phys  51: 1079–1187.  <ul class="find" data-citedArticleID="956374"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolution+of+networks.&amp;auth=&amp;atitle=Evolution+of+networks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000501-PastorSatorras1" id="pone.0000501-PastorSatorras1"></a>Pastor-Satorras R, Vazquez A, Vespignani A (2001) Dynamical and correlation properties of the internet. Phys Rev Lett  87: 258701.  <ul class="find" data-citedArticleID="956412" data-doi="10.1103/physrevlett.87.258701"><li><a href="http://dx.doi.org/10.1103/physrevlett.87.258701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamical+and+correlation+properties+of+the+internet." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamical+and+correlation+properties+of+the+internet.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000501-Colizza1" id="pone.0000501-Colizza1"></a>Colizza V, Flammini A, Maritan A, Vespignani A (2005) Characterization and modeling of protein-protein interaction networks. Physica A  352: 1–27.  <ul class="find" data-citedArticleID="956368" data-doi="10.1016/j.physa.2004.12.030"><li><a href="http://dx.doi.org/10.1016/j.physa.2004.12.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+and+modeling+of+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+and+modeling+of+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000501-Zhou1" id="pone.0000501-Zhou1"></a>Zhou S, Mondragon RJ (2004) The rich-club phenomenon in the internet topology. IEEE Commun Lett  8: 180–182.  <ul class="find" data-citedArticleID="956438" data-doi="10.1109/lcomm.2004.823426"><li><a href="http://dx.doi.org/10.1109/lcomm.2004.823426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rich-club+phenomenon+in+the+internet+topology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rich-club+phenomenon+in+the+internet+topology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000501-Colizza2" id="pone.0000501-Colizza2"></a>Colizza V, Flammini A, Serrano MA, Vespignani A (2006) Detecting rich-club ordering in complex networks. Nature Physics  2: 110–115.  <ul class="find" data-citedArticleID="956370" data-doi="10.1038/nphys209"><li><a href="http://dx.doi.org/10.1038/nphys209" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+rich-club+ordering+in+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+rich-club+ordering+in+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000501-Dobrin1" id="pone.0000501-Dobrin1"></a>Dobrin R, Beg QK, Barabasi AL, Oltvai ZN (2004) Aggregation of topological motifs in the <em>Escherichia coli</em> transcriptional regulatory network. BMC Bioinformatics  5: 10.  <ul class="find" data-citedArticleID="956372" data-doi="10.1186/1471-2105-5-10"><li><a href="http://dx.doi.org/10.1186/1471-2105-5-10" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aggregation+of+topological+motifs+in+the+Escherichia+coli+transcriptional+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aggregation+of+topological+motifs+in+the+Escherichia+coli+transcriptional+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000501-Milo1" id="pone.0000501-Milo1"></a>Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, et al.  (2002) Network motifs: simple building blocks of complex networks. Science  298: 824–827.  <ul class="find" data-citedArticleID="956406" data-doi="10.1126/science.298.5594.824"><li><a href="http://dx.doi.org/10.1126/science.298.5594.824" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+motifs%3A+simple+building+blocks+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+motifs%3A+simple+building+blocks+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000501-Ihmels2" id="pone.0000501-Ihmels2"></a>Ihmels J, Friedlander G, Bergmann S, Sarig O, Ziv Y, et al.  (2002) Revealing modular organization in the yeast transcriptional network. Nature Genetics  31: 370–377.  <ul class="find" data-citedArticleID="956386" data-doi="10.1038/ng941"><li><a href="http://dx.doi.org/10.1038/ng941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Revealing+modular+organization+in+the+yeast+transcriptional+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Revealing+modular+organization+in+the+yeast+transcriptional+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000501-Kellis1" id="pone.0000501-Kellis1"></a>Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES (2003) Sequenceing and comparison of yeast species to identify genes and regulatory elements. Nature  423: 241–254.  <ul class="find" data-citedArticleID="956392" data-doi="10.1038/nature01644"><li><a href="http://dx.doi.org/10.1038/nature01644" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequenceing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequenceing+and+comparison+of+yeast+species+to+identify+genes+and+regulatory+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000501-Vazques1" id="pone.0000501-Vazques1"></a>Vazques A, Dobrin R, Sergi D, Eckmann J-P, Oltvai ZN, et al.  (2004) The topological relationship between the large-scale attributes and local interaction patterns of complex networks. Proc Natl Acad Sci USA  101: 17940–17945.  <ul class="find" data-citedArticleID="956428" data-doi="10.1073/pnas.0406024101"><li><a href="http://dx.doi.org/10.1073/pnas.0406024101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+topological+relationship+between+the+large-scale+attributes+and+local+interaction+patterns+of+complex+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+topological+relationship+between+the+large-scale+attributes+and+local+interaction+patterns+of+complex+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000501-Bergmann1" id="pone.0000501-Bergmann1"></a>Bergmann S, Ihmels J, Barkai N (2004) Similarities and differences in genome-wide expression data of six organisms. PLoS Biol  2: 85–93.  <ul class="find" data-citedArticleID="956360" data-doi="10.1371/journal.pbio.0020009"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Similarities+and+differences+in+genome-wide+expression+data+of+six+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Similarities+and+differences+in+genome-wide+expression+data+of+six+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000501-Perelson1" id="pone.0000501-Perelson1"></a>Perelson AS, Weisbuch G (1997) Immunology for physicists. Rev Mod Phys  69: 1219–1267.  <ul class="find" data-citedArticleID="956414" data-doi="10.1103/revmodphys.69.1219"><li><a href="http://dx.doi.org/10.1103/revmodphys.69.1219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunology+for+physicists." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunology+for+physicists.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000501-Reil1" id="pone.0000501-Reil1"></a>Reil T (1999) Dynamics of Gene Expression in an Artificial Genome. Proceedings of the 5th European Conference on Advances in Artificial Life. Lecture Notes In Computer Science 1674. London: Springer-Verlag. pp. 457–466.  <ul class="find-nolinks"></ul></li><li><span class="label">35.
              </span><a name="pone.0000501-Geard1" id="pone.0000501-Geard1"></a>Geard N, Wiles J (2003) Structure and dynamics of a gene network model incorporating small RNAs. Proceedings of the 2003 Congress on Evolutionary Computation (CEC).   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0000501-Watson1" id="pone.0000501-Watson1"></a>Watson J, Geard N, Wiles J (2004) Towards more biological mutation operators in gene regulation studies. BioSystems  76: 239–248.  <ul class="find" data-citedArticleID="956434" data-doi="10.1016/j.biosystems.2004.05.016"><li><a href="http://dx.doi.org/10.1016/j.biosystems.2004.05.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+more+biological+mutation+operators+in+gene+regulation+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+more+biological+mutation+operators+in+gene+regulation+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000501-vanNoort1" id="pone.0000501-vanNoort1"></a>van Noort V, Snel B, Huynen MA (2004) The yeast coexpression network has a small-world, scale-free architecture and can be explained by a simple model. EMBO Rep  5: 280–284.  <ul class="find" data-citedArticleID="956440" data-doi="10.1038/sj.embor.7400090"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400090" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+yeast+coexpression+network+has+a+small-world%2C+scale-free+architecture+and+can+be+explained+by+a+simple+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+yeast+coexpression+network+has+a+small-world%2C+scale-free+architecture+and+can+be+explained+by+a+simple+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000501-Banzhaf1" id="pone.0000501-Banzhaf1"></a>Banzhaf W, Kuo PD (2004) Network motifs in natural and artificial transcriptional regulatory networks. J Biol Phys Chem  4: 85–92.  <ul class="find" data-citedArticleID="956354" data-doi="10.4024/2040405.jbpc.04.02"><li><a href="http://dx.doi.org/10.4024/2040405.jbpc.04.02" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+motifs+in+natural+and+artificial+transcriptional+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+motifs+in+natural+and+artificial+transcriptional+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000501-Wagner1" id="pone.0000501-Wagner1"></a>Wagner A (1994) Evolution of gene networks by gene duplications: a mathematical model and its implications on genome organization. Proc Natl Acad Sci USA  91: 4387–4391.  <ul class="find" data-citedArticleID="956430" data-doi="10.1073/pnas.91.10.4387"><li><a href="http://dx.doi.org/10.1073/pnas.91.10.4387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+gene+networks+by+gene+duplications%3A+a+mathematical+model+and+its+implications+on+genome+organization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+gene+networks+by+gene+duplications%3A+a+mathematical+model+and+its+implications+on+genome+organization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000501-Wagner2" id="pone.0000501-Wagner2"></a>Wagner A (2001) The yeast protein interaction network evolves rapidly and contains few redundant duplicate genes. Mol Biol Evol  18(7): 1283–1292.  <ul class="find" data-citedArticleID="956432" data-doi="10.1093/oxfordjournals.molbev.a003913"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003913" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+yeast+protein+interaction+network+evolves+rapidly+and+contains+few+redundant+duplicate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+yeast+protein+interaction+network+evolves+rapidly+and+contains+few+redundant+duplicate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000501-Sole1" id="pone.0000501-Sole1"></a>Sole RV, Pastor-Satorras R (2002)  Complex Networks in Genomics and Proteomics. In: Bornholdt S, Schuster HG, editors. Handbook of Graphs and Networks. Berlin: Wiley-VCH Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0000501-Kleffe1" id="pone.0000501-Kleffe1"></a>Kleffe J, Borodovsky M (1992) 1st and 2nd moment of counts of words in random texts generated by Markov-chains. Comp Appl Biosci  8: 433–441.  <ul class="find" data-citedArticleID="956394" data-doi="10.1093/bioinformatics/8.5.433"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.5.433" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=1st+and+2nd+moment+of+counts+of+words+in+random+texts+generated+by+Markov-chains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%221st+and+2nd+moment+of+counts+of+words+in+random+texts+generated+by+Markov-chains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000501-Shannon1" id="pone.0000501-Shannon1"></a>Shannon CE (1949) Communication in the Presence of Noise. Proc IRE  37: 10–21.  <ul class="find" data-citedArticleID="956418" data-doi="10.1109/jrproc.1949.232969"><li><a href="http://dx.doi.org/10.1109/jrproc.1949.232969" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Communication+in+the+Presence+of+Noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Communication+in+the+Presence+of+Noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000501-Molloy1" id="pone.0000501-Molloy1"></a>Molloy M, Reed B (1995) A critical point for random graphs with a given degree sequence. Random Structures and Algorithms  6: 161–180.  <ul class="find" data-citedArticleID="956408" data-doi="10.1002/rsa.3240060204"><li><a href="http://dx.doi.org/10.1002/rsa.3240060204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+critical+point+for+random+graphs+with+a+given+degree+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+critical+point+for+random+graphs+with+a+given+degree+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000501-Caldarelli1" id="pone.0000501-Caldarelli1"></a>Caldarelli G, Capocci A, De Los Rios P, Munoz MA (2002) Scale-free networks from varying vertex intrinsic fitness. Phys Rev Lett  89: 258702.  <ul class="find" data-citedArticleID="956364" data-doi="10.1103/physrevlett.89.258702"><li><a href="http://dx.doi.org/10.1103/physrevlett.89.258702" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scale-free+networks+from+varying+vertex+intrinsic+fitness." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scale-free+networks+from+varying+vertex+intrinsic+fitness.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000501-Kauffman2" id="pone.0000501-Kauffman2"></a>Kauffman S (2004) A proposal for using the ensemble approach to understand genetic regulatory networks. J Theor Biol  230: 581–590.  <ul class="find" data-citedArticleID="956390" data-doi="10.1016/j.jtbi.2003.12.017"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2003.12.017" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+proposal+for+using+the+ensemble+approach+to+understand+genetic+regulatory+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+proposal+for+using+the+ensemble+approach+to+understand+genetic+regulatory+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000501-Kullback1" id="pone.0000501-Kullback1"></a>Kullback S, Leibler RA (1951) On information and sufficiency. Annals of Mathematical Statistics  22: 79–86.  <ul class="find" data-citedArticleID="956398" data-doi="10.1214/aoms/1177729694"><li><a href="http://dx.doi.org/10.1214/aoms/1177729694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+information+and+sufficiency." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+information+and+sufficiency.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000501-Carmi1" id="pone.0000501-Carmi1"></a>Carmi S, Havlin S, Kirkpatrick S, Shavitt Y, Shir E (2006) MEDUSA-New Model of Internet Topology Using <em>k</em>-shell Decomposition. cond-mat/0601240. arXiv preprint.  <ul class="find" data-citedArticleID="956366"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=MEDUSA-New+Model+of+Internet+Topology+Using+k-shell+Decomposition.+cond-mat%2F0601240.+arXiv+preprint.&amp;auth=&amp;atitle=MEDUSA-New+Model+of+Internet+Topology+Using+k-shell+Decomposition.+cond-mat%2F0601240.+arXiv+preprint." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEDUSA-New+Model+of+Internet+Topology+Using+k-shell+Decomposition.+cond-mat%2F0601240.+arXiv+preprint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEDUSA-New+Model+of+Internet+Topology+Using+k-shell+Decomposition.+cond-mat%2F0601240.+arXiv+preprint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.XML" value="103297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.PDF" value="341571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.t001.PNG_L" value="90010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.t001.PNG_M" value="95612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.t001.PNG_S" value="3381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.t001.TIF" value="422392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.t001.PNG_I" value="42308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e001.PNG" value="3389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e001.TIF" value="24988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e002.PNG" value="349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e002.TIF" value="20836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e003.PNG" value="882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e003.TIF" value="21596"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e004.PNG" value="2358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e004.TIF" value="24100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e005.PNG" value="817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e005.TIF" value="21500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g001.PNG_L" value="20069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g001.PNG_M" value="31201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g001.PNG_S" value="2521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g001.TIF" value="155614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g001.PNG_I" value="16380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e006.PNG" value="1706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e006.TIF" value="22976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e007.PNG" value="339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e007.TIF" value="20852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g002.PNG_L" value="3490468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g002.PNG_M" value="161130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g002.PNG_S" value="4434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g002.TIF" value="5754240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g002.PNG_I" value="49895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g003.PNG_L" value="644027"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g003.PNG_M" value="109134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g003.PNG_S" value="3799"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g003.TIF" value="826564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.g003.PNG_I" value="44754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e008.PNG" value="1474"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e008.TIF" value="22508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e009.PNG" value="2358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e009.TIF" value="23928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e010.PNG" value="1678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e010.TIF" value="22860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e011.PNG" value="2365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e011.TIF" value="23864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e012.PNG" value="6756"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e012.TIF" value="30096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e013.PNG" value="2389"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e013.TIF" value="24064"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e014.PNG" value="3086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e014.TIF" value="25108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e015.PNG" value="3308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.e015.TIF" value="25196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.s001.PDF" value="53750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.s002.PDF" value="398838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.s003.PDF" value="16733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.s004.PDF" value="38729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000501.s005.PDF" value="2194613"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000501&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000501" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000501&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000501&volume=&issue=&title=The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network&author_name=Duygu%20Balcan%2C%20Alkan%20Kabak%C3%A7%C4%B1o%C4%9Flu%2C%20Muhittin%20Mungan%2C%20Ay%C5%9Fe%20Erzan&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000501" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501&amp;t=The%20Information%20Coded%20in%20the%20Yeast%20Response%20Elements%20Accounts%20for%20Most%20of%20the%20Topological%20Properties%20of%20Its%20Transcriptional%20Regulation%20Network" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501&title=The%20Information%20Coded%20in%20the%20Yeast%20Response%20Elements%20Accounts%20for%20Most%20of%20the%20Topological%20Properties%20of%20Its%20Transcriptional%20Regulation%20Network&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501&amp;title=The%20Information%20Coded%20in%20the%20Yeast%20Response%20Elements%20Accounts%20for%20Most%20of%20the%20Topological%20Properties%20of%20Its%20Transcriptional%20Regulation%20Network" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000501&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Information Coded in the Yeast Response Elements Accounts for Most of the Topological Properties of Its Transcriptional Regulation Network';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000501';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Information%20Coded%20in%20the%20Yeast%20Response%20Elements%20Accounts%20for%20Most%20of%20the%20Topological%20Properties%20of%20Its%20Transcriptional%20Regulation%20Network http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000501" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000501" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="23840"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulatory+networks%22" title="Search for articles in the subject area:'Gene regulatory networks'"><div class="flagText">Gene regulatory networks</div></a>
              <div data-categoryid="17555" data-articleid="23840"
                   data-categoryname="Gene regulatory networks"
                   class="flagImage" title="Flag 'Gene regulatory networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Network+motifs%22" title="Search for articles in the subject area:'Network motifs'"><div class="flagText">Network motifs</div></a>
              <div data-categoryid="18085" data-articleid="23840"
                   data-categoryname="Network motifs"
                   class="flagImage" title="Flag 'Network motifs' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+interaction+networks%22" title="Search for articles in the subject area:'Protein interaction networks'"><div class="flagText">Protein interaction networks</div></a>
              <div data-categoryid="18069" data-articleid="23840"
                   data-categoryname="Protein interaction networks"
                   class="flagImage" title="Flag 'Protein interaction networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="23840"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Topology%22" title="Search for articles in the subject area:'Topology'"><div class="flagText">Topology</div></a>
              <div data-categoryid="32019" data-articleid="23840"
                   data-categoryname="Topology"
                   class="flagImage" title="Flag 'Topology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcription+factors%22" title="Search for articles in the subject area:'Transcription factors'"><div class="flagText">Transcription factors</div></a>
              <div data-categoryid="17335" data-articleid="23840"
                   data-categoryname="Transcription factors"
                   class="flagImage" title="Flag 'Transcription factors' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast+and+fungal+models%22" title="Search for articles in the subject area:'Yeast and fungal models'"><div class="flagText">Yeast and fungal models</div></a>
              <div data-categoryid="35135" data-articleid="23840"
                   data-categoryname="Yeast and fungal models"
                   class="flagImage" title="Flag 'Yeast and fungal models' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2003'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3839'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6724&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=41557">Further reaching implications on evolution of networks</a><br>Posted by suihuang</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
