

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000849"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000849" />

    <meta name="citation_title" content="Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae"/>
    <meta itemprop="name" content="Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae"/>

      <meta name="citation_author" content="Mamadou B. Coulibaly"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
            <meta name="citation_author_institution" content="Malaria Research and Training Center, University of Bamako, Bamako, Mali"/>
      <meta name="citation_author" content="Neil F. Lobo"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
      <meta name="citation_author" content="Meagan C. Fitzpatrick"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
      <meta name="citation_author" content="Marcia Kern"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
      <meta name="citation_author" content="Olga Grushko"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
      <meta name="citation_author" content="Daniel V. Thaner"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
      <meta name="citation_author" content="Sékou F. Traoré"/>
            <meta name="citation_author_institution" content="Malaria Research and Training Center, University of Bamako, Bamako, Mali"/>
      <meta name="citation_author" content="Frank H. Collins"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>
      <meta name="citation_author" content="Nora J. Besansky"/>
            <meta name="citation_author_institution" content="Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America"/>

    <meta name="citation_date" content="2007/9/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000849.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e849"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genetics and the Origin of Species.; citation_author=T Dobzhansky; citation_number=1; citation_date=1937; citation_publisher=Columbia University Press; " />
      <meta name="citation_reference" content="citation_title=Chromosome evolution in eukaryotes: a multi-kingdom perspective.; citation_author=A Coghlan; citation_author=EE Eichler; citation_author=SG Oliver; citation_author=AH Paterson; citation_author=L Stein; citation_journal_title=Trends Genet; citation_volume=21; citation_number=2; citation_pages=673-682; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chromosomal inversion polymorphisms and adaptation.; citation_author=AA Hoffmann; citation_author=CM Sgro; citation_author=AR Weeks; citation_journal_title=Trends Ecol Evol; citation_volume=19; citation_number=3; citation_pages=482-488; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Progress and Prospects in Evolutionary Biology: The Drosophila Model.; citation_author=JR Powell; citation_number=4; citation_date=1997; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Chromosomal inversions and the reproductive isolation of species.; citation_author=MA Noor; citation_author=KL Grams; citation_author=LA Bertucci; citation_author=J Reiland; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=5; citation_pages=12084-12088; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Chromosomal rearrangements and speciation.; citation_author=LH Rieseberg; citation_journal_title=Trends Ecol Evol; citation_volume=16; citation_number=6; citation_pages=351-358; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=It takes two transposons to tango: transposable-element-mediated chromosomal rearrangements.; citation_author=YH Gray; citation_journal_title=Trends Genet; citation_volume=16; citation_number=7; citation_pages=461-468; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Transposable elements as sources of genomic variation.; citation_author=MG Kidwell; citation_author=DR Lisch; citation_number=8; citation_date=2002; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Eukaryotic Transposable Elements: Teaching Old Genomes New Tricks.; citation_author=SR Wessler; citation_number=9; citation_pages=138-165; citation_date=2006; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of the breakpoints of an inversion fixed between Drosophila melanogaster and D. subobscura.; citation_author=S Cirera; citation_author=JM Martin-Campos; citation_author=C Segarra; citation_author=M Aguade; citation_journal_title=Genetics; citation_volume=139; citation_number=10; citation_pages=321-326; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Comparative genome sequencing of Drosophila pseudoobscura: chromosomal, gene, and cis-element evolution.; citation_author=S Richards; citation_author=Y Liu; citation_author=BR Bettencourt; citation_author=P Hradecky; citation_author=S Letovsky; citation_journal_title=Genome Res; citation_volume=15; citation_number=11; citation_pages=1-18; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Breakpoint structure reveals the unique origin of an interspecific chromosomal inversion (2La) in the Anopheles gambiae complex.; citation_author=IV Sharakhov; citation_author=BJ White; citation_author=MV Sharakhova; citation_author=J Kayondo; citation_author=NF Lobo; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=12; citation_pages=6258-6262; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Isolation and analysis of the breakpoint sequences of chromosome inversion In(3L)Payne in Drosophila melanogaster.; citation_author=CS Wesley; citation_author=WF Eanes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=13; citation_pages=3132-3136; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The structure and population genetics of the breakpoints associated with the cosmopolitan chromosomal inversion In(3R)Payne in Drosophila melanogaster.; citation_author=LM Matzkin; citation_author=T Merritt; citation_author=CT Zhu; citation_author=WF Eanes; citation_journal_title=Genetics; citation_volume=170; citation_number=14; citation_pages=1143-1152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Segmental duplications: organization and impact within the current human genome project assembly.; citation_author=JA Bailey; citation_author=AM Yavor; citation_author=HF Massa; citation_author=BJ Trask; citation_author=EE Eichler; citation_journal_title=Genome Res; citation_volume=11; citation_number=15; citation_pages=1005-1017; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Long inverted repeats in eukaryotic genomes: recombinogenic motifs determine genomic plasticity.; citation_author=Y Wang; citation_author=FC Leung; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=16; citation_pages=1277-1284; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes.; citation_author=PE Warburton; citation_author=J Giordano; citation_author=F Cheung; citation_author=Y Gelfand; citation_author=G Benson; citation_journal_title=Genome Res; citation_volume=14; citation_number=17; citation_pages=1861-1869; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Study of intrachromosomal duplications among the eukaryote genomes.; citation_author=G Achaz; citation_author=P Netter; citation_author=E Coissac; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=18; citation_pages=2280-2288; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Segmental duplications and the evolution of the primate genome.; citation_author=RV Samonte; citation_author=EE Eichler; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=19; citation_pages=65-72; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular-evolutionary mechanisms for genomic disorders.; citation_author=P Stankiewicz; citation_author=JR Lupski; citation_journal_title=Curr Opin Genet Dev; citation_volume=12; citation_number=20; citation_pages=312-319; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A polytene chromosome analysis of the Anopheles gambiae species complex.; citation_author=M Coluzzi; citation_author=A Sabatini; citation_author=A Della Torre; citation_author=MA Di Deco; citation_author=V Petrarca; citation_journal_title=Science; citation_volume=298; citation_number=21; citation_pages=1415-1418; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Chromosomal inversion intergradation and incipient speciation in Anopheles gambiae.; citation_author=M Coluzzi; citation_author=V Petrarca; citation_author=MA DiDeco; citation_journal_title=Boll Zool; citation_volume=52; citation_number=22; citation_pages=45-63; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=The distribution and inversion polymorphism of chromosomally recognized taxa of the Anopheles gambiae complex in Mali, West Africa.; citation_author=YT Toure; citation_author=V Petrarca; citation_author=SF Traore; citation_author=A Coulibaly; citation_author=HM Maiga; citation_journal_title=Parassitologia; citation_volume=40; citation_number=23; citation_pages=477-511; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=On the distribution and genetic differentiation of Anopheles gambiae s.s. molecular forms.; citation_author=A Della Torre; citation_author=Z Tu; citation_author=V Petrarca; citation_journal_title=Insect Biochem Mol Biol; citation_volume=35; citation_number=24; citation_pages=755-769; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cloning of inversion breakpoints in the Anopheles gambiae complex traces a transposable element at the inversion junction.; citation_author=KD Mathiopoulos; citation_author=A della Torre; citation_author=V Predazzi; citation_author=V Petrarca; citation_author=M Coluzzi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=25; citation_pages=12444-12449; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Distribution of T1, Q, Pegasus and mariner transposable elements on the polytene chromosomes of PEST, a standard strain of Anopheles gambiae.; citation_author=O Mukabayire; citation_author=NJ Besansky; citation_journal_title=Chromosoma; citation_volume=104; citation_number=26; citation_pages=585-595; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Construction of a BAC library and generation of BAC end sequence-tagged connectors for genome sequencing of the African malaria mosquito Anopheles gambiae.; citation_author=YS Hong; citation_author=JR Hogan; citation_author=X Wang; citation_author=A Sarkar; citation_author=C Sim; citation_journal_title=Mol Genet Genomics; citation_volume=268; citation_number=27; citation_pages=720-728; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor.; citation_author=O Kohany; citation_author=AJ Gentles; citation_author=L Hankus; citation_author=J Jurka; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=28; citation_pages=474; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=RepeatMasker Open-3.0.; citation_author=AFA Smit; citation_author=R Hubley; citation_author=P Green; citation_number=29; citation_date=1996–2004; " />
      <meta name="citation_reference" content="citation_title=Unusual haplotype structure at the proximal breakpoint of In(2L)t in a natural population of Drosophila melanogaster.; citation_author=P Andolfatto; citation_author=JD Wall; citation_author=M Kreitman; citation_journal_title=Genetics; citation_volume=153; citation_number=30; citation_pages=1297-1311; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Inversion polymorphisms and nucleotide variability in Drosophila.; citation_author=P Andolfatto; citation_author=F Depaulis; citation_author=A Navarro; citation_journal_title=Genet Res; citation_volume=77; citation_number=31; citation_pages=1-8; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Eucaryotic transposable genetic elements with inverted terminal repeats.; citation_author=S Potter; citation_author=M Truett; citation_author=M Phillips; citation_author=A Maher; citation_journal_title=Cell; citation_volume=20; citation_number=32; citation_pages=639-647; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Retrotransposons and the FB transposon from Drosophila melanogaster.; citation_author=PM Bingham; citation_author=Z Zachar; citation_number=33; citation_pages=485-502; citation_date=1989; citation_publisher=American Society for Microbiology; " />
      <meta name="citation_reference" content="citation_title=Structural variation of the human genome.; citation_author=AJ Sharp; citation_author=Z Cheng; citation_author=EE Eichler; citation_journal_title=Annu Rev Genomics Hum Genet; citation_volume=7; citation_number=34; citation_pages=407-442; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Aberrant transposition of a Tc1-mariner element, impala, in the fungus Fusarium oxysporum.; citation_author=A Hua-Van; citation_author=T Langin; citation_author=MJ Daboussi; citation_journal_title=Mol Genet Genomics; citation_volume=267; citation_number=35; citation_pages=79-87; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Generation of a widespread Drosophila inversion by a transposable element.; citation_author=M Caceres; citation_author=JM Ranz; citation_author=A Barbadilla; citation_author=M Long; citation_author=A Ruiz; citation_journal_title=Science; citation_volume=285; citation_number=36; citation_pages=415-418; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Isolation and characterization of Y chromosome sequences from the African malaria mosquito Anopheles gambiae.; citation_author=J Krzywinski; citation_author=D Nusskern; citation_author=M Kern; citation_author=NJ Besansky; citation_journal_title=Genetics; citation_volume=166; citation_number=37; citation_pages=1291-1302; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Degenerating Y chromosome of Drosophila miranda: a trap for retrotransposons.; citation_author=M Steinemann; citation_author=S Steinemann; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=89; citation_number=38; citation_pages=7591-7595; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Factors affecting inverted repeat stimulation of recombination and deletion in Saccharomyces cerevisiae.; citation_author=KS Lobachev; citation_author=BM Shor; citation_author=HT Tran; citation_author=W Taylor; citation_author=JD Keen; citation_journal_title=Genetics; citation_volume=148; citation_number=39; citation_pages=1507-1524; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Excision of transposon Tn5 is dependent on the inverted repeats but not on the transposase function of Tn5.; citation_author=C Egner; citation_author=DE Berg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=78; citation_number=40; citation_pages=459-463; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=Chromosome speciation: Humans, Drosophila, and mosquitoes.; citation_author=FJ Ayala; citation_author=M Coluzzi; citation_journal_title=Proc Natl Acad Sci U S A 102 Suppl; citation_volume=1; citation_number=41; citation_pages=6535-6542; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization of two natural hotspots in the Drosophila buzzatii genome induced by transposon insertions.; citation_author=M Caceres; citation_author=M Puig; citation_author=A Ruiz; citation_journal_title=Genome Res; citation_volume=11; citation_number=42; citation_pages=1353-1364; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Polytene chromosome preparation from anopheline mosquitoes.; citation_author=A della Torre; citation_number=43; citation_pages=329-336; citation_date=1997; citation_publisher=Chapman & Hall; " />
      <meta name="citation_reference" content="citation_title=In situ hybridization to Anopheles polytene chromosomes.; citation_author=V Kumar; citation_author=AJ Cornel; citation_author=O Mukabayire; citation_number=44; citation_pages=337-345; citation_date=1997; citation_publisher=Chapman & Hall; " />
      <meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods.; citation_author=J Rozas; citation_author=JC Sanchez-DelBarrio; citation_author=X Messeguer; citation_author=R Rozas; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=45; citation_pages=2496-2497; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae"/>
    <meta name="twitter:description" content="The malaria vector Anopheles gambiae maintains high levels of inversion polymorphism that facilitate its exploitation of diverse ecological settings across tropical Africa. Molecular characterization of inversion breakpoints is a first step toward understanding the processes that generate and maintain inversions. Here we focused on inversion 2Rj because of its association with the assortatively mating Bamako chromosomal form of An. gambiae, whose distinctive breeding sites are rock pools beside the Niger River in Mali and Guinea. Sequence and computational analysis of 2Rj revealed the same 14.6 kb insertion between both breakpoints, which occurred near but not within predicted genes. Each insertion consists of 5.3 kb terminal inverted repeat arms separated by a 4 kb spacer. The insertions lack coding capacity, and are comprised of degraded remnants of repetitive sequences including class I and II transposable elements. Because of their large size and patchwork composition, and as no other instances of these insertions were identified in the An. gambiae genome, they do not appear to be transposable elements. The 14.6 kb modules inserted at both 2Rj breakpoint junctions represent low copy repeats (LCRs, also called segmental duplications) that are strongly implicated in the recent (&sim;0.4Ne generations) origin of 2Rj. The LCRs contribute to further genome instability, as demonstrated by an imprecise excision event at the proximal breakpoint of 2Rj in field isolates."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000849.g005"/>

  <meta property="og:title" content="Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3739'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1451'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2930&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000849">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000849" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000849&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Segmental Duplication Implicated in the Genesis of Inversion 2<i>Rj</i> of <i>Anopheles gambiae</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mamadou B. Coulibaly, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America, 
                  Malaria Research and Training Center, University of Bamako, Bamako, Mali
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Neil F. Lobo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Meagan C. Fitzpatrick, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marcia Kern, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olga Grushko, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Daniel V. Thaner, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sékou F. Traoré, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Malaria Research and Training Center, University of Bamako, Bamako, Mali
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank H. Collins, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nora J. Besansky
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:nbesansk@nd.edu">nbesansk@nd.edu</a></p>

                <p>Affiliation:
                  Center for Global Health and Infectious Diseases, Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000849</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000849-g001" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000849-g002" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000849-g003" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000849-g004" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000849-g005" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000849-t001" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000849">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000849" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The malaria vector <em>Anopheles gambiae</em> maintains high levels of inversion polymorphism that facilitate its exploitation of diverse ecological settings across tropical Africa. Molecular characterization of inversion breakpoints is a first step toward understanding the processes that generate and maintain inversions. Here we focused on inversion 2Rj because of its association with the assortatively mating Bamako chromosomal form of <em>An. gambiae,</em> whose distinctive breeding sites are rock pools beside the Niger River in Mali and Guinea. Sequence and computational analysis of 2Rj revealed the same 14.6 kb insertion between both breakpoints, which occurred near but not within predicted genes. Each insertion consists of 5.3 kb terminal inverted repeat arms separated by a 4 kb spacer. The insertions lack coding capacity, and are comprised of degraded remnants of repetitive sequences including class I and II transposable elements. Because of their large size and patchwork composition, and as no other instances of these insertions were identified in the <em>An. gambiae</em> genome, they do not appear to be transposable elements. The 14.6 kb modules inserted at both <em>2Rj</em> breakpoint junctions represent low copy repeats (LCRs, also called segmental duplications) that are strongly implicated in the recent (~0.4N<sub>e</sub> generations) origin of <em>2Rj</em>. The LCRs contribute to further genome instability, as demonstrated by an imprecise excision event at the proximal breakpoint of <em>2Rj</em> in field isolates.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Coulibaly MB, Lobo NF, Fitzpatrick MC, Kern M, Grushko O, et al.  (2007) Segmental Duplication Implicated in the Genesis of Inversion 2<em>Rj</em> of <em>Anopheles gambiae</em>. PLoS ONE 2(9):
          e849.
            doi:10.1371/journal.pone.0000849</p><p><strong>Academic Editor: </strong>Rodolfo Aramayo, Texas A&amp;M University, United States of America</p><p><strong>Received:</strong> June 5, 2007; <strong>Accepted:</strong> August 15, 2007; <strong>Published:</strong> September 5, 2007</p><p><strong>Copyright:</strong> © 2007 Coulibaly et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This investigation received financial support from the UNDP/World Bank/WHO Special Programme for Research and Training in Tropical Diseases (A30272 to NJB), and from the National Institutes of Health (U01 AI48846 to FHC). NJB was supported by NIH R01 AI48842 and R01 AI63508.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Seventy years ago, Dobzhansky expressed “…little doubt that chromosomal changes are one of the mainsprings of evolution” <a href="#pone.0000849-Dobzhansky1">[1]</a>. Although this view was founded on evidence from Drosophila in which rearrangements of the banding pattern are readily detected on giant (polytene) chromosomes, the widespread occurrence of chromosomal rearrangements is now apparent in light of the many whole genome sequences from yeasts to humans and plants. In dipteran flies, including anopheline mosquitoes, paracentric inversions are the most common type of chromosomal change. Inversion polymorphisms often segregate in nonrandom patterns with respect to environmental and phenotypic variation, suggesting that inversions are subject to strong selection <a href="#pone.0000849-Coghlan1">[2]</a>–<a href="#pone.0000849-Powell1">[4]</a>. Adaptations associated with alternative arrangements can be preserved, despite migration and gene flow between their carriers, through suppressed recombination in inversion heterozygotes. This mechanism may contribute to ongoing ecotypic differentiation and ultimately speciation <a href="#pone.0000849-Noor1">[5]</a>, <a href="#pone.0000849-Rieseberg1">[6]</a>, explaining why fixed chromosomal inversion differences often distinguish closely related species <a href="#pone.0000849-Powell1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Given their significance in evolution, an appreciation for how inversions arise in natural populations is of great interest. Transposable elements (TEs) are known to induce chromosomal rearrangements indirectly through ectopic recombination or aberrant gene conversion and directly by an alternative transposition process <a href="#pone.0000849-Gray1">[7]</a>. Both class I elements (retrotransposons) and class II elements (transposons, including foldback elements) have been implicated in the generation of rearrangements in a wide array of organisms <a href="#pone.0000849-Coghlan1">[2]</a>, <a href="#pone.0000849-Kidwell1">[8]</a>, <a href="#pone.0000849-Wessler1">[9]</a>. However, repetitive sequences recognizable as TEs are not inevitably associated with the breakpoints of chromosomal inversions <a href="#pone.0000849-Cirera1">[10]</a>–<a href="#pone.0000849-Wesley1">[13]</a>. Moreover, their occurrence at one or both breakpoints may not unambiguously implicate TEs in generating the inversion, as their arrival may have been secondary <a href="#pone.0000849-Matzkin1">[14]</a>. Although they constitute a proven mechanism, TEs are neither the exclusive nor perhaps the predominant means of chromosomal rearrangement.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Segmental duplications are direct or inverted repetitions of blocks of the genome from 1 to &gt;400 kb in length and 90–100% sequence identity <a href="#pone.0000849-Bailey1">[15]</a>. A growing body of evidence suggests that paralogous segmental duplications, also known as low-copy repeats (LCRs), may be important determinants of genomic plasticity in eukaryotes <a href="#pone.0000849-Wang1">[16]</a>–<a href="#pone.0000849-Achaz1">[18]</a>. In nine model eukaryotes, genome-wide computational analyses of inverted repeats revealed that these structures can be quite frequent, even after masking of known repetitive elements <a href="#pone.0000849-Wang1">[16]</a>, <a href="#pone.0000849-Warburton1">[17]</a>. Because the arms are inverse complements of one another, inverted repeats can form secondary structures (palindromes or stem-loops) that are prone to recombination, leading to chromosomal rearrangements including inversions. This mechanism underlies several human genomic disorders, contributes to tumorigenesis, and has played a major role in primate karyotype evolution <a href="#pone.0000849-Samonte1">[19]</a>, <a href="#pone.0000849-Stankiewicz1">[20]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p><em>Anopheles gambiae sensu stricto</em>, a principal Afrotropical vector of malaria, is one of seven recognized species in the <em>An. gambiae</em> sibling species complex. Chromosomal inversions have been and continue to be instrumental in the evolutionary diversification of this group. No less than 120 polymorphic paracentric rearrangements have been described within the complex, and all except one pair of species can be distinguished by at least one fixed inversion difference <a href="#pone.0000849-Coluzzi1">[21]</a>. <em>An. gambiae s.s.</em> (hereafter, <em>An. gambiae</em>) is one of the most chromosomally polymorphic species in the complex. Not coincidentally, it is also one of the most geographically widespread species, occupying diverse ecoclimatic zones and ecological settings across Africa south of the Sahara. Data compiled from several decades of study in West Africa, entailing nearly 18,000 karyotyped <em>An. gambiae</em>, show nonrandom associations of five inversions on the right arm of chromosome 2 that correspond to different ecophenotypes termed chromosomal forms <a href="#pone.0000849-Coluzzi2">[22]</a>, <a href="#pone.0000849-Toure1">[23]</a>. Three of these coexist in Mali–the Mopti, Savanna and Bamako chromosomal forms–each with characteristic larval breeding sites. Mopti is associated with semi-permanent sites such as rice fields, Savanna with rain-dependent sites, and Bamako with rock pools at the margins of the Niger River in southern Mali and Guinea (<a href="#pone-0000849-g001">Figure 1</a>). In addition to its distinctive larval ecology, Bamako is characterized by fixation of the 2<em>Rj</em> chromosomal inversion, which is absent from Mopti and present only at very low frequency in Savanna populations <a href="#pone.0000849-DellaTorre1">[24]</a>. Other 2R inversions associated with Bamako (<em>b,c,</em> and <em>u</em>) are shared between chromosomal forms, albeit at different frequencies.</p>
<div class="figure" id="pone-0000849-g001"><div class="img"><a name="pone-0000849-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Rock pools along the Niger River, typical breeding sites of the <em>An. gambiae</em> Bamako chromosomal form.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Courtesy of M. Fodde and M. Coluzzi.</p>
<span>doi:10.1371/journal.pone.0000849.g001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The only other inversions in the <em>An. gambiae</em> complex whose breakpoint structures have been characterized are 2<em>La</em> from <em>An. gambiae</em> and 2<em>Rd'</em> from <em>An. arabiensis</em> <a href="#pone.0000849-Sharakhov1">[12]</a>, <a href="#pone.0000849-Mathiopoulos1">[25]</a>. The geographically widespread and interspecific distribution of 2<em>La</em> point to a relatively ancient origin; no decisive evidence of TEs was found at the breakpoints, though repetitive structures suggestive of their remnants were present. By contrast, inversion 2<em>Rd'</em> is exclusive to <em>An. arabiensis</em> and is apparently absent from populations east of the Rift Valley, raising the possibility of a more recent origin. A TE fragment found at the distal breakpoint designated <em>Odysseus</em> was implicated in the generation of 2<em>Rd'</em> <a href="#pone.0000849-Mathiopoulos1">[25]</a>. Because 2<em>Rj</em> is limited to West African populations of <em>An. gambiae</em>, this inversion also may be relatively young. To gain insight into its origin, we cloned the 2<em>Rj</em> inversion breakpoints. Here, we describe the breakpoint structure and report the presence of giant perfectly conserved inverted repeat sequences at both junctions that are distinct from TEs. The implications of this structure are evaluated in light of genome-wide computational analysis, <em>in situ</em> hybridizations and sequencing of field specimens of <em>An. gambiae</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Isolation of the 2<em>Rj</em> breakpoints</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The <em>An. gambiae</em> reference genome, sequenced from the PEST strain, is homokaryotypic standard (uninverted) for all chromosome arrangements <a href="#pone.0000849-Mukabayire1">[26]</a>. From a BAC library prepared from PEST <a href="#pone.0000849-Hong1">[27]</a>, candidate clones known to map in the vicinity of the 2<em>R+<sup>j</sup></em> proximal breakpoint (cytogenetic subdivision 10C) were hybridized to 2<em>Rj</em> chromosomes from the BKO strain of the Bamako chromosomal form of <em>An. gambiae</em>. BAC clone 153N7 hybridized to both the proximal and the distal (subdivision 7C) ends of the 2<em>Rj</em> arrangement (<a href="#pone-0000849-g002">Figure 2</a>), indicating that it contained inversion breakpoints. Subsequent rounds of hybridization to 2<em>Rj</em> chromosomes using smaller fragments of 153N7 refined the location of the proximal breakpoint to a 1.3 kb segment of single-copy PEST DNA. This fragment was used as a probe to isolate the corresponding sequences from a plasmid genomic library prepared from the BKO (2<em>Rj</em>) strain.</p>
<div class="figure" id="pone-0000849-g002"><div class="img"><a name="pone-0000849-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Cloning strategy for the 2<em>Rj</em> inversion breakpoints.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Relative position of four paracentric inversions (<em>j</em>, <em>b</em>, <em>c</em>, <em>u</em>) on chromosome 2<em>R</em> in <em>An. gambiae</em>. Asterisk represents the centromere. (B) Schematic of alternative arrangements of 2<em>Rj</em> in PEST (+<sup>j</sup>) and BKO (<em>j</em>) strains (not to scale). The cytogenetic map represented beneath the PEST chromosome shows the breakpoints positioned in subdivision 7C and 10C. Numbers above the chromosomes are used in the text to refer to corresponding regions adjacent to the breakpoints, represented as identically hatched or shaded regions in alternative arrangements. Dashed line indicates repetitive DNA insertion. Shown beneath the PEST chromosome is a representation of BAC 153N7 that spans the proximal breakpoint. Below the BKO chromosome are plasmid (1, 5) and fosmid (59I22, 35D6) clones corresponding to the breakpoints of the <em>j</em> arrangement. Structure of the repetitive DNA in 59I22 and 35D6 is represented below as arrows (inverted repeats) separated by a solid line (central spacer) of indicated lengths in kb. (C) FISH performed on the chromosomes of the BKO strain (2<em>Rj</em>) with BAC 153N7. Signals in red indicate the distal and the proximal breakpoints of the <em>j</em> inversion.</p>
<span>doi:10.1371/journal.pone.0000849.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Two plasmids, clone 1 (11.3 kb) and clone 5 (11.5 kb), were recovered from the screen and completely sequenced. Each clone was characterized by a small terminal region of single-copy DNA corresponding to subdivision 10C in PEST (1.6kb and 168bp, respectively), but derived from opposite ends of the 2<em>Rj</em> inversion in BKO (<a href="#pone-0000849-g002">Figure 2</a>). The remaining sequences of clones 1 and 5 were repetitive and 99% identical between the two clones across their entire length. Neither clone contained sequences similar to subdivision 7C, indicating that the opposite breakpoint was not present.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To recover clones with inserts of sufficient length to span the breakpoints, a fosmid library constructed from the same BKO strain was screened by PCR, using primer sequences designed from the unique regions of clones 1 and 5. One clone from each breakpoint was isolated: fosmid 59I22 (distal) and fosmid 35D6 (proximal). Interrogating the <em>An. gambiae</em> PEST genome with end-sequences of both BKO fosmid clones revealed matches to unique regions of 7C and 10C at opposite ends, indicating that both clones spanned the long repetitive sequences at the breakpoints (<a href="#pone-0000849-g002">Figure 2</a>). Sequence was determined across the breakpoint junctions in both fosmids and extended 2–4 kb into flanking sequences at either end, totaling 22 kb for 59I22 and 19 kb for 35D6.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>The pairwise alignment of 2<em>R+<sup>j</sup></em> standard (PEST) and 2<em>Rj</em> inverted (BKO) sequences revealed the approximate nucleotide position of the inversion breakpoints (<a href="#pone-0000849-g003">Figure 3</a>). Spanning three divisions of the cytogenetic map (7C–10C), the 2<em>Rj</em> inversion physically encompasses 12.5 Mb between positions 3,262,186 and 15,750,717 on chromosome 2R in PEST assembly AgamP3. The distal breakpoint in 7C (between regions labeled in <a href="#pone-0000849-g002">Figure 2</a> as “1” and “2” on the standard arrangement of PEST) occurred between positions 3,262,186 and 3,262,296; the intervening 110 bp are missing from the inverted chromosome. Remarkably, the proximal breakpoint in 10C (between regions “3” and “4” on the PEST chromosome in <a href="#pone-0000849-g002">Figure 2</a>) seems to have occurred precisely between positions 15,750,716-15,750,717.</p>
<div class="figure" id="pone-0000849-g003"><div class="img"><a name="pone-0000849-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Sequences at the breakpoints of alternative arrangements of 2<em>Rj.</em></span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The standard arrangement represented by PEST is aligned to the inverted arrangement represented by BKO in the regions (1–4) immediately surrounding the breakpoints. Vertical lines indicate sequence identity. In bracketed bold text is the terminal sequence of the intervening inverted repeat (represented by a dashed line in the diagram). Underlined are putative target site duplications in inverse complementary orientation.</p>
<span>doi:10.1371/journal.pone.0000849.g003</span></div>

<h4>The 2<em>Rj</em> breakpoints contain long inverted repeats</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The 2<em>Rj</em> arrangement of BKO is framed by two 14.6 kb insertions that are absent from the corresponding locations on 2<em>R+<sup>j</sup></em> in PEST (<a href="#pone-0000849-g002">Figure 2</a>). The distal and proximal insertions are &gt;99% identical to each other across nearly their entire lengths (14,608 identities across 14,627 aligned positions). To search for direct or inverted repeats within each insertion, the sequences were aligned against themselves in both orientations and the results were summarized by dot-plot (not shown). These analyses revealed that each insertion is composed of 5.3 kb terminal inverted repeats (IRs) separated by a 4 kb middle segment (spacer). The IRs belonging to the same insertion are &gt;99% or 100% identical (proximal or distal insertion, respectively).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The insertions are repetitive in the <em>An. gambiae</em> genome, but not as a contiguous 14.6 kb sequence in the genome assembly. A nucleotide-level interrogation (BLASTN) of the <em>An. gambiae</em> genome using the 14.6 kb insertion sequence as a query returned &gt;3000 hits with expectation values (E) &lt;10<sup>−5</sup>, predominantly from relatively short unmapped scaffolds. However, no BLAST alignment exceeded 2kb in length, nor did multiple alignments from the same hit approximate the full-length insertion sequence, although the 118 kb unmapped scaffold AAAB01008935 bears three consecutive islands of similarity corresponding to ~7 kb of the insertion sequence. Nevertheless, we cannot rule out that the absence of full-length hits was an artifact of large gaps (represented by strings of N's) between contigs within repetitive unmapped scaffolds. Indeed, it seems likely that automated genome assembly pipelines would have difficulty assembling a sequence with 5.3 kb perfect (or nearly perfect) IRs.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>More refined BLASTN searches with smaller components of the sequence were conducted iteratively, in an effort to reconstruct consensus sequences from potential gene or TE fragments. These were followed by protein-level (BLASTX) searches at NCBI of the non-redundant GenBank database. Weak similarities (E≅10<sup>−8</sup>) were identified to a transposase in the IRs and to the Drosophila protein product of gene CG5555 in the spacer. Based on sequence similarity, CG5555 may have zinc ion binding and ubiquitin-protein ligase activities. However, any sequence similarities to the 14.6 kb sequence were only fragmentary and are unlikely to correspond to functional proteins. Overall, the 14.6 kb insertions lack obvious coding potential, as judged by the absence of significant open reading frames. Different parts of the insertion are highly repeated in the <em>An. gambiae</em> genome and share segments of sequence similarity with various Class I and II TEs, but are not themselves full-length elements. Screening of the insertion sequence against a reference collection of repeats (Repbase) using CENSOR software <a href="#pone.0000849-Kohany1">[28]</a> identified putative fragments of <em>An. gambiae</em> TEs, including Harbinger1_AG DNA transposon in the IRs and SINEX-1_AG and GYPSY9-LTR_AG retrotransposons in the spacer (Supplementary <a href="#pone.0000849.s001">Table S1</a>). RepeatMasker (<a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>; <a href="#pone.0000849-Smit1">[29]</a>) a program that screens sequences for interspersed and simple repeats, returned congruent results. No association with MITEs was observed.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>To avoid the implication that the 14.6 kb insertions are TEs, the full length structures have been designated as low-copy repeats (LCRs).</p>


<h4>Instability associated with LCRs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A preliminary assessment of the presence of these LCRs (or transposon/retrotransposon components of them) at the 2<em>Rj</em> breakpoints of other <em>An. gambiae</em> isolates was performed by fluorescent <em>in situ</em> hybridization (FISH) to polytene chromosomes, using one of the IRs as a probe. As expected, signals were detected at both breakpoints in the BKO strain from which the fosmid and plasmid clones were derived. The same pattern also was detected in a wild specimen of the Bamako chromosomal form (2<em>Rj</em> homozygote), but only one of the homologs gave a signal at the proximal breakpoint (not shown). None of the 2<em>R+<sup>j</sup></em> karyotypes from Mali, Burkina Faso or Cameroon were labeled at either breakpoint. Other sites of hybridization were detected in all specimens. Centric heterochromatin and the heterochromatin of subdivision 21A on 2L were weakly and diffusely labeled. Additional euchromatic sites were labeled on all arms except 2L: X (1C); 2R (14D, 16A, 18D, 19D); 3L (39A); 3R (30A, 33C). These sites were rarely occupied in more than one specimen, and sometimes appeared to be occupied only in one homolog (<a href="#pone-0000849-g004">Figure 4</a>). We found no more than four euchromatic sites labeled per specimen. At this level of resolution, it was not possible to determine whether the labeled sites are occupied by a complete LCR or merely related transposons of which they are composed.</p>
<div class="figure" id="pone-0000849-g004"><div class="img"><a name="pone-0000849-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>FISH performed on the chromosomes of the BKO strain (2<em>Rj</em>) with one arm of the LCR IR as a probe.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Signal in red maps to subdivision 14D (on 2<em>R</em>) in only one homolog. This site also was labeled in only one homolog on the chromosomes of a wild specimen from Bancoumana, Mali.</p>
<span>doi:10.1371/journal.pone.0000849.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>IR-containing sequences may be prone to deletion, as was hinted by the FISH result at the proximal inversion breakpoint in the 2<em>Rj</em> homozygote. The possibility of polymorphism for the presence of the LCR at one or both breakpoints was investigated further by PCR, using a panel of genomic DNA extracted from 249 2<em>Rj</em> homokaryotypic mosquitoes collected from Mali. Reasoning that amplification across the junction of regions 1–3 and 2–4 could only be successful if the 14.6 kb LCR had been deleted, we attempted PCR across both junctions. No products were obtained from the distal (region 1–3) junction. However, consistent with the FISH result, 70 mosquitoes (28%) yielded PCR products from the proximal (region 2–4) junction. Interestingly, the products were all identical in length and much shorter than the predicted size (184 bp instead of 606 bp) given precise deletion of the LCR. Sequence analysis of a subset (66 amplicons) revealed identical sequences, with the exception of a high frequency C–A polymorphism (SNP) at one position. The shorter than expected product size proved to be the result of an imprecise deletion that removed all except the last 17 bp of the distal end of the LCR as well as ~200 bp of flanking DNA from both region 2 and 4 (<a href="#pone-0000849-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0000849-g005"><div class="img"><a name="pone-0000849-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000849.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Structure of the LCR excision found at the proximal end of 2<em>Rj</em> in a subsample of the Bamako chromosomal form from Mali.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Uppercase letters within boxes represent intact chromosomal regions 2 and 4, and the complete 14.6 kb LCR (shaded in gray). Shown below is the corresponding sequence after excision of the LCR. Dashes represent deleted sequence. The excision event also removed the flanking 198 bp of region 2 and 232 bp of region 4, but left 17 bp at the 3′ end of the LCR.</p>
<span>doi:10.1371/journal.pone.0000849.g005</span></div>

<h4>Predicted genes immediately surrounding the breakpoints of 2<em>Rj</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Manual models of predicted genes nearest the breakpoints suggest that no genes were interrupted by the inversion, though this has not yet been experimentally verified. (The manual models for prospective genes in these regions can be visualized at VectorBase by navigating to the following URLs: <a href="http://agambiae.vectorbase.org/Genome/ContigView/?">http://agambiae.vectorbase.org/Genome/Co​ntigView/?</a>c = 2R:15750000;w = 20000 and <a href="http://agambiae.vectorbase.org/Genome/ContigView/?">http://agambiae.vectorbase.org/Genome/Co​ntigView/?</a>c = 2R:3262200;w = 20000, and selecting the manual annotation track available through the DAS sources menu.) The proximal breakpoint falls between two divergently transcribed genes. The first, AGAP002110-RA, potentially encodes a kinase related to the CDC7 family. The product of the second, AGAP002111-RA, contains a CCCH-type zinc finger with a G-patch domain suggesting its involvement in nucleic acid binding. These genes (henceforth, 2110 and 2111) respectively lie 6.9 kb and 407 bp from the breakpoint in regions 3 and 4 of <a href="#pone-0000849-g002">Figure 2</a>. The distal breakpoint (between regions 1 and 2 of <a href="#pone-0000849-g002">Figure 2</a>) also falls between two divergently transcribed genes, AGAP001334-RA and AGAP001335-RA (henceforth, 1334 and 1335). The potential product of 1334, 721 bp from the breakpoint, did not match any known conserved domains and has an unknown function, but was similar to mesenchymal stem cell protein DSC92. The latter gene, whose manual model is positioned only 371 bp from the breakpoint, appears to encode a dynein light chain of type-1. Sufficient sequence was determined from the BKO fosmids to align three of the four putative genes from BKO and PEST across their lengths. Gaps, if any, were limited to predicted introns and sequence identity was &gt;96%.</p>


<h4>Recent origin of 2R<em>j</em></h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>An estimate for the minimum age of 2R<em>j</em> was derived under a model that assumes a unique origin for the inversion, complete genetic isolation of alternative arrangements, and balancing selection which has maintained the inversion at its present frequency since immediately following its origin <a href="#pone.0000849-Andolfatto1">[30]</a>. The estimate is based on the observed number of segregating sites within the inverted and standard chromosomes at a locus inside the inversion adjacent to the breakpoint, where neither crossing over nor gene conversion are expected <a href="#pone.0000849-Andolfatto2">[31]</a>. Using homokaryotypic (2R<em>j</em>/<em>j</em> or 2R+<sup>j</sup>/+<sup>j</sup>) mosquitoes collected in Mali, sequences were determined from two predicted genes, one inside the inversion only 6.9 kb from the proximal breakpoint (2110) and another outside the inversion only 712 bp away from the distal breakpoint (1334). The results are summarized in <a href="#pone-0000849-t001">Table 1</a>. Although loci outside the inversion are homosequential on chromosomes carrying alternative arrangements, diversity is considerably reduced on inverted (derived) chromosomes due to asynapsis in heterokaryotypes <a href="#pone.0000849-Andolfatto2">[31]</a>. As expected, diversity was three-fold lower on 2Rj versus 2R+<sup>j</sup> chromosomes at locus 1334 outside the inversion. Within the inversion at locus 2110, diversity was reduced nearly 19-fold on inverted versus standard chromosomes. Using the data from locus 2110, we estimated the historical frequency of 2Rj as 0.108 and the minimum age as 0.4N<sub>e</sub> generations. There were no mutations shared between alternative arrangements but, consistent with a recent origin, there were also no fixed differences (Supplementary <a href="#pone.0000849.s002">Figure S1</a>).</p>
<div class="figure" id="pone-0000849-t001"><div class="img"><a name="pone-0000849-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000849" data-uri="info:doi/10.1371/journal.pone.0000849.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000849.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000849.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000849.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000849.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000849.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Levels of polymorphism at two loci near the <em>2Rj</em> breakpoint junction from standard and inverted chromosomes.</span></strong></p><span>doi:10.1371/journal.pone.0000849.t001</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Molecular analysis of both breakpoint junctions of 2<em>Rj</em> chromosomes revealed that they bear nearly identical 14.6 kb insertions that we have classified as LCRs. Each LCR is bounded by identical 5.3 kb IRs separated by a 4 kb spacer. There is no apparent coding capacity and no similar full-length structures were detected in the <em>An. gambiae</em> reference genome, not even at the <em>2R+<sup>j</sup></em> breakpoint junctions. Instead, the LCRs show weak and fragmentary similarity in portions of the IRs and spacer to several unrelated class I and II TEs, suggesting their origin in a graveyard of degenerate TEs. There is a superficial structural resemblance to foldback TEs, a distinctive category of Class II TEs first discovered in <em>Drosophila melanogaster</em> <a href="#pone.0000849-Potter1">[32]</a> and later found in a wide variety of organisms. Foldback elements carry long terminal IR arms from several hundred to a thousand base pairs long separated by a spacer of variable length. They move by an unknown mechanism and most apparently lack coding capacity <a href="#pone.0000849-Bingham1">[33]</a>. However, the similarity to the <em>An. gambiae</em> LCRs seems spurious for several reasons. First, the IRs of the <em>An. gambiae</em> LCR substantially exceed the 1–2 kb maximum length reported for any other foldback elements, and lack the terminal blocks of subrepeats often associated with this type of TE. Unlike foldback elements, the <em>An. gambiae</em> LCRs are not flanked by short target site duplications as expected from TE mobilization (however as discussed below, 8 bp sequences that may have been caused by an original insertion event are present). More important, no other copies could be recognized in the <em>An. gambiae</em> genome that would indicate replicative behavior typical of a foldback or any other kind of TE. Indeed, it seems unlikely that these LCRs are functional TEs or composite TEs. Available data suggest that they are segmental duplications, distinct from high copy mobile elements such as SINEs, LINEs and other TEs. The human genome is particularly enriched for segmental duplications <a href="#pone.0000849-Sharp1">[34]</a>, but they occur in all eukaryotic genomes <a href="#pone.0000849-Achaz1">[18]</a>. Unlike transposition that results in the amplification of specific TE sequences, segmental duplications involve “generic” DNA regions of variable length (1 to &gt;400 kb) arising from sister chromatid exchange or double-strand break repair associated with gene conversion.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Segmental duplications have been implicated in genome restructuring of primates, and have played a direct role in generating recurrent chromosomal rearrangements underlying human disease <a href="#pone.0000849-Samonte1">[19]</a>, <a href="#pone.0000849-Stankiewicz1">[20]</a>. Similarly, the 14.6 kb LCR is directly implicated in generating inversion 2<em>Rj</em>, given its presence at both junctions and the absence of any other repetitive structures at the breakpoints. The inversion could have originated by one of two alternative mechanisms. First, if both 14.6 kb LCRs were already present at both future breakpoints on the standard chromosome, these nearly identical sequences could serve as targets for non-allelic homologous recombination, leading to inversion of the intervening sequence. Normally, inversion is the outcome only if the non-allelic sequences are present in inverted orientation relative to one another. In the present case, because each 14.6 kb insertion is composed of 5.3 kb terminal IRs, inversion could be the outcome even if the full-length structures were present in direct orientation, via pairing of opposite IR arms. (Unfortunately, PCR experiments to determine the relative orientation of the two insertions based on the unique spacer sequence were unsuccessful).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A second possible mechanism that could have induced the inversion has been referred to as alternative or aberrant transposition in TE studies <a href="#pone.0000849-Gray1">[7]</a>, <a href="#pone.0000849-HuaVan1">[35]</a>, but this process need not be exclusive to TEs. Under this scenario, the starting point would have been a single 14.6 kb LCR at the proximal breakpoint on the standard chromosome. Synapsis between IRs on sister chromatids, followed by (1) a double strand break at one end and (2) insertion of the “hybrid” structure into a new target site (<em>i.e</em>., the other breakpoint of 2<em>Rj</em>) on one of the same chromosome arms involved in the hybrid would result in an inversion <a href="#pone.0000849-Gray1">[7]</a>. As the final outcomes of homologous recombination and alternative transposition are the same, the data do not allow us to conclusively distinguish these two mechanisms; either one is plausible. However, some evidence supports the second mechanism whereby a single proximal LCR promotes the inversion. Underlined in <a href="#pone-0000849-g003">Figure 3</a> is a putative 8 bp target site duplication. This duplication originally would have existed as a short direct repeat flanking the proximal 14.6 kb LCR on the standard chromosome, and may have resulted from double-strand break repair upon its integration. As a result of the inversion, the 8 bp sequence now exists in inverse complementary orientation with each site flanking different copies of the LCR, as expected (<em>e.g</em>., <a href="#pone-0000849-g003">Figure 3</a> of <a href="#pone.0000849-Caceres1">[36]</a>). No reciprocal set of target site duplications was noted, probably because they were never formed during resolution of the alternative transposition process.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>If the 2Rj inversion was caused by the 14.6 kb LCR, how might the LCR itself have arisen? No structure matching the full-length LCR (or its IR components) was identified in the <em>An. gambiae</em> reference genome. However, a short (118 kb) unmapped scaffold AAAB01008935 contained large adjacent segments of similarity extending consecutively across ~7 kb of the insertion sequence. Scaffolds that are both short and unmapped are typically composed of repetitive DNA, which interferes with assembly and ability to assign a unique map location. Regions of the <em>An. gambiae</em> genome where repetitive DNA is significantly overrepresented are pericentric heterochromatin and the fully heterochromatic Y chromosome (<em>e.g</em>., <a href="#pone.0000849-Krzywinski1">[37]</a>); such regions seem to be sinks for transposable elements and other repeat structures <a href="#pone.0000849-Steinemann1">[38]</a>. In these same regions of the human genome, segmental duplications are enriched 3–5 fold. It seems likely that the 14.6 kb LCR originated in one of these regions. Achaz <em>et al</em> <a href="#pone.0000849-Achaz1">[18]</a> proposed a model for the dynamics of duplication in eukaryotic genomes based on computational analysis. The engine driving their model is the continuous genesis of adjacent direct (tandem) repeats created by mispairing and recombination between sister chromatids. The tandem repeats are subsequently submitted to rounds of chromosomal rearrangements (deletions, insertions, inversions) that transform them into distantly spaced repeats. According to this model, the 14.6 kb structure could have been generated by a two-step process beginning with the tandem repetition of a 5.3 kb segment (itself a TE graveyard). A later inversion event involving a 9.3 kb DNA segment containing one of the tandem repeats could have produced 5.3 kb IRs separated by a 4 kb spacer. The presence of long IRs capable of forming a stem-loop structure may have destabilized the 14.6 kb module, leading to its excision from pericentric heterochromatin and reinsertion into the proximal breakpoint of the 2<em>Rj</em> inversion.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Long IRs are associated with genetic instability. The rate with which deletions and rearrangement events are stimulated depends upon length of the IRs and the distance between them: longer IRs and shorter spacers are more recombinogenic. In yeast, 185 bp IRs separated by 8 kb spacers did not stimulate recombination, but 1 kb IRs of a perfect palindrome increased recombination as much as 17,000-fold <a href="#pone.0000849-Lobachev1">[39]</a>. Because the 2<em>Rj</em> inversion spans a very large distance (12.5 Mb), interaction between IRs at either end–while not impossible—should be very rare. However, interaction between the relatively long 5.3 kb IRs separated by only 4 kb within a given 14.6 kb module may be more frequent. Excision of inverted repeats is an outcome that follows intra-strand hairpin formation and slippage during DNA replication <a href="#pone.0000849-Egner1">[40]</a>. PCR and sequence data from the proximal breakpoint junction of inverted (2<em>Rj</em>) chromosomes from Mali are consistent with this model. Almost one-third of the chromosomes tested revealed an excision of the proximal 14.6 kb module. All the sequences were identical or nearly so (only one SNP was segregating), suggesting that these sequences probably descended from the same event. This event left the footprint of an imprecise excision that included flanking DNA on both sides, although a short segment (17 bp) of the 3′-end of the module was spared, consistent with slippage during DNA replication (see <a href="#pone-0000849-g005">Figure 5</a>). Although only one type of excision event was detected and only at the proximal breakpoint, additional events could be unmasked with wider geographic sampling and/or a wider selection of PCR primers for screening.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Like most mosquitoes, <em>An. gambiae</em> possesses only 3 chromosomes: two autosomes and sex chromosomes. Accordingly, any randomly occurring inversion has a reasonable chance of capturing a significant fraction of the genome, potentially including sets of genes relevant to adaptive fitness in alternative environments. However, a striking pattern found in <em>An. gambiae</em> is the overrepresentation of polymorphic inversions on chromosome 2R. Although this arm represents &lt;30% of the polytene complement, it carries 58% of all polymorphic inversions <a href="#pone.0000849-Coluzzi1">[21]</a>. Rearrangements of this arm are associated with ecotypic differentiation and incipient speciation of <em>An. gambiae</em> <a href="#pone.0000849-Coluzzi1">[21]</a>, <a href="#pone.0000849-Ayala1">[41]</a>. Therefore, both the origin and the gene content of these inversions are of special interest in an evolutionary and epidemiological context: ecotypic differentiation is increasing the vectorial capacity of this already proficient carrier of malaria <a href="#pone.0000849-Coluzzi1">[21]</a>. Here we have shed light on the origin of the 2<em>Rj</em> inversion that is characteristic of the Bamako chromosomal form of <em>An. gambiae</em>. This 12.5 Mb inversion was not induced by TEs <em>per se</em>, but rather by almost identical long IR modules at both breakpoints that are genetically unstable LCRs. Sequence identity of &gt;99% between both IR arms within a LCR, as well as between the two LCRs, suggests either the homogenizing effect of gene conversion or the relatively recent origin of the 2<em>Rj</em> inversion. A recent origin is consistent with our estimates of the minimum age of the 2Rj inversion based on sequence polymorphism near a breakpoint (~0.4Ne generations) and the lack of fixed differences between alternative arrangements. Three other considerations support the hypothesis of recent origin. First is the absence of any other repetitive DNA at the breakpoints, despite the expectation that IRs at inversion breakpoints should be mutational hotspots for subsequent structural and nucleotide variation <a href="#pone.0000849-Caceres2">[42]</a>. Second is the apparent absence of these modules from the <em>An. gambiae</em> reference genome. Third is the restricted distribution of 2<em>Rj</em> in Africa. <em>An. gambiae</em> occurs throughout tropical Africa, but 2<em>Rj</em> is not found outside of West-Central Africa and the Bamako chromosomal form that is fixed for this inversion is limited to the Niger River and its tributaries in southern Mali and northern Guinea <a href="#pone.0000849-Coluzzi2">[22]</a>. If this rearrangement was indeed recent, the expected sequence conservation at the breakpoints of inverted chromosomes should facilitate ongoing efforts to design a robust molecular diagnostic assay to detect the inversion in natural populations. Successful development of a molecular diagnostic assay will facilitate future studies on the ecological genomics of 2<em>Rj</em>, as current efforts to study ecological and epidemiological correlates associated with this inversion are severely hampered by limited access to readable polytene chromosomes.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Mosquitoes</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The BKO colony of the <em>An. gambiae</em> Bamako chromosomal form used for genomic library construction originated from Moribabougou, Mali in 1998. It was established at the University of Rome “La Sapienza” and kindly provided by A. della Torre. The karyotype was 2<em>Rjcu</em>/<em>jcu</em>. Field specimens of <em>An. gambiae</em> Bamako were collected in Aug-Sep 2004 at four sites in Mali: Bancoumana (12°20′N, 8°20′W), Banambani (12°48′N, 08°03′W), Fanzana (13°20′N, 06°13′W) and Kela (11°88′N, 8°45′W).</p>


<h4>FISH</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Ovaries dissected from half gravid females were placed in modified Carnoy's solution (ethanol: glacial acetic acid, 3:1) for 24 h at RT. Polytene chromosome preparation and <em>in situ</em> hybridization were performed according to <a href="#pone.0000849-dellaTorre1">[43]</a>, <a href="#pone.0000849-Kumar1">[44]</a> as described previously <a href="#pone.0000849-Sharakhov1">[12]</a>. Following hybridization, chromosomes were washed with 0.2×SSC, counterstained with YOYO-1 (Sigma) and mounted in DABCO (Sigma) antifade solution. Fluorescent signals were detected using a Bio-Rad MRC-1024 Confocal Laser Scanning Imaging System running LaserSharp 3.2 software.</p>


<h4>Library Construction and Screening</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>High molecular weight genomic DNA extracted from the <em>An. gambiae</em> BKO strain (Qiagen Genomic-tip) was used to prepare plasmid (pSMART LCKan cloning vector) and fosmid (pCCFos cloning vector) libraries (Lucigen Corporation, Middleton, WI). Three genome-equivalents of the plasmid library (average insert size, 12–15 kb) were transferred to nitrocellulose supported membranes (Nitropure, GE Osmonics) and screened according to the manufacturer's recommended protocol. The probe was a 1.3 kb fragment of a BAC clone (153N7) derived from the <em>An. gambiae</em> PEST (2R+<sup>j</sup>/+<sup>j</sup>) strain by PCR using primer A (5′-GTTGACGTTTTGGGTCGTTT-3′) and primer C (5′-CTCGGTTTGGGAAAAAGTCA-3′). Labeling was by PCR in the presence of <sup>32</sup>P-dCTP. The 50 µl PCR reaction contained 20mM Tris-HCl (pH 8.4), 50 mM KCl, 1.5 mM of MgCl<sub>2</sub>, 200 µM of each dNTP excluding dCTP, 50 µCi of <sup>32</sup>P-dCTP (Amersham), 50 ng of 153N7 template, 25 pmol primers, and 1U Taq polymerase. Following an initial 4 min denaturation at 94° were 35 cycles of 94° for 1 min, 60° for 15 s, 72° for 4 min and a hold at 72° for 5 min. The probe was purified using a Centri-Sep column (Princeton Separations) as recommended by the supplier.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>The <em>An. gambiae</em> BKO fosmid library (average insert size ~40 kb) was arrayed onto 120 384-well plates, representing ~10 genome equivalents. Screening was achieved by PCR amplification of DNA pools (1-plate pools, 8-row and 12-column DNA pools per each plate). Primers were designed from the unique regions of two clones (1 and 5) isolated from the plasmid library, derived from the distal and proximal breakpoints of the 2<em>Rj</em> inversion (see <a href="#pone-0000849-g002">Figure 2</a>). Primers for the distal breakpoint were BfosF1 (5′-AAGCAAGACGCCGAGATTGC-3′) and BfosR1 (5′-TTTTCCAAACAATCGCGTGC-3′); those for the proximal breakpoint were BfosF5_2 (5′-TTCCCATTTCACCGTTGCCT-3′) and BfosR5_2 (5′-CATTATGCATTTCTGTTGCA-3′). Each 25 µl PCR reaction contained 20mM Tris-HCl (pH 8.4), 50 mM KCl, 1.5 mM MgCl<sub>2</sub>, 200 µM each dNTP, 5 pmol of each primer, 1 µl of the template DNA, and 1U of Taq polymerase. The PCR conditions were 94° for 2 min, 30 cycles of 94° for 30 s, 50° for 30 s and 72° for 30 s followed by 72° for 5 min and a 4° hold.</p>


<h4>DNA Isolation and Sequencing</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Plasmid DNA was isolated using a standard alkaline lysis protocol. Fosmid and BAC DNA was isolated using the Perfectprep Plasmid Midi Kit (Eppendorf). The sequencing strategy entailed a combination of primer walking using custom primers (Invitrogen) and the GPS-1 Genome Priming System (New England BioLabs), which places a transposon containing universal priming sites into target DNA at random locations. Sequencing reactions were performed with the Big Dye Terminator v3.1 Cycle Sequencing kit (Applied Biosystems) and analyzed on the ABI PRISM 3700 DNA Analyzer (Applied Biosystems). Individual sequences were inspected, verified and assembled into contigs using Lasergene Seqman software (DNASTAR). The sequenced regions of fosmids 35D6 and 59I22 were submitted to GenBank (accession nos. EF015880-EF015881).</p>


<h4>Analysis of Excision Events</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Genomic DNA was isolated from individual 2<em>Rj</em>/<em>j</em> mosquitoes using the DNeasy kit (Qiagen) with the slight modification of reducing elution volume to 30 µl. Prior to analysis by PCR, genomic DNA was diluted 1:8 in TE buffer (~5 ng/ul). PCR across the junctions of the inverted arrangement was attempted with a pair of distal primers (D1F, 5′-GCGTTGTCAATAATGCCTGA-3′; P3ST, 5′-CGAAGAGGAAGTCGTGCTTT-3′), and a pair of proximal primers (D2ST, 5′-GGCGGATTCTAGCAAATGTC-3′; P4U, 5′-GTTGACGTTTTGGGTCGTTT-3′). Each 50 µl PCR reaction contained 20mM Tris-HCl (pH 8.4), 50 mM KCl, 1.5 mM MgCl<sub>2</sub>, 200 µM each dNTP, 5 pmol of each primer, 1 µl of template DNA, and 1U of Taq polymerase. The PCR conditions were 94° for 2 min, 35 cycles of 94° for 30 s, 55° for 30 s and 72° for 45 s followed by 72° for 5 min and a 4° hold. PCR products were separated on 1.5% agarose gels, excised, and purified using the Pure-Link Gel Extraction Kit (Invitrogen). PCR products were sequenced on both strands using Big Dye Terminator chemistry as described above.</p>


<h4>Molecular dating of 2Rj</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Primer pairs for AGAP002110 (temp10F, 5′-CGGGAGGGTGTAATGGTATG-3′ and temp10R, <span class="gene">5′-AGAAAATTGCTCGGCTTCAC</span>) and AGAP001334 (temp54F, 5′-TTCCCATGCAATTCTGTTCA-3′ and temp54R, 5′-TTTGCCAGTTTTCCTTCACC-3′) were used in 50 µl PCR reactions containing 20mM Tris-HCl (pH 8.4), 50 mM KCl, 1.5 mM MgCl<sub>2</sub>, 200 µM each dNTP, 5 pmol of each primer, 1 µl of template DNA, and 1U of Taq polymerase. The PCR conditions were 94° for 2 min, 40 cycles of 94° for 20 s, 55° for 15 s and 72° for 20 s followed by 72° for 5 min and a 4° hold. PCR products were purified and sequenced directly as described above. Sequence polymorphism statistics in <a href="#pone-0000849-t001">Table 1</a> were calculated using DnaSP 4.0 software <a href="#pone.0000849-Rozas1">[45]</a>. Date of origin was estimated from the observed number of segregating sites within inverted and standard chromosomes as described <a href="#pone.0000849-Andolfatto1">[30]</a>.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000849"></div><a name="pone.0000849.s001" id="pone.0000849.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000849.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">CENSOR results of known transposons and retrotransposons in REPBASE to which similarity was found in the 14.6 kb Low Copy Repeat.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000849.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000849.s002" id="pone.0000849.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000849.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Polymorphic sites at AGAP002110 in <em>A. gambiae</em> field specimens homokaryotypic for 2<em>Rj</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000849.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the entomological staff of the Malaria Research and Training Center for logistical support during field collections and assistance with karyotyping. Y. Hong helped with extraction of high molecular weight DNA for library preparation, P. Romans gave helpful advice during plasmid library screening, D. Sangaré assisted with preparing plate, row and column pools of the fosmid library, and M. Unger assisted with FISH and figure preparation. Manual annotation of chromosome 2R was performed by K. Campbell through VectorBase (National Institutes of Health); J. M. C Ribeiro prepared Genscan predictions. M. Hammond provided constructive comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: NB FC MC ST. Performed the experiments: MC OG DT MF MK. Analyzed the data: NB MC NL MF. Contributed reagents/materials/analysis tools: FC NL ST. Wrote the paper: NB MC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000849-Dobzhansky1" id="pone.0000849-Dobzhansky1"></a>Dobzhansky T (1937) Genetics and the Origin of Species. New York: Columbia University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000849-Coghlan1" id="pone.0000849-Coghlan1"></a>Coghlan A, Eichler EE, Oliver SG, Paterson AH, Stein L (2005) Chromosome evolution in eukaryotes: a multi-kingdom perspective. Trends Genet  21: 673–682.  <ul class="find" data-citedArticleID="988192" data-doi="10.1016/j.tig.2005.09.009"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.09.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+evolution+in+eukaryotes%3A+a+multi-kingdom+perspective." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+evolution+in+eukaryotes%3A+a+multi-kingdom+perspective.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000849-Hoffmann1" id="pone.0000849-Hoffmann1"></a>Hoffmann AA, Sgro CM, Weeks AR (2004) Chromosomal inversion polymorphisms and adaptation. Trends Ecol Evol  19: 482–488.  <ul class="find" data-citedArticleID="988206" data-doi="10.1016/j.tree.2004.06.013"><li><a href="http://dx.doi.org/10.1016/j.tree.2004.06.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+inversion+polymorphisms+and+adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+inversion+polymorphisms+and+adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000849-Powell1" id="pone.0000849-Powell1"></a>Powell JR (1997) Progress and Prospects in Evolutionary Biology: The <em>Drosophila</em> Model. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000849-Noor1" id="pone.0000849-Noor1"></a>Noor MA, Grams KL, Bertucci LA, Reiland J (2001) Chromosomal inversions and the reproductive isolation of species. Proc Natl Acad Sci U S A  98: 12084–12088.  <ul class="find" data-citedArticleID="988228" data-doi="10.1073/pnas.221274498"><li><a href="http://dx.doi.org/10.1073/pnas.221274498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+inversions+and+the+reproductive+isolation+of+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+inversions+and+the+reproductive+isolation+of+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000849-Rieseberg1" id="pone.0000849-Rieseberg1"></a>Rieseberg LH (2001) Chromosomal rearrangements and speciation. Trends Ecol Evol  16: 351–358.  <ul class="find" data-citedArticleID="988236" data-doi="10.1016/s0169-5347(01)02187-5"><li><a href="http://dx.doi.org/10.1016/s0169-5347(01)02187-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+rearrangements+and+speciation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+rearrangements+and+speciation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000849-Gray1" id="pone.0000849-Gray1"></a>Gray YH (2000) It takes two transposons to tango: transposable-element-mediated chromosomal rearrangements. Trends Genet  16: 461–468.  <ul class="find" data-citedArticleID="988204" data-doi="10.1016/s0168-9525(00)02104-1"><li><a href="http://dx.doi.org/10.1016/s0168-9525(00)02104-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=It+takes+two+transposons+to+tango%3A+transposable-element-mediated+chromosomal+rearrangements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22It+takes+two+transposons+to+tango%3A+transposable-element-mediated+chromosomal+rearrangements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000849-Kidwell1" id="pone.0000849-Kidwell1"></a>Kidwell MG, Lisch DR (2002)  Transposable elements as sources of genomic variation. In: Craig NL, Craigie R, Gellert M, Lambowitz AM, editors. Mobile DNA II. Washington, D. C.: ASM Press.   <ul class="find-nolinks"></ul></li><li><span class="label">9.
              </span><a name="pone.0000849-Wessler1" id="pone.0000849-Wessler1"></a>Wessler SR (2006)  Eukaryotic Transposable Elements: Teaching Old Genomes New Tricks. In: Caporale L, editor. The Implicit Genome. New York: Oxford University Press. pp. 138–165.  <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0000849-Cirera1" id="pone.0000849-Cirera1"></a>Cirera S, Martin-Campos JM, Segarra C, Aguade M (1995) Molecular characterization of the breakpoints of an inversion fixed between <em>Drosophila melanogaster</em> and <em>D. subobscura</em>. Genetics  139: 321–326.  <ul class="find" data-citedArticleID="988190"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+characterization+of+the+breakpoints+of+an+inversion+fixed+between+Drosophila+melanogaster+and+D.+subobscura.&amp;auth=&amp;atitle=Molecular+characterization+of+the+breakpoints+of+an+inversion+fixed+between+Drosophila+melanogaster+and+D.+subobscura." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+the+breakpoints+of+an+inversion+fixed+between+Drosophila+melanogaster+and+D.+subobscura." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+the+breakpoints+of+an+inversion+fixed+between+Drosophila+melanogaster+and+D.+subobscura.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000849-Richards1" id="pone.0000849-Richards1"></a>Richards S, Liu Y, Bettencourt BR, Hradecky P, Letovsky S, et al.  (2005) Comparative genome sequencing of <em>Drosophila pseudoobscura</em>: chromosomal, gene, and cis-element evolution. Genome Res  15: 1–18.  <ul class="find" data-citedArticleID="988234" data-doi="10.1101/gr.3059305"><li><a href="http://dx.doi.org/10.1101/gr.3059305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+genome+sequencing+of+Drosophila+pseudoobscura%3A+chromosomal%2C+gene%2C+and+cis-element+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+genome+sequencing+of+Drosophila+pseudoobscura%3A+chromosomal%2C+gene%2C+and+cis-element+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000849-Sharakhov1" id="pone.0000849-Sharakhov1"></a>Sharakhov IV, White BJ, Sharakhova MV, Kayondo J, Lobo NF, et al.  (2006) Breakpoint structure reveals the unique origin of an interspecific chromosomal inversion (2La) in the <em>Anopheles gambiae</em> complex. Proc Natl Acad Sci U S A  103: 6258–6262.  <ul class="find" data-citedArticleID="988242" data-doi="10.1073/pnas.0509683103"><li><a href="http://dx.doi.org/10.1073/pnas.0509683103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breakpoint+structure+reveals+the+unique+origin+of+an+interspecific+chromosomal+inversion+%282La%29+in+the+Anopheles+gambiae+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breakpoint+structure+reveals+the+unique+origin+of+an+interspecific+chromosomal+inversion+%282La%29+in+the+Anopheles+gambiae+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000849-Wesley1" id="pone.0000849-Wesley1"></a>Wesley CS, Eanes WF (1994) Isolation and analysis of the breakpoint sequences of chromosome inversion In(3L)Payne in <em>Drosophila melanogaster</em>. Proc Natl Acad Sci U S A  91: 3132–3136.  <ul class="find" data-citedArticleID="988258" data-doi="10.1073/pnas.91.8.3132"><li><a href="http://dx.doi.org/10.1073/pnas.91.8.3132" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+analysis+of+the+breakpoint+sequences+of+chromosome+inversion+In%283L%29Payne+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+analysis+of+the+breakpoint+sequences+of+chromosome+inversion+In%283L%29Payne+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000849-Matzkin1" id="pone.0000849-Matzkin1"></a>Matzkin LM, Merritt T, Zhu CT, Eanes WF (2005) The structure and population genetics of the breakpoints associated with the cosmopolitan chromosomal inversion In(3R)Payne in <em>Drosophila melanogaster</em>. Genetics  170: 1143–1152.  <ul class="find" data-citedArticleID="988224" data-doi="10.1534/genetics.104.038810"><li><a href="http://dx.doi.org/10.1534/genetics.104.038810" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structure+and+population+genetics+of+the+breakpoints+associated+with+the+cosmopolitan+chromosomal+inversion+In%283R%29Payne+in+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structure+and+population+genetics+of+the+breakpoints+associated+with+the+cosmopolitan+chromosomal+inversion+In%283R%29Payne+in+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000849-Bailey1" id="pone.0000849-Bailey1"></a>Bailey JA, Yavor AM, Massa HF, Trask BJ, Eichler EE (2001) Segmental duplications: organization and impact within the current human genome project assembly. Genome Res  11: 1005–1017.  <ul class="find" data-citedArticleID="988182" data-doi="10.1101/gr.gr-1871r"><li><a href="http://dx.doi.org/10.1101/gr.gr-1871r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Segmental+duplications%3A+organization+and+impact+within+the+current+human+genome+project+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Segmental+duplications%3A+organization+and+impact+within+the+current+human+genome+project+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000849-Wang1" id="pone.0000849-Wang1"></a>Wang Y, Leung FC (2006) Long inverted repeats in eukaryotic genomes: recombinogenic motifs determine genomic plasticity. FEBS Lett  580: 1277–1284.  <ul class="find" data-citedArticleID="988254" data-doi="10.1016/j.febslet.2006.01.045"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.01.045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Long+inverted+repeats+in+eukaryotic+genomes%3A+recombinogenic+motifs+determine+genomic+plasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Long+inverted+repeats+in+eukaryotic+genomes%3A+recombinogenic+motifs+determine+genomic+plasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000849-Warburton1" id="pone.0000849-Warburton1"></a>Warburton PE, Giordano J, Cheung F, Gelfand Y, Benson G (2004) Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes. Genome Res  14: 1861–1869.  <ul class="find" data-citedArticleID="988256" data-doi="10.1101/gr.2542904"><li><a href="http://dx.doi.org/10.1101/gr.2542904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inverted+repeat+structure+of+the+human+genome%3A+the+X-chromosome+contains+a+preponderance+of+large%2C+highly+homologous+inverted+repeats+that+contain+testes+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inverted+repeat+structure+of+the+human+genome%3A+the+X-chromosome+contains+a+preponderance+of+large%2C+highly+homologous+inverted+repeats+that+contain+testes+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000849-Achaz1" id="pone.0000849-Achaz1"></a>Achaz G, Netter P, Coissac E (2001) Study of intrachromosomal duplications among the eukaryote genomes. Mol Biol Evol  18: 2280–2288.  <ul class="find" data-citedArticleID="988174" data-doi="10.1093/oxfordjournals.molbev.a003774"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Study+of+intrachromosomal+duplications+among+the+eukaryote+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Study+of+intrachromosomal+duplications+among+the+eukaryote+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000849-Samonte1" id="pone.0000849-Samonte1"></a>Samonte RV, Eichler EE (2002) Segmental duplications and the evolution of the primate genome. Nat Rev Genet  3: 65–72.  <ul class="find" data-citedArticleID="988240" data-doi="10.1038/nrg705"><li><a href="http://dx.doi.org/10.1038/nrg705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Segmental+duplications+and+the+evolution+of+the+primate+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Segmental+duplications+and+the+evolution+of+the+primate+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000849-Stankiewicz1" id="pone.0000849-Stankiewicz1"></a>Stankiewicz P, Lupski JR (2002) Molecular-evolutionary mechanisms for genomic disorders. Curr Opin Genet Dev  12: 312–319.  <ul class="find" data-citedArticleID="988248" data-doi="10.1016/s0959-437x(02)00304-0"><li><a href="http://dx.doi.org/10.1016/s0959-437x(02)00304-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular-evolutionary+mechanisms+for+genomic+disorders." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular-evolutionary+mechanisms+for+genomic+disorders.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000849-Coluzzi1" id="pone.0000849-Coluzzi1"></a>Coluzzi M, Sabatini A, Della Torre A, Di Deco MA, Petrarca V (2002) A polytene chromosome analysis of the <em>Anopheles gambiae</em> species complex. Science  298: 1415–1418.  <ul class="find" data-citedArticleID="988194" data-doi="10.1126/science.1077769"><li><a href="http://dx.doi.org/10.1126/science.1077769" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+polytene+chromosome+analysis+of+the+Anopheles+gambiae+species+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+polytene+chromosome+analysis+of+the+Anopheles+gambiae+species+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000849-Coluzzi2" id="pone.0000849-Coluzzi2"></a>Coluzzi M, Petrarca V, DiDeco MA (1985) Chromosomal inversion intergradation and incipient speciation in <em>Anopheles gambiae</em>. Boll Zool  52: 45–63.  <ul class="find" data-citedArticleID="988196" data-doi="10.1080/11250008509440343"><li><a href="http://dx.doi.org/10.1080/11250008509440343" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosomal+inversion+intergradation+and+incipient+speciation+in+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosomal+inversion+intergradation+and+incipient+speciation+in+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000849-Toure1" id="pone.0000849-Toure1"></a>Toure YT, Petrarca V, Traore SF, Coulibaly A, Maiga HM, et al.  (1998) The distribution and inversion polymorphism of chromosomally recognized taxa of the <em>Anopheles gambiae</em> complex in Mali, West Africa. Parassitologia  40: 477–511.  <ul class="find" data-citedArticleID="988252"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa.&amp;auth=&amp;atitle=The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+distribution+and+inversion+polymorphism+of+chromosomally+recognized+taxa+of+the+Anopheles+gambiae+complex+in+Mali%2C+West+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000849-DellaTorre1" id="pone.0000849-DellaTorre1"></a>Della Torre A, Tu Z, Petrarca V (2005) On the distribution and genetic differentiation of <em>Anopheles gambiae s.s.</em> molecular forms. Insect Biochem Mol Biol  35: 755–769.  <ul class="find" data-citedArticleID="988198" data-doi="10.1016/j.ibmb.2005.02.006"><li><a href="http://dx.doi.org/10.1016/j.ibmb.2005.02.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+distribution+and+genetic+differentiation+of+Anopheles+gambiae+s.s.+molecular+forms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+distribution+and+genetic+differentiation+of+Anopheles+gambiae+s.s.+molecular+forms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000849-Mathiopoulos1" id="pone.0000849-Mathiopoulos1"></a>Mathiopoulos KD, della Torre A, Predazzi V, Petrarca V, Coluzzi M (1998) Cloning of inversion breakpoints in the <em>Anopheles gambiae</em> complex traces a transposable element at the inversion junction. Proc Natl Acad Sci U S A  95: 12444–12449.  <ul class="find" data-citedArticleID="988222" data-doi="10.1073/pnas.95.21.12444"><li><a href="http://dx.doi.org/10.1073/pnas.95.21.12444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+inversion+breakpoints+in+the+Anopheles+gambiae+complex+traces+a+transposable+element+at+the+inversion+junction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+inversion+breakpoints+in+the+Anopheles+gambiae+complex+traces+a+transposable+element+at+the+inversion+junction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000849-Mukabayire1" id="pone.0000849-Mukabayire1"></a>Mukabayire O, Besansky NJ (1996) Distribution of T1, Q, Pegasus and mariner transposable elements on the polytene chromosomes of PEST, a standard strain of <em>Anopheles gambiae</em>. Chromosoma  104: 585–595.  <ul class="find" data-citedArticleID="988226" data-doi="10.1007/bf00352298"><li><a href="http://dx.doi.org/10.1007/bf00352298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+T1%2C+Q%2C+Pegasus+and+mariner+transposable+elements+on+the+polytene+chromosomes+of+PEST%2C+a+standard+strain+of+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+T1%2C+Q%2C+Pegasus+and+mariner+transposable+elements+on+the+polytene+chromosomes+of+PEST%2C+a+standard+strain+of+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000849-Hong1" id="pone.0000849-Hong1"></a>Hong YS, Hogan JR, Wang X, Sarkar A, Sim C, et al.  (2003) Construction of a BAC library and generation of BAC end sequence-tagged connectors for genome sequencing of the African malaria mosquito <em>Anopheles gambiae</em>. Mol Genet Genomics  268: 720–728.  <ul class="find" data-citedArticleID="988208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Construction+of+a+BAC+library+and+generation+of+BAC+end+sequence-tagged+connectors+for+genome+sequencing+of+the+African+malaria+mosquito+Anopheles+gambiae.&amp;auth=&amp;atitle=Construction+of+a+BAC+library+and+generation+of+BAC+end+sequence-tagged+connectors+for+genome+sequencing+of+the+African+malaria+mosquito+Anopheles+gambiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Construction+of+a+BAC+library+and+generation+of+BAC+end+sequence-tagged+connectors+for+genome+sequencing+of+the+African+malaria+mosquito+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Construction+of+a+BAC+library+and+generation+of+BAC+end+sequence-tagged+connectors+for+genome+sequencing+of+the+African+malaria+mosquito+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000849-Kohany1" id="pone.0000849-Kohany1"></a>Kohany O, Gentles AJ, Hankus L, Jurka J (2006) Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor. BMC Bioinformatics  7: 474.  <ul class="find" data-citedArticleID="988214" data-doi="10.1186/1471-2105-7-474"><li><a href="http://dx.doi.org/10.1186/1471-2105-7-474" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Annotation%2C+submission+and+screening+of+repetitive+elements+in+Repbase%3A+RepbaseSubmitter+and+Censor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Annotation%2C+submission+and+screening+of+repetitive+elements+in+Repbase%3A+RepbaseSubmitter+and+Censor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000849-Smit1" id="pone.0000849-Smit1"></a>Smit AFA, Hubley R, Green P (1996–2004) RepeatMasker Open-3.0.   <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0000849-Andolfatto1" id="pone.0000849-Andolfatto1"></a>Andolfatto P, Wall JD, Kreitman M (1999) Unusual haplotype structure at the proximal breakpoint of In(2L)t in a natural population of <em>Drosophila melanogaster</em>. Genetics  153: 1297–1311.  <ul class="find" data-citedArticleID="988176"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Unusual+haplotype+structure+at+the+proximal+breakpoint+of+In%282L%29t+in+a+natural+population+of+Drosophila+melanogaster.&amp;auth=&amp;atitle=Unusual+haplotype+structure+at+the+proximal+breakpoint+of+In%282L%29t+in+a+natural+population+of+Drosophila+melanogaster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unusual+haplotype+structure+at+the+proximal+breakpoint+of+In%282L%29t+in+a+natural+population+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unusual+haplotype+structure+at+the+proximal+breakpoint+of+In%282L%29t+in+a+natural+population+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000849-Andolfatto2" id="pone.0000849-Andolfatto2"></a>Andolfatto P, Depaulis F, Navarro A (2001) Inversion polymorphisms and nucleotide variability in Drosophila. Genet Res  77: 1–8.  <ul class="find" data-citedArticleID="988178" data-doi="10.1017/s0016672301004955"><li><a href="http://dx.doi.org/10.1017/s0016672301004955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inversion+polymorphisms+and+nucleotide+variability+in+Drosophila." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inversion+polymorphisms+and+nucleotide+variability+in+Drosophila.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000849-Potter1" id="pone.0000849-Potter1"></a>Potter S, Truett M, Phillips M, Maher A (1980) Eucaryotic transposable genetic elements with inverted terminal repeats. Cell  20: 639–647.  <ul class="find" data-citedArticleID="988230" data-doi="10.1016/0092-8674(80)90310-4"><li><a href="http://dx.doi.org/10.1016/0092-8674(80)90310-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eucaryotic+transposable+genetic+elements+with+inverted+terminal+repeats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eucaryotic+transposable+genetic+elements+with+inverted+terminal+repeats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000849-Bingham1" id="pone.0000849-Bingham1"></a>Bingham PM, Zachar Z (1989)  Retrotransposons and the FB transposon from <em>Drosophila melanogaster</em>. In: Berg DE, Howe MM, editors. Mobile DNA. Washington, D. C.: American Society for Microbiology. pp. 485–502.  <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0000849-Sharp1" id="pone.0000849-Sharp1"></a>Sharp AJ, Cheng Z, Eichler EE (2006) Structural variation of the human genome. Annu Rev Genomics Hum Genet  7: 407–442.  <ul class="find" data-citedArticleID="988244" data-doi="10.1146/annurev.genom.7.080505.115618"><li><a href="http://dx.doi.org/10.1146/annurev.genom.7.080505.115618" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+variation+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+variation+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000849-HuaVan1" id="pone.0000849-HuaVan1"></a>Hua-Van A, Langin T, Daboussi MJ (2002) Aberrant transposition of a Tc1-mariner element, <em>impala</em>, in the fungus <em>Fusarium oxysporum</em>. Mol Genet Genomics  267: 79–87.  <ul class="find" data-citedArticleID="988210" data-doi="10.1007/s00438-002-0638-9"><li><a href="http://dx.doi.org/10.1007/s00438-002-0638-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aberrant+transposition+of+a+Tc1-mariner+element%2C+impala%2C+in+the+fungus+Fusarium+oxysporum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aberrant+transposition+of+a+Tc1-mariner+element%2C+impala%2C+in+the+fungus+Fusarium+oxysporum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000849-Caceres1" id="pone.0000849-Caceres1"></a>Caceres M, Ranz JM, Barbadilla A, Long M, Ruiz A (1999) Generation of a widespread Drosophila inversion by a transposable element. Science  285: 415–418.  <ul class="find" data-citedArticleID="988186" data-doi="10.1126/science.285.5426.415"><li><a href="http://dx.doi.org/10.1126/science.285.5426.415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+a+widespread+Drosophila+inversion+by+a+transposable+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+a+widespread+Drosophila+inversion+by+a+transposable+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000849-Krzywinski1" id="pone.0000849-Krzywinski1"></a>Krzywinski J, Nusskern D, Kern M, Besansky NJ (2004) Isolation and characterization of Y chromosome sequences from the African malaria mosquito <em>Anopheles gambiae</em>. Genetics  166: 1291–1302.  <ul class="find" data-citedArticleID="988216" data-doi="10.1534/genetics.166.3.1291"><li><a href="http://dx.doi.org/10.1534/genetics.166.3.1291" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+and+characterization+of+Y+chromosome+sequences+from+the+African+malaria+mosquito+Anopheles+gambiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+and+characterization+of+Y+chromosome+sequences+from+the+African+malaria+mosquito+Anopheles+gambiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000849-Steinemann1" id="pone.0000849-Steinemann1"></a>Steinemann M, Steinemann S (1992) Degenerating Y chromosome of Drosophila miranda: a trap for retrotransposons. Proc Natl Acad Sci U S A  89: 7591–7595.  <ul class="find" data-citedArticleID="988250" data-doi="10.1073/pnas.89.16.7591"><li><a href="http://dx.doi.org/10.1073/pnas.89.16.7591" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Degenerating+Y+chromosome+of+Drosophila+miranda%3A+a+trap+for+retrotransposons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Degenerating+Y+chromosome+of+Drosophila+miranda%3A+a+trap+for+retrotransposons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000849-Lobachev1" id="pone.0000849-Lobachev1"></a>Lobachev KS, Shor BM, Tran HT, Taylor W, Keen JD, et al.  (1998) Factors affecting inverted repeat stimulation of recombination and deletion in <em>Saccharomyces cerevisiae</em>. Genetics  148: 1507–1524.  <ul class="find" data-citedArticleID="988220"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Factors+affecting+inverted+repeat+stimulation+of+recombination+and+deletion+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000849-Egner1" id="pone.0000849-Egner1"></a>Egner C, Berg DE (1981) Excision of transposon Tn5 is dependent on the inverted repeats but not on the transposase function of Tn5. Proc Natl Acad Sci U S A  78: 459–463.  <ul class="find" data-citedArticleID="988202" data-doi="10.1073/pnas.78.1.459"><li><a href="http://dx.doi.org/10.1073/pnas.78.1.459" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Excision+of+transposon+Tn5+is+dependent+on+the+inverted+repeats+but+not+on+the+transposase+function+of+Tn5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Excision+of+transposon+Tn5+is+dependent+on+the+inverted+repeats+but+not+on+the+transposase+function+of+Tn5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000849-Ayala1" id="pone.0000849-Ayala1"></a>Ayala FJ, Coluzzi M (2005) Chromosome speciation: Humans, Drosophila, and mosquitoes. Proc Natl Acad Sci U S A 102 Suppl  1: 6535–6542.  <ul class="find" data-citedArticleID="988180" data-doi="10.1073/pnas.0501847102"><li><a href="http://dx.doi.org/10.1073/pnas.0501847102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+speciation%3A+Humans%2C+Drosophila%2C+and+mosquitoes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+speciation%3A+Humans%2C+Drosophila%2C+and+mosquitoes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000849-Caceres2" id="pone.0000849-Caceres2"></a>Caceres M, Puig M, Ruiz A (2001) Molecular characterization of two natural hotspots in the <em>Drosophila buzzatii</em> genome induced by transposon insertions. Genome Res  11: 1353–1364.  <ul class="find" data-citedArticleID="988188" data-doi="10.1101/gr.174001"><li><a href="http://dx.doi.org/10.1101/gr.174001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+of+two+natural+hotspots+in+the+Drosophila+buzzatii+genome+induced+by+transposon+insertions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+of+two+natural+hotspots+in+the+Drosophila+buzzatii+genome+induced+by+transposon+insertions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000849-dellaTorre1" id="pone.0000849-dellaTorre1"></a>della Torre A (1997)  Polytene chromosome preparation from anopheline mosquitoes. In: Crampton JM, Beard CB, Louis C, editors. Molecular Biology of Disease Vectors: A Methods Manual. London: Chapman &amp; Hall. pp. 329–336.  <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0000849-Kumar1" id="pone.0000849-Kumar1"></a>Kumar V, Cornel AJ, Mukabayire O (1997)  <em>In situ</em> hybridization to <em>Anopheles</em> polytene chromosomes. In: Crampton JM, Beard CB, Louis K, editors. Molecular Biology of Insect Disease Vectors: A Methods Manual. London: Chapman &amp; Hall. pp. 337–345.  <ul class="find-nolinks"></ul></li><li><span class="label">45.
              </span><a name="pone.0000849-Rozas1" id="pone.0000849-Rozas1"></a>Rozas J, Sanchez-DelBarrio JC, Messeguer X, Rozas R (2003) DnaSP, DNA polymorphism analyses by the coalescent and other methods. Bioinformatics  19: 2496–2497.  <ul class="find" data-citedArticleID="988238" data-doi="10.1093/bioinformatics/btg359"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DnaSP%2C+DNA+polymorphism+analyses+by+the+coalescent+and+other+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.XML" value="98928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.PDF" value="285055"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g001.PNG_L" value="6487148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g001.PNG_M" value="485333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g001.PNG_S" value="21151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g001.TIF" value="8333032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g001.PNG_I" value="142558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g002.PNG_L" value="1258834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g002.PNG_M" value="175726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g002.PNG_S" value="12904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g002.TIF" value="1804074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g002.PNG_I" value="60678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g003.PNG_L" value="199568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g003.PNG_M" value="27737"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g003.PNG_S" value="2740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g003.TIF" value="376222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g003.PNG_I" value="10759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g004.PNG_L" value="3880777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g004.PNG_M" value="443805"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g004.PNG_S" value="12637"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g004.TIF" value="5352784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g004.PNG_I" value="176031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g005.PNG_L" value="97886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g005.PNG_M" value="16022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g005.PNG_S" value="2105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g005.TIF" value="210306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.g005.PNG_I" value="6993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.t001.PNG_L" value="96880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.t001.PNG_M" value="130380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.t001.PNG_S" value="17127"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.t001.TIF" value="399004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.t001.PNG_I" value="45164"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.s001.DOC" value="43520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000849.s002.DOC" value="27136"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000849&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000849" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000849&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000849&volume=&issue=&title=Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae&author_name=Mamadou%20B.%20Coulibaly%2C%20Neil%20F.%20Lobo%2C%20Meagan%20C.%20Fitzpatrick%2C%20Marcia%20Kern%2C%20Olga%20Grushko%2C%20Daniel%20V.%20Thaner%2C%20S%C3%A9kou%20F.%20Traor%C3%A9%2C%20Frank%20H.%20Collins%2C%20Nora%20J.%20Besansky&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000849" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849&amp;t=Segmental%20Duplication%20Implicated%20in%20the%20Genesis%20of%20Inversion%202Rj%20of%20Anopheles%20gambiae" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849&title=Segmental%20Duplication%20Implicated%20in%20the%20Genesis%20of%20Inversion%202Rj%20of%20Anopheles%20gambiae&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849&amp;title=Segmental%20Duplication%20Implicated%20in%20the%20Genesis%20of%20Inversion%202Rj%20of%20Anopheles%20gambiae" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000849&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Segmental Duplication Implicated in the Genesis of Inversion 2Rj of Anopheles gambiae';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000849';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Segmental%20Duplication%20Implicated%20in%20the%20Genesis%20of%20Inversion%202Rj%20of%20Anopheles%20gambiae http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000849" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000849" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosome+structure+and+function%22" title="Search for articles in the subject area:'Chromosome structure and function'"><div class="flagText">Chromosome structure and function</div></a>
              <div data-categoryid="18739" data-articleid="24536"
                   data-categoryname="Chromosome structure and function"
                   class="flagImage" title="Flag 'Chromosome structure and function' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="24536"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="18229" data-articleid="24536"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24536"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Inversions%22" title="Search for articles in the subject area:'Inversions'"><div class="flagText">Inversions</div></a>
              <div data-categoryid="43105" data-articleid="24536"
                   data-categoryname="Inversions"
                   class="flagImage" title="Flag 'Inversions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Invertebrate+genomics%22" title="Search for articles in the subject area:'Invertebrate genomics'"><div class="flagText">Invertebrate genomics</div></a>
              <div data-categoryid="17221" data-articleid="24536"
                   data-categoryname="Invertebrate genomics"
                   class="flagImage" title="Flag 'Invertebrate genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="24536"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transposable+elements%22" title="Search for articles in the subject area:'Transposable elements'"><div class="flagText">Transposable elements</div></a>
              <div data-categoryid="17435" data-articleid="24536"
                   data-categoryname="Transposable elements"
                   class="flagImage" title="Flag 'Transposable elements' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5300'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8280'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3939&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
