

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000814"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000814" />

    <meta name="citation_title" content="Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase"/>
    <meta itemprop="name" content="Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase"/>

      <meta name="citation_author" content="Qi Wang"/>
            <meta name="citation_author_institution" content="Center for Computational Biology, Molecular Biology Institute, Institute for Genomics and Proteomics, University of California at Los Angeles, Los Angeles, United States of America"/>
      <meta name="citation_author" content="Christopher Lee"/>
            <meta name="citation_author_institution" content="Center for Computational Biology, Molecular Biology Institute, Institute for Genomics and Proteomics, University of California at Los Angeles, Los Angeles, United States of America"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of California at Los Angeles, Los Angeles, United States of America"/>

    <meta name="citation_date" content="2007/8/29"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000814.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e814"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Correlation of co-ordinated amino acid substitutions with function in viruses related to tobacco mosaic virus.; citation_author=D Altschuh; citation_author=AM Lesk; citation_author=AC Bloomer; citation_author=A Klug; citation_journal_title=J Mol Biol; citation_volume=193; citation_number=1; citation_pages=693-707; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Correlated mutations and residue contacts in proteins.; citation_author=U Gobel; citation_author=C Sander; citation_author=R Schneider; citation_author=A Valencia; citation_journal_title=Proteins; citation_volume=18; citation_number=2; citation_pages=309-317; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Can three-dimensional contacts in protein structures be predicted by analysis of correlated mutations?; citation_author=IN Shindyalov; citation_author=NA Kolchanov; citation_author=C Sander; citation_journal_title=Protein Eng; citation_volume=7; citation_number=3; citation_pages=349-358; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The prediction of protein contacts from multiple sequence alignments.; citation_author=DJ Thomas; citation_author=G Casari; citation_author=C Sander; citation_journal_title=Protein Eng; citation_volume=9; citation_number=4; citation_pages=941-948; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Improving contact predictions by the combination of correlated mutations and other sources of sequence information.; citation_author=O Olmea; citation_author=A Valencia; citation_journal_title=Fold Des; citation_volume=2; citation_number=5; citation_pages=S25-32; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Effective use of sequence correlation and conservation in fold recognition.; citation_author=O Olmea; citation_author=B Rost; citation_author=A Valencia; citation_journal_title=J Mol Biol; citation_volume=293; citation_number=6; citation_pages=1221-1239; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Analysis of covariation in an SH3 domain sequence alignment: applications in tertiary contact prediction and the design of compensating hydrophobic core substitutions.; citation_author=SM Larson; citation_author=AA Di Nardo; citation_author=AR Davidson; citation_journal_title=J Mol Biol; citation_volume=303; citation_number=7; citation_pages=433-446; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Progress in predicting inter-residue contacts of proteins with neural networks and correlated mutations.; citation_author=P Fariselli; citation_author=O Olmea; citation_author=A Valencia; citation_author=R Casadio; citation_journal_title=Proteins Suppl; citation_volume=5; citation_number=8; citation_pages=157-162; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Influence of conservation on calculations of amino acid covariance in multiple sequence alignments.; citation_author=AA Fodor; citation_author=RW Aldrich; citation_journal_title=Proteins; citation_volume=56; citation_number=9; citation_pages=211-221; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved networks of residues mediate allosteric communication in proteins.; citation_author=GM Suel; citation_author=SW Lockless; citation_author=MA Wall; citation_author=R Ranganathan; citation_journal_title=Nat Struct Biol; citation_volume=10; citation_number=10; citation_pages=59-69; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolutionarily conserved pathways of energetic connectivity in protein families.; citation_author=SW Lockless; citation_author=R Ranganathan; citation_journal_title=Science; citation_volume=286; citation_number=11; citation_pages=295-299; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Allosteric determinants in guanine nucleotide-binding proteins.; citation_author=ME Hatley; citation_author=SW Lockless; citation_author=SK Gibson; citation_author=AG Gilman; citation_author=R Ranganathan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=12; citation_pages=14445-14450; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Covariation of mutations in the V3 loop of human immunodeficiency virus type 1 envelope protein: an information theoretic analysis.; citation_author=BT Korber; citation_author=RM Farber; citation_author=DH Wolpert; citation_author=AS Lapedes; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=13; citation_pages=7176-7180; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Covariability of V3 loop amino acids.; citation_author=PJ Bickel; citation_author=PC Cosman; citation_author=RA Olshen; citation_author=PC Spector; citation_author=AG Rodrigo; citation_journal_title=AIDS Res Hum Retroviruses; citation_volume=12; citation_number=14; citation_pages=1401-1411; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Extended spectrum of HIV-1 reverse transcriptase mutations in patients receiving multiple nucleoside analog inhibitors.; citation_author=MJ Gonzales; citation_author=TD Wu; citation_author=J Taylor; citation_author=I Belitskaya; citation_author=R Kantor; citation_journal_title=Aids; citation_volume=17; citation_number=15; citation_pages=791-799; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Covariability of selected amino acid positions for HIV type 1 subtypes C and B.; citation_author=PB Gilbert; citation_author=V Novitsky; citation_author=M Essex; citation_journal_title=AIDS Res Hum Retroviruses; citation_volume=21; citation_number=16; citation_pages=1016-1030; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Covariation of amino acid positions in HIV-1 protease.; citation_author=NG Hoffman; citation_author=CA Schiffer; citation_author=R Swanstrom; citation_journal_title=Virology; citation_volume=314; citation_number=17; citation_pages=536-548; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mutation patterns and structural correlates in human immunodeficiency virus type 1 protease following different protease inhibitor treatments.; citation_author=TD Wu; citation_author=CA Schiffer; citation_author=MJ Gonzales; citation_author=J Taylor; citation_author=R Kantor; citation_journal_title=J Virol; citation_volume=77; citation_number=18; citation_pages=4836-4847; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Novel human immunodeficiency virus type 1 protease mutations potentially involved in resistance to protease inhibitors.; citation_author=V Svicher; citation_author=F Ceccherini-Silberstein; citation_author=F Erba; citation_author=M Santoro; citation_author=C Gori; citation_journal_title=Antimicrob Agents Chemother; citation_volume=49; citation_number=19; citation_pages=2015-2025; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The genome-wide distribution of background linkage disequilibrium in a population isolate.; citation_author=SK Service; citation_author=RA Ophoff; citation_author=NB Freimer; citation_journal_title=Hum Mol Genet; citation_volume=10; citation_number=20; citation_pages=545-551; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Accurate reconstruction of a known HIV-1 transmission history by phylogenetic tree analysis.; citation_author=T Leitner; citation_author=D Escanilla; citation_author=C Franzen; citation_author=M Uhlen; citation_author=J Albert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=21; citation_pages=10864-10869; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Molecular epidemiology of HIV-1 variants in the global AIDS pandemic: an update.; citation_author=MM Thomson; citation_author=R Najera; citation_journal_title=AIDS Rev; citation_volume=7; citation_number=22; citation_pages=210-224; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lower in vivo mutation rate of human immunodeficiency virus type 1 than that predicted from the fidelity of purified reverse transcriptase.; citation_author=LM Mansky; citation_author=HM Temin; citation_journal_title=J Virol; citation_volume=69; citation_number=23; citation_pages=5087-5094; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Limitations of a molecular clock applied to considerations of the origin of HIV-1.; citation_author=B Korber; citation_author=J Theiler; citation_author=S Wolinsky; citation_journal_title=Science; citation_volume=280; citation_number=24; citation_pages=1868-1871; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=High rate of recombination throughout the human immunodeficiency virus type 1 genome.; citation_author=AE Jetzt; citation_author=H Yu; citation_author=GJ Klarmann; citation_author=Y Ron; citation_author=BD Preston; citation_journal_title=J Virol; citation_volume=74; citation_number=25; citation_pages=1234-1240; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus type 1 recombination: rate, fidelity, and putative hot spots.; citation_author=J Zhuang; citation_author=AE Jetzt; citation_author=G Sun; citation_author=H Yu; citation_author=G Klarmann; citation_journal_title=J Virol; citation_volume=76; citation_number=26; citation_pages=11273-11282; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=High rates of human immunodeficiency virus type 1 recombination: near-random segregation of markers one kilobase apart in one round of viral replication.; citation_author=T Rhodes; citation_author=H Wargo; citation_author=WS Hu; citation_journal_title=J Virol; citation_volume=77; citation_number=27; citation_pages=11193-11200; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Viral dynamics in human immunodeficiency virus type 1 infection.; citation_author=X Wei; citation_author=SK Ghosh; citation_author=ME Taylor; citation_author=VA Johnson; citation_author=EA Emini; citation_journal_title=Nature; citation_volume=373; citation_number=28; citation_pages=117-122; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Rapid turnover of plasma virions and CD4 lymphocytes in HIV-1 infection.; citation_author=DD Ho; citation_author=AU Neumann; citation_author=AS Perelson; citation_author=W Chen; citation_author=JM Leonard; citation_journal_title=Nature; citation_volume=373; citation_number=29; citation_pages=123-126; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=HIV-1 dynamics in vivo: virion clearance rate, infected cell life-span, and viral generation time.; citation_author=AS Perelson; citation_author=AU Neumann; citation_author=M Markowitz; citation_author=JM Leonard; citation_author=DD Ho; citation_journal_title=Science; citation_volume=271; citation_number=30; citation_pages=1582-1586; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Convergent evolution: the need to be explicit.; citation_author=RF Doolittle; citation_journal_title=Trends Biochem Sci; citation_volume=19; citation_number=31; citation_pages=15-18; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Cases in Which Parsimony or Compatibility Methods Will Be Positively Misleading.; citation_author=J Felsenstein; citation_journal_title=Systematic Zoology; citation_volume=27; citation_number=32; citation_pages=401-410; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Molecular footprint of drug-selective pressure in a human immunodeficiency virus transmission chain.; citation_author=P Lemey; citation_author=I Derdelinckx; citation_author=A Rambaut; citation_author=K Van Laethem; citation_author=S Dumont; citation_journal_title=J Virol; citation_volume=79; citation_number=33; citation_pages=11981-11989; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Separation of phylogenetic and functional associations in biological sequences by using the parametric bootstrap.; citation_author=KR Wollenberg; citation_author=WR Atchley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=34; citation_pages=3288-3291; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Detection and reduction of evolutionary noise in correlated mutation analysis.; citation_author=O Noivirt; citation_author=M Eisenstein; citation_author=A Horovitz; citation_journal_title=Protein Eng Des Sel; citation_volume=18; citation_number=35; citation_pages=247-253; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Automatic detection of conserved base pairing patterns in RNA virus genomes.; citation_author=IL Hofacker; citation_author=PF Stadler; citation_journal_title=Comput Chem; citation_volume=23; citation_number=36; citation_pages=401-414; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A phylogenetic method for detecting positive epistasis in gene sequences and its application to RNA virus evolution.; citation_author=B Shapiro; citation_author=A Rambaut; citation_author=OG Pybus; citation_author=EC Holmes; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=37; citation_pages=1724-1730; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Positive selection detection in 40,000 human immunodeficiency virus (HIV) type 1 sequences automatically identifies drug resistance and positive fitness mutations in HIV protease and reverse transcriptase.; citation_author=L Chen; citation_author=A Perlina; citation_author=CJ Lee; citation_journal_title=J Virol; citation_volume=78; citation_number=38; citation_pages=3722-3732; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=39; citation_pages=1299-1320; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A tutorial on statistical methods for population association studies.; citation_author=DJ Balding; citation_journal_title=Nat Rev Genet; citation_volume=7; citation_number=40; citation_pages=781-791; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium in humans: models and data.; citation_author=JK Pritchard; citation_author=M Przeworski; citation_journal_title=Am J Hum Genet; citation_volume=69; citation_number=41; citation_pages=1-14; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium in the human genome.; citation_author=DE Reich; citation_author=M Cargill; citation_author=S Bolk; citation_author=J Ireland; citation_author=PC Sabeti; citation_journal_title=Nature; citation_volume=411; citation_number=42; citation_pages=199-204; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A first-generation linkage disequilibrium map of human chromosome 22.; citation_author=E Dawson; citation_author=GR Abecasis; citation_author=S Bumpstead; citation_author=Y Chen; citation_author=S Hunt; citation_journal_title=Nature; citation_volume=418; citation_number=43; citation_pages=544-548; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Haplotype variation and linkage disequilibrium in 313 human genes.; citation_author=JC Stephens; citation_author=JA Schneider; citation_author=DA Tanguay; citation_author=J Choi; citation_author=T Acharya; citation_journal_title=Science; citation_volume=293; citation_number=44; citation_pages=489-493; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genotypic testing for human immunodeficiency virus type 1 drug resistance.; citation_author=RW Shafer; citation_journal_title=Clin Microbiol Rev; citation_volume=15; citation_number=45; citation_pages=247-277; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HIV drug resistance.; citation_author=F Clavel; citation_author=AJ Hance; citation_journal_title=N Engl J Med; citation_volume=350; citation_number=46; citation_pages=1023-1035; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hitch-Hiking Effect of a Favorable Gene.; citation_author=JM Smith; citation_author=J Haigh; citation_journal_title=Genetical Research; citation_volume=23; citation_number=47; citation_pages=23-35; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Genomic scans for selective sweeps using SNP data.; citation_author=R Nielsen; citation_author=S Williamson; citation_author=Y Kim; citation_author=MJ Hubisz; citation_author=AG Clark; citation_journal_title=Genome Research; citation_volume=15; citation_number=48; citation_pages=1566-1575; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Update of the drug resistance mutations in HIV-1: Fall 2006.; citation_author=VA Johnson; citation_author=F Brun-Vezinet; citation_author=B Clotet; citation_author=DR Kuritzkes; citation_author=D Pillay; citation_journal_title=Top HIV Med; citation_volume=14; citation_number=49; citation_pages=125-130; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evidence for positive epistasis in HIV-1.; citation_author=S Bonhoeffer; citation_author=C Chappey; citation_author=NT Parkin; citation_author=JM Whitcomb; citation_author=CJ Petropoulos; citation_journal_title=Science; citation_volume=306; citation_number=50; citation_pages=1547-1550; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Covarion structure in plastid genome evolution: a new statistical test.; citation_author=C Ane; citation_author=JG Burleigh; citation_author=MM McMahon; citation_author=MJ Sanderson; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=51; citation_pages=914-924; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The HIV positive selection mutation database.; citation_author=C Pan; citation_author=J Kim; citation_author=L Chen; citation_author=Q Wang; citation_author=C Lee; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=52; citation_pages=D371-375; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus reverse transcriptase and protease sequence database.; citation_author=SY Rhee; citation_author=MJ Gonzales; citation_author=R Kantor; citation_author=BJ Betts; citation_author=J Ravela; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=53; citation_pages=298-303; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=On the interpretation of x(2) from contingency tables, and the calculation of P.; citation_author=RA Fisher; citation_journal_title=Journal of the Royal Statistical Society; citation_volume=85; citation_number=54; citation_pages=87-94; citation_date=1922; " />
      <meta name="citation_reference" content="citation_title=A survey of exact inference for contingency tables.; citation_author=A Agresti; citation_journal_title=Statistical Science; citation_volume=7; citation_number=55; citation_pages=131-177; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Sample Sizes Required to Detect Linkage Disequilibrium between 2 or 3 Loci.; citation_author=AHD Brown; citation_journal_title=Theoretical Population Biology; citation_volume=8; citation_number=56; citation_pages=184-201; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=On Measures of Gametic Disequilibrium.; citation_author=RC Lewontin; citation_journal_title=Genetics; citation_volume=120; citation_number=57; citation_pages=849-852; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Development of Associative Overdominance through Linkage Disequilibrium in Finite Populations.; citation_author=T Ohta; citation_author=M Kimura; citation_journal_title=Genetical Research; citation_volume=16; citation_number=58; citation_pages=165-&; citation_date=1970; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase"/>
    <meta name="twitter:description" content="Correlated amino acid mutation analysis has been widely used to infer functional interactions between different sites in a protein. However, this analysis can be confounded by important phylogenetic effects broadly classifiable as background linkage disequilibrium (BLD). We have systematically separated the covariation induced by selective interactions between amino acids from background LD, using synonymous (S) vs. amino acid (A) mutations. Covariation between two amino acid mutations, (A,A), can be affected by selective interactions between amino acids, whereas covariation within (A,S) pairs or (S,S) pairs cannot. Our analysis of the pol gene &mdash; including the protease and the reverse transcriptase genes &mdash; in HIV reveals that (A,A) covariation levels are enormously higher than for either (A,S) or (S,S), and thus cannot be attributed to phylogenetic effects. The magnitude of these effects suggests that a large portion of (A,A) covariation in the HIV pol gene results from selective interactions. Inspection of the most prominent (A,A) interactions in the HIV pol gene showed that they are known sites of independently identified drug resistance mutations, and physically cluster around the drug binding site. Moreover, the specific set of (A,A) interaction pairs was reproducible in different drug treatment studies, and vanished in untreated HIV samples. The (S,S) covariation curves measured a low but detectable level of background LD in HIV."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000814.g005"/>

  <meta property="og:title" content="Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=2616'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7436'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1086&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000814">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000814" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000814&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Qi Wang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Computational Biology, Molecular Biology Institute, Institute for Genomics and Proteomics, University of California at Los Angeles, Los Angeles, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christopher Lee
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:leec@chem.ucla.edu">leec@chem.ucla.edu</a></p>

                <p>Affiliations:
                  Center for Computational Biology, Molecular Biology Institute, Institute for Genomics and Proteomics, University of California at Los Angeles, Los Angeles, United States of America, 
                  Department of Chemistry and Biochemistry, University of California at Los Angeles, Los Angeles, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 29, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000814</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000814-g001" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000814-g002" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000814-g003" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000814-g004" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000814-g005" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000814">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000814" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Correlated amino acid mutation analysis has been widely used to infer functional interactions between different sites in a protein. However, this analysis can be confounded by important phylogenetic effects broadly classifiable as background linkage disequilibrium (BLD). We have systematically separated the covariation induced by selective interactions between amino acids from background LD, using synonymous (S) <em>vs.</em> amino acid (A) mutations. Covariation between two amino acid mutations, (A,A), can be affected by selective interactions between amino acids, whereas covariation within (A,S) pairs or (S,S) pairs cannot. Our analysis of the <em>pol</em> gene — including the protease and the reverse transcriptase genes — in HIV reveals that (A,A) covariation levels are enormously higher than for either (A,S) or (S,S), and thus cannot be attributed to phylogenetic effects. The magnitude of these effects suggests that a large portion of (A,A) covariation in the HIV <em>pol</em> gene results from selective interactions. Inspection of the most prominent (A,A) interactions in the HIV <em>pol</em> gene showed that they are known sites of independently identified drug resistance mutations, and physically cluster around the drug binding site. Moreover, the specific set of (A,A) interaction pairs was reproducible in different drug treatment studies, and vanished in untreated HIV samples. The (S,S) covariation curves measured a low but detectable level of background LD in HIV.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang Q, Lee C (2007) Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase. PLoS ONE 2(8):
          e814.
            doi:10.1371/journal.pone.0000814</p><p><strong>Academic Editor: </strong>Jean Carr, Institute of Human Virology, United States of America</p><p><strong>Received:</strong> July 6, 2007; <strong>Accepted:</strong> August 1, 2007; <strong>Published:</strong> August 29, 2007</p><p><strong>Copyright:</strong> © 2007 Wang, Lee. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported by grants from the NIH (U54 RR021813) and DOE (DE-FC02-02ER63421), a Dreyfus Foundation Teacher-Scholar Award to CJL, as well as UCLA dissertation year fellowship and UCLA AIDS Institute fellowship to QW.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Correlated amino acid mutation analysis has been widely used to infer functional interactions between different sites in a protein <a href="#pone.0000814-Altschuh1">[1]</a>–<a href="#pone.0000814-Hatley1">[12]</a>. Typically, a strong correlation between amino acid mutations is interpreted as evidence of functional interactions under substantial selection pressure. For example, statistical covariation of amino acid mutations in HIV has revealed interesting biological interactions between sites, and constraints imposed by protein structure <a href="#pone.0000814-Korber1">[13]</a>–<a href="#pone.0000814-Svicher1">[19]</a>. Therefore, studying covariation of amino acid mutations in HIV will improve our understanding of HIV drug resistance as well as help vaccine design <a href="#pone.0000814-Korber1">[13]</a>, <a href="#pone.0000814-Wu1">[18]</a>. Studies of covariation in different regions of HIV genome have identified a number of correlated amino acid mutation pairs, many of which have known biological interactions <a href="#pone.0000814-Korber1">[13]</a>–<a href="#pone.0000814-Svicher1">[19]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>However, such covariation analysis can be confounded by important phylogenetic effects <a href="#pone.0000814-Korber1">[13]</a>, <a href="#pone.0000814-Bickel1">[14]</a>. One major challenge for covariation analysis is distinguishing covariation that is genuinely due to selection pressure, from covariation that is simply due to co-inheritance from a common ancestor. When a mutation first occurs in an individual chromosome, other mutations are already present in that chromosome, and initially this mutation will be inherited in 100% linkage with those other mutations. Such co-occurrence due to common ancestry is classified as background linkage disequilibrium (BLD) <a href="#pone.0000814-Service1">[20]</a> (<a href="#pone-0000814-g001">Fig. 1A</a>). Over time, however, such linkage will be scrambled by events such as recombination and mutation, returning to equilibrium (no statistical association between them). For example, homologous recombination events between any pair of mutations will gradually scramble any linkage between the mutation pair at a rate that is proportional to the physical distance between them, the recombination rate, and the passage of time. This raises several questions. How to distinguish BLD from the covariation due to selection pressure? What fraction of covariation is BLD? How strong is BLD in HIV? The evidence from different studies has been ambiguous. On one hand, studies indicate that phylogenetic effects in HIV are strong. Phylogenetic analysis has successfully inferred HIV transmission history from HIV sequences <a href="#pone.0000814-Leitner1">[21]</a>, <a href="#pone.0000814-Thomson1">[22]</a>. On the other hand, HIV's high mutation rate <a href="#pone.0000814-Mansky1">[23]</a>, <a href="#pone.0000814-Korber2">[24]</a>, recombination rate <a href="#pone.0000814-Jetzt1">[25]</a>–<a href="#pone.0000814-Rhodes1">[27]</a>, and short generation time <a href="#pone.0000814-Wei1">[28]</a>–<a href="#pone.0000814-Perelson1">[30]</a> should reduce the phylogenetic effect significantly.</p>
<div class="figure" id="pone-0000814-g001"><div class="img"><a name="pone-0000814-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Schema of Separating Selective Interactions from Background Linkage Disequilibrium (BLD).</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Mutation covariation due to BLD. Covariation of mutation A and R (shown in multiple sequence alignment, right) is caused by co-inheritance of the two mutations from a common ancestor (shown in the phylogenetic tree, left). (B) Mutation covariation due to selective interactions. Relative fitness models for mutations x and y, the double mutant (xy), and wildtype (0). Two models are contrasted: top, independent (additive) fitness effects don't cause amino acid mutation covariation; bottom, selective interactions cause covariation of x and y. (C) Distinguishing BLD <em>vs.</em> fitness using pairs of amino acid mutations (A) and synonymous (S) mutations.</p>
<span>doi:10.1371/journal.pone.0000814.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>It should also be emphasized that phylogenetic analysis can be confounded by strong selection pressure. Whereas phylogenetic analysis interprets the presence of the same mutation in several individuals as evidence of common ancestry, selection pressure creates bias for “convergent evolution” <a href="#pone.0000814-Doolittle1">[31]</a> in which the same mutation can evolve independently many times due to positive selection. Such hidden biases are incompatible with the assumptions of classical phylogenetic analysis <a href="#pone.0000814-Felsenstein1">[32]</a>. For example, one study has reported that selection pressure for drug resistance can cause incorrect phylogenetic inferences from HIV sequences (compared with the known transmission history) <a href="#pone.0000814-Lemey1">[33]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Thus, it is important to develop methods that can distinguish these two causes of covariation. Recently, analytical methods such as parametric bootstrap <a href="#pone.0000814-Wollenberg1">[34]</a> and phylogeny-based shuffling <a href="#pone.0000814-Noivirt1">[35]</a> have been applied to estimate what fraction of covariation arises from phylogenetic effects. In this paper we take a different approach, based on comparing levels of covariation between different types of mutation pairs, such as pairs of amino acid mutations (A,A) <em>vs.</em> pairs of synonymous mutations (S,S). Whereas (A,A) pairs are subject to both phylogenetic effects and selection pressure on functional interactions between amino acid sites, (S,S) pairs cannot be subject to this type of selection pressure (since they leave the amino acid sequence unchanged) (<a href="#pone-0000814-g001">Fig. 1</a>). We have therefore used synonymous mutations to measure background LD <a href="#pone.0000814-Service1">[20]</a>, to systematically distinguish covariation from phylogenetic effects <em>vs.</em> other sources of covariation, independent of mathematical model assumptions. Throughout this paper we will use the term “covariation” to refer to observed statistical association (without implying any specific interpretation of its cause); “background linkage disequilibrium” to indicate the specific interpretation of co-inheritance from a common ancestor; and “selective interactions” to indicate the specific interpretation of selection pressure for co-occurrence of a given pair of amino acid mutations. The “selective interactions” include 3D structural interactions (both local and long-range) as well as phenotypic covariation due to shared selection pressure. It should also be noted that selection on nucleotides (<em>e.g.</em> constraints on the RNA structure in viruses <a href="#pone.0000814-Hofacker1">[36]</a>, <a href="#pone.0000814-Shapiro1">[37]</a>) rather than on amino acids can also cause BLD, as measured by (S,S) covariation, thus may contribute to the covariation of all three types of mutation pairs, (A,A), (A,S) and (S,S), in this study.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Metrics of LD in HIV and Its Comparison with Background LD</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>First, we performed standard analyses of Linkage Disequilibrium (LD) on a dataset of about 50,000 HIV-1 <em>pol</em> gene sequences of subtype B, covering a 1.4 kb region of the HIV protease and reverse transcriptase (RT) genes, mostly from patients under antiretroviral drug treatment <a href="#pone.0000814-Chen1">[38]</a>. Following the procedure of the Human Genome HapMap project <a href="#pone.0000814-The1">[39]</a>, we applied a minimum frequency criteria to the data before measuring the LD. After applying the frequency cutoff of 2%, our dataset included 398 distinct single nucleotide mutations, each with 3260 observation counts on average. It should be noted that due to the very large size of this dataset and the high rate of mutation in HIV, we detected a very high density of mutations, including mutations at the majority of individual nucleotide sites, most with large numbers of observations. This provided a uniquely high-resolution mutation dataset for mapping LD. The density of mutations (observations per nucleotide) in this dataset is 100-fold higher than in the data from the Human Genome HapMap project <a href="#pone.0000814-The1">[39]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We computed D′ and r <a href="#pone.0000814-Balding1">[40]</a>, two measures of statistical association commonly used to measure LD in many organisms, <em>e.g.</em> human <a href="#pone.0000814-Pritchard1">[41]</a>–<a href="#pone.0000814-Stephens1">[44]</a>. Both metrics displayed a pattern in HIV similar to that in human, decaying as a function of distance (<a href="#pone-0000814-g002">Fig. 2A</a>, and <a href="#pone.0000814.s001">Fig. S1A</a>), as expected from population genetics theory. However, they indicated weaker LD than that in human <a href="#pone.0000814-Pritchard1">[41]</a>–<a href="#pone.0000814-Stephens1">[44]</a>, which is consistent with HIV's high mutation rate <a href="#pone.0000814-Mansky1">[23]</a>, <a href="#pone.0000814-Korber2">[24]</a>, recombination rate <a href="#pone.0000814-Jetzt1">[25]</a>–<a href="#pone.0000814-Rhodes1">[27]</a>, and short generation time <a href="#pone.0000814-Wei1">[28]</a>–<a href="#pone.0000814-Perelson1">[30]</a>, the factors that reduce LD according to the population genetics theory.</p>
<div class="figure" id="pone-0000814-g002"><div class="img"><a name="pone-0000814-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A,A) Covariation Is Dramatically Higher Than (A,S) and (S,S) Covariation in the Specialty Dataset.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) Sliding window results of average D′. All mutation pairs, black; silent mutation pairs (S,S) only, green. Each sliding window contains 4% of the data points in the set. (B) Sliding window results of average D′. Amino acid mutation pairs (A,A), red; amino acid mutations to silent mutations (A,S), blue; silent mutation pairs (S,S), green. Each sliding window contains 2% of the data points in the set. (C–E) Plots of D′ against the physical distance (base) within the mutation pair for C) (A,A), D) (A,S) and E) (S,S).</p>
<span>doi:10.1371/journal.pone.0000814.g002</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Another crucial difference between HIV and classical examples of LD analysis (<em>i.e.</em> the human data), is that this region of the HIV genome experiences very strong positive selection pressure due to antiviral drug treatments <a href="#pone.0000814-Shafer1">[45]</a>, <a href="#pone.0000814-Clavel1">[46]</a>. To control for possible LD caused by amino acid selection pressure, we repeated this LD analysis strictly for pairs of synonymous mutations. Such mutations do not change the amino acid sequence and thus are not subject to amino acid selection pressure, yielding a measurement of “background LD” free of amino acid selection artifacts <a href="#pone.0000814-Service1">[20]</a>. D′ and r metrics of the background LD decayed as a function of physical distance (<a href="#pone-0000814-g002">Fig. 2A</a>, and <a href="#pone.0000814.s001">Fig. S1A</a>). However, the average background LD by these metrics was smaller than the average LD, across the whole one kb region. The average D′ for background LD decayed from 0.02 for adjacent mutations to 0.01 for distances of 0.4 kb or more, about two-fold lower than the standard LD curve over the same distance range (<a href="#pone-0000814-g002">Fig. 2A</a>). The average r showed a similar pattern (<a href="#pone.0000814.s001">Fig. S1A</a>). This suggests that selection pressure plays an important role in shaping LD in HIV.</p>


<h4>Comparing Covariation of (A,A), (A,S) and (S,S) in HIV</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To examine this hypothesis further, we subdivided mutation pairs into three groups: pairs of synonymous mutations (S,S); pairs of amino acid mutations (A,A); and pairs consisting of one amino acid mutation and one synonymous mutation (A,S). (A,A) pairs experience both phylogenetic effects and possible selective interactions; that is, (A,A) pairs that together increase reproductive fitness may be selected for co-occurrence. By contrast, since synonymous mutations do not affect the amino acid sequence, both (S,S) and (A,S) pairs are not subject to amino acid selective interaction effects. Thus, the extent of selective interaction effects can be estimated by a systematic pattern of excess covariation specifically for (A,A) pairs relative to that for (S,S) and (A,S) pairs.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We first compared the (A,A), (A,S) and (S,S) covariation in the same dataset of 50,000 HIV-1 samples. After applying minimum frequency cutoff of 2%, 124 amino acid mutations and 274 silent mutations were included, yielding 7626 (A,A) pairs, 33976 (A,S) pairs, and 37401 (S,S) pairs. Compared with (A,S) and (S,S), (A,A) covariation was dramatically higher, in both D′ and r (<a href="#pone-0000814-g002">Fig. 2</a>, and <a href="#pone.0000814.s001">Fig. S1</a>). Only (A,A) pairs showed D′ greater than 0.8 and many more (A,A) pairs showed strong covariation (D′&gt;0.5) than (A,S) and (S,S) pairs (<a href="#pone-0000814-g002">Fig. 2C, 2D, 2E</a>). Furthermore, the average covariation of (A,A) was much higher than that of (A,S) and (S,S) (<a href="#pone-0000814-g002">Fig. 2B</a>). The average D′ of (A,A) gradually declined from 0.18 to 0.03 over about 1000 bases, while the average D′ of (A,S) and (A,A) started at less than 0.05 and rapidly dropped to 0.01 at around 300 bases. On average, (A,A) covariation levels were two- to five-fold higher than those of (A,S) and (S,S) across this range of distances. The conclusion also held for the frequency cutoff of 1% and 4% (<a href="#pone.0000814.s002">Fig. S2</a> and <a href="#pone.0000814.s003">S3</a>). In addition, the difference in distribution for covariation scores D′ of (A,A) <em>vs.</em> those of (A,S) and for (A,A) <em>vs.</em> (S,S) was statistically significant (both p-values less than 10<sup>−16</sup>, Wilcoxon rank sum test — see <a href="#s4">Materials and Methods</a>). Thus, a predominant fraction of (A,A) covariation does not appear to be attributable to background LD as measured by (S,S) covariation.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>It is also striking that the (A,S) and (S,S) covariation (measured by D′ and r) behaved similarly, in contrast with (A,A) covariation. The average D′ of (A,S) and (S,S) both started under 0.05 and gradually decayed until they reached a flat of around 0.01 at 300 bases (<a href="#pone-0000814-g002">Fig. 2B</a>). The same pattern was repeated in the average r curve (<a href="#pone.0000814.s001">Fig. S1B</a>). However, it is also interesting that there appear to be slight differences between (A,S) and (S,S) at short distances (less than 200 bases). The average D′ value for (A,S) was significantly higher (up to 0.04) than (S,S) for adjacent mutations, but decayed more rapidly, so that this difference vanished beyond 300 bases. This higher value of (A,S) <em>vs.</em> (S,S) is consistent with the known strong positive selection for amino acid mutations in this region <a href="#pone.0000814-Shafer1">[45]</a>, <a href="#pone.0000814-Clavel1">[46]</a>, since (A,S) pairs would be directly affected by such potential selective sweep events <a href="#pone.0000814-Smith1">[47]</a>, <a href="#pone.0000814-Nielsen1">[48]</a>, whereas (S,S) pairs can only be affected indirectly (<em>i.e.</em> only by selective sweep for a third mutation that is a positively selected amino acid mutation).</p>


<h4>Comparing Covariation of (A,A), (A,S) and (S,S) in the Stanford-Treated Dataset</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To assess the reproducibility of these results, we repeated this analysis of (A,A), (A,S) and (S,S) covariance in a second, independent dataset, containing about 7,000 drug-treated HIV samples of subtype B covering either protease or RT (Stanford-Treated; see <a href="#s4">Materials and Methods</a>). 73 amino acid mutations and 103 silent mutations (mutation frequency ≥5%; see <a href="#s4">Materials and Methods</a>) were included in the analysis.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Although the average number of samples per site in Stanford-Treated was less than one tenth of the Specialty dataset, we found the same covariance pattern — the (A,A) covariation (D′) was much stronger than that of (A,S) and (S,S) (both p-values less than 10<sup>−7</sup>, Wilcoxon rank sum test — see <a href="#s4">Materials and Methods</a>), and the covariation levels of (A,S) and (S,S) were similar (p-value = 0.89, Wilcoxon rank sum test — see <a href="#s4">Materials and Methods</a>). The average D′ of (A,A) started at around 0.20 and declined to 0.07 over a scale of 800 bases; while for (A,S) and (S,S), the average D′ started less than 0.07 and then both fluctuated at around 0.05 (<a href="#pone-0000814-g003">Fig. 3A</a>). The average r showed a similar pattern, differing from D′ mainly in measurement scale (<a href="#pone.0000814.s004">Fig. S4A</a>). Overall, (A,A) covariation was about two- to four-fold higher than (A,S) and (S,S) covariation levels. Again, the (A,S) and (S,S) curves were largely indistinguishable within the range of sampling variance inherent in the dataset (<a href="#pone-0000814-g003">Fig. 3A</a>). These data demonstrate again that most (A,A) covariation in this region of HIV is not attributable to background LD as measured by (S,S) covariation, suggesting a dominant role for selective interactions due to selection pressure imposed by antiviral drug treatment.</p>
<div class="figure" id="pone-0000814-g003"><div class="img"><a name="pone-0000814-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>(A,A) Covariation Is Dramatically Higher than (A,S) and (S,S) Covariation in the Stanford-Treated Dataset but not the Stanford-Untreated Dataset.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Sliding window results of average D′ in A) Stanford-Treated Dataset and B) Stanford-Untreated Dataset. Amino acid mutation pairs (A,A), red; amino acid mutations to silent mutations (A,S), blue; silent mutation pairs (S,S), green. Each sliding window contains 4% of the data points in the set.</p>
<span>doi:10.1371/journal.pone.0000814.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>To exclude the possibility that the consistent pattern between Specialty and Stanford-Treated datasets results from overlapping samples, we eliminated from the Specialty dataset all sequences with 98% or higher identity to samples in the Stanford dataset and re-analyzed the Specialty dataset. After the filtering, the (A,A) covariation level is still much higher than (A,S) and (S,S) across all distances (<a href="#pone.0000814.s005">Fig. S5</a>).</p>


<h4>Comparing Covariation of (A,A), (A,S) and (S,S) in the Stanford-Untreated Dataset</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To test the role of drug-induced selection via a negative control, we carried out the same analysis in a set of samples collected from untreated patients (Stanford-Untreated; see <a href="#s4">Materials and Methods</a>). Previous studies have showed that comparison of these Treated <em>vs.</em> Untreated datasets can identify the effects of antiviral drug treatment <a href="#pone.0000814-Gonzales1">[15]</a>, <a href="#pone.0000814-Wu1">[18]</a>. The Untreated dataset contained about 4,500 drug-naive samples covering either protease or RT (see <a href="#s4">Materials and Methods</a>). 42 amino acid mutations and 107 silent mutations (mutation frequency ≥5%) were included in the analysis.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Strikingly, the large difference in covariation between (A,A) and (A,S)/(S,S) disappeared in the untreated dataset. The average D′ of (A,A) fluctuated around the average D′ of (A,S) and (S,S), at approximately 0.07 (<a href="#pone-0000814-g003">Fig. 3B</a>). The same pattern was repeated in the average r curve (<a href="#pone.0000814.s004">Fig. S4B</a>). These data provide a clear, independent confirmation that drug-induced selection pressure is indeed the explanation for the surplus covariation of (A,A) pairs (relative to background LD measured by (S,S)) in the Specialty and Stanford-Treated datasets, both of which included drug-treated samples.</p>


<h4>(A,A) Pairs Near the RT Active Site Show Strong Covariation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The (A,A) covariation decay curve (<a href="#pone-0000814-g002">Fig. 2B</a>) revealed a clear double-peak for pairs between 400–550 base distance in the Specialty dataset. Strikingly, a similar double-peak was observed at the same location in the Stanford-Treated (A,A) curve (<a href="#pone-0000814-g003">Fig. 3A</a>). We analyzed the two datasets separately to identify the mutation pairs responsible for these two peaks. These data revealed that the peaks were caused by the same set of mutation pairs in both datasets. One peak resulted from strong covariation between a cluster of mutations RT 41L and 43E with another cluster RT 208Y and 210W; while the other peak reflected strong covariation between a cluster RT 67N and 70R with the cluster RT 208Y, 218E and 219E/Q. Interestingly, in the three-dimensional protein structure, all these residues lie close to the reverse transcriptase active site (<a href="#pone-0000814-g004">Fig. 4</a>), less than 25 Å apart. Furthermore, mutations RT 41L, 67N, 70R, 210W and 219E/Q are known RT drug resistance mutation <a href="#pone.0000814-Johnson1">[49]</a>. Thus every single one of the (A,A) covariation pairs observed in these peaks consisted of either one, or two known drug-resistance mutations. This analysis of the individual (A,A) covariation pairs provides independent confirmation that these specific residues are positively selected for drug-resistance.</p>
<div class="figure" id="pone-0000814-g004"><div class="img"><a name="pone-0000814-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Amino Acid Mutation Pairs that Show Strong Covariation Are Close to Active Sites in RT.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>HIV-1 reverse transcriptase (RT) structure (PDB accession number 3HVTA) is shown using Protein Explorer (<a href="http://www.proteinexplorer.org">www.proteinexplorer.org</a>). The RT41, 43 and 44, red; RT 67 and 70, green; RT 208, 210, 218, 219, yellow; active sites 110,185 and 186 in magenta. The grey sphere cluster is the nucleoside reverse transcriptase inhibitor — Nevirapine.</p>
<span>doi:10.1371/journal.pone.0000814.g004</span></div>

<h4>Comparison of Covariation Maps of Protease in the Specialty Dataset</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>To provide a comprehensive analysis of the specific (A,A) pairs that showed significant evidence of selective interactions, we constructed two-dimensional maps of the statistical strength of covariation between all possible pairs of codons positions in HIV protease, for (A,A), (A,S) and (S,S) (<a href="#pone-0000814-g005">Fig. 5</a>). We used Fisher's exact test to calculate the covariation score θ (see <a href="#s4">Materials and Methods</a>), which detects statistically significant covariation even in cases with smaller counts. No minimum frequency criteria was applied. This allows us to comprehensively compare the covariation of different types of mutation pairs. In this analysis, we used the Specialty dataset, due to its much larger number of samples (about 50,000). For each pair of codon positions, the map displays the highest level of covariation measured for mutations at that pair (see <a href="#s4">Materials and Methods</a>).</p>
<div class="figure" id="pone-0000814-g005"><div class="img"><a name="pone-0000814-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000814" data-uri="info:doi/10.1371/journal.pone.0000814.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000814.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000814.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000814.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000814.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The Covariation Maps of Three Different Types of Mutation Pairs in HIV Protease.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The covariation maps of A) amino acid mutation pairs (A,A), B) amino acid mutations to silent mutations (A,S) and C) silent mutation pairs (S,S). The X and Y axes represent the codon positions in protease. Each cell represents the strongest covariation value (θ; see <a href="#s4">Materials and Methods</a>) measured for any mutation pair of the designated type between the two positions. The strength of the covariation is depicted on a color scale, with yellow indicating covariation score (θ) larger than 1 and varying shades up to blue indicating covariation score (θ) larger than 5 (the covariation of two mutations is at least five times greater than random). White indicates no evidence of covariation.</p>
<span>doi:10.1371/journal.pone.0000814.g005</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>These data reveal several striking differences between (A,A), (A,S) and (S,S) covariation. First, the (A,A) map contains a large fraction (2.6%) of strong covariation effects (θ&gt;5 at 95% confidence; see <a href="#s4">Materials and Methods</a>), compared with only a small fraction for (A,S) (0.3%) and (S,S) (0.3%). Second, whereas strong (A,A) covariation broadly distributed across the whole map (<a href="#pone-0000814-g005">Fig. 5A</a>), in the (A,S) and (S,S) maps covariation clustered close to the diagonal (<a href="#pone-0000814-g005">Fig. 5B and 5C</a>), <em>i.e.</em> for codon positions that are close in the sequence. These data suggest that background LD decays rapidly, within 100 nt (about 30 amino acids). The fact that (A,A) covariation extended more broadly, indicates that it arises from a different process than background LD. Third, a large fraction (30 out of 53) of the codon positions identified by (A,A) covariation are known drug-resistance mutation sites <a href="#pone.0000814-Johnson1">[49]</a> in HIV protease (for the list of drug-resistance codons identified, see <a href="#s4">Materials and Methods</a>), confirming again that these covariation effects involve drug-resistance selection.</p>


<h4>Correlation of the Covariation Between Independent Datasets</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We have shown that the level of (A,A) covariation is higher than (A,S) and (S,S) covariation in both drug-treated datasets (Specialty; Stanford-Treated). However, do these independent datasets display the same covariation effects? To answer this question, we compared the covariation measurements for each (A,A) pair from these two datasets. For each (A,A) pair, we plotted the covariation value observed in the Specialty dataset against the covariation value observed in the Stanford-Treated dataset (<a href="#pone.0000814.s006">Fig. S6</a>). Strikingly, the (A,A) pairs that covaried in the Specialty dataset also covaried in the Stanford-Treated (<a href="#pone.0000814.s006">Fig. S6A</a>), and the covariation values in the two datasets showed strong quantitative agreement, yielding a high correlation coefficient of 0.83 (See <a href="#s4">Materials and Methods</a>) between these two independent datasets. In contrast, for (A,S) and (S,S) pairs, the covariation detected in these two datasets was low, both giving the correlation coefficients of 0.35. This indicates that a single, consistent pattern of selective interactions is reproducibly discovered in the two independent datasets, but only for (A,A) covariation.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>To assess whether drug treatment acts as the consistent amino acid selection in these two datasets, we compared the (A,A) covariation in the Treated dataset with that in the Untreated one. We found that the high consistence of (A,A) covariation between the Specialty and the Treated (correlation coefficient 0.83) disappeared in the comparison between the Untreated and the Treated, leaving a correlation coefficient of only 0.39 (<a href="#pone.0000814.s006">Fig. S6D</a>). This suggests that drug treatment (shared by the Specialty and the Treated datasets, but not the Untreated dataset) causes the nearly identical pattern of selective interactions found in these two independent datasets.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have systematically separated the covariation induced by selective interactions from background LD, using silent (S) and amino acid (A) mutations. Selective interactions between amino acids can be detected by (A,A) pairs, but not by (A,S) or (S,S) pairs. Our analysis of the <em>pol</em> gene in HIV suggests that a large portion of (A,A) covariation in HIV results from selective interactions. Meanwhile, the (S,S) covariation curves suggest a low but detectable level of background LD in HIV. Although HIV has extremely high mutation and recombination rate, as well as short generation time, the (S,S) covariation metrics were still able to detect some BLD, decreasing as a function of physical distance (<a href="#pone-0000814-g002">Fig. 2</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Several lines of evidence demonstrate the robustness of these conclusions. First, the same results were found by three different measurements of covariation: the widely used D′ and r metrics, and Fisher's exact test. Second, these results were reproduced in independent experimental studies (the Specialty and Stanford-Treated datasets). Third, the high level of consistency between independent (A,S) and (S,S) covariation curves suggests that the much higher level of covariation observed for (A,A) pairs cannot be attributed to background LD. Fourth, we also found direct evidence that the difference in covariation levels between (A,A) <em>vs.</em> (A,S)/(S,S) is due to selection, specifically, antiviral drug treatment, by comparing treated <em>vs.</em> untreated datasets. Fifth, the most prominent (A,A) interactions in the HIV <em>pol</em> gene have been independently identified as drug resistance mutations that physically cluster around the drug binding site. Finally, the specific set of (A,A) interaction pairs was reproducible in different drug treatment studies, and vanished in untreated HIV samples. Our result agrees with the ‘observation of positive epistasis in HIV <a href="#pone.0000814-Bonhoeffer1">[50]</a>. A previous study in plastid genomes also indicates that the significant covariation in plastid genomes is likely due to changes in the selective constraints of amino acids <a href="#pone.0000814-Ane1">[51]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Could the surplus of the (A,A) covariation compared with that of (A,S) and (S,S) in the treated datasets (Specialty and Stanford-Treated) be an artifact of differences in the intrinsic mutation rates between silent and amino acid mutations (<em>e.g.</em> silent mutations are more likely to be transitions than transversions, thus evolving faster)? We directly tested this possibility by performing the same analysis in samples from untreated patients (Stanford-Untreated). Such an artifact should have also have been observed in the untreated dataset. Yet, the difference between (A,A) <em>vs.</em> (A,S)/(S,S) disappeared in the untreated dataset (<a href="#pone-0000814-g003">Fig. 3</a>), indicating that this difference was due specifically to drug-treatment. It should also be noted that in addition to drug treatment, there are other sources of selection, such as immune pressure. Like the drug-induced selection, this too only causes (A,A) but not (A,S) or (S,S) covariation. However, we didn't detect a significant difference between (A,A) <em>vs.</em> (A,S)/(S,S) in the untreated samples, suggesting our approach is not sensitive enough to detect weaker selection.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>How might drug treatment cause the dramatic increase in covariation of amino acid mutation pairs observed in HIV? Several models are possible. 1) Drug treatment selects for mutations that directly cause drug resistance (called primary mutations), many of which may have secondary effects such as reducing protein stability and/or other aspects of viral fitness. These mutations can in turn induce selection pressure for mutations that compensate for these effects (called accessory mutations; <em>e.g.</em> a mutation that restores the protein stability). Such secondary selection effects will cause a pattern of covariation of primary mutations with their associated accessory mutations. By contrast, in the untreated samples, where such positive selection forces are presumably weaker, we did not detect significant evidence of selective interactions. 2) The covariation can be caused by shared selection pressure among amino acid mutations. If mutation X and Y are independently selected for under the same drug treatment, the two mutations are likely to covary. To distinguish the aforementioned two possibilities, we need to estimate the fitness of mutation X and Y separately and compare the sum with the fitness of the XY double mutation, which is beyond the scope of this study.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our data also indicate that accurate measurement of background LD is useful to improve the accuracy of functional interaction prediction. Accurate measurement of the background LD would enable calculation of a threshold value above which the statistics will have a specific probability of resulting from causes other than background LD. Comparison of statistical values of covariation calculated from (A,A) pairs with the background (<em>i.e.</em> (S,S) covariation) allows identification of pairs of sites having a specific probability of interacting due to selection on amino acids. Such phylogenetic effects should be taken into consideration in covariation analysis of amino acid interactions.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In this paper, we have only analyzed the <em>pol</em> gene, which is known to have experienced strong drug selection. The same method should be applied in the other regions of the HIV genome. We expect the (A,S) and (S,S) covariation, while still consistent with each other, varies across the HIV genome due to different phylogeny across the genome. The comparison of (A,A) covariation with that of (A,S)/(S,S) across the HIV genome will provide a global view of the influence of selection on mutation covariation. For example, such comparison in the <em>env</em> gene will hopefully improve our understanding of the interplay between the host selection and phylogeny in that region. The same analysis could also be done in subtypes other than subtype B.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>HIV-1 sequence data</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The Specialty dataset contained 48,927 subtype B sequences, mostly from patients under antiretroviral drug treatment. Multiple sequence alignments and mutation detection were performed as previously described <a href="#pone.0000814-Chen1">[38]</a>, <a href="#pone.0000814-Pan1">[52]</a>. All these sequences covered the whole protease and part of the RT. The Treated and the Untreated datasets were downloaded from Stanford database (<a href="http://hivdb.stanford.edu/">http://hivdb.stanford.edu/</a>) <a href="#pone.0000814-Rhee1">[53]</a>, selecting only subtype B sequences. In the Treated dataset, there were 1795 protease sequences treated with protease inhibitor (this subset were used to calculate the covariation between mutations in protease) and 5121 reverse transcriptase sequences treated with either nucleoside reverse transcriptase inhibitor (NRI) or non-nucleoside reverse transcriptase inhibitor (NNRI) (this subset were used to calculate the covariation between mutations in RT), including 1320 samples that cover both protease and reverse transcriptase sequences (this subset were used to calculate the covariation between mutations in protease and those in RT). In each subset, we required a minimum mutation frequency of 5%. In the Untreated dataset, there were 2620 PI-naïve protease sequences and 1795 NRI/NNRI-naïve reverse transcriptase sequences, including 1208 samples that have both protease and reverse transcriptase treatment-naïve.</p>


<h4>Measurements of covariation for individual mutation pairs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We used the Fisher exact test <a href="#pone.0000814-Fisher1">[54]</a>, <a href="#pone.0000814-Agresti1">[55]</a> to test for non-random associations between mutation α at position X and mutation β at position Y, by computing the p-value for the two-sided test using the 2×2 contingency table: N<sub>XαYβ</sub>, N<sub>XαY0</sub>, N<sub>X0Yβ</sub> and N<sub>X0Y0</sub>·N<sub>XαYβ</sub> is the number of samples that have mutation α at position X and also mutation β at position Y; N<sub>XαY0</sub> is the number of samples that have mutation α at position X and but no kind of mutation at position Y; N<sub>X0Yβ</sub> is the number of samples that have no mutation at position X and have mutation β at position Y; N<sub>X0Y0</sub> is the number of samples that have mutation at neither position. We computed the odds ratio, its confidence interval (95% two-sided) and p-value using the fisher.test function from the statistical software package R. Note: the maximum likelihood estimator for θ is provided by (N<sub>XαYβ</sub>·N<sub>X0Y0</sub>)/(N<sub>XαY0</sub>·N<sub>X0Yβ</sub>); for independent mutations X<sub>α</sub> and Y<sub>β</sub>, θ = 1. We calculated D′ and r following the standard procedures <a href="#pone.0000814-Brown1">[56]</a>–<a href="#pone.0000814-Ohta1">[58]</a>, using p<sub>1</sub> = (N<sub>XαYβ</sub>+N<sub>XαY0</sub>)/N, q<sub>1</sub> = (N<sub>XαYβ</sub>+N<sub>X0Yβ</sub>)/N, x<sub>11</sub> = N<sub>XαYβ</sub>/N, where N = N<sub>XαYβ</sub>+N<sub>XαY0</sub>+N<sub>X0Yβ</sub>+N<sub>X0Y0.</sub>. Finally, we used Wilcoxon rank sum test (wilcox.test function in the R package) to compare different types of mutation pairs with respect to their covariation score (<em>e.g.</em> D′). The p-value is calculated for the null hypothesis that the covariation scores for the two types of mutation pairs are from the same distribution.</p>


<h4>Average LD as a function of distance</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Mutation pairs with negative LD were excluded. Mutation pairs were ranked by physical distance. We calculated smoothed curves using a sliding window, the window width of 2% or 4% of the total data, and an offset for neighboring windows of the 1/2 the window width.</p>


<h4>Covariation map</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>For each pair of mutations, we used Fisher exact test to compute a p-value for statistically significant covariation, along with a lower-bound estimate for the strength of covariation θ based on the 95% confidence interval. Only statistically significant mutation pairs (p&lt;10<sup>−6</sup> for a single pair, yielding a significance level of 0.01 after the Bonferroni correction) were included in our analysis. For a given codon position pair, the strongest covariation value θ for any pair of mutations at the two positions (of the designated type: (A,A), (A,S), or (S,S)) was displayed in the map.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>In protease, the drug-resistant codons identified by (A,A) covariation are 10, 13, 16, 20, 24, 30, 32, 33, 35, 36, 43, 46, 47, 48, 50, 53, 54, 58, 60, 62, 63, 71, 73, 74, 82, 84, 85, 88, 90 and 93.</p>


<h4>Comparing covariation between datasets</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>To test whether the covariation derived from two datasets, X and Y, was consistent, we plotted for every mutation pair the covariation measurement r in X <em>vs.</em> that in Y. We also calculated the correlation coefficient between the two datasets. Since correlation coefficient is very sensitive to outliers, the lowest correlation from 2000 bootstrap replicates (the R boot library) was taken as the correlation score between X and Y.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000814"></div><a name="pone.0000814.s001" id="pone.0000814.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000814.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">(A,A) Covariation Measured by r Is Dramatically Higher than (A,S) and (S,S) Covariation in the Specialty Dataset. (A) Sliding window results of average r. All mutation pairs, black; silent mutation pairs (S,S) only, green. Each sliding window contains 4% of the data points in the set. (B) Sliding window results of average r. Amino acid mutation pairs (A,A), red; amino acid mutations to silent mutations (A,S), blue; silent mutation pairs (S,S), green. Each sliding window contains 2% of the data points in the set. (C–E) Plots of r against the physical distance (base) within the mutation pair for C) (A,A), D) (A,S) and E) (S,S).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000814.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.84 MB TIF)</p>
<a name="pone.0000814.s002" id="pone.0000814.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000814.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">(A,A) Covariation Is Still Dramatically Higher than (A,S) and (S,S) Covariation in the Specialty Dataset Using 0.01 As the Mutation Frequency Cutoff. (A–C) Plots of D' against the physical distance (base) within the mutation pair for A) amino acid mutation pairs (A,A), B) amino acid mutations to silent mutations (A,S) and C) silent mutation pairs (S,S). (D, E) Sliding window results of average D) D' and E) r. (A,A), red; (A,S), blue; (S,S), green. Each sliding window contains 2% of the data points in the set.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000814.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.88 MB TIF)</p>
<a name="pone.0000814.s003" id="pone.0000814.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000814.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">(A,A) Covariation Is Still Dramatically Higher than (A,S) and (S,S) Covariation in the Specialty Dataset Using 0.04 As the Mutation Frequency Cutoff. (A–C) Plots of D' against the physical distance (base) within the mutation pair for A) amino acid mutation pairs (A,A), B) amino acid mutations to silent mutations (A,S) and C) silent mutation pairs (S,S). (D, E) Sliding window results of average D) D' and E) r. (A,A), red; (A,S), blue; (S,S), green. Each sliding window contains 2% of the data points in the set.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000814.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.55 MB TIF)</p>
<a name="pone.0000814.s004" id="pone.0000814.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000814.s004">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">(A,A) Covariation Measured by r Is Dramatically Higher than (A,S) and (S,S) Covariation in the Stanford-Treated Dataset But Not the Stanford-Untreated Dataset. Sliding window results of average r in (A) Stanford-Treated Dataset and (B) Stanford-Untreated Dataset. Amino acid mutation pairs (A,A), red; amino acid mutations to silent mutations (A,S), blue; silent mutation pairs (S,S), green. Each sliding window contains 4% of the data points in the set.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000814.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.17 MB TIF)</p>
<a name="pone.0000814.s005" id="pone.0000814.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000814.s005">Figure S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">(A,A) Covariation Is Still Dramatically Higher than (A,S) and (S,S) Covariation in the Specialty Dataset After Excluding Samples That Have Nucleotide Sequence Similarity 98% Or Greater With Any Sample In the Stanford-Treated Dataset. (A–C) Plots of D' against the physical distance (base) within the mutation pair for A) amino acid mutation pairs (A,A), B) amino acid mutations to silent mutations (A,S) and C) silent mutation pairs (S,S). (D, E) Sliding window results of average D) D' and E) r. (A,A), red; (A,S), blue; (S,S), green. Each sliding window contains 2% of the data points in the set.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000814.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.88 MB TIF)</p>
<a name="pone.0000814.s006" id="pone.0000814.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000814.s006">Figure S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Shared Drug Treatment Leads to High Consistency of Amino Acid Covariation between Independent Datasets. (A–C) The covariation measurement r in the Specialty dataset plotted against that in the Stanford-Treated dataset for A) amino acid mutation pairs (A,A), B) amino acid mutations to silent mutations (A,S) and C) silent mutation pairs (S,S). (D–F) The covariation measurement r in the Stanford-Untreated dataset plotted against that in the Treated dataset, for D) (A,A), E) (A,S) and F) (S,S).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000814.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.65 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We wish to thank A. Alekseyenko, L. Chen, N. Kim, H. Lu, C. Pan and Y. Xing for helpful discussion and Prof. N. Freimer, Prof. C. Sabatti and the reviews for their constructive comments on this manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CL QW. Performed the experiments: QW. Analyzed the data: QW. Contributed reagents/materials/analysis tools: CL. Wrote the paper: CL QW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000814-Altschuh1" id="pone.0000814-Altschuh1"></a>Altschuh D, Lesk AM, Bloomer AC, Klug A (1987) Correlation of co-ordinated amino acid substitutions with function in viruses related to tobacco mosaic virus. J Mol Biol  193: 693–707.  <ul class="find" data-citedArticleID="984590" data-doi="10.1016/0022-2836(87)90352-4"><li><a href="http://dx.doi.org/10.1016/0022-2836(87)90352-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlation+of+co-ordinated+amino+acid+substitutions+with+function+in+viruses+related+to+tobacco+mosaic+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlation+of+co-ordinated+amino+acid+substitutions+with+function+in+viruses+related+to+tobacco+mosaic+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000814-Gobel1" id="pone.0000814-Gobel1"></a>Gobel U, Sander C, Schneider R, Valencia A (1994) Correlated mutations and residue contacts in proteins. Proteins  18: 309–317.  <ul class="find" data-citedArticleID="984620" data-doi="10.1002/prot.340180402"><li><a href="http://dx.doi.org/10.1002/prot.340180402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correlated+mutations+and+residue+contacts+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correlated+mutations+and+residue+contacts+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000814-Shindyalov1" id="pone.0000814-Shindyalov1"></a>Shindyalov IN, Kolchanov NA, Sander C (1994) Can three-dimensional contacts in protein structures be predicted by analysis of correlated mutations? Protein Eng  7: 349–358.  <ul class="find" data-citedArticleID="984680" data-doi="10.1093/protein/7.3.349"><li><a href="http://dx.doi.org/10.1093/protein/7.3.349" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+three-dimensional+contacts+in+protein+structures+be+predicted+by+analysis+of+correlated+mutations%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+three-dimensional+contacts+in+protein+structures+be+predicted+by+analysis+of+correlated+mutations%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000814-Thomas1" id="pone.0000814-Thomas1"></a>Thomas DJ, Casari G, Sander C (1996) The prediction of protein contacts from multiple sequence alignments. Protein Eng  9: 941–948.  <ul class="find" data-citedArticleID="984692" data-doi="10.1093/protein/9.11.941"><li><a href="http://dx.doi.org/10.1093/protein/9.11.941" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+prediction+of+protein+contacts+from+multiple+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+prediction+of+protein+contacts+from+multiple+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000814-Olmea1" id="pone.0000814-Olmea1"></a>Olmea O, Valencia A (1997) Improving contact predictions by the combination of correlated mutations and other sources of sequence information. Fold Des  2: S25–32.  <ul class="find" data-citedArticleID="984658" data-doi="10.1016/s1359-0278(97)00060-6"><li><a href="http://dx.doi.org/10.1016/s1359-0278(97)00060-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+contact+predictions+by+the+combination+of+correlated+mutations+and+other+sources+of+sequence+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+contact+predictions+by+the+combination+of+correlated+mutations+and+other+sources+of+sequence+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000814-Olmea2" id="pone.0000814-Olmea2"></a>Olmea O, Rost B, Valencia A (1999) Effective use of sequence correlation and conservation in fold recognition. J Mol Biol  293: 1221–1239.  <ul class="find" data-citedArticleID="984660" data-doi="10.1006/jmbi.1999.3208"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effective+use+of+sequence+correlation+and+conservation+in+fold+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effective+use+of+sequence+correlation+and+conservation+in+fold+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000814-Larson1" id="pone.0000814-Larson1"></a>Larson SM, Di Nardo AA, Davidson AR (2000) Analysis of covariation in an SH3 domain sequence alignment: applications in tertiary contact prediction and the design of compensating hydrophobic core substitutions. J Mol Biol  303: 433–446.  <ul class="find" data-citedArticleID="984640" data-doi="10.1006/jmbi.2000.4146"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4146" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+covariation+in+an+SH3+domain+sequence+alignment%3A+applications+in+tertiary+contact+prediction+and+the+design+of+compensating+hydrophobic+core+substitutions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+covariation+in+an+SH3+domain+sequence+alignment%3A+applications+in+tertiary+contact+prediction+and+the+design+of+compensating+hydrophobic+core+substitutions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000814-Fariselli1" id="pone.0000814-Fariselli1"></a>Fariselli P, Olmea O, Valencia A, Casadio R (2001) Progress in predicting inter-residue contacts of proteins with neural networks and correlated mutations. Proteins Suppl  5: 157–162.  <ul class="find" data-citedArticleID="984610" data-doi="10.1002/prot.1173"><li><a href="http://dx.doi.org/10.1002/prot.1173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Progress+in+predicting+inter-residue+contacts+of+proteins+with+neural+networks+and+correlated+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Progress+in+predicting+inter-residue+contacts+of+proteins+with+neural+networks+and+correlated+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000814-Fodor1" id="pone.0000814-Fodor1"></a>Fodor AA, Aldrich RW (2004) Influence of conservation on calculations of amino acid covariance in multiple sequence alignments. Proteins  56: 211–221.  <ul class="find" data-citedArticleID="984616" data-doi="10.1002/prot.20098"><li><a href="http://dx.doi.org/10.1002/prot.20098" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+conservation+on+calculations+of+amino+acid+covariance+in+multiple+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+conservation+on+calculations+of+amino+acid+covariance+in+multiple+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000814-Suel1" id="pone.0000814-Suel1"></a>Suel GM, Lockless SW, Wall MA, Ranganathan R (2003) Evolutionarily conserved networks of residues mediate allosteric communication in proteins. Nat Struct Biol  10: 59–69.  <ul class="find" data-citedArticleID="984686" data-doi="10.1038/nsb881"><li><a href="http://dx.doi.org/10.1038/nsb881" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+networks+of+residues+mediate+allosteric+communication+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+networks+of+residues+mediate+allosteric+communication+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000814-Lockless1" id="pone.0000814-Lockless1"></a>Lockless SW, Ranganathan R (1999) Evolutionarily conserved pathways of energetic connectivity in protein families. Science  286: 295–299.  <ul class="find" data-citedArticleID="984648" data-doi="10.1126/science.286.5438.295"><li><a href="http://dx.doi.org/10.1126/science.286.5438.295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionarily+conserved+pathways+of+energetic+connectivity+in+protein+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionarily+conserved+pathways+of+energetic+connectivity+in+protein+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000814-Hatley1" id="pone.0000814-Hatley1"></a>Hatley ME, Lockless SW, Gibson SK, Gilman AG, Ranganathan R (2003) Allosteric determinants in guanine nucleotide-binding proteins. Proc Natl Acad Sci U S A  100: 14445–14450.  <ul class="find" data-citedArticleID="984624" data-doi="10.1073/pnas.1835919100"><li><a href="http://dx.doi.org/10.1073/pnas.1835919100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Allosteric+determinants+in+guanine+nucleotide-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Allosteric+determinants+in+guanine+nucleotide-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000814-Korber1" id="pone.0000814-Korber1"></a>Korber BT, Farber RM, Wolpert DH, Lapedes AS (1993) Covariation of mutations in the V3 loop of human immunodeficiency virus type 1 envelope protein: an information theoretic analysis. Proc Natl Acad Sci U S A  90: 7176–7180.  <ul class="find" data-citedArticleID="984636" data-doi="10.1073/pnas.90.15.7176"><li><a href="http://dx.doi.org/10.1073/pnas.90.15.7176" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covariation+of+mutations+in+the+V3+loop+of+human+immunodeficiency+virus+type+1+envelope+protein%3A+an+information+theoretic+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covariation+of+mutations+in+the+V3+loop+of+human+immunodeficiency+virus+type+1+envelope+protein%3A+an+information+theoretic+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000814-Bickel1" id="pone.0000814-Bickel1"></a>Bickel PJ, Cosman PC, Olshen RA, Spector PC, Rodrigo AG, et al.  (1996) Covariability of V3 loop amino acids. AIDS Res Hum Retroviruses  12: 1401–1411.  <ul class="find" data-citedArticleID="984596" data-doi="10.1089/aid.1996.12.1401"><li><a href="http://dx.doi.org/10.1089/aid.1996.12.1401" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covariability+of+V3+loop+amino+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covariability+of+V3+loop+amino+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000814-Gonzales1" id="pone.0000814-Gonzales1"></a>Gonzales MJ, Wu TD, Taylor J, Belitskaya I, Kantor R, et al.  (2003) Extended spectrum of HIV-1 reverse transcriptase mutations in patients receiving multiple nucleoside analog inhibitors. Aids  17: 791–799.  <ul class="find" data-citedArticleID="984622" data-doi="10.1097/00002030-200304110-00003"><li><a href="http://dx.doi.org/10.1097/00002030-200304110-00003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extended+spectrum+of+HIV-1+reverse+transcriptase+mutations+in+patients+receiving+multiple+nucleoside+analog+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extended+spectrum+of+HIV-1+reverse+transcriptase+mutations+in+patients+receiving+multiple+nucleoside+analog+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000814-Gilbert1" id="pone.0000814-Gilbert1"></a>Gilbert PB, Novitsky V, Essex M (2005) Covariability of selected amino acid positions for HIV type 1 subtypes C and B. AIDS Res Hum Retroviruses  21: 1016–1030.  <ul class="find" data-citedArticleID="984618" data-doi="10.1089/aid.2005.21.1016"><li><a href="http://dx.doi.org/10.1089/aid.2005.21.1016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covariability+of+selected+amino+acid+positions+for+HIV+type+1+subtypes+C+and+B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covariability+of+selected+amino+acid+positions+for+HIV+type+1+subtypes+C+and+B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000814-Hoffman1" id="pone.0000814-Hoffman1"></a>Hoffman NG, Schiffer CA, Swanstrom R (2003) Covariation of amino acid positions in HIV-1 protease. Virology  314: 536–548.  <ul class="find" data-citedArticleID="984630" data-doi="10.1016/s0042-6822(03)00484-7"><li><a href="http://dx.doi.org/10.1016/s0042-6822(03)00484-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covariation+of+amino+acid+positions+in+HIV-1+protease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covariation+of+amino+acid+positions+in+HIV-1+protease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000814-Wu1" id="pone.0000814-Wu1"></a>Wu TD, Schiffer CA, Gonzales MJ, Taylor J, Kantor R, et al.  (2003) Mutation patterns and structural correlates in human immunodeficiency virus type 1 protease following different protease inhibitor treatments. J Virol  77: 4836–4847.  <ul class="find" data-citedArticleID="984700" data-doi="10.1128/jvi.77.8.4836-4847.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.8.4836-4847.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+patterns+and+structural+correlates+in+human+immunodeficiency+virus+type+1+protease+following+different+protease+inhibitor+treatments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+patterns+and+structural+correlates+in+human+immunodeficiency+virus+type+1+protease+following+different+protease+inhibitor+treatments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000814-Svicher1" id="pone.0000814-Svicher1"></a>Svicher V, Ceccherini-Silberstein F, Erba F, Santoro M, Gori C, et al.  (2005) Novel human immunodeficiency virus type 1 protease mutations potentially involved in resistance to protease inhibitors. Antimicrob Agents Chemother  49: 2015–2025.  <ul class="find" data-citedArticleID="984688" data-doi="10.1128/aac.49.5.2015-2025.2005"><li><a href="http://dx.doi.org/10.1128/aac.49.5.2015-2025.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+human+immunodeficiency+virus+type+1+protease+mutations+potentially+involved+in+resistance+to+protease+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+human+immunodeficiency+virus+type+1+protease+mutations+potentially+involved+in+resistance+to+protease+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000814-Service1" id="pone.0000814-Service1"></a>Service SK, Ophoff RA, Freimer NB (2001) The genome-wide distribution of background linkage disequilibrium in a population isolate. Hum Mol Genet  10: 545–551.  <ul class="find" data-citedArticleID="984674" data-doi="10.1093/hmg/10.5.545"><li><a href="http://dx.doi.org/10.1093/hmg/10.5.545" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome-wide+distribution+of+background+linkage+disequilibrium+in+a+population+isolate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome-wide+distribution+of+background+linkage+disequilibrium+in+a+population+isolate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000814-Leitner1" id="pone.0000814-Leitner1"></a>Leitner T, Escanilla D, Franzen C, Uhlen M, Albert J (1996) Accurate reconstruction of a known HIV-1 transmission history by phylogenetic tree analysis. Proc Natl Acad Sci U S A  93: 10864–10869.  <ul class="find" data-citedArticleID="984642" data-doi="10.1073/pnas.93.20.10864"><li><a href="http://dx.doi.org/10.1073/pnas.93.20.10864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+reconstruction+of+a+known+HIV-1+transmission+history+by+phylogenetic+tree+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+reconstruction+of+a+known+HIV-1+transmission+history+by+phylogenetic+tree+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000814-Thomson1" id="pone.0000814-Thomson1"></a>Thomson MM, Najera R (2005) Molecular epidemiology of HIV-1 variants in the global AIDS pandemic: an update. AIDS Rev  7: 210–224.  <ul class="find" data-citedArticleID="984694"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+epidemiology+of+HIV-1+variants+in+the+global+AIDS+pandemic%3A+an+update.&amp;auth=&amp;atitle=Molecular+epidemiology+of+HIV-1+variants+in+the+global+AIDS+pandemic%3A+an+update." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+epidemiology+of+HIV-1+variants+in+the+global+AIDS+pandemic%3A+an+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+epidemiology+of+HIV-1+variants+in+the+global+AIDS+pandemic%3A+an+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000814-Mansky1" id="pone.0000814-Mansky1"></a>Mansky LM, Temin HM (1995) Lower in vivo mutation rate of human immunodeficiency virus type 1 than that predicted from the fidelity of purified reverse transcriptase. J Virol  69: 5087–5094.  <ul class="find" data-citedArticleID="984650"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+that+predicted+from+the+fidelity+of+purified+reverse+transcriptase.&amp;auth=&amp;atitle=Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+that+predicted+from+the+fidelity+of+purified+reverse+transcriptase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+that+predicted+from+the+fidelity+of+purified+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lower+in+vivo+mutation+rate+of+human+immunodeficiency+virus+type+1+than+that+predicted+from+the+fidelity+of+purified+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000814-Korber2" id="pone.0000814-Korber2"></a>Korber B, Theiler J, Wolinsky S (1998) Limitations of a molecular clock applied to considerations of the origin of HIV-1. Science  280: 1868–1871.  <ul class="find" data-citedArticleID="984638" data-doi="10.1126/science.280.5371.1868"><li><a href="http://dx.doi.org/10.1126/science.280.5371.1868" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Limitations+of+a+molecular+clock+applied+to+considerations+of+the+origin+of+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Limitations+of+a+molecular+clock+applied+to+considerations+of+the+origin+of+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000814-Jetzt1" id="pone.0000814-Jetzt1"></a>Jetzt AE, Yu H, Klarmann GJ, Ron Y, Preston BD, et al.  (2000) High rate of recombination throughout the human immunodeficiency virus type 1 genome. J Virol  74: 1234–1240.  <ul class="find" data-citedArticleID="984632" data-doi="10.1128/jvi.74.3.1234-1240.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.3.1234-1240.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+rate+of+recombination+throughout+the+human+immunodeficiency+virus+type+1+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+rate+of+recombination+throughout+the+human+immunodeficiency+virus+type+1+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000814-Zhuang1" id="pone.0000814-Zhuang1"></a>Zhuang J, Jetzt AE, Sun G, Yu H, Klarmann G, et al.  (2002) Human immunodeficiency virus type 1 recombination: rate, fidelity, and putative hot spots. J Virol  76: 11273–11282.  <ul class="find" data-citedArticleID="984702" data-doi="10.1128/jvi.76.22.11273-11282.2002"><li><a href="http://dx.doi.org/10.1128/jvi.76.22.11273-11282.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+type+1+recombination%3A+rate%2C+fidelity%2C+and+putative+hot+spots." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+type+1+recombination%3A+rate%2C+fidelity%2C+and+putative+hot+spots.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000814-Rhodes1" id="pone.0000814-Rhodes1"></a>Rhodes T, Wargo H, Hu WS (2003) High rates of human immunodeficiency virus type 1 recombination: near-random segregation of markers one kilobase apart in one round of viral replication. J Virol  77: 11193–11200.  <ul class="find" data-citedArticleID="984672" data-doi="10.1128/jvi.77.20.11193-11200.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.20.11193-11200.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+rates+of+human+immunodeficiency+virus+type+1+recombination%3A+near-random+segregation+of+markers+one+kilobase+apart+in+one+round+of+viral+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+rates+of+human+immunodeficiency+virus+type+1+recombination%3A+near-random+segregation+of+markers+one+kilobase+apart+in+one+round+of+viral+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000814-Wei1" id="pone.0000814-Wei1"></a>Wei X, Ghosh SK, Taylor ME, Johnson VA, Emini EA, et al.  (1995) Viral dynamics in human immunodeficiency virus type 1 infection. Nature  373: 117–122.  <ul class="find" data-citedArticleID="984696" data-doi="10.1038/373117a0"><li><a href="http://dx.doi.org/10.1038/373117a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Viral+dynamics+in+human+immunodeficiency+virus+type+1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Viral+dynamics+in+human+immunodeficiency+virus+type+1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000814-Ho1" id="pone.0000814-Ho1"></a>Ho DD, Neumann AU, Perelson AS, Chen W, Leonard JM, et al.  (1995) Rapid turnover of plasma virions and CD4 lymphocytes in HIV-1 infection. Nature  373: 123–126.  <ul class="find" data-citedArticleID="984626" data-doi="10.1038/373123a0"><li><a href="http://dx.doi.org/10.1038/373123a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+turnover+of+plasma+virions+and+CD4+lymphocytes+in+HIV-1+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+turnover+of+plasma+virions+and+CD4+lymphocytes+in+HIV-1+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000814-Perelson1" id="pone.0000814-Perelson1"></a>Perelson AS, Neumann AU, Markowitz M, Leonard JM, Ho DD (1996) HIV-1 dynamics in vivo: virion clearance rate, infected cell life-span, and viral generation time. Science  271: 1582–1586.  <ul class="find" data-citedArticleID="984664" data-doi="10.1126/science.271.5255.1582"><li><a href="http://dx.doi.org/10.1126/science.271.5255.1582" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+dynamics+in+vivo%3A+virion+clearance+rate%2C+infected+cell+life-span%2C+and+viral+generation+time." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+dynamics+in+vivo%3A+virion+clearance+rate%2C+infected+cell+life-span%2C+and+viral+generation+time.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000814-Doolittle1" id="pone.0000814-Doolittle1"></a>Doolittle RF (1994) Convergent evolution: the need to be explicit. Trends Biochem Sci  19: 15–18.  <ul class="find" data-citedArticleID="984608" data-doi="10.1016/0968-0004(94)90167-8"><li><a href="http://dx.doi.org/10.1016/0968-0004(94)90167-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Convergent+evolution%3A+the+need+to+be+explicit." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Convergent+evolution%3A+the+need+to+be+explicit.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000814-Felsenstein1" id="pone.0000814-Felsenstein1"></a>Felsenstein J (1978) Cases in Which Parsimony or Compatibility Methods Will Be Positively Misleading. Systematic Zoology  27: 401–410.  <ul class="find" data-citedArticleID="984612" data-doi="10.2307/2412923"><li><a href="http://dx.doi.org/10.2307/2412923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cases+in+Which+Parsimony+or+Compatibility+Methods+Will+Be+Positively+Misleading." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cases+in+Which+Parsimony+or+Compatibility+Methods+Will+Be+Positively+Misleading.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000814-Lemey1" id="pone.0000814-Lemey1"></a>Lemey P, Derdelinckx I, Rambaut A, Van Laethem K, Dumont S, et al.  (2005) Molecular footprint of drug-selective pressure in a human immunodeficiency virus transmission chain. J Virol  79: 11981–11989.  <ul class="find" data-citedArticleID="984644" data-doi="10.1128/jvi.79.18.11981-11989.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.18.11981-11989.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+footprint+of+drug-selective+pressure+in+a+human+immunodeficiency+virus+transmission+chain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+footprint+of+drug-selective+pressure+in+a+human+immunodeficiency+virus+transmission+chain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000814-Wollenberg1" id="pone.0000814-Wollenberg1"></a>Wollenberg KR, Atchley WR (2000) Separation of phylogenetic and functional associations in biological sequences by using the parametric bootstrap. Proc Natl Acad Sci U S A  97: 3288–3291.  <ul class="find" data-citedArticleID="984698" data-doi="10.1073/pnas.97.7.3288"><li><a href="http://dx.doi.org/10.1073/pnas.97.7.3288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Separation+of+phylogenetic+and+functional+associations+in+biological+sequences+by+using+the+parametric+bootstrap." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Separation+of+phylogenetic+and+functional+associations+in+biological+sequences+by+using+the+parametric+bootstrap.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000814-Noivirt1" id="pone.0000814-Noivirt1"></a>Noivirt O, Eisenstein M, Horovitz A (2005) Detection and reduction of evolutionary noise in correlated mutation analysis. Protein Eng Des Sel  18: 247–253.  <ul class="find" data-citedArticleID="984654" data-doi="10.1093/protein/gzi029"><li><a href="http://dx.doi.org/10.1093/protein/gzi029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+and+reduction+of+evolutionary+noise+in+correlated+mutation+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+and+reduction+of+evolutionary+noise+in+correlated+mutation+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000814-Hofacker1" id="pone.0000814-Hofacker1"></a>Hofacker IL, Stadler PF (1999) Automatic detection of conserved base pairing patterns in RNA virus genomes. Comput Chem  23: 401–414.  <ul class="find" data-citedArticleID="984628" data-doi="10.1016/s0097-8485(99)00013-3"><li><a href="http://dx.doi.org/10.1016/s0097-8485(99)00013-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+detection+of+conserved+base+pairing+patterns+in+RNA+virus+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+detection+of+conserved+base+pairing+patterns+in+RNA+virus+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000814-Shapiro1" id="pone.0000814-Shapiro1"></a>Shapiro B, Rambaut A, Pybus OG, Holmes EC (2006) A phylogenetic method for detecting positive epistasis in gene sequences and its application to RNA virus evolution. Mol Biol Evol  23: 1724–1730.  <ul class="find" data-citedArticleID="984678" data-doi="10.1093/molbev/msl037"><li><a href="http://dx.doi.org/10.1093/molbev/msl037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenetic+method+for+detecting+positive+epistasis+in+gene+sequences+and+its+application+to+RNA+virus+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenetic+method+for+detecting+positive+epistasis+in+gene+sequences+and+its+application+to+RNA+virus+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000814-Chen1" id="pone.0000814-Chen1"></a>Chen L, Perlina A, Lee CJ (2004) Positive selection detection in 40,000 human immunodeficiency virus (HIV) type 1 sequences automatically identifies drug resistance and positive fitness mutations in HIV protease and reverse transcriptase. J Virol  78: 3722–3732.  <ul class="find" data-citedArticleID="984602" data-doi="10.1128/jvi.78.7.3722-3732.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.7.3722-3732.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+selection+detection+in+40%2C000+human+immunodeficiency+virus+%28HIV%29+type+1+sequences+automatically+identifies+drug+resistance+and+positive+fitness+mutations+in+HIV+protease+and+reverse+transcriptase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+selection+detection+in+40%2C000+human+immunodeficiency+virus+%28HIV%29+type+1+sequences+automatically+identifies+drug+resistance+and+positive+fitness+mutations+in+HIV+protease+and+reverse+transcriptase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000814-The1" id="pone.0000814-The1"></a>The International HapMap Consortium (2005) A haplotype map of the human genome. Nature  437: 1299–1320.  <ul class="find" data-citedArticleID="984690" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000814-Balding1" id="pone.0000814-Balding1"></a>Balding DJ (2006) A tutorial on statistical methods for population association studies. Nat Rev Genet  7: 781–791.  <ul class="find" data-citedArticleID="984594" data-doi="10.1038/nrg1916"><li><a href="http://dx.doi.org/10.1038/nrg1916" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+tutorial+on+statistical+methods+for+population+association+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+tutorial+on+statistical+methods+for+population+association+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000814-Pritchard1" id="pone.0000814-Pritchard1"></a>Pritchard JK, Przeworski M (2001) Linkage disequilibrium in humans: models and data. Am J Hum Genet  69: 1–14.  <ul class="find" data-citedArticleID="984666" data-doi="10.1086/321275"><li><a href="http://dx.doi.org/10.1086/321275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+in+humans%3A+models+and+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+in+humans%3A+models+and+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000814-Reich1" id="pone.0000814-Reich1"></a>Reich DE, Cargill M, Bolk S, Ireland J, Sabeti PC, et al.  (2001) Linkage disequilibrium in the human genome. Nature  411: 199–204.  <ul class="find" data-citedArticleID="984668" data-doi="10.1038/35075590"><li><a href="http://dx.doi.org/10.1038/35075590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000814-Dawson1" id="pone.0000814-Dawson1"></a>Dawson E, Abecasis GR, Bumpstead S, Chen Y, Hunt S, et al.  (2002) A first-generation linkage disequilibrium map of human chromosome 22. Nature  418: 544–548.  <ul class="find" data-citedArticleID="984606" data-doi="10.1038/nature00864"><li><a href="http://dx.doi.org/10.1038/nature00864" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+first-generation+linkage+disequilibrium+map+of+human+chromosome+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+first-generation+linkage+disequilibrium+map+of+human+chromosome+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000814-Stephens1" id="pone.0000814-Stephens1"></a>Stephens JC, Schneider JA, Tanguay DA, Choi J, Acharya T, et al.  (2001) Haplotype variation and linkage disequilibrium in 313 human genes. Science  293: 489–493.  <ul class="find" data-citedArticleID="984684" data-doi="10.1126/science.1059431"><li><a href="http://dx.doi.org/10.1126/science.1059431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haplotype+variation+and+linkage+disequilibrium+in+313+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haplotype+variation+and+linkage+disequilibrium+in+313+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000814-Shafer1" id="pone.0000814-Shafer1"></a>Shafer RW (2002) Genotypic testing for human immunodeficiency virus type 1 drug resistance. Clin Microbiol Rev  15: 247–277.  <ul class="find" data-citedArticleID="984676" data-doi="10.1128/cmr.15.2.247-277.2002"><li><a href="http://dx.doi.org/10.1128/cmr.15.2.247-277.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genotypic+testing+for+human+immunodeficiency+virus+type+1+drug+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genotypic+testing+for+human+immunodeficiency+virus+type+1+drug+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000814-Clavel1" id="pone.0000814-Clavel1"></a>Clavel F, Hance AJ (2004) HIV drug resistance. N Engl J Med  350: 1023–1035.  <ul class="find" data-citedArticleID="984604" data-doi="10.1056/nejmra025195"><li><a href="http://dx.doi.org/10.1056/nejmra025195" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+drug+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+drug+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000814-Smith1" id="pone.0000814-Smith1"></a>Smith JM, Haigh J (1974) Hitch-Hiking Effect of a Favorable Gene. Genetical Research  23: 23–35.  <ul class="find" data-citedArticleID="984682" data-doi="10.1017/s0016672300014634"><li><a href="http://dx.doi.org/10.1017/s0016672300014634" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hitch-Hiking+Effect+of+a+Favorable+Gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hitch-Hiking+Effect+of+a+Favorable+Gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000814-Nielsen1" id="pone.0000814-Nielsen1"></a>Nielsen R, Williamson S, Kim Y, Hubisz MJ, Clark AG, et al.  (2005) Genomic scans for selective sweeps using SNP data. Genome Research  15: 1566–1575.  <ul class="find" data-citedArticleID="984652" data-doi="10.1101/gr.4252305"><li><a href="http://dx.doi.org/10.1101/gr.4252305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+scans+for+selective+sweeps+using+SNP+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+scans+for+selective+sweeps+using+SNP+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000814-Johnson1" id="pone.0000814-Johnson1"></a>Johnson VA, Brun-Vezinet F, Clotet B, Kuritzkes DR, Pillay D, et al.  (2006) Update of the drug resistance mutations in HIV-1: Fall 2006. Top HIV Med  14: 125–130.  <ul class="find" data-citedArticleID="984634"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006.&amp;auth=&amp;atitle=Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Update+of+the+drug+resistance+mutations+in+HIV-1%3A+Fall+2006.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000814-Bonhoeffer1" id="pone.0000814-Bonhoeffer1"></a>Bonhoeffer S, Chappey C, Parkin NT, Whitcomb JM, Petropoulos CJ (2004) Evidence for positive epistasis in HIV-1. Science  306: 1547–1550.  <ul class="find" data-citedArticleID="984598" data-doi="10.1126/science.1101786"><li><a href="http://dx.doi.org/10.1126/science.1101786" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+positive+epistasis+in+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+positive+epistasis+in+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000814-Ane1" id="pone.0000814-Ane1"></a>Ane C, Burleigh JG, McMahon MM, Sanderson MJ (2005) Covarion structure in plastid genome evolution: a new statistical test. Mol Biol Evol  22: 914–924.  <ul class="find" data-citedArticleID="984592" data-doi="10.1093/molbev/msi076"><li><a href="http://dx.doi.org/10.1093/molbev/msi076" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Covarion+structure+in+plastid+genome+evolution%3A+a+new+statistical+test." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Covarion+structure+in+plastid+genome+evolution%3A+a+new+statistical+test.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000814-Pan1" id="pone.0000814-Pan1"></a>Pan C, Kim J, Chen L, Wang Q, Lee C (2007) The HIV positive selection mutation database. Nucleic Acids Res  35: D371–375.  <ul class="find" data-citedArticleID="984662" data-doi="10.1093/nar/gkl855"><li><a href="http://dx.doi.org/10.1093/nar/gkl855" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HIV+positive+selection+mutation+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HIV+positive+selection+mutation+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000814-Rhee1" id="pone.0000814-Rhee1"></a>Rhee SY, Gonzales MJ, Kantor R, Betts BJ, Ravela J, et al.  (2003) Human immunodeficiency virus reverse transcriptase and protease sequence database. Nucleic Acids Res  31: 298–303.  <ul class="find" data-citedArticleID="984670" data-doi="10.1093/nar/gkg100"><li><a href="http://dx.doi.org/10.1093/nar/gkg100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+reverse+transcriptase+and+protease+sequence+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+reverse+transcriptase+and+protease+sequence+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000814-Fisher1" id="pone.0000814-Fisher1"></a>Fisher RA (1922) On the interpretation of x(2) from contingency tables, and the calculation of P. Journal of the Royal Statistical Society  85: 87–94.  <ul class="find" data-citedArticleID="984614" data-doi="10.2307/2340521"><li><a href="http://dx.doi.org/10.2307/2340521" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+interpretation+of+x%282%29+from+contingency+tables%2C+and+the+calculation+of+P." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+interpretation+of+x%282%29+from+contingency+tables%2C+and+the+calculation+of+P.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000814-Agresti1" id="pone.0000814-Agresti1"></a>Agresti A (1992) A survey of exact inference for contingency tables. Statistical Science  7: 131–177.  <ul class="find" data-citedArticleID="984588" data-doi="10.1214/ss/1177011462"><li><a href="http://dx.doi.org/10.1214/ss/1177011462" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+survey+of+exact+inference+for+contingency+tables." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+survey+of+exact+inference+for+contingency+tables.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000814-Brown1" id="pone.0000814-Brown1"></a>Brown AHD (1975) Sample Sizes Required to Detect Linkage Disequilibrium between 2 or 3 Loci. Theoretical Population Biology  8: 184–201.  <ul class="find" data-citedArticleID="984600" data-doi="10.1016/0040-5809(75)90031-3"><li><a href="http://dx.doi.org/10.1016/0040-5809(75)90031-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sample+Sizes+Required+to+Detect+Linkage+Disequilibrium+between+2+or+3+Loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sample+Sizes+Required+to+Detect+Linkage+Disequilibrium+between+2+or+3+Loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000814-Lewontin1" id="pone.0000814-Lewontin1"></a>Lewontin RC (1988) On Measures of Gametic Disequilibrium. Genetics  120: 849–852.  <ul class="find" data-citedArticleID="984646"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+Measures+of+Gametic+Disequilibrium.&amp;auth=&amp;atitle=On+Measures+of+Gametic+Disequilibrium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+Measures+of+Gametic+Disequilibrium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+Measures+of+Gametic+Disequilibrium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000814-Ohta1" id="pone.0000814-Ohta1"></a>Ohta T, Kimura M (1970) Development of Associative Overdominance through Linkage Disequilibrium in Finite Populations. Genetical Research  16: 165–&amp;.  <ul class="find" data-citedArticleID="984656" data-doi="10.1017/s0016672300002391"><li><a href="http://dx.doi.org/10.1017/s0016672300002391" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+Associative+Overdominance+through+Linkage+Disequilibrium+in+Finite+Populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+Associative+Overdominance+through+Linkage+Disequilibrium+in+Finite+Populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.XML" value="100912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.PDF" value="368237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g001.PNG_L" value="743610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g001.PNG_M" value="116112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g001.PNG_S" value="17680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g001.TIF" value="1269330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g001.PNG_I" value="66377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g002.PNG_L" value="865163"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g002.PNG_M" value="100823"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g002.PNG_S" value="13075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g002.TIF" value="1360302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g002.PNG_I" value="35845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g003.PNG_L" value="70197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g003.PNG_M" value="49937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g003.PNG_S" value="9326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g003.TIF" value="332832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g003.PNG_I" value="17100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g004.PNG_L" value="1044449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g004.PNG_M" value="212482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g004.PNG_S" value="15678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g004.TIF" value="1613990"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g004.PNG_I" value="71442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g005.PNG_L" value="3182174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g005.PNG_M" value="169553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g005.PNG_S" value="10121"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g005.TIF" value="5133406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.g005.PNG_I" value="53867"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.s001.TIF" value="1395694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.s002.TIF" value="1417366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.s003.TIF" value="1108476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.s004.TIF" value="363440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.s005.TIF" value="1428758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000814.s006.TIF" value="690570"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000814&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000814" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000814&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000814&volume=&issue=&title=Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase&author_name=Qi%20Wang%2C%20Christopher%20Lee&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000814" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814&amp;t=Distinguishing%20Functional%20Amino%20Acid%20Covariation%20from%20Background%20Linkage%20Disequilibrium%20in%20HIV%20Protease%20and%20Reverse%20Transcriptase" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814&title=Distinguishing%20Functional%20Amino%20Acid%20Covariation%20from%20Background%20Linkage%20Disequilibrium%20in%20HIV%20Protease%20and%20Reverse%20Transcriptase&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814&amp;title=Distinguishing%20Functional%20Amino%20Acid%20Covariation%20from%20Background%20Linkage%20Disequilibrium%20in%20HIV%20Protease%20and%20Reverse%20Transcriptase" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000814&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Distinguishing Functional Amino Acid Covariation from Background Linkage Disequilibrium in HIV Protease and Reverse Transcriptase';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000814';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Distinguishing%20Functional%20Amino%20Acid%20Covariation%20from%20Background%20Linkage%20Disequilibrium%20in%20HIV%20Protease%20and%20Reverse%20Transcriptase http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000814" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000814" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+analysis%22" title="Search for articles in the subject area:'Amino acid analysis'"><div class="flagText">Amino acid analysis</div></a>
              <div data-categoryid="47251" data-articleid="24466"
                   data-categoryname="Amino acid analysis"
                   class="flagImage" title="Flag 'Amino acid analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drug+interactions%22" title="Search for articles in the subject area:'Drug interactions'"><div class="flagText">Drug interactions</div></a>
              <div data-categoryid="46103" data-articleid="24466"
                   data-categoryname="Drug interactions"
                   class="flagImage" title="Flag 'Drug interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Drug+therapy%22" title="Search for articles in the subject area:'Drug therapy'"><div class="flagText">Drug therapy</div></a>
              <div data-categoryid="18461" data-articleid="24466"
                   data-categoryname="Drug therapy"
                   class="flagImage" title="Flag 'Drug therapy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV%22" title="Search for articles in the subject area:'HIV'"><div class="flagText">HIV</div></a>
              <div data-categoryid="42663" data-articleid="24466"
                   data-categoryname="HIV"
                   class="flagImage" title="Flag 'HIV' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="24466"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="24466"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteases%22" title="Search for articles in the subject area:'Proteases'"><div class="flagText">Proteases</div></a>
              <div data-categoryid="19447" data-articleid="24466"
                   data-categoryname="Proteases"
                   class="flagImage" title="Flag 'Proteases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Silent+mutation%22" title="Search for articles in the subject area:'Silent mutation'"><div class="flagText">Silent mutation</div></a>
              <div data-categoryid="35777" data-articleid="24466"
                   data-categoryname="Silent mutation"
                   class="flagImage" title="Flag 'Silent mutation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3864'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3929'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8163&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
