

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000656"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000656" />

    <meta name="citation_title" content="Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates"/>
    <meta itemprop="name" content="Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates"/>

      <meta name="citation_author" content="Emmanuel J. Chang"/>
            <meta name="citation_author_institution" content="Department of Chemistry, York College of the City University of New York, Jamaica, New York, United States of America"/>
            <meta name="citation_author_institution" content="Laboratory of Mass Spectrometry and Gaseous Ion Chemistry, The Rockefeller University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Rashida Begum"/>
            <meta name="citation_author_institution" content="Department of Chemistry, York College of the City University of New York, Jamaica, New York, United States of America"/>
      <meta name="citation_author" content="Brian T. Chait"/>
            <meta name="citation_author_institution" content="Laboratory of Mass Spectrometry and Gaseous Ion Chemistry, The Rockefeller University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Terry Gaasterland"/>
            <meta name="citation_author_institution" content="Scripps Institute of Oceanography, University of California at San Diego, La Jolla, California, United States of America"/>

    <meta name="citation_date" content="2007/8/1"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000656.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e656"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The regulation of protein function by multisite phosphorylation–a 25 year update.; citation_author=P Cohen; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=1; citation_pages=596-601; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Signaling–2000 and beyond.; citation_author=T Hunter; citation_journal_title=Cell; citation_volume=100; citation_number=2; citation_pages=113-127; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation- and ubiquitin-dependent degradation of the cyclin-dependent kinase inhibitor Far1p in budding yeast.; citation_author=S Henchoz; citation_author=Y Chi; citation_author=B Catarin; citation_author=I Herskowitz; citation_author=RJ Deshaies; citation_journal_title=Genes Dev; citation_volume=11; citation_number=3; citation_pages=3046-3060; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The structural basis for specificity of substrate and recruitment peptides for cyclin-dependent kinases.; citation_author=N Brown; citation_author=M Noble; citation_author=J Endicott; citation_author=L Johnson; citation_journal_title=Nat Cell Biology; citation_volume=1; citation_number=4; citation_pages=438-443; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Starting the cell cycle: what's the point?; citation_author=FR Cross; citation_journal_title=Curr Opin Cell Biol; citation_volume=7; citation_number=5; citation_pages=790-797; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Testing a mathematical model of the yeast cell cycle.; citation_author=FR Cross; citation_author=V Archambault; citation_author=M Miller; citation_author=M Klovstad; citation_journal_title=Mol Biol Cell; citation_volume=13; citation_number=6; citation_pages=52-70; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation by cyclin B-Cdk underlies release of mitotic exit activator Cdc14 from the nucleolus.; citation_author=R Azzam; citation_author=S Chen; citation_author=W Shou; citation_author=A Mah; citation_author=G Alexandru; citation_journal_title=Science; citation_volume=305; citation_number=7; citation_pages=516-519; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometry-based methods for phosphorylation site mapping of hyperphosphorylated proteins applied to Net1, a regulator of exit from mitosis in yeast.; citation_author=S Loughrey Chen; citation_author=MJ Huddleston; citation_author=W Shou; citation_author=RJ Deshaies; citation_author=RS Annan; citation_journal_title=Mol Cell Proteomics; citation_volume=1; citation_number=8; citation_pages=186-196; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence.; citation_author=N Blom; citation_author=T Sicheritz-Ponten; citation_author=R Gupta; citation_author=S Gammeltoft; citation_author=S Brunak; citation_journal_title=Proteomics; citation_volume=4; citation_number=9; citation_pages=1633-1649; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Kinomic: methods for deciphering the kinome.; citation_author=S Johnson; citation_author=T Hunter; citation_journal_title=Nat Methods; citation_volume=2; citation_number=10; citation_pages=17-25; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=How do protein kinases recognize their substrates?; citation_author=L Pinna; citation_author=M Ruzzene; citation_journal_title=Biochimica et Biophysica Acta; citation_volume=1314; citation_number=11; citation_pages=191-225; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cellular substrates of p34(cdc2) and its companion cyclin-dependent kinases.; citation_author=EA Nigg; citation_journal_title=Trends Cell Biol; citation_volume=3; citation_number=12; citation_pages=296-301; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=PROSITE: a dictionary of sites and patterns in proteins.; citation_author=A Bairoch; citation_journal_title=Nucleic Acids Res; citation_volume=19; citation_number=13; citation_pages=2241-2245; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=ELM server: A new resource for investigating short functional sites in modular eukarotic proteins.; citation_author=P Puntervoll; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=14; citation_pages=3625-3630; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The PROSITE database, its status in 2002.; citation_author=L Falquet; citation_author=M Pagni; citation_author=P Bucher; citation_author=N Hulo; citation_author=C Sigrist; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=15; citation_pages=235-238; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Scansite 2.0: Proteome-wide prediction of cell signaling interactions using short sequence motifs.; citation_author=J Obenauer; citation_author=LC Cantley; citation_author=M Yaffe; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=16; citation_pages=3635-3641; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A motif-based profile scanning approach for genome-wide prediction of signaling pathways.; citation_author=M Yaffe; citation_author=G Leparc; citation_author=J Lai; citation_author=T Obata; citation_author=S Volinia; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=17; citation_pages=348-355; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=KinasePhos: a web tool for identifying protein kinase-specific phosphorylation sites.; citation_author=H Huang; citation_author=T Lee; citation_author=S Tzeng; citation_author=J Horng; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=18; citation_pages=W226-W229; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Incorporating hidden Markov models for identifying protein kinase-specific phosphorylation sites.; citation_author=H Huang; citation_author=T Lee; citation_author=S Tzeng; citation_author=L Wu; citation_author=J Horng; citation_journal_title=J Comput Chem; citation_volume=26; citation_number=19; citation_pages=1032-1041; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Predicting the phosphorylation sites using hidden Markov models and machine learning methods.; citation_author=P Senawongse; citation_author=A Dalby; citation_author=Z Yang; citation_journal_title=J Chem Inf Model; citation_volume=45; citation_number=20; citation_pages=1147-1152; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sequence and structure-based prediction of eukaryotic protein phosphorylation sites.; citation_author=N Blom; citation_author=S Gammeltoft; citation_author=S Brunak; citation_journal_title=J Mol Biol; citation_volume=294; citation_number=21; citation_pages=1351-1362; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Artificial neural networks for molecular sequence analysis.; citation_author=C Wu; citation_journal_title=Comput Chem; citation_volume=21; citation_number=22; citation_pages=237-256; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Structural basis and prediction of substrate specificity in protein serine/threonine kinases.; citation_author=R Brinkworth; citation_author=R Breinl; citation_author=B Kobe; citation_journal_title=Proc Natl Acad Sci, USA; citation_volume=100; citation_number=23; citation_pages=74-79; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The cyclin family of budding yeast: abundant use of a good idea.; citation_author=B Andrews; citation_author=V Measday; citation_journal_title=Trends in Genetics; citation_volume=14; citation_number=24; citation_pages=66-72; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Control of the yeast cell cycle by the Cdc28 protein kinase.; citation_author=K Nasmyth; citation_journal_title=Curr Opin in Cell Biol; citation_volume=5; citation_number=25; citation_pages=166-179; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of cyclin-dependent kinase regulation: structures of Cdks, their cyclin activators and Cip and INK4 inhibitors.; citation_author=N Pavletich; citation_journal_title=J Mol Biol; citation_volume=287; citation_number=26; citation_pages=821-828; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cyclin-dependent kinases: engines, clocks and microprocessors.; citation_author=DO Morgan; citation_journal_title=Annu Rev Cell Dev Biol; citation_volume=13; citation_number=27; citation_pages=261-291; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A predictive scale for evaluating cyclin-dependent kinase substrates. A comparison of p34cdc2 and p33cdk2.; citation_author=J Holmes; citation_author=M Solomon; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=28; citation_pages=25240-25246; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Use of an oriented peptide library to determine the optimal substrates of protein kinases.; citation_author=Z Songyang; citation_author=S Blechner; citation_author=N Hoagland; citation_author=MF Hoekstra; citation_author=H Piwnica-Worms; citation_journal_title=Curr Biol; citation_volume=4; citation_number=29; citation_pages=973-982; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Clustering of phosphorylation site recognition motifs can be exploited to predict the targets of cyclin-dependent kinase.; citation_author=A Moses; citation_author=J Heriche; citation_author=R Durbin; citation_journal_title=Genome Biol; citation_volume=8; citation_number=30; citation_pages=R23; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of Sic1p by G1 Cdk required for its degradation and entry into S phase.; citation_author=R Verma; citation_author=RS Annan; citation_author=MJ Huddleston; citation_author=SA Carr; citation_author=G Reynard; citation_journal_title=Science; citation_volume=278; citation_number=31; citation_pages=455-460; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Inhibitory phosphorylation of the APC regulator Hct1 is controlled by the kinase Cdc28 and the phosphatase Cdc14.; citation_author=S Jaspersen; citation_author=C JF; citation_author=D Morgan; citation_journal_title=Curr Biol; citation_volume=9; citation_number=32; citation_pages=227-236; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Pheromone-dependent G1 cell cycle arrest requires Far1 phosphorylation, but may not involve inhibition of Cdc28-Cln2, in vivo.; citation_author=A Gartner; citation_author=A Jovanovic; citation_author=D Jeoung; citation_author=S Bourlat; citation_author=FR Cross; citation_journal_title=Mol Cell Biol; citation_volume=18; citation_number=33; citation_pages=3681-3691; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The role of phosphorylation and the Cdc28 protein kinase in cell cycle-regulated nuclear import of the S. cerevisiae transcription factor SWI5.; citation_author=T Moll; citation_author=G Tebb; citation_author=U Surana; citation_author=H Robitsch; citation_author=K Nasmyth; citation_journal_title=Cell; citation_volume=66; citation_number=34; citation_pages=743-758; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A conserved cyclin-binding domain determines functional interplay between anaphase-promoting complex-Cdh1 and cyclin A-Cdk2 during cell cycle progression.; citation_author=C Sorensen; citation_author=C Lukas; citation_author=E Kramer; citation_author=J-M Oeters; citation_author=J Bartek; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=35; citation_pages=3692-3703; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A bipartitie substrate recognition motif for cyclin-dependent kinases.; citation_author=D Takeda; citation_author=J Wohlschlegel; citation_author=A Dutta; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=36; citation_pages=1993-1997; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Multi-site phosphorylation of Pho4 by the cyclin-Cdk Pho80-Pho85 is semi-processive with site preferences.; citation_author=D Jeffery; citation_author=M Springer; citation_author=D King; citation_author=E O'Shea; citation_journal_title=J Mol Biol; citation_volume=306; citation_number=37; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Analysis of Protein Phosphorylation by Hypothesis-Driven Multiple-Stage Mass Spectrometry.; citation_author=EJ Chang; citation_author=V Archambault; citation_author=DT McLachlin; citation_author=AN Krutchinsky; citation_author=BT Chait; citation_journal_title=Anal Chem; citation_volume=76; citation_number=38; citation_pages=4472-4483; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multisite phosphorylation of a Cdk inhibitor sets a threshold for the onset of DNA replication.; citation_author=P Nash; citation_author=X Tang; citation_author=S Orlicky; citation_author=Q Chen; citation_author=F Gerlter; citation_journal_title=Nature; citation_volume=414; citation_number=39; citation_pages=514-521; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Accurate quantitation of protein expression and site-specific phosphorylation.; citation_author=Y Oda; citation_author=K Huang; citation_author=FR Cross; citation_author=D Cowburn; citation_author=BT Chait; citation_journal_title=Proc Natl Acad Sci, USA; citation_volume=96; citation_number=40; citation_pages=6591-6596; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Distinct regions of the Swi5 and Ace2 transcription factors are required for specific gene activation.; citation_author=H McBride; citation_author=Y Yu; citation_author=D Stillmain; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=41; citation_pages=21029-21036; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Interaction between yeast Cdc6 protein and B-type cyclin/Cdc28 kinases.; citation_author=S Elasser; citation_author=F Lou; citation_author=B Wang; citation_author=J Campbell; citation_author=A Jong; citation_journal_title=Mol Biol Cell; citation_volume=7; citation_number=42; citation_pages=1723-1735; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cyclin-dependent kinases prevent DNA re-replication through multiple mechanisms.; citation_author=VQ Nguyen; citation_author=C Co; citation_author=J Li; citation_journal_title=Nature; citation_volume=411; citation_number=43; citation_pages=1068-1073; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=S-Cdk-dependent phosphorylation of Sld2 essential fo chromosomal DNA replication in budding yeast.; citation_author=H Masumoto; citation_author=S Muramatsu; citation_author=Y Kamimura; citation_author=H Araki; citation_journal_title=Nature; citation_volume=415; citation_number=44; citation_pages=651-655; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Regulation of transcription at the Saccharomyces cerevisiae start transition by Stb1, a Swi6-binding protein.; citation_author=Y Ho; citation_author=M Costanzo; citation_author=L Moore; citation_author=R Kobayashi; citation_author=B Andrews; citation_journal_title=Mol Biol Cell; citation_volume=19; citation_number=45; citation_pages=5267-5278; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cell cycle- and Cln2-Cdc2p-dependent phosphroylation of the Yeast Ste20p protein kinase.; citation_author=C Wu; citation_author=T Leeuw; citation_author=E Leberer; citation_author=D Thomas; citation_author=M Whiteway; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=46; citation_pages=28107-28115; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Targets of the cyclin-dependent kinase Cdk1.; citation_author=JA Ubersax; citation_author=EL Woodbury; citation_author=PN Quang; citation_author=M Paraz; citation_author=JD Blethrow; citation_journal_title=Nature; citation_volume=425; citation_number=47; citation_pages=859-864; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Targeted proteomic study of the Cyclin-Cdk module.; citation_author=V Archambault; citation_author=EJ Chang; citation_author=DB J; citation_author=FR Cross; citation_author=BT Chait; citation_journal_title=Mol Cell; citation_volume=14; citation_number=48; citation_pages=699-711; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of sic1, a cyclin-dependent kianse (Cdk) inhibitor, by Cdk including Pho85 kinase is required for its prompt degradation.; citation_author=M Nishizawa; citation_author=M Kawasumi; citation_author=M Fujino; citation_author=T-e A; citation_journal_title=Mol Biol Cell; citation_volume=9; citation_number=49; citation_pages=2393-2405; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The role of Thr160 phosphorylation of Cdk2 in substrate recognition.; citation_author=J Holmes; citation_author=M Solomon; citation_journal_title=Eur J Biochem; citation_volume=268; citation_number=50; citation_pages=4647-4652; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Overlapping and distinct roles of the duplicated yeast transcription factors Ace2p and Swi5p.; citation_author=M Doolin; citation_author=A Johnson; citation_author=L Johnson; citation_author=G Butler; citation_journal_title=Mol Microbiol; citation_volume=40; citation_number=51; citation_pages=422-432; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cyclin-dependent kinases prevent DNA re-replication through multiple mechanisms.; citation_author=VQ Nguyen; citation_author=C Co; citation_author=JJ Li; citation_journal_title=Nature; citation_volume=411; citation_number=52; citation_pages=1068-1073; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Recruitment of Thr 319-phosphorylated Ndd1p to the FHA domain of Fkh2p requires Clb kinase activity: a mechanism for CLB cluster gene activation.; citation_author=D Reynolds; citation_author=B Shi; citation_author=C McClean; citation_author=F Katsis; citation_author=B Kemp; citation_journal_title=Genes Dev; citation_volume=17; citation_number=53; citation_pages=1789-1802; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation of the mitotic regulator Pds1/securin by Cdc28 is required for efficient nuclear localization of Esp1/separase.; citation_author=R Agarwal; citation_author=O Cohen-Fix; citation_journal_title=Genes Dev; citation_volume=16; citation_number=54; citation_pages=1371-1382; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detection of secreted peptides by using hypothesis-driven multistage mass spectrometry.; citation_author=M Kalkum; citation_author=GJ Lyon; citation_author=BT Chait; citation_journal_title=Proc Natl Acad Sci, USA; citation_volume=100; citation_number=55; citation_pages=2795-27800; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Protein structure prediction in genomics.; citation_author=D Jones; citation_journal_title=Brief Bioinform; citation_volume=2; citation_number=56; citation_pages=111-125; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Tools for comparative protein structure modeling and analysis.; citation_author=N Eswar; citation_author=B John; citation_author=N Mirkovic; citation_author=A Fiser; citation_author=V Ilyin; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=57; citation_pages=3375-3380; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Phosphorylation by Cdc28 activates the Cdc20- dependent activity of the anaphase promoting complex.; citation_author=A Rudner; citation_author=A Murray; citation_journal_title=J Cell Biol; citation_volume=149; citation_number=58; citation_pages=1377-1390; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification and characterization of protein subcomplexes in yeast.; citation_author=J Hollunder; citation_author=A Beyer; citation_author=T Wilhelm; citation_journal_title=Proteomics; citation_volume=5; citation_number=59; citation_pages=2082-2089; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry.; citation_author=Y Ho; citation_journal_title=Nature; citation_volume=415; citation_number=60; citation_pages=180-183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Functional organization of the yeast proteome by systematic analysis of protein complexes.; citation_author=A Gavin; citation_journal_title=Nature; citation_volume=415; citation_number=61; citation_pages=141-147; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phosphate groups as substrate determinants for casein kinase I action.; citation_author=H Flotow; citation_author=PR Graves; citation_author=A Wang; citation_author=CJ Fiol; citation_author=RW Roeske; citation_journal_title=J Biol Chem; citation_volume=265; citation_number=62; citation_pages=14264-14269; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Role of acidic residues as substrate determinants for casein kinase I. J Biol Chem; citation_author=H Flotow; citation_author=PJ Roach; citation_volume=266; citation_number=63; citation_pages=3724-3727; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Formation of protein kinase recognition sites by covalent modification of the substrate. Molecular mechanism for the synergistic action of casein kinase II and glycogen synthase kinase 3.; citation_author=CJ Fiol; citation_author=A Marenholz; citation_author=Y Wang; citation_author=R Roeske; citation_author=PJ Roach; citation_journal_title=J Biol Chem; citation_volume=262; citation_number=64; citation_pages=14042-14048; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=A Common Phosphate Binding Site Explains the Unique Substrate Specificity of GSK3 and Its Inactivation by Phosphorylation.; citation_author=S Frame; citation_author=P Cohen; citation_author=R Biondi; citation_journal_title=Mol Cell; citation_volume=7; citation_number=65; citation_pages=1321-1327; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mechanistic Insights into Sky1p, a Yeast Homologue of the Mammalian SR Protein Kinases.; citation_author=B Aubol; citation_author=B Nolen; citation_author=D Vu; citation_author=G Gourisankar; citation_author=J Adams; citation_journal_title=Biochemistry; citation_volume=41; citation_number=66; citation_pages=10002-10009; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mass spectrometric and kinetic analysis of ASF/SF2 phosphorylation by SRPK1 and Clk/Sty.; citation_author=A Velazquez-Dones; citation_author=J Hagopian; citation_author=C Ma; citation_author=X Zhong; citation_author=H Zhou; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=67; citation_pages=41761; citation_date=2005; " />
      <meta name="citation_reference" content="citation_number=68; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs.; citation_author=C Ramu; citation_author=H Sugawar; citation_author=T Koike; citation_author=R Lopez; citation_author=T Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=69; citation_pages=3497-3500; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates"/>
    <meta name="twitter:description" content="Protein phosphorylation, mediated by a family of enzymes called cyclin-dependent kinases (Cdks), plays a central role in the cell-division cycle of eukaryotes. Phosphorylation by Cdks directs the cell cycle by modifying the function of regulators of key processes such as DNA replication and mitotic progression. Here, we present a novel computational procedure to predict substrates of the cyclin-dependent kinase Cdc28 (Cdk1) in the Saccharomyces cerevisiae. Currently, most computational phosphorylation site prediction procedures focus solely on local sequence characteristics. In the present procedure, we model Cdk substrates based on both local and global characteristics of the substrates. Thus, we define the local sequence motifs that represent the Cdc28 phosphorylation sites and subsequently model clustering of these motifs within the protein sequences. This restraint reflects the observation that many known Cdk substrates contain multiple clustered phosphorylation sites. The present strategy defines a subset of the proteome that is highly enriched for Cdk substrates, as validated by comparing it to a set of bona fide, published, experimentally characterized Cdk substrates which was to our knowledge, comprehensive at the time of writing. To corroborate our model, we compared its predictions with three experimentally independent Cdk proteomic datasets and found significant overlap. Finally, we directly detected in vivo phosphorylation at Cdk motifs for selected putative substrates using mass spectrometry."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000656.g002"/>

  <meta property="og:title" content="Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1460'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8545'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=4468&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000656">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000656" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000656&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Emmanuel J. Chang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:echang@york.cuny.edu">echang@york.cuny.edu</a></p>

                <p>Affiliations:
                  Department of Chemistry, York College of the City University of New York, Jamaica, New York, United States of America, 
                  Laboratory of Mass Spectrometry and Gaseous Ion Chemistry, The Rockefeller University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rashida Begum, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemistry, York College of the City University of New York, Jamaica, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Brian T. Chait, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Mass Spectrometry and Gaseous Ion Chemistry, The Rockefeller University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Terry Gaasterland
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Scripps Institute of Oceanography, University of California at San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 01, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000656</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000656-t001" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000656-g001" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000656-t002" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000656-g002" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000656">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000656" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Protein phosphorylation, mediated by a family of enzymes called cyclin-dependent kinases (Cdks), plays a central role in the cell-division cycle of eukaryotes. Phosphorylation by Cdks directs the cell cycle by modifying the function of regulators of key processes such as DNA replication and mitotic progression. Here, we present a novel computational procedure to predict substrates of the cyclin-dependent kinase Cdc28 (Cdk1) in the <em>Saccharomyces cerevisiae</em>. Currently, most computational phosphorylation site prediction procedures focus solely on local sequence characteristics. In the present procedure, we model Cdk substrates based on both local and global characteristics of the substrates. Thus, we define the local sequence motifs that represent the Cdc28 phosphorylation sites and subsequently model clustering of these motifs within the protein sequences. This restraint reflects the observation that many known Cdk substrates contain multiple clustered phosphorylation sites. The present strategy defines a subset of the proteome that is highly enriched for Cdk substrates, as validated by comparing it to a set of <em>bona fide</em>, published, experimentally characterized Cdk substrates which was to our knowledge, comprehensive at the time of writing. To corroborate our model, we compared its predictions with three experimentally independent Cdk proteomic datasets and found significant overlap. Finally, we directly detected <em>in vivo</em> phosphorylation at Cdk motifs for selected putative substrates using mass spectrometry.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Chang EJ, Begum R, Chait BT, Gaasterland T (2007) Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates. PLoS ONE 2(8):
          e656.
            doi:10.1371/journal.pone.0000656</p><p><strong>Academic Editor: </strong>Raya Khanin, University of Glasgow, United Kingdom</p><p><strong>Received:</strong> April 23, 2007; <strong>Accepted:</strong> June 24, 2007; <strong>Published:</strong> August 1, 2007</p><p><strong>Copyright:</strong> © 2007 Chang et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>EJC was supported in part by a Burroughs-Wellcome Interfaces in Science fellowship, and a Graduate Research Technology Initiative grant from the State of New York. BTC acknowledges support from the National Institutes of Health (Grant RR00862).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The reversible modification of proteins by covalent addition and removal of phosphate is a major means by which cellular function is regulated <a href="#pone.0000656-Cohen1">[1]</a>, <a href="#pone.0000656-Hunter1">[2]</a>. The addition of phosphate, which is a sterically bulky and negatively charged moiety, can alter a protein's biochemical properties and affect its structure and activity. For example, phosphorylation can create docking sites to mediate protein interactions <a href="#pone.0000656-Hunter1">[2]</a>, modify signal sequences on proteins to regulate their subcellular localization <a href="#pone.0000656-Henchoz1">[3]</a>, or activate enzymes by bringing their active sites into proper alignment <a href="#pone.0000656-Brown1">[4]</a>. Networks of phosphorylation-induced signaling can result in complex effects such as signal amplification, feedback inhibition or induction of cyclical oscillation between different cellular states <a href="#pone.0000656-Cross1">[5]</a>–<a href="#pone.0000656-LoughreyChen1">[8]</a>. Therefore, a computational tool that accurately predicts phosphorylation events could contribute to a more complete understanding of cell function <a href="#pone.0000656-Blom1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Phosphorylation prediction algorithms must select, from all amino acid sequence space, a subset of amino acid sequences that are able to interact with one or more kinases as phosphate acceptors. The somewhat limited success of current phosphorylation prediction algorithms likely arises from the very large number and variety of both kinases and potential phosphate acceptors (Ser, Thr and Tyr residues) <a href="#pone.0000656-Hunter1">[2]</a>, <a href="#pone.0000656-Johnson1">[10]</a>. A major difficulty in protein phosphorylation prediction stems from the fact that each kinase has its own particular specificity determinants <a href="#pone.0000656-Pinna1">[11]</a>, <a href="#pone.0000656-Nigg1">[12]</a>. In some cases a particular kinase may require its substrate to have a highly stringent recognition site, whereas other kinases may be relatively promiscuous. Other kinases require restraints that may be distal to the recognition site, or consensus motif. Furthermore, it is possible that in certain cases, different kinases may have partially overlapping specificity, so that a single acceptor residue can be phosphorylated by more than one kinase. The challenge in developing a phosphorylation prediction tool is to effectively model molecular recognition mechanisms between individual kinases and their substrates, where the mechanisms can vary broadly for different kinases, and for which little experimental data may be available.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Most current strategies for the prediction of phosphorylation sites model the amino acid sequence (or a so-called consensus motif), which represents a kinase-specific phosphorylation site. Proteins that contain an instance of a given kinase's consensus motif are predicted to be substrates of that kinase. The simplest example of this type of strategy is linear motif searching, using computational tools such as PROSITE <a href="#pone.0000656-Bairoch1">[13]</a> and ELM <a href="#pone.0000656-Puntervoll1">[14]</a>. This type of strategy searches for instances of phosphorylation consensus motifs represented by regular expressions. Other algorithms, such as ScanSite and PHOSITE utilize position-specific profile searches, which allow for more flexible definitions of consensus motifs <a href="#pone.0000656-Falquet1">[15]</a>–<a href="#pone.0000656-Yaffe1">[17]</a>. Machine learning approaches, (<em>e.g.</em> hidden Markov models <a href="#pone.0000656-Huang1">[18]</a>–<a href="#pone.0000656-Senawongse1">[20]</a> and artificial neural networks <a href="#pone.0000656-Blom1">[9]</a>, <a href="#pone.0000656-Blom2">[21]</a>, <a href="#pone.0000656-Wu1">[22]</a>) have been used to model interdependencies between amino acids within a given consensus motif. NetPhos and NetPhosK are leading methods for phosphorylation site prediction that utilize artificial neural networks. Certain other procedures such as PREDIKIN utilize three-dimensional structural modeling to try to predict kinase-specific phosphorylation <a href="#pone.0000656-Brinkworth1">[23]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Attempts to evaluate the effectiveness of any phosphorylation prediction method face a two-fold difficulty. First, since the complete set of phosphorylation sites on all proteins is not known, it is not possible to assess the comprehensiveness of phosphorylation prediction. Therefore, both “false” positive and “true” negative designations may actually be assigned incorrectly to <em>true</em> phosphorylation sites that have yet to be discovered. Second, since only a limited number of sites are known for many kinases, it is likely that the known set of phosphorylation sites for a particular kinase is systematically biased and that any given algorithm may be unwittingly designed or trained to miss true positives. The most reliable measure of confirmation of phosphorylation-site prediction is the identification of such sites as <em>bona fide in vivo</em> phosphorylation sites through experiment. Because this is often laborious and not straightforward, few broad-based computational phosphorylation prediction procedures have had their results substantially confirmed through experimental verification.</p>

<h4>Strategy</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>One hallmark of nearly all published phosphorylation prediction procedures is that they employ a strategy to model the substrate specificity for as many kinases as possible. Such tools, which utilize the same strategy for different kinases, may inadvertently miss elements of substrate recognition that are in some way unique to a particular kinase system. Here, rather than developing a general model to predict substrates for many kinases, we instead propose a targeted procedure that models the substrate specificity in a more detailed way for a single well-studied family of kinases, <em>i.e.</em>, the cyclin-dependent kinases (Cdks) <a href="#pone.0000656-Cross1">[5]</a>, <a href="#pone.0000656-Andrews1">[24]</a>, <a href="#pone.0000656-Nasmyth1">[25]</a>. By targeting a single family of kinases, we should be better positioned to consider discriminating factors specific to this family. Thus, in the present procedure we are able to incorporate additional global characteristics that occur specifically between Cdks and their substrates, introducing a second factor that are not considered when modeling only local sequence motifs.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>Cdks are the master regulators of eukaryotic cell cycle progression, coordinating events such as DNA synthesis and mitosis that are necessary for proper cell division and driving the cell-division process in a regulated manner <a href="#pone.0000656-Cross1">[5]</a>, <a href="#pone.0000656-Andrews1">[24]</a>, <a href="#pone.0000656-Nasmyth1">[25]</a>. In order for Cdks to exhibit enzymatic activity, they must be associated with a binding partner protein called a cyclin. Particular Cdks are associated with one or more cyclins at different points in the cell cycle, and the sequential, temporally coordinated activity of the Cdk/cyclin combinations organizes and orders the molecular events in the cell cycle.</p>
<a id="article1.body1.sec1.sec1.p3" name="article1.body1.sec1.sec1.p3"></a><p>Cdks are obligatory proline-directed serine/threonine kinases. Empirical studies of Cdk substrates indicate a strict requirement for a proline residue one amino acid C-terminal to the acceptor residue (the “+1” site) <a href="#pone.0000656-Nigg1">[12]</a> as well as a strong preference for basic amino acids proximal to the acceptor site, especially arginine or lysine residues at or around the +3 site (<em>i.e.</em>, 3 residues C-terminal to the acceptor). These sequence characteristics are supported by X-ray crystal structures that reveal a large binding pocket for docking of the requisite proline residue, and an acidic patch for binding the C-terminal basic region <a href="#pone.0000656-Brown1">[4]</a>, <a href="#pone.0000656-Pavletich1">[26]</a>, <a href="#pone.0000656-Morgan1">[27]</a>. The identity of other residues surrounding the acceptor site also plays a role, albeit smaller, in Cdk substrate preference. Studies on the catalytic activity of Cdc28 towards <em>in vitro</em> peptide phosphorylation show that substrates of different cyclin/Cdk combinations have largely the same primary sequence characteristics, although different combinations do exhibit slightly different preferences <a href="#pone.0000656-Holmes1">[28]</a>, <a href="#pone.0000656-Songyang1">[29]</a>. These subtle differences may have a considerable impact on cyclin/Cdk specificity, but other factors such as cyclin abundance, substrate binding and the presence or absence of substrate proteins may also play a significant role.</p>
<a id="article1.body1.sec1.sec1.p4" name="article1.body1.sec1.sec1.p4"></a><p>Studies by Holmes and Solomon <a href="#pone.0000656-Holmes1">[28]</a> directly assayed for amino acid sequence specificity for Cdk phosphorylation. Their approach involved a series of experiments based on a GST fusion constructs, each containing a peptide based on the sequence KSPRK derived from the histone H1 Cdk substrate. The effects of all possible single amino acid substitutions at the −1, +2 and +3 positions (position 0 is the acceptor site, and +1 is the obligatory proline residue) were detailed for <em>Xenopus laevis</em> cyclin B-Cdc2 and human cyclin A-Cdc2, cyclin A-Cdk2, cyclin E-Cdk2, and cyclin B-Cdc2. Varying the -1 position was shown to have the least effect, with efficiency of phosphorylation changing, for example, about 2-fold between the worst (Pro) to the best (Gln and Met, followed by His and Gly) amino acids for the <em>X. laevis</em> cyclin B-Cdc2. The +2 position shows strong selectivity against Pro, Gln, Glu and Asp, with about one order of magnitude lower reaction efficiency than for Lys, Arg and Met, the amino acids contributing most positively to catalytic efficiency. Nearly all other amino acids are tolerated at this position, showing about 20-60% of wild-type efficiency. The +3 position is the most selective, with Arg and Lys being strongly preferred, His and Pro showing efficiency ~20% of wild type, and all the others showing efficiency ~5% of wild type, except for the acidic residues Glu and Asp which showed virtually no activity. The activity profiles for other cyclin-Cdk complexes were essential similar to that for cyclin B-Cdc2.</p>
<a id="article1.body1.sec1.sec1.p5" name="article1.body1.sec1.sec1.p5"></a><p>Our computational strategy focused on two characteristics of Cdk-substrate recognition. We first considered data to determine the primary substrate sequence preference, using published crystal structure and biochemical assay data. Second, we incorporated a number of observations indicating clustering <a href="#pone.0000656-Moses1">[30]</a> of phosphorylation sites within Cdk substrates. Many of the known Cdk substrates were phosphorylated at multiple sites in their sequence <a href="#pone.0000656-Henchoz1">[3]</a>, <a href="#pone.0000656-Verma1">[31]</a>-<a href="#pone.0000656-Moll1">[34]</a>. Additionally, certain substrates were found to have a specific patch in their structure that bound cyclins (cyclin-binding, or Cy motif) <a href="#pone.0000656-Sorensen1">[35]</a>, <a href="#pone.0000656-Takeda1">[36]</a>, suggesting that the molecular recognition of substrate was influenced by contacts distal from the catalytic site. Biophysical studies on Pho85, a kinase in <em>S. cerevisiae</em> homologous to Cdc28, further showed semi-processive phosphorylation—i.e., one kinase-substrate binding event may be followed by several phosphate transfer events without dissociation of the enzyme and substrate proteins <a href="#pone.0000656-Jeffery1">[37]</a>. These findings led us to hypothesize that in many cases, Cdk substrates might contain clusters of phosphorylation sites, and therefore that Cdk substrate prediction could be improved not only by optimizing the consensus motif sequence, but also by following consensus site identification with selection of proteins whose sequences are enriched for repeats of that motif. If correct, such an approach will account for the physical mode of phosphorylation and will also overcome the statistical likelihood of false positive predictions based on single site predictions. Multi-site phosphorylation has been previously observed in several different Cdk substrates<a href="#pone.0000656-Jeffery1">[37]</a>–<a href="#pone.0000656-Oda1">[40]</a> One of the best examples of this is phosphorylation of Sic1<a href="#pone.0000656-Nash1">[39]</a> by Cln-Cdc28 complexes, where multisite phosphorylation acts as a switch that sets a threshold for the onset of DNA synthesis during cell cycle.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Based on all the preceding considerations, we modeled Cdk substrates by identifying clusters of both the canonical Cdk motif represented by the regular expression [ST]PX[RK] and a PSSM [<a href="#pone-0000656-t001">Table 1</a>] profile generated from Holmes and Solomon's kinetic data <a href="#pone.0000656-Holmes1">[28]</a>. Proteins in the proteome of the budding yeast <em>Saccharomyces cerevisiae</em> were scored according to both models, and the distribution of scores for each method was compared to the distribution of scores for sequences from a randomly generated mock proteome (see <a href="#s4">Methods</a>). Based on these comparisons, we identified a set of candidate Cdk substrate proteins from <em>S. cerevisiae</em>, and evaluated that set against experimental data.</p>
<div class="figure" id="pone-0000656-t001"><div class="img"><a name="pone-0000656-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000656.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000656.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Position-specific scoring matrix representing the Cdk phosphorylation motif</span></strong></p><span>doi:10.1371/journal.pone.0000656.t001</span></div>
<h4>Clustered canonical motif–based modeling of Cdk substrates</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The canonical Cdk phosphorylation motif, represented by the regular expression <strong>[ST]PX[RK]</strong>, represents the most salient features of Cdk phosphorylation site composition and the largest contributions to catalytic efficiency of phosphorylation. It is a highly restrictive statement of a potential Cdk phosphorylation site, in the sense that it does not allow at all for phosphorylation site sequences that may deviate from these features. It accentuates the most influential aspects of site recognition and disregards the rest. The benefit of such an exclusive statement of the phosphorylation motif, then, is that it highlights the most likely phosphorylation sites. However, it is probable that this type of statement will result in underprediction, since not all substrates will have all of their actual phosphorylation sites in these stringent motifs.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Proteins from the yeast proteome were observed to contain between zero and 9 copies of the canonical phosphorylation motif. The majority of proteins in the yeast proteome (i.e., 4800) had no occurrences of the motif (score = 0), and as a trend, the number of proteins decreased as the score (i.e., the number of canonical motifs) increased [<a href="#pone-0000656-g001">Figure 1A</a>]. A similar general trend was observed in the mock proteome. However, the rate of decrease was higher for the mock than for the yeast proteome [<a href="#pone-0000656-g001">Figure 1A</a>]. In other words, the yeast proteome was enriched for high scoring proteins—suggesting that high scores may indeed be indicative of selection for function as Cdk substrates.</p>
<div class="figure" id="pone-0000656-g001"><div class="img"><a name="pone-0000656-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000656.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000656.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Analysis of Canonical Cdk Motif Clustering in Yeast and Mock Proteomes.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(A) The number of proteins having a given score decreases as that score increases. Yeast proteins are represented in navy, and mock proteins are represented in magenta. At low score, (i.e. less than ~4), yeast and mock are similar– the ratio of mock to yeast, shown by black squares, approximates unity (B). However at higher scores (i.e. 5 and above), yeast proteome contains substantially more proteins than mock (A), and the ratio of mock/yeast approaches zero (B). All proteins from the yeast proteome scoring 5 or higher are considered candidate substrates.</p>
<span>doi:10.1371/journal.pone.0000656.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>The following procedure was used to predict potential Cdk substrates in an unbiased fashion. For each integral score j between 0 and 9 inclusive, we calculated the ratio r<sub>j</sub> that represents the ratio of the proportion of proteins from the randomly generated mock proteome with score j to the proportion of yeast proteins with score j [<a href="#pone-0000656-g001">Figure 1B</a>]. It appeared that at low scores of j, r<sub>j</sub> values were clustered close to unity (i.e., similar in real and mock), but at high scores of j, r<sub>j</sub> tended toward zero (i.e., enriched in real proteins and therefore candidate Cdk substrate) [<a href="#pone-0000656-g001">Figure 1B</a>]. We determined a cut-off score k that would divide the yeast proteome into 2 groups, a group scoring below k where the number of real proteins is similar to the number of mock proteins, and a group scoring above k, that is enriched for real proteins. Therefore we solved for the value k that minimized the sum of the standard errors of the mean (SEM) over (i) all r<sub>j</sub> such that j&lt;k, and (ii) all r<sub>j </sub>such that j&gt; = k. We found this value of k to be equal to 5, yielding a lower scoring cluster with an SEM of 0.079 and a higher scoring cluster with an SEM of 0.032. Moreover, this value of k also maximizes the differences between the means of r<sub>j </sub>for the two clusters. The mean of r<sub>j&lt;5</sub> = 1.01, and the mean of r<sub>j&gt; = 5</sub> = 0.078.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>A total of 38 yeast proteins scored above the threshold value (k = 5) that separated random from significant predicted substrates [<a href="#pone-0000656-t002">Table 2</a>, <a href="#pone.0000656.s001">Table S1</a>]. These 38 included the known Cdk substrates Ace2, Cdc6, Cdh1, Orc2, Sld2, Stb1 and Ste20 [<a href="#pone-0000656-t002">Table 2</a>].<a href="#pone.0000656-Jaspersen1">[32]</a>, <a href="#pone.0000656-Nash1">[39]</a>–<a href="#pone.0000656-Wu2">[46]</a> When compared to the results of a proteomic survey of in vitro Cdc28 phosphorylation by Ubersax et al.<a href="#pone.0000656-Ubersax1">[47]</a>, 25 of the 38 proteins were found in their set of 186 best candidate Cdc28 substrates [<a href="#pone-0000656-t002">Table 2</a>]. In addition, six of the 38 proteins, Cdh1, Lte1, Bem3, Bud3, Ace2 and Ypl267 have been found to physically interact with cyclin/Cdc28 complexes via co-immunoaffinity purification [<a href="#pone-0000656-t002">Table 2</a>] <a href="#pone.0000656-Archambault1">[48]</a>.</p>
<div class="figure" id="pone-0000656-t002"><div class="img"><a name="pone-0000656-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000656.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000656.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Bioinformatic screen for candidates Cdc28 substrates</span></strong></p><span>doi:10.1371/journal.pone.0000656.t002</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>This method did not predict all known <em>in vivo</em> Cdc28 substrates [Supplementary <a href="#pone.0000656.s002">Table S2</a> reviews and references known Cdc28 substrates]. For example, some known substrates such as Sic1<a href="#pone.0000656-Verma1">[31]</a>, <a href="#pone.0000656-Nash1">[39]</a>, <a href="#pone.0000656-Nishizawa1">[49]</a>, although containing clustered minimal Cdk motifs, do not contain sufficient copies of the full canonical consensus motif to exceed the cut-off value of k = 5.</p>


<h4>Clustered kinetics-based PSSM modeling of Cdk substrates</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Based on kinetic phosphorylation data <a href="#pone.0000656-Holmes1">[28]</a>, we used the PSSM-based approach to model the probability for each of the 20 amino acids at positions –1 through +4 to be present surrounding minimal Cdk phosphorylation motifs <a href="#pone.0000656-Holmes2">[50]</a>. The score for a protein equals the sum of the PSSM score for each potential Cdk site, as defined by Equations 1 and 2 and the PSSM in <a href="#pone-0000656-t001">Table 1</a>. The general trends using this scoring model were similar to those using the canonical consensus regular expression motif scoring system: as the score increased, the occurrence of proteins decreased, with more real proteins than mock proteins at high scores [<a href="#pone-0000656-g002">Figure 2A</a>]. The range of PSSM scores are continuous values, rather than the discrete integral values obtained from regular expression scoring. Therefore, in order to perform analogous discrete analysis for the two scoring systems, we grouped the proteins into bins 0.4 units wide according to their summed PSSM scores.</p>
<div class="figure" id="pone-0000656-g002"><div class="img"><a name="pone-0000656-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000656" data-uri="info:doi/10.1371/journal.pone.0000656.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000656.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000656.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000656.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000656.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Analysis of kinetic-derived PSSM motif clustering.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The number of proteins having a given score decreases as that score increases. Yeast proteins are represented in navy, and mock proteins are represented in magenta. At low score, (I.e. less than ~3.2), yeast and mock are similar– the ratio of mock to yeast, shown by black squares, approximates unity. (Proteins are grouped in bins 0.4 units wide) However at higher scores (I.e. 4.4 and above), yeast proteome contains substantially more proteins than mock, and the ratio of mock/yeast approaches zero. All proteins from the yeast proteome scoring 4.4or higher are considered candidate substrates. The region between 3.2 and 4.4 is considered a transition region and yeast proteins with these scores are considered borderline candidate substrates</p>
<span>doi:10.1371/journal.pone.0000656.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In this way, we determined that a value of k = 4.4 minimized the sum of the SEM of r<sub>j&lt;k</sub> and the SEM of r<sub>j&gt; = k</sub> and maximized the differences between the means of the two clusters; the mean for the lower scoring group r<sub>j&lt;k</sub> = 0.96 and the mean for the higher scoring group r<sub>j&gt; = k</sub> = 0.11 [<a href="#pone-0000656-g002">Figure 2B</a>]. Here, there appears to be a region of transition from high to low, between the scores of 3.2 to 4.0 (as opposed to the sharp break between scores of 4 and 5 observed with the regular expression scoring system). To determine the transition area in an unbiased manner, we calculated two values, l and m (such that l&lt; = m) that also minimizes the sum of the SEM of r<sub>j&lt;l</sub> and the SEM of r<sub>j&gt; = m</sub>. We found values of l = 3.2 and m = 4.4. The values of l and m define respectively the upper boundary of a SEM-minimized cluster of low scoring proteins (with a mean r<sub>j&lt;l = </sub>1.04) where the enrichment of Cdk substrates is likely low, and the lower boundary of a SEM-minimized cluster of high scoring proteins (with a mean r<sub>j&gt; = m = </sub>0.11), which is likely highly enriched for <em>bona fide</em> substrates. The region between l and m defines a borderline area that likely contains a mix of substrates and non-substrates. In this case, we found that m = k, defining exactly the same high scoring region likely to contain Cdk substrates, regardless of whether or not we choose to define the borderline region.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The above-described unbiased analysis determined a set of 35 likely Cdk substrate proteins scoring above 4.4, and 55 borderline proteins scoring between 3.2 and 4.4 [<a href="#pone-0000656-t002">Table 2</a>]. Twenty-three of the 35 high scoring candidate substrates were also predicted using regular expression scoring; 6 of the 55 borderline candidates overlapped with regular expression motif scoring candidates [<a href="#pone-0000656-t002">Table 2</a>]. Ace2, Cdh1, Orc2, Sld2, Stb1 were among the known substrates that were predicted both using the canonical regular expression motif and the kinetic PSSM <a href="#pone.0000656-Jaspersen1">[32]</a>, <a href="#pone.0000656-Nguyen1">[43]</a>–<a href="#pone.0000656-Ho1">[45]</a>, <a href="#pone.0000656-Doolin1">[51]</a>, <a href="#pone.0000656-Nguyen2">[52]</a>. Cdc6 <a href="#pone.0000656-Elasser1">[42]</a>, <a href="#pone.0000656-Nguyen1">[43]</a> and Ste20 <a href="#pone.0000656-Oda1">[40]</a>, <a href="#pone.0000656-Wu2">[46]</a> are known substrates predicted using the canonical regular expression motif and considered borderline proteins using the kinetic PSSM. Orc6 <a href="#pone.0000656-Nguyen1">[43]</a> and Swi5 <a href="#pone.0000656-Moll1">[34]</a> are known substrates that were predicted by the PSSM method only. Far1, Ndd1, Net 1 and Pds1 are known substrates that were missed using the canonical regular expression motif and considered borderline proteins using the kinetic PSSM <a href="#pone.0000656-Azzam1">[7]</a>, <a href="#pone.0000656-LoughreyChen1">[8]</a>, <a href="#pone.0000656-Gartner1">[33]</a>, <a href="#pone.0000656-Reynolds1">[53]</a>, <a href="#pone.0000656-Agarwal1">[54]</a>.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Additionally, 22 of these 35 candidates matched the top scoring <em>in vitro</em> substrates <a href="#pone.0000656-Ubersax1">[47]</a>. Four candidates, Ace2, Cdh1, Bem3 and Ypl267, were found to physically interact with cyclin/Cdc28 complexes by co-immunoaffinity purification <a href="#pone.0000656-Archambault1">[48]</a>. Twenty-two of the 55 borderline candidates were found to be substrates in the <em>in vitro</em> study and two, Bud3 and Far1, were found previously to interact physically with cyclin/Cdc28 complexes.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Using mass spectrometry <a href="#pone.0000656-Chang1">[38]</a>, we were able to determine phosphorylation at Cdk motifs for several predicted substrates. In these experiments, we found <em>in vitro</em> phosphorylation of recombinant Mcm3 which had been incubated with ATP and affinity-purified Cdc28 complexes. We also found <em>in vivo</em> phosphorylation at Cdk motifs on Ace2, Swi5, Orc2 and Orc6.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have presented here a model for cyclin-dependent kinase substrates. The model first defines a bioinformatic representation of the Cdk phosphorylation motif, either as a regular expression or a PSSM. In addition, the model proposes that a significant proportion of Cdk phosphorylation occurs on proteins that contain multiple phosphorylation sites. The non-random clustering of potential Cdk sites in particular proteins serves as evidence of biological function selected for by nature.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The canonical motif and PSSM strategies, combined, define a set of 91 candidate Cdk substrate proteins comprising 1.5% the yeast proteome. Of these, 46 (0.73% of the yeast proteome) were defined as strong candidates, either being detected using the canonical-motif scoring method, or scoring above the upper cutoff using PSSM-motif method. Twenty-seven were detected using only the canonical-motif method, 8 using only the PSSM-motif method, and 11 by both methods. The remaining 45 (0.72% of the yeast proteome) predicted candidates were “borderline” PSSM candidates only.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>By comparison, only 0.10% of the sequences in the randomized mock proteome scored above the threshold for inclusion as strong candidates, and 0.45% of the sequences in the mock proteome met the score criteria for borderline, PSSM candidates (but not strong candidates). The ratio of candidate substrates detected in yeast-to-candidates substrates detected in mock yields an estimated false positive rate of 14% for the strong candidates and 63% for the borderline candidates. These values indicate that there is indeed clustering on the sequence level beyond what would be expected by random. From them we can infer that ~40 of the 46 strong candidates and ~17 of the 45 borderline candidates are <em>bona fide</em> Cdk substrates. Thus, although the false positive rate for the borderline candidates is high, that subset is nevertheless not inconsequential to biological researchers, since greater than 1 in 3 are likely to be <em>bona fide</em> substrates.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Out of the total set of 91 candidate substrates, 13 proteins (14%) are contained in the set of experimentally characterized <em>in vivo</em> substrates. To our knowledge, at the time of writing there are 26 proteins in that set (<a href="#pone.0000656.s002">Table S2</a>); thus 50% of the currently known substrates were detected as candidates. For reasons detailed below, we expect this method to be less than comprehensive, but rather to yield a set of likely candidate substrates useful for biological researchers while maintaining a reasonably low false positive rate. Extrapolating from our false positive and false negative rates, we expect there to be approximately 114 total proteins (1.9% of the yeast proteome) that are Cdc28 substrates.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Many of our candidate substrates were also predicted to contain Cdk phosphorylation sites using other leading phosphorylation detection algorithms, such as Scansite and NetPhosK. Scansite, using a threshold setting of “high” returns 265 yeast proteins (4.2% of the proteome) as candidate Cdk substrates. Of these, 35 are contained in our set of 91 candidate substrates (38%). Scansite predicts 8 of the 24 well-characterized candidate substrates (33%), as compared to the 50% hit rate using our method. When Scansite was run on our random sequence database, 2.8% of the sequences were detected as candidate Cdk substrates -a false positive rate of 67% for Scansite, for Cdk substrate prediction in this dataset. Therefore, although the present method was only somewhat more comprehensive (50% to 33%) than Scansite with respect to true positive detection, it was much more accurate in terms of false positive rate. Our method generates a set of strong candidates with an estimated false positive rate of 14%, while Scansite, even set to high stringency yields a false positive rate of 67%. Scansite yields a false positive rate similar to that of the borderline candidates (63%) generated using the current method.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>NetPhosK<a href="#pone.0000656-Blom1">[9]</a> detected 88 of our 91 (97%) candidates as containing Cdk substrates, using a scoring threshold of 0.60− a similar true positive rate as Scansite. However, our simulations indicate that fully 21% of the proteome, or 1300 proteins, is predicted by NetPhosK to be Cdk substrates, and so the false positive rate is expected to be even higher for NetPhosK than for Scansite. Thus, the major difference between two leading current phosphorylation prediction methods and the one presented here—protein-level motif clustering—is recognized as an increase in accuracy as measured by a reduced false positive rate.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Our method predicts approximately half of the known yeast Cdk substrates. Therefore, in this study, we make no claim at completeness. Instead, we show the utility of a targeted bioinformatic tool that produces a set of predictions that can be validated using experimental techniques. Our pilot proteomic study, in which we assayed for <em>in vivo</em> phosphorylation using hypothesis-driven mass spectrometry <a href="#pone.0000656-Chang1">[38]</a>, <a href="#pone.0000656-Kalkum1">[55]</a>, confirms a number of our predictions [<a href="#pone-0000656-t002">Table 2</a>]. In addition, our predictions are also consistent with many of the high scoring proteins from the high-throughput <em>in vitro</em> phosphorylation study by Ubersax <em>et al.</em> <a href="#pone.0000656-Ubersax1">[47]</a>, although most of these are as of yet unconfirmed <em>in vivo</em>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Our model, as it stands, is particularly useful for organisms with small proteomes, such as <em>S. cerevisiae</em>. Larger proteomes may be problematic because the false positive rate likely will increase with the number and size of proteins. To extend this procedure effectively may require additional filtering procedures. For example, phosphorylation sites are largely expected to occur on solvent-accessible portions of proteins, particularly loops, so an additional weight could be added to motifs that are expected to occur in such regions, as determined by existing secondary structure prediction <a href="#pone.0000656-Jones1">[56]</a> or homology modeling algorithms <a href="#pone.0000656-Eswar1">[57]</a>. Incorporating the conservation of phosphorylation motifs across related species into the model might also increase its specificity by adding additional biological restraints. However, this has proven to be not a straightforward task, complicated by the fact that orthologous candidate substrates show homologous regions that are enriched for Cdk motifs, but where in many cases the number and precise positioning of the motifs are <em>not</em> very precisely conserved. Supplemental <a href="#pone.0000656.s003">Table S3</a> shows some examples of the imperfect conservation of Cdk motifs across taxa in Cdk substrates. New algorithms are needed in order to properly account for these factors when performing multiple alignments of Cdk substrates.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Furthermore, the semi-processive physical model <a href="#pone.0000656-Jeffery1">[37]</a> of Cdk phosphorylation also suggests that the clustering of sites likely occurs on contiguous surfaces or individual domains of proteins. The average spacing between motifs for candidate substrates identified in our study by canonical motif scoring is 103+/−63 (mean+/−standard deviation) amino acids residues, and by PSSM scoring is 69+/−46 residues. Among the candidate substrates, the subset that overlaps with known, experimentally characterized Cdk substrates, the average spacing was smaller than (63+/−37 for canonical motif scoring, and 38+/−20 for PSSM scoring) but statistically indistinguishable from spacing for the overall set of candidate substrates. Such large spaces between sites suggest that three-dimensional, domain level proximity, rather than simply linear spacing plays an important role in the processivity of Cdk2. Further exploration is necessary to determine the feasibility of using spacing data, or 3-D data for increasing the selectivity of the procedure.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>The algorithm missed certain known yeast substrates such as Cdc23 <a href="#pone.0000656-Rudner1">[58]</a> that are thought to contain single phosphorylation sites. However Cdc23 is present in cells in complex with the proteins Cdc16 and Cdc27 <a href="#pone.0000656-Rudner1">[58]</a>, both of which also have multiple putative Cdk phosphorylation sties. Therefore, it is reasonable to hypothesize that the kinase recognizes and phosphorylates a surface of the entire complex that is formed by the junction of all three proteins. As data on protein complexes <a href="#pone.0000656-Hollunder1">[59]</a>–<a href="#pone.0000656-Gavin1">[61]</a> becomes more comprehensive and reliable, it may become feasible to statistically analyze the presence of Cdk motifs within complexes in a similar manner to that done for individual proteins. We note here that the domain-level clustering of motifs here likely differs from the local clustering observed in the substrates of kinases such as the casein kinases<a href="#pone.0000656-Flotow1">[62]</a>–<a href="#pone.0000656-Fiol1">[64]</a>, GSK3<a href="#pone.0000656-Fiol1">[64]</a>, <a href="#pone.0000656-Frame1">[65]</a> and SR specific protein kinases<a href="#pone.0000656-Aubol1">[66]</a>, <a href="#pone.0000656-VelazquezDones1">[67]</a>, where multiple phosphorylation sites are observed within a single extended motif or repeat region.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>The success of the computational procedure presented here stresses the importance of not being limited to local sequence characteristics for functional prediction. The difficulty in the prediction of post-translational modifications and in phosphorylation prediction in particular, is that short, local sequences—even those that match an extremely well defined consensus—can occur frequently by random sequence drift. In the present study, we found useful the fact that Cdk substrates not only have consensus motifs that have been well studied and could be quite precisely defined, but also had the characteristic of site clustering. We incorporated both global and local sequence characteristics of Cdk substrates into a bioinformatic model that proved successful in predicting a significant number of putative substrates. A substantial amount of experimental information obtained by us and other leads us to believe that this set of putative substrates is, in fact, highly enriched for <em>bona fide</em> Cdk substrates. This set of proteins includes a substantial proportion of known substrates from previous <em>in vivo</em> and <em>in vitro</em> studies, as well as substrates that were confirmed as <em>in vivo</em> phosphorylation sites by mass spectrometry. In the future, these types of approaches—incorporating biochemical details into bioinformatics, and interfacing bioinformatics with experimental testing—should prove to be a useful strategy in predictive computational biology.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>For regular expression consensus motif searches, an algorithm was implemented that scored all proteins in the yeast proteome according to the number of occurrences of the motif. Proteins were scored as the number of phosphorylation motifs within their sequence. For PSSM consensus motif scoring, a PSSM was constructed by assigning a score to each amino acid in each relevant position directly proportional to its effect on catalytic efficiency based on Holmes and Solomon's <a href="#pone.0000656-Holmes1">[28]</a> kinetic data. The specific structure and values of this PSSM can be found in <a href="#pone-0000656-t001">Table 1</a>. These scores were stored in a table—the positions relative to the phosphate acceptor residue was represented on one axis of the table, and the twenty individual amino acids were represented on the other axis. Each protein was scored as follows. First, the information content for each position was calculated from the PSSM using the standard relative entropy definition at each position using the equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where p<sub>i</sub> is the observed probability of amino acid i (at a given position) in the motif, and f<sub>i</sub> is the background frequency of amino acid i in the proteome. The information content at each position should be directly related to its discriminatory power in predicting phosphorylation substrates of Cdk. Then, for each protein, all Ser-Pro and Thr-Pro (the minimal requirement for phosphorylation by Cdk) sequences in a protein sequence were located, and each Ser-Pro and Thr-Pro sequences were scored based on the 5 amino acid window around it (from −1 to +3) around it as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000656.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where I<sub>bits</sub>(n) is the total information content at position n, as defined above, and P<sub>aa</sub>(n<sub>k</sub>) is the probability of amino acid n (any one of the 20 amino acids) at position k. This scheme yields a score for each motif that is weighted both by the information content at each position, and by the relative likelihood of the amino acid found at that position. This gives proportionally more weight to positions that possess more discriminatory power.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Proteins are scored as the aggregate of the score of their individual potential Cdk phosphorylation sites. This type of scoring accounts for both the ‘goodness’ of each potential phosphorylation site and the enrichment (and possible clustering) of potential sites within the protein sequence. The score of a protein is defined as the sum of all scores (S/T<sub>i</sub>P<sub>i+1</sub>) for that protein sequence. The scorings of protein using the regular expression version of the phosphorylation motif can also be represented using this system, by assigning a value of one for relative information to each relevant position and assigning a score of 1 for each match to the regular expression found in a given protein sequence.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>A set of randomly generated amino acid sequences, collectively having identical amino acid composition and protein length distributions as the actual yeast proteome, was used as a negative control. The Cdk phosphorylation motifs found in this ‘mock proteome’ represent the amino acid distribution if it were truly random. Deviations from the random distribution are likely to result from selective pressure on protein sequences, and therefore to reflect biological functionality as phosphorylation substrates.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Scripts were written in PERL on and executed on a multi-CPU Sun server running Solaris 10 to find putative phosphorylation sites and compute their scores for each yeast protein sequence and mock protein sequence using the formulae (1) and (2) as described above. Multiple alignments were performed using the World-Wide Web based clustalW <a href="#pone.0000656-Available1">[68]</a>, <a href="#pone.0000656-Ramu1">[69]</a> server hosted by EMBL.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000656"></div><a name="pone.0000656.s001" id="pone.0000656.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000656.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Accession numbers and descriptions of candidate substrates.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000656.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0000656.s002" id="pone.0000656.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000656.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Compilation of currently known substrates of Cdc28.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000656.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0000656.s003" id="pone.0000656.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000656.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Conservation and alignment of Cdk phosphorylation motifs. Sequences matching canonical and minimal Cdk motifs are highlighted in bold, demonstrating imperfect conservation of motifs across organisms. While some motifs show near perfect alignment, other sites appear in the same general area across the organisms, but are not aligned precisely by the ClustalW organism, either due to differing numbers of sites, or different locations within the protein sequence. Such imperfect alignment corroborates the proposition that selection has occurred on Cdk substrates to favor domain-level clustered phosphorylation. Note for example, that the S. cerevisiae Orc6 (example A) sequence contains four motifs around residue 105–124, three of which nearly perfectly align with the corresponding A. gossypii sequence, while K. lactis contains only two corresponding motifs, and C. albicans only one. Another good example is in the region corresponding to residues 300–340 in S. cerevisiae Swi5 (example C), which contains four Cdk motifs. The corresponding region in A gosyppi contains 5 motifs, and in C. albicans contains 6 motifs, none of which align well with the S. cerevisiae motifs, while the K. lactis contains only 1 single motif in the regions.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000656.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EC. Performed the experiments: EC RB. Analyzed the data: BC TG EC RB. Contributed reagents/materials/analysis tools: BC TG EC. Wrote the paper: BC EC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000656-Cohen1" id="pone.0000656-Cohen1"></a>Cohen P (2000) The regulation of protein function by multisite phosphorylation–a 25 year update. Trends Biochem Sci  25: 596–601.  <ul class="find" data-citedArticleID="969856" data-doi="10.1016/s0968-0004(00)01712-6"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01712-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+regulation+of+protein+function+by+multisite+phosphorylation%26%23x2013%3Ba+25+year+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+regulation+of+protein+function+by+multisite+phosphorylation%26%23x2013%3Ba+25+year+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000656-Hunter1" id="pone.0000656-Hunter1"></a>Hunter T (2000) Signaling–2000 and beyond. Cell  100: 113–127.  <ul class="find" data-citedArticleID="969898" data-doi="10.1016/s0092-8674(00)81688-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81688-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signaling%26%23x2013%3B2000+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signaling%26%23x2013%3B2000+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000656-Henchoz1" id="pone.0000656-Henchoz1"></a>Henchoz S, Chi Y, Catarin B, Herskowitz I, Deshaies RJ, et al.  (1997) Phosphorylation- and ubiquitin-dependent degradation of the cyclin-dependent kinase inhibitor Far1p in budding yeast. Genes Dev  11: 3046–3060.  <ul class="find" data-citedArticleID="969882" data-doi="10.1101/gad.11.22.3046"><li><a href="http://dx.doi.org/10.1101/gad.11.22.3046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation-+and+ubiquitin-dependent+degradation+of+the+cyclin-dependent+kinase+inhibitor+Far1p+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation-+and+ubiquitin-dependent+degradation+of+the+cyclin-dependent+kinase+inhibitor+Far1p+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000656-Brown1" id="pone.0000656-Brown1"></a>Brown N, Noble M, Endicott J, Johnson L (1999) The structural basis for specificity of substrate and recruitment peptides for cyclin-dependent kinases. Nat Cell Biology  1: 438–443.  <ul class="find" data-citedArticleID="969852"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+structural+basis+for+specificity+of+substrate+and+recruitment+peptides+for+cyclin-dependent+kinases.&amp;auth=&amp;atitle=The+structural+basis+for+specificity+of+substrate+and+recruitment+peptides+for+cyclin-dependent+kinases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+structural+basis+for+specificity+of+substrate+and+recruitment+peptides+for+cyclin-dependent+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+structural+basis+for+specificity+of+substrate+and+recruitment+peptides+for+cyclin-dependent+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000656-Cross1" id="pone.0000656-Cross1"></a>Cross FR (1995) Starting the cell cycle: what's the point? Curr Opin Cell Biol  7: 790–797.  <ul class="find" data-citedArticleID="969858" data-doi="10.1016/0955-0674(95)80062-x"><li><a href="http://dx.doi.org/10.1016/0955-0674(95)80062-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Starting+the+cell+cycle%3A+what%27s+the+point%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Starting+the+cell+cycle%3A+what%27s+the+point%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000656-Cross2" id="pone.0000656-Cross2"></a>Cross FR, Archambault V, Miller M, Klovstad M (2002) Testing a mathematical model of the yeast cell cycle. Mol Biol Cell  13: 52–70.  <ul class="find" data-citedArticleID="969860"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Testing+a+mathematical+model+of+the+yeast+cell+cycle.&amp;auth=&amp;atitle=Testing+a+mathematical+model+of+the+yeast+cell+cycle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Testing+a+mathematical+model+of+the+yeast+cell+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Testing+a+mathematical+model+of+the+yeast+cell+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000656-Azzam1" id="pone.0000656-Azzam1"></a>Azzam R, Chen S, Shou W, Mah A, Alexandru G, et al.  (2004) Phosphorylation by cyclin B-Cdk underlies release of mitotic exit activator Cdc14 from the nucleolus. Science  305: 516–519.  <ul class="find" data-citedArticleID="969842" data-doi="10.1126/science.1099402"><li><a href="http://dx.doi.org/10.1126/science.1099402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+by+cyclin+B-Cdk+underlies+release+of+mitotic+exit+activator+Cdc14+from+the+nucleolus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+by+cyclin+B-Cdk+underlies+release+of+mitotic+exit+activator+Cdc14+from+the+nucleolus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000656-LoughreyChen1" id="pone.0000656-LoughreyChen1"></a>Loughrey Chen S, Huddleston MJ, Shou W, Deshaies RJ, Annan RS, et al.  (2002) Mass spectrometry-based methods for phosphorylation site mapping of hyperphosphorylated proteins applied to Net1, a regulator of exit from mitosis in yeast. Mol Cell Proteomics  1: 186–196.  <ul class="find" data-citedArticleID="969910" data-doi="10.1074/mcp.m100032-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m100032-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometry-based+methods+for+phosphorylation+site+mapping+of+hyperphosphorylated+proteins+applied+to+Net1%2C+a+regulator+of+exit+from+mitosis+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometry-based+methods+for+phosphorylation+site+mapping+of+hyperphosphorylated+proteins+applied+to+Net1%2C+a+regulator+of+exit+from+mitosis+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000656-Blom1" id="pone.0000656-Blom1"></a>Blom N, Sicheritz-Ponten T, Gupta R, Gammeltoft S, Brunak S (2004) Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. Proteomics  4: 1633–1649.  <ul class="find" data-citedArticleID="969846" data-doi="10.1002/pmic.200300771"><li><a href="http://dx.doi.org/10.1002/pmic.200300771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+post-translational+glycosylation+and+phosphorylation+of+proteins+from+the+amino+acid+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+post-translational+glycosylation+and+phosphorylation+of+proteins+from+the+amino+acid+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000656-Johnson1" id="pone.0000656-Johnson1"></a>Johnson S, Hunter T (2005) Kinomic: methods for deciphering the kinome. Nat Methods  2: 17–25.  <ul class="find" data-citedArticleID="969904" data-doi="10.1038/nmeth731"><li><a href="http://dx.doi.org/10.1038/nmeth731" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinomic%3A+methods+for+deciphering+the+kinome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinomic%3A+methods+for+deciphering+the+kinome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000656-Pinna1" id="pone.0000656-Pinna1"></a>Pinna L, Ruzzene M (1996) How do protein kinases recognize their substrates? Biochimica et Biophysica Acta  1314: 191–225.  <ul class="find" data-citedArticleID="969940" data-doi="10.1016/s0167-4889(96)00083-3"><li><a href="http://dx.doi.org/10.1016/s0167-4889(96)00083-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+do+protein+kinases+recognize+their+substrates%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+do+protein+kinases+recognize+their+substrates%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000656-Nigg1" id="pone.0000656-Nigg1"></a>Nigg EA (1993) Cellular substrates of p34(cdc2) and its companion cyclin-dependent kinases. Trends Cell Biol  3: 296–301.  <ul class="find" data-citedArticleID="969930" data-doi="10.1016/0962-8924(93)90011-o"><li><a href="http://dx.doi.org/10.1016/0962-8924(93)90011-o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+substrates+of+p34%28cdc2%29+and+its+companion+cyclin-dependent+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+substrates+of+p34%28cdc2%29+and+its+companion+cyclin-dependent+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000656-Bairoch1" id="pone.0000656-Bairoch1"></a>Bairoch A (1991) PROSITE: a dictionary of sites and patterns in proteins. Nucleic Acids Res  19: 2241–2245.  <ul class="find" data-citedArticleID="969844" data-doi="10.1093/nar/19.suppl.2241"><li><a href="http://dx.doi.org/10.1093/nar/19.suppl.2241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PROSITE%3A+a+dictionary+of+sites+and+patterns+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PROSITE%3A+a+dictionary+of+sites+and+patterns+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000656-Puntervoll1" id="pone.0000656-Puntervoll1"></a>Puntervoll P al. e (2003) ELM server: A new resource for investigating short functional sites in modular eukarotic proteins. Nucleic Acids Res  31: 3625–3630.  <ul class="find" data-citedArticleID="969942" data-doi="10.1093/nar/gkg545"><li><a href="http://dx.doi.org/10.1093/nar/gkg545" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ELM+server%3A+A+new+resource+for+investigating+short+functional+sites+in+modular+eukarotic+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ELM+server%3A+A+new+resource+for+investigating+short+functional+sites+in+modular+eukarotic+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000656-Falquet1" id="pone.0000656-Falquet1"></a>Falquet L, Pagni M, Bucher P, Hulo N, Sigrist C, et al.  (2002) The PROSITE database, its status in 2002. Nucleic Acids Res  30: 235–238.  <ul class="find" data-citedArticleID="969868" data-doi="10.1093/nar/30.1.235"><li><a href="http://dx.doi.org/10.1093/nar/30.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+PROSITE+database%2C+its+status+in+2002." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+PROSITE+database%2C+its+status+in+2002.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000656-Obenauer1" id="pone.0000656-Obenauer1"></a>Obenauer J, Cantley LC, Yaffe M (2003) Scansite 2.0: Proteome-wide prediction of cell signaling interactions using short sequence motifs. Nucleic Acids Res  31: 3635–3641.  <ul class="find" data-citedArticleID="969934" data-doi="10.1093/nar/gkg584"><li><a href="http://dx.doi.org/10.1093/nar/gkg584" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scansite+2.0%3A+Proteome-wide+prediction+of+cell+signaling+interactions+using+short+sequence+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scansite+2.0%3A+Proteome-wide+prediction+of+cell+signaling+interactions+using+short+sequence+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000656-Yaffe1" id="pone.0000656-Yaffe1"></a>Yaffe M, Leparc G, Lai J, Obata T, Volinia S, et al.  (2001) A motif-based profile scanning approach for genome-wide prediction of signaling pathways. Nat Biotechnol  19: 348–355.  <ul class="find" data-citedArticleID="969968" data-doi="10.1038/86737"><li><a href="http://dx.doi.org/10.1038/86737" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+motif-based+profile+scanning+approach+for+genome-wide+prediction+of+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+motif-based+profile+scanning+approach+for+genome-wide+prediction+of+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000656-Huang1" id="pone.0000656-Huang1"></a>Huang H, Lee T, Tzeng S, Horng J (2005) KinasePhos: a web tool for identifying protein kinase-specific phosphorylation sites. Nucleic Acids Res  33: W226–W229.  <ul class="find" data-citedArticleID="969894" data-doi="10.1093/nar/gki471"><li><a href="http://dx.doi.org/10.1093/nar/gki471" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=KinasePhos%3A+a+web+tool+for+identifying+protein+kinase-specific+phosphorylation+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22KinasePhos%3A+a+web+tool+for+identifying+protein+kinase-specific+phosphorylation+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000656-Huang2" id="pone.0000656-Huang2"></a>Huang H, Lee T, Tzeng S, Wu L, Horng J, et al.  (2005) Incorporating hidden Markov models for identifying protein kinase-specific phosphorylation sites. J Comput Chem  26: 1032–1041.  <ul class="find" data-citedArticleID="969896" data-doi="10.1002/jcc.20235"><li><a href="http://dx.doi.org/10.1002/jcc.20235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporating+hidden+Markov+models+for+identifying+protein+kinase-specific+phosphorylation+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporating+hidden+Markov+models+for+identifying+protein+kinase-specific+phosphorylation+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000656-Senawongse1" id="pone.0000656-Senawongse1"></a>Senawongse P, Dalby A, Yang Z (2005) Predicting the phosphorylation sites using hidden Markov models and machine learning methods. J Chem Inf Model  45: 1147–1152.  <ul class="find" data-citedArticleID="969950" data-doi="10.1021/ci050047+"><li><a href="http://dx.doi.org/10.1021/ci050047+" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+phosphorylation+sites+using+hidden+Markov+models+and+machine+learning+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+phosphorylation+sites+using+hidden+Markov+models+and+machine+learning+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000656-Blom2" id="pone.0000656-Blom2"></a>Blom N, Gammeltoft S, Brunak S (1999) Sequence and structure-based prediction of eukaryotic protein phosphorylation sites. J Mol Biol  294: 1351–1362.  <ul class="find" data-citedArticleID="969848" data-doi="10.1006/jmbi.1999.3310"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.3310" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+structure-based+prediction+of+eukaryotic+protein+phosphorylation+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+structure-based+prediction+of+eukaryotic+protein+phosphorylation+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000656-Wu1" id="pone.0000656-Wu1"></a>Wu C (1997) Artificial neural networks for molecular sequence analysis. Comput Chem  21: 237–256.  <ul class="find" data-citedArticleID="969964" data-doi="10.1016/s0097-8485(96)00038-1"><li><a href="http://dx.doi.org/10.1016/s0097-8485(96)00038-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Artificial+neural+networks+for+molecular+sequence+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Artificial+neural+networks+for+molecular+sequence+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000656-Brinkworth1" id="pone.0000656-Brinkworth1"></a>Brinkworth R, Breinl R, Kobe B (2003) Structural basis and prediction of substrate specificity in protein serine/threonine kinases. Proc Natl Acad Sci, USA  100: 74–79.  <ul class="find" data-citedArticleID="969850" data-doi="10.1073/pnas.0134224100"><li><a href="http://dx.doi.org/10.1073/pnas.0134224100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+and+prediction+of+substrate+specificity+in+protein+serine%2Fthreonine+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+and+prediction+of+substrate+specificity+in+protein+serine%2Fthreonine+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000656-Andrews1" id="pone.0000656-Andrews1"></a>Andrews B, Measday V (1998) The cyclin family of budding yeast: abundant use of a good idea. Trends in Genetics  14: 66–72.  <ul class="find" data-citedArticleID="969834" data-doi="10.1016/s0168-9525(97)01322-x"><li><a href="http://dx.doi.org/10.1016/s0168-9525(97)01322-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+cyclin+family+of+budding+yeast%3A+abundant+use+of+a+good+idea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+cyclin+family+of+budding+yeast%3A+abundant+use+of+a+good+idea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000656-Nasmyth1" id="pone.0000656-Nasmyth1"></a>Nasmyth K (1993) Control of the yeast cell cycle by the Cdc28 protein kinase. Curr Opin in Cell Biol  5: 166–179.  <ul class="find" data-citedArticleID="969924" data-doi="10.1016/0955-0674(93)90099-c"><li><a href="http://dx.doi.org/10.1016/0955-0674(93)90099-c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+the+yeast+cell+cycle+by+the+Cdc28+protein+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+the+yeast+cell+cycle+by+the+Cdc28+protein+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000656-Pavletich1" id="pone.0000656-Pavletich1"></a>Pavletich N (1999) Mechanisms of cyclin-dependent kinase regulation: structures of Cdks, their cyclin activators and Cip and INK4 inhibitors. J Mol Biol  287: 821–828.  <ul class="find" data-citedArticleID="969938" data-doi="10.1006/jmbi.1999.2640"><li><a href="http://dx.doi.org/10.1006/jmbi.1999.2640" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+cyclin-dependent+kinase+regulation%3A+structures+of+Cdks%2C+their+cyclin+activators+and+Cip+and+INK4+inhibitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+cyclin-dependent+kinase+regulation%3A+structures+of+Cdks%2C+their+cyclin+activators+and+Cip+and+INK4+inhibitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000656-Morgan1" id="pone.0000656-Morgan1"></a>Morgan DO (1997) Cyclin-dependent kinases: engines, clocks and microprocessors. Annu Rev Cell Dev Biol  13: 261–291.  <ul class="find" data-citedArticleID="969918" data-doi="10.1146/annurev.cellbio.13.1.261"><li><a href="http://dx.doi.org/10.1146/annurev.cellbio.13.1.261" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyclin-dependent+kinases%3A+engines%2C+clocks+and+microprocessors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyclin-dependent+kinases%3A+engines%2C+clocks+and+microprocessors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000656-Holmes1" id="pone.0000656-Holmes1"></a>Holmes J, Solomon M (1996) A predictive scale for evaluating cyclin-dependent kinase substrates. A comparison of p34cdc2 and p33cdk2. J Biol Chem  271: 25240–25246.  <ul class="find" data-citedArticleID="969890" data-doi="10.1074/jbc.271.41.25240"><li><a href="http://dx.doi.org/10.1074/jbc.271.41.25240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+predictive+scale+for+evaluating+cyclin-dependent+kinase+substrates.+A+comparison+of+p34cdc2+and+p33cdk2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+predictive+scale+for+evaluating+cyclin-dependent+kinase+substrates.+A+comparison+of+p34cdc2+and+p33cdk2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000656-Songyang1" id="pone.0000656-Songyang1"></a>Songyang Z, Blechner S, Hoagland N, Hoekstra MF, Piwnica-Worms H, et al.  (1994) Use of an oriented peptide library to determine the optimal substrates of protein kinases. Curr Biol  4: 973–982.  <ul class="find" data-citedArticleID="969952" data-doi="10.1016/s0960-9822(00)00221-9"><li><a href="http://dx.doi.org/10.1016/s0960-9822(00)00221-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+an+oriented+peptide+library+to+determine+the+optimal+substrates+of+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+an+oriented+peptide+library+to+determine+the+optimal+substrates+of+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000656-Moses1" id="pone.0000656-Moses1"></a>Moses A, Heriche J, Durbin R (2007) Clustering of phosphorylation site recognition motifs can be exploited to predict the targets of cyclin-dependent kinase. Genome Biol  8: R23.  <ul class="find" data-citedArticleID="969920" data-doi="10.1186/gb-2007-8-2-r23"><li><a href="http://dx.doi.org/10.1186/gb-2007-8-2-r23" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clustering+of+phosphorylation+site+recognition+motifs+can+be+exploited+to+predict+the+targets+of+cyclin-dependent+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clustering+of+phosphorylation+site+recognition+motifs+can+be+exploited+to+predict+the+targets+of+cyclin-dependent+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000656-Verma1" id="pone.0000656-Verma1"></a>Verma R, Annan RS, Huddleston MJ, Carr SA, Reynard G, et al.  (1997) Phosphorylation of Sic1p by G1 Cdk required for its degradation and entry into S phase. Science  278: 455–460.  <ul class="find" data-citedArticleID="969962" data-doi="10.1126/science.278.5337.455"><li><a href="http://dx.doi.org/10.1126/science.278.5337.455" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+Sic1p+by+G1+Cdk+required+for+its+degradation+and+entry+into+S+phase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+Sic1p+by+G1+Cdk+required+for+its+degradation+and+entry+into+S+phase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000656-Jaspersen1" id="pone.0000656-Jaspersen1"></a>Jaspersen S, JF C, Morgan D (1999) Inhibitory phosphorylation of the APC regulator Hct1 is controlled by the kinase Cdc28 and the phosphatase Cdc14. Curr Biol  9: 227–236.  <ul class="find" data-citedArticleID="969900" data-doi="10.1016/s0960-9822(99)80111-0"><li><a href="http://dx.doi.org/10.1016/s0960-9822(99)80111-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibitory+phosphorylation+of+the+APC+regulator+Hct1+is+controlled+by+the+kinase+Cdc28+and+the+phosphatase+Cdc14." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibitory+phosphorylation+of+the+APC+regulator+Hct1+is+controlled+by+the+kinase+Cdc28+and+the+phosphatase+Cdc14.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000656-Gartner1" id="pone.0000656-Gartner1"></a>Gartner A, Jovanovic A, Jeoung D, Bourlat S, Cross FR, et al.  (1998) Pheromone-dependent G1 cell cycle arrest requires Far1 phosphorylation, but may not involve inhibition of Cdc28-Cln2, in vivo. Mol Cell Biol  18: 3681–3691.  <ul class="find" data-citedArticleID="969878"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pheromone-dependent+G1+cell+cycle+arrest+requires+Far1+phosphorylation%2C+but+may+not+involve+inhibition+of+Cdc28-Cln2%2C+in+vivo.&amp;auth=&amp;atitle=Pheromone-dependent+G1+cell+cycle+arrest+requires+Far1+phosphorylation%2C+but+may+not+involve+inhibition+of+Cdc28-Cln2%2C+in+vivo." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pheromone-dependent+G1+cell+cycle+arrest+requires+Far1+phosphorylation%2C+but+may+not+involve+inhibition+of+Cdc28-Cln2%2C+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pheromone-dependent+G1+cell+cycle+arrest+requires+Far1+phosphorylation%2C+but+may+not+involve+inhibition+of+Cdc28-Cln2%2C+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000656-Moll1" id="pone.0000656-Moll1"></a>Moll T, Tebb G, Surana U, Robitsch H, Nasmyth K (1991) The role of phosphorylation and the Cdc28 protein kinase in cell cycle-regulated nuclear import of the S. cerevisiae transcription factor SWI5. Cell  66: 743–758.  <ul class="find" data-citedArticleID="969916" data-doi="10.1016/0092-8674(91)90118-i"><li><a href="http://dx.doi.org/10.1016/0092-8674(91)90118-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+phosphorylation+and+the+Cdc28+protein+kinase+in+cell+cycle-regulated+nuclear+import+of+the+S.+cerevisiae+transcription+factor+SWI5." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+phosphorylation+and+the+Cdc28+protein+kinase+in+cell+cycle-regulated+nuclear+import+of+the+S.+cerevisiae+transcription+factor+SWI5.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000656-Sorensen1" id="pone.0000656-Sorensen1"></a>Sorensen C, Lukas C, Kramer E, Oeters J-M, Bartek J, et al.  (2001) A conserved cyclin-binding domain determines functional interplay between anaphase-promoting complex-Cdh1 and cyclin A-Cdk2 during cell cycle progression. Mol Cell Biol  21: 3692–3703.  <ul class="find" data-citedArticleID="969954" data-doi="10.1128/mcb.21.11.3692-3703.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.11.3692-3703.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+conserved+cyclin-binding+domain+determines+functional+interplay+between+anaphase-promoting+complex-Cdh1+and+cyclin+A-Cdk2+during+cell+cycle+progression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+conserved+cyclin-binding+domain+determines+functional+interplay+between+anaphase-promoting+complex-Cdh1+and+cyclin+A-Cdk2+during+cell+cycle+progression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000656-Takeda1" id="pone.0000656-Takeda1"></a>Takeda D, Wohlschlegel J, Dutta A (2001) A bipartitie substrate recognition motif for cyclin-dependent kinases. J Biol Chem  276: 1993–1997.  <ul class="find" data-citedArticleID="969956" data-doi="10.1074/jbc.m005719200"><li><a href="http://dx.doi.org/10.1074/jbc.m005719200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+bipartitie+substrate+recognition+motif+for+cyclin-dependent+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+bipartitie+substrate+recognition+motif+for+cyclin-dependent+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000656-Jeffery1" id="pone.0000656-Jeffery1"></a>Jeffery D, Springer M, King D, O'Shea E (2001) Multi-site phosphorylation of Pho4 by the cyclin-Cdk Pho80-Pho85 is semi-processive with site preferences. J Mol Biol  306:   <ul class="find" data-citedArticleID="969902" data-doi="10.1006/jmbi.2000.4417"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4417" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multi-site+phosphorylation+of+Pho4+by+the+cyclin-Cdk+Pho80-Pho85+is+semi-processive+with+site+preferences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multi-site+phosphorylation+of+Pho4+by+the+cyclin-Cdk+Pho80-Pho85+is+semi-processive+with+site+preferences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000656-Chang1" id="pone.0000656-Chang1"></a>Chang EJ, Archambault V, McLachlin DT, Krutchinsky AN, Chait BT (2004) Analysis of Protein Phosphorylation by Hypothesis-Driven Multiple-Stage Mass Spectrometry. Anal Chem  76: 4472–4483.  <ul class="find" data-citedArticleID="969854" data-doi="10.1021/ac049637h"><li><a href="http://dx.doi.org/10.1021/ac049637h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+Protein+Phosphorylation+by+Hypothesis-Driven+Multiple-Stage+Mass+Spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+Protein+Phosphorylation+by+Hypothesis-Driven+Multiple-Stage+Mass+Spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000656-Nash1" id="pone.0000656-Nash1"></a>Nash P, Tang X, Orlicky S, Chen Q, Gerlter F, et al.  (2001) Multisite phosphorylation of a Cdk inhibitor sets a threshold for the onset of DNA replication. Nature  414: 514–521.  <ul class="find" data-citedArticleID="969922" data-doi="10.1038/35107009"><li><a href="http://dx.doi.org/10.1038/35107009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multisite+phosphorylation+of+a+Cdk+inhibitor+sets+a+threshold+for+the+onset+of+DNA+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multisite+phosphorylation+of+a+Cdk+inhibitor+sets+a+threshold+for+the+onset+of+DNA+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000656-Oda1" id="pone.0000656-Oda1"></a>Oda Y, Huang K, Cross FR, Cowburn D, Chait BT (1999) Accurate quantitation of protein expression and site-specific phosphorylation. Proc Natl Acad Sci, USA  96: 6591–6596.  <ul class="find" data-citedArticleID="969936" data-doi="10.1073/pnas.96.12.6591"><li><a href="http://dx.doi.org/10.1073/pnas.96.12.6591" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+quantitation+of+protein+expression+and+site-specific+phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+quantitation+of+protein+expression+and+site-specific+phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000656-McBride1" id="pone.0000656-McBride1"></a>McBride H, Yu Y, Stillmain D (1999) Distinct regions of the Swi5 and Ace2 transcription factors are required for specific gene activation. J Biol Chem  274: 21029–21036.  <ul class="find" data-citedArticleID="969914" data-doi="10.1074/jbc.274.30.21029"><li><a href="http://dx.doi.org/10.1074/jbc.274.30.21029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+regions+of+the+Swi5+and+Ace2+transcription+factors+are+required+for+specific+gene+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+regions+of+the+Swi5+and+Ace2+transcription+factors+are+required+for+specific+gene+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000656-Elasser1" id="pone.0000656-Elasser1"></a>Elasser S, Lou F, Wang B, Campbell J, Jong A (1996) Interaction between yeast Cdc6 protein and B-type cyclin/Cdc28 kinases. Mol Biol Cell  7: 1723–1735.  <ul class="find" data-citedArticleID="969864" data-doi="10.1091/mbc.7.11.1723"><li><a href="http://dx.doi.org/10.1091/mbc.7.11.1723" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interaction+between+yeast+Cdc6+protein+and+B-type+cyclin%2FCdc28+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interaction+between+yeast+Cdc6+protein+and+B-type+cyclin%2FCdc28+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000656-Nguyen1" id="pone.0000656-Nguyen1"></a>Nguyen VQ, Co C, Li J (2001) Cyclin-dependent kinases prevent DNA re-replication through multiple mechanisms. Nature  411: 1068–1073.  <ul class="find" data-citedArticleID="969926"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms.&amp;auth=&amp;atitle=Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000656-Masumoto1" id="pone.0000656-Masumoto1"></a>Masumoto H, Muramatsu S, Kamimura Y, Araki H (2002) S-Cdk-dependent phosphorylation of Sld2 essential fo chromosomal DNA replication in budding yeast. Nature  415: 651–655.  <ul class="find" data-citedArticleID="969912" data-doi="10.1038/nature713"><li><a href="http://dx.doi.org/10.1038/nature713" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=S-Cdk-dependent+phosphorylation+of+Sld2+essential+fo+chromosomal+DNA+replication+in+budding+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22S-Cdk-dependent+phosphorylation+of+Sld2+essential+fo+chromosomal+DNA+replication+in+budding+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000656-Ho1" id="pone.0000656-Ho1"></a>Ho Y, Costanzo M, Moore L, Kobayashi R, Andrews B (1999) Regulation of transcription at the Saccharomyces cerevisiae start transition by Stb1, a Swi6-binding protein. Mol Biol Cell  19: 5267–5278.  <ul class="find" data-citedArticleID="969884"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+transcription+at+the+Saccharomyces+cerevisiae+start+transition+by+Stb1%2C+a+Swi6-binding+protein.&amp;auth=&amp;atitle=Regulation+of+transcription+at+the+Saccharomyces+cerevisiae+start+transition+by+Stb1%2C+a+Swi6-binding+protein." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+transcription+at+the+Saccharomyces+cerevisiae+start+transition+by+Stb1%2C+a+Swi6-binding+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+transcription+at+the+Saccharomyces+cerevisiae+start+transition+by+Stb1%2C+a+Swi6-binding+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000656-Wu2" id="pone.0000656-Wu2"></a>Wu C, Leeuw T, Leberer E, Thomas D, Whiteway M (1998) Cell cycle- and Cln2-Cdc2p-dependent phosphroylation of the Yeast Ste20p protein kinase. J Biol Chem  273: 28107–28115.  <ul class="find" data-citedArticleID="969966" data-doi="10.1074/jbc.273.43.28107"><li><a href="http://dx.doi.org/10.1074/jbc.273.43.28107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle-+and+Cln2-Cdc2p-dependent+phosphroylation+of+the+Yeast+Ste20p+protein+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle-+and+Cln2-Cdc2p-dependent+phosphroylation+of+the+Yeast+Ste20p+protein+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000656-Ubersax1" id="pone.0000656-Ubersax1"></a>Ubersax JA, Woodbury EL, Quang PN, Paraz M, Blethrow JD, et al.  (2003) Targets of the cyclin-dependent kinase Cdk1. Nature  425: 859–864.  <ul class="find" data-citedArticleID="969958" data-doi="10.1038/nature02062"><li><a href="http://dx.doi.org/10.1038/nature02062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targets+of+the+cyclin-dependent+kinase+Cdk1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targets+of+the+cyclin-dependent+kinase+Cdk1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000656-Archambault1" id="pone.0000656-Archambault1"></a>Archambault V, Chang EJ, J DB, Cross FR, Chait BT, et al.  (2004) Targeted proteomic study of the Cyclin-Cdk module. Mol Cell  14: 699–711.  <ul class="find" data-citedArticleID="969836" data-doi="10.1016/j.molcel.2004.05.025"><li><a href="http://dx.doi.org/10.1016/j.molcel.2004.05.025" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Targeted+proteomic+study+of+the+Cyclin-Cdk+module." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Targeted+proteomic+study+of+the+Cyclin-Cdk+module.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000656-Nishizawa1" id="pone.0000656-Nishizawa1"></a>Nishizawa M, Kawasumi M, Fujino M, A T-e (1998) Phosphorylation of sic1, a cyclin-dependent kianse (Cdk) inhibitor, by Cdk including Pho85 kinase is required for its prompt degradation. Mol Biol Cell  9: 2393–2405.  <ul class="find" data-citedArticleID="969932" data-doi="10.1091/mbc.9.9.2393"><li><a href="http://dx.doi.org/10.1091/mbc.9.9.2393" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+sic1%2C+a+cyclin-dependent+kianse+%28Cdk%29+inhibitor%2C+by+Cdk+including+Pho85+kinase+is+required+for+its+prompt+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+sic1%2C+a+cyclin-dependent+kianse+%28Cdk%29+inhibitor%2C+by+Cdk+including+Pho85+kinase+is+required+for+its+prompt+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000656-Holmes2" id="pone.0000656-Holmes2"></a>Holmes J, Solomon M (2001) The role of Thr160 phosphorylation of Cdk2 in substrate recognition. Eur J Biochem  268: 4647–4652.  <ul class="find" data-citedArticleID="969892" data-doi="10.1046/j.1432-1327.2001.02392.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1327.2001.02392.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+Thr160+phosphorylation+of+Cdk2+in+substrate+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+Thr160+phosphorylation+of+Cdk2+in+substrate+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000656-Doolin1" id="pone.0000656-Doolin1"></a>Doolin M, Johnson A, Johnson L, Butler G (2001) Overlapping and distinct roles of the duplicated yeast transcription factors Ace2p and Swi5p. Mol Microbiol  40: 422–432.  <ul class="find" data-citedArticleID="969862" data-doi="10.1046/j.1365-2958.2001.02388.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02388.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overlapping+and+distinct+roles+of+the+duplicated+yeast+transcription+factors+Ace2p+and+Swi5p." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overlapping+and+distinct+roles+of+the+duplicated+yeast+transcription+factors+Ace2p+and+Swi5p.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000656-Nguyen2" id="pone.0000656-Nguyen2"></a>Nguyen VQ, Co C, Li JJ (2001) Cyclin-dependent kinases prevent DNA re-replication through multiple mechanisms. Nature  411: 1068–1073.  <ul class="find" data-citedArticleID="969928"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms.&amp;auth=&amp;atitle=Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyclin-dependent+kinases+prevent+DNA+re-replication+through+multiple+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000656-Reynolds1" id="pone.0000656-Reynolds1"></a>Reynolds D, Shi B, McClean C, Katsis F, Kemp B, et al.  (2003) Recruitment of Thr 319-phosphorylated Ndd1p to the FHA domain of Fkh2p requires Clb kinase activity: a mechanism for CLB cluster gene activation. Genes Dev  17: 1789–1802.  <ul class="find" data-citedArticleID="969946" data-doi="10.1101/gad.1074103"><li><a href="http://dx.doi.org/10.1101/gad.1074103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recruitment+of+Thr+319-phosphorylated+Ndd1p+to+the+FHA+domain+of+Fkh2p+requires+Clb+kinase+activity%3A+a+mechanism+for+CLB+cluster+gene+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recruitment+of+Thr+319-phosphorylated+Ndd1p+to+the+FHA+domain+of+Fkh2p+requires+Clb+kinase+activity%3A+a+mechanism+for+CLB+cluster+gene+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000656-Agarwal1" id="pone.0000656-Agarwal1"></a>Agarwal R, Cohen-Fix O (2002) Phosphorylation of the mitotic regulator Pds1/securin by Cdc28 is required for efficient nuclear localization of Esp1/separase. Genes Dev  16: 1371–1382.  <ul class="find" data-citedArticleID="969832" data-doi="10.1101/gad.971402"><li><a href="http://dx.doi.org/10.1101/gad.971402" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+of+the+mitotic+regulator+Pds1%2Fsecurin+by+Cdc28+is+required+for+efficient+nuclear+localization+of+Esp1%2Fseparase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+of+the+mitotic+regulator+Pds1%2Fsecurin+by+Cdc28+is+required+for+efficient+nuclear+localization+of+Esp1%2Fseparase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000656-Kalkum1" id="pone.0000656-Kalkum1"></a>Kalkum M, Lyon GJ, Chait BT (2003) Detection of secreted peptides by using hypothesis-driven multistage mass spectrometry. Proc Natl Acad Sci, USA  100: 2795–27800.  <ul class="find" data-citedArticleID="969908" data-doi="10.1073/pnas.0436605100"><li><a href="http://dx.doi.org/10.1073/pnas.0436605100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+secreted+peptides+by+using+hypothesis-driven+multistage+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+secreted+peptides+by+using+hypothesis-driven+multistage+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000656-Jones1" id="pone.0000656-Jones1"></a>Jones D (2001) Protein structure prediction in genomics. Brief Bioinform  2: 111–125.  <ul class="find" data-citedArticleID="969906" data-doi="10.1093/bib/2.2.111"><li><a href="http://dx.doi.org/10.1093/bib/2.2.111" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+structure+prediction+in+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+structure+prediction+in+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000656-Eswar1" id="pone.0000656-Eswar1"></a>Eswar N, John B, Mirkovic N, Fiser A, Ilyin V, et al.  (2003) Tools for comparative protein structure modeling and analysis. Nucleic Acids Res  31: 3375–3380.  <ul class="find" data-citedArticleID="969866" data-doi="10.1093/nar/gkg543"><li><a href="http://dx.doi.org/10.1093/nar/gkg543" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tools+for+comparative+protein+structure+modeling+and+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tools+for+comparative+protein+structure+modeling+and+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000656-Rudner1" id="pone.0000656-Rudner1"></a>Rudner A, Murray A (2000) Phosphorylation by Cdc28 activates the Cdc20- dependent activity of the anaphase promoting complex. J Cell Biol  149: 1377–1390.  <ul class="find" data-citedArticleID="969948" data-doi="10.1083/jcb.149.7.1377"><li><a href="http://dx.doi.org/10.1083/jcb.149.7.1377" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphorylation+by+Cdc28+activates+the+Cdc20-+dependent+activity+of+the+anaphase+promoting+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphorylation+by+Cdc28+activates+the+Cdc20-+dependent+activity+of+the+anaphase+promoting+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000656-Hollunder1" id="pone.0000656-Hollunder1"></a>Hollunder J, Beyer A, Wilhelm T (2005) Identification and characterization of protein subcomplexes in yeast. Proteomics  5: 2082–2089.  <ul class="find" data-citedArticleID="969888" data-doi="10.1002/pmic.200401121"><li><a href="http://dx.doi.org/10.1002/pmic.200401121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+characterization+of+protein+subcomplexes+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+characterization+of+protein+subcomplexes+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000656-Ho2" id="pone.0000656-Ho2"></a>Ho Y al. e (2002) Systematic identification of protein complexes in Saccharomyces cerevisiae by mass spectrometry. Nature  415: 180–183.  <ul class="find" data-citedArticleID="969886" data-doi="10.1038/415180a"><li><a href="http://dx.doi.org/10.1038/415180a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+identification+of+protein+complexes+in+Saccharomyces+cerevisiae+by+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+identification+of+protein+complexes+in+Saccharomyces+cerevisiae+by+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000656-Gavin1" id="pone.0000656-Gavin1"></a>Gavin A al. e (2002) Functional organization of the yeast proteome by systematic analysis of protein complexes. Nature  415: 141–147.  <ul class="find" data-citedArticleID="969880" data-doi="10.1038/415141a"><li><a href="http://dx.doi.org/10.1038/415141a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+organization+of+the+yeast+proteome+by+systematic+analysis+of+protein+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+organization+of+the+yeast+proteome+by+systematic+analysis+of+protein+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000656-Flotow1" id="pone.0000656-Flotow1"></a>Flotow H, Graves PR, Wang A, Fiol CJ, Roeske RW, et al.  (1990) Phosphate groups as substrate determinants for casein kinase I action. J Biol Chem  265: 14264–14269.  <ul class="find" data-citedArticleID="969872"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phosphate+groups+as+substrate+determinants+for+casein+kinase+I+action.&amp;auth=&amp;atitle=Phosphate+groups+as+substrate+determinants+for+casein+kinase+I+action." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphate+groups+as+substrate+determinants+for+casein+kinase+I+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphate+groups+as+substrate+determinants+for+casein+kinase+I+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000656-Flotow2" id="pone.0000656-Flotow2"></a>Flotow H, Roach PJ (1991) Role of acidic residues as substrate determinants for casein kinase I. J Biol Chem 266: 3724–3727.  <ul class="find" data-citedArticleID="969874"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Role+of+acidic+residues+as+substrate+determinants+for+casein+kinase+I.+J+Biol+Chem&amp;auth=&amp;atitle=Role+of+acidic+residues+as+substrate+determinants+for+casein+kinase+I.+J+Biol+Chem" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+acidic+residues+as+substrate+determinants+for+casein+kinase+I.+J+Biol+Chem" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+acidic+residues+as+substrate+determinants+for+casein+kinase+I.+J+Biol+Chem%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000656-Fiol1" id="pone.0000656-Fiol1"></a>Fiol CJ, Marenholz A, Wang Y, Roeske R, Roach PJ (1987) Formation of protein kinase recognition sites by covalent modification of the substrate. Molecular mechanism for the synergistic action of casein kinase II and glycogen synthase kinase 3. J Biol Chem  262: 14042–14048.  <ul class="find" data-citedArticleID="969870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Formation+of+protein+kinase+recognition+sites+by+covalent+modification+of+the+substrate.+Molecular+mechanism+for+the+synergistic+action+of+casein+kinase+II+and+glycogen+synthase+kinase+3.&amp;auth=&amp;atitle=Formation+of+protein+kinase+recognition+sites+by+covalent+modification+of+the+substrate.+Molecular+mechanism+for+the+synergistic+action+of+casein+kinase+II+and+glycogen+synthase+kinase+3." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Formation+of+protein+kinase+recognition+sites+by+covalent+modification+of+the+substrate.+Molecular+mechanism+for+the+synergistic+action+of+casein+kinase+II+and+glycogen+synthase+kinase+3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Formation+of+protein+kinase+recognition+sites+by+covalent+modification+of+the+substrate.+Molecular+mechanism+for+the+synergistic+action+of+casein+kinase+II+and+glycogen+synthase+kinase+3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000656-Frame1" id="pone.0000656-Frame1"></a>Frame S, Cohen P, Biondi R (2001) A Common Phosphate Binding Site Explains the Unique Substrate Specificity of GSK3 and Its Inactivation by Phosphorylation. Mol Cell  7: 1321–1327.  <ul class="find" data-citedArticleID="969876" data-doi="10.1016/s1097-2765(01)00253-2"><li><a href="http://dx.doi.org/10.1016/s1097-2765(01)00253-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Common+Phosphate+Binding+Site+Explains+the+Unique+Substrate+Specificity+of+GSK3+and+Its+Inactivation+by+Phosphorylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Common+Phosphate+Binding+Site+Explains+the+Unique+Substrate+Specificity+of+GSK3+and+Its+Inactivation+by+Phosphorylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000656-Aubol1" id="pone.0000656-Aubol1"></a>Aubol B, Nolen B, Vu D, Gourisankar G, Adams J (2002) Mechanistic Insights into Sky1p, a Yeast Homologue of the Mammalian SR Protein Kinases. Biochemistry  41: 10002–10009.  <ul class="find" data-citedArticleID="969838" data-doi="10.1021/bi020233y"><li><a href="http://dx.doi.org/10.1021/bi020233y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanistic+Insights+into+Sky1p%2C+a+Yeast+Homologue+of+the+Mammalian+SR+Protein+Kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanistic+Insights+into+Sky1p%2C+a+Yeast+Homologue+of+the+Mammalian+SR+Protein+Kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000656-VelazquezDones1" id="pone.0000656-VelazquezDones1"></a>Velazquez-Dones A, Hagopian J, Ma C, Zhong X, Zhou H, et al.  (2005) Mass spectrometric and kinetic analysis of ASF/SF2 phosphorylation by SRPK1 and Clk/Sty. J Biol Chem  280: 41761.  <ul class="find" data-citedArticleID="969960" data-doi="10.1074/jbc.m504156200"><li><a href="http://dx.doi.org/10.1074/jbc.m504156200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mass+spectrometric+and+kinetic+analysis+of+ASF%2FSF2+phosphorylation+by+SRPK1+and+Clk%2FSty." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mass+spectrometric+and+kinetic+analysis+of+ASF%2FSF2+phosphorylation+by+SRPK1+and+Clk%2FSty.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000656-Available1" id="pone.0000656-Available1"></a>  Available: <a href="http://www.ebi.ac.uk/clustalw/">http://www.ebi.ac.uk/clustalw/</a>.  <ul class="find-nolinks"></ul></li><li><span class="label">69.
              </span><a name="pone.0000656-Ramu1" id="pone.0000656-Ramu1"></a>Ramu C, Sugawar H, Koike T, Lopez R, Gibson T, et al.  (2003) Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res  31: 3497–3500.  <ul class="find" data-citedArticleID="969944" data-doi="10.1093/nar/gkg500"><li><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+the+Clustal+series+of+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+the+Clustal+series+of+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.XML" value="147748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.PDF" value="288200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t001.PNG_L" value="90774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t001.PNG_M" value="104120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t001.PNG_S" value="17529"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t001.TIF" value="526014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t001.PNG_I" value="46420"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g001.PNG_L" value="110856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g001.PNG_M" value="70160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g001.PNG_S" value="8768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g001.TIF" value="1057908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g001.PNG_I" value="35175"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t002.PNG_L" value="316566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t002.PNG_M" value="46486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t002.PNG_S" value="14445"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t002.TIF" value="2550918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.t002.PNG_I" value="63432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g002.PNG_L" value="653065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g002.PNG_M" value="73196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g002.PNG_S" value="9299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g002.TIF" value="1754362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.g002.PNG_I" value="37779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.e001.PNG" value="11302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.e001.TIF" value="27184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.e002.PNG" value="14964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.e002.TIF" value="31360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.s001.DOC" value="32768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.s002.DOC" value="36864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000656.s003.DOC" value="40960"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000656&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000656" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000656&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000656&volume=&issue=&title=Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates&author_name=Emmanuel%20J.%20Chang%2C%20Rashida%20Begum%2C%20Brian%20T.%20Chait%2C%20Terry%20Gaasterland&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000656" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656&amp;t=Prediction%20of%20Cyclin-Dependent%20Kinase%20Phosphorylation%20Substrates" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656&title=Prediction%20of%20Cyclin-Dependent%20Kinase%20Phosphorylation%20Substrates&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656&amp;title=Prediction%20of%20Cyclin-Dependent%20Kinase%20Phosphorylation%20Substrates" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000656&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Prediction of Cyclin-Dependent Kinase Phosphorylation Substrates';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000656';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Prediction%20of%20Cyclin-Dependent%20Kinase%20Phosphorylation%20Substrates http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000656" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000656" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cyclins%22" title="Search for articles in the subject area:'Cyclins'"><div class="flagText">Cyclins</div></a>
              <div data-categoryid="22357" data-articleid="24150"
                   data-categoryname="Cyclins"
                   class="flagImage" title="Flag 'Cyclins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phosphorylation%22" title="Search for articles in the subject area:'Phosphorylation'"><div class="flagText">Phosphorylation</div></a>
              <div data-categoryid="42723" data-articleid="24150"
                   data-categoryname="Phosphorylation"
                   class="flagImage" title="Flag 'Phosphorylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+sequencing%22" title="Search for articles in the subject area:'Protein sequencing'"><div class="flagText">Protein sequencing</div></a>
              <div data-categoryid="20569" data-articleid="24150"
                   data-categoryname="Protein sequencing"
                   class="flagImage" title="Flag 'Protein sequencing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomes%22" title="Search for articles in the subject area:'Proteomes'"><div class="flagText">Proteomes</div></a>
              <div data-categoryid="18649" data-articleid="24150"
                   data-categoryname="Proteomes"
                   class="flagImage" title="Flag 'Proteomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="43143" data-articleid="24150"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="47429" data-articleid="24150"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24150"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="24150"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4057'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8337'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2054&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=42457">Useful, incremental </a><br>Posted by pedrobeltrao</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
