

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Regulation of Plant Developmental Processes by a Novel Splicing Factor</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000471"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000471" />

    <meta name="citation_title" content="Regulation of Plant Developmental Processes by a Novel Splicing Factor"/>
    <meta itemprop="name" content="Regulation of Plant Developmental Processes by a Novel Splicing Factor"/>

      <meta name="citation_author" content="Gul Shad Ali"/>
            <meta name="citation_author_institution" content="Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America"/>
      <meta name="citation_author" content="Saiprasad G. Palusa"/>
            <meta name="citation_author_institution" content="Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America"/>
      <meta name="citation_author" content="Maxim Golovkin"/>
            <meta name="citation_author_institution" content="Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America"/>
      <meta name="citation_author" content="Jayendra Prasad"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Columbia University, New York, New York, United States of America"/>
      <meta name="citation_author" content="James L. Manley"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, Columbia University, New York, New York, United States of America"/>
      <meta name="citation_author" content="Anireddy S.N. Reddy"/>
            <meta name="citation_author_institution" content="Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America"/>

    <meta name="citation_date" content="2007/5/30"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000471.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e471"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Alternative Splicing of Pre-Messenger RNAs in Plants in the Genomic Era.; citation_author=AS Reddy; citation_journal_title=Annu Rev Plant Biol; citation_volume=58; citation_number=1; citation_pages=267-294; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=SRprises along a messenger's journey.; citation_author=Y Huang; citation_author=JA Steitz; citation_journal_title=Mol Cell; citation_volume=17; citation_number=2; citation_pages=613-615; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sorting out the complexity of SR protein functions.; citation_author=BR Graveley; citation_journal_title=RNA; citation_volume=6; citation_number=3; citation_pages=1197-1211; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiple roles of the SR protein family in splicing regulation.; citation_author=JR Sanford; citation_author=D Longman; citation_author=JF Caceres; citation_number=4; citation_pages=33-58; citation_date=2003; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=Nuclear pre-mRNA splicing in plants.; citation_author=ASN Reddy; citation_journal_title=CRC Crit Rev Plant Sci; citation_volume=20; citation_number=5; citation_pages=523-572; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Arginine-serine-rich domains bound at splicing enhancers contact the branchpoint to promote prespliceosome assembly.; citation_author=H Shen; citation_author=JL Kan; citation_author=MR Green; citation_journal_title=Mol Cell; citation_volume=13; citation_number=6; citation_pages=367-376; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=SR proteins and splicing control.; citation_author=JL Manley; citation_author=R Tacke; citation_journal_title=Genes Dev; citation_volume=10; citation_number=7; citation_pages=1569-1579; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Specific interactions between proteins implicated in splice site selection and regulated alternative splicing.; citation_author=JY Wu; citation_author=T Maniatis; citation_journal_title=Cell; citation_volume=75; citation_number=8; citation_pages=1061-1070; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Structure and expression of a plant U1 snRNP 70K gene: alternative splicing of U1 snRNP 70K pre-mRNAs produces two different transcripts.; citation_author=M Golovkin; citation_author=ASN Reddy; citation_journal_title=Plant Cell; citation_volume=8; citation_number=9; citation_pages=1421-1435; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Plant serine/arginine-rich proteins and their role in pre-mRNA splicing.; citation_author=AS Reddy; citation_journal_title=Trends Plant Sci; citation_volume=9; citation_number=10; citation_pages=541-547; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A plethora of plant serine/arginine-rich proteins: redundancy or evolution of novel gene functions?; citation_author=M Kalyna; citation_author=A Barta; citation_journal_title=Biochem Soc Trans; citation_volume=32; citation_number=11; citation_pages=561-564; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The serine/arginine-rich protein family in rice plays important roles in constitutive and alternative splicing of pre-mRNA.; citation_author=M Isshiki; citation_author=A Tsumoto; citation_author=K Shimamoto; citation_journal_title=Plant Cell; citation_volume=18; citation_number=12; citation_pages=146-158; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Survey of Conserved Alternative Splicing Events of mRNAs Encoding SR Proteins in Land Plants.; citation_author=K Iida; citation_author=M Go; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=13; citation_pages=1085-1094; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Overexpression of the arginine-rich carboxy-terminal region of U1 snRNP 70K inhibits both splicing and nucleocytoplasmic transport of mRNA.; citation_author=JM Romac; citation_author=JD Keene; citation_journal_title=Genes Dev; citation_volume=9; citation_number=14; citation_pages=1400-1410; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Protein-protein interactions and 5′-splice-site recognition in mammalian mRNA precursors.; citation_author=JD Kohtz; citation_author=SF Jamison; citation_author=CL Will; citation_author=P Zuo; citation_author=R Lührmann; citation_journal_title=Nature; citation_volume=368; citation_number=15; citation_pages=119-124; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The plant U1 small nuclear ribonucleoprotein particle 70K protein interacts with two novel serine/arginine-rich proteins.; citation_author=M Golovkin; citation_author=ASN Reddy; citation_journal_title=Plant Cell; citation_volume=10; citation_number=16; citation_pages=1637-1648; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=An SC35-like protein and a novel serine/arginine-rich protein interact with Arabidopsis U1-70K protein.; citation_author=M Golovkin; citation_author=ASN Reddy; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=17; citation_pages=36428-36438; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Network of interactions of a novel plant-specific Arg/Ser-rich protein, atRSZ33, with atSC35-like splicing factors.; citation_author=S Lopato; citation_author=C Forstner; citation_author=M Kalyna; citation_author=J Hilscher; citation_author=U Langhammer; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=18; citation_pages=39989-39998; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ATP, phosphorylation and transcription regulate the mobility of plant splicing factors.; citation_author=GS Ali; citation_author=AS Reddy; citation_journal_title=J Cell Sci; citation_volume=119; citation_number=19; citation_pages=3527-3538; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nuclear localization and in vivo dynamics of a plant-specific serine/arginine-rich protein.; citation_author=GS Ali; citation_author=M Golovkin; citation_author=AS Reddy; citation_journal_title=Plant J; citation_volume=36; citation_number=20; citation_pages=883-893; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The ASRG database: identification and survey of Arabidopsis thaliana genes involved in pre-mRNA splicing.; citation_author=BB Wang; citation_author=V Brendel; citation_journal_title=Genome Biol; citation_volume=5; citation_number=21; citation_pages=R102; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Human Tra2 proteins are sequence-specific activators of pre-mRNA splicing.; citation_author=R Tacke; citation_author=M Tohyama; citation_author=S Ogawa; citation_author=JL Manley; citation_journal_title=Cell; citation_volume=93; citation_number=22; citation_pages=139-148; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=REMEMBERING WINTER: Toward a Molecular Understanding of Vernalization.; citation_author=S Sung; citation_author=RM Amasino; citation_journal_title=Annu Rev Plant Biol; citation_volume=56; citation_number=23; citation_pages=491-508; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Multiple Pathways in the Decision to Flower: Enabling, Promoting, and Resetting.; citation_author=PK Boss; citation_author=RM Bastow; citation_author=JS Mylne; citation_author=C Dean; citation_volume=16; citation_number=24; citation_pages=S18-S31; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The FLF MADS box gene: a repressor of flowering in Arabidopsis regulated by vernalization and methylation.; citation_author=CC Sheldon; citation_author=JE Burn; citation_author=PP Perez; citation_author=J Metzger; citation_author=JA Edwards; citation_journal_title=Plant Cell; citation_volume=11; citation_number=25; citation_pages=445-458; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=FLOWERING LOCUS C encodes a novel MADS domain protein that acts as a repressor of flowering.; citation_author=SD Michaels; citation_author=RM Amasino; citation_journal_title=Plant Cell; citation_volume=11; citation_number=26; citation_pages=949-956; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Functional significance of the alternative transcript processing of the Arabidopsis floral promoter FCA.; citation_author=R Macknight; citation_author=M Duroux; citation_author=R Laurie; citation_author=P Dijkwel; citation_author=G Simpson; citation_journal_title=Plant Cell; citation_volume=14; citation_number=27; citation_pages=877-888; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=FY is an RNA 3′ end-processing factor that interacts with FCA to control the Arabidopsis floral transition.; citation_author=GG Simpson; citation_author=PP Dijkwel; citation_author=V Quesada; citation_author=I Henderson; citation_author=C Dean; citation_journal_title=Cell; citation_volume=113; citation_number=28; citation_pages=777-787; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Regulation of flowering time by FVE, a retinoblastoma-associated protein.; citation_author=I Ausin; citation_author=C Alonso-Blanco; citation_author=JA Jarillo; citation_author=L Ruiz-Garcia; citation_author=JM Martinez-Zapater; citation_journal_title=Nat Genet; citation_volume=36; citation_number=29; citation_pages=162-166; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Regulation of flowering time by histone acetylation in Arabidopsis.; citation_author=Y He; citation_author=SD Michaels; citation_author=RM Amasino; citation_journal_title=Science; citation_volume=302; citation_number=30; citation_pages=1751-1754; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Isolation of LUMINIDEPENDENS: a gene involved in the control of flowering time in Arabidopsis.; citation_author=I Lee; citation_author=MJ Aukerman; citation_author=SL Gore; citation_author=KN Lohman; citation_author=SD Michaels; citation_journal_title=Plant Cell; citation_volume=6; citation_number=31; citation_pages=75-83; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Regulation of flowering time in Arabidopsis by K homology domain proteins.; citation_author=TC Mockler; citation_author=X Yu; citation_author=D Shalitin; citation_author=D Parikh; citation_author=TP Michael; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=32; citation_pages=12759-12764; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A new Arabidopsis gene, FLK, encodes an RNA binding protein with K homology motifs and regulates flowering time via FLOWERING LOCUS C.; citation_author=MH Lim; citation_author=J Kim; citation_author=YS Kim; citation_author=KS Chung; citation_author=YH Seo; citation_journal_title=Plant Cell; citation_volume=16; citation_number=33; citation_pages=731-740; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=FPA, a gene involved in floral induction in Arabidopsis, encodes a protein containing RNA-recognition motifs.; citation_author=FM Schomburg; citation_author=DA Patton; citation_author=DW Meinke; citation_author=RM Amasino; citation_journal_title=Plant Cell; citation_volume=13; citation_number=34; citation_pages=1427-1436; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The VERNALIZATION 2 gene mediates the epigenetic regulation of vernalization in Arabidopsis.; citation_author=AR Gendall; citation_author=YY Levy; citation_author=A Wilson; citation_author=C Dean; citation_journal_title=Cell; citation_volume=107; citation_number=35; citation_pages=525-535; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The AGAMOUS-LIKE 20 MADS domain protein integrates floral inductive pathways in Arabidopsis.; citation_author=H Lee; citation_author=SS Suh; citation_author=E Park; citation_author=E Cho; citation_author=JH Ahn; citation_journal_title=Genes Dev; citation_volume=14; citation_number=36; citation_pages=2366-2376; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Activation tagging of the floral inducer FT.; citation_author=I Kardailsky; citation_author=VK Shukla; citation_author=JH Ahn; citation_author=N Dagenais; citation_author=SK Christensen; citation_journal_title=Science; citation_volume=286; citation_number=37; citation_pages=1962-1965; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The CONSTANS gene of Arabidopsis promotes flowering and encodes a protein showing similarities to zinc finger transcription factors.; citation_author=J Putterill; citation_author=F Robson; citation_author=K Lee; citation_author=R Simon; citation_author=G Coupland; citation_journal_title=Cell; citation_volume=80; citation_number=38; citation_pages=847-857; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Nuclear speckles: a model for nuclear organelles.; citation_author=AI Lamond; citation_author=DL Spector; citation_journal_title=Nat Rev Mol Cell Biol; citation_volume=4; citation_number=39; citation_pages=605-612; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mechanism of Leaf Shape Determination.; citation_author=H Tsukaya; citation_journal_title=Annu Rev Plant Biol; citation_volume=57; citation_number=40; citation_pages=477-496; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Networks in leaf development.; citation_author=ME Byrne; citation_journal_title=Curr Opin Plant Biol; citation_volume=8; citation_number=41; citation_pages=59-66; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The transcription factor AtGRF5 and the transcription coactivator AN3 regulate cell proliferation in leaf primordia of Arabidopsis thaliana.; citation_author=G Horiguchi; citation_author=GT Kim; citation_author=H Tsukaya; citation_journal_title=Plant J; citation_volume=43; citation_number=42; citation_pages=68-78; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The CURLY LEAF gene controls both division and elongation of cells during the expansion of the leaf blade in Arabidopsis thaliana.; citation_author=GT Kim; citation_author=H Tsukaya; citation_author=H Uchimiya; citation_journal_title=Planta; citation_volume=206; citation_number=43; citation_pages=175-183; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=DRL1, a homolog of the yeast TOT4/KTI12 protein, has a function in meristem activity and organ growth in plants.; citation_author=H Nelissen; citation_author=JH Clarke; citation_author=M De Block; citation_author=S De Block; citation_author=R Vanderhaeghen; citation_journal_title=Plant Cell; citation_volume=15; citation_number=44; citation_pages=639-654; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Conservation and molecular dissection of ROUGH SHEATH2 and ASYMMETRIC LEAVES1 function in leaf development.; citation_author=G Theodoris; citation_author=N Inada; citation_author=M Freeling; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=45; citation_pages=6837-6842; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ASYMMETRIC LEAVES2-LIKE1 gene, a member of the AS2/LOB family, controls proximal-distal patterning in Arabidopsis petals.; citation_author=A Chalfun-Junior; citation_author=J Franken; citation_author=JJ Mes; citation_author=N Marsch-Martinez; citation_author=A Pereira; citation_journal_title=Plant Mol Biol; citation_volume=57; citation_number=46; citation_pages=559-575; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Sugar-dependent alterations in cotyledon and leaf development in transgenic plants expressing the HDZhdip gene ATHB13.; citation_author=J Hanson; citation_author=H Johannesson; citation_author=P Engstrom; citation_journal_title=Plant Mol Biol; citation_volume=45; citation_number=47; citation_pages=247-262; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Wall-associated kinases are expressed throughout plant development and are required for cell expansion.; citation_author=TA Wagner; citation_author=BD Kohorn; citation_journal_title=Plant Cell; citation_volume=13; citation_number=48; citation_pages=303-318; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The too many mouths and four lips mutations affect stomatal production in Arabidopsis.; citation_author=M Yang; citation_author=FD Sack; citation_journal_title=Plant Cell; citation_volume=7; citation_number=49; citation_pages=2227-2239; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A subtilisin-like serine protease involved in the regulation of stomatal density and distribution in Arabidopsis thaliana.; citation_author=D Berger; citation_author=T Altmann; citation_journal_title=Genes Dev; citation_volume=14; citation_number=50; citation_pages=1119-1131; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Stomatal development and pattern controlled by a MAPKK kinase.; citation_author=DC Bergmann; citation_author=W Lukowitz; citation_author=CR Somerville; citation_journal_title=Science; citation_volume=304; citation_number=51; citation_pages=1494-1497; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stomatal patterning and differentiation by synergistic interactions of receptor kinases.; citation_author=ED Shpak; citation_author=JM McAbee; citation_author=LJ Pillitteri; citation_author=KU Torii; citation_journal_title=Science; citation_volume=309; citation_number=52; citation_pages=290-293; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular and Genetic Mechanisms of Floral Control.; citation_author=T Jack; citation_journal_title=The Plant Cell; citation_volume=16; citation_number=53; citation_pages=S1-S17; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cloning of the Arabidopsis WIGGUM gene identifies a role for farnesylation in meristem development.; citation_author=EC Ziegelhoffer; citation_author=LJ Medrano; citation_author=EM Meyerowitz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=54; citation_pages=7633-7638; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The ULTRAPETALA gene controls shoot and floral meristem size in Arabidopsis.; citation_author=JC Fletcher; citation_journal_title=Development; citation_volume=128; citation_number=55; citation_pages=1323-1333; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Enlarged meristems and delayed growth in plp mutants result from lack of CaaX prenyltransferases.; citation_author=MP Running; citation_author=M Lavy; citation_author=H Sternberg; citation_author=A Galichet; citation_author=W Gruissem; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=56; citation_pages=7815-7820; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=PIE1, an ISWI family gene, is required for FLC activation and floral repression in Arabidopsis.; citation_author=YS Noh; citation_author=RM Amasino; citation_journal_title=Plant Cell; citation_volume=15; citation_number=57; citation_pages=1671-1682; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CLAVATA1, a regulator of meristem and flower development in Arabidopsis.; citation_author=SE Clark; citation_author=MP Running; citation_author=EM Meyerowitz; citation_journal_title=Development; citation_volume=119; citation_number=58; citation_pages=397-418; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=CLAVATA2, a regulator of meristem and organ development in Arabidopsis.; citation_author=JM Kayes; citation_author=SE Clark; citation_journal_title=Development; citation_volume=125; citation_number=59; citation_pages=3843-3851; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=CLAVATA3 is a specific regulator of shoot and floral meristem development affecting the same processes as CLAVATA1.; citation_author=SE Clark; citation_author=MP Running; citation_author=EM Meyerowitz; citation_journal_title=Development; citation_volume=121; citation_number=60; citation_pages=2057-2067; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Alternative splicing of pre-mRNAs of Arabidopsis serine/arginine-rich proteins: regulation by hormones and stresses.; citation_author=SG Palusa; citation_author=GS Ali; citation_author=AS Reddy; citation_journal_title=Plant J; citation_volume=49; citation_number=61; citation_pages=1091-1107; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=atSRp30, one of two SF2/ASF-like proteins from Arabidopsis thaliana, regulates splicing of specific plant genes.; citation_author=S Lopato; citation_author=M Kalyna; citation_author=S Dorner; citation_author=R Kobayashi; citation_author=AR Krainer; citation_journal_title=Genes Dev; citation_volume=13; citation_number=62; citation_pages=987-1001; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The Arabidopsis splicing factor SR1 is regulated by alternative splicing.; citation_author=G Lazar; citation_author=HM Goodman; citation_journal_title=Plant Mol Biol; citation_volume=42; citation_number=63; citation_pages=571-581; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ectopic expression of atRSZ33 reveals its function in splicing and causes pleiotropic changes in development.; citation_author=M Kalyna; citation_author=S Lopato; citation_author=A Barta; citation_journal_title=Mol Biol Cell; citation_volume=14; citation_number=64; citation_pages=3565-3577; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools.; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=65; citation_pages=4876-4882; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The protein kinase Clk/Sty directly modulates SR protein activity: both hyper-and hypophosphorylation inhibit splicing.; citation_author=J Prasad; citation_author=K Colwill; citation_author=T Pawson; citation_author=JL Manley; citation_journal_title=Mol Cell Biol; citation_volume=19; citation_number=66; citation_pages=6991-7000; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection.; citation_author=C Li; citation_author=WH Wong; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=67; citation_pages=31-36; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Controlling the false discovery rate—A practical and powerful approach to multiple testing.; citation_author=Y Benjamini; citation_author=Y Hochberg; citation_journal_title=J R Stat Soc Ser B Methodol; citation_volume=57; citation_number=68; citation_pages=289-300; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=KIC, a novel Ca2+ binding protein with one EF-hand motif, interacts with a microtubule motor protein and regulates trichome morphogenesis.; citation_author=VS Reddy; citation_author=IS Day; citation_author=T Thomas; citation_author=AS Reddy; citation_journal_title=Plant Cell; citation_volume=16; citation_number=69; citation_pages=185-200; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Role of chromatin modification in flowering-time control.; citation_author=Y He; citation_author=RM Amasino; citation_journal_title=Trends Plant Sci; citation_volume=10; citation_number=70; citation_pages=30-35; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Regulation of Plant Developmental Processes by a Novel Splicing Factor"/>
    <meta name="twitter:description" content="Serine/arginine-rich (SR) proteins play important roles in constitutive and alternative splicing and other aspects of mRNA metabolism. We have previously isolated a unique plant SR protein (SR45) with atypical domain organization. However, the biological and molecular functions of this novel SR protein are not known. Here, we report biological and molecular functions of this protein. Using an in vitro splicing complementation assay, we showed that SR45 functions as an essential splicing factor. Furthermore, the alternative splicing pattern of transcripts of several other SR genes was altered in a mutant, sr45-1, suggesting that the observed phenotypic abnormalities in sr45-1 are likely due to altered levels of SR protein isoforms, which in turn modulate splicing of other pre-mRNAs. sr45-1 exhibited developmental abnormalities, including delayed flowering, narrow leaves and altered number of petals and stamens. The late flowering phenotype was observed under both long days and short days and was rescued by vernalization. FLC, a key flowering repressor, is up-regulated in sr45-1 demonstrating that SR45 influences the autonomous flowering pathway. Changes in the alternative splicing of SR genes and the phenotypic defects in the mutant were rescued by SR45 cDNA, further confirming that the observed defects in the mutant are due to the lack of SR45. These results indicate that SR45 is a novel plant-specific splicing factor that plays a crucial role in regulating developmental processes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000471.g008"/>

  <meta property="og:title" content="Regulation of Plant Developmental Processes by a Novel Splicing Factor" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3927'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4113'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3376&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000471">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000471" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000471&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Regulation of Plant Developmental Processes by a Novel Splicing Factor
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gul Shad Ali, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Saiprasad G. Palusa, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maxim Golovkin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America
                </p>

                <p>
                    Current address: Thomas Jefferson University, Philadelphia, Pennsylvania 19107, USA.
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jayendra Prasad, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Columbia University, New York, New York, United States of America
                </p>

                <p>
                    Current address: Department of Immunology, NE4-254, Lerner Research Institute, Cleveland Clinic Foundation, Cleveland, Ohio 44195, USA.
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            James L. Manley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, Columbia University, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Anireddy S.N. Reddy
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:reddy@colostate.edu">reddy@colostate.edu</a></p>

                <p>Affiliation:
                  Department of Biology and Program in Molecular Plant Biology, Colorado State University, Fort Collins, Colorado, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: May 30, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000471</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000471-g001" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g002" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g003" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g004" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g005" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g006" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g007" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000471-g008" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000471">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000471" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Serine/arginine-rich (SR) proteins play important roles in constitutive and alternative splicing and other aspects of mRNA metabolism. We have previously isolated a unique plant SR protein (SR45) with atypical domain organization. However, the biological and molecular functions of this novel SR protein are not known. Here, we report biological and molecular functions of this protein. Using an in vitro splicing complementation assay, we showed that SR45 functions as an essential splicing factor. Furthermore, the alternative splicing pattern of transcripts of several other SR genes was altered in a mutant, <em>sr45-1</em>, suggesting that the observed phenotypic abnormalities in <em>sr45-1</em> are likely due to altered levels of SR protein isoforms, which in turn modulate splicing of other pre-mRNAs. <em>sr45-1</em> exhibited developmental abnormalities, including delayed flowering, narrow leaves and altered number of petals and stamens. The late flowering phenotype was observed under both long days and short days and was rescued by vernalization. <em>FLC</em>, a key flowering repressor, is up-regulated in <em>sr45-1</em> demonstrating that SR45 influences the autonomous flowering pathway. Changes in the alternative splicing of <em>SR</em> genes and the phenotypic defects in the mutant were rescued by <em>SR45</em> cDNA, further confirming that the observed defects in the mutant are due to the lack of SR45. These results indicate that SR45 is a novel plant-specific splicing factor that plays a crucial role in regulating developmental processes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Ali GS, Palusa SG, Golovkin M, Prasad J, Manley JL, et al.  (2007) Regulation of Plant Developmental Processes by a Novel Splicing Factor. PLoS ONE 2(5):
          e471.
            doi:10.1371/journal.pone.0000471</p><p><strong>Academic Editor: </strong>Stefan Kepinski, University of Leeds, United Kingdom</p><p><strong>Received:</strong> January 8, 2007; <strong>Accepted:</strong> April 28, 2007; <strong>Published:</strong> May 30, 2007</p><p><strong>Copyright:</strong> © 2007 Ali et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant from the Department of Energy (DE-FG02-04ER15556) to ASNR and NIH grant GM048259 to JLM.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>SR proteins constitute a highly conserved family of structurally and functionally related non-snRNP proteins with multiple roles in pre-mRNA splicing and other aspects of RNA metabolism <a href="#pone.0000471-Reddy1">[1]</a>–<a href="#pone.0000471-Reddy2">[5]</a>. These proteins have a modular domain structure with one or two N-terminal RNA recognition motifs (RRMs) and a C-terminal arginine/serine-rich (RS) domain. The RRM, which confers RNA-binding specificity, binds to specific regulatory sequences in pre-mRNA, and the RS domain mediates protein-protein and protein-RNA interactions in the splicing machinery <a href="#pone.0000471-Graveley1">[3]</a>, <a href="#pone.0000471-Shen1">[6]</a>. An important feature of SR proteins is that any one of them can complement splicing-deficient S100 extracts in splicing of pre-mRNA substrates with consensus splice sites <a href="#pone.0000471-Manley1">[7]</a>. Based on this property, SR proteins have been called essential or general splicing factors. SR proteins in animals function as essential splicing factors in constitutive pre-mRNA splicing and also regulate alternative splicing (AS) by influencing splice site selection in a concentration-dependent manner <a href="#pone.0000471-Graveley1">[3]</a>, <a href="#pone.0000471-Manley1">[7]</a>. During formation of the spliceosomal E-complex, ASF (alternative splicing factor)/SF2 (splicing factor 2), one of the SR proteins, recruits U1 snRNP to the 5′ splice site by interacting simultaneously with the pre-mRNA and the U1-70K protein <a href="#pone.0000471-Graveley1">[3]</a>, <a href="#pone.0000471-Shen1">[6]</a>. SR proteins (e.g., SC35 and ASF/SF2) are also involved in bridging 5′ and 3′ splice sites by interacting concurrently with U1-70K and U2AF<sup>35</sup> <a href="#pone.0000471-Wu1">[8]</a>. Furthermore, SR proteins facilitate incorporation of the tri-snRNP complex (U4/U6.U5 snRNP) into the spliceosome and promote base pairing between U2 and U6 snRNA <a href="#pone.0000471-Graveley1">[3]</a>, <a href="#pone.0000471-Manley1">[7]</a>. Some animal SR proteins that shuttle between the nucleus and cytoplasm function in mRNA export, mRNA stability and/or translation <a href="#pone.0000471-Reddy1">[1]</a>. These studies underscore the importance of this family of proteins in RNA metabolism. Although, SR proteins show functional redundancy in in vitro splicing assays, in vivo studies with non-plant systems indicate that some SR proteins are redundant whereas others are not <a href="#pone.0000471-Sanford1">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In animals there are 11 SR splicing factors whereas this family of proteins is considerably expanded in plants with 19 in Arabidopsis <a href="#pone.0000471-Golovkin1">[9]</a>–<a href="#pone.0000471-Kalyna1">[11]</a> and 23 in rice <a href="#pone.0000471-Isshiki1">[12]</a>, <a href="#pone.0000471-Iida1">[13]</a>. Part of this expansion is attributed to differences in splice site recognition between plants and animals <a href="#pone.0000471-Reddy3">[10]</a>. In support of this notion, several SR proteins have been shown to interact with U1-70K, a U1 snRNP specific protein that plays important roles in constitutive and regulated splicing <a href="#pone.0000471-Reddy1">[1]</a>, <a href="#pone.0000471-Romac1">[14]</a>–<a href="#pone.0000471-Lopato1">[18]</a>. Some plant SR proteins appear to be orthologs of metazoan SR proteins whereas others are unique to plants with novel structural features. Seven of the 19 Arabidopsis SR proteins have no counterparts in animal systems <a href="#pone.0000471-Reddy3">[10]</a>, <a href="#pone.0000471-Kalyna1">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>SR45 was isolated as a U1-70K interacting protein in a yeast two-hybrid screen <a href="#pone.0000471-Golovkin3">[17]</a>. Unlike typical SR proteins, which contain a single RS domain at the C-terminus, SR45 has two distinct RS domains, one on either side of the RRM (<a href="#pone-0000471-g001">Figure 1A</a>). Using a GFP-SR45 fusion protein, we showed that SR45 localizes to nuclear speckles and its mobility is regulated by ATP and transcription <a href="#pone.0000471-Ali1">[19]</a>, <a href="#pone.0000471-Ali2">[20]</a>. However, the molecular and biological functions of SR45 protein are not known. In this study, using an in vitro splicing complementation assay, we show that SR45 is an essential splicing factor. Further, the splicing patterns of several other SR genes is affected in a T-DNA insertion mutant, <em>sr45-1</em>, suggesting that, in addition to its role in constitutive splicing, it also plays a role in regulated splicing. Phenotypic analyses of <em>sr45-1</em> plants indicate an important role for SR45 in regulating multiple plant-specific developmental processes including plant size, flowering time and morphology of organs. The work reported here is significant because it shows not only that SR45 affects several developmental processes but also that a plant-specific SR protein with novel domain organization is a bonafide SR splicing factor.</p>
<div class="figure" id="pone-0000471-g001"><div class="img"><a name="pone-0000471-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Arabidopsis SR45 is a splicing factor.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Alignment of SR45 amino acid sequences from Arabidopsis (AtSR45, At1g16610), rice (RiceSR45_1, Accession AK070420; RiceSR45_2, Accession AK063761) and maize (MaizeSR45, Accession BT016650). The RRM domain is underscored and the RS domains at the N-and C-terminus are indicated by dashed underlines. Identical amino acids are shown by reverse lettering. Dashes indicate gaps in alignment. (B) In vitro splicing of β-globin pre-mRNA in S100 cell extract supplemented with either 25 ng of ASF/SF2 (ASF) or increasing amounts (10, 30 and 90 ng) of purified SR45 (SR45) expressed in insect cells. The positions of pre-mRNA, spliced mRNA and 5′ exon are indicated to the right of the blot. Boxes and a line indicate exons and an intron, respectively.</p>
<span>doi:10.1371/journal.pone.0000471.g001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>SR45 Is Present in Flowering Plants but Not in Algae and Animals</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To identify Arabidopsis SR45 homologs in other organism we searched all the available plant, fungal and animal genomes and EST sequence databases (see <a href="#s4">Materials and Methods</a> ). These searches revealed the presence of SR45 in rice, maize and in many other flowering plants, but not in algae (<em>Chlamydomonas reinhardtii</em>; <em>Cyanidioschyzon merolae</em>), a diatom (<em>Thalassiosira pseudonana</em>) and animals. These results indicate that SR45 has appeared later in evolution only in the flowering plants clade. Interestingly, there are two <em>SR45</em> genes in rice (accessions AK070420 and AK063761) as opposed to one in Arabidopsis, indicating that <em>SR45</em> may have undergone duplication after the divergence of monocots and dicots. A comparison of the amino acid sequences of Arabidopsis SR45 with homologs from rice and maize (accession BT016650) showed similar domain organization and considerable sequence conservation (51–80% similarity) (<a href="#pone-0000471-g001">Figure 1A</a>). The absence of SR45 in animals and algae and its presence in flowering plants suggest that SR45 may have evolved to perform functions that are specific to flowering plants.</p>


<h4>SR45 Protein Expressed in Insect Cells Complements Splicing-Deficient S100 Cell Extract</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>An important property of SR proteins is that each one is sufficient to complement splicing-deficient HeLa cell S100 extract in splicing of at least some pre-mRNA substrates. Because its domain structure is different from typical SR proteins, SR45 is not classified as an SR protein in a recent report <a href="#pone.0000471-Wang1">[21]</a>. Furthermore, animal proteins with similar domain organization such as human Tra2a and Trab cannot substitute for the essential splicing function of SR proteins <a href="#pone.0000471-Tacke1">[22]</a>. To determine if SR45 functions as an essential splicing factor, we expressed SR45 as a His.tag fusion in insects cells and used the purified protein to analyze its splicing activity in S100 extract with a β-globin pre-mRNA substrate. Remarkably, the purified SR45 was able to activate splicing of β-globin pre-mRNA in a concentration-dependent manner and at a level comparable to recombinant human ASF/SF2 (<a href="#pone-0000471-g001">Figure 1B</a>). These results provide evidence that SR45, despite its unique domain organization, is a bonafide SR protein and functions as an essential splicing factor.</p>


<h4>Molecular Characterization of the <em>sr45-1</em> Mutant</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To analyze the in vivo function of SR45, we characterized a T-DNA insertion mutant of <em>SR45</em> (<a href="#pone-0000471-g002">Figure 2A</a>). Genomic PCR analyses with <em>SR45</em>-specific primers (LP+RP) that flanked the T-DNA insertion site amplified the expected size <em>SR45</em> product from wild-type (WT) but not from the mutant indicating that <em>SR45</em> is disrupted (<a href="#pone-0000471-g002">Figure 2B</a>). In contrast PCR with the T-DNA-specific primer (LBb1) and <em>SR45</em>-specific primer (LP) amplified a product from the mutant but not from WT (<a href="#pone-0000471-g002">Figure 2B</a>), confirming that the insertion is in <em>SR45</em>. Insertion of T-DNA at a single locus was verified by Southern analysis (<a href="#pone-0000471-g002">Figure 2C</a>). RT-PCR with <em>SR45</em>-specific forward and reverse primers (1F+414R) amplified the expected size product from WT but not from the mutant (<a href="#pone-0000471-g002">Figure 2D</a>). To test if a truncated transcript upstream of the insertion is produced, PCR was performed with forward primer, 1F and a primer that is complementary to SR45 before the T-DNA (172R). The expected size product was amplified indicating that a truncated SR45 transcript is produced in the mutant (<a href="#pone-0000471-g002">Figure 2D</a>). However, the level of the truncated transcript is only 8% of the full-length transcript in WT, indicating that it may be unstable. This mutant was named <em>sr45-1</em> and was used in the experiments reported below.</p>
<div class="figure" id="pone-0000471-g002"><div class="img"><a name="pone-0000471-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Molecular characterization of <em>sr45-1</em>plants.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(A) Gene structure of <em>SR45</em> (At1g16610). Filled rectangles are exons; thin lines are introns; inverted open triangle indicates the position of the T-DNA insertion in the 7<sup>th</sup> exon. The location of various primers used in PCR is indicated by half-head arrows. The schematic diagram below the gene structure shows domain organization of SR45 protein. The corresponding gene positions coding for the N-terminal arginine/serine rich (RS1) domain, the middle RNA recognition motif (RRM) and the C-terminal RS2 domain are indicated by downward arrows. (B) Verification of the <em>sr45-1</em> insertion in the genomic DNA by PCR with <em>SR45</em>-specific primers (LP and RP) and the T-DNA-specific primer LBb1; the locations of these primers relative to the T-DNA insertion are shown in (A). (C) Southern blot showing single insertion of T-DNA. Genomic DNA from WT and <em>sr45-1</em> was digested with either SacI or EcoRI and probed with a <sup>32</sup>P-labeled T-DNA probe. (D) RT-PCR expression analyses of <em>SR45</em> in WT and <em>sr45-1</em> plants using <em>SR45</em>-specific primers (1F and 414R, which will amplify full-length <em>SR45</em>; 1F and 172R will amplify a truncated <em>SR45</em> transcript before the T-DNA insertion). Cyc, cyclophilin product was amplified to show equal amount of cDNA template in PCR. PCR product in WT and <em>sr45-1</em> was normalized to cyclophilin expression in WT and <em>sr45-1</em>, respectively. Numbers below each panel indicate the level of transcript in WT and <em>sr45-1</em> plants. The level of <em>SR45</em> transcript in WT is considered as 1.</p>
<span>doi:10.1371/journal.pone.0000471.g002</span></div>

<h4>The Alternative Splicing Pattern of Several Related SR Genes is Altered in the <em>sr45-1</em> mutant</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To gain insight into the molecular mechanism of SR45, we analyzed the expression pattern of the alternative transcripts of all 19 SR genes in different organs in <em>sr45-1</em> and WT. <a href="#pone-0000471-g003">Figure 3</a> shows that the AS patterns of <em>SR30, RS31, RS31a, SR34</em> and <em>SR34b</em> were different between <em>sr45-1</em> and WT. Sequencing of the alternative transcripts showed that, except for <em>SR34b</em>, the smallest transcript codes for a functional protein, whereas the rest code for truncated proteins (<a href="#pone-0000471-g003">Figure 3</a>). In the vegetative tissues of <em>sr45-1</em>, the amounts of <em>SR30</em> mRNA isoform 1 and 3 diminished to negligible levels, whereas the level of isoform 5 increased by approximately 2 fold (<a href="#pone-0000471-g003">Figure 3</a>, SR30). In inflorescence, isoform 1 and 3 showed reduced abundance relative to isoform 5. Similarly, in <em>sr45-1</em>, the smaller size transcripts of <em>RS31</em> (isoform 1 and 2) and <em>RS31a</em> (isoform 1, 3 and 4) were also diminished. In some cases, this reduction was accompanied by a moderate increase in the longer size products in some genes (e.g. <em>SR30</em>). In the case of <em>SR34</em>, there was an overall decrease in all transcripts in <em>sr45-1</em>. The abundance of the two smaller transcripts (isoform 1 and 3) in all tissues decreased. In stems and leaves, but not in roots and inflorescence, this reduction was accompanied by an increase in isoform 5. The abundance of the longest isoforms significantly diminished but this may have resulted due to an overall decrease in transcript level in <em>sr45-1</em> relative to WT. In all tissues, the abundance of the intermediate size transcript (isoform 6) of <em>SR34b</em>, which codes for full-length protein, remained similar between <em>sr45-1</em> and WT. In vegetative tissues, the abundance of the longer transcripts (isoform 7 and 8) decreased and the smaller size transcripts (isofrom 1 and 2) increased in <em>sr45-1</em>. In inflorescence of <em>sr45-1</em>, however, isoform 7 and 8 decreased but there was no significant change in the levels of isoform 1 and 2. The pattern of alternative transcripts of the rest of the SR genes, except for very subtle changes in a few transcripts, remained similar between <em>sr45-1</em> and WT (<em>RSZ32</em> and data not shown). Sequence analyses of the affected transcripts show that the longest intron is either partially retained or excluded in the alternative transcripts. Since mutation in <em>SR45</em> caused a decrease in the smaller size products and an increase in the longer size product, SR45 likely functions by influencing splicing of the longest intron to maintain a balance in the relative abundance of alternative transcripts. Overall these analyses indicate that SR45 affects the AS of a subset of transcripts of SR protein-encoding genes, which in turn may regulate the splicing of other transcripts and eventually control the different phenotypes observed in <em>sr45-1</em>plants.</p>
<div class="figure" id="pone-0000471-g003"><div class="img"><a name="pone-0000471-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Expression and alternative splicing of pre-mRNAs of Arabidopsis <em>SR</em> genes in different organs is altered in the <em>sr45-1</em> plants.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Expression levels were analyzed by RT-PCR with primers specific to each <em>SR</em> gene. Sequences of forward and reverse primers used are shown in <a href="#pone.0000471.s005">Table S3</a>. An equal amount of template in each reaction was verified by amplifying a constitutively expressed cyclophilin. The name of the <em>SR</em> gene is shown on the left of each panel. DNA sizes are indicated on the right. Isoform number is indicated on the left side of the gel. R, root; S, stem; L, leaf and I, inflorescence. Schematic diagrams in the bottom panel for each gene show the gene structure and its alternatively spliced mRNA isoforms (Numbers below each isoform indicate the number of nucleotides). Predicted proteins from splice variants are shown to the right of each isoform. Exons are filled rectangles and introns are thin lines. Black rectangles represent constitutively spliced exons whereas the red rectangles indicate the included regions in splice variants. Vertical arrowhead and ‘*’ show start and stop codons, respectively; Horizontal green and red arrowheads above and below gene structures indicate the position of forward and reverse primers, respectively. In the schematics of predicted proteins, numbers to the right are the number of amino acids in the protein. RRM, RNA recognition motif, RS, Arginine/Serine-rich domain. Blue rectangle indicates a stretch of amino acids that are not present in functional SR proteins.</p>
<span>doi:10.1371/journal.pone.0000471.g003</span></div>

<h4><em>sr45-1</em> Plants Show Delayed Flowering and Altered Leaf Morphology</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>For phenotypic characterization, <em>sr45-1</em> and WT plants were grown under identical conditions. At the seedling stage, the leaves of the <em>sr45-1</em> plants were elongated and curly (<a href="#pone-0000471-g004">Figure 4A and B</a>). In general <em>sr45-1</em> plants were smaller (about one-third) than WT throughout their life cycle (<a href="#pone-0000471-g004">Figure 4A–D</a>). The mutant showed delay in transition to reproductive phase (<a href="#pone-0000471-g004">Figure 4C and D</a>). Root growth was also slower in the mutant as compared to the WT (<a href="#pone-0000471-g004">Figure 4E and F</a>). Under most conditions <em>sr45-1</em> plants were bushy (<a href="#pone-0000471-g004">Figure 4D</a>). The most pronounced phenotypes, flowering time and leaf morphology were chosen for further detailed characterization.</p>
<div class="figure" id="pone-0000471-g004"><div class="img"><a name="pone-0000471-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Phenotypic characterization of <em>sr45-1</em>plants.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A–D) Phenotypes of <em>sr45-1</em> at different growth stages grown under long-days (16-h photoperiod). WT plants are shown on the left side and <em>sr45-1</em> plants are on the right side of each panel. (A) Eight day-old seedlings on MS plates; Note that <em>sr45-1</em> has narrow true leaves. (B) Twenty-day-old plants in the soil; several leaves in the <em>sr45-1</em> plant are curled downward and inward, indicated by arrow-heads. Inset graph in (B) shows the size of <em>sr45-1</em> compared to WT. (C) Thirty-five-day old plants in soil. (D) Fifty-four-day-old plants in soil; note that WT has completed flowering and most of the siliques have turned brown, whereas, the <em>sr45-1</em> plant is still flowering. (E) Root growth in <em>sr45-1</em> and WT plants. A representative photograph of <em>sr45-1</em> and WT seedlings (27 days after germination) illustrates reduced root growth. (F) Quantification of root growth. Each data point is the mean±SEM of 5 plants.</p>
<span>doi:10.1371/journal.pone.0000471.g004</span></div>

<h4><em>sr45-1</em> Plants Flowered Later than WT in Both Long-day and Short-day Photoperiods</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Genetic analysis of flowering time in Arabidopsis has uncovered four major flowering pathways–the photoperiodic, the vernalization, the autonomous and the hormonal pathways <a href="#pone.0000471-Sung1">[23]</a>, <a href="#pone.0000471-Boss1">[24]</a>. To determine which of these pathways is affected in <em>sr45-1</em>, we quantified flowering time under short-day (SD, 8 h:16 h light:dark) and long-day (LD, 16 h:8 h light:dark) conditions. Flowering time was measured as days to bolting and number of rosette leaves at the first appearance of a flower. Under LD a majority of WT plants flowered around 29 days after germination (<a href="#pone-0000471-g005">Figure 5A</a>). In contrast, <em>sr45-1</em> plants flowered around 40 days, reflecting, on average, 11 days of delay in flowering compared to the WT (<a href="#pone-0000471-g005">Figure 5A</a>). Similarly, at the appearance of the first flower, <em>sr45-1</em> had 17 leaves whereas WT had only 12 leaves. Under SD for <em>sr45-1</em>, the average number of days to bolting was 102, which was about twice the time required for WT to start bolting (<a href="#pone-0000471-g005">Figure 5A</a>). Likewise, the number of leaves at flowering was 97 in <em>sr45-1</em> as compared to 36 in WT. Flowering time under a 12-h photoperiod was very similar to LD (<a href="#pone-0000471-g005">Figure 5A</a>). Together, these data indicate that <em>sr45-1</em> exhibits delayed flowering under both LD and SD, although the delay is more prominent under SD.</p>
<div class="figure" id="pone-0000471-g005"><div class="img"><a name="pone-0000471-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>sr45-1 plants are late flowering.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Top panel: <em>sr45-1</em> plants are considerably later flowering than WT under long-day (LD, 16 h:8 h light:dark), short-day (SD, 8 h:16 h light:dark) and 12 h:12 h light:dark conditions. Age of the plants at the time they were photographed is indicated on each panel. Bottom Panel: Quantification of flowering time. Transition to reproductive stage was measured as days to bolting and number of rosette leaves at the appearance of 1<sup>st</sup> flower. Each bar is the mean±SEM of 48 to 72 plants. Significant differences (<em>p</em>&lt;0.05) between WT and <em>sr45-1</em> plants are indicated by ‘*’. Each experiment was repeated three times. (B) Effect of vernalization on flowering of <em>sr45-1</em> plants. <em>sr45-1</em> and WT seeds were stratified at 4°C for 2 days or vernalized for 40 days at 4°C and grown under LD (16 h photoperiod) or SD (8 h photoperiod) conditions in soil. Flowering time was measured as described in experimental procedures. V+and V-indicate vernalized and unvernalized plants, respectively. Significant differences (<em>p</em>&lt;0.05) between vernalized and unvernalized plants are indicated by ‘*’. Each experiment was repeated three times. (C) Expression analyses of flowering related genes in <em>sr45-1</em> and WT plants. Expression of representative genes in various flowering pathways was analyzed by RT-PCR. <em>VRN2</em> belongs to the vernalization pathway, <em>FCA</em> to the autonomous pathway; <em>FLC</em> integrates signals from these two pathways. <em>CO</em> belongs to the photoperiod pathway. <em>SOC1</em> and <em>FT</em> function downstream of <em>FLC</em>. Each RT-PCR was repeated at least three times. (D) Expression levels of various genes in the flowering time pathways of Arabidopsis. Model is adapted from <a href="#pone.0000471-Boss1">[24]</a>, <a href="#pone.0000471-He2">[70]</a>. Value in parentheses next to a gene indicates induction or repression of that gene in <em>sr45-1</em>. NC, No change.</p>
<span>doi:10.1371/journal.pone.0000471.g005</span></div>

<h4>The Late Flowering Phenotype of <em>sr45-1</em> Was Rescued by Vernalization but Was Insensitive to Photoperiod, Placing it in The Autonomous Pathway of Flowering</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To determine the effect of vernalization on the flowering of <em>sr45-1</em>, we vernalized <em>sr45-1</em> and WT seeds for 40 days and grew them under LD and SD conditions. Under LD, in WT vernalization marginally decreased days to bolting by one day and number of leaves at flowering from 12 to 10. On the other hand, vernalization of <em>sr45-1</em> reduced the number of days to bolting and leaves at flowering close to WT (<a href="#pone-0000471-g005">Figure 5B</a>). Similarly under SD, vernalization also reverted flowering time closer to WT. However, unlike under LD, vernalization was not fully effective. Vernalized <em>sr45-1</em> plants flowered 31 days earlier than unvernalized plants, however, they were still 34 days later in flowering than vernalized WT plants, indicating a slight interaction of photoperiod with vernalization on flowering. In summary, the SR45-dependent late flowering phenotype was rescued by vernalization, demonstrating that SR45 affects the autonomous pathway.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>To gain insight into the molecular effects of SR45 on flowering, we performed microarray experiments consisting of three biological replicates of <em>sr45-1</em> and WT using the Affymetrix Arabidopsis 22K gene chip and analyzed the data for flowering time genes (<a href="#pone.0000471.s003">Table S1</a> and <a href="#pone.0000471.s001">Figure S1</a>). One of the prominent genes that displayed increased expression in <em>sr45-1</em> was the <em>FLOWERING LOCUS C</em> (<em>FLC</em>) (<a href="#pone-0000471-g005">Figure 5C</a>, <a href="#pone.0000471.s001">Figure S1B</a> and <a href="#pone.0000471.s003">Table S1</a>), which is a MADS-box transcription factor and was originally identified as a mutation that was early flowering under LD <a href="#pone.0000471-Sheldon1">[25]</a>, <a href="#pone.0000471-Michaels1">[26]</a>. The <em>FLC</em> gene is a convergence point for several flowering pathways <a href="#pone.0000471-Sung1">[23]</a>, <a href="#pone.0000471-Boss1">[24]</a>. Close to a dozen genetic loci that affect the expression of <em>FLC</em> have been characterized phenotypically and at the molecular level <a href="#pone.0000471-Sung1">[23]</a>, <a href="#pone.0000471-Boss1">[24]</a>. Seven of these genes, <em>FCA</em> <a href="#pone.0000471-Macknight1">[27]</a>, <em>FY</em> <a href="#pone.0000471-Simpson1">[28]</a>, <em>FVE</em> <a href="#pone.0000471-Ausin1">[29]</a>, <em>FLD</em> <a href="#pone.0000471-He1">[30]</a>, <em>LD</em> <a href="#pone.0000471-Lee1">[31]</a>, <em>FLK</em> <a href="#pone.0000471-Mockler1">[32]</a>, <a href="#pone.0000471-Lim1">[33]</a> and <em>FPA</em> <a href="#pone.0000471-Schomburg1">[34]</a> belong to the autonomous pathway and three (<em>VRN1, VRN2</em>, and <em>VIN3)</em> belong to the vernalization pathway (<a href="#pone-0000471-g005">Figure 5D</a>). Expression analyses of these genes in the microarray expriment combined with verification by RT-PCR showed that these genes were not differentially expressed in <em>sr45-1</em> as compared to WT (<a href="#pone-0000471-g005">Figure 5D</a> and <a href="#pone.0000471.s001">Figure S1</a> and <a href="#pone.0000471.s003">Table S1</a>). <em>VRN2</em> <a href="#pone.0000471-Gendall1">[35]</a> and <em>FCA</em>, which is known to regulate flowering by producing multiple transcripts <a href="#pone.0000471-Macknight1">[27]</a>, did not reveal any distinguishable alteration in their expression level or AS pattern between <em>sr45-1</em> and WT (<a href="#pone-0000471-g005">Figure 5C</a> and <a href="#pone.0000471.s001">Figure S1</a>). These analyses, however, do not rule out the possibility that subtle changes in the AS, limited to only a few nucleotides and indistinguishable by RT-PCR, are affected by SR45. Similarly, the AS pattern of <em>FLC</em> gene was also unchanged in <em>sr45-1</em>. FLC represses the downstream target genes <em>SOC1</em> <a href="#pone.0000471-Lee2">[36]</a> and <em>FT</em> <a href="#pone.0000471-Kardailsky1">[37]</a>. RT-PCR analyses showed no differential expression of <em>FT</em>, whereas there was some repression of <em>SOC1</em> in <em>sr45-1</em> compared to WT (<a href="#pone-0000471-g005">Figure 5C and D</a>), suggesting that the elevated level of <em>FLC</em> in <em>sr45-1</em> is causing late-flowering phenotype by affecting the activity of some other downstream components. Microarray and RT-PCR analyses with the <em>CONSTANS (CO)</em> gene (photoperiod pathway) <a href="#pone.0000471-Putterill1">[38]</a>, showed no alteration of expression in <em>sr45-1</em> indicating that it does not affect the photoperiod flowering pathway. Similarly, the expression of several genes that positively regulate <em>FLC</em>, such as the members of the PAF1-complex, were also not changed (<a href="#pone.0000471.s001">Figure S1</a> and <a href="#pone.0000471.s003">Table S1</a>). Overall, these analyses indicate that SR45 affects flowering in an <em>FLC</em>-dependent manner.</p>


<h4>Characterization of Leaf Morphology</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>To quantify the change in leaf shape, we analyzed the length to width ratio of leaves in WT and <em>sr45-1</em>. Significant differences in the leaf length/width ratio were observed between WT and <em>sr45-1</em>, which were more obvious in juvenile (leaf 1 to 4) than in mature leaves (leaf 5 to 7) (<a href="#pone-0000471-g006">Figure 6A and B</a>). On average <em>sr45-1</em> leaves were about two-fold more elongated than in WT. Since floral organs are modified leaves, we also compared their morphology in fully opened flowers of <em>sr45-1</em> and WT. Like vegetative leaves, petals were also elongated in <em>sr45-1</em> (<a href="#pone-0000471-g006">Figure 6C</a>). Together, these data indicate that SR45 is necessary for the normal expansion of organs.</p>
<div class="figure" id="pone-0000471-g006"><div class="img"><a name="pone-0000471-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span><em>sr45-1</em> plants have elongated leaves and petals.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>(A) WT and <em>sr45-1</em> plants grown on MS plates. Note that <em>sr45-1</em> has elongated leaves. Cotyledonary leaves (C1 and C2) and true leaves are numbered from left (older) to right (younger). (B) Leaf length/width ratios (Mean±SEM) of WT and <em>sr45-1</em> plants. (<em>n</em> = 11 leaves from 11 plants, Student's <em>t</em>-test <em>p</em>-values are as follows: Cotyledon (C1), <em>p</em> = 0.52; Cotyledon (C2), <em>p</em> = 0.39; leaf 1, <em>p</em> = 0.002; leaf 2, <em>p</em> = 0.021; leaf 3, <em>p</em> = 0.022; leaf 4, <em>p</em> = 0.0003; leaf 5, <em>p</em> = 0.33; leaf 6, <em>p</em> = 0.49; leaf 7, <em>p</em> = 0.29). The symbol ‘*’ on the top of bars indicate that the difference between WT and <em>sr45-1</em> is significant (<em>p</em>&lt;0.05). (C) Fully opened flowers of <em>sr45-1</em> plants have elongated petals. The bottom panel shows quantification of length, width and length/width ratio (Mean±SEM) of petals in WT (WT) and mutant (<em>sr45-1</em>) plants (<em>n</em> = 16, Student's <em>t</em>-test <em>p</em>-values: length, <em>p</em> = 10<sup>−9</sup>, width, <em>p</em> = 10<sup>−8</sup><em>;</em>length/width <em>p</em> = 10<sup>−10</sup>). The symbol ‘*’ on the top of bars indicate that the difference between WT and <em>sr45-1</em> is significant (<em>p</em>&lt;0.05). (D) The leaves of <em>sr45-1</em> plants exhibit abnormal expansion of pavement cells. Scanning electron micrographs of the abaxial and adaxial surfaces of WT and <em>sr45-1</em> plants. Note the presence of abnormal elongated abaxial pavement cells in <em>sr45-1</em>. For clarity, the boundaries of pavement cells on the adaxial surfaces are outlined. Note the presence of markedly enlarged and elongated pavement cells in <em>sr45-1</em> leaves. Bars = 50 µm. (E) Quantitative analyses of the number of pavement cells and stomata in WT and <em>sr45-1</em>. Each bar is the average±SEM of cells counted in at least three 160 mm<sup>2</sup> areas in three different leaves. The <em>p</em>-values of the <em>t</em>-test statistics of pavement cell and stomata were 0.55 (non-significant) and 0.03 (significant, indicated by ‘*’), respectively. (F) Box plots of the distribution of cell sizes in WT and <em>sr45-1</em> pavement cells. Each box plot was made from the cell sizes of WT (<em>n</em> = 112) and <em>sr45-1</em> (<em>n</em> = 128) from at least three different leaves. Boxes indicate the interquartile range between the 1st and 3rd quartile, whereas line in the middle of the box represents median. Ends of the vertical lines indicate the range of the data.</p>
<span>doi:10.1371/journal.pone.0000471.g006</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>To better characterize the leaf morphology, we compared epidermal cell-shape in the leaves of <em>sr45-1</em> to WT with scanning electron microscopy (SEM). Compared to WT, the abaxial surfaces of <em>sr45-1</em> leaves had some pavement cells that were abnormal in shape (<a href="#pone-0000471-g006">Figure 6D</a>). These cells were significantly more elongated along the length of the leaf and did not display clear indentations and lobes as in WT cells. The presence of these cells probably interferes with normal expansion of leaf cells resulting in the curly phenotype of <em>sr45-1</em> leaves. Comparison of the adaxial surfaces of <em>sr45-1</em> to WT revealed two distinct phenotypes. First, several pavement cells of <em>sr45-1</em> displayed abnormal expansion along all axes. In these cells, the indentations and lobes, although detectable, were less prominent as compared to WT making them look more elongated and arranged length-wise along the length of leaf (<a href="#pone-0000471-g006">Figure 6D</a>). Second, contrary to regularly spaced stomata in WT leaves, in <em>sr45-1</em> they were clustered together. To quantify the contribution of cell size and cell number to the elongated phenotype of leaves, we compared the number of pavement cells and stomata in <em>sr45-1</em> to WT. Stomata were approximately 30% more in <em>sr45-1</em> than in WT (<a href="#pone-0000471-g006">Figure 6E</a>). The number of pavement cells per unit area in <em>sr45-1</em> was slightly more than in WT (65/mm<sup>2</sup> in <em>sr45-1</em> to 61/mm<sup>2</sup> in WT), however this difference was statistically not significant (<em>p</em> = 0.55) (<a href="#pone-0000471-g006">Figure 6E</a>). The fact that there are more stomata in the mutant suggests that cell division may have been affected early in leaf development. Cell size and the orientation of expansion on the contrary were dramatically changed in <em>sr45-1</em>. Although, the means of pavement cells size of <em>sr45-1</em> and WT were not significantly different (<em>p</em> = 0.3), visual examination revealed obvious differences in the distribution of their cell sizes. Quantification of the cell size data is displayed as box plots in <a href="#pone-0000471-g006">Figure 6F</a>, which show that the cell size distribution in <em>sr45-1</em> is more varied than in WT (size range was 6347 mm<sup>2 </sup>in <em>sr45-1</em> compared to 4199 mm<sup>2</sup> in WT). Additionally, the scatter of cell sizes in the 4<sup>th</sup> quartile was about two-fold greater for <em>sr45-1</em> than for WT. This shows that a substantial proportion of <em>sr45-1</em> cells were bigger than the biggest cells in WT. Together, these data indicate that the elongated leaf shape phenotype is probably caused by abnormal cell expansion and by subtle changes in cell division.</p>


<h4>Flowers of the <em>sr45-1</em> Plants Have Abnormal Petal and Stamen Numbers</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>The mutant plants grown under short day conditions had higher occurrence of flowers with altered number of petals and stamens (<a href="#pone-0000471-g007">Figure 7</a>). This phenotype was more prominent in the early flowers and was rarely observed in later flowers. This phenotype is different from the well-studied phenotype of mutants of floral homeotic genes, where whole whorls of floral organs change from one type to another. Instead, <em>sr45-1</em> flowers have all four whorls but the number of petals and stamens varied from 3 to 8 in several combinations (<a href="#pone-0000471-g007">Figure 7B–I</a>). The number of sepals and carpels remained unchanged. In the same inflorescence several flowers had the WT organ numbers (<a href="#pone-0000471-g007">Figure 7B and C</a>). However, most of the inflorescences had at least one flower with altered petal and stamen numbers. Overall, of the 67 flowers counted, 11 had the abnormal petal and stamen number, which corresponds to a penetrance of approximately 15%. SR45 thus partially affects the petal and stamen number in a development-specific manner.</p>
<div class="figure" id="pone-0000471-g007"><div class="img"><a name="pone-0000471-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span><em>sr45-1</em> Flowers have Altered Number of Sepals and Petals.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>(A) WT inflorescence shows that all flowers had four petals. (B and C) Inflorescences from two separate <em>sr45-1</em> plants are shown. Each panel shows 3 of 6 flowers with abnormal petal and stamen numbers, indicated by arrowhead. (D) Close-up of a WT flower with 4 petals 6 stamens. (E–I) <em>sr45-1</em> flowers with abnormal petal and stamen numbers. Number of sepals and carpels were not changed. (E) 4 petals 5 stamens (F) 5 petals 5 stamens (G) 6 petals 3 stamens (H) 6 petals 4 stamens (I) 8 petals 3 stamens.</p>
<span>doi:10.1371/journal.pone.0000471.g007</span></div>

<h4>Full-length <em>SR45</em> Rescued <em>sr45-1</em></h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>To demonstrate that the lack of SR45 was responsible for the <em>sr45-1</em> phenotype, we complemented the mutant with a <em>GFP</em>-tagged <em>SR45</em> (<em>GFP-SR45</em>). The <em>GFP-SR45</em> construct was described earlier and was shown to be functional <a href="#pone.0000471-Ali2">[20]</a>. These plants exhibited a characteristic speckled pattern of GFP-SR45 in the nuclei indicating that it was expressed and localized correctly (<a href="#pone-0000471-g008">Figure 8A</a>, inset). The ectopic expression of <em>GFP-SR45</em> in transgenic <em>sr45-1</em> plants reverted their narrow-leaf, narrow-petal and flowering phenotype back to the WT (<a href="#pone-0000471-g008">Figure 8A, B and C</a>). Consistent with the flowering time of the rescued <em>GFP-SR45/sr45-1</em> plants, the <em>FLC</em> levels were also reverted back to WT (<a href="#pone-0000471-g008">Figure 8D</a>, <em>FLC</em>), further confirming that the mutant phenotypes were due to <em>SR45</em> mutation. Similarly, RT-PCR analysis of several independent transformants showed the expected size <em>SR45</em> product (<a href="#pone-0000471-g008">Figure 8D</a>, <em>SR45</em>). In addition, the splicing pattern of <em>SR30, SR34, SR34b, RS31</em> and <em>RS31a</em> was also rescued in three independent <em>GFP-SR45/sr45-1</em> transgenic lines (<a href="#pone-0000471-g008">Figure 8D</a>).</p>
<div class="figure" id="pone-0000471-g008"><div class="img"><a name="pone-0000471-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000471" data-uri="info:doi/10.1371/journal.pone.0000471.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000471.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000471.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000471.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000471.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Full-length <em>GFP</em>-tagged <em>SR45</em> rescued the mutant phenotypes of <em>sr45-1</em> plants.</span></strong></p><a id="article1.body1.sec2.sec10.fig1.caption1.p1" name="article1.body1.sec2.sec10.fig1.caption1.p1"></a><p>(A) Growth of WT, <em>sr45-1</em> and <em>GFP-SR45</em>/<em>sr45-1</em> plants. The narrow-leaf and stunted growth phenotype of <em>sr45-1</em> was rescued by expression of <em>GFP-SR45</em> in <em>sr45-1</em> plants. Inset in the lower right corner shows a root epidermal nucleus of a <em>GFP-SR45</em>/<em>sr45-1</em> displaying the characteristic speckled distribution of GFP-SR45. (B) Petal shape phenotype of WT, <em>sr45-1</em> and <em>GFP-SR45</em>/<em>sr45-1</em>. The <em>p</em>-values of <em>t</em>-test statistics are as follows: WT vs <em>sr45-1</em>, <em>p</em> = 0.000012 (significant); WT vs <em>GFPSR45/sr45-1</em>, <em>p</em> = 0.32 (non-significant). Significant difference is indicated by ‘*’. (C) Quantification of flowering time. Transition to reproductive stage was measured as days to bolting and number of rosette leaves at the appearance of 1<sup>st</sup> flower. Each bar is the mean±SEM of 10 to 22 plants. Bars with same letter and the same color indicate non-significant difference (<em>p</em>&gt;0.05). (D) Expression pattern of alternatively spliced transcripts of <em>SR</em> genes in WT, <em>sr45-1</em> and <em>GFP-SR45</em>/<em>sr45-1</em> plants. Expression pattern of three independent <em>GFP-SR45</em>/<em>sr45-1</em> lines is shown. In all transgenic lines the splicing patterns of pre-mRNAs encoding SR proteins is restored to wild-type. Similar results were obtained in two independent RT-PCR analyses. For details on RT-PCR and primer sets used see <a href="#pone-0000471-g003">Figure 3</a>. Cyclophilin amplification was used to demonstrate equal amount of template in each PCR.</p>
<span>doi:10.1371/journal.pone.0000471.g008</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>SR45 Is a Plant-Specific Splicing Factor</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>SR45 was identified as a U1-70k interacting protein in a yeast two-hybrid screen <a href="#pone.0000471-Golovkin3">[17]</a> and verified to co-localize with U1-70K and another splicing factor, SR1, in the nucleus as speckles <a href="#pone.0000471-Ali2">[20]</a>, a hallmark feature of all SR proteins both in plants and metazoans <a href="#pone.0000471-Reddy3">[10]</a>, <a href="#pone.0000471-Lamond1">[39]</a>. These observations, together with its unique domain organization (an N-terminal and a C-terminal RS domain separated by an RRM domain), suggested that SR45 plays a role in splicing (<a href="#pone-0000471-g002">Figure 2</a>). To be classified as a bonafide splicing factor, a protein is usually tested for its ability to complement splicing in a splicing-deficient extract. We showed that SR45 was able to splice a pre-mRNA substrate in a splicing-deficient extract, suggesting that it is a splicing factor (<a href="#pone-0000471-g001">Figure 1B</a>). This finding is further supported by the fact that a mutation in <em>SR45</em> led to changes in the splicing pattern of several other SR genes (discussed below).</p>


<h4>SR45 Is Involved in Plant-Specific Developmental Processes</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Homologs of SR45 are found only in higher plants, which suggests, that SR45 performs a plant-specific function. This notion is supported by the functional characterization of SR45 at the phenotypic and molecular levels. The most prominent plant-specific function affected by SR45 was flowering. The <em>sr45-1</em> plants were late flowering in both SD and LD conditions, essentially placing it in the autonomous pathway. This was validated by physiological, molecular and genetic analyses. First, the late-flowering phenotype was overcome by vernalization under LD (<a href="#pone-0000471-g005">Figure 5</a>) consistent with the late-flowering characteristics of other autonomous pathway mutants such as <em>fve, fld, fca, fy, fpa, ld</em> and <em>flk,</em> all of which are late flowering under both SD and LD and are rescued by vernalization <a href="#pone.0000471-Simpson1">[28]</a>–<a href="#pone.0000471-Schomburg1">[34]</a>. Second, <em>sr45-1</em> had an elevated level of <em>FLC</em> (<a href="#pone-0000471-g005">Figure 5C</a>, <a href="#pone.0000471.s001">Figure S1B</a>), whose expression is elevated in all autonomous pathway mutants. Third, expression of <em>CO</em>, which is a photoperiod pathway gene, was not changed in the <em>sr45-1</em> mutant, indicating that SR45 does not operate in the photoperiod pathway.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>RT-PCR analyses showed that a truncated transcript of <em>SR45</em> is produced in <em>sr45-1</em> plants, which may code for a truncated version of SR45 that would lack the last one-third of the protein. This may result in a non-functional or a protein with abnormal function ultimately resulting in impaired activity in splicing. However, the level of the truncated transcript in the mutant is very low (about 8% of WT level, <a href="#pone-0000471-g002">Fig. 2D</a>). In support of this conclusion, the heterozygous plants were similar to WT (data not shown). Hence, the observed phenotypes in <em>sr45-1</em> plants are not likely due to a hypomorphic allele or a dominant negative allele of SR45.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>Arabidopsis leaf development requires the establishment of proximodistal, adaxio-abaxio and mediolateral asymmetry, which is a complex process controlled by several genes that are involved in polar cell expansion and differentiation <a href="#pone.0000471-Tsukaya1">[40]</a>, <a href="#pone.0000471-Byrne1">[41]</a>. The leaf morphology of <em>sr45-1</em> bears resemblance to those mutants that are defective in the leaf length-width expansion. Analysis of our microarray and RT-PCR data for several known genes involved in leaf morphology such as <em>ANGUSTIFOLIA</em> (<em>AN</em>), <em>ROT3</em> <a href="#pone.0000471-Tsukaya1">[40]</a>, <em>AN3</em> <a href="#pone.0000471-Horiguchi1">[42]</a>, <em>CURLY LEAF</em> (<em>CLF</em>) <a href="#pone.0000471-Kim1">[43]</a>, <em>DRL</em> <a href="#pone.0000471-Nelissen1">[44]</a>, <em>AS1</em> <a href="#pone.0000471-Theodoris1">[45]</a>, <em>AS2</em> <a href="#pone.0000471-ChalfunJunior1">[46]</a> and <em>ATHB13</em> <a href="#pone.0000471-Hanson1">[47]</a>, however, did not reveal a significant change in their expression level between the mutant and WT (<a href="#pone.0000471.s002">Figure S2</a> and <a href="#pone.0000471.s004">Table S2</a>). This indicates that these genes are not affected by SR45. Comparison of cell shapes and number in <em>sr45-1</em> to WT showed that SR45 likely affects both cell expansion and division. Several genes involved in diverse cellular processes affect cell expansion. For example, wall-associated kinase2 (<em>WAK2</em>) antisense plants had smaller cell sizes than control plants <a href="#pone.0000471-Wagner1">[48]</a>. Interestingly, in <em>sr45-1</em> leaves, which had more expanded cells, the expression of this gene (259560_at) together with several other WAK-like genes such as, <em>WAK1</em> (259561_at) and <em>WAK3</em> (259559_at) were moderately (2 to 3-fold over WT) induced (<a href="#pone.0000471.s004">Table S2</a>, and RT-PCR data not shown) suggesting a positive correlation between the expression of these genes with cell expansion. The stomatal density phenotype in <em>sr45-1</em> showed resemblance to the stomatal density mutants such as <em>tmm</em> <a href="#pone.0000471-Yang1">[49]</a> <em>sdd</em> <a href="#pone.0000471-Berger1">[50]</a> <em>yda</em> <a href="#pone.0000471-Bergmann1">[51]</a> and a synergistically interacting group of receptor-like kinases, <em>er erl1 erl2</em> <a href="#pone.0000471-Shpak1">[52]</a>. However, an analysis of the expression of these genes in the microarray data did not reveal significant differences between <em>sr45-1</em> and WT plants, indicating that SR45 acts on yet to be identified genes for controlling stomatal density. Our work revealed the importance of splicing in development and provides a framework for investigating the role of splicing in the development of leaves and other organs.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>In addition to late-flowering phenotypes, <em>sr45-1</em> plants also had defects in the number of petals and stamens in a significant proportion of plants (<a href="#pone-0000471-g007">Figure 7</a>). The abnormal organ number observed in <em>sr45-1</em> did not follow the well-studied characteristic ABC model, which specify organ identity in each of the four whorls of a flower rather than their number in each whorl <a href="#pone.0000471-Jack1">[53]</a>. Therefore, SR45 likely controls the activities of gene(s) involved in specifying the number of floral organs. Altered organ numbers in the Arabidopsis flowers is observed in lesions in several genes, such as <em>WIGGUM/ERA1</em> <a href="#pone.0000471-Ziegelhoffer1">[54]</a>, <em>ULTRAPETALA</em> <a href="#pone.0000471-Fletcher1">[55]</a>, <em>PLURIPETALA</em> <a href="#pone.0000471-Running1">[56]</a>, <em>PIE1</em> <a href="#pone.0000471-Noh1">[57]</a>, <em>CLV1</em> <a href="#pone.0000471-Clark1">[58]</a>, <em>CLV2</em> <a href="#pone.0000471-Kayes1">[59]</a> and <em>CLV3</em> <a href="#pone.0000471-Clark2">[60]</a>. None of these genes were, however, differentially expressed (<a href="#pone.0000471.s002">Figure S2</a> and <a href="#pone.0000471.s004">Table S2</a>), suggesting that other genes, which may or may not be related to the reported genes, may be affecting the organ numbers in an SR45-dpenedent manner.</p>


<h4>RNA Targets of SR45 Include Other SR Genes</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>SR45 is a splicing factor and therefore would affect a phenotypic outcome by affecting splicing of other genes. An effect on constitutive splicing could lead to the accumulation of unspliced pre-mRNAs, which may encode truncated proteins and/or have reduced stability. Transcripts of plant SR genes themselves are the targets of AS regulation in a tissue-and development-specific manner <a href="#pone.0000471-Palusa1">[61]</a>–<a href="#pone.0000471-Kalyna2">[64]</a>. To understand the mechanism of SR45 in splicing, we analyzed the AS pattern of all SR genes in <em>sr45-1</em> and WT plants. Analyses of SR genes showed changes in the AS pattern of several SR genes, which in turn are likely to regulate the splicing of other genes. In affected SR genes, the longest intron was alternatively spliced. In most cases, the general pattern is a reduction in the usage of the distal 3′ splice sites that generate a smaller transcript and an increase in the usage of the proximal 3′ splice sites in <em>sr45-1</em> compared to WT. This indicates that the WT SR45 protein favors the usage of the distal 3′ AS sites and is responsible for a specific balance of alternative transcripts. Although the effect of depleting a specific plant SR protein on the splicing pattern of other genes has not been reported so far, studies with overexpression of two different types of plant SR proteins, SR30 and RSZ33, resulted in abnormalities in development and morphology, which were accompanied by a marked change in the splicing pattern of <em>SR30</em> and <em>SR1</em> <a href="#pone.0000471-Lopato2">[62]</a>, <a href="#pone.0000471-Kalyna2">[64]</a>. These studies indicate that the ratio of the splice variants is critical for normal plant development. SR45 also changed the splicing pattern of transcripts from several other SR genes. This, together with the fact that SR45 complements S100 cell extract, suggests that it is involved in both constitutive and regulated splicing. Based on our observation that the AS pattern of several other SR genes is differentially modulated by SR45, it is likely that the splice variants of SR proteins modulate the splicing and/or other RNA processing activities of genes involved in the developmental processes affected in the <em>sr45-1</em> plants.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plant Material and Growth Conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Seeds of T-DNA insertion lines in the <em>SR45</em> (At1g16610) in Columbia (Col) background were obtained from the SALK collection (<a href="http://signal.salk.edu/">http://signal.salk.edu/</a>; Salk_004132) at the ABRC. The T-DNA insert was verified by genomic PCR with <em>SR45</em>-specific and the T-DNA-specific primers. WT and homozygous mutant plants were grown under identical conditions of 100 µmoles m<sup>−2</sup> s<sup>−1</sup>, 16 h:8 h dark:light cycle at 22°C and 70% RH. Seeds from fully mature siliques were collected and used in subsequent experiments. Seeds from WT and <em>sr45-1</em> were surface sterilized, suspended in 0.1% Phytagar and stratified for 2 days at 4°C and depending on the experiment plated on Murashige and Skooge (MS) plates (MS salts, 1% sucrose, 0.8% Phytagar) or directly sown in soil. Root length of WT and <em>sr45-1</em> plants was measured on every second day. Size of twenty-day-old WT and <em>sr45-1</em> plants was determined from photographs with the NIH Image J (<a href="http://rsb.info.nih.gov/ij/">http://rsb.info.nih.gov/ij/</a>). In determining the relative size of <em>sr45-1</em>, WT size was considered as 100 percent.</p>


<h4>Database searches and amino acid sequence alignment</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Arabidopsis (<a href="http://www.arabidopsis.org">www.arabidopsis.org</a>), rice (<em>Oryza sativa</em> cv. <em>japonica</em>, <a href="http://cdna01.dna.affrc.go.jp/cDNA/">http://cdna01.dna.affrc.go.jp/cDNA/</a>), algae (<em>Chlamydomonas reinhardtii,</em> <a href="http://genome.jgi-psf.org/Chlre3/Chlre3.home.html">http://genome.jgi-psf.org/Chlre3/Chlre3.​home.html</a>; <em>Cyanidioschyzon merolae</em> , <a href="http://merolae.biol.s.u-tokyo.ac.jp/">http://merolae.biol.s.u-tokyo.ac.jp/</a>) a diatom (<em>Thalassiosira pseudonana,</em> <a href="http://genome.jgi-psf.org/thaps1/thaps1.home.html">http://genome.jgi-psf.org/thaps1/thaps1.​home.html</a>), fission yeast (<em>Schizosaccharomyces pombe,</em> <a href="http://www.sanger.ac.uk/Projects/S_pombe/">http://www.sanger.ac.uk/Projects/S_pombe​/</a>), budding yeast (<em>Saccharomyces cerevisiae</em>, <a href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>), human (<em>Homo sapiens</em>, <a href="http://www.ensembl.org/">http://www.ensembl.org/</a>), <em>Caenorhabditis elegans</em> (<a href="http://www.wormebase.org">www.wormebase.org</a>), fruit fly (<em>Drosophila melanogastor</em>, <a href="http://www.flybase.org">www.flybase.org</a>) and Eukaryotic Gene Orthologs [EGO; <a href="http://compbio.dfci.harvard.edu/tgi/cgi-bin/tgi/gimain.pl?">http://compbio.dfci.harvard.edu/tgi/cgi-​bin/tgi/gimain.pl?</a>gudb = maize]) databases were searched with the Arabidopsis SR45 protein sequence using BLASTP and TBLASTX. Downloaded sequences were analyzed for the presence of RRM using Interproscan (<a href="http://www.ebi.ac.uk/InterProScan/">http://www.ebi.ac.uk/InterProScan/</a>) and the presence of RS domain manually. The amino acid sequences of Arabidopsis (At1g16610), rice (AK070420 and AK063761) and maize (translated from BT016650) SR45 proteins were aligned with ClustalX using default parameters <a href="#pone.0000471-Thompson1">[65]</a>.</p>


<h4>In Vitro Splicing Assay</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The coding region of <em>SR45</em> was amplified from a cDNA clone with forward (5′CG<strong>GGATCC</strong>GCGAAACCAAGTCGTGGC3′) and reverse primers (5′CCG<strong>CTCGAG</strong>TTAAGTTTTACGAGGTGGAG3′) containing BamHI and XhoI sites (indicated in bold), respectively. The amplified product was digested with BamHI and XhoI and ligated into pFASTBAC vector and verified by sequencing. The recombinant baculovirus was prepared as recommended by the manufacturer (Invitrogen). Recombinant virus was used to infect Sf21 cells and the recombinant protein fused to His.tag was purified using Ni<sup>2+</sup> affinity chromatography <a href="#pone.0000471-Prasad1">[66]</a>. ASF/SF2, which was also fused to His.tag and expressed in insect cells, was purified essentially as SR45. For in vitro S100 complementation splicing assays, 10, 30 and 90 ng of SR45 was used with the β-globin pre-mRNA as a substrate. Twenty-five ng of His-tagged ASF/SF2 was used as a positive control. Splicing assays were carried out for 2 hours at 30°C. Splicing reactions were deproteinized and precipitated with ethanol. Spliced products were fractionated in a 6% denaturing PAGE and visualized by autoradiography.</p>


<h4>Quantification of Leaf and Petal Morphology</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Cotyledonary and rosette leaves from 3 week old WT and <em>sr45-1</em> plants grown on MS plates were excised, arranged in the order of age and photographed together. Length and width of each leaf was determined using Adobe Photoshop and the length/width ratio was calculated. Similarly, fully-opened flowers from soil-grown WT and <em>sr45-1</em> plants were photographed and the length/width ratio of petals was determined. Data were statistically analyzed for significant differences using the Students' <em>t</em>-test in Microsoft Excel.</p>


<h4>Flowering Time Measurement and Vernalization Treatment</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>For vernalization, seeds were surface-sterilized, suspended in 0.1% Phytagar and stratified for 2 days at 4°C. Seeds were incubated at 22°C for 2 days and transferred back to 4°C for another 40 days. Unvernalized seeds were surface-sterilized and stratified for 2 days at 4°C. Vernalized and unvernalized seeds were germinated and grown under LD (16 h:8 h light:dark) or SD (8 h:16 h light:dark) conditions. Flowering time was measured both as days to bolting and the number of rosette leaves at flowering. Data for 48 to 72 plants were statistically analyzed for significant differences using the Students' <em>t</em>-test in Microsoft Excel.</p>


<h4>RT-PCR Analyses of Flowering Genes and Analysis of Alternative Splicing of Pre-mRNAs of SR Proteins</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>For flowering time, leaf morphology and floral organ number, RT-PCR analyses were performed with cDNA prepared from 2-week old Arabidopsis seedlings with Takara EX Taq™ polymerase and gene specific primers (Sequences of the primers are provided in <a href="#pone.0000471.s005">Table S3</a>). PCR products were quantified with the NIH Image J software. The intensities of PCR products were normalized to cyclophilin. For determining the AS patterns of the 19 Arabidopsis SR genes, RT-PCRs with gene-specific primers were performed with total RNA isolated from root, stem, leaf and inflorescence of five-week old WT and <em>sr45-1</em> plants. Equal amount of template in each assay was verified with cyclophilin primers. The sequences of SR gene primers, which in most cases corresponded to the first and last exons, are given in <a href="#pone.0000471.s005">Table S3</a>.</p>


<h4>Microarray Experiments and Data Analyses</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Total RNA was isolated from fifteen-day old seedlings grown on MS plates at 16h:8h light:dark cycle and 22°C according to the Trizol method (Invitrogen). The RNA samples were treated with DNAseI and purified using the Qiagen RNA isolation columns. Complementary RNA synthesis, hybridization to the Affymetrix Arabidopsis Genome ATH1 Array, data acquisition, processing and analyses were done according to the Affymetrix GeneChip Expression Analysis Technical Manual (Affymetrix, Santa Clara, CA). Two additional experiments using similar conditions and GeneChip analyses as above were repeated several months apart as a biological replicate. These three experiments generated three control and three <em>sr45-1</em> data sets. Each data set was initially analyzed separately with the Affymetrix MAS5.0 software, which yielded log ratios of <em>sr45-1</em> signal to WT and an associated <em>p</em>-value indicating the significance of the change. For statistical comparison analyses, gene expression data were calculated using the model-based expression index-perfect match (MBEI-PM) algorithm with invariant set normalization in the dChip software (<a href="http://www.dchip.org">http://www.dchip.org</a>) <a href="#pone.0000471-Li1">[67]</a>. The Gene expression data were analyzed for significant differential expression with a moderated paired <em>t</em>-test in the R/Bioconductor software (<a href="http://www.R-project.org">http://www.R-project.org</a>). The resultant <em>p</em>-values were corrected for false discovery rate (FDR) using the Benjamini-Hochberg (HB) algorithm in R/Bioconductor <a href="#pone.0000471-Benjamini1">[68]</a>. Genes with at least two-fold change with an associated FDR-corrected <em>p</em>-value≤0.05 were considered significantly changed in <em>sr45-1</em> relative to WT.</p>


<h4>Scanning electron microscopy</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Scanning electron microscopy was conducted as described <a href="#pone.0000471-Reddy4">[69]</a>. Cell sizes were measured with the NIH Image J software in at least three SEM images of WT and <em>sr45-1</em> leaves.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000471"></div><a name="pone.0000471.s001" id="pone.0000471.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000471.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Expression analyses of flowering time genes in WT and <em>sr45-1</em> plants. (A) RT-PCR was performed with total RNA isolated from two week old plants with gene-specific primers as described in the Experimental procedures. For full name of genes see the legend of <a href="#pone.0000471.s001">Figure S1B</a> below. (B) PCR bands were quantified with NIH Image J software (<a href="http://rsb.info.nih.gov/ij/">http://rsb.info.nih.gov/ij/</a>). Each PCR product in WT and <em>sr45-1</em> was normalized to cyclophilin transcript level in WT and <em>sr45-1</em>, respectively. Data shown are the percent of WT, with WT levels adjusted to 100 percent. VRN, The vernalization pathway; Autonomous, The autonomous flowering pathway; PAF1, RNA polymerase II (Pol II) Associated Factor 1-complex; VIP, vernalization independence; Integrator; Floral pathway integrators; Photoperiod, Photoperiod pathway genes; MIG, meristem identity genes. <em>VRN1,VERNALIZATION 1; VRN2, VERNALIZATION 2; FCA, FCA protein; FY, FY protein; FLD, Flowering Locus D; FVE, FVE protein; LD, LUMINIDEPENDENS; FLK, FLOWERING LATE KH DOMAIN; PIE1, PHOTOPERIOD INDEPENDENT EARLY FLOWERING1; ELF7, EARLY FLOWERING 7; ELF8, EARLY FLOWERING 8; VIP4, VERNALIZATION INDEPENDENCE 4; VIP3, VERNALIZATION INDEPENDENCE 3; FRI, FRIGIDA; FRL1, FRIGIDA-LIKE 1; GAI, GA INSENSITIVE; FLC, FLOWERING LOCUS C; SOC1, SUPPRESSER OF OVEREXPRESSER OF CONSTANS 1; FT, FLOWERING TIME T; CRY2, CRYPTOCHROME2; PHYA, PHYTOCHROME A; AP1, APETELLA 1; LFY, LEAFY; CYC, CYCLOPHILLIN</em>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000471.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.19 MB TIF)</p>
<a name="pone.0000471.s002" id="pone.0000471.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000471.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Expression analyses of leaf shape and floral organ number genes in WT and <em>sr45-1</em> plants. (A) RT-PCR was performed with total RNA isolated from two week old plants with gene-specific primers as described in the Experimental procedures. Left panel consists of RT-PCR of leaf morphology genes; right panel consists of genes affecting petal numbers. <em>AS1, ASYMMETRIC LEAVES1; AS2, ASYMMETRIC LEAVES2; AN, ANGUSTIFOLIA; AN3, ANGUSTIFOLIA3; ROT3, ROTUNDIFOLIA3; DRL, DEFORMED ROOTS AND LEAVES 1; ATHB13, HOMEODOMAIN LEUCINE-ZIPPER PROTEIN ATHB13; WIG, WIGGUM; ULT, ULTRAPETALLA; CYC, CYCLOPHILLIN</em>. (B) PCR bands were quantified with NIH Image J software (<a href="http://rsb.info.nih.gov/ij/">http://rsb.info.nih.gov/ij/</a>). Each PCR product in WT and <em>sr45-1</em> was normalized to cyclophilin transcript level in WT and <em>sr45-1</em>, respectively. Data shown are the percent of WT, with WT levels adjusted to 100 percent.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000471.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.74 MB TIF)</p>
<a name="pone.0000471.s003" id="pone.0000471.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000471.s003">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Summary of Microarray Analyses of Flowering Time Genes in <em>sr45-1</em> and WT Arabidopsis. Gene expression data of <em>sr45-1</em> and WT consisting of three biological replicates were statistically analyzed as described in <a href="#s4">Materials and Methods</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000471.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.11 MB DOC)</p>
<a name="pone.0000471.s004" id="pone.0000471.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000471.s004">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Gene expression analyses of all genes present on the Affymetrix Arabidopsis gene chip.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000471.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(3.25 MB XLS)</p>
<a name="pone.0000471.s005" id="pone.0000471.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000471.s005">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Sequences of gene-specific primers of Arabidopsis SR, flowering pathway and leaf morphology genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000471.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.08 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Arabidopsis Biological Resource Center for providing the T-DNA mutant line and Dr. Irene Day for critical reading of the manuscript. We also thank Dr. Ann Hess for help with the microarray data analyses.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: AR GA. Performed the experiments: GA SP MG JP. Analyzed the data: JM AR GA SP. Wrote the paper: AR GA.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000471-Reddy1" id="pone.0000471-Reddy1"></a>Reddy AS (2007) Alternative Splicing of Pre-Messenger RNAs in Plants in the Genomic Era. Annu Rev Plant Biol  58: 267–294.  <ul class="find" data-citedArticleID="953944" data-doi="10.1146/annurev.arplant.58.032806.103754"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.58.032806.103754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+Splicing+of+Pre-Messenger+RNAs+in+Plants+in+the+Genomic+Era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+Splicing+of+Pre-Messenger+RNAs+in+Plants+in+the+Genomic+Era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000471-Huang1" id="pone.0000471-Huang1"></a>Huang Y, Steitz JA (2005) SRprises along a messenger's journey. Mol Cell  17: 613–615.  <ul class="find" data-citedArticleID="953890" data-doi="10.1016/j.molcel.2005.02.020"><li><a href="http://dx.doi.org/10.1016/j.molcel.2005.02.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SRprises+along+a+messenger%27s+journey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SRprises+along+a+messenger%27s+journey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000471-Graveley1" id="pone.0000471-Graveley1"></a>Graveley BR (2000) Sorting out the complexity of SR protein functions. RNA  6: 1197–1211.  <ul class="find" data-citedArticleID="953880" data-doi="10.1017/s1355838200000960"><li><a href="http://dx.doi.org/10.1017/s1355838200000960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sorting+out+the+complexity+of+SR+protein+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sorting+out+the+complexity+of+SR+protein+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000471-Sanford1" id="pone.0000471-Sanford1"></a>Sanford JR, Longman D, Caceres JF (2003)  Multiple roles of the SR protein family in splicing regulation. In: Jeanteur P, editor. Regulation of alternative splicing. New York: Springer. pp. 33–58.  <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000471-Reddy2" id="pone.0000471-Reddy2"></a>Reddy ASN (2001) Nuclear pre-mRNA splicing in plants. CRC Crit Rev Plant Sci  20: 523–572.  <ul class="find" data-citedArticleID="953946" data-doi="10.1080/20013591099272"><li><a href="http://dx.doi.org/10.1080/20013591099272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+pre-mRNA+splicing+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+pre-mRNA+splicing+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000471-Shen1" id="pone.0000471-Shen1"></a>Shen H, Kan JL, Green MR (2004) Arginine-serine-rich domains bound at splicing enhancers contact the branchpoint to promote prespliceosome assembly. Mol Cell  13: 367–376.  <ul class="find" data-citedArticleID="953962" data-doi="10.1016/s1097-2765(04)00025-5"><li><a href="http://dx.doi.org/10.1016/s1097-2765(04)00025-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arginine-serine-rich+domains+bound+at+splicing+enhancers+contact+the+branchpoint+to+promote+prespliceosome+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arginine-serine-rich+domains+bound+at+splicing+enhancers+contact+the+branchpoint+to+promote+prespliceosome+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000471-Manley1" id="pone.0000471-Manley1"></a>Manley JL, Tacke R (1996) SR proteins and splicing control. Genes Dev  10: 1569–1579.  <ul class="find" data-citedArticleID="953928" data-doi="10.1101/gad.10.13.1569"><li><a href="http://dx.doi.org/10.1101/gad.10.13.1569" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SR+proteins+and+splicing+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SR+proteins+and+splicing+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000471-Wu1" id="pone.0000471-Wu1"></a>Wu JY, Maniatis T (1993) Specific interactions between proteins implicated in splice site selection and regulated alternative splicing. Cell  75: 1061–1070.  <ul class="find" data-citedArticleID="953982" data-doi="10.1016/0092-8674(93)90316-i"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90316-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+interactions+between+proteins+implicated+in+splice+site+selection+and+regulated+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+interactions+between+proteins+implicated+in+splice+site+selection+and+regulated+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000471-Golovkin1" id="pone.0000471-Golovkin1"></a>Golovkin M, Reddy ASN (1996) Structure and expression of a plant U1 snRNP 70K gene: alternative splicing of U1 snRNP 70K pre-mRNAs produces two different transcripts. Plant Cell  8: 1421–1435.  <ul class="find" data-citedArticleID="953874" data-doi="10.2307/3870311"><li><a href="http://dx.doi.org/10.2307/3870311" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+expression+of+a+plant+U1+snRNP+70K+gene%3A+alternative+splicing+of+U1+snRNP+70K+pre-mRNAs+produces+two+different+transcripts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+expression+of+a+plant+U1+snRNP+70K+gene%3A+alternative+splicing+of+U1+snRNP+70K+pre-mRNAs+produces+two+different+transcripts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000471-Reddy3" id="pone.0000471-Reddy3"></a>Reddy AS (2004) Plant serine/arginine-rich proteins and their role in pre-mRNA splicing. Trends Plant Sci  9: 541–547.  <ul class="find" data-citedArticleID="953948" data-doi="10.1016/j.tplants.2004.09.007"><li><a href="http://dx.doi.org/10.1016/j.tplants.2004.09.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+serine%2Farginine-rich+proteins+and+their+role+in+pre-mRNA+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+serine%2Farginine-rich+proteins+and+their+role+in+pre-mRNA+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000471-Kalyna1" id="pone.0000471-Kalyna1"></a>Kalyna M, Barta A (2004) A plethora of plant serine/arginine-rich proteins: redundancy or evolution of novel gene functions? Biochem Soc Trans  32: 561–564.  <ul class="find" data-citedArticleID="953898" data-doi="10.1042/bst0320561"><li><a href="http://dx.doi.org/10.1042/bst0320561" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+plethora+of+plant+serine%2Farginine-rich+proteins%3A+redundancy+or+evolution+of+novel+gene+functions%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+plethora+of+plant+serine%2Farginine-rich+proteins%3A+redundancy+or+evolution+of+novel+gene+functions%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000471-Isshiki1" id="pone.0000471-Isshiki1"></a>Isshiki M, Tsumoto A, Shimamoto K (2006) The serine/arginine-rich protein family in rice plays important roles in constitutive and alternative splicing of pre-mRNA. Plant Cell  18: 146–158.  <ul class="find" data-citedArticleID="953894" data-doi="10.1105/tpc.105.037069"><li><a href="http://dx.doi.org/10.1105/tpc.105.037069" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+serine%2Farginine-rich+protein+family+in+rice+plays+important+roles+in+constitutive+and+alternative+splicing+of+pre-mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+serine%2Farginine-rich+protein+family+in+rice+plays+important+roles+in+constitutive+and+alternative+splicing+of+pre-mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000471-Iida1" id="pone.0000471-Iida1"></a>Iida K, Go M (2006) Survey of Conserved Alternative Splicing Events of mRNAs Encoding SR Proteins in Land Plants. Mol Biol Evol  23: 1085–1094.  <ul class="find" data-citedArticleID="953892" data-doi="10.1093/molbev/msj118"><li><a href="http://dx.doi.org/10.1093/molbev/msj118" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survey+of+Conserved+Alternative+Splicing+Events+of+mRNAs+Encoding+SR+Proteins+in+Land+Plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survey+of+Conserved+Alternative+Splicing+Events+of+mRNAs+Encoding+SR+Proteins+in+Land+Plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000471-Romac1" id="pone.0000471-Romac1"></a>Romac JM, Keene JD (1995) Overexpression of the arginine-rich carboxy-terminal region of U1 snRNP 70K inhibits both splicing and nucleocytoplasmic transport of mRNA. Genes Dev  9: 1400–1410.  <ul class="find" data-citedArticleID="953952" data-doi="10.1101/gad.9.11.1400"><li><a href="http://dx.doi.org/10.1101/gad.9.11.1400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+the+arginine-rich+carboxy-terminal+region+of+U1+snRNP+70K+inhibits+both+splicing+and+nucleocytoplasmic+transport+of+mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+the+arginine-rich+carboxy-terminal+region+of+U1+snRNP+70K+inhibits+both+splicing+and+nucleocytoplasmic+transport+of+mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000471-Kohtz1" id="pone.0000471-Kohtz1"></a>Kohtz JD, Jamison SF, Will CL, Zuo P, Lührmann R, et al.  (1994) Protein-protein interactions and 5′-splice-site recognition in mammalian mRNA precursors. Nature  368: 119–124.  <ul class="find" data-citedArticleID="953908" data-doi="10.1038/368119a0"><li><a href="http://dx.doi.org/10.1038/368119a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein-protein+interactions+and+5%26%23x2032%3B-splice-site+recognition+in+mammalian+mRNA+precursors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein-protein+interactions+and+5%26%23x2032%3B-splice-site+recognition+in+mammalian+mRNA+precursors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000471-Golovkin2" id="pone.0000471-Golovkin2"></a>Golovkin M, Reddy ASN (1998) The plant U1 small nuclear ribonucleoprotein particle 70K protein interacts with two novel serine/arginine-rich proteins. Plant Cell  10: 1637–1648.  <ul class="find" data-citedArticleID="953876" data-doi="10.1105/tpc.10.10.1637"><li><a href="http://dx.doi.org/10.1105/tpc.10.10.1637" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+plant+U1+small+nuclear+ribonucleoprotein+particle+70K+protein+interacts+with+two+novel+serine%2Farginine-rich+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+plant+U1+small+nuclear+ribonucleoprotein+particle+70K+protein+interacts+with+two+novel+serine%2Farginine-rich+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000471-Golovkin3" id="pone.0000471-Golovkin3"></a>Golovkin M, Reddy ASN (1999) An SC35-like protein and a novel serine/arginine-rich protein interact with Arabidopsis U1-70K protein. J Biol Chem  274: 36428–36438.  <ul class="find" data-citedArticleID="953878" data-doi="10.1074/jbc.274.51.36428"><li><a href="http://dx.doi.org/10.1074/jbc.274.51.36428" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+SC35-like+protein+and+a+novel+serine%2Farginine-rich+protein+interact+with+Arabidopsis+U1-70K+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+SC35-like+protein+and+a+novel+serine%2Farginine-rich+protein+interact+with+Arabidopsis+U1-70K+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000471-Lopato1" id="pone.0000471-Lopato1"></a>Lopato S, Forstner C, Kalyna M, Hilscher J, Langhammer U, et al.  (2002) Network of interactions of a novel plant-specific Arg/Ser-rich protein, atRSZ33, with atSC35-like splicing factors. J Biol Chem  277: 39989–39998.  <ul class="find" data-citedArticleID="953922" data-doi="10.1074/jbc.m206455200"><li><a href="http://dx.doi.org/10.1074/jbc.m206455200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Network+of+interactions+of+a+novel+plant-specific+Arg%2FSer-rich+protein%2C+atRSZ33%2C+with+atSC35-like+splicing+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Network+of+interactions+of+a+novel+plant-specific+Arg%2FSer-rich+protein%2C+atRSZ33%2C+with+atSC35-like+splicing+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000471-Ali1" id="pone.0000471-Ali1"></a>Ali GS, Reddy AS (2006) ATP, phosphorylation and transcription regulate the mobility of plant splicing factors. J Cell Sci  119: 3527–3538.  <ul class="find" data-citedArticleID="953848" data-doi="10.1242/jcs.03144"><li><a href="http://dx.doi.org/10.1242/jcs.03144" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ATP%2C+phosphorylation+and+transcription+regulate+the+mobility+of+plant+splicing+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ATP%2C+phosphorylation+and+transcription+regulate+the+mobility+of+plant+splicing+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000471-Ali2" id="pone.0000471-Ali2"></a>Ali GS, Golovkin M, Reddy AS (2003) Nuclear localization and in vivo dynamics of a plant-specific serine/arginine-rich protein. Plant J  36: 883–893.  <ul class="find" data-citedArticleID="953850" data-doi="10.1046/j.1365-313x.2003.01932.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.01932.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+localization+and+in+vivo+dynamics+of+a+plant-specific+serine%2Farginine-rich+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+localization+and+in+vivo+dynamics+of+a+plant-specific+serine%2Farginine-rich+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000471-Wang1" id="pone.0000471-Wang1"></a>Wang BB, Brendel V (2004) The ASRG database: identification and survey of Arabidopsis thaliana genes involved in pre-mRNA splicing. Genome Biol  5: R102.  <ul class="find" data-citedArticleID="953980"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+ASRG+database%3A+identification+and+survey+of+Arabidopsis+thaliana+genes+involved+in+pre-mRNA+splicing.&amp;auth=&amp;atitle=The+ASRG+database%3A+identification+and+survey+of+Arabidopsis+thaliana+genes+involved+in+pre-mRNA+splicing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ASRG+database%3A+identification+and+survey+of+Arabidopsis+thaliana+genes+involved+in+pre-mRNA+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ASRG+database%3A+identification+and+survey+of+Arabidopsis+thaliana+genes+involved+in+pre-mRNA+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000471-Tacke1" id="pone.0000471-Tacke1"></a>Tacke R, Tohyama M, Ogawa S, Manley JL (1998) Human Tra2 proteins are sequence-specific activators of pre-mRNA splicing. Cell  93: 139–148.  <ul class="find" data-citedArticleID="953970" data-doi="10.1016/s0092-8674(00)81153-8"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81153-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Tra2+proteins+are+sequence-specific+activators+of+pre-mRNA+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Tra2+proteins+are+sequence-specific+activators+of+pre-mRNA+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000471-Sung1" id="pone.0000471-Sung1"></a>Sung S, Amasino RM (2005) REMEMBERING WINTER: Toward a Molecular Understanding of Vernalization. Annu Rev Plant Biol  56: 491–508.  <ul class="find" data-citedArticleID="953968" data-doi="10.1146/annurev.arplant.56.032604.144307"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.56.032604.144307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=REMEMBERING+WINTER%3A+Toward+a+Molecular+Understanding+of+Vernalization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22REMEMBERING+WINTER%3A+Toward+a+Molecular+Understanding+of+Vernalization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000471-Boss1" id="pone.0000471-Boss1"></a>Boss PK, Bastow RM, Mylne JS, Dean C (2004) Multiple Pathways in the Decision to Flower: Enabling, Promoting, and Resetting. 16: S18–S31.  <ul class="find" data-citedArticleID="953860" data-doi="10.1105/tpc.015958"><li><a href="http://dx.doi.org/10.1105/tpc.015958" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+Pathways+in+the+Decision+to+Flower%3A+Enabling%2C+Promoting%2C+and+Resetting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+Pathways+in+the+Decision+to+Flower%3A+Enabling%2C+Promoting%2C+and+Resetting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000471-Sheldon1" id="pone.0000471-Sheldon1"></a>Sheldon CC, Burn JE, Perez PP, Metzger J, Edwards JA, et al.  (1999) The <em>FLF</em> MADS box gene: a repressor of flowering in Arabidopsis regulated by vernalization and methylation. Plant Cell  11: 445–458.  <ul class="find" data-citedArticleID="953960" data-doi="10.1105/tpc.11.3.445"><li><a href="http://dx.doi.org/10.1105/tpc.11.3.445" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+FLF+MADS+box+gene%3A+a+repressor+of+flowering+in+Arabidopsis+regulated+by+vernalization+and+methylation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+FLF+MADS+box+gene%3A+a+repressor+of+flowering+in+Arabidopsis+regulated+by+vernalization+and+methylation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000471-Michaels1" id="pone.0000471-Michaels1"></a>Michaels SD, Amasino RM (1999) FLOWERING LOCUS C encodes a novel MADS domain protein that acts as a repressor of flowering. Plant Cell  11: 949–956.  <ul class="find" data-citedArticleID="953930" data-doi="10.1105/tpc.11.5.949"><li><a href="http://dx.doi.org/10.1105/tpc.11.5.949" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FLOWERING+LOCUS+C+encodes+a+novel+MADS+domain+protein+that+acts+as+a+repressor+of+flowering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FLOWERING+LOCUS+C+encodes+a+novel+MADS+domain+protein+that+acts+as+a+repressor+of+flowering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000471-Macknight1" id="pone.0000471-Macknight1"></a>Macknight R, Duroux M, Laurie R, Dijkwel P, Simpson G, et al.  (2002) Functional significance of the alternative transcript processing of the Arabidopsis floral promoter <em>FCA</em>. Plant Cell  14: 877–888.  <ul class="find" data-citedArticleID="953926" data-doi="10.1105/tpc.010456"><li><a href="http://dx.doi.org/10.1105/tpc.010456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+significance+of+the+alternative+transcript+processing+of+the+Arabidopsis+floral+promoter+FCA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+significance+of+the+alternative+transcript+processing+of+the+Arabidopsis+floral+promoter+FCA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000471-Simpson1" id="pone.0000471-Simpson1"></a>Simpson GG, Dijkwel PP, Quesada V, Henderson I, Dean C (2003) FY is an RNA 3′ end-processing factor that interacts with FCA to control the Arabidopsis floral transition. Cell  113: 777–787.  <ul class="find" data-citedArticleID="953966" data-doi="10.1016/s0092-8674(03)00425-2"><li><a href="http://dx.doi.org/10.1016/s0092-8674(03)00425-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FY+is+an+RNA+3%26%23x2032%3B+end-processing+factor+that+interacts+with+FCA+to+control+the+Arabidopsis+floral+transition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FY+is+an+RNA+3%26%23x2032%3B+end-processing+factor+that+interacts+with+FCA+to+control+the+Arabidopsis+floral+transition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000471-Ausin1" id="pone.0000471-Ausin1"></a>Ausin I, Alonso-Blanco C, Jarillo JA, Ruiz-Garcia L, Martinez-Zapater JM (2004) Regulation of flowering time by FVE, a retinoblastoma-associated protein. Nat Genet  36: 162–166.  <ul class="find" data-citedArticleID="953852" data-doi="10.1038/ng1295"><li><a href="http://dx.doi.org/10.1038/ng1295" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+flowering+time+by+FVE%2C+a+retinoblastoma-associated+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+flowering+time+by+FVE%2C+a+retinoblastoma-associated+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000471-He1" id="pone.0000471-He1"></a>He Y, Michaels SD, Amasino RM (2003) Regulation of flowering time by histone acetylation in Arabidopsis. Science  302: 1751–1754.  <ul class="find" data-citedArticleID="953884" data-doi="10.1126/science.1091109"><li><a href="http://dx.doi.org/10.1126/science.1091109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+flowering+time+by+histone+acetylation+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+flowering+time+by+histone+acetylation+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000471-Lee1" id="pone.0000471-Lee1"></a>Lee I, Aukerman MJ, Gore SL, Lohman KN, Michaels SD, et al.  (1994) Isolation of <em>LUMINIDEPENDENS</em>: a gene involved in the control of flowering time in Arabidopsis. Plant Cell  6: 75–83.  <ul class="find" data-citedArticleID="953914" data-doi="10.2307/3869676"><li><a href="http://dx.doi.org/10.2307/3869676" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+LUMINIDEPENDENS%3A+a+gene+involved+in+the+control+of+flowering+time+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+LUMINIDEPENDENS%3A+a+gene+involved+in+the+control+of+flowering+time+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000471-Mockler1" id="pone.0000471-Mockler1"></a>Mockler TC, Yu X, Shalitin D, Parikh D, Michael TP, et al.  (2004) Regulation of flowering time in Arabidopsis by K homology domain proteins. Proc Natl Acad Sci U S A  101: 12759–12764.  <ul class="find" data-citedArticleID="953932" data-doi="10.1073/pnas.0404552101"><li><a href="http://dx.doi.org/10.1073/pnas.0404552101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+flowering+time+in+Arabidopsis+by+K+homology+domain+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+flowering+time+in+Arabidopsis+by+K+homology+domain+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000471-Lim1" id="pone.0000471-Lim1"></a>Lim MH, Kim J, Kim YS, Chung KS, Seo YH, et al.  (2004) A new Arabidopsis gene, <em>FLK</em>, encodes an RNA binding protein with K homology motifs and regulates flowering time via <em>FLOWERING LOCUS C</em>. Plant Cell  16: 731–740.  <ul class="find" data-citedArticleID="953920" data-doi="10.1105/tpc.019331"><li><a href="http://dx.doi.org/10.1105/tpc.019331" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+Arabidopsis+gene%2C+FLK%2C+encodes+an+RNA+binding+protein+with+K+homology+motifs+and+regulates+flowering+time+via+FLOWERING+LOCUS+C." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+Arabidopsis+gene%2C+FLK%2C+encodes+an+RNA+binding+protein+with+K+homology+motifs+and+regulates+flowering+time+via+FLOWERING+LOCUS+C.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000471-Schomburg1" id="pone.0000471-Schomburg1"></a>Schomburg FM, Patton DA, Meinke DW, Amasino RM (2001) <em>FPA</em>, a gene involved in floral induction in Arabidopsis, encodes a protein containing RNA-recognition motifs. Plant Cell  13: 1427–1436.  <ul class="find" data-citedArticleID="953958" data-doi="10.2307/3871305"><li><a href="http://dx.doi.org/10.2307/3871305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FPA%2C+a+gene+involved+in+floral+induction+in+Arabidopsis%2C+encodes+a+protein+containing+RNA-recognition+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FPA%2C+a+gene+involved+in+floral+induction+in+Arabidopsis%2C+encodes+a+protein+containing+RNA-recognition+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000471-Gendall1" id="pone.0000471-Gendall1"></a>Gendall AR, Levy YY, Wilson A, Dean C (2001) The VERNALIZATION 2 gene mediates the epigenetic regulation of vernalization in Arabidopsis. Cell  107: 525–535.  <ul class="find" data-citedArticleID="953872" data-doi="10.1016/s0092-8674(01)00573-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00573-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+VERNALIZATION+2+gene+mediates+the+epigenetic+regulation+of+vernalization+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+VERNALIZATION+2+gene+mediates+the+epigenetic+regulation+of+vernalization+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000471-Lee2" id="pone.0000471-Lee2"></a>Lee H, Suh SS, Park E, Cho E, Ahn JH, et al.  (2000) The AGAMOUS-LIKE 20 MADS domain protein integrates floral inductive pathways in Arabidopsis. Genes Dev  14: 2366–2376.  <ul class="find" data-citedArticleID="953916" data-doi="10.1101/gad.813600"><li><a href="http://dx.doi.org/10.1101/gad.813600" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+AGAMOUS-LIKE+20+MADS+domain+protein+integrates+floral+inductive+pathways+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+AGAMOUS-LIKE+20+MADS+domain+protein+integrates+floral+inductive+pathways+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000471-Kardailsky1" id="pone.0000471-Kardailsky1"></a>Kardailsky I, Shukla VK, Ahn JH, Dagenais N, Christensen SK, et al.  (1999) Activation tagging of the floral inducer FT. Science  286: 1962–1965.  <ul class="find" data-citedArticleID="953902" data-doi="10.1126/science.286.5446.1962"><li><a href="http://dx.doi.org/10.1126/science.286.5446.1962" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activation+tagging+of+the+floral+inducer+FT." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activation+tagging+of+the+floral+inducer+FT.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000471-Putterill1" id="pone.0000471-Putterill1"></a>Putterill J, Robson F, Lee K, Simon R, Coupland G (1995) The <em>CONSTANS</em> gene of Arabidopsis promotes flowering and encodes a protein showing similarities to zinc finger transcription factors. Cell  80: 847–857.  <ul class="find" data-citedArticleID="953942" data-doi="10.1016/0092-8674(95)90288-0"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90288-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CONSTANS+gene+of+Arabidopsis+promotes+flowering+and+encodes+a+protein+showing+similarities+to+zinc+finger+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CONSTANS+gene+of+Arabidopsis+promotes+flowering+and+encodes+a+protein+showing+similarities+to+zinc+finger+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000471-Lamond1" id="pone.0000471-Lamond1"></a>Lamond AI, Spector DL (2003) Nuclear speckles: a model for nuclear organelles. Nat Rev Mol Cell Biol  4: 605–612.  <ul class="find" data-citedArticleID="953910" data-doi="10.1038/nrm1172"><li><a href="http://dx.doi.org/10.1038/nrm1172" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+speckles%3A+a+model+for+nuclear+organelles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+speckles%3A+a+model+for+nuclear+organelles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000471-Tsukaya1" id="pone.0000471-Tsukaya1"></a>Tsukaya H (2006) Mechanism of Leaf Shape Determination. Annu Rev Plant Biol  57: 477–496.  <ul class="find" data-citedArticleID="953976" data-doi="10.1146/annurev.arplant.57.032905.105320"><li><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105320" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+Leaf+Shape+Determination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+Leaf+Shape+Determination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000471-Byrne1" id="pone.0000471-Byrne1"></a>Byrne ME (2005) Networks in leaf development. Curr Opin Plant Biol  8: 59–66.  <ul class="find" data-citedArticleID="953862" data-doi="10.1016/j.pbi.2004.11.009"><li><a href="http://dx.doi.org/10.1016/j.pbi.2004.11.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Networks+in+leaf+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Networks+in+leaf+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000471-Horiguchi1" id="pone.0000471-Horiguchi1"></a>Horiguchi G, Kim GT, Tsukaya H (2005) The transcription factor AtGRF5 and the transcription coactivator AN3 regulate cell proliferation in leaf primordia of Arabidopsis thaliana. Plant J  43: 68–78.  <ul class="find" data-citedArticleID="953888" data-doi="10.1111/j.1365-313x.2005.02429.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2005.02429.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcription+factor+AtGRF5+and+the+transcription+coactivator+AN3+regulate+cell+proliferation+in+leaf+primordia+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcription+factor+AtGRF5+and+the+transcription+coactivator+AN3+regulate+cell+proliferation+in+leaf+primordia+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000471-Kim1" id="pone.0000471-Kim1"></a>Kim GT, Tsukaya H, Uchimiya H (1998) <em>The CURLY LEAF</em> gene controls both division and elongation of cells during the expansion of the leaf blade in <em>Arabidopsis thaliana</em>. Planta  206: 175–183.  <ul class="find" data-citedArticleID="953906" data-doi="10.1007/s004250050389"><li><a href="http://dx.doi.org/10.1007/s004250050389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CURLY+LEAF+gene+controls+both+division+and+elongation+of+cells+during+the+expansion+of+the+leaf+blade+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CURLY+LEAF+gene+controls+both+division+and+elongation+of+cells+during+the+expansion+of+the+leaf+blade+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000471-Nelissen1" id="pone.0000471-Nelissen1"></a>Nelissen H, Clarke JH, De Block M, De Block S, Vanderhaeghen R, et al.  (2003) DRL1, a homolog of the yeast TOT4/KTI12 protein, has a function in meristem activity and organ growth in plants. Plant Cell  15: 639–654.  <ul class="find" data-citedArticleID="953934" data-doi="10.1105/tpc.007062"><li><a href="http://dx.doi.org/10.1105/tpc.007062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DRL1%2C+a+homolog+of+the+yeast+TOT4%2FKTI12+protein%2C+has+a+function+in+meristem+activity+and+organ+growth+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DRL1%2C+a+homolog+of+the+yeast+TOT4%2FKTI12+protein%2C+has+a+function+in+meristem+activity+and+organ+growth+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000471-Theodoris1" id="pone.0000471-Theodoris1"></a>Theodoris G, Inada N, Freeling M (2003) Conservation and molecular dissection of ROUGH SHEATH2 and ASYMMETRIC LEAVES1 function in leaf development. Proc Natl Acad Sci U S A  100: 6837–6842.  <ul class="find" data-citedArticleID="953972" data-doi="10.1073/pnas.1132113100"><li><a href="http://dx.doi.org/10.1073/pnas.1132113100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conservation+and+molecular+dissection+of+ROUGH+SHEATH2+and+ASYMMETRIC+LEAVES1+function+in+leaf+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conservation+and+molecular+dissection+of+ROUGH+SHEATH2+and+ASYMMETRIC+LEAVES1+function+in+leaf+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000471-ChalfunJunior1" id="pone.0000471-ChalfunJunior1"></a>Chalfun-Junior A, Franken J, Mes JJ, Marsch-Martinez N, Pereira A, et al.  (2005) ASYMMETRIC LEAVES2-LIKE1 gene, a member of the AS2/LOB family, controls proximal-distal patterning in Arabidopsis petals. Plant Mol Biol  57: 559–575.  <ul class="find" data-citedArticleID="953864" data-doi="10.1007/s11103-005-0698-4"><li><a href="http://dx.doi.org/10.1007/s11103-005-0698-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ASYMMETRIC+LEAVES2-LIKE1+gene%2C+a+member+of+the+AS2%2FLOB+family%2C+controls+proximal-distal+patterning+in+Arabidopsis+petals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ASYMMETRIC+LEAVES2-LIKE1+gene%2C+a+member+of+the+AS2%2FLOB+family%2C+controls+proximal-distal+patterning+in+Arabidopsis+petals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000471-Hanson1" id="pone.0000471-Hanson1"></a>Hanson J, Johannesson H, Engstrom P (2001) Sugar-dependent alterations in cotyledon and leaf development in transgenic plants expressing the HDZhdip gene ATHB13. Plant Mol Biol  45: 247–262.  <ul class="find" data-citedArticleID="953882"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sugar-dependent+alterations+in+cotyledon+and+leaf+development+in+transgenic+plants+expressing+the+HDZhdip+gene+ATHB13.&amp;auth=&amp;atitle=Sugar-dependent+alterations+in+cotyledon+and+leaf+development+in+transgenic+plants+expressing+the+HDZhdip+gene+ATHB13." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sugar-dependent+alterations+in+cotyledon+and+leaf+development+in+transgenic+plants+expressing+the+HDZhdip+gene+ATHB13." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sugar-dependent+alterations+in+cotyledon+and+leaf+development+in+transgenic+plants+expressing+the+HDZhdip+gene+ATHB13.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000471-Wagner1" id="pone.0000471-Wagner1"></a>Wagner TA, Kohorn BD (2001) Wall-associated kinases are expressed throughout plant development and are required for cell expansion. Plant Cell  13: 303–318.  <ul class="find" data-citedArticleID="953978" data-doi="10.1105/tpc.13.2.303"><li><a href="http://dx.doi.org/10.1105/tpc.13.2.303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wall-associated+kinases+are+expressed+throughout+plant+development+and+are+required+for+cell+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wall-associated+kinases+are+expressed+throughout+plant+development+and+are+required+for+cell+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000471-Yang1" id="pone.0000471-Yang1"></a>Yang M, Sack FD (1995) The too many mouths and four lips mutations affect stomatal production in Arabidopsis. Plant Cell  7: 2227–2239.  <ul class="find" data-citedArticleID="953984" data-doi="10.2307/3870164"><li><a href="http://dx.doi.org/10.2307/3870164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+too+many+mouths+and+four+lips+mutations+affect+stomatal+production+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+too+many+mouths+and+four+lips+mutations+affect+stomatal+production+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000471-Berger1" id="pone.0000471-Berger1"></a>Berger D, Altmann T (2000) A subtilisin-like serine protease involved in the regulation of stomatal density and distribution in Arabidopsis thaliana. Genes Dev  14: 1119–1131.  <ul class="find" data-citedArticleID="953856"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+subtilisin-like+serine+protease+involved+in+the+regulation+of+stomatal+density+and+distribution+in+Arabidopsis+thaliana.&amp;auth=&amp;atitle=A+subtilisin-like+serine+protease+involved+in+the+regulation+of+stomatal+density+and+distribution+in+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+subtilisin-like+serine+protease+involved+in+the+regulation+of+stomatal+density+and+distribution+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+subtilisin-like+serine+protease+involved+in+the+regulation+of+stomatal+density+and+distribution+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000471-Bergmann1" id="pone.0000471-Bergmann1"></a>Bergmann DC, Lukowitz W, Somerville CR (2004) Stomatal development and pattern controlled by a MAPKK kinase. Science  304: 1494–1497.  <ul class="find" data-citedArticleID="953858" data-doi="10.1126/science.1096014"><li><a href="http://dx.doi.org/10.1126/science.1096014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stomatal+development+and+pattern+controlled+by+a+MAPKK+kinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stomatal+development+and+pattern+controlled+by+a+MAPKK+kinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000471-Shpak1" id="pone.0000471-Shpak1"></a>Shpak ED, McAbee JM, Pillitteri LJ, Torii KU (2005) Stomatal patterning and differentiation by synergistic interactions of receptor kinases. Science  309: 290–293.  <ul class="find" data-citedArticleID="953964" data-doi="10.1126/science.1109710"><li><a href="http://dx.doi.org/10.1126/science.1109710" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stomatal+patterning+and+differentiation+by+synergistic+interactions+of+receptor+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stomatal+patterning+and+differentiation+by+synergistic+interactions+of+receptor+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000471-Jack1" id="pone.0000471-Jack1"></a>Jack T (2004) Molecular and Genetic Mechanisms of Floral Control. The Plant Cell  16: S1–S17.  <ul class="find" data-citedArticleID="953896" data-doi="10.1105/tpc.017038"><li><a href="http://dx.doi.org/10.1105/tpc.017038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+and+Genetic+Mechanisms+of+Floral+Control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+and+Genetic+Mechanisms+of+Floral+Control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000471-Ziegelhoffer1" id="pone.0000471-Ziegelhoffer1"></a>Ziegelhoffer EC, Medrano LJ, Meyerowitz EM (2000) Cloning of the Arabidopsis <em>WIGGUM</em> gene identifies a role for farnesylation in meristem development. Proc Natl Acad Sci U S A  97: 7633–7638.  <ul class="find" data-citedArticleID="953986" data-doi="10.1073/pnas.130189397"><li><a href="http://dx.doi.org/10.1073/pnas.130189397" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+the+Arabidopsis+WIGGUM+gene+identifies+a+role+for+farnesylation+in+meristem+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+the+Arabidopsis+WIGGUM+gene+identifies+a+role+for+farnesylation+in+meristem+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000471-Fletcher1" id="pone.0000471-Fletcher1"></a>Fletcher JC (2001) <em>The ULTRAPETALA</em> gene controls shoot and floral meristem size in Arabidopsis. Development  128: 1323–1333.  <ul class="find" data-citedArticleID="953870"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+ULTRAPETALA+gene+controls+shoot+and+floral+meristem+size+in+Arabidopsis.&amp;auth=&amp;atitle=The+ULTRAPETALA+gene+controls+shoot+and+floral+meristem+size+in+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ULTRAPETALA+gene+controls+shoot+and+floral+meristem+size+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ULTRAPETALA+gene+controls+shoot+and+floral+meristem+size+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000471-Running1" id="pone.0000471-Running1"></a>Running MP, Lavy M, Sternberg H, Galichet A, Gruissem W, et al.  (2004) Enlarged meristems and delayed growth in <em>plp</em> mutants result from lack of CaaX prenyltransferases. Proc Natl Acad Sci U S A  101: 7815–7820.  <ul class="find" data-citedArticleID="953954" data-doi="10.1073/pnas.0402385101"><li><a href="http://dx.doi.org/10.1073/pnas.0402385101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enlarged+meristems+and+delayed+growth+in+plp+mutants+result+from+lack+of+CaaX+prenyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enlarged+meristems+and+delayed+growth+in+plp+mutants+result+from+lack+of+CaaX+prenyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000471-Noh1" id="pone.0000471-Noh1"></a>Noh YS, Amasino RM (2003) <em>PIE1</em>, an ISWI family gene, is required for FLC activation and floral repression in Arabidopsis. Plant Cell  15: 1671–1682.  <ul class="find" data-citedArticleID="953936" data-doi="10.1105/tpc.012161"><li><a href="http://dx.doi.org/10.1105/tpc.012161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PIE1%2C+an+ISWI+family+gene%2C+is+required+for+FLC+activation+and+floral+repression+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PIE1%2C+an+ISWI+family+gene%2C+is+required+for+FLC+activation+and+floral+repression+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000471-Clark1" id="pone.0000471-Clark1"></a>Clark SE, Running MP, Meyerowitz EM (1993) <em>CLAVATA1</em>, a regulator of meristem and flower development in Arabidopsis. Development  119: 397–418.  <ul class="find" data-citedArticleID="953866"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CLAVATA1%2C+a+regulator+of+meristem+and+flower+development+in+Arabidopsis.&amp;auth=&amp;atitle=CLAVATA1%2C+a+regulator+of+meristem+and+flower+development+in+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLAVATA1%2C+a+regulator+of+meristem+and+flower+development+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLAVATA1%2C+a+regulator+of+meristem+and+flower+development+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000471-Kayes1" id="pone.0000471-Kayes1"></a>Kayes JM, Clark SE (1998) <em>CLAVATA2</em>, a regulator of meristem and organ development in Arabidopsis. Development  125: 3843–3851.  <ul class="find" data-citedArticleID="953904"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CLAVATA2%2C+a+regulator+of+meristem+and+organ+development+in+Arabidopsis.&amp;auth=&amp;atitle=CLAVATA2%2C+a+regulator+of+meristem+and+organ+development+in+Arabidopsis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLAVATA2%2C+a+regulator+of+meristem+and+organ+development+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLAVATA2%2C+a+regulator+of+meristem+and+organ+development+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000471-Clark2" id="pone.0000471-Clark2"></a>Clark SE, Running MP, Meyerowitz EM (1995) <em>CLAVATA3</em> is a specific regulator of shoot and floral meristem development affecting the same processes as <em>CLAVATA1</em>. Development  121: 2057–2067.  <ul class="find" data-citedArticleID="953868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=CLAVATA3+is+a+specific+regulator+of+shoot+and+floral+meristem+development+affecting+the+same+processes+as+CLAVATA1.&amp;auth=&amp;atitle=CLAVATA3+is+a+specific+regulator+of+shoot+and+floral+meristem+development+affecting+the+same+processes+as+CLAVATA1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLAVATA3+is+a+specific+regulator+of+shoot+and+floral+meristem+development+affecting+the+same+processes+as+CLAVATA1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLAVATA3+is+a+specific+regulator+of+shoot+and+floral+meristem+development+affecting+the+same+processes+as+CLAVATA1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000471-Palusa1" id="pone.0000471-Palusa1"></a>Palusa SG, Ali GS, Reddy AS (2007) Alternative splicing of pre-mRNAs of Arabidopsis serine/arginine-rich proteins: regulation by hormones and stresses. Plant J  49: 1091–1107.  <ul class="find" data-citedArticleID="953938" data-doi="10.1111/j.1365-313x.2006.03020.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.03020.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+splicing+of+pre-mRNAs+of+Arabidopsis+serine%2Farginine-rich+proteins%3A+regulation+by+hormones+and+stresses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+splicing+of+pre-mRNAs+of+Arabidopsis+serine%2Farginine-rich+proteins%3A+regulation+by+hormones+and+stresses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000471-Lopato2" id="pone.0000471-Lopato2"></a>Lopato S, Kalyna M, Dorner S, Kobayashi R, Krainer AR, et al.  (1999) atSRp30, one of two SF2/ASF-like proteins from Arabidopsis thaliana, regulates splicing of specific plant genes. Genes Dev  13: 987–1001.  <ul class="find" data-citedArticleID="953924" data-doi="10.1101/gad.13.8.987"><li><a href="http://dx.doi.org/10.1101/gad.13.8.987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=atSRp30%2C+one+of+two+SF2%2FASF-like+proteins+from+Arabidopsis+thaliana%2C+regulates+splicing+of+specific+plant+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22atSRp30%2C+one+of+two+SF2%2FASF-like+proteins+from+Arabidopsis+thaliana%2C+regulates+splicing+of+specific+plant+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000471-Lazar1" id="pone.0000471-Lazar1"></a>Lazar G, Goodman HM (2000) The Arabidopsis splicing factor SR1 is regulated by alternative splicing. Plant Mol Biol  42: 571–581.  <ul class="find" data-citedArticleID="953912"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+Arabidopsis+splicing+factor+SR1+is+regulated+by+alternative+splicing.&amp;auth=&amp;atitle=The+Arabidopsis+splicing+factor+SR1+is+regulated+by+alternative+splicing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Arabidopsis+splicing+factor+SR1+is+regulated+by+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Arabidopsis+splicing+factor+SR1+is+regulated+by+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000471-Kalyna2" id="pone.0000471-Kalyna2"></a>Kalyna M, Lopato S, Barta A (2003) Ectopic expression of atRSZ33 reveals its function in splicing and causes pleiotropic changes in development. Mol Biol Cell  14: 3565–3577.  <ul class="find" data-citedArticleID="953900" data-doi="10.1091/mbc.e03-02-0109"><li><a href="http://dx.doi.org/10.1091/mbc.e03-02-0109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ectopic+expression+of+atRSZ33+reveals+its+function+in+splicing+and+causes+pleiotropic+changes+in+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ectopic+expression+of+atRSZ33+reveals+its+function+in+splicing+and+causes+pleiotropic+changes+in+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000471-Thompson1" id="pone.0000471-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1997) The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="953974" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000471-Prasad1" id="pone.0000471-Prasad1"></a>Prasad J, Colwill K, Pawson T, Manley JL (1999) The protein kinase Clk/Sty directly modulates SR protein activity: both hyper-and hypophosphorylation inhibit splicing. Mol Cell Biol  19: 6991–7000.  <ul class="find" data-citedArticleID="953940"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+protein+kinase+Clk%2FSty+directly+modulates+SR+protein+activity%3A+both+hyper-and+hypophosphorylation+inhibit+splicing.&amp;auth=&amp;atitle=The+protein+kinase+Clk%2FSty+directly+modulates+SR+protein+activity%3A+both+hyper-and+hypophosphorylation+inhibit+splicing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+kinase+Clk%2FSty+directly+modulates+SR+protein+activity%3A+both+hyper-and+hypophosphorylation+inhibit+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+kinase+Clk%2FSty+directly+modulates+SR+protein+activity%3A+both+hyper-and+hypophosphorylation+inhibit+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000471-Li1" id="pone.0000471-Li1"></a>Li C, Wong WH (2001) Model-based analysis of oligonucleotide arrays: expression index computation and outlier detection. Proc Natl Acad Sci U S A  98: 31–36.  <ul class="find" data-citedArticleID="953918" data-doi="10.1073/pnas.98.1.31"><li><a href="http://dx.doi.org/10.1073/pnas.98.1.31" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model-based+analysis+of+oligonucleotide+arrays%3A+expression+index+computation+and+outlier+detection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model-based+analysis+of+oligonucleotide+arrays%3A+expression+index+computation+and+outlier+detection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000471-Benjamini1" id="pone.0000471-Benjamini1"></a>Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate—A practical and powerful approach to multiple testing. J R Stat Soc Ser B Methodol  57: 289–300.  <ul class="find" data-citedArticleID="953854"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Controlling+the+false+discovery+rate%26%23x2014%3BA+practical+and+powerful+approach+to+multiple+testing.&amp;auth=&amp;atitle=Controlling+the+false+discovery+rate%26%23x2014%3BA+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Controlling+the+false+discovery+rate%26%23x2014%3BA+practical+and+powerful+approach+to+multiple+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Controlling+the+false+discovery+rate%26%23x2014%3BA+practical+and+powerful+approach+to+multiple+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000471-Reddy4" id="pone.0000471-Reddy4"></a>Reddy VS, Day IS, Thomas T, Reddy AS (2004) KIC, a novel Ca2+ binding protein with one EF-hand motif, interacts with a microtubule motor protein and regulates trichome morphogenesis. Plant Cell  16: 185–200.  <ul class="find" data-citedArticleID="953950" data-doi="10.1105/tpc.016600"><li><a href="http://dx.doi.org/10.1105/tpc.016600" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=KIC%2C+a+novel+Ca2%2B+binding+protein+with+one+EF-hand+motif%2C+interacts+with+a+microtubule+motor+protein+and+regulates+trichome+morphogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22KIC%2C+a+novel+Ca2%2B+binding+protein+with+one+EF-hand+motif%2C+interacts+with+a+microtubule+motor+protein+and+regulates+trichome+morphogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000471-He2" id="pone.0000471-He2"></a>He Y, Amasino RM (2005) Role of chromatin modification in flowering-time control. Trends Plant Sci  10: 30–35.  <ul class="find" data-citedArticleID="953886" data-doi="10.1016/j.tplants.2004.11.003"><li><a href="http://dx.doi.org/10.1016/j.tplants.2004.11.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+chromatin+modification+in+flowering-time+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+chromatin+modification+in+flowering-time+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.XML" value="135017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.PDF" value="2215359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g001.PNG_L" value="1553269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g001.PNG_M" value="76340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g001.PNG_S" value="3172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g001.TIF" value="1966710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g001.PNG_I" value="31831"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g002.PNG_L" value="469228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g002.PNG_M" value="68355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g002.PNG_S" value="3817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g002.TIF" value="673910"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g002.PNG_I" value="27535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g003.PNG_L" value="807708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g003.PNG_M" value="98189"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g003.PNG_S" value="9651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g003.TIF" value="1077130"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g003.PNG_I" value="116479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g004.PNG_L" value="3143246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g004.PNG_M" value="437704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g004.PNG_S" value="12670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g004.TIF" value="4303250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g004.PNG_I" value="186208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g005.PNG_L" value="2248922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g005.PNG_M" value="183194"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g005.PNG_S" value="10049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g005.TIF" value="2976166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g005.PNG_I" value="127074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g006.PNG_L" value="2705177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g006.PNG_M" value="279275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g006.PNG_S" value="9207"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g006.TIF" value="3273334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g006.PNG_I" value="115256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g007.PNG_L" value="3725231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g007.PNG_M" value="572482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g007.PNG_S" value="12998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g007.TIF" value="5192594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g007.PNG_I" value="180709"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g008.PNG_L" value="1117348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g008.PNG_M" value="90478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g008.PNG_S" value="13824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g008.TIF" value="1493630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.g008.PNG_I" value="165022"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.s001.TIF" value="1588604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.s002.TIF" value="1044322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.s003.DOC" value="109056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.s004.XLS" value="3253248"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000471.s005.DOC" value="75776"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000471&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000471" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000471&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000471&volume=&issue=&title=Regulation of Plant Developmental Processes by a Novel Splicing Factor&author_name=Gul%20Shad%20Ali%2C%20Saiprasad%20G.%20Palusa%2C%20Maxim%20Golovkin%2C%20Jayendra%20Prasad%2C%20James%20L.%20Manley%2C%20Anireddy%20S.N.%20Reddy&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000471" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471&amp;t=Regulation%20of%20Plant%20Developmental%20Processes%20by%20a%20Novel%20Splicing%20Factor" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471&title=Regulation%20of%20Plant%20Developmental%20Processes%20by%20a%20Novel%20Splicing%20Factor&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471&amp;title=Regulation%20of%20Plant%20Developmental%20Processes%20by%20a%20Novel%20Splicing%20Factor" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000471&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Regulation of Plant Developmental Processes by a Novel Splicing Factor';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000471';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Regulation%20of%20Plant%20Developmental%20Processes%20by%20a%20Novel%20Splicing%20Factor http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000471" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000471" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="23780"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flowering+plants%22" title="Search for articles in the subject area:'Flowering plants'"><div class="flagText">Flowering plants</div></a>
              <div data-categoryid="43015" data-articleid="23780"
                   data-categoryname="Flowering plants"
                   class="flagImage" title="Flag 'Flowering plants' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flowers%22" title="Search for articles in the subject area:'Flowers'"><div class="flagText">Flowers</div></a>
              <div data-categoryid="17311" data-articleid="23780"
                   data-categoryname="Flowers"
                   class="flagImage" title="Flag 'Flowers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leaves%22" title="Search for articles in the subject area:'Leaves'"><div class="flagText">Leaves</div></a>
              <div data-categoryid="17987" data-articleid="23780"
                   data-categoryname="Leaves"
                   class="flagImage" title="Flag 'Leaves' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Petals%22" title="Search for articles in the subject area:'Petals'"><div class="flagText">Petals</div></a>
              <div data-categoryid="23223" data-articleid="23780"
                   data-categoryname="Petals"
                   class="flagImage" title="Flag 'Petals' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="23780"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Reverse+transcriptase-polymerase+chain+reaction%22" title="Search for articles in the subject area:'Reverse transcriptase-polymerase chain reaction'"><div class="flagText">Reverse transcriptase-polymerase chain reaction</div></a>
              <div data-categoryid="47921" data-articleid="23780"
                   data-categoryname="Reverse transcriptase-polymerase chain reaction"
                   class="flagImage" title="Flag 'Reverse transcriptase-polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Stamens%22" title="Search for articles in the subject area:'Stamens'"><div class="flagText">Stamens</div></a>
              <div data-categoryid="18057" data-articleid="23780"
                   data-categoryname="Stamens"
                   class="flagImage" title="Flag 'Stamens' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2169'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2573'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3699&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
