

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000847"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000847" />

    <meta name="citation_title" content="Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure"/>
    <meta itemprop="name" content="Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure"/>

      <meta name="citation_author" content="Gabriela Moura"/>
            <meta name="citation_author_institution" content="Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Miguel Pinheiro"/>
            <meta name="citation_author_institution" content="Institute of Electronics and Telematics Engineering, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Joel Arrais"/>
            <meta name="citation_author_institution" content="Institute of Electronics and Telematics Engineering, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Ana Cristina Gomes"/>
            <meta name="citation_author_institution" content="Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Laura Carreto"/>
            <meta name="citation_author_institution" content="Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Adelaide Freitas"/>
            <meta name="citation_author_institution" content="Department of Mathematics, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="José L. Oliveira"/>
            <meta name="citation_author_institution" content="Institute of Electronics and Telematics Engineering, University of Aveiro, Aveiro, Portugal"/>
      <meta name="citation_author" content="Manuel A. S. Santos"/>
            <meta name="citation_author_institution" content="Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal"/>

    <meta name="citation_date" content="2007/9/5"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000847.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e847"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=After the duplication: gene loss and adaptation in Saccharomyces genomes.; citation_author=PF Cliften; citation_author=RS Fulton; citation_author=RK Wilson; citation_author=M Johnston; citation_journal_title=Genetics; citation_volume=172; citation_number=1; citation_pages=863-872; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genomic deletions and precise removal of transposable elements mediated by short identical DNA segments in primates.; citation_author=LN van de Lagemaat; citation_author=L Gagnier; citation_author=P Medstrand; citation_author=DL Mager; citation_journal_title=Genome Res; citation_volume=15; citation_number=2; citation_pages=1243-1249; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Polymorphisms associated with the DAZ genes on the human Y chromosome.; citation_author=YW Lin; citation_author=DA Thi; citation_author=PL Kuo; citation_author=CC Hsu; citation_author=BD Huang; citation_journal_title=Genomics; citation_volume=86; citation_number=3; citation_pages=431-438; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Codon usage between genomes is constrained by genome-wide mutational processes.; citation_author=SL Chen; citation_author=W Lee; citation_author=AK Hottes; citation_author=L Shapiro; citation_author=HH McAdams; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=4; citation_pages=3480-3485; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gardening the genome: DNA methylation in Arabidopsis thaliana.; citation_author=SW Chan; citation_author=IR Henderson; citation_author=SE Jacobsen; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=5; citation_pages=351-360; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=DNA methylation and human disease.; citation_author=KD Robertson; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=6; citation_pages=597-610; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Mammalian mutation pressure, synonymous codon choice, and mRNA degradation.; citation_author=J Duan; citation_author=MA Antezana; citation_journal_title=J Mol Evol; citation_volume=57; citation_number=7; citation_pages=694-701; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Codon bias in Escherichia coli: the influence of codon context on mutation and selection.; citation_author=OG Berg; citation_author=PJ Silva; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=8; citation_pages=1397-1404; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy.; citation_author=H Akashi; citation_journal_title=Genetics; citation_volume=136; citation_number=9; citation_pages=927-935; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Rates of aminoacyl-tRNA selection at 29 sense codons in vivo.; citation_author=JF Curran; citation_author=M Yarus; citation_journal_title=J Mol Biol; citation_volume=209; citation_number=10; citation_pages=65-77; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Selection at the wobble position of codons read by the same tRNA in Saccharomyces cerevisiae.; citation_author=R Percudani; citation_author=S Ottonello; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=11; citation_pages=1752-1762; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Codon pairs in the genome of Escherichia coli.; citation_author=S Boycheva; citation_author=G Chkodrov; citation_author=I Ivanov; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=12; citation_pages=987-998; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural insights into translational fidelity.; citation_author=JM Ogle; citation_author=V Ramakrishnan; citation_journal_title=Annu Rev Biochem; citation_volume=74; citation_number=13; citation_pages=129-177; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Codon pair utilization biases influence translational elongation step times.; citation_author=B Irwin; citation_author=JD Heck; citation_author=GW Hatfield; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=14; citation_pages=22801-22806; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Computational identification of putative programmed translational frameshift sites.; citation_author=AA Shah; citation_author=MC Giddings; citation_author=RF Gesteland; citation_author=JF Atkins; citation_author=IP Ivanov; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=15; citation_pages=1046-1053; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The accuracy of mRNA-tRNA recognition.; citation_author=RH Buckingham; citation_author=H Grosjean; citation_number=16; citation_pages=83-126; citation_date=1986; citation_publisher=Chapman and Hall; " />
      <meta name="citation_reference" content="citation_title=Transfer RNA gene redundancy and translational selection in Saccharomyces cerevisiae.; citation_author=R Percudani; citation_author=A Pavesi; citation_author=S Ottonello; citation_journal_title=J Mol Biol; citation_volume=268; citation_number=17; citation_pages=322-330; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Selection of aminoacyl-tRNAs at sense codons: the size of the tRNA variable loop determines whether the immediate 3′ nucleotide to the codon has a context effect.; citation_author=JF Curran; citation_author=ES Poole; citation_author=WP Tate; citation_author=BL Gross; citation_journal_title=Nucleic Acids Res; citation_volume=23; citation_number=18; citation_pages=4104-4108; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Regularities of context-dependent codon bias in eukaryotic genes.; citation_author=A Fedorov; citation_author=S Saxonov; citation_author=W Gilbert; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=19; citation_pages=1192-1197; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparative context analysis of codon pairs on an ORFeome scale.; citation_author=G Moura; citation_author=M Pinheiro; citation_author=R Silva; citation_author=I Miranda; citation_author=V Afreixo; citation_journal_title=Genome Biol; citation_volume=6; citation_number=20; citation_pages=R28; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hidden infidelities of the translational stop signal.; citation_author=WP Tate; citation_author=ES Poole; citation_author=SA Mannering; citation_journal_title=Prog Nucleic Acid Res Mol Biol; citation_volume=52; citation_number=21; citation_pages=293-335; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Codon context effects in missense suppression.; citation_author=EJ Murgola; citation_author=FT Pagel; citation_author=KA Hijazi; citation_journal_title=J Mol Biol; citation_volume=175; citation_number=22; citation_pages=19-27; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=The major 5′ determinant in stop codon read-through involves two adjacent adenines.; citation_author=S Tork; citation_author=I Hatin; citation_author=JP Rousset; citation_author=C Fabret; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=23; citation_pages=415-421; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fidelity of aminoacyl-tRNA selection on the ribosome: kinetic and structural mechanisms.; citation_author=MV Rodnina; citation_author=W Wintermeyer; citation_journal_title=Annu Rev Biochem; citation_volume=70; citation_number=24; citation_pages=415-435; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The frequency of translational misreading errors in E. coli is largely determined by tRNA competition.; citation_author=EB Kramer; citation_author=PJ Farabaugh; citation_journal_title=RNA; citation_volume=13; citation_number=25; citation_pages=87-96; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Towards a genetic dissection of the basis of triplet decoding, and its natural subversion: programmed reading frame shifts and hops.; citation_author=JF Atkins; citation_author=RB Weiss; citation_author=S Thompson; citation_author=RF Gesteland; citation_journal_title=Annu Rev Genet; citation_volume=25; citation_number=26; citation_pages=201-228; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The accuracy of codon recognition by polypeptide release factors.; citation_author=DV Freistroffer; citation_author=M Kwiatkowski; citation_author=RH Buckingham; citation_author=M Ehrenberg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=27; citation_pages=2046-2051; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Quantitating protein synthesis, degradation, and endogenous antigen processing.; citation_author=MF Princiotta; citation_author=D Finzi; citation_author=SB Qian; citation_author=J Gibbs; citation_author=S Schuchmann; citation_journal_title=Immunity; citation_volume=18; citation_number=28; citation_pages=343-354; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Statistical, computational and visualization methodologies to unveil gene primary structure features.; citation_author=M Pinheiro; citation_author=V Afreixo; citation_author=G Moura; citation_author=A Freitas; citation_author=MA Santos; citation_journal_title=Methods Inf Med; citation_volume=45; citation_number=29; citation_pages=163-168; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA.; citation_author=A Campbell; citation_author=J Mrazek; citation_author=S Karlin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=30; citation_pages=9184-9189; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Differences in dinucleotide frequencies of human, yeast, and Escherichia coli genes.; citation_author=H Nakashima; citation_author=K Nishikawa; citation_author=T Ooi; citation_journal_title=DNA Res; citation_volume=4; citation_number=31; citation_pages=185-192; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Detection of genes with atypical nucleotide sequence in microbial genomes.; citation_author=SD Hooper; citation_author=OG Berg; citation_journal_title=J Mol Evol; citation_volume=54; citation_number=32; citation_pages=365-375; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gradients in nucleotide and codon usage along Escherichia coli genes.; citation_author=SD Hooper; citation_author=OG Berg; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=33; citation_pages=3517-3523; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ribosome structure: revisiting the connection between translational accuracy and unconventional decoding.; citation_author=G Stahl; citation_author=GP McCarty; citation_author=PJ Farabaugh; citation_journal_title=Trends Biochem Sci; citation_volume=27; citation_number=34; citation_pages=178-183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Codon choice and gene expression: synonymous codons differ in translational accuracy.; citation_author=DB Dix; citation_author=RC Thompson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=35; citation_pages=6888-6892; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Nonrandom utilization of codon pairs in Escherichia coli.; citation_author=GA Gutman; citation_author=GW Hatfield; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=36; citation_pages=3699-3703; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=tRNA properties help shape codon pair preferences in open reading frames.; citation_author=JR Buchan; citation_author=LS Aucott; citation_author=I Stansfield; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=37; citation_pages=1015-1027; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Universal replication biases in bacteria.; citation_author=EP Rocha; citation_author=A Danchin; citation_author=A Viari; citation_journal_title=Mol Microbiol; citation_volume=32; citation_number=38; citation_pages=11-16; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Codon catalog usage and the genome hypothesis.; citation_author=R Grantham; citation_author=C Gautier; citation_author=M Gouy; citation_author=R Mercier; citation_author=A Pave; citation_journal_title=Nucleic Acids Res; citation_volume=8; citation_number=39; citation_pages=r49-r62; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Codon context.; citation_author=RH Buckingham; citation_journal_title=Experientia; citation_volume=46; citation_number=40; citation_pages=1126-1133; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Evolutionary lability of context-dependent codon bias in bacteria.; citation_author=GAT McVean; citation_author=GDD Hurst; citation_journal_title=J Mol Evol; citation_volume=50; citation_number=41; citation_pages=264-275; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Thermophilic bacteria strictly obey Szybalski's transcription direction rule and politely purine-load RNAs with both adenine and guanine.; citation_author=PJ Lao; citation_author=DR Forsdyke; citation_journal_title=Genome Res; citation_volume=10; citation_number=42; citation_pages=228-236; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Understanding the adaptation of Halobacterium species NRC-1 to its extreme environment through computational analysis of its genome sequence.; citation_author=SP Kennedy; citation_author=WV Ng; citation_author=SL Salzberg; citation_author=L Hood; citation_author=S DasSarma; citation_journal_title=Genome Res; citation_volume=11; citation_number=43; citation_pages=1641-1650; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Amino acid composition of genomes, lifestyles of organisms, and evolutionary trends: a global picture with correspondence analysis.; citation_author=F Tekaia; citation_author=E Yeramian; citation_author=B Dujon; citation_journal_title=Gene; citation_volume=297; citation_number=44; citation_pages=51-60; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolution of the genome and the genetic code: selection at the dinucleotide level by methylation and polyribonucleotide cleavage.; citation_author=E Beutler; citation_author=T Gelbart; citation_author=JH Han; citation_author=JA Koziol; citation_author=B Beutler; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=45; citation_pages=192-196; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Genes from nine genomes are separated into their organisms in the dinucleotide composition space.; citation_author=H Nakashima; citation_author=M Ota; citation_author=K Nishikawa; citation_author=T Ooi; citation_journal_title=DNA Res; citation_volume=5; citation_number=46; citation_pages=251-259; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features.; citation_author=C Marck; citation_author=H Grosjean; citation_journal_title=RNA; citation_volume=8; citation_number=47; citation_pages=1189-1232; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Codon–anticodon pairing: the wobble hypothesis.; citation_author=FH Crick; citation_journal_title=J Mol Biol; citation_volume=19; citation_number=48; citation_pages=548-555; citation_date=1966; " />
      <meta name="citation_reference" content="citation_title=A genomic basis for the evolution of vertebrate transcription factors containing amino Acid runs.; citation_author=S Caburet; citation_author=D Vaiman; citation_author=RA Veitia; citation_journal_title=Genetics; citation_volume=167; citation_number=49; citation_pages=1813-1820; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tandem repeats in protein coding regions of primate genes.; citation_author=B Borstnik; citation_author=D Pumpernik; citation_journal_title=Genome Res; citation_volume=12; citation_number=50; citation_pages=909-915; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Over-representation of repeats in stress response genes: a strategy to increase versatility under stressful conditions?; citation_author=EP Rocha; citation_author=I Matic; citation_author=F Taddei; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=51; citation_pages=1886-1894; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Analysis of residuals in cross-classified tables.; citation_author=S Haberman; citation_journal_title=Biometrics; citation_volume=29; citation_number=52; citation_pages=205-220; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Analyzing categorical data.; citation_author=JS Simenoff; citation_number=53; citation_date=2003; citation_publisher=Springer-Verlag; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure"/>
    <meta name="twitter:description" content="BackgroundCodon usage and codon-pair context are important gene primary structure features that influence mRNA decoding fidelity. In order to identify general rules that shape codon-pair context and minimize mRNA decoding error, we have carried out a large scale comparative codon-pair context analysis of 119 fully sequenced genomes.Methodologies/Principal FindingsWe have developed mathematical and software tools for large scale comparative codon-pair context analysis. These methodologies unveiled general and species specific codon-pair context rules that govern evolution of mRNAs in the 3 domains of life. We show that evolution of bacterial and archeal mRNA primary structure is mainly dependent on constraints imposed by the translational machinery, while in eukaryotes DNA methylation and tri-nucleotide repeats impose strong biases on codon-pair context.ConclusionsThe data highlight fundamental differences between prokaryotic and eukaryotic mRNA decoding rules, which are partially independent of codon usage."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000847.g006"/>

  <meta property="og:title" content="Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9209'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9657'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3220&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000847">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000847" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000847&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gabriela Moura, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Miguel Pinheiro, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Electronics and Telematics Engineering, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joel Arrais, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Electronics and Telematics Engineering, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ana Cristina Gomes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Laura Carreto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adelaide Freitas, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Mathematics, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            José L. Oliveira, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Electronics and Telematics Engineering, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Manuel A. S. Santos
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:msantos@bio.ua.pt">msantos@bio.ua.pt</a></p>

                <p>Affiliation:
                  Department of Biology, Center for Environmental and Marine Studies, University of Aveiro, Aveiro, Portugal
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 05, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000847</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000847-g001" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-g002" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-t001" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-t002" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-g003" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-g004" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-g005" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000847-g006" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000847">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000847" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Codon usage and codon-pair context are important gene primary structure features that influence mRNA decoding fidelity. In order to identify general rules that shape codon-pair context and minimize mRNA decoding error, we have carried out a large scale comparative codon-pair context analysis of 119 fully sequenced genomes.</p>


<h3>Methodologies/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have developed mathematical and software tools for large scale comparative codon-pair context analysis. These methodologies unveiled general and species specific codon-pair context rules that govern evolution of mRNAs in the 3 domains of life. We show that evolution of bacterial and archeal mRNA primary structure is mainly dependent on constraints imposed by the translational machinery, while in eukaryotes DNA methylation and tri-nucleotide repeats impose strong biases on codon-pair context.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The data highlight fundamental differences between prokaryotic and eukaryotic mRNA decoding rules, which are partially independent of codon usage.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Moura G, Pinheiro M, Arrais J, Gomes AC, Carreto L, et al.  (2007) Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure. PLoS ONE 2(9):
          e847.
            doi:10.1371/journal.pone.0000847</p><p><strong>Academic Editor: </strong>Alan Christoffels, Temasek Life Sciences Laboratory, Singapore</p><p><strong>Received:</strong> May 25, 2007; <strong>Accepted:</strong> July 31, 2007; <strong>Published:</strong> September 5, 2007</p><p><strong>Copyright:</strong> © 2007 Moura et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by FEDER/FCT projects POCTI/BME/39030; SAU-MMO/55476; BIA-PRO/55472; BIA-MIC/55466; PTDC/MAT/72974/2006 and Human Frontier Science Programme project RGP45/2005.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>A myriad of evolutionary forces shape the primary structure of coding components (ORFs) of genomes, herein called ORFeomes. These include genome and gene duplication, chromosome rearrangements, DNA recombination, deletions and insertions, transposition of mobile elements, single nucleotide polymorphisms, nucleotide repeats and biased G+C pressure <a href="#pone.0000847-Cliften1">[1]</a>–<a href="#pone.0000847-Chen1">[4]</a>. Apart from these DNA replication derived phenomena others arising from DNA transcription, mRNA stability and translation <a href="#pone.0000847-Chan1">[5]</a>–<a href="#pone.0000847-Duan1">[7]</a> are also likely to fine tune ORFeomes' primary structure, but their significance is not yet fully understood.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>At the mRNA translation level, synonymous codon usage and codon-pair context (representing the pair of codons located in the A and P- ribosome sites) are expected to be under selective pressure since they affect mRNA decoding speed and accuracy <a href="#pone.0000847-Berg1">[8]</a>–<a href="#pone.0000847-Shah1">[15]</a>. Synonymous codon usage biases are explained mainly by G+C content and only secondarily by constraints imposed by mRNA translation variables <a href="#pone.0000847-Chen1">[4]</a>, namely tRNA abundance, efficiency of tRNA charging, mRNA decoding efficiency (speed plus accuracy), mRNA stability and structure, gene expression, and amino acid composition <a href="#pone.0000847-Duan1">[7]</a>, <a href="#pone.0000847-Ogle1">[13]</a>, <a href="#pone.0000847-Buckingham1">[16]</a>–<a href="#pone.0000847-Curran2">[18]</a>. The nucleotides surrounding a codon also influence synonymous codon usage, with the strongest influence arising from the interplay between the last nucleotide of a codon and the first nucleotide of the neighbor codon (N<sub>1</sub>N<sub>2</sub><strong>N<sub>3</sub></strong> <strong>N<sub>1</sub></strong>N<sub>2</sub>N<sub>3</sub>), the so called N<sub>3</sub>-N<sub>1</sub> context <a href="#pone.0000847-Duan1">[7]</a>, <a href="#pone.0000847-Fedorov1">[19]</a>, <a href="#pone.0000847-Moura1">[20]</a>. Conversely to codon usage, the forces that modulate codon-pair context, with the exception of the context of initiation and termination codons <a href="#pone.0000847-Buckingham1">[16]</a>, <a href="#pone.0000847-Tate1">[21]</a>, are still poorly understood. The few studies carried out to date show, however, that codon-pair context has a direct impact on missense, nonsense and frameshifting errors <a href="#pone.0000847-Shah1">[15]</a>, <a href="#pone.0000847-Murgola1">[22]</a>, <a href="#pone.0000847-Tork1">[23]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In <em>E. coli</em>, missense error <em>in vivo</em>, under standard growth conditions, is in the order of 10<sup>−3</sup> to 10<sup>−4</sup> per codon decoded <a href="#pone.0000847-Rodnina1">[24]</a>, <a href="#pone.0000847-Kramer1">[25]</a>. Frameshifting and stop codon readthrough errors happen at levels of 3×10<sup>−4</sup> to 10<sup>−5</sup> and of 10<sup>−3</sup> to 10<sup>−6</sup>, respectively <a href="#pone.0000847-Atkins1">[26]</a>, <a href="#pone.0000847-Freistroffer1">[27]</a>. Under stress, namely amino acid starvation, these basal error rates increase significantly <a href="#pone.0000847-Buckingham1">[16]</a>, indicating that decoding error in nature may be significantly higher than in optimal laboratory conditions. Furthermore, 30% of the newly synthesized proteins in HeLa, lymph node, L-K<sup>b</sup> and dendritic cells are defective ribosomal products (DRiPs) that arise from missense, frameshifting and ribosome drop off at mRNA pausing sites <a href="#pone.0000847-Princiotta1">[28]</a>. Since protein synthesis utilizes 45% of the cell ATP, 30% DRiP rate represents 11% of wastage of total cellular energy <a href="#pone.0000847-Princiotta1">[28]</a>. Whether this is a common trend in all type of cells is unknown, however, peptides resulting from proteasome degradation of DRiPs are a major source of peptides for MHC class I molecules, highlighting an unanticipated role of mistranslation in immune cells <a href="#pone.0000847-Princiotta1">[28]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>It is not yet clear whether the ribosome drops off randomly or preferentially at specific mRNA drop off hot spots. In other words, it is important to elucidate whether average decoding error (10<sup>−4</sup> to 10<sup>−5</sup>) is evenly distributed along mRNAs (average error) or whether it fluctuates along the mRNA? If so, how can decoding error hot spots be identified? In order to obtain insight into these questions and identify mRNA primary structural features that influence mRNA decoding error, we have developed a software package, statistical and graphical tools to study codon-pairs corresponding to ribosomal A- and P-site codons, using genome wide approaches (ANACONDA vs 1.0) <a href="#pone.0000847-Moura1">[20]</a>, <a href="#pone.0000847-Pinheiro1">[29]</a>. ANACONDA 1.0 already allowed us to demonstrate that codon-pair context is weakly modulated by G+C pressure <a href="#pone.0000847-Moura1">[20]</a>. In the present study, we have significantly improved ANACONDA (creating its version 2.0) and used it to carry out large scale comparative codon-pair context analysis using complete ORFeome sequences of 81 Eubacteria, 18 Archaea and 20 Eukaryota. The data show that i) codon-pair context is species specific, ii) there are general rules governing its evolution in the three domains of life and iii) in eubacteria and archeae codon-pair context is mainly determined by constraints imposed by the translational machinery, while, iv) in eukaryotes the emergence of DNA methylation and tri-nucleotide repeats influenced codon-pair context. The data suggests the existence of fundamental differences between prokaryotic and eukaryotic mRNA decoding rules and shows that codon-pair context is partially independent of codon usage.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>New tools for large scale comparative codon-pair context analysis</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The ANACONDA 1.0 algorithm developed previously <a href="#pone.0000847-Moura1">[20]</a>, <a href="#pone.0000847-Pinheiro1">[29]</a> simulates the ribosome during decoding by reading Open Reading Frames (ORFs) sequences, starting at the AUG initiation codon and moving the reading window three nucleotides at a time (<a href="#pone-0000847-g001">Figure 1A</a>). While doing this, it memorizes all codon-pairs, which represent A- and P-site codons during mRNA decoding. It then builds a codon-pair contingency table (<a href="#pone-0000847-g001">Figure 1B</a>) that renders itself to statistical analysis and permits determination of the codon-pair context bias <a href="#pone.0000847-Moura1">[20]</a>. The existence of association between codon-pairs is determined through the chi-square (χ<sup>2</sup>) test of independence and preferred and rejected pairs of codons are identified through the analysis of adjusted residuals for contingency tables. These rejected and preferred pairs of codons are then displayed in a 61x64 green (preferred) and red (rejected) color coded map that generates a global view of the codon-pair context data for any ORFeome (<a href="#pone-0000847-g001">Figure 1C</a>). ANACONDA 1.0 also clusters the data according to the context preferences and rejections (residuals values) and builds Differential Display Maps (DDM), which represent codon-pair context differences between two different ORFeomes (<a href="#pone-0000847-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000847-g001"><div class="img"><a name="pone-0000847-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Flowchart of the codon-pair context analysis performed by ANACONDA.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A) The software selects valid ORFs from the total set available for each species (ORFeome) and counts all combinations of two consecutive codons (codon-pair context) that are present in the sequences. B) The observed values are incorporated into a contingency table in which the lines correspond to the 5′ codon (ribosome P-site) and the columns to the 3′ codon (ribosome A-site) of each pair. C) The contingency table of observed values is then compared to another table in which the values expected under independence are calculated. The cell corresponding to each pair of codons was colored in green for preferred contexts or red for rejected ones. This produces a color-coded map for the 61×64 two-codon contexts of one ORFeome. D) To aid simultaneous comparison of a large set of ORFeomes the 61×64 map is automatically converted into one single column with 3904 lines, one for each pair of codons. E) Finally, the columns that illustrate the two-codon context bias of each individual ORFeome are placed side by side, yielding a large-scale codon context comparison map. Both maps for codon context bias, i.e. the 61×64 map for a single species and the large-scale codon context comparison map can be rearranged using clustering methodologies that highlight similar codon-pair context patterns. For detailed description of statistics and software, see <a href="#s4">Methods</a> or <a href="#pone.0000847-Moura1">[20]</a>, <a href="#pone.0000847-Pinheiro1">[29]</a>.</p>
<span>doi:10.1371/journal.pone.0000847.g001</span></div><div class="figure" id="pone-0000847-g002"><div class="img"><a name="pone-0000847-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Codon-pair context is species specific.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>A) Individual codon-pair context maps built for various genomes followed phylogeny indicating that codon-pair context is species specific. For instance, the human ORFeome map is more similar to that of chimpanzee (<em>Pan troglodytes</em>) than to the mouse (<em>Mus musculus</em>) map. B) This result was confirmed using differential display maps (DDM) that subtract two codon-pair context maps. For example, <em>H. sapiens</em>–<em>M. musculus</em> (<em>H.s</em> vs <em>M.m</em>); <em>H. sapiens</em>–<em>P. troglodytes</em> (<em>H.s</em> vs <em>P.t</em>). In these differential display maps major codon-pair context differences (above 15) are shown in light blue and darker maps correspond to species with more similar codon-pair context biases. In the present example, the maps of <em>H.s</em> vs <em>M.m</em> and <em>H.s</em> vs <em>P.t</em> have 6% and 1% of blue cells, respectively. C and D) The same phylogenetical relationship could be detected for bacterial ORFeomes, as exemplified for <em>Escherichia coli</em>, <em>Bacillus cereus</em> and <em>Salmonella typhi</em>. The DDM built with these species have 55% (<em>E.c</em> vs <em>B.c</em>) and 20% (<em>E.c</em> vs <em>S.t</em>) of blue cells. E) Finally, the phylogenetical relationship was maintained when the above species were clustered according to the similarities of the codon-pair context maps. The yeasts <em>Saccharomyces cerevisiae</em> and <em>Schizosaccharomyces pombe</em> were added to include an intermediate group of lower eukaryotes in the tree. Adjusted residuals are colored in the maps according to the color scale shown, so that green cells correspond to preferred and red cells to rejected contexts.</p>
<span>doi:10.1371/journal.pone.0000847.g002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In an attempt to identify putative general rules that govern codon-pair context, we have carried out large scale codon-pair comparisons, using ANACONDA version 2.0. For this, new algorithms and tools were developed to convert the 61x64 codon-pair context colour-coded maps into a single colour-coded column containing 3904 lines, representing all possible combinations of pairs of the 64 codons (<a href="#pone-0000847-g001">Figure 1D</a>). ANACONDA 2.0 compared these colour-coded columns, clustered the data and highlighted groups of codons that had similar pair preference and rejection patterns (<a href="#pone-0000847-g001">Figures 1E</a>–<a href="#pone-0000847-g002"></a><a href="#pone-0000847-g003">3</a>). Since the size of ORFeomes varied significantly between bacteria and eukaryotes, ANACONDA 2.0 normalized the data using the biggest ORFeome as a reference data set (<a href="#pone.0000847.s001">Figure S1</a>). This permitted carrying out direct comparisons of large and small ORFeomes and allowed the study of codon-pair context preferences (positive residual value; green color in the map) and rejections (negative residual values; red color in the map) of 119 ORFeomes of Eubacteria, Archaea and Eukarya, including the human and chimpanzee ORFeomes (<a href="#pone.0000847.s001">Figure S2A</a>–<a href="#pone.0000847.s003">C</a>).</p>


<h4>Codon-pair context preferences are species specific</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Codon-pair context maps showed remarkable diversity from bacteria to high eukaryotes (<a href="#pone-0000847-g002">Figure 2</a>; <a href="#pone.0000847.s006">Figure S3A</a>–<a href="#pone.0000847.s014">J</a>). For example, codon-pair context preferences of the human (<em>Homo sapiens</em> or <em>H.s</em>) and mouse (<em>Mus musculus</em> or <em>M.m</em>) ORFeomes showed several differences, which were unveiled by direct comparison of ORFeomes and construction of Differential Display Maps (DDM) (<a href="#pone-0000847-g002">Figure 2A,B</a>), as described in our previous study <a href="#pone.0000847-Moura1">[20]</a>. Conversely, the codon-pair context maps for the chimpanzee (<em>Pan troglodytes</em> or <em>P.t</em>) and human ORFeomes were remarkably similar (<a href="#pone-0000847-g002">Figure 2B</a>), which was in agreement with the high homogeneity found for codon-pair distributions of both ORFeomes (data not shown). The same trend was found in bacteria. Indeed, the <em>Escherichia coli</em> (<em>E.c</em>) ORFeome codon-pair context map was more similar to that of <em>Salmonella typhi</em> (<em>S.t</em>) than to <em>Bacillus cereus</em> (<em>B.c</em> in <a href="#pone-0000847-g002">Figure 2C,D</a>). Clustering of the codon-pair context maps showed that codon-pair context follows rRNA phylogeny (<a href="#pone-0000847-g002">Figure 2E</a>), highlighting the possibility of using codon-pair context maps as species specific fingerprints. Furthermore, the overall correlation between the 3 domains of life was lower than that calculated within each domain, as the Spearman's correlations of the ranks (<a href="#pone.0000847.s018">Table S1</a>) showed low correlation coefficients between species of different domains of life, i.e. 0,452 for Eukarya vs Archaea; 0,450 for Eukarya vs Eubacteria; and 0,500 for Archaea vs Eubacteria. While correlation coefficients calculated between species of the same domain were high, i.e. 0,988 among Eukarya (between <em>H. sapiens</em> and <em>P. troglodytes</em>); 0,823 among Archaea (between <em>P. abyssi</em> and <em>P. horikoshii</em>); and 0,959 among Eubacteria (between <em>E. coli</em> and <em>S. flexneri</em>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The distribution of residual values over the entire set of ORFeomes showed that the 3 domains of life have significantly different codon-pair preferences (<a href="#pone-0000847-t001">Tables 1</a>, <a href="#pone-0000847-t002">2</a>). For example, codon-pair contexts with highest and lowest adjusted residual values showed no common codon-pairs in the 3 domains of life, suggesting fundamental differences between eukarya, eubacteria and archeae in codon-pair rules and in the evolutionary forces that shape ORFeomes primary structure. Interestingly, 9 out of the 10 codon-pair contexts with highest residual values (best codon-pairs) of all eukaryotic ORFeomes were pairs formed by identical codons (codon repeats) (<a href="#pone-0000847-t001">Table 1</a>). The same trend was also detected when the most frequently preferred codon-pair contexts for each domain were compared (<a href="#pone-0000847-t002">Table 2</a>). With this approach, common codon-pair contexts were identified for the 3 domains of life. For example, AAU-CCA and GGC-UGU had positive residuals in Eubacteria and Archaea. In Eukarya and Archaea ACU-AAG had negative residuals and AGA-AGA had positive residuals in Eubacteria and Eukarya. This suggested that, despite the species specificity of codon-pair context maps, at least some of the evolutionary constraints that shaped codon-pair context are conserved across species in the three domains of life.</p>
<div class="figure" id="pone-0000847-t001"><div class="img"><a name="pone-0000847-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>The most biased codon-pair contexts.</span></strong></p><span>doi:10.1371/journal.pone.0000847.t001</span></div><div class="figure" id="pone-0000847-t002"><div class="img"><a name="pone-0000847-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>General codon-pair contexts.</span></strong></p><span>doi:10.1371/journal.pone.0000847.t002</span></div>

<h4>Context preferences exist in coding and non-coding sequences</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>A large-scale codon-pair context comparison was carried out to visualize general context patterns, using clustering tools (<a href="#pone-0000847-g003">Figure 3A</a>). Interestingly, a red region, corresponding to negative residual values (rejected context), appeared across the 119 ORFeomes studied (blue box in <a href="#pone-0000847-g003">Figure 3A</a>). These rejected codon-pairs were of the general type NNU<sub>3</sub>-A<sub>1</sub>NN, where N represents any base. Other general patterns in the map represented either preferred codon-pairs in Archaea and Eukarya (<a href="#pone-0000847-g003">Figure 3A</a>; region-Y), rejected codon-pairs in Archaea and Eukarya (<a href="#pone-0000847-g003">Figure 3A</a>; region-Z) or strongly rejected codon-pairs in Eubacteria (<a href="#pone-0000847-g003">Figure 3A</a>; region-X).</p>
<div class="figure" id="pone-0000847-g003"><div class="img"><a name="pone-0000847-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Nucleotide context preferences can be detected in total genome sequences.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A large-scale map for codon-pair context was produced using either the ORFeome (panel A) or the total genome (panel B) sequences of 119 species (see <a href="#pone-0000847-g001">Figure 1</a> and Methods). Such patterns are either universal i.e. present in every species, or visible only in special phylogenetic groups. Surprisingly, most of the ORFeome patterns were also present in total genome sequences, implying that the major forces that drive the evolution of coding sequences are not necessarily connected to mRNA translation. Moreover, when a Differential Map Display (DDM) was built to compare the two former maps (panel C) it became clear that eukaryotes have a more heterogeneous behavior, since they showed greater resemblance between coding and non-coding sequences (darker pattern in the DDM), but they also produced the larger differences found in the DDM (*). These differences correspond either to two-codon context rules imposed by the translational machinery and hence specific of ORFeomes, or to genome biases that are strongly repressed in coding sequences, where they are probably associated to increased decoding error rates. ORFeomes were arranged in the map by domain of life (Eukaryota, Archaea and Eubacteria from left to right) and sorted as shown in <a href="#pone.0000847.s002">Figure S2</a>. Adjusted residuals are colored in the maps so that green cells correspond to preferred and red cells to rejected contexts, while in the DDM major differences (above 15) between residuals of the previous maps are shown in light blue.</p>
<span>doi:10.1371/journal.pone.0000847.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In order to evaluate whether those general codon-pair context patterns arose from DNA replication biases, a second large scale comparative map was built using complete genome sequences (coding + non-coding) of the 119 organisms under study. For this, ANACONDA 2.0 scanned full chromosome sequences starting at the first six nucleotides and moved the scanning window three nucleotides at each step. In this way, both coding and non-coding sequences were analyzed and the frequency of all hexanucleotides was computed, without worrying about the DNA strand location or the reading frame of coding sequences, i.e. ORFs were scanned randomly in the frames 0, +1 or +2. This full genome context map (<a href="#pone-0000847-g003">Figure 3B</a>) showed patterns that were also observed in the ORFeome map (<a href="#pone-0000847-g003">Figure 3A</a>), confirming that DNA replication biases strongly influence codon-pair context. Since the difference between full genome (<a href="#pone-0000847-g003">Figure 3B</a>) and ORFeome (<a href="#pone-0000847-g003">Figure 3A</a>) codon context maps could separate global genome biases from translational biases, a DDM was built and the differences between the two were colored using a blue color scale (<a href="#pone-0000847-g003">Figure 3C</a>), as before (<a href="#pone-0000847-g002">Figure 2</a>). The DDM showed significant differences between full genome and ORFeome maps indicating that codon-pair context is also influenced by evolutionary forces that are not related to DNA replication biases. Interestingly, the column corresponding to eukaryotes was generally darker than the rest of the map, meaning that coding and non-coding sequences are similar in eukaryotes (i.e. stronger influence of DNA replication biases). However, the eukaryotic region of the DDM included the highest differences between ORFeomes and genomes (marked with * in <a href="#pone-0000847-g003">Figure 3C</a>), suggesting that the eukaryotic translational machinery also imposes strong selective pressure on specific combinations of codons, resulting in a localized higher divergence between coding and non-coding sequences.</p>


<h4>Codon-pair context is influenced by genome and mRNA translation biases</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Since DNA replication biases are partly visible at the dinucleotide level <a href="#pone.0000847-Campbell1">[30]</a>–<a href="#pone.0000847-Hooper1">[32]</a>, we have constructed individual codon-pair context maps in which rows and columns were sorted to separate P-site codons ending with a particular nucleotide (N3; rows) and A-site codons starting with a particular nucleotide (N1; columns) (<a href="#pone-0000847-g004">Figure 4A</a>). These two consecutive positions of codon-pair context discriminated rather well codon-pair preferences and such discrimination was very strong for high eukaryotes and weak for low eukaryotes and bacteria (<a href="#pone-0000847-g004">Figure 4A</a>). In order to determine whether such dinucleotide bias was linked to translational selection or to overall genome dinucleotide preferences, the dinucleotide bias was determined for the full set of 119 genomes under study (<a href="#pone-0000847-g004">Figure 4B</a>). Overall, rejection of UA dinucleotides in the 3 domains of life was evident; a trend that corresponded to the negative codon-pair context rule (NNU<sub>3</sub>-A<sub>1</sub>NN) described above. The overall dinucleotide biases were also in agreement with the codon-pair context pattern (<a href="#pone-0000847-g004">Figure 4A</a>). For example, the rejection of CpG dinucleotides in higher eukaryotes (with the surprising exception of the honeybee, <em>Apis mellifera</em>), was also observed in NNC<sub>3</sub>-G<sub>1</sub>NN codon-pairs (<a href="#pone-0000847-g004">Figure 4A</a>). Other examples were UpG and CpA dinucleotides that were strongly preferred in higher eukaryotes (<a href="#pone-0000847-g004">Figure 4B</a>), a characteristic that was also reflected in codon-pair context maps (<a href="#pone-0000847-g004">Figure 4A</a>). Finally, the dinucleotide biases (<a href="#pone-0000847-g004">Figure 4B</a>) showed overall preference for ApA and UpU dinucleotides. This feature originated from frequent tandem repeats of 3 and more identical bases (<a href="#pone.0000847.s015">Figure S4</a>).</p>
<div class="figure" id="pone-0000847-g004"><div class="img"><a name="pone-0000847-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Influence of dinucleotide bias on the codon-pair context preferences.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) In order to highlight the influence of dinucleotide bias on codon-pair contexts, the maps of <em>H. sapiens</em>, <em>M. musculus</em>, <em>S. cerevisiae</em> and <em>E. coli</em> were arranged according to their (N<sub>3</sub>-N<sub>1</sub>) context. High degree of context discrimination was achieved by these two positions in higher eukaryotes, especially for the dinucleotide CpG (blue square), however this effect was weak in yeast and <em>E. coli</em> showed an opposite preference pattern (green). Adjusted residuals are colored in the maps so that green cells correspond to preferred and red cells to rejected contexts. B) In order to further evaluate the role of the dinucleotide N<sub>3</sub>-N<sub>1</sub> bias on codon-pair context biases dinucleotide preferences were determined using total genome sequences. The dinucleotide combinations with highest bias were displayed in green (preferred dinucleotides) or red (rejected ones) and correspond to dinucleotides that appear 1% above or bellow the expected level, respectively. The UpA dinucleotide is strongly repressed throughout all domains of life. Other constraints imposed on ORFeomes by genomes biases include the rejection of CpG dinucleotides in higher eukaryotes and the accumulation of CpA and UpG in higher eukaryotes or UpU and ApA in almost all organisms. The last preference is related to high number of tandem repeats of more than 3 consecutive Us or As (<a href="#pone.0000847.s015">Figure S4</a>). ORFeomes were arranged in both maps by domain of life (Eukaryota, Archaea and Eubacteria from left to right) and sorted as shown in <a href="#pone.0000847.s002">Figure S2</a>.</p>
<span>doi:10.1371/journal.pone.0000847.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The only universal rule detected in the large-scale comparison (<a href="#pone-0000847-g003">Figure 3</a>) contained codon pairs of the type NNU<sub>3</sub>-A<sub>1</sub>NN. Since this rule included out-of-frame stop codons, namely UAA or UAG (i.e. NNU<sub>3</sub>-A<sub>1</sub>A<sub>2</sub>N or NNU<sub>3</sub>-A<sub>1</sub>G<sub>2</sub>N), we investigated whether NNU<sub>3</sub>-A<sub>1</sub>NN rejection was related to premature translation termination. For this, we constructed a subset of codon-pair context maps in which the contexts containing out-of-frame stop codons were represented (<a href="#pone-0000847-g005">Figure 5</a>). This approach showed that NNU<sub>3</sub>-A<sub>1</sub>A<sub>2</sub>N and NNU<sub>3</sub>-A<sub>1</sub>G<sub>2</sub>N type contexts were indeed the most negative in almost all ORFeomes. However, NNU<sub>3</sub>-G<sub>1</sub>A<sub>2</sub>N; NU<sub>2</sub>A<sub>3</sub>-A<sub>1</sub>NN and NU<sub>2</sub>G<sub>3</sub>-A<sub>1</sub>NN contexts which also contained out-of-frame stop codons had a majority of positive residual values (green), while NNU<sub>3</sub>-A<sub>1</sub>C<sub>2</sub>N and NNU<sub>3</sub>-A<sub>1</sub>U<sub>2</sub>N contexts that did not contain out-of-frame stop codons had a majority of negative residual values (red). Since some of the positive context rules (<a href="#pone-0000847-g005">Figure 5</a>) included the dinucleotide UpA, which was rejected in the total genomes map (<a href="#pone-0000847-g004">Figure 4</a>), it is likely that dinucleotide bias is not the only cause for the rejection of codon-pair contexts. On the other hand, premature termination was not the only potential problem here, because NNU<sub>3</sub>-A<sub>1</sub>C<sub>2</sub>N and NNU<sub>3</sub>-A<sub>1</sub>U<sub>2</sub>N did not correspond to out-of-frame stop codons and were also strongly rejected in ORFeomes (<a href="#pone-0000847-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0000847-g005"><div class="img"><a name="pone-0000847-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Genome dinucleotide bias has a strong influence on codon-pair context.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Since the most generalized negative codon-pair context rule detected corresponds to the general expression NNU<sub>3</sub>-A<sub>1</sub>NN, which includes the out-of-frame translation termination contexts NNU<sub>3</sub>-A<sub>1</sub>A<sub>2</sub>N and NNU<sub>3</sub>-A<sub>1</sub>G<sub>2</sub>N, other out-of-frame context terminators were analyzed separately. For this, the adjusted residuals of such contexts were included in an ORFeome comparison map. It was clear that NNU<sub>3</sub>-A<sub>1</sub>GN and NNU<sub>3</sub>-A<sub>1</sub>AN were indeed the most negative codon-pair contexts bearing out-of-frame stops, followed by NUA<sub>3</sub>-G<sub>1</sub>NN. The other groups of contexts tested did not generate codon-pair context rules, although some of them contained the strongly repressed UpA dinucleotide. The hypothesis that rejection of codon-pair contexts containing out-of-frame stop codons, namely NNU<sub>3</sub>-A<sub>1</sub>A<sub>2</sub>N and NNU<sub>3</sub>-A<sub>1</sub>G<sub>2</sub>N evolved to avoid premature termination was partially contradicted by the existence of similar patterns of NNU<sub>3</sub>-A<sub>1</sub>NN-type contexts that do not include any out-of-frame stops, namely NNU<sub>3</sub>-A<sub>1</sub>C<sub>2</sub>N and NNU<sub>3</sub>-A<sub>1</sub>U<sub>2</sub>N. ORFeomes were arranged in the map by domain of life (Eukaryota, Archaea and Eubacteria from left to right) and sorted as shown in <a href="#pone.0000847.s002">Figure S2</a>. Adjusted residuals are colored in the maps so that green cells correspond to preferred and red cells to rejected contexts.</p>
<span>doi:10.1371/journal.pone.0000847.g005</span></div>

<h4>General codon-pair context rules</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In order to highlight the codon-pair context preferences that were exclusive of coding sequences, the original map of ORFeomes (<a href="#pone-0000847-g003">Figure 3A</a>) was rebuilt to show in black cells whose residuals values were similar to those of identical contexts in the complete genomes map (<a href="#pone-0000847-g003">Figure 3B</a>). In this filtered map (<a href="#pone-0000847-g006">Figure 6A</a>) green and red colored cells corresponded to those context residual values calculated for ORFeomes that were significantly different from those calculated for complete genomes, i.e. cells that were colored in blue in <a href="#pone-0000847-g003">Figure 3C</a>. This large-scale comparative map allowed extraction of ORFeome specific codon-context patterns, while the converse filtering originated a complete genomes map that permitted extraction of genome specific patterns (<a href="#pone-0000847-g006">Figure 6B</a>). This approach highlighted clear codon-pair context differences between ORFeome and complete genome maps (<a href="#pone-0000847-g006">Figure 6A,B</a>). Interestingly, these patterns corresponded to different sets of codon-pair contexts that could be easily described by the expressions annotated on the side of each map (<a href="#pone-0000847-g006">Figure 6A,B</a> see also <a href="#pone.0000847.s016">Figure S5A</a>,<a href="#pone.0000847.s017">B</a> for different thresholds of visualization). In almost all cases, these codon-pair context rules fixed the last nucleotide of the first codon and the first nucleotide of the second codon, confirming that (N<sub>3</sub>-N<sub>1</sub>) positions shape codon context. Remarkably, the major patterns that appeared in the filtered map for complete genomes (<a href="#pone-0000847-g006">Figure 6B</a>) were related to UpA-rich hexanucleotides that produce weak codon-anticodon interactions in coding regions and should hence be under differential selective pressure in both types of sequences.</p>
<div class="figure" id="pone-0000847-g006"><div class="img"><a name="pone-0000847-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000847" data-uri="info:doi/10.1371/journal.pone.0000847.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000847.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000847.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000847.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Some codon-pair context patterns are associated to mRNA primary structure biases.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A) In order to identify ORFeome specific codon-pair context biases the two large scale context maps were filtered in such a way that only cells that yielded residual differences above 15 between the ORFeomes and total genomes sequences were shown. All other cases were colored in black (see <a href="#pone.0000847.s016">Figure S5</a> for different display thresholds). Codon-pair context patterns specific of ORFeomes are highlighted on the side of panel A. B) To visualize the patterns that appear in genomes and are absent in ORfeomes, large-scale comparative maps obtained with total genomes and ORFeomes were subtracted and only the cells that yielded differences above 15 were displayed. This highlighted patterns that are strongly preferred or repressed in coding sequences and may correspond to mistranslation hot spots.</p>
<span>doi:10.1371/journal.pone.0000847.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Mistranslation is a poorly understood biological phenomenon which is influenced by various protein synthesis factors and mRNA primary structure features <a href="#pone.0000847-Ogle1">[13]</a>, <a href="#pone.0000847-Hooper2">[33]</a>, <a href="#pone.0000847-Stahl1">[34]</a>. In order to shed new light on how the later influences decoding error and extend previous studies carried out mainly on the effect of codon usage on mistranslation <a href="#pone.0000847-Kramer1">[25]</a>, <a href="#pone.0000847-Dix1">[35]</a>, we are investigating the effect of codon-pair context on decoding fidelity. Our comparative genomics approaches unveiled the effect of both genome replication and translation specific biases on codon-pair context. The few studies carried out to date on codon-pair context were unable to distinguish those two types of biases <a href="#pone.0000847-Boycheva1">[12]</a>, <a href="#pone.0000847-Gutman1">[36]</a>–<a href="#pone.0000847-Rocha1">[38]</a>. Our large scale approach confirmed the importance of genomic biases but also unveiled important translational biases that shape codon-pair context and should be primary targets for mistranslation hot spots.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Large-scale genomic analysis, such as the one that we have performed, allows for obtaining a global view of mistranslation in a way that is totally out of reach from analysis of single ORFeomes. Indeed, comparison of large sets of codon-pair context data unveiled the main codon-pair context patterns that exist in the 3 domains of life. Interestingly, when the most preferred or repressed codon-pair contexts of all organisms were considered (<a href="#pone-0000847-t001">Table 1</a>), but also when common rules were selected (<a href="#pone-0000847-t002">Table 2</a>), there was little or no overlapping between the context patterns of the 3 domains of life. This suggests that genome replication and/or mRNA translation in each domain imposes specific constraints to decoding sequences which produce different codon-pair context outcomes. Also, the phylogeny of individual species appeared as an important determinant of its codon-pair context behavior (<a href="#pone-0000847-g002">Figure 2</a>), in a similar manner to that described for codon usage bias <a href="#pone.0000847-Grantham1">[39]</a> or dinucleotide genome signatures <a href="#pone.0000847-Nakashima1">[31]</a>, <a href="#pone.0000847-Hooper1">[32]</a>.</p>

<h4>Influence of genome wide biases on codon-pair context</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Our observation that ORFeomes and total genomes produce similar patterns of codon-pair context (<a href="#pone-0000847-g003">Figure 3</a>) confirmed previous studies <a href="#pone.0000847-Chen1">[4]</a>, <a href="#pone.0000847-Buckingham2">[40]</a>, <a href="#pone.0000847-McVean1">[41]</a>. This implies that most sequence constraints that can be detected in coding sequences are not imposed by the translational machinery, but arise from selective pressure imposed by DNA replication and related biases. That codon usage biases were mainly due to mutational pressure and only secondarily to translational selection further confirmed the relevance of DNA replication biases on codon-pair context <a href="#pone.0000847-Duan1">[7]</a>. In this scenario, one is prompted to hypothesize that the translational process may work with sub-optimized mRNA sequences since codon-context fine tunes decoding fidelity <a href="#pone.0000847-Shah1">[15]</a>, <a href="#pone.0000847-Murgola1">[22]</a>, <a href="#pone.0000847-Tork1">[23]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Genomes are known to have biased dinucleotide frequencies <a href="#pone.0000847-Nakashima1">[31]</a>, a feature that has frequently been used to produce genomic signatures of phylogenetical and taxonomical relevance <a href="#pone.0000847-Nakashima1">[31]</a>, <a href="#pone.0000847-Hooper1">[32]</a>. At the ORFeome level this bias influences codon usage <a href="#pone.0000847-Hooper1">[32]</a> but may also interfere with codon-context, whenever the last nucleotide of one codon is associated with the first nucleotide of the second codon of the pair. Indeed, (N<sub>3</sub>–N<sub>1</sub>) contexts explained part of our results (<a href="#pone-0000847-g006">Figure 6</a>) and confirmed the good discrimination obtained when one ORFeome map for codon context was arranged according to the last position of the first codon and the first position of the next codon (<a href="#pone-0000847-g004">Figure 4A</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>The association bias of two consecutive nucleotides is a characteristic of genomes which results from global selective pressures acting upon DNA at the level of repair and replication mechanisms <a href="#pone.0000847-Hooper1">[32]</a> or ecological constraints that may influence, for instance, the overall G+C content of the genome <a href="#pone.0000847-Lao1">[42]</a>–<a href="#pone.0000847-Tekaia1">[44]</a>. Regulatory activity acting upon the entire genome is another cause of dinucleotide bias. CpG dinucleotides, for example, are signals for DNA methylation, a mechanism commonly used by higher organisms to protect their genome from selfish DNA elements and to regulate gene expression <a href="#pone.0000847-Chan1">[5]</a>, <a href="#pone.0000847-Robertson1">[6]</a>. Our dinucleotide bias analysis for the 119 organisms confirmed a clear rejection of CpG methylation in coding sequences of high eukaryotes, as would be expected, since methylated DNA becomes unavailable for transcription and hence translation <a href="#pone.0000847-Chan1">[5]</a>. On the other hand, UpA dinucleotides are highly repressed in DNA sequences of most organisms <a href="#pone.0000847-Duan1">[7]</a>, <a href="#pone.0000847-Nakashima1">[31]</a>, <a href="#pone.0000847-Beutler1">[45]</a>, <a href="#pone.0000847-Nakashima2">[46]</a>. Interestingly, UpA dinucleotides are sites for preferential hydrolysis of RNA by macrophage ribonucleases <a href="#pone.0000847-Beutler1">[45]</a> destabilizing RNA molecules <a href="#pone.0000847-Duan1">[7]</a> and should hence be avoided <a href="#pone.0000847-Beutler1">[45]</a>. Furthermore, Duan and colleagues <a href="#pone.0000847-Duan1">[7]</a> proposed that mRNA stability imposes strong selective pressure on synonymous codon usage and it is likely that this is also true for codon-pair context. Our data confirmed that hypothesis since NNU<sub>3</sub>-A<sub>1</sub>NN contexts were highly repressed in the 119 different genomes analyzed.</p>


<h4>Influence of translational biases on codon-pair context</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>As already mentioned, the unique universal rule that could be detected in the 119 genomes analyzed was rejection of most codon-pair contexts of the type NNU<sub>3</sub>-A<sub>1</sub>NN (<a href="#pone-0000847-g003">Figure 3</a>). Clearly, this trend is a direct result of repression of the TpA dinucleotide in total DNA sequences (<a href="#pone-0000847-g003">Figure 3</a>). However, it was surprising that other UpA bearing contexts did not show strong rejection. For example, NU<sub>2</sub>A<sub>3</sub>-A<sub>1</sub>NN contexts are mainly preferred in coding sequences (<a href="#pone-0000847-g005">Figure 5</a>) indicating strong differences between codon-pairs containing UpA dinucleotides and suggesting that translation does influence codon-pair choice.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>When a large-scale comparison of codon-pair context excluded global genome biases (<a href="#pone-0000847-g006">Figure 6A</a>) it became evident that contexts that were truly produced by translation-driven selection were grouped in negative or positive rules depending on the phylogeny of the organisms. This was in agreement with the previous observation that strongly biased codon-pair contexts were different between the 3 domains of life (<a href="#pone-0000847-t001">Tables 1</a>, <a href="#pone-0000847-t002">2</a>), and supported the hypothesis that differences in the translational machineries of different organisms reshape mRNA primary structure in different ways. For example, the NNC<sub>3</sub>-N<sub>4</sub>NN contexts pattern of higher eukaryotes (B1 and B2 in <a href="#pone-0000847-g006">Figure 6A</a>) could be explained by specific decoding rules of C-ending codons in Eukarya. Indeed, eukaryotic species translate several C-ending codons by wobble pairing rules using inosine <a href="#pone.0000847-Marck1">[47]</a>, which recognizes A, C, or U at the wobbling position <a href="#pone.0000847-Crick1">[48]</a> while bacterial species decode most C-ending codons with Watson-Crick C-G base pairing between codon and anticodon <a href="#pone.0000847-Marck1">[47]</a>.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>As to the other minor rules highlighted on the left side of <a href="#pone-0000847-g006">Figure 6</a>, namely NNU<sub>3</sub>-G<sub>1</sub>G<sub>2</sub>R, NRU<sub>3</sub>-G<sub>1</sub>A<sub>2</sub>N and NG<sub>2</sub>N-NG<sub>2</sub>N or NC<sub>2</sub>N-NC<sub>2</sub>N, they may be related to both canonical decoding of U-ending codons in eukaryotes (A1 and A2 in <a href="#pone-0000847-g006">Figure 6A</a>) and to the existence of runs of special sets of amino acids, namely serine/proline/threonine/alanine and arginine/glycine (C in <a href="#pone-0000847-g006">Figure 6A</a>). That contexts of repeated codons are preferred in eukaryotic genes (<a href="#pone-0000847-t001">Tables 1</a>,<a href="#pone-0000847-t002">2</a>) and that proline, alanine and glycine are frequently found in amino acid runs of human genes <a href="#pone.0000847-Caburet1">[49]</a> corroborates the above hypothesis.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>On the other hand, most of the major genomic constraints that were not present in coding sequences, namely NNU<sub>3</sub>-A<sub>1</sub>NN, NYU<sub>3</sub>-A<sub>1</sub>RN and N(U/A)<sub>2</sub>U<sub>3</sub>-U<sub>1</sub>(U/A)<sub>2</sub>N or N(U/A)<sub>2</sub>A<sub>3</sub>-A<sub>1</sub>(U/A)<sub>2</sub>N rules (<a href="#pone-0000847-g006">Figure 6B</a>) were associated to weak decoding interactions involving A-U codon-anticodon pairing. Moreover, these rules are produced by either strong genomic dinucleotide bias against UpA (<a href="#pone-0000847-g006">Figure 6B</a>, rule D) or by rejection of error prone UA-rich codon-pair contexts in coding sequences (<a href="#pone-0000847-g006">Figure 6B</a>, rule F), in a clear confirmation of the additive effect of translational and non-translational selective pressures. Finally, we could also see a preference for trinucleotide repeats in non-coding sequences that was not detectable in coding regions, at least in eukaryotes (<a href="#pone-0000847-g006">Figure 6B</a>, rule E). This has already been described in primates and is related to strong mRNA primary structure constraints associated to high mRNA decoding efficiency <a href="#pone.0000847-Borstnik1">[50]</a>.</p>


<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Codon-pair contexts are biased in ORFeomes and such bias is the result of both translation and non-translation driven processes. Indeed, translational and DNA replication/repair and cis regulatory elements act synergistically on codon-pair context. This myriad of selective pressures creates significant difficulties to the identification of codon-context biases associated to mRNA translation only. Our large scale comparative genome approach indicated that: i) there is a strong influence of non-translational selective pressures upon coding sequences, especially in eukaryotic organisms since these have a higher degree of resemblance between ORFeome and total genome biases; ii) the strongest non-translational selective pressures that could be identified were dinucleotide biases, mainly imposed by regulatory cis-elements linked to DNA methylation or mRNA stability <a href="#pone.0000847-Chan1">[5]</a>, <a href="#pone.0000847-Beutler1">[45]</a>, and preference for trinucleotide repeats, usually associated with DNA polymerase slippage during replication <a href="#pone.0000847-Rocha2">[51]</a>; iii) apart from this non-translational noise, DNA coding sequences showed specific features that could be related to mRNA translation, namely repression of usage of premature termination or error-prone contexts associated to weak codon-anticodon interactions. It will now be most interesting to validate these <em>in silico</em> data <em>in vivo</em>, and identify experimentally the codon-pair contexts that are strongly selected for high mRNA decoding fidelity.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Primary data sources</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Nucleotide sequences, of complete genomes and assembled ORFeomes, were downloaded from GenBank or Ensembl Web sites (Genbank: <a href="ftp://ftp.ncbi.nih.gov/genomes/">ftp://ftp.ncbi.nih.gov/genomes/</a>; Ensembl: <a href="ftp://ftp.ensembl.org/pub/">ftp://ftp.ensembl.org/pub/</a>) between December 2005 and January 2006. These included the DNA sequences of 81 eubacterial, 18 archaeal and 20 eukaryotic species. Plasmid sequences were not included in the analysis and all chromosomal sequences from one genome were analyzed together by ANACONDA 2.0. The total set of files downloaded and used in this study is documented as supplementary data (<a href="#pone.0000847.s002">Figure S2</a>).</p>


<h4>Statistical analyses</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Two-codon context bias was studied in complete ORFeome sequences using the residual analysis tools available in the software package ANACONDA 1.0 (a detailed explanation of this software can be found in <a href="#pone.0000847-Moura1">[20]</a>, <a href="#pone.0000847-Pinheiro1">[29]</a>. ANACONDA is publicly available at <a href="http://bioinformatics.ua.pt/submited-papers">http://bioinformatics.ua.pt/submited-pap​ers</a>).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Briefly, this methodology counts all consecutive pairs of codons and uses statistical analysis for contingency tables where a multinomial distribution is assumed (<a href="#pone-0000847-g001">Figure 1B</a>). The final result of such statistical approach is the calculation of adjusted residuals for each codon pair present in any ORFeome. The adjusted residuals give direct information about preference or rejection of these codon pairs in relation to what would be expected assuming independence of the distribution (<a href="#pone-0000847-g001">Figure 1B</a>).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Since, under independence between two consecutive codons, the adjusted residuals <em>d<sub>ij</sub></em> have a standardized normal probability distribution <a href="#pone.0000847-Haberman1">[52]</a>, we have concluded that: <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000847.e001&amp;representation=PNG" class="inline-graphic"></span>, as the total number of observations is very high. This means that, for a 99,73% confidence level, an adjusted residue was statistically significant if its absolute value was greater than 3 <a href="#pone.0000847-Moura1">[20]</a>. However, this approach was based on a local analysis for each residual value. Herein, we considered a global analysis for each species and have thus constructed a simultaneous confidence region for all residual values. Since there are K = 61×64 different intervals we have introduced the Bonferroni correction to ensure an overall level of significance of α (usually α = 0.05, 0.01, 0.001). The Bonferroni correction is used for correction where each interval is constructed at a 100×(1–α/K) level (see, for example, <a href="#pone.0000847-Simenoff1">[53]</a>). Therefore, a–<em>a</em> to <em>a</em> interval at a confidence level of 100×(1−α/(61×64)) was constructed for each adjusted residual value <em>d<sub>ij</sub></em>. Considering again the asymptotic normal distribution of <em>d<sub>ij</sub></em> <a href="#pone.0000847-Haberman1">[52]</a> we had <em>a</em>≈4,70341 when 1–α = 0,99, <em>a</em>≈5,15350 when 1–α = 0,999, <em>a</em>≈8,16204 when 1–α = 0.01×10<sup>−10</sup>. Thus, we assumed that the codon-pair adjusted residuals that fall within the interval −5 to 5 were not statistically significant, for a global confidence level of 99% (colored in black in all maps shown herein).</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>The final output of residual analysis performed by ANACONDA is a codon-pair context map for each ORFeome being studied (<a href="#pone-0000847-g001">Figure 1C</a>). These maps show one colored square for each codon-pair, the first codon corresponding to rows and the second corresponding to columns in the map. The color scale chosen for this layout determines that preferred contexts are shown in green while repressed ones appear in red (<a href="#pone-0000847-g001">Figure 1C</a>).</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>Taking advantage of the automated statistical analysis performed by ANACONDA, individual maps for all 119 ORFeomes were built (see Figure S3). In order to facilitate large-scale comparison of maps these were converted into single lines and clustered together (<a href="#pone-0000847-g001">Figures 1D,E</a> and <a href="#pone-0000847-g003">3</a>). The patterns that appear in the resulting comparative map were then characterized by the codon-pair contexts that were present in each pattern. Also, the values of the adjusted residuals calculated for each species were corrected for ORFeome size to allow direct comparisons among ORFeomes.</p>
<a id="article1.body1.sec4.sec2.p6" name="article1.body1.sec4.sec2.p6"></a><p>The above approach was also used to study total genome sequences of the same 119 species in order to differentiate between the effect of translational selection acting upon coding sequences alone and genome mutational biases. With the same purpose, the bias for dinucleotides was studied in total genome sequences, and shown as observed frequencies, colored in green or red whenever the result was 1% above or below the expected value, respectively (<a href="#pone-0000847-g004">Figure 4B</a>).</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000847"></div><a name="pone.0000847.s001" id="pone.0000847.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Data normalization. In order to correct the size differences of ORFeomes, particularly between eukaryotes and non-eukaryotes, the adjusted residuals were normalized for 21 million codons which correspond approximately to the larger ORFeome analyzed (X. tropicalis). Normalization of codon-pair data for human chromosomes 1, 2, 3, 22 and ORFeome are displayed. The normalization effect is shown by the brightness of the maps, which is variable in non-normalized maps (above) and constant in normalized ones (below). After data normalization the differences between maps could be compared as shown in the DDM (right end of the Figure).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000847.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s002" id="pone.0000847.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s002">Figure S2A. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of species used. All species used in the study are listed according to the download order. The database of origin and respective accession numbers are indicated. A - Eukaryotes; B - Archaea and Eubacteria; C - Eubacteria (cont.).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000847.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.54 MB TIF)</p>
<a name="pone.0000847.s003" id="pone.0000847.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s003">Figure S2B. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.54 MB TIF)</p>
<a name="pone.0000847.s004" id="pone.0000847.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s004">Figure S2C. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.54 MB TIF)</p>
<a name="pone.0000847.s005" id="pone.0000847.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s005">Figure S3A. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Individual codon-pair context maps of the 119 species. The codon-pair context maps built with ANACONDA software for individual ORFeomes are shown as ordered in Suppl. <a href="#pone.0000847.s002">Figure S2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000847.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s006" id="pone.0000847.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s006">Figure S3B. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s007" id="pone.0000847.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s007">Figure S3C. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s008" id="pone.0000847.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s008">Figure S3D. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s009" id="pone.0000847.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s009">Figure S3E. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s010" id="pone.0000847.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s010">Figure S3F. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="postSiDOI">(4.32 MB TIF)</p>
<a name="pone.0000847.s011" id="pone.0000847.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s011">Figure S3G. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="postSiDOI">(2.16 MB TIF)</p>
<a name="pone.0000847.s012" id="pone.0000847.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s012">Figure S3H. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="postSiDOI">(2.16 MB TIF)</p>
<a name="pone.0000847.s013" id="pone.0000847.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s013">Figure S3I. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="postSiDOI">(2.16 MB TIF)</p>
<a name="pone.0000847.s014" id="pone.0000847.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s014">Figure S3J. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="postSiDOI">(2.16 MB TIF)</p>
<a name="pone.0000847.s015" id="pone.0000847.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s015">Figure S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">A and U bases are preferentially arranged in polynucleotide strings. In order to check if the preference detected for AA and UU dinucleotides in total genomes (<a href="#pone-0000847-g004">Figure 4B</a>) was due to a tendency for these bases to appear as polynucleotide strings we counted the number of times each individual base appeared isolated or in strings of two, three or more equal bases. The result of this approach points to a clear positive bias towards the accumulation of 3 or more consecutive A or U bases in total genomes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000847.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(0.10 MB TIF)</p>
<a name="pone.0000847.s016" id="pone.0000847.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s016">Figure S5A. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">Codon-pair context patterns that are exclusive of ORFeomes or genomes. The filtering technique that was used to determine the biases of codon-pair contexts in coding and total sequences (<a href="#pone-0000847-g006">Figure 6</a>) was further explored in here to evaluate the strength of those biases. This was done by gradually increasing the threshold of the residuals (D) that are significantly different in both maps, i.e. those that were allowed to appear in their original colors in the filtered map. When D was increased, only major differences between ORFeomes and genomes were visible, corresponding to differences between the residuals of ORFeomes and genomes maps that stay above 15, 20, 30 or 50. The strongest rules detected by this approach correspond to those identified as B1 and B2 in the filtered map for ORFeomes (map A in <a href="#pone-0000847-g006">Figure 6</a>), and F1, F2 and E2 in the filtered map for genomes (map B in <a href="#pone-0000847-g006">Figure 6</a>), because they are still visible when D = 50.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000847.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.55 MB TIF)</p>
<a name="pone.0000847.s017" id="pone.0000847.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s017">Figure S5B. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000847.s017</p><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1"></a><p class="postSiDOI">(0.53 MB TIF)</p>
<a name="pone.0000847.s018" id="pone.0000847.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000847.s018">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Codon-pair distribution similarities between the 3 domains of life. In order to compare the overall distribution of codon-pair contexts among the 119 organisms we have calculated the Spearman's correlation coefficients between all pairs of ORFeomes, producing a triangular colored map. The 119 species were organized by domain of life and sorted alphabetically in each domain. Pairs of species that were not statistically correlated (for a level of significance of 5%) are colored in grey, while green colored cells indicate pairs of species that were highly correlated (correlation coefficient above 0,80), and blue colored cells correspond to the major values fount inside each domain.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000847.s018</p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.32 MB XLS)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS GM MP JO. Performed the experiments: GM MP AG LC. Analyzed the data: GM MP JA AF JO. Contributed reagents/materials/analysis tools: GM. Wrote the paper: MS GM. Other: Developed software: JO JA MP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000847-Cliften1" id="pone.0000847-Cliften1"></a>Cliften PF, Fulton RS, Wilson RK, Johnston M (2006) After the duplication: gene loss and adaptation in Saccharomyces genomes. Genetics  172: 863–872.  <ul class="find" data-citedArticleID="987884" data-doi="10.1534/genetics.105.048900"><li><a href="http://dx.doi.org/10.1534/genetics.105.048900" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=After+the+duplication%3A+gene+loss+and+adaptation+in+Saccharomyces+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22After+the+duplication%3A+gene+loss+and+adaptation+in+Saccharomyces+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000847-vandeLagemaat1" id="pone.0000847-vandeLagemaat1"></a>van de Lagemaat LN, Gagnier L, Medstrand P, Mager DL (2005) Genomic deletions and precise removal of transposable elements mediated by short identical DNA segments in primates. Genome Res  15: 1243–1249.  <ul class="find" data-citedArticleID="987962" data-doi="10.1101/gr.3910705"><li><a href="http://dx.doi.org/10.1101/gr.3910705" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+deletions+and+precise+removal+of+transposable+elements+mediated+by+short+identical+DNA+segments+in+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+deletions+and+precise+removal+of+transposable+elements+mediated+by+short+identical+DNA+segments+in+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000847-Lin1" id="pone.0000847-Lin1"></a>Lin YW, Thi DA, Kuo PL, Hsu CC, Huang BD, et al.  (2005) Polymorphisms associated with the DAZ genes on the human Y chromosome. Genomics  86: 431–438.  <ul class="find" data-citedArticleID="987918" data-doi="10.1016/j.ygeno.2005.07.003"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.07.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polymorphisms+associated+with+the+DAZ+genes+on+the+human+Y+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polymorphisms+associated+with+the+DAZ+genes+on+the+human+Y+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000847-Chen1" id="pone.0000847-Chen1"></a>Chen SL, Lee W, Hottes AK, Shapiro L, McAdams HH (2004) Codon usage between genomes is constrained by genome-wide mutational processes. Proc Natl Acad Sci U S A  101: 3480–3485.  <ul class="find" data-citedArticleID="987882" data-doi="10.1073/pnas.0307827100"><li><a href="http://dx.doi.org/10.1073/pnas.0307827100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+usage+between+genomes+is+constrained+by+genome-wide+mutational+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+usage+between+genomes+is+constrained+by+genome-wide+mutational+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000847-Chan1" id="pone.0000847-Chan1"></a>Chan SW, Henderson IR, Jacobsen SE (2005) Gardening the genome: DNA methylation in Arabidopsis thaliana. Nat Rev Genet  6: 351–360.  <ul class="find" data-citedArticleID="987880" data-doi="10.1038/nrg1601"><li><a href="http://dx.doi.org/10.1038/nrg1601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gardening+the+genome%3A+DNA+methylation+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gardening+the+genome%3A+DNA+methylation+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000847-Robertson1" id="pone.0000847-Robertson1"></a>Robertson KD (2005) DNA methylation and human disease. Nat Rev Genet  6: 597–610.  <ul class="find" data-citedArticleID="987942" data-doi="10.1038/nrg1655"><li><a href="http://dx.doi.org/10.1038/nrg1655" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+methylation+and+human+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+methylation+and+human+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000847-Duan1" id="pone.0000847-Duan1"></a>Duan J, Antezana MA (2003) Mammalian mutation pressure, synonymous codon choice, and mRNA degradation. J Mol Evol  57: 694–701.  <ul class="find" data-citedArticleID="987894" data-doi="10.1007/s00239-003-2519-1"><li><a href="http://dx.doi.org/10.1007/s00239-003-2519-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mammalian+mutation+pressure%2C+synonymous+codon+choice%2C+and+mRNA+degradation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mammalian+mutation+pressure%2C+synonymous+codon+choice%2C+and+mRNA+degradation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000847-Berg1" id="pone.0000847-Berg1"></a>Berg OG, Silva PJ (1997) Codon bias in Escherichia coli: the influence of codon context on mutation and selection. Nucleic Acids Res  25: 1397–1404.  <ul class="find" data-citedArticleID="987862" data-doi="10.1093/nar/25.7.1397"><li><a href="http://dx.doi.org/10.1093/nar/25.7.1397" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+bias+in+Escherichia+coli%3A+the+influence+of+codon+context+on+mutation+and+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+bias+in+Escherichia+coli%3A+the+influence+of+codon+context+on+mutation+and+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000847-Akashi1" id="pone.0000847-Akashi1"></a>Akashi H (1994) Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy. Genetics  136: 927–935.  <ul class="find" data-citedArticleID="987858"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy.&amp;auth=&amp;atitle=Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synonymous+codon+usage+in+Drosophila+melanogaster%3A+natural+selection+and+translational+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000847-Curran1" id="pone.0000847-Curran1"></a>Curran JF, Yarus M (1989) Rates of aminoacyl-tRNA selection at 29 sense codons in vivo. J Mol Biol  209: 65–77.  <ul class="find" data-citedArticleID="987888" data-doi="10.1016/0022-2836(89)90170-8"><li><a href="http://dx.doi.org/10.1016/0022-2836(89)90170-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rates+of+aminoacyl-tRNA+selection+at+29+sense+codons+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rates+of+aminoacyl-tRNA+selection+at+29+sense+codons+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000847-Percudani1" id="pone.0000847-Percudani1"></a>Percudani R, Ottonello S (1999) Selection at the wobble position of codons read by the same tRNA in Saccharomyces cerevisiae. Mol Biol Evol  16: 1752–1762.  <ul class="find" data-citedArticleID="987934" data-doi="10.1093/oxfordjournals.molbev.a026087"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026087" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+at+the+wobble+position+of+codons+read+by+the+same+tRNA+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+at+the+wobble+position+of+codons+read+by+the+same+tRNA+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000847-Boycheva1" id="pone.0000847-Boycheva1"></a>Boycheva S, Chkodrov G, Ivanov I (2003) Codon pairs in the genome of Escherichia coli. Bioinformatics  19: 987–998.  <ul class="find" data-citedArticleID="987868" data-doi="10.1093/bioinformatics/btg082"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+pairs+in+the+genome+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+pairs+in+the+genome+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000847-Ogle1" id="pone.0000847-Ogle1"></a>Ogle JM, Ramakrishnan V (2005) Structural insights into translational fidelity. Annu Rev Biochem  74: 129–177.  <ul class="find" data-citedArticleID="987932" data-doi="10.1146/annurev.biochem.74.061903.155440"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.74.061903.155440" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+insights+into+translational+fidelity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+insights+into+translational+fidelity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000847-Irwin1" id="pone.0000847-Irwin1"></a>Irwin B, Heck JD, Hatfield GW (1995) Codon pair utilization biases influence translational elongation step times. J Biol Chem  270: 22801–22806.  <ul class="find" data-citedArticleID="987910" data-doi="10.1074/jbc.270.39.22801"><li><a href="http://dx.doi.org/10.1074/jbc.270.39.22801" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+pair+utilization+biases+influence+translational+elongation+step+times." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+pair+utilization+biases+influence+translational+elongation+step+times.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000847-Shah1" id="pone.0000847-Shah1"></a>Shah AA, Giddings MC, Gesteland RF, Atkins JF, Ivanov IP (2002) Computational identification of putative programmed translational frameshift sites. Bioinformatics  18: 1046–1053.  <ul class="find" data-citedArticleID="987950" data-doi="10.1093/bioinformatics/18.8.1046"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.8.1046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+identification+of+putative+programmed+translational+frameshift+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+identification+of+putative+programmed+translational+frameshift+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000847-Buckingham1" id="pone.0000847-Buckingham1"></a>Buckingham RH, Grosjean H (1986)  The accuracy of mRNA-tRNA recognition. In: Kirkwood TBL, Rosenberger RF, Galas DJ, editors. Accuracy in Molecular Processes: Its Control and Relevance to Living Systems. London: Chapman and Hall. pp. 83–126.  <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0000847-Percudani2" id="pone.0000847-Percudani2"></a>Percudani R, Pavesi A, Ottonello S (1997) Transfer RNA gene redundancy and translational selection in Saccharomyces cerevisiae. J Mol Biol  268: 322–330.  <ul class="find" data-citedArticleID="987936" data-doi="10.1006/jmbi.1997.0942"><li><a href="http://dx.doi.org/10.1006/jmbi.1997.0942" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transfer+RNA+gene+redundancy+and+translational+selection+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transfer+RNA+gene+redundancy+and+translational+selection+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000847-Curran2" id="pone.0000847-Curran2"></a>Curran JF, Poole ES, Tate WP, Gross BL (1995) Selection of aminoacyl-tRNAs at sense codons: the size of the tRNA variable loop determines whether the immediate 3′ nucleotide to the codon has a context effect. Nucleic Acids Res  23: 4104–4108.  <ul class="find" data-citedArticleID="987890" data-doi="10.1093/nar/23.20.4104"><li><a href="http://dx.doi.org/10.1093/nar/23.20.4104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selection+of+aminoacyl-tRNAs+at+sense+codons%3A+the+size+of+the+tRNA+variable+loop+determines+whether+the+immediate+3%26%23x2032%3B+nucleotide+to+the+codon+has+a+context+effect." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selection+of+aminoacyl-tRNAs+at+sense+codons%3A+the+size+of+the+tRNA+variable+loop+determines+whether+the+immediate+3%26%23x2032%3B+nucleotide+to+the+codon+has+a+context+effect.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000847-Fedorov1" id="pone.0000847-Fedorov1"></a>Fedorov A, Saxonov S, Gilbert W (2002) Regularities of context-dependent codon bias in eukaryotic genes. Nucleic Acids Res  30: 1192–1197.  <ul class="find" data-citedArticleID="987896" data-doi="10.1093/nar/30.5.1192"><li><a href="http://dx.doi.org/10.1093/nar/30.5.1192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regularities+of+context-dependent+codon+bias+in+eukaryotic+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regularities+of+context-dependent+codon+bias+in+eukaryotic+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000847-Moura1" id="pone.0000847-Moura1"></a>Moura G, Pinheiro M, Silva R, Miranda I, Afreixo V, et al.  (2005) Comparative context analysis of codon pairs on an ORFeome scale. Genome Biol  6: R28.  <ul class="find" data-citedArticleID="987924" data-doi="10.1186/gb-2005-6-3-r28"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-3-r28" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+context+analysis+of+codon+pairs+on+an+ORFeome+scale." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+context+analysis+of+codon+pairs+on+an+ORFeome+scale.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000847-Tate1" id="pone.0000847-Tate1"></a>Tate WP, Poole ES, Mannering SA (1996) Hidden infidelities of the translational stop signal. Prog Nucleic Acid Res Mol Biol  52: 293–335.  <ul class="find" data-citedArticleID="987956" data-doi="10.1016/s0079-6603(08)60970-8"><li><a href="http://dx.doi.org/10.1016/s0079-6603(08)60970-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hidden+infidelities+of+the+translational+stop+signal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hidden+infidelities+of+the+translational+stop+signal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000847-Murgola1" id="pone.0000847-Murgola1"></a>Murgola EJ, Pagel FT, Hijazi KA (1984) Codon context effects in missense suppression. J Mol Biol  175: 19–27.  <ul class="find" data-citedArticleID="987926" data-doi="10.1016/0022-2836(84)90442-x"><li><a href="http://dx.doi.org/10.1016/0022-2836(84)90442-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+context+effects+in+missense+suppression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+context+effects+in+missense+suppression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000847-Tork1" id="pone.0000847-Tork1"></a>Tork S, Hatin I, Rousset JP, Fabret C (2004) The major 5′ determinant in stop codon read-through involves two adjacent adenines. Nucleic Acids Res  32: 415–421.  <ul class="find" data-citedArticleID="987960" data-doi="10.1093/nar/gkh201"><li><a href="http://dx.doi.org/10.1093/nar/gkh201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+major+5%26%23x2032%3B+determinant+in+stop+codon+read-through+involves+two+adjacent+adenines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+major+5%26%23x2032%3B+determinant+in+stop+codon+read-through+involves+two+adjacent+adenines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000847-Rodnina1" id="pone.0000847-Rodnina1"></a>Rodnina MV, Wintermeyer W (2001) Fidelity of aminoacyl-tRNA selection on the ribosome: kinetic and structural mechanisms. Annu Rev Biochem  70: 415–435.  <ul class="find" data-citedArticleID="987948" data-doi="10.1146/annurev.biochem.70.1.415"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.70.1.415" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fidelity+of+aminoacyl-tRNA+selection+on+the+ribosome%3A+kinetic+and+structural+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fidelity+of+aminoacyl-tRNA+selection+on+the+ribosome%3A+kinetic+and+structural+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000847-Kramer1" id="pone.0000847-Kramer1"></a>Kramer EB, Farabaugh PJ (2007) The frequency of translational misreading errors in E. coli is largely determined by tRNA competition. RNA  13: 87–96.  <ul class="find" data-citedArticleID="987914" data-doi="10.1261/rna.294907"><li><a href="http://dx.doi.org/10.1261/rna.294907" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+frequency+of+translational+misreading+errors+in+E.+coli+is+largely+determined+by+tRNA+competition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+frequency+of+translational+misreading+errors+in+E.+coli+is+largely+determined+by+tRNA+competition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000847-Atkins1" id="pone.0000847-Atkins1"></a>Atkins JF, Weiss RB, Thompson S, Gesteland RF (1991) Towards a genetic dissection of the basis of triplet decoding, and its natural subversion: programmed reading frame shifts and hops. Annu Rev Genet  25: 201–228.  <ul class="find" data-citedArticleID="987860" data-doi="10.1146/annurev.ge.25.120191.001221"><li><a href="http://dx.doi.org/10.1146/annurev.ge.25.120191.001221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+genetic+dissection+of+the+basis+of+triplet+decoding%2C+and+its+natural+subversion%3A+programmed+reading+frame+shifts+and+hops." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+genetic+dissection+of+the+basis+of+triplet+decoding%2C+and+its+natural+subversion%3A+programmed+reading+frame+shifts+and+hops.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000847-Freistroffer1" id="pone.0000847-Freistroffer1"></a>Freistroffer DV, Kwiatkowski M, Buckingham RH, Ehrenberg M (2000) The accuracy of codon recognition by polypeptide release factors. Proc Natl Acad Sci U S A  97: 2046–2051.  <ul class="find" data-citedArticleID="987898" data-doi="10.1073/pnas.030541097"><li><a href="http://dx.doi.org/10.1073/pnas.030541097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+accuracy+of+codon+recognition+by+polypeptide+release+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+accuracy+of+codon+recognition+by+polypeptide+release+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000847-Princiotta1" id="pone.0000847-Princiotta1"></a>Princiotta MF, Finzi D, Qian SB, Gibbs J, Schuchmann S, et al.  (2003) Quantitating protein synthesis, degradation, and endogenous antigen processing. Immunity  18: 343–354.  <ul class="find" data-citedArticleID="987940" data-doi="10.1016/s1074-7613(03)00051-7"><li><a href="http://dx.doi.org/10.1016/s1074-7613(03)00051-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitating+protein+synthesis%2C+degradation%2C+and+endogenous+antigen+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitating+protein+synthesis%2C+degradation%2C+and+endogenous+antigen+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000847-Pinheiro1" id="pone.0000847-Pinheiro1"></a>Pinheiro M, Afreixo V, Moura G, Freitas A, Santos MA, et al.  (2006) Statistical, computational and visualization methodologies to unveil gene primary structure features. Methods Inf Med  45: 163–168.  <ul class="find" data-citedArticleID="987938"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical%2C+computational+and+visualization+methodologies+to+unveil+gene+primary+structure+features.&amp;auth=&amp;atitle=Statistical%2C+computational+and+visualization+methodologies+to+unveil+gene+primary+structure+features." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical%2C+computational+and+visualization+methodologies+to+unveil+gene+primary+structure+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical%2C+computational+and+visualization+methodologies+to+unveil+gene+primary+structure+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000847-Campbell1" id="pone.0000847-Campbell1"></a>Campbell A, Mrazek J, Karlin S (1999) Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA. Proc Natl Acad Sci U S A  96: 9184–9189.  <ul class="find" data-citedArticleID="987878" data-doi="10.1073/pnas.96.16.9184"><li><a href="http://dx.doi.org/10.1073/pnas.96.16.9184" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+signature+comparisons+among+prokaryote%2C+plasmid%2C+and+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+signature+comparisons+among+prokaryote%2C+plasmid%2C+and+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000847-Nakashima1" id="pone.0000847-Nakashima1"></a>Nakashima H, Nishikawa K, Ooi T (1997) Differences in dinucleotide frequencies of human, yeast, and Escherichia coli genes. DNA Res  4: 185–192.  <ul class="find" data-citedArticleID="987928" data-doi="10.1093/dnares/4.3.185"><li><a href="http://dx.doi.org/10.1093/dnares/4.3.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Differences+in+dinucleotide+frequencies+of+human%2C+yeast%2C+and+Escherichia+coli+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Differences+in+dinucleotide+frequencies+of+human%2C+yeast%2C+and+Escherichia+coli+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000847-Hooper1" id="pone.0000847-Hooper1"></a>Hooper SD, Berg OG (2002) Detection of genes with atypical nucleotide sequence in microbial genomes. J Mol Evol  54: 365–375.  <ul class="find" data-citedArticleID="987906" data-doi="10.1007/s00239-001-0051-8"><li><a href="http://dx.doi.org/10.1007/s00239-001-0051-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detection+of+genes+with+atypical+nucleotide+sequence+in+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detection+of+genes+with+atypical+nucleotide+sequence+in+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000847-Hooper2" id="pone.0000847-Hooper2"></a>Hooper SD, Berg OG (2000) Gradients in nucleotide and codon usage along Escherichia coli genes. Nucleic Acids Res  28: 3517–3523.  <ul class="find" data-citedArticleID="987908" data-doi="10.1093/nar/28.18.3517"><li><a href="http://dx.doi.org/10.1093/nar/28.18.3517" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gradients+in+nucleotide+and+codon+usage+along+Escherichia+coli+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gradients+in+nucleotide+and+codon+usage+along+Escherichia+coli+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000847-Stahl1" id="pone.0000847-Stahl1"></a>Stahl G, McCarty GP, Farabaugh PJ (2002) Ribosome structure: revisiting the connection between translational accuracy and unconventional decoding. Trends Biochem Sci  27: 178–183.  <ul class="find" data-citedArticleID="987954" data-doi="10.1016/s0968-0004(02)02064-9"><li><a href="http://dx.doi.org/10.1016/s0968-0004(02)02064-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosome+structure%3A+revisiting+the+connection+between+translational+accuracy+and+unconventional+decoding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosome+structure%3A+revisiting+the+connection+between+translational+accuracy+and+unconventional+decoding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000847-Dix1" id="pone.0000847-Dix1"></a>Dix DB, Thompson RC (1989) Codon choice and gene expression: synonymous codons differ in translational accuracy. Proc Natl Acad Sci U S A  86: 6888–6892.  <ul class="find" data-citedArticleID="987892" data-doi="10.1073/pnas.86.18.6888"><li><a href="http://dx.doi.org/10.1073/pnas.86.18.6888" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+choice+and+gene+expression%3A+synonymous+codons+differ+in+translational+accuracy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+choice+and+gene+expression%3A+synonymous+codons+differ+in+translational+accuracy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000847-Gutman1" id="pone.0000847-Gutman1"></a>Gutman GA, Hatfield GW (1989) Nonrandom utilization of codon pairs in Escherichia coli. Proc Natl Acad Sci U S A  86: 3699–3703.  <ul class="find" data-citedArticleID="987902" data-doi="10.1073/pnas.86.10.3699"><li><a href="http://dx.doi.org/10.1073/pnas.86.10.3699" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonrandom+utilization+of+codon+pairs+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonrandom+utilization+of+codon+pairs+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000847-Buchan1" id="pone.0000847-Buchan1"></a>Buchan JR, Aucott LS, Stansfield I (2006) tRNA properties help shape codon pair preferences in open reading frames. Nucleic Acids Res  34: 1015–1027.  <ul class="find" data-citedArticleID="987870" data-doi="10.1093/nar/gkj488"><li><a href="http://dx.doi.org/10.1093/nar/gkj488" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNA+properties+help+shape+codon+pair+preferences+in+open+reading+frames." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNA+properties+help+shape+codon+pair+preferences+in+open+reading+frames.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000847-Rocha1" id="pone.0000847-Rocha1"></a>Rocha EP, Danchin A, Viari A (1999) Universal replication biases in bacteria. Mol Microbiol  32: 11–16.  <ul class="find" data-citedArticleID="987944" data-doi="10.1046/j.1365-2958.1999.01334.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01334.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Universal+replication+biases+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Universal+replication+biases+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000847-Grantham1" id="pone.0000847-Grantham1"></a>Grantham R, Gautier C, Gouy M, Mercier R, Pave A (1980) Codon catalog usage and the genome hypothesis. Nucleic Acids Res  8: r49–r62.  <ul class="find" data-citedArticleID="987900" data-doi="10.1093/nar/8.1.197-c"><li><a href="http://dx.doi.org/10.1093/nar/8.1.197-c" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+catalog+usage+and+the+genome+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+catalog+usage+and+the+genome+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000847-Buckingham2" id="pone.0000847-Buckingham2"></a>Buckingham RH (1990) Codon context. Experientia  46: 1126–1133.  <ul class="find" data-citedArticleID="987874" data-doi="10.1007/bf01936922"><li><a href="http://dx.doi.org/10.1007/bf01936922" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon+context." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon+context.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000847-McVean1" id="pone.0000847-McVean1"></a>McVean GAT, Hurst GDD (2000) Evolutionary lability of context-dependent codon bias in bacteria. J Mol Evol  50: 264–275.  <ul class="find" data-citedArticleID="987922"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolutionary+lability+of+context-dependent+codon+bias+in+bacteria.&amp;auth=&amp;atitle=Evolutionary+lability+of+context-dependent+codon+bias+in+bacteria." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+lability+of+context-dependent+codon+bias+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+lability+of+context-dependent+codon+bias+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000847-Lao1" id="pone.0000847-Lao1"></a>Lao PJ, Forsdyke DR (2000) Thermophilic bacteria strictly obey Szybalski's transcription direction rule and politely purine-load RNAs with both adenine and guanine. Genome Res  10: 228–236.  <ul class="find" data-citedArticleID="987916" data-doi="10.1101/gr.10.2.228"><li><a href="http://dx.doi.org/10.1101/gr.10.2.228" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thermophilic+bacteria+strictly+obey+Szybalski%27s+transcription+direction+rule+and+politely+purine-load+RNAs+with+both+adenine+and+guanine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thermophilic+bacteria+strictly+obey+Szybalski%27s+transcription+direction+rule+and+politely+purine-load+RNAs+with+both+adenine+and+guanine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000847-Kennedy1" id="pone.0000847-Kennedy1"></a>Kennedy SP, Ng WV, Salzberg SL, Hood L, DasSarma S (2001) Understanding the adaptation of Halobacterium species NRC-1 to its extreme environment through computational analysis of its genome sequence. Genome Res  11: 1641–1650.  <ul class="find" data-citedArticleID="987912" data-doi="10.1101/gr.190201"><li><a href="http://dx.doi.org/10.1101/gr.190201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+the+adaptation+of+Halobacterium+species+NRC-1+to+its+extreme+environment+through+computational+analysis+of+its+genome+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+the+adaptation+of+Halobacterium+species+NRC-1+to+its+extreme+environment+through+computational+analysis+of+its+genome+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000847-Tekaia1" id="pone.0000847-Tekaia1"></a>Tekaia F, Yeramian E, Dujon B (2002) Amino acid composition of genomes, lifestyles of organisms, and evolutionary trends: a global picture with correspondence analysis. Gene  297: 51–60.  <ul class="find" data-citedArticleID="987958" data-doi="10.1016/s0378-1119(02)00871-5"><li><a href="http://dx.doi.org/10.1016/s0378-1119(02)00871-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+composition+of+genomes%2C+lifestyles+of+organisms%2C+and+evolutionary+trends%3A+a+global+picture+with+correspondence+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+composition+of+genomes%2C+lifestyles+of+organisms%2C+and+evolutionary+trends%3A+a+global+picture+with+correspondence+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000847-Beutler1" id="pone.0000847-Beutler1"></a>Beutler E, Gelbart T, Han JH, Koziol JA, Beutler B (1989) Evolution of the genome and the genetic code: selection at the dinucleotide level by methylation and polyribonucleotide cleavage. Proc Natl Acad Sci U S A  86: 192–196.  <ul class="find" data-citedArticleID="987864" data-doi="10.1073/pnas.86.1.192"><li><a href="http://dx.doi.org/10.1073/pnas.86.1.192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+the+genome+and+the+genetic+code%3A+selection+at+the+dinucleotide+level+by+methylation+and+polyribonucleotide+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+the+genome+and+the+genetic+code%3A+selection+at+the+dinucleotide+level+by+methylation+and+polyribonucleotide+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000847-Nakashima2" id="pone.0000847-Nakashima2"></a>Nakashima H, Ota M, Nishikawa K, Ooi T (1998) Genes from nine genomes are separated into their organisms in the dinucleotide composition space. DNA Res  5: 251–259.  <ul class="find" data-citedArticleID="987930" data-doi="10.1093/dnares/5.5.251"><li><a href="http://dx.doi.org/10.1093/dnares/5.5.251" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+from+nine+genomes+are+separated+into+their+organisms+in+the+dinucleotide+composition+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+from+nine+genomes+are+separated+into+their+organisms+in+the+dinucleotide+composition+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000847-Marck1" id="pone.0000847-Marck1"></a>Marck C, Grosjean H (2002) tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features. RNA  8: 1189–1232.  <ul class="find" data-citedArticleID="987920" data-doi="10.1017/s1355838202022021"><li><a href="http://dx.doi.org/10.1017/s1355838202022021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=tRNomics%3A+analysis+of+tRNA+genes+from+50+genomes+of+Eukarya%2C+Archaea%2C+and+Bacteria+reveals+anticodon-sparing+strategies+and+domain-specific+features." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22tRNomics%3A+analysis+of+tRNA+genes+from+50+genomes+of+Eukarya%2C+Archaea%2C+and+Bacteria+reveals+anticodon-sparing+strategies+and+domain-specific+features.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000847-Crick1" id="pone.0000847-Crick1"></a>Crick FH (1966) Codon–anticodon pairing: the wobble hypothesis. J Mol Biol  19: 548–555.  <ul class="find" data-citedArticleID="987886" data-doi="10.1016/s0022-2836(66)80022-0"><li><a href="http://dx.doi.org/10.1016/s0022-2836(66)80022-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon%26%23x2013%3Banticodon+pairing%3A+the+wobble+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon%26%23x2013%3Banticodon+pairing%3A+the+wobble+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000847-Caburet1" id="pone.0000847-Caburet1"></a>Caburet S, Vaiman D, Veitia RA (2004) A genomic basis for the evolution of vertebrate transcription factors containing amino Acid runs. Genetics  167: 1813–1820.  <ul class="find" data-citedArticleID="987876" data-doi="10.1534/genetics.104.029082"><li><a href="http://dx.doi.org/10.1534/genetics.104.029082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+basis+for+the+evolution+of+vertebrate+transcription+factors+containing+amino+Acid+runs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+basis+for+the+evolution+of+vertebrate+transcription+factors+containing+amino+Acid+runs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000847-Borstnik1" id="pone.0000847-Borstnik1"></a>Borstnik B, Pumpernik D (2002) Tandem repeats in protein coding regions of primate genes. Genome Res  12: 909–915.  <ul class="find" data-citedArticleID="987866" data-doi="10.1101/gr.138802"><li><a href="http://dx.doi.org/10.1101/gr.138802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tandem+repeats+in+protein+coding+regions+of+primate+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tandem+repeats+in+protein+coding+regions+of+primate+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000847-Rocha2" id="pone.0000847-Rocha2"></a>Rocha EP, Matic I, Taddei F (2002) Over-representation of repeats in stress response genes: a strategy to increase versatility under stressful conditions? Nucleic Acids Res  30: 1886–1894.  <ul class="find" data-citedArticleID="987946" data-doi="10.1093/nar/30.9.1886"><li><a href="http://dx.doi.org/10.1093/nar/30.9.1886" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Over-representation+of+repeats+in+stress+response+genes%3A+a+strategy+to+increase+versatility+under+stressful+conditions%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Over-representation+of+repeats+in+stress+response+genes%3A+a+strategy+to+increase+versatility+under+stressful+conditions%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000847-Haberman1" id="pone.0000847-Haberman1"></a>Haberman S (1973) Analysis of residuals in cross-classified tables. Biometrics  29: 205–220.  <ul class="find" data-citedArticleID="987904" data-doi="10.2307/2529686"><li><a href="http://dx.doi.org/10.2307/2529686" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+residuals+in+cross-classified+tables." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+residuals+in+cross-classified+tables.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000847-Simenoff1" id="pone.0000847-Simenoff1"></a>Simenoff JS (2003) Analyzing categorical data. New York: Springer-Verlag.   <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.XML" value="130706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.PDF" value="502981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g001.PNG_L" value="362360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g001.PNG_M" value="173008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g001.PNG_S" value="16700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g001.TIF" value="1029706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g001.PNG_I" value="111856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g002.PNG_L" value="350104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g002.PNG_M" value="184897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g002.PNG_S" value="15098"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g002.TIF" value="1101740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g002.PNG_I" value="101609"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t001.PNG_L" value="274777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t001.PNG_M" value="217969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t001.PNG_S" value="25542"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t001.TIF" value="1118496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t001.PNG_I" value="182786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t002.PNG_L" value="254966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t002.PNG_M" value="196434"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t002.PNG_S" value="24942"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t002.TIF" value="1106062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.t002.PNG_I" value="171201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g003.PNG_L" value="1186080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g003.PNG_M" value="331268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g003.PNG_S" value="14640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g003.TIF" value="2964146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g003.PNG_I" value="135608"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g004.PNG_L" value="105332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g004.PNG_M" value="166270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g004.PNG_S" value="18499"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g004.TIF" value="663930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g004.PNG_I" value="91946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g005.PNG_L" value="777366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g005.PNG_M" value="275069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g005.PNG_S" value="19092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g005.TIF" value="1682754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g005.PNG_I" value="169842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g006.PNG_L" value="771111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g006.PNG_M" value="262171"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g006.PNG_S" value="16322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g006.TIF" value="1624372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.g006.PNG_I" value="127741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.e001.PNG" value="8945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.e001.TIF" value="24844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s001.TIF" value="4328459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s002.TIF" value="542247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s003.TIF" value="541996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s004.TIF" value="541996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s005.TIF" value="4328087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s006.TIF" value="4327808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s007.TIF" value="4327808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s008.TIF" value="4327808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s009.TIF" value="4327808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s010.TIF" value="4327808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s011.TIF" value="2163704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s012.TIF" value="2163704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s013.TIF" value="2163704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s014.TIF" value="2163704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s015.TIF" value="114384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s016.TIF" value="574158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s017.TIF" value="549326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000847.s018.XLS" value="320512"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000847&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000847" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000847&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000847&volume=&issue=&title=Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure&author_name=Gabriela%20Moura%2C%20Miguel%20Pinheiro%2C%20Joel%20Arrais%2C%20Ana%20Cristina%20Gomes%2C%20Laura%20Carreto%2C%20Adelaide%20Freitas%2C%20Jos%C3%A9%20L.%20Oliveira%2C%20Manuel%20A.%20S.%20Santos&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000847" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847&amp;t=Large%20Scale%20Comparative%20Codon-Pair%20Context%20Analysis%20Unveils%20General%20Rules%20that%20Fine-Tune%20Evolution%20of%20mRNA%20Primary%20Structure" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847&title=Large%20Scale%20Comparative%20Codon-Pair%20Context%20Analysis%20Unveils%20General%20Rules%20that%20Fine-Tune%20Evolution%20of%20mRNA%20Primary%20Structure&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847&amp;title=Large%20Scale%20Comparative%20Codon-Pair%20Context%20Analysis%20Unveils%20General%20Rules%20that%20Fine-Tune%20Evolution%20of%20mRNA%20Primary%20Structure" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000847&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Large Scale Comparative Codon-Pair Context Analysis Unveils General Rules that Fine-Tune Evolution of mRNA Primary Structure';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000847';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Large%20Scale%20Comparative%20Codon-Pair%20Context%20Analysis%20Unveils%20General%20Rules%20that%20Fine-Tune%20Evolution%20of%20mRNA%20Primary%20Structure http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000847" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000847" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Archaean+biology%22" title="Search for articles in the subject area:'Archaean biology'"><div class="flagText">Archaean biology</div></a>
              <div data-categoryid="43097" data-articleid="24532"
                   data-categoryname="Archaean biology"
                   class="flagImage" title="Flag 'Archaean biology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="24532"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="24532"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+methylation%22" title="Search for articles in the subject area:'DNA methylation'"><div class="flagText">DNA methylation</div></a>
              <div data-categoryid="32275" data-articleid="24532"
                   data-categoryname="DNA methylation"
                   class="flagImage" title="Flag 'DNA methylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Eubacterium%22" title="Search for articles in the subject area:'Eubacterium'"><div class="flagText">Eubacterium</div></a>
              <div data-categoryid="42793" data-articleid="24532"
                   data-categoryname="Eubacterium"
                   class="flagImage" title="Flag 'Eubacterium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24532"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Messenger+RNA%22" title="Search for articles in the subject area:'Messenger RNA'"><div class="flagText">Messenger RNA</div></a>
              <div data-categoryid="18353" data-articleid="24532"
                   data-categoryname="Messenger RNA"
                   class="flagImage" title="Flag 'Messenger RNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+translation%22" title="Search for articles in the subject area:'Protein translation'"><div class="flagText">Protein translation</div></a>
              <div data-categoryid="24691" data-articleid="24532"
                   data-categoryname="Protein translation"
                   class="flagImage" title="Flag 'Protein translation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4318'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1385'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8123&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=12947">PLoS ONE Group</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
