

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001815"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001815" />

    <meta name="citation_title" content="The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course"/>
    <meta itemprop="name" content="The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course"/>

      <meta name="citation_author" content="Ingrid B. Müller"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany"/>
      <meta name="citation_author" content="Julia Knöckel"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany"/>
      <meta name="citation_author" content="Matthew R. Groves"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory-Hamburg Outstation, Hamburg, Germany"/>
      <meta name="citation_author" content="Rositsa Jordanova"/>
            <meta name="citation_author_institution" content="European Molecular Biology Laboratory-Hamburg Outstation, Hamburg, Germany"/>
      <meta name="citation_author" content="Steven E. Ealick"/>
            <meta name="citation_author_institution" content="Department of Chemistry and Chemical Biology, Cornell University, Ithaca, New York, United States of America"/>
      <meta name="citation_author" content="Rolf D. Walter"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany"/>
      <meta name="citation_author" content="Carsten Wrenger"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany"/>

    <meta name="citation_date" content="2008/3/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001815.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1815"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A genomic overview of pyridoxal-phosphate-dependent enzymes.; citation_author=R Percudani; citation_author=A Peracchi; citation_journal_title=EMBO Rep; citation_volume=4; citation_number=1; citation_pages=850-854; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Tpn1p, the plasma membrane vitamin B6 transporter of Saccharomyces cerevisiae.; citation_author=J Stolz; citation_author=M Vielreicher; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=2; citation_pages=18990-18996; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Brain pyridoxal kinase. Purification and characterization.; citation_author=JA Kerry; citation_author=M Rohde; citation_author=F Kwok; citation_journal_title=Eur J Biochem; citation_volume=158; citation_number=3; citation_pages=581-585; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Physical and enzymological interaction of Bacillus subtilis proteins required for de novo pyridoxal 5-phosphate biosynthesis.; citation_author=BR Belitsky; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=4; citation_pages=1191-1196; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Characterization of the products of the genes SNO1 and SNZ1 involved in pyridoxine synthesis in Saccharomyces cerevisiae.; citation_author=YX Dong; citation_author=S Sueda; citation_author=J Nikawa; citation_author=H Kondo; citation_journal_title=Eur J Biochem; citation_volume=271; citation_number=5; citation_pages=745-752; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analysis of the vitamin B6 biosynthesis pathway in the human malaria parasite Plasmodium falciparum.; citation_author=C Wrenger; citation_author=ML Eschbach; citation_author=IB Müller; citation_author=D Warnecke; citation_author=RD Walter; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=6; citation_pages=5242-5248; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reconstitution and biochemical characterization of a new pyridoxal-5-phosphate biosynthetic pathway.; citation_author=KE Burns; citation_author=Y Xiang; citation_author=CL Kinsland; citation_author=FW McLaffert; citation_author=TP Begley; citation_journal_title=J Am Chem Soc; citation_volume=127; citation_number=7; citation_pages=3682-3683; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the two components of pyridoxal 5-phosphate synthase from Bacillus subtilis.; citation_author=T Raschle; citation_author=N Amrhein; citation_author=TB Fitzpatrick; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=8; citation_pages=32291-32300; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Vitamin B6 biosynthesis in higher plants.; citation_author=M Tambasco-Studart; citation_author=O Titiz; citation_author=T Raschle; citation_author=G Forster; citation_author=N Amrhein; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=9; citation_pages=13687-13692; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Suppression of insertions in the complex pdxJ operon of Escherichia coli K-12 by lon and other mutations.; citation_author=HM Lam; citation_author=E Tancula; citation_author=WB Dempsey; citation_author=ME Winkler; citation_journal_title=J Bacteriol; citation_volume=174; citation_number=10; citation_pages=1554-1567; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Characterization of the complex pdxH-tyrS operon of Escherichia coli K-12 and pleiotropic phenotypes caused by pdxH insertion mutations.; citation_author=HM Lam; citation_author=ME Winkler; citation_journal_title=J Bacteriol; citation_volume=174; citation_number=11; citation_pages=6033-6045; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Three-dimensional structure of YaaE from Bacillus subtilis, a glutaminase implicated in pyridoxal-5-phosphate biosynthesis.; citation_author=JA Bauer; citation_author=EM Bennett; citation_author=TP Begley; citation_author=SE Ealick; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=12; citation_pages=2704-2711; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A new arrangement of (β/α)8 barrels in the synthase subunit of PLP synthase.; citation_author=J Zhu; citation_author=JW Burgner; citation_author=E Harms; citation_author=BR Belitsky; citation_author=JL Smith; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=13; citation_pages=27914-27923; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Vitamin B6 biosynthesis by the malaria parasite Plasmodium falciparum: biochemical and structural insights.; citation_author=M Gengenbacher; citation_author=TB Fitzpatrick; citation_author=T Raschle; citation_author=K Flicker; citation_author=I Sinning; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=14; citation_pages=3633-3641; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structural insights into the mechanism of the PLP synthase holoenzyme from Thermotoga maritima.; citation_author=F Zein; citation_author=Y Zhang; citation_author=YN Kang; citation_author=K Burns; citation_author=TP Begley; citation_journal_title=Biochemistry; citation_volume=45; citation_number=15; citation_pages=14609-14620; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Structure of a bacterial pyridoxal 5′-phosphate synthase complex.; citation_author=M Strohmeier; citation_author=T Raschle; citation_author=J Mazurkiewicz; citation_author=K Rippe; citation_author=I Sinning; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=103; citation_number=16; citation_pages=19284-19289; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Malaria: new developments in treatment and prevention.; citation_author=P Newton; citation_author=N White; citation_journal_title=Annu Rev Med; citation_volume=50; citation_number=17; citation_pages=179-192; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A highly conserved sequence is a novel gene involved in de novo vitamin B6 biosynthesis.; citation_author=M Ehrenshaft; citation_author=P Bilski; citation_author=MY Li; citation_author=CF Chignell; citation_author=ME Daub; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=96; citation_number=18; citation_pages=9374-9378; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The extremely conserved pyroA gene of Aspergillus nidulans is required for pyridoxine synthesis and is required indirectly for resistance to photosensitizers.; citation_author=AH Osmani; citation_author=GS May; citation_author=SA Osmani; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=19; citation_pages=23565-23569; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Overlap between pdxA and ksgA in the complex pdxA-ksgA-apaG-apaH operon of Escherichia coli K-12.; citation_author=BB Roa; citation_author=DM Connolly; citation_author=ME Winkler; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=20; citation_pages=4767-4777; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The biogenetic anatomy of vitamin B6. A 13C NMR investigation of the biosynthesis of pyridoxol in Escherichia coli.; citation_author=RE Hill; citation_author=K Himmeldirk; citation_author=IA Kennedy; citation_author=RM Pauloski; citation_author=BG Sayer; citation_journal_title=J Biol Chem; citation_volume=271; citation_number=21; citation_pages=30426-30435; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=SOR1, a gene required for photosensitizer and singlet oxygen resistance in Cercospora fungi, is highly conserved in divergent organisms.; citation_author=M Ehrenshaft; citation_author=AE Jenns; citation_author=KR Chung; citation_author=ME Daub; citation_journal_title=Mol Cell; citation_volume=1; citation_number=22; citation_pages=603-609; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The apicomplexan parasite Toxoplasma gondii generates pyridoxal phosphate de novo.; citation_author=J Knöckel; citation_author=IB Müller; citation_author=B Bergmann; citation_author=RD Walter; citation_author=C Wrenger; citation_journal_title=Mol Biochem Parasitol; citation_volume=152; citation_number=23; citation_pages=108-111; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Occurrence of pydixaminephosphate oxidase and pyridoxal kinase in Gram-negative and Gram-positive bacteria.; citation_author=W Pflug; citation_author=F Lingens; citation_journal_title=Hoppe Seylers Z Physiol Chem; citation_volume=364; citation_number=24; citation_pages=1627-1630; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Reaction mechanism of pyridoxal 5′-phosphate synthase: Detection of an enzyme bound chromophoric intermediate.; citation_author=T Raschle; citation_author=D Arigoni; citation_author=R Brunisholz; citation_author=N Amrhein; citation_author=TB Fitzpatrick; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=25; citation_pages=6098-6105; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=EF Fritsch; citation_author=T Maniatis; citation_number=26; citation_date=1989; citation_publisher=Cold Spring Harbor Laboratory; " />
      <meta name="citation_reference" content="citation_title=A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding.; citation_author=MM Bradford; citation_journal_title=Anal Biochem; citation_volume=72; citation_number=27; citation_pages=248-254; citation_date=1976; " />
      <meta name="citation_reference" content="citation_title=The Plasmodium falciparum bifunctional ornithine decarboxylase, S-adenosyl-L-methionine decarboxylase, enables a well balanced polyamine synthesis without domain-domain interaction.; citation_author=C Wrenger; citation_author=K Lüersen; citation_author=T Krause; citation_author=S Müller; citation_author=RD Walter; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=28; citation_pages=29651-29656; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The impact of protein characterization in structural proteomics.; citation_author=A Geerlof; citation_author=J Brown; citation_author=B Coutard; citation_author=MP Egloff; citation_author=FJ Enguita; citation_journal_title=Acta Crystallogr D Biol Crystallogr; citation_volume=62; citation_number=29; citation_pages=1125-1136; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SWISS-MODEL: An automated protein homology-modeling server.; citation_author=T Schwede; citation_author=J Kopp; citation_author=N Guex; citation_author=MC Peitsch; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=30; citation_pages=3381-3385; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=D Higgins; citation_author=J Thompson; citation_author=T Gibson; citation_author=JD Thompson; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Research; citation_volume=22; citation_number=31; citation_pages=4673-4680; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course"/>
    <meta name="twitter:description" content="BackgroundPlants, fungi, bacteria and the apicomplexan parasite Plasmodium falciparum are able to synthesize vitamin B6 de novo, whereas mammals depend upon the uptake of this essential nutrient from their diet. The active form of vitamin B6 is pyridoxal 5-phosphate (PLP). For its synthesis two enzymes, Pdx1 and Pdx2, act together, forming a multimeric complex consisting of 12 Pdx1 and 12 Pdx2 protomers.Methodology/Principal FindingsHere we report amino acid residues responsible for stabilization of the structural and enzymatic integrity of the plasmodial PLP synthase, identified by using distinct mutational analysis and biochemical approaches. Residues R85, H88 and E91 (RHE) are located at the Pdx1:Pdx1 interface and play an important role in Pdx1 complex assembly. Mutation of these residues to alanine impedes both Pdx1 activity and Pdx2 binding. Furthermore, changing D26, K83 and K151 (DKK), amino acids from the active site of Pdx1, to alanine obstructs not only enzyme activity but also formation of the complex. In contrast to the monomeric appearance of the RHE mutant, alteration of the DKK residues results in a hexameric assembly, and does not affect Pdx2 binding or its activity. While the modelled position of K151 is distal to the Pdx1:Pdx1 interface, it affects the assembly of hexameric Pdx1 into a functional dodecamer, which is crucial for PLP synthesis.Conclusions/SignificanceTaken together, our data suggest that the assembly of a functional Pdx1:Pdx2 complex follows a defined pathway and that inhibition of this assembly results in an inactive holoenzyme."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001815.g006"/>

  <meta property="og:title" content="The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9735'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6666'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7301&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001815">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001815" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001815&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ingrid B. Müller
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ingrid B. Müller, Julia Knöckel</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Julia Knöckel
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ingrid B. Müller, Julia Knöckel</p>

              
              

                <p>Affiliation:
                  Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew R. Groves, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  European Molecular Biology Laboratory-Hamburg Outstation, Hamburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rositsa Jordanova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  European Molecular Biology Laboratory-Hamburg Outstation, Hamburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Steven E. Ealick, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Chemistry and Chemical Biology, Cornell University, Ithaca, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rolf D. Walter, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carsten Wrenger
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:wrenger@bni-hamburg.de">wrenger@bni-hamburg.de</a></p>

                <p>Affiliation:
                  Department of Biochemistry, Bernhard Nocht Institute for Tropical Medicine, Hamburg, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 19, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001815</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001815-g001" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-g002" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-g003" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-g004" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-t001" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-g005" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-g006" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001815-t002" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001815">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001815" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Plants, fungi, bacteria and the apicomplexan parasite <em>Plasmodium falciparum</em> are able to synthesize vitamin B6 <em>de novo</em>, whereas mammals depend upon the uptake of this essential nutrient from their diet. The active form of vitamin B6 is pyridoxal 5-phosphate (PLP). For its synthesis two enzymes, Pdx1 and Pdx2, act together, forming a multimeric complex consisting of 12 Pdx1 and 12 Pdx2 protomers.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we report amino acid residues responsible for stabilization of the structural and enzymatic integrity of the plasmodial PLP synthase, identified by using distinct mutational analysis and biochemical approaches. Residues R85, H88 and E91 (RHE) are located at the Pdx1:Pdx1 interface and play an important role in Pdx1 complex assembly. Mutation of these residues to alanine impedes both Pdx1 activity and Pdx2 binding. Furthermore, changing D26, K83 and K151 (DKK), amino acids from the active site of Pdx1, to alanine obstructs not only enzyme activity but also formation of the complex. In contrast to the monomeric appearance of the RHE mutant, alteration of the DKK residues results in a hexameric assembly, and does not affect Pdx2 binding or its activity. While the modelled position of K151 is distal to the Pdx1:Pdx1 interface, it affects the assembly of hexameric Pdx1 into a functional dodecamer, which is crucial for PLP synthesis.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Taken together, our data suggest that the assembly of a functional Pdx1:Pdx2 complex follows a defined pathway and that inhibition of this assembly results in an inactive holoenzyme.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Müller IB, Knöckel J, Groves MR, Jordanova R, Ealick SE, et al.  (2008) The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course. PLoS ONE 3(3):
          e1815.
            doi:10.1371/journal.pone.0001815</p><p><strong>Editor: </strong>Haiwei Song, Institute of Molecular and Cell Biology, Singapore</p><p><strong>Received:</strong> November 15, 2007; <strong>Accepted:</strong> February 14, 2008; <strong>Published:</strong> March 19, 2008</p><p><strong>Copyright:</strong> © 2008 Müller et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by Deutsche Forschungsgemeinschaft Grant WA 395/15.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The active form of vitamin B6 is pyridoxal 5-phosphate (PLP), which is an essential cofactor for more than 100 enzymes and thereby involved in catalytic reactions such as amino acid decarboxylation, elimination and amino-transfer <a href="#pone.0001815-Percudani1">[1]</a>. PLP is synthesized <em>de novo</em> by plants, almost all bacteria and fungi; however, mammals depend entirely on the uptake of this indispensable nutrient from their diet. As shown for yeast, the non-phosphorylated inactive cofactor is imported via specific transporters and finally trapped within the cell by phosphorylation catalyzed by pyridoxal kinase (PdxK) <a href="#pone.0001815-Stolz1">[2]</a>, <a href="#pone.0001815-Kerry1">[3]</a>. Thus, the dual provision of PLP by <em>de novo</em> synthesis and salvage indicates necessity and importance of this cofactor for the survival of yeast and other organisms.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Currently two different pathways for the biosynthesis of PLP are known. The <em>E. coli</em>- (DOXP-dependent) creates from the substrates 4-phosphohydroxy-L-threonine, 1-deoxyxylulose 5-phosphate (DOXP) and glutamate pyridoxine. In contrast the fungi-like- (DOXP-independent) pathway, which has been firstly described in the fungus <em>Cercospora nicotianae</em>, synthesises the active cofactor PLP from ribose 5-phosphate, glyceraldehyde 3-phosphate and glutamine by an enzyme complex consisting of two proteins - Pdx1 and Pdx2 <a href="#pone.0001815-Belitsky1">[4]</a>–<a href="#pone.0001815-Lam2">[11]</a>. The Pdx2 protein exhibits glutaminase activity and delivers ammonia to Pdx1 <a href="#pone.0001815-Dong1">[5]</a>, <a href="#pone.0001815-Wrenger1">[6]</a> (<a href="#pone-0001815-g001">Fig. 1</a>). The crystal structures of Pdx1 (YaaD or PdxS), and Pdx2 (YaaE or PdxT) from various organisms have been analyzed <a href="#pone.0001815-Bauer1">[12]</a>–<a href="#pone.0001815-Gengenbacher1">[14]</a>. Only recently the structures of the B6 biosynthesis complex (PLP synthase) from <em>T. maritima</em> and <em>B. subtilis</em> have been solved. Pdx1 assembles into a dodecamer, consisting of two hexameric crowns, each decorated by six Pdx2 molecules <a href="#pone.0001815-Zein1">[15]</a>, <a href="#pone.0001815-Strohmeier1">[16]</a>.</p>
<div class="figure" id="pone-0001815-g001"><div class="img"><a name="pone-0001815-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Reaction scheme of the Pdx1 and Pdx2 proteins.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>The <em>Pf</em>Pdx2 enzyme depends on its interaction with <em>Pf</em>Pdx1 for glutaminase activity. The Pdx1 enzyme is responsible for the <em>de novo</em> synthesis of PLP by utilizing the substrates ribose 5-phosphate, glyceraldehyde 3-phosphate and ammonia, whereas the latter is provided by deaminating of glutamine by Pdx2 via substrate channelling to the attached <em>Pf</em>Pdx1 protein. The reactions of each enzyme are indicated.</p>
<span>doi:10.1371/journal.pone.0001815.g001</span></div><a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we report on biochemical analyses of the interaction of the plasmodial Pdx1:Pdx1 interfaces as well as about effects on Pdx2 binding and catalysis. In contrast to the static nature of the Pdx1 crystal structures, this study allows functional insights into the behaviour of the highly dynamic plasmodial complex and suggests a possible path for its assembly. The amino acids that are crucial for the plasmodial PLP complex should be further exploited for the design of specific drugs which will be restricted to the malaria parasite and not harming the human host.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Malaria is one of the most serious infectious diseases in the world (WHO, Communicable Disease Report). Antimalarial drugs are losing more and more efficacy against the deadliest agent, <em>Plasmodium falciparum</em>. Since a vaccine is not available, an urgent need exists to identify novel targets for the development of new chemotherapeutics <a href="#pone.0001815-Newton1">[17]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Specific activity and mutagenic analyses of the active sites of Pdx1 and Pdx2</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The specific activity of the plasmodial vitamin B6 biosynthesis complex consisting of <em>Pf</em>Pdx1 and <em>Pf</em>Pdx2 (1:1 ratio) was determined to be 662±54 pmol min<sup>−1</sup> mg<sup>−1</sup> protein if ribose 5-phosphate, glyceraldehyde 3-phosphate and glutamine were used as substrates. This is about seven-fold higher than the specific activity previously observed for the plasmodial enzymes by Gengenbacher <em>et al</em>. <a href="#pone.0001815-Gengenbacher1">[14]</a> and might result from a different expression system and purification methods. In the presence of ammonium chloride instead of glutamine, the plasmodial Pdx1 protein alone revealed a specific activity of 746±76 pmol min<sup>−1</sup> mg<sup>−1</sup> protein. Because of the observed similar activity, analyses on the amino acid residues from the <em>Pf</em>Pdx1 active site as well as for the Pdx1:Pdx1 interface were carried out by the latter enzyme assay without <em>Pf</em>Pdx2.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Structural analysis of Pdx1 (PdxS) from <em>Geobacillus stearothermophilu</em>s suggested a participation of residues D24, K81 and K149 in the binding of ribulose 5-phosphate or catalysis <a href="#pone.0001815-Zhu1">[13]</a>. Subsequently the amino acids K149 from <em>B. subtilis</em> and K82 in the PLP synthase subunit (YaaD) from <em>Thermatoga maritima</em>, respectively, were observed to be covalently attached to ribulose 5-phosphate <a href="#pone.0001815-Burns1">[7]</a>, <a href="#pone.0001815-Zein1">[15]</a>. The homologues of these residues in Pdx1 of the plasmodial enzyme are D26, K83 and K151, respectively (<a href="#pone-0001815-g002">Fig. 2A</a>). These amino acid residues were substituted by alanine using site directed mutagenesis and the mutant proteins were analysed for enzymatic activity (<a href="#pone-0001815-g003">Fig. 3</a>). The <em>Pf</em>Pdx1 DKK (D26A/K83A/K151A) triple mutant enzyme as well as each individual mutant were inactive. The results confirmed that residues D26, K83 and K151 are important for <em>Pf</em>Pdx1 activity; however, a glutaminase assay with the <em>Pf</em>Pdx2 wild-type enzyme and the respective <em>Pf</em>Pdx1 showed that the ability of Pdx2 to hydrolyze glutamine was not affected with activities in the same range as previously reported for the wild-type enzymes <a href="#pone.0001815-Wrenger1">[6]</a> (<a href="#pone-0001815-t001">Table 1</a>). This indicates that the <em>Pf</em>Pdx1 DKK mutant still binds to <em>Pf</em>Pdx2. To verify the ability of the <em>Pf</em>Pdx1 DKK mutant to bind to the C-terminal 6× HIS-Tag harbouring <em>Pf</em>Pdx2HIS wild-type protein, co-purification analyses were performed (<a href="#pone-0001815-g004">Fig. 4A</a>). The results clearly demonstrate that modification of these active site residues of <em>Pf</em>Pdx1 does not affect its ability to interact with <em>Pf</em>Pdx2.</p>
<div class="figure" id="pone-0001815-g002"><div class="img"><a name="pone-0001815-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>(A) Homology model of one plasmodial Pdx1 monomer showing the analyzed amino acid residues as indicated.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>(B) The interface region between two <em>Pf</em>Pdx1 proteins within the same hexameric ring illustrating the amino acid residues R85, H88 and E91, which are involved in Pdx1:Pdx1 binding. The model was generated by Swiss-Model <a href="#pone.0001815-Schwede1">[30]</a> and visualised by PyMOL (<a href="http://www.pymol.org">www.pymol.org</a>).</p>
<span>doi:10.1371/journal.pone.0001815.g002</span></div><div class="figure" id="pone-0001815-g003"><div class="img"><a name="pone-0001815-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Alignment of the deduced amino acid sequences of the YaaD protein from <em>Thermotoga maritima</em> (<em>Tm,</em> AC: Q9WYU4), the Pdx1 protein from <em>Bacillus subtilis</em> (<em>Bs,</em> AC: NP_387892), the PdxS protein from <em>Geobacillus stearothermophilu</em>s (<em>Gs,</em> AC: Q5L3Y2) with the Pdx1 protein from <em>P. falciparum</em> (<em>Pf,</em> AC: XP_966196).</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>The alignment was performed by using CLUSTALW <a href="#pone.0001815-Higgins1">[31]</a>. Identical (*) and similar (:) amino acid residues are indicated below the protein sequence. Gaps (-) were introduced into the sequence to maximize homology and to compensate for the different chain lengths. Mutated amino acid residues, which are predicted to be participating within the active site and the additional phosphate binding site, are labelled above by (+) and (§), respectively. The exchanged amino acid residues, proposed to be involved in the Pdx1:Pdx1 interaction, are labelled above the sequence by (#).</p>
<span>doi:10.1371/journal.pone.0001815.g003</span></div><div class="figure" id="pone-0001815-g004"><div class="img"><a name="pone-0001815-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Western blot analysis of the co-purified <em>Pf</em>Pdx1 and <em>Pf</em>Pdx2HIS proteins.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Cell homogenates of the recombinant expression of the plasmodial Pdx1 protein and Pdx2HIS protein (6× His-tag instead of a Strep-tag) were mixed and subsequently purified via Strep-tag affinity chromatography of the <em>Pf</em>Pdx1 protein. The co-purification was visualized by Western blot analysis using a monoclonal anti-Strep antibody (IBA) and the HIS-probe-HRP (Pierce) as described in <a href="#s4">Material and Methods</a>. (A) Co-purification of the <em>Pf</em>Pdx1 wild-type (3) and the <em>Pf</em>Pdx1 DKK (D26A, K83A and K151A) triple mutant protein (4) with the <em>Pf</em>Pdx2HIS wild-type. As a control the <em>Pf</em>Pdx1 wild-type (WT) (1) and the <em>Pf</em>Pdx2HIS (2) single proteins were purified via Strep-tag affinity chromatography and applied in Western blot analysis. (B) Co-purification of the <em>Pf</em>Pdx1 ERR (5), <em>Pf</em>Pdx1 RHE (6) as well as the <em>Pf</em>Pdx1 H88A (7) and <em>Pf</em>Pdx1 E91A (8) mutant proteins with the plasmodial Pdx2HIS wild-type. (Note: The <em>Pf</em>Pdx1 R85A mutant was not recombinantly expressible.) (C) The gate-keeper <em>Pf</em>Pdx2HIS E53Y (9) and <em>Pf</em>Pdx2HIS R154W (10) mutants were co-purified with the <em>Pf</em>Pdx1 wild-type protein via its Strep-tag.</p>
<span>doi:10.1371/journal.pone.0001815.g004</span></div><div class="figure" id="pone-0001815-t001"><div class="img"><a name="pone-0001815-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Properties of <em>P. falciparum</em> mutated Pdx1 and Pdx2 proteins in comparison to the wild type.</span></strong></p><span>doi:10.1371/journal.pone.0001815.t001</span></div>

<h4>K151 is crucial for the dodecameric conformation of the plasmodial Pdx1</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The molecular mass of the wild-type <em>Pf</em>Pdx1 dodecamer is 424±12 kDa, as determined by static light scattering (SLS) (<a href="#pone-0001815-g005">Fig. 5</a>). These data indicate that the wild-type <em>Pf</em>Pdx1 assembles into a dodecameric structure (double crown formation), as has been shown for the counterparts in <em>T. maritima</em> and <em>B. subtilis</em> <a href="#pone.0001815-Zein1">15</a>, <a href="#pone.0001815-Strohmeier1">16</a>. Surprisingly, the double crown formation is inhibited in the <em>Pf</em>Pdx1 DKK mutant, which falls apart into two separate crowns, thereby revealing a hexameric structure with molecular mass of 238±5 kDa as determined by SLS (<a href="#pone-0001815-g005">Fig. 5</a>). In order to narrow down which amino acid residues of the <em>Pf</em>Pdx1 DKK triple mutant are involved in <em>Pf</em>Pdx1:<em>Pf</em>Pdx1 interactions, the three amino acids were mutated individually and the derived proteins were analysed by SLS. Mutation of the <em>Pf</em>Pdx1 D26 and K83, which are expected to lie within the active site, does not alter the assembly state of <em>Pf</em>Pdx1 wild-type protein; however the mutation of <em>Pf</em>Pdx1 K151 to alanine - as shown for the DKK triple mutant in <a href="#pone-0001815-g005">Figure 5</a> - destabilizes the <em>Pf</em>Pdx1 assembly (<a href="#pone-0001815-t001">Table 1</a>). The result indicates that this amino acid residue, previously suggested to participate in the formation of the active site of <em>Pf</em>Pdx1, is also involved in the formation of the dodecameric structure of Pdx1 in <em>P. falciparum</em>.</p>
<div class="figure" id="pone-0001815-g005"><div class="img"><a name="pone-0001815-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Static light scattering of the plasmodial Pdx1 DKK and RHE mutant proteins in comparison to the wild-type (WT).</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>In order to analyze the structural conformation of the <em>Pf</em>Pdx1 proteins, size exclusion chromatography was performed with the <em>Pf</em>Pdx1 WT (—), <em>Pf</em>Pdx1 DKK (----) and the <em>Pf</em>Pdx1 RHE (……) mutant proteins and the masses measured by static light scattering. The corresponding molecular masses and the proposed structural assemblies are given below. Aliquots of the three peak fractions were subsequently analysed by SDS-PAGE, which confirmed the presence of <em>Pf</em>Pdx1 (data not shown).</p>
<span>doi:10.1371/journal.pone.0001815.g005</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Interestingly, addition of 10 mM glutamine (Q) to the buffer used in size exclusion chromatography, resulted in an approximate 50% shift in the assembly of the hexameric formation of the DKK mutant towards the dodecameric structure as estimated from the molecular mass plot obtained by SLS (<a href="#pone-0001815-g006">Fig. 6A</a>). A similar effect was obtained by the addition of 10 mM glutamic acid (E) as well as asparagine (N), although to a lesser extent. In contrast alanine (A) does not affect dodecamer formation.</p>
<div class="figure" id="pone-0001815-g006"><div class="img"><a name="pone-0001815-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>(A) Influence of glutamine on the conformation of the <em>Pf</em>Pdx1 DKK mutant.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Static light scattering analysis of the plasmodial Pdx1 DKK protein was performed in the presence of 10 mM glutamine. The proposed structural conformations are given above the peak fractions. (B) Affect of mutations on Pdx1 interactions between hexamers. Monomers coloured in green and violet belong to the upper hexameric crown and the monomer coloured in salmon belongs to the lower hexameric crown. Mutations in R85, H88 and E91 of the upper crown affect the conformation of loop 113–119, which interacts with loop-helix 179–191 in the lower crown. Helix 88–98 interacts directly with Pdx2 (not shown).</p>
<span>doi:10.1371/journal.pone.0001815.g006</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The amino acid residues E107, E136 and K189 were identified as being proximal to K151 using a homology model of the plasmodial Pdx1 protein (<a href="#pone-0001815-g002">Fig. 2A</a>). E107 and E136 were mutated to alanine and analyzed for their conformation by size exclusion chromatography and SLS. None of these mutants, including the double mutation of both glutamic acids, E107 and E136, showed an effect on dodecamer assembly (data not shown).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Previous crystal structure analyses showed that H116, R131, E135, R138, R139, K150 and K188 in the <em>T. maritima</em> structure are located in a phosphate-binding site <a href="#pone.0001815-Zein1">[15]</a>. All of these amino acid residues are conserved in the plasmodial Pdx1 protein at the positions H117, R132, E136, R139, R140, K151 and K189 (<a href="#pone-0001815-g003">Fig. 3</a>). Mutating the amino acid residues E136, R139 and R140 to alanine (ERR) abolishes the enzyme activity of <em>Pf</em>Pdx1, implying the importance of these residues for PLP synthesis. Nevertheless, the <em>Pf</em>Pdx1 ERR mutant reveals a molecular mass of 421±20 kDa as measured by SLS, which correlates well with the dodecameric formation of the wild-type complex. The glutaminase activity of <em>Pf</em>Pdx2 is not affected (<a href="#pone-0001815-t001">Table 1</a>), indicating binding of Pdx2 to Pdx1 as confirmed by co-purification experiments (<a href="#pone-0001815-g004">Fig. 4B</a>).</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>As a result of these experiments, we propose that the role of K151 in the assembly of dodecameric Pdx1 does not depend upon the neighbouring residues in the final assembled structure, rather that its interaction partners during the assembly must be distal from its final position, possibly even on adjacent monomers of the dodecamer.</p>


<h4>Analysis of the interaction of Pdx1 monomers in P. falciparum</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As already mentioned, biochemical analyses of the plasmodial PLP synthase revealed a 1:1 ratio of both Pdx1 and Pdx2 proteins <a href="#pone.0001815-Wrenger1">[6]</a>, <a href="#pone.0001815-Gengenbacher1">[14]</a>. The crystal structures of the PLP synthase of <em>T. maritima</em> and <em>B. subtilis</em> demonstrate that the dodecameric complex of Pdx1 is decorated by the identical number of Pdx2 monomers, which in turn have no direct contact to one another and are therefore interacting only with the respective Pdx1 protein <a href="#pone.0001815-Zein1">[15]</a>, <a href="#pone.0001815-Strohmeier1">[16]</a>. Thus, the double crown complex depends solely on the interaction of Pdx1. Through an analysis of the protein sequence of the plasmodial Pdx1 and the homology model, a number of amino acids within the <em>Pf</em>Pdx1:<em>Pf</em>Pdx1 interface region were identified (<a href="#pone-0001815-g002">Fig. 2B</a>). These residues include R85, H88 and E91 in <em>P. falciparum</em>, corresponding to amino acids conserved in other organisms as indicated in <a href="#pone-0001815-g002">Figures 2A,B</a> and <a href="#pone-0001815-g003">3</a>. To analyse the impact of these residues on enzyme activity as well as their importance for the assembly of the entire complex, they were mutated to alanine. No PLP synthesis activity of the plasmodial Pdx1 mutants was obtained; however, the glutaminase activity of <em>Pf</em>Pdx2 remained, although it was reduced to approx. 66% and 36% in the presence of the <em>Pf</em>Pdx1 mutants H88A and E91A, respectively (<a href="#pone-0001815-t001">Table 1</a>). The reaction catalyzed by <em>Pf</em>Pdx2 complexed with the <em>Pf</em>Pdx1 triple mutant RHE did not reveal any detectable glutaminase activity, indicating that the interaction of the <em>Pf</em>Pdx1 monomers might affect <em>Pf</em>Pdx2 binding. Furthermore, to verify whether the homomeric relations influence the assembly of <em>Pf</em>Pdx1 and <em>Pf</em>Pdx2, co-purification experiments were performed. The <em>Pf</em>Pdx1 wild-type protein and mutants as well as the <em>Pf</em>Pdx2HIS wild-type protein were heterologously expressed and the bacterial homogenates were mixed in an approx. 1:1 ratio. Subsequently, the proteins were purified via affinity chromatography using the Strep-tag of the respective <em>Pf</em>Pdx1 wild-type or mutant proteins. Western blot analysis clearly demonstrated co-purification of the wild-type forms, whereas the <em>Pf</em>Pdx1 mutants were found to be incompetent in binding <em>Pf</em>Pdx2 (<a href="#pone-0001815-g004">Fig. 4B</a>). The oligomeric states of the <em>Pf</em>Pdx1 RHE, H88A and E91A mutant proteins were further analyzed by size exclusion chromatography and SLS. No significant assembly to dodecameric or hexameric formation was detected in either the absence (<a href="#pone-0001815-g005">Fig. 5</a>/<a href="#pone-0001815-g006">6A</a>) or presence of glutamine. Therefore, the <em>Pf</em>Pdx1 RHE triple mutant impedes formation of the entire complex and exists in solution only as a monomer. Furthermore, even the single mutants H88A and E91A led to a monomeric state. The R85A mutant did not express and no further analysis of this mutant was performed.</p>


<h4>Ammonia shuttle from PfPdx2 to PfPdx1</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>PLP synthesis by <em>Pf</em>Pdx1 depends on ammonia, which is supplied by the hydrolysis of glutamine to glutamate and ammonia by <em>Pf</em>Pdx2, the glutaminase attached to <em>Pf</em>Pdx1 <a href="#pone.0001815-Wrenger1">[6]</a>, <a href="#pone.0001815-Gengenbacher1">[14]</a>. Ammonia is thought to be transferred to the catalytic site of Pdx1 via a protein channel <a href="#pone.0001815-Zhu1">[13]</a>, <a href="#pone.0001815-Zein1">[15]</a>, <a href="#pone.0001815-Strohmeier1">[16]</a>. Analysis of the Pdx2 structure led to the suggestion that the amino acid residues E53 and R154 are the gate-keepers of the ammonia translocation to Pdx1 <a href="#pone.0001815-Zein1">[15]</a>. Exchanging these amino acids to tyrosine (E53Y) and tryptophan (R154W), respectively, resulted in the total loss of the <em>Pf</em>Pdx2 glutaminase activity, while the ability of <em>Pf</em>Pdx1 to generate PLP is not affected if ammonium is added as an ammonia source. These mutations did not influence the ability of <em>Pf</em>Pdx2 to form a complex with <em>Pf</em>Pdx1 as clearly demonstrated by co-purification experiments (<a href="#pone-0001815-g004">Fig. 4C</a>).</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Two different vitamin B6 biosynthetic pathways have been previously described <a href="#pone.0001815-Ehrenshaft1">[18]</a>, <a href="#pone.0001815-Osmani1">[19]</a>. While the DOXP-dependent pathway, consisting of the Pdx protein family, has been well characterised <a href="#pone.0001815-Lam1">[10]</a>, <a href="#pone.0001815-Lam2">[11]</a>, <a href="#pone.0001815-Roa1">[20]</a>, <a href="#pone.0001815-Hill1">[21]</a>, the DOXP-independent pathway, first found in fungi, has only recently been identified <a href="#pone.0001815-Ehrenshaft1">[18]</a>, <a href="#pone.0001815-Osmani1">[19]</a>. The enzymes involved in the DOXP-independent pathway share neither sequence similarities nor substrate acceptance with <em>E. coli</em> Pdx proteins. Furthermore, vitamin B6 biosynthesis in <em>E. coli</em> leads to pyridoxine phosphate, which is subsequently oxidized to PLP by pyridoxine oxidase. The synthesis of the DOXP-independent pathway seems to be more efficient since it results directly in the formation of pyridoxal phosphate, the active form of vitamin B6. This pathway, originally identified by Ehrenshaft <em>et al.</em> <a href="#pone.0001815-Ehrenshaft2">[22]</a> in <em>Cercospora nicotianae</em>, a plant pathogen, is also found in <em>S. cerevisiae</em> as well as some archaebacteria, eubacteria, the plant <em>A. thaliana</em>, and recently in the protozoan parasites <em>P. falciparum</em> and <em>T. gondii</em> <a href="#pone.0001815-Dong1">[5]</a>, <a href="#pone.0001815-Wrenger1">[6]</a>, <a href="#pone.0001815-TambascoStudart1">[9]</a>, <a href="#pone.0001815-Ehrenshaft1">[18]</a>, <a href="#pone.0001815-Knckel1">[23]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>PLP is an essential cofactor for various enzymes many of which are involved in fundamental metabolic reactions <a href="#pone.0001815-Percudani1">[1]</a>. Deletion of vitamin B6 biosynthesis enzymes resulted in auxotrophy for this nutrient in bacterial cells <a href="#pone.0001815-Pflug1">[24]</a> and led to a developmental arrest in plant embryos <a href="#pone.0001815-TambascoStudart1">[9]</a>. Attempts to disrupt the open reading frame of the plasmodial <em>pdx1</em> have failed so far, which might indicate an indispensability of the gene for the survival of the parasite.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Analyses of the crystal structures of the entire PLP synthase complex in <em>T. maritima</em> and <em>B. subtilis</em> revealed a dodecameric Pdx1 conformation, which is decorated by twelve Pdx2 proteins <a href="#pone.0001815-Zein1">[15]</a>, <a href="#pone.0001815-Strohmeier1">[16]</a>. Very recently, the interface of Pdx1 and Pdx2 has been described <a href="#pone.0001815-Strohmeier1">[16]</a>; however, amino acid residues involved in the interaction of the Pdx1 subunits to form the double crown remained to be explored. Therefore, site directed mutagenesis was employed to modify highly conserved amino acid residues that were suggested to form a buried charge cluster and might be located in the Pdx1:Pdx1 interface <a href="#pone.0001815-Zhu1">[13]</a>, <a href="#pone.0001815-Zein1">[15]</a>. These highly conserved residues are R85, H88, E91 and D222 in the plasmodial Pdx1 enzyme. Substitution of H88 and E91 by alanine as well as the derived triple mutant RHE (R85A, H88A, E91A) resulted in the loss of <em>Pf</em>Pdx1 activity. Interestingly, these mutant proteins were neither able to form a dodecamer nor to associate with <em>Pf</em>Pdx2. While these residues primarily interact within the hexameric crown, they interact indirectly with the opposing hexamer through loop 113–119. This loop in turn contacts helix 179–191 of the second hexamer and may mediate dodecameric formation through this mechanism (<a href="#pone-0001815-g006">Fig. 6B</a>). The effect of RHE on Pdx1-Pdx2 interactions is less clear. These residues are positioned at the N-terminus of helix 88–98 and the loop following this helix interacts directly with Pdx2; thus, it is possible that R85, H88 and E91 are required indirectly for Pdx1-Pdx2 interactions as well.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In <em>T. maritima</em> the active site of YaaD contains the amino acid residues D25, K82 and K150 <a href="#pone.0001815-Zein1">[15]</a>. Mutation of these residues in the bacterial Pdx1 enzyme impedes PLP synthesis <a href="#pone.0001815-Zein1">[15]</a>, <a href="#pone.0001815-Raschle2">[25]</a>. This motif is also present in the plasmodial Pdx1 protein and mutagenic analyses of the respective residues (D26A, K83A and K151A) show loss of Pdx1 activity. However, the mutations have no impact on the <em>Pf</em>Pdx1:<em>Pf</em>Pdx2 binding ability and on the glutaminase activity of <em>Pf</em>Pdx2 (<a href="#pone-0001815-t001">Table 1</a>). Interestingly, the <em>Pf</em>Pdx1 DKK mutation leads to a split of the dodecamer into two hexamers. In the presence of glutamine, the destabilizing effect of the DKK triple mutation is reduced by about 50% suggesting an involvement of glutamine in the stabilization of the <em>Pf</em>Pdx1 assembly.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The dissociation of the dodecamer is demonstrated to be a result of the K151A mutation. Examination of the structural data of the <em>B. subtilis</em> Pdx1 complex (2NV2; <a href="http://www.pdb.org">www.pdb.org</a>) demonstrates that the equivalent residue (K149) orientates such that the terminal amino group points away from the active site and towards the central cavity of the assembled Pdx1 dodecamer. Nevertheless, it has been shown that K149 in the <em>B. subtilis</em> enzyme is involved in ribulose 5-phosphate binding <a href="#pone.0001815-Burns1">[7]</a>. K151 in the homologous <em>P. falciparum</em> Pdx1 is located in neither the Pdx1 nor the Pdx2 interface and consequently its role in formation of the Pdx1 dodecamer is intriguing. In the structure of <em>T. maritima</em> YaaD, this residue is pointed towards the phosphate binding site containing ERR <a href="#pone.0001815-Zein1">[15]</a>. It has been proposed that this site marks the glyceraldehyde 3-phosphate binding site or an alternative ribulose 5-phosphate binding site and that K151 participates in this binding <a href="#pone.0001815-Zein1">[15]</a>. ERR directly interact with helix 183–191 of the opposing hexameric crown forming extensive interactions. Interestingly, addition of exogenous amino acids with similar sizes and/or charge to K151 can mimic the function of this lysine by partially restoring the dodecameric conformation. However, this result was observed <em>in vitro</em> and remains for elucidation whether this effect is of physiological relevance.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>The observation that Pdx1 exists as monomers, hexamers and dodecamers, and that only the hexamers and dodecamers interact with Pdx2 suggests a possible order of assembly. Hexameric crowns probably form first, likely mediated by residues R85, H88 and E91. Once a crown has formed, it can bind Pdx2, as evidenced by the data from our co-purification assays, followed by the assembly of two crowns to form a dodecameric Pdx1 core. Alternatively, the dodecameric core could form first, followed by addition of 12 Pdx2 monomers. These two possibilities are not easily differentiated and clarification of the exact mechanism will require additional biochemical and structural studies.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>It has been reported that the bacterial YaaD and PdxS proteins from <em>T. maritima</em> and <em>G. stearothermophilus</em>, respectively, possess an additional phosphate binding site, which might interact with the phosphate group of the second substrate, glyceraldehyde 3-phosphate <a href="#pone.0001815-Zhu1">[13]</a>, <a href="#pone.0001815-Zein1">[15]</a>. Modification of the homologous amino acid residues in <em>P. falciparum</em>, E136, R139 and R140, abolished <em>Pf</em>Pdx1 activity, which emphasizes the important role for catalysis of these amino acid residues.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>For the generation of the active cofactor PLP, an ammonia source is required, which is provided by the glutaminase Pdx2 via substrate channelling <a href="#pone.0001815-Wrenger1">[6]</a>, <a href="#pone.0001815-Gengenbacher1">[14]</a>, <a href="#pone.0001815-Knckel1">[23]</a>. Structural analysis of the bacterial YaaD (Pdx1) and YaaE (Pdx2) complexes suggested a putative ammonia channel <a href="#pone.0001815-Zein1">[15]</a>, <a href="#pone.0001815-Strohmeier1">[16]</a>. The passage of ammonia seems to be modulated by a gate consisting of E47 and R135 in the <em>T. maritima</em> enzyme <a href="#pone.0001815-Zein1">[15]</a>. Mutagenesis of the respective conserved counterparts in <em>P. falciparum</em> (E53 and R154) indeed results in the loss of <em>Pf</em>Pdx2 activity, suggesting a potential steric interference within the ammonia tunnel, thereby blocking the transmission of ammonia towards <em>Pf</em>Pdx1. Interestingly, despite the proximity of the gate to <em>Pf</em>Pdx1, the mutation of these two amino acid residues does not influence the binding capability of <em>Pf</em>Pdx2 to <em>Pf</em>Pdx1.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>In conclusion, the results presented here suggest a possible path through which the plasmodial PLP synthase forms a hierarchical complex using a defined assembly sequence. Confirmation of the PLP synthase assembly path will require additional experiments to probe the dynamic behaviour of assembly. This sequence of events provides a further opportunity to interfere with the assembly of the complex and can be exploited for the development of novel chemotherapeutics to combat malaria.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Materials</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Restriction enzymes and ligase were purchased from New England Biolabs, USA. Oligonucleotides were from Operon, Germany. The cloning vector pASK-IBA3, Strep-Tactin-Sepharose, anhydrotetracycline and desthiobiotin were from IBA (Institut für Bioanalytik, Germany). All other used chemicals were from Sigma, Germany.</p>


<h4>Expression and purification of the PfPdx1 and PfPdx2</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p><em>E. coli</em> BLR (DE3) (Stratagene, Germany) was transformed with <em>P. falciparum</em> Pdx1 and Pdx2 previously cloned into the expression vector pASK-IBA3 <a href="#pone.0001815-Wrenger1">[6]</a>. Single colonies were picked and grown overnight in Luria-Bertani medium containing 50 µg mL<sup>−1</sup> ampicillin. The bacterial culture was diluted 1:50 and grown at 37°C until the A<sub>600</sub> reached 0.5. The expression was initiated with 200 ng mL<sup>−1</sup> of anhydrotetracycline and the cells were grown for 4 h at 37°C before being harvested. The cell pellet was resuspended in 100 mM Tris-HCl, pH 8.0, 150 mM NaCl containing 0.1 mM phenylmethylsulfonyl fluoride, sonicated, and centrifuged at 50,000× g for 1 hour at 4°C. The recombinant Strep-Tag fusion protein was purified according to the manufacturer's recommendation (IBA). The eluate of the affinity chromatography was analysed by SDS-PAGE <a href="#pone.0001815-Sambrook1">[26]</a>. The concentration of the purified recombinant protein was determined according to Bradford <a href="#pone.0001815-Bradford1">[27]</a>.</p>


<h4>Oligonucleotides and site-directed mutagenesis of PfPdx1 and PfPdx2</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Oligonucleotides were designed to replace amino acid residues in the proposed active site as well as in the interface of Pdx1:Pdx1 (<a href="#pone-0001815-t002">Table 2</a>). The putative active site residues of the Pdx1 domain (D26, K83 and K151) were mutated to alanine and the putative amino acid residues involved in ammonia channelling of Pdx2 were substituted by tyrosine and tryptophan (<a href="#pone-0001815-t002">Table 2</a>). 35 ng of the double-stranded supercoiled expression plasmid <em>Pf</em>Pdx1-IBA3 or <em>Pf</em>Pdx2-IBA3 <a href="#pone.0001815-Wrenger1">[6]</a> and 100 ng of mutagenic sense and antisense primers were used in a 50 µL PCR containing deoxyribonucleotides, reaction buffer, and <em>Pfu</em> DNA polymerase as described previously <a href="#pone.0001815-Wrenger2">[28]</a>. The cycling parameters were 95°C for 50 s, 55°C for 60 s, and 68°C for 9 min for 17 cycles. The linear amplification product was treated with endonuclease <em>Dpn</em>I (New England Biolabs) for 1 h to eliminate the parental template. A 10 µL aliquot from each PCR was used for the transformation of competent <em>E. coli</em> XL10GOLD cells (Stratagene). All mutations were verified by automatic sequencing (AGOWA, Germany). Finally, one clone of each construct was transformed for the expression in competent <em>E. coli</em> BLR (DE3) cells. The expressed proteins were purified as described above.</p>
<div class="figure" id="pone-0001815-t002"><div class="img"><a name="pone-0001815-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001815" data-uri="info:doi/10.1371/journal.pone.0001815.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001815.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001815.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001815.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001815.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001815.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Oligonucleotides which are used for cloning or site directed mutagenesis of <em>Pf</em>Pdx1 and <em>Pf</em>Pdx2. Mutation sites are underlined and in bold.</span></strong></p><span>doi:10.1371/journal.pone.0001815.t002</span></div>

<h4>Enzyme assays</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The glutaminase activity of <em>Pf</em>Pdx2 was assayed in two steps, according to <a href="#pone.0001815-Dong1">[5]</a>, <a href="#pone.0001815-Wrenger1">[6]</a>, by measuring the formation of glutamate, which is subsequently converted to 2-oxoglutarate by glutamate dehydrogenase with acetylpyridine adenine dinucleotide (APAD) as co-substrate. For activity the enzyme complex consisting of Pdx1 and Pdx2 is required <a href="#pone.0001815-Wrenger1">[6]</a>; therefore, both enzymes were mixed in an equimolar ratio (total amount 30 µg). The assay was performed in 50 mM Tris-HCl, pH 8.0 in the presence of 10 mM glutamine in a total volume of 300 µL at 37°C for 20 min. The enzymatic reaction was stopped by boiling for 1 min. A 50 mM Tris-HCl, pH 8.0 buffer containing 1 mM EDTA, 0.5 mM APAD and 7 units of glutamate dehydrogenase was added to a final volume of 1 ml and incubated for up to 90 min at 37°C. Finally, the samples were centrifuged for 1 min at 14,000× <em>g</em> and the absorbance of the supernatant was determined at a wavelength of 363 nm. The specific activity was calculated with the molar extinction coefficient of APADH (reduced form of APAD) of 8900 M<sup>−1</sup> cm<sup>−1</sup>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>For the synthesis of PLP, the enzymes Pdx1 and Pdx2 are required. The reaction was performed in the presence of the substrates 0.5 mM ribose 5-phosphate, 1 mM glyceraldehyde 3-phosphate and 10 mM glutamine in a buffer containing 100 mM Tris-HCl, pH 8.0, and 150 mM NaCl <a href="#pone.0001815-Pflug1">[24]</a>. PLP biosynthesis by Pdx1 alone was carried out under equivalent conditions; however glutamine was replaced by 10 mM ammonium salt. The total volume of 1 mL per reaction was incubated for 30 min at 37°C. Subsequently, the supernatant was analysed at 414 nm for the formation of a Schiff base between PLP and the Tris base using a spectrophotometer <a href="#pone.0001815-Burns1">[7]</a>, <a href="#pone.0001815-Raschle1">[8]</a>.</p>


<h4>Co-purification experiments of PfPdx1 and PfPdx2</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The plasmodial proteins were recombinantly expressed as described above. After sonication and centrifugation, the supernatants of the respective Pdx1 wild-type or mutant proteins were combined with the supernatants of the <em>Pf</em>Pdx2 expression. For co-purification the Strep-tag encoded by the pASK-IBA3 expression vector was substituted by a 6× His-tag using a PCR reaction containing the oligonucleotides <em>Pf</em>Pdx2-IBA3-S and <em>Pf</em>Pdx2HIS-IBA3-AS according to <a href="#pone.0001815-Wrenger1">[6]</a>. Subsequently, the mixture of <em>Pf</em>Pdx1 and <em>Pf</em>Pdx2HIS was purified via Strep-Tactin affinity chromatography as described above. For visualizing the co-purification, Western blot analyses were performed employing a monoclonal Strep-tag II antibody (IBA) at a dilution of 1:20000 and a secondary anti-mouse horseradish peroxidase labelled goat antibody (BioRad, Germany) at a dilution of 1:20000. The <em>Pf</em>Pdx2HIS protein was detected by the HIS-Probe-HRP reagent (Pierce, USA) at a dilution of 1:5000. The hybridization signals were visualised on X-ray films (Retina, Germany) using the ECL plus detection system, according to the manufacturer's instructions (GE Healthcare).</p>


<h4>Determination of the oligomeric state of PfPdx1</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>In order to investigate the complex formation of <em>Pf</em>Pdx1, the Strep-tagged protein was purified as described above. Subsequently 100 µg of the protein were separated by gel filtration on a Superdex 200 10/30 column (GE Healthcare). The elution buffer contained 100 mM Tris-HCl, pH 8.0, 150 mM NaCl with and without 10 mM glutamine, glutamic acid, asparagine or alanine, respectively. A miniDAWN Tristar (Wyatt Technologies, USA) was connected immediately downstream of the separation media and used to collect static light scattering (SLS) data <a href="#pone.0001815-Geerlof1">[29]</a>. The SLS data were analyzed using the package ASTRA, based on the absorption coefficient for <em>Pf</em>Pdx1 of 19855 M<sup>−1</sup> cm<sup>−1</sup> and the molecular mass of 34.2 kDa.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Marie-Luise Eschbach and Bärbel Bergmann for excellent technical assistance. J. K. conducted part of this work in fulfilment of the requirement for a Ph.D. from the University of Hamburg.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SE CW IM JK MG RJ RW. Performed the experiments: SE CW IM JK MG RJ. Analyzed the data: SE CW IM JK MG RJ RW. Wrote the paper: SE CW IM JK MG RJ RW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001815-Percudani1" id="pone.0001815-Percudani1"></a>Percudani R, Peracchi A (2003) A genomic overview of pyridoxal-phosphate-dependent enzymes. EMBO Rep  4: 850–854.  <ul class="find" data-citedArticleID="1076202" data-doi="10.1038/sj.embor.embor914"><li><a href="http://dx.doi.org/10.1038/sj.embor.embor914" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genomic+overview+of+pyridoxal-phosphate-dependent+enzymes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genomic+overview+of+pyridoxal-phosphate-dependent+enzymes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001815-Stolz1" id="pone.0001815-Stolz1"></a>Stolz J, Vielreicher M (2003) Tpn1p, the plasma membrane vitamin B6 transporter of Saccharomyces cerevisiae. J Biol Chem  278: 18990–18996.  <ul class="find" data-citedArticleID="1076216" data-doi="10.1074/jbc.m300949200"><li><a href="http://dx.doi.org/10.1074/jbc.m300949200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tpn1p%2C+the+plasma+membrane+vitamin+B6+transporter+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tpn1p%2C+the+plasma+membrane+vitamin+B6+transporter+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001815-Kerry1" id="pone.0001815-Kerry1"></a>Kerry JA, Rohde M, Kwok F (1986) Brain pyridoxal kinase. Purification and characterization. Eur J Biochem  158: 581–585.  <ul class="find" data-citedArticleID="1076190" data-doi="10.1111/j.1432-1033.1986.tb09794.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.1986.tb09794.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Brain+pyridoxal+kinase.+Purification+and+characterization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Brain+pyridoxal+kinase.+Purification+and+characterization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001815-Belitsky1" id="pone.0001815-Belitsky1"></a>Belitsky BR (2004) Physical and enzymological interaction of Bacillus subtilis proteins required for de novo pyridoxal 5-phosphate biosynthesis. J Bacteriol  186: 1191–1196.  <ul class="find" data-citedArticleID="1076170" data-doi="10.1128/jb.186.4.1191-1196.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.4.1191-1196.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical+and+enzymological+interaction+of+Bacillus+subtilis+proteins+required+for+de+novo+pyridoxal+5-phosphate+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical+and+enzymological+interaction+of+Bacillus+subtilis+proteins+required+for+de+novo+pyridoxal+5-phosphate+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001815-Dong1" id="pone.0001815-Dong1"></a>Dong YX, Sueda S, Nikawa J, Kondo H (2004) Characterization of the products of the genes SNO1 and SNZ1 involved in pyridoxine synthesis in Saccharomyces cerevisiae. Eur J Biochem  271: 745–752.  <ul class="find" data-citedArticleID="1076176" data-doi="10.1111/j.1432-1033.2003.03973.x"><li><a href="http://dx.doi.org/10.1111/j.1432-1033.2003.03973.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+products+of+the+genes+SNO1+and+SNZ1+involved+in+pyridoxine+synthesis+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+products+of+the+genes+SNO1+and+SNZ1+involved+in+pyridoxine+synthesis+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001815-Wrenger1" id="pone.0001815-Wrenger1"></a>Wrenger C, Eschbach ML, Müller IB, Warnecke D, Walter RD (2005) Analysis of the vitamin B6 biosynthesis pathway in the human malaria parasite Plasmodium falciparum. J Biol Chem  280: 5242–5248.  <ul class="find" data-citedArticleID="1076222" data-doi="10.1074/jbc.m412475200"><li><a href="http://dx.doi.org/10.1074/jbc.m412475200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+vitamin+B6+biosynthesis+pathway+in+the+human+malaria+parasite+Plasmodium+falciparum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+vitamin+B6+biosynthesis+pathway+in+the+human+malaria+parasite+Plasmodium+falciparum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001815-Burns1" id="pone.0001815-Burns1"></a>Burns KE, Xiang Y, Kinsland CL, McLaffert FW, Begley TP (2005) Reconstitution and biochemical characterization of a new pyridoxal-5-phosphate biosynthetic pathway. J Am Chem Soc  127: 3682–3683.  <ul class="find" data-citedArticleID="1076174" data-doi="10.1021/ja042792t"><li><a href="http://dx.doi.org/10.1021/ja042792t" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstitution+and+biochemical+characterization+of+a+new+pyridoxal-5-phosphate+biosynthetic+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstitution+and+biochemical+characterization+of+a+new+pyridoxal-5-phosphate+biosynthetic+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001815-Raschle1" id="pone.0001815-Raschle1"></a>Raschle T, Amrhein N, Fitzpatrick TB (2005) On the two components of pyridoxal 5-phosphate synthase from Bacillus subtilis. J Biol Chem  280: 32291–32300.  <ul class="find" data-citedArticleID="1076206" data-doi="10.1074/jbc.m501356200"><li><a href="http://dx.doi.org/10.1074/jbc.m501356200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+two+components+of+pyridoxal+5-phosphate+synthase+from+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+two+components+of+pyridoxal+5-phosphate+synthase+from+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001815-TambascoStudart1" id="pone.0001815-TambascoStudart1"></a>Tambasco-Studart M, Titiz O, Raschle T, Forster G, Amrhein N, et al.  (2005) Vitamin B6 biosynthesis in higher plants. Proc Natl Acad Sci USA  102: 13687–13692.  <ul class="find" data-citedArticleID="1076220" data-doi="10.1073/pnas.0506228102"><li><a href="http://dx.doi.org/10.1073/pnas.0506228102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vitamin+B6+biosynthesis+in+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vitamin+B6+biosynthesis+in+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001815-Lam1" id="pone.0001815-Lam1"></a>Lam HM, Tancula E, Dempsey WB, Winkler ME (1992) Suppression of insertions in the complex pdxJ operon of Escherichia coli K-12 by lon and other mutations. J Bacteriol  174: 1554–1567.  <ul class="find" data-citedArticleID="1076194"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Suppression+of+insertions+in+the+complex+pdxJ+operon+of+Escherichia+coli+K-12+by+lon+and+other+mutations.&amp;auth=&amp;atitle=Suppression+of+insertions+in+the+complex+pdxJ+operon+of+Escherichia+coli+K-12+by+lon+and+other+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppression+of+insertions+in+the+complex+pdxJ+operon+of+Escherichia+coli+K-12+by+lon+and+other+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppression+of+insertions+in+the+complex+pdxJ+operon+of+Escherichia+coli+K-12+by+lon+and+other+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001815-Lam2" id="pone.0001815-Lam2"></a>Lam HM, Winkler ME (1992) Characterization of the complex pdxH-tyrS operon of Escherichia coli K-12 and pleiotropic phenotypes caused by pdxH insertion mutations. J Bacteriol  174: 6033–6045.  <ul class="find" data-citedArticleID="1076196"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Characterization+of+the+complex+pdxH-tyrS+operon+of+Escherichia+coli+K-12+and+pleiotropic+phenotypes+caused+by+pdxH+insertion+mutations.&amp;auth=&amp;atitle=Characterization+of+the+complex+pdxH-tyrS+operon+of+Escherichia+coli+K-12+and+pleiotropic+phenotypes+caused+by+pdxH+insertion+mutations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+complex+pdxH-tyrS+operon+of+Escherichia+coli+K-12+and+pleiotropic+phenotypes+caused+by+pdxH+insertion+mutations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+complex+pdxH-tyrS+operon+of+Escherichia+coli+K-12+and+pleiotropic+phenotypes+caused+by+pdxH+insertion+mutations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001815-Bauer1" id="pone.0001815-Bauer1"></a>Bauer JA, Bennett EM, Begley TP, Ealick SE (2004) Three-dimensional structure of YaaE from Bacillus subtilis, a glutaminase implicated in pyridoxal-5-phosphate biosynthesis. J Biol Chem  279: 2704–2711.  <ul class="find" data-citedArticleID="1076168" data-doi="10.1074/jbc.m310311200"><li><a href="http://dx.doi.org/10.1074/jbc.m310311200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three-dimensional+structure+of+YaaE+from+Bacillus+subtilis%2C+a+glutaminase+implicated+in+pyridoxal-5-phosphate+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three-dimensional+structure+of+YaaE+from+Bacillus+subtilis%2C+a+glutaminase+implicated+in+pyridoxal-5-phosphate+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001815-Zhu1" id="pone.0001815-Zhu1"></a>Zhu J, Burgner JW, Harms E, Belitsky BR, Smith JL (2005) A new arrangement of (β/α)<sub>8</sub> barrels in the synthase subunit of PLP synthase. J Biol Chem  280: 27914–27923.  <ul class="find" data-citedArticleID="1076228" data-doi="10.1074/jbc.m503642200"><li><a href="http://dx.doi.org/10.1074/jbc.m503642200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+arrangement+of+%28%CE%B2%2F%CE%B1%298+barrels+in+the+synthase+subunit+of+PLP+synthase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+arrangement+of+%28%CE%B2%2F%CE%B1%298+barrels+in+the+synthase+subunit+of+PLP+synthase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001815-Gengenbacher1" id="pone.0001815-Gengenbacher1"></a>Gengenbacher M, Fitzpatrick TB, Raschle T, Flicker K, Sinning I, et al.  (2006) Vitamin B6 biosynthesis by the malaria parasite Plasmodium falciparum: biochemical and structural insights. J Biol Chem  281: 3633–3641.  <ul class="find" data-citedArticleID="1076184" data-doi="10.1074/jbc.m508696200"><li><a href="http://dx.doi.org/10.1074/jbc.m508696200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vitamin+B6+biosynthesis+by+the+malaria+parasite+Plasmodium+falciparum%3A+biochemical+and+structural+insights." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vitamin+B6+biosynthesis+by+the+malaria+parasite+Plasmodium+falciparum%3A+biochemical+and+structural+insights.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001815-Zein1" id="pone.0001815-Zein1"></a>Zein F, Zhang Y, Kang YN, Burns K, Begley TP, et al.  (2006) Structural insights into the mechanism of the PLP synthase holoenzyme from Thermotoga maritima. Biochemistry  45: 14609–14620.  <ul class="find" data-citedArticleID="1076226" data-doi="10.1021/bi061464y"><li><a href="http://dx.doi.org/10.1021/bi061464y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+insights+into+the+mechanism+of+the+PLP+synthase+holoenzyme+from+Thermotoga+maritima." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+insights+into+the+mechanism+of+the+PLP+synthase+holoenzyme+from+Thermotoga+maritima.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001815-Strohmeier1" id="pone.0001815-Strohmeier1"></a>Strohmeier M, Raschle T, Mazurkiewicz J, Rippe K, Sinning I, et al.  (2006) Structure of a bacterial pyridoxal 5′-phosphate synthase complex. Proc Natl Acad Sci USA  103: 19284–19289.  <ul class="find" data-citedArticleID="1076218" data-doi="10.1073/pnas.0604950103"><li><a href="http://dx.doi.org/10.1073/pnas.0604950103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+a+bacterial+pyridoxal+5%E2%80%B2-phosphate+synthase+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+a+bacterial+pyridoxal+5%E2%80%B2-phosphate+synthase+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001815-Newton1" id="pone.0001815-Newton1"></a>Newton P, White N (1999) Malaria: new developments in treatment and prevention. Annu Rev Med  50: 179–192.  <ul class="find" data-citedArticleID="1076198" data-doi="10.1146/annurev.med.50.1.179"><li><a href="http://dx.doi.org/10.1146/annurev.med.50.1.179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Malaria%3A+new+developments+in+treatment+and+prevention." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Malaria%3A+new+developments+in+treatment+and+prevention.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001815-Ehrenshaft1" id="pone.0001815-Ehrenshaft1"></a>Ehrenshaft M, Bilski P, Li MY, Chignell CF, Daub ME (1999) A highly conserved sequence is a novel gene involved in de novo vitamin B6 biosynthesis. Proc Natl Acad Sci USA  96: 9374–9378.  <ul class="find" data-citedArticleID="1076178" data-doi="10.1073/pnas.96.16.9374"><li><a href="http://dx.doi.org/10.1073/pnas.96.16.9374" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+highly+conserved+sequence+is+a+novel+gene+involved+in+de+novo+vitamin+B6+biosynthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+highly+conserved+sequence+is+a+novel+gene+involved+in+de+novo+vitamin+B6+biosynthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001815-Osmani1" id="pone.0001815-Osmani1"></a>Osmani AH, May GS, Osmani SA (1999) The extremely conserved pyroA gene of Aspergillus nidulans is required for pyridoxine synthesis and is required indirectly for resistance to photosensitizers. J Biol Chem  274: 23565–23569.  <ul class="find" data-citedArticleID="1076200" data-doi="10.1074/jbc.274.33.23565"><li><a href="http://dx.doi.org/10.1074/jbc.274.33.23565" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+extremely+conserved+pyroA+gene+of+Aspergillus+nidulans+is+required+for+pyridoxine+synthesis+and+is+required+indirectly+for+resistance+to+photosensitizers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+extremely+conserved+pyroA+gene+of+Aspergillus+nidulans+is+required+for+pyridoxine+synthesis+and+is+required+indirectly+for+resistance+to+photosensitizers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001815-Roa1" id="pone.0001815-Roa1"></a>Roa BB, Connolly DM, Winkler ME (1989) Overlap between pdxA and ksgA in the complex pdxA-ksgA-apaG-apaH operon of Escherichia coli K-12. J Bacteriol  171: 4767–4777.  <ul class="find" data-citedArticleID="1076210"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Overlap+between+pdxA+and+ksgA+in+the+complex+pdxA-ksgA-apaG-apaH+operon+of+Escherichia+coli+K-12.&amp;auth=&amp;atitle=Overlap+between+pdxA+and+ksgA+in+the+complex+pdxA-ksgA-apaG-apaH+operon+of+Escherichia+coli+K-12." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overlap+between+pdxA+and+ksgA+in+the+complex+pdxA-ksgA-apaG-apaH+operon+of+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overlap+between+pdxA+and+ksgA+in+the+complex+pdxA-ksgA-apaG-apaH+operon+of+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001815-Hill1" id="pone.0001815-Hill1"></a>Hill RE, Himmeldirk K, Kennedy IA, Pauloski RM, Sayer BG, et al.  (1996) The biogenetic anatomy of vitamin B6. A 13C NMR investigation of the biosynthesis of pyridoxol in Escherichia coli. J Biol Chem  271: 30426–30435.  <ul class="find" data-citedArticleID="1076188" data-doi="10.1074/jbc.271.48.30426"><li><a href="http://dx.doi.org/10.1074/jbc.271.48.30426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+biogenetic+anatomy+of+vitamin+B6.+A+13C+NMR+investigation+of+the+biosynthesis+of+pyridoxol+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+biogenetic+anatomy+of+vitamin+B6.+A+13C+NMR+investigation+of+the+biosynthesis+of+pyridoxol+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001815-Ehrenshaft2" id="pone.0001815-Ehrenshaft2"></a>Ehrenshaft M, Jenns AE, Chung KR, Daub ME (1998) SOR1, a gene required for photosensitizer and singlet oxygen resistance in Cercospora fungi, is highly conserved in divergent organisms. Mol Cell  1: 603–609.  <ul class="find" data-citedArticleID="1076180" data-doi="10.1016/s1097-2765(00)80060-x"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80060-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SOR1%2C+a+gene+required+for+photosensitizer+and+singlet+oxygen+resistance+in+Cercospora+fungi%2C+is+highly+conserved+in+divergent+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SOR1%2C+a+gene+required+for+photosensitizer+and+singlet+oxygen+resistance+in+Cercospora+fungi%2C+is+highly+conserved+in+divergent+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001815-Knckel1" id="pone.0001815-Knckel1"></a>Knöckel J, Müller IB, Bergmann B, Walter RD, Wrenger C (2006) The apicomplexan parasite Toxoplasma gondii generates pyridoxal phosphate de novo. Mol Biochem Parasitol  152: 108–111.  <ul class="find" data-citedArticleID="1076192" data-doi="10.1016/j.molbiopara.2006.12.005"><li><a href="http://dx.doi.org/10.1016/j.molbiopara.2006.12.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+apicomplexan+parasite+Toxoplasma+gondii+generates+pyridoxal+phosphate+de+novo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+apicomplexan+parasite+Toxoplasma+gondii+generates+pyridoxal+phosphate+de+novo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001815-Pflug1" id="pone.0001815-Pflug1"></a>Pflug W, Lingens F (1983) Occurrence of pydixaminephosphate oxidase and pyridoxal kinase in Gram-negative and Gram-positive bacteria. Hoppe Seylers Z Physiol Chem  364: 1627–1630.  <ul class="find" data-citedArticleID="1076204" data-doi="10.1515/bchm2.1983.364.2.1627"><li><a href="http://dx.doi.org/10.1515/bchm2.1983.364.2.1627" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Occurrence+of+pydixaminephosphate+oxidase+and+pyridoxal+kinase+in+Gram-negative+and+Gram-positive+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Occurrence+of+pydixaminephosphate+oxidase+and+pyridoxal+kinase+in+Gram-negative+and+Gram-positive+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001815-Raschle2" id="pone.0001815-Raschle2"></a>Raschle T, Arigoni D, Brunisholz R, Amrhein N, Fitzpatrick TB (2007) Reaction mechanism of pyridoxal 5′-phosphate synthase: Detection of an enzyme bound chromophoric intermediate. J Biol Chem  282: 6098–6105.  <ul class="find" data-citedArticleID="1076208" data-doi="10.1074/jbc.m610614200"><li><a href="http://dx.doi.org/10.1074/jbc.m610614200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reaction+mechanism+of+pyridoxal+5%E2%80%B2-phosphate+synthase%3A+Detection+of+an+enzyme+bound+chromophoric+intermediate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reaction+mechanism+of+pyridoxal+5%E2%80%B2-phosphate+synthase%3A+Detection+of+an+enzyme+bound+chromophoric+intermediate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001815-Sambrook1" id="pone.0001815-Sambrook1"></a>Sambrook J, Fritsch EF, Maniatis T (1989) Molecular Cloning: A Laboratory Manual, 2nd Ed. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0001815-Bradford1" id="pone.0001815-Bradford1"></a>Bradford MM (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem  72: 248–254.  <ul class="find" data-citedArticleID="1076172" data-doi="10.1006/abio.1976.9999"><li><a href="http://dx.doi.org/10.1006/abio.1976.9999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001815-Wrenger2" id="pone.0001815-Wrenger2"></a>Wrenger C, Lüersen K, Krause T, Müller S, Walter RD (2001) The Plasmodium falciparum bifunctional ornithine decarboxylase, S-adenosyl-L-methionine decarboxylase, enables a well balanced polyamine synthesis without domain-domain interaction. J Biol Chem  276: 29651–29656.  <ul class="find" data-citedArticleID="1076224" data-doi="10.1074/jbc.m100578200"><li><a href="http://dx.doi.org/10.1074/jbc.m100578200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Plasmodium+falciparum+bifunctional+ornithine+decarboxylase%2C+S-adenosyl-L-methionine+decarboxylase%2C+enables+a+well+balanced+polyamine+synthesis+without+domain-domain+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Plasmodium+falciparum+bifunctional+ornithine+decarboxylase%2C+S-adenosyl-L-methionine+decarboxylase%2C+enables+a+well+balanced+polyamine+synthesis+without+domain-domain+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001815-Geerlof1" id="pone.0001815-Geerlof1"></a>Geerlof A, Brown J, Coutard B, Egloff MP, Enguita FJ, et al.  (2006) The impact of protein characterization in structural proteomics. Acta Crystallogr D Biol Crystallogr  62: 1125–1136.  <ul class="find" data-citedArticleID="1076182" data-doi="10.1107/s0907444906030307"><li><a href="http://dx.doi.org/10.1107/s0907444906030307" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+impact+of+protein+characterization+in+structural+proteomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+impact+of+protein+characterization+in+structural+proteomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001815-Schwede1" id="pone.0001815-Schwede1"></a>Schwede T, Kopp J, Guex N, Peitsch MC (2003) SWISS-MODEL: An automated protein homology-modeling server. Nucleic Acids Res  31: 3381–3385.  <ul class="find" data-citedArticleID="1076214" data-doi="10.1093/nar/gkg520"><li><a href="http://dx.doi.org/10.1093/nar/gkg520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SWISS-MODEL%3A+An+automated+protein+homology-modeling+server." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SWISS-MODEL%3A+An+automated+protein+homology-modeling+server.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001815-Higgins1" id="pone.0001815-Higgins1"></a>Higgins D, Thompson J, Gibson T, Thompson JD, Higgins DG, et al.  (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research  22: 4673–4680.  <ul class="find" data-citedArticleID="1076186" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.XML" value="88785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.PDF" value="1024005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g001.PNG_L" value="140407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g001.PNG_M" value="56958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g001.PNG_S" value="8872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g001.TIF" value="202968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g001.PNG_I" value="18935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g002.PNG_L" value="1319748"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g002.PNG_M" value="191278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g002.PNG_S" value="6621"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g002.TIF" value="1809856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g002.PNG_I" value="69724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g003.PNG_L" value="341577"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g003.PNG_M" value="145945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g003.PNG_S" value="12776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g003.TIF" value="520716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g003.PNG_I" value="48716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g004.PNG_L" value="288543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g004.PNG_M" value="34117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g004.PNG_S" value="3293"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g004.TIF" value="344492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g004.PNG_I" value="13144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t001.PNG_L" value="168768"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t001.PNG_M" value="96937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t001.PNG_S" value="11523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t001.TIF" value="744864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t001.PNG_I" value="30464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g005.PNG_L" value="125760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g005.PNG_M" value="26331"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g005.PNG_S" value="2986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g005.TIF" value="297402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g005.PNG_I" value="11200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g006.PNG_L" value="672417"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g006.PNG_M" value="104543"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g006.PNG_S" value="5524"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g006.TIF" value="984088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.g006.PNG_I" value="40620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t002.PNG_L" value="245920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t002.PNG_M" value="175783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t002.PNG_S" value="13507"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t002.TIF" value="1208188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001815.t002.PNG_I" value="55030"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001815&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001815" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001815&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001815&volume=&issue=&title=The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course&author_name=Ingrid%20B.%20M%C3%BCller%2C%20Julia%20Kn%C3%B6ckel%2C%20Matthew%20R.%20Groves%2C%20Rositsa%20Jordanova%2C%20Steven%20E.%20Ealick%2C%20Rolf%20D.%20Walter%2C%20Carsten%20Wrenger&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001815" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815&amp;t=The%20Assembly%20of%20the%20Plasmodial%20PLP%20Synthase%20Complex%20Follows%20a%20Defined%20Course" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815&title=The%20Assembly%20of%20the%20Plasmodial%20PLP%20Synthase%20Complex%20Follows%20a%20Defined%20Course&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815&amp;title=The%20Assembly%20of%20the%20Plasmodial%20PLP%20Synthase%20Complex%20Follows%20a%20Defined%20Course" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001815&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'The Assembly of the Plasmodial PLP Synthase Complex Follows a Defined Course';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001815';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20The%20Assembly%20of%20the%20Plasmodial%20PLP%20Synthase%20Complex%20Follows%20a%20Defined%20Course http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001815" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001815" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alanine%22" title="Search for articles in the subject area:'Alanine'"><div class="flagText">Alanine</div></a>
              <div data-categoryid="40577" data-articleid="26454"
                   data-categoryname="Alanine"
                   class="flagImage" title="Flag 'Alanine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Amino+acid+analysis%22" title="Search for articles in the subject area:'Amino acid analysis'"><div class="flagText">Amino acid analysis</div></a>
              <div data-categoryid="47251" data-articleid="26454"
                   data-categoryname="Amino acid analysis"
                   class="flagImage" title="Flag 'Amino acid analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ammonia%22" title="Search for articles in the subject area:'Ammonia'"><div class="flagText">Ammonia</div></a>
              <div data-categoryid="21769" data-articleid="26454"
                   data-categoryname="Ammonia"
                   class="flagImage" title="Flag 'Ammonia' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="26454"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glutamine%22" title="Search for articles in the subject area:'Glutamine'"><div class="flagText">Glutamine</div></a>
              <div data-categoryid="40885" data-articleid="26454"
                   data-categoryname="Glutamine"
                   class="flagImage" title="Flag 'Glutamine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plasmodium%22" title="Search for articles in the subject area:'Plasmodium'"><div class="flagText">Plasmodium</div></a>
              <div data-categoryid="18205" data-articleid="26454"
                   data-categoryname="Plasmodium"
                   class="flagImage" title="Flag 'Plasmodium' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pyridoxine%22" title="Search for articles in the subject area:'Pyridoxine'"><div class="flagText">Pyridoxine</div></a>
              <div data-categoryid="44271" data-articleid="26454"
                   data-categoryname="Pyridoxine"
                   class="flagImage" title="Flag 'Pyridoxine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="26454"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5071'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6181'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9924&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=6697">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=427">Referee comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
