

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000324"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000324" />

    <meta name="citation_title" content="Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes"/>
    <meta itemprop="name" content="Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes"/>

      <meta name="citation_author" content="Gareth A. Wilson"/>
            <meta name="citation_author_institution" content="Centre for Ecology and Hydrology (CEH) Oxford, Oxford, United Kindgom"/>
      <meta name="citation_author" content="Edward J. Feil"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Bath, Bath, United Kingdom"/>
      <meta name="citation_author" content="Andrew K. Lilley"/>
            <meta name="citation_author_institution" content="Centre for Ecology and Hydrology (CEH) Oxford, Oxford, United Kindgom"/>
      <meta name="citation_author" content="Dawn Field"/>
            <meta name="citation_author_institution" content="Centre for Ecology and Hydrology (CEH) Oxford, Oxford, United Kindgom"/>

    <meta name="citation_date" content="2007/3/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000324.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e324"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes.; citation_author=RL Tatusov; citation_author=ND Fedorova; citation_author=JD Jackson; citation_author=AR Jacobs; citation_author=B Kiryutin; citation_journal_title=BMC Bioinformatics; citation_volume=4; citation_number=1; citation_pages=41; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computing prokaryotic gene ubiquity: rescuing the core from extinction.; citation_author=RL Charlebois; citation_author=WF Doolittle; citation_journal_title=Genome Res; citation_volume=14; citation_number=2; citation_pages=2469-2477; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Toward automatic reconstruction of a highly resolved tree of life.; citation_author=FD Ciccarelli; citation_author=T Doerks; citation_author=C von Mering; citation_author=CJ Creevey; citation_author=B Snel; citation_journal_title=Science; citation_volume=311; citation_number=3; citation_pages=1283-1287; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Orphans as taxonomically restricted and ecologically important genes.; citation_author=GA Wilson; citation_author=N Bertrand; citation_author=Y Patel; citation_author=JB Hughes; citation_author=EJ Feil; citation_journal_title=Microbiology; citation_volume=151; citation_number=4; citation_pages=2499-2501; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacterial Genomes as new gene homes: The genealogy of ORFans in E-coli.; citation_author=V Daubin; citation_author=H Ochman; citation_journal_title=Genome Research; citation_volume=14; citation_number=5; citation_pages=1036-1042; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=On the total number of genes and their length distribution in complete microbial genomes.; citation_author=M Skovgaard; citation_author=LJ Jensen; citation_author=S Brunak; citation_author=D Ussery; citation_author=A Krogh; citation_journal_title=Trends in Genetics; citation_volume=17; citation_number=6; citation_pages=425-428; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of an aerobic hyper-thermophilic crenarchaeon, Aeropyrum pernix K1.; citation_author=Y Kawarabayasi; citation_author=Y Hino; citation_author=H Horikawa; citation_author=S Yamazaki; citation_author=Y Haikawa; citation_journal_title=DNA Res; citation_volume=6; citation_number=7; citation_pages=83–101, 145-152; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Estimation of the number of authentic orphan genes in bacterial genomes.; citation_author=S Fukuchi; citation_author=K Nishikawa; citation_journal_title=DNA Res; citation_volume=11; citation_number=8; citation_pages=219–231, 311-313; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome Update: annotation quality in sequenced microbial genomes.; citation_author=DW Ussery; citation_author=PF Hallin; citation_journal_title=Microbiology; citation_volume=150; citation_number=9; citation_pages=2015-2017; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Twenty thousand ORFan microbial protein families for the biologist?; citation_author=N Siew; citation_author=D Fischer; citation_journal_title=Structure; citation_volume=11; citation_number=10; citation_pages=7-9; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ecological Perspectives on our complete genome collection.; citation_author=JBH Martiny; citation_author=D Field; citation_journal_title=Ecology Letters; citation_volume=8; citation_number=11; citation_pages=1334-1345; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Reverse transcriptase-polymerase chain reaction validation of 25 “orphan” genes from Escherichia coli K-12 MG1655.; citation_author=JP Alimi; citation_author=O Poirot; citation_author=F Lopez; citation_author=JM Claverie; citation_journal_title=Genome Research; citation_volume=10; citation_number=12; citation_pages=959-966; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification and functional analysis of ‘hypothetical’ genes expressed in Haemophilus influenzae.; citation_author=E Kolker; citation_author=KS Makarova; citation_author=S Shabalina; citation_author=AF Picone; citation_author=S Purvine; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=13; citation_pages=2353-2361; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Poorly conserved ORFs in the genome of the archaea Halobacterium sp NRC-1 correspond to expressed proteins.; citation_author=H Shmuely; citation_author=E Dinitz; citation_author=I Dahan; citation_author=J Eichler; citation_author=D Fischer; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=14; citation_pages=1248-1253; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Vibrio fischeri: a symbiotic bacterium with pathogenic congeners.; citation_author=EG Ruby; citation_author=M Urbanowski; citation_author=J Campbell; citation_author=A Dunn; citation_author=M Faini; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=15; citation_pages=3004-3009; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Lysogenic conversion by a filamentous phage encoding cholera toxin.; citation_author=MK Waldor; citation_author=JJ Mekalanos; citation_journal_title=Science; citation_volume=272; citation_number=16; citation_pages=1910-1914; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The tcp gene cluster of Vibrio cholerae.; citation_author=PA Manning; citation_journal_title=Gene; citation_volume=192; citation_number=17; citation_pages=63-70; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genome analysis of multiple pathogenic isolates of Streptococcus agalactiae: implications for the microbial “pan-genome”.; citation_author=H Tettelin; citation_author=V Masignani; citation_author=MJ Cieslewicz; citation_author=C Donati; citation_author=D Medini; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=18; citation_pages=13950-13955; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The microbial pan-genome.; citation_author=D Medini; citation_author=C Donati; citation_author=H Tettelin; citation_author=V Masignani; citation_author=R Rappuoli; citation_journal_title=Curr Opin Genet Dev; citation_volume=15; citation_number=19; citation_pages=589-594; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS).; citation_author=T Kosuge; citation_author=T Abe; citation_author=T Okido; citation_author=N Tanaka; citation_author=M Hirahata; citation_journal_title=DNA Res; citation_volume=13; citation_number=20; citation_pages=13; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Issues in searching molecular sequence databases.; citation_author=SF Altschul; citation_author=MS Boguski; citation_author=W Gish; citation_author=JC Wootton; citation_journal_title=Nat Genet; citation_volume=6; citation_number=21; citation_pages=119-129; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella.; citation_author=WW Navarre; citation_author=S Porwollik; citation_author=Y Wang; citation_author=M McClelland; citation_author=H Rosen; citation_journal_title=Science; citation_volume=313; citation_number=22; citation_pages=236-238; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Metabolic efficiency and amino acid composition in the proteomes of Escherichia coli and Bacillus subtilis.; citation_author=H Akashi; citation_author=T Gojobori; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=23; citation_pages=3695-3700; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Amino acid cost and codon-usage biases in 6 prokaryotic genomes: a whole-genome analysis.; citation_author=EM Heizer; citation_author=DW Raiford; citation_author=ML Raymer; citation_author=TE Doom; citation_author=RV Miller; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=24; citation_pages=1670-1680; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic detection of conserved gene clusters in microbial genomes.; citation_author=Y Zheng; citation_author=BP Anton; citation_author=RJ Roberts; citation_author=S Kasif; citation_journal_title=BMC Bioinformatics; citation_volume=6; citation_number=25; citation_pages=243; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=On the origin of microbial ORFans: quantifying the strength of the evidence for viral lateral transfer.; citation_author=Y Yin; citation_author=D Fischer; citation_journal_title=BMC Evol Biol; citation_volume=6; citation_number=26; citation_pages=63; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The MicrobesOnline Web site for comparative genomics.; citation_author=EJ Alm; citation_author=KH Huang; citation_author=MN Price; citation_author=RP Koche; citation_author=K Keller; citation_journal_title=Genome Res; citation_volume=15; citation_number=27; citation_pages=1015-1022; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulatory network of Escherichia coli: consistency between literature knowledge and microarray profiles.; citation_author=RM Gutierrez-Rios; citation_author=DA Rosenblueth; citation_author=JA Loza; citation_author=AM Huerta; citation_author=JD Glasner; citation_journal_title=Genome Res; citation_volume=13; citation_number=28; citation_pages=2435-2443; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function.; citation_author=Y Kang; citation_author=KD Weber; citation_author=Y Qiu; citation_author=PJ Kiley; citation_author=FR Blattner; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=29; citation_pages=1135-1160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comparative gene expression profiles following UV exposure in wild-type and SOS-deficient Escherichia coli.; citation_author=J Courcelle; citation_author=A Khodursky; citation_author=B Peter; citation_author=PO Brown; citation_author=PC Hanawalt; citation_journal_title=Genetics; citation_volume=158; citation_number=30; citation_pages=41-64; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA microarray analysis of gene expression in response to physiological and genetic changes that affect tryptophan metabolism in Escherichia coli.; citation_author=AB Khodursky; citation_author=BJ Peter; citation_author=NR Cozzarelli; citation_author=D Botstein; citation_author=PO Brown; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=31; citation_pages=12170-12175; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Toward a protein profile of Escherichia coli: comparison to its transcription profile.; citation_author=RW Corbin; citation_author=O Paliy; citation_author=F Yang; citation_author=J Shabanowitz; citation_author=M Platt; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=32; citation_pages=9232-9237; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Chromatographic isolation of methionine-containing peptides for gel-free proteome analysis: identification of more than 800 Escherichia coli proteins.; citation_author=K Gevaert; citation_author=J Van Damme; citation_author=M Goethals; citation_author=GR Thomas; citation_author=B Hoorelbeke; citation_journal_title=Mol Cell Proteomics; citation_volume=1; citation_number=33; citation_pages=896-903; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Only a small subset of the horizontally transferred chromosomal genes in Escherichia coli are translated into proteins.; citation_author=M Taoka; citation_author=Y Yamauchi; citation_author=T Shinkawa; citation_author=H Kaji; citation_author=W Motohashi; citation_journal_title=Mol Cell Proteomics; citation_volume=3; citation_number=34; citation_pages=780-787; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identifying protein function–a call for community action.; citation_author=RJ Roberts; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=35; citation_pages=E42; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An Experimental Approach to Genome Annotation.; citation_author=RJ Roberts; citation_author=P Karp; citation_author=S Kasif; citation_author=S Linn; citation_author=MR Buckley; citation_journal_title=Critical Issues Colloquia Report, American Society of Microbiology; citation_number=36; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=‘Conserved hypothetical’ proteins: prioritization of targets for experimental study.; citation_author=MY Galperin; citation_author=EV Koonin; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=37; citation_pages=5452-5463; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Selectionism and neutralism in molecular evolution.; citation_author=M Nei; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=38; citation_pages=2318-2342; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Protein evolution: causes of trends in amino-acid gain and loss.; citation_author=LD Hurst; citation_author=EJ Feil; citation_author=EP Rocha; citation_journal_title=Nature; citation_volume=442; citation_number=39; citation_pages=E11-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Patterns of nucleotide substitution in Drosophila and mammalian genomes.; citation_author=DA Petrov; citation_author=DL Hartl; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=40; citation_pages=1475-1479; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Neutral mutations and neutral substitutions in bacterial genomes.; citation_author=H Ochman; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=41; citation_pages=2091-2096; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Environments shape the nucleotide composition of genomes.; citation_author=KU Foerstner; citation_author=C von Mering; citation_author=SD Hooper; citation_author=P Bork; citation_journal_title=EMBO Rep; citation_volume=6; citation_number=42; citation_pages=1208-1213; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome evolution in bacterial endosymbionts of insects.; citation_author=JJ Wernegreen; citation_journal_title=Nature Review Genetics; citation_volume=3; citation_number=43; citation_pages=850-861; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=New metrics for comparative genomics.; citation_author=MY Galperin; citation_author=E Kolker; citation_journal_title=Curr Opin Biotechnol; citation_volume=17; citation_number=44; citation_pages=440-447; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Birth and death of orphan genes in Rickettsia.; citation_author=H Amiri; citation_author=W Davids; citation_author=SG Andersson; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=45; citation_pages=1575-1587; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=STRING 7–recent developments in the integration and prediction of protein interactions.; citation_author=C von Mering; citation_author=JJ L; citation_author=M Kuhn; citation_author=S Chaffron; citation_author=T Doerks; citation_journal_title=Nucleic Acids Res; citation_volume=10; citation_number=46; citation_pages=10; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Validation of Shewanella oneidensis MR-1 small proteins by AMT tag-based proteome analysis.; citation_author=MF Romine; citation_author=DA Elias; citation_author=ME Monroe; citation_author=K Auberry; citation_author=R Fang; citation_journal_title=Omics; citation_volume=8; citation_number=47; citation_pages=239-254; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Global profiling of Shewanella oneidensis MR-1: expression of hypothetical genes and improved functional annotations.; citation_author=E Kolker; citation_author=AF Picone; citation_author=MY Galperin; citation_author=MF Romine; citation_author=R Higdon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=48; citation_pages=2099-2104; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Towards a richer description of our complete collection of genomes and metagenomes: the “Minimal Information about a Genome Sequence”.; citation_author=D Field; citation_author=GM Garrity; citation_author=T Gray; citation_author=N Morrison; citation_author=JD Selengut; citation_journal_title=Nature Biotechnology (in review); citation_number=49; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=50; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER.; citation_author=AL Delcher; citation_author=D Harmon; citation_author=S Kasif; citation_author=O White; citation_author=SL Salzberg; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=51; citation_pages=4636-4641; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=EchoBASE: an integrated post-genomic database for Escherichia coli.; citation_author=RV Misra; citation_author=RS Horler; citation_author=W Reindl; citation_author= Goryanin; citation_author=GH Thomas; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=52; citation_pages=D329-333; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes"/>
    <meta name="twitter:description" content="BackgroundLineage-specific, or taxonomically restricted genes (TRGs), especially those that are species and strain-specific, are of special interest because they are expected to play a role in defining exclusive ecological adaptations to particular niches. Despite this, they are relatively poorly studied and little understood, in large part because many are still orphans or only have homologues in very closely related isolates. This lack of homology confounds attempts to establish the likelihood that a hypothetical gene is expressed and, if so, to determine the putative function of the protein.Methodology/Principal FindingsWe have developed &ldquo;QIPP&rdquo; (&ldquo;Quality Index for Predicted Proteins&rdquo;), an index that scores the &ldquo;quality&rdquo; of a protein based on non-homology-based criteria. QIPP can be used to assign a value between zero and one to any protein based on comparing its features to other proteins in a given genome. We have used QIPP to rank the predicted proteins in the proteomes of Bacteria and Archaea. This ranking reveals that there is a large amount of variation in QIPP scores, and identifies many high-scoring orphans as potentially &ldquo;authentic&rdquo; (expressed) orphans. There are significant differences in the distributions of QIPP scores between orphan and non-orphan genes for many genomes and a trend for less well-conserved genes to have lower QIPP scores.ConclusionsThe implication of this work is that QIPP scores can be used to further annotate predicted proteins with information that is independent of homology. Such information can be used to prioritize candidates for further analysis. Data generated for this study can be found in the OrphanMine at http://www.genomics.ceh.ac.uk/orphan_mine."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000324.g004"/>

  <meta property="og:title" content="Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5571'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5268'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3951&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000324">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000324" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000324&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gareth A. Wilson
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gawi@ceh.ac.uk">gawi@ceh.ac.uk</a></p>

                <p>Affiliation:
                  Centre for Ecology and Hydrology (CEH) Oxford, Oxford, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edward J. Feil, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Bath, Bath, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Andrew K. Lilley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Ecology and Hydrology (CEH) Oxford, Oxford, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Dawn Field
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre for Ecology and Hydrology (CEH) Oxford, Oxford, United Kindgom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000324</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000324-g001" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-t001" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-g002" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-g003" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-g004" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-t002" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-t003" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000324-t004" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000324">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000324" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Lineage-specific, or taxonomically restricted genes (TRGs), especially those that are species and strain-specific, are of special interest because they are expected to play a role in defining exclusive ecological adaptations to particular niches. Despite this, they are relatively poorly studied and little understood, in large part because many are still orphans or only have homologues in very closely related isolates. This lack of homology confounds attempts to establish the likelihood that a hypothetical gene is expressed and, if so, to determine the putative function of the protein.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have developed “QIPP” (“Quality Index for Predicted Proteins”), an index that scores the “quality” of a protein based on non-homology-based criteria. QIPP can be used to assign a value between zero and one to any protein based on comparing its features to other proteins in a given genome. We have used QIPP to rank the predicted proteins in the proteomes of Bacteria and Archaea. This ranking reveals that there is a large amount of variation in QIPP scores, and identifies many high-scoring orphans as potentially “authentic” (expressed) orphans. There are significant differences in the distributions of QIPP scores between orphan and non-orphan genes for many genomes and a trend for less well-conserved genes to have lower QIPP scores.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The implication of this work is that QIPP scores can be used to further annotate predicted proteins with information that is independent of homology. Such information can be used to prioritize candidates for further analysis. Data generated for this study can be found in the <em>OrphanMine</em> at <a href="http://www.genomics.ceh.ac.uk/orphan_mine">http://www.genomics.ceh.ac.uk/orphan_min​e</a>.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wilson GA, Feil EJ, Lilley AK, Field D (2007) Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes. PLoS ONE 2(3):
          e324.
            doi:10.1371/journal.pone.0000324</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> January 26, 2007; <strong>Accepted:</strong> February 18, 2007; <strong>Published:</strong> March 28, 2007</p><p><strong>Copyright:</strong> © 2007 Wilson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was funded through a Natural Environment Research Council Ph.D studentship awarded to GW (NER/S/A/2003/11341).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The availability of hundreds of complete bacterial genome sequences has made it possible to explore how the evolutionary diversification of gene content reflects the ecological needs and opportunities of different taxa. It is well known that the gene content of bacterial and archaeal genomes can vary widely and that only a very few genes are truly universal <a href="#pone.0000324-Tatusov1">[1]</a>–<a href="#pone.0000324-Ciccarelli1">[3]</a>. As a consequence, genes can differ significantly in their taxonomic distributions, with more broadly conserved genes having ‘housekeeping’ functions and less conserved genes being responsible for the phenotypic differences observed between organisms. Lineage-specific, or “taxonomically restricted” genes (TRGs), are defined as being exclusively restricted to a particular taxonomic group <a href="#pone.0000324-Wilson1">[4]</a>. In such a framework, genes may be TRGs at any taxonomic level (<em>i.e.</em> domain-, family, genus-, species- or strain-specific). TRGs at the species and strain-levels are of most interest in the search for genotypes which help define exclusive ecological adaptations to particular niches.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The study of narrowly distributed TRG's is confounded by the fact that many are short, repetitive or have unusual A+T contents <a href="#pone.0000324-Daubin1">[5]</a>, and the assumption that many such short coding sequences (CDS) represent annotation errors <a href="#pone.0000324-Skovgaard1">[6]</a>. Over-annotation of genomes, resulting in an excess of small predicted proteins is clearly evident in certain genomes (<em>e.g</em> the initial annotation of <em>Aeropyrum pernix</em> <a href="#pone.0000324-Kawarabayasi1">[7]</a>) and is proposed to be an unfortunate feature of many genomic annotations <a href="#pone.0000324-Skovgaard1">[6]</a>, <a href="#pone.0000324-Fukuchi1">[8]</a>, <a href="#pone.0000324-Ussery1">[9]</a>. This overannotation could mask intergenic regions containing small non-coding RNAs. It is also possible that many TRGs remain ‘orphaned’ for no other reason than the sampling bias in public genome databases <a href="#pone.0000324-Siew1">[10]</a>. It is well-known that the current collection is highly biased towards certain organisms (most notably pathogens, γ-Proteobacteria, and Firmicutes) <a href="#pone.0000324-Martiny1">[11]</a>. This results in the trend that taxonomic isolation is correlated with an increased percentage of orphans <a href="#pone.0000324-Fukuchi1">[8]</a>. It is therefore expected that homologues for many orphan predicted proteins in taxonomically isolated lineages that lack close relatives in genomic databases will be found once the taxonomic gaps in the genomic database begin to be filled <a href="#pone.0000324-Siew1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Despite potential errors in our current estimation of the numbers and identities of narrowly distributed TRGs, there is growing evidence that many, including those that are currently orphaned, are of biological significance. Hence there is a growing need to untangle erroneous CDS from authentic species- and strain-level TRGs <a href="#pone.0000324-Alimi1">[12]</a>–<a href="#pone.0000324-Shmuely1">[14]</a>. Dispersed examples of the latter are most frequently found as the result of in depth <em>in silico</em> <a href="#pone.0000324-Daubin1">[5]</a> or empirical studies <a href="#pone.0000324-Alimi1">[12]</a> of a particular organism or small group of organisms. Increasingly, examples are being identified as the result of whole genome sequencing <a href="#pone.0000324-Shmuely1">[14]</a>. One example to come from complete genome sequencing is the TCP virulence locus of <em>Vibrio cholerae</em> Tor N16961. Once a cluster of largely orphaned CDS, a homologous region has recently been found in the squid symbiont <em>Vibrio fischeri</em> <a href="#pone.0000324-Ruby1">[15]</a>. The TCP genes code for the toxin co-regulated pili in <em>V. cholerae</em> and serve as its critical intestinal colonisation factor, providing the receptor for entry of the temperate filamentous phage CTX<sup>Φ</sup>, which contains the cholera toxin genes, <em>ctxAB</em> <a href="#pone.0000324-Waldor1">[16]</a> into the cell <a href="#pone.0000324-Manning1">[17]</a>. Likewise, the sequencing of many genomes is confirming the presence of many strain-specific genes which form the “pan-genome” of many species <a href="#pone.0000324-Tettelin1">[18]</a>, <a href="#pone.0000324-Medini1">[19]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Given the potential significance of orphaned and narrow-range TRGs and the confounding sources of error associated with currently annotated genomes, it is clear that a reliable objective measure of the potential ‘quality’ of a given CDS would be useful [See 20 for the scoring of CDS with homology]. This could be used to prioritize it either as a candidate for further characterization or as an error. Motivated by this requirement, and with a specific focus on orphans and narrow-range TRGs, we have devised a scoring system that allows the ‘ranking’ of predicted proteins based on a variety of features, reflecting the likelihood that a given CDS encodes a protein.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We previously reported that the absolute number of single-copy TRGs from the complete and published genomes of Bacteria and Archaea is increasing <a href="#pone.0000324-Wilson1">[4]</a>. The most phylogenetically and ecologically unique species contribute the most unique genes, in part due to undersampling of these genetic lineages <a href="#pone.0000324-Wilson1">[4]</a>. For that study we generated two datasets. The first contained all orphans as defined by BLAST (using a threshold of 10<sup>−3</sup>), the second applied an arbitrary length cutoff of ≥150 amino acids and excluded all CDS with low complexity (highly repetitive) regions to remove likely CDS enriched in artefacts. The method of scoring CDS described here extends this ‘selective filtering’ approach and is called the ‘Quality Index for Predicted Proteins’ (QIPP). We describe the use of QIPP as it is applied to the reanalysis of this data set based on the inclusion of five criteria selected for their presumed ability to detect purifying selection and CDS which are unlikely to occur by chance alone. These are length <a href="#pone.0000324-Skovgaard1">[6]</a>, percentage low complexity (a measure of the degree of repetition) <a href="#pone.0000324-Altschul1">[21]</a>, difference in G+C composition of sequence and genome <a href="#pone.0000324-Navarre1">[22]</a>, average amino acid cost <a href="#pone.0000324-Akashi1">[23]</a>, <a href="#pone.0000324-Heizer1">[24]</a> and neighbourhood distribution (ND) <a href="#pone.0000324-Zheng1">[25]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>The orphan and non-orphan components of many proteomes have different overall characteristics</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To examine whether orphaned CDS, which are expected to be on average smaller <a href="#pone.0000324-Skovgaard1">[6]</a> and more A+T rich <a href="#pone.0000324-Daubin1">[5]</a>, <a href="#pone.0000324-Yin1">[26]</a> have significantly different QIPP scores than non-orphans, we re-examined our original data set <a href="#pone.0000324-Wilson1">[4]</a>. QIPP scores were calculated for each protein in this data set of 122 proteomes <a href="#pone.0000324-Wilson1">[4]</a> as described in the Materials &amp; Methods. In total, the distributions of all five criteria (length, low complexity, G+C content, amino acid cost and neighbourhood distribution (<a href="#pone-0000324-t001">Table 1</a>)) differ significantly between orphans and non-orphans in 61 of the 122 species examined (p&lt;0.05, Mann-Whitney). 3 or more criteria are significant in 117/122 species. Four of the remaining five species contained fewer than 10 orphans, and when all such genomes (n = 6) were excluded 115 of the remaining 116 species had orphans that differed significantly from the non-orphans for three or more criteria. The strikingly different values for <em>Escherichia coli</em> K12 can be seen in <a href="#pone-0000324-g001">Figure 1</a> as an example of these trends. The distribution of the QIPP scores for orphan and non-orphan TRG's were found to be significantly different for 119 of the 122 genomes (p&lt;0.05, Mann-Whitney). The remaining three genomes contained 2 or less orphans and thus could not provide significant discriminatory power. Overall, the QIPP scores for all orphan (mean = 0.38, +0.14) and non-orphan (mean = 0.54, +0.14) TRG's were significantly different (p = 0.000, Mann-Whitney). These results confirm that the criteria used for the QIPP scores can reliably distinguish between “orphan-like” (less well conserved) and “non-orphan-like” (more widely conserved) genes.</p>
<div class="figure" id="pone-0000324-g001"><div class="img"><a name="pone-0000324-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Distributions of orphans and non-orphans in E.coli K12.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The predicted proteins in E.coli K12 that were found to be unique (light gray) when compared to 122 bacterial proteomes (shown in <a href="#pone.0000324.s001">Table S1</a>) were designated as orphans (n = 174). All remaining proteins (dark gray) were non-orphans (n = 4137). Distributions of values for both groups were calculated as a percentage for (a) length, (b) percent low complexity, (c) G+C difference from the mean, (d) Cost and (e) Neighbourhood Distribution.</p>
<span>doi:10.1371/journal.pone.0000324.g001</span></div><div class="figure" id="pone-0000324-t001"><div class="img"><a name="pone-0000324-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Criteria used for the calculation of QIPP</span></strong></p><span>doi:10.1371/journal.pone.0000324.t001</span></div>

<h4>Ranking orphan CDS using QIPP scores</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The distribution of QIPP scores across the orphans in this data set was examined to determine if there was sufficient variation to rank them. <a href="#pone-0000324-g002">Figure 2a</a> shows that QIPP scores range from 0.0 to 0.9 (out of a possible range from zero to one) and so the index does have discriminatory power. The overall QIPP scores for each proteome deviate from the normal distribution for all five reference genomes, with too few high-scoring CDS and a longer than expected left-hand tail of low-scoring proteins (Darling-Anderson p&lt;.005). This is due to the fact that for each criterion (with the exception of low complexity) there are few proteins with very high ranks (<a href="#pone-0000324-g002">Figure 2b–f</a>).</p>
<div class="figure" id="pone-0000324-g002"><div class="img"><a name="pone-0000324-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>QIPP and Criterion Distributions of orphans in 122 bacterial genomes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The orphans (n = 43513) obtained from 122 bacterial genomes were scored and the distribution plotted according to (a) QIPP and the individual criteria that constitute QIPP: (b) length, (c) percent low complexity, (d) G+C difference from the mean, (e) cost and (f) Neighbourhood Distribution.</p>
<span>doi:10.1371/journal.pone.0000324.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We then examined the quality of the highest-scoring orphans to see if our list contained a significant number of potentially ‘authentic’ orphans–i.e. those unlikely to occur by chance. The extreme right hand distribution of these QIPP scores contains a total of 2,010 single-copy TRGs (≥95<sup>th</sup> percentile with a minimum score of 0.62), 1,260 are longer than 200 amino acids, a criterion that when used in isolation, is generally accepted to signify ‘authentic’ CDS <a href="#pone.0000324-Skovgaard1">[6]</a>. Relaxing the QIPP score threshold, and using only length as a criterion, a total of 9858 (22.66%) single-copy TRGs are found in this data set which are ≥200 amino acids. A subset of these, 2,445 (5.62%), are ≥400 amino acids.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>When interpreting the origins of such high-quality single-copy TRGs, the taxonomic uniqueness of each parent genome must be considered. Of those with QIPP scores above the 95th percentile (&gt; = 0.62), only 467 (23%) are from 62 species (8 per genome) sampled down to the species level (<em>i.e.</em> another species from the same genus is available in the data set) (average QIPP score = 0.66). In contrast, 1,543 (77%) originate from 60 species which only have more distant relatives in this data set. It is presumed that these genomes include many TRGs exclusive to higher taxonomic levels; 24 genomes are unique at the genus level (259 orphans, 11 per genome, average QIPP score = 0.66), 30 at the family level (931 orphans, 31 per genome, average QIPP score = 0.67) and 6 at the division level (353 orphans, 59 per genome, average QIPP score = 0.67). Of those larger than 200 amino acids, 2,878 (29%) are from 62 species (46 per genome) sampled down to the species with an average QIPP score of 0.43. The remaining 6,980 (71%), originate from 60 species unique at the genus level (1,439 total, 60 orphans per genome, average QIPP score = 0.44), the family level (4,263 total, 142 orphans per genome, average QIPP score = 0.48) and the division level (1,278 total, 213 orphans per genome, average QIPP score = 0.50).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>When plotted against genetic similarity, more distantly related genomes contribute on average more high-quality, single-copy TRGs (<a href="#pone.0000324.s001">Table S1</a> and <a href="#pone.0000324.s002">Figure S1</a>). Chi-squared tests were used to identify genomes that made a greater contribution than expected to the top 50% of the ranked list (<a href="#pone-0000324-g003">Figure 3</a>). Genomes that did not contain enough orphans (&gt;5) to perform a chi-squared test were removed from the analysis (n = 6). Genomes that contribute more high ranking QIPP scores are more distantly related (<a href="#pone-0000324-g003">Figure 3</a>, Anova p = 0.000) but only a low proportion of variability in top-ranking scores is explained by a regression analysis (p = 0.000, R-squared = 10.63%).</p>
<div class="figure" id="pone-0000324-g003"><div class="img"><a name="pone-0000324-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Genomes which are more taxonomically isolated have larger numbers of high-scoring orphan predicted proteins.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Chi-squared tests were used to determine which genomes had significantly more predicted proteins in the top 50% of the list of ranked orphan predicted proteins than would be expected by chance (−1 = significantly less orphans than expected in top 50% rank, 0 = no significant difference and 1 = significantly more orphans than expected in top 50% rank).</p>
<span>doi:10.1371/journal.pone.0000324.g003</span></div>

<h4>Less conserved genes have lower QIPP scores</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The difference between orphan and non-orphan QIPP scores suggests that it might be possible to predict <em>a priori</em> how conserved a particular CDS might be using QIPP scores in the absence of homology. To explore this further, we selected a subset of five reference genomes from the best-sampled taxa in our original dataset for which intra-specific comparisons yielding high numbers of strain-specific orphans were also available (<a href="#pone-0000324-t002">Table 2</a>). For each reference genome the taxonomic distribution of all predicted proteins at the Archaea/Bacteria level, domain, division, family, genus, species and strain level (<a href="#pone-0000324-g004">Figure 4</a>) was determined.</p>
<div class="figure" id="pone-0000324-g004"><div class="img"><a name="pone-0000324-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Calculated QIPP scores for 5 bacterial genomes split into taxonomic classes.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Every predicted protein in (a) <em>E.coli</em> K12, (b) <em>H.pylori</em> 26695, (c) <em>N.meningitides</em> MC58, (d) <em>P.marinus</em> CCMP1375 and (e) <em>V.vulnificus</em> CMCP6 was put into the taxonomic level at which it was restricted and scored according to QIPP.</p>
<span>doi:10.1371/journal.pone.0000324.g004</span></div><div class="figure" id="pone-0000324-t002"><div class="img"><a name="pone-0000324-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Numbers and percentages of species-specific and strain-specific genes after the addition of a second strain in five bacterial species.</span></strong></p><span>doi:10.1371/journal.pone.0000324.t002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>The average QIPP scores and percentages of predicted proteins exclusive to each of these taxonomic levels are given in <a href="#pone-0000324-t003">Table 3</a>. Overall, average scores are relatively uniform across the five genomes at each of the 7 taxonomic levels examined. Scores range from an average of 0.60 for proteins conserved across bacteria and archaea down to 0.35 for proteins conserved at the strain-level. These average scores are significantly different across TRG's exclusive to different taxonomic levels (Anova, p = 0.000 for every genome). The data show an overall decrease in QIPP score as the degree of conservation narrows (<a href="#pone-0000324-g004">Figure 4</a>). For the five genomes, when all CDS are taken into account, a regression analysis provides a p-value of 0.000 with R-squared values ranging from 20.3% to 36.3%.</p>
<div class="figure" id="pone-0000324-t003"><div class="img"><a name="pone-0000324-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Table showing the average QIPP score for predicted proteins at each taxonomic level for five selected bacterial genomes.</span></strong></p><span>doi:10.1371/journal.pone.0000324.t003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The differences in mean QIPP scores between different groups of TRG's are largest for comparisons between groups of CDS conserved above the level of division and those conserved at the species- and strain-level (<a href="#pone-0000324-t003">Table 3</a>). Still, average QIPP scores, are significantly different between all higher TRG groups when compared to the average for species-level TRGs, while groups of species- and strain-level TRGs cannot be distinguished (<a href="#pone-0000324-t004">Table 4</a>). Interestingly, scores from the gene prediction software Glimmer could be used to separate only 7 of the 15 comparisons presented in <a href="#pone-0000324-t004">Table 4</a>. Hence QIPP provides additional information which is useful for post-processing gene predictions such as those made by Glimmer, in the absence of homology.</p>
<div class="figure" id="pone-0000324-t004"><div class="img"><a name="pone-0000324-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000324" data-uri="info:doi/10.1371/journal.pone.0000324.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000324.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000324.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000324.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000324.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Statistical significance of QIPP (Q) and Glimmer (G) scores when differentiating between species-specific genes and a respective taxonomic rank.</span></strong></p><span>doi:10.1371/journal.pone.0000324.t004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>In addition to using QIPP to rank individual CDS, we also investigated whether the data had biological meaning. Using quartile analysis, 50% of the CDS in each of these genomes fall uniformly between the absolute values of 0.43 and 0.64 (<a href="#pone-0000324-t003">Table 3</a>), suggesting rule of thumb cut-offs for QIPP scores associated with the least (below 0.43) and most (above 0.64) highly conserved CDS in a genome. The data further suggest that the most extreme values of QIPP have the highest degree of predictive power for level of conservation (<a href="#pone-0000324-g004">Figure 4</a>). For example, using a minimum threshold score of ≥0.8, 98% of all CDS are members of the most conserved gene families (above the division-level). A total of 58% of CDS with scores less than 0.2 are species- and strain-specific TRGs.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>To observe the range of QIPP scores that might be expected from the most highly conserved CDS we examined a subset of universally conserved genes <a href="#pone.0000324-Ciccarelli1">[3]</a>. We found the homologues of these 31 previously defined protein families <a href="#pone.0000324-Ciccarelli1">[3]</a> in the <em>E. coli</em> K12 genome and examined their QIPP scores. These QIPP scores range from 0.5 to 0.87 with a mean of 0.69 (±0.099). A large number of these proteins are ribosomal proteins, which are all of shorter than average size for <em>E. coli</em>. QIPP score is very poorly correlated with the overall length of these proteins (R-squared = 0.012) suggesting that QIPP is not overly sensitive to any one component criterion. The two highest-scoring proteins, both with a QIPP score of 0.89, are extremely different in length (1,138 for the DNA-directed RNA polymerase, beta subunit versus 323 for the DNA-directed RNA polymerase, alpha subunit). When length is removed as a component criterion of QIPP, the scores of the shortest proteins increase by up to 0.16, while those of the very longest proteins decrease by a maximum of 0.09 giving a new mean value of 0.75 (±0.141).</p>


<h4>Validation of orphans with low QIPP scores using results from transcriptomic and proteomic studies</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To test whether we could validate the expression of orphans with low QIPP scores in a well-studied model organism, we searched the MicrobesOnline database <a href="#pone.0000324-Alm1">[27]</a> for <em>E.coli</em> K12 orphans identified in this study. This database provides experimental microarray results for this organism for four stress conditions: heat shock <a href="#pone.0000324-GutierrezRios1">[28]</a>, pH <a href="#pone.0000324-Kang1">[29]</a>, UV exposure <a href="#pone.0000324-Courcelle1">[30]</a> and tryptophan metabolism <a href="#pone.0000324-Khodursky1">[31]</a>. We examined the fifty highest and lowest ranked species-level TRGs (N = 100). The scores of the top ranking CDS ranged from 0.41–0.64 and the bottom from 0.02–0.28. To illustrate the range of CDS involved, the top scoring CDS was 547 amino acids in length, zero percent low complexity, average G+C content, but was more costly than average and came from a poorly characterized region of the genome. By contrast, the CDS with the lowest score of 0.02 was only 60 amino acids in length, 35% low complexity, had a highly deviant base composition, it was also more costly than average and was found in a poorly characterized region of the genome. Of these 100 orphans, 17 had identifiers not found in the MicrobesOnline database and were excluded. Of the remaining 83, only one failed to show any change in expression levels in any of these experiments. In total there were 46 occasions (involving 35 of these 100 orphans) when one of these orphans was included in the list of the 200 proteins reported in Microbes Online showing the largest (up or down) fold change in expression in one of these experiments. Of particular interest was the pH stress experiment where 12 (three in the top and nine from the bottom 50) of the top 100 up-regulated genes were orphans (p = 0.00, chi-square).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p><em>E.coli</em> K12 proteomic data sets <a href="#pone.0000324-Corbin1">[32]</a>–<a href="#pone.0000324-Taoka1">[34]</a> were also searched. When combined these investigations identified approximately 1,800 expressed proteins. While mRNA was found for 64 of the 174 CDS in <em>E. coli</em>, only 4 proteins could be identified for all 174 single-copy TRGs in this data set. These four CDS had an average QIPP score of 0.32 compared to mean score of 0.35 for all <em>E.coli</em> orphans.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We have developed an index called “QIPP” (“Quality Index for Predicted Proteins”) which can be used to assign a value between zero and one for a CDS compared to the rest of the genome on the basis of a set of selective criteria. This provides an objective measure of the probability that a given CDS either encodes a protein or is an annotation artefact (incorrect). Very long CDS, with typical nucleotide and amino-acid compositions, no low complexity regions, and which are found in well conserved regions have the highest QIPP scores and are considered most likely to encode proteins.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The distributions of QIPP scores, and trends in the component variables, confirm that orphans show consistent differences when compared with well characterised protein-coding genes <em>i.e.</em> they are short, repetitive, possess atypical G+C content, have high average cost for amino acids and are located in poorly characterised regions of the genome. The significant differences in the distributions of QIPP scores between orphan genes and non-orphan genes confirms that QIPP scores represent a valid means to rationalise and automate the identification of those CDS most likely to encode proteins (and find homologues among other available sequences). Because orphans generally have low QIPP scores it is also possible to meaningfully rank them as a subset of all CDS, selectively filter for high-scoring ‘authentic’ orphans, and begin to address the issue of correcting for the high percentage of orphans in current databases that are simply an artefact of sampling bias.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our data show that the lowest-scoring CDS encode the least evolutionary conserved proteins, <em>i.e</em>. those orphans restricted to single strains or species. As such, this approach can also provide evidence on the likely taxonomic range of a CDS in the absence of any useful homology. This is particularly significant given the unrepresentative sampling of the current genomic databases. Low-scoring, taxonomically restricted orphans are most likely to be annotation artefacts: we tested this in the case of <em>E. coli</em> K12 by reference to online transcriptomic and proteomic expression data. Surprisingly, these data revealed that even these low-scoring CDS are potentially expressed (given the caveats associated with using microarray data to validate orphans <a href="#pone.0000324-Skovgaard1">[6]</a> and the fact that <em>E.</em> coli is one of the most thoroughly studied organisms) and therefore suggest that annotation artefacts may not be as common as previously suspected. It is clear that empirical validation of genomic annotations is necessary and should be of the highest priority <a href="#pone.0000324-Roberts1">[35]</a>–<a href="#pone.0000324-Galperin1">[37]</a>. At a minimum it would appear premature to dismiss all very low-scoring orphans as having little biological relevance without further evidence.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>In effect, the criteria used in the QIPP score reflect the extent of purifying selection acting upon a sequence, which, in the absence of homology, preclude the use of more widely-used methods such as examination of dN/dS ratios <a href="#pone.0000324-Nei1">[38]</a>. Purifying selection should over time preferentially purge substitutions leading to the use of more metabolically costly amino-acids <a href="#pone.0000324-Hurst1">[39]</a>. Similarly, mutation pressure tends to move in the direction GC-&gt;AT rather than <em>vice versa</em> <a href="#pone.0000324-Petrov1">[40]</a>, <a href="#pone.0000324-Ochman1">[41]</a> and AT enrichment has commonly been cited as a footprint for relaxed or inefficient purifying selection (but see <a href="#pone.0000324-Foerstner1">[42]</a>). This can explain the high AT content of obligate endosymbionts or intracellular parasites which are adapted to a restricted niche, undergo restricted gene exchange, and possibly mutate at a high rate due to the loss of DNA repair genes <a href="#pone.0000324-Wernegreen1">[43]</a>. It is also well documented that phage and other mobile elements tend to show a higher AT content than the host bacterial genome <a href="#pone.0000324-Daubin1">[5]</a>, <a href="#pone.0000324-Hurst1">[39]</a>. As highly conserved proteins are likely to encode essential housekeeping functions, and therefore be subject to high levels of purifying selection, the noted correlation between the taxonomic range and QIPP score can be explained within this selective framework. This phenomenon also provides further validation for the use of the QIPP score in identifying “real” genes, as it is expected that CDS which are simply annotation artefacts should be evolving neutrally and hence have very low QIPP scores.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>This analysis provides proof of principal that the combined use of different criteria can be a powerful approach to determining the biological relevance of putative CDS. The power of the QIPP score could be improved by the use of additional criteria which are likely to reflect purifying selection, such as codon bias, for example. It is acknowledged that the criteria presently used are unlikely to be independent, and multivariate analysis is required to determine the interactions between the variables and to put corrections in place to improve the predictive power of the index. Preliminary analysis on five reference genomes has revealed a significant correlation (p&lt; = 0.05) between sequence length and complexity, with longer proteins showing more low complexity regions. Further, a significant correlation between G+C content and amino-acid cost was noted in four out of five genomes (the exception being <em>V. vulnificus</em>; data not shown).</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>There is a growing need for metrics that offer a deeper understanding of the detailed content of genomes, especially now that we have such large numbers <a href="#pone.0000324-Galperin2">[44]</a>. QIPP provides such a metric and can be used in combination with other <em>in silico</em> methods that can now be used to sift out potentially authentic orphans and improve genomic annotation. Such complementary methods include the analysis and removal of short CDS <a href="#pone.0000324-Skovgaard1">[6]</a>, gene fragments <a href="#pone.0000324-Amiri1">[45]</a>, and pseudogenes <a href="#pone.0000324-Fukuchi1">[8]</a> and the ranking of CDS based on the availability of homology-based information <a href="#pone.0000324-Kosuge1">[20]</a>. Integration of the information from such studies would provide the foundation for a single, global list of uncharacterized predicted proteins that could be used to systematically subject them to further <em>in silico</em> examination <a href="#pone.0000324-Kosuge1">[20]</a>, <a href="#pone.0000324-Roberts1">[35]</a>, <a href="#pone.0000324-Galperin1">[37]</a>. This data set could further be integrated with empirical evidence from a range of experimental studies, especially high throughput ‘omic studies, as is the case for databases like STRING <a href="#pone.0000324-vonMering1">[46]</a>. <em>In silico</em> studies of predicted proteins can help identify candidates for further examination, but any validation of the biological relevance of a particular protein must be based on empirical evidence <a href="#pone.0000324-Kolker1">[13]</a>, <a href="#pone.0000324-Roberts1">[35]</a>, <a href="#pone.0000324-Romine1">[47]</a>, <a href="#pone.0000324-Kolker2">[48]</a>. In order to comply with the principle of the transparent access to data for the sake of integration <a href="#pone.0000324-Field1">[49]</a>, all of the data generated in this study is available online in a searchable database, the <em>OrphanMine</em>, a database that supports wide-scale downloads of data, including lists of CDS with rich annotations in GFF3 (Generic Feature Format Version 3) (<a href="http://song.sourceforge.net/gff3.shtml">http://song.sourceforge.net/gff3.shtml</a>) format.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In conclusion, the QIPP index supports an objective rationale for prioritising predicted genes for further study, including ‘authentic’ single-copy TRGs. Although further work is required to refine the approach, this represents an important step in the standardisation and automation of identifying biologically important genes in the absence of homology.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Material and Methods"></a><h3>Material and Methods</h3>
<h4>Processing of Genomes and Proteomes</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>All genomic annotations and proteomes as both amino acid and DNA were downloaded from the NCBI Refseq FTP site. Orphans were detected as previously described <a href="#pone.0000324-Wilson1">[4]</a> using NCBI Blast <a href="#pone.0000324-Altschul2">[50]</a> and a cutoff of 10<sup>−3</sup> and then loaded into the <em>OrphanMine</em> database for post-processing. The <em>OrphanMine</em> interface was used to generate groups of TRGs for each taxonomic level. A custom Perl script was used to calculate length, G+C content and cost and to parse blast reports to generate a “neighbourhood distribution” (ND) for each CDS. All of the data used in this study is publicly available through the <em>OrphanMine</em>. The code used to generate lists of orphans from proteomes is available in the YAMAP package (<a href="http://www.genomics.ceh.ac.uk/yamap/">www.genomics.ceh.ac.uk/yamap/</a>) and all other code (any additional Perl scripts) is available on request <a href="mailto:(gawi@ceh.ac.uk)">(gawi@ceh.ac.uk)</a>.</p>


<h4>Calculation of QIPP scores</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For each genome and for each of the five selected criteria, the distribution of non-orphans was generated and the percentiles for that distribution were calculated. For the criteria of length and ND, the absolute value of each component criterion (e.g. length of 200 amino acids) was transformed into a sub-score from 0 to 100 depending on the percentile in which it fell (e.g. the 35<sup>th</sup> percentile from the shortest CDS found would be given a score of 35). For low complexity and cost, where more of either actually suggests a less probable CDS, the score was subtracted from 100 (e.g. a protein with 50% low complexity might fall in the 70<sup>th</sup> percentile and therefore be given a low score of 30). G+C content had to be calculated as the deviation from the mean value. Values above the 50th percentile were corrected by the equation 100 minus the percentile value multiplied by two and values below had their percentile doubled.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Length was calculated as the total number of amino acids and percentage low complexity regions was calculated from regions masked with the seg program <a href="#pone.0000324-Altschul1">[21]</a> using default parameters. G+C content was calculated from the proteome as DNA. The average amino acid cost of a sequence was calculated using the relative costs for each amino acid according to the values given in Akashi &amp; Gojobori <a href="#pone.0000324-Akashi1">[23]</a>. Randomized proteomes (<em>i.e.</em> any sequence evolving neutrally) are the most costly as purifying selection appears to select for amino acids that are less metabolically expensive <a href="#pone.0000324-Akashi1">[23]</a>. ND was calculated by determining the level of conservation of the five flanking CDS on either side of a particular CDS. For each of these ten genes, the number of species in which a similar sequence was found was recorded (maximum of 121 for this dataset). Those numbers were then summed, averaged and percentiles generated for the distribution.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>The scores from all five criteria are normalized with respect to each particular genome and can therefore be summed. To obtain a final QIPP score between zero and 1, the average is taken and divided by 100. Zero would be the worst possible candidate for a real gene while 1 would be ideal. Using the interface to the <em>OrphanMine</em> it is possible to perform user-selected rankings of subsets of the CDS held in the database on the basis of 1 or all of the component criteria used in QIPP. To compare QIPP and Glimmer scores, the five reference genomes were run through Glimmer (v2.13) <a href="#pone.0000324-Delcher1">[51]</a> with default settings.</p>


<h4>Genetic Similarity of genomes and the taxonomic distribution of TRGs</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The Index of Isolation of an Organism (IIO) similarity measure was calculated by averaging the logarithm of the best e-value for each CDS in a proteome as described by Fukuchi &amp; Nishikawa <a href="#pone.0000324-Fukuchi1">[8]</a>. The taxonomic distribution of each CDS in the five references genomes (<a href="#pone-0000324-t003">Table 3</a>) was obtained through interrogation of the <em>OrphanMine</em> database <a href="#pone.0000324-Wilson1">[4]</a>. For each genome, appropriate queries were performed to find genes restricted to each taxonomic level. The output was scored and downloaded in a tab-delimited format. A Perl script was written to parse the output to ensure that every predicted protein was only counted once and each protein could be classed according to its lineage specificity.</p>


<h4>Obtaining Empirical Data from Microarray and Proteomic Studies</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The MicrobesOnline database <a href="#pone.0000324-Alm1">[27]</a> was queried for the <em>E.coli</em> orphan genes using their unique VIMSS ID. A file was provided by Keith Keller to map the GenBank IDs of the orphan genes obtained from <em>OrphanMine</em> to the VIMSS ID. EchoBASE is a database that curates information regarding the genes and gene products of the model bacterium <em>E. coli</em> K-12, including links to literature describing proteomic analyses of this bacterium <a href="#pone.0000324-Misra1">[<em>52</em>]</a>. The ‘b number’ identifiers provided in the literature were used to map data from the proteomic analyses to the <em>E.coli</em> orphan genes obtained from <em>OrphanMine</em>. When ‘b numbers’ were not provided, the gene name, if present, was used.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000324"></div><a name="pone.0000324.s001" id="pone.0000324.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000324.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">The number of predicted proteins, orphans, percentage orphans, isolation index and taxonomic uniqueness for each of the 122 bacterial genomes used in this analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0000324.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.22 MB DOC)</p>
<a name="pone.0000324.s002" id="pone.0000324.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000324.s002">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Relationship between the numbers of orphans as a percentage of total predicted proteins and Isolation Index of an Organism. The IIO for each genome in our dataset (full list of genomes given in <a href="#pone.0000324.s001">Table S1</a>) is plotted against (a) percentage of orphans, (b) the number of orphans greater than 200 aa's and (c) the percentage of total orphans greater than 200 aa's in length. In addition, each genome is classed according to the taxonomic level at which it is the only sequenced representative.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000324.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.29 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Andrew Spiers, Rob Edwards, Dave Ussery, Chris Taylor, Peter Sterk, Mike Allen, Anil Wipat, Alice MorningStar, Lisa Crossman, Jack Gilbert and Sarah Turner for critical reading of the manuscript and Keith Keller for mapping of unique identifiers for use in the MicrobesOnline database <a href="#pone.0000324-Alm1">[27]</a>.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: EF DF GW. Performed the experiments: GW. Analyzed the data: DF GW. Contributed reagents/materials/analysis tools: GW. Wrote the paper: DF GW. Other: Provided expert statistical input which made the paper possible: AL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000324-Tatusov1" id="pone.0000324-Tatusov1"></a>Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, et al.  (2003) The COG database: an updated version includes eukaryotes. BMC Bioinformatics  4: 41.  Epub 2003 Sep 2011.  <ul class="find" data-citedArticleID="940308"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+COG+database%3A+an+updated+version+includes+eukaryotes.&amp;auth=&amp;atitle=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+COG+database%3A+an+updated+version+includes+eukaryotes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+COG+database%3A+an+updated+version+includes+eukaryotes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000324-Charlebois1" id="pone.0000324-Charlebois1"></a>Charlebois RL, Doolittle WF (2004) Computing prokaryotic gene ubiquity: rescuing the core from extinction. Genome Res  14: 2469–2477.  <ul class="find" data-citedArticleID="940234" data-doi="10.1101/gr.3024704"><li><a href="http://dx.doi.org/10.1101/gr.3024704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computing+prokaryotic+gene+ubiquity%3A+rescuing+the+core+from+extinction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computing+prokaryotic+gene+ubiquity%3A+rescuing+the+core+from+extinction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000324-Ciccarelli1" id="pone.0000324-Ciccarelli1"></a>Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, et al.  (2006) Toward automatic reconstruction of a highly resolved tree of life. Science  311: 1283–1287.  <ul class="find" data-citedArticleID="940236" data-doi="10.1126/science.1123061"><li><a href="http://dx.doi.org/10.1126/science.1123061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+automatic+reconstruction+of+a+highly+resolved+tree+of+life.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000324-Wilson1" id="pone.0000324-Wilson1"></a>Wilson GA, Bertrand N, Patel Y, Hughes JB, Feil EJ, et al.  (2005) Orphans as taxonomically restricted and ecologically important genes. Microbiology  151: 2499–2501.  <ul class="find" data-citedArticleID="940318" data-doi="10.1099/mic.0.28146-0"><li><a href="http://dx.doi.org/10.1099/mic.0.28146-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Orphans+as+taxonomically+restricted+and+ecologically+important+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Orphans+as+taxonomically+restricted+and+ecologically+important+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000324-Daubin1" id="pone.0000324-Daubin1"></a>Daubin V, Ochman H (2004) Bacterial Genomes as new gene homes: The genealogy of ORFans in E-coli. Genome Research  14: 1036–1042.  <ul class="find" data-citedArticleID="940242" data-doi="10.1101/gr.2231904"><li><a href="http://dx.doi.org/10.1101/gr.2231904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+Genomes+as+new+gene+homes%3A+The+genealogy+of+ORFans+in+E-coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+Genomes+as+new+gene+homes%3A+The+genealogy+of+ORFans+in+E-coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000324-Skovgaard1" id="pone.0000324-Skovgaard1"></a>Skovgaard M, Jensen LJ, Brunak S, Ussery D, Krogh A (2001) On the total number of genes and their length distribution in complete microbial genomes. Trends in Genetics  17: 425–428.  <ul class="find" data-citedArticleID="940304" data-doi="10.1016/s0168-9525(01)02372-1"><li><a href="http://dx.doi.org/10.1016/s0168-9525(01)02372-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+total+number+of+genes+and+their+length+distribution+in+complete+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+total+number+of+genes+and+their+length+distribution+in+complete+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000324-Kawarabayasi1" id="pone.0000324-Kawarabayasi1"></a>Kawarabayasi Y, Hino Y, Horikawa H, Yamazaki S, Haikawa Y, et al.  (1999) Complete genome sequence of an aerobic hyper-thermophilic crenarchaeon, Aeropyrum pernix K1. DNA Res  6: 83–101, 145–152.  <ul class="find" data-citedArticleID="940266" data-doi="10.1093/dnares/6.2.145"><li><a href="http://dx.doi.org/10.1093/dnares/6.2.145" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+an+aerobic+hyper-thermophilic+crenarchaeon%2C+Aeropyrum+pernix+K1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+an+aerobic+hyper-thermophilic+crenarchaeon%2C+Aeropyrum+pernix+K1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000324-Fukuchi1" id="pone.0000324-Fukuchi1"></a>Fukuchi S, Nishikawa K (2004) Estimation of the number of authentic orphan genes in bacterial genomes. DNA Res  11: 219–231, 311–313.  <ul class="find" data-citedArticleID="940250" data-doi="10.1093/dnares/11.4.311"><li><a href="http://dx.doi.org/10.1093/dnares/11.4.311" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimation+of+the+number+of+authentic+orphan+genes+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimation+of+the+number+of+authentic+orphan+genes+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000324-Ussery1" id="pone.0000324-Ussery1"></a>Ussery DW, Hallin PF (2004) Genome Update: annotation quality in sequenced microbial genomes. Microbiology  150: 2015–2017.  <ul class="find" data-citedArticleID="940312" data-doi="10.1099/mic.0.27338-0"><li><a href="http://dx.doi.org/10.1099/mic.0.27338-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+Update%3A+annotation+quality+in+sequenced+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+Update%3A+annotation+quality+in+sequenced+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000324-Siew1" id="pone.0000324-Siew1"></a>Siew N, Fischer D (2003) Twenty thousand ORFan microbial protein families for the biologist? Structure  11: 7–9.  <ul class="find" data-citedArticleID="940302" data-doi="10.1016/s0969-2126(02)00938-3"><li><a href="http://dx.doi.org/10.1016/s0969-2126(02)00938-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Twenty+thousand+ORFan+microbial+protein+families+for+the+biologist%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Twenty+thousand+ORFan+microbial+protein+families+for+the+biologist%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000324-Martiny1" id="pone.0000324-Martiny1"></a>Martiny JBH, Field D (2005) Ecological Perspectives on our complete genome collection. Ecology Letters  8: 1334–1345.  <ul class="find" data-citedArticleID="940278" data-doi="10.1111/j.1461-0248.2005.00837.x"><li><a href="http://dx.doi.org/10.1111/j.1461-0248.2005.00837.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ecological+Perspectives+on+our+complete+genome+collection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ecological+Perspectives+on+our+complete+genome+collection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000324-Alimi1" id="pone.0000324-Alimi1"></a>Alimi JP, Poirot O, Lopez F, Claverie JM (2000) Reverse transcriptase-polymerase chain reaction validation of 25 “orphan” genes from Escherichia coli K-12 MG1655. Genome Research  10: 959–966.  <ul class="find" data-citedArticleID="940224" data-doi="10.1101/gr.10.7.959"><li><a href="http://dx.doi.org/10.1101/gr.10.7.959" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+transcriptase-polymerase+chain+reaction+validation+of+25+%26%23x201c%3Borphan%26%23x201d%3B+genes+from+Escherichia+coli+K-12+MG1655." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+transcriptase-polymerase+chain+reaction+validation+of+25+%26%23x201c%3Borphan%26%23x201d%3B+genes+from+Escherichia+coli+K-12+MG1655.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000324-Kolker1" id="pone.0000324-Kolker1"></a>Kolker E, Makarova KS, Shabalina S, Picone AF, Purvine S, et al.  (2004) Identification and functional analysis of ‘hypothetical’ genes expressed in Haemophilus influenzae. Nucleic Acids Res  32: 2353–2361.  <ul class="find" data-citedArticleID="940270" data-doi="10.1093/nar/gkh555"><li><a href="http://dx.doi.org/10.1093/nar/gkh555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+functional+analysis+of+%26%23x2018%3Bhypothetical%26%23x2019%3B+genes+expressed+in+Haemophilus+influenzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+functional+analysis+of+%26%23x2018%3Bhypothetical%26%23x2019%3B+genes+expressed+in+Haemophilus+influenzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000324-Shmuely1" id="pone.0000324-Shmuely1"></a>Shmuely H, Dinitz E, Dahan I, Eichler J, Fischer D, et al.  (2004) Poorly conserved ORFs in the genome of the archaea Halobacterium sp NRC-1 correspond to expressed proteins. Bioinformatics  20: 1248–1253.  <ul class="find" data-citedArticleID="940300" data-doi="10.1093/bioinformatics/bth075"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Poorly+conserved+ORFs+in+the+genome+of+the+archaea+Halobacterium+sp+NRC-1+correspond+to+expressed+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Poorly+conserved+ORFs+in+the+genome+of+the+archaea+Halobacterium+sp+NRC-1+correspond+to+expressed+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000324-Ruby1" id="pone.0000324-Ruby1"></a>Ruby EG, Urbanowski M, Campbell J, Dunn A, Faini M, et al.  (2005) Complete genome sequence of Vibrio fischeri: a symbiotic bacterium with pathogenic congeners. Proc Natl Acad Sci U S A  102: 3004–3009.  <ul class="find" data-citedArticleID="940298" data-doi="10.1073/pnas.0409900102"><li><a href="http://dx.doi.org/10.1073/pnas.0409900102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Vibrio+fischeri%3A+a+symbiotic+bacterium+with+pathogenic+congeners." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Vibrio+fischeri%3A+a+symbiotic+bacterium+with+pathogenic+congeners.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000324-Waldor1" id="pone.0000324-Waldor1"></a>Waldor MK, Mekalanos JJ (1996) Lysogenic conversion by a filamentous phage encoding cholera toxin. Science  272: 1910–1914.  <ul class="find" data-citedArticleID="940314" data-doi="10.1126/science.272.5270.1910"><li><a href="http://dx.doi.org/10.1126/science.272.5270.1910" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lysogenic+conversion+by+a+filamentous+phage+encoding+cholera+toxin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lysogenic+conversion+by+a+filamentous+phage+encoding+cholera+toxin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000324-Manning1" id="pone.0000324-Manning1"></a>Manning PA (1997) The tcp gene cluster of Vibrio cholerae. Gene  192: 63–70.  <ul class="find" data-citedArticleID="940276" data-doi="10.1016/s0378-1119(97)00036-x"><li><a href="http://dx.doi.org/10.1016/s0378-1119(97)00036-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tcp+gene+cluster+of+Vibrio+cholerae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tcp+gene+cluster+of+Vibrio+cholerae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000324-Tettelin1" id="pone.0000324-Tettelin1"></a>Tettelin H, Masignani V, Cieslewicz MJ, Donati C, Medini D, et al.  (2005) Genome analysis of multiple pathogenic isolates of Streptococcus agalactiae: implications for the microbial “pan-genome”. Proc Natl Acad Sci U S A  102: 13950–13955.  Epub 12005 Sep 13919.  <ul class="find" data-citedArticleID="940310" data-doi="10.1073/pnas.0506758102"><li><a href="http://dx.doi.org/10.1073/pnas.0506758102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+analysis+of+multiple+pathogenic+isolates+of+Streptococcus+agalactiae%3A+implications+for+the+microbial+%26%23x201c%3Bpan-genome%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+analysis+of+multiple+pathogenic+isolates+of+Streptococcus+agalactiae%3A+implications+for+the+microbial+%26%23x201c%3Bpan-genome%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000324-Medini1" id="pone.0000324-Medini1"></a>Medini D, Donati C, Tettelin H, Masignani V, Rappuoli R (2005) The microbial pan-genome. Curr Opin Genet Dev  15: 589–594.  Epub 2005 Sep 2026.  <ul class="find" data-citedArticleID="940280" data-doi="10.1016/j.gde.2005.09.006"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.09.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microbial+pan-genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microbial+pan-genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000324-Kosuge1" id="pone.0000324-Kosuge1"></a>Kosuge T, Abe T, Okido T, Tanaka N, Hirahata M, et al.  (2006) Exploration and Grading of Possible Genes from 183 Bacterial Strains by a Common Protocol to Identification of New Genes: Gene Trek in Prokaryote Space (GTPS). DNA Res  13: 13.  <ul class="find" data-citedArticleID="940274" data-doi="10.1093/dnares/dsl014"><li><a href="http://dx.doi.org/10.1093/dnares/dsl014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration+and+Grading+of+Possible+Genes+from+183+Bacterial+Strains+by+a+Common+Protocol+to+Identification+of+New+Genes%3A+Gene+Trek+in+Prokaryote+Space+%28GTPS%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration+and+Grading+of+Possible+Genes+from+183+Bacterial+Strains+by+a+Common+Protocol+to+Identification+of+New+Genes%3A+Gene+Trek+in+Prokaryote+Space+%28GTPS%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000324-Altschul1" id="pone.0000324-Altschul1"></a>Altschul SF, Boguski MS, Gish W, Wootton JC (1994) Issues in searching molecular sequence databases. Nat Genet  6: 119–129.  <ul class="find" data-citedArticleID="940228" data-doi="10.1038/ng0294-119"><li><a href="http://dx.doi.org/10.1038/ng0294-119" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Issues+in+searching+molecular+sequence+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Issues+in+searching+molecular+sequence+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000324-Navarre1" id="pone.0000324-Navarre1"></a>Navarre WW, Porwollik S, Wang Y, McClelland M, Rosen H, et al.  (2006) Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella. Science  313: 236–238.  <ul class="find" data-citedArticleID="940284" data-doi="10.1126/science.1128794"><li><a href="http://dx.doi.org/10.1126/science.1128794" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selective+silencing+of+foreign+DNA+with+low+GC+content+by+the+H-NS+protein+in+Salmonella." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selective+silencing+of+foreign+DNA+with+low+GC+content+by+the+H-NS+protein+in+Salmonella.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000324-Akashi1" id="pone.0000324-Akashi1"></a>Akashi H, Gojobori T (2002) Metabolic efficiency and amino acid composition in the proteomes of Escherichia coli and Bacillus subtilis. Proc Natl Acad Sci U S A  99: 3695–3700.  <ul class="find" data-citedArticleID="940222" data-doi="10.1073/pnas.062526999"><li><a href="http://dx.doi.org/10.1073/pnas.062526999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolic+efficiency+and+amino+acid+composition+in+the+proteomes+of+Escherichia+coli+and+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolic+efficiency+and+amino+acid+composition+in+the+proteomes+of+Escherichia+coli+and+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000324-Heizer1" id="pone.0000324-Heizer1"></a>Heizer EM Jr, Raiford DW, Raymer ML, Doom TE, Miller RV, et al.  (2006) Amino acid cost and codon-usage biases in 6 prokaryotic genomes: a whole-genome analysis. Mol Biol Evol  23: 1670–1680.  <ul class="find" data-citedArticleID="940260" data-doi="10.1093/molbev/msl029"><li><a href="http://dx.doi.org/10.1093/molbev/msl029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+cost+and+codon-usage+biases+in+6+prokaryotic+genomes%3A+a+whole-genome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+cost+and+codon-usage+biases+in+6+prokaryotic+genomes%3A+a+whole-genome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000324-Zheng1" id="pone.0000324-Zheng1"></a>Zheng Y, Anton BP, Roberts RJ, Kasif S (2005) Phylogenetic detection of conserved gene clusters in microbial genomes. BMC Bioinformatics  6: 243.  <ul class="find" data-citedArticleID="940322" data-doi="10.1186/1471-2105-6-243"><li><a href="http://dx.doi.org/10.1186/1471-2105-6-243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+detection+of+conserved+gene+clusters+in+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+detection+of+conserved+gene+clusters+in+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000324-Yin1" id="pone.0000324-Yin1"></a>Yin Y, Fischer D (2006) On the origin of microbial ORFans: quantifying the strength of the evidence for viral lateral transfer. BMC Evol Biol  6: 63.  <ul class="find" data-citedArticleID="940320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=On+the+origin+of+microbial+ORFans%3A+quantifying+the+strength+of+the+evidence+for+viral+lateral+transfer.&amp;auth=&amp;atitle=On+the+origin+of+microbial+ORFans%3A+quantifying+the+strength+of+the+evidence+for+viral+lateral+transfer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+the+origin+of+microbial+ORFans%3A+quantifying+the+strength+of+the+evidence+for+viral+lateral+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+the+origin+of+microbial+ORFans%3A+quantifying+the+strength+of+the+evidence+for+viral+lateral+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000324-Alm1" id="pone.0000324-Alm1"></a>Alm EJ, Huang KH, Price MN, Koche RP, Keller K, et al.  (2005) The MicrobesOnline Web site for comparative genomics. Genome Res  15: 1015–1022.  <ul class="find" data-citedArticleID="940226" data-doi="10.1101/gr.3844805"><li><a href="http://dx.doi.org/10.1101/gr.3844805" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+MicrobesOnline+Web+site+for+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+MicrobesOnline+Web+site+for+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000324-GutierrezRios1" id="pone.0000324-GutierrezRios1"></a>Gutierrez-Rios RM, Rosenblueth DA, Loza JA, Huerta AM, Glasner JD, et al.  (2003) Regulatory network of Escherichia coli: consistency between literature knowledge and microarray profiles. Genome Res  13: 2435–2443.  <ul class="find" data-citedArticleID="940258" data-doi="10.1101/gr.1387003"><li><a href="http://dx.doi.org/10.1101/gr.1387003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+network+of+Escherichia+coli%3A+consistency+between+literature+knowledge+and+microarray+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+network+of+Escherichia+coli%3A+consistency+between+literature+knowledge+and+microarray+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000324-Kang1" id="pone.0000324-Kang1"></a>Kang Y, Weber KD, Qiu Y, Kiley PJ, Blattner FR (2005) Genome-wide expression analysis indicates that FNR of Escherichia coli K-12 regulates a large number of genes of unknown function. J Bacteriol  187: 1135–1160.  <ul class="find" data-citedArticleID="940264" data-doi="10.1128/jb.187.3.1135-1160.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.3.1135-1160.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+expression+analysis+indicates+that+FNR+of+Escherichia+coli+K-12+regulates+a+large+number+of+genes+of+unknown+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+expression+analysis+indicates+that+FNR+of+Escherichia+coli+K-12+regulates+a+large+number+of+genes+of+unknown+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000324-Courcelle1" id="pone.0000324-Courcelle1"></a>Courcelle J, Khodursky A, Peter B, Brown PO, Hanawalt PC (2001) Comparative gene expression profiles following UV exposure in wild-type and SOS-deficient Escherichia coli. Genetics  158: 41–64.  <ul class="find" data-citedArticleID="940240"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli.&amp;auth=&amp;atitle=Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+gene+expression+profiles+following+UV+exposure+in+wild-type+and+SOS-deficient+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000324-Khodursky1" id="pone.0000324-Khodursky1"></a>Khodursky AB, Peter BJ, Cozzarelli NR, Botstein D, Brown PO, et al.  (2000) DNA microarray analysis of gene expression in response to physiological and genetic changes that affect tryptophan metabolism in Escherichia coli. Proc Natl Acad Sci U S A  97: 12170–12175.  <ul class="find" data-citedArticleID="940268" data-doi="10.1073/pnas.220414297"><li><a href="http://dx.doi.org/10.1073/pnas.220414297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+microarray+analysis+of+gene+expression+in+response+to+physiological+and+genetic+changes+that+affect+tryptophan+metabolism+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+microarray+analysis+of+gene+expression+in+response+to+physiological+and+genetic+changes+that+affect+tryptophan+metabolism+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000324-Corbin1" id="pone.0000324-Corbin1"></a>Corbin RW, Paliy O, Yang F, Shabanowitz J, Platt M, et al.  (2003) Toward a protein profile of Escherichia coli: comparison to its transcription profile. Proc Natl Acad Sci U S A  100: 9232–9237.  <ul class="find" data-citedArticleID="940238" data-doi="10.1073/pnas.1533294100"><li><a href="http://dx.doi.org/10.1073/pnas.1533294100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Toward+a+protein+profile+of+Escherichia+coli%3A+comparison+to+its+transcription+profile." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Toward+a+protein+profile+of+Escherichia+coli%3A+comparison+to+its+transcription+profile.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000324-Gevaert1" id="pone.0000324-Gevaert1"></a>Gevaert K, Van Damme J, Goethals M, Thomas GR, Hoorelbeke B, et al.  (2002) Chromatographic isolation of methionine-containing peptides for gel-free proteome analysis: identification of more than 800 Escherichia coli proteins. Mol Cell Proteomics  1: 896–903.  <ul class="find" data-citedArticleID="940256" data-doi="10.1074/mcp.m200061-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m200061-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatographic+isolation+of+methionine-containing+peptides+for+gel-free+proteome+analysis%3A+identification+of+more+than+800+Escherichia+coli+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatographic+isolation+of+methionine-containing+peptides+for+gel-free+proteome+analysis%3A+identification+of+more+than+800+Escherichia+coli+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000324-Taoka1" id="pone.0000324-Taoka1"></a>Taoka M, Yamauchi Y, Shinkawa T, Kaji H, Motohashi W, et al.  (2004) Only a small subset of the horizontally transferred chromosomal genes in Escherichia coli are translated into proteins. Mol Cell Proteomics  3: 780–787.  <ul class="find" data-citedArticleID="940306" data-doi="10.1074/mcp.m400030-mcp200"><li><a href="http://dx.doi.org/10.1074/mcp.m400030-mcp200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Only+a+small+subset+of+the+horizontally+transferred+chromosomal+genes+in+Escherichia+coli+are+translated+into+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Only+a+small+subset+of+the+horizontally+transferred+chromosomal+genes+in+Escherichia+coli+are+translated+into+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000324-Roberts1" id="pone.0000324-Roberts1"></a>Roberts RJ (2004) Identifying protein function–a call for community action. PLoS Biol  2: E42.  Epub 2004 Mar 2016.  <ul class="find" data-citedArticleID="940292" data-doi="10.1371/journal.pbio.0020042"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+protein+function%26%23x2013%3Ba+call+for+community+action." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+protein+function%26%23x2013%3Ba+call+for+community+action.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000324-Roberts2" id="pone.0000324-Roberts2"></a>Roberts RJ, Karp P, Kasif S, Linn S, Buckley MR (2005) An Experimental Approach to Genome Annotation. Critical Issues Colloquia Report, American Society of Microbiology.   <ul class="find" data-citedArticleID="940294"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+Experimental+Approach+to+Genome+Annotation.&amp;auth=&amp;atitle=An+Experimental+Approach+to+Genome+Annotation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+Experimental+Approach+to+Genome+Annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+Experimental+Approach+to+Genome+Annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000324-Galperin1" id="pone.0000324-Galperin1"></a>Galperin MY, Koonin EV (2004) ‘Conserved hypothetical’ proteins: prioritization of targets for experimental study. Nucleic Acids Res  32: 5452–5463.  Print 2004.  <ul class="find" data-citedArticleID="940252" data-doi="10.1093/nar/gkh885"><li><a href="http://dx.doi.org/10.1093/nar/gkh885" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%26%23x2018%3BConserved+hypothetical%26%23x2019%3B+proteins%3A+prioritization+of+targets+for+experimental+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%26%23x2018%3BConserved+hypothetical%26%23x2019%3B+proteins%3A+prioritization+of+targets+for+experimental+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000324-Nei1" id="pone.0000324-Nei1"></a>Nei M (2005) Selectionism and neutralism in molecular evolution. Mol Biol Evol  22: 2318–2342.  <ul class="find" data-citedArticleID="940286" data-doi="10.1093/molbev/msi242"><li><a href="http://dx.doi.org/10.1093/molbev/msi242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Selectionism+and+neutralism+in+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Selectionism+and+neutralism+in+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000324-Hurst1" id="pone.0000324-Hurst1"></a>Hurst LD, Feil EJ, Rocha EP (2006) Protein evolution: causes of trends in amino-acid gain and loss. Nature  442: E11–12.  discussion E12.  <ul class="find" data-citedArticleID="940262" data-doi="10.1038/nature05137"><li><a href="http://dx.doi.org/10.1038/nature05137" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+evolution%3A+causes+of+trends+in+amino-acid+gain+and+loss." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+evolution%3A+causes+of+trends+in+amino-acid+gain+and+loss.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000324-Petrov1" id="pone.0000324-Petrov1"></a>Petrov DA, Hartl DL (1999) Patterns of nucleotide substitution in Drosophila and mammalian genomes. Proc Natl Acad Sci U S A  96: 1475–1479.  <ul class="find" data-citedArticleID="940290" data-doi="10.1073/pnas.96.4.1475"><li><a href="http://dx.doi.org/10.1073/pnas.96.4.1475" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterns+of+nucleotide+substitution+in+Drosophila+and+mammalian+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterns+of+nucleotide+substitution+in+Drosophila+and+mammalian+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000324-Ochman1" id="pone.0000324-Ochman1"></a>Ochman H (2003) Neutral mutations and neutral substitutions in bacterial genomes. Mol Biol Evol  20: 2091–2096.  Epub 2003 Aug 2029.  <ul class="find" data-citedArticleID="940288" data-doi="10.1093/molbev/msg229"><li><a href="http://dx.doi.org/10.1093/molbev/msg229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutral+mutations+and+neutral+substitutions+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutral+mutations+and+neutral+substitutions+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000324-Foerstner1" id="pone.0000324-Foerstner1"></a>Foerstner KU, von Mering C, Hooper SD, Bork P (2005) Environments shape the nucleotide composition of genomes. EMBO Rep  6: 1208–1213.  <ul class="find" data-citedArticleID="940248" data-doi="10.1038/sj.embor.7400538"><li><a href="http://dx.doi.org/10.1038/sj.embor.7400538" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Environments+shape+the+nucleotide+composition+of+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Environments+shape+the+nucleotide+composition+of+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000324-Wernegreen1" id="pone.0000324-Wernegreen1"></a>Wernegreen JJ (2002) Genome evolution in bacterial endosymbionts of insects. Nature Review Genetics  3: 850–861.  <ul class="find" data-citedArticleID="940316" data-doi="10.1038/nrg931"><li><a href="http://dx.doi.org/10.1038/nrg931" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+evolution+in+bacterial+endosymbionts+of+insects." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+evolution+in+bacterial+endosymbionts+of+insects.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000324-Galperin2" id="pone.0000324-Galperin2"></a>Galperin MY, Kolker E (2006) New metrics for comparative genomics. Curr Opin Biotechnol  17: 440–447.  Epub 2006 Sep 2015.  <ul class="find" data-citedArticleID="940254" data-doi="10.1016/j.copbio.2006.08.007"><li><a href="http://dx.doi.org/10.1016/j.copbio.2006.08.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+metrics+for+comparative+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+metrics+for+comparative+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000324-Amiri1" id="pone.0000324-Amiri1"></a>Amiri H, Davids W, Andersson SG (2003) Birth and death of orphan genes in Rickettsia. Mol Biol Evol  20: 1575–1587.  Epub 2003 Jun 1527.  <ul class="find" data-citedArticleID="940232" data-doi="10.1093/molbev/msg175"><li><a href="http://dx.doi.org/10.1093/molbev/msg175" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Birth+and+death+of+orphan+genes+in+Rickettsia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Birth+and+death+of+orphan+genes+in+Rickettsia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000324-vonMering1" id="pone.0000324-vonMering1"></a>von Mering C, L JJ, Kuhn M, Chaffron S, Doerks T, et al.  (2006) STRING 7–recent developments in the integration and prediction of protein interactions. Nucleic Acids Res  10: 10.  <ul class="find" data-citedArticleID="940324" data-doi="10.1093/nar/gkl825"><li><a href="http://dx.doi.org/10.1093/nar/gkl825" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=STRING+7%26%23x2013%3Brecent+developments+in+the+integration+and+prediction+of+protein+interactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22STRING+7%26%23x2013%3Brecent+developments+in+the+integration+and+prediction+of+protein+interactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000324-Romine1" id="pone.0000324-Romine1"></a>Romine MF, Elias DA, Monroe ME, Auberry K, Fang R, et al.  (2004) Validation of Shewanella oneidensis MR-1 small proteins by AMT tag-based proteome analysis. Omics  8: 239–254.  <ul class="find" data-citedArticleID="940296" data-doi="10.1089/omi.2004.8.239"><li><a href="http://dx.doi.org/10.1089/omi.2004.8.239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Validation+of+Shewanella+oneidensis+MR-1+small+proteins+by+AMT+tag-based+proteome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Validation+of+Shewanella+oneidensis+MR-1+small+proteins+by+AMT+tag-based+proteome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000324-Kolker2" id="pone.0000324-Kolker2"></a>Kolker E, Picone AF, Galperin MY, Romine MF, Higdon R, et al.  (2005) Global profiling of Shewanella oneidensis MR-1: expression of hypothetical genes and improved functional annotations. Proc Natl Acad Sci U S A  102: 2099–2104.  Epub 2005 Jan 2031.  <ul class="find" data-citedArticleID="940272" data-doi="10.1073/pnas.0409111102"><li><a href="http://dx.doi.org/10.1073/pnas.0409111102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+profiling+of+Shewanella+oneidensis+MR-1%3A+expression+of+hypothetical+genes+and+improved+functional+annotations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+profiling+of+Shewanella+oneidensis+MR-1%3A+expression+of+hypothetical+genes+and+improved+functional+annotations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000324-Field1" id="pone.0000324-Field1"></a>Field D, Garrity GM, Gray T, Morrison N, Selengut JD, et al.  (2007) Towards a richer description of our complete collection of genomes and metagenomes: the “Minimal Information about a Genome Sequence”. Nature Biotechnology (in review).   <ul class="find" data-citedArticleID="940246"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Towards+a+richer+description+of+our+complete+collection+of+genomes+and+metagenomes%3A+the+%26%23x201c%3BMinimal+Information+about+a+Genome+Sequence%26%23x201d%3B.&amp;auth=&amp;atitle=Towards+a+richer+description+of+our+complete+collection+of+genomes+and+metagenomes%3A+the+%26%23x201c%3BMinimal+Information+about+a+Genome+Sequence%26%23x201d%3B." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+richer+description+of+our+complete+collection+of+genomes+and+metagenomes%3A+the+%26%23x201c%3BMinimal+Information+about+a+Genome+Sequence%26%23x201d%3B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+richer+description+of+our+complete+collection+of+genomes+and+metagenomes%3A+the+%26%23x201c%3BMinimal+Information+about+a+Genome+Sequence%26%23x201d%3B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000324-Altschul2" id="pone.0000324-Altschul2"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="940230" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000324-Delcher1" id="pone.0000324-Delcher1"></a>Delcher AL, Harmon D, Kasif S, White O, Salzberg SL (1999) Improved microbial gene identification with GLIMMER. Nucleic Acids Res  27: 4636–4641.  <ul class="find" data-citedArticleID="940244" data-doi="10.1093/nar/27.23.4636"><li><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+microbial+gene+identification+with+GLIMMER." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+microbial+gene+identification+with+GLIMMER.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000324-Misra1" id="pone.0000324-Misra1"></a>Misra RV, Horler RS, Reindl W, Goryanin , Thomas GH (2005) EchoBASE: an integrated post-genomic database for Escherichia coli. Nucleic Acids Res  33: D329–333.  <ul class="find" data-citedArticleID="940282" data-doi="10.1093/nar/gki028"><li><a href="http://dx.doi.org/10.1093/nar/gki028" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=EchoBASE%3A+an+integrated+post-genomic+database+for+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22EchoBASE%3A+an+integrated+post-genomic+database+for+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.XML" value="108793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.PDF" value="382920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g001.PNG_L" value="89122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g001.PNG_M" value="24075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g001.PNG_S" value="2278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g001.TIF" value="345958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g001.PNG_I" value="14190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t001.PNG_L" value="80425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t001.PNG_M" value="31034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t001.PNG_S" value="2262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t001.TIF" value="384964"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t001.PNG_I" value="13880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g002.PNG_L" value="101780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g002.PNG_M" value="27586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g002.PNG_S" value="2168"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g002.TIF" value="365938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g002.PNG_I" value="14505"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g003.PNG_L" value="43511"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g003.PNG_M" value="16238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g003.PNG_S" value="1870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g003.TIF" value="92930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g003.PNG_I" value="8157"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g004.PNG_L" value="203339"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g004.PNG_M" value="35309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g004.PNG_S" value="2314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g004.TIF" value="359112"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.g004.PNG_I" value="20615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t002.PNG_L" value="111993"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t002.PNG_M" value="39521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t002.PNG_S" value="2489"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t002.TIF" value="460134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t002.PNG_I" value="17281"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t003.PNG_L" value="130613"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t003.PNG_M" value="49856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t003.PNG_S" value="2653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t003.TIF" value="526560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t003.PNG_I" value="21497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t004.PNG_L" value="61721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t004.PNG_M" value="22062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t004.PNG_S" value="2134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t004.TIF" value="342324"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.t004.PNG_I" value="10384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.s001.DOC" value="222720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000324.s002.TIF" value="3290000"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000324&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000324" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000324&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000324&volume=&issue=&title=Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes&author_name=Gareth%20A.%20Wilson%2C%20Edward%20J.%20Feil%2C%20Andrew%20K.%20Lilley%2C%20Dawn%20Field&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000324" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324&amp;t=Large-Scale%20Comparative%20Genomic%20Ranking%20of%20Taxonomically%20Restricted%20Genes%20(TRGs)%20in%20Bacterial%20and%20Archaeal%20Genomes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324&title=Large-Scale%20Comparative%20Genomic%20Ranking%20of%20Taxonomically%20Restricted%20Genes%20(TRGs)%20in%20Bacterial%20and%20Archaeal%20Genomes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324&amp;title=Large-Scale%20Comparative%20Genomic%20Ranking%20of%20Taxonomically%20Restricted%20Genes%20(TRGs)%20in%20Bacterial%20and%20Archaeal%20Genomes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000324&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Large-Scale Comparative Genomic Ranking of Taxonomically Restricted Genes (TRGs) in Bacterial and Archaeal Genomes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000324';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Large-Scale%20Comparative%20Genomic%20Ranking%20of%20Taxonomically%20Restricted%20Genes%20(TRGs)%20in%20Bacterial%20and%20Archaeal%20Genomes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000324" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000324" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="17227" data-articleid="23486"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="23486"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+annotation%22" title="Search for articles in the subject area:'Genome annotation'"><div class="flagText">Genome annotation</div></a>
              <div data-categoryid="40359" data-articleid="23486"
                   data-categoryname="Genome annotation"
                   class="flagImage" title="Flag 'Genome annotation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+complexity%22" title="Search for articles in the subject area:'Genome complexity'"><div class="flagText">Genome complexity</div></a>
              <div data-categoryid="18711" data-articleid="23486"
                   data-categoryname="Genome complexity"
                   class="flagImage" title="Flag 'Genome complexity' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23486"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23486"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomes%22" title="Search for articles in the subject area:'Proteomes'"><div class="flagText">Proteomes</div></a>
              <div data-categoryid="18649" data-articleid="23486"
                   data-categoryname="Proteomes"
                   class="flagImage" title="Flag 'Proteomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Proteomic+databases%22" title="Search for articles in the subject area:'Proteomic databases'"><div class="flagText">Proteomic databases</div></a>
              <div data-categoryid="43483" data-articleid="23486"
                   data-categoryname="Proteomic databases"
                   class="flagImage" title="Flag 'Proteomic databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1480'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4488'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6403&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=8761">TRG and TRGs</a><br>Posted by pvittata</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
