

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000928"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000928" />

    <meta name="citation_title" content="Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032"/>
    <meta itemprop="name" content="Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032"/>

      <meta name="citation_author" content="Jason Gioia"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Shailaja Yerrapragada"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Xiang Qin"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Huaiyang Jiang"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Okezie C. Igboeli"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Donna Muzny"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Shannon Dugan-Rocha"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Yan Ding"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Alicia Hawes"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Wen Liu"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Lesette Perez"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Christie Kovar"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Huyen Dinh"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Sandra Lee"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Lynne Nazareth"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Peter Blyth"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Michael Holder"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Christian Buhay"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Madhan R. Tirumalai"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Yamei Liu"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Indrani Dasgupta"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Lina Bokhetache"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Masaya Fujita"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Fathi Karouia"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Prahathees Eswara Moorthy"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Johnathan Siefert"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Akif Uzman"/>
            <meta name="citation_author_institution" content="Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Prince Buzumbo"/>
            <meta name="citation_author_institution" content="Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Avani Verma"/>
            <meta name="citation_author_institution" content="Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Hiba Zwiya"/>
            <meta name="citation_author_institution" content="Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Brian D. McWilliams"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Adeola Olowu"/>
            <meta name="citation_author_institution" content="University of St. Thomas, Houston Texas, United States of America"/>
      <meta name="citation_author" content="Kenneth D. Clinkenbeard"/>
            <meta name="citation_author_institution" content="Department of Veterinary Pathobiology, Center for Veterinary Health Sciences, Oklahoma State University, Stillwater, Oklahoma, United States of America"/>
      <meta name="citation_author" content="David Newcombe"/>
            <meta name="citation_author_institution" content="University of Idaho Coeur d'Alene, Coeur d'Alene, Idaho, United States of America"/>
            <meta name="citation_author_institution" content="Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, United States of America"/>
      <meta name="citation_author" content="Lisa Golebiewski"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Joseph F. Petrosino"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Wayne L. Nicholson"/>
            <meta name="citation_author_institution" content="Department of Microbiology and Cell Science, University of Florida Space Life Sciences Laboratory, Kennedy Space Center, Florida, United States of America"/>
      <meta name="citation_author" content="George E. Fox"/>
            <meta name="citation_author_institution" content="Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="Kasthuri Venkateswaran"/>
            <meta name="citation_author_institution" content="Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, United States of America"/>
      <meta name="citation_author" content="Sarah K. Highlander"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>
      <meta name="citation_author" content="George M. Weinstock"/>
            <meta name="citation_author_institution" content="Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America"/>
            <meta name="citation_author_institution" content="Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America"/>

    <meta name="citation_date" content="2007/9/26"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000928.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e928"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Systematics and Ecology of Bacillus.; citation_author=FG Priest; citation_number=1; citation_pages=3-16; citation_date=1993; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Spore UV and acceleration resistance of endolithic Bacillus pumilus and Bacillus subtilis isolates obtained from Sonoran desert basalt: implications for lithopanspermia.; citation_author=JN Benardini; citation_author=J Sawyer; citation_author=K Venkateswaran; citation_author=WL Nicholson; citation_journal_title=Astrobiology; citation_volume=3; citation_number=2; citation_pages=709-717; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recurrent isolation of hydrogen peroxide-resistant spores of Bacillus pumilus from a spacecraft assembly facility.; citation_author=MJ Kempf; citation_author=F Chen; citation_author=R Kern; citation_author=K Venkateswaran; citation_journal_title=Astrobiology; citation_volume=5; citation_number=3; citation_pages=391-405; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Survival of spacecraft-associated microorganisms under simulated martian UV irradiation.; citation_author=DA Newcombe; citation_author=AC Schuerger; citation_author=JN Benardini; citation_author=D Dickinson; citation_author=R Tanner; citation_journal_title=Applied Environmental Microbiology; citation_volume=71; citation_number=4; citation_pages=8147-8156; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Spores of Bacillus subtilis: their resistance to and killing by radiation, heat and chemicals.; citation_author=P Setlow; citation_journal_title=Journal of Applied Microbiology; citation_volume=101; citation_number=5; citation_pages=514-525; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Resistance of spores of Bacillus species to ultraviolet light.; citation_author=P Setlow; citation_journal_title=Environ Mol Mutagen; citation_volume=38; citation_number=6; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Analysis of factors that influence the sensitivity of spores of Bacillus subtilis to DNA damaging chemicals.; citation_author=DE Cortezzo; citation_author=P Setlow; citation_journal_title=Journal of Applied Microbiology; citation_volume=98; citation_number=7; citation_pages=606-617; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Oxidative stress responses in Escherichia coli and Salmonella typhimurium.; citation_author=SB Farr; citation_author=T Kogoma; citation_journal_title=Microbiological Reviews; citation_volume=55; citation_number=8; citation_pages=561-585; citation_date=1991; " />
      <meta name="citation_reference" content="citation_author=J Sambrook; citation_author=DW Russell; citation_number=9; citation_date=2001; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=10; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A “double adaptor” method for improved shotgun library construction.; citation_author=B Andersson; citation_author=MA Wentland; citation_author=JY Ricafrente; citation_author=W Liu; citation_author=RA Gibbs; citation_journal_title=Analytical Chemistry; citation_volume=236; citation_number=11; citation_pages=107-113; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The Atlas genome assembly system.; citation_author=P Havlak; citation_author=R Chen; citation_author=KJ Durbin; citation_author=A Egan; citation_author=Y Ren; citation_journal_title=Genome Research; citation_volume=14; citation_number=12; citation_pages=721-732; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complete genome sequence of Rickettsia typhi and comparison with sequences of other rickettsiae.; citation_author=MP McLeod; citation_author=X Qin; citation_author=SE Karpathy; citation_author=J Gioia; citation_author=SK Highlander; citation_journal_title=Journal of Bacteriology; citation_volume=186; citation_number=13; citation_pages=5842-5855; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Improved microbial identification with GLIMMER.; citation_author=AL Delcher; citation_author=D Harmon; citation_author=S Kasif; citation_author=O White; citation_author=SL Salzberg; citation_journal_title=Nucleic Acids Research; citation_volume=27; citation_number=14; citation_pages=4636-4641; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding.; citation_author=A Lukashin; citation_author=M Borodovsky; citation_journal_title=Nucleic Acids Research; citation_volume=26; citation_number=15; citation_pages=1107-1115; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Research; citation_volume=33; citation_number=16; citation_pages=D39-45; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=InterProScan - an integration platform for the signature-recognition methods in InterPro.; citation_author=EM Zdobnov; citation_author=R Apweiler; citation_journal_title=Bioinformatics; citation_volume=17; citation_number=17; citation_pages=847-848; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Enhanced automated function prediction using distantly related sequences and contextual association by PFP.; citation_author=T Hawkins; citation_author=S Luban; citation_author=D Kihara; citation_journal_title=Protein Science; citation_volume=15; citation_number=18; citation_pages=1550-1556; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PSORTb v.2.0: Expanded prediction of bacterial protein subcellular localization and insights gained from comparative proteome analysis.; citation_author=JL Gardy; citation_author=MR Laird; citation_author=F Chen; citation_author=S Rey; citation_author=CJ Walsh; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=19; citation_pages=617-623; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=ExPASy: The proteomics server for in-depth protein knowledge and analysis.; citation_author=E Gasteiger; citation_author=A Gattiker; citation_author=C Hoogland; citation_author=I Ivanyi; citation_author=RD Appel; citation_journal_title=Nucleic Acids Research; citation_volume=31; citation_number=20; citation_pages=3784-3788; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The prediction of helix-turn-helix DNA-binding regions in proteins.; citation_author=IB Dodd; citation_author=JB Egan; citation_journal_title=Protein Engineering; citation_volume=2; citation_number=21; citation_pages=174-175; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=MEROPS: the peptidase database.; citation_author=ND Rawlings; citation_author=FR Morton; citation_author=AJ Barrett; citation_journal_title=Nucleic Acids Research; citation_volume=34; citation_number=22; citation_pages=D270-D272; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=TCDB: the Transporter Classification Database for membrane transport protein analyses and information.; citation_author=MH Saier; citation_author=CV Tran; citation_author=RD Barabote; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=23; citation_pages=D181-186; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Analysis of temporal gene expression during Bacillus subtilis spore germination and outgrowth.; citation_author=BJ Keijser; citation_author=A Ter Beek; citation_author=H Rauwerda; citation_author=F Schuren; citation_author=R Montijn; citation_journal_title=Journal of Bacteriology; citation_volume=189; citation_number=24; citation_pages=3624-3634; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=DNA bipyrimidine photoproduct repair and transcriptional response of UV-C irradiated Bacillus subtilis.; citation_author=R Moeller; citation_author=E Stackebrandt; citation_author=T Douki; citation_author=J Cadet; citation_author=P Rettberg; citation_journal_title=Archives of Microbiology; citation_number=25; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Resistance of Bacillus endospores to extreme terrestrial and extraterrestrial environments.; citation_author=WL Nicholson; citation_author=N Munakata; citation_author=G Horneck; citation_author=HJ Melosh; citation_author=P Setlow; citation_journal_title=Microbiology and Molecular Biology Reviews; citation_volume=64; citation_number=26; citation_pages=548-572; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mechanisms for the prevention of damage to DNA in spores of Bacillus species.; citation_author=P Setlow; citation_journal_title=Annu Rev Microbiol; citation_volume=49; citation_number=27; citation_pages=29-54; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=New small, acid-soluble proteins unique to spores of Bacillus subtilis: identification of the coding genes and regulation and function of two of these genes.; citation_author=I Bagyan; citation_author=B Setlow; citation_author=P Setlow; citation_journal_title=Journal of Bacteriology; citation_volume=180; citation_number=28; citation_pages=6704-6712; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Effects of mutant small, acid-soluble spore proteins from Bacillus subtilis on DNA in vivo and in vitro.; citation_author=F Tovar-Rojo; citation_author=P Setlow; citation_journal_title=Journal of Bacteriology; citation_volume=173; citation_number=29; citation_pages=4827-4835; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Regulation of four genes encoding small, acid-soluble spore proteins in Bacillus subtilis.; citation_author=A Cabrera-Hernandez; citation_author=JL Sanchez-Salas; citation_author=M Paidhungat; citation_author=P Setlow; citation_journal_title=Gene; citation_volume=232; citation_number=30; citation_pages=1-10; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Properties of Bacillus subtilis small, acid-soluble spore proteins with changes in the sequence recognized by their specific protease.; citation_author=Y Carrillo-Martinez; citation_author=P Setlow; citation_journal_title=Journal of Bacteriology; citation_volume=176; citation_number=31; citation_pages=5357-5363; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A phylogenomic study of DNA repair genes, proteins, and processes.; citation_author=JA Eisen; citation_author=PC Hanawalt; citation_journal_title=Mutation Research; citation_volume=435; citation_number=32; citation_pages=171-213; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genome analysis of DNA repair genes in the alpha proteobacterium Caulobacter crescentus.; citation_author=M Martins-Pinheiro; citation_author=RC Marques; citation_author=CF Menck; citation_journal_title=BMC Microbiology; citation_volume=7; citation_number=33; citation_pages=17; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Role of the Nfo (YqfS) and ExoA apurinic/apyrimidinic endonucleases in protecting Bacillus subtilis spores from DNA damage.; citation_author=JM Salas-Pacheco; citation_author=B Setlow; citation_author=P Setlow; citation_author=M Pedraza-Reyes; citation_journal_title=Journal of Bacteriology; citation_volume=187; citation_number=34; citation_pages=7374-7381; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Bacillus subtilis counterpart of the mammalian 3-methyladenine DNA glycosylase has hypoxanthine and 1,N6-ethenoadenine as preferred substrates.; citation_author=RM Aamodt; citation_author=PO Falnes; citation_author=RF Johansen; citation_author=E Seeberg; citation_author=M Bjoras; citation_journal_title=Journal of Biological Chemistry; citation_volume=279; citation_number=35; citation_pages=13601-13606; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Prokaryotic nucleotide excision repair: the UvrABC system.; citation_author=JJ Truglio; citation_author=DL Croteau; citation_author=B Van Houten; citation_author=C Kisker; citation_journal_title=Chemical Reviews; citation_volume=106; citation_number=36; citation_pages=233-252; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=PcrA is an essential DNA helicase of Bacillus subtilis fulfilling functions both in repair and rolling-circle replication.; citation_author=MA Petit; citation_author=E Dervyn; citation_author=M Rose; citation_author=KD Entian; citation_author=S McGovern; citation_journal_title=Molecular Microbiology; citation_volume=29; citation_number=37; citation_pages=261-273; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Contribution of the mismatch DNA repair system to the generation of stationary-phase-induced mutants of Bacillus subtilis.; citation_author=M Pedraza-Reyes; citation_author=RE Yasbin; citation_journal_title=Journal of Bacteriology; citation_volume=186; citation_number=38; citation_pages=6485-6491; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Visualization of mismatch repair in bacterial cells.; citation_author=BT Smith; citation_author=AD Grossman; citation_author=GC Walker; citation_journal_title=Molecular Cell; citation_volume=8; citation_number=39; citation_pages=1197-1206; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cell cycle and genetic requirements of two pathways of nonhomologous end-joining repair of double-strand breaks in Saccharomyces cerevisiae.; citation_author=JK Moore; citation_author=JE Haber; citation_journal_title=Molecular and Cellular Biology; citation_volume=16; citation_number=40; citation_pages=2164-2173; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Identification of a DNA nonhomologous end-joining complex in bacteria.; citation_author=GR Weller; citation_author=B Kysela; citation_author=R Roy; citation_author=LM Tonkin; citation_author=E Scanlan; citation_journal_title=Science; citation_volume=297; citation_number=41; citation_pages=1686-1689; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The forespore line of gene expression in Bacillus subtilis.; citation_author=ST Wang; citation_author=B Setlow; citation_author=EM Conlon; citation_author=JL Lyon; citation_author=D Imamura; citation_journal_title=Journal of Molecular Biology; citation_volume=358; citation_number=42; citation_pages=16-37; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Role of DNA repair by non-homologous end joining (NHEJ) in Bacillus subtilis spore resistance to extreme dryness, mono- and polychromatic UV and ionizing radiation.; citation_author=R Moeller; citation_author=E Stackebrandt; citation_author=G Reitz; citation_author=T Berger; citation_author=P Rettberg; citation_journal_title=Journal of Bacteriology; citation_number=43; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The Bacillus subtilis addAB genes are fully functional in Escherichia coli.; citation_author=J Kooistra; citation_author=BJ Haijema; citation_author=G Venema; citation_journal_title=Molecular Microbiology; citation_volume=7; citation_number=44; citation_pages=915-923; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Inducible DNA repair and differentiation in Bacillus subtilis: interactions between global regulons.; citation_author=RE Yasbin; citation_author=DL Cheo; citation_author=KW Bayles; citation_journal_title=Molecular Microbiology; citation_volume=6; citation_number=45; citation_pages=1263-1270; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Genetic composition of the Bacillus subtilis SOS system.; citation_author=N Au; citation_author=E Kuester-Schoeck; citation_author=V Mandava; citation_author=LE Bothwell; citation_author=SP Canny; citation_journal_title=Journal of Bacteriology; citation_volume=187; citation_number=46; citation_pages=7655-7666; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Binding of the Bacillus subtilis LexA protein to the SOS operator.; citation_author=ES Groban; citation_author=MB Johnson; citation_author=P Banky; citation_author=PG Burnett; citation_author=GL Calderon; citation_journal_title=Nucleic Acids Research; citation_volume=33; citation_number=47; citation_pages=6287-6295; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolutionary pathways of an ancient gene recX.; citation_author=J Lin; citation_author=ZZ Chen; citation_author=B Tian; citation_author=YJ Hua; citation_journal_title=Gene; citation_volume=387; citation_number=48; citation_pages=15-20; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Repair of apurinic/apyrimidinic sites by UV damage endonuclease; a repair protein for UV and oxidative damage.; citation_author=S Kanno; citation_author=S Iwai; citation_author=M Takao; citation_author=A Yasui; citation_journal_title=Nucleic Acids Research; citation_volume=27; citation_number=49; citation_pages=3096-3103; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Distinctive genetic features exhibited by the Y-family DNA polymerases in Bacillus subtilis.; citation_author=S Duigou; citation_author=SD Ehrlich; citation_author=P Noirot; citation_author=MF Noirot-Gros; citation_journal_title=Molecular Microbiology; citation_volume=54; citation_number=50; citation_pages=439-451; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Relative efficiencies of the bacterial, yeast, and human DNA methyltransferases for the repair of O6-methylguanine and O4-methylthymine. Suggestive evidence for O4-methylthymine repair by eukaryotic methyltransferases.; citation_author=M Sassanfar; citation_author=MK Dosanjh; citation_author=JM Essigmann; citation_author=L Samson; citation_journal_title=Journal of Biological Chemistry; citation_volume=266; citation_number=51; citation_pages=2767-2771; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The AidB component of the Escherichia coli adaptive response to alkylating agents is a flavin-containing, DNA-binding protein.; citation_author=MS Rohankhedkar; citation_author=SB Mulrooney; citation_author=WJ Wedemeyer; citation_author=RP Hausinger; citation_journal_title=Journal of Bacteriology; citation_volume=188; citation_number=52; citation_pages=223-230; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bacillus subtilis ada operon encodes two DNA alkyltransferases.; citation_author=F Morohoshi; citation_author=K Hayashi; citation_author=N Munakata; citation_journal_title=Nucleic Acids Research; citation_volume=18; citation_number=53; citation_pages=5473-5480; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Characterization of the DNA damage-inducible helicase DinG from Escherichia coli.; citation_author=ON Voloshin; citation_author=F Vanevski; citation_author=PP Khil; citation_author=RD Camerini-Otero; citation_journal_title=Journal of Biological Chemistry; citation_volume=278; citation_number=54; citation_pages=28284-28293; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inhibition of Escherichia coli RecA coprotease activities by DinI.; citation_author=T Yasuda; citation_author=K Morimatsu; citation_author=T Horii; citation_author=T Nagata; citation_author=H Ohmori; citation_journal_title=The EMBO Journal; citation_volume=17; citation_number=55; citation_pages=3207-3216; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The Nudix hydrolase superfamily.; citation_author=AG McLennan; citation_journal_title=Cellular and Molecular Life Sciences; citation_volume=63; citation_number=56; citation_pages=123-143; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Light-driven DNA repair by photolyases.; citation_author=LO Essen; citation_author=T Klar; citation_journal_title=Cellular and Molecular Life Sciences; citation_volume=63; citation_number=57; citation_pages=1266-1277; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Helicase structure and mechanism.; citation_author=JM Caruthers; citation_author=DB McKay; citation_journal_title=Current Opinion in Structural Biology; citation_volume=12; citation_number=58; citation_pages=123-133; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A point mutation in Escherichia coli DNA helicase II renders the enzyme nonfunctional in two DNA repair pathways. Evidence for initiation of unwinding from a nick in vivo.; citation_author=RM Brosh; citation_author=SW Matson; citation_journal_title=Journal of Biological Chemistry; citation_volume=272; citation_number=59; citation_pages=572-579; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Transcriptome and proteome analysis of Bacillus subtilis gene expression in response to superoxide and peroxide stress.; citation_author=J Mostertz; citation_author=C Scharf; citation_author=M Hecker; citation_author=G Homuth; citation_journal_title=Microbiology; citation_volume=150; citation_number=60; citation_pages=497-512; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The katX gene, which codes for the catalase in spores of Bacillus subtilis, is a forespore-specific gene controlled by sigmaF, and KatX is essential for hydrogen peroxide resistance of the germinating spore.; citation_author=I Bagyan; citation_author=L Casillas-Martinez; citation_author=P Setlow; citation_journal_title=Journal of Bacteriology; citation_volume=180; citation_number=61; citation_pages=2057-2062; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Peroxiredoxins: a historical overview and speculative preview of novel mechanisms and emerging concepts in cell signaling.; citation_author=SG Rhee; citation_author=HZ Chae; citation_author=K Kim; citation_journal_title=Free Radical Biology & Medicine; citation_volume=38; citation_number=62; citation_pages=1543-1552; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Thioredoxins in bacteria: functions in oxidative stress response and regulation of thioredoxin genes.; citation_author=T Zeller; citation_author=G Klug; citation_journal_title=Naturwissenschaften; citation_volume=93; citation_number=63; citation_pages=259-266; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tricksy business: transcriptome analysis reveals the involvement of thioredoxin A in redox homeostasis, oxidative stress, sulfur metabolism, and cellular differentiation in Bacillus subtilis.; citation_author=WK Smits; citation_author=JY Dubois; citation_author=S Bron; citation_author=JM van Dijl; citation_author=OP Kuipers; citation_journal_title=Journal of Bacteriology; citation_volume=187; citation_number=64; citation_pages=3921-3930; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=OhrR is a repressor of ohrA, a key organic hydroperoxide resistance determinant in Bacillus subtilis.; citation_author=M Fuangthong; citation_author=S Atichartpongkul; citation_author=S Mongkolsuk; citation_author=JD Helmann; citation_journal_title=Journal of Bacteriology; citation_volume=183; citation_number=65; citation_pages=4134-4141; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Paired Bacillus anthracis Dps (mini-ferritin) have different reactivities with peroxide.; citation_author=X Liu; citation_author=K Kim; citation_author=T Leighton; citation_author=EC Theil; citation_journal_title=Journal of Biological Chemistry; citation_volume=281; citation_number=66; citation_pages=27827-27835; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Alkyl hydroperoxide reductase, catalase, MrgA, and superoxide dismutase are not involved in resistance of Bacillus subtilis spores to heat or oxidizing agents.; citation_author=L Casillas-Martinez; citation_author=P Setlow; citation_journal_title=Journal of Bacteriology; citation_volume=179; citation_number=67; citation_pages=7420-7425; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Bacillus subtilis paraquat resistance is directed by sigmaM, an extracytoplasmic function sigma factor, and is conferred by YqjL and BcrC.; citation_author=M Cao; citation_author=CM Moore; citation_author=JD Helmann; citation_journal_title=Journal of Bacteriology; citation_volume=187; citation_number=68; citation_pages=2948-2956; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Transcriptional analysis of the ylaABCD operon of Bacillus subtilis encoding a sigma factor of extracytoplasmic function family.; citation_author=T Matsumoto; citation_author=K Nakanishi; citation_author=K Asai; citation_author=Y Sadaie; citation_journal_title=Genes & Genetic Systems; citation_volume=80; citation_number=69; citation_pages=385-393; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bacillus species proteins involved in spore formation and degradation: from identification in the genome, to sequence analysis, and determination of function and structure.; citation_author=MJ Jedrzejas; citation_author=WJ Huang; citation_journal_title=Critical Reviews in Biochemistry and Molecular Biology; citation_volume=38; citation_number=70; citation_pages=173-198; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032"/>
    <meta name="twitter:description" content="BackgroundBacillus spores are notoriously resistant to unfavorable conditions such as UV radiation, &gamma;-radiation, H2O2, desiccation, chemical disinfection, or starvation. Bacillus pumilus SAFR-032 survives standard decontamination procedures of the Jet Propulsion Lab spacecraft assembly facility, and both spores and vegetative cells of this strain exhibit elevated resistance to UV radiation and H2O2 compared to other Bacillus species.Principal FindingsThe genome of B. pumilus SAFR-032 was sequenced and annotated. Lists of genes relevant to DNA repair and the oxidative stress response were generated and compared to B. subtilis and B. licheniformis. Differences in conservation of genes, gene order, and protein sequences are highlighted because they potentially explain the extreme resistance phenotype of B. pumilus. The B. pumilus genome includes genes not found in B. subtilis or B. licheniformis and conserved genes with sequence divergence, but paradoxically lacks several genes that function in UV or H2O2 resistance in other Bacillus species.SignificanceThis study identifies several candidate genes for further research into UV and H2O2 resistance. These findings will help explain the resistance of B. pumilus and are applicable to understanding sterilization survival strategies of microbes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000928.g001"/>

  <meta property="og:title" content="Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8600'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7126'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8416&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000928">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000928" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000928&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in <i>Bacillus pumilus</i> SAFR-032
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jason Gioia
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jason Gioia, Shailaja Yerrapragada</p>

              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shailaja Yerrapragada
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Jason Gioia, Shailaja Yerrapragada</p>

              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiang Qin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huaiyang Jiang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Okezie C. Igboeli, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Donna Muzny, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Shannon Dugan-Rocha, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yan Ding, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alicia Hawes, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wen Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lesette Perez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christie Kovar, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huyen Dinh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sandra Lee, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lynne Nazareth, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Blyth, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael Holder, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Buhay, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Madhan R. Tirumalai, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yamei Liu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Indrani Dasgupta, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lina Bokhetache, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Masaya Fujita, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fathi Karouia, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Prahathees Eswara Moorthy, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Johnathan Siefert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Akif Uzman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Prince Buzumbo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Avani Verma, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hiba Zwiya, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Natural Sciences, University of Houston‐Downtown, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Brian D. McWilliams, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adeola Olowu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  University of St. Thomas, Houston Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenneth D. Clinkenbeard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Veterinary Pathobiology, Center for Veterinary Health Sciences, Oklahoma State University, Stillwater, Oklahoma, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David Newcombe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  University of Idaho Coeur d'Alene, Coeur d'Alene, Idaho, United States of America, 
                  Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lisa Golebiewski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph F. Petrosino, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wayne L. Nicholson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Microbiology and Cell Science, University of Florida Space Life Sciences Laboratory, Kennedy Space Center, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            George E. Fox, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Biochemistry, University of Houston, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kasthuri Venkateswaran, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sarah K. Highlander, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            George M. Weinstock
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:gwstock@bcm.tmc.edu">gwstock@bcm.tmc.edu</a></p>

                <p>Affiliations:
                  Human Genome Sequencing Center, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Texas, United States of America, 
                  Department of Molecular Virology and Microbiology, Baylor College of Medicine, Houston, Texas, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 26, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000928</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000928-g001" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000928-t001" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000928-t002" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000928-t003" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000928-t004" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000928">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000928" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p><em>Bacillus</em> spores are notoriously resistant to unfavorable conditions such as UV radiation, γ-radiation, H<sub>2</sub>O<sub>2</sub>, desiccation, chemical disinfection, or starvation. <em>Bacillus pumilus</em> SAFR-032 survives standard decontamination procedures of the Jet Propulsion Lab spacecraft assembly facility, and both spores and vegetative cells of this strain exhibit elevated resistance to UV radiation and H<sub>2</sub>O<sub>2</sub> compared to other <em>Bacillus</em> species.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>The genome of <em>B. pumilus</em> SAFR-032 was sequenced and annotated. Lists of genes relevant to DNA repair and the oxidative stress response were generated and compared to <em>B. subtilis</em> and <em>B. licheniformis</em>. Differences in conservation of genes, gene order, and protein sequences are highlighted because they potentially explain the extreme resistance phenotype of <em>B. pumilus</em>. The <em>B. pumilus</em> genome includes genes not found in <em>B. subtilis</em> or <em>B. licheniformis</em> and conserved genes with sequence divergence, but paradoxically lacks several genes that function in UV or H<sub>2</sub>O<sub>2</sub> resistance in other <em>Bacillus</em> species.</p>


<h3>Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study identifies several candidate genes for further research into UV and H<sub>2</sub>O<sub>2</sub> resistance. These findings will help explain the resistance of <em>B. pumilus</em> and are applicable to understanding sterilization survival strategies of microbes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gioia J, Yerrapragada S, Qin X, Jiang H, Igboeli OC, et al.  (2007) Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in <em>Bacillus pumilus</em> SAFR-032. PLoS ONE 2(9):
          e928.
            doi:10.1371/journal.pone.0000928</p><p><strong>Academic Editor: </strong>Abraham Sonenshein, Tufts University, United States of America</p><p><strong>Received:</strong> May 18, 2007; <strong>Accepted:</strong> August 31, 2007; <strong>Published:</strong> September 26, 2007</p><p><strong>Copyright:</strong> © 2007 Gioia et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported in part by grants to GMW (NSF-414410), GEF (NASA-Exobiology #NNG05GN75G) and KV (contract to JPL/Caltech from NASA and funded by NRA ROSS 2005). The sponsors had no role in the conduct of this study.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p><em>Bacillus pumilus</em> is a Gram-positive, aerobic, rod-shaped, soil-dwelling bacterium <a href="#pone.0000928-Priest1">[1]</a>. Like other <em>Bacillus</em> species, <em>B. pumilus</em> produces spores that are more resistant than vegetative cells to heat, desiccation, UV radiation, γ-radiation, H<sub>2</sub>O<sub>2</sub>, and starvation. <em>B. pumilus</em> has been found in extreme environments such as the interior of Sonoran desert basalt and the Mars Odyssey spacecraft <a href="#pone.0000928-Benardini1">[2]</a>, <a href="#pone.0000928-Kempf1">[3]</a>. Spores and vegetative cells of <em>B. pumilus</em> SAFR-032, a strain originally recovered from the Jet Propulsion Lab (Pasadena, CA) spacecraft assembly facility, are endowed with UV radiation and H<sub>2</sub>O<sub>2</sub> resistance capabilities that significantly exceed other <em>Bacillus</em> species and allow survival of standard sterilization practices <a href="#pone.0000928-Kempf1">[3]</a>–<a href="#pone.0000928-Setlow1">[5]</a>. Sterilization is significant not only for prevention of contamination of extraterrestrial environments via spacecraft, but also for fundamental processes in bacteriology, medicine, the pharmaceutical industry, and counter-bioterrorism measures, and hence such resistance is cause for concern.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>UV radiation induces the formation of deleterious DNA lesions such as pyrimidine dimers <a href="#pone.0000928-Setlow1">[5]</a>, <a href="#pone.0000928-Setlow2">[6]</a>. <em>Bacillus</em> spores are more resistant to UV radiation than vegetative cells because desiccation and the presence of small acid soluble spore proteins (SASP) mitigate DNA damage. A variety of DNA repair mechanisms that become active upon germination also permit survival of UV radiation. H<sub>2</sub>O<sub>2</sub> kills spores by oxidative damage to the inner membrane and it also causes oxidative damage to cellular proteins and DNA <a href="#pone.0000928-Cortezzo1">[7]</a>, <a href="#pone.0000928-Farr1">[8]</a>. H<sub>2</sub>O<sub>2</sub>-induced damage is combated by a variety of reducing agents that react with oxidative agents or oxidized cellular components.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we present an analysis of the <em>B. pumilus</em> SAFR-032 genome. In comparing this genome to less UV- and H<sub>2</sub>O<sub>2</sub>-resistant <em>Bacillus</em> species (<em>B. subtilis</em> and <em>B. licheniformis</em>) we identify genomic differences that provide important insights into the DNA repair pathways and oxidative stress response pathways of <em>B. pumilus</em>. The genes identified in this study are candidates for further experimental research.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Bacterial strain growth and DNA isolation</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A single <em>B. pumilus</em> SAFR-032 colony exhibiting circular, crateriform morphology and raised ridges on its surface, was used to inoculate trypticase soy yeast (TSY) broth. The culture was grown overnight at 37°C with vigorous shaking. Genomic DNA was purified from CsCl gradients of whole cell lysates <a href="#pone.0000928-Sambrook1">[9]</a>.</p>


<h4>DNA sequencing and genome assembly</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>DNA sequencing was performed by a combined approach using traditional Sanger dideoxy whole genome shotgun (WGS) sequencing and 454 Life Sciences pyrosequencing strategies <a href="#pone.0000928-Margulies1">[10]</a>. Genomic DNA was nebulized into 5 kb fragments, and cloned into a derivative of pUC18 <a href="#pone.0000928-Andersson1">[11]</a>. The clones were used for WGS DNA sequencing using ABI 3700 sequencers, and reads were assembled using the ATLAS assembler <a href="#pone.0000928-Havlak1">[12]</a>. Read-pair information was used to create higher order scaffolds. WGS reads were sequenced to ten-fold coverage. The WGS plasmid libraries were not random, but had cloning bias of unknown cause. Consequently, the WGS sequence was supplemented with short reads generated on a 454 Life Sciences GS20 sequencer and lacking cloning bias. Here the coverage was thirteen fold.</p>


<h4>Gene identification and annotation</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Previously described gene prediction and manual annotation protocols were followed <a href="#pone.0000928-McLeod1">[13]</a>. Glimmer <a href="#pone.0000928-Delcher1">[14]</a> and GeneMark <a href="#pone.0000928-Lukashin1">[15]</a> were used independently to predict open reading frames (ORFs). Visualization of gene predictions was performed using the Genboree system (<a href="http://www.genboree.org">www.genboree.org</a>) and the CONAN database <a href="#pone.0000928-McLeod1">[13]</a>. DNA comparisons were performed with BLASTN and BLASTZ. Protein sequences were analyzed by BLASTP vs. the nr database at NCBI <a href="#pone.0000928-Wheeler1">[16]</a>. When appropriate, other predictive tools such as InterProScan <a href="#pone.0000928-Zdobnov1">[17]</a>, PFP <a href="#pone.0000928-Hawkins1">[18]</a>, PSORTb <a href="#pone.0000928-Gardy1">[19]</a>, ExPASy ENZYME <a href="#pone.0000928-Gasteiger1">[20]</a>, Helix-Turn-Helix Predictor <a href="#pone.0000928-Dodd1">[21]</a>, MEROPs <a href="#pone.0000928-Rawlings1">[22]</a>, and the Transport Classification Database <a href="#pone.0000928-Saier1">[23]</a> were used. The <em>B. pumilus</em> SAFR-032 genome is 3.7 Mb and 3848 features (3687 ORFs, 12 frameshifts, 38 pseudogenes, 7 rRNA operons, 69 tRNAs, and 21 ncRNAs) were annotated. The <em>B. pumilus</em> genome has been deposited in GenBank under the accession number CP000813. Locus tags of genes discussed in this paper are listed in Supplementary <a href="#pone.0000928.s001">Table S1</a>.</p>


<h4>Comparative Genomic Analysis</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The database of annotated genes was searched for genes relevant to DNA repair and H<sub>2</sub>O<sub>2</sub> resistance. <em>B. pumilus</em> genes were considered homologs of <em>B. subtilis</em> and <em>B. licheniformis</em> genes if their translated sequences aligned with ≥50% identity to the homolog of either species. Exceptions were made in deference to conserved gene order and local alignments to functional domains characteristic of specific proteins. We examined the <em>B. subtilis</em> and <em>B. licheniformis</em> genomes and available literature to find DNA repair and H<sub>2</sub>O<sub>2</sub> resistance genes not found in our <em>B. pumilus</em> gene list. Relevant genes absent from the <em>B. pumilus</em> gene list were confirmed as absent using <em>B. subtilis</em> and <em>B. licheniformis</em> sequences as queries for local BLAST against the <em>B. pumilus</em> genome.</p>


<h4>Spore survivability to UV radiation and H<sub>2</sub>O<sub>2</sub></h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Methods of measuring survival of spores exposed to UV radiation and H<sub>2</sub>O<sub>2</sub> have been previously described <a href="#pone.0000928-Kempf1">[3]</a>, <a href="#pone.0000928-Newcombe1">[4]</a>. Data presented here include but are not limited to measurements previously reported in those studies.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><em>B. pumilus</em> SAFR-032 was selected for genome sequencing and analysis because its spores exhibited unusually high resistance to UV radiation and H<sub>2</sub>O<sub>2</sub> compared to the standard dosimetric strains <em>B. subtilis</em> 168 and <em>B. licheniformis</em>. Whereas &gt;90% lethality of <em>B. subtilis</em> and <em>B. lichenifiormis</em> spores is achieved by exposure to 200 J/m<sup>2</sup> UV254, 1500 J/m<sup>2</sup> are required to kill 90% of <em>B. pumilus</em> SAFR-032 spores (<a href="#pone-0000928-g001">Figure 1a</a>). Twelve percent of <em>B. pumilus</em> SAFR-032 spores survive 5% liquid H<sub>2</sub>O<sub>2</sub>, which is nearly thrice the survival rate of <em>B. subtilis</em> spores (<a href="#pone-0000928-g001">Figure 1b</a>).</p>
<div class="figure" id="pone-0000928-g001"><div class="img"><a name="pone-0000928-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000928.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000928.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Resistance of <em>B. pumilus</em> SAFR-032 spores to UV radiation and H<sub>2</sub>O<sub>2</sub>.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>a) Survivability of spores exposed to varying doses of UV254 (100 uW sec<sup>−1</sup>cm<sup>−2</sup>). Key: <em>B. pumilus</em> SAFR-032, circles; <em>B. subtilis</em> 168, squares; <em>B. licheniformis</em> ME-13-1, triangles. b) Survivability of spores exposed to 5% H<sub>2</sub>O<sub>2</sub> liquid for one hour.</p>
<span>doi:10.1371/journal.pone.0000928.g001</span></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The <em>B. pumilus</em> SAFR-032 genome was annotated and analyzed for features relevant to UV radiation resistance and H<sub>2</sub>O<sub>2</sub> resistance. Mechanisms of DNA repair and the oxidative stress response were compared among <em>B. pumilus</em>, <em>B. subtilis</em>, and <em>B. licheniformis</em> to generate lists of genes common to all three species, genes unique to <em>B. pumilus</em>, and genes absent in <em>B. pumilus</em> (<a href="#pone-0000928-t001">Table 1</a>). The presence or absence of genes is indicative of unique functions that may explain phenotypic differences. Despite gene conservation, the possibility of altered functions of homologous genes due to sequence divergence cannot be excluded. Therefore, the translated sequences of common genes were also compared (<a href="#pone-0000928-t002">Tables 2</a> and <a href="#pone-0000928-t003">3</a>). In addition to gene conservation and sequence similarity it is also important to understand gene functions in context of the organism's growth phase. Although the temporal activity of only some proteins discussed here are known, two recent studies describe transcription of many <em>B. subtilis</em> DNA repair and H<sub>2</sub>O<sub>2</sub> resistance genes. Keijser et al. identified transcripts more abundant in spores and germinating cells than in vegetative cells <a href="#pone.0000928-Keijser1">[24]</a>. Moeller et al. identified transcripts induced after exposure of vegetative cells to UVC radiation (200–280 nm) <a href="#pone.0000928-Moeller1">[25]</a>. We cross-referenced our gene lists with these temporal transcription data to augment our genomic comparisons (<a href="#pone-0000928-t001">Table 1</a>). However, it should be understood that because spore survivability assays entail growing surviving spores to countable levels in liquid or solid media, resistance mechanisms at any stage of growth may be important to survivability.</p>
<div class="figure" id="pone-0000928-t001"><div class="img"><a name="pone-0000928-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>List of <em>Bacillus</em> genes involved in DNA repair and oxidative stress resistance.</span></strong></p><span>doi:10.1371/journal.pone.0000928.t001</span></div><div class="figure" id="pone-0000928-t002"><div class="img"><a name="pone-0000928-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Sequence conservation of DNA repair proteins among <em>Bacillus</em> species.</span></strong></p><span>doi:10.1371/journal.pone.0000928.t002</span></div><div class="figure" id="pone-0000928-t003"><div class="img"><a name="pone-0000928-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Sequence conservation of H<sub>2</sub>O<sub>2</sub> resistance proteins among <em>Bacillus</em> species</span></strong></p><span>doi:10.1371/journal.pone.0000928.t003</span></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Previous analyses of the resistance properties of <em>Bacillus</em> spores centered on small acid-soluble spore proteins (SASP) and the spore photoproduct lyase DNA repair system <a href="#pone.0000928-Nicholson1">[26]</a>, <a href="#pone.0000928-Setlow3">[27]</a>. SASP are spore core proteins that play a crucial role in resistance to UV radiation, heat, desiccation, and oxidative damage by binding DNA and altering its reactivity <a href="#pone.0000928-Setlow1">[5]</a>. When exposed to UV radiation, SASP-bound DNA more readily forms the spore photoproduct (SP), 5-thyminyl-5,6-dihydrothymine, rather than cyclobutane dimers or (6-4)-photoproducts, which are formed in the absence of SASP. Unlike these other DNA lesions, SP is easily repaired by the spore photoproduct lyase (SP lyase), which is encoded by <em>splB</em> gene and is negatively regulated by the <em>splA</em> gene product <a href="#pone.0000928-Setlow2">[6]</a>. <em>B. pumilus</em> has an intact <em>splAB</em> operon. The translated SplB (BPUM_1283) sequence is highly conserved in <em>B. pumilus</em>, but SplA (BPUM_1282) shows much more sequence diversity among <em>B. pumilus</em>, <em>B. subtilis</em>, and <em>B. licheniformis</em> (<a href="#pone-0000928-t002">Table 2</a>), indicating possible differences in SP lyase genetic regulation.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p><em>Bacillus subtilis</em> produces 18 SASPs, whose sequences are short (40–100 amino acids) and highly conserved. The α/β-type SspA and SspB predominate, and there are also minor α/β-type SASPs, a γ–type, and novel SASPs <a href="#pone.0000928-Bagyan1">[28]</a>. Fifteen SASP genes were annotated in <em>B. pumilus</em> (<a href="#pone-0000928-t001">Table 1</a>); homologs of SspC, SspG, and SspH were not found. SspC is a minor α/β-type SASP that contributes to UV radiation resistance <a href="#pone.0000928-TovarRojo1">[29]</a>, hence its absence from <em>B. pumilus</em> is paradoxical. SspH and SspG are novel type SASPs, that have no effect on <em>B. subtilis</em> UV radiation resistance <a href="#pone.0000928-Bagyan1">[28]</a>, <a href="#pone.0000928-CabreraHernandez1">[30]</a>. The <em>B. pumilus</em> and <em>B. licheniformis</em> homologs of the γ–type SASP appear to be amino-terminal truncations of the 84 amino acid SspE of <em>B. subtilis</em>. The significance of such a truncation is unclear, as the only known function of SspE is as an amino acid source for germinating spores <a href="#pone.0000928-CarrilloMartinez1">[31]</a>. Although these differences in gene content and sequence conservation may contribute to the enhanced UV and oxidation resistance of <em>B. pumilus</em>, other important factors are likely to be found among DNA repair and oxidative stress response genes</p>

<h4>DNA Repair Mechanisms–Single Strand Repair pathways</h4>
<h5>Base Excision Repair (BER).</h5><a id="article1.body1.sec3.sec1.sec1.p1" name="article1.body1.sec3.sec1.sec1.p1"></a><p>Oxidative damage to DNA is repaired by BER, which is performed by DNA glycosylases and AP (apurinic/apyrimidinic) endonucleases <a href="#pone.0000928-Eisen1">[32]</a>, <a href="#pone.0000928-MartinsPinheiro1">[33]</a>. DNA glycosylases remove damaged bases from the DNA backbone to create an AP site. AP endonucleases bind to this site and cleave the DNA 5′ to the abasic site, forming a free 3′-hydroxyl which is repaired by DNA polymerases. Monofunctional DNA glycosylases only have glycosylase activity, whereas bifunctional DNA glycosylases have both glycosylase and lyase activities as well as the ability to cleave the phosphodiester backbone 3′ to the AP site. <em>B. pumilus</em> encodes both monofunctional [AlkA (BPUM_0752), Ung (BPUM_03444)] and bifunctional [MutM (BPUM_2550), Nth (BPUM_1966)] DNA glycosylases in addition to the AP endonuclease IV, Nfo (BPUM_2246). Nth and Nfo are highly conserved among <em>B. pumilus</em>, <em>B. subtilis</em>, and <em>B. licheniformis</em>, but AlkA, Ung, and MutM have greater sequence divergence (<a href="#pone-0000928-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec1.sec1.p2" name="article1.body1.sec3.sec1.sec1.p2"></a><p><em>B. pumilus</em> lacks a homolog of the AP endonuclease ExoA and the DNA glycosylase YxlJ, both of which are present in <em>B. subtilis</em> and <em>B. licheniformis.</em> The lack of ExoA is not surprising, as <em>B. subtilis exoA</em> mutants do not exhibit enhanced sensitivity to H<sub>2</sub>O<sub>2</sub> <a href="#pone.0000928-SalasPacheco1">[34]</a>. YxlJ functions in the repair of DNA alkylation damage and removal of deaminated purines and cyclic etheno adducts <a href="#pone.0000928-Aamodt1">[35]</a>, and it is transcribed during spore germination and outgrowth <a href="#pone.0000928-Keijser1">[24]</a>; its absence suggests that another protein compensates for its loss.</p>

<h5>Nucleotide Excision Repair (NER).</h5><a id="article1.body1.sec3.sec1.sec2.p1" name="article1.body1.sec3.sec1.sec2.p1"></a><p>While BER recognizes and repairs individual bases by specific DNA glycosylases, NER identifies multi-base distortions in the double helix and removes bulky single-stranded lesions, which are repaired by DNA polymerase I <a href="#pone.0000928-Truglio1">[36]</a>. The <em>E. coli</em> NER pathway consists of UvrA and UvrB, which recognize DNA lesions, the UvrC nuclease, and the UvrD helicase. The NER machinery can be recruited to DNA damage by the Mfd protein in a process called transcription-coupled NER. In <em>B. subtilis</em>, NER is associated with UV radiation resistance in vegetative cells <a href="#pone.0000928-Nicholson1">[26]</a>, and <em>uvrA</em> and <em>uvrB</em> are transcribed in germinating/outgrowing spores <a href="#pone.0000928-Keijser1">[24]</a>. <em>B. subtilis</em> lacks UvrD, but likely uses PcrA to perform the UvrD helicase function <a href="#pone.0000928-Eisen1">[32]</a>, <a href="#pone.0000928-Petit1">[37]</a>. <em>B. pumilus</em> encodes homologs of UvrABC (BPUM_3147, 3148, &amp; 2506), PcrA (BPUM_0625), and Mfd (BPUM_0039), the amino acid sequences of which are conserved with respect to <em>B. subtilis</em> and <em>B. licheniformis</em> (<a href="#pone-0000928-t002">Table 2</a>).</p>

<h5>Mismatch Repair (MMR).</h5><a id="article1.body1.sec3.sec1.sec3.p1" name="article1.body1.sec3.sec1.sec3.p1"></a><p>MMR recognizes and repairs mismatched bases in newly synthesized DNA daughter strands, and although not associated with DNA repair related to UV radiation or oxidative damage, it is important in maintaining genomic integrity <a href="#pone.0000928-PedrazaReyes1">[38]</a>. In <em>E. coli</em>, MMR involves MutS and MutL, which recognize mismatches, and endonuclease MutH. <em>Bacillus</em> species lack MutH and must use another, unidentified mechanism <a href="#pone.0000928-Smith1">[39]</a>. <em>B. pumilus</em> MutS (BPUM_1608) and MutL (BPUM_1609) homologs are moderately well-conserved compared to those of <em>B. subtilis</em> and <em>B. licheniformis</em> (<a href="#pone-0000928-t002">Table 2</a>). Homologs of XseA (BPUM_2162) and XseB (BPUM_2161), subunits of a MMR exonuclease, were also annotated in <em>B. pumilus</em>.</p>



<h4>DNA Repair Mechanisms–Double Strand Repair pathways</h4>
<h5>Non-Homologous End-Joining (NHEJ).</h5><a id="article1.body1.sec3.sec2.sec1.p1" name="article1.body1.sec3.sec2.sec1.p1"></a><p>The NHEJ pathway repairs double-strand DNA (DSB) breaks by directly joining DNA ends without requiring a homologous template to guide the repair <a href="#pone.0000928-Moore1">[40]</a>. Prokaryotic homologs of the eukaryotic DNA-end-binding protein, Ku, and DNA ligase IV were recently identified in several bacteria <a href="#pone.0000928-Weller1">[41]</a>. In <em>B. subtilis</em>, the NHEJ proteins are encoded on the <em>ykoUVW</em> operon, and <em>ykoU</em> and <em>ykoV</em> are transcribed both in vegetative cells and germinating/outgrowing spores <a href="#pone.0000928-Wang1">[42]</a>. <em>B. subtilis</em> YkoV (Ku) specifically recruits YkoU (DNA ligase IV) to DNA ends to stimulate DNA ligation, and loss of these proteins leads to hypersensitivity to UV radiation in <em>B. subtilis</em> <a href="#pone.0000928-Moeller2">[43]</a>. YkoW is hypothesized to interact with dsDNA ends.</p>
<a id="article1.body1.sec3.sec2.sec1.p2" name="article1.body1.sec3.sec2.sec1.p2"></a><p>There is significant amino acid sequence variation in NHEJ proteins among <em>B. pumilus</em>, <em>B. subtilis</em>, and <em>B. licheniformis</em> (<a href="#pone-0000928-t002">Table 2</a>). The YkoV and YkoU sequences of <em>B. subtilis</em> and <em>B. licheniformis</em> are more closely related to each other than to their <em>B. pumilus</em> homologs (BPUM_1667 &amp; BPUM_1666). Additionally, <em>B. pumilus</em> YkoW (BPUM_1234), at 807 amino acids in length, is much longer than <em>B. subtilis</em> YkoW (749 amino acids) and <em>B. licheniformis</em> (549 amino acids). Beyond these differences in amino acid sequences, which may affect protein function, the regulation of NHEJ genes appears to be different in <em>B. pumilus</em>. The <em>ykoUVW</em> operon structure of <em>B. subtilis</em> and <em>B. licheniformis</em> is not conserved in <em>B. pumilus</em>. In <em>B. pumilus, ykoU</em> and <em>ykoV</em> are adjacent and divergently transcribed, while <em>ykoW</em> is located on a separate locus.</p>

<h5>Recombinational Repair.</h5><a id="article1.body1.sec3.sec2.sec2.p1" name="article1.body1.sec3.sec2.sec2.p1"></a><p>Homologous recombination (HR) is a ubiquitous process that is crucial for DNA repair and maintenance. It is a multi-step pathway involving several proteins that facilitate the invasion of dsDNA by a ssDNA substrate. As DNA is unwound by a helicase, the migrating strand replaces damaged DNA and the intermediate structure is resolved by an endonuclease <a href="#pone.0000928-Eisen1">[32]</a>. HR in <em>Bacillus</em> species can be initiated by the AddAB pathway, which is analogous to <em>E. coli</em> RecBCD <a href="#pone.0000928-Kooistra1">[44]</a>, or the RecFOR pathway <a href="#pone.0000928-Eisen1">[32]</a>. Strand invasion and exchange is catalyzed by RecA, and <em>recA</em> mutations increase sensitivity to UV radiation <a href="#pone.0000928-Nicholson1">[26]</a>. Branch migration is performed by the RuvAB proteins, and resolution is performed by the RecU (RuvC in <em>E. coli</em>), and RecG proteins <a href="#pone.0000928-Eisen1">[32]</a>. <em>B. pumilus</em> encodes homologs of all HR proteins common to <em>B. subtilis</em> and <em>B. licheniformis</em> (<a href="#pone-0000928-t001">Tables 1</a> &amp; <a href="#pone-0000928-t002">2</a>).</p>
<a id="article1.body1.sec3.sec2.sec2.p2" name="article1.body1.sec3.sec2.sec2.p2"></a><p>Control of HR is closely related to the SOS repair system. In <em>B. subtilis</em>, the SOS regulon is similar to that in <em>E. coli</em> but it is also induced in competent cells in the absence of any DNA-damaging treatment <a href="#pone.0000928-Yasbin1">[45]</a>. RecA and the SOS transcriptional repressor LexA are the two main proteins involved in this coordinated cellular response to UV-light and DNA-damaging agents <a href="#pone.0000928-Au1">[46]</a>. RecA is activated by ssDNA and promotes LexA self-cleavage, causing it to lose affinity to DNA and allowing expression of the SOS-response genes. The <em>B. pumilus</em> LexA (BPUM_1686) sequence is 87% similar to the <em>B. subtilis</em> and <em>B. licheniformis</em> LexA homologs, and their DNA-binding motifs are identical <a href="#pone.0000928-Groban1">[47]</a>, suggesting that their activities are similar in these three species. Several SOS proteins have been identified in <em>E. coli</em> and <em>B. subtilis</em>, but the identification of <em>B. pumilus</em> SOS proteins will require experimental verification of regulation by RecA and LexA <a href="#pone.0000928-Au1">[46]</a>. HR is also under the influence of RecX, a repressor of <em>recA</em> <a href="#pone.0000928-Lin1">[48]</a>. <em>B. pumilus</em> RecX (BPUM_0795) has moderate sequence conservation with its <em>B. subtilis</em> and <em>B. licheniformis</em> homologs.</p>



<h4>Other DNA repair systems</h4>
<h5>UVDE-dependent excision repair.</h5><a id="article1.body1.sec3.sec3.sec1.p1" name="article1.body1.sec3.sec3.sec1.p1"></a><p>YwjD is a <em>B. subtilis</em> homolog of UVDE, a eukaryotic protein that repairs UV radiation-induced cyclobutane pyrimidine dimers and 6-4 photoproducts <a href="#pone.0000928-Kanno1">[49]</a>. <em>B. pumilus</em> encodes a YwjD (BPUM_3376) homolog that shares only moderate sequence identity with <em>B. subtilis</em> YwjD, which is produced in vegetative cells, and there is no <em>B. licheniformis</em> homolog. Because the sequence conservation is poor, it is possible that <em>B. pumilus</em> YwjD functions in a way that enhances its DNA repair activity.</p>

<h5>Y-family polymerases.</h5><a id="article1.body1.sec3.sec3.sec2.p1" name="article1.body1.sec3.sec3.sec2.p1"></a><p>The Y family polymerases are error-prone, translesional DNA polymerases that are processive through DNA lesions that block the replicative polymerase <a href="#pone.0000928-Duigou1">[50]</a>. Two Y-family polymerases were annotated in <em>B. pumilus</em> and named for their <em>B. subtilus</em> homologs, PolY1 (YqjH; BPUM_2125) and PolY2 (YqjW; BPUM_2102). In <em>B. subtilis</em> PolY2 is an SOS inducible polymerase that functions in UV damage repair and is necessary for UV-induced mutagenesis <a href="#pone.0000928-Duigou1">[50]</a>. PolY2 is missing in <em>B. licheniformis,</em> which may contribute to its relative UV sensitivity. The fact that PolY2 is present in both <em>B. pumilus</em> and <em>B. subtilis</em> means that it alone cannot account for the UV resistance of <em>B. pumilus</em>. However, sequence variation (<a href="#pone-0000928-t002">Table 2</a>) and differences in expression may influence its activity. In contrast, PolY1 is a DinB subfamily polymerase that is constitutively expressed and functions in untargeted mutagenesis rather than UV-induced mutagenesis <a href="#pone.0000928-Duigou1">[50]</a>. PolY1 is common to <em>B. pumilus</em>, <em>B. subtilis</em>, and <em>B. licheniformis</em>, and, therefore, unlikely to be responsible for UV resistance. Two other <em>B. subtilis</em> Y-family polymerases, UvrX and YozK-YobH are encoded on integrated prophages that are not present in <em>B. pumilus</em> <a href="#pone.0000928-Duigou1">[50]</a>.</p>

<h5>Alkyltransferases.</h5><a id="article1.body1.sec3.sec3.sec3.p1" name="article1.body1.sec3.sec3.sec3.p1"></a><p>Alkylating chemicals can mutate DNA bases or the phosphodiester backbone by adding an alkyl group to the nitrogen or oxygen atoms. Ogt is a methyltransferase that removes the alkyl group from O<sup>6</sup>-alkyl guanine or, preferentially, O<sup>4</sup>-alkyl thymine <a href="#pone.0000928-Sassanfar1">[51]</a>. Ogt also exhibits suicide inactivation by transferring an alkyl group to a cysteine residue in its own structure. Ogt (BPUM_1248) is found in <em>B</em>. <em>pumilus, B. subtilis</em> and <em>B. licheniformis</em>, although the protein sequence is not well-conserved (<a href="#pone-0000928-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec3.sec3.p2" name="article1.body1.sec3.sec3.sec3.p2"></a><p>The <em>B. pumilus</em> genome encodes a second alkyltransferase, Ada (BPUM_1200), which, like Ogt, removes alkyl moities from DNA by suicide inactivation <a href="#pone.0000928-Eisen1">[32]</a>. Ada also initiates the adaptive response, which activates several DNA repair enzymes <a href="#pone.0000928-Rohankhedkar1">[52]</a>. There are notable differences between <em>B. pumilus</em> Ada and the homologs of <em>B. subtilis</em> and <em>B. licheniformis</em> that may be significant in DNA repair. <em>B. pumilus</em> Ada, like <em>E. coli</em> Ada, incorporates a regulatory domain fused to the alkyl glycosylase domain. However, in both <em>B. subtilis</em> and <em>B. licheniformis</em>, these domains are split into two proteins <a href="#pone.0000928-Morohoshi1">[53]</a>, AdaA and AdaB, neither of which align with greater than 50% identity to <em>B. pumilus</em> Ada. The fusion of the two proteins in <em>B. pumilus</em> raises the possibility that the function and transcriptional regulation of this alkyltransferase may be different in <em>B. pumilus</em> compared to <em>B. subtilis</em> and <em>B. licheniformis</em>.</p>

<h5>Other proteins.</h5><a id="article1.body1.sec3.sec3.sec4.p1" name="article1.body1.sec3.sec3.sec4.p1"></a><p>The ATP-dependent DNA helicase DinG can unwind RNA or DNA, and it is a bacterial homolog of a human DNA repair helicase <a href="#pone.0000928-Voloshin1">[54]</a>, <a href="#pone.0000928-Yasuda1">[55]</a>. Homologs of DinG (BPUM_1971) are present in <em>B. pumilus, B. subtilis</em> and <em>B. licheniformis,</em> although their sequences are not well conserved (<a href="#pone-0000928-t002">Table 2</a>).</p>
<a id="article1.body1.sec3.sec3.sec4.p2" name="article1.body1.sec3.sec3.sec4.p2"></a><p>The Nudix hydrolase superfamily MutT protein hydrolyzes 8-oxo-dGTP (a reactive oxygen species) and prevents its incorporation into DNA <a href="#pone.0000928-McLennan1">[56]</a>. <em>B. subtilis</em> has three MutT superfamily genes, <em>mutT,</em> which is transcribed in germinating/outgrowing spores <a href="#pone.0000928-Keijser1">[24]</a>, <em>yjhB</em>, and <em>yvcI</em>. The <em>B. pumilus</em> genome has one <em>yvcI</em> gene (BPUM_3116), but no <em>mutT</em>; or <em>yjhB</em> homologs.</p>
<a id="article1.body1.sec3.sec3.sec4.p3" name="article1.body1.sec3.sec3.sec4.p3"></a><p>YshD is a MutS2 family protein that maintains genome integrity by inhibiting intergenomic recombination. The YshD (BPUM_2516) sequence is conserved among <em>B. pumilus</em>, <em>B. subtilis,</em> and <em>B. licheniformis</em> (<a href="#pone-0000928-t002">Table 2</a>), but it is unclear if it has an effect on UV or H<sub>2</sub>O<sub>2</sub> resistance.</p>



<h4>DNA repair proteins unique to B. pumilus</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The <em>B. pumilus</em> genome encodes PhrB (BPUM_1378), a DNA photolyase enzyme that repairs cyclobutane-pyrimidine dimers <a href="#pone.0000928-Essen1">[57]</a>. Although no homolog exists in <em>B. subtilis</em> and <em>B. licheniformis</em>, there are homologs in other <em>Bacillus</em> species such as <em>B. firmus</em>, <em>B. cereus, B. anthracis</em>, and <em>B. thuringiensis</em>. Nevertheless, none of these species exhibit UV radiation resistance comparable to <em>B. pumilus</em>. The <em>B. subtilis</em> photolyase amino acid sequence is diverse with respect to other photolyases. It shares 32% amino acid identity with <em>E. coli</em> PhrB and only 46% sequence identity with its closest homolog from <em>B. firmus</em>. It is logical that the presence of a photolyase gives <em>B. pumilus</em> UV resistance capabilities that <em>B. subtilis</em> and <em>B. licheniformis</em> lack. However, because less UV-resistant <em>Bacillus</em> species also have photolyase enzymes, the relation of photolyase to enhanced UV resistance is not clear. Although the sequence divergence in the <em>B. pumilus</em> photolyase may indicate altered function, <em>B. pumilus</em> may rely on a combination of other factors for its UV resistance properties.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Genes encoding two DNA repair/modification proteins not found in <em>B. subtilis</em> and <em>B. licheniformis</em> were also annotated in <em>B. pumilus</em>. One sequence (BPUM_0608) is similar to a Superfamily II (SF-2) helicase based upon the presence of a DExD Walker B motif in conserved motif II <a href="#pone.0000928-Caruthers1">[58]</a>. SF-2 helicases are known to function in NER and recombinational repair in yeast <a href="#pone.0000928-Brosh1">[59]</a>. Although it cannot be predicted that this helicase functions in DNA repair, if it does have such a function it would be a feature that <em>B. subtilis</em> and <em>B. licheniformis</em> lack, possibly contributing to the enhanced UV radiation resistance. <em>B. pumilus</em> also encodes a C-5 cytosine-specific DNA methyltransferase (BPUM_0656) that has no homolog in either <em>B. subtilis</em> or <em>B. licheniformis</em>. Though unlikely to be directly implicated in DNA repair, it is possible that a unique DNA-modifying protein may contribute to genomic stability in <em>B. pumilus</em>. Additionally, the <em>B. pumilus</em> genome has 517 coding sequences that are not common to <em>B. subtilis</em> and <em>B. licheniformis</em>, including 218 hypothetical proteins that have no sequence similarity to any known sequence in the nr database. It is possible that one more of these coding sequences of unknown function may contribute to UV radiation resistance.</p>


<h4>H<sub>2</sub>O<sub>2</sub> resistance</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p><em>Bacillus</em> species use a variety of proteins to resist the toxic effects of H<sub>2</sub>O<sub>2</sub>, including catalases, and various reducing proteins such as alkyl hydroperoxide reductase and peroxiredoxins <a href="#pone.0000928-Mostertz1">[60]</a>. Analysis of the <em>B. pumilus</em> genome reveals many striking differences compared to similar proteins in <em>B. subtilis</em> and <em>B. licheniformis</em>.</p>
<h5>Catalase.</h5><a id="article1.body1.sec3.sec5.sec1.p1" name="article1.body1.sec3.sec5.sec1.p1"></a><p>Catalases convert H<sub>2</sub>O<sub>2</sub> into water and oxygen in a highly efficient reaction that requires neither ATP nor an exogenous reducing agent <a href="#pone.0000928-Farr1">[8]</a>. <em>B. subtilis</em> and <em>B. licheniformis</em> produce two vegetative catalases, KatA and KatB (KatE), and one germination catalase, KatX, which is present in spores and protects germinating cells from H<sub>2</sub>O<sub>2</sub> <a href="#pone.0000928-Bagyan2">[61]</a>. All three catalases are transcribed in germinating/outgrowing spores <a href="#pone.0000928-Keijser1">[24]</a>. <em>B. pumilus</em> has no homolog to either vegetative catalase, however, it has two KatX homologs. The sequence conservation of KatX1 (BPUM_3712) is moderate, but KatX2 (BPUM_0892) is more diverse, sharing less than 50% identity with <em>B. subtilis</em> and <em>B. licheniformis</em> KatX. A second germination-specific catalase with substantial sequence diversity is a candidate protein that may explain the enhanced peroxide resistance of <em>B. pumilus</em> spores.</p>
<a id="article1.body1.sec3.sec5.sec1.p2" name="article1.body1.sec3.sec5.sec1.p2"></a><p>YjqC and YdbD are two additional proteins with catalase domains that are found in <em>B. subtilis</em> and <em>B. licheniformis</em>, although little is known of their functions. <em>B. pumilus</em> YjqC (BPUM_2346) shares moderate sequence identity with its <em>Bacillus</em> homologs (<a href="#pone-0000928-t003">Table 3</a>), but there is no YdbD homolog in <em>B. pumilus</em>. However, a <em>B. pumilus</em> sequence containing a manganese catalase domain does exist (YdbD uses Mn<sup>2+</sup> as a cofactor). It is possible that this catalase (BPUM_1305), which differs greatly from YdbD, may have properties that contribute to the H<sub>2</sub>O<sub>2</sub> resistance of <em>B. pumilus</em>.</p>
<a id="article1.body1.sec3.sec5.sec1.p3" name="article1.body1.sec3.sec5.sec1.p3"></a><p>The spore coat protein CotJC contains a predicted catalase domain in its amino acid sequence. Although CotJC is present in <em>B. subtilis</em> and <em>B. licheniformis</em>, no homolog was identified in <em>B. pumilus</em>, suggesting that it is not necessary for elevated peroxide resistance.</p>

<h5>Peroxiredoxins.</h5><a id="article1.body1.sec3.sec5.sec2.p1" name="article1.body1.sec3.sec5.sec2.p1"></a><p>Bacteria use peroxiredoxins to reduce H<sub>2</sub>O<sub>2</sub> to water <a href="#pone.0000928-Rhee1">[62]</a>. Four peroxiredoxins were annotated in <em>B. pumilus</em>. Three peroxiredoxin protein sequences, YkuU (BPUM_1319), YgaF (BPUM_0826), and Tpx (BPUM_2581), are highly conserved with respect to their <em>B. subtilis</em> and <em>B. licheniformis</em> homologs (<a href="#pone-0000928-t003">Table 3</a>). The fourth peroxiredoxin (BPUM_3690) annotated in <em>B. pumilus</em> has no obvious homolog in <em>B. subtilis</em> or <em>B. licheniformis</em>. Instead, these two species produce an alkyl hydroperoxide reductase that is induced upon H<sub>2</sub>O<sub>2</sub> stress. The enzyme is a heterodimer of AhpC and AhpF, and it uses NADH or NADPH as a reducing agent. In <em>B. subtilis</em> and <em>B. licheniformis</em> the subunits are encoded on the <em>ahpCF</em> operon, and their translated sequences are highly conserved (&gt;90% identity for AhpC and AhpF). The <em>B. pumilus</em> genome does not contain a homologous operon. However, it does it does have a gene encoding an NADH dehydrogenase (BPUM_2106), which, if coupled with the peroxiredoxin (BPUM_3690), could hypothetically function as an alkyl hydroperoxide reductase. Given the lack of sequence and gene order conservation, the function may be distinct from <em>B. subtilis</em> and <em>B. licheniformis</em> AphCF, possibly explaining the abnormal H<sub>2</sub>O<sub>2</sub> resistance of <em>B. pumilus</em>.</p>

<h5>Peroxidases.</h5><a id="article1.body1.sec3.sec5.sec3.p1" name="article1.body1.sec3.sec5.sec3.p1"></a><p>Peroxidases also reduce H<sub>2</sub>O<sub>2</sub> to water using NADH or NADPH as a cofactor <a href="#pone.0000928-Farr1">[8]</a>. A glutathione peroxidase, BsaA (BPUM_1925), was annotated in <em>B. pumilus</em>. BsaA uses glutathione as a reducing agent to reduce lipid hydroxyperoxides formed by peroxide stress, and <em>bsaA</em> is transcribed during spore germination/outgrowth <a href="#pone.0000928-Keijser1">[24]</a>. There is substantial sequence diversity among the BsaA homologs of <em>B. pumilus</em>, <em>B. subtilis</em>, and <em>B. licheniformis</em> (<a href="#pone-0000928-t003">Table 3</a>), raising the possibility that differences in <em>B. pumilus</em> BsaA may contribute to H<sub>2</sub>O<sub>2</sub> resistance.</p>

<h5>Other reducing agents.</h5><a id="article1.body1.sec3.sec5.sec4.p1" name="article1.body1.sec3.sec5.sec4.p1"></a><p>Thioredoxins and glutaredoxins are instrumental to peroxide stress resistance. They reduce peroxiredoxins and peroxidases, facilitating their functions, and act as hydroxyl radical scavengers <a href="#pone.0000928-Zeller1">[63]</a>. They also maintain oxidation states of cytoplasmic proteins, preventing illegitimate disulfide bond formation <a href="#pone.0000928-Smits1">[64]</a>. Several redox proteins were annotated in <em>B. pumilus</em>, but only those known to be related to peroxide stress and those unique compared to <em>B. subtilis</em> and <em>B. licheniformis</em> are mentioned in this work.</p>
<a id="article1.body1.sec3.sec5.sec4.p2" name="article1.body1.sec3.sec5.sec4.p2"></a><p>TrxA is the product of the thioredoxin A gene, which is essential in <em>B. subtilis</em>. The reducing potential of TrxA is recycled by the thioredoxin reductase, TrxB. <em>B. pumilus</em> TrxA (BPUM_2507) and TrxB (BPUM_3117) share approximately 90% identity with their <em>B. subtilis</em> homologs. A second TrxB-like thioredoxin reductase (BPUM_0664) was annotated in <em>B. pumilus</em> (<a href="#pone-0000928-t004">Table 4</a>). This protein has no <em>B. subtilis</em> homolog and a poor alignment to a <em>B. licheniformis</em> reductase, so it may provide peroxide stress resistance capabilities not available to these species. YcgT (BPUM_0777), another thioredoxin-disulfide reductase, is present in <em>B. pumilus</em>, but its sequence is not well-conserved with <em>B. subtilis</em> and <em>B. licheniformis</em>, raising the possibility that differences in YcgT activity may be important to <em>B. pumilus</em> oxidative stress resistance.</p>
<div class="figure" id="pone-0000928-t004"><div class="img"><a name="pone-0000928-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000928" data-uri="info:doi/10.1371/journal.pone.0000928.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000928.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000928.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000928.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000928.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Unique DNA repair and H<sub>2</sub>O<sub>2</sub> resistance proteins of <em>B. pumilus</em>.</span></strong></p><span>doi:10.1371/journal.pone.0000928.t004</span></div>
<h5>The <em>ohr</em> operon.</h5><a id="article1.body1.sec3.sec5.sec5.p1" name="article1.body1.sec3.sec5.sec5.p1"></a><p>In <em>B. subtilis</em> resistance to organic peroxides is encoded by the <em>ohr</em> locus, which produces the peroxide resistance proteins OhrA and OhrB, and OhrR, a transcriptional regulator of <em>ohrA</em> <a href="#pone.0000928-Fuangthong1">[65]</a>. The <em>B. pumilus</em> homologs (BPUM_1211-1213) of these proteins share moderate homology with their <em>B. subtilis</em> and <em>B. licheniformis</em> homologs (<a href="#pone-0000928-t003">Table 3</a>), so they may have altered function due to sequence diversity.</p>

<h5>Regulation of the oxidative stress response.</h5><a id="article1.body1.sec3.sec5.sec6.p1" name="article1.body1.sec3.sec5.sec6.p1"></a><p>Several transcriptional regulators of the <em>B. subtilis</em> oxidative stress response are known, including PerR, Spx, and sigma factors SigM and SigB. All four of these proteins are conserved in <em>B. pumilus</em> (<a href="#pone-0000928-t003">Table 3</a>). Dps proteins are DNA- binding proteins that protect bacteria from oxidative stress by sequestering iron and oxidants and storing them as benign ferric oxide minerals <a href="#pone.0000928-Liu1">[66]</a>. Two Dps proteins, DpsA (YktB) and MrgA are known in <em>B. subtilis</em> and <em>B. licheniformis</em>. <em>B. pumilus</em> does encode a DpsA homolog (<a href="#pone-0000928-t003">Table 3</a>; BPUM_2703), however, it has no MrgA homolog. Although MrgA is important for peroxide resistance proteins in vegetative cells, it has no effect on peroxide resistance in spores <a href="#pone.0000928-CasillasMartinez1">[67]</a>.</p>

<h5>Other oxidative stress proteins.</h5><a id="article1.body1.sec3.sec5.sec7.p1" name="article1.body1.sec3.sec5.sec7.p1"></a><p>Oxidative stress also occurs in the form of superoxide, O<sub>2</sub><sup>−</sup>. Although the O<sub>2</sub><sup>−</sup> and H<sub>2</sub>O<sub>2</sub> stress responses are distinct, the conditions are related, via the chemical conversion of O<sub>2</sub><sup>−</sup> to H<sub>2</sub>O<sub>2</sub> by superoxide dismutases. <em>B. pumilus</em> has three superoxide dismutases: SodA (BPUM_2230), which uses a manganese cofactor, SodF (BPUM_1859), which uses an iron cofactor, and YojM (BPUM_1865), which uses copper or zinc as a cofactor <a href="#pone.0000928-Farr1">[8]</a>. <em>B. pumilus</em> SodA is highly conserved with respect to the homologs of <em>B. subtilis</em> and <em>B. licheniformis</em>, but there is much greater sequence diversity in SodF and YojM (<a href="#pone-0000928-t003">Table 3</a>). If these changes in sequence have any effect on protein function, it is difficult to speculate what benefit there would be for H<sub>2</sub>O<sub>2</sub> resistance, as any decrease in superoxide reductase-mediated H<sub>2</sub>O<sub>2</sub> production would mean less efficient removal of O<sub>2</sub><sup>−</sup>. The hydrolase YqjL (BPUM_2113) and the efflux protein BcrC (BPUM_3294) both contribute to O<sub>2</sub><sup>−</sup> resistance by unknown mechanisms <a href="#pone.0000928-Cao1">[68]</a>. The <em>B. pumilus</em> homologs of these proteins are not well-conserved with respect to their <em>B. subtilis</em> and <em>B. licheniformis</em> homologs (<a href="#pone-0000928-t003">Table 3</a>). It is possible that <em>B. pumilus</em> YjqL and/or BcrC may be more adept at O<sub>2</sub><sup>−</sup> detoxification than their <em>B. subtilis</em> and <em>B. licheniformis</em> homologs, and that their activities may alleviate the production of H<sub>2</sub>O<sub>2</sub> by superoxide dismutases.</p>
<a id="article1.body1.sec3.sec5.sec7.p2" name="article1.body1.sec3.sec5.sec7.p2"></a><p>Two additional proteins related to oxidative stress resistance in <em>B. subtilis</em> were not annotated in the <em>B. pumilus</em> genome. YlaC is a <em>B. subtilis</em> extracytoplasmic sigma factor that is regulated by the anti-sigma factor YlaD, which contains an oxidative stress-sensing domain <a href="#pone.0000928-Matsumoto1">[69]</a>. Transcription of <em>ylaCD</em> was also shown to be Spx-dependent, further linking it to the oxidative stress response. Nevertheless, the lack of YlaC and YlaD homologs in <em>B. pumilus</em> indicates that despite the function of these proteins, they are not essential to H<sub>2</sub>O<sub>2</sub> resistance.</p>
<a id="article1.body1.sec3.sec5.sec7.p3" name="article1.body1.sec3.sec5.sec7.p3"></a><p>The database of annotated <em>B. pumilus</em> coding sequences was examined for oxygenases, oxidoreductases, and redoxins without homologs in <em>B. subtilis</em> or <em>B. licheniformis</em>. The predicted proteins and functions associated with these regions are listed in <a href="#pone-0000928-t004">Table 4</a>. Additionally, it is possible that proteins functioning in peroxide resistance are among the hypothetical proteins and other undefined ORFs that have no <em>B. subtilis</em> or <em>B. licheniformis</em> homolog.</p>



<h4>Conclusion</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Given the phenotypic differences between <em>B. pumilus</em> and <em>B. subtilis</em> and <em>B. licheniformis</em> in terms of UV radiation resistance and H<sub>2</sub>O<sub>2</sub> resistance, it was expected that a comparison of the genomes of these species would point to unique <em>B. pumilus</em> genes related to these functions. Most genes related to DNA repair and H<sub>2</sub>O<sub>2</sub> resistance are conserved among these species. Paradoxically, <em>B. pumilus</em> lacks several DNA repair and oxidative stress response genes found in <em>B. subtilis</em> and <em>B. licheniformis</em>. Nevetheless, this analysis has identified several <em>B. pumilus</em> genes worthy of further study because of their absence in related organisms, differences in amino acid sequence, or predicted differences in genetic regulation.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000928"></div><a name="pone.0000928.s001" id="pone.0000928.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000928.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Locus tag numbers of genes in <a href="#pone-0000928-t001">Table 1</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000928.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.09 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We like to especially acknowledge John Rummel, NASA's Planetary Protection Officer for his constant encouragement of our efforts in this area.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GW KV GF WN SH. Performed the experiments: KV DM SD YD AH WL LP CK HD SL LN PB CB MH XQ HJ. Analyzed the data: GW AU KV AV GF JG SY MT YL ID LB MF FK PE JS PB AA HZ BM AO KC DN. Contributed reagents/materials/analysis tools: GW JP KV DM SD YD AH WL LP CK HD SL LN PB CB MH SH JG SY XQ HJ OI MF LG. Wrote the paper: GW JG SY.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000928-Priest1" id="pone.0000928-Priest1"></a>Priest FG (1993)  Systematics and Ecology of <em>Bacillus</em>. In: Sonenshein AL, Hoch JA, Losick R, editors. <em>Bacillus subtilis</em> and Other Gram-Positive Bacteria: Biochemistry, Physiology, and Molecular Genetics. Washington, D.C.: ASM Press. pp. 3–16.  <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0000928-Benardini1" id="pone.0000928-Benardini1"></a>Benardini JN, Sawyer J, Venkateswaran K, Nicholson WL (2003) Spore UV and acceleration resistance of endolithic Bacillus pumilus and Bacillus subtilis isolates obtained from Sonoran desert basalt: implications for lithopanspermia. Astrobiology  3: 709–717.  <ul class="find" data-citedArticleID="995196" data-doi="10.1089/153110703322736033"><li><a href="http://dx.doi.org/10.1089/153110703322736033" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spore+UV+and+acceleration+resistance+of+endolithic+Bacillus+pumilus+and+Bacillus+subtilis+isolates+obtained+from+Sonoran+desert+basalt%3A+implications+for+lithopanspermia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spore+UV+and+acceleration+resistance+of+endolithic+Bacillus+pumilus+and+Bacillus+subtilis+isolates+obtained+from+Sonoran+desert+basalt%3A+implications+for+lithopanspermia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000928-Kempf1" id="pone.0000928-Kempf1"></a>Kempf MJ, Chen F, Kern R, Venkateswaran K (2005) Recurrent isolation of hydrogen peroxide-resistant spores of <em>Bacillus pumilus</em> from a spacecraft assembly facility. Astrobiology  5: 391–405.  <ul class="find" data-citedArticleID="995242" data-doi="10.1089/ast.2005.5.391"><li><a href="http://dx.doi.org/10.1089/ast.2005.5.391" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recurrent+isolation+of+hydrogen+peroxide-resistant+spores+of+Bacillus+pumilus+from+a+spacecraft+assembly+facility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recurrent+isolation+of+hydrogen+peroxide-resistant+spores+of+Bacillus+pumilus+from+a+spacecraft+assembly+facility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000928-Newcombe1" id="pone.0000928-Newcombe1"></a>Newcombe DA, Schuerger AC, Benardini JN, Dickinson D, Tanner R, et al.  (2005) Survival of spacecraft-associated microorganisms under simulated martian UV irradiation. Applied Environmental Microbiology  71: 8147–8156.  <ul class="find" data-citedArticleID="995272" data-doi="10.1128/aem.71.12.8147-8156.2005"><li><a href="http://dx.doi.org/10.1128/aem.71.12.8147-8156.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survival+of+spacecraft-associated+microorganisms+under+simulated+martian+UV+irradiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survival+of+spacecraft-associated+microorganisms+under+simulated+martian+UV+irradiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000928-Setlow1" id="pone.0000928-Setlow1"></a>Setlow P (2006) Spores of <em>Bacillus subtilis</em>: their resistance to and killing by radiation, heat and chemicals. Journal of Applied Microbiology  101: 514–525.  <ul class="find" data-citedArticleID="995296" data-doi="10.1111/j.1365-2672.2005.02736.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2672.2005.02736.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spores+of+Bacillus+subtilis%3A+their+resistance+to+and+killing+by+radiation%2C+heat+and+chemicals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spores+of+Bacillus+subtilis%3A+their+resistance+to+and+killing+by+radiation%2C+heat+and+chemicals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000928-Setlow2" id="pone.0000928-Setlow2"></a>Setlow P (2001) Resistance of spores of Bacillus species to ultraviolet light. Environ Mol Mutagen  38: 97–104.  <ul class="find" data-citedArticleID="995298" data-doi="10.1002/em.1058"><li><a href="http://dx.doi.org/10.1002/em.1058" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+of+spores+of+Bacillus+species+to+ultraviolet+light." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+of+spores+of+Bacillus+species+to+ultraviolet+light.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000928-Cortezzo1" id="pone.0000928-Cortezzo1"></a>Cortezzo DE, Setlow P (2005) Analysis of factors that influence the sensitivity of spores of <em>Bacillus subtilis</em> to DNA damaging chemicals. Journal of Applied Microbiology  98: 606–617.  <ul class="find" data-citedArticleID="995210" data-doi="10.1111/j.1365-2672.2004.02495.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2672.2004.02495.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+factors+that+influence+the+sensitivity+of+spores+of+Bacillus+subtilis+to+DNA+damaging+chemicals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+factors+that+influence+the+sensitivity+of+spores+of+Bacillus+subtilis+to+DNA+damaging+chemicals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000928-Farr1" id="pone.0000928-Farr1"></a>Farr SB, Kogoma T (1991) Oxidative stress responses in <em>Escherichia coli</em> and <em>Salmonella typhimurium</em>. Microbiological Reviews  55: 561–585.  <ul class="find" data-citedArticleID="995222"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Oxidative+stress+responses+in+Escherichia+coli+and+Salmonella+typhimurium.&amp;auth=&amp;atitle=Oxidative+stress+responses+in+Escherichia+coli+and+Salmonella+typhimurium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oxidative+stress+responses+in+Escherichia+coli+and+Salmonella+typhimurium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oxidative+stress+responses+in+Escherichia+coli+and+Salmonella+typhimurium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000928-Sambrook1" id="pone.0000928-Sambrook1"></a>Sambrook J, Russell DW (2001) Molecular cloning: a laboratory manual. Cold Spring Harbor, New York: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0000928-Margulies1" id="pone.0000928-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="995252"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000928-Andersson1" id="pone.0000928-Andersson1"></a>Andersson B, Wentland MA, Ricafrente JY, Liu W, Gibbs RA (1996) A “double adaptor” method for improved shotgun library construction. Analytical Chemistry  236: 107–113.  <ul class="find" data-citedArticleID="995188" data-doi="10.1006/abio.1996.0138"><li><a href="http://dx.doi.org/10.1006/abio.1996.0138" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+%E2%80%9Cdouble+adaptor%E2%80%9D+method+for+improved+shotgun+library+construction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+%E2%80%9Cdouble+adaptor%E2%80%9D+method+for+improved+shotgun+library+construction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000928-Havlak1" id="pone.0000928-Havlak1"></a>Havlak P, Chen R, Durbin KJ, Egan A, Ren Y, et al.  (2004) The Atlas genome assembly system. Genome Research  14: 721–732.  <ul class="find" data-citedArticleID="995232" data-doi="10.1101/gr.2264004"><li><a href="http://dx.doi.org/10.1101/gr.2264004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Atlas+genome+assembly+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Atlas+genome+assembly+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000928-McLeod1" id="pone.0000928-McLeod1"></a>McLeod MP, Qin X, Karpathy SE, Gioia J, Highlander SK, et al.  (2004) Complete genome sequence of <em>Rickettsia typhi</em> and comparison with sequences of other rickettsiae. Journal of Bacteriology  186: 5842–5855.  <ul class="find" data-citedArticleID="995260" data-doi="10.1128/jb.186.17.5842-5855.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.17.5842-5855.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complete+genome+sequence+of+Rickettsia+typhi+and+comparison+with+sequences+of+other+rickettsiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complete+genome+sequence+of+Rickettsia+typhi+and+comparison+with+sequences+of+other+rickettsiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000928-Delcher1" id="pone.0000928-Delcher1"></a>Delcher AL, Harmon D, Kasif S, White O, Salzberg SL (1999) Improved microbial identification with GLIMMER. Nucleic Acids Research  27: 4636–4641.  <ul class="find" data-citedArticleID="995212" data-doi="10.1093/nar/27.23.4636"><li><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+microbial+identification+with+GLIMMER." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+microbial+identification+with+GLIMMER.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000928-Lukashin1" id="pone.0000928-Lukashin1"></a>Lukashin A, Borodovsky M (1998) GeneMark.hmm: new solutions for gene finding. Nucleic Acids Research  26: 1107–1115.  <ul class="find" data-citedArticleID="995250" data-doi="10.1093/nar/26.4.1107"><li><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneMark.hmm%3A+new+solutions+for+gene+finding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneMark.hmm%3A+new+solutions+for+gene+finding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000928-Wheeler1" id="pone.0000928-Wheeler1"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2005) Database resources of the National Center for Biotechnology Information. Nucleic Acids Research  33: D39–45.  <ul class="find" data-citedArticleID="995316" data-doi="10.1093/nar/gki062"><li><a href="http://dx.doi.org/10.1093/nar/gki062" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000928-Zdobnov1" id="pone.0000928-Zdobnov1"></a>Zdobnov EM, Apweiler R (2001) InterProScan - an integration platform for the signature-recognition methods in InterPro. Bioinformatics  17: 847–848.  <ul class="find" data-citedArticleID="995322" data-doi="10.1093/bioinformatics/17.9.847"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.847" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=InterProScan+-+an+integration+platform+for+the+signature-recognition+methods+in+InterPro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22InterProScan+-+an+integration+platform+for+the+signature-recognition+methods+in+InterPro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000928-Hawkins1" id="pone.0000928-Hawkins1"></a>Hawkins T, Luban S, Kihara D (2006) Enhanced automated function prediction using distantly related sequences and contextual association by PFP. Protein Science  15: 1550–1556.  <ul class="find" data-citedArticleID="995234" data-doi="10.1110/ps.062153506"><li><a href="http://dx.doi.org/10.1110/ps.062153506" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhanced+automated+function+prediction+using+distantly+related+sequences+and+contextual+association+by+PFP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhanced+automated+function+prediction+using+distantly+related+sequences+and+contextual+association+by+PFP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000928-Gardy1" id="pone.0000928-Gardy1"></a>Gardy JL, Laird MR, Chen F, Rey S, Walsh CJ, et al.  (2004) PSORTb v.2.0: Expanded prediction of bacterial protein subcellular localization and insights gained from comparative proteome analysis. Bioinformatics  21: 617–623.  <ul class="find" data-citedArticleID="995226" data-doi="10.1093/bioinformatics/bti057"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PSORTb+v.2.0%3A+Expanded+prediction+of+bacterial+protein+subcellular+localization+and+insights+gained+from+comparative+proteome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PSORTb+v.2.0%3A+Expanded+prediction+of+bacterial+protein+subcellular+localization+and+insights+gained+from+comparative+proteome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000928-Gasteiger1" id="pone.0000928-Gasteiger1"></a>Gasteiger E, Gattiker A, Hoogland C, Ivanyi I, Appel RD, et al.  (2003) ExPASy: The proteomics server for in-depth protein knowledge and analysis. Nucleic Acids Research  31: 3784–3788.  <ul class="find" data-citedArticleID="995228" data-doi="10.1093/nar/gkg563"><li><a href="http://dx.doi.org/10.1093/nar/gkg563" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ExPASy%3A+The+proteomics+server+for+in-depth+protein+knowledge+and+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ExPASy%3A+The+proteomics+server+for+in-depth+protein+knowledge+and+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000928-Dodd1" id="pone.0000928-Dodd1"></a>Dodd IB, Egan JB (1988) The prediction of helix-turn-helix DNA-binding regions in proteins. Protein Engineering  2: 174–175.  <ul class="find" data-citedArticleID="995214" data-doi="10.1093/protein/2.3.174"><li><a href="http://dx.doi.org/10.1093/protein/2.3.174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+prediction+of+helix-turn-helix+DNA-binding+regions+in+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+prediction+of+helix-turn-helix+DNA-binding+regions+in+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000928-Rawlings1" id="pone.0000928-Rawlings1"></a>Rawlings ND, Morton FR, Barrett AJ (2006) MEROPS: the peptidase database. Nucleic Acids Research  34: D270–D272.  <ul class="find" data-citedArticleID="995282" data-doi="10.1093/nar/gkj089"><li><a href="http://dx.doi.org/10.1093/nar/gkj089" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEROPS%3A+the+peptidase+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEROPS%3A+the+peptidase+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000928-Saier1" id="pone.0000928-Saier1"></a>Saier MH Jr., Tran CV, Barabote RD (2006) TCDB: the Transporter Classification Database for membrane transport protein analyses and information. Nucleic Acids Res  34: D181–186.  <ul class="find" data-citedArticleID="995288" data-doi="10.1093/nar/gkj001"><li><a href="http://dx.doi.org/10.1093/nar/gkj001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TCDB%3A+the+Transporter+Classification+Database+for+membrane+transport+protein+analyses+and+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TCDB%3A+the+Transporter+Classification+Database+for+membrane+transport+protein+analyses+and+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000928-Keijser1" id="pone.0000928-Keijser1"></a>Keijser BJ, Ter Beek A, Rauwerda H, Schuren F, Montijn R, et al.  (2007) Analysis of temporal gene expression during <em>Bacillus subtilis</em> spore germination and outgrowth. Journal of Bacteriology  189: 3624–3634.  <ul class="find" data-citedArticleID="995240" data-doi="10.1128/jb.01736-06"><li><a href="http://dx.doi.org/10.1128/jb.01736-06" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+temporal+gene+expression+during+Bacillus+subtilis+spore+germination+and+outgrowth." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+temporal+gene+expression+during+Bacillus+subtilis+spore+germination+and+outgrowth.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000928-Moeller1" id="pone.0000928-Moeller1"></a>Moeller R, Stackebrandt E, Douki T, Cadet J, Rettberg P, et al.  (2007) DNA bipyrimidine photoproduct repair and transcriptional response of UV-C irradiated <em>Bacillus subtilis</em>. Archives of Microbiology.   <ul class="find" data-citedArticleID="995262" data-doi="10.1007/s00203-007-0263-4"><li><a href="http://dx.doi.org/10.1007/s00203-007-0263-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+bipyrimidine+photoproduct+repair+and+transcriptional+response+of+UV-C+irradiated+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+bipyrimidine+photoproduct+repair+and+transcriptional+response+of+UV-C+irradiated+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000928-Nicholson1" id="pone.0000928-Nicholson1"></a>Nicholson WL, Munakata N, Horneck G, Melosh HJ, Setlow P (2000) Resistance of <em>Bacillus</em> endospores to extreme terrestrial and extraterrestrial environments. Microbiology and Molecular Biology Reviews  64: 548–572.  <ul class="find" data-citedArticleID="995274" data-doi="10.1128/mmbr.64.3.548-572.2000"><li><a href="http://dx.doi.org/10.1128/mmbr.64.3.548-572.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resistance+of+Bacillus+endospores+to+extreme+terrestrial+and+extraterrestrial+environments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resistance+of+Bacillus+endospores+to+extreme+terrestrial+and+extraterrestrial+environments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000928-Setlow3" id="pone.0000928-Setlow3"></a>Setlow P (1995) Mechanisms for the prevention of damage to DNA in spores of Bacillus species. Annu Rev Microbiol  49: 29–54.  <ul class="find" data-citedArticleID="995300" data-doi="10.1146/annurev.mi.49.100195.000333"><li><a href="http://dx.doi.org/10.1146/annurev.mi.49.100195.000333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+for+the+prevention+of+damage+to+DNA+in+spores+of+Bacillus+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+for+the+prevention+of+damage+to+DNA+in+spores+of+Bacillus+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000928-Bagyan1" id="pone.0000928-Bagyan1"></a>Bagyan I, Setlow B, Setlow P (1998) New small, acid-soluble proteins unique to spores of <em>Bacillus subtilis</em>: identification of the coding genes and regulation and function of two of these genes. Journal of Bacteriology  180: 6704–6712.  <ul class="find" data-citedArticleID="995192"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=New+small%2C+acid-soluble+proteins+unique+to+spores+of+Bacillus+subtilis%3A+identification+of+the+coding+genes+and+regulation+and+function+of+two+of+these+genes.&amp;auth=&amp;atitle=New+small%2C+acid-soluble+proteins+unique+to+spores+of+Bacillus+subtilis%3A+identification+of+the+coding+genes+and+regulation+and+function+of+two+of+these+genes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=New+small%2C+acid-soluble+proteins+unique+to+spores+of+Bacillus+subtilis%3A+identification+of+the+coding+genes+and+regulation+and+function+of+two+of+these+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22New+small%2C+acid-soluble+proteins+unique+to+spores+of+Bacillus+subtilis%3A+identification+of+the+coding+genes+and+regulation+and+function+of+two+of+these+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000928-TovarRojo1" id="pone.0000928-TovarRojo1"></a>Tovar-Rojo F, Setlow P (1991) Effects of mutant small, acid-soluble spore proteins from <em>Bacillus subtilis</em> on DNA <em>in vivo</em> and <em>in vitro</em>. Journal of Bacteriology  173: 4827–4835.  <ul class="find" data-citedArticleID="995306"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+mutant+small%2C+acid-soluble+spore+proteins+from+Bacillus+subtilis+on+DNA+in+vivo+and+in+vitro.&amp;auth=&amp;atitle=Effects+of+mutant+small%2C+acid-soluble+spore+proteins+from+Bacillus+subtilis+on+DNA+in+vivo+and+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+mutant+small%2C+acid-soluble+spore+proteins+from+Bacillus+subtilis+on+DNA+in+vivo+and+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+mutant+small%2C+acid-soluble+spore+proteins+from+Bacillus+subtilis+on+DNA+in+vivo+and+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000928-CabreraHernandez1" id="pone.0000928-CabreraHernandez1"></a>Cabrera-Hernandez A, Sanchez-Salas JL, Paidhungat M, Setlow P (1999) Regulation of four genes encoding small, acid-soluble spore proteins in <em>Bacillus subtilis</em>. Gene  232: 1–10.  <ul class="find" data-citedArticleID="995200" data-doi="10.1016/s0378-1119(99)00124-9"><li><a href="http://dx.doi.org/10.1016/s0378-1119(99)00124-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+four+genes+encoding+small%2C+acid-soluble+spore+proteins+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+four+genes+encoding+small%2C+acid-soluble+spore+proteins+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000928-CarrilloMartinez1" id="pone.0000928-CarrilloMartinez1"></a>Carrillo-Martinez Y, Setlow P (1994) Properties of <em>Bacillus subtilis</em> small, acid-soluble spore proteins with changes in the sequence recognized by their specific protease. Journal of Bacteriology  176: 5357–5363.  <ul class="find" data-citedArticleID="995204"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Properties+of+Bacillus+subtilis+small%2C+acid-soluble+spore+proteins+with+changes+in+the+sequence+recognized+by+their+specific+protease.&amp;auth=&amp;atitle=Properties+of+Bacillus+subtilis+small%2C+acid-soluble+spore+proteins+with+changes+in+the+sequence+recognized+by+their+specific+protease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properties+of+Bacillus+subtilis+small%2C+acid-soluble+spore+proteins+with+changes+in+the+sequence+recognized+by+their+specific+protease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properties+of+Bacillus+subtilis+small%2C+acid-soluble+spore+proteins+with+changes+in+the+sequence+recognized+by+their+specific+protease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000928-Eisen1" id="pone.0000928-Eisen1"></a>Eisen JA, Hanawalt PC (1999) A phylogenomic study of DNA repair genes, proteins, and processes. Mutation Research  435: 171–213.  <ul class="find" data-citedArticleID="995218" data-doi="10.1016/s0921-8777(99)00050-6"><li><a href="http://dx.doi.org/10.1016/s0921-8777(99)00050-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenomic+study+of+DNA+repair+genes%2C+proteins%2C+and+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenomic+study+of+DNA+repair+genes%2C+proteins%2C+and+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000928-MartinsPinheiro1" id="pone.0000928-MartinsPinheiro1"></a>Martins-Pinheiro M, Marques RC, Menck CF (2007) Genome analysis of DNA repair genes in the alpha proteobacterium <em>Caulobacter crescentus</em>. BMC Microbiology  7: 17.  <ul class="find" data-citedArticleID="995254" data-doi="10.1186/1471-2180-7-17"><li><a href="http://dx.doi.org/10.1186/1471-2180-7-17" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+analysis+of+DNA+repair+genes+in+the+alpha+proteobacterium+Caulobacter+crescentus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+analysis+of+DNA+repair+genes+in+the+alpha+proteobacterium+Caulobacter+crescentus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000928-SalasPacheco1" id="pone.0000928-SalasPacheco1"></a>Salas-Pacheco JM, Setlow B, Setlow P, Pedraza-Reyes M (2005) Role of the Nfo (YqfS) and ExoA apurinic/apyrimidinic endonucleases in protecting <em>Bacillus subtilis</em> spores from DNA damage. Journal of Bacteriology  187: 7374–7381.  <ul class="find" data-citedArticleID="995290" data-doi="10.1128/jb.187.21.7374-7381.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.21.7374-7381.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+Nfo+%28YqfS%29+and+ExoA+apurinic%2Fapyrimidinic+endonucleases+in+protecting+Bacillus+subtilis+spores+from+DNA+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+Nfo+%28YqfS%29+and+ExoA+apurinic%2Fapyrimidinic+endonucleases+in+protecting+Bacillus+subtilis+spores+from+DNA+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000928-Aamodt1" id="pone.0000928-Aamodt1"></a>Aamodt RM, Falnes PO, Johansen RF, Seeberg E, Bjoras M (2004) The <em>Bacillus subtilis</em> counterpart of the mammalian 3-methyladenine DNA glycosylase has hypoxanthine and 1,N6-ethenoadenine as preferred substrates. Journal of Biological Chemistry  279: 13601–13606.  <ul class="find" data-citedArticleID="995186" data-doi="10.1074/jbc.m314277200"><li><a href="http://dx.doi.org/10.1074/jbc.m314277200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bacillus+subtilis+counterpart+of+the+mammalian+3-methyladenine+DNA+glycosylase+has+hypoxanthine+and+1%2CN6-ethenoadenine+as+preferred+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bacillus+subtilis+counterpart+of+the+mammalian+3-methyladenine+DNA+glycosylase+has+hypoxanthine+and+1%2CN6-ethenoadenine+as+preferred+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000928-Truglio1" id="pone.0000928-Truglio1"></a>Truglio JJ, Croteau DL, Van Houten B, Kisker C (2006) Prokaryotic nucleotide excision repair: the UvrABC system. Chemical Reviews  106: 233–252.  <ul class="find" data-citedArticleID="995308" data-doi="10.1021/cr040471u"><li><a href="http://dx.doi.org/10.1021/cr040471u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prokaryotic+nucleotide+excision+repair%3A+the+UvrABC+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prokaryotic+nucleotide+excision+repair%3A+the+UvrABC+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000928-Petit1" id="pone.0000928-Petit1"></a>Petit MA, Dervyn E, Rose M, Entian KD, McGovern S, et al.  (1998) PcrA is an essential DNA helicase of <em>Bacillus subtilis</em> fulfilling functions both in repair and rolling-circle replication. Molecular Microbiology  29: 261–273.  <ul class="find" data-citedArticleID="995278" data-doi="10.1046/j.1365-2958.1998.00927.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00927.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PcrA+is+an+essential+DNA+helicase+of+Bacillus+subtilis+fulfilling+functions+both+in+repair+and+rolling-circle+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PcrA+is+an+essential+DNA+helicase+of+Bacillus+subtilis+fulfilling+functions+both+in+repair+and+rolling-circle+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000928-PedrazaReyes1" id="pone.0000928-PedrazaReyes1"></a>Pedraza-Reyes M, Yasbin RE (2004) Contribution of the mismatch DNA repair system to the generation of stationary-phase-induced mutants of <em>Bacillus subtilis</em>. Journal of Bacteriology  186: 6485–6491.  <ul class="find" data-citedArticleID="995276" data-doi="10.1128/jb.186.19.6485-6491.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.19.6485-6491.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contribution+of+the+mismatch+DNA+repair+system+to+the+generation+of+stationary-phase-induced+mutants+of+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contribution+of+the+mismatch+DNA+repair+system+to+the+generation+of+stationary-phase-induced+mutants+of+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000928-Smith1" id="pone.0000928-Smith1"></a>Smith BT, Grossman AD, Walker GC (2001) Visualization of mismatch repair in bacterial cells. Molecular Cell  8: 1197–1206.  <ul class="find" data-citedArticleID="995302" data-doi="10.1016/s1097-2765(01)00402-6"><li><a href="http://dx.doi.org/10.1016/s1097-2765(01)00402-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Visualization+of+mismatch+repair+in+bacterial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Visualization+of+mismatch+repair+in+bacterial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000928-Moore1" id="pone.0000928-Moore1"></a>Moore JK, Haber JE (1996) Cell cycle and genetic requirements of two pathways of nonhomologous end-joining repair of double-strand breaks in <em>Saccharomyces cerevisiae</em>. Molecular and Cellular Biology  16: 2164–2173.  <ul class="find" data-citedArticleID="995266"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+cycle+and+genetic+requirements+of+two+pathways+of+nonhomologous+end-joining+repair+of+double-strand+breaks+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Cell+cycle+and+genetic+requirements+of+two+pathways+of+nonhomologous+end-joining+repair+of+double-strand+breaks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle+and+genetic+requirements+of+two+pathways+of+nonhomologous+end-joining+repair+of+double-strand+breaks+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle+and+genetic+requirements+of+two+pathways+of+nonhomologous+end-joining+repair+of+double-strand+breaks+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000928-Weller1" id="pone.0000928-Weller1"></a>Weller GR, Kysela B, Roy R, Tonkin LM, Scanlan E, et al.  (2002) Identification of a DNA nonhomologous end-joining complex in bacteria. Science  297: 1686–1689.  <ul class="find" data-citedArticleID="995314" data-doi="10.1126/science.1074584"><li><a href="http://dx.doi.org/10.1126/science.1074584" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+DNA+nonhomologous+end-joining+complex+in+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+DNA+nonhomologous+end-joining+complex+in+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000928-Wang1" id="pone.0000928-Wang1"></a>Wang ST, Setlow B, Conlon EM, Lyon JL, Imamura D, et al.  (2006) The forespore line of gene expression in <em>Bacillus subtilis</em>. Journal of Molecular Biology  358: 16–37.  <ul class="find" data-citedArticleID="995312" data-doi="10.1016/j.jmb.2006.01.059"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.01.059" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+forespore+line+of+gene+expression+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+forespore+line+of+gene+expression+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000928-Moeller2" id="pone.0000928-Moeller2"></a>Moeller R, Stackebrandt E, Reitz G, Berger T, Rettberg P, et al.  (2007) Role of DNA repair by non-homologous end joining (NHEJ) in <em>Bacillus subtilis</em> spore resistance to extreme dryness, mono- and polychromatic UV and ionizing radiation. Journal of Bacteriology.   <ul class="find" data-citedArticleID="995264" data-doi="10.1128/jb.00018-07"><li><a href="http://dx.doi.org/10.1128/jb.00018-07" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+DNA+repair+by+non-homologous+end+joining+%28NHEJ%29+in+Bacillus+subtilis+spore+resistance+to+extreme+dryness%2C+mono-+and+polychromatic+UV+and+ionizing+radiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+DNA+repair+by+non-homologous+end+joining+%28NHEJ%29+in+Bacillus+subtilis+spore+resistance+to+extreme+dryness%2C+mono-+and+polychromatic+UV+and+ionizing+radiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000928-Kooistra1" id="pone.0000928-Kooistra1"></a>Kooistra J, Haijema BJ, Venema G (1993) The <em>Bacillus subtilis addAB</em> genes are fully functional in <em>Escherichia coli</em>. Molecular Microbiology  7: 915–923.  <ul class="find" data-citedArticleID="995244" data-doi="10.1111/j.1365-2958.1993.tb01182.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1993.tb01182.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Bacillus+subtilis+addAB+genes+are+fully+functional+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Bacillus+subtilis+addAB+genes+are+fully+functional+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000928-Yasbin1" id="pone.0000928-Yasbin1"></a>Yasbin RE, Cheo DL, Bayles KW (1992) Inducible DNA repair and differentiation in <em>Bacillus subtilis</em>: interactions between global regulons. Molecular Microbiology  6: 1263–1270.  <ul class="find" data-citedArticleID="995318" data-doi="10.1111/j.1365-2958.1992.tb00847.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.1992.tb00847.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inducible+DNA+repair+and+differentiation+in+Bacillus+subtilis%3A+interactions+between+global+regulons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inducible+DNA+repair+and+differentiation+in+Bacillus+subtilis%3A+interactions+between+global+regulons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000928-Au1" id="pone.0000928-Au1"></a>Au N, Kuester-Schoeck E, Mandava V, Bothwell LE, Canny SP, et al.  (2005) Genetic composition of the <em>Bacillus subtilis</em> SOS system. Journal of Bacteriology  187: 7655–7666.  <ul class="find" data-citedArticleID="995190" data-doi="10.1128/jb.187.22.7655-7666.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.22.7655-7666.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+composition+of+the+Bacillus+subtilis+SOS+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+composition+of+the+Bacillus+subtilis+SOS+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000928-Groban1" id="pone.0000928-Groban1"></a>Groban ES, Johnson MB, Banky P, Burnett PG, Calderon GL, et al.  (2005) Binding of the <em>Bacillus subtilis</em> LexA protein to the SOS operator. Nucleic Acids Research  33: 6287–6295.  <ul class="find" data-citedArticleID="995230" data-doi="10.1093/nar/gki939"><li><a href="http://dx.doi.org/10.1093/nar/gki939" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Binding+of+the+Bacillus+subtilis+LexA+protein+to+the+SOS+operator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Binding+of+the+Bacillus+subtilis+LexA+protein+to+the+SOS+operator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000928-Lin1" id="pone.0000928-Lin1"></a>Lin J, Chen ZZ, Tian B, Hua YJ (2007) Evolutionary pathways of an ancient gene <em>recX</em>. Gene  387: 15–20.  <ul class="find" data-citedArticleID="995246" data-doi="10.1016/j.gene.2006.07.031"><li><a href="http://dx.doi.org/10.1016/j.gene.2006.07.031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+pathways+of+an+ancient+gene+recX." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+pathways+of+an+ancient+gene+recX.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000928-Kanno1" id="pone.0000928-Kanno1"></a>Kanno S, Iwai S, Takao M, Yasui A (1999) Repair of apurinic/apyrimidinic sites by UV damage endonuclease; a repair protein for UV and oxidative damage. Nucleic Acids Research  27: 3096–3103.  <ul class="find" data-citedArticleID="995238" data-doi="10.1093/nar/27.15.3096"><li><a href="http://dx.doi.org/10.1093/nar/27.15.3096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repair+of+apurinic%2Fapyrimidinic+sites+by+UV+damage+endonuclease%3B+a+repair+protein+for+UV+and+oxidative+damage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repair+of+apurinic%2Fapyrimidinic+sites+by+UV+damage+endonuclease%3B+a+repair+protein+for+UV+and+oxidative+damage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000928-Duigou1" id="pone.0000928-Duigou1"></a>Duigou S, Ehrlich SD, Noirot P, Noirot-Gros MF (2004) Distinctive genetic features exhibited by the Y-family DNA polymerases in <em>Bacillus subtilis</em>. Molecular Microbiology  54: 439–451.  <ul class="find" data-citedArticleID="995216" data-doi="10.1111/j.1365-2958.2004.04259.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04259.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinctive+genetic+features+exhibited+by+the+Y-family+DNA+polymerases+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinctive+genetic+features+exhibited+by+the+Y-family+DNA+polymerases+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000928-Sassanfar1" id="pone.0000928-Sassanfar1"></a>Sassanfar M, Dosanjh MK, Essigmann JM, Samson L (1991) Relative efficiencies of the bacterial, yeast, and human DNA methyltransferases for the repair of O6-methylguanine and O4-methylthymine. Suggestive evidence for O4-methylthymine repair by eukaryotic methyltransferases. Journal of Biological Chemistry  266: 2767–2771.  <ul class="find" data-citedArticleID="995294"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Relative+efficiencies+of+the+bacterial%2C+yeast%2C+and+human+DNA+methyltransferases+for+the+repair+of+O6-methylguanine+and+O4-methylthymine.+Suggestive+evidence+for+O4-methylthymine+repair+by+eukaryotic+methyltransferases.&amp;auth=&amp;atitle=Relative+efficiencies+of+the+bacterial%2C+yeast%2C+and+human+DNA+methyltransferases+for+the+repair+of+O6-methylguanine+and+O4-methylthymine.+Suggestive+evidence+for+O4-methylthymine+repair+by+eukaryotic+methyltransferases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relative+efficiencies+of+the+bacterial%2C+yeast%2C+and+human+DNA+methyltransferases+for+the+repair+of+O6-methylguanine+and+O4-methylthymine.+Suggestive+evidence+for+O4-methylthymine+repair+by+eukaryotic+methyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relative+efficiencies+of+the+bacterial%2C+yeast%2C+and+human+DNA+methyltransferases+for+the+repair+of+O6-methylguanine+and+O4-methylthymine.+Suggestive+evidence+for+O4-methylthymine+repair+by+eukaryotic+methyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000928-Rohankhedkar1" id="pone.0000928-Rohankhedkar1"></a>Rohankhedkar MS, Mulrooney SB, Wedemeyer WJ, Hausinger RP (2006) The AidB component of the <em>Escherichia coli</em> adaptive response to alkylating agents is a flavin-containing, DNA-binding protein. Journal of Bacteriology  188: 223–230.  <ul class="find" data-citedArticleID="995286" data-doi="10.1128/jb.188.1.223-230.2006"><li><a href="http://dx.doi.org/10.1128/jb.188.1.223-230.2006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+AidB+component+of+the+Escherichia+coli+adaptive+response+to+alkylating+agents+is+a+flavin-containing%2C+DNA-binding+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+AidB+component+of+the+Escherichia+coli+adaptive+response+to+alkylating+agents+is+a+flavin-containing%2C+DNA-binding+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000928-Morohoshi1" id="pone.0000928-Morohoshi1"></a>Morohoshi F, Hayashi K, Munakata N (1990) <em>Bacillus subtilis ada</em> operon encodes two DNA alkyltransferases. Nucleic Acids Research  18: 5473–5480.  <ul class="find" data-citedArticleID="995268" data-doi="10.1093/nar/18.18.5473"><li><a href="http://dx.doi.org/10.1093/nar/18.18.5473" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+subtilis+ada+operon+encodes+two+DNA+alkyltransferases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+subtilis+ada+operon+encodes+two+DNA+alkyltransferases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000928-Voloshin1" id="pone.0000928-Voloshin1"></a>Voloshin ON, Vanevski F, Khil PP, Camerini-Otero RD (2003) Characterization of the DNA damage-inducible helicase DinG from <em>Escherichia coli</em>. Journal of Biological Chemistry  278: 28284–28293.  <ul class="find" data-citedArticleID="995310" data-doi="10.1074/jbc.m301188200"><li><a href="http://dx.doi.org/10.1074/jbc.m301188200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+the+DNA+damage-inducible+helicase+DinG+from+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+the+DNA+damage-inducible+helicase+DinG+from+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000928-Yasuda1" id="pone.0000928-Yasuda1"></a>Yasuda T, Morimatsu K, Horii T, Nagata T, Ohmori H (1998) Inhibition of <em>Escherichia coli</em> RecA coprotease activities by DinI. The EMBO Journal  17: 3207–3216.  <ul class="find" data-citedArticleID="995320" data-doi="10.1093/emboj/17.11.3207"><li><a href="http://dx.doi.org/10.1093/emboj/17.11.3207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+Escherichia+coli+RecA+coprotease+activities+by+DinI." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+Escherichia+coli+RecA+coprotease+activities+by+DinI.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000928-McLennan1" id="pone.0000928-McLennan1"></a>McLennan AG (2006) The Nudix hydrolase superfamily. Cellular and Molecular Life Sciences  63: 123–143.  <ul class="find" data-citedArticleID="995258" data-doi="10.1007/s00018-005-5386-7"><li><a href="http://dx.doi.org/10.1007/s00018-005-5386-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Nudix+hydrolase+superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Nudix+hydrolase+superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000928-Essen1" id="pone.0000928-Essen1"></a>Essen LO, Klar T (2006) Light-driven DNA repair by photolyases. Cellular and Molecular Life Sciences  63: 1266–1277.  <ul class="find" data-citedArticleID="995220" data-doi="10.1007/s00018-005-5447-y"><li><a href="http://dx.doi.org/10.1007/s00018-005-5447-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Light-driven+DNA+repair+by+photolyases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Light-driven+DNA+repair+by+photolyases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000928-Caruthers1" id="pone.0000928-Caruthers1"></a>Caruthers JM, McKay DB (2002) Helicase structure and mechanism. Current Opinion in Structural Biology  12: 123–133.  <ul class="find" data-citedArticleID="995206" data-doi="10.1016/s0959-440x(02)00298-1"><li><a href="http://dx.doi.org/10.1016/s0959-440x(02)00298-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Helicase+structure+and+mechanism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Helicase+structure+and+mechanism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000928-Brosh1" id="pone.0000928-Brosh1"></a>Brosh RM Jr., Matson SW (1997) A point mutation in <em>Escherichia coli</em> DNA helicase II renders the enzyme nonfunctional in two DNA repair pathways. Evidence for initiation of unwinding from a nick <em>in vivo</em>. Journal of Biological Chemistry  272: 572–579.  <ul class="find" data-citedArticleID="995198" data-doi="10.1074/jbc.272.1.572"><li><a href="http://dx.doi.org/10.1074/jbc.272.1.572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+point+mutation+in+Escherichia+coli+DNA+helicase+II+renders+the+enzyme+nonfunctional+in+two+DNA+repair+pathways.+Evidence+for+initiation+of+unwinding+from+a+nick+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+point+mutation+in+Escherichia+coli+DNA+helicase+II+renders+the+enzyme+nonfunctional+in+two+DNA+repair+pathways.+Evidence+for+initiation+of+unwinding+from+a+nick+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000928-Mostertz1" id="pone.0000928-Mostertz1"></a>Mostertz J, Scharf C, Hecker M, Homuth G (2004) Transcriptome and proteome analysis of <em>Bacillus subtilis</em> gene expression in response to superoxide and peroxide stress. Microbiology  150: 497–512.  <ul class="find" data-citedArticleID="995270" data-doi="10.1099/mic.0.26665-0"><li><a href="http://dx.doi.org/10.1099/mic.0.26665-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+and+proteome+analysis+of+Bacillus+subtilis+gene+expression+in+response+to+superoxide+and+peroxide+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+and+proteome+analysis+of+Bacillus+subtilis+gene+expression+in+response+to+superoxide+and+peroxide+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000928-Bagyan2" id="pone.0000928-Bagyan2"></a>Bagyan I, Casillas-Martinez L, Setlow P (1998) The <em>katX</em> gene, which codes for the catalase in spores of <em>Bacillus subtilis</em>, is a forespore-specific gene controlled by sigmaF, and KatX is essential for hydrogen peroxide resistance of the germinating spore. Journal of Bacteriology  180: 2057–2062.  <ul class="find" data-citedArticleID="995194"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+katX+gene%2C+which+codes+for+the+catalase+in+spores+of+Bacillus+subtilis%2C+is+a+forespore-specific+gene+controlled+by+sigmaF%2C+and+KatX+is+essential+for+hydrogen+peroxide+resistance+of+the+germinating+spore.&amp;auth=&amp;atitle=The+katX+gene%2C+which+codes+for+the+catalase+in+spores+of+Bacillus+subtilis%2C+is+a+forespore-specific+gene+controlled+by+sigmaF%2C+and+KatX+is+essential+for+hydrogen+peroxide+resistance+of+the+germinating+spore." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+katX+gene%2C+which+codes+for+the+catalase+in+spores+of+Bacillus+subtilis%2C+is+a+forespore-specific+gene+controlled+by+sigmaF%2C+and+KatX+is+essential+for+hydrogen+peroxide+resistance+of+the+germinating+spore." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+katX+gene%2C+which+codes+for+the+catalase+in+spores+of+Bacillus+subtilis%2C+is+a+forespore-specific+gene+controlled+by+sigmaF%2C+and+KatX+is+essential+for+hydrogen+peroxide+resistance+of+the+germinating+spore.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000928-Rhee1" id="pone.0000928-Rhee1"></a>Rhee SG, Chae HZ, Kim K (2005) Peroxiredoxins: a historical overview and speculative preview of novel mechanisms and emerging concepts in cell signaling. Free Radical Biology &amp; Medicine  38: 1543–1552.  <ul class="find" data-citedArticleID="995284" data-doi="10.1016/j.freeradbiomed.2005.02.026"><li><a href="http://dx.doi.org/10.1016/j.freeradbiomed.2005.02.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Peroxiredoxins%3A+a+historical+overview+and+speculative+preview+of+novel+mechanisms+and+emerging+concepts+in+cell+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Peroxiredoxins%3A+a+historical+overview+and+speculative+preview+of+novel+mechanisms+and+emerging+concepts+in+cell+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000928-Zeller1" id="pone.0000928-Zeller1"></a>Zeller T, Klug G (2006) Thioredoxins in bacteria: functions in oxidative stress response and regulation of thioredoxin genes. Naturwissenschaften  93: 259–266.  <ul class="find" data-citedArticleID="995324" data-doi="10.1007/s00114-006-0106-1"><li><a href="http://dx.doi.org/10.1007/s00114-006-0106-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Thioredoxins+in+bacteria%3A+functions+in+oxidative+stress+response+and+regulation+of+thioredoxin+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Thioredoxins+in+bacteria%3A+functions+in+oxidative+stress+response+and+regulation+of+thioredoxin+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000928-Smits1" id="pone.0000928-Smits1"></a>Smits WK, Dubois JY, Bron S, van Dijl JM, Kuipers OP (2005) Tricksy business: transcriptome analysis reveals the involvement of thioredoxin A in redox homeostasis, oxidative stress, sulfur metabolism, and cellular differentiation in <em>Bacillus subtilis</em>. Journal of Bacteriology  187: 3921–3930.  <ul class="find" data-citedArticleID="995304" data-doi="10.1128/jb.187.12.3921-3930.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.12.3921-3930.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tricksy+business%3A+transcriptome+analysis+reveals+the+involvement+of+thioredoxin+A+in+redox+homeostasis%2C+oxidative+stress%2C+sulfur+metabolism%2C+and+cellular+differentiation+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tricksy+business%3A+transcriptome+analysis+reveals+the+involvement+of+thioredoxin+A+in+redox+homeostasis%2C+oxidative+stress%2C+sulfur+metabolism%2C+and+cellular+differentiation+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000928-Fuangthong1" id="pone.0000928-Fuangthong1"></a>Fuangthong M, Atichartpongkul S, Mongkolsuk S, Helmann JD (2001) OhrR is a repressor of <em>ohrA</em>, a key organic hydroperoxide resistance determinant in <em>Bacillus subtilis</em>. Journal of Bacteriology  183: 4134–4141.  <ul class="find" data-citedArticleID="995224" data-doi="10.1128/jb.183.14.4134-4141.2001"><li><a href="http://dx.doi.org/10.1128/jb.183.14.4134-4141.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OhrR+is+a+repressor+of+ohrA%2C+a+key+organic+hydroperoxide+resistance+determinant+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OhrR+is+a+repressor+of+ohrA%2C+a+key+organic+hydroperoxide+resistance+determinant+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000928-Liu1" id="pone.0000928-Liu1"></a>Liu X, Kim K, Leighton T, Theil EC (2006) Paired <em>Bacillus anthracis</em> Dps (mini-ferritin) have different reactivities with peroxide. Journal of Biological Chemistry  281: 27827–27835.  <ul class="find" data-citedArticleID="995248" data-doi="10.1074/jbc.m601398200"><li><a href="http://dx.doi.org/10.1074/jbc.m601398200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Paired+Bacillus+anthracis+Dps+%28mini-ferritin%29+have+different+reactivities+with+peroxide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Paired+Bacillus+anthracis+Dps+%28mini-ferritin%29+have+different+reactivities+with+peroxide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000928-CasillasMartinez1" id="pone.0000928-CasillasMartinez1"></a>Casillas-Martinez L, Setlow P (1997) Alkyl hydroperoxide reductase, catalase, MrgA, and superoxide dismutase are not involved in resistance of <em>Bacillus subtilis</em> spores to heat or oxidizing agents. Journal of Bacteriology  179: 7420–7425.  <ul class="find" data-citedArticleID="995208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alkyl+hydroperoxide+reductase%2C+catalase%2C+MrgA%2C+and+superoxide+dismutase+are+not+involved+in+resistance+of+Bacillus+subtilis+spores+to+heat+or+oxidizing+agents.&amp;auth=&amp;atitle=Alkyl+hydroperoxide+reductase%2C+catalase%2C+MrgA%2C+and+superoxide+dismutase+are+not+involved+in+resistance+of+Bacillus+subtilis+spores+to+heat+or+oxidizing+agents." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alkyl+hydroperoxide+reductase%2C+catalase%2C+MrgA%2C+and+superoxide+dismutase+are+not+involved+in+resistance+of+Bacillus+subtilis+spores+to+heat+or+oxidizing+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alkyl+hydroperoxide+reductase%2C+catalase%2C+MrgA%2C+and+superoxide+dismutase+are+not+involved+in+resistance+of+Bacillus+subtilis+spores+to+heat+or+oxidizing+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000928-Cao1" id="pone.0000928-Cao1"></a>Cao M, Moore CM, Helmann JD (2005) <em>Bacillus subtilis</em> paraquat resistance is directed by sigmaM, an extracytoplasmic function sigma factor, and is conferred by YqjL and BcrC. Journal of Bacteriology  187: 2948–2956.  <ul class="find" data-citedArticleID="995202" data-doi="10.1128/jb.187.9.2948-2956.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.9.2948-2956.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+subtilis+paraquat+resistance+is+directed+by+sigmaM%2C+an+extracytoplasmic+function+sigma+factor%2C+and+is+conferred+by+YqjL+and+BcrC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+subtilis+paraquat+resistance+is+directed+by+sigmaM%2C+an+extracytoplasmic+function+sigma+factor%2C+and+is+conferred+by+YqjL+and+BcrC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000928-Matsumoto1" id="pone.0000928-Matsumoto1"></a>Matsumoto T, Nakanishi K, Asai K, Sadaie Y (2005) Transcriptional analysis of the <em>ylaABCD</em> operon of <em>Bacillus subtilis</em> encoding a sigma factor of extracytoplasmic function family. Genes &amp; Genetic Systems  80: 385–393.  <ul class="find" data-citedArticleID="995256" data-doi="10.1266/ggs.80.385"><li><a href="http://dx.doi.org/10.1266/ggs.80.385" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+analysis+of+the+ylaABCD+operon+of+Bacillus+subtilis+encoding+a+sigma+factor+of+extracytoplasmic+function+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+analysis+of+the+ylaABCD+operon+of+Bacillus+subtilis+encoding+a+sigma+factor+of+extracytoplasmic+function+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000928-Jedrzejas1" id="pone.0000928-Jedrzejas1"></a>Jedrzejas MJ, Huang WJ (2003) <em>Bacillus</em> species proteins involved in spore formation and degradation: from identification in the genome, to sequence analysis, and determination of function and structure. Critical Reviews in Biochemistry and Molecular Biology  38: 173–198.  <ul class="find" data-citedArticleID="995236" data-doi="10.1080/713609234"><li><a href="http://dx.doi.org/10.1080/713609234" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacillus+species+proteins+involved+in+spore+formation+and+degradation%3A+from+identification+in+the+genome%2C+to+sequence+analysis%2C+and+determination+of+function+and+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacillus+species+proteins+involved+in+spore+formation+and+degradation%3A+from+identification+in+the+genome%2C+to+sequence+analysis%2C+and+determination+of+function+and+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.XML" value="160384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.PDF" value="191056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.g001.PNG_L" value="460197"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.g001.PNG_M" value="60056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.g001.PNG_S" value="10089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.g001.TIF" value="1123916"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.g001.PNG_I" value="27824"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t001.PNG_L" value="247953"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t001.PNG_M" value="82501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t001.PNG_S" value="10469"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t001.TIF" value="1056362"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t001.PNG_I" value="24598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t002.PNG_L" value="215659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t002.PNG_M" value="57944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t002.PNG_S" value="22299"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t002.TIF" value="1387854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t002.PNG_I" value="135800"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t003.PNG_L" value="128537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t003.PNG_M" value="78135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t003.PNG_S" value="17095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t003.TIF" value="761602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t003.PNG_I" value="68605"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t004.PNG_L" value="135497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t004.PNG_M" value="164180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t004.PNG_S" value="18571"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t004.TIF" value="602668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.t004.PNG_I" value="62724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000928.s001.DOC" value="91136"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000928&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000928" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000928&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000928&volume=&issue=&title=Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032&author_name=Jason%20Gioia%2C%20Shailaja%20Yerrapragada%2C%20Xiang%20Qin%2C%20Huaiyang%20Jiang%2C%20Okezie%20C.%20Igboeli%2C%20Donna%20Muzny%2C%20Shannon%20Dugan-Rocha%2C%20Yan%20Ding%2C%20Alicia%20Hawes%2C%20Wen%20Liu%2C%20Lesette%20Perez%2C%20Christie%20Kovar%2C%20Huyen%20Dinh%2C%20Sandra%20Lee%2C%20Lynne%20Nazareth%2C%20Peter%20Blyth%2C%20Michael%20Holder%2C%20Christian%20Buhay%2C%20Madhan%20R.%20Tirumalai%2C%20Yamei%20Liu%2C%20Indrani%20Dasgupta%2C%20Lina%20Bokhetache%2C%20Masaya%20Fujita%2C%20Fathi%20Karouia%2C%20Prahathees%20Eswara%20Moorthy%2C%20Johnathan%20Siefert%2C%20Akif%20Uzman%2C%20Prince%20Buzumbo%2C%20Avani%20Verma%2C%20Hiba%20Zwiya%2C%20Brian%20D.%20McWilliams%2C%20Adeola%20Olowu%2C%20Kenneth%20D.%20Clinkenbeard%2C%20David%20Newcombe%2C%20Lisa%20Golebiewski%2C%20Joseph%20F.%20Petrosino%2C%20Wayne%20L.%20Nicholson%2C%20George%20E.%20Fox%2C%20Kasthuri%20Venkateswaran%2C%20Sarah%20K.%20Highlander%2C%20George%20M.%20Weinstock&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000928" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928&amp;t=Paradoxical%20DNA%20Repair%20and%20Peroxide%20Resistance%20Gene%20Conservation%20in%20Bacillus%20pumilus%20SAFR-032" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928&title=Paradoxical%20DNA%20Repair%20and%20Peroxide%20Resistance%20Gene%20Conservation%20in%20Bacillus%20pumilus%20SAFR-032&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928&amp;title=Paradoxical%20DNA%20Repair%20and%20Peroxide%20Resistance%20Gene%20Conservation%20in%20Bacillus%20pumilus%20SAFR-032" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000928&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Paradoxical DNA Repair and Peroxide Resistance Gene Conservation in Bacillus pumilus SAFR-032';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000928';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Paradoxical%20DNA%20Repair%20and%20Peroxide%20Resistance%20Gene%20Conservation%20in%20Bacillus%20pumilus%20SAFR-032 http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000928" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000928" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacillus+subtilis%22" title="Search for articles in the subject area:'Bacillus subtilis'"><div class="flagText">Bacillus subtilis</div></a>
              <div data-categoryid="32939" data-articleid="24694"
                   data-categoryname="Bacillus subtilis"
                   class="flagImage" title="Flag 'Bacillus subtilis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+spores%22" title="Search for articles in the subject area:'Bacterial spores'"><div class="flagText">Bacterial spores</div></a>
              <div data-categoryid="42749" data-articleid="24694"
                   data-categoryname="Bacterial spores"
                   class="flagImage" title="Flag 'Bacterial spores' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+repair%22" title="Search for articles in the subject area:'DNA repair'"><div class="flagText">DNA repair</div></a>
              <div data-categoryid="17453" data-articleid="24694"
                   data-categoryname="DNA repair"
                   class="flagImage" title="Flag 'DNA repair' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24694"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="24694"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lichenology%22" title="Search for articles in the subject area:'Lichenology'"><div class="flagText">Lichenology</div></a>
              <div data-categoryid="27487" data-articleid="24694"
                   data-categoryname="Lichenology"
                   class="flagImage" title="Flag 'Lichenology' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24694"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ultraviolet+radiation%22" title="Search for articles in the subject area:'Ultraviolet radiation'"><div class="flagText">Ultraviolet radiation</div></a>
              <div data-categoryid="18913" data-articleid="24694"
                   data-categoryname="Ultraviolet radiation"
                   class="flagImage" title="Flag 'Ultraviolet radiation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6431'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2131'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6907&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
