

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: A Framework for Exploring Functional Variability in Olfactory Receptor Genes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000682"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000682" />

    <meta name="citation_title" content="A Framework for Exploring Functional Variability in Olfactory Receptor Genes"/>
    <meta itemprop="name" content="A Framework for Exploring Functional Variability in Olfactory Receptor Genes"/>

      <meta name="citation_author" content="Orna Man"/>
            <meta name="citation_author_institution" content="Department of Molecular Genetics, Weizmann Institute of Science, Rehovot, Israel"/>
            <meta name="citation_author_institution" content="Department of Structural Biology, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="David C. Willhite"/>
            <meta name="citation_author_institution" content="Department of Neurobiology, Yale University School of Medicine, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Chiquito J. Crasto"/>
            <meta name="citation_author_institution" content="Department of Neurobiology, Yale University School of Medicine, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Gordon M. Shepherd"/>
            <meta name="citation_author_institution" content="Department of Neurobiology, Yale University School of Medicine, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Yoav Gilad"/>
            <meta name="citation_author_institution" content="Department of Human Genetics, University of Chicago, Chicago, Illinois, United States of America"/>

    <meta name="citation_date" content="2007/8/1"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000682.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e682"/>
    <meta name="citation_issue" content="8"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A novel multigene family may encode odorant receptors: a molecular basis for odor recognition.; citation_author=L Buck; citation_author=R Axel; citation_journal_title=Cell; citation_volume=65; citation_number=1; citation_pages=175-187; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Seven-transmembrane proteins as odorant and chemosensory receptors.; citation_author=P Mombaerts; citation_journal_title=Science; citation_volume=286; citation_number=2; citation_pages=707-711; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The complete human olfactory subgenome.; citation_author=G Glusman; citation_author=I Yanai; citation_author=I Rubin; citation_author=D Lancet; citation_journal_title=Genome Res; citation_volume=11; citation_number=3; citation_pages=685-702; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The olfactory receptor gene superfamily: data mining, classification, and nomenclature.; citation_author=G Glusman; citation_author=A Bahar; citation_author=D Sharon; citation_author=Y Pilpel; citation_author=J White; citation_journal_title=Mamm Genome; citation_volume=11; citation_number=4; citation_pages=1016-1023; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The family of genes encoding odorant receptors in the channel catfish.; citation_author=J Ngai; citation_author=MM Dowling; citation_author=L Buck; citation_author=R Axel; citation_author=A Chess; citation_journal_title=Cell; citation_volume=72; citation_number=5; citation_pages=657-666; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Evolutionary dynamics of olfactory receptor genes in fishes and tetrapods.; citation_author=Y Niimura; citation_author=M Nei; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=6; citation_pages=6039-6044; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The olfactory receptor gene superfamily of the mouse.; citation_author=X Zhang; citation_author=S Firestein; citation_journal_title=Nat Neurosci; citation_volume=5; citation_number=7; citation_pages=124-133; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Comparison of the canine and human olfactory receptor gene repertoires.; citation_author=P Quignon; citation_author=E Kirkness; citation_author=E Cadieu; citation_author=N Touleimat; citation_author=R Guyon; citation_journal_title=Genome Biol; citation_volume=4; citation_number=8; citation_pages=R80; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of the Brown Norway rat yields insights into mammalian evolution.; citation_author=RA Gibbs; citation_author=GM Weinstock; citation_author=ML Metzker; citation_author=DM Muzny; citation_author=EJ Sodergren; citation_journal_title=Nature; citation_volume=428; citation_number=9; citation_pages=493-521; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A comparison of the human and chimpanzee olfactory receptor gene repertoires.; citation_author=Y Gilad; citation_author=O Man; citation_author=G Glusman; citation_journal_title=Genome Res; citation_volume=15; citation_number=10; citation_pages=224-230; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The human olfactory receptor repertoire.; citation_author=S Zozulya; citation_author=F Echeverri; citation_author=T Nguyen; citation_journal_title=Genome Biol; citation_volume=2; citation_number=11; citation_pages=RESEARCH0018; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The olfactory receptor universe–from whole genome analysis to structure and evolution.; citation_author=T Olender; citation_author=E Feldmesser; citation_author=T Atarot; citation_author=M Eisenstein; citation_author=D Lancet; citation_journal_title=Genet Mol Res; citation_volume=3; citation_number=12; citation_pages=545-553; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Different evolutionary processes shaped the mouse and human olfactory receptor gene families.; citation_author=JM Young; citation_author=C Friedman; citation_author=EM Williams; citation_author=JA Ross; citation_author=L Tonnes-Priddy; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=13; citation_pages=535-546; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The mouse olfactory receptor gene family.; citation_author=PA Godfrey; citation_author=B Malnic; citation_author=LB Buck; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=14; citation_pages=2156-2161; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The sense of smell: genomics of vertebrate odorant receptors.; citation_author=JM Young; citation_author=BJ Trask; citation_journal_title=Hum Mol Genet; citation_volume=11; citation_number=15; citation_pages=1153-1160; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The canine olfactory subgenome.; citation_author=T Olender; citation_author=T Fuchs; citation_author=C Linhart; citation_author=R Shamir; citation_author=M Adams; citation_journal_title=Genomics; citation_volume=83; citation_number=16; citation_pages=361-372; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The olfactory receptor gene repertoire in primates and mouse: evidence for reduction of the functional fraction in primates.; citation_author=S Rouquier; citation_author=A Blancher; citation_author=D Giorgi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=17; citation_pages=2870-2874; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Human specific loss of olfactory receptor genes.; citation_author=Y Gilad; citation_author=O Man; citation_author=S Paabo; citation_author=D Lancet; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=18; citation_pages=3324-3327; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Loss of olfactory receptor genes coincides with the acquisition of full trichromatic vision in primates.; citation_author=Y Gilad; citation_author=V Wiebe; citation_author=M Przeworski; citation_author=D Lancet; citation_author=S Paabo; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=19; citation_pages=E5; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=After the holy grail: establishing a molecular basis for Mammalian olfaction.; citation_author=RR Reed; citation_journal_title=Cell; citation_volume=116; citation_number=20; citation_pages=329-336; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=‘Microsmatic’ primates revisited: olfactory sensitivity in the squirrel monkey.; citation_author=M Laska; citation_author=A Seibt; citation_author=A Weber; citation_journal_title=Chem Senses; citation_volume=25; citation_number=21; citation_pages=47-53; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mutant G-protein-coupled receptors as a cause of human diseases.; citation_author=T Schoneberg; citation_author=A Schulz; citation_author=H Biebermann; citation_author=T Hermsdorf; citation_author=H Rompler; citation_journal_title=Pharmacol Ther; citation_volume=104; citation_number=22; citation_pages=173-206; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The substrate specificity-determining amino acid code of 4-coumarate:CoA ligase.; citation_author=K Schneider; citation_author=K Hovel; citation_author=K Witzel; citation_author=B Hamberger; citation_author=D Schomburg; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=23; citation_pages=8601-8606; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Altering substrate specificity of phosphatidylcholine-preferring phospholipase C of Bacillus cereus by random mutagenesis of the headgroup binding site.; citation_author=NM Antikainen; citation_author=PJ Hergenrother; citation_author=MM Harris; citation_author=W Corbett; citation_author=SF Martin; citation_journal_title=Biochemistry; citation_volume=42; citation_number=24; citation_pages=1603-1610; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Prediction of the odorant binding site of olfactory receptor proteins by human-mouse comparisons.; citation_author=O Man; citation_author=Y Gilad; citation_author=D Lancet; citation_journal_title=Protein Sci; citation_volume=13; citation_number=25; citation_pages=240-254; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Amino acid difference formula to help explain protein evolution.; citation_author=R Grantham; citation_journal_title=Science; citation_volume=185; citation_number=26; citation_pages=862-864; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=A global geometric framework for nonlinear dimensionality reduction.; citation_author=JB Tenenbaum; citation_author=V de Silva; citation_author=JC Langford; citation_journal_title=Science; citation_volume=290; citation_number=27; citation_pages=2319-2323; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The isomap algorithm and topological stability.; citation_author=M Balasubramanian; citation_author=EL Schwartz; citation_journal_title=Science; citation_volume=295; citation_number=28; citation_pages=7; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genetic basis for specific anosmia to isovaleric acid in the mouse.; citation_author=IC Griff; citation_author=RR Reed; citation_journal_title=Cell; citation_volume=83; citation_number=29; citation_pages=407-414; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Individual differences in the perceived intensity and quality of specific odors following self- and cross-adaptation.; citation_author=RJ O'Connell; citation_author=DA Stevens; citation_author=LM Zogby; citation_journal_title=Chem Senses; citation_volume=19; citation_number=30; citation_pages=197-208; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Probability model for molecular recognition in biological receptor repertoires: significance to the olfactory system.; citation_author=D Lancet; citation_author=E Sadovsky; citation_author=E Seidemann; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=31; citation_pages=3715-3719; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Combinatorial receptor codes for odors.; citation_author=B Malnic; citation_author=J Hirono; citation_author=T Sato; citation_author=LB Buck; citation_journal_title=Cell; citation_volume=96; citation_number=32; citation_pages=713-723; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mouse-human orthology relationships in an olfactory receptor gene cluster.; citation_author=M Lapidot; citation_author=Y Pilpel; citation_author=Y Gilad; citation_author=A Falcovitz; citation_author=D Sharon; citation_journal_title=Genomics; citation_volume=71; citation_number=33; citation_pages=296-306; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular bases of odor discrimination: Reconstitution of olfactory receptors that recognize overlapping sets of odorants.; citation_author=K Kajiya; citation_author=K Inaki; citation_author=M Tanaka; citation_author=T Haga; citation_author=H Kataoka; citation_journal_title=J Neurosci; citation_volume=21; citation_number=34; citation_pages=6018-6025; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identification of ligands for olfactory receptors by functional expression of a receptor library.; citation_author=D Krautwurst; citation_author=KW Yau; citation_author=RR Reed; citation_journal_title=Cell; citation_volume=95; citation_number=35; citation_pages=917-926; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genes and ligands for odorant, vomeronasal and taste receptors.; citation_author=P Mombaerts; citation_journal_title=Nat Rev Neurosci; citation_volume=5; citation_number=36; citation_pages=263-278; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Using orthologous and paralogous proteins to identify specificity-determining residues in bacterial transcription factors.; citation_author=LA Mirny; citation_author=MS Gelfand; citation_journal_title=J Mol Biol; citation_volume=321; citation_number=37; citation_pages=7-20; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Amino acids determining enzyme-substrate specificity in prokaryotic and eukaryotic protein kinases.; citation_author=L Li; citation_author=EI Shakhnovich; citation_author=LA Mirny; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=38; citation_pages=4463-4468; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Odorant receptor expression defines functional units in the mouse olfactory system.; citation_author=T Bozza; citation_author=P Feinstein; citation_author=C Zheng; citation_author=P Mombaerts; citation_journal_title=J Neurosci; citation_volume=22; citation_number=39; citation_pages=3033-3043; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of rhodopsin: A G protein-coupled receptor.; citation_author=K Palczewski; citation_author=T Kumasaka; citation_author=T Hori; citation_author=CA Behnke; citation_author=H Motoshima; citation_journal_title=Science; citation_volume=289; citation_number=40; citation_pages=739-745; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Multiple sequence alignment with the Clustal series of programs.; citation_author=R Chenna; citation_author=H Sugawara; citation_author=T Koike; citation_author=R Lopez; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=41; citation_pages=3497-3500; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Structural basis for a broad but selective ligand spectrum of a mouse olfactory receptor: mapping the odorant-binding site.; citation_author=S Katada; citation_author=T Hirokawa; citation_author=Y Oka; citation_author=M Suwa; citation_author=K Touhara; citation_journal_title=J Neurosci; citation_volume=25; citation_number=42; citation_pages=1806-1815; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Understanding human disease mutations through the use of interspecific genetic variation.; citation_author=MP Miller; citation_author=S Kumar; citation_journal_title=Hum Mol Genet; citation_volume=10; citation_number=43; citation_pages=2319-2328; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=AAindex: Amino Acid Index Database.; citation_author=S Kawashima; citation_author=H Ogata; citation_author=M Kanehisa; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=44; citation_pages=368-369; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Determining the Number of Clusters/Segments in Hierarchical Clustering/Segmentation Algorithms;; citation_author=S Salvador; citation_author=P Chan; citation_number=45; citation_pages=576-584; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="A Framework for Exploring Functional Variability in Olfactory Receptor Genes"/>
    <meta name="twitter:description" content="BackgroundOlfactory receptors (ORs) are the largest gene family in mammalian genomes. Since nearly all OR genes are orphan receptors, inference of functional similarity or differences between odorant receptors typically relies on sequence comparisons. Based on the alignment of entire coding region sequence, OR genes are classified into families and subfamilies, a classification that is believed to be a proxy for OR gene functional variability. However, the assumption that overall protein sequence diversity is a good proxy for functional properties is untested.MethodologyHere, we propose an alternative sequence-based approach to infer the similarities and differences in OR binding capacity. Our approach is based on similarities and differences in the predicted binding pockets of OR genes, rather than on the entire OR coding region.ConclusionsInterestingly, our approach yields markedly different results compared to the analysis based on the entire OR coding-regions. While neither approach can be tested at this time, the discrepancy between the two calls into question the assumption that the current classification reliably reflects OR gene functional variability."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000682.g003"/>

  <meta property="og:title" content="A Framework for Exploring Functional Variability in Olfactory Receptor Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=783'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=71'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5153&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000682">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000682" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000682&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    A Framework for Exploring Functional Variability in Olfactory Receptor Genes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Orna Man
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Orna Man, David C. Willhite</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:orna.man@weizmann.ac.il">orna.man@weizmann.ac.il</a> (OM); <a href="mailto:gilad@uchicago.edu">gilad@uchicago.edu</a> (YG)</p>

                <p>Affiliations:
                  Department of Molecular Genetics, Weizmann Institute of Science, Rehovot, Israel, 
                  Department of Structural Biology, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David C. Willhite
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Orna Man, David C. Willhite</p>

              
              

                <p>Affiliation:
                  Department of Neurobiology, Yale University School of Medicine, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chiquito J. Crasto, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurobiology, Yale University School of Medicine, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gordon M. Shepherd, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurobiology, Yale University School of Medicine, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yoav Gilad
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:orna.man@weizmann.ac.il">orna.man@weizmann.ac.il</a> (OM); <a href="mailto:gilad@uchicago.edu">gilad@uchicago.edu</a> (YG)</p>

                <p>Affiliation:
                  Department of Human Genetics, University of Chicago, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: August 01, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000682</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000682-g001" data-doi="info:doi/10.1371/journal.pone.0000682" data-uri="info:doi/10.1371/journal.pone.0000682.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000682-g002" data-doi="info:doi/10.1371/journal.pone.0000682" data-uri="info:doi/10.1371/journal.pone.0000682.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000682-g003" data-doi="info:doi/10.1371/journal.pone.0000682" data-uri="info:doi/10.1371/journal.pone.0000682.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000682">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000682" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Olfactory receptors (ORs) are the largest gene family in mammalian genomes. Since nearly all OR genes are orphan receptors, inference of functional similarity or differences between odorant receptors typically relies on sequence comparisons. Based on the alignment of entire coding region sequence, OR genes are classified into families and subfamilies, a classification that is believed to be a proxy for OR gene functional variability. However, the assumption that overall protein sequence diversity is a good proxy for functional properties is untested.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here, we propose an alternative sequence-based approach to infer the similarities and differences in OR binding capacity. Our approach is based on similarities and differences in the predicted binding pockets of OR genes, rather than on the entire OR coding region.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Interestingly, our approach yields markedly different results compared to the analysis based on the entire OR coding-regions. While neither approach can be tested at this time, the discrepancy between the two calls into question the assumption that the current classification reliably reflects OR gene functional variability.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Man O, Willhite DC, Crasto CJ, Shepherd GM, Gilad Y (2007) A Framework for Exploring Functional Variability in Olfactory Receptor Genes. PLoS ONE 2(8):
          e682.
            doi:10.1371/journal.pone.0000682</p><p><strong>Academic Editor: </strong>Ivan Baxter, Purdue University, United States of America</p><p><strong>Received:</strong> March 23, 2007; <strong>Accepted:</strong> June 28, 2007; <strong>Published:</strong> August 1, 2007</p><p><strong>Copyright:</strong> © 2007 Man et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>GMS, CC, and DCW were supported by NIH grant DC00086 (GMS).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Olfactory receptor (OR) genes, the largest gene family in mammalian genomes, constitute the basis for the sense of smell <a href="#pone.0000682-Buck1">[1]</a>–<a href="#pone.0000682-Glusman1">[3]</a>. Based on their full length protein sequence similarity, mammalian OR genes are divided into two classes, 17 families and ~250 subfamilies <a href="#pone.0000682-Glusman2">[4]</a>. OR genes from the same subfamily are defined as sharing 60% or more of their overall amino acid sequence. Class I genes are closely related to OR genes found in fish and are hence referred to as ‘fish like’ <a href="#pone.0000682-Ngai1">[5]</a>, while class II OR genes are specific to terrestrials <a href="#pone.0000682-Glusman1">[3]</a>; but see <a href="#pone.0000682-Niimura1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Recently, the complete OR gene repertoires of a number of mammalian species became available, permitting interspecies comparisons of complete OR gene repertoires <a href="#pone.0000682-Glusman1">[3]</a>, <a href="#pone.0000682-Zhang1">[7]</a>–<a href="#pone.0000682-Gilad1">[10]</a>. Such inter-species comparisons may increase our understanding of the similarities and differences between the sense of smell of different species. Humans, for example, have roughly 900 OR genes <a href="#pone.0000682-Glusman1">[3]</a>, <a href="#pone.0000682-Zozulya1">[11]</a>, <a href="#pone.0000682-Olender1">[12]</a>, but 54% of them carry one or more coding region disruptions and therefore are annotated as pseudogenes. In contrast, the mouse OR gene repertoire is ~30% larger than that of man <a href="#pone.0000682-Zhang1">[7]</a>, <a href="#pone.0000682-Young1">[13]</a>, <a href="#pone.0000682-Godfrey1">[14]</a>, but contains only 20% pseudogenes. Thus, the mouse putative functional OR gene repertoire is more than three times larger than that of humans <a href="#pone.0000682-Young2">[15]</a>. Similarly low proportions of OR pseudogenes were found in dogs and rats <a href="#pone.0000682-Quignon1">[8]</a>, <a href="#pone.0000682-Gibbs1">[9]</a>, <a href="#pone.0000682-Olender2">[16]</a>. In fact, it appears that humans have been accumulating OR pseudogenes faster than other primates <a href="#pone.0000682-Rouquier1">[17]</a>–<a href="#pone.0000682-Gilad3">[19]</a>, and as a result, have fewer intact (and putatively functional) OR genes, even when compared with chimpanzee, our closest living evolutionary relative <a href="#pone.0000682-Gilad1">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>However, nearly all mammalian OR genes are orphan receptors, as very few ligand (odorant) – receptor interactions have been demonstrated for OR genes <a href="#pone.0000682-Reed1">[20]</a>. Indeed, direct inter-species functional studies are extremely demanding and hence rare <a href="#pone.0000682-Laska1">[21]</a>. As a proxy for functional variability, similarities and differences between the protein sequence of OR genes are often used <a href="#pone.0000682-Zhang1">[7]</a>, <a href="#pone.0000682-Godfrey1">[14]</a>. In this approach, it is assumed that when orthologous OR genes are identical in sequence, both species maintain the same olfactory capability. More difficult is the interpretation of sequence <em>differences</em> between orthologous genes. To date, OR genes have been thought to have a similar function so long as they were classified into the same subfamily based on the full length of the OR protein <a href="#pone.0000682-Glusman2">[4]</a>, <a href="#pone.0000682-Zhang1">[7]</a>, <a href="#pone.0000682-Godfrey1">[14]</a>. This assumption remains untested.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Differences in specific binding properties are expected to be affected primarily by changes in the receptor's binding site(s) <a href="#pone.0000682-Schoneberg1">[22]</a>–<a href="#pone.0000682-Antikainen1">[24]</a>. Hence, sequence similarity in binding sites, rather than in the entire protein, may be a more reliable predictor of functional similarity. To examine this possibility, we assess functional variability across OR gene repertoires of two species, mouse and human, by considering only the putative OR protein binding site <a href="#pone.0000682-Man1">[25]</a>. We assume that OR genes with identical binding sites have the same binding properties and use Grantham's amino acid property scales <a href="#pone.0000682-Grantham1">[26]</a> to model functional differences among binding sites. We note that the assumption underlying our approach is also not tested. Moreover, at this time we cannot assess whether our analysis provides more accurate functional inference than the commonly used analysis based on the entire OR coding region. Instead, we propose our approach as a second, sensible solution to a problem that to date has only been addressed using only a single type of analysis. Interestingly, we find that the two approaches yield markedly different results, with potentially important implications for the interpretation of OR gene families.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Representation of the odorant space</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The species' odorant space can be defined as the collection of all odorants that a species is capable of detecting as well as their detection thresholds. Direct mapping of a species' odorant space could be achieved by either <em>in vivo</em> testing of the detection threshold for all possible odorants, or by identifying the binding affinity properties of all functional OR genes <em>in vitro</em>. Currently, high throughput data of this type do not exist.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Instead, we take an indirect route to infer the odorant space of human and mouse by mapping the chemical distances between the putative specificity determining residues (SDRs) <a href="#pone.0000682-Man1">[25]</a> of all OR genes. These residues were predicted based on protein sequence analysis and are supported by experimental results (see <a href="#s4">methods</a>). In particular, a functional role for most of these putative SDRs was confirmed by a recent OR protein functional study (Kristin Schmiedeberg, Elena Shirokova, Hans-Peter Weber, Jens Reden, Thomas Hummel, Boris Schilling, Wolfgang Meyerhof, and Dietmar Krautwurst; personal communication). Henceforth, we refer to the SDR residues as the putative binding site.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>For our inter-species comparison of OR repertoire variability, we use the entire human and mouse OR intact gene repertoires, the only two species for which we have well characterized repertoires from finished genomic sequence (the dog OR repertoire is not yet fully described <a href="#pone.0000682-Quignon1">[8]</a> and the draft of the chimpanzee genome has relatively low coverage for this type of analysis <a href="#pone.0000682-Gilad1">[10]</a>). We use previously published multiple alignment of OR genes from human and mouse <a href="#pone.0000682-Man1">[25]</a> in order to identify the putative binding sites of all OR proteins. To estimate distances between binding sites, we construct a distance matrix for the 22 amino acid residues of each binding site, utilizing Grantham indices <a href="#pone.0000682-Grantham1">[26]</a>. We visualize this distance matrix in two and/or three dimensional space by using the isomap algorithm <a href="#pone.0000682-Tenenbaum1">[27]</a>, <a href="#pone.0000682-Balasubramanian1">[28]</a> for geodesic mapping (see <a href="#s4">methods</a>). This approach assumes that proximity in binding sites reflects similarity in binding properties (i.e., that the ORs bind related odorants with similar binding affinity). Conversely, distant binding sites are assumed to have highly distinct binding properties. This set of assumptions has also been made when inferring functional variability from phylogenetic tree distances between OR genes based on the entire protein sequence (e.g., <a href="#pone.0000682-Godfrey1">[14]</a>).</p>


<h4>The human and mouse odorant spaces</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We superimpose two-dimensional representations of the human and mouse odorant spaces to compare the relative space coverage. As can be seen in <a href="#pone-0000682-g001">figure 1</a>, the odorant spaces of human and mouse largely overlap. However, even at low resolution (a high resolution 3D map of the entire data is available as supplementary material at <a href="http://senselab.med.yale.edu/senselab/ORDB/ordb_ent.html">http://senselab.med.yale.edu/senselab/OR​DB/ordb_ent.html</a>), regions of the map enriched in ORs from only one of the species are readily apparent (<em>e.g</em>. regions a–d in <a href="#pone-0000682-g001">Figure 1</a>). We note that, as expected, representing the human and mouse OR repertoires using randomly selected groups of 22 residues of the OR protein yields vastly different results than those shown here (<a href="#pone.0000682.s001">Figure S1</a>).</p>
<div class="figure" id="pone-0000682-g001"><div class="img"><a name="pone-0000682-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000682" data-uri="info:doi/10.1371/journal.pone.0000682.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000682.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000682.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>2D representation of the human (red) and mouse (blue) odorant spaces.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Map regions with an over-representation of binding sites from either human or mouse are circled (a–d). An example of a pair of human-mouse orthologs (MOR27-1 and OR52P1) with identical binding site is indicated in (e).</p>
<span>doi:10.1371/journal.pone.0000682.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>To identify groups of OR genes with highly similar binding sites, we perform a clustering analysis (see <a href="#s4">methods</a>). This yields 258 clusters, 64 of which contained only a single gene. Of the 194 clusters with more than one gene, six (<em>P</em> [of observing as many or more clusters] = 0.34) and 104 (<em>P</em> = 0.01) have greater than 70% of the genes from either human or mouse, respectively. Moreover, we find 41 clusters only in one of the two species (5 human-specific (<em>P</em> = 0.03) and 36 mouse-specific (<em>P</em> = 0.09). Interestingly, these do not correspond to the classification to OR gene subfamilies. In fact, inspection of a full-length protein sequence phylogenetic tree of OR genes reveals that clusters of OR genes with highly similar binding sites in our analysis are not often monophyletic, and often are not even in close proximity with one another (as illustrated in <a href="#pone-0000682-g002">Figure 2A, B</a>). Specifically, only six of the 41 OR clusters specific to one species consist entirely of genes that form a monophyletic clade in the phylogenetic tree based on the entire protein sequences of human and mouse OR genes.</p>
<div class="figure" id="pone-0000682-g002"><div class="img"><a name="pone-0000682-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000682" data-uri="info:doi/10.1371/journal.pone.0000682.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000682.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000682.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>A. 3D representation of the human (red) and mouse (blue) odorant spaces.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Shown in dark green is a mouse-specific cluster (consisting of MOR232-2, MOR232-5, MOR233-1, MOR233-7, MOR235-1). B. A phylogenetic tree based on the full protein sequence that includes all five genes (dark green) in the mouse specific cluster shown in A. The five genes do not form a monophyletic clade, but instead are interspersed among human (red) and other mouse (blue) genes.</p>
<span>doi:10.1371/journal.pone.0000682.g002</span></div>

<h4>Clustering analysis and the case of isovaleric acid</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Our analysis suggests an explanation of the apparent contradiction between the sensitivity of mice and humans to isovaleric acid (IVA) and the lack of IVA OR orthologs in humans when full length sequences are compared. Specifically, the genomic regions responsible for specific anosmia to Iva in the mouse were identified <a href="#pone.0000682-Griff1">[29]</a> and, following the classification of the complete mouse OR gene repertoire, 12 intact OR genes (as well as two OR pseudogene with more than two coding region disruptions) were mapped to these regions <a href="#pone.0000682-Zhang1">[7]</a>. Based on full-length protein sequence analysis, the 12 intact OR genes were classified into two small subfamilies that were found to be specific to mouse <a href="#pone.0000682-Zhang1">[7]</a>. Based on the view that sub-family classification is a proxy for functional similarity, the lack of human OR gene orthologs to either of the two subfamilies would indicate that humans cannot detect Iva. However, O'Connell et al. <a href="#pone.0000682-OConnell1">[30]</a> reported that humans can detect Iva at relatively low threshold concentration. Using the putative binding sites to cluster OR genes helps resolve this apparent contradiction: By our approach, the 12 mouse OR genes that confer the ability to detect Iva cannot be clustered without including human OR genes <em>i.e.,</em> genes from the two species are interspersed (<a href="#pone-0000682-g003">Figure 3</a>). Thus, consideration of only the binding sites suggests that genes with similar binding properties are present in both species and that both humans and mice can detect Iva (possibly with similar capacity), in agreement with the finding of O'Connell et al.</p>
<div class="figure" id="pone-0000682-g003"><div class="img"><a name="pone-0000682-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000682" data-uri="info:doi/10.1371/journal.pone.0000682.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000682.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000682.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000682.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000682.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000682.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>2D representation of the human (red) and mouse (blue) odorant spaces.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>The ‘Iva mouse OR genes’ (yellow) do not form a single cluster (A). High resolution 2D representations of map regions I (B) and II (C) reveal that human OR binding sites are interspersed with the mouse Iva ORs in both regions.</p>
<span>doi:10.1371/journal.pone.0000682.g003</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>To date, the functional variability within and between OR gene repertoires has been assessed by considering overall protein similarity <a href="#pone.0000682-Zhang1">[7]</a>, <a href="#pone.0000682-Godfrey1">[14]</a>. However, following gene duplication, the binding properties of an OR gene (or any receptor) may be modified by a few changes to the binding site. As a result, full-length protein sequence phylogenetic analysis captures the evolutionary relationship between different OR genes, but not necessarily the similarity in binding properties. In other words, recently duplicated OR genes may be highly similar at the overall protein level, but differ markedly in their binding capacities due to few changes to their binding sites. Here, we assess functional variability within OR gene repertoires by comparing the putative binding sites <a href="#pone.0000682-Man1">[25]</a> of all human and mouse putatively functional OR genes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>We find that the odorant spaces of the two species largely overlap, but that there is a somewhat larger than expected number of human-specific clusters of genes with highly similar binding sites, as well as a highly significant number of mouse-enriched (&gt;70%) clusters. These clusters of genes do not form monophyletic, species-specific lineages when the entire OR protein sequences were analyzed. The finding that two sets of assumptions yield such disparate results is interesting in its own right as it calls into question the reliability of the standard subfamily-based classification of OR genes into functional groups.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>It also raises the question of how to interpret OR gene clusters. A single receptor is thought to bind a series of chemically related odorants, with different affinities associated with each interaction <a href="#pone.0000682-Lancet1">[31]</a>, <a href="#pone.0000682-Malnic1">[32]</a>. If so, then, even with fewer functional receptors, humans may have preferentially retained ORs with broad specificities (‘generalists’) in order to maintain the ability to detect most odorants <a href="#pone.0000682-Lapidot1">[33]</a>. Alternatively, humans may have kept sets of OR genes that differ subtly in their binding sites in order to maintain strong binding affinity for each and every odorant in a group of chemically related compounds that are important in human evolution. Our analysis may be able to generate hypotheses to distinguish between these two possibilities: when a species requires the ability to detect and discriminate between chemically related odorants at low concentrations, this pressure should give rise to a cluster of OR genes with only subtly different binding sites. Conversely, if a region of the odorant space is represented by only one (or very few) ORs, this might suggest that most of the related odorants in that space are detected with low affinity (i.e., only at high concentration).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Overall, our analysis suggests that the odorant spaces of human and mouse largely overlap suggesting that, at saturating concentrations, both species are able to detect most odorants. In this respect, our results are consistent with the behavioral findings of Laska et al. <a href="#pone.0000682-Laska1">[21]</a>. However, we also find dozens of clusters of similar binding sites that are exclusive to one species, or nearly so. We speculate that these clusters reflect differences between human and mouse with respect to threshold detections of odorants bound by the OR genes that these clusters contain.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>This said, we cannot test our predictions, (or the predictions based on the commonly used analysis of the entire coding region). Mapping receptor-odorant interactions has proven to be a challenging task. Indeed, only a few receptor-odorant interactions have been reported <a href="#pone.0000682-Malnic1">[32]</a>, <a href="#pone.0000682-Kajiya1">[34]</a>, <a href="#pone.0000682-Krautwurst1">[35]</a>, and a large number of them relate to a single gene, OR I7 from rat. While the rat I7-odorant interactions have been reported in independent publications using different assays <a href="#pone.0000682-Mombaerts2">[36]</a>, most of the other putative receptor-odorant interactions have not yet been replicated, and were found using only one assay, often with unclear implications for in-vivo receptor-odorant interaction. Moreover, some of the reports have been contradictory. Thus, existing data are neither comprehensive nor reliable enough to be used for validation of our predictions. Since neither our approach nor that of using the full coding sequence can be validated experimentally, it is unclear which yields more accurate results.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Moreover, our findings are based on the current prediction of the OR protein SDR, which may change when a validated OR protein structure is available, or when other residues that determine the binding specificity are recognized. Clearly, our results are dependent on the identity of the 22 residues we analyze. However, our analysis can be easily repeated for a different set of residues.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In conclusion, our approach can not replace empirical studies of receptor-odorant interactions, but can help to generate prediction that can be tested empirically. In particular, as demonstrated for the case of the IVA, hypotheses generated using our approach may be quite different from those generated under the widely used classification approach which is based on the analysis of the entire OR protein sequence.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Prediction of the specificity determining residues</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We rely on previous prediction of 22 specificity determining residues (SDRs) in OR proteins <a href="#pone.0000682-Man1">[25]</a>. The assumption underlying the prediction of these SDRs was that these residues would be more conserved among orthologs, which are assumed to have a similar binding spectrum, than among paralogs, which are assumed to have a divergent binding spectrum. The implementation of this concept relied solely on protein sequence information. The same concept has been previously utilized to predict SDRs in bacterial transcription factors <a href="#pone.0000682-Mirny1">[37]</a> and eukaryotic and eubacterial protein kinases <a href="#pone.0000682-Li1">[38]</a>. In both cases the predictions were consistent with solved structures as well as experimental results. For OR proteins, no X-ray crystallographic structure has been determined and the only report of a mutation affecting specificity <a href="#pone.0000682-Krautwurst1">[35]</a> is now in question <a href="#pone.0000682-Bozza1">[39]</a>. Thus, the only validation of the predicted SDRs utilized information available from homologous GPCRs: In a homology model based on a high-resolution X-ray crystallographic structure of rhodopsin <a href="#pone.0000682-Palczewski1">[40]</a>, the 22 putative SDRs occupy the region corresponding to the retinal binding site. In addition, 21 of the 22 predicted SDRs correspond to amino acid positions that were previously associated with ligand-binding in at least one other GPCR <a href="#pone.0000682-Man1">[25]</a>.</p>


<h4>Multiple sequence alignment of OR proteins</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We used the alignment of <a href="#pone.0000682-Man1">[25]</a>, which contains 1441 protein sequences (402 and 1039 sequences from human and mouse, respectively). The alignment was constructed from all human and mouse OR protein sequences that span the seven putative transmembrane regions, contain no ambiguous residues, and no more than two coding region disruptions (in the text, we refer to all these as ‘intact’ genes). The approach taken in aligning the sequences was hierarchical in nature, combining automatic alignment of closely related sequences and the merger of such small alignments into larger alignments, each containing all the sequences belonging to a specific OR family. These family alignments were manually edited in order to ensure the correct alignment of the transmembrane regions, as well as of other OR protein motifs (such as an N-glycosylation site). Also, positions containing a gap in more than half of the members of the family were edited out. Finally, the family-wise alignments were manually merged into a single one, containing all 1441 OR sequences <a href="#pone.0000682-Man1">[25]</a>.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>A neighbor-joining tree was constructed from the most reliable positions in the OR multiple sequence alignment (positions containing fewer than 1% gaps), using ClustalX v1.83 <a href="#pone.0000682-Chenna1">[41]</a>. Trees were drawn using TreeExplorer (K. Tamura; <a href="http://evolgen.biol.metro-u.ac.jp/TE/TE_man.html">http://evolgen.biol.metro-u.ac.jp/TE/TE_​man.html</a>).</p>


<h4>Mapping of the odorant spaces of human and mouse</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Currently, there are no solved structures of OR proteins. Here, we rely on a prediction of the putative binding site based on the premise that orthologous OR genes are likely to have similar binding sites while the binding sites of paralogous OR genes are likely to be different <a href="#pone.0000682-Man1">[25]</a>. This theoretical prediction is in general agreement with the empirical results of Katada et al. <a href="#pone.0000682-Katada1">[42]</a>. We extracted the 22 residues corresponding to the putative odorant binding site from each OR sequence in the multiple alignment <a href="#pone.0000682-Man1">[25]</a>. Nine OR genes (human genes: OR9G3P, OR11J2P, OR2AE1, OR56B4; mouse genes: MOR188-2, MOR126-1, MOR126-2, MOR176-3, MOR204-25P) were found to contain a gap within the putative binding site, and were excluded from subsequent analyses.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Next, we constructed a distance matrix comparing the putative binding sites of all possible pairs of OR genes. In modeling the distance between binding sites, we used the Grantham chemical difference matrix <a href="#pone.0000682-Grantham1">[26]</a>. It has been shown that differences between disease alleles and wild-type alleles computed using the Grantham matrix are on average greater than those observed between putatively neutral polymorphic alleles, as well as inter-species differences <a href="#pone.0000682-Miller1">[43]</a>. This suggests that protein distances predicted by this matrix are functionally relevant.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>We downloaded from the AAIndex database <a href="#pone.0000682-Kawashima1">[44]</a> the three amino acid scales used in the construction of the Grantham chemical difference matrix <a href="#pone.0000682-Grantham1">[26]</a>, namely aax1:GRAR740101 (Composition), aax1:GRAR740102 (Polarity), and aax1:GRAR740103 (Volume). We used these indices to translate the putative binding site of each OR protein into a 66-dimensional vector. We then computed all pairwise distances between the vectors, using the standardized Euclidean distance as the distance measure, i.e., weighting each coordinate by the inverse of its variance.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>In order to to visualize the 66-dimensional space in two- and/or three-dimensions, we used the Isomap algorithm for geodesic dimensionality reduction <a href="#pone.0000682-Tenenbaum1">[27]</a>. We downloaded the Isomap Matlab package from <a href="http://isomap.stanford.edu/">http://isomap.stanford.edu/</a>. We then applied the IsomapII function (in Matlab/Math Works Inc v7 (R14)) to the previously computed distance matrix, using all available points as landmarks, and requesting that ten dimensions be calculated. In order to select the appropriate ε value, the neighborhood size parameter, we applied the tuning methodology described in <a href="#pone.0000682-Balasubramanian1">[28]</a>. Briefly, in this methodology, the algorithm is run multiple times, varying only the neighborhood size parameter, and recording each time the values of two cost- functions: the fraction of variance in geodesic distance estimates not accounted for by the Euclidean embedding and the fraction of points not included in the largest connected component of the neighborhood graph, and thus not included in the Euclidean embedding. The optimal neighborhood size is then determined as a tradeoff between these two cost functions. We found ε = 14.5 to be the optimal value for both the three-dimensional (accounting for 68.9% of the variance) as well as for the two-dimensional (accounting for 58.1% of the variance) embeddings. We confirmed the validity of the projection method by calculating the correlation of the matrix of standardized Euclidean distances based on the 66 dimensions with the Euclidean distances based on projections to two (r<sup>2</sup> = 0.75) or three (r<sup>2</sup> = 0.81) dimensions.</p>


<h4>Comparison of the mapping of odorant space obtained using the SDRs with those obtained using random sets of OR protein positions</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>In order to repeat our analysis using random sets of 22 residues, we used the set of sequence alignments that include only residues with no gaps as a pool, excluding the 22 SDRs themselves. This screening step resulted in a pool of 154 residues (of 327 positions in the original alignment) to choose from. Out of this pool we randomly chose five sets of positions, each of size 22. We then repeated the mapping procedure, using these random sets of positions instead of the SDRs, and reproduced <a href="#pone-0000682-g001">figure 1</a> for each random set of residues (<a href="#pone.0000682.s001">Figure S1</a>).</p>


<h4>Cluster analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>We applied agglomerative hierarchical clustering with average linkage, as implemented in Matlab/Math Works Inc v7 (R14), to the three-dimensional coordinates obtained from the Isomap procedure, using Euclidean distances as a distance measure. To determine the appropriate number of clusters, we utilized the unrefined L-method <a href="#pone.0000682-Salvador1">[45]</a>. For this purpose, the score for a specific number of clusters was the distance between the two nearest clusters (<em>i.e.</em> the clusters that would be merged if we were to decrease the number of clusters by one). By plotting scores against the number of clusters we determined the knee of the curve, and thus the number of optimal clusters, to be 110.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The 110 clusters obtained ranged in size from 1 (singleton clusters) to 323 ORs. Because of the wide range of cluster sizes, we decided to re-apply the hierarchical clustering. In order to determine which clusters would be divided further, we used the unrefined L-method <a href="#pone.0000682-Salvador1">[45]</a> again, this time plotting cluster size against cluster rank, where the rank is determined by size. We found that the knee of the curve corresponded to a cluster of size 46, and, thus, re-applied the hierarchical clustering procedure to only the seven clusters containing at least 46 OR genes. Our final analysis yielded 258 clusters, ranging in size from 1 to 29 (mean size: 5.56).</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>To test the significance of observing clusters specific to one species or enriched (&gt;70%) in one species, we took the following approach: For each group of clusters that are specific or enriched in one species, containing N clusters of sizes s<sub>1</sub>≥s<sub>2</sub>≥…≥s<sub>N</sub>, we randomly permuted the species-assignments 100,000 times, and counted the number of permutations fulfilling the following conditions (excluding singletons):</p>


<ol class="order">

<li>M, the number of clusters belonging to class X in the random permutation, fulfills N≤M.</li>

<li>Let t<sub>1</sub>≥t<sub>2</sub>≥…≥t<sub>M</sub> be the sizes of the clusters belonging to class X in the random permutation, then: t<sub>i</sub>≥s<sub>i</sub> for every 1≤i≤N</li>

</ol>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000682"></div><a name="pone.0000682.s001" id="pone.0000682.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000682.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000682.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank M. Przeworski for comments on the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GS YG OM DW. Analyzed the data: YG OM CC. Wrote the paper: GS YG OM DW CC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000682-Buck1" id="pone.0000682-Buck1"></a>Buck L, Axel R (1991) A novel multigene family may encode odorant receptors: a molecular basis for odor recognition. Cell  65: 175–187.  <ul class="find" data-citedArticleID="972288" data-doi="10.1016/0092-8674(91)90418-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(91)90418-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+multigene+family+may+encode+odorant+receptors%3A+a+molecular+basis+for+odor+recognition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+multigene+family+may+encode+odorant+receptors%3A+a+molecular+basis+for+odor+recognition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000682-Mombaerts1" id="pone.0000682-Mombaerts1"></a>Mombaerts P (1999) Seven-transmembrane proteins as odorant and chemosensory receptors. Science  286: 707–711.  <ul class="find" data-citedArticleID="972334" data-doi="10.1126/science.286.5440.707"><li><a href="http://dx.doi.org/10.1126/science.286.5440.707" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Seven-transmembrane+proteins+as+odorant+and+chemosensory+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Seven-transmembrane+proteins+as+odorant+and+chemosensory+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000682-Glusman1" id="pone.0000682-Glusman1"></a>Glusman G, Yanai I, Rubin I, Lancet D (2001) The complete human olfactory subgenome. Genome Res  11: 685–702.  <ul class="find" data-citedArticleID="972300" data-doi="10.1101/gr.171001"><li><a href="http://dx.doi.org/10.1101/gr.171001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+human+olfactory+subgenome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+human+olfactory+subgenome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000682-Glusman2" id="pone.0000682-Glusman2"></a>Glusman G, Bahar A, Sharon D, Pilpel Y, White J, et al.  (2000) The olfactory receptor gene superfamily: data mining, classification, and nomenclature. Mamm Genome  11: 1016–1023.  <ul class="find" data-citedArticleID="972302" data-doi="10.1007/s003350010196"><li><a href="http://dx.doi.org/10.1007/s003350010196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+olfactory+receptor+gene+superfamily%3A+data+mining%2C+classification%2C+and+nomenclature." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+olfactory+receptor+gene+superfamily%3A+data+mining%2C+classification%2C+and+nomenclature.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000682-Ngai1" id="pone.0000682-Ngai1"></a>Ngai J, Dowling MM, Buck L, Axel R, Chess A (1993) The family of genes encoding odorant receptors in the channel catfish. Cell  72: 657–666.  <ul class="find" data-citedArticleID="972338" data-doi="10.1016/0092-8674(93)90395-7"><li><a href="http://dx.doi.org/10.1016/0092-8674(93)90395-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+family+of+genes+encoding+odorant+receptors+in+the+channel+catfish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+family+of+genes+encoding+odorant+receptors+in+the+channel+catfish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000682-Niimura1" id="pone.0000682-Niimura1"></a>Niimura Y, Nei M (2005) Evolutionary dynamics of olfactory receptor genes in fishes and tetrapods. Proc Natl Acad Sci U S A  102: 6039–6044.  <ul class="find" data-citedArticleID="972340" data-doi="10.1073/pnas.0501922102"><li><a href="http://dx.doi.org/10.1073/pnas.0501922102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+dynamics+of+olfactory+receptor+genes+in+fishes+and+tetrapods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+dynamics+of+olfactory+receptor+genes+in+fishes+and+tetrapods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000682-Zhang1" id="pone.0000682-Zhang1"></a>Zhang X, Firestein S (2002) The olfactory receptor gene superfamily of the mouse. Nat Neurosci  5: 124–133.  <ul class="find" data-citedArticleID="972368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+olfactory+receptor+gene+superfamily+of+the+mouse.&amp;auth=&amp;atitle=The+olfactory+receptor+gene+superfamily+of+the+mouse." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+olfactory+receptor+gene+superfamily+of+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+olfactory+receptor+gene+superfamily+of+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000682-Quignon1" id="pone.0000682-Quignon1"></a>Quignon P, Kirkness E, Cadieu E, Touleimat N, Guyon R, et al.  (2003) Comparison of the canine and human olfactory receptor gene repertoires. Genome Biol  4: R80.  <ul class="find" data-citedArticleID="972350" data-doi="10.1186/gb-2003-4-12-r80"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-12-r80" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparison+of+the+canine+and+human+olfactory+receptor+gene+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparison+of+the+canine+and+human+olfactory+receptor+gene+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000682-Gibbs1" id="pone.0000682-Gibbs1"></a>Gibbs RA, Weinstock GM, Metzker ML, Muzny DM, Sodergren EJ, et al.  (2004) Genome sequence of the Brown Norway rat yields insights into mammalian evolution. Nature  428: 493–521.  <ul class="find" data-citedArticleID="972292"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution.&amp;auth=&amp;atitle=Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+the+Brown+Norway+rat+yields+insights+into+mammalian+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000682-Gilad1" id="pone.0000682-Gilad1"></a>Gilad Y, Man O, Glusman G (2005) A comparison of the human and chimpanzee olfactory receptor gene repertoires. Genome Res  15: 224–230.  <ul class="find" data-citedArticleID="972294" data-doi="10.1101/gr.2846405"><li><a href="http://dx.doi.org/10.1101/gr.2846405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+the+human+and+chimpanzee+olfactory+receptor+gene+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+the+human+and+chimpanzee+olfactory+receptor+gene+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000682-Zozulya1" id="pone.0000682-Zozulya1"></a>Zozulya S, Echeverri F, Nguyen T (2001) The human olfactory receptor repertoire. Genome Biol  2: RESEARCH0018.  <ul class="find" data-citedArticleID="972370" data-doi="10.1186/gb-2001-2-6-research0018"><li><a href="http://dx.doi.org/10.1186/gb-2001-2-6-research0018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+olfactory+receptor+repertoire." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+olfactory+receptor+repertoire.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000682-Olender1" id="pone.0000682-Olender1"></a>Olender T, Feldmesser E, Atarot T, Eisenstein M, Lancet D (2004) The olfactory receptor universe–from whole genome analysis to structure and evolution. Genet Mol Res  3: 545–553.  <ul class="find" data-citedArticleID="972344"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+olfactory+receptor+universe%26%23x2013%3Bfrom+whole+genome+analysis+to+structure+and+evolution.&amp;auth=&amp;atitle=The+olfactory+receptor+universe%26%23x2013%3Bfrom+whole+genome+analysis+to+structure+and+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+olfactory+receptor+universe%26%23x2013%3Bfrom+whole+genome+analysis+to+structure+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+olfactory+receptor+universe%26%23x2013%3Bfrom+whole+genome+analysis+to+structure+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000682-Young1" id="pone.0000682-Young1"></a>Young JM, Friedman C, Williams EM, Ross JA, Tonnes-Priddy L, et al.  (2002) Different evolutionary processes shaped the mouse and human olfactory receptor gene families. Hum Mol Genet  11: 535–546.  <ul class="find" data-citedArticleID="972364" data-doi="10.1093/hmg/11.5.535"><li><a href="http://dx.doi.org/10.1093/hmg/11.5.535" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+evolutionary+processes+shaped+the+mouse+and+human+olfactory+receptor+gene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+evolutionary+processes+shaped+the+mouse+and+human+olfactory+receptor+gene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000682-Godfrey1" id="pone.0000682-Godfrey1"></a>Godfrey PA, Malnic B, Buck LB (2004) The mouse olfactory receptor gene family. Proc Natl Acad Sci U S A  101: 2156–2161.  <ul class="find" data-citedArticleID="972304" data-doi="10.1073/pnas.0308051100"><li><a href="http://dx.doi.org/10.1073/pnas.0308051100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mouse+olfactory+receptor+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mouse+olfactory+receptor+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000682-Young2" id="pone.0000682-Young2"></a>Young JM, Trask BJ (2002) The sense of smell: genomics of vertebrate odorant receptors. Hum Mol Genet  11: 1153–1160.  <ul class="find" data-citedArticleID="972366" data-doi="10.1093/hmg/11.10.1153"><li><a href="http://dx.doi.org/10.1093/hmg/11.10.1153" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+sense+of+smell%3A+genomics+of+vertebrate+odorant+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+sense+of+smell%3A+genomics+of+vertebrate+odorant+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000682-Olender2" id="pone.0000682-Olender2"></a>Olender T, Fuchs T, Linhart C, Shamir R, Adams M, et al.  (2004) The canine olfactory subgenome. Genomics  83: 361–372.  <ul class="find" data-citedArticleID="972346" data-doi="10.1016/j.ygeno.2003.08.009"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2003.08.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+canine+olfactory+subgenome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+canine+olfactory+subgenome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000682-Rouquier1" id="pone.0000682-Rouquier1"></a>Rouquier S, Blancher A, Giorgi D (2000) The olfactory receptor gene repertoire in primates and mouse: evidence for reduction of the functional fraction in primates. Proc Natl Acad Sci U S A  97: 2870–2874.  <ul class="find" data-citedArticleID="972354" data-doi="10.1073/pnas.040580197"><li><a href="http://dx.doi.org/10.1073/pnas.040580197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+olfactory+receptor+gene+repertoire+in+primates+and+mouse%3A+evidence+for+reduction+of+the+functional+fraction+in+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+olfactory+receptor+gene+repertoire+in+primates+and+mouse%3A+evidence+for+reduction+of+the+functional+fraction+in+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000682-Gilad2" id="pone.0000682-Gilad2"></a>Gilad Y, Man O, Paabo S, Lancet D (2003) Human specific loss of olfactory receptor genes. Proc Natl Acad Sci U S A  100: 3324–3327.  <ul class="find" data-citedArticleID="972296" data-doi="10.1073/pnas.0535697100"><li><a href="http://dx.doi.org/10.1073/pnas.0535697100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+specific+loss+of+olfactory+receptor+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+specific+loss+of+olfactory+receptor+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000682-Gilad3" id="pone.0000682-Gilad3"></a>Gilad Y, Wiebe V, Przeworski M, Lancet D, Paabo S (2004) Loss of olfactory receptor genes coincides with the acquisition of full trichromatic vision in primates. PLoS Biol  2: E5.  <ul class="find" data-citedArticleID="972298" data-doi="10.1371/journal.pbio.0020005"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Loss+of+olfactory+receptor+genes+coincides+with+the+acquisition+of+full+trichromatic+vision+in+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Loss+of+olfactory+receptor+genes+coincides+with+the+acquisition+of+full+trichromatic+vision+in+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000682-Reed1" id="pone.0000682-Reed1"></a>Reed RR (2004) After the holy grail: establishing a molecular basis for Mammalian olfaction. Cell  116: 329–336.  <ul class="find" data-citedArticleID="972352" data-doi="10.1016/s0092-8674(04)00047-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00047-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=After+the+holy+grail%3A+establishing+a+molecular+basis+for+Mammalian+olfaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22After+the+holy+grail%3A+establishing+a+molecular+basis+for+Mammalian+olfaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000682-Laska1" id="pone.0000682-Laska1"></a>Laska M, Seibt A, Weber A (2000) ‘Microsmatic’ primates revisited: olfactory sensitivity in the squirrel monkey. Chem Senses  25: 47–53.  <ul class="find" data-citedArticleID="972322" data-doi="10.1093/chemse/25.1.47"><li><a href="http://dx.doi.org/10.1093/chemse/25.1.47" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%26%23x2018%3BMicrosmatic%26%23x2019%3B+primates+revisited%3A+olfactory+sensitivity+in+the+squirrel+monkey." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%26%23x2018%3BMicrosmatic%26%23x2019%3B+primates+revisited%3A+olfactory+sensitivity+in+the+squirrel+monkey.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000682-Schoneberg1" id="pone.0000682-Schoneberg1"></a>Schoneberg T, Schulz A, Biebermann H, Hermsdorf T, Rompler H, et al.  (2004) Mutant G-protein-coupled receptors as a cause of human diseases. Pharmacol Ther  104: 173–206.  <ul class="find" data-citedArticleID="972360" data-doi="10.1016/j.pharmthera.2004.08.008"><li><a href="http://dx.doi.org/10.1016/j.pharmthera.2004.08.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutant+G-protein-coupled+receptors+as+a+cause+of+human+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutant+G-protein-coupled+receptors+as+a+cause+of+human+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000682-Schneider1" id="pone.0000682-Schneider1"></a>Schneider K, Hovel K, Witzel K, Hamberger B, Schomburg D, et al.  (2003) The substrate specificity-determining amino acid code of 4-coumarate:CoA ligase. Proc Natl Acad Sci U S A  100: 8601–8606.  <ul class="find" data-citedArticleID="972358" data-doi="10.1073/pnas.1430550100"><li><a href="http://dx.doi.org/10.1073/pnas.1430550100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+substrate+specificity-determining+amino+acid+code+of+4-coumarate%3ACoA+ligase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+substrate+specificity-determining+amino+acid+code+of+4-coumarate%3ACoA+ligase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000682-Antikainen1" id="pone.0000682-Antikainen1"></a>Antikainen NM, Hergenrother PJ, Harris MM, Corbett W, Martin SF (2003) Altering substrate specificity of phosphatidylcholine-preferring phospholipase C of Bacillus cereus by random mutagenesis of the headgroup binding site. Biochemistry  42: 1603–1610.  <ul class="find" data-citedArticleID="972282" data-doi="10.1021/bi0267285"><li><a href="http://dx.doi.org/10.1021/bi0267285" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Altering+substrate+specificity+of+phosphatidylcholine-preferring+phospholipase+C+of+Bacillus+cereus+by+random+mutagenesis+of+the+headgroup+binding+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Altering+substrate+specificity+of+phosphatidylcholine-preferring+phospholipase+C+of+Bacillus+cereus+by+random+mutagenesis+of+the+headgroup+binding+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000682-Man1" id="pone.0000682-Man1"></a>Man O, Gilad Y, Lancet D (2004) Prediction of the odorant binding site of olfactory receptor proteins by human-mouse comparisons. Protein Sci  13: 240–254.  <ul class="find" data-citedArticleID="972328" data-doi="10.1110/ps.03296404"><li><a href="http://dx.doi.org/10.1110/ps.03296404" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+the+odorant+binding+site+of+olfactory+receptor+proteins+by+human-mouse+comparisons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+the+odorant+binding+site+of+olfactory+receptor+proteins+by+human-mouse+comparisons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000682-Grantham1" id="pone.0000682-Grantham1"></a>Grantham R (1974) Amino acid difference formula to help explain protein evolution. Science  185: 862–864.  <ul class="find" data-citedArticleID="972306" data-doi="10.1126/science.185.4154.862"><li><a href="http://dx.doi.org/10.1126/science.185.4154.862" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+difference+formula+to+help+explain+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+difference+formula+to+help+explain+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000682-Tenenbaum1" id="pone.0000682-Tenenbaum1"></a>Tenenbaum JB, de Silva V, Langford JC (2000) A global geometric framework for nonlinear dimensionality reduction. Science  290: 2319–2323.  <ul class="find" data-citedArticleID="972362" data-doi="10.1126/science.290.5500.2319"><li><a href="http://dx.doi.org/10.1126/science.290.5500.2319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+geometric+framework+for+nonlinear+dimensionality+reduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+geometric+framework+for+nonlinear+dimensionality+reduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000682-Balasubramanian1" id="pone.0000682-Balasubramanian1"></a>Balasubramanian M, Schwartz EL (2002) The isomap algorithm and topological stability. Science  295: 7.  <ul class="find" data-citedArticleID="972284" data-doi="10.1126/science.295.5552.7a"><li><a href="http://dx.doi.org/10.1126/science.295.5552.7a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+isomap+algorithm+and+topological+stability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+isomap+algorithm+and+topological+stability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000682-Griff1" id="pone.0000682-Griff1"></a>Griff IC, Reed RR (1995) The genetic basis for specific anosmia to isovaleric acid in the mouse. Cell  83: 407–414.  <ul class="find" data-citedArticleID="972308" data-doi="10.1016/0092-8674(95)90118-3"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90118-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+basis+for+specific+anosmia+to+isovaleric+acid+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+basis+for+specific+anosmia+to+isovaleric+acid+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000682-OConnell1" id="pone.0000682-OConnell1"></a>O'Connell RJ, Stevens DA, Zogby LM (1994) Individual differences in the perceived intensity and quality of specific odors following self- and cross-adaptation. Chem Senses  19: 197–208.  <ul class="find" data-citedArticleID="972342" data-doi="10.1093/chemse/19.3.197"><li><a href="http://dx.doi.org/10.1093/chemse/19.3.197" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Individual+differences+in+the+perceived+intensity+and+quality+of+specific+odors+following+self-+and+cross-adaptation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Individual+differences+in+the+perceived+intensity+and+quality+of+specific+odors+following+self-+and+cross-adaptation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000682-Lancet1" id="pone.0000682-Lancet1"></a>Lancet D, Sadovsky E, Seidemann E (1993) Probability model for molecular recognition in biological receptor repertoires: significance to the olfactory system. Proc Natl Acad Sci U S A  90: 3715–3719.  <ul class="find" data-citedArticleID="972318" data-doi="10.1073/pnas.90.8.3715"><li><a href="http://dx.doi.org/10.1073/pnas.90.8.3715" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probability+model+for+molecular+recognition+in+biological+receptor+repertoires%3A+significance+to+the+olfactory+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probability+model+for+molecular+recognition+in+biological+receptor+repertoires%3A+significance+to+the+olfactory+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000682-Malnic1" id="pone.0000682-Malnic1"></a>Malnic B, Hirono J, Sato T, Buck LB (1999) Combinatorial receptor codes for odors. Cell  96: 713–723.  <ul class="find" data-citedArticleID="972326" data-doi="10.1016/s0092-8674(00)80581-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80581-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Combinatorial+receptor+codes+for+odors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Combinatorial+receptor+codes+for+odors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000682-Lapidot1" id="pone.0000682-Lapidot1"></a>Lapidot M, Pilpel Y, Gilad Y, Falcovitz A, Sharon D, et al.  (2001) Mouse-human orthology relationships in an olfactory receptor gene cluster. Genomics  71: 296–306.  <ul class="find" data-citedArticleID="972320" data-doi="10.1006/geno.2000.6431"><li><a href="http://dx.doi.org/10.1006/geno.2000.6431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mouse-human+orthology+relationships+in+an+olfactory+receptor+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mouse-human+orthology+relationships+in+an+olfactory+receptor+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000682-Kajiya1" id="pone.0000682-Kajiya1"></a>Kajiya K, Inaki K, Tanaka M, Haga T, Kataoka H, et al.  (2001) Molecular bases of odor discrimination: Reconstitution of olfactory receptors that recognize overlapping sets of odorants. J Neurosci  21: 6018–6025.  <ul class="find" data-citedArticleID="972310"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+bases+of+odor+discrimination%3A+Reconstitution+of+olfactory+receptors+that+recognize+overlapping+sets+of+odorants.&amp;auth=&amp;atitle=Molecular+bases+of+odor+discrimination%3A+Reconstitution+of+olfactory+receptors+that+recognize+overlapping+sets+of+odorants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+bases+of+odor+discrimination%3A+Reconstitution+of+olfactory+receptors+that+recognize+overlapping+sets+of+odorants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+bases+of+odor+discrimination%3A+Reconstitution+of+olfactory+receptors+that+recognize+overlapping+sets+of+odorants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000682-Krautwurst1" id="pone.0000682-Krautwurst1"></a>Krautwurst D, Yau KW, Reed RR (1998) Identification of ligands for olfactory receptors by functional expression of a receptor library. Cell  95: 917–926.  <ul class="find" data-citedArticleID="972316" data-doi="10.1016/s0092-8674(00)81716-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81716-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+ligands+for+olfactory+receptors+by+functional+expression+of+a+receptor+library." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+ligands+for+olfactory+receptors+by+functional+expression+of+a+receptor+library.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000682-Mombaerts2" id="pone.0000682-Mombaerts2"></a>Mombaerts P (2004) Genes and ligands for odorant, vomeronasal and taste receptors. Nat Rev Neurosci  5: 263–278.  <ul class="find" data-citedArticleID="972336" data-doi="10.1038/nrn1365"><li><a href="http://dx.doi.org/10.1038/nrn1365" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+and+ligands+for+odorant%2C+vomeronasal+and+taste+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+and+ligands+for+odorant%2C+vomeronasal+and+taste+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000682-Mirny1" id="pone.0000682-Mirny1"></a>Mirny LA, Gelfand MS (2002) Using orthologous and paralogous proteins to identify specificity-determining residues in bacterial transcription factors. J Mol Biol  321: 7–20.  <ul class="find" data-citedArticleID="972332" data-doi="10.1016/s0022-2836(02)00587-9"><li><a href="http://dx.doi.org/10.1016/s0022-2836(02)00587-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+orthologous+and+paralogous+proteins+to+identify+specificity-determining+residues+in+bacterial+transcription+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+orthologous+and+paralogous+proteins+to+identify+specificity-determining+residues+in+bacterial+transcription+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000682-Li1" id="pone.0000682-Li1"></a>Li L, Shakhnovich EI, Mirny LA (2003) Amino acids determining enzyme-substrate specificity in prokaryotic and eukaryotic protein kinases. Proc Natl Acad Sci U S A  100: 4463–4468.  <ul class="find" data-citedArticleID="972324" data-doi="10.1073/pnas.0737647100"><li><a href="http://dx.doi.org/10.1073/pnas.0737647100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acids+determining+enzyme-substrate+specificity+in+prokaryotic+and+eukaryotic+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acids+determining+enzyme-substrate+specificity+in+prokaryotic+and+eukaryotic+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000682-Bozza1" id="pone.0000682-Bozza1"></a>Bozza T, Feinstein P, Zheng C, Mombaerts P (2002) Odorant receptor expression defines functional units in the mouse olfactory system. J Neurosci  22: 3033–3043.  <ul class="find" data-citedArticleID="972286"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Odorant+receptor+expression+defines+functional+units+in+the+mouse+olfactory+system.&amp;auth=&amp;atitle=Odorant+receptor+expression+defines+functional+units+in+the+mouse+olfactory+system." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Odorant+receptor+expression+defines+functional+units+in+the+mouse+olfactory+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Odorant+receptor+expression+defines+functional+units+in+the+mouse+olfactory+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000682-Palczewski1" id="pone.0000682-Palczewski1"></a>Palczewski K, Kumasaka T, Hori T, Behnke CA, Motoshima H, et al.  (2000) Crystal structure of rhodopsin: A G protein-coupled receptor. Science  289: 739–745.  <ul class="find" data-citedArticleID="972348" data-doi="10.1016/s0002-9394(00)00841-2"><li><a href="http://dx.doi.org/10.1016/s0002-9394(00)00841-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+rhodopsin%3A+A+G+protein-coupled+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+rhodopsin%3A+A+G+protein-coupled+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000682-Chenna1" id="pone.0000682-Chenna1"></a>Chenna R, Sugawara H, Koike T, Lopez R, Gibson TJ, et al.  (2003) Multiple sequence alignment with the Clustal series of programs. Nucleic Acids Res  31: 3497–3500.  <ul class="find" data-citedArticleID="972290" data-doi="10.1093/nar/gkg500"><li><a href="http://dx.doi.org/10.1093/nar/gkg500" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+sequence+alignment+with+the+Clustal+series+of+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+sequence+alignment+with+the+Clustal+series+of+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000682-Katada1" id="pone.0000682-Katada1"></a>Katada S, Hirokawa T, Oka Y, Suwa M, Touhara K (2005) Structural basis for a broad but selective ligand spectrum of a mouse olfactory receptor: mapping the odorant-binding site. J Neurosci  25: 1806–1815.  <ul class="find" data-citedArticleID="972312" data-doi="10.1523/jneurosci.4723-04.2005"><li><a href="http://dx.doi.org/10.1523/jneurosci.4723-04.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+a+broad+but+selective+ligand+spectrum+of+a+mouse+olfactory+receptor%3A+mapping+the+odorant-binding+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+a+broad+but+selective+ligand+spectrum+of+a+mouse+olfactory+receptor%3A+mapping+the+odorant-binding+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000682-Miller1" id="pone.0000682-Miller1"></a>Miller MP, Kumar S (2001) Understanding human disease mutations through the use of interspecific genetic variation. Hum Mol Genet  10: 2319–2328.  <ul class="find" data-citedArticleID="972330" data-doi="10.1093/hmg/10.21.2319"><li><a href="http://dx.doi.org/10.1093/hmg/10.21.2319" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+human+disease+mutations+through+the+use+of+interspecific+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+human+disease+mutations+through+the+use+of+interspecific+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000682-Kawashima1" id="pone.0000682-Kawashima1"></a>Kawashima S, Ogata H, Kanehisa M (1999) AAindex: Amino Acid Index Database. Nucleic Acids Res  27: 368–369.  <ul class="find" data-citedArticleID="972314" data-doi="10.1093/nar/27.1.368"><li><a href="http://dx.doi.org/10.1093/nar/27.1.368" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=AAindex%3A+Amino+Acid+Index+Database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22AAindex%3A+Amino+Acid+Index+Database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000682-Salvador1" id="pone.0000682-Salvador1"></a>Salvador S, Chan P (2004) Determining the Number of Clusters/Segments in Hierarchical Clustering/Segmentation Algorithms;. pp. 576–584.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.XML" value="74880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.PDF" value="272007"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g001.PNG_L" value="758784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g001.PNG_M" value="120873"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g001.PNG_S" value="9684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g001.TIF" value="1249156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g001.PNG_I" value="41435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g002.PNG_L" value="1096554"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g002.PNG_M" value="107721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g002.PNG_S" value="11095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g002.TIF" value="1698476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g002.PNG_I" value="58534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g003.PNG_L" value="753829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g003.PNG_M" value="81192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g003.PNG_S" value="14848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g003.TIF" value="1387018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.g003.PNG_I" value="96117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000682.s001.DOC" value="52224"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000682&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000682" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000682&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000682&volume=&issue=&title=A Framework for Exploring Functional Variability in Olfactory Receptor Genes&author_name=Orna%20Man%2C%20David%20C.%20Willhite%2C%20Chiquito%20J.%20Crasto%2C%20Gordon%20M.%20Shepherd%2C%20Yoav%20Gilad&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000682" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682&amp;t=A%20Framework%20for%20Exploring%20Functional%20Variability%20in%20Olfactory%20Receptor%20Genes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682&title=A%20Framework%20for%20Exploring%20Functional%20Variability%20in%20Olfactory%20Receptor%20Genes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682&amp;title=A%20Framework%20for%20Exploring%20Functional%20Variability%20in%20Olfactory%20Receptor%20Genes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000682&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'A Framework for Exploring Functional Variability in Olfactory Receptor Genes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000682';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20A%20Framework%20for%20Exploring%20Functional%20Variability%20in%20Olfactory%20Receptor%20Genes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000682" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000682" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binding+analysis%22" title="Search for articles in the subject area:'Binding analysis'"><div class="flagText">Binding analysis</div></a>
              <div data-categoryid="19597" data-articleid="24202"
                   data-categoryname="Binding analysis"
                   class="flagImage" title="Flag 'Binding analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Multiple+alignment+calculation%22" title="Search for articles in the subject area:'Multiple alignment calculation'"><div class="flagText">Multiple alignment calculation</div></a>
              <div data-categoryid="20427" data-articleid="24202"
                   data-categoryname="Multiple alignment calculation"
                   class="flagImage" title="Flag 'Multiple alignment calculation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Odorants%22" title="Search for articles in the subject area:'Odorants'"><div class="flagText">Odorants</div></a>
              <div data-categoryid="18097" data-articleid="24202"
                   data-categoryname="Odorants"
                   class="flagImage" title="Flag 'Odorants' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24202"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure+prediction%22" title="Search for articles in the subject area:'Protein structure prediction'"><div class="flagText">Protein structure prediction</div></a>
              <div data-categoryid="33203" data-articleid="24202"
                   data-categoryname="Protein structure prediction"
                   class="flagImage" title="Flag 'Protein structure prediction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pseudogenes%22" title="Search for articles in the subject area:'Pseudogenes'"><div class="flagText">Pseudogenes</div></a>
              <div data-categoryid="33795" data-articleid="24202"
                   data-categoryname="Pseudogenes"
                   class="flagImage" title="Flag 'Pseudogenes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24202"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="24202"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1336'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1716'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1936&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
