

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001597"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001597" />

    <meta name="citation_title" content="Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases"/>
    <meta itemprop="name" content="Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases"/>

      <meta name="citation_author" content="Kristine M. Briedis"/>
            <meta name="citation_author_institution" content="Bioinformatics Program, University of California San Diego, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Ayelet Starr"/>
            <meta name="citation_author_institution" content="Department of Bioengineering, University of California San Diego, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Philip E. Bourne"/>
            <meta name="citation_author_institution" content="Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California San Diego, La Jolla, California, United States of America"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001597.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1597"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The impact of structural genomics: expectations and outcomes.; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Science; citation_volume=311; citation_number=1; citation_pages=347-351; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Twilight zone of protein sequence alignments.; citation_author=B Rost; citation_journal_title=Protein Eng; citation_volume=12; citation_number=2; citation_pages=85-94; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=An integrated approach to the analysis and modeling of protein sequences and structures. II. On the relationship between sequence and structural similarity for proteins that are not obviously related in sequence.; citation_author=AS Yang; citation_author=B Honig; citation_journal_title=J Mol Biol; citation_volume=301; citation_number=3; citation_pages=679-689; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Assessing sequence comparison methods with reliable structurally identified distant evolutionary relationships.; citation_author=SE Brenner; citation_author=C Chothia; citation_author=TJ Hubbard; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=4; citation_pages=6073-6078; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Structure of an enzyme required for aminoglycoside antibiotic resistance reveals homology to eukaryotic protein kinases.; citation_author=WC Hon; citation_author=GA McKay; citation_author=PR Thompson; citation_author=RM Sweet; citation_author=DS Yang; citation_journal_title=Cell; citation_volume=89; citation_number=5; citation_pages=887-895; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evolutionary link between glycogen phosphorylase and a DNA modifying enzyme.; citation_author=L Holm; citation_author=C Sander; citation_journal_title=Embo J; citation_volume=14; citation_number=6; citation_pages=1287-1293; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Inferring protein function from structure.; citation_author=GJ Bartlett; citation_author=AE Todd; citation_author=JM Thornton; citation_journal_title=Methods Biochem Anal; citation_volume=44; citation_number=7; citation_pages=387-407; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A comparative proteomics resource: proteins of Arabidopsis thaliana.; citation_author=WW Li; citation_author=GB Quinn; citation_author=NN Alexandrov; citation_author=PE Bourne; citation_author=IN Shindyalov; citation_journal_title=Genome Biol; citation_volume=4; citation_number=8; citation_pages=R51; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Protein Data Bank.; citation_author=HM Berman; citation_author=J Westbrook; citation_author=Z Feng; citation_author=G Gilliland; citation_author=TN Bhat; citation_journal_title=Nucleic Acids Res; citation_volume=28; citation_number=9; citation_pages=235-242; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=SCOP: a structural classification of proteins database for the investigation of sequences and structures.; citation_author=AG Murzin; citation_author=SE Brenner; citation_author=T Hubbard; citation_author=C Chothia; citation_journal_title=J Mol Biol; citation_volume=247; citation_number=10; citation_pages=536-540; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=PDP: protein domain parser.; citation_author=N Alexandrov; citation_author=I Shindyalov; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=11; citation_pages=429-430; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=WU-Blast2 server at the European Bioinformatics Institute.; citation_author=R Lopez; citation_author=V Silventoinen; citation_author=S Robinson; citation_author=A Kibria; citation_author=W Gish; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=12; citation_pages=3795-3798; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Iterated profile searches with PSI-BLAST–a tool for discovery in protein databases.; citation_author=SF Altschul; citation_author=EV Koonin; citation_journal_title=Trends Biochem Sci; citation_volume=23; citation_number=13; citation_pages=444-447; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Fast protein fold recognition via sequence to structure alignment and contact capacity potentials.; citation_author=NN Alexandrov; citation_author=R Nussinov; citation_author=RM Zimmer; citation_journal_title=Pac Symp Biocomput; citation_number=14; citation_pages=53-72; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The protein kinase complement of the human genome.; citation_author=G Manning; citation_author=DB Whyte; citation_author=R Martinez; citation_author=T Hunter; citation_author=S Sudarsanam; citation_journal_title=Science; citation_volume=298; citation_number=15; citation_pages=1912-1934; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A comprehensive update of the sequence and structure classification of kinases.; citation_author=S Cheek; citation_author=K Ginalski; citation_author=H Zhang; citation_author=NV Grishin; citation_journal_title=BMC Struct Biol; citation_volume=5; citation_number=16; citation_pages=6; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2006.; citation_author=E Birney; citation_author=D Andrews; citation_author=M Caccamo; citation_author=Y Chen; citation_author=L Clarke; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=17; citation_pages=D556-561; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Protein kinases 6. The eukaryotic protein kinase superfamily: kinase (catalytic) domain structure and classification.; citation_author=SK Hanks; citation_author=T Hunter; citation_journal_title=Faseb J; citation_volume=9; citation_number=18; citation_pages=576-596; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The Pfam protein families database.; citation_author=A Bateman; citation_author=L Coin; citation_author=R Durbin; citation_author=RD Finn; citation_author=V Hollich; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=19; citation_pages=D138-141; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt): an expanding universe of protein information.; citation_author=CH Wu; citation_author=R Apweiler; citation_author=A Bairoch; citation_author=DA Natale; citation_author=WC Barker; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=20; citation_pages=D187-191; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=21; citation_pages=D5-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cloning and characterization of a human cDNA ACAD10 mapped to chromosome 12q24.1.; citation_author=X Ye; citation_author=C Ji; citation_author=C Zhou; citation_author=L Zeng; citation_author=S Gu; citation_journal_title=Mol Biol Rep; citation_volume=31; citation_number=22; citation_pages=191-195; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Phosphotransferase sequence homology.; citation_author=S Brenner; citation_journal_title=Nature; citation_volume=329; citation_number=23; citation_pages=21; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Identification of critical residues of choline kinase A2 from Caenorhabditis elegans.; citation_author=C Yuan; citation_author=C Kent; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=24; citation_pages=17801-17809; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural Evolution of the Protein Kinase-Like Superfamily.; citation_author=ED Scheeff; citation_author=PE Bourne; citation_journal_title=PLoS Comput Biol; citation_volume=1; citation_number=25; citation_pages=e49; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Assignment of homology to genome sequences using a library of hidden Markov models that represent all proteins of known structure.; citation_author=J Gough; citation_author=K Karplus; citation_author=R Hughey; citation_author=C Chothia; citation_journal_title=J Mol Biol; citation_volume=313; citation_number=26; citation_pages=903-919; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Structure and function of choline kinase isoforms in mammalian cells.; citation_author=C Aoyama; citation_author=H Liao; citation_author=K Ishidate; citation_journal_title=Prog Lipid Res; citation_volume=43; citation_number=27; citation_pages=266-281; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular causes of the aberrant choline phospholipid metabolism in breast cancer.; citation_author=K Glunde; citation_author=C Jie; citation_author=ZM Bhujwalla; citation_journal_title=Cancer Res; citation_volume=64; citation_number=28; citation_pages=4270-4276; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Choline phospholipid metabolism: a target in cancer cells?; citation_author=E Ackerstaff; citation_author=K Glunde; citation_author=ZM Bhujwalla; citation_journal_title=J Cell Biochem; citation_volume=90; citation_number=29; citation_pages=525-533; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Aminoglycoside antibiotic phosphotransferases are also serine protein kinases.; citation_author=DM Daigle; citation_author=GA McKay; citation_author=PR Thompson; citation_author=GD Wright; citation_journal_title=Chem Biol; citation_volume=6; citation_number=30; citation_pages=11-18; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Did protein kinase regulatory mechanisms evolve through elaboration of a simple structural component?; citation_author=N Kannan; citation_author=AF Neuwald; citation_journal_title=J Mol Biol; citation_volume=351; citation_number=31; citation_pages=956-972; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases"/>
    <meta name="twitter:description" content="BackgroundProtein sequence similarity is a commonly used criterion for inferring the unknown function of a protein from a protein of known function. However, proteins can diverge significantly over time such that sequence similarity is difficult, if not impossible, to find. In some cases, a structural similarity remains over long evolutionary time scales and once detected can be used to predict function.Methodology/Principal FindingsHere we employed a high-throughput approach to assign structural and functional annotation to the human proteome, focusing on the collection of human protein kinases, the human kinome. We compared human protein sequences to a library of domains from known structures using WU-BLAST, PSI-BLAST, and 123D. This approach utilized both sequence comparison and fold recognition methods. The resulting set of potential protein kinases was cross-checked against previously identified human protein kinases, and analyzed for conserved kinase motifs.Conclusions/SignificanceWe demonstrate that our structure-based method can be used to identify both typical and atypical human protein kinases. We also identify two potentially novel kinases that contain an interesting combination of kinase and acyl-CoA dehydrogenase domains."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001597.g003"/>

  <meta property="og:title" content="Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1273'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4321'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5040&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001597">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001597" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001597&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kristine M. Briedis
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Kristine M. Briedis, Ayelet Starr</p>

              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:kbriedis@ucsd.edu">kbriedis@ucsd.edu</a></p>

                <p>Affiliation:
                  Bioinformatics Program, University of California San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ayelet Starr
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Kristine M. Briedis, Ayelet Starr</p>

              
              

                <p>Affiliation:
                  Department of Bioengineering, University of California San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Philip E. Bourne
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California San Diego, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001597</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001597-g001" data-doi="info:doi/10.1371/journal.pone.0001597" data-uri="info:doi/10.1371/journal.pone.0001597.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001597-g002" data-doi="info:doi/10.1371/journal.pone.0001597" data-uri="info:doi/10.1371/journal.pone.0001597.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001597-g003" data-doi="info:doi/10.1371/journal.pone.0001597" data-uri="info:doi/10.1371/journal.pone.0001597.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001597">Reader Comments (6)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001597" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Protein sequence similarity is a commonly used criterion for inferring the unknown function of a protein from a protein of known function. However, proteins can diverge significantly over time such that sequence similarity is difficult, if not impossible, to find. In some cases, a structural similarity remains over long evolutionary time scales and once detected can be used to predict function.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we employed a high-throughput approach to assign structural and functional annotation to the human proteome, focusing on the collection of human protein kinases, the human kinome. We compared human protein sequences to a library of domains from known structures using WU-BLAST, PSI-BLAST, and 123D. This approach utilized both sequence comparison and fold recognition methods. The resulting set of potential protein kinases was cross-checked against previously identified human protein kinases, and analyzed for conserved kinase motifs.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We demonstrate that our structure-based method can be used to identify both typical and atypical human protein kinases. We also identify two potentially novel kinases that contain an interesting combination of kinase and acyl-CoA dehydrogenase domains.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Briedis KM, Starr A, Bourne PE (2008) Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases. PLoS ONE 3(2):
          e1597.
            doi:10.1371/journal.pone.0001597</p><p><strong>Academic Editor: </strong>Bostjan Kobe, University of Queensland, Australia</p><p><strong>Received:</strong> March 26, 2007; <strong>Accepted:</strong> December 20, 2007; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Briedis et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported in part by NIH GM63208.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Most proteome-wide functional annotation focuses on sequence similarity, however, this ignores valuable information that protein structure can provide–an important consideration in the era of structural genomics when many more protein structures are becoming available <a href="#pone.0001597-Chandonia1">[1]</a>. In some cases, the sequence between two proteins has diverged too far to find any significant sequence similarity with current methods, but a structural similarity can still be seen <a href="#pone.0001597-Rost1">[2]</a>–<a href="#pone.0001597-Brenner1">[4]</a>. For example, Hon <em>et al.</em> crystallized the aminoglycoside phosphotransferase APH(3′)-IIIa and found a surprising homology to eukaryotic protein kinases (ePKs) <a href="#pone.0001597-Hon1">[5]</a>. About half of the sequence folded into a structure typical of ePKs, despite a very low sequence identity. The major structural differences were found in the area of the protein that determined substrate specificity <a href="#pone.0001597-Hon1">[5]</a>. Likewise, Holm and Sander found two glucosyltransferases that shared less than 10% sequence identity, but still contained strong structural similarities that indicated evolutionary relatedness <a href="#pone.0001597-Holm1">[6]</a>. These two examples illustrate that the structures of proteins can reveal surprising similarities that are undetected by sequence identity alone. Notwithstanding, one must be cautious in assigning relatedness based on structural similarity alone. It is possible for two proteins with a similar structure to function in different ways. For example, lysozyme and α-lactalbumin have similar structures and a 40% sequence identity, but differ in function <a href="#pone.0001597-Bartlett1">[7]</a>. It is also possible for proteins to arrive at a similar structure through convergent rather than divergent evolution. Subtilisin and chymotrypsin are serine endopeptidases that share a catalytic triad, but no other sequence or fold similarity <a href="#pone.0001597-Bartlett1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>We have established a high-throughput approach to provide accurate structure and functional annotation termed the Encyclopedia of Life (EOL) <a href="#pone.0001597-Li1">[8]</a>, based on the desire to annotate a large number of sequenced proteomes. EOL uses a pipeline approach termed the integrated Genome Annotation Pipeline (iGAP), which we have applied in examining the set of human kinases, the human kinome, in an attempt to uncover distant homologs not previously seen.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>iGAP (<a href="#pone-0001597-g001"><strong>Figure 1</strong></a><strong>)</strong> compares already identified protein sequences from whole proteomes against a comprehensive structure fold library (FOLDLIB). The fold library was built from a combination of Protein Data Bank (PDB) protein chains <a href="#pone.0001597-Berman1">[9]</a> and protein domains defined by SCOP <a href="#pone.0001597-Murzin1">[10]</a> and PDP <a href="#pone.0001597-Alexandrov1">[11]</a>. SCOP domain sequences were filtered at 90% identity. Since there is a delay between protein structures being added to the PDB and classified by SCOP, PDB chains were clustered at 90% identity, parsed with PDP, and added to the SCOP domains to generate a more complete library. The collection of SCOP, PDP and PDB sequences were then clustered at 90% identity to determine the final FOLDLIB composition <a href="#pone.0001597-Li1">[8]</a>.</p>
<div class="figure" id="pone-0001597-g001"><div class="img"><a name="pone-0001597-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001597" data-uri="info:doi/10.1371/journal.pone.0001597.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001597.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001597.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>iGAP annotation pipeline. Diagram of the iGAP pipeline.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Protein sequences are compared to a domain library using WU-BLAST, PSI-BLAST, and 123D.</p>
<span>doi:10.1371/journal.pone.0001597.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The core of the pipeline consists of tools that search for sequence and fold similarity, including the sequence comparison programs WU-BLAST <a href="#pone.0001597-Lopez1">[12]</a> and PSI-BLAST <a href="#pone.0001597-Altschul1">[13]</a>, and the threading program 123D <a href="#pone.0001597-Li1">[8]</a>, <a href="#pone.0001597-Alexandrov2">[14]</a>. Protein sequences from completed proteomes were first compared to FOLDLIB using WU-BLAST. Then, PSI-BLAST profiles were generated for each input protein sequence using three iterations and a default H-value of 1e-06. Lastly, the protein sequences were compared to FOLDLIB using the fold recognition program 123D <a href="#pone.0001597-Li1">[8]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The result is a set of putative structure and function assignments including a novel statistical measure of reliability (Shindyalov <em>et al.</em> unpublished). Reliability is defined using a consensus approach with SCOP as a benchmark. Using a test set of non-redundant SCOP folds, Shindyalov <em>et al.</em> counted the number of consistently and inconsistently predicted assignments by WU-BLAST for each target sequence. The hits were binned by E-value and the specificity was averaged over all values in the bin, resulting in a reliability assignment. Reliability is defined as the number of positions with consistent predictions divided by the total number of positions having two or more hits to the same SCOP fold.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Using this method, it is found that the probability of traditional E-value assignments being correct varies between proteomes since they are not random, and indeed are not random in different ways. For example, using WU-BLAST to assign SCOP folds to proteomes, to reach a level of 1 error per 1000 annotations, one must use an E-value cutoff of 1×10<sup>−8</sup> for <em>Arabidopsis thaliana</em> but only 1×10<sup>−2</sup> for <em>Caenorhabditis elegans</em>. EOL individually benchmarks every genome and assigns a reliability index that can be used to compare different genomes. The reliability index is set by determining the E-values required for a sequence to be consistently identified with a fold and binning the hits by E-value. The resultant reliability index is termed A through E and corresponds to 99.9%, 99%, 90%, 50%, and 10% specificity, respectively <a href="#pone.0001597-Li1">[8]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>We utilized this pipeline to characterize the collection of human protein kinases. Eukaryotic protein kinases (ePKs) regulate signal transduction reactions in the cell, influencing many processes including metabolism, apoptosis and transcription <a href="#pone.0001597-Manning1">[15]</a>.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>The collection of kinases has previously been defined by several groups including Cheek <em>et al</em>. <a href="#pone.0001597-Cheek1">[16]</a> and Manning <em>et al</em>. <a href="#pone.0001597-Manning1">[15]</a>. Cheek <em>et al.</em> searched multiple species for all enzymes that catalyze the transfer of an ATP terminal phosphate group, while Manning <em>et al</em>. focused on both typical and atypical protein kinases in humans. Atypical protein kinases (aPKs) were defined by Manning <em>et al.</em> as proteins that have weak sequence similarity to the ePKs, but still have protein kinase activity.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>Since our study focuses on the human protein kinase superfamily, we compared our results with that of Manning <em>et al.</em> <a href="#pone.0001597-Manning1">[15]</a>. They published the “complete” human kinome paper in 2002 based on homologies detected using Hidden Markov Models (HMMs). HMMs were developed by Manning <em>et al.</em> for the ePK family and the PIKK, RIO, ABC1, PDK, and alpha kinase atypical families. The HMMs were used to search against Genbank, SwissProt, dbEST, Celera human genome, Incyte LifeSeqGold, and internal SUGEN and Pharmacia sequence databases. Full-length gene predictions were determined for putative kinase hits, and confirmed in most cases by cDNA cloning <a href="#pone.0001597-Manning1">[15]</a>.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>Our approach differs in several ways. By including the threading program 123D, we incorporate fold recognition along with sequence similarity, possibly leading to the identification of more distant homologs. We also searched Ensembl's <a href="#pone.0001597-Birney1">[17]</a> draft assembly 34 v19.34.a.1 of the human genome, which presumably differs from the genome draft used by Manning <em>et al.</em> in 2001–2002.</p>
<a id="article1.body1.sec1.p11" name="article1.body1.sec1.p11"></a><p>Utilizing Hidden Markov Models (HMMs) along with EST and cDNA data, Manning <em>et al.</em> found 518 human protein kinases. This accounts for almost 2% of all human genes, and makes protein kinases one of the largest eukaryotic gene families <a href="#pone.0001597-Manning1">[15]</a>. Most human kinases contain a eukaryotic protein kinase (ePK) catalytic domain. This catalytic domain shows remarkable conservation, specifically with respect to critical residues and motifs, as previously described by Hanks and Hunter <a href="#pone.0001597-Hanks1">[18]</a>. However, the HMM method employed by Manning <em>et al.</em> is only one approach to identifying specific protein families across a whole proteome. We thus compared the human kinome as classified by the EOL pipeline to the Manning set. We determined that our method performs well in classifying the kinome and we present here two putative novel kinases.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>Overall, the human kinase set identified here by EOL agreed with the set of kinases found by Manning <em>et al.</em> In addition, we analyzed 153 potential novel protein kinase sequences (selected as described in the <a href="#s3">Methods</a> section) using Pfam <a href="#pone.0001597-Bateman1">[19]</a> and found 44 contained an assignment for either an ePK or atypical kinase domain. Based on these Pfam results, our sequences were classified into the following groups (followed by the sequence count in parentheses): choline/ethanolamine kinase (5), fructosamine kinase (2), protein kinase (20), PI3_PI4 kinase (17) and not kinase (109) (See Supplementary <a href="#pone.0001597.s001">Tables S1</a> and <a href="#pone.0001597.s002">S2</a> for data).</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>Most of the differences between our human kinome and that found by Manning <em>et al.</em> can be attributed to analyzing a different draft of the human genome. Only one kinase exists in both Manning <em>et al.'s</em> human kinome and our Ensembl human genome draft that EOL did not identify (LRRK2 UniProt:Q5S2007 <a href="#pone.0001597-Wu1">[20]</a>). Upon further investigation, it was discovered that the Ensembl LRRK2 protein was only 400 amino acids long in our draft, and was missing the protein kinase domain. Ensembl lengthened the LRRK2 sequence in a subsequent draft to 2527 amino acids, including the protein kinase domain. Ten other protein kinases from the Manning <em>et al.</em> kinome match proteins in our set at a lower score than our cutoff for mapping, probably due to using slightly different gene predictions and data sets. These ten proteins, upon closer inspection, were manually mapped to the Manning <em>et al.</em> kinome. For example, ENSP00000330379 has a 98% local sequence identity to EphA10 in Manning <em>et al.'s</em> human kinome, but is 462 amino acids shorter. It is annotated in Ensembl as EphA10 precursor. The ten proteins, along with reasons for their poor mapping, are described in further detail in Supplementary <a href="#pone.0001597.s003">Table S3</a>.</p>
<a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>Some of the kinases identified by EOL are from protein families that are part of the protein kinase-like SCOP superfamily (d.144.1), but are not classified in the “protein kinases, catalytic subunit” family. This includes the atypical kinase families actin-fragmin kinases, MHCK/EF2 kinases, phosphoinositide 3-kinases, choline kinases, aminoglycoside phosphotransferases, and the RIO1-like kinases <a href="#pone.0001597-Murzin1">[10]</a>. Some of these EOL kinases were not present in Manning <em>et al.'s</em> set, but were already deposited and identified in NCBI's <a href="#pone.0001597-Wheeler1">[21]</a> Non-Redundant database (NR) as kinases. In an effort to pinpoint the source of differences, we looked at the methods used by Manning <em>et al.</em> to classify the sequences. Manning <em>et al.'s</em> paper states that they developed Hidden Markov Models (HMMs) for some of the atypical families, including PIKK, RIO, ABC1, PDK, and alpha kinase <a href="#pone.0001597-Manning1">[15]</a>. In comparison, the EOL search included only those atypical kinases present in the ePK superfamily, as defined by SCOP (d.144.1) <a href="#pone.0001597-Murzin1">[10]</a>. Thus, it is not surprising that the EOL human kinome contains a different set of atypical kinases than Manning <em>et al.'s</em> kinome. For example, to the best of our knowledge Manning's group did not build an HMM to look for choline/ethanolamine kinases. EOL's human kinome, however, correctly classified five such proteins (SCOP family d.144.1.8) in the human proteome.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>Here we focus on two particularly interesting potential kinases that were classified by Ensembl as acyl-CoA dehydrogenase family members. A BLAST search against NR showed these proteins to be ACAD10 [UniProt: Q6JQN1; Ensembl: ENSP00000325137] and ACAD11 [UniProt:Q709F0; Ensembl:ENSP00000264990].</p>
<a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>ACAD10 has been previously identified as being involved in the β-oxidation of fatty acids <a href="#pone.0001597-Ye1">[22]</a>. EOL recognized the acyl-CoA dehydrogenase domain, but also assigned a kinase domain as part of the sequence. 123D produced the strongest kinase hit, with a weaker hit from PSI-BLAST. <a href="#pone-0001597-g002"><strong>Figure 2</strong></a> shows an alignment of the protein to common kinase motifs. Clearly, the nucleotide position loop and Brenner's phosphotransferase motif <a href="#pone.0001597-Brenner2">[23]</a> are well conserved. Less well conserved is the choline kinase motif. It is interesting to note, however, that some of the most critical functional residues of choline kinases as identified by Yuan <em>et al.</em> are conserved <a href="#pone.0001597-Yuan1">[24]</a>.</p>
<div class="figure" id="pone-0001597-g002"><div class="img"><a name="pone-0001597-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001597" data-uri="info:doi/10.1371/journal.pone.0001597.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001597.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001597.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Conserved kinase motifs in ACAD10 and ACAD11.</span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>ACAD10 and ACAD11 contain conserved kinase motifs such as the nucleotide position loop, a phosphotransferase motif, and part of a choline kinase motif. Residues in pink are highly conserved; residues in green are commonly large hydrophobic amino acids. ACAD10 and ACAD11 are aligned with the choline kinase 1NW1 and aminoglycoside phosphotransferases 1ND4 and 1J7L for comparison.</p>
<span>doi:10.1371/journal.pone.0001597.g002</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>ACAD11 is 279 amino acids shorter than ACAD10, and has a similar arrangement of acyl-CoA dehydrogenase and kinase domains. The difference in length is mostly attributable to a hydrolase domain that is present in ACAD10, but not ACAD11. A BLAST alignment between ACAD10 and ACAD11 shared a 46% sequence identity overall (excluding the hydrolase domain), and a 48% sequence identity in the kinase domain. At the time of our initial study, the protein corresponding to ACAD11 in Ensembl did not contain a kinase domain. However, it has since been lengthened in a subsequent release and appears to contain a kinase domain with similar features to ACAD10, as shown in <a href="#pone-0001597-g002"><strong>Figure 2</strong></a> (see Supplementary <a href="#pone.0001597.s004">Figure S1</a> for a longer alignment).</p>
<a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>The kinase domains of ACAD10 and ACAD11 appear to be most similar to a choline kinase or an aminoglycoside phosphotransferase (APH) domain. The similarity between the APH and choline kinase families was previously noted by Scheeff and Bourne <a href="#pone.0001597-Scheeff1">[25]</a> in a study of the structural evolution of the protein kinase-like superfamily. Structural analysis revealed conservation in their C-terminal subdomains that was not observed to exist in other kinase families <a href="#pone.0001597-Scheeff1">[25]</a>. EOL, Pfam, and Superfamily <a href="#pone.0001597-Gough1">[26]</a> annotate the protein kinase domains of ACAD10 and ACAD11 as an APH domain with higher confidence than a choline kinase domain, however, the aforementioned similarities to the choline kinase motif are intriguing.</p>
<a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>Choline kinases phosphorylate choline to produce phophocholine <a href="#pone.0001597-Aoyama1">[27]</a>. This pathway eventually produces phosphatidylcholine, a component of cell membranes. Choline kinase is a particularly important atypical kinase as it has been shown to play a role in several types of cancer. Over-activity of choline kinase and increased concentrations of phosphocholine have been identified in breast cancer cells <a href="#pone.0001597-Glunde1">[28]</a>. Increased phosphocholine levels have also been reported in prostate and brain tumors <a href="#pone.0001597-Ackerstaff1">[29]</a>.</p>
<a id="article1.body1.sec2.p9" name="article1.body1.sec2.p9"></a><p>Aminoglycoside phosphotransferases (APHs) are also an interesting atypical kinase family, present in bacteria. As previously mentioned, Hon <em>et al.</em> revealed a surprising structural similarity between APH and eukaryotic protein kinases (ePK) <a href="#pone.0001597-Hon1">[5]</a>. APHs have been implicated in antibiotic resistance. They phosphorylate aminoglycoside hydroxyl groups. In bacteria this can result in inactivation of aminoglycoside antibiotics such as kanamycin and gentamicin. However, APHs have also been shown to phosphorylate some ePK substrates. Daigle <em>et al.</em> demonstrated that two APHs had the ability to phosphorylate some Ser/Thr protein kinase substrates, though at a slower rate than aminoglycoside phosphorylation <a href="#pone.0001597-Daigle1">[30]</a>. This could perhaps offer an explanation as to how a kinase domain with similarities to APHs would function in eukaryotes.</p>
<a id="article1.body1.sec2.p10" name="article1.body1.sec2.p10"></a><p>The domain arrangement of ACAD10 shown in <a href="#pone-0001597-g003"><strong>Figure 3</strong></a> was the only human protein identified as such in the Superfamily database <a href="#pone.0001597-Gough1">[26]</a>. Superfamily and Pfam found proteins with the same domain structure in <em>Mus musculus</em> (mouse), <em>Caenorhabditis elegans</em> (worm), <em>Caenorhabditis briggsae</em> (worm), <em>Bos taurus</em> (cow), <em>Ciona intestinalis</em> (sea squirt), <em>Macaca mulatta</em> (rhesus monkey), <em>Monodelphis domestica</em> (opossum) and <em>Pan troglodytes</em> (chimp) <a href="#pone.0001597-Bateman1">[19]</a>, <a href="#pone.0001597-Gough1">[26]</a>. Similar proteins also exist in bacteria <a href="#pone.0001597-Kannan1">[31]</a>.</p>
<div class="figure" id="pone-0001597-g003"><div class="img"><a name="pone-0001597-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001597" data-uri="info:doi/10.1371/journal.pone.0001597.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001597.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001597.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001597.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001597.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001597.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Domains identified in ACAD10.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>iGAP identified hydrolase, kinase, and acyl-CoA dehydrogenase domains in ACAD10.</p>
<span>doi:10.1371/journal.pone.0001597.g003</span></div><a id="article1.body1.sec2.p11" name="article1.body1.sec2.p11"></a><p>In conclusion, we have utilized both sequence and structure-based tools to annotate the human kinome. We were successful in identifying both ePK and atypical kinases. We were particularly intrigued by ACAD10 and ACAD11, which contain acyl-CoA dehydrogenase and apparent kinase domains. The cellular function of such a combination of domains and the level of kinase activity for these proteins remains to be determined.</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Assembly 34 v19.34.a.1 of the Ensembl <a href="#pone.0001597-Birney1">[17]</a> human genome draft was run through iGAP <a href="#pone.0001597-Li1">[8]</a>, including WU-BLAST <a href="#pone.0001597-Lopez1">[12]</a>, PSI-BLAST <a href="#pone.0001597-Altschul1">[13]</a>, and the threading program 123D <a href="#pone.0001597-Alexandrov2">[14]</a>. The subset predicted to contain the protein kinase superfamily was selected for further study. Protein kinase domains are generally 250–300 amino acids in length <a href="#pone.0001597-Hanks1">[18]</a>. Thus, our set of candidate proteins was filtered to exclude near-identical sequences and those shorter than 200 amino acids to exclude proteins that despite a short sequence or structural similarity cannot contain a full, active kinase domain. Since it was unknown at the beginning of the study how sensitive iGAP would be in identifying full kinase domains, we selected proteins with a predicted kinase domain of 120 amino acids (roughly half the length of a typical protein kinase domain) or greater for further study. To ensure we didn't miss any abnormally short kinases, we also included any proteins that did not meet the above criteria, but appeared to contain at least two conserved subdomains from Hanks and Hunter's ePK domain analysis <a href="#pone.0001597-Hanks1">[18]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The proteins found were mapped to the Manning <em>et al.'s</em> human kinome using BLAST at a 90% sequence identity cutoff point. This strict threshold was set so proteins were not erroneously mapped to each other. However, it was done with the understanding that given human genome draft changes, some proteins may fall below this identity threshold that should be considered equivalent to each other.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Of the remaining 324 potentially unique proteins, 234 were selected that matched to a kinase domain by 123D, in hopes of exploiting any distant structural similarities that would be overlooked when considering sequence alone. Many of these predictions were at a lower reliability and were deemed false positives. These false positives likely share some structural, but not functional, similarity to the kinase fold. Sequences of “A” or “B” reliability were analyzed for conserved kinase domain motifs and blasted against NCBI's NR database <a href="#pone.0001597-Wheeler1">[21]</a>. Including the aforementioned sequences that contained Hanks and Hunter ePK subdomains <a href="#pone.0001597-Hanks1">[18]</a>, our final data set consisted of 153 sequences (<a href="#pone.0001597.s001">Table S1</a>).</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001597"></div><a name="pone.0001597.s001" id="pone.0001597.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001597.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Set of possible protein kinases, as identified by EOL. 153 proteins identified by EOL as possible protein kinases were analyzed in depth for conserved kinase motifs. This table contains the Ensembl id, the reliability of the kinase match, the method that determined the match, the PDB ID of the kinase matched, and author comments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001597.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB XLS)</p>
<a name="pone.0001597.s002" id="pone.0001597.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001597.s002">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">20 Protein Kinase Matches. This table contains details regarding the 20 protein kinase Pfam matches. The table includes the Ensembl ID, the closest protein kinase mapping, and author comments.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001597.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001597.s003" id="pone.0001597.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001597.s003">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Reclassified Ensembl Proteins. This table describes Ensembl IDs that showed poor mapping to Sugen kinases and had to be manually verified. The table contains the Ensembl ID, the kinase name, and the reason why the protein had to be mapped manually.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001597.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001597.s004" id="pone.0001597.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001597.s004">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Kinase Domain Alignment of APH and Choline Kinase Proteins. This is a ClustalW alignment of sample choline kinase and aminoglycoside phosphotransferase domains with ACAD10 and ACAD11. Accession numbers are included in sequence names.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001597.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Eric Scheeff for valuable discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: PB. Performed the experiments: KB AS. Analyzed the data: KB AS. Wrote the paper: KB AS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001597-Chandonia1" id="pone.0001597-Chandonia1"></a>Chandonia JM, Brenner SE (2006) The impact of structural genomics: expectations and outcomes. Science  311: 347–351.  <ul class="find" data-citedArticleID="1055660" data-doi="10.1126/science.1121018"><li><a href="http://dx.doi.org/10.1126/science.1121018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+impact+of+structural+genomics%3A+expectations+and+outcomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+impact+of+structural+genomics%3A+expectations+and+outcomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001597-Rost1" id="pone.0001597-Rost1"></a>Rost B (1999) Twilight zone of protein sequence alignments. Protein Eng  12: 85–94.  <ul class="find" data-citedArticleID="1055686" data-doi="10.1093/protein/12.2.85"><li><a href="http://dx.doi.org/10.1093/protein/12.2.85" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Twilight+zone+of+protein+sequence+alignments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Twilight+zone+of+protein+sequence+alignments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001597-Yang1" id="pone.0001597-Yang1"></a>Yang AS, Honig B (2000) An integrated approach to the analysis and modeling of protein sequences and structures. II. On the relationship between sequence and structural similarity for proteins that are not obviously related in sequence. J Mol Biol  301: 679–689.  <ul class="find" data-citedArticleID="1055694" data-doi="10.1006/jmbi.2000.3974"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3974" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrated+approach+to+the+analysis+and+modeling+of+protein+sequences+and+structures.+II.+On+the+relationship+between+sequence+and+structural+similarity+for+proteins+that+are+not+obviously+related+in+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrated+approach+to+the+analysis+and+modeling+of+protein+sequences+and+structures.+II.+On+the+relationship+between+sequence+and+structural+similarity+for+proteins+that+are+not+obviously+related+in+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001597-Brenner1" id="pone.0001597-Brenner1"></a>Brenner SE, Chothia C, Hubbard TJ (1998) Assessing sequence comparison methods with reliable structurally identified distant evolutionary relationships. Proc Natl Acad Sci U S A  95: 6073–6078.  <ul class="find" data-citedArticleID="1055656" data-doi="10.1073/pnas.95.11.6073"><li><a href="http://dx.doi.org/10.1073/pnas.95.11.6073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assessing+sequence+comparison+methods+with+reliable+structurally+identified+distant+evolutionary+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assessing+sequence+comparison+methods+with+reliable+structurally+identified+distant+evolutionary+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001597-Hon1" id="pone.0001597-Hon1"></a>Hon WC, McKay GA, Thompson PR, Sweet RM, Yang DS, et al.  (1997) Structure of an enzyme required for aminoglycoside antibiotic resistance reveals homology to eukaryotic protein kinases. Cell  89: 887–895.  <ul class="find" data-citedArticleID="1055674" data-doi="10.1016/s0092-8674(00)80274-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80274-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+of+an+enzyme+required+for+aminoglycoside+antibiotic+resistance+reveals+homology+to+eukaryotic+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+of+an+enzyme+required+for+aminoglycoside+antibiotic+resistance+reveals+homology+to+eukaryotic+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001597-Holm1" id="pone.0001597-Holm1"></a>Holm L, Sander C (1995) Evolutionary link between glycogen phosphorylase and a DNA modifying enzyme. Embo J  14: 1287–1293.  <ul class="find" data-citedArticleID="1055672"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Evolutionary+link+between+glycogen+phosphorylase+and+a+DNA+modifying+enzyme.&amp;auth=&amp;atitle=Evolutionary+link+between+glycogen+phosphorylase+and+a+DNA+modifying+enzyme." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+link+between+glycogen+phosphorylase+and+a+DNA+modifying+enzyme." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+link+between+glycogen+phosphorylase+and+a+DNA+modifying+enzyme.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001597-Bartlett1" id="pone.0001597-Bartlett1"></a>Bartlett GJ, Todd AE, Thornton JM (2003) Inferring protein function from structure. Methods Biochem Anal  44: 387–407.  <ul class="find" data-citedArticleID="1055648" data-doi="10.1002/0471721204.ch19"><li><a href="http://dx.doi.org/10.1002/0471721204.ch19" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+protein+function+from+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+protein+function+from+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001597-Li1" id="pone.0001597-Li1"></a>Li WW, Quinn GB, Alexandrov NN, Bourne PE, Shindyalov IN (2003) A comparative proteomics resource: proteins of Arabidopsis thaliana. Genome Biol  4: R51.  <ul class="find" data-citedArticleID="1055678" data-doi="10.1186/gb-2003-4-8-r51"><li><a href="http://dx.doi.org/10.1186/gb-2003-4-8-r51" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparative+proteomics+resource%3A+proteins+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparative+proteomics+resource%3A+proteins+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001597-Berman1" id="pone.0001597-Berman1"></a>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, et al.  (2000) The Protein Data Bank. Nucleic Acids Res  28: 235–242.  <ul class="find" data-citedArticleID="1055652" data-doi="10.1093/nar/28.1.235"><li><a href="http://dx.doi.org/10.1093/nar/28.1.235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Protein+Data+Bank." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Protein+Data+Bank.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001597-Murzin1" id="pone.0001597-Murzin1"></a>Murzin AG, Brenner SE, Hubbard T, Chothia C (1995) SCOP: a structural classification of proteins database for the investigation of sequences and structures. J Mol Biol  247: 536–540.  <ul class="find" data-citedArticleID="1055684" data-doi="10.1016/s0022-2836(05)80134-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80134-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SCOP%3A+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001597-Alexandrov1" id="pone.0001597-Alexandrov1"></a>Alexandrov N, Shindyalov I (2003) PDP: protein domain parser. Bioinformatics  19: 429–430.  <ul class="find" data-citedArticleID="1055640" data-doi="10.1093/bioinformatics/btg006"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PDP%3A+protein+domain+parser." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PDP%3A+protein+domain+parser.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001597-Lopez1" id="pone.0001597-Lopez1"></a>Lopez R, Silventoinen V, Robinson S, Kibria A, Gish W (2003) WU-Blast2 server at the European Bioinformatics Institute. Nucleic Acids Res  31: 3795–3798.  <ul class="find" data-citedArticleID="1055680" data-doi="10.1093/nar/gkg573"><li><a href="http://dx.doi.org/10.1093/nar/gkg573" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WU-Blast2+server+at+the+European+Bioinformatics+Institute." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WU-Blast2+server+at+the+European+Bioinformatics+Institute.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001597-Altschul1" id="pone.0001597-Altschul1"></a>Altschul SF, Koonin EV (1998) Iterated profile searches with PSI-BLAST–a tool for discovery in protein databases. Trends Biochem Sci  23: 444–447.  <ul class="find" data-citedArticleID="1055644" data-doi="10.1016/s0968-0004(98)01298-5"><li><a href="http://dx.doi.org/10.1016/s0968-0004(98)01298-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iterated+profile+searches+with+PSI-BLAST%E2%80%93a+tool+for+discovery+in+protein+databases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iterated+profile+searches+with+PSI-BLAST%E2%80%93a+tool+for+discovery+in+protein+databases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001597-Alexandrov2" id="pone.0001597-Alexandrov2"></a>Alexandrov NN, Nussinov R, Zimmer RM (1996) Fast protein fold recognition via sequence to structure alignment and contact capacity potentials. Pac Symp Biocomput 53–72.  <ul class="find" data-citedArticleID="1055642"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fast+protein+fold+recognition+via+sequence+to+structure+alignment+and+contact+capacity+potentials.&amp;auth=&amp;atitle=Fast+protein+fold+recognition+via+sequence+to+structure+alignment+and+contact+capacity+potentials." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+protein+fold+recognition+via+sequence+to+structure+alignment+and+contact+capacity+potentials." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+protein+fold+recognition+via+sequence+to+structure+alignment+and+contact+capacity+potentials.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001597-Manning1" id="pone.0001597-Manning1"></a>Manning G, Whyte DB, Martinez R, Hunter T, Sudarsanam S (2002) The protein kinase complement of the human genome. Science  298: 1912–1934.  <ul class="find" data-citedArticleID="1055682" data-doi="10.1126/science.1075762"><li><a href="http://dx.doi.org/10.1126/science.1075762" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+kinase+complement+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+kinase+complement+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001597-Cheek1" id="pone.0001597-Cheek1"></a>Cheek S, Ginalski K, Zhang H, Grishin NV (2005) A comprehensive update of the sequence and structure classification of kinases. BMC Struct Biol  5: 6.  <ul class="find" data-citedArticleID="1055662" data-doi="10.1186/1472-6807-5-6"><li><a href="http://dx.doi.org/10.1186/1472-6807-5-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comprehensive+update+of+the+sequence+and+structure+classification+of+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comprehensive+update+of+the+sequence+and+structure+classification+of+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001597-Birney1" id="pone.0001597-Birney1"></a>Birney E, Andrews D, Caccamo M, Chen Y, Clarke L, et al.  (2006) Ensembl 2006. Nucleic Acids Res  34: D556–561.  <ul class="find" data-citedArticleID="1055654" data-doi="10.1093/nar/gkj133"><li><a href="http://dx.doi.org/10.1093/nar/gkj133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2006." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2006.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001597-Hanks1" id="pone.0001597-Hanks1"></a>Hanks SK, Hunter T (1995) Protein kinases 6. The eukaryotic protein kinase superfamily: kinase (catalytic) domain structure and classification. Faseb J  9: 576–596.  <ul class="find" data-citedArticleID="1055670" data-doi="10.1016/b978-012324719-3/50003-0"><li><a href="http://dx.doi.org/10.1016/b978-012324719-3/50003-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+kinases+6.+The+eukaryotic+protein+kinase+superfamily%3A+kinase+%28catalytic%29+domain+structure+and+classification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+kinases+6.+The+eukaryotic+protein+kinase+superfamily%3A+kinase+%28catalytic%29+domain+structure+and+classification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001597-Bateman1" id="pone.0001597-Bateman1"></a>Bateman A, Coin L, Durbin R, Finn RD, Hollich V, et al.  (2004) The Pfam protein families database. Nucleic Acids Res  32: D138–141.  <ul class="find" data-citedArticleID="1055650" data-doi="10.1093/nar/gkh121"><li><a href="http://dx.doi.org/10.1093/nar/gkh121" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pfam+protein+families+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pfam+protein+families+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001597-Wu1" id="pone.0001597-Wu1"></a>Wu CH, Apweiler R, Bairoch A, Natale DA, Barker WC, et al.  (2006) The Universal Protein Resource (UniProt): an expanding universe of protein information. Nucleic Acids Res  34: D187–191.  <ul class="find" data-citedArticleID="1055692" data-doi="10.1093/nar/gkj161"><li><a href="http://dx.doi.org/10.1093/nar/gkj161" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Universal+Protein+Resource+%28UniProt%29%3A+an+expanding+universe+of+protein+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Universal+Protein+Resource+%28UniProt%29%3A+an+expanding+universe+of+protein+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001597-Wheeler1" id="pone.0001597-Wheeler1"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2007) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  35: D5–12.  <ul class="find" data-citedArticleID="1055690" data-doi="10.1093/nar/gkl1031"><li><a href="http://dx.doi.org/10.1093/nar/gkl1031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001597-Ye1" id="pone.0001597-Ye1"></a>Ye X, Ji C, Zhou C, Zeng L, Gu S, et al.  (2004) Cloning and characterization of a human cDNA ACAD10 mapped to chromosome 12q24.1. Mol Biol Rep  31: 191–195.  <ul class="find" data-citedArticleID="1055696" data-doi="10.1023/b:mole.0000043622.57408.6b"><li><a href="http://dx.doi.org/10.1023/b:mole.0000043622.57408.6b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+characterization+of+a+human+cDNA+ACAD10+mapped+to+chromosome+12q24.1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+characterization+of+a+human+cDNA+ACAD10+mapped+to+chromosome+12q24.1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001597-Brenner2" id="pone.0001597-Brenner2"></a>Brenner S (1987) Phosphotransferase sequence homology. Nature  329: 21.  <ul class="find" data-citedArticleID="1055658" data-doi="10.1038/329021a0"><li><a href="http://dx.doi.org/10.1038/329021a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphotransferase+sequence+homology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphotransferase+sequence+homology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001597-Yuan1" id="pone.0001597-Yuan1"></a>Yuan C, Kent C (2004) Identification of critical residues of choline kinase A2 from Caenorhabditis elegans. J Biol Chem  279: 17801–17809.  <ul class="find" data-citedArticleID="1055698" data-doi="10.1074/jbc.m401382200"><li><a href="http://dx.doi.org/10.1074/jbc.m401382200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+critical+residues+of+choline+kinase+A2+from+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+critical+residues+of+choline+kinase+A2+from+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001597-Scheeff1" id="pone.0001597-Scheeff1"></a>Scheeff ED, Bourne PE (2005) Structural Evolution of the Protein Kinase-Like Superfamily. PLoS Comput Biol  1: e49.  <ul class="find" data-citedArticleID="1055688" data-doi="10.1371/journal.pcbi.0010049.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0010049.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+Evolution+of+the+Protein+Kinase-Like+Superfamily." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+Evolution+of+the+Protein+Kinase-Like+Superfamily.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001597-Gough1" id="pone.0001597-Gough1"></a>Gough J, Karplus K, Hughey R, Chothia C (2001) Assignment of homology to genome sequences using a library of hidden Markov models that represent all proteins of known structure. J Mol Biol  313: 903–919.  <ul class="find" data-citedArticleID="1055668" data-doi="10.1006/jmbi.2001.5080"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.5080" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assignment+of+homology+to+genome+sequences+using+a+library+of+hidden+Markov+models+that+represent+all+proteins+of+known+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assignment+of+homology+to+genome+sequences+using+a+library+of+hidden+Markov+models+that+represent+all+proteins+of+known+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001597-Aoyama1" id="pone.0001597-Aoyama1"></a>Aoyama C, Liao H, Ishidate K (2004) Structure and function of choline kinase isoforms in mammalian cells. Prog Lipid Res  43: 266–281.  <ul class="find" data-citedArticleID="1055646" data-doi="10.1016/j.plipres.2004.03.003"><li><a href="http://dx.doi.org/10.1016/j.plipres.2004.03.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+choline+kinase+isoforms+in+mammalian+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+choline+kinase+isoforms+in+mammalian+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001597-Glunde1" id="pone.0001597-Glunde1"></a>Glunde K, Jie C, Bhujwalla ZM (2004) Molecular causes of the aberrant choline phospholipid metabolism in breast cancer. Cancer Res  64: 4270–4276.  <ul class="find" data-citedArticleID="1055666" data-doi="10.1158/0008-5472.can-03-3829"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-03-3829" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+causes+of+the+aberrant+choline+phospholipid+metabolism+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+causes+of+the+aberrant+choline+phospholipid+metabolism+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001597-Ackerstaff1" id="pone.0001597-Ackerstaff1"></a>Ackerstaff E, Glunde K, Bhujwalla ZM (2003) Choline phospholipid metabolism: a target in cancer cells? J Cell Biochem  90: 525–533.  <ul class="find" data-citedArticleID="1055638" data-doi="10.1002/jcb.10659"><li><a href="http://dx.doi.org/10.1002/jcb.10659" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Choline+phospholipid+metabolism%3A+a+target+in+cancer+cells%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Choline+phospholipid+metabolism%3A+a+target+in+cancer+cells%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001597-Daigle1" id="pone.0001597-Daigle1"></a>Daigle DM, McKay GA, Thompson PR, Wright GD (1999) Aminoglycoside antibiotic phosphotransferases are also serine protein kinases. Chem Biol  6: 11–18.  <ul class="find" data-citedArticleID="1055664" data-doi="10.1016/s1074-5521(99)80016-7"><li><a href="http://dx.doi.org/10.1016/s1074-5521(99)80016-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aminoglycoside+antibiotic+phosphotransferases+are+also+serine+protein+kinases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aminoglycoside+antibiotic+phosphotransferases+are+also+serine+protein+kinases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001597-Kannan1" id="pone.0001597-Kannan1"></a>Kannan N, Neuwald AF (2005) Did protein kinase regulatory mechanisms evolve through elaboration of a simple structural component? J Mol Biol  351: 956–972.  <ul class="find" data-citedArticleID="1055676" data-doi="10.1016/j.jmb.2005.06.057"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.06.057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Did+protein+kinase+regulatory+mechanisms+evolve+through+elaboration+of+a+simple+structural+component%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Did+protein+kinase+regulatory+mechanisms+evolve+through+elaboration+of+a+simple+structural+component%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.XML" value="56782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.PDF" value="185676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g001.PNG_L" value="95330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g001.PNG_M" value="95729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g001.PNG_S" value="11352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g001.TIF" value="673088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g001.PNG_I" value="31181"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g002.PNG_L" value="50442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g002.PNG_M" value="146458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g002.PNG_S" value="15320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g002.TIF" value="395212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g002.PNG_I" value="74866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g003.PNG_L" value="20412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g003.PNG_M" value="15166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g003.PNG_S" value="6354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g003.TIF" value="67104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.g003.PNG_I" value="4653"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.s001.XLS" value="58880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.s002.XLS" value="23552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.s003.XLS" value="21504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001597.s004.DOC" value="37376"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001597&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001597" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001597&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001597&volume=&issue=&title=Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases&author_name=Kristine%20M.%20Briedis%2C%20Ayelet%20Starr%2C%20Philip%20E.%20Bourne&start_page=1&end_page=5" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001597" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597&amp;t=Analysis%20of%20the%20Human%20Kinome%20Using%20Methods%20Including%20Fold%20Recognition%20Reveals%20Two%20Novel%20Kinases" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597&title=Analysis%20of%20the%20Human%20Kinome%20Using%20Methods%20Including%20Fold%20Recognition%20Reveals%20Two%20Novel%20Kinases&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597&amp;title=Analysis%20of%20the%20Human%20Kinome%20Using%20Methods%20Including%20Fold%20Recognition%20Reveals%20Two%20Novel%20Kinases" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001597&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Analysis of the Human Kinome Using Methods Including Fold Recognition Reveals Two Novel Kinases';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001597';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Analysis%20of%20the%20Human%20Kinome%20Using%20Methods%20Including%20Fold%20Recognition%20Reveals%20Two%20Novel%20Kinases http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001597" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001597" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cholines%22" title="Search for articles in the subject area:'Cholines'"><div class="flagText">Cholines</div></a>
              <div data-categoryid="25585" data-articleid="26020"
                   data-categoryname="Cholines"
                   class="flagImage" title="Flag 'Cholines' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="26020"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hidden+Markov+models%22" title="Search for articles in the subject area:'Hidden Markov models'"><div class="flagText">Hidden Markov models</div></a>
              <div data-categoryid="33299" data-articleid="26020"
                   data-categoryname="Hidden Markov models"
                   class="flagImage" title="Flag 'Hidden Markov models' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+families%22" title="Search for articles in the subject area:'Human families'"><div class="flagText">Human families</div></a>
              <div data-categoryid="18723" data-articleid="26020"
                   data-categoryname="Human families"
                   class="flagImage" title="Flag 'Human families' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+structure%22" title="Search for articles in the subject area:'Protein structure'"><div class="flagText">Protein structure</div></a>
              <div data-categoryid="31877" data-articleid="26020"
                   data-categoryname="Protein structure"
                   class="flagImage" title="Flag 'Protein structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="26020"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="26020"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+similarity+searching%22" title="Search for articles in the subject area:'Sequence similarity searching'"><div class="flagText">Sequence similarity searching</div></a>
              <div data-categoryid="35177" data-articleid="26020"
                   data-categoryname="Sequence similarity searching"
                   class="flagImage" title="Flag 'Sequence similarity searching' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=836'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=7449'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9437&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9563">Maybe only 4 choline/ethanolamine kinases</a><br>Posted by Gerard_Manning</p>
      <p><a href="/annotation/listThread.action?root=17923">They are CAK family members</a><br>Posted by Gerard_Manning</p>
      <p><a href="/annotation/listThread.action?root=18547">Which?</a><br>Posted by Gerard_Manning</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
