

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000608"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000608" />

    <meta name="citation_title" content="Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase"/>
    <meta itemprop="name" content="Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase"/>

      <meta name="citation_author" content="Olivier Delelis"/>
            <meta name="citation_author_institution" content="LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France"/>
      <meta name="citation_author" content="Vincent Parissi"/>
            <meta name="citation_author_institution" content="Laboratoire REGER, IFR 66 “Pathologies infectieuses et cancer”, Bordeaux, France"/>
      <meta name="citation_author" content="Hervé Leh"/>
            <meta name="citation_author_institution" content="LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France"/>
      <meta name="citation_author" content="Gladys Mbemba"/>
            <meta name="citation_author_institution" content="LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France"/>
      <meta name="citation_author" content="Caroline Petit"/>
            <meta name="citation_author_institution" content="Génétique des Virus, Département des Maladies Infectieuses, Institut Cochin, INSERM U567, CNRS UMR8104, Université René Descartes, Paris, France"/>
      <meta name="citation_author" content="Pierre Sonigo"/>
            <meta name="citation_author_institution" content="Génétique des Virus, Département des Maladies Infectieuses, Institut Cochin, INSERM U567, CNRS UMR8104, Université René Descartes, Paris, France"/>
      <meta name="citation_author" content="Eric Deprez"/>
            <meta name="citation_author_institution" content="LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France"/>
      <meta name="citation_author" content="Jean-François Mouscadet"/>
            <meta name="citation_author_institution" content="LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France"/>

    <meta name="citation_date" content="2007/7/11"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000608.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e608"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=HIV integrase, a brief overview from chemistry to therapeutics.; citation_author=R Craigie; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=1; citation_pages=23213-23216; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Retroviral DNA integration–mechanism and consequences.; citation_author=MK Lewinski; citation_author=FD Bushman; citation_journal_title=Adv Genet; citation_volume=55; citation_number=2; citation_pages=147-181; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Processing of viral DNA ends channels the HIV-1 integration reaction to concerted integration.; citation_author=M Li; citation_author=R Craigie; citation_journal_title=J Biol Chem; citation_volume=280; citation_number=3; citation_pages=29334-29339; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Retroviral DNA integration: reaction pathway and critical intermediates.; citation_author=M Li; citation_author=M Mizuuchi; citation_author=TR Burke; citation_author=R Craigie; citation_journal_title=EMBO J; citation_volume=25; citation_number=4; citation_pages=1295-1304; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recombinant human immunodeficiency virus type 1 integrase exhibits a capacity for full-site integration in vitro that is comparable to that of purified preintegration complexes from virus-infected cells.; citation_author=S Sinha; citation_author=DP Grandgenett; citation_journal_title=J Virol; citation_volume=79; citation_number=5; citation_pages=8208-8216; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Probing of HIV-1 integrase/DNA interactions using novel analogs of viral DNA.; citation_author=J Agapkina; citation_author=M Smolov; citation_author=S Barbe; citation_author=E Zubin; citation_author=T Zatsepin; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=6; citation_pages=11530-11540; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Sequence specificity of viral end DNA binding by HIV-1 integrase reveals critical regions for protein-DNA interaction.; citation_author=D Esposito; citation_author=R Craigie; citation_journal_title=EMBO J; citation_volume=17; citation_number=7; citation_pages=5832-5843; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Human immunodeficiency virus integrase protein requires a subterminal position of its viral DNA recognition sequence for efficient cleavage.; citation_author=C Vink; citation_author=DC van Gent; citation_author=Y Elgersma; citation_author=RH Plasterk; citation_journal_title=J Virol; citation_volume=65; citation_number=8; citation_pages=4636-4644; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Retroviral DNA integration: viral and cellular determinants of target-site selection.; citation_author=MK Lewinski; citation_author=M Yamashita; citation_author=M Emerman; citation_author=A Ciuffi; citation_author=H Marshall; citation_journal_title=PLoS Pathog; citation_volume=2; citation_number=9; citation_pages=e60; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV-1 integrase crosslinked oligomers are active in vitro.; citation_author=A Faure; citation_author=C Calmels; citation_author=C Desjobert; citation_author=M Castroviejo; citation_author=A Caumont-Sarcos; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=10; citation_pages=977-986; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Relationship between the oligomeric status of HIV-1 integrase on DNA and enzymatic activity.; citation_author=E Guiot; citation_author=K Carayon; citation_author=O Delelis; citation_author=F Simon; citation_author=P Tauc; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=11; citation_pages=22707-22719; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Weak palindromic consensus sequences are a common feature found at the integration target sites of many retroviruses.; citation_author=X Wu; citation_author=Y Li; citation_author=B Crise; citation_author=SM Burgess; citation_author=DJ Munroe; citation_journal_title=J Virol; citation_volume=79; citation_number=12; citation_pages=5211-5214; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Symmetrical base preferences surrounding HIV-1, avian sarcoma/leukosis virus, and murine leukemia virus integration sites.; citation_author=AG Holman; citation_author=JM Coffin; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=13; citation_pages=6103-6107; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Symmetrical recognition of cellular DNA target sequences during retroviral integration.; citation_author=DP Grandgenett; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=14; citation_pages=5903-5904; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Effect of polypurine tract (PPT) mutations on human immunodeficiency virus type 1 replication: a virus with a completely randomized PPT retains low infectivity.; citation_author=LR Miles; citation_author=BE Agresta; citation_author=MB Khan; citation_author=S Tang; citation_author=JG Levin; citation_journal_title=J Virol; citation_volume=79; citation_number=15; citation_pages=6859-6867; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Azido-containing diketo acid derivatives inhibit human immunodeficiency virus type 1 integrase in vivo and influence the frequency of deletions at two-long-terminal-repeat-circle junctions.; citation_author=ES Svarovskaia; citation_author=R Barr; citation_author=X Zhang; citation_author=GC Pais; citation_author=C Marchand; citation_journal_title=J Virol; citation_volume=78; citation_number=16; citation_pages=3210-3222; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Rearrangements in unintegrated retroviral DNA are complex and are the result of multiple genetic determinants.; citation_author=JC Olsen; citation_author=C Bova-Hill; citation_author=DP Grandgenett; citation_author=TP Quinn; citation_author=JP Manfredi; citation_journal_title=J Virol; citation_volume=64; citation_number=17; citation_pages=5475-5484; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Early detection of a two-long-terminal-repeat junction molecule in the cytoplasm of recombinant murine leukemia virus-infected cells.; citation_author=F Serhan; citation_author=M Penaud; citation_author=C Petit; citation_author=T Leste-Lasserre; citation_author=S Trajcevski; citation_journal_title=J Virol; citation_volume=78; citation_number=18; citation_pages=6190-6199; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Roles of host cell factors in circularization of retroviral dna.; citation_author=JM Kilzer; citation_author=T Stracker; citation_author=B Beitzel; citation_author=K Meek; citation_author=M Weitzman; citation_journal_title=Virology; citation_volume=314; citation_number=19; citation_pages=460-467; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Role of the non-homologous DNA end joining pathway in the early steps of retroviral infection.; citation_author=L Li; citation_author=JM Olvera; citation_author=KE Yoder; citation_author=RS Mitchell; citation_author=SL Butler; citation_journal_title=EMBO J; citation_volume=20; citation_number=20; citation_pages=3272-3281; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Determinants of Mg2+-dependent activities of recombinant human immunodeficiency virus type 1 integrase.; citation_author=H Leh; citation_author=P Brodin; citation_author=J Bischerour; citation_author=E Deprez; citation_author=P Tauc; citation_journal_title=Biochemistry; citation_volume=39; citation_number=21; citation_pages=9285-9294; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Mechanism of HIV-1 integrase inhibition by styrylquinoline derivatives in vitro.; citation_author=E Deprez; citation_author=S Barbe; citation_author=M Kolaski; citation_author=H Leh; citation_author=F Zouhiri; citation_journal_title=Mol Pharmacol; citation_volume=65; citation_number=22; citation_pages=85-98; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The core domain of HIV-1 integrase recognizes key features of its DNA substrates.; citation_author=JL Gerton; citation_author=PO Brown; citation_journal_title=J Biol Chem; citation_volume=272; citation_number=23; citation_pages=25809-25815; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=HIV-1 integrase catalytic core: molecular dynamics and simulated fluorescence decays.; citation_author=C Laboulais; citation_author=E Deprez; citation_author=H Leh; citation_author=JF Mouscadet; citation_author=JC Brochon; citation_journal_title=Biophys J; citation_volume=81; citation_number=24; citation_pages=473-489; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A novel function for spumaretrovirus integrase: an early requirement for integrase-mediated cleavage of 2 LTR circles.; citation_author=O Delelis; citation_author=C Petit; citation_author=H Leh; citation_author=G Mbemba; citation_author=JF Mouscadet; citation_journal_title=Retrovirology; citation_volume=2; citation_number=25; citation_pages=31; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Kinetic study of the HIV-1 DNA 3′-end processing.; citation_author=M Smolov; citation_author=M Gottikh; citation_author=V Tashlitskii; citation_author=S Korolev; citation_author=I Demidyuk; citation_journal_title=FEBS J; citation_volume=273; citation_number=26; citation_pages=1137-1151; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nonspecific alcoholysis, a novel endonuclease activity of human immunodeficiency virus type 1 and other retroviral integrases.; citation_author=M Katzman; citation_author=M Sudol; citation_journal_title=J Virol; citation_volume=70; citation_number=27; citation_pages=2598-2604; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mapping target site selection for the non-specific nuclease activities of retroviral integrase.; citation_author=M Katzman; citation_author=M Sudol; citation_author=JS Pufnock; citation_author=S Zeto; citation_author=LM Skinner; citation_journal_title=Virus Res; citation_volume=66; citation_number=28; citation_pages=87-100; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Nucleophile selection for the endonuclease activities of human, ovine, and avian retroviral integrases.; citation_author=LM Skinner; citation_author=M Sudol; citation_author=AL Harper; citation_author=M Katzman; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=29; citation_pages=114-124; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Mapping domains of retroviral integrase responsible for viral DNA specificity and target site selection by analysis of chimeras between human immunodeficiency virus type 1 and visna virus integrases.; citation_author=M Katzman; citation_author=M Sudol; citation_journal_title=J Virol; citation_volume=69; citation_number=30; citation_pages=5687-5696; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Importance of the conserved CA dinucleotide at Mu termini.; citation_author=I Lee; citation_author=RM Harshey; citation_journal_title=J Mol Biol; citation_volume=314; citation_number=31; citation_pages=433-444; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Invariant and variable base stacking geometries in B-DNA and A-DNA.; citation_author=S Neugebauerova; citation_author=J Kypr; citation_journal_title=J Biomol Struct Dyn; citation_volume=18; citation_number=32; citation_pages=73-81; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Circles with two tandem long terminal repeats are specifically cleaved by pol gene-associated endonuclease from avian sarcoma and leukosis viruses: nucleotide sequences required for site-specific cleavage.; citation_author=G Duyk; citation_author=M Longiaru; citation_author=D Cobrinik; citation_author=R Kowal; citation_author=P deHaseth; citation_journal_title=J Virol; citation_volume=56; citation_number=33; citation_pages=589-599; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Site-specific nicking at the avian retrovirus LTR circle junction by the viral pp32 DNA endonuclease.; citation_author=DP Grandgenett; citation_author=AC Vora; citation_journal_title=Nucleic Acids Res; citation_volume=13; citation_number=34; citation_pages=6205-6221; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Both substrate and target oligonucleotide sequences affect in vitro integration mediated by human immunodeficiency virus type 1 integrase protein produced in Saccharomyces cerevisiae.; citation_author=AD Leavitt; citation_author=RB Rose; citation_author=HE Varmus; citation_journal_title=J Virol; citation_volume=66; citation_number=35; citation_pages=2359-2368; citation_date=1992; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase"/>
    <meta name="twitter:description" content="BackgroundHIV-1 integrase (IN) catalyses the retroviral integration process, removing two nucleotides from each long terminal repeat and inserting the processed viral DNA into the target DNA. It is widely assumed that the strand transfer step has no sequence specificity. However, recently, it has been reported by several groups that integration sites display a preference for palindromic sequences, suggesting that a symmetry in the target DNA may stabilise the tetrameric organisation of IN in the synaptic complex.Methodology/Principal FindingsWe assessed the ability of several palindrome-containing sequences to organise tetrameric IN and investigated the ability of IN to catalyse DNA cleavage at internal positions. Only one palindromic sequence was successfully cleaved by IN. Interestingly, this symmetrical sequence corresponded to the 2-LTR junction of retroviral DNA circles&mdash;a palindrome similar but not identical to the consensus sequence found at integration sites. This reaction depended strictly on the cognate retroviral sequence of IN and required a full-length wild-type IN. Furthermore, the oligomeric state of IN responsible for this cleavage differed from that involved in the 3&prime;-processing reaction. Palindromic cleavage strictly required the tetrameric form, whereas 3&prime;-processing was efficiently catalysed by a dimer.Conclusions/SignificanceOur findings suggest that the restriction-like cleavage of palindromic sequences may be a general physiological activity of retroviral INs and that IN tetramerisation is strongly favoured by DNA symmetry, either at the target site for the concerted integration or when the DNA contains the 2-LTR junction in the case of the palindromic internal cleavage."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000608.g006"/>

  <meta property="og:title" content="Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6961'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=1395'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=1549&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000608">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000608" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000608&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Olivier Delelis
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:delelis@lbpa.ens-cachan.fr">delelis@lbpa.ens-cachan.fr</a></p>

                <p>Affiliation:
                  LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vincent Parissi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratoire REGER, IFR 66 “Pathologies infectieuses et cancer”, Bordeaux, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hervé Leh, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gladys Mbemba, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Caroline Petit, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génétique des Virus, Département des Maladies Infectieuses, Institut Cochin, INSERM U567, CNRS UMR8104, Université René Descartes, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pierre Sonigo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Génétique des Virus, Département des Maladies Infectieuses, Institut Cochin, INSERM U567, CNRS UMR8104, Université René Descartes, Paris, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eric Deprez, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean-François Mouscadet
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  LBPA, CNRS UMR8113, Ecole Normale Supérieure de Cachan, Cachan, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 11, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000608</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000608-g001" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000608-g002" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000608-g003" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000608-g004" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000608-g005" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000608-g006" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000608-t001" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000608">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000608" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>HIV-1 integrase (IN) catalyses the retroviral integration process, removing two nucleotides from each long terminal repeat and inserting the processed viral DNA into the target DNA. It is widely assumed that the strand transfer step has no sequence specificity. However, recently, it has been reported by several groups that integration sites display a preference for palindromic sequences, suggesting that a symmetry in the target DNA may stabilise the tetrameric organisation of IN in the synaptic complex.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We assessed the ability of several palindrome-containing sequences to organise tetrameric IN and investigated the ability of IN to catalyse DNA cleavage at internal positions. Only one palindromic sequence was successfully cleaved by IN. Interestingly, this symmetrical sequence corresponded to the 2-LTR junction of retroviral DNA circles—a palindrome similar but not identical to the consensus sequence found at integration sites. This reaction depended strictly on the cognate retroviral sequence of IN and required a full-length wild-type IN. Furthermore, the oligomeric state of IN responsible for this cleavage differed from that involved in the 3′-processing reaction. Palindromic cleavage strictly required the tetrameric form, whereas 3′-processing was efficiently catalysed by a dimer.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our findings suggest that the restriction-like cleavage of palindromic sequences may be a general physiological activity of retroviral INs and that IN tetramerisation is strongly favoured by DNA symmetry, either at the target site for the concerted integration or when the DNA contains the 2-LTR junction in the case of the palindromic internal cleavage.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Delelis O, Parissi V, Leh H, Mbemba G, Petit C, et al.  (2007) Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase. PLoS ONE 2(7):
          e608.
            doi:10.1371/journal.pone.0000608</p><p><strong>Academic Editor: </strong>Olivier Schwartz, Institut Pasteur, France</p><p><strong>Received:</strong> April 30, 2007; <strong>Accepted:</strong> June 12, 2007; <strong>Published:</strong> July 11, 2007</p><p><strong>Copyright:</strong> © 2007 Delelis et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the TrioH European project (FP6 grant 503480), the Centre National de la Recherche Scientifique, and the French National Agency for Research against AIDS (ANRS).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Once human immunodeficiency type 1 virus (HIV-1) enters the host cell, its genomic RNA is reverse transcribed to generate a double-stranded linear DNA that is subsequently covalently inserted into host cell chromosomes by integrase (IN). Complete integration involves two spatially and temporally distinct reactions. The first reaction, 3′-processing, occurs in the cytoplasm. The viral DNA is trimmed by IN, releasing a 3′ terminal dinucleotide from each viral DNA extremity, downstream from the canonical subterminal 5′-CA sequence. The second reaction, strand transfer, which takes place after the matured viral DNA has been translocated to the nucleus, results in covalent insertion of the viral DNA into chromosomal DNA. Both reactions require the physiological cofactor, Mg<sup>2+</sup> <a href="#pone.0000608-Craigie1">[1]</a>, <a href="#pone.0000608-Lewinski1">[2]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Both 3′-processing and strand transfer reactions can be modelled <em>in vitro</em> with the recombinant IN and short oligodeoxynucleotides (ODN) mimicking the extremities of the viral genome. The simultaneous presence in the same complex of the newly exposed 3′-OH extremities of the 3′ processed viral DNA, target DNA and IN allows complete integration to take place <em>in vitro</em> <a href="#pone.0000608-Li1">[3]</a>–<a href="#pone.0000608-Sinha1">[5]</a>. A specific viral sequence (<em>att</em>) is required for 3′-processing, as shown by the inefficiency of this process if the CA sequence is mutated or located more than two nucleotides away from the 3′ end of the DNA <a href="#pone.0000608-Agapkina1">[6]</a>–<a href="#pone.0000608-Vink1">[8]</a>. However, internal cleavage has been reported, but this endonucleolytic activity is not sequence-specific as it occurs independently of the 5′-CA dinucleotide and only in the presence of Mn<sup>2+</sup>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p><em>In vivo</em>, 3′-processing and strand transfer occur within the pre-integration complex, a large nucleoprotein complex comprising the viral IN bound to the short cognate sequences located at each end of the long terminal repeat (LTR). The duplicated sequence present at the integration site—the hallmark of retroviral integration—demonstrates the intrinsically symmetric nature of the concerted integration process <a href="#pone.0000608-Lewinski2">[9]</a>. The 3′-processing and strand transfer reactions require the formation of an IN oligomer. Both <em>in vitro</em> and cellular complementation studies have shown that single inactive IN mutants may combine to form a catalytically competent nucleoprotein, confirming the oligomeric nature of functional IN <a href="#pone.0000608-Craigie1">[1]</a>. Although IN dimers are competent for the 3′-processing reaction, tetramers (dimers of dimers) are required for coupled insertion of the two viral DNA ends <a href="#pone.0000608-Li1">[3]</a>, <a href="#pone.0000608-Faure1">[10]</a>, <a href="#pone.0000608-Guiot1">[11]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The 3′-processing reaction is strictly dependent on the presence of the <em>att</em> sequence. This is a highly specific reaction as it requires the presence of the CA dinucleotide, at position 3/4 from the DNA 3′-extremity (see sequence HIV38B in <a href="#pone-0000608-g001">Fig. 1B</a>), regardless the cationic cofactor, Mn<sup>2+</sup> or Mg<sup>2+</sup>. In the presence of Mg<sup>2+</sup>, more positions are involved in the reaction specificity (about the 10 terminal positions) <a href="#pone.0000608-Agapkina1">[6]</a>, <a href="#pone.0000608-Esposito1">[7]</a>. No such reaction specificity in term of target sequence has been found <em>in vitro</em> for the strand transfer reaction. However, large-scale sequencing of <em>ex vivo</em> integration sites have revealed a bias towards palindromic sequences, thus providing evidence that target are organised symmetrically rather than randomly <a href="#pone.0000608-Wu1">[12]</a>–<a href="#pone.0000608-Grandgenett1">[14]</a>. Thus, tetrameric IN may display significant specificity for symmetric sequences, and the recognition of such sequences in the target may favour the stabilisation of tetramers.</p>
<div class="figure" id="pone-0000608-g001"><div class="img"><a name="pone-0000608-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Sequences of retroviral LTR-LTR junctions and oligonucleotide substrates.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>A) The retroviral sequences found at the LTR-LTR junctions of 2-LTR circles are almost perfect palindromes <a href="#pone.0000608-Miles1">[15]</a>–<a href="#pone.0000608-Serhan1">[18]</a>. The bases underlined correspond to imperfect palindromic sequences. The 5′-CA dinucleotide (U5-LTR) or its complementary sequence GT (U3-LTR), essential for the 3′-processing reaction, is shown in bold. The vertical dashed line indicates the axis of symmetry. B) Summary of the various DNA substrates used for palindrome cleavage and 3′-processing reactions (only the top strands are shown). The mutations in the Mut1 and Mut2 sequences are underlined.</p>
<span>doi:10.1371/journal.pone.0000608.g001</span></div><a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We investigated several palindromic sequences, focusing on possible determinants promoting IN tetramerisation and on the ability of IN to cleave symmetric DNA sequences at internal positions. It is important to note that a symmetric sequence is also present at the LTR-LTR junction of covalently linked viral extremities. Interestingly, this sequence is similar, although not identical, to the palindromic consensus found at the integration sites <a href="#pone.0000608-Wu1">[12]</a>–<a href="#pone.0000608-Serhan1">[18]</a>. The palindromic sequence in the 2-LTR circles generated in retroviral replication junctions is a general feature of retroviruses (<a href="#pone-0000608-g001">Figure 1A</a>) <a href="#pone.0000608-Miles1">[15]</a>–<a href="#pone.0000608-Serhan1">[18]</a>. For HIV-1, all strains in infected cells yield in majority the LTR-LTR junction sequence shown in <a href="#pone-0000608-g001">Fig. 1A</a>. The percentage of these junctions in infected cells which matches perfectly with the expected junction sequence (deduced from the sequences of viral DNA extremities) is about 52–58%, the variability in the sequence mostly originates in deletion/insertion at the junction <a href="#pone.0000608-Miles1">[15]</a>, <a href="#pone.0000608-Svarovskaia1">[16]</a>. Two-LTR circles accumulate in cells when IN mutations or the use of strand transfer inhibitors impair integration. They are considered to be dead-end products originating from the cell-mediated ligation of linear viral DNA via the NHEJ pathway <a href="#pone.0000608-Kilzer1">[19]</a>, <a href="#pone.0000608-Li3">[20]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>We then tested either palindromic sequences derived from the preferential integration sites or the retroviral sequence present at the LTR-LTR junction. We investigated <em>in vitro</em> cleavage conditions and demonstrated, for the first time, that HIV-1 IN cleaves DNA at internal positions, in a sequence-specific manner with the same efficiency as compared to the 3′-processing reaction. This cleavage is symmetric and occurs only in the LTR-LTR palindromic sequence—it mainly occurs at the canonical CA positions on opposite strands and is restricted to the cognate substrate of HIV-1 IN. This activity differs from other IN activities, such as disintegration or 3′-processing, as it is restricted to the full-length tetrameric form of the protein.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Methods"></a><h3>Methods</h3>
<h4>Oligonucleotides and plasmids</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The sequences of the different ODN substrates are shown in <a href="#pone-0000608-g001">Figure 1B</a>. ODNs were purchased from Eurogentec (Liege, Belgium) and further purified by electrophoresis in a denaturing 18% acrylamide/urea gel. For activity assays, ODNs were radiolabelled with T4 polynucleotide kinase (Biolabs) and γ[<sup>32−</sup>P]ATP (3000 Ci/mmol) (Amersham), and purified on a Sephadex G-10 column (GE Healthcare). Double-stranded ODNs were obtained by mixing equimolar amounts of complementary strands in the presence of 100 mM NaCl. We constructed pJCT, a plasmid containing the LTR-LTR junction, as follows: The HIV-1 LTR-LTR junction was amplified from HIV-1-infected CEM cells using gag 5′-gaattcgcgcttcagcaagccgagtc and env 5′-gaattcacccaaaaggtcagtgtggagtcc primers. The 1440-bp DNA fragment containing the LTR-LTR junction was digested with <em>Eco</em>RI and inserted into pCR2.1-TOPO (3.9 kb) (Invitrogen), yielding pJCT.</p>


<h4>IN purification and activity assays</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The full-length HIV-1 WT IN, the 50-212 catalytic core domain (CC), the 1-212 two-domain protein (ΔC) and the E152A point mutant were prepared as previously described <a href="#pone.0000608-Leh1">[21]</a>. IN activity assays—3′-processing, disintegration and internal cleavage—were carried out at 37°C, in a buffer containing 10 mM HEPES (pH 7.2), 1 mM DTT, 7.5 mM MgCl<sub>2</sub> or MnCl<sub>2</sub> in the presence of 12.5 nM DNA substrate. Products were separated by electrophoresis in denaturing 18% acrylamide/urea gels. Gels were analysed with a Molecular Dynamics STORM phosphoimager and quantified with Image Quant™ 4.1 software. For palindrome cleavage, <em>Sca</em>I was used as a restriction control. <em>Sca</em>I recognises the <span class="gene">5′-AGTACT</span> sequence and specifically cleaves after the 5′-GT.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>For plasmid cleavage assays, pJCT containing the HIV LTR-LTR junction were incubated with 3 µM crosslinked IN oligomers for four hours, under conditions similar to those described for 3′-processing. Cleavage products were then analysed by electrophoresis in 1% agarose gels, with detection by BET staining. Crosslinked IN oligomers were produced and purified as previously described <a href="#pone.0000608-Faure1">[10]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>All experiments (activity and cross-linking experiments) were performed several times (up to five times each). Results were reproducible and each figure (<a href="#pone-0000608-g002">Fig. 2</a>–<a href="#pone-0000608-g003"></a><a href="#pone-0000608-g004"></a><a href="#pone-0000608-g005"></a><a href="#pone-0000608-g006">6</a>) displays one representative experiment.</p>
<div class="figure" id="pone-0000608-g002"><div class="img"><a name="pone-0000608-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Endonucleolytic activity of HIV-1 IN on the palindromic U5-U3 junction.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A) IN was incubated with 12.5 nM of 38-mer PalA/PalB duplex mimicking the HIV-1 palindromic U5-U3 junction in the presence of 7.5 mM divalent cation, for 2 h at 37°C. The 38-mer duplex was radiolabelled either on the 5′-extremity of the PalB ODN (top strand) (lanes 1-7) or on the 5′-extremity of the PalA ODN (bottom strand) (lanes 8–10). The metallic cofactor was Mg<sup>2+</sup> (lanes 2–4, 9–10) or Mn<sup>2+</sup> (lanes 5–7). Lanes 3, 6 and 9: 1.5 µM IN; Lanes 2, 5 and 10: 3 µM IN. Lanes 4 and 7: 3 µM IN + 1 mM EDTA. Lanes 1 and 8: PalA/PalB DNA substrate digested with <em>Sca</em>I. B) Relative cleavage efficiencies for the various DNA positions in the HIV-1 palindromic junction and 3′-processing substrate. The relative cleavage efficiency corresponds to a ratio between cleavage occurring at one position and total IN cleavage activity. This ratio is directly related to the specificity of the cleavage.</p>
<span>doi:10.1371/journal.pone.0000608.g002</span></div><div class="figure" id="pone-0000608-g003"><div class="img"><a name="pone-0000608-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>HIV-1 IN is highly specific for its cognate palindromic sequence and fails to cleave PVF and mutated HIV-1 palindromes.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>A) Cleavage activity of HIV-1 IN on PFV and HIV-1 palindromes. IN was incubated with DNA substrates in a Mg<sup>2+</sup>-containing buffer for 2 h at 37°C. Lanes 1 and 2: FP53B/FP53A substrate (PFV). Lanes 3 and 4: PalB/PalA substrate (HIV). Lanes 5–8: Mut1 substrate (CA-&gt;GT). Lanes 9–12: Mut2 substrate (GTAC-&gt;CATG). Lanes 2, 4, 6 and 10: 3 µM IN. Lanes 7 and 11: 1.5 µM IN. Lanes 1, 3, 5 and 9: negative control with 3 µM IN + 1 mM EDTA. Lanes 8 and 12: <em>Sca</em>I activity on mutated DNA duplexes. B) Histogram of cleavage efficiencies for the different DNA positions in the HIV-1 LTR-LTR junction. Black bars, wt. White bars, Mut1.</p>
<span>doi:10.1371/journal.pone.0000608.g003</span></div><div class="figure" id="pone-0000608-g004"><div class="img"><a name="pone-0000608-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Differential responses of palindrome cleavage and 3′-processing activities to increasing IN concentrations.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>PalB/PalA or HIV38A/HIV38B duplexes (12.5 nM) were incubated with increasing concentrations of IN for 2 h at 37°C. Palindrome cleavage and 3′-processing activities were quantified as indicated in <a href="#s2">Materials and Methods</a> and plotted versus IN concentration: Palindrome cleavage (straight line); 3′-processing (dashed line). A) Ionic strength increased with IN concentration. B) The experiment was performed as in A, except that ionic strength was kept constant ([NaCl]  =  200 mM final concentration). The time courses of product formation for palindrome cleavage and 3′-processing were compared under conditions of optimal IN concentration (3 µM), with 200 mM NaCl (inset). F<sub>DNA</sub> represents the fractional saturation function of DNA sites, as measured by fluorescence anisotropy (see <a href="#s2">Materials and Methods</a>), using either PalB/PalA or HIV38A/HIV38B duplexes. F<sub>DNA</sub> is indicated for two IN concentrations (1.5 and 3 µM).</p>
<span>doi:10.1371/journal.pone.0000608.g004</span></div><div class="figure" id="pone-0000608-g005"><div class="img"><a name="pone-0000608-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>HIV-1 IN oligomer recruitment to the LTR-LTR junction.</span></strong></p><a id="article1.body1.sec2.sec2.fig4.caption1.p1" name="article1.body1.sec2.sec2.fig4.caption1.p1"></a><p>The 38-bp duplex PalB/PalA (1 pmol) was incubated with IN (5 pmol) for 0 to 60 min (lanes 2–7) in the presence of AHDAP (300 µM) in a final volume of 10 µL (yielding DNA and IN concentrations of 0.1 and 0.5 µM, respectively). Crosslinked products were then subjected to SDS-PAGE analysis and gel autoradiography. MW: Molecular weight markers (kDa). Lane 1: no IN. The weak reduction of the signal observed lanes 6 and 7 as comparison to the lane 5 is due to the time-dependent formation of higher-order oligomeric states of IN which are dependent on the AHDAP <a href="#pone.0000608-Faure1">[10]</a>.</p>
<span>doi:10.1371/journal.pone.0000608.g005</span></div><div class="figure" id="pone-0000608-g006"><div class="img"><a name="pone-0000608-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Palindrome cleavage by purified IN oligomers.</span></strong></p><a id="article1.body1.sec2.sec2.fig5.caption1.p1" name="article1.body1.sec2.sec2.fig5.caption1.p1"></a><p>The pJCT plasmid (12.5 nM) containing the LTR-LTR junction was incubated alone (lanes 1 and 4) or with 3 µM of either pure crosslinked dimers (Di) (lanes 2–3) or pure crosslinked tetramers (Te) of IN (lanes 5–6) for 4 hours at 37°C. It was then subjected to electrophoresis in an agarose gel (1%, 50V, 30 min) in the presence of BET (50 µg). Plasmid cleavage by IN was followed by partial digestion with <em>Eco</em>RI (lanes 3 and 6). MW: Molecular weight markers (kb).</p>
<span>doi:10.1371/journal.pone.0000608.g006</span></div>

<h4>Chemical IN-DNA crosslinking</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>HIV-1 IN (5 pmol) was incubated with the PalA/PalB duplex (1 pmol) radiolabelled at its 5′-end in the presence of <em>cis</em>-aquahydroxydiamino platinum (AHDAP, 300 µM), 0.05% NP40, 10 mM DTT, 20 mM HEPES pH 7.5, at 37°C in the dark (final volume, 10 µl). The reaction was stopped by eliminating excess AHDAP, after crosslinking, by elution through a G25 MicroSpin column (Amersham) <a href="#pone.0000608-Faure1">[10]</a>.</p>


<h4>Steady-state fluorescence anisotropy assay</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Fluorescence anisotropy parameters were recorded with a Beacon 2000 instrument (PanVera, Madison, USA) in a cell maintained at 25°C. The formation of IN-DNA complexes was monitored by incubating fluorescein-labelled double-stranded ODNs with IN in 20 mM Tris (pH 7.2), 1 mM DTT, 100 or 200 mM NaCl, 5 mM MgCl<sub>2</sub>. The fractional saturation function of DNA (F<sub>DNA</sub>) was calculated as previously described <a href="#pone.0000608-Guiot1">[11]</a>, <a href="#pone.0000608-Deprez1">[22]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>HIV-1 WT specifically cleaves the U5-U3 palindromic junction in vitro</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Analyses of HIV-1 integration sites have suggested that IN displays a preference for symmetric DNA sequences for the integration process <a href="#pone.0000608-Wu1">[12]</a>–<a href="#pone.0000608-Grandgenett1">[14]</a>. We reasoned that IN might display specific recognition properties with DNA substrates containing symmetric or palindromic sequences, possibly leading to specific internal cleavage.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>We tested this hypothesis, by assaying HIV-1 IN endonuclease activity in the presence of Mg<sup>2+</sup> or Mn<sup>2+</sup>, using 38-bp ODNs containing several symmetric sequences in internal positions (<a href="#pone-0000608-g001">Fig. 1B</a>). We first tested the WP1 and WP2 palindromic sequences which mimic the weak palindrome consensus found at integration sites <em>in vivo</em> <a href="#pone.0000608-Wu1">[12]</a>–<a href="#pone.0000608-Grandgenett1">[14]</a>. This sequence (..GTXAC..) displays an additional central base as compared to the palindromic LTR-LTR junction (<a href="#pone-0000608-g001">Fig. 1B</a>), where X represents any base with a bias toward A or T <a href="#pone.0000608-Wu1">[12]</a>. No significant cleavage was observed with WP1 and WP2 (data not shown). We next assessed the internal cleavage by IN of a palindromic 38-bp ODN that mimics the HIV-1 LTR-LTR junction. We observed a clear and reproducible cleavage of the palindromic sequence by radio-labelling the top strand (Pal B), regardless the divalent cation cofactor used (<a href="#pone-0000608-g002">Fig. 2A</a>; lanes 1–7) (no cleavage occurred in the absence of metallic cofactor or with single-stranded ODNs; data not shown). A control cleavage reaction with <em>Sca</em>I showed that cleavage occurred preferentially at a precise position downstream from the conserved 5′-CA sequence (primary site) matching the cleavage site observed for the standard 3′-processing of a single LTR extremity (band <em>a</em>). A weaker secondary cleavage site was also observed. This cleavage occurred reproducibly within the palindromic sequence after the next A residue in the 5′-3′ direction (band <em>b</em>). In addition to these two main sites, minor cleavage sites characteristic of the non-specific endonuclease activity of IN were also found throughout the length of the DNA.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>We assessed the symmetry of specific cleavage by investigating IN activity on the other (bottom) strand of this palindrome (PalA; <a href="#pone-0000608-g002">Fig. 2A</a>, lanes 8–10). Again, specific cleavage was observed (bands a' and b'), with cleavage efficiency highest downstream from the 5′-CA position. Altogether, our data show that internal cleavage was clearly symmetric at the palindromic LTR-LTR junction. The efficiency of cleavage was dependent on the IN concentration for both strands. A better activity was obtained at 3 µM as compared to 1.5 µM (see also <a href="#pone-0000608-g004">figure 4</a>). The relative cleavage efficiencies for palindrome cleavage and 3′-processing reactions are indicated for each position in <a href="#pone-0000608-g002">Fig. 2B</a>. For these two reactions, the specific sites represent about 80% of the total IN activity on ODNs. Therefore, both reactions display similar reaction specificity. However, two major cleavage sites were observed in a reproducible manner for the palindrome cleavage and one site for the 3′-processing reaction. Moreover, it is important to note that efficiencies of 3′-processing and palindrome cleavage are comparable (see also the complete kinetics study in <a href="#pone-0000608-g004">Fig. 4</a>). Therefore, our results show that the cleavages occurring either at the DNA extremity or at the internal palindromic sequence are both specific and display similar catalytic efficiencies.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>We then investigated the structural features of IN required for this activity, by testing the ability of point and deletion IN mutants to cleave the palindromic sequence. First, the catalytically inactive E152A mutant was unable to cleave the palindromic junction (<a href="#pone-0000608-t001">Table I</a>), regardless of the cationic cofactor used, showing that the DDE catalytic triad is directly involved in this cleavage. Second, we tested two IN mutants, one devoid of the C-terminal domain (ΔC) and the other, lacking both the N-terminal and the C-terminal domains (CC). These two deletion mutants were active in the disintegration assay in the presence of Mn<sup>2+</sup> conditions (<a href="#pone-0000608-t001">Table I</a>), consistent with previous results <a href="#pone.0000608-Leh1">[21]</a>, <a href="#pone.0000608-Gerton1">[23]</a>, <a href="#pone.0000608-Laboulais1">[24]</a>. However, they were not competent for palindrome cleavage (<a href="#pone-0000608-t001">Table I</a>). Thus, cleavage of the palindromic junction depends on the integrity of the full-length HIV-1 IN, closely paralleling the stringency observed for the 3′-processing reaction.</p>
<div class="figure" id="pone-0000608-t001"><div class="img"><a name="pone-0000608-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000608" data-uri="info:doi/10.1371/journal.pone.0000608.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000608.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000608.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000608.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000608.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000608.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Activities of the wild-type IN and mutants</span></strong></p><span>doi:10.1371/journal.pone.0000608.t001</span></div><a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>We then investigated whether the palindromic organisation of the DNA substrate containing the canonical 5′-CA dinucleotide was sufficient to ensure cleavage and assessed the tolerance to sequence changes of this cleavage. We recently showed that another retroviral IN, the primate foamy virus (PFV-1) IN, can cleave the palindromic sequence found at the PFV LTR-LTR junction <a href="#pone.0000608-Delelis1">[25]</a>. We assessed the ability of HIV-1 IN to specifically cleave a palindromic sequence derived from the PFV LTR-LTR junction as well as several mutated versions of the HIV-1 palindromic sequence (<a href="#pone-0000608-g003">Fig. 3A</a>; see <a href="#pone-0000608-g001">Fig. 1A</a> for ODN sequences). The HIV-1 IN efficiently cleaved the eight-base HIV-1 palindromic sequence but did not cleave the six-base PFV 2-LTR junction which is specifically cleaved by the PFV-1 IN <a href="#pone.0000608-Delelis1">[25]</a>. This suggests that retroviral INs display no cross-specificity for palindrome cleavage. We then investigated whether the four base pairs located between the canonical CA sequences of HIV-1 were required for cleavage using an inverted eight-base palindrome in which the four central bases were replaced by CATG (Mut2 substrate, see <a href="#pone-0000608-g001">Fig. 1B</a>). Again, no significant cleavage was detected, demonstrating that the palindromic organisation of the sequence was not sufficient for cleavage. We then investigated the role of the canonical 5′-CA, by replacing this dinucleotide with a 5′-GT sequence (Mut1 substrate, see <a href="#pone-0000608-g001">Fig. 1B</a>). Mut1 displayed clear cleavage sites within the boundaries of the palindrome, indicating that the 5′-CA is not crucial for the specific recruitment of IN to the viral substrate. However, cleavage levels were lower at the a position and higher at the b position than for the wild-type sequence (<a href="#pone-0000608-g003">Fig. 3B</a>). Interestingly, a third significant cleavage site was observed for Mut1 (band c), corresponding to a cleavage downstream from a 5′-TA sequence. This suggests that, in the absence of the canonical 5′-CA, IN probes the most favourable cleavage sites in the vicinity of the palindrome. Overall, our results show that palindrome cleavage has more or similar stringent requirements for the structural integrity of IN and viral DNA sequence than does the disintegration or the 3′-processing reactions, respectively.</p>


<h4>3′-processing and palindrome cleavage require different oligomeric states</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>The 3′-processing activity of IN depends on the IN/DNA ratio, which controls the level of active complexes <a href="#pone.0000608-Guiot1">[11]</a>. 3′-processing is optimal for IN/DNA ratio conditions corresponding to the dimerisation of IN on DNA <a href="#pone.0000608-Faure1">[10]</a>, <a href="#pone.0000608-Guiot1">[11]</a>, <a href="#pone.0000608-Smolov1">[26]</a>. We investigated the extent to which palindrome cleavage and 3′-processing could be compared, by determining the efficiencies of these processes as a function of IN concentration. The DNA substrates used were of identical size, to avoid a bias in IN binding due to oligonucleotide length <a href="#pone.0000608-Guiot1">[11]</a>. We observed a typical bell-shaped curve for the 3′-processing reaction, and the optimal IN concentration was close to 1.5 µM (<a href="#pone-0000608-g004">Fig. 4A</a>, dashed line). We carried out fluorescence anisotropy experiments under these conditions and showed that all the DNA substrates in solution were bound by IN (F<sub>DNA</sub>  =  100%). In contrast to 3′-processing, palindrome cleavage activity was weak at 1.5 µM, although F<sub>DNA</sub> was also 100%. However, more efficient palindrome cleavage was observed at higher protein concentrations, peaking at 3 µM IN (<a href="#pone-0000608-g004">Fig. 4A</a>, straight line). The amount of product obtained at this concentration was consistent with the fractional saturation function (F<sub>DNA</sub>  =  20%). The lower amount of active complexes can be fully explained by the higher ionic strength at the protein concentration of 3 µM ([NaCl]  =  200 mM). Taking into account the global effect of ionic strength, it is suggested that the activity per IN/DNA complexes is similar for palindrome cleavage and 3′-processing reactions. Consistently, both reactions display similar efficiencies for 3 µM IN concentration.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>As ionic strength strongly influences the number of IN/DNA complexes, we next compared the palindrome cleavage and 3′-processing reactions in the presence of increasing IN concentrations, keeping salt concentration constant (200 mM) (<a href="#pone-0000608-g004">Fig. 4B</a>). With 1.5 µM IN, the efficiency of 3′-processing was about one third that obtained with the same IN concentration at lower ionic strength (compare <a href="#pone-0000608-g004">Fig. 4A and 4B</a>). This lower level of activity is due to the smaller number of complexes formed in the presence of 200 mM NaCl. Interestingly, the activity curve for 3′-processing was characterized by two maxima, one centred on 1.5 µM IN and the other centred on 3 µM (<a href="#pone-0000608-g004">Fig. 4B</a>). This suggests that 3′-processing occurs optimally with two types of complex, differing in the number of IN protomers complexed to DNA. In contrast, only the higher IN concentration allowed efficient palindrome cleavage, although similar numbers of IN/DNA complexes were obtained with the palindrome and the 3′-processing substrate (20% of complexes) for both IN concentrations (1.5 and 3 µM). Thus, only one type of complex is competent for palindrome cleavage and a low IN:DNA stoichiometry is not compatible with this reaction in contrast to what was observed for the 3′-processing reaction. In the second peak, corresponding to a high IN/DNA ratio, both reactions—3′-processing and palindrome cleavage—were equally efficient.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>We compared the efficiency of the two reactions further at optimal IN concentration, by carrying out time-dependent experiments (<a href="#pone-0000608-g004">Fig. 4B</a>, inset). Both reactions were slow and linear over time for the first 10 hours, consistent with slow IN turnover <a href="#pone.0000608-Smolov1">[26]</a>. In both cases, 90% of the product had been obtained by 16 hours of incubation. The kinetics of the two reactions were indistinguishable, confirming that palindrome cleavage is as efficient as 3′-processing.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>Palindrome cleavage and 3′-processing responded differently to IN concentration, suggesting that these two reactions may be optimised by different oligomeric states of IN bound to DNA. We have previously shown that DNA-bound IN was mainly dimeric under conditions corresponding to the first peak of 3′-processing activity <a href="#pone.0000608-Guiot1">[11]</a>. Here, we found that optimal palindrome cleavage activity as well as the second peak of 3′-processing were obtained with a higher IN concentration. Altogether, our results suggest that the dimeric IN catalyses the 3′-processing reaction but not the palindrome cleavage, whereas a higher-order multimeric state catalyses both reactions indifferently. It was previously shown that the decreasing phase observed for 3′-processing at high protein concentration is due to the formation of protein aggregates <a href="#pone.0000608-Guiot1">[11]</a>. Most likely, protein aggregation onto DNA also accounts for the decreasing phases observed for the single peak of palindrome cleavage and the second peak of the 3′-processing reaction. For 3′-processing, we hypothesized that the decreasing phase observed between the two peaks of activity could be due to the formation of an intermediary oligomeric state, catalytically inactive.</p>


<h4>Palindromic DNA substrate recruits tetrameric IN</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Recently, Faure <em>et al.</em> have identified by cross-linking analysis one IN dimer per LTR extremity and observed that a DNA fragment lacking the LTR sequence did not recruit discrete complexes <a href="#pone.0000608-Li2">[4]</a>, <a href="#pone.0000608-Faure1">[10]</a>. In this study, we investigated whether the presence of an internal LTR-LTR junction led to the recruitment or stabilisation of such discrete IN complexes using the same experimental approach. We carried out cross-linking experiments with a DNA fragment containing an internal HIV-1 LTR-LTR junction. This DNA fragment was radiolabelled, cross-linked to IN by incubation for various lengths of time and subjected to SDS-PAGE (<a href="#pone-0000608-g005">Fig. 5</a>). In contrast to what was observed for the one-LTR sequence, we found two cross-linked products with the LTR-LTR junction, corresponding to the dimeric and tetrameric forms of IN bound to DNA. Together with the activity results obtained, this experiment indicates that the LTR-LTR junction can recruit both multimeric forms—dimers and tetramers—although only the tetramer is competent for internal cleavage.</p>


<h4>Restriction-like activity of IN tetramers</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Our results on short ODNs suggested that only tetramers were capable of cleaving the palindrome junction. We therefore investigated whether HIV-1 IN could cleave similarly longer DNA at internal positions. Purified covalent IN oligomers such as dimers and tetramers were obtained by chemically crosslinking and separated by size-exclusion chromatography, as previously described <a href="#pone.0000608-Faure1">[10]</a>. Dimers and tetramers were recovered and independently assayed for their ability to cleave a plasmid containing the palindromic LTR-LTR junction (the principle of the assay is explained in <a href="#pone-0000608-g006">Figure 6</a>). The incubation of IN dimers with DNA led to the accumulation of linear plasmid, indicating that double-strand cleavage had occurred (<a href="#pone-0000608-g006">Fig. 6</a>). After further digestion with <em>Eco</em>RI, we observed several bands, showing that the cleavage catalysed by the dimeric form occurred essentially non-specifically, at different sites in the DNA molecule. Linearisation of the plasmid was also obtained with IN tetramers, but further digestion with <em>Eco</em>RI yielded a single 700-bp fragment, showing that cleavage of a 5,300-bp plasmid had occurred, in a restriction-like manner, at a single position corresponding to the LTR-LTR junction. The specific cleavage at the LTR-LTR junction also occurs with dimers but this cleavage is rather weak as compared to the cleavage observed with the tetrameric IN. This could be explained either by a weak palindrome cleavage activity of dimers or a significant formation of tetramers (i.e. non-covalent dimer of dimers) which are competent for palindrome cleavage.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>We also assessed the palindrome cleavage activity of purified tetramers, dimers and monomers, using the 38-bp DNA substrate PalA/PalB. Again, the efficiencies of the various forms of IN differed—tetramers (50% cleavage)&gt;dimers (3% cleavage)&gt;monomers (no cleavage) (data not shown)—confirming that the tetrameric form was the most efficient for internal cleavage. In conclusion, our results provide for the first time the demonstration that HIV-1 is highly competent to cleave the LTR-LTR junction.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>We have identified a new type of endonuclease activity of HIV-1 IN that requires the enzyme to be organised as a tetramer on a specific DNA sequence. IN specifically cleaves, in a reproducible manner, DNA at internal positions, only if the cleavage site is formed by the palindromic sequence corresponding to the HIV-1 LTR-LTR junction. IN has been reported to cleave internal sequences in several studies <a href="#pone.0000608-Katzman1">[27]</a>–<a href="#pone.0000608-Skinner1">[29]</a>. However, the reported activity was weak and non-specific in all these cases. In particular, truncated IN proteins (CC, ΔC, ΔN) inactive in 3′-processing or strand transfer retain this endonuclease activity <a href="#pone.0000608-Katzman3">[30]</a>. In contrast, the palindromic junction cleavage described here is highly sequence-specific, requires the full-length protein and is efficient with the physiologically relevant cofactor, Mg<sup>2+</sup>. Palindrome cleavage also differs markedly in this respect from the disintegration reaction that can be performed by CC, ΔC or ΔN truncated proteins only in the presence of Mn<sup>2+</sup>, or by the full-length IN with a large preference for Mn<sup>2+</sup> over Mg<sup>2+</sup> <a href="#pone.0000608-Gerton1">[23]</a>, <a href="#pone.0000608-Laboulais1">[24]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Internal cleavage mostly occurs at the 5′-CA position corresponding to the CA of the 3′-processing reaction, symmetrically on the minus and plus strands. We also identified a secondary cleavage site downstream from the 5′-CA, at the 5′-TA sequence. Interestingly, mutation of the 5′-CA canonical dinucleotide revealed a supplementary cleavage at the 5′-TA sequence upstream from the mutated position, suggesting that cleavage may be delocalised, occurring at adjacent sites in the absence of the canonical 5′-CA. Studies of the conformational properties of nine different dinucleotides for which adequate data from crystal structures were available indicated that CA was by far the most flexible, resulting in its selection at transposon termini due to its significant conformational mobility <a href="#pone.0000608-Lee1">[31]</a>. The TA dinucleotide is the next most flexible <a href="#pone.0000608-Neugebauerova1">[32]</a>. It has been suggested that this flexibility of the CA dinucleotide is critical for DNA melting/distortion events before cleavage, making it possible for this strand to be engaged by the active site of the MuA transposase. These observations suggest that IN recruitment to the DNA substrate may be followed by a probing phase enabling IN to find a favourable site for cleavage—5′-CA or 5′-TA sequences. According to this model, the presence of the canonical 5′-CA in the WT palindrome would restrict upward movement, whereas movement in both directions is possible if this sequence is missing.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>Using purified cross-linked IN multimers, we showed that although LTR-LTR junctions recruited both the dimeric and tetrameric forms of IN, only tetramers specifically and efficiently cleaved the junction. Early studies reported that the avian sarcoma virus integrase could cleave the LTR-LTR junction <a href="#pone.0000608-Duyk1">[33]</a>, <a href="#pone.0000608-Grandgenett2">[34]</a>. However, this result was not reproduced with the HIV-1 IN <a href="#pone.0000608-Leavitt1">[35]</a>. Our results suggest that this may be due to the stringent conditions for palindrome cleavage, which is strictly dependent on the ability of HIV-1 IN to form an active tetrameric enzyme. This may account for the low efficiency or absence of palindrome cleavage at IN concentrations maximising the 3′-processing of short ODNs, which is known to occur with a dimeric enzyme <a href="#pone.0000608-Faure1">[10]</a>, <a href="#pone.0000608-Guiot1">[11]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>The requirement of a tetrameric form for palindrome cleavage parallels the results recently described for concerted integration showing that a dimer of dimers allows insertion of the two DNA ends into two different strands of the target DNA <a href="#pone.0000608-Li1">[3]</a>–<a href="#pone.0000608-Sinha1">[5]</a>, <a href="#pone.0000608-Grandgenett1">[14]</a>. However, the weak palindromic consensus which has been observed at the integration sites <em>in vivo</em> <a href="#pone.0000608-Wu1">[12]</a>–<a href="#pone.0000608-Grandgenett1">[14]</a> differs from the LTR-LTR junction by an additional central base. Hence, we did not observe a specific cleavage with DNA substrates mimicking this consensus sequence, indicating that the additional central base is a specific feature of integration sites. Moreover, our results indicate that the cleavage of the palindromic LTR-LTR junction is not tolerant to sequence modifications and thus is a highly specific reaction, whereas the integration sites, characterized <em>in vivo</em>, only exhibit a weak consensus. The symmetrical consensus, found at integration sites <em>in vivo</em>, means that a palindromic sequence in the DNA target favours the transesterification reaction (nucleophilic attack conducted by the 3′-OH viral DNA extremity) resulting in the full integration process. Our assay does not mimic a strand transfer (or integration) assay but rather reveals endonucleolytic reactions (nucleophilic attack mediated by a water molecule). This means that the specific cleavage of the LTR-LTR junction is primarily mediated by a water molecule and the catalytic mechanism of IN for this cleavage is similar to the well-known mechanism for 3′-processing. The catalytic reaction is more specific in term of sequence when a water molecule is involved in the nucleophilic attack, as observed for 3′-processing <a href="#pone.0000608-Agapkina1">[6]</a>, <a href="#pone.0000608-Esposito1">[7]</a> and 2-LTR junction cleavage (this study) as compared to reactions directly involving the terminal 3′-hydroxyl of the viral DNA as a nucleophilic agent (half transfer <a href="#pone.0000608-Leh1">[21]</a> and the full site integration in symmetrical sequences <a href="#pone.0000608-Wu1">[12]</a>–<a href="#pone.0000608-Grandgenett1">[14]</a>). These results suggest that both palindrome cleavage and integration require a tetrameric form but are based on different mechanisms. Nevertheless, both the strict dependence on palindromic sequences for internal cleavage and the palindromic bias of the integration demonstrate that tetrameric IN intrinsically prefers to bind to symmetric DNA sites.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>We recently reported an equivalent activity for PFV-1 IN with its cognate palindromic substrate. These observations suggest that palindrome cleavage may be an intrinsic property of retroviral INs <a href="#pone.0000608-Delelis1">[25]</a>. The strong specificity of such symmetric internal cleavage and the fact that the only palindrome cleaved corresponded to the cognate LTR-LTR junction naturally present in infected cells, within 2-LTR circles, suggest that palindrome recognition by IN may play an important role at different stages of the replication cycle. <em>In vitro</em>, internal cleavage at the HIV-1 LTR-LTR junction mostly produces extremities with a four-base overhang, different from the two-base overhang generated by 3′-processing. No such intermediate DNA product has yet been identified in infected cells, in contrast to the canonical 3′-processing product, which constitutes the precursor of the integrated provirus. Thus, in the absence of further cellular evidence, it remains unclear whether the palindrome cleavage of LTR-LTR junctions contributes to the overall equilibrium of the different forms of viral DNA in the cell.</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JM OD ED. Performed the experiments: HL GM OD VP. Analyzed the data: CP JM PS OD ED. Wrote the paper: JM OD ED.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000608-Craigie1" id="pone.0000608-Craigie1"></a>Craigie R (2001) HIV integrase, a brief overview from chemistry to therapeutics. J Biol Chem  276: 23213–23216.  <ul class="find" data-citedArticleID="965734" data-doi="10.1074/jbc.r100027200"><li><a href="http://dx.doi.org/10.1074/jbc.r100027200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV+integrase%2C+a+brief+overview+from+chemistry+to+therapeutics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV+integrase%2C+a+brief+overview+from+chemistry+to+therapeutics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000608-Lewinski1" id="pone.0000608-Lewinski1"></a>Lewinski MK, Bushman FD (2005) Retroviral DNA integration–mechanism and consequences. Adv Genet  55: 147–181.  <ul class="find" data-citedArticleID="965772" data-doi="10.1016/s0065-2660(05)55005-3"><li><a href="http://dx.doi.org/10.1016/s0065-2660(05)55005-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+DNA+integration%26%23x2013%3Bmechanism+and+consequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+DNA+integration%26%23x2013%3Bmechanism+and+consequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000608-Li1" id="pone.0000608-Li1"></a>Li M, Craigie R (2005) Processing of viral DNA ends channels the HIV-1 integration reaction to concerted integration. J Biol Chem  280: 29334–29339.  <ul class="find" data-citedArticleID="965776" data-doi="10.1074/jbc.m505367200"><li><a href="http://dx.doi.org/10.1074/jbc.m505367200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Processing+of+viral+DNA+ends+channels+the+HIV-1+integration+reaction+to+concerted+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Processing+of+viral+DNA+ends+channels+the+HIV-1+integration+reaction+to+concerted+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000608-Li2" id="pone.0000608-Li2"></a>Li M, Mizuuchi M, Burke TR Jr, Craigie R (2006) Retroviral DNA integration: reaction pathway and critical intermediates. EMBO J  25: 1295–1304.  <ul class="find" data-citedArticleID="965778" data-doi="10.1038/sj.emboj.7601005"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+DNA+integration%3A+reaction+pathway+and+critical+intermediates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+DNA+integration%3A+reaction+pathway+and+critical+intermediates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000608-Sinha1" id="pone.0000608-Sinha1"></a>Sinha S, Grandgenett DP (2005) Recombinant human immunodeficiency virus type 1 integrase exhibits a capacity for full-site integration in vitro that is comparable to that of purified preintegration complexes from virus-infected cells. J Virol  79: 8208–8216.  <ul class="find" data-citedArticleID="965790" data-doi="10.1128/jvi.79.13.8208-8216.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.13.8208-8216.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinant+human+immunodeficiency+virus+type+1+integrase+exhibits+a+capacity+for+full-site+integration+in+vitro+that+is+comparable+to+that+of+purified+preintegration+complexes+from+virus-infected+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinant+human+immunodeficiency+virus+type+1+integrase+exhibits+a+capacity+for+full-site+integration+in+vitro+that+is+comparable+to+that+of+purified+preintegration+complexes+from+virus-infected+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000608-Agapkina1" id="pone.0000608-Agapkina1"></a>Agapkina J, Smolov M, Barbe S, Zubin E, Zatsepin T, et al.  (2006) Probing of HIV-1 integrase/DNA interactions using novel analogs of viral DNA. J Biol Chem  281: 11530–11540.  <ul class="find" data-citedArticleID="965732" data-doi="10.1074/jbc.m512271200"><li><a href="http://dx.doi.org/10.1074/jbc.m512271200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Probing+of+HIV-1+integrase%2FDNA+interactions+using+novel+analogs+of+viral+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Probing+of+HIV-1+integrase%2FDNA+interactions+using+novel+analogs+of+viral+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000608-Esposito1" id="pone.0000608-Esposito1"></a>Esposito D, Craigie R (1998) Sequence specificity of viral end DNA binding by HIV-1 integrase reveals critical regions for protein-DNA interaction. EMBO J  17: 5832–5843.  <ul class="find" data-citedArticleID="965742" data-doi="10.1093/emboj/17.19.5832"><li><a href="http://dx.doi.org/10.1093/emboj/17.19.5832" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+specificity+of+viral+end+DNA+binding+by+HIV-1+integrase+reveals+critical+regions+for+protein-DNA+interaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+specificity+of+viral+end+DNA+binding+by+HIV-1+integrase+reveals+critical+regions+for+protein-DNA+interaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000608-Vink1" id="pone.0000608-Vink1"></a>Vink C, van Gent DC, Elgersma Y, Plasterk RH (1991) Human immunodeficiency virus integrase protein requires a subterminal position of its viral DNA recognition sequence for efficient cleavage. J Virol  65: 4636–4644.  <ul class="find" data-citedArticleID="965798"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Human+immunodeficiency+virus+integrase+protein+requires+a+subterminal+position+of+its+viral+DNA+recognition+sequence+for+efficient+cleavage.&amp;auth=&amp;atitle=Human+immunodeficiency+virus+integrase+protein+requires+a+subterminal+position+of+its+viral+DNA+recognition+sequence+for+efficient+cleavage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+immunodeficiency+virus+integrase+protein+requires+a+subterminal+position+of+its+viral+DNA+recognition+sequence+for+efficient+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+immunodeficiency+virus+integrase+protein+requires+a+subterminal+position+of+its+viral+DNA+recognition+sequence+for+efficient+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000608-Lewinski2" id="pone.0000608-Lewinski2"></a>Lewinski MK, Yamashita M, Emerman M, Ciuffi A, Marshall H, et al.  (2006) Retroviral DNA integration: viral and cellular determinants of target-site selection. PLoS Pathog  2: e60.  <ul class="find" data-citedArticleID="965774" data-doi="10.1371/journal.ppat.0020060"><li><a href="http://dx.doi.org/10.1371/journal.ppat.0020060" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroviral+DNA+integration%3A+viral+and+cellular+determinants+of+target-site+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroviral+DNA+integration%3A+viral+and+cellular+determinants+of+target-site+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000608-Faure1" id="pone.0000608-Faure1"></a>Faure A, Calmels C, Desjobert C, Castroviejo M, Caumont-Sarcos A, et al.  (2005) HIV-1 integrase crosslinked oligomers are active in vitro. Nucleic Acids Res  33: 977–986.  <ul class="find" data-citedArticleID="965744" data-doi="10.1093/nar/gki241"><li><a href="http://dx.doi.org/10.1093/nar/gki241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+integrase+crosslinked+oligomers+are+active+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+integrase+crosslinked+oligomers+are+active+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000608-Guiot1" id="pone.0000608-Guiot1"></a>Guiot E, Carayon K, Delelis O, Simon F, Tauc P, et al.  (2006) Relationship between the oligomeric status of HIV-1 integrase on DNA and enzymatic activity. J Biol Chem  281: 22707–22719.  <ul class="find" data-citedArticleID="965752" data-doi="10.1074/jbc.m602198200"><li><a href="http://dx.doi.org/10.1074/jbc.m602198200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+the+oligomeric+status+of+HIV-1+integrase+on+DNA+and+enzymatic+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+the+oligomeric+status+of+HIV-1+integrase+on+DNA+and+enzymatic+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000608-Wu1" id="pone.0000608-Wu1"></a>Wu X, Li Y, Crise B, Burgess SM, Munroe DJ (2005) Weak palindromic consensus sequences are a common feature found at the integration target sites of many retroviruses. J Virol  79: 5211–5214.  <ul class="find" data-citedArticleID="965800" data-doi="10.1128/jvi.79.8.5211-5214.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.8.5211-5214.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Weak+palindromic+consensus+sequences+are+a+common+feature+found+at+the+integration+target+sites+of+many+retroviruses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Weak+palindromic+consensus+sequences+are+a+common+feature+found+at+the+integration+target+sites+of+many+retroviruses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000608-Holman1" id="pone.0000608-Holman1"></a>Holman AG, Coffin JM (2005) Symmetrical base preferences surrounding HIV-1, avian sarcoma/leukosis virus, and murine leukemia virus integration sites. Proc Natl Acad Sci U S A  102: 6103–6107.  <ul class="find" data-citedArticleID="965754" data-doi="10.1073/pnas.0501646102"><li><a href="http://dx.doi.org/10.1073/pnas.0501646102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symmetrical+base+preferences+surrounding+HIV-1%2C+avian+sarcoma%2Fleukosis+virus%2C+and+murine+leukemia+virus+integration+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symmetrical+base+preferences+surrounding+HIV-1%2C+avian+sarcoma%2Fleukosis+virus%2C+and+murine+leukemia+virus+integration+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000608-Grandgenett1" id="pone.0000608-Grandgenett1"></a>Grandgenett DP (2005) Symmetrical recognition of cellular DNA target sequences during retroviral integration. Proc Natl Acad Sci U S A  102: 5903–5904.  <ul class="find" data-citedArticleID="965748" data-doi="10.1073/pnas.0502045102"><li><a href="http://dx.doi.org/10.1073/pnas.0502045102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Symmetrical+recognition+of+cellular+DNA+target+sequences+during+retroviral+integration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Symmetrical+recognition+of+cellular+DNA+target+sequences+during+retroviral+integration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000608-Miles1" id="pone.0000608-Miles1"></a>Miles LR, Agresta BE, Khan MB, Tang S, Levin JG, et al.  (2005) Effect of polypurine tract (PPT) mutations on human immunodeficiency virus type 1 replication: a virus with a completely randomized PPT retains low infectivity. J Virol  79: 6859–6867.  <ul class="find" data-citedArticleID="965782" data-doi="10.1128/jvi.79.11.6859-6867.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.11.6859-6867.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+polypurine+tract+%28PPT%29+mutations+on+human+immunodeficiency+virus+type+1+replication%3A+a+virus+with+a+completely+randomized+PPT+retains+low+infectivity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+polypurine+tract+%28PPT%29+mutations+on+human+immunodeficiency+virus+type+1+replication%3A+a+virus+with+a+completely+randomized+PPT+retains+low+infectivity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000608-Svarovskaia1" id="pone.0000608-Svarovskaia1"></a>Svarovskaia ES, Barr R, Zhang X, Pais GC, Marchand C, et al.  (2004) Azido-containing diketo acid derivatives inhibit human immunodeficiency virus type 1 integrase in vivo and influence the frequency of deletions at two-long-terminal-repeat-circle junctions. J Virol  78: 3210–3222.  <ul class="find" data-citedArticleID="965796" data-doi="10.1128/jvi.78.7.3210-3222.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.7.3210-3222.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Azido-containing+diketo+acid+derivatives+inhibit+human+immunodeficiency+virus+type+1+integrase+in+vivo+and+influence+the+frequency+of+deletions+at+two-long-terminal-repeat-circle+junctions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Azido-containing+diketo+acid+derivatives+inhibit+human+immunodeficiency+virus+type+1+integrase+in+vivo+and+influence+the+frequency+of+deletions+at+two-long-terminal-repeat-circle+junctions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000608-Olsen1" id="pone.0000608-Olsen1"></a>Olsen JC, Bova-Hill C, Grandgenett DP, Quinn TP, Manfredi JP, et al.  (1990) Rearrangements in unintegrated retroviral DNA are complex and are the result of multiple genetic determinants. J Virol  64: 5475–5484.  <ul class="find" data-citedArticleID="965786"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Rearrangements+in+unintegrated+retroviral+DNA+are+complex+and+are+the+result+of+multiple+genetic+determinants.&amp;auth=&amp;atitle=Rearrangements+in+unintegrated+retroviral+DNA+are+complex+and+are+the+result+of+multiple+genetic+determinants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rearrangements+in+unintegrated+retroviral+DNA+are+complex+and+are+the+result+of+multiple+genetic+determinants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rearrangements+in+unintegrated+retroviral+DNA+are+complex+and+are+the+result+of+multiple+genetic+determinants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000608-Serhan1" id="pone.0000608-Serhan1"></a>Serhan F, Penaud M, Petit C, Leste-Lasserre T, Trajcevski S, et al.  (2004) Early detection of a two-long-terminal-repeat junction molecule in the cytoplasm of recombinant murine leukemia virus-infected cells. J Virol  78: 6190–6199.  <ul class="find" data-citedArticleID="965788" data-doi="10.1128/jvi.78.12.6190-6199.2004"><li><a href="http://dx.doi.org/10.1128/jvi.78.12.6190-6199.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+detection+of+a+two-long-terminal-repeat+junction+molecule+in+the+cytoplasm+of+recombinant+murine+leukemia+virus-infected+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+detection+of+a+two-long-terminal-repeat+junction+molecule+in+the+cytoplasm+of+recombinant+murine+leukemia+virus-infected+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000608-Kilzer1" id="pone.0000608-Kilzer1"></a>Kilzer JM, Stracker T, Beitzel B, Meek K, Weitzman M, et al.  (2003) Roles of host cell factors in circularization of retroviral dna. Virology  314: 460–467.  <ul class="find" data-citedArticleID="965762" data-doi="10.1016/s0042-6822(03)00455-0"><li><a href="http://dx.doi.org/10.1016/s0042-6822(03)00455-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Roles+of+host+cell+factors+in+circularization+of+retroviral+dna." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Roles+of+host+cell+factors+in+circularization+of+retroviral+dna.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000608-Li3" id="pone.0000608-Li3"></a>Li L, Olvera JM, Yoder KE, Mitchell RS, Butler SL, et al.  (2001) Role of the non-homologous DNA end joining pathway in the early steps of retroviral infection. EMBO J  20: 3272–3281.  <ul class="find" data-citedArticleID="965780" data-doi="10.1093/emboj/20.12.3272"><li><a href="http://dx.doi.org/10.1093/emboj/20.12.3272" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+the+non-homologous+DNA+end+joining+pathway+in+the+early+steps+of+retroviral+infection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+the+non-homologous+DNA+end+joining+pathway+in+the+early+steps+of+retroviral+infection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000608-Leh1" id="pone.0000608-Leh1"></a>Leh H, Brodin P, Bischerour J, Deprez E, Tauc P, et al.  (2000) Determinants of Mg2+-dependent activities of recombinant human immunodeficiency virus type 1 integrase. Biochemistry  39: 9285–9294.  <ul class="find" data-citedArticleID="965770" data-doi="10.1021/bi000398b"><li><a href="http://dx.doi.org/10.1021/bi000398b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Determinants+of+Mg2%2B-dependent+activities+of+recombinant+human+immunodeficiency+virus+type+1+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Determinants+of+Mg2%2B-dependent+activities+of+recombinant+human+immunodeficiency+virus+type+1+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000608-Deprez1" id="pone.0000608-Deprez1"></a>Deprez E, Barbe S, Kolaski M, Leh H, Zouhiri F, et al.  (2004) Mechanism of HIV-1 integrase inhibition by styrylquinoline derivatives in vitro. Mol Pharmacol  65: 85–98.  <ul class="find" data-citedArticleID="965738" data-doi="10.1124/mol.65.1.85"><li><a href="http://dx.doi.org/10.1124/mol.65.1.85" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+HIV-1+integrase+inhibition+by+styrylquinoline+derivatives+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+HIV-1+integrase+inhibition+by+styrylquinoline+derivatives+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000608-Gerton1" id="pone.0000608-Gerton1"></a>Gerton JL, Brown PO (1997) The core domain of HIV-1 integrase recognizes key features of its DNA substrates. J Biol Chem  272: 25809–25815.  <ul class="find" data-citedArticleID="965746" data-doi="10.1074/jbc.272.41.25809"><li><a href="http://dx.doi.org/10.1074/jbc.272.41.25809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+core+domain+of+HIV-1+integrase+recognizes+key+features+of+its+DNA+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+core+domain+of+HIV-1+integrase+recognizes+key+features+of+its+DNA+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000608-Laboulais1" id="pone.0000608-Laboulais1"></a>Laboulais C, Deprez E, Leh H, Mouscadet JF, Brochon JC, et al.  (2001) HIV-1 integrase catalytic core: molecular dynamics and simulated fluorescence decays. Biophys J  81: 473–489.  <ul class="find" data-citedArticleID="965764" data-doi="10.1016/s0006-3495(01)75715-3"><li><a href="http://dx.doi.org/10.1016/s0006-3495(01)75715-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+integrase+catalytic+core%3A+molecular+dynamics+and+simulated+fluorescence+decays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+integrase+catalytic+core%3A+molecular+dynamics+and+simulated+fluorescence+decays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000608-Delelis1" id="pone.0000608-Delelis1"></a>Delelis O, Petit C, Leh H, Mbemba G, Mouscadet JF, et al.  (2005) A novel function for spumaretrovirus integrase: an early requirement for integrase-mediated cleavage of 2 LTR circles. Retrovirology  2: 31.  <ul class="find" data-citedArticleID="965736"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+novel+function+for+spumaretrovirus+integrase%3A+an+early+requirement+for+integrase-mediated+cleavage+of+2+LTR+circles.&amp;auth=&amp;atitle=A+novel+function+for+spumaretrovirus+integrase%3A+an+early+requirement+for+integrase-mediated+cleavage+of+2+LTR+circles." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+function+for+spumaretrovirus+integrase%3A+an+early+requirement+for+integrase-mediated+cleavage+of+2+LTR+circles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+function+for+spumaretrovirus+integrase%3A+an+early+requirement+for+integrase-mediated+cleavage+of+2+LTR+circles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000608-Smolov1" id="pone.0000608-Smolov1"></a>Smolov M, Gottikh M, Tashlitskii V, Korolev S, Demidyuk I, et al.  (2006) Kinetic study of the HIV-1 DNA 3′-end processing. FEBS J  273: 1137–1151.  <ul class="find" data-citedArticleID="965794" data-doi="10.1111/j.1742-4658.2006.05139.x"><li><a href="http://dx.doi.org/10.1111/j.1742-4658.2006.05139.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Kinetic+study+of+the+HIV-1+DNA+3%26%23x2032%3B-end+processing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Kinetic+study+of+the+HIV-1+DNA+3%26%23x2032%3B-end+processing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000608-Katzman1" id="pone.0000608-Katzman1"></a>Katzman M, Sudol M (1996) Nonspecific alcoholysis, a novel endonuclease activity of human immunodeficiency virus type 1 and other retroviral integrases. J Virol  70: 2598–2604.  <ul class="find" data-citedArticleID="965756"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nonspecific+alcoholysis%2C+a+novel+endonuclease+activity+of+human+immunodeficiency+virus+type+1+and+other+retroviral+integrases.&amp;auth=&amp;atitle=Nonspecific+alcoholysis%2C+a+novel+endonuclease+activity+of+human+immunodeficiency+virus+type+1+and+other+retroviral+integrases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonspecific+alcoholysis%2C+a+novel+endonuclease+activity+of+human+immunodeficiency+virus+type+1+and+other+retroviral+integrases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonspecific+alcoholysis%2C+a+novel+endonuclease+activity+of+human+immunodeficiency+virus+type+1+and+other+retroviral+integrases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000608-Katzman2" id="pone.0000608-Katzman2"></a>Katzman M, Sudol M, Pufnock JS, Zeto S, Skinner LM (2000) Mapping target site selection for the non-specific nuclease activities of retroviral integrase. Virus Res  66: 87–100.  <ul class="find" data-citedArticleID="965758" data-doi="10.1016/s0168-1702(99)00126-4"><li><a href="http://dx.doi.org/10.1016/s0168-1702(99)00126-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+target+site+selection+for+the+non-specific+nuclease+activities+of+retroviral+integrase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+target+site+selection+for+the+non-specific+nuclease+activities+of+retroviral+integrase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000608-Skinner1" id="pone.0000608-Skinner1"></a>Skinner LM, Sudol M, Harper AL, Katzman M (2001) Nucleophile selection for the endonuclease activities of human, ovine, and avian retroviral integrases. J Biol Chem  276: 114–124.  <ul class="find" data-citedArticleID="965792" data-doi="10.1074/jbc.m007032200"><li><a href="http://dx.doi.org/10.1074/jbc.m007032200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleophile+selection+for+the+endonuclease+activities+of+human%2C+ovine%2C+and+avian+retroviral+integrases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleophile+selection+for+the+endonuclease+activities+of+human%2C+ovine%2C+and+avian+retroviral+integrases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000608-Katzman3" id="pone.0000608-Katzman3"></a>Katzman M, Sudol M (1995) Mapping domains of retroviral integrase responsible for viral DNA specificity and target site selection by analysis of chimeras between human immunodeficiency virus type 1 and visna virus integrases. J Virol  69: 5687–5696.  <ul class="find" data-citedArticleID="965760"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mapping+domains+of+retroviral+integrase+responsible+for+viral+DNA+specificity+and+target+site+selection+by+analysis+of+chimeras+between+human+immunodeficiency+virus+type+1+and+visna+virus+integrases.&amp;auth=&amp;atitle=Mapping+domains+of+retroviral+integrase+responsible+for+viral+DNA+specificity+and+target+site+selection+by+analysis+of+chimeras+between+human+immunodeficiency+virus+type+1+and+visna+virus+integrases." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+domains+of+retroviral+integrase+responsible+for+viral+DNA+specificity+and+target+site+selection+by+analysis+of+chimeras+between+human+immunodeficiency+virus+type+1+and+visna+virus+integrases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+domains+of+retroviral+integrase+responsible+for+viral+DNA+specificity+and+target+site+selection+by+analysis+of+chimeras+between+human+immunodeficiency+virus+type+1+and+visna+virus+integrases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000608-Lee1" id="pone.0000608-Lee1"></a>Lee I, Harshey RM (2001) Importance of the conserved CA dinucleotide at Mu termini. J Mol Biol  314: 433–444.  <ul class="find" data-citedArticleID="965768" data-doi="10.1006/jmbi.2001.5177"><li><a href="http://dx.doi.org/10.1006/jmbi.2001.5177" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Importance+of+the+conserved+CA+dinucleotide+at+Mu+termini." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Importance+of+the+conserved+CA+dinucleotide+at+Mu+termini.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000608-Neugebauerova1" id="pone.0000608-Neugebauerova1"></a>Neugebauerova S, Kypr J (2000) Invariant and variable base stacking geometries in B-DNA and A-DNA. J Biomol Struct Dyn  18: 73–81.  <ul class="find" data-citedArticleID="965784" data-doi="10.1080/07391102.2000.10506648"><li><a href="http://dx.doi.org/10.1080/07391102.2000.10506648" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invariant+and+variable+base+stacking+geometries+in+B-DNA+and+A-DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invariant+and+variable+base+stacking+geometries+in+B-DNA+and+A-DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000608-Duyk1" id="pone.0000608-Duyk1"></a>Duyk G, Longiaru M, Cobrinik D, Kowal R, deHaseth P, et al.  (1985) Circles with two tandem long terminal repeats are specifically cleaved by pol gene-associated endonuclease from avian sarcoma and leukosis viruses: nucleotide sequences required for site-specific cleavage. J Virol  56: 589–599.  <ul class="find" data-citedArticleID="965740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Circles+with+two+tandem+long+terminal+repeats+are+specifically+cleaved+by+pol+gene-associated+endonuclease+from+avian+sarcoma+and+leukosis+viruses%3A+nucleotide+sequences+required+for+site-specific+cleavage.&amp;auth=&amp;atitle=Circles+with+two+tandem+long+terminal+repeats+are+specifically+cleaved+by+pol+gene-associated+endonuclease+from+avian+sarcoma+and+leukosis+viruses%3A+nucleotide+sequences+required+for+site-specific+cleavage." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Circles+with+two+tandem+long+terminal+repeats+are+specifically+cleaved+by+pol+gene-associated+endonuclease+from+avian+sarcoma+and+leukosis+viruses%3A+nucleotide+sequences+required+for+site-specific+cleavage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Circles+with+two+tandem+long+terminal+repeats+are+specifically+cleaved+by+pol+gene-associated+endonuclease+from+avian+sarcoma+and+leukosis+viruses%3A+nucleotide+sequences+required+for+site-specific+cleavage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000608-Grandgenett2" id="pone.0000608-Grandgenett2"></a>Grandgenett DP, Vora AC (1985) Site-specific nicking at the avian retrovirus LTR circle junction by the viral pp32 DNA endonuclease. Nucleic Acids Res  13: 6205–6221.  <ul class="find" data-citedArticleID="965750" data-doi="10.1093/nar/13.17.6205"><li><a href="http://dx.doi.org/10.1093/nar/13.17.6205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-specific+nicking+at+the+avian+retrovirus+LTR+circle+junction+by+the+viral+pp32+DNA+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-specific+nicking+at+the+avian+retrovirus+LTR+circle+junction+by+the+viral+pp32+DNA+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000608-Leavitt1" id="pone.0000608-Leavitt1"></a>Leavitt AD, Rose RB, Varmus HE (1992) Both substrate and target oligonucleotide sequences affect in vitro integration mediated by human immunodeficiency virus type 1 integrase protein produced in Saccharomyces cerevisiae. J Virol  66: 2359–2368.  <ul class="find" data-citedArticleID="965766"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Both+substrate+and+target+oligonucleotide+sequences+affect+in+vitro+integration+mediated+by+human+immunodeficiency+virus+type+1+integrase+protein+produced+in+Saccharomyces+cerevisiae.&amp;auth=&amp;atitle=Both+substrate+and+target+oligonucleotide+sequences+affect+in+vitro+integration+mediated+by+human+immunodeficiency+virus+type+1+integrase+protein+produced+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Both+substrate+and+target+oligonucleotide+sequences+affect+in+vitro+integration+mediated+by+human+immunodeficiency+virus+type+1+integrase+protein+produced+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Both+substrate+and+target+oligonucleotide+sequences+affect+in+vitro+integration+mediated+by+human+immunodeficiency+virus+type+1+integrase+protein+produced+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.XML" value="83552"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.PDF" value="746361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g001.PNG_L" value="92821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g001.PNG_M" value="61418"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g001.PNG_S" value="7508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g001.TIF" value="132784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g001.PNG_I" value="20664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g002.PNG_L" value="256535"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g002.PNG_M" value="121403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g002.PNG_S" value="10458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g002.TIF" value="259592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g002.PNG_I" value="38724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g003.PNG_L" value="346582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g003.PNG_M" value="97996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g003.PNG_S" value="15462"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g003.TIF" value="766114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g003.PNG_I" value="118858"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g004.PNG_L" value="214460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g004.PNG_M" value="88148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g004.PNG_S" value="8896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g004.TIF" value="296848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g004.PNG_I" value="65230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g005.PNG_L" value="347834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g005.PNG_M" value="175551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g005.PNG_S" value="9978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g005.TIF" value="439442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g005.PNG_I" value="59423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g006.PNG_L" value="207558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g006.PNG_M" value="105570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g006.PNG_S" value="11531"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g006.TIF" value="380238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.g006.PNG_I" value="69454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.t001.PNG_L" value="48662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.t001.PNG_M" value="61546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.t001.PNG_S" value="11028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.t001.TIF" value="222668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000608.t001.PNG_I" value="21381"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000608&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000608" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000608&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000608&volume=&issue=&title=Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase&author_name=Olivier%20Delelis%2C%20Vincent%20Parissi%2C%20Herv%C3%A9%20Leh%2C%20Gladys%20Mbemba%2C%20Caroline%20Petit%2C%20Pierre%20Sonigo%2C%20Eric%20Deprez%2C%20Jean-Fran%C3%A7ois%20Mouscadet&start_page=1&end_page=8" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000608" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608&amp;t=Efficient%20and%20Specific%20Internal%20Cleavage%20of%20a%20Retroviral%20Palindromic%20DNA%20Sequence%20by%20Tetrameric%20HIV-1%20Integrase" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608&title=Efficient%20and%20Specific%20Internal%20Cleavage%20of%20a%20Retroviral%20Palindromic%20DNA%20Sequence%20by%20Tetrameric%20HIV-1%20Integrase&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608&amp;title=Efficient%20and%20Specific%20Internal%20Cleavage%20of%20a%20Retroviral%20Palindromic%20DNA%20Sequence%20by%20Tetrameric%20HIV-1%20Integrase" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000608&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Efficient and Specific Internal Cleavage of a Retroviral Palindromic DNA Sequence by Tetrameric HIV-1 Integrase';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000608';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Efficient%20and%20Specific%20Internal%20Cleavage%20of%20a%20Retroviral%20Palindromic%20DNA%20Sequence%20by%20Tetrameric%20HIV-1%20Integrase http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000608" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000608" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">














          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cofactors+%28biochemistry%29%22" title="Search for articles in the subject area:'Cofactors (biochemistry)'"><div class="flagText">Cofactors (biochemistry)</div></a>
              <div data-categoryid="31901" data-articleid="24054"
                   data-categoryname="Cofactors (biochemistry)"
                   class="flagImage" title="Flag 'Cofactors (biochemistry)' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cross-linking%22" title="Search for articles in the subject area:'Cross-linking'"><div class="flagText">Cross-linking</div></a>
              <div data-categoryid="22007" data-articleid="24054"
                   data-categoryname="Cross-linking"
                   class="flagImage" title="Flag 'Cross-linking' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+cleavage%22" title="Search for articles in the subject area:'DNA cleavage'"><div class="flagText">DNA cleavage</div></a>
              <div data-categoryid="31951" data-articleid="24054"
                   data-categoryname="DNA cleavage"
                   class="flagImage" title="Flag 'DNA cleavage' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequence+analysis%22" title="Search for articles in the subject area:'DNA sequence analysis'"><div class="flagText">DNA sequence analysis</div></a>
              <div data-categoryid="47935" data-articleid="24054"
                   data-categoryname="DNA sequence analysis"
                   class="flagImage" title="Flag 'DNA sequence analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="24054"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="24054"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oligomers%22" title="Search for articles in the subject area:'Oligomers'"><div class="flagText">Oligomers</div></a>
              <div data-categoryid="20627" data-articleid="24054"
                   data-categoryname="Oligomers"
                   class="flagImage" title="Flag 'Oligomers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Repeated+sequences%22" title="Search for articles in the subject area:'Repeated sequences'"><div class="flagText">Repeated sequences</div></a>
              <div data-categoryid="20009" data-articleid="24054"
                   data-categoryname="Repeated sequences"
                   class="flagImage" title="Flag 'Repeated sequences' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=3096'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2318'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5539&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=16295">Referee Comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=2301">Referee Comments: Referee 2</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
