

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001203"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001203" />

    <meta name="citation_title" content="Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues"/>
    <meta itemprop="name" content="Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues"/>

      <meta name="citation_author" content="Henrik Hornshøj"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark"/>
      <meta name="citation_author" content="Lene Nagstrup Conley"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark"/>
      <meta name="citation_author" content="Jakob Hedegaard"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark"/>
      <meta name="citation_author" content="Peter Sørensen"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark"/>
      <meta name="citation_author" content="Frank Panitz"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark"/>
      <meta name="citation_author" content="Christian Bendixen"/>
            <meta name="citation_author_institution" content="Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark"/>

    <meta name="citation_date" content="2007/11/21"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001203.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1203"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A compendium of gene expression in normal human tissues.; citation_author=LL Hsiao; citation_author=F Dangond; citation_author=T Yoshida; citation_author=R Hong; citation_author=RV Jensen; citation_journal_title=Physiol Genomics; citation_volume=7; citation_number=1; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Interpreting expression profiles of cancers by genome-wide survey of breadth of expression in normal tissues.; citation_author=X Ge; citation_author=S Yamamoto; citation_author=S Tsutsumi; citation_author=Y Midorikawa; citation_author=S Ihara; citation_journal_title=Genomics; citation_volume=86; citation_number=2; citation_pages=127-141; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=GeneNote: whole genome expression profiles in normal human tissues.; citation_author=O Shmueli; citation_author=S Horn-Saban; citation_author=V Chalifa-Caspi; citation_author=M Shmoish; citation_author=R Ophir; citation_journal_title=C R Biol; citation_volume=326; citation_number=3; citation_pages=1067-1072; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.; citation_author=AI Su; citation_author=T Wiltshire; citation_author=S Batalov; citation_author=H Lapp; citation_author=KA Ching; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=4; citation_pages=6062-6067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The functional landscape of mouse gene expression.; citation_author=W Zhang; citation_author=QD Morris; citation_author=R Chang; citation_author=O Shai; citation_author=MA Bakowski; citation_journal_title=J Biol; citation_volume=3; citation_number=5; citation_pages=21; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genome-wide profiling of gene expression in 29 normal human tissues with a cDNA microarray.; citation_author=A Saito-Hisaminato; citation_author=T Katagiri; citation_author=S Kakiuchi; citation_author=T Nakamura; citation_author=T Tsunoda; citation_journal_title=DNA Res; citation_volume=9; citation_number=6; citation_pages=35-45; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A DNA microarray survey of gene expression in normal human tissues.; citation_author=R Shyamsundar; citation_author=YH Kim; citation_author=JP Higgins; citation_author=K Montgomery; citation_author=M Jorden; citation_journal_title=Genome Biol; citation_volume=6; citation_number=7; citation_pages=R22; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Large-scale analysis of the human and mouse transcriptomes.; citation_author=AI Su; citation_author=MP Cooke; citation_author=KA Ching; citation_author=Y Hakak; citation_author=JR Walker; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=8; citation_pages=4465-4470; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A compendium of gene expression in normal human tissues.; citation_author=LL Hsiao; citation_author=F Dangond; citation_author=T Yoshida; citation_author=R Hong; citation_author=RV Jensen; citation_journal_title=Physiol Genomics; citation_volume=7; citation_number=9; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Interpreting expression profiles of cancers by genome-wide survey of breadth of expression in normal tissues.; citation_author=X Ge; citation_author=S Yamamoto; citation_author=S Tsutsumi; citation_author=Y Midorikawa; citation_author=S Ihara; citation_journal_title=Genomics; citation_volume=86; citation_number=10; citation_pages=127-141; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A compendium of gene expression in normal human tissues.; citation_author=LL Hsiao; citation_author=F Dangond; citation_author=T Yoshida; citation_author=R Hong; citation_author=RV Jensen; citation_journal_title=Physiol Genomics; citation_volume=7; citation_number=11; citation_pages=97-104; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=GeneNote: whole genome expression profiles in normal human tissues.; citation_author=O Shmueli; citation_author=S Horn-Saban; citation_author=V Chalifa-Caspi; citation_author=M Shmoish; citation_author=R Ophir; citation_journal_title=C R Biol; citation_volume=326; citation_number=12; citation_pages=1067-1072; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A gene atlas of the mouse and human protein-encoding transcriptomes.; citation_author=AI Su; citation_author=T Wiltshire; citation_author=S Batalov; citation_author=H Lapp; citation_author=KA Ching; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=13; citation_pages=6062-6067; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=NCBI GEO: mining tens of millions of expression profiles–database and tools update.; citation_author=T Barrett; citation_author=DB Troup; citation_author=SE Wilhite; citation_author=P Ledoux; citation_author=D Rudnev; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=14; citation_pages=D760-D765; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.; citation_author=M Ashburner; citation_author=CA Ball; citation_author=JA Blake; citation_author=D Botstein; citation_author=H Butler; citation_journal_title=Nat Genet; citation_volume=25; citation_number=15; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=BayGO: Bayesian analysis of ontology term enrichment in microarray data.; citation_author=RZ Vencio; citation_author=T Koide; citation_author=SL Gomes; citation_author=CA Pereira; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=16; citation_pages=86; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The KEGG resource for deciphering the genome.; citation_author=M Kanehisa; citation_author=S Goto; citation_author=S Kawashima; citation_author=Y Okuno; citation_author=M Hattori; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=17; citation_pages=D277-D280; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A global test for groups of genes: testing association with a clinical outcome.; citation_author=JJ Goeman; citation_author=SA van de Geer; citation_author=F de Kort; citation_author=HC van Houwelingen; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=18; citation_pages=93-99; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2007.; citation_author=TJ Hubbard; citation_author=BL Aken; citation_author=K Beal; citation_author=B Ballester; citation_author=M Caccamo; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=19; citation_pages=D610-D617; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=20; citation_pages=D5-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=SMART 5: domains in the context of genomes and networks.; citation_author=I Letunic; citation_author=RR Copley; citation_author=B Pils; citation_author=S Pinkert; citation_author=J Schultz; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=21; citation_pages=D257-D260; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary conservation of expression profiles between human and mouse orthologous genes.; citation_author=BY Liao; citation_author=J Zhang; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=22; citation_pages=530-540; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cloning of the cDNA for a human homologue of the Drosophila white gene and mapping to chromosome 21q22.3.; citation_author=H Chen; citation_author=C Rossier; citation_author=MD Lalioti; citation_author=A Lynn; citation_author=A Chakravarti; citation_journal_title=Am J Hum Genet; citation_volume=59; citation_number=23; citation_pages=66-75; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Muscle protein breakdown and the critical role of the ubiquitin-proteasome pathway in normal and disease states.; citation_author=SH Lecker; citation_author=V Solomon; citation_author=WE Mitch; citation_author=AL Goldberg; citation_journal_title=J Nutr; citation_volume=129; citation_number=24; citation_pages=227S-237S; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Interpreting expression profiles of cancers by genome-wide survey of breadth of expression in normal tissues.; citation_author=X Ge; citation_author=S Yamamoto; citation_author=S Tsutsumi; citation_author=Y Midorikawa; citation_author=S Ihara; citation_journal_title=Genomics; citation_volume=86; citation_number=25; citation_pages=127-141; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Osteopontin identified as lead marker of colon cancer progression, using pooled sample expression profiling.; citation_author=D Agrawal; citation_author=T Chen; citation_author=R Irby; citation_author=J Quackenbush; citation_author=AF Chambers; citation_journal_title=J Natl Cancer Inst; citation_volume=94; citation_number=26; citation_pages=513-521; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Porcine transcriptome analysis based on 97 non-normalized cDNA libraries and assembly of 1,021,891 expressed sequence tags.; citation_author=J Gorodkin; citation_author=S Cirera; citation_author=J Hedegaard; citation_author=MJ Gilchrist; citation_author=F Panitz; citation_journal_title=Genome Biol; citation_volume=8; citation_number=27; citation_pages=R45; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=28; citation_pages=D5-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=An extensible application for assembling annotation for genomic data.; citation_author=J Zhang; citation_author=V Carey; citation_author=R Gentleman; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=29; citation_pages=155-156; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Bioconductor: an open source framework for bioinformatics and computational biology.; citation_author=M Reimers; citation_author=VJ Carey; citation_journal_title=Methods Enzymol; citation_volume=411; citation_number=30; citation_pages=119-134; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation.; citation_author=YH Yang; citation_author=S Dudoit; citation_author=P Luu; citation_author=DM Lin; citation_author=V Peng; citation_journal_title=Nucleic Acids Res; citation_volume=30; citation_number=31; citation_pages=e15; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Linear models and empirical bayes methods for assessing differential expression in microarray experiments.; citation_author=GK Smyth; citation_journal_title=Stat Appl Genet Mol Biol; citation_volume=3; citation_number=32; citation_pages=Article3; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=NCBI GEO: mining tens of millions of expression profiles–database and tools update.; citation_author=T Barrett; citation_author=DB Troup; citation_author=SE Wilhite; citation_author=P Ledoux; citation_author=D Rudnev; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=33; citation_pages=D760-D765; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A global test for groups of genes: testing association with a clinical outcome.; citation_author=JJ Goeman; citation_author=SA van de Geer; citation_author=F de Kort; citation_author=HC van Houwelingen; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=34; citation_pages=93-99; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ensembl 2007.; citation_author=TJ Hubbard; citation_author=BL Aken; citation_author=K Beal; citation_author=B Ballester; citation_author=M Caccamo; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=35; citation_pages=D610-D617; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information.; citation_author=DL Wheeler; citation_author=T Barrett; citation_author=DA Benson; citation_author=SH Bryant; citation_author=K Canese; citation_journal_title=Nucleic Acids Res; citation_volume=35; citation_number=36; citation_pages=D5-12; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=SMART 5: domains in the context of genomes and networks.; citation_author=I Letunic; citation_author=RR Copley; citation_author=B Pils; citation_author=S Pinkert; citation_author=J Schultz; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=37; citation_pages=D257-D260; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Evolutionary conservation of expression profiles between human and mouse orthologous genes.; citation_author=BY Liao; citation_author=J Zhang; citation_journal_title=Mol Biol Evol; citation_volume=23; citation_number=38; citation_pages=530-540; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues"/>
    <meta name="twitter:description" content="BackgroundGene expression microarrays have been intensively applied to screen for genes involved in specific biological processes of interest such as diseases or responses to environmental stimuli. For mammalian species, cataloging of the global gene expression profiles in large tissue collections under normal conditions have been focusing on human and mouse genomes but is lacking for the pig genome.Methodology/Principal FindingsHere we present the results from a large-scale porcine study establishing microarray cDNA expression profiles of approximately 20.000 genes across 23 healthy tissues. As expected, a large portion of the genes show tissue specific expression in agreement with mappings to gene descriptions, Gene Ontology terms and KEGG pathways. Two-way hierarchical clustering identified expected tissue clusters in accordance with tissue type and a number of cDNA clusters having similar gene expression patterns across tissues. For one of these cDNA clusters, we demonstrate that possible tissue associated gene function can be inferred for previously uncharacterized genes based on their shared expression patterns with functionally annotated genes. We show that gene expression in common porcine tissues is similar to the expression in homologous tissues of human.Conclusions/SignificanceThe results from this study constitute a valuable and publicly available resource of basic gene expression profiles in normal porcine tissues and will contribute to the identification and functional annotation of porcine genes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001203.g005"/>

  <meta property="og:title" content="Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6279'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=5107'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7762&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001203">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001203" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001203&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Henrik Hornshøj, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lene Nagstrup Conley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jakob Hedegaard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter Sørensen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Frank Panitz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christian Bendixen
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:christian.bendixen@agrsci.dk">christian.bendixen@agrsci.dk</a></p>

                <p>Affiliation:
                  Department of Genetics and Biotechnology, Faculty of Agricultural Sciences, University of Aarhus, Tjele, Denmark
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 21, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001203</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001203-g001" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001203-g002" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001203-g003" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001203-g004" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001203-g005" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001203">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001203" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Gene expression microarrays have been intensively applied to screen for genes involved in specific biological processes of interest such as diseases or responses to environmental stimuli. For mammalian species, cataloging of the global gene expression profiles in large tissue collections under normal conditions have been focusing on human and mouse genomes but is lacking for the pig genome.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Here we present the results from a large-scale porcine study establishing microarray cDNA expression profiles of approximately 20.000 genes across 23 healthy tissues. As expected, a large portion of the genes show tissue specific expression in agreement with mappings to gene descriptions, Gene Ontology terms and KEGG pathways. Two-way hierarchical clustering identified expected tissue clusters in accordance with tissue type and a number of cDNA clusters having similar gene expression patterns across tissues. For one of these cDNA clusters, we demonstrate that possible tissue associated gene function can be inferred for previously uncharacterized genes based on their shared expression patterns with functionally annotated genes. We show that gene expression in common porcine tissues is similar to the expression in homologous tissues of human.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The results from this study constitute a valuable and publicly available resource of basic gene expression profiles in normal porcine tissues and will contribute to the identification and functional annotation of porcine genes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hornshøj H, Conley LN, Hedegaard J, Sørensen P, Panitz F, et al.  (2007) Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues. PLoS ONE 2(11):
          e1203.
            doi:10.1371/journal.pone.0001203</p><p><strong>Academic Editor: </strong>Jörg Hoheisel, Deutsches Krebsforschungszentrum, Germany</p><p><strong>Received:</strong> June 29, 2007; <strong>Accepted:</strong> October 26, 2007; <strong>Published:</strong> November 21, 2007</p><p><strong>Copyright:</strong> © 2007 Hornshøj et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Basic university funding provided. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The microarray technology is ideal for whole-genome and large-scale profiling of gene expression under various conditions. For instance, microarray-based experiments have been widely used to identify differentially expressed genes involved in specific biological processes such as disease or response to environmental stimuli. These experiments have found new gene functions and provide insights into the transcriptional regulation that underlies biological processes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Several gene expression surveys from human and mouse studies have demonstrated important applications of gene expression profiles obtained from normal functioning organs and tissues <a href="#pone.0001203-Hsiao1">[1]</a>–<a href="#pone.0001203-Su2">[8]</a>. For example, genes that are tissue specific have been identified and used to describe the biological processes associated with human organs <a href="#pone.0001203-Hsiao2">[9]</a>. In addition, integration of gene expression profiles from healthy tissues has been shown to be valuable in the biological interpretation of expression profiles from human cancer cells <a href="#pone.0001203-Ge2">[10]</a>. Compendiums of gene expression, such as Human Gene Expression (HuGE) Index <a href="#pone.0001203-Hsiao3">[11]</a>, Gene Normal Tissue Expression (GeneNote) <a href="#pone.0001203-Shmueli2">[12]</a> and SymAtlas <a href="#pone.0001203-Su3">[13]</a>, have also been created as publicly available web resource as a result of gene expression surveys. They provide easy access to tissue expression levels for single genes. Finally, co-expressed genes have been used to predict function of previously uncharacterized genes <a href="#pone.0001203-Zhang1">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Domesticated pig (Sus scrofa) was our choice of model organism for large scale gene expression profiling as it provides easy access to samples of tissues, which are physiologically and anatomically similar to those of other mammalians used in biomedical research, such as human and mouse. At present, no porcine whole genome surveys of gene expression across large tissue collections have been reported. Here we present the results from a large-scale porcine survey of expression profiles of 26.877 microarray cDNAs representing approximately 20.000 genes in 23 healthy tissues. The overall intention of this study has been to catalog the basic expression profiles of as many genes as possible in a large collection of normal functioning porcine tissues and make this publicly available. The results from this study have been made publicly available via Gene Expression Omnibus (GEO) <a href="#pone.0001203-Barrett1">[14]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Gene annotations such as Gene Ontology (GO) terms and KEGG pathways are integrated in the analysis and interpretation of expression profiles. GO terms constitute a controlled vocabulary of Biological Processes (BP), Molecular Functions (MF) and Cellular Components (CC) for gene products <a href="#pone.0001203-Ashburner1">[15]</a>. GO has been widely used as a tool for the interpretation of microarray differential gene expression by grouping genes according to mapped GO terms instead of looking at single genes. One common approach is to statistically test for enriched ontology terms in microarray data <a href="#pone.0001203-Vencio1">[16]</a>. The Kyoto Encyclopedia of Genes and Genomes (KEGG) represent the current knowledge on molecular interaction networks such as pathways <a href="#pone.0001203-Kanehisa1">[17]</a>. By KEGG classification of microarray genes one can identify pathways that are associated with certain phenotypes or disease states, for example leukemia <a href="#pone.0001203-Goeman1">[18]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>We identify single genes with differences in expression across tissues and use two-way hierarchical clustering to group tissues and genes according to their expression profiles. In an attempt to infer potential tissue functions for previously uncharacterized genes, we investigate the expression patterns, functional annotations and cDNA sequences of a gene cluster. Finally, we compare the gene expression profiles of orthologous genes in nine common tissues across pig and human.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Establishing microarray gene expression profiles</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In total, 46 two-channel hybridizations were carried out corresponding to 23 tissue samples each with two independent RNA extractions. A common reference sample was constructed from a pool of all labeled RNA extracts. The ratio between the mRNA levels in each tissue and the common reference was computed for all microarray cDNA spots generating 46 sets of relative gene expression profiles for 26.877 porcine cDNAs (PCs). The data was normalized using the print tip loess method as described in the methods section. The raw intensities and normalized gene expression ratios were made publicly available by submission to GEO and can be accessed via accession number GSE4918.</p>


<h4>Assigning known processes and pathways to tissue gene expression</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We first identified differences between the expression in each of the 23 tissues and the overall expression defined by the expression in other 22 tissues. The microarray cDNAs were mapped to GO terms of class BP to assign known biological processes to the identified tissue differences in expression. The GO terms mapped to positively regulated genes were subjected to an enrichment test to compare tissue expression with known biological processes (<a href="#pone-0001203-g001">Figure 1</a>). As expected, we were able to find numerous expected matches between observed tissue specific gene expression and GO biological processes that directly associate with specific tissues. For the genes showing increased expression in muscle type tissues we found ‘muscle contraction’ to be a significantly enriched GO BP term in all of the nine muscle type tissues used in the experiment. Similarly ‘muscle development’ was found to be an enriched GO BP term in five muscle tissues (Biceps femoris, Longissimus dorsi, Semimembranosus, Supraspinatus and Vastus intermedius). Increased expression of genes assigned to ‘circulation’ was observed exclusively in heart tissue. Longissimus dorsi and Semimembranosus appear to be the most similar tissues among the muscle type tissues in terms of sharing most GO BP terms (10 out of 49). Genes assigned to the liver associated process ‘steroid metabolism’ are positively regulated in the liver tissue. Genes assigned to brain processes such as ‘synaptic transmission’, ‘nervous system development’ and ‘intracellular signaling’ were found to have specifically higher expression levels in Cerebellum and Frontal cortex. ‘Epidermis development’, a process associated with skin, is assigned to genes showing increased expression levels in skin. ‘Biosynthesis’ is the most frequent process associated with increased gene expression by its appearance in 15 of the 23 tissues.</p>
<div class="figure" id="pone-0001203-g001"><div class="img"><a name="pone-0001203-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Enriched GO BP terms for cDNAs representing positively regulated genes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Size of dots corresponds to the number of cDNAs that were tested (minimum = 6 and maximum = 1025) and color codes indicate tissue type. Only significant (P≤0.01) GO-BP terms represented by 50 or more cDNAs on the array were included.</p>
<span>doi:10.1371/journal.pone.0001203.g001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In general, none of the tissue-specific terms were found to be enriched for genes positively regulated in other tissues than expected. However, some tissue-specific terms were not enriched in all those tissues as might have been expected. For example the process ‘muscle development’ was only found for positively regulated genes in five of the nine muscle tissues.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The genes represented on the microarray where also mapped to available KEGG pathways based on the gene accession IDs. The cDNAs were grouped by these pathways and a global test was carried out on the normalized data set to assign possible biochemical pathways with positively regulated genes (<a href="#pone-0001203-g002">Figure 2</a>). For several pathway genes we identified expected tissue specific expression profiles in clear agreement with the pathway function. For example, ‘regulation of actin cytoskeleton’ is a muscle associated pathway and the genes that represent this pathway on the microarray show increased expression in four muscle type tissues (Biceps femoris, Longissimus dorsi, Semimembranosus and Triceps brachii). Again, Longissimus dorsi and Semimembranosus appear to be the most similar tissues among the muscle type tissues. These two tissues share 24 out of 68 KEGG pathways whose genes are positively regulated. Two known liver tissue associated pathways named ‘metabolism of xenobiotics by cytochrome P450’ and ‘complement and coagulation cascades’ are represented by genes that are positively regulated in the liver tissue.</p>
<div class="figure" id="pone-0001203-g002"><div class="img"><a name="pone-0001203-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>KEGG pathways for cDNAs representing differently expressed genes across tissues.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Size of dots corresponds to the number of cDNAs (minimum = 10 and maximum = 253) that were tested to have a positive influence on the expression levels and color codes indicate tissue type. Only significant (P≤0.05) KEGG pathways represented by 50 or more cDNAs on the array were included.</p>
<span>doi:10.1371/journal.pone.0001203.g002</span></div>

<h4>Clustering of gene expression profiles by tissues</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Two-way hierarchical clustering was applied to the normalized expression profiles to identify clusters of genes and tissues displaying similar expression patterns (<a href="#pone-0001203-g003">Figure 3A</a>). The tissue dendogram shows that the tissues cluster into groups as expected from their tissue types. The nine muscle type tissues Biceps femoris, heart, Infraspinatus, Longissimus dorsi, Semimembranosus, Supraspinatus, Semitendinosus, Triceps brachii and Vastus intermedius cluster into one single tissue group. Although located in this muscle cluster, the expression profiles of heart tissue is slightly different than those of the other muscle tissues. Infraspinatus and Supraspinatus, which are both muscles on the dorsum of the scapula, form a subcluster in the muscle tissue group. Thyroid gland and pituitary gland are members of the same cluster, which surprisingly also includes lung. The fact that thyroid gland and lung tend to cluster together may result from shared positively regulated genes involved in amino acid metabolisms and cell adhesion (see <a href="#pone-0001203-g002">Figure 2</a>). Large intenstine, small intestine and stomach form a group of digestive tissues of which the first two are located in a separate subcluster. Likewise, Frontal cortex and Cerebellum, define a cluster of brain tissues. Pancreas and liver tissues show very distinct expression patterns compared to any other tissues.</p>
<div class="figure" id="pone-0001203-g003"><div class="img"><a name="pone-0001203-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Two-way hierarchical clustering of gene expression ratios.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Heatmaps displays the log2(M) on a color scale from green indicating lower expression to red indicating a higher expression, interpolated over black for log2(M) = 0. A. Overview of entire hierarchical clustering showing 26.877 cDNAs (row-wise) and 23 tissues (column-wise). B. Enlarged view of selected cDNA cluster. BFE, Biceps femoris; CBE, Cerebellum; FAT, fat; FCO, Frontal cortex; HEA, heart; ISP, Infraspinatus; KID, kidney; LDO, Longissimus dorsi; LIN, large intestine; LIV, liver; LUN, lung; PAN, pancreas; PGL, pituitary gland; SIN, small intestine; SKI, skin; SME, Semimembranosus; SPL, spleen; SSP, Supraspinatus; STE, Semitendinosus; STO, stomach; TBR, Triceps brachii; THG, thyroid gland; VIN, Vastus intermedius.</p>
<span>doi:10.1371/journal.pone.0001203.g003</span></div>

<h4>Co-expression of uncharacterized genes with known genes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Several cDNA clusters, whose members share expression profiles across tissues, can be identified on top part of the heatmap in <a href="#pone-0001203-g003">Figure 3A</a>. One of the identified cDNA clusters has 60 members of which most are positively regulated in muscle type tissues and negatively regulated in non-muscle type tissues (see <a href="#pone-0001203-g003">Figure 3B</a>). We selected this cDNA cluster for further investigation in an attempt to infer functions for uncharacterized genes based on co-expression with known genes and sequence analysis. As anticipated, the description for the genes mapped to these cluster cDNAs reveals many genes with muscle-specific function, for example myosins (MYH2, MYH4, MYH6 and MYL4), tropomyosin (TPM2), myotilin (MYOT), troponins (TNNC2, TNNI2, TNNI3, TNNT1, TNNT2 and TNNT3), carbonic anhydrase III (CA3) and actins (ACTC and ACTG1). Thus, the observed muscle specific expression profiles of this cluster are largely confirmed by the assigned gene annotations.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>There are four non-annotated cDNA members (PC_206947, PC_207214, PC_207218, PC_207224) of this particular cluster (see <a href="#pone-0001203-g003">Figure 3B</a>). The sequences of these cDNA members do not show significant nucleotide BLASTN similarity (P-value≤0.1) to any known human or other mammalian gene transcript. These cDNAs were considered to represent porcine genes with no available annotations, but whose expression is predominantly muscle specific. We further investigated the cDNA sequences and gene expression patterns of this cluster to infer the possible function of the corresponding uncharacterized genes. We first compared the cDNA sequences with publicly available porcine genome sequences at Ensembl Trace Server <a href="#pone.0001203-Hubbard1">[19]</a> to increase the sequence representation and found genome sequence(s) with similarity to three of the cDNAs (PC_207214, PC_207218, PC_207224). A BLASTP comparison of the amino acid translated genome sequences to NCBI's non-redundant (nr) sequence database <a href="#pone.0001203-Wheeler1">[20]</a> identified similarity (P-value = 5×10<sup>−37</sup>) to an additional protein for PC_207218 termed ATP-binding cassette transporter G1 (ABCG1) from human. Another cDNA (PC_207826) is found on the microarray whose gene also encodes an ABC transporter protein termed ATP-binding cassette, sub-family A (ABC1), member 10 (ABCA10). Also, the expression patterns of these two ABC transporter genes represented by PC_207218 and PC_207826 are similar (see <a href="#pone-0001203-g004">Figure 4</a>). Protein domain prediction in the amino acid sequence translated from cDNA and genome sequences using the SMART web program <a href="#pone.0001203-Letunic1">[21]</a> showed the presence of transmembrane regions in the protein products of the assumed genes represented by PC_206947, PC_207214, PC_207218. Signal peptides were also found in the amino acid sequences of PC_207214 and PC_207218. The amino acid sequence for PC_207224 was predicted to contain an ubiquitin conjugating-like (UBC-like) protein domain (P-value = 1.7×10<sup>−2</sup>). Another microarray cDNA named PC_214110 represents the gene encoding ubiquitin-conjugating enzyme E2D 2 (UBE2D2) and the expression patterns if this gene appears to be similar to PC_207224 (see <a href="#pone-0001203-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001203-g004"><div class="img"><a name="pone-0001203-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Expression profiles of uncharacterized genes and co-expressed genes with known function.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Each histogram represents the expression profiles of a single cDNA across all 23 tissues for two uncharacterized protein coding genes (PC_207218, PC_207224) and four co-expressed genes with known function (PC_207826: ABCA10, PC_214110: UBE2D2). Each bar represents the gene expression ratio (M) between the tissue sample and the common reference sample on the logarithmic scale. M values below zero, indicating lower gene expression level, are shown by green bars. M values above zero, indicating higher expression, are shown by red bars.</p>
<span>doi:10.1371/journal.pone.0001203.g004</span></div>

<h4>Correlation with gene expression in human tissues</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To identify similarities in tissue gene expression between pig and human we computed Pearson's correlation coefficients for pairs of common tissues across these two species. We first identified a study, in which the gene expression profiles of 35 human tissue types were also established using a cDNA-based microarray experiment with a common reference design <a href="#pone.0001203-Shyamsundar1">[7]</a>. Nine common tissues were found to be present in both our study and the human study (Frontal cortex, heart, kidney, liver, lung, pancreas, spleen, stomach and Thyroid gland). We identified orthologous relationship for 3.861 genes across the microarray platforms of these two studies based on the best reciprocal BLASTN hit between cDNA sequences and identical mapping of cDNA sequences to gene IDs (see <a href="#s4">Methods</a>). We then extracted the expression data for the 3.861 orthologous genes in the nine common tissues of these two studies. In order to make the gene expression profiles from these two studies as comparable as possible we followed a recently proposed approach, in which relative abundance (RA) values are computed as the measurement for gene expression levels and used for cross-species comparison <a href="#pone.0001203-Liao1">[22]</a>. RA is defined by the raw signal intensities in the individual tissues divided by the sum of raw signal intensity in all nine tissues.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Using Pearson's method we computed pair wise correlation coefficients between common tissues across pig and human, which is shown in <a href="#pone-0001203-g005">Figure 5</a>. The correlation in gene expression between porcine and human is clearly higher for homologous tissue pairs in comparison to the correlation between non-homologous tissues with the exception of pancreas. The highest correlation was found for liver followed by Frontal cortex, heart, lung, stomach, thyroid, spleen and kidney.</p>
<div class="figure" id="pone-0001203-g005"><div class="img"><a name="pone-0001203-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001203" data-uri="info:doi/10.1371/journal.pone.0001203.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001203.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001203.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001203.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001203.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Correlation in gene expression between common tissues of pig and human.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Pearson's correlation coefficients computed for pairs of common tissues from pig and human. Color scale from white to dark blue represents correlation coefficients from −0.10 to 0.31.</p>
<span>doi:10.1371/journal.pone.0001203.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We carried out a large-scale survey of gene expression in porcine by establishing expression profiles for approximately 20.000 genes represented by 26.877 microarray cDNAs across 23 healthy and diverse tissue samples. We analyzed the expression in each tissue compared the overall expression.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Increased tissue specific gene expression corresponds to numerous of the major tissue specific GO BP terms and KEGG pathways as expected (see <a href="#pone-0001203-g001">Figure 1</a> and <a href="#pone-0001203-g002">2</a>). The muscle type tissues Longissimus dorsi and Semimembranosus share most enriched GO terms and KEGG pathways among the muscle type tissues. Longissimus dorsi is one of the deep muscles of the back whereas Semimembranosus is situated at the back of the thigh, but both muscles are type II muscle fibers. The integration of functional gene annotations in the analysis confirms the detected differences in gene expression across tissues and confirms the expression of porcine genes being as expected in terms of assigned known tissue associated function.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Two-way hierarchical clustering of the gene expression profiles formed expected clusters according to tissue types (see <a href="#pone-0001203-g003">Figure 3</a>). The gland tissue thyroid gland and pituitary gland are both members of the same cluster. Thyroid gland forms a subcluster together with lung. This might be explained from the fact that these two tissues share positive regulation of genes that are mapped to the KEGG pathways of several amino acid metabolisms and cell adhesion molecules (CAMs) (see <a href="#pone-0001203-g002">Figure 2</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Clusters of cDNAs who share expression profiles across one or more tissues were identified by two-way hierarchical clustering. In an attempt to identify expression of uncharacterized genes and infer possible function for these genes we selected one of these clusters and investigated the shared expression patterns with known genes and cDNA sequences. This cluster has 60 cDNA members predominantly with high expression levels in muscle type tissues and low expression levels in non-muscle type tissues (see <a href="#pone-0001203-g003">Figure 3</a>).</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Four members of the selected cDNA cluster did not produce significant BLASTN nucleotide similarity to known human protein coding sequences in NCBI's nr sequence database. These cDNAs were considered to represent previously uncharacterized genes. A further analysis of expression profiles and sequence analysis was carried out on the cluster cDNAs in order to infer potential function for these muscle specific genes. The fact that protein domains can be predicted in the translated amino acid sequences suggest that the cDNA sequences are in fact protein coding and the missing sequence similarity to other protein coding sequences suggest the existence of novel and expressed porcine genes. These assumed novel genes encode proteins that are likely involved in muscle-specific processes such as ‘muscle development’ and ‘muscle contraction’, which are frequent GO terms in this cluster. A possible reason for the missing transmembrane domain and signal peptide protein domain predictions for PC_207224 could be that it represents an untranslated region (UTR) rather than protein coding regions. An additional gene product from human, termed ATP-binding cassette transport G1 (ABCG1) was found to be similar to the translated genome sequences for the PC207218 cDNA. The ABCG1 gene is expressed in many human tissues <a href="#pone.0001203-Chen1">[23]</a>, but apparently not in a muscle specific manner. The presence of an ABCG1 gene is supported by the predicted transmembrane domain in the translated amino acid sequence of PC_207218. Also the cDNA termed PC_207826 represents a member of the ABC transporter protein family named ATP-binding cassette, sub-family A (ABC1), member 10 (ABCA10), which shows similar expression across the tissues (see <a href="#pone-0001203-g004">Figure 4</a>). This gene has no mappings to GO terms, which could have indicated a tissue related function. The expression profiles observed in this study, however, suggest that ABC transporter proteins play a role in a muscle associated process.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>In the amino acid sequence translated from the PC_207224 cDNA sequence, the SMART program predicts an ubiquitin conjugating-like (UBC-like) protein domain, although with low significance (P-value = 0.017). Increased expression of the ubiquitin - proteasome proteolytic pathway, which includes ubiquitin conjugating enzymes (E2s), has been associated with loss of muscle protein <a href="#pone.0001203-Lecker1">[24]</a>. Assuming that the gene represented by PC_207224 encodes an E2 enzyme, one could speculate that this enzyme is part of a pathway associated with a process in muscle involving protein loss. Another E2 gene, termed UBE2D2, is also present on the microarray represented by PC_214110, which shows a similar expression pattern (see <a href="#pone-0001203-g004">Figure 4</a>).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The presence of predicted protein domains in the translated amino acid sequence suggest that the corresponding genes are novel because the cDNA sequences are predicted to be protein coding and not UTR sequence, which otherwise might have explained the missing similarity to other protein coding sequences. The increased expression patterns of these assumed novel genes in muscle type tissues suggest that the gene functions are related to muscle-specific processes. The combination of sequence analysis with gene expression profiles provides important clues to the function of uncharacterized genes and this approach makes it worthwhile to analyze the other cDNA clusters identified. Full-length cDNA cloning and sequencing should also be applied to cDNAs representing uncharacterized genes as part of the further annotation of these genes.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>We have compared gene expression profiles for nine common tissues across pig and human using 3.861 orthologous genes by means of computing Pearson's correlation coefficients. We have found clear similarity between homologous tissues across these two species as would be expected. The highest similarity was observed between pig and human liver followed by the other eight tissues Frontal cortex, heart, lung, stomach, thyroid, spleen, kidney and pancreas. The fact that cross-species correlation in gene expression is higher between homologous tissue-pairs compared to the correlation between non-homologous tissue-pairs further confirms the established expression profiles of this study, since similar processes and therefore similar gene expression are expected to occur in the same common tissues of these two mammalian species. However, differences in technical aspects of the compared tissue expression studies are likely to reduce correlations and therefore the true biological tissue correlations are probably higher than the correlations obtained here.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Tissue samples and RNA extractions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Each of the 23 tissue samples was prepared from five healthy Hampshire gilts at age four to six months. We have used tissue sample pooling of the five gilts, a cost-effective approach for reducing effects from individuals <a href="#pone.0001203-Ge3">[25]</a> and identifying the most common differences in gene expression <a href="#pone.0001203-Agrawal1">[26]</a>. The tissue samples were immediately frozen in liquid nitrogen and subsequently kept at −80°C. Two independent RNA extractions (46 in total) were carried out from each tissue sample using the RNeasy Maxi Kit from Qiagen. The 23 tissues and their abbreviations used in this experiment: BFE, Biceps femoris; CBE, Cerebellum; FAT, fat; FCO, Frontal cortex; HEA, heart; ISP, Infraspinatus; KID, kidney; LDO, Longissimus dorsi; LIN, large intestine; LIV, liver; LUN, lung; PAN, pancreas; PGL, pituitary gland; SIN, small intestine; SKI, skin; SME, Semimembranosus; SPL, spleen; SSP, Supraspinatus; STE, Semitendinosus; STO, stomach; TBR, Triceps brachii; THG, thyroid gland; VIN, Vastus intermedius.</p>


<h4>Microarray cDNAs</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The DNA fragments were amplified from cDNA clones generated as part of large-scale sequencing of expressed sequence tags (ESTs) in the Sino-Danish Pig Genome Sequencing Project <a href="#pone.0001203-Gorodkin1">[27]</a>. The cDNA clones for PCR amplification were selected such that the cDNAs covered the largest possible number of human gene transcripts. EST clusters for human gene transcripts in NCBI's RefSeq database release 17 <a href="#pone.0001203-Wheeler2">[28]</a> were created using BLASTN sequence similarity program implemented to run on a DeCypher computer [<a href="http://www.timelogic.com">http://www.timelogic.com</a>] with P-value at or below 10<sup>−8</sup>. Within each cluster one cDNA with the minimum predicted distance to the 3′ end of the human gene transcript was selected. Microarray cDNAs were mapped to GO terms and KEGG pathways based on the human accession id, obtained from BLASTN, using the AnnBuilder package <a href="#pone.0001203-Zhang2">[29]</a> from Bioconductor <a href="#pone.0001203-Reimers1">[30]</a>. To represent uncharacterized genes on the microarray, a set of EST sequence clusters without BLASTN sequence similarity to any known human gene transcript was created and clustered using the “all-vs-all” TERACLU algorithm on a DeCypher computer from TimeLogic [<a href="http://www.timelogic.com">http://www.timelogic.com</a>]. We added one cDNA to the selection list for each of EST clusters with a minimum depth of 3 ESTs and minimum predicted distance to the 3′ end of the assembled EST contig. A total list of 27.744 elements consisting of 26.877 cDNAs and 867 control elements was created for spotting. Of the 26.877 cDNAs, 21.417 map to 15.831 human gene transcript IDs corresponding to roughly 1.35 cDNAs per gene transcript. The remaining 5.460 cDNAs were thus estimated to cover around 4.036 gene transcripts yielding approximately 19.867 gene transcripts in total. All elements were spotted in duplicates on UltraGAPS slides from Corning with an SDDC-2 ChipWriter (Biorad) yielding a total of 55.488 spots on each microarray. A full description of the cDNA microarray platform can be viewed at GEO via accession ID GPL3608.</p>


<h4>Microarray experiment</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The microarray experiment was carried out using a two-channel common reference design with two independent RNA extractions from each tissue sample. For each total-RNA extraction, 20 µg was labeled with Alexa 594 and 20 µg with Alexa 488 using SuperScript Indirect cDNA labeling System from Invitrogen. The RNA extracts labeled with Alexa 488 was collected from all tissue samples, mixed and used as reference sample referred to here as a the common reference. The RNA extracts labeled with Alexa 594 were used individually as labeled tissue samples. The labeled RNA extracts were used in two batches of 23 hybridizations corresponding to the two independent RNA extraction batches. One labeled tissue sample RNA (Alexa 594) and one labeled common reference sample RNA (Alexa 488) was thus hybridized to each of 46 microarrays. The same common reference sample was thus used in all the 46 (2×23) hybridizations. Two rounds of hybridizations on a Discovery XT hybridization station from Ventana were carried out corresponding to the two RNA batches. The hybridized microarrays were scanned and converted into TIFF images using Scanner and ScanArray Express software from Perkin Elmer.</p>


<h4>Data processing and analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Spot detection and spot intensity quantification was done using GenePix Pro version 6.0 software from Molecular Devices. Processing of the data and computation of the gene expression ratios between the tissue samples and the common reference was carried out in the R statistical programming environment version 2.3.0 [<a href="http://www.r-project.org/">http://www.r-project.org/</a>] using various Bioconductor packages <a href="#pone.0001203-Reimers1">[30]</a>. Expression ratios were defined by the signal intensity of channel one (tissue sample), divided by the signal intensity of channel two (common reference sample). The expression ratios were normalized without background correction using the print tip loess method <a href="#pone.0001203-Yang1">[31]</a> of the limma package <a href="#pone.0001203-Smyth1">[32]</a> from Bioconductor. The entire data set from the microarray gene expression experiment set has been submitted to GEO <a href="#pone.0001203-Barrett2">[33]</a> and can be queried via accession ID GSE4918. To identify cDNAs representing genes with significant differences between expression in a given tissue and the overall expression, defined by the expression in the other 22 tissues, we subjected the normalized data to an empirical Bayes method and adjusted the P-values for multiple testing using the false discovery rate (“fdr”) method <a href="#pone.0001203-Smyth1">[32]</a>. This test was carried out for all the 23 tissues. We used 5% significance level (P-values≤0.05) as a threshold and Log2(ratio) &gt;0 to select cDNAs representing positively regulated genes. GO BP term enrichment tests were carried out using the GOHyperG function of the GOStats library from Bioconductor <a href="#pone.0001203-Reimers1">[30]</a> and the global testing of genes grouped by KEGG pathway was done using Bioconductor's globaltest package <a href="#pone.0001203-Goeman2">[34]</a>. Only GO terms and KEGG pathways represented by 50 or more microarray cDNAs were included. Visualization plots and hierarchical clustering was done using SpotFire software version 8.2.1 with package DecisionSite for Functional Genomics. Additional porcine genome sequence representations for uncharacterized microarray genes were retrieved from Emsembl Trace Server <a href="#pone.0001203-Hubbard2">[35]</a>. For further analysis of the porcine cDNA and genomic sequences we used NCBI's TBLASTX web program to compare amino acid translated sequences with NCBI's nr sequence database <a href="#pone.0001203-Wheeler3">[36]</a> and the SMART web program <a href="#pone.0001203-Letunic2">[37]</a> to predict protein domains in the translated sequences. To compare the porcine gene expression profiles with corresponding gene expression in homologous tissues of human we identified a similar experiment from human, which also used a cDNA-based microarray platform and a common reference design <a href="#pone.0001203-Shyamsundar1">[7]</a>. The data set from this experiment was downloaded at GEO using Accession ID GSE2193. Using the best reciprocal BLASTN [<a href="http://www.timelogic.com/">http://www.timelogic.com/</a>] similarity hit with P-values at or below 10<sup>−8</sup> between the microarray cDNA sequences in the two experiments and requiring that the cDNA sequences for orthologous genes be mapped to the same gene ID in the two experiments, we identified 3.861 orthologous gene pairs. For each of the 3.861 orthologous genes we randomly picked one cDNA per gene in both experiment. By tissue sample comparison we were able to identify nine common tissues represented in both experiments (Frontal cortex, heart, kidney, liver, lung, pancreas, spleen, stomach and thyroid). We used relative abundance (RA) values as a measurement for gene expression levels, a previously proposed approach for optimal cross-experiment comparability <a href="#pone.0001203-Liao2">[38]</a>. RA is defined by the raw signal intensity in each individual tissue divided by the total signal intensity in all nine tissues. A data matrix with 3.861 gene rows, 2×9 = 18 tissue columns and average RA values was created. For comparative analysis we applied computation of Pearson's correlation coefficients between common tissues across pig and human on the RA data matrix using the R statistical programming environment [<a href="http://www.r-project.org/">http://www.r-project.org/</a>].</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>Authors would like to acknowledge the support from the Danish Meat Association and the Danish Ministry of Agriculture and Fisheries. The construction of the genome-wide cDNA arrays were made possible by the work of the Sino-Danish Porcine Genome Consortium.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Proposed and supervised the experimental work and contributed to the interpretation of analysis results and manuscript evaluation: CB. Revised the manuscript: FP. Contributed to the data analysis: PS. Involved in data analysis, interpretation of the analysis results and manuscript evaluation: JH. Carried out experimental work and data acquisition: LC. Participated in the experimental design/work and carried out the data analysis and drafting of the manuscript: HH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001203-Hsiao1" id="pone.0001203-Hsiao1"></a>Hsiao LL, Dangond F, Yoshida T, Hong R, Jensen RV, et al.  (2001) A compendium of gene expression in normal human tissues. Physiol Genomics  7: 97–104.  <ul class="find" data-citedArticleID="1020036"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+compendium+of+gene+expression+in+normal+human+tissues.&amp;auth=&amp;atitle=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+compendium+of+gene+expression+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001203-Ge1" id="pone.0001203-Ge1"></a>Ge X, Yamamoto S, Tsutsumi S, Midorikawa Y, Ihara S, et al.  (2005) Interpreting expression profiles of cancers by genome-wide survey of breadth of expression in normal tissues. Genomics  86: 127–141.  <ul class="find" data-citedArticleID="1020024" data-doi="10.1016/j.ygeno.2005.04.008"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.04.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpreting+expression+profiles+of+cancers+by+genome-wide+survey+of+breadth+of+expression+in+normal+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpreting+expression+profiles+of+cancers+by+genome-wide+survey+of+breadth+of+expression+in+normal+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001203-Shmueli1" id="pone.0001203-Shmueli1"></a>Shmueli O, Horn-Saban S, Chalifa-Caspi V, Shmoish M, Ophir R, et al.  (2003) GeneNote: whole genome expression profiles in normal human tissues. C R Biol  326: 1067–1072.  <ul class="find" data-citedArticleID="1020062" data-doi="10.1016/j.crvi.2003.09.012"><li><a href="http://dx.doi.org/10.1016/j.crvi.2003.09.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneNote%3A+whole+genome+expression+profiles+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneNote%3A+whole+genome+expression+profiles+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001203-Su1" id="pone.0001203-Su1"></a>Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, et al.  (2004) A gene atlas of the mouse and human protein-encoding transcriptomes. Proc Natl Acad Sci U S A  101: 6062–6067.  <ul class="find" data-citedArticleID="1020070" data-doi="10.1073/pnas.0400782101"><li><a href="http://dx.doi.org/10.1073/pnas.0400782101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001203-Zhang1" id="pone.0001203-Zhang1"></a>Zhang W, Morris QD, Chang R, Shai O, Bakowski MA, et al.  (2004) The functional landscape of mouse gene expression. J Biol  3: 21.  <ul class="find" data-citedArticleID="1020086"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+functional+landscape+of+mouse+gene+expression.&amp;auth=&amp;atitle=The+functional+landscape+of+mouse+gene+expression." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+functional+landscape+of+mouse+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+functional+landscape+of+mouse+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001203-SaitoHisaminato1" id="pone.0001203-SaitoHisaminato1"></a>Saito-Hisaminato A, Katagiri T, Kakiuchi S, Nakamura T, Tsunoda T, et al.  (2002) Genome-wide profiling of gene expression in 29 normal human tissues with a cDNA microarray. DNA Res  9: 35–45.  <ul class="find" data-citedArticleID="1020060" data-doi="10.1093/dnares/9.2.35"><li><a href="http://dx.doi.org/10.1093/dnares/9.2.35" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+profiling+of+gene+expression+in+29+normal+human+tissues+with+a+cDNA+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+profiling+of+gene+expression+in+29+normal+human+tissues+with+a+cDNA+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001203-Shyamsundar1" id="pone.0001203-Shyamsundar1"></a>Shyamsundar R, Kim YH, Higgins JP, Montgomery K, Jorden M, et al.  (2005) A DNA microarray survey of gene expression in normal human tissues. Genome Biol  6: R22.  <ul class="find" data-citedArticleID="1020066" data-doi="10.1186/gb-2005-6-3-r22"><li><a href="http://dx.doi.org/10.1186/gb-2005-6-3-r22" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+DNA+microarray+survey+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+DNA+microarray+survey+of+gene+expression+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001203-Su2" id="pone.0001203-Su2"></a>Su AI, Cooke MP, Ching KA, Hakak Y, Walker JR, et al.  (2002) Large-scale analysis of the human and mouse transcriptomes. Proc Natl Acad Sci U S A  99: 4465–4470.  <ul class="find" data-citedArticleID="1020072" data-doi="10.1073/pnas.012025199"><li><a href="http://dx.doi.org/10.1073/pnas.012025199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+analysis+of+the+human+and+mouse+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+analysis+of+the+human+and+mouse+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001203-Hsiao2" id="pone.0001203-Hsiao2"></a>Hsiao LL, Dangond F, Yoshida T, Hong R, Jensen RV, et al.  (2001) A compendium of gene expression in normal human tissues. Physiol Genomics  7: 97–104.  <ul class="find" data-citedArticleID="1020038"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+compendium+of+gene+expression+in+normal+human+tissues.&amp;auth=&amp;atitle=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+compendium+of+gene+expression+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001203-Ge2" id="pone.0001203-Ge2"></a>Ge X, Yamamoto S, Tsutsumi S, Midorikawa Y, Ihara S, et al.  (2005) Interpreting expression profiles of cancers by genome-wide survey of breadth of expression in normal tissues. Genomics  86: 127–141.  <ul class="find" data-citedArticleID="1020026" data-doi="10.1016/j.ygeno.2005.04.008"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.04.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpreting+expression+profiles+of+cancers+by+genome-wide+survey+of+breadth+of+expression+in+normal+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpreting+expression+profiles+of+cancers+by+genome-wide+survey+of+breadth+of+expression+in+normal+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001203-Hsiao3" id="pone.0001203-Hsiao3"></a>Hsiao LL, Dangond F, Yoshida T, Hong R, Jensen RV, et al.  (2001) A compendium of gene expression in normal human tissues. Physiol Genomics  7: 97–104.  <ul class="find" data-citedArticleID="1020040"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+compendium+of+gene+expression+in+normal+human+tissues.&amp;auth=&amp;atitle=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+compendium+of+gene+expression+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+compendium+of+gene+expression+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001203-Shmueli2" id="pone.0001203-Shmueli2"></a>Shmueli O, Horn-Saban S, Chalifa-Caspi V, Shmoish M, Ophir R, et al.  (2003) GeneNote: whole genome expression profiles in normal human tissues. C R Biol  326: 1067–1072.  <ul class="find" data-citedArticleID="1020064" data-doi="10.1016/j.crvi.2003.09.012"><li><a href="http://dx.doi.org/10.1016/j.crvi.2003.09.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneNote%3A+whole+genome+expression+profiles+in+normal+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneNote%3A+whole+genome+expression+profiles+in+normal+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001203-Su3" id="pone.0001203-Su3"></a>Su AI, Wiltshire T, Batalov S, Lapp H, Ching KA, et al.  (2004) A gene atlas of the mouse and human protein-encoding transcriptomes. Proc Natl Acad Sci U S A  101: 6062–6067.  <ul class="find" data-citedArticleID="1020074" data-doi="10.1073/pnas.0400782101"><li><a href="http://dx.doi.org/10.1073/pnas.0400782101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001203-Barrett1" id="pone.0001203-Barrett1"></a>Barrett T, Troup DB, Wilhite SE, Ledoux P, Rudnev D, et al.  (2007) NCBI GEO: mining tens of millions of expression profiles–database and tools update. Nucleic Acids Res  35: D760–D765.  <ul class="find" data-citedArticleID="1020018" data-doi="10.1093/nar/gkl887"><li><a href="http://dx.doi.org/10.1093/nar/gkl887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+GEO%3A+mining+tens+of+millions+of+expression+profiles%E2%80%93database+and+tools+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+GEO%3A+mining+tens+of+millions+of+expression+profiles%E2%80%93database+and+tools+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001203-Ashburner1" id="pone.0001203-Ashburner1"></a>Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, et al.  (2000) Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="1020016"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001203-Vencio1" id="pone.0001203-Vencio1"></a>Vencio RZ, Koide T, Gomes SL, Pereira CA (2006) BayGO: Bayesian analysis of ontology term enrichment in microarray data. BMC Bioinformatics  7: 86.  <ul class="find" data-citedArticleID="1020076"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=BayGO%3A+Bayesian+analysis+of+ontology+term+enrichment+in+microarray+data.&amp;auth=&amp;atitle=BayGO%3A+Bayesian+analysis+of+ontology+term+enrichment+in+microarray+data." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=BayGO%3A+Bayesian+analysis+of+ontology+term+enrichment+in+microarray+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22BayGO%3A+Bayesian+analysis+of+ontology+term+enrichment+in+microarray+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001203-Kanehisa1" id="pone.0001203-Kanehisa1"></a>Kanehisa M, Goto S, Kawashima S, Okuno Y, Hattori M (2004) The KEGG resource for deciphering the genome. Nucleic Acids Res  32: D277–D280.  <ul class="find" data-citedArticleID="1020046" data-doi="10.1093/nar/gkh063"><li><a href="http://dx.doi.org/10.1093/nar/gkh063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+KEGG+resource+for+deciphering+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+KEGG+resource+for+deciphering+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001203-Goeman1" id="pone.0001203-Goeman1"></a>Goeman JJ, van de Geer SA, de Kort F, van Houwelingen HC (2004) A global test for groups of genes: testing association with a clinical outcome. Bioinformatics  20: 93–99.  <ul class="find" data-citedArticleID="1020030" data-doi="10.1093/bioinformatics/btg382"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+test+for+groups+of+genes%3A+testing+association+with+a+clinical+outcome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+test+for+groups+of+genes%3A+testing+association+with+a+clinical+outcome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001203-Hubbard1" id="pone.0001203-Hubbard1"></a>Hubbard TJ, Aken BL, Beal K, Ballester B, Caccamo M, et al.  (2007) Ensembl 2007. Nucleic Acids Res  35: D610–D617.  <ul class="find" data-citedArticleID="1020042" data-doi="10.1093/nar/gkl996"><li><a href="http://dx.doi.org/10.1093/nar/gkl996" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2007." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2007.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001203-Wheeler1" id="pone.0001203-Wheeler1"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2007) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  35: D5–12.  <ul class="find" data-citedArticleID="1020078" data-doi="10.1093/nar/gkl1031"><li><a href="http://dx.doi.org/10.1093/nar/gkl1031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001203-Letunic1" id="pone.0001203-Letunic1"></a>Letunic I, Copley RR, Pils B, Pinkert S, Schultz J, et al.  (2006) SMART 5: domains in the context of genomes and networks. Nucleic Acids Res  34: D257–D260.  <ul class="find" data-citedArticleID="1020050" data-doi="10.1093/nar/gkj079"><li><a href="http://dx.doi.org/10.1093/nar/gkj079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMART+5%3A+domains+in+the+context+of+genomes+and+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMART+5%3A+domains+in+the+context+of+genomes+and+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001203-Liao1" id="pone.0001203-Liao1"></a>Liao BY, Zhang J (2006) Evolutionary conservation of expression profiles between human and mouse orthologous genes. Mol Biol Evol  23: 530–540.  <ul class="find" data-citedArticleID="1020054" data-doi="10.1093/molbev/msj054"><li><a href="http://dx.doi.org/10.1093/molbev/msj054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+conservation+of+expression+profiles+between+human+and+mouse+orthologous+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+conservation+of+expression+profiles+between+human+and+mouse+orthologous+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001203-Chen1" id="pone.0001203-Chen1"></a>Chen H, Rossier C, Lalioti MD, Lynn A, Chakravarti A, et al.  (1996) Cloning of the cDNA for a human homologue of the Drosophila white gene and mapping to chromosome 21q22.3. Am J Hum Genet  59: 66–75.  <ul class="find" data-citedArticleID="1020022"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cloning+of+the+cDNA+for+a+human+homologue+of+the+Drosophila+white+gene+and+mapping+to+chromosome+21q22.3.&amp;auth=&amp;atitle=Cloning+of+the+cDNA+for+a+human+homologue+of+the+Drosophila+white+gene+and+mapping+to+chromosome+21q22.3." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+the+cDNA+for+a+human+homologue+of+the+Drosophila+white+gene+and+mapping+to+chromosome+21q22.3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+the+cDNA+for+a+human+homologue+of+the+Drosophila+white+gene+and+mapping+to+chromosome+21q22.3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001203-Lecker1" id="pone.0001203-Lecker1"></a>Lecker SH, Solomon V, Mitch WE, Goldberg AL (1999) Muscle protein breakdown and the critical role of the ubiquitin-proteasome pathway in normal and disease states. J Nutr  129: 227S–237S.  <ul class="find" data-citedArticleID="1020048"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Muscle+protein+breakdown+and+the+critical+role+of+the+ubiquitin-proteasome+pathway+in+normal+and+disease+states.&amp;auth=&amp;atitle=Muscle+protein+breakdown+and+the+critical+role+of+the+ubiquitin-proteasome+pathway+in+normal+and+disease+states." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Muscle+protein+breakdown+and+the+critical+role+of+the+ubiquitin-proteasome+pathway+in+normal+and+disease+states." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Muscle+protein+breakdown+and+the+critical+role+of+the+ubiquitin-proteasome+pathway+in+normal+and+disease+states.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001203-Ge3" id="pone.0001203-Ge3"></a>Ge X, Yamamoto S, Tsutsumi S, Midorikawa Y, Ihara S, et al.  (2005) Interpreting expression profiles of cancers by genome-wide survey of breadth of expression in normal tissues. Genomics  86: 127–141.  <ul class="find" data-citedArticleID="1020028" data-doi="10.1016/j.ygeno.2005.04.008"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2005.04.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpreting+expression+profiles+of+cancers+by+genome-wide+survey+of+breadth+of+expression+in+normal+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpreting+expression+profiles+of+cancers+by+genome-wide+survey+of+breadth+of+expression+in+normal+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001203-Agrawal1" id="pone.0001203-Agrawal1"></a>Agrawal D, Chen T, Irby R, Quackenbush J, Chambers AF, et al.  (2002) Osteopontin identified as lead marker of colon cancer progression, using pooled sample expression profiling. J Natl Cancer Inst  94: 513–521.  <ul class="find" data-citedArticleID="1020014" data-doi="10.1093/jnci/94.7.513"><li><a href="http://dx.doi.org/10.1093/jnci/94.7.513" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Osteopontin+identified+as+lead+marker+of+colon+cancer+progression%2C+using+pooled+sample+expression+profiling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Osteopontin+identified+as+lead+marker+of+colon+cancer+progression%2C+using+pooled+sample+expression+profiling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001203-Gorodkin1" id="pone.0001203-Gorodkin1"></a>Gorodkin J, Cirera S, Hedegaard J, Gilchrist MJ, Panitz F, et al.  (2007) Porcine transcriptome analysis based on 97 non-normalized cDNA libraries and assembly of 1,021,891 expressed sequence tags. Genome Biol  8: R45.  <ul class="find" data-citedArticleID="1020034" data-doi="10.1186/gb-2007-8-4-r45"><li><a href="http://dx.doi.org/10.1186/gb-2007-8-4-r45" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Porcine+transcriptome+analysis+based+on+97+non-normalized+cDNA+libraries+and+assembly+of+1%2C021%2C891+expressed+sequence+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Porcine+transcriptome+analysis+based+on+97+non-normalized+cDNA+libraries+and+assembly+of+1%2C021%2C891+expressed+sequence+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001203-Wheeler2" id="pone.0001203-Wheeler2"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2007) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  35: D5–12.  <ul class="find" data-citedArticleID="1020080" data-doi="10.1093/nar/gkl1031"><li><a href="http://dx.doi.org/10.1093/nar/gkl1031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001203-Zhang2" id="pone.0001203-Zhang2"></a>Zhang J, Carey V, Gentleman R (2003) An extensible application for assembling annotation for genomic data. Bioinformatics  19: 155–156.  <ul class="find" data-citedArticleID="1020088" data-doi="10.1093/bioinformatics/19.1.155"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.1.155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+extensible+application+for+assembling+annotation+for+genomic+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+extensible+application+for+assembling+annotation+for+genomic+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001203-Reimers1" id="pone.0001203-Reimers1"></a>Reimers M, Carey VJ (2006) Bioconductor: an open source framework for bioinformatics and computational biology. Methods Enzymol  411: 119–134.  <ul class="find" data-citedArticleID="1020058" data-doi="10.1016/s0076-6879(06)11008-3"><li><a href="http://dx.doi.org/10.1016/s0076-6879(06)11008-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bioconductor%3A+an+open+source+framework+for+bioinformatics+and+computational+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bioconductor%3A+an+open+source+framework+for+bioinformatics+and+computational+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001203-Yang1" id="pone.0001203-Yang1"></a>Yang YH, Dudoit S, Luu P, Lin DM, Peng V, et al.  (2002) Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation. Nucleic Acids Res  30: e15.  <ul class="find" data-citedArticleID="1020084" data-doi="10.1093/nar/30.4.e15"><li><a href="http://dx.doi.org/10.1093/nar/30.4.e15" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Normalization+for+cDNA+microarray+data%3A+a+robust+composite+method+addressing+single+and+multiple+slide+systematic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Normalization+for+cDNA+microarray+data%3A+a+robust+composite+method+addressing+single+and+multiple+slide+systematic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001203-Smyth1" id="pone.0001203-Smyth1"></a>Smyth GK (2004) Linear models and empirical bayes methods for assessing differential expression in microarray experiments. Stat Appl Genet Mol Biol  3: Article3.  <ul class="find" data-citedArticleID="1020068" data-doi="10.2202/1544-6115.1027"><li><a href="http://dx.doi.org/10.2202/1544-6115.1027" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linear+models+and+empirical+bayes+methods+for+assessing+differential+expression+in+microarray+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001203-Barrett2" id="pone.0001203-Barrett2"></a>Barrett T, Troup DB, Wilhite SE, Ledoux P, Rudnev D, et al.  (2007) NCBI GEO: mining tens of millions of expression profiles–database and tools update. Nucleic Acids Res  35: D760–D765.  <ul class="find" data-citedArticleID="1020020" data-doi="10.1093/nar/gkl887"><li><a href="http://dx.doi.org/10.1093/nar/gkl887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+GEO%3A+mining+tens+of+millions+of+expression+profiles%E2%80%93database+and+tools+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+GEO%3A+mining+tens+of+millions+of+expression+profiles%E2%80%93database+and+tools+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001203-Goeman2" id="pone.0001203-Goeman2"></a>Goeman JJ, van de Geer SA, de Kort F, van Houwelingen HC (2004) A global test for groups of genes: testing association with a clinical outcome. Bioinformatics  20: 93–99.  <ul class="find" data-citedArticleID="1020032" data-doi="10.1093/bioinformatics/btg382"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg382" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+test+for+groups+of+genes%3A+testing+association+with+a+clinical+outcome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+test+for+groups+of+genes%3A+testing+association+with+a+clinical+outcome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001203-Hubbard2" id="pone.0001203-Hubbard2"></a>Hubbard TJ, Aken BL, Beal K, Ballester B, Caccamo M, et al.  (2007) Ensembl 2007. Nucleic Acids Res  35: D610–D617.  <ul class="find" data-citedArticleID="1020044" data-doi="10.1093/nar/gkl996"><li><a href="http://dx.doi.org/10.1093/nar/gkl996" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ensembl+2007." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ensembl+2007.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001203-Wheeler3" id="pone.0001203-Wheeler3"></a>Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, et al.  (2007) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res  35: D5–12.  <ul class="find" data-citedArticleID="1020082" data-doi="10.1093/nar/gkl1031"><li><a href="http://dx.doi.org/10.1093/nar/gkl1031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Database+resources+of+the+National+Center+for+Biotechnology+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Database+resources+of+the+National+Center+for+Biotechnology+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001203-Letunic2" id="pone.0001203-Letunic2"></a>Letunic I, Copley RR, Pils B, Pinkert S, Schultz J, et al.  (2006) SMART 5: domains in the context of genomes and networks. Nucleic Acids Res  34: D257–D260.  <ul class="find" data-citedArticleID="1020052" data-doi="10.1093/nar/gkj079"><li><a href="http://dx.doi.org/10.1093/nar/gkj079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SMART+5%3A+domains+in+the+context+of+genomes+and+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SMART+5%3A+domains+in+the+context+of+genomes+and+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001203-Liao2" id="pone.0001203-Liao2"></a>Liao BY, Zhang J (2006) Evolutionary conservation of expression profiles between human and mouse orthologous genes. Mol Biol Evol  23: 530–540.  <ul class="find" data-citedArticleID="1020056" data-doi="10.1093/molbev/msj054"><li><a href="http://dx.doi.org/10.1093/molbev/msj054" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+conservation+of+expression+profiles+between+human+and+mouse+orthologous+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+conservation+of+expression+profiles+between+human+and+mouse+orthologous+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.XML" value="78750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.PDF" value="1044141"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g001.PNG_L" value="641323"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g001.PNG_M" value="238078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g001.PNG_S" value="13728"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g001.TIF" value="2248950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g001.PNG_I" value="102372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g002.PNG_L" value="941174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g002.PNG_M" value="282520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g002.PNG_S" value="14635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g002.TIF" value="2478934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g002.PNG_I" value="121597"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g003.PNG_L" value="513969"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g003.PNG_M" value="301504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g003.PNG_S" value="13770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g003.TIF" value="5848600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g003.PNG_I" value="130908"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g004.PNG_L" value="377506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g004.PNG_M" value="145700"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g004.PNG_S" value="13844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g004.TIF" value="3224162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g004.PNG_I" value="64907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g005.PNG_L" value="70498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g005.PNG_M" value="84935"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g005.PNG_S" value="14282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g005.TIF" value="748278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001203.g005.PNG_I" value="41999"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001203&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001203" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001203&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001203&volume=&issue=&title=Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues&author_name=Henrik%20Hornsh%C3%B8j%2C%20Lene%20Nagstrup%20Conley%2C%20Jakob%20Hedegaard%2C%20Peter%20S%C3%B8rensen%2C%20Frank%20Panitz%2C%20Christian%20Bendixen&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001203" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203&amp;t=Microarray%20Expression%20Profiles%20of%2020.000%20Genes%20across%2023%20Healthy%20Porcine%20Tissues" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203&title=Microarray%20Expression%20Profiles%20of%2020.000%20Genes%20across%2023%20Healthy%20Porcine%20Tissues&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203&amp;title=Microarray%20Expression%20Profiles%20of%2020.000%20Genes%20across%2023%20Healthy%20Porcine%20Tissues" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001203&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Microarray Expression Profiles of 20.000 Genes across 23 Healthy Porcine Tissues';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001203';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Microarray%20Expression%20Profiles%20of%2020.000%20Genes%20across%2023%20Healthy%20Porcine%20Tissues http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001203" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001203" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complementary+DNA%22" title="Search for articles in the subject area:'Complementary DNA'"><div class="flagText">Complementary DNA</div></a>
              <div data-categoryid="32053" data-articleid="25238"
                   data-categoryname="Complementary DNA"
                   class="flagImage" title="Flag 'Complementary DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25238"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25238"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25238"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Muscle+tissue%22" title="Search for articles in the subject area:'Muscle tissue'"><div class="flagText">Muscle tissue</div></a>
              <div data-categoryid="49177" data-articleid="25238"
                   data-categoryname="Muscle tissue"
                   class="flagImage" title="Flag 'Muscle tissue' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22RNA+extraction%22" title="Search for articles in the subject area:'RNA extraction'"><div class="flagText">RNA extraction</div></a>
              <div data-categoryid="20119" data-articleid="25238"
                   data-categoryname="RNA extraction"
                   class="flagImage" title="Flag 'RNA extraction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="25238"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Thyroid%22" title="Search for articles in the subject area:'Thyroid'"><div class="flagText">Thyroid</div></a>
              <div data-categoryid="47211" data-articleid="25238"
                   data-categoryname="Thyroid"
                   class="flagImage" title="Flag 'Thyroid' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4276'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4561'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=87&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
