

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001457"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001457" />

    <meta name="citation_title" content="Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening"/>
    <meta itemprop="name" content="Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening"/>

      <meta name="citation_author" content="Suha Naffar-Abu-Amara"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Tal Shay"/>
            <meta name="citation_author_institution" content="Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Meirav Galun"/>
            <meta name="citation_author_institution" content="Department of Computer Science and Applied Mathematics, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Naomi Cohen"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Steven J. Isakoff"/>
            <meta name="citation_author_institution" content="Massachusetts General Hospital Cancer Center, Boston, Massachusetts, United States of America"/>
            <meta name="citation_author_institution" content="Department of Cell Biology, Harvard Medical School, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Zvi Kam"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel"/>
      <meta name="citation_author" content="Benjamin Geiger"/>
            <meta name="citation_author_institution" content="Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel"/>

    <meta name="citation_date" content="2008/1/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001457.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1457"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Vasculogenesis and angiogenesis as mechanisms of vascular network formation, growth and remodeling.; citation_author=S Patan; citation_journal_title=J Neurooncol; citation_volume=50(1–2); citation_number=1; citation_pages=1-15; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Regulation of the cytoskeleton by Rho-family GTPases: implications for tumour cell invasion.; citation_author=LS Price; citation_author=JG Collard; citation_journal_title=Semin Cancer Biol; citation_volume=11(2); citation_number=2; citation_pages=167-73; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Polarity, protrusion-retraction dynamics and their interplay during keratinocyte cell migration.; citation_author=T Libotte; citation_author=HW Kaiser; citation_author=W Alt; citation_author=T Bretschneider; citation_journal_title=Exp Cell Res; citation_volume=270(2); citation_number=3; citation_pages=129-37; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Compensation mechanism in tumor cell migration: mesenchymal-amoeboid transition after blocking of pericellular proteolysis.; citation_author=K Wolf; citation_author=I Mazo; citation_author=H Leung; citation_author=K Engelke; citation_author=UH von Andrian; citation_journal_title=J Cell Biol; citation_volume=160(2); citation_number=4; citation_pages=267-77; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The reorganization of microfilaments, centrosomes, and microtubules during in vitro small wound reendothelialization.; citation_author=MK Wong; citation_author=AI Gotlieb; citation_journal_title=J Cell Biol; citation_volume=107(5); citation_number=5; citation_pages=1777-83; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=In vitro endothelial wound repair. Interaction of cell migration and proliferation.; citation_author=BL Coomber; citation_author=AI Gotlieb; citation_journal_title=Arteriosclerosis; citation_volume=10(2); citation_number=6; citation_pages=215-22; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Cell migration and proliferation during the in vitro wound repair of the respiratory epithelium.; citation_author=JM Zahm; citation_author=H Kaplan; citation_author=AL Herard; citation_author=F Doriot; citation_author=D Pierrot; citation_journal_title=Cell Motil Cytoskeleton; citation_volume=37(1); citation_number=7; citation_pages=33-43; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The chemotactic effect of mixtures of antibody and antigen on polymorphonuclear leucocytes.; citation_author=S Boyden; citation_journal_title=J Exp Med; citation_volume=115; citation_number=8; citation_pages=453-66; citation_date=1962; " />
      <meta name="citation_reference" content="citation_title=A quantitative high-throughput endothelial cell migration assay.; citation_author=V Mastyugin; citation_author=E McWhinnie; citation_author=M Labow; citation_author=F Buxton; citation_journal_title=J Biomol Screen; citation_volume=9(8); citation_number=9; citation_pages=712-8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The phagokinetic tracks of 3T3 cells.; citation_author=G Albrecht-Buehler; citation_journal_title=Cell; citation_volume=11(2); citation_number=10; citation_pages=395-404; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=A Rac switch regulates random versus directionally persistent cell migration.; citation_author=R Pankov; citation_author=Y Endo; citation_author=S Even-Ram; citation_author=M Araki; citation_author=K Clark; citation_journal_title=J Cell Biol; citation_volume=170(5); citation_number=11; citation_pages=793-802; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Type IV collagen and laminin enhance the motility, adhesion, and proliferation of hepatoma cells.; citation_author=R Ogata; citation_journal_title=Kurume Med J; citation_volume=45(1); citation_number=12; citation_pages=11-20; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Cooperation of fibronectin with lysophosphatidic acid induces motility and transcellular migration of rat ascites hepatoma cells.; citation_author=M Ayaki; citation_author=M Mukai; citation_author=F Imamura; citation_author=T Iwasaki; citation_author=T Mammoto; citation_journal_title=Biochim Biophys Acta; citation_volume=1495(1); citation_number=13; citation_pages=40-50; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Inhibition of chemotactic motility and trans-endothelial migration of human neutrophils by sphingosine 1-phosphate.; citation_author=S Kawa; citation_author=S Kimura; citation_author=S Hakomori; citation_author=Y Igarashi; citation_journal_title=FEBS Lett; citation_volume=420(2-3); citation_number=14; citation_pages=196-200; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The tetraspanin CD9 associates with the integrin alpha6beta4 in cultured human epidermal keratinocytes and is involved in cell motility.; citation_author=B Baudoux; citation_author=D Castanares-Zapatero; citation_author=M Leclercq-Smekens; citation_author=N Berna; citation_author=Y Poumay; citation_journal_title=Eur J Cell Biol; citation_volume=79(1); citation_number=15; citation_pages=41-51; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=G(i)-mediated Cas tyrosine phosphorylation in vascular endothelial cells stimulated with sphingosine 1-phosphate: possible involvement in cell motility enhancement in cooperation with Rho-mediated pathways.; citation_author=T Ohmori; citation_author=Y Yatomi; citation_author=H Okamoto; citation_author=Y Miura; citation_author=G Rile; citation_journal_title=J Biol Chem; citation_volume=276(7); citation_number=16; citation_pages=5274-80; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Autocrine motility factor receptor gene expression and cell motility in lung cancer cell lines.; citation_author=I Takanami; citation_author=K Takeuchi; citation_author=H Watanabe; citation_author=T Yanagawa; citation_author=K Takagishi; citation_journal_title=Oncol Rep; citation_volume=9(1); citation_number=17; citation_pages=125-8; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Cyclic phosphatidic acid inhibits RhoA-mediated autophosphorylation of FAK at Tyr-397 and subsequent tumor-cell invasion.; citation_author=M Mukai; citation_author=T Iwasaki; citation_author=M Tatsuta; citation_author=A Togawa; citation_author=H Nakamura; citation_journal_title=Int J Oncol; citation_volume=22(6); citation_number=18; citation_pages=1247-56; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Overexpression of autocrine motility factor receptor (AMFR) in NIH3T3 fibroblasts induces cell transformation.; citation_author=Y Onishi; citation_author=K Tsukada; citation_author=J Yokota; citation_author=A Raz; citation_journal_title=Clin Exp Metastasis; citation_volume=20(1); citation_number=19; citation_pages=51-8; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cell-cycle-dependent invasion in vitro by rat ascites hepatoma cells.; citation_author=T Iwasaki; citation_author=K Shinkai; citation_author=M Mukai; citation_author=K Yoshioka; citation_author=Y Fujii; citation_journal_title=Int J Cancer; citation_volume=63(2); citation_number=20; citation_pages=282-7; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=An essential part for Rho-associated kinase in the transcellular invasion of tumor cells.; citation_author=K Itoh; citation_author=K Yoshioka; citation_author=H Akedo; citation_author=M Uehata; citation_author=T Ishizaki; citation_journal_title=Nat Med; citation_volume=5(2); citation_number=21; citation_pages=221-5; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genomic analysis of metastasis reveals an essential role for RhoC.; citation_author=EA Clark; citation_author=TR Golub; citation_author=ES Lander; citation_author=RO Hynes; citation_journal_title=Nature; citation_volume=406(6795); citation_number=22; citation_pages=532-5; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The microenvironment of the tumour-host interface.; citation_author=LA Liotta; citation_author=EC Kohn; citation_journal_title=Nature; citation_volume=411(6835); citation_number=23; citation_pages=375-9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=High-throughput screening of cellular features using high-resolution light-microscopy; application for profiling drug effects on cell adhesion.; citation_author=Y Paran; citation_author=M Ilan; citation_author=Y Kashman; citation_author=S Goldstein; citation_author=Y Liron; citation_journal_title=J Struct Biol; citation_volume=158(2); citation_number=24; citation_pages=233-43; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Laser autofocusing system for high-resolution cell biological imaging.; citation_author=Y Liron; citation_author=Y Paran; citation_author=NG Zatorsky; citation_author=B Geiger; citation_author=Z Kam; citation_journal_title=J Microsc; citation_volume=221(Pt 2); citation_number=25; citation_pages=145-51; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Segmentation and boundary detection using multiscale intensity measurements.; citation_author=E Sharon; citation_author=A Brandet; citation_author=R Basri; citation_journal_title=IEEE I469-I476; citation_number=26; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Functional proteomics approach to investigate the biological activities of cDNAs implicated in breast cancer.; citation_author=AE Witt; citation_author=LM Hines; citation_author=NL Collins; citation_author=Y Hu; citation_author=RN Gunawardane; citation_journal_title=J Proteome Res; citation_volume=5(3); citation_number=27; citation_pages=599-610; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Breast cancer-associated PIK3CA mutations are oncogenic in mammary epithelial cells.; citation_author=SJ Isakoff; citation_author=JA Engelman; citation_author=HY Irie; citation_author=J Luo; citation_author=SM Brachmann; citation_journal_title=Cancer Res; citation_volume=65(23); citation_number=28; citation_pages=10992-1000; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening"/>
    <meta name="twitter:description" content="BackgroundCell migration is a highly complex process, regulated by multiple genes, signaling pathways and external stimuli. To discover genes or pharmacological agents that can modulate the migratory activity of cells, screening strategies that enable the monitoring of diverse migratory parameters in a large number of samples are necessary.MethodologyIn the present study, we describe the development of a quantitative, high-throughput cell migration assay, based on a modified phagokinetic tracks (PKT) procedure, and apply it for identifying novel pro-migratory genes in a cancer-related gene library. In brief, cells are seeded on fibronectin-coated 96-well plates, covered with a monolayer of carboxylated latex beads. Motile cells clear the beads, located along their migratory paths, forming tracks that are visualized using an automated, transmitted-light screening microscope. The tracks are then segmented and characterized by multi-parametric, morphometric analysis, resolving a variety of morphological and kinetic features.ConclusionsIn this screen we identified 4 novel genes derived from breast carcinoma related cDNA library, whose over-expression induces major alteration in the migration of the stationary MCF7 cells. This approach can serve for high throughput screening for novel ways to modulate cellular migration in pathological states such as tumor metastasis and invasion."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001457.g005"/>

  <meta property="og:title" content="Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9888'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3625'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5139&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001457">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001457" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001457&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Suha Naffar-Abu-Amara, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tal Shay, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Meirav Galun, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Computer Science and Applied Mathematics, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Naomi Cohen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Steven J. Isakoff, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Massachusetts General Hospital Cancer Center, Boston, Massachusetts, United States of America, 
                  Department of Cell Biology, Harvard Medical School, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zvi Kam, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Benjamin Geiger
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:benny.geiger@weizmann.ac.il">benny.geiger@weizmann.ac.il</a></p>

                <p>Affiliation:
                  Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001457</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001457-g001" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001457-g002" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001457-g003" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001457-g004" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001457-g005" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001457">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001457" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Cell migration is a highly complex process, regulated by multiple genes, signaling pathways and external stimuli. To discover genes or pharmacological agents that can modulate the migratory activity of cells, screening strategies that enable the monitoring of diverse migratory parameters in a large number of samples are necessary.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>In the present study, we describe the development of a quantitative, high-throughput cell migration assay, based on a modified phagokinetic tracks (PKT) procedure, and apply it for identifying novel pro-migratory genes in a cancer-related gene library. In brief, cells are seeded on fibronectin-coated 96-well plates, covered with a monolayer of carboxylated latex beads. Motile cells clear the beads, located along their migratory paths, forming tracks that are visualized using an automated, transmitted-light screening microscope. The tracks are then segmented and characterized by multi-parametric, morphometric analysis, resolving a variety of morphological and kinetic features.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>In this screen we identified 4 novel genes derived from breast carcinoma related cDNA library, whose over-expression induces major alteration in the migration of the stationary MCF7 cells. This approach can serve for high throughput screening for novel ways to modulate cellular migration in pathological states such as tumor metastasis and invasion.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Naffar-Abu-Amara S, Shay T, Galun M, Cohen N, Isakoff SJ, et al.  (2008) Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening. PLoS ONE 3(1):
          e1457.
            doi:10.1371/journal.pone.0001457</p><p><strong>Academic Editor: </strong>Neil Hotchin, University of Birmingham, United Kingdom</p><p><strong>Received:</strong> October 21, 2007; <strong>Accepted:</strong> December 18, 2007; <strong>Published:</strong> January 23, 2008</p><p><strong>Copyright:</strong> © 2008 Naffar-Abu-Amara et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by grants from the Israel Science Foundation and an NIGMS grant for the Cell Migration Consortium (NIH Grant U54GM64346), and by the Kahn fund for systems biology at the Weizmann Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Cell migration plays a critical role in numerous physiological processes, including embryonic development, inflammatory responses, wound healing, and angiogenesis, as well as in pathological states such as tumor invasion and metastasis <a href="#pone.0001457-Patan1">[1]</a>, <a href="#pone.0001457-Price1">[2]</a>. To explore the mechanisms underlying the regulation of cell migration, a variety of qualitative and quantitative approaches have been developed. These include 2- and 3-dimensional time-lapse movies, tracking the migration of cultured or tissue-embedded cells <a href="#pone.0001457-Libotte1">[3]</a>, <a href="#pone.0001457-Wolf1">[4]</a>, wound-closure assays <a href="#pone.0001457-Wong1">[5]</a>–<a href="#pone.0001457-Zahm1">[7]</a>, matrix-permeation assays <a href="#pone.0001457-Boyden1">[8]</a>, <a href="#pone.0001457-Mastyugin1">[9]</a> and “recording” of the cells' migration “history,” based on assays such as PKT formation <a href="#pone.0001457-AlbrechtBuehler1">[10]</a>. The latter assay is widely used for studying the migratory activities of different cell types <a href="#pone.0001457-Libotte1">[3]</a>, <a href="#pone.0001457-Pankov1">[11]</a>, matrix remodeling <a href="#pone.0001457-Ogata1">[12]</a>, <a href="#pone.0001457-Ayaki1">[13]</a> and perturbation of cell migration by chemical or genetic modulators <a href="#pone.0001457-Kawa1">[14]</a>–<a href="#pone.0001457-Onishi1">[19]</a>. Such studies are of particular relevance to cancer cell motility, which is believed to reflect the invasive or metastatic potential of these cells in vivo <a href="#pone.0001457-Kawa1">[14]</a>, <a href="#pone.0001457-Iwasaki1">[20]</a>–<a href="#pone.0001457-Liotta1">[23]</a>. Thus, identification of chemicals that alter cell migration, or specific genes whose perturbation affects cell migration could potentially be used for the modulation of metastatic cell migration.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Our objective in the present study was to develop a PKT-based approach for tracking cell migration, which is reproducible, compatible with high-throughput microscopy, and provides quantitative information, morphological and dynamic, on the migratory process. We show here that while the PKT records the integrated history of migratory activity at a single time point, the quantitative imaging software, enables the calculation of both “static” parameters such as track length and area, and “dynamic” parameters such as migration rates, persistence, and lamellar activity.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The high-throughput migration assay described herein, and the imaging software developed for measuring different features of the migratory process, provide a rapid, reliable and quantitative approach for assessing cell migration in diverse cell types, cultured under varying conditions, and exposed to a variety of chemical or genetic perturbations.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Development of a bead-based high-throughput PKT assay</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Critical to the development of this PKT assay was the selection of suitable beads, with optimal dimensions and chemical properties (<a href="#pone.0001457.s001">Table S1</a>). The beads that were found most suitable for PKT assays applied to a wide variety of cell types were carboxylate-modified latex (CML) white polystyrene beads, with an average diameter of 340 nm, and a negative charge content of 184.7 µEq/g. These beads form a homogenous and visible monolayer; their attachment to the substrate is firm enough to prevent spontaneous detachment, but still susceptible to removal by migrating cells.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The surface chemistry of the beads was found to have a strong effect on the PKT assay: beads with an aldehyde-modified surface attached firmly to the substrate, and could not be removed by migrating cells. Beads with a sulfated surface tended to aggregate, yielding a non-uniform monolayer. Carboxylated beads, with or without additional sulfate groups, tended to form rather homogenous suspensions after centrifugation. The surface density of the carboxylate groups also affected track formation: a low charge density (23.9 µEq/g) caused the bead to interact strongly with the surface, such that many cell types failed to effectively remove the beads as they migrated. Beads with carboxylate groups of intermediate density (91.4 µEq/g) were found optimal for some adherent cells (e.g., H1299, REF52) but not for cells with weaker adhesions (e.g., MCF7; B16-F10). Beads containing carboxyl groups with a density of 160–185 µEq/g were found to be optimal for assays applied to a wide range of cell types.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Moreover, the diameter of the beads had a major effect on the visibility of the tracks and on the stability of the monolayer. Thus, small beads (&lt;300 nm in diameter) could hardly be visualized, while large beads (~1,000 nm in diameter) tended to detach from the surface and then spontaneously reattach, resulting in poorly defined tracks. The optimal bead diameter for automated PKT assays was found to be about 400 nm.</p>


<h4>Development of the automated microscopy system</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>PKT assays were recorded using a cell-screening microscope <a href="#pone.0001457-Paran1">[24]</a> equipped with a laser autofocus device <a href="#pone.0001457-Liron1">[25]</a>. The microscope operating program and the image acquisition software were written as an application within the UCSF PRIISM environment (<a href="http://msg.ucsf.edu/ive">http://msg.ucsf.edu/ive</a>). For this application, images were taken using a 10×/0.4 objective, under transmitted light illumination. A light diffuser was inserted above the multi-well plate, to minimize non-homogenous illumination and avoid the shadows commonly caused by the narrow well walls.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The dedicated acquisition program controls the illumination; autofocusing and image acquisition steps for all selected fields within each well, and for all selected wells in the plate, while optimizing the experimental details (e.g. well number, field position, exposure time, objective, optical setting, and the like). In order to record a maximal number of complete cell tracks, images of adjacent fields were fused into a seamless montage, in which tracks spanning more than one image are merged at high precision (<a href="#pone.0001457.s007">Figure S1</a>).</p>


<h4>Quantification of migratory parameters, based on PKT morphometry</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To identify individual tracks, images were subjected to “flattening”, thereby compensating for non-homogeneous illumination, smoothing and contrast enhancement. The resulting intensity histogram yielded a major peak, corresponding to the unperturbed background (<a href="#pone-0001457-g001">Figure 1c</a>, black asterisk); a high intensity peak corresponding to the bead-free tracks (<a href="#pone-0001457-g001">Figure 1c</a>, blue asterisk); and low-intensity pixels corresponding to bead-loaded cells (<a href="#pone-0001457-g001">Figure 1c</a>, red asterisk). By applying two binary thresholds, cells (<a href="#pone-0001457-g001">Figure 1d</a>), and tracks (<a href="#pone-0001457-g001">Figure 1e</a>) could be differentiated from each other. Debris, scratches, tracks with no, or more than one cell, intersecting tracks, and tracks extending beyond the border of the montage, were identified and discarded (<a href="#pone-0001457-g001">Figure 1f</a>, segments outlined in blue). To obtain fine definition of track borders, which were slightly blurred by the smoothing step (<a href="#pone-0001457-g001">Figure 1g</a>), track boundaries were recalculated from the original images, using multiscale segmentation analysis <a href="#pone.0001457-Sharon1">[26]</a> (<a href="#pone-0001457-g001">Figure 1h</a>).</p>
<div class="figure" id="pone-0001457-g001"><div class="img"><a name="pone-0001457-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Computerized identification of individual PKT formed by H1299 cells.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(a) A montage of 4×4 fields (1024×1024 pixels), each acquired using a 10×/0.4 objective. (b) A single field (512×512 pixels, binned 2×2) (c) Histogram of pixel intensity: the major peak (*) corresponds to background pixels; the minor peak of bright intensity (*) corresponds to track pixels; and the dark pixels represent cell bodies and debris (*). The red vertical lines mark the two thresholds separating the track pixels from the cells and the background. (d,e) Binary images, after applying thresholds. Pixels with intensities below the lower threshold (cells and debris) are colored white in (d), and pixels with intensities above the higher threshold (tracks) are colored white in (e). (f) All connected components of the entire montage are outlined: Tracks are outlined in red. Objects either too small or too large in area to be included in the image analyses, or located on the borders of the montage, are outlined in blue. (g) Enlargement of a segmented field following binary segmentation. (h) The same area depicted in (g), following multi-scale segmentation, and including the fine outline of the track and the track axes. Scale bars: 250 µm.</p>
<span>doi:10.1371/journal.pone.0001457.g001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Following the segmentation step, we quantified the various morphometric parameters for each cell type. The track parameters that were automatically measured included track area, perimeter, major axis, minor axis, axial ratio and solidity. Track path length and end-to-end length were manually measured. The track parameters calculated from these morphometric parameters include persistence, effective velocity, average migration velocity, lamellar activity, and overall directionality. These morphological and “dynamic” parameters are defined in <a href="#pone.0001457.s002">Table S2</a>, and graphically presented in <a href="#pone-0001457-g002">Figure 2</a><strong>.</strong></p>
<div class="figure" id="pone-0001457-g002"><div class="img"><a name="pone-0001457-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Morphometric parameters of PKT.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>The automatic calculation of the various morphometric parameters used in this study, are demonstrated using PKTs formed by H1299, B16-F10 and MCF7 cells. The calculated parameters include: Total track area (µm<sup>2</sup>), delineated in red; net track area after cell area is subtracted; minor and major axes (µm) of the best-fit ellipse; ratio of the axes; migration velocity [length of skeleton and branches (green+blue)/migration time], effective velocity [end-to-end distance (colored red)/migration time]; track perimeter; roughness [Perimeter<sup>2</sup>/(4π*Area)] and Solidity [track area (blue)/area of the convex hull (red+blue) enclosing the track]. The values indicated in the figure refer to the single track shown.</p>
<span>doi:10.1371/journal.pone.0001457.g002</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Notably, even apparently similar parameters (e.g., “migration velocity” and “effective velocity”) can vary greatly. For example, B16-F10 cell, shown in <a href="#pone-0001457-g002">Figure 2</a>, exhibited almost the same migration velocity (62.4 µm/hr) as H1299 cell (67.26 µm/hr), while the latter cell type displayed a much higher effective velocity (57.26 µm/hr, compared to 20.4 µm/hr in B16-F10 cell), indicating a more persistent migration than the former. To assess “lateral” lamellar activity, which affects the width and roughness of track borders, we measured the track perimeter, and calculated roughness and solidity parameters. This analysis showed that while the perimeters of tracks formed by H1299 and B16-F10 cells were nearly the same, the roughness parameter was considerably higher in B16-F10 cell (5.2) than in H1299 cell (3.2), indicating higher lamellar activity in the melanoma cells. This finding was directly confirmed by time-lapse movies (<a href="#pone-0001457-g003">Figure 3</a>, and <a href="#pone.0001457.s010">Movies S1</a> and <a href="#pone.0001457.s011">S2</a>).</p>
<div class="figure" id="pone-0001457-g003"><div class="img"><a name="pone-0001457-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Track border roughness indicates lateral lamellar activity.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>Different time points are shown at which lateral lamellar activity of an H1299 (upper panel) or B16-F10 (lower panel) cells leave behind marks on the shape and roughness of the track border. (Full-length movies are available online as <a href="#pone.0001457.s010">Movie S1</a> and <a href="#pone.0001457.s011">Movie S2</a>, respectively.)</p>
<span>doi:10.1371/journal.pone.0001457.g003</span></div>

<h4>Application of PKT morphometry for measuring cell-type specific and drug-induced effects on migratory parameters</h4>
<h5>a) Different cell types produce PKT with distinct characteristics.</h5><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1"></a><p>The PKT assay described herein was used to test the migratory behavior of a variety of cell lines. These include: B16-F10 and B16-F1 melanoma cells; MDA-MB-231 and MCF7 breast carcinoma cells, REF52, SV80 and NIH3T3 fibroblast lines; H1299 lung carcinoma cells, and several prostate carcinoma lines (DU145, PC3 and CL1). Four of these cell lines (MDA-MB-231, MCF7, H1299 and B16-F10) are utilized for the purposes of illustration (see <a href="#pone-0001457-g004">Figure 4a</a> and <a href="#pone.0001457.s003">Table S3</a>). MCF7 cells, for example, hardly migrate, producing only a small, bead-free zone around each cell, with an average net track area of 4,900±2,400 µm<sup>2</sup> (n = 93). The B16-F10 melanoma cells produce branched tracks due to the extension of multiple filopodia and thin lamella largely perpendicular to the main migratory track, with an average net area of 7,400±2,800 µm<sup>2</sup> (n = 124). The MDA-MB-231 cells are highly migratory metastatic cells, producing both long and wide tracks with an average net area of 13,500±6,300 µm<sup>2</sup> (n = 149). H1299 cells are characterized by rapid and highly persistent migration, forming tracks with an average net area of 14,000±7,600 µm<sup>2</sup> (n = 104).</p>
<div class="figure" id="pone-0001457-g004"><div class="img"><a name="pone-0001457-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>PKT features of different cell lines and the effects of cytoskeletal drugs on tracks parameters.</span></strong></p><a id="article1.body1.sec2.sec4.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec4.sec1.fig1.caption1.p1"></a><p>(a) A montage of 2×3 images of the PKT of MCF7 cells, as well as for MDA-MB-231 cells, B16-F10 cells, and H1299 cells. Note the differences in track net area (A<sub>N</sub>) and axial ratio (X), depending on the cell line. (b) A montage of 2×3 image of H1299 control cells, which exhibit long migratory paths that are highly persistent. The montage images of the Latrunculin A (4 µM)- and Nocadazole (2.5 µM)-treated wells indicate inhibited cell motility; PMA (100 ng/ml)-treated well shows an increase in cell motility. Scale bars: 250 µm.</p>
<span>doi:10.1371/journal.pone.0001457.g004</span></div>
<h5>b) The effects of cytoskeletal drugs on PKT structural and dynamic features.</h5><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1"></a><p>In order to determine whether our automated screening system was capable of detecting changes in specific migratory features induced by genetic or chemical perturbations, we treated H1299 cells with various compounds (e.g., Latrunculin-A, Nocodazole and PMA) known to affect cell motility. The effect of each drug on the different morphometric parameters was then measured (<a href="#pone-0001457-g004">Figure 4b</a>). Since the values for each PKT parameter did not appear to have normal statistical distributions, we based our comparison on changes in percentile values for each morphometric parameter, rather than on changes in average values (<a href="#pone.0001457.s004">Table S4</a>). Analysis of the PKT area of H1299 cells treated with the various inhibitors indicated that Latrunculin-A or Nocodazole markedly reduced track areas, axial ratios, migration velocities and roughness, compared to control cells. In the PMA-treated cells, the PKT area, major axis and calculated migration velocities increased (p&lt;0.05), but the minor axis, axial ratios and solidity did not significantly differ from those of control cells.</p>



<h4>Application of the PKT assay to the identification of pro-migratory genes in a breast carcinoma-related gene library (BC1000)</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The PKT assay described herein was used to screen a library of cancer-related genes for their ability to induce a migratory phenotype in largely stationary breast epithelial cells. For that purpose, we infected cultured MCF7 cells with retroviral vectors encoding 55 genes selected from the BC1000 library (<a href="#pone.0001457.s005">Table S5</a>) and tested their migratory activity by means of quantitative PKT screening.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>As shown in <a href="#pone-0001457-g005">Figure 5a</a> and <a href="#pone.0001457.s006">Table S6</a>, the PKT screen enabled us to identify four novel pro-migratory candidates: HOXB7, FGF7, ERBB3 and PKCζ. The 80<sup>th</sup> percentile values, as well as the average and standard deviation, are presented in <a href="#pone.0001457.s006">Table S6</a>. While all four genes stimulated MCF7 cell migration, their effects on the various migratory parameters differed. Thus, while FGF7 and PKCζ clones induced the formation of long, persistent tracks, due to the enhancement of directional membrane protrusions, the elongated tracks, induced by HOXB7 and ERBB3 appeared to be associated with multiple membrane protrusions in all directions. Therefore, while prominence of directional “forward protrusions” cannot be directly assessed by track morphometry, it is apparent that enhancement of track length which is not accompanied by high roughness values is, most likely, driven by increased directional lamellipodial activity (<a href="#pone.0001457.s006">Table S6</a> and <a href="#pone-0001457-g005">Figure 5</a>). It is interesting to note that in the screen of the BC1000 library, changes in migratory behavior were noted for another gene, namely MFGE8 (also known as breast epithelial antigen BA46). While the overexpression of this gene induced only minor enhancement in PKT area, it did greatly enhance track roughness, suggesting that it enhances non-directional membrane protrusion (Naffar Abu-Amara, unpublished data).</p>
<div class="figure" id="pone-0001457-g005"><div class="img"><a name="pone-0001457-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001457" data-uri="info:doi/10.1371/journal.pone.0001457.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001457.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001457.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001457.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001457.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The pro-migratory effect of genes derived from the BC1000 library.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(a) A montage of 4×4 images, comparing PKT produced by GFP-MCF7 control cells, to those produced by MCF7 cells expressing the BC1000-derived genes HOXB7, PKCζ, FGF7, and ERBB3. Magnification: 10×. Scale bars: 250 µm. (b) Calculated ratio between each of the migratory morphometric parameters of the different BC1000 library candidates, and those of control cells (GFP-MCF7). The primary statistical approach utilized was based on calculating, for each parameter, the “80<sup>th</sup> percentile.” The normalized effect of GFP-control is always defined as zero: a zero value indicates no difference between the “80<sup>th</sup> percentile” value of the candidate gene, and of the control cells. Numbers that are higher or lower than zero indicate an increase or decrease, respectively, in the 80<sup>th</sup> percentile value of the tested parameters. (For additional details, see <a href="#pone.0001457.s006">Table S6</a>).</p>
<span>doi:10.1371/journal.pone.0001457.g005</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>To determine the interrelationships between the various migratory parameters, we applied the Pearson correlation test to all PKT produced by unperturbed cells (<a href="#pone.0001457.s008">Figure S2</a>). This analysis revealed, for example, that track length (major axis and axial ratio) is negatively correlated with track solidity, indicating that the production of elongated tracks by the control cells is highly correlated with the lateral protrusive activity. This analysis was also applied to those cells expressing the different pro-migratory genes, in order to determine their capacity to disrupt the apparent interdependence between the various migratory parameters. These analyses demonstrate that, overexpression of PKCζ, HOXB7, and ERBB3 diminished the reciprocal relationship between track length and solidity.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Concerning the relationships between the axial ratio and the dimensions of the long and short axes, the control cells show the predominant contribution of the long axis to the axial ratio value, with the minor axis exerting a limited effect. In cells expressing FGF7 and PKCζ the axial ratio was mainly correlated to the major axis, while ERBB3 and HOXB7 affected the axial ratio by changing the track width. It thus appears that enhancement of overall cell migration by different pro-migratory genes can be achieved by selective modulation of a variety of dynamic cellular features such as cell polarization and protrusive membrane activity.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The primary objective of this study was the development of a quantitative assay for cell migration, which could measure multiple migratory features, and would be compatible with high-throughput screening. The major experimental challenge involved in designing such an assay is the apparent conflict between the rapid acquisition of vast amount of migration data, and the need to obtain “high content” information about the migratory behavior of many cells, including dynamic features such as migration velocity and lamellipodial activity. Since collection of <span class="underline">direct</span> dynamic information about cell migration is incompatible with high throughput, we therefore chose to explore indirect, yet reproducible and robust, approaches for obtaining such information. We propose herein that detailed morphometric analysis of PKT generated by individual migrating cells can provide such quantitative, morphological and apparently dynamic data.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The novel aspects of this study which deserve specific discussion are: (i) the choice of beads; (ii) track segmentation and morphometry; and (iii) statistical analysis of the data. The selection of beads for the PKT assay was based on three properties of the “migration field”: visibility of the track (primarily affected by the bead size, with an optimal bead diameter of 300–400 nm); their ability to form a homogenous layer (optimal with aldehyde or carboxylated surfaces; poor with sulfate-modified beads), and the capacity to be removed by migrating cells (optimal with carboxylated- and poor with aldehyde-modified beads). Notably, the susceptibility of carboxylated beads to removal by migrating cells is inversely correlated to the surface density of the carboxylate groups, enabling “fine tuning” of the bead layer for the particular cell type to be tested.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>PKT segmentation and morphometry provided important insights into the basic features of the migratory process. These include the overall migratory activity of the cell, corresponding to the net PKT area; migration polarity, corresponding to the length of the major axis of migration (or the axial ratio); and “lateral” lamellar activity, measured by the track's solidity, among other values. Based on these measurements, several dynamic parameters were calculated, including average and effective migration rates, and lamellar activity. Naturally, the dynamic information, inferred from the static PKT morphology, is rather indirect, yet the high-resolution data [especially the roughness of track borders, measured by multiscale segmentation analysis <a href="#pone.0001457-Sharon1">[26]</a>; (<a href="#pone-0001457-g001">Figure 1h</a>)] which could be correlated to dynamic information, based on time-lapse video microscopy (<a href="#pone-0001457-g003">Figure 3</a>). Once calculated for each track, the multiple parameters could be correlated to each other, either in control cells, or following chemical or genetic perturbation.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>For our purposes, the PKT assay was utilized to discover novel pro-migratory genes in a breast carcinoma-related gene library. The rationale underlying this approach is that despite the obvious molecular complexity of the cell migration process, there might be “master genes” that could induce migratory features in a stationary cell. The unique capacity of the PKT assay we developed to highlight and quantify individual features of the migratory phenotype, could then link a given pro-migratory gene to specific migratory mechanisms. The new pro-migratory genes identified here include: HOXB7, ERBB3, PKCζ and FGF7. While all these genes are known to be “cancer-related”, the current study points to the possibility that their contribution to the malignant process might be related to their pro-migratory activity.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Preparation of bead-coated 96-well plates</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Glass-bottomed 96-well plates (Whatmann, Inc., Clifton, NJ, USA, Cat. # 7706-2370) were incubated for 2 hr at room temperature, with 50 µl of 10 µg/ml fibronectin solution dissolved in PBS (Fibronectin, F-1141; Sigma Chemical Co., St Louis, MO, USA). The wells were then washed twice with PBS and coated with 340 nm white polystyrene latex beads (Interfacial Dynamics Corporation-Molecular Probes Microspheres Technologies, USA; Product no. 2-300; Batch no. 1344). The bead suspension (3.2 ml) was centrifuged for 5 minutes at 20,800×g, and the pellet resuspended in 4 ml PBS, until all visible bead clumps were dispersed. The sedimentation procedure was then repeated once more, after which the beads were resuspended in 7 ml PBS, to a final concentration of 0.9<sup>12</sup> particles/ml. Aliquots of 70 µl of the bead suspension were added to each well, which had been pre-coated with fibronectin, and incubated at 37°C for 2 hr, followed by gentle washing with PBS (×5) using a plate washer (Colombus Plus, Tecan, Switzerland). Before cell plating, the PBS was replaced with 50 µl culture medium suitable for the particular cell type used in the assay (see <a href="#pone.0001457.s009">Figure S3</a>).</p>


<h4>Cell preparation for the PKT assay</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>MCF7 (ATCC-HTB-22), MDA-MB-231 (ATCC-HBT-26), and B16-F10 (ATCC-CRL-6475) cells were cultured in Dulbecco's Modified Eagle's Medium (DMEM); H1299 cells (ATCC-CRL-5803) were grown in RPMI-1640. Both culture media were supplemented with 10% FCS, 2 mM glutamine, 100 International Units/ml penicillin, and 100 µg/ml streptomycin (Biological Industries, Beit Haemek, Israel), and maintained in a 5% CO<sub>2</sub> humidified incubator at 37°C. For the PKT assay, 200–400 cells (in 50 µl of medium) were cultured in each well. Depending on the typical track dimensions, as determined in preliminary experiments, the number of plated cells, and the time of incubation were calibrated to maximize the number of single, non-intersecting cell tracks. Typically, 200–400 cells/well and 7 hours of incubation were found to be optimal parameters for most cells.</p>


<h4>Pharmacological perturbations of PKT formation</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>To determine the effects of various pharmacological inhibitors on the H1299 cell line, cells were plated and incubated for one hour, after which they were treated with either 4 µM Latrunculin A; 2.5 µM Nocodazole; or 100 ng/ml PMA (Phorbol 12-mirystate 13-acetate). The cells were then incubated for an additional 4 hr, fixed with 3% paraformaldehyde and washed twice with PBS. Plates were either examined immediately by means of a screening autofocus microscope, or stored at 4°C for later inspection.</p>


<h4>Screening for migration-inducing genes</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To screen for migration-inducing genes, we selected 55 candidate genes from the BC1000 library: (<a href="http://www.hip.harvard.edu/research/breast_cancer/index.htm">http://www.hip.harvard.edu/research/brea​st_cancer/index.htm</a>), assembled at the Harvard Institute of Proteomics using literature-mining software <a href="#pone.0001457-Witt1">[27]</a>. This library consists of a collection of full-length cDNAs known to be associated with breast cancer development. The cDNAs are cloned into a puromycin- selectable retroviral vector (JP1520) <a href="#pone.0001457-Isakoff1">[28]</a> using the Creator™ recombination system (Clontech, Mountain View, CA). The 55 genes tested (see <a href="#pone.0001457.s005">Table S5</a>) were randomly selected from this library, and were generously provided by Prof. Joan Brugge (Department of Cell Biology, Harvard Medical School, USA). Genes were introduced into the MCF7 cells by means of retroviral infection. Each clone was tested for its impact on cell migration, using the PKT assay. As controls, we also generated JP1520 GFP-expressing MCF7 cells. The PKT assay was carried out in 96-well plates. Four hundred cells per well were seeded, and 8 wells were tested for each clone. The cells were incubated for 7 hours, and then fixed using 3% PFA. Data was collected using the autofocus-screening microscope.</p>


<h4>Statistical Analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Since the distribution of track parameters displays non-normal distributions, we used the percentile statistical tool to estimates parameters variability. For example, the 80<sup>th</sup> percentile for track area is the value bellow which 80% of the tracks area are found.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Differences between control and treated cultures were evaluated for significance using the Two-Sample Kolmogorov-Smirnov goodness-of-fit hypothesis test. A p-value of &lt;0.05 was considered to be statistically significant.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>The Pearson's correlation test was conducted with values ranging from +1 (a perfect positive linear relationship between two tested variables) to −1 (a perfect negative linear relationship). A p-value of &lt;0.0014 was considered statistically significant.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001457"></div><a name="pone.0001457.s001" id="pone.0001457.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Comparison of the properties of different beads used for the PKT assay.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001457.s002" id="pone.0001457.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Parameters annotation.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001457.s003" id="pone.0001457.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">PKT analysis for the various cell lines.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001457.s004" id="pone.0001457.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">PKT analysis of H1299 cells treated by various drugs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001457.s005" id="pone.0001457.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">List of tested genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001457.s006" id="pone.0001457.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">PKT analysis of MCF7 cells overexpressing a given pro-migratory gene.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001457.s007" id="pone.0001457.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s007">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">A scheme describing the image acquisition and display process. (a) A template of a 96-well plate. (b) The positions of 52 fields that can be acquired within one well, using a 10× objective. (c) A montage of 4×4 images (1024×1024 pixels), corresponding to the marked area of the well. (d) A full-resolution image of one field (512×512 pixels) within the montage (marked in c). Scale bar: 250 µm.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(2.03 MB TIF)</p>
<a name="pone.0001457.s008" id="pone.0001457.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s008">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Auto-correlation between the PKT morphometric parameters in control cells, and in cells expressing pro-migratory genes. Auto-correlation between the various morphometric parameters was calculated for the control (GFP-MCF7) library, as well as for cells overexpressing the different pro-migratory genes described in <a href="#pone-0001457-g005">figure 5</a>. Each rectangle is divided by a white line into two triangles; each triangle shows the correlation test of a different candidate. The p-value of each correlation result is indicated beneath the correlation score number.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(4.16 MB TIF)</p>
<a name="pone.0001457.s009" id="pone.0001457.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s009">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">Schematic outline of the 96-well plate preparation for the PKT assay.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.96 MB TIF)</p>
<a name="pone.0001457.s010" id="pone.0001457.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s010">Movie S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">PKT formation by H1299, plated on polystyrene beads. H1299 cells were plated on a glass-bottomed 35 mm dish covered with 10 µg/ml fibronectin, and coated with polystyrene beads. Cells were maintained at 370C in CO2-buffered RPMI 1640 medium with 10% FCS and penicillin/streptomycin antibiotics. Before transferring the cells to the microscope, the CO2- buffered RPMI 1640 medium was replaced with HEPES buffered pre-warmed medium. Images were acquired using a 10× objective, every 5 minutes over 5 hours.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(2.93 MB MOV)</p>
<a name="pone.0001457.s011" id="pone.0001457.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001457.s011">Movie S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">PKT formation by B16-F10 melanoma cells, plated on polystyrene beads. B16-F10 cells were plated on a glass-bottomed 35 mm dish covered with 10 µg/ml fibronectin, and coated with polystyrene beads. Cells were maintained at 370C in CO2-buffered DMEM medium with 10% FCS and penicillin/streptomycin antibiotics. Before transferring the cells to the microscope, the CO2-buffered DMEM medium was replaced with pre-warmed HEPES-buffered medium. Images were acquired using a 10× objective, every 5 minutes for 7 hours.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001457.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(2.14 MB MOV)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>SNA is a recipient of a special fellowship from the Planning and Budgeting Committee of the Israel Ministry of Education. BG holds the Erwin Neter Professorial Chair in Cell and Tumor Biology. ZK holds the Israel Pollak Professorial Chair in Biophysics. We thank the Harvard Institute of Proteomics and Joshua LaBaer, Joseph Pearlberg and Joan Brugge, for making the BC1000 gene collection available to us.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SN. Performed the experiments: SN. Analyzed the data: SN. Wrote the paper: BG SN. Other: Contributed to the development of the image analysis software: NC MG TS. Principal investigator: BG. Co-Principal investigator: ZK. Contributed to the establishment of the BC1000 library examined in this work: SI.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001457-Patan1" id="pone.0001457-Patan1"></a>Patan S (2000) Vasculogenesis and angiogenesis as mechanisms of vascular network formation, growth and remodeling. J Neurooncol  50(1–2): 1–15.  <ul class="find" data-citedArticleID="1043950"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Vasculogenesis+and+angiogenesis+as+mechanisms+of+vascular+network+formation%2C+growth+and+remodeling.&amp;auth=&amp;atitle=Vasculogenesis+and+angiogenesis+as+mechanisms+of+vascular+network+formation%2C+growth+and+remodeling." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vasculogenesis+and+angiogenesis+as+mechanisms+of+vascular+network+formation%2C+growth+and+remodeling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vasculogenesis+and+angiogenesis+as+mechanisms+of+vascular+network+formation%2C+growth+and+remodeling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001457-Price1" id="pone.0001457-Price1"></a>Price LS, Collard JG (2001) Regulation of the cytoskeleton by Rho-family GTPases: implications for tumour cell invasion. Semin Cancer Biol  11(2): 167–73.  <ul class="find" data-citedArticleID="1043952" data-doi="10.1006/scbi.2000.0367"><li><a href="http://dx.doi.org/10.1006/scbi.2000.0367" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+the+cytoskeleton+by+Rho-family+GTPases%3A+implications+for+tumour+cell+invasion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+the+cytoskeleton+by+Rho-family+GTPases%3A+implications+for+tumour+cell+invasion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001457-Libotte1" id="pone.0001457-Libotte1"></a>Libotte T, Kaiser HW, Alt W, Bretschneider T (2001) Polarity, protrusion-retraction dynamics and their interplay during keratinocyte cell migration. Exp Cell Res  270(2): 129–37.  <ul class="find" data-citedArticleID="1043930" data-doi="10.1006/excr.2001.5339"><li><a href="http://dx.doi.org/10.1006/excr.2001.5339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polarity%2C+protrusion-retraction+dynamics+and+their+interplay+during+keratinocyte+cell+migration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polarity%2C+protrusion-retraction+dynamics+and+their+interplay+during+keratinocyte+cell+migration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001457-Wolf1" id="pone.0001457-Wolf1"></a>Wolf K, Mazo I, Leung H, Engelke K, von Andrian UH, et al.  (2003) Compensation mechanism in tumor cell migration: mesenchymal-amoeboid transition after blocking of pericellular proteolysis. J Cell Biol  160(2): 267–77.  <ul class="find" data-citedArticleID="1043960" data-doi="10.1083/jcb.200209006"><li><a href="http://dx.doi.org/10.1083/jcb.200209006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Compensation+mechanism+in+tumor+cell+migration%3A+mesenchymal-amoeboid+transition+after+blocking+of+pericellular+proteolysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Compensation+mechanism+in+tumor+cell+migration%3A+mesenchymal-amoeboid+transition+after+blocking+of+pericellular+proteolysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001457-Wong1" id="pone.0001457-Wong1"></a>Wong MK, Gotlieb AI (1988) The reorganization of microfilaments, centrosomes, and microtubules during in vitro small wound reendothelialization. J Cell Biol  107(5): 1777–83.  <ul class="find" data-citedArticleID="1043962" data-doi="10.1083/jcb.107.5.1777"><li><a href="http://dx.doi.org/10.1083/jcb.107.5.1777" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+reorganization+of+microfilaments%2C+centrosomes%2C+and+microtubules+during+in+vitro+small+wound+reendothelialization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+reorganization+of+microfilaments%2C+centrosomes%2C+and+microtubules+during+in+vitro+small+wound+reendothelialization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001457-Coomber1" id="pone.0001457-Coomber1"></a>Coomber BL, Gotlieb AI (1990) In vitro endothelial wound repair. Interaction of cell migration and proliferation. Arteriosclerosis  10(2): 215–22.  <ul class="find" data-citedArticleID="1043920" data-doi="10.1161/01.atv.10.2.215"><li><a href="http://dx.doi.org/10.1161/01.atv.10.2.215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+endothelial+wound+repair.+Interaction+of+cell+migration+and+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+endothelial+wound+repair.+Interaction+of+cell+migration+and+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001457-Zahm1" id="pone.0001457-Zahm1"></a>Zahm JM, Kaplan H, Herard AL, Doriot F, Pierrot D, et al.  (1997) Cell migration and proliferation during the in vitro wound repair of the respiratory epithelium. Cell Motil Cytoskeleton  37(1): 33–43.  <ul class="find" data-citedArticleID="1043964" data-doi="10.1002/(sici)1097-0169(1997)37:1<33::aid-cm4&gt;3.0.co;2-i"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0169(1997)37:1<33::aid-cm4&gt;3.0.co;2-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+migration+and+proliferation+during+the+in+vitro+wound+repair+of+the+respiratory+epithelium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+migration+and+proliferation+during+the+in+vitro+wound+repair+of+the+respiratory+epithelium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001457-Boyden1" id="pone.0001457-Boyden1"></a>Boyden S (1962) The chemotactic effect of mixtures of antibody and antigen on polymorphonuclear leucocytes. J Exp Med  115: 453–66.  <ul class="find" data-citedArticleID="1043916" data-doi="10.1084/jem.115.3.453"><li><a href="http://dx.doi.org/10.1084/jem.115.3.453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chemotactic+effect+of+mixtures+of+antibody+and+antigen+on+polymorphonuclear+leucocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chemotactic+effect+of+mixtures+of+antibody+and+antigen+on+polymorphonuclear+leucocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001457-Mastyugin1" id="pone.0001457-Mastyugin1"></a>Mastyugin V, McWhinnie E, Labow M, Buxton F (2004) A quantitative high-throughput endothelial cell migration assay. J Biomol Screen  9(8): 712–8.  <ul class="find" data-citedArticleID="1043936"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+quantitative+high-throughput+endothelial+cell+migration+assay.&amp;auth=&amp;atitle=A+quantitative+high-throughput+endothelial+cell+migration+assay." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+quantitative+high-throughput+endothelial+cell+migration+assay." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+quantitative+high-throughput+endothelial+cell+migration+assay.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001457-AlbrechtBuehler1" id="pone.0001457-AlbrechtBuehler1"></a>Albrecht-Buehler G (1977) The phagokinetic tracks of 3T3 cells. Cell  11(2): 395–404.  <ul class="find" data-citedArticleID="1043910" data-doi="10.1016/0092-8674(77)90057-5"><li><a href="http://dx.doi.org/10.1016/0092-8674(77)90057-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+phagokinetic+tracks+of+3T3+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+phagokinetic+tracks+of+3T3+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001457-Pankov1" id="pone.0001457-Pankov1"></a>Pankov R, Endo Y, Even-Ram S, Araki M, Clark K, et al.  (2005) A Rac switch regulates random versus directionally persistent cell migration. J Cell Biol  170(5): 793–802.  <ul class="find" data-citedArticleID="1043946" data-doi="10.1083/jcb.200503152"><li><a href="http://dx.doi.org/10.1083/jcb.200503152" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Rac+switch+regulates+random+versus+directionally+persistent+cell+migration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Rac+switch+regulates+random+versus+directionally+persistent+cell+migration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001457-Ogata1" id="pone.0001457-Ogata1"></a>Ogata R (1998) Type IV collagen and laminin enhance the motility, adhesion, and proliferation of hepatoma cells. Kurume Med J  45(1): 11–20.  <ul class="find" data-citedArticleID="1043940" data-doi="10.2739/kurumemedj.45.11"><li><a href="http://dx.doi.org/10.2739/kurumemedj.45.11" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Type+IV+collagen+and+laminin+enhance+the+motility%2C+adhesion%2C+and+proliferation+of+hepatoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Type+IV+collagen+and+laminin+enhance+the+motility%2C+adhesion%2C+and+proliferation+of+hepatoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001457-Ayaki1" id="pone.0001457-Ayaki1"></a>Ayaki M, Mukai M, Imamura F, Iwasaki T, Mammoto T, et al.  (2000) Cooperation of fibronectin with lysophosphatidic acid induces motility and transcellular migration of rat ascites hepatoma cells. Biochim Biophys Acta  1495(1): 40–50.  <ul class="find" data-citedArticleID="1043912" data-doi="10.1016/s0167-4889(99)00153-6"><li><a href="http://dx.doi.org/10.1016/s0167-4889(99)00153-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cooperation+of+fibronectin+with+lysophosphatidic+acid+induces+motility+and+transcellular+migration+of+rat+ascites+hepatoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cooperation+of+fibronectin+with+lysophosphatidic+acid+induces+motility+and+transcellular+migration+of+rat+ascites+hepatoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001457-Kawa1" id="pone.0001457-Kawa1"></a>Kawa S, Kimura S, Hakomori S, Igarashi Y (1997) Inhibition of chemotactic motility and trans-endothelial migration of human neutrophils by sphingosine 1-phosphate. FEBS Lett  420(2-3): 196–200.  <ul class="find" data-citedArticleID="1043928" data-doi="10.1016/s0014-5793(97)01516-0"><li><a href="http://dx.doi.org/10.1016/s0014-5793(97)01516-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+chemotactic+motility+and+trans-endothelial+migration+of+human+neutrophils+by+sphingosine+1-phosphate." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+chemotactic+motility+and+trans-endothelial+migration+of+human+neutrophils+by+sphingosine+1-phosphate.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001457-Baudoux1" id="pone.0001457-Baudoux1"></a>Baudoux B, Castanares-Zapatero D, Leclercq-Smekens M, Berna N, Poumay Y (2000) The tetraspanin CD9 associates with the integrin alpha6beta4 in cultured human epidermal keratinocytes and is involved in cell motility. Eur J Cell Biol  79(1): 41–51.  <ul class="find" data-citedArticleID="1043914" data-doi="10.1078/s0171-9335(04)70006-0"><li><a href="http://dx.doi.org/10.1078/s0171-9335(04)70006-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+tetraspanin+CD9+associates+with+the+integrin+alpha6beta4+in+cultured+human+epidermal+keratinocytes+and+is+involved+in+cell+motility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+tetraspanin+CD9+associates+with+the+integrin+alpha6beta4+in+cultured+human+epidermal+keratinocytes+and+is+involved+in+cell+motility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001457-Ohmori1" id="pone.0001457-Ohmori1"></a>Ohmori T, Yatomi Y, Okamoto H, Miura Y, Rile G, et al.  (2001) G(i)-mediated Cas tyrosine phosphorylation in vascular endothelial cells stimulated with sphingosine 1-phosphate: possible involvement in cell motility enhancement in cooperation with Rho-mediated pathways. J Biol Chem  276(7): 5274–80.  <ul class="find" data-citedArticleID="1043942" data-doi="10.1074/jbc.m005405200"><li><a href="http://dx.doi.org/10.1074/jbc.m005405200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G%28i%29-mediated+Cas+tyrosine+phosphorylation+in+vascular+endothelial+cells+stimulated+with+sphingosine+1-phosphate%3A+possible+involvement+in+cell+motility+enhancement+in+cooperation+with+Rho-mediated+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G%28i%29-mediated+Cas+tyrosine+phosphorylation+in+vascular+endothelial+cells+stimulated+with+sphingosine+1-phosphate%3A+possible+involvement+in+cell+motility+enhancement+in+cooperation+with+Rho-mediated+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001457-Takanami1" id="pone.0001457-Takanami1"></a>Takanami I, Takeuchi K, Watanabe H, Yanagawa T, Takagishi K (2002) Autocrine motility factor receptor gene expression and cell motility in lung cancer cell lines. Oncol Rep  9(1): 125–8.  <ul class="find" data-citedArticleID="1043956" data-doi="10.3892/or.9.1.125"><li><a href="http://dx.doi.org/10.3892/or.9.1.125" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Autocrine+motility+factor+receptor+gene+expression+and+cell+motility+in+lung+cancer+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Autocrine+motility+factor+receptor+gene+expression+and+cell+motility+in+lung+cancer+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001457-Mukai1" id="pone.0001457-Mukai1"></a>Mukai M, Iwasaki T, Tatsuta M, Togawa A, Nakamura H, et al.  (2003) Cyclic phosphatidic acid inhibits RhoA-mediated autophosphorylation of FAK at Tyr-397 and subsequent tumor-cell invasion. Int J Oncol  22(6): 1247–56.  <ul class="find" data-citedArticleID="1043938" data-doi="10.3892/ijo.22.6.1247"><li><a href="http://dx.doi.org/10.3892/ijo.22.6.1247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cyclic+phosphatidic+acid+inhibits+RhoA-mediated+autophosphorylation+of+FAK+at+Tyr-397+and+subsequent+tumor-cell+invasion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cyclic+phosphatidic+acid+inhibits+RhoA-mediated+autophosphorylation+of+FAK+at+Tyr-397+and+subsequent+tumor-cell+invasion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001457-Onishi1" id="pone.0001457-Onishi1"></a>Onishi Y, Tsukada K, Yokota J, Raz A (2003) Overexpression of autocrine motility factor receptor (AMFR) in NIH3T3 fibroblasts induces cell transformation. Clin Exp Metastasis  20(1): 51–8.  <ul class="find" data-citedArticleID="1043944"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Overexpression+of+autocrine+motility+factor+receptor+%28AMFR%29+in+NIH3T3+fibroblasts+induces+cell+transformation.&amp;auth=&amp;atitle=Overexpression+of+autocrine+motility+factor+receptor+%28AMFR%29+in+NIH3T3+fibroblasts+induces+cell+transformation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Overexpression+of+autocrine+motility+factor+receptor+%28AMFR%29+in+NIH3T3+fibroblasts+induces+cell+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Overexpression+of+autocrine+motility+factor+receptor+%28AMFR%29+in+NIH3T3+fibroblasts+induces+cell+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001457-Iwasaki1" id="pone.0001457-Iwasaki1"></a>Iwasaki T, Shinkai K, Mukai M, Yoshioka K, Fujii Y, et al.  (1995) Cell-cycle-dependent invasion in vitro by rat ascites hepatoma cells. Int J Cancer  63(2): 282–7.  <ul class="find" data-citedArticleID="1043926" data-doi="10.1002/ijc.2910630223"><li><a href="http://dx.doi.org/10.1002/ijc.2910630223" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-cycle-dependent+invasion+in+vitro+by+rat+ascites+hepatoma+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-cycle-dependent+invasion+in+vitro+by+rat+ascites+hepatoma+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001457-Itoh1" id="pone.0001457-Itoh1"></a>Itoh K, Yoshioka K, Akedo H, Uehata M, Ishizaki T, et al.  (1999) An essential part for Rho-associated kinase in the transcellular invasion of tumor cells. Nat Med  5(2): 221–5.  <ul class="find" data-citedArticleID="1043924"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+essential+part+for+Rho-associated+kinase+in+the+transcellular+invasion+of+tumor+cells.&amp;auth=&amp;atitle=An+essential+part+for+Rho-associated+kinase+in+the+transcellular+invasion+of+tumor+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+essential+part+for+Rho-associated+kinase+in+the+transcellular+invasion+of+tumor+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+essential+part+for+Rho-associated+kinase+in+the+transcellular+invasion+of+tumor+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001457-Clark1" id="pone.0001457-Clark1"></a>Clark EA, Golub TR, Lander ES, Hynes RO (2000) Genomic analysis of metastasis reveals an essential role for RhoC. Nature  406(6795): 532–5.  <ul class="find" data-citedArticleID="1043918" data-doi="10.1038/35020106"><li><a href="http://dx.doi.org/10.1038/35020106" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+analysis+of+metastasis+reveals+an+essential+role+for+RhoC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+analysis+of+metastasis+reveals+an+essential+role+for+RhoC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001457-Liotta1" id="pone.0001457-Liotta1"></a>Liotta LA, Kohn EC (2001) The microenvironment of the tumour-host interface. Nature  411(6835): 375–9.  <ul class="find" data-citedArticleID="1043932" data-doi="10.1038/35077241"><li><a href="http://dx.doi.org/10.1038/35077241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+microenvironment+of+the+tumour-host+interface." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+microenvironment+of+the+tumour-host+interface.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001457-Paran1" id="pone.0001457-Paran1"></a>Paran Y, Ilan M, Kashman Y, Goldstein S, Liron Y, et al.  (2007) High-throughput screening of cellular features using high-resolution light-microscopy; application for profiling drug effects on cell adhesion. J Struct Biol  158(2): 233–43.  <ul class="find" data-citedArticleID="1043948" data-doi="10.1016/j.jsb.2006.12.013"><li><a href="http://dx.doi.org/10.1016/j.jsb.2006.12.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High-throughput+screening+of+cellular+features+using+high-resolution+light-microscopy%3B+application+for+profiling+drug+effects+on+cell+adhesion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High-throughput+screening+of+cellular+features+using+high-resolution+light-microscopy%3B+application+for+profiling+drug+effects+on+cell+adhesion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001457-Liron1" id="pone.0001457-Liron1"></a>Liron Y, Paran Y, Zatorsky NG, Geiger B, Kam Z (2006) Laser autofocusing system for high-resolution cell biological imaging. J Microsc  221(Pt 2): 145–51.  <ul class="find" data-citedArticleID="1043934" data-doi="10.1111/j.1365-2818.2006.01550.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2818.2006.01550.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Laser+autofocusing+system+for+high-resolution+cell+biological+imaging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Laser+autofocusing+system+for+high-resolution+cell+biological+imaging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001457-Sharon1" id="pone.0001457-Sharon1"></a>Sharon E, Brandet A, Basri R (2001) Segmentation and boundary detection using multiscale intensity measurements. IEEE I469-I476.   <ul class="find" data-citedArticleID="1043954"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Segmentation+and+boundary+detection+using+multiscale+intensity+measurements.&amp;auth=&amp;atitle=Segmentation+and+boundary+detection+using+multiscale+intensity+measurements." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Segmentation+and+boundary+detection+using+multiscale+intensity+measurements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Segmentation+and+boundary+detection+using+multiscale+intensity+measurements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001457-Witt1" id="pone.0001457-Witt1"></a>Witt AE, Hines LM, Collins NL, Hu Y, Gunawardane RN, et al.  (2006) Functional proteomics approach to investigate the biological activities of cDNAs implicated in breast cancer. J Proteome Res  5(3): 599–610.  <ul class="find" data-citedArticleID="1043958" data-doi="10.1021/pr050395r"><li><a href="http://dx.doi.org/10.1021/pr050395r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+proteomics+approach+to+investigate+the+biological+activities+of+cDNAs+implicated+in+breast+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+proteomics+approach+to+investigate+the+biological+activities+of+cDNAs+implicated+in+breast+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001457-Isakoff1" id="pone.0001457-Isakoff1"></a>Isakoff SJ, Engelman JA, Irie HY, Luo J, Brachmann SM, et al.  (2005) Breast cancer-associated PIK3CA mutations are oncogenic in mammary epithelial cells. Cancer Res  65(23): 10992–1000.  <ul class="find" data-citedArticleID="1043922" data-doi="10.1158/0008-5472.can-05-2612"><li><a href="http://dx.doi.org/10.1158/0008-5472.can-05-2612" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Breast+cancer-associated+PIK3CA+mutations+are+oncogenic+in+mammary+epithelial+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Breast+cancer-associated+PIK3CA+mutations+are+oncogenic+in+mammary+epithelial+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.XML" value="69550"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.PDF" value="731065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g001.PNG_L" value="1557790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g001.PNG_M" value="241719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g001.PNG_S" value="10936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g001.TIF" value="2252484"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g001.PNG_I" value="76533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g002.PNG_L" value="1016048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g002.PNG_M" value="154963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g002.PNG_S" value="16515"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g002.TIF" value="1855886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g002.PNG_I" value="88646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g003.PNG_L" value="700272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g003.PNG_M" value="82549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g003.PNG_S" value="5840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g003.TIF" value="717498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g003.PNG_I" value="24343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g004.PNG_L" value="2738410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g004.PNG_M" value="176160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g004.PNG_S" value="10442"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g004.TIF" value="3189348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g004.PNG_I" value="87808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g005.PNG_L" value="1543210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g005.PNG_M" value="178929"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g005.PNG_S" value="11686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g005.TIF" value="2387732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.g005.PNG_I" value="60961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s001.DOC" value="31744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s002.DOC" value="25600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s003.DOC" value="27648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s004.DOC" value="28672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s005.DOC" value="42496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s006.DOC" value="48640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s007.TIF" value="2028058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s008.TIF" value="5616520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s009.TIF" value="831118"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s010.MOV" value="2926950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001457.s011.MOV" value="2137978"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001457&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001457" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001457&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001457&volume=&issue=&title=Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening&author_name=Suha%20Naffar-Abu-Amara%2C%20Tal%20Shay%2C%20Meirav%20Galun%2C%20Naomi%20Cohen%2C%20Steven%20J.%20Isakoff%2C%20Zvi%20Kam%2C%20Benjamin%20Geiger&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001457" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457&amp;t=Identification%20of%20Novel%20Pro-Migratory%2C%20Cancer-Associated%20Genes%20Using%20Quantitative%2C%20Microscopy-Based%20Screening" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457&title=Identification%20of%20Novel%20Pro-Migratory%2C%20Cancer-Associated%20Genes%20Using%20Quantitative%2C%20Microscopy-Based%20Screening&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457&amp;title=Identification%20of%20Novel%20Pro-Migratory%2C%20Cancer-Associated%20Genes%20Using%20Quantitative%2C%20Microscopy-Based%20Screening" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001457&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Identification of Novel Pro-Migratory, Cancer-Associated Genes Using Quantitative, Microscopy-Based Screening';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001457';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Identification%20of%20Novel%20Pro-Migratory%2C%20Cancer-Associated%20Genes%20Using%20Quantitative%2C%20Microscopy-Based%20Screening http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001457" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001457" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cancer+cell+migration%22" title="Search for articles in the subject area:'Cancer cell migration'"><div class="flagText">Cancer cell migration</div></a>
              <div data-categoryid="33965" data-articleid="25742"
                   data-categoryname="Cancer cell migration"
                   class="flagImage" title="Flag 'Cancer cell migration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+migration%22" title="Search for articles in the subject area:'Cell migration'"><div class="flagText">Cell migration</div></a>
              <div data-categoryid="17279" data-articleid="25742"
                   data-categoryname="Cell migration"
                   class="flagImage" title="Flag 'Cell migration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+motility%22" title="Search for articles in the subject area:'Cell motility'"><div class="flagText">Cell motility</div></a>
              <div data-categoryid="17359" data-articleid="25742"
                   data-categoryname="Cell motility"
                   class="flagImage" title="Flag 'Cell motility' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+screens%22" title="Search for articles in the subject area:'Genetic screens'"><div class="flagText">Genetic screens</div></a>
              <div data-categoryid="47743" data-articleid="25742"
                   data-categoryname="Genetic screens"
                   class="flagImage" title="Flag 'Genetic screens' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Library+screening%22" title="Search for articles in the subject area:'Library screening'"><div class="flagText">Library screening</div></a>
              <div data-categoryid="17295" data-articleid="25742"
                   data-categoryname="Library screening"
                   class="flagImage" title="Flag 'Library screening' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Morphometry%22" title="Search for articles in the subject area:'Morphometry'"><div class="flagText">Morphometry</div></a>
              <div data-categoryid="24039" data-articleid="25742"
                   data-categoryname="Morphometry"
                   class="flagImage" title="Flag 'Morphometry' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Perimeters%22" title="Search for articles in the subject area:'Perimeters'"><div class="flagText">Perimeters</div></a>
              <div data-categoryid="36323" data-articleid="25742"
                   data-categoryname="Perimeters"
                   class="flagImage" title="Flag 'Perimeters' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polystyrene%22" title="Search for articles in the subject area:'Polystyrene'"><div class="flagText">Polystyrene</div></a>
              <div data-categoryid="28977" data-articleid="25742"
                   data-categoryname="Polystyrene"
                   class="flagImage" title="Flag 'Polystyrene' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4354'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4353'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1212&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=2647">Referee comments: Referee 1 (J. Victor Small)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
