

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000972"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000972" />

    <meta name="citation_title" content="An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides"/>
    <meta itemprop="name" content="An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides"/>

      <meta name="citation_author" content="Matthew C. T. Hartman"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Kristopher Josephson"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Chi-Wang Lin"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Jack W. Szostak"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/10/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000972.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e972"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Ribosomal synthesis of unnatural peptides.; citation_author=K Josephson; citation_author=MCT Hartman; citation_author=JW Szostak; citation_journal_title=J Am Chem Soc; citation_volume=127; citation_number=1; citation_pages=11727-11735; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A simplified reconstitution of mRNA-directed peptide synthesis: activity of the epsilon enhancer and an unnatural amino acid.; citation_author=AC Forster; citation_author=H Weissbach; citation_author=SC Blacklow; citation_journal_title=Anal Biochem; citation_volume=297; citation_number=2; citation_pages=60-70; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Programming peptidomimetic syntheses by translating genetic codes designed de novo.; citation_author=AC Forster; citation_author=Z Tan; citation_author=MN Nalam; citation_author=H Lin; citation_author=H Qu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=3; citation_pages=6353-6357; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cell-free translation reconstituted with purified components.; citation_author=Y Shimizu; citation_author=A Inoue; citation_author=Y Tomari; citation_author=T Suzuki; citation_author=T Yokogawa; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=4; citation_pages=751-755; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=T4 RNA ligase mediated preparation of novel chemically misacylated transfer-RNA Phes.; citation_author=TG Heckler; citation_author=LH Chang; citation_author=Y Zama; citation_author=T Naka; citation_author=MS Chorghade; citation_journal_title=Biochemistry; citation_volume=23; citation_number=5; citation_pages=1468-1473; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=A general method for site-specific incorporation of unnatural amino acids into proteins.; citation_author=CJ, J. A-CS Noren; citation_author=MC Griffith; citation_author=PG Schultz; citation_journal_title=Science; citation_volume=244; citation_number=6; citation_pages=182-188; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Biosynthetic site-specific incorporation of a non-natural amino-acid into a polypeptide.; citation_author=JD Bain; citation_author=ES Diala; citation_author=CG Glabe; citation_author=TA Dix; citation_author=AR Chamberlin; citation_journal_title=J Am Chem Soc; citation_volume=111; citation_number=7; citation_pages=8013-8014; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=A highly flexible tRNA acylation method for non-natural polypeptide synthesis.; citation_author=H Murakami; citation_author=M Ohta; citation_author=H Ashigai; citation_author=H Suga; citation_journal_title=Nat Methods; citation_volume=3; citation_number=8; citation_pages=357-359; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=In situ chemical aminoacylation with amino acid thioesters linked to a peptide nucleic acid.; citation_author=K Ninomiya; citation_author=T Minohata; citation_author=M Nishimura; citation_author=M Sisido; citation_journal_title=J Am Chem Soc; citation_volume=126; citation_number=9; citation_pages=15984-15989; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Expanding the genetic code.; citation_author=L Wang; citation_author=J Xie; citation_author=PG Schultz; citation_journal_title=Annu Rev Biophys Biomol Struct; citation_volume=35; citation_number=10; citation_pages=225-249; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Enzymatic aminoacylation of tRNA with unnatural amino acids.; citation_author=MCT Hartman; citation_author=K Josephson; citation_author=JW Szostak; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=11; citation_pages=4356-4361; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Adaptability of nonnatural aromatic amino acids to the active center of the E. coli ribosomal A site.; citation_author=T Hohsaka; citation_author=K Sato; citation_author=M Sisido; citation_author=K Takai; citation_author=S Yokoyama; citation_journal_title=FEBS Lett; citation_volume=335; citation_number=12; citation_pages=47-50; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Efficient incorporation of nonnatural amino acids with large aromatic groups into streptavidin in in vitro protein synthesizing systems.; citation_author=T Hohsaka; citation_author=D Kajihara; citation_author=Y Ashizuka; citation_author=H Murakami; citation_author=M Sisido; citation_journal_title=J Am Chem Soc; citation_volume=121; citation_number=13; citation_pages=34-40; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Predicting the binding affinities of misacylated tRNAs for Thermus thermophilus EF-Tu.GTP.; citation_author=H Asahara; citation_author=OC Uhlenbeck; citation_journal_title=Biochemistry; citation_volume=44; citation_number=14; citation_pages=11254-11261; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Amino acid backbone specificity of the Escherichia coli translation machinery.; citation_author=Z Tan; citation_author=AC Forster; citation_author=SC Blacklow; citation_author=VW Cornish; citation_journal_title=J Am Chem Soc; citation_volume=126; citation_number=15; citation_pages=12752-12753; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Self-assembling amphiphilic siderophores from marine bacteria.; citation_author=JS Martinez; citation_author=GP Zhang; citation_author=PD Holt; citation_author=HT Jung; citation_author=CJ Carrano; citation_journal_title=Science; citation_volume=287; citation_number=16; citation_pages=1245-1247; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A fast screening method for the identification of siderophores from fluorescent Pseudomonas spp. by liquid chromatography/electrospray mass spectrometry.; citation_author=S Kilz; citation_author=C Lenz; citation_author=R Fuchs; citation_author=H Budzikiewicz; citation_journal_title=J Mass Spectrom; citation_volume=34; citation_number=17; citation_pages=281-290; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Amino acid substrate specificity of asparaginyl-,aspartyl-, and glutaminyl-tRNA synthetase isolated from higher plants.; citation_author=PJ Lea; citation_author=L Fowden; citation_journal_title=Phytochemistry; citation_volume=12; citation_number=18; citation_pages=1903-1916; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=Divergence in noncognate amino acid recognition between class I and class II lysyl-tRNA synthetases.; citation_author=J Levengood; citation_author=SF Ataide; citation_author=H Roy; citation_author=M Ibba; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=19; citation_pages=17707-17714; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Effects of incorporation of trans-4,5-dehydrolysine on collagen biosynthesis and extrusion in embryonic chick tibiae.; citation_author=PJ Christner; citation_author=J Rosenbloom; citation_journal_title=J Biol Chem; citation_volume=246; citation_number=20; citation_pages=7551-7556; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=Antibacterial lysine analogs that target lysine riboswitches.; citation_author=KF Blount; citation_author=JX Wang; citation_author=J Lim; citation_author=N Sudarsan; citation_author=RR Breaker; citation_journal_title=Nat Chem Biol; citation_volume=3; citation_number=21; citation_pages=44-49; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Phenotypic suppression by incorporation of an alien amino acid.; citation_author=B Lemeignan; citation_author=P Sonigo; citation_author=P Marliere; citation_journal_title=J Mol Biol; citation_volume=231; citation_number=22; citation_pages=161-166; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Interfering with different steps of protein synthesis explored by transcriptional profiling of Escherichia coli K-12.; citation_author=J Sabina; citation_author=N Dover; citation_author=LJ Templeton; citation_author=DR Smulski; citation_author=D Soll; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=23; citation_pages=6158-6170; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Applications of amino acid analogs for studying co and posttranslational modifications of proteins.; citation_author=G Hortin; citation_author=I Boime; citation_journal_title=Methods Enzymol; citation_volume=96; citation_number=24; citation_pages=777-784; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Microviridins, elastase inhibitors from the cyanobacterium Nostoc Minutum (NIES-26).; citation_author=M Murakami; citation_author=Q Sun; citation_author=K Ishida; citation_author=H Matsuda; citation_author=T Okino; citation_journal_title=Photochemistry; citation_volume=45; citation_number=25; citation_pages=1197-1202; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Influence of γ-glutamyl hydrazide on protein synthesis of Pseudomonas aeruginosa.; citation_author=AL Neal; citation_author=L Libman; citation_author=ME Smulson; citation_journal_title=Arch Biochem Biophys; citation_volume=127; citation_number=26; citation_pages=426-428; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Metabolism and action of amino acid analog anti-cancer agents.; citation_author=GS Ahluwalia; citation_author=JL Grem; citation_author=Z Hao; citation_author=DA Cooney; citation_journal_title=Pharmacol Ther; citation_volume=46; citation_number=27; citation_pages=243-271; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Synthesis of a novel histidine analogue and its efficient incorporation into a protein in vivo.; citation_author=Y Ikeda; citation_author=S Kawahara; citation_author=M Taki; citation_author=A Kuno; citation_author=T Hasegawa; citation_journal_title=Protein Eng; citation_volume=16; citation_number=28; citation_pages=699-706; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The effect of amino acid analogues on alkaline phosphatase formation in Escherichia coli K-12. I. Substitution of triazolealanine for histidine.; citation_author=S Schlesinger; citation_author=MJ Schlesinger; citation_journal_title=J Biol Chem; citation_volume=242; citation_number=29; citation_pages=3369-3372; citation_date=1967; " />
      <meta name="citation_reference" content="citation_title=Incorporation of an unnatural amino acid in the active site of porcine pancreatic phospholipase A2. Substitution of histidine by 1,2,4-triazole-3-alanine yields an enzyme with high activity at acidic pH.; citation_author=SH Beiboer; citation_author=B van den Berg; citation_author=N Dekker; citation_author=RC Cox; citation_author=HM Verheij; citation_journal_title=Protein Eng; citation_volume=9; citation_number=30; citation_pages=345-352; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Towards new protein engineering: in vivo building and folding of protein shuttles for drug delivery and targeting by the selective pressure incorporation (SPI) method.; citation_author=C Minks; citation_author=S Alefelder; citation_author=L Moroder; citation_author=R Huber; citation_author=N Budisa; citation_journal_title=Tetrahedron; citation_volume=56; citation_number=31; citation_pages=9431-9442; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Biosynthesis of proteins incorporating a versatile set of phenylalanine analogues.; citation_author=K Kirshenbaum; citation_author=IS Carrico; citation_author=DA Tirrell; citation_journal_title=Chembiochem; citation_volume=3; citation_number=32; citation_pages=235-237; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Proteins with beta-(thienopyrrolyl)alanines as alternative chromophores and pharmaceutically active amino acids.; citation_author=N Budisa; citation_author=S Alefelder; citation_author=JH Bae; citation_author=R Golbik; citation_author=C Minks; citation_journal_title=Protein Sci; citation_volume=10; citation_number=33; citation_pages=1281-1292; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Incorporation of tryptophan and its benzo(b)thiophene, 1-methylindole, and indene analogs into protein of Escherichia coli.; citation_author=LE Hall; citation_author=GD Hegeman; citation_author=TR Bosin; citation_journal_title=Res Commun Chem Pathol Pharmacol; citation_volume=9; citation_number=34; citation_pages=145-153; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Inhibitory glutamate receptor channels.; citation_author=TA Cleland; citation_journal_title=Mol Neurobiol; citation_volume=13; citation_number=35; citation_pages=97-136; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Receptors for the excitatory amino acids in the mammalian central nervous system.; citation_author=H McLennan; citation_journal_title=Prog Neurobiol; citation_volume=20; citation_number=36; citation_pages=251-271; citation_date=1983; " />
      <meta name="citation_reference" content="citation_title=Site-Specific Incorporation of non-natural residues into peptides: Effect of residue structure on suppression and translation efficiencies.; citation_author=JD Bain; citation_author=DA Wacker; citation_author=EE Kuo; citation_author=AR Chamberlin; citation_journal_title=Tetrahedron; citation_volume=47; citation_number=37; citation_pages=2389-2400; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The effect of ethionine on the synthesis of beta galactosidase: formation of an immunologically cross-reacting protein.; citation_author=J Spizek; citation_author=J Janacek; citation_journal_title=Biochem Biophys Res Commun; citation_volume=34; citation_number=38; citation_pages=17-21; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Efficient incorporation of unsaturated methionine analogues into proteins in vivo.; citation_author=JCM van Hest; citation_author=KL Kiick; citation_author=DA Tirrell; citation_journal_title=J Am Chem Soc; citation_volume=122; citation_number=39; citation_pages=1282-1288; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Incorporation of azides into recombinant proteins for chemoselective modification by the Staudinger ligation.; citation_author=KL Kiick; citation_author=E Saxon; citation_author=DA Tirrell; citation_author=CR Bertozzi; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=40; citation_pages=19-24; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Identification of an expanded set of translationally active methionine analogues in Escherichia coli.; citation_author=KL Kiick; citation_author=R Weberskirch; citation_author=DA Tirrell; citation_journal_title=FEBS Lett; citation_volume=502; citation_number=41; citation_pages=25-30; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Attenuation of the editing activity of the Escerichia coli leucyl-tRNA synthetase allows incorporation of novel amino acids into proteins in vivo.; citation_author=Y Tang; citation_author=DA Tirrell; citation_journal_title=Biochemistry; citation_volume=41; citation_number=42; citation_pages=10635-10645; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural and mechanistic basis of pre- and posttransfer editing by leucyl-tRNA synthetase.; citation_author=TL Lincecum; citation_author=M Tukalo; citation_author=A Yaremchuk; citation_author=RS Mursinna; citation_author=AM Williams; citation_journal_title=Mol Cell; citation_volume=11; citation_number=43; citation_pages=951-963; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Photo-leucine and photo-methionine allow identification of protein-protein interactions in living cells.; citation_author=M Suchanek; citation_author=A Radzikowska; citation_author=C Thiele; citation_journal_title=Nat Methods; citation_volume=2; citation_number=44; citation_pages=261-267; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Action of thiazolidine-2-carboxylic acid, a proline analog, on protein synthesizing systems.; citation_author=V Busiello; citation_author=M di Girolamo; citation_author=C Cini; citation_author=C De Marco; citation_journal_title=Biochim Biophys Acta; citation_volume=564; citation_number=45; citation_pages=311-321; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Analysis of the amino acid binding to the proline transfer ribonucleic acid synthetase of Escherichia coli.; citation_author=TS Papas; citation_author=AH Mehler; citation_journal_title=J Biol Chem; citation_volume=245; citation_number=46; citation_pages=1588-1595; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Effects of proline analogues on the formation of alkaline phosphatase in Escherichia coli.; citation_author=H Morris; citation_author=MJ Schlesinger; citation_journal_title=J Bacteriol; citation_volume=111; citation_number=47; citation_pages=203-210; citation_date=1972; " />
      <meta name="citation_reference" content="citation_title=Effect of 3,4-dehydro-DL-proline on growth and protein synthesis.; citation_author=L Fowden; citation_author=S Neale; citation_author=H Tristram; citation_journal_title=Nature; citation_volume=199; citation_number=48; citation_pages=35-38; citation_date=1963; " />
      <meta name="citation_reference" content="citation_title=Structural modification of a periodic polypeptide through biosynthetic replacement of proline with azetidine-2-carboxylic acid.; citation_author=TJ Deming; citation_author=MJ Fournier; citation_author=TL Mason; citation_author=DA Tirrell; citation_journal_title=Macromolecules; citation_volume=29; citation_number=49; citation_pages=1442-1444; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Preparation of misacylated aminoacyl-tRNA(Phe)'s useful as probes of the ribosomal acceptor site.; citation_author=JR Roesser; citation_author=C Xu; citation_author=RC Payne; citation_author=CK Surratt; citation_author=SM Hecht; citation_journal_title=Biochemistry; citation_volume=28; citation_number=50; citation_pages=5185-5195; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Site-specific incorporation of novel backbone structures into proteins.; citation_author=JA Ellman; citation_author=D Mendel; citation_author=PG Schultz; citation_journal_title=Science; citation_volume=255; citation_number=51; citation_pages=197-200; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Ribosomal binding and dipeptide formation by misacylated tRNA(Phe)'s.; citation_author=TG Heckler; citation_author=JR Roesser; citation_author=C Xu; citation_author=PI Chang; citation_author=SM Hecht; citation_journal_title=Biochemistry; citation_volume=27; citation_number=52; citation_pages=7254-7262; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Unexpected preference of the E. coli translation system for the ester bond during incorporation of backbone-elongated substrates.; citation_author=S Sando; citation_author=K Abe; citation_author=N Sato; citation_author=T Shibata; citation_author=K Mizusawa; citation_journal_title=J Am Chem Soc; citation_volume=129; citation_number=53; citation_pages=6180-6186; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Metabolism of D-aminoacyl-tRNAs in Escherichia coli and Saccharomyces cerevisiae cells.; citation_author=J Soutourina; citation_author=P Plateau; citation_author=S Blanquet; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=54; citation_pages=32535-32542; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Molecular properties that influence the oral bioavailability of drug candidates.; citation_author=DF Veber; citation_author=SR Johnson; citation_author=HY Cheng; citation_author=BR Smith; citation_author=KW Ward; citation_journal_title=J Med Chem; citation_volume=45; citation_number=55; citation_pages=2615-2623; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Ribosome-catalyzed polyester formation.; citation_author=S Fahnestock; citation_author=A Rich; citation_journal_title=Science; citation_volume=173; citation_number=56; citation_pages=340-343; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=An experimental approach to evaluating the role of backbone interactions in proteins using unnatural amino acid mutagenesis.; citation_author=JT Koh; citation_author=VW Cornish; citation_author=PG Schultz; citation_journal_title=Biochemistry; citation_volume=36; citation_number=57; citation_pages=11314-11322; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A general route for post-translational cyclization of mRNA-displayed libraries.; citation_author=SW Millward; citation_author=TT Takahashi; citation_author=RW Roberts; citation_journal_title=J Am Chem Soc; citation_volume=127; citation_number=58; citation_pages=14142-14143; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular dissection of a critical specificity determinant within the amino acid editing domain of leucyl-tRNA synthetase.; citation_author=RS Mursinna; citation_author=KW Lee; citation_author=JM Briggs; citation_author=SA Martinis; citation_journal_title=Biochemistry; citation_volume=43; citation_number=59; citation_pages=155-165; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Escherichia coli tryptophanyl-tRNA synthetase mutants selected for tryptophan auxotrophy implicate the dimer interface in optimizing amino acid binding.; citation_author=S Sever; citation_author=K Rogers; citation_author=MJ Rogers; citation_author=C Carter; citation_author=D Soll; citation_journal_title=Biochemistry; citation_volume=35; citation_number=60; citation_pages=32-40; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The genetic incorporation of a distance probe into proteins in Escherichia coli.; citation_author=M-L Tsao; citation_author=D Summerer; citation_author=Y Ryu; citation_author=PG Schultz; citation_journal_title=J Am Chem Soc; citation_volume=128; citation_number=61; citation_pages=4572-4573; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Assay of ester and polyester formation by the ribosomal peptidyltransferase.; citation_author=S Fahnestock; citation_author=H Neumann; citation_author=A Rich; citation_journal_title=Methods Enzymol; citation_volume=30; citation_number=62; citation_pages=489-497; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Transformation of aminoacyl tRNAs for the in vitro selection of “drug-like” molecules.; citation_author=C Merryman; citation_author=R Green; citation_journal_title=Chem Biol; citation_volume=11; citation_number=63; citation_pages=575-582; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The suppression of defective translation by ppGpp and its role in the stringent response.; citation_author=PH O'Farrell; citation_journal_title=Cell; citation_volume=14; citation_number=64; citation_pages=545-557; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Anticodon sequence mutants of Escherichia coli initiator tRNA: effects of overproduction of aminoacyl-tRNA synthetases, methionyl-tRNA formyltransferase, and initiation factor 2 on activity in initiation.; citation_author=C Mayer; citation_author=C Kohrer; citation_author=E Kenny; citation_author=C Prusko; citation_author=UL RajBhandary; citation_journal_title=Biochemistry; citation_volume=42; citation_number=65; citation_pages=4787-4799; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=RNA-peptide fusions for the in vitro selection of peptides and proteins.; citation_author=RW Roberts; citation_author=JW Szostak; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=66; citation_pages=12297-12302; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=In vitro selection and evolution of functional proteins by using ribosome display.; citation_author=J Hanes; citation_author=A Pluckthun; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=67; citation_pages=4937-4942; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=An in vitro polysome display system for identifying ligands from very large peptide libraries.; citation_author=LC Mattheakis; citation_author=RR Bhatt; citation_author=WJ Dower; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=91; citation_number=68; citation_pages=9022-9026; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Properties of known drugs. 2. Side chains.; citation_author=GW Bemis; citation_author=MA Murcko; citation_journal_title=J Med Chem; citation_volume=42; citation_number=69; citation_pages=5095-5099; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Reactions of Phosphorus Compounds. XIX. Reactions of 3-(o-formylphenoxy)propyltriphenylphosphonium bromide and 3-(p-formylphenoxy)propyltriphenylphosphonium bromide).; citation_author=EE Schweizer; citation_author=CJ Berninger; citation_author=DM Crouse; citation_author=RA Davis; citation_author=RS Logothetis; citation_journal_title=J Org Chem; citation_volume=34; citation_number=70; citation_pages=207-212; citation_date=1968; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides"/>
    <meta name="twitter:description" content="BackgroundThe application of in vitro translation to the synthesis of unnatural peptides may allow the production of extremely large libraries of highly modified peptides, which are a potential source of lead compounds in the search for new pharmaceutical agents. The specificity of the translation apparatus, however, limits the diversity of unnatural amino acids that can be incorporated into peptides by ribosomal translation. We have previously shown that over 90 unnatural amino acids can be enzymatically loaded onto tRNA.Methodology/Principal FindingsWe have now used a competition assay to assess the efficiency of tRNA-aminoacylation of these analogs. We have also used a series of peptide translation assays to measure the efficiency with which these analogs are incorporated into peptides. The translation apparatus tolerates most side chain derivatives, a few &alpha;,&alpha; disubstituted, N-methyl and &alpha;-hydroxy derivatives, but no &beta;-amino acids. We show that over 50 unnatural amino acids can be incorporated into peptides by ribosomal translation. Using a set of analogs that are efficiently charged and translated we were able to prepare individual peptides containing up to 13 different unnatural amino acids.Conclusions/SignificanceOur results demonstrate that a diverse array of unnatural building blocks can be translationally incorporated into peptides. These building blocks provide new opportunities for in vitro selections with highly modified drug-like peptides."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000972.g011"/>

  <meta property="og:title" content="An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6850'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4292'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2249&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000972">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000972" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000972&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Matthew C. T. Hartman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Department of Chemistry, Massey Cancer Center, Virginia Commonwealth University, Richmond, Virginia, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kristopher Josephson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America
                </p>

                <p>
                    Current address: EMD Serono Research Institute, Rockland, Massachusetts, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chi-Wang Lin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jack W. Szostak
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:szostak@molbio.mgh.harvard.edu">szostak@molbio.mgh.harvard.edu</a></p>

                <p>Affiliation:
                  Howard Hughes Medical Institute, Department of Molecular Biology, Center for Computational and Integrative Biology, Simches Research Center, Massachusetts General Hospital, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000972</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000972-g001" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-t001" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g002" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g003" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g004" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g005" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g006" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g007" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g008" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g009" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g010" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-t002" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-g011" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-t003" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000972-t004" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000972">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000972" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The application of <em>in vitro</em> translation to the synthesis of unnatural peptides may allow the production of extremely large libraries of highly modified peptides, which are a potential source of lead compounds in the search for new pharmaceutical agents. The specificity of the translation apparatus, however, limits the diversity of unnatural amino acids that can be incorporated into peptides by ribosomal translation. We have previously shown that over 90 unnatural amino acids can be enzymatically loaded onto tRNA.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We have now used a competition assay to assess the efficiency of tRNA-aminoacylation of these analogs. We have also used a series of peptide translation assays to measure the efficiency with which these analogs are incorporated into peptides. The translation apparatus tolerates most side chain derivatives, a few α,α disubstituted, <em>N</em>-methyl and α-hydroxy derivatives, but no β-amino acids. We show that over 50 unnatural amino acids can be incorporated into peptides by ribosomal translation. Using a set of analogs that are efficiently charged and translated we were able to prepare individual peptides containing up to 13 different unnatural amino acids.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results demonstrate that a diverse array of unnatural building blocks can be translationally incorporated into peptides. These building blocks provide new opportunities for in vitro selections with highly modified drug-like peptides.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hartman MCT, Josephson K, Lin C-W, Szostak JW (2007) An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides. PLoS ONE 2(10):
          e972.
            doi:10.1371/journal.pone.0000972</p><p><strong>Academic Editor: </strong>Christophe Herman, Baylor College of Medicine, United States of America</p><p><strong>Received:</strong> July 13, 2007; <strong>Accepted:</strong> September 12, 2007; <strong>Published:</strong> October 3, 2007</p><p><strong>Copyright:</strong> © 2007 Hartman et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Howard Hughes Medical Institute. Jack Szostak is an Investigator, and Matthew Hartman, Kris Josephson, and Chi-Wang Lin are/were HHMI Research Associates.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The recent development of translation systems reconstituted entirely from purified components <a href="#pone.0000972-Josephson1">[1]</a>–<a href="#pone.0000972-Shimizu1">[4]</a> has enabled the ribosomal synthesis of peptides composed primarily of unnatural (i.e. non-proteinogenic) amino acids. The major roadblock to the ribosomal synthesis of highly modified drug-like peptides is the limited number of unnatural building blocks known to be compatible with the translation apparatus. This dearth of unnatural building blocks results in part from difficulties in loading unnatural amino acids onto tRNA, the key first step in translation. Several techniques for charging tRNAs with unnatural amino acids have been developed such as chemoenzymatic tRNA acylation <a href="#pone.0000972-Heckler1">[5]</a>–<a href="#pone.0000972-Bain1">[7]</a>, ribozyme acylation <a href="#pone.0000972-Murakami1">[8]</a>, chemical acylation using PNA thioesters <a href="#pone.0000972-Ninomiya1">[9]</a>, and the use of engineered aminoacyl-tRNA synthetases (AARSs) <a href="#pone.0000972-Wang1">[10]</a>. Unfortunately all of these approaches require specialized reagents and/or multistep syntheses. In contrast, an all-enzymatic system using the wild-type AARSs would be as convenient as a standard translation experiment using the twenty proteinogenic amino acids. To this end we have recently developed a MALDI-TOF MS screening assay <a href="#pone.0000972-Hartman1">[11]</a> that has uncovered over 90 unnatural building blocks that are AARS substrates. The incorporation of these amino acids into peptides, however, depends on the efficiency of aminoacylation, and on whether or not they are compatible with the translational steps subsequent to aminoacylation. The specificity of many of these steps with respect to unnatural amino acids is unknown. It is clear; however, that the ribosome does not tolerate certain very sterically demanding aminoacyl-tRNAs <a href="#pone.0000972-Hohsaka1">[12]</a>, <a href="#pone.0000972-Hohsaka2">[13]</a>, and recent studies suggest that a threshold EF-Tu affinity is required for aminoacyl-tRNAs to enter the A-site of the ribosome <a href="#pone.0000972-Asahara1">[14]</a>. Particular types of backbone analogs also seem to be excluded during these translational steps <a href="#pone.0000972-Tan1">[15]</a>, although exactly how they are rejected is not well-understood.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Our newly defined pool of AARS substrates is ideal for addressing these specificity questions because it contains a diverse collection of side chain, <em>N</em>-methyl, α,α-disubstituted, and β-aminoacyl-tRNAs. Here we describe a straightforward means of determining whether an enzymatically charged unnatural amino acid can be effectively incorporated into a peptide. The results of these experiments have allowed us to identify over 50 unnatural amino acids that can be incorporated at a single defined position in a peptide with high efficiency using an all-enzymatic translation system. Our work identifies building blocks that can be combined to prepare highly modified peptides that contain multiple unnatural amino acids. We also identify many cases where the engineering of synthetases, EF-Tu or the ribosome itself may be required to enable efficient incorporation of interesting analogs.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Aminoacylation competition assay</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The MALDI-TOF MS assay that we used to identify unnatural amino acids that are AARS substrates <a href="#pone.0000972-Hartman1">[11]</a> does not provide a quantitative measure of tRNA charging efficiency. The rate of synthesis of aminoacylated tRNA, and the degree to which an analog competes effectively against traces of contamination with the natural amino acid, can greatly affect the yield and purity of analog-containing peptides synthesized by in vitro translation. However, detailed kinetic analysis of large numbers of analogs is not feasible. We therefore devised an AARS competitive inhibition assay in which the charging of a natural amino acid in the presence and absence of an analog could be quantitatively compared by MS through the use of an isotopically labeled derivatizing reagent <a href="#pone.0000972-Hartman1">[11]</a>. In order to compare analog inhibition among different AARSs, we set the natural amino acid at roughly 2x its <em>K<sub>m</sub></em> and then added the unnatural amino acid analog at 1000-fold higher concentration. As an example, the assays for the inhibition measurement of valine analog 3-fluoro-valine (N15) are shown in <a href="#pone-0000972-g001">Figure 1</a>. For each assay an identical amount of non-isotopically derivatized Val-AMP was added as an internal standard, and experiments in the presence or absence of the analog were performed using the deuterium (<em>d</em><sub>15</sub>) labeled derivatizing reagent <a href="#pone.0000972-Hartman1">[11]</a>. The presence of competing N15 clearly led to a strong decrease in the intensity of the <em>d<sub>15</sub></em>-Val-AMP peak (<a href="#pone-0000972-g001">Figure 1B</a>) when compared to the assay with valine alone (<a href="#pone-0000972-g001">Figure 1A</a>). Comparison of the ratio of Val-AMP to <em>d<sub>15</sub></em>-Val-AMP in the presence and absence of the analog allowed determination of the % inhibition value.</p>
<div class="figure" id="pone-0000972-g001"><div class="img"><a name="pone-0000972-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Inhibition of the charging of valine by analog N15.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The intensity of the <em>d</em><sub>15</sub>-Val-AMP in the absence (A) or presence (B) of analog N15 (50 mM) is shown. Valine was present in both assays at 50 µM. The difference in the ratio of Val-AMP and <em>d</em><sub>15</sub>-Val-AMP in the assays was used for the inhibition calculation. As expected for a ValRS substrate, a peak corresponding to the <em>d</em><sub>15</sub>-N15-AMP was also observed in the analog-containing assay. This peak was not used in the determination of inhibition.</p>
<span>doi:10.1371/journal.pone.0000972.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The inhibition data for the analogs is presented in <a href="#pone-0000972-t001">Table 1</a>. As expected, different analogs vary widely in their ability to competitively inhibit the aminoacylation of the natural amino acids. In general, we expect good competitive inhibitors to be good substrates, although this correlation need not always hold. Nevertheless, as discussed below, analogs that are good competitive inhibitors tend to be translated more efficiently than analogs that are poor competitors, supporting the idea that the synthesis of high levels of aminoacylated tRNA during a translation reaction facilitates the synthesis of analog-bearing peptides.</p>
<div class="figure" id="pone-0000972-t001"><div class="img"><a name="pone-0000972-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>AARS competition data for unnatural amino acids</span></strong></p><span>doi:10.1371/journal.pone.0000972.t001</span></div>

<h4>Translation efficiency of analogs</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We have previously defined a set of ~90 amino acid analogs that can be enzymatically charged onto tRNA <a href="#pone.0000972-Hartman1">[11]</a>. To determine which of these could be incorporated into peptides, we used an assay based on the translation of short peptides containing a single unnatural amino acid. We prepared 5 separate mRNA templates containing a C-terminal FLAG or His<sub>6</sub> tag for purification; this set of templates is similar to but more complete than that previously described <a href="#pone.0000972-Josephson1">[1]</a>. Each template was designed to test analogs of 4 of the natural amino acids (<a href="#pone-0000972-g002">Figure 2</a>, <em>a–e</em>). The translation reactions were then carried out in the PURE translation system <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Shimizu1">[4]</a> with <sup>35</sup>S-methionine as the N-terminal radiolabel, except for Met analogs which were tested using <sup>3</sup>H-His and template e. In each test, one of the natural amino acids was omitted and a corresponding analog added in its place; thus synthesis of radiolabeled tagged peptide required readthrough of the codon assigned to the unnatural amino acid <a href="#pone.0000972-Josephson1">[1]</a>. The peptide yield for each unnatural amino acid was determined relative to the peptide yield for the corresponding natural amino acid, by comparison of the peptide radioactivity for the all-natural peptide with that containing the unnatural analog. To verify that the translated peptide contained the unnatural amino acid, the mass of the peptide was determined by MALDI-MS. For unnatural amino acids that differed in mass by +/−1 amu from their natural counterparts, we verified incorporation using templates requiring multiple (4–7) incorporations. For instance, incorporation of the tryptophan analog A18 was tested using a template with 4 tryptophan codons (see supporting information <a href="#pone.0000972.s001">Figure S1</a>) and was unambiguously identified due to its +4 mass difference from the tryptophan containing peptide.</p>
<div class="figure" id="pone-0000972-g002"><div class="img"><a name="pone-0000972-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Peptides encoded by mRNA templates used in this study.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The position(s) of the radiolabeled methionine or histidine are underlined.</p>
<span>doi:10.1371/journal.pone.0000972.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We grouped the amino acid analogs that we studied into three classes based on translation efficiency and specificity. Class I monomers resulted in a peptide yield that was &gt;75% of that obtained with the corresponding natural amino acid, and the translated peptide was homogeneous by MALDI-MS analysis. Class II monomers were translated less efficiently, with a peptide yield ranging from 25% to 75% of that obtained with the natural amino acid, but the translated peptide was still homogeneous. Class III monomers were either poorly translated ( &lt;25% peptide yield) and/or led to a heterogeneous mixture of peptides. In many cases inefficient translation correlated with poor aminoacylation, suggesting that the synthesis of aminoacylated tRNA may have limited peptide yield. Peptide heterogeneity was most often due to the misincorporation of natural amino acids in place of the desired analog. We discuss these analogs in groups based on their side chain and backbone compositions. We used the same amino acid abbreviations used in our previous paper <a href="#pone.0000972-Hartman1">[11]</a> (C = Charged, P = Polar, A = Aromatic, N = Nonpolar, α =  α,α disubstituted, β = beta amino acid, M = <em>N</em>-methyl). Since our first report we have found two additional unnatural amino acids that are efficiently aminoacylated and translated; these compounds (‘Photo-Leu’, N25; ‘Photo-Met’, N26) are also discussed below. Finally, we have described the translation results for twelve of these analogs in a preliminary report <a href="#pone.0000972-Josephson1">[1]</a>; because these analogs are included in the other experiments described herein, we have included them here.</p>


<h4>Charged side chain analogs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Most of the negatively charged GluRS and AspRS substrates shown in <a href="#pone-0000972-g003">Figure 3</a> were efficient translation substrates. All three 4-fluoro-glutamates (C3–C5) were efficiently incorporated, but both 4-methyl analogs (C1 and C2) were poorly translated. Amino acids C1 and C2 also compete much more poorly with glutamic acid for the GluRS active site than their fluoro counterparts, suggesting that inefficient charging is the cause of their poor translation.</p>
<div class="figure" id="pone-0000972-g003"><div class="img"><a name="pone-0000972-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Amino acids with charged or polar side chains and their translation yields relative to the cognate natural AA.</span></strong></p><span>doi:10.1371/journal.pone.0000972.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p><span class="small-caps">l-</span><em>threo</em>-<em>β</em>-hydroxy aspartic acid (C6) is an Asp analog found in certain NRPs, such as the iron-chelating aquachelins <a href="#pone.0000972-Martinez1">[16]</a> and pyoverdins <a href="#pone.0000972-Kilz1">[17]</a>. As observed previously, C6 was efficiently incorporated into peptides in place of Asp <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Lea1">[18]</a>; it also competes well with Asp in the tRNA-Asp aminoacylation assay.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>We have confirmed the efficient translation of the Lys analog, <em>S</em>-2-aminoethyl cysteine (C7) <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Levengood1">[19]</a>, and observed that another Lys analog, trans-dehydro lysine (C8), was translated with high efficiency. C8 is incorporated into collagen in a chick embryo system <a href="#pone.0000972-Christner1">[20]</a>; both C7 and C8 exhibit antibacterial activity due to their ability to bind and inhibit bacterial riboswitches involved in lysine biosynthesis <a href="#pone.0000972-Blount1">[21]</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The positively charged Leu analog aza-leucine (C9) was a strong competitive inhibitor of LeuRS, and was also translated efficiently. Incorporation of this analog into protein is consistent with several reports, including upregulation of heat-shock genes when <em>E. coli</em> is challenged with aza-leucine <a href="#pone.0000972-Lemeignan1">[22]</a>, <a href="#pone.0000972-Sabina1">[23]</a>.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>Of the four arginine analogs found to be AARS substrates, <span class="small-caps">l-</span>canavanine (C10) <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Hortin1">[24]</a> was translated efficiently, while <span class="small-caps">l-</span>NMMA (N<sup>G</sup>-methyl arginine) (C11) was translated with somewhat lower efficiency (65% peptide yield relative to Arg). N<sup>G</sup>-hydroxy arginine (C12) and vinyl-L-NIO (C13) did not yield any detectable peptide, and were also poor competitors in the AARS assay.</p>


<h4>Polar, uncharged</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Two polar but uncharged amino acid analogs are excellent translation substrates (<a href="#pone-0000972-g003">Figure 3</a>). Of these, we and others had previously observed <span class="small-caps">dl-</span><em>β</em>-hydroxy norvaline (P5) to be an excellent threonine surrogate in translation reactions <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Hortin1">[24]</a>. Remarkably, the <span class="small-caps">l-</span>Glu γ-methyl ester (P7) efficiently substituted for methionine in all steps of translation; fully formylated peptides containing the expected double incorporation of P7 were observed in experiments with template <em>e</em>. This analog is found in the microviridins, peptides produced by certain cyanobacteria, which were initially isolated as elastase inhibitors <a href="#pone.0000972-Murakami2">[25]</a>. In contrast, the <span class="small-caps">l-</span>Asp <em>β</em>-methyl ester (P6) led to the translation of a heterogeneous mixture of peptides, including the non-formylated peptide and peptides that initiated at codons other than methionine, suggesting that the initiation complex does not tolerate this analog.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Of the three analogs previously found to be GlnRS substrates (P1–P3), <span class="small-caps">l-</span>glutamic acid γ-hydrazide <a href="#pone.0000972-Lea1">[18]</a>, <a href="#pone.0000972-Neal1">[26]</a>, <span class="small-caps">l-</span>albizziine <a href="#pone.0000972-Lea1">[18]</a>, and <span class="small-caps">l-</span>theanine<a href="#pone.0000972-Lea1">[18]</a>, only the hydrazide was tolerated by the translation apparatus. Acivicin (P4), the cyclic anti-cancer agent and γ-glutamanyl transpeptidase inhibitor <a href="#pone.0000972-Ahluwalia1">[27]</a> was not translated efficiently, consistent with its poor activity in the IleRS competition assay.</p>


<h4>Aromatic Analogs</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>We examined two histidine analogs, <em>β</em>-2-thiazolyl-alanine (A1) and <em>β</em>-(1,2,4-triazol-3-yl-alanine) (A2) <a href="#pone.0000972-Ikeda1">[28]</a>–<a href="#pone.0000972-Beiboer1">[30]</a>; A2 allowed efficient peptide synthesis as described <a href="#pone.0000972-Josephson1">[1]</a>, but we could not detect any peptide product containing A1 (<a href="#pone-0000972-g004">Figure 4</a>). A1 also showed poorer competitive inhibition of HisRS (<a href="#pone-0000972-t001">Table 1</a>). Our observations are entirely consistent with earlier in vivo experiments in which a His-auxotrophic <em>E. coli</em> strain was used to show that A2, but not A1, could be incorporated into a specific protein <a href="#pone.0000972-Ikeda1">[28]</a>.</p>
<div class="figure" id="pone-0000972-g004"><div class="img"><a name="pone-0000972-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Amino acids with aromatic side chains and their translation yields relative to the cognate natural AA.</span></strong></p><span>doi:10.1371/journal.pone.0000972.g004</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The tyrosine analog 3-fluoro tyrosine <a href="#pone.0000972-Hortin1">[24]</a>, <a href="#pone.0000972-Minks1">[31]</a> (A3) was translated efficiently, as previously observed <a href="#pone.0000972-Josephson1">[1]</a>. However, 3-nitro tyrosine (A4), with a bulkier substituent at the 3-position, was not detectably incorporated into peptides. Again, these results correlate with those of the AARS competition assay.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Of the four phenylalanine surrogates previously shown to be PheRS substrates <a href="#pone.0000972-Hartman1">[11]</a>, 2-fluoro Phe (A5), 2-thienyl Ala (A6) <a href="#pone.0000972-Hortin1">[24]</a>, <em>β</em>-methyl Phe (A7), and <em>β</em>-thienyl Ser (A8), only the previously described A5 <a href="#pone.0000972-Josephson1">[1]</a> was translated efficiently. Use of the PheRS A294G mutant allowed several previously described <em>p</em>-substituted analogs (A9–A13) <a href="#pone.0000972-Kirshenbaum1">[32]</a> chloro, bromo, iodo, ethynyl, and azido, to be introduced into peptides in good yields. However, <em>p</em>-nitrophenylalanine (A14) was poorly translated in the presence of this synthetase. Each of the poorly translated Phe analogs were also poor competitors in the AARS assay (all showed less than 25% inhibition).</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>All of the tryptophan analogs examined (A17–A23) proved to be competent translation substrates, despite a wide variation in AARS competition ability. There is some disagreement in the literature <a href="#pone.0000972-Budisa1">[33]</a>, <a href="#pone.0000972-Hall1">[34]</a> whether 3-(thianaphthen-3-yl)-<span class="small-caps">l-</span>alanine (A23) can be translated; in our system it was a reasonably good substrate.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>The aromatic glutamate receptor agonists <a href="#pone.0000972-Cleland1">[35]</a>, <a href="#pone.0000972-McLennan1">[36]</a> and GluRS substrates <span class="small-caps">l-</span>quisqualic acid (A15) and ibotenic acid (A16) had different fates in translation—only <span class="small-caps">l-</span>quisqualic acid was translated in good yields. Ibotenic acid was also much weaker in the AARS competition assay.</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>Both of the aromatic isoleucine surrogates <span class="small-caps">dl-</span>α-(2-thienyl)glycine (A24) and <span class="small-caps">l-</span>phenylglycine (A25) were excellent translation substrates, consistent with the successful in vitro suppression experiments with <span class="small-caps">l-</span>phenylglycine <a href="#pone.0000972-Bain2">[37]</a>.</p>


<h4>Nonpolar</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>A wide variety of nonpolar substrates were translated efficiently (<a href="#pone-0000972-g005">Figure 5</a>). We examined a large number of Met analogs using template e, <sup>3</sup>H-His for labeling, and the His<sub>6</sub> tag for peptide purification. We confirmed the translatability of many previously reported methionine analogs <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Spizek1">[38]</a>–<a href="#pone.0000972-Kiick1">[40]</a> in our assay including 2-amino hex-5-ynoic acid, crotylglycine, <span class="small-caps">l-</span>norleucine,<span class="small-caps"> l-</span>norvaline, <span class="small-caps">l-</span>ethionine, and <span class="small-caps">l-</span>β-azidohomoalanine, and (N1-N6). On the other hand, four other MetRS substrates, 6,6,6-trifluoronorleucine (N7), 2-amino-4,4,4-trifluorobutyric acid (N8), <span class="small-caps">l-</span>C-propargyl glycine (N9) <a href="#pone.0000972-Kiick2">[41]</a>, and <span class="small-caps">l-</span>allyl glycine (N10) <a href="#pone.0000972-Kiick2">[41]</a> resulted in poor translation initiation and/or elongation without any evidence for misincorporation. Analog N10 (as well as N6) <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Tang1">[42]</a> could, however, be charged and incorporated using the editing deficient LeuRS D345A <a href="#pone.0000972-Lincecum1">[43]</a>. β-Cyclopropyl alanine (N11) was efficiently translated, but the resulting peptide was a mixture of full length peptide and peptide lacking the N-terminal residue, suggesting initiation with N11 was difficult. The photoactive diazirine bearing Met analog (N26) <a href="#pone.0000972-Suchanek1">[44]</a> was efficiently incorporated into peptide. This analog, also known as photo-Met, should be a useful probe for studying the binding sites of in vitro translated peptides on protein targets.</p>
<div class="figure" id="pone-0000972-g005"><div class="img"><a name="pone-0000972-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Amino acids with nonpolar side chains and their translation yields relative to the cognate natural AA.</span></strong></p><span>doi:10.1371/journal.pone.0000972.g005</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>Valine analogs 3-fluoro-valine (N12), <em>t</em>-butyl-glycine (N13), and <em>O-</em>methyl-<span class="small-caps">l-</span>threonine (N14) as well as (2S,3S)-2-amino-3-methoxybutanoic acid (N15) were incorporated into peptides in excellent yields. The analog 4-thia-isoleucine (N16) was charged by both ValRS and IleRS, but only the aminoacylated tRNA<sup>Ile</sup> resulted in efficient analog incorporation into peptide. It remains unclear as to whether this difference reflects a difference in tRNA charging efficiency (the AARS competition assay showed they were both moderate competitors) or a difference in the interaction of the charged tRNAs with EF-Tu and/or the ribosome. The isoleucine analog <span class="small-caps">l-</span>cyclohexylglycine (N20) was easily integrated into peptides.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>We and others have previously reported that translation reactions containing the leucine analog 5′,5′,5′-trifluoro leucine (N17) were high yielding <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Hortin1">[24]</a>. LeuRS substrates β-<em>t</em>-butyl-alanine (N18) and β-cyclopentyl alanine (N19) also could be translated in moderate to excellent yields. The photoreactive diazirine bearing Leu analog N25 is aminoacylated by LeuRS and efficiently incorporated into peptides. This analog (photo-Leu) <a href="#pone.0000972-Suchanek1">[44]</a> provides an additional probe of the interaction of translated peptides with protein targets.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>All four proline analogs: thiazolidine-2-carboxylic acid (N21) <a href="#pone.0000972-Busiello1">[45]</a> thiazolidine-4-carboxylic acid (N22) <a href="#pone.0000972-Papas1">[46]</a>, 3,4-dehydro proline (N23) <a href="#pone.0000972-Morris1">[47]</a>, <a href="#pone.0000972-Fowden1">[48]</a>, and <span class="small-caps">l-</span>azetidine-2-carboxylic acid (N24) <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Deming1">[49]</a>, were excellent substrates for the translation apparatus.</p>


<h4>α,α Disubstituted Amino Acids</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>After completing our analysis of side chain analogs, we proceeded to test those backbone analogs that can be enzymatically charged onto tRNA, including α,α disubstituted amino acids, <em>β</em>-amino acids, <em>N</em>-methyl amino acids and α-hydroxy acids (<a href="#pone-0000972-g006">Figure 6</a>). Only two of the seven α,α disubstituted amino acids that are AARS substrates could be incorporated into peptides during translation (<a href="#pone-0000972-g006">Figure 6</a>, α1–α7). These two, 1-amino cyclopentanoic acid (α1) and 1-aminocyclohexanoic acid (α2), were both charged by both ValRS and LeuRS, and both were good translation substrates in response to Val but not Leu codons. Interestingly, α1 and α2 were also the best AARS competitors among the α,α disubstituted amino acids in the competition assay, suggesting that α,α disubstituted AAs incorporation into peptide in our system is controlled at least in part by the AARS. The ribosome and/or EF-Tu are also implicated in low translation efficiencies for α-methyl substituted AAs by a recent study using chemically acylated tRNA <a href="#pone.0000972-Tan1">[15]</a>.</p>
<div class="figure" id="pone-0000972-g006"><div class="img"><a name="pone-0000972-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Backbone-modified amino acids and their translation yields relative to the cognate natural AA.</span></strong></p><span>doi:10.1371/journal.pone.0000972.g006</span></div>

<h4>β-amino acids</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>Although one β<sub>2</sub> (β10) and eleven β<sub>3</sub> (β1–β9, β11) amino acids are known to be AARS substrates <a href="#pone.0000972-Hartman1">[11]</a>, none were incorporated into peptides by the translation apparatus (<a href="#pone-0000972-g006">Figure 6</a>). Although this is a disappointing result, it is consistent with previous reports of poor to undetectable incorporation of β-amino acids in translation experiments <a href="#pone.0000972-Bain1">[7]</a>, <a href="#pone.0000972-Tan1">[15]</a>, <a href="#pone.0000972-Roesser1">[50]</a>, <a href="#pone.0000972-Ellman1">[51]</a>. Literature reports do suggest that β-amino acids can act as peptide donors at the ribosome P site <a href="#pone.0000972-Heckler2">[52]</a>. This is consistent with our finding that β9 is able to serve as an initiator amino acid, albeit inefficiently. Very recently, Sando and coworkers have found that β−hydroxy acids are competent translation substrates and have suggested that the elevated pKa of the amine in β−amino acids prevents deprotonation and peptidyl transfer <a href="#pone.0000972-Sando1">[53]</a>. If this interpretation is correct, ribosome engineering may be required to enable the efficient incorporation of β-amino acids into peptides.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>One difficulty with our studies of the aminoacylation of the β-amino acids is that some of them have an identical mass to their natural analogs (β2, β4–β6, and β11). Since it is impossible to rule out natural amino acid contamination using our MALDI-MS assay for tRNA aminoacylation, it was possible that these analogs were not actually charged. However, the observation that they are not translated confirms that they are charged onto tRNA, since tRNA charging with contaminating natural amino acid would have led to efficient peptide translation.</p>


<h4><em>N</em>-methyl amino acids</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>The <em>N</em>-methyl amino acids are of particular interest since they are very common in the natural product non-ribosomal peptides, to which they are thought to contribute protease resistance and membrane permeability. Of the six <em>N-</em>methyl amino acids that were AARS substrates (M1–M6, <a href="#pone-0000972-g006">Figure 6</a>) only <em>N-</em>methyl Asp (M3) and <em>N-</em>methyl His (M4) resulted in even moderate yields of translated peptide. One of the difficulties in testing the translation of these compounds was that many of them were contaminated with very small amounts of the unmethylated, natural amino acid. In order to remove this contamination an excess of the <em>N-</em>methyl amino acid was put into an AARS reaction. Since the contaminating natural amino acid is a more efficient substrate than the <em>N-</em>methyl analog and is at a much lower concentration, most of the natural amino acid became attached to the tRNA and was removed by EtOH precipitation <a href="#pone.0000972-Soutourina1">[54]</a>. The <em>N-</em>methyl AA containing EtOH layer was kept and evaporated. After repeating this procedure twice, formation of the natural amino acid-containing peptides was greatly suppressed. Unfortunately, the yields of <em>N</em>-methyl amino acid containing peptides remained low for M1, M2, M5 and M6, possibly due to the synthesis of limiting amounts of aminoacylated tRNA by the synthetase enzymes as evidenced by the poor ability of all of the <em>N</em>-methyl AAs to serve as competitors in our AARS assay.</p>


<h4>α-hydroxy carboxylic acids</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>Alpha-hydroxy acids are another interesting class of backbone analogs that are also the defining component of the depsipeptides, a subset of the natural product non-ribosomal peptides. Every substitution of an amide linkage for an ester linkage in a depsipeptide removes one hydrogen bond donor from the molecule, which may contribute significantly to membrane permeability <a href="#pone.0000972-Veber1">[55]</a>. The incorporation of α-hydroxy acids into peptides by ribosomal translation has generally been accomplished by chemical transformation of aminoacyl-tRNA after enzymatic charging <a href="#pone.0000972-Fahnestock1">[56]</a> or by chemical tRNA acylation <a href="#pone.0000972-Tan1">[15]</a>, <a href="#pone.0000972-Bain2">[37]</a>, <a href="#pone.0000972-Koh1">[57]</a>. Since the α-hydroxy acids lack an α-amino group, their potential activity as AARS substrates cannot be determined by our MALDI-MS assay. We therefore studied their ability to be translated directly, by replacing one amino acid at a time with the corresponding α-hydroxy acid using the appropriate templates (<em>a–e</em>). Of the 13 commercially available α-hydroxy analogs (Ala, Asp, Glu, Gly, His, Ile, Leu, Met, Phe, Ser, Val, Trp, Tyr), only one, α-hydroxy methionine (OM), was competent in translation. Interestingly, the peptide products of translation with OM using template e (<a href="#pone-0000972-g002">Figure 2</a>) showed defects in initiation including lack of formylation and initiation at internal codons (<a href="#pone-0000972-g007">Figure 7</a>). Since α-hydroxy methionine differs in mass from methionine by only +1, we desired corroborating chemical evidence for its incorporation. When incorporated into peptides, α-hydroxy acids introduce alkali-labile ester linkages <a href="#pone.0000972-Fahnestock1">[56]</a>. We examined the sensitivity to partial alkaline hydrolysis (<a href="#pone-0000972-g007">Figure 7</a>) of peptides designed to incorporate methionine between two His<sub>6</sub> tags (template f). Mild hydrolysis led to cleavage of the ester-containing peptides <a href="#pone.0000972-Fahnestock1">[56]</a>, <a href="#pone.0000972-Millward1">[58]</a> and gave peptide fragments of the expected masses. As expected, treatment of the methionine containing peptide showed no cleavage.</p>
<div class="figure" id="pone-0000972-g007"><div class="img"><a name="pone-0000972-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Ester hydrolysis shows that <em>α</em>-hydroxy methionine is incorporated.</span></strong></p><a id="article1.body1.sec2.sec10.fig1.caption1.p1" name="article1.body1.sec2.sec10.fig1.caption1.p1"></a><p>Top–Comparison of the peptide masses before and after treatment with NaOH which shows the expected hydrolyzed products labeled D, E and F. Bottom—Expected products from the hydrolysis of peptide containing an <em>α</em>-hydroxy methionine residue.</p>
<span>doi:10.1371/journal.pone.0000972.g007</span></div>

<h4>Fidelity of Analog Incorporation</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>The above experiments demonstrate that many different analogs can be incorporated into peptides at a single site. However, our ultimate goal is to prepare libraries of peptides composed primarily of unnatural AAs. The synthesis of such peptides requires that all analogs be mutually orthogonal, so that misincorporation events do not lead to a scrambling of the relationship between coding mRNA and translated peptide. Misincorporation at the level of the AARSs could occur either because an analog is charged by one or more additional synthetases, or by incorporation of contaminating natural amino acids in place of the analog. We therefore decided to begin by testing each analog in the presence of the other 19 natural amino acids and their synthetases, using an mRNA template coding for the incorporation of all 20 building blocks. This assay can detect many of the potential misincorporation events that would not be seen in the short templates used above for single-analog translation assays, since not all the competing amino acids and synthetases were present.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>In order to carry out this fidelity assay, we designed template <em>h</em>, which contained codons for each of the 20 amino acids and a C-terminal His-Tag. His analogs were tested in the similar template <em>i</em>, which contains a C-terminal FLAG tag. We tested all analogs that had &gt;25% translation efficiency as measured with templates <em>a–e</em>, using templates <em>h</em> or <em>i</em> in the presence of the other 19 natural amino acids and their synthetases. As above, we performed three experiments for each analog, a positive control with the cognate natural amino acid, a negative control with no amino acid added, and finally the experiment with the unnatural amino acid. Interestingly, in nearly all cases, the negative control experiment lacking a particular AA led to the synthesis of significant amounts of full-length peptide. The identity of these unexpected full length products varied depending on the AA withheld. For certain amino acids, peptides resulting from natural amino acid contamination (Ile, Glu, Asp) were observed. In other cases, specific misincorporation products were generated (e.g Trp→Phe, Tyr→His, and Leu→Gln/Lys, Val→Leu/Ile). Since the yield of these peptides was in most cases as high as the positive control containing 20 natural amino acids, we focused on the MALDI-MS analysis of the peptide products synthesized in the presence of an unnatural amino acid.</p>
<a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>Many of the analogs were incorporated efficiently in the presence of the other natural amino acids (<a href="#pone-0000972-g008">Figure 8</a>). Some analogs showed two (or more) peaks in the resulting peptide MS, one of the correct mass and one or more of other masses. These analogs were categorized into two subgroups based on the identity of the other peptides observed. In some examples (A15, N4 and N11), these other masses were the natural amino acid-containing peptides, suggesting that at the concentration tested (100 µM) these analogs could only partially compete with contaminating natural AAs. In other cases (C11 and A11) the undesired peptides corresponded to peptides that resulted from other misincorporation events, some of which could not be identified by mass. A third group of amino acids showed no incorporation under these conditions.</p>
<div class="figure" id="pone-0000972-g008"><div class="img"><a name="pone-0000972-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Analog incorporation efficiency and fidelity in the presence of the other 19 natural amino acids and their AARSs.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>Reactions used templates <em>h</em> and <em>i</em> and each reaction tested one unnatural amino acid analog. Analogs that yielded non-homogeneous peptide products at 0.1 mM, were tested again at 1 mM.</p>
<span>doi:10.1371/journal.pone.0000972.g008</span></div><a id="article1.body1.sec2.sec11.p4" name="article1.body1.sec2.sec11.p4"></a><p>The heterogeneous products observed with some of the analogs suggested that competition is occurring between the analog and one or more natural amino acids. We therefore re-tested all of analogs that gave heterogeneous products, as well as those that did not lead to measurable incorporation, at a 10-fold higher concentration (1 mM). At 1 mM all of the analogs except M4 showed improved incorporation into peptides including two (A1 and C11) that now gave entirely homogeneous products (<a href="#pone-0000972-g008">Figure 8</a>).</p>


<h4>Translation of a peptide containing multiple analogs</h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>The synthesis of peptides containing multiple unnatural amino acids requires a high fidelity of mRNA-directed incorporation, but in addition, the cumulative inefficiencies associated with each analog must not result in substantially diminished translation yields. We examined both of these requirements by translating peptides containing increasingly large numbers of different analogs. Prior this study, we were able to prepare peptides containing 12 different unnatural amino acids previously shown to incorporate efficiently <a href="#pone.0000972-Josephson1">[1]</a>. For the experiments described here, we focused as much as possible on new analogs that were uncovered with our AARS assay <a href="#pone.0000972-Hartman1">[11]</a>. We selected analogs that were efficiently incorporated in the 20 AA templates (<em>h</em> and <em>i</em>) at 1 mM as well as A24 which was chosen as an alternate to the well described Ile analog N16, and C6, which we had shown previously to be incorporated and is our only Asp analog <a href="#pone.0000972-Josephson1">[1]</a>. We were able to prepare peptides of the correct mass within error using 13 different unnatural amino acids using template <em>g</em> (M-&gt;N1, Y-&gt;A3, D-&gt;C6, E-&gt;A15, F-&gt;A10, T-&gt;P5, V-&gt;N12, W-&gt;A18, K-&gt;C7, I-&gt;A24, P-&gt;N23, R-&gt;C10, L-&gt;N18) (<a href="#pone-0000972-g009">Figure 9</a>). Five of these, A15, N12, A24, N23, and N18 are new analogs not previously incorporated translationally into peptides. Although the products of this translation reaction are not completely homogenous, the correct peptide is the dominant product. This level of purity should be sufficient for in vitro selections of unnatural peptides using mRNA display.</p>
<div class="figure" id="pone-0000972-g009"><div class="img"><a name="pone-0000972-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Incorporation of multiple unnatural AAs into a single peptide.</span></strong></p><a id="article1.body1.sec2.sec12.fig1.caption1.p1" name="article1.body1.sec2.sec12.fig1.caption1.p1"></a><p>(A) Mass spectrum of the peptides produced by in vitro translation of template <em>g</em> with all 15 natural amino acids. (B) Mass spectrum of the peptides produced from in vitro translation of template <em>g</em> with 13 unnatural and 2 natural amino acids. (C) Structure of the unnatural peptide that is the major product of in vitro translation of template <em>g</em> with unnatural amino acids.</p>
<span>doi:10.1371/journal.pone.0000972.g009</span></div><a id="article1.body1.sec2.sec12.p2" name="article1.body1.sec2.sec12.p2"></a><p>Unfortunately, attempts to simultaneously incorporate 15 different unnatural AAs (analogs for every natural AA except, Asn, Ala, Ser, Gly, Cys) did not result in peptides of the correct mass. We suspect that this failure was caused by incomplete AARS/AA orthogonality and by the presence of very small amounts of the highly efficient natural amino acids in the translation mix.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3>
<h4>Correlation between aminoacylation and translation assay</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A scatter plot comparing inhibition of the AARS reaction containing the cognate, natural amino acid and translation yield (single incorporation in templates <em>a</em>–<em>e</em>) is shown in <a href="#pone-0000972-g010">Figure 10</a>. Analogs that are efficient AARS competitors are located in the top half of the plot, and analogs that are translated efficiently appear on the right. From the plot it is clear that there is a correlation between competition with the natural AA for the AARS and translation efficiency—there are no highly efficient AARS competitors that translate poorly and thus would appear in the upper left quadrant of the plot. There are, on the other hand, a few analogs that translate well, but are inefficient AARS competitors and appear in lower right quadrant. These include A19, A21–A25, C9, C11, N11, N20, and P1. Several of these (A24, A25, C11, N11, N20, and P1) were among the more inefficient analogs with the 20 AA template experiments. Thus it seems reasonable to conclude that for these analogs the inefficiency is occurring during the AARS charging step. The synthetases (LeuRS and TrpRS) activating the remaining analogs in the lower right quadrant of the graph (A19, A21–23 and C9) have especially low <em>K<sub>m</sub></em>s (1.5 µM <a href="#pone.0000972-Mursinna1">[59]</a> and 12 µM <a href="#pone.0000972-Sever1">[60]</a> respectively) for their cognate AAs. This might allow them to compensate for their relative inefficiencies of charging since the analogs were tested in these particular translation experiments at 400 µM.</p>
<div class="figure" id="pone-0000972-g010"><div class="img"><a name="pone-0000972-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Comparison of % AARS competition and % translation yield for all of the analogs shown in <a href="#pone-0000972-t001">Table 1</a>.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The symbols are as follows: blue circles—side chain analogs, green squares—<em>α,α</em> disubstituted amino acids, black open triangles—<em>N-</em>methyl amino acids, red diamonds–<em>β</em>-amino acids. Translation yields for analogs that gave multiple peaks in the translation experiment were set to zero for this graph.</p>
<span>doi:10.1371/journal.pone.0000972.g010</span></div>

<h4>Specificity of the translation apparatus</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>As a step towards the synthesis of large libraries of NRP-like molecules using an in vitro translation system, we have studied the tolerance of the translation apparatus to a wide variety of amino acid analogs. The translation experiments are summarized by analog class in <a href="#pone-0000972-t002">Table 2</a>. Most side chain analogs are efficiently incorporated into peptides, but the backbone analogs are for the most part poorly translated. Of the 71 analogs tested, 41 side chain analogs are efficiently translated with high fidelity along with the other 19 AAs, and another 8 side chain and 2 backbone analogs should be efficiently translated if contamination of the translation system with natural, cognate AAs can be removed. The remaining analogs (mostly backbone), although AARS substrates, are poorly incorporated into peptides during translation.</p>
<div class="figure" id="pone-0000972-t002"><div class="img"><a name="pone-0000972-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Summary of translation results</span></strong></p><span>doi:10.1371/journal.pone.0000972.t002</span></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>Considering the side chain analogs as a class, poor charging onto tRNA appears to be the most common explanation for inefficient translation. This is supported by the correlation between competitive aminoacylation inhibition (a surrogate for aminoacylation activity) and translation efficiency (peptide yield and homogeneity) (<a href="#pone-0000972-g010">Figure 10</a>). For certain side chain analogs this explanation is certainly correct because they have been incorporated into peptides with good efficiency using stop codon suppression systems. For example, <em>p</em>-nitrophenylalanine (A14) has been efficiently incorporated into proteins in <em>E. coli</em> using an engineered AARS <a href="#pone.0000972-Tsao1">[61]</a>. Similarly β-cyclopentyl alanine (N19) <a href="#pone.0000972-Ellman1">[51]</a> and L-<em>C</em>-propargyl glycine (N9) <a href="#pone.0000972-Forster2">[3]</a> have been efficiently added using <em>E. coli</em> in vitro translation systems.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The correlation between charging efficiency and translatability suggests that enzyme engineering of the AARSs to better recognize certain analogs will be an important step to improve translation yield. Alternatively, enzymatic pre-charging of the tRNAs at high enzyme and amino acid concentrations prior to addition to the translation mixture may improve yields while minimizing product heterogeneity due to competition from efficiently acylated contaminating natural amino acids.</p>
<a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4"></a><p>For the backbone analogs, the relative importance of aminoacylation efficiency vs. later steps in translation varies with the type of backbone modification. In the case of the α-hydroxy acids, we cannot detect tRNA acylation directly. However, the ribosomes from <em>E. coli</em> are known to be able to generate ester linked products for the α-hydroxy analogs of Phe, Leu, Ile, Ala, <a href="#pone.0000972-Tan1">[15]</a>, <a href="#pone.0000972-Ellman1">[51]</a>, <a href="#pone.0000972-Koh1">[57]</a>, <a href="#pone.0000972-Fahnestock2">[62]</a> and now Met. Since only α-hydroxy methionine was incorporated into peptides in our assays of 13 α-hydroxy acids, either the synthetases or EF-Tu must discriminate against the other α-hydroxy acids. Given that many α-hydroxy acids are common metabolites (e.g. glycolate, lactate, glycerate, malate) while others may be generated at low levels in cells by hydrolysis of acyl-CoA intermediates or reduction of α-keto acids, it would not be surprising if many synthetases and perhaps EF-Tu have evolved the ability to discriminate against α-hydroxy acids. If this is the case, it will be necessary to engineer modified synthetases and perhaps EF-Tu to enable the efficient enzymatic incorporation of α-hydroxy acids into peptides. Similar synthetase engineering may be needed for the <em>N</em>-methyl amino acids, which, except for <em>N</em>-methyl His and Asp were poorly incorporated into peptides in our experiments and were poor AARS competitors as well. Previous experiments show that chemically generated <em>N</em>-methyl aminoacyl-tRNAs can be used to generate methylated peptides with reasonable efficiency <a href="#pone.0000972-Tan1">[15]</a>, <a href="#pone.0000972-Merryman1">[63]</a>, suggesting that the main block to incorporation is not at the level of EF-Tu or the ribosome. By default, poor efficiency of tRNA charging may be the limiting factor. In contrast, the β-amino acids are likely to be discriminated against by the active site of the peptidyl transferase center, possibly due to a higher pKa of the amino group <a href="#pone.0000972-Sando1">[53]</a>.</p>


<h4>Misincorporation/Contamination</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Our discovery that full length peptides resulting from misincorporation or contamination were formed when we used the 20 AA template <em>h</em> but withheld particular amino acids was surprising because we had only observed this in a few cases with our shorter templates. For example template <em>d</em> which contains both Gln and Glu codons often gives significant full length peptide when Glu is withheld; we expect that this results from contaminating Glu in the Gln stock. In the same vein, the Glu and Asp contamination products with the 20 AA template <em>h</em> most likely resulted from the Gln and Asn stocks. Consistent with this fact, lowering the concentration of Asn improved the incorporation efficiency of Asp analog C6 in this template (not shown). The misincorporation events we observed, however, require other explanations. We reasoned that misincorporation could occur either by mischarging of a non-cognate AA by a particular AARS (e.g. the Leu to Lys/Gln misincorporation could be caused by the formation Lys/Gln-tRNA<sup>Leu</sup> by LeuRS) or by readthrough of a non-cognate codon by a cognate AA-tRNA pair (e.g. Gln-tRNA<sup>Gln</sup> reading the Leu codon). Preliminary investigations into the Leu→Gln/Lys misincorporation suggest that it is codon misreading that is the cause, since changing the Leu codon from CUG to UUA led to misincorporation of Phe (standard codon UUU) instead of Gln (standard codon CAG). It is possible that these misincorporation events are artifacts of our in vitro translation system, or it may be that these miscoding events represent what happens in vivo when a particular AA-tRNA is depleted in cells lacking the stringent response <a href="#pone.0000972-OFarrell1">[64]</a>.</p>


<h4>Initiation</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Methionine analogs provide an opportunity to investigate the substrate specificity of the initiation steps of translation. Aminoacylated initiator tRNA is inspected twice prior to initiation, once by the methionyl-tRNA formyltransferase (MTF) and once by the initiation factor IF2 <a href="#pone.0000972-Mayer1">[65]</a>. Using template <em>e</em> which contains both initiator and elongator methionine codons, we found several methionine analogs that caused defects in initiation as evidenced by the production of peptides that lacked the N-terminal Met analog altogether, and appeared to either incorporate histidine in place of methionine, or to initiate at internal sites. Since our translation system lacks both the peptide deformylase and the N-terminal methionine aminopeptidase enzymes, the N-terminal residue cannot be removed posttranslationally, and therefore these peptide products must result from a failure of the initiation steps. Such translations were generally low yielding, but β-cyclopropyl alanine, N11, and α-hydroxy methionine (OM) are exceptions to this trend. Translation reactions with N11 were high yielding, but two peaks corresponding to the full-length formylated peptide and peptide lacking the N-terminal N11 residue were observed. Thus for this particular analog, the initiation machinery (most likely IF-2) appears to be more discriminating than the elongation complexes. For OM, no formylated full-length peptide was observed at all; initiation with non-formylated OM, however, proceeded with reasonable efficiency (<a href="#pone-0000972-g007">Figure 7</a>).</p>


<h4>Genetic code coverage</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We have examined the translation of analogs of 17 out of the 20 canonical amino acids–all except Gly, Ser, and Cys. For 15 of these 17, we have identified at least one analog that is translated efficiently. These amino acids correspond to 47 of the 64 codons, or reassignment of 73% of the genetic code. Cumulative inefficiencies have so far limited us to the simultaneous reassignment of 13 different amino acid codons (<a href="#pone-0000972-g009">Figure 9</a>). Further expansion of the monomer set that is compatible with enzymatic aminoacylation and ribosomal translation may be achievable in several different ways. Clearly, the continued generation of mutant AARSs with altered substrate specificities will greatly facilitate this program. An alternative strategy is to use a combination of enzymatic, chemo-enzymatic and chemical synthesis of pre-charged tRNAs. The use of pre-charged tRNAs greatly reduces the opportunities for mis-incorporation events during peptide synthesis, but also reduces the yield of translated peptide since aminoacylated tRNA is not continuously regenerated during the course of the translation reaction. However, this factor may be less important in the synthesis of large mRNA-display or ribosome-display libraries, where each ribosome is only required to synthesize one peptide and turnover is not needed <a href="#pone.0000972-Roberts1">[66]</a>–<a href="#pone.0000972-Mattheakis1">[68]</a>.</p>


<h4>Analog properties</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>It is our desire to prepare peptides using <em>in vitro</em> translation that are as drug-like as possible. To this end, we compared the side chain functional groups available to our system with a list of the most common functional groups in known drugs <a href="#pone.0000972-Bemis1">[69]</a>. Of the top 46 functional groups found in drugs, 9 are represented within the natural amino acid side chains (<a href="#pone-0000972-g011">Figure 11</a>). Using only the analogs that are &gt;75% efficient in our system, we can incorporate eleven more drug-like functional groups including methyl ethers, halogens, the dimethyl amino group, methyl esters and the <em>t</em>-butyl group (<a href="#pone-0000972-g011">Figure 11</a>). Drugs containing building blocks or functional groups present in our library include the HIV protease inhibitor Atazanavir which contains two <em>t</em>-butyl glycines (N13), Perindopril, an ACE inhibitor which includes norvaline (N4), and the antihypertensive drug Candoxatril which includes a cyclopentyl carboxylic acid moiety like that of α1. Other functional groups including acetates, nitriles, sulfonates, nitro groups, and ketones are targets for future unnatural translation building blocks.</p>
<div class="figure" id="pone-0000972-g011"><div class="img"><a name="pone-0000972-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>Functional groups found in known drugs and examples of drugs containing them.</span></strong></p><span>doi:10.1371/journal.pone.0000972.g011</span></div><a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Access to new functional groups that are found in current drugs is not enough to bestow drug-like qualities onto small molecules. These functional groups must be placed on a backbone or scaffold in precise orientations so that they can interact tightly with their targets. Coupling in vitro translation with mRNA-display <a href="#pone.0000972-Roberts1">[66]</a> or ribosome-display will allow selection from extremely large libraries to find the rare peptides that properly orient these drug-like functional groups. Rigidifying the library scaffold by cyclization <a href="#pone.0000972-Millward1">[58]</a> should also improve affinity and specificity. We hope that further work aimed at increasing the building block diversity accessible through ribosomal peptide translation will enable a new path to drug discovery.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p><em>Amino acids</em> Amino acid stocks were prepared at 10 mM concentration or their maximum solubility (always &gt;1 mM) and were stored at –20°C. For translation assays the pH of the stocks was adjusted with KOH to 7–7.5, and the stocks were filtered. Supplier information for the amino acids has been described previously <a href="#pone.0000972-Hartman1">[11]</a>. The α-hydroxy acids were purchased from Fluka (OF, OM, OS, OV), Sigma/Aldrich (OA, OP, OG, OH, OI, OL, OW, OY) or City Chemical (OE).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p><em>Protein purification</em> AARSs and translation factors were prepared as His-tagged constructs as previously described <a href="#pone.0000972-Josephson1">[1]</a>, <a href="#pone.0000972-Hartman1">[11]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p><em>Quantitative AARS Assays</em> Each assay contained 40 mM HEPES-KOH pH 7.4, 17 mM MgCl<sub>2</sub>, 45 mM KCl, 3.4 mM BME, 6 mM ATP, 6% glycerol, 0.02 units/mL yeast inorganic pyrophosphatase, 50–350 µM <em>E. coli</em> total tRNA (Roche), 0.09 mg/mL BSA, and various AARS, natural amino acid and analog concentrations (<a href="#pone-0000972-t003">Table 3</a>). For each analog inhibition experiment three different assays were performed (A, B, and C). These assays were then mixed together (Mixing protocols 1–3) to determine differences between the ratios of the natural AA [M]<sup>+</sup> and [M+15]<sup>+ </sup>peaks in the presence of the analog.</p>
<div class="figure" id="pone-0000972-t003"><div class="img"><a name="pone-0000972-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>AARS and AA concentrations in competition assays</span></strong></p><span>doi:10.1371/journal.pone.0000972.t003</span></div><a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p><strong>Assay A</strong>: Only the natural amino acid was included in this charging assay at the indicated concentration (<a href="#pone-0000972-t003">Table 3</a>). The assay was initiated by the addition of the appropriate AARS and was incubated for 30 min at 37°C. A solution of 3 M NaOAc pH 5.2 (0.1 vol., 2.5 µL) was added and the assay solution extracted with unbuffered phenol:CHCl<sub>3</sub>:isoamyl alcohol (25:24:1), and then CHCl<sub>3</sub>. A portion (45 µL) of the final aqueous layer was removed and precipitated with EtOH (3 vol.) and 3 M NaOAc pH 5.2 (0.1 vol.) at –20°C. The resulting pellet was washed twice with 70% EtOH (250 µL each), allowed to dry at room temperature (~5 min), and dissolved in 200 mM NaOAc pH 5.0 (12.5 µL). Half of the solution was frozen at –20°C, and the remaining half was added to water (3.75 µL), a freshly prepared solution of 4-formylphenoxypropyl triphenylphosphonium bromide <a href="#pone.0000972-Schweizer1">[70]</a> (each 69 mM in MeOH, 12.5 µL), and freshly prepared NaCNBH<sub>3</sub> (200 mM in 50 mM NaOAc pH 5.0, 2.5 µL). The solution was placed in a tumbler at 37°C (occasionally additional MeOH (1–2 µL) was required for complete dissolution). After 2 hours, 4.4 M NH<sub>4</sub>OAc pH 5.0 (0.2 vol., 5 µL) was added to quench and the assay was mixed as described below (Mixing protocol).</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p><strong>Assay B</strong>: Assay B followed the identical procedure, except the deuterated aldehyde 4-formylphenoxypropyl triphenylphosphonium-<em>d<sub>15</sub></em> bromide <a href="#pone.0000972-Hartman1">[11]</a> was added in place of the non-deuterated aldehyde.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p><strong>Assay C</strong>: Assay C included both the natural amino acid and the analog at the specified concentrations (<a href="#pone-0000972-t003">Table 3</a>) and used the deuterated aldehyde 4-formylphenoxypropyl triphenylphosphonium-<em>d<sub>15</sub></em> bromide. The contents of the three assays are summarized in <a href="#pone-0000972-t004">Table 4</a>.</p>
<div class="figure" id="pone-0000972-t004"><div class="img"><a name="pone-0000972-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000972" data-uri="info:doi/10.1371/journal.pone.0000972.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000972.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000972.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000972.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000972.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>AARS assay contents</span></strong></p><span>doi:10.1371/journal.pone.0000972.t004</span></div><a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p><strong>Mixing protocol #1</strong> (used to determine the ratio of the [M]<sup>+</sup> and [M+15]<sup>+</sup> peaks when no unnatural AA competitor was present). After the NH<sub>4</sub>OAc quenching of the reductive amination step, 25 µL of Assay A and 25 µL of Assay B were mixed together and the combined assays were precipitated with EtOH (3 vol.). The pellet was washed with 70% EtOH (2X) and 100% EtOH (2X) and was allowed to dry at room temperature. The dry pellet was then dissolved in 200 mM NH<sub>4</sub>OAc pH 5.0 (2.25 µL) and 1 U/µL Nuclease P1 in 200 mM NH<sub>4</sub>OAc pH 5.0 (~0.25 µL) was added. After 20 min at room temperature, an aliquot (1 µL) was removed and added to a saturated solution of CHCA (α-cyano-4-hydroxycinnamic acid) matrix in 1:1 MeCN:1% TFA (9 µL). An aliquot (1 µL) of the resulting suspension was added to a MALDI plate and analyzed.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8"></a><p><strong>Mixing protocol #2</strong> (used to determine the ratio of [M]<sup>+</sup> and [M+15]<sup>+</sup> that would be found if complete, 100%, inhibition by the analog was observed). For this protocol, assay A was used alone without mixing and treated exactly as described for Mixing protocol #1.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9"></a><p><strong>Mixing protocol #3</strong> (used to determine the ratio of [M]<sup>+</sup> and [M+15]<sup>+</sup> when the analog was present). For this protocol, Assays A and C (25 µL each) were mixed and treated exactly as in Mixing protocol #1.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10"></a><p><em>Data analysis</em> After software baseline subtraction, the ratio of peak heights for derivatized natural AA-AMP and <em>d</em><sub>15</sub>-AA-AMP was determined for the assay mixtures described above. To get the final % inhibition, the following equation was used % inh = (ratio A/C mix – ratio A mix)/(ratio A/B mix – ratio A mix)*100. For MetRS, ValRS, IleRS, GlnRS, GluRS, and AspRS analogs the % inhibition value was instead calculated with a calibration curve determined by mixing known ratios of d<sub>15</sub>-AA-AMP and H<sub>15</sub>-AA-AMP <a href="#pone.0000972-Hartman1">[11]</a>.</p>
<a id="article1.body1.sec4.p11" name="article1.body1.sec4.p11"></a><p><em>Translation Assay</em> Translation reactions (50 µL) were carried out as previously described <a href="#pone.0000972-Josephson1">[1]</a>. The mRNAs used for the translation reactions are described in the supporting information (<a href="#pone.0000972.s001">Figure S1</a>). For a particular template only the required natural or analog amino acids (400 µM each) and AARSs (0.1 µM MetRS, 0.3 µM LeuRS, 0.6 µM GluRS, 0.2 µM ProRS, 1.0 µM GlnRS, 1.0 µM HisRS, 0.3 µM PheRS or 0.25 µM PheRS A294G, 1.5 µM TrpRS, 0.2 µM SerRS, 0.2 µM IleRS, 0.4 µM ThrRS, 0.6 µM AsnRS, 0.6 µM AspRS, 0.5 µM TyrRS, 0.5 µM LysRS, 0.4 µM ArgRS, 0.2 µM ValRS, 0.2 µM AlaRS, 0.5 µM CysRS, 0.06 µM GlyRS) were added.</p>
<a id="article1.body1.sec4.p12" name="article1.body1.sec4.p12"></a><p><em>Multiple Analog template translation</em> Peptide synthesis was carried out with the following optimized amino acid concentrations: N1 (800 µM), A3 (1600 µM), C6 (800 µM), A15 (1600 µM), A10 (400 µM), P5 (800 µM), N12 (3200 µM), A18 (800 µM), C7 (400 µM), A24 (800 µM), N23 (800 µM), C10 (400 µM), N18 (1600 µM), Ala (400 µM), Gln (400 µM).</p>
<a id="article1.body1.sec4.p13" name="article1.body1.sec4.p13"></a><p><em>α-hydroxy methionine peptide cleavage assay</em> Translation reactions (100 µL) were performed as described <a href="#pone.0000972-Josephson1">[1]</a> using mRNA template <em>f</em>. After elution from the Ni-NTA resin (Qiagen) into 0.2% TFA (50 µL), the peptides were treated with 1 M NaOH (15 µL) for 60 min at 37°C. The assays were quenched with 10% TFA (6 µL) and 1 M HOAc (5 µL) and were purified by Zip-Tip C<sub>18</sub> chromatography as described above. The products were analyzed by MALDI-TOF MS.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000972"></div><a name="pone.0000972.s001" id="pone.0000972.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000972.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0000972.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors thank T. Ueda (University of Tokyo, Kashiwa, Japan) and P. Schimmel (Scripps Research Institute, La Jolla, CA) for gifts of AARS plasmids, J. K. Coward (Univ. of Michigan, Ann Arbor, MI) for gifts of the fluoroglutamates C3-C5, J. Taunton (University of California, San Francisco, CA) for compounds N25 and N26 and D. Tirrell (California Institute of Technology, Pasadena, CA) for compounds N1, N6, and A12.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JS MH KJ CL. Performed the experiments: MH KJ CL. Analyzed the data: JS MH KJ CL. Wrote the paper: JS MH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000972-Josephson1" id="pone.0000972-Josephson1"></a>Josephson K, Hartman MCT, Szostak JW (2005) Ribosomal synthesis of unnatural peptides. J Am Chem Soc  127: 11727–11735.  <ul class="find" data-citedArticleID="999308" data-doi="10.1021/ja0515809"><li><a href="http://dx.doi.org/10.1021/ja0515809" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+synthesis+of+unnatural+peptides." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+synthesis+of+unnatural+peptides.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000972-Forster1" id="pone.0000972-Forster1"></a>Forster AC, Weissbach H, Blacklow SC (2001) A simplified reconstitution of mRNA-directed peptide synthesis: activity of the epsilon enhancer and an unnatural amino acid. Anal Biochem  297: 60–70.  <ul class="find" data-citedArticleID="999284" data-doi="10.1006/abio.2001.5329"><li><a href="http://dx.doi.org/10.1006/abio.2001.5329" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simplified+reconstitution+of+mRNA-directed+peptide+synthesis%3A+activity+of+the+epsilon+enhancer+and+an+unnatural+amino+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simplified+reconstitution+of+mRNA-directed+peptide+synthesis%3A+activity+of+the+epsilon+enhancer+and+an+unnatural+amino+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000972-Forster2" id="pone.0000972-Forster2"></a>Forster AC, Tan Z, Nalam MN, Lin H, Qu H, et al.  (2003) Programming peptidomimetic syntheses by translating genetic codes designed de novo. Proc Natl Acad Sci U S A  100: 6353–6357.  <ul class="find" data-citedArticleID="999286" data-doi="10.1073/pnas.1132122100"><li><a href="http://dx.doi.org/10.1073/pnas.1132122100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Programming+peptidomimetic+syntheses+by+translating+genetic+codes+designed+de+novo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Programming+peptidomimetic+syntheses+by+translating+genetic+codes+designed+de+novo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000972-Shimizu1" id="pone.0000972-Shimizu1"></a>Shimizu Y, Inoue A, Tomari Y, Suzuki T, Yokogawa T, et al.  (2001) Cell-free translation reconstituted with purified components. Nat Biotechnol  19: 751–755.  <ul class="find" data-citedArticleID="999374" data-doi="10.1038/90802"><li><a href="http://dx.doi.org/10.1038/90802" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-free+translation+reconstituted+with+purified+components." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-free+translation+reconstituted+with+purified+components.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000972-Heckler1" id="pone.0000972-Heckler1"></a>Heckler TG, Chang LH, Zama Y, Naka T, Chorghade MS, et al.  (1984) T4 RNA ligase mediated preparation of novel chemically misacylated transfer-RNA Phes. Biochemistry  23: 1468–1473.  <ul class="find" data-citedArticleID="999296" data-doi="10.1021/bi00302a020"><li><a href="http://dx.doi.org/10.1021/bi00302a020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T4+RNA+ligase+mediated+preparation+of+novel+chemically+misacylated+transfer-RNA+Phes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T4+RNA+ligase+mediated+preparation+of+novel+chemically+misacylated+transfer-RNA+Phes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000972-Noren1" id="pone.0000972-Noren1"></a>Noren CJ, J. A-CS, Griffith MC, Schultz PG (1989) A general method for site-specific incorporation of unnatural amino acids into proteins. Science  244: 182–188.  <ul class="find" data-citedArticleID="999354" data-doi="10.1126/science.2649980"><li><a href="http://dx.doi.org/10.1126/science.2649980" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+method+for+site-specific+incorporation+of+unnatural+amino+acids+into+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+method+for+site-specific+incorporation+of+unnatural+amino+acids+into+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000972-Bain1" id="pone.0000972-Bain1"></a>Bain JD, Diala ES, Glabe CG, Dix TA, Chamberlin AR (1989) Biosynthetic site-specific incorporation of a non-natural amino-acid into a polypeptide. J Am Chem Soc  111: 8013–8014.  <ul class="find" data-citedArticleID="999258" data-doi="10.1021/ja00202a052"><li><a href="http://dx.doi.org/10.1021/ja00202a052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthetic+site-specific+incorporation+of+a+non-natural+amino-acid+into+a+polypeptide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthetic+site-specific+incorporation+of+a+non-natural+amino-acid+into+a+polypeptide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000972-Murakami1" id="pone.0000972-Murakami1"></a>Murakami H, Ohta M, Ashigai H, Suga H (2006) A highly flexible tRNA acylation method for non-natural polypeptide synthesis. Nat Methods  3: 357–359.  <ul class="find" data-citedArticleID="999344" data-doi="10.1038/nmeth877"><li><a href="http://dx.doi.org/10.1038/nmeth877" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+highly+flexible+tRNA+acylation+method+for+non-natural+polypeptide+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+highly+flexible+tRNA+acylation+method+for+non-natural+polypeptide+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000972-Ninomiya1" id="pone.0000972-Ninomiya1"></a>Ninomiya K, Minohata T, Nishimura M, Sisido M (2004) In situ chemical aminoacylation with amino acid thioesters linked to a peptide nucleic acid. J Am Chem Soc  126: 15984–15989.  <ul class="find" data-citedArticleID="999352" data-doi="10.1021/ja048200o"><li><a href="http://dx.doi.org/10.1021/ja048200o" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+situ+chemical+aminoacylation+with+amino+acid+thioesters+linked+to+a+peptide+nucleic+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+situ+chemical+aminoacylation+with+amino+acid+thioesters+linked+to+a+peptide+nucleic+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000972-Wang1" id="pone.0000972-Wang1"></a>Wang L, Xie J, Schultz PG (2006) Expanding the genetic code. Annu Rev Biophys Biomol Struct  35: 225–249.  <ul class="find" data-citedArticleID="999390" data-doi="10.1146/annurev.biophys.35.101105.121507"><li><a href="http://dx.doi.org/10.1146/annurev.biophys.35.101105.121507" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expanding+the+genetic+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expanding+the+genetic+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000972-Hartman1" id="pone.0000972-Hartman1"></a>Hartman MCT, Josephson K, Szostak JW (2006) Enzymatic aminoacylation of tRNA with unnatural amino acids. Proc Natl Acad Sci U S A  103: 4356–4361.  <ul class="find" data-citedArticleID="999294" data-doi="10.1073/pnas.0509219103"><li><a href="http://dx.doi.org/10.1073/pnas.0509219103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enzymatic+aminoacylation+of+tRNA+with+unnatural+amino+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enzymatic+aminoacylation+of+tRNA+with+unnatural+amino+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000972-Hohsaka1" id="pone.0000972-Hohsaka1"></a>Hohsaka T, Sato K, Sisido M, Takai K, Yokoyama S (1993) Adaptability of nonnatural aromatic amino acids to the active center of the <em>E. coli</em> ribosomal A site. FEBS Lett  335: 47–50.  <ul class="find" data-citedArticleID="999300" data-doi="10.1016/0014-5793(93)80436-x"><li><a href="http://dx.doi.org/10.1016/0014-5793(93)80436-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptability+of+nonnatural+aromatic+amino+acids+to+the+active+center+of+the+E.+coli+ribosomal+A+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptability+of+nonnatural+aromatic+amino+acids+to+the+active+center+of+the+E.+coli+ribosomal+A+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000972-Hohsaka2" id="pone.0000972-Hohsaka2"></a>Hohsaka T, Kajihara D, Ashizuka Y, Murakami H, Sisido M (1999) Efficient incorporation of nonnatural amino acids with large aromatic groups into streptavidin in in vitro protein synthesizing systems. J Am Chem Soc  121: 34–40.  <ul class="find" data-citedArticleID="999302" data-doi="10.1021/ja9813109"><li><a href="http://dx.doi.org/10.1021/ja9813109" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+incorporation+of+nonnatural+amino+acids+with+large+aromatic+groups+into+streptavidin+in+in+vitro+protein+synthesizing+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+incorporation+of+nonnatural+amino+acids+with+large+aromatic+groups+into+streptavidin+in+in+vitro+protein+synthesizing+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000972-Asahara1" id="pone.0000972-Asahara1"></a>Asahara H, Uhlenbeck OC (2005) Predicting the binding affinities of misacylated tRNAs for <em>Thermus thermophilus</em> EF-Tu.GTP. Biochemistry  44: 11254–11261.  <ul class="find" data-citedArticleID="999256" data-doi="10.1021/bi050204y"><li><a href="http://dx.doi.org/10.1021/bi050204y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+the+binding+affinities+of+misacylated+tRNAs+for+Thermus+thermophilus+EF-Tu.GTP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+the+binding+affinities+of+misacylated+tRNAs+for+Thermus+thermophilus+EF-Tu.GTP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000972-Tan1" id="pone.0000972-Tan1"></a>Tan Z, Forster AC, Blacklow SC, Cornish VW (2004) Amino acid backbone specificity of the <em>Escherichia coli</em> translation machinery. J Am Chem Soc  126: 12752–12753.  <ul class="find" data-citedArticleID="999382" data-doi="10.1021/ja0472174"><li><a href="http://dx.doi.org/10.1021/ja0472174" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+backbone+specificity+of+the+Escherichia+coli+translation+machinery." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+backbone+specificity+of+the+Escherichia+coli+translation+machinery.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000972-Martinez1" id="pone.0000972-Martinez1"></a>Martinez JS, Zhang GP, Holt PD, Jung HT, Carrano CJ, et al.  (2000) Self-assembling amphiphilic siderophores from marine bacteria. Science  287: 1245–1247.  <ul class="find" data-citedArticleID="999328" data-doi="10.1126/science.287.5456.1245"><li><a href="http://dx.doi.org/10.1126/science.287.5456.1245" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Self-assembling+amphiphilic+siderophores+from+marine+bacteria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Self-assembling+amphiphilic+siderophores+from+marine+bacteria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000972-Kilz1" id="pone.0000972-Kilz1"></a>Kilz S, Lenz C, Fuchs R, Budzikiewicz H (1999) A fast screening method for the identification of siderophores from fluorescent <em>Pseudomonas spp.</em> by liquid chromatography/electrospray mass spectrometry. J Mass Spectrom  34: 281–290.  <ul class="find" data-citedArticleID="999314" data-doi="10.1002/(sici)1096-9888(199904)34:4<281::aid-jms750&gt;3.0.co;2-m"><li><a href="http://dx.doi.org/10.1002/(sici)1096-9888(199904)34:4<281::aid-jms750&gt;3.0.co;2-m" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+fast+screening+method+for+the+identification+of+siderophores+from+fluorescent+Pseudomonas+spp.+by+liquid+chromatography%2Felectrospray+mass+spectrometry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+fast+screening+method+for+the+identification+of+siderophores+from+fluorescent+Pseudomonas+spp.+by+liquid+chromatography%2Felectrospray+mass+spectrometry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000972-Lea1" id="pone.0000972-Lea1"></a>Lea PJ, Fowden L (1973) Amino acid substrate specificity of asparaginyl-,aspartyl-, and glutaminyl-tRNA synthetase isolated from higher plants. Phytochemistry  12: 1903–1916.  <ul class="find" data-citedArticleID="999320" data-doi="10.1016/s0031-9422(00)91509-7"><li><a href="http://dx.doi.org/10.1016/s0031-9422(00)91509-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Amino+acid+substrate+specificity+of+asparaginyl-%2Caspartyl-%2C+and+glutaminyl-tRNA+synthetase+isolated+from+higher+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Amino+acid+substrate+specificity+of+asparaginyl-%2Caspartyl-%2C+and+glutaminyl-tRNA+synthetase+isolated+from+higher+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000972-Levengood1" id="pone.0000972-Levengood1"></a>Levengood J, Ataide SF, Roy H, Ibba M (2004) Divergence in noncognate amino acid recognition between class I and class II lysyl-tRNA synthetases. J Biol Chem  279: 17707–17714.  <ul class="find" data-citedArticleID="999324" data-doi="10.1074/jbc.m313665200"><li><a href="http://dx.doi.org/10.1074/jbc.m313665200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergence+in+noncognate+amino+acid+recognition+between+class+I+and+class+II+lysyl-tRNA+synthetases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergence+in+noncognate+amino+acid+recognition+between+class+I+and+class+II+lysyl-tRNA+synthetases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000972-Christner1" id="pone.0000972-Christner1"></a>Christner PJ, Rosenbloom J (1971) Effects of incorporation of <em>trans</em>-4,5-dehydrolysine on collagen biosynthesis and extrusion in embryonic chick tibiae. J Biol Chem  246: 7551–7556.  <ul class="find" data-citedArticleID="999272"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+incorporation+of+trans-4%2C5-dehydrolysine+on+collagen+biosynthesis+and+extrusion+in+embryonic+chick+tibiae.&amp;auth=&amp;atitle=Effects+of+incorporation+of+trans-4%2C5-dehydrolysine+on+collagen+biosynthesis+and+extrusion+in+embryonic+chick+tibiae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+incorporation+of+trans-4%2C5-dehydrolysine+on+collagen+biosynthesis+and+extrusion+in+embryonic+chick+tibiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+incorporation+of+trans-4%2C5-dehydrolysine+on+collagen+biosynthesis+and+extrusion+in+embryonic+chick+tibiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000972-Blount1" id="pone.0000972-Blount1"></a>Blount KF, Wang JX, Lim J, Sudarsan N, Breaker RR (2007) Antibacterial lysine analogs that target lysine riboswitches. Nat Chem Biol  3: 44–49.  <ul class="find" data-citedArticleID="999266" data-doi="10.1038/nchembio842"><li><a href="http://dx.doi.org/10.1038/nchembio842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antibacterial+lysine+analogs+that+target+lysine+riboswitches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antibacterial+lysine+analogs+that+target+lysine+riboswitches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000972-Lemeignan1" id="pone.0000972-Lemeignan1"></a>Lemeignan B, Sonigo P, Marliere P (1993) Phenotypic suppression by incorporation of an alien amino acid. J Mol Biol  231: 161–166.  <ul class="find" data-citedArticleID="999322" data-doi="10.1006/jmbi.1993.1269"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1269" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phenotypic+suppression+by+incorporation+of+an+alien+amino+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phenotypic+suppression+by+incorporation+of+an+alien+amino+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000972-Sabina1" id="pone.0000972-Sabina1"></a>Sabina J, Dover N, Templeton LJ, Smulski DR, Soll D, et al.  (2003) Interfering with different steps of protein synthesis explored by transcriptional profiling of Escherichia coli K-12. J Bacteriol  185: 6158–6170.  <ul class="find" data-citedArticleID="999364" data-doi="10.1128/jb.185.20.6158-6170.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.20.6158-6170.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interfering+with+different+steps+of+protein+synthesis+explored+by+transcriptional+profiling+of+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interfering+with+different+steps+of+protein+synthesis+explored+by+transcriptional+profiling+of+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000972-Hortin1" id="pone.0000972-Hortin1"></a>Hortin G, Boime I (1983) Applications of amino acid analogs for studying co and posttranslational modifications of proteins. Methods Enzymol  96: 777–784.  <ul class="find" data-citedArticleID="999304" data-doi="10.1016/s0076-6879(83)96065-2"><li><a href="http://dx.doi.org/10.1016/s0076-6879(83)96065-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applications+of+amino+acid+analogs+for+studying+co+and+posttranslational+modifications+of+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applications+of+amino+acid+analogs+for+studying+co+and+posttranslational+modifications+of+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000972-Murakami2" id="pone.0000972-Murakami2"></a>Murakami M, Sun Q, Ishida K, Matsuda H, Okino T, et al.  (1997) Microviridins, elastase inhibitors from the cyanobacterium <em>Nostoc Minutum</em> (NIES-26). Photochemistry  45: 1197–1202.  <ul class="find" data-citedArticleID="999346" data-doi="10.1016/s0031-9422(97)00131-3"><li><a href="http://dx.doi.org/10.1016/s0031-9422(97)00131-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microviridins%2C+elastase+inhibitors+from+the+cyanobacterium+Nostoc+Minutum+%28NIES-26%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microviridins%2C+elastase+inhibitors+from+the+cyanobacterium+Nostoc+Minutum+%28NIES-26%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000972-Neal1" id="pone.0000972-Neal1"></a>Neal AL, Libman L, Smulson ME (1968) Influence of γ-glutamyl hydrazide on protein synthesis of <em>Pseudomonas aeruginosa</em>. Arch Biochem Biophys  127: 426–428.  <ul class="find" data-citedArticleID="999350" data-doi="10.1016/0003-9861(68)90246-4"><li><a href="http://dx.doi.org/10.1016/0003-9861(68)90246-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+%CE%B3-glutamyl+hydrazide+on+protein+synthesis+of+Pseudomonas+aeruginosa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+%CE%B3-glutamyl+hydrazide+on+protein+synthesis+of+Pseudomonas+aeruginosa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000972-Ahluwalia1" id="pone.0000972-Ahluwalia1"></a>Ahluwalia GS, Grem JL, Hao Z, Cooney DA (1990) Metabolism and action of amino acid analog anti-cancer agents. Pharmacol Ther  46: 243–271.  <ul class="find" data-citedArticleID="999254" data-doi="10.1016/0163-7258(90)90094-i"><li><a href="http://dx.doi.org/10.1016/0163-7258(90)90094-i" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolism+and+action+of+amino+acid+analog+anti-cancer+agents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolism+and+action+of+amino+acid+analog+anti-cancer+agents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000972-Ikeda1" id="pone.0000972-Ikeda1"></a>Ikeda Y, Kawahara S, Taki M, Kuno A, Hasegawa T, et al.  (2003) Synthesis of a novel histidine analogue and its efficient incorporation into a protein in vivo. Protein Eng  16: 699–706.  <ul class="find" data-citedArticleID="999306" data-doi="10.1093/protein/gzg084"><li><a href="http://dx.doi.org/10.1093/protein/gzg084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synthesis+of+a+novel+histidine+analogue+and+its+efficient+incorporation+into+a+protein+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synthesis+of+a+novel+histidine+analogue+and+its+efficient+incorporation+into+a+protein+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000972-Schlesinger1" id="pone.0000972-Schlesinger1"></a>Schlesinger S, Schlesinger MJ (1967) The effect of amino acid analogues on alkaline phosphatase formation in <em>Escherichia coli</em> K-12. I. Substitution of triazolealanine for histidine. J Biol Chem  242: 3369–3372.  <ul class="find" data-citedArticleID="999368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+effect+of+amino+acid+analogues+on+alkaline+phosphatase+formation+in+Escherichia+coli+K-12.+I.+Substitution+of+triazolealanine+for+histidine.&amp;auth=&amp;atitle=The+effect+of+amino+acid+analogues+on+alkaline+phosphatase+formation+in+Escherichia+coli+K-12.+I.+Substitution+of+triazolealanine+for+histidine." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+amino+acid+analogues+on+alkaline+phosphatase+formation+in+Escherichia+coli+K-12.+I.+Substitution+of+triazolealanine+for+histidine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+amino+acid+analogues+on+alkaline+phosphatase+formation+in+Escherichia+coli+K-12.+I.+Substitution+of+triazolealanine+for+histidine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000972-Beiboer1" id="pone.0000972-Beiboer1"></a>Beiboer SH, van den Berg B, Dekker N, Cox RC, Verheij HM (1996) Incorporation of an unnatural amino acid in the active site of porcine pancreatic phospholipase A2. Substitution of histidine by 1,2,4-triazole-3-alanine yields an enzyme with high activity at acidic pH. Protein Eng  9: 345–352.  <ul class="find" data-citedArticleID="999262" data-doi="10.1093/protein/9.4.345"><li><a href="http://dx.doi.org/10.1093/protein/9.4.345" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporation+of+an+unnatural+amino+acid+in+the+active+site+of+porcine+pancreatic+phospholipase+A2.+Substitution+of+histidine+by+1%2C2%2C4-triazole-3-alanine+yields+an+enzyme+with+high+activity+at+acidic+pH." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporation+of+an+unnatural+amino+acid+in+the+active+site+of+porcine+pancreatic+phospholipase+A2.+Substitution+of+histidine+by+1%2C2%2C4-triazole-3-alanine+yields+an+enzyme+with+high+activity+at+acidic+pH.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000972-Minks1" id="pone.0000972-Minks1"></a>Minks C, Alefelder S, Moroder L, Huber R, Budisa N (2000) Towards new protein engineering: in vivo building and folding of protein shuttles for drug delivery and targeting by the selective pressure incorporation (SPI) method. Tetrahedron  56: 9431–9442.  <ul class="find" data-citedArticleID="999340" data-doi="10.1016/s0040-4020(00)00827-9"><li><a href="http://dx.doi.org/10.1016/s0040-4020(00)00827-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+new+protein+engineering%3A+in+vivo+building+and+folding+of+protein+shuttles+for+drug+delivery+and+targeting+by+the+selective+pressure+incorporation+%28SPI%29+method." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+new+protein+engineering%3A+in+vivo+building+and+folding+of+protein+shuttles+for+drug+delivery+and+targeting+by+the+selective+pressure+incorporation+%28SPI%29+method.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000972-Kirshenbaum1" id="pone.0000972-Kirshenbaum1"></a>Kirshenbaum K, Carrico IS, Tirrell DA (2002) Biosynthesis of proteins incorporating a versatile set of phenylalanine analogues. Chembiochem  3: 235–237.  <ul class="find" data-citedArticleID="999316" data-doi="10.1002/1439-7633(20020301)3:2/3<235::aid-cbic235&gt;3.0.co;2-7"><li><a href="http://dx.doi.org/10.1002/1439-7633(20020301)3:2/3<235::aid-cbic235&gt;3.0.co;2-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+of+proteins+incorporating+a+versatile+set+of+phenylalanine+analogues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+of+proteins+incorporating+a+versatile+set+of+phenylalanine+analogues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000972-Budisa1" id="pone.0000972-Budisa1"></a>Budisa N, Alefelder S, Bae JH, Golbik R, Minks C, et al.  (2001) Proteins with beta-(thienopyrrolyl)alanines as alternative chromophores and pharmaceutically active amino acids. Protein Sci  10: 1281–1292.  <ul class="find" data-citedArticleID="999268" data-doi="10.1110/ps.51601"><li><a href="http://dx.doi.org/10.1110/ps.51601" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteins+with+beta-%28thienopyrrolyl%29alanines+as+alternative+chromophores+and+pharmaceutically+active+amino+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteins+with+beta-%28thienopyrrolyl%29alanines+as+alternative+chromophores+and+pharmaceutically+active+amino+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000972-Hall1" id="pone.0000972-Hall1"></a>Hall LE, Hegeman GD, Bosin TR (1974) Incorporation of tryptophan and its benzo(b)thiophene, 1-methylindole, and indene analogs into protein of Escherichia coli. Res Commun Chem Pathol Pharmacol  9: 145–153.  <ul class="find" data-citedArticleID="999290"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Incorporation+of+tryptophan+and+its+benzo%28b%29thiophene%2C+1-methylindole%2C+and+indene+analogs+into+protein+of+Escherichia+coli.&amp;auth=&amp;atitle=Incorporation+of+tryptophan+and+its+benzo%28b%29thiophene%2C+1-methylindole%2C+and+indene+analogs+into+protein+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporation+of+tryptophan+and+its+benzo%28b%29thiophene%2C+1-methylindole%2C+and+indene+analogs+into+protein+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporation+of+tryptophan+and+its+benzo%28b%29thiophene%2C+1-methylindole%2C+and+indene+analogs+into+protein+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000972-Cleland1" id="pone.0000972-Cleland1"></a>Cleland TA (1996) Inhibitory glutamate receptor channels. Mol Neurobiol  13: 97–136.  <ul class="find" data-citedArticleID="999274" data-doi="10.1007/bf02740637"><li><a href="http://dx.doi.org/10.1007/bf02740637" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibitory+glutamate+receptor+channels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibitory+glutamate+receptor+channels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000972-McLennan1" id="pone.0000972-McLennan1"></a>McLennan H (1983) Receptors for the excitatory amino acids in the mammalian central nervous system. Prog Neurobiol  20: 251–271.  <ul class="find" data-citedArticleID="999334" data-doi="10.1016/0301-0082(83)90004-7"><li><a href="http://dx.doi.org/10.1016/0301-0082(83)90004-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Receptors+for+the+excitatory+amino+acids+in+the+mammalian+central+nervous+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Receptors+for+the+excitatory+amino+acids+in+the+mammalian+central+nervous+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000972-Bain2" id="pone.0000972-Bain2"></a>Bain JD, Wacker DA, Kuo EE, Chamberlin AR (1991) Site-Specific Incorporation of non-natural residues into peptides: Effect of residue structure on suppression and translation efficiencies. Tetrahedron  47: 2389–2400.  <ul class="find" data-citedArticleID="999260" data-doi="10.1016/s0040-4020(01)81776-2"><li><a href="http://dx.doi.org/10.1016/s0040-4020(01)81776-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-Specific+Incorporation+of+non-natural+residues+into+peptides%3A+Effect+of+residue+structure+on+suppression+and+translation+efficiencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-Specific+Incorporation+of+non-natural+residues+into+peptides%3A+Effect+of+residue+structure+on+suppression+and+translation+efficiencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000972-Spizek1" id="pone.0000972-Spizek1"></a>Spizek J, Janacek J (1969) The effect of ethionine on the synthesis of beta galactosidase: formation of an immunologically cross-reacting protein. Biochem Biophys Res Commun  34: 17–21.  <ul class="find" data-citedArticleID="999378" data-doi="10.1016/0006-291x(69)90521-x"><li><a href="http://dx.doi.org/10.1016/0006-291x(69)90521-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+ethionine+on+the+synthesis+of+beta+galactosidase%3A+formation+of+an+immunologically+cross-reacting+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+ethionine+on+the+synthesis+of+beta+galactosidase%3A+formation+of+an+immunologically+cross-reacting+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000972-vanHest1" id="pone.0000972-vanHest1"></a>van Hest JCM, Kiick KL, Tirrell DA (2000) Efficient incorporation of unsaturated methionine analogues into proteins in vivo. J Am Chem Soc  122: 1282–1288.  <ul class="find" data-citedArticleID="999392" data-doi="10.1021/ja992749j"><li><a href="http://dx.doi.org/10.1021/ja992749j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+incorporation+of+unsaturated+methionine+analogues+into+proteins+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+incorporation+of+unsaturated+methionine+analogues+into+proteins+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000972-Kiick1" id="pone.0000972-Kiick1"></a>Kiick KL, Saxon E, Tirrell DA, Bertozzi CR (2002) Incorporation of azides into recombinant proteins for chemoselective modification by the Staudinger ligation. Proc Natl Acad Sci U S A  99: 19–24.  <ul class="find" data-citedArticleID="999310" data-doi="10.1073/pnas.012583299"><li><a href="http://dx.doi.org/10.1073/pnas.012583299" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Incorporation+of+azides+into+recombinant+proteins+for+chemoselective+modification+by+the+Staudinger+ligation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Incorporation+of+azides+into+recombinant+proteins+for+chemoselective+modification+by+the+Staudinger+ligation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000972-Kiick2" id="pone.0000972-Kiick2"></a>Kiick KL, Weberskirch R, Tirrell DA (2001) Identification of an expanded set of translationally active methionine analogues in <em>Escherichia coli</em>. FEBS Lett  502: 25–30.  <ul class="find" data-citedArticleID="999312" data-doi="10.1016/s0014-5793(01)02657-6"><li><a href="http://dx.doi.org/10.1016/s0014-5793(01)02657-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+an+expanded+set+of+translationally+active+methionine+analogues+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+an+expanded+set+of+translationally+active+methionine+analogues+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000972-Tang1" id="pone.0000972-Tang1"></a>Tang Y, Tirrell DA (2002) Attenuation of the editing activity of the <em>Escerichia coli</em> leucyl-tRNA synthetase allows incorporation of novel amino acids into proteins in vivo. Biochemistry  41: 10635–10645.  <ul class="find" data-citedArticleID="999384" data-doi="10.1021/bi026130x"><li><a href="http://dx.doi.org/10.1021/bi026130x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Attenuation+of+the+editing+activity+of+the+Escerichia+coli+leucyl-tRNA+synthetase+allows+incorporation+of+novel+amino+acids+into+proteins+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Attenuation+of+the+editing+activity+of+the+Escerichia+coli+leucyl-tRNA+synthetase+allows+incorporation+of+novel+amino+acids+into+proteins+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000972-Lincecum1" id="pone.0000972-Lincecum1"></a>Lincecum TL Jr, Tukalo M, Yaremchuk A, Mursinna RS, Williams AM, et al.  (2003) Structural and mechanistic basis of pre- and posttransfer editing by leucyl-tRNA synthetase. Mol Cell  11: 951–963.  <ul class="find" data-citedArticleID="999326" data-doi="10.1016/s1097-2765(03)00098-4"><li><a href="http://dx.doi.org/10.1016/s1097-2765(03)00098-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+and+mechanistic+basis+of+pre-+and+posttransfer+editing+by+leucyl-tRNA+synthetase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+and+mechanistic+basis+of+pre-+and+posttransfer+editing+by+leucyl-tRNA+synthetase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000972-Suchanek1" id="pone.0000972-Suchanek1"></a>Suchanek M, Radzikowska A, Thiele C (2005) Photo-leucine and photo-methionine allow identification of protein-protein interactions in living cells. Nat Methods  2: 261–267.  <ul class="find" data-citedArticleID="999380" data-doi="10.1038/nmeth752"><li><a href="http://dx.doi.org/10.1038/nmeth752" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Photo-leucine+and+photo-methionine+allow+identification+of+protein-protein+interactions+in+living+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Photo-leucine+and+photo-methionine+allow+identification+of+protein-protein+interactions+in+living+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000972-Busiello1" id="pone.0000972-Busiello1"></a>Busiello V, di Girolamo M, Cini C, De Marco C (1979) Action of thiazolidine-2-carboxylic acid, a proline analog, on protein synthesizing systems. Biochim Biophys Acta  564: 311–321.  <ul class="find" data-citedArticleID="999270" data-doi="10.1016/0005-2787(79)90228-4"><li><a href="http://dx.doi.org/10.1016/0005-2787(79)90228-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Action+of+thiazolidine-2-carboxylic+acid%2C+a+proline+analog%2C+on+protein+synthesizing+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Action+of+thiazolidine-2-carboxylic+acid%2C+a+proline+analog%2C+on+protein+synthesizing+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000972-Papas1" id="pone.0000972-Papas1"></a>Papas TS, Mehler AH (1970) Analysis of the amino acid binding to the proline transfer ribonucleic acid synthetase of <em>Escherichia coli</em>. J Biol Chem  245: 1588–1595.  <ul class="find" data-citedArticleID="999358"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+the+amino+acid+binding+to+the+proline+transfer+ribonucleic+acid+synthetase+of+Escherichia+coli.&amp;auth=&amp;atitle=Analysis+of+the+amino+acid+binding+to+the+proline+transfer+ribonucleic+acid+synthetase+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+amino+acid+binding+to+the+proline+transfer+ribonucleic+acid+synthetase+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+amino+acid+binding+to+the+proline+transfer+ribonucleic+acid+synthetase+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000972-Morris1" id="pone.0000972-Morris1"></a>Morris H, Schlesinger MJ (1972) Effects of proline analogues on the formation of alkaline phosphatase in <em>Escherichia coli</em>. J Bacteriol  111: 203–210.  <ul class="find" data-citedArticleID="999342"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effects+of+proline+analogues+on+the+formation+of+alkaline+phosphatase+in+Escherichia+coli.&amp;auth=&amp;atitle=Effects+of+proline+analogues+on+the+formation+of+alkaline+phosphatase+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effects+of+proline+analogues+on+the+formation+of+alkaline+phosphatase+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effects+of+proline+analogues+on+the+formation+of+alkaline+phosphatase+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000972-Fowden1" id="pone.0000972-Fowden1"></a>Fowden L, Neale S, Tristram H (1963) Effect of 3,4-dehydro-DL-proline on growth and protein synthesis. Nature  199: 35–38.  <ul class="find" data-citedArticleID="999288" data-doi="10.1038/199035a0"><li><a href="http://dx.doi.org/10.1038/199035a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+3%2C4-dehydro-DL-proline+on+growth+and+protein+synthesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+3%2C4-dehydro-DL-proline+on+growth+and+protein+synthesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000972-Deming1" id="pone.0000972-Deming1"></a>Deming TJ, Fournier MJ, Mason TL, Tirrell DA (1996) Structural modification of a periodic polypeptide through biosynthetic replacement of proline with azetidine-2-carboxylic acid. Macromolecules  29: 1442–1444.  <ul class="find" data-citedArticleID="999276" data-doi="10.1021/ma9510698"><li><a href="http://dx.doi.org/10.1021/ma9510698" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+modification+of+a+periodic+polypeptide+through+biosynthetic+replacement+of+proline+with+azetidine-2-carboxylic+acid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+modification+of+a+periodic+polypeptide+through+biosynthetic+replacement+of+proline+with+azetidine-2-carboxylic+acid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000972-Roesser1" id="pone.0000972-Roesser1"></a>Roesser JR, Xu C, Payne RC, Surratt CK, Hecht SM (1989) Preparation of misacylated aminoacyl-tRNA(Phe)'s useful as probes of the ribosomal acceptor site. Biochemistry  28: 5185–5195.  <ul class="find" data-citedArticleID="999362" data-doi="10.1021/bi00438a041"><li><a href="http://dx.doi.org/10.1021/bi00438a041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Preparation+of+misacylated+aminoacyl-tRNA%28Phe%29%27s+useful+as+probes+of+the+ribosomal+acceptor+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Preparation+of+misacylated+aminoacyl-tRNA%28Phe%29%27s+useful+as+probes+of+the+ribosomal+acceptor+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000972-Ellman1" id="pone.0000972-Ellman1"></a>Ellman JA, Mendel D, Schultz PG (1992) Site-specific incorporation of novel backbone structures into proteins. Science  255: 197–200.  <ul class="find" data-citedArticleID="999278" data-doi="10.1126/science.1553546"><li><a href="http://dx.doi.org/10.1126/science.1553546" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-specific+incorporation+of+novel+backbone+structures+into+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-specific+incorporation+of+novel+backbone+structures+into+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000972-Heckler2" id="pone.0000972-Heckler2"></a>Heckler TG, Roesser JR, Xu C, Chang PI, Hecht SM (1988) Ribosomal binding and dipeptide formation by misacylated tRNA(Phe)'s. Biochemistry  27: 7254–7262.  <ul class="find" data-citedArticleID="999298" data-doi="10.1021/bi00419a012"><li><a href="http://dx.doi.org/10.1021/bi00419a012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosomal+binding+and+dipeptide+formation+by+misacylated+tRNA%28Phe%29%27s." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosomal+binding+and+dipeptide+formation+by+misacylated+tRNA%28Phe%29%27s.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000972-Sando1" id="pone.0000972-Sando1"></a>Sando S, Abe K, Sato N, Shibata T, Mizusawa K, et al.  (2007) Unexpected preference of the <em>E. coli</em> translation system for the ester bond during incorporation of backbone-elongated substrates. J Am Chem Soc  129: 6180–6186.  <ul class="find" data-citedArticleID="999366" data-doi="10.1021/ja068033n"><li><a href="http://dx.doi.org/10.1021/ja068033n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unexpected+preference+of+the+E.+coli+translation+system+for+the+ester+bond+during+incorporation+of+backbone-elongated+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unexpected+preference+of+the+E.+coli+translation+system+for+the+ester+bond+during+incorporation+of+backbone-elongated+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000972-Soutourina1" id="pone.0000972-Soutourina1"></a>Soutourina J, Plateau P, Blanquet S (2000) Metabolism of D-aminoacyl-tRNAs in <em>Escherichia coli</em> and <em>Saccharomyces cerevisiae</em> cells. J Biol Chem  275: 32535–32542.  <ul class="find" data-citedArticleID="999376" data-doi="10.1074/jbc.m005166200"><li><a href="http://dx.doi.org/10.1074/jbc.m005166200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Metabolism+of+D-aminoacyl-tRNAs+in+Escherichia+coli+and+Saccharomyces+cerevisiae+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Metabolism+of+D-aminoacyl-tRNAs+in+Escherichia+coli+and+Saccharomyces+cerevisiae+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000972-Veber1" id="pone.0000972-Veber1"></a>Veber DF, Johnson SR, Cheng HY, Smith BR, Ward KW, et al.  (2002) Molecular properties that influence the oral bioavailability of drug candidates. J Med Chem  45: 2615–2623.  <ul class="find" data-citedArticleID="999388" data-doi="10.1021/jm020017n"><li><a href="http://dx.doi.org/10.1021/jm020017n" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000972-Fahnestock1" id="pone.0000972-Fahnestock1"></a>Fahnestock S, Rich A (1971) Ribosome-catalyzed polyester formation. Science  173: 340–343.  <ul class="find" data-citedArticleID="999280" data-doi="10.1126/science.173.3994.340"><li><a href="http://dx.doi.org/10.1126/science.173.3994.340" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ribosome-catalyzed+polyester+formation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ribosome-catalyzed+polyester+formation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000972-Koh1" id="pone.0000972-Koh1"></a>Koh JT, Cornish VW, Schultz PG (1997) An experimental approach to evaluating the role of backbone interactions in proteins using unnatural amino acid mutagenesis. Biochemistry  36: 11314–11322.  <ul class="find" data-citedArticleID="999318" data-doi="10.1021/bi9707685"><li><a href="http://dx.doi.org/10.1021/bi9707685" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+experimental+approach+to+evaluating+the+role+of+backbone+interactions+in+proteins+using+unnatural+amino+acid+mutagenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+experimental+approach+to+evaluating+the+role+of+backbone+interactions+in+proteins+using+unnatural+amino+acid+mutagenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000972-Millward1" id="pone.0000972-Millward1"></a>Millward SW, Takahashi TT, Roberts RW (2005) A general route for post-translational cyclization of mRNA-displayed libraries. J Am Chem Soc  127: 14142–14143.  <ul class="find" data-citedArticleID="999338" data-doi="10.1021/ja054373h"><li><a href="http://dx.doi.org/10.1021/ja054373h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+general+route+for+post-translational+cyclization+of+mRNA-displayed+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+general+route+for+post-translational+cyclization+of+mRNA-displayed+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0000972-Mursinna1" id="pone.0000972-Mursinna1"></a>Mursinna RS, Lee KW, Briggs JM, Martinis SA (2004) Molecular dissection of a critical specificity determinant within the amino acid editing domain of leucyl-tRNA synthetase. Biochemistry  43: 155–165.  <ul class="find" data-citedArticleID="999348" data-doi="10.1021/bi034919h"><li><a href="http://dx.doi.org/10.1021/bi034919h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+dissection+of+a+critical+specificity+determinant+within+the+amino+acid+editing+domain+of+leucyl-tRNA+synthetase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+dissection+of+a+critical+specificity+determinant+within+the+amino+acid+editing+domain+of+leucyl-tRNA+synthetase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000972-Sever1" id="pone.0000972-Sever1"></a>Sever S, Rogers K, Rogers MJ, Carter C Jr, Soll D (1996) Escherichia coli tryptophanyl-tRNA synthetase mutants selected for tryptophan auxotrophy implicate the dimer interface in optimizing amino acid binding. Biochemistry  35: 32–40.  <ul class="find" data-citedArticleID="999372" data-doi="10.1021/bi952103d"><li><a href="http://dx.doi.org/10.1021/bi952103d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Escherichia+coli+tryptophanyl-tRNA+synthetase+mutants+selected+for+tryptophan+auxotrophy+implicate+the+dimer+interface+in+optimizing+amino+acid+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Escherichia+coli+tryptophanyl-tRNA+synthetase+mutants+selected+for+tryptophan+auxotrophy+implicate+the+dimer+interface+in+optimizing+amino+acid+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000972-Tsao1" id="pone.0000972-Tsao1"></a>Tsao M-L, Summerer D, Ryu Y, Schultz PG (2006) The genetic incorporation of a distance probe into proteins in <em>Escherichia coli</em>. J Am Chem Soc  128: 4572–4573.  <ul class="find" data-citedArticleID="999386" data-doi="10.1021/ja058262u"><li><a href="http://dx.doi.org/10.1021/ja058262u" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genetic+incorporation+of+a+distance+probe+into+proteins+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genetic+incorporation+of+a+distance+probe+into+proteins+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000972-Fahnestock2" id="pone.0000972-Fahnestock2"></a>Fahnestock S, Neumann H, Rich A (1974) Assay of ester and polyester formation by the ribosomal peptidyltransferase. Methods Enzymol  30: 489–497.  <ul class="find" data-citedArticleID="999282" data-doi="10.1016/0076-6879(74)30049-3"><li><a href="http://dx.doi.org/10.1016/0076-6879(74)30049-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assay+of+ester+and+polyester+formation+by+the+ribosomal+peptidyltransferase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assay+of+ester+and+polyester+formation+by+the+ribosomal+peptidyltransferase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0000972-Merryman1" id="pone.0000972-Merryman1"></a>Merryman C, Green R (2004) Transformation of aminoacyl tRNAs for the <em>in vitro</em> selection of “drug-like” molecules. Chem Biol  11: 575–582.  <ul class="find" data-citedArticleID="999336" data-doi="10.1016/j.chembiol.2004.03.009"><li><a href="http://dx.doi.org/10.1016/j.chembiol.2004.03.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transformation+of+aminoacyl+tRNAs+for+the+in+vitro+selection+of+%E2%80%9Cdrug-like%E2%80%9D+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transformation+of+aminoacyl+tRNAs+for+the+in+vitro+selection+of+%E2%80%9Cdrug-like%E2%80%9D+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0000972-OFarrell1" id="pone.0000972-OFarrell1"></a>O'Farrell PH (1978) The suppression of defective translation by ppGpp and its role in the stringent response. Cell  14: 545–557.  <ul class="find" data-citedArticleID="999356" data-doi="10.1016/0092-8674(78)90241-6"><li><a href="http://dx.doi.org/10.1016/0092-8674(78)90241-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+suppression+of+defective+translation+by+ppGpp+and+its+role+in+the+stringent+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+suppression+of+defective+translation+by+ppGpp+and+its+role+in+the+stringent+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0000972-Mayer1" id="pone.0000972-Mayer1"></a>Mayer C, Kohrer C, Kenny E, Prusko C, RajBhandary UL (2003) Anticodon sequence mutants of <em>Escherichia coli</em> initiator tRNA: effects of overproduction of aminoacyl-tRNA synthetases, methionyl-tRNA formyltransferase, and initiation factor 2 on activity in initiation. Biochemistry  42: 4787–4799.  <ul class="find" data-citedArticleID="999332" data-doi="10.1021/bi034011r"><li><a href="http://dx.doi.org/10.1021/bi034011r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anticodon+sequence+mutants+of+Escherichia+coli+initiator+tRNA%3A+effects+of+overproduction+of+aminoacyl-tRNA+synthetases%2C+methionyl-tRNA+formyltransferase%2C+and+initiation+factor+2+on+activity+in+initiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anticodon+sequence+mutants+of+Escherichia+coli+initiator+tRNA%3A+effects+of+overproduction+of+aminoacyl-tRNA+synthetases%2C+methionyl-tRNA+formyltransferase%2C+and+initiation+factor+2+on+activity+in+initiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0000972-Roberts1" id="pone.0000972-Roberts1"></a>Roberts RW, Szostak JW (1997) RNA-peptide fusions for the <em>in vitro</em> selection of peptides and proteins. Proc Natl Acad Sci U S A  94: 12297–12302.  <ul class="find" data-citedArticleID="999360" data-doi="10.1073/pnas.94.23.12297"><li><a href="http://dx.doi.org/10.1073/pnas.94.23.12297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA-peptide+fusions+for+the+in+vitro+selection+of+peptides+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA-peptide+fusions+for+the+in+vitro+selection+of+peptides+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0000972-Hanes1" id="pone.0000972-Hanes1"></a>Hanes J, Pluckthun A (1997) In vitro selection and evolution of functional proteins by using ribosome display. Proc Natl Acad Sci U S A  94: 4937–4942.  <ul class="find" data-citedArticleID="999292" data-doi="10.1073/pnas.94.10.4937"><li><a href="http://dx.doi.org/10.1073/pnas.94.10.4937" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+selection+and+evolution+of+functional+proteins+by+using+ribosome+display." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+selection+and+evolution+of+functional+proteins+by+using+ribosome+display.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0000972-Mattheakis1" id="pone.0000972-Mattheakis1"></a>Mattheakis LC, Bhatt RR, Dower WJ (1994) An in vitro polysome display system for identifying ligands from very large peptide libraries. Proc Natl Acad Sci U S A  91: 9022–9026.  <ul class="find" data-citedArticleID="999330" data-doi="10.1073/pnas.91.19.9022"><li><a href="http://dx.doi.org/10.1073/pnas.91.19.9022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+in+vitro+polysome+display+system+for+identifying+ligands+from+very+large+peptide+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+in+vitro+polysome+display+system+for+identifying+ligands+from+very+large+peptide+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0000972-Bemis1" id="pone.0000972-Bemis1"></a>Bemis GW, Murcko MA (1999) Properties of known drugs. 2. Side chains. J Med Chem  42: 5095–5099.  <ul class="find" data-citedArticleID="999264" data-doi="10.1021/jm9903996"><li><a href="http://dx.doi.org/10.1021/jm9903996" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properties+of+known+drugs.+2.+Side+chains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properties+of+known+drugs.+2.+Side+chains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0000972-Schweizer1" id="pone.0000972-Schweizer1"></a>Schweizer EE, Berninger CJ, Crouse DM, Davis RA, Logothetis RS (1968) Reactions of Phosphorus Compounds. XIX. Reactions of 3-(o-formylphenoxy)propyltriphenylphosph​oniumbromide and 3-(p-formylphenoxy)propyltriphenylphosph​oniumbromide). J Org Chem  34: 207–212.  <ul class="find" data-citedArticleID="999370" data-doi="10.1021/jo00838a045"><li><a href="http://dx.doi.org/10.1021/jo00838a045" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reactions+of+Phosphorus+Compounds.+XIX.+Reactions+of+3-%28o-formylphenoxy%29propyltriphenylphosphonium+bromide+and+3-%28p-formylphenoxy%29propyltriphenylphosphonium+bromide%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reactions+of+Phosphorus+Compounds.+XIX.+Reactions+of+3-%28o-formylphenoxy%29propyltriphenylphosphonium+bromide+and+3-%28p-formylphenoxy%29propyltriphenylphosphonium+bromide%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.XML" value="155998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.PDF" value="585286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g001.PNG_L" value="699395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g001.PNG_M" value="68392"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g001.PNG_S" value="7943"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g001.TIF" value="3477828"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g001.PNG_I" value="30888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t001.PNG_L" value="210598"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t001.PNG_M" value="117375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t001.PNG_S" value="16568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t001.TIF" value="1104944"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t001.PNG_I" value="36215"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g002.PNG_L" value="41659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g002.PNG_M" value="32874"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g002.PNG_S" value="4745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g002.TIF" value="71742"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g002.PNG_I" value="11963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g003.PNG_L" value="64885"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g003.PNG_M" value="40889"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g003.PNG_S" value="6159"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g003.TIF" value="101582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g003.PNG_I" value="14179"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g004.PNG_L" value="92617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g004.PNG_M" value="48615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g004.PNG_S" value="6517"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g004.TIF" value="127724"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g004.PNG_I" value="16620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g005.PNG_L" value="87006"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g005.PNG_M" value="52278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g005.PNG_S" value="7046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g005.TIF" value="131330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g005.PNG_I" value="17845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g006.PNG_L" value="78607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g006.PNG_M" value="45089"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g006.PNG_S" value="6610"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g006.TIF" value="117922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g006.PNG_I" value="15777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g007.PNG_L" value="1001679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g007.PNG_M" value="110302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g007.PNG_S" value="9958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g007.TIF" value="2542014"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g007.PNG_I" value="38589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g008.PNG_L" value="545863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g008.PNG_M" value="47062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g008.PNG_S" value="6658"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g008.TIF" value="1233886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g008.PNG_I" value="16468"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g009.PNG_L" value="728750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g009.PNG_M" value="72046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g009.PNG_S" value="5743"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g009.TIF" value="1314360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g009.PNG_I" value="25586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g010.PNG_L" value="432174"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g010.PNG_M" value="88501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g010.PNG_S" value="7749"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g010.TIF" value="2648312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g010.PNG_I" value="34222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t002.PNG_L" value="81689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t002.PNG_M" value="106310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t002.PNG_S" value="15105"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t002.TIF" value="370406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t002.PNG_I" value="36423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g011.PNG_L" value="139784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g011.PNG_M" value="123390"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g011.PNG_S" value="12428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g011.TIF" value="898342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.g011.PNG_I" value="50920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t003.PNG_L" value="75206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t003.PNG_M" value="96471"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t003.PNG_S" value="15573"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t003.TIF" value="423102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t003.PNG_I" value="32397"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t004.PNG_L" value="23635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t004.PNG_M" value="28494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t004.PNG_S" value="8415"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t004.TIF" value="123388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.t004.PNG_I" value="9519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000972.s001.DOC" value="36864"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000972&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000972" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000972&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000972&volume=&issue=&title=An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides&author_name=Matthew%20C.%20T.%20Hartman%2C%20Kristopher%20Josephson%2C%20Chi-Wang%20Lin%2C%20Jack%20W.%20Szostak&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000972" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972&amp;t=An%20Expanded%20Set%20of%20Amino%20Acid%20Analogs%20for%20the%20Ribosomal%20Translation%20of%20Unnatural%20Peptides" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972&title=An%20Expanded%20Set%20of%20Amino%20Acid%20Analogs%20for%20the%20Ribosomal%20Translation%20of%20Unnatural%20Peptides&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972&amp;title=An%20Expanded%20Set%20of%20Amino%20Acid%20Analogs%20for%20the%20Ribosomal%20Translation%20of%20Unnatural%20Peptides" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000972&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'An Expanded Set of Amino Acid Analogs for the Ribosomal Translation of Unnatural Peptides';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000972';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20An%20Expanded%20Set%20of%20Amino%20Acid%20Analogs%20for%20the%20Ribosomal%20Translation%20of%20Unnatural%20Peptides http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000972" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000972" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Aminoacylation%22" title="Search for articles in the subject area:'Aminoacylation'"><div class="flagText">Aminoacylation</div></a>
              <div data-categoryid="44783" data-articleid="24780"
                   data-categoryname="Aminoacylation"
                   class="flagImage" title="Flag 'Aminoacylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="31811" data-articleid="24780"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Methionine%22" title="Search for articles in the subject area:'Methionine'"><div class="flagText">Methionine</div></a>
              <div data-categoryid="40697" data-articleid="24780"
                   data-categoryname="Methionine"
                   class="flagImage" title="Flag 'Methionine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptide+libraries%22" title="Search for articles in the subject area:'Peptide libraries'"><div class="flagText">Peptide libraries</div></a>
              <div data-categoryid="20137" data-articleid="24780"
                   data-categoryname="Peptide libraries"
                   class="flagImage" title="Flag 'Peptide libraries' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptide+synthesis%22" title="Search for articles in the subject area:'Peptide synthesis'"><div class="flagText">Peptide synthesis</div></a>
              <div data-categoryid="40593" data-articleid="24780"
                   data-categoryname="Peptide synthesis"
                   class="flagImage" title="Flag 'Peptide synthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Peptides%22" title="Search for articles in the subject area:'Peptides'"><div class="flagText">Peptides</div></a>
              <div data-categoryid="17847" data-articleid="24780"
                   data-categoryname="Peptides"
                   class="flagImage" title="Flag 'Peptides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Protein+translation%22" title="Search for articles in the subject area:'Protein translation'"><div class="flagText">Protein translation</div></a>
              <div data-categoryid="24691" data-articleid="24780"
                   data-categoryname="Protein translation"
                   class="flagImage" title="Flag 'Protein translation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transfer+RNA%22" title="Search for articles in the subject area:'Transfer RNA'"><div class="flagText">Transfer RNA</div></a>
              <div data-categoryid="21239" data-articleid="24780"
                   data-categoryname="Transfer RNA"
                   class="flagImage" title="Flag 'Transfer RNA' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1713'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5437'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1607&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=45305">A lovely study</a><br>Posted by Chris_Surridge</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
