

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001400"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001400" />

    <meta name="citation_title" content="Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus"/>
    <meta itemprop="name" content="Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus"/>

      <meta name="citation_author" content="Bernard Couvreur"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Jérôme Beaufays"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Cédric Charon"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Kathia Lahaye"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="François Gensale"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Valérie Denis"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Benoît Charloteaux"/>
            <meta name="citation_author_institution" content="Centre de Biophysique Moléculaire Numérique, Gembloux Agricultural University, Gembloux, Belgium"/>
      <meta name="citation_author" content="Yves Decrem"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Pierre-Paul Prévôt"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Michel Brossard"/>
            <meta name="citation_author_institution" content="Institute of Zoology, University of Neuchâtel, Neuchâtel, Switzerland"/>
      <meta name="citation_author" content="Luc Vanhamme"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
            <meta name="citation_author_institution" content="Laboratory of Molecular Parasitology, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>
      <meta name="citation_author" content="Edmond Godfroid"/>
            <meta name="citation_author_institution" content="Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001400.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1400"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Evolutionary evolutionary biology of parasites. Monographs in population biology.; citation_author=PW Price; citation_number=1; citation_pages=237; citation_date=1980; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=The global importance of ticks.; citation_author=F Jongejan; citation_author=G Uilenberg; citation_journal_title=Parasitology; citation_volume=129; citation_number=2; citation_pages=S3-S14; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Biology of ticks. Vol. I.; citation_author=DE Sonenshine; citation_number=3; citation_date=1991; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Biochemical perspectives on paralysis and other forms of toxicoses caused by ticks.; citation_author=BJ Mans; citation_author=R Gothe; citation_author=AWH Neitz; citation_journal_title=Parasitology; citation_volume=129; citation_number=4; citation_pages=S95-S111; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Role of saliva in blood-feeding by arthropods.; citation_author=JMC Ribeiro; citation_journal_title=Ann Rev Entomol; citation_volume=32; citation_number=5; citation_pages=463-478; citation_date=1987a; " />
      <meta name="citation_reference" content="citation_title=Role of arthropod saliva in blood feeding: Sialome and post-sialome perspectives.; citation_author=JMC Ribeiro; citation_author=IMB Fancischetti; citation_journal_title=Ann Rev. Entomol; citation_volume=48; citation_number=6; citation_pages=73-88; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Exploring tick saliva: from biochemistry to “sialomes” and functional genomics.; citation_author=JG Valenzuela; citation_journal_title=Parasitology; citation_volume=129; citation_number=7; citation_pages=S83-S94; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Tick saliva in anti-tick immunity and pathogen transmission.; citation_author=L Kovar; citation_journal_title=Folia Microbiol; citation_volume=49; citation_number=8; citation_pages=327-336; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Immunobiology 5th ed., Garland Publishing.; citation_author=CA Janeway; citation_author=P Travers; citation_author=M Walport; citation_author=MJ Shlomchik; citation_number=9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The regulators of complement activation (RCA) gene cluster.; citation_author=D Hourcade; citation_author=VM Holers; citation_author=JP Atkinson; citation_journal_title=Adv Immunol; citation_volume=45; citation_number=10; citation_pages=381-416; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Properdin:binding to C3b and stabilization of the C3b-dependent C3 convertase.; citation_author=DT Fearon; citation_author=KF Austen; citation_journal_title=J Exp Med.; citation_volume=142; citation_number=11; citation_pages=856-63; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Analysis of the natural polymeric forms of human properdin and their functions in complement activation.; citation_author=MK Pangburn; citation_journal_title=J immunol; citation_volume=142; citation_number=12; citation_pages=202-207; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Molecular architecture of human properdin, a positive regulator of the alternative pathway of complement.; citation_author=CA Smith; citation_author=MK Pangburn; citation_author=CW Vogel; citation_author=HJ Müller-Eberhard; citation_journal_title=J Biol Chem; citation_volume=259; citation_number=13; citation_pages=4582-4588; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Properdin, the terminal complement components, thrombospondin and the circumsporozoite protein of malaria parasites contain similar sequence motifs.; citation_author=D Goundis; citation_author=KBM Reid; citation_journal_title=Nature; citation_volume=335; citation_number=14; citation_pages=82-85; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning of the cDNA coding for properdin, a positive regulator of the alternative pathway of human complement.; citation_author=KF Nolan; citation_author=W Schwaeble; citation_author=S Kaluz; citation_author=MP Dierich; citation_author=KB Reid; citation_journal_title=Eur J Immunol; citation_volume=21; citation_number=15; citation_pages=771-776; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Neutron and X-ray scattering studies on the human complement protein properdin provide an analysis of the thrombospondin repeat.; citation_author=KF Smith; citation_author=KF Nolan; citation_author=KB Reid; citation_author=SJ Perkins; citation_journal_title=Biochemistry; citation_volume=30; citation_number=16; citation_pages=8000-8008; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The binding of human complement proteins C5, factor B, beta 1H and properdin to complement fragment C3b on zymosan.; citation_author=RG DiScipio; citation_journal_title=Biochem J.; citation_volume=199; citation_number=17; citation_pages=485-496; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=The role of properdin in the assembly of the alternative pathway C3 convertases of complement.; citation_author=DE Hourcade; citation_journal_title=J Biol Chem; citation_volume=281; citation_number=18; citation_pages=2128-2132; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Inhibition of complement alternative pathway function with anti-properdin monoclonal antibodies.; citation_author=R Gupta-Bansal; citation_author=JB Parent; citation_author=KR Brunden; citation_journal_title=Mol Immunol.; citation_volume=37; citation_number=19; citation_pages=191-201; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Complement evasion by bacteria and parasites.; citation_author=KA Joiner; citation_journal_title=Ann Rev Microbiol; citation_volume=42; citation_number=20; citation_pages=201-230; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Acquired resistance to ticks. iii. Cobra venom factor and the resistance response.; citation_author=SK Wikel; citation_author=JR Allen; citation_journal_title=Immunology; citation_volume=32; citation_number=21; citation_pages=457-465; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Acquired resistance to ticks: expression of resistance by C4-deficient guinea pigs.; citation_author=SK Wikel; citation_journal_title=Am J Trop Med Hyg.; citation_volume=28; citation_number=22; citation_pages=586-590; citation_date=1979; " />
      <meta name="citation_reference" content="citation_title=Ixodes dammini: salivary anti-complement activity.; citation_author=JM Ribeiro; citation_journal_title=Exp Parasitol.; citation_volume=64; citation_number=23; citation_pages=347-353; citation_date=1987b; " />
      <meta name="citation_reference" content="citation_title=Ixodes ticks: serum species sensitivity of anticomplement activity.; citation_author=CH Lawrie; citation_author=SE Randolph; citation_author=PA Nuttall; citation_journal_title=Exp Parasitol; citation_volume=93; citation_number=24; citation_pages=207-214; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Purification, cloning, and expression of a novel salivary anticomplement protein from the tick, Ixodes scapularis.; citation_author=JG Valenzuela; citation_author=R Charlab; citation_author=TN Mather; citation_author=JM Ribeiro; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=25; citation_pages=18717-18723; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Immunosuppressive effects of Ixodes ricinus tick saliva or salivary gland extracts on innate and acquired immune response of BALB/c mice.; citation_author=N Mejri; citation_author=B Rutti; citation_author=M Brossard; citation_journal_title=Parasitol Res; citation_volume=88; citation_number=26; citation_pages=192-197; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Capillary feeding of specific dsRNA induces silencing of the Isac gene in nymphal Ixodes scapularis ticks.; citation_author=CA Soares; citation_author=CM Lima; citation_author=MC Dolan; citation_author=J Piesman; citation_author=CB Beard; citation_journal_title=Insect Mol Biol; citation_volume=14; citation_number=27; citation_pages=443-452; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Salp25D, an Ixodes scapularis antioxidant, is 1 of 14 immunodominant antigens in engorged tick salivary glands.; citation_author=S Das; citation_author=G Banerjee; citation_author=K DePonte; citation_author=N Marcantonio; citation_author=FS Kantor; citation_journal_title=J Infect Dis; citation_volume=184; citation_number=28; citation_pages=1056-1064; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Biochemical and functional characterization of Salp20, an Ixodes scapularis tick salivary protein that inhibits the complement pathway.; citation_author=K Tyson; citation_author=C Elkins; citation_author=H Patterson; citation_author=E Fikrig; citation_author=A de Silva; citation_journal_title=Insect Molecular Biology; citation_volume=16; citation_number=29; citation_pages=469-479; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The transcriptome of the salivary glands of the female western black-legged tick Ixodes pacificus (Acari: Ixodidae).; citation_author=IM Francischetti; citation_author=V My Pham; citation_author=BJ Mans; citation_author=JF Andersen; citation_author=TN Mather; citation_journal_title=Insect Biochem Mol Biol.; citation_volume=35; citation_number=30; citation_pages=1142-1161; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Ixodes ticks belonging to the Ixodes ricinus complex encode a family of anticomplement proteins.; citation_author=V Daix; citation_author=H Schroeder; citation_author=N Praet; citation_author=J-P Georgin; citation_author=I Chiappino; citation_journal_title=Insect Mol Biol.; citation_volume=16; citation_number=31; citation_pages=155-166; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Host immune response evasion strategies in Ornithodoros erraticus and O. moubata and their relationship to the development of an antiargasid vaccine.; citation_author=A Astigarraga; citation_author=A Oleaga-Perez; citation_author=R Perez-Sanchez; citation_author=JA Baranda; citation_author=A Encinas-Grandes; citation_journal_title=Parasite Immunol.; citation_volume=19; citation_number=32; citation_pages=401-410; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Complement inhibitor of C5 activation from the soft tick Ornithodoros moubata.; citation_author=MA Nunn; citation_author=A Sharma; citation_author=GC Paesen; citation_author=S Adamson; citation_author=O Lissina; citation_journal_title=J Immunol; citation_volume=174; citation_number=33; citation_pages=2084-2091; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exploring the sialome of the tick Ixodes scapularis.; citation_author=JG Valenzuela; citation_author=IM Francischetti; citation_author=VM Pham; citation_author=MK Garfield; citation_author=TN Mather; citation_journal_title=J Exp Biol; citation_volume=205(Pt 18); citation_number=34; citation_pages=2843-2864; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=An annotated catalog of salivary gland transcripts from Ixodes scapularis ticks.; citation_author=JM Ribeiro; citation_author=F Alarcon-Chaidez; citation_author=IM Francischetti; citation_author=BJ Mans; citation_author=TN Mather; citation_journal_title=Insect Biochem Mol Biol; citation_volume=36; citation_number=35; citation_pages=111-129; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Genome size and organization in the blacklegged tick, Ixodes scapularis and the Southern cattle tick, Boophilus microplus.; citation_author=AJ Ullmann; citation_author=CM Lima; citation_author=FD Guerrero; citation_author=J Piesman; citation_author=WC Black; citation_journal_title=Insect Mol Biol; citation_volume=4; citation_number=36; citation_pages=217-222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome size and organization in the ixodid tick Amblyomma americanum (L.).; citation_author=MJ Palmer; citation_author=JA Bantle; citation_author=X Guo; citation_author=WS Fargo; citation_journal_title=Insect Mol Biol; citation_volume=3; citation_number=37; citation_pages=57-62; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The third, fourth, and fifth components of human complement: isolation and biochemical properties.; citation_author=BF Tack; citation_author=J Janatova; citation_author=ML Thomas; citation_author=RA Harrison; citation_author=CH Hammer; citation_number=38; citation_pages=64; citation_date=1981; citation_publisher=Academic Press; " />
      <meta name="citation_reference" content="citation_title=Investigation of the mechanisms of anti-complement activity in Ixodes ricinus ticks.; citation_author=CH Lawrie; citation_author=RB Sim; citation_author=PA Nuttall; citation_journal_title=Mol Immunol.; citation_volume=42; citation_number=39; citation_pages=31-38; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chemical characterization of human factor B of the alternate pathway of complement activation.; citation_author=B Curman; citation_author=L Sandberg-Tragardh; citation_author=PA Peterson; citation_journal_title=Biochemistry.; citation_volume=16; citation_number=40; citation_pages=5368-5375; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Evolution by gene duplication.; citation_author=S Ohno; citation_number=41; citation_pages=150; citation_date=1970; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Adaptive Evolution of Genes and Genomes.; citation_author=AL Hughes; citation_number=42; citation_pages=270; citation_date=1999; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Duplication and divergence: the evolution of new genes and old ideas.; citation_author=JS Taylor; citation_author=J Raes; citation_journal_title=Annu Rev Genet.; citation_volume=38; citation_number=43; citation_pages=615-643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution of animal toxin multigene families.; citation_author=D Kordis; citation_author=F Gubensek; citation_journal_title=Gene.; citation_volume=261; citation_number=44; citation_pages=43-52; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Adaptive evolution in the snake venom Kunitz/BPTI protein family.; citation_author=V Zupunski; citation_author=D Kordis; citation_author=F Gubensek; citation_journal_title=FEBS Lett; citation_volume=547; citation_number=45; citation_pages=131-136; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution of hematophagy in ticks: common origins for blood coagulation and platelet aggregation inhibitors from soft ticks of the genus Ornithodoros.; citation_author=BJ Mans; citation_author=AI Louw; citation_author=AWH Neitz; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=46; citation_pages=1695-1705; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Variation in the salivary peptide, maxadilan, from species in the Lutzomyia longipalpis complex.; citation_author=GC Lanzaro; citation_author=AH Lopes; citation_author=JM Ribeiro; citation_author=CB Shoemaker; citation_author=A Warburg; citation_journal_title=Insect Mol Biol.; citation_volume=8; citation_number=47; citation_pages=267-275; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Antigenic diversity in maxadilan, a salivary protein from the sand fly vector of American visceral leishmaniasis.; citation_author=RS Milleron; citation_author=JP Mutebi; citation_author=S Valle; citation_author=A Montoya; citation_author=H Yin; citation_journal_title=Am J Trop Med Hyg; citation_volume=70; citation_number=48; citation_pages=286-293; citation_date=2004a; " />
      <meta name="citation_reference" content="citation_title=Negative effect of antibodies against maxadilan on the fitness of the sand fly vector of American visceral leishmaniasis.; citation_author=RS Milleron; citation_author=JM Ribeiro; citation_author=D Elnaime; citation_author=L Soong; citation_author=GC Lanzaro; citation_journal_title=Am J Trop Med Hyg; citation_volume=70; citation_number=49; citation_pages=278-285; citation_date=2004b; " />
      <meta name="citation_reference" content="citation_title=Assembly of the cytolytic alternative pathway of complement from 11 isolated plasma proteins.; citation_author=RD Schreiber; citation_author=HJ Muller-Eberhard; citation_journal_title=J Exp Med; citation_volume=148; citation_number=50; citation_pages=1722-1727; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=The C3 convertase of the alternative pathway of human complement. Enzymic properties of the bimolecular proteinase.; citation_author=MK Pangburn; citation_author=HJ Muller-Eberhard; citation_journal_title=Biochem J; citation_volume=235; citation_number=51; citation_pages=723-730; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Properdin can initiate complement activation by binding specific target surfaces and providing a platform for de novo convertase assembly.; citation_author=D Spitzer; citation_author=LM Mitchell; citation_author=JP Atkinson; citation_author=DE Hourcade; citation_journal_title=J Immunol; citation_volume=179; citation_number=52; citation_pages=2600-2608; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Activator-specific requirement of properdin in the initiation and amplification of the alternative pathway complement.; citation_author=Y Kimura; citation_author=T Miwa; citation_author=L Zhou; citation_author=WC Song; citation_journal_title=Blood.; citation_number=53; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Properdin.; citation_author=KF Nolan; citation_author=KD Reid; citation_journal_title=Methods Enzymol; citation_volume=223; citation_number=54; citation_pages=35-46; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The dimeric and trimeric solution structures of the multidomain complement protein properdin by X-ray scattering, analytical ultracentrifugation and constrained modelling.; citation_author=Z Sun; citation_author=KB Reid; citation_author=SJ Perkins; citation_journal_title=J Mol Biol; citation_volume=343; citation_number=55; citation_pages=1327-1343; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complement escape of human pathogenic bacteria by acquisition of complement regulators.; citation_author=P Kraiczy; citation_author=R Würzner; citation_journal_title=Mol Immunol; citation_volume=43; citation_number=56; citation_pages=31-44; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Anti-immunology: evasion of the host immune system by bacterial and viral pathogens.; citation_author=BB Finlay; citation_author=G McFadden; citation_journal_title=Cell.; citation_volume=124; citation_number=57; citation_pages=767-782; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=West Nile virus nonstructural protein NS1 inhibits complement activation by binding the regulatory protein factor H.; citation_author=KM Chung; citation_author=MK Liszewski; citation_author=G Nybakken; citation_author=AE Davis; citation_author=RR Townsend; citation_journal_title=Proc Natl Acad Sci U S A.; citation_volume=103; citation_number=58; citation_pages=19111-191116; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mechanism of complement inactivation by glycoprotein C of herpes simplex virus.; citation_author=I Kostavasili; citation_author=A Sahu; citation_author=HM Friedman; citation_author=RJ Eisenberg; citation_author=GH Cohen; citation_journal_title=J Immunol.; citation_volume=158; citation_number=59; citation_pages=1763-1771; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Evasion of innate immunity by parasitic protozoa.; citation_author=D Sacks; citation_author=A Sher; citation_journal_title=Nat Immunol; citation_volume=3; citation_number=60; citation_pages=1041-1047; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structure and function of recombinant cobra venom factor.; citation_author=MA Kock; citation_author=BE Hew; citation_author=H Bammert; citation_author=DC Fritzinger; citation_author=CW Vogel; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=61; citation_pages=30836-30843; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complement-targeted therapeutics.; citation_author=D Ricklin; citation_author=JD Lambris; citation_journal_title=Nat Biotechnol; citation_volume=25; citation_number=62; citation_pages=1265-1275; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Discovery and development of the complement inhibitor eculizumab for the treatment of paroxysmal nocturnal hemoglobinuria.; citation_author=RP Rother; citation_author=SA Rollins; citation_author=CF Mojcik; citation_author=RA Brodsky; citation_author=L Bell; citation_journal_title=Nat Biotechnol; citation_volume=25; citation_number=63; citation_pages=1256-1264; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Structure-function relationships of the complements components.; citation_author=KB Reid; citation_author=AJ Day; citation_journal_title=Immunol. Today; citation_volume=10; citation_number=64; citation_pages=177-180; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=The possibilities and pitfalls for anti-complement therapies in inflammatory diseases.; citation_author=M Mizuno; citation_author=BP Morgan; citation_journal_title=Curr Drug Targets Inflamm Allergy; citation_volume=3; citation_number=65; citation_pages=87-96; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Strategies of therapeutic complement inhibition.; citation_author=TE Mollnes; citation_author=M Kirschfink; citation_journal_title=Mol Immunol; citation_volume=43; citation_number=66; citation_pages=107-121; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Epizootiology of Lyme Borreliosis.; citation_author=JF Anderson; citation_journal_title=Scand J Infect Dis–Suppl.; citation_volume=77; citation_number=67; citation_pages=3-34; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=The Ecology of the sheep tick, Ixodes ricinus L., host relationship of the tick. Part 2. Observations on hill and moorland grazings in northern England.; citation_author=A Milne; citation_journal_title=Parasitology; citation_volume=39; citation_number=68; citation_pages=173-197; citation_date=1949; " />
      <meta name="citation_reference" content="citation_title=Tick-host specificity.; citation_author=H Hoogstraal; citation_author=A Aeschlimann; citation_journal_title=Bulletin de la société entomologique suisse; citation_volume=55; citation_number=69; citation_pages=5-32; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=The paralogous salivary anti-complement proteins IRAC I and IRAC II encoded by Ixodes ricinus ticks have broad and complementary inhibitory activities against the complement of different host species.; citation_author=H Schroeder; citation_author=V Daix; citation_author=L Gillet; citation_author=JC Renauld; citation_author=A Vanderplasschen; citation_journal_title=Microbes Infect; citation_volume=9; citation_number=70; citation_pages=247-250; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genomic view of the evolution of the complement system.; citation_author=M Nonaka; citation_author=A Kimura; citation_journal_title=Immunogenetics; citation_volume=58; citation_number=71; citation_pages=701-713; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The alternative pathway of complement in disease: opportunities for therapeutic targeting.; citation_author=VM Holers; citation_author=JM Thurman; citation_journal_title=Mol Immunol; citation_volume=41(2–3); citation_number=72; citation_pages=147-52; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The central role of the alternative complement pathway in human disease.; citation_author=JM Thurman; citation_author=VM Holers; citation_journal_title=J Immunol; citation_volume=176; citation_number=73; citation_pages=1305-1310; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recombinant soluble CR1 suppressed complement activation, inflammation, and necrosis associated with reperfusion of ischemic myocardium.; citation_author=HF Weisman; citation_author=T Bartow; citation_author=MK Leppo; citation_author=MP Boyle; citation_author=HC Marsh; citation_journal_title=Trans Assoc Am Physicians; citation_volume=103; citation_number=74; citation_pages=64-72; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Inhibition of complement activity by humanized anti-C5 antibody and single-chain Fv.; citation_author=TC Thomas; citation_author=SA Rollins; citation_author=RP Rother; citation_author=MA Giannoni; citation_author=SL Hartman; citation_journal_title=Mol Immunol; citation_volume=33; citation_number=75; citation_pages=1389-401; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=The scanning model for translation: an update.; citation_author=M Kozak; citation_journal_title=J Cell Biol; citation_volume=108; citation_number=76; citation_pages=229-241; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Tick histamine-binding proteins: isolation, cloning, and three-dimensional structure.; citation_author=GC Paesen; citation_author=PL Adams; citation_author=K Harlos; citation_author=PA Nuttall; citation_author=DI Stuart; citation_journal_title=Mol Cell; citation_volume=3; citation_number=77; citation_pages=661-671; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Anti-hemostatic effects of a serpin from the saliva of the tick Ixodes ricinus.; citation_author=PP Prevot; citation_author=B Adam; citation_author=KZ Boudjeltia; citation_author=M Brossard; citation_author=L Lins; citation_journal_title=J Biol Chem.; citation_volume=281; citation_number=78; citation_pages=26361-26369; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Locating proteins in the cell using TargetP, SignalP, and related tools.; citation_author=O Emanuelsson; citation_author=S Brunak; citation_author=G von Heijne; citation_author=H Nielsen; citation_journal_title=Nature Protocols; citation_volume=2; citation_number=79; citation_pages=953-971; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Prediction, conservation analysis, and structural characterization of mammalian mucin-type O-glycosylation sites.; citation_author=K Julenius; citation_author=A Molgaard; citation_author=R Gupta; citation_author=S Brunak; citation_journal_title=Glycobiology; citation_volume=15; citation_number=80; citation_pages=153-164; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes.; citation_author=A Krogh; citation_author=B Larsson; citation_author=G von Heijne; citation_author=ELL Sonnhammer; citation_journal_title=J Mol Biol; citation_volume=305; citation_number=81; citation_pages=567-580; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Disulfide Connectivity Prediction using Recursive Neural Networks and Evolutionary Information.; citation_author=A Vullo; citation_author=P Frasconi; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=82; citation_pages=653-659; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Hydrophobic cluster analysis: an efficient new way to compare and analyse amino acid sequences.; citation_author=C Gaboriaud; citation_author=V Bissery; citation_author=T Benchetrit; citation_author=JP Mornon; citation_journal_title=FEBS Lett; citation_volume=224; citation_number=83; citation_pages=149-155; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Deciphering protein sequence information through hydrophobic cluster analysis (HCA): current status and perspectives.; citation_author=I Callebaut; citation_author=G Labesse; citation_author=P Durand; citation_author=A Poupon; citation_author=L Canard; citation_journal_title=Cell Mol Life Sci.; citation_volume=53; citation_number=84; citation_pages=621-645; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=GeneDoc: Analysis and Visualization of Genetic Variation.; citation_author=KB Nicholas; citation_author=HB Nicholas; citation_author=DW Deerfield; citation_journal_title=EMBNEW NEWS; citation_volume=4; citation_number=85; citation_pages=14; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Systematic Biology; citation_volume=52; citation_number=86; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=TreeView: an application to display phylogenetic trees on personal computers.; citation_author=RD Page; citation_journal_title=Comput Appl Biosci.; citation_volume=12; citation_number=87; citation_pages=357-358; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Reconstruction of ancestral protein sequences and its applications.; citation_author=W Cai; citation_author=J Pei; citation_author=NV Grishin; citation_journal_title=BMC Evol Biol.; citation_volume=4; citation_number=88; citation_pages=33; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment.; citation_author=S Kumar; citation_author=K Tamura; citation_author=M Nei; citation_journal_title=Brief Bioinform.; citation_volume=5; citation_number=89; citation_pages=150-163; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Complement tests.; citation_author=PC Giclas; citation_number=90; citation_pages=181-186; citation_date=1997; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Complement.; citation_author=JE Colligan; citation_number=91; citation_pages=13.1.1-13.2.7; citation_date=1994; citation_publisher=Wiley/Interscience; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus"/>
    <meta name="twitter:description" content="BackgroundTicks are blood feeding arachnids that characteristically take a long blood meal. They must therefore counteract host defence mechanisms such as hemostasis, inflammation and the immune response. This is achieved by expressing batteries of salivary proteins coded by multigene families.Methodology/Principal FindingsWe report the in-depth analysis of a tick multigene family and describe five new anticomplement proteins in Ixodes ricinus. Compared to previously described Ixodes anticomplement proteins, these segregated into a new phylogenetic group or subfamily. These proteins have a novel action mechanism as they specifically bind to properdin, leading to the inhibition of C3 convertase and the alternative complement pathway. An excess of non-synonymous over synonymous changes indicated that coding sequences had undergone diversifying selection. Diversification was not associated with structural, biochemical or functional diversity, adaptation to host species or stage specificity but rather to differences in antigenicity.Conclusions/SignificanceAnticomplement proteins from I. ricinus are the first inhibitors that specifically target a positive regulator of complement, properdin. They may provide new tools for the investigation of role of properdin in physiological and pathophysiological mechanisms. They may also be useful in disorders affecting the alternative complement pathway. Looking for and detecting the different selection pressures involved will help in understanding the evolution of multigene families and hematophagy in arthropods."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001400.g011"/>

  <meta property="og:title" content="Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1235'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6503'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9653&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001400">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001400" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001400&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Variability and Action Mechanism of a Family of Anticomplement Proteins in <i>Ixodes ricinus</i>
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernard Couvreur
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Bernard Couvreur, Jérôme Beaufays, Cédric Charon</p>

              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jérôme Beaufays
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Bernard Couvreur, Jérôme Beaufays, Cédric Charon</p>

              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Cédric Charon
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Bernard Couvreur, Jérôme Beaufays, Cédric Charon</p>

              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kathia Lahaye, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            François Gensale, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Valérie Denis, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Benoît Charloteaux, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Centre de Biophysique Moléculaire Numérique, Gembloux Agricultural University, Gembloux, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yves Decrem, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pierre-Paul Prévôt, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michel Brossard, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Zoology, University of Neuchâtel, Neuchâtel, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Luc Vanhamme, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium, 
                  Laboratory of Molecular Parasitology, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Edmond Godfroid
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:edmond.godfroid@ulb.ac.be">edmond.godfroid@ulb.ac.be</a></p>

                <p>Affiliation:
                  Laboratory for Molecular Biology of Ectoparasites, Institut de Biologie et de Médecine Moléculaires (IBMM), Université Libre de Bruxelles, Gosselies, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001400</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001400-t001" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g001" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g002" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-t002" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-t003" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g003" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g004" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-t004" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g005" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g006" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g007" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g008" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g009" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-t005" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g010" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001400-g011" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001400">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001400" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Ticks are blood feeding arachnids that characteristically take a long blood meal. They must therefore counteract host defence mechanisms such as hemostasis, inflammation and the immune response. This is achieved by expressing batteries of salivary proteins coded by multigene families.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We report the in-depth analysis of a tick multigene family and describe five new anticomplement proteins in <em>Ixodes ricinus</em>. Compared to previously described <em>Ixodes</em> anticomplement proteins, these segregated into a new phylogenetic group or subfamily. These proteins have a novel action mechanism as they specifically bind to properdin, leading to the inhibition of C3 convertase and the alternative complement pathway. An excess of non-synonymous over synonymous changes indicated that coding sequences had undergone diversifying selection. Diversification was not associated with structural, biochemical or functional diversity, adaptation to host species or stage specificity but rather to differences in antigenicity.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Anticomplement proteins from <em>I. ricinus</em> are the first inhibitors that specifically target a positive regulator of complement, properdin. They may provide new tools for the investigation of role of properdin in physiological and pathophysiological mechanisms. They may also be useful in disorders affecting the alternative complement pathway. Looking for and detecting the different selection pressures involved will help in understanding the evolution of multigene families and hematophagy in arthropods.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Couvreur B, Beaufays J, Charon C, Lahaye K, Gensale F, et al.  (2008) Variability and Action Mechanism of a Family of Anticomplement Proteins in <em>Ixodes ricinus</em>. PLoS ONE 3(1):
          e1400.
            doi:10.1371/journal.pone.0001400</p><p><strong>Academic Editor: </strong>Jeffrey Gold, Oregon Health &amp; Science University, United States of America</p><p><strong>Received:</strong> September 25, 2007; <strong>Accepted:</strong> December 11, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Couvreur et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>LVH is a Senior Research Associate at the Belgian National Fund for Scientific Research. CC is supported by a PhD studentship from the Fonds pour la Formation à la Recherche dans l'Industrie et dans l'Agriculture of the Belgian Federal Governement. This work was supported by contracts 215107 and 415702 from the Ministère de la Région Wallonne in Belgium.</p><p><strong>Competing interests:</strong> The activities of IxAC-B1 to B5 are being patented.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Parasites probably affect every living organism and it may reasonably be estimated that at least half the animals on earth are parasites <a href="#pone.0001400-Price1">[1]</a>. By definition, parasites live at the expense of their host but hosts defend themselves and, in turn, parasites evolve counter-measures. Parasitism is probably therefore a major driving force in evolution <a href="#pone.0001400-Price1">[1]</a>. Bloodfeeding arthropods such as ticks constitute a very good example of the evolutionary arms race between hosts and parasites.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Ticks are obligate blood feeding arachnids. They infest many species of mammals, birds, reptiles and amphibians worldwide. They are the vectors of protozoan, bacterial and viral pathogens of prime medical and veterinary importance. Examples of such important pathogens are <em>Borrelia burgdorferi</em>, Tick-borne Encephalitis Virus (TBEV), <em>Babesia bovis</em> or <em>Theileria parva</em>, the respective agents of Lyme disease and viral encephalitis in humans, and babesiosis and theileriosis (East Coast Fever) in cattle <a href="#pone.0001400-Jongejan1">[2]</a>. Blood losses due to heavy infestation may weaken the animal, render it more susceptible to other diseases or cause death by exsanguination <a href="#pone.0001400-Sonenshine1">[3]</a>. In addition, the occurrence of tick toxicosis including tick paralysis is associated with the presence of toxins in the saliva <a href="#pone.0001400-Mans1">[4]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>There are two main families of ticks, <em>Ixodidae</em> or hard ticks and <em>Argasidae</em> or soft ticks. The <em>Ixodidae</em> family is further divided into two subdivisions: Prostriata, which contains only the subfamily <em>Ixodinae</em>, and Metastriata, which includes the subfamilies <em>Bothriocrotinae</em>, <em>Amblyomminae</em>, <em>Haemaphysalinae</em> and <em>Rhipicephalinae</em> <a href="#pone.0001400-Sonenshine1">[3]</a>. Argasid ticks typically feed for short periods of time (up to 2 hours) whereas Ixodid ticks remain attached to and feed on their vertebrate host for up to two weeks <a href="#pone.0001400-Sonenshine1">[3]</a>. The feeding time of an adult <em>Ixodes ricinus</em> female is typically 7–10 days <a href="#pone.0001400-Sonenshine1">[3]</a>. Such a long blood meal is only possible because these parasites have developed ways to circumvent host defense mechanisms including hemostasis (coagulation, platelet aggregation and vasoconstriction), the inflammatory response and innate and adaptive immunity <a href="#pone.0001400-Ribeiro1">[reviewed in 5]</a>, <a href="#pone.0001400-Ribeiro2">[6]</a>, <a href="#pone.0001400-Valenzuela1">[7]</a>, <a href="#pone.0001400-Kovar1">[8]</a>. Furthermore, pain or itching caused by the inflammatory response stimulates hosts to scratch and dislodge the parasite.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The complement system is a first line of defence against invading pathogens and it links the innate and adaptive responses of the vertebrate immune system <a href="#pone.0001400-Janeway1">[reviewed in 9]</a>. It consists of a cascade of plasma enzymes leading to activation of three effector mechanisms: (i) generation of the short potent pro-inflammatory peptides C3a and C5a, ii) deposition of opsonizing C3b proteins on cell surfaces, (iii) formation of the membrane attack complex (MAC). MACs create pores in the membrane, leading to cell death. Complement is activated <em>via</em> three main pathways. The classical pathway (CP) is initiated mainly when the C1 complex binds to the Fc region of certain antibody isotypes in immune complexes. The lectin-mediated pathway is activated by mannose-binding lectin interacting with mannose residues on microbial surfaces.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The alternative pathway (AP) is spontaneously activated by hydrolysis of plasma C3 into C3 (H<sub>2</sub>O). C3 (H<sub>2</sub>O) binds soluble factor B (fB). Bound fB is cleaved by serine protease factor D into soluble Ba peptide and the larger Bb fragment. The resulting C3 (H<sub>2</sub>O)Bb complex is the initial C3 convertase. It cleaves fluid-phase C3 into C3a peptide and metastable C3b. C3b binds covalently to a pathogen or cell surface via a short-lived thioester bond. Factor B interacts with C3b, leading to its cleavage by factor D and the formation of the C3 convertase (C3bBb). This complex generates new C3b molecules and amplifies the complement cascade by forming new C3 convertases or C5 convertases (C3b2Bb). C5 convertase cleaves C5 into C5a and C5b. C5b initiates the formation of MAC <a href="#pone.0001400-Janeway1">[9]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Host cells are protected from attack by the complement system by plasma and membrane-bound regulatory molecules that inactivate complement proteins. C3 convertases are deactivated by dissociation mediated by surface proteins such as Decay-Accelerating Factor (DAF) and Complement Receptor-1 (CR1), as well as soluble factor H. These proteins bind to C3b and displace Bb <a href="#pone.0001400-Janeway1">[9]</a>. They also act as co-factors for serine protease factor I which cleaves C3b <a href="#pone.0001400-Hourcade1">[10]</a>. On the other hand, the half-life of C3 convertase is increased at least 10-fold by properdin <a href="#pone.0001400-Fearon1">[11]</a>. It is present in the plasma in oligomer form (dimer, trimer or tetramer) <a href="#pone.0001400-Pangburn1">[12]</a>, <a href="#pone.0001400-Smith1">[13]</a>. Each monomer is a 53 kDa protein composed of six repetitive thrombospondin domains (TSP), flanked with an N-terminal and C-terminal region <a href="#pone.0001400-Goundis1">[14]</a>, <a href="#pone.0001400-Nolan1">[15]</a>, <a href="#pone.0001400-Smith2">[16]</a>. Properdin binds to surface-bound C3b and increases its ability to interact with factor B <a href="#pone.0001400-DiScipio1">[17]</a>. It also binds to pre-formed C3 convertases leading to increased stability and preventing inactivation by regulators such as factor H and factor I <a href="#pone.0001400-Janeway1">[9]</a>. Moreover, properdin oligomers attached to C3b on cell surfaces interact with preformed fluid-phase C3b or C3bBb through its other subunits <a href="#pone.0001400-Hourcade2">[18]</a>. The essential role of properdin in complement activation was demonstrated by the capacity of an anti-properdin monoclonal antibody to inhibit activation of the alternative pathway. This monoclonal antibody prevents the interaction between properdin and C3b <a href="#pone.0001400-GuptaBansal1">[19]</a>.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>The AP is the major line of defense against invading pathogens such as bacteria <a href="#pone.0001400-Joiner1">[20]</a>. It is also involved in guinea pig resistance to the hard tick <em>Dermacentor andersoni</em> <a href="#pone.0001400-Wikel1">[21]</a>, <a href="#pone.0001400-Wikel2">[22]</a>. Saliva or salivary gland extracts from <em>Ixodes dammini</em> <a href="#pone.0001400-Ribeiro3">[23]</a>, <em>I. hexagonus</em> and <em>I. uriae</em> <a href="#pone.0001400-Lawrie1">[24]</a>, <em>I. scapularis</em> <a href="#pone.0001400-Valenzuela2">[25]</a> and <em>I. ricinus</em> <a href="#pone.0001400-Lawrie1">[24]</a>, <a href="#pone.0001400-Mejri1">[26]</a> have been found to have an inhibitory activity on the alternative complement pathway. Valenzuela et al. <a href="#pone.0001400-Valenzuela2">[25]</a> purified the active anticomplement component from the saliva of adult <em>I. scapularis</em>. N-terminal sequencing combined with the screening of a cDNA library led to the description of the coding sequence of a tick anticomplement protein named ISAC (<em>I. scapularis</em> anti-complement). Recombinant ISAC mimics the anticomplement activity of tick saliva. It interferes with the formation of C3 convertase from C3 and fB and destabilizes pre-formed C3 convertase. Sequences closely related to ISAC were then cloned by RT/PCR from <em>I. scapularis</em> nymphs <a href="#pone.0001400-Soares1">[27]</a>, found by screening a cDNA library with sera from repeatedly infested guinea pigs <a href="#pone.0001400-Das1">[28]</a> or by PCR-screening of a nymph cDNA library <a href="#pone.0001400-Tyson1">[29]</a>. In <em>I. pacificus</em>, sequencing large numbers of cDNA clones from adult salivary glands led to the discovery of ISAC-I <a href="#pone.0001400-Francischetti1">[30]</a>. Finally, using degenerate primers designed from the published ISAC sequence, Daix et al. <a href="#pone.0001400-Daix1">[31]</a> recently cloned the related IRAC I and IRAC II from <em>I. ricinus</em>.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>In soft ticks too, anticomplement activity is present in saliva and salivary gland extracts <a href="#pone.0001400-Astigarraga1">[32]</a>. In <em>Ornithodoros moubata</em>, this activity is due to protein OmCI which inhibits both the alternative and classical pathways. Its sequence is unrelated to the <em>Ixodes</em> anticomplement molecules mentioned above (&lt;15% amino-acid identity). OmCI binds to C5 component of the complement cascade and belongs to the lipocalin superfamily <a href="#pone.0001400-Nunn1">[33]</a>.</p>
<a id="article1.body1.sec1.p9" name="article1.body1.sec1.p9"></a><p>The recent characterization of large numbers of cDNA sequences from salivary glands of Ixodid ticks including <em>I. scapularis</em> <a href="#pone.0001400-Valenzuela3">[34]</a>, <a href="#pone.0001400-Ribeiro4">[35]</a> and <em>I. pacificus</em> <a href="#pone.0001400-Francischetti1">[30]</a> indicated that most salivary proteins are expressed as large clusters of related proteins, probably coded by multigene families. Moreover, genome size and organization were examined in <em>Ixodes scapularis</em>, <em>Boophilus microplus</em> <a href="#pone.0001400-Ullmann1">[36]</a> and <em>Amblyomma americanum</em> <a href="#pone.0001400-Palmer1">[37]</a>. These genomes are large: 2.1×10<sup>9</sup>, 7.1×10<sup>9</sup> bp and 1.04×10<sup>9</sup> bp respectively. Reassociation rates of genomic DNA indicate that they are composed mainly of moderately repetitive elements, which include transposable elements and members of multigene families. This organization in multigene families is therefore probably a major feature of hard tick genome organization and perhaps an adaptation to bloodfeeding.</p>
<a id="article1.body1.sec1.p10" name="article1.body1.sec1.p10"></a><p>In the work described here we completed an inventory of sequences related to <em>I. scapularis</em> anticomplement protein ISAC in the salivary glands of <em>I. ricinus</em>. We were able to detect five new sequences that we used to study diversification mechanisms possibly at work in a family of tick salivary proteins (hereby referred to as the IxAC family) and we investigated their action mechanism. The results showed that <em>I. ricinus</em> anticomplement proteins specifically bind to properdin, leading to the inhibition of the formation of C3 convertase and inhibition of the alternative complement activation pathway. Sequence diversification is associated with antigenic diversity rather than major divergence in molecular characteristics or activity, host specificity or stage specificity.</p>
<a id="article1.body1.sec1.p11" name="article1.body1.sec1.p11"></a><p>In this study, we investigated the general significance of multigene families in the context of a host-parasite relationship. This is a specific in-depth analysis of a tick multigene family. It highlights the value of performing dedicated gene-targeted inventories when studying specific aspects of tick adaptation to a bloodfeeding lifestyle. Remarkably, IxACs from <em>I. ricinus</em> are also the first inhibitors that specifically target a positive regulator of complement.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A large family of anticomplement proteins in the hard tick <em>Ixodes ricinus</em></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The fragments of two genes related to the prototypical <em>Ixodes scapularis</em> anticomplement protein ISAC of Valenzuela et al. <a href="#pone.0001400-Valenzuela2">[25]</a> were found by serendipity in cDNA from pooled salivary glands. The complete coding sequences as well as parts of the 3′ and 5′ UTR's were then reconstituted by RACE. The new genes (accession numbers: AM407396 and AM407397) coded for two new proteins showing ~40 % identity with ISAC and the recently described IRAC I and IRAC II from <em>I. ricinus</em> <a href="#pone.0001400-Daix1">[31]</a>, but over 65 % identity with each other. This led us to suspect the existence of a much larger family of anticomplement proteins in <em>I. ricinus</em>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In order to make as complete an inventory of this family as possible, a total of 2 different reverse transcription experiments, 6 different PCR amplifications and 12 ligations were performed on polyA+ RNA from salivary glands of <em>I. ricinus</em> females (<a href="#pone.0001400.s004">Table S1</a>). 122 clones with inserts of the expected size (≥600 bp) were sequenced. A few were disregarded as they coded for proteins unrelated to anticomplement proteins (e.g. ribosomal proteins). A few additional clones with inserts smaller than expected (400–500 bp) were sequenced too. They were found to code for homologs of uncharacterized “putative salivary proteins” from <em>I. scapularis</em> and <em>I. pacificus</em>. Of the 118 anticomplement-like clones found, most could be assigned to previously described IRAC I, IRAC II, AM407396 and AM407397 on the basis of sequence identity. AM407396 was the most frequent (46.6 %) followed by AM407397 (23.3 %), IRAC I (15.5 %) and IRAC II (0.9%). Three additional new sequences were also identified. They were assigned accession numbers AM407398, AM407399 and AM407400 respectively. They accounted for 3.4 %, 8.6 % and 1.7% of clones, respectively. Sequences found to be identical in ≥3 independent clones were considered genuine. AM407400 was only represented by two clones which showed a difference of two nucleotides. Therefore, the latter sequence was confirmed independently by amplifying internal fragments with gene-specific primers from salivary gland cDNA. Finally, the same set of sequences was found from the various independent RT-PCR experiments (<a href="#pone.0001400.s004">Table S1</a>). Overall, the results suggested that a complete or near-complete inventory of IxAC anticomplement messengers from the <em>Ixodes ricinus</em> salivary gland was achieved at least for the population investigated.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>PCR experiments using the primers listed in <a href="#pone-0001400-t001">Table 1</a> were also performed on cDNA from pooled salivary glands of 3 day-fed male and female <em>R. appendiculatus</em>. No PCR products were observed (not shown). We were nevertheless able to amplify the coding sequence for the known lipocalin RaHBP-2 from the same cDNAs using specific primers. Interrogation of databases including the preliminary releases of sequences from the non-<em>Ixodes</em> hard tick <em>R. appendiculatus, B. microplus, A. americanum and A. variegatum</em> genome projects yielded no or only insignificant hits.</p>
<div class="figure" id="pone-0001400-t001"><div class="img"><a name="pone-0001400-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>PCR primers for RT-PCR inventory of <em>I. ricinus</em> genes coding for anticomplement proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001400.t001</span></div>

<h4>Phylogenetic analysis of tick anticomplement sequences</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Nucleotide and peptide sequences of ISAC, IRAC I and AM407396 were used to interrogate databases with the same results. A total of 48 entries were recovered (<a href="#pone.0001400.s005">Table S2</a>). These were from <em>I. scapularis</em> (45 entries), <em>I. ricinus</em> (2 entries) and <em>I. pacificus</em> (1 entry). They had been cloned from salivary glands (30), whole fed nymphs and nymph salivary glands (15) or unspecified tick material (3).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Two sequences containing ambiguous positions and ten sequences with incomplete coding sequences for the mature protein, including Salp9 (AF278574) and Isac-like clone 113 (AY956386), were initially discarded. These 12 entries were from <em>I. scapularis</em>. The remaining 36 entries were aligned with our five new sequences from <em>I. ricinus</em> (AM407396 to AM407400). Distance dendrograms were constructed from alignments of nucleotide sequences or predicted amino-acid sequences of putative mature proteins translated from the open reading frames (<a href="#pone-0001400-g001">Figure 1</a>). All sequences clustered into two main groups or subfamilies, IxAC-A and IxAC-B, which were strongly supported by bootstrap analysis (1000/1000). IxAC-A could be further divided into two clusters. A first, large, group contained only <em>I. scapularis</em> sequences closely related to prototypical ISAC. It was strongly supported by bootstrap analysis (&gt;900/1000). A second, smaller, cluster contained Isac-1 from <em>I. pacificus</em>, IRAC II from <em>I. ricinus</em>, Salp20-like protein 2 and EST n° DN970085 from <em>I. scapularis</em>. Bootstrap support was lower (≥830/1000). IRAC I could not be joined robustly (bootstrap value &lt;750/1000) to any of the previous two clusters. The IxAC-B subfamily contained our five new sequences AM407396 to AM407400 from <em>I. ricinus</em> but none from other tick species. No robust cluster emerged within this subfamily.</p>
<div class="figure" id="pone-0001400-g001"><div class="img"><a name="pone-0001400-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogenetic analysis of <em>Ixodes</em> anticomplement proteins.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A distance dendrogram was constructed from an alignment of 41 tick mature anticomplement proteins using programs in the Phylip 3.65 package (see text). Branch length is proportional to distances between peptide sequences. The bootstrap values are indicated near major nodes, calculated from 1000 replicates of the peptide and nucleotide sequence alignments, respectively. Bold characters: <em>I. ricinus</em> entries; *: <em>I. pacificus</em> sequence; all others are from <em>I. scapularis</em>. Prototypical ISAC is boxed. Sequences are identified by their accession number in databases or by descriptive names when available. Ad., isolated from adults; Ny., isolated from nymphs; NS, not specified.</p>
<span>doi:10.1371/journal.pone.0001400.g001</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The maximum-likelihood method was also applied to the initial nucleotide and amino-acid alignment of putative mature proteins. It supported the same topology as the distance method with slightly different bootstrap values (<a href="#pone-0001400-g001">Figure 1</a>). The two subfamilies were strongly supported (1000/1000). The two clusters within IxAC-A were also recovered but bootstrap support was lower than with the distance method. Again, IRAC I could not be placed robustly in any of the two clusters within IxAC-A (bootstrap support &lt;650/1000).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>The overall topology of the distance trees was not altered after including the leader peptide sequences in the alignments (not shown) or the 12 discarded sequences (not shown). Most of the latter clustered with ISAC within the ISAC-like cluster (not shown). Only Salp9 grouped with Isac-I, IRAC II and EST n° DN970085 within the second cluster in IxAC-A. Salp9, a 79 residues peptide, aligned to the C-terminal half of DN970085 to which it showed 90% identity. Because ISAC and Salp9 were the earliest tick anticomplement sequences published, we decided to name the large and small clusters within IxAC-A “ISAC-like” and “Salp9-like”, respectively.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>We therefore decided to rename the five new sequences (AM407396 to AM407400) IxAC-B1 to IxAC-B5 to indicate the fact that they clustered into the new group or subfamily IxAC-B and away from the previously described IRAC I and IRAC II which belong to the IxAC-A subfamily.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>Percentages of identity and similarity were calculated for representative IxACs (<a href="#pone-0001400-t002">Table 2</a>). Within a subfamily, amino-acid sequences were over 60 % identical whereas identity dropped to ~40 % between the two subfamilies. The two subfamilies could also be differentiated by an indel of 4 amino-acids (position 74 to 77 in <a href="#pone-0001400-g002">Figure 2</a>). Within the Isac-like cluster or within the Salp9-like cluster, amino-acid sequences were at least 70% identical.</p>
<div class="figure" id="pone-0001400-g002"><div class="img"><a name="pone-0001400-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Alignment of <em>Ixodes</em> anticomplement proteins.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>The 7 anticomplement proteins from <em>I. ricinus</em> (IRAC I and II; IxAC-B1 to B5) were aligned with the prototypical anticomplement protein from <em>I. scapularis</em> (ISAC) and the homolog from <em>I. pacificus</em> (ISAC I). Individual residues printed in white on a black background are conserved in all 9 aligned sequences; white residues on a grey background are conserved in 7 or 8 of 9 entries; black residues on a grey background are conserved in 5 or 6 entries; black residues on a white background are conserved in less than 5 entries. -, gap; ! !, region of predicted signal peptide cleavage; C1 to C4, conserved cysteine residues.</p>
<span>doi:10.1371/journal.pone.0001400.g002</span></div><div class="figure" id="pone-0001400-t002"><div class="img"><a name="pone-0001400-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Nucleotide and amino-acid identity/similarity of mature anticomplement proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001400.t002</span></div><a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>According to information provided on the entry files or in the original publications, the sequences had been obtained from adults (12), nymphs (34) or unspecified stages (2). However, no robust “adult-only” or “nymph-only” clusters could be discerned (<a href="#pone-0001400-g001">Figure 1</a>). Moreover, “adult” EST n° DN968378 was found to be 100% identical to “nymphal” Salp20 (AF209917) (<a href="#pone-0001400-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>To summarize, phylogenetic analysis of all available tick anticomplement sequences indicated that they robustly segregated into two distinct groups or subfamilies, which we termed IxAC-A and IxAC-B. Within-group amino-acid identity was &gt;60% whereas between-group identity dropped to ~40%. The larger IxAC-A contained sequences from <em>I. scapularis</em>, <em>I. ricinus</em> and <em>I. pacificus</em>. It could be subdivided into two or possibly three clusters. Our new sequences from <em>I. ricinus</em> constituted a completely new group which we named IxAC-B. No stage-specific group of sequences was identified at this point of the research.</p>


<h4>Protein properties</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The properties of these newly discovered proteins were predicted from their amino-acid sequences and compared to prototypical ISAC from <em>I. scapularis</em> and related Isac-1 from <em>I. pacificus</em>. Calculated PM and pI ranged from 17.46 to 18.03 and 4.01 to 4.29 respectively (<a href="#pone-0001400-t003">Table 3</a>).</p>
<div class="figure" id="pone-0001400-t003"><div class="img"><a name="pone-0001400-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Calculated properties of anticomplement proteins.</span></strong></p><span>doi:10.1371/journal.pone.0001400.t003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>All anticomplement proteins presented four conserved cysteine residues predicted to make two disulphide bridges (<a href="#pone-0001400-g002">Figure 2</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Most likely signal peptide cleavage sites for representative IxACs are indicated in <a href="#pone-0001400-t003">Table 3</a> and <a href="#pone-0001400-g002">Figure 2</a>. In each individual peptide sequence, SignalP predicted a second or even a third probable, though less likely, cleavage site. These were at position 19 (after C residue), 21 (after SS residues) or 22 (after SSN/E). Altogether, cleavage at any 3 locations within the C↑SS↑(S) ↑E/N motif is theoretically possible.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>The presence of a signal peptide and the absence of any hydrophobic transmembrane region suggests that these proteins are secreted. This was supported by a TargetP program analysis and confirmed experimentally as recombinant IxACs were recovered in the culture medium after transfection of COS7 or 293T cells.</p>
<a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5"></a><p>In a western blot analysis using an anti-V5 antibody, all recombinant IxACs from <em>I. ricinus</em> appeared as a series of thin bands at 50–70 kDa (<a href="#pone-0001400-g003">Figure 3a</a>). The apparent molecular weights are consistent with reported values for purified native anticomplement proteins from <em>I. scapularis</em> (~65 kDa) <a href="#pone.0001400-Valenzuela2">[25]</a> and <em>I. damini</em> (~49 kDa) <a href="#pone.0001400-Ribeiro3">[23]</a>. They contrast with the predicted MW of ~18 kDa. This difference and the appearance of the bands were in agreement with extensive glycosylation. Indeed, several consensus sites for N- and O- glycosylation were found in the sequences (<a href="#pone-0001400-t003">Table 3</a>). Furthermore, the presence of N-linked glycosylation was experimentally confirmed by treatment with N-glycosidase F leading to a fall in observed MW to 35–45 kDa (<a href="#pone-0001400-g003">Figure 3b</a>). Recombinant Salp20 expressed in insect cells also appears as a smear possibly representing differentially glycosylated forms of the protein <a href="#pone.0001400-Tyson1">[29]</a>. The authors experimentally confirmed the presence of N-linked and O-linked sugars.</p>
<div class="figure" id="pone-0001400-g003"><div class="img"><a name="pone-0001400-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Western blot analysis of recombinant IxAC-V5His proteins from <em>I. ricinus</em>.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Standardised amounts of recombinant IxAC-V5His proteins from supernatants of transfected 293T cells were analysed by SDS/PAGE and detected by western blotting using an anti-V5 monoclonal antibody. A) Parallel analysis of IxACs, B) N-deglycosylation of IxAC-B1-V5His.1, untreated, 2, incubated with PNGase (New England Biolabs).</p>
<span>doi:10.1371/journal.pone.0001400.g003</span></div><a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6"></a><p>Finally, hydrophobic cluster analysis (HCA) showed that the distribution of clusters of hydrophobic amino-acids had a nearly identical distribution in all 7 IxACs from <em>I. ricinus</em> as well as in ISAC and ISAC-1 (<a href="#pone-0001400-g004">Figure 4</a>). This suggested that these 9 proteins had identical folds or tertiary structures.</p>
<div class="figure" id="pone-0001400-g004"><div class="img"><a name="pone-0001400-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of <em>Ixodes</em> anticomplement protein tertiary structure.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Aligned IxAC amino-acid sequences from <em>I. ricinus</em> were submitted to hydrophobic cluster analysis (HCA). Groups of adjacent hydrophobic residues are outlined and shaded. Proline (asterisk), glycine (open rectangle), serine (dotted square) and threonine (open square) are highlighted. The overall distribution of hydrophobic clusters and their size, shape and orientation are very similar.</p>
<span>doi:10.1371/journal.pone.0001400.g004</span></div>

<h4>The excess of non-synonymous over synonymous changes indicates that the coding sequences had undergone diversifying selection</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Calculation of the percentage identity between 7 <em>I. ricinus</em> anticomplement sequences indicated that they were more closely related at the nucleotide level than at the amino-acid level (<a href="#pone-0001400-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>A theoretical ancestral sequence was also re-constructed using the Ancescon program. It was aligned to the 7 actual sequences. The numbers of synonymous changes per synonymous sites (dS) and non-synonymous changes per non-synonymous sites (dN) were calculated using the Nei-Gojobori method. Values for dN/dS were consistently &gt;1 for pairwise comparisons of actual sequences with one another and with the putative ancestral sequence (<a href="#pone-0001400-t004">Table 4</a>, higher-right triangle). The ratio from overall means of dN and dS values was 2.44. Fisher's exact test for positive selection did not reject the hypothesis of dN &gt;dS except in the case of IRAC I compared to IRAC II (P value &lt;0.05) (<a href="#pone-0001400-t004">Table 4</a>, lower left triangle).</p>
<div class="figure" id="pone-0001400-t004"><div class="img"><a name="pone-0001400-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Evidence for positive selection in <em>I. ricinus</em> IxAC coding sequences.</span></strong></p><span>doi:10.1371/journal.pone.0001400.t004</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>All the data therefore show that diversifying selection had taken place within the IxAC family in <em>I. ricinus</em>.</p>


<h4>IxACs from <em>I. ricinus</em> inhibit the alternative complement pathway (AP) but not the classical pathway (CP)</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The effect of similar amounts of the seven <em>I. ricinus</em> IxACs transiently expressed in 293T cells (<a href="#pone-0001400-g003">Figure 3</a>) were assessed in hemolytic assays of both the classical (CP) and alternative complement pathways (AP). A clear dose-dependent inhibition of the AP was observed for all seven recombinant proteins tested as they inhibited the lysis of rabbit erythrocytes by normal human serum. The shape of the curves and plateau values for hemolysis inhibition were identical for the seven proteins (<a href="#pone-0001400-g005">Figure 5</a>).</p>
<div class="figure" id="pone-0001400-g005"><div class="img"><a name="pone-0001400-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Effect of recombinant <em>I. ricinus</em> IxAC proteins on the alternative and classical pathways of complement activation.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Assays of the alternative (AP, solid lines) and classical (CP, dashed lines) complement activation pathway were conducted in the presence of normalized amounts of recombinant <em>I. ricinus</em> IxACs produced in the supernatant of transfected 293T cells. Values for the percent inhibition of rabbit red blood cell lysis in the presence of human serum are indicated. The values are means ± standard deviation of triplicates. RaHBP2 was used as negative control. Black diamond :IRAC I; black square:IRAC II; black triangle:IxAC-B1; cross:IxAC-B2; star:IxAC-B3; closed circle:IxAC-B4; Plus:IxAC-B5; open circle:RaHBP2.</p>
<span>doi:10.1371/journal.pone.0001400.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Addition of recombinant IxACs in the AP assay 15 minutes after adding red blood cells to human serum in the AP assay showed that they were able to inhibit ongoing hemolysis of rabbit erythrocytes. (<a href="#pone.0001400.s001">Figure S1</a>).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The same dilutions of the 7 recombinant proteins were retested in the CP assay in the presence of normal human serum. As shown in <a href="#pone-0001400-g005">Figure 5</a>, there was no inhibition of lysis of antibody-sensitized sheep erythrocytes.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>In both assays, no inhibition was observed after addition of recombinant RaHBP2.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>We therefore concluded that the 7 IxAC from <em>I. ricinus</em> all had a similar inhibitory effect on the alternative but not the classical complement activation pathway.</p>


<h4>IxACs inhibit the cleavage of human C3 and factor B</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>During complement activation by the alternative pathway, plasma protein C3 is cleaved into the large opsonizing factor C3b and the small pro-inflammatory peptide C3a and precursor B is cleaved into a large Bb fragment and a small Ba peptide. We investigated the effect of <em>I. ricinus</em> IxAC proteins on the cleavage of factor B and on the production of C3a in the AP assay. Supernatants of completed AP hemolytic assays were analyzed by Western blot using antisera to fB or C3a.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>As shown in <a href="#pone-0001400-g006">Figure 6A</a>, anti-C3a antibody recognized major bands at 116 kDa, 77kDa and ~10 kDa. Arrows indicate clearly identifiable bands; they corresponded to the α-chain of C3 (115 kDa) and the small C3a peptide (9 kDa) <a href="#pone.0001400-Tack1">[38]</a>, <a href="#pone.0001400-Lawrie2">[39]</a>. The latter was almost completely suppressed in samples from assays run in the presence of recombinant IxACs as compared to sample from assays run in the presence of RaHBP2 or without added protein.</p>
<div class="figure" id="pone-0001400-g006"><div class="img"><a name="pone-0001400-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Inhibition of C3a formation and factor B cleavage.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Aliquots of supernatant from AP hemolysis assays conducted in the presence of standardized amounts of IxACs from <em>I. ricinus</em> and unrelated control RaHBP2 were analyzed by western blotting. Panel A: Blots from gels run under denaturing conditions were probed with monospecific anti-C3a serum. The α-chain of precursor C3 (116 kDa) and the C3a peptide (~10 kDa) are indicated by arrows. Panel B: Blots from gels run under non-denaturing conditions were probed with a antiserum to factor B. Purified factor B was used as a positive control.</p>
<span>doi:10.1371/journal.pone.0001400.g006</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>Antiserum to factor B recognized purified factor B as a single band on a non-denaturating western blot (<a href="#pone-0001400-g006">Figure 6B</a>). A second band was recognized in samples from control AP assays run in the absence of added protein or in the presence of unrelated RaHBP2. It resolved into two distinct bands presumably corresponding to differently charged forms of Bb <a href="#pone.0001400-Lawrie2">[39]</a>, <a href="#pone.0001400-Curman1">[40]</a>. It was absent in a sample from AP assays run in the presence of recombinant IxACs.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>We concluded that <em>I. ricinus</em> IxAC inhibited the formation of C3a and cleavage of fB. Moreover, there were no detectable differences in the degree of this inhibition between different members of the two IxAC sub-families.</p>


<h4><em>I. ricinus</em> IxACs specifically interact with properdin</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>We then attempted to identify the target(s) of IxACs using ELISA methodology. Components of C3 convertase (i.e. C3, C3b, fB, fD or properdin) were coated on microtiter plates and incubated with recombinant IxAC_V5His. Binding of IxACs was monitored using an anti-V5 antibody. We first tested one member of the IxAC-A subfamily and one from the IxAC-B subfamily. IRAC II and IxAC-B1 purified from the baculovirus/Sf9 expression system, but not unrelated protein Iris, strongly bound to properdin. They did not bind to C3, C3b, fB or fD (<a href="#pone-0001400-g007">Figure 7A</a>). In addition, standardized amounts of the seven <em>I. ricinus</em> IxACs but not RaHBP2 bound to properdin in a dose-dependent manner (<a href="#pone-0001400-g007">Figure 7B</a>).</p>
<div class="figure" id="pone-0001400-g007"><div class="img"><a name="pone-0001400-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>ELISA analysis of the binding of IxAC proteins to immobilized C3 convertase components.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Panel A: Binding of IxACs to AP components. Purified recombinant IRAC II, IxAC-B1 or unrelated protein Iris were added to microtiter wells previously coated with purified factors C3, C3b, fB, fD or properdin (P). Bound proteins were detected with an anti-V5 monoclonal antibody using an ELISA format. Light dotted histogram: IRAC II; dark dotted histogram: IxAC-B1; black histogram: Iris. Panel B: Increasing amounts of normalized supernatant from transfected culture 293T cells were added to immobilized properdin. Bound IxACs were detected with an anti-V5 antibody. Black diamond: Iris; black square: IRAC I; black triangle: IRAC II; cross: IxAC-B1; star: IxAC-B2; closed circle: IxAC-B3; plus: IxAC-B4; minus : IxAC-B5. Panel C. Competition between properdin and IxACs for C3b binding. Purified properdin and increasing amounts of IRAC II, IxAC-B1 or unrelated control IRIS were added simultaneously to C3b-precoated microtiter wells. Bound properdin was detected with an anti-properdin monoclonal antibody. Black diamond: IRAC II; black square: IxAC-B1; black triangle: Iris.</p>
<span>doi:10.1371/journal.pone.0001400.g007</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>We also tested the binding of properdin to C3b-coated plates in the presence of increasing amounts of IRAC II, IxAC-B1 and control Iris. Binding was revealed by a monoclonal antibody to properdin (<a href="#pone-0001400-g007">Figure 7C</a>). Increasing amounts of IRAC II, IxAC-B1 but not Iris lead to a decrease in the amount of bound properdin. We concluded that <em>I. ricinus</em> IxACs specifically interacted with properdin and prevented its binding to C3b. Again, no difference could be discerned amongst IxACs.</p>


<h4>IxAC proteins inhibit the formation of the C3 convertase complex by interacting with properdin</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>We also studied the effect of <em>I. ricinus</em> IxACs on the formation and stability of the alternative pathway C3 convertase (C3bBbP). This was reconstituted <em>in vitro</em> by adding purified components fB, fD and properdin to C3b-coated plates. Bound Bb and properdin were detected using specific antibodies. Approximately 10 times less bound Bb was detected in the absence of properdin than in its presence (<a href="#pone-0001400-g008">Figure 8A</a>). We tested the effect of one member of each of the two IxAC subfamilies on the formation and stability of this complex.</p>
<div class="figure" id="pone-0001400-g008"><div class="img"><a name="pone-0001400-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Effect of IxAC proteins on the formation and stability of C3 convertase.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>Panel A and B: The effect of IxAC proteins on the formation of C3 convertase was evaluated by incubating simultaneously purified factors B, D and properdin with increasing amounts of IRAC II or IxAC-B1 on C3b-coated wells. Panel C and D: The effect of IxAC proteins on the stability of C3 convertase was assessed by incubating preformed C3 convertase (fB, fD and properdin pre-incubated for 1 hour) on C3b-coated plates with increasing amounts of recombinant IxACs. Bound factor B or properdin was detected with an anti-factor B antibody (A–C) or anti-properdin antibody (B–D), respectively. Recombinant IRIS was used as negative control. Black diamond: IRAC II; black square: IxAC-B1; black triangle: IRIS.</p>
<span>doi:10.1371/journal.pone.0001400.g008</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>In a first series of experiments, increasing amounts of purified IRAC II and IxAC-B1 were added together with the individual convertase components to C3b-coated plates. We observed a dose-dependent decrease in the amount of bound Bb (<a href="#pone-0001400-g008">Figure 8A</a>) or properdin (<a href="#pone-0001400-g008">Figure 8B</a>), indicating inhibition of complex formation. At the highest protein concentrations (≥200 mM), the amount of bound Bb dropped to values observed when reconstituting the C3 convertase without properdin (<a href="#pone-0001400-g008">Figure 8A</a>). No such effect was observed with the unrelated Iris protein (<a href="#pone-0001400-g008">Figure 8</a>).</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>In a second series of experiments, C3bBbP was pre-formed on ELISA plates and then incubated with increasing amounts of IRAC II and IxAC-B1 proteins and the unrelated control Iris protein (<a href="#pone-0001400-g008">Figure 8C and D</a>). The results indicated that IxAC proteins induced the displacement of all pre-bound factor Bb (<a href="#pone-0001400-g008">Figure 8C</a>) and about 50 % of pre-bound properdin.</p>
<a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4"></a><p>We also performed time-course experiments of C3 convertase formation with (C3bBbP) or without properdin (C3bBb) in the presence of 200 mM IRAC II, IxAC-B1 or Iris. The amount of bound Bb was much lower in the absence of properdin than in its presence. In this case, the presence of IxACs or Iris had no effect, indicating that the proteins had no direct effect on the interaction between C3b and Bb. On the contrary, the formation of C3 convertase in the presence of properdin was strongly affected by IxACs. Values of bound Bb dropped to values observed without properdin (<a href="#pone.0001400.s002">Figure S2</a>).</p>
<a id="article1.body1.sec2.sec8.p5" name="article1.body1.sec2.sec8.p5"></a><p>Overall, these results show that <em>I. ricinus</em> IxAC proteins inhibit the formation of the C3 convertase complex by interacting specifically with properdin. They also induce the displacement of pre-bound properdin, and indirectly, Bb, in a dose-dependent manner.</p>


<h4>IxAC proteins inhibit complement activation on agarose-coated ELISA plates</h4>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1"></a><p>We also tested the ability of IxACs to inhibit activation of the AP on agarose-coated ELISA plates using human serum as a source of complement factors. This experimental set-up is closer to physiological activation of complement than protein-protein interactions conducted on plastic surfaces. Recombinant IRAC II and IxAC-B1 were added in the assay at various time-points after addition of human serum (<a href="#pone-0001400-g009">Figure 9</a>).</p>
<div class="figure" id="pone-0001400-g009"><div class="img"><a name="pone-0001400-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Effect of IxAC proteins on the deposition of C3b and factor B on agarose-coated plates.</span></strong></p><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1"></a><p>Loading human serum on agarose-coated microplate wells activates the alternative complement pathway. Purified recombinant IRAC II (A–B) or IxAC-B1 (C–D) were added after 30, 45 or 60 minutes. The reactions were stopped at various times. C3b and factor B deposition was detected using anti-C3 antibody (A–C) or anti-factor B antibody (B–D), respectively. Black diamond: no added protein; black square: 30 min.; black triangle: 45 min.; cross: 60 min.</p>
<span>doi:10.1371/journal.pone.0001400.g009</span></div><a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2"></a><p>The results indicated that IxAC proteins prevented C3b and factor B deposition in agarose-coated wells when added together with serum (not shown). They also stopped further C3 and factor B deposition when they were added at various times (30, 45 and 60 min) after initiation of the reaction as shown by the immediate plateauing of the curve. Nevertheless, they did not displace bound C3 or Bb as the measured amounts of bound factors did not drop (<a href="#pone-0001400-g009">Figure 9</a>). The results also showed that members of the two IxAC subfamilies are able to inhibit the formation of C3 convertase (<a href="#pone-0001400-g009">Figure 9A–B</a> and <a href="#pone-0001400-g009">Figure 9C–D</a>) in a similar manner. Although they were able to stop the ongoing formation of C3 convertase they could not undo previously formed complexes.</p>
<a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3"></a><p>An additional experiment was also performed. After 60 min. of reaction, the reaction medium was replaced with fresh buffer containing the recombinant proteins but no human serum. A drop in the amount of bound Bb was observed in the presence of IRAC II or IxAC-B1 but not Iris (not shown).</p>
<a id="article1.body1.sec2.sec9.p4" name="article1.body1.sec2.sec9.p4"></a><p>These results therefore confirmed that IxAC proteins are able to inhibit the activation of the AP on a surface by preventing deposition of C3b and Bb.</p>


<h4>Class specificity is observed within Vertebrates</h4>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1"></a><p>We next tested the hypothesis that diversification of anticomplement proteins helps <em>I. ricinus</em> counteract the complement activity of its diverse host organisms. Freshly prepared sera from different vertebrate species were first titrated in the AP assay in order to define the volume causing 50% hemolysis (AH50). A wide range of AH50 values were observed, from the equivalent of 0.25 µl per microwell test (50 µl final volume) for <em>Boa constrictor</em> to 7.0 µl per test for Balb/c mice (<a href="#pone-0001400-t005">Table 5</a>). Heat-inactivated samples completely lost their hemolytic activity, confirming that this activity was indeed due to complement (not shown).</p>
<div class="figure" id="pone-0001400-t005"><div class="img"><a name="pone-0001400-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Host specificity of anticomplement activity by recombinant IxACs from <em>I. ricinus</em>.</span></strong></p><span>doi:10.1371/journal.pone.0001400.t005</span></div><a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2"></a><p>Identical amounts of normalized IxAC proteins from <em>I. ricinus</em> and control RaHBP2 were then added to the AP hemolysis assay in the presence of AH50 volumes of serum. The 7 anticomplement proteins reproducibly inhibited all mammalian sera in a dose-dependent manner (<a href="#pone-0001400-t005">Table 5</a> and <a href="#pone.0001400.s003">Figure S3</a>). In some species, such as humans (<a href="#pone-0001400-g003">Figure 3</a>) or <em>B. Taurus</em> and <em>M. musculus</em> (<a href="#pone.0001400.s003">Figure S3</a>) the dose-response curves were similar for the seven IxACs from <em>I. ricinus</em>. In other mammals such as, <em>C. familiaris</em>, <em>O. aries</em>, <em>S. domesticus</em> and <em>C. elaphus</em> (<a href="#pone.0001400.s003">Figure S3</a>) lower doses of the proteins had different efficiencies. They all reached similar plateau values at higher doses of protein, though. We also observed that most species were not equally sensitive to IxAC inhibition of the AP. Thus hemolysis of rabbit red blood cells by mouse serum was inhibited by not more than ~30% whereas the hemolytic activity of human serum was inhibited at ~85 %. Intermediate plateau values were observed for the other species tested (<a href="#pone.0001400.s003">Figure S3</a>). On the contrary, IxACs from <em>I. ricinus</em> did not affect most bird and squamate reptile sera, with the exception of IRAC II and IxAC-B4, which inhibited AP activity of one bird (<em>Phasianus colchicus</em>) and one snake (<em>Elaph guttata</em>), respectively.</p>
<a id="article1.body1.sec2.sec10.p3" name="article1.body1.sec2.sec10.p3"></a><p>To summarize, all 7 IxAC from <em>I. ricinus</em> inhibited the AP in all mammal species tested. Inhibition of the AP in only one bird by only IRAC II and only one squamate species by only IxAC-B4 was also observed.</p>


<h4>Expression patterns of individual <em>I. ricinus</em> IxACs</h4>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1"></a><p>Primer pairs designed to specifically detect each IxAC messenger (<a href="#pone.0001400.s006">Table S3</a>) were used to analyse their pattern of expression by RT/PCR in individual ticks, at various stages of their life cycle and during the bloodmeal.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2"></a><p>We first investigated IxAC expression during the life cycle. All 7 IxAC sequences were detected in polyA+ RNA from the original pool of 70 salivary glands from adult females. No PCR product was observed from poly A+ RNA that had not undergone reverse transcription, indicating that we did not amplify fragments of genomic DNA (<a href="#pone-0001400-g010">Figure 10a</a>). Younger stages of the life cycle (larvae and nymphs) expressed IRAC I, IxAC-B1, B3 and B5 (<a href="#pone-0001400-g010">Figure 10b</a>). We also investigated whether they expressed additional anticomplement proteins not found in adults by searching for the full coding sequences in 10 pooled nymph and 10 pooled larvae cDNA in high-fidelity conditions using primer pairs designed from the 5′ and 3′ ends of IxAC-A or IxAC-B coding sequences (<a href="#pone-0001400-t001">Table 1</a>). Primers amplified the expected ~600 bp products which were then inserted into pCRII and sequenced. Only IRAC I was retrieved from nymphs (7 independent clones), and IxAC-B1 and IxAC-B5 from larvae (2 and 1 clones respectively). The “nymphal” or “larvae” nucleotide sequences were 100% identical to “adult” ones.</p>
<div class="figure" id="pone-0001400-g010"><div class="img"><a name="pone-0001400-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Expression patterns of individual IxACs.</span></strong></p><a id="article1.body1.sec2.sec11.fig1.caption1.p1" name="article1.body1.sec2.sec11.fig1.caption1.p1"></a><p>PolyA+ RNA was extracted from various <em>I. ricinus</em> material as indicated and reverse transcribed. The resulting cDNAs were submitted to PCR analysis using pairs of primers specific for the indicated IxACs. Non- reverse transcribed polyA+ RNA from the pool was included as negative control (Ctrl). PCR products were run on 1.2 % agarose gels. M, DNA size markers. B1 to B5, IxAC-B1 to IxAC-B5. Sizes in bp are indicated. Panel A: Analysis of salivary glands of a tick population (70 specimens, pool) and from individual female ticks at day 5 of the bloodmeal (A to J). Panel B: Analysis of pooled salivary glands of tick female populations at day 0 (25 specimens), day 3 (25 specimens) and day 5 (70 specimens) of the bloodmeal as well as from pooled gorged nymphs (25 specimens) and larvae (25 specimens).</p>
<span>doi:10.1371/journal.pone.0001400.g010</span></div><a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3"></a><p>We then investigated the expression of the IxAC repertoire during the bloodmeal. Messengers for IRAC I, IRAC II, IxAC-B1, IxAC-B3 and IxAC-B5 were detected in unfed females and the complete repertoire was detected in salivary glands from females on day 3 of the bloodmeal and on day 5 (<a href="#pone-0001400-g010">Figure 10b</a>).</p>
<a id="article1.body1.sec2.sec11.p4" name="article1.body1.sec2.sec11.p4"></a><p>When analysing the pattern of IxAC expression in 10 individual adult females we found that no single specimen expressed the whole range of anticomplement IxAC proteins. Each individual tick expressed one (1 tick), two (6 ticks) or three proteins (3 ticks). IxAC-B1 was expressed in 8/10 ticks tested, followed by IxAC-B3 and B4 (3/10), then IRAC I, IRAC II, IxAC-B2 and B5 (2/10). Four individual ticks expressed members of both IxAC-A and IxAC-B and 6 individual ticks expressed only members of IxAC-B. None expressed solely members of IxAC-A.</p>
<a id="article1.body1.sec2.sec11.p5" name="article1.body1.sec2.sec11.p5"></a><p>Taken together, the results indicated that IxAC are expressed in all ticks tested, throughout the bloodmeal and at all the development stages examined. There was no evidence for stage-specific variants although some members of the family may be induced or up regulated during the first days of the bloodmeal and in the adult. Individual females expressed individual members of the family and no individual tick expressed the complete series of <em>I. ricinus</em> IxACs.</p>


<h4>Antigenic diversification in the IxAC family in <em>I. ricinus</em></h4>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1"></a><p>A monospecific mouse antiserum to IxAC-B1 was produced in mice by DNA immunization followed by a booster with purified recombinant IxAC-B1. It was used to perform western blot analysis of standardized amounts of IRAC I to IxAC-B5 (see <a href="#pone-0001400-g003">Figure 3a</a>) in parallel with the anti-V5 commercial antibody. As shown in <a href="#pone-0001400-g011">Figure 11A</a>, mouse anti-IxAC-B1 serum only recognized IxAC-B1 and none of the other IxACs from <em>I. ricinus</em>. This shows that epitopes recognized on IxAC-B1 in this assay were not present on any other member of the family.</p>
<div class="figure" id="pone-0001400-g011"><div class="img"><a name="pone-0001400-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001400" data-uri="info:doi/10.1371/journal.pone.0001400.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001400.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001400.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001400.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>Antigenic specificity of recombinant IxACs from <em>I. ricinus</em>.</span></strong></p><a id="article1.body1.sec2.sec12.fig1.caption1.p1" name="article1.body1.sec2.sec12.fig1.caption1.p1"></a><p>Standardised amounts of the seven <em>I. ricinus</em> recombinant IxACs were analysed for antigenic specificity. Panel A. Western blot analysis. The serum from a mouse immunized against IxAC-B1 by genetic immunization followed by a protein boost recognised solely recombinant IxAC B1. M, molecular weight markers (Mark12, Invitrogen). Panel B. Seroneutralization experiments. AP hemolysis assays were conducted with and without the seven recombinant IxACs. 100% hemolysis was obtained in the absence of anticomplement protein (light dotted histogram). Recombinant IxACs alone (dark dotted histogram) or recombinant IxACs plus heat-inactivated sera from mice immunized against IxAC-B1 (anti IxAC-B1, black histogram) or mock immunized mice (anti-PBS, white histogram) were added as indicated. Neutralization of activity as indicated by a recovery of hemolysis, was observed only on IxAC-B1. Upper panels: seroneutralization of IxAC-A subfamily. Lower panel: seroneutralization of IxAC-B subfamily. Error bars represent standard deviations.</p>
<span>doi:10.1371/journal.pone.0001400.g011</span></div><a id="article1.body1.sec2.sec12.p2" name="article1.body1.sec2.sec12.p2"></a><p>The neutralizing potential of these antibodies was also assessed. Standardized amounts of 7 recombinant IxACs from <em>I. ricinus</em> were pre-incubated with heat-inactivated anti-IxAC-B1 serum before assessing their ability to interfere with complement activity. Neutralization of AP inhibition activity as indicated by recovery of RBC lysis was observed only against IxAC-B1 (<a href="#pone-0001400-g011">Figure 11B</a>). Seroneutralization of AP inhibition by IxAC-B1 was not observed with pre-immune sera or with antisera directed to the unrelated protein Iris (not shown).</p>
<a id="article1.body1.sec2.sec12.p3" name="article1.body1.sec2.sec12.p3"></a><p>These data show that an antiserum raised against one member of the IxAC family is able to recognize and functionally inhibit this member alone and no other member of the family. It suggests that immunodominant epitopes of a member of the family are not shared by other members.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The interaction between hard ticks and their hosts is a good example of an ongoing “arms race” between a parasite and its host. As the tick feeds on its vertebrate host for periods of several days it must counteract all the host's defense mechanisms including hemostasis, inflammation and the immune response. This is accomplished by injecting batteries of active proteins in the saliva. Most are coded by multigene families only a few of which have been fully characterized. In this work we have attempted to provide an exhaustive analysis of one such family, the family of salivary inhibitors of the alternative complement pathway in <em>Ixodes ricinus</em>. Our analysis includes: i) a complete or nearly complete inventory of members of the family, ii) a detailed study of its action mechanism: these proteins bind to properdin, a positive activator of complement, thereby preventing early rejection of the tick by the innate response, and iii) investigation of the possible selective pressure causing diversification: sequence diversification comes with antigenic diversity and helps circumvent the host's adaptive immune response.</p>

<h4>A large family of complement inhibitors in the hard tick <em>I. ricinus</em></h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We describe five new anticomplement sequences (AM407396 to AM 407400) from the hard tick <em>Ixodes ricinus</em>. These share less than 40 % amino-acid identity with previously described ISAC of <em>I. scapularis</em> <a href="#pone.0001400-Valenzuela2">[25]</a> and with IRAC I and IRAC II from <em>I. ricinus</em> <a href="#pone.0001400-Daix1">[31]</a>. The new sequences clustered in a distinct group of anticomplement sequences as indicated by phylogenetic analysis (<a href="#pone-0001400-g001">Figure 1</a>). We also proposed the acronym “IxAC” for <em><span class="underline">Ix</span>odes</em> <span class="underline">A</span>nti-<span class="underline">C</span>omplement to the larger family of <em>Ixodes</em> anticomplement proteins related to prototypical ISAC.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>Our analysis provides a reasonably exhaustive and accurate inventory for the tick population under study. This is supported by several lines of evidence: i) 12 RT/PCR inventories performed independently on two different cDNA using 6 different primers pairs consistently yielded the same set of sequences; ii) Sequences identical to previously described IRAC I and IRAC II were recovered; iii) IxAC-B1 and IxAC-B2 were first discovered by serendipidity and then repeatedly found again by RT/PCR analysis of salivary gland cDNA. The existence of additional variants (either alleles or paralogs) in the larger <em>I. ricinus</em> species may be investigated on specimens captured in the field or on specimens from other laboratory colonies.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>We also noticed consistent differences in the frequency of the 7 <em>I. ricinus</em> IxACs in RT/PCR inventories (<a href="#pone.0001400.s004">Table S1</a>). This may be due to the differential expression of IxAC genes in individual ticks (<a href="#pone-0001400-g010">Figure 10</a>). Alternatively, different steady-state levels of messengers may also be responsible.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>The following arguments are consistent with the theory that the 7 <em>I. ricinus</em> IxAC genes are paralogs rather than alleles: i) The 7 <em>I. ricinus</em> IxAC proteins are not very closely related (65% amino-acid identity at most), ii) They could not be grouped together into a single phylogenetic cluster, iii) They were cloned from a restricted tick population, iv) Some individual female ticks expressed up to 3 IxACs simultaneously, indicating the existence of at least two loci.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>Daix et al. <a href="#pone.0001400-Daix1">[31]</a> developed specific monoclonal antibodies that can differentiate IRAC I and IRAC II by immunofluorescence analysis on dissected salivary glands. They observed co-expression of the two proteins in the salivary glands of 12 adult specimens collected in various woodland locations throughout Belgium. The authors concluded that IRAC I and IRAC II are not alleles but rather co-expressed members of a multigene family. However, in our RT/PCR analysis, IRAC I and IRAC II mRNA were detected together only twice in 10 individual specimens from the Neuchâtel breeding colony. This discrepancy may be due to cross detection of other members of the IxAC families by monoclonal antibodies directed against IRAC I and IRAC II.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Phylogenetic analysis of the nucleotide and amino-acid sequences by distance and maximum likelihood methods (<a href="#pone-0001400-g001">Figure 1</a>) indicated that all known <em>Ixodes</em> anticomplement proteins (IxACs) could robustly be grouped into two large groups or subfamilies: IxAC-A and IxAC-B. IxAC-B only contains our five new sequences from <em>I. ricinus</em> whereas IxAC-A contains all the other sequences. IxAC-A can be further subdivided into “ISAC-like” and “Salp9-like” clusters after the name of the founder sequences. Regardless of the method used, placement of IRAC I within the ISAC-like or Salp9-like cluster was not supported by bootstrap analysis. It therefore probably represents a third cluster within the IxAC-A family.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>The presence of IxAC sequences in <em>I. ricinus</em>, <em>I. scapularis</em> and <em>I. pacificus</em> indicated that their last common ancestor and possibly the ancestral <em>Ixodes</em> tick possessed an IxAC-like sequence. The lack of <em>I. scapularis</em> and <em>I. pacificus</em> representatives in IxAC-B is best explained by methodological differences when looking for IxAC sequences in these two species. Indeed, we performed a dedicated family-specific inventory in <em>I. ricinus</em> whereas the <em>I. scapularis</em> and one <em>I. pacificus</em> entries are the result of large-scale untargeted sequencing of large numbers of clones taken at random or PCR screening using unique primers designed from ISAC and Salp20. Dedicated RT/PCR inventories of IxAC sequences in <em>I. scapularis</em> and/or <em>I. pacificus</em> would help resolve this question by showing whether or not the two species express members of IxAC-B.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>IxAC homologs were not detected by RT/PCR analysis of <em>R. appendiculatus</em> salivary gland cDNA or by <em>in silico</em> interrogations of collections of available non-<em>Ixodes</em> (Metastriata) sequences. IxAC homologs, if any, in <em>R. appendiculatus</em> might be too divergent to be amplified by our PCR primers. Besides, the present coverage of Metastriata genomes in public databases may be very limited. On the other hand, our finding is consistent with the absence of reported inhibition of the alternative complement pathway by Metastriata.</p>


<h4>Positive (diversifying) selection of coding sequences</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Since Ohno <a href="#pone.0001400-Ohno1">[41]</a>, gene duplication has been considered to be an important factor in evolution as it leads to the evolution of new gene functions. In higher organisms most genes belong to families of related genes formed by repeated gene duplication events during evolution <a href="#pone.0001400-Hughes1">[42]</a>.Considerable research is currently being carried out in order to understand the forces leading to and shaping multigene families in living organisms <a href="#pone.0001400-Taylor1">[43]</a>.Ticks are particularly suited for this analysis as their salivary proteins are coded by multigene families, a feature probably related to their bloodfeeding lifestyle.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>As far as salivary proteins in general are concerned, the process of gene duplication accompanied by positive selection leading to the acquisition of novel protein functions is documented in snake venom proteins <a href="#pone.0001400-Kordis1">[44]</a>, <a href="#pone.0001400-Zupunski1">[45]</a> and in soft tick salivary proteins <a href="#pone.0001400-Mans2">[46]</a>. In the bloodfeeding diptera <em>Lutzomyia longipalpis,</em> variants of the maxadilan protein retain the same function and biochemical properties <a href="#pone.0001400-Lanzaro1">[47]</a> but have undergone diversifying selection (Lanzaro, personal communication). This is associated with antigenic diversity leading to escape from the host's antibody response <a href="#pone.0001400-Milleron1">[48]</a>, <a href="#pone.0001400-Milleron2">[49]</a>.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>We compared the coding sequences of the seven <em>I. ricinus</em> anticomplement sequences with one another as well as with a putative reconstructed ancestral sequence. We observed that percent identities between IxAC amino-acid sequences were consistently lower than percent identity at the nucleotide level. In addition, dN/dS ratios were consistently higher than 1. We concluded that IxAC coding sequences had been subjected to strong positive selection within the <em>I. ricinus</em> species. In other words, diversification of the IxAC amino-acid sequences was strongly selected for and this was not associated with speciation. This confirms the conclusion of a restricted analysis conducted on only two members of the family: Daix et al. <a href="#pone.0001400-Daix1">[31]</a> observed an excess of non-synonymous changes when they compared IRAC I, IRAC II and anticomplement sequences from <em>I. scapularis</em> and <em>I. pacificus</em> (i.e. members of the IxAC-A subfamily).</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>We examined the type of the selection pressure operating on the IxAC family in <em>I. ricinus</em> by specifically investigating the following possibilities i) structural and biochemical diversity, ii) mechanism and functional diversity, iii) adaptation to host species, iv) stage specificity, v) antigenic diversity.</p>


<h4>Structural and biochemical diversity</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>We first assessed the differences in the biochemical properties of the members of the IxAC family. Predicted and experimental biochemical properties of the seven proteins were analysed and compared to those of ISAC from <em>I. scapularis</em> and ISAC-I from <em>I. pacificus</em>. We observed very similar values for calculated pI and observed and calculated molecular weights. All recombinant proteins were exported in the supernatant as predicted by specialized algorithms. They were highly glycosylated and disulphide bonds were predicted in all of them (this article and ref. <a href="#pone.0001400-Valenzuela2">[25]</a>, <a href="#pone.0001400-Tyson1">[29]</a>). Finally, comparison of the distribution of hydrophobic clusters in the amino-acid sequences predicted identical or very similar folds or tertiary structure in all IxACs.</p>


<h4>Mechanism and function diversity</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>The kinetics of inhibition of the alternative complement pathway in human serum were similar for all 7 recombinant IxACs from <em>I. ricinus</em>. The classical pathway was not affected. This is also a feature of the <em>I. scapularis</em> anti-complement proteins ISAC <a href="#pone.0001400-Valenzuela2">[25]</a> and Salp20 <a href="#pone.0001400-Tyson1">[29]</a> and it has been reported recently for <em>I. ricinus</em> IRAC I and IRAC II by Daix et al. <a href="#pone.0001400-Daix1">[31]</a>. We then investigated the action mechanism of the 7 <em>I. ricinus</em> IxACs. Protein binding and competition experiments suggested that all 7 proteins had the same action mechanism. This is consistent with the similar properties of members of the family and also with the identical or very similar predicted folds or tertiary structures. We conclude that IxAC sequence diversification is not primarily driven by selection for different biochemical properties and tertiary structure, or a change in the roles of the proteins. Minute differences are possible but they remain to be investigated.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Our study of this action mechanism showed that IxAC molecules bind specifically to properdin, preventing its association with C3 and thereby reducing formation of C3 convertase complex to levels observed in the absence of properdin. They also induce destabilization of pre-formed C3bBbP convertase. They consequently inhibit the activation of complement via the alternative pathway. Properdin (factor P) is known to increase ten times the half-life of C3bBb convertase although functional convertase activity may be obtained in its absence <a href="#pone.0001400-Pangburn1">[12]</a>, <a href="#pone.0001400-Schreiber1">[50]</a>, <a href="#pone.0001400-Pangburn2">[51]</a>. The effect of IxAC proteins may therefore be explained by a decreased stability of C3 convertase due to blocking of properdin. This is reminiscent of the effect of monoclonal antibodies that bind to properdin and knock out of the properdin gene in mice, and both lead to inhibition of the AP <a href="#pone.0001400-GuptaBansal1">[19]</a>.To our knowledge this is the first time that direct interaction with properdin is described as a mechanism of complement regulation.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3"></a><p>Two recent reports further emphasize the importance of properdin in the AP: Spitzer et al. <a href="#pone.0001400-Spitzer1">[52]</a> and Kimura et al. <a href="#pone.0001400-Kimura1">[53]</a> showed that properdin can also bind directly to microbes, initiating assembly of C3BbP convertase and complement activation. Whether properdin is a positive regulator or an initiator of the AP, it is a central element and a critical molecular target for inhibitors. Nevertheless, it remains to be seen whether IxACs can also interfere with the binding of properdin to the target surface.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4"></a><p>Ixodid ticks are pool feeders, they dilacerate small blood vessels at the bite site, generating a small haemorrhage or pool of blood in which saliva is injected and from which they pump blood <a href="#pone.0001400-Sonenshine1">[3]</a>. Local inhibition of complement activation in the pool of blood is therefore beneficial for at least two reasons. Firstly, inhibition of the production of pro-inflammatory peptides will help prevent the inflammation response at the bite site; secondly, inhibition of MAC insertion on the mouthparts and midgut epithelium will prevent the destruction of tick tissue by complement factors present in the blood meal.</p>
<a id="article1.body1.sec3.sec5.p5" name="article1.body1.sec3.sec5.p5"></a><p>In order to obtain that effect, ticks must secrete sufficient amounts of the inhibitor. A very rough calculation of the amount required can be made. This must take into account properdin concentration (~5 µg/ml in normal human plasma) <a href="#pone.0001400-Nolan2">[54]</a>, <a href="#pone.0001400-Pangburn1">[12]</a>, the amount of ingested blood (little information is available about this volume which is probably several hundred microliters <a href="#pone.0001400-Sonenshine1">[3]</a>), the amount of secreted saliva (unknown but limited by the amount of ingested blood as blood water is recycled into saliva) and the concentration of IxAC in the saliva. If there is complete recycling of blood water into saliva and an equimolar properdin-IxAC interaction, the IxAC concentration required may be estimated to be approximatively 1 ng/µl. However, the anti-complement effect needs only be local (i.e. in the immediate proximity of the mouth parts and midgut epithelium), and this drastically reduces the amount of complement to be neutralized. Moreover, the IxAC might not mediate the only anti-complement mechanism in <em>Ixodes</em> saliva. Inhibition of the IxAC expression in ticks by RNAi may help reveal additional anticomplement molecules.</p>
<a id="article1.body1.sec3.sec5.p6" name="article1.body1.sec3.sec5.p6"></a><p>Finally our results suggest that one molecule of IxAC may interact with several molecules of properdin. Thus, in the experiment shown in <a href="#pone-0001400-g007">figure 7c</a>, complete inhibition of binding of 200 ng of properdin to C3b was observed with around 25 nm inhibitor, corresponding to an IxAC/properdin molecular ratio of 1/3 to ¼. This is consistent with the finding that properdin is present in the serum as dimer, trimer and tetramer <a href="#pone.0001400-Sun1">[55]</a> forms. One IxAC molecule may therefore interact with one properdin polymer. This hypothesis is open to experimentation and is currently being tested in our laboratory.</p>
<a id="article1.body1.sec3.sec5.p7" name="article1.body1.sec3.sec5.p7"></a><p>The mechanism by which salivary gland extract (SGE) from <em>I. ricinus</em> inhibits the AP has been explored by Lawrie et al. <a href="#pone.0001400-Lawrie2">[39]</a>. They observed inhibition of the cleavage of fB to Bb and C3a production when rabbit erythrocytes were used as activators and complete human serum as a source of complement. Little or no C3b was observed on erythrocytes in the presence of SGE. Reconstitution of the alternative C3 convertase (C3bBb) <em>in vitro</em> from purified C3b, fB and fD (but not properdin) was not affected by SGE. Moreover, SGE had no effect on the cleavage of <sup>125</sup>I-C3 into <sup>125</sup>I-C3b by preformed C3bBb or when it was added simultaneously with C3b, factor B and factor D. These findings are similar to our results and may therefore be fully explained by the action of IxAC proteins in the saliva.</p>
<a id="article1.body1.sec3.sec5.p8" name="article1.body1.sec3.sec5.p8"></a><p>Lawrie et al. <a href="#pone.0001400-Lawrie2">[39]</a> also noticed that SGE provoked the cleavage of a ~5 kDa peptide at the C-terminal end of purified C3 α chains. The product can apparently still be cleaved by preformed convertase to yield C3a. The authors also suggested that the larger fragment may not be able to participate in convertase formation. However, we were unable to reproduce this finding (data not shown). As saliva is a complex mixture of proteins, proteolytic cleavage of C3, as well as other possible additional mechanisms, may also be involved in inhibition of the AP.</p>
<a id="article1.body1.sec3.sec5.p9" name="article1.body1.sec3.sec5.p9"></a><p>Saliva, recombinant ISAC <a href="#pone.0001400-Valenzuela2">[25]</a> and Salp20 <a href="#pone.0001400-Tyson1">[29]</a> from <em>I. scapularis</em> as well as IRAC I and IRAC II from <em>I. ricinus</em> <a href="#pone.0001400-Daix1">[31]</a>, have been previously tested for their ability to inhibit AP activation in experiments using human serum as source of complement and agarose as activating surface. The authors observed inhibition of the deposition of both C3b and Bb and release of pre-bound Bb but not pre-bound C3b from the plates. Release of C3b is not observed because this protein is covalently linked to agarose. Finally, recombinant Salp20 also inhibits the production of C3a and deposition of C3b on the surface of red blood cells in the AP hemolytic assay <a href="#pone.0001400-Tyson1">[29]</a>. We conclude that the novel inhibition mechanism described here is consistent with published data concerning <em>Ixodes</em> tick saliva, salivary gland extracts, or recombinant anticomplement proteins.</p>


<h4>Comparison of IxACs with other anti-complement inhibitors</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Complement is a critical component of innate and adaptive immunity mainly acting through specific cell lysis, the release of potent pro-inflammatory peptides and opsonization of target cells. To prevent tissue damage by over-activation, complement activation is subject to tight regulation by many physiological negative regulators. These include surface and soluble proteins such as Decay-Accelerating Factor (DAF/CD55), Complement Receptor-1 (CR1/CD35), and factor H. They compete with factor B for binding to C3b and facilitate dissociation of the C3bBb complex. Together with the membrane protein MCP/CD46, they also act as cofactors for factor I (fI), a serine protease leading to C3b inactivation by proteolysis. Finally, other membrane proteins such as CR2/CD21, protectin or CD59 protect the host cell membrane from inappropriate complement activation and cell destruction <a href="#pone.0001400-Janeway1">[9]</a>.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>Many pathogens also target components of the complement system, sometimes taking advantage of existing physiological regulation mechanisms. Some pathogenic bacteria such as <em>Borrelia</em>, <em>Neisseria</em>, <em>Streptococcus</em> and <em>Yersinia</em> express receptors that bind host-derived soluble complement regulatory proteins, in particular factor H, FHL-1, and C4b binding protein <a href="#pone.0001400-Kraiczy1">[56]</a>. Certain pathogenic viruses express proteins homologous to vertebrate complement regulators such as vaccinia complement control protein or apparently unrelated functional analogs <a href="#pone.0001400-Finlay1">[57]</a>. Protein NS1 of West Nile Virus recruits soluble fH <a href="#pone.0001400-Chung1">[58]</a>. Surface protein gC of Herpes Simplex Virus binds to C3b and inhibits its association with C5 and properdin <a href="#pone.0001400-Kostavasili1">[59]</a>.</p>
<a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3"></a><p>Parasitic protozoa such as <em>Leishmania</em> spp. and <em>Trypanosoma cruzi</em> are also able to counteract complement activation. In the former, the major surface protein GP63 is a protease that can cleave C3b to inactive iC3b. The latter expresses a 160 kDa homolog of DAF which binds to C3b and C4b and prevents formation of the convertase <a href="#pone.0001400-Sacks1">[60]</a>.</p>
<a id="article1.body1.sec3.sec6.p4" name="article1.body1.sec3.sec6.p4"></a><p>In the soft tick <em>Ornithodoros moubata</em>, lipocalin OmCI binds to component C5 and prevents interaction with C5 convertase or blocks the C5a cleavage site. It prevents generation of C5a and MAC and thus suppresses complement hemolytic activity while preserving the immune clearance and opsonization functions <a href="#pone.0001400-Nunn1">[33]</a>.</p>
<a id="article1.body1.sec3.sec6.p5" name="article1.body1.sec3.sec6.p5"></a><p>Cobra venom factor (CVF) is a homolog of cobra C3. It is a functional analog of C3b as it binds to factor B which is cleaved by factor D to form the CVF:Bb complex. This is a stable convertase that continuously cleaves component C3 and C5, leading to depletion of serum complement components <a href="#pone.0001400-Kock1">[61]</a>.</p>
<a id="article1.body1.sec3.sec6.p6" name="article1.body1.sec3.sec6.p6"></a><p>Finally, complement is also considered as a privileged target for new therapeutic agents. They comprise serine protease inhibitors (as many components of the complement cascade are serine proteases), soluble versions of physiological complement regulators, chemical complement inhibitors, complement receptor antagonists and therapeutic antibodies <a href="#pone.0001400-Ricklin1">[62]</a>. For instance, the anti-C5 humanized antibody or eculizumab which inhibits the generation of both C5b and anaphylatoxin C5a is currently tested in clinical studies against autoimmune inflammatory diseases such as rheumatoid arthritis, glomerulonephritis and lupus erythematosus and has recently been shown to be effective against nocturnal paroxysmal hemoglobinuria <a href="#pone.0001400-Rother1">[63]</a>.</p>
<a id="article1.body1.sec3.sec6.p7" name="article1.body1.sec3.sec6.p7"></a><p>As compared to these inhibitors, IxACs have a completely new action mechanism as they bind and prevent the action of properdin, a factor that has not yet been found to be negatively regulated. The only noticeable structural feature of IxAC proteins is the presence of 4 conserved cysteine residues predicted to make two disulphide bridges. These are also found in repeated, conserved motifs of ~60 residues known as short consensus repeats (SCR's) characteristic of regulators such as fH <a href="#pone.0001400-Reid1">[64]</a>. Otherwise, there was no sequence homology or structural resemblance as indicated by the HCA method (data not shown). Moreover, no experimental determination of the 3D structure has yet been published and database interrogations yielded no hits with known 3 D structures (data not shown). Therefore, IxAC-mediated inhibition of the AP appears to be a novel mechanism both in terms of the molecular interactions involved and the structure of the inhibitor.</p>
<a id="article1.body1.sec3.sec6.p8" name="article1.body1.sec3.sec6.p8"></a><p>What would be the advantages of such a mechanism if IxACs are considered to be putative drugs? First, IxACs target the only physiological positive regulator of complement activation. Secondly, it acts very early in the AP cascade and in any case upstream of most known inhibitors. In addition, as properdin has recently been described to also be an initiator of AP <a href="#pone.0001400-Spitzer1">[52]</a>, <a href="#pone.0001400-Kimura1">[53]</a>, IxACs may have a double action on AP, both on initiation and stabilization. Furthermore, properdin strictly acts at membrane level, mostly by stabilizing preformed C3 convertases complexes. Therefore IxAC inhibitors have a localized, highly targeted action, unlike many putative anticomplement drugs which have a systemic action <a href="#pone.0001400-Mizuno1">[65]</a>. Finally, it acts specifically on the AP, while many drugs on trial target either the classical pathway or, like C3 inhibitors, both the classical and alternative pathways. Even anti-properdin antibodies also have a significant inhibitory action on the classical pathway of complement activation <a href="#pone.0001400-GuptaBansal1">[19]</a>. The specificity of IxACs for the AP has two major consequences. Insofar as clinical treatment is considered, a major concern for anticomplement drugs is that by inducing a complete blockade of the complement system, they increase susceptibility to infections and/or trigger autoimmune-like symptoms <a href="#pone.0001400-Mollnes1">[66]</a>. The detrimental effects of complement inhibition would be significantly lowered by specific inhibition of the sole AP, with no action on most of the CP effects. This application is theoretically feasible as the only critical adverse effect of AP inhibition is an increased susceptibility to neisserial meningitis <a href="#pone.0001400-Ricklin1">[62]</a>.</p>
<a id="article1.body1.sec3.sec6.p9" name="article1.body1.sec3.sec6.p9"></a><p>One limitation we foresee is the case of diseases in which the respective involvement of AP and CP are unknown or intricate. The therapeutical benefits of AP inhibition will then be difficult to evaluate.</p>


<h4>Absence of stage specificity</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>We also investigated the developmental and bloodmeal stage specificity of expression. In databases, <em>I. scapularis</em> IxAC entries are reported from adults and nymphs but no robust “adult-only” or “nymphal-only” cluster appeared in our phylogenetic analysis. Moreover, we found one instance where the same nucleotide sequence was reported in both stages. In <em>I. ricinus</em>, we detected IxAC messengers from both IxAC-A and IxAC-B by RT/PCR in all stages of the life cycle and there was no evidence for stage-specific sequences. However, not all seven IxACs were detected in larvae and nymphs, as we observed expression of only IRAC I, IxAC-B1, B3 and B5 in fed larvae and fed nymphs.</p>
<a id="article1.body1.sec3.sec7.p2" name="article1.body1.sec3.sec7.p2"></a><p>We also investigated the expression of the IxAC repertoire during the bloodmeal. IRAC I, IRAC II, IxAC-B1, B3 and B5 messengers were detected in unfed females and the complete repertoire was detected in salivary glands from pooled females at days 3 and 5 of the bloodmeal.</p>
<a id="article1.body1.sec3.sec7.p3" name="article1.body1.sec3.sec7.p3"></a><p>These results suggest that variability in the IxAC family is not due to the development of proteins specific for a development stage or a phase of the bloodmeal. It is still possible, however, that some members are induced or upregulated in the first days of the bloodmeal or in the adult. Alternatively, the apparent absence of certain messengers in unfed adults and in younger stages may be due to the fact that individual ticks expressed only one or a few IxAC. This may be answered by analyzing larger pools of larvae, nymphs and unfed adult salivary glands.</p>
<a id="article1.body1.sec3.sec7.p4" name="article1.body1.sec3.sec7.p4"></a><p>Finally, variability in IxAC expression by individual adult females was observed: individual adult females each expressed different individual or series (up to three) of IxACs although some variants such IxAC-B1 were expressed more often than others.</p>


<h4>Adaptation to host species</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>A rationale for IxAC variability may be the necessity for the tick to be able to inhibit AP activity from a wide range of hosts. Indeed, <em>I. ricinus</em> is known to infest a very large range of mammals, birds and some reptiles <a href="#pone.0001400-Sonenshine1">[3]</a>, <a href="#pone.0001400-Anderson1">[67]</a> and it is possible that <em>I. ricinus</em> can infest any terrestrial vertebrate it encounters <a href="#pone.0001400-Milne1">[68]</a>, <a href="#pone.0001400-Hoogstraal1">[69]</a>.</p>
<a id="article1.body1.sec3.sec8.p2" name="article1.body1.sec3.sec8.p2"></a><p>Lawrie et al <a href="#pone.0001400-Lawrie1">[24]</a> compared the anticomplement activity of adult salivary gland extract (SGE) from several <em>Ixodes</em> species against a series of birds and mammals. The authors attributed the observed differences in anti-complement activity to host specificity of different tick species.</p>
<a id="article1.body1.sec3.sec8.p3" name="article1.body1.sec3.sec8.p3"></a><p>Recently, Schroeder et al. <a href="#pone.0001400-Schroeder1">[70]</a> compared the anticomplement activity properties of recombinant IRAC I and IRAC II on fresh serum from human, dog, horse, sheep, rat, pheasant and pigeon. They observed inhibition of human, canine, sheep, horse, pig and pheasant serum by both IRAC I and IRAC II. Neither inhibited pigeon or rat serum.</p>
<a id="article1.body1.sec3.sec8.p4" name="article1.body1.sec3.sec8.p4"></a><p>We extended this analysis to the seven IxACs in <em>I. ricinus</em> and a larger and more diverse panel of host species. Dose-dependent inhibition of the AP by all recombinant IxACs was observed in sera from all mammals tested. For birds, only pheasant serum was inhibited, by IRAC II alone. In addition, AP activity in the serum of the snake <em>Elaph guttata</em> was inhibited by IxAC-B4. Our results with IRAC I and IRAC II were identical to those of Schroeder except for pheasant serum where we noted no inhibition with IRAC I. However, the inhibition was very low in their study <a href="#pone.0001400-Schroeder1">[70]</a>. Small experimental differences may explain the different conclusion, i.e. “low level” rather than “absence” of inhibition.</p>
<a id="article1.body1.sec3.sec8.p5" name="article1.body1.sec3.sec8.p5"></a><p>As every IxAC can inhibit the AP in all mammals tested, we conclude that diversification is not due to host specificity within the <em>Mammalia</em> class. Rather, it appears that IxAC evolved as a mammal-specific pathway for AP inhibition. The fact that inhibition of AP activity is only induced by some IxACs in only one bird and one reptile might seem surprising given the very wide specificity of <em>I. ricinus</em>. This is fully explained, though, by the absence of properdin in Birds and Squamates as reported by Nonaka &amp; Kimura <a href="#pone.0001400-Nonaka1">[71]</a>. IRAC II and IxAC-B4 might accidentally bind to other components of the complement system such as C6, C7, C8 and C9 of the MAC, as they also possess repeats of the TSR (TSP1) motif characteristic of Properdin <a href="#pone.0001400-Goundis1">[14]</a>. Therefore, <em>Ixodes</em> ticks might rely on other mechanisms to inhibit the AP or complement effector mechanisms when feeding on most birds and reptiles (e.g. proteolysis of the C-terminus of the C3 α chain described by Lawrie et al. <a href="#pone.0001400-Lawrie2">[39]</a>).</p>
<a id="article1.body1.sec3.sec8.p6" name="article1.body1.sec3.sec8.p6"></a><p>Alternatively, as birds and reptiles have not been studied as exhaustively as mammals, it cannot be ruled out that properdin is present in some species or lineages but not in others. Besides, bird-specific and/or lizard-specific IxAC variants may not yet have been found. All the ticks used in this study have been bred on mice and rabbits in the laboratory for over 20 years from founder specimens initially captured in the surroundings of Neuchâtel in Switzerland. Therefore, they probably have a reduced genetic diversity and have been selected for efficient blood feeding on mammals. Tick specimens collected on infested birds and lizards will help resolve these questions.</p>
<a id="article1.body1.sec3.sec8.p7" name="article1.body1.sec3.sec8.p7"></a><p>Although all recombinant IxACs inhibited the AP in all mammals tested, quantitative differences appeared in the ability of various IxACs to inhibit the AP in a given host and in the ability of a given IxAC to inhibit the AP in different hosts. Sequence differences between the IxACs might result in species specificity. Indeed although the general 3D structure of IxACs appears to have been conserved, small changes might modulate their interaction with properdin. Preliminary analysis of properdin amino-acid sequences from different eutherian mammals (<em>H. sapiens</em>, <em>M. musculus</em>, <em>C. familiaris</em>, <em>E. caballus</em>) indicated a 75 to 79% identity between the mature proteins (B. Couvreur, unpublished). Although properdin is a very highly conserved protein in eutherians, it is conceivable that small differences in amino acid sequences may lead to changes in affinity for the inhibitors. Furthermore, sequence comparison between individual thrombospondin domains suggested that TSR5 is more variable than the other TSR domains (B. Couvreur, unpublished). Because TSR5 is involved in the interaction of properdin with C3b <a href="#pone.0001400-Sun1">[55]</a> and as IxAC displaces this interaction, it is tempting to speculate that TSR5 variability could lead to different affinities for IxAC inhibitors, different degrees of inhibition of C3b binding and differences in the level of inhibition of the AP hemolysis assay.</p>
<a id="article1.body1.sec3.sec8.p8" name="article1.body1.sec3.sec8.p8"></a><p>Alternatively, several other factors might also explain the observed differences between mammalian sera. These include i) different properdin concentrations in the plasma from different species, ii) different concentrations of C3 and other complement components, iii) different relative roles of properdin (stabilizer of C3 convertase versus activator of the AP) in the various species tested.</p>
<a id="article1.body1.sec3.sec8.p9" name="article1.body1.sec3.sec8.p9"></a><p>Whatever the explanation, these differences were not further investigated as they have no impact on the general conclusion of this study.</p>


<h4>Antigenic diversification</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>One of the most frequent countermeasures adopted by parasites against the host's immune system is antigenic variation. It allows the parasite to escape the adaptive response but also to prevent immediate rejection in a new host by an immune memory reaction following a previous contact.</p>
<a id="article1.body1.sec3.sec9.p2" name="article1.body1.sec3.sec9.p2"></a><p>Tick anticomplement proteins are immunogenic to mammals and antibodies are present in the serum of repeatedly infested rabbits and guinea pigs <a href="#pone.0001400-Das1">[28]</a>. Furthermore, we showed here that mice antibodies can neutralize anticomplement activity. siRNA experiments on nymphs have shown that silencing ISAC -like molecules in <em>I. scapularis</em> leads to a dramatic reduction in bloodmeal survival of feeding nymphs <a href="#pone.0001400-Soares1">[27]</a> indicating the importance of anticomplement proteins for the parasite. An immune response to anticomplement proteins may therefore be a major threat to completion of the bloodmeal or even tick survival.</p>
<a id="article1.body1.sec3.sec9.p3" name="article1.body1.sec3.sec9.p3"></a><p>Our finding that members of the IxAC family of proteins in <em>I. ricinus</em> do not share immunodominant epitopes and that individual ticks express different IxAC variants indicate that part of a tick population are probably able to escape an anti-IxAC immune response. Antigenic variability may therefore be considered to be a selective advantage at population level. This is a likely explanation of why IxAC evolved in a multigene family undergoing diversification.</p>


<h4>Possible clinical roles of IxACs</h4>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1"></a><p>Many diseases are caused either by an insufficiency or over-functioning of complement. The former mostly leads to increased susceptibility to infectious agents <a href="#pone.0001400-Ricklin1">[62]</a>. The latter is associated with a large number of diseases including inflammatory disorders (e.g. asthma, sepsis); disorders related to severe tissue injuries (e.g. burn injury, myocardial infarction, ischemia/reperfusion), auto-immune diseases (e.g. lupus erythematosus,glomerulonephritis, rheumatoid arthritis, psoriasis, multiple sclerosis) and degenerative diseases (e.g. Alzheimer's disease) <a href="#pone.0001400-Ricklin1">[62]</a>, <a href="#pone.0001400-Mizuno1">[65]</a>, <a href="#pone.0001400-Mollnes1">[66]</a>. In some cases, the involvement of complement is only circumstantial as activated components have been localized at the sites of clinical disease. In other cases the involvement of complement was deduced by the analysis of inhibitors deficiencies in animal models. For example, the AP is known to be involved in disorders such as ischemia-reperfusion injury (I/R), systemic lupus erythematosus (SLE), asthma and rheumatoid arthritis (RA) <a href="#pone.0001400-Holers1">[72]</a>. In mouse models of diseases, the fB-/- genotype is associated with no or reduced alterations (e.g. reduced deposition of C3b in kidney I/R injury) <a href="#pone.0001400-Thurman1">[73]</a> confirming the role of AP and C3 convertase.</p>
<a id="article1.body1.sec3.sec10.p2" name="article1.body1.sec3.sec10.p2"></a><p>IxAC molecules may have a positive clinical effect in conditions where inappropriate complement activation involves the binding of properdin and C3b. This will lead to a reduced production of pro-inflammatory molecules, reduced deposition of C3b and reduced cell lysis due to deposition of the MAC. Again, in most complement-related disorders, the relative contribution of the AP and CP is not known. More information about these diseases is needed before IxAC can be used as candidate therapy. Therefore on the one hand, more information about these diseases is needed before the beneficial effects of IxAC can be evaluated. On the other hand, however, AP-specific inhibitors could not be restricted to diseases initiated by AP activation as, in many cases, the alternative pathway loop is triggered as a secondary effect to amplify the activation, irrespective of the initial trigger (AP, CP or lectin pathway). A definite indication for a putative role of IxACs as anticomplement therapy must await tests in animal models of candidate diseases. This is the subject of ongoing experiments in the laboratory.</p>
<a id="article1.body1.sec3.sec10.p3" name="article1.body1.sec3.sec10.p3"></a><p>IxACs may also be used as tools to investigate the role of complement activation in certain diseases. As its causal role is only suspected and not proved in many of these disorders, a specific inhibitor of complement activation may be used in animal models to determine this role. This was recently achieved in knockout mice deficient in some complement components. More specifically and interestingly, IxAC may make it possible to distinguish the respective contributions of the AP and CP in pathophysiological mechanisms.</p>


<h4>Concluding remarks</h4>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1"></a><p>To summarize, in this study we described five new anticomplement proteins from the hard tick <em>Ixodes ricinus</em> and we report the first specific in-depth analysis of a tick multigene family. Our data suggest that this inventory of anticomplement sequences is complete for the investigated population. This provided an opportunity to study possible gene diversification mechanisms related to the acquisition of a bloodfeeding lifestyle by hard ticks. Diversifying selection was shown to operate. Nevertheless, all proteins had very similar biochemical properties. They all inhibited formation of C3 convertase by specifically binding to properdin. Differences in host specificity or patterns of expression could not account for sequence diversity. However, sequence divergence was associated with antigenic differences. As individual ticks do not express the same range of anticomplement proteins, certain individuals in a population would be able to escape the host immune reaction.</p>
<a id="article1.body1.sec3.sec11.p2" name="article1.body1.sec3.sec11.p2"></a><p>This is the first time that inhibition of the alternative pathway <em>via</em> the specific binding of a positive regulator is described. This is an efficient way of blocking the innate immune system and preventing early rejection of the tick by the host. Recombinant IxACs may therefore be useful tools for the investigation of the role of properdin in physiological and pathophysiological mechanisms. Because activation of the alternative complement pathway is involved in major human diseases such as ischemia-reperfusion injury, systemic lupus erythematosus (SLE), asthma and rheumatoid arthritis (RA), AP inhibitors may prove to be therapeutically beneficial <a href="#pone.0001400-Holers1">[72]</a>. To date, only two complement inhibitors have reached the pre-clinical testing stage: recombinant soluble complement receptor 1 (sCR1) <a href="#pone.0001400-Weisman1">[74]</a> and anti-C5 monoclonal antibodies <a href="#pone.0001400-Thomas1">[75]</a>. However, they do not act specifically on AP <a href="#pone.0001400-GuptaBansal1">[19]</a>. Selective inhibition of the AP by properdin inhibitors such as IxACs may therefore be used to treat pathological effects of uncontrolled complement activation by this pathway, without compromising the classical and lectin pathways.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Tick material</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Specimens of <em>Ixodes ricinus</em> were raised in the tick breeding facility at the <em>Institut de Zoologie</em>, <em>Université de Neuchâtel</em> (Switzerland). Founders of the colony were initially collected in woodlands near Neuchâtel and have been maintained on rabbits (adults and nymphs) and SWISS mice (larvae) for over 20 years. Specimens of <em>Rhipicephalus appendiculatus</em> strain Mugaga were a kind gift of Dr Maxime Madder (Animal Health department, Prince Leopold Institute of Tropical Medicine, Antwerp, Belgium). The colony originated from individuals collected in East Africa. It was routinely maintained on rabbits. All specimens were devoid of transmissible pathogens.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>Pairs of salivary glands from <em>I. ricinus</em> were dissected from i) 70 adult female specimens at day 5 of the bloodmeal; ii) 25 females at day 3 of the bloodmeal, iii) 25 unfed females, iv) 10 individual females at day 5 of the bloodmeal. Pools of 25 fully gorged larvae and 25 fully gorged nymphs were also prepared. In addition, salivary glands were prepared from 25 adult female and 25 adult male <em>R. appendiculatus</em>.</p>


<h4>Nucleic acid extraction and analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>PolyA+ RNA was extracted from tick material using the Micro-Fasttrack 2.0 (Invitrogen).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>All polyA+ RNA were reverse transcribed with Superscript III (Invitrogen) in the presence of RNaseOUT ribonuclease inhibitor (Invitrogen) using Not1-d(T)18 bifunctional primer (Amersham Biosciences). PolyA+ RNA from pooled day 5 salivary glands was also reverse transcribed with Superscript II (Invitrogen) using the GeneRacer oligodT reverse transcription primer (Invitrogen).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>5′ and 3′ RACE experiments were performed using the Generacer kit (Invitrogen).</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4"></a><p>For restriction analysis, sequencing and small-scale transfection experiments, recombinant plasmid DNA was extracted using Genelute Plasmid Miniprep Kit (Sigma).</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5"></a><p>Plasmid constructs were sequenced using universal sequencing primers at Biovallée A.S.B.L. (Gosselies, Belgium) and at Genome Express (Meylan, France).</p>
<a id="article1.body1.sec4.sec2.p6" name="article1.body1.sec4.sec2.p6"></a><p>All commercial reagents and kits were used according to the manufacturers' instructions.</p>


<h4>RT/PCR inventories of IxAC sequences in <em>I. ricinus</em></h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>PolyA+ RNA extracted from 70 pooled pairs of salivary glands from <em>I. ricinus</em> females on day 5 of the bloodmeal was reverse transcribed using two different sets of reverse transcriptase and oligodT primers. cDNA1 was generated with Superscript II using the GeneRacer oligodT reverse transcription primer (Invitrogen). cDNA2 was produced with Superscript III (Invitrogen) in the presence of RNaseOUT ribonuclease inhibitor (Invitrogen) using Not1-d(T)18 bifunctional primer (Amersham Biosciences). Both were used for RT/PCR experiments designed to make an inventory of the family of anticomplement proteins in <em>I. ricinus</em>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Upstream primers were designed manually from the 5′ end of coding sequences of anticomplement proteins available early in this project from <em>I. ricinus</em> (IRAC I and II, AM407396, AM407397: <em>IRI</em> primer) or from all available <em>Ixodes</em> spp. sequences (IRAC I and II, AM407396, AM407397, ISAC, Isac-1: <em>IXO</em> primer). Commercial downstream primers <em>Generacer 3′</em> (Invitrogen) and <em>Not1</em> (Amersham Biosciences) are designed to anneal to the 5′ end of the modified oligodT primers used for reverse transcription. In addition, downstream primers <em>UTR1</em> and <em>UTR2</em> were designed from the 3′ UTR of anticomplement sequences. All primers are listed in <a href="#pone-0001400-t001">Table 1</a>. They were purchased from Eurogentec (Liège, Belgium) and diluted to 10 µM in MillliQ water.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Six primer pairs were thus constituted: i) <em>IRI</em>-<em>generacer3′</em>, ii) <em>IXO</em>-<em>generacer3′</em>, iii) <em>IRI</em>-<em>Not1</em>, iv) <em>IXO</em>-<em>Not1</em>, v) <em>IXO</em>-<em>UTR1</em>, vi) <em>IXO</em>-<em>UTR2</em>. Primer pairs (i) and ii) were used on cDNA1 whereas primer pairs (iii) to (vi) were used on cDNA2 (<a href="#pone.0001400.s004">Table S1</a>). PCR amplification experiments were performed with the Expand High Fidelity Plus PCR System (Roche) in a final volume of 50 µl of commercial buffer containing 1.5 mM MgCl<sub>2</sub>. PCR cycling parameters were as follows: 2 min denaturation at 94°C, then 30 cycles of 30 sec at 94°C, 30 sec annealing at 61 to 65°C (depending on the Tm of the primer pairs), 1 min at 72°C, then final incubation for 7 min at 72°C in a PTC-100 Programmable Thermal Controller (MJ Research). Annealing temperature was set at the lower Tm for a given primer pair minus 5°C.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>PCR products were routinely purified on polyacrylamide gels and inserted into both the pCRII and the pCDNA3.1/V5-His vectors (Invitrogen) by the TA method. Top10F' chemically competent <em>E. coli</em> cells (Invitrogen) were then transformed and plated onto LB-Agar-Ampicillin solid medium. Plasmid DNA from recombinant clones was analysed by <em>Eco</em>RI (pCRII) or <em>Bam</em>H1–<em>Xba</em>1 (pcDNA3.1/V5-His-TOPO) restriction and inserts ≥600 bp were sequenced on both strands.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>Therefore a total of 2 different reverse transcription experiments, 6 different PCR amplifications and 12 ligations were performed during the course of RT/PCR inventories of anticomplement sequences in salivary glands of <em>I. ricinus</em> females.</p>


<h4>RT-PCR analysis of expression of individual IxACs</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Pairs of PCR primers (<a href="#pone.0001400.s006">Table S3</a>) were designed from an alignment of the <em>I. ricinus</em> IxAC coding sequences to specifically amplify each of the family members one at a time, each primer pair generating a product of different size. They were synthesized by Sigma-Genosys and purified by HPLC. The specific messengers were searched by RT/PCR in polyA+ prepared from i) salivary glands of individual adult females at day 5 of the bloodmeal, ii) pooled salivary glands at day 0 of the bloodmeal (unfed females), iii) pooled salivary glands of females at day 3 of the bloodmeal, iv) pooled nymphs and v) pooled larvae.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>PCR was performed using Taq polymerase in a 50 µl reaction volume according to the manufacturer's instructions (Roche Biochemicals). Except for IxAC-B4, PCR cycling conditions were as follows: 2 min denaturation at 94°C then 40 cycles of 30 sec at 94°C, 30 sec annealing at 57°C, 30 sec elongation at 72°C followed by a final 7 min. elongation at 72°C on PTC-100. This set of PCR conditions had to be slightly modified to amplify the specific IxAC-B4 fragment: 45 cycles and annealing at 60°C. PolyA+ submitted to reverse transcription without the actual RT enzyme were used as negative control template.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>In parallel, family-specific reverse primers were designed from UTR or coding sequences to amplify the full coding sequences of members of IxAC-A (<em>IXO</em>-<em>CDSREV1</em>) and IxAC-B (<em>IXO</em>-<em>CDSREV2</em>) (<a href="#pone-0001400-t001">Table 1</a>). They were applied to cDNA from pooled nymphs and pooled larvae using the Expand-HF+ system for PCR amplification as described above.</p>


<h4>Expression and quantification of recombinant proteins</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The coding sequences for <em>I. ricinus</em> IxACS were amplified by PCR and inserted into the vector pCDNA3.1/V5-His-TOPO (Invitrogen). Upstream primers were designed from the first 20 nt of each coding sequences. Nucleotides surrounding the ATG were changed to ACC<span class="underline">ATG</span>G (IRAC I, IxAC-B1 to B5) or GCC<span class="underline">ATG</span> (IRAC II) according to Kozak's consensus for efficient initiation of translation <a href="#pone.0001400-Kozak1">[76]</a>. Downstream primers were designed from the 3′ end of the coding sequences omitting the TGA stop codon so as to create IxAC-V5His chimeras.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>The coding sequence for <em>Rhipicephalus appendiculatus</em> histamine-binding protein 2 (RaHBP2) was also amplified from salivary gland cDNA of adult <em>R. appendiculatus</em> females using PCR primers designed from the original published sequence (U96081) <a href="#pone.0001400-Paesen1">[77]</a> and inserted into vector pCDNA3.1V5His. Throughout this study, recombinant RaHBP2 was used as a negative control.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>Subconfluent 293T cells in 35 mm diameter wells (Orange Scientific) were transfected with 2 µg plasmid DNA and 6.0 µl Fugene 6 (Roche Biochemicals) in Dulbecco's modified Eagle's medium (DMEM, Invitrogen) without FCS. The medium was harvested after 72h. Pooled supernatants were cleared by centrifugation, concentrated ~50-fold by filtration on 30 kDa cut-off membranes (Millipore), dialyzed against GVB or VB buffers, and finally stored at –80°C in 40 µl aliquots.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>Concentrated culture supernatants were analyzed by western blotting on a Hybond ECL membrane (GE healthcare) using an anti-V5 primary antibody (Invitrogen), an IgHRP conjugate as secondary antibody and the ECL detection reagent (GE healthcare) following the manufacturer's instructions. Autoradiogram signals were quantified with ImageQuant TL Software (GE Healthcare). The relative amounts of protein were adjusted by diluting the most concentrated IxAC protein to the level of the less concentrated one. After normalization, new western blot analyses showed not more than 2.5-fold differences in protein concentrations.</p>
<a id="article1.body1.sec4.sec5.p5" name="article1.body1.sec4.sec5.p5"></a><p>Samples were also subjected to N-deglycosylation using N-glycosidase F (New England Biolabs) using conditions recommended by the manufacturer.</p>
<a id="article1.body1.sec4.sec5.p6" name="article1.body1.sec4.sec5.p6"></a><p>The coding regions of IRAC II and IxAC-B1 were amplified by PCR (94°C for 30 s, 56°C for 30 s, 72°C for 1 min.; 30 cycles) using ExTaq DNA Polymerase (Taqara). The PCR product was inserted into the pBlueBac4.5/V5-His Topo vector (Invitrogen) in frame with the coding sequence of the V5 and His epitopes at the C-terminus. Recombinant baculoviruses were generated by recombination between pBlueBac/IxAC and Bac-N-Blue linear DNA virus (Invitrogen). Recombinant viruses were selected and amplified according to the manufacturer's instruction. Sf9 cells were infected with a high-titer stock of recombinant baculovirus and were incubated for 72 hours at 27°C in Sf900 II serum-free medium (Invitrogen). Recombinant IxACs proteins were purified from the cell culture supernatant by affinity chromatography on a His-Trap column (GE Healthcare). The proteins were recovered in 50 mM NaH2PO4 buffer (pH 7.5) containing 300 mM NaCl and 50 mM of imidazole. In experiments with purified proteins, we used protein Iris, a serpin from the salivary gland of <em>I. ricinus</em> <a href="#pone.0001400-Prevot1">[78]</a>, as a negative control because it was also expressed in the baculovirus/Sf9 and purified in the same manner.</p>


<h4>Computer-assisted analysis of sequences and database interrogations</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Probable cellular targeting and eukaryotic leader peptide cleavage site were identified in amino-acid sequences with the Target P and Signal P programs respectively <a href="#pone.0001400-Emanuelsson1">[79]</a>. N-linked glycosylation and O-GalNAc (mucin type) glycosylation sites were predicted with the NetNGlyc 1.0 and NetOGlyc 3.1 <a href="#pone.0001400-Julenius1">[80]</a> programs respectively. An hydrophobic anchor at the C-terminal end of the peptide sequence was searched for with the TMHMM v. 2.0 <a href="#pone.0001400-Krogh1">[81]</a> program that predicts transmembrane helices in proteins. All were used online at the Center for Biological Analysis of the Technical University of Denmark (CBS prediction servers: <a href="http://www.cbs.dtu.dk/services/">http://www.cbs.dtu.dk/services/</a>). Calculation of molecular weight and isoelectric point were performed with the Pepstats program from EMBOSS online at the European Bioinformatics Institute (<a href="http://www.ebi.ac.uk/emboss/pepinfo/">http://www.ebi.ac.uk/emboss/pepinfo/</a>). Putative S-S links were searched with program Disulfind <a href="#pone.0001400-Vullo1">[82]</a> at <a href="http://cassandra.dsi.unifi.it/disulfind/">http://cassandra.dsi.unifi.it/disulfind/</a>.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>Tick anticomplement amino-acid sequences were submitted to Hydrophobic Cluster Analysis (HCA) <a href="#pone.0001400-Gaboriaud1">[83]</a>, <a href="#pone.0001400-Callebaut1">[84]</a>. Briefly, amino-acid sequences in an alignment are compared for the overall distribution of hydrophobic clusters, their sizes, shapes and orientations. They can also be compared for possible particular structures induced by specific residues.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>The non-redundant, EST, GSS and PDB databases were interrogated online at the NCBI server (<a href="http://www.ncbi.nlm.nih.gov/BLAST/">http://www.ncbi.nlm.nih.gov/BLAST/</a>) using Blast family programs. Nucleotide and amino acid sequences of ISAC, IRAC I, IxAC-B1 were used as queries. We also interrogated the preliminary releases of the genome projects for non-<em>Ixodes</em> hard ticks <em>Amblyomma variegatum</em>, <em>Boophilus microplus</em>, <em>Rhipicephalus appendiculatus</em> online at the Institute for Genomic Research (<a href="http://tigrblast.tigr.org/tgi">http://tigrblast.tigr.org/tgi</a>) and <em>A. americanum</em> available at the University of Oklahoma cDNA Blast Server (<a href="http://www.genome.ou.edu/tick.html">http://www.genome.ou.edu/tick.html</a>).</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Coding sequences were aligned using ClustalW under the Mega3 package to obtain inframe alignment of codons. Minor manual adjustments were made at the 3′ end (C-terminus) of the alignment. The Genedoc package <a href="#pone.0001400-Nicholas1">[85]</a>(<a href="http://www.psc.edu/biomed/genedoc/">http://www.psc.edu/biomed/genedoc/</a>) was used to visualize alignments and calculate percent identity of nucleotide and peptide sequences. Percent similarity of peptide sequences was also calculated with Genedoc using a Blosum 62 matrix. For phylogenetic analysis by the distance methods, we used programs in the Phylip 3.65 package (<a href="http://evolution.gs.washington.edu/phylip.html">http://evolution.gs.washington.edu/phyli​p.html</a>). The substitution model was F84 for nucleotide sequences and the JTT matrix for amino-acid sequences distance analysis. The PHYML 2.4.4 package <a href="#pone.0001400-Guindon1">[86]</a> (<a href="http://atgc.lirmm.fr/phyml/">http://atgc.lirmm.fr/phyml/</a>) was used for maximum likelihood analysis. We used the HKY model for nucleotide sequence analysis and the JTT model for amino-acid sequences analysis. Transition to transversion ratios and proportions of invariable sites were estimated from the dataset. No γ distribution of rates among sites was applied. Bootstrap analysis was performed on 1000 replicates of the initial datasets. Trees were visualized with Treeview 1.6.6 <a href="#pone.0001400-Page1">[87]</a>. Phylogenetic analyses were repeated on amino-acid and nucleotide alignments with or without the leader peptide sequences.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>We used the Ancescon package to reconstruct a probable ancestral coding sequence <a href="#pone.0001400-Cai1">[88]</a> (<a href="http://protevo.eb.tuebingen.mpg.de/toolkit/index.phpviewancescon">http://protevo.eb.tuebingen.mpg.de/toolk​it/index.phpviewancescon</a>) from an alignment of the coding sequences for <em>I. ricinus</em> IxAC mature proteins. The dN and dS values were calculated using the Nei-Gojobori method as implemented in the MEGA 3.1 package <a href="#pone.0001400-Kumar1">[89]</a> (<a href="http://www.megasoftware.net/">http://www.megasoftware.net/</a>).</p>


<h4>Serum samples</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Complement assays were conducted with freshly prepared sera from the mammals, birds and squamates (snakes and lizards) listed in <a href="#pone-0001400-t005">Table 5</a>. All animals were healthy and non-immunized. Except for red deer, they had been maintained in confined environments and there was no evidence or history of tick bites.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2"></a><p>Fresh blood samples were left to clot at room temperature for 2 to 6 hours. The sera were separated from the clot by centrifugation. They were then aliquoted and stored at –80°C. Hemolysed samples were discarded.</p>
<a id="article1.body1.sec4.sec8.p3" name="article1.body1.sec4.sec8.p3"></a><p>Human sera were obtained from four healthy male volunteers, Beagle dog (<em>Canis familiaris</em>) sera from three individuals, sheep (<em>Ovis aries</em>) sera from five individuals, pig (<em>Sus domesticus</em>) sera from two individuals, calf (<em>Bos taurus</em>) sera from three individuals, deer (<em>Cervus elaphus scoticus)</em> sera from three individuals. Ten female Balb/c mice (<em>Mus musculus</em>) purchased from Harlan Netherlands were bled by retro-orbital puncture. Chicken (<em>Gallus gallus</em>) sera were obtained and pooled from four individuals, pheasant (<em>Phasianus colchicus)</em> sera from five birds, domestic turkey (<em>Meleagris gallopavo</em>) serum from one individual, pigeon (<em>Columba liva</em>) sera from five individuals, lizard (<em>Tropidurus torquatus)</em> sera from two individuals, snake sera from one individual (<em>Boa constrictor</em>) and three individuals <em>(Elaph guttata).</em></p>


<h4>Assay of the alternative complement pathway (AP)</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Recombinant proteins were assessed for their capacity to inhibit the alternative complement pathway (AP) according to Giclas <a href="#pone.0001400-Giclas1">[90]</a> on red blood cells (RBC) from naïve healthy female New Zealand White rabbits. Briefly, fresh sera were diluted in gelatin-veronal-EGTA buffer (GVB) in microwell plates and washed RBCs were added. The surface of rabbit erythrocytes activated the alternative complement pathway leading to their lysis and release of hemoglobin in the buffer. After 60 min incubation at 37°C, supernatants were recovered to measure absorbance at 415 nm with a Model 680 microplate reader (Biorad). The volume of serum causing 50% hemolysis (AH50 value) was then determined by serial dilutions and used for further tests.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>1 to 10 µl of serum were introduced in the test depending of the host species considered. 100% lysis control consisted in total hemolysis produced by incubating 25 µl of MilliQ water. Background level (no hemolysis) was determined by incubating the erythrocytes in GVB buffer alone (without added serum). Each experimental point was done in triplicate and experiments were performed at least twice, by different investigators.</p>
<a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>In order to test the inhibitory effect of the new proteins, up to 10 µl of standardized supernatant (see above) were introduced in the AP test. The inhibitor was serially diluted in a final volume of 25 µl GVB in the presence of AH 50 volumes of the host serum under consideration. The assay was then performed as described above. Percent inhibition of hemolysis was then calculated with the equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.e001&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec9.p4" name="article1.body1.sec4.sec9.p4"></a><p>The kinetics of hemolysis inhibition were also investigated. The hemolytic AP assay was conducted as described above except that we added 200 ng of purified recombinant IRAC II, IxAC-B1 or unrelated protein IRIS to wells 10 minutes after the addition of RBC. Optical density at 415 nm was then taken every 5 or 10 minutes.</p>


<h4>Assay of the classical pathway (CP)</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>Recombinant proteins were also tested for inhibition of the classical complement pathway (CP) essentially as described by Colligan <a href="#pone.0001400-Colligan1">[91]</a>. Ready-to-use reagents were purchased from Institut Virion\Serion GmbH (Würtzburg, Germany). They included sheep erythrocytes pre-coated with rabbit anti-sheep RBC antibodies and Veronal Buffer pH 7.3 (VB) containing NaCl, CaCl<sub>2 </sub>and MgCl<sub>2</sub>. Briefly, diluted serum was incubated in the presence of antibody-coated sheep RBCs in microplates. Immune complexes on the surface of RBCs activate the classical pathway of complement leading to lysis and release of hemoglobin. As in the AP assay, absorbance at 415 nm in the supernatant is proportional to the amount of lysed RBC. Pooled human serum was first titrated to determine the volume that produces 50 % hemolysis (CH50 value). Starting with 10 µl, standardized amounts of recombinant proteins were diluted two-fold in VB buffer containing the equivalent of 0.8 µl human serum per test (total volume 25 µl). Pre-coated sheep erythrocytes were then added and the reaction performed as described above. Results were expressed as percent inhibition of hemolysis as for the AP pathway.</p>


<h4>Western blot analysis of fB cleavage and C3a formation in the AP assay</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>The generation of peptide C3a from C3 and the cleavage of factor B in human serum during the AP assay in the presence of <em>I. ricinus</em> IxACs were assessed by western blot analysis as described by Lawrie et al. <a href="#pone.0001400-Lawrie2">[39]</a>. 10 µl of supernatant from AP assays conducted in the presence of <em>I. ricinus</em> IxAC or control protein RaHBP2 were analyzed by denaturing SDS/PAGE (C3 cleavage) or by non-denaturating PAGE (fB cleavage). Purified factor B was also included as a control in the analysis. Briefly, 10% acrylamide SDS/PAGE in tris–tricine buffer were run according to standard methodology. 10% acrylamide non-denaturating gel electrophoresis was conducted as described except that SDS and β-mercaptoethanol were omitted in all buffers. The material was blotted onto nitrocellulose sheets according to standard methodology. Rabbit anti-human C3a serum (Calbiochem, dilution 1:5000) or goat anti-human factor B serum (Quidel, dilution 1:1000) were used as primary antibodies. Anti-rabbit IgG or Anti-goat IgG horseradish peroxidase conjugates (Promega, dilution 1:5000) were used as secondary antibodies. Blots were developed using the chemiluminescence substrate, ECL+ kit from Amersham (Bucks, UK) according to the manufacturer's instructions. Membranes were then exposed to KODAK X-ray film.</p>


<h4>Enzyme-linked Immunosorbent assay for measuring the binding of recombinant IxAC to components of C3 convertase</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>The binding of the seven recombinant <em>I. ricinus</em> IxAC proteins and control protein RaHBP2 to components of human C3 convertase was assessed using an enzyme-linked immunosorbent assay (ELISA). In some experiments, we also used recombinant IRAC II, IxAC-B1 and control protein Iris purified from the Baculovirus/Sf9 expression system.</p>
<a id="article1.body1.sec4.sec12.p2" name="article1.body1.sec4.sec12.p2"></a><p>Wells in 96-well polystyrene microtiter plates (Nunc) were coated with 200 ng of purified C3, C3b, factor B, factor D or properdin (Calbiochem) in 100 µl pH 7.4 phosphate-buffered saline (PBS, Invitrogen) overnight at 4°C. Wells were washed three times for 5 min. with washing buffer (8.1 mM Na<sub>2</sub>HPO<sub>4</sub>; 1.8 mM NAH<sub>2</sub>PO<sub>4</sub>; 0.05% Tween 20; 25 mM NaCl; 10 mM MgCl<sub>2</sub>) and blocked with PBS-Tween-BSA buffer (1% BSA, 0.1 % Tween 20 in PBS pH 7.4) for 1h at 37°C. Increasing amounts of recombinant <em>I. ricinus</em> IxACs and control proteins in 50 µl sample buffer (8.1 mM Na<sub>2</sub>HPO<sub>4</sub>; 1.8 mM NAH2PO4; 4% BSA; 0.05% Tween 20; 75 mM NaCl; 10 mM MgCl<sub>2</sub>) were added to the wells. After incubating for 1 h at 37°C, wells were washed 3 times with washing buffer. 100 µl antibody buffer (8.1 mM Na<sub>2</sub>HPO<sub>4</sub>; 1.8 mM NAH<sub>2</sub>PO<sub>4</sub>; 4% BSA; 0.05% Tween 20; 25 mM NaCl; 10 mM MgCl<sub>2</sub>) containing a mouse anti-V5 antibody (Invitrogen, dilution 1:5000) were added to the wells. Plates were incubated for 1h at 37°C then washed again three times. Horseradish peroxidase-conjugated anti-mouse IgG antibodies (Promega, diluted 1:7500) in 100 µl antibody buffer were incubated for 1h at 37°C. After 3 washes, 50 µl of 3,3′,5,5′ –Tetramethylbenzidine (TMB, Sigma) substrate was added. The reaction was stopped by adding 50 µl H<sub>2</sub>SO<sub>4</sub> 0.2 N. Optical density at 450 and 630 nm was measured with a Model 680 microplate reader (Biorad). In a first series of experiments we tested the binding of purified recombinant IRAC II and IxAC-B1 and unrelated protein IRIS to C3 convertase components. In the second series we tested standardized amounts of the 7 <em>I. ricinus</em> IxAC and control protein RaHBP2 as expressed in the supernatant of transfected 293T cells (see above) on purified properdin alone.</p>
<a id="article1.body1.sec4.sec12.p3" name="article1.body1.sec4.sec12.p3"></a><p>We also assessed the effect of recombinant IRAC II, IxAC-B1 and unrelated protein IRIS on the binding of Properdin to C3b. Wells were coated with 150 ng of C3b. 200 ng factor P and increasing concentrations of IRAC II, IxAC-B1 or unrelated protein IRIS in sample buffer were added simultaneously to the wells. The amount of bound properdin was estimated using a primary mouse monoclonal antibody to factor P (diluted 1:2000; Quidel) and horseradish peroxidase-conjugated anti-mouse IgG antibodies (diluted 1:7500; Promega) using the above-described ELISA method.</p>


<h4>Enzyme-linked immunosorbent assays for assessing the effect of IxACs on the formation and stability of C3 convertase</h4>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1"></a><p>We assessed the effect of recombinant IxAC proteins on the formation and stability of C3 convertase <em>in vitro</em> using the above-described ELISA method. 96-well polystyrene microtiter plates were first coated with 150 ng of C3b per well. The effect of IxAC proteins on the formation of C3 convertase was assessed by adding 200 ng of factor B, 20 ng of factor D, 200 ng of factor P in sample buffer to the wells together with increasing concentrations of IRAC II, IxAC-B1 or Iris. Plates were incubated for one hour at 37°C before antibody detection of fB or P. To assess the effect of IxACs on the stability of the C3 convertase, C3b-coated wells were incubated for 1 h with 200 ng of factor B, 20 ng of factor D, 200 ng of factor P in sample buffer. The wells were washed three times before deposition of IRAC II, IxAC-B1 or IRIS proteins. The plates were further incubated for one hour at 37°C. Anti-factor P (dilution 1:2000) and anti-factor B (dilution 1:1000) were used as primary antibody to detect bound proteins as described above. Results are expressed as % bound fB or properdin using the following formula:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.e002&amp;representation=PNG" class="inline-graphic"></span><br></p>
<a id="article1.body1.sec4.sec13.p2" name="article1.body1.sec4.sec13.p2"></a><p>Background values were obtained by measuring OD values of C3-coated plates revealed with anti-fB or anti-properdin antibodies. OD max was measured by performing the test without added IxAC.</p>
<a id="article1.body1.sec4.sec13.p3" name="article1.body1.sec4.sec13.p3"></a><p>In a complementary experiment, we compared the effect of IRAC I, IxAC-B1 and the control protein Iris on the formation of C3 convertase in the presence (C3bBbP) or absence of properdin (C3Bb). 200ng of factor B, 20 ng of D, with or without 20 ng of properdin, and 200 ng of IRAC I, IxAC-B1 or control protein Iris were added to microtiter wells pre-coated with 150 ng purified C3b. The reaction was stopped at 0, 20, 40 and 60 min. and bound factor B or properdin were detected with the respective specific antibodies using the above-described ELISA method.</p>


<h4>Enzyme –linked immunosorbent assays for measuring C3 and factor B deposition on agarose-coated wells</h4>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1"></a><p>The ability of <em>I. ricinus</em> IxACs to alter AP activation by a solid phase was assessed by agarose ELISA as described by Valenzuela et al. <a href="#pone.0001400-Valenzuela2">[25]</a> with minor modifications. Deposition of C3b and fB on agarose-coated microplates was used to measure AP activation and C3 convertase formation. Briefly, wells of polystyrene microplates (Nunc) were coated with 100 µl of 0.1% melted agarose in water and incubated at 37°C for 48 h. Wells were then incubated with 50 µl of 200 mM HEPES, pH 7.4, 150 mM NaCl, 5 mM EGTA, 2 mM MgCl<sub>2</sub>, and 10% human serum for 90 min at 37°C. After 30, 45 or 60 minutes, 200 ng of purified IRAC II or IxAC-B1 was added to the plates. The reaction was stopped by washing five times with 200 µl of HEPES saline containing 2 mM MgCl<sub>2</sub> and 10 mg/ml bovine serum albumin (HEPES-BSA buffer). Bound C3b and fB were revealed by Elisa. Goat anti-human C3 serum (Quidel, diluted 1:5000) or goat anti-human fB serum (Quidel, diluted 1:1000) used as primary antibodies were incubated for 1 h at 37°C in HEPES-BSA buffer. Wells were washed 5 times with 200 µl HEPES-BSA and IgG anti-goat peroxidase-conjugate (Promega, diluted 1:5000) was then added as secondary antibody and incubated for 1 h at 37°C in HEPES-BSA. Plates were washed twice with HEPES-BSA, then twice with HEPES containing 10% Tween 20, and finally twice with HEPES buffer alone. 50 µl of 3,3′,5,5′ –Tetramethylbenzidine substrate (TMB, Sigma) was then added to each well and the developed color reactions were stopped by adding 50 µl 0.2 M H<sub>2</sub>SO<sub>4. </sub>Optical density at 450 and 630 nm was measured with a Model 680 microplate reader (Biorad).</p>


<h4>Immunization and antigen assay</h4>
<a id="article1.body1.sec4.sec15.p1" name="article1.body1.sec4.sec15.p1"></a><p>IxAC-B1-specific antibodies were produced in Balb/c mice by DNA immunization followed by a booster protein injection. Briefly, 100 µg of plasmid pCDNA3.1/IxAC-B1_V5-His DNA in saline was injected four times in the anterior tibialis at three week intervals. Animals were then boosted once with 1 µg purified IxAC-B1 in Alum (Brenntag Biosector, DK). Sera were taken by retro-orbital puncture two to 5 weeks after the booster injection. Recombinant IxAC-B1_V5His protein was produced by transfecting subconfluent 293T cells with plasmid pCDNA3.1/IxAC-B1V5His in serum-free culture medium. The recombinant protein was purified from cell culture supernatants by chromatography on NiNTa columns (Invitrogen). Pre-immune and immune antisera were tested by western blot on the 7 <em>I. ricinus</em> IxACs and assessed <em>in vitro</em> for neutralization of anticomplement activity in the AP test. We reasoned that if seroneutralization occurred, antibodies would block inhibition of AP by IxACS and hemolysis would occur. 3.0 µl heat-inactivated mouse antisera were preincubated for 30 min at room temperature with 2.0 µl samples of standardized IxACS. The mixture was then added to 2.0 µl human serum in GVB buffer in a total volume of 25 µl. 25 µl RBC were then added and the AP assay was conducted as described above. Controls included i) sera from mice previously mock-immunized 3 times with PBS in Freund's adjuvant, ii) serum from mice immunized with the unrelated protein Iris <a href="#pone.0001400-Prevot1">[78]</a>, iii) GVB buffer. Results are expressed as % hemolysis as:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001400.e003&amp;representation=PNG" class="inline-graphic"></span><br></p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001400"></div><a name="pone.0001400.s001" id="pone.0001400.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001400.s001">Figure S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001400.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="postSiDOI">(0.14 MB TIF)</p>
<a name="pone.0001400.s002" id="pone.0001400.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001400.s002">Figure S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001400.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="postSiDOI">(0.36 MB TIF)</p>
<a name="pone.0001400.s003" id="pone.0001400.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001400.s003">Figure S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001400.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="postSiDOI">(0.37 MB TIF)</p>
<a name="pone.0001400.s004" id="pone.0001400.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001400.s004">Table S1. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001400.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001400.s005" id="pone.0001400.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001400.s005">Table S2. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001400.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001400.s006" id="pone.0001400.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001400.s006">Table S3. </a></strong></p><p class="siDoi">doi:10.1371/journal.pone.0001400.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Dr Maxime Madder, Ph D, at the Animal Health department, Prince Leopold Institute of Tropical Medicine, Antwerp, Belgium for the <em>R. appendiculatus</em> specimens; Dr. P. Kerkhofs and Dr Thierry Vandenberg, DVM, PhDs at the Veterinary and Agrochemical Research Centre (VAR, Bruxelles, Belgium) for sheep and chicken sera respectively; Dr Cathleen McEntee (ULB, Bruxelles) for serum from Beagle dogs, Dr A. Linden from Ulg (Liège Belgium) for deer serum samples, Mr. M. Marchand at the <em>Centre de Sélection Bovine</em> (Ciney, Belgium) for naïve calf sera; Dr Coulon, DVM, for pig serum samples; Dr Christophe Remy at the <em>Museum d'Histoire Naturelle de Tournai</em> (Tournai, Belgium) for serum samples from <em>Elaph guttata</em>; Prof. M. Millinkovitch (ULB) for serum sample from <em>Boa constrictor</em>; Mr. Lieven Gevart (Gentaur S.P.R.L., Bruxelles) for turkey samples, Dr. M. Fournel, DVM (ULB), for collecting sera from pheasants and domestic pigeons. The excellent technical assistance of Mr Louis Delhaye and staff at the IBMM animal facility is gratefully acknowledged.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: BC EG JB CC. Performed the experiments: BC JB CC KL FG VD PP. Analyzed the data: BC EG JB CC BC LV. Contributed reagents/materials/analysis tools: MB EG YD. Wrote the paper: BC EG JB CC LV. Other: First co-author: CC. Contributed equally: LV EG. Senior co-authors: LV EG CC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001400-Price1" id="pone.0001400-Price1"></a>Price PW (1980) Evolutionary evolutionary biology of parasites. Monographs in population biology. Princeton, New Jersey: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001400-Jongejan1" id="pone.0001400-Jongejan1"></a>Jongejan F, Uilenberg G (2004) The global importance of ticks. Parasitology  129: S3–S14.  <ul class="find" data-citedArticleID="1038354" data-doi="10.1017/s0031182004005967"><li><a href="http://dx.doi.org/10.1017/s0031182004005967" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+global+importance+of+ticks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+global+importance+of+ticks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001400-Sonenshine1" id="pone.0001400-Sonenshine1"></a>Sonenshine DE (1991) Biology of ticks. Vol. I. New York, Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001400-Mans1" id="pone.0001400-Mans1"></a>Mans BJ, Gothe R, Neitz AWH (2004) Biochemical perspectives on paralysis and other forms of toxicoses caused by ticks. Parasitology  129: S95–S111.  <ul class="find" data-citedArticleID="1038382" data-doi="10.1017/s0031182003004670"><li><a href="http://dx.doi.org/10.1017/s0031182003004670" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+perspectives+on+paralysis+and+other+forms+of+toxicoses+caused+by+ticks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+perspectives+on+paralysis+and+other+forms+of+toxicoses+caused+by+ticks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001400-Ribeiro1" id="pone.0001400-Ribeiro1"></a>Ribeiro JMC (1987a) Role of saliva in blood-feeding by arthropods. Ann Rev Entomol  32: 463–478.  <ul class="find" data-citedArticleID="1038426" data-doi="10.1146/annurev.en.32.010187.002335"><li><a href="http://dx.doi.org/10.1146/annurev.en.32.010187.002335" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+saliva+in+blood-feeding+by+arthropods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+saliva+in+blood-feeding+by+arthropods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001400-Ribeiro2" id="pone.0001400-Ribeiro2"></a>Ribeiro JMC, Fancischetti IMB (2003) Role of arthropod saliva in blood feeding: Sialome and post-sialome perspectives. Ann Rev. Entomol  48: 73–88.  <ul class="find" data-citedArticleID="1038428" data-doi="10.1146/annurev.ento.48.060402.102812"><li><a href="http://dx.doi.org/10.1146/annurev.ento.48.060402.102812" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+arthropod+saliva+in+blood+feeding%3A+Sialome+and+post-sialome+perspectives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+arthropod+saliva+in+blood+feeding%3A+Sialome+and+post-sialome+perspectives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001400-Valenzuela1" id="pone.0001400-Valenzuela1"></a>Valenzuela JG (2004) Exploring tick saliva: from biochemistry to “sialomes” and functional genomics. Parasitology  129: S83–S94.  <ul class="find" data-citedArticleID="1038468" data-doi="10.1017/s0031182004005189"><li><a href="http://dx.doi.org/10.1017/s0031182004005189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+tick+saliva%3A+from+biochemistry+to+%E2%80%9Csialomes%E2%80%9D+and+functional+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+tick+saliva%3A+from+biochemistry+to+%E2%80%9Csialomes%E2%80%9D+and+functional+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001400-Kovar1" id="pone.0001400-Kovar1"></a>Kovar L (2004) Tick saliva in anti-tick immunity and pathogen transmission. Folia Microbiol  49: 327–336.  <ul class="find" data-citedArticleID="1038366" data-doi="10.1007/bf02931051"><li><a href="http://dx.doi.org/10.1007/bf02931051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tick+saliva+in+anti-tick+immunity+and+pathogen+transmission." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tick+saliva+in+anti-tick+immunity+and+pathogen+transmission.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001400-Janeway1" id="pone.0001400-Janeway1"></a>Janeway CA Jr, Travers P, Walport M, Shlomchik MJ (2001) Immunobiology 5th ed., Garland Publishing.  <ul class="find" data-citedArticleID="1038350"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Immunobiology+5th+ed.%2C+Garland+Publishing.&amp;auth=&amp;atitle=Immunobiology+5th+ed.%2C+Garland+Publishing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunobiology+5th+ed.%2C+Garland+Publishing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunobiology+5th+ed.%2C+Garland+Publishing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001400-Hourcade1" id="pone.0001400-Hourcade1"></a>Hourcade D, Holers VM, Atkinson JP (1989) The regulators of complement activation (RCA) gene cluster. Adv Immunol  45: 381–416.  <ul class="find" data-citedArticleID="1038344" data-doi="10.1016/s0065-2776(08)60697-5"><li><a href="http://dx.doi.org/10.1016/s0065-2776(08)60697-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+regulators+of+complement+activation+%28RCA%29+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+regulators+of+complement+activation+%28RCA%29+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001400-Fearon1" id="pone.0001400-Fearon1"></a>Fearon DT, Austen KF (1975) Properdin:binding to C3b and stabilization of the C3b-dependent C3 convertase. J Exp Med.  142: 856–63.  <ul class="find" data-citedArticleID="1038324" data-doi="10.1084/jem.142.4.856"><li><a href="http://dx.doi.org/10.1084/jem.142.4.856" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properdin%3Abinding+to+C3b+and+stabilization+of+the+C3b-dependent+C3+convertase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properdin%3Abinding+to+C3b+and+stabilization+of+the+C3b-dependent+C3+convertase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001400-Pangburn1" id="pone.0001400-Pangburn1"></a>Pangburn MK (1989) Analysis of the natural polymeric forms of human properdin and their functions in complement activation. J immunol  142: 202–207.  <ul class="find" data-citedArticleID="1038416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+the+natural+polymeric+forms+of+human+properdin+and+their+functions+in+complement+activation.&amp;auth=&amp;atitle=Analysis+of+the+natural+polymeric+forms+of+human+properdin+and+their+functions+in+complement+activation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+natural+polymeric+forms+of+human+properdin+and+their+functions+in+complement+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+natural+polymeric+forms+of+human+properdin+and+their+functions+in+complement+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001400-Smith1" id="pone.0001400-Smith1"></a>Smith CA, Pangburn MK, Vogel CW, Müller-Eberhard HJ (1984) Molecular architecture of human properdin, a positive regulator of the alternative pathway of complement. J Biol Chem  259: 4582–4588.  <ul class="find" data-citedArticleID="1038444"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+architecture+of+human+properdin%2C+a+positive+regulator+of+the+alternative+pathway+of+complement.&amp;auth=&amp;atitle=Molecular+architecture+of+human+properdin%2C+a+positive+regulator+of+the+alternative+pathway+of+complement." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+architecture+of+human+properdin%2C+a+positive+regulator+of+the+alternative+pathway+of+complement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+architecture+of+human+properdin%2C+a+positive+regulator+of+the+alternative+pathway+of+complement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001400-Goundis1" id="pone.0001400-Goundis1"></a>Goundis D, Reid KBM (1988) Properdin, the terminal complement components, thrombospondin and the circumsporozoite protein of malaria parasites contain similar sequence motifs. Nature  335: 82–85.  <ul class="find" data-citedArticleID="1038334" data-doi="10.1038/335082a0"><li><a href="http://dx.doi.org/10.1038/335082a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properdin%2C+the+terminal+complement+components%2C+thrombospondin+and+the+circumsporozoite+protein+of+malaria+parasites+contain+similar+sequence+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properdin%2C+the+terminal+complement+components%2C+thrombospondin+and+the+circumsporozoite+protein+of+malaria+parasites+contain+similar+sequence+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001400-Nolan1" id="pone.0001400-Nolan1"></a>Nolan KF, Schwaeble W, Kaluz S, Dierich MP, Reid KB (1991) Molecular cloning of the cDNA coding for properdin, a positive regulator of the alternative pathway of human complement. Eur J Immunol  21: 771–776.  <ul class="find" data-citedArticleID="1038400" data-doi="10.1002/eji.1830210333"><li><a href="http://dx.doi.org/10.1002/eji.1830210333" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+of+the+cDNA+coding+for+properdin%2C+a+positive+regulator+of+the+alternative+pathway+of+human+complement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+of+the+cDNA+coding+for+properdin%2C+a+positive+regulator+of+the+alternative+pathway+of+human+complement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001400-Smith2" id="pone.0001400-Smith2"></a>Smith KF, Nolan KF, Reid KB, Perkins SJ (1991) Neutron and X-ray scattering studies on the human complement protein properdin provide an analysis of the thrombospondin repeat. Biochemistry  30: 8000–8008.  <ul class="find" data-citedArticleID="1038446" data-doi="10.1021/bi00246a018"><li><a href="http://dx.doi.org/10.1021/bi00246a018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neutron+and+X-ray+scattering+studies+on+the+human+complement+protein+properdin+provide+an+analysis+of+the+thrombospondin+repeat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neutron+and+X-ray+scattering+studies+on+the+human+complement+protein+properdin+provide+an+analysis+of+the+thrombospondin+repeat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001400-DiScipio1" id="pone.0001400-DiScipio1"></a>DiScipio RG (1981) The binding of human complement proteins C5, factor B, beta 1H and properdin to complement fragment C3b on zymosan. Biochem J.  199: 485–496.  <ul class="find" data-citedArticleID="1038320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+binding+of+human+complement+proteins+C5%2C+factor+B%2C+beta+1H+and+properdin+to+complement+fragment+C3b+on+zymosan.&amp;auth=&amp;atitle=The+binding+of+human+complement+proteins+C5%2C+factor+B%2C+beta+1H+and+properdin+to+complement+fragment+C3b+on+zymosan." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+binding+of+human+complement+proteins+C5%2C+factor+B%2C+beta+1H+and+properdin+to+complement+fragment+C3b+on+zymosan." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+binding+of+human+complement+proteins+C5%2C+factor+B%2C+beta+1H+and+properdin+to+complement+fragment+C3b+on+zymosan.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001400-Hourcade2" id="pone.0001400-Hourcade2"></a>Hourcade DE (2006) The role of properdin in the assembly of the alternative pathway C3 convertases of complement. J Biol Chem  281: 2128–2132.  <ul class="find" data-citedArticleID="1038346" data-doi="10.1074/jbc.m508928200"><li><a href="http://dx.doi.org/10.1074/jbc.m508928200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+properdin+in+the+assembly+of+the+alternative+pathway+C3+convertases+of+complement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+properdin+in+the+assembly+of+the+alternative+pathway+C3+convertases+of+complement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001400-GuptaBansal1" id="pone.0001400-GuptaBansal1"></a>Gupta-Bansal R, Parent JB, Brunden KR (2000) Inhibition of complement alternative pathway function with anti-properdin monoclonal antibodies. Mol Immunol.  37: 191–201.  <ul class="find" data-citedArticleID="1038338" data-doi="10.1016/s0161-5890(00)00047-x"><li><a href="http://dx.doi.org/10.1016/s0161-5890(00)00047-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+complement+alternative+pathway+function+with+anti-properdin+monoclonal+antibodies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+complement+alternative+pathway+function+with+anti-properdin+monoclonal+antibodies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001400-Joiner1" id="pone.0001400-Joiner1"></a>Joiner KA (1988) Complement evasion by bacteria and parasites. Ann Rev Microbiol  42: 201–230.  <ul class="find" data-citedArticleID="1038352" data-doi="10.1146/annurev.mi.42.100188.001221"><li><a href="http://dx.doi.org/10.1146/annurev.mi.42.100188.001221" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complement+evasion+by+bacteria+and+parasites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complement+evasion+by+bacteria+and+parasites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001400-Wikel1" id="pone.0001400-Wikel1"></a>Wikel SK, Allen JR (1977) Acquired resistance to ticks. iii. Cobra venom factor and the resistance response. Immunology  32: 457–465.  <ul class="find" data-citedArticleID="1038478"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Acquired+resistance+to+ticks.+iii.+Cobra+venom+factor+and+the+resistance+response.&amp;auth=&amp;atitle=Acquired+resistance+to+ticks.+iii.+Cobra+venom+factor+and+the+resistance+response." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acquired+resistance+to+ticks.+iii.+Cobra+venom+factor+and+the+resistance+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acquired+resistance+to+ticks.+iii.+Cobra+venom+factor+and+the+resistance+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001400-Wikel2" id="pone.0001400-Wikel2"></a>Wikel SK (1979) Acquired resistance to ticks: expression of resistance by C4-deficient guinea pigs. Am J Trop Med Hyg.  28: 586–590.  <ul class="find" data-citedArticleID="1038480"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Acquired+resistance+to+ticks%3A+expression+of+resistance+by+C4-deficient+guinea+pigs.&amp;auth=&amp;atitle=Acquired+resistance+to+ticks%3A+expression+of+resistance+by+C4-deficient+guinea+pigs." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Acquired+resistance+to+ticks%3A+expression+of+resistance+by+C4-deficient+guinea+pigs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Acquired+resistance+to+ticks%3A+expression+of+resistance+by+C4-deficient+guinea+pigs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001400-Ribeiro3" id="pone.0001400-Ribeiro3"></a>Ribeiro JM (1987b) <em>Ixodes dammini</em>: salivary anti-complement activity. Exp Parasitol.  64: 347–353.  <ul class="find" data-citedArticleID="1038430" data-doi="10.1016/0014-4894(87)90046-4"><li><a href="http://dx.doi.org/10.1016/0014-4894(87)90046-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ixodes+dammini%3A+salivary+anti-complement+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ixodes+dammini%3A+salivary+anti-complement+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001400-Lawrie1" id="pone.0001400-Lawrie1"></a>Lawrie CH, Randolph SE, Nuttall PA (1999) <em>Ixodes</em> ticks: serum species sensitivity of anticomplement activity. Exp Parasitol  93: 207–214.  <ul class="find" data-citedArticleID="1038378" data-doi="10.1006/expr.1999.4456"><li><a href="http://dx.doi.org/10.1006/expr.1999.4456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ixodes+ticks%3A+serum+species+sensitivity+of+anticomplement+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ixodes+ticks%3A+serum+species+sensitivity+of+anticomplement+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001400-Valenzuela2" id="pone.0001400-Valenzuela2"></a>Valenzuela JG, Charlab R, Mather TN, Ribeiro JM (2000) Purification, cloning, and expression of a novel salivary anticomplement protein from the tick, <em>Ixodes scapularis.</em> J Biol Chem  275: 18717–18723.  <ul class="find" data-citedArticleID="1038470" data-doi="10.1074/jbc.m001486200"><li><a href="http://dx.doi.org/10.1074/jbc.m001486200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Purification%2C+cloning%2C+and+expression+of+a+novel+salivary+anticomplement+protein+from+the+tick%2C+Ixodes+scapularis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Purification%2C+cloning%2C+and+expression+of+a+novel+salivary+anticomplement+protein+from+the+tick%2C+Ixodes+scapularis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001400-Mejri1" id="pone.0001400-Mejri1"></a>Mejri N, Rutti B, Brossard M (2002) Immunosuppressive effects of <em>Ixodes ricinus</em> tick saliva or salivary gland extracts on innate and acquired immune response of BALB/c mice. Parasitol Res  88: 192–197.  <ul class="find" data-citedArticleID="1038386" data-doi="10.1007/s00436-001-0515-1"><li><a href="http://dx.doi.org/10.1007/s00436-001-0515-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunosuppressive+effects+of+Ixodes+ricinus+tick+saliva+or+salivary+gland+extracts+on+innate+and+acquired+immune+response+of+BALB%2Fc+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunosuppressive+effects+of+Ixodes+ricinus+tick+saliva+or+salivary+gland+extracts+on+innate+and+acquired+immune+response+of+BALB%2Fc+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001400-Soares1" id="pone.0001400-Soares1"></a>Soares CA, Lima CM, Dolan MC, Piesman J, Beard CB, et al.  (2005) Capillary feeding of specific dsRNA induces silencing of the Isac gene in nymphal <em>Ixodes scapularis</em> ticks. Insect Mol Biol  14: 443–452.  <ul class="find" data-citedArticleID="1038448" data-doi="10.1111/j.1365-2583.2005.00575.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.2005.00575.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Capillary+feeding+of+specific+dsRNA+induces+silencing+of+the+Isac+gene+in+nymphal+Ixodes+scapularis+ticks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Capillary+feeding+of+specific+dsRNA+induces+silencing+of+the+Isac+gene+in+nymphal+Ixodes+scapularis+ticks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001400-Das1" id="pone.0001400-Das1"></a>Das S, Banerjee G, DePonte K, Marcantonio N, Kantor FS, et al.  (2001) Salp25D, an <em>Ixodes scapularis</em> antioxidant, is 1 of 14 immunodominant antigens in engorged tick salivary glands. J Infect Dis  184: 1056–1064.  <ul class="find" data-citedArticleID="1038318" data-doi="10.1086/323351"><li><a href="http://dx.doi.org/10.1086/323351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Salp25D%2C+an+Ixodes+scapularis+antioxidant%2C+is+1+of+14+immunodominant+antigens+in+engorged+tick+salivary+glands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Salp25D%2C+an+Ixodes+scapularis+antioxidant%2C+is+1+of+14+immunodominant+antigens+in+engorged+tick+salivary+glands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001400-Tyson1" id="pone.0001400-Tyson1"></a>Tyson K, Elkins C, Patterson H, Fikrig E, de Silva A (2007) Biochemical and functional characterization of Salp20, an Ixodes scapularis tick salivary protein that inhibits the complement pathway. Insect Molecular Biology  16: 469–479.  <ul class="find" data-citedArticleID="1038464" data-doi="10.1111/j.1365-2583.2007.00742.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.2007.00742.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical+and+functional+characterization+of+Salp20%2C+an+Ixodes+scapularis+tick+salivary+protein+that+inhibits+the+complement+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical+and+functional+characterization+of+Salp20%2C+an+Ixodes+scapularis+tick+salivary+protein+that+inhibits+the+complement+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001400-Francischetti1" id="pone.0001400-Francischetti1"></a>Francischetti IM, My Pham V, Mans BJ, Andersen JF, Mather TN, et al.  (2005) The transcriptome of the salivary glands of the female western black-legged tick <em>Ixodes pacificus</em> (<em>Acari: Ixodidae</em>). Insect Biochem Mol Biol.  35: 1142–1161.  <ul class="find" data-citedArticleID="1038328" data-doi="10.1016/j.ibmb.2005.05.007"><li><a href="http://dx.doi.org/10.1016/j.ibmb.2005.05.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+transcriptome+of+the+salivary+glands+of+the+female+western+black-legged+tick+Ixodes+pacificus+%28Acari%3A+Ixodidae%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+transcriptome+of+the+salivary+glands+of+the+female+western+black-legged+tick+Ixodes+pacificus+%28Acari%3A+Ixodidae%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001400-Daix1" id="pone.0001400-Daix1"></a>Daix V, Schroeder H, Praet N, Georgin J-P, Chiappino I, et al.  (2007) <em>Ixodes</em> ticks belonging to the <em>Ixodes ricinus</em> complex encode a family of anticomplement proteins. Insect Mol Biol.  16: 155–166.  <ul class="find" data-citedArticleID="1038316" data-doi="10.1111/j.1365-2583.2006.00710.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.2006.00710.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ixodes+ticks+belonging+to+the+Ixodes+ricinus+complex+encode+a+family+of+anticomplement+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ixodes+ticks+belonging+to+the+Ixodes+ricinus+complex+encode+a+family+of+anticomplement+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001400-Astigarraga1" id="pone.0001400-Astigarraga1"></a>Astigarraga A, Oleaga-Perez A, Perez-Sanchez R, Baranda JA, Encinas-Grandes A (1997) Host immune response evasion strategies in <em>Ornithodoros erraticus</em> and <em>O. moubata</em> and their relationship to the development of an antiargasid vaccine. Parasite Immunol.  19: 401–410.  <ul class="find" data-citedArticleID="1038304" data-doi="10.1046/j.1365-3024.1997.d01-236.x"><li><a href="http://dx.doi.org/10.1046/j.1365-3024.1997.d01-236.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Host+immune+response+evasion+strategies+in+Ornithodoros+erraticus+and+O.+moubata+and+their+relationship+to+the+development+of+an+antiargasid+vaccine." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Host+immune+response+evasion+strategies+in+Ornithodoros+erraticus+and+O.+moubata+and+their+relationship+to+the+development+of+an+antiargasid+vaccine.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001400-Nunn1" id="pone.0001400-Nunn1"></a>Nunn MA, Sharma A, Paesen GC, Adamson S, Lissina O, et al.  (2005) Complement inhibitor of C5 activation from the soft tick <em>Ornithodoros moubata</em>. J Immunol  174: 2084–2091.  <ul class="find" data-citedArticleID="1038406"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Complement+inhibitor+of+C5+activation+from+the+soft+tick+Ornithodoros+moubata.&amp;auth=&amp;atitle=Complement+inhibitor+of+C5+activation+from+the+soft+tick+Ornithodoros+moubata." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complement+inhibitor+of+C5+activation+from+the+soft+tick+Ornithodoros+moubata." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complement+inhibitor+of+C5+activation+from+the+soft+tick+Ornithodoros+moubata.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001400-Valenzuela3" id="pone.0001400-Valenzuela3"></a>Valenzuela JG, Francischetti IM, Pham VM, Garfield MK, Mather TN, et al.  (2002) Exploring the sialome of the tick <em>Ixodes scapularis</em>. J Exp Biol  205(Pt 18): 2843–2864.  <ul class="find" data-citedArticleID="1038472"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Exploring+the+sialome+of+the+tick+Ixodes+scapularis.&amp;auth=&amp;atitle=Exploring+the+sialome+of+the+tick+Ixodes+scapularis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploring+the+sialome+of+the+tick+Ixodes+scapularis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploring+the+sialome+of+the+tick+Ixodes+scapularis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001400-Ribeiro4" id="pone.0001400-Ribeiro4"></a>Ribeiro JM, Alarcon-Chaidez F, Francischetti IM, Mans BJ, Mather TN, et al.  (2006) An annotated catalog of salivary gland transcripts from <em>Ixodes scapularis</em> ticks. Insect Biochem Mol Biol  36: 111–129.  <ul class="find" data-citedArticleID="1038432" data-doi="10.1016/j.ibmb.2005.11.005"><li><a href="http://dx.doi.org/10.1016/j.ibmb.2005.11.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+annotated+catalog+of+salivary+gland+transcripts+from+Ixodes+scapularis+ticks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+annotated+catalog+of+salivary+gland+transcripts+from+Ixodes+scapularis+ticks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001400-Ullmann1" id="pone.0001400-Ullmann1"></a>Ullmann AJ, Lima CM, Guerrero FD, Piesman J, Black WC 4th (2005) Genome size and organization in the blacklegged tick, <em>Ixodes scapularis</em> and the Southern cattle tick, <em>Boophilus microplus</em>. Insect Mol Biol  4: 217–222.  <ul class="find" data-citedArticleID="1038466" data-doi="10.1111/j.1365-2583.2005.00551.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.2005.00551.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+size+and+organization+in+the+blacklegged+tick%2C+Ixodes+scapularis+and+the+Southern+cattle+tick%2C+Boophilus+microplus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+size+and+organization+in+the+blacklegged+tick%2C+Ixodes+scapularis+and+the+Southern+cattle+tick%2C+Boophilus+microplus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001400-Palmer1" id="pone.0001400-Palmer1"></a>Palmer MJ, Bantle JA, Guo X, Fargo WS (1994) Genome size and organization in the ixodid tick <em>Amblyomma americanum</em> (L.). Insect Mol Biol  3: 57–62.  <ul class="find" data-citedArticleID="1038414" data-doi="10.1111/j.1365-2583.1994.tb00151.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2583.1994.tb00151.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+size+and+organization+in+the+ixodid+tick+Amblyomma+americanum+%28L.%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+size+and+organization+in+the+ixodid+tick+Amblyomma+americanum+%28L.%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001400-Tack1" id="pone.0001400-Tack1"></a>Tack BF, Janatova J, Thomas ML, Harrison RA, Hammer CH (1981)  The third, fourth, and fifth components of human complement: isolation and biochemical properties. In: Colowick SP, Kaplan NO, editors. Methods in Enzymology 80. New York 1981: Academic Press.   <ul class="find-nolinks"></ul></li><li><span class="label">39.
              </span><a name="pone.0001400-Lawrie2" id="pone.0001400-Lawrie2"></a>Lawrie CH, Sim RB, Nuttall PA (2005) Investigation of the mechanisms of anti-complement activity in <em>Ixodes ricinus</em> ticks. Mol Immunol.  42: 31–38.  <ul class="find" data-citedArticleID="1038380" data-doi="10.1016/j.molimm.2004.07.001"><li><a href="http://dx.doi.org/10.1016/j.molimm.2004.07.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Investigation+of+the+mechanisms+of+anti-complement+activity+in+Ixodes+ricinus+ticks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Investigation+of+the+mechanisms+of+anti-complement+activity+in+Ixodes+ricinus+ticks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001400-Curman1" id="pone.0001400-Curman1"></a>Curman B, Sandberg-Tragardh L, Peterson PA (1977) Chemical characterization of human factor B of the alternate pathway of complement activation. Biochemistry.  16: 5368–5375.  <ul class="find" data-citedArticleID="1038314" data-doi="10.1021/bi00643a031"><li><a href="http://dx.doi.org/10.1021/bi00643a031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+characterization+of+human+factor+B+of+the+alternate+pathway+of+complement+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+characterization+of+human+factor+B+of+the+alternate+pathway+of+complement+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001400-Ohno1" id="pone.0001400-Ohno1"></a>Ohno S (1970) Evolution by gene duplication. New-York: Springer-Verlag.   <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001400-Hughes1" id="pone.0001400-Hughes1"></a>Hughes AL (1999) Adaptive Evolution of Genes and Genomes. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0001400-Taylor1" id="pone.0001400-Taylor1"></a>Taylor JS, Raes J (2004) Duplication and divergence: the evolution of new genes and old ideas. Annu Rev Genet.  38: 615–643.  <ul class="find" data-citedArticleID="1038458" data-doi="10.1146/annurev.genet.38.072902.092831"><li><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.092831" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Duplication+and+divergence%3A+the+evolution+of+new+genes+and+old+ideas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001400-Kordis1" id="pone.0001400-Kordis1"></a>Kordis D, Gubensek F (2000) Adaptive evolution of animal toxin multigene families. Gene.  261: 43–52.  <ul class="find" data-citedArticleID="1038362" data-doi="10.1016/s0378-1119(00)00490-x"><li><a href="http://dx.doi.org/10.1016/s0378-1119(00)00490-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+of+animal+toxin+multigene+families." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+of+animal+toxin+multigene+families.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001400-Zupunski1" id="pone.0001400-Zupunski1"></a>Zupunski V, Kordis D, Gubensek F (2003) Adaptive evolution in the snake venom Kunitz/BPTI protein family. FEBS Lett  547: 131–136.  <ul class="find" data-citedArticleID="1038482" data-doi="10.1016/s0014-5793(03)00693-8"><li><a href="http://dx.doi.org/10.1016/s0014-5793(03)00693-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+evolution+in+the+snake+venom+Kunitz%2FBPTI+protein+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+evolution+in+the+snake+venom+Kunitz%2FBPTI+protein+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001400-Mans2" id="pone.0001400-Mans2"></a>Mans BJ, Louw AI, Neitz AWH (2002) Evolution of hematophagy in ticks: common origins for blood coagulation and platelet aggregation inhibitors from soft ticks of the genus <em>Ornithodoros</em>. Mol Biol Evol  19: 1695–1705.  <ul class="find" data-citedArticleID="1038384" data-doi="10.1093/oxfordjournals.molbev.a003992"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003992" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+hematophagy+in+ticks%3A+common+origins+for+blood+coagulation+and+platelet+aggregation+inhibitors+from+soft+ticks+of+the+genus+Ornithodoros." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+hematophagy+in+ticks%3A+common+origins+for+blood+coagulation+and+platelet+aggregation+inhibitors+from+soft+ticks+of+the+genus+Ornithodoros.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001400-Lanzaro1" id="pone.0001400-Lanzaro1"></a>Lanzaro GC, Lopes AH, Ribeiro JM, Shoemaker CB, Warburg A, et al.  (1999) Variation in the salivary peptide, maxadilan, from species in the <em>Lutzomyia longipalpis</em> complex. Insect Mol Biol.  8: 267–275.  <ul class="find" data-citedArticleID="1038376" data-doi="10.1046/j.1365-2583.1999.820267.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2583.1999.820267.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+the+salivary+peptide%2C+maxadilan%2C+from+species+in+the+Lutzomyia+longipalpis+complex." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+the+salivary+peptide%2C+maxadilan%2C+from+species+in+the+Lutzomyia+longipalpis+complex.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001400-Milleron1" id="pone.0001400-Milleron1"></a>Milleron RS, Mutebi JP, Valle S, Montoya A, Yin H, et al.  (2004a) Antigenic diversity in maxadilan, a salivary protein from the sand fly vector of American visceral leishmaniasis. Am J Trop Med Hyg  70: 286–293.  <ul class="find" data-citedArticleID="1038388"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Antigenic+diversity+in+maxadilan%2C+a+salivary+protein+from+the+sand+fly+vector+of+American+visceral+leishmaniasis.&amp;auth=&amp;atitle=Antigenic+diversity+in+maxadilan%2C+a+salivary+protein+from+the+sand+fly+vector+of+American+visceral+leishmaniasis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antigenic+diversity+in+maxadilan%2C+a+salivary+protein+from+the+sand+fly+vector+of+American+visceral+leishmaniasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antigenic+diversity+in+maxadilan%2C+a+salivary+protein+from+the+sand+fly+vector+of+American+visceral+leishmaniasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001400-Milleron2" id="pone.0001400-Milleron2"></a>Milleron RS, Ribeiro JM, Elnaime D, Soong L, Lanzaro GC (2004b) Negative effect of antibodies against maxadilan on the fitness of the sand fly vector of American visceral leishmaniasis. Am J Trop Med Hyg  70: 278–285.  <ul class="find" data-citedArticleID="1038390"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Negative+effect+of+antibodies+against+maxadilan+on+the+fitness+of+the+sand+fly+vector+of+American+visceral+leishmaniasis.&amp;auth=&amp;atitle=Negative+effect+of+antibodies+against+maxadilan+on+the+fitness+of+the+sand+fly+vector+of+American+visceral+leishmaniasis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Negative+effect+of+antibodies+against+maxadilan+on+the+fitness+of+the+sand+fly+vector+of+American+visceral+leishmaniasis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Negative+effect+of+antibodies+against+maxadilan+on+the+fitness+of+the+sand+fly+vector+of+American+visceral+leishmaniasis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001400-Schreiber1" id="pone.0001400-Schreiber1"></a>Schreiber RD, Muller-Eberhard HJ (1978) Assembly of the cytolytic alternative pathway of complement from 11 isolated plasma proteins. J Exp Med  148: 1722–1727.  <ul class="find" data-citedArticleID="1038440" data-doi="10.1084/jem.148.6.1722"><li><a href="http://dx.doi.org/10.1084/jem.148.6.1722" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Assembly+of+the+cytolytic+alternative+pathway+of+complement+from+11+isolated+plasma+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Assembly+of+the+cytolytic+alternative+pathway+of+complement+from+11+isolated+plasma+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001400-Pangburn2" id="pone.0001400-Pangburn2"></a>Pangburn MK, Muller-Eberhard HJ (1986) The C3 convertase of the alternative pathway of human complement. Enzymic properties of the bimolecular proteinase. Biochem J  235: 723–730.  <ul class="find" data-citedArticleID="1038418"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+C3+convertase+of+the+alternative+pathway+of+human+complement.+Enzymic+properties+of+the+bimolecular+proteinase.&amp;auth=&amp;atitle=The+C3+convertase+of+the+alternative+pathway+of+human+complement.+Enzymic+properties+of+the+bimolecular+proteinase." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C3+convertase+of+the+alternative+pathway+of+human+complement.+Enzymic+properties+of+the+bimolecular+proteinase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C3+convertase+of+the+alternative+pathway+of+human+complement.+Enzymic+properties+of+the+bimolecular+proteinase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001400-Spitzer1" id="pone.0001400-Spitzer1"></a>Spitzer D, Mitchell LM, Atkinson JP, Hourcade DE (2007) Properdin can initiate complement activation by binding specific target surfaces and providing a platform for de novo convertase assembly. J Immunol  179: 2600–2608.  <ul class="find" data-citedArticleID="1038452"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Properdin+can+initiate+complement+activation+by+binding+specific+target+surfaces+and+providing+a+platform+for+de+novo+convertase+assembly.&amp;auth=&amp;atitle=Properdin+can+initiate+complement+activation+by+binding+specific+target+surfaces+and+providing+a+platform+for+de+novo+convertase+assembly." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properdin+can+initiate+complement+activation+by+binding+specific+target+surfaces+and+providing+a+platform+for+de+novo+convertase+assembly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properdin+can+initiate+complement+activation+by+binding+specific+target+surfaces+and+providing+a+platform+for+de+novo+convertase+assembly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001400-Kimura1" id="pone.0001400-Kimura1"></a>Kimura Y, Miwa T, Zhou L, Song WC (2007) Activator-specific requirement of properdin in the initiation and amplification of the alternative pathway complement. Blood..   DOI 10.1182/blood-2007-05-089821.  <ul class="find" data-citedArticleID="1038358" data-doi="10.1182/blood-2007-05-089821"><li><a href="http://dx.doi.org/10.1182/blood-2007-05-089821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Activator-specific+requirement+of+properdin+in+the+initiation+and+amplification+of+the+alternative+pathway+complement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Activator-specific+requirement+of+properdin+in+the+initiation+and+amplification+of+the+alternative+pathway+complement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001400-Nolan2" id="pone.0001400-Nolan2"></a>Nolan KF, Reid KD (1993) Properdin. Methods Enzymol  223: 35–46.  <ul class="find" data-citedArticleID="1038402"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Properdin.&amp;auth=&amp;atitle=Properdin." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properdin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properdin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001400-Sun1" id="pone.0001400-Sun1"></a>Sun Z, Reid KB, Perkins SJ (2004) The dimeric and trimeric solution structures of the multidomain complement protein properdin by X-ray scattering, analytical ultracentrifugation and constrained modelling. J Mol Biol  343: 1327–1343.  <ul class="find" data-citedArticleID="1038454" data-doi="10.1016/j.jmb.2004.09.001"><li><a href="http://dx.doi.org/10.1016/j.jmb.2004.09.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+dimeric+and+trimeric+solution+structures+of+the+multidomain+complement+protein+properdin+by+X-ray+scattering%2C+analytical+ultracentrifugation+and+constrained+modelling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+dimeric+and+trimeric+solution+structures+of+the+multidomain+complement+protein+properdin+by+X-ray+scattering%2C+analytical+ultracentrifugation+and+constrained+modelling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001400-Kraiczy1" id="pone.0001400-Kraiczy1"></a>Kraiczy P, Würzner R (2006) Complement escape of human pathogenic bacteria by acquisition of complement regulators. Mol Immunol  43: 31–44.  <ul class="find" data-citedArticleID="1038370" data-doi="10.1016/j.molimm.2005.06.016"><li><a href="http://dx.doi.org/10.1016/j.molimm.2005.06.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complement+escape+of+human+pathogenic+bacteria+by+acquisition+of+complement+regulators." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complement+escape+of+human+pathogenic+bacteria+by+acquisition+of+complement+regulators.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001400-Finlay1" id="pone.0001400-Finlay1"></a>Finlay BB, McFadden G (2006) Anti-immunology: evasion of the host immune system by bacterial and viral pathogens. Cell.  124: 767–782.  <ul class="find" data-citedArticleID="1038326" data-doi="10.1016/j.cell.2006.01.034"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anti-immunology%3A+evasion+of+the+host+immune+system+by+bacterial+and+viral+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anti-immunology%3A+evasion+of+the+host+immune+system+by+bacterial+and+viral+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001400-Chung1" id="pone.0001400-Chung1"></a>Chung KM, Liszewski MK, Nybakken G, Davis AE, Townsend RR, et al.  (2006) West Nile virus nonstructural protein NS1 inhibits complement activation by binding the regulatory protein factor H. Proc Natl Acad Sci U S A.  103: 19111–191116.  <ul class="find" data-citedArticleID="1038310" data-doi="10.1073/pnas.0605668103"><li><a href="http://dx.doi.org/10.1073/pnas.0605668103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=West+Nile+virus+nonstructural+protein+NS1+inhibits+complement+activation+by+binding+the+regulatory+protein+factor+H." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22West+Nile+virus+nonstructural+protein+NS1+inhibits+complement+activation+by+binding+the+regulatory+protein+factor+H.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001400-Kostavasili1" id="pone.0001400-Kostavasili1"></a>Kostavasili I, Sahu A, Friedman HM, Eisenberg RJ, Cohen GH, et al.  (1997) Mechanism of complement inactivation by glycoprotein C of herpes simplex virus. J Immunol.  158: 1763–1771.  <ul class="find" data-citedArticleID="1038364"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanism+of+complement+inactivation+by+glycoprotein+C+of+herpes+simplex+virus.&amp;auth=&amp;atitle=Mechanism+of+complement+inactivation+by+glycoprotein+C+of+herpes+simplex+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanism+of+complement+inactivation+by+glycoprotein+C+of+herpes+simplex+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanism+of+complement+inactivation+by+glycoprotein+C+of+herpes+simplex+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001400-Sacks1" id="pone.0001400-Sacks1"></a>Sacks D, Sher A (2002) Evasion of innate immunity by parasitic protozoa. Nat Immunol  3: 1041–1047.  <ul class="find" data-citedArticleID="1038438" data-doi="10.1038/ni1102-1041"><li><a href="http://dx.doi.org/10.1038/ni1102-1041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evasion+of+innate+immunity+by+parasitic+protozoa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evasion+of+innate+immunity+by+parasitic+protozoa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001400-Kock1" id="pone.0001400-Kock1"></a>Kock MA, Hew BE, Bammert H, Fritzinger DC, Vogel CW (2004) Structure and function of recombinant cobra venom factor. J Biol Chem  279: 30836–30843.  <ul class="find" data-citedArticleID="1038360" data-doi="10.1074/jbc.m403196200"><li><a href="http://dx.doi.org/10.1074/jbc.m403196200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure+and+function+of+recombinant+cobra+venom+factor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure+and+function+of+recombinant+cobra+venom+factor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001400-Ricklin1" id="pone.0001400-Ricklin1"></a>Ricklin D, Lambris JD (2007) Complement-targeted therapeutics. Nat Biotechnol  25: 1265–1275.  <ul class="find" data-citedArticleID="1038434" data-doi="10.1038/nbt1342"><li><a href="http://dx.doi.org/10.1038/nbt1342" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complement-targeted+therapeutics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complement-targeted+therapeutics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001400-Rother1" id="pone.0001400-Rother1"></a>Rother RP, Rollins SA, Mojcik CF, Brodsky RA, Bell L (2007) Discovery and development of the complement inhibitor eculizumab for the treatment of paroxysmal nocturnal hemoglobinuria. Nat Biotechnol  25: 1256–1264.  <ul class="find" data-citedArticleID="1038436" data-doi="10.1038/nbt1344"><li><a href="http://dx.doi.org/10.1038/nbt1344" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discovery+and+development+of+the+complement+inhibitor+eculizumab+for+the+treatment+of+paroxysmal+nocturnal+hemoglobinuria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discovery+and+development+of+the+complement+inhibitor+eculizumab+for+the+treatment+of+paroxysmal+nocturnal+hemoglobinuria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001400-Reid1" id="pone.0001400-Reid1"></a>Reid KB, Day AJ (1989) Structure-function relationships of the complements components. Immunol. Today  10: 177–180.  <ul class="find" data-citedArticleID="1038424" data-doi="10.1016/0167-5699(89)90317-4"><li><a href="http://dx.doi.org/10.1016/0167-5699(89)90317-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structure-function+relationships+of+the+complements+components." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structure-function+relationships+of+the+complements+components.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001400-Mizuno1" id="pone.0001400-Mizuno1"></a>Mizuno M, Morgan BP (2004) The possibilities and pitfalls for anti-complement therapies in inflammatory diseases. Curr Drug Targets Inflamm Allergy  3: 87–96.  <ul class="find" data-citedArticleID="1038394" data-doi="10.2174/1568010043483890"><li><a href="http://dx.doi.org/10.2174/1568010043483890" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+possibilities+and+pitfalls+for+anti-complement+therapies+in+inflammatory+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+possibilities+and+pitfalls+for+anti-complement+therapies+in+inflammatory+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001400-Mollnes1" id="pone.0001400-Mollnes1"></a>Mollnes TE, Kirschfink M (2006) Strategies of therapeutic complement inhibition. Mol Immunol  43: 107–121.  <ul class="find" data-citedArticleID="1038396" data-doi="10.1016/j.molimm.2005.06.014"><li><a href="http://dx.doi.org/10.1016/j.molimm.2005.06.014" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Strategies+of+therapeutic+complement+inhibition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Strategies+of+therapeutic+complement+inhibition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001400-Anderson1" id="pone.0001400-Anderson1"></a>Anderson JF (1991) Epizootiology of Lyme Borreliosis. Scand J Infect Dis–Suppl.  77: 3–34.  <ul class="find" data-citedArticleID="1038302"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Epizootiology+of+Lyme+Borreliosis.&amp;auth=&amp;atitle=Epizootiology+of+Lyme+Borreliosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epizootiology+of+Lyme+Borreliosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epizootiology+of+Lyme+Borreliosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001400-Milne1" id="pone.0001400-Milne1"></a>Milne A (1949) The Ecology of the sheep tick, <em>Ixodes ricinus</em> L., host relationship of the tick. Part 2. Observations on hill and moorland grazings in northern England. Parasitology  39: 173–197.  <ul class="find" data-citedArticleID="1038392" data-doi="10.1017/s0031182000083736"><li><a href="http://dx.doi.org/10.1017/s0031182000083736" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Ecology+of+the+sheep+tick%2C+Ixodes+ricinus+L.%2C+host+relationship+of+the+tick.+Part+2.+Observations+on+hill+and+moorland+grazings+in+northern+England." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Ecology+of+the+sheep+tick%2C+Ixodes+ricinus+L.%2C+host+relationship+of+the+tick.+Part+2.+Observations+on+hill+and+moorland+grazings+in+northern+England.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001400-Hoogstraal1" id="pone.0001400-Hoogstraal1"></a>Hoogstraal H, Aeschlimann A (1982) Tick-host specificity. Bulletin de la société entomologique suisse  55: 5–32.  <ul class="find" data-citedArticleID="1038342"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tick-host+specificity.&amp;auth=&amp;atitle=Tick-host+specificity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tick-host+specificity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tick-host+specificity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001400-Schroeder1" id="pone.0001400-Schroeder1"></a>Schroeder H, Daix V, Gillet L, Renauld JC, Vanderplasschen A (2007) The paralogous salivary anti-complement proteins IRAC I and IRAC II encoded by <em>Ixodes ricinus</em> ticks have broad and complementary inhibitory activities against the complement of different host species. Microbes Infect  9: 247–250.  <ul class="find" data-citedArticleID="1038442" data-doi="10.1016/j.micinf.2006.10.020"><li><a href="http://dx.doi.org/10.1016/j.micinf.2006.10.020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+paralogous+salivary+anti-complement+proteins+IRAC+I+and+IRAC+II+encoded+by+Ixodes+ricinus+ticks+have+broad+and+complementary+inhibitory+activities+against+the+complement+of+different+host+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+paralogous+salivary+anti-complement+proteins+IRAC+I+and+IRAC+II+encoded+by+Ixodes+ricinus+ticks+have+broad+and+complementary+inhibitory+activities+against+the+complement+of+different+host+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001400-Nonaka1" id="pone.0001400-Nonaka1"></a>Nonaka M, Kimura A (2006) Genomic view of the evolution of the complement system. Immunogenetics  58: 701–713.  <ul class="find" data-citedArticleID="1038404" data-doi="10.1007/s00251-006-0142-1"><li><a href="http://dx.doi.org/10.1007/s00251-006-0142-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+view+of+the+evolution+of+the+complement+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+view+of+the+evolution+of+the+complement+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001400-Holers1" id="pone.0001400-Holers1"></a>Holers VM, Thurman JM (2004) The alternative pathway of complement in disease: opportunities for therapeutic targeting. Mol Immunol  41(2–3): 147–52.  <ul class="find" data-citedArticleID="1038340" data-doi="10.1016/j.molimm.2004.03.012"><li><a href="http://dx.doi.org/10.1016/j.molimm.2004.03.012" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+alternative+pathway+of+complement+in+disease%3A+opportunities+for+therapeutic+targeting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+alternative+pathway+of+complement+in+disease%3A+opportunities+for+therapeutic+targeting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001400-Thurman1" id="pone.0001400-Thurman1"></a>Thurman JM, Holers VM (2006) The central role of the alternative complement pathway in human disease. J Immunol  176: 1305–1310.  <ul class="find" data-citedArticleID="1038462"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+central+role+of+the+alternative+complement+pathway+in+human+disease.&amp;auth=&amp;atitle=The+central+role+of+the+alternative+complement+pathway+in+human+disease." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+central+role+of+the+alternative+complement+pathway+in+human+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+central+role+of+the+alternative+complement+pathway+in+human+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001400-Weisman1" id="pone.0001400-Weisman1"></a>Weisman HF, Bartow T, Leppo MK, Boyle MP, Marsh HC Jr, et al.  (1990) Recombinant soluble CR1 suppressed complement activation, inflammation, and necrosis associated with reperfusion of ischemic myocardium. Trans Assoc Am Physicians  103: 64–72.  <ul class="find" data-citedArticleID="1038476"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Recombinant+soluble+CR1+suppressed+complement+activation%2C+inflammation%2C+and+necrosis+associated+with+reperfusion+of+ischemic+myocardium.&amp;auth=&amp;atitle=Recombinant+soluble+CR1+suppressed+complement+activation%2C+inflammation%2C+and+necrosis+associated+with+reperfusion+of+ischemic+myocardium." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinant+soluble+CR1+suppressed+complement+activation%2C+inflammation%2C+and+necrosis+associated+with+reperfusion+of+ischemic+myocardium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinant+soluble+CR1+suppressed+complement+activation%2C+inflammation%2C+and+necrosis+associated+with+reperfusion+of+ischemic+myocardium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001400-Thomas1" id="pone.0001400-Thomas1"></a>Thomas TC, Rollins SA, Rother RP, Giannoni MA, Hartman SL, et al.  (1996) Inhibition of complement activity by humanized anti-C5 antibody and single-chain Fv. Mol Immunol  33: 1389–401.  <ul class="find" data-citedArticleID="1038460" data-doi="10.1016/s0161-5890(96)00078-8"><li><a href="http://dx.doi.org/10.1016/s0161-5890(96)00078-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+complement+activity+by+humanized+anti-C5+antibody+and+single-chain+Fv." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+complement+activity+by+humanized+anti-C5+antibody+and+single-chain+Fv.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001400-Kozak1" id="pone.0001400-Kozak1"></a>Kozak M (1989) The scanning model for translation: an update. J Cell Biol  108: 229–241.  <ul class="find" data-citedArticleID="1038368" data-doi="10.1083/jcb.108.2.229"><li><a href="http://dx.doi.org/10.1083/jcb.108.2.229" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+scanning+model+for+translation%3A+an+update." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+scanning+model+for+translation%3A+an+update.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001400-Paesen1" id="pone.0001400-Paesen1"></a>Paesen GC, Adams PL, Harlos K, Nuttall PA, Stuart DI (1999) Tick histamine-binding proteins: isolation, cloning, and three-dimensional structure. Mol Cell  3: 661–671.  <ul class="find" data-citedArticleID="1038410" data-doi="10.1016/s1097-2765(00)80359-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80359-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tick+histamine-binding+proteins%3A+isolation%2C+cloning%2C+and+three-dimensional+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tick+histamine-binding+proteins%3A+isolation%2C+cloning%2C+and+three-dimensional+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001400-Prevot1" id="pone.0001400-Prevot1"></a>Prevot PP, Adam B, Boudjeltia KZ, Brossard M, Lins L, et al.  (2006) Anti-hemostatic effects of a serpin from the saliva of the tick <em>Ixodes ricinus</em>. J Biol Chem.  281: 26361–26369.  <ul class="find" data-citedArticleID="1038420" data-doi="10.1074/jbc.m604197200"><li><a href="http://dx.doi.org/10.1074/jbc.m604197200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anti-hemostatic+effects+of+a+serpin+from+the+saliva+of+the+tick+Ixodes+ricinus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anti-hemostatic+effects+of+a+serpin+from+the+saliva+of+the+tick+Ixodes+ricinus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001400-Emanuelsson1" id="pone.0001400-Emanuelsson1"></a>Emanuelsson O, Brunak S, von Heijne G, Nielsen H (2007) Locating proteins in the cell using TargetP, SignalP, and related tools. Nature Protocols  2: 953–971.  <ul class="find" data-citedArticleID="1038322" data-doi="10.1038/nprot.2007.131"><li><a href="http://dx.doi.org/10.1038/nprot.2007.131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locating+proteins+in+the+cell+using+TargetP%2C+SignalP%2C+and+related+tools." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locating+proteins+in+the+cell+using+TargetP%2C+SignalP%2C+and+related+tools.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001400-Julenius1" id="pone.0001400-Julenius1"></a>Julenius K, Molgaard A, Gupta R, Brunak S (2005) Prediction, conservation analysis, and structural characterization of mammalian mucin-type O-glycosylation sites. Glycobiology  15: 153–164.  <ul class="find" data-citedArticleID="1038356" data-doi="10.1093/glycob/cwh151"><li><a href="http://dx.doi.org/10.1093/glycob/cwh151" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction%2C+conservation+analysis%2C+and+structural+characterization+of+mammalian+mucin-type+O-glycosylation+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction%2C+conservation+analysis%2C+and+structural+characterization+of+mammalian+mucin-type+O-glycosylation+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001400-Krogh1" id="pone.0001400-Krogh1"></a>Krogh A, Larsson B, von Heijne G, Sonnhammer ELL (2001) Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes. J Mol Biol  305: 567–580.  <ul class="find" data-citedArticleID="1038372" data-doi="10.1006/jmbi.2000.4315"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4315" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model%3A+Application+to+complete+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predicting+transmembrane+protein+topology+with+a+hidden+Markov+model%3A+Application+to+complete+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001400-Vullo1" id="pone.0001400-Vullo1"></a>Vullo A, Frasconi P (2004) Disulfide Connectivity Prediction using Recursive Neural Networks and Evolutionary Information. Bioinformatics  20: 653–659.  <ul class="find" data-citedArticleID="1038474" data-doi="10.1093/bioinformatics/btg463"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg463" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disulfide+Connectivity+Prediction+using+Recursive+Neural+Networks+and+Evolutionary+Information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disulfide+Connectivity+Prediction+using+Recursive+Neural+Networks+and+Evolutionary+Information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001400-Gaboriaud1" id="pone.0001400-Gaboriaud1"></a>Gaboriaud C, Bissery V, Benchetrit T, Mornon JP (1987) Hydrophobic cluster analysis: an efficient new way to compare and analyse amino acid sequences. FEBS Lett  224: 149–155.  <ul class="find" data-citedArticleID="1038330" data-doi="10.1016/0014-5793(87)80439-8"><li><a href="http://dx.doi.org/10.1016/0014-5793(87)80439-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hydrophobic+cluster+analysis%3A+an+efficient+new+way+to+compare+and+analyse+amino+acid+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hydrophobic+cluster+analysis%3A+an+efficient+new+way+to+compare+and+analyse+amino+acid+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001400-Callebaut1" id="pone.0001400-Callebaut1"></a>Callebaut I, Labesse G, Durand P, Poupon A, Canard L, et al.  (1997) Deciphering protein sequence information through hydrophobic cluster analysis (HCA): current status and perspectives. Cell Mol Life Sci.  53: 621–645.  <ul class="find" data-citedArticleID="1038308" data-doi="10.1007/s000180050082"><li><a href="http://dx.doi.org/10.1007/s000180050082" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deciphering+protein+sequence+information+through+hydrophobic+cluster+analysis+%28HCA%29%3A+current+status+and+perspectives." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deciphering+protein+sequence+information+through+hydrophobic+cluster+analysis+%28HCA%29%3A+current+status+and+perspectives.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001400-Nicholas1" id="pone.0001400-Nicholas1"></a>Nicholas KB, Nicholas HB Jr, Deerfield DW II (1997) GeneDoc: Analysis and Visualization of Genetic Variation. EMBNEW NEWS  4: 14.  <ul class="find" data-citedArticleID="1038398"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=GeneDoc%3A+Analysis+and+Visualization+of+Genetic+Variation.&amp;auth=&amp;atitle=GeneDoc%3A+Analysis+and+Visualization+of+Genetic+Variation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneDoc%3A+Analysis+and+Visualization+of+Genetic+Variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneDoc%3A+Analysis+and+Visualization+of+Genetic+Variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001400-Guindon1" id="pone.0001400-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Systematic Biology  52: 696–704.  <ul class="find" data-citedArticleID="1038336"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001400-Page1" id="pone.0001400-Page1"></a>Page RD (1996) TreeView: an application to display phylogenetic trees on personal computers. Comput Appl Biosci.  12: 357–358.  <ul class="find" data-citedArticleID="1038412" data-doi="10.1093/bioinformatics/12.4.357"><li><a href="http://dx.doi.org/10.1093/bioinformatics/12.4.357" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TreeView%3A+an+application+to+display+phylogenetic+trees+on+personal+computers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TreeView%3A+an+application+to+display+phylogenetic+trees+on+personal+computers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001400-Cai1" id="pone.0001400-Cai1"></a>Cai W, Pei J, Grishin NV (2004) Reconstruction of ancestral protein sequences and its applications. BMC Evol Biol.  4: 33.  <ul class="find" data-citedArticleID="1038306"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Reconstruction+of+ancestral+protein+sequences+and+its+applications.&amp;auth=&amp;atitle=Reconstruction+of+ancestral+protein+sequences+and+its+applications." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reconstruction+of+ancestral+protein+sequences+and+its+applications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reconstruction+of+ancestral+protein+sequences+and+its+applications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001400-Kumar1" id="pone.0001400-Kumar1"></a>Kumar S, Tamura K, Nei M (2004) MEGA3: Integrated software for Molecular Evolutionary Genetics Analysis and sequence alignment. Brief Bioinform.  5: 150–163.  <ul class="find" data-citedArticleID="1038374" data-doi="10.1093/bib/5.2.150"><li><a href="http://dx.doi.org/10.1093/bib/5.2.150" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MEGA3%3A+Integrated+software+for+Molecular+Evolutionary+Genetics+Analysis+and+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001400-Giclas1" id="pone.0001400-Giclas1"></a>Giclas PC (1997)  Complement tests. In: Rose NR, Conway de Macario E, Folds JD, Lane HC, Nakamura RM, editors. Manual of clinical laboratory immunology, 5th edition. Washinton DC: ASM Press. pp. 181–186.  <ul class="find-nolinks"></ul></li><li><span class="label">91.
              </span><a name="pone.0001400-Colligan1" id="pone.0001400-Colligan1"></a>Colligan JE (1994)  Complement. In: Coligan JE, Kruisbeek AM, Margulies DH, Shevach EM, Strober W, editors. Current Protocols in Immunology. New-York: Wiley/Interscience. pp. 13.1.1–13.2.7.  <ul class="find-nolinks"></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.XML" value="249160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.PDF" value="1657402"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t001.PNG_L" value="220435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t001.PNG_M" value="131528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t001.PNG_S" value="11661"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t001.TIF" value="859686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t001.PNG_I" value="41041"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g001.PNG_L" value="160018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g001.PNG_M" value="65559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g001.PNG_S" value="8177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g001.TIF" value="223506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g001.PNG_I" value="28676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g002.PNG_L" value="624273"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g002.PNG_M" value="101673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g002.PNG_S" value="10158"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g002.TIF" value="926146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g002.PNG_I" value="31887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t002.PNG_L" value="104491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t002.PNG_M" value="151982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t002.PNG_S" value="19770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t002.TIF" value="442386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t002.PNG_I" value="51477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t003.PNG_L" value="177321"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t003.PNG_M" value="103109"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t003.PNG_S" value="11652"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t003.TIF" value="721074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t003.PNG_I" value="32195"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g003.PNG_L" value="198372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g003.PNG_M" value="81567"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g003.PNG_S" value="5326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g003.TIF" value="211326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g003.PNG_I" value="24025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g004.PNG_L" value="5475087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g004.PNG_M" value="179200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g004.PNG_S" value="12951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g004.TIF" value="6429734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g004.PNG_I" value="97731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t004.PNG_L" value="118292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t004.PNG_M" value="70284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t004.PNG_S" value="9777"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t004.TIF" value="537540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t004.PNG_I" value="21988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g005.PNG_L" value="53958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g005.PNG_M" value="49983"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g005.PNG_S" value="9678"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g005.TIF" value="98806"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g005.PNG_I" value="16892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g006.PNG_L" value="719385"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g006.PNG_M" value="95522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g006.PNG_S" value="8998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g006.TIF" value="979366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g006.PNG_I" value="46322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g007.PNG_L" value="72279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g007.PNG_M" value="32033"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g007.PNG_S" value="10968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g007.TIF" value="144866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g007.PNG_I" value="33818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g008.PNG_L" value="117541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g008.PNG_M" value="49404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g008.PNG_S" value="9673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g008.TIF" value="238188"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g008.PNG_I" value="16178"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g009.PNG_L" value="94479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g009.PNG_M" value="56025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g009.PNG_S" value="10431"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g009.TIF" value="171408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g009.PNG_I" value="18864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t005.PNG_L" value="168335"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t005.PNG_M" value="97242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t005.PNG_S" value="11592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t005.TIF" value="772670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.t005.PNG_I" value="30960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g010.PNG_L" value="782740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g010.PNG_M" value="135891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g010.PNG_S" value="11094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g010.TIF" value="738686"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g010.PNG_I" value="41879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g011.PNG_L" value="334343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g011.PNG_M" value="110359"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g011.PNG_S" value="9117"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g011.TIF" value="443446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.g011.PNG_I" value="47562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.e001.PNG" value="17239"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.e001.TIF" value="34616"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.e002.PNG" value="15201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.e002.TIF" value="31808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.e003.PNG" value="13702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.e003.TIF" value="30180"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.s001.TIF" value="179378"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.s002.TIF" value="307244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.s003.TIF" value="444834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.s004.DOC" value="61952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.s005.DOC" value="66048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001400.s006.DOC" value="30720"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001400&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001400" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001400&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001400&volume=&issue=&title=Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus&author_name=Bernard%20Couvreur%2C%20J%C3%A9r%C3%B4me%20Beaufays%2C%20C%C3%A9dric%20Charon%2C%20Kathia%20Lahaye%2C%20Fran%C3%A7ois%20Gensale%2C%20Val%C3%A9rie%20Denis%2C%20Beno%C3%AEt%20Charloteaux%2C%20Yves%20Decrem%2C%20Pierre-Paul%20Pr%C3%A9v%C3%B4t%2C%20Michel%20Brossard%2C%20Luc%20Vanhamme%2C%20Edmond%20Godfroid&start_page=1&end_page=24" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001400" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400&amp;t=Variability%20and%20Action%20Mechanism%20of%20a%20Family%20of%20Anticomplement%20Proteins%20in%20Ixodes%20ricinus" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400&title=Variability%20and%20Action%20Mechanism%20of%20a%20Family%20of%20Anticomplement%20Proteins%20in%20Ixodes%20ricinus&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400&amp;title=Variability%20and%20Action%20Mechanism%20of%20a%20Family%20of%20Anticomplement%20Proteins%20in%20Ixodes%20ricinus" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001400&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Variability and Action Mechanism of a Family of Anticomplement Proteins in Ixodes ricinus';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001400';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Variability%20and%20Action%20Mechanism%20of%20a%20Family%20of%20Anticomplement%20Proteins%20in%20Ixodes%20ricinus http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001400" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001400" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">




















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Complement+system%22" title="Search for articles in the subject area:'Complement system'"><div class="flagText">Complement system</div></a>
              <div data-categoryid="34855" data-articleid="25630"
                   data-categoryname="Complement system"
                   class="flagImage" title="Flag 'Complement system' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25630"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="26251" data-articleid="25630"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Properdin%22" title="Search for articles in the subject area:'Properdin'"><div class="flagText">Properdin</div></a>
              <div data-categoryid="27645" data-articleid="25630"
                   data-categoryname="Properdin"
                   class="flagImage" title="Flag 'Properdin' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Recombinant+proteins%22" title="Search for articles in the subject area:'Recombinant proteins'"><div class="flagText">Recombinant proteins</div></a>
              <div data-categoryid="18471" data-articleid="25630"
                   data-categoryname="Recombinant proteins"
                   class="flagImage" title="Flag 'Recombinant proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ricinus%22" title="Search for articles in the subject area:'Ricinus'"><div class="flagText">Ricinus</div></a>
              <div data-categoryid="43337" data-articleid="25630"
                   data-categoryname="Ricinus"
                   class="flagImage" title="Flag 'Ricinus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Salivary+glands%22" title="Search for articles in the subject area:'Salivary glands'"><div class="flagText">Salivary glands</div></a>
              <div data-categoryid="47555" data-articleid="25630"
                   data-categoryname="Salivary glands"
                   class="flagImage" title="Flag 'Salivary glands' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="25630"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2247'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2105'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3840&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=61199">Change of affiliation</a><br>Posted by B_Couvreur</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
