

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001095"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001095" />

    <meta name="citation_title" content="Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains"/>
    <meta itemprop="name" content="Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains"/>

      <meta name="citation_author" content="Chao Cheng"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Paola Fabrizio"/>
            <meta name="citation_author_institution" content="Andrus Gerontology Center and Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Huanying Ge"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Min Wei"/>
            <meta name="citation_author_institution" content="Andrus Gerontology Center and Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Valter D. Longo"/>
            <meta name="citation_author_institution" content="Andrus Gerontology Center and Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
      <meta name="citation_author" content="Lei M. Li"/>
            <meta name="citation_author_institution" content="Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America"/>
            <meta name="citation_author_institution" content="Department of Mathematics, University of Southern California, Los Angeles, California, United States of America"/>

    <meta name="citation_date" content="2007/10/31"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001095.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1095"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A conserved regulatory system for aging.; citation_author=C Kenyon; citation_journal_title=Cell; citation_volume=105; citation_number=1; citation_pages=165-168; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Evolutionary medicine:from drawf model systems to healthy centenarians.; citation_author=VD Longo; citation_author=CE Finch; citation_journal_title=Science; citation_volume=299; citation_number=2; citation_pages=1342-1346; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The chronological life span of Saccharomyces cerevisiae.; citation_author=P Fabrizio; citation_author=V Longo; citation_journal_title=Aging Cell; citation_volume=2; citation_number=3; citation_pages=73-81; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Large-scale identification in yeast of conserved ageing genes.; citation_author=M Kaeberlein; citation_author=B Kennedy; citation_journal_title=Mechanisms of Ageing and Development; citation_volume=126; citation_number=4; citation_pages=17-21; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genes determining yeast replicative life span in a long-lived genetic background.; citation_author=M Kaeberlein; citation_author=KT Kirkland; citation_author=S Fields; citation_author=B Kennedy; citation_journal_title=Mechanisms of Ageing and Development; citation_volume=126; citation_number=5; citation_pages=491-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of longevity and stress resistance by Sch9 in yeast.; citation_author=P Fabrizio; citation_author=F Pozza; citation_author=SD Pletcher; citation_author=LVCM Gendron; citation_journal_title=Science; citation_volume=292; citation_number=6; citation_pages=288-90; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=SOD2 functions downstream of Sch9 to extend longevity in yeast.; citation_author=P Fabrizio; citation_author=LL Liou; citation_author=VN Moy; citation_author=A Diaspro; citation_author=J Selverstone Valentine; citation_journal_title=Genetics; citation_volume=163; citation_number=7; citation_pages=35-46; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Extension of chronological life span in yeast by decreased tor pathway signaling.; citation_author=R Powers; citation_author=K M; citation_author=C SD; citation_author=K BK; citation_author=F S; citation_journal_title=Genes & Development; citation_volume=20; citation_number=8; citation_pages=174-184; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The nature and causes of ageing.; citation_author=TB Kirkwood; citation_journal_title=Ciba Found Symp; citation_volume=134; citation_number=9; citation_pages=193-207; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Comparative life spans of species: why do species have the life spans they do?; citation_author=TB Kirkwood; citation_journal_title=Am J Clin Nutr; citation_volume=55; citation_number=10; citation_pages=1191S-1195S; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Evolution of ageing.; citation_author=TB Kirkwood; citation_journal_title=Mech Ageing Dev; citation_volume=123; citation_number=11; citation_pages=737-745; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolutionary theories of ageing applied to long-lived organisms.; citation_author=L Partridge; citation_journal_title=Exp Gerontol; citation_volume=36; citation_number=12; citation_pages=641-650; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Fitness cost of extended lifespan in Caenorhabditis elegans.; citation_author=NL Jenkins; citation_author=G McColl; citation_author=GJ Lithgow; citation_journal_title=Proc Biol Sci; citation_volume=271; citation_number=13; citation_pages=2523-2526; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Programmed and altruistic ageing.; citation_author=VD Longo; citation_author=J Mitteldorf; citation_author=VP Skulachev; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=14; citation_pages=866-872; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Aging: atheory based on free radical and radiation chemistry.; citation_author=D Harman; citation_journal_title=J Gerontol; citation_volume=11; citation_number=15; citation_pages=298-300; citation_date=1956; " />
      <meta name="citation_reference" content="citation_title=A biologic clock: the mitochondria?; citation_author=D Harman; citation_journal_title=Journal of the American Geriatrics Society; citation_volume=20; citation_number=16; citation_pages=145-147; citation_date=1972; " />
      <meta name="citation_reference" content="citation_title=Analyzing high-density oligonucleotide gene expression array data.; citation_author=EE Schadt; citation_author=C Li; citation_author=C Su; citation_author=WH Wong; citation_journal_title=Journal of Cellular Biochemistry; citation_volume=80; citation_number=17; citation_pages=192-202; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Feature extraction and normalization algorithms for high-density oligonucleotide gene expression array data.; citation_author=EE Schadt; citation_author=C Li; citation_author=B Ellis; citation_author=WH Wong; citation_journal_title=Journal of Cellular Biochemistry Supplement; citation_volume=37; citation_number=18; citation_pages=120-125; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A comparison of normalization methods for high density oligonucleotide array data based on bias and variance.; citation_author=B Bolstad; citation_author=RA Irizarry; citation_author=M Astrand; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=19; citation_pages=185-193; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data.; citation_author=RA Irizarry; citation_author=B Hobbs; citation_author=F Collin; citation_author=YD Beazer-Barclay; citation_author=KJ Antonellis; citation_journal_title=Biostatistics; citation_volume=4; citation_number=20; citation_pages=249-264; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sub-array normalization subject to differentiation.; citation_author=C Cheng; citation_author=LM Li; citation_journal_title=Nucleic Acids Research; citation_volume=33; citation_number=21; citation_pages=5565-5573; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=JW Tukey; citation_number=22; citation_date=1977; citation_publisher=Addison-Wesley; " />
      <meta name="citation_reference" content="citation_title=Differentiation detection in microarray normalization.; citation_author=LM Li; citation_author=C Cheng; citation_number=23; citation_date=2007; citation_publisher=CRC press; " />
      <meta name="citation_reference" content="citation_title=Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.; citation_author=A Subramanian; citation_author=P Tamayo; citation_author=V Mootha; citation_author=S Mukherjee; citation_author=B Ebert; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=24; citation_pages=15545-15550; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A direct approach to false discovery rates.; citation_author=JD Storey; citation_journal_title=Journal of the Royal Statistical Society, Series B; citation_volume=64; citation_number=25; citation_pages=479-498; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Statistical significance for genome-wide experiment.; citation_author=JD Storey; citation_author=R Tibshirani; citation_journal_title=Proc Natl Acad Sci; citation_volume=100; citation_number=26; citation_pages=9440-9445; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Function and regulation of yeast hexose transporters.; citation_author=S Ozcan; citation_author=M Johnston; citation_journal_title=Microbiol Mol Biol Rev; citation_volume=63; citation_number=27; citation_pages=554-69; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Three different regulatory mechanisms enable yeast hexose transporter (HXT) genes to be induced by different levels of glucose.; citation_author=S Ozcan; citation_author=M Johnston; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=28; citation_pages=1564-72; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The ras/protein kinase a pathway acts in parallel with the mob2/cbk1 pathway to effect cell cycle progression and proper bud site selection.; citation_author=L Schneper; citation_author=A Krauss; citation_author=R Miyamoto; citation_author=S Fang; citation_author=J Broach; citation_journal_title=Eukaryot Cell; citation_volume=3; citation_number=29; citation_pages=108-20; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Requirement of Saccharomyces cerevisiae ras for completion of mitosis.; citation_author=T Morishita; citation_author=H Mitsuzawa; citation_author=M Nakafuku; citation_author=S Nakamura; citation_author=S Hattori; citation_journal_title=Science; citation_volume=270; citation_number=30; citation_pages=1213-5; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A metabolic model for life span determination in Caenorhabditis elegans.; citation_author=S Rea; citation_author=T Johnson; citation_journal_title=Dev Cell; citation_volume=5; citation_number=31; citation_pages=197-203; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Mitochondria, oxidant, and aging.; citation_author=R Balaban; citation_author=S Nemoto; citation_author=T Finkel; citation_journal_title=Cell; citation_volume=120; citation_number=32; citation_pages=483-495; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The plasticity of aging: insight from long-lived mutants.; citation_author=C Kenyon; citation_journal_title=Cell; citation_volume=120; citation_number=33; citation_pages=449-460; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A systematic RNAi screen identifies a critical role for mitochondria in C. elegans longevity.; citation_author=S Lee; citation_author=R Lee; citation_author=A Fraser; citation_author=R Kamath; citation_author=G Ruvkun; citation_journal_title=Nat Genet; citation_volume=33; citation_number=34; citation_pages=40-48; citation_date=2003; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains"/>
    <meta name="twitter:description" content="BackgroundRecent studies suggest that the regulation of longevity may be partially conserved in many eukaryotes ranging from yeast to mammals. The three yeast mutants sch9&Delta;, ras2&Delta;, tor1&Delta; show extended chronological life span up to three folds. Our aim is to dissect the mechanisms that lead to the yeast life span extension.Methodology/Principal FindingsWe obtain gene expression profiles of sch9&Delta;, ras2&Delta;, tor1&Delta; as well as that for a wild type at day 2.5 in SDC medium using Affymetrix Yeast2.0 arrays. To accurately estimate the expression differentiation between the wild type and the long-lived mutants, we use sub-array normalization followed by a variant of the median-polishing summarization. The results are validated by the probe sets of S. pombe on the same chips. To translate the differentiation into changes of biological activities, we make statistical inference by integrating the expression profiles with biological gene subsets defined by Gene Ontology, KEGG pathways, and cellular localization of proteins. Other than subset-versus-other comparisons, we also make local comparisons between two directly-related gene subsets such as cytosolic and mitochondrial ribosomes. Our consensus is obtained by cross-examination of these inferences. The significant and systematic differentiation in the three long-lived strains includes: lower transcriptional activities; down-regulation of TCA cycle and oxidative phosphorylation versus up-regulation of the KEGG pathway Glycolysis/Gluconeogenesis; the overall reduction of mitochondrial activities. We also report some different expression patterns such as reduction of the activities relating to mitosis in ras2&Delta;.Conclusions/SignificanceThe modification of energy pathways and modification of compartment activities such as down-regulation of mitochondrial ribosome proteins versus up-regulation of cytosolic ribosome proteins are directly associated with the life span extension in yeast. The results provide a new and systematic S. cerevisiae version of the free radical theory from the perspective of functional genomics."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001095.g005"/>

  <meta property="og:title" content="Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=9102'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8959'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=7057&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001095">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001095" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001095&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Chao Cheng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paola Fabrizio, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Andrus Gerontology Center and Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Huanying Ge, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Min Wei, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Andrus Gerontology Center and Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Valter D. Longo, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Andrus Gerontology Center and Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lei M. Li
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:lilei@usc.edu">lilei@usc.edu</a></p>

                <p>Affiliations:
                  Molecular and Computational Biology Program, Department of Biological Sciences, University of Southern California, Los Angeles, California, United States of America, 
                  Department of Mathematics, University of Southern California, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 31, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001095</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001095-g001" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-g002" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-g003" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-t001" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-t002" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-t003" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-g004" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-g005" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-t004" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001095-t005" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001095">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001095" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Recent studies suggest that the regulation of longevity may be partially conserved in many eukaryotes ranging from yeast to mammals. The three yeast mutants <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ show extended chronological life span up to three folds. Our aim is to dissect the mechanisms that lead to the yeast life span extension.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We obtain gene expression profiles of <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ as well as that for a wild type at day 2.5 in SDC medium using Affymetrix Yeast2.0 arrays. To accurately estimate the expression differentiation between the wild type and the long-lived mutants, we use sub-array normalization followed by a variant of the median-polishing summarization. The results are validated by the probe sets of <em>S. pombe</em> on the same chips. To translate the differentiation into changes of biological activities, we make statistical inference by integrating the expression profiles with biological gene subsets defined by Gene Ontology, KEGG pathways, and cellular localization of proteins. Other than subset-versus-other comparisons, we also make local comparisons between two directly-related gene subsets such as cytosolic and mitochondrial ribosomes. Our consensus is obtained by cross-examination of these inferences. The significant and systematic differentiation in the three long-lived strains includes: lower transcriptional activities; down-regulation of TCA cycle and oxidative phosphorylation versus up-regulation of the KEGG pathway Glycolysis/Gluconeogenesis; the overall reduction of mitochondrial activities. We also report some different expression patterns such as reduction of the activities relating to mitosis in <em>ras2</em>Δ.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The modification of energy pathways and modification of compartment activities such as down-regulation of mitochondrial ribosome proteins versus up-regulation of cytosolic ribosome proteins are directly associated with the life span extension in yeast. The results provide a new and systematic <em>S. cerevisiae</em> version of the free radical theory from the perspective of functional genomics.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Cheng C, Fabrizio P, Ge H, Wei M, Longo VD, et al.  (2007) Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains. PLoS ONE 2(10):
          e1095.
            doi:10.1371/journal.pone.0001095</p><p><strong>Academic Editor: </strong>Ji Zhu, University of Michigan, United States of America</p><p><strong>Received:</strong> June 11, 2007; <strong>Accepted:</strong> October 9, 2007; <strong>Published:</strong> October 31, 2007</p><p><strong>Copyright:</strong> © 2007 Cheng et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work is supported by the grant R01 GM75308-01 from NIH, and is partially supported by Center of Excellence in Genome Science at University of Southern California, the NIH P50 HG002790 grant.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Recent findings suggest that ageing, like many other biological processes, is subject to regulation by pathways that may have been partially conserved throughout evolution <a href="#pone.0001095-Kenyon1">[1]</a>, <a href="#pone.0001095-Longo1">[2]</a>. In fact, the down-regulation of the glucose-sensing/insulin/IGF-1 pathway promotes life span extension in organisms ranging from yeast to mice. In <em>S. cerevisiae</em> two different paradigms are used to measure longevity: the replicative life span, which is defined as the total number of daughter cells generated by a mother cell; and the chronological life span which is measured by monitoring the mean/maximum survival time of a population of non-dividing yeast. Here we focus on the chronological life span, which represents a simple but valuable system to study how post-mitotic cells age <a href="#pone.0001095-Fabrizio1">[3]</a>. The mutations of a single gene within the principal yeast nutrient-sensing pathways can extend the chronological life span dramatically <a href="#pone.0001095-Kaeberlein1">[4]</a>, <a href="#pone.0001095-Kaeberlein2">[5]</a>. Among these pathways, the Sch9, Ras2/cAMP/PKA and TOR pathways are of most interest. Importantly, in higher eukaryotes pathways that appear to share a common evolutionary origin with the Sch9 and TOR pathways are also implicated in life span regulation. In budding yeast inactivation of <em>sch9</em>, homolog of mammalian serine/threonine protein kinase Akt, extends chronological life span by nearly three folds <a href="#pone.0001095-Fabrizio2">[6]</a>. Deletion of <em>ras2</em> or down-regulation of <em>cyr1</em> in the Ras2/cAMP/PKA pathway nearly doubles the chronological life span of yeast <a href="#pone.0001095-Fabrizio2">[6]</a>, <a href="#pone.0001095-Fabrizio3">[7]</a>. In a large scale screening in yeast, several genes that encode components of the nutrient-responsive TOR pathway were found to increase the chronological life span <a href="#pone.0001095-Powers1">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A number of theories have been proposed to explain the mechanism of ageing. Among them are the disposable soma theory of ageing first suggested by Weismann and later developed by Kirkwood <em>et al.</em> <a href="#pone.0001095-Kirkwood1">[9]</a>, <a href="#pone.0001095-Kirkwood2">[10]</a>, <a href="#pone.0001095-Kirkwood3">[11]</a>, the accumulated mutation theory first proposed by Medawar in 1952 <a href="#pone.0001095-Partridge1">[12]</a>, the antagonistic pleiotropy theory proposed by Williams in 1957 <a href="#pone.0001095-Jenkins1">[13]</a>, the programmed and altruistic ageing theory <a href="#pone.0001095-Longo2">[14]</a>. The free radical theory of ageing first proposed by Harman in the 1950s <a href="#pone.0001095-Harman1">[15]</a> is particularly relevant to the research reported in this article. According to this theory, ageing is a consequence of free radical damage. Later Harman extended the idea to implicate mitochondrial production of ROS in the 1970s <a href="#pone.0001095-Harman2">[16]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The partial conservation of the life-span regulatory pathways suggests that they may have evolved in ancestral unicellular organisms in order to overcome periods of starvation. Calorie restriction (CR), which resembles the starvation conditions used to assess chronological life span, causes longevity extension in all the ageing model systems. In this article we analyze the gene expression profiles of chronologically long-lived yeast strains. Under this ageing paradigm, haploid yeast are grown in synthetic complete medium (SDC) until nutrients are depleted. Once yeast stop dividing, they are kept in the depleted medium. The viability of the cultures is monitored over time by measuring the colony forming units (CFUs) <a href="#pone.0001095-Fabrizio1">[3]</a>. Incubation in nutrient-depleted SDC mimics the conditions normally encountered by yeast in the natural environment where microorganisms survive for long periods of time under starvation.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>The modification of the chronological life span, is the end effect of genetic interventions such as the knockout of <em>sch9</em>, and environmental changes such as calorie restriction. Extensive results have been obtained in relating genotypes and the phenotype of life span. Our effort aims to understand the intermediate steps of the ageing mechanism.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The microarray technology allows us to measure the expression profiles of a living cell. We obtain the gene expression profiles of the long-lived mutants <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ together with a wild type at day 2.5 in SDC medium using Affymetrix yeast2.0 arrays. How to accurately estimate the differentiation between the wild type and the long-lived mutants is a key problem in our functional genomic study of ageing. Usually the estimation consists of two steps: normalization and summarization. Normalization aims to remove any non-biological difference generated in the reaction and read-out process while keep the real biological differentiation between a target and a reference sample. The invariant-set <a href="#pone.0001095-Schadt1">[17]</a>, <a href="#pone.0001095-Schadt2">[18]</a> and quantiles <a href="#pone.0001095-Bolstad1">[19]</a>, <a href="#pone.0001095-Irizarry1">[20]</a> normalization are two widely used methods in the literature. However, in our case, it is possible that the expression profiles of the long-lived mutants have substantial differentiation compared to that of the wild type. It is also possible that the differentiation is not symmetric. Thereby we adopt the sub-array normalization that is designed to preserve biological differentiation <a href="#pone.0001095-Cheng1">[21]</a>, <a href="#pone.0001095-Li1">[23]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>How to translate the differentiation into changes of biological activities is another key problem in the functional genomics of ageing. Relatively complete bio- and genomic- databases exist of <em>S. cerevisiae</em>. They provide us with instruments for statistical inference. In this article, we infer significant modifications of biological activities by integrating the expression differentiation with three sources of biological knowledge: Gene Ontology (GO), KEGG pathways, and cellular localization of proteins. Our consensus inference is obtained by cross-examination of the inferences drawn from the three perspectives. Furthermore, to reduce the gap of statistical significance and biological significance, we compare the transcriptional activities of two “directly-related gene subsets” such as the first half and second half of an energetic pathway, or ribosome proteins in either mitochondria or cytosol. The idea of local inference follows the basic principles of statistical design proposed by Sir R. Fisher while the idea of consensus inference is one statistical view of systems biology.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>In this work, we study the yeast ageing mechanism from the perspective of functional genomics. The three mutants, <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ share the same phenotype: longer chronological life span. The significant and systematic expression differentiation underlying the phenotype can shed light on the mechanism of ageing. We show that the goal is achievable and from the current data set we identify some common and characteristic changes of biological activities, which may directly lead to longevity.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Sample preparation and Affymetrix GeneChip arrays</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We obtained the gene expression profiles of yeast strains including wild type, <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ cells at day 2.5. Specifically, all strains used were obtained from frozen stocks. Each strain was inoculated in 1 mL SDC and grown overnight. Saturated overnight cultures were then diluted into 3 flasks each containing 50 mL of culture. All samples were incubated at 30°C with shaking (2200 rpm) until day 2.5. Total RNA was isolated from day 2.5 post-diauxic yeast cultures (2.0×10<sup>8</sup> cells) according to the acid phenol protocol. Briefly, yeast were collected by centrifugation, washed with cold water once, and resuspended in 400 µl of 10 mM Tris pH 7.5, 10 mM EDTA, 0.5% SDS. After adding 400 µl of warm acid phenol the cell suspension was incubated at 65°C for 20 minutes with vortexing every 5 minutes, centrifuged and the supernatant extracted twice with acid phenol and once with chloroform. Total RNA was recovered by precipitation with ethanol and cleaned up by using the RNAsy kit (Qiagen). RNA (5 µg/sample) was sent to the UCLA DNA array Core Facility. Total RNA from independent cultures of each strain was used as a template to synthesize complementary RNA (cRNA). The biotin-labeled cRNA was hybridized to Affymetrix GeneChip® Yeast2.0 Array. In sum, three biological replicates were obtained for each of wild type, <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>In the SDC medium, a substantial proportion of yeast cells are still dividing before day 2. At older ages, such as day 3–5, most of the cells become hypometabolic, which is associated with a dramatic drop in transcription. We harvest mRNA at day 2.5 so that we can extract enough mRNA for microarray experiment while avoid the noise introduced by the transcriptional activities of dividing cells.</p>


<h4>Normalization and summarization</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>After imaging process, the expression of each sample is represented by a CEL file, which includes the fluorescence intensities of all probes. Denote the three arrays of wild type, <em>sch9</em>Δ <em>ras2</em>Δ, and <em>tor1</em>Δ respectively by W1, W2, W3, S1, S2, S3, R1, R2, R3, T1, T2, T3. The conversion of probe intensities to expression values requires two statistical procedures: normalization and summarization. We applied the Sub-Sub normalization <a href="#pone.0001095-Cheng1">[21]</a> to our data sets, aiming at giving enough protection to possible differentiation between the mutants and the wild type. The normalization is carried out in a pairwise fashion. Namely, for each wild type sample, three replicates of a mutant are normalized with respect to this reference. Take <em>sch9</em>Δ for example, the normalized arrays with respect to W1, W2, W3 are respectively denoted by S1\W1, S2\W1, S3\W1, S1\W2, S2\W2, S3\W2, S1\W3, S2\W3, S3\W3.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Our summarization is a modified version of the median polishing method <a href="#pone.0001095-Tukey1">[22]</a> in RMA (the Bioconductor affy package <a href="http://www.bioconductor.org/">http://www.bioconductor.org/</a>). The median polishing summarization method is based on a two-factor model, which include the sample effect and probe-specific effect. Namely, the gene expression of each sample is estimated by adjusting each individual probe effect. In our situation, we group the wild type and normalized mutant arrays by the reference, and then summarize each group. Take <em>sch9</em>Δ for example. We summarize the four arrays (the reference plus three normalized) W1, S1\W1, S2\W1, S3\W1 together. This leads to three estimates of expression fold changes of the mutant versus the wild type. In total, we have nine estimates from three wild type references, and we take their median difference as the final estimate. Due to the nature of normalization <a href="#pone.0001095-Li1">[23]</a>, only the portion of differentiation that is not confounded with the reference array is estimable. In the above scheme, arrays in a summarization group correspond to the same reference. Thus we expect that the difference between a (normalized) mutant and a wild type is, for the most part, real differentiation. The median is a robust estimate that is consistent with the median polishing method. Our treatment of reference in normalization is somewhat different from existing methods, and correspondingly we use this group median polishing summarization to take into account the reference effect. We note that summarization is done at the probe set level. Roughly speaking, the modified median-polishing summarization aims to remove the reference-specific effect for each probe set as well as the probe-specific effect.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>The Yeast2.0 Array contains probe sets for both <em>S. cerevisiae</em> and <em>S. pombe</em>. The observed fluorescence intensities of <em>S. pombe</em> probes are primarily due to cross hybridization, and we only use them in normalization. To some extent, they play the role of external controls. Most genes correspond to only one probe set in Yeast2.0 Array, and for genes with multiple probe sets we take the average of fold changes.</p>


<h4>Wilcoxon scoring of gene subsets</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Based on the expression fold changes of the three mutant strains with respect to the wild type, we make inference about the modifications of biological activities using gene subsets defined by Gene Ontology (GO), KEGG Pathways, and cellular organelle (GFP fusion localization). A common theme of these analyses is as follows. Suppose we have <em>m</em> gene subsets <em>S</em><sub>1</sub>,<em>S</em><sub>2</sub>,…,<em>S<sub>m</sub></em>. From the log ratios of expression levels of a mutant with respect to the wild type, we want to identify those subsets whose expressions are significantly up-regulated or down-regulated. Denote the union of these gene subsets by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.e001&amp;representation=PNG" class="inline-graphic"></span>. Our strategy consists of two steps. In the first step, for each subset <em>S<sub>i</sub></em>, we compare their expressions against those in the complement of <em>S<sub>i</sub></em> in <em>G</em> denoted by <em>G</em>−<em>S<sub>i</sub></em>. This is a typical two-sample problem in statistics. We use the Wilcoxon rank test to calculate p-value (one-sided) for each comparison. In the second step, we rank these subsets according to their significances. These subsets could be a GO category, a metabolic pathway, or protein genes localized in an organelle. Other tests and methods such as GSEA <a href="#pone.0001095-Subramanian1">[24]</a> could be applied to our study. We report the results by the Wilcoxon scoring due to its well-established statistical properties such as robustness and reasonably good efficiency.</p>


<h4>Multiple test correction</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To correct for multiple testing, we adjust the p-values by the method introduced by Storey <em>et al</em>. <a href="#pone.0001095-Storey1">[25]</a>, <a href="#pone.0001095-Storey2">[26]</a>. In this method, the q-value is defined to evaluate the false discovery rate. The computation of q-values is implemented by the “qvalue” package provided in the R software (<a href="http://www.r-project.org/">http://www.r-project.org/</a>).</p>


<h4>Gene Ontology analysis</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Gene ontology information is from “<a href="http://ftp://genome-ftp.stanford.edu/pub/go/">ftp://genome-ftp.stanford.edu/pub/go/</a>”. The Gene Ontology subsets are defined from three related yet different aspects: biological processes, molecular functions and cellular components. Data structure for gene ontology (GO) is a directed acyclic graph (DAG). Each node in the DAG is a set of genes with specific annotation. The closer the nodes are to the terminal, the more detailed annotations are given and thereby are more informative. To avoid redundancy and overlapping between GO nodes and to facilitate our statistical analysis, we select from the DAG the nodes that are closest to the terminal and have at least 30 genes. This selection ends up with nodes of 44 cellular components, 53 molecular functions and 109 biological processes. The gene subsets defined by these nodes are referred to as terminal informative GO categories (TIGO). Then we apply the Wilcoxon scoring method and multiple test correction to these TIGO categories. By taking only the terminal informative TIGO categories rather than all the GO nodes, results are easier to be interpreted.</p>


<h4>KEGG Pathway analysis</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The pathway information are from the KEGG database: <a href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>. In total our study uses 103 <em>S. cerevisiae</em> pathways, most of which are well-established metabolic pathways. To expand our knowledge of ageing, we seek pathways that are significantly changed in the long-lived mutants. We regard each pathway as a subset of genes, and apply our statistical scoring and significance analysis to the 103 pathways and obtain a p-value and a q-value for each of them.</p>


<h4>Cellular organelle analysis</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The cellular localization data are from <a href="http://yeastgfp.ucsf.edu/">http://yeastgfp.ucsf.edu/</a>. In this data set, 75% proteins were classified into 22 distinct subcellular localization categories, including mitochondria, nucleus, nucleolus, vacuole, vacuole membrane, budding neck, etc. Many research indicate that mitochondrion plays a central role in ageing. We also expected that the cellular organelle analysis would provide some information about the role of the different organelles in ageing. In this analysis, genes that function in the same cellular localization are regarded as one gene subset. The protein gene subsets from the yeast GFP fusion localization database are different from the cellular components in the GO categories.</p>


<h4>Consensus and local Inference</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>After making inference using each of the three biological instruments: Gene Ontology, KEGG Pathways, and cellular organelle, we can report a consensus by cross-examination. Our another approach is to compare two gene subsets in a natural “biological block”. For one example, we consider the first half and second half of an energetic pathway. For another example, we consider ribosomes in cytosol and ribosomes in mitochondria. Using this idea of local comparison, we get over the issue of multiple testing and thereby improve the statistical significance. The scheme of our inference is illustrated in <a href="#pone-0001095-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0001095-g001"><div class="img"><a name="pone-0001095-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Scheme of consensus inference and local inference.</span></strong></p><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1"></a><p>After the preprocessing of the microarray data for the long-lived cells versus the wild type, we translate the differentiation into changes of biological activities by consensus and local inference using three kinds of biological instruments: Gene Ontology (GO), KEGG pathways, and cellular localization of proteins.</p>
<span>doi:10.1371/journal.pone.0001095.g001</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Results"></a><h3>Results</h3>
<h4>Preprocessing of microarray data</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We obtained the gene expression profiles of yeast strains including wild type, <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ cells at day 2.5 using Affymetrix GeneChip® Yeast2.0 Array. In total, three biological replicates were generated for each strain. RNA of these replicates was obtained from independent populations which were grown in separate flasks under similar conditions. The expression fold changes of 5841 yeast genes were obtained for <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ with respect to the wild type by the Sub-Sub normalization <a href="#pone.0001095-Cheng1">[21]</a> followed by the modified median-polishing summarization (<a href="#s2">Materials and methods</a>) that aims to remove the reference-specific effect. We optimized the parameters in the Sub-Sub normalization by examining the results among replicates, and by checking probe sets of <em>S. pombe</em> on the Yeast2.0 Array. In this normalization, we divide each array into sub-arrays and normalize probe intensities within each sub-array by least trimmed squares to protect differentiation. The subarray size is selected to be 50 by 50; subarrays overlap by half the subarray size; and the trimming fraction of least trimmed squares is 0.45.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>If the experiment conditions and mRNA amount for the reference and target samples are similar, we argue in <a href="#pone.0001095-Li1">[23]</a> that a simple linear function is a good approximation in normalization even though the relationship between dye concentration and fluorescent intensity is nonlinear. Namely, to normalize a target array with respect to a reference, we shift and scale the probe intensities by α+β*target intensity in such a way that the differences with reference intensities are minimized. Specifically, the parameter α and β are estimated by least trimmed squares. In <a href="#pone-0001095-g002">Figure 2</a>, we show the estimates of the relative scale β in each subarray for <em>sch9</em>Δ versus wild type. Since we normalize each of the three target (mutant) arrays versus each of the three reference (wild type) arrays, total nine spatial patterns are shown in <a href="#pone-0001095-g002">Figure 2</a>, in which each row corresponds to a reference and each column corresponds to a target. The corresponding histograms of the scale parameters are shown in <a href="#pone.0001095.s001">Figure S1</a>. The adjustment of spatial effect can help reduce the variation between replicates, see examples in <a href="#pone.0001095-Cheng1">[21]</a>, <a href="#pone.0001095-Li1">[23]</a>. In this case, for each gene the standard deviation of the nine (three targets versus three references, see <a href="#s2">Materials and methods</a>) expression log-ratios of a mutant versus wild type is calculated. The medians of these standard deviations for <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ are respectively 0.146, 0.125, 0.134.</p>
<div class="figure" id="pone-0001095-g002"><div class="img"><a name="pone-0001095-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Non-homogeneous spatial patterns of relative scales.</span></strong></p><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1"></a><p>The estimates of the normalization scale in each subarray are shown by their spatial locations. We normalize each of the three target (sch9 mutant) arrays versus each of the three reference (wild type) arrays, and total nine spatial patterns are shown. Each row corresponds to a reference and each column corresponds to a target.</p>
<span>doi:10.1371/journal.pone.0001095.g002</span></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3"></a><p>In <a href="#pone-0001095-g003">Figure 3</a> we show the M-A plots of expression levels of three mutants versus the wild type. In the M-A plots, the x, y coordinate value of a dot respectively show the average and difference of a gene expression levels between a mutant and the wild type. The differentiation for probe sets of <em>S. pombe</em> are around zero as expected, especially at the left end. In fact, the medians of the average log ratios for <em>S. pombe</em> genes are 0.018, 0.047, 0.031 respectively for <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ. Some probe sets correspond to homologs of <em>S. pombe</em> and <em>S. cerevisiae</em>, and are expected to be expressed in both. We note that the results are obtained in a blind fashion, for we do not separate <em>S. pombe</em> and <em>S. cerevisiae</em> probes in normalization. Therefore, the probe sets of <em>S. pombe</em> are used in both training and validation. Some differences among the three M-A plots are observed. This is not a surprise because the three genes <em>sch9</em>, <em>ras2</em>, <em>tor1</em> do play some different roles according to what we know.</p>
<div class="figure" id="pone-0001095-g003"><div class="img"><a name="pone-0001095-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>M-A plots of sub-sub normalization followed by modified median polishing summarization.</span></strong></p><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1"></a><p>In the M-A plots, the x, y coordinate value of a dot respectively show the average and difference of a gene expression between the wild type and a mutant. The differentiation for most probe sets of S. pombe are around zero as expected. Black: S. cerevisiae; Magenta: S. pombe. Top: sch9 mutant; Middle: ras2 mutant; Bottom: tor1 mutant. Two horizontal lines at ±0.15 are also plotted.</p>
<span>doi:10.1371/journal.pone.0001095.g003</span></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4"></a><p>We check the M-A plot of raw data without any normalization. That is, for three replicates of a mutant and the wild type, we summarize their expression values by the median polishing method. Then we calculate the average and difference expression of two strains and show the result by M-A plots, see <a href="#pone.0001095.s002">Figure S2</a>. The differentiation for probe sets of <em>S. pombe</em> is mainly distributed along the horizontal direction. This suggests that we can shift the differentiation of <em>S. cerevisiae</em> genes by the median of the differentiation of <em>S. pombe</em> genes. We compare the expressions resulted from this simple median-shift normalization with expression from the above method, the correlation coefficients are respectively 0.990, 0.970, 0.988 for <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5"></a><p>The similarity indicates that quality of the microarrays is relatively good. Finally, the expression results are confirmed by quantitative RT-PCR for eleven genes and northern blots for two genes.</p>
<a id="article1.body1.sec3.sec1.p6" name="article1.body1.sec3.sec1.p6"></a><p>We calculated the one-sided Wilcoxon rank test score for each TIGO subset versus the rest genes, and ranked these subsets according to their corresponding q-values respectively for the up-regulated and down-regulated case. The same computation was carried out for KEGG pathways. These results are rather lengthy, and we report the most significant parts later in an integrative way. The complete spreadsheets can be found in the supplementary materials, <a href="#pone.0001095.s004">Text S1</a>, <a href="#pone.0001095.s006">Text S3</a> and <a href="#pone.0001095.s007">Text S4</a>. The result of the cellular organelle analysis (GFP fusion localization) is summarized in <a href="#pone-0001095-t001">Table 1</a>, and the details can be found in <a href="#pone.0001095.s005">Text S2</a>.</p>
<div class="figure" id="pone-0001095-t001"><div class="img"><a name="pone-0001095-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Positively- and negatively-regulated cellular organelles.</span></strong></p><span>doi:10.1371/journal.pone.0001095.t001</span></div>

<h4>Lower transcriptional activities</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Our analysis shows that the overall transcription activities of the three long-lived mutants are relatively lower than those of the wild type. In fact, in the comparison of KEGG pathway activities, “basal transcription factors” and “RNA polymerase”, are among the most negatively regulated pathways, which also include “DNA polymerase”, “Cell cycle” and “Proteasome”, see <a href="#pone-0001095-t002">Table 2</a>. The basal transcription factors form a complex that acts as a general transcription machine. One explanation is that <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ mutants in the nutrient-depleted environment can live a more economical life and hence a lower basal transcription is sufficient to maintain survival. In consistent with lower transcriptional level, proteasome, the complex in charge of protein degradation, is negatively regulated in the long-lived mutants. Moreover, in <a href="#pone-0001095-t003">Table 3</a>, we list the expression activities of relevant pre-transcription and post-transcription TIGO categories, which are all down-regulated with sufficient statistical evidence. From the perspective of protein localizations, we see in <a href="#pone-0001095-t001">Table 1</a> that the expression activities of the compartment nucleus and nucleolus are significantly lower in the mutants <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ compared to the wild type. It is noticed that reduction of transcriptional activities is most significant in <em>ras2</em>Δ, and this is confirmed by the overall expression profiles shown in <a href="#pone-0001095-g003">Figure 3</a>. All these evidences lead to the consensus that the three long-lived mutants somehow manage to lower their transcription activities in the SDC medium.</p>
<div class="figure" id="pone-0001095-t002"><div class="img"><a name="pone-0001095-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Most negatively regulated KEGG pathways in the long-lived mutants compared to the wild type.</span></strong></p><span>doi:10.1371/journal.pone.0001095.t002</span></div><div class="figure" id="pone-0001095-t003"><div class="img"><a name="pone-0001095-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Down-regulation of TIGO categories relating to transcription.</span></strong></p><span>doi:10.1371/journal.pone.0001095.t003</span></div>

<h4>Switch of energy pathway</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>The universal “currency” of chemical energy ATP in animal cells and most other non-photosynthetic cells is generated mainly by the aerobic oxidation process. In aerobic oxidation, glucose is metabolized to CO<sub>2</sub> and H<sub>2</sub>O, and the released energy is converted to the chemical energy of phosphoanhydride bonds in ATP. The initial steps of oxidation of glucose, referred to as glycolysis, convert glucose into pyruvate. The reactions of glycolysis occur in the cytosol in both eukaryotes and prokaryotes and do not require O<sub>2</sub>. In contrast, the final steps of oxidation require O<sub>2</sub> and occur in mitochondria in eukaryotes. The synthesis of ATP in mitochondria is driven by the flow of electrons from the reduced coenzymes NADH and FADH<sub>2</sub> to O<sub>2</sub>. This oxidative phosphorylation process depends on the generation of proton-motive force across the inner membrane, with electron transport and proton pumping. Other than glycolysis, reactions in the mitochondria such as the citric acid cycle (TCA) also generate the reduced coenzymes NADH and FADH<sub>2</sub>.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>In our KEGG pathway analysis, the most up-regulated metabolic pathways common to all three mutants with respect to the wild type is Glycolysis/Gluconeogenesis. Since much is known about the aerobic oxidation process, we examine the details of the expression differentiation between the long-lived strains and wild type. Specifically, we compare the expression activities of the initial steps and final steps using the Wilcoxon rank test. At the bottom of <a href="#pone-0001095-g004">Figure 4</a> we show the statistical results for the comparison of the KEGG pathways: glycolysis/gluconeogenesis versus TCA and oxidative phosphorylation. The analysis implies that in the long-lived mutants, TCA cycle and oxidative phosphorylation are negatively regulated compared with Glycolysis/Gluconeogenesis. The comparison is shown by box plots at the top of <a href="#pone-0001095-g004">Figure 4</a>. Usually yeast becomes hypo-metabolic (respiration rates decrease) around day 3–5. One explanation to the observation is that the long-lived become hypometabolic faster than the wild type.</p>
<div class="figure" id="pone-0001095-g004"><div class="img"><a name="pone-0001095-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Comparison of Glycolysis/Gluconeogenesis, TCA, and Oxidative phosphorylation.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Top panel: The box plots of the log-ratios of gene expressions for Glycolysis/Gluconeogenesis, TCA, and Oxidative phosphorylation in the long-lived mutants with respect to wild type; bottom: statistical significances of the above comparisons using the Wilcoxon rank test.</p>
<span>doi:10.1371/journal.pone.0001095.g004</span></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p><em>Hxt</em>2 and <em>Hxt</em>4 are both high-affinity glucose transporters, whose expressions are induced by low levels of glucose and repressed by high levels of glucose <a href="#pone.0001095-Ozcan1">[27]</a>, <a href="#pone.0001095-Ozcan2">[28]</a>. Our result indicates that dysfunction of either <em>sch9</em>Δ, <em>ras2</em>Δ, or <em>tor1</em>Δ leads to significant up-regulation of <em>Hxt</em>2 (log-ratios 1.63, 0.83 1.54) and <em>Hxt</em>4 (log-ratios 1.15, 1.66, 0.67), and thereby a more efficient usage of glucose. Consistently, the TIGO category associated with monosaccharide catabolism is also positively modified. Genes in this TIGO category participate in chemical reactions that lead to breakdown of monosaccharides and polyhydric alcohols.</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4"></a><p>Significant changes of other related pathways are also observed. The fructose and mannose metabolism are positively regulated in <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ, the galactose metabolism, starch and sucrose metabolism are positively regulated in <em>ras2</em>Δ, although not as significant as the Glycolysis/Gluconeogenesis pathway. They indicate that the up-regulation of Glycolysis/Gluconeogenesis is associated with modifications of other catabolic pathways in the mutant cells.</p>


<h4>Change of compartment activity</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>In yeast, the reactions of TCA cycle, electron transport and oxidative phosphorylation occur inside mitochondria whereas those of glycolysis occur in cytosol. In aerobic conditions, oxidative phosphorylation is efficient to generate ATPs, but at the same time it produces the reactive oxygen species (ROS) as byproducts, which is thought to be one of the causes of ageing.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>The change of energy pathways leads us to consider change of compartment activities. From <a href="#pone-0001095-t001">Table 1</a>, the overall expression levels of mitochondria is significantly lower in <em>sch9</em>Δ, <em>ras2</em>Δ, <em>tor1</em>Δ. Moreover, in <a href="#pone-0001095-t004">Table 4</a> we examine the expression differentiations for all five TIGO categories specifically associated with mitochondria, which include mitochondrial large ribosomal subunit, mitochondrial small ribosomal subunit, mitochondrial inner membrane, mitochondrion organization and biogenesis and protein-mitochondrial targeting. Their expression activities are consistently down-regulated in all three mutants. Thus we hypothesize that the reduction of biological activities in mitochondria may lead to elongation of chronological life span in <em>S. cerevisiae</em>. In contrast, our GO analysis shows that the TIGO categories, cytosolic large ribosomal subunit (GO:0005842), and cytosolic small ribosomal subunit (GO:0005843) are positively regulated in all the long-lived mutants. Consistent with the GO analysis, the ribosome pathway excluding mitochondria ribosomal subunits is positively regulated in the KEGG analysis. Furthermore, we directly examine expression differentiation between cytosolic ribosomes and mitochondrial ribosomes by the Wilcoxon rank test and all p-values are less than 10<sup>−10</sup>. The comparison is illustrated by box plots in <a href="#pone-0001095-g005">Figure 5</a>.</p>
<div class="figure" id="pone-0001095-g005"><div class="img"><a name="pone-0001095-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Box plots of fold changes between cytosolic and mitochondrial ribosome proteins.</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>The box plots of expression log-ratios for the cytosolic and mitochondrial ribosome proteins in the long-lived mutants with respect to wild type. We also compare the expression differentiation by the Wilcoxon rank test and all three p-values are 0.00.</p>
<span>doi:10.1371/journal.pone.0001095.g005</span></div><div class="figure" id="pone-0001095-t004"><div class="img"><a name="pone-0001095-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>The transcription activities of the long-lived strains with respect to the wild type yeast for five TIGO categories that are associated with mitochondria.</span></strong></p><span>doi:10.1371/journal.pone.0001095.t004</span></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>As shown in <a href="#pone-0001095-t001">Table 1</a>, ER-located and vacuole-located proteins are positively affected. The GO category (GO:0005789) endoplasmic reticulum membrane is up-regulated too. The endoplasmic reticulum is part of the endomembrane system, which modifies proteins, makes macromolecules, and transfers substances throughout the cell. In budding yeast cells, vacuoles are the storage compartments of amino acids and the detoxification compartments. Under conditions of starvation, proteins are degraded in vacuoles, which is called autophagy. The up-regulations of vacuole-located proteins may imply that autophagy in the cells of these long-lived mutants is enhanced to maintain survival in low nutrient conditions such as SDC medium.</p>


<h4>Differences among mutants</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>Despite the common expression patterns in <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ, we do observe various differences. r<em>as</em>2Δ shows the lowest overall transcriptional activities. In addition, we found those activities relating to mitosis in the mutant <em>ras2</em>Δ are all down-regulated, see <a href="#pone-0001095-t005">Table 5</a>. They include regulation of mitosis, mitotic sister chromatid segregation, and mitotic spindle organization and biogenesis. The activities in the cellular components (GO categories) such as spindle pole body, spindle pole body and microtubule cycle, bud neck, bud tip and incipient bud site, provide additional evidence. Similar and independent results are found in the cellular location analysis (<a href="#pone-0001095-t001">Table 1</a>). Other activities relating to DNA replication and DNA repair are down-regulated too. It is in consistency with the lower mitotic activities. It is known that Ras proteins regulate cell growth in response to nutrient availability through protein kinase A (PKA) activity, see <a href="#pone.0001095-Schneper1">[29]</a> for references. Also Ras proteins have PKA-independent functions in mitosis and actin repolarization <a href="#pone.0001095-Morishita1">[30]</a>. The expression profiles from our experiments provide systematic evidences to these stories. The down-regulation of the MAPK signaling pathway is significant in <em>ras</em>2Δ (q-value = 0.009), less slight in <em>tor1</em>Δ, and not in <em>sch9</em>Δ.</p>
<div class="figure" id="pone-0001095-t005"><div class="img"><a name="pone-0001095-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001095" data-uri="info:doi/10.1371/journal.pone.0001095.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001095.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001095.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001095.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001095.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>The transcription activities of <em>ras2</em>Δ with respect to the wild type yeast for TIGO categories relating to mitosis.</span></strong></p><span>doi:10.1371/journal.pone.0001095.t005</span></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>Some differences in metabolisms are also observed. For example, the inositol phosphate metabolism is down-regulated in <em>ras2</em>Δ, and <em>tor1</em>Δ (q-value = 0.013, 0.039), but not in <em>sch9</em>Δ. The phosphatidylinositol signaling system is down-regulated in <em>ras2</em>Δ, and <em>tor1</em>Δ (q-value = 0.036, 0.059), but not in <em>sch9</em>Δ. Although it is difficult to enumerate them all, differences among the three long-lived mutants, from another angle, suggest that the common expression patterns reported above are strongly linked to longevity.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>To understand the mechanisms of ageing, we identified the common and characteristic differentiation in the transcriptional profiles of the three long-lived strains <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ. The success of our effort hinges on the measurement accuracy of mRNA expression levels. In the design of Affymetrix GeneChip®, multiple (11-20) 25-mer probes are used for each ORF (open reading frame) and they serve as within-block statistical replicates. In addition, we do observe higher probe specificity and other improvement in the recent yeast2.0 chips. It should be noted that cross-hybridization always exists and the measured values tend to be smaller than real differentiation.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>Our analysis is based on the sub-array normalization that aims to improve accuracy and preserve differentiation. A simple linear function is sufficient, if not perfect, for the purpose of normalizing our yeast microarrays. First, all the microarray experiments were conducted under the same condition. Second, the estimated scale values are mostly in the range [0.7, 1.3]; see the histograms in <a href="#pone.0001095.s001">Figure S1</a>. It is argued in <a href="#pone.0001095-Li1">[23]</a> that a simple linear function is sufficient in such a normalization case. Third, the hybridization is a complicated process with various uncontrollable factors. It is important to use a highly robust estimator of the linear function to eliminate unpredictable probe intensities. Least trimmed squares is an appropriate choice due to its robustness in several senses. Fourth, the fair modification of expression profiles from normalization should vary from one situation to another. In our case, by comparing <a href="#pone-0001095-g003">Figure 3</a> and <a href="#pone.0001095.s002">Figure S2</a>, we feel the fair modification of expression profiles from normalization should not be large. We made an effort to preprocess this yeast microarray data set and examined the validity of the presented results from the perspectives of <em>S. pombe</em> probes sets, non-normalized expression profiles, and other considerations and supporting examples reported in our previous work. It is our belief that the future of functional genomics and proteomics lies not only in scale but also in measurement accuracy. To be consistent with the sub-array normalization, we use the median polishing summarization stratified by the reference selected from raw wild type arrays. Additional investigation of the reference effect is worthwhile in the future research.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3"></a><p>The common changes of biological activities in the differentiation are inferred by integrating the expression profiles with biological subsets defined by cellular organelles, metabolism pathways, biological process, and molecular functions. We choose to use those instruments from three sources: cellular localization of proteins, KEGG, and Gene Ontology. Gene Ontology compiles results from literature along three dimensions, and some categories overlap with those from the other two sources. Similarly, we can make inference about the transcriptional regulation in long-lived mutants using expression profiles together with information of ChIP-chip and binding motifs. The results are described in other reports. We use the q-value method developed by Storey <em>et al</em>. <a href="#pone.0001095-Storey1">[25]</a>, <a href="#pone.0001095-Storey2">[26]</a> to deal with the multiple test issue. The definitions and algorithms of q-values were initially obtained based on several assumptions, one of which is that the null distribution of the p-value is uniform[0,1]. In <a href="#pone.0001095.s003">Figure S3</a>, we show the histograms of p-values from the KEGG pathway analysis. Besides, the same genes could be shared by multiple subsets and dependence among hypotheses exists. The sensitivity of q-values to these assumptions in our study is a subtle problem and it is worth more investigation in our future work.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4"></a><p>Other than subset-versus-all comparisons, we also make local comparisons between two directly-related subsets such as cytosolic and mitochondrial ribosome proteins. If the “directly-related gene subsets” are appropriately selected, in our opinion, the conclusion drawn from microarray studies can be greatly strengthened by this local inference approach. Our consensus inference is obtained by the cross-examination of the inferences drawn from different instruments.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5"></a><p>Our results show that mutants <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ, which share the same phenotype: longer chronological life span, do share some common differentiation patterns. The commonality is particularly interesting in the presence of various differences. For example, the activities relating to mitosis in <em>ras2</em>Δ are significantly reduced. The significant and systematic expression differentiation underlying the phenotype is critical for understanding ageing in yeast. One such feature is lower pre- and post-transcriptional activities.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6"></a><p>Another common characteristics in long-lived strains is the down-regulation of TCA cycle and oxidative phosphorylation. In contrast, the upstream of this process, the Glycolysis/Gluconeogenesis pathway, is slightly or moderately up-regulated. The up-regulation of genes relating to Glycolysis/Gluconeogenesis implies that mutant cells consume the carbon sources in a different manner compared to the wild type. The adaption may be achieved through a mechanism similar to that in CR. On the other hand, the down-regulation of genes relating to TCA cycle and oxidative phosphorylation indicates that mutant cells switch to alternative energy pathways that likely depend on glycolysis. Rea <em>et al.</em> <a href="#pone.0001095-Rea1">[31]</a> proposed a metabolic model to describe the “Energy switch” hypothesis for longevity mutants in <em>C. elegans</em>. They suggested that most, if not all, long-lived mutants in <em>C. elegans</em> utilize anaerobic mitochondrial fermentation, which do not involve the electron transport chain and generate fewer radical species. Our results indicate that the notion of “energy switch” may be relevant for explaining life span extension in <em>S. cerevisiae</em>. However, rather than anaerobic mitochondrial fermentation, in the yeast strains <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ, the alternative energy pathway is likely to involve glycolysis and occur in cytosol or organelles other than mitochondria.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7"></a><p>Evidences from the analysis of cellular organelle, GO, and the comparison of cytosolic and mitochondrial ribosomes all indicate that activities of mitochondria are significantly reduced in <em>sch9</em>Δ, <em>ras2</em>Δ, and <em>tor1</em>Δ. In contrast, the expressions of cytosolic ribosomes are up-regulated. This change of compartment activities supports the ROS theory, which says reactive oxygen species (ROS) damage macromolecules and thereby accelerate ageing. The majority of cellular ROS (approximately 90%) is generated in mitochondria as a byproduct of oxidative phosphorylation during respiration <a href="#pone.0001095-Balaban1">[32]</a>. A number of mutations affecting respiration have been found to increase life span, and at least some may achieve this by decreasing ROS levels <a href="#pone.0001095-Lee1">[34]</a>. According to our analysis, many of the down-regulated genes encode mitochondrial proteins; conversely, the expression levels of genes that encode proteins localized in mitochondria tend to be negatively regulated in the long-lived mutants. Particularly, in the long-lived mutants, TCA and oxidative phosphorylation are negatively affected, both of which occur in the mitochondria. As a consequence, respiration is reduced and thereby less ROS are produced. Our observations and implications are consistent with results from a systematic RNA interference (RNAi) screen of 5,690 <em>Caenorhabditis elegans</em> genes for gene inactivations that increase lifespan <a href="#pone.0001095-Lee1">[34]</a>. They found that genes important for mitochondrial function stand out as a principal group of genes affecting <em>C. elegans</em> lifespan. Our results in yeast suggest that reduction of mitochondrial activities closely relates to extension of the yeast chronological life span.</p>
</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001095"></div><a name="pone.0001095.s001" id="pone.0001095.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Histograms of the scale values in <a href="#pone-0001095-g002">Figure 2</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.02 MB EPS)</p>
<a name="pone.0001095.s002" id="pone.0001095.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s002">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">M-A plots of data without normalization. In the M-A plots, the x, y coordinate value of a dot respectively show the average and difference of a gene expressions between the wild type and a mutant. Black: S. cerevisiae; Magenta: S. pombe. Top: sch9 mutant; Middle: ras2 mutant; Bottom: tor1 mutant. Two horizontal lines at ±0.15 are also plotted.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.86 MB EPS)</p>
<a name="pone.0001095.s003" id="pone.0001095.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s003">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Histograms of the p-values in the KEGG Pathway analysis. p-values are calculated for one-sided Wilcoxon tests that compare differentiation between mutants and wild type on gene subsets defined by KEGG Pathways. Left: sch9 mutant vs. wild; Middle: ras2 mutant vs. wild type; Bottom: tor1 mutant vs. wild type.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.01 MB EPS)</p>
<a name="pone.0001095.s004" id="pone.0001095.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s004">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Spreadsheet of KEGG analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
<a name="pone.0001095.s005" id="pone.0001095.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s005">Text S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Spreadsheet of Cellular organelle analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.00 MB XLS)</p>
<a name="pone.0001095.s006" id="pone.0001095.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s006">Text S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Spreadsheet of TIGO analysis</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001095.s007" id="pone.0001095.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001095.s007">Text S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Information of the TIGO categories</p>
<p class="siDoi">doi:10.1371/journal.pone.0001095.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.01 MB TXT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors would like to thank Dr. Minghua Deng for his help on the definition of TIGO.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: VL LL PF. Performed the experiments: PF MW. Analyzed the data: LL CC HG. Contributed reagents/materials/analysis tools: VL LL CC PF. Wrote the paper: LL CC.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001095-Kenyon1" id="pone.0001095-Kenyon1"></a>Kenyon C (2001) A conserved regulatory system for aging. Cell  105: 165–168.  <ul class="find" data-citedArticleID="1010702" data-doi="10.1016/s0092-8674(01)00306-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(01)00306-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+conserved+regulatory+system+for+aging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+conserved+regulatory+system+for+aging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001095-Longo1" id="pone.0001095-Longo1"></a>Longo VD, Finch CE (2003) Evolutionary medicine:from drawf model systems to healthy centenarians. Science  299: 1342–1346.  <ul class="find" data-citedArticleID="1010716" data-doi="10.1126/science.1077991"><li><a href="http://dx.doi.org/10.1126/science.1077991" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+medicine%3Afrom+drawf+model+systems+to+healthy+centenarians." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+medicine%3Afrom+drawf+model+systems+to+healthy+centenarians.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001095-Fabrizio1" id="pone.0001095-Fabrizio1"></a>Fabrizio P, Longo V (2003) The chronological life span of <em>Saccharomyces cerevisiae</em>. Aging Cell  2: 73–81.  <ul class="find" data-citedArticleID="1010684" data-doi="10.1046/j.1474-9728.2003.00033.x"><li><a href="http://dx.doi.org/10.1046/j.1474-9728.2003.00033.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chronological+life+span+of+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chronological+life+span+of+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001095-Kaeberlein1" id="pone.0001095-Kaeberlein1"></a>Kaeberlein M, Kennedy B (2005) Large-scale identification in yeast of conserved ageing genes. Mechanisms of Ageing and Development  126: 17–21.  <ul class="find" data-citedArticleID="1010698" data-doi="10.1016/j.mad.2004.09.013"><li><a href="http://dx.doi.org/10.1016/j.mad.2004.09.013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+identification+in+yeast+of+conserved+ageing+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+identification+in+yeast+of+conserved+ageing+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001095-Kaeberlein2" id="pone.0001095-Kaeberlein2"></a>Kaeberlein M, Kirkland KT, Fields S, Kennedy B (2005) Genes determining yeast replicative life span in a long-lived genetic background. Mechanisms of Ageing and Development  126: 491–504.  <ul class="find" data-citedArticleID="1010700" data-doi="10.1016/j.mad.2004.10.007"><li><a href="http://dx.doi.org/10.1016/j.mad.2004.10.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+determining+yeast+replicative+life+span+in+a+long-lived+genetic+background." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+determining+yeast+replicative+life+span+in+a+long-lived+genetic+background.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001095-Fabrizio2" id="pone.0001095-Fabrizio2"></a>Fabrizio P, Pozza F, Pletcher SD, Gendron LVCM (2001) Regulation of longevity and stress resistance by Sch9 in yeast. Science  292: 288–90.  <ul class="find" data-citedArticleID="1010686" data-doi="10.1126/science.1059497"><li><a href="http://dx.doi.org/10.1126/science.1059497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+longevity+and+stress+resistance+by+Sch9+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+longevity+and+stress+resistance+by+Sch9+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001095-Fabrizio3" id="pone.0001095-Fabrizio3"></a>Fabrizio P, Liou LL, Moy VN, Diaspro A, Selverstone Valentine J, et al.  (2003) SOD2 functions downstream of Sch9 to extend longevity in yeast. Genetics  163: 35–46.  <ul class="find" data-citedArticleID="1010688"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SOD2+functions+downstream+of+Sch9+to+extend+longevity+in+yeast.&amp;auth=&amp;atitle=SOD2+functions+downstream+of+Sch9+to+extend+longevity+in+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SOD2+functions+downstream+of+Sch9+to+extend+longevity+in+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SOD2+functions+downstream+of+Sch9+to+extend+longevity+in+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001095-Powers1" id="pone.0001095-Powers1"></a>Powers R, M K, SD C, BK K, S F (2006) Extension of chronological life span in yeast by decreased tor pathway signaling. Genes &amp; Development  20: 174–184.  <ul class="find" data-citedArticleID="1010728" data-doi="10.1101/gad.1381406"><li><a href="http://dx.doi.org/10.1101/gad.1381406" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extension+of+chronological+life+span+in+yeast+by+decreased+tor+pathway+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extension+of+chronological+life+span+in+yeast+by+decreased+tor+pathway+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001095-Kirkwood1" id="pone.0001095-Kirkwood1"></a>Kirkwood TB (1988) The nature and causes of ageing. Ciba Found Symp  134: 193–207.  <ul class="find" data-citedArticleID="1010706" data-doi="10.1002/9780470513583.ch13"><li><a href="http://dx.doi.org/10.1002/9780470513583.ch13" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+nature+and+causes+of+ageing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+nature+and+causes+of+ageing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001095-Kirkwood2" id="pone.0001095-Kirkwood2"></a>Kirkwood TB (1992) Comparative life spans of species: why do species have the life spans they do? Am J Clin Nutr  55: 1191S–1195S.  <ul class="find" data-citedArticleID="1010708"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Comparative+life+spans+of+species%3A+why+do+species+have+the+life+spans+they+do%3F&amp;auth=&amp;atitle=Comparative+life+spans+of+species%3A+why+do+species+have+the+life+spans+they+do%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+life+spans+of+species%3A+why+do+species+have+the+life+spans+they+do%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+life+spans+of+species%3A+why+do+species+have+the+life+spans+they+do%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001095-Kirkwood3" id="pone.0001095-Kirkwood3"></a>Kirkwood TB (2002) Evolution of ageing. Mech Ageing Dev  123: 737–745.  <ul class="find" data-citedArticleID="1010710" data-doi="10.1016/s0047-6374(01)00419-5"><li><a href="http://dx.doi.org/10.1016/s0047-6374(01)00419-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+ageing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+ageing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001095-Partridge1" id="pone.0001095-Partridge1"></a>Partridge L (2001) Evolutionary theories of ageing applied to long-lived organisms. Exp Gerontol  36: 641–650.  <ul class="find" data-citedArticleID="1010726" data-doi="10.1016/s0531-5565(00)00232-1"><li><a href="http://dx.doi.org/10.1016/s0531-5565(00)00232-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+theories+of+ageing+applied+to+long-lived+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+theories+of+ageing+applied+to+long-lived+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001095-Jenkins1" id="pone.0001095-Jenkins1"></a>Jenkins NL, McColl G, Lithgow GJ (2004) Fitness cost of extended lifespan in <em>Caenorhabditis elegans</em>. Proc Biol Sci  271: 2523–2526.  <ul class="find" data-citedArticleID="1010696" data-doi="10.1098/rspb.2004.2897"><li><a href="http://dx.doi.org/10.1098/rspb.2004.2897" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fitness+cost+of+extended+lifespan+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fitness+cost+of+extended+lifespan+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001095-Longo2" id="pone.0001095-Longo2"></a>Longo VD, Mitteldorf J, Skulachev VP (2005) Programmed and altruistic ageing. Nat Rev Genet  6: 866–872.  <ul class="find" data-citedArticleID="1010718" data-doi="10.1038/nrg1706"><li><a href="http://dx.doi.org/10.1038/nrg1706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Programmed+and+altruistic+ageing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Programmed+and+altruistic+ageing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001095-Harman1" id="pone.0001095-Harman1"></a>Harman D (1956) Aging: atheory based on free radical and radiation chemistry. J Gerontol  11: 298–300.  <ul class="find" data-citedArticleID="1010690" data-doi="10.1093/geronj/11.3.298"><li><a href="http://dx.doi.org/10.1093/geronj/11.3.298" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Aging%3A+atheory+based+on+free+radical+and+radiation+chemistry." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Aging%3A+atheory+based+on+free+radical+and+radiation+chemistry.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001095-Harman2" id="pone.0001095-Harman2"></a>Harman D (1972) A biologic clock: the mitochondria? Journal of the American Geriatrics Society  20: 145–147.  <ul class="find" data-citedArticleID="1010692"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+biologic+clock%3A+the+mitochondria%3F&amp;auth=&amp;atitle=A+biologic+clock%3A+the+mitochondria%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+biologic+clock%3A+the+mitochondria%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+biologic+clock%3A+the+mitochondria%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001095-Schadt1" id="pone.0001095-Schadt1"></a>Schadt EE, Li C, Su C, Wong WH (2000) Analyzing high-density oligonucleotide gene expression array data. Journal of Cellular Biochemistry  80: 192–202.  <ul class="find" data-citedArticleID="1010732" data-doi="10.1002/1097-4644(20010201)80:2<192::aid-jcb50&gt;3.0.co;2-w"><li><a href="http://dx.doi.org/10.1002/1097-4644(20010201)80:2<192::aid-jcb50&gt;3.0.co;2-w" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyzing+high-density+oligonucleotide+gene+expression+array+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyzing+high-density+oligonucleotide+gene+expression+array+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001095-Schadt2" id="pone.0001095-Schadt2"></a>Schadt EE, Li C, Ellis B, Wong WH (2001) Feature extraction and normalization algorithms for high-density oligonucleotide gene expression array data. Journal of Cellular Biochemistry Supplement  37: 120–125.  <ul class="find" data-citedArticleID="1010734" data-doi="10.1002/jcb.10073"><li><a href="http://dx.doi.org/10.1002/jcb.10073" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Feature+extraction+and+normalization+algorithms+for+high-density+oligonucleotide+gene+expression+array+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Feature+extraction+and+normalization+algorithms+for+high-density+oligonucleotide+gene+expression+array+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001095-Bolstad1" id="pone.0001095-Bolstad1"></a>Bolstad B, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on bias and variance. Bioinformatics  19: 185–193.  <ul class="find" data-citedArticleID="1010680" data-doi="10.1093/bioinformatics/19.2.185"><li><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+bias+and+variance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+comparison+of+normalization+methods+for+high+density+oligonucleotide+array+data+based+on+bias+and+variance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001095-Irizarry1" id="pone.0001095-Irizarry1"></a>Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, et al.  (2003) Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics  4: 249–264.  <ul class="find" data-citedArticleID="1010694" data-doi="10.1093/biostatistics/4.2.249"><li><a href="http://dx.doi.org/10.1093/biostatistics/4.2.249" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exploration%2C+normalization%2C+and+summaries+of+high+density+oligonucleotide+array+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001095-Cheng1" id="pone.0001095-Cheng1"></a>Cheng C, Li LM (2005) Sub-array normalization subject to differentiation. Nucleic Acids Research  33: 5565–5573.  <ul class="find" data-citedArticleID="1010682" data-doi="10.1093/nar/gki844"><li><a href="http://dx.doi.org/10.1093/nar/gki844" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sub-array+normalization+subject+to+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sub-array+normalization+subject+to+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001095-Tukey1" id="pone.0001095-Tukey1"></a>Tukey JW (1977) Exploratory Data Analysis. Cambridge, MA: Addison-Wesley.   <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0001095-Li1" id="pone.0001095-Li1"></a>Li LM, Cheng C (2007)  Differentiation detection in microarray normalization. In: Stafford P, editor. DNA Microarray Normalization. CRC press.   <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0001095-Subramanian1" id="pone.0001095-Subramanian1"></a>Subramanian A, Tamayo P, Mootha V, Mukherjee S, Ebert B, et al.  (2005) Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci USA  102: 15545–15550.  <ul class="find" data-citedArticleID="1010742" data-doi="10.1073/pnas.0506580102"><li><a href="http://dx.doi.org/10.1073/pnas.0506580102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+set+enrichment+analysis%3A+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001095-Storey1" id="pone.0001095-Storey1"></a>Storey JD (2002) A direct approach to false discovery rates. Journal of the Royal Statistical Society, Series B  64: 479–498.  <ul class="find" data-citedArticleID="1010738" data-doi="10.1111/1467-9868.00346"><li><a href="http://dx.doi.org/10.1111/1467-9868.00346" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+direct+approach+to+false+discovery+rates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+direct+approach+to+false+discovery+rates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001095-Storey2" id="pone.0001095-Storey2"></a>Storey JD, Tibshirani R (2003) Statistical significance for genome-wide experiment. Proc Natl Acad Sci  100: 9440–9445.  <ul class="find" data-citedArticleID="1010740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Statistical+significance+for+genome-wide+experiment.&amp;auth=&amp;atitle=Statistical+significance+for+genome-wide+experiment." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical+significance+for+genome-wide+experiment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical+significance+for+genome-wide+experiment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001095-Ozcan1" id="pone.0001095-Ozcan1"></a>Ozcan S, Johnston M (1999) Function and regulation of yeast hexose transporters. Microbiol Mol Biol Rev  63: 554–69.  <ul class="find" data-citedArticleID="1010722"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Function+and+regulation+of+yeast+hexose+transporters.&amp;auth=&amp;atitle=Function+and+regulation+of+yeast+hexose+transporters." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Function+and+regulation+of+yeast+hexose+transporters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Function+and+regulation+of+yeast+hexose+transporters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001095-Ozcan2" id="pone.0001095-Ozcan2"></a>Ozcan S, Johnston M (1995) Three different regulatory mechanisms enable yeast hexose transporter (HXT) genes to be induced by different levels of glucose. Mol Cell Biol  15: 1564–72.  <ul class="find" data-citedArticleID="1010724"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Three+different+regulatory+mechanisms+enable+yeast+hexose+transporter+%28HXT%29+genes+to+be+induced+by+different+levels+of+glucose.&amp;auth=&amp;atitle=Three+different+regulatory+mechanisms+enable+yeast+hexose+transporter+%28HXT%29+genes+to+be+induced+by+different+levels+of+glucose." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+different+regulatory+mechanisms+enable+yeast+hexose+transporter+%28HXT%29+genes+to+be+induced+by+different+levels+of+glucose." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+different+regulatory+mechanisms+enable+yeast+hexose+transporter+%28HXT%29+genes+to+be+induced+by+different+levels+of+glucose.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001095-Schneper1" id="pone.0001095-Schneper1"></a>Schneper L, Krauss A, Miyamoto R, Fang S, Broach J (2004) The ras/protein kinase a pathway acts in parallel with the mob2/cbk1 pathway to effect cell cycle progression and proper bud site selection. Eukaryot Cell  3: 108–20.  <ul class="find" data-citedArticleID="1010736" data-doi="10.1128/ec.3.1.108-120.2004"><li><a href="http://dx.doi.org/10.1128/ec.3.1.108-120.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+ras%2Fprotein+kinase+a+pathway+acts+in+parallel+with+the+mob2%2Fcbk1+pathway+to+effect+cell+cycle+progression+and+proper+bud+site+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+ras%2Fprotein+kinase+a+pathway+acts+in+parallel+with+the+mob2%2Fcbk1+pathway+to+effect+cell+cycle+progression+and+proper+bud+site+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001095-Morishita1" id="pone.0001095-Morishita1"></a>Morishita T, Mitsuzawa H, Nakafuku M, Nakamura S, Hattori S, et al.  (1995) Requirement of <em>Saccharomyces cerevisiae</em> ras for completion of mitosis. Science  270: 1213–5.  <ul class="find" data-citedArticleID="1010720" data-doi="10.1126/science.270.5239.1213"><li><a href="http://dx.doi.org/10.1126/science.270.5239.1213" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+of+Saccharomyces+cerevisiae+ras+for+completion+of+mitosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+of+Saccharomyces+cerevisiae+ras+for+completion+of+mitosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001095-Rea1" id="pone.0001095-Rea1"></a>Rea S, Johnson T (2003) A metabolic model for life span determination in <em>Caenorhabditis elegans</em>. Dev Cell  5: 197–203.  <ul class="find" data-citedArticleID="1010730" data-doi="10.1016/s1534-5807(03)00242-9"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00242-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+metabolic+model+for+life+span+determination+in+Caenorhabditis+elegans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+metabolic+model+for+life+span+determination+in+Caenorhabditis+elegans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001095-Balaban1" id="pone.0001095-Balaban1"></a>Balaban R, Nemoto S, Finkel T (2005) Mitochondria, oxidant, and aging. Cell  120: 483–495.  <ul class="find" data-citedArticleID="1010678" data-doi="10.1016/j.cell.2005.02.001"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.02.001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mitochondria%2C+oxidant%2C+and+aging." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mitochondria%2C+oxidant%2C+and+aging.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001095-Kenyon2" id="pone.0001095-Kenyon2"></a>Kenyon C (2005) The plasticity of aging: insight from long-lived mutants. Cell  120: 449–460.  <ul class="find" data-citedArticleID="1010704" data-doi="10.1016/j.cell.2005.02.002"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.02.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+plasticity+of+aging%3A+insight+from+long-lived+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+plasticity+of+aging%3A+insight+from+long-lived+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001095-Lee1" id="pone.0001095-Lee1"></a>Lee S, Lee R, Fraser A, Kamath R, Ruvkun G (2003) A systematic RNAi screen identifies a critical role for mitochondria in <em>C. elegans</em> longevity. Nat Genet  33: 40–48.  <ul class="find" data-citedArticleID="1010712" data-doi="10.1038/ng1056"><li><a href="http://dx.doi.org/10.1038/ng1056" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+systematic+RNAi+screen+identifies+a+critical+role+for+mitochondria+in+C.+elegans+longevity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+systematic+RNAi+screen+identifies+a+critical+role+for+mitochondria+in+C.+elegans+longevity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.XML" value="110067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.PDF" value="2588894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.e001.PNG" value="6523"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.e001.TIF" value="22504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g001.PNG_L" value="184966"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g001.PNG_M" value="39260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g001.PNG_S" value="8600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g001.TIF" value="311922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g001.PNG_I" value="13287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g002.PNG_L" value="64792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g002.PNG_M" value="76095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g002.PNG_S" value="12120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g002.TIF" value="520432"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g002.PNG_I" value="30659"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g003.PNG_L" value="135782"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g003.PNG_M" value="107545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g003.PNG_S" value="13763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g003.TIF" value="611192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g003.PNG_I" value="92078"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t001.PNG_L" value="145722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t001.PNG_M" value="87177"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t001.PNG_S" value="12147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t001.TIF" value="729460"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t001.PNG_I" value="26537"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t002.PNG_L" value="85043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t002.PNG_M" value="52808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t002.PNG_S" value="9333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t002.TIF" value="388792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t002.PNG_I" value="16348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t003.PNG_L" value="108689"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t003.PNG_M" value="69470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t003.PNG_S" value="9795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t003.TIF" value="467830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t003.PNG_I" value="21122"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g004.PNG_L" value="272569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g004.PNG_M" value="85383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g004.PNG_S" value="11933"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g004.TIF" value="595558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g004.PNG_I" value="29870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g005.PNG_L" value="18940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g005.PNG_M" value="23937"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g005.PNG_S" value="2374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g005.TIF" value="166280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.g005.PNG_I" value="10525"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t004.PNG_L" value="87657"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t004.PNG_M" value="55429"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t004.PNG_S" value="9106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t004.TIF" value="372982"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t004.PNG_I" value="16992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t005.PNG_L" value="123221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t005.PNG_M" value="183588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t005.PNG_S" value="21268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t005.TIF" value="551898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.t005.PNG_I" value="61941"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s001.EPS" value="20985"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s002.EPS" value="860247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s003.EPS" value="7200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s004.XLS" value="51200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s005.XLS" value="3459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s006.XLS" value="32644"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001095.s007.TXT" value="13993"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001095&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001095" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001095&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001095&volume=&issue=&title=Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains&author_name=Chao%20Cheng%2C%20Paola%20Fabrizio%2C%20Huanying%20Ge%2C%20Min%20Wei%2C%20Valter%20D.%20Longo%2C%20Lei%20M.%20Li&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001095" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095&amp;t=Significant%20and%20Systematic%20Expression%20Differentiation%20in%20Long-Lived%20Yeast%20Strains" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095&title=Significant%20and%20Systematic%20Expression%20Differentiation%20in%20Long-Lived%20Yeast%20Strains&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095&amp;title=Significant%20and%20Systematic%20Expression%20Differentiation%20in%20Long-Lived%20Yeast%20Strains" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001095&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Significant and Systematic Expression Differentiation in Long-Lived Yeast Strains';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001095';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Significant%20and%20Systematic%20Expression%20Differentiation%20in%20Long-Lived%20Yeast%20Strains http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001095" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001095" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25024"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Glycolysis%22" title="Search for articles in the subject area:'Glycolysis'"><div class="flagText">Glycolysis</div></a>
              <div data-categoryid="44115" data-articleid="25024"
                   data-categoryname="Glycolysis"
                   class="flagImage" title="Flag 'Glycolysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mitochondria%22" title="Search for articles in the subject area:'Mitochondria'"><div class="flagText">Mitochondria</div></a>
              <div data-categoryid="32453" data-articleid="25024"
                   data-categoryname="Mitochondria"
                   class="flagImage" title="Flag 'Mitochondria' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Oxidative+phosphorylation%22" title="Search for articles in the subject area:'Oxidative phosphorylation'"><div class="flagText">Oxidative phosphorylation</div></a>
              <div data-categoryid="44083" data-articleid="25024"
                   data-categoryname="Oxidative phosphorylation"
                   class="flagImage" title="Flag 'Oxidative phosphorylation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ribosomes%22" title="Search for articles in the subject area:'Ribosomes'"><div class="flagText">Ribosomes</div></a>
              <div data-categoryid="32243" data-articleid="25024"
                   data-categoryname="Ribosomes"
                   class="flagImage" title="Flag 'Ribosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Saccharomyces+cerevisiae%22" title="Search for articles in the subject area:'Saccharomyces cerevisiae'"><div class="flagText">Saccharomyces cerevisiae</div></a>
              <div data-categoryid="43143" data-articleid="25024"
                   data-categoryname="Saccharomyces cerevisiae"
                   class="flagImage" title="Flag 'Saccharomyces cerevisiae' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Schizosaccharomyces+pombe%22" title="Search for articles in the subject area:'Schizosaccharomyces pombe'"><div class="flagText">Schizosaccharomyces pombe</div></a>
              <div data-categoryid="33499" data-articleid="25024"
                   data-categoryname="Schizosaccharomyces pombe"
                   class="flagImage" title="Flag 'Schizosaccharomyces pombe' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="25024"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1246'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9017'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=4818&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
