

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001303"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001303" />

    <meta name="citation_title" content="Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice"/>
    <meta itemprop="name" content="Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice"/>

      <meta name="citation_author" content="Luis M. Valor"/>
            <meta name="citation_author_institution" content="Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, United Kingdom"/>
      <meta name="citation_author" content="Seth G. N. Grant"/>
            <meta name="citation_author_institution" content="Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, United Kingdom"/>

    <meta name="citation_date" content="2007/12/12"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001303.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1303"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Gene knockout protocols (Methods in Molecular Biology).; citation_author=M Tymms; citation_author=I Kola; citation_number=1; citation_pages=431; citation_date=1999; citation_publisher=Humana Press, Inc; " />
      <meta name="citation_reference" content="citation_title=Flanking gene and genetic background problems in genetically manipulated mice.; citation_author=WE Crusio; citation_journal_title=Biol Psychiatry; citation_volume=56; citation_number=2; citation_pages=381-385; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Synapse-associated protein 102/dlgh3 couples the NMDA receptor to specific plasticity pathways and learning strategies.; citation_author=PC Cuthbert; citation_author=LE Stanford; citation_author=MP Coba; citation_author=JA Ainge; citation_author=AE Fink; citation_journal_title=J Neurosci; citation_volume=27; citation_number=3; citation_pages=2673-2682; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Enhanced long-term potentiation and impaired learning in mice with mutant postsynaptic density-95 protein.; citation_author=M Migaud; citation_author=P Charlesworth; citation_author=M Dempster; citation_author=LC Webster; citation_author=AM Watabe; citation_journal_title=Nature; citation_volume=396; citation_number=4; citation_pages=433-439; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Requirement of a critical period of transcription for induction of a late phase of LTP.; citation_author=PV Nguyen; citation_author=T Abel; citation_author=ER Kandel; citation_journal_title=Science; citation_volume=265; citation_number=5; citation_pages=1104-1107; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=A protein synthesis-dependent late phase of cerebellar long-term depression.; citation_author=DJ Linden; citation_journal_title=Neuron; citation_volume=17; citation_number=6; citation_pages=483-490; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cognitive neuroscience and the study of memory.; citation_author=B Milner; citation_author=LR Squire; citation_author=ER Kandel; citation_journal_title=Neuron; citation_volume=20; citation_number=7; citation_pages=445-468; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Genes to remember.; citation_author=CM Alberini; citation_journal_title=J Exp Biol; citation_volume=202; citation_number=8; citation_pages=2887-2891; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Transcription-dependent neuronal plasticity the nuclear calcium hypothesis.; citation_author=H Bading; citation_journal_title=Eur J Biochem; citation_volume=267; citation_number=9; citation_pages=5280-5283; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Tissue-plasminogen activator is induced as an immediate-early gene during seizure, kindling and long-term potentiation.; citation_author=Z Qian; citation_author=ME Gilbert; citation_author=MA Colicos; citation_author=ER Kandel; citation_author=D Kuhl; citation_journal_title=Nature; citation_volume=361; citation_number=10; citation_pages=453-457; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Regulation of transcription factors by neuronal activity.; citation_author=AE West; citation_author=EC Griffith; citation_author=ME Greenberg; citation_journal_title=Nat Rev Neurosci; citation_volume=3; citation_number=11; citation_pages=921-931; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Coupling of mGluR/Homer and PSD-95 complexes by the Shank family of postsynaptic density proteins.; citation_author=JC Tu; citation_author=B Xiao; citation_author=S Naisbitt; citation_author=JP Yuan; citation_author=RS Petralia; citation_journal_title=Neuron; citation_volume=23; citation_number=12; citation_pages=583-592; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Shank, a novel family of postsynaptic density proteins that binds to the NMDA receptor/PSD-95/GKAP complex and cortactin.; citation_author=S Naisbitt; citation_author=E Kim; citation_author=JC Tu; citation_author=B Xiao; citation_author=C Sala; citation_journal_title=Neuron; citation_volume=23; citation_number=13; citation_pages=569-582; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Proteomic analysis of NMDA receptor-adhesion protein signaling complexes.; citation_author=H Husi; citation_author=MA Ward; citation_author=JS Choudhary; citation_author=WP Blackstock; citation_author=SG Grant; citation_journal_title=Nat Neurosci; citation_volume=3; citation_number=14; citation_pages=661-669; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Isolation of 2000-kDa complexes of N-methyl-D-aspartate receptor and postsynaptic density 95 from mouse brain.; citation_author=H Husi; citation_author=SG Grant; citation_journal_title=J Neurochem; citation_volume=77; citation_number=15; citation_pages=281-291; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular characterization and comparison of the components and multiprotein complexes in the postsynaptic proteome.; citation_author=MO Collins; citation_author=H Husi; citation_author=L Yu; citation_author=JM Brandon; citation_author=CN Anderson; citation_journal_title=J Neurochem; citation_number=16; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Organization of brain complexity–synapse proteome form and function.; citation_author=AJ Pocklington; citation_author=JD Armstrong; citation_author=SG Grant; citation_journal_title=Brief Funct Genomic Proteomic; citation_volume=5; citation_number=17; citation_pages=66-73; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Proteomics of the nervous system.; citation_author=H Husi; citation_author=SG Grant; citation_journal_title=Trends Neurosci; citation_volume=24; citation_number=18; citation_pages=259-266; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The discs-large tumor suppressor gene of Drosophila encodes a guanylate kinase homolog localized at septate junctions.; citation_author=DF Woods; citation_author=PJ Bryant; citation_journal_title=Cell; citation_volume=66; citation_number=19; citation_pages=451-464; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=SynGAP regulates ERK/MAPK signaling, synaptic plasticity, and learning in the complex with postsynaptic density 95 and NMDA receptor.; citation_author=NH Komiyama; citation_author=AM Watabe; citation_author=HJ Carlisle; citation_author=K Porter; citation_author=P Charlesworth; citation_journal_title=J Neurosci; citation_volume=22; citation_number=20; citation_pages=9721-9732; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Phosphatidylinositol 3-kinase regulates the induction of long-term potentiation through extracellular signal-related kinase-independent mechanisms.; citation_author=P Opazo; citation_author=AM Watabe; citation_author=SG Grant; citation_author=TJ O'Dell; citation_journal_title=J Neurosci; citation_volume=23; citation_number=21; citation_pages=3679-3688; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of PSD-95 as a regulator of dopamine-mediated synaptic and behavioral plasticity.; citation_author=WD Yao; citation_author=RR Gainetdinov; citation_author=MI Arbuckle; citation_author=TD Sotnikova; citation_author=M Cyr; citation_journal_title=Neuron; citation_volume=41; citation_number=22; citation_pages=625-638; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Separable features of visual cortical plasticity revealed by N-methyl-D-aspartate receptor 2A signaling.; citation_author=M Fagiolini; citation_author=H Katagiri; citation_author=H Miyamoto; citation_author=H Mori; citation_author=SG Grant; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=23; citation_pages=2854-2859; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Neuropathic sensitization of behavioral reflexes and spinal NMDA receptor/CaM kinase II interactions are disrupted in PSD-95 mutant mice.; citation_author=EM Garry; citation_author=A Moss; citation_author=A Delaney; citation_author=F O'Neill; citation_author=J Blakemore; citation_journal_title=Curr Biol; citation_volume=13; citation_number=24; citation_pages=321-328; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A potential animal model for Lesch-Nyhan syndrome through introduction of HPRT mutations into mice.; citation_author=MR Kuehn; citation_author=A Bradley; citation_author=EJ Robertson; citation_author=MJ Evans; citation_journal_title=Nature; citation_volume=326; citation_number=25; citation_pages=295-298; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=HPRT-deficient (Lesch-Nyhan) mouse embryos derived from germline colonization by cultured cells.; citation_author=M Hooper; citation_author=K Hardy; citation_author=A Handyside; citation_author=S Hunter; citation_author=M Monk; citation_journal_title=Nature; citation_volume=326; citation_number=26; citation_pages=292-295; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Double replacement gene targeting for the production of a series of mouse strains with different prion protein gene alterations.; citation_author=RC Moore; citation_author=NJ Redhead; citation_author=J Selfridge; citation_author=J Hope; citation_author=JC Manson; citation_journal_title=Biotechnology (N Y); citation_volume=13; citation_number=27; citation_pages=999-1004; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Efficient BLG-Cre mediated gene deletion in the mammary gland.; citation_author=S Selbert; citation_author=DJ Bentley; citation_author=DW Melton; citation_author=D Rannie; citation_author=P Lourenco; citation_journal_title=Transgenic Res; citation_volume=7; citation_number=28; citation_pages=387-396; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Inactivation of the mouse HPRT locus by a 203-bp retroposon insertion and a 55-kb gene-targeted deletion: establishment of new HPRT-deficient mouse embryonic stem cell lines.; citation_author=H Tsuda; citation_author=CE Maynard-Currie; citation_author=LH Reid; citation_author=T Yoshida; citation_author=K Edamura; citation_journal_title=Genomics; citation_volume=42; citation_number=29; citation_pages=413-421; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A simple genotyping assay for the Hprt null allele in mice produced from the HM-1 and E14TG2a mouse embryonic stem cell lines.; citation_author=C McEwan; citation_author=DW Melton; citation_journal_title=Transgenic Res; citation_volume=12; citation_number=30; citation_pages=519-520; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genomic organization of the mottled gene, the mouse homologue of the human Menkes disease gene.; citation_author=C Cecchi; citation_author=P Avner; citation_journal_title=Genomics; citation_volume=37; citation_number=31; citation_pages=96-104; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Impaired male fertility and atrophy of semineforous tubules caused by haploinsufficiency for Foxa3.; citation_author=R Behr; citation_author=S Sackett; citation_author=I Bochkis; citation_author=L Phillip; citation_author=K Kaestner; citation_journal_title=Dev Biol In press; citation_number=32; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Nebulin regulates thin filament length, contractility, and Z-disk structure in vivo.; citation_author=CC Witt; citation_author=C Burkart; citation_author=D Labeit; citation_author=M McNabb; citation_author=Y Wu; citation_journal_title=Embo J; citation_volume=25; citation_number=33; citation_pages=3843-3855; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A new progeroid syndrome reveals that genotoxic stress suppresses the somatotroph axis.; citation_author=LJ Niedernhofer; citation_author=GA Garinis; citation_author=A Raams; citation_author=AS Lalai; citation_author=AR Robinson; citation_journal_title=Nature; citation_volume=444; citation_number=34; citation_pages=1038-1043; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Perturbations of vascular homeostasis and aortic valve abnormalities in fibulin-4 deficient mice.; citation_author=K Hanada; citation_author=M Vermeij; citation_author=GA Garinis; citation_author=MC de Waard; citation_author=MG Kunen; citation_journal_title=Circ Res; citation_volume=100; citation_number=35; citation_pages=738-746; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Biochemical, molecular and behavioral phenotypes of Rab3A mutations in the mouse.; citation_author=S Yang; citation_author=M Farias; citation_author=D Kapfhamer; citation_author=J Tobias; citation_author=G Grant; citation_journal_title=Genes Brain Behav; citation_volume=6; citation_number=36; citation_pages=77-96; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Interpretation of knockout experiments: the congenic footprint.; citation_author=LC Schalkwyk; citation_author=C Fernandes; citation_author=MW Nash; citation_author=K Kurrikoff; citation_author=E Vasar; citation_journal_title=Genes Brain Behav; citation_volume=6; citation_number=37; citation_pages=299-303; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Genetic variation among 129 substrains and its importance for targeted mutagenesis in mice.; citation_author=EM Simpson; citation_author=CC Linder; citation_author=EE Sargent; citation_author=MT Davisson; citation_author=LE Mobraaten; citation_journal_title=Nat Genet; citation_volume=16; citation_number=38; citation_pages=19-27; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genealogy of the 129 inbred strains: 129/SvJ is a contaminated inbred strain.; citation_author=DW Threadgill; citation_author=D Yee; citation_author=A Matin; citation_author=JH Nadeau; citation_author=T Magnuson; citation_journal_title=Mamm Genome; citation_volume=8; citation_number=39; citation_pages=390-393; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Knockout mice: simple solutions to the problems of genetic background and flanking genes.; citation_author=DP Wolfer; citation_author=WE Crusio; citation_author=HP Lipp; citation_journal_title=Trends Neurosci; citation_volume=25; citation_number=40; citation_pages=336-340; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Model-based analysis of oligonucleotide arrays: model validation, design issues and standard error application.; citation_author=C Li; citation_author=W Hung Wong; citation_journal_title=Genome Biol; citation_volume=2; citation_number=41; citation_pages=RESEARCH0032; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice"/>
    <meta name="twitter:description" content="BackgroundGene expression profiling using microarrays is a powerful technology widely used to study regulatory networks. Profiling of mRNA levels in mutant organisms has the potential to identify genes regulated by the mutated protein.Methodology/Principle FindingsUsing tissues from multiple lines of knockout mice we have examined genome-wide changes in gene expression. We report that a significant proportion of changed genes were found near the targeted gene.Conclusions/SignificanceThe apparent clustering of these genes was explained by the presence of flanking DNA from the parental ES cell. We provide recommendations for the analysis and reporting of microarray data from knockout mice"/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001303.g004"/>

  <meta property="og:title" content="Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4337'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6099'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8911&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001303">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001303" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001303&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Luis M. Valor, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Seth G. N. Grant
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:sg3@sanger.ac.uk">sg3@sanger.ac.uk</a></p>

                <p>Affiliation:
                  Genes to Cognition Programme, The Wellcome Trust Sanger Institute, Cambridge, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 12, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001303</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001303-g001" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-t001" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-g002" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-t002" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-t003" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-t004" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-g003" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001303-g004" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001303">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001303" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Gene expression profiling using microarrays is a powerful technology widely used to study regulatory networks. Profiling of mRNA levels in mutant organisms has the potential to identify genes regulated by the mutated protein.</p>


<h3>Methodology/Principle Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Using tissues from multiple lines of knockout mice we have examined genome-wide changes in gene expression. We report that a significant proportion of changed genes were found near the targeted gene.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>The apparent clustering of these genes was explained by the presence of flanking DNA from the parental ES cell. We provide recommendations for the analysis and reporting of microarray data from knockout mice</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Valor LM, Grant SGN (2007) Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice. PLoS ONE 2(12):
          e1303.
            doi:10.1371/journal.pone.0001303</p><p><strong>Academic Editor: </strong>Abraham Sonenshein, Tufts University, United States of America</p><p><strong>Received:</strong> July 5, 2007; <strong>Accepted:</strong> November 15, 2007; <strong>Published:</strong> December 12, 2007</p><p><strong>Copyright:</strong> © 2007 Valor, Grant. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by the Wellcome Trust. They supported the research and played no specific role in the details of the research.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Microarray technology has been widely assimilated by gene expression researchers and its success is largely due to its power of assaying thousands of transcripts in a genome wide manner. One of the most popular experimental designs has been to determine the gene expression changes that occur in knockout mice where it is correlated with phenotypic changes. The usual objective is to obtain insights into the role of the mutated protein by identifying particular genes under its control, and retrieve candidate genes that may explain the altered phenotype in the mutant animal.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The creation of knockout mice involves the modification of the genome of mouse embryonic stem (ES) cells through a variety of processes including homologous recombination or other mutagenic procedures <a href="#pone.0001303-Tymms1">[1]</a>. The majority of gene targeting experiments have been performed in ES cells derived from the 129 strain of mice. Following germ-line transmission of the mutant allele, the founder mice are often backcrossed onto other strains, creating hybrid animals. The relative contribution of the genetic background of the ES cell and the strain to which it was bred has been widely studied <a href="#pone.0001303-Crusio1">[2]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>We have studied the effects of knockout (KO) mutations on the mechanisms of learning and memory <a href="#pone.0001303-Cuthbert1">[3]</a>, <a href="#pone.0001303-Migaud1">[4]</a>. Patterns of neural activity induced by learning or artificial stimuli lead to activation of signal transduction pathways, which ultimately lead to the cellular changes underlying learning and memory. The two major cellular changes involve strength of synaptic transmission (synaptic plasticity) and gene expression (activity-dependent gene expression). It is clear from studies in the sensory-motor synapses of <em>Aplysia californica</em> and the hippocampus of mice and rats that neurotransmitter receptors initiate the signalling events that lead to both synaptic plasticity and activity-dependent gene expression <a href="#pone.0001303-Nguyen1">[5]</a>, <a href="#pone.0001303-Linden1">[6]</a>, <a href="#pone.0001303-Milner1">[7]</a>, <a href="#pone.0001303-Alberini1">[8]</a>, <a href="#pone.0001303-Bading1">[9]</a>, <a href="#pone.0001303-Qian1">[10]</a>, <a href="#pone.0001303-West1">[11]</a>. L-glutamate is the dominant excitatory neurotransmitter in the central nervous system of vertebrates and glutamate receptors are of special relevance to synaptic plasticity. The ionotropic <em>N</em>-methyl-D-aspartate (NMDA) and metabotropic (mGLuR) subtypes of glutamate receptor are commonly involved in experimental plasticity-driven transcriptional changes. Both the NMDA and mGluR receptors are connected by a series of adaptor proteins <a href="#pone.0001303-Tu1">[12]</a>, <a href="#pone.0001303-Naisbitt1">[13]</a> and can be physically isolated from mouse brain in ~2000-kDa multiprotein complexes called NRC or MASC (NRC, NMDA Receptor Complex; MASC, MAGUK Associated Signaling Complex) <a href="#pone.0001303-Husi1">[14]</a>, <a href="#pone.0001303-Husi2">[15]</a>, <a href="#pone.0001303-Collins1">[16]</a>, <a href="#pone.0001303-Pocklington1">[17]</a>. NRCs are organized into receptor, adaptor, signalling, cytoskeletal and novel proteins, and includes cell adhesion molecules, kinases, phosphatases, GTPase-activating proteins and Ras and MAPK pathway components. NRC/MASC form a subset of the post-synaptic density which is thought to be comprised of a set of complexes. Such a complex offers an attractive model for the organisation and coordination signalling networks <a href="#pone.0001303-Husi1">[14]</a>, <a href="#pone.0001303-Pocklington1">[17]</a>, <a href="#pone.0001303-Husi3">[18]</a>. It was hypothesised that the function of the NRC is to orchestrate the signalling pathways downstream of receptor activation and coordinate both synaptic plasticity and activity-dependent gene expression amongst other changes.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We were specifically interested in the previously characterized KO mice for PSD-95 and SAP102 <a href="#pone.0001303-Cuthbert1">[3]</a>, <a href="#pone.0001303-Migaud1">[4]</a>. Both proteins are from the MAGUK (<span class="underline">M</span>embrane <span class="underline">A</span>ssociated <span class="underline">GU</span>anylate <span class="underline">K</span>inase) family, which members share a common structure of three PDZ (PSD-95/Dlg/ZO-1), an SH3 (Src-homology-3) and guanylate kinase (GK) domains and together enable protein-protein interactions. The postsynaptic density 95 (PSD-95) protein is a homolog of the <em>Drosophila</em> suppressor gene, <em>dlg</em> <a href="#pone.0001303-Woods1">[19]</a> and acts as a scaffold to organize postsynaptic proteins into signalling complexes <a href="#pone.0001303-Migaud1">[4]</a> or anchoring complexes for glutamate receptors and cytoskeletal proteins, as reviewed elsewhere <a href="#pone.0001303-Tu1">[12]</a>. Mice lacking PSD-95 or SAP102 were found to have enhanced forms of NMDA receptor-dependent long-term potentiation <a href="#pone.0001303-Cuthbert1">[3]</a>, <a href="#pone.0001303-Migaud1">[4]</a>, <a href="#pone.0001303-Komiyama1">[20]</a>, <a href="#pone.0001303-Opazo1">[21]</a> and exhibited impairments in spatial learning (and other forms of behavioural plasticity) <a href="#pone.0001303-Yao1">[22]</a>, <a href="#pone.0001303-Fagiolini1">[23]</a>, <a href="#pone.0001303-Garry1">[24]</a>. Our original aim was to identify those transcripts that were controlled by PSD-95 and SAP102 and therefore try to track their dependent signalling pathways. Surprisingly, and because of the unbiased nature of microarray analyses, we found that the most significantly changing genes were clustered in the neighbourhood of the KO gene. We observed that this phenomenon was frequent in KO studies and, therefore, special care should be taken in the interpretation of this kind of experiments.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>To determine the gene expression changes in the hippocampus of mice lacking SAP102 or PSD-95, total RNA was isolated from the hippocampus of SAP102<sup>−/Y</sup> and PSD-95<sup>−/−</sup> mutant mice and littermate wild-types. RNA was hybridized to MG-430 2.0 arrays (Affymetrix) that contained 45,037 probe sets excluding controls. The data was normalized and filtered (see <a href="#s4">Materials and Methods</a>). We first examined the expression data from SAP102 mutants and found that 65% (13/20) of the significantly changed probe sets were encoded by chromosome X, which also encodes SAP102. This unexpected finding suggests that either SAP102 preferentially regulates genes on the X-chromosome or that there is some <em>cis</em>-acting effect of the mutation.</p>
<a id="article1.body1.sec2.p2" name="article1.body1.sec2.p2"></a><p>We focused on the SAP102 regulated genes on the X-chromosome and found they were clustered around the SAP102/<em>dlgh3</em> gene (<a href="#pone-0001303-g001">Figure 1</a>). These genes spanned a region of ~98 Mb (<a href="#pone-0001303-g001">Figure 1A</a>, <a href="#pone-0001303-t001">Table 1</a>). Within this cluster was the <em>hprt</em> (hypoxanthine guanine phosphoribosyl transferase) gene, a selection marker extensively used in ES cells to generate transgenic mice <a href="#pone.0001303-Kuehn1">[25]</a>, <a href="#pone.0001303-Hooper1">[26]</a>, <a href="#pone.0001303-Moore1">[27]</a>, <a href="#pone.0001303-Selbert1">[28]</a>. In our case, the ES cell clone used to generate our knockout mice was E14TG2a HM-1, which harbours an <em>hprt</em> allele containing a 55 kb-deletion in its 5′-flanking region <a href="#pone.0001303-Tsuda1">[29]</a>. Using a PCR protocol specifically designed to genotype this deletion <a href="#pone.0001303-McEwan1">[30]</a>, we found it was correlated with the reduction of HPRT expression (data not shown). The <em>hprt</em> and SAP102/<em>dlgh3</em> loci are separated by ~48 Mb and since the <em>hprt</em> locus of 129S5 mice does not contain the deletion, these data suggested that the observed changes in <em>hprt</em> gene expression in SAP102 mutant mice was the result of a linked mutation originated in the ES cells. Following gene targeting of SAP102/<em>dlgh3</em> gene in E14TG2a ES cells (129P2 background), the chimera was crossed to 129S5 and subsequently backcrossed. This provided an important clue that genetic background of the ES cells might be contributing to the gene expression profiling.</p>
<div class="figure" id="pone-0001303-g001"><div class="img"><a name="pone-0001303-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Clustering of changed genes around <em>SAP102/dlgh3</em> and <em>PSD-95/dlgh4</em> loci. <em>A</em>, significant genes (x-axis) from SAP102<sup>−/Y</sup> hippocampus along chromosome X (y-axis), indicating the location of SAP102 gene (<em>dlgh3</em>) and <em>hprt.</em></span></strong></p><a id="article1.body1.sec2.fig1.caption1.p1" name="article1.body1.sec2.fig1.caption1.p1"></a><p>White histogram, results from 129S5 strain animals; black histogram, results from C57BL/6j after selection of <em>hprt</em> deletion (see text). <em>B</em>, probe sets from PSD-95<sup>−/−</sup> along chromosome 11, where the position of the mutated gene (<em>dlgh4</em>) is indicated. White histogram, hippocampus mRNA; black histogram; cortex mRNA.</p>
<span>doi:10.1371/journal.pone.0001303.g001</span></div><div class="figure" id="pone-0001303-t001"><div class="img"><a name="pone-0001303-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>List of significant gene expression probe sets from X chromosome in SAP102<sup>−/Y</sup> samples.</span></strong></p><span>doi:10.1371/journal.pone.0001303.t001</span></div><a id="article1.body1.sec2.p3" name="article1.body1.sec2.p3"></a><p>To further explore this hypothesis, we examined the degree of linkage between both loci. Using the MGI database (<a href="http://www.informatics.jax.org">www.informatics.jax.org</a>) we estimated the percentage of independent segregation of hprt and <em>SAP102/dlg3</em> to be 27% (27 cM). We tested this using a cross between <em>SAP102/dlg3</em> heterozygous females and 129S5 (wild type) males, which produced 37 litters, and subsequent genotyping. Only 27% of the heterozygous females had two wild-type alleles of <em>hprt</em> and 29% of E17.5 male embryos showed segregation of <em>hprt</em> and <em>SAP102/dlg3</em> mutations. These data are in keeping with the conclusion that the entire portion between both loci originated in the original E14TG2a ES cells.</p>
<a id="article1.body1.sec2.p4" name="article1.body1.sec2.p4"></a><p>These observations suggest that further backcrossing and marker selection would result in a reduction of the E14TG2a DNA. To explore this possibility we analyzed microarray data from SAP102<sup>−/Y</sup> hippocampi that contained the wild-type allele of <em>hprt</em> mutation. In this experiment we used SAP102<sup>−/Y</sup> mutant mice that had been backcrossed 3 generations onto the C57BL/6j background. In this case, only six probe sets were changed from the X chromosome (~29% of the significant probe sets; 6/21) and spanned a shorter region of ~48 Mb (<a href="#pone-0001303-g001">Figure 1A</a>, <a href="#pone-0001303-t001">Table 1</a>). This clustering was significantly different from a random distribution along the X chromosome (<em>P</em>&lt;0.05, χ<sup>2</sup> = 26.6 d.f. = 16). Four of the five probe sets were common to the previous experiment (<a href="#pone-0001303-t001">Table 1</a>) and the change was similar in direction and magnitude, thus reproducing the effect of the ES cell-derived sequences and suggesting changes were independent of the host backcrossing genetic background. Since we used C57BL/6j animals for the new analysis, we could take advantage of the availability of primers that distinguished C57- and 129-derived strains genotypes. We focused on <em>Atp7a</em>, which expression was changed in the microarray experiment and is found ~5 Mb from <em>SAP102/dlg3</em> and tested if it was of 129 or C57BL/6 origin. Using a published genotyping protocol (in which the amplicon covers exon 8 of Atp7a <a href="#pone.0001303-Cecchi1">[31]</a>), we found that the all samples that showed changes in Atp7a expression were of 129 origin (<a href="#pone-0001303-g002">Figure 2</a>). These data further support the conclusion that the changes in gene expression in the mutant mice involving the chromosome X can be explained by genetic background effects.</p>
<div class="figure" id="pone-0001303-g002"><div class="img"><a name="pone-0001303-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>PCR genotyping results for Atp7a and Dlgh3 genomic regions.</span></strong></p><a id="article1.body1.sec2.fig2.caption1.p1" name="article1.body1.sec2.fig2.caption1.p1"></a><p>Two bands were observed for Atp7a reactions, one of 694 bp corresponding to 129-derived sequences and another of 637 bp corresponding to C57Bl6-derived sequences (upper panel). The appearance of the 694 bp-band coincided with the presence of the SAP102 mutation (-/y) (lower panel). Wild-type 129S5 (129) and C57BL/6j (BL6) were used as positive controls.</p>
<span>doi:10.1371/journal.pone.0001303.g002</span></div><a id="article1.body1.sec2.p5" name="article1.body1.sec2.p5"></a><p>Similar effects were observed for other mutations involving autosomes. We took advantage of the results obtained in another knockout for a MAGUK protein, PSD-95, which is encoded on chromosome 11. Using an identical microarray platform to that used for SAP102 mutants, we found that ~47% (8/17) of the significantly changed probes in hippocampus were encoded on chromosome 11. As shown in <a href="#pone-0001303-g001">Figure 1B</a> there was significant clustering of probe sets around the <em>dlgh4</em> locus and spanned a region of ~5 Mb (<em>P</em>&lt;0.001, χ<sup>2</sup> = 33.7 d.f. = 11). This shorter length of the ES cell-derived region in PSD-95 mutants may be due to the larger number of backcrosses (&gt;10 in PSD-95 onto a C57BL/6 background) compared to the SAP102 mutants (~48 Mb after 3 backcrossed on C57BL/6). These data are consistent with the view that the number of backcrosses helps to shorten the length of ES cell-derived sequences as well as marker selection (<em>hprt</em> in the SAP102 mutants) and should be considered in microarray profiling experiments in knock-out animals. The above data is based on studies of hippocampus transcript levels and we asked if similar results would be observed in other regions of the nervous system. Using cortex RNA from PSD-95 mutants we found that 4 of the 6 probe sets that were significant in the hippocampus were also significant in the cortex of the same animals (<a href="#pone-0001303-t002">Table 2</a>), thus reproducing the effects of ES-cell derived sequences in gene expression. For further details on the statistical analysis of both SAP102 and PSD-95 mutants, see <a href="#pone-0001303-t003">Tables 3</a> and <a href="#pone-0001303-t004">4</a>.</p>
<div class="figure" id="pone-0001303-t002"><div class="img"><a name="pone-0001303-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>List of significant gene expression probe sets on chromosome 11 from PSD-95<sup>−/−</sup> mutants.</span></strong></p><span>doi:10.1371/journal.pone.0001303.t002</span></div><div class="figure" id="pone-0001303-t003"><div class="img"><a name="pone-0001303-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Distribution of significant genes within the chromosome of the ablated gene.</span></strong></p><span>doi:10.1371/journal.pone.0001303.t003</span></div><div class="figure" id="pone-0001303-t004"><div class="img"><a name="pone-0001303-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Distribution of significant genes between all the chromosomes.</span></strong></p><span>doi:10.1371/journal.pone.0001303.t004</span></div><a id="article1.body1.sec2.p6" name="article1.body1.sec2.p6"></a><p>We also validated the microarray experiments performed in SAP102 and PSD-95 mutants by qRT-PCR. As shown in <a href="#pone-0001303-g003">Figure 3</a> the changes in gene expression detected with microarrays were confirmed using qRT-PCR for SAP102, HPRT, Fundc2, and BB315069 (3′ to Fundc2) in the SAP102 mutants and PSD-95, Fbxo39 and BG092359 (3′ to Fbxo39) in the PSD-95 mutants.</p>
<div class="figure" id="pone-0001303-g003"><div class="img"><a name="pone-0001303-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Validation of microarray analyses by qRT-PCR assays in PSD-95 and SAP102 mutants.</span></strong></p><a id="article1.body1.sec2.fig3.caption1.p1" name="article1.body1.sec2.fig3.caption1.p1"></a><p>Hippocampus mRNA was extracted and level of mRNA detected using microarrays or qRT-PCR. <em>A</em>, SAP102 mutant mice; results for SAP102, Hprt, Fundc2, BB315069 (3′ to Fundc2) are shown. B. PSD-95 mutant mice; results for PSD-95, Fbxo39 and BG092359 (3′ to Fbxo39). y-axis: fold change of mutant mRNA levels compared to control; mean±s.e.m. P, qRT-PCR data; M, microarray data. M1, M2… refers to different probe sets for the same gene. Note different scale of y-axes.</p>
<span>doi:10.1371/journal.pone.0001303.g003</span></div><a id="article1.body1.sec2.p7" name="article1.body1.sec2.p7"></a><p>From the above experiments on brain mRNA from SAP102 and PSD-95 mutants it seems likely that these effects of genes linked to the mutant allele might be generally important in other knockout lines and other tissues. To address this issue we analyzed recent data deposited in ArrayExpress (<a href="http://www.ebi.ac.uk/arrayexpress/">www.ebi.ac.uk/arrayexpress/</a>) that employed the same format of arrays as used in our study. The purpose was not to exhaustively compile the existent data but to provide further examples. Data from four mutant genes were analyzed: Foxa3 <a href="#pone.0001303-Behr1">[32]</a>, Nebulin (<em>neb</em>) <a href="#pone.0001303-Witt1">[33]</a>, Ercc1 <a href="#pone.0001303-Niedernhofer1">[34]</a> and Fibulin-4 (<em>efemp2</em>) <a href="#pone.0001303-Hanada1">[35]</a> (<a href="#pone-0001303-g004">Figure 4</a>), using RNA derived from testes, skeletal muscle, liver and aorta respectively.</p>
<div class="figure" id="pone-0001303-g004"><div class="img"><a name="pone-0001303-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001303" data-uri="info:doi/10.1371/journal.pone.0001303.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001303.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001303.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001303.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001303.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Number of significantly changed genes on targeted chromosome for Foxa3, Neb, Ercc1 and Fibulin-4 mutants.</span></strong></p><a id="article1.body1.sec2.fig4.caption1.p1" name="article1.body1.sec2.fig4.caption1.p1"></a><p>y-axis: number of changed genes, x-axis: chromosomal position. The arrow indicates the location of the mutated genes in each case.</p>
<span>doi:10.1371/journal.pone.0001303.g004</span></div><a id="article1.body1.sec2.p8" name="article1.body1.sec2.p8"></a><p>In all four knockouts we found changes in gene expression in genes near to the targeted locus; however the potential boundary between ES cell and host DNA was not clear in all mutants (<a href="#pone-0001303-t003">Table 3</a>, <a href="#pone-0001303-g004">Figure 4</a>). The number of changed genes was plotted for the length of the relevant chromosome and the position of the targeted gene indicated. This allows us to examine clustering ‘within chromosomes’ and to compare with random distributions (see <a href="#s4">Materials and Methods</a>). For chromosome 7 and the <em>foxa3</em> locus, of the 8 genes that changed on this chromosome (including the mutated gene), all were clustered within 28 Mb of the mutation (<em>P</em>&lt;0.01, χ<sup>2</sup> = 31.1 d.f. = 14), despite 11 backcrosses onto C57BL/6 strain <a href="#pone.0001303-Behr1">[32]</a>. For <em>ercc1</em> and <em>neb</em> mutants, the changed genes were distributed more widely on chromosome 7 and 2 respectively. Although there was a trend toward clustering with the mutant locus, this distribution was not statistically significant (<a href="#pone-0001303-t003">Table 3</a>): for <em>ercc1</em> there were 72.7% of significant genes within ~60 Mb (~42% of the chromosome); for <em>neb</em> there were 53.3% within ~75 Mb (~42% of the chromosome). The inability to identify more conclusive evidence of linkage in <em>ercc1</em> or <em>neb</em> mutants was probably due to the large number of changed genes: compared to SAP102, PSD-95 and Foxa3 where 21, 12 and 10 genes respectively were changed, 140 genes were changed in <em>ercc1</em> mutants and 92 in <em>neb</em> mutants (<a href="#pone-0001303-t004">Table 4</a>). In the case of the <em>efemp2</em> locus on chromosome 19, there were 4 genes changed; however clustering was not statistically significant (<a href="#pone-0001303-t003">Table 3</a>) perhaps because chromosome 19 is too short and only a small portion (~8% of the chromosome, which was absent of significant genes) was estimated to segregate independently and thus it is possible that the entire chromosome was derived from ES-cell background. Thus using the ‘within chromosome’ clustering does not reveal statistically significant results for all knockout mice and appears to be influenced by chromosome size and number of significantly changed genes. To overcome these limitations we used an alternative approach where we compared the frequency of changed genes and their contribution to deviation from random distributions on the targeted chromosome with other chromosomes (see <a href="#s4">Materials and Methods</a>). Using this ‘between chromosome’ method we found that of 7 mutant examined, 6 showed evidence of clustering of changed genes to the targeted chromosome (<a href="#pone-0001303-t004">Table 4</a>).</p>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Microarrays are a powerful method for detecting changes in gene expression and here we examine data from a range of different knockout mice. When comparing RNA from mutant mice and wild type controls, we observed a higher than expected frequency of changed genes near to the targeted/knocked-out gene. These genes were either clustered around the targeted allele or over-represented on that chromosome. Recent reports found evidence of linked changes in two knockouts <a href="#pone.0001303-Yang1">[36]</a>, <a href="#pone.0001303-Schalkwyk1">[37]</a> (one of them published during the preparation of this manuscript). In our studies of PSD-95 and SAP102, the gene was disrupted in 129P2 ES cells and the RNA extracted from mice on a hybrid background (since a distinct mouse strain was used during breeding of the mutants). We consider the most likely explanation for the changes in gene expression of the genes clustered around the knockout locus to be that these genes were of ES cell origin (flanking DNA). In this model, the flanking DNA (from ES cells) contains polymorphisms that alter gene expression relative to the genetic background of the backcrossed mice. We found changes in gene expression in SAP102 mice (from 129P2 ES cells) backcrossed onto 129S5 and C57BL/6 strains. Although 129P2 and 129S5 might be closely related genetic backgrounds, a large variability has been reported between 129 substrains <a href="#pone.0001303-Simpson1">[38]</a>, <a href="#pone.0001303-Threadgill1">[39]</a>. The notion that flanking DNA can contribute to complications in mouse behavioural studies has been recognised and breeding strategies recommended to minimise its effects <a href="#pone.0001303-Wolfer1">[40]</a>. We have reported here that although backcrosses shorten the length of ES cell flanking regions more than ten backcrosses was insufficient to completely eliminate them (see PSD-95 and Foxa3 examples), as recognized in <a href="#pone.0001303-Wolfer1">[40]</a>. Marker selection (as the example provided by <em>hprt</em>) can accelerate the identification of mice with shorter flanking DNA.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>An alternative model, which we cannot exclude, is that the changes in gene expression around the knockout locus were because of the modifications introduced during gene targeting (e.g. neomycin gene) that produce a <em>cis</em>-acting effect on neighbouring genes. This could arise either by disrupting a functional region in the host genome (i.e., a regulatory element) or interaction of the foreign sequence with the flanking DNA. The number of changing genes in the microarray experiments in the SAP102 and PSD-95 samples was inversely correlated with the number of backcrosses and active marker selection, suggesting the <em>cis</em>-acting effect, if it exists, only partially contributes. To explore a <em>cis</em> effect in more detail, a future approach could consist of analyzing microarray data from knock-in animals, where the insertion is made into a region lacking functional elements (i.e., coding region or a regulatory element).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Microarray analysis in knock-out animals should be carefully examined to avoid misinterpretation in the gene expression profile. Rather than assign the observed changes to the mutation, the genotype of the changed genes needs to be tested. This current study highlights the effects of flanking DNA of ES cell origin, which is selected during breeding of the mutant mice. It may therefore be necessary to genotype all changed genes before studying mutation-dependent effects. Although this may not be practical in all situations, the simplest approach would be to treat those changed genes on the targeted chromosome with extreme caution and only consider changes on other chromosomes or in those portions of the targeted chromosome estimated to segregate independently. Future array based methods could include mRNA profiling and a separate array for profiling the genotype of the corresponding probe sets and their surrounding regions. We would also recommend that when gene expression data is reported on mutant mice, that the chromosome location of the genes encoding those mRNAs is clearly reported. Studies on mutant mice on the inbred background of the ES cells would obviate the need to perform genotyping.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Animals</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>For microarray studies, the ages of mice used were as follows: 6 weeks old (129S5 strain); 12–13 weeks old (C57BL/6j strain). Number of mice: five SAP102<sup>−/Y</sup> and five littermate wild-types (129S5 strain); four SAP102<sup>−/Y</sup> and three littermate wild-types (C57BL/6j strain); five PSD-95<sup>−/−</sup> and four littermate wild-types (hippocampus, C57BL/6j strain); two PSD-95<sup>−/−</sup> and two littermate wild-types (cortex, C57BL/6j strain). Animals were dispatched by cervical dislocation. All dissections were performed consecutively on the same day at 4°C and tissue kept moist with chilled PBS (Gibco). Collected tissue (hippocampus and cortex) was immediately frozen in liquid nitrogen or immersed into RNAlater reagent (Qiagen) on ice for RNA isolation. Animals were treated in accordance with the UK Animal Scientific Procedures Act (1986).</p>


<h4>Nucleic acids isolation</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>RNeasy Mini and Midi columns (Qiagen) were used to purify the RNA from hippocampus and cortex, respectively. On-column DNase treatment was carried out to remove traces of DNA. During all the purifications, manufacturer's instructions were followed. Samples were concentrated with the addition of 1/10 vol of 3 M NaOAc pH 6 and 3 vol of 100% EtOH and precipitated at −20°C. Once resuspended in 12 µl of RNase-free H<sub>2</sub>O, an aliquot was used to check the RNA integrity in 1% formaldehyde-agarose gels.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>To isolate genomic DNA, mouse tails were cut into 4–5 pieces and treated with 500 µl of the Lysis Buffer (10 mM Tris-HCl pH = 7.5, 1 mM EDTA, 1% SDS, 10 mM NaCl) and 2.5 µl Proteinase K (20 mg/ml) during 37 °C O/N. Then, 500 µl buffered phenol was added and the mix was incubated 1–2 h with gentle rotation at 4 °C. Once taking the supernatant after a low-speed spin (5,000 rpm 5 min), 500 µl of isoamylic-chloroform-phenol was used and spin at 13,000 rpm for 5 min. DNA was immediately precipitated by adding 1 ml isopropyl alcohol. After spinning at 13,000 rpm for 5 min, the pellet was washed with 500 µl 70% ethanol and resuspend in 100 µl H<sub>2</sub>O.</p>


<h4>Microarray hybridization, data acquisition and statistical analyses</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>For the transcriptome profiling, MG-430 2.0 arrays (Affymetrix) were used, that contain 45,101 probe sets including controls, one array per sample. Briefly, 3 µg of total RNA was reverse transcribed, labelled and hybridized using One-cycle Labelling Kit instructions (Affymetrix). Fluidics Station 450 and GCS3000, both from Affymetrix, were used for the washing and scanning steps respectively.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Signal intensities values and detection flags were extracted using GCOS (Affymetrix), using a Target value of 500 in the scaling step. Data were normalized by the median per gene and per array using GeneSpring v7 (Silicon Genetics): i) ANOVA test (p&lt;0.05) without assuming equal variances, ii) signal intensity must be a Present or Marginal call in all wild-type samples when down-regulated and knock-out samples when up-regulated, and iii) fold change &gt;1.8 that was selected arbitrarily for the purposes of the present work. In parallel data was processed using dChip (PM/MM difference model) <a href="#pone.0001303-Li1">[41]</a>. Only those probe sets appearing in both analyses were considered.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>The significant genes obtained in the microarray analyses were grouped in bins of 10 Mb to simplify both visualization in the figures and χ<sup>2</sup> tests. Centromeres were not considered. We compared the distribution of the significant genes along the targeted chromosome to a random distribution. This random distribution was calculated by dividing the number of significant genes by the number of bins. When testing the distribution of significant genes between all the chromosomes, three types of random distribution were analyzed: equal proportion of significant genes in all the chromosomes, proportion normalized by the length of each chromosome and proportion normalized by the number of genes (based upon Ensembl release 45). The microarray data is deposited in the ArrayExpress database, <a href="http://www.ebi.ac.uk/arrayexpress">www.ebi.ac.uk/arrayexpress</a>, accession number E-MEXP-1159.</p>


<h4>Retrotranscription and PCR</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Typically, 1 µg of total RNA was used for the retrotranscription, following the recommendations of Invitrogen (SuperScript II Reverse Transcriptase). PCR reactions were set up per sample using 1 µl cDNA, 0.3 µM each primer, 25 µl 2× Quantitect SYBR Green PCR Master Mix (Qiagen) to a final volume of 50 µl. The sequences of the primers were 5′-GTGGATTCTTTCTCCTTCAGC-3′ and 5′-ATACAACCTCCTCAGCTTTGC-3′ (Fundc2), 5′-TTCTGGTTTGTTTTGTTTTGC-3′ and 5′-TTCATGCCTTTTGTAAGCAAG-3′ (BB315069), 5′-CTCTCCACTTCATGCTCCAC-3′ and 5′-ATGGACATCCAGATTGCTAAG-3′ (Fbxo39), 5′-AATCGTCATAGGGAGGTTCAC-3′ and 5′-AGACCTTGCTATAGGCTCCAG-3′ (BG092359), 5′-ACGAGAGTGGTCAAGGTTAAAG-3′ and 5′-GGAGAGAAGATCATCGTTGG-3′ (PSD-95), 5′-AAAGGACCTCTCGAAGTGTTG-3′ and 5′-ATTTGGCTTTTCCAGTTTCAC-3′ (Hprt), 5′-GTGCTACGGGTGAATGAG-3′ and 5′-TGGTCTATCTGAAGGTGGCC-3′ (Sap102). Reactions were run out in the 7500 Real Time PCR System (Applied Biosystems) with the following conditions: 1 cycle of 95°C for 15 min, 35–40 cycles of 94°C for 30 s, 55°C for 30 s, 72°C for 50 s, 1 cycle of 72°C for 5 min. The resulting amplicons were checked for specificity and size in agarose gels. The data generated was analysed in 7500 System SDS v1.2.2 software, that calculated the cycle threshold (C<sub>T</sub>) and the fold change ( = 2<sup>−ΔΔCT</sup>). GAPDH was used as endogenous controls. To analyze statistically the quantitative PCR results, Student's t-test was used.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>For genotyping, each PCR was performed using 1 µl genomic DNA, 1 U Platinum Taq DNA polymerase (Invitrogen), 5 µl PCR buffer 10× (Invitrogen), 10 mM each dNTP, 50 mM MgCl<sub>2</sub> and 0.5 µM each primer in a final volume of 50 µl. For <em>hprt</em> alleles genotyping, the protocol described in <a href="#pone.0001303-McEwan1">[30]</a> was used with few modifications. Reactions to detect wild-type and mutant alleles were set up in parallel for the same genomic DNA: 1 cycle of 94°C 5 min, 35 cycles of 94°C 30 s, 62°C 30 s for the wild-type allele or 67°C for the null allele, 72°C 60 s and 1 cycle of 72°C 5 min. Addition of 1% DMSO was required for the null allele genotyping. For <em>atp7a</em> allele genotyping, the protocol described in <a href="#pone.0001303-Cecchi1">[31]</a> was used with few modifications, in which the conditions were 1 cycle of 94°C 5 min, 35 cycles of 94°C 30 s, 55°C 30 s, 72°C 60 s and 1 cycle of 72°C 5 min.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Jane Robinson for the maintenance of the animals; Nurudeen Afinowi, Marcelo Coba and Peter Cuthbert for their technical assistance and discussion; Lawrence Humphreys and Paul Charlesworth for providing the material for genotyping; Noboru Komiyama and Douglas Strathdee for reviewing the manuscript; Jane Turner for editorial assistance.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SG LV. Performed the experiments: LV. Analyzed the data: SG LV. Wrote the paper: SG LV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001303-Tymms1" id="pone.0001303-Tymms1"></a>Tymms M, Kola I (1999) Gene knockout protocols (Methods in Molecular Biology). Humana Press, Inc.   <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0001303-Crusio1" id="pone.0001303-Crusio1"></a>Crusio WE (2004) Flanking gene and genetic background problems in genetically manipulated mice. Biol Psychiatry  56: 381–385.  <ul class="find" data-citedArticleID="1029146" data-doi="10.1016/j.biopsych.2003.12.026"><li><a href="http://dx.doi.org/10.1016/j.biopsych.2003.12.026" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flanking+gene+and+genetic+background+problems+in+genetically+manipulated+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flanking+gene+and+genetic+background+problems+in+genetically+manipulated+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001303-Cuthbert1" id="pone.0001303-Cuthbert1"></a>Cuthbert PC, Stanford LE, Coba MP, Ainge JA, Fink AE, et al.  (2007) Synapse-associated protein 102/dlgh3 couples the NMDA receptor to specific plasticity pathways and learning strategies. J Neurosci  27: 2673–2682.  <ul class="find" data-citedArticleID="1029148" data-doi="10.1523/jneurosci.4457-06.2007"><li><a href="http://dx.doi.org/10.1523/jneurosci.4457-06.2007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Synapse-associated+protein+102%2Fdlgh3+couples+the+NMDA+receptor+to+specific+plasticity+pathways+and+learning+strategies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Synapse-associated+protein+102%2Fdlgh3+couples+the+NMDA+receptor+to+specific+plasticity+pathways+and+learning+strategies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001303-Migaud1" id="pone.0001303-Migaud1"></a>Migaud M, Charlesworth P, Dempster M, Webster LC, Watabe AM, et al.  (1998) Enhanced long-term potentiation and impaired learning in mice with mutant postsynaptic density-95 protein. Nature  396: 433–439.  <ul class="find" data-citedArticleID="1029174"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Enhanced+long-term+potentiation+and+impaired+learning+in+mice+with+mutant+postsynaptic+density-95+protein.&amp;auth=&amp;atitle=Enhanced+long-term+potentiation+and+impaired+learning+in+mice+with+mutant+postsynaptic+density-95+protein." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhanced+long-term+potentiation+and+impaired+learning+in+mice+with+mutant+postsynaptic+density-95+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhanced+long-term+potentiation+and+impaired+learning+in+mice+with+mutant+postsynaptic+density-95+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001303-Nguyen1" id="pone.0001303-Nguyen1"></a>Nguyen PV, Abel T, Kandel ER (1994) Requirement of a critical period of transcription for induction of a late phase of LTP. Science  265: 1104–1107.  <ul class="find" data-citedArticleID="1029182" data-doi="10.1126/science.8066450"><li><a href="http://dx.doi.org/10.1126/science.8066450" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+of+a+critical+period+of+transcription+for+induction+of+a+late+phase+of+LTP." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+of+a+critical+period+of+transcription+for+induction+of+a+late+phase+of+LTP.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001303-Linden1" id="pone.0001303-Linden1"></a>Linden DJ (1996) A protein synthesis-dependent late phase of cerebellar long-term depression. Neuron  17: 483–490.  <ul class="find" data-citedArticleID="1029170" data-doi="10.1016/s0896-6273(00)80180-4"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)80180-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+protein+synthesis-dependent+late+phase+of+cerebellar+long-term+depression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+protein+synthesis-dependent+late+phase+of+cerebellar+long-term+depression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001303-Milner1" id="pone.0001303-Milner1"></a>Milner B, Squire LR, Kandel ER (1998) Cognitive neuroscience and the study of memory. Neuron  20: 445–468.  <ul class="find" data-citedArticleID="1029176" data-doi="10.1016/s0896-6273(00)80987-3"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)80987-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cognitive+neuroscience+and+the+study+of+memory." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cognitive+neuroscience+and+the+study+of+memory.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001303-Alberini1" id="pone.0001303-Alberini1"></a>Alberini CM (1999) Genes to remember. J Exp Biol  202: 2887–2891.  <ul class="find" data-citedArticleID="1029136"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genes+to+remember.&amp;auth=&amp;atitle=Genes+to+remember." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genes+to+remember." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genes+to+remember.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001303-Bading1" id="pone.0001303-Bading1"></a>Bading H (2000) Transcription-dependent neuronal plasticity the nuclear calcium hypothesis. Eur J Biochem  267: 5280–5283.  <ul class="find" data-citedArticleID="1029138" data-doi="10.1046/j.1432-1327.2000.01565.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1327.2000.01565.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcription-dependent+neuronal+plasticity+the+nuclear+calcium+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcription-dependent+neuronal+plasticity+the+nuclear+calcium+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001303-Qian1" id="pone.0001303-Qian1"></a>Qian Z, Gilbert ME, Colicos MA, Kandel ER, Kuhl D (1993) Tissue-plasminogen activator is induced as an immediate-early gene during seizure, kindling and long-term potentiation. Nature  361: 453–457.  <ul class="find" data-citedArticleID="1029190" data-doi="10.1038/361453a0"><li><a href="http://dx.doi.org/10.1038/361453a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tissue-plasminogen+activator+is+induced+as+an+immediate-early+gene+during+seizure%2C+kindling+and+long-term+potentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tissue-plasminogen+activator+is+induced+as+an+immediate-early+gene+during+seizure%2C+kindling+and+long-term+potentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001303-West1" id="pone.0001303-West1"></a>West AE, Griffith EC, Greenberg ME (2002) Regulation of transcription factors by neuronal activity. Nat Rev Neurosci  3: 921–931.  <ul class="find" data-citedArticleID="1029206" data-doi="10.1038/nrn987"><li><a href="http://dx.doi.org/10.1038/nrn987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+transcription+factors+by+neuronal+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+transcription+factors+by+neuronal+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001303-Tu1" id="pone.0001303-Tu1"></a>Tu JC, Xiao B, Naisbitt S, Yuan JP, Petralia RS, et al.  (1999) Coupling of mGluR/Homer and PSD-95 complexes by the Shank family of postsynaptic density proteins. Neuron  23: 583–592.  <ul class="find" data-citedArticleID="1029202" data-doi="10.1016/s0896-6273(00)80810-7"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)80810-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Coupling+of+mGluR%2FHomer+and+PSD-95+complexes+by+the+Shank+family+of+postsynaptic+density+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Coupling+of+mGluR%2FHomer+and+PSD-95+complexes+by+the+Shank+family+of+postsynaptic+density+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001303-Naisbitt1" id="pone.0001303-Naisbitt1"></a>Naisbitt S, Kim E, Tu JC, Xiao B, Sala C, et al.  (1999) Shank, a novel family of postsynaptic density proteins that binds to the NMDA receptor/PSD-95/GKAP complex and cortactin. Neuron  23: 569–582.  <ul class="find" data-citedArticleID="1029180" data-doi="10.1016/s0896-6273(00)80809-0"><li><a href="http://dx.doi.org/10.1016/s0896-6273(00)80809-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Shank%2C+a+novel+family+of+postsynaptic+density+proteins+that+binds+to+the+NMDA+receptor%2FPSD-95%2FGKAP+complex+and+cortactin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Shank%2C+a+novel+family+of+postsynaptic+density+proteins+that+binds+to+the+NMDA+receptor%2FPSD-95%2FGKAP+complex+and+cortactin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001303-Husi1" id="pone.0001303-Husi1"></a>Husi H, Ward MA, Choudhary JS, Blackstock WP, Grant SG (2000) Proteomic analysis of NMDA receptor-adhesion protein signaling complexes. Nat Neurosci  3: 661–669.  <ul class="find" data-citedArticleID="1029158" data-doi="10.1038/76615"><li><a href="http://dx.doi.org/10.1038/76615" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomic+analysis+of+NMDA+receptor-adhesion+protein+signaling+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomic+analysis+of+NMDA+receptor-adhesion+protein+signaling+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001303-Husi2" id="pone.0001303-Husi2"></a>Husi H, Grant SG (2001) Isolation of 2000-kDa complexes of N-methyl-D-aspartate receptor and postsynaptic density 95 from mouse brain. J Neurochem  77: 281–291.  <ul class="find" data-citedArticleID="1029160" data-doi="10.1046/j.1471-4159.2001.00248.x"><li><a href="http://dx.doi.org/10.1046/j.1471-4159.2001.00248.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Isolation+of+2000-kDa+complexes+of+N-methyl-D-aspartate+receptor+and+postsynaptic+density+95+from+mouse+brain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Isolation+of+2000-kDa+complexes+of+N-methyl-D-aspartate+receptor+and+postsynaptic+density+95+from+mouse+brain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001303-Collins1" id="pone.0001303-Collins1"></a>Collins MO, Husi H, Yu L, Brandon JM, Anderson CN, et al.  (2005) Molecular characterization and comparison of the components and multiprotein complexes in the postsynaptic proteome. J Neurochem.   <ul class="find" data-citedArticleID="1029144" data-doi="10.1111/j.1471-4159.2005.03507.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.2005.03507.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+characterization+and+comparison+of+the+components+and+multiprotein+complexes+in+the+postsynaptic+proteome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+characterization+and+comparison+of+the+components+and+multiprotein+complexes+in+the+postsynaptic+proteome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001303-Pocklington1" id="pone.0001303-Pocklington1"></a>Pocklington AJ, Armstrong JD, Grant SG (2006) Organization of brain complexity–synapse proteome form and function. Brief Funct Genomic Proteomic  5: 66–73.  <ul class="find" data-citedArticleID="1029188" data-doi="10.1093/bfgp/ell013"><li><a href="http://dx.doi.org/10.1093/bfgp/ell013" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organization+of+brain+complexity%E2%80%93synapse+proteome+form+and+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organization+of+brain+complexity%E2%80%93synapse+proteome+form+and+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001303-Husi3" id="pone.0001303-Husi3"></a>Husi H, Grant SG (2001) Proteomics of the nervous system. Trends Neurosci  24: 259–266.  <ul class="find" data-citedArticleID="1029162" data-doi="10.1016/s0166-2236(00)01792-6"><li><a href="http://dx.doi.org/10.1016/s0166-2236(00)01792-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proteomics+of+the+nervous+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proteomics+of+the+nervous+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001303-Woods1" id="pone.0001303-Woods1"></a>Woods DF, Bryant PJ (1991) The discs-large tumor suppressor gene of Drosophila encodes a guanylate kinase homolog localized at septate junctions. Cell  66: 451–464.  <ul class="find" data-citedArticleID="1029212" data-doi="10.1016/0092-8674(81)90009-x"><li><a href="http://dx.doi.org/10.1016/0092-8674(81)90009-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+discs-large+tumor+suppressor+gene+of+Drosophila+encodes+a+guanylate+kinase+homolog+localized+at+septate+junctions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+discs-large+tumor+suppressor+gene+of+Drosophila+encodes+a+guanylate+kinase+homolog+localized+at+septate+junctions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001303-Komiyama1" id="pone.0001303-Komiyama1"></a>Komiyama NH, Watabe AM, Carlisle HJ, Porter K, Charlesworth P, et al.  (2002) SynGAP regulates ERK/MAPK signaling, synaptic plasticity, and learning in the complex with postsynaptic density 95 and NMDA receptor. J Neurosci  22: 9721–9732.  <ul class="find" data-citedArticleID="1029164"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=SynGAP+regulates+ERK%2FMAPK+signaling%2C+synaptic+plasticity%2C+and+learning+in+the+complex+with+postsynaptic+density+95+and+NMDA+receptor.&amp;auth=&amp;atitle=SynGAP+regulates+ERK%2FMAPK+signaling%2C+synaptic+plasticity%2C+and+learning+in+the+complex+with+postsynaptic+density+95+and+NMDA+receptor." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SynGAP+regulates+ERK%2FMAPK+signaling%2C+synaptic+plasticity%2C+and+learning+in+the+complex+with+postsynaptic+density+95+and+NMDA+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SynGAP+regulates+ERK%2FMAPK+signaling%2C+synaptic+plasticity%2C+and+learning+in+the+complex+with+postsynaptic+density+95+and+NMDA+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001303-Opazo1" id="pone.0001303-Opazo1"></a>Opazo P, Watabe AM, Grant SG, O'Dell TJ (2003) Phosphatidylinositol 3-kinase regulates the induction of long-term potentiation through extracellular signal-related kinase-independent mechanisms. J Neurosci  23: 3679–3688.  <ul class="find" data-citedArticleID="1029186"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Phosphatidylinositol+3-kinase+regulates+the+induction+of+long-term+potentiation+through+extracellular+signal-related+kinase-independent+mechanisms.&amp;auth=&amp;atitle=Phosphatidylinositol+3-kinase+regulates+the+induction+of+long-term+potentiation+through+extracellular+signal-related+kinase-independent+mechanisms." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phosphatidylinositol+3-kinase+regulates+the+induction+of+long-term+potentiation+through+extracellular+signal-related+kinase-independent+mechanisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phosphatidylinositol+3-kinase+regulates+the+induction+of+long-term+potentiation+through+extracellular+signal-related+kinase-independent+mechanisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001303-Yao1" id="pone.0001303-Yao1"></a>Yao WD, Gainetdinov RR, Arbuckle MI, Sotnikova TD, Cyr M, et al.  (2004) Identification of PSD-95 as a regulator of dopamine-mediated synaptic and behavioral plasticity. Neuron  41: 625–638.  <ul class="find" data-citedArticleID="1029216" data-doi="10.1016/s0896-6273(04)00048-0"><li><a href="http://dx.doi.org/10.1016/s0896-6273(04)00048-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+PSD-95+as+a+regulator+of+dopamine-mediated+synaptic+and+behavioral+plasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+PSD-95+as+a+regulator+of+dopamine-mediated+synaptic+and+behavioral+plasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001303-Fagiolini1" id="pone.0001303-Fagiolini1"></a>Fagiolini M, Katagiri H, Miyamoto H, Mori H, Grant SG, et al.  (2003) Separable features of visual cortical plasticity revealed by N-methyl-D-aspartate receptor 2A signaling. Proc Natl Acad Sci U S A  100: 2854–2859.  <ul class="find" data-citedArticleID="1029150" data-doi="10.1073/pnas.0536089100"><li><a href="http://dx.doi.org/10.1073/pnas.0536089100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Separable+features+of+visual+cortical+plasticity+revealed+by+N-methyl-D-aspartate+receptor+2A+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Separable+features+of+visual+cortical+plasticity+revealed+by+N-methyl-D-aspartate+receptor+2A+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001303-Garry1" id="pone.0001303-Garry1"></a>Garry EM, Moss A, Delaney A, O'Neill F, Blakemore J, et al.  (2003) Neuropathic sensitization of behavioral reflexes and spinal NMDA receptor/CaM kinase II interactions are disrupted in PSD-95 mutant mice. Curr Biol  13: 321–328.  <ul class="find" data-citedArticleID="1029152" data-doi="10.1016/s0960-9822(03)00084-8"><li><a href="http://dx.doi.org/10.1016/s0960-9822(03)00084-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuropathic+sensitization+of+behavioral+reflexes+and+spinal+NMDA+receptor%2FCaM+kinase+II+interactions+are+disrupted+in+PSD-95+mutant+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuropathic+sensitization+of+behavioral+reflexes+and+spinal+NMDA+receptor%2FCaM+kinase+II+interactions+are+disrupted+in+PSD-95+mutant+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001303-Kuehn1" id="pone.0001303-Kuehn1"></a>Kuehn MR, Bradley A, Robertson EJ, Evans MJ (1987) A potential animal model for Lesch-Nyhan syndrome through introduction of HPRT mutations into mice. Nature  326: 295–298.  <ul class="find" data-citedArticleID="1029166" data-doi="10.1038/326295a0"><li><a href="http://dx.doi.org/10.1038/326295a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+potential+animal+model+for+Lesch-Nyhan+syndrome+through+introduction+of+HPRT+mutations+into+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+potential+animal+model+for+Lesch-Nyhan+syndrome+through+introduction+of+HPRT+mutations+into+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001303-Hooper1" id="pone.0001303-Hooper1"></a>Hooper M, Hardy K, Handyside A, Hunter S, Monk M (1987) HPRT-deficient (Lesch-Nyhan) mouse embryos derived from germline colonization by cultured cells. Nature  326: 292–295.  <ul class="find" data-citedArticleID="1029156" data-doi="10.1038/326292a0"><li><a href="http://dx.doi.org/10.1038/326292a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HPRT-deficient+%28Lesch-Nyhan%29+mouse+embryos+derived+from+germline+colonization+by+cultured+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HPRT-deficient+%28Lesch-Nyhan%29+mouse+embryos+derived+from+germline+colonization+by+cultured+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001303-Moore1" id="pone.0001303-Moore1"></a>Moore RC, Redhead NJ, Selfridge J, Hope J, Manson JC, et al.  (1995) Double replacement gene targeting for the production of a series of mouse strains with different prion protein gene alterations. Biotechnology (N Y)  13: 999–1004.  <ul class="find" data-citedArticleID="1029178" data-doi="10.1038/nbt0995-999"><li><a href="http://dx.doi.org/10.1038/nbt0995-999" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Double+replacement+gene+targeting+for+the+production+of+a+series+of+mouse+strains+with+different+prion+protein+gene+alterations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Double+replacement+gene+targeting+for+the+production+of+a+series+of+mouse+strains+with+different+prion+protein+gene+alterations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001303-Selbert1" id="pone.0001303-Selbert1"></a>Selbert S, Bentley DJ, Melton DW, Rannie D, Lourenco P, et al.  (1998) Efficient BLG-Cre mediated gene deletion in the mammary gland. Transgenic Res  7: 387–396.  <ul class="find" data-citedArticleID="1029194"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Efficient+BLG-Cre+mediated+gene+deletion+in+the+mammary+gland.&amp;auth=&amp;atitle=Efficient+BLG-Cre+mediated+gene+deletion+in+the+mammary+gland." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+BLG-Cre+mediated+gene+deletion+in+the+mammary+gland." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+BLG-Cre+mediated+gene+deletion+in+the+mammary+gland.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001303-Tsuda1" id="pone.0001303-Tsuda1"></a>Tsuda H, Maynard-Currie CE, Reid LH, Yoshida T, Edamura K, et al.  (1997) Inactivation of the mouse HPRT locus by a 203-bp retroposon insertion and a 55-kb gene-targeted deletion: establishment of new HPRT-deficient mouse embryonic stem cell lines. Genomics  42: 413–421.  <ul class="find" data-citedArticleID="1029200" data-doi="10.1006/geno.1997.4771"><li><a href="http://dx.doi.org/10.1006/geno.1997.4771" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+the+mouse+HPRT+locus+by+a+203-bp+retroposon+insertion+and+a+55-kb+gene-targeted+deletion%3A+establishment+of+new+HPRT-deficient+mouse+embryonic+stem+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+the+mouse+HPRT+locus+by+a+203-bp+retroposon+insertion+and+a+55-kb+gene-targeted+deletion%3A+establishment+of+new+HPRT-deficient+mouse+embryonic+stem+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001303-McEwan1" id="pone.0001303-McEwan1"></a>McEwan C, Melton DW (2003) A simple genotyping assay for the Hprt null allele in mice produced from the HM-1 and E14TG2a mouse embryonic stem cell lines. Transgenic Res  12: 519–520.  <ul class="find" data-citedArticleID="1029172"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple+genotyping+assay+for+the+Hprt+null+allele+in+mice+produced+from+the+HM-1+and+E14TG2a+mouse+embryonic+stem+cell+lines.&amp;auth=&amp;atitle=A+simple+genotyping+assay+for+the+Hprt+null+allele+in+mice+produced+from+the+HM-1+and+E14TG2a+mouse+embryonic+stem+cell+lines." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple+genotyping+assay+for+the+Hprt+null+allele+in+mice+produced+from+the+HM-1+and+E14TG2a+mouse+embryonic+stem+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple+genotyping+assay+for+the+Hprt+null+allele+in+mice+produced+from+the+HM-1+and+E14TG2a+mouse+embryonic+stem+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001303-Cecchi1" id="pone.0001303-Cecchi1"></a>Cecchi C, Avner P (1996) Genomic organization of the mottled gene, the mouse homologue of the human Menkes disease gene. Genomics  37: 96–104.  <ul class="find" data-citedArticleID="1029142" data-doi="10.1006/geno.1996.0525"><li><a href="http://dx.doi.org/10.1006/geno.1996.0525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+organization+of+the+mottled+gene%2C+the+mouse+homologue+of+the+human+Menkes+disease+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+organization+of+the+mottled+gene%2C+the+mouse+homologue+of+the+human+Menkes+disease+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001303-Behr1" id="pone.0001303-Behr1"></a>Behr R, Sackett S, Bochkis I, Phillip L, Kaestner K (2007) Impaired male fertility and atrophy of semineforous tubules caused by haploinsufficiency for Foxa3. Dev Biol In press.   <ul class="find" data-citedArticleID="1029140" data-doi="10.1016/j.ydbio.2007.03.525"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2007.03.525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impaired+male+fertility+and+atrophy+of+semineforous+tubules+caused+by+haploinsufficiency+for+Foxa3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impaired+male+fertility+and+atrophy+of+semineforous+tubules+caused+by+haploinsufficiency+for+Foxa3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001303-Witt1" id="pone.0001303-Witt1"></a>Witt CC, Burkart C, Labeit D, McNabb M, Wu Y, et al.  (2006) Nebulin regulates thin filament length, contractility, and Z-disk structure in vivo. Embo J  25: 3843–3855.  <ul class="find" data-citedArticleID="1029208" data-doi="10.1038/sj.emboj.7601242"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601242" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nebulin+regulates+thin+filament+length%2C+contractility%2C+and+Z-disk+structure+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nebulin+regulates+thin+filament+length%2C+contractility%2C+and+Z-disk+structure+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001303-Niedernhofer1" id="pone.0001303-Niedernhofer1"></a>Niedernhofer LJ, Garinis GA, Raams A, Lalai AS, Robinson AR, et al.  (2006) A new progeroid syndrome reveals that genotoxic stress suppresses the somatotroph axis. Nature  444: 1038–1043.  <ul class="find" data-citedArticleID="1029184" data-doi="10.1038/nature05456"><li><a href="http://dx.doi.org/10.1038/nature05456" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+progeroid+syndrome+reveals+that+genotoxic+stress+suppresses+the+somatotroph+axis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+progeroid+syndrome+reveals+that+genotoxic+stress+suppresses+the+somatotroph+axis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001303-Hanada1" id="pone.0001303-Hanada1"></a>Hanada K, Vermeij M, Garinis GA, de Waard MC, Kunen MG, et al.  (2007) Perturbations of vascular homeostasis and aortic valve abnormalities in fibulin-4 deficient mice. Circ Res  100: 738–746.  <ul class="find" data-citedArticleID="1029154" data-doi="10.1161/01.res.0000260181.19449.95"><li><a href="http://dx.doi.org/10.1161/01.res.0000260181.19449.95" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Perturbations+of+vascular+homeostasis+and+aortic+valve+abnormalities+in+fibulin-4+deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Perturbations+of+vascular+homeostasis+and+aortic+valve+abnormalities+in+fibulin-4+deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001303-Yang1" id="pone.0001303-Yang1"></a>Yang S, Farias M, Kapfhamer D, Tobias J, Grant G, et al.  (2007) Biochemical, molecular and behavioral phenotypes of Rab3A mutations in the mouse. Genes Brain Behav  6: 77–96.  <ul class="find" data-citedArticleID="1029214" data-doi="10.1111/j.1601-183x.2006.00235.x"><li><a href="http://dx.doi.org/10.1111/j.1601-183x.2006.00235.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biochemical%2C+molecular+and+behavioral+phenotypes+of+Rab3A+mutations+in+the+mouse." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biochemical%2C+molecular+and+behavioral+phenotypes+of+Rab3A+mutations+in+the+mouse.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001303-Schalkwyk1" id="pone.0001303-Schalkwyk1"></a>Schalkwyk LC, Fernandes C, Nash MW, Kurrikoff K, Vasar E, et al.  (2007) Interpretation of knockout experiments: the congenic footprint. Genes Brain Behav  6: 299–303.  <ul class="find" data-citedArticleID="1029192" data-doi="10.1111/j.1601-183x.2007.00304.x"><li><a href="http://dx.doi.org/10.1111/j.1601-183x.2007.00304.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interpretation+of+knockout+experiments%3A+the+congenic+footprint." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interpretation+of+knockout+experiments%3A+the+congenic+footprint.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001303-Simpson1" id="pone.0001303-Simpson1"></a>Simpson EM, Linder CC, Sargent EE, Davisson MT, Mobraaten LE, et al.  (1997) Genetic variation among 129 substrains and its importance for targeted mutagenesis in mice. Nat Genet  16: 19–27.  <ul class="find" data-citedArticleID="1029196" data-doi="10.1038/ng0597-19"><li><a href="http://dx.doi.org/10.1038/ng0597-19" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variation+among+129+substrains+and+its+importance+for+targeted+mutagenesis+in+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variation+among+129+substrains+and+its+importance+for+targeted+mutagenesis+in+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001303-Threadgill1" id="pone.0001303-Threadgill1"></a>Threadgill DW, Yee D, Matin A, Nadeau JH, Magnuson T (1997) Genealogy of the 129 inbred strains: 129/SvJ is a contaminated inbred strain. Mamm Genome  8: 390–393.  <ul class="find" data-citedArticleID="1029198" data-doi="10.1007/s003359900453"><li><a href="http://dx.doi.org/10.1007/s003359900453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genealogy+of+the+129+inbred+strains%3A+129%2FSvJ+is+a+contaminated+inbred+strain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genealogy+of+the+129+inbred+strains%3A+129%2FSvJ+is+a+contaminated+inbred+strain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001303-Wolfer1" id="pone.0001303-Wolfer1"></a>Wolfer DP, Crusio WE, Lipp HP (2002) Knockout mice: simple solutions to the problems of genetic background and flanking genes. Trends Neurosci  25: 336–340.  <ul class="find" data-citedArticleID="1029210" data-doi="10.1016/s0166-2236(02)02192-6"><li><a href="http://dx.doi.org/10.1016/s0166-2236(02)02192-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Knockout+mice%3A+simple+solutions+to+the+problems+of+genetic+background+and+flanking+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Knockout+mice%3A+simple+solutions+to+the+problems+of+genetic+background+and+flanking+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001303-Li1" id="pone.0001303-Li1"></a>Li C, Hung Wong W (2001) Model-based analysis of oligonucleotide arrays: model validation, design issues and standard error application. Genome Biol  2: RESEARCH0032.  <ul class="find" data-citedArticleID="1029168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Model-based+analysis+of+oligonucleotide+arrays%3A+model+validation%2C+design+issues+and+standard+error+application.&amp;auth=&amp;atitle=Model-based+analysis+of+oligonucleotide+arrays%3A+model+validation%2C+design+issues+and+standard+error+application." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model-based+analysis+of+oligonucleotide+arrays%3A+model+validation%2C+design+issues+and+standard+error+application." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model-based+analysis+of+oligonucleotide+arrays%3A+model+validation%2C+design+issues+and+standard+error+application.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.XML" value="94940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.PDF" value="262029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g001.PNG_L" value="146878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g001.PNG_M" value="26755"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g001.PNG_S" value="6340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g001.TIF" value="443342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g001.PNG_I" value="15973"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t001.PNG_L" value="276146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t001.PNG_M" value="158246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t001.PNG_S" value="14784"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t001.TIF" value="1131842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t001.PNG_I" value="48705"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g002.PNG_L" value="441899"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g002.PNG_M" value="25813"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g002.PNG_S" value="3976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g002.TIF" value="465630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g002.PNG_I" value="8730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t002.PNG_L" value="231502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t002.PNG_M" value="130155"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t002.PNG_S" value="13269"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t002.TIF" value="941624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t002.PNG_I" value="39199"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t003.PNG_L" value="137425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t003.PNG_M" value="189116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t003.PNG_S" value="21148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t003.TIF" value="537502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t003.PNG_I" value="65995"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t004.PNG_L" value="154827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t004.PNG_M" value="216135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t004.PNG_S" value="22640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t004.TIF" value="601818"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.t004.PNG_I" value="75151"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g003.PNG_L" value="106825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g003.PNG_M" value="22518"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g003.PNG_S" value="6674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g003.TIF" value="338920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g003.PNG_I" value="11740"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g004.PNG_L" value="143845"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g004.PNG_M" value="22185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g004.PNG_S" value="8123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g004.TIF" value="339046"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001303.g004.PNG_I" value="40630"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001303&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001303" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001303&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001303&volume=&issue=&title=Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice&author_name=Luis%20M.%20Valor%2C%20Seth%20G.%20N.%20Grant&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001303" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303&amp;t=Clustered%20Gene%20Expression%20Changes%20Flank%20Targeted%20Gene%20Loci%20in%20Knockout%20Mice" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303&title=Clustered%20Gene%20Expression%20Changes%20Flank%20Targeted%20Gene%20Loci%20in%20Knockout%20Mice&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303&amp;title=Clustered%20Gene%20Expression%20Changes%20Flank%20Targeted%20Gene%20Loci%20in%20Knockout%20Mice" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001303&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Clustered Gene Expression Changes Flank Targeted Gene Loci in Knockout Mice';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001303';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Clustered%20Gene%20Expression%20Changes%20Flank%20Targeted%20Gene%20Loci%20in%20Knockout%20Mice http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001303" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001303" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="25438"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25438"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
              <div data-categoryid="17701" data-articleid="25438"
                   data-categoryname="Gene targeting"
                   class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="25438"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hippocampus%22" title="Search for articles in the subject area:'Hippocampus'"><div class="flagText">Hippocampus</div></a>
              <div data-categoryid="47345" data-articleid="25438"
                   data-categoryname="Hippocampus"
                   class="flagImage" title="Flag 'Hippocampus' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25438"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mouse+models%22" title="Search for articles in the subject area:'Mouse models'"><div class="flagText">Mouse models</div></a>
              <div data-categoryid="32069" data-articleid="25438"
                   data-categoryname="Mouse models"
                   class="flagImage" title="Flag 'Mouse models' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
              <div data-categoryid="31809" data-articleid="25438"
                   data-categoryname="Mutation"
                   class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1194'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8039'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2943&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
