

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001390"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001390" />

    <meta name="citation_title" content="High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania"/>
    <meta itemprop="name" content="High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania"/>

      <meta name="citation_author" content="Marco Salemi"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>
      <meta name="citation_author" content="Tulio de Oliveira"/>
            <meta name="citation_author_institution" content="Medical Research Council (MRC) Bioinformatics Unit, South African National Bioinformatics Institute, University of Western Cape, Cape Town, South Africa"/>
      <meta name="citation_author" content="Massimo Ciccozzi"/>
            <meta name="citation_author_institution" content="Istituto Superiore di Sanita', Roma, Italy"/>
      <meta name="citation_author" content="Giovanni Rezza"/>
            <meta name="citation_author_institution" content="Istituto Superiore di Sanita', Roma, Italy"/>
      <meta name="citation_author" content="Maureen M. Goodenow"/>
            <meta name="citation_author_institution" content="Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America"/>

    <meta name="citation_date" content="2008/1/2"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001390.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1390"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Molecular cloning and characterization of the HTLV-III virus associated with AIDS.; citation_author=BH Hahn; citation_author=GM Shaw; citation_author=SK Arya; citation_author=M Popovic; citation_author=RC Gallo; citation_journal_title=Nature; citation_volume=312; citation_number=1; citation_pages=166-169; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=Genetic variability of the human immunodeficiency virus: statistical and biological issues.; citation_author=F Seillier-Moiseiwitsch; citation_author=BH Margolin; citation_author=R Swanstrom; citation_journal_title=Annu. Rev. Genet.; citation_volume=28; citation_number=2; citation_pages=559-96; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Genetic diversity of HIV-1: the moving target.; citation_author=M Peeters; citation_author=PM Sharp; citation_journal_title=AIDS; citation_volume=14; citation_number=3; citation_pages=S129-S140; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=HIV-1 nomenclature proposal.; citation_author=DL Robertson; citation_author=JP Anderson; citation_author=JA Bradac; citation_author=JK Carr; citation_author=B Foley; citation_journal_title=Science; citation_volume=288; citation_number=4; citation_pages=55-56; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Understanding the genetic diversity of HIV-1.; citation_author=FE McCutchan; citation_journal_title=AIDS; citation_volume=14; citation_number=5; citation_pages=S31-S44; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of breakpoints in intergenotypic recombinants of HIV type 1 by bootscanning.; citation_author=MO Salminen; citation_author=JK Carr; citation_author=DS Burke; citation_author=FE McCutchan; citation_journal_title=AIDS Res. Hum. Retrovir.; citation_volume=11; citation_number=6; citation_pages=1423-1425; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Origin of HIV-1 in the chimpanzee Pan troglodytes troglodytes.; citation_author=F Gao; citation_author=E Bailes; citation_author=DL Robertson; citation_author=Y Chen; citation_author=CM Rodenburg; citation_journal_title=Nature.; citation_volume=397; citation_number=7; citation_pages=436-441; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Timing the ancestor of the HIV-1 pandemic strains.; citation_author=B Korber; citation_author=M Muldoon; citation_author=J Theiler; citation_author=F Gao; citation_author=R Gupta; citation_journal_title=Science; citation_volume=288; citation_number=8; citation_pages=1789-1796; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Dating the common ancestor of SIVcpz and HIV-1 group M and the origin of HIV-1 subtypes using a new method to uncover clock-like molecular evolution.; citation_author=M Salemi; citation_author=K Strimmer; citation_author=WW Hall; citation_author=M Duffy; citation_author=E Delaporte; citation_journal_title=FASEB J; citation_volume=15; citation_number=9; citation_pages=276-278; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Accelerating the development and future availability of HIV-1 vaccines: why, when, where, and how?; citation_author=J Esparza; citation_author=N Bhamarapravati; citation_journal_title=Lancet; citation_volume=355; citation_number=10; citation_pages=2061-2066; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Circulation of subtype A and gagA/envB recombinant HIV type 1 strains among injecting drug users in St. Petersburg, Russia, correlates with geographical origin of infections.; citation_author=VV Lukashov; citation_author=R Huismans; citation_author=AG Rakhmanova; citation_author=ZN Lisistina; citation_author=NA Akhtyrskaya; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=15; citation_number=11; citation_pages=1577-1583; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The HIV type 1 epidemic in Belarus: predominance of Eastern European subtype A strains and circulation of subtype B strains.; citation_author=NV Lazouskaya; citation_author=VF Eremin; citation_author=KW Adema; citation_author=EL Gasich; citation_author= Baan; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=21; citation_number=12; citation_pages=830-833; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Molecular epidemiology of HIV type 1 in Ukraine: birthplace of an epidemic.; citation_author=MD Saad; citation_author=AM Shcherbinskaya; citation_author=Y Nadai; citation_author=YV Kruglov; citation_author=SV Antonenko; citation_journal_title=AIDS Res Hum Retrovir; citation_volume=22; citation_number=13; citation_pages=709-714; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=HIV-1 subtypes in Yugoslavia.; citation_author=M Stanojevic; citation_author=A Papa; citation_author=E Papadimitriou; citation_author=E Zerjav; citation_author=D Jevtovic; citation_journal_title=AIDS Res Hum; citation_volume=Retroviruses18; citation_number=14; citation_pages=519-22; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Molecular diversity of HIV in Albania.; citation_author=M Ciccozzi; citation_author=C Gori; citation_author=S Boros; citation_author=MJ Ruiz-Alvarez; citation_author=A Harxhi; citation_journal_title=J Infect Dis; citation_volume=192; citation_number=15; citation_pages=475-479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The epidemic behavior of the hepatitis C virus.; citation_author=OG Pybus; citation_author=MA Charleston; citation_author=S Gupta; citation_author=A Rambaut; citation_author=EC Holmes; citation_journal_title=Science; citation_volume=292; citation_number=16; citation_pages=2323-2325; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic analysis reveals the complex structure of HIV-1 transmission within defined risk groups.; citation_author=S Hue; citation_author=D Pillay; citation_author=JP Clewley; citation_author=OG Pybus; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=17; citation_pages=4425-4429; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Tracing the origin and history of HIV-2 epidemics.; citation_author=P Lemey; citation_author=OG Pybus; citation_author=B Wang; citation_author=NK Saksena; citation_author=M Salemi; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=100; citation_number=18; citation_pages=6588-6592; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Characterization of Early U.S. HIV-1 Strains, Dating of the Introduction of HIV-1 into the U.S. and Estimation of Population History of the U.S. Epidemic.; citation_author=E Robbins; citation_author=P Lemey; citation_author=OG Pybus; citation_author=HW Jaffe; citation_author=AM Saekhou; citation_journal_title=J Virol; citation_volume=77; citation_number=19; citation_pages=6359-6366; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Different epidemic potentials of HIV-1 B and C subtypes.; citation_author=M Salemi; citation_author=T de Oliveira; citation_author=TM Soares; citation_author=OG Pybus; citation_author=AT Dumans; citation_journal_title=J Mol Evol; citation_volume=60; citation_number=20; citation_pages=598-605; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=HIV-1 pol gene variation is sufficient for reconstruction of transmissions in the era of antiretroviral therapy.; citation_author=S Hue; citation_author=JP Clewley; citation_author=PA Cane; citation_author=D Pillay; citation_journal_title=AIDS; citation_volume=18(5); citation_number=21; citation_pages=719-28; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Distinct patterns of natural selection in the reverse transcriptase gene of HIV-1 in the presence and absence of antiretroviral therapy.; citation_author=E de S Leal; citation_author=EC Holmes; citation_author=PM Zanotto; citation_journal_title=Virology; citation_volume=325; citation_number=22; citation_pages=181-191; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Nevirapine resistance mutations of human immunodeficiency virus type 1 selected during therapy.; citation_author=DD Richman; citation_author=D Havlir; citation_author=J Corbeil; citation_author=D Looney; citation_author=C Ignacio; citation_journal_title=J Virol; citation_volume=68; citation_number=23; citation_pages=1660-1666; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Evolution of lamivudine resistance in human immunodeficiency virus type 1-infected individuals: the relative roles of drift and selection.; citation_author=SD Frost; citation_author=M Nijhuis; citation_author=R Schuurman; citation_author=CA Boucher; citation_author=AJ Brown; citation_journal_title=J Virol; citation_volume=74; citation_number=24; citation_pages=6262-6268; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Search for the mechanism of genetic variation in the pro gene of human immunodeficiency virus.; citation_author=IM Rouzine; citation_author=JM Coffin; citation_journal_title=J Virol; citation_volume=73; citation_number=25; citation_pages=8167-8178; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Unifying the epidemiological and evolutionary dynamics of pathogens.; citation_author=BT Grenfell; citation_author=OG Pybus; citation_author=JR Gog; citation_author=JL Wood; citation_author=JM Daly; citation_journal_title=Science; citation_volume=303; citation_number=26; citation_pages=327-333; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A cladistic measure of gene flow inferred from the phylogenies of alleles.; citation_author=M Slatkin; citation_author=WP Maddison; citation_journal_title=Genetics; citation_volume=123; citation_number=27; citation_pages=401-416; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools; citation_author=JD Thompson; citation_author=TJ Gibson; citation_author=F Plewniak; citation_author=F Jeanmougin; citation_author=DG Higgins; citation_journal_title=Nucleic Acids Res; citation_volume=25; citation_number=28; citation_pages=4876-4882; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Phylogeny Inference based on parsimony and other methods with PAUP*.; citation_author=D Swofford; citation_author=J Sullivan; citation_number=29; citation_pages=160-206; citation_date=2003; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=Estimating divergence dates from molecular sequences.; citation_author=A Rambaut; citation_author=L Bromham; citation_journal_title=Mol Biol Evol; citation_volume=15; citation_number=30; citation_pages=442-448; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Estimating the rate of molecular evolution: incorporating non-contemporaneous sequences into maximum likelihood phylogenies.; citation_author=A Rambaut; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=31; citation_pages=395-399; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence alignment.; citation_author=K Strimmer; citation_author=A von Haeseler; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=94; citation_number=32; citation_pages=6815-6819; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=TREE-PUZZLE: a maximum likelhood phylogenetic analysis using quartets and parallel computing.; citation_author=HA Schmidt; citation_author=K Strimmer; citation_author=M Vingron; citation_author=A von Haeseler; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=33; citation_pages=502-504; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Relaxed Phylogenetics and Dating with Confidence.; citation_author=AJ Drummond; citation_author=SYW Ho; citation_author=MJ Phillips; citation_author=A Rambaut; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=34; citation_pages=1-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Bayesian coalescent inference of past population dynamics from molecular sequences.; citation_author=A Drummond; citation_author=A Rambaut; citation_author=B Shapiro; citation_author=OG Pybus; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=35; citation_pages=1185-1192; " />
      <meta name="citation_reference" content="citation_title=The coalescent.; citation_author=JFC Kingman; citation_journal_title=Stoch. Proc. Appl.; citation_volume=13; citation_number=36; citation_pages=235-248; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Sampling theory for neutral alleles in a varying environment.; citation_author=RC Griffiths; citation_author=S Tavarè; citation_journal_title=Philos. Trans. R. Soc. London Ser. B; citation_volume=344; citation_number=37; citation_pages=403-410; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=An integrated framework for the inference of viral population history from reconstructed genealogies.; citation_author=OG Pybus; citation_author=A Rambaut; citation_author=PH Harvey; citation_journal_title=Genetics; citation_volume=155; citation_number=38; citation_pages=1429-1437; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Bayes Factors.; citation_author=RE Kass; citation_author=AE Raftery; citation_journal_title=J Am Stat Ass; citation_volume=90; citation_number=39; citation_pages=773-795; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Bayesian selection of continuous-time Markov chain evolutionary models.; citation_author=MA Suchard; citation_author=RE Weiss; citation_author=JS Sinsheimer; citation_journal_title=Mol Biol Evol; citation_volume=18; citation_number=40; citation_pages=1001-1013; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=HIV-1 phylodynamic analysis in distinct brain compartments provides a model for the neuropathogenesis of AIDS.; citation_author=M Salemi; citation_author=SL Lamers; citation_author=S Yu; citation_author=T de Oliveira; citation_author=WM Fitch; citation_journal_title=J Virol; citation_volume=79; citation_number=41; citation_pages=11343-11352; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania"/>
    <meta name="twitter:description" content="BackgroundHIV-1 epidemic in Western Europe is largely due to subtype B. Little is known about the HIV-1 in Eastern Europe, but a few studies have shown that non-B subtypes are quite common. In Albania, where a recent study estimated a ten-fold increase of AIDS incidence during the last six years, subtype A and B account for 90% of the know infections.Methodology/Principal FindingsWe investigated the demographic history of HIV-1 subtype A and B in Albania by using a statistical framework based on coalescent theory and phylogeography. High-resolution phylogenetic and molecular clock analysis showed a limited introduction to the Balkan country of subtype A during the late 1980s followed by an epidemic outburst in the early 1990s. In contrast, subtype B was apparently introduced multiple times between the mid-1970s and mid-1980s. Both subtypes are growing exponentially, although the HIV-1A epidemic displays a faster growth rate, and a significantly higher basic reproductive number R0. HIV-1A gene flow occurs primarily from the capital Tirane, in the center of the country, to the periphery, while HIV-1B flow is characterized by a balanced exchange between center and periphery. Finally, we calculated that the actual number of infections in Albania is at least two orders of magnitude higher than previously thought.Conclusions/SignificanceOur analysis demonstrates the power of recently developed computational tools to investigate molecular epidemiology of pathogens, and emphasize the complex factors involved in the establishment of HIV-1 epidemics. We suggest that a significant correlation exists between HIV-1 exponential spread and the socio-political changes occurred during the Balkan wars. The fast growth of a relatively new non-B epidemic in the Balkans may have significant consequences for the evolution of HIV-1 epidemiology in neighboring countries in Eastern and Western Europe."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001390.g005"/>

  <meta property="og:title" content="High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6730'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6898'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8745&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001390">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001390" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001390&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marco Salemi
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:salemi@pathology.ufl.edu">salemi@pathology.ufl.edu</a></p>

                <p>Affiliation:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Tulio de Oliveira, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Medical Research Council (MRC) Bioinformatics Unit, South African National Bioinformatics Institute, University of Western Cape, Cape Town, South Africa
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Massimo Ciccozzi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Istituto Superiore di Sanita', Roma, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Giovanni Rezza, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Istituto Superiore di Sanita', Roma, Italy
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maureen M. Goodenow
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Pathology, Immunology, and Laboratory Medicine, University of Florida, Gainesville, Florida, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 02, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001390</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001390-g001" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001390-g002" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001390-g003" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001390-t001" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001390-g004" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001390-g005" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001390">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001390" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>HIV-1 epidemic in Western Europe is largely due to subtype B. Little is known about the HIV-1 in Eastern Europe, but a few studies have shown that non-B subtypes are quite common. In Albania, where a recent study estimated a ten-fold increase of AIDS incidence during the last six years, subtype A and B account for 90% of the know infections.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We investigated the demographic history of HIV-1 subtype A and B in Albania by using a statistical framework based on coalescent theory and phylogeography. High-resolution phylogenetic and molecular clock analysis showed a limited introduction to the Balkan country of subtype A during the late 1980s followed by an epidemic outburst in the early 1990s. In contrast, subtype B was apparently introduced multiple times between the mid-1970s and mid-1980s. Both subtypes are growing exponentially, although the HIV-1A epidemic displays a faster growth rate, and a significantly higher basic reproductive number R<sub>0</sub>. HIV-1A gene flow occurs primarily from the capital Tirane, in the center of the country, to the periphery, while HIV-1B flow is characterized by a balanced exchange between center and periphery. Finally, we calculated that the actual number of infections in Albania is at least two orders of magnitude higher than previously thought.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our analysis demonstrates the power of recently developed computational tools to investigate molecular epidemiology of pathogens, and emphasize the complex factors involved in the establishment of HIV-1 epidemics. We suggest that a significant correlation exists between HIV-1 exponential spread and the socio-political changes occurred during the Balkan wars. The fast growth of a relatively new non-B epidemic in the Balkans may have significant consequences for the evolution of HIV-1 epidemiology in neighboring countries in Eastern and Western Europe.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Salemi M, de Oliveira T, Ciccozzi M, Rezza G, Goodenow MM (2008) High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania. PLoS ONE 3(1):
          e1390.
            doi:10.1371/journal.pone.0001390</p><p><strong>Academic Editor: </strong>Jason Stajich, University of California at Berkeley, United States of America</p><p><strong>Received:</strong> September 11, 2007; <strong>Accepted:</strong> December 7, 2007; <strong>Published:</strong> January 2, 2008</p><p><strong>Copyright:</strong> © 2008 Salemi et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The research was supported by PHS R01 awards HD032259, AI047723, and AI065265; Center for Research in Pediatric Immune Deficiency; Stephany W. Holloway University Chair for AIDS Research (MMG).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Human immunodeficiency virus type 1 (HIV-1) infection continues to spread rapidly throughout the world. According to the UNAIDS/WHO (June 2006) report, an estimated 39.5 million people are living with the virus worldwide. HIV-1 is characterized by high genetic variability, rapid evolution, and diversification <a href="#pone.0001390-Hahn1">[1]</a>, <a href="#pone.0001390-SeillierMoiseiwitsch1">[2]</a>. Indeed, recombination coupled with the elevated error rate of the reverse transcriptase, and the rapid turnover of HIV-1 in infected individuals, are at the origin of the high genetic variability of the virus <a href="#pone.0001390-Peeters1">[3]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The majority of HIV-1 strains cluster within a large group called M (for Main), which is responsible for the pandemic <a href="#pone.0001390-Peeters1">[3]</a>. Group M includes ten phylogenetically distinct subtypes, two sub-subtypes, and several inter-subtype HIV-1 recombinants, known as circulating recombinant forms (CRFs) <a href="#pone.0001390-Robertson1">[4]</a>–<a href="#pone.0001390-Salminen1">[6]</a>. Most subtypes, as well as CRFs, are present in Africa, reflecting the African origin of the epidemic during the 1930s <a href="#pone.0001390-Gao1">[7]</a>–<a href="#pone.0001390-Salemi1">[9]</a>. HIV-1 subtype C is the most prevalent worldwide accounting for more than half of all infections, whereas HIV-1B is the subtype responsible for most of the infections in Western Europe, United States, and Australia <a href="#pone.0001390-Esparza1">[10]</a>. Little is known about the HIV-1 epidemic in Eastern Europe, but a few studies have shown that non-B subtypes are quite common. For example, HIV-1A is highly prevalent in Russia, Belarus, and Ukraine <a href="#pone.0001390-Lukashov1">[11]</a>–<a href="#pone.0001390-Saad1">[13]</a>, as well as former Yugoslavia <a href="#pone.0001390-Stanojevic1">[14]</a>. In Albania, a recent report has shown the presence of both HIV-1A and B subtypes, which account for about 90% of the known infections <a href="#pone.0001390-Ciccozzi1">[15]</a>. Although the registered HIV-1 cases in Albania are still low the social, political, and economical instability of the country has its own risk determinants with regard to sexually transmitted diseases in general and for HIV/AIDS in particular. A detailed characterization of the demographic parameters shaping the Albanian epidemic, including the growth rate in the number of effective infections, the time of introduction of a lineage in a population, and the viral gene flow (migration) to/from neighboring countries, has important implication for the evolution of the HIV-1 epidemic in the European continent as a whole.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Phylogenies reconstructed from randomly sampled viral gene sequences contain valuable and unique information about population-level processes such as change in population size and growth rate, and can be used to understand the course of a viral epidemic over time <a href="#pone.0001390-Pybus1">[16]</a>, <a href="#pone.0001390-Hue1">[17]</a>. Indeed, coalescent theory has successfully been employed to infer the history of a pathogen population and investigate the evolutionary dynamics of different HIV-1 subtypes within a specific geographic region <a href="#pone.0001390-Pybus1">[16]</a>–<a href="#pone.0001390-Salemi2">[20]</a>. To this end, it has been demonstrated that the <em>pol</em> gene contains sufficient evolutionary information to reconstruct demographic histories in spite of the potential bias introduced by the emergence of drug resistance mutations <a href="#pone.0001390-Salemi2">[20]</a>, <a href="#pone.0001390-Hue2">[21]</a>. Moreover, the Albanian data set included only therapy-naïve subjects and no major mutations associated with drug resistance were identified in such a cohort <a href="#pone.0001390-Ciccozzi1">[15]</a>. The coalescent framework also assumes neutral evolution and several studies have shown that the <em>pol</em> gene is under both positive and negative selection <a href="#pone.0001390-deSLeal1">[22]</a>–<a href="#pone.0001390-Rouzine1">[25]</a>. However, even when an HIV-1 gene evolving within a host is under strong selection, the genealogies generated at the epidemiological (inter-individual) level appear to be neutral and do not significantly violate the assumptions of the coalescence <a href="#pone.0001390-Grenfell1">[26]</a>. In fact, even when different HIV-1 genes undergo significantly different selection pressure remarkably similar and reliable demographic estimates are still obtained from coalescent analysis <a href="#pone.0001390-Lemey1">[18]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present study we used high-resolution phylogenetic analysis and the coalescent framework, as well as a phylogeographic approach <a href="#pone.0001390-Slatkin1">[27]</a>, to reconstruct the history and evolution of HIV-1A and HIV-1B epidemics in Albania. Our data not only provide important insights about the extent of the epidemic in this country, but also indicate how, in a time of intense socio-political change and migration trends, the emergent epidemics in the Balkans may have profound implications for the future of HIV-1 epidemiology in Europe and worldwide.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Origin of HIV-1A and HIV-1B Albanian epidemics</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To investigate the origin of HIV-1 subtype A and B in Albania we compiled different data sets of Albanian and reference sequences from different geographic origin (supplemental <a href="#pone.0001390.s001">Table S1</a>). Phylogenetic analysis of HIV-1A and HIV-1B Albania+reference sequences alignments pointed to different origins for the two subtypes in the Balkan country. HIV-1A ML inferred genealogy (<a href="#pone-0001390-g001">Figure 1A</a>) clustered all Albanian strains but one within a monophyletic clade highly supported both by boostrapping (&gt;70%) and zero branch length test (<em>p</em>&lt;0.001). The Albanian clade emerged as a subcluster of a monophyletic group joining sequences from Greece and also including one Albanian strain (the ML genealogy displaying full sequence names is given in supplemental <a href="#pone.0001390.s003">Figure S1A</a>). The finding that 99% of the HIV-1A Albanian sequences could be traced back to a unique most recent common ancestor (MRCA) suggested a single major introduction of HIV-1A from Greece followed by local epidemic spread. The presence of one Greek sequence within the Albanian clade and viceversa (see <a href="#pone-0001390-g001">Figure 1A</a>) could also indicate a limited ongoing viral gene flow between the two neighboring countries. HIV-1B ML inferred genealogy (<a href="#pone-0001390-g001">Figure 1B</a>) showed Albanian sequences highly scattered among strains collected from several other European countries. Three well-supported Albanian clades (bootstrap &gt;75% and the zero branch length test <em>p</em>&lt;0.001) are visible in the tree. Each clade belonged to a separate phylogenetic lineage supporting a scenario of multiple independent introductions to Albania of subtype B from different geographic areas (the ML genealogy displaying full sequence names is given in supplemental <a href="#pone.0001390.s003">Figure S1B</a>). The existence of several independent Albanian clades was also an indication of separate transmission networks, originating from different introductions of the virus at different times, within the country.</p>
<div class="figure" id="pone-0001390-g001"><div class="img"><a name="pone-0001390-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Maximum likelihood phylogenetic analysis of HIV-1A pol sequences.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Branch lengths were estimated with the best fitting nucleotide substitution model according to a hierarchical likelihood ratio test <a href="#pone.0001390-Swofford1">[29]</a>, and were drawn in scale with the bar at the bottom indicating 0.1 nucleotide substitutions per site. One * along a branch represents significant statistical support for the clade subtending that branch (p&lt;0.001 in the zero-branch-length test and bootstrap support &gt;75%). The color of each tip branch represents the country (or geographic region) of origin of sequence corresponding to that branch, according to the legend in the figure. A. Phylogenetic tree of 31 HIV-1A strains from Bulgaria and 122 subtype A reference sequences downloaded from the Los Alamos HIV database. The tree was rooted using two HIV-1B strains as outgroup. The arrow indicates the most recent common ancestor (MRCA) of the Albanian monophyletic clade. B. Phylogenetic tree of 21 HIV-1B strains from Bulgaria and 46 subtype B reference sequences downloaded from the Los Alamos HIV database. The tree was rooted using two HIV-1A strains as outgroup. Solid boxes highlight statistically supported clades of Albanian sequences.</p>
<span>doi:10.1371/journal.pone.0001390.g001</span></div>

<h4>Likelihood mapping analysis</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The phylogenetic structure of the Albanian epidemics was also investigated by likelihood mapping analysis. The evaluation of 10,000 random quartets with the likelihood-mapping method showed a significant difference between HIV-1A and HIV-1B epidemic. More than 60% of the randomly chosen quartets from the HIV-1A alignment of Albanian strains were distributed in the center of the likelihood map, indicating a strong signal for star-like phylogeny (<a href="#pone-0001390-g002">Figure 2A</a>, left panel). In contrast, only 23.1% of quartets from the HIV-1B Albanian alignment fell in the center of the map with most of the quartets equally distributed in the three corners, which represent tree-like signal (<a href="#pone-0001390-g002">Figure 2A</a>, right panel). On the other hand, HIV-1A and HIV-1B maps in the rest of Europe (after excluding the Albanian sequences) exhibited very similar structure, although subtype A still showed a larger star-like signal with respect to subtype B (<a href="#pone-0001390-g002">Figure 2B</a>). The detailed distribution of dots in each region of the likelihood mappings is given in supplemental <a href="#pone.0001390.s004">Figure S2</a>.</p>
<div class="figure" id="pone-0001390-g002"><div class="img"><a name="pone-0001390-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Likelihood mapping of HIV-1A and B pol sequences.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Each dot represents the likelihoods of the three possible unrooted trees for a set of four sequences (quartets) selected randomly from the data set (see <a href="#s4">Materials and Methods</a>): dots close to the corners or the sides represent, respectively, tree-like, or network-like phylogenetic signal in the data. The central area of the likelihood map, highlighted by a smaller blue triangle inside the map, represents star-like signal. The percentage of dots in the central area is given at the basis of each map. A. Likelihood mapping of 10,000 random quartets of HIV-1A (left) and HIV-1B (right) Albanian sequences. B. Likelihood mapping of 10,000 random quartets of HIV-1A (left) and HIV-1B (right) Albanian+reference sequences downloaded from the Los Alamos HIV databases.</p>
<span>doi:10.1371/journal.pone.0001390.g002</span></div>

<h4>Demographic history of HIV-1 Albanian A and B subtypes</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>HIV-1A and HIV-1B Albanian sequences were analyzed separately to infer the demographic history of the two subtypes within the Balkan country. We calculated approximate marginal likelihoods of six different demographic models (supplemental <a href="#pone.0001390.s002">Table S2</a>): four parametric models (constant population size with strict or relaxed molecular clock, exponential growth with strict or relaxed molecular clock), and two non-parametric models (Bayesian skyline plot, BSP, with strict or relaxed molecular clock). For both data sets, the Bayes Factor (BF) favored, in general, models enforcing a relaxed molecular clock over strict clock models (<a href="#pone-0001390-t001">Table 1</a>). Models assuming exponential population growth performed always better than models assuming constant population size. In contrast, the BF was not significant when exponential models were compared with BSPs (<a href="#pone-0001390-t001">Table 1</a>) suggesting that both the parametric and the non-parametric models fit the data equally well. The BSP of HIV-1A dated the origin of the MRCA of the Albanian epidemic in the late 1980s (<a href="#pone-0001390-g003">Figure 3</a>, bottom panel) with an initial median estimate of 4 effective infections (95% high posterior density interval, 95% HPD, 0.4 to 22). The final median estimate in the year 2003 (the latest sampling date available) resulted of about 757 effective infections (95%HPD = 101–6,000). The reconstruction of HIV-1B demographic history estimated the mid-1970s as the origin of the MRCA (<a href="#pone-0001390-g003">Figure 3</a>, top panel). However, It is important to keep in mind that because HIV-1B Albanian strains were scattered among other European sequences, such a date indicated rather the origin of the MRCA of HIV-1 in Europe than in Albania where, in fact, multiple independent introductions over time were postulated. The origin of the Albanian transmission clades within the HIV-1B tree (<a href="#pone-0001390-g002">Figure 2</a>) was traced back between 1983 and 1986 (<a href="#pone-0001390-g003">Figure 3</a>, top panel). The median estimate of <em>Ne</em> in the year 2002 (the latest sampling date available) resulted of about 243 effective infections (95%HPD = 36–1,919), about one third of the estimate for HIV-1A and in substantial agreement with epidemiological data showing HIV-1A as the prevalent infection <a href="#pone.0001390-Ciccozzi1">[15]</a>. Moreover, comparison of HIV-1A and HIV-1B demographic histories showed that subtype B epidemic expanded initially at a much slower than the HIV-1A epidemic. Both parametric and non-parametric subtype B demographic curves looked approximately linear during the first decade of the epidemic, and experienced a sharper increase only within the next fifteen years during the same time-frame of the introduction and exponential growth of subtype A epidemic (<a href="#pone-0001390-g003">Figure 3</a>). In fact, by 1996, nine years after the first introduction in Albania of HIV-1A, the effective number of subtype A infections was already larger than the one due to HIV-1B.</p>
<div class="figure" id="pone-0001390-g003"><div class="img"><a name="pone-0001390-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Bayesian skyline plots of HIV-1 subtypes in Albania.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Non-parametric curves of HIV-1 effective population size (effective number of infections, Ne) over time in Albania were estimated from pol gene sequences employing a Bayesian framework. Genetic distances were transformed into a timescale of years taking into account the different sampling times of the viral strains and by enforcing a relaxed molecular clock model (see <a href="#s4">Materials and Methods</a>). Solid lines indicate median, and 95% upper and lower high posterior density (HPD) estimates of Ne, according to the color legend to the right; dotted blue lines are parametric estimate of Ne(t) according to the exponential demographic model. Broken lines between the top and bottom panels highlight the time window common to the two epidemics. Top panel. HIV-1 subtype B. Each arrow indicates the estimated origin of the corresponding Albanian clade (indicated by the roman numeral) in the tree in <a href="#pone-0001390-g001">Figure 1B</a>. The first arrow from the left indicates the time of the MRCA. 95% HPD intervals are given in parenthesis below each estimate. Bottom panel. HIV-1 subtype A. The arrow indicates the estimated origin of the MRCA of the Albanian clade shown by the tree in <a href="#pone-0001390-g001">Figure 1A</a>. 95% HPD intervals are given in parenthesis below each estimate.</p>
<span>doi:10.1371/journal.pone.0001390.g003</span></div><div class="figure" id="pone-0001390-t001"><div class="img"><a name="pone-0001390-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001390.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001390.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Bayes Factors between different demographic models of HIV-1A and B in Albania.</span></strong></p><span>doi:10.1371/journal.pone.0001390.t001</span></div>

<h4>Grow rate estimates of HIV-1 subtype A and B Albanian epidemics</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Under the exponential model, the effective number of infections <em>Ne</em> grows exponentially at rate <em>r</em>. It has to be noted that <em>Ne</em> represents the number of infections actually contributing to new infections, rather than the total number of infected individuals. Such observation is reflected by the estimates of the growth rates that resulted 0.69 years<sup>−1</sup> (95%HPD = 0.47–0.905) for HIV-1A, but only 0.136 years<sup>−1</sup> (95%HPD = 0.064–0.205) for HIV-1B. Basic reproductive numbers (R<sub>0</sub>) of HIV-1A and HIV-1B in Albania, obtained from the parametric estimates of the growth rate <em>r</em>, was also significantly different (<em>p</em>&lt;0.01, paired t-test) for the two subtypes (<a href="#pone-0001390-g004">Figure 4</a>). Assuming the virus is transmitted at the same rate during the total length of infection, a short term progressor would transmit the infection 1 to 2 times if infected by HIV-1B, but 2 to 4 times if infected with HIV-1A. An HIV-1A long term progressor would transmit the infection 6 to 8 times compared to the 1 to 3 times of an HIV-1B long term progressor.</p>
<div class="figure" id="pone-0001390-g004"><div class="img"><a name="pone-0001390-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Estimates of HIV-1A and HIV-1B basic reproductive number (R0) in Albania.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>R0 estimates for different average duration of infection (D) were inferred from the Bayesian estimates of population growth rate (see <a href="#s4">materials and Methods</a>). A Black bar indicates the 95% high posterior density intervals of the estimate. An * on top of the bar indicate a statistically significant difference (paired t-test, p&lt;0.01) between R0 estimates for two different subtypes.</p>
<span>doi:10.1371/journal.pone.0001390.g004</span></div>

<h4>Phylogeography of HIV-1 Albanian subtypes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The gene flow (migration) of HIV-1 subtype A and B among different geographic areas and major cities in Albania was investigated with a modified version of the Slatkin and Maddison method <a href="#pone.0001390-Slatkin1">[27]</a>. After superimposing the city of origin of the Albanian sequences on the tip branches of the ML genealogy, we inferred the city of origin of each ancestral node (i.e. ancestral sequence) in the tree using the maximum parsimony algorithm (supplemental <a href="#pone.0001390.s005">Figure S3</a>). The general migration trend between Tirane, the capital roughly at the center of the country, and the major cities at North, South, and West of Tirane was then estimated as observed migration in the genealogy (<a href="#pone-0001390-g005">Figure 5A</a>). The null hypothesis of panmixia (i.e. no population subdivision or complete intermixing of sequences from different geographic areas) was rejected by the randomization test <a href="#pone.0001390-Slatkin1">[27]</a> for subtype A (<em>p</em>&lt;0.001), but not for subtype B (<em>p</em> &gt;0.05). Subtype A gene flow appeared highly asymmetrical with the virual infections mainly expanding from the center to the periphery (<a href="#pone-0001390-g005">Figure 5A</a> top panel). In contrast, HIV-1B flow appeared symmetrical with no preferred migration route (<a href="#pone-0001390-g005">Figure 5A</a> bottom panel). Tirane appeared to be the epicenter of HIV-1A epidemic, exporting the infections toward most of the major cities in the country (<a href="#pone-0001390-g005">Figure 5B</a>).</p>
<div class="figure" id="pone-0001390-g005"><div class="img"><a name="pone-0001390-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001390" data-uri="info:doi/10.1371/journal.pone.0001390.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001390.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001390.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001390.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Phylogeographic mapping of HIV-1A and B epidemic in Albania.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>A. The bubblegrams show the frequency of gene flow (migrations) in Albania to/from different geographic areas and the country capital Tirane. The surface of each circle is proportional to the percentage of observed migrations in the ML genealogy (supplemental <a href="#pone.0001390.s005">Figure S3</a>). Migrations were inferred with a modified version of the Slatkin and Maddison algorithm <a href="#pone.0001390-Slatkin1">[27]</a>, <a href="#pone.0001390-Salemi3">[41]</a> for the HIV-1A (top panel) and HIV-1B (bottom panel) subtype from the maximum likelihood inferred genealogies given in <a href="#pone-0001390-g001">Figure 1</a> and <a href="#pone-0001390-g002">2</a>, respectively. B. Detailed mapping of HIV-1A (indicated by the red arrows) and HIV-1B (indicated by the blue arrows) gene flow among different Albanian geographic areas.</p>
<span>doi:10.1371/journal.pone.0001390.g005</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The application of high-resolution phylogenetic methods and the coalescence as tools to study the evolutionary dynamics of pathogens both within and among hosts has been undergoing a dramatic development within the recent years <a href="#pone.0001390-Grenfell1">[26]</a>. The present study is an example of the detailed epidemiological and evolutionary information that can successfully be extracted from sampled gene sequence data. By employing both a coalescent framework and a phylogeographic approach we were able to investigate the origin and demographic history of HIV-1A and HIV-1B epidemics in Albania to a significant depth. A recent survey of different subtypes circulating in the Balkan country has shown that HIV-1A and B were responsible for about 90% of the known infections <a href="#pone.0001390-Ciccozzi1">[15]</a>. The study pointed out that subtype A prevalence increased over time and became predominant even if, surprisingly, in neighboring countries hosting a large number of Albanian immigrants, such as Italy and Greece, subtype B is still the predominant epidemic. Our data demonstrated that while HIV-1B epidemic in Albania has been characterized by multiple introductions throughout the 1980s and a relatively slow growth, HIV-1A has been spreading, right after its limited introduction in the late 1980s, at an explosive rate. Demographic curves and likelihood mappings were both consistent with a star-like outburst of subtype A during a relatively short time interval. In contrast, subtype B epidemic appeared more structured and characterized by discrete transmission clusters evolving as independent phylogenetic lineages over a longer period of time. The reconstruction of HIV-1 demographic history in the Balkan country coupled with phylogeographic analysis pointed out how complex socio-political and economic changes can dramatically influence the epidemiology of a newly introduced pathogen in a population. The Wars in the Balkans resulted in significant loss of human life, infrastructure damage and the severe disruption of social and health services, especially those supporting women and children. In 1995 over half of Albanian population were refugees or displaced. The war created conditions favorable to HIV/AIDS transmission. The breakdown in the health and public services together with increased risk behaviors including substance abuse, commercial sex work, and sexual behavior in young people are enabling factors for increased HIV/AIDS transmission. An inadequate public health and public awareness programs could have significantly contributed to the spread of HIV-1 among risk groups. The early 1990s were the years when, according to our estimates, both subtype A and B began to spread fast with a considerable gene flow from the capital Tirane, the most developed city in the center of the country, to the periphery. However, the estimated growth rate of subtype A resulted five times larger than subtype B. Such finding does not imply anything about increased pathogenicity of subtype A that, although it cannot be excluded, has not been substantiated so far by any other data. The expansion of both epidemics can be traced back to the dramatic socio-political changes occurred in Albania during the early 1990s. Their subsequent different growth, with HIV-1A becoming quickly the predominant infection, may be due to their introduction in different susceptible populations living in urban or rural areas. A similar explanation has been proposed for the faster growth rate of HIV-1 subtype C with respect to subtype B in Southern Brazil <a href="#pone.0001390-Salemi2">[20]</a>. It is important to notice that although R<sub>0</sub> estimates for subtype A and B are in agreement with this scenario, they should be taken with caution, since they rely on the assumption that the virus is transmitted at the same rate during the total length of infection. Such an assumption seems to be reasonable for pathogens like HCV <a href="#pone.0001390-Pybus1">[16]</a>, but it is less clear whether it could be valid for HIV-1 as well.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The only epidemiological survey in Albania conducted so far at the national level, of which we have used the molecular data analyzed in the present work, collected 72 plasma samples representing about 50% of the country's known individuals with HIV/AIDS <a href="#pone.0001390-Ciccozzi1">[15]</a>. The study estimated an increase in the incidence of AIDS from 0.3 cases/million population in 1996 to 3.3 cases/million population in 2002. Although a ten-fold increase is in agreement with our overall finding, the parametric estimates of HIV-1A and B demographic histories suggested a much larger number of effective infections. Moreover, the number of effective infections (i.e. those one contributing to onward transmission) is always significantly lower than the total number of infected individuals. As an example, the number of effective infections in 1995 U.S.A reached 5,000, while the prevalence was actually 200,000 <a href="#pone.0001390-Hue1">[17]</a>. If a similar relationship held for the Albanian epidemics, our estimates would indicate the presence of about 30,000 people infected by subtype A and about 10,000 by subtype B in a country of about 3.6 millions people. Even if these numbers may be overestimates due to a bias in the coalescent models used, it has to be considered that to date national surveillance mechanisms and reporting systems in Albania are still underdeveloped resulting in a lack of adequate epidemiological information and behavioral data. Our analysis suggested that the extent of HIV-1 epidemic in Albania is much larger than previously thought with HIV-1A infections growing at an alarming rate. Through phylogeographic analysis we have shown that HIV-1A gene flow has been occurring form the capital Tirane to the cities of Vlore and Durres, the two major Albanian ports on the Adriatic sea close to Southern Italy, Shkoder, Northern city close to Montenegro, and Sarande, Southern city close to the Greek border. Such a trend, coupled with the significant Albanian emigration during the last recent years due to high rate of unemployment has the potential to introduce and spread a new wave of HIV-1A epidemic into Southern Europe significantly affecting, on the long run, HIV-1 epidemiology in Europe as well as other Western countries.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>In conclusion, we showed how the coalescent and phylogeographic framework could be employed to investigate in detail the demographic history and the complex factors underlying the spread of pathogens in a population. Our findings highlight the dramatic influence that social, political, and economical factors can have in determining the outcome of a pathogen newly introduced in a susceptible population, and call for considerable improvements in prevention campaigns and monitoring of the HIV-1 infection in the Balkans.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Data sets</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Two main data sets of previously published HIV-1 <em>pol</em> sequences (protease+RT) from Albania were compiled including, respectively, 31 HIV-1A strains collected between 1997 and 2003, and 21 HIV-1B strains collected between 1995 and 2002 <a href="#pone.0001390-Ciccozzi1">[15]</a>. Two additional data sets were obtained using subtype A and B reference sequences downloaded from the Los Alamos HIV databases (<a href="http://hiv-web.lanl.gov/">http://hiv-web.lanl.gov/</a>). Full details on the alignments compiled for the present study are given in supplemental <a href="#pone.0001390.s001">Table S1</a>. To assemble data sets suitable for demographic analysis (see below), reference strains from epidemiologically unlinked individuals were chosen according to the following criteria: 1. known country were the infection was acquired; 2. known year of sampling, and 3. sequences from therapy naïve subjects and/or without known mutations associated with drug-resistance. Sequences were aligned with the Clustal algorithm <a href="#pone.0001390-Thompson1">[28]</a> followed by manual editing. Positions containing gaps were removed from the final alignment. Alignments are available from the authors upon request.</p>


<h4>Phylogenetic analysis</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For each data set, the best fitting nucleotide substitution model was tested with a hierarchical likelihood ratio test following the strategy described by Swofford and Sullivan (2003) <a href="#pone.0001390-Swofford1">[29]</a>, using a neighbor-joining (NJ) base-tree with LogDet corrected distances. Maximum likelihood (ML) trees were then inferred with the selected model and ML-estimated substitution parameters. Model and model parameters for each data set are given in supplemental <a href="#pone.0001390.s001">Table S1</a>. The heuristic search for the ML tree was performed using an NJ tree as starting tree and the TBR branch-swapping algorithm. NJ trees were also estimated using pair-wise distances inferred by ML with the best fitting nucleotide substitution model. Calculations were performed with PAUP* 4.0b10 written by David L. Swofford. Statistical support for internal branches in the NJ trees was obtained by bootstrapping (1000 replicates) and with the ML-based zero branch length test for the ML trees <a href="#pone.0001390-Swofford1">[29]</a>. Trees were rooted by ML rooting by selecting the rooted tree with the best likelihood under the molecular clock constraint (taking into account the different sampling times of the <em>taxa</em>) <a href="#pone.0001390-Rambaut1">[30]</a>, <a href="#pone.0001390-Rambaut2">[31]</a>, or by outgroup rooting using two Albanian HIV-1A strains as outgroup for the HIV-1B data set, and two HIV-1B Albanian strains for the HIV-1A data sets. Both methods inferred the same root for each data set.</p>


<h4>Likelihood mapping</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The phylogenetic signal in a data set of aligned DNA or amino acid sequences can be investigated with the likelihood mapping method by analyzing groups of four sequences, randomly chosen, called quartets <a href="#pone.0001390-Strimmer1">[32]</a>. For a quartet, just three unrooted tree topologies are possible. The likelihood of each topology is estimated with the maximum likelihood method and the three likelihoods are reported as a dot in an equilateral triangle (the likelihood map). Three main areas in the map can be distinguished as shown in supplemental <a href="#pone.0001390.s004">Figure S2</a> <a href="#pone.0001390-Strimmer1">[32]</a>: the three corners representing fully resolved tree topologies, i.e. the presence of tree-like phylogenetic signal in the data; the center, which represents star-like phylogeny, and the three areas on the sides indicating network-like phylogeny, i.e. presence of recombination or conflicting phylogenetic signals. For N sequences <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001390.e001&amp;representation=PNG" class="inline-graphic"></span> possible quartets exist and that the distribution (percentage) of dots within each area gives an idea about the mode of evolution in the data set under investigation. Extensive simulation studies have shown that &gt;33% dots falling within the central area indicate substantial star-like signal, i.e. a star-like outburst of multiple phylogenetic lineages <a href="#pone.0001390-Strimmer1">[32]</a>. Likelihood mapping analyses were performed with the program TREE-PUZZLE <a href="#pone.0001390-Schmidt1">[33]</a> for each data set by analyzing 10,000 random quartets.</p>


<h4>Evolutionary rates of HIV-1A and HIV-1B subtypes</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>For HIV-1B, a previous estimate of the evolutionary rate (3.55 10<sup>−3</sup> nucleotide substitutions/site year) from a data set of 106 sequences sampled between 1983 and 2000 was available <a href="#pone.0001390-Hue1">[17]</a>. For HIV-1A, the evolutionary rate was inferred from the data (1.4 10<sup>−3</sup> nucleotide substitutions/site year, 95% high posterior density interval 0.28–2.6 10<sup>−3</sup>) since sequences were sampled at different time points and a molecular clock with non-contemporaneous tips could be calibrated <a href="#pone.0001390-Rambaut2">[31]</a>, as done by Hue and coll. for HIV-1B <a href="#pone.0001390-Hue1">[17]</a>. To this end, the evolutionary rate was estimated by employing the Bayesian Markov Chains Monte Carlo (MCMC) clock method <a href="#pone.0001390-Drummond1">[34]</a> implemented in the BEAST software package (<a href="http://evolve.zoo.ox.ac.uk/beast/">http://evolve.zoo.ox.ac.uk/beast/</a>). As a first approximation, we estimated the evolutionary rate using the full data set, including Albanian as well as reference sequences, with a strict Bayesian MCMC clock <a href="#pone.0001390-Drummond1">[34]</a>. For all the subsequent calculations implementing various demographic models (see below) the subtype specific rate was employed as mean fixed rate for models enforcing a strict molecular clock, and as mean fixed rate with exponentially distributed evolutionary rates as prior for models enforcing a relaxed molecular clock <a href="#pone.0001390-Drummond1">[34]</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>The Bayesian calculation consisted of three independent 10,000,000 generations MCMC with sampling every 1000<sup>th</sup> generation. The three independent runs were combined with LogCombiner version 1.4. Convergence of the MCMC was assessed by calculating the effective sampling size (ESS) of the runs <a href="#pone.0001390-Drummond2">[35]</a>. All parameter estimates showed significant ESS (&gt;250).</p>


<h4>Coalescent models</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>By using coalescent theory, we can infer the demographic history of a population from the genealogical relationships of sampled individuals <a href="#pone.0001390-Kingman1">[36]</a>, <a href="#pone.0001390-Griffiths1">[37]</a>. A genealogy reconstructed from randomly sampled HIV sequences contains information about population-level processes such as change in population size, and growth rate <a href="#pone.0001390-Pybus2">[38]</a>. Given a viral phylogeny <em>P</em> and a vector φ representing the parameters of a demographic model <em>N</em>(<em>t</em>), it is possible to calculate the log of the conditional probability <em>ln</em>[φ|<em>P</em>]. Bayesian estimates of φ can be found by MCMC sampling procedure <a href="#pone.0001390-Pybus2">[38]</a>. We considered six demographic models for the HIV-1A and HIV-1B epidemic in Albania: constant population size with strict or relaxed molecular clock, exponential growth with strict or relaxed molecular clock, and Bayesian skyline plot (BSP) with strict or relaxed molecular clock. Both parametric (constant or exponential model) and non-parametric (BSP) estimates of demographic history were performed with BEAST version 1.4.1 by running one MCMC for 100,000,000 generations with sampling every 10,000<sup>th</sup> generation. Each aligned data set was partitioned in 1<sup>st</sup>+2<sup>nd</sup> and 3<sup>rd</sup> codon positions and the parameters of the nucleotide substitution (HKY+Γ+I) and demographic model were estimated independently for ach partition. All parameter estimates showed significant ESS (&gt;300).</p>


<h4>Bayesian selection of coalescent models</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>While the constant and the exponential model are nested, the BSP is a non-parametric model that cannot be compared with the other two by comparing the mean log posterior probabilities. Model comparison in a Bayesian framework can be achieved, however, by calculating the Bayes Factor (BF), which is the ratio of the marginal likelihoods (marginal with respect to the prior) of the two models being compared <a href="#pone.0001390-Kass1">[39]</a>. We calculated approximate marginal likelihoods for each coalescent model via importance sampling using the harmonic mean of the sampled likelihoods (with the posterior as the importance distribution) <a href="#pone.0001390-Suchard1">[40]</a>. The difference (in log<sub>e</sub> space) of marginal likelihood between any two models is the log<sub>e</sub> of the Bayes Factor, log<sub>e</sub>(BF). Evidence against the null model (i.e. the one with lower marginal likelihood) is assessed in the following way <a href="#pone.0001390-Kass1">[39]</a>: 2&gt;2· log<sub>e</sub>(BF)&gt;6 indicates positive evidence against the null model; 6&gt;2·log<sub>e</sub>(BF)&gt;10 indicates strong evidence against the null model; 2·log<sub>e</sub>(BF)&gt;10 indicates very strong evidence against the null model. The calculations were performed with BEAST 1.5 alpha (<a href="http://code.google.com/p/beast-mcmc/">http://code.google.com/p/beast-mcmc/</a>) according to the instructions on the BEAST website (<a href="http://beast.bio.ed.ac.uk/Model_comparison">http://beast.bio.ed.ac.uk/Model_comparis​on</a>).</p>


<h4>Grow rate of HIV-1 subtype A and B epidemic</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Since the exponential model was the one with the highest marginal likelihood for both HIV-1A and B Albanian data sets (see supplemental <a href="#pone.0001390.s002">Table S2</a>), we could use the population growth rate, <em>r</em>, which is one of the two free parameters (the other is the number of effective infections <em>Ne</em>) of the model to infer the epidemiological quantity R<sub>0</sub>. R<sub>0</sub> is the basic reproductive number (infectivity) of a pathogen, i.e. the average number of secondary infections caused by each primary infected individual. In a viral population exponentially growing at rate <em>r</em>, with <em>D</em> as the average duration of infectiousness, it can be shown that if the virus is transmitted at the same rate during the total length of infection, then R<sub>0</sub> = <em>rD</em>+1<a href="#pone.0001390-Pybus1">[16]</a>. The limitations of such an assumption for HIV-1 are discussed in the Discussion section.</p>


<h4>Phylogeographic analysis</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The hypothesis of restricted gene flow among distinct HIV-1 populations within different geographic regions and cities in Albania was tested with a modified version of the Slatkin and Maddison test <a href="#pone.0001390-Slatkin1">[27]</a> that allows to count migration to/from different viral subpopulations <a href="#pone.0001390-Salemi3">[41]</a>, using the MacClade version 4 program (Sinauer Associates, Sunderland, MA), as described below. A one-character data matrix is obtained from the original data set by assigning to each <em>taxon</em> in the tree a one-letter code indicating its country (or geographic region) of origin. Then, the putative origin of each ancestral sequence (i.e. internal node) in the tree is inferred by finding the most parsimonious reconstruction (MPR) of the ancestral character (supplemental <a href="#pone.0001390.s005">Figure S3</a>). The final tree-length, i.e. the number of observed migrations in the genealogy, can easily be computed and compared to the tree-length distribution of 10,000 trees obtained by random joining-splitting. Observed genealogies significantly shorter than random trees indicate the presence of subdivided populations with restricted gene flow <a href="#pone.0001390-Slatkin1">[27]</a>. Specific migrations among different countries (character states) were traced with the <em>State changes and stasis</em> tool (MacClade software), which counts the number of changes in a tree for each pair-wise character state. When multiple MPRs were present (as in our data sets), the algorithm calculated the average migration count over all possible MPRs for each pair. The resulting pair-wise migration matrix was then normalized, and a randomization test with 10,000 matrices obtained from 10,000 random trees (by random joining-splitting of the original tree) was performed to assess the statistical significance of the observed migration counts.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001390"></div><a name="pone.0001390.s001" id="pone.0001390.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001390.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">HIV-1 pol gene data sets assembled for the present study.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001390.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB DOC)</p>
<a name="pone.0001390.s002" id="pone.0001390.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001390.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Marginal Likelihoods for different epidemiological models of HIV-1A and B in Albania.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001390.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001390.s003" id="pone.0001390.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001390.s003">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Maximum likelihood phylogenetic analysis of HIV-1 A and B pol sequences. The maximum likelihood trees display the full names of the HIV-1 sequences used in the analysis. Two digits at the beginning of the name indicate the sampling year, and the next two characters the country of origin according to the guidelines at the Los Alamos HIV databases. Albanian sequences are indicated as “Alb”. A. The tree is the same as the one given in <a href="#pone-0001390-g001">Figure 1A</a> for HIV-1A pol sequences. No Sampling year was available for the sequences from Greece and for one Yugoslavian strain. B. The tree is the same as the one given in <a href="#pone-0001390-g001">Figure 1B</a> for HIV-1B pol sequences.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001390.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.27 MB PPT)</p>
<a name="pone.0001390.s004" id="pone.0001390.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001390.s004">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Detailed distribution of likelihood mapping of HIV-1A and B pol sequences. The likelihood mappings are the same as the ones reported in <a href="#pone-0001390-g002">Figure 2</a>. The detailed distribution of dots in each region of the map is given as percentage values. Seven main regions can be distinguished: the three corners representing tree-like signal; the three sides representing network-like signal, and the center representing star-like signal. A. Likelihood mapping of 10,000 random quartets of HIV-1A (left) and HIV-1B (right) Albanian sequences. B. Likelihood mapping of 10,000 random quartets of HIV-1A (left) and HIV-1B (right) Albanian+reference sequences downloaded from the Los Alamos HIV databases.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001390.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.10 MB PPT)</p>
<a name="pone.0001390.s005" id="pone.0001390.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001390.s005">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Maximum parsimony gene-flow analysis of HIV-1 subtype A and B in Albania. Rooted trees were estimated by maximum likelihood with the best fitting nucleotide substitution model and a molecular clock taking into account different sampling dates (see <a href="#s4">Materials and Methods</a>). Edges are drawn not proportional to genetic distances. Each color represents the geographic location of a city as north, south or west of Tirane (the capital, at the center), according to the color legend to the right. The color of a tip branch indicates from which city the actual sequence was sampled. The color of an internal branch is the city origin of the ancestral sequences at the top of that branch, as inferred by the maximum parsimonious reconstruction of ancestral states. A. HIV-1A pol sequences from Albania. B. HIV-1B pol sequences from Albania.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001390.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.05 MB PPT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are thankful to Andrew Rambaut for helpful suggestions about the Bayesian analyses and critical reading of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS. Performed the experiments: MS. Analyzed the data: MS MG Td. Contributed reagents/materials/analysis tools: MC GR. Wrote the paper: MS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001390-Hahn1" id="pone.0001390-Hahn1"></a>Hahn BH, Shaw GM, Arya SK, Popovic M, Gallo RC, et al.  (1984) Molecular cloning and characterization of the HTLV-III virus associated with AIDS. Nature  312: 166–169.  <ul class="find" data-citedArticleID="1037390" data-doi="10.1038/312166a0"><li><a href="http://dx.doi.org/10.1038/312166a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+and+characterization+of+the+HTLV-III+virus+associated+with+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+and+characterization+of+the+HTLV-III+virus+associated+with+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001390-SeillierMoiseiwitsch1" id="pone.0001390-SeillierMoiseiwitsch1"></a>Seillier-Moiseiwitsch F, Margolin BH, Swanstrom R (1994) Genetic variability of the human immunodeficiency virus: statistical and biological issues. Annu. Rev. Genet.  28: 559–96.  <ul class="find" data-citedArticleID="1037440" data-doi="10.1146/annurev.ge.28.120194.003015"><li><a href="http://dx.doi.org/10.1146/annurev.ge.28.120194.003015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+variability+of+the+human+immunodeficiency+virus%3A+statistical+and+biological+issues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+variability+of+the+human+immunodeficiency+virus%3A+statistical+and+biological+issues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001390-Peeters1" id="pone.0001390-Peeters1"></a>Peeters M, Sharp PM (2000) Genetic diversity of HIV-1: the moving target. AIDS  14: Suppl. 13S129–S140.  <ul class="find" data-citedArticleID="1037410"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genetic+diversity+of+HIV-1%3A+the+moving+target.&amp;auth=&amp;atitle=Genetic+diversity+of+HIV-1%3A+the+moving+target." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+diversity+of+HIV-1%3A+the+moving+target." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+diversity+of+HIV-1%3A+the+moving+target.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001390-Robertson1" id="pone.0001390-Robertson1"></a>Robertson DL, Anderson JP, Bradac JA, Carr JK, Foley B, et al.  (2000) HIV-1 nomenclature proposal. Science  288: 55–56.  <ul class="find" data-citedArticleID="1037424" data-doi="10.1126/science.288.5463.55d"><li><a href="http://dx.doi.org/10.1126/science.288.5463.55d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+nomenclature+proposal." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+nomenclature+proposal.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001390-McCutchan1" id="pone.0001390-McCutchan1"></a>McCutchan FE (2000) Understanding the genetic diversity of HIV-1. AIDS  14: S31–S44.  <ul class="find" data-citedArticleID="1037408"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Understanding+the+genetic+diversity+of+HIV-1.&amp;auth=&amp;atitle=Understanding+the+genetic+diversity+of+HIV-1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+the+genetic+diversity+of+HIV-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+the+genetic+diversity+of+HIV-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001390-Salminen1" id="pone.0001390-Salminen1"></a>Salminen MO, Carr JK, Burke DS, McCutchan FE (1995) Identification of breakpoints in intergenotypic recombinants of HIV type 1 by bootscanning. AIDS Res. Hum. Retrovir.  11: 1423–1425.  <ul class="find" data-citedArticleID="1037436" data-doi="10.1089/aid.1995.11.1423"><li><a href="http://dx.doi.org/10.1089/aid.1995.11.1423" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+breakpoints+in+intergenotypic+recombinants+of+HIV+type+1+by+bootscanning." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+breakpoints+in+intergenotypic+recombinants+of+HIV+type+1+by+bootscanning.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001390-Gao1" id="pone.0001390-Gao1"></a>Gao F, Bailes E, Robertson DL, Chen Y, Rodenburg CM, et al.  (1999) Origin of HIV-1 in the chimpanzee Pan troglodytes troglodytes. Nature.  397: 436–441.  <ul class="find" data-citedArticleID="1037384"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Origin+of+HIV-1+in+the+chimpanzee+Pan+troglodytes+troglodytes.&amp;auth=&amp;atitle=Origin+of+HIV-1+in+the+chimpanzee+Pan+troglodytes+troglodytes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Origin+of+HIV-1+in+the+chimpanzee+Pan+troglodytes+troglodytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Origin+of+HIV-1+in+the+chimpanzee+Pan+troglodytes+troglodytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001390-Korber1" id="pone.0001390-Korber1"></a>Korber B, Muldoon M, Theiler J, Gao F, Gupta R, et al.  (2000) Timing the ancestor of the HIV-1 pandemic strains. Science  288: 1789–1796.  <ul class="find" data-citedArticleID="1037400" data-doi="10.1126/science.288.5472.1789"><li><a href="http://dx.doi.org/10.1126/science.288.5472.1789" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Timing+the+ancestor+of+the+HIV-1+pandemic+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Timing+the+ancestor+of+the+HIV-1+pandemic+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001390-Salemi1" id="pone.0001390-Salemi1"></a>Salemi M, Strimmer K, Hall WW, Duffy M, Delaporte E, et al.  (2001) Dating the common ancestor of SIVcpz and HIV-1 group M and the origin of HIV-1 subtypes using a new method to uncover clock-like molecular evolution. FASEB J  15: 276–278.  <ul class="find" data-citedArticleID="1037430" data-doi="10.1096/fj.00-0449fje"><li><a href="http://dx.doi.org/10.1096/fj.00-0449fje" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+the+common+ancestor+of+SIVcpz+and+HIV-1+group+M+and+the+origin+of+HIV-1+subtypes+using+a+new+method+to+uncover+clock-like+molecular+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+the+common+ancestor+of+SIVcpz+and+HIV-1+group+M+and+the+origin+of+HIV-1+subtypes+using+a+new+method+to+uncover+clock-like+molecular+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001390-Esparza1" id="pone.0001390-Esparza1"></a>Esparza J, Bhamarapravati N (2000) Accelerating the development and future availability of HIV-1 vaccines: why, when, where, and how? Lancet  355: 2061–2066.  <ul class="find" data-citedArticleID="1037380" data-doi="10.1016/s0140-6736(00)02360-6"><li><a href="http://dx.doi.org/10.1016/s0140-6736(00)02360-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accelerating+the+development+and+future+availability+of+HIV-1+vaccines%3A+why%2C+when%2C+where%2C+and+how%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accelerating+the+development+and+future+availability+of+HIV-1+vaccines%3A+why%2C+when%2C+where%2C+and+how%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001390-Lukashov1" id="pone.0001390-Lukashov1"></a>Lukashov VV, Huismans R, Rakhmanova AG, Lisistina ZN, Akhtyrskaya NA, et al.  (1999) Circulation of subtype A and gagA/envB recombinant HIV type 1 strains among injecting drug users in St. Petersburg, Russia, correlates with geographical origin of infections. AIDS Res Hum Retrovir  15: 1577–1583.  <ul class="find" data-citedArticleID="1037406" data-doi="10.1089/088922299309874"><li><a href="http://dx.doi.org/10.1089/088922299309874" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Circulation+of+subtype+A+and+gagA%2FenvB+recombinant+HIV+type+1+strains+among+injecting+drug+users+in+St.+Petersburg%2C+Russia%2C+correlates+with+geographical+origin+of+infections." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Circulation+of+subtype+A+and+gagA%2FenvB+recombinant+HIV+type+1+strains+among+injecting+drug+users+in+St.+Petersburg%2C+Russia%2C+correlates+with+geographical+origin+of+infections.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001390-Lazouskaya1" id="pone.0001390-Lazouskaya1"></a>Lazouskaya NV, Eremin VF, Adema KW, Gasich EL, Baan , et al.  (2005) The HIV type 1 epidemic in Belarus: predominance of Eastern European subtype A strains and circulation of subtype B strains. AIDS Res Hum Retrovir  21: 830–833.  <ul class="find" data-citedArticleID="1037402" data-doi="10.1089/aid.2005.21.830"><li><a href="http://dx.doi.org/10.1089/aid.2005.21.830" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+HIV+type+1+epidemic+in+Belarus%3A+predominance+of+Eastern+European+subtype+A+strains+and+circulation+of+subtype+B+strains." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+HIV+type+1+epidemic+in+Belarus%3A+predominance+of+Eastern+European+subtype+A+strains+and+circulation+of+subtype+B+strains.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001390-Saad1" id="pone.0001390-Saad1"></a>Saad MD, Shcherbinskaya AM, Nadai Y, Kruglov YV, Antonenko SV, et al.  (2006) Molecular epidemiology of HIV type 1 in Ukraine: birthplace of an epidemic. AIDS Res Hum Retrovir  22: 709–714.  <ul class="find" data-citedArticleID="1037428" data-doi="10.1089/aid.2006.22.709"><li><a href="http://dx.doi.org/10.1089/aid.2006.22.709" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+epidemiology+of+HIV+type+1+in+Ukraine%3A+birthplace+of+an+epidemic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+epidemiology+of+HIV+type+1+in+Ukraine%3A+birthplace+of+an+epidemic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001390-Stanojevic1" id="pone.0001390-Stanojevic1"></a>Stanojevic M, Papa A, Papadimitriou E, Zerjav E, Jevtovic D, et al.  (2002) HIV-1 subtypes in Yugoslavia. AIDS Res Hum  Retroviruses18: 519–22.  <ul class="find" data-citedArticleID="1037444" data-doi="10.1089/088922202317406673"><li><a href="http://dx.doi.org/10.1089/088922202317406673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+subtypes+in+Yugoslavia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+subtypes+in+Yugoslavia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001390-Ciccozzi1" id="pone.0001390-Ciccozzi1"></a>Ciccozzi M, Gori C, Boros S, Ruiz-Alvarez MJ, Harxhi A, et al.  (2005) Molecular diversity of HIV in Albania. J Infect Dis  192: 475–479.  <ul class="find" data-citedArticleID="1037374" data-doi="10.1086/431599"><li><a href="http://dx.doi.org/10.1086/431599" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+diversity+of+HIV+in+Albania." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+diversity+of+HIV+in+Albania.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001390-Pybus1" id="pone.0001390-Pybus1"></a>Pybus OG, Charleston MA, Gupta S, Rambaut A, Holmes EC, et al.  (2001) The epidemic behavior of the hepatitis C virus. Science  292: 2323–2325.  <ul class="find" data-citedArticleID="1037412" data-doi="10.1126/science.1058321"><li><a href="http://dx.doi.org/10.1126/science.1058321" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+epidemic+behavior+of+the+hepatitis+C+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+epidemic+behavior+of+the+hepatitis+C+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001390-Hue1" id="pone.0001390-Hue1"></a>Hue S, Pillay D, Clewley JP, Pybus OG (2005) Genetic analysis reveals the complex structure of HIV-1 transmission within defined risk groups. Proc Natl Acad Sci USA  102: 4425–4429.  <ul class="find" data-citedArticleID="1037392" data-doi="10.1073/pnas.0407534102"><li><a href="http://dx.doi.org/10.1073/pnas.0407534102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+analysis+reveals+the+complex+structure+of+HIV-1+transmission+within+defined+risk+groups." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+analysis+reveals+the+complex+structure+of+HIV-1+transmission+within+defined+risk+groups.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001390-Lemey1" id="pone.0001390-Lemey1"></a>Lemey P, Pybus OG, Wang B, Saksena NK, Salemi M, et al.  (2003) Tracing the origin and history of HIV-2 epidemics. Proc Natl Acad Sci USA  100: 6588–6592.  <ul class="find" data-citedArticleID="1037404" data-doi="10.1073/pnas.0936469100"><li><a href="http://dx.doi.org/10.1073/pnas.0936469100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tracing+the+origin+and+history+of+HIV-2+epidemics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tracing+the+origin+and+history+of+HIV-2+epidemics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001390-Robbins1" id="pone.0001390-Robbins1"></a>Robbins E, Lemey P, Pybus OG, Jaffe HW, Saekhou AM, et al.  (2003) Characterization of Early U.S. HIV-1 Strains, Dating of the Introduction of HIV-1 into the U.S. and Estimation of Population History of the U.S. Epidemic. J Virol  77: 6359–6366.  <ul class="find" data-citedArticleID="1037422" data-doi="10.1128/jvi.77.11.6359-6366.2003"><li><a href="http://dx.doi.org/10.1128/jvi.77.11.6359-6366.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+Early+U.S.+HIV-1+Strains%2C+Dating+of+the+Introduction+of+HIV-1+into+the+U.S.+and+Estimation+of+Population+History+of+the+U.S.+Epidemic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+Early+U.S.+HIV-1+Strains%2C+Dating+of+the+Introduction+of+HIV-1+into+the+U.S.+and+Estimation+of+Population+History+of+the+U.S.+Epidemic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001390-Salemi2" id="pone.0001390-Salemi2"></a>Salemi M, de Oliveira T, Soares TM, Pybus OG, Dumans AT, et al.  (2005) Different epidemic potentials of HIV-1 B and C subtypes. J Mol Evol  60: 598–605.  <ul class="find" data-citedArticleID="1037432" data-doi="10.1007/s00239-004-0206-5"><li><a href="http://dx.doi.org/10.1007/s00239-004-0206-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Different+epidemic+potentials+of+HIV-1+B+and+C+subtypes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Different+epidemic+potentials+of+HIV-1+B+and+C+subtypes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001390-Hue2" id="pone.0001390-Hue2"></a>Hue S, Clewley JP, Cane PA, Pillay D (2004) HIV-1 pol gene variation is sufficient for reconstruction of transmissions in the era of antiretroviral therapy. AIDS  18(5): 719–28.  <ul class="find" data-citedArticleID="1037394" data-doi="10.1097/00002030-200403260-00002"><li><a href="http://dx.doi.org/10.1097/00002030-200403260-00002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+pol+gene+variation+is+sufficient+for+reconstruction+of+transmissions+in+the+era+of+antiretroviral+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+pol+gene+variation+is+sufficient+for+reconstruction+of+transmissions+in+the+era+of+antiretroviral+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001390-deSLeal1" id="pone.0001390-deSLeal1"></a>de S Leal E, Holmes EC, Zanotto PM (2004) Distinct patterns of natural selection in the reverse transcriptase gene of HIV-1 in the presence and absence of antiretroviral therapy. Virology  325: 181–191.  <ul class="find" data-citedArticleID="1037454" data-doi="10.1016/j.virol.2004.04.004"><li><a href="http://dx.doi.org/10.1016/j.virol.2004.04.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+patterns+of+natural+selection+in+the+reverse+transcriptase+gene+of+HIV-1+in+the+presence+and+absence+of+antiretroviral+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+patterns+of+natural+selection+in+the+reverse+transcriptase+gene+of+HIV-1+in+the+presence+and+absence+of+antiretroviral+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001390-Richman1" id="pone.0001390-Richman1"></a>Richman DD, Havlir D, Corbeil J, Looney D, Ignacio C, et al.  (1994) Nevirapine resistance mutations of human immunodeficiency virus type 1 selected during therapy. J Virol  68: 1660–1666.  <ul class="find" data-citedArticleID="1037420"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nevirapine+resistance+mutations+of+human+immunodeficiency+virus+type+1+selected+during+therapy.&amp;auth=&amp;atitle=Nevirapine+resistance+mutations+of+human+immunodeficiency+virus+type+1+selected+during+therapy." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nevirapine+resistance+mutations+of+human+immunodeficiency+virus+type+1+selected+during+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nevirapine+resistance+mutations+of+human+immunodeficiency+virus+type+1+selected+during+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001390-Frost1" id="pone.0001390-Frost1"></a>Frost SD, Nijhuis M, Schuurman R, Boucher CA, Brown AJ (2000) Evolution of lamivudine resistance in human immunodeficiency virus type 1-infected individuals: the relative roles of drift and selection. J Virol  74: 6262–6268.  <ul class="find" data-citedArticleID="1037382" data-doi="10.1128/jvi.74.14.6262-6268.2000"><li><a href="http://dx.doi.org/10.1128/jvi.74.14.6262-6268.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+lamivudine+resistance+in+human+immunodeficiency+virus+type+1-infected+individuals%3A+the+relative+roles+of+drift+and+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+lamivudine+resistance+in+human+immunodeficiency+virus+type+1-infected+individuals%3A+the+relative+roles+of+drift+and+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001390-Rouzine1" id="pone.0001390-Rouzine1"></a>Rouzine IM, Coffin JM (1999) Search for the mechanism of genetic variation in the pro gene of human immunodeficiency virus. J Virol  73: 8167–8178.  <ul class="find" data-citedArticleID="1037426"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Search+for+the+mechanism+of+genetic+variation+in+the+pro+gene+of+human+immunodeficiency+virus.&amp;auth=&amp;atitle=Search+for+the+mechanism+of+genetic+variation+in+the+pro+gene+of+human+immunodeficiency+virus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Search+for+the+mechanism+of+genetic+variation+in+the+pro+gene+of+human+immunodeficiency+virus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Search+for+the+mechanism+of+genetic+variation+in+the+pro+gene+of+human+immunodeficiency+virus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001390-Grenfell1" id="pone.0001390-Grenfell1"></a>Grenfell BT, Pybus OG, Gog JR, Wood JL, Daly JM, et al.  (2004) Unifying the epidemiological and evolutionary dynamics of pathogens. Science  303: 327–333.  <ul class="find" data-citedArticleID="1037386" data-doi="10.1126/science.1090727"><li><a href="http://dx.doi.org/10.1126/science.1090727" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Unifying+the+epidemiological+and+evolutionary+dynamics+of+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Unifying+the+epidemiological+and+evolutionary+dynamics+of+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001390-Slatkin1" id="pone.0001390-Slatkin1"></a>Slatkin M, Maddison WP (1989) A cladistic measure of gene flow inferred from the phylogenies of alleles. Genetics  123: 401–416.  <ul class="find" data-citedArticleID="1037442"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles.&amp;auth=&amp;atitle=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+cladistic+measure+of+gene+flow+inferred+from+the+phylogenies+of+alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001390-Thompson1" id="pone.0001390-Thompson1"></a>Thompson JD, Gibson TJ, Plewniak F, Jeanmougin F, Higgins DG (1994) CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools Nucleic Acids Res  25: 4876–4882.  <ul class="find" data-citedArticleID="1037452" data-doi="10.1093/nar/25.24.4876"><li><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL_X+windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001390-Swofford1" id="pone.0001390-Swofford1"></a>Swofford D, Sullivan J (2003)  Phylogeny Inference based on parsimony and other methods with PAUP*. In: Salemi M, Vandamme AM, editors. The Phylogenetic Handbook - a practical approach to DNA and protein phylogeny. New York: Cambridge University Press. pp. 160–206.  <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0001390-Rambaut1" id="pone.0001390-Rambaut1"></a>Rambaut A, Bromham L (1998) Estimating divergence dates from molecular sequences. Mol Biol Evol  15: 442–448.  <ul class="find" data-citedArticleID="1037416" data-doi="10.1093/oxfordjournals.molbev.a025940"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025940" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+divergence+dates+from+molecular+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+divergence+dates+from+molecular+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001390-Rambaut2" id="pone.0001390-Rambaut2"></a>Rambaut A (2000) Estimating the rate of molecular evolution: incorporating non-contemporaneous sequences into maximum likelihood phylogenies. Bioinformatics  16: 395–399.  <ul class="find" data-citedArticleID="1037418" data-doi="10.1093/bioinformatics/16.4.395"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.4.395" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Estimating+the+rate+of+molecular+evolution%3A+incorporating+non-contemporaneous+sequences+into+maximum+likelihood+phylogenies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Estimating+the+rate+of+molecular+evolution%3A+incorporating+non-contemporaneous+sequences+into+maximum+likelihood+phylogenies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001390-Strimmer1" id="pone.0001390-Strimmer1"></a>Strimmer K, von Haeseler A (1997) Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence alignment. Proc Natl Acad Sci USA  94: 6815–6819.  <ul class="find" data-citedArticleID="1037446" data-doi="10.1073/pnas.94.13.6815"><li><a href="http://dx.doi.org/10.1073/pnas.94.13.6815" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Likelihood-mapping%3A+a+simple+method+to+visualize+phylogenetic+content+of+a+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Likelihood-mapping%3A+a+simple+method+to+visualize+phylogenetic+content+of+a+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001390-Schmidt1" id="pone.0001390-Schmidt1"></a>Schmidt HA, Strimmer K, Vingron M, von Haeseler A (2002) TREE-PUZZLE: a maximum likelhood phylogenetic analysis using quartets and parallel computing. Bioinformatics  18: 502–504.  <ul class="find" data-citedArticleID="1037438" data-doi="10.1093/bioinformatics/18.3.502"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.3.502" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TREE-PUZZLE%3A+a+maximum+likelhood+phylogenetic+analysis+using+quartets+and+parallel+computing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TREE-PUZZLE%3A+a+maximum+likelhood+phylogenetic+analysis+using+quartets+and+parallel+computing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001390-Drummond1" id="pone.0001390-Drummond1"></a>Drummond AJ, Ho SYW, Phillips MJ, Rambaut A (2006) Relaxed Phylogenetics and Dating with Confidence. PLoS Biol  4: 1–12.  <ul class="find" data-citedArticleID="1037376" data-doi="10.1371/journal.pbio.0040088"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relaxed+Phylogenetics+and+Dating+with+Confidence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relaxed+Phylogenetics+and+Dating+with+Confidence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001390-Drummond2" id="pone.0001390-Drummond2"></a>Drummond A, Rambaut A, Shapiro B, Pybus OGBayesian coalescent inference of past population dynamics from molecular sequences. Mol Biol Evol  22: 1185–1192.  <ul class="find" data-citedArticleID="1037378" data-doi="10.1093/molbev/msi103"><li><a href="http://dx.doi.org/10.1093/molbev/msi103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001390-Kingman1" id="pone.0001390-Kingman1"></a>Kingman JFC (1982) The coalescent. Stoch. Proc. Appl.  13: 235–248.  <ul class="find" data-citedArticleID="1037398" data-doi="10.1016/0304-4149(82)90011-4"><li><a href="http://dx.doi.org/10.1016/0304-4149(82)90011-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+coalescent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+coalescent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001390-Griffiths1" id="pone.0001390-Griffiths1"></a>Griffiths RC, Tavarè S (1994) Sampling theory for neutral alleles in a varying environment. Philos. Trans. R. Soc. London Ser. B  344: 403–410.  <ul class="find" data-citedArticleID="1037388" data-doi="10.1098/rstb.1994.0079"><li><a href="http://dx.doi.org/10.1098/rstb.1994.0079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sampling+theory+for+neutral+alleles+in+a+varying+environment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sampling+theory+for+neutral+alleles+in+a+varying+environment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001390-Pybus2" id="pone.0001390-Pybus2"></a>Pybus OG, Rambaut A, Harvey PH (2000) An integrated framework for the inference of viral population history from reconstructed genealogies. Genetics  155: 1429–1437.  <ul class="find" data-citedArticleID="1037414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+integrated+framework+for+the+inference+of+viral+population+history+from+reconstructed+genealogies.&amp;auth=&amp;atitle=An+integrated+framework+for+the+inference+of+viral+population+history+from+reconstructed+genealogies." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+integrated+framework+for+the+inference+of+viral+population+history+from+reconstructed+genealogies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+integrated+framework+for+the+inference+of+viral+population+history+from+reconstructed+genealogies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001390-Kass1" id="pone.0001390-Kass1"></a>Kass RE, Raftery AE (1995) Bayes Factors. J Am Stat Ass  90: 773–795.  <ul class="find" data-citedArticleID="1037396" data-doi="10.1080/01621459.1995.10476572"><li><a href="http://dx.doi.org/10.1080/01621459.1995.10476572" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+Factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+Factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001390-Suchard1" id="pone.0001390-Suchard1"></a>Suchard MA, Weiss RE, Sinsheimer JS (2001) Bayesian selection of continuous-time Markov chain evolutionary models. Mol Biol Evol  18: 1001–1013.  <ul class="find" data-citedArticleID="1037448" data-doi="10.1093/oxfordjournals.molbev.a003872"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003872" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+selection+of+continuous-time+Markov+chain+evolutionary+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+selection+of+continuous-time+Markov+chain+evolutionary+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001390-Salemi3" id="pone.0001390-Salemi3"></a>Salemi M, Lamers SL, Yu S, de Oliveira T, Fitch WM, et al.  (2005) HIV-1 phylodynamic analysis in distinct brain compartments provides a model for the neuropathogenesis of AIDS. J Virol  79: 11343–11352.  <ul class="find" data-citedArticleID="1037434" data-doi="10.1128/jvi.79.17.11343-11352.2005"><li><a href="http://dx.doi.org/10.1128/jvi.79.17.11343-11352.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HIV-1+phylodynamic+analysis+in+distinct+brain+compartments+provides+a+model+for+the+neuropathogenesis+of+AIDS." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HIV-1+phylodynamic+analysis+in+distinct+brain+compartments+provides+a+model+for+the+neuropathogenesis+of+AIDS.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.XML" value="95750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.PDF" value="416646"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g001.PNG_L" value="544316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g001.PNG_M" value="78602"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g001.PNG_S" value="14984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g001.TIF" value="1190776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g001.PNG_I" value="85647"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g002.PNG_L" value="1339911"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g002.PNG_M" value="213631"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g002.PNG_S" value="11920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g002.TIF" value="1780212"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g002.PNG_I" value="69255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g003.PNG_L" value="742951"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g003.PNG_M" value="125617"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g003.PNG_S" value="10978"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g003.TIF" value="1341998"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g003.PNG_I" value="43850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.t001.PNG_L" value="191454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.t001.PNG_M" value="107345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.t001.PNG_S" value="10731"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.t001.TIF" value="766774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.t001.PNG_I" value="33028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g004.PNG_L" value="121077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g004.PNG_M" value="41904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g004.PNG_S" value="7854"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g004.TIF" value="357360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g004.PNG_I" value="14790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g005.PNG_L" value="3111863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g005.PNG_M" value="230219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g005.PNG_S" value="11454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g005.TIF" value="5477454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.g005.PNG_I" value="70166"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.e001.PNG" value="5559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.e001.TIF" value="21476"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.s001.DOC" value="58880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.s002.DOC" value="38912"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.s003.PPT" value="267264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.s004.PPT" value="102400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001390.s005.PPT" value="47104"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001390&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001390" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001390&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001390&volume=&issue=&title=High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania&author_name=Marco%20Salemi%2C%20Tulio%20de%20Oliveira%2C%20Massimo%20Ciccozzi%2C%20Giovanni%20Rezza%2C%20Maureen%20M.%20Goodenow&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001390" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390&amp;t=High-Resolution%20Molecular%20Epidemiology%20and%20Evolutionary%20History%20of%20HIV-1%20Subtypes%20in%20Albania" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390&title=High-Resolution%20Molecular%20Epidemiology%20and%20Evolutionary%20History%20of%20HIV-1%20Subtypes%20in%20Albania&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390&amp;title=High-Resolution%20Molecular%20Epidemiology%20and%20Evolutionary%20History%20of%20HIV-1%20Subtypes%20in%20Albania" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001390&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'High-Resolution Molecular Epidemiology and Evolutionary History of HIV-1 Subtypes in Albania';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001390';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20High-Resolution%20Molecular%20Epidemiology%20and%20Evolutionary%20History%20of%20HIV-1%20Subtypes%20in%20Albania http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001390" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001390" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Albania%22" title="Search for articles in the subject area:'Albania'"><div class="flagText">Albania</div></a>
              <div data-categoryid="41403" data-articleid="25610"
                   data-categoryname="Albania"
                   class="flagImage" title="Flag 'Albania' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Albanians%22" title="Search for articles in the subject area:'Albanians'"><div class="flagText">Albanians</div></a>
              <div data-categoryid="41495" data-articleid="25610"
                   data-categoryname="Albanians"
                   class="flagImage" title="Flag 'Albanians' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22HIV-1%22" title="Search for articles in the subject area:'HIV-1'"><div class="flagText">HIV-1</div></a>
              <div data-categoryid="42865" data-articleid="25610"
                   data-categoryname="HIV-1"
                   class="flagImage" title="Flag 'HIV-1' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22NMR+relaxation%22" title="Search for articles in the subject area:'NMR relaxation'"><div class="flagText">NMR relaxation</div></a>
              <div data-categoryid="21901" data-articleid="25610"
                   data-categoryname="NMR relaxation"
                   class="flagImage" title="Flag 'NMR relaxation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="25610"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetics%22" title="Search for articles in the subject area:'Phylogenetics'"><div class="flagText">Phylogenetics</div></a>
              <div data-categoryid="46093" data-articleid="25610"
                   data-categoryname="Phylogenetics"
                   class="flagImage" title="Flag 'Phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Population+groupings%22" title="Search for articles in the subject area:'Population groupings'"><div class="flagText">Population groupings</div></a>
              <div data-categoryid="40421" data-articleid="25610"
                   data-categoryname="Population groupings"
                   class="flagImage" title="Flag 'Population groupings' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="25610"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7267'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1516'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1506&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
