

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Structure and Reaction Mechanism of Basil Eugenol Synthase</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000993"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000993" />

    <meta name="citation_title" content="Structure and Reaction Mechanism of Basil Eugenol Synthase"/>
    <meta itemprop="name" content="Structure and Reaction Mechanism of Basil Eugenol Synthase"/>

      <meta name="citation_author" content="Gordon V. Louie"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Thomas J. Baiga"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Marianne E. Bowman"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Takao Koeduka"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Michigan, Ann Arbor, Michigan, United States of America"/>
      <meta name="citation_author" content="John H. Taylor"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Snejina M. Spassova"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Eran Pichersky"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular and Developmental Biology, University of Michigan, Ann Arbor, Michigan, United States of America"/>
      <meta name="citation_author" content="Joseph P. Noel"/>
            <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America"/>

    <meta name="citation_date" content="2007/10/3"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000993.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e993"/>
    <meta name="citation_issue" content="10"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=An investigation of the storage and biosynthesis of phenylpropenes in sweet basil.; citation_author=DR Gang; citation_author=J Wang; citation_author=N Dudareva; citation_author=KH Nam; citation_author=JE Simon; citation_journal_title=Plant Physiol; citation_volume=125; citation_number=1; citation_pages=539-555; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Biosynthesis of plant volatiles: nature's diversity and ingenuity.; citation_author=E Pichersky; citation_author=JP Noel; citation_author=N Dudareva; citation_journal_title=Science; citation_volume=311; citation_number=2; citation_pages=808-811; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization of a petunia acetyltransferase involved in the biosynthesis of the floral volatile isoeugenol.; citation_author=R Dexter; citation_author=A Qualley; citation_author=CM Kish; citation_author=CJ Ma; citation_author=T Koeduka; citation_journal_title=Plant J; citation_volume=49; citation_number=3; citation_pages=265-275; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Eugenol and isoeugenol, characteristic aromatic constituents of spices, are biosynthesized via reduction of a coniferyl alcohol ester.; citation_author=T Koeduka; citation_author=E Fridman; citation_author=DR Gang; citation_author=DG Vassao; citation_author=BL Jackson; citation_journal_title=Proc Natl Acad Sci; citation_volume=103; citation_number=4; citation_pages=10128-10133; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chavicol formation in sweet basil (Ocimum basilicum): cleavage of an esterified C9 hydroxyl group with NAD(P)H-dependent reduction.; citation_author=DG Vassao; citation_author=DR Gang; citation_author=T Koeduka; citation_author=B Jackson; citation_author=E Pichersky; citation_journal_title=Org Biomol Chem; citation_volume=4; citation_number=5; citation_pages=2733-2744; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structures of pinoresinol-lariciresinol and phenylcoumaran benzylic ether reductases and their relationship to isoflavone reductases.; citation_author=T Min; citation_author=H Kasahara; citation_author=DL Bedgar; citation_author=B Youn; citation_author=PK Lawrence; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=6; citation_pages=50714-50723; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of isoflavone reductase from alfalfa (Medicago sativa L.).; citation_author=X Wang; citation_author=X He; citation_author=J Lin; citation_author=H Shao; citation_author=Z Chang; citation_journal_title=J Mol Biol; citation_volume=358; citation_number=7; citation_pages=1341-1352; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Proanthocyanidin biosynthesis in plants. Purification of legume leucoanthocyanidin reductase and molecular cloning of its cDNA.; citation_author=GJ Tanner; citation_author=KT Francki; citation_author=S Abrahams; citation_author=JM Watson; citation_author=PJ Larkin; citation_journal_title=J Biol Chem; citation_volume=278; citation_number=8; citation_pages=31647-31656; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Identification of cDNAs encoding pterocarpan reductase involved in isoflavan phytoalexin biosynthesis in Lotus japonicus by EST mining.; citation_author=T Akashi; citation_author=S Koshimizu; citation_author=T Aoki; citation_author=S Ayabe; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=9; citation_pages=5666-5670; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recombinant pinoresinol-lariciresinol reductases from western red cedar (Thuja plicata) catalyze opposite enantiospecific conversions.; citation_author=M Fujita; citation_author=DR Gang; citation_author=LB Davin; citation_author=NG Lewis; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=10; citation_pages=618-627; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Evolution of plant defense mechanisms. Relationships of phenylcoumaran benzylic ether reductases to pinoresinol-lariciresinol and isoflavone reductases.; citation_author=DR Gang; citation_author=H Kasahara; citation_author=ZQ Xia; citation_author=K Vander Mijnsbrugge; citation_author=G Bauw; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=11; citation_pages=7516-7527; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=NADP-dependent enzymes. I: Conserved stereochemistry of cofactor binding.; citation_author=O Carugo; citation_author=P Argos; citation_journal_title=Proteins; citation_volume=28; citation_number=12; citation_pages=10-28; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Short-chain dehydrogenases/reductases (SDRs).; citation_author=Y Kallberg; citation_author=U Oppermann; citation_author=H Jornvall; citation_author=B Persson; citation_journal_title=Eur J Biochem; citation_volume=269; citation_number=13; citation_pages=4409-17; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Structural analysis of UDP-sugar binding to UDP-galactose 4-epimerase from Escherichia coli.; citation_author=JB Thoden; citation_author=AD Hegeman; citation_author=G Wesenberg; citation_author=MC Chapeau; citation_author=PA Frey; citation_journal_title=Biochemistry; citation_volume=36; citation_number=14; citation_pages=6294-6304; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Cofactor hydrogen bonding onto the protein main chain is conserved in the short chain dehydrogenase/reductase family and contributes to nicotinamide orientation.; citation_author=R Shi; citation_author=S-X Lin; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=15; citation_pages=16778-16785; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=An unbiased cell morphology-based screen for new, biologically active small molecules.; citation_author=M Tanaka; citation_author=R Bateman; citation_author=D Rauh; citation_author=E Vaisberg; citation_author=S Ramachandani; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=16; citation_pages=128-135; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Critical residues for structure and catalysis in short-chain dehydrogenases/reductases.; citation_author=C Filling; citation_author=KD Berndt; citation_author=J Benach; citation_author=S Knapp; citation_author=T Prozorovski; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=17; citation_pages=25677-25684; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Lysine 190 is the catalytic base in MenF, the menaquinone-specific isochorismate synthase from Escherichia coli: implications for an enzyme family.; citation_author=S Kolappan; citation_author=J Zwahlen; citation_author=R Zhou; citation_author=JJ Truglio; citation_author=PJ Tonge; citation_journal_title=Biochemistry; citation_volume=46; citation_number=18; citation_pages=946-953; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=The catalytic mechanism of Drosophila alcohol dehydrogenase: evidence for a proton relay modulated by the coupled ionization of the active site lysine/tyrosine pair and a NAD+ ribose OH switch.; citation_author=A Koumanov; citation_author=J Benach; citation_author=S Atrian; citation_author=R Gonzalez-Duarte; citation_author=A Karshikoff; citation_journal_title=Proteins; citation_volume=51; citation_number=19; citation_pages=289-98; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Substituent effects on carbocation stability: the pK(R) for p-quinone methide.; citation_author=MM Toteva; citation_author=M Moran; citation_author=TL Amyes; citation_author=JP Richard; citation_journal_title=J Am Chem Soc; citation_volume=125; citation_number=20; citation_pages=8814-8819; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dissection of malonyl-coenzyme A decarboxylation from polyketide formation in the reaction mechanism of a plant polyketide synthase.; citation_author=JM Jez; citation_author=JL Ferrer; citation_author=ME Bowman; citation_author=RA Dixon; citation_author=JP Noel; citation_journal_title=Biochemistry; citation_volume=39; citation_number=21; citation_pages=890-902; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Site-directed mutagenesis by overlap extension using the polymerase chain reaction.; citation_author=SN Ho; citation_author=HD Hunt; citation_author=RM Horton; citation_author=JK Pullen; citation_author=LR Pease; citation_journal_title=Gene; citation_volume=77; citation_number=22; citation_pages=51-59; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants.; citation_author=W Kabsch; citation_journal_title=J Appl Crystallog; citation_volume=26; citation_number=23; citation_pages=795-800; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=The integration of macromolecular diffraction data.; citation_author=AG Leslie; citation_journal_title=Acta Crystallogr; citation_volume=D62; citation_number=24; citation_pages=48-57; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Scaling and assessment of data quality.; citation_author=P Evans; citation_journal_title=Acta Crystallogr; citation_volume=D62; citation_number=25; citation_pages=72-82; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=MOLREP: an automated program for molecular replacement.; citation_author=A Vagin; citation_author=A Teplyakov; citation_journal_title=J Appl Crystallog; citation_volume=30; citation_number=26; citation_pages=1022-1025; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Comparative protein modeling by satisfaction of spatial restraints.; citation_author=A Sali; citation_author=TL Blundell; citation_journal_title=J Mol Biol; citation_volume=234; citation_number=27; citation_pages=779-815; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Crystallography and NMR system: a new software suite for macromolecular structure determination.; citation_author=AT Brunger; citation_author=GL Warren; citation_journal_title=Acta Crystallog; citation_volume=D54; citation_number=28; citation_pages=905-921; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=XtalView/Xfit: a versatile program for manipulating atomic coordinates and electron density.; citation_author=DE McRee; citation_journal_title=J Struct Biol; citation_volume=125; citation_number=29; citation_pages=156-165; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The CCP4 suite: programs for protein crystallography.; citation_journal_title=Acta Crystallog; citation_volume=D50; citation_number=30; citation_pages=760-763; citation_date=1994; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Structure and Reaction Mechanism of Basil Eugenol Synthase"/>
    <meta name="twitter:description" content="Phenylpropenes, a large group of plant volatile compounds that serve in multiple roles in defense and pollinator attraction, contain a propenyl side chain. Eugenol synthase (EGS) catalyzes the reductive displacement of acetate from the propenyl side chain of the substrate coniferyl acetate to produce the allyl-phenylpropene eugenol. We report here the structure determination of EGS from basil (Ocimum basilicum) by protein x-ray crystallography. EGS is structurally related to the short-chain dehydrogenase/reductases (SDRs), and in particular, enzymes in the isoflavone-reductase-like subfamily. The structure of a ternary complex of EGS bound to the cofactor NADP(H) and a mixed competitive inhibitor EMDF ((7S,8S)-ethyl (7,8-methylene)-dihydroferulate) provides a detailed view of the binding interactions within the EGS active site and a starting point for mutagenic examination of the unusual reductive mechanism of EGS. The key interactions between EMDF and the EGS-holoenzyme include stacking of the phenyl ring of EMDF against the cofactor's nicotinamide ring and a water-mediated hydrogen-bonding interaction between the EMDF 4-hydroxy group and the side-chain amino moiety of a conserved lysine residue, Lys132. The C4 carbon of nicotinamide resides immediately adjacent to the site of hydride addition, the C7 carbon of cinnamyl acetate substrates. The inhibitor-bound EGS structure suggests a two-step reaction mechanism involving the formation of a quinone-methide prior to reduction. The formation of this intermediate is promoted by a hydrogen-bonding network that favors deprotonation of the substrate's 4-hydroxyl group and disfavors binding of the acetate moiety, akin to a push-pull catalytic mechanism. Notably, the catalytic involvement in EGS of the conserved Lys132 in preparing the phenolic substrate for quinone methide formation through the proton-relay network appears to be an adaptation of the analogous role in hydrogen bonding played by the equivalent lysine residue in other enzymes of the SDR family."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000993.g009"/>

  <meta property="og:title" content="Structure and Reaction Mechanism of Basil Eugenol Synthase" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=808'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4369'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5796&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000993">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000993" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000993&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Structure and Reaction Mechanism of Basil Eugenol Synthase
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gordon V. Louie, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas J. Baiga, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marianne E. Bowman, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Takao Koeduka, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular, Cellular and Developmental Biology, University of Michigan, Ann Arbor, Michigan, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            John H. Taylor, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Snejina M. Spassova, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Eran Pichersky, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular, Cellular and Developmental Biology, University of Michigan, Ann Arbor, Michigan, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joseph P. Noel
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:noel@salk.edu">noel@salk.edu</a></p>

                <p>Affiliation:
                  Howard Hughes Medical Institute, Jack H. Skirball Center for Chemical Biology and Proteomics, The Salk Institute for Biological Studies, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: October 03, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000993</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000993-g001" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-t001" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g002" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g003" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g004" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g005" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-t002" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g006" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g007" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g008" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000993-g009" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000993">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000993" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Phenylpropenes, a large group of plant volatile compounds that serve in multiple roles in defense and pollinator attraction, contain a propenyl side chain. Eugenol synthase (EGS) catalyzes the reductive displacement of acetate from the propenyl side chain of the substrate coniferyl acetate to produce the allyl-phenylpropene eugenol. We report here the structure determination of EGS from basil (<em>Ocimum basilicum</em>) by protein x-ray crystallography. EGS is structurally related to the short-chain dehydrogenase/reductases (SDRs), and in particular, enzymes in the isoflavone-reductase-like subfamily. The structure of a ternary complex of EGS bound to the cofactor NADP(H) and a mixed competitive inhibitor EMDF ((7S,8S)-ethyl (7,8-methylene)-dihydroferulate) provides a detailed view of the binding interactions within the EGS active site and a starting point for mutagenic examination of the unusual reductive mechanism of EGS. The key interactions between EMDF and the EGS-holoenzyme include stacking of the phenyl ring of EMDF against the cofactor's nicotinamide ring and a water-mediated hydrogen-bonding interaction between the EMDF 4-hydroxy group and the side-chain amino moiety of a conserved lysine residue, Lys132. The C4 carbon of nicotinamide resides immediately adjacent to the site of hydride addition, the C7 carbon of cinnamyl acetate substrates. The inhibitor-bound EGS structure suggests a two-step reaction mechanism involving the formation of a quinone-methide prior to reduction. The formation of this intermediate is promoted by a hydrogen-bonding network that favors deprotonation of the substrate's 4-hydroxyl group and disfavors binding of the acetate moiety, akin to a push-pull catalytic mechanism. Notably, the catalytic involvement in EGS of the conserved Lys132 in preparing the phenolic substrate for quinone methide formation through the proton-relay network appears to be an adaptation of the analogous role in hydrogen bonding played by the equivalent lysine residue in other enzymes of the SDR family.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Louie GV, Baiga TJ, Bowman ME, Koeduka T, Taylor JH, et al.  (2007) Structure and Reaction Mechanism of Basil Eugenol Synthase. PLoS ONE 2(10):
          e993.
            doi:10.1371/journal.pone.0000993</p><p><strong>Academic Editor: </strong>Martin Egli, Vanderbilt University, United States of America</p><p><strong>Received:</strong> June 26, 2007; <strong>Accepted:</strong> September 13, 2007; <strong>Published:</strong> October 3, 2007</p><p><strong>Copyright:</strong> © 2007 Louie et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by National Science Foundation grants 0331353, 0312466 and 0718152 to EP and 0236027 and 0718064 to JNP. JPN is an investigator of the Howard Hughes Medical Institute.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The phenylpropenes are a diverse group of plant secondary metabolites characterized by a phenyl ring bearing a propenyl side chain (<a href="#pone-0000993-g001">Figure 1A</a>). A variety of phenylpropenes occur in angiosperms, whereas a more limited subset of these compounds exist in gymnosperms. In plants, the phenylpropenes function in defense and interspecies communication. Because some phenylpropenes are toxic to animals and microorganisms, these compounds are typically produced and stored in plant vegetative tissues to act as deterrents against herbivores and microbial pathogens <a href="#pone.0000993-Gang1">[1]</a>. Moreover, some volatile phenylpropenes are emitted by flowering plants and serve as attractants for insect pollinators <a href="#pone.0000993-Pichersky1">[2]</a>. Historically, humans have exploited both the aromatic and toxic properties of the phenylpropenes in perfumes, flavorings, preservatives, and general antiseptics.</p>
<div class="figure" id="pone-0000993-g001"><div class="img"><a name="pone-0000993-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Structure and biosynthesis of the phenylpropenes.</span></strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>(A) Phenylpropene diversity. The basic skeleton of an allyl phenylpropene is shown, along with the numbering of the carbon atoms used in this manuscript. The iso phenylpropenes differ in the position of the double bond (C7 = C8) in the propene side group. For eugenol (an allyl phenylpropene), the phenyl-ring substituents are 2 = H, 3 = OCH<sub>3</sub>, 4 = OH, and 5 = H. The possible methylenedioxy-bridge modifications are also shown. (B) Simplified reaction scheme for the biosynthesis of eugenol from L-phenylalanine and the monolignol alcohol, coniferyl alcohol (CFAT: coniferyl alcohol acetyl transferase; EGS: eugenol synthase).</p>
<span>doi:10.1371/journal.pone.0000993.g001</span></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The phenylpropenes are derived from coumaryl, coniferyl, and sinapyl alcohol, which are also intermediates in the lignin and lignan biosynthetic pathways. As precursors for phenylpropene production, the monolignol alcohols undergo first acetylation of the C9 hydroxyl group <a href="#pone.0000993-Dexter1">[3]</a> and then reductive cleavage of the acetate moiety to yield the propenyl side group <a href="#pone.0000993-Koeduka1">[4]</a>, <a href="#pone.0000993-Vassao1">[5]</a>. This reduction reaction is catalyzed by enzymes that produce an allyl propene (with the double bond between C9 and C8) or an “iso” propene (with the double bond between C8 and C7). An example of the former is basil (<em>Ocimum basilicum</em>) eugenol synthase (EGS), which converts coniferyl acetate to eugenol, and an example of the latter is petunia (<em>Petunia hybrida</em>) isoeugenol synthase (IGS), which converts coniferyl acetate to isoeugenol (<a href="#pone-0000993-g001">Figure 1</a>). Further modifications required for formation of the known natural phenylpropenes include additional hydroxylation of the benzene ring, methylation of any of the hydroxyl groups on the benzene ring, and formation of a methylenedioxy bridge (<a href="#pone-0000993-g001">Figure 1</a>). Some of these modifications may occur before the formation of the propene moiety, but a free hydroxyl group at the <em>para</em> position appears to be a requirement for the reduction reaction <a href="#pone.0000993-Koeduka1">[4]</a>. The biosynthetic routes to the phenylpropenes generate considerable chemical diversity.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The basil EGS and petunia IGS are closely related to a number of other NADPH-dependent enzymes that act on phenylpropanoid-derived substrates. These enzymes constitute the PIP family, named after the three initially identified members, pinoresinol-lariciresinol reductase (PLR) <a href="#pone.0000993-Min1">[6]</a>, isoflavone reductase (IFR) <a href="#pone.0000993-Wang1">[7]</a>, and phenylcoumaran benzylic ether reductase (PCBER) <a href="#pone.0000993-Min1">[6]</a>. Other enzymes in this family are leucocyanidin reductase <a href="#pone.0000993-Tanner1">[8]</a>, and pterocarpan reductase <a href="#pone.0000993-Akashi1">[9]</a>. Notably, several PIP enzymes in addition to EGS and IGS catalyze the reductive cleavage of a carbon-oxygen bond that occurs in a phenyl-ring substituent positioned <em>para</em> to the 4-hydroxyl group. The PIP-enzyme catalyzed reductions all involve A-type stereospecificity of hydride transfer from the NAD(P)H cofactor (donation of the nicotinamide C4 pro-R hydride). However, the reaction mechanism of the PIP enzymes remains to be fully characterized. In particular, the cleavage of a carbon-oxygen bond represents an unusual application of a nicotinamide-cofactor dependent reduction, which more typically adds a hydride anion and proton across a double bond in the substrate. The involvement of a quinone-methide (conjugated enone) intermediate in the bond cleavage <a href="#pone.0000993-Koeduka1">[4]</a>, <a href="#pone.0000993-Akashi1">[9]</a>–<a href="#pone.0000993-Gang2">[11]</a> is frequently assumed, although solid experimental support has not been reported. Therefore, a direct reductive displacement <a href="#pone.0000993-Koeduka1">[4]</a> of the oxygen function by hydride ion cannot be excluded.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>We describe here the crystal structure of basil EGS in apo and holo forms, and also as a ternary complex with the cofactor NADP(H) and a designed inhibitor, (7S,8S)-ethyl (7,8-methylene)-dihydroferulate). Previous crystallographic studies of PIP-family enzymes yielded only apo-structures <a href="#pone.0000993-Min1">[6]</a>, <a href="#pone.0000993-Wang1">[7]</a>, and thus, reliable pictures of cofactor and substrate binding and catalytic mechanism were significantly hampered. Our EGS structures now clearly reveal the interactions formed by the substrate within the enzyme's active site and identify possible catalytic residues. These studies, together with the analysis of the activity of the protein following in vitro mutagenesis of specific residues, provide circumstantial support for a reaction mechanism involving a quinone-methide intermediate and the participation of a key lysine residue in an unusual push-pull like two-step catalytic mechanism.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Crystallographic structure elucidation for basil EGS complexed with NADP<sup>+</sup></h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A structure solution was obtained initially for the orthorhombic crystal form of holo-EGS, which contains two EGS/NADP<sup>+</sup> complexes per asymmetric unit. A three-dimensional model was refined at 1.7-Å resolution resulting in crystallographic R-factors of 0.244 and 0.267 (working and FreeR, respectively; see <a href="#pone-0000993-t001">Table 1</a>). This refined structure then served as the search model for MR analysis of the monoclinic crystal form. The monoclinic structure also contains two EGS/NADP<sup>+</sup> complexes per asymmetric unit and was refined at 1.6-Å resolution resulting in crystallographic R-factors of 0.210 and 0.229 (<a href="#pone-0000993-t001">Table 1</a>). For each of the other EGS crystal forms, MR solutions were obtained with either the monoclinic or orthorhombic crystal structure of EGS serving as the search model. A 2-fold rotationally symmetric homodimer is consistently observed as the asymmetric unit in all crystal forms analyzed and solved to date. However, the inter-monomer association within the dimer is not extensive, and in agreement with elution behavior on gel-exclusion chromatography, monomeric EGS is likely the functionally relevant form. In all cases, the entire polypeptide chain of EGS is visible in electron-density maps, except for four residues at the N-terminus, and additionally the NADP<sup>+</sup> cofactor is extremely well ordered.</p>
<div class="figure" id="pone-0000993-t001"><div class="img"><a name="pone-0000993-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of data collection and refinement statistics for EGS structures</span></strong></p><span>doi:10.1371/journal.pone.0000993.t001</span></div>

<h4>Overall structure of basil EGS and structure comparisons with other IFR-like proteins</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>EGS is very similar in polypeptide-chain fold to three other PIP-family proteins that have been structurally characterized, pinoresinol-lariciresinol reductase (PLR) <a href="#pone.0000993-Min1">[6]</a>, isoflavone reductase (IFR) <a href="#pone.0000993-Wang1">[7]</a>, and PCBER <a href="#pone.0000993-Min1">[6]</a>. Structurally, the PIP-family proteins organize around an N-terminal, Rossman-fold domain, containing a core, six-stranded parallel β-sheet flanked on each face by an α-helical layer (<a href="#pone-0000993-g002">Figure 2A</a>). One edge of the core β-sheet provides the extended binding surface for the NADP<sup>+</sup> cofactor (as discussed further below). The C-terminal polypeptide-chain segment of the PIP-family proteins forms a predominantly α-helical domain, and this C-terminal segment also contributes three additional β-strands to the Rossman-fold domain. The C-terminal domain is presumed (see below) to function in substrate binding. Indeed, this domain together with the last α-helix of the Rossman-fold domain surround a cavity located immediately adjacent to the nicotinamide ring of the NADP<sup>+</sup> cofactor. Within the IFR-like PIP family, the substrate-binding domains (residues 154–314 in EGS) appear more structurally divergent than the nicotinamide-cofactor binding domains (residues 1–153 in EGS). For example, from comparisons of polypeptide-chain backbones (<a href="#pone-0000993-g002">Figure 2B</a>), EGS differs from PCBER by 1.40 Å (rmsd) overall, but by only 0.83 Å for the Rossman-fold domain alone; and similarly, EGS differs from IFR by 1.63 Å overall and 1.09 Å for the Rossman-fold domain alone. Curiously, PCBER, PLR, and IFR-like EGS-all form 2-fold rotationally symmetric homodimers, but the various homodimeric associations are distinct in each case. An additional structural element unique to EGS (absent in the other PIP-family proteins) is a proline-rich extension at the C-terminus. This tail segment passes across the mouth of the active-site region, and the side chain of the C-terminal phenylalanine residue participates directly in forming the substrate-binding pocket. The positioning of the tail segment in EGS precludes the formation of the homodimeric associations observed in PCBER and IFR.</p>
<div class="figure" id="pone-0000993-g002"><div class="img"><a name="pone-0000993-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Structure of EGS and comparison with other PIP-family enzymes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(A) Orthogonal views of EGS/NADP<sup>+</sup> monomer. The polypeptide chain of EGS is represented as a ribbon, with coloring varying from blue for the N-terminus to red for the C-terminus. The atoms of the NADP<sup>+</sup> cofactor are drawn as balls and sticks, and are colored coded according to element (carbon: gray; nitrogen: blue; oxygen: red; phosphorus: orange). (B) Superposition of the polypeptide-chain backbones of EGS and other PIP-family enzymes (color coding as shown in inset). The NADP<sup>+</sup> cofactor of EGS is also shown (the structures of the other PIP-family enzymes were determined in the absence of cofactor).</p>
<span>doi:10.1371/journal.pone.0000993.g002</span></div>

<h4>Binding of the NADP(H) cofactor to EGS</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The structures of EGS complexed with NADP<sup>+</sup> or NADPH provide the first structural characterization of nicotinamide-cofactor binding by the PIP family of enzymes (previous crystallographic analyses had yielded only apo-enzyme structures). The cofactor is bound through a large number of polar and non-polar interactions (<a href="#pone-0000993-g003">Figure 3</a>). The adenine ring adopts the anti conformation and is sandwiched between the δ-guanido group of Arg39 and the carboxamide group of Gln87. The adenine-ribose lies in the C3′-endo conformation. The ribose ring is packed against the α-carbons of both Gly14 and Gly17, and the central diphosphate group forms hydrogen bonds with the backbone amide-nitrogens of residues 18 and 19. The protein residues involved in these interactions reside within the Gly14-Xaa-Xaa-Gly17-Xaa-Xaa-Gly20 segment, a canonical sequence-motif for NAD(P) binding <a href="#pone.0000993-Carugo1">[12]</a>. The 2′-phosphate is sequestered by a short loop formed by residues 38–42, and is hydrogen bonded to the side chains of Thr38, Arg39 and Ser42. Thr16 from the glycine-rich loop also hydrogen bonds the 2′-phosphate, as well as the adjacent 3′-hydroxyl group. The nicotinamide-ribose has the C2′-endo conformation, and its hydroxyl groups are involved in hydrogen bonds with residue 111′s carbonyl oxygen, Ser110′s side chain hydroxyl moiety, and the side-chain amino group of Lys132. The nicotinamide ring adopts the anti conformation with its B-face stacked against the side chain of Phe154 and its A-face directed towards the substrate-binding pocket. The nicotinamide carboxamide group forms hydrogen bonds to three backbone atoms (ON7 to 154 N, and NN7 to 112 O and 152 O).</p>
<div class="figure" id="pone-0000993-g003"><div class="img"><a name="pone-0000993-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Interactions between EGS and the NADP<sup>+</sup> cofactor.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Only the EGS polypeptide-chain segments that form direct interactions with the NADP<sup>+</sup> cofactor are shown. Hydrogen-bond interactions formed by the cofactor are represented as magenta dashed lines. Atom coloring is the same as in <a href="#pone-0000993-g002">Figure 2A</a>, except that the carbon atoms of the polypeptide-chain segments are green. The blue-colored contours envelope regions greater than 3σ in the NADP-omit electron-density map.</p>
<span>doi:10.1371/journal.pone.0000993.g003</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Structural differences between the NADP<sup>+</sup> and NADPH complexes of EGS are small, and center around the nicotinamide ring of the cofactor. The nicotinamide forms a planar group in the oxidized state, but is distorted slightly to a more boat-like conformation in the reduced state. From a comparison of apo-EGS and the EGS-NADP(H) complexes, it also appears that little structural perturbation results from binding of NADP(H), aside from better ordering of the polypeptide-chain segments that form the cofactor-binding cleft.</p>


<h4>Structural comparison of EGS with UDP-galactose 4-epimerase</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>The PIP-family enzymes belong to a larger superfamily of NAD(P)-dependent dehydrogenases, the short-chain dehydrogenases/reductases (SDRs) <a href="#pone.0000993-Kallberg1">[13]</a>. The most similar member of the larger superfamily is UDP-galactose 4-epimerase <a href="#pone.0000993-Thoden1">[14]</a> (PDB entry 1KVQ), which provided a template for the binding modes of both the nicotinamide cofactor and substrate in the earlier structural analyses of the apo-forms of IFR, PCBER and PLR <a href="#pone.0000993-Min1">[6]</a>, <a href="#pone.0000993-Wang1">[7]</a>. Indeed, UDP-galactose 4-epimerase possesses a C-terminal domain that is similar topologically to the C-terminal domains of the PIP-family proteins (<a href="#pone-0000993-g004">Figure 4A</a>). In the UDP-galactose 4-epimerase crystal structure, a cavity within the C-terminal domain is positioned next to the nicotinamide ring of the NAD<sup>+</sup> cofactor and is occupied by the substrate, UDP galactose. The corresponding cavity in EGS is much smaller in volume and the side chains lining the cavity are more non-polar in character. These properties of the EGS substrate-binding pocket are consistent with the smaller size and greater hydrophobicity of the EGS substrate, the acetate ester of coniferyl alcohol. Notably, the conformation of NAD<sup>+</sup> bound to UDP-galactose 4-epimerase differs markedly from that of NADP<sup>+</sup> bound to EGS, particularly in the conformation of the nicotinamide ring (<a href="#pone-0000993-g004">Figure 4B</a>). In the UDP-galactose 4-epimerase/NAD<sup>+</sup> complex, the nicotinamide ring adopts the syn conformation, consistent with the class-B oxidoreductase activity of the enzyme. In contrast, the anti-conformer of the nicotinamide ring observed in EGS is consistent with the class-A reductase (donation of the pro-R hydride) activity of the PIP-family enzymes. The orientation of the nicotinamide ring in EGS appears to be influenced largely by interactions of the carboxamide group with the polypeptide-chain backbone, an observation also made previously for the SDRs <a href="#pone.0000993-Shi1">[15]</a>. Also, EGS possesses an additional loop (residues 38–42) that forms a binding pocket for the 2′-phosphate group of NADP(H). This loop is absent in UDP-galactose 4-epimerase, but occurs with variable length in all PIP-family proteins.</p>
<div class="figure" id="pone-0000993-g004"><div class="img"><a name="pone-0000993-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Structural comparison of EGS and UDP-galactose epimerase.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(A) Superposition of polypeptide-chain backbones of EGS and UDP-galactose epimerase (color coding as shown in inset). For clarity, only the NADP<sup>+</sup> cofactor of EGS is shown. (B) Comparison of NAD(P)-cofactor conformation and substrate-analog binding in EGS and UDP-galactose epimerase. The binding of the EGS competitive inhibitor (7S,8S)-ethyl (7,8-methylene)-dihydroferulate (EMDF) is described in detail in the text and <a href="#pone-0000993-g005">Figure 5</a>. The coloring of the polypeptide-chain segments is the same as in (A). The inset shows the coloring used for the carbon atoms of the nicotinamide cofactors, EMDF bound to EGS, and UDP-glucose bound to UDP-galactose-4-epimerase.</p>
<span>doi:10.1371/journal.pone.0000993.g004</span></div>

<h4>Binding of the EGS-inhibitor EMDF</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The co-crystal structure of EGS complexed with a specifically designed inhibitor, EMDF ((7S,8S)-ethyl (7,8-methylene)-dihydroferulate) provides a view of the substrate-binding mode within the active site of EGS (<a href="#pone-0000993-g004">Figures 4B</a> and <a href="#pone-0000993-g005">5A</a>). This inhibitor is a close structural analog of coniferyl acetate, and carries the same functional groups on the C3 and C4 (<em>para</em>) positions of the phenyl (guaiacol) ring. However, EMDF cannot serve as a substrate of EGS because within the C1 substituent, a cyclopropyl group replaces of the C7-C8 double bond and the orientation of the ester is reversed. Our measurements indicate that EMDF acts as a competitive inhibitor of EGS, with an inhibition constant (K<sub>i</sub> = 0.8 mM) similar to the K<sub>m</sub> for coniferyl acetate, 0.57 mM. A key interaction between EMDF and EGS is the packing of the inhibitor's phenyl ring parallel to the A-face of the cofactor's nicotinamide ring (interplanar separation 3.4 Å). Notably, stacking of a substrate aromatic-ring against the NAD(P) nicotinamide ring is a common feature of the binding modes of SDRs <a href="#pone.0000993-Tanaka1">[16]</a>. In EGS, the nicotinamide ring is, in turn, stacked against the side chain of Phe154. The <em>para</em>-hydroxy group of the inhibitor's guaiacol moiety hydrogen bonds with the backbone amide-nitrogen of Val114 and also interacts via a bridging water molecule with the side-chain amino group of Lys132. The residues lining the binding pocket are otherwise predominantly aromatic (Phe85, Phe125, Tyr157, Phe158, Tyr161, and Phe314) and aliphatic (Val114, Ile261, Leu262, and Leu265). In the known EGS and IGS sequences, Lys132, Tyr157, Phe158, and Ile261 are invariant, whereas conservative amino-acid substitutions are observed at the other binding-pocket residues. The inhibitor's 3-hydroxymethyl group is accommodated within a small, non-polar pocket. The observed orientation of the guaiacol moiety would be clearly favored over the reverse orientation (resulting from a 180° rotation around the C7-C1 bond), which would position the hydroxymethyl group in close contact with the nicotinamide ribose and the side chain of Phe85. However, the absence of specific interactions formed by the hydroxymethyl group within its binding pocket is perhaps consistent with the limited ability of EGS to utilize as substrate coumaryl acetate (unpublished data), which lacks a substituent at the 3-position. In addition, the acetate ester of sinapyl alcohol, which bears hydroxymethyl groups at both the 3- and 5-positions, would be expected to be incapable of binding to EGS. The inhibitor's C1 substituent bearing the cyclopropyl and ethyl-ester moieties projects into a cavity formed by the C-terminal domain of the protein. This cavity is capped by the side chains of Tyr157, Tyr161, Pro258, Leu262 and Phe314. With the exception of the C-terminal Phe314, these capping residues form a relatively rigid cage, as indicated by their invariant positioning in all EGS crystal structures and low crystallographic temperature factors. The capping region appears to lack sufficient volume to accommodate a C1 substituent larger than an acetate-esterified propenol. This finding is in agreement with the observed inactivity of EGS toward other esters of coniferyl alcohol, for example coniferyl coumarate <a href="#pone.0000993-Koeduka1">[4]</a>, which bears a much bulkier substituent.</p>
<div class="figure" id="pone-0000993-g005"><div class="img"><a name="pone-0000993-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Binding of the competitive inhibitor EMDF by EGS.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) Orthogonal views of the 7S,8S- stereoisomer of the competitive inhibitor EMDF bound to EGS. Hydrogen-bond interactions formed by the EMDF molecule (cyan colored carbons) are represented as magenta dashed lines. The blue-colored contours envelope regions greater than 2.5σ in the initial F<sub>obs</sub>-F<sub>calc</sub> electron-density map. The direction of view used in the right panel (approximately perpendicular to the plane of the nicotinamide ring) is maintained roughly in <a href="#pone-0000993-g005">figures 5B–D</a> and <a href="#pone-0000993-g006">6B</a>. The chemical structure of EMDF is shown in the inset. (B) Modeled binding of coniferyl acetate to EGS. The atom coloring is the same as in (A), with magenta carbon atoms for the coniferyl acetate. The chemical structures of coniferyl acetate and EMDF are compared in the inset. The close interaction between the EMDF C7-atom and the hydride donor of the nicotinamide (C4) is shown as a yellow dashed line. (C) Binding of EMDF to the Lys132Gln variant of EGS. Hydrogen-bond interactions formed by the EMDF molecule (cyan colored carbons) are represented as magenta dashed lines. Hydrogen bonds involving the side chain of Gln132 are shown as orange dashed lines. The blue-colored contours envelope regions greater than 2.0σ in the initial F<sub>obs</sub>-F<sub>calc</sub> electron-density map. (D) Binding of EMDF to the Lys132Arg variant of EGS (stereo representation). The blue-colored contours envelope regions greater than 2σ in the initial F<sub>obs</sub>-F<sub>calc</sub> electron-density map for the EGS-Arg132/EMDF complex (green). The altered positioning of the Arg132 side-chain and neighboring residues (most notably Phe85, Ile88, and Ile129) and the disordering of the C-terminal tail (residues 310–314) are apparent with respect to the holo-EGS-Arg132 structure (magenta). For comparison, the position of the wild-type Lys132 side chain and the key bridging water molecule shown in <a href="#pone-0000993-g005">Figure 5A</a> are also shown (yellow).</p>
<span>doi:10.1371/journal.pone.0000993.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Curiously, at the inhibitor-binding site described above, some residual electron density is invariably observed, even with crystal samples prepared with EGS protein that had not been purposely exposed to a potential ligand. This density might be due to low-occupancy binding of a small, eugenol-resembling compound that originated from bacterial growth-media derived from yeast extracts. However, soaking experiments of EGS crystals with coniferyl acetate, or other substrate analogs (e.g. coumaryl acetate and 4-bromo-cinnamyl acetate) failed to produce stable complexes with EGS. The difficulty in obtaining EGS complexes with these compounds is possibly explained by the implications of the finding that binding of the true substrate, coniferyl acetate, cannot be readily modeled based on the observed positioning of the EMDF inhibitor compound. The determinative structural feature of coniferyl acetate is the planarity of the propene moiety (C7-C8-C9), which results in steric clashes between the terminal acetate moiety and neighboring residues in the active-site capping region, in particular Tyr157, Ile261 and Phe314. In contrast, the cyclopropyl group of EMDF induces a distinct kink in the conformation of the side group, which steers the end of the side group into a hole formed within the capping region (<a href="#pone-0000993-g005">Figure 5B</a>). The stringent binding selectivity of EGS is further emphasized by the observation that only the 7-(S),8-(S) stereoisomer of EMDF complexed with EGS; the 7-(R),8-(R) isomer (present at low levels in the EMDF preparation) was excluded, due to poorer steric complementarity with the EGS active site.</p>


<h4>3D structure determination and in vitro mutagenesis suggests a reaction mechanism for EGS</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Although the binding of EMDF exploits shape features of the EGS active site that are inaccessible to the coniferyl acetate substrate, the structure of the EGS-NADP<sup>+</sup>-EMDF complex nevertheless provides a useful framework for probing the EGS enzymatic mechanism. Together with the observed effects on catalytic activity of specific amino-acid replacements (<a href="#pone-0000993-t002">Table 2</a>), as described below, the structure provides compelling support for the involvement of a quinone-methide intermediate both in promoting carbon-oxygen bond cleavage of the acetate moiety and in serving as the actual substrate of the reduction reaction via NADPH-mediated hydride transfer.</p>
<div class="figure" id="pone-0000993-t002"><div class="img"><a name="pone-0000993-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Relative activity of EGS variants</span></strong></p><span>doi:10.1371/journal.pone.0000993.t002</span></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>A prominent active-site residue is Lys132, which occurs in all PIP-family enzymes as well as most SDRs <a href="#pone.0000993-Filling1">[17]</a>. The structure of the EGS-NADP<sup>+</sup>-EMDF complex shows that the ε-amino group of Lys132 forms interactions with both the nicotinamide-ribose of NADP(H), and potentially, the substrate molecule (<a href="#pone-0000993-g005">Figure 5A</a>). The Lys132 interaction with the substrate is particularly intriguing, as it is not proximal to the site of hydride addition (as suggested in the case of IFR <a href="#pone.0000993-Wang1">[7]</a>), but instead involves the p-hydroxyl group via a bridging water molecule. Notably, a p-hydroxyl group is a distinguishing feature of the substrates of all PIP-family enzymes, and is a requirement for reduction by EGS <a href="#pone.0000993-Koeduka1">[4]</a>. For both PLR and IFR, alanine replacements of the lysine that is equivalent to EGS-Lys132 abolish enzyme activity <a href="#pone.0000993-Min1">[6]</a>, <a href="#pone.0000993-Wang1">[7]</a>. In EGS, Lys132Ala and Lys132Gln mutants are completely inactive, whereas the Lys132Arg mutant retains partial (71%) activity (<a href="#pone-0000993-t002">Table 2</a>). Crystallographic analyses confirmed that for the Ala132 and Gln132 mutants, both NADP<sup>+</sup> and EMDF binding are little affected (<a href="#pone-0000993-g005">Figure 5C</a>), despite the loss of the binding interactions normally contributed by Lys132. These results therefore point to a catalytic role for Lys132. In particular, the involvement of a catalytic group acting at the p-hydroxyl group clearly argues for the formation of a quinone methide as a reaction intermediate as opposed to direct nucleophilic replacement by a NADPH derived hydride anion.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>The formation of a quinone methide can be promoted by abstraction of the proton from the p-hydroxyl group of substrate. Detailed inspection of the hydrogen-bonding network involving the ε-amino group of Lys132 (<a href="#pone-0000993-g006">Figure 6A</a>) suggests that this group exists formally in the unprotonated–NH<sub>2</sub> state and is the donor in hydrogen-bond interactions with the 2′-hydroxyl group of the nicotinamide-ribose and the backbone carbonyl oxygen of residue 110. With an available lone pair of electrons, the amino group can serve as the acceptor in a hydrogen bond with the bridging water molecule, and most importantly, thereby act as a general base. The water molecule (as a hydroxide ion) can in turn facilitate deprotonation of the substrate's p-hydroxyl group. Intruigingly, in the monoclinic structure of unliganded EGS-NADP<sup>+</sup>, a nitrate anion from the crystallization medium occupies the site of the bridging water molecule, and may mimic the hydroxide ion that develops during the catalytic reaction.</p>
<div class="figure" id="pone-0000993-g006"><div class="img"><a name="pone-0000993-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Hydrogen-bonding interactions in the EGS active-site and proposed reaction mechanism of EGS.</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>(A) Hydrogen-bonding network involving the Lys132 side-chain amino group, the 4-hydroxyl group of EMDF, and the bridging water molecule. Inferred hydrogen-atom positions are shown in blue. Hydrogen-bond interactions are represented as magenta dashed lines. (B) Comparison of the hydrogen-bond interactions made by the catalytic lysine residue in EGS (Lys132) and the SDR UDP-galactose-4-epimerase (Lys153). The inset shows the coloring used for the carbon atoms of EMDF bound to EGS, and UDP-glucose bound to UDP-galactose-4-epimerase. Hydrogen bonds are drawn as thin dashed lines. Water molecules are drawn as red spheres. Those outlined in the pale red form part of a postulated proton-relay network in UDP-galactose-4-epimerase, whereas the presumed catalytic water molecule in EGS is outlined in green and marked with an asterisk. (C) Proposed reaction mechanism of EGS (and IGS) involving a quinone-methide intermediate. The catalytic base (B:) that promotes deprotonation of the 4-hydroxyl group of the substrate is the hydroxide ion that is activated by the side-chain amino group of Lys132. The loss of acetate generates the quinone-methide intermediate. The attack at C7 of this intermediate by the NADPH-derived hydride yields the allyl phenylpropene eugenol, whereas hydride attack at C9 presumably yields the iso-phenylpropene isoeugenol.</p>
<span>doi:10.1371/journal.pone.0000993.g006</span></div><a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>The loss in activity of EGS Lys132-mutants can be interpreted in terms of the proposed mechanistic model, in conjunction with results from structural analyses. In the EGS (Lys132Gln)-NADP<sup>+</sup>-EMDF complex, the Gln132 side chain retains an interaction with the nicotinamide ribose (also through an intervening water molecule), but is unable to form a direct or water-mediated interaction with the p-hydroxyl group of EMDF. Likewise, Ala132 obviously lacks a catalytic group capable of promoting deprotonation of the substrate's p-hydroxyl group, and therefore, the observed inactivity of the Lys132Ala and Lys132Gln mutants can be readily explained. The Lys132Arg mutant is partially active, and in this case, with the higher pK<sub>a</sub> and greater length of the Arg side chain, the positively charged guanidinium moiety could possibly participate directly (i.e. without the requirement for an intervening water molecule) in lowering the pK<sub>a</sub> of the substrate's p-hydroxyl group. Interestingly, preliminary structural analysis of the holo and EMDF-bound forms of the Lys132Arg mutant shows that the Arg132 guanidinium moiety is displaced slightly by the EMDF guaiacol ring (<a href="#pone-0000993-g005">Figure 5D</a>), thus diminishing the potential influence of the Arg132 on catalysis.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5"></a><p>One difficulty with the proposed catalytic role for a lysine residue is the relatively high pK<sub>a</sub> (normally ~10.4 in solution) of the side-chain amino group, which would disfavor acquisition of the initial free-base state required for proton abstraction from the substrate. However, the pK<sub>a</sub>s of ionizable groups in proteins can be greatly influenced by the local structural environment, in particular, involvement in hydrogen-bonding networks and hydrophobic interactions. Such factors have been suggested to account for the catalytic-base activity of the lysine ε-amino group in a number of enzymes. A notable example is isochorismate synthase <a href="#pone.0000993-Kolappan1">[18]</a>, in which a catalytic lysine is proposed to deprotonate and thereby activate a nucleophilic water molecule. Furthermore, from theoretical calculations on the conserved Lys-Tyr-Ser catalytic triad in an SDR-type alcohol dehydrogenase, the catalytically important lysine residue is suggested to exist in a partially unprotonated state, and in this state, participate in a proton-relay network that involves hydroxyl groups on the catalytic tyrosine and nicotinamide ribose <a href="#pone.0000993-Koumanov1">[19]</a>. This network functions ultimately to abstract a proton from the alcohol substrate. Intriguingly, although EGS lacks the catalytic tyrosine of the SDR enzyme, Lys132 in EGS corresponds exactly to the catalytic lysine of the SDRs, and the p-hydroxyl group of EGS-bound EMDF occurs at roughly the same position as the Tyr ζ-hydroxyl group of the catalytic tyrosine of SDRs (<a href="#pone-0000993-g006">Figure 6B</a>).</p>
<a id="article1.body1.sec2.sec6.p6" name="article1.body1.sec2.sec6.p6"></a><p>For the EGS-catalyzed reaction with the coniferyl acetate substrate, formation of the quinone-methide intermediate would be concomitant with displacement of an acetate ion (<a href="#pone-0000993-g006">Figure 6C</a>). In concert with proton abstraction from the p-hydroxyl group, EGS may therefore exploit an additional driving force for generation of the reaction intermediate—promoting the loss of the acetate. In particular, steric restrictions within the enzyme's active site (as discussed above) appear to disfavor the binding of an extended C1 substituent on the substrate. Furthermore, due to the predominantly non-polar character of the capping region of the EGS active site, only a single residue, Tyr157, is available for hydrogen-bond interactions with the polar oxygen atoms of the acetate moiety. In fact, the cluster of aromatic side-chains in this region of the active site (see <a href="#pone-0000993-g005">Figure 5A</a>) may provide a favorable environment for stabilizing the carbocationic character of C7 and C9 <a href="#pone.0000993-Toteva1">[20]</a> within the extended quinone-methide. The proposed model for disfavoring binding of the acetate group is consistent with the diminished catalytic activity of mutants of EGS that carry smaller active-site capping residues (Tyr157Ala, Tyr157Phe, Phe314Ala, and Δ311–314; see <a href="#pone-0000993-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec6.p7" name="article1.body1.sec2.sec6.p7"></a><p>Both the Tyr157Ala and Tyr157Phe mutants retain partial enzyme activity (21% and 33%, respectively; <a href="#pone-0000993-t002">Table 2</a>), and therefore, the hydrogen-bonding capacity of Tyr157 is apparently not essential for catalysis. The lack of a suitable proton donor within the active site of EGS to catalyze the carbon-oxygen bond cleavage may account for the requirement of EGS for an esterified substrate: coniferyl alcohol carries a much poorer leaving group (free hydroxide ion) than its acetylated form (the resonance-stabilized acetate ion) (see <a href="#pone-0000993-g006">Figure 6C</a>). The absence of a well-defined binding site for the acetate, underscored by the finding that soaking of EGS-NADP<sup>+</sup> crystals with a high concentration (0.5 M) of sodium acetate yielded no ordered binding of acetate ion, may also contribute to the apparent irreversibility of the reaction that generates eugenol and acetate from coniferyl acetate.</p>
<a id="article1.body1.sec2.sec6.p8" name="article1.body1.sec2.sec6.p8"></a><p>In summary, a mechanistic scheme emerges in which binding of the coniferyl acetate substrate within the active site of EGS leads to deprotonation of the p-hydroxy group (PUSH) coupled with expulsion of acetate ion from the C1 substituent (PULL). The resultant extended quinone methide intermediate serves as a hydride acceptor at C7 to yield the product eugenol. For NAD(P)H-mediated reduction reactions, the acceptance of hydride by a substrate is typically accompanied by the acquistion of a proton at an adjacent atom, to maintain charge neutrality. As discussed above, EGS lacks an appropriately positioned proton donor near the site of hydride addition (C7); protonation instead occurs at the p-hydroxyl group, in concert with rearrangement of the double-bond system of the quinone methide and re-aromatization of the phenyl ring. Furthermore, the reduction of a double bond within a quinone methide intermediate more closely resembles a typical reaction catalyzed by a nicotinamide-cofactor enzyme, and indeed the reaction catalyzed by the PIP-family member IFR <a href="#pone.0000993-Fujita1">[10]</a>.</p>


<h4>Determinants of the regioselectivity of the EGS-catalyzed reduction reaction</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>In the structure of the EGS-NADP<sup>+</sup>-EMDF complex, the C4 atom of the cofactor's nicotinamide ring, which serves as the donor of the pro-R hydride, is directly apposed to the C7 atom of the inhibitor's side group (with an interatomic separation of 3.5 Å). Such a relative positioning of the nicotinamide ring and the side group of coniferyl acetate would appear to be ideal for hydride attack on the substrate C7-carbon and the consequent production of the expected allylphenylpropene, eugenol. In contrast, the production of the isophenylpropene, isoeugenol, would seemingly require hydride attack on the C9 carbon of the side group. Thus, a binding mode that more appropriately apposes the coniferyl-acetate C9 and nicotinamide C4 atoms would provide a possible means for conferring isoeugenol production from by the related enzyme, IGS from petunia. Further structural and mutagenic studies and analyses of additional, newly discovered IGS-type enzymes (Koeduka et al., submitted) are underway to probe the enzymic determinants of the regioselectivity of the EGS/IGS catalyzed reactions.</p>


<h4>Modeling of NADP(H) and substrate binding in other IFR-like proteins</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>The EGS residues that are involved in interactions with the NADP(H) cofactor (see above) are highly conserved in the other IFR-like enzymes, and thus these enzymes can be expected to maintain a cofactor-binding site very similar to that observed in EGS. Most of these other enzymes have also been characterized as A-type reductases. On the basis of the holo-EGS structures, the NADP(H) cofactor can be readily modeled into the apo-structures of the other IFR-like enzymes, although small, accommodating adjustments to the surrounding protein are necessary.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>The substrates of the PLR, PCBER and IFR enzymes all possess a phenyl ring with a C4-hydroxyl group, and the expected sites of hydride addition occur near the C7 atom of the substrate. For these enzymes, substrate binding can be modeled on the basis of the positioning of the guaiacol moiety of EMDF bound to EGS. As expected, in all cases, the substrate C7-atom is positioned very close to the C4 atom of the cofactor's nicotinamide ring. The substrate-binding pocket in general appears less enclosed in the other IFR-like proteins than in EGS, consistent perhaps with the larger size of the C1 substituents of the cognate substrates. The more open binding pockets are due primarily to the absence of the C-terminal tail that occurs in EGS, as well as the substitution of smaller residues within the active-site capping region.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>Moreover, the substrates for PLR and PCBER contain a cyclic-ether linkage adjacent to the site of hydride addition, and will therefore generate a transient alkoxide intermediate upon carbon-oxygen bond cleavage and ring opening. These enzymes may employ two means to promote protonation of the alkoxide intermediate: a potential proton donor or hydrogen-bonding group within the active site (His271 in PLR; Ser263 and Ser267 in PCBER), and a more open binding site that may allow access to the intermediate by bulk water.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Protein expression and purification</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>A DNA fragment encoding the entire amino-acid sequence (residues 1–314) of <em>Ocimum basilicum</em> EGS1 <a href="#pone.0000993-Koeduka1">[4]</a> was inserted between the NcoI and BamH1 sites of the expression vector pHIS8, which, under the control of a T7 promoter, yields the target protein fused to an N-terminal octahistidine tag <a href="#pone.0000993-Jez1">[21]</a>. For heterologous over-expression of the EGS protein, the plasmid pHIS8(EGS) was transformed into the expression host <em>E. coli</em> BL21(DE3) (Novagen). <em>E. coli</em> cultures in TB medium were grown at 37°C to an optical density (600 nm) of 1.5, induced with 1 mM isopropyl-β-D-thiogalactoside, and allowed to grow for an additional 6 hrs at 20°C. Bacterial cells were harvested by centrifugation, resuspended in lysis buffer (50 mM TrisHCl, pH 8.0; 0.5 M NaCl; 20 mM imidazole; 1% v/v Tween20; 10% v/v glycerol; and 20 mM 2-mercaptoethanol), and lysed by sonication. The EGS protein was isolated from the <em>E. coli</em> lysate by affinity chromatography with nickel-nitrilotriacetic-acid coupled agarose (Qiagen), and eluted with lysis buffer supplemented with 0.25 M imidazole. The partially purified EGS protein was treated with thrombin for cleavage of the octahistidine tag, and then further purified by gel-exclusion chromatography using a Superdex 200 HR26/60 column (Pharmacia Biosystems).</p>


<h4>Site-directed mutagenesis of <em>O. basilicum</em> EGS</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Site-directed mutants of the EGS gene were created in the plasmid pHIS8(EGS) with the PCR method <a href="#pone.0000993-Ho1">[22]</a>. The DNA sequences of the mutant constructs were confirmed by sequencing of the entire EGS insert in both the forward and reverse directions.</p>


<h4>Chemoenzymatic synthesis of coniferyl acetate ((E)-4-hydroxy-3-methoxycinnamylacetate)</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Coniferyl alcohol (180.2 mg, 1.0 mmol), <em>Candida antarctica</em> lipase B (CAL-B, 25 mg, Aldrich, ≥10,000 U/g), vinyl acetate (401 µL, 5.0 mmol), and dry diethyl ether (Et<sub>2</sub>O, 50 mL, 0.2 M) were stirred in a 125-mL Erlenmeyer flask at room temperature for 2 h (<a href="#pone-0000993-g007">Figure 7</a>). The reaction mixture was filtered through glass wool and concentrated <em>in vacuo</em>. The reaction yielded quantitatively the desired product (222 mg colorless oil). <sup>1</sup>H NMR (500 MHz, CDCl3) δ 6.91 (m, 3H), 6.59 (d, <em>J</em> = 15.92 Hz, 1H), 6.15 (dt, <em>J</em> = 6.62, 15.92 Hz, 1H), 4.72 (d, <em>J</em> = 6.62 Hz, 2H), 3.92 (s, 3H), 2.11 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl3) δ 171.2, 146.8, 146.1, 134.7, 129.0, 121.0, 120.9, 114.7, 108.6, 65.5, 56.1, 21.3. LCMS [M-H]<sup>-</sup> calculated for C<sub>12</sub>H<sub>13</sub>O<sub>4</sub>: 221.08, found: 221.2.</p>
<div class="figure" id="pone-0000993-g007"><div class="img"><a name="pone-0000993-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Chemoenzymatic synthesis of coniferyl acetate.</span></strong></p><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1"></a><p>Coniferyl acetate was obtained as a colorless oil with a yield of nearly 100%.</p>
<span>doi:10.1371/journal.pone.0000993.g007</span></div>

<h4>Chemical synthesis of (7S,8S)-ethyl (7,8-methylene)-dihydroferulate (EMDF)</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>The compound (7S,8S)-ethyl (7,8-methylene)-dihydroferulate (EMDF, (1S,2S)-ethyl 2-(4-hydroxy-3-methoxyphenyl) cyclo-propanecarboxylate) was synthesized in three steps (<a href="#pone-0000993-g008">Figure 8</a>).</p>
<div class="figure" id="pone-0000993-g008"><div class="img"><a name="pone-0000993-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Chemical synthesis of (7S,8S)-ethyl (7,8-methylene)-dihydroferulate (EMDF).</span></strong></p><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1"></a><p>EMDF was obtained with an overall yield of 10%.</p>
<span>doi:10.1371/journal.pone.0000993.g008</span></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p><em>4-((2-methoxyethoxy)methoxy)-3-methoxy benzaldehyde (step 1).</em> K<sub>2</sub>CO<sub>3</sub> (1.38 g, 10.0 mmol, 1.0 eq.) was added to solution of vanillin (1.52 g, 10.0 mmol, 1.0 eq.) in dry acetone (70 mL, 0.15 M), and the mixture was stirred under argon at 0°C for 30 min. Then MeOCH<sub>2</sub>CH<sub>2</sub>OCH<sub>2</sub>Cl (MEM-Cl, 1.49 g, 12.0 mmol, 1.2 eq.) was added drop wise, and the mixture was stirred an additional 5 h. The mixture was then concentrated <em>in vacuo</em> to ~20 mL, combined with 20 mL of water, and extracted with diethyl ether (3×25 mL). The organic layers were pooled, washed with brine, dried with MgSO<sub>4</sub>, filtered and concentrated <em>in vacuo</em>. The resulting yellow oil (2.38 g, 98% yield) was used without further purification. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.86 (s, 1H), 7.43 (s, 1H), 7.42 (d, <em>J</em> = 8.5 Hz, 2H), 7.32 (d, <em>J</em> = 8.5 Hz, 2H), 5.41 (s, 2H), 3.93 (s, 3H), 3.86 (t, <em>J</em> = 4.7 Hz, 2H), 3.54 (t, <em>J</em> = 4.7 Hz, 2H), 3.35 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 191.0, 152.0, 150.0, 131.1, 126.5, 114.8, 109.4, 94.0, 71.4, 68.2, 59.0, 56.0; LCMS [M+Na]<sup>+</sup> calculated for C<sub>12</sub>H<sub>16</sub>O<sub>5</sub>Na: 263.09, found: 263.3.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p><em>1-((2-methoxyethoxy)methoxy)-2-methoxy-4​-vinylbenzene(step 2).</em> To a suspension of methyltriphenylphosphonium bromide (MeP(Ph)<sub>3</sub>Br, 3.57 g, 10.0 mmol, 2.0 eq.) in anhydrous tetrahydrofuran (THF, 30 mL, 0.33 M), under argon at room temperature, was added potassium bis-trimethylsilylamide (KN(TMS)<sub>2</sub>, 4.5 mL, 9.0 mmol, 1.8 eq., 0.5 M in toluene). After 30 min, a solution containing 4-((2-methoxyethoxy)methoxy)-3-methoxy benzaldehyde (1.20 g, 5.0 mmol, 1.0 eq.) was added to the yellow colored ylide solution via cannula. The reaction was completed within 30–60 min (as monitored by TLC, hexanes/ethyl acetate, 7:3). The reaction mixture was quenched with saturated NH<sub>4</sub>Cl and extracted with diethyl ether (3×100 mL). The organic layers were combined, washed with water (2×100 mL), brine (2×100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, concentrated <em>in vacuo</em>, and subjected to Dry Column Vacuum Chromatography (90% hexanes/ethyl acetate) to afford a colorless oil (2.01 g, 84% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.15 (d, <em>J</em> = 8.3 Hz, 1H), 6.96 (s, 1H), 6.93 (d, <em>J</em> = 8.3 Hz, 1H), 6.65 (dd, <em>J</em> = 10.9, 17.5 Hz, 1H), 5.63 (d, <em>J</em> = 17.4 Hz, 1H), 5.32 (s, 2H), 5.17 (d, <em>J</em> = 10.9 Hz, 1H), 3.90 (s, 3H), 3.87 (t, <em>J</em> = 4.7 Hz, 2H), 3.56 (t, <em>J</em> = 4.7 Hz, 2H), 3.37 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 149.7, 146.4, 136.4, 132.3, 119.4, 116.2, 112.4, 109.1, 94.4, 71.5, 67.8, 59.0, 55.8; LCMS [M+Na]<sup>+</sup> calculated for C<sub>13</sub>H<sub>18</sub>O<sub>4</sub>Na: 261.1, found: 260.9.</p>
<a id="article1.body1.sec3.sec4.p4" name="article1.body1.sec3.sec4.p4"></a><p><em>(1S,2S)-ethyl 2-(4-hydroxy-3-methoxyphenyl) cyclo-propanecarboxylate (step 3).</em> To a solution of copper-ligand complex prepared from Cu<sup>I</sup>OTf (toluene)<sub>0.5 </sub>(129.3 mg, 0.25 mmol, 2.0 mol%) and ligand <strong>(A)</strong> (75.3 mg, 0.25 mmol, 2.0 mol%) in 50 mL of dry dichloromethane (DCM) was added 1-((2-methoxyethoxy)methoxy)-2-methoxy-4​-vinylbenzene(1.0 g, 4.2 mmol, 1.0 eq.). Ethyl diazoacetate (CHN<sub>2</sub>CO<sub>2</sub>Et, 485.8 µL, 4.62 mmol, 1.1 eq.) was dissolved in 0.5 mL dry dichloromethane and added to the stirring reaction mixture by slow addition via syringe pump over 10 h. The crude reaction mixture was filtered through a plug of silica gel and concentrated <em>in vacuo</em>. <sup>1</sup>H NMR and LCMS showed a complex product mixture consisting of 80% conversion to cyclopropane products with a <em>trans</em> to <em>cis</em> ratio of 2.5:1. Chiral HPLC analysis (Chiralcel-OD) determined 95% <em>ee</em> for the <em>trans</em> product. In addition to the MEM-protected trans- and cis-cyclopropane products, the crude reaction mixture contained the desired final product, the deprotected trans-cyclopropane, (1S,2S)-ethyl 2-(4-hydroxy-3-methoxyphenyl) cyclo-propanecarboxylate (EMDF), which was isolated by column chromatography (90% hexanes/ethyl acetate → 100% ethyl acetate) (99.2 mg as a colorless oil, 10% yield). Subsequent X-ray structure analysis of EMDF complexed with EGS confirmed the compound's absolute stereochemistry. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 6.82 (d, <em>J</em> = 8.2 Hz, 1H), 6.64 (s, 1H), 6.59 (d, <em>J</em> = 8.2 Hz, 1H), 5.56, ( br s, Ar-OH, 1H), 4.17 (q, <em>J</em> = 7.0 Hz, 2H), 3.87 (s, 3H), 2.47 (ddd, <em>J</em> = 4.6, 6.4, 9.6 Hz, 1H), 1.82 (ddd, <em>J</em> = 4.6, 5.6, 8.8 Hz, 1H), 1.54 (ddd, <em>J</em> = 4.8, 5.6, 9.2 Hz, 1H), 1.28 (t, <em>J</em> = 7.1 Hz, 3H)1.25 (ddd, buried, 1H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 173.6, 146.5, 144.3, 131.9, 118.8, 114.3, 109.4, 60.7, 55.9, 26.1, 23.8, 16.7, 14.3; LCMS [M-H]<sup>-</sup> calculated for C<sub>13</sub>H<sub>15</sub>O<sub>4</sub>Na: 235.1, found: 234.8.</p>


<h4>EGS enzyme assay</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>EGS enzyme activity was measured by gas chromatography/mass spectrometry as described previously <a href="#pone.0000993-Koeduka1">[4]</a>. The assay mixture (total volume 0.15 mL) contained 0.05 M MES-KOH (pH 6.5), 1 mM NADPH, 1 mM coniferyl acetate, and 2 µg of EGS. Reaction mixtures were incubated at 25°C for 15 min followed by extraction with 1 mL of hexane. For determination of the specific activities of crude preparations of EGS, enzyme concentrations were assessed from western blots with an EGS antibody. For detailed kinetic analyses, substrate concentrations ranged from 0.1 to 5.0 mM, and for EMDF-inhibition determinations, the inhibitor concentrations used were 0, 0.4 and 0.8 mM (<a href="#pone-0000993-g009">Figure 9</a>).</p>
<div class="figure" id="pone-0000993-g009"><div class="img"><a name="pone-0000993-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000993" data-uri="info:doi/10.1371/journal.pone.0000993.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000993.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000993.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000993.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000993.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Double reciprocal plot for EGS activity in the presence and absence of EMDF.</span></strong></p><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1"></a><p>The plots illustrate the mixed nature of the EMDF EGS inhibitor. Ki was estimated from nonlinear fitting to a modified Michaelis-Menten equation.</p>
<span>doi:10.1371/journal.pone.0000993.g009</span></div>

<h4>Crystallization of basil EGS</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Wild-type EGS from basil (<em>Ocimum basilicum</em>) in complex with NADP<sup>+</sup> was crystallized at 4°C from buffered solutions of protein mixed with polyethylene glycol (PEG) and a salt. The typical crystallization solutions employed were 0.1 M sodium succinate (pH 5.5), 5 mM NADP<sup>+</sup> (Sigma Aldrich), 0.3 M KCl, 2 mM dithiothreitol and 21% (w/v) PEG 3350; or 0.1 M MOPSO (pH 6.5-7.0), 5 mM NADP<sup>+</sup>, 0.3 M KNO<sub>3</sub>, 2 mM dithiothreitol and 28% (w/v) PEG monomethylether 5000. These conditions yielded a number of distinct (but related) crystal forms. Morphologically, all of the crystal forms grew as thin plates. The two most commonly observed forms were monoclinic (space group <em>P</em>2<sub>1</sub>, with unit-cell parameters a = 53.8, b = 85.9, c = 76.2 Å, β = 107.3°) and orthorhombic (space group <em>P</em>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, with unit-cell dimensions a = 79.3, b = 85.9, c = 99.2 Å), and both diffracted X-rays to high resolution (typically 1.6 to 2.0 Å). Similar crystallization conditions were also employed for wild-type EGS complexed with NADPH (the reduced form of the cofactor) or without added cofactor (apo-EGS), and for Lys132-mutant EGS proteins complexed with NADP<sup>+</sup>. (It should be noted that the apo-EGS protein likely contained a small amount of nicotinamide cofactor incorporated during expression in <em>E. coli</em>.) Micro-seeding was useful for promoting crystal growth of the mutant forms of EGS. Ternary complexes of EGS, NADP<sup>+</sup> and the EGS inhibitor EMDF were obtained by soaking EGS/NADP<sup>+</sup> crystals in crystallization solution supplemented with 5–10 mM EMDF.</p>


<h4>X-ray diffraction data</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>Crystals were transferred briefly to a cryoprotectant solution (consisting of reservoir solution supplemented with 17–20% v/v glycerol) prior to immersion in liquid nitrogen. X-ray diffraction data were measured from frozen crystals at beamlines 8.2.1 and 8.2.2 of the Advanced Light Source (Lawrence Berkeley National Laboratory) on ADSC Quantum 210 or 315 CCD detectors. Diffraction intensities were indexed, integrated and scaled with the programs XDS and XSCALE <a href="#pone.0000993-Kabsch1">[23]</a> or Mosflm <a href="#pone.0000993-Leslie1">[24]</a> and Scala <a href="#pone.0000993-Evans1">[25]</a>.</p>


<h4>X-ray structure determination of EGS</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>Initial crystallographic phases were determined for the orthorhombic crystal form of the EGS/NADP<sup>+</sup> complex through molecular-replacement (MR) with the program Molrep <a href="#pone.0000993-Vagin1">[26]</a>. A homology model for EGS was constructed with the program Modeller <a href="#pone.0000993-Sali1">[27]</a> based on the structure of phenylcoumaran benzylic ether reductase (PCBER, PDB entry 1QYC <a href="#pone.0000993-Min1">[6]</a>). The program ARP/wARP was used for automated rebuilding of the initial structural model against a two-fold, non-crystallographic-symmetry averaged map. Subsequent structural refinement used the program CNS <a href="#pone.0000993-Brunger1">[28]</a>. Xfit <a href="#pone.0000993-McRee1">[29]</a> was used for graphical map inspection and manual rebuilding of the atomic model. Programs from the CCP4 <a href="#pone.0000993-Collaborative1">[30]</a> suite were employed for all other crystallographic calculations. Structural depictions were generated with the program Pymol (Delano Scientific, San Carlos, CA).</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the staff of the Advanced Light Source, Berkeley, California for assistance during data collection on beamlines 8.2.1 and 8.2.2.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JN EP GL TB TK. Performed the experiments: GL TB MB JT SS TK. Analyzed the data: JN EP GL TB MB JT SS TK. Contributed reagents/materials/analysis tools: JN EP GL TB MB SS TK. Wrote the paper: JN EP GL TB.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000993-Gang1" id="pone.0000993-Gang1"></a>Gang DR, Wang J, Dudareva N, Nam KH, Simon JE, et al.  (2001) An investigation of the storage and biosynthesis of phenylpropenes in sweet basil. Plant Physiol  125: 539–555.  <ul class="find" data-citedArticleID="1001460" data-doi="10.1104/pp.125.2.539"><li><a href="http://dx.doi.org/10.1104/pp.125.2.539" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+investigation+of+the+storage+and+biosynthesis+of+phenylpropenes+in+sweet+basil." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+investigation+of+the+storage+and+biosynthesis+of+phenylpropenes+in+sweet+basil.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000993-Pichersky1" id="pone.0000993-Pichersky1"></a>Pichersky E, Noel JP, Dudareva N (2006) Biosynthesis of plant volatiles: nature's diversity and ingenuity. Science  311: 808–811.  <ul class="find" data-citedArticleID="1001484" data-doi="10.1126/science.1118510"><li><a href="http://dx.doi.org/10.1126/science.1118510" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biosynthesis+of+plant+volatiles%3A+nature%27s+diversity+and+ingenuity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biosynthesis+of+plant+volatiles%3A+nature%27s+diversity+and+ingenuity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000993-Dexter1" id="pone.0000993-Dexter1"></a>Dexter R, Qualley A, Kish CM, Ma CJ, Koeduka T, et al.  (2007) Characterization of a petunia acetyltransferase involved in the biosynthesis of the floral volatile isoeugenol. Plant J  49: 265–275.  <ul class="find" data-citedArticleID="1001452" data-doi="10.1111/j.1365-313x.2006.02954.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02954.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+of+a+petunia+acetyltransferase+involved+in+the+biosynthesis+of+the+floral+volatile+isoeugenol." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+of+a+petunia+acetyltransferase+involved+in+the+biosynthesis+of+the+floral+volatile+isoeugenol.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000993-Koeduka1" id="pone.0000993-Koeduka1"></a>Koeduka T, Fridman E, Gang DR, Vassao DG, Jackson BL, et al.  (2006) Eugenol and isoeugenol, characteristic aromatic constituents of spices, are biosynthesized via reduction of a coniferyl alcohol ester. Proc Natl Acad Sci  103: 10128–10133.  <ul class="find" data-citedArticleID="1001472" data-doi="10.1073/pnas.0603732103"><li><a href="http://dx.doi.org/10.1073/pnas.0603732103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eugenol+and+isoeugenol%2C+characteristic+aromatic+constituents+of+spices%2C+are+biosynthesized+via+reduction+of+a+coniferyl+alcohol+ester." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eugenol+and+isoeugenol%2C+characteristic+aromatic+constituents+of+spices%2C+are+biosynthesized+via+reduction+of+a+coniferyl+alcohol+ester.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000993-Vassao1" id="pone.0000993-Vassao1"></a>Vassao DG, Gang DR, Koeduka T, Jackson B, Pichersky E, et al.  (2006) Chavicol formation in sweet basil (<em>Ocimum basilicum</em>): cleavage of an esterified C9 hydroxyl group with NAD(P)H-dependent reduction. Org Biomol Chem  4: 2733–2744.  <ul class="find" data-citedArticleID="1001500" data-doi="10.1039/b605407b"><li><a href="http://dx.doi.org/10.1039/b605407b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chavicol+formation+in+sweet+basil+%28Ocimum+basilicum%29%3A+cleavage+of+an+esterified+C9+hydroxyl+group+with+NAD%28P%29H-dependent+reduction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chavicol+formation+in+sweet+basil+%28Ocimum+basilicum%29%3A+cleavage+of+an+esterified+C9+hydroxyl+group+with+NAD%28P%29H-dependent+reduction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000993-Min1" id="pone.0000993-Min1"></a>Min T, Kasahara H, Bedgar DL, Youn B, Lawrence PK, et al.  (2003) Crystal structures of pinoresinol-lariciresinol and phenylcoumaran benzylic ether reductases and their relationship to isoflavone reductases. J Biol Chem  278: 50714–50723.  <ul class="find" data-citedArticleID="1001482" data-doi="10.1074/jbc.m308493200"><li><a href="http://dx.doi.org/10.1074/jbc.m308493200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structures+of+pinoresinol-lariciresinol+and+phenylcoumaran+benzylic+ether+reductases+and+their+relationship+to+isoflavone+reductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structures+of+pinoresinol-lariciresinol+and+phenylcoumaran+benzylic+ether+reductases+and+their+relationship+to+isoflavone+reductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000993-Wang1" id="pone.0000993-Wang1"></a>Wang X, He X, Lin J, Shao H, Chang Z, et al.  (2006) Crystal structure of isoflavone reductase from alfalfa (Medicago sativa L.). J Mol Biol  358: 1341–1352.  <ul class="find" data-citedArticleID="1001502" data-doi="10.1016/j.jmb.2006.03.022"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.03.022" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+isoflavone+reductase+from+alfalfa+%28Medicago+sativa+L.%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+isoflavone+reductase+from+alfalfa+%28Medicago+sativa+L.%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000993-Tanner1" id="pone.0000993-Tanner1"></a>Tanner GJ, Francki KT, Abrahams S, Watson JM, Larkin PJ, et al.  (2003) Proanthocyanidin biosynthesis in plants. Purification of legume leucoanthocyanidin reductase and molecular cloning of its cDNA. J Biol Chem  278: 31647–31656.  <ul class="find" data-citedArticleID="1001492" data-doi="10.1074/jbc.m302783200"><li><a href="http://dx.doi.org/10.1074/jbc.m302783200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Proanthocyanidin+biosynthesis+in+plants.+Purification+of+legume+leucoanthocyanidin+reductase+and+molecular+cloning+of+its+cDNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Proanthocyanidin+biosynthesis+in+plants.+Purification+of+legume+leucoanthocyanidin+reductase+and+molecular+cloning+of+its+cDNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000993-Akashi1" id="pone.0000993-Akashi1"></a>Akashi T, Koshimizu S, Aoki T, Ayabe S (2006) Identification of cDNAs encoding pterocarpan reductase involved in isoflavan phytoalexin biosynthesis in <em>Lotus japonicus</em> by EST mining. FEBS Lett  580: 5666–5670.  <ul class="find" data-citedArticleID="1001444" data-doi="10.1016/j.febslet.2006.09.016"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.09.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+cDNAs+encoding+pterocarpan+reductase+involved+in+isoflavan+phytoalexin+biosynthesis+in+Lotus+japonicus+by+EST+mining." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+cDNAs+encoding+pterocarpan+reductase+involved+in+isoflavan+phytoalexin+biosynthesis+in+Lotus+japonicus+by+EST+mining.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000993-Fujita1" id="pone.0000993-Fujita1"></a>Fujita M, Gang DR, Davin LB, Lewis NG (1999) Recombinant pinoresinol-lariciresinol reductases from western red cedar (<em>Thuja plicata</em>) catalyze opposite enantiospecific conversions. J Biol Chem  274: 618–627.  <ul class="find" data-citedArticleID="1001458" data-doi="10.1074/jbc.274.2.618"><li><a href="http://dx.doi.org/10.1074/jbc.274.2.618" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recombinant+pinoresinol-lariciresinol+reductases+from+western+red+cedar+%28Thuja+plicata%29+catalyze+opposite+enantiospecific+conversions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recombinant+pinoresinol-lariciresinol+reductases+from+western+red+cedar+%28Thuja+plicata%29+catalyze+opposite+enantiospecific+conversions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000993-Gang2" id="pone.0000993-Gang2"></a>Gang DR, Kasahara H, Xia ZQ, Vander Mijnsbrugge K, Bauw G, et al.  (1999) Evolution of plant defense mechanisms. Relationships of phenylcoumaran benzylic ether reductases to pinoresinol-lariciresinol and isoflavone reductases. J Biol Chem  274: 7516–7527.  <ul class="find" data-citedArticleID="1001462" data-doi="10.1074/jbc.274.11.7516"><li><a href="http://dx.doi.org/10.1074/jbc.274.11.7516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+plant+defense+mechanisms.+Relationships+of+phenylcoumaran+benzylic+ether+reductases+to+pinoresinol-lariciresinol+and+isoflavone+reductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+plant+defense+mechanisms.+Relationships+of+phenylcoumaran+benzylic+ether+reductases+to+pinoresinol-lariciresinol+and+isoflavone+reductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000993-Carugo1" id="pone.0000993-Carugo1"></a>Carugo O, Argos P (1997) NADP-dependent enzymes. I: Conserved stereochemistry of cofactor binding. Proteins  28: 10–28.  <ul class="find" data-citedArticleID="1001448" data-doi="10.1002/(sici)1097-0134(199705)28:1<10::aid-prot2&gt;3.3.co;2-r"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0134(199705)28:1<10::aid-prot2&gt;3.3.co;2-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NADP-dependent+enzymes.+I%3A+Conserved+stereochemistry+of+cofactor+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NADP-dependent+enzymes.+I%3A+Conserved+stereochemistry+of+cofactor+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000993-Kallberg1" id="pone.0000993-Kallberg1"></a>Kallberg Y, Oppermann U, Jornvall H, Persson B (2002) Short-chain dehydrogenases/reductases (SDRs). Eur J Biochem  269: 4409–17.  <ul class="find" data-citedArticleID="1001470" data-doi="10.1046/j.1432-1033.2002.03130.x"><li><a href="http://dx.doi.org/10.1046/j.1432-1033.2002.03130.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Short-chain+dehydrogenases%2Freductases+%28SDRs%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Short-chain+dehydrogenases%2Freductases+%28SDRs%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000993-Thoden1" id="pone.0000993-Thoden1"></a>Thoden JB, Hegeman AD, Wesenberg G, Chapeau MC, Frey PA, et al.  (1997) Structural analysis of UDP-sugar binding to UDP-galactose 4-epimerase from Escherichia coli. Biochemistry  36: 6294–6304.  <ul class="find" data-citedArticleID="1001494" data-doi="10.1021/bi970025j"><li><a href="http://dx.doi.org/10.1021/bi970025j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+analysis+of+UDP-sugar+binding+to+UDP-galactose+4-epimerase+from+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+analysis+of+UDP-sugar+binding+to+UDP-galactose+4-epimerase+from+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000993-Shi1" id="pone.0000993-Shi1"></a>Shi R, Lin S-X (2004) Cofactor hydrogen bonding onto the protein main chain is conserved in the short chain dehydrogenase/reductase family and contributes to nicotinamide orientation. J Biol Chem  279: 16778–16785.  <ul class="find" data-citedArticleID="1001488" data-doi="10.1074/jbc.m313156200"><li><a href="http://dx.doi.org/10.1074/jbc.m313156200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cofactor+hydrogen+bonding+onto+the+protein+main+chain+is+conserved+in+the+short+chain+dehydrogenase%2Freductase+family+and+contributes+to+nicotinamide+orientation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cofactor+hydrogen+bonding+onto+the+protein+main+chain+is+conserved+in+the+short+chain+dehydrogenase%2Freductase+family+and+contributes+to+nicotinamide+orientation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000993-Tanaka1" id="pone.0000993-Tanaka1"></a>Tanaka M, Bateman R, Rauh D, Vaisberg E, Ramachandani S, et al.  (2005) An unbiased cell morphology-based screen for new, biologically active small molecules. PLoS Biol  3: 128–135.  <ul class="find" data-citedArticleID="1001490" data-doi="10.1371/journal.pbio.0030128"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030128" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+unbiased+cell+morphology-based+screen+for+new%2C+biologically+active+small+molecules." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+unbiased+cell+morphology-based+screen+for+new%2C+biologically+active+small+molecules.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000993-Filling1" id="pone.0000993-Filling1"></a>Filling C, Berndt KD, Benach J, Knapp S, Prozorovski T, et al.  (2002) Critical residues for structure and catalysis in short-chain dehydrogenases/reductases. J Biol Chem  277: 25677–25684.  <ul class="find" data-citedArticleID="1001456" data-doi="10.1074/jbc.m202160200"><li><a href="http://dx.doi.org/10.1074/jbc.m202160200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Critical+residues+for+structure+and+catalysis+in+short-chain+dehydrogenases%2Freductases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Critical+residues+for+structure+and+catalysis+in+short-chain+dehydrogenases%2Freductases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000993-Kolappan1" id="pone.0000993-Kolappan1"></a>Kolappan S, Zwahlen J, Zhou R, Truglio JJ, Tonge PJ, et al.  (2007) Lysine 190 is the catalytic base in MenF, the menaquinone-specific isochorismate synthase from <em>Escherichia coli</em>: implications for an enzyme family. Biochemistry  46: 946–953.  <ul class="find" data-citedArticleID="1001474" data-doi="10.1021/bi0608515"><li><a href="http://dx.doi.org/10.1021/bi0608515" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Lysine+190+is+the+catalytic+base+in+MenF%2C+the+menaquinone-specific+isochorismate+synthase+from+Escherichia+coli%3A+implications+for+an+enzyme+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Lysine+190+is+the+catalytic+base+in+MenF%2C+the+menaquinone-specific+isochorismate+synthase+from+Escherichia+coli%3A+implications+for+an+enzyme+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000993-Koumanov1" id="pone.0000993-Koumanov1"></a>Koumanov A, Benach J, Atrian S, Gonzalez-Duarte R, Karshikoff A, et al.  (2003) The catalytic mechanism of <em>Drosophila</em> alcohol dehydrogenase: evidence for a proton relay modulated by the coupled ionization of the active site lysine/tyrosine pair and a NAD+ ribose OH switch. Proteins  51: 289–98.  <ul class="find" data-citedArticleID="1001476" data-doi="10.1002/prot.10354"><li><a href="http://dx.doi.org/10.1002/prot.10354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+catalytic+mechanism+of+Drosophila+alcohol+dehydrogenase%3A+evidence+for+a+proton+relay+modulated+by+the+coupled+ionization+of+the+active+site+lysine%2Ftyrosine+pair+and+a+NAD%2B+ribose+OH+switch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+catalytic+mechanism+of+Drosophila+alcohol+dehydrogenase%3A+evidence+for+a+proton+relay+modulated+by+the+coupled+ionization+of+the+active+site+lysine%2Ftyrosine+pair+and+a+NAD%2B+ribose+OH+switch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000993-Toteva1" id="pone.0000993-Toteva1"></a>Toteva MM, Moran M, Amyes TL, Richard JP (2003) Substituent effects on carbocation stability: the pK(R) for p-quinone methide. J Am Chem Soc  125: 8814–8819.  <ul class="find" data-citedArticleID="1001496" data-doi="10.1021/ja029588v"><li><a href="http://dx.doi.org/10.1021/ja029588v" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Substituent+effects+on+carbocation+stability%3A+the+pK%28R%29+for+p-quinone+methide." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Substituent+effects+on+carbocation+stability%3A+the+pK%28R%29+for+p-quinone+methide.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000993-Jez1" id="pone.0000993-Jez1"></a>Jez JM, Ferrer JL, Bowman ME, Dixon RA, Noel JP (2000) Dissection of malonyl-coenzyme A decarboxylation from polyketide formation in the reaction mechanism of a plant polyketide synthase. Biochemistry  39: 890–902.  <ul class="find" data-citedArticleID="1001466" data-doi="10.1021/bi991489f"><li><a href="http://dx.doi.org/10.1021/bi991489f" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissection+of+malonyl-coenzyme+A+decarboxylation+from+polyketide+formation+in+the+reaction+mechanism+of+a+plant+polyketide+synthase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissection+of+malonyl-coenzyme+A+decarboxylation+from+polyketide+formation+in+the+reaction+mechanism+of+a+plant+polyketide+synthase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000993-Ho1" id="pone.0000993-Ho1"></a>Ho SN, Hunt HD, Horton RM, Pullen JK, Pease LR (1989) Site-directed mutagenesis by overlap extension using the polymerase chain reaction. Gene  77: 51–59.  <ul class="find" data-citedArticleID="1001464" data-doi="10.1016/0378-1119(89)90358-2"><li><a href="http://dx.doi.org/10.1016/0378-1119(89)90358-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Site-directed+mutagenesis+by+overlap+extension+using+the+polymerase+chain+reaction." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Site-directed+mutagenesis+by+overlap+extension+using+the+polymerase+chain+reaction.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000993-Kabsch1" id="pone.0000993-Kabsch1"></a>Kabsch W (1993) Automatic processing of rotation diffraction data from crystals of initially unknown symmetry and cell constants. J Appl Crystallog  26: 795–800.  <ul class="find" data-citedArticleID="1001468" data-doi="10.1107/s0021889893005588"><li><a href="http://dx.doi.org/10.1107/s0021889893005588" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Automatic+processing+of+rotation+diffraction+data+from+crystals+of+initially+unknown+symmetry+and+cell+constants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Automatic+processing+of+rotation+diffraction+data+from+crystals+of+initially+unknown+symmetry+and+cell+constants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000993-Leslie1" id="pone.0000993-Leslie1"></a>Leslie AG (2006) The integration of macromolecular diffraction data. Acta Crystallogr  D62: 48–57.  <ul class="find" data-citedArticleID="1001478" data-doi="10.1107/s0907444905039107"><li><a href="http://dx.doi.org/10.1107/s0907444905039107" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+integration+of+macromolecular+diffraction+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+integration+of+macromolecular+diffraction+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000993-Evans1" id="pone.0000993-Evans1"></a>Evans P (2006) Scaling and assessment of data quality. Acta Crystallogr  D62: 72–82.  <ul class="find" data-citedArticleID="1001454" data-doi="10.1107/s0907444905036693"><li><a href="http://dx.doi.org/10.1107/s0907444905036693" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Scaling+and+assessment+of+data+quality." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Scaling+and+assessment+of+data+quality.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000993-Vagin1" id="pone.0000993-Vagin1"></a>Vagin A, Teplyakov A (1997) MOLREP: an automated program for molecular replacement. J Appl Crystallog  30: 1022–1025.  <ul class="find" data-citedArticleID="1001498" data-doi="10.1107/s0021889897006766"><li><a href="http://dx.doi.org/10.1107/s0021889897006766" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MOLREP%3A+an+automated+program+for+molecular+replacement." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MOLREP%3A+an+automated+program+for+molecular+replacement.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000993-Sali1" id="pone.0000993-Sali1"></a>Sali A, Blundell TL (1993) Comparative protein modeling by satisfaction of spatial restraints. J Mol Biol  234: 779–815.  <ul class="find" data-citedArticleID="1001486" data-doi="10.1006/jmbi.1993.1626"><li><a href="http://dx.doi.org/10.1006/jmbi.1993.1626" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+protein+modeling+by+satisfaction+of+spatial+restraints." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+protein+modeling+by+satisfaction+of+spatial+restraints.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000993-Brunger1" id="pone.0000993-Brunger1"></a>Brunger AT, Warren GL (1998) Crystallography and NMR system: a new software suite for macromolecular structure determination. Acta Crystallog  D54: 905–921.  <ul class="find" data-citedArticleID="1001446" data-doi="10.1107/s0907444998003254"><li><a href="http://dx.doi.org/10.1107/s0907444998003254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystallography+and+NMR+system%3A+a+new+software+suite+for+macromolecular+structure+determination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystallography+and+NMR+system%3A+a+new+software+suite+for+macromolecular+structure+determination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000993-McRee1" id="pone.0000993-McRee1"></a>McRee DE (1999) XtalView/Xfit: a versatile program for manipulating atomic coordinates and electron density. J Struct Biol  125: 156–165.  <ul class="find" data-citedArticleID="1001480" data-doi="10.1006/jsbi.1999.4094"><li><a href="http://dx.doi.org/10.1006/jsbi.1999.4094" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=XtalView%2FXfit%3A+a+versatile+program+for+manipulating+atomic+coordinates+and+electron+density." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22XtalView%2FXfit%3A+a+versatile+program+for+manipulating+atomic+coordinates+and+electron+density.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000993-Collaborative1" id="pone.0000993-Collaborative1"></a>Collaborative Computational Project Number 4 (1994) The CCP4 suite: programs for protein crystallography. Acta Crystallog  D50: 760–763.  <ul class="find" data-citedArticleID="1001450" data-doi="10.1107/s0907444994003112"><li><a href="http://dx.doi.org/10.1107/s0907444994003112" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+CCP4+suite%3A+programs+for+protein+crystallography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+CCP4+suite%3A+programs+for+protein+crystallography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.XML" value="102932"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.PDF" value="607556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g001.PNG_L" value="577196"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g001.PNG_M" value="71688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g001.PNG_S" value="8779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g001.TIF" value="1020660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g001.PNG_I" value="26545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t001.PNG_L" value="246231"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t001.PNG_M" value="149363"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t001.PNG_S" value="14886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t001.TIF" value="1107384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t001.PNG_I" value="45909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g002.PNG_L" value="3569143"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g002.PNG_M" value="188367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g002.PNG_S" value="10288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g002.TIF" value="4977802"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g002.PNG_I" value="60780"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g003.PNG_L" value="1373726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g003.PNG_M" value="202774"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g003.PNG_S" value="11549"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g003.TIF" value="1953280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g003.PNG_I" value="64096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g004.PNG_L" value="2667750"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g004.PNG_M" value="175455"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g004.PNG_S" value="19000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g004.TIF" value="4097278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g004.PNG_I" value="157113"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g005.PNG_L" value="4776679"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g005.PNG_M" value="237081"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g005.PNG_S" value="16747"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g005.TIF" value="6904210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g005.PNG_I" value="111490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t002.PNG_L" value="73958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t002.PNG_M" value="96447"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t002.PNG_S" value="13528"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t002.TIF" value="374882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.t002.PNG_I" value="33108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g006.PNG_L" value="1008475"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g006.PNG_M" value="98217"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g006.PNG_S" value="15349"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g006.TIF" value="1799882"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g006.PNG_I" value="106778"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g007.PNG_L" value="49729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g007.PNG_M" value="19534"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g007.PNG_S" value="4860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g007.TIF" value="83000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g007.PNG_I" value="6968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g008.PNG_L" value="152663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g008.PNG_M" value="48570"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g008.PNG_S" value="7744"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g008.TIF" value="258566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g008.PNG_I" value="16763"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g009.PNG_L" value="220061"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g009.PNG_M" value="61292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g009.PNG_S" value="8383"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g009.TIF" value="536852"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000993.g009.PNG_I" value="23753"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000993&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000993" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000993&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000993&volume=&issue=&title=Structure and Reaction Mechanism of Basil Eugenol Synthase&author_name=Gordon%20V.%20Louie%2C%20Thomas%20J.%20Baiga%2C%20Marianne%20E.%20Bowman%2C%20Takao%20Koeduka%2C%20John%20H.%20Taylor%2C%20Snejina%20M.%20Spassova%2C%20Eran%20Pichersky%2C%20Joseph%20P.%20Noel&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000993" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993&amp;t=Structure%20and%20Reaction%20Mechanism%20of%20Basil%20Eugenol%20Synthase" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993&title=Structure%20and%20Reaction%20Mechanism%20of%20Basil%20Eugenol%20Synthase&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993&amp;title=Structure%20and%20Reaction%20Mechanism%20of%20Basil%20Eugenol%20Synthase" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000993&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Structure and Reaction Mechanism of Basil Eugenol Synthase';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000993';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Structure%20and%20Reaction%20Mechanism%20of%20Basil%20Eugenol%20Synthase http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000993" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000993" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alcohols%22" title="Search for articles in the subject area:'Alcohols'"><div class="flagText">Alcohols</div></a>
              <div data-categoryid="32035" data-articleid="24820"
                   data-categoryname="Alcohols"
                   class="flagImage" title="Flag 'Alcohols' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cofactors+%28biochemistry%29%22" title="Search for articles in the subject area:'Cofactors (biochemistry)'"><div class="flagText">Cofactors (biochemistry)</div></a>
              <div data-categoryid="31901" data-articleid="24820"
                   data-categoryname="Cofactors (biochemistry)"
                   class="flagImage" title="Flag 'Cofactors (biochemistry)' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Crystal+structure%22" title="Search for articles in the subject area:'Crystal structure'"><div class="flagText">Crystal structure</div></a>
              <div data-categoryid="17301" data-articleid="24820"
                   data-categoryname="Crystal structure"
                   class="flagImage" title="Flag 'Crystal structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzyme+structure%22" title="Search for articles in the subject area:'Enzyme structure'"><div class="flagText">Enzyme structure</div></a>
              <div data-categoryid="18161" data-articleid="24820"
                   data-categoryname="Enzyme structure"
                   class="flagImage" title="Flag 'Enzyme structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Enzymes%22" title="Search for articles in the subject area:'Enzymes'"><div class="flagText">Enzymes</div></a>
              <div data-categoryid="17863" data-articleid="24820"
                   data-categoryname="Enzymes"
                   class="flagImage" title="Flag 'Enzymes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Hydrides%22" title="Search for articles in the subject area:'Hydrides'"><div class="flagText">Hydrides</div></a>
              <div data-categoryid="23547" data-articleid="24820"
                   data-categoryname="Hydrides"
                   class="flagImage" title="Flag 'Hydrides' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Lysine%22" title="Search for articles in the subject area:'Lysine'"><div class="flagText">Lysine</div></a>
              <div data-categoryid="40333" data-articleid="24820"
                   data-categoryname="Lysine"
                   class="flagImage" title="Flag 'Lysine' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Nicotine%22" title="Search for articles in the subject area:'Nicotine'"><div class="flagText">Nicotine</div></a>
              <div data-categoryid="20375" data-articleid="24820"
                   data-categoryname="Nicotine"
                   class="flagImage" title="Flag 'Nicotine' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=7217'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4970'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=9315&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
