

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001252"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001252" />

    <meta name="citation_title" content="Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates"/>
    <meta itemprop="name" content="Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates"/>

      <meta name="citation_author" content="Ivan Rusyn"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Rebecca C. Fry"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Thomas J. Begley"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Joanna Klapacz"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="J. Peter Svensson"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Mark Ambrose"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Leona D. Samson"/>
            <meta name="citation_author_institution" content="Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001252.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1252"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Regulatory networks revealed by transcriptional profiling of damaged Saccharomyces cerevisiae cells: Rpn4 links base excision repair with proteasomes.; citation_author=SA Jelinsky; citation_author=P Estep; citation_author=GM Church; citation_author=LD Samson; citation_journal_title=Mol Cell Biol; citation_volume=20; citation_number=1; citation_pages=8157-8167; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Global response of Saccharomyces cerevisiae to an alkylating agent.; citation_author=SA Jelinsky; citation_author=LD Samson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=96; citation_number=2; citation_pages=1486-1491; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Genomic expression programs in the response of yeast cells to environmental changes.; citation_author=AP Gasch; citation_author=PT Spellman; citation_author=CM Kao; citation_author=O Carmel-Harel; citation_author=MB Eisen; citation_journal_title=Mol Biol Cell; citation_volume=11; citation_number=3; citation_pages=4241-4257; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Abasic sites in DNA: repair and biological consequences in Saccharomyces cerevisiae.; citation_author=S Boiteux; citation_author=M Guillet; citation_journal_title=DNA Repair (Amst); citation_volume=3; citation_number=4; citation_pages=1-12; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Effect of acute doses of 2-acetylaminofluorene on the capacity of rat liver to repair methylated purines in DNA in vivo and in vitro.; citation_author=DP Cooper; citation_author=PJ O'Connor; citation_author=GP Margison; citation_journal_title=Cancer Res; citation_volume=42; citation_number=5; citation_pages=4203-4209; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Chemical carcinogen induction of DNA-repair synthesis in human peripheral blood monocytes.; citation_author=RS Lake; citation_author=ML Kropko; citation_author=S McLachlan; citation_author=MR Pezzutti; citation_author=RH Shoemaker; citation_journal_title=Mutat Res; citation_volume=74; citation_number=6; citation_pages=357-377; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Effect of chronic treatment of rats with dimethylnitrosamine on the removal of O6-methylguanine from DNA.; citation_author=R Montesano; citation_author=H Bresil; citation_author=G Planche-Martel; citation_author=GP Margison; citation_author=AE Pegg; citation_journal_title=Cancer Res; citation_volume=40; citation_number=7; citation_pages=452-458; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Species differences in the inducibility of hepatic O6-alkylguanine repair in rodents.; citation_author=PJ O'Connor; citation_author=YH Chu; citation_author=DP Cooper; citation_author=GB Maru; citation_author=RA Smith; citation_journal_title=Biochimie; citation_volume=64; citation_number=8; citation_pages=769-773; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=A new pathway for DNA repair in Escherichia coli.; citation_author=L Samson; citation_author=J Cairns; citation_journal_title=Nature; citation_volume=267; citation_number=9; citation_pages=281-283; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=Alternative pathways for the in vivo repair of O6-alkylguanine and O4-alkylthymine in Escherichia coli: the adaptive response and nucleotide excision repair.; citation_author=L Samson; citation_author=J Thomale; citation_author=MF Rajewsky; citation_journal_title=Embo J; citation_volume=7; citation_number=10; citation_pages=2261-2267; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=DNA Repair and Mutagenesis.; citation_author=EC Friedberg; citation_author=GC Walker; citation_author=W Siede; citation_number=11; citation_date=2005; citation_publisher=American Society of Microbiology; " />
      <meta name="citation_reference" content="citation_title=Generation of a strong mutator phenotype in yeast by imbalanced base excision repair.; citation_author=BJ Glassner; citation_author=LJ Rasmussen; citation_author=MT Najarian; citation_author=LM Posnick; citation_author=LD Samson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=12; citation_pages=9997-10002; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Imbalanced base excision repair increases spontaneous mutation and alkylation sensitivity in Escherichia coli.; citation_author=LM Posnick; citation_author=LD Samson; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=13; citation_pages=6763-6771; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Influence of S-adenosylmethionine pool size on spontaneous mutation, dam methylation, and cell growth of Escherichia coli.; citation_author=LM Posnick; citation_author=LD Samson; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=14; citation_pages=6756-6762; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=In vivo evidence for endogenous DNA alkylation damage as a source of spontaneous mutation in eukaryotic cells.; citation_author=W Xiao; citation_author=L Samson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=90; citation_number=15; citation_pages=2117-2121; citation_date=1993; " />
      <meta name="citation_reference" content="citation_title=Base excision repair in yeast and mammals.; citation_author=A Memisoglu; citation_author=L Samson; citation_journal_title=Mutat Res; citation_volume=451; citation_number=16; citation_pages=39-51; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A probabilistic functional network of yeast genes.; citation_author=I Lee; citation_author=SV Date; citation_author=AT Adai; citation_author=EM Marcotte; citation_journal_title=Science; citation_volume=306; citation_number=17; citation_pages=1555-1558; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Genomic expression responses to DNA-damaging agents and the regulatory role of the yeast ATR homolog Mec1p.; citation_author=AP Gasch; citation_author=M Huang; citation_author=S Metzner; citation_author=D Botstein; citation_author=SJ Elledge; citation_journal_title=Mol Biol Cell; citation_volume=12; citation_number=18; citation_pages=2987-3003; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Protein oxidation in G0 cells of Saccharomyces cerevisiae depends on the state rather than rate of respiration and is enhanced in pos9 but not yap1 mutants.; citation_author=H Aguilaniu; citation_author=L Gustafsson; citation_author=M Rigoulet; citation_author=T Nystrom; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=19; citation_pages=35396-35404; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=5′-nicked apurinic/apyrimidinic sites are resistant to beta-elimination by beta-polymerase and are persistent in human cultured cells after oxidative stress.; citation_author=J Nakamura; citation_author=DK La; citation_author=JA Swenberg; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=20; citation_pages=5323-5328; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Endogenous apurinic/apyrimidinic sites in genomic DNA of mammalian tissues.; citation_author=J Nakamura; citation_author=JA Swenberg; citation_journal_title=Cancer Res; citation_volume=59; citation_number=21; citation_pages=2522-2526; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants.; citation_author=HT Tran; citation_author=JD Keen; citation_author=M Kricker; citation_author=MA Resnick; citation_author=DA Gordenin; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=22; citation_pages=2859-2865; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Summaries of Affymetrix GeneChip probe level data.; citation_author=RA Irizarry; citation_author=BM Bolstad; citation_author=F Collin; citation_author=LM Cope; citation_author=B Hobbs; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=23; citation_pages=e15; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks.; citation_author=P Shannon; citation_author=A Markiel; citation_author=O Ozier; citation_author=NS Baliga; citation_author=JT Wang; citation_journal_title=Genome Res; citation_volume=13; citation_number=24; citation_pages=2498-2504; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=FunSpec: a web-based cluster interpreter for yeast.; citation_author=MD Robinson; citation_author=J Grigull; citation_author=N Mohammad; citation_author=TR Hughes; citation_journal_title=BMC Bioinformatics; citation_volume=3; citation_number=25; citation_pages=35; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=3-methyladenine DNA glycosylases: structure, function, and biological importance.; citation_author=MD Wyatt; citation_author=JM Allan; citation_author=AY Lau; citation_author=TE Ellenberger; citation_author=LD Samson; citation_journal_title=Bioessays; citation_volume=21; citation_number=26; citation_pages=668-676; citation_date=1999; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates"/>
    <meta name="twitter:description" content="Upon exposure to DNA damaging agents, Saccharomyces cerevisiae respond by activating a massive transcriptional program that reflects the fact that &ldquo;DNA damaging&rdquo; agents also damage other cellular macromolecules. To identify the transcriptional response that is specific to DNA damage, we have modulated the first two enzymes in the base excision repair (BER) pathway generating yeast strains with varied levels of the repair intermediates, abasic sites or strand breaks. We show that the number of abasic sites is significantly increased when the 3-methyladenine DNA glycosylase (Mag): AP endonuclease (Apn1) ratio is increased and that spontaneous frame shift mutation is considerably elevated when either Mag, or Mag plus Apn1, expression is elevated. Expression profiling identified 633 ORFs with differential expression associated with BER modulation. Analysis of transcriptional networks associated with the accumulation of DNA repair intermediates identifies an enrichment for numerous biological processes. Moreover, most of the BER-activated transcriptional response was independent of the classical yeast environmental stress response (ESR). This study highlights that DNA damage in the form of abasic sites or strand breaks resulting from BER modulation is a trigger for substantial genome-wide change and that this response is largely ESR-independent. Taken together, these results suggest that a branch point exists in the current model for DNA damage-signaled transcription in S. cerevisiae."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001252.g006"/>

  <meta property="og:title" content="Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5618'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8739'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=8599&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001252">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001252" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001252&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Transcriptional Networks in <i>S. cerevisiae</i> Linked to an Accumulation of Base Excision Repair Intermediates
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ivan Rusyn
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ivan Rusyn, Rebecca C. Fry</p>

              
              

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Department of Environmental Sciences and Engineering, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rebecca C. Fry
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Ivan Rusyn, Rebecca C. Fry</p>

              
              

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thomas J. Begley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Department of Biomedical Sciences, Gen*NY*Sis Center for Excellence in Cancer Genomics, University at Albany–State University of New York (SUNY), Rensselaer, New York, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joanna Klapacz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            J. Peter Svensson, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Ambrose, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Department of Cellular and Molecular Pathology, University of California at Los Angeles, Los Angeles, California, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leona D. Samson
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:lsamson@mit.edu">lsamson@mit.edu</a></p>

                <p>Affiliation:
                  Center for Environmental Health Sciences and Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001252</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001252-g001" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001252-t001" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001252-g002" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001252-g003" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001252-g004" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001252-g005" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001252-g006" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001252">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001252" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Upon exposure to DNA damaging agents, <em>Saccharomyces cerevisiae</em> respond by activating a massive transcriptional program that reflects the fact that “DNA damaging” agents also damage other cellular macromolecules. To identify the transcriptional response that is specific to DNA damage, we have modulated the first two enzymes in the base excision repair (BER) pathway generating yeast strains with varied levels of the repair intermediates, abasic sites or strand breaks. We show that the number of abasic sites is significantly increased when the 3-methyladenine DNA glycosylase (Mag): AP endonuclease (Apn1) ratio is increased and that spontaneous frame shift mutation is considerably elevated when either Mag, or Mag plus Apn1, expression is elevated. Expression profiling identified 633 ORFs with differential expression associated with BER modulation. Analysis of transcriptional networks associated with the accumulation of DNA repair intermediates identifies an enrichment for numerous biological processes. Moreover, most of the BER-activated transcriptional response was independent of the classical yeast environmental stress response (ESR). This study highlights that DNA damage in the form of abasic sites or strand breaks resulting from BER modulation is a trigger for substantial genome-wide change and that this response is largely ESR-independent. Taken together, these results suggest that a branch point exists in the current model for DNA damage-signaled transcription in <em>S. cerevisiae</em>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Rusyn I, Fry RC, Begley TJ, Klapacz J, Svensson JP, et al.  (2007) Transcriptional Networks in <em>S. cerevisiae</em> Linked to an Accumulation of Base Excision Repair Intermediates. PLoS ONE 2(11):
          e1252.
            doi:10.1371/journal.pone.0001252</p><p><strong>Academic Editor: </strong>Cecile Fairhead, Pasteur Institute, France</p><p><strong>Received:</strong> October 10, 2007; <strong>Accepted:</strong> November 7, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Rusyn et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the National Institute of Environmental Health Sciences (ES11399 and ES002109), and the National Cancer Institute(CA055042). LDS is an American Cancer Society Research Professor.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Previously, we have shown that yeast exposed to alkylating agents initiate a broad gene expression response modulating a staggering 30% of the yeast genome, i.e. up to 2000 genes <a href="#pone.0001252-Jelinsky1">[1]</a>, <a href="#pone.0001252-Jelinsky2">[2]</a>. Upon exposure to exogenous alkylation damage, yeast respond by modulating a host of biological processes that includes, as expected, DNA repair and cell cycle checkpoint pathways, but also includes the following: (i) activation of protein degradation machinery, (ii) increased lipid metabolism, (iii) increased RNA processing and (iv) decreased protein synthesis (ribosomal biosynthesis) <a href="#pone.0001252-Jelinsky1">[1]</a>, <a href="#pone.0001252-Jelinsky2">[2]</a>. That numerous biological processes are modulated when yeast are exposed to alkylating agents likely reflects the fact that in addition to damaging DNA, these so-called “DNA damaging” agents also damage proteins, lipids and RNA. In addition, the expression of a core set of ~900 genes changes under a variety of stressful conditions, including DNA damage, and is termed the environmental stress response (ESR) <a href="#pone.0001252-Gasch1">[3]</a>. Here, we set out to identify the transcriptional response that is specifically caused by damage to DNA and not to other cellular molecules; instead of introducing damage with reactive electrophilic agents (e.g. an alkylating agent) we generated DNA damage by imbalancing the base excision repair (BER) pathway such that abasic sites or gaps and nicks accumulate in the genome during the processing of repairing endogenous DNA damage.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In its simplest form, the first two steps of the BER pathway are represented by a DNA glycosylase and an AP endonuclease. DNA glycosylases can recognize damaged bases and remove them via cleavage of the <em>N</em>-glycosidic bond to initiate BER. In <em>S. cerevisiae</em> there are at least five distinct DNA glycosylases, namely Ung, Nth1, Nth2, Ogg1, and Mag, that collectively recognize damaged bases and process them to generate abasic sites. The abasic site is substrate for AP endonucleases that carry out the next step in BER. <em>S. cerevisiae</em> has two distinct AP endonucleases, the major enzyme Apn1 and a damage-inducible enzyme, Apn2 <a href="#pone.0001252-Boiteux1">[4]</a>. Apn1 and Apn2 cleave abasic sites 5′ to the site of base loss, generating a single strand break. The abasic sugar residue is removed by deoxyribophophodiesterase, the gap is filled by DNA polymerase and the remaining nick sealed by DNA ligase.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Across phylogeny, the regulation of enzymes involved in the first two steps of BER has been noted at the level of transcription. In <em>E. coli</em>, yeast, mouse and human cells, treatments with xenobiotics can induce the transcription of BER genes <a href="#pone.0001252-Cooper1">[5]</a>–<a href="#pone.0001252-Samson2">[10]</a>. The induced expression of such DNA repair pathways has long been thought to confer protection against both the killing and mutagenic effects of DNA damage <a href="#pone.0001252-Friedberg1">[11]</a>. However, it is now apparent that in some circumstances increased expression of DNA repair genes may actually lead to the accumulation of mutations. In fact, our previous studies demonstrated the generation of a strong mutator phenotype in <em>S. cerevisiae</em> and <em>E. coli</em> by imbalanced base excision repair <a href="#pone.0001252-Glassner1">[12]</a>–<a href="#pone.0001252-Xiao1">[15]</a>. Specifically, it was shown that Mag expression in yeast produces up to a 300-fold increase in the spontaneous base substitution mutation rate when Mag levels are high relative to Apn1 <a href="#pone.0001252-Glassner1">[12]</a>. It was suggested that such a dramatic increase in spontaneous mutation rate is due to the fact that Mag excises normal DNA bases, in addition to endogenously damaged bases, producing mutagenic AP sites that can be further processed into mutations by the <em>Rev1/Rev3/Rev7</em> lesion bypass polymerases <a href="#pone.0001252-Glassner1">[12]</a>, <a href="#pone.0001252-Xiao1">[15]</a>, <a href="#pone.0001252-Memisoglu1">[16]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In these studies, we explore how imbalanced BER affects the persistence of AP sites and frameshift mutations in yeast and how the accumulation of BER intermediates is linked to a DNA damage specific transcriptional response. We show that Mag expression in <em>S. cerevisiae</em> produces a 2- to 5-fold increase in AP sites, depending on the Mag:Apn1 ratio, an effect that mirrors the changes in spontaneous base substitution mutation rates in the same strains of yeast <a href="#pone.0001252-Glassner1">[12]</a>. This finding confirms the hypothesis that increased Mag:Apn1 ratios lead to an increased steady state level of mutagenic AP sites. Furthermore, frameshift mutations are elevated when Mag:Apn1 ratios are high and these mutations are reduced with increased expression of Apn1. These studies also show that in response to the DNA damage of accumulated BER repair intermediates, yeast respond by modulating the expression of ~600 ORFs that are integrated into transcriptional networks encoding DNA replication/metabolism, protein transport, energy metabolism and ribosomal biosynthesis machinery. Surprisingly, this BER-modulated transcriptional response is largely independent of the environmental stress response (ESR), suggesting that a branch point exists in the current model for DNA damage signaled transcription.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Imbalance in the first two steps in BER leads to accumulation of AP sites</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In this study, the first two enzymes in the BER pathway, Mag and Apn1, were modulated such that the BER intermediates, abasic sites or strand breaks, accumulate (<a href="#pone-0001252-g001">Fig. 1</a>). Specifically, the impact of accumulating abasic sites was examined by (i) increased expression of Mag while maintaining wild type levels of Apn1 (WT/p<em>MAG</em>), (ii) maintaining wild type levels of Mag in an Apn1 deficient strain (Δ<em>apn1</em>), or (iii) increased expression of Mag in an Apn1 deficient strain (Δ<em>apn1</em>/p<em>MAG</em>). The impact of accumulating gaps and nicks was examined by (i) expressing Apn1 (Δ<em>apn1</em>/p<em>APN1</em>), or (ii) expressing both Mag and Apn1 (Δ<em>apn1</em>/p<em>MAG</em>/p<em>APN1</em>) (<a href="#pone-0001252-t001">Table 1</a>).</p>
<div class="figure" id="pone-0001252-g001"><div class="img"><a name="pone-0001252-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Imbalances of base excision repair.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The pathway on the left indicates normal processing in the base excision repair pathway. Central arrows highlight the potential imbalances that may lead to toxicity, clastogenecity and mutagenicity on the right (modified from <a href="#pone.0001252-Wyatt1">[26]</a>).</p>
<span>doi:10.1371/journal.pone.0001252.g001</span></div><div class="figure" id="pone-0001252-t001"><div class="img"><a name="pone-0001252-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Yeast strains used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0001252.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The persistence of AP sites in the yeast strains with altered levels of expression of Apn1 and Mag was examined. Plasmid based Mag expression in the wild type host increased steady-state level of genomic AP sites by ~2.5 fold (<a href="#pone-0001252-g002">Fig. 2</a>, Expt. #1). In the Δ<em>apn1</em> strain, AP sites increased ~2-fold, an effect that was essentially reversed by expressing Apn1 from a plasmid (<a href="#pone-0001252-g002">Fig. 2</a>, Expt. #2, #4). As one might expect, when Mag was expressed in the Δ<em>apn1</em> strain, AP sites were further elevated to ~5-fold that in wild type cells (<a href="#pone-0001252-g002">Fig. 2</a>, Expt. #3). When Apn1 was coexpressed with Mag in the Δ<em>apn1</em> strain, the steady state level of AP sites fell dramatically, but it should be noted that it did not quite return to wild type levels (<a href="#pone-0001252-g002">Fig. 2</a>, Expt. #5).</p>
<div class="figure" id="pone-0001252-g002"><div class="img"><a name="pone-0001252-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Five BER modulated strains were compared for measurements of AP site accumulation.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Five experimental conditions (#1–5) each with different ratios of Mag:Apn1 were assessed for AP site accumulation measured in genomic DNA. Statistical difference (t-test, p&lt;0.05) from wild type (WT, *) or from Δ<em>apn1/MAG</em> group (**) is indicated. The expression of Mag of Apn1 are shown as: (0) = low expression, (+) = wild type expression, (++++++++++) = high expression.</p>
<span>doi:10.1371/journal.pone.0001252.g002</span></div>

<h4>Frame shift mutations accumulate under conditions of increased abasic sites and increased gaps and nicks</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In addition to the AP site measurements in the BER modulated strain, we set out to examine another common type of spontaneous-induced damage, namely (+1) and (−1) frame shift mutations. We find that Apn1 expression does not affect frame shift mutation rates (<a href="#pone-0001252-g003">Fig. 3</a>). However, Mag expression caused a dramatic increase of 12 and 38-fold in (+1) and (−1) frame shifts respectively when compared to the Apn1-deficient strain. Interestingly, co-expression of Apn1 and Mag did not fully suppress the increased frame shift mutagenesis with a remarkable 5- and 14-fold increase in (+1) and (−1) frame shifts, respectively, still remaining (<a href="#pone-0001252-g003">Fig. 3</a>).</p>
<div class="figure" id="pone-0001252-g003"><div class="img"><a name="pone-0001252-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Induction of base excision repair leads to an accumulation of frameshift mutations.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Frameshift mutation frequency in Δ<em>apn1</em> strains that expressed Apn1 and/or Mag was determined as detailed in <a href="#s4">Materials and Methods</a>. Asterisk (*) denotes statistical difference (t-test, p&lt;0.05) relative to the background strain (Δ<em>apn1</em>).</p>
<span>doi:10.1371/journal.pone.0001252.g003</span></div>

<h4>Accumulation of BER intermediates results in DNA damage specific genome-wide signatures distinct from signatures induced by MMS</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The genome-wide response of yeast to the accumulation of DNA repair intermediates was examined by performing transcriptional profiling of the five yeast strains with altered expression of BER repair enzymes (see <a href="#s4">Materials and Methods</a>). The genome-wide expression levels of the BER modulated strains were compared to wild type yeast and the differentially expressed genes were identified (see <a href="#s4">Materials and Methods</a>); a total of 633 ORFs were identified with statistically significant changes compared to wild type yeast in at least one condition (<a href="#pone-0001252-g004">Fig. 4</a>, <a href="#pone.0001252.s002">Dataset S1</a>). It should be emphasized here that these strains were not exposed to any exogenous DNA damaging agents. The extent of change across the yeast genome in response to BER expression was quite varied across the five experiments, with yeast expressing Mag (WT/p<em>MAG</em>) showing the most transcriptional change, specifically 334 modulated ORFs (<a href="#pone-0001252-g004">Fig. 4</a>).</p>
<div class="figure" id="pone-0001252-g004"><div class="img"><a name="pone-0001252-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Gene expression changes in <em>S. cerevisiae</em> caused by an induction of base excision repair or methyl methane sulfonate (MMS).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>MMS transcriptional data were obtained from two doses (0.05% and 0.1%) from <a href="#pone.0001252-Jelinsky1">[1]</a>. Unsupervised hierarchical clustering of 633 ORFs whose transcription was significantly changed in at least one of the BER strains used in this study (as compared to the wild type). The number of ORFs identified for each experiment as statistically significant is shown in brackets below the column. Fold changes were Z-score transformed; red indicates increased relative expression, and blue indicates decreased relative expression.</p>
<span>doi:10.1371/journal.pone.0001252.g004</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>We further set out to compare the transcriptional response that was specifically caused by the accumulation of DNA repair intermediates, to that caused by exposure to an alkylating agent. The expression patterns of the BER modulated strains were compared to our previously published data of the transcriptional response of yeast when exposed to the alkylating agent, methyl methanesulfonate (MMS), at two doses 0.05% and 0.1% <a href="#pone.0001252-Jelinsky1">[1]</a>. The expression patterns of the five yeast strains with modulated levels of BER were strikingly different from the expression patterns of yeast exposed to MMS (<a href="#pone-0001252-g004">Fig. 4</a>).</p>


<h4>The accumulation of BER intermediates activates ESR-independent transcriptional networks enriched for numerous biological processes</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To identify whether significant molecular interactions exist among the gene products of the transcripts that were modulated by BER intermediates, the 633 ORFs that were transcriptionally modulated in at least one BER condition were overlayed on a functional yeast network comprised of 4,691 nodes (proteins) and 34,000 edges (interactions) <a href="#pone.0001252-Lee1">[17]</a>. Of the 633 ORFs modulated by expressing BER enzymes, 501 were contained in the large yeast network (<a href="#pone.0001252.s001">Fig. S1</a>, <a href="#pone.0001252.s002">Dataset S1</a>).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To identify transcriptionally enriched modules within the protein network, we further analyzed this large interactome for clusters of proteins that interact and also are modulated by BER intermediate accumulation; five subnetworks were identified (<a href="#pone.0001252.s001">Fig. S1</a>). It should be mentioned again that these subnetworks are encoded entirely by genes that are transcriptionally modulated by BER intermediate accumulation. The subnetworks that are modulated by BER intermediate accumulation are enriched for numerous biological processes including: protein biosynthesis (<a href="#pone-0001252-g005">Fig. 5A, C</a>), protein-mitochondrial targeting (<a href="#pone-0001252-g005">Fig. 5A</a>), energy pathways (<a href="#pone-0001252-g005">Fig. 5B</a>), chromatin assembly/disassembly (<a href="#pone-0001252-g005">Fig. 5C</a>), amino acid biosynthesis (<a href="#pone-0001252-g005">Fig. 5D</a>), and DNA replication/metabolism (<a href="#pone-0001252-g005">Fig. 5E</a>). Of these biological processes that are modulated by BER intermediate accumulation, those that are transcriptionally activated are: (i) amino acid biosynthesis, (ii) DNA replication/metabolism, and (iii) energy pathways (<a href="#pone-0001252-g005">Fig. 5</a>). The biological processes that are down regulated by BER intermediate accumulation are protein biosynthesis, chromatin assembly/disassembly and protein-mitochondrial targeting (<a href="#pone-0001252-g005">Fig. 5</a>).</p>
<div class="figure" id="pone-0001252-g005"><div class="img"><a name="pone-0001252-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Yeast transcriptional networks activated in response to BER intermediate accumulation are largely ESR-independent.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>A) Subnetwork #1 enriched for protein-mitochondrial targeting and protein biosynthesis, B) Subnetwork #2 enriched for energy pathways, C) Subnetwork #3 enriched for ribosome biogenesis and chromatin assembly, D) Subnetwork #4 enriched for amino acid biosynthesis, E) Subnetwork #5 enriched for DNA replication/metabolism. P-values for gene ontology enrichment are indicated in legend. Pie charts in the upper right hand corner of each figure represent the percentage of network nodes that are ESR (yellow) or non ESR (blue) (P-value of ESR enrichment is shown). Proteins encoded by transcripts that are transcriptionally activated are indicated with a (+), all other proteins are encoded by transcripts that are transcriptionally repressed by BER intermediate accumulation.</p>
<span>doi:10.1371/journal.pone.0001252.g005</span></div>

<h4>The DNA damage-specific response to BER intermediates is largely ESR independent</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The current model of DNA damage-induced transcriptional response in yeast suggests that DNA damage signals for the modulation of three groups of genes; (i) the activation or repression of a battery of ~900 genes classified as the environmental stress response (ESR), (ii) the modulation of ~50 cell cycle associated genes, and (iii) the modulation of 9 genes termed the “DNA damage signature” <a href="#pone.0001252-Gasch2">[18]</a>. We set out to determine whether or not the transcriptional networks that were activated by the accumulation of BER intermediates were largely comprised of the ESR, as would be predicted by the current model. The statistical enrichment of ESR genes was calculated for each of the subnetworks, as well as for the entire list of 633 ORFs modulated by BER intermediate accumulation (see <a href="#s4">Materials and Methods</a>) (<a href="#pone-0001252-g005">Fig. 5</a> A–E). Surprisingly, the enrichment of the ESR in the set of 633 ORFs was not statistically significant (P = 0.12). Furthermore, only one of the five subnetworks was significantly enriched for genes that belong to the ESR (<a href="#pone-0001252-g005">Fig. 5</a> C). The third subnetwork is highly enriched for proteins that are involved in ribosome biogenesis; note that a classic hallmark of the ESR is that ribosomal protein genes are repressed (<a href="#pone-0001252-g005">Fig. 5C</a>) <a href="#pone.0001252-Gasch2">[18]</a>. All of the other subnetworks that were activated by the accumulation of BER intermediates were statistically ESR-independent.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>It should be mentioned that contained within these BER intermediate modulated transcriptional networks are a few members of the “DNA damage signature” and transcripts that encode members of the cell-cycle related genes. Specifically, the fifth subnetwork that is enriched for DNA replication/metabolism integrates two members of the yeast “DNA damage signature,” namely the ribonucleotide reductases Rnr2 and Rnr4. The third subnetwork that is enriched for chromatin assembly/disassembly-related proteins contains two cell-cycle related transcripts that encode histones, Htb1 and Htb2, found previously to be repressed by MMS and IR <a href="#pone.0001252-Gasch2">[18]</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Our network findings suggest that the current model of DNA damage-induced transcriptional activation in yeast should be expanded in a few ways (<a href="#pone-0001252-g006">Fig. 6</a>). First, our results show that DNA damage in the form of repair intermediates is insufficient to activate the bulk of the environmental stress response. Second, only a subset of the MMS and IR “DNA damage signature” is activated by the accumulation of repair intermediates. As the DNA damage-specific response to BER intermediates is largely ESR independent, these data suggest that a branch point in the current model of DNA damage-induced transcriptional signaling exists (<a href="#pone-0001252-g006">Fig. 6</a>). Specifically, we find transcripts that encode proteins involved in protein-mitochondial targeting, chromatin assembly/disassembly, DNA replication/metabolism and energy pathways, are members of the DNA damage activated gene set.</p>
<div class="figure" id="pone-0001252-g006"><div class="img"><a name="pone-0001252-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001252" data-uri="info:doi/10.1371/journal.pone.0001252.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001252.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001252.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001252.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001252.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Diagram of a DNA damage-triggered transcriptional response in <em>S. cerevisiae</em>.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>DNA base damage (e.g. alkylated bases induced by endogenous or exogenous agents) is processed by repair machinery resulting in repair intermediates (e.g. abasic sites, gaps and nicks). DNA damage signals for the transcriptional modulation of (i) the induction or repression of the ~900 genes belonging to the environmental stress response (ESR), and (ii) ~50 cell cycle regulated genes, and (iii) DNA damage-triggered networks that include the MMS and IR DNA damage signature. Novel branch points in the DNA damage-triggered ESR independent transcriptional response are shown in red.</p>
<span>doi:10.1371/journal.pone.0001252.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study we set out to identify the transcriptional response in yeast that is specifically caused by damage to DNA and not to other cellular molecules. As an alternative to introducing damage with reactive electrophilic agents (e.g. an alkylating agent), we generated DNA damage by imbalancing the base excision repair (BER) pathway such that abasic sites or gaps and nicks accumulate in the genome during the process of repairing endogenous DNA damage. Five yeast strains with altered expression levels of Apn1 and Mag were analyzed for the persistence of AP sites. Mag expression in the wild type host increased the steady-state level of genomic AP sites by ~2.5 fold. As expected, when Mag was expressed in the Δ<em>apn1</em> strain, AP sites were further elevated to ~5-fold that in wild type, whereas coexpression of Apn1 with Mag in the Δ<em>apn1</em> strain resulted in a dramatic decrease in the steady state level of AP sites. These results complement our findings for single base substitution mutation rates in the same strains of yeast where an increase in Mag: Apn1 ratio leads to the generation of a strong mutator phenotype in yeast <a href="#pone.0001252-Glassner1">[12]</a>. It was found that spontaneous mutations were increased several hundred fold when Mag expression was relatively high and Apn1 relatively low <a href="#pone.0001252-Glassner1">[12]</a>, <a href="#pone.0001252-Xiao1">[15]</a>. Intriguingly, while Mag expression in AP endonuclease-deficient yeast led to a nearly 300-fold increase in spontaneous mutation rate, here we observed only a 5-fold increase in steady state levels of AP sites. However, unlike the current experiments with logarithmically growing cells, spontaneous mutation rates were determined in stationary phase cells over a period of 14 days <a href="#pone.0001252-Glassner1">[12]</a>, <a href="#pone.0001252-Xiao1">[15]</a>. Since stationary phase yeast cells are known to exhibit significantly higher levels of oxidant production <a href="#pone.0001252-Aguilaniu1">[19]</a>, the difference in experimental conditions might account for this apparent divergence. Indeed, when AP sites were measured in 7 day old stationary phase cells, a further &gt;10-fold increase in AP sites was detected in wild type and Mag expressing strains (data not shown). We further examined the BER modulated yeast strains for another type of spontaneous-induced damage, that of frame shift mutations. Similar to our results with spontaneous base substitution rates <a href="#pone.0001252-Glassner1">[12]</a>, <a href="#pone.0001252-Xiao1">[15]</a>, we find that Apn1 expression does not affect frame shift mutation rates. Mag expression, however, caused a dramatic increase in frame shift mutation rate when compared to the Apn1-deficient strain.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To determine the impact of the accumulation of DNA repair intermediates on gene expression, the five yeast strains with altered expression of BER repair enzymes were analyzed for differential gene expression. A total of 633 ORFs were identified with statistically significant changes in at least one condition compared to wild type yeast. The extent of change across the yeast genome in response to modulated BER expression was quite varied across the five experiments, with yeast expressing Mag (WT/p<em>MAG</em>) showing the most transcriptional change. It is noteworthy that even without exposure to any exogenous DNA damaging agents there is considerable gene expression change.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>The genome-wide impact of the accumulation of DNA repair intermediates was compared to our previous results of gene expression changes in yeast upon exposure to an alkylating agent, namely methyl methanesulfonate (MMS) <a href="#pone.0001252-Jelinsky1">[1]</a>. It was a surprise to find the expression patterns of the five yeast strains with modulated levels of BER were strikingly different from the expression patterns of yeast exposed to MMS. These differences in the expression patterns of the BER modulated yeast strains and yeast exposed to MMS may, in part, represent the difference between responses to an acute influx of damage, as for MMS, rather than a chronic exposure to damage, as would be generated by BER modulation.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>To classify the biological processes and pathways that were affected by BER intermediate accumulation, the gene products of the transcripts that were modulated by BER intermediates, were overlayed on a functional yeast network <a href="#pone.0001252-Lee1">[17]</a>. A large interacting networks of proteins was identified that contained five transcriptionally enriched subnetworks. These subnetworks are encoded entirely by genes that are transcriptionally modulated by BER intermediate accumulation. These subnetworks are enriched for numerous biological processes including: protein biosynthesis, protein-mitochondrial targeting, energy pathways, chromatin assembly/disassembly, amino acid biosynthesis and DNA replication/metabolism. Three of these biological processes that are modulated by BER intermediate accumulation are transcriptionally activated; these are amino acid biosynthesis, DNA replication/metabolism, and energy pathways. The protein biosynthesis and protein-mitochondrial targeting processes are down regulated by BER intermediate accumulation.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>The panel of biological processes that in this study are modulated by BER intermediate accumulation is strikingly absent of a number of cellular processes previously shown to be modulated in yeast exposed to the alkylating agent MMS <a href="#pone.0001252-Jelinsky1">[1]</a>, <a href="#pone.0001252-Jelinsky2">[2]</a>. Notably missing from the list of biological processes modulated by endogenous DNA damage are (i) the 26S proteasome, (ii) lipid metabolism machinery, and (iii) RNA processing machinery. That these processes are modulated when yeast strains are exposed to exogenously generated alkylation damage likely represents the non-specific nature of the damage (e.g. damage to proteins, lipids and RNA) induced by alkylating agents.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>To date, the proposed model of DNA damage-induced transcriptional response in yeast suggests that DNA damage signals for the modulation of ~900 genes classified as the environmental stress response (ESR), but also the modulation of ~50 cell cycle associated genes, and 9 genes termed the “DNA damage signature” <a href="#pone.0001252-Gasch2">[18]</a>. In addition to their response to DNA damaging agents, the ESR genes in yeast are also responsive to a host of changes in environmental conditions <a href="#pone.0001252-Gasch1">[3]</a>. We set out to determine whether or not the transcriptional networks that were activated by the accumulation of BER intermediates were largely comprised of the ESR, as would be predicted by the current model of DNA damage-induced transcriptional response. The statistical enrichment of ESR genes was calculated for each of the subnetworks, as well as for the entire list of ORFs modulated by BER intermediate accumulation in the yeast strains. Surprisingly, the enrichment of the ESR in the set of 633 ORFs was not statistically significant. Furthermore, only one of the five subnetworks was significantly enriched for genes that belong to the ESR. All of the other subnetworks that were activated by the accumulation of BER intermediates were statistically ESR-independent.</p>

<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>Our results show that imbalancing the BER pathway leads to an accumulation of repair intermediates, such as AP sites, and stimulates the induction of frame shift mutations. Furthermore, we show that the induction of DNA repair intermediates, even in the absence of an exogenous damaging agent, causes a profound change in gene expression <em>S. cerevisiae</em>. This DNA damage-specific expression pattern of ~600 ORFs varies markedly from the expression pattern produced by exogenous exposure to the alkyating agent MMS. The biological processes modulated by the presence of DNA repair intermediates include protein synthesis, protein targeting, amino acid biosynthesis, and chromatin assembly. Notably lacking from the biological processes modulated by DNA repair intermediates is that of the protein degradation machinery (e.g. 26S proteasome) which is robustly induced upon exposure to exogenous DNA damaging agents; this suggests that the trigger for upregulating the proteasome emanates from damaged molecules other than DNA, most likely from damaged proteins. The DNA damage-triggered transcripts encode molecular networks that are comprised of proteins involved, not surprisingly, in DNA replication/metabolism, but also in numerous other biological processes but that the triggered responses is largely ESR-independent. As the DNA damage-specific response to BER intermediates is largely ESR independent, these data suggest that a branch point in the current model of DNA damage-induced transcriptional signaling exists. This study highlights that DNA damage, specifically, is the trigger for the modulation of transcripts that encode proteins with broad biological functions.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Strains and plasmids</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p><a href="#pone-0001252-t001">Table 1</a> lists the strains used in this study. All strains, plasmids and cell growth conditions were as previously described <a href="#pone.0001252-Glassner1">[12]</a>. Briefly, strains were grown from overnights in SD-<em>ura</em>+GAL media at 30°C with constant shaking until cell density of 10<sup>7</sup> cells/ml. They were then pelleted by centrifugation, quick-frozen using a dry ice/ethanol bath, and stored at −80°C for use in the assays described below. All experiments were performed in biological triplicate.</p>


<h4>Isolation of DNA</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>DNA was extracted by a procedure slightly modified from the method reported previously <a href="#pone.0001252-Nakamura1">[20]</a>. To minimize formation of oxidative artifacts during isolation, 2,2,6,6-tetramethylpiperidinoxyl (TEMPO, 20 mM final concentration) was added to all solutions and all procedures were performed on ice. Briefly, frozen cells were thawed and homogenized in PBS with a Tehran homogenizer (Wheaton Instruments, Millville, NJ). After centrifugation at 2,000×g for 10 min, the nuclear pellets were incubated in lysis buffer (Applied Biosystems) overnight at 4°C with proteinase K (500 mg/ml; Applied Biosystems). DNA was then extracted twice with a mixture of phenol/chloroform/water followed by ethanol precipitation. The extracted DNA was incubated in PBS (pH 7.4) with RNase A followed by DNA precipitation with cold ethanol. Then, the DNA pellet was resuspended in sterilized distilled water. The DNA solution was stored at −80°C until assayed.</p>


<h4>Abasic sites</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Abasic (AP) sites were measured based on a procedure reported by Nakamura and Swenberg <a href="#pone.0001252-Nakamura2">[21]</a>. Briefly, 8 µg of DNA in 150 µl of phosphate-buffered saline was incubated with 1 mM aldehyde reactive probe at 37°C for 10 min. After precipitation using cold ethanol, DNA was suspended in TE buffer. DNA (250 ng) in TE buffer was heat-denatured and loaded on a nitrocellulose membrane (110 ng DNA/slot, Hybond-C Super, Amersham Pharmacia Biotech) and soaked with 5× SSC then baked in a vacuum oven for 30 min. The membrane was preincubated with 10 ml of Tris-HCl containing bovine serum albumin for 15 min and then incubated in the same solution containing streptavidin-conjugated horseradish peroxidase at room temperature for 45 min. After rinsing the nitrocellulose membrane, the enzymatic activity on the membrane was visualized by enhanced chemiluminescence reagents. The nitrocellulose filter was exposed to x-ray film, and the developed film was analyzed using a Kodak Image Station 440. Quantitation was based on comparisons to internal standard DNA containing a known amount of AP sites.</p>


<h4>Fluctuation Analysis of Frameshift Mutations</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The plasmids used in the study were described previously <a href="#pone.0001252-Glassner1">[12]</a>. Lithium acetate method (Geitz kit, Genomics One International Inc. Buffalo, NY) was used to introduce YEpAPN1, pYES2.0, and pYES-MAG plasmids into isogenic haploid <em>apn1</em> null E133 and E134 yeast strains <a href="#pone.0001252-Tran1">[22]</a>. The lys2:<em>InsE</em> inserts inactivate the <em>LYS2</em> gene with 12 and 14 nucleotide runs of A, respectively, resulting in a Lys<sup>−</sup> phenotype. E133 reverts to Lys<sup>+</sup> solely by a +1 frameshift, whereas E134 reverts by a −1 frameshift mutation. <em>S. cerevisiae</em> 3MeA DNA glycosylase (MAG) was expressed in yeast when galactose, at final concentration of 2%, was added to SC drop-out media lacking uracil (Q-BioGene, Carlsbad, CA). The <em>S. cerevisiae</em> AP endonuclease 1 (APN1) was expressed from the YEpAPN1 plasmid maintained in culture by excluding leucine from the SC drop-out media. Transformant cultures were grown overnight in 2% synthetic glucose media without uracil for pYES2.0 and pYES-MAG or without uracil and leucine when YEpAPN1 was additionally included in combination with the above plasmids. When cultures reached saturation, they were diluted to a density of ~4,000 cells/ml in 2% galactose SC media and divided into ten parallel 10 ml cultures. After incubation at 30°C for ~4 days, cells were harvested, washed, and resuspended in 1 ml water. Dilutions were spread on SC–uracil or SC–uracil–leucine plates, respectively, to determine the average number of cells per culture for three randomly chosen samples per each condition. Duplicates of 0.1 ml aliquots of each culture were spread on SC +galactose plates lacking lysine to select for Lys<sup>+</sup> revertants. After ~4–5 day incubation at 30°C, colonies were counted. The median number of revertants per 10<sup>8 </sup>cells was obtained and is expressed as mean±standard deviation from at least 3 independent experiments.</p>


<h4>RNA extraction and Microarray Analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Total RNA was isolated using hot phenol extraction and hybridized to Affymetrix® (Santa Clara, CA) Yeast Genome S98 arrays in biological triplicate. RNA clean-up, cRNA labeling and fragmentation, array hybridization and staining were done as described in the Affymetrix® eukaryotic labeling protocol (Santa Clara, CA). Processed microarrays were scanned and raw image intensity data files were processed and normalized using robust multi-array average (RMA) software <a href="#pone.0001252-Irizarry1">[23]</a>. Statistical significance of expression was determined for each gene using triplicate arrays for each experimental group of BER relative to the wild type strain (ANOVA, p&lt;0.05; Fold change ≥1.5, ≤−1.5). All microarray data have been submitted to the Gene Expression Omnibus, accession number GSE9295 (<a href="http://www.ncbi.nlm.nih.gov/geo/">www.ncbi.nlm.nih.gov/geo/</a>).</p>


<h4>Network Analysis, Gene Ontology and ESR Enrichment</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Network analysis was carried out using the Cytoscape software <a href="#pone.0001252-Shannon1">[24]</a>. Transcriptionally modulated ORFs were linked into the large-scale probabilistic network <a href="#pone.0001252-Lee1">[17]</a>. Statistical evaluation of co-regulated interacting groups of genes was carried out through Gene Ontology Enrichment Analysis within the Functional Specification Database <a href="#pone.0001252-Robinson1">[25]</a>. To classify transcripts as activated or repressed across the BER experiments a mean expression value was calculated and queried for Gene Ontology enrichment as above. Co-regulated yeast ORFs were classified according to the Gene Ontology Biological Process and the hypergeometric distribution used to assess enrichment of a particular gene category. Statistical significance of enrichment of environmental stress genes was calculated through the Fisher's Exact Test.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001252"></div><a name="pone.0001252.s001" id="pone.0001252.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001252.s001">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">BER modulated transcriptional networks in <em>S. cerevisiae</em>. A) The transcripts modulated under BER intermediate accumulation were integrated with their gene products and overlayed on all known protein-protein interactions in yeast (4,681 nodes and 34,000 edges). Of the BER modulated ORFs, 501 ORFs were identified in the database of known protein-protein interactions in the large interacting network (indicated in red). B) Analysis requiring that all interacting nodes are transcriptionally modulated by BER intermediate accumulation results in the identification of five subnetworks (1–5) significantly enriched for GO biological processes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001252.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(7.69 MB EPS)</p>
<a name="pone.0001252.s002" id="pone.0001252.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001252.s002">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of 633 ORFs modulated by BER imbalance. ORFs modulated in at least one condition of base excision repair imbalance in yeast were identified using differential expression testing.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001252.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.21 MB XLS)</p>
</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: LS IR. Performed the experiments: TB IR JK MA. Analyzed the data: LS JS RF IR. Wrote the paper: LS RF IR.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001252-Jelinsky1" id="pone.0001252-Jelinsky1"></a>Jelinsky SA, Estep P, Church GM, Samson LD (2000) Regulatory networks revealed by transcriptional profiling of damaged Saccharomyces cerevisiae cells: Rpn4 links base excision repair with proteasomes. Mol Cell Biol  20: 8157–8167.  <ul class="find" data-citedArticleID="1025010" data-doi="10.1128/mcb.20.21.8157-8167.2000"><li><a href="http://dx.doi.org/10.1128/mcb.20.21.8157-8167.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulatory+networks+revealed+by+transcriptional+profiling+of+damaged+Saccharomyces+cerevisiae+cells%3A+Rpn4+links+base+excision+repair+with+proteasomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulatory+networks+revealed+by+transcriptional+profiling+of+damaged+Saccharomyces+cerevisiae+cells%3A+Rpn4+links+base+excision+repair+with+proteasomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001252-Jelinsky2" id="pone.0001252-Jelinsky2"></a>Jelinsky SA, Samson LD (1999) Global response of Saccharomyces cerevisiae to an alkylating agent. Proc Natl Acad Sci U S A  96: 1486–1491.  <ul class="find" data-citedArticleID="1025012" data-doi="10.1073/pnas.96.4.1486"><li><a href="http://dx.doi.org/10.1073/pnas.96.4.1486" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+response+of+Saccharomyces+cerevisiae+to+an+alkylating+agent." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+response+of+Saccharomyces+cerevisiae+to+an+alkylating+agent.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001252-Gasch1" id="pone.0001252-Gasch1"></a>Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, et al.  (2000) Genomic expression programs in the response of yeast cells to environmental changes. Mol Biol Cell  11: 4241–4257.  <ul class="find" data-citedArticleID="1025002" data-doi="10.1091/mbc.11.12.4241"><li><a href="http://dx.doi.org/10.1091/mbc.11.12.4241" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+programs+in+the+response+of+yeast+cells+to+environmental+changes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001252-Boiteux1" id="pone.0001252-Boiteux1"></a>Boiteux S, Guillet M (2004) Abasic sites in DNA: repair and biological consequences in Saccharomyces cerevisiae. DNA Repair (Amst)  3: 1–12.  <ul class="find" data-citedArticleID="1024996" data-doi="10.1016/j.dnarep.2003.10.002"><li><a href="http://dx.doi.org/10.1016/j.dnarep.2003.10.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Abasic+sites+in+DNA%3A+repair+and+biological+consequences+in+Saccharomyces+cerevisiae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Abasic+sites+in+DNA%3A+repair+and+biological+consequences+in+Saccharomyces+cerevisiae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001252-Cooper1" id="pone.0001252-Cooper1"></a>Cooper DP, O'Connor PJ, Margison GP (1982) Effect of acute doses of 2-acetylaminofluorene on the capacity of rat liver to repair methylated purines in DNA in vivo and in vitro. Cancer Res  42: 4203–4209.  <ul class="find" data-citedArticleID="1024998"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+acute+doses+of+2-acetylaminofluorene+on+the+capacity+of+rat+liver+to+repair+methylated+purines+in+DNA+in+vivo+and+in+vitro.&amp;auth=&amp;atitle=Effect+of+acute+doses+of+2-acetylaminofluorene+on+the+capacity+of+rat+liver+to+repair+methylated+purines+in+DNA+in+vivo+and+in+vitro." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+acute+doses+of+2-acetylaminofluorene+on+the+capacity+of+rat+liver+to+repair+methylated+purines+in+DNA+in+vivo+and+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+acute+doses+of+2-acetylaminofluorene+on+the+capacity+of+rat+liver+to+repair+methylated+purines+in+DNA+in+vivo+and+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001252-Lake1" id="pone.0001252-Lake1"></a>Lake RS, Kropko ML, McLachlan S, Pezzutti MR, Shoemaker RH, et al.  (1980) Chemical carcinogen induction of DNA-repair synthesis in human peripheral blood monocytes. Mutat Res  74: 357–377.  <ul class="find" data-citedArticleID="1025014" data-doi="10.1016/0165-1161(80)90194-6"><li><a href="http://dx.doi.org/10.1016/0165-1161(80)90194-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chemical+carcinogen+induction+of+DNA-repair+synthesis+in+human+peripheral+blood+monocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chemical+carcinogen+induction+of+DNA-repair+synthesis+in+human+peripheral+blood+monocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001252-Montesano1" id="pone.0001252-Montesano1"></a>Montesano R, Bresil H, Planche-Martel G, Margison GP, Pegg AE (1980) Effect of chronic treatment of rats with dimethylnitrosamine on the removal of O6-methylguanine from DNA. Cancer Res  40: 452–458.  <ul class="find" data-citedArticleID="1025020"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Effect+of+chronic+treatment+of+rats+with+dimethylnitrosamine+on+the+removal+of+O6-methylguanine+from+DNA.&amp;auth=&amp;atitle=Effect+of+chronic+treatment+of+rats+with+dimethylnitrosamine+on+the+removal+of+O6-methylguanine+from+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Effect+of+chronic+treatment+of+rats+with+dimethylnitrosamine+on+the+removal+of+O6-methylguanine+from+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Effect+of+chronic+treatment+of+rats+with+dimethylnitrosamine+on+the+removal+of+O6-methylguanine+from+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001252-OConnor1" id="pone.0001252-OConnor1"></a>O'Connor PJ, Chu YH, Cooper DP, Maru GB, Smith RA, et al.  (1982) Species differences in the inducibility of hepatic O6-alkylguanine repair in rodents. Biochimie  64: 769–773.  <ul class="find" data-citedArticleID="1025026" data-doi="10.1016/s0300-9084(82)80127-2"><li><a href="http://dx.doi.org/10.1016/s0300-9084(82)80127-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species+differences+in+the+inducibility+of+hepatic+O6-alkylguanine+repair+in+rodents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species+differences+in+the+inducibility+of+hepatic+O6-alkylguanine+repair+in+rodents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001252-Samson1" id="pone.0001252-Samson1"></a>Samson L, Cairns J (1977) A new pathway for DNA repair in Escherichia coli. Nature  267: 281–283.  <ul class="find" data-citedArticleID="1025034" data-doi="10.1038/267281a0"><li><a href="http://dx.doi.org/10.1038/267281a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+pathway+for+DNA+repair+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+pathway+for+DNA+repair+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001252-Samson2" id="pone.0001252-Samson2"></a>Samson L, Thomale J, Rajewsky MF (1988) Alternative pathways for the in vivo repair of O6-alkylguanine and O4-alkylthymine in Escherichia coli: the adaptive response and nucleotide excision repair. Embo J  7: 2261–2267.  <ul class="find" data-citedArticleID="1025036"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Alternative+pathways+for+the+in+vivo+repair+of+O6-alkylguanine+and+O4-alkylthymine+in+Escherichia+coli%3A+the+adaptive+response+and+nucleotide+excision+repair.&amp;auth=&amp;atitle=Alternative+pathways+for+the+in+vivo+repair+of+O6-alkylguanine+and+O4-alkylthymine+in+Escherichia+coli%3A+the+adaptive+response+and+nucleotide+excision+repair." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+pathways+for+the+in+vivo+repair+of+O6-alkylguanine+and+O4-alkylthymine+in+Escherichia+coli%3A+the+adaptive+response+and+nucleotide+excision+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+pathways+for+the+in+vivo+repair+of+O6-alkylguanine+and+O4-alkylthymine+in+Escherichia+coli%3A+the+adaptive+response+and+nucleotide+excision+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001252-Friedberg1" id="pone.0001252-Friedberg1"></a>Friedberg EC, Walker GC, Siede W (2005) DNA Repair and Mutagenesis. Washington, DC: American Society of Microbiology.   <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0001252-Glassner1" id="pone.0001252-Glassner1"></a>Glassner BJ, Rasmussen LJ, Najarian MT, Posnick LM, Samson LD (1998) Generation of a strong mutator phenotype in yeast by imbalanced base excision repair. Proc Natl Acad Sci U S A  95: 9997–10002.  <ul class="find" data-citedArticleID="1025006" data-doi="10.1073/pnas.95.17.9997"><li><a href="http://dx.doi.org/10.1073/pnas.95.17.9997" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+a+strong+mutator+phenotype+in+yeast+by+imbalanced+base+excision+repair." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+a+strong+mutator+phenotype+in+yeast+by+imbalanced+base+excision+repair.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001252-Posnick1" id="pone.0001252-Posnick1"></a>Posnick LM, Samson LD (1999) Imbalanced base excision repair increases spontaneous mutation and alkylation sensitivity in Escherichia coli. J Bacteriol  181: 6763–6771.  <ul class="find" data-citedArticleID="1025028"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Imbalanced+base+excision+repair+increases+spontaneous+mutation+and+alkylation+sensitivity+in+Escherichia+coli.&amp;auth=&amp;atitle=Imbalanced+base+excision+repair+increases+spontaneous+mutation+and+alkylation+sensitivity+in+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Imbalanced+base+excision+repair+increases+spontaneous+mutation+and+alkylation+sensitivity+in+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Imbalanced+base+excision+repair+increases+spontaneous+mutation+and+alkylation+sensitivity+in+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001252-Posnick2" id="pone.0001252-Posnick2"></a>Posnick LM, Samson LD (1999) Influence of S-adenosylmethionine pool size on spontaneous mutation, dam methylation, and cell growth of Escherichia coli. J Bacteriol  181: 6756–6762.  <ul class="find" data-citedArticleID="1025030"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Influence+of+S-adenosylmethionine+pool+size+on+spontaneous+mutation%2C+dam+methylation%2C+and+cell+growth+of+Escherichia+coli.&amp;auth=&amp;atitle=Influence+of+S-adenosylmethionine+pool+size+on+spontaneous+mutation%2C+dam+methylation%2C+and+cell+growth+of+Escherichia+coli." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+S-adenosylmethionine+pool+size+on+spontaneous+mutation%2C+dam+methylation%2C+and+cell+growth+of+Escherichia+coli." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+S-adenosylmethionine+pool+size+on+spontaneous+mutation%2C+dam+methylation%2C+and+cell+growth+of+Escherichia+coli.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001252-Xiao1" id="pone.0001252-Xiao1"></a>Xiao W, Samson L (1993) In vivo evidence for endogenous DNA alkylation damage as a source of spontaneous mutation in eukaryotic cells. Proc Natl Acad Sci U S A  90: 2117–2121.  <ul class="find" data-citedArticleID="1025044" data-doi="10.1073/pnas.90.6.2117"><li><a href="http://dx.doi.org/10.1073/pnas.90.6.2117" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+evidence+for+endogenous+DNA+alkylation+damage+as+a+source+of+spontaneous+mutation+in+eukaryotic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+evidence+for+endogenous+DNA+alkylation+damage+as+a+source+of+spontaneous+mutation+in+eukaryotic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001252-Memisoglu1" id="pone.0001252-Memisoglu1"></a>Memisoglu A, Samson L (2000) Base excision repair in yeast and mammals. Mutat Res  451: 39–51.  <ul class="find" data-citedArticleID="1025018" data-doi="10.1016/s0027-5107(00)00039-7"><li><a href="http://dx.doi.org/10.1016/s0027-5107(00)00039-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Base+excision+repair+in+yeast+and+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Base+excision+repair+in+yeast+and+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001252-Lee1" id="pone.0001252-Lee1"></a>Lee I, Date SV, Adai AT, Marcotte EM (2004) A probabilistic functional network of yeast genes. Science  306: 1555–1558.  <ul class="find" data-citedArticleID="1025016" data-doi="10.1126/science.1099511"><li><a href="http://dx.doi.org/10.1126/science.1099511" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+probabilistic+functional+network+of+yeast+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+probabilistic+functional+network+of+yeast+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001252-Gasch2" id="pone.0001252-Gasch2"></a>Gasch AP, Huang M, Metzner S, Botstein D, Elledge SJ, et al.  (2001) Genomic expression responses to DNA-damaging agents and the regulatory role of the yeast ATR homolog Mec1p. Mol Biol Cell  12: 2987–3003.  <ul class="find" data-citedArticleID="1025004" data-doi="10.1091/mbc.12.10.2987"><li><a href="http://dx.doi.org/10.1091/mbc.12.10.2987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+expression+responses+to+DNA-damaging+agents+and+the+regulatory+role+of+the+yeast+ATR+homolog+Mec1p." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+expression+responses+to+DNA-damaging+agents+and+the+regulatory+role+of+the+yeast+ATR+homolog+Mec1p.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001252-Aguilaniu1" id="pone.0001252-Aguilaniu1"></a>Aguilaniu H, Gustafsson L, Rigoulet M, Nystrom T (2001) Protein oxidation in G0 cells of Saccharomyces cerevisiae depends on the state rather than rate of respiration and is enhanced in pos9 but not yap1 mutants. J Biol Chem  276: 35396–35404.  <ul class="find" data-citedArticleID="1024994" data-doi="10.1074/jbc.m101796200"><li><a href="http://dx.doi.org/10.1074/jbc.m101796200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+oxidation+in+G0+cells+of+Saccharomyces+cerevisiae+depends+on+the+state+rather+than+rate+of+respiration+and+is+enhanced+in+pos9+but+not+yap1+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+oxidation+in+G0+cells+of+Saccharomyces+cerevisiae+depends+on+the+state+rather+than+rate+of+respiration+and+is+enhanced+in+pos9+but+not+yap1+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001252-Nakamura1" id="pone.0001252-Nakamura1"></a>Nakamura J, La DK, Swenberg JA (2000) 5′-nicked apurinic/apyrimidinic sites are resistant to beta-elimination by beta-polymerase and are persistent in human cultured cells after oxidative stress. J Biol Chem  275: 5323–5328.  <ul class="find" data-citedArticleID="1025022" data-doi="10.1074/jbc.275.8.5323"><li><a href="http://dx.doi.org/10.1074/jbc.275.8.5323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=5%E2%80%B2-nicked+apurinic%2Fapyrimidinic+sites+are+resistant+to+beta-elimination+by+beta-polymerase+and+are+persistent+in+human+cultured+cells+after+oxidative+stress." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%225%E2%80%B2-nicked+apurinic%2Fapyrimidinic+sites+are+resistant+to+beta-elimination+by+beta-polymerase+and+are+persistent+in+human+cultured+cells+after+oxidative+stress.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001252-Nakamura2" id="pone.0001252-Nakamura2"></a>Nakamura J, Swenberg JA (1999) Endogenous apurinic/apyrimidinic sites in genomic DNA of mammalian tissues. Cancer Res  59: 2522–2526.  <ul class="find" data-citedArticleID="1025024"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Endogenous+apurinic%2Fapyrimidinic+sites+in+genomic+DNA+of+mammalian+tissues.&amp;auth=&amp;atitle=Endogenous+apurinic%2Fapyrimidinic+sites+in+genomic+DNA+of+mammalian+tissues." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endogenous+apurinic%2Fapyrimidinic+sites+in+genomic+DNA+of+mammalian+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endogenous+apurinic%2Fapyrimidinic+sites+in+genomic+DNA+of+mammalian+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001252-Tran1" id="pone.0001252-Tran1"></a>Tran HT, Keen JD, Kricker M, Resnick MA, Gordenin DA (1997) Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants. Mol Cell Biol  17: 2859–2865.  <ul class="find" data-citedArticleID="1025040"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hypermutability+of+homonucleotide+runs+in+mismatch+repair+and+DNA+polymerase+proofreading+yeast+mutants.&amp;auth=&amp;atitle=Hypermutability+of+homonucleotide+runs+in+mismatch+repair+and+DNA+polymerase+proofreading+yeast+mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hypermutability+of+homonucleotide+runs+in+mismatch+repair+and+DNA+polymerase+proofreading+yeast+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hypermutability+of+homonucleotide+runs+in+mismatch+repair+and+DNA+polymerase+proofreading+yeast+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001252-Irizarry1" id="pone.0001252-Irizarry1"></a>Irizarry RA, Bolstad BM, Collin F, Cope LM, Hobbs B, et al.  (2003) Summaries of Affymetrix GeneChip probe level data. Nucleic Acids Res  31: e15.  <ul class="find" data-citedArticleID="1025008" data-doi="10.1093/nar/gng015"><li><a href="http://dx.doi.org/10.1093/nar/gng015" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Summaries+of+Affymetrix+GeneChip+probe+level+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Summaries+of+Affymetrix+GeneChip+probe+level+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001252-Shannon1" id="pone.0001252-Shannon1"></a>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, et al.  (2003) Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res  13: 2498–2504.  <ul class="find" data-citedArticleID="1025038" data-doi="10.1101/gr.1239303"><li><a href="http://dx.doi.org/10.1101/gr.1239303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytoscape%3A+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001252-Robinson1" id="pone.0001252-Robinson1"></a>Robinson MD, Grigull J, Mohammad N, Hughes TR (2002) FunSpec: a web-based cluster interpreter for yeast. BMC Bioinformatics  3: 35.  <ul class="find" data-citedArticleID="1025032"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=FunSpec%3A+a+web-based+cluster+interpreter+for+yeast.&amp;auth=&amp;atitle=FunSpec%3A+a+web-based+cluster+interpreter+for+yeast." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=FunSpec%3A+a+web-based+cluster+interpreter+for+yeast." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22FunSpec%3A+a+web-based+cluster+interpreter+for+yeast.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001252-Wyatt1" id="pone.0001252-Wyatt1"></a>Wyatt MD, Allan JM, Lau AY, Ellenberger TE, Samson LD (1999) 3-methyladenine DNA glycosylases: structure, function, and biological importance. Bioessays  21: 668–676.  <ul class="find" data-citedArticleID="1025042" data-doi="10.1002/(sici)1521-1878(199908)21:8<668::aid-bies6&gt;3.0.co;2-d"><li><a href="http://dx.doi.org/10.1002/(sici)1521-1878(199908)21:8<668::aid-bies6&gt;3.0.co;2-d" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=3-methyladenine+DNA+glycosylases%3A+structure%2C+function%2C+and+biological+importance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%223-methyladenine+DNA+glycosylases%3A+structure%2C+function%2C+and+biological+importance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.XML" value="72486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.PDF" value="513959"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g001.PNG_L" value="80477"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g001.PNG_M" value="93520"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g001.PNG_S" value="11268"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g001.TIF" value="773134"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g001.PNG_I" value="36727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.t001.PNG_L" value="79963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.t001.PNG_M" value="109838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.t001.PNG_S" value="14464"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.t001.TIF" value="311076"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.t001.PNG_I" value="35994"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g002.PNG_L" value="182783"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g002.PNG_M" value="34060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g002.PNG_S" value="7635"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g002.TIF" value="374760"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g002.PNG_I" value="11692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g003.PNG_L" value="210375"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g003.PNG_M" value="28065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g003.PNG_S" value="6949"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g003.TIF" value="392422"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g003.PNG_I" value="18785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g004.PNG_L" value="561365"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g004.PNG_M" value="131896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g004.PNG_S" value="11917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g004.TIF" value="1016304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g004.PNG_I" value="47461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g005.PNG_L" value="1375830"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g005.PNG_M" value="249371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g005.PNG_S" value="15123"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g005.TIF" value="1795354"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g005.PNG_I" value="83135"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g006.PNG_L" value="87630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g006.PNG_M" value="101409"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g006.PNG_S" value="8891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g006.TIF" value="607302"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.g006.PNG_I" value="38725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.s001.EPS" value="7689107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001252.s002.XLS" value="213504"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001252&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001252" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001252&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001252&volume=&issue=&title=Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates&author_name=Ivan%20Rusyn%2C%20Rebecca%20C.%20Fry%2C%20Thomas%20J.%20Begley%2C%20Joanna%20Klapacz%2C%20J.%20Peter%20Svensson%2C%20Mark%20Ambrose%2C%20Leona%20D.%20Samson&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001252" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252&amp;t=Transcriptional%20Networks%20in%20S.%20cerevisiae%20Linked%20to%20an%20Accumulation%20of%20Base%20Excision%20Repair%20Intermediates" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252&title=Transcriptional%20Networks%20in%20S.%20cerevisiae%20Linked%20to%20an%20Accumulation%20of%20Base%20Excision%20Repair%20Intermediates&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252&amp;title=Transcriptional%20Networks%20in%20S.%20cerevisiae%20Linked%20to%20an%20Accumulation%20of%20Base%20Excision%20Repair%20Intermediates" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001252&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Transcriptional Networks in S. cerevisiae Linked to an Accumulation of Base Excision Repair Intermediates';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001252';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Transcriptional%20Networks%20in%20S.%20cerevisiae%20Linked%20to%20an%20Accumulation%20of%20Base%20Excision%20Repair%20Intermediates http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001252" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001252" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Base+excision+repair%22" title="Search for articles in the subject area:'Base excision repair'"><div class="flagText">Base excision repair</div></a>
              <div data-categoryid="37975" data-articleid="25336"
                   data-categoryname="Base excision repair"
                   class="flagImage" title="Flag 'Base excision repair' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Biosynthesis%22" title="Search for articles in the subject area:'Biosynthesis'"><div class="flagText">Biosynthesis</div></a>
              <div data-categoryid="40369" data-articleid="25336"
                   data-categoryname="Biosynthesis"
                   class="flagImage" title="Flag 'Biosynthesis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="25336"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+damage%22" title="Search for articles in the subject area:'DNA damage'"><div class="flagText">DNA damage</div></a>
              <div data-categoryid="17965" data-articleid="25336"
                   data-categoryname="DNA damage"
                   class="flagImage" title="Flag 'DNA damage' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+repair%22" title="Search for articles in the subject area:'DNA repair'"><div class="flagText">DNA repair</div></a>
              <div data-categoryid="17453" data-articleid="25336"
                   data-categoryname="DNA repair"
                   class="flagImage" title="Flag 'DNA repair' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="25336"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Damage+mechanics%22" title="Search for articles in the subject area:'Damage mechanics'"><div class="flagText">Damage mechanics</div></a>
              <div data-categoryid="18039" data-articleid="25336"
                   data-categoryname="Damage mechanics"
                   class="flagImage" title="Flag 'Damage mechanics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Yeast%22" title="Search for articles in the subject area:'Yeast'"><div class="flagText">Yeast</div></a>
              <div data-categoryid="43039" data-articleid="25336"
                   data-categoryname="Yeast"
                   class="flagImage" title="Flag 'Yeast' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5106'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=3047'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=6365&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
