

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000547"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000547" />

    <meta name="citation_title" content="Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations"/>
    <meta itemprop="name" content="Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations"/>

      <meta name="citation_author" content="Diego Calzolari"/>
            <meta name="citation_author_institution" content="Burnham Institute for Medical Research, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Giovanni Paternostro"/>
            <meta name="citation_author_institution" content="Burnham Institute for Medical Research, La Jolla, California, United States of America"/>
      <meta name="citation_author" content="Patrick L. Harrington Jr."/>
            <meta name="citation_author_institution" content="Physics and Astronomy Department, Michigan State University, East Lansing, Michigan, United States of America"/>
      <meta name="citation_author" content="Carlo Piermarocchi"/>
            <meta name="citation_author_institution" content="Physics and Astronomy Department, Michigan State University, East Lansing, Michigan, United States of America"/>
      <meta name="citation_author" content="Phillip M. Duxbury"/>
            <meta name="citation_author_institution" content="Physics and Astronomy Department, Michigan State University, East Lansing, Michigan, United States of America"/>

    <meta name="citation_date" content="2007/6/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000547.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e547"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Systems biology and combination therapy in the quest for clinical efficacy.; citation_author=J Fitzgerald; citation_author=B Schoeber; citation_author=UB Nielsen; citation_author=PK Sorger; citation_journal_title=Nature Chemical Biology; citation_volume=2; citation_number=1; citation_pages=458-466; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Apoptosis: A link between cancer genetics and chemotherapy.; citation_author=RW Johnstone; citation_author=AA Ruefli; citation_author=SW Lowe; citation_journal_title=Cell; citation_volume=108; citation_number=2; citation_pages=153-164; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Signal transduction therapy for cancer - whither now.; citation_author=S Klein; citation_author=A Levitzki; citation_journal_title=Current Signal Transduction Therapy; citation_volume=1; citation_number=3; citation_pages=1-12; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular targeting of cell death signal transduction pathways in cancer.; citation_author=I Petak; citation_author=JA Houghton; citation_author=L Kopper; citation_journal_title=Current Signal Transduction Therapy; citation_volume=1; citation_number=4; citation_pages=113-131; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Biomarkers for apoptosis in Alzheimer's disease.; citation_author=M Ankarcrona; citation_author=B Winblad; citation_journal_title=International Journal of Geriatric Psychiatry; citation_volume=20; citation_number=5; citation_pages=101-105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pinning down cell signaling, cancer and Alzheimer's disease.; citation_author=KP Lu; citation_journal_title=TRENDS in Biochemical Sciences; citation_volume=29; citation_number=6; citation_pages=200-209; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Caspase gene expression in the brain as a function of the clinical progression of Alzheimer's disease.; citation_author=PN Pompl; citation_author=S Yemul; citation_author=ZM Xiang; citation_author=L Ho; citation_author=V Haroutunian; citation_journal_title=Archives of Neurology; citation_volume=60; citation_number=7; citation_pages=369-376; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Iterative reconstruction of transcriptional regulatory networks: An algorithmic approach.; citation_author=CL Barrett; citation_author=BO Palsson; citation_journal_title=PLOS computational biology; citation_volume=2; citation_number=8; citation_pages=429-438; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reverse engineering the gap gene network of drosophila melanogaster.; citation_author=TJ Perkins; citation_author=J Jaeger; citation_author=L Glass; citation_journal_title=PLOS computational biology; citation_volume=2; citation_number=9; citation_pages=417-428; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A computational framework for the topological analysis and targeted disruption of signal transduction networks.; citation_author=S Dealy; citation_author=S Kauffman; citation_author=J Socolar; citation_journal_title=Complexity; citation_volume=11; citation_number=10; citation_pages=52-60; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The yeast cell-cycle network is robustly designed.; citation_author=F Li; citation_author=T Long; citation_author=Y Lu; citation_author=Q Ouyang; citation_author=C Tang; citation_journal_title=Proceedings of the National Academy of Science U S A; citation_volume=101; citation_number=11; citation_pages=4781-4786; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Eukaryotic cells are dynamically ordered or critical but not chaotic.; citation_author=I Shmulevich; citation_author=SA Kauffman; citation_author=M Aldana; citation_journal_title=Proceedings of the National Academy of Science U S A; citation_volume=102; citation_number=12; citation_pages=13439-13444; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Physiochemical modeling of cell signaling pathways.; citation_author=BB Aldridge; citation_author=JM Burke; citation_author=DA Lauffenburger; citation_author=PK Sorger; citation_journal_title=Nature Cell Biology; citation_volume=8; citation_number=13; citation_pages=1195-1203; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Cell-signalling dynamics in time and space.; citation_author=BN Kholodenko; citation_journal_title=Nature Reviews Molecular Cell Biology; citation_volume=7; citation_number=14; citation_pages=165-176; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Characterization and modeling of protein-protein interaction networks.; citation_author=V Colizza; citation_author=A Flammini; citation_author=A Maritan; citation_author=A Vespignani; citation_journal_title=Physica A; citation_volume=352; citation_number=15; citation_pages=1-27; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Number of attractors in random boolean networks.; citation_author=B Drossel; citation_journal_title=Physical Review E; citation_volume=72; citation_number=16; citation_pages=16110; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cell fates as high-dimensional attractor states of a complex gene regulatory network.; citation_author=S Huang; citation_author=G Eichler; citation_author=Y Bar-Yam; citation_author=DE Ingber; citation_journal_title=Physical Review Letters; citation_volume=94; citation_number=17; citation_pages=128701; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A computational framework for the topological analysis and targeted disruption of signal transduction networks.; citation_author=K Klemm; citation_author=S Bornholdt; citation_journal_title=Physical Review E; citation_volume=72; citation_number=18; citation_pages=55101; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dynamic pattern evolution on scale-free networks.; citation_author=H Zhou; citation_author=R Lipowsky; citation_journal_title=Proceedings of the National Academy of Science USA; citation_volume=102; citation_number=19; citation_pages=10052-10057; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bistability in apoptosis: Roles of bax, bcl-2 and mitochondrial permeability transition pores.; citation_author=EZ Bagci; citation_author=Y Vodovotz; citation_author=TR Billiar; citation_author=GB Ermentrout; citation_author=I Bahar; citation_journal_title=Biophysical Journal; citation_volume=91; citation_number=20; citation_pages=1546-1549; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Nonlinear protein degradation and the function of genetic circuits.; citation_author=NE Buchler; citation_author=U Gerland; citation_author=T Hwa; citation_journal_title=Proceedings of the National Academy of Science USA; citation_volume=102; citation_number=21; citation_pages=9559-9564; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Robustness properties of apoptosis models with respect to parameter variations and intrinsic noise.; citation_author=T Eißing; citation_author=F Allgöwer; citation_author=E Bullinger; citation_journal_title=IEEE Proceedings on Systems Biology; citation_volume=152; citation_number=22; citation_pages=221-228; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Regulation of gene expression in flux balance models of metabolism.; citation_author=MW Covert; citation_author=CH Schilling; citation_author=B Palsson; citation_journal_title=Journal of Theoretical Biology; citation_volume=213; citation_number=23; citation_pages=73-78; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Flux balance analysis in the era of metabolomics.; citation_author=JM Lee; citation_author=EP Gianchandani; citation_author=JA Papin; citation_journal_title=Briefings in Bioinformatics; citation_volume=7; citation_number=24; citation_pages=140-150; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Apoptosis-based therapies for hematologic malignancies.; citation_author=JC Reed; citation_author=M Pellecchia; citation_journal_title=Blood; citation_volume=106; citation_number=25; citation_pages=408-418; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The topology of the regulatory interactions predicts the expression pattern of the segment polarity genes in drosophila melanogaster.; citation_author=R Albert; citation_author=HG Othmer; citation_journal_title=Journal of Theoretical Biology; citation_volume=223; citation_number=26; citation_pages=1-18; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Global reconstruction of the human metabolic network based on genomic and bibliomic data.; citation_author=NC Duarte; citation_author=SA Becker; citation_author=N Jamshidi; citation_author=NI Thiele; citation_author=ML Mo; citation_journal_title=Proceedings of the National Academy of Science U S A; citation_volume=104; citation_number=27; citation_pages=1777-1782; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=SA Kauffman; citation_number=28; citation_date=1993; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=The hill equation revisited: uses and misuses.; citation_author=JN Weiss; citation_journal_title=FASEB Journal; citation_volume=11; citation_number=29; citation_pages=835-841; citation_date=1997; " />
      <meta name="citation_reference" content="citation_author=TH Cormen; citation_author=CE Leiserson; citation_author=RL Rivest; citation_author=C Stein; citation_number=30; citation_date=2001; citation_publisher=MIT Press and McGraw-Hill; " />
      <meta name="citation_reference" content="citation_title=Optimal identification of biochemical reaction networks.; citation_author=XJ Feng; citation_author=H Rabitz; citation_journal_title=Biophysical Journal; citation_volume=86; citation_number=31; citation_pages=1270-1281; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Optimizing genetic circuits by global sensitivity analysis.; citation_author=S Hooshangi; citation_author=XJ Feng; citation_author=D Chen; citation_author=G Li; citation_author=R Weiss; citation_author=H Rabitz; citation_journal_title=Biophysical Journal; citation_volume=87; citation_number=32; citation_pages=2195-2202; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Closed-loop learning control of bio-networks.; citation_author=J Ku; citation_author=X Feng; citation_author=H Rabitz; citation_journal_title=Journal of Computational Biology; citation_volume=11; citation_number=33; citation_pages=642-659; citation_date=2004; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations"/>
    <meta name="twitter:description" content="BackgroundSelective control in a population is the ability to control a member of the population while leaving the other members relatively unaffected. The concept of selective control is developed using cell death or apoptosis in heterogeneous cell populations as an example. Control of apoptosis is essential in a variety of therapeutic environments, including cancer where cancer cell death is a desired outcome and Alzheimer's disease where neuron survival is the desired outcome. However, in both cases these responses must occur with minimal response in other cells exposed to treatment; that is, the response must be selective.Methodology and Principal FindingsApoptosis signaling in heterogeneous cells is described by an ensemble of gene networks with identical topology but different link strengths. Selective control depends on the statistics of signaling in the ensemble of networks, and we analyze the effects of superposition, non-linearity and feedback on these statistics. Parallel pathways promote normal statistics while series pathways promote skew distributions, which in the most extreme cases become log-normal. We also show that feedback and non-linearity can produce bimodal signaling statistics, as can discreteness and non-linearity. Two methods for optimizing selective control are presented. The first is an exhaustive search method and the second is a linear programming based approach. Though control of a single gene in the signaling network yields little selectivity, control of a few genes typically yields higher levels of selectivity. The statistics of gene combinations susceptible to selective control in heterogeneous apoptosis networks is studied and is used to identify general control strategies.Conclusions and SignificanceWe have explored two methods for the study of selectivity in cell populations. The first is an exhaustive search method limited to three node perturbations. The second is an effective linear model, based on interpolation of single node sensitivity, in which the selective combinations can be found by linear programming optimization. We found that selectivity is promoted by acting on the least sensitive nodes in the case of weak populations, while selective control of robust populations is optimized through perturbations of more sensitive nodes. High throughput experiments with heterogeneous cell lines could be designed in an analogous manner, with the further possibility of incorporating the selectivity optimization process into a closed-loop control system."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000547.g016"/>

  <meta property="og:title" content="Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=5413'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7817'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=510&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000547">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000547" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000547&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Diego Calzolari, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Burnham Institute for Medical Research, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Giovanni Paternostro, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Burnham Institute for Medical Research, La Jolla, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick L. Harrington Jr., 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Physics and Astronomy Department, Michigan State University, East Lansing, Michigan, United States of America
                </p>

                <p>
                    Current address: Bioinformatics Graduate Program, University of Michigan, Ann Arbor, Michigan, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Carlo Piermarocchi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Physics and Astronomy Department, Michigan State University, East Lansing, Michigan, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Phillip M. Duxbury
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:duxbury@pa.msu.edu">duxbury@pa.msu.edu</a></p>

                <p>Affiliation:
                  Physics and Astronomy Department, Michigan State University, East Lansing, Michigan, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: June 20, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000547</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000547-g001" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-t001" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g002" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g003" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g004" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g005" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g006" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g007" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g008" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g009" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g009" title="Figure 9">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g009&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-t002" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g010" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g010" title="Figure 10">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g010&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g011" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g011" title="Figure 11">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g011&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g012" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g012" title="Figure 12">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g012&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g013" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g013" title="Figure 13">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g013&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g014" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g014" title="Figure 14">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g014&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g015" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g015" title="Figure 15">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g015&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000547-g016" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g016" title="Figure 16">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g016&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000547">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000547" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Selective control in a population is the ability to control a member of the population while leaving the other members relatively unaffected. The concept of selective control is developed using cell death or apoptosis in heterogeneous cell populations as an example. Control of apoptosis is essential in a variety of therapeutic environments, including cancer where cancer cell death is a desired outcome and Alzheimer's disease where neuron survival is the desired outcome. However, in both cases these responses must occur with minimal response in other cells exposed to treatment; that is, the response must be selective.</p>


<h3>Methodology and Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Apoptosis signaling in heterogeneous cells is described by an ensemble of gene networks with identical topology but different link strengths. Selective control depends on the statistics of signaling in the ensemble of networks, and we analyze the effects of superposition, non-linearity and feedback on these statistics. Parallel pathways promote normal statistics while series pathways promote skew distributions, which in the most extreme cases become log-normal. We also show that feedback and non-linearity can produce bimodal signaling statistics, as can discreteness and non-linearity. Two methods for optimizing selective control are presented. The first is an exhaustive search method and the second is a linear programming based approach. Though control of a single gene in the signaling network yields little selectivity, control of a few genes typically yields higher levels of selectivity. The statistics of gene combinations susceptible to selective control in heterogeneous apoptosis networks is studied and is used to identify general control strategies.</p>


<h3>Conclusions and Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>We have explored two methods for the study of selectivity in cell populations. The first is an exhaustive search method limited to three node perturbations. The second is an effective linear model, based on interpolation of single node sensitivity, in which the selective combinations can be found by linear programming optimization. We found that selectivity is promoted by acting on the least sensitive nodes in the case of weak populations, while selective control of robust populations is optimized through perturbations of more sensitive nodes. High throughput experiments with heterogeneous cell lines could be designed in an analogous manner, with the further possibility of incorporating the selectivity optimization process into a closed-loop control system.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Calzolari D, Paternostro G, Harrington PL Jr, Piermarocchi C, Duxbury PM (2007) Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations. PLoS ONE 2(6):
          e547.
            doi:10.1371/journal.pone.0000547</p><p><strong>Academic Editor: </strong>Mark Cookson, Laboratory of Neurogenetics, National Institutes of Health, United States of America</p><p><strong>Received:</strong> March 28, 2007; <strong>Accepted:</strong> May 9, 2007; <strong>Published:</strong> June 20, 2007</p><p><strong>Copyright:</strong> © 2007 Calzolari et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work was supported by the Burnham Institute and by Michigan State University. The funders had no role in the work described in the manuscript or in its preparation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Living cells carry out their functions, such as working, reproducing and dying, by appropriate response to extracellular and intracellular inputs to a complex network of signaling pathways. Genes which code for the proteins in these pathways are controlled by regulatory proteins which up-regulate or down-regulate these genes, depending on inputs to the signaling network. The enormous effort currently directed at understanding signaling networks may be subdivided into two areas, firstly extracting faithful wiring diagrams for the networks and, secondly developing methods to understand and control the messages which pass through them. In this contribution we develop the concept of selective control in diverse cell populations, and introduce computational methods which optimize selectivity for a particular signal for a designated member of a cell population.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>We introduce the concept of selective control or selectivity in cell populations as the requirement of finding a set of inputs which induce one member of the population to produce a desired response while ensuring that the remaining members of the population have a minimal response. In the case of apoptosis or cell death, which we use as an illustrative example, we consider a population of cells and seek methods to kill a selected member of the population while ensuring the survival of the others in the population. Control of apoptosis is essential in a variety of therapeutic environments including cancer where cancer cell death is a desired outcome<a href="#pone.0000547-Fitzgerald1">[1]</a>–<a href="#pone.0000547-Petak1">[4]</a> and Alzheimer's disease where neuron survival is the desired outcome<a href="#pone.0000547-Ankarcrona1">[5]</a>–<a href="#pone.0000547-Pompl1">[7]</a>. However in both cases these responses must occur with minimal response in other cells exposed to treatment that is, the response must be selective.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Though striking progress is occuring in the extraction of networks using a range of experimental data<a href="#pone.0000547-Barrett1">[8]</a>, <a href="#pone.0000547-Perkins1">[9]</a>, knowledge of signaling networks remains predominantly at the level of topology rather than detailed knowledge of the rate constants and non-linear message passing which occurs in the networks. Models to distinguish between members of a population of cells, for example different cancer cells and different normal tissue types, require differences in message passing parameters and/or expression levels of the genes in the network. Here, the computational procedures for selectivity in cell populations are elucidated using heterogeneous populations, where members of the population are distinguished by having message-passing efficiencies drawn from homogeneous random distributions.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Models of message passing in gene networks range from binary models with discrete message passing rules<a href="#pone.0000547-Dealy1">[10]</a>–<a href="#pone.0000547-Shmulevich1">[12]</a> to non-linear ordinary differential equations<a href="#pone.0000547-Aldridge1">[13]</a> and to stochastic spatio-temporal models<a href="#pone.0000547-Kholodenko1">[14]</a> which are simulated using partial differential equations or Monte Carlo methods. Questions of interest also vary greatly, from generic questions about the number of attractors and their stability in random networks<a href="#pone.0000547-Colizza1">[15]</a>–<a href="#pone.0000547-Zhou1">[19]</a> to modeling the detailed dynamics of gene concentrations in particular pathways<a href="#pone.0000547-Bagci1">[20]</a>–<a href="#pone.0000547-Eiing1">[22]</a>, and to the cellular response such as control of flagellar rotation in bacteria responding to chemotaxis. Some of the tools developed for the analysis of metabolic networks, both dynamically and using steady state flux balance approaches, can be profitably extended to signaling networks<a href="#pone.0000547-Covert1">[23]</a>, <a href="#pone.0000547-Lee1">[24]</a>. In the flux balance approach (FBA), the ouput of a cell may be optimized with respect to an objective function and subject to the constraints of flux balance at each node in the network.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Though there are conceptual similarities between the FBA and our approach, there are also critical differences. Instead of flux balance, we require message passing rules which describe how a gene responds to the state of its neighbors. To illustrate the effects of different rules, we use an important example from systems biology, the apoptosis network. In particular we discuss the statistics of death signals produced by continuum and discrete message passing rules in this network. We also develop the concept of selective control. Rather than optimizing the objective for one cell or metabolic network, as occurs in FBA, we seek to optimize the response of one cell in a population while minimizing the response of other cells in the population. Selective control is demonstrated using exhaustive search over drug combinations in discrete models and using an approximate linear programming approach. Though drugs affecting specifically every node of the apoptosis network are not yet available, this is a very active field of pharmacological research and it is probably one of the biological networks where this ideal situation, from the control point of view, is closest to reality<a href="#pone.0000547-Reed1">[25]</a>. Moreover, the network we use is probably still an incomplete representation of the apoptosis network, both for the topology and for the kinetic parameters. Nevertheless, several authors have shown that useful results can be obtained from partially characterized models of biological networks<a href="#pone.0000547-Albert1">[26]</a>, <a href="#pone.0000547-Duarte1">[27]</a>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Statistics of signaling</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We have modified the apoptosis network, hsa04210, of the Kegg database to that presented in <a href="#pone-0000547-g001">Fig. 1</a>, where network complexes consisting of several genes are split into individual nodes. Recent work emphasizes the importance of positive feedback between CASP3 and CASP8 denoted by the long dashed arrow in the figure<a href="#pone.0000547-Bagci1">[20]</a>, <a href="#pone.0000547-Eiing1">[22]</a>, though the importance of this feedback is not universally accepted. There are 47 genes in <a href="#pone-0000547-g001">Figure 1</a> and an additional node which we label the output node. The 47 genes may be catagorized (see <a href="#pone-0000547-t001">Table 1</a>) as: input genes (dashed circles in the <a href="#pone-0000547-g001">Fig. 1</a>), which transmit signals to the network from the other parts of the cellular network; membrane genes which code for membrane proteins and complexes bound to membrane proteins; death genes which signal onset and execution of apoptosis; life genes which reduce apoptotic signals and are upregulated in many cancer cells; and finally the remaining genes in the network. The output, or “death” node (48 in <a href="#pone-0000547-g001">Fig. 1</a>) is added to represent the cumulative effect of many genes implicated in the onset of the cell death machinery.</p>
<div class="figure" id="pone-0000547-g001"><div class="img"><a name="pone-0000547-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The human apoptosis network modified, as described in the text, from that in the Kegg database (hsa04210).</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>There are 47 genes in the figure and an additional node which we label the output node. The 47 genes may be catagorized (see <a href="#pone-0000547-t001">Table 1</a>) as: input genes (dashed circles); membrane genes; death genes; life genes; and finally the remaining genes in the network. The output, or “death” node (48) is added to represent the cumulative effect of many genes implicated in the onset of the cell death machinery. In this figure, solid lines indicate promotion while short dashed links indicate inhibition. The long dashed link between CASP3 and CASP8 adds the possibility of feedback in the apoptosome.</p>
<span>doi:10.1371/journal.pone.0000547.g001</span></div><div class="figure" id="pone-0000547-t001"><div class="img"><a name="pone-0000547-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Typical roles of genes in the signaling network.</span></strong></p><span>doi:10.1371/journal.pone.0000547.t001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We consider two types of models, those where the gene activities, <em>a<sub>i</sub></em>, are continuous and those with discrete gene activities, <em>m<sub>i</sub></em>. We use models with continuous activities to illustrate the generic statistics of signal propagation in the apoptosis network, while discrete models are more convenient for the exhaustive search methods used in the selectivity studies. In the discrete models the gene activity has discrete values up to a maximum value <em>M</em>, so that <em>m<sub>l</sub></em> = 0,1…,<em>M</em>. Binary networks, where <em>M</em> = 1 have received the most attention, following the work of Kauffman<a href="#pone.0000547-Kauffman1">[28]</a>. We considered three discrete cases, <em>M</em> = 1,2,10, though here we focus upon M = 10 which is closer to the non-linear continuous behavior observed in experiment.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>Each gene receives signals from the genes that it is connected to in the signaling network of <a href="#pone-0000547-g001">Figure 1</a>. The signal arriving at a gene depends on the strength of the connections to its neighbors in the network. We define the strength of these connections to be ω<em><sub>ij</sub></em> between the <em>i<sup>th</sup></em> and <em>j<sup>th</sup></em> genes. Since the network is directed, ω<em><sub>ij</sub></em>≠ω<em><sub>ji</sub></em>. The values of ω<em><sub>ij</sub></em> are poorly characterized even in metabolic networks where they correspond to reaction rates. In the absence of detailed knowledge about these connections we take them to be random variables and in this way develop a generic understanding of signal propagation in heterogeneous cell populations. The link weights, ω<em><sub>ij</sub></em>, have positive random values for promotion links and negative random values for inhibition links.</p>


<h4>Continuous models</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>In the continuous models, the edge weights ω<em><sub>ij</sub></em> are uniform continuous random variables and each gene has activity <em>a<sub>i</sub></em> which is a continuous variable. The signal arriving at a gene is given by the sum,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>n</em>(<em>l</em>) is the set of genes which send signals to gene <em>l</em>. The signal <em>s<sub>l</sub></em> arriving at gene <em>l</em> may be positive or negative, where a negative signal implies inhibition. However, the activity of a gene must be positive or zero so that a negative signal at a gene implies complete inhibition and the gene is switched off, so that its activity is set to zero. This is a basic non-linearity in signaling networks.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>In addition, gene activity levels are often observed to depend in a non-linear way on the signals arriving at the gene. A common approximation to the nonlinear response in gene activity is the Hill equation<a href="#pone.0000547-Weiss1">[29]</a>,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>where <em>c</em>/<em>d</em> is the saturation value of the gene activity, <em>d</em> determines the onset of saturation and the exponent <em>b</em> is the cooperativity index. The case, <em>b</em> = 1, is Michaelis-Menten behavior characteristic of a chemical reaction in the presence of a substrate. The simplest case <em>b</em> = 1, <em>c</em> = 1, <em>d</em> = 0 is the majority rule signaling procedure, given by <em>a<sub>l</sub></em> = <em>s<sub>l</sub></em> for <em>s<sub>l</sub></em> positive and <em>a<sub>l</sub></em> = 0 for <em>s<sub>l</sub></em> negative.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>There are several procedures for simulating signal propagation through networks. In binary networks there has been considerable study of synchronous as opposed to asynchronous updates, where in the former case the gene activity levels at time <em>t</em> are used to update all of the activity levels at time <em>t</em>+1. In contrast asynchronous methods update gene activity randomly, for example one randomly chosen gene at a time, to model stochastic behavior. The number of attractors found in binary networks appears to depend on the update procedure 16]. In the absence of the feedback link between nodes 42 and 28 in <a href="#pone-0000547-g001">Fig. 1</a>, the network presented there has no loops. In this case signal propagation through the network is deterministic and non-chaotic. Furthermore signal propagation through the loopless network can be carried out in one sweep of the network by ordering the nodes according to their distance from the inputs. The nodes are then updated in order of their distance from the input, a procedure which we denote the optimal signaling algorithm(OSA). As described later, this procedure can be modified to take into account the feedback<a href="#pone.0000547-Bagci1">[20]</a>, <a href="#pone.0000547-Eiing1">[22]</a> induced by the link between CASP3 and CASP8 in <a href="#pone-0000547-g001">Fig. 1</a>. In the absence of this link, the longest path from the inputs to the output has 15 links and hence the OSA algorithm is completed in 15 time steps.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>We directly tested the effect of asynchronous, synchronous and OSA procedures on signaling in the loopless apoptosis network and found that they produce essentially the same results. This result is at first counterintuitive as undirected random networks show more complex behavior, such as chaos, and larger sets of attractors. However directed networks are deterministic so that for a given set of inputs in a network with fixed edge weights, and using deterministic message passing rules, there is a unique output. Statistical variations do occur however when the link weights are varied or stochastic noise is added to the message passing rules. Since we are interested in heterogeneous populations, which are analogous to quenched disorder, we consider variations in signaling due to variations in the edge weights.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Typical statistical behavior of signals passing through the loopless apoptosis network are presented in <a href="#pone-0000547-g002">Fig. 2</a> for an important gene, NFkB, in the interior of the network and also for the cumulative death signal at the output. These distributions are found by simulating 50000 different networks, where each network has links (values of ω<em><sub>ij</sub></em>) having weights drawn from a uniform distribution on the interval [0,1]. These simulations were carried out for a model with continuous activities <em>a<sub>i</sub></em>, where Eq. (1) is used to find the total signal arriving at a gene and we use the linear relation <em>a<sub>i</sub></em> = <em>s<sub>i</sub></em> for positive signals and <em>a<sub>i</sub></em> = 0 for negative signals. The input genes in the network were assigned random values on the interval [0,1]. Positive values of the signal arriving at the output node indicate cell death, while negative values denote life. Although the output node statistics (see <a href="#pone-0000547-g002">Fig. 2a</a>) is somewhat skew it is not too far from a normal distribution, however the statistics at NFkB (<a href="#pone-0000547-g002">Fig. 2b</a>) is highly skew and is almost lognormal. We now provide a simple explanation for the contrasting statistical behavior occurring for NFkB (34 in <a href="#pone-0000547-g001">Fig. 1</a>) and the output node (48 in <a href="#pone-0000547-g001">Fig. 1</a>).</p>
<div class="figure" id="pone-0000547-g002"><div class="img"><a name="pone-0000547-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The distribution of gene activities in heterogeneous cell populations with a population size of 50,000.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>These distribution are for majority rule signaling with positive and continuous gene activities. a) The activity of the output node is close to a normal distribution due to many pathways arriving at the output node; b) Signal statistics at the gene NFkB, which is an internal node lying at the end of a chain of links (see <a href="#pone-0000547-g001">Fig. 1</a>), is close to log-normal. (see <a href="#s3">discussion</a> in the text).</p>
<span>doi:10.1371/journal.pone.0000547.g002</span></div><a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>Simplified models elucidating the origin of the signaling statistics observed in <a href="#pone-0000547-g002">Fig. 2</a> are presented in <a href="#pone-0000547-g003">Figs. 3</a> and <a href="#pone-0000547-g004">4</a>. Many paths enter the death node (48) and this is simplified to a set of independent parallel paths in <a href="#pone-0000547-g003">Fig. 3</a>. According to Eq. (1), the death signal is then a sum of random variables and it is well known in that case that the statistics of the signal should be a normal distribution, in the asymptotic limit. The observed near normal behavior observed for the death node is then due to the fact that many parallel paths enter the death node. Deviations from the ideal normal distribution are expected for several reasons, including the fact that the activities cannot be negative, the presence of correlations in the signals entering the death node, and due to the fact that we are far from the asymptotic limit.</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7"></a><p>In contrast, NFkB is at the end of a chain of connections (see <a href="#pone-0000547-g001">Fig. 1</a>) and a simplified model of this connectivity is illustrated in <a href="#pone-0000547-g004">Figure 4</a>. In this case Eq. (1) yields,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>This is a random multiplicative process, so that if the link variables ω have random noise, then the output signal, <em>a<sub>out</sub></em> asymptotically obeys log-normal statistics. The log normal distribution, in the variable <em>x</em>, is given by,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e004&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>which is typically highly skew and exhibits large fluctuations. Here σ, μ are parameters in the distribution. The statistical variations of signals arriving at genes in complex networks clearly varies a great deal depending on the local connectivity of the genes. In cases where there are mostly linear pathways, as is believed to occur in some cancer cells, there is a greater potential for strong fluctuations in signaling statistics.</p>
<div class="figure" id="pone-0000547-g003"><div class="img"><a name="pone-0000547-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>A parallel combination of signaling pathways with no series connections.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>For a large number of parallel connections, the output activity <em>a<sub>out</sub></em> is normally distributed (see text).</p>
<span>doi:10.1371/journal.pone.0000547.g003</span></div><div class="figure" id="pone-0000547-g004"><div class="img"><a name="pone-0000547-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>A signaling pathway with four steps in series and with no parallel connections.</span></strong></p><a id="article1.body1.sec2.sec2.fig3.caption1.p1" name="article1.body1.sec2.sec2.fig3.caption1.p1"></a><p>If the number of steps in the pathway is large, the ouput activity <em>a<sub>out</sub></em> obeys log-normal statistics (see text).</p>
<span>doi:10.1371/journal.pone.0000547.g004</span></div><a id="article1.body1.sec2.sec2.p8" name="article1.body1.sec2.sec2.p8"></a><p>Non-linearity is a hallmark of genetic response and we have studied the effect of a variety of non-linear activity-signal behaviors on signaling in the apoptosis network. We tested the effect of various Hill equation parameters on the activity statistics of the apoptosis network. In these calculations, each gene has the same non-linear behavior given by Eq. (2). We found that the generic behavior was similar to that presented in <a href="#pone-0000547-g002">Fig. 2</a>. One example, where we used the Michaelis-Menten limit of Eq. (2), is presented in <a href="#pone-0000547-g005">Fig. 5</a>. The statistics of the death node (<a href="#pone-0000547-g005">Fig. 5a</a>) remain close to a normal distribution, while the statistics of NFkB remains close to log-normal. The geometry of the network thus controls the signaling statistics even in the presence of non-linearity.</p>
<div class="figure" id="pone-0000547-g005"><div class="img"><a name="pone-0000547-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Majority rule signaling with non-linearity.</span></strong></p><a id="article1.body1.sec2.sec2.fig4.caption1.p1" name="article1.body1.sec2.sec2.fig4.caption1.p1"></a><p>The distribution of signals in heterogeneous cell populations with a population size of 50,000. These distribution are for majority rule signaling with positive, continuous gene activities calculated from the signal by using the Michaelis-Menten law <em>a</em> = <em>s</em>/(1+<em>s</em>). a) Signal statistics at the output node is similar to a normal distribution; b) Signal statistics at the gene NFkB remains close to log-normal despite the non-linear dependence of the activity on the signal.</p>
<span>doi:10.1371/journal.pone.0000547.g005</span></div><a id="article1.body1.sec2.sec2.p9" name="article1.body1.sec2.sec2.p9"></a><p>An important feature absent from the Kegg apoptosis network is feedback. The heavy dashed connection between CASP3 and CASP8 (nodes 42 and 28) produces feedback which has recently been found to be important in the apoptosome<a href="#pone.0000547-Bagci1">[20]</a>, <a href="#pone.0000547-Eiing1">[22]</a>. This link leads to feedback as illustrated in the subgraph of <a href="#pone-0000547-g006">Fig. 6</a>. To elucidate the effect of feedback on signaling in the apoptosis network, we studied the response of the network in <a href="#pone-0000547-g006">Fig. 6</a> with random weights on the edges and a range of signal strengths arriving at CASP8 (a<sub>1</sub> in <a href="#pone-0000547-g006">Fig. 6</a>).</p>
<div class="figure" id="pone-0000547-g006"><div class="img"><a name="pone-0000547-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Feedback loop in the apoptosis network, following </span></strong></p><a id="article1.body1.sec2.sec2.fig5.caption1.p1" name="article1.body1.sec2.sec2.fig5.caption1.p1"></a><p><a href="#pone.0000547-Bagci1">[20]</a>.</p>
<span>doi:10.1371/journal.pone.0000547.g006</span></div><a id="article1.body1.sec2.sec2.p10" name="article1.body1.sec2.sec2.p10"></a><p>The equations for the signals, <em>s<sub>i</sub></em>(<em>t</em>+1), arriving at time <em>t</em>+1 at the five genes in <a href="#pone-0000547-g006">Fig. 6</a> are,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e005&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e006&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e007&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>In this equation <em>S</em> is the input signal and ω<em><sub>ij</sub></em> is the strength of the signaling between nodes <em>i</em> and <em>j</em>. In heterogeneous population studies these links are taken to be random. The activity of each gene at time <em>t</em>+1 is found using a non-linear relation to the signal <em>s<sub>i</sub></em>(<em>t</em>+1), given by the Hill equation,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br>where <em>b</em>,<em>c</em>,<em>d</em> are model parameters. The typical activities of the five genes, <em>a<sub>i</sub></em>(<em>t</em>), as a function of the input signal strength, <em>S</em>, are presented in <a href="#pone-0000547-g007">Fig. 7</a> for three types of Hill equation parameters, linear (top figure), Michaelis-Menten (middle figure) and co-operative (bottom figure). As observed in modeling using ODE's<a href="#pone.0000547-Bagci1">[20]</a>, co-operative signaling leads to new behavior and a sharp onset of a transition between a low activity state and a high activity state. The behavior of <a href="#pone-0000547-g007">Fig. 7c</a> is typical of the co-operative case, and the location of the jump discontinuity and the values of the gene activities depend on the parameter values used in the simulations. One example is presented in this figure. We found that for each parameter set there is a steady state response at long times and this is the value that is plotted in the figures.</p>
<div class="figure" id="pone-0000547-g007"><div class="img"><a name="pone-0000547-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>The steady state activity of genes in the feedback loop illustrated in <a href="#pone-0000547-g006">Fig. 6</a>.</span></strong></p><a id="article1.body1.sec2.sec2.fig6.caption1.p1" name="article1.body1.sec2.sec2.fig6.caption1.p1"></a><p>The top figure is the case of linear signaling (<em>d</em> = 0, <em>c</em> = <em>b</em> = 1 in Eq. (10)), the middle figure is for the Michaelis-Menten case (<em>b</em> = <em>c</em> = <em>d</em> = 1 in Eq. (10)), and the bottom figure was found using Eq. (10) with <em>c</em> = 2, <em>d</em> = 1 and <em>b</em> = 2 corresponding to co-operative non-linearity. In the top figure the link weights are ω<sub>12</sub> = 0.367, ω<sub>23</sub> = 0.864, ω<sub>34</sub> = 0.754, ω<sub>45</sub> = 0.617, ω<sub>51</sub> = 0.718, ω<sub>51</sub> = 0.828. In the middle and bottom figures the weights used were (0.417,0.847,0.287,0.456,0.614,0.521) and (1.812,1,207, 0.971,1.158, 1.924,1.489) respectively. The link weights were chosen to be random on the interval [0,<em>c</em>].</p>
<span>doi:10.1371/journal.pone.0000547.g007</span></div><a id="article1.body1.sec2.sec2.p11" name="article1.body1.sec2.sec2.p11"></a><p>We have also studied the effect of feedback on signaling statistics in heterogeneous cell populations using the full apoptosis network, <a href="#pone-0000547-g001">Fig. 1</a>. Signaling in this network is carried out by using the OSA procedure in combination with full iteration of the loopy sub-graph of <a href="#pone-0000547-g006">Fig. 6</a>. In the linear and Michaelis-Menten cases illustrated in the top two figures of <a href="#pone-0000547-g007">Fig. 7</a>, feedback amplifies the signal, but does not qualitatively change the statistics of death signals. However in the case of co-operative non-linearity where bi-stability and strong sensitivity to the input signal occurs (see <a href="#pone-0000547-g007">Fig. 7c</a>), the statistics of the death signal can become bimodal, as illustrated in <a href="#pone-0000547-g008">Fig. 8</a>. The signaling statistics can then be controlled by controlling the feedback and non-linearity in the network.</p>
<div class="figure" id="pone-0000547-g008"><div class="img"><a name="pone-0000547-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Death statistics in a heterogeneous population of 50,000 cells with co-operative non-linearity and feedback.</span></strong></p><a id="article1.body1.sec2.sec2.fig7.caption1.p1" name="article1.body1.sec2.sec2.fig7.caption1.p1"></a><p>The non-linearity parameters used in Eq. (10) were <em>c</em> = 1.8, <em>d</em> = 1, <em>b</em> = 2 and all of the link weights were chosen to be random on the interval [0,<em>c</em>].</p>
<span>doi:10.1371/journal.pone.0000547.g008</span></div>

<h4>Discrete models</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Discrete models enable exhaustive search over the activity states of the genes and, as elucidated below, identification of the optimal combinations for selective control. In these models we discretize the weights and the activities into the same number of discrete levels, so that <em>m<sub>i</sub></em> = 0,1,…<em>M</em> and |ω<em><sub>ij</sub></em>| = 1,…<em>M</em>, for a model where the activity level of a gene has a maximum integer value of <em>M</em>. We find that the generic behaviors for large values of <em>M</em> are similar to that of the continuous models of the last subsection. On the other hand, for binary networks where each gene has activity zero or one, the behavior is quite different, with a key novelty the fact that many genes have zero incoming signal and hence a decision must be made about their activity in this case. Using the momentum rule, where the state is maintained unless changed by an incoming signal, leads to a strong dependence on initial conditions as the initial state is unchanged unless a signal is received to change it. As in Eq. (1), the signal, <em>s<sub>l</sub></em> arriving at gene <em>l</em> is a linear superposition given by,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>where <em>n</em>(<em>l</em>) is the set of genes which signal directly to gene <em>l</em>. In the discrete models, the signal, <em>s<sub>l</sub></em>, produced by the superposition rule above is then normalised to <em>s<sub>l</sub></em>/<em>n</em>(<em>l</em>) where <em>n</em>(<em>l</em>) is the number of neighbors of gene <em>l</em>. We use several different linear and non-linear relations to find the discrete activity of a gene, <em>m<sub>l</sub></em>, from the normalized signal <em>s<sub>l</sub></em>/<em>n</em>(<em>l</em>), as described below. In all cases, if the signal arriving at a gene is negative, the gene is completely inhibited and <em>m<sub>l</sub></em> = 0, which is a basic non-linearity in both continuous and discrete signaling models.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In the linear rule, the discrete activity is found from the normalized signal using,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br>where <em>c</em><sub>1</sub> is a constant which we usually take to be <em>c</em><sub>1</sub> = 1. <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e023&amp;representation=PNG" class="inline-graphic"></span> is the ceiling function, which raises a floating point number, <em>x</em>, to its next largest integer value. For the logarithmic rule the discrete activity is given by,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>where the constant <em>c</em><sub>2</sub> is chosen so that the maximum signal corresponds to <em>m<sub>l</sub></em> = <em>M</em>. In our case with <em>M</em> = 10, we take <em>c</em><sub>2</sub> = 2.17. The sigmoidal rule is given by,<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>with the parameters α = 20 and β = 10. This has a form similar to the Fermi function in physics and to dose-response curves in radiation therapy. For <em>M</em> = 10 the maximum normalized signal which can arrive at a gene is 100. The functions (12–14) are constructed so that for small signals the gene activity is one, while signals of maximum value yield gene activity 10, ensuring that all possible relations between signal and activity are represented. These behaviors are presented in <a href="#pone-0000547-g009">Fig. 9a</a>.</p>
<div class="figure" id="pone-0000547-g009"><div class="img"><a name="pone-0000547-g009" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g009&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g009"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g009&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g009/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g009/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g009/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g009/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g009.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g009/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g009/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g009.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 9.  <span>Discrete signal death statistics.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>a) Relations between the normalized signal, <em>s<sub>l</sub></em>/<em>n</em>(<em>l</em>), arriving at a gene and it's discrete activity, from the top the behaviors are for the logarithmic function, Eq. (13), the sigmoidal function, Eq. (14) and the linear function, Eq. (12). The death statistics for these three models: b) linear function; c) sigmoidal function; d) logarithmic function. It is clear that the distribution of death activities becomes bimodal and broader as we go from the linear function to the logarithmic function.</p>
<span>doi:10.1371/journal.pone.0000547.g009</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>The death statistics resulting from these discrete models are presented in <a href="#pone-0000547-g009">Figs. 9b–d</a>. The discrete linear model leads to unimodal statistics, however the sigmoidal and logarithmic functions lead to bimodal statistics. This is due to the fact that the latter functions amplify small signals, as is evident in <a href="#pone-0000547-g009">Fig. 9a</a>. In the next section we use these three discrete models in studies of selective control.</p>


<h4>Statistics of selective control</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Selective control aims at changing the life/death signal of one member of a population with a minimal change of the remaining members of the population. We address in this section the question of how network topology and general signal propagation properties can be used to design strategies for selective control. The control of the life/death signal is realized by acting with external perturbations (drugs) on the nodes and on the signaling flow. As we have seen in the previous sections, the characteristics of the signaling through the network can be strongly dependent on the OSA rule. We will see below how strategies for selectivity are affected by these rules. General strategies for selectivity can be inferred by analyzing the statistics of the nodes involved in selective perturbations. For a given network topology, we can identify the nodes that are more likely to appear in a selective perturbation, and analyze their correlations. Important insight, such as the role of balancing pro-apoptosis and anti-apoptosis perturbations in selectivity, can be obtained from this analysis. Moreover, the correlation analysis revealed that for robust populations selective nodes tend to be the ones that produce the stronger change in the output signal. The opposite is true in the case of weak populations, for which selectivity is improved by acting on nodes that produce weak signal changes in the output.</p>


<h4>Exhaustive search in discrete models</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>In this subsection we carry out an exhaustive search of selective perturbations by discretizing the control parameters and signaling variables. In the next section, we will show how some of the key features of selectivity statistics can be captured with a simplified method based on linear programming optimization, which is less demanding from a computational point of view.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We start by generating a population of different apoptosis networks with the same topology, but random values for the initial gene expressions <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e024&amp;representation=PNG" class="inline-graphic"></span> and random strength of the links ω<em><sub>ij</sub></em>Є[−<em>M</em>,−1] for inhibition, and <em>ω<sub>ij</sub></em>Є[1,<em>M</em>] for stimulation (<em>M</em> = 10 in the numerical calculations). The chosen population needs to represent living cells, <em>i.e.</em> it must have the property<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br>where <em>s̅</em><em><sub>o</sub></em> is the signal life/death threshold value, λ is the index labeling individuals in the population, and <em>s<sub>o</sub></em><sub>,λ</sub> is the output node signal. We take homeostasis into account by adding the constraint that each individual λ remains alive under fluctuations on the input nodes. After <em>s<sub>o</sub></em><sub>,λ</sub> has been calculated for a given input, we let the input nodes fluctuate and we recalculate its value. If the output of a network is less than a threshold value, that is <em>s<sub>o</sub></em><sub>,λ</sub>&lt;<em>s̅</em><em><sub>o</sub></em>, for ten random fluctuations of the input nodes, then we keep the individual λ in the living population. In the analysis of selectivity, described below, a population of 100 was chosen as the number of different cell types in the human body is of this order. Preliminary studies indicate that, as expected, selective control is easier for smaller populations, provided the number of control nodes is fixed.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Once we have created a living population, we can start to study the effect of external perturbations on the nodes. These perturbations represent the effect of drugs that stimulate or inhibit one or more gene. We will therefore represent the effect of the drug by changing the gene expression levels in the nodes by δ<em>m<sub>i</sub></em>. We will say that an individual λ̅ can be <em>selectively controlled</em> if we can find a perturbation on the gene expression levels δ<em>m<sub>i</sub></em> with the property<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(17)</span></span><br>Though there are 48 nodes, 11 are input nodes and one is the output node, so there are 36 control nodes in the network. There are <em>M</em>−1 possible perturbations on each control node so the total number of possible perturbations on all the nodes, (<em>M</em>−1)<sup>36</sup>, which is too large to be explored exhaustively. Therefore, we considered perturbations that act only on <em>k</em>-subsets of all possible perturbations. For <em>k</em> = 1 this means that we are considering only single node perturbations, which requires a search over 36(<em>M</em>−1) possibilities. For <em>k</em> = 2 we are considering pairs of nodes, with 630(<em>M</em>−1)<sup>2</sup> perturbation combinations. For an arbitrary <em>k</em>-subset the number of combinations is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e025&amp;representation=PNG" class="inline-graphic"></span>.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>In <a href="#pone-0000547-t002">Table 2</a> we present the selectivity results for ten different populations with 100 individuals each and three different OSA rules. The columns in the <a href="#pone-0000547-t002">Table 2</a> represent different <em>k</em>-subsets for the three different rules. The value for the threshold <em>s̅</em><em><sub>o</sub></em> was set to 1. In the linear case, the average percentage of individuals that can be selectively killed within the <em>k</em> = 3 subset is about 63%. We have found that this average depends strongly on the value of the threshold, and it decreases for higher threshold values. For instance, by setting the threshold at <em>s̅</em><em><sub>o</sub></em> = 7 the average selectivity (within the <em>k</em> = 3 manifold) is reduced to 8.3%. The logarithmic and sigmoid rules give a overall higher selectivity, with an average selectivity of 63.5% and 73.6%, respectively, for death threshold <em>s̅</em><em><sub>o</sub></em> = 1.</p>
<div class="figure" id="pone-0000547-t002"><div class="img"><a name="pone-0000547-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Number of individuals that can be selectively controlled in 10 populations with 100 individuals each.</span></strong></p><span>doi:10.1371/journal.pone.0000547.t002</span></div><a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>We have studied the statistics of nodes entering in selective combinations. In the linear case, single-node selectivity is obtained by acting on BAD (30), IkBa (33), NFkB (34), BAX (40), CASP3 (42), CASP7 (43) or TP53 (44). These nodes can be divided into those that are pro- and anti-apoptosis, based on their average effect on the output signal. Single drug selective apoptosis is generally induced by stimulating a pro-apoptosis node, or by inhibiting an anti-apoptosis node. For instance, nodes IkBa (33) and NFkB (34) have on average an anti-apoptosis behavior, so they are mainly associated with negative δ<em>m</em>. The dotted line in <a href="#pone-0000547-g010">Fig. 10</a> shows the distribution of nodes entering in all the selective combinations found using the linear OSA rule. Notice the peaks corresponding to the same nodes that can induce single-drug selectivity discussed above. In the figure, we also plot the distribution of nodes for the sigmoid (dashed line) and the logarithmic (solid line) rules. The sigmoid and the linear rule identify a very similar set of nodes that are more likely to enter in selective perturbations. Both BAD (30) and CASP3 (42) are strongly present in the linear and sigmoid model. The sigmoid rule slightly enhances the number of combinations in which the inhibition of anti-apoptosis genes represents the mechanism for selectivity, see e.g. CHUCK (32), IkBa (33), NFkB (34), BIRC2 (35) and BCL2 (37). Both linear and sigmoid rules suggest that selectivity might be better achieved by a direct up-regulation of caspases, or by acting on pro- or anti- apoptosis proteins of the BCL2 family. The logarithmic rule results in a different distribution, suggesting a different strategy for selectivity. Most of the selective combinations in the logarithmic case involve nodes upstream in the signaling network, indicating that the best strategy for selectivity is an action on cell-membrane FAS or TNF pathways.</p>
<div class="figure" id="pone-0000547-g010"><div class="img"><a name="pone-0000547-g010" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g010&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g010"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g010&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g010/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g010/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g010/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g010/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g010.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g010/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g010/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g010.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 10.  <span>Distribution of nodes entering in selective combinations for the Linear (dotted), Sigmoid (dashed), and Logarithmic (solid) OSA rules.</span></strong></p><span>doi:10.1371/journal.pone.0000547.g010</span></div><a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>Interesting correlations can be observed between the statistics of selective and mortal perturbations. Any perturbation that kills at least one member of the population is defined as <em>mortal</em>. Selective perturbations are mortal perturbations, but there are many more nonselective mortal combinations which kill more than one individual in the population. <a href="#pone-0000547-g011">Fig. 11</a> shows the distribution of nodes entering in selective and mortal combinations in the case of linear (top panel) and logarithmic OSA rules (lower panel). The linear rule shows a strong correlation between nodes that appear in selective combinations and nodes that appear in mortal combinations. The correlation between the mortality and and selectivity distribution is 0.88. The logarithmic case is completely different (lower panel) and exhibits a strong anti-correlation between selectivity and mortality (−0.80). The anti-correlation in the logarithmic case can be reduced by increasing the threshold, and switches to correlation for values of the threshold larger than the average value of the output signal. This behavior can be understood in the following way. For a very high life/death threshold it is very difficult to find individuals that can be killed. In that case we can say that the population is very robust with respect to external perturbations. Therefore, if one individual can be killed, the perturbation that kills that individual is very likely to be selective. We have observed that by pushing the threshold to very high values the correlation between selectivity and mortality approaches one. The nodes that are involved in this case are the ones that are able to produce the strongest change in the output, and are the same nodes usually involved in many mortal combinations. On the other hand, if the population is weak, nodes that are highly mortal are likely to kill more than one individual at the same time, therefore selectivity is associated with nodes that are less mortal, i.e. those that produce small changes in the output signal. The robustness or weakness of a population is determined not only by the value of the life/death threshold, but also by the OSA rules giving different signaling statistics as discussed in the previous section. In fact, the behavior in <a href="#pone-0000547-g011">Fig. 11</a> was obtained using the same threshold (<em>s̅</em><em><sub>o</sub></em> = 1). There, the correlation/anticorrelation is a consequence of the higher sensitivity of the logarithmic rule to external perturbations, which implies that the population is much weaker compared to the linear case.</p>
<div class="figure" id="pone-0000547-g011"><div class="img"><a name="pone-0000547-g011" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g011&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g011"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g011&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g011/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g011/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g011/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g011/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g011.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g011/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g011/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g011.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 11.  <span>Distribution of nodes appearing in selective and mortal perturbations in the linear (top panel) and logarithmic (lower panel) OSA rules.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>The life/death threshold is set to <em>s̅</em><em><sub>o</sub></em> = 1 for the two rules. The correlation between the mortality and selectivity distributions is 0.88 for the linear case and −.80 for logarithmic case.</p>
<span>doi:10.1371/journal.pone.0000547.g011</span></div><a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7"></a><p>We have analyzed correlations between two nodes appearing in selective combinations. For the linear rule, this is shown in <a href="#pone-0000547-g012">Fig. 12</a> using a correlation matrix plot. Darker dots indicate a higher number of selective combinations containing the two nodes given by the row and column of the matrix. Notice the strong presence of the mitochondrial BAD (30) and BAX (40), and the caspases CASP3 (42) and CASP7(43). However, these nodes often appear in combination with other nodes, many of which have an anti-apoptosis character. The balance of pro and anti-apoptosis perturbations is the key element which increases the selectivity from 1.5% in the <em>k</em> = 1 case to the 63% in the three-node perturbation. Notice also that in this linear case the nodes involved in selective combinations are often downstream (i.e. close to the output node) in the signaling network. We show in <a href="#pone-0000547-g013">Fig. 13</a> and <a href="#pone-0000547-g014">Fig. 14</a> the same correlation matrices in the case of the sigmoid and logarithmic OSA rules. In these models more nodes are involved in the selective combinations. The correlation pattern for the sigmoid rule shows strong similarities to the linear case. However, the logarithmic rule results in a qualitatively different pattern in the correlation matrix. Notice for instance that the role of BAD (30), which was dominant in the linear and sigmoid rule, is strongly reduced in this case. In contrast to the linear and sigmoid case, nodes in long pathways dominate in the selective combinations. Overall, the presence of nonlinearity in the OSA rules seems to enhance the possibility of selective control. This is also confirmed by the trend in the total number of <em>k</em> = 3 selective combinations, being 9×10<sup>6</sup>, 11×10<sup>6</sup>, and 26×10<sup>6</sup> for the linear, sigmoid and logarithmic OSA rules, respectively.</p>
<div class="figure" id="pone-0000547-g012"><div class="img"><a name="pone-0000547-g012" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g012&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g012"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g012&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g012/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g012/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g012/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g012/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g012.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g012/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g012/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g012.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 12.  <span>Correlation matrix for nodes appearing in the same selective perturbation in the linear OSA rule.</span></strong></p><a id="article1.body1.sec2.sec5.fig3.caption1.p1" name="article1.body1.sec2.sec5.fig3.caption1.p1"></a><p>Darker dots indicate that the two nodes given by the row and column of the matrix appear more often in the selective drug combinations.</p>
<span>doi:10.1371/journal.pone.0000547.g012</span></div><div class="figure" id="pone-0000547-g013"><div class="img"><a name="pone-0000547-g013" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g013&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g013"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g013&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g013/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g013/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g013/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g013/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g013.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g013/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g013/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g013.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 13.  <span>Correlation matrix for nodes appearing in the same selective perturbation in the sigmoidal OSA rule.</span></strong></p><a id="article1.body1.sec2.sec5.fig4.caption1.p1" name="article1.body1.sec2.sec5.fig4.caption1.p1"></a><p>Darker dots indicate that the two nodes given by the row and column of the matrix appear more often in the selective drug combinations.</p>
<span>doi:10.1371/journal.pone.0000547.g013</span></div><div class="figure" id="pone-0000547-g014"><div class="img"><a name="pone-0000547-g014" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g014&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g014"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g014&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g014/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g014/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g014/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g014/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g014.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g014/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g014/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g014.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 14.  <span>Correlation matrix for nodes appearing in the same selective perturbation in the logarithmic OSA rule.</span></strong></p><a id="article1.body1.sec2.sec5.fig5.caption1.p1" name="article1.body1.sec2.sec5.fig5.caption1.p1"></a><p>Darker dots indicate that the two nodes given by the row and column of the matrix appear more often in the selective drug combinations.</p>
<span>doi:10.1371/journal.pone.0000547.g014</span></div>

<h4>Linear programming methods</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The exhaustive search method, discussed in the previous subsection, for selective combinations becomes very demanding when combinations with a large number of drugs are involved. A different approach consists in defining an effective model for the dependence of the output signal on the perturbations. The effective model is derived from the original OSA approach using a sensitivity analysis, and some of the selective perturbations found with the effective model are also selective perturbations for the original OSA problem. The advantage of the effective model is that selective combinations can be efficiently obtained by linear programming methods<a href="#pone.0000547-Cormen1">[30]</a>. We will analyze below the statistics of the selective combinations for the linear OSA method obtained through the effective model. The form of the solutions in many cases involves a number of nodes larger than three, in contrast to the exhaustive method discussed in the previous section. The effective method therefore provides a different sampling of the full space of selective combinations. However, we will see that this different sampling leads to very a similar statistics for the selective nodes.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>The output signal from a OSA can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(18)</span></span><br>and depends in a nonlinear way on the perturbations of the internal nodes. A typical single node dependence is shown in <a href="#pone-0000547-g015">Fig. 15</a> for a pro-apoptosis BAD (30) and an anti-apoptosis IkBa (33) node for two different individuals (solid and dashed lines) as a function of the strength of the single node perturbation. Notice that the single drug dependence follows a step-like dependence as a consequence of the discreteness of the gene expression levels. Moreover, due to the constraint that the activity must remain positive, there are regions in which the dependence on the external perturbation saturates. The effective linear model can be defined as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(19)</span></span><br>where the coefficient <em>c</em><sub>λ,<em>i</em></sub> can be estimated by approximating the single drug dependence such as the ones in <a href="#pone-0000547-g015">Fig. 15</a> with a linear dependence using interpolation methods. The linear interpolation works well only for some nodes and individuals, since in many cases the dependence is non-monotonic and highly nonlinear. Moreover, notice that we are neglecting higher order many-node effects that are included in the OSA approach. However, we will see below that this does not affect considerably the statistics of the genes that are more likely to appear in selective combinations. The interpolation method also allows us to identify nodes that lead to the highest variations of the output, and the selective combinations can be restricted to nodes within that set. The smallest coefficients <em>c</em><sub>λ,<em>i</em></sub> can therefore be neglected. The reduction of the control parameter phase space by sensitivity analysis is often a key element in global optimization problems. This was shown explicitly in the case of parameter identification in biochemical reaction networks<a href="#pone.0000547-Feng1">[31]</a>, <a href="#pone.0000547-Hooshangi1">[32]</a>. Once the coefficient <em>c</em><sub>λ,<em>i</em></sub> and the threshold value <em>s̅</em><em><sub>o</sub></em> have been fixed, the selectivity problem can be recast in the form of a linear programming optimization problem where we minimize the cost function<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(20)</span></span><br>on the polytope defined by the constraint equations<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e021&amp;representation=PNG" class="inline-graphic"><span class="note">(21)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e022&amp;representation=PNG" class="inline-graphic"><span class="note">(22)</span></span><br>The solution provided by the linear programming method is optimal in the sense that it gives the global minimum of the function in Eq. (20) <a href="#pone.0000547-Cormen1">[30]</a>.</p>
<div class="figure" id="pone-0000547-g015"><div class="img"><a name="pone-0000547-g015" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g015&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g015"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g015&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g015/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g015/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g015/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g015/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g015.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g015/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g015/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g015.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 15.  <span>Output signal as a function of the perturbation strength on a pro-apoptosis (BAD, red increasing line) and anti-apoptosis node (IkBa, green decreasing line).</span></strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>The dashed line and the solid line refers to two different individuals.</p>
<span>doi:10.1371/journal.pone.0000547.g015</span></div><a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>We show in <a href="#pone-0000547-g016">Fig. 16</a> the statistics of the nodes found in the linear programming optimization (solid line) compared to the same statistics obtained with the exhaustive search described in the previous section (dotted line for linear, dashed line for sigmoid). The two approaches identify basically the same nodes as the ones that are more likely to be present in selective combinations. Notice the strong presence of BAD (30), BAX (40) and Caspases (42–44) in the three approaches. Also, the relatively strong peaks at IL-3R (17), PI3K (20), and AKT3 (26) are captured by the effective approach. These peaks suggest the possibility of selective control by acting on the AKT signaling pathway.</p>
<div class="figure" id="pone-0000547-g016"><div class="img"><a name="pone-0000547-g016" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g016&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000547" data-uri="info:doi/10.1371/journal.pone.0000547.g016"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.g016&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g016/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g016/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g016/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g016/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g016.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000547.g016/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000547.g016/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000547.g016.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 16.  <span>Distribution of nodes entering in selective combinations obtained with the Linear OSA (dotted), sigmoid (dashed) and with the effective linear programming method (solid line).</span></strong></p><span>doi:10.1371/journal.pone.0000547.g016</span></div><a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4"></a><p>We also have used the optimal control parameter <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000547.e026&amp;representation=PNG" class="inline-graphic"></span> obtained with the linear programming on the original nonlinear OSA and checked the system for selectivity. Typically, we found that the linear programming solution is only partially selective for the original OSA rule. The drug combinations found from the linear programming method can selectively kill only 20% of the 100 individuals in the survivor population, which is considerably smaller than that found using exhaustive search. However, many selective combinations found by the linear programming approach are <em>quasi-selective</em>, in the sense that they kill two/three individuals in the population rather than one which is requirement for selectivity. This quasi-selectivity captured by the linear programming method is at the origin of the strong similarity in the distributions of <a href="#pone-0000547-g016">Fig. 16</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Analysis of the statistical behavior of genes in the apoptosis network illustrates the importance of local connectivity on gene activity variations. Genes which receive signals from many parallel paths exhibit normal statistics while genes which lie at the end of a single pathway are prone to large statistical variations and highly skew statistics. Though non-linearity alone does not alter these broad conclusions, the combination of non-linearity and feedback may lead to bimodal statistics. This bimodality occurs due to combination of feedback and bistability<a href="#pone.0000547-Bagci1">[20]</a>, which in our signaling models is reflected in a sharp rise in activity with signal strength.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The concept of selective control in heterogeneous cell populations was developed, using the apoptosis network as an illustrative example. Selective control within a heterogeneous cell population is the ability to control one member of the population while leaving the other members relatively unaffected. General selective control strategies that are only dependent on the topology of the network and signaling rules can be inferred from analysis of networks with random link weights. For instance, linear and sigmoid rules identify the same set of nodes that are most efficient in selective control. These nodes can lead to a high degree of selectivity within a given population, especially by balancing pro- and anti- apoptosis perturbations. We have explored two methods for the study of selectivity. The first is an exhaustive search method limited to three node perturbations. The second is an effective linear model, based on interpolation of single node sensitivity, in which the selective combinations can be found by linear programming optimization. The two approaches identify the same strategies for selectivity. We have also identified a general rule that relates the life/death switching robustness of a population to the optimal selectivity strategy. Selectivity is promoted by acting on the least sensitive nodes in the case of weak populations, while selective control of robust populations is optimized through perturbations of more sensitive nodes. More generally selective control is a computational challenge in a broad range of systems biology problems where intervention needs to be directed at subsets of a diverse population.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>At a more practical level, high throughput experiments with heterogeneous cell lines could be designed in such a way that the selectivity optimization process is part of a closed-loop control system<a href="#pone.0000547-Ku1">[33]</a>. Single drug measurements could be used to obtain directly the sensitivity in a given heterogeneous cell population. The linear programming optimization method can then use the sensitivity measurements to identify selective combinations in a model free way. The sensitivity/selectivity optimization can then be improved through iterated experiments in which various computational algorithms and experimental protocols are integrated.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1"></a><p>Two programs were used to generate the signaling statistics and to analyse selectivity. One procedure was written in Mathematica and the other in C++. These independent programs were used to check the accuracy of the numerical analysis of the signaling procedures and the signaling statistics.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2"></a><p>The software used to run the exhaustive search tests is written in C++ and compiled for both the Linux 32 bit and 64 bit operating systems. We used two clusters, available at the Burnham Institute for Medical Research, Falcon and Bsrc. Both of them are Portable Batch System (PBS) clusters, so that they have a native implementation of queues management enabling submission of the same program multiple times to achieve virtual parallelism. Falcon has 128 CPUs organized in 64 nodes and each of them is a x86 32 bit 2.4 GHz with 1 GB of private memory; Bsrc has 128 CPU organized in 32 nodes and each of them is a x86 64 bit 2.0 GHz with 2 GB of private memory. Exhaustive search requires an exponential computational time in the number of nodes and projecting from the time needed for combinations of 1,2,3 nodes, that is respectively 1 minute, 1.5 hours and 7.5 hours, we estimate that around 90 days would be needed to study all the combinations of 4 drugs. The calculations on the continuous models and the linear programming optimization were implemented using Mathematica on a personal computer. The NMinimize function in Mathematica finds the global minimum when the objective function and constraints are linear.</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>PMD thanks Leon Glass and Bard Ermentrout for useful discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GP PD PH DC CP. Performed the experiments: PD PH DC CP. Analyzed the data: GP PD PH DC CP. Contributed reagents/materials/analysis tools: GP PD PH DC CP. Wrote the paper: PD CP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000547-Fitzgerald1" id="pone.0000547-Fitzgerald1"></a>Fitzgerald J, Schoeber B, Nielsen UB, Sorger PK (2006) Systems biology and combination therapy in the quest for clinical efficacy. Nature Chemical Biology  2(9): 458–466.  <ul class="find" data-citedArticleID="960380" data-doi="10.1038/nchembio817"><li><a href="http://dx.doi.org/10.1038/nchembio817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systems+biology+and+combination+therapy+in+the+quest+for+clinical+efficacy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systems+biology+and+combination+therapy+in+the+quest+for+clinical+efficacy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000547-Johnstone1" id="pone.0000547-Johnstone1"></a>Johnstone RW, Ruefli AA, Lowe SW (2002) Apoptosis: A link between cancer genetics and chemotherapy. Cell  108: 153–164.  <ul class="find" data-citedArticleID="960386" data-doi="10.1016/s0092-8674(02)00625-6"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00625-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apoptosis%3A+A+link+between+cancer+genetics+and+chemotherapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apoptosis%3A+A+link+between+cancer+genetics+and+chemotherapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000547-Klein1" id="pone.0000547-Klein1"></a>Klein S, Levitzki A (2006) Signal transduction therapy for cancer - whither now. Current Signal Transduction Therapy  1: 1–12.  <ul class="find" data-citedArticleID="960392" data-doi="10.2174/157436206775269244"><li><a href="http://dx.doi.org/10.2174/157436206775269244" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal+transduction+therapy+for+cancer+-+whither+now." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal+transduction+therapy+for+cancer+-+whither+now.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000547-Petak1" id="pone.0000547-Petak1"></a>Petak I, Houghton JA, Kopper L (2006) Molecular targeting of cell death signal transduction pathways in cancer. Current Signal Transduction Therapy  1: 113–131.  <ul class="find" data-citedArticleID="960406" data-doi="10.2174/157436206775269217"><li><a href="http://dx.doi.org/10.2174/157436206775269217" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+targeting+of+cell+death+signal+transduction+pathways+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+targeting+of+cell+death+signal+transduction+pathways+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000547-Ankarcrona1" id="pone.0000547-Ankarcrona1"></a>Ankarcrona M, Winblad B (2005) Biomarkers for apoptosis in Alzheimer's disease. International Journal of Geriatric Psychiatry  20: 101–105.  <ul class="find" data-citedArticleID="960356" data-doi="10.1002/gps.1260"><li><a href="http://dx.doi.org/10.1002/gps.1260" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biomarkers+for+apoptosis+in+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biomarkers+for+apoptosis+in+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000547-Lu1" id="pone.0000547-Lu1"></a>Lu KP (2004) Pinning down cell signaling, cancer and Alzheimer's disease. TRENDS in Biochemical Sciences  29: 200–209.  <ul class="find" data-citedArticleID="960402" data-doi="10.1016/j.tibs.2004.02.002"><li><a href="http://dx.doi.org/10.1016/j.tibs.2004.02.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pinning+down+cell+signaling%2C+cancer+and+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pinning+down+cell+signaling%2C+cancer+and+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000547-Pompl1" id="pone.0000547-Pompl1"></a>Pompl PN, Yemul S, Xiang ZM, Ho L, Haroutunian V, et al.  (2003) Caspase gene expression in the brain as a function of the clinical progression of Alzheimer's disease. Archives of Neurology  60: 369–376.  <ul class="find" data-citedArticleID="960408" data-doi="10.1001/archneur.60.3.369"><li><a href="http://dx.doi.org/10.1001/archneur.60.3.369" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caspase+gene+expression+in+the+brain+as+a+function+of+the+clinical+progression+of+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caspase+gene+expression+in+the+brain+as+a+function+of+the+clinical+progression+of+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000547-Barrett1" id="pone.0000547-Barrett1"></a>Barrett CL, Palsson BO (2006) Iterative reconstruction of transcriptional regulatory networks: An algorithmic approach. PLOS computational biology  2: 429–438.  <ul class="find" data-citedArticleID="960360" data-doi="10.1371/journal.pcbi.0020052.eor"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020052.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Iterative+reconstruction+of+transcriptional+regulatory+networks%3A+An+algorithmic+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Iterative+reconstruction+of+transcriptional+regulatory+networks%3A+An+algorithmic+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000547-Perkins1" id="pone.0000547-Perkins1"></a>Perkins TJ, Jaeger J, Glass L (2006) Reverse engineering the gap gene network of drosophila melanogaster. PLOS computational biology  2: 417–428.  <ul class="find" data-citedArticleID="960404" data-doi="10.1371/journal.pcbi.0020051"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0020051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reverse+engineering+the+gap+gene+network+of+drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reverse+engineering+the+gap+gene+network+of+drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000547-Dealy1" id="pone.0000547-Dealy1"></a>Dealy S, Kauffman S, Socolar J (2005) A computational framework for the topological analysis and targeted disruption of signal transduction networks. Complexity  11: 52–60.  <ul class="find" data-citedArticleID="960370" data-doi="10.1002/cplx.20100"><li><a href="http://dx.doi.org/10.1002/cplx.20100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computational+framework+for+the+topological+analysis+and+targeted+disruption+of+signal+transduction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computational+framework+for+the+topological+analysis+and+targeted+disruption+of+signal+transduction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000547-Li1" id="pone.0000547-Li1"></a>Li F, Long T, Lu Y, Ouyang Q, Tang C (2004) The yeast cell-cycle network is robustly designed. Proceedings of the National Academy of Science U S A  101: 4781–4786.  <ul class="find" data-citedArticleID="960400" data-doi="10.1073/pnas.0305937101"><li><a href="http://dx.doi.org/10.1073/pnas.0305937101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+yeast+cell-cycle+network+is+robustly+designed." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+yeast+cell-cycle+network+is+robustly+designed.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000547-Shmulevich1" id="pone.0000547-Shmulevich1"></a>Shmulevich I, Kauffman SA, Aldana M (2005) Eukaryotic cells are dynamically ordered or critical but not chaotic. Proceedings of the National Academy of Science U S A  102: 13439–13444.  <ul class="find" data-citedArticleID="960412" data-doi="10.1073/pnas.0506771102"><li><a href="http://dx.doi.org/10.1073/pnas.0506771102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Eukaryotic+cells+are+dynamically+ordered+or+critical+but+not+chaotic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Eukaryotic+cells+are+dynamically+ordered+or+critical+but+not+chaotic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000547-Aldridge1" id="pone.0000547-Aldridge1"></a>Aldridge BB, Burke JM, Lauffenburger DA, Sorger PK (2006) Physiochemical modeling of cell signaling pathways. Nature Cell Biology  8: 1195–1203.  <ul class="find" data-citedArticleID="960354" data-doi="10.1038/ncb1497"><li><a href="http://dx.doi.org/10.1038/ncb1497" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiochemical+modeling+of+cell+signaling+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiochemical+modeling+of+cell+signaling+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000547-Kholodenko1" id="pone.0000547-Kholodenko1"></a>Kholodenko BN (2006) Cell-signalling dynamics in time and space. Nature Reviews Molecular Cell Biology  7: 165–176.  <ul class="find" data-citedArticleID="960390" data-doi="10.1038/nrm1838"><li><a href="http://dx.doi.org/10.1038/nrm1838" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell-signalling+dynamics+in+time+and+space." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell-signalling+dynamics+in+time+and+space.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000547-Colizza1" id="pone.0000547-Colizza1"></a>Colizza V, Flammini A, Maritan A, Vespignani A (2005) Characterization and modeling of protein-protein interaction networks. Physica A  352: 1–27.  <ul class="find" data-citedArticleID="960364" data-doi="10.1016/j.physa.2004.12.030"><li><a href="http://dx.doi.org/10.1016/j.physa.2004.12.030" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Characterization+and+modeling+of+protein-protein+interaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Characterization+and+modeling+of+protein-protein+interaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000547-Drossel1" id="pone.0000547-Drossel1"></a>Drossel B (2005) Number of attractors in random boolean networks. Physical Review E  72: 16110.  <ul class="find" data-citedArticleID="960372" data-doi="10.1103/physreve.72.016110"><li><a href="http://dx.doi.org/10.1103/physreve.72.016110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Number+of+attractors+in+random+boolean+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Number+of+attractors+in+random+boolean+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000547-Huang1" id="pone.0000547-Huang1"></a>Huang S, Eichler G, Bar-Yam Y, Ingber DE (2005) Cell fates as high-dimensional attractor states of a complex gene regulatory network. Physical Review Letters  94: 128701.  <ul class="find" data-citedArticleID="960384" data-doi="10.1103/physrevlett.94.128701"><li><a href="http://dx.doi.org/10.1103/physrevlett.94.128701" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+fates+as+high-dimensional+attractor+states+of+a+complex+gene+regulatory+network." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+fates+as+high-dimensional+attractor+states+of+a+complex+gene+regulatory+network.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000547-Klemm1" id="pone.0000547-Klemm1"></a>Klemm K, Bornholdt S (2005) A computational framework for the topological analysis and targeted disruption of signal transduction networks. Physical Review E  72: 55101.  <ul class="find" data-citedArticleID="960394"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+computational+framework+for+the+topological+analysis+and+targeted+disruption+of+signal+transduction+networks.&amp;auth=&amp;atitle=A+computational+framework+for+the+topological+analysis+and+targeted+disruption+of+signal+transduction+networks." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computational+framework+for+the+topological+analysis+and+targeted+disruption+of+signal+transduction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computational+framework+for+the+topological+analysis+and+targeted+disruption+of+signal+transduction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000547-Zhou1" id="pone.0000547-Zhou1"></a>Zhou H, Lipowsky R (2005) Dynamic pattern evolution on scale-free networks. Proceedings of the National Academy of Science USA  102: 10052–10057.  <ul class="find" data-citedArticleID="960416" data-doi="10.1073/pnas.0409296102"><li><a href="http://dx.doi.org/10.1073/pnas.0409296102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+pattern+evolution+on+scale-free+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+pattern+evolution+on+scale-free+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000547-Bagci1" id="pone.0000547-Bagci1"></a>Bagci EZ, Vodovotz Y, Billiar TR, Ermentrout GB, Bahar I (2006) Bistability in apoptosis: Roles of bax, bcl-2 and mitochondrial permeability transition pores. Biophysical Journal  91: 1546–1549.  <ul class="find" data-citedArticleID="960358" data-doi="10.1529/biophysj.105.068122"><li><a href="http://dx.doi.org/10.1529/biophysj.105.068122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bistability+in+apoptosis%3A+Roles+of+bax%2C+bcl-2+and+mitochondrial+permeability+transition+pores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bistability+in+apoptosis%3A+Roles+of+bax%2C+bcl-2+and+mitochondrial+permeability+transition+pores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000547-Buchler1" id="pone.0000547-Buchler1"></a>Buchler NE, Gerland U, Hwa T (2005) Nonlinear protein degradation and the function of genetic circuits. Proceedings of the National Academy of Science USA  102: 9559–9564.  <ul class="find" data-citedArticleID="960362" data-doi="10.1073/pnas.0409553102"><li><a href="http://dx.doi.org/10.1073/pnas.0409553102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nonlinear+protein+degradation+and+the+function+of+genetic+circuits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nonlinear+protein+degradation+and+the+function+of+genetic+circuits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000547-Eiing1" id="pone.0000547-Eiing1"></a>Eißing T, Allgöwer F, Bullinger E (2005) Robustness properties of apoptosis models with respect to parameter variations and intrinsic noise. IEEE Proceedings on Systems Biology  152: 221–228.  <ul class="find" data-citedArticleID="960376" data-doi="10.1049/ip-syb:20050046"><li><a href="http://dx.doi.org/10.1049/ip-syb:20050046" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Robustness+properties+of+apoptosis+models+with+respect+to+parameter+variations+and+intrinsic+noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Robustness+properties+of+apoptosis+models+with+respect+to+parameter+variations+and+intrinsic+noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000547-Covert1" id="pone.0000547-Covert1"></a>Covert MW, Schilling CH, Palsson B (2001) Regulation of gene expression in flux balance models of metabolism. Journal of Theoretical Biology  213: 73–78.  <ul class="find" data-citedArticleID="960368" data-doi="10.1006/jtbi.2001.2405"><li><a href="http://dx.doi.org/10.1006/jtbi.2001.2405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+gene+expression+in+flux+balance+models+of+metabolism." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+gene+expression+in+flux+balance+models+of+metabolism.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000547-Lee1" id="pone.0000547-Lee1"></a>Lee JM, Gianchandani EP, Papin JA (2006) Flux balance analysis in the era of metabolomics. Briefings in Bioinformatics  7: 140–150.  <ul class="find" data-citedArticleID="960398" data-doi="10.1093/bib/bbl007"><li><a href="http://dx.doi.org/10.1093/bib/bbl007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flux+balance+analysis+in+the+era+of+metabolomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flux+balance+analysis+in+the+era+of+metabolomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000547-Reed1" id="pone.0000547-Reed1"></a>Reed JC, Pellecchia M (2005) Apoptosis-based therapies for hematologic malignancies. Blood  106: 408–418.  <ul class="find" data-citedArticleID="960410" data-doi="10.1182/blood-2004-07-2761"><li><a href="http://dx.doi.org/10.1182/blood-2004-07-2761" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apoptosis-based+therapies+for+hematologic+malignancies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apoptosis-based+therapies+for+hematologic+malignancies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000547-Albert1" id="pone.0000547-Albert1"></a>Albert R, Othmer HG (2003) The topology of the regulatory interactions predicts the expression pattern of the segment polarity genes in drosophila melanogaster. Journal of Theoretical Biology  223: 1–18.  <ul class="find" data-citedArticleID="960352" data-doi="10.1016/s0022-5193(03)00035-3"><li><a href="http://dx.doi.org/10.1016/s0022-5193(03)00035-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+topology+of+the+regulatory+interactions+predicts+the+expression+pattern+of+the+segment+polarity+genes+in+drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+topology+of+the+regulatory+interactions+predicts+the+expression+pattern+of+the+segment+polarity+genes+in+drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000547-Duarte1" id="pone.0000547-Duarte1"></a>Duarte NC, Becker SA, Jamshidi N, Thiele NI, Mo ML, et al.  (2007) Global reconstruction of the human metabolic network based on genomic and bibliomic data. Proceedings of the National Academy of Science U S A  104: 1777–1782.  <ul class="find" data-citedArticleID="960374" data-doi="10.1073/pnas.0610772104"><li><a href="http://dx.doi.org/10.1073/pnas.0610772104" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+reconstruction+of+the+human+metabolic+network+based+on+genomic+and+bibliomic+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+reconstruction+of+the+human+metabolic+network+based+on+genomic+and+bibliomic+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000547-Kauffman1" id="pone.0000547-Kauffman1"></a>Kauffman SA (1993) The origins of order: Self-organization and selection in evolution. Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0000547-Weiss1" id="pone.0000547-Weiss1"></a>Weiss JN (1997) The hill equation revisited: uses and misuses. FASEB Journal  11: 835–841.  <ul class="find" data-citedArticleID="960414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+hill+equation+revisited%3A+uses+and+misuses.&amp;auth=&amp;atitle=The+hill+equation+revisited%3A+uses+and+misuses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+hill+equation+revisited%3A+uses+and+misuses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+hill+equation+revisited%3A+uses+and+misuses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000547-Cormen1" id="pone.0000547-Cormen1"></a>Cormen TH, Leiserson CE, Rivest RL, Stein C (2001) Introduction to Algorithms. MIT Press and McGraw-Hill.   <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0000547-Feng1" id="pone.0000547-Feng1"></a>Feng XJ, Rabitz H (2004) Optimal identification of biochemical reaction networks. Biophysical Journal  86: 1270–1281.  <ul class="find" data-citedArticleID="960378" data-doi="10.1016/s0006-3495(04)74201-0"><li><a href="http://dx.doi.org/10.1016/s0006-3495(04)74201-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimal+identification+of+biochemical+reaction+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimal+identification+of+biochemical+reaction+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000547-Hooshangi1" id="pone.0000547-Hooshangi1"></a>Hooshangi S, Feng XJ, Chen D, Li G, Weiss R, Rabitz H (2004) Optimizing genetic circuits by global sensitivity analysis. Biophysical Journal  87: 2195–2202.  <ul class="find" data-citedArticleID="960382" data-doi="10.1529/biophysj.104.044131"><li><a href="http://dx.doi.org/10.1529/biophysj.104.044131" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimizing+genetic+circuits+by+global+sensitivity+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimizing+genetic+circuits+by+global+sensitivity+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000547-Ku1" id="pone.0000547-Ku1"></a>Ku J, Feng X, Rabitz H (2004) Closed-loop learning control of bio-networks. Journal of Computational Biology  11: 642–659.  <ul class="find" data-citedArticleID="960396" data-doi="10.1089/cmb.2004.11.642"><li><a href="http://dx.doi.org/10.1089/cmb.2004.11.642" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Closed-loop+learning+control+of+bio-networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Closed-loop+learning+control+of+bio-networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.XML" value="107277"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.PDF" value="690767"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g001.PNG_L" value="341368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g001.PNG_M" value="256148"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g001.PNG_S" value="6923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g001.TIF" value="440890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g001.PNG_I" value="95860"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t001.PNG_L" value="40922"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t001.PNG_M" value="36775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t001.PNG_S" value="2538"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t001.TIF" value="333680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t001.PNG_I" value="17285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e001.PNG" value="2295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e001.TIF" value="23620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e002.PNG" value="2147"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e002.TIF" value="23488"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g002.PNG_L" value="92246"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g002.PNG_M" value="28051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g002.PNG_S" value="2923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g002.TIF" value="412896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g002.PNG_I" value="25917"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e003.PNG" value="1842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e003.TIF" value="23128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e004.PNG" value="3999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e004.TIF" value="25592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g003.PNG_L" value="18952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g003.PNG_M" value="14438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g003.PNG_S" value="1674"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g003.TIF" value="35366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g003.PNG_I" value="6320"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g004.PNG_L" value="9372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g004.PNG_M" value="7399"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g004.PNG_S" value="1463"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g004.TIF" value="19614"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g004.PNG_I" value="3611"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g005.PNG_L" value="38984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g005.PNG_M" value="36427"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g005.PNG_S" value="3496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g005.TIF" value="269240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g005.PNG_I" value="28946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g006.PNG_L" value="4419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g006.PNG_M" value="4419"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g006.PNG_S" value="2676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g006.TIF" value="21630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g006.PNG_I" value="14934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e005.PNG" value="2424"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e005.TIF" value="24092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e006.PNG" value="2042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e006.TIF" value="23504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e007.PNG" value="2126"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e007.TIF" value="23572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e008.PNG" value="2129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e008.TIF" value="23560"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e009.PNG" value="3107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e009.TIF" value="25040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e010.PNG" value="4660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e010.TIF" value="27160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g007.PNG_L" value="46829"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g007.PNG_M" value="16790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g007.PNG_S" value="3107"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g007.TIF" value="84274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g007.PNG_I" value="24441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g008.PNG_L" value="16379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g008.PNG_M" value="23820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g008.PNG_S" value="2018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g008.TIF" value="47206"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g008.PNG_I" value="10450"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e011.PNG" value="2400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e011.TIF" value="23688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e012.PNG" value="2211"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e012.TIF" value="23816"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e023.PNG" value="353"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e023.TIF" value="20948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e013.PNG" value="2355"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e013.TIF" value="24104"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e014.PNG" value="4592"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e014.TIF" value="26016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g009.PNG_L" value="63793"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g009.PNG_M" value="35961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g009.PNG_S" value="2185"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g009.TIF" value="119394"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g009.PNG_I" value="15492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e024.PNG" value="2139"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e024.TIF" value="23620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e015.PNG" value="1565"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e015.TIF" value="22556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e016.PNG" value="1170"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e016.TIF" value="21880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e017.PNG" value="1952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e017.TIF" value="23120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e025.PNG" value="1735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e025.TIF" value="22888"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t002.PNG_L" value="64372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t002.PNG_M" value="64604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t002.PNG_S" value="3384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t002.TIF" value="376234"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.t002.PNG_I" value="30638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g010.PNG_L" value="10454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g010.PNG_M" value="78729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g010.PNG_S" value="3062"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g010.TIF" value="54092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g010.PNG_I" value="33384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g011.PNG_L" value="155757"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g011.PNG_M" value="64090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g011.PNG_S" value="4049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g011.TIF" value="313176"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g011.PNG_I" value="50967"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g012.PNG_L" value="29599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g012.PNG_M" value="25546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g012.PNG_S" value="3270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g012.TIF" value="98406"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g012.PNG_I" value="12490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g013.PNG_L" value="8583"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g013.PNG_M" value="29839"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g013.PNG_S" value="3810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g013.TIF" value="64284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g013.PNG_I" value="15257"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g014.PNG_L" value="8721"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g014.PNG_M" value="34521"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g014.PNG_S" value="4096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g014.TIF" value="69752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g014.PNG_I" value="17909"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e018.PNG" value="2821"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e018.TIF" value="24636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e019.PNG" value="5825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e019.TIF" value="26736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e020.PNG" value="3894"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e020.TIF" value="25692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e021.PNG" value="2599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e021.TIF" value="24052"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e022.PNG" value="2625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e022.TIF" value="24116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g015.PNG_L" value="6111"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g015.PNG_M" value="44558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g015.PNG_S" value="2275"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g015.TIF" value="44208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g015.PNG_I" value="19023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g016.PNG_L" value="10798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g016.PNG_M" value="86877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g016.PNG_S" value="3309"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g016.TIF" value="57988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.g016.PNG_I" value="35810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e026.PNG" value="796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000547.e026.TIF" value="21440"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000547&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000547" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000547&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000547&volume=&issue=&title=Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations&author_name=Diego%20Calzolari%2C%20Giovanni%20Paternostro%2C%20Patrick%20L.%20Harrington%20Jr.%2C%20Carlo%20Piermarocchi%2C%20Phillip%20M.%20Duxbury&start_page=1&end_page=12" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000547" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547&amp;t=Selective%20Control%20of%20the%20Apoptosis%20Signaling%20Network%20in%20Heterogeneous%20Cell%20Populations" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547&title=Selective%20Control%20of%20the%20Apoptosis%20Signaling%20Network%20in%20Heterogeneous%20Cell%20Populations&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547&amp;title=Selective%20Control%20of%20the%20Apoptosis%20Signaling%20Network%20in%20Heterogeneous%20Cell%20Populations" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000547&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Selective Control of the Apoptosis Signaling Network in Heterogeneous Cell Populations';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000547';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Selective%20Control%20of%20the%20Apoptosis%20Signaling%20Network%20in%20Heterogeneous%20Cell%20Populations http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000547" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000547" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Apoptosis%22" title="Search for articles in the subject area:'Apoptosis'"><div class="flagText">Apoptosis</div></a>
              <div data-categoryid="17189" data-articleid="23932"
                   data-categoryname="Apoptosis"
                   class="flagImage" title="Flag 'Apoptosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+networks%22" title="Search for articles in the subject area:'Genetic networks'"><div class="flagText">Genetic networks</div></a>
              <div data-categoryid="40201" data-articleid="23932"
                   data-categoryname="Genetic networks"
                   class="flagImage" title="Flag 'Genetic networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Linear+programming%22" title="Search for articles in the subject area:'Linear programming'"><div class="flagText">Linear programming</div></a>
              <div data-categoryid="24261" data-articleid="23932"
                   data-categoryname="Linear programming"
                   class="flagImage" title="Flag 'Linear programming' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Normal+distribution%22" title="Search for articles in the subject area:'Normal distribution'"><div class="flagText">Normal distribution</div></a>
              <div data-categoryid="32203" data-articleid="23932"
                   data-categoryname="Normal distribution"
                   class="flagImage" title="Flag 'Normal distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Optimization%22" title="Search for articles in the subject area:'Optimization'"><div class="flagText">Optimization</div></a>
              <div data-categoryid="31839" data-articleid="23932"
                   data-categoryname="Optimization"
                   class="flagImage" title="Flag 'Optimization' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Signaling+networks%22" title="Search for articles in the subject area:'Signaling networks'"><div class="flagText">Signaling networks</div></a>
              <div data-categoryid="19723" data-articleid="23932"
                   data-categoryname="Signaling networks"
                   class="flagImage" title="Flag 'Signaling networks' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Statistical+distributions%22" title="Search for articles in the subject area:'Statistical distributions'"><div class="flagText">Statistical distributions</div></a>
              <div data-categoryid="32973" data-articleid="23932"
                   data-categoryname="Statistical distributions"
                   class="flagImage" title="Flag 'Statistical distributions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Statistical+signal+processing%22" title="Search for articles in the subject area:'Statistical signal processing'"><div class="flagText">Statistical signal processing</div></a>
              <div data-categoryid="23951" data-articleid="23932"
                   data-categoryname="Statistical signal processing"
                   class="flagImage" title="Flag 'Statistical signal processing' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2171'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=548'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8406&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
