

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001591"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001591" />

    <meta name="citation_title" content="Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability"/>
    <meta itemprop="name" content="Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability"/>

      <meta name="citation_author" content="Marc Mangel"/>
            <meta name="citation_author_institution" content="Center for Biomolecular Science and Engineering, Department of Applied Mathematics and Statistics, University of California Santa Cruz, Santa Cruz, California, United States or America"/>
      <meta name="citation_author" content="Michael B. Bonsall"/>
            <meta name="citation_author_institution" content="Mathematical Ecology Research Group, Department of Zoology, University of Oxford, Oxford, United Kingdom"/>

    <meta name="citation_date" content="2008/2/13"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001591.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1591"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Model systems in stem cell biology.; citation_author=JS Robert; citation_journal_title=BioEssays; citation_volume=26; citation_number=1; citation_pages=1005-1012; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Socializing with their neighbors: stem cells and their niche.; citation_author=E Fuchs; citation_author=T Tumbar; citation_author=G Guasch; citation_journal_title=Cell; citation_volume=116; citation_number=2; citation_pages=769-778; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Stem cells and heart disease.; citation_author=P Anversa; citation_author=A Leri; citation_author=J Kajstura; citation_number=3; citation_pages=425-430; citation_date=2006; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_author=R Lanza; citation_author=J Gearhart; citation_author=B Hogan; citation_author=D Melton; citation_author=R Pedersen; citation_number=4; citation_pages=548; citation_date=2006; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_title=Biology, molecular and oganismic.; citation_author=T Dobzhansky; citation_journal_title=Integ Compar Biol; citation_volume=4; citation_number=5; citation_pages=443-452; citation_date=1964; " />
      <meta name="citation_reference" content="citation_title=Stem cells and pattern formation in the nervous system: The possible versus the actual.; citation_author=DJ Anderson; citation_journal_title=Neuron; citation_volume=30; citation_number=6; citation_pages=19-35; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Adult stem cell plasticity: fact or artifact?; citation_author=M Raff; citation_journal_title=Annu Rev Cell and Devel Biol; citation_volume=19; citation_number=7; citation_pages=1-22; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The evolutionary ecology of stem cells and their niches-the time is now.; citation_author=M Mangel; citation_author=MB Bonsall; citation_journal_title=Oikos; citation_volume=116; citation_number=8; citation_pages=1779-1781; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Clonal evolution of stem and differentiated cells can be predicted by integrating cell-intrinsic and extrinsic parameters.; citation_author=S Viswanathan; citation_author=R Davey; citation_author=D Cheng; citation_author=RC Raghu; citation_author=DA Lauffenburger; citation_journal_title=Biotechnol Appl Bioc; citation_volume=42; citation_number=9; citation_pages=119-131; citation_date=2005; " />
      <meta name="citation_reference" content="citation_author=E Mayr; citation_number=10; citation_pages=1048; citation_date=1982; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_title=Out of Eden: Stem cells and their niches.; citation_author=FM Watt; citation_author=BLM Hogan; citation_journal_title=Science; citation_volume=287; citation_number=11; citation_pages=1427-1430; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Stochastic development of stem cells.; citation_author=H Vogel; citation_author=H Niewisch; citation_author=G Matioli; citation_journal_title=J Theor Biol; citation_volume=22; citation_number=12; citation_pages=249-270; citation_date=1969; " />
      <meta name="citation_reference" content="citation_title=Survival of the fittest: in vivo selection and stem cell gene therapy.; citation_author=T Neff; citation_author=BC Beard; citation_author=H-P Kiem; citation_journal_title=Blood; citation_volume=107; citation_number=13; citation_pages=1751-1760; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Stem cells: attributes, cycles, spirals, pitfalls and uncertainties lessons for and from the crypt.; citation_author=CS Potten; citation_author=M Loeffler; citation_journal_title=Development; citation_volume=110; citation_number=14; citation_pages=1001-1020; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Keratinocyte stem cells: A commentary.; citation_author=CS Potten; citation_author=C Booth; citation_journal_title=J Invest Dermatol; citation_volume=119; citation_number=15; citation_pages=88; citation_date=2002; " />
      <meta name="citation_reference" content="citation_author=U Alon; citation_number=16; citation_pages=320; citation_date=2007; citation_publisher=Chapman and Hall/CRC; " />
      <meta name="citation_reference" content="citation_title=Uses and abuses of mathematics in biology.; citation_author=RM May; citation_journal_title=Science; citation_volume=303; citation_number=17; citation_pages=790-793; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=RM May; citation_author=A McLean; citation_number=18; citation_pages=272; citation_date=2007; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_author=RA Fisher; citation_number=19; citation_pages=360; citation_date=1930; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_author=SA Frank; citation_number=20; citation_pages=400; citation_date=2007; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_author=PR Grant; citation_author=BR Grant; citation_number=21; citation_pages=272; citation_date=2007; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=Intravenous infusion of bone marrow in patients receiving radiation and chemotherapy.; citation_author=ED Thomas; citation_author=HL Lochte; citation_author=WC Lu; citation_author=JW Ferrebee; citation_journal_title=New Engl J Med; citation_volume=257; citation_number=22; citation_pages=491-496; citation_date=1957; " />
      <meta name="citation_reference" content="citation_title=Hematopoietic-cell transplantation at 50.; citation_author=FR Appelbaum; citation_journal_title=New Engl J Med; citation_volume=357; citation_number=23; citation_pages=1472-1475; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=A stochastic model of stem cell proliferation based on the growth of spleen colony-forming cells.; citation_author=JE Till; citation_author=EA McCulloch; citation_author=L Siminovitch; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=51; citation_number=24; citation_pages=29-36; citation_date=1964; " />
      <meta name="citation_reference" content="citation_author=M Mangel; citation_number=25; citation_pages=376; citation_date=2006; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_author=JD Murray; citation_number=26; citation_pages=736; citation_date=2003; citation_publisher=Springer; " />
      <meta name="citation_reference" content="citation_title=(Re)defining stem cells.; citation_author=S Shostak; citation_journal_title=BioEssays; citation_volume=28; citation_number=27; citation_pages=301-308; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Understanding cellular networks to improve hematopoietic stem cell expansion cultures.; citation_author=DC Kirouac; citation_author=PW Zandstra; citation_journal_title=Curr Opin Biotech; citation_volume=17; citation_number=28; citation_pages=538-547; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mutation selection and the natural history of cancer.; citation_author=J Cairns; citation_journal_title=Nature; citation_volume=255; citation_number=29; citation_pages=197-200; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=An adaptationist view of apoptosis.; citation_author=EK LeGrand; citation_journal_title=Q Rev Biol; citation_volume=72; citation_number=30; citation_pages=135-147; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Intestinal stem cells protect their genome by segregation of template DNA strands.; citation_author=CS Potten; citation_author=G Owen; citation_author=D Booth; citation_journal_title=J Cell Sci; citation_volume=115; citation_number=31; citation_pages=2381-2388; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Stem cell bioengineering.; citation_author=PW Zandstra; citation_author=A Nagy; citation_journal_title=Annu Rev of Biomed Eng; citation_volume=3; citation_number=32; citation_pages=275-305; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=M Mangel; citation_author=CW Clark; citation_number=33; citation_pages=308; citation_date=1988; citation_publisher=Princeton University Press; " />
      <meta name="citation_reference" content="citation_title=Definition and evaluation of behavioral and developmental programs.; citation_author=M Mangel; citation_author=D Ludwig; citation_journal_title=Annu Rev Ecol Syst; citation_volume=23; citation_number=34; citation_pages=507-536; citation_date=1992; " />
      <meta name="citation_reference" content="citation_author=AI Houston; citation_author=JM McNamara; citation_number=35; citation_pages=392; citation_date=1999; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_author=CW Clark; citation_author=M Mangel; citation_number=36; citation_pages=289; citation_date=2000; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_author=SE Kingsland; citation_number=37; citation_pages=324; citation_date=1985; citation_publisher=University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=Life-history trade-offs and ecological dynamics in the evolution of longevity.; citation_author=MB Bonsall; citation_author=M Mangel; citation_journal_title=Proc R Soc Lond B; citation_volume=271; citation_number=38; citation_pages=1143-1150; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=MA Nowak; citation_number=39; citation_pages=384; citation_date=2006; citation_publisher=Harvard University Press; " />
      <meta name="citation_reference" content="citation_author=D Wodarz; citation_author=N Komarova; citation_number=40; citation_pages=268; citation_date=2005; citation_publisher=World Scientific; " />
      <meta name="citation_reference" content="citation_title=The development of epithelial stem cell concepts.; citation_author=CS Potten; citation_author=JW Wilson; citation_number=41; citation_pages=11-21; citation_date=2006; citation_publisher=Elsevier; " />
      <meta name="citation_reference" content="citation_title=Mathematics of microbial populations.; citation_author=PR Painter; citation_author=AG Marr; citation_journal_title=Ann Rev Microbiol; citation_volume=22; citation_number=42; citation_pages=519-548; citation_date=1968; " />
      <meta name="citation_reference" content="citation_title=Sensitivity analysis of intracelluar signaling pathway kinetics predicts targets of stem cell fate control.; citation_author=A Madhavi; citation_author=RE Davey; citation_author=P Bhola; citation_author=T Yin; citation_author=PW Zandstra; citation_journal_title=PLoS Comput Biol; citation_volume=3; citation_number=43; citation_pages=1-11; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=LIF-mediated control of embryonic stem cell self-renewal emerges due to an autoregulatory loop.; citation_author=RE Davey; citation_author=K Onishi; citation_author=A Mahdavi; citation_author=PW Zandstra; citation_journal_title=FASEB J; citation_volume=21; citation_number=44; citation_pages=2020-2032; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=N MacDonald; citation_number=45; citation_pages=256; citation_date=1989; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_title=The shape of things to come: using models with physiological structure to predict mortality trajectories.; citation_author=M Mangel; citation_author=MB Bonsall; citation_journal_title=Theor Popul Biol; citation_volume=65; citation_number=46; citation_pages=353-359; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Efficient exact stochastic simulation of chemical systems with many species and many channels.; citation_author=MA Gibson; citation_author=J Bruck; citation_journal_title=J Chem Phys; citation_volume=104; citation_number=47; citation_pages=1876; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=The chemical Langevin equation.; citation_author=DT Gillespie; citation_journal_title=J Chem Phys; citation_volume=113; citation_number=48; citation_pages=297; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Approximate accelerated stochastic simulation of chemically reacting systems.; citation_author=DT Gillespie; citation_journal_title=J Chem Phys; citation_volume=115; citation_number=49; citation_pages=1716; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Improved leap-size selection for accelerated stochastic simulation.; citation_author=DT Gillespie; citation_author=LR Petzold; citation_journal_title=J Chem Phys; citation_volume=119; citation_number=50; citation_pages=8229; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=On a functional central limit theorem for Markov population processes.; citation_author=AD Barbour; citation_journal_title=Adv Appl Probab; citation_volume=6; citation_number=51; citation_pages=21-39; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Equilibrium distributions for Markov population processes.; citation_author=AD Barbour; citation_journal_title=Adv App Probab; citation_volume=12; citation_number=52; citation_pages=591-614; citation_date=1980; " />
      <meta name="citation_reference" content="citation_title=Solutions of ordinary differential equations as limits of pure jump Markov processes.; citation_author=TG Kurtz; citation_journal_title=J Appl Probab; citation_volume=7; citation_number=53; citation_pages=49-58; citation_date=1970; " />
      <meta name="citation_reference" content="citation_title=Limit theorems for sequences of jump Markov processes approximating ordinary differential processes.; citation_author=TG Kurtz; citation_journal_title=J Appl Probab; citation_volume=8; citation_number=54; citation_pages=344-356; citation_date=1971; " />
      <meta name="citation_reference" content="citation_title=The relationship between stochastic and deterministic models for chemical reactions.; citation_author=TG Kurtz; citation_journal_title=J Chem Phys; citation_volume=57; citation_number=55; citation_pages=2976-2978; citation_date=1972; " />
      <meta name="citation_reference" content="citation_title=Barrier transitions driven by fluctuations, with applications to ecology and evolution.; citation_author=M Mangel; citation_journal_title=Theor Popul Biol; citation_volume=45; citation_number=56; citation_pages=16-40; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Can adaptive dynamics invade?; citation_author=U Dieckmann; citation_journal_title=Trends Ecol Evol; citation_volume=12; citation_number=57; citation_pages=128-131; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=How should we define fitness for general ecological scenarios?; citation_author=JAJ Metz; citation_author=RM Nisbet; citation_author=SAH Geritz; citation_journal_title=Trends Ecol Evol; citation_volume=7; citation_number=58; citation_pages=198-202; citation_date=1992; " />
      <meta name="citation_reference" content="citation_author=TL Vincent; citation_author=JS Brown; citation_number=59; citation_pages=400; citation_date=2005; citation_publisher=Cambridge University Press; " />
      <meta name="citation_reference" content="citation_author=EC Pielou; citation_number=60; citation_pages=385; citation_date=1977; citation_publisher=John Wiley and Sons; " />
      <meta name="citation_reference" content="citation_title=Evolutionary analysis of life span, competition, and adaptive radiation, motivated by the Pacific rockfishes (Sebastes).; citation_author=M Mangel; citation_author=HK Kindsvater; citation_author=MB Bonsall; citation_journal_title=Evolution; citation_volume=61; citation_number=61; citation_pages=1208-1224; citation_date=2007; " />
      <meta name="citation_reference" content="citation_author=ML Puterman; citation_number=62; citation_pages=672; citation_date=1994; citation_publisher=Wiley Interscience; " />
      <meta name="citation_reference" content="citation_title=Dynamic optimization over infinite-time horizon: Web-building strategy in an orb-weaving spider as a case study.; citation_author=S Venner; citation_author=I Chads; citation_author=M-C Bel-Venner; citation_author=A Pasquet; citation_author=F Charpillet; citation_journal_title=J Theor Biol; citation_volume=241; citation_number=63; citation_pages=725-733; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability"/>
    <meta name="twitter:description" content="Phenotypic evolutionary models have been used with great success in many areas of biology, but thus far have not been applied to the study of stem cells except for investigations of cancer. We develop a framework that allows such modeling techniques to be applied to stem cells more generally. The fundamental modeling structure is the stochastic kinetics of stem cells in their niche and of transit amplifying and fully differentiated cells elsewhere in the organism, with positive and negative feedback. This formulation allows graded signals to be turned into all or nothing responses, and shows the importance of looking beyond the niche for understanding how stem cells behave. Using the deterministic version of this framework, we show how competition between different stem cell lines can be analyzed, and under what circumstances stem cells in a niche will be replaced by other stem cells with different phenotypic characteristics. Using the stochastic version of our framework and state dependent life history theory, we show that the optimal behavior of a focal stem cell will involve long periods of quiescence and that a population of identical stem cells will show great variability in the times at which activity occurs; we compare our results with classic ones on quiescence and variability in the hematopoietic system."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001591.g006"/>

  <meta property="og:title" content="Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1428'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=7701'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=982&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001591">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001591" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001591&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Marc Mangel
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:msmangel@ucsc.edu">msmangel@ucsc.edu</a></p>

                <p>Affiliation:
                  Center for Biomolecular Science and Engineering, Department of Applied Mathematics and Statistics, University of California Santa Cruz, Santa Cruz, California, United States or America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Michael B. Bonsall
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Mathematical Ecology Research Group, Department of Zoology, University of Oxford, Oxford, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 13, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001591</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001591-g001" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001591-t001" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001591-g002" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001591-g003" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001591-g004" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001591-g005" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001591-g006" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001591">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001591" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Phenotypic evolutionary models have been used with great success in many areas of biology, but thus far have not been applied to the study of stem cells except for investigations of cancer. We develop a framework that allows such modeling techniques to be applied to stem cells more generally. The fundamental modeling structure is the stochastic kinetics of stem cells in their niche and of transit amplifying and fully differentiated cells elsewhere in the organism, with positive and negative feedback. This formulation allows graded signals to be turned into all or nothing responses, and shows the importance of looking beyond the niche for understanding how stem cells behave. Using the deterministic version of this framework, we show how competition between different stem cell lines can be analyzed, and under what circumstances stem cells in a niche will be replaced by other stem cells with different phenotypic characteristics. Using the stochastic version of our framework and state dependent life history theory, we show that the optimal behavior of a focal stem cell will involve long periods of quiescence and that a population of identical stem cells will show great variability in the times at which activity occurs; we compare our results with classic ones on quiescence and variability in the hematopoietic system.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Mangel M, Bonsall MB (2008) Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability. PLoS ONE 3(2):
          e1591.
            doi:10.1371/journal.pone.0001591</p><p><strong>Academic Editor: </strong>Mark Rees, University of Sheffield, United Kingdom</p><p><strong>Received:</strong> January 9, 2008; <strong>Accepted:</strong> January 16, 2008; <strong>Published:</strong> February 13, 2008</p><p><strong>Copyright:</strong> © 2008 Mangel, Bonsall. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the NSF, Royal Society, and the Astor Travel Fund, University of Oxford.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The current enthusiasm for stem cells (both adult and embryonic) <a href="#pone.0001591-Robert1">[1]</a>, and their role in regenerative medicine is based on the assumption that we can remove stem cells from their natural habitat, propagate them in culture, transplant them into a foreign environment and assume that the transplanted cells will do as we wish or that we can manipulate them <em>in vivo</em> with desired results <a href="#pone.0001591-Fuchs1">[2]</a>. For example, stem cells can differentiate into cell lineages that are different from the tissue in which they originate and adult stem cells can migrate to sites of injury <a href="#pone.0001591-Anversa1">[3]</a>. Harold Varmus has said that “It is not unrealistic to say that [stem cell] research has the potential to revolutionize the practice of medicine and improve the quality and length of life” (pg. 513 in <a href="#pone.0001591-Lanza1">[4]</a>). But long before that, Theodosius Dobzhansky said that us “nothing makes sense in biology except in the light of evolution” <a href="#pone.0001591-Dobzhansky1">[5, pg 449]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There may be enormous differences between what stem cells do in their original tissue and what they can and will do when put into culture or when transplanted to a new location <a href="#pone.0001591-Anderson1">[6]</a>. Raff <a href="#pone.0001591-Raff1">[7]</a> noted that “perhaps the greatest challenge in stem cell biology is to uncover the…mechanisms that determine whether a daughter of a stem cell division self-renews or commits to a particular pathway of differentiation. Cracking this problem for the adult mammalian stem cells of interest will be a crucial step for both developmental biology and cell therapy.” Solutions in this area of stem-cell biology will require all sorts of different biology, and innovative ideas. Elsewhere <a href="#pone.0001591-Mangel1">[8]</a> we have argued that the time is right for the development of evolutionary theory about stem cells and the microenvironment (their niche) that maintains them.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>It is generally agreed that a focal stem cell in its niche is influenced by signals (both positive and negative feedback) from the other stem cells in the niche, the support cells in the niche, and from more differentiated cells throughout the body. How exactly to model this is unclear, with the consequence that most models of stem cells focus on a single stem cell (or a population of identically behaving stem cells) in a single niche. We show how to make operational the recommendation in <a href="#pone.0001591-Viswanathan1">[9]</a> that the next advances will combine probability-based events and mechanistic parameters as the best approximation to actual phenomena. Our models are rooted in evolution by natural selection, which is ultimately <span class="underline">the</span> mechanism for all of biology.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Theoretical approaches to stem cell biology have usually involved typological thinking <a href="#pone.0001591-Mayr1">[10]</a>, although there are examples of population thinking <a href="#pone.0001591-Watt1">[11]</a>. Typically, one assumes that all stem cells behave in a similar manner and writes either a set of deterministic differential equations or a probabilistic master equation <a href="#pone.0001591-Vogel1">[12]</a> to characterize the dynamics of stem cells, but ignores natural selection. What is now required is a kinetic theory of stem cell dynamics that accounts for variation and natural selection <a href="#pone.0001591-Neff1">[13]</a>. We offer the first such theory here. We also show how the biological version of the Heisenberg uncertainty principle <a href="#pone.0001591-Potten1">[14]</a>, <a href="#pone.0001591-Potten2">[15]</a> can be made operational to show how manipulations may alter the behavior and progeny of stem cells. To achieve our goals, we use a somewhat stylized model, motivated by the hematopoietic system. There is much to be learned from such stylized models, in systems biology <a href="#pone.0001591-Alon1">[16]</a>, ecology <a href="#pone.0001591-May1">[17]</a>, <a href="#pone.0001591-May2">[18]</a>, and evolution <a href="#pone.0001591-Fisher1">[19]</a>–<a href="#pone.0001591-Grant1">[21]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>September 2007 was the 50th anniversary of the publication of the first successful demonstration of the intravenous infusion of bone marrow in patients who had experienced radiation or chemotherapy that destroyed bone marrow <a href="#pone.0001591-Thomas1">[22]</a>, <a href="#pone.0001591-Appelbaum1">[23]</a>. In a subsequent classic study of the hematopoietic stem cell system <a href="#pone.0001591-Till1">[24]</a>, stem cell colony forming units showed enormous epigenetic variation, leading to highly overdispersed numbers of descendant cells in individual colonies in spleens of experimental animals. We will show how to interpret this result in terms of variability in the cell cycle <a href="#pone.0001591-Mangel2">[25]</a>; this begs the question of why such variation has not been eliminated, or alternatively how natural selection maintains such variation.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>In its niche, a stem cell responds to its local environment (signals received from the support cells of and the other stem cells in the niche) and its global environment (signals received from more differentiated cells) but we assume that the behavior of the focal stem cell does not affect the behavior of the other stem or more fully differentiated cells. We are thus able to avoid, at least initially, game theoretical aspects of stem cell biology; ultimately they must be considered. Even with this simplified framework, we are able to understand how natural selection will act on competition between stem cells with different phenotypic characteristics (explained below) and how natural selection can lead to both quiescence and great variation in the behavior of stem cells.</p>

<h4>Overview of the Models</h4>
<a id="article1.body1.sec1.sec1.p1" name="article1.body1.sec1.sec1.p1"></a><p>Stem cells have the ability to both self-renew and to differentiate. The niche of the stem cell is the local microenvironment that supports the maintenance, renewal, and differentiation of stem cells. In the case of the villi of the small intestine or bone marrow it is easy to conceive of the niche as a defined physical environment. In other cases, such as the epidermis, the niche will be determined by the spatial extent of signaling molecules. Signals and feedback controls (both negative and positive) affect the pattern of behavior of the stem cells in their niche. These signals include cell-autonomous promotors (such as <em>piwi</em>); self-feedback, autocrine and paracrine influences; hormones, growth factors, cytokines, and short-range cell to cell signalling pathways (e.g. Notch, Hedgehog, Wnt); hormones from remote sources; positional cues from chemical gradients as in classical Turing systems <a href="#pone.0001591-Murray1">[26]</a>; and adhesion <a href="#pone.0001591-Shostak1">[27]</a>. Thus, stem cells are not simply single input-output systems, but are complex, nonlinear, multiple input-output systems <a href="#pone.0001591-Kirouac1">[28]</a>.</p>
<a id="article1.body1.sec1.sec1.p2" name="article1.body1.sec1.sec1.p2"></a><p>The simplest population biology that one can imagine is to focus on the number of stem <em>S</em>(<em>t</em>), transit amplifying progenitor <em>A</em>(<em>t</em>), and fully differentiated <em>D</em>(<em>t</em>) cells at time <em>t</em> associated with the stem cells in the niche being modeled. In doing so, for simplicity we compress the differentiation stages of a transit amplifying cell, but they could be unpacked should one wish to do so. In addition, there will be other transit amplifying and fully differentiated cells, produced by other stem cells, that create an additional signaling background. We denote those cells by <em>B</em>. For simplicity we consider only one kind of progenitor cell and only one kind of differentiated cell, but extensions are clear and obvious. The positive and negative feedback links affecting stem cell fate are illustrated in <a href="#pone-0001591-g001">Figure 1</a>.</p>
<div class="figure" id="pone-0001591-g001"><div class="img"><a name="pone-0001591-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The feedback network affecting a stem cell in its niche.</span></strong></p><a id="article1.body1.sec1.sec1.fig1.caption1.p1" name="article1.body1.sec1.sec1.fig1.caption1.p1"></a><p>Here stem cells are clear circles, transit amplifying cells colored circles, and fully differentiated cells in black. A focal stem cell has positive feedback (+) on itself but shares inhibition and negative feedback (−) with other stem cells. Transit amplifying cells receive positive feedback from stem cells and exert positive feedback on fully differentiated cells. Both transit amplifying and fully differentiated cells exhibit negative feedback on stem cells. The symbols that characterize these processes are explained in the <a href="#s4">Materials and Methods</a>.</p>
<span>doi:10.1371/journal.pone.0001591.g001</span></div>

<h4>The transitions of stem, transit amplifying, and fully differentiated cells</h4>
<a id="article1.body1.sec1.sec2.p1" name="article1.body1.sec1.sec2.p1"></a><p>Most simply, stem cells divide symmetrically – doubling and producing two stem cells, divide asymmetrically – doubling and producing one stem cell and one cell that will be a progenitor of a fully differentiated cell, undergo apoptosis (to maintain an error-free genome <a href="#pone.0001591-Cairns1">[29]</a>, <a href="#pone.0001591-LeGrand1">[30]</a>) or migrate out of the niche <a href="#pone.0001591-Potten3">[31]</a>. Almost all of the literature focuses on whether stem cells will divide symmetrically or asymmetrically.</p>
<a id="article1.body1.sec1.sec2.p2" name="article1.body1.sec1.sec2.p2"></a><p>In fact, there are many more possible transitions of stem cells and their progeny. We use the following indexing system for transitions of stem, transit amplifying and fully differentiated cells: 1) a stem cell divides symmetrically, 2) a stem cell divides asymmetrically, 3) a stem cell divides and differentiates symmetrically producing two transit amplifying cells, 4) a stem cell under goes apoptosis or migrates out of the niche, 5) a transit amplifying cell divides, 6) a transit amplifying cell undergoes apoptosis, 7) a transit amplifying cell fully differentiates, 8) a fully differentiated cell undergoes apoptosis. To fully understand the evolutionary ecology of stem cells we need to consider the entire range of possibilities of transitions and both positive and negative feedback. This is implicit in <a href="#pone.0001591-Zandstra1">[32]</a>. In <a href="#s4"><strong>Materials and Methods</strong></a> we describe how to model these transitions, implement the model and associate a system of ordinary differential equations with the conditional means of the stochastic numbers of stem, transit amplifying, and fully differentiated cells.</p>


<h4>Trade offs</h4>
<a id="article1.body1.sec1.sec3.p1" name="article1.body1.sec1.sec3.p1"></a><p>The transitions of stem, transit amplifying, and differentiated cells are characterized by a number of parameters. Biological realities will constrain some choices for parameter values. For example, we may anticipate that the probability that all stem cells in the niche either undergo apoptosis or migrate out of the niche is not too high since if it were the system could not persist. That is, if the niche empties (either through apoptosis or migration) it has essentially gone extinct. Similarly, in most situations, the population of transit amplifying cells is much more active than the population of stem cells (with consequence that the number of amplifying cells will thus be larger than the number of stem cells) and we require parameter values that make this so. We also require that when there is just one stem cell in the niche, the numbers of stem cells increase.</p>
<a id="article1.body1.sec1.sec3.p2" name="article1.body1.sec1.sec3.p2"></a><p>In addition, we may expect there to exist links and trade-offs between the different parameters. For example, since symmetric renewal, asymmetric renewal and symmetric differentiation involve many of the same signals and processes, we may expect that their rates are correlated. Cells that undergo high rates of replication are more likely to have errors in them and thus are likely to undergo apoptosis. When transit amplifying cells disappear at a higher rate because of replication error, fewer of them will become fully differentiated cells and this too can be captured by a trade-off.</p>
<a id="article1.body1.sec1.sec3.p3" name="article1.body1.sec1.sec3.p3"></a><p>In summary, one may conceive of the parameter set describing the transitions of the stochastic kinetics as the phenotype of a stem cell and its descendants, or if all of the stem cells in a niche are identical descendants of a single cell, then these parameters are the phenotype of the stem cell clone. These trade-offs are fully described in <a href="#s4"><strong>Materials and Methods</strong></a>.</p>


<h4>The focal stem cell in the niche</h4>
<a id="article1.body1.sec1.sec4.p1" name="article1.body1.sec1.sec4.p1"></a><p>To understand fully how natural selection acts on a stem cell, we need to consider the fitness of a focal stem cell. Stem cells (and transit amplifying cells) do not by themselves achieve fitness. Rather, they support the organism – which can be viewed as an organized collection of fully differentiated cells. Fitness of the focal stem cell depends upon what it does, what the other stem cells in the niche do, and how many transit amplifying and differentiated cells are present. We thus define a fitness function <em>F</em>(<em>y</em>, <em>s</em>, <em>a</em>, <em>d</em>) to be the maximum expected accumulated fitness through differentiated cells, given that the focal stem cell has accumulated <em>y</em> resources towards the next division, that there are <em>s</em> stem cells in the niche and that there are <em>a</em> and <em>d</em> transit amplifying and differentiated cells sending signals back to the focal stem cell. State dependent life history theory, as implemented through stochastic dynamic programming <a href="#pone.0001591-Mangel3">[33]</a>–<a href="#pone.0001591-Clark1">[36]</a>, allows us to compute the fitness-maximizing responses of the focal stem cell.</p>

</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1"></a><p>A variety of parameters are used in this section; they are explained in <a href="#pone-0001591-t001">Table 1</a> and constitute the phenotype of the stem cell and its descendants. For modeling, it is generally easier to work with the rates of cell cycles (generically, λ) rather than cell cycle times (generically <em>T</em>).</p>
<div class="figure" id="pone-0001591-t001"><div class="img"><a name="pone-0001591-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Parameters, Their Interpretations, and Values.</span></strong></p><span>doi:10.1371/journal.pone.0001591.t001</span></div>
<h4>Another look at Till et al (1964)</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We begin with another look at Till et al <a href="#pone.0001591-Till1">[24]</a>. Their data lead to a frequency distribution for cell cycle times that is shown in <a href="#pone-0001591-g002">Figure 2</a>. We are again led to ask how natural selection maintains such a broad and slowly declining distribution.</p>
<div class="figure" id="pone-0001591-g002"><div class="img"><a name="pone-0001591-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The distribution of cell cycle times inferred from the overdispersed data of Till et al.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>This should be compared to <a href="#pone-0001591-g006">Figure 6</a>.</p>
<span>doi:10.1371/journal.pone.0001591.g002</span></div>

<h4>A special case of the differential equations</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>If the rate of symmetric differentiation is 0, we can solve for the steady state levels of stem and differentiated cells explicitly and easily obtain that of the amplifying cells numerically. In particular, the steady state number of stem cells is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e001&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>which shows that the steady state number of stem cells increases with increasing values of λ<sub>1</sub> and with decreasing values of μ<sub>4</sub> (<a href="#pone-0001591-t001">Table 1</a>), both of which accord with intuition. If the steady state number of transit amplifying cells were known, the steady state number of differentiated cells is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e002&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Thus, by adjustment of any of φ, μ<sub>6</sub> or μ<sub>8</sub> (<a href="#pone-0001591-t001">Table 1</a>), natural selection can lead to different numbers of fully differentiated cells without changing the fundamental structure of the system of stem, transit amplifying and differentiated cells. That is, we can create a ratio of fully differentiated to stem cell that ranges from the observed 3:1 to 2–5,000:1 through the evolution of phenotypic characteristics.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>We obtain a single nonlinear equation for the steady state level of transit amplifying cells<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e003&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e004&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e005&amp;representation=PNG" class="inline-graphic"></span>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>This special case also serves as a check of our numerical methods.</p>


<h4>Qualitative properties of the stochastic system</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>As described in the <a href="#s4"><strong>Materials and Methods</strong></a>, under very general conditions, <em>S</em> = 0 is dynamically unstable and has maximum value <em>K</em> (determined by feedback inhibition). Thus, the solution of the set of differential equations will move from <em>S</em> = 1 (the minimum non-zero value of the number of stem cells in the niche) towards <em>K</em>, determined by the balance of μ<sub>4</sub> and λ<sub>1</sub>. In the full stochastic simulation, all the same properties hold, as can be seen from the equations characterizing the transition rates. One important consequence of <em>S</em> = 0 being dynamically unstable is that when a niche goes extinct, it will do so quickly <a href="#pone.0001591-Mangel2">[25]</a>. Thus, we expect extant clones of stem cells to have much longer lifetimes than those of clones heading for extinction. Similarly, the number of transit amplifying cells is determined by the balance of μ<sub>6</sub>−λ<sub>5</sub> and φ. The number of fully differentiated cells is determined by μ<sub>8</sub>, φ, and μ<sub>6</sub> – thus a variety of simple mechanisms can create tissues with different structures.</p>


<h4>Quantitative results</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>In <a href="#pone-0001591-g003">Figure 3</a>, we show the trajectories for transit amplifying (<a href="#pone-0001591-g003">Figure 3</a>, upper panel) and fully differentiated (<a href="#pone-0001591-g003">Figure 3</a>, lower panel) cells and the associated solution of the differential equation system. We see that the set of differential equations is a good proxy for the mean trajectory of the stochastic system and that the number of fully differentiated cells has a trajectory that mimics that of the transit amplifying cells, as it must. In <a href="#pone-0001591-g004">Figure 4</a>, we show the realizations of the stochastic system as phase plane plots of the density of stem cells and transit amplifying cells (upper panel) or stem cells and fully differentiated cells (lower panel). We see that a wide-range of cell numbers is expected due to the inherent stochasticity in the system.</p>
<div class="figure" id="pone-0001591-g003"><div class="img"><a name="pone-0001591-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The stochastic trajectories for 100 simulations of the stem-transit amplifying-differentiated system described in the text and the solution of the differential equations 25–27 (red line).</span></strong></p><span>doi:10.1371/journal.pone.0001591.g003</span></div><div class="figure" id="pone-0001591-g004"><div class="img"><a name="pone-0001591-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>The results of the stochastic simulation can be shown as phase plots of stem cells and transit amplifying cells (uppere panel) or stem cells and fully differentiated cells (lower panel)</span></strong></p><span>doi:10.1371/journal.pone.0001591.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In <a href="#pone-0001591-g005">Figure 5</a>, we show two results of the invasion/replacement analysis. First (upper panel) we consider the invasion of a resident population of stem cells by a new phenotype with a different value of cell cycle rate for symmetric renewal <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e006&amp;representation=PNG" class="inline-graphic"></span>. Here we see that in general if λ<sub>1</sub> exceeds <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e007&amp;representation=PNG" class="inline-graphic"></span> then the residents will resist the invasion by the new stem cell type and in general if the converse is true then the residents are excluded by the invading stem cell. However, there is a small region in which both kinds of stem cells coexist. Second (lower panel) we consider the replacement analysis when the value of the rate of the cell cycle for symmetric differentiation λ<sub>3</sub> varies between the resident and invading stem cells. Here the region of coexistence is greater and the boundary curves, while still symmetrical are more complicated.</p>
<div class="figure" id="pone-0001591-g005"><div class="img"><a name="pone-0001591-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Deterministic replacement analysis allow us to determine when a mutation in one of the phenotypic parameters or the introduction of stem cells with different phenotypic parameters into a niche will lead to replacement of one stem cell clone by another, or coexistence of clones in the same niche.</span></strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>Upper panel) Stem cells differ in the cell cycle rate for symmetric renewal λ<sub>1</sub>. Lower panel) Stem cells differ in the cell cycle rate for symmetric differentiation λ<sub>3</sub>.</p>
<span>doi:10.1371/journal.pone.0001591.g005</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>In <a href="#pone-0001591-g006">Figure 6</a> we show the frequency distributions of transitions by the focal stem cell following the behavior determined from the stochastic dynamic programming model. These data refer to 22,200 simulations in which there are about 3507 transitions of the simulated stem cells, of which only 6.42 per cent were symmetric renewal; the remaining transitions were asymmetric renewal. The minimum value of resources needed for a transition is <em>y<sub>d</sub></em> = 5.5, but in the simulated stem cell population the average value of resources at the time of transition is 7.1, and the variance of those resources is 0.84. The average number of transit amplifying and background cells at the time of transition is 11.1, and the variance in those numbers is 53.5.</p>
<div class="figure" id="pone-0001591-g006"><div class="img"><a name="pone-0001591-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001591" data-uri="info:doi/10.1371/journal.pone.0001591.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001591.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001591.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001591.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>The forward simulation of the decisions by the focal stem cell allows us to generate the frequency distribution of the rates of transition, comparable to <a href="#pone-0001591-g002">Figure 2</a>.</span></strong></p><span>doi:10.1371/journal.pone.0001591.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Phenotypic models of evolutionary processes have made important and long-lasting contributions in ecology and evolutionary biology <a href="#pone.0001591-Kingsland1">[37]</a>–<a href="#pone.0001591-Nowak1">[39]</a> and in the study of cancer <a href="#pone.0001591-Frank1">[20]</a>, <a href="#pone.0001591-Wodarz1">[40]</a>. They offer promise to help us understand great swaths of the life history and dynamics of stem cells and their descendants. Most of our work in this paper is in developing the population biology of stem, transit amplifying and fully differentiated cells in their niche to provide a fundamental structure for the analyses shown in <a href="#pone-0001591-g005">Figures 5</a> and <a href="#pone-0001591-g006">6</a>. This structure provides the framework for answering a wide range of questions about the evolutionary biology of stem cells. The results of <a href="#pone-0001591-g005">Figure 5</a> show that it is possible to use evolutionary replacement analyses to understand consequences of manipulation of stem cell life history parameters <em>in vivo</em> and show, for example, that not all manipulations will be successful (the invading stem cell line cannot persist) or perhaps may be too successful (the invading stem cells fully replace the resident ones). We have not allowed mutations in the phenotypic parameters of the stem cell clones, but this is possible too with our methods <a href="#pone.0001591-Bonsall1">[38]</a> and would allow one to investigate how the clone changes over time due to mutation.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p><a href="#pone-0001591-g006">Figure 6</a> shows that it is possible to understand the key properties of stem cells – quiescence and great variability in activity – in terms of natural selection acting on the decisions of stem cells in response to the signals from the local microenvironment (the other stem cells) and from the more differentiated cells in the rest of the organism. Although it is comforting to see the similarity of <a href="#pone-0001591-g002">Figure 2</a> and <a href="#pone-0001591-g006">6</a>, one should remember that pattern does not imply process. However, the mechanistic model that underlies <a href="#pone-0001591-g006">Figure 6</a> will allow us (elsewhere) to explore the key components of the process that lead to the pattern.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>An empirical goal suggested by our work would be to more carefully identify these trade-offs and the feedback functions (described in detail in <a href="#s4"><strong>Materials and Methods</strong></a>) that characterize asymmetric renewal and symmetric differentiation. We have used simple exponential functions, but others – in which the all-or-nothing response is built in – are clearly possible <a href="#pone.0001591-Alon1">[16]</a>. How the strength of these signals will evolve also remains an open question.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our work also provides a number of qualitative insights. For example, in healthy organisms, we should not expect the stem populations to be at their maximum sizes. Rather, the stochastic processes of birth and death will lead to population sizes smaller than the maximum. This, of course, makes identifying pathological situations more difficult, but should also help us avoid leaping to inappropriate conclusions. Furthermore, one cannot conceive of the transitions of stem cells (quiescence, symmetric renewal, asymmetric renewal, symmetric differentiation, apoptosis, or migration) in the absence of feedback from the amplifying transit cells and the fully differentiated cells. Indeed, they are crucial to what happens in the niche. We have also shown that a single framework can account for the observed ratio of fully differentiated to stem cells <a href="#pone.0001591-Potten4">[41]</a> by varying the phenotypic characteristics.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Many questions remain and there is much work to be done, ranging from the development of systems biology models for the phenotypic parameters, to the evolution of the signalling functions and their parameters, to the development of more complicated game theoretical aspects. Each will provide new insight into the evolutionary ecology of stem cells and their niches.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Another look at Till et al 1964</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Till et al. <a href="#pone.0001591-Till1">[24]</a> recognized that overdispersed data (in which the variance is much larger than the mean) such as theirs could not be described by a Poisson distribution. Instead, they fit a gamma distribution <a href="#pone.0001591-Mangel2">[25]</a> to their data. The Poisson process is equivalent to the assumption that the probability of a stem cell being active in the next <em>dt</em> is λΔ<em>t</em>+<em>o</em>(Δ<em>t</em>), where <em>o</em>(Δ<em>t</em>) indicates times that are higher powers of Δ<em>t</em>. An improvement upon their approach is the following. It is known that there is variation in cell cycle times <a href="#pone.0001591-Painter1">[42]</a>. Thus, rather than fitting the counts with a gamma density, we assign a gamma density to λ, leading to a negative binomial distribution for the counts <a href="#pone.0001591-Mangel2">[25]</a>. We use the method of moments to infer the parameters of this gamma density. Since the cell cycle time <em>T</em> = 1/λ, it is then an elementary calculation to determine the frequency distribution of cell cycle time, given the inferred gamma density.</p>


<h4>The stochastic kinetics of transitions</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We use <em>s</em>, <em>a</em>, <em>d</em> for particular values of the number of stem, transit amplifying, or fully differentiated cells. We characterize the feedback network in <a href="#pone-0001591-g001">Figure 1</a> as follows. We let Δ<em>S</em>, Δ<em>A</em> and Δ<em>D</em> denote the change in the number of stem, amplifying, or differentiated cells in the interval of time Δ<em>t</em>, which is assumed to be small. As noted before, <em>o</em>(Δ<em>t</em>) denotes terms that are higher powers of Δ<em>t</em> (typically Δ<em>t</em><sup>2</sup>, Δ<em>t</em><sup>3</sup> etc.).</p>
<h5>Symmetric Renewal.</h5><a id="article1.body1.sec4.sec2.sec1.p1" name="article1.body1.sec4.sec2.sec1.p1"></a><p><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e008&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e009&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br></p>
<a id="article1.body1.sec4.sec2.sec1.p2" name="article1.body1.sec4.sec2.sec1.p2"></a><p>In this equation, λ<sub>1</sub> = 1/<em>T</em><sub>1</sub> is the cell cycle time for stem cells that renew symmetrically. We assume no variation in this rate. The function Φ<sub>1</sub>(<em>s</em>) characterizes the inhibition of stem cells upon each other <a href="#pone.0001591-Madhavi1">[43]</a>. A variety of function forms are possible <a href="#pone.0001591-Alon1">[16]</a> but the key is that Φ<sub>1</sub>(<em>s</em>) decreases as <em>s</em> increases (which captures the inhibition); for computations we use Φ<sub>1</sub>(<em>s</em>) = <em>ln</em>(<em>K</em>)−<em>ln</em>(<em>s</em>).</p>

<h5>Asymmetric Renewal.</h5><a id="article1.body1.sec4.sec2.sec2.p1" name="article1.body1.sec4.sec2.sec2.p1"></a><p><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e011&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>In analogy to Eqn (5), λ<sub>2</sub> is the cell cycle rate when a stem cell renews asymmetrically, with associated cell cycle time <em>T</em><sub>2</sub>. Inhibition of symmetric renewal occurs for two reasons. First, negative feedback from stem cells on stem cells decreases overall activity. Second, negative feedback from existing transit amplifying and fully differentiated cells inhibits the production of transit amplifying cells. Thus Φ<sub>2</sub>(<em>a</em>,<em>d</em>,<em>b</em>) characterizes the signal from the transit amplifying and fully differentiated cells to the stem cells. Once again, this function will decrease as the number of transit amplifying and fully differentiated cells increases. For computations or analysis we use Φ<sub>2</sub>(<em>a</em>,<em>d</em>,<em>b</em>) = <em>e</em><sup>−<em>s</em>(<em>a+d+b</em>)</sup> which is equivalent to a Hill function when the argument is small. ε measures the sensitivity of the stem cell population to what is happening in the population of transit amplifying and fully differentiated cells. Stem cells convert graded stimuli into all or nothing responses <a href="#pone.0001591-Davey1">[44]</a> and we have specifically chosen a graded rather than all or nothing response <a href="#pone.0001591-Alon1">[16]</a> because that allows the all or nothing response to emerge, rather than to be built into the model.</p>

<h5>Symmetric Differentiation.</h5><a id="article1.body1.sec4.sec2.sec3.p1" name="article1.body1.sec4.sec2.sec3.p1"></a><p><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br>Here λ<sub>3</sub> and Φ<sub>3</sub>(<em>a</em>,<em>d</em>,<em>b</em>) are analogous to those above but can be understood as a ‘more desperate’ signal. For computations or analysis we use Φ<sub>3</sub>(<em>a</em>,<em>d</em>,<em>b</em>) = <em>e</em><sup>−γ(<em>a</em>+<em>d</em>+<em>b</em>)</sup> where <em>γ</em> is the sensitivity parameter for symmetric differentiation.</p>

<h5>Migration/Apoptosis.</h5><a id="article1.body1.sec4.sec2.sec4.p1" name="article1.body1.sec4.sec2.sec4.p1"></a><p>From the viewpoint of the focal stem cell, whether another stem cell undergoes apoptosis or migrates out of the niche (which is surely important for the state of the organism) is immaterial – the consequence is the same, a reduction in the number of stem cells in this niche. Thus we combine them in<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br>Here μ<sub>4</sub> measures the rate of migration of stem cells out of the niche plus the rate of apoptosis, both on a per stem cell basis.</p>

<h5>Transit Cell Amplification.</h5><a id="article1.body1.sec4.sec2.sec5.p1" name="article1.body1.sec4.sec2.sec5.p1"></a><p>Transit cells amplify and since they send a signal back to the niche, their dynamics must be included. Hence<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br>Here λ<sub>5</sub> is the cell cycle rate for transit amplifying cells. As mentioned above, we have compressed the <em>n</em> stages of development from transit amplifying to fully differentiated cell; unpacking this assumption can be done through a linear chain <a href="#pone.0001591-Bonsall1">[38]</a>, <a href="#pone.0001591-MacDonald1">[45]</a>, <a href="#pone.0001591-Mangel5">[46]</a>.</p>

<h5>Transit Cell Mortality.</h5><a id="article1.body1.sec4.sec2.sec6.p1" name="article1.body1.sec4.sec2.sec6.p1"></a><p>Transit amplifying cells will disappear because of apoptosis (e.g. because they have too many DNA errors) and because they fully differentiate. We let μ<sub>6</sub> denote the total per capita rate at which transit amplifying cells disappear; a fraction φ of them are converted to differentiated cells and the remainder experience apoptosis. Thus the mortality of transit amplifying cells is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e018&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e019&amp;representation=PNG" class="inline-graphic"><span class="note">(15)</span></span><br></p>

<h5>Transit Cell Differentiation.</h5><a id="article1.body1.sec4.sec2.sec7.p1" name="article1.body1.sec4.sec2.sec7.p1"></a><p>Transit amplifying cells that disappear but do not die are converted into fully differentiated cells as in<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(16)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e021&amp;representation=PNG" class="inline-graphic"><span class="note">(17)</span></span><br>Note that ρ<sub>6</sub>+ρ<sub>7</sub> = μ<sub>6</sub><em>a,</em> the total rate at which transit amplifying cells disappear.</p>

<h5>Mortality of Differentiated Cells.</h5><a id="article1.body1.sec4.sec2.sec8.p1" name="article1.body1.sec4.sec2.sec8.p1"></a><p>Finally, fully differentiated cells die, so that<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e022&amp;representation=PNG" class="inline-graphic"><span class="note">(18)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(19)</span></span><br>Here μ<sub>8</sub> accounts for all sources of mortality of fully differentiated cells.</p>



<h4>State dynamics using Gillespie's τ method</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>We use the Gillespie τ-method <a href="#pone.0001591-Gibson1">[47]</a>–<a href="#pone.0001591-Gillespie3">[50]</a> to convert from rates of transitions in a small interval of time Δ<em>t</em> to the dynamics of the populations of cells. We assume that the time to the next transition of the system, τ, is a random variable with exponential distribution and mean time determined by the rate of all processes. The overall rate is<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e024&amp;representation=PNG" class="inline-graphic"><span class="note">(20)</span></span><br>so that<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e025&amp;representation=PNG" class="inline-graphic"><span class="note">(21)</span></span><br>Given the current values of <em>S</em>(<em>t</em>), <em>A</em>(<em>t</em>) and <em>D</em>(<em>t</em>) we first compute the time of the next transition by comparing the right hand side of Eqn 21 with a uniformly distributed random variable.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Next, given that a transition has occurred we compute<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(22)</span></span><br>Since when time τ has elapsed one of the transitions has occurred, we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(23)</span></span><br>Thus, the <em>f<sub>i</sub></em> are the probability density function for the transitions of the system. It is also useful in numerical implementation to deal with the cumulative distribution function <em>F<sub>i</sub></em>(<em>s</em>,<em>a</em>,<em>d</em>) defined by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e028&amp;representation=PNG" class="inline-graphic"><span class="note">(24)</span></span><br></p>


<h4>Associated differential equations</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The numbers of stem, transit amplifying, and fully differentiated cells are random variables. The rates of transitions of stem cells are nonlinear functions of the states, thus it is a nontrivial matter to connect the transition rates to ordinary differential equations for the states <a href="#pone.0001591-Barbour1">[51]</a>–<a href="#pone.0001591-Mangel6">[56]</a>. It is perhaps easiest to associate differential equations with the transitions if we interpret the differential equations as the mean of conditioned kinetics <a href="#pone.0001591-Mangel6">[56]</a>. Thus, taking expectations and following the method of <a href="#pone.0001591-Mangel6">[56]</a> we associate the following differential equations with the transitions described in the previous section<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e029&amp;representation=PNG" class="inline-graphic"><span class="note">(25)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e030&amp;representation=PNG" class="inline-graphic"><span class="note">(26)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e031&amp;representation=PNG" class="inline-graphic"><span class="note">(27)</span></span><br>This set of ordinary differential equations will allow us to develop a number of novel insights about stem cells, their niche, and replacement dynamics of stem cells in their niche.</p>


<h4>Biological constraints on parameter values</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The transitions of stem, transit amplifying, and differentiated cells are characterized by a number of parameters. Biological realities will constrain some choices for parameter values. For example, we may anticipate that the probability that all stem cells in the niche either undergo apoptosis or migrate out of the niche is not too high since if it were the system could not persist. That is, if the niche empties (either through apoptosis or migration), it has essentially gone extinct. In computation, we choose parameters so that the probability of extinction of the niche is less than 20 percent.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>Similarly, in most situations, the population of transit amplifying cells is much more active than the population of stem cells (with consequence that the number of amplifying cells will thus be much larger than the number of stem cells). For purposes of computation we use the condition ρ<sub>5</sub>&gt;4ρ<sub>1</sub>.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>We require that the number of support cells is sufficient such that when there is just one stem cell in the niche, the numbers of stem cells increase. In terms of the associated ordinary differential equations, this is equivalent to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e032&amp;representation=PNG" class="inline-graphic"></span> when <em>S</em> = 1. Since Φ<sub>3</sub>(<em>A</em>,<em>D</em>,<em>B</em>) will be a decreasing function of <em>A</em>, <em>D</em> and <em>B</em>, it follows from Eqn 25 that we require<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e033&amp;representation=PNG" class="inline-graphic"><span class="note">(28)</span></span><br>In addition, we may expect there to exist links and trade-offs between the different parameters. For example, since symmetric renewal, asymmetric renewal and symmetric differentiation involve many of the same signals and processes, we may expect that the cell cycle rates λ<sub>1</sub>, λ<sub>2</sub>, and λ<sub>3</sub> are correlated. Thus, for computations, we use λ<sub>2</sub> = 2·λ<sub>1</sub> and λ<sub>3</sub> = 0.05·λ<sub>2</sub>. Similarly, cells that undergo high rates of replication are more likely to have errors in them and thus be more likely to undergo apoptosis. The simplest version of that trade-off is a linear one in which<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e034&amp;representation=PNG" class="inline-graphic"><span class="note">(29)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e035&amp;representation=PNG" class="inline-graphic"><span class="note">(30)</span></span><br>For example, the disappearance of stem cells from the niche is decomposed into terms characterizing migration to other niches (μ<sub>40</sub>) and mortality due to apoptosis. The second equation above allows us to separate the disappearance of transit amplifying cells into those potentially converting to differentiated cells and those suffering apoptosis.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>When transit amplifying cells disappear at a higher rate because of replication error, fewer of them will become fully differentiated cells. This can be captured by a trade-off between φ and λ<sub>1</sub> as in<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e036&amp;representation=PNG" class="inline-graphic"><span class="note">(31)</span></span><br>In summary, we may think of the parameter set [λ<sub>1</sub>,λ<sub>2</sub>,λ<sub>3</sub>,μ<sub>40</sub>,μ<sub>41</sub>,λ<sub>5</sub>,μ<sub>60</sub>,μ<sub>61</sub>,φ<sub>0</sub>,φ<sub>1</sub>,μ<sub>8</sub>] as the phenotype of a stem cell and its descendants, or if all of the stem cells in a niche are identical descendants of a single cell, then these parameters are the phenotype of the stem cell clone. But, of course, all the stem cells in a niche need not be conspecifics. We thus turn to the competition between different stem cell phenotypes.</p>


<h4>Coexistence, replacement, and invasion</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>We can use evolutionary replacement analysis <a href="#pone.0001591-Bonsall1">[38]</a>, <a href="#pone.0001591-Dieckmann1">[57]</a>–<a href="#pone.0001591-Mangel7">[61]</a> with the system of ordinary differential equations to explore the competition between and modification of stem cell clones and when manipulation of stem cell transition parameters will be successful. We now use <em>i</em> and <em>j</em> to index the stem cell clone with different phenotypic parameters, let γ<em><sub>ji</sub></em> denote the influence of transit amplifying and fully differentiated cells of type <em>j</em> on stem cell clone <em>i</em> so that<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e037&amp;representation=PNG" class="inline-graphic"><span class="note">(32)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e038&amp;representation=PNG" class="inline-graphic"><span class="note">(33)</span></span><br><a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e039&amp;representation=PNG" class="inline-graphic"><span class="note">(34)</span></span><br></p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>In these equations, we assume that all stem cells share the functional forms for the influence of transit amplifying and fully differentiated cells on asymmetric renewal and symmetric differentiation. After specifying the set of phenotypic parameters, we can integrate the coupled differential equations to determine the outcome of competition between any number of stem cell clones that differ in their phenotypic parameters.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>However, to illustrate the ideas and explain them in as simple a manner as possible, consider the situation in which we wish to manipulate some of the phenotypic parameters in a niche by inserting a new stem cell with different parameters into an existing clone. In that case, we need to consider the stem cells present (the ‘resident’ stem cells) and the new one (the ‘invading’ stem cell). For definiteness, let us assume that the rate of symmetric differentiation of the residents is λ<sub>3</sub> and of the invader is <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e040&amp;representation=PNG" class="inline-graphic"></span>. We first solve Eqns 25–27 for the steady state numbers of stem, transit amplifying, and fully differentiated cells with the resident phenotype which we denote by <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e041&amp;representation=PNG" class="inline-graphic"></span> to focus our attention on these steady state values as they depend upon the value of λ<sub>3</sub> (clearly, they depend upon the other phenotypic parameters too, but we only consider the manipulation of λ<sub>3</sub> here).</p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4"></a><p>To make the notation simpler let us return to Eqn 25 and rewrite it as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e042&amp;representation=PNG" class="inline-graphic"><span class="note">(35)</span></span><br>Then <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e043&amp;representation=PNG" class="inline-graphic"></span> since those values correspond to a steady state. Now imagine that we introduce a stem cell with different phenotypic parameters. We let <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e044&amp;representation=PNG" class="inline-graphic"></span> denote the number of stem cells with symmetric differentiation rate <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e045&amp;representation=PNG" class="inline-graphic"></span>. A number of outcomes are possible.</p>
<a id="article1.body1.sec4.sec6.p5" name="article1.body1.sec4.sec6.p5"></a><p>First, it is possible that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e046&amp;representation=PNG" class="inline-graphic"></span> when <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e047&amp;representation=PNG" class="inline-graphic"></span> (initially there are no transit amplifying or fully differentiated cells descended from the invading stem cells). In this case, the number of stem cells with symmetric differentiation rate <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e048&amp;representation=PNG" class="inline-graphic"></span> declines and we say that the resident stem cells resist invasion or replacement.</p>
<a id="article1.body1.sec4.sec6.p6" name="article1.body1.sec4.sec6.p6"></a><p>Second, it is possible that <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e049&amp;representation=PNG" class="inline-graphic"></span> when <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e050&amp;representation=PNG" class="inline-graphic"></span>. In this case, the number of invading stem cells will increase, and lead to transit amplifying and fully differentiated descendants. One effect will be that the resident stem cells will be disturbed from their steady state. Then their numbers may decline, in which case we say that the invading stem cell has replaced or excluded the resident stem cells. Alternatively, after the perturbation by the invading them cell (and the production of transit amplifying and fully differentiated cells from the invader) the resident stem cells may remain steady or even increase. In that case, we say that the two kinds of stem cells coexist.</p>


<h4>The focal stem cell in its niche</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>We now describe how one may characterize the behavior of a focal stem cell in its niche. To be sure, thinking of a single stem cell having behavior while others cells in the niche (and the transit amplifying and fully differentiated cells) have a fixed behavior is a simplification. Ultimately, one should view stem cells in their niche as playing a dynamic cooperative game, but we reserve that for subsequent work. For the time being, we focus on the state dependent life history of a focal stem cell, determining the optimal life history through stochastic dynamic programming <a href="#pone.0001591-Mangel3">[33]</a>–<a href="#pone.0001591-Clark1">[36]</a>.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>Like all cells, the focal stem cell must accumulate resources for division. We denote by <em>Y</em>(<em>t</em>) the resources available for cell division at time <em>t</em>, assuming that a minimum resource level <em>y<sub>d</sub></em>. We assume that the inhibition of the focal stem cell by other stem cells also affects the flow of resources to the focal stem cell and has the same functional form as used in the inhibition of cell cycling. Thus<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e051&amp;representation=PNG" class="inline-graphic"><span class="note">(36)</span></span><br>Here <em>q</em> can be scaled out by interpreting <em>t</em> as multiples of 1/<em>q</em>, so we set it equal to 1.</p>
<a id="article1.body1.sec4.sec7.p3" name="article1.body1.sec4.sec7.p3"></a><p>We assume that the organism gains fitness from these fully differentiated cells at rate <em>R<sub>F</sub></em>(<em>d</em>) when there are <em>d</em> fully differentiated cells sending signals to the focal stem cell. We assume that the decision of the focal stem cell is made at the time of the next transition of the stem, amplifying or differentiated cells. Using the methods describe above, we compute τ(<em>s</em>,<em>a</em>,<em>d</em>), which is the random variable describing the time to the next transition, and the vector Δ = (Δ<em>S</em>,Δ<em>A</em>,Δ<em>D</em>) that describes the changes in stem, transit amplifying and differentiated cells when a transition occurs.</p>
<a id="article1.body1.sec4.sec7.p4" name="article1.body1.sec4.sec7.p4"></a><p>When this transition occurs, the focal stem cell has level of resources <em>y</em>. If <em>y</em> is less than the resources available for differentiation, then the focal stem cell cannot do anything other than remain quiescent. In that case<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e052&amp;representation=PNG" class="inline-graphic"><span class="note">(37)</span></span><br>which we define as <em>V</em><sub>0</sub>, the fitness value of remaining quiescent. The right hand side of this equation represents the future accumulation of fitness and is averaged over the time of the next transition and the change in the cell population when that transition occurs.</p>
<a id="article1.body1.sec4.sec7.p5" name="article1.body1.sec4.sec7.p5"></a><p>If <em>y</em> exceeds <em>y<sub>d</sub></em>, then in the simplest case the focal stem cell may remain quiescent (with value <em>V</em><sub>0</sub> given above), renew symmetrically (with value <em>V</em><sub>1</sub> to be determined) or renew asymmetrically (with value <em>V</em><sub>2</sub> also to be determined). Allowing for full differentiation of the focal stem cell, or migration out of the niche requires certain complications which we will postpone for a later paper. After a division resources <em>y</em>−<em>y<sub>d</sub></em> remain for correcting errors in the daughter cells. We thus let Ψ(<em>y</em>−<em>y<sub>d</sub></em>) denote the probability that a daughter cell is error free (i.e. does not undergo apoptosis), given that <em>y</em>−<em>y<sub>d</sub></em> resources are used for correcting errors. For computations we use Ψ(<em>z</em>) = 1−exp(−γ<em><sub>c</sub>z</em>) where γ<em><sub>c</sub></em> is a parameter.</p>
<a id="article1.body1.sec4.sec7.p6" name="article1.body1.sec4.sec7.p6"></a><p>If the stem cell renews symmetrically, we assume that the focal stem cell remains error-free and a daughter stem cell joins the other stem cells in the niche. Assuming segregation of errors upon division <a href="#pone.0001591-Cairns1">[29]</a>, <a href="#pone.0001591-Potten3">[31]</a>, all resources remaining after division are used to ensure that the daughter stem cell is error free. We thus have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e053&amp;representation=PNG" class="inline-graphic"><span class="note">(38)</span></span><br>On the other hand, if the stem cell renews asymmetrically, a new transit amplifying cell is produced and the resources remaining after division are used to correct errors in the new transit amplifying cell. In this case<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e054&amp;representation=PNG" class="inline-graphic"><span class="note">(39)</span></span><br>Thus, when <em>y</em> exceeds <em>y<sub>d</sub></em> we have<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001591.e055&amp;representation=PNG" class="inline-graphic"><span class="note">(40)</span></span><br>Note that there is no explicit time in any of these equations. Thus, they cannot be solved by the usual method of backward iteration <a href="#pone.0001591-Mangel3">[33]</a>–<a href="#pone.0001591-Clark1">[36]</a>. Rather, we determine <em>F</em>(<em>y</em>,<em>s</em>,<em>a</em>,<em>d</em>) by value iteration (see <a href="#pone.0001591-Puterman1">[62]</a> for a general discussion and <a href="#pone.0001591-Venner1">[63]</a> for an application in behavioral biology). When the value iteration has stabilized, we have found in addition to fitness, the optimal decisions <em>i</em><sup>*</sup>(<em>y</em>,<em>s</em>,<em>a</em>,<em>d</em>) of the focal stem cell when its accumulated resources for division are <em>y</em>, there are <em>s</em> other stem cells in the niche and the number of transit amplifying and differentiated cells sending signals to the niche are <em>a</em> and <em>d</em> respectively. We then use the Markov simulation described previously to create the environment of the focal stem cell and follow its subsequent behavior (quiescence, symmetric renewal, or asymmetrical renewal) as the number of stem cells in the niche and the number of amplifying and differentiated cells change through the stochastic processes described previously. In this way, we are able to construct the times between transitions of the focal stem cell, and thus the analogue of the distribution of Till et al. <a href="#pone.0001591-Till1">[24]</a>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Richard Gardner, Chris Graham, and Arthur Lander for very useful discussions and Arthur Lander for comments on a previous version of the manuscript. Kate Richerson and Laith Yakob provided excellent and timely help in the preparation of the final version.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Wrote the paper: MM MB. Other: Conceived and designed the models: MM. Analyzed the models: MM. First proposed the problem: MM. Worked extensively on the modeling and the paper: MB MM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001591-Robert1" id="pone.0001591-Robert1"></a>Robert JS (2004) Model systems in stem cell biology. BioEssays  26: 1005–1012.  <ul class="find" data-citedArticleID="1055184" data-doi="10.1002/bies.20100"><li><a href="http://dx.doi.org/10.1002/bies.20100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Model+systems+in+stem+cell+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Model+systems+in+stem+cell+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001591-Fuchs1" id="pone.0001591-Fuchs1"></a>Fuchs E, Tumbar T, Guasch G (2004) Socializing with their neighbors: stem cells and their niche. Cell  116: 769–778.  <ul class="find" data-citedArticleID="1055108" data-doi="10.1016/s0092-8674(04)00255-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(04)00255-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Socializing+with+their+neighbors%3A+stem+cells+and+their+niche." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Socializing+with+their+neighbors%3A+stem+cells+and+their+niche.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001591-Anversa1" id="pone.0001591-Anversa1"></a>Anversa P, Leri A, Kajstura J (2006)  Stem cells and heart disease. In: Lanza R, Gearhart J, Hogan B, Melton D, Pedersen R, et al, editors. Essentials of stem cell biology. Amsterdam: Elsevier. pp. 425–430.  <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0001591-Lanza1" id="pone.0001591-Lanza1"></a>Lanza R, Gearhart J, Hogan B, Melton D, Pedersen R, et al.  (2006) Essentials of stem cell biology. Amsterdam: Elsevier.   <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0001591-Dobzhansky1" id="pone.0001591-Dobzhansky1"></a>Dobzhansky T (1964) Biology, molecular and oganismic. Integ Compar Biol  4: 443–452.  <ul class="find" data-citedArticleID="1055102"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Biology%2C+molecular+and+oganismic.&amp;auth=&amp;atitle=Biology%2C+molecular+and+oganismic." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biology%2C+molecular+and+oganismic." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biology%2C+molecular+and+oganismic.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001591-Anderson1" id="pone.0001591-Anderson1"></a>Anderson DJ (2001) Stem cells and pattern formation in the nervous system: The possible versus the actual. Neuron  30: 19–35.  <ul class="find" data-citedArticleID="1055082" data-doi="10.1016/s0896-6273(01)00260-4"><li><a href="http://dx.doi.org/10.1016/s0896-6273(01)00260-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stem+cells+and+pattern+formation+in+the+nervous+system%3A+The+possible+versus+the+actual." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stem+cells+and+pattern+formation+in+the+nervous+system%3A+The+possible+versus+the+actual.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001591-Raff1" id="pone.0001591-Raff1"></a>Raff M (2003) Adult stem cell plasticity: fact or artifact? Annu Rev Cell and Devel Biol  19: 1–22.  <ul class="find" data-citedArticleID="1055182"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Adult+stem+cell+plasticity%3A+fact+or+artifact%3F&amp;auth=&amp;atitle=Adult+stem+cell+plasticity%3A+fact+or+artifact%3F" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adult+stem+cell+plasticity%3A+fact+or+artifact%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adult+stem+cell+plasticity%3A+fact+or+artifact%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001591-Mangel1" id="pone.0001591-Mangel1"></a>Mangel M, Bonsall MB (2007) The evolutionary ecology of stem cells and their niches-the time is now. Oikos  116: 1779–1781.  <ul class="find" data-citedArticleID="1055140" data-doi="10.1111/j.0030-1299.2007.16248.x"><li><a href="http://dx.doi.org/10.1111/j.0030-1299.2007.16248.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+ecology+of+stem+cells+and+their+niches-the+time+is+now." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+ecology+of+stem+cells+and+their+niches-the+time+is+now.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001591-Viswanathan1" id="pone.0001591-Viswanathan1"></a>Viswanathan S, Davey R, Cheng D, Raghu RC, Lauffenburger DA, et al.  (2005) Clonal evolution of stem and differentiated cells can be predicted by integrating cell-intrinsic and extrinsic parameters. Biotechnol Appl Bioc  42: 119–131.  <ul class="find" data-citedArticleID="1055196" data-doi="10.1042/ba20040207"><li><a href="http://dx.doi.org/10.1042/ba20040207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Clonal+evolution+of+stem+and+differentiated+cells+can+be+predicted+by+integrating+cell-intrinsic+and+extrinsic+parameters." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Clonal+evolution+of+stem+and+differentiated+cells+can+be+predicted+by+integrating+cell-intrinsic+and+extrinsic+parameters.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001591-Mayr1" id="pone.0001591-Mayr1"></a>Mayr E (1982) The growth of biological thought: Diversity, evolution, and inheritance. Cambridge: Harvard University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0001591-Watt1" id="pone.0001591-Watt1"></a>Watt FM, Hogan BLM (2000) Out of Eden: Stem cells and their niches. Science  287: 1427–1430.  <ul class="find" data-citedArticleID="1055200" data-doi="10.1126/science.287.5457.1427"><li><a href="http://dx.doi.org/10.1126/science.287.5457.1427" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Out+of+Eden%3A+Stem+cells+and+their+niches." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Out+of+Eden%3A+Stem+cells+and+their+niches.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001591-Vogel1" id="pone.0001591-Vogel1"></a>Vogel H, Niewisch H, Matioli G (1969) Stochastic development of stem cells. J Theor Biol  22: 249–270.  <ul class="find" data-citedArticleID="1055198" data-doi="10.1016/0022-5193(69)90004-6"><li><a href="http://dx.doi.org/10.1016/0022-5193(69)90004-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+development+of+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+development+of+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001591-Neff1" id="pone.0001591-Neff1"></a>Neff T, Beard BC, Kiem H-P (2006) Survival of the fittest: in vivo selection and stem cell gene therapy. Blood  107: 1751–1760.  <ul class="find" data-citedArticleID="1055164" data-doi="10.1182/blood-2005-06-2335"><li><a href="http://dx.doi.org/10.1182/blood-2005-06-2335" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Survival+of+the+fittest%3A+in+vivo+selection+and+stem+cell+gene+therapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Survival+of+the+fittest%3A+in+vivo+selection+and+stem+cell+gene+therapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001591-Potten1" id="pone.0001591-Potten1"></a>Potten CS, Loeffler M (1990) Stem cells: attributes, cycles, spirals, pitfalls and uncertainties lessons for and from the crypt. Development  110: 1001–1020.  <ul class="find" data-citedArticleID="1055172"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Stem+cells%3A+attributes%2C+cycles%2C+spirals%2C+pitfalls+and+uncertainties+lessons+for+and+from+the+crypt.&amp;auth=&amp;atitle=Stem+cells%3A+attributes%2C+cycles%2C+spirals%2C+pitfalls+and+uncertainties+lessons+for+and+from+the+crypt." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stem+cells%3A+attributes%2C+cycles%2C+spirals%2C+pitfalls+and+uncertainties+lessons+for+and+from+the+crypt." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stem+cells%3A+attributes%2C+cycles%2C+spirals%2C+pitfalls+and+uncertainties+lessons+for+and+from+the+crypt.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001591-Potten2" id="pone.0001591-Potten2"></a>Potten CS, Booth C (2002) Keratinocyte stem cells: A commentary. J Invest Dermatol  119: 88.  <ul class="find" data-citedArticleID="1055174" data-doi="10.1046/j.1523-1747.2002.00020.x"><li><a href="http://dx.doi.org/10.1046/j.1523-1747.2002.00020.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Keratinocyte+stem+cells%3A+A+commentary." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Keratinocyte+stem+cells%3A+A+commentary.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001591-Alon1" id="pone.0001591-Alon1"></a>Alon U (2007) An Introduction to Systems Biology. Design Principles of Biological Circuits. Boca Raton: Chapman and Hall/CRC.   <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0001591-May1" id="pone.0001591-May1"></a>May RM (2004) Uses and abuses of mathematics in biology. Science  303: 790–793.  <ul class="find" data-citedArticleID="1055154" data-doi="10.1126/science.1094442"><li><a href="http://dx.doi.org/10.1126/science.1094442" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Uses+and+abuses+of+mathematics+in+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Uses+and+abuses+of+mathematics+in+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001591-May2" id="pone.0001591-May2"></a>May RM, McLean A (2007) Theoretical ecology. Principles and applications. Third Edition. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0001591-Fisher1" id="pone.0001591-Fisher1"></a>Fisher RA (1930) The genetical theory of natural selection. Oxford: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0001591-Frank1" id="pone.0001591-Frank1"></a>Frank SA (2007) Dynamics of cancer: Incidence, inheritance, and evolution. Princeton: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0001591-Grant1" id="pone.0001591-Grant1"></a>Grant PR, Grant BR (2007) How and why species multiply: The radiation of Darwin's finches. Princeton: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0001591-Thomas1" id="pone.0001591-Thomas1"></a>Thomas ED, Lochte HL, Lu WC, Ferrebee JW (1957) Intravenous infusion of bone marrow in patients receiving radiation and chemotherapy. New Engl J Med  257: 491–496.  <ul class="find" data-citedArticleID="1055188" data-doi="10.1056/nejm195709122571102"><li><a href="http://dx.doi.org/10.1056/nejm195709122571102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intravenous+infusion+of+bone+marrow+in+patients+receiving+radiation+and+chemotherapy." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intravenous+infusion+of+bone+marrow+in+patients+receiving+radiation+and+chemotherapy.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001591-Appelbaum1" id="pone.0001591-Appelbaum1"></a>Appelbaum FR (2007) Hematopoietic-cell transplantation at 50. New Engl J Med  357: 1472–1475.  <ul class="find" data-citedArticleID="1055086" data-doi="10.1056/nejmp078166"><li><a href="http://dx.doi.org/10.1056/nejmp078166" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hematopoietic-cell+transplantation+at+50." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hematopoietic-cell+transplantation+at+50.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001591-Till1" id="pone.0001591-Till1"></a>Till JE, McCulloch EA, Siminovitch L (1964) A stochastic model of stem cell proliferation based on the growth of spleen colony-forming cells. Proc Natl Acad Sci U S A  51: 29–36.  <ul class="find" data-citedArticleID="1055190" data-doi="10.1073/pnas.51.1.29"><li><a href="http://dx.doi.org/10.1073/pnas.51.1.29" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+stochastic+model+of+stem+cell+proliferation+based+on+the+growth+of+spleen+colony-forming+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+stochastic+model+of+stem+cell+proliferation+based+on+the+growth+of+spleen+colony-forming+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001591-Mangel2" id="pone.0001591-Mangel2"></a>Mangel M (2006) The theoretical biologist's toolbox: Quantitative methods for ecology and evolutionary biology. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0001591-Murray1" id="pone.0001591-Murray1"></a>Murray JD (2003) Mathematical Biology, Volume 2. Heidelberg: Springer.   <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0001591-Shostak1" id="pone.0001591-Shostak1"></a>Shostak S (2006) (Re)defining stem cells. BioEssays  28: 301–308.  <ul class="find" data-citedArticleID="1055186" data-doi="10.1002/bies.20376"><li><a href="http://dx.doi.org/10.1002/bies.20376" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%28Re%29defining+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%28Re%29defining+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001591-Kirouac1" id="pone.0001591-Kirouac1"></a>Kirouac DC, Zandstra PW (2006) Understanding cellular networks to improve hematopoietic stem cell expansion cultures. Curr Opin Biotech  17: 538–547.  <ul class="find" data-citedArticleID="1055124" data-doi="10.1016/j.copbio.2006.07.003"><li><a href="http://dx.doi.org/10.1016/j.copbio.2006.07.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Understanding+cellular+networks+to+improve+hematopoietic+stem+cell+expansion+cultures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Understanding+cellular+networks+to+improve+hematopoietic+stem+cell+expansion+cultures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001591-Cairns1" id="pone.0001591-Cairns1"></a>Cairns J (1975) Mutation selection and the natural history of cancer. Nature  255: 197–200.  <ul class="find" data-citedArticleID="1055094" data-doi="10.1038/255197a0"><li><a href="http://dx.doi.org/10.1038/255197a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+selection+and+the+natural+history+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+selection+and+the+natural+history+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001591-LeGrand1" id="pone.0001591-LeGrand1"></a>LeGrand EK (1997) An adaptationist view of apoptosis. Q Rev Biol  72: 135–147.  <ul class="find" data-citedArticleID="1055134" data-doi="10.1086/419763"><li><a href="http://dx.doi.org/10.1086/419763" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+adaptationist+view+of+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+adaptationist+view+of+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001591-Potten3" id="pone.0001591-Potten3"></a>Potten CS, Owen G, Booth D (2002) Intestinal stem cells protect their genome by segregation of template DNA strands. J Cell Sci  115: 2381–2388.  <ul class="find" data-citedArticleID="1055176"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Intestinal+stem+cells+protect+their+genome+by+segregation+of+template+DNA+strands.&amp;auth=&amp;atitle=Intestinal+stem+cells+protect+their+genome+by+segregation+of+template+DNA+strands." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intestinal+stem+cells+protect+their+genome+by+segregation+of+template+DNA+strands." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intestinal+stem+cells+protect+their+genome+by+segregation+of+template+DNA+strands.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001591-Zandstra1" id="pone.0001591-Zandstra1"></a>Zandstra PW, Nagy A (2001) Stem cell bioengineering. Annu Rev of Biomed Eng  3: 275–305.  <ul class="find" data-citedArticleID="1055204" data-doi="10.1146/annurev.bioeng.3.1.275"><li><a href="http://dx.doi.org/10.1146/annurev.bioeng.3.1.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stem+cell+bioengineering." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stem+cell+bioengineering.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001591-Mangel3" id="pone.0001591-Mangel3"></a>Mangel M, Clark CW (1988) Dynamic modeling in behavioral ecology. Princeton: Princeton University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0001591-Mangel4" id="pone.0001591-Mangel4"></a>Mangel M, Ludwig D (1992) Definition and evaluation of behavioral and developmental programs. Annu Rev Ecol Syst  23: 507–536.  <ul class="find" data-citedArticleID="1055146" data-doi="10.1146/annurev.es.23.110192.002451"><li><a href="http://dx.doi.org/10.1146/annurev.es.23.110192.002451" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Definition+and+evaluation+of+behavioral+and+developmental+programs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Definition+and+evaluation+of+behavioral+and+developmental+programs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001591-Houston1" id="pone.0001591-Houston1"></a>Houston AI, McNamara JM (1999) Models of adaptive behavior. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">36.
              </span><a name="pone.0001591-Clark1" id="pone.0001591-Clark1"></a>Clark CW, Mangel M (2000) Dynamic state variable models in ecology: Methods and applications. New York: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0001591-Kingsland1" id="pone.0001591-Kingsland1"></a>Kingsland SE (1985) Modeling nature: Episodes in the history of population ecology. Chicago: University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">38.
              </span><a name="pone.0001591-Bonsall1" id="pone.0001591-Bonsall1"></a>Bonsall MB, Mangel M (2004) Life-history trade-offs and ecological dynamics in the evolution of longevity. Proc R Soc Lond B  271: 1143–1150.  <ul class="find" data-citedArticleID="1055092" data-doi="10.1098/rspb.2004.2722"><li><a href="http://dx.doi.org/10.1098/rspb.2004.2722" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Life-history+trade-offs+and+ecological+dynamics+in+the+evolution+of+longevity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Life-history+trade-offs+and+ecological+dynamics+in+the+evolution+of+longevity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001591-Nowak1" id="pone.0001591-Nowak1"></a>Nowak MA (2006) Evolutionary dynamics. Cambridge: Harvard University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0001591-Wodarz1" id="pone.0001591-Wodarz1"></a>Wodarz D, Komarova N (2005) Computational biology of cancer: Lecture notes and mathematical modeling. Singapore: World Scientific.   <ul class="find-nolinks"></ul></li><li><span class="label">41.
              </span><a name="pone.0001591-Potten4" id="pone.0001591-Potten4"></a>Potten CS, Wilson JW (2006)  The development of epithelial stem cell concepts. In: Lanza R, Gearhart J, Hogan B, Melton D, Pedersen R, et al, editors. Essentials of stem cell biology. Amsterdam: Elsevier. pp. 11–21.  <ul class="find-nolinks"></ul></li><li><span class="label">42.
              </span><a name="pone.0001591-Painter1" id="pone.0001591-Painter1"></a>Painter PR, Marr AG (1968) Mathematics of microbial populations. Ann Rev Microbiol  22: 519–548.  <ul class="find" data-citedArticleID="1055168" data-doi="10.1146/annurev.mi.22.100168.002511"><li><a href="http://dx.doi.org/10.1146/annurev.mi.22.100168.002511" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mathematics+of+microbial+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mathematics+of+microbial+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001591-Madhavi1" id="pone.0001591-Madhavi1"></a>Madhavi A, Davey RE, Bhola P, Yin T, Zandstra PW (2007) Sensitivity analysis of intracelluar signaling pathway kinetics predicts targets of stem cell fate control. PLoS Comput Biol  3: 1–11.  <ul class="find" data-citedArticleID="1055138" data-doi="10.1371/journal.pcbi.0030130"><li><a href="http://dx.doi.org/10.1371/journal.pcbi.0030130" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sensitivity+analysis+of+intracelluar+signaling+pathway+kinetics+predicts+targets+of+stem+cell+fate+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sensitivity+analysis+of+intracelluar+signaling+pathway+kinetics+predicts+targets+of+stem+cell+fate+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001591-Davey1" id="pone.0001591-Davey1"></a>Davey RE, Onishi K, Mahdavi A, Zandstra PW (2007) LIF-mediated control of embryonic stem cell self-renewal emerges due to an autoregulatory loop. FASEB J  21: 2020–2032.  <ul class="find" data-citedArticleID="1055098" data-doi="10.1096/fj.06-7852com"><li><a href="http://dx.doi.org/10.1096/fj.06-7852com" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LIF-mediated+control+of+embryonic+stem+cell+self-renewal+emerges+due+to+an+autoregulatory+loop." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LIF-mediated+control+of+embryonic+stem+cell+self-renewal+emerges+due+to+an+autoregulatory+loop.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001591-MacDonald1" id="pone.0001591-MacDonald1"></a>MacDonald N (1989) Biological delay systems: Linear stability theory. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">46.
              </span><a name="pone.0001591-Mangel5" id="pone.0001591-Mangel5"></a>Mangel M, Bonsall MB (2004) The shape of things to come: using models with physiological structure to predict mortality trajectories. Theor Popul Biol  65: 353–359.  <ul class="find" data-citedArticleID="1055148" data-doi="10.1016/j.tpb.2003.07.005"><li><a href="http://dx.doi.org/10.1016/j.tpb.2003.07.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+shape+of+things+to+come%3A+using+models+with+physiological+structure+to+predict+mortality+trajectories." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+shape+of+things+to+come%3A+using+models+with+physiological+structure+to+predict+mortality+trajectories.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001591-Gibson1" id="pone.0001591-Gibson1"></a>Gibson MA, Bruck J (2000) Efficient exact stochastic simulation of chemical systems with many species and many channels. J Chem Phys  104: 1876.  <ul class="find" data-citedArticleID="1055110" data-doi="10.1021/jp993732q"><li><a href="http://dx.doi.org/10.1021/jp993732q" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+exact+stochastic+simulation+of+chemical+systems+with+many+species+and+many+channels." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+exact+stochastic+simulation+of+chemical+systems+with+many+species+and+many+channels.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001591-Gillespie1" id="pone.0001591-Gillespie1"></a>Gillespie DT (2000) The chemical Langevin equation. J Chem Phys  113: 297.  <ul class="find" data-citedArticleID="1055112" data-doi="10.1063/1.481811"><li><a href="http://dx.doi.org/10.1063/1.481811" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chemical+Langevin+equation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chemical+Langevin+equation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001591-Gillespie2" id="pone.0001591-Gillespie2"></a>Gillespie DT (2001) Approximate accelerated stochastic simulation of chemically reacting systems. J Chem Phys  115: 1716.  <ul class="find" data-citedArticleID="1055114" data-doi="10.1063/1.1378322"><li><a href="http://dx.doi.org/10.1063/1.1378322" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Approximate+accelerated+stochastic+simulation+of+chemically+reacting+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Approximate+accelerated+stochastic+simulation+of+chemically+reacting+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001591-Gillespie3" id="pone.0001591-Gillespie3"></a>Gillespie DT, Petzold LR (2003) Improved leap-size selection for accelerated stochastic simulation. J Chem Phys  119: 8229.  <ul class="find" data-citedArticleID="1055116" data-doi="10.1063/1.1613254"><li><a href="http://dx.doi.org/10.1063/1.1613254" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+leap-size+selection+for+accelerated+stochastic+simulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+leap-size+selection+for+accelerated+stochastic+simulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001591-Barbour1" id="pone.0001591-Barbour1"></a>Barbour AD (1974) On a functional central limit theorem for Markov population processes. Adv Appl Probab  6: 21–39.  <ul class="find" data-citedArticleID="1055088" data-doi="10.2307/1426205"><li><a href="http://dx.doi.org/10.2307/1426205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+a+functional+central+limit+theorem+for+Markov+population+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+a+functional+central+limit+theorem+for+Markov+population+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001591-Barbour2" id="pone.0001591-Barbour2"></a>Barbour AD (1980) Equilibrium distributions for Markov population processes. Adv App Probab  12: 591–614.  <ul class="find" data-citedArticleID="1055090" data-doi="10.2307/1426422"><li><a href="http://dx.doi.org/10.2307/1426422" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Equilibrium+distributions+for+Markov+population+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Equilibrium+distributions+for+Markov+population+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001591-Kurtz1" id="pone.0001591-Kurtz1"></a>Kurtz TG (1970) Solutions of ordinary differential equations as limits of pure jump Markov processes. J Appl Probab  7: 49–58.  <ul class="find" data-citedArticleID="1055126" data-doi="10.2307/3212147"><li><a href="http://dx.doi.org/10.2307/3212147" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Solutions+of+ordinary+differential+equations+as+limits+of+pure+jump+Markov+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Solutions+of+ordinary+differential+equations+as+limits+of+pure+jump+Markov+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001591-Kurtz2" id="pone.0001591-Kurtz2"></a>Kurtz TG (1971) Limit theorems for sequences of jump Markov processes approximating ordinary differential processes. J Appl Probab  8: 344–356.  <ul class="find" data-citedArticleID="1055128" data-doi="10.2307/3211904"><li><a href="http://dx.doi.org/10.2307/3211904" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Limit+theorems+for+sequences+of+jump+Markov+processes+approximating+ordinary+differential+processes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Limit+theorems+for+sequences+of+jump+Markov+processes+approximating+ordinary+differential+processes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001591-Kurtz3" id="pone.0001591-Kurtz3"></a>Kurtz TG (1972) The relationship between stochastic and deterministic models for chemical reactions. J Chem Phys  57: 2976–2978.  <ul class="find" data-citedArticleID="1055130" data-doi="10.1063/1.1678692"><li><a href="http://dx.doi.org/10.1063/1.1678692" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relationship+between+stochastic+and+deterministic+models+for+chemical+reactions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relationship+between+stochastic+and+deterministic+models+for+chemical+reactions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001591-Mangel6" id="pone.0001591-Mangel6"></a>Mangel M (1994) Barrier transitions driven by fluctuations, with applications to ecology and evolution. Theor Popul Biol  45: 16–40.  <ul class="find" data-citedArticleID="1055150" data-doi="10.1006/tpbi.1994.1002"><li><a href="http://dx.doi.org/10.1006/tpbi.1994.1002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Barrier+transitions+driven+by+fluctuations%2C+with+applications+to+ecology+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Barrier+transitions+driven+by+fluctuations%2C+with+applications+to+ecology+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001591-Dieckmann1" id="pone.0001591-Dieckmann1"></a>Dieckmann U (1997) Can adaptive dynamics invade? Trends Ecol Evol  12: 128–131.  <ul class="find" data-citedArticleID="1055100" data-doi="10.1016/s0169-5347(97)01004-5"><li><a href="http://dx.doi.org/10.1016/s0169-5347(97)01004-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+adaptive+dynamics+invade%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+adaptive+dynamics+invade%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001591-Metz1" id="pone.0001591-Metz1"></a>Metz JAJ, Nisbet RM, Geritz SAH (1992) How should we define fitness for general ecological scenarios? Trends Ecol Evol  7: 198–202.  <ul class="find" data-citedArticleID="1055160" data-doi="10.1016/0169-5347(92)90073-k"><li><a href="http://dx.doi.org/10.1016/0169-5347(92)90073-k" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+should+we+define+fitness+for+general+ecological+scenarios%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+should+we+define+fitness+for+general+ecological+scenarios%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001591-Vincent1" id="pone.0001591-Vincent1"></a>Vincent TL, Brown JS (2005) Evolutionary game theory, natural selection and Darwinian dynamics. Cambridge: Cambridge University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">60.
              </span><a name="pone.0001591-Pielou1" id="pone.0001591-Pielou1"></a>Pielou EC (1977) Mathematical ecology. London: John Wiley and Sons.   <ul class="find-nolinks"></ul></li><li><span class="label">61.
              </span><a name="pone.0001591-Mangel7" id="pone.0001591-Mangel7"></a>Mangel M, Kindsvater HK, Bonsall MB (2007) Evolutionary analysis of life span, competition, and adaptive radiation, motivated by the Pacific rockfishes (<em>Sebastes</em>). Evolution  61: 1208–1224.  <ul class="find" data-citedArticleID="1055152" data-doi="10.1111/j.1558-5646.2007.00094.x"><li><a href="http://dx.doi.org/10.1111/j.1558-5646.2007.00094.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+analysis+of+life+span%2C+competition%2C+and+adaptive+radiation%2C+motivated+by+the+Pacific+rockfishes+%28Sebastes%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+analysis+of+life+span%2C+competition%2C+and+adaptive+radiation%2C+motivated+by+the+Pacific+rockfishes+%28Sebastes%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001591-Puterman1" id="pone.0001591-Puterman1"></a>Puterman ML (1994) Markov decision processes: Discrete dynamic stochastic programming. New York: Wiley Interscience.   <ul class="find-nolinks"></ul></li><li><span class="label">63.
              </span><a name="pone.0001591-Venner1" id="pone.0001591-Venner1"></a>Venner S, Chads I, Bel-Venner M-C, Pasquet A, Charpillet F, et al.  (2006) Dynamic optimization over infinite-time horizon: Web-building strategy in an orb-weaving spider as a case study. J Theor Biol  241: 725–733.  <ul class="find" data-citedArticleID="1055192" data-doi="10.1016/j.jtbi.2006.01.008"><li><a href="http://dx.doi.org/10.1016/j.jtbi.2006.01.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dynamic+optimization+over+infinite-time+horizon%3A+Web-building+strategy+in+an+orb-weaving+spider+as+a+case+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dynamic+optimization+over+infinite-time+horizon%3A+Web-building+strategy+in+an+orb-weaving+spider+as+a+case+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.XML" value="114295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.PDF" value="426326"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g001.PNG_L" value="100699"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g001.PNG_M" value="62191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g001.PNG_S" value="7808"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g001.TIF" value="139962"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g001.PNG_I" value="22270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.t001.PNG_L" value="177996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.t001.PNG_M" value="166851"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.t001.PNG_S" value="21140"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.t001.TIF" value="877886"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.t001.PNG_I" value="104896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g002.PNG_L" value="56325"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g002.PNG_M" value="47539"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g002.PNG_S" value="7018"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g002.TIF" value="150232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g002.PNG_I" value="17669"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e001.PNG" value="8582"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e001.TIF" value="24380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e002.PNG" value="7161"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e002.TIF" value="23108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e003.PNG" value="7733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e003.TIF" value="23632"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e004.PNG" value="8482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e004.TIF" value="24272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e005.PNG" value="10461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e005.TIF" value="26256"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g003.PNG_L" value="512222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g003.PNG_M" value="142238"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g003.PNG_S" value="15298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g003.TIF" value="865448"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g003.PNG_I" value="113946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g004.PNG_L" value="344545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g004.PNG_M" value="95466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g004.PNG_S" value="12904"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g004.TIF" value="728050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g004.PNG_I" value="78096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e006.PNG" value="5588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e006.TIF" value="21508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e007.PNG" value="5051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e007.TIF" value="20992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g005.PNG_L" value="373595"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g005.PNG_M" value="89735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g005.PNG_S" value="12798"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g005.TIF" value="603340"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g005.PNG_I" value="80506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g006.PNG_L" value="21250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g006.PNG_M" value="16557"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g006.PNG_S" value="4887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g006.TIF" value="77958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.g006.PNG_I" value="6228"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e008.PNG" value="12856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e008.TIF" value="29348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e009.PNG" value="8044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e009.TIF" value="24040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e010.PNG" value="12810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e010.TIF" value="29312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e011.PNG" value="9381"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e011.TIF" value="25408"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e012.PNG" value="12842"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e012.TIF" value="29300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e013.PNG" value="9458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e013.TIF" value="25456"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e014.PNG" value="12819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e014.TIF" value="29344"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e015.PNG" value="7230"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e015.TIF" value="23172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e016.PNG" value="12837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e016.TIF" value="29308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e017.PNG" value="7254"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e017.TIF" value="23192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e018.PNG" value="12893"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e018.TIF" value="29356"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e019.PNG" value="8292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e019.TIF" value="24208"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e020.PNG" value="12688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e020.TIF" value="29252"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e021.PNG" value="7688"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e021.TIF" value="23600"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e022.PNG" value="12914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e022.TIF" value="29360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e023.PNG" value="7412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e023.TIF" value="23312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e024.PNG" value="9741"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e024.TIF" value="25508"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e025.PNG" value="8850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e025.TIF" value="24880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e026.PNG" value="12791"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e026.TIF" value="29492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e027.PNG" value="10012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e027.TIF" value="25856"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e028.PNG" value="9847"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e028.TIF" value="25612"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e029.PNG" value="10491"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e029.TIF" value="26380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e030.PNG" value="13380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e030.TIF" value="29504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e031.PNG" value="8510"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e031.TIF" value="24288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e032.PNG" value="6437"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e032.TIF" value="22388"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e033.PNG" value="9804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e033.TIF" value="25620"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e034.PNG" value="6844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e034.TIF" value="22840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e035.PNG" value="6914"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e035.TIF" value="22872"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e036.PNG" value="6672"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e036.TIF" value="22624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e037.PNG" value="15333"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e037.TIF" value="31288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e038.PNG" value="22004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e038.TIF" value="37704"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e039.PNG" value="9288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e039.TIF" value="25032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e040.PNG" value="5067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e040.TIF" value="21004"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e041.PNG" value="8562"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e041.TIF" value="24556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e042.PNG" value="9100"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e042.TIF" value="24936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e043.PNG" value="9541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e043.TIF" value="25496"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e044.PNG" value="6097"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e044.TIF" value="22048"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e045.PNG" value="5087"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e045.TIF" value="20988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e046.PNG" value="10412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e046.TIF" value="26492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e047.PNG" value="6403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e047.TIF" value="22384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e048.PNG" value="5085"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e048.TIF" value="21000"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e049.PNG" value="7334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e049.TIF" value="26352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e050.PNG" value="6374"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e050.TIF" value="22352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e051.PNG" value="7683"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e051.TIF" value="23584"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e052.PNG" value="11989"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e052.TIF" value="28556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e053.PNG" value="16054"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e053.TIF" value="32720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e054.PNG" value="16745"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e054.TIF" value="33680"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e055.PNG" value="10348"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001591.e055.TIF" value="26680"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001591&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001591" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001591&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001591&volume=&issue=&title=Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability&author_name=Marc%20Mangel%2C%20Michael%20B.%20Bonsall&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001591" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591&amp;t=Phenotypic%20Evolutionary%20Models%20in%20Stem%20Cell%20Biology%3A%20Replacement%2C%20Quiescence%2C%20and%20Variability" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591&title=Phenotypic%20Evolutionary%20Models%20in%20Stem%20Cell%20Biology%3A%20Replacement%2C%20Quiescence%2C%20and%20Variability&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591&amp;title=Phenotypic%20Evolutionary%20Models%20in%20Stem%20Cell%20Biology%3A%20Replacement%2C%20Quiescence%2C%20and%20Variability" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001591&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Phenotypic Evolutionary Models in Stem Cell Biology: Replacement, Quiescence, and Variability';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001591';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Phenotypic%20Evolutionary%20Models%20in%20Stem%20Cell%20Biology%3A%20Replacement%2C%20Quiescence%2C%20and%20Variability http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001591" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001591" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">










          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Apoptosis%22" title="Search for articles in the subject area:'Apoptosis'"><div class="flagText">Apoptosis</div></a>
              <div data-categoryid="17189" data-articleid="26008"
                   data-categoryname="Apoptosis"
                   class="flagImage" title="Flag 'Apoptosis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+cycle+and+cell+division%22" title="Search for articles in the subject area:'Cell cycle and cell division'"><div class="flagText">Cell cycle and cell division</div></a>
              <div data-categoryid="17353" data-articleid="26008"
                   data-categoryname="Cell cycle and cell division"
                   class="flagImage" title="Flag 'Cell cycle and cell division' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+differentiation%22" title="Search for articles in the subject area:'Cell differentiation'"><div class="flagText">Cell differentiation</div></a>
              <div data-categoryid="17923" data-articleid="26008"
                   data-categoryname="Cell differentiation"
                   class="flagImage" title="Flag 'Cell differentiation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Cell+migration%22" title="Search for articles in the subject area:'Cell migration'"><div class="flagText">Cell migration</div></a>
              <div data-categoryid="17279" data-articleid="26008"
                   data-categoryname="Cell migration"
                   class="flagImage" title="Flag 'Cell migration' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Differential+equations%22" title="Search for articles in the subject area:'Differential equations'"><div class="flagText">Differential equations</div></a>
              <div data-categoryid="35491" data-articleid="26008"
                   data-categoryname="Differential equations"
                   class="flagImage" title="Flag 'Differential equations' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ecological+niches%22" title="Search for articles in the subject area:'Ecological niches'"><div class="flagText">Ecological niches</div></a>
              <div data-categoryid="48505" data-articleid="26008"
                   data-categoryname="Ecological niches"
                   class="flagImage" title="Flag 'Ecological niches' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Stem+cell+niche%22" title="Search for articles in the subject area:'Stem cell niche'"><div class="flagText">Stem cell niche</div></a>
              <div data-categoryid="22409" data-articleid="26008"
                   data-categoryname="Stem cell niche"
                   class="flagImage" title="Flag 'Stem cell niche' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Stem+cells%22" title="Search for articles in the subject area:'Stem cells'"><div class="flagText">Stem cells</div></a>
              <div data-categoryid="17585" data-articleid="26008"
                   data-categoryname="Stem cells"
                   class="flagImage" title="Flag 'Stem cells' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1045'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4341'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=2393&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
