

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000894"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000894" />

    <meta name="citation_title" content="Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits"/>
    <meta itemprop="name" content="Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits"/>

      <meta name="citation_author" content="Christopher J. Martyniuk"/>
            <meta name="citation_author_institution" content="Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Stéphane Aris-Brosou"/>
            <meta name="citation_author_institution" content="Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada"/>
            <meta name="citation_author_institution" content="Department of Mathematics and Statistics, University of Ottawa, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Guy Drouin"/>
            <meta name="citation_author_institution" content="Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Joel Cahn"/>
            <meta name="citation_author_institution" content="Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada"/>
      <meta name="citation_author" content="Vance L. Trudeau"/>
            <meta name="citation_author_institution" content="Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada"/>

    <meta name="citation_date" content="2007/9/19"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000894.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e894"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Role of gene duplication in evolution.; citation_author=T Ohta; citation_journal_title=Genome; citation_volume=31; citation_number=1; citation_pages=304-310; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Vertebrate innovations.; citation_author=SM Shimeld; citation_author=PW Holland; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=97; citation_number=2; citation_pages=4449-4452; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Evolutionary history of the ligand-gated ion-channel superfamily of receptors.; citation_author=MO Ortells; citation_author=GG Lunt; citation_journal_title=Trends Neurosci; citation_volume=18; citation_number=3; citation_pages=121-127; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Early developmental modulation of GABAA receptor function. Influence on adaptive responses.; citation_author=CK Kellogg; citation_journal_title=Perspect Dev Neurobiol; citation_volume=5; citation_number=4; citation_pages=219-234; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Benzodiazepines and anterior pituitary function.; citation_author=E Arvat; citation_author=R Giordano; citation_author=S Grottoli; citation_author=E Ghigo; citation_journal_title=J Endocrinol Invest; citation_volume=25; citation_number=5; citation_pages=735-747; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Neurotransmitter systems involved in learning and memory in the rat: a meta-analysis based on studies of four behavioral tasks.; citation_author=T Myhrer; citation_journal_title=Brain Res Brain Res Rev; citation_volume=41; citation_number=6; citation_pages=268-287; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=gamma-Aminobutyric acid in the nervous system of a planarian.; citation_author=KS Eriksson; citation_author=P Panula; citation_journal_title=J Comp Neurol; citation_volume=345; citation_number=7; citation_pages=528-536; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Immunohistochemical and biochemical evidence for the putative inhibitory neurotransmitters histamine and GABA in lobster olfactory lobes.; citation_author=E Orona; citation_author=BA Battelle; citation_author=BW Ache; citation_journal_title=J Comp Neurol; citation_volume=294; citation_number=8; citation_pages=633-646; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Invertebrate GABA and glutamate receptors: molecular biology reveals predictable structures but some unusual pharmacologies.; citation_author=MG Darlison; citation_journal_title=Trends Neurosci; citation_volume=15; citation_number=9; citation_pages=469-474; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Distribution of GABA immunoreactivity in the central and peripheral nervous system of amphioxus (Branchiostoma lanceolatum Pallas).; citation_author=R Anadon; citation_author=F Adrio; citation_author=I Rodriguez-Moldes; citation_journal_title=J Comp Neurol; citation_volume=401; citation_number=10; citation_pages=293-307; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The GABA-A receptor gene family: new targets for therapeutic intervention.; citation_author=PJ Whiting; citation_journal_title=Neurochem Int; citation_volume=34; citation_number=11; citation_pages=387-390; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=From ion currents to genomic analysis: recent advances in GABAA receptor research.; citation_author=LE Rabow; citation_author=SJ Russek; citation_author=DH Farb; citation_journal_title=Synapse; citation_volume=21; citation_number=12; citation_pages=189-274; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=GABAA receptors and benzodiazepines: a role for dendritic resident subunit mRNAs.; citation_author=E Costa; citation_author=J Auta; citation_author=DR Grayson; citation_author=K Matsumoto; citation_author=GD Pappas; citation_journal_title=Neuropharmacology; citation_volume=43; citation_number=13; citation_pages=925-937; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The ‘ABC’ of GABA receptors.; citation_author=J Bormann; citation_journal_title=Trends Pharmacol Sci; citation_volume=21; citation_number=14; citation_pages=16-19; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=GABAB receptors: a new paradigm in G protein signaling.; citation_author=A Couve; citation_author=SJ Moss; citation_author=MN Pangalos; citation_journal_title=Mol Cell Neurosci; citation_volume=16; citation_number=15; citation_pages=296-312; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=GABAC receptors.; citation_author=J Bormann; citation_author=A Feigenspan; citation_journal_title=Trends Neurosci; citation_volume=18; citation_number=16; citation_pages=515-519; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Immunocytochemical localization of the GABA(C) receptor rho subunits in the cat, goldfish, and chicken retina.; citation_author=P Koulen; citation_author=JH Brandstatter; citation_author=S Kroger; citation_author=R Enz; citation_author=J Bormann; citation_journal_title=J Comp Neurol; citation_volume=380; citation_number=17; citation_pages=520-532; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Medicinal chemistry and molecular pharmacology of GABA(C) receptors.; citation_author=GA Johnston; citation_journal_title=Curr Top Med Chem; citation_volume=2; citation_number=18; citation_pages=903-913; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evolution of GABA(A) receptor diversity in the human genome.; citation_author=SJ Russek; citation_journal_title=Gene; citation_volume=227; citation_number=19; citation_pages=213-222; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Consequences of the evolution of the GABA(A) receptor gene family.; citation_author=MG Darlison; citation_author=I Pahal; citation_author=C Thode; citation_journal_title=Cell Mol Neurobiol; citation_volume=25; citation_number=20; citation_pages=607-624; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of the set of GABA(A) receptor genes in the human genome.; citation_author=J Simon; citation_author=H Wakimoto; citation_author=N Fujita; citation_author=M Lalande; citation_author=EA Barnard; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=21; citation_pages=41422-41435; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Identification of a putative gamma-aminobutyric acid (GABA) receptor subunit rho2 cDNA and colocalization of the genes encoding rho2 (GABRR2) and rho1 (GABRR1) to human chromosome 6q14-q21 and mouse chromosome 4.; citation_author=GR Cutting; citation_author=S Curristin; citation_author=H Zoghbi; citation_author=B O'Hara; citation_author=MF Seldin; citation_journal_title=Genomics; citation_volume=12; citation_number=22; citation_pages=801-806; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Identification of major phylogenetic branches of inhibitory ligand-gated channel receptors.; citation_author=H Xue; citation_journal_title=J Mol Evol; citation_volume=47; citation_number=23; citation_pages=323-333; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evidence for a diverse Cys-loop ligand-gated ion channel superfamily in early bilateria.; citation_author=JA Dent; citation_journal_title=J Mol Evol; citation_volume=62; citation_number=24; citation_pages=523-535; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The evolution of GABAA receptor-like genes.; citation_author=SY Tsang; citation_author=SK Ng; citation_author=Z Xu; citation_author=H Xue; citation_journal_title=Mol Biol Evol; citation_volume=24; citation_number=25; citation_pages=599-610; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Functional genomics and enzyme evolution. Homologous and analogous enzymes encoded in microbial genomes.; citation_author=MY Galperin; citation_author=EV Koonin; citation_journal_title=Genetica; citation_volume=106; citation_number=26; citation_pages=159-170; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=GABA(A) receptor epsilon and theta subunits display unusual structural variation between species and are enriched in the rat locus ceruleus.; citation_author=ST Sinkkonen; citation_author=MC Hanna; citation_author=EF Kirkness; citation_author=ER Korpi; citation_journal_title=J Neurosci; citation_volume=20; citation_number=27; citation_pages=3588-3595; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A codon-based model of nucleotide substitution for protein-coding DNA sequences.; citation_author=N Goldman; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=11; citation_number=28; citation_pages=725-736; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=29; citation_pages=2472-2479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages.; citation_author=Z Yang; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=30; citation_pages=908-917; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling.; citation_author=K Arnold; citation_author=L Bordoli; citation_author=J Kopp; citation_author=T Schwede; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=31; citation_pages=195-201; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of an ACh-binding protein reveals the ligand-binding domain of nicotinic receptors.; citation_author=K Brejc; citation_author=WJ van Dijk; citation_author=RV Klaassen; citation_author=M Schuurmans; citation_author=J van Der Oost; citation_journal_title=Nature; citation_volume=411; citation_number=32; citation_pages=269-276; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Enhancement of protein modeling by human intervention in applying the automatic programs 3D-JIGSAW and 3D-PSSM.; citation_author=PA Bates; citation_author=LA Kelley; citation_author=RM MacCallum; citation_author=MJ Sternberg; citation_journal_title=Proteins Suppl; citation_volume=5; citation_number=33; citation_pages=39-46; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Nicotinic receptors at the amino acid level.; citation_author=PJ Corringer; citation_author=N Le Novere; citation_author=JP Changeux; citation_journal_title=Annu Rev Pharmacol Toxicol; citation_volume=40; citation_number=34; citation_pages=431-458; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for heterogeneous selection pressure at amino acid sites.; citation_author=Z Yang; citation_author=R Nielsen; citation_author=N Goldman; citation_author=AM Pedersen; citation_journal_title=Genetics; citation_volume=155; citation_number=35; citation_pages=431-449; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Not so different after all: a comparison of methods for detecting amino acid sites under selection.; citation_author=SL Kosakovsky Pond; citation_author=SD Frost; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=36; citation_pages=1208-1222; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical Bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WS Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=37; citation_pages=1107-1118; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Comprehensive analysis of the ascidian genome reveals novel insights into the molecular evolution of ion channel genes.; citation_author=Y Okamura; citation_author=A Nishino; citation_author=Y Murata; citation_author=K Nakajo; citation_author=H Iwasaki; citation_journal_title=Physiol Genomics; citation_volume=22; citation_number=38; citation_pages=269-282; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evidence for a late evolutionary appearance of brain-specific benzodiazepine receptors: an investigation of 18 vertebrate and 5 invertebrate species.; citation_author=M Nielsen; citation_author=C Braestrup; citation_author=RF Squires; citation_journal_title=Brain Res; citation_volume=141; citation_number=39; citation_pages=342-346; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic comparison of the photoaffinity-labeled benzodiazepine receptor subunits.; citation_author=J Hebebrand; citation_author=W Friedl; citation_author=B Breidenbach; citation_author=P Propping; citation_journal_title=J Neurochem; citation_volume=48; citation_number=40; citation_pages=1103-1108; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=GABA and glutamate receptors are involved in modulating pacemaker activity in hydra.; citation_author=G Kass-Simon; citation_author=A Pannaccione; citation_author=P Pierobon; citation_journal_title=Comp Biochem Physiol A Mol Integr Physiol; citation_volume=136; citation_number=41; citation_pages=329-342; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Evolution of new hormone function: loss and gain of a receptor.; citation_author=DM Irwin; citation_author=K Wong; citation_journal_title=J Hered; citation_volume=96; citation_number=42; citation_pages=205-211; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish.; citation_author=S Hoegg; citation_author=H Brinkmann; citation_author=JS Taylor; citation_author=A Meyer; citation_journal_title=J Mol Evol; citation_volume=59; citation_number=43; citation_pages=190-203; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Major events in the genome evolution of vertebrates: paranome age and size differ considerably between ray-finned fishes and land vertebrates.; citation_author=K Vandepoele; citation_author=W De Vos; citation_author=JS Taylor; citation_author=A Meyer; citation_author=Y Van de Peer; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=101; citation_number=44; citation_pages=1638-1643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=GABAA receptor epsilon subunit expression in identified peptidergic neurons of the rat hypothalamus.; citation_author=N Moragues; citation_author=P Ciofi; citation_author=P Lafon; citation_author=G Tramu; citation_author=M Garret; citation_journal_title=Brain Res; citation_volume=967; citation_number=45; citation_pages=285-289; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Impact of epsilon and theta subunits on pharmacological properties of alpha3beta1 GABAA receptors expressed in Xenopus oocytes.; citation_author=M Ranna; citation_author=ST Sinkkonen; citation_author=T Moykkynen; citation_author=M Uusi-Oukari; citation_author=ER Korpi; citation_journal_title=BMC Pharmacol; citation_volume=6; citation_number=46; citation_pages=1; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Locus coeruleus lesions decrease norepinephrine input into the medial preoptic area and medial basal hypothalamus and block the LH, FSH and prolactin preovulatory surge.; citation_author=JA Anselmo-Franci; citation_author=CR Franci; citation_author=L Krulich; citation_author=J Antunes-Rodrigues; citation_author=SM McCann; citation_journal_title=Brain Res; citation_volume=767; citation_number=47; citation_pages=289-296; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Neuropeptide Y in the medial basal hypothalamus and medial preoptic area during the induction of LH surge may be controlled by locus coeruleus.; citation_author=AA Rocha; citation_author=R Guerra-Sa; citation_author=NA Silveira; citation_author=JA Anselmo-Franci; citation_author=CR Franci; citation_journal_title=Neuropeptides; citation_volume=40; citation_number=48; citation_pages=57-63; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Identification of a residue in the gamma-aminobutyric acid type A receptor alpha subunit that differentially affects diazepam-sensitive and -insensitive benzodiazepine site binding.; citation_author=JM Derry; citation_author=SM Dunn; citation_author=M Davies; citation_journal_title=J Neurochem; citation_volume=88; citation_number=49; citation_pages=1431-1438; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Mutation of glutamate 155 of the GABAA receptor beta2 subunit produces a spontaneously open channel: a trigger for channel activation.; citation_author=JG Newell; citation_author=RA McDevitt; citation_author=C Czajkowski; citation_journal_title=J Neurosci; citation_volume=24; citation_number=50; citation_pages=11226-11235; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Basic local alignment search tool.; citation_author=SF Altschul; citation_author=W Gish; citation_author=W Miller; citation_author=EW Myers; citation_author=DJ Lipman; citation_journal_title=J Mol Biol; citation_volume=215; citation_number=51; citation_pages=403-410; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=52; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=53; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=ProtTest: selection of best-fit models of protein evolution.; citation_author=F Abascal; citation_author=R Zardoya; citation_author=D Posada; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=54; citation_pages=2104-2105; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences.; citation_author=DT Jones; citation_author=WR Taylor; citation_author=JM Thornton; citation_journal_title=Comput Appl Biosci; citation_volume=8; citation_number=55; citation_pages=275-282; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods.; citation_author=Z Yang; citation_journal_title=J Mol Evol; citation_volume=39; citation_number=56; citation_pages=306-314; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Bayesian phylogenetic model selection using reversible jump Markov chain Monte Carlo.; citation_author=JP Huelsenbeck; citation_author=B Larget; citation_author=ME Alfaro; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=57; citation_pages=1123-1133; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Bayesian inference in molecular phylogenetics.; citation_author=Z Yang; citation_number=58; citation_pages=63-90; citation_date=2005; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=Maximum likelihood methods for detecting adaptive evolution after gene duplication.; citation_author=JP Bielawski; citation_author=Z Yang; citation_journal_title=J Struct Funct Genomics; citation_volume=3; citation_number=59; citation_pages=201-212; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=60; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene.; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Genetics; citation_volume=148; citation_number=61; citation_pages=929-936; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Comparative modeling in CASP6 using consensus approach to template selection, sequence-structure alignment, and structure assessment.; citation_author=C Venclovas; citation_author=M Margelevicius; citation_journal_title=Proteins 61 Suppl; citation_volume=7; citation_number=62; citation_pages=99-105; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits"/>
    <meta name="twitter:description" content="BackgroundThe amino acid neurotransmitter GABA is abundant in the central nervous system (CNS) of both invertebrates and vertebrates. Receptors of this neurotransmitter play a key role in important processes such as learning and memory. Yet, little is known about the mode and tempo of evolution of the receptors of this neurotransmitter. Here, we investigate the phylogenetic relationships of GABA receptor subunits across the chordates and detail their mode of evolution among mammals.Principal FindingsOur analyses support two major monophyletic clades: one clade containing GABAA receptor &alpha;, &gamma;, and &epsilon; subunits, and another one containing GABAA receptor &rho;, &beta;, &delta;, &theta;, and &pi; subunits. The presence of GABA receptor subunits from each of the major clades in the Ciona intestinalis genome suggests that these ancestral duplication events occurred before the divergence of urochordates. However, while gene divergence proceeded at similar rates on most receptor subunits, we show that the mammalian-specific subunits &theta; and &epsilon; experienced an episode of positive selection and of relaxed constraints, respectively, after the duplication event. Sites putatively under positive selection are placed on a three-dimensional model obtained by homology-modeling.ConclusionsOur results suggest an early divergence of the GABA receptor subunits, before the split from urochordates. We show that functional changes occurred in the lineages leading to the mammalian-specific subunit &theta;, and we identify the amino acid sites putatively responsible for the functional divergence. We discuss potential consequences for the evolution of mammals and of their CNS."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000894.g005"/>

  <meta property="og:title" content="Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7532'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6735'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3882&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000894">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000894" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000894&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christopher J. Martyniuk, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada
                </p>

                <p>
                    Current address: Center for Environmental and Human Toxicology, University of Florida, Gainesville, Florida, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Stéphane Aris-Brosou
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:sarisbro@uottawa.ca">sarisbro@uottawa.ca</a></p>

                <p>Affiliations:
                  Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada, 
                  Department of Mathematics and Statistics, University of Ottawa, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Guy Drouin, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joel Cahn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Vance L. Trudeau
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biology and Centre for Advanced Research in Environmental Genomics, University of Ottawa, Ottawa, Ontario, Canada
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: September 19, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000894</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000894-g001" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000894-g002" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000894-g003" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000894-t001" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000894-t002" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000894-g004" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000894-g005" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000894">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000894" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>The amino acid neurotransmitter GABA is abundant in the central nervous system (CNS) of both invertebrates and vertebrates. Receptors of this neurotransmitter play a key role in important processes such as learning and memory. Yet, little is known about the mode and tempo of evolution of the receptors of this neurotransmitter. Here, we investigate the phylogenetic relationships of GABA receptor subunits across the chordates and detail their mode of evolution among mammals.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>Our analyses support two major monophyletic clades: one clade containing GABA<sub>A</sub> receptor α, γ, and ε subunits, and another one containing GABA<sub>A</sub> receptor ρ, β, δ, θ, and π subunits. The presence of GABA receptor subunits from each of the major clades in the <em>Ciona intestinalis</em> genome suggests that these ancestral duplication events occurred before the divergence of urochordates. However, while gene divergence proceeded at similar rates on most receptor subunits, we show that the mammalian-specific subunits θ and ε experienced an episode of positive selection and of relaxed constraints, respectively, after the duplication event. Sites putatively under positive selection are placed on a three-dimensional model obtained by homology-modeling.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our results suggest an early divergence of the GABA receptor subunits, before the split from urochordates. We show that functional changes occurred in the lineages leading to the mammalian-specific subunit θ, and we identify the amino acid sites putatively responsible for the functional divergence. We discuss potential consequences for the evolution of mammals and of their CNS.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Martyniuk CJ, Aris-Brosou S, Drouin G, Cahn J, Trudeau VL (2007) Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits. PLoS ONE 2(9):
          e894.
            doi:10.1371/journal.pone.0000894</p><p><strong>Academic Editor: </strong>Jason Stajich, University of California at Berkeley, United States of America</p><p><strong>Received:</strong> June 11, 2007; <strong>Accepted:</strong> August 14, 2007; <strong>Published:</strong> September 19, 2007</p><p><strong>Copyright:</strong> © 2007 Martyniuk et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by an Ontario Graduate Scholarship to CJM, and by Natural Sciences and Engineering Research Council Discovery Grants to VLT, SAB and GD.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Gene duplication followed by gene divergence is one of the major mechanisms responsible for the evolution of new functions <a href="#pone.0000894-Ohta1">[1]</a>. It is thought to underlie the evolution of vertebrates and more particularly their complex and specialized central nervous system (CNS) <a href="#pone.0000894-Shimeld1">[2]</a>. Two classes of proteins that underwent such a mechanism are cationic (e.g., acetylcholine, serotonin) and anionic (e.g., glycine, γ-aminobutyric acid) ligand-gated channels, estimated to have diverged before the origin of eukaryotes <a href="#pone.0000894-Ortells1">[3]</a>. However, little more is known about the mode and tempo of evolution of these receptors. Such knowledge may provide us with insights into the structural and functional complexity of these receptors in the central nervous system and in their role in mammalian evolution.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>The γ-aminobutyric acid (GABA) is the major inhibitory neurotransmitter found in the vertebrate brain and is involved in CNS development and organization <a href="#pone.0000894-Kellogg1">[4]</a>, neuroendocrine function <a href="#pone.0000894-Arvat1">[5]</a>, and neural processes such as learning and memory <a href="#pone.0000894-Myhrer1">[6]</a>. GABA is also present in the nervous system of non-vertebrate taxa, for example, flatworms <a href="#pone.0000894-Eriksson1">[7]</a>, arthropods <a href="#pone.0000894-Orona1">[8]</a>, <a href="#pone.0000894-Darlison1">[9]</a> and early chordates <a href="#pone.0000894-Anadon1">[10]</a>. GABA synaptic transmission is achieved through membrane bound postsynaptic receptors. Currently, there are three major classes of GABA receptors identified in the mammalian CNS: GABA<sub>A</sub>, GABA<sub>B</sub> and GABA<sub>C</sub>. These are distinguished according to their composition, pharmacology and localization. Ionotropic GABA<sub>A</sub> receptors are ligand-gated chloride (Cl<sup>−</sup>) channels consisting of both high abundance subunits (α1-6, β1-4, γ1-3, δ) and low abundance subunits (ε, θ, and π) <a href="#pone.0000894-Whiting1">[11]</a>. Changes in the abundance and composition of these subunits have been shown to induce differences in GABA<sub>A</sub> receptor sensitivity and response <a href="#pone.0000894-Rabow1">[12]</a>, <a href="#pone.0000894-Costa1">[13]</a>. The current structural model of the GABA<sub>A</sub> receptor is a pentameric receptor with binding sites for the GABA ligand and for receptor modulation by benzodiazepines, neurosteroids, ethanol, and barbiturates <a href="#pone.0000894-Bormann1">[14]</a>. Metabotropic GABA<sub>B</sub> receptors are members of the seven transmembrane domain family and are coupled to downstream calcium and potassium channels via G-proteins <a href="#pone.0000894-Couve1">[15]</a>. Finally, GABA<sub>C</sub> receptors are also pentameric ionotropic Cl<sup>−</sup> channels that show similar membrane topology as the GABA<sub>A</sub> receptors. However, GABA<sub>C</sub> receptors have unique functional and electrophysiological characteristics, including a slower Cl<sup>−</sup> conductance and insensitivity to bicuculline and other GABA<sub>A</sub> receptor modulators <a href="#pone.0000894-Bormann1">[14]</a>, <a href="#pone.0000894-Bormann2">[16]</a>. Ionotropic GABA<sub>C</sub> receptors are composed of ρ subunits that are highly expressed in the vertebrate retina and preferentially localized to bipolar cells <a href="#pone.0000894-Koulen1">[17]</a> but are also found in the spinal cord and pituitary <a href="#pone.0000894-Johnston1">[18]</a>. Both spatial and temporal regulation of GABA receptor subunit expression provide functional diversity to the GABA receptor family.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Fourteen of the human GABA<sub>A</sub> receptor genes are clustered on four chromosomes (chromosomes 4, 5, 15, and X: <a href="#pone.0000894-Russek1">[19]</a>, <a href="#pone.0000894-Darlison2">[20]</a>). Two of these clusters contain two genes coding for receptor α subunits, one gene coding for a receptor β subunit and one gene coding for a receptor γ subunit whereas the other two clusters contain single genes coding for a receptor α, β and γ subunit–with the ε subunit gene replacing the γ subunit gene on the X chromosome <a href="#pone.0000894-Russek1">[19]</a>. Evidence based on chromosomal organization, intron/exon structure and transcriptional organization suggests that these four clusters originated from duplications of (and within) an ancestral GABA<sub>A</sub> receptor gene cluster containing single genes coding for a receptor α, β and γ subunit <a href="#pone.0000894-Russek1">[19]</a>–<a href="#pone.0000894-Simon1">[21]</a>. The other two GABA<sub>A</sub> receptor genes, coding for the receptor δ and the π subunits, also likely arose by duplication of an ancestral GABA<sub>A</sub> receptor gene(s) but are not part of the four GABA<sub>A</sub> receptor gene clusters <a href="#pone.0000894-Darlison2">[20]</a>. GABA<sub>C</sub> receptor genes share only about 35% amino acid sequence identity with GABA<sub>A</sub> receptor genes <a href="#pone.0000894-Darlison2">[20]</a>. This suggests that these two types of genes may have a common origin and have since acquired distinct functions <a href="#pone.0000894-Darlison2">[20]</a>, <a href="#pone.0000894-Cutting1">[22]</a>. However, these studies largely focused on mammalian sequences and the evolutionary history of ionotropic GABA receptor subunits before the origin of mammals is unclear.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Our goal here is twofold. First, we extend our knowledge of the phylogenetic relationships between ionotropic GABA receptor subunits, in particular by including recently available genomes such as those of the sea squirt <em>Ciona intestinalis</em> and the pufferfish <em>Takifugu rubripes</em>. Second, we test whether positive selection can explain the evolution of the mammalian-specific GABA receptor subunits θ and ε.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>GABA<sub>A</sub> receptor phylogeny</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Our estimate of the GABA<sub>A</sub> receptor phylogeny shows two major monophyletic clades (<a href="#pone-0000894-g001">Fig. 1</a>), which is consistent with previous studies <a href="#pone.0000894-Ortells1">[3]</a>, <a href="#pone.0000894-Xue1">[23]</a>–<a href="#pone.0000894-Tsang1">[25]</a>. This topology is robust to both the search algorithm used and the model of evolution. Indeed, the same topology was obtained by both maximum likelihood and Bayesian approaches, with the latter integrating over different substitution models (where the substitution model used in maximum likelihood [see <a href="#s4">Methods</a>] had a posterior probability of one). The phylogenetic tree of vertebrate sequences containing the GABA<sub>A</sub> receptor α, γ, and ε subunits shows that these receptor subunits do not result from non-orthologous gene displacement <a href="#pone.0000894-Galperin1">[26]</a> but appear to be derived from a common ancestor (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g002">2</a>). Subunits α are divided into two strongly supported groups: a clade composed of subunits α4 and α6 and the other composed of subunits α1-3 and α5. Subunits γ1 and γ2 form a strongly supported group, whereas the γ3, γ4 and ε subunits all group together. This suggests that the GABA<sub>A</sub> receptor ε subunits are derived from γ subunits. This is in agreement with previous suggestions that the GABA<sub>A</sub> receptor subunits γ4 of chicken and ε found in mammals are likely to be orthologous <a href="#pone.0000894-Darlison2">[20]</a>, <a href="#pone.0000894-Sinkkonen1">[27]</a>.</p>
<div class="figure" id="pone-0000894-g001"><div class="img"><a name="pone-0000894-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Phylogeny of GABA receptor protein sequences.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The two clades, based on the presence (α, γ, and ε) or absence (ρ, β, δ, θ, and π) of a benzodiazepine binding site, are indicated by BZ+ and BZ–, respectively. Bootstrap values (maximum likelihood analysis) are indicated at all nodes while posterior probabilities (Bayesian analysis) are only indicated when smaller than 100%. The scale bar represents 1 amino acid substitutions per site. Ach: acetylcholine receptor.</p>
<span>doi:10.1371/journal.pone.0000894.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The relationship of the GABA receptor subunits involved in benzodiazepine binding (subunits ρ, β, δ, θ, and π) is uncertain and depends on whether invertebrate GABA/glycine-like receptor sequences are included or not in the analysis. Indeed, when the invertebrate GABA/glycine-like receptor sequences are included (81-sequence data set), subunits δ and π are found to be the sister group to the other GABA receptor sequences in this clade (<a href="#pone-0000894-g001">Fig. 1</a>). In contrast, the tree estimated without the invertebrate GABA/glycine-like receptor sequences (55-sequence data set) shows that subunits ρ are the sister group to other subunit families within this clade (<a href="#pone-0000894-g003">Fig. 3</a>). However and importantly, for both trees, the GABA<sub>C</sub> receptor ρ subunits, the GABA<sub>A</sub> receptor θ and β subunits, and the GABA<sub>A</sub> receptor π and δ subunits, form three strongly supported groups (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g003">3</a>).</p>
<div class="figure" id="pone-0000894-g002"><div class="img"><a name="pone-0000894-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Phylogeny of vertebrate GABA receptor protein sequences with benzodiazepine binding sites.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Bootstrap values are indicated at the nodes and the scale bar represents 1 amino acid substitutions per site.</p>
<span>doi:10.1371/journal.pone.0000894.g002</span></div><div class="figure" id="pone-0000894-g003"><div class="img"><a name="pone-0000894-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Phylogeny of vertebrate GABA receptor protein sequences without benzodiazepine binding sites.</span></strong></p><a id="article1.body1.sec2.sec1.fig3.caption1.p1" name="article1.body1.sec2.sec1.fig3.caption1.p1"></a><p>Bootstrap values are indicated at the nodes and the scale bar represents 1 amino acid substitutions per site.</p>
<span>doi:10.1371/journal.pone.0000894.g003</span></div>

<h4>Analysis of selective pressures</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Our phylogenetic analyses suggested that the θ and ε paralogs have undergone a period of accelerated evolution following the duplication event. Such a period of accelerated evolution could be due to an episode of positive selection affecting these branches. To test this hypothesis, we used a codon substitution model, which measures selective pressures by estimating the nonsynonymous to synonymous rate ratio. This ratio is denoted <em>ω</em>, with <em>ω</em> = 1, &lt;1 and &gt;1 indicating neutral evolution, purifying and positive selection, respectively <a href="#pone.0000894-Goldman1">[28]</a>. The comparison of a null codon model where selective pressures are constant both along lineages and among sites (H<sub>0</sub>) against a branch-specific or “free-ratio” model (denoted Br in <a href="#pone-0000894-t001">Tables 1</a> and <a href="#pone-0000894-t002">2</a>) showed an extensive variation of the <em>ω</em> rate ratio among branches in both data sets (<a href="#pone-0000894-t001">Tables 1</a> and <a href="#pone-0000894-t002">2</a>).</p>
<div class="figure" id="pone-0000894-t001"><div class="img"><a name="pone-0000894-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Model comparisons and parameter estimates under models of constant (H<sub>0</sub>) or variable <em>ω</em> rate ratios across branches (Br), clades (H<sub>1</sub> to H<sub>3</sub>) or both clades and sites (A and B) for the θ subunit.</span></strong></p><span>doi:10.1371/journal.pone.0000894.t001</span></div><div class="figure" id="pone-0000894-t002"><div class="img"><a name="pone-0000894-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Model comparisons and parameter estimates under models of constant (H<sub>0</sub>) or variable <em>ω</em> rate ratios across branches (Br), clades (H<sub>1</sub> to H<sub>3</sub>) or both clades and sites (A and B) for the ε subunit.</span></strong></p><span>doi:10.1371/journal.pone.0000894.t002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Our results suggest that the GABA<sub>A</sub> receptor θ subunit was subjected to positive selection. For reference, we label branches and clades following the conventions set in <a href="#pone-0000894-g004">Figure 4a</a>. When allowing selective pressures to differ specifically in the branch leading to this θ subunit (branch number 5: model H<sub>1</sub>), the model does not significantly explain the data better (at the 1% level) than the constant <em>ω</em> rate ratio model (H<sub>0</sub>; <a href="#pone-0000894-t001">Table 1</a>). However, specifically allowing for a different rate in the branch of interest (branch number 5) and allowing <em>ω</em> to vary among sites within this branch suggested the existence of sites evolving under positive selection in this branch. Indeed, both the test of positive selection (model A; see <a href="#s4">Methods</a> and <a href="#pone.0000894-Zhang1">[29]</a>) and its variant (model B; see <a href="#s4">Methods</a> and <a href="#pone.0000894-Yang1">[30]</a>) have highly significant <em>P</em>-values (<a href="#pone-0000894-t001">Table 1</a>). The estimate of <em>ω</em> for the sites allowed to be under positive selection in branch 5 or <em>ω</em><sub>5</sub><sup>(2)</sup> under model B (<a href="#pone-0000894-t001">Table 1</a>) is infinity because the synonymous rate along this branch is almost zero. Over all the branches, the average synonymous rate was .356 (excluding root branches), which suggests that saturation may not be an issue for these analyses. However, the sites estimated under models A and B are all different, except for site M274W. While both models consistently detect some evidence for positive selection, the identity of these sites is most likely unreliable, save for the possible exception of site M274W.</p>
<div class="figure" id="pone-0000894-g004"><div class="img"><a name="pone-0000894-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Phylogenetic trees used to formulate hypotheses about the selective forces that shaped the evolution of the GABA<sub>A</sub> subunits.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>(a) the GABA<sub>A</sub> receptor θ and β subunits ; (b) the GABA<sub>A</sub> receptor ε and γ subunits. Selective pressures are allowed to vary among sets of branches to test for the a priori hypotheses listed at the bottom of the figure: H<sub>0</sub>: the null hypothesis that all branch-specific rates are equal; H<sub>1</sub>: burst of evolution following the main gene duplication event (ε/γ and θ/β); H<sub>2</sub>: also allows for a rate change after the burst of evolution; H<sub>3</sub>: extends H<sub>2</sub> to allow for burst of evolution after all duplication events depicted on each tree. Hypotheses test are reported in <a href="#pone-0000894-t001">Tables 1</a> and <a href="#pone-0000894-t002">2</a>. The scale bar represents 0.1 substitutions per codon site, with branch lengths estimated under the null model (H<sub>0</sub>).</p>
<span>doi:10.1371/journal.pone.0000894.g004</span></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>As of July 2007, no three-dimensional (3D) structures for GABA receptor subunits were deposited in the Protein DataBank (PDB: <a href="http://www.pdb.org">www.pdb.org</a>). Similarity searches based on BLASTp using the protein sequence of the human θ paralog (accession number: AAI09211) returned no structure from PDB. Homology modeling with SWISS-MODEL <a href="#pone.0000894-Arnold1">[31]</a> only produced a model between amino acid positions 291 and 347. This prediction was based on the NMR-obtained template whose PDB accession number is 1VRY, which corresponds to the second and third transmembrane domains of the α-1 subunit of human glycine receptor. The use of recently produced structure of the acetylcholine binding receptor (PDB: 1I9B; <a href="#pone.0000894-Brejc1">[32]</a>) as a template failed to produce any 3D model: this is because the 17% similarity with the human θ paralog is below the 25% threshold of SWISS-MODEL. 3D-JIGSAW <a href="#pone.0000894-Bates1">[33]</a> produced a larger model that encompasses the previous one and spans from position 62 to 372 (<a href="#pone-0000894-g005">Fig. 5</a>). Because 3D-JIGSAW is more liberal than SWISS-MODEL, its prediction is expected to be somewhat inaccurate outside of the glycine receptor domain. To assess the robustness of our 3D model, we also produced 3D models based on the amino acid sequence of human paralogs β1 (CAA32875), β2 (AAB29370) and β3 (AAA52511). The models produced by 3D-JIGSAW had, after structural alignment, root mean square deviations of 1.21Å, 1.15Å and 1.60Å, respectively, with the model produced with the human θ paralog. These figures are smaller than the 2.5Å resolution usually obtained by X-ray diffraction, so that our model is relatively robust to the protein sequence used within the β/θ group. Although the interpretation of these models should be based on a more rigorous analysis, three domains are apparent on this model (<a href="#pone-0000894-g005">Fig. 5</a>): an N-terminal domain composed of essentially of β-sheets that might correspond to the ligand-binding domain, an intermediate domain composed essentially of α-helices that might correspond to the transmembrane region and a C-terminal domain that might correspond to an intracellular region. This interpretation is consistent with what is known of our model's template (1VRY) and, more generally, of pentameric ligand-gated ion channels such as nAChR, 5-HT<sub>3</sub>, GABA<sub>C</sub>, and glycine receptors <a href="#pone.0000894-Corringer1">[34]</a>. The sites putatively evolving under positive selection along branch 5 are located in all three putative domains of the receptor (<a href="#pone-0000894-g005">Fig. 5a–b</a>). Note that site M274W, identified by both model A and model B is located within the putative transmembrane region on the first α-helices (M1: <a href="#pone-0000894-g005">Fig.5</a>; <a href="#pone.0000894-Corringer1">[34]</a>). Further analyses allowing <em>ω</em> to differ among branches leading to the β paralogs (models H<sub>2</sub> and H<sub>3</sub>) are consistent with an episode of positive selection leading to the θ paralogs, followed by a regime of negative selection. To summarize, evidence for positive selection acting in this branch is strong, seems to have affected all three putative domains of the receptor subunit, but the identification of the actual sites should be taken with caution due to the relatively small number of sequences analyzed here.</p>
<div class="figure" id="pone-0000894-g005"><div class="img"><a name="pone-0000894-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000894" data-uri="info:doi/10.1371/journal.pone.0000894.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000894.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000894.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000894.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000894.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Mapping of the sites putatively under positive selection on the three-dimensional models of the θ subunit of the GABA<sub>A</sub> receptor.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>Sites putatively detected to be under positive selection on subunit θ under: (a) model A and (b) model B.</p>
<span>doi:10.1371/journal.pone.0000894.g005</span></div><a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Results from the ε data set only show evidence for relaxed selective pressures along the branch leading to the ε subunit. Here also, branches and clades are numbered following conventions in <a href="#pone-0000894-g004">Figure 4b</a>. As shown above, the reconstruction of this region of the GABA tree is unstable (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g002">2</a>). However, all results shown below were identical to the fifth decimal place under both tree reconstructions. This probably reflects (i) the low support for the ε clade (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g002">2</a>) and (ii) the relative robustness of site codon models to the exact topology of the tree as long as this tree is reasonable <a href="#pone.0000894-Yang2">[35]</a>, <a href="#pone.0000894-KosakovskyPond1">[36]</a>. As with subunit θ, the fit of model H<sub>1</sub> is not significantly better than that of the one ratio model (H<sub>0</sub>), but neither test with model A or with model B are significant (<a href="#pone-0000894-t002">Table 2</a>). Importantly, the comparison of model A with a model called M1a <a href="#pone.0000894-Yang3">[37]</a> that assumes only two classes of sites, with 0&lt;<em>ω</em>&lt;0 and <em>ω</em> = 1, is highly significant (<a href="#pone-0000894-t002">Table 2</a>). This latest comparison, called “test 1” in <a href="#pone.0000894-Zhang1">[29]</a>, cannot discriminate efficiently between positive selection and relaxed selective pressures <a href="#pone.0000894-Zhang1">[29]</a>. Taken together, a significant “test 1” and a non-significant “test 2” suggest that the ε subunit only underwent relaxed constraints. Because of the conservative nature of “test 2” (see <a href="#s4">Methods</a> and <a href="#pone.0000894-Zhang1">[29]</a>), and because the average synonymous rate was 4.105 (excluding root branches) in the ε/γ data set, more sequences will be required to definitely rule out the action of positive selection during the evolution of the ε subunit.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>The present analyses and previous work <a href="#pone.0000894-Ortells1">[3]</a>, <a href="#pone.0000894-Xue1">[23]</a> all suggest that an ancestral GABA-like receptor subunit gave rise to two monophyletic clades, categorized as subunits involved (α, γ, and ε) or not involved (ρ, β, δ, θ, and π) in benzodiazepine binding. As previously found <a href="#pone.0000894-Dent1">[24]</a>, <a href="#pone.0000894-Tsang1">[25]</a>, <a href="#pone.0000894-Okamura1">[38]</a>, the presence of putative GABA α-, β- and ρ-like receptor subunits in the genome of <em>C. intestinalis</em> support that this ancestral duplication most likely occurred before the divergence of urochordates. These results suggest that benzodiazepine sensitivity evolved early, in marked contrast to previous proposals <a href="#pone.0000894-Nielsen1">[39]</a>, <a href="#pone.0000894-Hebebrand1">[40]</a>. In line with our finding, recent electrophysiological data suggest that invertebrates (e.g., <em>Hydra vulgaris</em>) are responsive to benzodiazepine modulation and this response is similar to the response to GABA <a href="#pone.0000894-KassSimon1">[41]</a>. On the other hand, the presence of α and β-like subunits in <em>C. intestinalis</em> does not necessarily indicate benzodiazepine sensitivity.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>It was previously suggested that the GABA<sub>A</sub> receptor δ subunit is the most primitive subunit within the GABA<sub>A</sub> clade not involved in benzodiazepine binding <a href="#pone.0000894-Ortells1">[3]</a>. Our phylogenetic analyses were unable to confidently determine the probable progenitor for receptor subunits in this clade (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g003">3</a>). Within the GABA<sub>A</sub> receptor β subunit clade, it is interesting to note that our genome searches did not identify any β1 sequences in pufferfish or chicken genomes. This suggests that the pufferfish and chicken lineages may have independently lost their β1 paralog.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our phylogenetic analyses also indicate that the mammalian-specific GABA<sub>A</sub> subunits ε are derived from the γ4 subunit (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g002">2</a>), which is consistent with their chromosomal organization: Human GABA<sub>A</sub> receptor ε genes are positioned on a location of the X chromosome that corresponds to the position of GABA<sub>A</sub> receptor γ4 genes on human chromosomes 4, 5 and 15 <a href="#pone.0000894-Russek1">[19]</a>, <a href="#pone.0000894-Darlison2">[20]</a>, <a href="#pone.0000894-Sinkkonen1">[27]</a>. The higher rate of evolution of the GABA<sub>A</sub> receptor ε subunits may explain why these subunits have so far only been found in mammals while the receptor γ4 subunits are present in both birds and reptiles <a href="#pone.0000894-Darlison2">[20]</a>. Similarly, chromosomal location <a href="#pone.0000894-Darlison2">[20]</a>, intron-exon organization <a href="#pone.0000894-Simon1">[21]</a> and our phylogenetic analyses are consistent with the suggestion that the mammalian-specific GABA<sub>A</sub> receptor θ subunits are derived from GABA<sub>A</sub> receptor β4 subunits (<a href="#pone-0000894-g001">Fig. 1</a> and <a href="#pone-0000894-g003">3</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although we could only identify a small number of GABA<sub>A</sub> receptor sequences from the <em>C. intestinalis</em> genome, the pufferfish genome contains orthologs to most of the GABA<sub>A</sub> receptor sequences found in mammalian genomes. Furthermore, all the vertebrate genomes queried, including that of pufferfish, contain a single gene coding for most of the 20 GABA<sub>A</sub> receptor subunits (α1-6, γ1-3, ε, δ, π, ρ1-3, β1-4 and θ). However in pufferfish, five of these gene families contain two paralogous gene copies instead of a single copy. Three of these pairs (pufferfish α2, β4 and ρ1) are composed of very similar genes that have undergone recent duplication events. It would be interesting to test experimentally whether the subunits encoded by these gene copies have similar functions. Examples of recent gene duplication events show that it is not necessarily the case: receptors for proglucagon-derived peptides for instance exhibit differences in ligand-binding capabilities between recent paralogs <a href="#pone.0000894-Irwin1">[42]</a>. The presence of two paralogous gene copies in some of the GABA<sub>A</sub> receptor subunit families found in the pufferfish genome is consistent with a complete genome duplication event that occurred in the pufferfish lineage <a href="#pone.0000894-Hoegg1">[43]</a>, <a href="#pone.0000894-Vandepoele1">[44]</a>. Further studies should elucidate whether duplicated GABA receptor subunits have a functional significance for GABAergic transmission in fish.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Our codon analyses showed that both GABA<sub>A</sub> receptor subunits θ and ε experienced positive selection and relaxed constraints, respectively (<a href="#pone-0000894-t001">Table 1</a>–<a href="#pone-0000894-t002">2</a>). We further identified the putative amino acid sites that may be responsible for the functional divergence of subunit θ (<a href="#pone-0000894-t001">Table 1</a> and <a href="#pone-0000894-g005">Fig. 5</a>). One position in subunit θ (274) was consistently identified as being under positive selection by different models. This position is located in putative transmenbrane segment M1, which, like M3, is though to interact with their neighboring subunits in the pentameric receptor <a href="#pone.0000894-Corringer1">[34]</a>. The functional significance both of this location and of the adaptive substitution by a larger and aromatic residue at this position in subunit θ are unclear. Yet, positive selection acting on the sites in the N-terminal domain might have affected the ligand-binding affinity of GABA agonists and or antagonists, and maybe also allosteric transitions between different conformational states, that are the functions associated to this domain in nicotinic receptors <a href="#pone.0000894-Corringer1">[34]</a>. On the other hand, positive selection acting on the C-terminal domain might have affected the sensitivity of the receptor, a property that is determined by the five loops constituting the pentameric receptor, at least in nicotinic receptors <a href="#pone.0000894-Corringer1">[34]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Relaxed constraints in the ε subunit would be consistent with the observation that rodent ε subunit have acquired an unusually large insertion of 483 amino acids in their second exon <a href="#pone.0000894-Sinkkonen1">[27]</a>. These subunits are expressed in the CNS, and are less abundant than other subunits <a href="#pone.0000894-Whiting1">[11]</a>. In rats, the expression of ε subunits is associated with peptidergic neurons, such as those producing orexin, oxytocin, and gonadotropin-releasing hormone (GnRH). This suggests that GABA<sub>A</sub> receptors with ε subunits might have a role in neuroendocrine function, such as that involved in the control of feeding and reproduction <a href="#pone.0000894-Moragues1">[45]</a>. A recent study showed that ε subunits could increase GABA sensitivity up to 100-fold in <em>Xenopus</em> oocytes <a href="#pone.0000894-Ranna1">[46]</a>. It was also shown that receptors containing the ε subunit could be insensitive to the GABA receptor modulators pregnanolone and pentobarbital <a href="#pone.0000894-Ranna1">[46]</a>. Interestingly, these GABA receptor subtypes are predominant in the locus coeruleus, a nucleus in the brain stem that contains a large population of noradrenergic positive neurons. When this region is lesioned in rats, there is a disruption in the preovulatory surge of luteinizing hormone (LH), follicle-stimulating hormone (FSH), and prolactin <a href="#pone.0000894-AnselmoFranci1">[47]</a> and a significant reduction in circulating plasma LH <a href="#pone.0000894-Rocha1">[48]</a>. Further studies should assess the role of the GABA<sub>A</sub> receptor θ and ε subunits in this neuroendocrine pathway. Should these subunits be involved in this pathway, our results would suggest that the rapid divergence of the GABA<sub>A</sub> receptor θ and ε subunits played a role in the evolution of the neuroendocrine system in mammals. Recent studies utilizing point-mutations have shown that single amino acid changes in a GABA receptor subunit will have dramatic effects on the kinetics of the receptor <a href="#pone.0000894-Derry1">[49]</a>, <a href="#pone.0000894-Newell1">[50]</a>. Future studies are needed to determine whether or not these amino acid changes confer significant alterations in GABA receptor kinetics and function.</p>

<h4>Conclusions</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>To conclude, our results show that (1) the two major clades of ionotropic GABA receptors arose before the split from urochordates, (2) the GABA<sub>A</sub> receptor family evolved by both gains and losses of subtypes (e.g., teleost β4, chicken γ4, mammalian ε and θ) and (3) the function of the GABA receptor subunits might have changed adaptively in the mammalian-specific GABA<sub>A</sub> subunit θ, while relaxed constraints acted on subunit ε. These changes of selective regime might have played a role in the evolution of neuroendocrine functions controlling feeding and reproduction in mammals. We caution however that further research should be performed to experimentally test these functional divergence hypotheses.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Genome database searches</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Homologous gene queries were performed on the National Center for Biotechnology Information (NCBI; <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) server. BLASTn and BLASTp searches <a href="#pone.0000894-Altschul1">[51]</a> were used to find homologous genes of full-length GABA receptor subunits α1-6, β1-3, γ1-3, δ, ε, θ, and π present in the completed rat genome project. The <em>Ciona intestinalis</em> sequences were obtained from the Joint Genome Institute (<a href="http://genome.jgipsf.org/ciona4/ciona4.home.html">http://genome.jgipsf.org/ciona4/ciona4.h​ome.html</a>). Sequence alignments are available as supplementary <a href="#pone.0000894.s001">Text S1</a> and <a href="#pone.0000894.s002">S2</a>.</p>


<h4>GABA receptor phylogeny</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>We obtained and aligned a data set of 81 homologous protein sequences of the GABA receptor subunits queried above. This data set contains almost exclusively complete protein sequences with 997 aligned amino acid positions; this alignment is much longer than the actual length of GABA receptors (ca. 470 amino acids) because of the presence of large indels in some specific subunits. All aligned positions were conserved to help tease apart highly conserved proteins. To test for robustness of these analyses, we increased the species sampling of receptors involved in benzodiazepine binding (BZ+: α, γ, ε) and those that are not (BZ–: ρ, β, δ, θ, and π) <a href="#pone.0000894-Ortells1">[3]</a>, <a href="#pone.0000894-Xue1">[23]</a>. Note that this BZ+/BZ– classification of subunits is somewhat artificial as two subunits are actually required to form a benzodiazepine sites <a href="#pone.0000894-Whiting1">[11]</a>. We sampled 55 chordates including sea squirt and vertebrates for the BZ+ GABA<sub>A</sub> receptor protein sequences (1,007 aligned amino acid positions) and 55 chordates including sea squirt and vertebrate BZ– protein sequences (795 aligned amino acid positions). Again, discrepancies between actual protein lengths and length of the alignment reflect the presence of indels.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>Phylogenetic trees were inferred using both the maximum likelihood approach as implemented in PHYML 2.4.4 <a href="#pone.0000894-Guindon1">[52]</a> and the Bayesian approach implemented in MrBayes 3.1.1 <a href="#pone.0000894-Ronquist1">[53]</a>. In both approaches, the 81-sequence data set was rooted with acetylcholine receptor protein sequences <a href="#pone.0000894-Ortells1">[3]</a>, <a href="#pone.0000894-Xue1">[23]</a>. Based on the results from the analysis of this data set (<a href="#pone-0000894-g001">Fig. 1</a>), the BZ+ and BZ– data sets (<a href="#pone-0000894-g002">Fig. 2</a> and <a href="#pone-0000894-g003">3</a>, respectively) were rooted with the lobster β1 (accession number AY098945) because this sequence provides a closer outgroup to both data sets (and that closer outgroups minimize possible errors in phylogenetic trees). ProtTest <a href="#pone.0000894-Abascal1">[54]</a> was used to determine, based on the Akaike Information Criterion (min<em>AIC</em>), that the JTT+I+Γ model of amino acid substitution <a href="#pone.0000894-Jones1">[55]</a>, <a href="#pone.0000894-Yang4">[56]</a> was the most appropriate model of evolution for this data set (<em>pˆ<sub>I</sub></em> = 0.01; <em>αˆ</em> = 0.819); the second to most appropriate model was JTT+Γ (Δ<em>AIC</em> = 48.81), while the second to most appropriate rate matrix was WAG (+I+Γ: Δ<em>AIC</em> = 444.74; +Γ: Δ<em>AIC</em> = 523.11). Among-site rate variation modeled by a discrete Γ distribution with eight rate categories. This maximum likelihood model was fitted independently to each data set.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3"></a><p>Because model choice as performed by ProtTest can lead to underestimating uncertainty, the maximum likelihood analyses were complemented by a Bayesian analysis that integrates over different models of evolution: rather than selecting an a priori empirical model of substitution, a reversible-jump Markov chain Monte Carlo (RJ-MCMC) was constructed to integrate over model uncertainty <a href="#pone.0000894-Huelsenbeck1">[57]</a>. Models with equal prior probability were: Poisson, JTT, Dayhoff, MtREV, MtMAMM, WAG, rtREV, CpREV, VT and BLOSUM62 as described and implemented in MrBayes <a href="#pone.0000894-Ronquist1">[53]</a>. Among-site rate variation was modeled using a discrete Γ distribution with five rate categories <a href="#pone.0000894-Yang4">[56]</a> plus a class of invariable sites. Under this mixed model of protein evolution, four independent RJ-MCMC samplers were run for ten million steps using different starting values. To decrease autocorrelation of the samples taken from the target distribution, steps along the chain were sampled every 1,000 accepted steps, a method known as thinning (e.g., <a href="#pone.0000894-Yang5">[58]</a>). To improve mixing, each sampler consisted of four chains, three of which were heated to different temperatures (e.g., <a href="#pone.0000894-Yang4">[56]</a>). By raising the likelihood function to a power &lt;1, deep valleys of the likelihood surface become shallow, which facilitates their crossing by the sampler and hence improves the ability of the chain to explore the entire parameter space efficiently, in proportion of the target density. Sampling was realized from the non-heated chain. Burn-in length and convergence of the samplers were checked by plotting time series plots and checking that average standard deviations of split frequencies were lower than 0.01 <a href="#pone.0000894-Ronquist1">[53]</a>. The chains appeared to have converged by 10,000 steps; to be conservative, 100,000 steps were discarded as a burn-in.</p>


<h4>Analysis of selective pressures</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Codon data were obtained and split into two smaller data sets, one for each of the paralogous clade of interest: θ and ε. Each data set contained 13 sequences human (<em>Homo sapiens</em>), rat (<em>Rattus norvegicus</em>) and mouse (<em>Mus musculus</em>) copies of θ and ε paralogs, as well as the corresponding members of the group in which this clade was located: β for θ; γ for ε. Each tree was rooted by the closest human paralog (δ and α, respectively). Sites with ambiguous data were removed. A statistical approach was then used to detect functional divergence at individual codon sites within the pre-specified branches based on a procedure similar to that by Bielawski and Yang <a href="#pone.0000894-Bielawski1">[59]</a>: for each paralog the null hypothesis H<sub>0</sub> was that of no variable selective pressure among branches and among sites (<a href="#pone-0000894-g004">Fig. 4</a>). This hypothesis was contrasted by means of likelihood ratio tests against three potential alternative modes of evolution. In H<sub>1</sub>, only the branch leading to the θ or to the ε clade was allowed to evolve at a different <em>ω</em> rate ratio. This branch (branch number five) is said to have a foreground rate, while all the other branches have the same background rate. In a second model, H<sub>2</sub>, the θ or the ε clade was allowed to evolve at a rate that differs from both the foreground and the background rates. Finally in H<sub>3</sub>, each paralog was allowed to evolve at its own rate after duplication.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>In some cases, only a few sites are affected by an episode of positive selection within a given branch. A more powerful approach to detecting sites undergoing positive selection in such cases is to allow <em>ω</em> to vary among sites within the branch of interest using the “test of positive selection” or “test 2” described in <a href="#pone.0000894-Zhang1">[29]</a> and implemented in PAML version 3.15 <a href="#pone.0000894-Yang6">[60]</a>. These “branch-site” models allow the <em>ω</em> ratio to vary both among branches and among the sites in the foreground branch. Model A has four classes of sites: class 0 includes conserved codons with 0&lt;<em>ω</em><sub>5</sub><sup>(0)</sup>&lt;1 is estimated from the data; class 1 includes codons that evolve neutrally (<em>ω</em><sub>5</sub><sup>(1)</sup> = 1); classes 2a and 2b include codons that are conserved or neutral on the background branches, but are under positive selection on the foreground branch, with <em>ω</em><sub>5</sub><sup>(2)</sup>&gt;1 estimated from the data. The “test of positive selection” compares this model against a simpler (null) model, that does not allow for positive selection (<em>ω</em><sub>5</sub><sup>(2)</sup> = 1), so that this model estimates one fewer parameters than its alternative. To be conservative, we used <em>χ</em><sub>1</sub><sup>2</sup> as an approximation to the distribution of the test statistic under the null <a href="#pone.0000894-Zhang1">[29]</a>. Robustness of these branch-site codon models was assessed by using a second test based on “model B” as described in <a href="#pone.0000894-Zhang1">[29]</a>, which is identical to model A described above except that <em>ω</em><sub>5</sub><sup>(1)</sup> is free to vary. Model B is compared to model M3 <a href="#pone.0000894-Yang2">[35]</a> which is a site model with two discrete rate categories. Sites putatively under adaptive evolution in the “test of positive selection” were identified by the Bayes empirical Bayes (BEB) procedure <a href="#pone.0000894-Yang3">[37]</a>, that improves on the naïve empirical Bayes approach (NEB: <a href="#pone.0000894-Nielsen2">[61]</a>) by accommodating uncertainties of the maximum likelihood estimates; with “model B”, only BEB is implemented to assign individual sites to rate categories. All codon models were run at least twice to check convergence. Ancestral amino acid residues were determined by parsimony (for <a href="#pone-0000894-t001">Tables 1</a> and <a href="#pone-0000894-t002">2</a>).</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>Three-dimensional (3D) structure predictions were carried out with 3D-JIGSAW <a href="#pone.0000894-Bates1">[33]</a> and with SWISS-MODEL <a href="#pone.0000894-Arnold1">[31]</a>. Both tools predict structures by homology modeling, a technique that can be decomposed into five steps: (i) a query or parts thereof are aligned to one or more template protein sequences, as determined by BLASTp searches; templates must have a resolved 3D structure; (ii) these 3D segments are put together to form a preliminary model; (iii) side chains are adjusted to account for substitutions between the query and the templates; (iv) the model is examined for potential collisions between atoms; finally (v) the model is refined by limited energy minimization. The major inaccuracies of homology modeling usually stem from two sources: low sequence similarity and improper template selection <a href="#pone.0000894-Venclovas1">[62]</a>. These two sources of inaccuracies were assessed as explained in the text. Structural alignments of the backbones of the models and computation of root mean square deviations or RMSDs were performed with DeepView, which is available through the ExPASy Web site.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000894"></div><a name="pone.0000894.s001" id="pone.0000894.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000894.s001">Text S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Amino acid alignment of the 81 full-length GABA receptor subunits.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000894.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.08 MB TXT)</p>
<a name="pone.0000894.s002" id="pone.0000894.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000894.s002">Text S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Codon alignment of the GABA receptor subunits β, θ, γ and ɛ for human.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000894.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.09 MB TXT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>The authors want to thank Jason Stajich and two anonymous reviewers for their comments, which improved the manuscript considerably.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: SA GD VT. Performed the experiments: CM JC. Analyzed the data: SA CM GD. Wrote the paper: SA CM GD JC VT.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000894-Ohta1" id="pone.0000894-Ohta1"></a>Ohta T (1989) Role of gene duplication in evolution. Genome  31: 304–310.  <ul class="find" data-citedArticleID="992164" data-doi="10.1139/g89-048"><li><a href="http://dx.doi.org/10.1139/g89-048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+gene+duplication+in+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+gene+duplication+in+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000894-Shimeld1" id="pone.0000894-Shimeld1"></a>Shimeld SM, Holland PW (2000) Vertebrate innovations. Proc Natl Acad Sci USA  97: 4449–4452.  <ul class="find" data-citedArticleID="992182" data-doi="10.1073/pnas.97.9.4449"><li><a href="http://dx.doi.org/10.1073/pnas.97.9.4449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Vertebrate+innovations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Vertebrate+innovations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000894-Ortells1" id="pone.0000894-Ortells1"></a>Ortells MO, Lunt GG (1995) Evolutionary history of the ligand-gated ion-channel superfamily of receptors. Trends Neurosci  18: 121–127.  <ul class="find" data-citedArticleID="992170" data-doi="10.1016/0166-2236(95)93887-4"><li><a href="http://dx.doi.org/10.1016/0166-2236(95)93887-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+history+of+the+ligand-gated+ion-channel+superfamily+of+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+history+of+the+ligand-gated+ion-channel+superfamily+of+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000894-Kellogg1" id="pone.0000894-Kellogg1"></a>Kellogg CK (1998) Early developmental modulation of GABA<sub>A</sub> receptor function. Influence on adaptive responses. Perspect Dev Neurobiol  5: 219–234.  <ul class="find" data-citedArticleID="992148"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Early+developmental+modulation+of+GABAA+receptor+function.+Influence+on+adaptive+responses.&amp;auth=&amp;atitle=Early+developmental+modulation+of+GABAA+receptor+function.+Influence+on+adaptive+responses." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Early+developmental+modulation+of+GABAA+receptor+function.+Influence+on+adaptive+responses." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Early+developmental+modulation+of+GABAA+receptor+function.+Influence+on+adaptive+responses.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000894-Arvat1" id="pone.0000894-Arvat1"></a>Arvat E, Giordano R, Grottoli S, Ghigo E (2002) Benzodiazepines and anterior pituitary function. J Endocrinol Invest  25: 735–747.  <ul class="find" data-citedArticleID="992098" data-doi="10.1007/bf03345110"><li><a href="http://dx.doi.org/10.1007/bf03345110" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Benzodiazepines+and+anterior+pituitary+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Benzodiazepines+and+anterior+pituitary+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000894-Myhrer1" id="pone.0000894-Myhrer1"></a>Myhrer T (2003) Neurotransmitter systems involved in learning and memory in the rat: a meta-analysis based on studies of four behavioral tasks. Brain Res Brain Res Rev  41: 268–287.  <ul class="find" data-citedArticleID="992156" data-doi="10.1016/s0165-0173(02)00268-0"><li><a href="http://dx.doi.org/10.1016/s0165-0173(02)00268-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neurotransmitter+systems+involved+in+learning+and+memory+in+the+rat%3A+a+meta-analysis+based+on+studies+of+four+behavioral+tasks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neurotransmitter+systems+involved+in+learning+and+memory+in+the+rat%3A+a+meta-analysis+based+on+studies+of+four+behavioral+tasks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000894-Eriksson1" id="pone.0000894-Eriksson1"></a>Eriksson KS, Panula P (1994) gamma-Aminobutyric acid in the nervous system of a planarian. J Comp Neurol  345: 528–536.  <ul class="find" data-citedArticleID="992126" data-doi="10.1002/cne.903450405"><li><a href="http://dx.doi.org/10.1002/cne.903450405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=gamma-Aminobutyric+acid+in+the+nervous+system+of+a+planarian." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22gamma-Aminobutyric+acid+in+the+nervous+system+of+a+planarian.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000894-Orona1" id="pone.0000894-Orona1"></a>Orona E, Battelle BA, Ache BW (1990) Immunohistochemical and biochemical evidence for the putative inhibitory neurotransmitters histamine and GABA in lobster olfactory lobes. J Comp Neurol  294: 633–646.  <ul class="find" data-citedArticleID="992168" data-doi="10.1002/cne.902940410"><li><a href="http://dx.doi.org/10.1002/cne.902940410" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunohistochemical+and+biochemical+evidence+for+the+putative+inhibitory+neurotransmitters+histamine+and+GABA+in+lobster+olfactory+lobes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunohistochemical+and+biochemical+evidence+for+the+putative+inhibitory+neurotransmitters+histamine+and+GABA+in+lobster+olfactory+lobes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000894-Darlison1" id="pone.0000894-Darlison1"></a>Darlison MG (1992) Invertebrate GABA and glutamate receptors: molecular biology reveals predictable structures but some unusual pharmacologies. Trends Neurosci  15: 469–474.  <ul class="find" data-citedArticleID="992118" data-doi="10.1016/0166-2236(92)90091-l"><li><a href="http://dx.doi.org/10.1016/0166-2236(92)90091-l" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Invertebrate+GABA+and+glutamate+receptors%3A+molecular+biology+reveals+predictable+structures+but+some+unusual+pharmacologies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Invertebrate+GABA+and+glutamate+receptors%3A+molecular+biology+reveals+predictable+structures+but+some+unusual+pharmacologies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000894-Anadon1" id="pone.0000894-Anadon1"></a>Anadon R, Adrio F, Rodriguez-Moldes I (1998) Distribution of GABA immunoreactivity in the central and peripheral nervous system of amphioxus (<em>Branchiostoma lanceolatum</em> Pallas). J Comp Neurol  401: 293–307.  <ul class="find" data-citedArticleID="992092" data-doi="10.1002/(sici)1096-9861(19981123)401:3<293::aid-cne1&gt;3.3.co;2-6"><li><a href="http://dx.doi.org/10.1002/(sici)1096-9861(19981123)401:3<293::aid-cne1&gt;3.3.co;2-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distribution+of+GABA+immunoreactivity+in+the+central+and+peripheral+nervous+system+of+amphioxus+%28Branchiostoma+lanceolatum+Pallas%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distribution+of+GABA+immunoreactivity+in+the+central+and+peripheral+nervous+system+of+amphioxus+%28Branchiostoma+lanceolatum+Pallas%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000894-Whiting1" id="pone.0000894-Whiting1"></a>Whiting PJ (1999) The GABA-A receptor gene family: new targets for therapeutic intervention. Neurochem Int  34: 387–390.  <ul class="find" data-citedArticleID="992194" data-doi="10.1016/s0197-0186(99)00048-0"><li><a href="http://dx.doi.org/10.1016/s0197-0186(99)00048-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+GABA-A+receptor+gene+family%3A+new+targets+for+therapeutic+intervention." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+GABA-A+receptor+gene+family%3A+new+targets+for+therapeutic+intervention.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000894-Rabow1" id="pone.0000894-Rabow1"></a>Rabow LE, Russek SJ, Farb DH (1995) From ion currents to genomic analysis: recent advances in GABA<sub>A</sub> receptor research. Synapse  21: 189–274.  <ul class="find" data-citedArticleID="992172" data-doi="10.1002/syn.890210302"><li><a href="http://dx.doi.org/10.1002/syn.890210302" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+ion+currents+to+genomic+analysis%3A+recent+advances+in+GABAA+receptor+research." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+ion+currents+to+genomic+analysis%3A+recent+advances+in+GABAA+receptor+research.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000894-Costa1" id="pone.0000894-Costa1"></a>Costa E, Auta J, Grayson DR, Matsumoto K, Pappas GD, et al.  (2002) GABA<sub>A</sub> receptors and benzodiazepines: a role for dendritic resident subunit mRNAs. Neuropharmacology  43: 925–937.  <ul class="find" data-citedArticleID="992112" data-doi="10.1016/s0028-3908(02)00199-5"><li><a href="http://dx.doi.org/10.1016/s0028-3908(02)00199-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABAA+receptors+and+benzodiazepines%3A+a+role+for+dendritic+resident+subunit+mRNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABAA+receptors+and+benzodiazepines%3A+a+role+for+dendritic+resident+subunit+mRNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000894-Bormann1" id="pone.0000894-Bormann1"></a>Bormann J (2000) The ‘ABC’ of GABA receptors. Trends Pharmacol Sci  21: 16–19.  <ul class="find" data-citedArticleID="992104" data-doi="10.1016/s0165-6147(99)01413-3"><li><a href="http://dx.doi.org/10.1016/s0165-6147(99)01413-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+%26%23x2018%3BABC%26%23x2019%3B+of+GABA+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+%26%23x2018%3BABC%26%23x2019%3B+of+GABA+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000894-Couve1" id="pone.0000894-Couve1"></a>Couve A, Moss SJ, Pangalos MN (2000) GABA<sub>B</sub> receptors: a new paradigm in G protein signaling. Mol Cell Neurosci  16: 296–312.  <ul class="find" data-citedArticleID="992114" data-doi="10.1006/mcne.2000.0908"><li><a href="http://dx.doi.org/10.1006/mcne.2000.0908" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABAB+receptors%3A+a+new+paradigm+in+G+protein+signaling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABAB+receptors%3A+a+new+paradigm+in+G+protein+signaling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000894-Bormann2" id="pone.0000894-Bormann2"></a>Bormann J, Feigenspan A (1995) GABA<sub>C</sub> receptors. Trends Neurosci  18: 515–519.  <ul class="find" data-citedArticleID="992106" data-doi="10.1016/0166-2236(95)98370-e"><li><a href="http://dx.doi.org/10.1016/0166-2236(95)98370-e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABAC+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABAC+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000894-Koulen1" id="pone.0000894-Koulen1"></a>Koulen P, Brandstatter JH, Kroger S, Enz R, Bormann J, et al.  (1997) Immunocytochemical localization of the GABA(C) receptor rho subunits in the cat, goldfish, and chicken retina. J Comp Neurol  380: 520–532.  <ul class="find" data-citedArticleID="992152" data-doi="10.1002/(sici)1096-9861(19970421)380:4<520::aid-cne8&gt;3.0.co;2-3"><li><a href="http://dx.doi.org/10.1002/(sici)1096-9861(19970421)380:4<520::aid-cne8&gt;3.0.co;2-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Immunocytochemical+localization+of+the+GABA%28C%29+receptor+rho+subunits+in+the+cat%2C+goldfish%2C+and+chicken+retina." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Immunocytochemical+localization+of+the+GABA%28C%29+receptor+rho+subunits+in+the+cat%2C+goldfish%2C+and+chicken+retina.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000894-Johnston1" id="pone.0000894-Johnston1"></a>Johnston GA (2002) Medicinal chemistry and molecular pharmacology of GABA(C) receptors. Curr Top Med Chem  2: 903–913.  <ul class="find" data-citedArticleID="992142" data-doi="10.2174/1568026023393453"><li><a href="http://dx.doi.org/10.2174/1568026023393453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Medicinal+chemistry+and+molecular+pharmacology+of+GABA%28C%29+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Medicinal+chemistry+and+molecular+pharmacology+of+GABA%28C%29+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000894-Russek1" id="pone.0000894-Russek1"></a>Russek SJ (1999) Evolution of GABA(A) receptor diversity in the human genome. Gene  227: 213–222.  <ul class="find" data-citedArticleID="992180" data-doi="10.1016/s0378-1119(98)00594-0"><li><a href="http://dx.doi.org/10.1016/s0378-1119(98)00594-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+GABA%28A%29+receptor+diversity+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+GABA%28A%29+receptor+diversity+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000894-Darlison2" id="pone.0000894-Darlison2"></a>Darlison MG, Pahal I, Thode C (2005) Consequences of the evolution of the GABA(A) receptor gene family. Cell Mol Neurobiol  25: 607–624.  <ul class="find" data-citedArticleID="992120" data-doi="10.1007/s10571-005-4004-4"><li><a href="http://dx.doi.org/10.1007/s10571-005-4004-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Consequences+of+the+evolution+of+the+GABA%28A%29+receptor+gene+family." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Consequences+of+the+evolution+of+the+GABA%28A%29+receptor+gene+family.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000894-Simon1" id="pone.0000894-Simon1"></a>Simon J, Wakimoto H, Fujita N, Lalande M, Barnard EA (2004) Analysis of the set of GABA(A) receptor genes in the human genome. J Biol Chem  279: 41422–41435.  <ul class="find" data-citedArticleID="992184" data-doi="10.1074/jbc.m401354200"><li><a href="http://dx.doi.org/10.1074/jbc.m401354200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+set+of+GABA%28A%29+receptor+genes+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+set+of+GABA%28A%29+receptor+genes+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000894-Cutting1" id="pone.0000894-Cutting1"></a>Cutting GR, Curristin S, Zoghbi H, O'Hara B, Seldin MF, et al.  (1992) Identification of a putative gamma-aminobutyric acid (GABA) receptor subunit rho2 cDNA and colocalization of the genes encoding rho2 (GABRR2) and rho1 (GABRR1) to human chromosome 6q14-q21 and mouse chromosome 4. Genomics  12: 801–806.  <ul class="find" data-citedArticleID="992116" data-doi="10.1016/0888-7543(92)90312-g"><li><a href="http://dx.doi.org/10.1016/0888-7543(92)90312-g" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+putative+gamma-aminobutyric+acid+%28GABA%29+receptor+subunit+rho2+cDNA+and+colocalization+of+the+genes+encoding+rho2+%28GABRR2%29+and+rho1+%28GABRR1%29+to+human+chromosome+6q14-q21+and+mouse+chromosome+4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+putative+gamma-aminobutyric+acid+%28GABA%29+receptor+subunit+rho2+cDNA+and+colocalization+of+the+genes+encoding+rho2+%28GABRR2%29+and+rho1+%28GABRR1%29+to+human+chromosome+6q14-q21+and+mouse+chromosome+4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000894-Xue1" id="pone.0000894-Xue1"></a>Xue H (1998) Identification of major phylogenetic branches of inhibitory ligand-gated channel receptors. J Mol Evol  47: 323–333.  <ul class="find" data-citedArticleID="992196" data-doi="10.1007/pl00006390"><li><a href="http://dx.doi.org/10.1007/pl00006390" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+major+phylogenetic+branches+of+inhibitory+ligand-gated+channel+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+major+phylogenetic+branches+of+inhibitory+ligand-gated+channel+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000894-Dent1" id="pone.0000894-Dent1"></a>Dent JA (2006) Evidence for a diverse Cys-loop ligand-gated ion channel superfamily in early bilateria. J Mol Evol  62: 523–535.  <ul class="find" data-citedArticleID="992122" data-doi="10.1007/s00239-005-0018-2"><li><a href="http://dx.doi.org/10.1007/s00239-005-0018-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+diverse+Cys-loop+ligand-gated+ion+channel+superfamily+in+early+bilateria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+diverse+Cys-loop+ligand-gated+ion+channel+superfamily+in+early+bilateria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000894-Tsang1" id="pone.0000894-Tsang1"></a>Tsang SY, Ng SK, Xu Z, Xue H (2007) The evolution of GABA<sub>A</sub> receptor-like genes. Mol Biol Evol  24: 599–610.  <ul class="find" data-citedArticleID="992188" data-doi="10.1093/molbev/msl188"><li><a href="http://dx.doi.org/10.1093/molbev/msl188" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolution+of+GABAA+receptor-like+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolution+of+GABAA+receptor-like+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000894-Galperin1" id="pone.0000894-Galperin1"></a>Galperin MY, Koonin EV (1999) Functional genomics and enzyme evolution. Homologous and analogous enzymes encoded in microbial genomes. Genetica  106: 159–170.  <ul class="find" data-citedArticleID="992128"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Functional+genomics+and+enzyme+evolution.+Homologous+and+analogous+enzymes+encoded+in+microbial+genomes.&amp;auth=&amp;atitle=Functional+genomics+and+enzyme+evolution.+Homologous+and+analogous+enzymes+encoded+in+microbial+genomes." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Functional+genomics+and+enzyme+evolution.+Homologous+and+analogous+enzymes+encoded+in+microbial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Functional+genomics+and+enzyme+evolution.+Homologous+and+analogous+enzymes+encoded+in+microbial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000894-Sinkkonen1" id="pone.0000894-Sinkkonen1"></a>Sinkkonen ST, Hanna MC, Kirkness EF, Korpi ER (2000) GABA(A) receptor epsilon and theta subunits display unusual structural variation between species and are enriched in the rat locus ceruleus. J Neurosci  20: 3588–3595.  <ul class="find" data-citedArticleID="992186"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=GABA%28A%29+receptor+epsilon+and+theta+subunits+display+unusual+structural+variation+between+species+and+are+enriched+in+the+rat+locus+ceruleus.&amp;auth=&amp;atitle=GABA%28A%29+receptor+epsilon+and+theta+subunits+display+unusual+structural+variation+between+species+and+are+enriched+in+the+rat+locus+ceruleus." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABA%28A%29+receptor+epsilon+and+theta+subunits+display+unusual+structural+variation+between+species+and+are+enriched+in+the+rat+locus+ceruleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABA%28A%29+receptor+epsilon+and+theta+subunits+display+unusual+structural+variation+between+species+and+are+enriched+in+the+rat+locus+ceruleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000894-Goldman1" id="pone.0000894-Goldman1"></a>Goldman N, Yang Z (1994) A codon-based model of nucleotide substitution for protein-coding DNA sequences. Mol Biol Evol  11: 725–736.  <ul class="find" data-citedArticleID="992130"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences.&amp;auth=&amp;atitle=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+codon-based+model+of+nucleotide+substitution+for+protein-coding+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000894-Zhang1" id="pone.0000894-Zhang1"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 2472–2479.  <ul class="find" data-citedArticleID="992210" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000894-Yang1" id="pone.0000894-Yang1"></a>Yang Z, Nielsen R (2002) Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages. Mol Biol Evol  19: 908–917.  <ul class="find" data-citedArticleID="992198" data-doi="10.1093/oxfordjournals.molbev.a004148"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000894-Arnold1" id="pone.0000894-Arnold1"></a>Arnold K, Bordoli L, Kopp J, Schwede T (2006) The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. Bioinformatics  22: 195–201.  <ul class="find" data-citedArticleID="992096" data-doi="10.1093/bioinformatics/bti770"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti770" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SWISS-MODEL+workspace%3A+a+web-based+environment+for+protein+structure+homology+modelling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SWISS-MODEL+workspace%3A+a+web-based+environment+for+protein+structure+homology+modelling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000894-Brejc1" id="pone.0000894-Brejc1"></a>Brejc K, van Dijk WJ, Klaassen RV, Schuurmans M, van Der Oost J, et al.  (2001) Crystal structure of an ACh-binding protein reveals the ligand-binding domain of nicotinic receptors. Nature  411: 269–276.  <ul class="find" data-citedArticleID="992108" data-doi="10.1038/35077011"><li><a href="http://dx.doi.org/10.1038/35077011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+an+ACh-binding+protein+reveals+the+ligand-binding+domain+of+nicotinic+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+an+ACh-binding+protein+reveals+the+ligand-binding+domain+of+nicotinic+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000894-Bates1" id="pone.0000894-Bates1"></a>Bates PA, Kelley LA, MacCallum RM, Sternberg MJ (2001) Enhancement of protein modeling by human intervention in applying the automatic programs 3D-JIGSAW and 3D-PSSM. Proteins Suppl  5: 39–46.  <ul class="find" data-citedArticleID="992100" data-doi="10.1002/prot.1168"><li><a href="http://dx.doi.org/10.1002/prot.1168" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Enhancement+of+protein+modeling+by+human+intervention+in+applying+the+automatic+programs+3D-JIGSAW+and+3D-PSSM." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Enhancement+of+protein+modeling+by+human+intervention+in+applying+the+automatic+programs+3D-JIGSAW+and+3D-PSSM.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000894-Corringer1" id="pone.0000894-Corringer1"></a>Corringer PJ, Le Novere N, Changeux JP (2000) Nicotinic receptors at the amino acid level. Annu Rev Pharmacol Toxicol  40: 431–458.  <ul class="find" data-citedArticleID="992110" data-doi="10.1146/annurev.pharmtox.40.1.431"><li><a href="http://dx.doi.org/10.1146/annurev.pharmtox.40.1.431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nicotinic+receptors+at+the+amino+acid+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nicotinic+receptors+at+the+amino+acid+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000894-Yang2" id="pone.0000894-Yang2"></a>Yang Z, Nielsen R, Goldman N, Pedersen AM (2000) Codon-substitution models for heterogeneous selection pressure at amino acid sites. Genetics  155: 431–449.  <ul class="find" data-citedArticleID="992200"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.&amp;auth=&amp;atitle=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+heterogeneous+selection+pressure+at+amino+acid+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000894-KosakovskyPond1" id="pone.0000894-KosakovskyPond1"></a>Kosakovsky Pond SL, Frost SD (2005) Not so different after all: a comparison of methods for detecting amino acid sites under selection. Mol Biol Evol  22: 1208–1222.  <ul class="find" data-citedArticleID="992150" data-doi="10.1093/molbev/msi105"><li><a href="http://dx.doi.org/10.1093/molbev/msi105" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Not+so+different+after+all%3A+a+comparison+of+methods+for+detecting+amino+acid+sites+under+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000894-Yang3" id="pone.0000894-Yang3"></a>Yang Z, Wong WS, Nielsen R (2005) Bayes empirical Bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="992202" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+Bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+Bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000894-Okamura1" id="pone.0000894-Okamura1"></a>Okamura Y, Nishino A, Murata Y, Nakajo K, Iwasaki H, et al.  (2005) Comprehensive analysis of the ascidian genome reveals novel insights into the molecular evolution of ion channel genes. Physiol Genomics  22: 269–282.  <ul class="find" data-citedArticleID="992166" data-doi="10.1152/physiolgenomics.00229.2004"><li><a href="http://dx.doi.org/10.1152/physiolgenomics.00229.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comprehensive+analysis+of+the+ascidian+genome+reveals+novel+insights+into+the+molecular+evolution+of+ion+channel+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comprehensive+analysis+of+the+ascidian+genome+reveals+novel+insights+into+the+molecular+evolution+of+ion+channel+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000894-Nielsen1" id="pone.0000894-Nielsen1"></a>Nielsen M, Braestrup C, Squires RF (1978) Evidence for a late evolutionary appearance of brain-specific benzodiazepine receptors: an investigation of 18 vertebrate and 5 invertebrate species. Brain Res  141: 342–346.  <ul class="find" data-citedArticleID="992160" data-doi="10.1016/0006-8993(78)90203-2"><li><a href="http://dx.doi.org/10.1016/0006-8993(78)90203-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+a+late+evolutionary+appearance+of+brain-specific+benzodiazepine+receptors%3A+an+investigation+of+18+vertebrate+and+5+invertebrate+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+a+late+evolutionary+appearance+of+brain-specific+benzodiazepine+receptors%3A+an+investigation+of+18+vertebrate+and+5+invertebrate+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000894-Hebebrand1" id="pone.0000894-Hebebrand1"></a>Hebebrand J, Friedl W, Breidenbach B, Propping P (1987) Phylogenetic comparison of the photoaffinity-labeled benzodiazepine receptor subunits. J Neurochem  48: 1103–1108.  <ul class="find" data-citedArticleID="992134" data-doi="10.1111/j.1471-4159.1987.tb05633.x"><li><a href="http://dx.doi.org/10.1111/j.1471-4159.1987.tb05633.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+comparison+of+the+photoaffinity-labeled+benzodiazepine+receptor+subunits." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+comparison+of+the+photoaffinity-labeled+benzodiazepine+receptor+subunits.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000894-KassSimon1" id="pone.0000894-KassSimon1"></a>Kass-Simon G, Pannaccione A, Pierobon P (2003) GABA and glutamate receptors are involved in modulating pacemaker activity in hydra. Comp Biochem Physiol A Mol Integr Physiol  136: 329–342.  <ul class="find" data-citedArticleID="992146" data-doi="10.1016/s1095-6433(03)00168-5"><li><a href="http://dx.doi.org/10.1016/s1095-6433(03)00168-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABA+and+glutamate+receptors+are+involved+in+modulating+pacemaker+activity+in+hydra." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABA+and+glutamate+receptors+are+involved+in+modulating+pacemaker+activity+in+hydra.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000894-Irwin1" id="pone.0000894-Irwin1"></a>Irwin DM, Wong K (2005) Evolution of new hormone function: loss and gain of a receptor. J Hered  96: 205–211.  <ul class="find" data-citedArticleID="992140" data-doi="10.1093/jhered/esi024"><li><a href="http://dx.doi.org/10.1093/jhered/esi024" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+new+hormone+function%3A+loss+and+gain+of+a+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+new+hormone+function%3A+loss+and+gain+of+a+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000894-Hoegg1" id="pone.0000894-Hoegg1"></a>Hoegg S, Brinkmann H, Taylor JS, Meyer A (2004) Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish. J Mol Evol  59: 190–203.  <ul class="find" data-citedArticleID="992136" data-doi="10.1007/s00239-004-2613-z"><li><a href="http://dx.doi.org/10.1007/s00239-004-2613-z" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Phylogenetic+timing+of+the+fish-specific+genome+duplication+correlates+with+the+diversification+of+teleost+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Phylogenetic+timing+of+the+fish-specific+genome+duplication+correlates+with+the+diversification+of+teleost+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000894-Vandepoele1" id="pone.0000894-Vandepoele1"></a>Vandepoele K, De Vos W, Taylor JS, Meyer A, Van de Peer Y (2004) Major events in the genome evolution of vertebrates: paranome age and size differ considerably between ray-finned fishes and land vertebrates. Proc Natl Acad Sci USA  101: 1638–1643.  <ul class="find" data-citedArticleID="992190" data-doi="10.1073/pnas.0307968100"><li><a href="http://dx.doi.org/10.1073/pnas.0307968100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+events+in+the+genome+evolution+of+vertebrates%3A+paranome+age+and+size+differ+considerably+between+ray-finned+fishes+and+land+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+events+in+the+genome+evolution+of+vertebrates%3A+paranome+age+and+size+differ+considerably+between+ray-finned+fishes+and+land+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000894-Moragues1" id="pone.0000894-Moragues1"></a>Moragues N, Ciofi P, Lafon P, Tramu G, Garret M (2003) GABA<sub>A</sub> receptor epsilon subunit expression in identified peptidergic neurons of the rat hypothalamus. Brain Res  967: 285–289.  <ul class="find" data-citedArticleID="992154" data-doi="10.1016/s0006-8993(02)04270-1"><li><a href="http://dx.doi.org/10.1016/s0006-8993(02)04270-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GABAA+receptor+epsilon+subunit+expression+in+identified+peptidergic+neurons+of+the+rat+hypothalamus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GABAA+receptor+epsilon+subunit+expression+in+identified+peptidergic+neurons+of+the+rat+hypothalamus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000894-Ranna1" id="pone.0000894-Ranna1"></a>Ranna M, Sinkkonen ST, Moykkynen T, Uusi-Oukari M, Korpi ER (2006) Impact of epsilon and theta subunits on pharmacological properties of alpha3beta1 GABA<sub>A</sub> receptors expressed in Xenopus oocytes. BMC Pharmacol  6: 1.  <ul class="find" data-citedArticleID="992174" data-doi="10.1186/1471-2210-6-1"><li><a href="http://dx.doi.org/10.1186/1471-2210-6-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Impact+of+epsilon+and+theta+subunits+on+pharmacological+properties+of+alpha3beta1+GABAA+receptors+expressed+in+Xenopus+oocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Impact+of+epsilon+and+theta+subunits+on+pharmacological+properties+of+alpha3beta1+GABAA+receptors+expressed+in+Xenopus+oocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000894-AnselmoFranci1" id="pone.0000894-AnselmoFranci1"></a>Anselmo-Franci JA, Franci CR, Krulich L, Antunes-Rodrigues J, McCann SM (1997) Locus coeruleus lesions decrease norepinephrine input into the medial preoptic area and medial basal hypothalamus and block the LH, FSH and prolactin preovulatory surge. Brain Res  767: 289–296.  <ul class="find" data-citedArticleID="992094" data-doi="10.1016/s0006-8993(97)00613-6"><li><a href="http://dx.doi.org/10.1016/s0006-8993(97)00613-6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Locus+coeruleus+lesions+decrease+norepinephrine+input+into+the+medial+preoptic+area+and+medial+basal+hypothalamus+and+block+the+LH%2C+FSH+and+prolactin+preovulatory+surge." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Locus+coeruleus+lesions+decrease+norepinephrine+input+into+the+medial+preoptic+area+and+medial+basal+hypothalamus+and+block+the+LH%2C+FSH+and+prolactin+preovulatory+surge.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000894-Rocha1" id="pone.0000894-Rocha1"></a>Rocha AA, Guerra-Sa R, Silveira NA, Anselmo-Franci JA, Franci CR (2006) Neuropeptide Y in the medial basal hypothalamus and medial preoptic area during the induction of LH surge may be controlled by locus coeruleus. Neuropeptides  40: 57–63.  <ul class="find" data-citedArticleID="992176" data-doi="10.1016/j.npep.2005.10.005"><li><a href="http://dx.doi.org/10.1016/j.npep.2005.10.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Neuropeptide+Y+in+the+medial+basal+hypothalamus+and+medial+preoptic+area+during+the+induction+of+LH+surge+may+be+controlled+by+locus+coeruleus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Neuropeptide+Y+in+the+medial+basal+hypothalamus+and+medial+preoptic+area+during+the+induction+of+LH+surge+may+be+controlled+by+locus+coeruleus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000894-Derry1" id="pone.0000894-Derry1"></a>Derry JM, Dunn SM, Davies M (2004) Identification of a residue in the gamma-aminobutyric acid type A receptor alpha subunit that differentially affects diazepam-sensitive and -insensitive benzodiazepine site binding. J Neurochem  88: 1431–1438.  <ul class="find" data-citedArticleID="992124" data-doi="10.1046/j.1471-4159.2003.02264.x"><li><a href="http://dx.doi.org/10.1046/j.1471-4159.2003.02264.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+a+residue+in+the+gamma-aminobutyric+acid+type+A+receptor+alpha+subunit+that+differentially+affects+diazepam-sensitive+and+-insensitive+benzodiazepine+site+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+a+residue+in+the+gamma-aminobutyric+acid+type+A+receptor+alpha+subunit+that+differentially+affects+diazepam-sensitive+and+-insensitive+benzodiazepine+site+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000894-Newell1" id="pone.0000894-Newell1"></a>Newell JG, McDevitt RA, Czajkowski C (2004) Mutation of glutamate 155 of the GABA<sub>A</sub> receptor beta2 subunit produces a spontaneously open channel: a trigger for channel activation. J Neurosci  24: 11226–11235.  <ul class="find" data-citedArticleID="992158" data-doi="10.1523/jneurosci.3746-04.2004"><li><a href="http://dx.doi.org/10.1523/jneurosci.3746-04.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutation+of+glutamate+155+of+the+GABAA+receptor+beta2+subunit+produces+a+spontaneously+open+channel%3A+a+trigger+for+channel+activation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutation+of+glutamate+155+of+the+GABAA+receptor+beta2+subunit+produces+a+spontaneously+open+channel%3A+a+trigger+for+channel+activation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000894-Altschul1" id="pone.0000894-Altschul1"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol  215: 403–410.  <ul class="find" data-citedArticleID="992090" data-doi="10.1016/s0022-2836(05)80360-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(05)80360-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Basic+local+alignment+search+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Basic+local+alignment+search+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000894-Guindon1" id="pone.0000894-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="992132"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000894-Ronquist1" id="pone.0000894-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="992178" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0000894-Abascal1" id="pone.0000894-Abascal1"></a>Abascal F, Zardoya R, Posada D (2005) ProtTest: selection of best-fit models of protein evolution. Bioinformatics  21: 2104–2105.  <ul class="find" data-citedArticleID="992088" data-doi="10.1093/bioinformatics/bti263"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti263" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ProtTest%3A+selection+of+best-fit+models+of+protein+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ProtTest%3A+selection+of+best-fit+models+of+protein+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0000894-Jones1" id="pone.0000894-Jones1"></a>Jones DT, Taylor WR, Thornton JM (1992) The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci  8: 275–282.  <ul class="find" data-citedArticleID="992144" data-doi="10.1093/bioinformatics/8.3.275"><li><a href="http://dx.doi.org/10.1093/bioinformatics/8.3.275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+rapid+generation+of+mutation+data+matrices+from+protein+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+rapid+generation+of+mutation+data+matrices+from+protein+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0000894-Yang4" id="pone.0000894-Yang4"></a>Yang Z (1994) Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods. J Mol Evol  39: 306–314.  <ul class="find" data-citedArticleID="992204" data-doi="10.1007/bf00160154"><li><a href="http://dx.doi.org/10.1007/bf00160154" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+phylogenetic+estimation+from+DNA+sequences+with+variable+rates+over+sites%3A+approximate+methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+phylogenetic+estimation+from+DNA+sequences+with+variable+rates+over+sites%3A+approximate+methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0000894-Huelsenbeck1" id="pone.0000894-Huelsenbeck1"></a>Huelsenbeck JP, Larget B, Alfaro ME (2004) Bayesian phylogenetic model selection using reversible jump Markov chain Monte Carlo. Mol Biol Evol  21: 1123–1133.  <ul class="find" data-citedArticleID="992138" data-doi="10.1093/molbev/msh123"><li><a href="http://dx.doi.org/10.1093/molbev/msh123" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+phylogenetic+model+selection+using+reversible+jump+Markov+chain+Monte+Carlo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+phylogenetic+model+selection+using+reversible+jump+Markov+chain+Monte+Carlo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0000894-Yang5" id="pone.0000894-Yang5"></a>Yang Z (2005)  Bayesian inference in molecular phylogenetics. In: Gascuel O, editor. Mathematics of evolution and phylogeny. Oxford: Oxford University Press. pp. 63–90.  <ul class="find-nolinks"></ul></li><li><span class="label">59.
              </span><a name="pone.0000894-Bielawski1" id="pone.0000894-Bielawski1"></a>Bielawski JP, Yang Z (2003) Maximum likelihood methods for detecting adaptive evolution after gene duplication. J Struct Funct Genomics  3: 201–212.  <ul class="find" data-citedArticleID="992102" data-doi="10.1007/978-94-010-0263-9_20"><li><a href="http://dx.doi.org/10.1007/978-94-010-0263-9_20" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maximum+likelihood+methods+for+detecting+adaptive+evolution+after+gene+duplication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maximum+likelihood+methods+for+detecting+adaptive+evolution+after+gene+duplication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0000894-Yang6" id="pone.0000894-Yang6"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="992208" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0000894-Nielsen2" id="pone.0000894-Nielsen2"></a>Nielsen R, Yang Z (1998) Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene. Genetics  148: 929–936.  <ul class="find" data-citedArticleID="992162"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene.&amp;auth=&amp;atitle=Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Likelihood+models+for+detecting+positively+selected+amino+acid+sites+and+applications+to+the+HIV-1+envelope+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0000894-Venclovas1" id="pone.0000894-Venclovas1"></a>Venclovas C, Margelevicius M (2005) Comparative modeling in CASP6 using consensus approach to template selection, sequence-structure alignment, and structure assessment. Proteins 61 Suppl  7: 99–105.  <ul class="find" data-citedArticleID="992192" data-doi="10.1002/prot.20725"><li><a href="http://dx.doi.org/10.1002/prot.20725" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Comparative+modeling+in+CASP6+using+consensus+approach+to+template+selection%2C+sequence-structure+alignment%2C+and+structure+assessment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Comparative+modeling+in+CASP6+using+consensus+approach+to+template+selection%2C+sequence-structure+alignment%2C+and+structure+assessment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.XML" value="118075"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.PDF" value="365986"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g001.PNG_L" value="121458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g001.PNG_M" value="108878"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g001.PNG_S" value="12071"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g001.TIF" value="690706"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g001.PNG_I" value="44371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g002.PNG_L" value="108187"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g002.PNG_M" value="79838"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g002.PNG_S" value="12483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g002.TIF" value="703466"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g002.PNG_I" value="46907"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g003.PNG_L" value="87377"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g003.PNG_M" value="84735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g003.PNG_S" value="12952"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g003.TIF" value="568540"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g003.PNG_I" value="48506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t001.PNG_L" value="212960"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t001.PNG_M" value="120251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t001.PNG_S" value="12357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t001.TIF" value="895002"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t001.PNG_I" value="37789"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t002.PNG_L" value="217501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t002.PNG_M" value="124384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t002.PNG_S" value="12497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t002.TIF" value="921092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.t002.PNG_I" value="38833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g004.PNG_L" value="55272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g004.PNG_M" value="68441"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g004.PNG_S" value="10664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g004.TIF" value="409010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g005.PNG_L" value="224226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g005.PNG_M" value="145501"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g005.PNG_S" value="14407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g005.TIF" value="394020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.g005.PNG_I" value="55288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.s001.TXT" value="84876"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000894.s002.TXT" value="91496"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000894&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000894" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000894&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000894&volume=&issue=&title=Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits&author_name=Christopher%20J.%20Martyniuk%2C%20St%C3%A9phane%20Aris-Brosou%2C%20Guy%20Drouin%2C%20Joel%20Cahn%2C%20Vance%20L.%20Trudeau&start_page=1&end_page=11" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000894" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894&amp;t=Early%20Evolution%20of%20Ionotropic%20GABA%20Receptors%20and%20Selective%20Regimes%20Acting%20on%20the%20Mammalian-Specific%20Theta%20and%20Epsilon%20Subunits" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894&title=Early%20Evolution%20of%20Ionotropic%20GABA%20Receptors%20and%20Selective%20Regimes%20Acting%20on%20the%20Mammalian-Specific%20Theta%20and%20Epsilon%20Subunits&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894&amp;title=Early%20Evolution%20of%20Ionotropic%20GABA%20Receptors%20and%20Selective%20Regimes%20Acting%20on%20the%20Mammalian-Specific%20Theta%20and%20Epsilon%20Subunits" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000894&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Early Evolution of Ionotropic GABA Receptors and Selective Regimes Acting on the Mammalian-Specific Theta and Epsilon Subunits';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000894';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Early%20Evolution%20of%20Ionotropic%20GABA%20Receptors%20and%20Selective%20Regimes%20Acting%20on%20the%20Mammalian-Specific%20Theta%20and%20Epsilon%20Subunits http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000894" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000894" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">

























          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Central+nervous+system%22" title="Search for articles in the subject area:'Central nervous system'"><div class="flagText">Central nervous system</div></a>
              <div data-categoryid="47337" data-articleid="24626"
                   data-categoryname="Central nervous system"
                   class="flagImage" title="Flag 'Central nervous system' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Codons%22" title="Search for articles in the subject area:'Codons'"><div class="flagText">Codons</div></a>
              <div data-categoryid="18427" data-articleid="24626"
                   data-categoryname="Codons"
                   class="flagImage" title="Flag 'Codons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gamma-aminobutyric+acid%22" title="Search for articles in the subject area:'Gamma-aminobutyric acid'"><div class="flagText">Gamma-aminobutyric acid</div></a>
              <div data-categoryid="19879" data-articleid="24626"
                   data-categoryname="Gamma-aminobutyric acid"
                   class="flagImage" title="Flag 'Gamma-aminobutyric acid' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="24626"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ligand-gated+ion+channels%22" title="Search for articles in the subject area:'Ligand-gated ion channels'"><div class="flagText">Ligand-gated ion channels</div></a>
              <div data-categoryid="52165" data-articleid="24626"
                   data-categoryname="Ligand-gated ion channels"
                   class="flagImage" title="Flag 'Ligand-gated ion channels' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="24626"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="24626"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+alignment%22" title="Search for articles in the subject area:'Sequence alignment'"><div class="flagText">Sequence alignment</div></a>
              <div data-categoryid="18051" data-articleid="24626"
                   data-categoryname="Sequence alignment"
                   class="flagImage" title="Flag 'Sequence alignment' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=215'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=968'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=7392&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
