

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Evolutionary History of LINE-1 in the Major Clades of Placental Mammals</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000158"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000158" />

    <meta name="citation_title" content="Evolutionary History of LINE-1 in the Major Clades of Placental Mammals"/>
    <meta itemprop="name" content="Evolutionary History of LINE-1 in the Major Clades of Placental Mammals"/>

      <meta name="citation_author" content="Paul D. Waters"/>
            <meta name="citation_author_institution" content="Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Matieland, South Africa"/>
            <meta name="citation_author_institution" content="Comparative Genomics Group, Research School of Biological Sciences, The Australian National University, Canberra, Australia"/>
      <meta name="citation_author" content="Gauthier Dobigny"/>
            <meta name="citation_author_institution" content="Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Matieland, South Africa"/>
            <meta name="citation_author_institution" content="Institut de Recherche pour le Développement, Centre de Biologie pour la Gestion des Populations, Montferrier-sur-Lez, France"/>
      <meta name="citation_author" content="Peter J. Waddell"/>
            <meta name="citation_author_institution" content="Laboratory of Biometry and Bioinformatics, University of Tokyo, Tokyo, Japan"/>
            <meta name="citation_author_institution" content="South Carolina Cancer Center, University of South Carolina, Columbia, South Carolina, United States of America"/>
      <meta name="citation_author" content="Terence J. Robinson"/>
            <meta name="citation_author_institution" content="Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Matieland, South Africa"/>

    <meta name="citation_date" content="2007/1/17"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000158.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e158"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Biology of mammalian L1 retrotransposons.; citation_author=EM Ostertag; citation_author=HH Kazazian; citation_journal_title=Annu Rev Genet; citation_volume=35; citation_number=1; citation_pages=501-538; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_author=ES Lander; citation_author=LM Linton; citation_author=B Birren; citation_author=C Nusbaum; citation_author=MC Zody; citation_journal_title=Nature; citation_volume=409; citation_number=2; citation_pages=860-921; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=LINE drive. retrotransposition and genome instability.; citation_author=HH Kazazian; citation_author=JL Goodier; citation_journal_title=Cell; citation_volume=110; citation_number=3; citation_pages=277-280; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Apolipoprotein(a) gene enhancer resides within a LINE element.; citation_author=Z Yang; citation_author=D Boffelli; citation_author=N Boonmark; citation_author=K Schwartz; citation_author=R Lawn; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=4; citation_pages=891-897; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Transcriptional disruption by the L1 retrotransposon and implications for mammalian transcriptomes.; citation_author=JS Han; citation_author=ST Szak; citation_author=JD Boeke; citation_journal_title=Nature; citation_volume=429; citation_number=5; citation_pages=268-274; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=X-chromosome inactivation: a repeat hypothesis.; citation_author=MF Lyon; citation_journal_title=Cytogenet Cell Genet; citation_volume=80; citation_number=6; citation_pages=133-137; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=X inactivation-specific methylation of LINE-1 elements by DNMT3B: implications for the Lyon repeat hypothesis.; citation_author=RS Hansen; citation_journal_title=Hum Mol Genet; citation_volume=12; citation_number=7; citation_pages=2559-2567; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=LINE-mediated retrotransposition of marked Alu sequences.; citation_author=M Dewannieux; citation_author=C Esnault; citation_author=T Heidmann; citation_journal_title=Nat Genet; citation_volume=35; citation_number=8; citation_pages=41-48; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Whole-genome screening indicates a possible burst of formation of processed pseudogenes and Alu repeats by particular L1 subfamilies in ancestral primates.; citation_author=K Ohshima; citation_author=M Hattori; citation_author=T Yada; citation_author=T Gojobori; citation_author=Y Sakaki; citation_journal_title=Genome Biol; citation_volume=4; citation_number=9; citation_pages=R74; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Ancestral, mammalian-wide subfamilies of LINE-1 repetitive sequences.; citation_author=AF Smit; citation_author=G Toth; citation_author=AD Riggs; citation_author=J Jurka; citation_journal_title=J Mol Biol; citation_volume=264; citation_number=10; citation_pages=401-417; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=A phylogenetic foundation for comparative mammalian genomics.; citation_author=PJ Waddell; citation_author=H Kishino; citation_author=R Ota; citation_journal_title=Genome Inform Ser Workshop Genome Inform; citation_volume=12; citation_number=11; citation_pages=141-154; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Towards resolving the interordinal relationships of placental mammals. Systematic Biology.; citation_author=PJ Waddell; citation_author=N Okada; citation_author=M Hasegawa; citation_journal_title=Syst Biol; citation_volume=48; citation_number=12; citation_pages=1-5; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Parallel adaptive radiations in two major clades of placental mammals.; citation_author=O Madsen; citation_author=M Scally; citation_author=CJ Douady; citation_author=DJ Kao; citation_author=RW DeBry; citation_journal_title=Nature; citation_volume=409; citation_number=13; citation_pages=610-614; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogenetics and the origins of placental mammals.; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=WE Johnson; citation_author=YP Zhang; citation_author=OA Ryder; citation_journal_title=Nature; citation_volume=409; citation_number=14; citation_pages=614-618; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Resolution of the Early Placental Mammal Radiation Using Bayesian Phylogenetics.; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=SJ O'Brien; citation_author=O Madsen; citation_author=M Scally; citation_journal_title=Science; citation_volume=294; citation_number=15; citation_pages=2348-2351; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Pig genome analysis: differential distribution of SINE and LINE sequences is less pronounced than in the human and mouse genomes.; citation_author=PD Thomsen; citation_author=JR Miller; citation_journal_title=Mamm Genome; citation_volume=7; citation_number=16; citation_pages=42-46; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=LINE-1 distribution in Afrotheria and Xenarthra: implications for understanding the evolution of LINE-1 in eutherian genomes.; citation_author=PD Waters; citation_author=G Dobigny; citation_author=AT Pardini; citation_author=TJ Robinson; citation_journal_title=Chromosoma; citation_volume=113; citation_number=17; citation_pages=137-144; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Molecular evidence for the major clades of placental mammals.; citation_author=M Scully; citation_author=O Madsen; citation_author=CJ Douady; citation_author=WW de Jong; citation_author=MJ Stanhope; citation_journal_title=J Mamm Evol; citation_volume=8; citation_number=18; citation_pages=239-277; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The use of composite taxa in supermatrices.; citation_author=MS Springer; citation_author=M Scally; citation_author=O Madsen; citation_author=WW de Jong; citation_author=CJ Douady; citation_journal_title=Mol Phylogenet Evol; citation_volume=30; citation_number=19; citation_pages=883-884; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Retroposed elements as archives for the evolutionary history of placental mammals.; citation_author=JO Kriegs; citation_author=G Churakov; citation_author=M Kiefmann; citation_author=U Jordan; citation_author=J Brosius; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=20; citation_pages=e91; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pylogenetic relationships of the warf boas and a comparison of Bayesian and boostrap measures of phylogenetic support.; citation_author=TP Wilcox; citation_author=DJ Zwickl; citation_author=TA Heath; citation_author=DM Hillis; citation_journal_title=Mol Phylogenet Evol; citation_volume=25; citation_number=21; citation_pages=361-371; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Evaluating placental inter-ordinal phylogenies with novel sequences including RAG1, gamma-fibrinogen, ND6, and mt-tRNA, plus MCMC-driven nucleotide, amino acid, and codon models.; citation_author=PJ Waddell; citation_author=S Shelley; citation_journal_title=Mol Phylogenet Evol; citation_volume=28; citation_number=22; citation_pages=197-224; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Placental diversification and the Cretaceous-Tertiary boundary.; citation_author=MS Springer; citation_author=WJ Murphy; citation_author=E Eizirik; citation_author=SJ O'Brien; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=23; citation_pages=1056-1061; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Influence of Tertiary paleoenvironmental changes on the diversification of South American mammals: a relaxed molecular clock study within xenarthrans.; citation_author=F Delsuc; citation_author=SF Vizcaino; citation_author=EJ Douzery; citation_journal_title=BMC Evol Biol; citation_volume=4; citation_number=24; citation_pages=11; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ancient SINEs from African endemic mammals.; citation_author=M Nikaido; citation_author=H Nishihara; citation_author=Y Hukumoto; citation_author=N Okada; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=25; citation_pages=522-527; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Parallel adaptive radiations in two major clades of placental mammals.; citation_author=O Madsen; citation_author=M Scally; citation_author=CJ Douady; citation_author=DJ Kao; citation_author=RW DeBry; citation_journal_title=Nature; citation_volume=409; citation_number=26; citation_pages=610-614; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A new phylogenetic marker, apolipoprotein B, provides compelling evidence for eutherian relationships.; citation_author=H Amrine-Madsen; citation_author=KP Koepfli; citation_author=RK Wayne; citation_author=MS Springer; citation_journal_title=Mol Phylogenet Evol; citation_volume=28; citation_number=27; citation_pages=225-240; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cross-species chromosome painting in the golden mole and elephant-shrew: support for the mammalian clades Afrotheria and Afroinsectiphillia but not Afroinsectivora.; citation_author=TJ Robinson; citation_author=B Fu; citation_author=MA Ferguson-Smith; citation_author=F Yang; citation_journal_title=Proc R Soc Lond B Biol Sci; citation_volume=271; citation_number=28; citation_pages=1477-1484; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A retroposon analysis of Afrotherian phylogeny.; citation_author=H Nishihara; citation_author=Y Satta; citation_author=M Nikaido; citation_author=JG Thewissen; citation_author=MJ Stanhope; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=29; citation_pages=1823-1833; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Definition and relationships of Lipotyphla.; citation_author=RDE MacPhee; citation_author=M Novacek; citation_number=30; citation_pages=13-29; citation_date=1993; citation_publisher=Springer-Verlag; " />
      <meta name="citation_reference" content="citation_title=Higher taxonomic relationships among extant mammals based on morphology, with selected comparisons of results from molecular data.; citation_author=J Shoshani; citation_author=MC McKenna; citation_journal_title=Mol Phylogenet Evol; citation_volume=9; citation_number=31; citation_pages=572-584; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Using novel phylogenetic methods to evaluate mammalian mtDNA, including amino acid-invariant sites-LogDet plus site stripping, to detect internal conflicts in the data, with special reference to the positions of hedgehog, armadillo, and elephant.; citation_author=PJ Waddell; citation_author=Y Cao; citation_author=J Hauf; citation_author=M Hasegawa; citation_journal_title=Syst Biol; citation_volume=48; citation_number=32; citation_pages=31-53; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Molecular phylogeny of living xenarthrans and the impact of character and taxon sampling on the placental tree rooting.; citation_author=F Delsuc; citation_author=M Scally; citation_author=O Madsen; citation_author=MJ Stanhope; citation_author=WW de Jong; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=33; citation_pages=1656-1671; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=L1 (LINE-1) retrotransposon diversity differs dramatically between mammals and fish.; citation_author=AV Furano; citation_author=DD Duvernell; citation_author=S Boissinot; citation_journal_title=Trends Genet; citation_volume=20; citation_number=34; citation_pages=9-14; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=L1 (LINE-1) retrotransposon evolution and amplification in recent human history.; citation_author=S Boissinot; citation_author=P Chevret; citation_author=AV Furano; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=35; citation_pages=915-928; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Rapid evolution of a young L1 (LINE-1) clade in recently speciated Rattus taxa.; citation_author=EL Cabot; citation_author=B Angeletti; citation_author=K Usdin; citation_author=AV Furano; citation_journal_title=J Mol Evol; citation_volume=45; citation_number=36; citation_pages=412-423; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The age and evolution of non-LTR retrotransposable elements.; citation_author=HS Malik; citation_author=WD Burke; citation_author=TH Eickbush; citation_journal_title=Mol Biol Evol; citation_volume=16; citation_number=37; citation_pages=793-805; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=T-Coffee: A novel method for fast and accurate multiple sequence alignment.; citation_author=C Notredame; citation_author=DG Higgins; citation_author=J Heringa; citation_journal_title=J Mol Biol; citation_volume=302; citation_number=38; citation_pages=205-217; citation_date=2000; " />
      <meta name="citation_reference" content="citation_author=DL Swofford; citation_number=39; citation_publisher=Sinauer Associates; " />
      <meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models.; citation_author=F Ronquist; citation_author=JP Huelsenbeck; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=40; citation_pages=1572-1574; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Dating the human-ape splitting by a molecular clock of mitochondrial DNA.; citation_author=M Hasegawa; citation_author=K Kishino; citation_author=T Yano; citation_journal_title=J Mol Evol; citation_volume=22; citation_number=41; citation_pages=160-174; citation_date=1985; " />
      <meta name="citation_reference" content="citation_title=Phylogenetic inference.; citation_author=DL Swofford; citation_author=GJ Olsen; citation_author=PJ Waddell; citation_author=DM Hillis; citation_number=42; citation_pages=407-514; citation_date=1996; citation_publisher=Sinauer; " />
      <meta name="citation_reference" content="citation_title=Evolutionary trees of apes and humans from DNA sequences.; citation_author=PJ Waddell; citation_author=D Penny; citation_number=43; citation_pages=53-73; citation_date=1996; citation_publisher=Clarendon Press; " />
      <meta name="citation_reference" content="citation_title=General time-reversible distances with unequal rates across sites: mixing gamma and inverse Gaussian distributions with invariant sites.; citation_author=PJ Waddell; citation_author=MA Steel; citation_journal_title=Mol Phylogenet Evol; citation_volume=8; citation_number=44; citation_pages=398-414; citation_date=1997; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Evolutionary History of LINE-1 in the Major Clades of Placental Mammals"/>
    <meta name="twitter:description" content="BackgroundLINE-1 constitutes an important component of mammalian genomes. It has a dynamic evolutionary history characterized by the rise, fall and replacement of subfamilies. Most data concerning LINE-1 biology and evolution are derived from the human and mouse genomes and are often assumed to hold for all placentals.MethodologyTo examine LINE-1 relationships, sequences from the 3&prime; region of the reverse transcriptase from 21 species (representing 13 orders across Afrotheria, Xenarthra, Supraprimates and Laurasiatheria) were obtained from whole genome sequence assemblies, or by PCR with degenerate primers. These sequences were aligned and analysed.Principal FindingsOur analysis reflects accepted placental relationships suggesting mostly lineage-specific LINE-1 families. The data provide clear support for several clades including Glires, Supraprimates, Laurasiatheria, Boreoeutheria, Xenarthra and Afrotheria. Within the afrotherian LINE-1 (AfroLINE) clade, our tree supports Paenungulata, Afroinsectivora and Afroinsectiphillia. Xenarthran LINE-1 (XenaLINE) falls sister to AfroLINE, providing some support for the Atlantogenata (Xenarthra+Afrotheria) hypothesis.SignificanceLINEs and SINEs make up approximately half of all placental genomes, so understanding their dynamics is an essential aspect of comparative genomics. Importantly, a tree of LINE-1 offers a different view of the root, as long edges (branches) such as that to marsupials are shortened and/or broken up. Additionally, a robust phylogeny of diverse LINE-1 is essential in testing that site-specific LINE-1 insertions, often regarded as homoplasy-free phylogenetic markers, are indeed unique and not convergent."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000158.g002"/>

  <meta property="og:title" content="Evolutionary History of LINE-1 in the Major Clades of Placental Mammals" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4798'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=8284'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6927&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000158">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000158" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000158&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Evolutionary History of LINE-1 in the Major Clades of Placental Mammals
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Paul D. Waters, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Matieland, South Africa, 
                  Comparative Genomics Group, Research School of Biological Sciences, The Australian National University, Canberra, Australia
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gauthier Dobigny, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Matieland, South Africa, 
                  Institut de Recherche pour le Développement, Centre de Biologie pour la Gestion des Populations, Montferrier-sur-Lez, France
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Peter J. Waddell, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Laboratory of Biometry and Bioinformatics, University of Tokyo, Tokyo, Japan, 
                  South Carolina Cancer Center, University of South Carolina, Columbia, South Carolina, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Terence J. Robinson
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:tjr@sun.ac.za">tjr@sun.ac.za</a>
            </p>

                <p>Affiliation:
                  Evolutionary Genomics Group, Department of Botany and Zoology, University of Stellenbosch, Matieland, South Africa
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 17, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000158</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000158-g001" data-doi="info:doi/10.1371/journal.pone.0000158" data-uri="info:doi/10.1371/journal.pone.0000158.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000158.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000158-g002" data-doi="info:doi/10.1371/journal.pone.0000158" data-uri="info:doi/10.1371/journal.pone.0000158.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000158.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000158">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000158" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>LINE-1 constitutes an important component of mammalian genomes. It has a dynamic evolutionary history characterized by the rise, fall and replacement of subfamilies. Most data concerning LINE-1 biology and evolution are derived from the human and mouse genomes and are often assumed to hold for all placentals.</p>


<h3>Methodology</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>To examine LINE-1 relationships, sequences from the 3′ region of the reverse transcriptase from 21 species (representing 13 orders across Afrotheria, Xenarthra, Supraprimates and Laurasiatheria) were obtained from whole genome sequence assemblies, or by PCR with degenerate primers. These sequences were aligned and analysed.</p>


<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Our analysis reflects accepted placental relationships suggesting mostly lineage-specific LINE-1 families. The data provide clear support for several clades including Glires, Supraprimates, Laurasiatheria, Boreoeutheria, Xenarthra and Afrotheria. Within the afrotherian LINE-1 (AfroLINE) clade, our tree supports Paenungulata, Afroinsectivora and Afroinsectiphillia. Xenarthran LINE-1 (XenaLINE) falls sister to AfroLINE, providing some support for the Atlantogenata (Xenarthra+Afrotheria) hypothesis.</p>


<h3>Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec4.p1" name="article1.front1.article-meta1.abstract1.sec4.p1"></a><p>LINEs and SINEs make up approximately half of all placental genomes, so understanding their dynamics is an essential aspect of comparative genomics. Importantly, a tree of LINE-1 offers a different view of the root, as long edges (branches) such as that to marsupials are shortened and/or broken up. Additionally, a robust phylogeny of diverse LINE-1 is essential in testing that site-specific LINE-1 insertions, often regarded as homoplasy-free phylogenetic markers, are indeed unique and not convergent.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Waters PD, Dobigny G, Waddell PJ, Robinson TJ (2007) Evolutionary History of LINE-1 in the Major Clades of Placental Mammals. PLoS ONE 2(1):
          e158.
            doi:10.1371/journal.pone.0000158</p><p><strong>Academic Editor: </strong>James Fraser, University of Queensland, Australia</p><p><strong>Received:</strong> November 1, 2006; <strong>Accepted:</strong> December 15, 2006; <strong>Published:</strong> January 17, 2007</p><p><strong>Copyright:</strong> © 2007 Waters et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Financial support from the Wellcome Trust, the University of Stellenbosch and the National Research Foundation (GUN 2053812) to TJR is gratefully acknowledged. PJW thanks JSPS for support via their fellowship program, NIH grant 5R01LM8626.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The non-LTR retrotransposons Long Interspersed Nuclear Elements-1 (LINE-1, or L1) are a major component of mammalian genomes (~20% of that of human) that transpose through an RNA intermediate (reviewed in <a href="#pone.0000158-Ostertag1">[1]</a>). Most LINE-1 elements are 5′ truncated upon transposition and are therefore inactive. In fact, only an estimated 60 LINE-1 copies are potentially active in the human genome <a href="#pone.0000158-Lander1">[2]</a>. LINE-1 have been shown to be responsible for many genetic disorders such as gene disruption, nucleotide deletions, duplications and chromosomal instability through heterologous recombination (<a href="#pone.0000158-Kazazian1">[3]</a> and references therein). However, they are also involved in important genomic functions. These include regulation of gene expression <a href="#pone.0000158-Yang1">[4]</a>, <a href="#pone.0000158-Han1">[5]</a> and possibly X-inactivation in females <a href="#pone.0000158-Lyon1">[6]</a>, <a href="#pone.0000158-Hansen1">[7]</a>. Further, LINE-1 provides the Reverse-Transcriptase (RTase) necessary for transposition of the ALU SINE sequences <a href="#pone.0000158-Dewannieux1">[8]</a>, and may also have a role in the generation of processed pseudogenes <a href="#pone.0000158-Ohshima1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Most of the current data concerning LINE-1 biology and evolution result from investigations of the human and mouse genomes and are often assumed to hold for all eutherians <a href="#pone.0000158-Ostertag1">[1]</a>. Previous studies have investigated the paleohistory of LINE-1 families based on the “genomic fossil record of pseudogenes retroposed at different times from active source genes” <a href="#pone.0000158-Smit1">[10]</a>. Unfortunately this usually relies heavily on human and/or mouse genomes. It is now widely accepted that rodents and primates fall within the same supraordinal clade that is often called Euarchontoglires, but is formally named and defined as the crown group Supraprimates (comprising the orders Primates, Dermoptera, Scandentia, Rodentia and Lagomorpha <a href="#pone.0000158-Waddell1">[11]</a>). Supraprimates is recognised as only one of the four main lineages of eutherian mammals, the others being Laurasiatheria (Pholidota, Carnivora, Perissodactyla, Cetartiodactyla, Chiroptera, Eulipotyphla), Xenarthra (Cingulata, Vermilingua, Folivora) and Afrotheria (Proboscidea, Sirenia, Hyracoidea, Tubulidentata, Macroscelidae, plus Afrosoricida = Tenrecomorpha = Chrysochloridae+Tenrecidae) <a href="#pone.0000158-Waddell1">[11]</a>–<a href="#pone.0000158-Murphy2">[15]</a>, and not especially close to the root. Significantly, investigations of LINE-1 distribution in species representative of other eutherian clades have demonstrated that Supraprimates display distinct patterns to the others <a href="#pone.0000158-Thomsen1">[16]</a>, <a href="#pone.0000158-Waters1">[17]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Here we extend the understanding of LINE-1 using PCR amplification from genomes of a broad range of placental species complemented with Blastn searches of available databases, with emphasis on the two most basal placental clades, Afrotheria and Xenarthra. Using these data we examine how closely a tree of LINE elements follows the generally accepted tree of placental mammals <a href="#pone.0000158-Waddell1">[11]</a>–<a href="#pone.0000158-Murphy2">[15]</a>, <a href="#pone.0000158-Scully1">[18]</a>, <a href="#pone.0000158-Springer1">[19]</a>. Thus, taxon-specific LINE-1 activity is identified. There is strong evidence for autapomorphic groups of LINE-1 active in Afrotheria, Xenarthra and Boreoeutheria, i.e. AfroLINEs, XenaLINEs, and BoreoLINEs respectively.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Finally, LINE and SINE transposition, or site-specific insertion events, are considered rare genomic changes and as such are increasingly used as phylogenetic characters to test other reconstructions <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Kriegs1">[20]</a>. However, the potential for convergent insertions, and the question of how often this occurs, needs to be addressed. The construction of an accurate tree of LINEs across mammals is an essential step in addressing this question.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Sequence data</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Since most LINE-1 copies are 5′-truncated and subsequently evolve as pseudogenes <a href="#pone.0000158-Ostertag1">[1]</a> they accumulate open reading frame (ORF) terminating, nonsense, and indel mutations. As expected, most of the 52 clones sequenced herein exhibit gaps and are non-active elements. Overall, five aardvark, three golden mole, one elephant and one bat clone displayed an ORF. It is probable that these represent a recent class of transposon not having had sufficient time to acquire mutations in this region. To further determine which lineages appeared recently active, all elements from the extended dataset were BLASTed against their genome of origin, and any that gave full-length hits with homologies higher than 98% have their branches coloured grey in <a href="#pone-0000158-g001">Figure 1</a>. A divergence of 2% is approximately 5 million years at the relatively slow rate that apes (e.g., chimp, human) evolve, or ~1–3 million years for murid rodents which show the most rapid rates of change (along with tenrec) on our trees. These blast searches can of course be done only in those species for which there are at least whole genome shotgun sequence data (armadillo, elephant, tenrec, human, chimp, macaque, rabbit, mouse, rat, cow, dog). Therefore, to estimate recent activity in the absence of these data (for Cape serotine bat, aardvark, Cape golden mole, Cape elephant shrew, Cape rock hyrax, Cowan's shrew tenrec, Florida manatee, six-banded armadillo, tree anteater, pale-throated three-toed sloth), pairwise distances of closely related LINE-1 were checked, and any pair of sequences that were &gt;98% similar to each other are coloured yellow in the tree (<a href="#pone-0000158-g001">Figure 1</a>). This gives a reasonable estimate of recently active LINE-1 in our dataset.</p>
<div class="figure" id="pone-0000158-g001"><div class="img"><a name="pone-0000158-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000158.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000158" data-uri="info:doi/10.1371/journal.pone.0000158.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000158.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000158.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000158.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000158.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000158.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000158.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000158.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000158.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000158.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Phylogenetic tree of LINE-1: combined dataset. Bayesian consensus tree generated by a GTR invariant-sites plus Γ model applied to our concatenated dataset (69 long and 109 short sequences; see text for details). Posterior probability values ≥95% are shown. Species in blue reflect sequences that are 1050 bp in length, whereas those in black correspond to the 300 bp sequences. Grey branches indicate sequences with &gt;98% homology in their respective genomes. For species lacking whole genome sequencing projects, that is manatee, hyrax, golden mole, sloth, bat, among others, yellow is used to indicate pairs of sequences with &gt;98% homology. This yields a minimum estimate of the number of copies of potentially recently active L1 in these species. A species key shows the abbreviated names, scientific names and common names. The tree is broken into two sections. The inset shows which part of the tree is displayed.</p>
<span>doi:10.1371/journal.pone.0000158.g001</span></div>

<h4>The Tree</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>All trees reveal a similar history, irrespective of the methods used. <a href="#pone-0000158-g001">Figure 1</a> shows the hierarchical Bayesian consensus tree generated by a GTR invariant-sites plus gamma model. This method with posterior probability (pp) values is useful for a number of reasons. Firstly, with short sequences (and many of these are only 300 bp) the bootstrap approach tends to be very severe on resolving clades. To illustrate this consider, for example, a transversional change that defines a clade without contradiction by other characters. This will receive a bootstrap proportion of ~67%, which is low. However, if the model identifies this as a substitution pattern that is very unlikely to be due to multiple hits, it should receive a high pp value. Secondly, under the model pp values tend to be conservative <a href="#pone.0000158-Wilcox1">[21]</a> yet we need to be cautious since the data do not fit the model so pp values can easily become too extreme as the likelihood function becomes inexact <a href="#pone.0000158-Waddell1">[11]</a>. For other mammalian nuclear genes, pp values seemed to produce few enough strongly misleading results as to retain utility for evaluating branches of trees from short sequences <a href="#pone.0000158-Waddell3">[22]</a>. Finally, and reassuringly, a careful visual inspection of our results shows there are no high (&gt;0.95) pp values that clash with <em>a priori</em> expectations.</p>


<h4>Lineage-specific LINEs and systematic implications</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Moreover, our two data sets (longer sequences only vs. combined data) retrieved similar topologies (<a href="#pone-0000158-g001">Figure 1</a> and <a href="#pone-0000158-g002">Figure 2</a>) with the shorter sequences grouping/falling in expected positions around the corresponding longer sequences. Consequently, the following discussion largely focuses on the outcomes reflected in the combined tree (<a href="#pone-0000158-g001">Figure 1</a>). Although we are aware that our investigations represent gene phylogenies, since most sequences are paralogous many clades of LINE-1 elements were, nonetheless, found to reflect closely what is known about species relationships in mammals. This clearly suggests that for many lineages all LINE-1 active at any one time coalesce to a common and not too distant ancestor. Hence these signatures of ancestral but exclusively shared TE activity can reasonably be used as potential synapomorphies, and are thus useful for inferring phylogenetic relationships between species.</p>
<div class="figure" id="pone-0000158-g002"><div class="img"><a name="pone-0000158-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000158.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000158" data-uri="info:doi/10.1371/journal.pone.0000158.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000158.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000158.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000158.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000158.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000158.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000158.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000158.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000158.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000158.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Phylogenetic tree of LINE-1: longer sequences only. Bayesian consensus tree generated by a GTR invariant-sites plus Γ model applied to our long (1050 bp) dataset that included 69 sequences. Posterior probability values ≥95% are shown. Grey branches indicate sequences with &gt;98% homology to other LINE-1 copies in their respective genomes. A species key shows the abbreviated names, scientific names and common names.</p>
<span>doi:10.1371/journal.pone.0000158.g002</span></div><h5>AfroLINEs, XenaLINEs and BoreoLINEs.</h5><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1"></a><p>Three major clades of L1 appear in our tree corresponding to three main placental lineages. They are monophyletic assemblages of LINE-1 sequences obtained exclusively from: 1) Boreoeutheria represented here by primates (human, chimpanzee and macaque), rodents (rat and mouse) plus lagomorphs (rabbit), i.e. Supraprimates, and Laurasiatheria (pig, cat, bat, horse, cow and dog). 2) Xenarthra represented in our analysis by a monophyletic group of nine-banded and six-banded armadillo L1, with sequences of other xenathrans (sloth and anteater) clustered at their base. 3) Afrotheria represented by elephant, manatee, hyrax, golden mole, elephant shrew, tenrec and aardvark (<a href="#pone-0000158-g001">Figure 1</a>). The support for the monophyly of LINE-1 elements specific to many of these taxa, some of which have high homology to other copies in their respective genomes (and therefore were probably active relatively recently), suggests that there has been continued LINE-1 activity in nearly all these placental lineages. Molecular dating indicates that Afrotheria differentiated from other Placentalia ~110–95 million years ago (MYA) and subsequently radiated ~85 MYA <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Murphy1">[14]</a>, <a href="#pone.0000158-Springer2">[23]</a>, <a href="#pone.0000158-Delsuc1">[24]</a>. A potentially good geological calibration that agrees with such dates is the separation of Afrotheria and Xenathra by the opening of the South Atlantic about 100 MYA <a href="#pone.0000158-Waddell2">[12]</a>.</p>
<a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2"></a><p>The activity of a unique family of SINEs in afrotherians (AfroSINES <a href="#pone.0000158-Nikaido1">[25]</a>) supports early AfroLINE activity since SINE mobility is reliant on the RTase from transposable elements other than themselves, primarily LINE-1 <a href="#pone.0000158-Dewannieux1">[8]</a>. Other rare genomic characters supporting Afrotheria include a 9 bp deletion in BRCA1 <a href="#pone.0000158-Madsen2">[26]</a>, a 237-246 bp deletion in APOB <a href="#pone.0000158-AmrineMadsen1">[27]</a>, two chromosomal syntenic associations (HSA1+19q, HSA5+3+21 <a href="#pone.0000158-Robinson1">[28]</a>) and several TE insertions <a href="#pone.0000158-Nishihara1">[29]</a>.</p>

<h5>Intra BoreoLINEs.</h5><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1"></a><p>Relationships of active L1 lineages within Boreoeutheria closely follow the expected species relationships. There are monophyletic groups of rat and mouse L1 to which the rabbit elements are basal. Sister to this grouping are primate LINE-1, which include the primate-specific L1PA2 element consensus sequence. Interestingly, there are two clades of primate specific L1, one of which appears to have been recently active with elements that have &gt;98% homology to other copies in the genome. The L1PA2 consensus sequence falls within the clade with no recent activity in human, and therefore most likely represents a LINE-1 lineage that has become extinct in human (<a href="#pone-0000158-g001">Figure 1</a>). Collectively these elements represent supraprimate retrotransposons. Also included in the Boreoeutheria are monophyletic groupings of cow and dog L1 together with the pig, cat and bat L1 that collectively represent laurasiatherian elements. Additionally, we identified three orthologous L1 between human and chimpanzee (as indicated by their identical location and orientation to the same markers in both species). All three pairs share 98–99% homology to each other: HSA_3 and PTR_2 fall within the primate L1; the older HSA_2 and PTR_1 elements fall at the base of Boreoeutheria; and the ancient HSA_4 and PTR_3 fall near the root of the tree with other ancient elements (<a href="#pone-0000158-g001">Figure 1</a>).</p>



<h4>Intra AfroLINEs</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Many robust LINE-1 groups were retrieved within Afrotheria. Most hyrax, manatee and elephant LINE-1 sequences were grouped into two clades providing evidence for at least two clade-specific elements, thus strongly consistent with Paenungulata. This suggests that LINE-1 activity and evolution occurred in the ancestral Paenungulate genome subsequent to their differentiation from other afrotherians but before the hyrax/manatee/elephant split <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Murphy1">[14]</a>, <a href="#pone.0000158-Springer2">[23]</a>, <a href="#pone.0000158-Delsuc1">[24]</a> which could not be resolved using our LINE-1 investigations.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Equally interesting from the phylogenetic viewpoint is recovery of a large group of LINEs only in taxa of the superorder Afroinsectiphillia (aardvark, tenrecs, golden moles and elephant shrews <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Murphy1">[14]</a>, <a href="#pone.0000158-Robinson1">[28]</a>) and, within it, Afroinsectivora (the former taxa minus aardvark). Indeed all LINEs sampled within these orders coalesce within their respective orders (with the exception of two tenrec sequences obtained from the database). Surprisingly, we do not find support for Afrosoricida (tenrecs plus golden moles). This can be a difficult group to recover using nucleotide sequences <a href="#pone.0000158-Waddell3">[22]</a> and has yet to be confirmed by a suite of conservative characters (although it is consistent with some of the morphology <a href="#pone.0000158-MacPhee1">[30]</a>). In our investigation a distinct branching pattern emerged with golden mole closest to elephant shrew (<a href="#pone-0000158-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Within aardvark there is good evidence of recent activity as indicated by our identification of closely related sequences (yellow branches in <a href="#pone-0000158-g001">Figure 1</a>) and the finding of five intact open reading frames (ORFs) in the seven aardvark clones sequenced. This is consistent with fluorescent <em>in situ</em> hybridization patterns showing aardvark to be highly enriched with LINE-1 relative to other placental mammals <a href="#pone.0000158-Waters1">[17]</a> suggesting that this was a relatively recent event which is perhaps ongoing.</p>


<h4>XenaLINEs and the root</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Use of the consensus sequences L1M4 and L1ME to root the tree is further justified by a diverse assemblage of apparently very old LINE-1 insertions, shared by many of the main placental lineages, and appearing sister to them. All have seemingly been inactive for a considerable period as suggested by their long terminal lineages plus numerous indels and non-sense mutations (data not shown).</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The relationship between the four major placental clades is still hotly debated. There are three competing hypotheses, the Epitheria hypothesis that Xenarthra is sister to all other placentals <a href="#pone.0000158-Kriegs1">[20]</a>, <a href="#pone.0000158-Shoshani1">[31]</a>, the Atlantogenata hypothesis that Xenarthra and Afrotheria are sister taxa and sister to all other placentals <a href="#pone.0000158-Waddell4">[32]</a>, and the Exafroplacentalia hypothesis that Afrotheria is at the root <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Murphy1">[14]</a>, <a href="#pone.0000158-Waddell3">[22]</a>, <a href="#pone.0000158-Delsuc2">[33]</a>. Molecular analyses, even using long concatenated sequences, fail to provide consistent statistical support to any of these (compounded by the fact that taxon sampling makes a considerable difference) suggesting the phylogenetic models are breaking down <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Delsuc2">[33]</a>. Recently, however, support for the Epitheria hypothesis was shown by Kriegs <em>et al</em> 
               <a href="#pone.0000158-Kriegs1">[20]</a> using retroposed elements. We on the other hand show that the L1 sequences from the xenarthran species fall sister to the AfroLINEs (<a href="#pone-0000158-g001">Figure 1</a>) favouring the Atlantogenata hypothesis. In the long dataset tree the AfroLINEs (represented by elephant and tenrec) and XenaLINEs (represented by nine-banded armadillo) association has a pp value of 0.97 (<a href="#pone-0000158-g002">Figure 2</a>). This is also consistent with results by Waddell, Umehara, Griche and Kishino (unpublished) whose analyses of the 17 aligned genomes at the UCSC browser, identify 15 highly conserved indels of 5 bp or greater in favour of Atlantogenata, but only four for Epitheria and three for Exafroplacentalia (a highly significant result by the test in <a href="#pone.0000158-Waddell1">[11]</a>).</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>There is strong support for two lineages of nine-banded armadillo L1. Members of only one of these clades have homology &gt;98% to other elements in the genome. L1 members of the remaining clade appear to have been inactive for longer with homologies of 92–95% to other L1 copies in the genome. They also cluster with the three L1 isolated from six-banded armadillo and, therefore represent a family of L1 that was active prior to the divergence these two armadillo species (<a href="#pone-0000158-g001">Figure 1</a>).</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Assuming our tree is accurate, at least two interpretations of our results can be made depending on which topology of the placental tree is considered correct. First, if either the Exafroplacentalia or Epitheria hypothesis is correct, then a variety of LINEs must have been active just before the three main placental groups split. These remained active after the first branching and by chance a fairly closely related pair of LINE-1 lineages came to dominate the genomes of afrotherians (AfroLINEs) and xenarthrans (XenaLINEs) with all others apparently going extinct. In boreoeutherians, the LINE lineages that came to dominate Afrotheria and Xenarthra went extinct and a third (more distantly related) assemblage of LINE lineages (BoreoLINEs) eventually dominated. The alternative, and <em>a priori</em> more likely explanation is that LINEs follow the species tree, and the Atlantogenata hypothesis is correct. Either hypothesis is consistent with what is known presently about LINE evolutionary history and functioning. This includes L1 activity before and during the first branching within placentals <a href="#pone.0000158-Lander1">[2]</a>, <a href="#pone.0000158-Smit1">[10]</a>, plus cycles of competition, extinction and replacement of L1 which is well documented in primates and rodents <a href="#pone.0000158-Furano1">[34]</a>–<a href="#pone.0000158-Cabot1">[36]</a>.</p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>A serious concern in all sequence analyses of placental orders is that the model of sequence evolution assumed is inadequate, and as a consequence there will be systematic errors in the tree that swamp stochastic errors. In order to further assess the potential for these biases we ran a set of transversion only analyses. In general the stochastic error of edges rose but the same basic topology was retrieved. That is, the L1 consensus based root was surrounded by old copies of L1, and there were four main groups of LINEs. Those in taxa from Afrotheria and Xenarthra on one side, and those from taxa within Supraprimates and Laurasiatheria on the other side of the root, thus supporting the Atlantogenata postulate.</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6"></a><p>Although we are unable to definitely determine which of the hypotheses outlined above is correct, the recently initiated sequencing of complete genomes for a wide variety of mammals including armadillo, elephant, tenrec, rabbit, hedgehog, and guinea pig (see <a href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a>) should allow refined testing of the alternatives. Indeed at last count, there were at least 2X draft sequences completed for nearly 30 mammals representing all but four orders - Tubulidentata (aardvark), Sirenia (manatee and dugong), Pholidota (pangolin) and flying lemur (Dermoptera). Such an extensive evaluation will be an important test of how informative a PCR/phylogenetic survey such as this is in determining the history and activity of LINE-1 in a diverse group, and how episodic LINE activity has been in the evolutionary past.</p>
<a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7"></a><p>Finally, as the testing of deeper phylogenetic relationships (such as inter-ordinal relationships) moves to include L1 (also SINE and other TE) insertion events <a href="#pone.0000158-Waddell1">[11]</a>, <a href="#pone.0000158-Kriegs1">[20]</a>, the need to sample L1 across many taxa will be a critical test to determine whether particular insertion events can be considered appropriately rare genomic changes. Convergent L1 insertion events will tend to appear paraphyletic within the tree, whereas synapomorphic insertions should be monophyletic. Thus an accurate tree of LINEs for all genomes should go hand in hand with using these characters as phylogenetic markers. Our work represents an important step in this direction.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Material and Methods"></a><h3>Material and Methods</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Previously designed general primers <a href="#pone.0000158-Waters1">[17]</a> were used to amplify approximately 300 bp of ORF 2 (240 bp within the 7<sup>th</sup> and 8<sup>th</sup> subdomains of the RT domain plus 60 bp extending into the region directly 3′ to the RT <a href="#pone.0000158-Malik1">[37]</a>). Such PCR preferentially targets the most numerous intact LINE-1 family members in the genome. PCR products were cloned and a random sample sequenced. Homology of the clones with LINE-1 sequence was systematically verified using the RepeatMasker program (<a href="http://repeatmasker.genome.washington.edu">http://repeatmasker.genome.washington.ed​u</a>).</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Two phylogenetic data matrices were prepared. The first included 66 LINE-1 sequences of 1050 bp from 11 eutherian species (six sequences from each species) with genome sequencing projects available from <a href="http://www.ensembl.org">www.ensembl.org</a> ( <a href="#pone.0000158.s001">Supplementary table 1</a>) plus three consensus sequences (L1PMA2, L1M4 and L1ME <a href="#pone.0000158-Smit1">[10]</a>). Our second matrix included these 69 sequences (1050 bp), plus 52 additional ~300 bp sequences we obtained by PCR, and 57 partial sequences from Genbank. This second matrix comprised 1050 characters and 178 sequences representing a total of 22 placental species in 13 orders (<a href="#pone.0000158.s001">Supplementary table 1</a>). The shorter 300 bp sequences are homologous to the 3′ end of the 1050 bp sequences. Sequences were aligned using T-coffee v1.35 <a href="#pone.0000158-Notredame1">[38]</a> then refined manually. Nucleotide sites that clearly appeared to be post-transposition insertions (e.g., insertions present in only one sequence that also cause a major frameshift) were removed from the data set.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>A variety of methods were used to explore the data. These included parsimony, distance based trees, and maximum likelihood (using PAUP* <a href="#pone.0000158-Swofford1">[39]</a>), plus a “hierarchical Bayesian” method implemented using (MC)<sup>3</sup> chains (MrBayes 3.0 <a href="#pone.0000158-Ronquist1">[40]</a>). The hierarchical Bayesian (or marginal likelihood) trees, in particular, tended to best reflect biological expectations by recovering well-established clades suggesting fewer errors in reconstruction. General transition matrices, such as the HKY <a href="#pone.0000158-Hasegawa1">[41]</a>, or general time reversible GTR models with site rate variability following an invariant sites (p<sub>inv</sub>) plus gamma (Γ) distribution <a href="#pone.0000158-Swofford2">[42]</a>–<a href="#pone.0000158-Waddell6">[44]</a>, were used for model-based methods. The Γ distribution was approximated with four discrete rate classes of equal size. Five chains (four hot, one cold) plus a random starting tree was used for each (MC)<sup>3</sup> run. Chains were run to at least 4 million steps with sampling of trees every 50 steps. Plateaus (supposed convergences) in likelihood tended to appear by ~200,000 steps, but all trees prior to 500,000 steps were discarded in order to be conservative. All other settings were left at their defaults. Each model was run at least twice and both topology and posterior probability (pp) values of edges were checked for conformity between runs.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>To further test deeper portions of the tree, and in particular possible misrooting, transversion only invariant-sites plus gamma models were used <a href="#pone.0000158-Swofford2">[42]</a>. Since the average instantaneous transversion rate is approximately four times less than that of transitions, this should strongly reduce the extent to which multiple hits confound tree estimation. These were implemented by converting all As in the data into Gs and all Cs into Ts. PAUP* and MrBayes then default to transversion models. Note that, in the case of MrBayes, the Dirichlet priors remained those for four states, but these allowed the frequencies of A and C to go to much less than zero, closely approximating a transversion only model.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Serious issues with the rooting of placentals using sequence data are: (1) Breakdown of the fit of data to any currently used phylogenetic model, (2) the long distance to the marsupial outgroup <a href="#pone.0000158-Madsen2">[26]</a>, and (3) long unbranched ingroup sequence <a href="#pone.0000158-Nishihara1">[29]</a>. In the present investigation the root of the tree was identified <em>a priori</em> as a L1M4 consensus sequence and a L1ME (ancient L1M4 subfamily) consensus sequence which was postulated as predating the earliest splits among living placental mammals <a href="#pone.0000158-Smit1">[10]</a>. Using consensus sequences of L1 common to all placentals offers a root much closer than marsupials, while L1 elements themselves break up long branches of the species tree such as the aardvark lineage (the only living species in Tubulidentata). These two effects together offer considerable robustness against break-down of the model <a href="#pone.0000158-Swofford2">[42]</a>. To further increase robusteness, we ran transversion-only models to explore the root.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000158"></div><a name="pone.0000158.s001" id="pone.0000158.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000158.s001">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Species and common names of sequences used in this study. The sequence names are as they appear on the phylogeny of <a href="#pone-0000158-g001">Figure 1</a>. The location of the sequence is given as a position on either a chromosome, scaffold or clone, along with its accession number. Sequences generated in this study have an accession number but no strand location as these are not part of a genome assembly.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000158.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.06 MB XLS)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>GD and PDW were postdoctoral fellows in the Evolutionary Genomics Group. PJW thanks the Laboratory of Biometry and Bioinformatics, University of Tokyo for computational resources in running phylogenetic analyses. PJW was the 2005 recipient of the John Ellerman Travelling Scholarship.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: GD TR PW PW. Performed the experiments: GD PW. Analyzed the data: PW PW. Contributed reagents/materials/analysis tools: TR. Wrote the paper: GD TR PW PW. Other: Did some of the bench-work, preliminary phylogenetic analyses, and assisted with the write-up of the manuscript: GD. Provided funding, direction and assisted with the write-up of the manuscript: TR. Wrote the first draft, did most of the bench-work, the bioinformatic analysis, and data mining: PW. Undertook most of the phylogenetic analyses and assisted with the write-up of the manuscript: PW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000158-Ostertag1" id="pone.0000158-Ostertag1"></a>Ostertag EM, Kazazian HH Jr (2001) Biology of mammalian L1 retrotransposons. Annu Rev Genet  35: 501–538.  <ul class="find" data-citedArticleID="926210" data-doi="10.1146/annurev.genet.35.102401.091032"><li><a href="http://dx.doi.org/10.1146/annurev.genet.35.102401.091032" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Biology+of+mammalian+L1+retrotransposons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Biology+of+mammalian+L1+retrotransposons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000158-Lander1" id="pone.0000158-Lander1"></a>Lander ES, Linton LM, Birren B, Nusbaum C, Zody MC, et al.  (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–921.  <ul class="find" data-citedArticleID="926186"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Initial+sequencing+and+analysis+of+the+human+genome.&amp;auth=&amp;atitle=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000158-Kazazian1" id="pone.0000158-Kazazian1"></a>Kazazian HH Jr, Goodier JL (2002) LINE drive. retrotransposition and genome instability. Cell  110: 277–280.  <ul class="find" data-citedArticleID="926182" data-doi="10.1016/s0092-8674(02)00868-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(02)00868-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LINE+drive.+retrotransposition+and+genome+instability." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LINE+drive.+retrotransposition+and+genome+instability.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000158-Yang1" id="pone.0000158-Yang1"></a>Yang Z, Boffelli D, Boonmark N, Schwartz K, Lawn R (1998) Apolipoprotein(a) gene enhancer resides within a LINE element. J Biol Chem  273: 891–897.  <ul class="find" data-citedArticleID="926248" data-doi="10.1074/jbc.273.2.891"><li><a href="http://dx.doi.org/10.1074/jbc.273.2.891" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Apolipoprotein%28a%29+gene+enhancer+resides+within+a+LINE+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Apolipoprotein%28a%29+gene+enhancer+resides+within+a+LINE+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000158-Han1" id="pone.0000158-Han1"></a>Han JS, Szak ST, Boeke JD (2004) Transcriptional disruption by the L1 retrotransposon and implications for mammalian transcriptomes. Nature  429: 268–274.  <ul class="find" data-citedArticleID="926176" data-doi="10.1038/nature02536"><li><a href="http://dx.doi.org/10.1038/nature02536" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+disruption+by+the+L1+retrotransposon+and+implications+for+mammalian+transcriptomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+disruption+by+the+L1+retrotransposon+and+implications+for+mammalian+transcriptomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000158-Lyon1" id="pone.0000158-Lyon1"></a>Lyon MF (1998) X-chromosome inactivation: a repeat hypothesis. Cytogenet Cell Genet  80: 133–137.  <ul class="find" data-citedArticleID="926188" data-doi="10.1159/000014969"><li><a href="http://dx.doi.org/10.1159/000014969" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X-chromosome+inactivation%3A+a+repeat+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X-chromosome+inactivation%3A+a+repeat+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000158-Hansen1" id="pone.0000158-Hansen1"></a>Hansen RS (2003) X inactivation-specific methylation of LINE-1 elements by DNMT3B: implications for the Lyon repeat hypothesis. Hum Mol Genet  12: 2559–2567.  <ul class="find" data-citedArticleID="926178" data-doi="10.1093/hmg/ddg268"><li><a href="http://dx.doi.org/10.1093/hmg/ddg268" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=X+inactivation-specific+methylation+of+LINE-1+elements+by+DNMT3B%3A+implications+for+the+Lyon+repeat+hypothesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22X+inactivation-specific+methylation+of+LINE-1+elements+by+DNMT3B%3A+implications+for+the+Lyon+repeat+hypothesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000158-Dewannieux1" id="pone.0000158-Dewannieux1"></a>Dewannieux M, Esnault C, Heidmann T (2003) LINE-mediated retrotransposition of marked Alu sequences. Nat Genet  35: 41–48.  <ul class="find" data-citedArticleID="926172" data-doi="10.1038/ng1223"><li><a href="http://dx.doi.org/10.1038/ng1223" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LINE-mediated+retrotransposition+of+marked+Alu+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LINE-mediated+retrotransposition+of+marked+Alu+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000158-Ohshima1" id="pone.0000158-Ohshima1"></a>Ohshima K, Hattori M, Yada T, Gojobori T, Sakaki Y, et al.  (2003) Whole-genome screening indicates a possible burst of formation of processed pseudogenes and Alu repeats by particular L1 subfamilies in ancestral primates. Genome Biol  4: R74.  <ul class="find" data-citedArticleID="926208"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Whole-genome+screening+indicates+a+possible+burst+of+formation+of+processed+pseudogenes+and+Alu+repeats+by+particular+L1+subfamilies+in+ancestral+primates.&amp;auth=&amp;atitle=Whole-genome+screening+indicates+a+possible+burst+of+formation+of+processed+pseudogenes+and+Alu+repeats+by+particular+L1+subfamilies+in+ancestral+primates." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-genome+screening+indicates+a+possible+burst+of+formation+of+processed+pseudogenes+and+Alu+repeats+by+particular+L1+subfamilies+in+ancestral+primates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-genome+screening+indicates+a+possible+burst+of+formation+of+processed+pseudogenes+and+Alu+repeats+by+particular+L1+subfamilies+in+ancestral+primates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000158-Smit1" id="pone.0000158-Smit1"></a>Smit AF, Toth G, Riggs AD, Jurka J (1995) Ancestral, mammalian-wide subfamilies of LINE-1 repetitive sequences. J Mol Biol  264: 401–417.  <ul class="find" data-citedArticleID="926220" data-doi="10.1006/jmbi.1994.0095"><li><a href="http://dx.doi.org/10.1006/jmbi.1994.0095" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancestral%2C+mammalian-wide+subfamilies+of+LINE-1+repetitive+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancestral%2C+mammalian-wide+subfamilies+of+LINE-1+repetitive+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000158-Waddell1" id="pone.0000158-Waddell1"></a>Waddell PJ, Kishino H, Ota R (2001) A phylogenetic foundation for comparative mammalian genomics. Genome Inform Ser Workshop Genome Inform  12: 141–154.  <ul class="find" data-citedArticleID="926232"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+phylogenetic+foundation+for+comparative+mammalian+genomics.&amp;auth=&amp;atitle=A+phylogenetic+foundation+for+comparative+mammalian+genomics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+phylogenetic+foundation+for+comparative+mammalian+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+phylogenetic+foundation+for+comparative+mammalian+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000158-Waddell2" id="pone.0000158-Waddell2"></a>Waddell PJ, Okada N, Hasegawa M (1999) Towards resolving the interordinal relationships of placental mammals. Systematic Biology. Syst Biol  48: 1–5.  <ul class="find" data-citedArticleID="926234"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Towards+resolving+the+interordinal+relationships+of+placental+mammals.+Systematic+Biology.&amp;auth=&amp;atitle=Towards+resolving+the+interordinal+relationships+of+placental+mammals.+Systematic+Biology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+resolving+the+interordinal+relationships+of+placental+mammals.+Systematic+Biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+resolving+the+interordinal+relationships+of+placental+mammals.+Systematic+Biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000158-Madsen1" id="pone.0000158-Madsen1"></a>Madsen O, Scally M, Douady CJ, Kao DJ, DeBry RW, et al.  (2001) Parallel adaptive radiations in two major clades of placental mammals. Nature  409: 610–614.  <ul class="find" data-citedArticleID="926192" data-doi="10.1038/35054544"><li><a href="http://dx.doi.org/10.1038/35054544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+adaptive+radiations+in+two+major+clades+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+adaptive+radiations+in+two+major+clades+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000158-Murphy1" id="pone.0000158-Murphy1"></a>Murphy WJ, Eizirik E, Johnson WE, Zhang YP, Ryder OA, et al.  (2001) Molecular phylogenetics and the origins of placental mammals. Nature  409: 614–618.  <ul class="find" data-citedArticleID="926198" data-doi="10.1038/35054550"><li><a href="http://dx.doi.org/10.1038/35054550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogenetics+and+the+origins+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogenetics+and+the+origins+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000158-Murphy2" id="pone.0000158-Murphy2"></a>Murphy WJ, Eizirik E, O'Brien SJ, Madsen O, Scally M, et al.  (2001) Resolution of the Early Placental Mammal Radiation Using Bayesian Phylogenetics. Science  294: 2348–2351.  <ul class="find" data-citedArticleID="926200" data-doi="10.1126/science.1067179"><li><a href="http://dx.doi.org/10.1126/science.1067179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Resolution+of+the+Early+Placental+Mammal+Radiation+Using+Bayesian+Phylogenetics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Resolution+of+the+Early+Placental+Mammal+Radiation+Using+Bayesian+Phylogenetics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000158-Thomsen1" id="pone.0000158-Thomsen1"></a>Thomsen PD, Miller JR (1996) Pig genome analysis: differential distribution of SINE and LINE sequences is less pronounced than in the human and mouse genomes. Mamm Genome  7: 42–46.  <ul class="find" data-citedArticleID="926230" data-doi="10.1007/s003359900010"><li><a href="http://dx.doi.org/10.1007/s003359900010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pig+genome+analysis%3A+differential+distribution+of+SINE+and+LINE+sequences+is+less+pronounced+than+in+the+human+and+mouse+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pig+genome+analysis%3A+differential+distribution+of+SINE+and+LINE+sequences+is+less+pronounced+than+in+the+human+and+mouse+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000158-Waters1" id="pone.0000158-Waters1"></a>Waters PD, Dobigny G, Pardini AT, Robinson TJ (2004) LINE-1 distribution in Afrotheria and Xenarthra: implications for understanding the evolution of LINE-1 in eutherian genomes. Chromosoma  113: 137–144.  <ul class="find" data-citedArticleID="926244" data-doi="10.1007/s00412-004-0301-9"><li><a href="http://dx.doi.org/10.1007/s00412-004-0301-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LINE-1+distribution+in+Afrotheria+and+Xenarthra%3A+implications+for+understanding+the+evolution+of+LINE-1+in+eutherian+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LINE-1+distribution+in+Afrotheria+and+Xenarthra%3A+implications+for+understanding+the+evolution+of+LINE-1+in+eutherian+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000158-Scully1" id="pone.0000158-Scully1"></a>Scully M, Madsen O, Douady CJ, de Jong WW, Stanhope MJ, et al.  (2001) Molecular evidence for the major clades of placental mammals. J Mamm Evol  8: 239–277.  <ul class="find" data-citedArticleID="926216"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Molecular+evidence+for+the+major+clades+of+placental+mammals.&amp;auth=&amp;atitle=Molecular+evidence+for+the+major+clades+of+placental+mammals." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+evidence+for+the+major+clades+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+evidence+for+the+major+clades+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000158-Springer1" id="pone.0000158-Springer1"></a>Springer MS, Scally M, Madsen O, de Jong WW, Douady CJ, et al.  (2004) The use of composite taxa in supermatrices. Mol Phylogenet Evol  30: 883–884.  <ul class="find" data-citedArticleID="926222" data-doi="10.1016/j.ympev.2003.09.019"><li><a href="http://dx.doi.org/10.1016/j.ympev.2003.09.019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+use+of+composite+taxa+in+supermatrices." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+use+of+composite+taxa+in+supermatrices.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000158-Kriegs1" id="pone.0000158-Kriegs1"></a>Kriegs JO, Churakov G, Kiefmann M, Jordan U, Brosius J, et al.  (2006) Retroposed elements as archives for the evolutionary history of placental mammals. PLoS Biol  4: e91.  <ul class="find" data-citedArticleID="926184" data-doi="10.1371/journal.pbio.0040091"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040091" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Retroposed+elements+as+archives+for+the+evolutionary+history+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Retroposed+elements+as+archives+for+the+evolutionary+history+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000158-Wilcox1" id="pone.0000158-Wilcox1"></a>Wilcox TP, Zwickl DJ, Heath TA, Hillis DM (2002) Pylogenetic relationships of the warf boas and a comparison of Bayesian and boostrap measures of phylogenetic support. Mol Phylogenet Evol  25: 361–371.  <ul class="find" data-citedArticleID="926246"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Pylogenetic+relationships+of+the+warf+boas+and+a+comparison+of+Bayesian+and+boostrap+measures+of+phylogenetic+support.&amp;auth=&amp;atitle=Pylogenetic+relationships+of+the+warf+boas+and+a+comparison+of+Bayesian+and+boostrap+measures+of+phylogenetic+support." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pylogenetic+relationships+of+the+warf+boas+and+a+comparison+of+Bayesian+and+boostrap+measures+of+phylogenetic+support." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pylogenetic+relationships+of+the+warf+boas+and+a+comparison+of+Bayesian+and+boostrap+measures+of+phylogenetic+support.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000158-Waddell3" id="pone.0000158-Waddell3"></a>Waddell PJ, Shelley S (2003) Evaluating placental inter-ordinal phylogenies with novel sequences including RAG1, gamma-fibrinogen, ND6, and mt-tRNA, plus MCMC-driven nucleotide, amino acid, and codon models. Mol Phylogenet Evol  28: 197–224.  <ul class="find" data-citedArticleID="926236" data-doi="10.1016/s1055-7903(03)00115-5"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00115-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluating+placental+inter-ordinal+phylogenies+with+novel+sequences+including+RAG1%2C+gamma-fibrinogen%2C+ND6%2C+and+mt-tRNA%2C+plus+MCMC-driven+nucleotide%2C+amino+acid%2C+and+codon+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluating+placental+inter-ordinal+phylogenies+with+novel+sequences+including+RAG1%2C+gamma-fibrinogen%2C+ND6%2C+and+mt-tRNA%2C+plus+MCMC-driven+nucleotide%2C+amino+acid%2C+and+codon+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000158-Springer2" id="pone.0000158-Springer2"></a>Springer MS, Murphy WJ, Eizirik E, O'Brien SJ (2003) Placental diversification and the Cretaceous-Tertiary boundary. Proc Natl Acad Sci U S A  100: 1056–1061.  <ul class="find" data-citedArticleID="926224" data-doi="10.1073/pnas.0334222100"><li><a href="http://dx.doi.org/10.1073/pnas.0334222100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Placental+diversification+and+the+Cretaceous-Tertiary+boundary." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Placental+diversification+and+the+Cretaceous-Tertiary+boundary.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000158-Delsuc1" id="pone.0000158-Delsuc1"></a>Delsuc F, Vizcaino SF, Douzery EJ (2004) Influence of Tertiary paleoenvironmental changes on the diversification of South American mammals: a relaxed molecular clock study within xenarthrans. BMC Evol Biol  4: 11.  <ul class="find" data-citedArticleID="926168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Influence+of+Tertiary+paleoenvironmental+changes+on+the+diversification+of+South+American+mammals%3A+a+relaxed+molecular+clock+study+within+xenarthrans.&amp;auth=&amp;atitle=Influence+of+Tertiary+paleoenvironmental+changes+on+the+diversification+of+South+American+mammals%3A+a+relaxed+molecular+clock+study+within+xenarthrans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Influence+of+Tertiary+paleoenvironmental+changes+on+the+diversification+of+South+American+mammals%3A+a+relaxed+molecular+clock+study+within+xenarthrans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Influence+of+Tertiary+paleoenvironmental+changes+on+the+diversification+of+South+American+mammals%3A+a+relaxed+molecular+clock+study+within+xenarthrans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000158-Nikaido1" id="pone.0000158-Nikaido1"></a>Nikaido M, Nishihara H, Hukumoto Y, Okada N (2003) Ancient SINEs from African endemic mammals. Mol Biol Evol  20: 522–527.  <ul class="find" data-citedArticleID="926202" data-doi="10.1093/molbev/msg052"><li><a href="http://dx.doi.org/10.1093/molbev/msg052" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ancient+SINEs+from+African+endemic+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ancient+SINEs+from+African+endemic+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000158-Madsen2" id="pone.0000158-Madsen2"></a>Madsen O, Scally M, Douady CJ, Kao DJ, DeBry RW, et al.  (2001) Parallel adaptive radiations in two major clades of placental mammals. Nature  409: 610–614.  <ul class="find" data-citedArticleID="926194" data-doi="10.1038/35054544"><li><a href="http://dx.doi.org/10.1038/35054544" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parallel+adaptive+radiations+in+two+major+clades+of+placental+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parallel+adaptive+radiations+in+two+major+clades+of+placental+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000158-AmrineMadsen1" id="pone.0000158-AmrineMadsen1"></a>Amrine-Madsen H, Koepfli KP, Wayne RK, Springer MS (2003) A new phylogenetic marker, apolipoprotein B, provides compelling evidence for eutherian relationships. Mol Phylogenet Evol  28: 225–240.  <ul class="find" data-citedArticleID="926162" data-doi="10.1016/s1055-7903(03)00118-0"><li><a href="http://dx.doi.org/10.1016/s1055-7903(03)00118-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+phylogenetic+marker%2C+apolipoprotein+B%2C+provides+compelling+evidence+for+eutherian+relationships." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+phylogenetic+marker%2C+apolipoprotein+B%2C+provides+compelling+evidence+for+eutherian+relationships.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000158-Robinson1" id="pone.0000158-Robinson1"></a>Robinson TJ, Fu B, Ferguson-Smith MA, Yang F (2004) Cross-species chromosome painting in the golden mole and elephant-shrew: support for the mammalian clades Afrotheria and Afroinsectiphillia but not Afroinsectivora. Proc R Soc Lond B Biol Sci  271: 1477–1484.  <ul class="find" data-citedArticleID="926212" data-doi="10.1098/rspb.2004.2754"><li><a href="http://dx.doi.org/10.1098/rspb.2004.2754" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cross-species+chromosome+painting+in+the+golden+mole+and+elephant-shrew%3A+support+for+the+mammalian+clades+Afrotheria+and+Afroinsectiphillia+but+not+Afroinsectivora." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cross-species+chromosome+painting+in+the+golden+mole+and+elephant-shrew%3A+support+for+the+mammalian+clades+Afrotheria+and+Afroinsectiphillia+but+not+Afroinsectivora.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000158-Nishihara1" id="pone.0000158-Nishihara1"></a>Nishihara H, Satta Y, Nikaido M, Thewissen JG, Stanhope MJ, et al.  (2005) A retroposon analysis of Afrotherian phylogeny. Mol Biol Evol  22: 1823–1833.  <ul class="find" data-citedArticleID="926204" data-doi="10.1093/molbev/msi179"><li><a href="http://dx.doi.org/10.1093/molbev/msi179" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+retroposon+analysis+of+Afrotherian+phylogeny." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+retroposon+analysis+of+Afrotherian+phylogeny.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000158-MacPhee1" id="pone.0000158-MacPhee1"></a>MacPhee RDE, Novacek M (1993)  Definition and relationships of Lipotyphla. In: Szalay FS, Novacek MJ, McKenna MC, editors. Mammalian Phylogeny: Placentals. New York: Springer-Verlag. pp. 13–29.  <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0000158-Shoshani1" id="pone.0000158-Shoshani1"></a>Shoshani J, McKenna MC (1998) Higher taxonomic relationships among extant mammals based on morphology, with selected comparisons of results from molecular data. Mol Phylogenet Evol  9: 572–584.  <ul class="find" data-citedArticleID="926218" data-doi="10.1006/mpev.1998.0520"><li><a href="http://dx.doi.org/10.1006/mpev.1998.0520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher+taxonomic+relationships+among+extant+mammals+based+on+morphology%2C+with+selected+comparisons+of+results+from+molecular+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher+taxonomic+relationships+among+extant+mammals+based+on+morphology%2C+with+selected+comparisons+of+results+from+molecular+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000158-Waddell4" id="pone.0000158-Waddell4"></a>Waddell PJ, Cao Y, Hauf J, Hasegawa M (1999) Using novel phylogenetic methods to evaluate mammalian mtDNA, including amino acid-invariant sites-LogDet plus site stripping, to detect internal conflicts in the data, with special reference to the positions of hedgehog, armadillo, and elephant. Syst Biol  48: 31–53.  <ul class="find" data-citedArticleID="926238"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Using+novel+phylogenetic+methods+to+evaluate+mammalian+mtDNA%2C+including+amino+acid-invariant+sites-LogDet+plus+site+stripping%2C+to+detect+internal+conflicts+in+the+data%2C+with+special+reference+to+the+positions+of+hedgehog%2C+armadillo%2C+and+elephant.&amp;auth=&amp;atitle=Using+novel+phylogenetic+methods+to+evaluate+mammalian+mtDNA%2C+including+amino+acid-invariant+sites-LogDet+plus+site+stripping%2C+to+detect+internal+conflicts+in+the+data%2C+with+special+reference+to+the+positions+of+hedgehog%2C+armadillo%2C+and+elephant." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+novel+phylogenetic+methods+to+evaluate+mammalian+mtDNA%2C+including+amino+acid-invariant+sites-LogDet+plus+site+stripping%2C+to+detect+internal+conflicts+in+the+data%2C+with+special+reference+to+the+positions+of+hedgehog%2C+armadillo%2C+and+elephant." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+novel+phylogenetic+methods+to+evaluate+mammalian+mtDNA%2C+including+amino+acid-invariant+sites-LogDet+plus+site+stripping%2C+to+detect+internal+conflicts+in+the+data%2C+with+special+reference+to+the+positions+of+hedgehog%2C+armadillo%2C+and+elephant.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000158-Delsuc2" id="pone.0000158-Delsuc2"></a>Delsuc F, Scally M, Madsen O, Stanhope MJ, de Jong WW, et al.  (2002) Molecular phylogeny of living xenarthrans and the impact of character and taxon sampling on the placental tree rooting. Mol Biol Evol  19: 1656–1671.  <ul class="find" data-citedArticleID="926170" data-doi="10.1093/oxfordjournals.molbev.a003989"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003989" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+phylogeny+of+living+xenarthrans+and+the+impact+of+character+and+taxon+sampling+on+the+placental+tree+rooting." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+phylogeny+of+living+xenarthrans+and+the+impact+of+character+and+taxon+sampling+on+the+placental+tree+rooting.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000158-Furano1" id="pone.0000158-Furano1"></a>Furano AV, Duvernell DD, Boissinot S (2004) L1 (LINE-1) retrotransposon diversity differs dramatically between mammals and fish. Trends Genet  20: 9–14.  <ul class="find" data-citedArticleID="926174" data-doi="10.1016/j.tig.2003.11.006"><li><a href="http://dx.doi.org/10.1016/j.tig.2003.11.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=L1+%28LINE-1%29+retrotransposon+diversity+differs+dramatically+between+mammals+and+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22L1+%28LINE-1%29+retrotransposon+diversity+differs+dramatically+between+mammals+and+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000158-Boissinot1" id="pone.0000158-Boissinot1"></a>Boissinot S, Chevret P, Furano AV (2000) L1 (LINE-1) retrotransposon evolution and amplification in recent human history. Mol Biol Evol  17: 915–928.  <ul class="find" data-citedArticleID="926164" data-doi="10.1093/oxfordjournals.molbev.a026372"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026372" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=L1+%28LINE-1%29+retrotransposon+evolution+and+amplification+in+recent+human+history." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22L1+%28LINE-1%29+retrotransposon+evolution+and+amplification+in+recent+human+history.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000158-Cabot1" id="pone.0000158-Cabot1"></a>Cabot EL, Angeletti B, Usdin K, Furano AV (1997) Rapid evolution of a young L1 (LINE-1) clade in recently speciated Rattus taxa. J Mol Evol  45: 412–423.  <ul class="find" data-citedArticleID="926166" data-doi="10.1007/pl00006246"><li><a href="http://dx.doi.org/10.1007/pl00006246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+evolution+of+a+young+L1+%28LINE-1%29+clade+in+recently+speciated+Rattus+taxa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+evolution+of+a+young+L1+%28LINE-1%29+clade+in+recently+speciated+Rattus+taxa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000158-Malik1" id="pone.0000158-Malik1"></a>Malik HS, Burke WD, Eickbush TH (1999) The age and evolution of non-LTR retrotransposable elements. Mol Biol Evol  16: 793–805.  <ul class="find" data-citedArticleID="926196" data-doi="10.1093/oxfordjournals.molbev.a026164"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026164" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+age+and+evolution+of+non-LTR+retrotransposable+elements." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+age+and+evolution+of+non-LTR+retrotransposable+elements.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000158-Notredame1" id="pone.0000158-Notredame1"></a>Notredame C, Higgins DG, Heringa J (2000) T-Coffee: A novel method for fast and accurate multiple sequence alignment. J Mol Biol  302: 205–217.  <ul class="find" data-citedArticleID="926206" data-doi="10.1006/jmbi.2000.4042"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.4042" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000158-Swofford1" id="pone.0000158-Swofford1"></a>Swofford DLPAUP: Phylogenetic Analysis using Parsimony (and other methods). 4 ed. Sunderland, Massachusetts: Sinauer Associates.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0000158-Ronquist1" id="pone.0000158-Ronquist1"></a>Ronquist F, Huelsenbeck JP (2003) MrBayes 3: Bayesian phylogenetic inference under mixed models. Bioinformatics  19: 1572–1574.  <ul class="find" data-citedArticleID="926214" data-doi="10.1093/bioinformatics/btg180"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MrBayes+3%3A+Bayesian+phylogenetic+inference+under+mixed+models.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000158-Hasegawa1" id="pone.0000158-Hasegawa1"></a>Hasegawa M, Kishino K, Yano T (1985) Dating the human-ape splitting by a molecular clock of mitochondrial DNA. J Mol Evol  22: 160–174.  <ul class="find" data-citedArticleID="926180" data-doi="10.1007/bf02101694"><li><a href="http://dx.doi.org/10.1007/bf02101694" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dating+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dating+the+human-ape+splitting+by+a+molecular+clock+of+mitochondrial+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000158-Swofford2" id="pone.0000158-Swofford2"></a>Swofford DL, Olsen GJ, Waddell PJ, Hillis DM (1996)  Phylogenetic inference. In: Hillis DM, Moritz C, Mable BK, editors. Molecular systematics. Sunderland, Massachusetts: Sinauer. pp. 407–514.  2nd ed. <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0000158-Waddell5" id="pone.0000158-Waddell5"></a>Waddell PJ, Penny D (1996)  Evolutionary trees of apes and humans from DNA sequences. In: Lock AJ, Peters CR, editors. Handbook of Symbolic Evolution. Oxford: Clarendon Press. pp. 53–73.  <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0000158-Waddell6" id="pone.0000158-Waddell6"></a>Waddell PJ, Steel MA (1997) General time-reversible distances with unequal rates across sites: mixing gamma and inverse Gaussian distributions with invariant sites. Mol Phylogenet Evol  8: 398–414.  <ul class="find" data-citedArticleID="926242" data-doi="10.1006/mpev.1997.0452"><li><a href="http://dx.doi.org/10.1006/mpev.1997.0452" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=General+time-reversible+distances+with+unequal+rates+across+sites%3A+mixing+gamma+and+inverse+Gaussian+distributions+with+invariant+sites." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22General+time-reversible+distances+with+unequal+rates+across+sites%3A+mixing+gamma+and+inverse+Gaussian+distributions+with+invariant+sites.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.XML" value="101423"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.PDF" value="284629"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g001.PNG_L" value="847710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g001.PNG_M" value="82137"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g001.PNG_S" value="3086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g001.TIF" value="1425044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g001.PNG_I" value="34702"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g002.PNG_L" value="277247"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g002.PNG_M" value="26459"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g002.PNG_S" value="2297"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g002.TIF" value="544132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.g002.PNG_I" value="17494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000158.s001.XLS" value="55296"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000158&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000158" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000158&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000158&volume=&issue=&title=Evolutionary History of LINE-1 in the Major Clades of Placental Mammals&author_name=Paul%20D.%20Waters%2C%20Gauthier%20Dobigny%2C%20Peter%20J.%20Waddell%2C%20Terence%20J.%20Robinson&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000158" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158&amp;t=Evolutionary%20History%20of%20LINE-1%20in%20the%20Major%20Clades%20of%20Placental%20Mammals" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158&title=Evolutionary%20History%20of%20LINE-1%20in%20the%20Major%20Clades%20of%20Placental%20Mammals&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158&amp;title=Evolutionary%20History%20of%20LINE-1%20in%20the%20Major%20Clades%20of%20Placental%20Mammals" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000158&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Evolutionary History of LINE-1 in the Major Clades of Placental Mammals';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000158';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Evolutionary%20History%20of%20LINE-1%20in%20the%20Major%20Clades%20of%20Placental%20Mammals http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000158" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000158" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Animal+phylogenetics%22" title="Search for articles in the subject area:'Animal phylogenetics'"><div class="flagText">Animal phylogenetics</div></a>
              <div data-categoryid="20335" data-articleid="23154"
                   data-categoryname="Animal phylogenetics"
                   class="flagImage" title="Flag 'Animal phylogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Elephants%22" title="Search for articles in the subject area:'Elephants'"><div class="flagText">Elephants</div></a>
              <div data-categoryid="43025" data-articleid="23154"
                   data-categoryname="Elephants"
                   class="flagImage" title="Flag 'Elephants' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23154"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23154"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Paleogenetics%22" title="Search for articles in the subject area:'Paleogenetics'"><div class="flagText">Paleogenetics</div></a>
              <div data-categoryid="33815" data-articleid="23154"
                   data-categoryname="Paleogenetics"
                   class="flagImage" title="Flag 'Paleogenetics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23154"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="23154"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Xenarthra%22" title="Search for articles in the subject area:'Xenarthra'"><div class="flagText">Xenarthra</div></a>
              <div data-categoryid="43767" data-articleid="23154"
                   data-categoryname="Xenarthra"
                   class="flagImage" title="Flag 'Xenarthra' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2960'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=8901'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=8830&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=3139">How many steps</a><br>Posted by jeales</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
