

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Divergent Evolution among Teleost V1r Receptor Genes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000379"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000379" />

    <meta name="citation_title" content="Divergent Evolution among Teleost V1r Receptor Genes"/>
    <meta itemprop="name" content="Divergent Evolution among Teleost V1r Receptor Genes"/>

      <meta name="citation_author" content="Patrick Pfister"/>
            <meta name="citation_author_institution" content="Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland"/>
            <meta name="citation_author_institution" content="National Center of Competence in Research (NCCR) Frontiers in Genetics, University of Geneva, Geneva, Switzerland"/>
      <meta name="citation_author" content="Jerome Randall"/>
            <meta name="citation_author_institution" content="Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland"/>
            <meta name="citation_author_institution" content="National Center of Competence in Research (NCCR) Frontiers in Genetics, University of Geneva, Geneva, Switzerland"/>
      <meta name="citation_author" content="Juan I. Montoya-Burgos"/>
            <meta name="citation_author_institution" content="Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland"/>
      <meta name="citation_author" content="Ivan Rodriguez"/>
            <meta name="citation_author_institution" content="Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland"/>
            <meta name="citation_author_institution" content="National Center of Competence in Research (NCCR) Frontiers in Genetics, University of Geneva, Geneva, Switzerland"/>

    <meta name="citation_date" content="2007/4/18"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000379.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e379"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=The odorant receptor repertoire of teleost fish.; citation_author=TS Alioto; citation_author=J Ngai; citation_journal_title=BMC Genomics; citation_volume=6; citation_number=1; citation_pages=173; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A second class of chemosensory receptors in the olfactory epithelium.; citation_author=SD Liberles; citation_author=LB Buck; citation_journal_title=Nature; citation_volume=442; citation_number=2; citation_pages=645-650; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The repertoire of trace amine G-protein-coupled receptors: large expansion in zebrafish.; citation_author=DE Gloriam; citation_author=TK Bjarnadottir; citation_author=YL Yan; citation_author=JH Postlethwait; citation_author=HB Schioth; citation_journal_title=Mol Phylogenet Evol; citation_volume=35; citation_number=3; citation_pages=470-482; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Evolution of vomeronasal-type odorant receptor genes in the zebrafish genome.; citation_author=Y Hashiguchi; citation_author=M Nishida; citation_journal_title=Gene; citation_volume=362; citation_number=4; citation_pages=19-28; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Cloning and localization of two multigene receptor families in goldfish olfactory epithelium.; citation_author=Y Cao; citation_author=BC Oh; citation_author=L Stryer; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=95; citation_number=5; citation_pages=11987-11992; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The repertoire of olfactory C family G protein-coupled receptors in zebrafish: candidate chemosensory receptors for amino acids.; citation_author=TS Alioto; citation_author=J Ngai; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=6; citation_pages=309; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The gene repertoire and the common evolutionary history of glutamate, pheromone (V2R), taste(1) and other related G protein-coupled receptors.; citation_author=TK Bjarnadottir; citation_author=R Fredriksson; citation_author=HB Schioth; citation_journal_title=Gene; citation_volume=362; citation_number=7; citation_pages=70-84; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pheromone detection mediated by a V1r vomeronasal receptor.; citation_author=C Boschat; citation_author=C Pelofi; citation_author=O Randin; citation_author=D Roppolo; citation_author=C Luscher; citation_journal_title=Nat Neurosci; citation_volume=5; citation_number=8; citation_pages=1261-1262; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Deficient pheromone responses in mice lacking a cluster of vomeronasal receptor genes.; citation_author=K Del Punta; citation_author=T Leinders-Zufall; citation_author=I Rodriguez; citation_author=D Jukam; citation_author=CJ Wysocki; citation_journal_title=Nature; citation_volume=419; citation_number=9; citation_pages=70-74; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Divergent V1R repertoires in five species: Amplification in rodents, decimation in primates, and a surprisingly small repertoire in dogs.; citation_author=JM Young; citation_author=M Kambere; citation_author=BJ Trask; citation_author=RP Lane; citation_journal_title=Genome Res; citation_volume=15; citation_number=10; citation_pages=231-240; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Species specificity in rodent pheromone receptor repertoires.; citation_author=RP Lane; citation_author=J Young; citation_author=T Newman; citation_author=BJ Trask; citation_journal_title=Genome Res; citation_volume=14; citation_number=11; citation_pages=603-608; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Dramatic variation of the vomeronasal pheromone gene repertoire among five orders of placental and marsupial mammals.; citation_author=W Grus; citation_author=P Shi; citation_author=Y Zhang; citation_author=J Zhang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=12; citation_pages=5767-5772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Rapid turnover and species-specificity of vomeronasal pheromone receptor genes in mice and rats.; citation_author=WE Grus; citation_author=J Zhang; citation_journal_title=Gene; citation_volume=340; citation_number=13; citation_pages=303-312; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Multiple new and isolated families within the mouse superfamily of V1r vomeronasal receptors.; citation_author=I Rodriguez; citation_author=K Del Punta; citation_author=A Rothman; citation_author=T Ishii; citation_author=P Mombaerts; citation_journal_title=Nat Neurosci; citation_volume=5; citation_number=14; citation_pages=134-140; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Odorant and vomeronasal receptor genes in two mouse genome assemblies.; citation_author=X Zhang; citation_author=I Rodriguez; citation_author=P Mombaerts; citation_author=S Firestein; citation_journal_title=Genomics; citation_volume=83; citation_number=15; citation_pages=802-811; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Olfactory expression of a single and highly variable V1r pheromone receptor-like gene in fish species.; citation_author=P Pfister; citation_author=I Rodriguez; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=16; citation_pages=5489-5494; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Positive Selection During the Diversification of Class I Vomeronasal Receptor-like (V1RL) Genes, Putative Pheromone Receptor Genes, in Human and Primate Evolution.; citation_author=NI Mundy; citation_author=S Cook; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=17; citation_pages=1805-1810; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Sequence analysis of mouse vomeronasal receptor gene clusters reveals common promoter motifs and a history of recent expansion.; citation_author=RP Lane; citation_author=T Cutforth; citation_author=R Axel; citation_author=L Hood; citation_author=BJ Trask; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=18; citation_pages=291-296; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Adaptive diversification of vomeronasal receptor 1 genes in rodents.; citation_author=P Shi; citation_author=JP Bielawski; citation_author=H Yang; citation_author=YP Zhang; citation_journal_title=J Mol Evol; citation_volume=60; citation_number=19; citation_pages=566-576; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene and genome duplications in vertebrates: the one-to-four (-to-eight in fish) rule and the evolution of novel gene functions.; citation_author=A Meyer; citation_author=M Schartl; citation_journal_title=Curr Opin Cell Biol; citation_volume=11; citation_number=20; citation_pages=699-704; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Gen(om)e duplications in the evolution of early vertebrates.; citation_author=A Sidow; citation_journal_title=Curr Opin Genet Dev; citation_volume=6; citation_number=21; citation_pages=715-722; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Extensive gene duplication in the early evolution of animals before the parazoan-eumetazoan split demonstrated by G proteins and protein tyrosine kinases from sponge and hydra.; citation_author=H Suga; citation_author=M Koyanagi; citation_author=D Hoshiyama; citation_author=K Ono; citation_author=N Iwabe; citation_journal_title=J Mol Evol; citation_volume=48; citation_number=22; citation_pages=646-653; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Major events in the genome evolution of vertebrates: paranome age and size differ considerably between ray-finned fishes and land vertebrates.; citation_author=K Vandepoele; citation_author=W De Vos; citation_author=JS Taylor; citation_author=A Meyer; citation_author=Y Van de Peer; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=23; citation_pages=1638-1643; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Fugu genome analysis provides evidence for a whole-genome duplication early during the evolution of ray-finned fishes.; citation_author=A Christoffels; citation_author=EG Koh; citation_author=JM Chia; citation_author=S Brenner; citation_author=S Aparicio; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=24; citation_pages=1146-1151; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Medaka–a model organism from the far East.; citation_author=J Wittbrodt; citation_author=A Shima; citation_author=M Schartl; citation_journal_title=Nat Rev Genet; citation_volume=3; citation_number=25; citation_pages=53-64; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Tetraodon fluviatilis, a new puffer fish model for genome studies.; citation_author=T Crnogorac-Jurcevic; citation_author=JR Brown; citation_author=H Lehrach; citation_author=LC Schalkwyk; citation_journal_title=Genomics; citation_volume=41; citation_number=26; citation_pages=177-184; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Novel evolutionary relationship among four fish model systems.; citation_author=WJ Chen; citation_author=G Orti; citation_author=A Meyer; citation_journal_title=Trends Genet; citation_volume=20; citation_number=27; citation_pages=424-431; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Medaka and zebrafish, an evolutionary twin study.; citation_author=M Furutani-Seiki; citation_author=J Wittbrodt; citation_journal_title=Mech Dev; citation_volume=121; citation_number=28; citation_pages=629-637; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A novel family of genes encoding putative pheromone receptors in mammals.; citation_author=C Dulac; citation_author=R Axel; citation_journal_title=Cell; citation_volume=83; citation_number=29; citation_pages=195-206; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=PipMaker--a web server for aligning two genomic DNA sequences.; citation_author=S Schwartz; citation_author=Z Zha ng; citation_author=KA Frazer; citation_author=A Smit; citation_author=C Riemer; citation_journal_title=Genome Res; citation_volume=10; citation_number=30; citation_pages=577-586; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator.; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_journal_title=Genome Res; citation_volume=14; citation_number=31; citation_pages=1188-1190; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice.; citation_author=JD Thompson; citation_author=DG Higgins; citation_author=TJ Gibson; citation_journal_title=Nucleic Acids Res; citation_volume=22; citation_number=32; citation_pages=4673-4680; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution.; citation_author=D Posada; citation_author=KA Crandall; citation_journal_title=Bioinformatics; citation_volume=14; citation_number=33; citation_pages=817-818; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.; citation_author=S Guindon; citation_author=O Gascuel; citation_journal_title=Syst Biol; citation_volume=52; citation_number=34; citation_pages=696-704; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=RRTree: relative-rate tests between groups of sequences on a phylogenetic tree.; citation_author=M Robinson-Rechavi; citation_author=D Huchon; citation_journal_title=Bioinformatics; citation_volume=16; citation_number=35; citation_pages=296-297; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood.; citation_author=Z Yang; citation_journal_title=Comput Appl Biosci; citation_volume=13; citation_number=36; citation_pages=555-556; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=A maximum likelihood method for detecting functional divergence at individual codon sites, with application to gene family evolution.; citation_author=JP Bielawski; citation_author=Z Yang; citation_journal_title=J Mol Evol; citation_volume=59; citation_number=37; citation_pages=121-132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level.; citation_author=J Zhang; citation_author=R Nielsen; citation_author=Z Yang; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=38; citation_pages=2472-2479; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Bayes empirical bayes inference of amino acid sites under positive selection.; citation_author=Z Yang; citation_author=WS Wong; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=39; citation_pages=1107-1118; citation_date=2005; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Divergent Evolution among Teleost V1r Receptor Genes"/>
    <meta name="twitter:description" content="The survival of vertebrate species is dependent on the ability of individuals to adequately interact with each other, a function often mediated by the olfactory system. Diverse olfactory receptor repertoires are used by this system to recognize chemicals. Among these receptors, the V1rs, encoded by a very large gene family in most mammals, are able to detect pheromones. Teleosts, which also express V1r receptors, possess a very limited V1r repertoire. Here, taking advantage of the possibility to unequivocally identify V1r orthologs in teleosts, we analyzed the olfactory expression and evolutionary constraints of a pair of clustered fish V1r receptor genes, V1r1 and V1r2. Orthologs of the two genes were found in zebrafish, medaka, and threespine stickleback, but a single representative was observed in tetraodontidae species. Analysis of V1r1 and V1r2 sequences from 12 different euteleost species indicate different evolutionary rates between the two paralogous genes, leading to a highly conserved V1r2 gene and a V1r1 gene under more relaxed selective constraint. Moreover, positively-selected sites were detected in specific branches of the V1r1 clade. Our results suggest a conserved agonist specificity of the V1R2 receptor between euteleost species, its loss in the tetraodontidae lineage, and the acquisition of different chemosensory characteristics for the V1R1 receptor."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000379.g006"/>

  <meta property="og:title" content="Divergent Evolution among Teleost V1r Receptor Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8170'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4270'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6634&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000379">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000379" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000379&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Divergent Evolution among Teleost V1r Receptor Genes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Patrick Pfister, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland, 
                  National Center of Competence in Research (NCCR) Frontiers in Genetics, University of Geneva, Geneva, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jerome Randall, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland, 
                  National Center of Competence in Research (NCCR) Frontiers in Genetics, University of Geneva, Geneva, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Juan I. Montoya-Burgos, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ivan Rodriguez
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:ivan.rodriguez@zoo.unige.ch">ivan.rodriguez@zoo.unige.ch</a></p>

                <p>Affiliations:
                  Department of Zoology and Animal Biology, University of Geneva, Geneva, Switzerland, 
                  National Center of Competence in Research (NCCR) Frontiers in Genetics, University of Geneva, Geneva, Switzerland
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 18, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000379</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000379-g001" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-t001" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-g002" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-g003" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-g004" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-g005" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-g006" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-t002" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000379-t003" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000379">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000379" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The survival of vertebrate species is dependent on the ability of individuals to adequately interact with each other, a function often mediated by the olfactory system. Diverse olfactory receptor repertoires are used by this system to recognize chemicals. Among these receptors, the V1rs, encoded by a very large gene family in most mammals, are able to detect pheromones. Teleosts, which also express V1r receptors, possess a very limited V1r repertoire. Here, taking advantage of the possibility to unequivocally identify V1r orthologs in teleosts, we analyzed the olfactory expression and evolutionary constraints of a pair of clustered fish V1r receptor genes, <em>V1r1</em> and <em>V1r2</em>. Orthologs of the two genes were found in zebrafish, medaka, and threespine stickleback, but a single representative was observed in tetraodontidae species. Analysis of <em>V1r1</em> and <em>V1r2</em> sequences from 12 different euteleost species indicate different evolutionary rates between the two paralogous genes, leading to a highly conserved <em>V1r2</em> gene and a <em>V1r1</em> gene under more relaxed selective constraint. Moreover, positively-selected sites were detected in specific branches of the <em>V1r1</em> clade. Our results suggest a conserved agonist specificity of the V1R2 receptor between euteleost species, its loss in the tetraodontidae lineage, and the acquisition of different chemosensory characteristics for the V1R1 receptor.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Pfister P, Randall J, Montoya-Burgos JI, Rodriguez I (2007) Divergent Evolution among Teleost V1r Receptor Genes. PLoS ONE 2(4):
          e379.
            doi:10.1371/journal.pone.0000379</p><p><strong>Academic Editor: </strong>Suzannah Rutherford, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received:</strong> February 9, 2007; <strong>Accepted:</strong> March 20, 2007; <strong>Published:</strong> April 18, 2007</p><p><strong>Copyright:</strong> © 2007 Pfister et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was funded by the Swiss National Science Foundation.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>In vertebrates, interindividual interactions related to reproduction are largely dependent on pheromonal communication. Chemosensory structures located in mammals in the nasal cavity and in teleosts in the olfactory rosette represent the major tools used for such exchanges. These structures contain thousands of olfactory sensory neurons, organized in pseudostratified neuroepithelia. Each olfactory sensory neuron extends a single dendrite towards the outside world and an axon which directly connects to the olfactory bulb, in the brain. A single or a few members of a remarkably large olfactory receptor gene repertoire are expressed by each olfactory sensory neuron. Thus, chemical information from the outside world is transported by multiple parallel lines to the olfactory bulb, where it is processed, directed towards various brain areas, and translated into a conscious or unconscious perception.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Four different classes of olfactory receptors have been described in teleosts: odorant, trace amine-associated, vomeronasal type 2 (V2r) and vomeronasal type 1 (V1r) receptors. Odorant receptors number 143 in zebrafish <a href="#pone.0000379-Alioto1">[1]</a>, and represent, as in mammals, one of the largest gene families in the genome. Trace amine-associated receptors <a href="#pone.0000379-Liberles1">[2]</a>, are also present in fish species (57 genes in zebrafish) <a href="#pone.0000379-Gloriam1">[3]</a>. V2rs react to amino acids in teleosts, and are relatively numerous (at least 24 potentially functional V2rs in zebrafish) <a href="#pone.0000379-Hashiguchi1">[4]</a>–<a href="#pone.0000379-Bjarnadottir1">[7]</a>. V1rs mediate pheromone detection in mice <a href="#pone.0000379-Boschat1">[8]</a>, <a href="#pone.0000379-DelPunta1">[9]</a> and are thought to play a similar role in other species. In mammals, V1r genes are highly divergent, often species-specific, and in rodents number over 100 <a href="#pone.0000379-Young1">[10]</a>–<a href="#pone.0000379-Rodriguez1">[14]</a>. The role played by the size and variability of these V1r repertoires is not understood yet. Thus, basic questions, such as the degree of potential redundancy between paralogous V1r genes, remain unanswered. However, positive Darwinian pressure has been shown to act on some of these receptor genes, between paralogous or orthologous sequences <a href="#pone.0000379-Grus1">[12]</a>, <a href="#pone.0000379-Zhang1">[15]</a>–<a href="#pone.0000379-Shi1">[19]</a>, indicating, at least that for some novel genes, novel functions were acquired.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The variability and profusion of V1r genes in some species result from a high rate of gene birth and death<a href="#pone.0000379-Grus1">[12]</a>, <a href="#pone.0000379-Grus2">[13]</a>, <a href="#pone.0000379-Shi1">[19]</a>. This unusual characteristic renders the identification of orthologous V1r pairs, if existent at the functional level, difficult to perform with certainty, even in species as closely related as mice and rats. Despite the extensive genomic duplications which affected teleost genomes <a href="#pone.0000379-Meyer1">[20]</a>–<a href="#pone.0000379-Christoffels1">[24]</a>, fish V1r genes appear not to have gone through the flourishing diversification and amplification characterizing mammalian pheromone receptor genes. This peculiar situation allows the study of orthologous V1rs between euteleost species (<a href="#pone-0000379-t001">Table 1</a>) whose radiation dates back over 110 mio. years (<a href="#pone-0000379-g001">Figure 1</a>) <a href="#pone.0000379-Wittbrodt1">[25]</a>–<a href="#pone.0000379-FurutaniSeiki1">[28]</a>. We here report such an analysis.</p>
<div class="figure" id="pone-0000379-g001"><div class="img"><a name="pone-0000379-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Phylogenetic relationships between zebrafish, stickleback, medaka, tetraodon and fugu. The polytomy at the base of the Acanthomorpha superorder is still debated.</p>
<span>doi:10.1371/journal.pone.0000379.g001</span></div><div class="figure" id="pone-0000379-t001"><div class="img"><a name="pone-0000379-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Names, abbreviations, families, orders and superorders corresponding to the teleost species used in this study.</span></strong></p><span>doi:10.1371/journal.pone.0000379.t001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A Linked Pair of Teleost V1r Receptor Genes</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>A single V1r gene, based on the zebrafish (<em>Danio rerio, Dr</em>) (Zv4, 2005), medaka (<em>Oryzias latipes</em>) (Medaka 1, 2005), tetraodon <em>(Tetraodon nigroviridis, Tn)</em> (Tetraodon 7, 2004) and fugu <em>(Takifugu rubripes, Tr)</em> (Fugu 2.0, 2004) assemblies, was previously identified in teleosts <a href="#pone.0000379-Pfister1">[16]</a>. Taking advantage of the newly released zebrafish (Zv6, 2006), medaka (Medaka 1, 2006) and threespine stickleback (<em>Gasterosteus aculeatus, Ga</em>) (BROAD S1, 2006) assemblies (see <a href="#s4">Materials and Methods</a>), and including the fugu (Fugu 4.0, 2005) and tetraodon databases, we performed a search for potential novel V1r sequences. Databases were mined with the TBLASTN and BLASTN algorithms using as queries all known fish V1r, mouse <em>V1rb2, V1rf3</em> and <em>V1re4</em> sequences. Our criteria for inclusion were an uninterrupted coding sequence of at least 850 nucleotides contained in a single exon (all known vertebrate V1r genes exhibit such characteristics), a potential translated product containing 7 transmembrane segments, and an expected value inferior to 10<sup>−2</sup>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We identified two different V1r sequences fulfilling these criteria in <em>Dr</em>, and found their corresponding orthologs in <em>Ga</em> and <em>Ol</em>, one of them corresponding to a previously described zebrafish V1r <a href="#pone.0000379-Pfister1">[16]</a>. A single V1r gene was found in <em>Tn</em> and <em>Tr</em>. We named the V1r common to all tested species <em>V1r1</em> and the one missing in the tetraodontidae lineage <em>V1r2</em>. We hypothesized the existence of a potential pseudogenized or distant <em>V1r2</em> sequence in tetraodontidae species, and tested this hypothesis by searching for sequences with a potential to encode 10 amino acid triplets conserved in <em>Ol, Dr, and Ga V1r2</em>: HLA (49), LTR (59), PQT (64), GCK (80), NMA (140), APR (153), GFC (167), TRA (228), FGI (245) and PVV (263), in sequences surrounding tetraodontidae <em>V1r1</em> genes (100kb upstream and downstream of tetraodon <em>V1r1</em>, and 11kb upstream and 10kb downstream of fugu <em>V1r1</em>). We estimated that for a sequence to be considered potentially related to a <em>V1r2</em> gene, at least 4 of the 10 conserved triplets were to be present, irrespective of the reading frame or the transcription orientation, and with a distance between each triplet not exceeding+/−12 amino acids relative to the one in the known <em>V1r2</em> sequences. No sequence fulfilling these criteria was identified, suggesting the absence of a <em>V1r2</em> gene in the vicinity of the <em>V1r1</em> tetraodontidae gene (but without excluding the potential presence of a highly degenerate <em>V1r2</em> pseudogene).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>We also performed PCRs at low stringency and/or used degenerate primers (see <a href="#s4">Materials and Methods</a>) on genomic <em>Tr</em> and <em>Tn</em> DNA, using <em>V1r2</em> primers located at positions conserved in all other identified fish <em>V1r2</em> genes. No amplicon corresponding to a potential <em>V1r2</em> gene or pseudogene was obtained, again supporting the potential absence of this gene in Tetraodontidae. However, our own experience makes us wary of such negative observations <a href="#pone.0000379-Pfister1">[16]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4"></a><p>Mammalian V1Rs exhibit 10 highly conserved amino acid residues (some of which are exclusively observed in V1Rs), and a conserved glycosylation site in extracellular loop 2 <a href="#pone.0000379-Rodriguez1">[14]</a>. 7 of the conserved residues and the glycosylation site were present in all teleost V1r sequences (<a href="#pone-0000379-g002">Figure 2</a>).</p>
<div class="figure" id="pone-0000379-g002"><div class="img"><a name="pone-0000379-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Alignment between teleost V1R1 (light blue) and V1R2 (blue) receptors. <em>Dr, Ga, Ol, Tr, Tn</em> and mouse (<em>Mm</em>) V1RF3 proteins are shown. Conserved residues (at least seven out of ten) are highlighted in blue. Asterisks indicate conserved residues in virtually all mouse V1Rs, and red letters correspond to the ones also conserved in teleosts. Empty squares show the position of positively selected sites in some genes of the V1r1 clade. Green boxes indicate the position of the conserved N-linked glycosylation sites (NXS/T). TM, transmembrane, IC, intracellular, EC, extracellular domains.</p>
<span>doi:10.1371/journal.pone.0000379.g002</span></div><a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5"></a><p>To dispose of a larger and therefore more reliable dataset, we then expanded our analysis to species pertaining to a single genus, <em>Danio</em>. Using PCR primers specific for <em>Dr V1r1</em> and <em>V1r2</em> sequences, we investigated the presence of the two genes in the following species: <em>D. albolineatus (Da), D. choprae (Dc), D. dangila (Dd), D. kerri (Dk), D. kyathit (Dky), D. malabaricus (Dm), D. nigrofasciatus (Dn), D. pantheri (Dp)</em> and <em>D. yoma (Dy)</em> (<a href="#pone-0000379-t001">Table 1</a>).<em>V1r1</em> and <em>V1r2</em> sequences were found in all tested <em>Danio</em> species.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6"></a><p>Analysis of the genomic location of the two V1r genes indicated that they were clustered on chromosomes 22 in zebrafish, 5 in medaka, and 17 in stickleback. Their transcription directions were inverted in all cases observed, in a bidirectional orientation, with an unusual short distance between coding sequences <a href="#pone.0000379-Rodriguez1">[14]</a>, ranging from 1.1kb (<em>Dm</em>) to 3.4 kb (<em>Ol</em>) (<a href="#pone-0000379-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0000379-g003"><div class="img"><a name="pone-0000379-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Teleost <em>V1r1</em> and <em>V1r2</em> genes are clustered in the genome. Relative positions and orientations of <em>V1r1</em> and <em>V1r2</em> genes in <em>Dr, Dm, Ol, Ga, Tr</em> and <em>Tn. V1r2</em> was not found in the tetraodontidae species. The grey square represents the position of the conserved <em>V1r1</em> 5′UTR region. Horizontal lines above the zebrafish sequence represent the position and size of the RNA probes used for the in situ hybridizations (probes I, II and III) (see <a href="#pone-0000379-g005">Figure 5</a>). The <em>V1r1</em> transcription start site is indicated by an asterisk.</p>
<span>doi:10.1371/journal.pone.0000379.g003</span></div>

<h4>A Conserved Sequence in the 5′UTR of <em>V1r1</em></h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>V1r receptor genes in mice exhibit a characteristic not observed in odorant receptor genes: an unusual conservation of non-coding sequences between members of a given family, which includes transcribed and non-transcribed regions <a href="#pone.0000379-Lane2">[18]</a>. We evaluated the potential homology of non-coding sequences between orthologous teleost <em>V1r1</em> genes (<em>Dr, Ol, Ga, Tr, Dm</em>, and <em>Tn</em>) by pairwise comparisons of the sequences using the Pipmaker analysis tool. We identified a 75 nucleotide segment located 5′ to the <em>V1r1</em> coding sequence containing 31 invariant nucleotides (<a href="#pone-0000379-g004">Figure 4</a>). The position of this conserved sequence relative to the start site ranged from -81 (<em>Dr</em>) to -185 (<em>Ol</em>) (<a href="#pone-0000379-g003">Figure 3</a>). 5′ RACE was performed and one transription start site was identified 404 bp upstream of the <em>V1r1</em> first ATG codon (<a href="#pone-0000379-g003">Figure 3</a>), indicating that the conserved sequence was part of the <em>V1r1</em> 5′UTR.</p>
<div class="figure" id="pone-0000379-g004"><div class="img"><a name="pone-0000379-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A sequence is conserved in the 5′UTR of <em>V1r1</em>. An alignment of the conserved residues in the 5′UTR of <em>Dr V1r1</em> mRNA is shown. A 75 base pair region is conserved in all tested species upstream of the translation start site of <em>V1r1</em>. A sequence logo was generated.</p>
<span>doi:10.1371/journal.pone.0000379.g004</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The conserved segment was not found duplicated 5′ to<em>V1r2</em> sequence, and was not identified associated with any V1r mouse gene. No significant conservation of other non-coding segments was identified neighboring <em>V1r2</em> sequences.</p>


<h4>Expression of <em>V1r1</em> and <em>V1r2</em> in the Zebrafish Olfactory Rosette</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We investigated the potential expression of <em>V1r1</em> and <em>V1r2</em> in the olfactory rosette using two approaches. We first performed RT-PCRs on olfactory rosette extracts from male and female adult zebrafish using primers specific for <em>V1r1</em> or <em>V1r2. V1r1</em> and <em>V1r2</em> transcripts were consistently observed in this tissue (<a href="#pone-0000379-g005">Figure 5a</a>). No transcripts were found in other organs, including barbels, lips, heart, gills, muscle and brain (data not shown). Second, and in order to obtain a more precise picture of the cells expressing the V1r receptor genes, we performed in situ hybridizations on olfactory rosette sections. Single neurons located in the sensory neuroepithelium did express <em>V1r1</em> transcripts, similarly to what was observed for cells expressing <em>V1r2</em> (<a href="#pone-0000379-g005">Figure 5b–d</a>). No evidence of sexual dimorphism in the expression of the V1r receptor genes was observed (data not shown).</p>
<div class="figure" id="pone-0000379-g005"><div class="img"><a name="pone-0000379-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p><em>Dr V1r1</em> and <em>V1r2</em> are expressed in the olfactory rosette. (a) RT-PCR indicating transcription of <em>V1r1</em> and <em>V1r2</em> in olfactory rosette extracts. OMP (olfactory marker protein) and βactin were used as positive controls. (b) In situ hybridization of a horizontal olfactory rosette section with an anti-sense <em>Dr V1r1</em> probe (probe III in <a href="#pone-0000379-g003">Figure 3</a>). (c) In situ hybridization with anti-sense <em>Dr V1r2</em> probes (probe I in <a href="#pone-0000379-g003">Figure 3</a>). (d) In situ hybridization with an antisense 5′UTR <em>V1r1</em> probe (probe II in <a href="#pone-0000379-g003">Figure 3</a>). Arrows indicate cells reacting to the probes. Asterisks and empty squares correspond respectively to luminal and cartilaginous parts of the rosette. Scalebar: 40 µm.</p>
<span>doi:10.1371/journal.pone.0000379.g005</span></div>

<h4>Unequal Evolutionary Rates Between <em>V1r1</em> and <em>V1r2</em></h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Two phylogenetic trees, including 24 <em>V1r1</em> and <em>V1r2</em> teleost genes or their corresponding proteins, were generated using Maximum Likelihood methods (<a href="#pone-0000379-g006">Figure 6</a>). We evaluated potential evolutionary differences between the V1r1 and V1r2 clades by performing a Relative Rate Test, and found that the V1r1 clade evolved significantly faster than the V1r2 clade (p = 0.015) (see <a href="#s4">Materials and Methods</a>).</p>
<div class="figure" id="pone-0000379-g006"><div class="img"><a name="pone-0000379-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Teleost <em>V1r1</em> and <em>V1r2</em> exhibit different evolutionary rates. (a) Phylogenetic analysis of teleost V1r genes. A rooted DNA tree was generated, based on an alignment of 24 teleost V1r sequences and three mouse V1rs (<em>V1rf3, V1re4</em> and <em>V1rb2</em>). Similar ω were obtained for each V1r family using two-or three-ratio models. The V1r1 clade ω value was 3×times higher than the one of the V1r2 clade (see <a href="#pone-0000379-t002">Table 2</a>). Positive selection was detected on branches A–F and asterisks indicate branches in which positively selected sites were identified. (b) Unrooted tree based on the amino acid alignment of the 24 teleost V1r receptors.</p>
<span>doi:10.1371/journal.pone.0000379.g006</span></div><div class="figure" id="pone-0000379-t002"><div class="img"><a name="pone-0000379-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Likelihood Ratio Tests for substitution models performed on the complete dataset and estimates of the corresponding parameters.</span></strong></p><span>doi:10.1371/journal.pone.0000379.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>To determine the origin of the acceleration detected in the V1r1 clade, we performed a series of tests of increasing sensitivity, looking at the whole dataset or at specific clades, at full-length sequences or codon by codon, and aimed at measuring the selective pressures acting on these genes. Two potential explanations for the observed differential evolutionary rates, which are not mutually exclusive, involve relaxation of the purifying selection, and positive Darwinian selection.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>The dN/dS ratio (ω) for the overall dataset was first calculated using the one-ratio model (M0) as implemented in the PAML software (see <a href="#s4">Materials and Methods</a>). An ω of 0.099 was estimated, suggesting that a strong purifying selection is globally acting on V1r receptors. Using the two-ratio model, the ω estimates for the V1r1 and V1r2 clades were 0.147 and 0.050 respectively (<a href="#pone-0000379-g006">Figure 6</a>). The Likelihood Ratio Test (LRT) indicated that the two-ratio model fit the dataset significantly better than the one-ratio model (<a href="#pone-0000379-t002">Table 2</a>), confirming that the V1r1 clade has a 3×times higher rate of non-synonymous mutations. However, no evidence for positive selection (ω&gt;1) was found.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>To further investigate potential changes in selective pressure on specific residues, we applied site-specific models to our dataset (see <a href="#s4">Materials and Methods</a>). The comparison of different site-specific models suggested heterogeneous evolutionary rates within V1r sequences. Thus, the M3 model (discrete with K = 2) fit the data significantly better than the M0 model (one ratio). The same M3 model allowing 3 different ω, fit the data better than with 2 different ω (<a href="#pone-0000379-t002">Table 2</a>). This last model proposes 29.3% of highly conserved sites (ω0 = 0.016), 60.2% of sites under strong purifying selection (ω1 = 0.103), and 10.6% of the sites under moderate purifying selection (ω2 = 0.455). Furthermore, the M8 model, which allows positive selection and which fit the data better than the M7 model (<a href="#pone-0000379-t002">Table 2</a>), indicated that 3% of the amino acid sites were under positive selection. The identities of the positively selected residues were however not statistically supported (BEB calculations, see <a href="#s4">Materials and Methods</a>).</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>We then analyzed our dataset with a site-specific approach, but looking at specific branches. We used the branch-site model D (see <a href="#s4">Materials and Methods</a>) to identify potential selective pressure differences for a category of sites between the two paralogous clades. The LRTs involving model D (with K = 2 or K = 3 categories of sites) and its nested model M3 indicated that model D with K = 3 fit our data the best (<a href="#pone-0000379-t002">Table 2</a>). This model proposed that for both genes, 45% of sites were under marked purifying selection (ω0 = 0.13), 9% under moderate purifying selection (ω1 = 0.48), and that 46% of the sites displayed a 10×relaxation in the V1r1 clade when compared to the V1r2 clade (ω2V1r2 = 0.005 vs. ω2background = 0.048). However, this large proportion of relaxed sites was still under strong purifying selection (low ω values).</p>
<a id="article1.body1.sec2.sec4.p6" name="article1.body1.sec2.sec4.p6"></a><p>Thus, models with additional categories of ω, applied either to the whole dataset or to specific groups within the tree, were systematically preferred to models of lower complexities (<a href="#pone-0000379-t002">Table 2</a>).</p>
<a id="article1.body1.sec2.sec4.p7" name="article1.body1.sec2.sec4.p7"></a><p>The potential existence of rare positively selected sites in a subset of branches was then assessed. Using again a branch-site model, model A (see <a href="#s4">Materials and Methods</a>), we analyzed the 21 longest internal and terminal branches of the V1r1/V1r2 clades. We found 5 branches exhibiting positively selected sites in the V1r1 clade and a single branch in the V1r2 clade. Among these branches, we were able to identify 9 significant positively selected sites, 5 of which were located in transmembrane regions (<a href="#pone-0000379-g002">Figure 2</a>). All significant sites pertained to the 5 branches of the V1r1 clade (<a href="#pone-0000379-t003">Table 3</a>, <a href="#pone-0000379-g006">Figure 6</a>).</p>
<div class="figure" id="pone-0000379-t003"><div class="img"><a name="pone-0000379-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000379" data-uri="info:doi/10.1371/journal.pone.0000379.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000379.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000379.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000379.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000379.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000379.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Branch-site specific Test 2 for positive selection and branches displaying sites under positive selection (including corresponding positively selected sites).</span></strong></p><span>doi:10.1371/journal.pone.0000379.t003</span></div><a id="article1.body1.sec2.sec4.p8" name="article1.body1.sec2.sec4.p8"></a><p>In accordance with our previous estimates (results of model M8), the percentage of these residues was limited, ranging from 0.4% to 7.0% (<a href="#pone-0000379-t003">Table 3</a>). 1 to 3 sites per branch displayed an ω significantly above 1 (BEB calculations) (<a href="#pone-0000379-t003">Table 3</a>).</p>
<a id="article1.body1.sec2.sec4.p9" name="article1.body1.sec2.sec4.p9"></a><p>Our approach, involving the use of models of increasing finesse, thus led to the confirmation of differential evolutionary rates between paralogous V1r clades, and to the detection of positive selection between orthologous V1r genes.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>We here describe two linked teleost V1r genes, <em>V1r1</em> and <em>V1r2</em>, and identify their orthologs in multiple fish species, taking advantage of their physical proximity and relative conservation.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Apparently, not all teleosts share the same V1r repertoire; we were indeed unable to identify a <em>V1r2</em> receptor gene in the two tested tetraodontidae species. This negative observation may naturally reflect a failure to identify an existing sequence, due to a still incomplete coverage of the tetraodon and fugu genomes. This seems however unlikely since these genomes are close to being fully sequenced. Moreover, since <em>V1r1</em> and <em>V1r2</em> genes are found in close association in all teleost species tested (less than 3.5 kb apart), and since relatively large genomic sequences surrounding the tetraodon and fugu <em>V1r1</em> were available and searched, we propose that our findings reflect a gene loss, which specifically affected the tetraodontidae lineage. This potential genus-specific gene loss is consistent with the remarkably compact genome of members of this lineage, genome that can be as small as 390 Mb in fugu and 340 Mb in tetraodon (while zebrafish and medaka genome sizes are 1.7 Gb and 800 Mb respectively). This limited genomic material translates into a contraction of gene number, shrinking known to affect many gene families, including olfactory receptor gene repertoires. Thus, a comparison between fugu and zebrafish odorant receptor genes indicates a drastic reduction in the fugu repertoire size (143 vs. 44 and 42 odorant receptor genes in zebrafish, fugu and tetraodon respectively) <a href="#pone.0000379-Alioto1">[1]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We identified a short sequence in the 5′UTR of <em>V1r1</em> transcripts, conserved between multiple teleost species, including some whose common ancestor dates back over 100 mio. years. This observation is reminiscent of the remarkable 5′UTR conservation present in mouse V1ra and V1rb transcripts <a href="#pone.0000379-Lane2">[18]</a>. It may point to a transcriptional or translational regulatory element acting on fish <em>V1r1</em> genes, and possibly also on <em>V1r2</em>. We also report a peculiar genomic organization of the fish <em>V1r1</em> and <em>V1r2</em> genes. As little as 1.1 kb separates the coding sequences of the two genes, leaving limited space for two 5′UTRs and two promoters since their transcriptional direction is inverted. The zebrafish V1r intergenic sequence, whose very limited size renders it easily amenable to genetic manipulation, will surely prove to be of interest in the study of the mechanisms regulating V1r expression, which at present remain completely unknown. We observed that unequal selective pressures affected the evolution of the V1r1 and V1r2 clades. The V1r1 clade exhibits a relatively relaxed negative pressure, with on given branches, specific residues under positive Darwinian selection, a situation observed for some mammalian V1rs. This finding suggests a possible variable role played by orthologous V1R1 receptors, or at least non-homogenous agonist response profiles between the latter (due for example to species-specific coevolution of agonist-receptor pairs). The situation is clearly different for the V1r2 clade: a strong purifying pressure is at work on V1R2 receptors, suggesting a common role and/or agonist profile between orthologous V1R2 proteins. Such theories will be put to test as soon as we identify natural agonists for fish and mammalian V1rs. But the road is apparently long before we dispose of a general view of agonist-V1r receptor pairs in vertebrates. Indeed, despite an identification of the first V1r genes over a decade ago <a href="#pone.0000379-Dulac1">[29]</a>, only a single such pair has been identified <a href="#pone.0000379-Boschat1">[8]</a>. This situation, partly resulting from the difficulties faced to express these receptors in vitro, also reflects a more problematic situation, which is the very limited number of known potential agonists.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Identification of V1r genes</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The following ENSEMBL databases were searched using the BLASTN and TBLASTN algorithms: <em>Dr</em> Zv6 (August 2006 release, 7×coverage), <em>Ga</em> BROAD S1 (June 2006 release, 11×coverage), <em>Ol</em> MEDAKA1 (May 2006 release, 6.7×coverage), <em>Tn</em> Tetraodon 7 (September 2004 release, 8.3×coverage), and <em>Tr</em> Fugu 4.0 (May 2005 release, 90% coverage). Database mined sequences from <em>Dr, Ol</em> and <em>Ga</em> were PCR amplified from genomic DNA and sequenced for confirmation. <em>Da, Dc, Dd, Dk, Dky, Dm, Dn, Dp</em> and <em>Dy V1r1 and V1r2</em> were PCR amplified from genomic DNA using the following primers: TTC CCG GTA ACA CCG CTG TCA TCT G, GAA GAA GAG CCG GAG GTC AGT GAT CAG, TAT GGA CCT GTG TGT CAC, CAG CCT TTA TGA AAC ACA TTC AC, TGG ACC TGT GTG TCA CCA TCA AAA GG, TCA GTT CTT GCC GCT GGA GTT CTT GCC, and sequenced. The <em>Dm</em> intergenic sequence was amplified from genomic DNA and sequenced. Conditions for degenerate PCRs performed on fugu and tetraodon genomic DNA were: 4 min. at 95°C followed by 48 cycles of 45 sec. at 95°C, 4 min. at 50°C and 3 min at 72°C, followed by a final extension time of 10 min at 72°C, using the following primers: TCT GSM TCA CCT KCA TKC TGA GYG YST WCC A and GGG CTG AGV GMK GCR TAS MAY GAG GAR AAA AA. Low stringency amplifications were performed with an annealing temperature of 52°C. Sequences were deposited in GenBank with the following accession numbers: DQ887609, DQ887610, DQ887610, DQ887612, DQ887613, AY279523, Q887614, DQ887615, DQ887616, 880989, 884592, 884596, 884602, 884604, 884606, 881043, 881043, 881049, 881053, 884622, and 884628.</p>


<h4>RT-PCR</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>mRNA was extracted (RNeasy extraction kit, Qiagen) from 40 male and female adult olfactory rosettes. cDNA was synthesized using a random hexamer primer. Primers used for <em>Dr V1r2</em>, β-actin and OMP genes were previously described <a href="#pone.0000379-Pfister1">[16]</a>. Primers for <em>Dr V1r1</em> were the following: TGT TTC TGT CCC TGG TGC TGG T and AGC CGG AGG TCA GTG ATC AG. PCR conditions were previously described <a href="#pone.0000379-Pfister1">[16]</a>.</p>


<h4>5′ RACE</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>5′ RACE (Rapid Amplification of cDNA Ends) was performed following the Roche Applied Science RACE kit protocol. First strand cDNA synthesis, and subsequent PCR cycles were performed using the following <em>Dr V1r1</em> primers: AGC ACG ATG GCA TCA GCA GGC GAG A, TCG CGG CGC ACC AGC AAG ATG AAG, and CGC AGA TGA CAG CGG TGT TAC C. Amplification conditions were: 3 min. at 95°C, followed by 34 cycles of 15 sec. at 95°C, 30 sec. at 55°C (or 60°C depending on the primer), and 1 min. at 72°C, with a final extension of 10 min. at 72°C.</p>


<h4>5′UTR Sequence Logo</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The 75 base pair sequences corresponding to the <em>V1r1</em> 5′UTR homology region of <em>Dr, Dm, Ol, Ga, Tr</em>, and <em>Tn</em>, initially identified using the Pipmaker analysis tool <a href="#pone.0000379-Schwartz1">[30]</a> available at <a href="http://pipmaker.bx.psu.edu/pipmaker">http://pipmaker.bx.psu.edu/pipmaker</a>, were aligned. A logo was generated using the sequence logo software <a href="#pone.0000379-Crooks1">[31]</a> available at <a href="http://weblogo.berkeley.edu/logo.cgi">http://weblogo.berkeley.edu/logo.cgi</a>.</p>


<h4>In situ hybridizations</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p><em>Dr V1r1</em> and <em>V1r2</em> probes corresponded to genomic amplicons obtained with the following primers: ATG TGG TGC CGA TGG TGC TGA TGG, AGC CGT GTG TTG GAG ACG AGG ATC AG, TAT GGA CCT GTG TGT CAC, and TCA TGG AAG TCC ACA TGG CAG AAG. The conserved 5′UTR <em>V1r1</em> box probe was flanked by the primers AGT GGA AAT GCA GTG TGC GC and CAA TTA CCT GAA TGA TGT GC. RNA digoxygenin labeled probes were synthesized according to the DIG RNA labeling kit supplier protocol (Roche Molecular Biochemicals). 10 µm cryosections were fixed for 20 minutes with 4% paraformaldehyde at 4°C. Hybridizations were performed in hybridization buffer (50% deionized formamide, 10% dextran sulphate, 1 µg/µl tRNA, 1×Denhardt's, 1×salt solution (2M NaCl, 100mM TRIS, 50mM NaH<sub>2</sub>PO<sub>4</sub>, 50 mM Na<sub>2</sub>HPO<sub>4</sub>, 50mM EDTA, pH 7.5)) overnight at 65°C. Primary anti-DIG antibody (1:1000) coupled to alkaline phosphatase (Roche Molecular Biochemicals) and FastRed (DAKO) were used for signal revelation. 1 ng/µl DAPI was used to counterstain nuclei. Signal detection was achieved either by an Axiocam camera connected to an Axioplan2 fluorescence microscope (Zeiss), or by confocal imaging using a TCS SP2 AOBS microscope (Leica).</p>


<h4>Phylogenetic trees</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Nucleotide sequences of 24 teleost V1r sequences and mouse <em>V1rb2, V1re4</em> and <em>V1rf3</em> were aligned using ClustalX <a href="#pone.0000379-Thompson1">[32]</a> and manually rearranged using the Bioedit alignment software (v. 7.0.5.3).</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>The MODELTEST v3.7 program <a href="#pone.0000379-Posada1">[33]</a> was used to determine the model of DNA sequence evolution that fit our data best. The best fit model was the General Time Reversible (GTR) with a gamma shape distribution of evolutionary rates (α = 1.18; 8 categories of sites). Phylogenetic tree reconstruction based on DNA sequences was performed using the Maximum Likelihood (ML) method as implemented in the program Phyml <a href="#pone.0000379-Guindon1">[34]</a> and using NNI and SPR branch swapping methods, or imposing different starting topologies to avoid local optima. The amino acid tree reconstruction was performed using the JTT model of amino acid changes and a gamma shape distribution of evolutionary rates (α = 2.15; 4 categories of sites). The same topology was retrieved when using two different starting trees: the Bionj Tree or the best topology found using the DNA alignment. For DNA and amino acid analyses, support for branches was assessed by bootstrap analyses of 500 replicates.</p>


<h4>Evolutionary rates and selective pressures</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>Differences in the evolutionary rates of the two paralogous V1r copies were statistically tested using the RRtree program <a href="#pone.0000379-RobinsonRechavi1">[35]</a>. Selective pressures acting on the V1r receptor genes were assessed using different models as implemented in the PAML software v.3.15 <a href="#pone.0000379-Yang1">[36]</a>. Couples of nested models were compared using Likelihood Ratio Tests (LRTs) to statistically determine the best model. Twice the difference of the likelihood values of the tree obtained under each model approximately follows a chi-square distribution and, together with the number of degrees of freedom (df) separating the two models, allows the calculation of the associated <em>p</em>-value.</p>
<a id="article1.body1.sec4.sec7.p2" name="article1.body1.sec4.sec7.p2"></a><p>To evaluate a possible unequal ratio of non-synonymous vs. synonymous substitutions (ω) in the two paralogous V1r copies, we performed an LRT comparing the two-ratio model (one ω per clade) with the one-ratio model (M0) where ω is constant. Varying selective pressure along the genes was assessed by a LRT test comparing the site-specific model M3 (discrete) with 2 or 3 classes of sites displaying different ω, against the M0 model (one-ratio). The branch-site model D <a href="#pone.0000379-Bielawski1">[37]</a>, an extension of the model M3 which allows selective pressure at one class of sites to be different in two parts of the phylogeny, was used to test for divergent selective pressure between the two paralogous V1r genes, by applying an LRT on models D vs. M3.</p>


<h4>Positive selection</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>Positive selection was first assessed using two LRTs involving site-specific models of increasing complexity. Model M2a (which allows one group of sites to have an ω&gt;1), was compared to model M1a (where ω cannot exceed 1). Discrete model M8 (in which the ω value can vary according to a beta distribution (between 0 and 1) and which includes an extra category of sites with ω&gt;1), was compared to model M7 (in which ω can only vary according to the beta distribution). Branch-site models were used to identify positive selection acting on sites along specific branches of the phylogenetic tree. Model A <a href="#pone.0000379-Zhang2">[38]</a> was used to identify positive selection acting on sites along specific branches of the tree. This model, which allows one category of sites to evolve under positive selection (ω2&gt;1), was compared by LRT to that same model with an ω2 fixed at 1 (Test 2). This LRT test is the most reliable test according to Zhang et al. <a href="#pone.0000379-Zhang2">[38]</a>. Significance of positively selected sites was evaluated using Bayes Empirical Bayes (BEB) calculations <a href="#pone.0000379-Yang2">[39]</a>.</p>


<h4>Animals</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>Animals were housed and handled accordingly to the guidelines and regulations of the institution and of the state of Geneva.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank J. Wittbrodt for medaka genomic DNA, and C. Bauer for help on the confocal imaging platform.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: IR JM PP. Performed the experiments: JM PP JR. Analyzed the data: IR JM PP JR. Wrote the paper: IR JM PP.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000379-Alioto1" id="pone.0000379-Alioto1"></a>Alioto TS, Ngai J (2005) The odorant receptor repertoire of teleost fish. BMC Genomics  6: 173.  <ul class="find" data-citedArticleID="945224" data-doi="10.1186/1471-2164-6-173"><li><a href="http://dx.doi.org/10.1186/1471-2164-6-173" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+odorant+receptor+repertoire+of+teleost+fish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+odorant+receptor+repertoire+of+teleost+fish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000379-Liberles1" id="pone.0000379-Liberles1"></a>Liberles SD, Buck LB (2006) A second class of chemosensory receptors in the olfactory epithelium. Nature  442: 645–650.  <ul class="find" data-citedArticleID="945264" data-doi="10.1038/nature05066"><li><a href="http://dx.doi.org/10.1038/nature05066" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+second+class+of+chemosensory+receptors+in+the+olfactory+epithelium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+second+class+of+chemosensory+receptors+in+the+olfactory+epithelium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000379-Gloriam1" id="pone.0000379-Gloriam1"></a>Gloriam DE, Bjarnadottir TK, Yan YL, Postlethwait JH, Schioth HB, et al.  (2005) The repertoire of trace amine G-protein-coupled receptors: large expansion in zebrafish. Mol Phylogenet Evol  35: 470–482.  <ul class="find" data-citedArticleID="945250" data-doi="10.1016/j.ympev.2004.12.003"><li><a href="http://dx.doi.org/10.1016/j.ympev.2004.12.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+repertoire+of+trace+amine+G-protein-coupled+receptors%3A+large+expansion+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+repertoire+of+trace+amine+G-protein-coupled+receptors%3A+large+expansion+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000379-Hashiguchi1" id="pone.0000379-Hashiguchi1"></a>Hashiguchi Y, Nishida M (2005) Evolution of vomeronasal-type odorant receptor genes in the zebrafish genome. Gene  362: 19–28.  <ul class="find" data-citedArticleID="945258" data-doi="10.1016/j.gene.2005.07.044"><li><a href="http://dx.doi.org/10.1016/j.gene.2005.07.044" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolution+of+vomeronasal-type+odorant+receptor+genes+in+the+zebrafish+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolution+of+vomeronasal-type+odorant+receptor+genes+in+the+zebrafish+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000379-Cao1" id="pone.0000379-Cao1"></a>Cao Y, Oh BC, Stryer L (1998) Cloning and localization of two multigene receptor families in goldfish olfactory epithelium. Proc Natl Acad Sci USA  95: 11987–11992.  <ul class="find" data-citedArticleID="945234" data-doi="10.1073/pnas.95.20.11987"><li><a href="http://dx.doi.org/10.1073/pnas.95.20.11987" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+and+localization+of+two+multigene+receptor+families+in+goldfish+olfactory+epithelium." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+and+localization+of+two+multigene+receptor+families+in+goldfish+olfactory+epithelium.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000379-Alioto2" id="pone.0000379-Alioto2"></a>Alioto TS, Ngai J (2006) The repertoire of olfactory C family G protein-coupled receptors in zebrafish: candidate chemosensory receptors for amino acids. BMC Genomics  7: 309.  <ul class="find" data-citedArticleID="945226"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+repertoire+of+olfactory+C+family+G+protein-coupled+receptors+in+zebrafish%3A+candidate+chemosensory+receptors+for+amino+acids.&amp;auth=&amp;atitle=The+repertoire+of+olfactory+C+family+G+protein-coupled+receptors+in+zebrafish%3A+candidate+chemosensory+receptors+for+amino+acids." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+repertoire+of+olfactory+C+family+G+protein-coupled+receptors+in+zebrafish%3A+candidate+chemosensory+receptors+for+amino+acids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+repertoire+of+olfactory+C+family+G+protein-coupled+receptors+in+zebrafish%3A+candidate+chemosensory+receptors+for+amino+acids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000379-Bjarnadottir1" id="pone.0000379-Bjarnadottir1"></a>Bjarnadottir TK, Fredriksson R, Schioth HB (2005) The gene repertoire and the common evolutionary history of glutamate, pheromone (V2R), taste(1) and other related G protein-coupled receptors. Gene  362: 70–84.  <ul class="find" data-citedArticleID="945230" data-doi="10.1016/j.gene.2005.07.029"><li><a href="http://dx.doi.org/10.1016/j.gene.2005.07.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+gene+repertoire+and+the+common+evolutionary+history+of+glutamate%2C+pheromone+%28V2R%29%2C+taste%281%29+and+other+related+G+protein-coupled+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+gene+repertoire+and+the+common+evolutionary+history+of+glutamate%2C+pheromone+%28V2R%29%2C+taste%281%29+and+other+related+G+protein-coupled+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000379-Boschat1" id="pone.0000379-Boschat1"></a>Boschat C, Pelofi C, Randin O, Roppolo D, Luscher C, et al.  (2002) Pheromone detection mediated by a V1r vomeronasal receptor. Nat Neurosci  5: 1261–1262.  <ul class="find" data-citedArticleID="945232" data-doi="10.1038/nn978"><li><a href="http://dx.doi.org/10.1038/nn978" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pheromone+detection+mediated+by+a+V1r+vomeronasal+receptor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pheromone+detection+mediated+by+a+V1r+vomeronasal+receptor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000379-DelPunta1" id="pone.0000379-DelPunta1"></a>Del Punta K, Leinders-Zufall T, Rodriguez I, Jukam D, Wysocki CJ, et al.  (2002) Deficient pheromone responses in mice lacking a cluster of vomeronasal receptor genes. Nature  419: 70–74.  <ul class="find" data-citedArticleID="945244" data-doi="10.1038/nature00955"><li><a href="http://dx.doi.org/10.1038/nature00955" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Deficient+pheromone+responses+in+mice+lacking+a+cluster+of+vomeronasal+receptor+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Deficient+pheromone+responses+in+mice+lacking+a+cluster+of+vomeronasal+receptor+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000379-Young1" id="pone.0000379-Young1"></a>Young JM, Kambere M, Trask BJ, Lane RP (2005) Divergent V1R repertoires in five species: Amplification in rodents, decimation in primates, and a surprisingly small repertoire in dogs. Genome Res  15: 231–240.  <ul class="find" data-citedArticleID="945296" data-doi="10.1101/gr.3339905"><li><a href="http://dx.doi.org/10.1101/gr.3339905" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Divergent+V1R+repertoires+in+five+species%3A+Amplification+in+rodents%2C+decimation+in+primates%2C+and+a+surprisingly+small+repertoire+in+dogs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Divergent+V1R+repertoires+in+five+species%3A+Amplification+in+rodents%2C+decimation+in+primates%2C+and+a+surprisingly+small+repertoire+in+dogs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000379-Lane1" id="pone.0000379-Lane1"></a>Lane RP, Young J, Newman T, Trask BJ (2004) Species specificity in rodent pheromone receptor repertoires. Genome Res  14: 603–608.  <ul class="find" data-citedArticleID="945260" data-doi="10.1101/gr.2117004"><li><a href="http://dx.doi.org/10.1101/gr.2117004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Species+specificity+in+rodent+pheromone+receptor+repertoires." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Species+specificity+in+rodent+pheromone+receptor+repertoires.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000379-Grus1" id="pone.0000379-Grus1"></a>Grus W, Shi P, Zhang Y, Zhang J (2005) Dramatic variation of the vomeronasal pheromone gene repertoire among five orders of placental and marsupial mammals. Proc Natl Acad Sci U S A  102: 5767–5772.  <ul class="find" data-citedArticleID="945252" data-doi="10.1073/pnas.0501589102"><li><a href="http://dx.doi.org/10.1073/pnas.0501589102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dramatic+variation+of+the+vomeronasal+pheromone+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dramatic+variation+of+the+vomeronasal+pheromone+gene+repertoire+among+five+orders+of+placental+and+marsupial+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000379-Grus2" id="pone.0000379-Grus2"></a>Grus WE, Zhang J (2004) Rapid turnover and species-specificity of vomeronasal pheromone receptor genes in mice and rats. Gene  340: 303–312.  <ul class="find" data-citedArticleID="945254" data-doi="10.1016/j.gene.2004.07.037"><li><a href="http://dx.doi.org/10.1016/j.gene.2004.07.037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rapid+turnover+and+species-specificity+of+vomeronasal+pheromone+receptor+genes+in+mice+and+rats." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rapid+turnover+and+species-specificity+of+vomeronasal+pheromone+receptor+genes+in+mice+and+rats.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000379-Rodriguez1" id="pone.0000379-Rodriguez1"></a>Rodriguez I, Del Punta K, Rothman A, Ishii T, Mombaerts P (2002) Multiple new and isolated families within the mouse superfamily of V1r vomeronasal receptors. Nat Neurosci  5: 134–140.  <ul class="find" data-citedArticleID="945276" data-doi="10.1038/nn795"><li><a href="http://dx.doi.org/10.1038/nn795" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+new+and+isolated+families+within+the+mouse+superfamily+of+V1r+vomeronasal+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+new+and+isolated+families+within+the+mouse+superfamily+of+V1r+vomeronasal+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000379-Zhang1" id="pone.0000379-Zhang1"></a>Zhang X, Rodriguez I, Mombaerts P, Firestein S (2004) Odorant and vomeronasal receptor genes in two mouse genome assemblies. Genomics  83: 802–811.  <ul class="find" data-citedArticleID="945298" data-doi="10.1016/j.ygeno.2003.10.009"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2003.10.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Odorant+and+vomeronasal+receptor+genes+in+two+mouse+genome+assemblies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Odorant+and+vomeronasal+receptor+genes+in+two+mouse+genome+assemblies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000379-Pfister1" id="pone.0000379-Pfister1"></a>Pfister P, Rodriguez I (2005) Olfactory expression of a single and highly variable V1r pheromone receptor-like gene in fish species. Proc Natl Acad Sci U S A  102: 5489–5494.  <ul class="find" data-citedArticleID="945270" data-doi="10.1073/pnas.0402581102"><li><a href="http://dx.doi.org/10.1073/pnas.0402581102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Olfactory+expression+of+a+single+and+highly+variable+V1r+pheromone+receptor-like+gene+in+fish+species." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Olfactory+expression+of+a+single+and+highly+variable+V1r+pheromone+receptor-like+gene+in+fish+species.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000379-Mundy1" id="pone.0000379-Mundy1"></a>Mundy NI, Cook S (2003) Positive Selection During the Diversification of Class I Vomeronasal Receptor-like (V1RL) Genes, Putative Pheromone Receptor Genes, in Human and Primate Evolution. Mol Biol Evol  20: 1805–1810.  <ul class="find" data-citedArticleID="945268" data-doi="10.1093/molbev/msg192"><li><a href="http://dx.doi.org/10.1093/molbev/msg192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+Selection+During+the+Diversification+of+Class+I+Vomeronasal+Receptor-like+%28V1RL%29+Genes%2C+Putative+Pheromone+Receptor+Genes%2C+in+Human+and+Primate+Evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+Selection+During+the+Diversification+of+Class+I+Vomeronasal+Receptor-like+%28V1RL%29+Genes%2C+Putative+Pheromone+Receptor+Genes%2C+in+Human+and+Primate+Evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000379-Lane2" id="pone.0000379-Lane2"></a>Lane RP, Cutforth T, Axel R, Hood L, Trask BJ (2002) Sequence analysis of mouse vomeronasal receptor gene clusters reveals common promoter motifs and a history of recent expansion. Proc Natl Acad Sci U S A  99: 291–296.  <ul class="find" data-citedArticleID="945262" data-doi="10.1073/pnas.012608399"><li><a href="http://dx.doi.org/10.1073/pnas.012608399" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+analysis+of+mouse+vomeronasal+receptor+gene+clusters+reveals+common+promoter+motifs+and+a+history+of+recent+expansion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+analysis+of+mouse+vomeronasal+receptor+gene+clusters+reveals+common+promoter+motifs+and+a+history+of+recent+expansion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000379-Shi1" id="pone.0000379-Shi1"></a>Shi P, Bielawski JP, Yang H, Zhang YP (2005) Adaptive diversification of vomeronasal receptor 1 genes in rodents. J Mol Evol  60: 566–576.  <ul class="find" data-citedArticleID="945280" data-doi="10.1007/s00239-004-0172-y"><li><a href="http://dx.doi.org/10.1007/s00239-004-0172-y" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Adaptive+diversification+of+vomeronasal+receptor+1+genes+in+rodents." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Adaptive+diversification+of+vomeronasal+receptor+1+genes+in+rodents.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000379-Meyer1" id="pone.0000379-Meyer1"></a>Meyer A, Schartl M (1999) Gene and genome duplications in vertebrates: the one-to-four (-to-eight in fish) rule and the evolution of novel gene functions. Curr Opin Cell Biol  11: 699–704.  <ul class="find" data-citedArticleID="945266" data-doi="10.1016/s0955-0674(99)00039-3"><li><a href="http://dx.doi.org/10.1016/s0955-0674(99)00039-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+and+genome+duplications+in+vertebrates%3A+the+one-to-four+%28-to-eight+in+fish%29+rule+and+the+evolution+of+novel+gene+functions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+and+genome+duplications+in+vertebrates%3A+the+one-to-four+%28-to-eight+in+fish%29+rule+and+the+evolution+of+novel+gene+functions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000379-Sidow1" id="pone.0000379-Sidow1"></a>Sidow A (1996) Gen(om)e duplications in the evolution of early vertebrates. Curr Opin Genet Dev  6: 715–722.  <ul class="find" data-citedArticleID="945282" data-doi="10.1016/s0959-437x(96)80026-8"><li><a href="http://dx.doi.org/10.1016/s0959-437x(96)80026-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gen%28om%29e+duplications+in+the+evolution+of+early+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gen%28om%29e+duplications+in+the+evolution+of+early+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000379-Suga1" id="pone.0000379-Suga1"></a>Suga H, Koyanagi M, Hoshiyama D, Ono K, Iwabe N, et al.  (1999) Extensive gene duplication in the early evolution of animals before the parazoan-eumetazoan split demonstrated by G proteins and protein tyrosine kinases from sponge and hydra. J Mol Evol  48: 646–653.  <ul class="find" data-citedArticleID="945284" data-doi="10.1007/pl00006508"><li><a href="http://dx.doi.org/10.1007/pl00006508" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extensive+gene+duplication+in+the+early+evolution+of+animals+before+the+parazoan-eumetazoan+split+demonstrated+by+G+proteins+and+protein+tyrosine+kinases+from+sponge+and+hydra." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extensive+gene+duplication+in+the+early+evolution+of+animals+before+the+parazoan-eumetazoan+split+demonstrated+by+G+proteins+and+protein+tyrosine+kinases+from+sponge+and+hydra.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000379-Vandepoele1" id="pone.0000379-Vandepoele1"></a>Vandepoele K, De Vos W, Taylor JS, Meyer A, Van de Peer Y (2004) Major events in the genome evolution of vertebrates: paranome age and size differ considerably between ray-finned fishes and land vertebrates. Proc Natl Acad Sci U S A  101: 1638–1643.  <ul class="find" data-citedArticleID="945288" data-doi="10.1073/pnas.0307968100"><li><a href="http://dx.doi.org/10.1073/pnas.0307968100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Major+events+in+the+genome+evolution+of+vertebrates%3A+paranome+age+and+size+differ+considerably+between+ray-finned+fishes+and+land+vertebrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Major+events+in+the+genome+evolution+of+vertebrates%3A+paranome+age+and+size+differ+considerably+between+ray-finned+fishes+and+land+vertebrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000379-Christoffels1" id="pone.0000379-Christoffels1"></a>Christoffels A, Koh EG, Chia JM, Brenner S, Aparicio S, et al.  (2004) Fugu genome analysis provides evidence for a whole-genome duplication early during the evolution of ray-finned fishes. Mol Biol Evol  21: 1146–1151.  <ul class="find" data-citedArticleID="945238" data-doi="10.1093/molbev/msh114"><li><a href="http://dx.doi.org/10.1093/molbev/msh114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fugu+genome+analysis+provides+evidence+for+a+whole-genome+duplication+early+during+the+evolution+of+ray-finned+fishes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fugu+genome+analysis+provides+evidence+for+a+whole-genome+duplication+early+during+the+evolution+of+ray-finned+fishes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000379-Wittbrodt1" id="pone.0000379-Wittbrodt1"></a>Wittbrodt J, Shima A, Schartl M (2002) Medaka–a model organism from the far East. Nat Rev Genet  3: 53–64.  <ul class="find" data-citedArticleID="945290" data-doi="10.1038/nrg704"><li><a href="http://dx.doi.org/10.1038/nrg704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Medaka%26%23x2013%3Ba+model+organism+from+the+far+East." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Medaka%26%23x2013%3Ba+model+organism+from+the+far+East.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000379-CrnogoracJurcevic1" id="pone.0000379-CrnogoracJurcevic1"></a>Crnogorac-Jurcevic T, Brown JR, Lehrach H, Schalkwyk LC (1997) Tetraodon fluviatilis, a new puffer fish model for genome studies. Genomics  41: 177–184.  <ul class="find" data-citedArticleID="945240" data-doi="10.1006/geno.1997.4646"><li><a href="http://dx.doi.org/10.1006/geno.1997.4646" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tetraodon+fluviatilis%2C+a+new+puffer+fish+model+for+genome+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tetraodon+fluviatilis%2C+a+new+puffer+fish+model+for+genome+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000379-Chen1" id="pone.0000379-Chen1"></a>Chen WJ, Orti G, Meyer A (2004) Novel evolutionary relationship among four fish model systems. Trends Genet  20: 424–431.  <ul class="find" data-citedArticleID="945236" data-doi="10.1016/j.tig.2004.07.005"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.07.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+evolutionary+relationship+among+four+fish+model+systems." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+evolutionary+relationship+among+four+fish+model+systems.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000379-FurutaniSeiki1" id="pone.0000379-FurutaniSeiki1"></a>Furutani-Seiki M, Wittbrodt J (2004) Medaka and zebrafish, an evolutionary twin study. Mech Dev  121: 629–637.  <ul class="find" data-citedArticleID="945248" data-doi="10.1016/j.mod.2004.05.010"><li><a href="http://dx.doi.org/10.1016/j.mod.2004.05.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Medaka+and+zebrafish%2C+an+evolutionary+twin+study." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Medaka+and+zebrafish%2C+an+evolutionary+twin+study.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000379-Dulac1" id="pone.0000379-Dulac1"></a>Dulac C, Axel R (1995) A novel family of genes encoding putative pheromone receptors in mammals. Cell  83: 195–206.  <ul class="find" data-citedArticleID="945246" data-doi="10.1016/0092-8674(95)90161-2"><li><a href="http://dx.doi.org/10.1016/0092-8674(95)90161-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+family+of+genes+encoding+putative+pheromone+receptors+in+mammals." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+family+of+genes+encoding+putative+pheromone+receptors+in+mammals.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000379-Schwartz1" id="pone.0000379-Schwartz1"></a>Schwartz S, Zha ng Z, Frazer KA, Smit A, Riemer C, et al.  (2000) PipMaker--a web server for aligning two genomic DNA sequences. Genome Res  10: 577–586.  <ul class="find" data-citedArticleID="945278" data-doi="10.1101/gr.10.4.577"><li><a href="http://dx.doi.org/10.1101/gr.10.4.577" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PipMaker--a+web+server+for+aligning+two+genomic+DNA+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PipMaker--a+web+server+for+aligning+two+genomic+DNA+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000379-Crooks1" id="pone.0000379-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: a sequence logo generator. Genome Res  14: 1188–1190.  <ul class="find" data-citedArticleID="945242" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+a+sequence+logo+generator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+a+sequence+logo+generator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000379-Thompson1" id="pone.0000379-Thompson1"></a>Thompson JD, Higgins DG, Gibson TJ (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res  22: 4673–4680.  <ul class="find" data-citedArticleID="945286" data-doi="10.1093/nar/22.22.4673"><li><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000379-Posada1" id="pone.0000379-Posada1"></a>Posada D, Crandall KA (1998) MODELTEST: testing the model of DNA substitution. Bioinformatics  14: 817–818.  <ul class="find" data-citedArticleID="945272" data-doi="10.1093/bioinformatics/14.9.817"><li><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MODELTEST%3A+testing+the+model+of+DNA+substitution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MODELTEST%3A+testing+the+model+of+DNA+substitution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000379-Guindon1" id="pone.0000379-Guindon1"></a>Guindon S, Gascuel O (2003) A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol  52: 696–704.  <ul class="find" data-citedArticleID="945256"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.&amp;auth=&amp;atitle=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+simple%2C+fast%2C+and+accurate+algorithm+to+estimate+large+phylogenies+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000379-RobinsonRechavi1" id="pone.0000379-RobinsonRechavi1"></a>Robinson-Rechavi M, Huchon D (2000) RRTree: relative-rate tests between groups of sequences on a phylogenetic tree. Bioinformatics  16: 296–297.  <ul class="find" data-citedArticleID="945274" data-doi="10.1093/bioinformatics/16.3.296"><li><a href="http://dx.doi.org/10.1093/bioinformatics/16.3.296" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RRTree%3A+relative-rate+tests+between+groups+of+sequences+on+a+phylogenetic+tree." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RRTree%3A+relative-rate+tests+between+groups+of+sequences+on+a+phylogenetic+tree.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000379-Yang1" id="pone.0000379-Yang1"></a>Yang Z (1997) PAML: a program package for phylogenetic analysis by maximum likelihood. Comput Appl Biosci  13: 555–556.  <ul class="find" data-citedArticleID="945292" data-doi="10.1093/bioinformatics/13.5.555"><li><a href="http://dx.doi.org/10.1093/bioinformatics/13.5.555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22PAML%3A+a+program+package+for+phylogenetic+analysis+by+maximum+likelihood.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000379-Bielawski1" id="pone.0000379-Bielawski1"></a>Bielawski JP, Yang Z (2004) A maximum likelihood method for detecting functional divergence at individual codon sites, with application to gene family evolution. J Mol Evol  59: 121–132.  <ul class="find" data-citedArticleID="945228" data-doi="10.1007/s00239-004-2597-8"><li><a href="http://dx.doi.org/10.1007/s00239-004-2597-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+maximum+likelihood+method+for+detecting+functional+divergence+at+individual+codon+sites%2C+with+application+to+gene+family+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+maximum+likelihood+method+for+detecting+functional+divergence+at+individual+codon+sites%2C+with+application+to+gene+family+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000379-Zhang2" id="pone.0000379-Zhang2"></a>Zhang J, Nielsen R, Yang Z (2005) Evaluation of an improved branch-site likelihood method for detecting positive selection at the molecular level. Mol Biol Evol  22: 2472–2479.  <ul class="find" data-citedArticleID="945300" data-doi="10.1093/molbev/msi237"><li><a href="http://dx.doi.org/10.1093/molbev/msi237" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evaluation+of+an+improved+branch-site+likelihood+method+for+detecting+positive+selection+at+the+molecular+level.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000379-Yang2" id="pone.0000379-Yang2"></a>Yang Z, Wong WS, Nielsen R (2005) Bayes empirical bayes inference of amino acid sites under positive selection. Mol Biol Evol  22: 1107–1118.  <ul class="find" data-citedArticleID="945294" data-doi="10.1093/molbev/msi097"><li><a href="http://dx.doi.org/10.1093/molbev/msi097" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayes+empirical+bayes+inference+of+amino+acid+sites+under+positive+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.XML" value="90758"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.PDF" value="890051"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g001.PNG_L" value="328272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g001.PNG_M" value="56287"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g001.PNG_S" value="2405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g001.TIF" value="525556"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g001.PNG_I" value="23169"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t001.PNG_L" value="95723"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t001.PNG_M" value="32229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t001.PNG_S" value="2371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t001.TIF" value="556548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t001.PNG_I" value="13444"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g002.PNG_L" value="2281559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g002.PNG_M" value="403173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g002.PNG_S" value="8961"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g002.TIF" value="3034564"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g003.PNG_L" value="79559"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g003.PNG_M" value="36012"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g003.PNG_S" value="2308"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g003.TIF" value="130988"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g003.PNG_I" value="15192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g004.PNG_L" value="487251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g004.PNG_M" value="132042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g004.PNG_S" value="4042"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g004.TIF" value="765276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g004.PNG_I" value="48668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g005.PNG_L" value="1387825"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g005.PNG_M" value="441717"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g005.PNG_S" value="11332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g005.TIF" value="1839494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g005.PNG_I" value="154049"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g006.PNG_L" value="683804"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g006.PNG_M" value="81132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g006.PNG_S" value="2779"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g006.TIF" value="875096"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.g006.PNG_I" value="28209"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t002.PNG_L" value="199891"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t002.PNG_M" value="69229"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t002.PNG_S" value="3286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t002.TIF" value="1008446"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t002.PNG_I" value="29262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t003.PNG_L" value="80863"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t003.PNG_M" value="28226"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t003.PNG_S" value="2270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t003.TIF" value="464486"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000379.t003.PNG_I" value="12370"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000379&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000379" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000379&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000379&volume=&issue=&title=Divergent Evolution among Teleost V1r Receptor Genes&author_name=Patrick%20Pfister%2C%20Jerome%20Randall%2C%20Juan%20I.%20Montoya-Burgos%2C%20Ivan%20Rodriguez&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000379" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379&amp;t=Divergent%20Evolution%20among%20Teleost%20V1r%20Receptor%20Genes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379&title=Divergent%20Evolution%20among%20Teleost%20V1r%20Receptor%20Genes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379&amp;title=Divergent%20Evolution%20among%20Teleost%20V1r%20Receptor%20Genes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000379&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Divergent Evolution among Teleost V1r Receptor Genes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000379';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Divergent%20Evolution%20among%20Teleost%20V1r%20Receptor%20Genes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000379" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000379" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Conservation+science%22" title="Search for articles in the subject area:'Conservation science'"><div class="flagText">Conservation science</div></a>
              <div data-categoryid="40107" data-articleid="23596"
                   data-categoryname="Conservation science"
                   class="flagImage" title="Flag 'Conservation science' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="23596"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23596"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23596"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="23596"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Olfactory+receptor+neurons%22" title="Search for articles in the subject area:'Olfactory receptor neurons'"><div class="flagText">Olfactory receptor neurons</div></a>
              <div data-categoryid="21543" data-articleid="23596"
                   data-categoryname="Olfactory receptor neurons"
                   class="flagImage" title="Flag 'Olfactory receptor neurons' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phylogenetic+analysis%22" title="Search for articles in the subject area:'Phylogenetic analysis'"><div class="flagText">Phylogenetic analysis</div></a>
              <div data-categoryid="17851" data-articleid="23596"
                   data-categoryname="Phylogenetic analysis"
                   class="flagImage" title="Flag 'Phylogenetic analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Zebrafish%22" title="Search for articles in the subject area:'Zebrafish'"><div class="flagText">Zebrafish</div></a>
              <div data-categoryid="32787" data-articleid="23596"
                   data-categoryname="Zebrafish"
                   class="flagImage" title="Flag 'Zebrafish' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=9318'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4998'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3114&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=7921">reference</a><br>Posted by Rodriguez</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
