

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Parallel Selection on TRPV6 in Human Populations</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001686"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001686" />

    <meta name="citation_title" content="Parallel Selection on TRPV6 in Human Populations"/>
    <meta itemprop="name" content="Parallel Selection on TRPV6 in Human Populations"/>

      <meta name="citation_author" content="David A. Hughes"/>
            <meta name="citation_author_institution" content="Max Plank Institute for Evolutionary Anthropology, Leipzig, Germany"/>
      <meta name="citation_author" content="Kun Tang"/>
            <meta name="citation_author_institution" content="Max Plank Institute for Evolutionary Anthropology, Leipzig, Germany"/>
      <meta name="citation_author" content="Rainer Strotmann"/>
            <meta name="citation_author_institution" content="Institute of Biochemistry, Molecular Biochemistry, Medical Faculty, University of Leipzig, Germany"/>
      <meta name="citation_author" content="Torsten Schöneberg"/>
            <meta name="citation_author_institution" content="Institute of Biochemistry, Molecular Biochemistry, Medical Faculty, University of Leipzig, Germany"/>
      <meta name="citation_author" content="Jean Prenen"/>
            <meta name="citation_author_institution" content="Department of Physiology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
      <meta name="citation_author" content="Bernd Nilius"/>
            <meta name="citation_author_institution" content="Department of Physiology, Katholieke Universiteit Leuven, Leuven, Belgium"/>
      <meta name="citation_author" content="Mark Stoneking"/>
            <meta name="citation_author_institution" content="Max Plank Institute for Evolutionary Anthropology, Leipzig, Germany"/>

    <meta name="citation_date" content="2008/2/27"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001686.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1686"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Population History and Natural Selection Shape Patterns of Genetic Variation in 132 Genes.; citation_author=JM Akey; citation_author=MA Eberle; citation_author=MJ Rieder; citation_author=CS Carlson; citation_author=MD Shriver; citation_journal_title=PLoS Biol; citation_volume=2; citation_number=1; citation_pages=E286; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Interrogating a high-density SNP map for signatures of natural selection.; citation_author=JM Akey; citation_author=G Zhang; citation_author=K Zhang; citation_author=L Jin; citation_author=MD Shriver; citation_journal_title=Genome Res; citation_volume=12; citation_number=2; citation_pages=1805-1814; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Natural selection on protein-coding genes in the human genome.; citation_author=CD Bustamante; citation_author=A Fledel-Alon; citation_author=S Williamson; citation_author=R Nielsen; citation_author=M Todd Hubisz; citation_journal_title=Nature; citation_volume=437; citation_number=3; citation_pages=1153; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomic regions exhibiting positive selection identified from dense genotype data.; citation_author=CS Carlson; citation_author=DJ Thomas; citation_author=MA Eberle; citation_author=JE Swanson; citation_author=RJ Livingston; citation_journal_title=Genome Res; citation_volume=15; citation_number=4; citation_pages=1553-1565; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inferring Nonneutral Evolution from Human-Chimp-Mouse Orthologous Gene Trios.; citation_author=AG Clark; citation_author=S Glanowski; citation_author=R Nielsen; citation_author=PD Thomas; citation_author=A Kejariwal; citation_journal_title=Science; citation_volume=302; citation_number=5; citation_pages=1960-1963; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A genome scan to detect candidate regions influenced by local natural selection in human populations.; citation_author=M Kayser; citation_author=S Brauer; citation_author=M Stoneking; citation_journal_title=Mol Biol Evol; citation_volume=20; citation_number=6; citation_pages=893-900; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=A scan for positively selected genes in the genomes of humans and chimpanzees.; citation_author=R Nielsen; citation_author=C Bustamante; citation_author=AG Clark; citation_author=S Glanowski; citation_author=TB Sackton; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=7; citation_pages=e170; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Disentangling the effects of demography and selection in human history.; citation_author=JE Stajich; citation_author=MW Hahn; citation_journal_title=Mol Biol Evol; citation_volume=22; citation_number=8; citation_pages=63-73; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome scans of DNA variability in humans reveal evidence for selective sweeps outside of Africa.; citation_author=JF Storz; citation_author=BA Payseur; citation_author=MW Nachman; citation_journal_title=Mol Biol Evol; citation_volume=21; citation_number=9; citation_pages=1800-1811; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A map of recent positive selection in the human genome.; citation_author=BF Voight; citation_author=S Kudaravalli; citation_author=X Wen; citation_author=JK Pritchard; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=10; citation_pages=e72; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Searching for signals of evolutionary selection in 168 genes related to immune function.; citation_author=EC Walsh; citation_author=P Sabeti; citation_author=HB Hutcheson; citation_author=B Fry; citation_author=SF Schaffner; citation_journal_title=Hum Genet; citation_volume=119; citation_number=11; citation_pages=92-102; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Function and expression of the epithelial Ca2+ channel family: comparison of the epithelial Ca2+ channel 1 and 2.; citation_author=JGJ Hoenderop; citation_author=R Vennekens; citation_author=D Müller; citation_author=J Prenen; citation_author=G Droogmans; citation_journal_title=J Physiol (Lond); citation_volume=537; citation_number=12; citation_pages=747-761; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Genetic signatures of strong recent positive selection at the lactase gene.; citation_author=T Bersaglieri; citation_author=PC Sabeti; citation_author=N Patterson; citation_author=T Vanderploeg; citation_author=SF Schaffner; citation_journal_title=Am J Hum Genet; citation_volume=74; citation_number=13; citation_pages=1111-1120; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The effect of a selected locus on linked neutral loci.; citation_author=G Thomson; citation_journal_title=Genetics; citation_volume=85; citation_number=14; citation_pages=753-788; citation_date=1977; " />
      <meta name="citation_reference" content="citation_title=TRPV6 exhibits unusual patterns of polymorphism and divergence in worldwide populations.; citation_author=JM Akey; citation_author=WJ Swanson; citation_author=J Madeoy; citation_author=M Eberle; citation_author=MD Shriver; citation_journal_title=Hum Mol Genet; citation_volume=15; citation_number=15; citation_pages=2106-2113; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages.; citation_author=Z Yang; citation_author=R Nielsen; citation_journal_title=Mol Biol Evol; citation_volume=19; citation_number=16; citation_pages=908-917; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A human genome diversity cell line panel.; citation_author=HM Cann; citation_author=C de Toma; citation_author=L Cazes; citation_author=MF Legrand; citation_author=V Morel; citation_journal_title=Science; citation_volume=296; citation_number=17; citation_pages=261-262; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Detecting recent positive selection in the human genome from haplotype structure.; citation_author=PC Sabeti; citation_author=DE Reich; citation_author=JM Higgins; citation_author=HZ Levine; citation_author=DJ Richter; citation_journal_title=Nature; citation_volume=419; citation_number=18; citation_pages=832-837; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A New Approach for Using Genome Scans to Detect Recent Positive Selection in the Human Genome.; citation_author=K Tang; citation_author=KR Thornton; citation_author=M Stoneking; citation_journal_title=PLoS Biol; citation_volume=5; citation_number=19; citation_pages=e171; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Haplotype homozygosity and derived alleles in the human genome.; citation_author=AE Fry; citation_author=CJ Trafford; citation_author=MA Kimber; citation_author=M Chan; citation_author=KA Rockett; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=20; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Positive natural selection in the human lineage.; citation_author=PC Sabeti; citation_author=SF Schaffner; citation_author=B Fry; citation_author=J Lohmueller; citation_author=P Varilly; citation_journal_title=Science; citation_volume=312; citation_number=21; citation_pages=1614-1620; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A Fine-Scale Linkage-Disequilibrium Measure Based on Length of Haplotype Sharing.; citation_author=Y Wang; citation_author=LP Zhao; citation_author=S Dudoit; citation_journal_title=Am J Hum Genet; citation_volume=78; citation_number=22; citation_pages=615-628; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=A haplotype map of the human genome.; citation_journal_title=Nature; citation_volume=437; citation_number=23; citation_pages=1299; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Linkage disequilibrium in the human genome.; citation_author=DE Reich; citation_author=M Cargill; citation_author=S Bolk; citation_author=J Ireland; citation_author=PC Sabeti; citation_journal_title=Nature; citation_volume=411; citation_number=24; citation_pages=199; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The International HapMap Project.; citation_journal_title=Nature; citation_volume=426; citation_number=25; citation_pages=789; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genetic traces of ancient demography.; citation_author=HC Harpending; citation_author=MA Batzer; citation_author=M Gurven; citation_author=LB Jorde; citation_author=AR Rogers; citation_journal_title=PNAS; citation_volume=95; citation_number=26; citation_pages=1961-1967; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Using mitochondrial and nuclear DNA markers to reconstruct human evolution.; citation_author=LB Jorde; citation_author=M Bamshad; citation_author=AR Rogers; citation_journal_title=Bioessays; citation_volume=20; citation_number=27; citation_pages=126-136; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=The allele frequency spectrum in genome-wide human variation data reveals signals of differential demographic history in three large world populations.; citation_author=GT Marth; citation_author=E Czabarka; citation_author=J Murvai; citation_author=ST Sherry; citation_journal_title=Genetics; citation_volume=166; citation_number=28; citation_pages=351-372; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Recovering the geographic origin of early modern humans by realistic and spatially explicit simulations.; citation_author=N Ray; citation_author=M Currat; citation_author=P Berthier; citation_author=L Excoffier; citation_journal_title=Genome Res; citation_volume=15; citation_number=29; citation_pages=1161-1167; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Calibrating a coalescent simulation of human genome sequence variation.; citation_author=SF Schaffner; citation_author=C Foo; citation_author=S Gabriel; citation_author=D Reich; citation_author=MJ Daly; citation_journal_title=Genome Res; citation_volume=15; citation_number=30; citation_pages=1576-1583; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Human Population Expansion and Microsatellite Variation.; citation_author=LA Zhivotovsky; citation_author=L Bennett; citation_author=AM Bowcock; citation_author=MW Feldman; citation_journal_title=Mol Biol Evol; citation_volume=17; citation_number=31; citation_pages=757-767; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Features of evolution and expansion of modern humans, inferred from genomewide microsatellite markers.; citation_author=LA Zhivotovsky; citation_author=NA Rosenberg; citation_author=MW Feldman; citation_journal_title=Am J Hum Genet; citation_volume=72; citation_number=32; citation_pages=1171-1186; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Generating samples under a Wright-Fisher neutral model of genetic variation.; citation_author=RR Hudson; citation_journal_title=Bioinformatics; citation_volume=18; citation_number=33; citation_pages=337-338; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The signature of positive selection on standing genetic variation.; citation_author=M Przeworski; citation_author=G Coop; citation_author=JD Wall; citation_journal_title=Evolution Int J Org Evolution; citation_volume=59; citation_number=34; citation_pages=2312-2323; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=How reliable are empirical genomic scans for selective sweeps?; citation_author=KM Teshima; citation_author=G Coop; citation_author=M Przeworski; citation_journal_title=Genome Res; citation_volume=16; citation_number=35; citation_pages=702-712; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Mg2+-dependent gating and strong inward rectification of the cation channel TRPV6.; citation_author=T Voets; citation_author=A Janssens; citation_author=J Prenen; citation_author=D Droogmans; citation_author=G Nilius; citation_journal_title=J Gen Physiol; citation_volume=121; citation_number=36; citation_pages=245-260; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Permeation and gating properties of the novel epithelial Ca2+ channel.; citation_author=R Vennekens; citation_author=JG Hoenderop; citation_author=J Prenen; citation_author=M Stuiver; citation_author=PH Willems; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=37; citation_pages=3963-3969; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Whole-cell and single channel monovalent cation currents through the novel rabbit epithelial Ca2+ channel ECaC.; citation_author=B Nilius; citation_author=R Vennekens; citation_author=J Prenen; citation_author=JG Hoenderop; citation_author=RJ Bindels; citation_journal_title=J Physiol Lond; citation_volume=527; citation_number=38; citation_pages=239-248; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Modulation of the epithelial calcium channel, ECaC, by intracellular Ca2+.; citation_author=B Nilius; citation_author=J Prenen; citation_author=R Vennekens; citation_author=JG Hoenderop; citation_author=RJ Bindels; citation_journal_title=Cell Calcium; citation_volume=29; citation_number=39; citation_pages=417-428; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A single pore residue determines calcium selectivity, inward rectification and pore size of TRPV6 (CaT1/ECaC2).; citation_author=T Voets; citation_author=A Janssens; citation_author=G Droogmans; citation_author=B Nilius; citation_journal_title=Biophys J (Annual Meeting Abstracts) 2724-Pos; citation_number=40; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The single pore residue Asp542 determines Ca2+ permeation and Mg2+ block of the epithelial Ca2+ channel.; citation_author=B Nilius; citation_author=R Vennekens; citation_author=J Prenen; citation_author=JG Hoenderop; citation_author=G Droogmans; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=41; citation_pages=1020-1025; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Fast and slow inactivation kinetics of the Ca2+ channels ECaC1 and ECaC2 (TRPV5 and TRPV6). Role of the intracellular loop located between transmembrane segments 2 and 3.; citation_author=B Nilius; citation_author=J Prenen; citation_author=JG Hoenderop; citation_author=R Vennekens; citation_author=S Hoefs; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=42; citation_pages=30852-30858; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The carboxyl terminus of the epithelial Ca(2+) channel ECaC1 is involved in Ca(2+)-dependent inactivation.; citation_author=B Nilius; citation_author=F Weidema; citation_author=J Prenen; citation_author=JG Hoenderop; citation_author=R Vennekens; citation_journal_title=Pflugers Arch; citation_volume=445; citation_number=43; citation_pages=584-588; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Calcium absorption across epithelia.; citation_author=JG Hoenderop; citation_author=B Nilius; citation_author=RJ Bindels; citation_journal_title=Physiol Rev; citation_volume=85; citation_number=44; citation_pages=373-422; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pore properties and ionic block of the rabbit epithelial calcium channel expressed in HEK 293 cells.; citation_author=R Vennekens; citation_author=J Prenen; citation_author=JG Hoenderop; citation_author=RJ Bindels; citation_author=G Droogmans; citation_journal_title=J Physiol; citation_volume=530; citation_number=45; citation_pages=183-191; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Ca2+-selective transient receptor potential V channel architecture and function require a specific ankyrin repeat.; citation_author=I Erler; citation_author=D Hirnet; citation_author=U Wissenbach; citation_author=V Flockerzi; citation_author=BA Niemeyer; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=46; citation_pages=34456-34463; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Competitive regulation of CaT-like-mediated Ca2+ entry by protein kinase C and calmodulin.; citation_author=BA Niemeyer; citation_author=C Bergs; citation_author=U Wissenbach; citation_author=V Flockerzi; citation_author=C Trost; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=47; citation_pages=3600-3605; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=(Patho)physiological implications of the novel epithelial Ca2+ channels TRPV5 and TRPV6.; citation_author=T Nijenhuis; citation_author=JG Hoenderop; citation_author=B Nilius; citation_author=RJ Bindels; citation_journal_title=Pflugers Arch; citation_volume=446; citation_number=48; citation_pages=401-409; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Widespread Parallel Evolution in Sticklebacks by Repeated Fixation of Ectodysplasin Alleles.; citation_author=PF Colosimo; citation_author=KE Hosemann; citation_author=S Balabhadra; citation_author=G Villarreal; citation_author=M Dickson; citation_journal_title=Science; citation_volume=307; citation_number=49; citation_pages=1928-1933; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Soft sweeps: molecular population genetics of adaptation from standing genetic variation.; citation_author=J Hermisson; citation_author=PS Pennings; citation_journal_title=Genetics; citation_volume=169; citation_number=50; citation_pages=2335-2352; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Haldane's sieve and adaptation from the standing genetic variation.; citation_author=HA Orr; citation_author=AJ Betancourt; citation_journal_title=Genetics; citation_volume=157; citation_number=51; citation_pages=875-884; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Soft Sweeps III: The Signature of Positive Selection from Recurrent Mutation.; citation_author=PS Pennings; citation_author=J Hermisson; citation_journal_title=PLoS Genet; citation_volume=2; citation_number=52; citation_pages=e186; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Geography predicts neutral genetic diversity of human populations.; citation_author=F Prugnolle; citation_author=A Manica; citation_author=F Balloux; citation_journal_title=Curr Biol; citation_volume=15; citation_number=53; citation_pages=R159-160; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The Archaeology of Ushki Lake, Kamchatka, and the Pleistocene Peopling of the Americas.; citation_author=T Goebel; citation_author=MR Waters; citation_author=M Dikova; citation_journal_title=Science; citation_volume=301; citation_number=54; citation_pages=501-505; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The Early Settlement of North America: The Clovis Era.; citation_author=G Haynes; citation_number=55; citation_date=2002; citation_publisher=Cambridge Univ. Press; " />
      <meta name="citation_reference" content="citation_title=The history of the human landscapes of New Guinea.; citation_author=GS Hope; citation_author=SG Haberle; citation_number=56; citation_pages=541-554; citation_date=2005; citation_publisher=Pacific Linguistics, Research School of Pacific and Asian Studies, Australian National University; " />
      <meta name="citation_reference" content="citation_title=TRPV6 potentiates calcium-dependent cell proliferation.; citation_author=EC Schwarz; citation_author=U Wissenbach; citation_author=BA Niemeyer; citation_author=B Strauss; citation_author=SE Philipp; citation_journal_title=Cell Calcium; citation_volume=39; citation_number=57; citation_pages=163-173; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=TRPV6 channel controls prostate cancer cell proliferation via Ca(2+)/NFAT-dependent pathways.; citation_author=V Lehen'kyi; citation_author=M Flourakis; citation_author=R Skryma; citation_author=N Prevarskaya; citation_journal_title=Oncogene; citation_volume=26; citation_number=58; citation_pages=7380-7385; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=TRP channels in cancer.; citation_author=N Prevarskaya; citation_author=L Zhang; citation_author=G Barritt; citation_journal_title=Biochim Biophys Acta; citation_volume=1772; citation_number=59; citation_pages=937-946; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=TRPV6 Is a Ca2+ Entry Channel Essential for Ca2+-induced Differentiation of Human Keratinocytes.; citation_author=V Lehen'kyi; citation_author=B Beck; citation_author=R Polakowska; citation_author=M Charveron; citation_author=P Bordat; citation_journal_title=J Biol Chem; citation_volume=282; citation_number=60; citation_pages=22582-22591; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Regulation of the epithelial Ca2+ channels TRPV5 and TRPV6 by 1alpha,25-dihydroxy Vitamin D3 and dietary Ca2+.; citation_author=SF van de Graaf; citation_author=I Boullart; citation_author=JG Hoenderop; citation_author=RJ Bindels; citation_journal_title=J Steroid Biochem Mol Biol; citation_volume=89–90; citation_number=61; citation_pages=303-308; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Chimeric human epidermal reconstructs to study the role of melanocytes and keratinocytes in pigmentation and photoprotection.; citation_author=S Bessou-Touya; citation_author=M Picardo; citation_author=V Maresca; citation_author=JE Surleve-Bazeille; citation_author=C Pain; citation_journal_title=J Invest Dermatol; citation_volume=111; citation_number=62; citation_pages=1103-1108; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Keratinocytes play a role in regulating distribution patterns of recipient melanosomes in vitro.; citation_author=L Minwalla; citation_author=Y Zhao; citation_author=IC Le Poole; citation_author=RR Wickett; citation_author=RE Boissy; citation_journal_title=J Invest Dermatol; citation_volume=117; citation_number=63; citation_pages=341-347; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Farmers and Their Languages: The First Expansions.; citation_author=J Diamond; citation_author=P Bellwood; citation_journal_title=Science; citation_volume=300; citation_number=64; citation_pages=597-603; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Recent origin and cultural reversion of a hunter-gatherer group.; citation_author=H Oota; citation_author=B Pakendorf; citation_author=G Weiss; citation_author=A von Haeseler; citation_author=S Pookajorn; citation_journal_title=PLoS Biol; citation_volume=3; citation_number=65; citation_pages=e71; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation.; citation_author=M Stephens; citation_author=P Scheet; citation_journal_title=Am J Hum Genet; citation_volume=76; citation_number=66; citation_pages=449-462; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A new statistical method for haplotype reconstruction from population data.; citation_author=M Stephens; citation_author=NJ Smith; citation_author=P Donnelly; citation_journal_title=Am J Hum Genet; citation_volume=68; citation_number=67; citation_pages=978-989; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Insights into recombination from patterns of linkage disequilibrium in humans.; citation_author=SE Ptak; citation_author=K Voelpel; citation_author=M Przeworski; citation_journal_title=Genetics; citation_volume=167; citation_number=68; citation_pages=387-397; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Geographic variation in human mitochondrial DNA from Papua New Guinea.; citation_author=M Stoneking; citation_author=LB Jorde; citation_author=K Bhatia; citation_author=AC Wilson; citation_journal_title=Genetics; citation_volume=124; citation_number=69; citation_pages=717-733; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Haplotype variation and linkage disequilibrium in 313 human genes.; citation_author=JC Stephens; citation_author=JA Schneider; citation_author=DA Tanguay; citation_author=J Choi; citation_author=T Acharya; citation_journal_title=Science; citation_volume=293; citation_number=70; citation_pages=489-493; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins.; citation_author=KD Pruitt; citation_author=T Tatusova; citation_author=DR Maglott; citation_journal_title=Nucl Acids Res; citation_volume=33; citation_number=71; citation_pages=D501-504; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Epithelial Ca channel 1 (ECaC1) in autosomal dominant hypercalciuria.; citation_author=D Muller; citation_author=H JGJ; citation_author=R Vennekens; citation_author=P Eggert; citation_author=F Harangi; citation_journal_title=Nephrol Dial Transplant; citation_volume=17: (9); citation_number=72; citation_pages=1614-1620; citation_date=2002; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Parallel Selection on TRPV6 in Human Populations"/>
    <meta name="twitter:description" content="We identified and examined a candidate gene for local directional selection in Europeans, TRPV6, and conclude that selection has acted on standing genetic variation at this locus, creating parallel soft sweep events in humans. A novel modification of the extended haplotype homozygosity (EHH) test was utilized, which compares EHH for a single allele across populations, to investigate the signature of selection at TRPV6 and neighboring linked loci in published data sets for Europeans, Asians and African-Americans, as well as in newly-obtained sequence data for additional populations. We find that all non-African populations carry a signature of selection on the same haplotype at the TRPV6 locus. The selective footprints, however, are significantly differentiated between non-African populations and estimated to be younger than an ancestral population of non-Africans. The possibility of a single selection event occurring in an ancestral population of non-Africans was tested by simulations and rejected. The putatively-selected TRPV6 haplotype contains three candidate sites for functional differences, namely derived non-synonymous substitutions C157R, M378V and M681T. Potential functional differences between the ancestral and derived TRPV6 proteins were investigated by cloning the ancestral and derived forms, transfecting cell lines, and carrying out electrophysiology experiments via patch clamp analysis. No statistically-significant differences in biophysical channel function were found, although one property of the protein, namely Ca2+ dependent inactivation, may show functionally relevant differences between the ancestral and derived forms. Although the reason for selection on this locus remains elusive, this is the first demonstration of a widespread parallel selection event acting on standing genetic variation in humans, and highlights the utility of between population EHH statistics."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001686.g006"/>

  <meta property="og:title" content="Parallel Selection on TRPV6 in Human Populations" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1790'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4264'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9357&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001686">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001686" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001686&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Parallel Selection on <i>TRPV6</i> in Human Populations
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            David A. Hughes
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p>*E-mail: <a href="mailto:hughes@eva.mpg.de">hughes@eva.mpg.de</a></p>

                <p>Affiliation:
                  Max Plank Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kun Tang, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Plank Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rainer Strotmann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biochemistry, Molecular Biochemistry, Medical Faculty, University of Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Torsten Schöneberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Biochemistry, Molecular Biochemistry, Medical Faculty, University of Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jean Prenen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physiology, Katholieke Universiteit Leuven, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Bernd Nilius, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Physiology, Katholieke Universiteit Leuven, Leuven, Belgium
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Stoneking
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Max Plank Institute for Evolutionary Anthropology, Leipzig, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 27, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001686</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001686-g001" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-g002" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-g003" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-t001" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-t002" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-g004" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-g005" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001686-g006" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001686">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001686" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>We identified and examined a candidate gene for local directional selection in Europeans, <em>TRPV6</em>, and conclude that selection has acted on standing genetic variation at this locus, creating parallel soft sweep events in humans. A novel modification of the extended haplotype homozygosity (EHH) test was utilized, which compares EHH for a single allele across populations, to investigate the signature of selection at <em>TRPV6</em> and neighboring linked loci in published data sets for Europeans, Asians and African-Americans, as well as in newly-obtained sequence data for additional populations. We find that all non-African populations carry a signature of selection on the same haplotype at the <em>TRPV6</em> locus. The selective footprints, however, are significantly differentiated between non-African populations and estimated to be younger than an ancestral population of non-Africans. The possibility of a single selection event occurring in an ancestral population of non-Africans was tested by simulations and rejected. The putatively-selected <em>TRPV6</em> haplotype contains three candidate sites for functional differences, namely derived non-synonymous substitutions C157R, M378V and M681T. Potential functional differences between the ancestral and derived <em>TRPV6</em> proteins were investigated by cloning the ancestral and derived forms, transfecting cell lines, and carrying out electrophysiology experiments via patch clamp analysis. No statistically-significant differences in biophysical channel function were found, although one property of the protein, namely Ca<sup>2+</sup> dependent inactivation, may show functionally relevant differences between the ancestral and derived forms. Although the reason for selection on this locus remains elusive, this is the first demonstration of a widespread parallel selection event acting on standing genetic variation in humans, and highlights the utility of between population EHH statistics.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hughes DA, Tang K, Strotmann R, Schöneberg T, Prenen J, et al.  (2008) Parallel Selection on <em>TRPV6</em> in Human Populations. PLoS ONE 3(2):
          e1686.
            doi:10.1371/journal.pone.0001686</p><p><strong>Academic Editor: </strong>Matthew W. Hahn, Indiana University, United States of America</p><p><strong>Received:</strong> September 24, 2007; <strong>Accepted:</strong> January 25, 2008; <strong>Published:</strong> February 27, 2008</p><p><strong>Copyright:</strong> © 2008 Hughes et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The work by DAH, KT and MS was funded by the Max Planck Society. The work of RS and TS is funded by the Bundesministerium für Bildung und Forschung and the work of JP and BN was supported by the Human Frontiers Science Programme (RGP 32/2004), the Belgian Federal Government, the Flemish Government and the Onderzoeksraad KU Leuven (GOA 2004/07, F.W.O. G.0214.99, F.W.O. G. 0136.00; F.W.O. G.0172.03, Poles of Attraction Program, Prime Ministers Office IUAP Nr.3P4/23, Excellentiefinanciering EF/95/010).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Recently, with the advent of large, publicly-available data sets, the completion of many mammalian genome sequences, and large-scale genotyping resources, there has been much effort in attempting to identify loci which have potentially been under selection in the entire human species, or locally in specific human populations <a href="#pone.0001686-Akey1">[1]</a>–<a href="#pone.0001686-Walsh1">[11]</a>. While there has been much success in producing lists of candidate genes, less effort has gone into a detailed characterization of the candidate genes, to verify the signature of selection obtained from genome-wide data, elucidate the functional differences between selected and non-selected alleles and their phenotypic consequences, and (ultimately) identify the nature of the selective force that produced the footprint of selection. Such further characterization is necessary for a complete understanding of the role and consequences of selection in shaping human genetic variation.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Here we present an analysis of independent parallel local selection on a candidate gene, <em>TRPV6</em>. <em>TRPV6</em> encodes a Ca<sup>2+</sup> selective ion channel of the transient receptor potential (TRP) superfamily, which is critically involved in Ca<sup>2+</sup> (re)absorption <a href="#pone.0001686-Hoenderop1">[12]</a>. <em>TPRV6</em> is the primary route of dietary calcium uptake, and has previously been identified as a candidate for local selection in two other studies <a href="#pone.0001686-Akey1">[1]</a>, <a href="#pone.0001686-Stajich1">[8]</a>. These studies both used the SeattleSNPs data base, consisting at the time of full sequence data for over 130 genes in twenty-four individuals of African American decent and twenty-three individuals of European decent. Both studies identified putative selected genes by comparing the empirical data to simulated data that was based on an assumed demographic history. These studies found that the <em>TRPV6</em> locus has an excess of high frequency derived alleles, an excess of rare alleles, and reduced diversity in Europeans relative to African-Americans, all consistent with local directional selection in Europeans. Furthermore, the date of selection in Europeans was estimated to be around 10,000 years before present, suggesting that <em>TRPV6</em> may have coevolved with lactose tolerance <a href="#pone.0001686-Bersaglieri1">[13]</a> as fresh milk is a major source of calcium in European populations.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The <em>TRPV6</em> locus contains three non-synonymous substitutions (C157R, M378V and M681T) for which the derived allele is fixed (or nearly so) in Europeans, but are each polymorphic in African-Americans with a derived allele frequency of ~50%. The lack of any unique high frequency allele in Europeans suggested that selection on <em>TRPV6</em> must have occurred on a “preexisting mutation in the ancestral African population…”, which subsequently “…became advantageous in a new environment and rose to high frequency” <a href="#pone.0001686-Stajich1">[8]</a>. However, three genes flanking <em>TRPV6</em>, namely <em>EPHB6</em>, <em>TRPV5</em> and <em>KEL</em>, were also identified as potential target(s) of selection that could then be influencing genetic variation in neighboring genes through hitchhiking <a href="#pone.0001686-Akey1">[1]</a>, <a href="#pone.0001686-Thomson1">[14]</a>. Nonetheless, since <em>TRPV6</em> exhibits the most extreme departures from neutrality and is the only gene among these four with any high frequency derived amino acid substitutions, it is considered to be the most likely target of selection <a href="#pone.0001686-Akey1">[1]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>A subsequent study found that <em>TRPV6</em> exhibits accelerated protein evolution on the human lineage, as compared to Pan, Mus and Rattus <a href="#pone.0001686-Akey3">[15]</a>. A branch-sites model <a href="#pone.0001686-Yang1">[16]</a> identified six sites with a high posterior probability of having been subject to selection; three of these sites are the nonsynonymous substitutions for which the derived alleles define the putatively selected haplotype in Europeans <a href="#pone.0001686-Akey3">[15]</a>. This study also sequenced the three exons containing these non-synonymous substitutions in 90 individuals from six human populations, as well as genotyped these three sites in the CEPH diversity panel, a collection of DNAs from 51 human populations <a href="#pone.0001686-Cann1">[17]</a>. The three non-synonymous sites were fixed or nearly so for the derived alleles in all non-African populations, and each of the non-African populations exhibited reduced diversity as compared to sub-Saharan African populations, consistent with the action of selection <a href="#pone.0001686-Akey3">[15]</a>. This shared signature of selection among all non-African populations studied led to the suggestion that the selective event occurred around the time of the major human migration out of Africa some 60,000–100,000 years before present, and moreover indicated that dairying could not be the selective pressure <a href="#pone.0001686-Akey3">[15]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>In the present study we have also independently identified <em>TRPV6</em> as a candidate gene of local directional selection, using the SeattleSNPs data set and a different genome scan approach. We further characterize the footprint of selection on <em>TRPV6</em> in the aforementioned and additional populations, using a novel extended haplotype homozygosity approach, and find evidence for independent, parallel selection in Europeans and Asians (and possibly other populations). Our data and analyses suggest selection on standing genetic variation, also known as soft sweeps, in different populations for the same haplotype that differs from the ancestral haplotype by the three previously mentioned nonsynonymous substitutions. In addition, we also investigated the functionality of the ancestral form of <em>TRPV6</em> and the putatively-selected, derived form of <em>TRPV6</em>. Although some electrophysiological differences in protein function were found between the two forms in <em>TRPV6</em>, these differences were not statistically-significant. Thus, while functional consequences of the variation at <em>TRPV6</em> and the underlying selective force remain to be determined, this study does advance our knowledge about the temporal and spatial pattern of selection on <em>TRPV6</em>.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>SeattleSNP Genome Scan, Neutrality tests, and Dating Time since Selection</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We identified <em>TRPV6</em> as a putative target of local selection by implementing a genome scan based on two summary statistics of the sequence data (<a href="#pone.0001686.s001">Figure S1</a>), analogous to that performed previously <a href="#pone.0001686-Kayser1">[6]</a>. A drastic reduction in diversity was identified at the <em>TRPV6</em> locus (<a href="#pone.0001686.s002">Figure S2</a>) and at the neighboring loci <em>EPHB6</em>, <em>TRPV5</em> and <em>KEL</em>. Additionally, statistically significant evidence for selection based on summary statistics of the allele frequency spectrum and the HKA test for each of these genes was found (<a href="#pone.0001686.s010">Table S2</a>). However, <em>TRPV6</em> exhibited the most significant departures from neutrality. Given these data and previous studies <a href="#pone.0001686-Akey1">[1]</a>, <a href="#pone.0001686-Stajich1">[8]</a>, <a href="#pone.0001686-Akey3">[15]</a> we focused on the <em>TRPV6</em> locus as the putative site of selection. We then used the full sequence data from SeattleSNPs to estimate the time since fixation in Europeans, using two different methods. We estimated the age since fixation in Europeans to be approximately 7 ky (<a href="#pone.0001686.s003">Figure S3</a>). These results are comparable to a previous estimated date of 10 ky <a href="#pone.0001686-Akey1">[1]</a>, but are not consistent with a suggested date of 60–100 ky <a href="#pone.0001686-Akey3">[15]</a>, which was based solely on the apparent global distribution of selection at this locus. Finally, we also genotyped the CEPH diversity panel for the three non-synonymous substitutions (C157R, M378V and M681T) in a MALDI-TOF assay (<a href="#pone.0001686.s012">Table S4</a>) and confirmed near fixation for the derived allele at each of these sites in all non-African populations <a href="#pone.0001686-Akey3">[15]</a>. Further, discussion of these results, and the relevant materials and methods for these analyses, which confirm previous work, can be found in Supporting Information.</p>


<h4>Rab (Between Populations EHH) at <em>TRPV6</em> in Europeans</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Recently, new methods have been developed for investigating loci for selection by evaluating the extended haplotype homozygosity (EHH) associated with an allele <a href="#pone.0001686-Voight1">[10]</a>, <a href="#pone.0001686-Sabeti1">[18]</a>, <a href="#pone.0001686-Tang1">[19]</a>. The premise of the EHH test is to evaluate the age of an allele <a href="#pone.0001686-Fry1">[20]</a> by comparing the decay of haplotype homozygosity among haplotypes with a pre-defined set of core SNPs. Haplotypes that have been subject to selection will have increased in frequency more rapidly than expected under neutrality, and hence will be associated with extended haplotypes relative to haplotypes that have not been subject to selection. A caveat and yet a useful feature of EHH tests is that discernable selective signatures may be found for selective events that occurred up to 400 generations ago <a href="#pone.0001686-Sabeti1">[18]</a> but are not expected for much more than 1,000 generations <a href="#pone.0001686-Voight1">[10]</a>, <a href="#pone.0001686-Sabeti2">[21]</a>. Moreover, it has recently been shown that EHH tests have greater power to detect recent selection than other tests <a href="#pone.0001686-Voight1">[10]</a>. We utilized this methodology and calculated Raw values (<span class="underline">r</span>elative iEHH of an <span class="underline">a</span>llele <span class="underline">w</span>ithin a population), which are based on comparing the EHH associated with the two different alleles at a SNP within a single population <a href="#pone.0001686-Tang1">[19]</a>. We also introduce a novel EHH statistic called Rab (<span class="underline">r</span>elative iEHH of an <span class="underline">a</span>llele <span class="underline">b</span>etween populations), which compares the iEHH (integrated EHH) of the same allele between two different populations (<a href="#pone-0001686-g001">Figure 1</a>, see <a href="#s3">Materials and Methods</a>). The Rab statistic is directly analogous to the recently developed Rsb statistic (<span class="underline">r</span>elative iEHH of a <span class="underline">s</span>ite <span class="underline">b</span>etween populations) <a href="#pone.0001686-Tang1">[19]</a>, and allows us to analyze SNPs which are fixed (or nearly so) in a population and for which there is thus little information in a within-population comparison. Extreme Rab values would be associated with alleles that have potentially been under selection in one of the two populations. The median Rab value between Europeans and African-Americans is 1.54 in the SeattleSNP data, indicating that on average there is more EHH in Europeans than in African-Americans, consistent with previous findings of more linkage disequilibrium in Europeans than in African-Americans <a href="#pone.0001686-Wang1">[22]</a>–<a href="#pone.0001686-Reich1">[24]</a>. To determine if the <em>TRPV6</em> locus in Europeans has a preponderance of sites that have extreme Rab values with respect to African-Americans, we calculated the number of sites found above the 95<sup>th</sup> quantile of the Rab distribution for each gene in the SeattleSNPs data set and in a simulated data set (see <a href="#s3">Materials and Methods</a>). From the empirical distribution, 56.25% of the 112 <em>TRPV6</em> SNPs are above the 95<sup>th</sup> percentile and 28.57% are above the 99<sup>th</sup> percentile. The extent of the difference in EHH between Europeans and African-Americans can be seen in <a href="#pone-0001686-g001">Figure 1</a>. None of the other 220 genes in the SeattleSNPs dataset exceed <em>TRPV6</em> in terms of the number of SNPs with Rab values above either the 95<sup>th</sup> or the 99<sup>th</sup> percentile. Based on the quantiles of the simulated data of 10,000 loci, 58.92% (p-value = 0.0021) and 57.14% (p-value = 0.0001) of <em>TRPV6</em> SNPs are above the 95<sup>th</sup> and 99<sup>th</sup> quantile respectively. These analyses demonstrate that the EHH associated with <em>TRPV6</em> in Europeans is highly unusual when compared to both other genes in the SeattleSNPs data and to simulated data, suggesting that local directional selection has indeed influenced variation and haplotype homozygosity at the <em>TRPV6</em> locus in Europeans.</p>
<div class="figure" id="pone-0001686-g001"><div class="img"><a name="pone-0001686-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>An EHH plot of the <em>TRPV6</em> locus from the SeattleSNP data, illustrating the calculation of Rab and Raw statistics.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The core SNP is the M378V mutation, where allele A is derived and allele G is ancestral.</p>
<span>doi:10.1371/journal.pone.0001686.g001</span></div>

<h4>Signature of selection on <em>TRPV6</em> in other non-African populations</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>All non-African populations are fixed or nearly so for the derived, putatively selected haplotype in Europeans (see <a href="#pone.0001686.s005">Figure S5</a>); do they also show the same signature of selection as in Europeans? To address this question, we sequenced 4.1 kb of non-consecutive sequence in a 52 kb region around <em>TRPV6</em> (<a href="#pone.0001686.s006">Figure S6</a>) in four additional non-African populations: Karitiana from Brazil, Pathan from Pakistan, Han Chinese, and highland Papua New Guineans. As in Europeans, there is an extreme paucity of variation in these populations as compared to Africans (<a href="#pone-0001686-g002">Figure 2</a>), as well as large EHH when compared to the same alleles in Africans (<a href="#pone-0001686-t001">Table 1</a>). Moreover, each of these populations has haplotype homozygosity levels and Rab values that are comparable to or even greater than that found in Europeans (<a href="#pone-0001686-g003">Figure 3A</a>; <a href="#pone-0001686-t001">Table 1</a>). Strikingly, the polymorphisms in this 4.1 kb of sequence include only one singleton and one doubleton in the Han Chinese, only one singleton in the Karitiana, the presence of a single African-like chromosome in highland New Guinea (as also found in Europeans), and the presence in the Pathan of two distinct haplotypes both carrying the derived, putatively-selected alleles, possibly as a result of a recombination event prior to or early in the selective phase (<a href="#pone-0001686-g003">Figure 3A</a>). These results indicate that local directional selection has acted on each of these populations, and by extension in all non-African populations, as it has in Europeans.</p>
<div class="figure" id="pone-0001686-g002"><div class="img"><a name="pone-0001686-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Visual haplotype graph and network for sequence data around the <em>TRPV6</em> locus in 6 populations.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>A) Visual haplotype graph of sequence data generated for four populations (New Guinea highlanders, the Pathan of Pakistan, Han Chinese, and the Karitiana of Brazil), compared to the European and African sequences from the Seattle SNPs dataset. B) A network of the sequence data from the Karitiana, Pathan, New Guinea highlanders, Han Chinese, Europeans, and Africans.</p>
<span>doi:10.1371/journal.pone.0001686.g002</span></div><div class="figure" id="pone-0001686-g003"><div class="img"><a name="pone-0001686-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>EHH decay plots for sequence data and HapMap data.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>A) An EHH decay plot with the C157R mutation as the core SNP from sequence data generated for the Karitiana, Han Chinese, New Guinea highlanders, and Pathan, along with data for Europeans and Africans (light green is the ancestral allele, dark green is the derived allele) from the SeattleSNPs dataset. Only the derived allele is plotted for each non-African population. B) EHH decay plot from HapMap data with the M378T mutation as the core. For the Yoruba, light green indicates the ancestral haplotype and dark green indicates the derived haplotype.</p>
<span>doi:10.1371/journal.pone.0001686.g003</span></div><div class="figure" id="pone-0001686-t001"><div class="img"><a name="pone-0001686-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary statistics for approximately 4.1 kb of sequence data in and around <em>TRPV6</em> for six populations.</span></strong></p><span>doi:10.1371/journal.pone.0001686.t001</span></div>

<h4>EHH and Rab at <em>TRPV6</em> in HapMap data:</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To further investigate the putative selection on <em>TRPV6</em> in non-Africans, we analyzed the EHH patterns in the HapMap data <a href="#pone.0001686-The2">[25]</a>, which provides an analysis of EHH over a much larger physical distance for Europeans and Africans (Yoruba) than with the SeattleSNPs data, as well as the addition of two Asian populations (Han Chinese and Japanese). We calculated Rab values using HapMap Phase I data for each non-African population compared to the Yoruba for approximately 20,000 genes. We then determined the proportion of sites which were above the 95<sup>th</sup> percentile for each gene and compared these to <em>TRPV6</em> and neighboring genes (<a href="#pone-0001686-t002">Table 2</a>). In the two Asian groups, the <em>TRPV6</em> locus had a significant proportion of SNPs that were above the 95<sup>th</sup> percentile of their respective Rab bins, while Europeans were nearly significant. None of the other genes closely-linked to <em>TRPV6</em> were significant in this test, although they do harbor alleles which have significant Rab values. Previously, the region surrounding <em>TRPV6</em> (including <em>TRPV5</em> and <em>EPHB6</em>) was identified as a target of selection using the analogous Rsb statistic <a href="#pone.0001686-Tang1">[19]</a>. This difference most likely is because our analysis focuses on individual genes, whereas the previous study identified larger genomic regions that had been influenced by local selection <a href="#pone.0001686-Tang1">[19]</a>.</p>
<div class="figure" id="pone-0001686-t002"><div class="img"><a name="pone-0001686-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Rab significance values for <em>EPHB6</em>, <em>TRPV6</em>, <em>TRPV5</em>, and <em>KEL</em> in pairwise population comparisons from the HapMap data.</span></strong></p><span>doi:10.1371/journal.pone.0001686.t002</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>Remarkably, the EHH around <em>TRPV6</em> extends more than 820 kb in Asians (both Chinese and Japanese), compared to 582 kb in Europeans and 190 kb in the Yoruba (<a href="#pone-0001686-g003">Figure 3B</a>). This raises the question as to whether the signal of selection at <em>TRPV6</em> differs between Europeans and Asians. We therefore compared the iEHH of each allele between the Asians and Europeans; the Rab values are significant (or nearly so) for all four genes in the TPRV6 region when either Asian group is compared to Europeans, as compared to other genes in the HapMap phase 1 data (<a href="#pone-0001686-t002">Table 2</a>). On average, the amount of EHH found in Han Chinese and Japanese at the <em>TRPV6</em> locus is 2.18 and 2.39 times greater than that in Europeans, and both are 8.23 times greater than that in Africans. Thus, the <em>TRPV6</em> region in Asians has a significant excess of alleles with significant Rab (iEHH), as compared to the remainder of the genome and to either Africans or Europeans.</p>


<h4>Parallel Selection events or a Single Selective Event</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Although it was previously hypothesized that selection on <em>TRPV6</em> occurred in a population ancestral to all non-African populations<a href="#pone.0001686-Akey3">[15]</a>, the significant Rab values obtained when comparing Asian populations to European, as well as the estimated time since fixation in Europeans of 7 ky, raises the intriguing possibility that selection may have occurred independently in Asians and Europeans, with stronger and/or more recent selection in Asians. We therefore used simulations to investigate if the observed differences in EHH between Europeans and Asians (Rab of ~2.3) could be explained by a single selection event which occurred in the ancestral population of Europeans and Asians, some 50 ky <a href="#pone.0001686-Harpending1">[26]</a>–<a href="#pone.0001686-Zhivotovsky2">[32]</a>, as suggested previously <a href="#pone.0001686-Akey3">[15]</a>. Using Hudson's ms program <a href="#pone.0001686-Hudson1">[33]</a> and demographic parameters previously found to provide the best fit to human data <a href="#pone.0001686-Schaffner1">[30]</a> we simulated 850kbp segments (roughly the length needed for EHH in Asians to reach 0.05) for Africans, Europeans and Asians. To simulate a very strong selective event, we introduced a severe bottleneck in the ancestral population of non-Africans, setting the effective populations size (Ne) to 1 at 50,100 years before present for five generations. This would have the same effect as a selective event, in that a single haplotype will be fixed in this ancestral population, prior to the split of Europeans and Asians. This scenario provides a maximum amount of opportunity for the subsequent independent demographic histories of Asian and European populations to accumulate mutations and recombination events, which may then recapitulate the observed difference in EHH (Rab) seen in the empirical data. Our null hypothesis is that these independent demographic histories could, following a shared selective event, explain the observed difference in EHH between Asians and Europeans; rejecting this null hypothesis would imply independent selection for the same haplotype in Asians and Europeans. Average Rab values across all sites and all simulations were 1.08 for Asians as compared to Europeans, stressing the similarity in EHH between the two populations. As expected following a selective sweep, large Rab values were observed between non-Africans (who experienced the simulated selective event) and Africans (who did not experience the selection): average Rab equals 6.42 for Asians as compared to Africans, and 6.41 for Europeans compared to Africans. Moreover, a significant difference was observed between empirical and simulated Rab values between Europeans and Asians (p = 0.023, see <a href="#s3">Materials and Methods</a>). These results suggest that the observed, significantly different EHH between Asians and Europeans is unlikely to reflect a single selection event in an ancestral population of these two populations some 50,000 years ago. Rather, we conclude from the data that the same <em>TRPV6</em> haplotype was subject to more recent, and hence independent, parallel selection in these different populations, with stronger and/or more recent selection in Asians than in Europeans.</p>


<h4>Selection on <em>TRPV6</em> in Africans</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>Allele frequency and the HKA tests revealed no significant deviations from neutrality in African-Americans (<a href="#pone.0001686.s009">Table S1</a>), however there were a large number of African-American haplotypes which were closely related to the star-shaped cluster of European haplotypes (<a href="#pone-0001686-g002">Figure 2B</a>). Although this may reflect European admixture, we also observed the selected haplotype at moderate to high frequency in some populations of Africa (<a href="#pone.0001686.s005">Figure S5</a>). Thus, is there any signal of selection on <em>TRPV6</em> in African populations?</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We first investigated EHH at the <em>TRPV6</em> locus in African-Americans by calculating Raw values <a href="#pone.0001686-Tang1">[19]</a> for the SeattleSNPs African-American data; these Raw values were then compared to both the empirical distribution of Raw values for all genes in the Seattle SNPs data for African-Americans, and to a simulated distribution. The proportion of Raw values above the 95<sup>th</sup> percentile approaches statistical significance when compared to the empirical SeattleSNPs distribution (p = 0.064 with 12.9% of 109 sites significant), and is highly significant when compared to the simulated data, with 41.3% of the 109 sites above the 95<sup>th</sup> quantile (p = 0.012) and with 22% of the sites above the 99<sup>th</sup> quantile (p = 0.006). Since this signature of selection could reflect European admixture in the SeattleSNPs African-American dataset, we also calculated Raw statistics for each SNP for the Yoruba from the HapMap data. No allele at the <em>TRPV6</em> locus in the Yoruba has a significant Raw value. The signature of selection that is present in the SeattleSNPs data therefore is likely to be the result of admixture with individuals of European origin, or (less likely) reflects very recent selection in this African-American population.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>It was previously reported that the Mozabite population of North Africa has an excess of high frequency derived alleles and a significant Fay and Wu's H value <a href="#pone.0001686-Akey3">[15]</a>, which is consistent with the high frequency of the putative-selected haplotype in this population (<a href="#pone.0001686.s005">Figure S5</a>). However, this could be explained by admixture of this North African population with Europeans, and not necessarily selection operating on this population. Furthermore it was also previously suggested that South Africans and African Americans may be experiencing some form of balancing selection at the <em>TRPV6</em> locus, based on large positive Tajima's D values <a href="#pone.0001686-Akey3">[15]</a>. Alternatively, given the data presented here, these large positive Tajima's D values could reflect an incomplete sweep that is in the process of driving some alleles to moderate frequency, while others are decreasing in frequency. Another possibility is the action of a soft sweep (i.e. selection on standing variation), shortening EHH, creating spurious values in neutrality tests, and in general confounding the signature of selection <a href="#pone.0001686-Przeworski1">[34]</a>, <a href="#pone.0001686-Teshima1">[35]</a>. It would be necessary to obtain more sequence data in and around the <em>TRPV6</em> locus from more African populations, to determine if <em>TRPV6</em> has experienced non-neutral evolution in African populations, while considering the possible confounding affects of a soft sweep.</p>


<h4>Functional analysis of <em>TRPV6</em> haplotypes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The putatively-selected <em>TRPV6</em> haplotype is defined by the derived state of three amino acid substitutions: C157R, M378V, and M681T. The presence of three amino acid substitutions on a recently selected haplotype suggests that one (or more) of these substitutions may result in some phenotypic difference that may then be influenced by selection. We therefore investigated possible functional differences between the ancestral and derived <em>TRPV6</em> proteins by cloning each form into expression vectors and transfecting the constructs into a human cell line; we then used patch clamp analysis to investigate the electrophysiology of the <em>TRPV6</em> calcium channel in cells expressing either the ancestral or derived <em>TRPV6</em> proteins. Our functional screening comprised the most typical features of <em>TRPV6</em>, namely, the strong inward rectification (i.e. the inward currents are much larger than outward currents) <a href="#pone.0001686-Voets1">[36]</a>, the very positive reversal potentials if Ba<sup>2+</sup> or Ca<sup>2+</sup> are the charge carriers (indicating high selectivity for both cations), block by Mg<sup>2+</sup> of <em>TRPV6</em> activity, and Ca<sup>2+</sup> dependent inactivation.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The intracellular expression pattern of both constructs was assessed by epifluorescence microscopy and revealed largely plasma membrane-bound CFP fluorescence, compatible with surface expression of the proteins. No differences in the expression levels or surface expression of derived and ancestral <em>TRPV6</em> were observed (data not shown). <a href="#pone-0001686-g004">Figure 4</a> shows a typical patch clamp experiment in which a voltage ramp protocol was used under different ion conditions. In the absence of any divalent cations in the bath solution, e.g. with the current carried by Na<sup>+</sup>, large and strongly inward rectifying currents were measured which reversed close to 0 mV. This is the typical hallmark of <em>TRPV5</em> and <em>TRPV6</em> currents <a href="#pone.0001686-Vennekens1">[37]</a>–<a href="#pone.0001686-Nilius2">[39]</a>. However, when Ca<sup>2+</sup> or Ba<sup>2+</sup> were the only charge carrier for inward currents, the current density increased and the reversal potentials were shifted to more than +50 mV, indicating high selectivity for these divalent cations <a href="#pone.0001686-Vennekens1">[37]</a>. This experiment shows that major changes in permeability cannot be expected between derived and ancestral <em>TRPV6</em>. This is not unexpected because the pore region, which is responsible for this unique permeation pattern <a href="#pone.0001686-Voets1">[36]</a>, <a href="#pone.0001686-Voets2">[40]</a>, <a href="#pone.0001686-Nilius3">[41]</a>, is the same in the two versions of <em>TRPV6</em>. A fast initial decay was observed when Ca<sup>2+</sup> was the charge carrier (<a href="#pone-0001686-g004">Figure 4A</a>). This fast decrease in the Ca<sup>2+</sup> entry is due to channels closing, i.e. inactivation, a process which is mediated by Ca<sup>2+</sup> itself entering the cell through the channel and is known as Ca<sup>2+</sup> dependent inactivation <a href="#pone.0001686-Hoenderop1">[12]</a>, <a href="#pone.0001686-Nilius2">[39]</a>, <a href="#pone.0001686-Nilius4">[42]</a>, <a href="#pone.0001686-Nilius5">[43]</a>. Ca<sup>2+</sup>-dependent inactivation acts in the sense of a negative feedback on Ca<sup>2+</sup> entry through the channel. This functional fingerprint was also observed for the ancestral construct (<a href="#pone-0001686-g004">Figure 4B</a>). By using voltage steps from +70 mV to −100 mV, inactivation of currents through <em>TRPV6</em> was measured. If Ca<sup>2+</sup> is the charge carrier, an initial fast inactivation is followed by a slower decay (<a href="#pone-0001686-g004">Figure 4C</a>). In divalent-free solution (DVF) and with Ba<sup>2+ </sup>as the charge carrier, no inactivation was observed during the first 400 ms. This behavior is again typical for the Ca<sup>2+</sup>-dependent inactivation of <em>TRPV6</em>, as described in detail previously <a href="#pone.0001686-Nilius5">[43]</a>. An identical behavior was observed for the ancestral construct (<a href="#pone-0001686-g004">Figure 4D</a>).</p>
<div class="figure" id="pone-0001686-g004"><div class="img"><a name="pone-0001686-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Current–voltage relationships (IV curves) for the derived (CMM) and ancestral (RVT) forms of <em>TRPV6.</em></span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>A) IV curves for the derived form obtained from the voltage ramps shown in the inset. Note the strong inward rectification and the shift of the reversal potentials when Ca<sup>2+</sup> or Ba<sup>2+</sup> are the charge carriers (DVF: divalent free solution, Ca<sup>2+</sup>: 30 mM CaCl<sub>2</sub> with NMDG<sup>+</sup> as substitute for Na<sup>+</sup>, see <a href="#s3">methods</a> for details). B) IV curves for the ancestral form, using the same protocol as in A. C) Current activation for the derived <em>TRPV6</em> form by voltage steps from +70 mV to −100 mV as shown in the inset. Note the fast initial inactivation when Ca<sup>2+</sup> is the charge carrier and the absence of current decay in DVF conditions and with Ba<sup>2+</sup> as the charge carrier. D) Current activation for the ancestral <em>TRPV6</em> form, with the same protocol as in C.</p>
<span>doi:10.1371/journal.pone.0001686.g004</span></div><a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>The typical response to changes in the extracellular solution for the ancestral and the derived construct are shown in <a href="#pone-0001686-g005">Figure 5</a>. Large currents were observed in ramp protocols at −100mV in DVF solution. Substitution of all extracellular cations by the non-permeable N-methyl d-glucamine (NMDG<sup>+</sup>) abolished the inward current. If the solution was changed to Ba<sup>2+</sup> as the only charge carrier, the current increased again and remained stable. Application of a solution with Ca<sup>2+</sup> as the only charge carrier induced a transiently increased current, which rapidly decayed (<a href="#pone-0001686-g005">Figure 5A</a>). The respective IV curves obtained from voltage ramps are shown in <a href="#pone-0001686-g005">figure 5B</a> and represent the typical IV fingerprint shown in <a href="#pone-0001686-g004">figure 4A</a>. The same pattern was observed for the ancestral construct: large inward currents in DVF solution, relative stable or only slowly decaying currents with Ba<sup>2+</sup> as charge carrier, and rapidly inactivating current when the charge carrier changed to Ca<sup>2+</sup> (<a href="#pone-0001686-g005">Figure 5C</a>). Also, the respective currents obtained from the time course are similar to the ones shown in <a href="#pone-0001686-g004">figure 4B</a>.</p>
<div class="figure" id="pone-0001686-g005"><div class="img"><a name="pone-0001686-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Time course of <em>TRPV6</em> currents with changing extracellular ionic conditions.</span></strong></p><a id="article1.body1.sec2.sec7.fig2.caption1.p1" name="article1.body1.sec2.sec7.fig2.caption1.p1"></a><p>A) Current density for the derived <em>TRPV6</em> form at −100 mV obtained from voltage ramps. Bars denoted DVF indicates superfusion of the cell with a divalent free solution; bars above the time course indicate changes to NMGD<sup>+</sup>, Ba<sup>2+</sup> and Ca<sup>2+</sup>. Note the typical large currents in DVF, the complete inhibition of inward current with NMDG<sup>+</sup>, the relatively stable current in Ba<sup>2+</sup>, and the fast decay with Ca<sup>2+ </sup>as the charge carrier. Also note the slow recovery from Ca<sup>2+</sup> dependent inactivation. B) IV curves from the time course shown in A, measured at the times which are indicated by solid points (red DVF, blue Ba<sup>2+</sup>, green Ca<sup>2+</sup>). C and D) Identical experiments as in A and B, respectively, with the ancestral <em>TRPV6</em> form.</p>
<span>doi:10.1371/journal.pone.0001686.g005</span></div><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>To compare these functional data statistically, we measured several parameters. First, current density at −150mV did not differ significantly between the derived and ancestral <em>TRPV6</em> proteins under all three conditions: DVF; charge carrier Ca<sup>2+</sup>; or charge carrier Ba<sup>2+</sup> (<a href="#pone-0001686-g006">Figure 6A</a>). Second, the same holds when only data were taken from cells in which all three protocols could be measured (DVF, Ba<sup>2+</sup>, Ca<sup>2+</sup>) and the current density with Ca<sup>2+</sup> and Ba<sup>2+</sup> as the charge carrier could be normalized to the monovalent (DVF) current at −150 mV (<a href="#pone-0001686-g006">Figure 6B</a>). Third, as an estimate for Ca<sup>2+</sup> dependent inactivation, we compared the ratio of the currents at the end of the 400 ms voltage step to −100 mV and the maximal inward current during the step (I<sub>400ms,−100mV</sub>/I<sub>max</sub>) for derived and ancestral <em>TRPV6</em>. This ratio reflects the extent of negative feedback inhibition of <em>TRPV6</em> by Ca<sup>2+</sup>. No inactivation is present with Ba<sup>2+</sup> and under the DVF condition. In the presence of 30 mM Ca<sup>2+</sup>, inactivation of 48.9 +/− 5% and 34.9 +/− 6% was observed for the ancestral and derived <em>TRPV6</em> proteins. This difference in inactivation has a low, albeit not statistically-significant, probability of occurring by chance (p = 0.094, <a href="#pone-0001686-g006">Figure 6C</a>); however, this difference could be biologically significant as feedback inhibition and expression are together perhaps the most important regulators of the protein. Continuing, it has been shown in detail that extracellular Mg<sup>2+</sup> blocks both <em>TRPV5</em> and <em>TRPV6</em> currents <a href="#pone.0001686-Hoenderop1">[12]</a>, <a href="#pone.0001686-Nilius1">[38]</a>, <a href="#pone.0001686-Hoenderop2">[44]</a>, <a href="#pone.0001686-Vennekens2">[45]</a>. We have therefore, fourth, measured the current decrease when 1 mM Mg<sup>2+</sup> was added to the DVF solution in voltage step experiments. Currents were inhibited by ~50% and again no significant difference between the ancestral and derived forms was observed (<a href="#pone-0001686-g006">Figure 6D</a>).</p>
<div class="figure" id="pone-0001686-g006"><div class="img"><a name="pone-0001686-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001686" data-uri="info:doi/10.1371/journal.pone.0001686.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001686.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001686.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001686.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001686.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Analyses of three key feature of <em>TRPV6</em>: current density, Ca<sup>2+</sup> dependent inactivation and block by Mg<sup>2+</sup>.</span></strong></p><a id="article1.body1.sec2.sec7.fig3.caption1.p1" name="article1.body1.sec2.sec7.fig3.caption1.p1"></a><p>Data from between 3 and 5 cells for all graphs (except where stated otherwise) with p values are indicated. A) Current density was measured at −150 mV under three ionic conditions for derived <em>TRPV6</em> and for ancestral <em>TRPV6</em>. DVF: divalent free (13 derived and 12 ancestral cells measured); Ba<sup>2+</sup>, Ca<sup>2+</sup> refer to these divalent cations as the only charge carrier for the inward currents. Note that high current densities have been measured indicating high functional expression of the channels. B) Data were taken only from the same cells in which all three condition (DVF, Ba<sup>2+</sup>, Ca<sup>2+</sup>) could be measured (see protocol panel A). Current densities were normalized to the current in DVF solution at −150 mV. C) Ca<sup>2+</sup> dependent inactivation was estimated by the ratio of the currents at the end of the 400 ms voltage step and the maximal inward current during the step (I<sub>400ms,−100mV</sub>/I<sub>max</sub>). Eleven derived and 13 ancestral alleles were tested for the effect in which Ca<sup>2+</sup> has on Ca<sup>2+</sup> dependent inactivation D) Block of <em>TRPV6</em> by Mg<sup>2+</sup> was estimated from step protocols. I<sub>1Mg2+,−100mV</sub>/I<sub>DVF,−100mV</sub> is the ratio of the maximal inward current measured from the same cell after changing from DVF to addition of 1 mM Mg<sup>2+</sup>. (1-ratio) gives the inhibition.</p>
<span>doi:10.1371/journal.pone.0001686.g006</span></div><a id="article1.body1.sec2.sec7.p5" name="article1.body1.sec2.sec7.p5"></a><p>One of the amino acid differences between the derived and ancestral versions of <em>TRPV6</em>, C157R, is located between the third and the fourth ankyrin repeats of <em>TRPV6</em> (see <a href="#pone.0001686.s004">Figure S4</a>), which have been shown to be functionally important for multimerization, may initiate a molecular zippering process for protein-protein interaction, and may create an intracellular anchor necessary for functional subunit assembly <a href="#pone.0001686-Erler1">[46]</a>. Therefore, a C157R mutation could be functionally-significant in protein-protein interactions, and we cannot exclude at this point that protein-protein interactions might differ between the derived and ancestral versions of <em>TRPV6</em>. However, channel function per se is not significantly changed by the C157R mutation. It is more difficult to predict any functional changes for the M378V mutant, which is located extracellularly, in the loop between the first and the second transmembrane helix, TM1 and TM2. The residue M681T is located in a functionally important region, very close to the putative Ca<sup>2+</sup>/calmodulin binding site between residues 691 and 711 <a href="#pone.0001686-Nilius5">[43]</a>. Ca<sup>2+</sup>/calmodulin binding to this region initiates Ca<sup>2+</sup>-dependent fast inactivation of <em>TRPV6</em>, which can be reversed by PKC dependent phosphorylation <a href="#pone.0001686-Niemeyer1">[47]</a>. It is this feature of the two protein alleles which may be functionally relevant. The data therefore suggest the possibility that the ancestral form may be a better Ca<sup>2+</sup> re-absorber than the putatively selected form of <em>TRPV6</em>, i.e the derived allele is more sensitive to Ca<sup>2+</sup> feedback. Nonetheless, the differences we find in biochemical/biophysical function of the <em>TRPV6</em> protein (expressed as homotetramers) between the two forms are not statistically-significant. It is also possible that other aspects of <em>TRPV6</em> could still be affected, such as the putative site of protein-protein interaction in the N-terminus or the site close to a region which is critical for Ca<sup>2+</sup> dependent inactivation <a href="#pone.0001686-Nilius5">[43]</a>. Or, it may be that the amount of <em>TRPV6</em> channel differs between these two forms, because of expression or stability differences <em>in vivo</em>. Despite these extensive electrophysiological experiments, the phenotypic consequences of these different <em>TRPV6</em> haplotypes remain enigmatic.</p>


<h4>Conclusion</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p><em>TRPV6</em> is an epithelial calcium channel which is thought to be the primary protein in the absorption of calcium in the intestine and hence is key to absorbing calcium from the diet <a href="#pone.0001686-Hoenderop2">[44]</a>, <a href="#pone.0001686-Nijenhuis1">[48]</a>. This, and the proposed date of selection of about 10,000 years before present, led to the hypothesis that this locus may have co-evolved with lactase persistence with the onset of dairying in Europeans <a href="#pone.0001686-Akey1">[1]</a>. However, both this work and previous work <a href="#pone.0001686-Akey3">[15]</a> find signatures of selection in all non-African populations examined; there is thus no correlation between dairying and selection at this locus.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>We conclude that the best explanation for the observed patterns of variation at <em>TRPV6</em> in non-African populations is the result of independent, parallel local selection acting on standing genetic variation (a soft sweep), analogous to the parallel evolution recently described in sticklebacks <a href="#pone.0001686-Colosimo1">[49]</a>. The putative haplotype is present in all tested populations of the world, consistent with this haplotype being present in the ancestral population of all humans. The selected allele therefore must have been at a high enough frequency to survive population bottlenecks during human migrations, but also at a low enough frequency so that it did not become associated with multiple haplotypes during the selection phase, thereby reducing or even eliminating the signature of selection <a href="#pone.0001686-Teshima1">[35]</a>, <a href="#pone.0001686-Hermisson1">[50]</a>–<a href="#pone.0001686-Pennings1">[52]</a>.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3"></a><p>Bottlenecks such as those that non-Africans are thought to have experienced will remove haplotypes at low frequency, so that subsequent soft sweeps may start from alleles carried on a single haplotype <a href="#pone.0001686-Teshima1">[35]</a>. While there is good evidence that the migrations of modern humans out of Africa were accompanied by bottlenecks <a href="#pone.0001686-Marth1">[28]</a>, <a href="#pone.0001686-Schaffner1">[30]</a>, <a href="#pone.0001686-Prugnolle1">[53]</a>, the extent to which such bottlenecks would have been sufficient to remove existing multiple haplotypes depends upon the frequency at which the standing genetic variation is present prior to the bottleneck and the magnitude and length of the bottleneck. In the SeattleSNPs data there is evidence suggesting the presence of multiple haplotypes in Europeans at the start of or during the sweep. At the <em>TRPV5</em> locus (which is 22 kb from the <em>TRPV6</em> locus) there are two major haplotypes present in Europeans that differ at 9 sites, all of which are also polymorphic in Africans, and thus by inference were present in the ancestral population (<a href="#pone.0001686.s007">Figure S7</a>). These two distinct haplotypes could be explained either by the presence of multiple haplotypes on the background of the selected allele prior to the onset of selection, or a recombination event which occurred early during the selective phase.</p>
<a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4"></a><p>The timing of the selection event in Europeans and the apparent parallel, independent local selection in Europeans and Asians suggest that independent selective events may have also occurred in the Americas and in highland New Guinea, since both of these regions were colonized well before 7,000 years ago <a href="#pone.0001686-Goebel1">[54]</a>–<a href="#pone.0001686-Hope1">[56]</a>. Moreover, the greater extent of EHH observed in Han Chinese and Japanese suggests that the selective event must have been more recent in time and/or stronger than that of Europeans.</p>
<a id="article1.body1.sec2.sec8.p5" name="article1.body1.sec2.sec8.p5"></a><p>Our analysis of several electrophysiological characteristics of <em>TRPV6</em> demonstrated no significant differences between the two alleles. However, the observed difference in Ca<sup>2+</sup>-dependent inactivation may nonetheless be biologically relevant. If it were, then the derived form of <em>TRPV6</em> would be quicker to “close” in response to Ca<sup>2+</sup> feedback, signifying that it would reabsorb less Ca<sup>2+</sup> than the ancestral form. Nonetheless, it is possible that the ancestral and derived forms differ not in electrophysiological characteristics, but rather via novel protein-protein interactions in the intracellular domains, or differences in the regulation of gene expression. These properties should be explored in the future. We further note that <em>TRPV6</em> not only absorbs calcium from the diet, but also increases cell proliferation of a variety of different cancer cells <a href="#pone.0001686-Schwarz1">[57]</a>–<a href="#pone.0001686-Prevarskaya1">[59]</a>, as well serving an essential role in the differentiation of kerationcytes via a Ca<sup>2+</sup> gradient in skin tissue <a href="#pone.0001686-Lehenkyi2">[60]</a>. This is an intriguing relationship due to the transcriptional regulation of <em>TRPV6</em> by 1,25-dihydroxyvitamin D3 <a href="#pone.0001686-vandeGraaf1">[61]</a>, which is in turn derived from cholesterol via ultraviolet light. Since the availability of light is regulated by the amount of melanin in the skin, and the distribution of melanosomes throughout the skin is regulated by kerationcytes <a href="#pone.0001686-BessouTouya1">[62]</a>, <a href="#pone.0001686-Minwalla1">[63]</a>, there is potentially a complex regulatory circuit involving <em>TRPV6</em>.</p>
<a id="article1.body1.sec2.sec8.p6" name="article1.body1.sec2.sec8.p6"></a><p>A potential selective force that could result in parallel selection in different populations is agriculture. The independent development of agriculture in several different parts of the world within the past 11,000 years <a href="#pone.0001686-Diamond1">[64]</a> including the Middle East (11,000 ybp), East Asia (9,000 ybp), the Americas (5,000–4000 ybp), and the New Guinea highlands (9,000–6,000 ybp), fits remarkably well with the demonstrated parallel selection on <em>TRPV6</em> in the ancestors of Europeans and Asians, and potentially in the Americas and New Guinea as well. It may be that dietary changes related to the switch to agriculture underlie the selective event, or that selection on <em>TRPV6</em> is related to resistance to a particular disease that became important with the increased population densities associated with agriculture.</p>
<a id="article1.body1.sec2.sec8.p7" name="article1.body1.sec2.sec8.p7"></a><p>As a simple test of this hypothesis, we included three hunter-gatherer groups from India in our genotyping of the three tagging SNPs in worldwide populations (Koragas, Mullukurunan, and Mullukurumba; <a href="#pone.0001686.s011">Table S3</a>); however, all three populations were fixed for the derived, putatively-selected haplotype, suggesting that selection for this haplotype also occurred in the ancestors of these groups. While to the best of our knowledge these populations have always been hunter-gatherers, it is possible that they have reverted from an agricultural to a hunting-gathering lifestyle, as has been documented for other hunting-gathering groups <a href="#pone.0001686-Oota1">[65]</a>. Genotyping of additional hunter-gatherer groups would therefore be desirable.</p>
<a id="article1.body1.sec2.sec8.p8" name="article1.body1.sec2.sec8.p8"></a><p>In sum, in the absence of any current demonstrable functional difference between the ancestral and putatively-selected, derived form of <em>TRPV6</em>, the reason(s) for selection on <em>TRPV6</em> remain(s) enigmatic. However we have demonstrated that the most parsimonious explanation for the observed variation in humans at <em>TRPV6</em> is independent parallel soft sweeps occurring multiple times in human history. This is the first record, to our knowledge, of parallel selection acting on standing genetic variation in humans and suggests, as others have observed <a href="#pone.0001686-Voight1">[10]</a>, that this may not be uncommon in recent human adaptive history.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Public Data Resources</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>We downloaded data from the SeattleSNPs Variation Discovery Resource at <a href="http://pga.mbt.washington.edu/on">http://pga.mbt.washington.edu/on</a> April 26, 2005 for 221 genes, of which 200 were sequenced from panel 1 and 21 sequenced from panel 2. The samples from panel 1 include 24 African-Americans from the Human Variation Panel and 23 European-Americans, while the samples from Panel 2 are HapMap samples <a href="#pone.0001686-The2">[25]</a>, consisting of 24 Yoruba from Ibadan, Nigeria, and 23 CEPH European-Americans. Introns, exons, and UTRs were sequenced for each gene in each sample and in one <em>Pan troglodytes</em>. HapMap Phase I data as well as individual genotype data was downloaded in bulk from the International HapMap Project. HapMap data consists of genotype and phased data from four populations: 30 trios from the Yoruba people of Ibadan, Nigeria; 45 unrelated Japanese individuals from the greater Tokyo area; 45 unrelated Han Chinese from Beijing; and 30 trios from Americans of Northern and Western European origin.</p>


<h4>Extended Haplotype Homozygosity</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Extended haplotype homozygosity (EHH) values <a href="#pone.0001686-Sabeti1">[18]</a> were calculated for every allele in each gene in the SeattleSNPs dataset using the inferred haplotypes from Phase v 2.0, obtained from the SeattleSNPs website. Following previous procedures <a href="#pone.0001686-Voight1">[10]</a>, <a href="#pone.0001686-Tang1">[19]</a> we defined every SNP allele as a core and calculated the decay of EHH as the probability of selecting two identical haplotypes, starting with a value of 1 at the core site and decaying to 0 as the distance from the core site increases. To summarize the decay of EHH we calculated the integrated area under the EHH curve (iEHH) for each SNP allele. iEHH values were then used to calculate two relative EHH statistics, Raw (<span class="underline">r</span>elative EHH of an <span class="underline">a</span>llele <span class="underline">w</span>ithin a population) and Rab (<span class="underline">r</span>elative EHH of an <span class="underline">a</span>llele <span class="underline">b</span>etween populations). Raw is a within-population statistic and is defined as the ratio of the iEHH values for the haplotypes defined by the two alleles at each SNP, as proposed previously <a href="#pone.0001686-Sabeti1">[18]</a>. Raw was calculated for each SNP, with the allele with the larger iEHH in the numerator. We introduce here a new between-population statistic, Rab, which is defined as the ratio of the iEHH values for the same core SNP allele in two different populations. Rab was calculated for every allele in the data with the European or non-African iEHH in the numerator. To determine if a locus has a preponderance of large Rab or Raw values, we calculated an empirical distribution of Rab and Raw values based on the other 220 genes in the SeattleSNP data, as well as generating Rab and Raw values for a simulated data set. For the empirical analysis we used the Phase v2.0 <a href="#pone.0001686-Stephens1">[66]</a>, <a href="#pone.0001686-Stephens2">[67]</a> files that were supplied on the SeattleSNPs website, in which all sites with a minor allele frequency below 5% were removed prior to the Phase analysis. Rab and Raw values were calculated for every allele and binned according to the frequency of the European allele, or in the case of Raw according to the allele with the largest iEHH value. Alleles that did not occur at least twice in both populations were removed from the analysis and the 95% and 99% percentiles for each frequency bin were determined. For each gene we then determined the proportion of sites that were above each of these percentiles.</p>


<h4>Neutral Simulations</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>For the comparison to simulated data, we used Hudson's ms program <a href="#pone.0001686-Hudson1">[33]</a> to simulate 10,000 loci with demographic parameters based on a model previously found to provide the best fit to human data <a href="#pone.0001686-Schaffner1">[30]</a>, <a href="#pone.0001686-Hudson1">[33]</a>. This model includes three populations (Africans, Europeans, and Asians), allows for migration between all populations, and includes a bottleneck for both non-African populations and a post-agricultural population expansion for each population. In the simulations used in EHH analyses (Rab and Raw) we chose a theta value for the simulations that yielded a median value of the observed number of segregating sites in <em>TRPV6</em> (146) and a recombination rate of 0.00103/base across the entire sequenced region. The recombination rate used here is the median recombination rate for African Americans, estimated by a previous study of the SeattleSNPs data <a href="#pone.0001686-Ptak1">[68]</a>. In simulations testing neutrality based on the allele frequency spectrum, we used the observed number of segregating sites observed in the population under question.</p>


<h4>Selection Simulations</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>Simulations of a selective sweep were carried out as described above, with the following exceptions: migration between populations was excluded; the sequence length was set to 850kb; and a bottleneck was simulated by setting Ne to 1 for 5 generations starting 50,100 ybp for the ancestral population of Asians and Europeans. Simulations were based on the theta value observed in the African-American sequence data. This creates a drastic bottleneck equivalent to the effect of an instantaneous selective sweep. After five generations Ne was increased to 7700 and 50 generations later the European and Asian populations split and followed subsequent independent demographic histories with no migration. To simulate the ascertainment bias in the empirical data, each simulation included an additional 28 chromosomes, comprised of 4 African, 5 European and 5 Asians, from which a discovery panel was created to identify SNPs which were then used in the subsequent analysis. To determine if the eight empirical <em>TRPV6</em> Rab values are greater than those simulated, we performed two conservative one-sided Mann-Whitney U tests. The first compares all sites within the window in which <em>TRPV6</em> would lie for each simulation to the distribution of observed HapMap Rab values, while the second compares the top eight sites in each simulation from the previous test that also have an allele frequency greater than 0.95. We used this more stringent frequency cutoff because SNPs in the empirical data were also fixed or nearly so. P values were calculated as the proportion of simulations that had a Mann-Whitney U p-value greater than 0.05.</p>


<h4>DNA Amplification for Sequencing and Genotyping</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>25 µl PCR reactions were carried out in MJ Research Thermal Cyclers (MR Research, Waltham, MA, USA). Each PCR consisted of an initial DNA denaturation and <em>Taq</em> activation step at 95° for 15 min followed by 34 repeated cycles of denaturation at 95° for 1 min, an annealing step for 1 min and extension at 72° for 1 min. After amplification there was a final extension step at 72° for 10 min. The reactions included 20 ng of template DNA, 1× PCR buffer (Applied Biosystems, Foster City, CA, USA), 200 µM dNTPs (Amersham Biosciences, Uppsala, Sweden), 400 nM of each primer (Biotez, Berlin, Germany) and 1.25 units of AmpliTaq Gold DNA polymerase (Applied Biosystems, Foster City, CA, USA). All primers were designed with Primer3, and in cases of multiplexing the primers were further analyzed with NetPrimer.</p>


<h4>Additional Sequencing</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>Approximately 4.1 kb of non-consecutive sequence in a 52 kb region around <em>TRPV6</em> (<a href="#pone.0001686.s006">Figure S6</a>) was obtained from four additional populations: 23 Karitiana from Brazil, 23 Pathan from Pakistan, and 23 Han Chinese, all from the HGCP-CEPH Diversity Panel <a href="#pone.0001686-Cann1">[17]</a>; and 23 highland Papua New Guineans <a href="#pone.0001686-Stoneking1">[69]</a>. Regions for sequencing were selected to capture informative polymorphic sites, based on the data from SeattleSNPs. Primers are provided in <a href="#pone.0001686.s013">Table S5</a>. Sequences were aligned, concatenated and phased with Phase v2.0 <a href="#pone.0001686-Stephens1">[66]</a>, <a href="#pone.0001686-Stephens3">[70]</a> and then iEHH and Rab values, with Africans as the comparative population, were calculated for each site in each population. Each PCR product was subsequently sequenced in both directions in a total volume of 10 µl. PCR amplification primers (Supplemental <a href="#pone.0001686.s013">Table 5</a>) were used as sequencing primers in reactions consisting of 1 µl of Applied Biosystems Big Dye Terminator v 1.1, 0.75× sequencing buffer, 320 nM primer (Biotez, Berlin, Germany) and 2ng of amplified template DNA. Sequencing was carried out in an ABI 3730 DNA Analyzer. Sequence data was analyzed manually with Applied Biosystems SeqScape Software V2.5 and reference sequences (Applied Biosystems, Foster City, CA, USA).</p>


<h4>HapMap Analysis</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>We defined the genomic location for 19386 RefSeq <a href="#pone.0001686-Pruitt1">[71]</a> genes from Human Genome Build release 34 and used HapMap phase 1 release data to calculate EHH and iEHH values for each SNP allele within the boundaries of each gene, allowing each allele to decay to an EHH value of at least 0.05, before obtaining the Raw and Rab statistics. We calculated the Rab statistic for each non-African population vs. Africans, as well as for Han Chinese vs. Europeans and Japanese vs. Europeans. As with the SeattleSNPs data, we binned each Rab value by the frequency of the allele in the population in the numerator, calculated the 95% percentile of each bin, and determined the proportion of alleles found above the 95% percentile for each gene. The p-values were determined by comparing the proportion of significant alleles within the gene of interest vs. all other genes in the HapMap Phase I data set. Only genes with at least four Rab or four Raw values were considered, to avoid spurious results caused by too few SNPs.</p>


<h4>Construction of mammalian expression vectors</h4>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1"></a><p>The open reading frame of the human <em>TRPV6</em> ortholog was cloned from a human European placenta and the cDNA sequence verified by sequencing. The ancestral form (157R, 378V, 681T) was generated from the derived form by overlap PCR. Both the derived (CMM) and the ancestral (RVT) forms were ligated into a customized vector (pcDNA3-CFP) that includes the cyan fluorescent protein (CFP) coding sequence 3′ to the multiple cloning sites, to obtain C-terminally CFP-tagged proteins. This construct revealed similar currents as obtained previously with a bicistronic expression vector, pCINeo/IRES-GFP/h<em>TRPV6</em>, which was used to co-express human <em>TRPV6</em> and enhanced green fluorescent protein (GFP) <a href="#pone.0001686-Muller1">[72]</a>.</p>


<h4>Cell culture and transfection</h4>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1"></a><p>Human embryonic kidney cells (HEK293) were grown in DMEM containing 10% (v/v) human serum, 2 mM L-glutamine, 2 U/ml penicillin and 2 mg/ml streptomycin at 37°C in a humidity-controlled incubator with 10% CO<sub>2</sub>. HEK293 cells were transiently transfected with the vector constructs. Transfected cells were visually identified in the patch clamp apparatus by their CFP fluorescence.</p>


<h4>Electrophysiology</h4>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1"></a><p>Electrophysiological methods used here were described previously in detail <a href="#pone.0001686-Nilius1">[38]</a>. Whole-cell currents were measured with an EPC-9 (HEKA Elektronik, Lambrecht, Germany, sampling rate 0.2 ms, 8-Pole Bessel filter 10 kHz) or an L/M-EPC-7 (List Elektronics, Darmstadt, Germany) using ruptured patches. Electrode resistances were between 2 and 5 MΩ, capacitance and access resistances were monitored continuously, and series resistance was maximally compensated. The ramp protocol consisted of linear voltage ramps changing from −150 mV to +100 mV within 400 ms, applied every 5 s from a holding potential of 0 mV. The step protocol consisted of a series of 400 ms long voltage steps applied from a holding potential of +70 mV to −100 mV. Current densities, expressed per unit membrane capacitance, were measured from the current at −100 mV during the ramp protocols.</p>


<h4>Solutions and experimental procedures</h4>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1"></a><p>The standard extracellular solution (“Krebs”) contained (in mM) 150 NaCl, 6 CsCl, 1.5 CaCl<sub>2</sub>, 1 MgCl<sub>2</sub>, 10 HEPES and 10 glucose, pH 7.4 with CsOH. Nominally free Ca<sup>2+</sup> solutions were buffered by 5 mM EGTA at a free [Ca<sup>2+</sup>] below 1 nM, as calculated by the CaBuf program (<a href="http://ftp://ftp.cc.kuleuven.be/pub/droogmans/cabuf.zip">ftp://ftp.cc.kuleuven.be/pub/droogmans/c​abuf.zip</a>). The standard internal (pipette) solution contained (in mM) 40 CsCl, 100 Cs-aspartate, 1 MgCl<sub>2</sub>, 10 BAPTA, 4 Na<sub>2</sub>ATP, 10 HEPES, pH 7.2 with CsOH. Cells were kept in a nominally Ca<sup>2+</sup>-free medium to prevent Ca<sup>2+</sup> overload and exposed for maximal 5 min to a Krebs solution containing 1.5 mM Ca<sup>2+</sup> before sealing the patch pipette to the cell. If Ca<sup>2+</sup> (Ba<sup>2+</sup>) was used as the only charge carrier for inward currents, 30mM CaCl<sub>2</sub> (BaCl<sub>2</sub>) was used in the pipette; all other cations were substituted by NMDG-Cl. All experiments were performed at room temperature (20–22°C).</p>


<h4>Statistical analysis</h4>
<a id="article1.body1.sec3.sec12.p1" name="article1.body1.sec3.sec12.p1"></a><p>In all protein function experiments the data are expressed as mean±SEM. Overall statistical significance was determined by analysis of variance (ANOVA, significance level p&lt;0.05).</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001686"></div><a name="pone.0001686.s001" id="pone.0001686.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">A scatter plot of molecular Fst and lnRHap values for each of the 221 genes from the SeattleSNPs data set. Genes from the candidate region of chromosome 7q34, including the TRPV6 locus, are identified in the plot.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.84 MB TIF)</p>
<a name="pone.0001686.s002" id="pone.0001686.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Visual haplotype graph and network of the SeattleSNPs sequence data for the TRPV6 locus. A) A visual haplotype graph of the TRPV6 locus. Each horizontal line is an individual haplotype from a given individual, and each vertical column is a SNP marked by the position of that SNP in the sequence. The major alleles are in blue, while minor alleles are in yellow. B) A MJ network for the European and African SeattleSNP sequence data, with a box around the star-like portion of the network. Branch lengths for the haplotype nodes inside the box were artificially lengthened for illustration purposes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.08 MB TIF)</p>
<a name="pone.0001686.s003" id="pone.0001686.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">A density plot of the time since fixation for the putative allele in Europeans, with estimations are based on twelve and ten segregating sites. The 2.5% and 97.5% credible intervals are plotted for both time estimations. The average mode values is ~7,000 ybp.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(1.12 MB TIF)</p>
<a name="pone.0001686.s004" id="pone.0001686.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">A topology graph of the TRPV6 protein illustrating the location of the three non-synonymous mutations, C157R, M378R and M681T and their relationship with putative important functional areas.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.42 MB TIF)</p>
<a name="pone.0001686.s005" id="pone.0001686.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Haplotype frequencies of three non-synonymous polymorphisms used as tagging SNPs (rs4987657 (C157R), rs4987667 (M378V) and rs4987682 (M681T)) and genotyped in the CEPH Human Diversity panel and additionally in 31 Ethiopians, 51 Germans, 11 South African Bantu-speakers, and three hunter-gatherer groups from India (44 Koragas, 16 Mullukurunan, and 2 Mullukurumba). The key for inferred haplotypes is in the bottom right corner, with CGC corresponding to the ancestral haplotype and TAT to the derived haplotype.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(4.14 MB TIF)</p>
<a name="pone.0001686.s006" id="pone.0001686.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">An overview of the genomic region encompassing the TRPV6/TRPV5 region used to sequence eight blocks in the Karitiana, Han Chinese, highland Papua New Guineans, and the Pathan, previously sequenced in Europeans and African-Americans. The illustration is from the UCSC Genome Browser (chr7:142,278,427-142,330,351) and with the inclusion of a custom track.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.83 MB TIF)</p>
<a name="pone.0001686.s007" id="pone.0001686.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s007">Figure S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Visual haplotype graph of the TRPV5 locus from the SeattleSNPs data set. Each horizontal line is a haplotype and each vertical column is a SNP marked by its position in the TRPV5 sequence. Major alleles are marked in blue, and minor alleles are in yellow. All genotypes were used to infer the haplotypes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(3.64 MB TIF)</p>
<a name="pone.0001686.s008" id="pone.0001686.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s008">Text S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1"></a><p class="preSiDOI">Supplementary Results and Discussion, and Materials and Methods</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s008</p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.11 MB PDF)</p>
<a name="pone.0001686.s009" id="pone.0001686.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s009">Table S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1"></a><p class="preSiDOI">SeattleSNPs Genome Scan Statistics</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s009</p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.04 MB DOC)</p>
<a name="pone.0001686.s010" id="pone.0001686.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s010">Table S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1"></a><p class="preSiDOI">Summary Statistics for EPHB6, TRPV6, TRPV5 and KEL</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s010</p><a id="article1.body1.sec4.supplementary-material10.caption1.p2" name="article1.body1.sec4.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.05 MB DOC)</p>
<a name="pone.0001686.s011" id="pone.0001686.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s011">Table S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material11.caption1.p1" name="article1.body1.sec4.supplementary-material11.caption1.p1"></a><p class="preSiDOI">World Haplotype Frequencies for C157R, M378V and M681T</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s011</p><a id="article1.body1.sec4.supplementary-material11.caption1.p2" name="article1.body1.sec4.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001686.s012" id="pone.0001686.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s012">Table S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material12.caption1.p1" name="article1.body1.sec4.supplementary-material12.caption1.p1"></a><p class="preSiDOI">PCR primers for MALDI-TOF assay</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s012</p><a id="article1.body1.sec4.supplementary-material12.caption1.p2" name="article1.body1.sec4.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
<a name="pone.0001686.s013" id="pone.0001686.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001686.s013">Table S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material13.caption1.p1" name="article1.body1.sec4.supplementary-material13.caption1.p1"></a><p class="preSiDOI">Table of PCR and sequencing primers</p>
<p class="siDoi">doi:10.1371/journal.pone.0001686.s013</p><a id="article1.body1.sec4.supplementary-material13.caption1.p2" name="article1.body1.sec4.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(0.03 MB DOC)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We are indebted to Bjoern Woltering, Mehmet Somel, Susan Ptak, Naim Matasci, Ines Hellmann, Wolfgang Enard, Sean Myles, and Michael Hofreiter for useful discussion of the work and review of the manuscript.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MS BN DH RS TS JP. Performed the experiments: KT DH RS JP. Analyzed the data: BN DH JP. Contributed reagents/materials/analysis tools: MS KT BN DH RS TS JP. Wrote the paper: MS BN DH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001686-Akey1" id="pone.0001686-Akey1"></a>Akey JM, Eberle MA, Rieder MJ, Carlson CS, Shriver MD, et al.  (2004) Population History and Natural Selection Shape Patterns of Genetic Variation in 132 Genes. PLoS Biol  2: E286.  <ul class="find" data-citedArticleID="1063760" data-doi="10.1371/journal.pbio.0020286"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0020286" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+History+and+Natural+Selection+Shape+Patterns+of+Genetic+Variation+in+132+Genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+History+and+Natural+Selection+Shape+Patterns+of+Genetic+Variation+in+132+Genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001686-Akey2" id="pone.0001686-Akey2"></a>Akey JM, Zhang G, Zhang K, Jin L, Shriver MD (2002) Interrogating a high-density SNP map for signatures of natural selection. Genome Res  12: 1805–1814.  <ul class="find" data-citedArticleID="1063762" data-doi="10.1101/gr.631202"><li><a href="http://dx.doi.org/10.1101/gr.631202" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interrogating+a+high-density+SNP+map+for+signatures+of+natural+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interrogating+a+high-density+SNP+map+for+signatures+of+natural+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001686-Bustamante1" id="pone.0001686-Bustamante1"></a>Bustamante CD, Fledel-Alon A, Williamson S, Nielsen R, Todd Hubisz M, et al.  (2005) Natural selection on protein-coding genes in the human genome. Nature  437: 1153.  <ul class="find" data-citedArticleID="1063770" data-doi="10.1038/nature04240"><li><a href="http://dx.doi.org/10.1038/nature04240" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Natural+selection+on+protein-coding+genes+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Natural+selection+on+protein-coding+genes+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001686-Carlson1" id="pone.0001686-Carlson1"></a>Carlson CS, Thomas DJ, Eberle MA, Swanson JE, Livingston RJ, et al.  (2005) Genomic regions exhibiting positive selection identified from dense genotype data. Genome Res  15: 1553–1565.  <ul class="find" data-citedArticleID="1063774" data-doi="10.1101/gr.4326505"><li><a href="http://dx.doi.org/10.1101/gr.4326505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomic+regions+exhibiting+positive+selection+identified+from+dense+genotype+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomic+regions+exhibiting+positive+selection+identified+from+dense+genotype+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001686-Clark1" id="pone.0001686-Clark1"></a>Clark AG, Glanowski S, Nielsen R, Thomas PD, Kejariwal A, et al.  (2003) Inferring Nonneutral Evolution from Human-Chimp-Mouse Orthologous Gene Trios. Science  302: 1960–1963.  <ul class="find" data-citedArticleID="1063776" data-doi="10.1126/science.1088821"><li><a href="http://dx.doi.org/10.1126/science.1088821" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+Nonneutral+Evolution+from+Human-Chimp-Mouse+Orthologous+Gene+Trios." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+Nonneutral+Evolution+from+Human-Chimp-Mouse+Orthologous+Gene+Trios.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001686-Kayser1" id="pone.0001686-Kayser1"></a>Kayser M, Brauer S, Stoneking M (2003) A genome scan to detect candidate regions influenced by local natural selection in human populations. Mol Biol Evol  20: 893–900.  <ul class="find" data-citedArticleID="1063804" data-doi="10.1093/molbev/msg092"><li><a href="http://dx.doi.org/10.1093/molbev/msg092" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+genome+scan+to+detect+candidate+regions+influenced+by+local+natural+selection+in+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+genome+scan+to+detect+candidate+regions+influenced+by+local+natural+selection+in+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001686-Nielsen1" id="pone.0001686-Nielsen1"></a>Nielsen R, Bustamante C, Clark AG, Glanowski S, Sackton TB, et al.  (2005) A scan for positively selected genes in the genomes of humans and chimpanzees. PLoS Biol  3: e170.  <ul class="find" data-citedArticleID="1063816" data-doi="10.1371/journal.pbio.0030170"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030170" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+scan+for+positively+selected+genes+in+the+genomes+of+humans+and+chimpanzees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+scan+for+positively+selected+genes+in+the+genomes+of+humans+and+chimpanzees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001686-Stajich1" id="pone.0001686-Stajich1"></a>Stajich JE, Hahn MW (2005) Disentangling the effects of demography and selection in human history. Mol Biol Evol  22: 63–73.  <ul class="find" data-citedArticleID="1063860" data-doi="10.1093/molbev/msh252"><li><a href="http://dx.doi.org/10.1093/molbev/msh252" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Disentangling+the+effects+of+demography+and+selection+in+human+history." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Disentangling+the+effects+of+demography+and+selection+in+human+history.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001686-Storz1" id="pone.0001686-Storz1"></a>Storz JF, Payseur BA, Nachman MW (2004) Genome scans of DNA variability in humans reveal evidence for selective sweeps outside of Africa. Mol Biol Evol  21: 1800–1811.  <ul class="find" data-citedArticleID="1063870" data-doi="10.1093/molbev/msh192"><li><a href="http://dx.doi.org/10.1093/molbev/msh192" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+scans+of+DNA+variability+in+humans+reveal+evidence+for+selective+sweeps+outside+of+Africa." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+scans+of+DNA+variability+in+humans+reveal+evidence+for+selective+sweeps+outside+of+Africa.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001686-Voight1" id="pone.0001686-Voight1"></a>Voight BF, Kudaravalli S, Wen X, Pritchard JK (2006) A map of recent positive selection in the human genome. PLoS Biol  4: e72.  <ul class="find" data-citedArticleID="1063890" data-doi="10.1371/journal.pbio.0040072"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+map+of+recent+positive+selection+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+map+of+recent+positive+selection+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001686-Walsh1" id="pone.0001686-Walsh1"></a>Walsh EC, Sabeti P, Hutcheson HB, Fry B, Schaffner SF, et al.  (2006) Searching for signals of evolutionary selection in 168 genes related to immune function. Hum Genet  119: 92–102.  <ul class="find" data-citedArticleID="1063892" data-doi="10.1007/s00439-005-0090-0"><li><a href="http://dx.doi.org/10.1007/s00439-005-0090-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Searching+for+signals+of+evolutionary+selection+in+168+genes+related+to+immune+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Searching+for+signals+of+evolutionary+selection+in+168+genes+related+to+immune+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001686-Hoenderop1" id="pone.0001686-Hoenderop1"></a>Hoenderop JGJ, Vennekens R, Müller D, Prenen J, Droogmans G, et al.  (2001) Function and expression of the epithelial Ca<sup>2+</sup> channel family: comparison of the epithelial Ca<sup>2+</sup> channel 1 and 2. J Physiol (Lond)  537: 747–761.  <ul class="find" data-citedArticleID="1063794" data-doi="10.1113/jphysiol.2001.012917"><li><a href="http://dx.doi.org/10.1113/jphysiol.2001.012917" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Function+and+expression+of+the+epithelial+Ca2%2B+channel+family%3A+comparison+of+the+epithelial+Ca2%2B+channel+1+and+2." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Function+and+expression+of+the+epithelial+Ca2%2B+channel+family%3A+comparison+of+the+epithelial+Ca2%2B+channel+1+and+2.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001686-Bersaglieri1" id="pone.0001686-Bersaglieri1"></a>Bersaglieri T, Sabeti PC, Patterson N, Vanderploeg T, Schaffner SF, et al.  (2004) Genetic signatures of strong recent positive selection at the lactase gene. Am J Hum Genet  74: 1111–1120.  <ul class="find" data-citedArticleID="1063766" data-doi="10.1086/421051"><li><a href="http://dx.doi.org/10.1086/421051" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+signatures+of+strong+recent+positive+selection+at+the+lactase+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+signatures+of+strong+recent+positive+selection+at+the+lactase+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001686-Thomson1" id="pone.0001686-Thomson1"></a>Thomson G (1977) The effect of a selected locus on linked neutral loci. Genetics  85: 753–788.  <ul class="find" data-citedArticleID="1063880"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+effect+of+a+selected+locus+on+linked+neutral+loci.&amp;auth=&amp;atitle=The+effect+of+a+selected+locus+on+linked+neutral+loci." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+effect+of+a+selected+locus+on+linked+neutral+loci." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+effect+of+a+selected+locus+on+linked+neutral+loci.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001686-Akey3" id="pone.0001686-Akey3"></a>Akey JM, Swanson WJ, Madeoy J, Eberle M, Shriver MD (2006) TRPV6 exhibits unusual patterns of polymorphism and divergence in worldwide populations. Hum Mol Genet  15: 2106–2113.  <ul class="find" data-citedArticleID="1063764" data-doi="10.1093/hmg/ddl134"><li><a href="http://dx.doi.org/10.1093/hmg/ddl134" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRPV6+exhibits+unusual+patterns+of+polymorphism+and+divergence+in+worldwide+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRPV6+exhibits+unusual+patterns+of+polymorphism+and+divergence+in+worldwide+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001686-Yang1" id="pone.0001686-Yang1"></a>Yang Z, Nielsen R (2002) Codon-substitution models for detecting molecular adaptation at individual sites along specific lineages. Mol Biol Evol  19: 908–917.  <ul class="find" data-citedArticleID="1063896" data-doi="10.1093/oxfordjournals.molbev.a004148"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004148" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Codon-substitution+models+for+detecting+molecular+adaptation+at+individual+sites+along+specific+lineages.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001686-Cann1" id="pone.0001686-Cann1"></a>Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, et al.  (2002) A human genome diversity cell line panel. Science  296: 261–262.  <ul class="find" data-citedArticleID="1063772" data-doi="10.1126/science.296.5566.261b"><li><a href="http://dx.doi.org/10.1126/science.296.5566.261b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+human+genome+diversity+cell+line+panel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+human+genome+diversity+cell+line+panel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001686-Sabeti1" id="pone.0001686-Sabeti1"></a>Sabeti PC, Reich DE, Higgins JM, Levine HZ, Richter DJ, et al.  (2002) Detecting recent positive selection in the human genome from haplotype structure. Nature  419: 832–837.  <ul class="find" data-citedArticleID="1063852" data-doi="10.1038/nature01140"><li><a href="http://dx.doi.org/10.1038/nature01140" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+recent+positive+selection+in+the+human+genome+from+haplotype+structure." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+recent+positive+selection+in+the+human+genome+from+haplotype+structure.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001686-Tang1" id="pone.0001686-Tang1"></a>Tang K, Thornton KR, Stoneking M (2007) A New Approach for Using Genome Scans to Detect Recent Positive Selection in the Human Genome. PLoS Biol  5: e171.  <ul class="find" data-citedArticleID="1063872" data-doi="10.1371/journal.pbio.0050171"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0050171" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+New+Approach+for+Using+Genome+Scans+to+Detect+Recent+Positive+Selection+in+the+Human+Genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+New+Approach+for+Using+Genome+Scans+to+Detect+Recent+Positive+Selection+in+the+Human+Genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001686-Fry1" id="pone.0001686-Fry1"></a>Fry AE, Trafford CJ, Kimber MA, Chan M, Rockett KA, et al.  (2006) Haplotype homozygosity and derived alleles in the human genome. Am J Hum Genet  78:   <ul class="find" data-citedArticleID="1063784" data-doi="10.1086/504160"><li><a href="http://dx.doi.org/10.1086/504160" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haplotype+homozygosity+and+derived+alleles+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haplotype+homozygosity+and+derived+alleles+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001686-Sabeti2" id="pone.0001686-Sabeti2"></a>Sabeti PC, Schaffner SF, Fry B, Lohmueller J, Varilly P, et al.  (2006) Positive natural selection in the human lineage. Science  312: 1614–1620.  <ul class="find" data-citedArticleID="1063854" data-doi="10.1126/science.1124309"><li><a href="http://dx.doi.org/10.1126/science.1124309" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Positive+natural+selection+in+the+human+lineage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Positive+natural+selection+in+the+human+lineage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001686-Wang1" id="pone.0001686-Wang1"></a>Wang Y, Zhao LP, Dudoit S (2006) A Fine-Scale Linkage-Disequilibrium Measure Based on Length of Haplotype Sharing. Am J Hum Genet  78: 615–628.  <ul class="find" data-citedArticleID="1063894" data-doi="10.1086/502632"><li><a href="http://dx.doi.org/10.1086/502632" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Fine-Scale+Linkage-Disequilibrium+Measure+Based+on+Length+of+Haplotype+Sharing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Fine-Scale+Linkage-Disequilibrium+Measure+Based+on+Length+of+Haplotype+Sharing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001686-The1" id="pone.0001686-The1"></a>The International HapMap C (2005) A haplotype map of the human genome. Nature  437: 1299.  <ul class="find" data-citedArticleID="1063876" data-doi="10.1038/nature04226"><li><a href="http://dx.doi.org/10.1038/nature04226" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+haplotype+map+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+haplotype+map+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001686-Reich1" id="pone.0001686-Reich1"></a>Reich DE, Cargill M, Bolk S, Ireland J, Sabeti PC, et al.  (2001) Linkage disequilibrium in the human genome. Nature  411: 199.  <ul class="find" data-citedArticleID="1063850" data-doi="10.1038/35075590"><li><a href="http://dx.doi.org/10.1038/35075590" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Linkage+disequilibrium+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Linkage+disequilibrium+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001686-The2" id="pone.0001686-The2"></a>The International HapMap Consortium (2003) The International HapMap Project. Nature  426: 789.  <ul class="find" data-citedArticleID="1063878" data-doi="10.1038/nrg1351"><li><a href="http://dx.doi.org/10.1038/nrg1351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+International+HapMap+Project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+International+HapMap+Project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001686-Harpending1" id="pone.0001686-Harpending1"></a>Harpending HC, Batzer MA, Gurven M, Jorde LB, Rogers AR, et al.  (1998) Genetic traces of ancient demography. PNAS  95: 1961–1967.  <ul class="find" data-citedArticleID="1063788" data-doi="10.1073/pnas.95.4.1961"><li><a href="http://dx.doi.org/10.1073/pnas.95.4.1961" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genetic+traces+of+ancient+demography." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genetic+traces+of+ancient+demography.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001686-Jorde1" id="pone.0001686-Jorde1"></a>Jorde LB, Bamshad M, Rogers AR (1998) Using mitochondrial and nuclear DNA markers to reconstruct human evolution. Bioessays  20: 126–136.  <ul class="find" data-citedArticleID="1063802" data-doi="10.1002/(sici)1521-1878(199802)20:2<126::aid-bies5&gt;3.0.co;2-r"><li><a href="http://dx.doi.org/10.1002/(sici)1521-1878(199802)20:2<126::aid-bies5&gt;3.0.co;2-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+mitochondrial+and+nuclear+DNA+markers+to+reconstruct+human+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+mitochondrial+and+nuclear+DNA+markers+to+reconstruct+human+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001686-Marth1" id="pone.0001686-Marth1"></a>Marth GT, Czabarka E, Murvai J, Sherry ST (2004) The allele frequency spectrum in genome-wide human variation data reveals signals of differential demographic history in three large world populations. Genetics  166: 351–372.  <ul class="find" data-citedArticleID="1063810" data-doi="10.1534/genetics.166.1.351"><li><a href="http://dx.doi.org/10.1534/genetics.166.1.351" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+allele+frequency+spectrum+in+genome-wide+human+variation+data+reveals+signals+of+differential+demographic+history+in+three+large+world+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+allele+frequency+spectrum+in+genome-wide+human+variation+data+reveals+signals+of+differential+demographic+history+in+three+large+world+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001686-Ray1" id="pone.0001686-Ray1"></a>Ray N, Currat M, Berthier P, Excoffier L (2005) Recovering the geographic origin of early modern humans by realistic and spatially explicit simulations. Genome Res  15: 1161–1167.  <ul class="find" data-citedArticleID="1063848" data-doi="10.1101/gr.3708505"><li><a href="http://dx.doi.org/10.1101/gr.3708505" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recovering+the+geographic+origin+of+early+modern+humans+by+realistic+and+spatially+explicit+simulations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recovering+the+geographic+origin+of+early+modern+humans+by+realistic+and+spatially+explicit+simulations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001686-Schaffner1" id="pone.0001686-Schaffner1"></a>Schaffner SF, Foo C, Gabriel S, Reich D, Daly MJ, et al.  (2005) Calibrating a coalescent simulation of human genome sequence variation. Genome Res  15: 1576–1583.  <ul class="find" data-citedArticleID="1063856" data-doi="10.1101/gr.3709305"><li><a href="http://dx.doi.org/10.1101/gr.3709305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Calibrating+a+coalescent+simulation+of+human+genome+sequence+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Calibrating+a+coalescent+simulation+of+human+genome+sequence+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001686-Zhivotovsky1" id="pone.0001686-Zhivotovsky1"></a>Zhivotovsky LA, Bennett L, Bowcock AM, Feldman MW (2000) Human Population Expansion and Microsatellite Variation. Mol Biol Evol  17: 757–767.  <ul class="find" data-citedArticleID="1063898" data-doi="10.1093/oxfordjournals.molbev.a026354"><li><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026354" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Human+Population+Expansion+and+Microsatellite+Variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Human+Population+Expansion+and+Microsatellite+Variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001686-Zhivotovsky2" id="pone.0001686-Zhivotovsky2"></a>Zhivotovsky LA, Rosenberg NA, Feldman MW (2003) Features of evolution and expansion of modern humans, inferred from genomewide microsatellite markers. Am J Hum Genet  72: 1171–1186.  <ul class="find" data-citedArticleID="1063900" data-doi="10.1086/375120"><li><a href="http://dx.doi.org/10.1086/375120" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Features+of+evolution+and+expansion+of+modern+humans%2C+inferred+from+genomewide+microsatellite+markers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Features+of+evolution+and+expansion+of+modern+humans%2C+inferred+from+genomewide+microsatellite+markers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001686-Hudson1" id="pone.0001686-Hudson1"></a>Hudson RR (2002) Generating samples under a Wright-Fisher neutral model of genetic variation. Bioinformatics  18: 337–338.  <ul class="find" data-citedArticleID="1063800" data-doi="10.1093/bioinformatics/18.2.337"><li><a href="http://dx.doi.org/10.1093/bioinformatics/18.2.337" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generating+samples+under+a+Wright-Fisher+neutral+model+of+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generating+samples+under+a+Wright-Fisher+neutral+model+of+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001686-Przeworski1" id="pone.0001686-Przeworski1"></a>Przeworski M, Coop G, Wall JD (2005) The signature of positive selection on standing genetic variation. Evolution Int J Org Evolution  59: 2312–2323.  <ul class="find" data-citedArticleID="1063844" data-doi="10.1554/05-273.1"><li><a href="http://dx.doi.org/10.1554/05-273.1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+signature+of+positive+selection+on+standing+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+signature+of+positive+selection+on+standing+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001686-Teshima1" id="pone.0001686-Teshima1"></a>Teshima KM, Coop G, Przeworski M (2006) How reliable are empirical genomic scans for selective sweeps? Genome Res  16: 702–712.  <ul class="find" data-citedArticleID="1063874" data-doi="10.1101/gr.5105206"><li><a href="http://dx.doi.org/10.1101/gr.5105206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=How+reliable+are+empirical+genomic+scans+for+selective+sweeps%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22How+reliable+are+empirical+genomic+scans+for+selective+sweeps%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001686-Voets1" id="pone.0001686-Voets1"></a>Voets T, Janssens A, Prenen J, Droogmans D, Nilius G (2003) Mg<sup>2+</sup>-dependent gating and strong inward rectification of the cation channel TRPV6. J Gen Physiol  121: 245–260.  <ul class="find" data-citedArticleID="1063886" data-doi="10.1085/jgp.20028752"><li><a href="http://dx.doi.org/10.1085/jgp.20028752" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mg2%2B-dependent+gating+and+strong+inward+rectification+of+the+cation+channel+TRPV6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mg2%2B-dependent+gating+and+strong+inward+rectification+of+the+cation+channel+TRPV6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001686-Vennekens1" id="pone.0001686-Vennekens1"></a>Vennekens R, Hoenderop JG, Prenen J, Stuiver M, Willems PH, et al.  (2000) Permeation and gating properties of the novel epithelial Ca<sup>2+</sup> channel. J Biol Chem  275: 3963–3969.  <ul class="find" data-citedArticleID="1063882" data-doi="10.1074/jbc.275.6.3963"><li><a href="http://dx.doi.org/10.1074/jbc.275.6.3963" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Permeation+and+gating+properties+of+the+novel+epithelial+Ca2%2B+channel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Permeation+and+gating+properties+of+the+novel+epithelial+Ca2%2B+channel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001686-Nilius1" id="pone.0001686-Nilius1"></a>Nilius B, Vennekens R, Prenen J, Hoenderop JG, Bindels RJ, et al.  (2000) Whole-cell and single channel monovalent cation currents through the novel rabbit epithelial Ca<sup>2+</sup> channel ECaC. J Physiol Lond  527: 239–248.  <ul class="find" data-citedArticleID="1063822" data-doi="10.1111/j.1469-7793.2000.00239.x"><li><a href="http://dx.doi.org/10.1111/j.1469-7793.2000.00239.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Whole-cell+and+single+channel+monovalent+cation+currents+through+the+novel+rabbit+epithelial+Ca2%2B+channel+ECaC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Whole-cell+and+single+channel+monovalent+cation+currents+through+the+novel+rabbit+epithelial+Ca2%2B+channel+ECaC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001686-Nilius2" id="pone.0001686-Nilius2"></a>Nilius B, Prenen J, Vennekens R, Hoenderop JG, Bindels RJ, et al.  (2001) Modulation of the epithelial calcium channel, ECaC, by intracellular Ca<sup>2+</sup>. Cell Calcium  29: 417–428.  <ul class="find" data-citedArticleID="1063824" data-doi="10.1054/ceca.2001.0201"><li><a href="http://dx.doi.org/10.1054/ceca.2001.0201" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modulation+of+the+epithelial+calcium+channel%2C+ECaC%2C+by+intracellular+Ca2%2B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modulation+of+the+epithelial+calcium+channel%2C+ECaC%2C+by+intracellular+Ca2%2B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001686-Voets2" id="pone.0001686-Voets2"></a>Voets T, Janssens A, Droogmans G, Nilius B (2003) A single pore residue determines calcium selectivity, inward rectification and pore size of TRPV6 (CaT1/ECaC2). Biophys J (Annual Meeting Abstracts) 2724-Pos.   <ul class="find" data-citedArticleID="1063888"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+single+pore+residue+determines+calcium+selectivity%2C+inward+rectification+and+pore+size+of+TRPV6+%28CaT1%2FECaC2%29.&amp;auth=&amp;atitle=A+single+pore+residue+determines+calcium+selectivity%2C+inward+rectification+and+pore+size+of+TRPV6+%28CaT1%2FECaC2%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+single+pore+residue+determines+calcium+selectivity%2C+inward+rectification+and+pore+size+of+TRPV6+%28CaT1%2FECaC2%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+single+pore+residue+determines+calcium+selectivity%2C+inward+rectification+and+pore+size+of+TRPV6+%28CaT1%2FECaC2%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001686-Nilius3" id="pone.0001686-Nilius3"></a>Nilius B, Vennekens R, Prenen J, Hoenderop JG, Droogmans G, et al.  (2001) The single pore residue Asp542 determines Ca<sup>2+</sup> permeation and Mg<sup>2+</sup> block of the epithelial Ca<sup>2+</sup> channel. J Biol Chem  276: 1020–1025.  <ul class="find" data-citedArticleID="1063826" data-doi="10.1074/jbc.m006184200"><li><a href="http://dx.doi.org/10.1074/jbc.m006184200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+single+pore+residue+Asp542+determines+Ca2%2B+permeation+and+Mg2%2B+block+of+the+epithelial+Ca2%2B+channel." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+single+pore+residue+Asp542+determines+Ca2%2B+permeation+and+Mg2%2B+block+of+the+epithelial+Ca2%2B+channel.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001686-Nilius4" id="pone.0001686-Nilius4"></a>Nilius B, Prenen J, Hoenderop JG, Vennekens R, Hoefs S, et al.  (2002) Fast and slow inactivation kinetics of the Ca2+ channels ECaC1 and ECaC2 (TRPV5 and TRPV6). Role of the intracellular loop located between transmembrane segments 2 and 3. J Biol Chem  277: 30852–30858.  <ul class="find" data-citedArticleID="1063828" data-doi="10.1074/jbc.m202418200"><li><a href="http://dx.doi.org/10.1074/jbc.m202418200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+and+slow+inactivation+kinetics+of+the+Ca2%2B+channels+ECaC1+and+ECaC2+%28TRPV5+and+TRPV6%29.+Role+of+the+intracellular+loop+located+between+transmembrane+segments+2+and+3." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+and+slow+inactivation+kinetics+of+the+Ca2%2B+channels+ECaC1+and+ECaC2+%28TRPV5+and+TRPV6%29.+Role+of+the+intracellular+loop+located+between+transmembrane+segments+2+and+3.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001686-Nilius5" id="pone.0001686-Nilius5"></a>Nilius B, Weidema F, Prenen J, Hoenderop JG, Vennekens R, et al.  (2003) The carboxyl terminus of the epithelial Ca(2+) channel ECaC1 is involved in Ca(2+)-dependent inactivation. Pflugers Arch  445: 584–588.  <ul class="find" data-citedArticleID="1063830"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+carboxyl+terminus+of+the+epithelial+Ca%282%2B%29+channel+ECaC1+is+involved+in+Ca%282%2B%29-dependent+inactivation.&amp;auth=&amp;atitle=The+carboxyl+terminus+of+the+epithelial+Ca%282%2B%29+channel+ECaC1+is+involved+in+Ca%282%2B%29-dependent+inactivation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+carboxyl+terminus+of+the+epithelial+Ca%282%2B%29+channel+ECaC1+is+involved+in+Ca%282%2B%29-dependent+inactivation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+carboxyl+terminus+of+the+epithelial+Ca%282%2B%29+channel+ECaC1+is+involved+in+Ca%282%2B%29-dependent+inactivation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001686-Hoenderop2" id="pone.0001686-Hoenderop2"></a>Hoenderop JG, Nilius B, Bindels RJ (2005) Calcium absorption across epithelia. Physiol Rev  85: 373–422.  <ul class="find" data-citedArticleID="1063796" data-doi="10.1152/physrev.00003.2004"><li><a href="http://dx.doi.org/10.1152/physrev.00003.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Calcium+absorption+across+epithelia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Calcium+absorption+across+epithelia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001686-Vennekens2" id="pone.0001686-Vennekens2"></a>Vennekens R, Prenen J, Hoenderop JG, Bindels RJ, Droogmans G, et al.  (2001) Pore properties and ionic block of the rabbit epithelial calcium channel expressed in HEK 293 cells. J Physiol  530: 183–191.  <ul class="find" data-citedArticleID="1063884" data-doi="10.1111/j.1469-7793.2001.0183l.x"><li><a href="http://dx.doi.org/10.1111/j.1469-7793.2001.0183l.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pore+properties+and+ionic+block+of+the+rabbit+epithelial+calcium+channel+expressed+in+HEK+293+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pore+properties+and+ionic+block+of+the+rabbit+epithelial+calcium+channel+expressed+in+HEK+293+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001686-Erler1" id="pone.0001686-Erler1"></a>Erler I, Hirnet D, Wissenbach U, Flockerzi V, Niemeyer BA (2004) Ca2+-selective transient receptor potential V channel architecture and function require a specific ankyrin repeat. J Biol Chem  279: 34456–34463.  <ul class="find" data-citedArticleID="1063782" data-doi="10.1074/jbc.m404778200"><li><a href="http://dx.doi.org/10.1074/jbc.m404778200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ca2%2B-selective+transient+receptor+potential+V+channel+architecture+and+function+require+a+specific+ankyrin+repeat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ca2%2B-selective+transient+receptor+potential+V+channel+architecture+and+function+require+a+specific+ankyrin+repeat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001686-Niemeyer1" id="pone.0001686-Niemeyer1"></a>Niemeyer BA, Bergs C, Wissenbach U, Flockerzi V, Trost C (2001) Competitive regulation of CaT-like-mediated Ca2+ entry by protein kinase C and calmodulin. Proc Natl Acad Sci U S A  98: 3600–3605.  <ul class="find" data-citedArticleID="1063818" data-doi="10.1073/pnas.051511398"><li><a href="http://dx.doi.org/10.1073/pnas.051511398" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Competitive+regulation+of+CaT-like-mediated+Ca2%2B+entry+by+protein+kinase+C+and+calmodulin." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Competitive+regulation+of+CaT-like-mediated+Ca2%2B+entry+by+protein+kinase+C+and+calmodulin.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001686-Nijenhuis1" id="pone.0001686-Nijenhuis1"></a>Nijenhuis T, Hoenderop JG, Nilius B, Bindels RJ (2003) (Patho)physiological implications of the novel epithelial Ca2+ channels TRPV5 and TRPV6. Pflugers Arch  446: 401–409.  <ul class="find" data-citedArticleID="1063820" data-doi="10.1007/s00424-003-1038-7"><li><a href="http://dx.doi.org/10.1007/s00424-003-1038-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=%28Patho%29physiological+implications+of+the+novel+epithelial+Ca2%2B+channels+TRPV5+and+TRPV6." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22%28Patho%29physiological+implications+of+the+novel+epithelial+Ca2%2B+channels+TRPV5+and+TRPV6.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001686-Colosimo1" id="pone.0001686-Colosimo1"></a>Colosimo PF, Hosemann KE, Balabhadra S, Villarreal G Jr, Dickson M, et al.  (2005) Widespread Parallel Evolution in Sticklebacks by Repeated Fixation of Ectodysplasin Alleles. Science  307: 1928–1933.  <ul class="find" data-citedArticleID="1063778" data-doi="10.1126/science.1107239"><li><a href="http://dx.doi.org/10.1126/science.1107239" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+Parallel+Evolution+in+Sticklebacks+by+Repeated+Fixation+of+Ectodysplasin+Alleles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+Parallel+Evolution+in+Sticklebacks+by+Repeated+Fixation+of+Ectodysplasin+Alleles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001686-Hermisson1" id="pone.0001686-Hermisson1"></a>Hermisson J, Pennings PS (2005) Soft sweeps: molecular population genetics of adaptation from standing genetic variation. Genetics  169: 2335–2352.  <ul class="find" data-citedArticleID="1063792" data-doi="10.1534/genetics.104.036947"><li><a href="http://dx.doi.org/10.1534/genetics.104.036947" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Soft+sweeps%3A+molecular+population+genetics+of+adaptation+from+standing+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Soft+sweeps%3A+molecular+population+genetics+of+adaptation+from+standing+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001686-Orr1" id="pone.0001686-Orr1"></a>Orr HA, Betancourt AJ (2001) Haldane's sieve and adaptation from the standing genetic variation. Genetics  157: 875–884.  <ul class="find" data-citedArticleID="1063834"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Haldane%27s+sieve+and+adaptation+from+the+standing+genetic+variation.&amp;auth=&amp;atitle=Haldane%27s+sieve+and+adaptation+from+the+standing+genetic+variation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haldane%27s+sieve+and+adaptation+from+the+standing+genetic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haldane%27s+sieve+and+adaptation+from+the+standing+genetic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001686-Pennings1" id="pone.0001686-Pennings1"></a>Pennings PS, Hermisson J (2006) Soft Sweeps III: The Signature of Positive Selection from Recurrent Mutation. PLoS Genet  2: e186.  <ul class="find" data-citedArticleID="1063836" data-doi="10.1371/journal.pgen.0020186.eor"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0020186.eor" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Soft+Sweeps+III%3A+The+Signature+of+Positive+Selection+from+Recurrent+Mutation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Soft+Sweeps+III%3A+The+Signature+of+Positive+Selection+from+Recurrent+Mutation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001686-Prugnolle1" id="pone.0001686-Prugnolle1"></a>Prugnolle F, Manica A, Balloux F (2005) Geography predicts neutral genetic diversity of human populations. Curr Biol  15: R159–160.  <ul class="find" data-citedArticleID="1063840" data-doi="10.1016/j.cub.2005.02.038"><li><a href="http://dx.doi.org/10.1016/j.cub.2005.02.038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geography+predicts+neutral+genetic+diversity+of+human+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geography+predicts+neutral+genetic+diversity+of+human+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001686-Goebel1" id="pone.0001686-Goebel1"></a>Goebel T, Waters MR, Dikova M (2003) The Archaeology of Ushki Lake, Kamchatka, and the Pleistocene Peopling of the Americas. Science  301: 501–505.  <ul class="find" data-citedArticleID="1063786" data-doi="10.1126/science.1086555"><li><a href="http://dx.doi.org/10.1126/science.1086555" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Archaeology+of+Ushki+Lake%2C+Kamchatka%2C+and+the+Pleistocene+Peopling+of+the+Americas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Archaeology+of+Ushki+Lake%2C+Kamchatka%2C+and+the+Pleistocene+Peopling+of+the+Americas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001686-Haynes1" id="pone.0001686-Haynes1"></a>Haynes G (2002) The Early Settlement of North America: The Clovis Era. Cambridge: Cambridge Univ. Press.   <ul class="find-nolinks"></ul></li><li><span class="label">56.
              </span><a name="pone.0001686-Hope1" id="pone.0001686-Hope1"></a>Hope GS, Haberle SG (2005)  The history of the human landscapes of New Guinea. In: Pawley A, Attenborough R, Golson J, Hide R, editors. Papuan Pasts: Cultural, linguistic and biological histories of Papuan-speaking peoples. Canberra: Pacific Linguistics, Research School of Pacific and Asian Studies, Australian National University. pp. 541–554.  <ul class="find-nolinks"></ul></li><li><span class="label">57.
              </span><a name="pone.0001686-Schwarz1" id="pone.0001686-Schwarz1"></a>Schwarz EC, Wissenbach U, Niemeyer BA, Strauss B, Philipp SE, et al.  (2006) TRPV6 potentiates calcium-dependent cell proliferation. Cell Calcium  39: 163–173.  <ul class="find" data-citedArticleID="1063858" data-doi="10.1016/j.ceca.2005.10.006"><li><a href="http://dx.doi.org/10.1016/j.ceca.2005.10.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRPV6+potentiates+calcium-dependent+cell+proliferation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRPV6+potentiates+calcium-dependent+cell+proliferation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001686-Lehenkyi1" id="pone.0001686-Lehenkyi1"></a>Lehen'kyi V, Flourakis M, Skryma R, Prevarskaya N (2007) TRPV6 channel controls prostate cancer cell proliferation via Ca(2+)/NFAT-dependent pathways. Oncogene  26: 7380–7385.  <ul class="find" data-citedArticleID="1063806" data-doi="10.1038/sj.onc.1210545"><li><a href="http://dx.doi.org/10.1038/sj.onc.1210545" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRPV6+channel+controls+prostate+cancer+cell+proliferation+via+Ca%282%2B%29%2FNFAT-dependent+pathways." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRPV6+channel+controls+prostate+cancer+cell+proliferation+via+Ca%282%2B%29%2FNFAT-dependent+pathways.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001686-Prevarskaya1" id="pone.0001686-Prevarskaya1"></a>Prevarskaya N, Zhang L, Barritt G (2007) TRP channels in cancer. Biochim Biophys Acta  1772: 937–946.  <ul class="find" data-citedArticleID="1063838" data-doi="10.1016/j.bbadis.2007.05.006"><li><a href="http://dx.doi.org/10.1016/j.bbadis.2007.05.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRP+channels+in+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRP+channels+in+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001686-Lehenkyi2" id="pone.0001686-Lehenkyi2"></a>Lehen'kyi V, Beck B, Polakowska R, Charveron M, Bordat P, et al.  (2007) TRPV6 Is a Ca2+ Entry Channel Essential for Ca2+-induced Differentiation of Human Keratinocytes. J Biol Chem  282: 22582–22591.  <ul class="find" data-citedArticleID="1063808" data-doi="10.1074/jbc.m611398200"><li><a href="http://dx.doi.org/10.1074/jbc.m611398200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TRPV6+Is+a+Ca2%2B+Entry+Channel+Essential+for+Ca2%2B-induced+Differentiation+of+Human+Keratinocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TRPV6+Is+a+Ca2%2B+Entry+Channel+Essential+for+Ca2%2B-induced+Differentiation+of+Human+Keratinocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001686-vandeGraaf1" id="pone.0001686-vandeGraaf1"></a>van de Graaf SF, Boullart I, Hoenderop JG, Bindels RJ (2004) Regulation of the epithelial Ca2+ channels TRPV5 and TRPV6 by 1alpha,25-dihydroxy Vitamin D3 and dietary Ca2+. J Steroid Biochem Mol Biol  89–90: 303–308.  <ul class="find" data-citedArticleID="1063902" data-doi="10.1016/j.jsbmb.2004.03.029"><li><a href="http://dx.doi.org/10.1016/j.jsbmb.2004.03.029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+the+epithelial+Ca2%2B+channels+TRPV5+and+TRPV6+by+1alpha%2C25-dihydroxy+Vitamin+D3+and+dietary+Ca2%2B." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+the+epithelial+Ca2%2B+channels+TRPV5+and+TRPV6+by+1alpha%2C25-dihydroxy+Vitamin+D3+and+dietary+Ca2%2B.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001686-BessouTouya1" id="pone.0001686-BessouTouya1"></a>Bessou-Touya S, Picardo M, Maresca V, Surleve-Bazeille JE, Pain C, et al.  (1998) Chimeric human epidermal reconstructs to study the role of melanocytes and keratinocytes in pigmentation and photoprotection. J Invest Dermatol  111: 1103–1108.  <ul class="find" data-citedArticleID="1063768" data-doi="10.1046/j.1523-1747.1998.00405.x"><li><a href="http://dx.doi.org/10.1046/j.1523-1747.1998.00405.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chimeric+human+epidermal+reconstructs+to+study+the+role+of+melanocytes+and+keratinocytes+in+pigmentation+and+photoprotection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chimeric+human+epidermal+reconstructs+to+study+the+role+of+melanocytes+and+keratinocytes+in+pigmentation+and+photoprotection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001686-Minwalla1" id="pone.0001686-Minwalla1"></a>Minwalla L, Zhao Y, Le Poole IC, Wickett RR, Boissy RE (2001) Keratinocytes play a role in regulating distribution patterns of recipient melanosomes in vitro. J Invest Dermatol  117: 341–347.  <ul class="find" data-citedArticleID="1063812" data-doi="10.1046/j.0022-202x.2001.01411.x"><li><a href="http://dx.doi.org/10.1046/j.0022-202x.2001.01411.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Keratinocytes+play+a+role+in+regulating+distribution+patterns+of+recipient+melanosomes+in+vitro." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Keratinocytes+play+a+role+in+regulating+distribution+patterns+of+recipient+melanosomes+in+vitro.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001686-Diamond1" id="pone.0001686-Diamond1"></a>Diamond J, Bellwood P (2003) Farmers and Their Languages: The First Expansions. Science  300: 597–603.  <ul class="find" data-citedArticleID="1063780" data-doi="10.1126/science.1078208"><li><a href="http://dx.doi.org/10.1126/science.1078208" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Farmers+and+Their+Languages%3A+The+First+Expansions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Farmers+and+Their+Languages%3A+The+First+Expansions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001686-Oota1" id="pone.0001686-Oota1"></a>Oota H, Pakendorf B, Weiss G, von Haeseler A, Pookajorn S, et al.  (2005) Recent origin and cultural reversion of a hunter-gatherer group. PLoS Biol  3: e71.  <ul class="find" data-citedArticleID="1063832" data-doi="10.1371/journal.pbio.0030071"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030071" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recent+origin+and+cultural+reversion+of+a+hunter-gatherer+group." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recent+origin+and+cultural+reversion+of+a+hunter-gatherer+group.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001686-Stephens1" id="pone.0001686-Stephens1"></a>Stephens M, Scheet P (2005) Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation. Am J Hum Genet  76: 449–462.  <ul class="find" data-citedArticleID="1063862" data-doi="10.1086/428594"><li><a href="http://dx.doi.org/10.1086/428594" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accounting+for+decay+of+linkage+disequilibrium+in+haplotype+inference+and+missing-data+imputation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accounting+for+decay+of+linkage+disequilibrium+in+haplotype+inference+and+missing-data+imputation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001686-Stephens2" id="pone.0001686-Stephens2"></a>Stephens M, Smith NJ, Donnelly P (2001) A new statistical method for haplotype reconstruction from population data. Am J Hum Genet  68: 978–989.  <ul class="find" data-citedArticleID="1063864" data-doi="10.1086/319501"><li><a href="http://dx.doi.org/10.1086/319501" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+new+statistical+method+for+haplotype+reconstruction+from+population+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+new+statistical+method+for+haplotype+reconstruction+from+population+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001686-Ptak1" id="pone.0001686-Ptak1"></a>Ptak SE, Voelpel K, Przeworski M (2004) Insights into recombination from patterns of linkage disequilibrium in humans. Genetics  167: 387–397.  <ul class="find" data-citedArticleID="1063846" data-doi="10.1534/genetics.167.1.387"><li><a href="http://dx.doi.org/10.1534/genetics.167.1.387" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insights+into+recombination+from+patterns+of+linkage+disequilibrium+in+humans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insights+into+recombination+from+patterns+of+linkage+disequilibrium+in+humans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001686-Stoneking1" id="pone.0001686-Stoneking1"></a>Stoneking M, Jorde LB, Bhatia K, Wilson AC (1990) Geographic variation in human mitochondrial DNA from Papua New Guinea. Genetics  124: 717–733.  <ul class="find" data-citedArticleID="1063868"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Geographic+variation+in+human+mitochondrial+DNA+from+Papua+New+Guinea.&amp;auth=&amp;atitle=Geographic+variation+in+human+mitochondrial+DNA+from+Papua+New+Guinea." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Geographic+variation+in+human+mitochondrial+DNA+from+Papua+New+Guinea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Geographic+variation+in+human+mitochondrial+DNA+from+Papua+New+Guinea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001686-Stephens3" id="pone.0001686-Stephens3"></a>Stephens JC, Schneider JA, Tanguay DA, Choi J, Acharya T, et al.  (2001) Haplotype variation and linkage disequilibrium in 313 human genes. Science  293: 489–493.  <ul class="find" data-citedArticleID="1063866" data-doi="10.1126/science.1059431"><li><a href="http://dx.doi.org/10.1126/science.1059431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Haplotype+variation+and+linkage+disequilibrium+in+313+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Haplotype+variation+and+linkage+disequilibrium+in+313+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001686-Pruitt1" id="pone.0001686-Pruitt1"></a>Pruitt KD, Tatusova T, Maglott DR (2005) NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucl Acids Res  33: D501–504.  <ul class="find" data-citedArticleID="1063842" data-doi="10.1093/nar/gkl842"><li><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NCBI+Reference+Sequence+%28RefSeq%29%3A+a+curated+non-redundant+sequence+database+of+genomes%2C+transcripts+and+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001686-Muller1" id="pone.0001686-Muller1"></a>Muller D, JGJ H, Vennekens R, Eggert P, Harangi F, et al.  (2002) Epithelial Ca channel 1 (ECaC1) in autosomal dominant hypercalciuria. Nephrol Dial Transplant  17: (9): 1614–1620.  <ul class="find" data-citedArticleID="1063814" data-doi="10.1093/ndt/17.9.1614"><li><a href="http://dx.doi.org/10.1093/ndt/17.9.1614" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Epithelial+Ca+channel+1+%28ECaC1%29+in+autosomal+dominant+hypercalciuria." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Epithelial+Ca+channel+1+%28ECaC1%29+in+autosomal+dominant+hypercalciuria.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.XML" value="150963"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.PDF" value="576191"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g001.PNG_L" value="243880"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g001.PNG_M" value="58425"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g001.PNG_S" value="7981"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g001.TIF" value="616696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g001.PNG_I" value="20895"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g002.PNG_L" value="2618725"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g002.PNG_M" value="284656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g002.PNG_S" value="13693"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g002.TIF" value="6367820"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g002.PNG_I" value="116720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g003.PNG_L" value="417697"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g003.PNG_M" value="81897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g003.PNG_S" value="11638"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g003.TIF" value="1183264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g003.PNG_I" value="40284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t001.PNG_L" value="120671"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t001.PNG_M" value="169720"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t001.PNG_S" value="20005"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t001.TIF" value="503766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t001.PNG_I" value="58149"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t002.PNG_L" value="84088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t002.PNG_M" value="51233"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t002.PNG_S" value="8836"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t002.TIF" value="372080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.t002.PNG_I" value="16301"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g004.PNG_L" value="431947"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g004.PNG_M" value="100926"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g004.PNG_S" value="10341"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g004.TIF" value="827050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g004.PNG_I" value="35871"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g005.PNG_L" value="464866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g005.PNG_M" value="95965"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g005.PNG_S" value="10120"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g005.TIF" value="788522"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g005.PNG_I" value="34435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g006.PNG_L" value="341817"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g006.PNG_M" value="83145"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g006.PNG_S" value="11946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g006.TIF" value="877358"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.g006.PNG_I" value="30373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s001.TIF" value="509575"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s002.TIF" value="2945642"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s003.TIF" value="996245"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s004.TIF" value="1261028"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s005.TIF" value="3490479"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s006.TIF" value="834906"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s007.TIF" value="3587172"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s008.PDF" value="110972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s009.DOC" value="36352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s010.DOC" value="49152"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s011.DOC" value="31232"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s012.DOC" value="29696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001686.s013.DOC" value="34816"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001686&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001686" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001686&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001686&volume=&issue=&title=Parallel Selection on TRPV6 in Human Populations&author_name=David%20A.%20Hughes%2C%20Kun%20Tang%2C%20Rainer%20Strotmann%2C%20Torsten%20Sch%C3%B6neberg%2C%20Jean%20Prenen%2C%20Bernd%20Nilius%2C%20Mark%20Stoneking&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001686" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686&amp;t=Parallel%20Selection%20on%20TRPV6%20in%20Human%20Populations" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686&title=Parallel%20Selection%20on%20TRPV6%20in%20Human%20Populations&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686&amp;title=Parallel%20Selection%20on%20TRPV6%20in%20Human%20Populations" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001686&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Parallel Selection on TRPV6 in Human Populations';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001686';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Parallel%20Selection%20on%20TRPV6%20in%20Human%20Populations http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001686" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001686" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22African+Americans%22" title="Search for articles in the subject area:'African Americans'"><div class="flagText">African Americans</div></a>
              <div data-categoryid="40667" data-articleid="26198"
                   data-categoryname="African Americans"
                   class="flagImage" title="Flag 'African Americans' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="26198"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Charge+carriers%22" title="Search for articles in the subject area:'Charge carriers'"><div class="flagText">Charge carriers</div></a>
              <div data-categoryid="24585" data-articleid="26198"
                   data-categoryname="Charge carriers"
                   class="flagImage" title="Flag 'Charge carriers' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genetic+loci%22" title="Search for articles in the subject area:'Genetic loci'"><div class="flagText">Genetic loci</div></a>
              <div data-categoryid="17713" data-articleid="26198"
                   data-categoryname="Genetic loci"
                   class="flagImage" title="Flag 'Genetic loci' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Han+Chinese%22" title="Search for articles in the subject area:'Han Chinese'"><div class="flagText">Han Chinese</div></a>
              <div data-categoryid="40115" data-articleid="26198"
                   data-categoryname="Han Chinese"
                   class="flagImage" title="Flag 'Han Chinese' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Haplotypes%22" title="Search for articles in the subject area:'Haplotypes'"><div class="flagText">Haplotypes</div></a>
              <div data-categoryid="19681" data-articleid="26198"
                   data-categoryname="Haplotypes"
                   class="flagImage" title="Flag 'Haplotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Natural+selection%22" title="Search for articles in the subject area:'Natural selection'"><div class="flagText">Natural selection</div></a>
              <div data-categoryid="34433" data-articleid="26198"
                   data-categoryname="Natural selection"
                   class="flagImage" title="Flag 'Natural selection' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Papua+New+Guinea%22" title="Search for articles in the subject area:'Papua New Guinea'"><div class="flagText">Papua New Guinea</div></a>
              <div data-categoryid="40451" data-articleid="26198"
                   data-categoryname="Papua New Guinea"
                   class="flagImage" title="Flag 'Papua New Guinea' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8510'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4395'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3470&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10499">Referee comments: Referee 1</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
