

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Modeling Transcriptome Based on Transcript-Sampling Data</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001659"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001659" />

    <meta name="citation_title" content="Modeling Transcriptome Based on Transcript-Sampling Data"/>
    <meta itemprop="name" content="Modeling Transcriptome Based on Transcript-Sampling Data"/>

      <meta name="citation_author" content="Jiang Zhu"/>
            <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China"/>
            <meta name="citation_author_institution" content="Graduate University of Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Fuhong He"/>
            <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China"/>
            <meta name="citation_author_institution" content="Graduate University of Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Jing Wang"/>
            <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Jun Yu"/>
            <meta name="citation_author_institution" content="Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China"/>

    <meta name="citation_date" content="2008/2/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001659.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1659"/>
    <meta name="citation_issue" content="2"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Interrogating the transcriptome.; citation_author=Y Ruan; citation_author=P Le Ber; citation_author=HH Ng; citation_author=ET Liu; citation_journal_title=Trends Biotechnol; citation_volume=22; citation_number=1; citation_pages=23-30; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Quantitative monitoring of gene expression patterns with a complementary DNA microarray.; citation_author=M Schena; citation_author=D Shalon; citation_author=RW Davis; citation_author=PO Brown; citation_journal_title=Science; citation_volume=270; citation_number=2; citation_pages=467-470; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Expression monitoring by hybridization to high-density oligonucleotide arrays.; citation_author=DJ Lockhart; citation_author=H Dong; citation_author=MC Byrne; citation_author=MT Follettie; citation_author=MV Gallo; citation_journal_title=Nat Biotechnol; citation_volume=14; citation_number=3; citation_pages=1675-1680; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Large-scale transcriptional activity in chromosomes 21 and 22.; citation_author=P Kapranov; citation_author=SE Cawley; citation_author=J Drenkow; citation_author=S Bekiranov; citation_author=RL Strausberg; citation_journal_title=Science; citation_volume=296; citation_number=4; citation_pages=916-919; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Global identification of human transcribed sequences with genome tiling arrays.; citation_author=P Bertone; citation_author=V Stolc; citation_author=TE Royce; citation_author=JS Rozowsky; citation_author=AE Urban; citation_journal_title=Science; citation_volume=306; citation_number=5; citation_pages=2242-2246; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution.; citation_author=J Cheng; citation_author=P Kapranov; citation_author=J Drenkow; citation_author=S Dike; citation_author=S Brubaker; citation_journal_title=Science; citation_volume=308; citation_number=6; citation_pages=1149-1154; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide location and function of DNA binding proteins.; citation_author=B Ren; citation_author=F Robert; citation_author=JJ Wyrick; citation_author=O Aparicio; citation_author=EG Jennings; citation_journal_title=Science; citation_volume=290; citation_number=7; citation_pages=2306-2309; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Complementary DNA sequencing: expressed sequence tags and human genome project.; citation_author=MD Adams; citation_author=JM Kelley; citation_author=JD Gocayne; citation_author=M Dubnick; citation_author=MH Polymeropoulos; citation_journal_title=Science; citation_volume=252; citation_number=8; citation_pages=1651-1656; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Serial analysis of gene expression.; citation_author=VE Velculescu; citation_author=L Zhang; citation_author=B Vogelstein; citation_author=KW Kinzler; citation_journal_title=Science; citation_volume=270; citation_number=9; citation_pages=484-487; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Using the transcriptome to annotate the genome.; citation_author=S Saha; citation_author=A Sparks; citation_author=C Rago; citation_author=V Akmaev; citation_author=C Wang; citation_journal_title=Nat Biotechnol; citation_volume=20; citation_number=10; citation_pages=508-512; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays.; citation_author=S Brenner; citation_author=M Johnson; citation_author=J Bridgham; citation_author=G Golda; citation_author=DH Lloyd; citation_journal_title=Nat Biotechnol; citation_volume=18; citation_number=11; citation_pages=630-634; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=In vitro cloning of complex mixtures of DNA on microbeads: physical separation of differentially expressed cDNAs.; citation_author=S Brenner; citation_author=SR Williams; citation_author=EH Vermaas; citation_author=T Storck; citation_author=K Moon; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=12; citation_pages=1665-1670; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage.; citation_author=T Shiraki; citation_author=S Kondo; citation_author=S Katayama; citation_author=K Waki; citation_author=T Kasukawa; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=13; citation_pages=15776-15781; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=5′ Long serial analysis of gene expression (LongSAGE) and 3′ LongSAGE for transcriptome characterization and genome annotation.; citation_author=CL Wei; citation_author=P Ng; citation_author=KP Chiu; citation_author=CH Wong; citation_author=CC Ang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=101; citation_number=14; citation_pages=11701-11706; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation.; citation_author=P Ng; citation_author=CL Wei; citation_author=WK Sung; citation_author=KP Chiu; citation_author=L Lipovich; citation_journal_title=Nat Methods; citation_volume=2; citation_number=15; citation_pages=105-111; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Tag-based approaches for transcriptome research and genome annotation.; citation_author=M Harbers; citation_author=P Carninci; citation_journal_title=Nat Methods; citation_volume=2; citation_number=16; citation_pages=495-502; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors.; citation_author=M Margulies; citation_author=M Egholm; citation_author=WE Altman; citation_author=S Attiya; citation_author=JS Bader; citation_journal_title=Nature; citation_volume=437; citation_number=17; citation_pages=376-380; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Accurate multiplex polony sequencing of an evolved bacterial genome.; citation_author=J Shendure; citation_author=GJ Porreca; citation_author=NB Reppas; citation_author=X Lin; citation_author=JP McCutcheon; citation_journal_title=Science; citation_volume=309; citation_number=18; citation_pages=1728-1732; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Microfabricated bioprocessor for integrated nanoliter-scale Sanger DNA sequencing.; citation_author=RG Blazej; citation_author=P Kumaresan; citation_author=RA Mathies; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=19; citation_pages=7240-7245; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiplex sequencing of paired-end ditags (MS-PET): a strategy for the ultra-high-throughput analysis of transcriptomes and genomes.; citation_author=P Ng; citation_author=JJ Tan; citation_author=HS Ooi; citation_author=YL Lee; citation_author=KP Chiu; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=20; citation_pages=e84; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Can transcriptome size be estimated from SAGE catalogs?; citation_author=M Stern; citation_author=S Anisimov; citation_author=K Boheler; citation_journal_title=Binformatics; citation_volume=19; citation_number=21; citation_pages=443-448; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=General statistics of stochastic process of gene expression in eukaryotic cells.; citation_author=VA Kuznetsov; citation_author=GD Knott; citation_author=RF Bonner; citation_journal_title=Genetics; citation_volume=161; citation_number=22; citation_pages=1321-1332; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Modeling Sage data with a truncated gamma-Poisson model.; citation_author=HH Thygesen; citation_author=AH Zwinderman; citation_journal_title=BMC Bioinformatics; citation_volume=7; citation_number=23; citation_pages=157; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=An anatomy of normal and malignant gene expression.; citation_author=K Boon; citation_author=EC Osorio; citation_author=SF Greenhut; citation_author=CF Schaefer; citation_author=J Shoemaker; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=24; citation_pages=11287-11292; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Large-scale production of SAGE libraries from microdissected tissues, flow-sorted cells, and cell lines.; citation_author=J Khattra; citation_author=AD Delaney; citation_author=Y Zhao; citation_author=A Siddiqui; citation_author=J Asano; citation_journal_title=Genome Res; citation_volume=17; citation_number=25; citation_pages=108-116; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=LongSAGE profiling of nine human embryonic stem cell lines.; citation_author=M Hirst; citation_author=A Delaney; citation_author=SA Rogers; citation_author=A Schnerch; citation_author=DR Persaud; citation_journal_title=Genome Biol; citation_volume=8; citation_number=26; citation_pages=R113; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=MPSS profiling of human embryonic stem cells.; citation_author=R Brandenberger; citation_author=I Khrebtukova; citation_author=RS Thies; citation_author=T Miura; citation_author=C Jingli; citation_journal_title=BMC Dev Biol; citation_volume=4; citation_number=27; citation_pages=10; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Properties of four human embryonic stem cell lines maintained in a feeder-free culture system.; citation_author=MK Carpenter; citation_author=ES Rosler; citation_author=GJ Fisk; citation_author=R Brandenberger; citation_author=X Ares; citation_journal_title=Dev Dyn; citation_volume=229; citation_number=28; citation_pages=243-258; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The human genome browser at UCSC.; citation_author=WJ Kent; citation_author=CW Sugnet; citation_author=TS Furey; citation_author=KM Roskin; citation_author=TH Pringle; citation_journal_title=Genome Res; citation_volume=12; citation_number=29; citation_pages=996-1006; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Bayesian shrinkage estimation of the relative abundance of mRNA transcripts using SAGE.; citation_author=J Morris; citation_author=K Baggerly; citation_author=K Coombes; citation_journal_title=Biometrics; citation_volume=59; citation_number=30; citation_pages=476-486; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Three abundance classes in HeLa cell messenger RNA.; citation_author=JO Bishop; citation_author=JG Morton; citation_author=M Rosbash; citation_author=M Richardson; citation_journal_title=Nature; citation_volume=250; citation_number=31; citation_pages=199-204; citation_date=1974; " />
      <meta name="citation_reference" content="citation_title=Analysis of mRNA populations.; citation_author=JO Bishop; citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_volume=283; citation_number=32; citation_pages=373-374; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Dissecting the regulatory circuitry of a eukaryotic genome.; citation_author=FC Holstege; citation_author=EG Jennings; citation_author=JJ Wyrick; citation_author=TI Lee; citation_author=CJ Hengartner; citation_journal_title=Cell; citation_volume=95; citation_number=33; citation_pages=717-728; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Mapping DNA-protein interactions in large genomes by sequence tag analysis of genomic enrichment.; citation_author=J Kim; citation_author=AA Bhinge; citation_author=XC Morgan; citation_author=VR Iyer; citation_journal_title=Nat Methods; citation_volume=2; citation_number=34; citation_pages=47-53; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A global map of p53 transcription-factor binding sites in the human genome.; citation_author=CL Wei; citation_author=Q Wu; citation_author=VB Vega; citation_author=KP Chiu; citation_author=P Ng; citation_journal_title=Cell; citation_volume=124; citation_number=35; citation_pages=207-219; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=ECgene: genome-based EST clustering and gene modeling for alternative splicing.; citation_author=N Kim; citation_author=S Shin; citation_author=S Lee; citation_journal_title=Genome Res; citation_volume=15; citation_number=36; citation_pages=566-576; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=ESTGenes: alternative splicing from ESTs in Ensembl.; citation_author=E Eyras; citation_author=M Caccamo; citation_author=V Curwen; citation_author=M Clamp; citation_journal_title=Genome Res; citation_volume=14; citation_number=37; citation_pages=976-987; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Detecting the impact of sequencing errors on SAGE data.; citation_author=J Colinge; citation_author=G Feger; citation_journal_title=Binformatics; citation_volume=17; citation_number=38; citation_pages=840-842; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Identifying novel transcripts and novel genes in the human genome by using novel SAGE tags.; citation_author=J Chen; citation_author=M Sun; citation_author=S Lee; citation_author=G Zhou; citation_author=JD Rowley; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=39; citation_pages=12257-12262; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A novel algorithm for computational identification of contaminated EST libraries.; citation_author=R Sorek; citation_author=HM Safer; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=40; citation_pages=1067-1074; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Correct identification of genes from serial analysis of gene expression tag sequences.; citation_author=S Lee; citation_author=T Clark; citation_author=J Chen; citation_author=G Zhou; citation_author=LR Scott; citation_journal_title=Genomics; citation_volume=79; citation_number=41; citation_pages=598-602; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Oligo(dT) primer generates a high frequency of truncated cDNAs through internal poly(A) priming during reverse transcription.; citation_author=DK Nam; citation_author=S Lee; citation_author=G Zhou; citation_author=X Cao; citation_author=C Wang; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=42; citation_pages=6152-6156; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Heterogeneity in polyadenylation cleavage sites in mammalian mRNA sequences: implications for SAGE analysis.; citation_author=E Pauws; citation_author=AH van Kampen; citation_author=SA van de Graaf; citation_author=JJ de Vijlder; citation_author=C Ris-Stalpers; citation_journal_title=Nucleic Acids Res; citation_volume=29; citation_number=43; citation_pages=1690-1694; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The impact of SNPs on the interpretation of SAGE and MPSS experimental data.; citation_author=AP Silva; citation_author=JE De Souza; citation_author=PA Galante; citation_author=GJ Riggins; citation_author=SJ De Souza; citation_journal_title=Nucleic Acids Res; citation_volume=32; citation_number=44; citation_pages=6104-6110; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=On a Distribution Law for Word Frequencies.; citation_author=HS Sichel; citation_journal_title=J Am Stat Assoc; citation_volume=70; citation_number=45; citation_pages=542-547; citation_date=1975; " />
      <meta name="citation_reference" content="citation_title=Word Frequency Distributions and Type-Token Characteristics.; citation_author=HS Sichel; citation_journal_title=Mathematical Scientist; citation_volume=11; citation_number=46; citation_pages=45-72; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Anatomy of the generalized inverse Gaussian-Poisson distribution with special applications to bibliometric studies.; citation_author=HS Sichel; citation_journal_title=Inform Process Manag; citation_volume=28; citation_number=47; citation_pages=5-17; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Inference for Sichel's Compound Poisson Distribution.; citation_author=AC Atkinson; citation_journal_title=J Am Stat Assoc; citation_volume=77; citation_number=48; citation_pages=153-158; citation_date=1982; " />
      <meta name="citation_reference" content="citation_title=Parameter Estimation for the Sichel Distribution and Its Multivariate Extension.; citation_author=GZ Stein; citation_journal_title=J Am Stat Assoc; citation_volume=82; citation_number=49; citation_pages=938-944; citation_date=1987; " />
      <meta name="citation_reference" content="citation_title=Yes, the GIGP really does work- and is workable!; citation_author=QL Burrell; citation_journal_title=J Am Soc Inf Sci; citation_volume=44; citation_number=50; citation_pages=61-69; citation_date=1993; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Modeling Transcriptome Based on Transcript-Sampling Data"/>
    <meta name="twitter:description" content="BackgroundNewly-evolved multiplex sequencing technology has been bringing transcriptome sequencing into an unprecedented depth. Millions of transcript tags now can be acquired in a single experiment through parallelization. The significant increase in throughput and reduction in cost required us to address some fundamental questions, such as how many transcript tags do we have to sequence for a given transcriptome? How could we estimate the total number of unique transcripts for different cell types (transcriptome diversity) and the distribution of their copy numbers (transcriptome dynamics)? What is the probability that a transcript with a given expression level to be detected at a certain sampling depth?Methodology/Principal FindingsWe developed a statistical model to evaluate these parameters based on transcriptome-sampling data. Three mixture models were exploited for their potentials to model the sampling frequencies. We demonstrated that relative abundances of all transcripts in a transcriptome follow the generalized inverse Gaussian distribution. The widely known beta and gamma distributions failed to fulfill the singular characteristics of relative abundance distribution, i.e., highly skewed toward zero and with a long tail. An estimator of transcriptome diversity and an analytical form of sampling growth curve were proposed in a coherent framework. Experimental data fitted this model very well and Monte Carlo simulations based on this model replicated sampling experiments in a remarkable precision.ConclusionsTaking human embryonic stem cell as a prototype, we demonstrated that sequencing tens of thousands of transcript tags in an ordinary EST/SAGE experiment was far from sufficient. In order to fully characterize a human transcriptome, millions of transcript tags had to be sequenced. This model lays a statistical basis for transcriptome-sampling experiments and in essence can be used in all sampling-based data."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001659.g007"/>

  <meta property="og:title" content="Modeling Transcriptome Based on Transcript-Sampling Data" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6875'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=9354'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=6318&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001659">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001659" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001659&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Modeling Transcriptome Based on Transcript-Sampling Data
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jiang Zhu, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China, 
                  Graduate University of Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Fuhong He, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China, 
                  Graduate University of Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jing Wang
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:wangjing@big.ac.cn">wangjing@big.ac.cn</a> (JW); <a href="mailto:junyu@big.ac.cn">junyu@big.ac.cn</a> (JY)</p>

                <p>Affiliation:
                  Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jun Yu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:wangjing@big.ac.cn">wangjing@big.ac.cn</a> (JW); <a href="mailto:junyu@big.ac.cn">junyu@big.ac.cn</a> (JY)</p>

                <p>Affiliation:
                  Chinese Academy of Sciences (CAS) Key Laboratory of Genome Sciences and Information, Beijing Institute of Genomics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: February 20, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001659</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001659-t001" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g001" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g002" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g003" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-t002" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g004" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g005" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g006" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001659-g007" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001659">Reader Comments (3)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001659" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Newly-evolved multiplex sequencing technology has been bringing transcriptome sequencing into an unprecedented depth. Millions of transcript tags now can be acquired in a single experiment through parallelization. The significant increase in throughput and reduction in cost required us to address some fundamental questions, such as how many transcript tags do we have to sequence for a given transcriptome? How could we estimate the total number of unique transcripts for different cell types (transcriptome diversity) and the distribution of their copy numbers (transcriptome dynamics)? What is the probability that a transcript with a given expression level to be detected at a certain sampling depth?</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We developed a statistical model to evaluate these parameters based on transcriptome-sampling data. Three mixture models were exploited for their potentials to model the sampling frequencies. We demonstrated that relative abundances of all transcripts in a transcriptome follow the generalized inverse Gaussian distribution. The widely known beta and gamma distributions failed to fulfill the singular characteristics of relative abundance distribution, i.e., highly skewed toward zero and with a long tail. An estimator of transcriptome diversity and an analytical form of sampling growth curve were proposed in a coherent framework. Experimental data fitted this model very well and Monte Carlo simulations based on this model replicated sampling experiments in a remarkable precision.</p>


<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>Taking human embryonic stem cell as a prototype, we demonstrated that sequencing tens of thousands of transcript tags in an ordinary EST/SAGE experiment was far from sufficient. In order to fully characterize a human transcriptome, millions of transcript tags had to be sequenced. This model lays a statistical basis for transcriptome-sampling experiments and in essence can be used in all sampling-based data.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zhu J, He F, Wang J, Yu J (2008) Modeling Transcriptome Based on Transcript-Sampling Data. PLoS ONE 3(2):
          e1659.
            doi:10.1371/journal.pone.0001659</p><p><strong>Academic Editor: </strong>Janet Kelso, Max Planck Institute for Evolutionary Anthropology, Germany</p><p><strong>Received:</strong> December 10, 2007; <strong>Accepted:</strong> January 21, 2008; <strong>Published:</strong> February 20, 2008</p><p><strong>Copyright:</strong> © 2008 Zhu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the National Natural Science Foundation of China (NSFC) (Grant No. 90608003, 30221004) and the National Basic Research Program of China (973 Program) (Grant No. 2006CB910404).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Transcriptomes vary significantly according to specialization of cell types as well as their life cycle or dynamic status, such as growth and apoptosis under various physiological and pathological conditions. This extremely dynamic nature of transcriptomes requires thorough and unbiased profiling experiments to identify as many transcripts as possible, including alternative spliced variants and non-coding RNAs <a href="#pone.0001659-Ruan1">[1]</a>. There are two basic approaches for transcriptomic studies in terms of methodology: hybridization-based and sequencing-based. Hybridization-based microarray technology, due to its high throughput and affordability, is widely used for mapping gene expression patterns <a href="#pone.0001659-Schena1">[2]</a>, <a href="#pone.0001659-Lockhart1">[3]</a>, transcriptional activities (genome tiling array) <a href="#pone.0001659-Kapranov1">[4]</a>–<a href="#pone.0001659-Cheng1">[6]</a>, and binding sites of regulatory proteins (ChIP-on-chip) <a href="#pone.0001659-Ren1">[7]</a>. However, it relies on a predefined probe set and suffers from poor sensitivity for low abundant targets. In contrast, sequencing-based transcript-sampling experiments extract sequence tags to interrogate transcriptomes, such as expressed sequence tag (EST) sequencing <a href="#pone.0001659-Adams1">[8]</a>, serial analysis of gene expression (SAGE) <a href="#pone.0001659-Velculescu1">[9]</a>, <a href="#pone.0001659-Saha1">[10]</a>, massively parallel signature sequencing (MPSS) <a href="#pone.0001659-Brenner1">[11]</a>, <a href="#pone.0001659-Brenner2">[12]</a>, cap analysis gene expression (CAGE) <a href="#pone.0001659-Shiraki1">[13]</a>, and most recently paired-end ditags (PETs) technique <a href="#pone.0001659-Wei1">[14]</a>, <a href="#pone.0001659-Ng1">[15]</a> (see reference <a href="#pone.0001659-Harbers1">[16]</a> for a thorough review). All these techniques share an assumption that the sampling frequency of a tag (or the number of overlapping ESTs) is proportional to the abundance of the corresponding transcript in a given cellular mRNA pool. The sequencing-based methods do not depend on any prior knowledge about the transcriptomes so that in theory they can identify as many targeted transcripts as possible to reach an adequate coverage. A comprehensive survey of transcriptomes by transcript or its tag sampling, followed by extensive microarray experiments for repeated measurements under various physiological conditions should be able to significantly accelerate <em>de novo</em> analyses and functional annotations of unknown transcriptomes, especially when the genome sequence of the targeted organism is available. In recent years, sequencing technology is undergoing a revolution where highly-multiplexed sequencing instruments allow effective acquisition of sequence reads by millions in a single experiment <a href="#pone.0001659-Margulies1">[17]</a>–<a href="#pone.0001659-Blazej1">[19]</a>. Although the read length of some current techniques, typically 30–150 nt in length, is not long enough for <em>de novo</em> sequencing of large and complex genomes, it is sufficient for transcript tag sequencing. As their throughputs and protocols are being improved constantly, sequencing-based methods are expected to gain a great momentum in the years to come <a href="#pone.0001659-Ng2">[20]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>There have been several attempts to model transcriptome-sampling data in recent years. Stern and colleagues empirically estimated the relative abundance of a transcript as the ratio of its sampling frequency over the sample size and transcriptome diversity by a simple correction of sampling errors <a href="#pone.0001659-Stern1">[21]</a>. Although this is mathematically valid when the sample size is sufficiently large, the empirical estimation might lead to biases for the low-abundant transcripts. Kuznestsov and colleagues <a href="#pone.0001659-Kuznetsov1">[22]</a> extended discrete Pareto-like distribution to model the sampling frequencies directly, but gave no implication on the distribution of true relative abundances. Very recently, Thygesen and Zwinderman <a href="#pone.0001659-Thygesen1">[23]</a> used the gamma distribution to model the relative abundances but, as we demonstrated in this manuscript, it was not suitable despite of their mathematical simplicity. Statistically determining the distribution of relative abundances not only provides a theoretical basis for accurately estimating transcriptome diversity but also sheds light on the dynamics of a transcriptome.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study, we proposed an effective statistical model for systematically analyzing transcriptome-sampling data. We used continuous probability distribution to model relative abundances of all transcripts in a transcriptome, and then mixed it with a binomial or Poisson model to derive the distribution of sampling frequencies. The resulted distribution was explicitly distinguished from the underlying relative abundance distribution since it has taken sampling errors into account. We exploited the beta-binomial, gamma-Poisson, and generalized inverse Gaussian-Poisson models, and concluded that the relative abundances of a transcriptome followed the generalized inverse Gaussian distribution. We proposed an effective estimator for transcriptome diversity and provided an analytical form of sampling growth curve. Our results were derived from a coherent statistical model thus superior to other empirical curve-fitting methods. Monte Carlo simulations of transcriptome-sampling process were also carried out, and both the experimental and simulated data fitted our model fairly well. Through extensive simulations, we could determine the probability of detecting transcripts with a certain expression level at a given sampling stage, which provides important implications for future experimental design. Our method can be readily programmed with a moderate demand for computing time.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Experimental data</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>For illustration of our model, we used a selected dataset from SAGE Genie website <a href="#pone.0001659-Boon1">[24]</a>, including ten libraries constructed from normal human embryonic stem cells (hESC) generated by Marco Marra's group in Canada according to a LongSAGE protocol <a href="#pone.0001659-Khattra1">[25]</a>, <a href="#pone.0001659-Hirst1">[26]</a>. Among these libraries, SAGE Genie library 843 (or Lib843, derived from undifferentiated hESC cell line H9 over 38 passages) has the largest sample size. We pooled it with two other libraries, Lib1390 (hESC cell line H1 over 31 passages) and Lib1313 (hESC cell line H7 over 33 passages), to represent a more in-depth sampling. Pooling the three hESC cell lines has been rationalized to represent the overall state of hESC as sampling single cell line may lead to variations due to culture conditions rather than intrinsic differences <a href="#pone.0001659-Brandenberger1">[27]</a>. Previous microarray analysis has suggested remarkably similar expression pattern between the three cell lines <a href="#pone.0001659-Carpenter1">[28]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We eliminated erroneous tags by two criteria. First, we matched all tags to human genomic sequences (UCSC Golden Path hg18) <a href="#pone.0001659-Kent1">[29]</a>, and only matched tags went through further analyses. About 90% of the unmatched tags were found in the one-count bin, and 97% were present in first three count bins, thus a significant fraction of them were likely resulted by sequencing errors. For matched tags, only those observed in more than one of ten libraries were finally regarded as reliable tags. Finally, Lib843 had 311,175 tags corresponding to 38,244 unique tags and the pooled library had 747,778 tags corresponding to 51,470 unique tags. The libraries used for demonstration were summarized in <a href="#pone-0001659-t001">Table 1</a>; analyses on other libraries gave the similar results (data not shown).</p>
<div class="figure" id="pone-0001659-t001"><div class="img"><a name="pone-0001659-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Parameters and estimations under GIGP model.</span></strong></p><span>doi:10.1371/journal.pone.0001659.t001</span></div>

<h4>Mixture model</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We model transcriptome-sampling data as follows. When <em>N</em> transcripts (or transcript tags) are sequenced from a transcriptome of a given cell type, let <em>f<sub>x</sub></em> be the number of unique transcripts that are detected <em>x</em> times. {<em>f</em><sub>1</sub>, <em>f</em><sub>2</sub>,…} is termed as the frequencies of frequencies (FOF), as it is irrelevant to the identity of transcripts. The sample size <em>N</em> = Σ<em>x</em>⋅<em>f<sub>x</sub></em> and <em>s</em> = Σ <em>f<sub>x</sub></em> is the total number of unique transcripts detected in the library (<em>x</em>≥1). Assuming that there are <em>S</em> (unknown) unique transcripts expressed in the RNA preparation (transcriptome diversity), <em>f</em><sub>0</sub> = <em>S</em>−<em>s</em> stands for those undetected transcripts.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Previous studies estimated the relative abundances of all transcripts in <em>f<sub>x</sub></em> as <em>x</em>/<em>N</em> and used FOF to formulate the distribution of relative abundances directly <a href="#pone.0001659-Stern1">[21]</a>. Although this is statistically valid when sample size is sufficiently large, in practice the empirical estimation may be seriously biased due to sampling errors <a href="#pone.0001659-Morris1">[30]</a>. In this study, we used continuous probability distribution <em>φ</em>(<em>π</em>), 0&lt;π&lt;1 to model relative abundance distribution (RAD). That is, any transcript has a probability <em>φ</em>(<em>π</em>)<em>dπ</em> to be expressed at relative abundance π. RAD was then mixed with a basic sampling model, binomial or Poisson distribution, to give sampling frequency distribution (SFD) <em>P</em>(<em>x</em>|<em>N</em>), <em>x</em> = 0,…, <em>N</em>, which gives the probability for any transcript of being detected <em>x</em> times when <em>N</em> transcripts are sequenced. That is, a proportion <em>P</em>(<em>x</em>|<em>N</em>) of total unique transcripts are expected to occur <em>x</em> times in a sample of size <em>N</em>. Since FOF is generated from SFD, we used FOF to fit SFD rather than empirically formulate RAD.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>When the mixture model is fitted, one can deduce the estimator of transcriptome diversity and sampling growth curve in a unified statistical framework. When <em>N</em> transcripts are sequenced, there are <em>s</em>(<em>N</em>) = <em>S</em>[1−<em>P</em>(0|<em>N</em>)] unique transcripts expected to be detected. If we actually detect <em>s</em> unique transcripts, the total number of unique transcripts can be estimated as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e001&amp;representation=PNG" class="inline-graphic"></span>. In addition, RAD <em>φ</em>(<em>π</em>) has expectation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e002&amp;representation=PNG" class="inline-graphic"></span>, giving an alternative estimator of transcriptome diversity <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e003&amp;representation=PNG" class="inline-graphic"></span> as <em>S</em> is large. Using the estimated transcriptome diversity <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e004&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e005&amp;representation=PNG" class="inline-graphic"></span> given by the fitted model, we can deduce an explicit formula for sampling growth curve as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e006&amp;representation=PNG" class="inline-graphic"></span>.</p>


<h4>Evaluation of mixture model</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We exploited three potential mixture models, beta-binomial (BB), gamma-Poisson (GP), and generalized inverse Gaussian-Poisson (GIGP) distribution. We first used Lib843 to demonstrate their performances in fitting the experimental data. The error-filtered SAGE data were first formulated as FOF data, and SFD were fitted by using maximum likelihood method. Once fitted, the expected FOF can be written as <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e007&amp;representation=PNG" class="inline-graphic"></span>, where <em>x</em> = 1,…, <em>N</em> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e008&amp;representation=PNG" class="inline-graphic"></span> is the estimated transcriptome diversity that is generated based on sampling models simultaneously. We plotted the expected FOF under each model against experimental observations (<a href="#pone-0001659-g001">Figure 1</a>). The magnitude of sample size <em>N</em> in our study made BB and GP mixtures behave in the same way, consistent with their theoretical behaviors. From a practical point of view, there was no difference found between these two mixtures; both fitted the FOF data poorly (<a href="#pone-0001659-g001">Figure 1A and 1B</a>). The fitted BB mixture had parameters <em>α</em> = 1.2841e-005, <em>β</em> = 11,573, under which transcriptome diversity was grossly overestimated as <em>S</em> = 8.9492e+008. For GP mixture, the parameters were <em>α</em> = 2.0108e-012, <em>β</em> = 0.036825, and <em>S</em> = 5.6984e+015. As the parameter is very approximate to zero, the estimate may have been seriously biased due to rounding errors. Although both beta and gamma distributions are mathematically simple and straightforward to form probability mixtures, they are not flexible enough to represent RAD, i.e., being highly skewed and with a long tail. This phenomenon was recognized by Thygesen and Zwinderman, leading to a separation of FOF into two parts and introduction of another nonparametric component to model the high frequency bins. Their model resulted in unnecessary mathematical complexity and in essence an incomplete RAD <a href="#pone.0001659-Thygesen1">[23]</a>.</p>
<div class="figure" id="pone-0001659-g001"><div class="img"><a name="pone-0001659-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>The expected FOF plotted against experimental observations.</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Both axes are in logarithmic scale to make FOF data more legible. The expected FOFs under BB (A) and GP (B) models are plotted against the experimentally observed FOF from Lib843. The expected FOF under GIGP model is plotted against the experimental observations from Lib843 (C) and the pooled library (D).</p>
<span>doi:10.1371/journal.pone.0001659.g001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In contrast, GIGP mixture with parameters <em>γ</em> = −0.6439, <em>b</em> = 0.0518 and <em>c</em> = 0.0008 predicted FOF data fairly well for Lib843 (<a href="#pone-0001659-g001">Figure 1C</a>). The large dispersion at the tail was attributed to inconsistent logarithmic scale rather than model errors. Transcriptome diversity was estimated as <em>S</em> = 81,645 under GIGP model. For comparison, we also fitted GIGP mixture using the pooled library (<a href="#pone-0001659-g001">Figure 1D</a>), which gave a consistent estimation <em>S</em> = 77,152. The minor difference was likely due to variations of the original cDNA libraries. The results under GIGP model were summarized in <a href="#pone-0001659-t001">Table 1</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Since BB and GP mixtures fitted the data poorly, we limited further analyses only on GIGP mixture. Once SFD is determined by experimentally observed FOF data, RAD and transcriptome diversity <em>S</em> also become known. To validate the fitted RAD, we did Monte Carlo simulation to imitate the sampling process in SAGE experiments. Based on the fitted RAD and estimated <em>S</em> under GIGP model, a simulated library with the same sample size <em>N</em> as the pooled library was generated and the FOF was plotted in <a href="#pone-0001659-g002">Figure 2</a>, showing that the simulation exactly replicated the experimental result. This gave solid confidence on our fitted RAD and estimated <em>S</em>.</p>
<div class="figure" id="pone-0001659-g002"><div class="img"><a name="pone-0001659-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Monte Carlo simulation for the pooled library.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>Both axes are in logarithmic scale to make FOF data more legible. A virtual transcriptome is generated with <em>S</em> = 77,152 according to the fitted RAD. With the same sample size as the pooled library, the simulated FOF data is plotted against the experimental observation. The simulation based on the fitted RAD and estimated transcriptome diversity <em>S</em> exactly replicates the SAGE experiment.</p>
<span>doi:10.1371/journal.pone.0001659.g002</span></div>

<h4>Relative abundance distribution</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Under GIGP model, RAD is the generalized inverse Gaussian distribution; it is unimodal and very flexible in shape. Being fitted with the pooled library, the RAD—highly skewed toward zero and with a long tail—had values of mode, mean, and median, 4.11e-7, 1.30e-5, and 1.65e-6, respectively (<a href="#pone-0001659-g003">Figure 3</a>). The 75% confidence interval with minimum length was at [1.00e-7, 4.80e-6], and 90% of the unique transcripts had relative abundances less than 1.66e-5. Although it has been previously recognized that most transcripts are expressed at low abundances and highly abundant transcripts are rare, the fitted RAD in this study for the first time precisely described the constitution of transcriptomes.</p>
<div class="figure" id="pone-0001659-g003"><div class="img"><a name="pone-0001659-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Relative abundance distribution for hESC transcriptome.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The GIGP mixture is fitted with the pooled library that represents the hESC transcriptome. The probability density of the fitted generalized inverse Gaussian distribution is plotted (mode: 4.11e-7, mean: 1.30e-5, and median: 1.65e-6). The 75% confidence interval (CI) with minimum length is at [1.00e-7, 4.80e-6]. It is highly skewed toward zero and has a long tail. Inset: the distribution function of RAD, showing that 90% of the transcripts have relative abundances less than 1.66e-5.</p>
<span>doi:10.1371/journal.pone.0001659.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In order to make the concept of relative abundance more biologically relevant, the copy number of a transcript in a given cell can be calculated by multiplying its relative abundance with the estimated total number of transcripts in that cell. A lower bound of this estimated total was based on the RNA-DNA hybridization experiment; it was about 300,000 mRNA molecules in HeLa cell <a href="#pone.0001659-Bishop1">[31]</a>, <a href="#pone.0001659-Bishop2">[32]</a>. As this number may vary across different cell types <em>in vivo</em> and under different culture conditions, it is often hard to determine precisely. We converted the relative abundances into copy numbers under different assumptions within a nearly true range on the total copy number per cell (or per cell type), from 100,000 to 5,000,000. Based on the fitted RAD and transcriptome diversity <em>S</em>, the copy numbers were clustered into different expression level bins and the number of unique transcripts in each bin was formulated in <a href="#pone-0001659-t002">Table 2</a>. As the total copy number per cell increases, most transcripts centre at 1–5 copies per cell. For instance, if we assume there are 1,000,000 copies of transcript per cell, the mean and median copy numbers are 12.96 and 1.65 copies per cell respectively and 90% of transcripts have expression levels less than 16.60 copies per cell. These results have been supported by the experimental evidence in yeast <a href="#pone.0001659-Holstege1">[33]</a>.</p>
<div class="figure" id="pone-0001659-t002"><div class="img"><a name="pone-0001659-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Distribution of expression level.</span></strong></p><span>doi:10.1371/journal.pone.0001659.t002</span></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Based on repeated Monte Carlo simulations and assuming there are 1,000,000 total transcripts per cell, we calculated the mean and median copy numbers of detected transcripts (<a href="#pone-0001659-g004">Figure 4</a>) and the probability of detection in different expression level bins (<a href="#pone-0001659-g005">Figure 5</a>) at different sampling stages. When sampling 10,000 transcripts, the experiment (typical for EST studies) should have enough power to identify all abundant transcripts with expression level greater than 500 copies per cell. For sample sizes ranging from 50,000 to 300,000 (typical for SAGE experiments), only 10% to 47% of the transcripts at expression levels of 1–5 copies per cell are expected to be detected. When a million tags are acquired, 40% and 85% of the transcripts with an expression level of &lt;1 copy per cell and 1–5 copies per cell become detectable, respectively; other high frequency bins should have been saturated to different extents in this sampling range.</p>
<div class="figure" id="pone-0001659-g004"><div class="img"><a name="pone-0001659-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Mean and median copy numbers of detected transcripts at different sampling stages.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Monte Carlo simulation is done with the fitted RAD and estimated transcriptome diversity <em>S</em> of the pooled library. Assuming there are 1,000,000 copies of transcript per cell, the mean and median copy numbers of all detected transcripts at each sampling stage are plotted.</p>
<span>doi:10.1371/journal.pone.0001659.g004</span></div><div class="figure" id="pone-0001659-g005"><div class="img"><a name="pone-0001659-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Detecting probabilities in different expression level bins at different sampling stages.</span></strong></p><a id="article1.body1.sec2.sec4.fig3.caption1.p1" name="article1.body1.sec2.sec4.fig3.caption1.p1"></a><p>Monte Carlo simulation is done with the fitted RAD and estimated transcriptome diversity <em>S</em> of the pooled library. Assuming there are 1,000,000 copies of transcript per cell, the detecting probabilities of transcripts in different expression level bins are plotted as a function of sample size <em>N</em>.</p>
<span>doi:10.1371/journal.pone.0001659.g005</span></div>

<h4>Growth curve of transcriptome sampling</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Another important result of our sampling model is an explicit analytical form of the sampling growth curve (Equation 12). In general, sampling histories are not available for SAGE data archived in public databases. Since tags are assumed to be randomly sampled, one can approximate the sampling history by drawing tags from the library without replacement, and at each sampling point, the observed number of unique transcript tags <em>s</em>(<em>N</em>) can be recorded. We did so and plotted the expected growth curve against the simulated one (<a href="#pone-0001659-g006">Figure 6</a>), showing that the equation (12) faithfully predicted the sampling processes for both Lib843 and the pooled library. We noted that only about 47% of the transcripts were identified in Lib843—the deepest transcriptome sampling by far from SAGE experiments; even for the pooled library with doubled sample size, nearly 33% of the transcripts were still missed.</p>
<div class="figure" id="pone-0001659-g006"><div class="img"><a name="pone-0001659-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Sampling growth curve and transcriptome diversity estimation.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Number of unique transcripts (solid square) identified in Lib843 (A) and the pooled library (B) as well as the sampling histories (red open circle) and predicted growth curve (blue solid line) are plotted. Blue-shaded areas divide the estimated transcriptome diversity <em>S</em> (black dashed line) into four quarters.</p>
<span>doi:10.1371/journal.pone.0001659.g006</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We further used Monte Carlo simulation to evaluate the overall behavior of transcriptome sampling. For sampling effort <em>N</em> from 0 up to 3,000,000 with step length 120,000, the simulated growth curve and that predicted by equation (12) were plotted in <a href="#pone-0001659-g007">Figure 7</a>. Even for this long sampling range, equation (12) still predicted the sampling growth curve quite well. As most transcripts in a given transcriptome exist at very low levels, the sampling efficiency significantly decreases as sampling proceeds. A sampling size of 100,000 is rather minimal for covering the first quarter of the transcriptome. To cover the second and the third quarter, 300,000 and 1,000,000 additional tags have to be acquired, respectively. To identify 90% of the expressed genes, a transcriptome project should aim at sequencing at least 3 million tags. To reach this goal the newly-evolved sequencing technology is indispensable.</p>
<div class="figure" id="pone-0001659-g007"><div class="img"><a name="pone-0001659-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001659" data-uri="info:doi/10.1371/journal.pone.0001659.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001659.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001659.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001659.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Monte Carlo simulation of a deep transcriptome sampling.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>Monte Carlo simulation is done with the fitted RAD and estimated transcriptome diversity <em>S</em> of the pooled library, for a deep sampling that ranges from 0 up to 3,000,000 with step length of 120,000. The predicted growth curve (blue solid line) aligns well with the simulation (red open circle). Both the simulated and the predicted growth curves intercept at the data point for the original library (solid square). Blue-shaded areas divide the estimated transcriptome diversity <em>S</em> (black dashed line) into four quarters.</p>
<span>doi:10.1371/journal.pone.0001659.g007</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Although we used SAGE data for illustration in this manuscript, our method is certainly applicable to other types of transcriptome-sampling experiments such as EST and MPSS as well as other large-scale sampling-based methodologies. For example, our method may still be useful for analyzing chromatin immunoprecipitation data (ChIP-tag) <a href="#pone.0001659-Kim1">[34]</a>, <a href="#pone.0001659-Wei2">[35]</a>. We have found that the relative abundances of ChIP-enriched DNA fragments also follow the generalized inverse Gaussian distribution (data not shown). In general, as long as sampling frequencies are formulated as FOF, our sampling model can be used for statistical evaluation and is independent of experimental details in most circumstances. In this context, the sampling frequency of a target may be the number of short sequence tags from a particular transcript in a SAGE experiment, the number of overlapping ESTs when properly clustered to form a gene (or a transcript) model, or the number of overlapping tags from an immunoprecipitated DNA fragment in a ChIP-tag experiment. Methodology concerning detailed data processing for different types of experiments has been discussed intensively in the literatures <a href="#pone.0001659-Kim1">[34]</a>–<a href="#pone.0001659-Eyras1">[37]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>All tag-based methods essentially depend on the assumption that tags contain sufficient information to establish one-to-one correspondence between tags and transcripts. However, this assumption may collapse to some extents due to many factors. First, sequencing and PCR amplification errors often contribute a large fraction to unmatched tags <a href="#pone.0001659-Colinge1">[38]</a>. The tags with low frequencies are often suspicious but have been revealed corresponding to legitimate transcripts <a href="#pone.0001659-Chen1">[39]</a>. In addition, aberrant tags may also be produced from genomic contaminations <a href="#pone.0001659-Sorek1">[40]</a>. Second, the assumption that the long SAGE tags of 21 bp in length are long enough to ensure unique transcript identification is imperfect <a href="#pone.0001659-Lee1">[41]</a>. Identical sequence tags can be generated from isoforms of a gene, produced by alternative splicing and initiation/termination, as well as different paralogs in a gene family. Third, internally primed reverse transcription, incomplete digestion of tag site and alternative poly(A) cleavage may produce different tags for one unique transcript <a href="#pone.0001659-Boon1">[24]</a>, <a href="#pone.0001659-Nam1">[42]</a>, <a href="#pone.0001659-Pauws1">[43]</a>. Forth, the existence of SNPs in an outbred population also complicates the interpretation of transcript tags <a href="#pone.0001659-Silva1">[44]</a>. The overall impact of these factors on the relationship between tags and transcripts is rather complicated and needs further investigations. In this study we used a very strict filtering process and ambiguities from the first factor have been reduced to a large extent. The biases introduced by the latter two factors are opposite, i.e., one makes several transcripts correspond to one tag and the other matches several tags to one transcript. Overall, we suppose that the two biases would cancel out each other and our estimation has effectively captured the reality.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Sampling frequency distribution</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Binomial distribution is a fundamental statistical assumption about sampling process. For a given transcript with relative abundance <em>π</em>, the sampling frequency when <em>N</em> transcripts are sampled can be modeled by <em>Binomial</em>(<em>N</em>, <em>π</em>). The binomial distribution is often approximated by Poisson distribution <em>Poisson</em>(<em>λ</em>) with <em>λ</em> = <em>π</em>⋅<em>N</em> when <em>N</em> is large, <em>π</em> is small, and <em>π</em>⋅<em>N</em> is moderate, which is precisely the case even for the most abundant transcript. Assuming there are <em>S</em> (unknown) unique transcripts expressed in a given cell type, and each of them has relative abundance <em>π</em><sub>1</sub>, <em>π</em><sub>2</sub>,…, <em>π</em><sub>S</sub> respectively. For mathematical convenience, we assume that <em>π<sub>i</sub></em> s are distributed as a continuous probability density (RAD) <em>φ</em>(<em>π</em>), 0&lt;<em>π</em>&lt;1 under the constraint <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e009&amp;representation=PNG" class="inline-graphic"></span>. By basic probability calculus, for any transcript, the unconditional distribution of sampling frequency (SFD) is written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e010&amp;representation=PNG" class="inline-graphic"><span class="note">(1)</span></span><br>where <em>x</em> = 0,…, <em>N</em>. As <em>φ</em>(<em>π</em>) is necessarily highly skewed toward zero in our context, the binomial in (1) can be approximated by the Poisson distribution. Writing <em>λ</em> = <em>π</em>⋅<em>N</em> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e011&amp;representation=PNG" class="inline-graphic"></span>, it follows that<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e012&amp;representation=PNG" class="inline-graphic"><span class="note">(2)</span></span><br>Probability (2) is the counterpart of (1) under Poisson assumption, and <em>ψ</em>(<em>λ</em>) is a re-parameterized form of RAD <em>φ</em>(<em>π</em>). Extending the upper integration limit <em>N</em> to infinity is justifiable as the integration between <em>N</em> and infinity is negligibly small. Using different RAD leads to different SFD; the justification for one or another depends on its ability to characterize the transcriptome.</p>


<h4>Beta-binomial (BB) and gamma-Poisson (GP) mixtures</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Beta distribution is straightforward for modeling how proportions vary. Mixing it with binomial distribution according to (1) leads to the widely known BB mixture<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e013&amp;representation=PNG" class="inline-graphic"><span class="note">(3)</span></span><br>where <em>x</em> = 0,…, <em>N</em>; <em>α</em>,<em>β</em>&gt;0 are two parameters and <em>B</em>(<em>α</em>,<em>β</em>) is the beta function. Under the Poisson assumption, let the parameter <em>λ</em> follow the gamma distribution <em>ψ</em>(<em>λ</em>). According to (2), GP mixture can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e014&amp;representation=PNG" class="inline-graphic"><span class="note">(4)</span></span><br>where <em>x</em> = 0,…, <em>N</em>; <em>α</em>,<em>β</em>&gt;0 are two <em>N</em>-dependent parameters and Γ(<em>α</em>)is the gamma function. Note that <em>ψ</em>(<em>λ</em>) depends on the sample size <em>N</em>, and so do its parameters. RAD can be obtained simply as <em>φ</em>(<em>π</em>) = <em>N</em>⋅<em>ψ</em>(<em>πN</em>). It is worth noting that gamma distribution can be obtained from beta distribution analogous to that Poisson approximates to binomial. GP mixture thus is an approximate form of BB mixture when <em>N</em> is large.</p>


<h4>Generalized inverse Gaussian-Poisson (GIGP) mixture</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>To capture the singular characteristics of RAD, i.e., highly skewed toward zero and having a long tail, some sophisticated distributions are to be applied. Generalized inverse Gaussian distribution is such a flexible distribution <a href="#pone.0001659-Sichel1">[45]</a>. It has density<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e015&amp;representation=PNG" class="inline-graphic"><span class="note">(5)</span></span><br>where −∞&lt;γ&lt;+∞, <em>b</em>&gt;0 and <em>c</em>&gt;0 are three parameters. <em>K<sub>γ</sub></em>(<em>α</em>) is the second kind of modified Bessel function of order <em>γ</em>. Under the Poisson approximation, according to (2) the GIGP mixture can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e016&amp;representation=PNG" class="inline-graphic"><span class="note">(6)</span></span><br>which has been previously studied by Sichel <a href="#pone.0001659-Sichel2">[46]</a>, <a href="#pone.0001659-Sichel3">[47]</a>. The complicated mathematical form of GIGP mixture, especially the appearance of the modified Bessel function, seems daunting for practical use; this is likely the primary reason for its failure to be widely used. However, by using recurrence relation <a href="#pone.0001659-Atkinson1">[48]</a>, <a href="#pone.0001659-Stein1">[49]</a> all the seeming drawbacks are trivial and probability (6) can be evaluated very readily.</p>


<h4>Transcriptome diversity and sampling growth curve</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>Based on SFD, one can deduce the estimator of transcriptome diversity and sampling growth curve in a systematical manner. According to the probability (1) or (2), when a sample of size <em>N</em> is sequenced, any transcript has probability <em>P</em>(0|<em>N</em>) to be missed. That is, there are<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e017&amp;representation=PNG" class="inline-graphic"><span class="note">(7)</span></span><br>unique transcripts are expected to be detected. Plugging any estimated <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e018&amp;representation=PNG" class="inline-graphic"></span> and <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e019&amp;representation=PNG" class="inline-graphic"></span> given by the fitted mixture into (7) yields a sampling growth curve. If we actually detect <em>s</em> unique transcripts when totally <em>N</em> transcript tags are sequenced, the total number of unique transcripts can be estimated as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e020&amp;representation=PNG" class="inline-graphic"><span class="note">(8)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>In addition, it is worth noting that RAD <em>φ</em>(<em>π</em>) has expectation <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e021&amp;representation=PNG" class="inline-graphic"></span>. This gives an alternative estimator of transcriptome diversity<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e022&amp;representation=PNG" class="inline-graphic"><span class="note">(9)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>Equation (7), (8) and (9) can be applied to any probability mixture. As BB and GP mixtures fit experimental data rather poorly, we only present results of GIGP mixture; yet that of BB and GP mixture can be written out in a similar way. Under GIGP mixture, according to (8) it is straightforward to obtain<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e023&amp;representation=PNG" class="inline-graphic"><span class="note">(10)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4"></a><p>As distribution (5) has mean <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e024&amp;representation=PNG" class="inline-graphic"></span>, according to (9) <em>S</em> can be estimated alternatively as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e025&amp;representation=PNG" class="inline-graphic"><span class="note">(11)</span></span><br></p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5"></a><p>In our study, estimator (10) and (11) give identical estimate to the second decimal. Using (11) as an estimator of <em>S</em>, one can write out an analytical form of the sampling growth curve under GIGP mixture according to (7) as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e026&amp;representation=PNG" class="inline-graphic"><span class="note">(12)</span></span><br></p>


<h4>Fitting probability mixture</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The parameters of SFD (3), (4) and (6) can be fitted using experimentally observed FOF data. Since the zero frequency bin <em>f</em><sub>0</sub> representing the number of undetected transcripts is unknown, the FOF {<em>f</em><sub>1</sub>, <em>f</em><sub>2</sub>,…} is actually drawn from the <em>zero-truncated</em> SFD given by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e027&amp;representation=PNG" class="inline-graphic"><span class="note">(13)</span></span><br></p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>In this study, we used maximum likelihood method to fit the parameters. The log-likelihood of {<em>f</em><sub>1</sub>, <em>f</em><sub>2</sub>,…} can be written as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e028&amp;representation=PNG" class="inline-graphic"><span class="note">(14)</span></span><br>where <em>θ</em> represents the general model parameter. We highly recommend to evaluate the probability involved in (14) using recurrence formula under each mixture, as in our experiences, directly evaluating high order Bessel function through easy mathematical routine often leads to computational overflow. The maximum likelihood estimation of model parameters can be computed by maximizing (14) numerically. Burrell and Fenton <a href="#pone.0001659-Burrell1">[50]</a> proposed to use derivative of log-likelihood in Quasi-Newton method to accelerate the maximizing procedure. In our experiences, taking advantage of modern computational power, direct maximization methods without using derivative information are efficient enough. In this study, we used the Nelder-Meed algorithm to maximize (14). The convergence was quite rapid.</p>


<h4>Monte Carlo simulation of transcriptome sampling</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Once SFD is fitted based on FOF data, RAD and transcriptome diversity <em>S</em> are determined simultaneously under the sampling model. Based on these parameters, one can carry out Monte Carlo simulation to <em>ab initio</em> imitate experimental sampling processes. At first, a virtual transcriptome with <em>S</em> transcripts indexed by 1,…,<em>S</em> is created. Relative abundances π<sub>1</sub>,…, <em>π<sub>S</sub></em> are randomly generated from fitted RAD and normalized to fulfill the constraint <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001659.e029&amp;representation=PNG" class="inline-graphic"></span>. A random number <em>r</em> is then chosen for each tag and its identity is determined by looking up <em>r</em> in a table of the cumulative sum of the simulated relative abundances. This ensures that the <em>i</em>th transcript has probability <em>π<sub>i</sub></em> to be detected. Repeatedly choosing <em>N</em> random numbers generates a virtual library of size <em>N</em>.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We appreciate Drs. Gane Ka-Shu Wong, Liping Wei and Hongyu Zhao for helpful suggestions and critical reading of the manuscript. We also thank all staffs in Beijing Institute of Genomics for their sincere support.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: JZ JY. Analyzed the data: JZ FH JW. Wrote the paper: JZ JY. Other: Responsible for project management: JW.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001659-Ruan1" id="pone.0001659-Ruan1"></a>Ruan Y, Le Ber P, Ng HH, Liu ET (2004) Interrogating the transcriptome. Trends Biotechnol  22: 23–30.  <ul class="find" data-citedArticleID="1061392" data-doi="10.1016/j.tibtech.2003.11.002"><li><a href="http://dx.doi.org/10.1016/j.tibtech.2003.11.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Interrogating+the+transcriptome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Interrogating+the+transcriptome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001659-Schena1" id="pone.0001659-Schena1"></a>Schena M, Shalon D, Davis RW, Brown PO (1995) Quantitative monitoring of gene expression patterns with a complementary DNA microarray. Science  270: 467–470.  <ul class="find" data-citedArticleID="1061396" data-doi="10.1126/science.270.5235.467"><li><a href="http://dx.doi.org/10.1126/science.270.5235.467" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Quantitative+monitoring+of+gene+expression+patterns+with+a+complementary+DNA+microarray." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Quantitative+monitoring+of+gene+expression+patterns+with+a+complementary+DNA+microarray.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001659-Lockhart1" id="pone.0001659-Lockhart1"></a>Lockhart DJ, Dong H, Byrne MC, Follettie MT, Gallo MV, et al.  (1996) Expression monitoring by hybridization to high-density oligonucleotide arrays. Nat Biotechnol  14: 1675–1680.  <ul class="find" data-citedArticleID="1061376" data-doi="10.1038/nbt1296-1675"><li><a href="http://dx.doi.org/10.1038/nbt1296-1675" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+monitoring+by+hybridization+to+high-density+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+monitoring+by+hybridization+to+high-density+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001659-Kapranov1" id="pone.0001659-Kapranov1"></a>Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, et al.  (2002) Large-scale transcriptional activity in chromosomes 21 and 22. Science  296: 916–919.  <ul class="find" data-citedArticleID="1061362" data-doi="10.1126/science.1068597"><li><a href="http://dx.doi.org/10.1126/science.1068597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+transcriptional+activity+in+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+transcriptional+activity+in+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001659-Bertone1" id="pone.0001659-Bertone1"></a>Bertone P, Stolc V, Royce TE, Rozowsky JS, Urban AE, et al.  (2004) Global identification of human transcribed sequences with genome tiling arrays. Science  306: 2242–2246.  <ul class="find" data-citedArticleID="1061328" data-doi="10.1126/science.1103388"><li><a href="http://dx.doi.org/10.1126/science.1103388" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+identification+of+human+transcribed+sequences+with+genome+tiling+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+identification+of+human+transcribed+sequences+with+genome+tiling+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001659-Cheng1" id="pone.0001659-Cheng1"></a>Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, et al.  (2005) Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution. Science  308: 1149–1154.  <ul class="find" data-citedArticleID="1061350" data-doi="10.1126/science.1108625"><li><a href="http://dx.doi.org/10.1126/science.1108625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001659-Ren1" id="pone.0001659-Ren1"></a>Ren B, Robert F, Wyrick JJ, Aparicio O, Jennings EG, et al.  (2000) Genome-wide location and function of DNA binding proteins. Science  290: 2306–2309.  <ul class="find" data-citedArticleID="1061390" data-doi="10.1126/science.290.5500.2306"><li><a href="http://dx.doi.org/10.1126/science.290.5500.2306" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+location+and+function+of+DNA+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+location+and+function+of+DNA+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001659-Adams1" id="pone.0001659-Adams1"></a>Adams MD, Kelley JM, Gocayne JD, Dubnick M, Polymeropoulos MH, et al.  (1991) Complementary DNA sequencing: expressed sequence tags and human genome project. Science  252: 1651–1656.  <ul class="find" data-citedArticleID="1061324" data-doi="10.1126/science.2047873"><li><a href="http://dx.doi.org/10.1126/science.2047873" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Complementary+DNA+sequencing%3A+expressed+sequence+tags+and+human+genome+project." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Complementary+DNA+sequencing%3A+expressed+sequence+tags+and+human+genome+project.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001659-Velculescu1" id="pone.0001659-Velculescu1"></a>Velculescu VE, Zhang L, Vogelstein B, Kinzler KW (1995) Serial analysis of gene expression. Science  270: 484–487.  <ul class="find" data-citedArticleID="1061418" data-doi="10.1126/science.270.5235.484"><li><a href="http://dx.doi.org/10.1126/science.270.5235.484" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Serial+analysis+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Serial+analysis+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001659-Saha1" id="pone.0001659-Saha1"></a>Saha S, Sparks A, Rago C, Akmaev V, Wang C, et al.  (2002) Using the transcriptome to annotate the genome. Nat Biotechnol  20: 508–512.  <ul class="find" data-citedArticleID="1061394" data-doi="10.1038/nbt0502-508"><li><a href="http://dx.doi.org/10.1038/nbt0502-508" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Using+the+transcriptome+to+annotate+the+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Using+the+transcriptome+to+annotate+the+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001659-Brenner1" id="pone.0001659-Brenner1"></a>Brenner S, Johnson M, Bridgham J, Golda G, Lloyd DH, et al.  (2000) Gene expression analysis by massively parallel signature sequencing (MPSS) on microbead arrays. Nat Biotechnol  18: 630–634.  <ul class="find" data-citedArticleID="1061340" data-doi="10.1038/76469"><li><a href="http://dx.doi.org/10.1038/76469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+analysis+by+massively+parallel+signature+sequencing+%28MPSS%29+on+microbead+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+analysis+by+massively+parallel+signature+sequencing+%28MPSS%29+on+microbead+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001659-Brenner2" id="pone.0001659-Brenner2"></a>Brenner S, Williams SR, Vermaas EH, Storck T, Moon K, et al.  (2000) In vitro cloning of complex mixtures of DNA on microbeads: physical separation of differentially expressed cDNAs. Proc Natl Acad Sci U S A  97: 1665–1670.  <ul class="find" data-citedArticleID="1061342" data-doi="10.1073/pnas.97.4.1665"><li><a href="http://dx.doi.org/10.1073/pnas.97.4.1665" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vitro+cloning+of+complex+mixtures+of+DNA+on+microbeads%3A+physical+separation+of+differentially+expressed+cDNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vitro+cloning+of+complex+mixtures+of+DNA+on+microbeads%3A+physical+separation+of+differentially+expressed+cDNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001659-Shiraki1" id="pone.0001659-Shiraki1"></a>Shiraki T, Kondo S, Katayama S, Waki K, Kasukawa T, et al.  (2003) Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage. Proc Natl Acad Sci U S A  100: 15776–15781.  <ul class="find" data-citedArticleID="1061400" data-doi="10.1073/pnas.2136655100"><li><a href="http://dx.doi.org/10.1073/pnas.2136655100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cap+analysis+gene+expression+for+high-throughput+analysis+of+transcriptional+starting+point+and+identification+of+promoter+usage." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cap+analysis+gene+expression+for+high-throughput+analysis+of+transcriptional+starting+point+and+identification+of+promoter+usage.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001659-Wei1" id="pone.0001659-Wei1"></a>Wei CL, Ng P, Chiu KP, Wong CH, Ang CC, et al.  (2004) 5′ Long serial analysis of gene expression (LongSAGE) and 3′ LongSAGE for transcriptome characterization and genome annotation. Proc Natl Acad Sci U S A  101: 11701–11706.  <ul class="find" data-citedArticleID="1061420" data-doi="10.1073/pnas.0403514101"><li><a href="http://dx.doi.org/10.1073/pnas.0403514101" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=5%E2%80%B2+Long+serial+analysis+of+gene+expression+%28LongSAGE%29+and+3%E2%80%B2+LongSAGE+for+transcriptome+characterization+and+genome+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%225%E2%80%B2+Long+serial+analysis+of+gene+expression+%28LongSAGE%29+and+3%E2%80%B2+LongSAGE+for+transcriptome+characterization+and+genome+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001659-Ng1" id="pone.0001659-Ng1"></a>Ng P, Wei CL, Sung WK, Chiu KP, Lipovich L, et al.  (2005) Gene identification signature (GIS) analysis for transcriptome characterization and genome annotation. Nat Methods  2: 105–111.  <ul class="find" data-citedArticleID="1061384" data-doi="10.1038/nmeth733"><li><a href="http://dx.doi.org/10.1038/nmeth733" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+identification+signature+%28GIS%29+analysis+for+transcriptome+characterization+and+genome+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+identification+signature+%28GIS%29+analysis+for+transcriptome+characterization+and+genome+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001659-Harbers1" id="pone.0001659-Harbers1"></a>Harbers M, Carninci P (2005) Tag-based approaches for transcriptome research and genome annotation. Nat Methods  2: 495–502.  <ul class="find" data-citedArticleID="1061356" data-doi="10.1038/nmeth768"><li><a href="http://dx.doi.org/10.1038/nmeth768" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tag-based+approaches+for+transcriptome+research+and+genome+annotation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tag-based+approaches+for+transcriptome+research+and+genome+annotation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001659-Margulies1" id="pone.0001659-Margulies1"></a>Margulies M, Egholm M, Altman WE, Attiya S, Bader JS, et al.  (2005) Genome sequencing in microfabricated high-density picolitre reactors. Nature  437: 376–380.  <ul class="find" data-citedArticleID="1061378"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.&amp;auth=&amp;atitle=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequencing+in+microfabricated+high-density+picolitre+reactors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequencing+in+microfabricated+high-density+picolitre+reactors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001659-Shendure1" id="pone.0001659-Shendure1"></a>Shendure J, Porreca GJ, Reppas NB, Lin X, McCutcheon JP, et al.  (2005) Accurate multiplex polony sequencing of an evolved bacterial genome. Science  309: 1728–1732.  <ul class="find" data-citedArticleID="1061398" data-doi="10.1126/science.1117389"><li><a href="http://dx.doi.org/10.1126/science.1117389" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Accurate+multiplex+polony+sequencing+of+an+evolved+bacterial+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001659-Blazej1" id="pone.0001659-Blazej1"></a>Blazej RG, Kumaresan P, Mathies RA (2006) Microfabricated bioprocessor for integrated nanoliter-scale Sanger DNA sequencing. Proc Natl Acad Sci U S A  103: 7240–7245.  <ul class="find" data-citedArticleID="1061334" data-doi="10.1073/pnas.0602476103"><li><a href="http://dx.doi.org/10.1073/pnas.0602476103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microfabricated+bioprocessor+for+integrated+nanoliter-scale+Sanger+DNA+sequencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microfabricated+bioprocessor+for+integrated+nanoliter-scale+Sanger+DNA+sequencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001659-Ng2" id="pone.0001659-Ng2"></a>Ng P, Tan JJ, Ooi HS, Lee YL, Chiu KP, et al.  (2006) Multiplex sequencing of paired-end ditags (MS-PET): a strategy for the ultra-high-throughput analysis of transcriptomes and genomes. Nucleic Acids Res  34: e84.  <ul class="find" data-citedArticleID="1061386" data-doi="10.1093/nar/gkl444"><li><a href="http://dx.doi.org/10.1093/nar/gkl444" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplex+sequencing+of+paired-end+ditags+%28MS-PET%29%3A+a+strategy+for+the+ultra-high-throughput+analysis+of+transcriptomes+and+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplex+sequencing+of+paired-end+ditags+%28MS-PET%29%3A+a+strategy+for+the+ultra-high-throughput+analysis+of+transcriptomes+and+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001659-Stern1" id="pone.0001659-Stern1"></a>Stern M, Anisimov S, Boheler K (2003) Can transcriptome size be estimated from SAGE catalogs? Binformatics  19: 443–448.  <ul class="find" data-citedArticleID="1061414" data-doi="10.1093/bioinformatics/btg018"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Can+transcriptome+size+be+estimated+from+SAGE+catalogs%3F" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Can+transcriptome+size+be+estimated+from+SAGE+catalogs%3F%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001659-Kuznetsov1" id="pone.0001659-Kuznetsov1"></a>Kuznetsov VA, Knott GD, Bonner RF (2002) General statistics of stochastic process of gene expression in eukaryotic cells. Genetics  161: 1321–1332.  <ul class="find" data-citedArticleID="1061372"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=General+statistics+of+stochastic+process+of+gene+expression+in+eukaryotic+cells.&amp;auth=&amp;atitle=General+statistics+of+stochastic+process+of+gene+expression+in+eukaryotic+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=General+statistics+of+stochastic+process+of+gene+expression+in+eukaryotic+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22General+statistics+of+stochastic+process+of+gene+expression+in+eukaryotic+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001659-Thygesen1" id="pone.0001659-Thygesen1"></a>Thygesen HH, Zwinderman AH (2006) Modeling Sage data with a truncated gamma-Poisson model. BMC Bioinformatics  7: 157.  <ul class="find" data-citedArticleID="1061416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Modeling+Sage+data+with+a+truncated+gamma-Poisson+model.&amp;auth=&amp;atitle=Modeling+Sage+data+with+a+truncated+gamma-Poisson+model." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+Sage+data+with+a+truncated+gamma-Poisson+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+Sage+data+with+a+truncated+gamma-Poisson+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001659-Boon1" id="pone.0001659-Boon1"></a>Boon K, Osorio EC, Greenhut SF, Schaefer CF, Shoemaker J, et al.  (2002) An anatomy of normal and malignant gene expression. Proc Natl Acad Sci U S A  99: 11287–11292.  <ul class="find" data-citedArticleID="1061336" data-doi="10.1073/pnas.152324199"><li><a href="http://dx.doi.org/10.1073/pnas.152324199" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+anatomy+of+normal+and+malignant+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+anatomy+of+normal+and+malignant+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001659-Khattra1" id="pone.0001659-Khattra1"></a>Khattra J, Delaney AD, Zhao Y, Siddiqui A, Asano J, et al.  (2007) Large-scale production of SAGE libraries from microdissected tissues, flow-sorted cells, and cell lines. Genome Res  17: 108–116.  <ul class="find" data-citedArticleID="1061366" data-doi="10.1101/gr.5488207"><li><a href="http://dx.doi.org/10.1101/gr.5488207" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+production+of+SAGE+libraries+from+microdissected+tissues%2C+flow-sorted+cells%2C+and+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+production+of+SAGE+libraries+from+microdissected+tissues%2C+flow-sorted+cells%2C+and+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001659-Hirst1" id="pone.0001659-Hirst1"></a>Hirst M, Delaney A, Rogers SA, Schnerch A, Persaud DR, et al.  (2007) LongSAGE profiling of nine human embryonic stem cell lines. Genome Biol  8: R113.  <ul class="find" data-citedArticleID="1061358" data-doi="10.1186/gb-2007-8-6-r113"><li><a href="http://dx.doi.org/10.1186/gb-2007-8-6-r113" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=LongSAGE+profiling+of+nine+human+embryonic+stem+cell+lines." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22LongSAGE+profiling+of+nine+human+embryonic+stem+cell+lines.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001659-Brandenberger1" id="pone.0001659-Brandenberger1"></a>Brandenberger R, Khrebtukova I, Thies RS, Miura T, Jingli C, et al.  (2004) MPSS profiling of human embryonic stem cells. BMC Dev Biol  4: 10.  <ul class="find" data-citedArticleID="1061338" data-doi="10.1186/1471-213x-4-10"><li><a href="http://dx.doi.org/10.1186/1471-213x-4-10" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MPSS+profiling+of+human+embryonic+stem+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MPSS+profiling+of+human+embryonic+stem+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001659-Carpenter1" id="pone.0001659-Carpenter1"></a>Carpenter MK, Rosler ES, Fisk GJ, Brandenberger R, Ares X, et al.  (2004) Properties of four human embryonic stem cell lines maintained in a feeder-free culture system. Dev Dyn  229: 243–258.  <ul class="find" data-citedArticleID="1061346" data-doi="10.1002/dvdy.10431"><li><a href="http://dx.doi.org/10.1002/dvdy.10431" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Properties+of+four+human+embryonic+stem+cell+lines+maintained+in+a+feeder-free+culture+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Properties+of+four+human+embryonic+stem+cell+lines+maintained+in+a+feeder-free+culture+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001659-Kent1" id="pone.0001659-Kent1"></a>Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, et al.  (2002) The human genome browser at UCSC. Genome Res  12: 996–1006.  <ul class="find" data-citedArticleID="1061364" data-doi="10.1101/gr.229102"><li><a href="http://dx.doi.org/10.1101/gr.229102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+human+genome+browser+at+UCSC." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+human+genome+browser+at+UCSC.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001659-Morris1" id="pone.0001659-Morris1"></a>Morris J, Baggerly K, Coombes K (2003) Bayesian shrinkage estimation of the relative abundance of mRNA transcripts using SAGE. Biometrics  59: 476–486.  <ul class="find" data-citedArticleID="1061380" data-doi="10.1111/1541-0420.00057"><li><a href="http://dx.doi.org/10.1111/1541-0420.00057" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bayesian+shrinkage+estimation+of+the+relative+abundance+of+mRNA+transcripts+using+SAGE." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bayesian+shrinkage+estimation+of+the+relative+abundance+of+mRNA+transcripts+using+SAGE.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001659-Bishop1" id="pone.0001659-Bishop1"></a>Bishop JO, Morton JG, Rosbash M, Richardson M (1974) Three abundance classes in HeLa cell messenger RNA. Nature  250: 199–204.  <ul class="find" data-citedArticleID="1061330" data-doi="10.1038/250199a0"><li><a href="http://dx.doi.org/10.1038/250199a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Three+abundance+classes+in+HeLa+cell+messenger+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Three+abundance+classes+in+HeLa+cell+messenger+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001659-Bishop2" id="pone.0001659-Bishop2"></a>Bishop JO (1978) Analysis of mRNA populations. Philos Trans R Soc Lond B Biol Sci  283: 373–374.  <ul class="find" data-citedArticleID="1061332"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+mRNA+populations.&amp;auth=&amp;atitle=Analysis+of+mRNA+populations." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+mRNA+populations." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+mRNA+populations.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001659-Holstege1" id="pone.0001659-Holstege1"></a>Holstege FC, Jennings EG, Wyrick JJ, Lee TI, Hengartner CJ, et al.  (1998) Dissecting the regulatory circuitry of a eukaryotic genome. Cell  95: 717–728.  <ul class="find" data-citedArticleID="1061360" data-doi="10.1016/s0092-8674(00)81641-4"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81641-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001659-Kim1" id="pone.0001659-Kim1"></a>Kim J, Bhinge AA, Morgan XC, Iyer VR (2005) Mapping DNA-protein interactions in large genomes by sequence tag analysis of genomic enrichment. Nat Methods  2: 47–53.  <ul class="find" data-citedArticleID="1061368" data-doi="10.1038/nmeth726"><li><a href="http://dx.doi.org/10.1038/nmeth726" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+DNA-protein+interactions+in+large+genomes+by+sequence+tag+analysis+of+genomic+enrichment." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+DNA-protein+interactions+in+large+genomes+by+sequence+tag+analysis+of+genomic+enrichment.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001659-Wei2" id="pone.0001659-Wei2"></a>Wei CL, Wu Q, Vega VB, Chiu KP, Ng P, et al.  (2006) A global map of p53 transcription-factor binding sites in the human genome. Cell  124: 207–219.  <ul class="find" data-citedArticleID="1061422" data-doi="10.1016/j.cell.2005.10.043"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.10.043" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+global+map+of+p53+transcription-factor+binding+sites+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+global+map+of+p53+transcription-factor+binding+sites+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001659-Kim2" id="pone.0001659-Kim2"></a>Kim N, Shin S, Lee S (2005) ECgene: genome-based EST clustering and gene modeling for alternative splicing. Genome Res  15: 566–576.  <ul class="find" data-citedArticleID="1061370" data-doi="10.1101/gr.3030405"><li><a href="http://dx.doi.org/10.1101/gr.3030405" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ECgene%3A+genome-based+EST+clustering+and+gene+modeling+for+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ECgene%3A+genome-based+EST+clustering+and+gene+modeling+for+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001659-Eyras1" id="pone.0001659-Eyras1"></a>Eyras E, Caccamo M, Curwen V, Clamp M (2004) ESTGenes: alternative splicing from ESTs in Ensembl. Genome Res  14: 976–987.  <ul class="find" data-citedArticleID="1061354" data-doi="10.1101/gr.1862204"><li><a href="http://dx.doi.org/10.1101/gr.1862204" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ESTGenes%3A+alternative+splicing+from+ESTs+in+Ensembl." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ESTGenes%3A+alternative+splicing+from+ESTs+in+Ensembl.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001659-Colinge1" id="pone.0001659-Colinge1"></a>Colinge J, Feger G (2001) Detecting the impact of sequencing errors on SAGE data. <em>Binformatics</em>  17: 840–842.  <ul class="find" data-citedArticleID="1061352" data-doi="10.1093/bioinformatics/17.9.840"><li><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.840" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Detecting+the+impact+of+sequencing+errors+on+SAGE+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Detecting+the+impact+of+sequencing+errors+on+SAGE+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001659-Chen1" id="pone.0001659-Chen1"></a>Chen J, Sun M, Lee S, Zhou G, Rowley JD, et al.  (2002) Identifying novel transcripts and novel genes in the human genome by using novel SAGE tags. Proc Natl Acad Sci U S A  99: 12257–12262.  <ul class="find" data-citedArticleID="1061348" data-doi="10.1073/pnas.192436499"><li><a href="http://dx.doi.org/10.1073/pnas.192436499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identifying+novel+transcripts+and+novel+genes+in+the+human+genome+by+using+novel+SAGE+tags." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identifying+novel+transcripts+and+novel+genes+in+the+human+genome+by+using+novel+SAGE+tags.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001659-Sorek1" id="pone.0001659-Sorek1"></a>Sorek R, Safer HM (2003) A novel algorithm for computational identification of contaminated EST libraries. Nucleic Acids Res  31: 1067–1074.  <ul class="find" data-citedArticleID="1061410" data-doi="10.1093/nar/gkg170"><li><a href="http://dx.doi.org/10.1093/nar/gkg170" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+algorithm+for+computational+identification+of+contaminated+EST+libraries." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+algorithm+for+computational+identification+of+contaminated+EST+libraries.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001659-Lee1" id="pone.0001659-Lee1"></a>Lee S, Clark T, Chen J, Zhou G, Scott LR, et al.  (2002) Correct identification of genes from serial analysis of gene expression tag sequences. Genomics  79: 598–602.  <ul class="find" data-citedArticleID="1061374" data-doi="10.1006/geno.2002.6730"><li><a href="http://dx.doi.org/10.1006/geno.2002.6730" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Correct+identification+of+genes+from+serial+analysis+of+gene+expression+tag+sequences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Correct+identification+of+genes+from+serial+analysis+of+gene+expression+tag+sequences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001659-Nam1" id="pone.0001659-Nam1"></a>Nam DK, Lee S, Zhou G, Cao X, Wang C, et al.  (2002) Oligo(dT) primer generates a high frequency of truncated cDNAs through internal poly(A) priming during reverse transcription. Proc Natl Acad Sci U S A  99: 6152–6156.  <ul class="find" data-citedArticleID="1061382" data-doi="10.1073/pnas.092140899"><li><a href="http://dx.doi.org/10.1073/pnas.092140899" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oligo%28dT%29+primer+generates+a+high+frequency+of+truncated+cDNAs+through+internal+poly%28A%29+priming+during+reverse+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oligo%28dT%29+primer+generates+a+high+frequency+of+truncated+cDNAs+through+internal+poly%28A%29+priming+during+reverse+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001659-Pauws1" id="pone.0001659-Pauws1"></a>Pauws E, van Kampen AH, van de Graaf SA, de Vijlder JJ, Ris-Stalpers C (2001) Heterogeneity in polyadenylation cleavage sites in mammalian mRNA sequences: implications for SAGE analysis. Nucleic Acids Res  29: 1690–1694.  <ul class="find" data-citedArticleID="1061388" data-doi="10.1093/nar/29.8.1690"><li><a href="http://dx.doi.org/10.1093/nar/29.8.1690" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneity+in+polyadenylation+cleavage+sites+in+mammalian+mRNA+sequences%3A+implications+for+SAGE+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneity+in+polyadenylation+cleavage+sites+in+mammalian+mRNA+sequences%3A+implications+for+SAGE+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001659-Silva1" id="pone.0001659-Silva1"></a>Silva AP, De Souza JE, Galante PA, Riggins GJ, De Souza SJ, et al.  (2004) The impact of SNPs on the interpretation of SAGE and MPSS experimental data. Nucleic Acids Res  32: 6104–6110.  <ul class="find" data-citedArticleID="1061408" data-doi="10.1093/nar/gkh937"><li><a href="http://dx.doi.org/10.1093/nar/gkh937" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+impact+of+SNPs+on+the+interpretation+of+SAGE+and+MPSS+experimental+data." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+impact+of+SNPs+on+the+interpretation+of+SAGE+and+MPSS+experimental+data.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001659-Sichel1" id="pone.0001659-Sichel1"></a>Sichel HS (1975) On a Distribution Law for Word Frequencies. <em>J Am Stat Assoc</em>  70: 542–547.  <ul class="find" data-citedArticleID="1061402" data-doi="10.1080/01621459.1975.10482469"><li><a href="http://dx.doi.org/10.1080/01621459.1975.10482469" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=On+a+Distribution+Law+for+Word+Frequencies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22On+a+Distribution+Law+for+Word+Frequencies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001659-Sichel2" id="pone.0001659-Sichel2"></a>Sichel HS (1986) Word Frequency Distributions and Type-Token Characteristics. Mathematical Scientist  11: 45–72.  <ul class="find" data-citedArticleID="1061404"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Word+Frequency+Distributions+and+Type-Token+Characteristics.&amp;auth=&amp;atitle=Word+Frequency+Distributions+and+Type-Token+Characteristics." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Word+Frequency+Distributions+and+Type-Token+Characteristics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Word+Frequency+Distributions+and+Type-Token+Characteristics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001659-Sichel3" id="pone.0001659-Sichel3"></a>Sichel HS (1992) Anatomy of the generalized inverse Gaussian-Poisson distribution with special applications to bibliometric studies. Inform <em>Process Manag</em>  28: 5–17.  <ul class="find" data-citedArticleID="1061406" data-doi="10.1016/0306-4573(92)90088-h"><li><a href="http://dx.doi.org/10.1016/0306-4573(92)90088-h" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Anatomy+of+the+generalized+inverse+Gaussian-Poisson+distribution+with+special+applications+to+bibliometric+studies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Anatomy+of+the+generalized+inverse+Gaussian-Poisson+distribution+with+special+applications+to+bibliometric+studies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001659-Atkinson1" id="pone.0001659-Atkinson1"></a>Atkinson AC (1982) Inference for Sichel's Compound Poisson Distribution. <em>J Am Stat Assoc</em>  77: 153–158.  <ul class="find" data-citedArticleID="1061326" data-doi="10.2307/2287782"><li><a href="http://dx.doi.org/10.2307/2287782" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inference+for+Sichel%27s+Compound+Poisson+Distribution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inference+for+Sichel%27s+Compound+Poisson+Distribution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001659-Stein1" id="pone.0001659-Stein1"></a>Stein GZ (1987) Parameter Estimation for the Sichel Distribution and Its Multivariate Extension. <em>J Am Stat Assoc</em>  82: 938–944.  <ul class="find" data-citedArticleID="1061412" data-doi="10.1080/01621459.1987.10478520"><li><a href="http://dx.doi.org/10.1080/01621459.1987.10478520" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Parameter+Estimation+for+the+Sichel+Distribution+and+Its+Multivariate+Extension." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Parameter+Estimation+for+the+Sichel+Distribution+and+Its+Multivariate+Extension.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001659-Burrell1" id="pone.0001659-Burrell1"></a>Burrell QL (1993) Yes, the GIGP really does work- and is workable! J Am Soc Inf Sci  44: 61–69.  <ul class="find" data-citedArticleID="1061344" data-doi="10.1002/(sici)1097-4571(199303)44:2<61::aid-asi1&gt;3.0.co;2-j"><li><a href="http://dx.doi.org/10.1002/(sici)1097-4571(199303)44:2<61::aid-asi1&gt;3.0.co;2-j" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yes%2C+the+GIGP+really+does+work-+and+is+workable%21" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yes%2C+the+GIGP+really+does+work-+and+is+workable%21%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.XML" value="103336"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.PDF" value="396272"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t001.PNG_L" value="112296"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t001.PNG_M" value="67645"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t001.PNG_S" value="9736"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t001.TIF" value="501264"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t001.PNG_I" value="21343"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e001.PNG" value="8380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e001.TIF" value="24332"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e002.PNG" value="9310"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e002.TIF" value="25080"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e003.PNG" value="6615"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e003.TIF" value="22532"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e004.PNG" value="5154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e004.TIF" value="21072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e005.PNG" value="6407"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e005.TIF" value="22428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e006.PNG" value="8625"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e006.TIF" value="24668"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e007.PNG" value="7586"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e007.TIF" value="23536"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e008.PNG" value="5133"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e008.TIF" value="21068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g001.PNG_L" value="405330"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g001.PNG_M" value="154010"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g001.PNG_S" value="13662"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g001.TIF" value="894034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g001.PNG_I" value="52546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g002.PNG_L" value="53883"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g002.PNG_M" value="96146"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g002.PNG_S" value="11125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g002.TIF" value="228790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g002.PNG_I" value="32224"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g003.PNG_L" value="41361"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g003.PNG_M" value="79165"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g003.PNG_S" value="11516"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g003.TIF" value="210548"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g003.PNG_I" value="27716"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t002.PNG_L" value="139379"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t002.PNG_M" value="183589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t002.PNG_S" value="22312"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t002.TIF" value="613690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.t002.PNG_I" value="66827"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g004.PNG_L" value="35401"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g004.PNG_M" value="76461"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g004.PNG_S" value="11069"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g004.TIF" value="176664"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g004.PNG_I" value="27545"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g005.PNG_L" value="42237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g005.PNG_M" value="86815"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g005.PNG_S" value="12368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g005.TIF" value="205056"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g005.PNG_I" value="30991"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g006.PNG_L" value="119958"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g006.PNG_M" value="63735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g006.PNG_S" value="14541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g006.TIF" value="359696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g006.PNG_I" value="58533"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g007.PNG_L" value="70727"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g007.PNG_M" value="70225"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g007.PNG_S" value="11526"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g007.TIF" value="188490"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.g007.PNG_I" value="24694"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e009.PNG" value="6754"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e009.TIF" value="22660"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e010.PNG" value="10726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e010.TIF" value="26492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e011.PNG" value="8819"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e011.TIF" value="24648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e012.PNG" value="12065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e012.TIF" value="27976"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e013.PNG" value="11775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e013.TIF" value="27276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e014.PNG" value="10787"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e014.TIF" value="26492"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e015.PNG" value="12132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e015.TIF" value="28072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e016.PNG" value="15453"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e016.TIF" value="31160"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e017.PNG" value="7955"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e017.TIF" value="24020"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e018.PNG" value="5114"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e018.TIF" value="21040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e019.PNG" value="6405"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e019.TIF" value="22400"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e020.PNG" value="7766"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e020.TIF" value="23708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e021.PNG" value="9223"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e021.TIF" value="24996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e022.PNG" value="6714"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e022.TIF" value="22656"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e023.PNG" value="12382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e023.TIF" value="28044"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e024.PNG" value="9837"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e024.TIF" value="25588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e025.PNG" value="9261"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e025.TIF" value="24980"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e026.PNG" value="13938"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e026.TIF" value="29848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e027.PNG" value="10144"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e027.TIF" value="25948"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e028.PNG" value="10848"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e028.TIF" value="27032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e029.PNG" value="6734"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001659.e029.TIF" value="22652"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001659&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001659" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001659&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001659&volume=&issue=&title=Modeling Transcriptome Based on Transcript-Sampling Data&author_name=Jiang%20Zhu%2C%20Fuhong%20He%2C%20Jing%20Wang%2C%20Jun%20Yu&start_page=1&end_page=9" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001659" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659&amp;t=Modeling%20Transcriptome%20Based%20on%20Transcript-Sampling%20Data" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659&title=Modeling%20Transcriptome%20Based%20on%20Transcript-Sampling%20Data&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659&amp;title=Modeling%20Transcriptome%20Based%20on%20Transcript-Sampling%20Data" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001659&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Modeling Transcriptome Based on Transcript-Sampling Data';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001659';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Modeling%20Transcriptome%20Based%20on%20Transcript-Sampling%20Data http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001659" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001659" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Binomials%22" title="Search for articles in the subject area:'Binomials'"><div class="flagText">Binomials</div></a>
              <div data-categoryid="32283" data-articleid="26144"
                   data-categoryname="Binomials"
                   class="flagImage" title="Flag 'Binomials' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Expressed+sequence+tags+analysis%22" title="Search for articles in the subject area:'Expressed sequence tags analysis'"><div class="flagText">Expressed sequence tags analysis</div></a>
              <div data-categoryid="23371" data-articleid="26144"
                   data-categoryname="Expressed sequence tags analysis"
                   class="flagImage" title="Flag 'Expressed sequence tags analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Monte+Carlo+method%22" title="Search for articles in the subject area:'Monte Carlo method'"><div class="flagText">Monte Carlo method</div></a>
              <div data-categoryid="34521" data-articleid="26144"
                   data-categoryname="Monte Carlo method"
                   class="flagImage" title="Flag 'Monte Carlo method' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Probability+distribution%22" title="Search for articles in the subject area:'Probability distribution'"><div class="flagText">Probability distribution</div></a>
              <div data-categoryid="32037" data-articleid="26144"
                   data-categoryname="Probability distribution"
                   class="flagImage" title="Flag 'Probability distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Relative+abundance+distribution%22" title="Search for articles in the subject area:'Relative abundance distribution'"><div class="flagText">Relative abundance distribution</div></a>
              <div data-categoryid="50703" data-articleid="26144"
                   data-categoryname="Relative abundance distribution"
                   class="flagImage" title="Flag 'Relative abundance distribution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+tagged+site+analysis%22" title="Search for articles in the subject area:'Sequence tagged site analysis'"><div class="flagText">Sequence tagged site analysis</div></a>
              <div data-categoryid="48851" data-articleid="26144"
                   data-categoryname="Sequence tagged site analysis"
                   class="flagImage" title="Flag 'Sequence tagged site analysis' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Serial+analysis+of+gene+expression%22" title="Search for articles in the subject area:'Serial analysis of gene expression'"><div class="flagText">Serial analysis of gene expression</div></a>
              <div data-categoryid="48515" data-articleid="26144"
                   data-categoryname="Serial analysis of gene expression"
                   class="flagImage" title="Flag 'Serial analysis of gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Transcriptome+analysis%22" title="Search for articles in the subject area:'Transcriptome analysis'"><div class="flagText">Transcriptome analysis</div></a>
              <div data-categoryid="40151" data-articleid="26144"
                   data-categoryname="Transcriptome analysis"
                   class="flagImage" title="Flag 'Transcriptome analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=1258'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=6829'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1473&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=41989">A start to more computational analyses of the transcriptome</a><br>Posted by JoachimJacob</p>
      <p><a href="/annotation/listThread.action?root=20369">Publisher's Note: Updated Figure 5</a><br>Posted by PLoS_ONE_Group</p>
      <p><a href="/annotation/listThread.action?root=21929">Referee comments: Referee 1 (Christian Iseli)</a><br>Posted by PLoS_ONE_Group</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
