

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000355"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000355" />

    <meta name="citation_title" content="Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes"/>
    <meta itemprop="name" content="Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes"/>

      <meta name="citation_author" content="Leonor Saiz"/>
            <meta name="citation_author_institution" content="Integrative Biological Modeling Laboratory, Computational Biology Program, Memorial Sloan-Kettering Cancer Center, New York, New York, United States of America"/>
      <meta name="citation_author" content="Jose M.G. Vilar"/>
            <meta name="citation_author_institution" content="Integrative Biological Modeling Laboratory, Computational Biology Program, Memorial Sloan-Kettering Cancer Center, New York, New York, United States of America"/>

    <meta name="citation_date" content="2007/4/4"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000355.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e355"/>
    <meta name="citation_issue" content="4"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Multipartite genetic control elements: communication by DNA loop.; citation_author=S Adhya; citation_journal_title=Annu Rev Genet; citation_volume=23; citation_number=1; citation_pages=227-250; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=DNA Looping.; citation_author=KS Matthews; citation_journal_title=Microbiological Reviews; citation_volume=56; citation_number=2; citation_pages=123-136; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=DNA looping.; citation_author=R Schleif; citation_journal_title=Annu Rev Biochem; citation_volume=61; citation_number=3; citation_pages=199-223; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=DNA looping in gene regulation: from the assembly of macromolecular complexes to the control of transcriptional noise.; citation_author=JMG Vilar; citation_author=L Saiz; citation_journal_title=Current Opinion in Genetics & Development; citation_volume=15; citation_number=4; citation_pages=136-144; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Chromatin looping and the probability of transcription.; citation_author=QL Li; citation_author=G Barkess; citation_author=H Qian; citation_journal_title=Trends in Genetics; citation_volume=22; citation_number=5; citation_pages=197-202; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Spreading of silent chromatin: inaction at a distance.; citation_author=PB Talbert; citation_author=S Henikoff; citation_journal_title=Nature Reviews Genetics; citation_volume=7; citation_number=6; citation_pages=793-803; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Macromolecular crowding: obvious but underappreciated.; citation_author=RJ Ellis; citation_journal_title=Trends in Biochemical Sciences; citation_volume=26; citation_number=7; citation_pages=597-604; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Physical-Properties of DNA Invivo as Probed by the Length Dependence of the Lac Operator Looping Process.; citation_author=GR Bellomy; citation_author=MC Mossing; citation_author=MT Record; citation_journal_title=Biochemistry; citation_volume=27; citation_number=8; citation_pages=3900-3906; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=DNA looping and physical constraints on transcription regulation.; citation_author=JMG Vilar; citation_author=S Leibler; citation_journal_title=J Mol Biol; citation_volume=331; citation_number=9; citation_pages=981-989; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Inferring the in vivo looping properties of DNA.; citation_author=L Saiz; citation_author=JM Rubi; citation_author=JMG Vilar; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=10; citation_pages=17642-17645; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Repression of lac promoter as a function of distance, phase and quality of an auxiliary lac operator.; citation_author=J Muller; citation_author=S Oehler; citation_author=B Muller-Hill; citation_journal_title=J Mol Biol; citation_volume=257; citation_number=11; citation_pages=21-29; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=In vivo DNA loops in araCBAD: size limits and helical repeat.; citation_author=DH Lee; citation_author=RF Schleif; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=86; citation_number=12; citation_pages=476-480; citation_date=1989; " />
      <meta name="citation_reference" content="citation_author=M Ptashne; citation_author=A Gann; citation_number=13; citation_pages=xvi, 192 p; citation_date=2002; citation_publisher=Cold Spring Harbor Laboratory Press; " />
      <meta name="citation_reference" content="citation_author=VA Bloomfield; citation_author=DM Crothers; citation_author=I Tinoco; citation_number=14; citation_pages=672 p; citation_date=2000; citation_publisher=University Science Books; " />
      <meta name="citation_reference" content="citation_title=Localized single-stranded bubble mechanism for cyclization of short double helix DNA.; citation_author=J Yan; citation_author=JF Marko; citation_journal_title=Phys Rev Lett; citation_volume=93; citation_number=15; citation_pages=108108; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Implications of the dependence of the elastic properties of DNA on nucleotide sequence.; citation_author=WK Olson; citation_author=D Swigon; citation_author=BD Coleman; citation_journal_title=Philosophical Transactions of the Royal Society of London Series a-Mathematical Physical and Engineering Sciences; citation_volume=362; citation_number=16; citation_pages=1403-1422; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Structural dynamics of the lac repressor-DNA complex revealed by a multiscale simulation.; citation_author=E Villa; citation_author=A Balaeff; citation_author=K Schulten; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=17; citation_pages=6783-6788; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Two are better than one.; citation_author=SE Halford; citation_author=DM Gowers; citation_author=RB Sessions; citation_journal_title=Nature Structural Biology; citation_volume=7; citation_number=18; citation_pages=705-707; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A gamut of loops: meandering DNA.; citation_author=S Semsey; citation_author=K Virnik; citation_author=S Adhya; citation_journal_title=Trends Biochem Sci; citation_volume=30; citation_number=19; citation_pages=334-341; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Statistical-mechanical theory of DNA looping.; citation_author=Y Zhang; citation_author=AE McEwen; citation_author=DM Crothers; citation_author=SD Levene; citation_journal_title=Biophys J; citation_volume=90; citation_number=20; citation_pages=1903-1912; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Alternative geometries of DNA looping: an analysis using the SfiI endonuclease.; citation_author=MA Watson; citation_author=DM Gowers; citation_author=SE Halford; citation_journal_title=J Mol Biol; citation_volume=298; citation_number=21; citation_pages=461-475; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Bacterial repression loops require enhanced DNA flexibility.; citation_author=NA Becker; citation_author=JD Kahn; citation_author=LJ Maher; citation_journal_title=Journal of Molecular Biology; citation_volume=349; citation_number=22; citation_pages=716-730; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Modeling the Lac repressor-operator assembly: The influence of DNA looping on Lac repressor conformation.; citation_author=D Swigon; citation_author=BD Coleman; citation_author=WK Olson; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=103; citation_number=23; citation_pages=9879-9884; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Architectural elements in nucleoprotein complexes: interchangeability of specific and non-specific DNA binding proteins.; citation_author=AM Segall; citation_author=SD Goodman; citation_author=HA Nash; citation_journal_title=Embo J; citation_volume=13; citation_number=24; citation_pages=4536-4548; citation_date=1994; " />
      <meta name="citation_reference" content="citation_author=TL Hill; citation_number=25; citation_pages=508 p; citation_date=1960; citation_publisher=Addison-Wesley Pub. Co; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of the lactose operon repressor and its complexes with DNA and inducer.; citation_author=M Lewis; citation_author=G Chang; citation_author=NC Horton; citation_author=MA Kercher; citation_author=HC Pace; citation_journal_title=Science; citation_volume=271; citation_number=26; citation_pages=1247-1254; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of lac repressor core tetramer and its implications for DNA looping.; citation_author=AM Friedman; citation_author=TO Fischmann; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=268; citation_number=27; citation_pages=1721-1727; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Conformation of Lac repressor tetramer in solution, bound and unbound to operator DNA.; citation_author=GC Ruben; citation_author=TB Roos; citation_journal_title=Microsc Res Tech; citation_volume=36; citation_number=28; citation_pages=400-416; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Fluorescence resonance energy transfer over approximately 130 basepairs in hyperstable lac repressor-DNA loops.; citation_author=LM Edelman; citation_author=R Cheong; citation_author=JD Kahn; citation_journal_title=Biophys J; citation_volume=84; citation_number=29; citation_pages=1131-1145; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Higher-Order Nucleoprotein Complexes in Transcription - Analogies with Site-Specific Recombination.; citation_author=R Grosschedl; citation_journal_title=Current Opinion in Cell Biology; citation_volume=7; citation_number=30; citation_pages=362-370; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Architectural transcription factors: proteins that remodel DNA.; citation_author=MH Werner; citation_author=SK Burley; citation_journal_title=Cell; citation_volume=88; citation_number=31; citation_pages=733-736; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=HMG1 and 2, and related ‘architectural’ DNA-binding proteins.; citation_author=JO Thomas; citation_author=AA Travers; citation_journal_title=Trends Biochem Sci; citation_volume=26; citation_number=32; citation_pages=167-174; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA-Sequence Determinants of Cap-Induced Bending and Protein-Binding Affinity.; citation_author=MR Gartenberg; citation_author=DM Crothers; citation_journal_title=Nature; citation_volume=333; citation_number=33; citation_pages=824-829; citation_date=1988; " />
      <meta name="citation_reference" content="citation_title=Crystal-Structure of a Cap-DNA Complex - the DNA Is Bent by 90-Degrees.; citation_author=SC Schultz; citation_author=GC Shields; citation_author=TA Steitz; citation_journal_title=Science; citation_volume=253; citation_number=34; citation_pages=1001-1007; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Crystal structure of an IHF-DNA complex: a protein-induced DNA U-turn.; citation_author=PA Rice; citation_author=S Yang; citation_author=K Mizuuchi; citation_author=HA Nash; citation_journal_title=Cell; citation_volume=87; citation_number=35; citation_pages=1295-1306; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cooperative Interactions between the Catabolite Gene Activator Protein and the Lac Repressor at the Lactose Promoter.; citation_author=JM Hudson; citation_author=MG Fried; citation_journal_title=Journal of Molecular Biology; citation_volume=214; citation_number=36; citation_pages=381-396; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Structural basis for cooperative DNA binding by CAP and lac repressor.; citation_author=A Balaeff; citation_author=L Mahadevan; citation_author=K Schulten; citation_journal_title=Structure; citation_volume=12; citation_number=37; citation_pages=123-132; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gal repressosome contains an antiparallel DNA loop.; citation_author=M Geanacopoulos; citation_author=G Vasmatzis; citation_author=VB Zhurkin; citation_author=S Adhya; citation_journal_title=Nat Struct Biol; citation_volume=8; citation_number=38; citation_pages=432-436; citation_date=2001; " />
      <meta name="citation_reference" content="citation_author=AD Bates; citation_author=A Maxwell; citation_number=39; citation_date=2005; citation_publisher=Oxford University Press; " />
      <meta name="citation_reference" content="citation_title=DNA twisting flexibility and the formation of sharply looped protein-DNA complexes.; citation_author=TE Cloutier; citation_author=J Widom; citation_journal_title=Proceedings of the National Academy of Sciences of the United States of America; citation_volume=102; citation_number=40; citation_pages=3645-3650; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Stochastic dynamics of macromolecular-assembly networks.; citation_author=L Saiz; citation_author=JMG Vilar; citation_journal_title=Mol Syst Biol; citation_volume=2; citation_number=41; citation_pages=2006.0024; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Multiprotein DNA looping.; citation_author=JMG Vilar; citation_author=L Saiz; citation_journal_title=Phys Rev Lett; citation_volume=96; citation_number=42; citation_pages=238103; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes"/>
    <meta name="twitter:description" content="Protein-DNA complexes with loops play a fundamental role in a wide variety of cellular processes, ranging from the regulation of DNA transcription to telomere maintenance. As ubiquitous as they are, their precise in vivo properties and their integration into the cellular function still remain largely unexplored. Here, we present a multilevel approach that efficiently connects in both directions molecular properties with cell physiology and use it to characterize the molecular properties of the looped DNA-lac repressor complex while functioning in vivo. The properties we uncover include the presence of two representative conformations of the complex, the stabilization of one conformation by DNA architectural proteins, and precise values of the underlying twisting elastic constants and bending free energies. Incorporation of all this molecular information into gene-regulation models reveals an unprecedented versatility of looped DNA-protein complexes at shaping the properties of gene expression."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000355.g004"/>

  <meta property="og:title" content="Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=7139'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=3654'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2068&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000355">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000355" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000355&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Multilevel Deconstruction of the <i>In Vivo</i> Behavior of Looped DNA-Protein Complexes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Leonor Saiz, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Integrative Biological Modeling Laboratory, Computational Biology Program, Memorial Sloan-Kettering Cancer Center, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jose M.G. Vilar
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:vilar@cbio.mskcc.org">vilar@cbio.mskcc.org</a></p>

                <p>Affiliation:
                  Integrative Biological Modeling Laboratory, Computational Biology Program, Memorial Sloan-Kettering Cancer Center, New York, New York, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: April 04, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000355</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000355-g001" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000355-g002" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000355-t001" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000355-g003" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000355-g004" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000355">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000355" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Protein-DNA complexes with loops play a fundamental role in a wide variety of cellular processes, ranging from the regulation of DNA transcription to telomere maintenance. As ubiquitous as they are, their precise <em>in vivo</em> properties and their integration into the cellular function still remain largely unexplored. Here, we present a multilevel approach that efficiently connects in both directions molecular properties with cell physiology and use it to characterize the molecular properties of the looped DNA-<em>lac</em> repressor complex while functioning <em>in vivo</em>. The properties we uncover include the presence of two representative conformations of the complex, the stabilization of one conformation by DNA architectural proteins, and precise values of the underlying twisting elastic constants and bending free energies. Incorporation of all this molecular information into gene-regulation models reveals an unprecedented versatility of looped DNA-protein complexes at shaping the properties of gene expression.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Saiz L, Vilar JM (2007) Multilevel Deconstruction of the <em>In Vivo</em> Behavior of Looped DNA-Protein Complexes. PLoS ONE 2(4):
          e355.
            doi:10.1371/journal.pone.0000355</p><p><strong>Academic Editor: </strong>Martin Egli, Vanderbilt University, United States of America</p><p><strong>Received:</strong> December 18, 2006; <strong>Accepted:</strong> March 14, 2007; <strong>Published:</strong> April 4, 2007</p><p><strong>Copyright:</strong> © 2007 Saiz, Vilar. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>The authors have no support or funding to report.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Many fundamental cellular processes, including transcription regulation, recombination, replication, gene silencing, and telomere maintenance, rely on the formation of DNA loops and higher order looped DNA packing structures, such as chromatin looping <a href="#pone.0000355-Adhya1">[1]</a>–<a href="#pone.0000355-Talbert1">[6]</a>. In the regulation of gene expression, proteins bound far away from the genes they control can be brought to the initiation of transcription region by looping the intervening DNA. The free energy cost of this process determines how easily DNA loops can form and therefore the extent to which distal DNA sites affect each other <a href="#pone.0000355-Vilar1">[4]</a>. Assessing directly the <em>in vivo</em> value of the free energy of DNA looping is remarkably difficult, not only because the properties of the components can change when studied <em>in vitro</em>, but also because the <em>in vivo</em> probing of the cell can perturb the process under study <a href="#pone.0000355-Ellis1">[7]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Computational and mathematical models of gene regulation provide an avenue to connect the physical properties of DNA in its <em>in vivo</em> natural environment with the resulting cellular behavior <a href="#pone.0000355-Bellomy1">[8]</a>, <a href="#pone.0000355-Vilar2">[9]</a>. This type of approach was used recently to infer the <em>in vivo</em> free energies of DNA looping by the <em>lac</em> repressor as a function of the loop length <a href="#pone.0000355-Saiz1">[10]</a> from measurements of enzyme production in the <em>lac</em> operon <a href="#pone.0000355-Muller1">[11]</a>, which proved to be a very accurate alternative to obtain molecular properties of the macromolecular complexes <em>in vivo</em>. The results of this analysis <a href="#pone.0000355-Saiz1">[10]</a> showed that the free energy for short loops oscillates with the helical periodicity of DNA, as expected, because the operators must have the right phase to bind simultaneously to the repressor <a href="#pone.0000355-Bellomy1">[8]</a>, <a href="#pone.0000355-Lee1">[12]</a> and, unexpectedly, that the free energy in a cycle behaves asymmetrically. A Fourier analysis of the oscillations indicated that this asymmetry can be characterized by a second representative oscillatory component with a period of ~5.6 bp, in addition to the component with the <em>in vivo</em> helical period (~10.9 bp). Another striking feature of the <em>in vivo</em> free energy of looping is that the amplitude of the oscillations is as small as ~2.5 kcal/mol, similar to the typical free energy of cooperative interactions between regulatory molecules <a href="#pone.0000355-Ptashne1">[13]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Uncovering the origin of the <em>in vivo</em> properties is important for understanding DNA looping and its effects in gene regulation, especially because current theories based on semiflexible polymer models of DNA predict symmetric and, at least, twice as large oscillations <a href="#pone.0000355-Bloomfield1">[14]</a>, <a href="#pone.0000355-Yan1">[15]</a>. Different contributions, such as the anisotropic flexibility of DNA, local features resulting from the DNA sequence <a href="#pone.0000355-Olson1">[16]</a>, and interactions with the <em>lac</em> repressor <a href="#pone.0000355-Villa1">[17]</a> and other DNA binding proteins, might be at play. Another potential source of complexity is the number of trajectories that DNA can follow to loop <a href="#pone.0000355-Halford1">[18]</a>–<a href="#pone.0000355-Watson1">[21]</a>. Thus, the observed behavior could be the result of loops with several representative conformations (<a href="#pone-0000355-g001">Figure 1</a>). Yet, only the lowest free energy conformation is typically considered.</p>
<div class="figure" id="pone-0000355-g001"><div class="img"><a name="pone-0000355-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1"></a><p>Two plausible alternative loop conformations of the <em>lac</em> repressor-DNA complex. The bidentate repressor, with the two dimers that form the functional tetramer shown in red, simultaneously binds DNA, colored orange, at two sites. The two structures represent two plausible trajectories of the DNA loop and two plausible conformations of the <em>lac</em> repressor (V-shaped and extended).</p>
<span>doi:10.1371/journal.pone.0000355.g001</span></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Here, we develop a statistical thermodynamics approach to deconstruct the observed behavior of the expression of the <em>lac</em> operon in <em>Escherichia coli</em> cells and use it to obtain the <em>in vivo</em> properties of DNA looping by the <em>lac</em> repressor at different levels of cellular organization. At the molecular level, we propose an elastic model for DNA loop formation that considers multiple structures of the DNA-protein complex and show that, at the cellular level, the <em>in vivo</em> behavior of the free energy of looping is accurately accounted for by the presence of two distinct types of DNA loops, corresponding to two main looped DNA-protein conformations, with different relative optimal free energies, phases, and interactions with key architectural proteins. We explore in detail the effects of multiple conformations on shaping the free energy of looping DNA and the consequences that resulting free energies have for gene regulation at the cell-population level.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A multi-conformation elastic DNA model</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We consider that the DNA loop can be in two distinct representative conformations (<a href="#pone-0000355-g001">Figure 1</a>) through the free energy of looping Δ<em>G<sub>l</sub></em>, which can be expressed in terms of the free energy of each of the conformations as (see <a href="#s4">Methods</a>)<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e001&amp;representation=PNG" class="inline-graphic"><span class="note">1</span></span><br>where the index <em>i</em> indicates whether the loop is in the conformation labeled 1 or 2 and <em>RT</em> (≈0.6 kcal/mol) is the gas constant, <em>R</em>, times the absolute temperature, <em>T</em>. The integer index <em>n</em> ranges from-infinity to +infinity and accounts for the 2<em>π</em> degeneracy in the twisting angle. In general, a system could have <em>M</em> representative conformations of the nucleoprotein-DNA complex and the summation in the previous expression of <em>i</em> would extend from 1 to <em>M</em> (see <a href="#s4">Methods</a> for the general case).</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>The free energy of a particular state includes bending and twisting contributions and is given following the classic elasticity theory of DNA <a href="#pone.0000355-Bloomfield1">[14]</a> by<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e002&amp;representation=PNG" class="inline-graphic"><span class="note">2</span></span><br>where <em>L</em> is the length of the loop (in bp), <em>L<sub>opt,i</sub></em> is the optimal spacing or phase (in bp), and Δ<em>G<sub>0,i</sub></em> is the corresponding optimal free energy (in kcal/mol), which depends on the type (<em>i</em>) of loop formed. In principle, the term Δ<em>G<sub>0,i</sub></em> could also depend on <em>L</em> because of the bending contribution <a href="#pone.0000355-Bloomfield1">[14]</a> but the <em>in vivo</em> results <a href="#pone.0000355-Saiz1">[10]</a> indicate that it is practically constant for the range of lengths analyzed. The twisting force constant (in kcal/mol bp), <em>C</em>, and the <em>in vivo</em> helical repeat (in bp), <em>hr</em>, are considered here to be the same for the two types of loops. The free energy Δ<em>G<sub>i</sub></em> of a conformation <em>i</em> is given by the equality <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e003&amp;representation=PNG" class="inline-graphic"></span>, which includes the sum over the states of a loop conformation.</p>


<h4><em>In vivo</em> free energy of DNA looping: complex average behavior from simple individual contributions</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>The free energy of looping Δ<em>G<sub>l</sub></em> given by Equations 1 and 2 closely reproduces the broad range of observed types of behavior (<a href="#pone-0000355-g002">Figure 2</a>), which consist of the <em>in vivo</em> free energies of looping DNA <a href="#pone.0000355-Saiz1">[10]</a> obtained from the measured repression levels (see <a href="#s4">Methods</a>) for two wild type situations <a href="#pone.0000355-Muller1">[11]</a>, <a href="#pone.0000355-Becker1">[22]</a> and a mutant lacking the architectural HU protein <a href="#pone.0000355-Becker1">[22]</a>. The <em>in vivo</em> free energies display not only asymmetric oscillations with reduced amplitude but also plateaus and secondary maxima. Therefore, our model indicates that the complex behavior of DNA looping <em>in vivo</em> emerges from a combination of the simple behavior of the individual conformations rather than from the individual conformations themselves.</p>
<div class="figure" id="pone-0000355-g002"><div class="img"><a name="pone-0000355-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Two-conformation analysis of the <em>in vivo</em> free energy of DNA looping. The <em>in vivo</em> free energy of looping DNA by the <em>lac</em> repressor (blue symbols) was obtained as described in Saiz et al. <a href="#pone.0000355-Saiz1">[10]</a> (see also <a href="#s4">Methods</a>) from the measured repression levels of Muller et al. <a href="#pone.0000355-Muller1">[11]</a> for wild type (WT1) and of Becker et al. <a href="#pone.0000355-Becker1">[22]</a> for wild type (WT2) and a mutant that does not express the architectural HU protein (ΔHU). As repression levels in the absence of looping (see <a href="#s4">Methods</a> and Saiz et al. <a href="#pone.0000355-Saiz1">[10]</a>) we have used 135 (WT1), 2.3 (WT2), and 1.7 (ΔHU). The thick black continuous lines correspond in each case to the best fit to the free energy Δ<em>G<sub>l</sub></em> given by Equations 1 and 2, which considers the contributions of two looped conformations. The contributions of each conformation are shown separately as red (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e004&amp;representation=PNG" class="inline-graphic"></span>) and black (<span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e005&amp;representation=PNG" class="inline-graphic"></span>) dashed lines. The values of the parameters for the best fit are shown in <a href="#pone-0000355-t001">Table 1</a>.</p>
<span>doi:10.1371/journal.pone.0000355.g002</span></div><div class="figure" id="pone-0000355-t001"><div class="img"><a name="pone-0000355-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000355.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000355.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span><em>In vivo</em> values of the molecular parameters of the looped DNA-<em>lac</em> repressor complex.</span></strong></p><span>doi:10.1371/journal.pone.0000355.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The values of the parameters for the best fit (continuous black thick curves) to the data inferred from the experiments (blue symbols) are gathered in <a href="#pone-0000355-t001">Table 1</a>. The free energy of looping for each conformation (<a href="#pone-0000355-g002">Figure 2</a> in dashed red and gray for the conformation with lowest and highest optimal free energy, respectively) depends on the length of the loop as expected for an elastic rod model of DNA, displaying symmetric oscillations with the periodicity of the DNA helix and relatively high amplitudes. The magnitudes of the amplitudes, in the order of 5 kcal/mol, are in excellent agreement with recent sequence-dependent DNA elasticity calculations for different types of <em>lac</em> repressor-DNA loops <a href="#pone.0000355-Swigon1">[23]</a>, which lead to oscillations of ~6 kcal/mol. Another interesting feature is the lack of a sharp increase of the looping free energy for short loops, which would be expected to arise from the bending free energy contribution. The observed behavior might originate from the high flexibility of the repressor <a href="#pone.0000355-Villa1">[17]</a> in the extended conformations <a href="#pone.0000355-Swigon1">[23]</a> or from the interaction of the DNA loop with architectural proteins that help bending <a href="#pone.0000355-Segall1">[24]</a>.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>In both wild type situations analyzed (<a href="#pone-0000355-g002">Figure 2</a>, WT1 and WT2), the presence of two looped conformations (one more stable than the other by 1.0 kcal/mol and with shifts in the optimal phases of 4.3 bp or −4.2 bp) is responsible for the reduced amplitude of the oscillations and the asymmetry, including secondary maxima and/or shoulders, of the free energy curves. The inferred <em>in vivo</em> data from the two experiments is in excellent agreement with the two-conformation analysis (compare experimental blue symbols and model black thick lines in <a href="#pone-0000355-g002">Figure 2</a>). Our results indicate that the behavior of the <em>in vivo</em> system depends strongly on the properties of the different loop conformations, especially on the optimal free energies and optimal phases (<a href="#pone-0000355-t001">Table 1</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Note that optimal phases and free energies between the two conformations are different for different wild type experiments (WT1 and WT2 in <a href="#pone-0000355-t001">Table 1</a>). These differences might arise from the differences in the experimental conditions, which are significant. For instance, the repression level in the absence of DNA looping is 135 for WT1 and 2.3 for WT2. They can also be due to potentially different boundary conditions because the loop is formed between the ideal and the main operator O<sub>1</sub> in WT1 and between the ideal and the auxiliary operator O<sub>2</sub> in WT2. The main operator is both more symmetric and ~10 times stronger than O<sub>2</sub>.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5"></a><p>Optimal energies and phases determine the relative contributions of the different conformations to the observed behavior and how they change with the length of the loop. Explicitly, the probabilities for each conformation to be present, <em>P</em><sub>1</sub> and <em>P</em><sub>2</sub>, are related to each other through the expression <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e006&amp;representation=PNG" class="inline-graphic"></span>, which results from the general principles of statistical thermodynamics <a href="#pone.0000355-Hill1">[25]</a>. As the distance between the two operators is changed, the less stable loop can become the most stable one. In some cases, such as for those loop lengths for which both conformations have the same free energies (when red and gray curves in <a href="#pone-0000355-g002">Figure 2</a> intersect each other), the two structures are equally probable and both conformations alternate in time in a single cell and occur simultaneously in a population of cells. In the other cases, when the difference is larger than <em>RT</em>, the conformation with the lowest free energy dominates over the other one.</p>
<a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6"></a><p>These two conformations of the DNA-protein looped complex, whose elastic properties we have characterized in detail, could consist of two ways of binding of the repressor to DNA, such as antiparallel and parallel DNA trajectories, which for a specific repressor conformation, i.e., the typical V-shape observed in the crystalline state <a href="#pone.0000355-Lewis1">[26]</a>, <a href="#pone.0000355-Friedman1">[27]</a>, would give rise in principle to four different loop geometries <a href="#pone.0000355-Semsey1">[19]</a>. Similarly, they could correspond to two different conformations of the <em>lac</em> repressor; namely, the V-shaped repressor and the extended conformation proposed from electron microscopy and fluorescence resonance energy transfer experiments in solution <a href="#pone.0000355-Ruben1">[28]</a>, <a href="#pone.0000355-Edelman1">[29]</a>.</p>


<h4>Effects of architectural proteins</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The free energy of looping DNA <em>in vivo</em> determines the cost of forming the loop in the natural environment of the cell, which includes the double-stranded DNA molecule, the proteins that tie the DNA loops, other DNA binding proteins, and the different proteins confined within the <em>E. coli</em> cell. Architectural proteins both in eukaryotes and prokaryotes play an important role in assisting the assembly of nucleoprotein complexes and contribute to the control of gene expression as well as other DNA transactions <a href="#pone.0000355-Grosschedl1">[30]</a>–<a href="#pone.0000355-Thomas1">[32]</a>. These proteins locally bend or kink DNA facilitating the formation of protein-DNA looped structures <a href="#pone.0000355-Gartenberg1">[33]</a>–<a href="#pone.0000355-Rice1">[35]</a> and thus are expected to affect the DNA looping properties <em>in vivo</em>. In particular, the stability of different types of looped DNA-<em>lac</em> repressor conformations has been shown to be affected by binding of the catabolite activator protein <a href="#pone.0000355-Hudson1">[36]</a>, <a href="#pone.0000355-Balaeff1">[37]</a>. Other bacterial architectural proteins, such as the heat unstable nucleoid protein (HU) also referred to as histone-like protein, do not have sequence specific DNA binding sites but also bend DNA.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>In the <em>E. coli</em> mutant without architectural HU protein (<a href="#pone-0000355-g002">Figure 2</a>, ΔHU), the <em>in vivo</em> free energy of DNA looping is compatible with the presence of two loop conformations that are similarly stable (0.2 kcal/mol difference) but have different optimal phases. In this case, the phase shift (3.3 bp) also leads to reduced amplitude of the oscillations, as in the wild-type case where HU protein is normally expressed, yet the asymmetric behavior is practically lost; now the presence of two loop conformations results in almost-symmetric oscillations with smaller amplitude. Comparison between wild type and ΔHU mutant results (<a href="#pone-0000355-t001">Table 1</a>) indicates that architectural proteins lower the optimal free energy of one conformation, leading to subtle differences of ~1 kcal/mol between the two most stable conformations. In systems like the Gal repressosome <a href="#pone.0000355-Geanacopoulos1">[38]</a>, the architectural HU protein is required to form the loop, which implies strong stabilizing effects and a single dominant conformation. In the <em>lac</em> operon, in contrast, we find that both HU stabilized and non-stabilized conformations contribute to the free energy of looping (<a href="#pone-0000355-g002">Figure 2</a>), which is responsible for the observed asymmetric behavior.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>In all three cases studied here (<a href="#pone-0000355-g002">Figure 2</a> and <a href="#pone-0000355-t001">Table 1</a>), the results obtained for the apparent <em>in vivo</em> twisting force constants, which also include the contributions from the repressor, are in the range 48–68 kcal/mol bp. These twisting force constants are a factor 2 smaller than the canonical value <a href="#pone.0000355-Bloomfield1">[14]</a>, <a href="#pone.0000355-Bates1">[39]</a> of 105 kcal/mol bp or 2.5×10<sup>−19</sup> erg cm, and are similar to those reported recently in cyclization experiments <a href="#pone.0000355-Cloutier1">[40]</a>.</p>


<h4>Shaping the behavior of the two-conformation free energy of looping</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Our analysis has shown that the complex behavior of the in <em>vivo</em> free energy of looping is accurately accounted for by combination of the rather simple behavior of two representative looped conformations (<a href="#pone-0000355-g002">Figure 2</a>). The major differences observed between wild type and the mutant without architectural HU protein arise mainly from the way in which the two conformations are combined; namely, from the differences in the optimal free energies and optimal phases between the two conformations. To explore the potential types of behavior that can arise when two conformations are combined, we have computed the free energy of looping by taking as reference the values of the parameters obtained for wild type (<a href="#pone-0000355-t001">Table 1</a>, WT1), keeping the values for one conformation fixed (conformation 1 of WT1), and systematically changing the values for the other one (<a href="#pone-0000355-g003">Figure 3</a>).</p>
<div class="figure" id="pone-0000355-g003"><div class="img"><a name="pone-0000355-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Free energy of looping for a two-conformation elastic DNA model. Different types of behavior are obtained by changing two key parameters: the difference in optimal free energies (Δ<em>G</em><sub>0,1–</sub>Δ<em>G</em><sub>0,2</sub>) and optimal phases (<em>L<sub>opt</sub></em><sub>,1</sub>–<em>L<sub>opt</sub></em><sub>,2</sub>). (A) The difference in optimal free energies between the two configurations increases from 0 kcal/mol (blue) to 1.5 kcal/mol (red) in increments of 0.5 kcal/mol whereas the difference in optimal phases is kept fixed at 4.2 bp. (B, C) The difference in optimal phases between the two conformations increases from −5.5 bp (blue) to 0 bp (red) in increments of 5.5/3 bp whereas the difference in optimal free energies is kept fixed at 1 kcal/mol (B) and at 0 kcal/mol (C).</p>
<span>doi:10.1371/journal.pone.0000355.g003</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>As the optimal free energy difference between conformations increases (<a href="#pone-0000355-g003">Figure 3A</a>), the behavior of the free energy changes from symmetric multiwell and wide minima, as in the ΔHU mutant, through asymmetric, typical of the wild type system, to symmetric with high amplitude oscillations (curve not shown), typical of “single-conformation” systems. A similarly broad range of types of behavior is also obtained when the difference between optimal phases changes. We have considered these changes in the context of two differences between optimal free energies: ~1.0 kcal/mol, like in wild type (<a href="#pone-0000355-g003">Figure 3B</a>), and ~0.0 kcal/mol, like in ΔHU mutants (<a href="#pone-0000355-g003">Figure 3C</a>). In both cases, as the difference between the optimal phases decreases, the amplitude of the oscillations increases. In the wild type-like situation, the oscillations of the free energy are asymmetric except for precisely tuned values of the parameters. In the ΔHU mutant-like situation, the oscillations are symmetric, and for precisely tuned values of the parameters, it is even possible to obtain oscillations with a period of half the helicity of DNA (<a href="#pone-0000355-g003">Figure 3C</a>, blue curve). All these results together show that the experimentally observed free energies of looping, as diverse as they are, provide just three examples of an even richer number of potential types of behavior.</p>


<h4>Across multiple levels: from DNA looping to gene regulation and cellular physiology</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The high versatility of multi-conformation protein-DNA complexes at shaping the free energy of looping DNA propagates to the cell physiology through the effects of DNA looping in gene regulation. In a similar way as we have inferred and analyzed the <em>in vivo</em> free energy of DNA looping, we can predict the effect of a given free energy of looping on gene regulation by inverting the mathematical expression that connects the free energy of looping with the repression levels for the <em>lac</em> operon (see <a href="#s4">Methods</a>). Explicitly, given the repression level for the system with a single operator (<em>R<sub>noloop</sub></em>), the repression level for two operators with looping follows from the free energy of looping through the expression<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e007&amp;representation=PNG" class="inline-graphic"><span class="note">3</span></span><br>where [<em>N</em>] is the concentration of repressors.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>As in the DNA looping free energy (<a href="#pone-0000355-g003">Figure 3</a>), the precise values of the differences in the optimal free energies and optimal phases between the two conformations strongly affect the repression level (<a href="#pone-0000355-g004">Figure 4</a>), leading also to a large variety of types of behaviors and degrees of repression. In general, the typical asymmetry of the free energy is less marked in the repression level (<a href="#pone-0000355-g004">Figure 4A</a>), to the extent that it might not be obvious in the raw experimental data, as happens in the classical experiments on the repression of the <em>lac</em> operon <a href="#pone.0000355-Muller1">[11]</a>. This loss of features leads to robust repression levels with respect to changes in the optimal phase (<a href="#pone-0000355-g004">Figure 4B</a>) when the optimal free energies differences are similar to the wild type value (~1 kcal/mol), whereas such robustness is not present when the optimal free energies of both conformations are similar (<a href="#pone-0000355-g004">Figure 4C</a>). The particular shape can thus be controlled <em>in vivo</em> by the HU architectural protein to produce either robust or sensitive gene expression patterns.</p>
<div class="figure" id="pone-0000355-g004"><div class="img"><a name="pone-0000355-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000355" data-uri="info:doi/10.1371/journal.pone.0000355.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000355.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000355.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000355.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Effects in gene expression of the free energy of looping for a two-conformation elastic DNA model. Repression levels obtained with Equation 3 using the corresponding free energies of <a href="#pone-0000355-g003">Figure 3</a>. (A) Differences in optimal free energies are varied and the optimal phases are kept fixed. (B, C) Differences in optimal phases are varied and optimal free energies are kept fixed at two different values: 1 kcal/mol (B) and 0 kcal/mol (C).</p>
<span>doi:10.1371/journal.pone.0000355.g004</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Computational and mathematical methods provide a unique avenue to connect cellular physiology with molecular properties in a living organism <a href="#pone.0000355-Vilar2">[9]</a>, <a href="#pone.0000355-Saiz1">[10]</a>, <a href="#pone.0000355-Saiz2">[41]</a>, <a href="#pone.0000355-Vilar3">[42]</a>. The statistical thermodynamics approach we have developed to deconstruct the observed behavior of the expression of the <em>lac</em> operon in <em>E. coli</em> cells has allowed us to obtain the <em>in vivo</em> properties of DNA looping by the <em>lac</em> repressor at different levels of biological organization.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>It was previously shown that classic experimental data on the expression of the <em>lac</em> operon in cell populations led to an unexpected, rather complex, behavior of the free energy of looping DNA <em>in vivo</em>, with small-amplitude asymmetric oscillations as a function of the length of the loop <a href="#pone.0000355-Saiz1">[10]</a>. Here, we have shown that this striking behavior has its molecular origin in the ability of the <em>lac</em> repressor to loop DNA <em>in vivo</em> in at least two different ways. Thus, the intricate <em>in vivo</em> behavior of the free energy of looping is the result of combining the relatively simple behavior of each of the two looped conformations. These two types of loops have different properties and interact distinctly with the HU architectural protein. Explicitly, we found that DNA loops that interact with the HU architectural protein are ~1 kcal/mol more stable than loops that do not. Our approach has also allowed us to accurately obtain the elastic properties of the protein-DNA complexes <em>in vivo</em>, including twisting force constants, which turned out to be a factor 2 smaller than the canonical value of 105 kcal/mol bp (2.5×10<sup>−19</sup> erg cm).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Our analysis of the effects of the molecular properties in the free energy of DNA looping at the cellular level, and their propagation to gene expression at the cell-population level, shows that there is a wide range of potential types of behavior that can arise from combining single-conformation free energies of looping. The mathematical expression for the free energy of looping (Equations 1 and 2) indicates that optimal free energies and phases, which in single-conformation systems affect only quantitative details, are key determinants of the qualitative behavior. In particular, the asymmetry in the oscillations is the consequence of the presence of a slightly preferred loop conformation with different optimal phase. Symmetric oscillations in the free energy result from equally stable loop conformations, a strongly dominant conformation, or conformations with the same optimal phases. In <em>E. coli</em> cells, as shown by our results, the HU protein preferentially affects one loop conformation making it slightly more stable, thus leading to the observed asymmetry.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Different loop trajectories have been observed <em>in vitro</em> for diverse nucleoprotein complexes <a href="#pone.0000355-Semsey1">[19]</a>, <a href="#pone.0000355-Watson1">[21]</a>. In particular, <em>in vitro</em> experiments of DNA cleavage by the <em>Sfi</em>I endonuclease, a type II restriction endonuclease that binds to two DNA sites as a tetramer by looping out the intervening DNA, have shown coexistence and alternative conformations <a href="#pone.0000355-Watson1">[21]</a> as the DNA spacer between binding sites is changed for loop sizes of 109–170 bp. They also observed similar periodicities for the two conformations as well as different phases in <em>in vitro</em> electrophoresis experiments. There are also studies on the Gal repressor showing that several non-simultaneous trajectories can exist and that there is a single configuration of the complex for a particular loop length when the HU protein is present <a href="#pone.0000355-Geanacopoulos1">[38]</a>. Our results provide evidence that shows, for the first time, that alternative and simultaneous nucleoprotein-DNA configurations are present <em>in vivo</em>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>At the cell-population level, whether the typical asymmetry of the free energy propagates to the repression level is controlled by the values of the optimal free energies and phases. In general, the asymmetry in the repression level is less marked than in the free energy, to the extent that it might not be obvious in the raw experimental data <a href="#pone.0000355-Muller1">[11]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Our results indicate that the biological consequences of having two or more DNA-looped conformations include a reduced dependence on the positioning of the DNA binding sites. For instance, by combining two DNA conformations, it is possible to reduce the amplitude of the typical oscillations in the free energy as a function of the length of the loop from ~5 kcal/mol to ~1 kcal/mol (<a href="#pone-0000355-g003">Figure 3C</a>). In this way, DNA appears to the cell to be much more malleable than it actually is in a single conformation. The presence of multiple DNA conformations also provides an extra layer of control of the properties of gene regulation. In the case of the <em>lac</em> operon, we have shown that the HU protein stabilizes one DNA conformation. Similarly, it has also been shown that the Catabolite Activator Protein (CAP) stabilizes preferentially certain loop conformations <a href="#pone.0000355-Balaeff1">[37]</a>. Thus, expression of CAP, HU, and other architectural proteins can change the DNA looping properties in a conformation-dependent manner and select the precise details of the interactions between distal DNA sites.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>In broad terms, our analysis has revealed that the formation of DNA loops <em>in vivo</em> is tightly coupled to the molecular properties of the proteins and protein complexes that form the loop. There is a high versatility of looped DNA-protein complexes at establishing different conformations in the intracellular environment and at adapting from one conformation to another. This versatility underlies the unanticipated behavior of the <em>in vivo</em> free energy of DNA looping and can be responsible not only for asymmetric oscillations with decreased amplitude but also for plateaus and secondary maxima. All these features indicate that the physical properties of DNA can actively be selected to control the cooperative binding of regulatory proteins and to achieve different cellular behaviors.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Methods"></a><h3>Methods</h3>
<h4>Free energy of DNA looping from multiple conformations</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>Following the statistical thermodynamics approach <a href="#pone.0000355-Hill1">[25]</a>, the free energy of looping, Δ<em>G<sub>l</sub></em>, can be expressed in terms of the free energy for each individual conformation as<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e008&amp;representation=PNG" class="inline-graphic"></span><br>where the right hand side of the equation has as many terms as the number of possible representative conformations of the looped DNA-protein complex. In practice, only the conformations with lowest free energy will have a significant effect in the observed behavior. In particular, we have shown that typically only two distinct conformations contribute significantly, and thus <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e009&amp;representation=PNG" class="inline-graphic"></span>, which leads to <span class="inline-formula"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e010&amp;representation=PNG" class="inline-graphic"></span> for the free energy of DNA looping.</p>


<h4>In vivo free energy of DNA looping from physiological measurements</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>The <em>in vivo</em> free energy of DNA looping by the <em>lac</em> repressor's binding to the main and an auxiliary operator can be expressed in terms of the measured repression levels through a well-established model for gene regulation by the <em>lac</em> repressor <a href="#pone.0000355-Vilar2">[9]</a>. For the experimental conditions consisting of a strong auxiliary operator, which are those of the experiments considered here, the free energy of looping DNA <a href="#pone.0000355-Saiz1">[10]</a> for an inter-operator distance <em>L</em> is given by:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000355.e011&amp;representation=PNG" class="inline-graphic"></span><br>where <em>R<sub>loop</sub></em> (<em>L</em>) is the measured repression level, a dimensionless quantity used to quantify the extent of repression of a gene; <em>R<sub>noloop</sub></em> is the repression level in the absence of DNA looping; [<em>N</em>] is the concentration of repressors; and <em>RT</em> is the gas constant times the absolute temperature.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Designed research, performed research, analyzed data, and wrote the paper: LS JV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000355-Adhya1" id="pone.0000355-Adhya1"></a>Adhya S (1989) Multipartite genetic control elements: communication by DNA loop. Annu Rev Genet  23: 227–250.  <ul class="find" data-citedArticleID="942702" data-doi="10.1146/annurev.ge.23.120189.001303"><li><a href="http://dx.doi.org/10.1146/annurev.ge.23.120189.001303" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multipartite+genetic+control+elements%3A+communication+by+DNA+loop." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multipartite+genetic+control+elements%3A+communication+by+DNA+loop.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000355-Matthews1" id="pone.0000355-Matthews1"></a>Matthews KS (1992) DNA Looping. Microbiological Reviews  56: 123–136.  <ul class="find" data-citedArticleID="942740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=DNA+Looping.&amp;auth=&amp;atitle=DNA+Looping." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+Looping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+Looping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000355-Schleif1" id="pone.0000355-Schleif1"></a>Schleif R (1992) DNA looping. Annu Rev Biochem  61: 199–223.  <ul class="find" data-citedArticleID="942756" data-doi="10.1146/annurev.bi.61.070192.001215"><li><a href="http://dx.doi.org/10.1146/annurev.bi.61.070192.001215" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+looping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+looping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000355-Vilar1" id="pone.0000355-Vilar1"></a>Vilar JMG, Saiz L (2005) DNA looping in gene regulation: from the assembly of macromolecular complexes to the control of transcriptional noise. Current Opinion in Genetics &amp; Development  15: 136–144.  <ul class="find" data-citedArticleID="942770" data-doi="10.1016/j.gde.2005.02.005"><li><a href="http://dx.doi.org/10.1016/j.gde.2005.02.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+looping+in+gene+regulation%3A+from+the+assembly+of+macromolecular+complexes+to+the+control+of+transcriptional+noise." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+looping+in+gene+regulation%3A+from+the+assembly+of+macromolecular+complexes+to+the+control+of+transcriptional+noise.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000355-Li1" id="pone.0000355-Li1"></a>Li QL, Barkess G, Qian H (2006) Chromatin looping and the probability of transcription. Trends in Genetics  22: 197–202.  <ul class="find" data-citedArticleID="942738" data-doi="10.1016/j.tig.2006.02.004"><li><a href="http://dx.doi.org/10.1016/j.tig.2006.02.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromatin+looping+and+the+probability+of+transcription." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromatin+looping+and+the+probability+of+transcription.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000355-Talbert1" id="pone.0000355-Talbert1"></a>Talbert PB, Henikoff S (2006) Spreading of silent chromatin: inaction at a distance. Nature Reviews Genetics  7: 793–803.  <ul class="find" data-citedArticleID="942766" data-doi="10.1038/nrg1920"><li><a href="http://dx.doi.org/10.1038/nrg1920" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Spreading+of+silent+chromatin%3A+inaction+at+a+distance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Spreading+of+silent+chromatin%3A+inaction+at+a+distance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000355-Ellis1" id="pone.0000355-Ellis1"></a>Ellis RJ (2001) Macromolecular crowding: obvious but underappreciated. Trends in Biochemical Sciences  26: 597–604.  <ul class="find" data-citedArticleID="942718" data-doi="10.1016/s0968-0004(01)01938-7"><li><a href="http://dx.doi.org/10.1016/s0968-0004(01)01938-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Macromolecular+crowding%3A+obvious+but+underappreciated." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Macromolecular+crowding%3A+obvious+but+underappreciated.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000355-Bellomy1" id="pone.0000355-Bellomy1"></a>Bellomy GR, Mossing MC, Record MT (1988) Physical-Properties of DNA Invivo as Probed by the Length Dependence of the Lac Operator Looping Process. Biochemistry  27: 3900–3906.  <ul class="find" data-citedArticleID="942710" data-doi="10.1021/bi00411a002"><li><a href="http://dx.doi.org/10.1021/bi00411a002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physical-Properties+of+DNA+Invivo+as+Probed+by+the+Length+Dependence+of+the+Lac+Operator+Looping+Process." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physical-Properties+of+DNA+Invivo+as+Probed+by+the+Length+Dependence+of+the+Lac+Operator+Looping+Process.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000355-Vilar2" id="pone.0000355-Vilar2"></a>Vilar JMG, Leibler S (2003) DNA looping and physical constraints on transcription regulation. J Mol Biol  331: 981–989.  <ul class="find" data-citedArticleID="942772" data-doi="10.1016/s0022-2836(03)00764-2"><li><a href="http://dx.doi.org/10.1016/s0022-2836(03)00764-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+looping+and+physical+constraints+on+transcription+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+looping+and+physical+constraints+on+transcription+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000355-Saiz1" id="pone.0000355-Saiz1"></a>Saiz L, Rubi JM, Vilar JMG (2005) Inferring the in vivo looping properties of DNA. Proceedings of the National Academy of Sciences of the United States of America  102: 17642–17645.  <ul class="find" data-citedArticleID="942752" data-doi="10.1073/pnas.0505693102"><li><a href="http://dx.doi.org/10.1073/pnas.0505693102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inferring+the+in+vivo+looping+properties+of+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inferring+the+in+vivo+looping+properties+of+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000355-Muller1" id="pone.0000355-Muller1"></a>Muller J, Oehler S, Muller-Hill B (1996) Repression of lac promoter as a function of distance, phase and quality of an auxiliary lac operator. J Mol Biol  257: 21–29.  <ul class="find" data-citedArticleID="942742" data-doi="10.1006/jmbi.1996.0143"><li><a href="http://dx.doi.org/10.1006/jmbi.1996.0143" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Repression+of+lac+promoter+as+a+function+of+distance%2C+phase+and+quality+of+an+auxiliary+lac+operator." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Repression+of+lac+promoter+as+a+function+of+distance%2C+phase+and+quality+of+an+auxiliary+lac+operator.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000355-Lee1" id="pone.0000355-Lee1"></a>Lee DH, Schleif RF (1989) In vivo DNA loops in araCBAD: size limits and helical repeat. Proc Natl Acad Sci U S A  86: 476–480.  <ul class="find" data-citedArticleID="942734" data-doi="10.1073/pnas.86.2.476"><li><a href="http://dx.doi.org/10.1073/pnas.86.2.476" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+DNA+loops+in+araCBAD%3A+size+limits+and+helical+repeat." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+DNA+loops+in+araCBAD%3A+size+limits+and+helical+repeat.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000355-Ptashne1" id="pone.0000355-Ptashne1"></a>Ptashne M, Gann A (2002) Genes &amp; signals. Cold Spring Harbor, , N.Y.: Cold Spring Harbor Laboratory Press.   <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0000355-Bloomfield1" id="pone.0000355-Bloomfield1"></a>Bloomfield VA, Crothers DM, Tinoco I (2000) Nucleic Acids: Structures, Properties, and Functions. Sausalito, CA: University Science Books.   <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0000355-Yan1" id="pone.0000355-Yan1"></a>Yan J, Marko JF (2004) Localized single-stranded bubble mechanism for cyclization of short double helix DNA. Phys Rev Lett  93: 108108.  <ul class="find" data-citedArticleID="942782" data-doi="10.1103/physrevlett.93.108108"><li><a href="http://dx.doi.org/10.1103/physrevlett.93.108108" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localized+single-stranded+bubble+mechanism+for+cyclization+of+short+double+helix+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localized+single-stranded+bubble+mechanism+for+cyclization+of+short+double+helix+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000355-Olson1" id="pone.0000355-Olson1"></a>Olson WK, Swigon D, Coleman BD (2004) Implications of the dependence of the elastic properties of DNA on nucleotide sequence. Philosophical Transactions of the Royal Society of London Series a-Mathematical Physical and Engineering Sciences  362: 1403–1422.  <ul class="find" data-citedArticleID="942744" data-doi="10.1098/rsta.2004.1380"><li><a href="http://dx.doi.org/10.1098/rsta.2004.1380" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Implications+of+the+dependence+of+the+elastic+properties+of+DNA+on+nucleotide+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Implications+of+the+dependence+of+the+elastic+properties+of+DNA+on+nucleotide+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000355-Villa1" id="pone.0000355-Villa1"></a>Villa E, Balaeff A, Schulten K (2005) Structural dynamics of the lac repressor-DNA complex revealed by a multiscale simulation. Proc Natl Acad Sci U S A  102: 6783–6788.  <ul class="find" data-citedArticleID="942776" data-doi="10.1073/pnas.0409387102"><li><a href="http://dx.doi.org/10.1073/pnas.0409387102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+dynamics+of+the+lac+repressor-DNA+complex+revealed+by+a+multiscale+simulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+dynamics+of+the+lac+repressor-DNA+complex+revealed+by+a+multiscale+simulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000355-Halford1" id="pone.0000355-Halford1"></a>Halford SE, Gowers DM, Sessions RB (2000) Two are better than one. Nature Structural Biology  7: 705–707.  <ul class="find" data-citedArticleID="942728" data-doi="10.1038/78911"><li><a href="http://dx.doi.org/10.1038/78911" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Two+are+better+than+one." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Two+are+better+than+one.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000355-Semsey1" id="pone.0000355-Semsey1"></a>Semsey S, Virnik K, Adhya S (2005) A gamut of loops: meandering DNA. Trends Biochem Sci  30: 334–341.  <ul class="find" data-citedArticleID="942762" data-doi="10.1016/j.tibs.2005.04.009"><li><a href="http://dx.doi.org/10.1016/j.tibs.2005.04.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gamut+of+loops%3A+meandering+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gamut+of+loops%3A+meandering+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000355-Zhang1" id="pone.0000355-Zhang1"></a>Zhang Y, McEwen AE, Crothers DM, Levene SD (2006) Statistical-mechanical theory of DNA looping. Biophys J  90: 1903–1912.  <ul class="find" data-citedArticleID="942784" data-doi="10.1529/biophysj.105.070490"><li><a href="http://dx.doi.org/10.1529/biophysj.105.070490" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Statistical-mechanical+theory+of+DNA+looping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Statistical-mechanical+theory+of+DNA+looping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000355-Watson1" id="pone.0000355-Watson1"></a>Watson MA, Gowers DM, Halford SE (2000) Alternative geometries of DNA looping: an analysis using the SfiI endonuclease. J Mol Biol  298: 461–475.  <ul class="find" data-citedArticleID="942778" data-doi="10.1006/jmbi.2000.3676"><li><a href="http://dx.doi.org/10.1006/jmbi.2000.3676" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+geometries+of+DNA+looping%3A+an+analysis+using+the+SfiI+endonuclease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+geometries+of+DNA+looping%3A+an+analysis+using+the+SfiI+endonuclease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000355-Becker1" id="pone.0000355-Becker1"></a>Becker NA, Kahn JD, Maher LJ (2005) Bacterial repression loops require enhanced DNA flexibility. Journal of Molecular Biology  349: 716–730.  <ul class="find" data-citedArticleID="942708" data-doi="10.1016/j.jmb.2005.09.005"><li><a href="http://dx.doi.org/10.1016/j.jmb.2005.09.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacterial+repression+loops+require+enhanced+DNA+flexibility." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacterial+repression+loops+require+enhanced+DNA+flexibility.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000355-Swigon1" id="pone.0000355-Swigon1"></a>Swigon D, Coleman BD, Olson WK (2006) Modeling the Lac repressor-operator assembly: The influence of DNA looping on Lac repressor conformation. Proc Natl Acad Sci U S A  103: 9879–9884.  <ul class="find" data-citedArticleID="942764" data-doi="10.1073/pnas.0603557103"><li><a href="http://dx.doi.org/10.1073/pnas.0603557103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Modeling+the+Lac+repressor-operator+assembly%3A+The+influence+of+DNA+looping+on+Lac+repressor+conformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Modeling+the+Lac+repressor-operator+assembly%3A+The+influence+of+DNA+looping+on+Lac+repressor+conformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000355-Segall1" id="pone.0000355-Segall1"></a>Segall AM, Goodman SD, Nash HA (1994) Architectural elements in nucleoprotein complexes: interchangeability of specific and non-specific DNA binding proteins. Embo J  13: 4536–4548.  <ul class="find" data-citedArticleID="942760"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Architectural+elements+in+nucleoprotein+complexes%3A+interchangeability+of+specific+and+non-specific+DNA+binding+proteins.&amp;auth=&amp;atitle=Architectural+elements+in+nucleoprotein+complexes%3A+interchangeability+of+specific+and+non-specific+DNA+binding+proteins." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Architectural+elements+in+nucleoprotein+complexes%3A+interchangeability+of+specific+and+non-specific+DNA+binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Architectural+elements+in+nucleoprotein+complexes%3A+interchangeability+of+specific+and+non-specific+DNA+binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000355-Hill1" id="pone.0000355-Hill1"></a>Hill TL (1960) An introduction to statistical thermodynamics. Reading, , Mass.: Addison-Wesley Pub. Co.   <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0000355-Lewis1" id="pone.0000355-Lewis1"></a>Lewis M, Chang G, Horton NC, Kercher MA, Pace HC, et al.  (1996) Crystal structure of the lactose operon repressor and its complexes with DNA and inducer. Science  271: 1247–1254.  <ul class="find" data-citedArticleID="942736" data-doi="10.1126/science.271.5253.1247"><li><a href="http://dx.doi.org/10.1126/science.271.5253.1247" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+the+lactose+operon+repressor+and+its+complexes+with+DNA+and+inducer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+the+lactose+operon+repressor+and+its+complexes+with+DNA+and+inducer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000355-Friedman1" id="pone.0000355-Friedman1"></a>Friedman AM, Fischmann TO, Steitz TA (1995) Crystal structure of lac repressor core tetramer and its implications for DNA looping. Science  268: 1721–1727.  <ul class="find" data-citedArticleID="942720" data-doi="10.1126/science.7792597"><li><a href="http://dx.doi.org/10.1126/science.7792597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+lac+repressor+core+tetramer+and+its+implications+for+DNA+looping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+lac+repressor+core+tetramer+and+its+implications+for+DNA+looping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000355-Ruben1" id="pone.0000355-Ruben1"></a>Ruben GC, Roos TB (1997) Conformation of Lac repressor tetramer in solution, bound and unbound to operator DNA. Microsc Res Tech  36: 400–416.  <ul class="find" data-citedArticleID="942750" data-doi="10.1002/(sici)1097-0029(19970301)36:5<400::aid-jemt10&gt;3.3.co;2-e"><li><a href="http://dx.doi.org/10.1002/(sici)1097-0029(19970301)36:5<400::aid-jemt10&gt;3.3.co;2-e" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conformation+of+Lac+repressor+tetramer+in+solution%2C+bound+and+unbound+to+operator+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conformation+of+Lac+repressor+tetramer+in+solution%2C+bound+and+unbound+to+operator+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000355-Edelman1" id="pone.0000355-Edelman1"></a>Edelman LM, Cheong R, Kahn JD (2003) Fluorescence resonance energy transfer over approximately 130 basepairs in hyperstable lac repressor-DNA loops. Biophys J  84: 1131–1145.  <ul class="find" data-citedArticleID="942716" data-doi="10.1016/s0006-3495(03)74929-7"><li><a href="http://dx.doi.org/10.1016/s0006-3495(03)74929-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fluorescence+resonance+energy+transfer+over+approximately+130+basepairs+in+hyperstable+lac+repressor-DNA+loops." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fluorescence+resonance+energy+transfer+over+approximately+130+basepairs+in+hyperstable+lac+repressor-DNA+loops.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000355-Grosschedl1" id="pone.0000355-Grosschedl1"></a>Grosschedl R (1995) Higher-Order Nucleoprotein Complexes in Transcription - Analogies with Site-Specific Recombination. Current Opinion in Cell Biology  7: 362–370.  <ul class="find" data-citedArticleID="942726" data-doi="10.1016/0955-0674(95)80091-3"><li><a href="http://dx.doi.org/10.1016/0955-0674(95)80091-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Higher-Order+Nucleoprotein+Complexes+in+Transcription+-+Analogies+with+Site-Specific+Recombination." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Higher-Order+Nucleoprotein+Complexes+in+Transcription+-+Analogies+with+Site-Specific+Recombination.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000355-Werner1" id="pone.0000355-Werner1"></a>Werner MH, Burley SK (1997) Architectural transcription factors: proteins that remodel DNA. Cell  88: 733–736.  <ul class="find" data-citedArticleID="942780" data-doi="10.1016/s0092-8674(00)81917-0"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81917-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Architectural+transcription+factors%3A+proteins+that+remodel+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Architectural+transcription+factors%3A+proteins+that+remodel+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000355-Thomas1" id="pone.0000355-Thomas1"></a>Thomas JO, Travers AA (2001) HMG1 and 2, and related ‘architectural’ DNA-binding proteins. Trends Biochem Sci  26: 167–174.  <ul class="find" data-citedArticleID="942768" data-doi="10.1016/s0968-0004(01)01801-1"><li><a href="http://dx.doi.org/10.1016/s0968-0004(01)01801-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HMG1+and+2%2C+and+related+%26%23x2018%3Barchitectural%26%23x2019%3B+DNA-binding+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HMG1+and+2%2C+and+related+%26%23x2018%3Barchitectural%26%23x2019%3B+DNA-binding+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000355-Gartenberg1" id="pone.0000355-Gartenberg1"></a>Gartenberg MR, Crothers DM (1988) DNA-Sequence Determinants of Cap-Induced Bending and Protein-Binding Affinity. Nature  333: 824–829.  <ul class="find" data-citedArticleID="942722" data-doi="10.1038/333824a0"><li><a href="http://dx.doi.org/10.1038/333824a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA-Sequence+Determinants+of+Cap-Induced+Bending+and+Protein-Binding+Affinity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA-Sequence+Determinants+of+Cap-Induced+Bending+and+Protein-Binding+Affinity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000355-Schultz1" id="pone.0000355-Schultz1"></a>Schultz SC, Shields GC, Steitz TA (1991) Crystal-Structure of a Cap-DNA Complex - the DNA Is Bent by 90-Degrees. Science  253: 1001–1007.  <ul class="find" data-citedArticleID="942758" data-doi="10.1126/science.1653449"><li><a href="http://dx.doi.org/10.1126/science.1653449" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal-Structure+of+a+Cap-DNA+Complex+-+the+DNA+Is+Bent+by+90-Degrees." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal-Structure+of+a+Cap-DNA+Complex+-+the+DNA+Is+Bent+by+90-Degrees.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000355-Rice1" id="pone.0000355-Rice1"></a>Rice PA, Yang S, Mizuuchi K, Nash HA (1996) Crystal structure of an IHF-DNA complex: a protein-induced DNA U-turn. Cell  87: 1295–1306.  <ul class="find" data-citedArticleID="942748" data-doi="10.1016/s0092-8674(00)81824-3"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81824-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Crystal+structure+of+an+IHF-DNA+complex%3A+a+protein-induced+DNA+U-turn." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Crystal+structure+of+an+IHF-DNA+complex%3A+a+protein-induced+DNA+U-turn.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000355-Hudson1" id="pone.0000355-Hudson1"></a>Hudson JM, Fried MG (1990) Cooperative Interactions between the Catabolite Gene Activator Protein and the Lac Repressor at the Lactose Promoter. Journal of Molecular Biology  214: 381–396.  <ul class="find" data-citedArticleID="942732" data-doi="10.1016/0022-2836(90)90188-r"><li><a href="http://dx.doi.org/10.1016/0022-2836(90)90188-r" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cooperative+Interactions+between+the+Catabolite+Gene+Activator+Protein+and+the+Lac+Repressor+at+the+Lactose+Promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cooperative+Interactions+between+the+Catabolite+Gene+Activator+Protein+and+the+Lac+Repressor+at+the+Lactose+Promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000355-Balaeff1" id="pone.0000355-Balaeff1"></a>Balaeff A, Mahadevan L, Schulten K (2004) Structural basis for cooperative DNA binding by CAP and lac repressor. Structure  12: 123–132.  <ul class="find" data-citedArticleID="942704" data-doi="10.1016/j.str.2003.12.004"><li><a href="http://dx.doi.org/10.1016/j.str.2003.12.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Structural+basis+for+cooperative+DNA+binding+by+CAP+and+lac+repressor." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Structural+basis+for+cooperative+DNA+binding+by+CAP+and+lac+repressor.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000355-Geanacopoulos1" id="pone.0000355-Geanacopoulos1"></a>Geanacopoulos M, Vasmatzis G, Zhurkin VB, Adhya S (2001) Gal repressosome contains an antiparallel DNA loop. Nat Struct Biol  8: 432–436.  <ul class="find" data-citedArticleID="942724" data-doi="10.1038/87595"><li><a href="http://dx.doi.org/10.1038/87595" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gal+repressosome+contains+an+antiparallel+DNA+loop." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gal+repressosome+contains+an+antiparallel+DNA+loop.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000355-Bates1" id="pone.0000355-Bates1"></a>Bates AD, Maxwell A (2005) DNA Topology. Oxford, UK: Oxford University Press.   <ul class="find-nolinks"></ul></li><li><span class="label">40.
              </span><a name="pone.0000355-Cloutier1" id="pone.0000355-Cloutier1"></a>Cloutier TE, Widom J (2005) DNA twisting flexibility and the formation of sharply looped protein-DNA complexes. Proceedings of the National Academy of Sciences of the United States of America  102: 3645–3650.  <ul class="find" data-citedArticleID="942714" data-doi="10.1073/pnas.0409059102"><li><a href="http://dx.doi.org/10.1073/pnas.0409059102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+twisting+flexibility+and+the+formation+of+sharply+looped+protein-DNA+complexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+twisting+flexibility+and+the+formation+of+sharply+looped+protein-DNA+complexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000355-Saiz2" id="pone.0000355-Saiz2"></a>Saiz L, Vilar JMG (2006) Stochastic dynamics of macromolecular-assembly networks. Mol Syst Biol  2: 2006.0024.  <ul class="find" data-citedArticleID="942754" data-doi="10.1038/msb4100061"><li><a href="http://dx.doi.org/10.1038/msb4100061" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Stochastic+dynamics+of+macromolecular-assembly+networks." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Stochastic+dynamics+of+macromolecular-assembly+networks.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000355-Vilar3" id="pone.0000355-Vilar3"></a>Vilar JMG, Saiz L (2006) Multiprotein DNA looping. Phys Rev Lett  96: 238103.  <ul class="find" data-citedArticleID="942774" data-doi="10.1103/physrevlett.96.238103"><li><a href="http://dx.doi.org/10.1103/physrevlett.96.238103" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiprotein+DNA+looping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiprotein+DNA+looping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.XML" value="76569"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.PDF" value="338173"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g001.PNG_L" value="72067"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g001.PNG_M" value="67630"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g001.PNG_S" value="3563"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g001.TIF" value="155648"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g001.PNG_I" value="27458"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e001.PNG" value="4870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e001.TIF" value="26316"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e002.PNG" value="5352"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e002.TIF" value="26412"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e003.PNG" value="3367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e003.TIF" value="24868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e004.PNG" value="3237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e004.TIF" value="24776"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e005.PNG" value="1923"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e005.TIF" value="23024"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g002.PNG_L" value="116897"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g002.PNG_M" value="95372"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g002.PNG_S" value="4771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g002.TIF" value="226236"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g002.PNG_I" value="60240"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.t001.PNG_L" value="74527"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.t001.PNG_M" value="26124"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.t001.PNG_S" value="1968"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.t001.TIF" value="335322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.t001.PNG_I" value="11626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e006.PNG" value="2506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e006.TIF" value="24192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g003.PNG_L" value="118946"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g003.PNG_M" value="113367"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g003.PNG_S" value="4732"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g003.TIF" value="267936"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g003.PNG_I" value="60366"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e007.PNG" value="4975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e007.TIF" value="26200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g004.PNG_L" value="133204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g004.PNG_M" value="135636"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g004.PNG_S" value="4887"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g004.TIF" value="300276"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.g004.PNG_I" value="67286"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e008.PNG" value="4393"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e008.TIF" value="25132"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e009.PNG" value="3060"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e009.TIF" value="24300"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e010.PNG" value="3547"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e010.TIF" value="24984"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e011.PNG" value="5449"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000355.e011.TIF" value="26856"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000355&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000355" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000355&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000355&volume=&issue=&title=Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes&author_name=Leonor%20Saiz%2C%20Jose%20M.G.%20Vilar&start_page=1&end_page=7" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000355" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355&amp;t=Multilevel%20Deconstruction%20of%20the%20In%20Vivo%20Behavior%20of%20Looped%20DNA-Protein%20Complexes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355&title=Multilevel%20Deconstruction%20of%20the%20In%20Vivo%20Behavior%20of%20Looped%20DNA-Protein%20Complexes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355&amp;title=Multilevel%20Deconstruction%20of%20the%20In%20Vivo%20Behavior%20of%20Looped%20DNA-Protein%20Complexes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000355&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Multilevel Deconstruction of the In Vivo Behavior of Looped DNA-Protein Complexes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000355';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Multilevel%20Deconstruction%20of%20the%20In%20Vivo%20Behavior%20of%20Looped%20DNA-Protein%20Complexes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000355" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000355" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">













          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bending%22" title="Search for articles in the subject area:'Bending'"><div class="flagText">Bending</div></a>
              <div data-categoryid="22873" data-articleid="23548"
                   data-categoryname="Bending"
                   class="flagImage" title="Flag 'Bending' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA%22" title="Search for articles in the subject area:'DNA'"><div class="flagText">DNA</div></a>
              <div data-categoryid="32449" data-articleid="23548"
                   data-categoryname="DNA"
                   class="flagImage" title="Flag 'DNA' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+structure%22" title="Search for articles in the subject area:'DNA structure'"><div class="flagText">DNA structure</div></a>
              <div data-categoryid="34399" data-articleid="23548"
                   data-categoryname="DNA structure"
                   class="flagImage" title="Flag 'DNA structure' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA-binding+proteins%22" title="Search for articles in the subject area:'DNA-binding proteins'"><div class="flagText">DNA-binding proteins</div></a>
              <div data-categoryid="17441" data-articleid="23548"
                   data-categoryname="DNA-binding proteins"
                   class="flagImage" title="Flag 'DNA-binding proteins' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Free+energy%22" title="Search for articles in the subject area:'Free energy'"><div class="flagText">Free energy</div></a>
              <div data-categoryid="18159" data-articleid="23548"
                   data-categoryname="Free energy"
                   class="flagImage" title="Flag 'Free energy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="23548"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+regulation%22" title="Search for articles in the subject area:'Gene regulation'"><div class="flagText">Gene regulation</div></a>
              <div data-categoryid="17553" data-articleid="23548"
                   data-categoryname="Gene regulation"
                   class="flagImage" title="Flag 'Gene regulation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22lac+operon%22" title="Search for articles in the subject area:'lac operon'"><div class="flagText">lac operon</div></a>
              <div data-categoryid="35779" data-articleid="23548"
                   data-categoryname="lac operon"
                   class="flagImage" title="Flag 'lac operon' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2147'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5931'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=654&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
