

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000648"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000648" />

    <meta name="citation_title" content="Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels"/>
    <meta itemprop="name" content="Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels"/>

      <meta name="citation_author" content="Todd A. Sangster"/>
            <meta name="citation_author_institution" content="Committee on Genetics, University of Chicago, Chicago, Illinois, United States of America"/>
            <meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Adam Bahrami"/>
            <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Amity Wilczek"/>
            <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Etsuko Watanabe"/>
            <meta name="citation_author_institution" content="FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Kurt Schellenberg"/>
            <meta name="citation_author_institution" content="FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Catherine McLellan"/>
            <meta name="citation_author_institution" content="FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Alicia Kelley"/>
            <meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Sek Won Kong"/>
            <meta name="citation_author_institution" content="Informatics Program, Children's Hospital Boston, Boston, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Christine Queitsch"/>
            <meta name="citation_author_institution" content="FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America"/>
      <meta name="citation_author" content="Susan Lindquist"/>
            <meta name="citation_author_institution" content="Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America"/>

    <meta name="citation_date" content="2007/7/25"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000648.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e648"/>
    <meta name="citation_issue" content="7"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Hsp90: a specialized but essential protein-folding tool.; citation_author=JC Young; citation_author=I Moarefi; citation_author=FU Hartl; citation_journal_title=J Cell Biol; citation_volume=154; citation_number=1; citation_pages=267-273; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Heat-shock protein 90, a chaperone for folding and regulation.; citation_author=D Picard; citation_journal_title=Cell Mol Life Sci; citation_volume=59; citation_number=2; citation_pages=1640-1648; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hsp70 and Hsp90–a relay team for protein folding.; citation_author=H Wegele; citation_author=L Muller; citation_author=J Buchner; citation_journal_title=Rev Physiol Biochem Pharmacol; citation_volume=151; citation_number=3; citation_pages=1-44; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=HSP90 and the chaperoning of cancer.; citation_author=L Whitesell; citation_author=SL Lindquist; citation_journal_title=Nat Rev Cancer; citation_volume=5; citation_number=4; citation_pages=761-772; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Hsp90 as a capacitor of phenotypic variation.; citation_author=C Queitsch; citation_author=TA Sangster; citation_author=S Lindquist; citation_journal_title=Nature; citation_volume=417; citation_number=5; citation_pages=618-624; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hsp90 as a capacitor for morphological evolution.; citation_author=SL Rutherford; citation_author=S Lindquist; citation_journal_title=Nature; citation_volume=396; citation_number=6; citation_pages=336-342; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Evidence for an epigenetic mechanism by which Hsp90 acts as a capacitor for morphological evolution.; citation_author=V Sollars; citation_author=X Lu; citation_author=L Xiao; citation_author=X Wang; citation_author=MD Garfinkel; citation_journal_title=Nat Genet; citation_volume=33; citation_number=7; citation_pages=70-74; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Hsp90 potentiates the rapid evolution of new traits: drug resistance in diverse fungi.; citation_author=LE Cowen; citation_author=S Lindquist; citation_journal_title=Science; citation_volume=309; citation_number=8; citation_pages=2185-2189; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Control of canalization and evolvability by hsp90.; citation_author=CC Milton; citation_author=CM Ulane; citation_author=S Rutherford; citation_journal_title=PLoS ONE; citation_volume=1; citation_number=9; citation_pages=e75; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Chaperoning of glucocorticoid receptors.; citation_author=WB Pratt; citation_author=Y Morishima; citation_author=M Murphy; citation_author=M Harrell; citation_journal_title=Handb Exp Pharmacol; citation_number=10; citation_pages=111-138; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Reduced levels of hsp90 compromise steroid receptor action in vivo.; citation_author=D Picard; citation_author=B Khursheed; citation_author=MJ Garabedian; citation_author=MG Fortin; citation_author=S Lindquist; citation_journal_title=Nature; citation_volume=348; citation_number=11; citation_pages=166-168; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Molecular interaction network of the Hsp90 chaperone system.; citation_author=R Zhao; citation_author=WA Houry; citation_journal_title=Adv Exp Med Biol; citation_volume=594; citation_number=12; citation_pages=27-36; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Animal and plant cell lysates share a conserved chaperone system that assembles the glucocorticoid receptor into a functional heterocomplex with hsp90.; citation_author=LF Stancato; citation_author=KA Hutchison; citation_author=P Krishna; citation_author=WB Pratt; citation_journal_title=Biochemistry; citation_volume=35; citation_number=13; citation_pages=554-561; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Analysis of tissue-specific expression of Arabidopsis thaliana HSP90-family gene HSP81.; citation_author=N Yabe; citation_author=T Takahashi; citation_author=Y Komeda; citation_journal_title=Plant Cell Physiol; citation_volume=35; citation_number=14; citation_pages=1207-1219; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=The Hsp90 family of proteins in Arabidopsis thaliana.; citation_author=P Krishna; citation_author=G Gloor; citation_journal_title=Cell Stress Chaperones; citation_volume=6; citation_number=15; citation_pages=238-246; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The chlorate-resistant and photomorphogenesis-defective mutant cr88 encodes a chloroplast-targeted HSP90.; citation_author=D Cao; citation_author=JE Froehlich; citation_author=H Zhang; citation_author=CL Cheng; citation_journal_title=Plant J; citation_volume=33; citation_number=16; citation_pages=107-118; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=SHEPHERD is the Arabidopsis GRP94 responsible for the formation of functional CLAVATA proteins.; citation_author=S Ishiguro; citation_author=Y Watanabe; citation_author=N Ito; citation_author=H Nonaka; citation_author=N Takeda; citation_journal_title=Embo J; citation_volume=21; citation_number=17; citation_pages=898-908; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=HSP90 interacts with RAR1 and SGT1 and is essential for RPS2-mediated disease resistance in Arabidopsis.; citation_author=A Takahashi; citation_author=C Casais; citation_author=K Ichimura; citation_author=K Shirasu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=18; citation_pages=11777-11782; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Cytosolic HSP90 associates with and modulates the Arabidopsis RPM1 disease resistance protein.; citation_author=DA Hubert; citation_author=P Tornero; citation_author=Y Belkhadir; citation_author=P Krishna; citation_author=A Takahashi; citation_journal_title=Embo J; citation_volume=22; citation_number=19; citation_pages=5679-5689; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Molecular chaperone Hsp90 associates with resistance protein N and its signaling proteins SGT1 and Rar1 to modulate an innate immune response in plants.; citation_author=Y Liu; citation_author=T Burch-Smith; citation_author=M Schiff; citation_author=S Feng; citation_author=SP Dinesh-Kumar; citation_journal_title=J Biol Chem; citation_volume=279; citation_number=20; citation_pages=2101-2108; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=High throughput virus-induced gene silencing implicates heat shock protein 90 in plant disease resistance.; citation_author=R Lu; citation_author=I Malcuit; citation_author=P Moffett; citation_author=MT Ruiz; citation_author=J Peart; citation_journal_title=Embo J; citation_volume=22; citation_number=21; citation_pages=5690-5699; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The C-terminal half of Phytophthora infestans RXLR effector AVR3a is sufficient to trigger R3a-mediated hypersensitivity and suppress INF1-induced cell death in Nicotiana benthamiana.; citation_author=JI Bos; citation_author=TD Kanneganti; citation_author=C Young; citation_author=C Cakir; citation_author=E Huitema; citation_journal_title=Plant J; citation_volume=48; citation_number=22; citation_pages=165-176; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The Mi-1-Mediated Pest Resistance Requires Hsp90 and Sgt1.; citation_author=KK Bhattarai; citation_author=Q Li; citation_author=Y Liu; citation_author=SP Dinesh-Kumar; citation_author=I Kaloshian; citation_journal_title=Plant Physiol.; citation_number=23; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Cytosolic HSP90 and HSP70 are essential components of INF1-mediated hypersensitive response and non-host resistance to Pseudomonas cichorii in Nicotiana benthamiana.; citation_author=H Kanzaki; citation_author=H Saitoh; citation_author=A Ito; citation_author=S Fujisawa; citation_author=S Kamoun; citation_journal_title=Molecular Plant Pathology; citation_volume=4; citation_number=24; citation_pages=383-391; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Contrasting mechanisms of defense against biotrophic and necrotrophic pathogens.; citation_author=J Glazebrook; citation_journal_title=Annu Rev Phytopathol; citation_volume=43; citation_number=25; citation_pages=205-227; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Role of SGT1 in resistance protein accumulation in plant immunity.; citation_author=C Azevedo; citation_author=S Betsuyaku; citation_author=J Peart; citation_author=A Takahashi; citation_author=L Noel; citation_journal_title=Embo J; citation_volume=25; citation_number=26; citation_pages=2007-2016; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Antagonistic control of disease resistance protein stability in the plant immune system.; citation_author=BF Holt; citation_author=Y Belkhadir; citation_author=JL Dangl; citation_journal_title=Science; citation_volume=309; citation_number=27; citation_pages=929-932; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Pharmacologic shifting of a balance between protein refolding and degradation mediated by Hsp90.; citation_author=C Schneider; citation_author=L Sepp-Lorenzino; citation_author=E Nimmesgern; citation_author=O Ouerfelli; citation_author=S Danishefsky; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=28; citation_pages=14536-14541; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Identification and structural characterization of the ATP/ADP-binding site in the Hsp90 molecular chaperone.; citation_author=C Prodromou; citation_author=SM Roe; citation_author=R O'Brien; citation_author=JE Ladbury; citation_author=PW Piper; citation_journal_title=Cell; citation_volume=90; citation_number=29; citation_pages=65-75; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The importance of ATP binding and hydrolysis by hsp90 in formation and function of protein heterocomplexes.; citation_author=JP Grenert; citation_author=BD Johnson; citation_author=DO Toft; citation_journal_title=J Biol Chem; citation_volume=274; citation_number=30; citation_pages=17525-17533; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Polypeptide release by Hsp90 involves ATP hydrolysis and is enhanced by the co-chaperone p23.; citation_author=JC Young; citation_author=FU Hartl; citation_journal_title=Embo J; citation_volume=19; citation_number=31; citation_pages=5930-5940; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Genome-wide insertional mutagenesis of Arabidopsis thaliana.; citation_author=JM Alonso; citation_author=AN Stepanova; citation_author=TJ Leisse; citation_author=CJ Kim; citation_author=H Chen; citation_journal_title=Science; citation_volume=301; citation_number=32; citation_pages=653-657; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=TAIR7 Genome Release.; citation_number=33; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Specific and heritable genetic interference by double-stranded RNA in Arabidopsis thaliana.; citation_author=CF Chuang; citation_author=EM Meyerowitz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=34; citation_pages=4985-4990; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Efficient isolation and mapping of Arabidopsis thaliana T-DNA insert junctions by thermal asymmetric interlaced PCR.; citation_author=YG Liu; citation_author=N Mitsukawa; citation_author=T Oosumi; citation_author=RF Whittier; citation_journal_title=Plant J; citation_volume=8; citation_number=35; citation_pages=457-463; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Analysis of the native forms of the 90 kDa heat shock protein (hsp90) in plant cytosolic extracts.; citation_author=P Krishna; citation_author=RK Reddy; citation_author=M Sacco; citation_author=JR Frappier; citation_author=RF Felsheim; citation_journal_title=Plant Mol Biol; citation_volume=33; citation_number=36; citation_pages=457-466; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Requirement for Hsp90 and a CyP-40-type cyclophilin in negative regulation of the heat shock response.; citation_author=AA Duina; citation_author=HM Kalton; citation_author=RF Gaber; citation_journal_title=J Biol Chem; citation_volume=273; citation_number=37; citation_pages=18974-18978; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Induction of hsp 72/73 by herbimycin A, an inhibitor of transformation by tyrosine kinase oncogenes.; citation_author=Y Murakami; citation_author=Y Uehara; citation_author=C Yamamoto; citation_author=H Fukazawa; citation_author=S Mizuno; citation_journal_title=Exp Cell Res; citation_volume=195; citation_number=38; citation_pages=338-344; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Population Genetic-Structure and Outcrossing Rate of Arabidopsis-Thaliana (L) Heynh.; citation_author=RJ Abbott; citation_author=MF Gomes; citation_journal_title=Heredity; citation_volume=62; citation_number=39; citation_pages=411-418; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Under cover: causes, effects and implications of Hsp90-mediated genetic capacitance.; citation_author=TA Sangster; citation_author=S Lindquist; citation_author=C Queitsch; citation_journal_title=Bioessays; citation_volume=26; citation_number=40; citation_pages=348-362; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=From genotype to phenotype: buffering mechanisms and the storage of genetic information.; citation_author=SL Rutherford; citation_journal_title=Bioessays; citation_volume=22; citation_number=41; citation_pages=1095-1105; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology.; citation_journal_title=Nat Genet; citation_volume=25; citation_number=42; citation_pages=25-29; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=GeneMerge–post-genomic analysis, data mining, and hypothesis testing.; citation_author=CI Castillo-Davis; citation_author=DL Hartl; citation_journal_title=Bioinformatics; citation_volume=19; citation_number=43; citation_pages=891-892; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Induced systemic resistance (ISR) against pathogens in the context of induced plant defences.; citation_author=M Heil; citation_author=RM Bostock; citation_journal_title=Ann Bot (Lond); citation_volume=89; citation_number=44; citation_pages=503-512; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Pseudomonas syringae manipulates systemic plant defenses against pathogens and herbivores.; citation_author=J Cui; citation_author=AK Bahrami; citation_author=EG Pringle; citation_author=G Hernandez-Guzman; citation_author=CL Bender; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=102; citation_number=45; citation_pages=1791-1796; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Signals involved in Arabidopsis resistance to Trichoplusia ni caterpillars induced by virulent and avirulent strains of the phytopathogen Pseudomonas syringae.; citation_author=J Cui; citation_author=G Jander; citation_author=LR Racki; citation_author=PD Kim; citation_author=NE Pierce; citation_journal_title=Plant Physiol; citation_volume=129; citation_number=46; citation_pages=551-564; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Induced Responses to Herbivory.; citation_author=R Karban; citation_author=IT Baldwin; citation_number=47; citation_date=1997; citation_publisher=University of Chicago Press; " />
      <meta name="citation_reference" content="citation_title=Improved binary vectors for Agrobacterium-mediated plant transformation.; citation_author=KE McBride; citation_author=KR Summerfelt; citation_journal_title=Plant Mol Biol; citation_volume=14; citation_number=48; citation_pages=269-276; citation_date=1990; " />
      <meta name="citation_reference" content="citation_title=Mapping the protein regions responsible for the functional specificities of the Arabidopsis MADS domain organ-identity proteins.; citation_author=BA Krizek; citation_author=EM Meyerowitz; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=49; citation_pages=4063-4070; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Removal of a cryptic intron and subcellular localization of green fluorescent protein are required to mark transgenic Arabidopsis plants brightly.; citation_author=J Haseloff; citation_author=KR Siemering; citation_author=DC Prasher; citation_author=S Hodge; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=94; citation_number=50; citation_pages=2122-2127; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=In planta Agrobacterium-mediated transformation of adult Arabidopsis thaliana plants by vacuum infiltration.; citation_author=N Bechtold; citation_author=G Pelletier; citation_journal_title=Methods Mol Biol; citation_volume=82; citation_number=51; citation_pages=259-266; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Generation of enhancer trap lines in Arabidopsis and characterization of expression patterns in the inflorescence.; citation_author=L Campisi; citation_author=Y Yang; citation_author=Y Yi; citation_author=E Heilig; citation_author=B Herman; citation_journal_title=Plant J; citation_volume=17; citation_number=52; citation_pages=699-707; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The water-culture method for growing plants without soils.; citation_author=DR Hoagland; citation_author=DI Arnon; citation_journal_title=California Agr Expt Sta Circ; citation_volume=347; citation_number=53; citation_date=1950; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels"/>
    <meta name="twitter:description" content="The molecular chaperone HSP90 aids the maturation of a diverse but select set of metastable protein clients, many of which are key to a variety of signal transduction pathways. HSP90 function has been best investigated in animal and fungal systems, where inhibition of the chaperone has exceptionally diverse effects, ranging from reversing oncogenic transformation to preventing the acquisition of drug resistance. Inhibition of HSP90 in the model plant Arabidopsis thaliana uncovers novel morphologies dependent on normally cryptic genetic variation and increases stochastic variation inherent to developmental processes. The biochemical activity of HSP90 is strictly conserved between animals and plants. However, the substrates and pathways dependent on HSP90 in plants are poorly understood. Progress has been impeded by the necessity of reliance on light-sensitive HSP90 inhibitors due to redundancy in the A. thaliana HSP90 gene family. Here we present phenotypic and genome-wide expression analyses of A. thaliana with constitutively reduced HSP90 levels achieved by RNAi targeting. HSP90 reduction affects a variety of quantitative life-history traits, including flowering time and total seed set, increases morphological diversity, and decreases the developmental stability of repeated characters. Several morphologies are synergistically affected by HSP90 and growth temperature. Genome-wide expression analyses also suggest a central role for HSP90 in the genesis and maintenance of plastic responses. The expression results are substantiated by examination of the response of HSP90-reduced plants to attack by caterpillars of the generalist herbivore Trichoplusia ni. HSP90 reduction potentiates a more robust herbivore defense response. In sum, we propose that HSP90 exerts global effects on the environmental responsiveness of plants to many different stimuli. The comprehensive set of HSP90-reduced lines described here is a vital instrument to further examine the role of HSP90 as a central interface between organism, development, and environment."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000648.g008"/>

  <meta property="og:title" content="Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=4613'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=590'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9722&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000648">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000648" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000648&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Phenotypic Diversity and Altered Environmental Plasticity in <i>Arabidopsis thaliana</i> with Reduced Hsp90 Levels
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Todd A. Sangster, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Committee on Genetics, University of Chicago, Chicago, Illinois, United States of America, 
                  Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Adam Bahrami, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Amity Wilczek, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Department of Ecology and Evolutionary Biology, Brown University, Providence, Rhode Island, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Etsuko Watanabe, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kurt Schellenberg, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Catherine McLellan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alicia Kelley, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Plants, Soils, and Biometeorology Department, Utah State University, Logan, Utah, United States of America,
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sek Won Kong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Informatics Program, Children's Hospital Boston, Boston, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Christine Queitsch
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:queitsch@u.washington.edu">queitsch@u.washington.edu</a></p>

                <p>Affiliation:
                  FAS Center for Systems Biology, Harvard University, Cambridge, Massachusetts, United States of America
                </p>

                <p>
                    Current address: Department of Genome Sciences, University of Washington, Seattle, Washington, United States of America
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Susan Lindquist
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Whitehead Institute for Biomedical Research, Cambridge, Massachusetts, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: July 25, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000648</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000648-g001" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-t001" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g002" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g003" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g004" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g005" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g006" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g007" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-t002" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000648-g008" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g008" title="Figure 8">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g008&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000648">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000648" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The molecular chaperone HSP90 aids the maturation of a diverse but select set of metastable protein clients, many of which are key to a variety of signal transduction pathways. HSP90 function has been best investigated in animal and fungal systems, where inhibition of the chaperone has exceptionally diverse effects, ranging from reversing oncogenic transformation to preventing the acquisition of drug resistance. Inhibition of HSP90 in the model plant <em>Arabidopsis thaliana</em> uncovers novel morphologies dependent on normally cryptic genetic variation and increases stochastic variation inherent to developmental processes. The biochemical activity of HSP90 is strictly conserved between animals and plants. However, the substrates and pathways dependent on HSP90 in plants are poorly understood. Progress has been impeded by the necessity of reliance on light-sensitive HSP90 inhibitors due to redundancy in the <em>A. thaliana</em> HSP90 gene family. Here we present phenotypic and genome-wide expression analyses of <em>A. thaliana</em> with constitutively reduced HSP90 levels achieved by RNAi targeting. HSP90 reduction affects a variety of quantitative life-history traits, including flowering time and total seed set, increases morphological diversity, and decreases the developmental stability of repeated characters. Several morphologies are synergistically affected by HSP90 and growth temperature. Genome-wide expression analyses also suggest a central role for HSP90 in the genesis and maintenance of plastic responses. The expression results are substantiated by examination of the response of HSP90-reduced plants to attack by caterpillars of the generalist herbivore <em>Trichoplusia ni</em>. HSP90 reduction potentiates a more robust herbivore defense response. In sum, we propose that HSP90 exerts global effects on the environmental responsiveness of plants to many different stimuli. The comprehensive set of HSP90-reduced lines described here is a vital instrument to further examine the role of HSP90 as a central interface between organism, development, and environment.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Sangster TA, Bahrami A, Wilczek A, Watanabe E, Schellenberg K, et al.  (2007) Phenotypic Diversity and Altered Environmental Plasticity in <em>Arabidopsis thaliana</em> with Reduced Hsp90 Levels. PLoS ONE 2(7):
          e648.
            doi:10.1371/journal.pone.0000648</p><p><strong>Academic Editor: </strong>Justin Borevitz, University of Chicago, United States of America</p><p><strong>Received:</strong> May 18, 2007; <strong>Accepted:</strong> June 11, 2007; <strong>Published:</strong> July 25, 2007</p><p><strong>Copyright:</strong> © 2007 Sangster et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding for this project was provided by the G. Harold and Lelia Y. Mathers Foundation to S.L. TAS was supported by a Howard Hughes Medical Institute pre-doctoral fellowship.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The essential eukaryotic chaperone HSP90 facilitates the maturation of a variety of metastable protein substrates, many of which are central regulators of biological circuits <a href="#pone.0000648-Young1">[1]</a>–<a href="#pone.0000648-Wegele1">[3]</a>. Due to HSP90's central position in numerous pathways regulating growth and development, the chaperone has emerged as a principal focus of investigation in diverse fields. For example, as key proteins controlling cell proliferation depend on HSP90, it is an intensively investigated target for anti-cancer therapies <a href="#pone.0000648-Whitesell1">[4]</a>. More broadly, HSP90 activity has been shown to have widespread effects on the genesis of phenotypic diversity. HSP90 can suppress the phenotypic consequences of existing genetic polymorphisms <a href="#pone.0000648-Queitsch1">[5]</a>, <a href="#pone.0000648-Rutherford1">[6]</a>, affect epigenetically inherited phenotypes <a href="#pone.0000648-Sollars1">[7]</a>, and facilitate the spread of novel mutations <a href="#pone.0000648-Cowen1">[8]</a>. Further, reduced HSP90 function can increase the probability that stochastic events inherent to development result in phenotypic differences <a href="#pone.0000648-Queitsch1">[5]</a>, <a href="#pone.0000648-Milton1">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Molecular chaperones are proteins which can modulate the folding of a variety of other polypeptides without permanent alteration to themselves <a href="#pone.0000648-Wegele1">[3]</a>. HSP90 is distinct among molecular chaperones in its functionally diverse but numerically restricted set of substrates (client proteins), which tend to share an inherent conformational instability. HSP90's interactions with certain mammalian hormone receptors, such as the glucocorticoid receptor <a href="#pone.0000648-Pratt1">[10]</a>, <a href="#pone.0000648-Picard2">[11]</a>, serve as a paradigm for its molecular function. HSP90 maintains such substrates in a signaling competent state, enabling them to receive upstream signals, such as hormone binding. Substrates then typically undergo a conformational change, thereby propagating signal transduction to downstream effectors <a href="#pone.0000648-Picard1">[2]</a>, <a href="#pone.0000648-Wegele1">[3]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>HSP90 function and substrates have been best investigated in mammalian and fungal systems <a href="#pone.0000648-Zhao1">[12]</a>. However, the biochemical function of HSP90—to promote the maturation of metastable client proteins—is conserved in plants, as illustrated by the fact that obligately HSP90-dependent mammalian steroid hormone receptors are functional in the plant cytoplasm, even though plants lack any homologous receptors <a href="#pone.0000648-Stancato1">[13]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the plant <em>Arabidopsis thaliana</em>, HSP90 homologs are encoded by seven different genetic loci. Of these, one is expressed in the endoplasmic reticulum (<em>HSP90.7</em>), one in the mitochondrion (<em>HSP90.6</em>), one in the chloroplast (<em>HSP90.5</em>), and four in the cytosol. The gene encoding one cytosolic protein (<em>HSP90.1</em>/At5g52640) is highly stress-inducible, whereas the other three (<em>HSP90.2</em>/At5g56030, <em>HSP90.3</em>/At5g56010, and <em>HSP90.4</em>/At5g56000) are constitutively expressed and are the products of very recent duplication events <a href="#pone.0000648-Yabe1">[14]</a>, <a href="#pone.0000648-Krishna1">[15]</a>. Two of the organelle specific HSP90s, have been shown to affect plant development. A choloroplast HSP90 mutant, <em>cr88</em>, alters responses to red light, chlorate resistance, and delays chloroplast development <a href="#pone.0000648-Cao1">[16]</a>. An endoplasmic reticulum HSP90 mutant, <em>shepherd</em>, affects apical meristem maintenance, presumably due to the failure to adequately chaperone the <em>CLAVATA1/2</em> complex <a href="#pone.0000648-Ishiguro1">[17]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>The functions of the cytosolic HSP90s are not well understood; mutations in these genes have only been phenotypically connected to increased sensitivity to biotrophic parasites, primarily microbial pathogens and viruses <a href="#pone.0000648-Takahashi1">[18]</a>–<a href="#pone.0000648-Kanzaki1">[24]</a>. The pathways that HSP90 is known to influence share dependency on R-proteins, which facilitate pathogen recognition and initiate downstream defense responses, such as localized cell death (hypersensitive response) and induced systemic resistance <a href="#pone.0000648-Glazebrook1">[25]</a>. Several groups have demonstrated the essential role of HSP90 in maintaining the stability of several R-proteins <a href="#pone.0000648-Takahashi1">[18]</a>, <a href="#pone.0000648-Hubert1">[19]</a>, <a href="#pone.0000648-Lu1">[21]</a>, <a href="#pone.0000648-Azevedo1">[26]</a>, <a href="#pone.0000648-Holt1">[27]</a>. If HSP90 function is missing, these R-proteins are rapidly degraded, akin to observations with HSP90 substrates in other systems <a href="#pone.0000648-Schneider1">[28]</a>, presumably leading to pathogen sensitivity.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>A point mutation (<em>hsp90.2-3</em>) in the ATP-binding domain of one of the constitutive cytosolic HSP90s, HSP90.2, renders the plant sensitive to pathogens <a href="#pone.0000648-Hubert1">[19]</a>. However, a null mutation in <em>HSP90.2</em> failed to display the same phenotype. Point mutations in similar positions to <em>hsp90.2-3</em> in yeast HSP90 proteins have been analyzed biochemically and found to prevent either ATP binding or hydrolysis <a href="#pone.0000648-Prodromou1">[29]</a>, <a href="#pone.0000648-Grenert1">[30]</a>, thereby interfering with substrate release <a href="#pone.0000648-Young2">[31]</a>. Therefore, the likely substrate-trapping properties of the <em>hsp90.2-3</em> mutation may result in some dominant characteristics and stronger phenotypes when compared to a corresponding null mutation.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7"></a><p>Neither the <em>HSP90.2</em> null mutation nor <em>hsp90.2-3</em> displayed common obvious morphological abnormalities in previous work <a href="#pone.0000648-Hubert1">[19]</a>, suggesting redundancy between different cytosolic isoforms. Inhibition of HSP90 activity with geldanamycin (GDA), an inhibitor that is specific to the HSP90 family of proteins, resulted in dramatically increased phenotypic variation in ten day old seedlings, affecting leaf shape and color, hypocotyl elongation, and gravitropism responses among others <a href="#pone.0000648-Queitsch1">[5]</a>. As GDA is light-sensitive, HSP90-dependent phenotypes could only be analyzed at the seedling stage.</p>
<a id="article1.body1.sec1.p8" name="article1.body1.sec1.p8"></a><p>To comprehensively investigate the phenotypic consequences of reduced cytosolic HSP90 function, we obtained lines with transfer DNA (T-DNA) insertions in <em>HSP90.1, HSP90.2</em>, or <em>HSP90.3</em>, representing null mutations in the respective isoforms <a href="#pone.0000648-Alonso1">[32]</a>. As <em>HSP90.2, HSP90.3</em>, and <em>HSP90.4</em> are located within 12000 base pairs of each other <a href="#pone.0000648-The1">[33]</a>, the construction of double mutants for constitutive isoforms is extremely difficult. Therefore, we decided to target the cytosolic HSP90s with a RNA interference (RNAi) based approach, creating a set of lines with reductions in the levels of several cytosolic isoforms. Herein, we describe the phenotypic and molecular characterization of all of these HSP90-reduced lines.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Construction of HSP90-Reduced Lines</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>To interfere with the expression of multiple cytosolic HSP90 isoforms, we cloned four regions of HSP90 with differing sequence identity across the four cytosolic isoforms. Construction of double-strand RNA producing vectors (<a href="#pone-0000648-g001">Fig. 1</a>) and transformation into the <em>A. thaliana</em> Col-0 accession were performed as described in <a href="#pone.0000648-Chuang1">[34]</a> with the modifications detailed in the <a href="#s4">Methods</a>. Control lines were created with an identical vector lacking any HSP90 sequence. The sequence-verified constructs contained a kanamycin resistance cassette, allowing screening for positive transformants in the T<sub>1</sub> generation. In total, vectors containing four different regions of HSP90 were tested. Screening of ~50,000 total seeds for each construct from three independent transformations yielded at least ten transgenic lines for three of the constructs. No transformants were obtained for the fourth construct (RNAi-D), which had the highest likelihood of interfering with expression of all four isoforms. Given the success with the other constructs, it appears likely that HSP90-reduction caused by the RNAi-D construct is severe enough to cause lethality under standard growth conditions.</p>
<div class="figure" id="pone-0000648-g001"><div class="img"><a name="pone-0000648-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Targeting HSP90 with double-strand RNA constructs.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The four cytosolic HSP90s are depicted in their genomic context. Introns are in black, exons in white. Regions cloned to create the double-strand RNA (RNAi) constructs are indicated in green (RNAi-A), blue (RNAi-B), red (RNAi-C), and yellow (RNAi-D) bars. Below each isoform the percent identity and the longest consecutive number of bases of perfect identity to the cloned region are noted for each construct.</p>
<span>doi:10.1371/journal.pone.0000648.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Positive transformants were self-propagated and analyzed for kanamycin resistance frequency in the T<sub>2</sub>; lines with frequencies consistent with segregation of a single insert were selected for further analysis. Kanamycin resistance was used to isolate true-breeding lines for the constructs from the T<sub>3</sub> generation. Of these, five lines, including three containing HSP90-RNAi construct A and one containing each of constructs B or C, were chosen for subsequent experiments, along with three independent control lines. Insert locations for most lines were determined by standard methods <a href="#pone.0000648-Liu2">[35]</a>; none of the isolated insert locations were within a coding region (<a href="#s4">Methods</a>).</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>As a complementary resource, we obtained lines with single T-DNA insertions in the <em>HSP90.1</em>, <em>HSP90.2</em>, and <em>HSP90.3</em> isoforms (HSP90-TDNA lines) <a href="#pone.0000648-Alonso1">[32]</a> from the Arabidopsis Biological Resource Center. Lines homozygous for each insert were isolated by PCR screening.</p>


<h4>Assessment of HSP90-Reduction</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Quantitative Western blotting was used to confirm that HSP90 protein levels are constitutively decreased under ambient conditions in the HSP90-RNAi and HSP90-TDNA lines. The antibody employed should recognize all four cytosolic HSP90s <a href="#pone.0000648-Krishna2">[36]</a>. All lines exhibited significantly reduced HSP90 levels except for the HSP90.1-TDNA line (<a href="#pone-0000648-t001">Table 1</a>). As HSP90.1 is constitutively expressed at low levels, a null mutant in this isoform would not be expected to exhibit HSP90 reduction under ambient conditions.</p>
<div class="figure" id="pone-0000648-t001"><div class="img"><a name="pone-0000648-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>HSP90 levels in HSP90-RNAi and HSP90-TDNA lines.</span></strong></p><span>doi:10.1371/journal.pone.0000648.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Consistent with these results, quantitative RT-PCR of RNA derived from seedlings which had undergone a brief heat-shock revealed significantly lower levels of HSP90 RNA in the HSP90-RNAi lines (<a href="#pone-0000648-t001">Table 1</a>). The reduction is reproducibly at least two-fold (one cycle) in all lines. The primers used in this reaction amplify <em>HSP90.1</em> only. Importantly, the reduction in HSP90 levels also appears to be stable across generations (<a href="#pone-0000648-t001">Table 1</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3"></a><p>Further confirmation that HSP90 levels are reduced in both HSP90-RNAi and HSP90-TDNA lines was derived from genome-wide expression analyses. All HSP90-RNAi and HSP90-TDNA lines were analyzed, along with the <em>hsp90.2-3</em> mutant. Ten 14 day old seedlings per genotype were pooled for each sample, and replicate samples were analyzed with the Affymetrix ATH-1 microarray. All array data may be found in the Gene Expression Omnibus under accession number GSE7796.</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4"></a><p>Standard methods were used to determine which genes were differentially expressed between each HSP90-reduced line and the appropriate control (<a href="#s4">Methods</a>). Our expression data confirmed that the cytosolic HSP90 isoforms tended to be reduced between 1.5 and 2 fold in a statistically significant manner in all HSP90-RNAi lines; individual isoforms were reduced to different extents in different lines. The magnitude of the reduction of total HSP90 RNA levels in each line is consistent with the RT-PCR data. <em>HSP90.1</em> and <em>HSP90.2</em> transcripts were significantly reduced to nearly undetectable levels in the respective HSP90-TDNA lines. The probes on the array cannot differentiate between <em>HSP90.3</em> and <em>HSP90.4</em>; the combined transcript levels of these genes were significantly reduced six fold in the HSP90.3-TDNA line. Note that in the HSP90.2-TDNA and <em>hsp90.2-3</em> mutants, <em>HSP90.1</em> levels were two-fold upregulated, indicating cross-regulation between different isoforms. This subtle increase in <em>HSP90.1</em> expression is likely due to a slight induction of the heat-stress response in the HSP90-TDNA lines, as other heat-stress markers such as <em>HSP101</em> show similar levels of induction. The induction of the heat-stress response by reduced levels of HSP90 has been previously observed in other systems <a href="#pone.0000648-Duina1">[37]</a>, <a href="#pone.0000648-Murakami1">[38]</a>. Such induction was not observed in any of the HSP90-RNAi lines, likely due to their lower levels of total HSP90 reduction.</p>


<h4>Increased Phenotypic Diversity in HSP90-Reduced Seedlings</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We previously reported an increase in phenotypic diversity of <em>A. thaliana</em> seedlings when HSP90 was pharmacologically inhibited by growth on GDA <a href="#pone.0000648-Queitsch1">[5]</a>. In an initial examination, we observed similar phenotypes, as well as similar phenotypic diversity, in the HSP90-RNAi lines. These phenotypes included the accumulation of purple pigment, an HSP90-dependent phenotype specific to the Col-0 accession in the previous study (<a href="#pone-0000648-g002">Fig. 2</a>). As <em>A. thaliana</em> is an inbreeding species<a href="#pone.0000648-Abbott1">[39]</a>, common laboratory accessions such as Col-0 are homozygous at virtually all nucleotides. Therefore, the phenotypic diversity revealed within the Col-0 accession by HSP90 reduction is highly unlikely to be due to genetic differences between individual plants. Rather, the diversity represents the outcome of stochastic developmental processes <a href="#pone.0000648-Queitsch1">[5]</a>.</p>
<div class="figure" id="pone-0000648-g002"><div class="img"><a name="pone-0000648-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Similar morphological phenotypes of seedlings with reduced HSP90 function by RNAi or pharmacological means (GDA).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(a) and (b): purple pigment accumulation; (c) and (d): organ number defect; (e) and (f): narrowly-shaped deformed true leaves; (g) and (h): twisted rosettes; (i) and (j): lobed cotyledon. RNAi plants are T<sub>3</sub> generation with from line RNAi-A3. Size bar 2 mm for b and g–i, 1 mm for a, c–f, and 3 mm for normal phenotype. (b) and (f) originally published in <a href="#pone.0000648-Queitsch1">[5]</a>.</p>
<span>doi:10.1371/journal.pone.0000648.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>As plants are exquisitely sensitive to environmental perturbations and often display apparently abnormal phenotypes even under non-stressful conditions, we set out to systematically investigate the phenotypic diversity in HSP90-reduced lines compared with appropriate controls. To reduce the likelihood that unrelated mutations arising during the transformation process might alter the observed spectrum of phenotypes, all transgenic lines were backcrossed twice to the Col-0 parent, with subsequent selection for insert homozygosity. Three hundred seventy-eight seeds for all five HSP90-RNAi and the three control lines were plated in a randomized block design such that nine seeds of each line were planted on each block of two plates containing thirty-six seeds each. Morphologically deviating phenotypes were scored after ten days according to a rubric standardized before the experiment (<a href="#pone-0000648-g003">Fig. 3</a>). To prevent experimenter bias, seed plating and phenotype scoring were conducted by a single scientist fully blind with regard to genotype identity.</p>
<div class="figure" id="pone-0000648-g003"><div class="img"><a name="pone-0000648-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Examples of standardized seedling morphological phenotypes for quantitative assessment.</span></strong></p><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1"></a><p>(a) no scored phenotype; (b) extra leaf, concave cotyledons, aerial organs on growth medium; (c) missing leaf; (d) narrow leaves, delayed development; (e) shoot meristemless, delayed development; (f) necrotic spots on leaves; (g) purple pigmentation in cotyledons, delayed development; (h) curled hypocotyl with root extruded from medium; (i) no hypocotyl, aerial organs on growth medium, delayed development. All seedlings are ten days old. Size bar 2 mm.</p>
<span>doi:10.1371/journal.pone.0000648.g003</span></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>No trait frequencies differed significantly between the three control lines; all controls were therefore combined for statistical analysis. For all five genetically HSP90-reduced lines, significantly more plants were scored as displaying at least one standardized abnormal phenotype than for the control lines (<a href="#pone-0000648-g004">Fig. 4</a>; p&lt;0.0001 for each line, χ<sup>2</sup> test). With the exception of RNAi-B1, all HSP90-RNAi lines showed significant enrichment for the accumulation of purple pigment in the cotyledons (p&lt;&lt;0.001 for each line, χ<sup>2</sup> test), consistent with previous data <a href="#pone.0000648-Queitsch1">[5]</a>. Another cotyledon phenotype, concave cotyledons, was overrepresented in all HSP90-RNAi lines (p&lt;&lt;0.001 for each line, χ<sup>2</sup> test). A major phenotypic difference between control and HSP90-reduced lines, excepting RNAi-B1, appeared to be the shape and timing of formation of true leaves. These HSP90-RNAi plants tended to produce narrow leaves which were often temporally delayed and therefore smaller than their counterparts on plants without the phenotype (p&lt;&lt;0.001 for each line, χ<sup>2</sup> test); this phenotype tended to co-occur with the purple pigment accumulation (p&lt;&lt;0.001, χ<sup>2</sup> test). No other correlations were found between other discussed phenotypes after controlling for genotype. The most common scored phenotype in both control and HSP90-RNAi plants was failure to elevate plant organs above the growth substrate; however, this phenotype was significantly more prevalent in HSP90-RNAi lines. Notably, one phenotype was significantly more prevalent in control lines: appearance of chlorotic, seemingly dead, spots on the first true leaves.</p>
<div class="figure" id="pone-0000648-g004"><div class="img"><a name="pone-0000648-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Quantitation of selected standard seedling phenotypes.</span></strong></p><a id="article1.body1.sec2.sec3.fig3.caption1.p1" name="article1.body1.sec2.sec3.fig3.caption1.p1"></a><p>(a) comparison of phenotypic frequency in percent between five HSP90-RNAi lines and three control lines (control data merged). (b) similar comparison between three HSP90-TDNA lines and Col-0. Error bars represent standard error. *p&lt;0.01, **p&lt;0.001; χ<sup>2</sup> test.</p>
<span>doi:10.1371/journal.pone.0000648.g004</span></div><a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>A similar experiment was conducted to assess phenotypes in the HSP90-TDNA lines. Note that as several HSP90-dependent phenotypes have previously been demonstrated to be exceptionally environmentally sensitive, phenotypic frequencies from different experiments cannot be directly compared due to potential differences in microenvironment. Importantly, aberrant phenotypes in all HSP90-TDNA lines were significantly enriched relative to the control (<a href="#pone-0000648-g004">Fig. 4</a>). T-DNA insertions in the <em>HSP90.2</em> and <em>HSP90.3</em> constitutive isoforms yielded a highly significant overrepresentation of seedlings with narrow leaves and delayed development (p&lt;&lt;0.001 for both lines, χ<sup>2</sup> test), as observed with four of the HSP90-RNAi lines. Consistent with the observation that <em>HSP90.1</em> exhibits very low levels of constitutive expression <a href="#pone.0000648-Yabe1">[14]</a>, the HSP90.1-TDNA line was not significantly enriched for any particular scored phenotype in the non-stressful environment employed. However, significantly more plants were scored as displaying at least one abnormal phenotype (p&lt;0.001, χ<sup>2</sup> test), which may indicate that the lack of inducible HSP90 amplifies the stochastic effects of microenvironments. For a complete list of scored phenotypes, phenotype frequencies, and associated p-values, see <a href="#pone.0000648.s001">Tables S1</a> and <a href="#pone.0000648.s002">S2</a>. In summary, a reduction in HSP90 levels results in greater phenotypic diversity, consistent with previously published hypotheses on HSP90's role in canalization <a href="#pone.0000648-Queitsch1">[5]</a>, <a href="#pone.0000648-Sangster1">[40]</a>, <a href="#pone.0000648-Rutherford2">[41]</a>.</p>


<h4>Enhancement of HSP90-Dependent Phenotypes by Altered Temperature</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>As previous studies have described HSP90-dependent phenotypes that can be enhanced or mimicked by temperature treatment <a href="#pone.0000648-Queitsch1">[5]</a>, <a href="#pone.0000648-Rutherford1">[6]</a>, we investigated whether the genetically HSP90-reduced lines recapitulate this phenomenon (<a href="#pone.0000648.s003">Table S3</a>). We analyzed seedling phenotypes as above, but at three different growth temperatures. Since higher temperatures accelerate development, seedlings were scored at different times after germination but at comparable developmental stages. Our previous studies indicated that the narrow leaf phenotype was revealed at higher growth temperatures (27°C) in genetically predisposed lines <a href="#pone.0000648-Queitsch1">[5]</a>. In agreement, increased temperature significantly increased the frequency of this phenotype in predisposed HSP90-RNAi lines (p&lt;&lt;0.001 by logistic regression of temperature versus phenotype occurrence); no such increase was observed in the control (<a href="#pone-0000648-g005">Fig. 5</a>). Note that the severity of the phenotype was also dramatically enhanced, with 3.5 percent of HSP90-RNAi plants presenting as shoot meristemless at the highest temperature (p&lt;0.001 by logistic regression).</p>
<div class="figure" id="pone-0000648-g005"><div class="img"><a name="pone-0000648-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Response of selected seedling phenotypes to growth temperature.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>(a) narrow leaf phenotype; (b) shoot meristemless phenotype; (c) purple pigment accumulation; and (d) one of first true leaves missing. Phenotypes for two HSP90-RNAi lines and one control were measured at 17, 22, and 27°C. Error bars represent standard error.</p>
<span>doi:10.1371/journal.pone.0000648.g005</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>The number of seedlings missing one of the first true leaves was also significantly enhanced by increased temperature in the HSP90-RNAi lines (p = 0.001, χ<sup>2</sup> test of 22°C versus 27°C), but not in controls (<a href="#pone-0000648-g005">Fig. 5</a>). Significant deviations of the development of repeated characters are commonly associated with a decrease in developmental stability. Therefore, not only does a decrease in HSP90 function result in lower developmental stability, but this effect is further exacerbated by elevated temperature.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Notably, certain HSP90-dependent phenotypes are not exacerbated by increased temperature; rather, the frequency and severity decreases <a href="#pone.0000648-Rutherford1">[6]</a>. For example, the frequency of the accumulation of purple pigment in cotyledons is dramatically decreased at higher growth temperatures (<a href="#pone-0000648-g005">Fig. 5</a>; p&lt;&lt;0.001 by logistic regression). Indeed, even control plants significantly displayed this phenotype at the lowest tested temperature (p = 0.003, Fisher's Exact Test of 17°C versus 22°C), though at a lower frequency than HSP90-RNAi seedlings. Taken together, these data support a role for HSP90 in the genesis of plant plasticity, the maintenance of wild-type plastic responses, and the promulgation of developmental stability.</p>


<h4>Quantitative Phenotypic Differences in Adult HSP90-Reduced Plants</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The genetically HSP90-reduced sets are ideal instruments to examine HSP90's role in the adult plant, as our previous work was limited by the light-sensitivity of GDA to the assessment of the effects of reducing HSP90 function in seedlings. To investigate phenotypes enriched in adult HSP90-reduced plants, we measured a variety of life-history traits and altered morphologies. Forty plants of each of the HSP90-RNAi, HSP90-TDNA, and appropriate control lines were grown in a randomized design under standard greenhouse conditions (see <a href="#s4">Methods</a>).</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>Two phenotypes affecting gross morphology were significantly enriched in HSP90-RNAi lines (<a href="#pone-0000648-g006">Fig. 6</a>). The most frequent was loss of apical dominance, leading to the simultaneous emergence of multiple primary inflorescences. Plants which grew multiple inflorescences as adults tended to be those which exhibited narrow first true leaves or other shoot meristem defects as seedlings (<a href="#pone-0000648-g007">Fig. 7</a>), indicating that the adult phenotype is likely a result of the early defect. This phenotype was observed in all five HSP90-RNAi lines (48% total frequency; p&lt;&lt;0.001 versus control, Fisher's Exact Test) but not in controls or in HSP90-TDNA lines.</p>
<div class="figure" id="pone-0000648-g006"><div class="img"><a name="pone-0000648-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Examples of altered morphologies observed in adult HSP90-reduced plants.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(a) control inflorescence; (b and c) fasciated inflorescence; (d) normal morphology with single primary inflorescence; (e) multiple simultaneous developing inflorescences. White arrows in d and e denote primary inflorescences. Scale bar 5 mm for a–c, 1 cm for d,e.</p>
<span>doi:10.1371/journal.pone.0000648.g006</span></div><div class="figure" id="pone-0000648-g007"><div class="img"><a name="pone-0000648-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Developmental progression of common altered shoot meristem development phenotype.</span></strong></p><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1"></a><p>(a–f): Seedling with RNAi-A construct exhibiting delayed development of true leaves and shoot meristem organization defects resulting in overproliferation of leaves. (g–l): Seedling with control construct with normal morphology. (a and g) 7 days after germination; (b and h) 10 days; (c and i) 14 days; (d and j) 18 days; (e and k) 21 days; (f and l) 25 days. Scale bar 1 mm for a–c, g–h, 2 mm for i, and 1.5 cm for d–f, j–l. These seedlings are of the Shahdara ecotype and are used to illustrate the phenotype; the developmental progression of affected Col-0 seedlings is similar.</p>
<span>doi:10.1371/journal.pone.0000648.g007</span></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>The second abnormal morphological phenotype presented as multiple shoots which initially developed fused together but eventually separated (<a href="#pone-0000648-g006">Fig. 6</a>). This morphology occurred in all HSP90-RNAi lines except for RNAi-B1 (4.7% frequency in combined HSP90-RNAi lines, p = 0.0216 versus control, Fisher's Exact Test) but not in any controls or HSP90-TDNA plants. No homeotic transformations were observed in any plant.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4"></a><p>Several quantitative life-history traits were altered in Col-0 HSP90-reduced plants (<a href="#pone-0000648-t002">Table 2</a>). All lines demonstrated to have constitutively lower HSP90 expression levels flowered significantly later than controls (p&lt;0.05 for each line versus control, Tukey's HSD). As <em>A. thaliana</em> ceases to produce rosette leaves when flowering commences, leaf number is correlated with flowering time; all five HSP90-RNAi lines had a significantly greater number of leaves than the control (p&lt;0.05 for each line versus control, Tukey's HSD). Except for line RNAi-B1, all the RNAi lines were significantly reduced in rosette diameter (p&lt;0.05 for each line versus control, Tukey's HSD). The total weight of seeds produced by each HSP90-reduced line was lower than all control lines (p&lt;&lt;0.001 by ANOVA of all HSP90 reduced lines versus controls). As the average seed weight was not significantly affected by HSP90 levels, HSP90-reduced plants produce slightly fewer seeds and likely experience a small reduction in fitness in the environment tested. All of these phenotypes were affected by the presence of multiple primary inflorescences; however, all still significantly differed in HSP90-reduced plants after controlling for the altered morphology (data not shown). Therefore, the HSP90-dependent morphological and quantitative phenotypic differences do not constitute a single syndrome resulting from HSP90 reduction.</p>
<div class="figure" id="pone-0000648-t002"><div class="img"><a name="pone-0000648-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Quantitation of selected life-history traits.</span></strong></p><span>doi:10.1371/journal.pone.0000648.t002</span></div><a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5"></a><p>Several other measured phenotypes were significantly different between some HSP90-RNAi lines and the controls, including plant height, silique number, and dry aerial mass. These differences could be statistically explained by the presence of multiple primary meristems and are likely a consequence of the early morphological difference.</p>


<h4>Genome-Wide Expression Analyses Suggest Altered Plastic Responses in HSP90-Reduced Lines</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>As little is known about HSP90's functions in plants, we attempted to extract biologically relevant pathways modulated by HSP90 from the genome-wide expression analyses discussed above. Using the several hundred significantly changed genes in each line, we determined if the significant genes had an increased frequency, when compared to all genes on the array, of Gene Ontology (GO)-defined biological processes, molecular functions, and cellular components <a href="#pone.0000648-The2">[42]</a> with GeneMerge <a href="#pone.0000648-CastilloDavis1">[43]</a>. All enrichments and corresponding significances are included in <a href="#pone.0000648.s004">Dataset S1</a>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>A comparison of all HSP90-reduced lines versus all controls reveals that the environmental responsiveness of the HSP90-reduced plants is generally altered. For example, the biological processes “response to abscisic acid stimulus” (p = 0.0002; hypergeometric distribution test with multiple comparison correction), “response to water deprivation” (p = 0.0004), and “jasmonic acid biosynthetic process” (p = 0.037) were significantly enriched among upregulated genes. Similarly, the molecular function “transcription factor activity” (p = 0.008) was significantly enriched among upregulated genes, consistent with HSP90's established role in the maturation of signal transduction proteins in other organisms <a href="#pone.0000648-Picard1">[2]</a>, <a href="#pone.0000648-Wegele1">[3]</a>.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>When examining GO category enrichments in individual lines, many more significant values were found in the HSP90-RNAi lines than in the HSP90-TDNA lines, likely due to the observed heat-stress induction in the latter. The hypothesis that HSP90 functions to allow the plant to respond appropriately to environmental cues was strongly supported by the analysis of individual HSP90-RNAi lines. Enriched biological processes included responses to various hormones and abiotic and biotic stresses and stimuli. The “endomembrane system” cellular compartment category was consistently enriched in genes with significantly altered expression. Most of the genes contributing to this category enrichment were again those involved in responses to environmental stimuli. Unlike the single isoform insertion mutants, the <em>hsp90.2-3</em> mutant extensively shared the enrichments in environmental response categories observed with the HSP90-RNAi lines.</p>


<h4>Response to Herbivory</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>Previous studies have demonstrated that HSP90 inhibition results in the attenuation of several R-gene signaling pathways, leading to increased sensitivity to biotrophic pathogens, including viruses, microbial pathogens, and certain metazoan parasites <a href="#pone.0000648-Takahashi1">[18]</a>–<a href="#pone.0000648-Kanzaki1">[24]</a>. R-gene-dependent defense involves salicyclic acid (SA) mediated signaling <a href="#pone.0000648-Glazebrook1">[25]</a>. SA-mediated defense against microbial pathogens and jasmonic acid (JA)-mediated defense against insect herbivores <a href="#pone.0000648-Heil1">[44]</a> have been suggested to be antagonistically regulated <a href="#pone.0000648-Cui1">[45]</a>, <a href="#pone.0000648-Cui2">[46]</a>. Our expression analysis is in agreement with this antagonistic regulation, as it revealed that plants reduced in HSP90 generally display upregulation of genes involved in jasmonic acid biosynthesis and that several lines exhibit downregulation of salicylic acid biosynthesis genes. Therefore, we used caterpillars of the cabbage looper <em>Trichoplusia ni</em> to test the basal resistance of all HSP90-reduced lines to a generalist herbivore.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>Plant resistance to the herbivore was quantified by the dry weight of a single caterpillar after seven days of feeding on a single plant, as weight gain of the caterpillar is inversely correlated to plant resistance. To control for possible effects of microenvironments, all experiments were randomized with respect to plant genotype. No consistent significant change in basal resistance was observed for individual T-DNA insertion mutants or for the HSP90-RNAi lines except for RNAi-B1, which was more resistant than the control. The most significant and reproducible difference to controls was observed with the <em>hsp90.2-3</em> mutant, known to be more sensitive to biotrophic pathogens, which was significantly more resistant to herbivore attack. In this experiment, we also observed that the <em>hsp90.2-3</em> mutant plants flowered significantly later than the control in these growth conditions (data not shown), consistent with the later flowering time of other HSP90-reduced plants reported above.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Previous studies have established that plants induce lasting responses to herbivores upon attack <a href="#pone.0000648-Karban1">[47]</a>. We next addressed whether the constitutively upregulated defense responses of the <em>hsp90.2-3</em> mutant are already maximal or whether they could be further induced. One three-day old <em>T. ni</em> caterpillar was caged on one standardized leaf per plant for 24 hours. Plants were allowed to recover and induce resistance for four days, after which a second <em>T. ni</em> was added as before. The <em>hsp90.2-3</em> could indeed induce further resistance (<a href="#pone-0000648-g008">Fig. 8</a>). This experiment also included the HSP90.1-TDNA and HSP90.2-TDNA insertion mutants, which were not significantly different from the control without induction. Interestingly, both these lines were significantly more resistant than the control after the induction treatment.</p>
<div class="figure" id="pone-0000648-g008"><div class="img"><a name="pone-0000648-g008" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000648" data-uri="info:doi/10.1371/journal.pone.0000648.g008"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000648.g008&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g008/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g008/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g008/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g008/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g008.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000648.g008/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000648.g008/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000648.g008.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 8.  <span>Response to herbivore attack differs in HSP90-reduced lines.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Response to herbivory was measured by <em>T. ni</em> caterpillar dry weight. Least-squares means of caterpillar weight are plotted for each genotype and treatment (see <a href="#s4">Methods</a>). Top: Basal and induced herbivore responses. For each pair, lighter colors denote basal response, darker colors indicate response after prior herbivore attack. Bottom: Herbivore response after treatment with different concentrations of Me-JA. Control in shades of green; <em>hsp90.2-3</em> in shades of blue. Lighter colors indicate higher Me-JA concentration. Statistical analysis by pairwise Student's t-tests; the response of lines sharing any letter is not statistically different. Separate statistical analysis conducted for top and bottom panels.</p>
<span>doi:10.1371/journal.pone.0000648.g008</span></div><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>A final experiment examined whether the constitutive resistance observed in the <em>hsp90.2-3</em> mutant resulted in saturation of JA signaling. Three concentrations of methyl-JA (Me-JA) were applied to <em>hsp90.2-3</em> and control plants, followed by the <em>T. ni</em> resistance bioassay. In both genotypes, herbivore resistance increased with increasing concentrations of Me-JA. However, the <em>hsp90.2-3</em> mutant was more resistant than the control for a given concentration of Me-JA (<a href="#pone-0000648-g008">Fig. 8</a>). Thus, the signaling capacity of the JA pathway is not saturated in the <em>hsp90.2-3</em> mutant. The observed additive effect of genotype and treatment does not exclude the hypothesis that reductions in HSP90 capacity result in increased herbivore resistance by a JA-independent pathway. In summary, the increased sensitivity to microbial pathogens and decreased sensitivity to herbivores of the <em>hsp90.2-3</em> mutant suggest an important role for HSP90 in the integration of diverse environmental stimuli into appropriate organismal responses.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Our analysis of plants with genetically reduced HSP90 levels establishes that HSP90 is crucial to maintain wild-type morphology against stochastic developmental or environmental perturbation. Further, we demonstrate that HSP90 is a central regulator of the plastic responses of plants to their environment.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>To determine the phenotypic consequences of reduced HSP90 function in <em>A. thaliana</em>, we employed complementary sets of HSP90-reduced lines, comparing lines with multiple cytosolic isoforms reduced via an RNAi approach with T-DNA insertion lines representing null mutants of individual isoforms. Note that the HSP90 reduction in these lines is not severe enough to interfere with plant viability or the health of most tested individuals. Therefore, these lines allow the assessment of traits which are most dependent on full HSP90 function.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>To control for possible insertion site effects of the RNAi constructs, five independent HSP90-RNAi lines were analyzed, and constructs with sequence specificity to three different portions of the HSP90 isoforms were used. As HSP90-dependent phenotypes tend to be shared among the different HSP90-reduced lines and our analysis of the whole-genome expression data suggests that off-target effects are highly unlikely, we conclude that our results represent a true description of the effects of modest HSP90 reduction in <em>A. thaliana</em>. Indeed, the HSP90-RNAi set may more faithfully reveal effects of reduced HSP90, as these lines, unlike individual constitutive HSP90-TDNA lines, display neither compensation through upregulation of the inducible isoform nor mild induction of the heat-shock response. All the HSP90-reduced lines described in detail are in the Col-0 background; therefore, we do not here address the buffering of polymorphic genetic variation by HSP90 <a href="#pone.0000648-Queitsch1">[5]</a>. Analysis of other accessions may reveal that Col-0 is genetically predisposed to exhibit some of the revealed phenotypes upon reduction of HSP90 function.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Our comprehensive quantitative analysis of seedling phenotypes in genetically HSP90-reduced plants extended the previous qualitative study of seedlings with HSP90 partially inhibited by GDA <a href="#pone.0000648-Queitsch1">[5]</a>. In seedlings, both methods of HSP90 reduction resulted in similar morphological abnormalities, such as altered leaf shape, organ number, and pigment accumulation. As isogenic plant seedlings may display significant phenotypic variability even in highly controlled environments, we established a standardized set of common morphological abnormalities (<a href="#pone-0000648-g003">Fig. 3</a>) whose frequency was assessed across a population randomized by genotype by a scientist lacking knowledge of plant identity. Both sets of genetically reduced HSP90 lines showed a significant increase in morphological abnormalities relative to their respective controls (<a href="#pone-0000648-g004">Fig. 4</a>). </p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>These data are consistent with our previous findings that HSP90 may promote developmental robustness, such that a decrease in HSP90 function results in an increase in the frequency of stochastic developmental events, thereby amplifying phenotypic variation in an isogenic population. Developmental stability is often measured by the faithful development of repeated characters such as leaves. HSP90 reduction led to a significant increase in early leaf number defects, exacerbated by further challenge with higher temperature. Similarly, reduction of HSP90 function has also been linked to increased stochastic variation in highly canalized characters in <em>Drosophila melanogaster</em> <a href="#pone.0000648-Milton1">[9]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>Both sets of HSP90-reduced lines display significantly increased frequencies of abnormal shoot meristem development. These alterations display a range of temperature-dependent expressivity, from abnormally narrow leaves to complete absence of adult leaf development. Affected plants normally recover later in development; however, this initial defect commonly results in the maintenance of multiple primary shoot meristems in the adult plant.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>The timing of flowering is among the most important decisions in the plant lifecycle and requires the integration of multiple environmental signals through complex signaling pathways. HSP90-reduced lines consistently flowered later than controls, suggesting that the chaperone may play a role in either sensing or integrating environmental signals into the appropriate response.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>Curiously, one phenotype, necrotic spots on leaves, was more frequently observed in both sets of control lines. We speculate that, given HSP90's established role in the maintenance of R-proteins, these spots may correspond to localized cell death in response to inappropriate activation of biotroph resistance pathways. The reduced incidence of necrosis in HSP90-reduced plants is consistent with the destabilization and degradation of certain R-proteins in an HSP90-reduced background.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Such diminuation of SA-dependent defense responses suggests that plastic responses regulated in an antagonistic manner may be upregulated. Indeed, our whole-genome expression analysis revealed that genes affecting JA biosynthesis tended to be upregulated, indicating that HSP90-reduced plants may display increased resistance to insect herbivores. While basal resistance was not reproducibly affected in the HSP90-RNAi or HSP90-TDNA lines, the stronger <em>hsp90.2-3</em> mutant consistently showed greater resistance to the generalist herbivore <em>T. ni</em>. When resistance was induced by prior caterpillar herbivory, both the <em>hsp90.2-3</em> mutant as well as the tested HSP90.1-TDNA and HSP90.2-TDNA lines were more responsive to the mild induction treatment than the control. Therefore, the resistance of the <em>hsp90.2-3</em> mutant represents greater expressivity of the phenotype. Our data mirror the observed sensitivity of the <em>hsp90.2-3</em> mutant to microbial pathogens, consistent with the suggested cross-regulation between the pathogen defense and herbivore resistance pathways. Note that, as with basal herbivore resistance, basal defense against microbial pathogens was not affected in the HSP90.2 isoform mutant. Thus, HSP90 is key to maintaining the responsive plasticity of the defense system; reduction of HSP90 function predisposes the plant to herbivore defense but renders it sensitive to pathogen attack.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>Our results demonstrate the involvement of HSP90 in the plastic balance of defense responses. Our genome-wide expression analysis and previous studies reporting HSP90-dependent effects on other classical plastic responses, such as the response to light and gravitropism <a href="#pone.0000648-Queitsch1">[5]</a>, <a href="#pone.0000648-Cao1">[16]</a>, suggest that HSP90's effects on plant plasticity are generalizable to many different stimuli and that HSP90 plays a global role in the environmental responsiveness of plants. The plant system, in particular with the comprehensive set of HSP90-reduced lines presented here, provides a unique opportunity to further investigate the hypothesis of HSP90 as a central interface between organism, development, and environment.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Construction of HSP90-Reduced Lines</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>DNA constructs were created following the procedures in <a href="#pone.0000648-Chuang1">[34]</a>. Using the pCGN1547 vector backbone <a href="#pone.0000648-McBride1">[48]</a> modified with a cauliflower mosaic virus 35S promoter and a nopaline synthase terminator <a href="#pone.0000648-Krizek1">[49]</a>, fragments of HSP90 sequence were cloned in tandem antisense and sense orientation, thus forming a dsRNA hairpin upon transcription. Four different fragments of the coding sequences of HSP90 were used: bases 678-1154 of <em>HSP90.1</em> (At5g52640) (RNAi-A), bases 1572-2100 of <em>HSP90.1</em> (RNAi-B), bases 1551-1941 of <em>HSP90.2</em> (At5g56030) (RNAi-C), and bases 310-695 of <em>HSP90.3</em> (At5g56010) (RNAi-D). The coding sequence of mGFP4 <a href="#pone.0000648-Haseloff1">[50]</a> was inserted between the antisense and sense HSP90 sequences to improve construct stability in bacteria. Control constructs contained the GFP fragment without any HSP90 sequence. All constructs were verified by direct sequencing.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p><em>Agrobacterium tumefaciens</em> strain ASE was used to transform the constructs into <em>A. thaliana</em> (Col-0, laboratory stock from D. Preuss) plants by vacuum infiltration <a href="#pone.0000648-Bechtold1">[51]</a>. Transformed T<sub>1</sub> seedlings were selected on germination (GM) <a href="#pone.0000648-Queitsch1">[5]</a> medium containing kanamycin (50 µg/mL). Kanamycin resistance was assayed by root elongation after 14 days of growth. Resistant seedlings were transferred to soil and allowed to self-pollinate. At least 100 T<sub>2</sub> seeds per T<sub>1</sub> line were grown on kanamycin-containing medium to determine the resistance segregation ratio. Nine plants of each line exhibiting a segregation ratio consistent with an insertion at a single locus were transferred to soil and self-propagated. Resulting T<sub>3</sub> seeds were assayed for full kanamycin resistance; fully resistant lines were used for further experiments. For some experiments, lines were backcrossed twice to the wild-type Col-0 parent, self-propagated, and fully kanamycin-resistant lines were re-isolated. The Shahdara (CS6180) lines shown in <a href="#pone-0000648-g007">Fig. 7</a> were created in a similar manner.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3"></a><p>Insertion sites were localized by TAIL-PCR, using degenerate primers AD2 or AD3 and PCR conditions as described in <a href="#pone.0000648-Liu2">[35]</a>. Construct specific right and left border primers were from <a href="#pone.0000648-Campisi1">[52]</a>. The insert locations were: for RNAi-A2, ~700 bp 3′ of <em>At2g04870</em>; for RNAi-A3, ~100 bp 3′ of <em>At2g06200</em>; for RNAi-B1, ~500 bp 5′ of <em>At1g42980</em>; for control-1, in the 3′UTR of <em>At4g10790</em>; for control-2, ~100 bp 3′ of <em>At1g26330</em>; and for control-3, ~1200 bp 5′ of <em>At1g25570</em>. Exact locations could not be determined for RNAi-A1 or RNAi-C1.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4"></a><p><em>A. thaliana</em> (Col-0) containing a T-DNA insert in either <em>HSP90.1</em> (SALK_075596), <em>HSP90.2</em> (SALK_058553), or <em>HSP90.3</em> (SALK_011160) <a href="#pone.0000648-Alonso1">[32]</a> were obtained from the Arabidopsis Biological Resource Center. Lines homozygous for each insert were isolated by PCR and were verified to be true-breeding by the PCR analysis of several progeny.</p>


<h4>Assessment of HSP90-Reduction</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>For the analysis of HSP90 levels by quantitative Western blot, seeds were plated randomized with respect to genotype on GM medium. HSP90-RNAi and HSP90-TDNA lines were grown separately. The HSP90-RNAi seeds were T<sub>3</sub> generation without backcrossing. Plates were cold-treated at 4°C for four days to ensure equal germination and incubated at 22°C under 100 µmol m<sup>−2</sup> s<sup>−1</sup> cool-white light in a 16 hour light/8 hour dark (long-day) cycle. The aerial portions of individual plants were harvested after fourteen days of growth and were immediately frozen in liquid nitrogen. The plants were individually ground to a powder in microfuge tubes, on dry ice, using disposable plastic pestles. Lysis buffer (50 mM Tris-HCl, 100 mM NaCl, 2 mM EDTA, and 1% NP-40) was added to the powder on ice and mixed by pipetting. Five plants from each line were pooled for each data point, with two biological replicates for each line. Protein concentrations were measured using the Biorad DC protein assay and were equalized by addition of buffer. Equal amounts of each line were loaded onto an SDS Page gel. A standard curve of one of the control accessions, control-1, was run on each gel. Proteins were transferred from the gels to PVDF membranes and Western blotted for HSP90 using an <em>A. thaliana</em>-specific rabbit polyclonal antibody <a href="#pone.0000648-Krishna2">[36]</a>. Presence of the secondary was detected using ECL reagent and film. Bands on the film were quantitated using an Alpha Innotech Imager and its accompanying Chemi Imager Alpha Ease FC software, with experimental samples plotted on the standard curves generated by the control samples.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For the quantitation of HSP90 levels by RT-PCR, seedlings were grown on soil (Fafard #2), with 10 plants per 4” pot. The HSP90-RNAi seeds used were T<sub>3</sub> and T<sub>4</sub> generations without backcrossing. Aerial tissue from 14 day old plants which had been heat shocked at 37°C for two hours was harvested and flash frozen in liquid nitrogen. Plants (5−10) for individual genotypes were pooled and manually ground in liquid nitrogen. Total RNA was isolated using the SV Total RNA Isolation kit (Promega) protocol on the ground tissue, omitting the DNase treatment in the manufacturer's protocol. Ten µg of eluted total RNA was treated with DNA-free DNase Treatment (Ambion). One hundred µL of supernatant from this step was used in a second Turbo DNA-free DNase Treatment (Ambion). RNA was precipitated with NaOAC and isopropanol, washed with 75% ethanol, and resuspended in 20 µL water. For accurate quantitation, the RNA was assayed with the Quant-iT RiboGreen RNA Assay Kit from Molecular Probes (Invitrogen). After Ribogreen quantitation, 500 ng of RNA was used in a random primed first strand synthesis reaction: Five hundred ng RNA was mixed with 1 µL 250 ng/µL random primers (Invitrogen 48190-011), 1 µL dNTP's (Invitrogen 18427-013), and water for 10 µL total volume. The mixture was incubated at 65°C for five minutes, then 4°C for two minutes. Four µL first strand buffer (Invitrogen), 2 µL of 0.1M DTT, and 1 µL of water were then added to each reaction, and the mixture was incubated at 25°C for two minutes. One µL SuperScript II RNase H Reverse Transcriptase (Invitrogen 18064-014) was added, and the mixture was incubated at 25°C for ten minutes, followed by 50 minutes at 42°C, 15 minutes at 70°C, and cooled to 4°C. The resulting cDNA was labeled with the QuantiTect SYBR Green PCR Kit (Qiagen 204143) in a DNA Engine Opticon PCR cycler (MJ Research) following the kit protocol. The cDNA was amplified by primers <span class="gene">TGGTTCTGAAAACTTCTAATATGTCG</span> and <span class="gene">TGACACAAACCCAACCCTAGA</span>, which amplify a portion of the 3′ untranslated region of <em>HSP90.1</em>. The PCR cycle employed was 95°C for 15 minutes, followed by 40 cycles of 94°C for 15 seconds, 55°C for 30 seconds, and 72°C for 30 seconds, with product concentration measurement at the end of each cycle. The standard curves used in the SYBR Green RT-PCR assay were derived from genomic <em>A. thaliana</em> (Col-0) DNA, purified with DNeasy Plant Maxi Kit (Qiagen 68163). This DNA was sheared by passing it through a BD 18× 1B needle attached to a 1 mL BD slip-tip disposable tuberculin syringe. The DNA was then quantitated with a spectrophotometer, and the concentration diluted to 2.5 ng/µL. Four 10:1 serial dilutions were made from this stock. These five standards were used to generate the standard curve, with each dilution run in triplicate. All samples were also run in triplicate, along with template-only controls. Determination of cycle time values were done with the Opticon Monitor 2 analysis software.</p>


<h4>Assessment of Phenotypic Diversity in Seedlings</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Three hundred seventy-eight seeds from each of five HSP90-RNAi and three control lines, all backcrossed, were plated on gridded square petri plates containing GM media at an equally-spaced density of 36 seeds per plate. The experiment was randomized such that nine seeds of each line were placed randomly on each block of two plates. A second experiment was performed with the three T-DNA insertion lines, along with a Col-0 control (CS60000), with 504 seeds per mutant line and randomization within each plate. All seeds used in each experiment resulted from parental plants which had been propagated in the same flat and harvested at the same time.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>Plates were stratified and plants grown as for the assessment of HSP90-reduction above. Morphologically deviant phenotypes were analyzed visually after ten days of growth. At this time, most plants displayed fully developed first true leaves and had initiated spirally phyllotactic leaves. Scoring was performed according to a standard rubric developed during pilot experiments. Each line had been included in at least one pilot experiment; the data presented here are from single experiments for the HSP90-RNAi and HSP90-TDNA lines. Data from pilot experiments were consistent with those presented here. The following traits were scored (<a href="#pone-0000648-g002">Fig. 2</a>): missing/extra organs; aberrant cotyledon shape, color, or expansion; aberrant leaf shape or color; failure to lift apical organs off the growth medium; delayed development such that spirally phyllotactic leaves were not visible; curled hypocotyl; and failure to germinate. Plants classified as “shoot meristemless” displayed no growth of apical-meristem derived organs when viewed under a dissecting microscope at 20× magnification. Plants without any standardized phenotype were scored as “no scored phenotype”.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3"></a><p>A similar design was employed to assess the temperature dependence of observed morphological phenotypes. One control line and two HSP90-RNAi lines (RNAi-A2 and RNAi-C1) were used, with 360 seeds per temperature per line and randomization within each plate. Plant growth conditions were as above except that temperatures of 17°C, 22°C, and 27°C were used. To account for more rapid development at increased temperature, seedlings were scored at eight days after germination (27°C), ten days (22°C) and fourteen days (17°C), when the developmental stages were equivalent. A restricted set of phenotypes was scored.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4"></a><p>The identities of all lines were arbitrarily coded before being given to the experimenter. Therefore, all plating and scoring was performed without knowledge of the genotype of each plant.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5"></a><p>As no phenotypes were statistically different between the three control lines for the HSP90-RNAi lines, the controls were combined for statistical analysis. To compare the frequencies of extremely rare phenotypes (less than five occurrences in either group under comparison), we used Fisher's Exact Test (2-tailed). For more common phenotypes, a 2-tailed Yates-corrected χ<sup>2</sup> test was employed.</p>


<h4>Assessment of Phenotypic Diversity in Adult Plants</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>We utilized the same seed batch of the 12 lines discussed in the assessment of phenotypic diversity in seedlings. The experiment was randomized such that each line occupied eight pots in a block of three flats with 32 pots per flat; five blocks were employed for a total of forty pots per line. Four to five seeds were planted in each pot in thoroughly wetted 50% Metro-Mix 200:50% Pro-Mix BX soil. The seeds were stratified on soil for four days at 4°C, followed by growth in a greenhouse environment (~22°C with 100 µmol m<sup>−2</sup> s<sup>−1</sup> cool-white light in a long-day light cycle). Flat position was rotated daily to control for microenvironments within the greenhouse. Humidity domes were removed after ten days, and all seedlings were eliminated from each pot except for the one closest to the center. No germination defects were observed. Plants were watered as necessary until one week after the cessation of flowering and were then allowed to dry for at least a month before harvest.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2"></a><p>Flowering was defined as the date the primary bolt reached 2 cm in length. Longest leaf measurements were performed one week after flowering. Plant height and the number of siliques on the primary stem were measured at the cessation of flowering; the longest stem was used for plants with multiple primary stems. Seeds were harvested before the dry mass of above-ground tissues was measured. The mass of a counted number of seeds (~200 per line) was measured to determine average seed mass.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3"></a><p>No statistical differences were observed between the control lines for the HSP90-RNAi plants; therefore, these lines were combined for analysis. Multiple linear regression was used to estimate least-squares means of each trait for each line. Genotype, flat, and position within the flat (corner, edge, or interior) were treated as fixed variables. A second model adding the presence of multiple meristems as a fixed variable was created for each trait to test if observed genotypic differences could be explained by this morphological trait. Tukey's HSD post-hoc test was used to examine whether differences between lines were significant. The significance of primary shoot number distributions was tested with Wilcoxon tests.</p>


<h4>Genome-wide Expression Analysis</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>The same seed batches were used as for the quantitative Western blots, and seedlings were grown under the same conditions. Aerial tissue was harvested from 14 day old seedlings and flash frozen in liquid nitrogen. Ten plants were pooled for each genotype and manually ground in liquid nitrogen. Total RNA was isolated using the SV Total RNA Isolation kit (Promega) protocol on the ground tissue, omitting the DNase treatment in the manufacturer's protocol. Eluted total RNA was treated with RQ1 Rnase-free DNase (Promega). A 65°C, 15 min incubation step inactivated the enzyme, instead of the kit's Stop Solution. Multiple identical RNA isolations were pooled and concentrated with the RNeasy (Qiagen) cleanup protocol. 50–60 µL of solution was eluted, with 8 µg of the purified RNA was used in the standard Affymetrix cRNA labeling and hybridization protocols (<a href="http://www.affymetrix.com/support/technical/manual/expression_manual.affx">http://www.affymetrix.com/support/techni​cal/manual/expression_manual.affx</a>) on Affymetrix ATH-1 microarrays. Three differerent experiments were performed. One experiment included lines RNAi-A1, RNAi-A3, and Control-2, with two biological replicates and one technical replicate per line. The second included lines RNAi-A2, RNAi-B1, RNAi-C1, Control-1, and Control-3, with three biological replicates per line. The third included the three single-isoform T-DNA insertion lines, along with the Col-0 control, with three biological replicates per line. All array data have been submitted to the Gene Expression Omnibus with accession number GSE7796.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>All data from different experiments were pooled for preprocessing and gene expression index calculation using the Probe Logarithmic Intensity Error (PLIER) algorithm (<a href="http://www.affymetrix.com/support/technical/technotes/plier_technote.pdf">http://www.affymetrix.com/support/techni​cal/technotes/plier_technote.pdf</a>). Probe sets that were below detection range across all the samples were excluded before statistical comparison. Average expression for each line was calculated by averaging across technical replicates, then averaging across biological replicates. Statistical significance was calculated by a two-way ANOVA or by Welch's t-test, depending on whether the experiment contained technical replicates. Fold changes were calculated from the average expression for each line divided by the average expression in all controls used in the same experiment. Fold changes for combined HSP90-RNAi, HSP90-TDNA, and all HSP90-reduced lines were calculated by averaging single-line fold changes converted to a linear scale.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>For the analysis with GeneMerge<a href="#pone.0000648-CastilloDavis1">[43]</a>, all genes with significance less than α = 0.05 were discarded. Genes were then sorted by average fold change. Lists of genes significantly upregulated by 1.5 fold or 2 fold or downregulated by 1.5 fold or 2 fold for each HSP90-reduced line were submitted to GeneMerge to determine enrichment in biological process, cellular component, or molecular function GO annotations.</p>


<h4>Response to Herbivory</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>Seeds were planted in 36-pot flats randomized with respect to genotype in thoroughly wetted Jiffy peat pellets. Three seeds were planted per pellet and were stratified for four days at 4°C. Flats were sealed with domes and parafilm to maintain high humidity during germination. Plants were grown in 12 hour light/12 hour dark cycles at ~22°C under cool white light. Domes were removed after six days, weeded to a single plant per pot after eleven days, and fertilized with one-quarter strength Hoagland's solution <a href="#pone.0000648-Hoagland1">[53]</a> after fourteen days. The order of the flats was rotated daily to minimize the effects of local environmental variance.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p><em>Trichoplusia ni</em> eggs were obtained from a highly inbred population (Entopath, Easton, PA) and were incubated at 28°C for two days to hatch. For the <em>T. ni</em> bioassays, the neonate larvae were caged to feed freely on single whole four-week old non-flowering plants for seven days according to <a href="#pone.0000648-Cui2">[46]</a>. Subsequently, caterpillars were removed, lyophilized, dried and weighed. Initial larval dry weight (~10 µg) is insignificant relative to weight gain and was disregarded. For the induction experiments, at least 90 plants were used per genotype and treatment. For the induction treatment, one three-day-old <em>T. ni</em> larva was caged on the seventh leaf to feed for 24 hours. The leaf was subsequently removed and a bioassay caterpillar added as above. The same leaf was caged and removed on control plants which had no induction caterpillar.</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3"></a><p>Methyl-jasmonic acid (MeJA) was dissolved in water at three different concentrations (0.04mM, 0.20mM, 1mM) and applied to plants by aerosol spray. The plants were allowed to induce responses for two days, followed by addition of a bioassay caterpillar as above. Control plants were sprayed with water alone and were physically separated from MeJA treated plants only during spraying and induction to prevent cross-contamination. Thirty-six plants per genotype per treatment were used.</p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4"></a><p>Caterpillar weight was approximately normally distributed. Least-squares means for caterpillar weight on each genotype were estimated with multiple-linear regression model including flat, treatment, genotype, and genotype×treatment interaction as fixed effects. Student's t-tests were employed to test for significant differences between genotypes.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000648"></div><a name="pone.0000648.s001" id="pone.0000648.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000648.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Morphological phenotype frequencies and associated p-values for seedling phenotypes in HSP90-RNAi lines. Top: P-values. Fisher's exact tests used for small sample sizes, otherwise chi-square tests; all tests are 2-tailed. * denotes comparisons which are not meaningful due to extremely rare observance of the phenotype (less than 0.8 percent in both lines under comparison). Bottom: Fraction of plants observed with each defined phenotype.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000648.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000648.s002" id="pone.0000648.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000648.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Morphological phenotype frequencies and associated p-values for seedling phenotypes in HSP90-Salk lines. Top: P-values. Fisher's exact tests used for small sample sizes, otherwise chi-square tests; all tests are 2-tailed. * denotes comparisons which are not meaningful due to extremely rare observance of the phenotype (less than 0.8 percent in both lines under comparison). Bottom: Fraction of plants observed with each defined phenotype.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000648.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000648.s003" id="pone.0000648.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000648.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Morphological phenotype frequencies and associated p-values for seedling phenotypes in HSP90-RNAi lines at different growth temperatures. Top: P-values. Fisher's exact tests used for small sample sizes, otherwise chi-square tests; all tests are 2-tailed. * denotes comparisons which are not meaningful due to extremely rare observance of the phenotype (less than 0.8 percent in both lines under comparison). Temperature effect p-values by logistic regression of temperature as a continuous variable versus phenotype presence as a nominal variable. ** denotes comparisons for which the logistic regression was not valid for the full temperature range. Bottom: Fraction of plants observed with each defined phenotype.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000648.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.00 MB TXT)</p>
<a name="pone.0000648.s004" id="pone.0000648.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000648.s004">Dataset S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Gene-Ontology Analysis. Significance was determined for all genes on the ATH-1 array by ANOVA comparison of normalized data from all control arrays versus all arrays from one HSP90-reduced line. All genes with p-value less significant that 0.05 (uncorrected for multiple comparisons) were discarded. Genes were then sorted by average fold change. Genes upregulated by 1.5 fold/2 fold or downregulated by 1.5 fold/2 fold were submitted to GeneMerge (<a href="http://www.oeb.harvard.edu/hartl/lab/publications/GeneMerge/GeneMerge.html">http://www.oeb.harvard.edu/hartl/lab/pub​lications/GeneMerge/GeneMerge.html</a>) to determine enrichment in biological process, cellular component, or molecular function GO annotations. Fold changes for combined HSP90-RNAi, HSP90-Salk, and all HSP90-reduced sets were calculated by averaging fold changes converted to a linear scale. The following tables are in the format: Line name Fold Change GO Dataset GeneMerge header GO descriptor Fraction of genes with GO descriptor on ATH1 array Fraction of genes with GO descriptor significantly changed by fold change Raw e-score E-score corrected for multiple comparisons GO annotation Genes significantly changed with GO annotation</p>
<p class="siDoi">doi:10.1371/journal.pone.0000648.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.80 MB TXT)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We gratefully thank H. Wang, S. Chandra, N. Salathia, J. Russell, and K. Johnsson for help with experiments, D. Hubert and J. Dangl for the <em>hsp90.2-3</em> mutant line, P. Krishna for the HSP90 antibody, and the Arabidopsis Biological Resource Center for seeds.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CQ AB TS AW EW. Performed the experiments: CQ AB TS AW EW KS CM AK. Analyzed the data: CQ SK AB TS AW EW CM. Wrote the paper: CQ SL TS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000648-Young1" id="pone.0000648-Young1"></a>Young JC, Moarefi I, Hartl FU (2001) Hsp90: a specialized but essential protein-folding tool. J Cell Biol  154: 267–273.  <ul class="find" data-citedArticleID="969170" data-doi="10.1083/jcb.200104079"><li><a href="http://dx.doi.org/10.1083/jcb.200104079" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp90%3A+a+specialized+but+essential+protein-folding+tool." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp90%3A+a+specialized+but+essential+protein-folding+tool.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000648-Picard1" id="pone.0000648-Picard1"></a>Picard D (2002) Heat-shock protein 90, a chaperone for folding and regulation. Cell Mol Life Sci  59: 1640–1648.  <ul class="find" data-citedArticleID="969136" data-doi="10.1007/pl00012491"><li><a href="http://dx.doi.org/10.1007/pl00012491" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heat-shock+protein+90%2C+a+chaperone+for+folding+and+regulation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heat-shock+protein+90%2C+a+chaperone+for+folding+and+regulation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000648-Wegele1" id="pone.0000648-Wegele1"></a>Wegele H, Muller L, Buchner J (2004) Hsp70 and Hsp90–a relay team for protein folding. Rev Physiol Biochem Pharmacol  151: 1–44.  <ul class="find" data-citedArticleID="969164" data-doi="10.1007/s10254-003-0021-1"><li><a href="http://dx.doi.org/10.1007/s10254-003-0021-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp70+and+Hsp90%26%23x2013%3Ba+relay+team+for+protein+folding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp70+and+Hsp90%26%23x2013%3Ba+relay+team+for+protein+folding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000648-Whitesell1" id="pone.0000648-Whitesell1"></a>Whitesell L, Lindquist SL (2005) HSP90 and the chaperoning of cancer. Nat Rev Cancer  5: 761–772.  <ul class="find" data-citedArticleID="969166" data-doi="10.1038/nrc1716"><li><a href="http://dx.doi.org/10.1038/nrc1716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HSP90+and+the+chaperoning+of+cancer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HSP90+and+the+chaperoning+of+cancer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000648-Queitsch1" id="pone.0000648-Queitsch1"></a>Queitsch C, Sangster TA, Lindquist S (2002) Hsp90 as a capacitor of phenotypic variation. Nature  417: 618–624.  <ul class="find" data-citedArticleID="969144" data-doi="10.1038/nature749"><li><a href="http://dx.doi.org/10.1038/nature749" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp90+as+a+capacitor+of+phenotypic+variation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp90+as+a+capacitor+of+phenotypic+variation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000648-Rutherford1" id="pone.0000648-Rutherford1"></a>Rutherford SL, Lindquist S (1998) Hsp90 as a capacitor for morphological evolution. Nature  396: 336–342.  <ul class="find" data-citedArticleID="969146" data-doi="10.1038/24550"><li><a href="http://dx.doi.org/10.1038/24550" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp90+as+a+capacitor+for+morphological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp90+as+a+capacitor+for+morphological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000648-Sollars1" id="pone.0000648-Sollars1"></a>Sollars V, Lu X, Xiao L, Wang X, Garfinkel MD, et al.  (2003) Evidence for an epigenetic mechanism by which Hsp90 acts as a capacitor for morphological evolution. Nat Genet  33: 70–74.  <ul class="find" data-citedArticleID="969154" data-doi="10.1038/ng1067"><li><a href="http://dx.doi.org/10.1038/ng1067" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evidence+for+an+epigenetic+mechanism+by+which+Hsp90+acts+as+a+capacitor+for+morphological+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evidence+for+an+epigenetic+mechanism+by+which+Hsp90+acts+as+a+capacitor+for+morphological+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000648-Cowen1" id="pone.0000648-Cowen1"></a>Cowen LE, Lindquist S (2005) Hsp90 potentiates the rapid evolution of new traits: drug resistance in diverse fungi. Science  309: 2185–2189.  <ul class="find" data-citedArticleID="969090" data-doi="10.1126/science.1118370"><li><a href="http://dx.doi.org/10.1126/science.1118370" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hsp90+potentiates+the+rapid+evolution+of+new+traits%3A+drug+resistance+in+diverse+fungi." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hsp90+potentiates+the+rapid+evolution+of+new+traits%3A+drug+resistance+in+diverse+fungi.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000648-Milton1" id="pone.0000648-Milton1"></a>Milton CC, Ulane CM, Rutherford S (2006) Control of canalization and evolvability by hsp90. PLoS ONE  1: e75.  <ul class="find" data-citedArticleID="969132" data-doi="10.1371/journal.pone.0000075"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000075" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+canalization+and+evolvability+by+hsp90." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+canalization+and+evolvability+by+hsp90.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000648-Pratt1" id="pone.0000648-Pratt1"></a>Pratt WB, Morishima Y, Murphy M, Harrell M (2006) Chaperoning of glucocorticoid receptors. Handb Exp Pharmacol 111–138.  <ul class="find" data-citedArticleID="969140" data-doi="10.1007/3-540-29717-0_5"><li><a href="http://dx.doi.org/10.1007/3-540-29717-0_5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chaperoning+of+glucocorticoid+receptors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chaperoning+of+glucocorticoid+receptors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000648-Picard2" id="pone.0000648-Picard2"></a>Picard D, Khursheed B, Garabedian MJ, Fortin MG, Lindquist S, et al.  (1990) Reduced levels of hsp90 compromise steroid receptor action in vivo. Nature  348: 166–168.  <ul class="find" data-citedArticleID="969138" data-doi="10.1038/348166a0"><li><a href="http://dx.doi.org/10.1038/348166a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reduced+levels+of+hsp90+compromise+steroid+receptor+action+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Reduced+levels+of+hsp90+compromise+steroid+receptor+action+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000648-Zhao1" id="pone.0000648-Zhao1"></a>Zhao R, Houry WA (2007) Molecular interaction network of the Hsp90 chaperone system. Adv Exp Med Biol  594: 27–36.  <ul class="find" data-citedArticleID="969174" data-doi="10.1007/978-0-387-39975-1_3"><li><a href="http://dx.doi.org/10.1007/978-0-387-39975-1_3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+interaction+network+of+the+Hsp90+chaperone+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+interaction+network+of+the+Hsp90+chaperone+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000648-Stancato1" id="pone.0000648-Stancato1"></a>Stancato LF, Hutchison KA, Krishna P, Pratt WB (1996) Animal and plant cell lysates share a conserved chaperone system that assembles the glucocorticoid receptor into a functional heterocomplex with hsp90. Biochemistry  35: 554–561.  <ul class="find" data-citedArticleID="969156" data-doi="10.1021/bi9511649"><li><a href="http://dx.doi.org/10.1021/bi9511649" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Animal+and+plant+cell+lysates+share+a+conserved+chaperone+system+that+assembles+the+glucocorticoid+receptor+into+a+functional+heterocomplex+with+hsp90." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Animal+and+plant+cell+lysates+share+a+conserved+chaperone+system+that+assembles+the+glucocorticoid+receptor+into+a+functional+heterocomplex+with+hsp90.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000648-Yabe1" id="pone.0000648-Yabe1"></a>Yabe N, Takahashi T, Komeda Y (1994) Analysis of tissue-specific expression of Arabidopsis thaliana HSP90-family gene HSP81. Plant Cell Physiol  35: 1207–1219.  <ul class="find" data-citedArticleID="969168"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+tissue-specific+expression+of+Arabidopsis+thaliana+HSP90-family+gene+HSP81.&amp;auth=&amp;atitle=Analysis+of+tissue-specific+expression+of+Arabidopsis+thaliana+HSP90-family+gene+HSP81." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+tissue-specific+expression+of+Arabidopsis+thaliana+HSP90-family+gene+HSP81." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+tissue-specific+expression+of+Arabidopsis+thaliana+HSP90-family+gene+HSP81.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000648-Krishna1" id="pone.0000648-Krishna1"></a>Krishna P, Gloor G (2001) The Hsp90 family of proteins in Arabidopsis thaliana. Cell Stress Chaperones  6: 238–246.  <ul class="find" data-citedArticleID="969118" data-doi="10.1379/1466-1268(2001)006<0238:thfopi&gt;2.0.co;2"><li><a href="http://dx.doi.org/10.1379/1466-1268(2001)006<0238:thfopi&gt;2.0.co;2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Hsp90+family+of+proteins+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Hsp90+family+of+proteins+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000648-Cao1" id="pone.0000648-Cao1"></a>Cao D, Froehlich JE, Zhang H, Cheng CL (2003) The chlorate-resistant and photomorphogenesis-defective mutant cr88 encodes a chloroplast-targeted HSP90. Plant J  33: 107–118.  <ul class="find" data-citedArticleID="969084" data-doi="10.1046/j.1365-313x.2003.016011.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.2003.016011.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+chlorate-resistant+and+photomorphogenesis-defective+mutant+cr88+encodes+a+chloroplast-targeted+HSP90." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+chlorate-resistant+and+photomorphogenesis-defective+mutant+cr88+encodes+a+chloroplast-targeted+HSP90.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000648-Ishiguro1" id="pone.0000648-Ishiguro1"></a>Ishiguro S, Watanabe Y, Ito N, Nonaka H, Takeda N, et al.  (2002) SHEPHERD is the Arabidopsis GRP94 responsible for the formation of functional CLAVATA proteins. Embo J  21: 898–908.  <ul class="find" data-citedArticleID="969112" data-doi="10.1093/emboj/21.5.898"><li><a href="http://dx.doi.org/10.1093/emboj/21.5.898" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SHEPHERD+is+the+Arabidopsis+GRP94+responsible+for+the+formation+of+functional+CLAVATA+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SHEPHERD+is+the+Arabidopsis+GRP94+responsible+for+the+formation+of+functional+CLAVATA+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000648-Takahashi1" id="pone.0000648-Takahashi1"></a>Takahashi A, Casais C, Ichimura K, Shirasu K (2003) HSP90 interacts with RAR1 and SGT1 and is essential for RPS2-mediated disease resistance in Arabidopsis. Proc Natl Acad Sci U S A  100: 11777–11782.  <ul class="find" data-citedArticleID="969158" data-doi="10.1073/pnas.2033934100"><li><a href="http://dx.doi.org/10.1073/pnas.2033934100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=HSP90+interacts+with+RAR1+and+SGT1+and+is+essential+for+RPS2-mediated+disease+resistance+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22HSP90+interacts+with+RAR1+and+SGT1+and+is+essential+for+RPS2-mediated+disease+resistance+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000648-Hubert1" id="pone.0000648-Hubert1"></a>Hubert DA, Tornero P, Belkhadir Y, Krishna P, Takahashi A, et al.  (2003) Cytosolic HSP90 associates with and modulates the Arabidopsis RPM1 disease resistance protein. Embo J  22: 5679–5689.  <ul class="find" data-citedArticleID="969110" data-doi="10.1093/emboj/cdg547"><li><a href="http://dx.doi.org/10.1093/emboj/cdg547" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytosolic+HSP90+associates+with+and+modulates+the+Arabidopsis+RPM1+disease+resistance+protein." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytosolic+HSP90+associates+with+and+modulates+the+Arabidopsis+RPM1+disease+resistance+protein.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000648-Liu1" id="pone.0000648-Liu1"></a>Liu Y, Burch-Smith T, Schiff M, Feng S, Dinesh-Kumar SP (2004) Molecular chaperone Hsp90 associates with resistance protein N and its signaling proteins SGT1 and Rar1 to modulate an innate immune response in plants. J Biol Chem  279: 2101–2108.  <ul class="find" data-citedArticleID="969124" data-doi="10.1074/jbc.m310029200"><li><a href="http://dx.doi.org/10.1074/jbc.m310029200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+chaperone+Hsp90+associates+with+resistance+protein+N+and+its+signaling+proteins+SGT1+and+Rar1+to+modulate+an+innate+immune+response+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+chaperone+Hsp90+associates+with+resistance+protein+N+and+its+signaling+proteins+SGT1+and+Rar1+to+modulate+an+innate+immune+response+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000648-Lu1" id="pone.0000648-Lu1"></a>Lu R, Malcuit I, Moffett P, Ruiz MT, Peart J, et al.  (2003) High throughput virus-induced gene silencing implicates heat shock protein 90 in plant disease resistance. Embo J  22: 5690–5699.  <ul class="find" data-citedArticleID="969128" data-doi="10.1093/emboj/cdg546"><li><a href="http://dx.doi.org/10.1093/emboj/cdg546" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+throughput+virus-induced+gene+silencing+implicates+heat+shock+protein+90+in+plant+disease+resistance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+throughput+virus-induced+gene+silencing+implicates+heat+shock+protein+90+in+plant+disease+resistance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000648-Bos1" id="pone.0000648-Bos1"></a>Bos JI, Kanneganti TD, Young C, Cakir C, Huitema E, et al.  (2006) The C-terminal half of Phytophthora infestans RXLR effector AVR3a is sufficient to trigger R3a-mediated hypersensitivity and suppress INF1-induced cell death in Nicotiana benthamiana. Plant J  48: 165–176.  <ul class="find" data-citedArticleID="969080" data-doi="10.1111/j.1365-313x.2006.02866.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02866.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+C-terminal+half+of+Phytophthora+infestans+RXLR+effector+AVR3a+is+sufficient+to+trigger+R3a-mediated+hypersensitivity+and+suppress+INF1-induced+cell+death+in+Nicotiana+benthamiana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+C-terminal+half+of+Phytophthora+infestans+RXLR+effector+AVR3a+is+sufficient+to+trigger+R3a-mediated+hypersensitivity+and+suppress+INF1-induced+cell+death+in+Nicotiana+benthamiana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000648-Bhattarai1" id="pone.0000648-Bhattarai1"></a>Bhattarai KK, Li Q, Liu Y, Dinesh-Kumar SP, Kaloshian I (2007) The Mi-1-Mediated Pest Resistance Requires Hsp90 and Sgt1. Plant Physiol..   <ul class="find" data-citedArticleID="969078" data-doi="10.1104/pp.107.097246"><li><a href="http://dx.doi.org/10.1104/pp.107.097246" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Mi-1-Mediated+Pest+Resistance+Requires+Hsp90+and+Sgt1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Mi-1-Mediated+Pest+Resistance+Requires+Hsp90+and+Sgt1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000648-Kanzaki1" id="pone.0000648-Kanzaki1"></a>Kanzaki H, Saitoh H, Ito A, Fujisawa S, Kamoun S, et al.  (2003) Cytosolic HSP90 and HSP70 are essential components of INF1-mediated hypersensitive response and non-host resistance to Pseudomonas cichorii in Nicotiana benthamiana. Molecular Plant Pathology  4: 383–391.  <ul class="find" data-citedArticleID="969114" data-doi="10.1046/j.1364-3703.2003.00186.x"><li><a href="http://dx.doi.org/10.1046/j.1364-3703.2003.00186.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytosolic+HSP90+and+HSP70+are+essential+components+of+INF1-mediated+hypersensitive+response+and+non-host+resistance+to+Pseudomonas+cichorii+in+Nicotiana+benthamiana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytosolic+HSP90+and+HSP70+are+essential+components+of+INF1-mediated+hypersensitive+response+and+non-host+resistance+to+Pseudomonas+cichorii+in+Nicotiana+benthamiana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000648-Glazebrook1" id="pone.0000648-Glazebrook1"></a>Glazebrook J (2005) Contrasting mechanisms of defense against biotrophic and necrotrophic pathogens. Annu Rev Phytopathol  43: 205–227.  <ul class="find" data-citedArticleID="969098" data-doi="10.1146/annurev.phyto.43.040204.135923"><li><a href="http://dx.doi.org/10.1146/annurev.phyto.43.040204.135923" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Contrasting+mechanisms+of+defense+against+biotrophic+and+necrotrophic+pathogens." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Contrasting+mechanisms+of+defense+against+biotrophic+and+necrotrophic+pathogens.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000648-Azevedo1" id="pone.0000648-Azevedo1"></a>Azevedo C, Betsuyaku S, Peart J, Takahashi A, Noel L, et al.  (2006) Role of SGT1 in resistance protein accumulation in plant immunity. Embo J  25: 2007–2016.  <ul class="find" data-citedArticleID="969074" data-doi="10.1038/sj.emboj.7601084"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601084" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Role+of+SGT1+in+resistance+protein+accumulation+in+plant+immunity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Role+of+SGT1+in+resistance+protein+accumulation+in+plant+immunity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000648-Holt1" id="pone.0000648-Holt1"></a>Holt BF 3rd, Belkhadir Y, Dangl JL (2005) Antagonistic control of disease resistance protein stability in the plant immune system. Science  309: 929–932.  <ul class="find" data-citedArticleID="969108" data-doi="10.1126/science.1109977"><li><a href="http://dx.doi.org/10.1126/science.1109977" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antagonistic+control+of+disease+resistance+protein+stability+in+the+plant+immune+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antagonistic+control+of+disease+resistance+protein+stability+in+the+plant+immune+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000648-Schneider1" id="pone.0000648-Schneider1"></a>Schneider C, Sepp-Lorenzino L, Nimmesgern E, Ouerfelli O, Danishefsky S, et al.  (1996) Pharmacologic shifting of a balance between protein refolding and degradation mediated by Hsp90. Proc Natl Acad Sci U S A  93: 14536–14541.  <ul class="find" data-citedArticleID="969152" data-doi="10.1073/pnas.93.25.14536"><li><a href="http://dx.doi.org/10.1073/pnas.93.25.14536" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pharmacologic+shifting+of+a+balance+between+protein+refolding+and+degradation+mediated+by+Hsp90." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pharmacologic+shifting+of+a+balance+between+protein+refolding+and+degradation+mediated+by+Hsp90.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000648-Prodromou1" id="pone.0000648-Prodromou1"></a>Prodromou C, Roe SM, O'Brien R, Ladbury JE, Piper PW, et al.  (1997) Identification and structural characterization of the ATP/ADP-binding site in the Hsp90 molecular chaperone. Cell  90: 65–75.  <ul class="find" data-citedArticleID="969142" data-doi="10.1016/s0092-8674(00)80314-1"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80314-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+and+structural+characterization+of+the+ATP%2FADP-binding+site+in+the+Hsp90+molecular+chaperone." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+and+structural+characterization+of+the+ATP%2FADP-binding+site+in+the+Hsp90+molecular+chaperone.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000648-Grenert1" id="pone.0000648-Grenert1"></a>Grenert JP, Johnson BD, Toft DO (1999) The importance of ATP binding and hydrolysis by hsp90 in formation and function of protein heterocomplexes. J Biol Chem  274: 17525–17533.  <ul class="find" data-citedArticleID="969100" data-doi="10.1074/jbc.274.25.17525"><li><a href="http://dx.doi.org/10.1074/jbc.274.25.17525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+importance+of+ATP+binding+and+hydrolysis+by+hsp90+in+formation+and+function+of+protein+heterocomplexes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+importance+of+ATP+binding+and+hydrolysis+by+hsp90+in+formation+and+function+of+protein+heterocomplexes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000648-Young2" id="pone.0000648-Young2"></a>Young JC, Hartl FU (2000) Polypeptide release by Hsp90 involves ATP hydrolysis and is enhanced by the co-chaperone p23. Embo J  19: 5930–5940.  <ul class="find" data-citedArticleID="969172" data-doi="10.1093/emboj/19.21.5930"><li><a href="http://dx.doi.org/10.1093/emboj/19.21.5930" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polypeptide+release+by+Hsp90+involves+ATP+hydrolysis+and+is+enhanced+by+the+co-chaperone+p23." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polypeptide+release+by+Hsp90+involves+ATP+hydrolysis+and+is+enhanced+by+the+co-chaperone+p23.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000648-Alonso1" id="pone.0000648-Alonso1"></a>Alonso JM, Stepanova AN, Leisse TJ, Kim CJ, Chen H, et al.  (2003) Genome-wide insertional mutagenesis of Arabidopsis thaliana. Science  301: 653–657.  <ul class="find" data-citedArticleID="969072" data-doi="10.1126/science.1086391"><li><a href="http://dx.doi.org/10.1126/science.1086391" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+insertional+mutagenesis+of+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+insertional+mutagenesis+of+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000648-The1" id="pone.0000648-The1"></a>The Arabidopsis Information Resource (2007) TAIR7 Genome Release.   Available: <a href="http://www.arabidopsis.org">www.arabidopsis.org</a>. Accessed 2007 May 1.  <ul class="find-nolinks"></ul></li><li><span class="label">34.
              </span><a name="pone.0000648-Chuang1" id="pone.0000648-Chuang1"></a>Chuang CF, Meyerowitz EM (2000) Specific and heritable genetic interference by double-stranded RNA in Arabidopsis thaliana. Proc Natl Acad Sci U S A  97: 4985–4990.  <ul class="find" data-citedArticleID="969088" data-doi="10.1073/pnas.060034297"><li><a href="http://dx.doi.org/10.1073/pnas.060034297" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Specific+and+heritable+genetic+interference+by+double-stranded+RNA+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Specific+and+heritable+genetic+interference+by+double-stranded+RNA+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000648-Liu2" id="pone.0000648-Liu2"></a>Liu YG, Mitsukawa N, Oosumi T, Whittier RF (1995) Efficient isolation and mapping of Arabidopsis thaliana T-DNA insert junctions by thermal asymmetric interlaced PCR. Plant J  8: 457–463.  <ul class="find" data-citedArticleID="969126" data-doi="10.1046/j.1365-313x.1995.08030457.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1995.08030457.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Efficient+isolation+and+mapping+of+Arabidopsis+thaliana+T-DNA+insert+junctions+by+thermal+asymmetric+interlaced+PCR." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Efficient+isolation+and+mapping+of+Arabidopsis+thaliana+T-DNA+insert+junctions+by+thermal+asymmetric+interlaced+PCR.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000648-Krishna2" id="pone.0000648-Krishna2"></a>Krishna P, Reddy RK, Sacco M, Frappier JR, Felsheim RF (1997) Analysis of the native forms of the 90 kDa heat shock protein (hsp90) in plant cytosolic extracts. Plant Mol Biol  33: 457–466.  <ul class="find" data-citedArticleID="969120"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Analysis+of+the+native+forms+of+the+90+kDa+heat+shock+protein+%28hsp90%29+in+plant+cytosolic+extracts.&amp;auth=&amp;atitle=Analysis+of+the+native+forms+of+the+90+kDa+heat+shock+protein+%28hsp90%29+in+plant+cytosolic+extracts." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+native+forms+of+the+90+kDa+heat+shock+protein+%28hsp90%29+in+plant+cytosolic+extracts." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+native+forms+of+the+90+kDa+heat+shock+protein+%28hsp90%29+in+plant+cytosolic+extracts.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000648-Duina1" id="pone.0000648-Duina1"></a>Duina AA, Kalton HM, Gaber RF (1998) Requirement for Hsp90 and a CyP-40-type cyclophilin in negative regulation of the heat shock response. J Biol Chem  273: 18974–18978.  <ul class="find" data-citedArticleID="969096" data-doi="10.1074/jbc.273.30.18974"><li><a href="http://dx.doi.org/10.1074/jbc.273.30.18974" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+for+Hsp90+and+a+CyP-40-type+cyclophilin+in+negative+regulation+of+the+heat+shock+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+for+Hsp90+and+a+CyP-40-type+cyclophilin+in+negative+regulation+of+the+heat+shock+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000648-Murakami1" id="pone.0000648-Murakami1"></a>Murakami Y, Uehara Y, Yamamoto C, Fukazawa H, Mizuno S (1991) Induction of hsp 72/73 by herbimycin A, an inhibitor of transformation by tyrosine kinase oncogenes. Exp Cell Res  195: 338–344.  <ul class="find" data-citedArticleID="969134" data-doi="10.1016/0014-4827(91)90382-5"><li><a href="http://dx.doi.org/10.1016/0014-4827(91)90382-5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induction+of+hsp+72%2F73+by+herbimycin+A%2C+an+inhibitor+of+transformation+by+tyrosine+kinase+oncogenes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induction+of+hsp+72%2F73+by+herbimycin+A%2C+an+inhibitor+of+transformation+by+tyrosine+kinase+oncogenes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000648-Abbott1" id="pone.0000648-Abbott1"></a>Abbott RJ, Gomes MF (1989) Population Genetic-Structure and Outcrossing Rate of Arabidopsis-Thaliana (L) Heynh. Heredity  62: 411–418.  <ul class="find" data-citedArticleID="969070" data-doi="10.1038/hdy.1989.56"><li><a href="http://dx.doi.org/10.1038/hdy.1989.56" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Population+Genetic-Structure+and+Outcrossing+Rate+of+Arabidopsis-Thaliana+%28L%29+Heynh." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Population+Genetic-Structure+and+Outcrossing+Rate+of+Arabidopsis-Thaliana+%28L%29+Heynh.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000648-Sangster1" id="pone.0000648-Sangster1"></a>Sangster TA, Lindquist S, Queitsch C (2004) Under cover: causes, effects and implications of Hsp90-mediated genetic capacitance. Bioessays  26: 348–362.  <ul class="find" data-citedArticleID="969150" data-doi="10.1002/bies.20020"><li><a href="http://dx.doi.org/10.1002/bies.20020" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Under+cover%3A+causes%2C+effects+and+implications+of+Hsp90-mediated+genetic+capacitance." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Under+cover%3A+causes%2C+effects+and+implications+of+Hsp90-mediated+genetic+capacitance.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000648-Rutherford2" id="pone.0000648-Rutherford2"></a>Rutherford SL (2000) From genotype to phenotype: buffering mechanisms and the storage of genetic information. Bioessays  22: 1095–1105.  <ul class="find" data-citedArticleID="969148" data-doi="10.1002/1521-1878(200012)22:12<1095::aid-bies7&gt;3.0.co;2-a"><li><a href="http://dx.doi.org/10.1002/1521-1878(200012)22:12<1095::aid-bies7&gt;3.0.co;2-a" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genotype+to+phenotype%3A+buffering+mechanisms+and+the+storage+of+genetic+information." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genotype+to+phenotype%3A+buffering+mechanisms+and+the+storage+of+genetic+information.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000648-The2" id="pone.0000648-The2"></a>The Gene Ontology Consortium (2000) Gene ontology: tool for the unification of biology. Nat Genet  25: 25–29.  <ul class="find" data-citedArticleID="969162"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+ontology%3A+tool+for+the+unification+of+biology.&amp;auth=&amp;atitle=Gene+ontology%3A+tool+for+the+unification+of+biology." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+ontology%3A+tool+for+the+unification+of+biology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+ontology%3A+tool+for+the+unification+of+biology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000648-CastilloDavis1" id="pone.0000648-CastilloDavis1"></a>Castillo-Davis CI, Hartl DL (2003) GeneMerge–post-genomic analysis, data mining, and hypothesis testing. Bioinformatics  19: 891–892.  <ul class="find" data-citedArticleID="969086" data-doi="10.1093/bioinformatics/btg114"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btg114" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GeneMerge%26%23x2013%3Bpost-genomic+analysis%2C+data+mining%2C+and+hypothesis+testing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GeneMerge%26%23x2013%3Bpost-genomic+analysis%2C+data+mining%2C+and+hypothesis+testing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000648-Heil1" id="pone.0000648-Heil1"></a>Heil M, Bostock RM (2002) Induced systemic resistance (ISR) against pathogens in the context of induced plant defences. Ann Bot (Lond)  89: 503–512.  <ul class="find" data-citedArticleID="969104"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Induced+systemic+resistance+%28ISR%29+against+pathogens+in+the+context+of+induced+plant+defences.&amp;auth=&amp;atitle=Induced+systemic+resistance+%28ISR%29+against+pathogens+in+the+context+of+induced+plant+defences." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Induced+systemic+resistance+%28ISR%29+against+pathogens+in+the+context+of+induced+plant+defences." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Induced+systemic+resistance+%28ISR%29+against+pathogens+in+the+context+of+induced+plant+defences.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000648-Cui1" id="pone.0000648-Cui1"></a>Cui J, Bahrami AK, Pringle EG, Hernandez-Guzman G, Bender CL, et al.  (2005) Pseudomonas syringae manipulates systemic plant defenses against pathogens and herbivores. Proc Natl Acad Sci U S A  102: 1791–1796.  <ul class="find" data-citedArticleID="969092" data-doi="10.1073/pnas.0409450102"><li><a href="http://dx.doi.org/10.1073/pnas.0409450102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pseudomonas+syringae+manipulates+systemic+plant+defenses+against+pathogens+and+herbivores." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pseudomonas+syringae+manipulates+systemic+plant+defenses+against+pathogens+and+herbivores.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000648-Cui2" id="pone.0000648-Cui2"></a>Cui J, Jander G, Racki LR, Kim PD, Pierce NE, et al.  (2002) Signals involved in Arabidopsis resistance to Trichoplusia ni caterpillars induced by virulent and avirulent strains of the phytopathogen Pseudomonas syringae. Plant Physiol  129: 551–564.  <ul class="find" data-citedArticleID="969094" data-doi="10.1104/pp.010815"><li><a href="http://dx.doi.org/10.1104/pp.010815" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signals+involved+in+Arabidopsis+resistance+to+Trichoplusia+ni+caterpillars+induced+by+virulent+and+avirulent+strains+of+the+phytopathogen+Pseudomonas+syringae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signals+involved+in+Arabidopsis+resistance+to+Trichoplusia+ni+caterpillars+induced+by+virulent+and+avirulent+strains+of+the+phytopathogen+Pseudomonas+syringae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000648-Karban1" id="pone.0000648-Karban1"></a>Karban R, Baldwin IT (1997) Induced Responses to Herbivory. Chicago, IL: University of Chicago Press.   <ul class="find-nolinks"></ul></li><li><span class="label">48.
              </span><a name="pone.0000648-McBride1" id="pone.0000648-McBride1"></a>McBride KE, Summerfelt KR (1990) Improved binary vectors for Agrobacterium-mediated plant transformation. Plant Mol Biol  14: 269–276.  <ul class="find" data-citedArticleID="969130" data-doi="10.1007/bf00018567"><li><a href="http://dx.doi.org/10.1007/bf00018567" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improved+binary+vectors+for+Agrobacterium-mediated+plant+transformation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improved+binary+vectors+for+Agrobacterium-mediated+plant+transformation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0000648-Krizek1" id="pone.0000648-Krizek1"></a>Krizek BA, Meyerowitz EM (1996) Mapping the protein regions responsible for the functional specificities of the Arabidopsis MADS domain organ-identity proteins. Proc Natl Acad Sci U S A  93: 4063–4070.  <ul class="find" data-citedArticleID="969122" data-doi="10.1073/pnas.93.9.4063"><li><a href="http://dx.doi.org/10.1073/pnas.93.9.4063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mapping+the+protein+regions+responsible+for+the+functional+specificities+of+the+Arabidopsis+MADS+domain+organ-identity+proteins." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mapping+the+protein+regions+responsible+for+the+functional+specificities+of+the+Arabidopsis+MADS+domain+organ-identity+proteins.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0000648-Haseloff1" id="pone.0000648-Haseloff1"></a>Haseloff J, Siemering KR, Prasher DC, Hodge S (1997) Removal of a cryptic intron and subcellular localization of green fluorescent protein are required to mark transgenic Arabidopsis plants brightly. Proc Natl Acad Sci U S A  94: 2122–2127.  <ul class="find" data-citedArticleID="969102" data-doi="10.1073/pnas.94.6.2122"><li><a href="http://dx.doi.org/10.1073/pnas.94.6.2122" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Removal+of+a+cryptic+intron+and+subcellular+localization+of+green+fluorescent+protein+are+required+to+mark+transgenic+Arabidopsis+plants+brightly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Removal+of+a+cryptic+intron+and+subcellular+localization+of+green+fluorescent+protein+are+required+to+mark+transgenic+Arabidopsis+plants+brightly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0000648-Bechtold1" id="pone.0000648-Bechtold1"></a>Bechtold N, Pelletier G (1998) In planta Agrobacterium-mediated transformation of adult Arabidopsis thaliana plants by vacuum infiltration. Methods Mol Biol  82: 259–266.  <ul class="find" data-citedArticleID="969076" data-doi="10.1385/0-89603-391-0:259"><li><a href="http://dx.doi.org/10.1385/0-89603-391-0:259" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+planta+Agrobacterium-mediated+transformation+of+adult+Arabidopsis+thaliana+plants+by+vacuum+infiltration." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+planta+Agrobacterium-mediated+transformation+of+adult+Arabidopsis+thaliana+plants+by+vacuum+infiltration.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0000648-Campisi1" id="pone.0000648-Campisi1"></a>Campisi L, Yang Y, Yi Y, Heilig E, Herman B, et al.  (1999) Generation of enhancer trap lines in Arabidopsis and characterization of expression patterns in the inflorescence. Plant J  17: 699–707.  <ul class="find" data-citedArticleID="969082" data-doi="10.1046/j.1365-313x.1999.00409.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1999.00409.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Generation+of+enhancer+trap+lines+in+Arabidopsis+and+characterization+of+expression+patterns+in+the+inflorescence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Generation+of+enhancer+trap+lines+in+Arabidopsis+and+characterization+of+expression+patterns+in+the+inflorescence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0000648-Hoagland1" id="pone.0000648-Hoagland1"></a>Hoagland DR, Arnon DI (1950) The water-culture method for growing plants without soils. California Agr Expt Sta Circ  347:   <ul class="find" data-citedArticleID="969106"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+water-culture+method+for+growing+plants+without+soils.&amp;auth=&amp;atitle=The+water-culture+method+for+growing+plants+without+soils." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+water-culture+method+for+growing+plants+without+soils." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+water-culture+method+for+growing+plants+without+soils.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.XML" value="138128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.PDF" value="827426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g001.PNG_L" value="239255"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g001.PNG_M" value="33285"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g001.PNG_S" value="5404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g001.TIF" value="377500"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g001.PNG_I" value="12142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t001.PNG_L" value="158066"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t001.PNG_M" value="162386"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t001.PNG_S" value="20395"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t001.TIF" value="707470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t001.PNG_I" value="83639"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g002.PNG_L" value="3938628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g002.PNG_M" value="373029"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g002.PNG_S" value="12915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g002.TIF" value="6055440"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g002.PNG_I" value="128641"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g003.PNG_L" value="7418083"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g003.PNG_M" value="388896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g003.PNG_S" value="12357"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g003.TIF" value="9342214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g003.PNG_I" value="122015"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g004.PNG_L" value="298011"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g004.PNG_M" value="80928"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g004.PNG_S" value="11785"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g004.TIF" value="883430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g004.PNG_I" value="48204"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g005.PNG_L" value="331670"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g005.PNG_M" value="95604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g005.PNG_S" value="8771"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g005.TIF" value="569288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g005.PNG_I" value="34566"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g006.PNG_L" value="7866065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g006.PNG_M" value="502306"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g006.PNG_S" value="12594"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g006.TIF" value="10430368"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g006.PNG_I" value="151497"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g007.PNG_L" value="12252039"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g007.PNG_M" value="720086"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g007.PNG_S" value="16017"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g007.TIF" value="16403142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g007.PNG_I" value="218651"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t002.PNG_L" value="132541"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t002.PNG_M" value="74201"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t002.PNG_S" value="10811"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t002.TIF" value="573930"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.t002.PNG_I" value="23065"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g008.PNG_L" value="548154"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g008.PNG_M" value="119416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g008.PNG_S" value="9695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g008.TIF" value="1267016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.g008.PNG_I" value="43190"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.s001.TXT" value="4061"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.s002.TXT" value="2915"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.s003.TXT" value="2416"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000648.s004.TXT" value="801420"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000648&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000648" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000648&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000648&volume=&issue=&title=Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels&author_name=Todd%20A.%20Sangster%2C%20Adam%20Bahrami%2C%20Amity%20Wilczek%2C%20Etsuko%20Watanabe%2C%20Kurt%20Schellenberg%2C%20Catherine%20McLellan%2C%20Alicia%20Kelley%2C%20Sek%20Won%20Kong%2C%20Christine%20Queitsch%2C%20Susan%20Lindquist&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000648" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648&amp;t=Phenotypic%20Diversity%20and%20Altered%20Environmental%20Plasticity%20in%20Arabidopsis%20thaliana%20with%20Reduced%20Hsp90%20Levels" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648&title=Phenotypic%20Diversity%20and%20Altered%20Environmental%20Plasticity%20in%20Arabidopsis%20thaliana%20with%20Reduced%20Hsp90%20Levels&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648&amp;title=Phenotypic%20Diversity%20and%20Altered%20Environmental%20Plasticity%20in%20Arabidopsis%20thaliana%20with%20Reduced%20Hsp90%20Levels" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000648&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Phenotypic Diversity and Altered Environmental Plasticity in Arabidopsis thaliana with Reduced Hsp90 Levels';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000648';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Phenotypic%20Diversity%20and%20Altered%20Environmental%20Plasticity%20in%20Arabidopsis%20thaliana%20with%20Reduced%20Hsp90%20Levels http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000648" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000648" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">











          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Arabidopsis+thaliana%22" title="Search for articles in the subject area:'Arabidopsis thaliana'"><div class="flagText">Arabidopsis thaliana</div></a>
              <div data-categoryid="32845" data-articleid="24134"
                   data-categoryname="Arabidopsis thaliana"
                   class="flagImage" title="Flag 'Arabidopsis thaliana' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Flowering+plants%22" title="Search for articles in the subject area:'Flowering plants'"><div class="flagText">Flowering plants</div></a>
              <div data-categoryid="43015" data-articleid="24134"
                   data-categoryname="Flowering plants"
                   class="flagImage" title="Flag 'Flowering plants' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Heat+shock+response%22" title="Search for articles in the subject area:'Heat shock response'"><div class="flagText">Heat shock response</div></a>
              <div data-categoryid="46383" data-articleid="24134"
                   data-categoryname="Heat shock response"
                   class="flagImage" title="Flag 'Heat shock response' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Leaves%22" title="Search for articles in the subject area:'Leaves'"><div class="flagText">Leaves</div></a>
              <div data-categoryid="17987" data-articleid="24134"
                   data-categoryname="Leaves"
                   class="flagImage" title="Flag 'Leaves' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Phenotypes%22" title="Search for articles in the subject area:'Phenotypes'"><div class="flagText">Phenotypes</div></a>
              <div data-categoryid="40235" data-articleid="24134"
                   data-categoryname="Phenotypes"
                   class="flagImage" title="Flag 'Phenotypes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant-herbivore+interactions%22" title="Search for articles in the subject area:'Plant-herbivore interactions'"><div class="flagText">Plant-herbivore interactions</div></a>
              <div data-categoryid="33687" data-articleid="24134"
                   data-categoryname="Plant-herbivore interactions"
                   class="flagImage" title="Flag 'Plant-herbivore interactions' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Seedlings%22" title="Search for articles in the subject area:'Seedlings'"><div class="flagText">Seedlings</div></a>
              <div data-categoryid="43017" data-articleid="24134"
                   data-categoryname="Seedlings"
                   class="flagImage" title="Flag 'Seedlings' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Seeds%22" title="Search for articles in the subject area:'Seeds'"><div class="flagText">Seeds</div></a>
              <div data-categoryid="18605" data-articleid="24134"
                   data-categoryname="Seeds"
                   class="flagImage" title="Flag 'Seeds' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=8836'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=5848'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3176&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
