

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Regulated Polyploidy in Halophilic Archaea</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000092"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000092" />

    <meta name="citation_title" content="Regulated Polyploidy in Halophilic Archaea"/>
    <meta itemprop="name" content="Regulated Polyploidy in Halophilic Archaea"/>

      <meta name="citation_author" content="Sebastian Breuert"/>
            <meta name="citation_author_institution" content="Goethe University, Institute for Molecular Biosciences, Frankfurt, Germany"/>
      <meta name="citation_author" content="Thorsten Allers"/>
            <meta name="citation_author_institution" content="Institute of Genetics, University of Nottingham, Queen's Medical Centre, United Kingdom"/>
      <meta name="citation_author" content="Gabi Spohn"/>
            <meta name="citation_author_institution" content="Goethe University, Institute for Molecular Biosciences, Frankfurt, Germany"/>
      <meta name="citation_author" content="Jörg Soppa"/>
            <meta name="citation_author_institution" content="Goethe University, Institute for Molecular Biosciences, Frankfurt, Germany"/>

    <meta name="citation_date" content="2006/12/20"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000092.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e92"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="1"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Yesterday's polyploids and the mystery of diploidization.; citation_author=KH Wolfe; citation_journal_title=Nat Rev Genet; citation_volume=2; citation_number=1; citation_pages=333-341; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The advantages and disadvantages of being polyploid.; citation_author=L Comai; citation_journal_title=Nat Rev Genet; citation_volume=6; citation_number=2; citation_pages=836-846; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome evolution in polyploids.; citation_author=JF Wendel; citation_journal_title=Plant Mol Biol; citation_volume=42; citation_number=3; citation_pages=225-249; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Modulation of Chemical Composition and Other Parameters of the Cell Growth Rate.; citation_author=H Bremer; citation_author=PP Dennis; citation_number=4; citation_pages=1553-1566; citation_date=1996; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=Timing of initiation of chromosome replication in individual Escherichia coli cells.; citation_author=K Skarstad; citation_author=E Boye; citation_author=HB Steen; citation_journal_title=EMBO J; citation_volume=5; citation_number=5; citation_pages=1711-1717; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Use of time-lapse microscopy to visualize rapid movement of the replication origin region of the chromosome during the cell cycle in Bacillus subtilis.; citation_author=C Webb; citation_author=P Graumann; citation_author=J Kahana; citation_author=A Telemann; citation_author=P Silver; citation_journal_title=Mol Microbiol; citation_volume=28; citation_number=6; citation_pages=883-892; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Localization of bacterial DNA polymerase: evidence for a factory model of replication.; citation_author=KP Lemon; citation_author=AD Grossman; citation_journal_title=Science; citation_volume=282; citation_number=7; citation_pages=1516-1519; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Multiplicity of genome equivalents in the radiation-resistant bacterium Micrococcus radiodurans.; citation_author=MT Hansen; citation_journal_title=J Bacteriol; citation_volume=134; citation_number=8; citation_pages=71-75; citation_date=1978; " />
      <meta name="citation_reference" content="citation_title=Multiple chromosomes of Azotobacter vinelandii.; citation_author= Punita; citation_author=SJ Jafri; citation_author=MA Reddy; citation_author=HK Das; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=9; citation_pages=3133-3138; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Changes of ploidy during the Azotobacter vinelandii growth cycle.; citation_author=R Maldonado; citation_author=J Jimenez; citation_author=J Casadesus; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=10; citation_pages=3911-3919; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Chromosome replication patterns in the hyperthermophilic euryarchaea Archaeoglobus fulgidus and Methanocaldococcus (Methanococcus) jannaschii.; citation_author=S Maisnier-Patin; citation_author=L Malandrin; citation_author=NK Birkeland; citation_author=R Bernander; citation_journal_title=Mol Microbiol; citation_volume=45; citation_number=11; citation_pages=1443-1450; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Changes in cell size and DNA content in Sulfolobus cultures during dilution and temperature shift experiments.; citation_author=K Hjort; citation_author=R Bernander; citation_journal_title=J Bacteriol; citation_volume=181; citation_number=12; citation_pages=5669-5675; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Cell cycle characteristics of thermophilic archaea.; citation_author=R Bernander; citation_author=A Poplawski; citation_journal_title=J Bacteriol; citation_volume=179; citation_number=13; citation_pages=4963-4969; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=DNA content and nucleoid distribution in Methanothermobacter thermautotrophicus.; citation_author=AI Majernik; citation_author=M Lundgren; citation_author=P McDermott; citation_author=R Bernander; citation_author=JP Chong; citation_journal_title=J Bacteriol; citation_volume=187; citation_number=14; citation_pages=1856-1858; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Nucleoid structure and partition in Methanococcus jannaschii: an archaeon with multiple copies of the chromosome.; citation_author=L Malandrin; citation_author=H Huber; citation_author=R Bernander; citation_journal_title=Genetics; citation_volume=152; citation_number=15; citation_pages=1315-1323; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The protein synthesizing machinery of the Archaebacterium Halobacterium-cutirubrum- molecular characterization.; citation_author=J Chant; citation_author=I Hui; citation_author=D Dejongwong; citation_author=L Shimmin; citation_author=PP Dennis; citation_journal_title=Sys Appl Microbiol; citation_volume=7; citation_number=16; citation_pages=106-114; citation_date=1986; " />
      <meta name="citation_reference" content="citation_title=Cell cycle-dependent expression of an essential SMC-like protein and dynamic chromosome localization in the archaeon Halobacterium salinarum.; citation_author=U Herrmann; citation_author=J Soppa; citation_journal_title=Mol Microbiol; citation_volume=46; citation_number=17; citation_pages=395-409; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=From genomes to function: haloarchaea as model organisms.; citation_author=J Soppa; citation_journal_title=Microbiology; citation_volume=152; citation_number=18; citation_pages=585-590; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Flow cytometry analysis of changes in the DNA content of the polychlorinated biphenyl degrader Comamonas testosteroni TK102: effect of metabolites on cell-cell separation.; citation_author=Y Hiraoka; citation_author=T Yamada; citation_author=K Tone; citation_author=Y Futaesaku; citation_author=K Kimbara; citation_journal_title=Appl Environ Microbiol; citation_volume=68; citation_number=19; citation_pages=5104-5112; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Obligate Human Pathogen,Neisseria gonorrhoeae, Is Polyploid.; citation_author=DM Tobiason; citation_author=HS Seifert; citation_journal_title=PLoS Biol; citation_volume=4; citation_number=20; citation_pages=e185; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The genomes of Desulfovibrio gigas and D. vulgaris.; citation_author=JR Postgate; citation_author=HM Kent; citation_author=RL Robson; citation_author=JA Chesshyre; citation_journal_title=J Gen Microbiol; citation_volume=130; citation_number=21; citation_pages=1597-1601; citation_date=1984; " />
      <meta name="citation_reference" content="citation_title=The relapsing fever agent Borrelia hermsii has multiple copies of its chromosome and linear plasmids.; citation_author=T Kitten; citation_author=AG Barbour; citation_journal_title=Genetics; citation_volume=132; citation_number=22; citation_pages=311-324; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=Multiple chromosomes of Azotobacter vinelandii.; citation_author=P Nagpal; citation_author=S Jafri; citation_author=MA Reddy; citation_author=HK Das; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=23; citation_pages=3133-3138; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Gene dosage analysis in Azotobacter vinelandii.; citation_author=R Maldonado; citation_author=A Garzon; citation_author=DR Dean; citation_author=J Casadesus; citation_journal_title=Genetics; citation_volume=132; citation_number=24; citation_pages=869-878; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=
                  In vivo damage and recA-dependent repair of plasmid and chromosomal DNA in the radiation-resistant bacterium Deinococcus radiodurans.; citation_author=MJ Daly; citation_author=L Ouyang; citation_author=P Fuchs; citation_author=KW Minton; citation_journal_title=J Bacteriol; citation_volume=176; citation_number=25; citation_pages=3508-3517; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Genome multiplicity and radiation resistance in Micrococcus radiodurans.; citation_author=HarsojoS Kitayama; citation_author=A Matsuyama; citation_journal_title=J Biochem (Tokyo); citation_volume=90; citation_number=26; citation_pages=877-80; citation_date=1981; " />
      <meta name="citation_reference" content="citation_title=A data analysis of the irradiation parameter D10 for bacteria and spores under various conditions.; citation_author=SJ van Gerwen; citation_author=FM Rombouts; citation_author=K van't Riet; citation_author=MH Zwietering; citation_journal_title=J Food Prot; citation_volume=62; citation_number=27; citation_pages=1024-1032; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Physiological responses of the halophilic archaeon Halobacterium sp. strain NRC1 to desiccation and gamma irradiation.; citation_author=M Kottemann; citation_author=A Kish; citation_author=C Iloanusi; citation_author=S Bjork; citation_author=J DiRuggiero; citation_journal_title=Extremophiles; citation_volume=9; citation_number=28; citation_pages=219-227; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inactivation of Escherichia coli 0157:H7, Salmonellae, and Camphylobacter jejuni in raw ground beef by gamma irradiation.; citation_author=MRS Clavero; citation_author=JD Monk; citation_author=LR Beuchat; citation_author=MP Doyle; citation_author=RE Brackett; citation_journal_title=Appl Environ Microbiol; citation_volume=60; citation_number=29; citation_pages=2069-2075; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Optimization of the 5-bromo-2′-deoxyuridine selection and its application for the isolation of nitrate respiration-deficient mutants of Haloferax volcanii.; citation_author=J Soppa; citation_journal_title=J Microbiol Methods; citation_volume=34; citation_number=30; citation_pages=41-48; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Bacteriorhodopsin mutants of Halobacterium sp. GRB. II. Characterization of mutants.; citation_author=J Soppa; citation_author=J Otomo; citation_author=J Straub; citation_author=J Tittor; citation_author=S Meessen; citation_journal_title=J Biol Chem; citation_volume=264; citation_number=31; citation_pages=13049-13056; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Development of a gene knockout system for the halophilic archaeon Haloferax volcanii by use of the pyrE gene.; citation_author=G Bitan-Banin; citation_author=R Ortenberg; citation_author=M Mevarech; citation_journal_title=J Bacteriol; citation_volume=185; citation_number=32; citation_pages=772-778; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Development of additional selectable markers for the halophilic archaeon Haloferax volcanii based on the leuB and trpA genes.; citation_author=T Allers; citation_author=HP Ngo; citation_author=M Mevarech; citation_author=RG Lloyd; citation_journal_title=Appl Environ Microbiol; citation_volume=70; citation_number=33; citation_pages=943-953; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Homologous gene knockout in the archaeon Halobacterium salinarum with ura3 as a counterselectable marker.; citation_author=R Peck; citation_author=S Dassarma; citation_author=M Krebs; citation_journal_title=Mol Microbiol; citation_volume=35; citation_number=34; citation_pages=667-676; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Arsenic resistance in Halobacterium sp. strain NRC-1 examined by using an improved gene knockout system.; citation_author=G Wang; citation_author=SP Kennedy; citation_author=S Fasiludeen; citation_author=C Rensing; citation_author=S DasSarma; citation_journal_title=J Bacteriol; citation_volume=186; citation_number=35; citation_pages=3187-3194; citation_date=2004; " />
      <meta name="citation_reference" content="citation_author=I Rosenshine; citation_author=M Mevarech; citation_number=36; citation_pages=265-270; citation_date=1991; citation_publisher=Plenum Press; " />
      <meta name="citation_reference" content="citation_title=Gene conversion drives within genic sequences: concerted evolution of ribosomal RNA genes in bacteria and archaea.; citation_author=D Liao; citation_journal_title=J Mol Evol; citation_volume=51; citation_number=37; citation_pages=305-317; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Gene conversion and concerted evolution in bacterial genomes.; citation_author=G Santoyo; citation_author=D Romero; citation_journal_title=FEMS Microbiol Rev; citation_volume=29; citation_number=38; citation_pages=169-183; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Elimination of deleterious mutations in plastid genomes by gene conversion.; citation_author=O Khakhlova; citation_author=R Bock; citation_journal_title=Plant J.; citation_volume=46; citation_number=39; citation_pages=85-94; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Catabolic ornithine transcarbamylase of Halobacterium halobium (salinarium): purification, characterization, sequence determination, and evolution.; citation_author=A Ruepp; citation_author=HN Muller; citation_author=F Lottspeich; citation_author=J Soppa; citation_journal_title=J Bacteriol; citation_volume=177; citation_number=40; citation_pages=1129-1136; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Transformation of the archaebacterium Halobacterium volcanii with genomic DNA.; citation_author=SW Cline; citation_author=LC Schalkwyk; citation_author=WF Doolittle; citation_journal_title=J Bacteriol; citation_volume=171; citation_number=41; citation_pages=4987-4991; citation_date=1989; " />
      <meta name="citation_reference" content="citation_title=Techniques for transformation of E.coli.; citation_author=D Hanahan; citation_number=42; citation_pages=109-135; citation_date=1985; citation_publisher=Oxford; " />
      <meta name="citation_reference" content="citation_author=BJ Bachmann; citation_number=43; citation_pages=2460-2488; citation_date=1996; citation_publisher=ASM Press; " />
      <meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12.; citation_author=FR Blattner; citation_author=G Plunkett; citation_author=CA Bloch; citation_author=NT Perna; citation_author=V Burland; citation_journal_title=Science; citation_volume=277; citation_number=44; citation_pages=1453-1474; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Genome sequence of Halobacterium spec. NRC-1.; citation_author=WV Ng; citation_author=SP Kennedy; citation_author=GG Mahairas; citation_author=B Berquist; citation_author=M Pan; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=45; citation_pages=12176-12181; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Fermentative arginine degradation in Halobacterium salinarium (halobium): genes, gene products, and transcripts of the arcRACB gene cluster.; citation_author=A Ruepp; citation_author=J Soppa; citation_journal_title=J Bacteriol; citation_volume=178; citation_number=46; citation_pages=4942-4947; citation_date=1996; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Regulated Polyploidy in Halophilic Archaea"/>
    <meta name="twitter:description" content="Polyploidy is common in higher eukaryotes, especially in plants, but it is generally assumed that most prokaryotes contain a single copy of a circular chromosome and are therefore monoploid. We have used two independent methods to determine the genome copy number in halophilic archaea, 1) cell lysis in agarose blocks and Southern blot analysis, and 2) Real-Time quantitative PCR. Fast growing H. salinarum cells contain on average about 25 copies of the chromosome in exponential phase, and their ploidy is downregulated to 15 copies in early stationary phase. The chromosome copy number is identical in cultures with a twofold lower growth rate, in contrast to the results reported for several other prokaryotic species. Of three additional replicons of H. salinarum, two have a low copy number that is not growth-phase regulated, while one replicon even shows a higher degree of growth phase-dependent regulation than the main replicon. The genome copy number of H. volcanii is similarly high during exponential phase (on average 18 copies/cell), and it is also downregulated (to 10 copies) as the cells enter stationary phase. The variation of genome copy numbers in the population was addressed by fluorescence microscopy and by FACS analysis. These methods allowed us to verify the growth phase-dependent regulation of ploidy in H. salinarum, and they revealed that there is a wide variation in genome copy numbers in individual cells that is much larger in exponential than in stationary phase. Our results indicate that polyploidy might be more widespread in archaea (or even prokaryotes in general) than previously assumed. Moreover, the presence of so many genome copies in a prokaryote raises questions about the evolutionary significance of this strategy."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000092.g006"/>

  <meta property="og:title" content="Regulated Polyploidy in Halophilic Archaea" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=8004'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4264'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3114&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000092">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000092" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000092&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Regulated Polyploidy in Halophilic Archaea
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sebastian Breuert, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Goethe University, Institute for Molecular Biosciences, Frankfurt, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Thorsten Allers, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Institute of Genetics, University of Nottingham, Queen's Medical Centre, United Kingdom
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Gabi Spohn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Goethe University, Institute for Molecular Biosciences, Frankfurt, Germany
                </p>

                <p>
                    Current address: Institut für Genetik, Köln, Germany
                </p>

              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jörg Soppa
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:soppa@em.uni-frankfurt.de">soppa@em.uni-frankfurt.de</a>
            </p>

                <p>Affiliation:
                  Goethe University, Institute for Molecular Biosciences, Frankfurt, Germany
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: December 20, 2006</li>
    <li>DOI: 10.1371/journal.pone.0000092</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000092-g001" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000092-g002" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000092-g003" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000092-g004" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000092-g005" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000092-g006" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000092-t001" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000092">Reader Comments (1)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000092" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Polyploidy is common in higher eukaryotes, especially in plants, but it is generally assumed that most prokaryotes contain a single copy of a circular chromosome and are therefore monoploid. We have used two independent methods to determine the genome copy number in halophilic archaea, 1) cell lysis in agarose blocks and Southern blot analysis, and 2) Real-Time quantitative PCR. Fast growing <em>H. salinarum</em> cells contain on average about 25 copies of the chromosome in exponential phase, and their ploidy is downregulated to 15 copies in early stationary phase. The chromosome copy number is identical in cultures with a twofold lower growth rate, in contrast to the results reported for several other prokaryotic species. Of three additional replicons of <em>H. salinarum</em>, two have a low copy number that is not growth-phase regulated, while one replicon even shows a higher degree of growth phase-dependent regulation than the main replicon. The genome copy number of <em>H. volcanii</em> is similarly high during exponential phase (on average 18 copies/cell), and it is also downregulated (to 10 copies) as the cells enter stationary phase. The variation of genome copy numbers in the population was addressed by fluorescence microscopy and by FACS analysis. These methods allowed us to verify the growth phase-dependent regulation of ploidy in <em>H. salinarum</em>, and they revealed that there is a wide variation in genome copy numbers in individual cells that is much larger in exponential than in stationary phase. Our results indicate that polyploidy might be more widespread in archaea (or even prokaryotes in general) than previously assumed. Moreover, the presence of so many genome copies in a prokaryote raises questions about the evolutionary significance of this strategy.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Breuert S, Allers T, Spohn G, Soppa J (2006) Regulated Polyploidy in Halophilic Archaea. PLoS ONE 1(1):
          e92.
            doi:10.1371/journal.pone.0000092</p><p><strong>Academic Editor: </strong>Beth Sullivan, Duke University, United States of America</p><p><strong>Received:</strong> September 22, 2006; <strong>Accepted:</strong> October 2, 2006; <strong>Published:</strong> December 20, 2006</p><p><strong>Copyright:</strong> © 2006 Breuert et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by the Deutsche Forschungsgemeinschaft (German Research Council) through grant No. So264/7-2 (to JS), and by a Royal Society University Research Fellowship (to TA).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Polyploidy, the existence of multiple copies of the normal set of chromosomes, is widely distributed in eukaryotes. Polyploids are common among fish and amphibian and very common among plants. It has even been proposed that the diploid vertebrate genomes were derived by reduction from polyploids <a href="#pone.0000092-Wolfe1">[1]</a>. The advantages and disadvantages of being polyploid have recently been reviewed <a href="#pone.0000092-Comai1">[2]</a>, <a href="#pone.0000092-Wendel1">[3]</a>. In short, the advantages of polyploidy are heterosis (hybrid vigour, an increased perfomance of the allopolyploid compared with the inbred parents), loss of self-incompatibility leading to the gain of asexual reproduction, and gene redundancy. The specific advantages of gene redundancy may be 1) a reduced chance that deleterious recessive mutations become homozygous, and 2) the potential for gene diversification and the acquisition of new functions. Disadvantages of being polyploid are 1) a higher frequency of mitotic or meiotic problems leading to aneuploidy and 2) epigenetic instability. Several of these points like heterosis apply only to eukaryotic species with sexual reproduction, while others like gene redundancy are relevant for archaea, bacteria and eukaryotes.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>In contrast to eukaryotes, prokaryotes are commonly thought to contain one copy of a circular chromosome. The gram-negative bacterium <em>Escherichia coli</em> contains one complete chromosome when its generation time is longer than the time required for replication and segregation of the chromosome. Unlike eukaryotes, it does not have a G2 phase, and cell division is initiated soon after replication has been finished <a href="#pone.0000092-Bremer1">[4]</a>. Under optimal laboratory conditions, the generation time of <em>E. coli</em> can become less than the replication time, and a new round of replication is initiated before termination of the previous round. The cells become merodiploid or merooligoploid for the origin-proximal genes <a href="#pone.0000092-Bremer1">[4]</a>, and if replication reinitiation is prevented with rifampicin, end up with 2, 4 or 8 chromosomes <a href="#pone.0000092-Skarstad1">[5]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>The best-studied gram-positive bacterium, <em>Bacillus subtilis</em>, also harbors one copy of the chromosome <a href="#pone.0000092-Webb1">[6]</a>. For <em>B. subtilis</em> and subsequently also for other bacterial species it was found that the chromosome is not distributed randomly in the cell. Replication takes place at a fixed site in the middle of the cell, and the newly replicated regions of the chromosome are immediately transported toward the cell poles <a href="#pone.0000092-Webb1">[6]</a>, <a href="#pone.0000092-Lemon1">[7]</a>. Thus, the DNA-polymerases and the replication forks are somewhat stationary and the DNA is highly mobile (“factory model of replication”) <a href="#pone.0000092-Lemon1">[7]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Several bacteria are known to be polypoid. The best known example is the radioresistant species <em>Deinococcus radiodurans</em>, which harbours about 8 chromosomal copies <a href="#pone.0000092-Hansen1">[8]</a>. Another example is <em>Azotobacter vinelandii</em> that has been reported to contain up to 80 chromosome copies <a href="#pone.0000092-Punita1">[9]</a>. However, this is only seen in fast growing cultures, while cultures grown in synthetic medium are not polyploid <a href="#pone.0000092-Maldonado1">[10]</a>. A few other examples exist, but are thought to be exceptions from the rule that bacteria are monoploid.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Genome copiy numbers were also determined for several species from the third domain of life, the archaea. Two <em>Sulfolobus</em> species and <em>Archaeoglobus fulgidus</em> were found to have an extensive G2 phase; thus they contain two copies during most of the cell cycle, and one copy prior to replication <a href="#pone.0000092-MaisnierPatin1">[11]</a>–<a href="#pone.0000092-Bernander1">[13]</a>. <em>Methanothermobacter thermoautotrophicus</em> has no G2 phase, but grows in filaments that contain several nucleoids, each of which contains a single chromosome. <a href="#pone.0000092-Majernik1">[14]</a>. <em>Methanococcus jannaschii</em> contains multiple copies of the chromosome, but has a very relaxed cell cycle control. The division plane is not at mid-cell, and there is no even distribution of the copies to the daughter cells <a href="#pone.0000092-Malandrin1">[15]</a>.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>The genome copy number of the haloarchaea, had not been systematically investigated until now. It has been reported that <em>H. cutirubrum</em> may contain 6–10 genome copies <a href="#pone.0000092-Chant1">[16]</a>, and we have previously shown that in <em>H. salinarum</em>, intracellular localization of the DNA is highly regulated in the course of the cell cycle <a href="#pone.0000092-Herrmann1">[17]</a>. This prompted us to study the ploidy of <em>H. salinarum</em> and a second haloarchaeal model species, <em>H. volcanii</em>, during growth under different conditions. Surprisingly, it was found that both species are polyploid, and that the genome copy number is downregulated as cells enter stationary phase.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>
               <em>Halobacterium salinarum</em> has multiple copies of the chromosome</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>It was previously reported that <em>H. cutirubrum</em> contains six to ten copies of the chromosome <a href="#pone.0000092-Chant1">[16]</a>. This was calculated after quantitation of the total DNA content and of the cell density of a culture aliquot. We applied a similar approach to determine the ploidy of <em>H. salinarum</em> (see <a href="#s4">Materials and Methods</a>), using <em>E. coli</em> grown in synthetic medium as a control (under these conditions, <em>E coli</em> is known to contain on average slightly more than one copy of the chromosome). We confirmed that the genome copy number of <em>H. salinarum</em> is significantly higher than that of <em>E. coli</em>, similar to <em>H. cutirubrum</em> (data not shown). However, the variation in our results was somewhat high, especially if different methods of DNA isolation were used. We therefore wanted to confirm the results using independent methods with internal standardization.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>An overview of the first method is given in <a href="#pone-0000092-g001">Fig. 1A</a>. In short, a culture of known cell density is embedded in low melting point agarose and agarose blocks with a defined number of cells are prepared. A genomic fragment of about 1 kbp near the origin is visualized by Southern blot analysis. The same probe is used for the simultaneous detection of a 0.9 kbp PCR fragment that was added as an internal standard. A standard curve is generated by the variation of the ratio of the molecules of internal standard per cell (see <a href="#s4">Material and Methods</a>). The method allows the absolute determination of the intracellular copy number of specific DNA fragments and is not influenced by the presence of additional replicons.</p>
<div class="figure" id="pone-0000092-g001"><div class="img"><a name="pone-0000092-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>A. Overview of the method. A culture of known cell density is embedded in low melting point agarose (step 1), agarose blocks with a defined number of cells are prepared, the cells are lysed and protein is digested (step 2). The blocks are melted and a restriction enzyme (step 3) as well as an internal standard (step 4) are added. After overnight digestion, DNA fragments are size fractionated by electrophoresis and a Southern blot is performed (step 5). A 1 kbp genomic fragment near the replication origin and the 0.9 kbp internal standard are both visualized with a single probe. Multiple aliquots containing different standard concentration are used for quantitation. B. Quantitation of the genome copy number of exponential cells. After gel electrophoresis and southern blotting, a genomic fragment (upper band) and different concentrations of an internal standard (lower band) were visualized with the same probe (step 5 in A). C. Quantitation of the genome copy number of stationary phase cells. After gel electrophoresis and Southern blotting, a genomic fragment (upper band) and different concentrations of an internal standard (lower band) were visualized with the same probe (step 5 in A). D. An example of a standard curve generated after quantitation of the bands shown in B. and C. The horizontal and vertical lines show the usage of the standard curve to determine the genome copy number in the biological replicate No. 9 (see E.). E. Summary of the results of the independent cultures that were used to determine the genome copy number with the “agarose block method”. In the first five experiments, the standard curve ranged from 0.5 molecules/cell to 10 molecules/cell (as in B.). The signal of exponential phase cells was much higher than the highest signal of the standard curve and could not be quantitated. Therefore the standard curve was chosen to range from 1 molecules/cell to 40 molecules/cell (as in C.) in subsequent experiments.</p>
<span>doi:10.1371/journal.pone.0000092.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>
               <a href="#pone-0000092-g001">Fig. 1 B</a> shows the result for an exponential phase culture, <a href="#pone-0000092-g001">Fig. 1C</a> shows an example of a stationary phase culture, and <a href="#pone-0000092-g001">Fig. 1D</a> shows one representative standard curve used for quantitation. The results of 20 independent biological replicates are summarized in <a href="#pone-0000092-g001">Fig. 1 E</a>. It was found that stationary phase cells contain slightly more than 10 copies of the chromosome, while the ploidy of exponentially growing cells is considerably higher with around 30 copies/cell. Thus the polypoidy was verified and the results indicated that the ploidy of <em>H. salinarum</em> is growth phase-regulated. However, the “agarose block method” is time-consuming, inhibiting processing of large sample numbers, and the variability is somewhat greater than desired.</p>


<h4>The ploidy of <em>H. salinarum</em> is growth-phase regulated</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>A second method for genome copy number quantitation is shown in <a href="#pone-0000092-g002">Fig. 2A</a>, and was used to study growth phase dependent regulation of ploidy. In short, a defined number of cells was harvested and a specific genomic fragment was determined by Real Time PCR. Absolute quantitation was enabled by using external and internal standardization (see <a href="#s4">Material and Methods</a>). This novel method combined several advantages: it is very fast and precise, extremely sensitive, and includes only a small number of steps. <a href="#pone-0000092-g002">Fig. 2B</a> shows the Real Time PCR results using a dilution series of the cell lysate and of the standard as templates; all curves have virtually ideal slopes and offsets. A standard curve (<a href="#pone-0000092-g002">Fig. 2C</a>) shows the high precision of DNA quantitation by Real Time PCR.</p>
<div class="figure" id="pone-0000092-g002"><div class="img"><a name="pone-0000092-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>A. Overview of the method. In short, a defined number of cells was harvested and lysed (steps 1 and 2). Serial dilutions of the cell lysate (step 3) were used as templates in quantitative Real Time PCR assays (step 4). Quantitation was performed by comparison with an external and an internal standard curve (<a href="#s4">Materials and Methods</a>). B. Selected real time PCR results. The fluorescence intensity curves from four standard dilutions (solid lines) and three sample dilutions (broken lines) are shown. In both cases serial tenfold dilutions of the templates were used. Note the identical slope of all curves and the equidistance of the curves of a dilution series, which is very close to the theoretical offset of 3.32 cycles per tenfold dilution. In addition to the selected reactions shown here, each experiment included more standards and more sample dilutions (filling the gap between tenfold dilutions) as well as a sample dilutions including a dilutions series of the standard added as internal control of PCR efficiency. C. A standard curve including nine standard concentrations distributed over three orders of magnitude. D. Average genome copy number values of three independent cultures and their standard deviation. E. Growth phase-dependent regulation of ploidy of H. salinarum. E is a graphical representation of the results tabulated in D.</p>
<span>doi:10.1371/journal.pone.0000092.g002</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>The method was used to determine the ploidy of <em>H. salinarum</em> throughout its growth. Three independent experiments were performed, and the results are tabulated in <a href="#pone-0000092-g002">Fig. 2D</a> and shown graphically in <a href="#pone-0000092-g002">Fig. 2E</a>. Cells in the mid-exponential growth phase contain about 25 copies of the chromosome, and this value drops to 15 copies in early stationary phase. The two independent methods used are in good agreement, and show that replication and cell division can be regulated independently in <em>H. salinarum</em>; in late exponential phase one round of cell division occurs while replication has already ceased.</p>


<h4>Ploidy of <em>H. salinrum</em> at different growth rates</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The doubling time of <em>H. salinarum</em>in grown aerobically in complex medium at 42°C is about 4 hours. This is longer than the presumed replication time, given a genome size of 2.57 Mbp, and therefore meropolyploidy of origin proximal regions similar to fast growing <em>E. coli</em> is not expected. Nevertheless, we investigated whether the growth rate has an effect on ploidy, as had been reported for fast-growing <em>E. coli</em> and additional bacterial species (see <a href="#s3">Discussion</a>). Two different conditions were used that lower the growth rate by about a factor of two, 1) cultures were grown at 30°C instead of 42°C, and 2) they were grown anaerobically by arginine fermentation instead of aerobic respiration. In both cases, the doubling time is around 8 hours. Samples were taken at four time points representing early, mid- and late exponential as well as early stationary phase. We found no reduction in ploidy in exponential phase at lower growth rates, where cells contained at least 25 genome copies under all three conditions (<a href="#pone-0000092-g003">Fig. 3</a>). The only difference is seen in early stationary phase, where a smaller reduction of ploidy was seen in slow-growing cells (8 hour doubling time) than in fast-growing cells (4 hour doubling time). As a consequence, stationary phase cells have a varying DNA content that depends on their history and thus they seem to have a “molecular memory” of what conditions they had experienced before they became stationary.</p>
<div class="figure" id="pone-0000092-g003"><div class="img"><a name="pone-0000092-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>H. salinarum was grown by aerobic respiration at 42oC and 30 oC and by arginine fermentation at 42oC. The doubling times were 4 hours, 8 hours and 8 hours, respectively. For each condition three independent cultures were used. Aliquots representing early exponential phase (2–3×108 cells/ml), mid-exponential phase (4–5×108 cells/ml), late exponential phase (8–9×108 cells/ml) and early stationary phase (1–2×109 cells/ml) were used to determine the genome copy number using the Real Time PCR method. Average values of the three biological replicates and their standard deviation are shown.</p>
<span>doi:10.1371/journal.pone.0000092.g003</span></div>

<h4>Ploidy of different <em>H. salinarum</em> replicons and its regulation</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>
               <em>H. salinarum</em> contains several replicons [<a href="http://www.halolex.de/">http://www.halolex.de/</a>], and the results above pertain to the largest replicon (chromosome). We set out to quantitate the copy numbers of the additional replicons termed pHS1 to pHS4. Since the smallest replicon pHS4 is only present in the sequenced strain (DSM 671) but not in the strain used for this study (DSM 670), the analysis was restricted to pHS1 to pHS3. Real Time PCR assays for the three replicons were established, and the ploidy throughought the culture growth was quantitated in parallel with the chromosome (<a href="#pone-0000092-g004">Fig. 4</a>). Surprisingly the copy numbers of two of the plasmids, pHS 2 and pHS3, are much lower than that of the chromosome. Furthermore, their copy number is not regulated and is around five from early exponential phase to stationary phase. The ploidy of the third plasmid, pHS1, is regulated similar to that of the chromosome, but it has a much lower copy number in early exponential phase. Therefore at least two different mechanisms of replicon copy number regulation exist in <em>H. salinarum</em>.</p>
<div class="figure" id="pone-0000092-g004"><div class="img"><a name="pone-0000092-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Three independent cultures were used to determine the copy numbers of the chromosome and three additional replicons, i.e. pHS1 to pHS3, using the Real Time PCR method. One of the growth curves, the average replicon copy number per cell and the standard deviations are shown.</p>
<span>doi:10.1371/journal.pone.0000092.g004</span></div>

<h4>Ploidy of <em>Haloferax volcanii</em> and its regulation</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>To investigate whether the polyploidy is restricted to <em>H. salinarum</em> or might be more widely distributed in haloarchaea, <em>H.</em> volcanii was chosen as a second example. It belongs to another genus and differs considerable from <em>H. salinarum</em> in morphology, metabolism and behavior. The growth phase-dependent ploidy of <em>H. volcanii</em> was quantitated using the two different methods described above. Initial experiments were performed with the “agarose block method” and show that <em>H. volcanii</em> is also polyploid (<a href="#pone-0000092-g005">Fig. 5A</a>). Similar to <em>H. salinarum</em>, there is a negative correlation between the cell density and the chromosomal copy number, indicating growth phase-dependent copy number regulation.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>The genome copy number was also determined by the quantitative PCR method, which yielded near-identical results. <em>H. volcanii</em> has almost 20 genome copies in exponential growth phase, and around 12 copies in stationary phase. Thus, it has a slightly lower genome copy number and a somewhat lower degree of growth phase-dependent copy number regulation than <em>H. salinarum</em>.</p>


<h4>Distribution of genome copy numbers in single cells of the population</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>The two methods for genome copy number determination yielded information about the average values in the population, but gave no information about the variation of genome copy number in individual cells.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>As a first approach to address the ploidy of single cells, confocal laser scanning microscopy (CSLM) was used. <em>H. salinarum</em> cells of the mid-exponential as well as of the stationary growth phase were fixed with formaldehyde and the DNA was stained with the fluorescence dye Hoechst 33342. As a control that the physiological state had been preserved, it was verified that all different intracellular DNA localization patters could be found in the cells from the exponentially growing culture, i.e. distributed DNA, one focus in the middle of the cell, three foci, and two foci at the cell poles <a href="#pone.0000092-Herrmann1">[17]</a>. Furthermore it was verified that the localization patterns that are specific for ongoing replication were absent from cells from stationary phase cultures (data not shown). 50 cells of each culture were chosen randomly and the fluorescence signal was quantitated. The average values were 163 a.u. (standard deviation 75) for cells from the exponential phase and 83 a.u. (s.d. 18) for cells from the stationary growth phase. Thus the growth phase-dependent regulation of the genome copy number could also be verified on the single cell level. The high standard deviation especially in the exponential phase provided a first indication that the genome copy number might be variable in the population. However, the method was not suitable to quantitate the variation of genome copy numbers in the population since it is not exact enough and it is difficult to characterize sufficient cells to address the question in a statistically significant manner.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3"></a><p>As an alternative, we used a “fluorescence activated cell sorter” (FACS) to determine the variation of the DNA content in a high number of cells. Again, the growth phase-dependent regulation of ploidy could be verified on the single cell level (<a href="#pone-0000092-g006">Fig. 6</a>). Thus, the “agarose block method” and the “Real Time PCR” method on the one hand and the CLSM and the FACS analyses on the other hand yield identical results, while they have two important differences: 1) the former methods determine the average content in the population, while the latter methods address single cells, and 2) the former methods quantitate a specific DNA sequence, while the latter determine the overall DNA content with a fluorescent dye. The FACS analysis is the only method that allows to address the variation of DNA content in a large population of cells. <a href="#pone-0000092-g006">Fig. 6</a> shows that the variation is much larger in exponentially growing than in stationary phase cells, the latter being much more uniform in DNA content as well as in size.</p>
<div class="figure" id="pone-0000092-g005"><div class="img"><a name="pone-0000092-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1"></a><p>Determination of genome copy number using the agarose block method. Different aliquots from one culture were used to inoculate several new cultures that were grown overnight. Throughouht the next day aliquots were withdrawn, the cell density was determined with a counting chamber and the ploidy with the agarose block method. For each aliquot, the cell density was plotted against the genome copy number, and a trend line was calculated. B. Determination of the genome copy number using the Real Time PCR method. Three independent cultures were used to determine the genome copy number. A selected growth curve, the average ploidy values and their standard deviation are shown.</p>
<span>doi:10.1371/journal.pone.0000092.g005</span></div><div class="figure" id="pone-0000092-g006"><div class="img"><a name="pone-0000092-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1"></a><p>A. The forward light scatter as a measure of cell size is plotted against the fluorescence of the DNA-specific dye acridine orange as a measure of the DNA content. Aliquots representing early and middle exponential phase and early stationary phase (top to bottom) were analyzed, and the optical densities are indicated. B. The fluorescence as a measure of DNA content is plotted against the fraction of the population exhibiting a specific fluorescence. The three curves from left to right represent the three aliquots shown in A from top to bottom.</p>
<span>doi:10.1371/journal.pone.0000092.g006</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Two very different haloarchaea model species <a href="#pone.0000092-Soppa1">[18]</a> were chosen for the quantitation of the genome copy number. They are dissimilar concerning e.g. cell morphology, optimal salt concentration, motility and taxis, utilizable electron donors and receptors, and the presence of haloarchaeal retinal proteins and gas vesicles. However, both species were found to be highly polyploid and to downregulate the number of genome copies as cells enter stationary phase. This indicates that polyploidy might be widespread in the different genera of halobacteriaceae and might even be more widespread in archaea as previously anticipated. Until now <em>M. jannaschii</em> was the only archaeal species that had been found to contain multiple copies of the chromosome <a href="#pone.0000092-Malandrin1">[15]</a>. However, the number of species that were found to contain one chromosome in the G1 phase and two chromosomes in the G2 phase of the cell cycle is very limited (<em>Sulfolobus, Archaeoglobus, Methanothermobacter</em>) <a href="#pone.0000092-Hjort1">[12]</a>–<a href="#pone.0000092-Majernik1">[14]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The situation is similar for bacteria. It is generally assumed that most bacterial species are monoploid and contain a single copy of a circular chromosome. But also in the bacterial domain of life, the number of genome copies has only be quantitated for a limited number of species. <em>E. coli</em> has been characterized very thoroughly and it is clear that slow-growing cells contain one chromosomal copy while fast growing cells are merooligoploid for origin-proximal regions, e.g. cells growing with at a doubling time of 24 minutes contain, on average, 6.54 origins and 1.94 termini <a href="#pone.0000092-Bremer1">[4]</a>. A small number of other bacteria have also been shown to be monoploid, i.e. <em>Bacillus subtilis</em> and <em>Comamonas testosteroni</em> 
            <a href="#pone.0000092-Webb1">[6]</a>, <a href="#pone.0000092-Hiraoka1">[19]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>We have shown that <em>H. salinarum</em> is not only highly polyploid, but also possesses a strict cell cycle control. Upon addition of the DNA polymerase inhibitor aphidicolin, cells stop dividing immediately <a href="#pone.0000092-Herrmann1">[17]</a>, although the high genome copy number should allow several divisions without the risk of generating anucleate cells. <em>H. volcanii</em> is also highly polyploid, and we have isolated temperature sensitive mutants that stop dividing after a shift to the restrictive temperature (unpublished data). These data indicate that polyploidy is not just the result of the loss of proper genome copy number control, but that it offers selective advantages that led to its development in evolution.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Several bacterial species have been reported to contain multiple copies of the chromosome. Probably the best known example is the radioresistant species <em>Deinococcus radiodurans</em>. It contains at least four genome copies and no monoploid stage could be detected at different growth rates <a href="#pone.0000092-Hansen1">[8]</a>. The human pathogen <em>Neisseria gonorrhoeae</em> contains on average three genome copies, suggesting that nascent <em>Neisseria</em> cells contain two genome copies, which are replicated in a concerted fashion to generate a cell with four genome copies <a href="#pone.0000092-Tobiason1">[20]</a>. As both species are coccal organisms, a linkage between morphology and polyploidy has been postulated <a href="#pone.0000092-Tobiason1">[20]</a>. However, species of other morphologies have also been found to be polyploid. <em>Desulfovibrio gigas</em> contains about nine genome equivalents in ammonia-limited chemostat cultures and about 17 genome copies in fast growing batch cultures <a href="#pone.0000092-Postgate1">[21]</a>. <em>Borrelia hermsii</em>, a spirochete that causes relapsing fever, harbors about 16 genome copies if it is grown in mice but only 1/4 to 1/2 of this number if it is grown axenically in batch culture <a href="#pone.0000092-Kitten1">[22]</a>. The case of <em>Azotobacter vinelandii</em> is controversial. A gene hybridization study concluded that this species harbors as many as 80 genome copies <a href="#pone.0000092-Punita1">[9]</a>, <a href="#pone.0000092-Nagpal1">[23]</a>. In contrast, a genetic approach came to the conclusion that “<em>A. vinelandii</em> is not a polyploid bacterium” <a href="#pone.0000092-Maldonado2">[24]</a>. One line of evidence was that “heterozygotic transconjugants and transformants are unstable and segregate into homozygotic colonies even in the absence of selection”. A subsequent study by the same group used flow cytometry and came to the conclusion that the genetic approach had let to erroneous results (see below) and that <em>A. vinelandii</em> is indeed polyploid. Fast-growing cells in the early exponential phase contained about the same number of chromosomes than <em>E. coli</em>, but the number of genome copies highly increased to as many as 100 in late exponential and in early stationary phase. However, it was also found that in synthetic medium at lower growth rate the genome copy number did not change and the authors concluded that “the polyploidy of <em>A. vinelandii</em> may not exist outside of the laboratory” <a href="#pone.0000092-Maldonado1">[10]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>Taken together, there are more characterized “exceptions” from the rule that bacteria are monoploid than species that have been proven to follow the rule. Polyploidy might be more widespread in prokaryotes than presently anticipated, and it will be interesting to quantitate the genome copy number in more archaeal and bacterial species. No common pattern has emerged from the characterized archaeal and bacterial species that are polypoloid. For example, the range of genome copy numbers is extremely large (from 4 to greater 100), cells may have a higher genome copy number in exponential phase (haloarchaea) or in stationary phase (<em>Azotobacter</em>), and the genome copy number may change with growth rate (<em>Desulfovibrio, E. coli</em>) or is independent from growth rate (<em>Halobacterium</em>).</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>It has been proposed that a selective advantage of polyploidy in prokaryotes could be a higher resistance to DNA damaging conditions, especially those that induce DNA double strand breaks. The radioresistant species <em>D. radiodurans</em> can survive X-ray dosages that lead on average to more than 150 double strand breaks per chromosome <a href="#pone.0000092-Daly1">[25]</a>. However, a study where the genome copy number of <em>D. radiodurans</em> was altered by growth in different media found no correlation between ploidy and resistance to gamma or UV radiation <a href="#pone.0000092-Kitayama1">[26]</a>. <em>H. salinarum</em> is also extremely resistant to X-ray irradiation. The D<sub>10</sub> values (10% survival) are 10 kGy for <em>D. radiodurans</em> and 5 kGy for <em>H. salinarum</em> 
            <a href="#pone.0000092-vanGerwen1">[27]</a>, <a href="#pone.0000092-Kottemann1">[28]</a>. For comparison, the D<sub>10</sub> value of <em>E. coli</em> is 0.25 kGy <a href="#pone.0000092-Clavero1">[29]</a>. However, <em>H. volcanii</em>, which has a similar genome copy number to <em>H. salinarum</em>, is not particularly radioresistant (D<sub>10</sub> = 1 kGy, unpublished data).</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>It is possible that polypolidy has evolved in response to desiccation, which is known to induce DNA double strand breaks. The hypersaline environments of haloarchaea are characterized by high temperatures and a high intensity of sun light, therefore the cells are always in danger of desiccation. <em>H. salinarum</em> was shown to be extremely desiccation resistant; about 25% of the cells remained viable after the exposure to high vacuum (10<sup>−6</sup> Pa) for 20 days <a href="#pone.0000092-Kottemann1">[28]</a>.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>One selective advantage of polyploidy might be that many genome copies allow the accumulation of mutations in different alleles of any gene without losing the wildtype allele. This might be important in situations where growth of the cells is compromised and alleles coding for e.g. enzymes with an altered substrate specificity or affinity might restore the ability to grow.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>While different alleles at the same locus might exist under specific conditions, genetic evidence suggests that this is not the case under normal growth conditions. For example, we have found it easy to isolate haloarchaeal mutants in different metabolic pathways, such as bacteriorhodopsin-mediated phototrophic growth, arginine fermentation, nitrate respiration, and cell cycle progression [30, 31, unpublished data]. Only two rounds of selection with an enrichment factor of about 10<sup>3</sup> are necessary, which argues against random segregation of a mutated gene from 15 to 25 wildtype alleles <a href="#pone.0000092-Soppa2">[30]</a>. A second line of evidence is the ease of replacing wildtype genes of <em>H. salinarum</em> and <em>H. volcanii</em> with deletion mutations, indicating efficient homologous recombination <a href="#pone.0000092-BitanBanin1">[32]</a>–<a href="#pone.0000092-Wang1">[35]</a>. Thirdly, <em>H. volcanii</em> possesses an efficient genetic exchange system. Mutants can be crossed and the segregation of the different chromosomes that had been combined into one cell by cell fusion can be followed by phenotypic markers. After a limited number of generations, the alleles had segregated in the absence of selection, and cells showed only one of the two parental phenotypes <a href="#pone.0000092-Rosenshine1">[36]</a>.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>These results indicates that <em>H. salinarum</em> and<em> H. volcanii</em> possess an efficient gene conversion mechanism that guarantees that all genome copies in growing cells are equalized. It is clear that gene conversion exists in archaea and bacteria, but it was only thought to operate on multiple copies of a gene that are situated on the same chromosome, e.g. ribosomal RNA genes [e.g. 37, 38]. However, it was recently demonstrated that gene conversion is used to eliminate deleterious mutations in the genomes of plastids, which are polyploid <a href="#pone.0000092-Khakhlova1">[39]</a>. It will be interesting to study whether gene conversion also operates to equalize alleles on different chromosomes of polyploid archaea and bacteria.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11"></a><p>An additional selective advantage of polyploidy might be that it allows to globally control gene expression by gene dosage regulation. The degree of regulation is nearly twofold for the main replicon and nearly fivefold for pHS1. Under non-saturating concentrations of regulatory proteins, which is often physiologically relevant, the occupancies of regulator binding sites in the DNA change with altered replicon copy numbers. Depending on the regulatory mechanism this can result in repression or induction of gene expression. In addition, the presence of different regulons with different ploidy and different regulary patterns allows <em>H. salinarum</em> to include different gene dosages in a single cell, that vary by a factor of 5. However, if the gene dosage would influence the expression level of some or many genes, it would also mean that the expression of theses genes is not uniform in exponentially growing <em>H. salinarum</em> cells, since the genome content of single cells was found to be very variable (<a href="#pone-0000092-g006">Fig. 6</a>). Clearly, methods for quantitation of gene expression levels at the single cell level are needed to clarify this question. In stationary phase, the genome content of the cells within the population is much more uniform. However, it depends on the history of the cells, and thus stationary phase cells have a “molecular memory” of the conditions they were exposed to during their growth phase. In another project, the transcript level of a specific gene in stationary phase cells was also found to vary in dependence of their history (unpublished data).</p>

<h4>Coda</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>
               <em>H. salinarum</em> and <em>H. volcanii</em>, representing two very different haloarchaeal genera, were both found to be highly polyploid. The genome copy number is higher than in any other prokaryote, with the exception of fast-growing <em>A. vinelandii</em> under laboratory conditions. Four different methods have shown that the ploidy level is regulated and stationary phase cells habor a smaller number of chromosomes than exponentially growing cells. The copy numbers of different replicons were quantitated, two different species were investigated, the average population as well as single cells have been studied, and different growth conditions and temperatures were used. Thus, this is one of the most thorough reports describing the genome copy number in prokaryotes. The results warrant a closer look at the distribution of polyploidy in archaea and bacteria.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Archaeal and bacterial strains and growth conditions</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>
               <em>H. salinarum</em> was obtained from the German culture collection (<a href="http://www.dsmz.de/">http://www.dsmz.de/</a>; strain No. DSM670). It was grown in complex medium by aerobic respiration or by arginine fermentation as described previously <a href="#pone.0000092-Soppa3">[31]</a>, <a href="#pone.0000092-Ruepp1">[40]</a>. <em>H. volcanii</em> strain H26 was constructed and grown in complex medium as described <a href="#pone.0000092-Allers1">[33]</a>. The <em>H. volcanii</em> strain WR340 was obtained from Moshe Mevarech (Tel Aviv University, Israel) and grown in complex medium <a href="#pone.0000092-Cline1">[41]</a>. <em>Escherichia coli</em> strain Xl1-blue MRF' was obtained from Stratagene (Leiden, Netherlands) and grown in M9 synthetic medium <a href="#pone.0000092-Hanahan1">[42]</a>. <em>E. coli</em> strain MG1655 <a href="#pone.0000092-Bachmann1">[43]</a> was grown in LB broth or 56/2 salts medium with 0.4% glucose or 0.2% glycerol.</p>


<h4>Quantitation of ploidy using isolated genomic DNA</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Two different aproaches were used to isolate genomic DNA from <em>E. coli</em> and <em>H. salinarum</em>. The first approach was to use the DNeasy tissue kit according to the instructions of the manufacturer (Qiagen, Hilden, Germany). The second approach was to precipitate DNA with ethanol after cell lysis and protein digestion. For both species, the cell density was determined using a Neubauer counting chamber. For the isolation of DNA from <em>E. coli</em>, 15 ml culture was harvested by centrifugation (15 min., 4000 g, 4°C). The pellet was suspended in 3.6 ml lysis buffer (10 mM Tris/HCl, pH 7.2, 1 mM EDTA, 100 mM NaCl, 0.05% SDS). 30 µl lysozyme solution (100 mg/ml) and 400 µl proteinase K solution (1 mg/ml) were added and it was incubated for 4 hours at 37°C. The solution was extracted three times with an equal volume of phenol and once with ether. DNA was pelleted by the addition of 1/10 vol. sodium acetate solution (3 M, pH 5.2) and 2.5 vol. ethanol. After centrifugation (20 min., 12 000 g, 4°C), the pellet was solved in TE (10 mM Tris/HCl, pH 7.2, 1 mM EDTA). For the isolation of DNA from <em>H. salinarum</em>, 15 ml of culture was pelleted (see above) and the cells were suspended in 300 µl of basal salt solution (medium without carbon source). 2.5 ml lysis buffer (60 mM EDTA, 0.25% sodium sarconisate, 60 mM Tris/HCl, pH 8) were added and it was mixed until lysis was complete. 400 µl proteinase K solution were added and it was incubated for 4 hours at 37°C. Phenol extraction and ethanol precipitation was performed as described above.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>The DNA concentration was determined photometrically. To calculate the genome equivalents, genome sizes of 4.64 Mbp and 2.57 Mbp were used for <em>E. coli</em> and <em>H. salinarum</em> 
               <a href="#pone.0000092-Blattner1">[44]</a>, <a href="#pone.0000092-Ng1">[45]</a>, that correspond to 5.1 fg/genome and 2.8 fg/genome for the two species.</p>


<h4>Quantitation of ploidy using the “agarose block method”</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>The cell density was determined with a Neubauer counting chamber. Cells from culture aliquots were harvested by centrifugation (15 min., 6000 g, room temperature) and resuspended in 1/10<sup>th</sup> volume of basal salts <a href="#pone.0000092-Soppa3">[31]</a>. 4 ml of cell suspension were quickly mixed with 4 ml of a 2% (w/v) solution of low melting point agarose (agarose type VII with low gelling temperature, Sigma, Steinheim, Germany). A 1 ml aliquot is transferred to an eppendorf tube and the rest is poured into a Petri dish. The weight of the 1 ml aliquot is determined and–together with the known cell density of the suspension–allows to calculate the number of cells per unit weight. After the agarose had solidified in the Petri dish, small ararose blocks of 4×4×4 mm were generated with a self-constructed device. The weight of the agarose blocks were determined and the number of included cells was calculated. The blocks were washed twice for 30 minutes in 50 volumes TE (10 mM Tris/HCl, pH 7.2, 1 mM EDTA). After that, they were incubated overnight at 37°C in 20 volumes lysis buffer (60 mM EDTA, 0.25% (w/v) sodium sarconisate, 100 µg/ml proteinase K, 60 mM Tris/HCl, pH 8.0). They were washed three times for 20 minutes in 10 volumes TE+1 mM PMSF, then they were washed three times for 30 minutes in 10 volumes TE. Subsequently they were transferred invidivually into pcr tubes. The agarose was molten at 80°C and then cooled to 37°C. The desired ratio of internal standard molecules per cell was added. As an internal standard, a pcr fragment was used that was generated with the primers SB_002_ST and SB_003_ST (<a href="#pone-0000092-t001">Table 1</a>) and genomic DNA of <em>H. salinarum</em> as a template. It was purified using a preparative agarose gel and the Quiaex II gel extraction kit (Qiagen, Hilden, Germany). It represents a genomic region close to the presumed replication origin of <em>H. salinarum</em>. 4.5 µl of the restriction enzyme Sal I (10 U/µl) and 6.5 µl of 10× Sal I buffer were added. Sal I does not cleave the internal standard, but it generates a 1 kbp fragment from the genomic DNA that fully includes the sequence of the internal standard. After incubation at 37°C for three hours they samples were shortly heated to 50°C and transferred to a 1.4% (w/v) agarose gel. Gel electrophoresis, transfer to a nylon membrane, hybridization, washing and detection with a digoxigenin-labelled probe were performed as previously described <a href="#pone.0000092-Ruepp2">[46]</a>. The probe was constructed using the internal standard pcr fragment as a template (primers see <a href="#pone-0000092-t001">Table 1</a>). Thus the probe equally recognizes the 900 bp internal standard pcr fragment and the 1 kbp genomic SalI fragment. Hybridization signals were visualized by using an anti-DIG-alkaline phosphatase conjugate and the chemoluminscence substrate CDP-star according to the instructions of the manufacturer (Roche, Mannheim, Germany) and exposing the nylon membranes to an X-ray film. The film was scanned and the signals were quantitated with the software ImageJ 1.32 (NIH, Bethesda, USA). For each band the background was determined using a field of the same surface area directly above and below the band. The average background value was subtracted from the signal of the band. The signals from the internal standards were used to construct a standard curve, which was used to determine the genome copy number using the average signal from the 1 kbp genomic fragment.</p>
<div class="figure" id="pone-0000092-t001"><div class="img"><a name="pone-0000092-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000092" data-uri="info:doi/10.1371/journal.pone.0000092.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000092.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000092.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000092.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000092.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000092.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Primer used for determination of the genome copy numbers</span></strong></p><span>doi:10.1371/journal.pone.0000092.t001</span></div>

<h4>Quantitation of ploidy using the “Real Time PCR method”</h4>
<h5>Overview.</h5><a id="article1.body1.sec4.sec4.sec1.p1" name="article1.body1.sec4.sec4.sec1.p1"></a><p>The rationale of the “Real Time PCR method” is to harvest haloarchaeal cells and lyse them completely by an osmotic shock in low salt solution. Dilutions of the cell lysates are used directly as templates in Real Time PCR assays. Quantitation of the genome copy number is achieved by comparing the results with a dilution series of a pcr product of known concentration that is used as a standard. To ensure that the pcr amplification with the genomic templates has the same efficiency as the amplifications with the pcr product, a third series of Real Time PCR assays was used. A dilution series of the pcr standard was added to the cell lysates and it was verified that the efficiency of standard amplifiction was identical in the absence and presence of cell lysate.</p>

<h5>Real time pcr method.</h5><a id="article1.body1.sec4.sec4.sec2.p1" name="article1.body1.sec4.sec4.sec2.p1"></a><p>
                  <em>H. salinarum</em> or <em>H. volcanii</em> were grown in complex medium as described <a href="#pone.0000092-Soppa3">[31]</a>, <a href="#pone.0000092-Allers1">[33]</a>. It was taken care that they were growing for at least 25 generations in exponential phase before they were used for genome copy number determinations. At the times indicated for each experiment the cell density was determined using a Neubauer counting chamber, and 2 ml aliquot of the cell culture was harvested by centrifugation (5 min., 3800 g, room temperature). The pellet was suspended in 1 ml of basal salt solution <a href="#pone.0000092-Soppa3">[31]</a>, <a href="#pone.0000092-Allers1">[33]</a>. 200 µl of the cell suspension was added to 1.8 ml of a. bid., resulting in rapid cell lysis. Serial dilutions of the cell lysate were generated in a. bid., and 5 µl aliquots were directly used as template for Real Time PCR assays.</p>
<a id="article1.body1.sec4.sec4.sec2.p2" name="article1.body1.sec4.sec4.sec2.p2"></a><p>For the quantitation of the genome copy number by Real Time PCR a standard was needed. A 1 kbp pcr product was generated using genomic DNA of <em>H. salinarum</em> as a template (primers see <a href="#pone-0000092-t001">Table 1</a>). It was purified by preparative agarose gel electrophoresis, eluted with the Quiaex II gel extraction kit (Quiagen, Hilden, Germany), and the concentration was determined photometrically. A series of dilutions were prepared that contained defined numbers of standard molecules, and 5 µl aliquots were used as templates for Real Time PCR. To ensure that pcr efficiencies of the cell lysate dilution series and the standard dilution series were identical, a third series of samples were prepared. A standard dilution series was added to one of the cell lysate samples as an internal control.</p>
<a id="article1.body1.sec4.sec4.sec2.p3" name="article1.body1.sec4.sec4.sec2.p3"></a><p>The Real Time PCR assays contained in a 25 µl reaction volume 5 µl of template (cell lysate, standard, or cell lysate with added standard), 1 µM of primer qPCR_SB_001 and primer qPCR_SB_002, 2× qPCR Master Mix (Finnezymes OY, Espoo, Finnland), and 17.1% (v/v) glycerol. The master mix contained <em>Thermus brockianus</em> DNA polymerase, SYBR GREEN I, dNTPs, MgCl<sub>2</sub>, and buffer (concentrations not released by the manufacturer). The pcr reaction conditions were 10 min. at 96°C, 40 cycles of 30 sec. 96°C, 45 sec. 68°C, 80 sec. 72°C, and an additional incubation of 5 min at 72°C. The Real Time PCRs were performed in the “Rotor Gene 3000” (Corbett Research, Melbourne, Australia). 72 samples could be analyzed simultaneously. At the end of the pcr, the probes were heated to 96°C and the melting point of each sample was determined. Data analysis was performed using the software “Rotor Gene 6.35” (Corbett Research). For each sample the number of cycles was determined until its fluorescence intensity reached a threshold determined by the software (C<sub>t</sub> value). The C<sub>t</sub> values of the standards were used to construct a standard curve (compare <a href="#pone-0000092-g002">Fig. 2C</a>) that was used to quantitate the genome copy numbers in the cell lysates and to check the pcr efficiencies in the lysates including internal standards.</p>

<h5>Optimization of the method.</h5><a id="article1.body1.sec4.sec4.sec3.p1" name="article1.body1.sec4.sec4.sec3.p1"></a><p>As the method was established, different steps were optimized to generate the protocol outlined above. Different cell lysis methods was compared, and it turned out the results were most reproducible if the cells were first harvested and resuspended in basal salts before they were lysed with a. bid. The cell density of the culture and of the cell suspension after centrifugation and resuspension were determined and it was revealed that the loss of cells was about 7% (this was be included in the calculation). The calculated melting points of the primers should be at least 75°C to allow an annealing temperature of 68°C. Different pcr fragment lengths were tested (100 bp, 350 bp, and 500 bp), and the 350 bp fragment turned out to be amplified most reproducibly without any artificial byproducts. The addition of glycerol turned out to be important because it abolished degradation of the genomic DNA during Real Time PCR that was otherwise observed. Real time pcr kits of different suppliers were tested, and the kit “DyNAmo qPCR kit 400L-Fi” from Finnezymes turned out to be the best.</p>



<h4>Fluorescence microscopy</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>500 µl aliquots of cultures of the mid-exponential and the early stationary growth phase were harvested by centrifugation (5 min., 3800 g, room temperature). The pellet was resuspended in 500 µl of fixing buffer (4% (v/v) formaldehyde, 0.1 M sodium cacodylate, 4.2 M NaCl, 220 mM MgCl<sub>2</sub>, 40 mM MgSO<sub>4</sub>, pH 7.0) and incubated for 10 minutes at 42°C. The cells were pelleted again under identical conditions, resuspended in 500 µl of washing solution (20 mM MgCl<sub>2</sub>, 20 mM MgSO<sub>4</sub>, 10 mM Tris/HCl, pH 7.5) and incubated for 10 minutes at 42°C. The dye Hoechst 33342 was added to a final concentration of 15 µg/ml and it was incubated for 20 minutes at 42°C. Subsequently the cell suspension was layered onto agarose-covered microscope slides. Microscopic pictures were obtained using a confocal laser scanning microscope (Leica TCS SP5) and TIFF images were analyzed using the software Imaris 4.1.1 (Leica).</p>


<h4>FACS analysis</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>
               <em>Halobacterium salinarum</em> DSM670 was grown in complex medium as described <a href="#pone.0000092-Soppa3">[31]</a>, and the growth phase was determined by optical density. A sample corresponding to about 1×10<sup>8</sup> cells was harvested by centrifugation (6 minutes, 3800 g, room temperature) and resuspended in 450 µl of basal salt solution. 50 µl of acridine orange solution (Sigma, 0.1 mg/ml in basal salt solution) was added to a final concentration of 10 µg/ml, and samples were analyzed promptly. Sample analysis was performed with an Apogee A40 mini flow cytometer (equipped with a 50 mW 488 nm solid state laser (Coherent). A 510–580 nm bandpass filter was used to detect acridine orange fluorescence due to DNA binding. A PMT voltage of 415 V was used for forward light scatter and 590 V used for emission between 510 and 580 nm (gain = 1). The light scatter signal was used as the trigger parameter. Latex beads of a uniform size and fluorescence were used for calibration and adjustment of the flow cytometer.</p>

</div>



<div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: TA SB GS. Performed the experiments: TA SB GS. Analyzed the data: TA JS SB GS. Wrote the paper: JS. Other: Supervised the theses of Sebastian Breuert and Gabi Spohn: JS.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000092-Wolfe1" id="pone.0000092-Wolfe1"></a>Wolfe KH (2001) Yesterday's polyploids and the mystery of diploidization. Nat Rev Genet  2(5): 333–341.  <ul class="find" data-citedArticleID="920330" data-doi="10.1038/35072009"><li><a href="http://dx.doi.org/10.1038/35072009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Yesterday%27s+polyploids+and+the+mystery+of+diploidization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Yesterday%27s+polyploids+and+the+mystery+of+diploidization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000092-Comai1" id="pone.0000092-Comai1"></a>Comai L (2005) The advantages and disadvantages of being polyploid. Nat Rev Genet  6(11): 836–846.  <ul class="find" data-citedArticleID="920260" data-doi="10.1038/nrg1711"><li><a href="http://dx.doi.org/10.1038/nrg1711" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+advantages+and+disadvantages+of+being+polyploid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+advantages+and+disadvantages+of+being+polyploid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000092-Wendel1" id="pone.0000092-Wendel1"></a>Wendel JF (2000) Genome evolution in polyploids. Plant Mol Biol  42(1): 225–249.  <ul class="find" data-citedArticleID="920328" data-doi="10.1007/978-94-011-4221-2_12"><li><a href="http://dx.doi.org/10.1007/978-94-011-4221-2_12" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+evolution+in+polyploids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+evolution+in+polyploids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000092-Bremer1" id="pone.0000092-Bremer1"></a>Bremer H, Dennis PP (1996)  Modulation of Chemical Composition and Other Parameters of the Cell Growth Rate. In: Neidhardt FC, editor. Escherichia coli and Salmonella. Washington: ASM Press. pp. 1553–1566.  University of Michigan Medical School. <ul class="find-nolinks"></ul></li><li><span class="label">5.
              </span><a name="pone.0000092-Skarstad1" id="pone.0000092-Skarstad1"></a>Skarstad K, Boye E, Steen HB (1986) Timing of initiation of chromosome replication in individual <em>Escherichia coli</em> cells. EMBO J  5(7): 1711–1717.  <ul class="find" data-citedArticleID="920314"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Timing+of+initiation+of+chromosome+replication+in+individual+Escherichia+coli+cells.&amp;auth=&amp;atitle=Timing+of+initiation+of+chromosome+replication+in+individual+Escherichia+coli+cells." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Timing+of+initiation+of+chromosome+replication+in+individual+Escherichia+coli+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Timing+of+initiation+of+chromosome+replication+in+individual+Escherichia+coli+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000092-Webb1" id="pone.0000092-Webb1"></a>Webb C, Graumann P, Kahana J, Telemann A, Silver P, et al.  (1998) Use of time-lapse microscopy to visualize rapid movement of the replication origin region of the chromosome during the cell cycle in <em>Bacillus subtilis</em>. Mol Microbiol  28(5): 883–892.  <ul class="find" data-citedArticleID="920326" data-doi="10.1046/j.1365-2958.1998.00808.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00808.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Use+of+time-lapse+microscopy+to+visualize+rapid+movement+of+the+replication+origin+region+of+the+chromosome+during+the+cell+cycle+in+Bacillus+subtilis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Use+of+time-lapse+microscopy+to+visualize+rapid+movement+of+the+replication+origin+region+of+the+chromosome+during+the+cell+cycle+in+Bacillus+subtilis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000092-Lemon1" id="pone.0000092-Lemon1"></a>Lemon KP, Grossman AD (1998) Localization of bacterial DNA polymerase: evidence for a factory model of replication. Science  282(5393): 1516–1519.  <ul class="find" data-citedArticleID="920282" data-doi="10.1126/science.282.5393.1516"><li><a href="http://dx.doi.org/10.1126/science.282.5393.1516" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Localization+of+bacterial+DNA+polymerase%3A+evidence+for+a+factory+model+of+replication." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Localization+of+bacterial+DNA+polymerase%3A+evidence+for+a+factory+model+of+replication.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000092-Hansen1" id="pone.0000092-Hansen1"></a>Hansen MT (1978) Multiplicity of genome equivalents in the radiation-resistant bacterium <em>Micrococcus radiodurans</em>. J Bacteriol  134(1): 71–75.  <ul class="find" data-citedArticleID="920266"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiplicity+of+genome+equivalents+in+the+radiation-resistant+bacterium+Micrococcus+radiodurans.&amp;auth=&amp;atitle=Multiplicity+of+genome+equivalents+in+the+radiation-resistant+bacterium+Micrococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiplicity+of+genome+equivalents+in+the+radiation-resistant+bacterium+Micrococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiplicity+of+genome+equivalents+in+the+radiation-resistant+bacterium+Micrococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000092-Punita1" id="pone.0000092-Punita1"></a>Punita , Jafri SJ, Reddy MA, Das HK (1989) Multiple chromosomes of <em>Azotobacter vinelandii</em>. J Bacteriol  171(6): 3133–3138.  <ul class="find" data-citedArticleID="920304"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+chromosomes+of+Azotobacter+vinelandii.&amp;auth=&amp;atitle=Multiple+chromosomes+of+Azotobacter+vinelandii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+chromosomes+of+Azotobacter+vinelandii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+chromosomes+of+Azotobacter+vinelandii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000092-Maldonado1" id="pone.0000092-Maldonado1"></a>Maldonado R, Jimenez J, Casadesus J (1994) Changes of ploidy during the <em>Azotobacter vinelandii</em> growth cycle. J Bacteriol  176(13): 3911–3919.  <ul class="find" data-citedArticleID="920292"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Changes+of+ploidy+during+the+Azotobacter+vinelandii+growth+cycle.&amp;auth=&amp;atitle=Changes+of+ploidy+during+the+Azotobacter+vinelandii+growth+cycle." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+of+ploidy+during+the+Azotobacter+vinelandii+growth+cycle." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+of+ploidy+during+the+Azotobacter+vinelandii+growth+cycle.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000092-MaisnierPatin1" id="pone.0000092-MaisnierPatin1"></a>Maisnier-Patin S, Malandrin L, Birkeland NK, Bernander R (2002) Chromosome replication patterns in the hyperthermophilic euryarchaea <em>Archaeoglobus fulgidus</em> and <em>Methanocaldococcus (Methanococcus) jannaschii</em>. Mol Microbiol  45(5): 1443–1450.  <ul class="find" data-citedArticleID="920286" data-doi="10.1046/j.1365-2958.2002.03111.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03111.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Chromosome+replication+patterns+in+the+hyperthermophilic+euryarchaea+Archaeoglobus+fulgidus+and+Methanocaldococcus+%28Methanococcus%29+jannaschii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Chromosome+replication+patterns+in+the+hyperthermophilic+euryarchaea+Archaeoglobus+fulgidus+and+Methanocaldococcus+%28Methanococcus%29+jannaschii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000092-Hjort1" id="pone.0000092-Hjort1"></a>Hjort K, Bernander R (1999) Changes in cell size and DNA content in <em>Sulfolobus</em> cultures during dilution and temperature shift experiments. J Bacteriol  181(18): 5669–5675.  <ul class="find" data-citedArticleID="920272"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Changes+in+cell+size+and+DNA+content+in+Sulfolobus+cultures+during+dilution+and+temperature+shift+experiments.&amp;auth=&amp;atitle=Changes+in+cell+size+and+DNA+content+in+Sulfolobus+cultures+during+dilution+and+temperature+shift+experiments." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Changes+in+cell+size+and+DNA+content+in+Sulfolobus+cultures+during+dilution+and+temperature+shift+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Changes+in+cell+size+and+DNA+content+in+Sulfolobus+cultures+during+dilution+and+temperature+shift+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000092-Bernander1" id="pone.0000092-Bernander1"></a>Bernander R, Poplawski A (1997) Cell cycle characteristics of thermophilic archaea. J Bacteriol  179(16): 4963–4969.  <ul class="find" data-citedArticleID="920246"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cell+cycle+characteristics+of+thermophilic+archaea.&amp;auth=&amp;atitle=Cell+cycle+characteristics+of+thermophilic+archaea." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle+characteristics+of+thermophilic+archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle+characteristics+of+thermophilic+archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000092-Majernik1" id="pone.0000092-Majernik1"></a>Majernik AI, Lundgren M, McDermott P, Bernander R, Chong JP (2005) DNA content and nucleoid distribution in <em>Methanothermobacter thermautotrophicu</em>s. J Bacteriol  187(5): 1856–1858.  <ul class="find" data-citedArticleID="920288" data-doi="10.1128/jb.187.5.1856-1858.2005"><li><a href="http://dx.doi.org/10.1128/jb.187.5.1856-1858.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+content+and+nucleoid+distribution+in+Methanothermobacter+thermautotrophicus." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+content+and+nucleoid+distribution+in+Methanothermobacter+thermautotrophicus.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000092-Malandrin1" id="pone.0000092-Malandrin1"></a>Malandrin L, Huber H, Bernander R (1999) Nucleoid structure and partition in <em>Methanococcus jannaschii</em>: an archaeon with multiple copies of the chromosome. Genetics  152(4): 1315–1323.  <ul class="find" data-citedArticleID="920290"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Nucleoid+structure+and+partition+in+Methanococcus+jannaschii%3A+an+archaeon+with+multiple+copies+of+the+chromosome.&amp;auth=&amp;atitle=Nucleoid+structure+and+partition+in+Methanococcus+jannaschii%3A+an+archaeon+with+multiple+copies+of+the+chromosome." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nucleoid+structure+and+partition+in+Methanococcus+jannaschii%3A+an+archaeon+with+multiple+copies+of+the+chromosome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nucleoid+structure+and+partition+in+Methanococcus+jannaschii%3A+an+archaeon+with+multiple+copies+of+the+chromosome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000092-Chant1" id="pone.0000092-Chant1"></a>Chant J, Hui I, Dejongwong D, Shimmin L, Dennis PP (1986) The protein synthesizing machinery of the Archaebacterium<em> Halobacterium</em>-cutirubrum- molecular characterization. Sys Appl Microbiol  7(1): 106–114.  <ul class="find" data-citedArticleID="920254" data-doi="10.1016/s0723-2020(86)80132-1"><li><a href="http://dx.doi.org/10.1016/s0723-2020(86)80132-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+protein+synthesizing+machinery+of+the+Archaebacterium+Halobacterium-cutirubrum-+molecular+characterization." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+protein+synthesizing+machinery+of+the+Archaebacterium+Halobacterium-cutirubrum-+molecular+characterization.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000092-Herrmann1" id="pone.0000092-Herrmann1"></a>Herrmann U, Soppa J (2002) Cell cycle-dependent expression of an essential SMC-like protein and dynamic chromosome localization in the archaeon <em>Halobacterium salinarum</em>. Mol Microbiol  46(2): 395–409.  <ul class="find" data-citedArticleID="920268" data-doi="10.1046/j.1365-2958.2002.03181.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03181.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cell+cycle-dependent+expression+of+an+essential+SMC-like+protein+and+dynamic+chromosome+localization+in+the+archaeon+Halobacterium+salinarum." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cell+cycle-dependent+expression+of+an+essential+SMC-like+protein+and+dynamic+chromosome+localization+in+the+archaeon+Halobacterium+salinarum.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000092-Soppa1" id="pone.0000092-Soppa1"></a>Soppa J (2006) From genomes to function: haloarchaea as model organisms. Microbiology  152: Pt 3585–590.  <ul class="find" data-citedArticleID="920316" data-doi="10.1099/mic.0.28504-0"><li><a href="http://dx.doi.org/10.1099/mic.0.28504-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=From+genomes+to+function%3A+haloarchaea+as+model+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22From+genomes+to+function%3A+haloarchaea+as+model+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000092-Hiraoka1" id="pone.0000092-Hiraoka1"></a>Hiraoka Y, Yamada T, Tone K, Futaesaku Y, Kimbara K (2002) Flow cytometry analysis of changes in the DNA content of the polychlorinated biphenyl degrader <em>Comamonas testosteroni</em> TK102: effect of metabolites on cell-cell separation. Appl Environ Microbiol  68(10): 5104–5112.  <ul class="find" data-citedArticleID="920270" data-doi="10.1128/aem.68.10.5104-5112.2002"><li><a href="http://dx.doi.org/10.1128/aem.68.10.5104-5112.2002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Flow+cytometry+analysis+of+changes+in+the+DNA+content+of+the+polychlorinated+biphenyl+degrader+Comamonas+testosteroni+TK102%3A+effect+of+metabolites+on+cell-cell+separation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Flow+cytometry+analysis+of+changes+in+the+DNA+content+of+the+polychlorinated+biphenyl+degrader+Comamonas+testosteroni+TK102%3A+effect+of+metabolites+on+cell-cell+separation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000092-Tobiason1" id="pone.0000092-Tobiason1"></a>Tobiason DM, Seifert HS (2006) The Obligate Human Pathogen,<em>Neisseria gonorrhoeae</em>, Is Polyploid. PLoS Biol  4(6): e185.  <ul class="find" data-citedArticleID="920322" data-doi="10.1371/journal.pbio.0040185"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0040185" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Obligate+Human+Pathogen%2CNeisseria+gonorrhoeae%2C+Is+Polyploid." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Obligate+Human+Pathogen%2CNeisseria+gonorrhoeae%2C+Is+Polyploid.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000092-Postgate1" id="pone.0000092-Postgate1"></a>Postgate JR, Kent HM, Robson RL, Chesshyre JA (1984) The genomes of <em>Desulfovibrio gigas</em> and <em>D. vulgaris</em>. J Gen Microbiol  130(7): 1597–1601.  <ul class="find" data-citedArticleID="920302" data-doi="10.1099/00221287-130-7-1597"><li><a href="http://dx.doi.org/10.1099/00221287-130-7-1597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomes+of+Desulfovibrio+gigas+and+D.+vulgaris." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomes+of+Desulfovibrio+gigas+and+D.+vulgaris.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000092-Kitten1" id="pone.0000092-Kitten1"></a>Kitten T, Barbour AG (1992) The relapsing fever agent <em>Borrelia hermsii</em> has multiple copies of its chromosome and linear plasmids. Genetics  132(2): 311–324.  <ul class="find" data-citedArticleID="920278"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+relapsing+fever+agent+Borrelia+hermsii+has+multiple+copies+of+its+chromosome+and+linear+plasmids.&amp;auth=&amp;atitle=The+relapsing+fever+agent+Borrelia+hermsii+has+multiple+copies+of+its+chromosome+and+linear+plasmids." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+relapsing+fever+agent+Borrelia+hermsii+has+multiple+copies+of+its+chromosome+and+linear+plasmids." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+relapsing+fever+agent+Borrelia+hermsii+has+multiple+copies+of+its+chromosome+and+linear+plasmids.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000092-Nagpal1" id="pone.0000092-Nagpal1"></a>Nagpal P, Jafri S, Reddy MA, Das HK (1989) Multiple chromosomes of <em>Azotobacter vinelandii</em>. J Bacteriol  171(6): 3133–3138.  <ul class="find" data-citedArticleID="920296"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Multiple+chromosomes+of+Azotobacter+vinelandii.&amp;auth=&amp;atitle=Multiple+chromosomes+of+Azotobacter+vinelandii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Multiple+chromosomes+of+Azotobacter+vinelandii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Multiple+chromosomes+of+Azotobacter+vinelandii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000092-Maldonado2" id="pone.0000092-Maldonado2"></a>Maldonado R, Garzon A, Dean DR, Casadesus J (1992) Gene dosage analysis in <em>Azotobacter vinelandii</em>. Genetics  132(4): 869–878.  <ul class="find" data-citedArticleID="920294"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+dosage+analysis+in+Azotobacter+vinelandii.&amp;auth=&amp;atitle=Gene+dosage+analysis+in+Azotobacter+vinelandii." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+dosage+analysis+in+Azotobacter+vinelandii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+dosage+analysis+in+Azotobacter+vinelandii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000092-Daly1" id="pone.0000092-Daly1"></a>Daly MJ, Ouyang L, Fuchs P, Minton KW (1994) 
                  <em>In vivo</em> damage and recA-dependent repair of plasmid and chromosomal DNA in the radiation-resistant bacterium <em>Deinococcus radiodurans</em>. J Bacteriol  176(12): 3508–3517.  <ul class="find" data-citedArticleID="920262"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans.&amp;auth=&amp;atitle=In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In+vivo+damage+and+recA-dependent+repair+of+plasmid+and+chromosomal+DNA+in+the+radiation-resistant+bacterium+Deinococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000092-Kitayama1" id="pone.0000092-Kitayama1"></a>Kitayama HarsojoS, Matsuyama A (1981) Genome multiplicity and radiation resistance in <em>Micrococcus radiodurans</em>. J Biochem (Tokyo)  90: 877–80.  <ul class="find" data-citedArticleID="920276"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome+multiplicity+and+radiation+resistance+in+Micrococcus+radiodurans.&amp;auth=&amp;atitle=Genome+multiplicity+and+radiation+resistance+in+Micrococcus+radiodurans." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+multiplicity+and+radiation+resistance+in+Micrococcus+radiodurans." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+multiplicity+and+radiation+resistance+in+Micrococcus+radiodurans.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000092-vanGerwen1" id="pone.0000092-vanGerwen1"></a>van Gerwen SJ, Rombouts FM, van't Riet K, Zwietering MH (1999) A data analysis of the irradiation parameter D10 for bacteria and spores under various conditions. J Food Prot  62(9): 1024–1032.  <ul class="find" data-citedArticleID="920332"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+data+analysis+of+the+irradiation+parameter+D10+for+bacteria+and+spores+under+various+conditions.&amp;auth=&amp;atitle=A+data+analysis+of+the+irradiation+parameter+D10+for+bacteria+and+spores+under+various+conditions." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+data+analysis+of+the+irradiation+parameter+D10+for+bacteria+and+spores+under+various+conditions." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+data+analysis+of+the+irradiation+parameter+D10+for+bacteria+and+spores+under+various+conditions.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000092-Kottemann1" id="pone.0000092-Kottemann1"></a>Kottemann M, Kish A, Iloanusi C, Bjork S, DiRuggiero J (2005) Physiological responses of the halophilic archaeon <em>Halobacterium</em> sp. strain NRC1 to desiccation and gamma irradiation. Extremophiles  9(3): 219–227.  <ul class="find" data-citedArticleID="920280" data-doi="10.1007/s00792-005-0437-4"><li><a href="http://dx.doi.org/10.1007/s00792-005-0437-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Physiological+responses+of+the+halophilic+archaeon+Halobacterium+sp.+strain+NRC1+to+desiccation+and+gamma+irradiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Physiological+responses+of+the+halophilic+archaeon+Halobacterium+sp.+strain+NRC1+to+desiccation+and+gamma+irradiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000092-Clavero1" id="pone.0000092-Clavero1"></a>Clavero MRS, Monk JD, Beuchat LR, Doyle MP, Brackett RE (1994) Inactivation of <em>Escherichia coli</em> 0157:H7, <em>Salmonellae</em>, and <em>Camphylobacter jejuni</em> in raw ground beef by gamma irradiation. Appl Environ Microbiol  60(6): 2069–2075.  <ul class="find" data-citedArticleID="920256"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inactivation+of+Escherichia+coli+0157%3AH7%2C+Salmonellae%2C+and+Camphylobacter+jejuni+in+raw+ground+beef+by+gamma+irradiation.&amp;auth=&amp;atitle=Inactivation+of+Escherichia+coli+0157%3AH7%2C+Salmonellae%2C+and+Camphylobacter+jejuni+in+raw+ground+beef+by+gamma+irradiation." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inactivation+of+Escherichia+coli+0157%3AH7%2C+Salmonellae%2C+and+Camphylobacter+jejuni+in+raw+ground+beef+by+gamma+irradiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inactivation+of+Escherichia+coli+0157%3AH7%2C+Salmonellae%2C+and+Camphylobacter+jejuni+in+raw+ground+beef+by+gamma+irradiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000092-Soppa2" id="pone.0000092-Soppa2"></a>Soppa J (1998) Optimization of the 5-bromo-2′-deoxyuridine selection and its application for the isolation of nitrate respiration-deficient mutants of <em>Haloferax volcanii</em>. J Microbiol Methods  34: 41–48.  <ul class="find" data-citedArticleID="920318" data-doi="10.1016/s0167-7012(98)00067-0"><li><a href="http://dx.doi.org/10.1016/s0167-7012(98)00067-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Optimization+of+the+5-bromo-2%26%23x2032%3B-deoxyuridine+selection+and+its+application+for+the+isolation+of+nitrate+respiration-deficient+mutants+of+Haloferax+volcanii." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Optimization+of+the+5-bromo-2%26%23x2032%3B-deoxyuridine+selection+and+its+application+for+the+isolation+of+nitrate+respiration-deficient+mutants+of+Haloferax+volcanii.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000092-Soppa3" id="pone.0000092-Soppa3"></a>Soppa J, Otomo J, Straub J, Tittor J, Meessen S, et al.  (1989) Bacteriorhodopsin mutants of <em>Halobacterium sp. GRB</em>. II. Characterization of mutants. J Biol Chem  264(22): 13049–13056.  <ul class="find" data-citedArticleID="920320"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Bacteriorhodopsin+mutants+of+Halobacterium+sp.+GRB.+II.+Characterization+of+mutants.&amp;auth=&amp;atitle=Bacteriorhodopsin+mutants+of+Halobacterium+sp.+GRB.+II.+Characterization+of+mutants." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Bacteriorhodopsin+mutants+of+Halobacterium+sp.+GRB.+II.+Characterization+of+mutants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Bacteriorhodopsin+mutants+of+Halobacterium+sp.+GRB.+II.+Characterization+of+mutants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000092-BitanBanin1" id="pone.0000092-BitanBanin1"></a>Bitan-Banin G, Ortenberg R, Mevarech M (2003) Development of a gene knockout system for the halophilic archaeon <em>Haloferax volcanii</em> by use of the pyrE gene. J Bacteriol  185(3): 772–778.  <ul class="find" data-citedArticleID="920248" data-doi="10.1128/jb.185.3.772-778.2003"><li><a href="http://dx.doi.org/10.1128/jb.185.3.772-778.2003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+a+gene+knockout+system+for+the+halophilic+archaeon+Haloferax+volcanii+by+use+of+the+pyrE+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+a+gene+knockout+system+for+the+halophilic+archaeon+Haloferax+volcanii+by+use+of+the+pyrE+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000092-Allers1" id="pone.0000092-Allers1"></a>Allers T, Ngo HP, Mevarech M, Lloyd RG (2004) Development of additional selectable markers for the halophilic archaeon <em>Haloferax volcanii</em> based on the leuB and trpA genes. Appl Environ Microbiol  70(2): 943–953.  <ul class="find" data-citedArticleID="920242" data-doi="10.1128/aem.70.2.943-953.2004"><li><a href="http://dx.doi.org/10.1128/aem.70.2.943-953.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Development+of+additional+selectable+markers+for+the+halophilic+archaeon+Haloferax+volcanii+based+on+the+leuB+and+trpA+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Development+of+additional+selectable+markers+for+the+halophilic+archaeon+Haloferax+volcanii+based+on+the+leuB+and+trpA+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000092-Peck1" id="pone.0000092-Peck1"></a>Peck R, Dassarma S, Krebs M (2000) Homologous gene knockout in the archaeon <em>Halobacterium salinarum</em> with <em>ura3</em> as a counterselectable marker. Mol Microbiol  35(3): 667–676.  <ul class="find" data-citedArticleID="920300" data-doi="10.1046/j.1365-2958.2000.01739.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.01739.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Homologous+gene+knockout+in+the+archaeon+Halobacterium+salinarum+with+ura3+as+a+counterselectable+marker." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Homologous+gene+knockout+in+the+archaeon+Halobacterium+salinarum+with+ura3+as+a+counterselectable+marker.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000092-Wang1" id="pone.0000092-Wang1"></a>Wang G, Kennedy SP, Fasiludeen S, Rensing C, DasSarma S (2004) Arsenic resistance in <em>Halobacterium sp</em>. strain NRC-1 examined by using an improved gene knockout system. J Bacteriol  186(10): 3187–3194.  <ul class="find" data-citedArticleID="920324" data-doi="10.1128/jb.186.10.3187-3194.2004"><li><a href="http://dx.doi.org/10.1128/jb.186.10.3187-3194.2004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Arsenic+resistance+in+Halobacterium+sp.+strain+NRC-1+examined+by+using+an+improved+gene+knockout+system." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Arsenic+resistance+in+Halobacterium+sp.+strain+NRC-1+examined+by+using+an+improved+gene+knockout+system.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000092-Rosenshine1" id="pone.0000092-Rosenshine1"></a>Rosenshine I, Mevarech M (1991) The kinetic of the genetic exchange process. in: General Appl Aspects of Halophilic Microorgananisms.  Rodriguez-Valera , editor. Plenum Press. New York: pp. 265–270.  <ul class="find-nolinks"></ul></li><li><span class="label">37.
              </span><a name="pone.0000092-Liao1" id="pone.0000092-Liao1"></a>Liao D (2000) Gene conversion drives within genic sequences: concerted evolution of ribosomal RNA genes in bacteria and archaea. J Mol Evol  51(4): 305–317.  <ul class="find" data-citedArticleID="920284"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Gene+conversion+drives+within+genic+sequences%3A+concerted+evolution+of+ribosomal+RNA+genes+in+bacteria+and+archaea.&amp;auth=&amp;atitle=Gene+conversion+drives+within+genic+sequences%3A+concerted+evolution+of+ribosomal+RNA+genes+in+bacteria+and+archaea." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+conversion+drives+within+genic+sequences%3A+concerted+evolution+of+ribosomal+RNA+genes+in+bacteria+and+archaea." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+conversion+drives+within+genic+sequences%3A+concerted+evolution+of+ribosomal+RNA+genes+in+bacteria+and+archaea.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000092-Santoyo1" id="pone.0000092-Santoyo1"></a>Santoyo G, Romero D (2005) Gene conversion and concerted evolution in bacterial genomes. FEMS Microbiol Rev  29(2): 169–183.  <ul class="find" data-citedArticleID="920312" data-doi="10.1016/j.fmrre.2004.10.004"><li><a href="http://dx.doi.org/10.1016/j.fmrre.2004.10.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+conversion+and+concerted+evolution+in+bacterial+genomes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+conversion+and+concerted+evolution+in+bacterial+genomes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000092-Khakhlova1" id="pone.0000092-Khakhlova1"></a>Khakhlova O, Bock R (2006) Elimination of deleterious mutations in plastid genomes by gene conversion. Plant J.  46: 85–94.  <ul class="find" data-citedArticleID="920274" data-doi="10.1111/j.1365-313x.2006.02673.x"><li><a href="http://dx.doi.org/10.1111/j.1365-313x.2006.02673.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Elimination+of+deleterious+mutations+in+plastid+genomes+by+gene+conversion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Elimination+of+deleterious+mutations+in+plastid+genomes+by+gene+conversion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000092-Ruepp1" id="pone.0000092-Ruepp1"></a>Ruepp A, Muller HN, Lottspeich F, Soppa J (1995) Catabolic ornithine transcarbamylase of <em>Halobacterium halobium</em> (<em>salinarium</em>): purification, characterization, sequence determination, and evolution. J Bacteriol  177(5): 1129–1136.  <ul class="find" data-citedArticleID="920308"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Catabolic+ornithine+transcarbamylase+of+Halobacterium+halobium+%28salinarium%29%3A+purification%2C+characterization%2C+sequence+determination%2C+and+evolution.&amp;auth=&amp;atitle=Catabolic+ornithine+transcarbamylase+of+Halobacterium+halobium+%28salinarium%29%3A+purification%2C+characterization%2C+sequence+determination%2C+and+evolution." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Catabolic+ornithine+transcarbamylase+of+Halobacterium+halobium+%28salinarium%29%3A+purification%2C+characterization%2C+sequence+determination%2C+and+evolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Catabolic+ornithine+transcarbamylase+of+Halobacterium+halobium+%28salinarium%29%3A+purification%2C+characterization%2C+sequence+determination%2C+and+evolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000092-Cline1" id="pone.0000092-Cline1"></a>Cline SW, Schalkwyk LC, Doolittle WF (1989) Transformation of the archaebacterium <em>Halobacterium volcanii</em> with genomic DNA. J Bacteriol  171(9): 4987–4991.  <ul class="find" data-citedArticleID="920258"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Transformation+of+the+archaebacterium+Halobacterium+volcanii+with+genomic+DNA.&amp;auth=&amp;atitle=Transformation+of+the+archaebacterium+Halobacterium+volcanii+with+genomic+DNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transformation+of+the+archaebacterium+Halobacterium+volcanii+with+genomic+DNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transformation+of+the+archaebacterium+Halobacterium+volcanii+with+genomic+DNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000092-Hanahan1" id="pone.0000092-Hanahan1"></a>Hanahan D (1985)  Techniques for transformation of <em>E.coli</em>. In: Glover DMIP, editor. DNA cloning - A practical approach. Oxford. Washington D.C: pp. 109–135.  <ul class="find-nolinks"></ul></li><li><span class="label">43.
              </span><a name="pone.0000092-Bachmann1" id="pone.0000092-Bachmann1"></a>Bachmann BJ (1996) Derivations and genotypes of some mutant derivatives of <em>Escherichia coli</em> K-12, in <em>Escherichia coli</em> and <em>Salmonella</em> Cellular and Molecular Biology,.  Neidhardt FC, et al., editor. ASM Press. Washington, D.C: pp. 2460–2488.  (1996).  <ul class="find-nolinks"></ul></li><li><span class="label">44.
              </span><a name="pone.0000092-Blattner1" id="pone.0000092-Blattner1"></a>Blattner FR, Plunkett G 3rd, Bloch CA, Perna NT, Burland V, et al.  (1997) The complete genome sequence of<em> Escherichia coli</em> K-12. Science  277(5331): 1453–1474.  <ul class="find" data-citedArticleID="920250" data-doi="10.1126/science.277.5331.1453"><li><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+complete+genome+sequence+of+Escherichia+coli+K-12." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+complete+genome+sequence+of+Escherichia+coli+K-12.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000092-Ng1" id="pone.0000092-Ng1"></a>Ng WV, Kennedy SP, Mahairas GG, Berquist B, Pan M, et al.  (2000) Genome sequence of <em>Halobacterium</em> spec. NRC-1. Proc Natl Acad Sci U S A  97(22): 12176–12181.  <ul class="find" data-citedArticleID="920298" data-doi="10.1073/pnas.190337797"><li><a href="http://dx.doi.org/10.1073/pnas.190337797" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome+sequence+of+Halobacterium+spec.+NRC-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome+sequence+of+Halobacterium+spec.+NRC-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000092-Ruepp2" id="pone.0000092-Ruepp2"></a>Ruepp A, Soppa J (1996) Fermentative arginine degradation in <em>Halobacterium salinarium (halobium</em>): genes, gene products, and transcripts of the <em>arcRACB</em> gene cluster. J Bacteriol  178: 4942–4947.  <ul class="find" data-citedArticleID="920310"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster.&amp;auth=&amp;atitle=Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fermentative+arginine+degradation+in+Halobacterium+salinarium+%28halobium%29%3A+genes%2C+gene+products%2C+and+transcripts+of+the+arcRACB+gene+cluster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.XML" value="131428"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.PDF" value="345775"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g001.PNG_L" value="1145314"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g001.PNG_M" value="73090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g001.PNG_S" value="3730"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g001.TIF" value="1330494"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g001.PNG_I" value="26624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g002.PNG_L" value="1916719"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g002.PNG_M" value="69604"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g002.PNG_S" value="3478"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g002.TIF" value="2002452"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g002.PNG_I" value="25841"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g003.PNG_L" value="454438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g003.PNG_M" value="64578"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g003.PNG_S" value="2864"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g003.TIF" value="643502"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g003.PNG_I" value="21844"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g004.PNG_L" value="370244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g004.PNG_M" value="45125"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g004.PNG_S" value="2284"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g004.TIF" value="378074"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g004.PNG_I" value="15977"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g005.PNG_L" value="414184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g005.PNG_M" value="34487"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g005.PNG_S" value="2695"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g005.TIF" value="483684"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g005.PNG_I" value="18380"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g006.PNG_L" value="1152801"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g006.PNG_M" value="28772"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g006.PNG_S" value="6373"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g006.TIF" value="1208082"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.g006.PNG_I" value="87203"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.t001.PNG_L" value="220342"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.t001.PNG_M" value="34759"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.t001.PNG_S" value="2414"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.t001.TIF" value="8749068"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000092.t001.PNG_I" value="14122"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000092&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000092" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000092&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000092&volume=&issue=&title=Regulated Polyploidy in Halophilic Archaea&author_name=Sebastian%20Breuert%2C%20Thorsten%20Allers%2C%20Gabi%20Spohn%2C%20J%C3%B6rg%20Soppa&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000092" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092&amp;t=Regulated%20Polyploidy%20in%20Halophilic%20Archaea" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092&title=Regulated%20Polyploidy%20in%20Halophilic%20Archaea&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092&amp;title=Regulated%20Polyploidy%20in%20Halophilic%20Archaea" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000092&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Regulated Polyploidy in Halophilic Archaea';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000092';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Regulated%20Polyploidy%20in%20Halophilic%20Archaea http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000092" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000092" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Bacterial+genomics%22" title="Search for articles in the subject area:'Bacterial genomics'"><div class="flagText">Bacterial genomics</div></a>
              <div data-categoryid="32801" data-articleid="23022"
                   data-categoryname="Bacterial genomics"
                   class="flagImage" title="Flag 'Bacterial genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Chromosomes%22" title="Search for articles in the subject area:'Chromosomes'"><div class="flagText">Chromosomes</div></a>
              <div data-categoryid="17251" data-articleid="23022"
                   data-categoryname="Chromosomes"
                   class="flagImage" title="Flag 'Chromosomes' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+evolution%22" title="Search for articles in the subject area:'Genome evolution'"><div class="flagText">Genome evolution</div></a>
              <div data-categoryid="17207" data-articleid="23022"
                   data-categoryname="Genome evolution"
                   class="flagImage" title="Flag 'Genome evolution' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="23022"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Mammalian+genomics%22" title="Search for articles in the subject area:'Mammalian genomics'"><div class="flagText">Mammalian genomics</div></a>
              <div data-categoryid="17625" data-articleid="23022"
                   data-categoryname="Mammalian genomics"
                   class="flagImage" title="Flag 'Mammalian genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Ploidy%22" title="Search for articles in the subject area:'Ploidy'"><div class="flagText">Ploidy</div></a>
              <div data-categoryid="36025" data-articleid="23022"
                   data-categoryname="Ploidy"
                   class="flagImage" title="Flag 'Ploidy' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polymerase+chain+reaction%22" title="Search for articles in the subject area:'Polymerase chain reaction'"><div class="flagText">Polymerase chain reaction</div></a>
              <div data-categoryid="47289" data-articleid="23022"
                   data-categoryname="Polymerase chain reaction"
                   class="flagImage" title="Flag 'Polymerase chain reaction' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Polyploidy%22" title="Search for articles in the subject area:'Polyploidy'"><div class="flagText">Polyploidy</div></a>
              <div data-categoryid="46213" data-articleid="23022"
                   data-categoryname="Polyploidy"
                   class="flagImage" title="Flag 'Polyploidy' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=5124'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=4056'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=226&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=9469">More detail on radiation resistance of H. volcanii</a><br>Posted by jonathan_eisen</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
