

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001479"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001479" />

    <meta name="citation_title" content="Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes"/>
    <meta itemprop="name" content="Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes"/>

      <meta name="citation_author" content="Ashok Hegde"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Nick Chuanxin Qiu"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Xuehui Qiu"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Steven Hao-Kee Ho"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Kenny Qi-Ye Tay"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Joshy George"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Felicia Soo Lee Ng"/>
            <meta name="citation_author_institution" content="Bioinformatics Institute, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Kunde Ramamoorthy Govindarajan"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Zhiyuan Gong"/>
            <meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, Singapore, Singapore"/>
      <meta name="citation_author" content="Sinnakaruppan Mathavan"/>
            <meta name="citation_author_institution" content="Genome Institute of Singapore, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
      <meta name="citation_author" content="Yun-Jin Jiang"/>
            <meta name="citation_author_institution" content="Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore"/>
            <meta name="citation_author_institution" content="Department of Biochemistry, National University of Singapore, Singapore, Singapore"/>
            <meta name="citation_author_institution" content="School of Biological Sciences, Nanyang Technological University, Singapore, Singapore"/>

    <meta name="citation_date" content="2008/1/23"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001479.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1479"/>
    <meta name="citation_issue" content="1"/>
    <meta name="citation_volume" content="3"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Notch signaling: cell fate control and signal integration in development.; citation_author=S Artavanis-Tsakonas; citation_author=MD Rand; citation_author=RJ Lake; citation_journal_title=Science; citation_volume=284; citation_number=1; citation_pages=770-776; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The segmentation clock: converting embryonic time into spatial pattern.; citation_author=O Pourquié; citation_journal_title=Science; citation_volume=301; citation_number=2; citation_pages=328-330; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=TAN-1, the human homolog of the Drosophila Notch gene, is broken by chromosomal translocations in T lymphoblastic neoplasms.; citation_author=LW Ellisen; citation_author=J Bird; citation_author=DC West; citation_author=AL Soreng; citation_author=TC Reynolds; citation_journal_title=Cell; citation_volume=66; citation_number=3; citation_pages=649-661; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=A novel Notch ligand, Dll4, induces T-cell leukemia/lymphoma when overexpressed in mice by retroviral-mediated gene transfer.; citation_author=XQ Yan; citation_author=U Sarmiento; citation_author=Y Sun; citation_author=G Huang; citation_author=J Guo; citation_journal_title=Blood; citation_volume=98; citation_number=4; citation_pages=3793-3799; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=The SCFFBW7 ubiquitin ligase complex as a tumor suppressor in T cell leukemia.; citation_author=BJ Thompson; citation_author=S Buonamici; citation_author=ML Sulis; citation_author=T Palomero; citation_author=T Vilimas; citation_journal_title=J Exp Med; citation_volume=204; citation_number=5; citation_pages=1825-1835; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Mutations in the human Jagged1 gene are responsible for Alagille syndrome.; citation_author=T Oda; citation_author=AG Elkahloun; citation_author=BL Pike; citation_author=K Okajima; citation_author=ID Krantz; citation_journal_title=Nat Genet; citation_volume=16; citation_number=6; citation_pages=235-242; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Alagille syndrome is caused by mutations in human Jagged1, which encodes a ligand for Notch1.; citation_author=L Li; citation_author=ID Krantz; citation_author=Y Deng; citation_author=A Genin; citation_author=AB Banta; citation_journal_title=Nat Genet; citation_volume=16; citation_number=7; citation_pages=243-251; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=NOTCH2 mutations cause Alagille syndrome, a heterogeneous disorder of the notch signaling pathway.; citation_author=R McDaniell; citation_author=DM Warthen; citation_author=PA Sanchez-Lara; citation_author=A Pai; citation_author=ID Krantz; citation_journal_title=Am J Hum Genet; citation_volume=79; citation_number=8; citation_pages=169-173; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Notch3 mutations in CADASIL, a hereditary adult-onset condition causing stroke and dementia.; citation_author=A Joutel; citation_author=C Corpechot; citation_author=A Ducros; citation_author=K Vahedi; citation_author=H Chabriat; citation_journal_title=Nature; citation_volume=383; citation_number=9; citation_pages=707-710; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mutations in the human Delta homologue, DLL3, cause axial skeletal defects in spondylocostal dysostosis.; citation_author=MP Bulman; citation_author=K Kusumi; citation_author=TM Frayling; citation_author=C McKeown; citation_author=C Garrett; citation_journal_title=Nat Genet; citation_volume=24; citation_number=10; citation_pages=438-441; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=A novel proteolytic cleavage involved in Notch signaling: the role of the disintegrin-metalloprotease TACE.; citation_author=C Brou; citation_author=F Logeat; citation_author=N Gupta; citation_author=C Bessia; citation_author=O LeBail; citation_journal_title=Mol Cell; citation_volume=5; citation_number=11; citation_pages=207-216; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Notch and presenilin: a proteolytic mechanism emerges.; citation_author=ME Fortini; citation_journal_title=Curr Opin Cell Biol; citation_volume=13; citation_number=12; citation_pages=627-634; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=A common enzyme connects notch signaling and Alzheimer's disease.; citation_author=R Kopan; citation_author=A Goate; citation_journal_title=Genes Dev; citation_volume=14; citation_number=13; citation_pages=2799-2806; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Ligand-induced cleavage and regulation of nuclear entry of Notch in Drosophila melanogaster embryos.; citation_author=S Kidd; citation_author=T Lieber; citation_author=MW Young; citation_journal_title=Genes Dev; citation_volume=12; citation_number=14; citation_pages=3728-3740; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Signal transduction by activated mNotch: importance of proteolytic processing and its regulation by the extracellular domain.; citation_author=R Kopan; citation_author=EH Schroeter; citation_author=H Weintraub; citation_author=JS Nye; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=15; citation_pages=1683-1688; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Indirect evidence for Delta-dependent intracellular processing of Notch in Drosophila embryos.; citation_author=M Lecourtois; citation_author=F Schweisguth; citation_journal_title=Curr Biol; citation_volume=8; citation_number=16; citation_pages=771-774; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Nuclear access and action of Notch in vivo.; citation_author=G Struhl; citation_author=A Adachi; citation_journal_title=Cell; citation_volume=93; citation_number=17; citation_pages=649-660; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Signalling downstream of activated mammalian Notch.; citation_author=S Jarriault; citation_author=C Brou; citation_author=F Logeat; citation_author=EH Schroeter; citation_author=R Kopan; citation_journal_title=Nature; citation_volume=377; citation_number=18; citation_pages=355-358; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Constitutively active human Notch1 binds to the transcription factor CBF1 and stimulates transcription through a promoter containing a CBF1-responsive element.; citation_author=FM Lu; citation_author=SE Lux; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=93; citation_number=19; citation_pages=5663-5667; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Notch signaling: from the outside in.; citation_author=JS Mumm; citation_author=R Kopan; citation_journal_title=Dev Biol; citation_volume=228; citation_number=20; citation_pages=151-165; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Suppressor of hairless directly activates transcription of enhancer of split complex genes in response to Notch receptor activity.; citation_author=AM Bailey; citation_author=JW Posakony; citation_journal_title=Genes Dev; citation_volume=9; citation_number=21; citation_pages=2609-2622; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Hes7: a bHLH-type repressor gene regulated by Notch and expressed in the presomitic mesoderm.; citation_author=Y Bessho; citation_author=G Miyoshi; citation_author=R Sakata; citation_author=R Kageyama; citation_journal_title=Genes Cells; citation_volume=6; citation_number=22; citation_pages=175-185; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Oscillatory expression of the bHLH factor Hes1 regulated by a negative feedback loop.; citation_author=H Hirata; citation_author=S Yoshiura; citation_author=T Ohtsuka; citation_author=Y Bessho; citation_author=T Harada; citation_journal_title=Science; citation_volume=298; citation_number=23; citation_pages=840-843; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=her1 and the notch pathway function within the oscillator mechanism that regulates zebrafish somitogenesis.; citation_author=SA Holley; citation_author=D Jülich; citation_author=GJ Rauch; citation_author=R Geisler; citation_author=C Nüsslein-Volhard; citation_journal_title=Development; citation_volume=129; citation_number=24; citation_pages=1175-1183; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hairy/E(spl)-related (Her) genes are central components of the segmentation oscillator and display redundancy with the Delta/Notch signaling pathway in the formation of anterior segmental boundaries in the zebrafish.; citation_author=AC Oates; citation_author=RK Ho; citation_journal_title=Development; citation_volume=129; citation_number=25; citation_pages=2929-2946; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Hes1 and Hes5 as Notch effectors in mammalian neuronal differentiation.; citation_author=T Ohtsuka; citation_author=M Ishibashi; citation_author=G Gradwohl; citation_author=S Nakanishi; citation_author=F Guillemot; citation_journal_title=EMBO J; citation_volume=18; citation_number=26; citation_pages=2196-2207; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Mind bomb is a ubiquitin ligase that is essential for efficient activation of Notch signaling by Delta.; citation_author=M Itoh; citation_author=CH Kim; citation_author=G Palardy; citation_author=T Oda; citation_author=Y-J Jiang; citation_journal_title=Dev Cell; citation_volume=4; citation_number=27; citation_pages=67-82; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Protein degradation: four E3s for the Notch pathway.; citation_author=EC Lai; citation_journal_title=Curr Biol; citation_volume=12; citation_number=28; citation_pages=R74-R78; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mutations affecting neurogenesis and brain morphology in the zebrafish, Danio rerio.; citation_author=Y-J Jiang; citation_author=M Brand; citation_author=C-P Heisenberg; citation_author=D Beuchle; citation_author=M Furutani-Seiki; citation_journal_title=Development; citation_volume=123; citation_number=29; citation_pages=205-216; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Mutations affecting somite formation and patterning in the zebrafish Danio rerio.; citation_author=FJM van Eeden; citation_author=M Granato; citation_author=U Schach; citation_author=M Brand; citation_author=M Furutani-Seiki; citation_journal_title=Development; citation_volume=123; citation_number=30; citation_pages=153-164; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Control of her1 expression during zebrafish somitogenesis by a Delta-dependent oscillator and an independent wave-front activity.; citation_author=SA Holley; citation_author=R Geisler; citation_author=C Nüsslein-Volhard; citation_journal_title=Genes Dev; citation_volume=14; citation_number=31; citation_pages=1678-1690; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=beamter/deltaC and the role of Notch ligands in the zebrafish somite segmentation, hindbrain neurogenesis and hypochord differentiation.; citation_author=D Jülich; citation_author=CH Lim; citation_author=J Round; citation_author=C Nicolaije; citation_author=J Schroeder; citation_journal_title=Dev Biol; citation_volume=286; citation_number=32; citation_pages=391-404; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A Notch feeling of somite segmentation and beyond.; citation_author=PCG Rida; citation_author=N Le Minh; citation_author=Y-J Jiang; citation_journal_title=Dev Biol; citation_volume=265; citation_number=33; citation_pages=2-22; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The characterization of zebrafish antimorphic mib alleles reveals that Mib and Mind bomb-2 (Mib2) function redundantly.; citation_author=C Zhang; citation_author=Q Li; citation_author=C-H Lim; citation_author=X Qiu; citation_author=Y-J Jiang; citation_journal_title=Dev Biol; citation_volume=305; citation_number=34; citation_pages=14-27; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Zebrafish Mib and Mib2 are mutual E3 ubiquitin ligases with common and specific Delta substrates.; citation_author=C Zhang; citation_author=Q Li; citation_author=Y-J Jiang; citation_journal_title=J Mol Biol; citation_volume=366; citation_number=35; citation_pages=1115-1128; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Notch signalling controls pancreatic cell differentiation.; citation_author=Å Apelqvist; citation_author=H Li; citation_author=L Sommer; citation_author=P Beatus; citation_author=DJ Anderson; citation_journal_title=Nature; citation_volume=400; citation_number=36; citation_pages=877-881; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Notch inhibits Ptf1 function and acinar cell differentiation in developing mouse and zebrafish pancreas.; citation_author=F Esni; citation_author=B Ghosh; citation_author=AV Biankin; citation_author=JW Lin; citation_author=MA Albert; citation_journal_title=Development; citation_volume=131; citation_number=37; citation_pages=4213-4224; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Evolutionary conserved role of ptf1a in the specification of exocrine pancreatic fates.; citation_author=E Zecchin; citation_author=A Mavropoulos; citation_author=N Devos; citation_author=A Filippi; citation_author=N Tiso; citation_journal_title=Dev Biol; citation_volume=268; citation_number=38; citation_pages=174-184; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Exocrine pancreas development in zebrafish.; citation_author=NS Yee; citation_author=K Lorent; citation_author=M Pack; citation_journal_title=Dev Biol; citation_volume=284; citation_number=39; citation_pages=84-101; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Distinct delta and jagged genes control sequential segregation of pancreatic cell types from precursor pools in zebrafish.; citation_author=E Zecchin; citation_author=A Filippi; citation_author=F Biemar; citation_author=N Tiso; citation_author=S Pauls; citation_journal_title=Dev Biol; citation_volume=301; citation_number=40; citation_pages=192-204; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Control of endodermal endocrine development by Hes-1.; citation_author=J Jensen; citation_author=EE Pedersen; citation_author=P Galante; citation_author=J Hald; citation_author=RS Heller; citation_journal_title=Nat Genet; citation_volume=24; citation_number=41; citation_pages=36-44; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Direct evidence for the pancreatic lineage: NGN3+ cells are islet progenitors and are distinct from duct progenitors.; citation_author=G Gu; citation_author=J Dubauskaite; citation_author=DA Melton; citation_journal_title=Development; citation_volume=129; citation_number=42; citation_pages=2447-2457; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=neurogenin3 is required for the development of the four endocrine cell lineages of the pancreas.; citation_author=G Gradwohl; citation_author=A Dierich; citation_author=M LeMeur; citation_author=F Guillemot; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=97; citation_number=43; citation_pages=1607-1611; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Requirement for LIM homeobox gene Isl1 in motor neuron generation reveals a motor neuron-dependent step in interneuron differentiation.; citation_author=SL Pfaff; citation_author=M Mendelsohn; citation_author=CL Stewart; citation_author=T Edlund; citation_author=TM Jessell; citation_journal_title=Cell; citation_volume=84; citation_number=44; citation_pages=309-320; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Identification of beta-cell-specific insulin gene transcription factor RIPE3b1 as mammalian MafA.; citation_author=M Olbrot; citation_author=J Rud; citation_author=LG Moss; citation_author=A Sharma; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=99; citation_number=45; citation_pages=6737-6742; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The Pax4 gene is essential for differentiation of insulin-producing beta cells in the mammalian pancreas.; citation_author=B Sosa-Pineda; citation_author=K Chowdhury; citation_author=M Torres; citation_author=G Oliver; citation_author=P Gruss; citation_journal_title=Nature; citation_volume=386; citation_number=46; citation_pages=399-402; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Pax6 is required for differentiation of glucagon-producing alpha-cells in mouse pancreas.; citation_author=L St-Onge; citation_author=B Sosa-Pineda; citation_author=K Chowdhury; citation_author=A Mansouri; citation_author=P Gruss; citation_journal_title=Nature; citation_volume=387; citation_number=47; citation_pages=406-409; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Microarrays: biotechnology's discovery platform for functional genomics.; citation_author=M Schena; citation_author=RA Heller; citation_author=TP Theriault; citation_author=K Konrad; citation_author=E Lachenmeier; citation_journal_title=Trends Biotechnol; citation_volume=16; citation_number=48; citation_pages=301-306; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Single-cell transcript analysis of pancreas development.; citation_author=MK Chiang; citation_author=DA Melton; citation_journal_title=Dev Cell; citation_volume=4; citation_number=49; citation_pages=383-393; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Global expression analysis of gene regulatory pathways during endocrine pancreatic development.; citation_author=G Gu; citation_author=JM Wells; citation_author=D Dombkowski; citation_author=F Preffer; citation_author=B Aronow; citation_journal_title=Development; citation_volume=131; citation_number=50; citation_pages=165-179; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=IA1 is NGN3-dependent and essential for differentiation of the endocrine pancreas.; citation_author=G Mellitzer; citation_author=S Bonne; citation_author=RF Luco; citation_author=M Van De Casteele; citation_author=N Lenne-Samuel; citation_journal_title=EMBO J; citation_volume=25; citation_number=51; citation_pages=1344-1352; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptome analysis of zebrafish embryogenesis using microarrays.; citation_author=S Mathavan; citation_author=SG Lee; citation_author=A Mak; citation_author=LD Miller; citation_author=KR Murthy; citation_journal_title=PLoS Genet; citation_volume=1; citation_number=52; citation_pages=260-276; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genomewide expression profiling in the zebrafish embryo identifies target genes regulated by Hedgehog signaling during vertebrate development.; citation_author=J Xu; citation_author=BP Srinivas; citation_author=SY Tay; citation_author=A Mak; citation_author=X Yu; citation_journal_title=Genetics; citation_volume=174; citation_number=53; citation_pages=735-752; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Rhombomere boundaries are Wnt signaling centers that regulate metameric patterning in the zebrafish hindbrain.; citation_author=BB Riley; citation_author=MY Chiang; citation_author=EM Storch; citation_author=R Heck; citation_author=GR Buckles; citation_journal_title=Dev Dyn; citation_volume=231; citation_number=54; citation_pages=278-291; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Wnt1 regulates neurogenesis and mediates lateral inhibition of boundary cell specification in the zebrafish hindbrain.; citation_author=M Amoyel; citation_author=YC Cheng; citation_author=Y-J Jiang; citation_author=DG Wilkinson; citation_journal_title=Development; citation_volume=132; citation_number=55; citation_pages=775-785; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Delta-Notch signaling regulates oligodendrocyte specification.; citation_author=HC Park; citation_author=B Appel; citation_journal_title=Development; citation_volume=130; citation_number=56; citation_pages=3747-3755; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=her4, a zebrafish homologue of the Drosophila neurogenic gene E(spl), is a target of NOTCH signalling.; citation_author=C Takke; citation_author=P Dornseifer; citation_author=E v Weizsäcker; citation_author=JA Campos-Ortega; citation_journal_title=Development; citation_volume=126; citation_number=57; citation_pages=1811-1821; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Patterning of proneuronal and inter-proneuronal domains by hairy- and enhancer of split-related genes in zebrafish neuroectoderm.; citation_author=YK Bae; citation_author=T Shimizu; citation_author=M Hibi; citation_journal_title=Development; citation_volume=132; citation_number=58; citation_pages=1375-1385; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Completing the set of h/E(spl) cyclic genes in zebrafish: her12 and her15 reveal novel modes of expression and contribute to the segmentation clock.; citation_author=SS Shankaran; citation_author=D Sieger; citation_author=C Schroter; citation_author=C Czepe; citation_author=MC Pauly; citation_journal_title=Dev Biol; citation_volume=304; citation_number=59; citation_pages=615-632; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Regulation of neuronal specification in the zebrafish spinal cord by Delta function.; citation_author=B Appel; citation_author=JS Eisen; citation_journal_title=Development; citation_volume=125; citation_number=60; citation_pages=371-380; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Novel transcript nort is a downstream target gene of the Notch signaling pathway in zebrafish.; citation_author=M Tsutsumi; citation_author=M Itoh; citation_journal_title=Gene Expr Patterns; citation_volume=7; citation_number=61; citation_pages=227-232; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Integrinα5 and Delta/Notch signaling have complementary spatiotemporal requirements during zebrafish somitogenesis.; citation_author=D Jülich; citation_author=R Geisler; citation_author=SA Holley; citation_journal_title=Dev Cell; citation_volume=8; citation_number=62; citation_pages=575-586; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Off limits–integrins holding boundaries in somitogenesis.; citation_author=S-W Chong; citation_author=Y-J Jiang; citation_journal_title=Trends Cell Biol; citation_volume=15; citation_number=63; citation_pages=453-457; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Extrinsic and intrinsic factors governing cell fate in cortical progenitor cultures.; citation_author=DK Irvin; citation_author=A Dhaka; citation_author=C Hicks; citation_author=G Weinmaster; citation_author=HI Kornblum; citation_journal_title=Dev Neurosci; citation_volume=25; citation_number=64; citation_pages=162-172; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Visualization of spatiotemporal activation of Notch signaling: live monitoring and significance in neural development.; citation_author=J Kohyama; citation_author=A Tokunaga; citation_author=Y Fujita; citation_author=H Miyoshi; citation_author=T Nagai; citation_journal_title=Dev Biol; citation_volume=286; citation_number=65; citation_pages=311-325; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gefiltin in zebrafish embryos: sequential gene expression of two neurofilament proteins in retinal ganglion cells.; citation_author=D Leake; citation_author=WS Asch; citation_author=AK Canger; citation_author=N Schechter; citation_journal_title=Differentiation; citation_volume=65; citation_number=66; citation_pages=181-189; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=ICA69 is expressed equally in the human endocrine and exocrine pancreas.; citation_author=MI Mally; citation_author=V Cirulli; citation_author=A Hayek; citation_author=T Otonkoski; citation_journal_title=Diabetologia; citation_volume=39; citation_number=67; citation_pages=474-480; citation_date=1996; " />
      <meta name="citation_reference" content="citation_title=Cellular distribution and ontogeny of insulin-like growth factors (IGFs) and IGF binding protein messenger RNAs and peptides in developing rat pancreas.; citation_author=DJ Hill; citation_author=J Hogg; citation_author=J Petrik; citation_author=E Arany; citation_author=VK Han; citation_journal_title=J Endocrinol; citation_volume=160; citation_number=68; citation_pages=305-317; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Ontogeny of glutamic acid decarboxylase gene expression in the mouse pancreas.; citation_author=JM Pleau; citation_author=M Throsby; citation_author=A Esling; citation_author=M Dardenne; citation_journal_title=Biochem Biophys Res Commun; citation_volume=233; citation_number=69; citation_pages=227-230; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Insulin-like growth factors and pancreas beta cells.; citation_author=TW van Haeften; citation_author=TB Twickler; citation_journal_title=Eur J Clin Invest; citation_volume=34; citation_number=70; citation_pages=249-255; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Gene expression patterns in pancreatic tumors, cells and tissues.; citation_author=AW Lowe; citation_author=M Olsen; citation_author=Y Hao; citation_author=SP Lee; citation_author=K Taek Lee; citation_journal_title=PLoS ONE; citation_volume=2; citation_number=71; citation_pages=e323; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Developmental regulation of islet-1 mRNA expression during neuronal differentiation in embryonic zebrafish.; citation_author=A Inoue; citation_author=M Takahashi; citation_author=K Hatta; citation_author=Y Hotta; citation_author=H Okamoto; citation_journal_title=Dev Dyn; citation_volume=199; citation_number=72; citation_pages=1-11; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Islet1 and Islet2 have equivalent abilities to promote motoneuron formation and to specify motoneuron subtype identity.; citation_author=SA Hutchinson; citation_author=JS Eisen; citation_journal_title=Development; citation_volume=133; citation_number=73; citation_pages=2137-2147; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Hedgehog signalling controls zebrafish neural keel morphogenesis via its level-dependent effects on neurogenesis.; citation_author=M Takamiya; citation_author=JA Campos-Ortega; citation_journal_title=Dev Dyn; citation_volume=235; citation_number=74; citation_pages=978-997; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=High throughput expression analysis of ZF-models consortium clones.; citation_author=C Thisse; citation_author=B Thisse; citation_journal_title=ZFIN Direct Data Submission; citation_number=75; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=her3, a zebrafish member of the hairy-E(spl) family, is repressed by Notch signalling.; citation_author=S Hans; citation_author=N Scheer; citation_author=I Riedl; citation_author=E v Weizsäcker; citation_author=P Blader; citation_journal_title=Development; citation_volume=131; citation_number=76; citation_pages=2957-2969; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Wnt5 signaling in vertebrate pancreas development.; citation_author=HJ Kim; citation_author=JR Schleiffarth; citation_author=J Jessurun; citation_author=S Sumanas; citation_author=A Petryk; citation_journal_title=BMC Biol; citation_volume=3; citation_number=77; citation_pages=23; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Analysis of the zebrafish perplexed mutation reveals tissue-specific roles for de novo pyrimidine synthesis during development.; citation_author=GB Willer; citation_author=VM Lee; citation_author=RG Gregg; citation_author=BA Link; citation_journal_title=Genetics; citation_volume=170; citation_number=78; citation_pages=1827-1837; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression of the zebrafish genome during embryogenesis (NIH R01 RR15402).; citation_author=B Thisse; citation_author=S Pflumio; citation_author=M Fürthauer; citation_author=B Loppin; citation_author=V Heyer; citation_journal_title=ZFIN Direct Data Submission; citation_number=79; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Large scale real-time PCR validation on gene expression measurements from two commercial long-oligonucleotide microarrays.; citation_author=Y Wang; citation_author=C Barbacioru; citation_author=F Hyland; citation_author=W Xiao; citation_author=KL Hunkapiller; citation_journal_title=BMC Genomics; citation_volume=7; citation_number=80; citation_pages=59; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Transcriptome kinetics of arsenic-induced adaptive response in zebrafish liver.; citation_author=SH Lam; citation_author=CL Winata; citation_author=Y Tong; citation_author=S Korzh; citation_author=WS Lim; citation_journal_title=Physiol Genomics; citation_volume=27; citation_number=81; citation_pages=351-361; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Independent development of pancreatic alpha- and beta-cells from neurogenin3-expressing precursors: a role for the Notch pathway in repression of premature differentiation.; citation_author=J Jensen; citation_author=RS Heller; citation_author=T Funder-Nielsen; citation_author=EE Pedersen; citation_author=C Lindsell; citation_journal_title=Diabetes; citation_volume=49; citation_number=82; citation_pages=163-176; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Minireview: transcriptional regulation in pancreatic development.; citation_author=JF Habener; citation_author=DM Kemp; citation_author=MK Thomas; citation_journal_title=Endocrinology; citation_volume=146; citation_number=83; citation_pages=1025-1034; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Gene regulatory factors in pancreatic development.; citation_author=J Jensen; citation_journal_title=Dev Dyn; citation_volume=229; citation_number=84; citation_pages=176-200; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A death-associated protein kinase (DAPK)-interacting protein, DIP-1, is an E3 ubiquitin ligase that promotes tumor necrosis factor-induced apoptosis and regulates the cellular levels of DAPK.; citation_author=Y Jin; citation_author=EK Blue; citation_author=S Dixon; citation_author=Z Shao; citation_author=PJ Gallagher; citation_journal_title=J Biol Chem; citation_volume=277; citation_number=85; citation_pages=46980-46986; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Jagged2a-Notch signaling mediates cell fate choice in zebrafish pronephric duct.; citation_author=M Ma; citation_author=Y-J Jiang; citation_journal_title=PLoS Genet; citation_volume=3; citation_number=86; citation_pages=e18; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Snx5, as a Mind bomb-binding protein, is expressed in hematopoietic and endothelial precursor cells in zebrafish.; citation_author=KW Yoo; citation_author=EH Kim; citation_author=SH Jung; citation_author=M Rhee; citation_author=BK Koo; citation_journal_title=FEBS Lett; citation_volume=580; citation_number=87; citation_pages=4409-4416; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Recognition and ubiquitination of Notch by Itch, a hect-type E3 ubiquitin ligase.; citation_author=L Qiu; citation_author=C Joazeiro; citation_author=N Fang; citation_author=HY Wang; citation_author=C Elly; citation_journal_title=J Biol Chem; citation_volume=275; citation_number=88; citation_pages=35734-35737; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Itch genetically interacts with Notch1 in a mouse autoimmune disease model.; citation_author=LE Matesic; citation_author=DC Haines; citation_author=NG Copeland; citation_author=NA Jenkins; citation_journal_title=Hum Mol Genet; citation_volume=15; citation_number=89; citation_pages=3485-3497; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The E3 ubiquitin ligase Itch couples JNK activation to TNFα-induced cell death by inducing c-FLIPL turnover.; citation_author=L Chang; citation_author=H Kamata; citation_author=G Solinas; citation_author=JL Luo; citation_author=S Maeda; citation_journal_title=Cell; citation_volume=124; citation_number=90; citation_pages=601-613; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Discontinuous organization and specification of the lateral floor plate in zebrafish.; citation_author=M Schäfer; citation_author=D Kinzel; citation_author=C Winkler; citation_journal_title=Dev Biol; citation_volume=301; citation_number=91; citation_pages=117-129; citation_date=2007; " />
      <meta name="citation_reference" content="citation_title=Sequence and embryonic expression of three zebrafish fringe genes, lunatic fringe, radical fringe, and manic fringe.; citation_author=X Qiu; citation_author=H Xu; citation_author=C Haddon; citation_author=J Lewis; citation_author=Y-J Jiang; citation_journal_title=Dev Dyn; citation_volume=231; citation_number=92; citation_pages=621-630; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Analyses of pancreas development by generation of gfp transgenic zebrafish using an exocrine pancreas-specific elastaseA gene promoter.; citation_author=H Wan; citation_author=S Korzh; citation_author=Z Li; citation_author=SP Mudumana; citation_author=V Korzh; citation_journal_title=Exp Cell Res; citation_volume=312; citation_number=93; citation_pages=1526-1539; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Pdx-1 knockdown reduces insulin promoter activity in zebrafish.; citation_author=H Huang; citation_author=N Liu; citation_author=S Lin; citation_journal_title=Genesis; citation_volume=30; citation_number=94; citation_pages=134-136; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=DNA arrays for analysis of gene expression.; citation_author=MB Eisen; citation_author=PO Brown; citation_journal_title=Methods Enzymol; citation_volume=303; citation_number=95; citation_pages=179-205; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=A segmental nearest neighbor normalization and gene identification method gives superior results for DNA-array analysis.; citation_author=H Yang; citation_author=H Haddad; citation_author=C Tomas; citation_author=K Alsaker; citation_author=ET Papoutsakis; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=100; citation_number=96; citation_pages=1122-1127; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response.; citation_author=VG Tusher; citation_author=R Tibshirani; citation_author=G Chu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=97; citation_pages=5116-5121; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns.; citation_author=MB Eisen; citation_author=PT Spellman; citation_author=PO Brown; citation_author=D Botstein; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=98; citation_pages=14863-14868; citation_date=1998; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes"/>
    <meta name="twitter:description" content="BackgroundNotch signaling is an evolutionarily conserved developmental pathway. Zebrafish mind bomb (mib) mutants carry mutations on mib gene, which encodes a RING E3 ligase required for Notch activation via Delta/Jagged ubiquitylation and internalization.Methodology/Principal FindingsWe examined the mib mutants for defects in pancreas development using in situ hybridization and GFP expression analysis of pancreas-specific GFP lines, carried out the global gene expression profile analysis of three different mib mutant alleles and validated the microarray data using real-time PCR and fluorescent double in situ hybridization. Our study showed that the mib mutants have diminished exocrine pancreas and this defect was most severe in mibta52b followed by mibm132 and then mibtfi91, which is consistent with the compromised Notch activity found in corresponding mib mutant alleles. Global expression profile analysis of mib mutants showed that there is a significant difference in gene expression profile of wt and three mib mutant alleles. There are 91 differentially expressed genes that are common to all three mib alleles. Through detailed analysis of microarray data, we have identified several previously characterized genes and some putative Notch-responsive genes involved in pancreas development. Moreover, results from real-time PCR and fluorescent double in situ hybridization were largely consistent with microarray data.Conclusions/SignificanceThis study provides, for the first time, a global gene expression profile in mib mutants generating useful genomic resources and providing an opportunity to identify the function of novel genes involved in Notch signaling and Notch-regulated developmental processes."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001479.g007"/>

  <meta property="og:title" content="Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=1749'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6674'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=9065&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001479">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001479" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001479&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Genomewide Expression Analysis in Zebrafish <i>mind bomb</i> Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Ashok Hegde, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Nick Chuanxin Qiu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nick Chuanxin Qiu, Xuehui Qiu, Steven Hao-Kee Ho, Kenny Qi-Ye Tay</p>

              
              

                <p>Affiliation:
                  Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuehui Qiu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nick Chuanxin Qiu, Xuehui Qiu, Steven Hao-Kee Ho, Kenny Qi-Ye Tay</p>

              
              

                <p>Affiliation:
                  Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Steven Hao-Kee Ho
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nick Chuanxin Qiu, Xuehui Qiu, Steven Hao-Kee Ho, Kenny Qi-Ye Tay</p>

              
              

                <p>Affiliation:
                  Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kenny Qi-Ye Tay
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Nick Chuanxin Qiu, Xuehui Qiu, Steven Hao-Kee Ho, Kenny Qi-Ye Tay</p>

              
              

                <p>Affiliation:
                  Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Joshy George, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Felicia Soo Lee Ng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioinformatics Institute, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kunde Ramamoorthy Govindarajan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhiyuan Gong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Biological Sciences, National University of Singapore, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Sinnakaruppan Mathavan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Genome Institute of Singapore, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Yun-Jin Jiang
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:yjjiang@imcb.a-star.edu.sg">yjjiang@imcb.a-star.edu.sg</a></p>

                <p>Affiliations:
                  Laboratory of Developmental Signalling and Patterning, Institute of Molecular and Cell Biology, A*STAR (Agency for Science, Technology and Research), Singapore, Singapore, 
                  Department of Biochemistry, National University of Singapore, Singapore, Singapore, 
                  School of Biological Sciences, Nanyang Technological University, Singapore, Singapore
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: January 23, 2008</li>
    <li>DOI: 10.1371/journal.pone.0001479</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001479-g001" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-g002" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-g003" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-g004" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-t001" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-g005" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-g006" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-t002" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-t003" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-t004" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-g007" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001479-t005" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001479">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001479" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1"></a><p>Notch signaling is an evolutionarily conserved developmental pathway. Zebrafish <em>mind bomb</em> (<em>mib</em>) mutants carry mutations on <em>mib</em> gene, which encodes a RING E3 ligase required for Notch activation via Delta/Jagged ubiquitylation and internalization.</p>


<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1"></a><p>We examined the <em>mib</em> mutants for defects in pancreas development using <em>in situ</em> hybridization and GFP expression analysis of pancreas-specific GFP lines, carried out the global gene expression profile analysis of three different <em>mib</em> mutant alleles and validated the microarray data using real-time PCR and fluorescent double <em>in situ</em> hybridization. Our study showed that the <em>mib</em> mutants have diminished exocrine pancreas and this defect was most severe in <em>mib<sup>ta52b</sup></em> followed by <em>mib<sup>m132</sup></em> and then <em>mib<sup>tfi91</sup></em>, which is consistent with the compromised Notch activity found in corresponding <em>mib</em> mutant alleles. Global expression profile analysis of <em>mib</em> mutants showed that there is a significant difference in gene expression profile of wt and three <em>mib</em> mutant alleles. There are 91 differentially expressed genes that are common to all three <em>mib</em> alleles. Through detailed analysis of microarray data, we have identified several previously characterized genes and some putative Notch-responsive genes involved in pancreas development. Moreover, results from real-time PCR and fluorescent double <em>in situ</em> hybridization were largely consistent with microarray data.</p>


<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1"></a><p>This study provides, for the first time, a global gene expression profile in <em>mib</em> mutants generating useful genomic resources and providing an opportunity to identify the function of novel genes involved in Notch signaling and Notch-regulated developmental processes.</p>

</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hegde A, Qiu NC, Qiu X, Ho SH-K, Tay KQ-Y, et al.  (2008) Genomewide Expression Analysis in Zebrafish <em>mind bomb</em> Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes. PLoS ONE 3(1):
          e1479.
            doi:10.1371/journal.pone.0001479</p><p><strong>Academic Editor: </strong>Geraldine Butler, University College Dublin, Ireland</p><p><strong>Received:</strong> July 26, 2007; <strong>Accepted:</strong> December 11, 2007; <strong>Published:</strong> January 23, 2008</p><p><strong>Copyright:</strong> © 2008 Hegde et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a joint grant from the Biomedical Research Council of Singapore and Juvenile Diabetes Research Foundation International (BMRC-JDRF, 04/1/50/22/295) and the Biomedical Research Council of A*STAR (Agency for Science, Technology and Research), Singapore.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The Notch pathway is an evolutionarily conserved signal transduction cascade that plays essential roles in a variety of developmental processes, such as pattern formation, cell fate determination and organ formation through local cell-cell interactions (reviewed in <a href="#pone.0001479-ArtavanisTsakonas1">[1]</a>–<a href="#pone.0001479-Pourqui1">[2]</a>). Apart from being important for normal development, Notch signaling is also related to several human congenital diseases, such as T-cell acute lymphoblastic leukemia/lymphoma <a href="#pone.0001479-Ellisen1">[3]</a>–<a href="#pone.0001479-Thompson1">[5]</a>, Alagille syndrome <a href="#pone.0001479-Oda1">[6]</a>–<a href="#pone.0001479-McDaniell1">[8]</a>, a late onset neurological disease (CADASIL) <a href="#pone.0001479-Joutel1">[9]</a>, and spondylocostal dysostosis <a href="#pone.0001479-Bulman1">[10]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Notch receptor functions as a membrane-bound transcription factor that turns on specific genes in response to physiological cues that trigger ligand binding. Upon functional binding of DSL (<span class="underline">D</span>elta, <span class="underline">S</span>errate/Jagged and <span class="underline">L</span>ag-2) transmembrane ligands, the membrane-bound Notch is proteolyzed by TNFα-converting enzyme (TACE), metalloproteases <a href="#pone.0001479-Brou1">[11]</a> and Presenilin, and the active form–Notch intracellular domain (NICD)–is released <a href="#pone.0001479-Fortini1">[12]</a>, <a href="#pone.0001479-Kopan1">[13]</a>. The NICD is then translocated to the nucleus <a href="#pone.0001479-Kidd1">[14]</a>–<a href="#pone.0001479-Struhl1">[17]</a> and binds to the conserved CSL (<span class="underline">C</span>BF1/RBPjκ, <span class="underline">S</span>u(H) and <span class="underline">L</span>ag-1) DNA-binding protein <a href="#pone.0001479-Jarriault1">[18]</a>, <a href="#pone.0001479-Lu1">[19]</a>, which is converted from a transcriptional repressor to activator. This triggers the expression of downstream target genes such as Hes/Her (<span class="underline">h</span>airy/<span class="underline">E</span>nhancer of <span class="underline">s</span>plit <span class="underline">r</span>elated) family of bHLH transcription factors, which in turn modulate the expression of downstream genes and themselves <a href="#pone.0001479-Lecourtois1">[16]</a>, <a href="#pone.0001479-Mumm1">[20]</a>–<a href="#pone.0001479-Ohtsuka1">[26]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Several E3 ligases, such as Mind bomb (Mib), Su(dx), Sel-10, Neuralized, and Deltex, have been shown to modulate the Notch signaling through ubiquitin-dependent protein degradation and/or endocytosis <a href="#pone.0001479-Itoh1">[27]</a>, <a href="#pone.0001479-Lai1">[28]</a>. Mib is a ubiquitin ligase that is required cell non-autonomously for Notch signaling and lateral inhibition by controlling Delta protein internalization <a href="#pone.0001479-Itoh1">[27]</a>. Several mutants of zebrafish Notch signaling components, such as <em>after eight (aei)/deltaD</em>, <em>deadly seven (des)/notch1a</em>, <em>beamter (bea)/deltaC</em>, and <em>mind bomb</em> (<em>mib</em>) have been isolated in a large scale screen <a href="#pone.0001479-Holley1">[24]</a>, <a href="#pone.0001479-Itoh1">[27]</a>, <a href="#pone.0001479-Jiang1">[29]</a>–<a href="#pone.0001479-Jlich1">[32]</a>. In all these mutants, the anterior somites are formed normally but the posterior somites are irregularly formed <a href="#pone.0001479-Jiang1">[29]</a>, <a href="#pone.0001479-vanEeden1">[30]</a>, <a href="#pone.0001479-Rida1">[33]</a>. The zebrafish <em>mib2</em> ortholog was recently cloned <a href="#pone.0001479-Zhang1">[34]</a>. The zebrafish Mib and Mib2 have common and specific Delta substrates <a href="#pone.0001479-Zhang2">[35]</a> and function redundantly <a href="#pone.0001479-Zhang1">[34]</a>. Three different alleles of <em>mib</em>, viz., <em>mib<sup>ta52b</sup></em>, <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> with descending severity of Notch-dependent phenotypes have been characterized in our laboratory <a href="#pone.0001479-Zhang1">[34]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>Genetic studies in mice <a href="#pone.0001479-Apelqvist1">[36]</a> and zebrafish <a href="#pone.0001479-Esni1">[37]</a>–<a href="#pone.0001479-Zecchin2">[40]</a> have shown that Notch signaling is involved in pancreas development. Mice deficient in Notch signaling components, Dll1, RBP-jκ and Hes1, have an increase of endocrine pancreatic cells and depletion of progenitor cells <a href="#pone.0001479-Apelqvist1">[36]</a>, <a href="#pone.0001479-Jensen1">[41]</a>. Roles of some of the Notch-related transcription factors, such as Ngn3 <a href="#pone.0001479-Gu1">[42]</a>, <a href="#pone.0001479-Gradwohl1">[43]</a>, Isl1 <a href="#pone.0001479-Pfaff1">[44]</a>, MafA <a href="#pone.0001479-Olbrot1">[45]</a>, Pax4 and Pax6 <a href="#pone.0001479-SosaPineda1">[46]</a>, <a href="#pone.0001479-StOnge1">[47]</a>, in pancreas development have been demonstrated. However, the role of differential regulation of Notch signaling on pancreas development is not well understood.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5"></a><p>Microarray is a useful method for genomewide expression profile analysis <a href="#pone.0001479-Schena1">[48]</a>. With this approach, several downstream genes of signaling pathways involved in controlling mouse pancreas development have been reported <a href="#pone.0001479-Chiang1">[49]</a>, <a href="#pone.0001479-Gu2">[50]</a>. Following the leads from such microarray analysis, a novel transcription factor, Myt1 <a href="#pone.0001479-Gu2">[50]</a> has been shown to be involved in pancreas development. Similarly, Mellitzer et al. also showed that a novel transcription factor, IA1, is Ngn3-regulated and required for normal differentiation of endocrine pancreatic cells <a href="#pone.0001479-Mellitzer1">[51]</a>. High-density microarrays have been established for zebrafish and transcriptome profiles during embryogenesis have been documented <a href="#pone.0001479-Mathavan1">[52]</a>. In addition, new target genes involved in sonic hedgehog signaling pathway has been identified in zebrafish using microarrays <a href="#pone.0001479-Xu1">[53]</a>. However, a genomewide analysis of Notch signaling defective mutants has not been reported.</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6"></a><p>Hence, we have carried out this study with three major objectives, viz. (1) to examine the three <em>mib</em> mutant alleles for pancreas defects and understand the consequence of differential activation of Notch signaling on pancreas development using <em>in situ</em> hybridization and pancreas-specific GFP expression analysis, (2) to identify differentially regulated genes between the wt zebrafish and three different <em>mib</em> mutant alleles through microarray analysis and (3) to identify and validate the Notch-responsive genes involved in pancreas development. Through our present work, we have shown that the development of exocrine pancreas is diminished in <em>mib</em> mutants and this defect is most severe in <em>mib<sup>ta52b</sup></em> followed by that in <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em>. Using microarray analysis, we have identified several characterized genes, and novel genes/ESTs potentially involved in pancreas development. Validation of representative genes with real-time PCR and fluorescent double <em>in situ</em> hybridization supported the microarray data. Furthermore, this study has generated useful genomic resources identifying a number of uncharacterized ESTs/genes that may play a significant role in Notch-regulated developmental process.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Pancreas defects in <em>mib</em> mutant alleles</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>We carried out <em>in situ</em> hybridization for four pancreas-specific genes in <em>mib</em> mutants of 4-day post-fertilization (dpf) and their wild type (wt) siblings (<a href="#pone-0001479-g001">Figure 1</a>). The expression of exocrine pancreas-specific genes, <em>elastaseA</em> (<a href="#pone-0001479-g001">Figure 1A and 1A</a>′–1D′) and <em>trypsin</em> (<a href="#pone-0001479-g001">Figure 1E and 1E</a>′–H′) were diminished in <em>mib</em> mutants. In comparison to their expression in wt embryos, these two genes were highly down-regulated in <em>mib<sup>ta52</sup></em> followed by those in <em>mib<sup>m132</sup></em> and then <em>mib<sup>tfi91</sup></em>, which are less down-regulated. This observation is in concurrence with the diminished Notch activity in these <em>mib</em> mutants <a href="#pone.0001479-Zhang1">[34]</a>. We also analyzed the <em>elastaseA</em>-GFP expression in 4-dpf <em>mib</em> mutants and their wt siblings (<a href="#pone-0001479-g002">Figure 2A–2D</a>). The <em>elastaseA</em>-GFP expression was diminished in <em>mib</em> mutant alleles in the same order of decreasing Notch activity in these mutants, which corroborates our observation shown by <em>in situ</em> hybridization (<a href="#pone-0001479-g001">Figure 1A and 1A</a>′–1D′).</p>
<div class="figure" id="pone-0001479-g001"><div class="img"><a name="pone-0001479-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Expression pattern of pancreas-specific genes at 96 hpf in <em>mib</em> mutants and their wild type (wt) siblings.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>The RNA probes used for <em>in situ</em> hybridization are (A and A′–D′) <em>elastaseA</em>, (E and E′–H′) <em>trypsin</em>, (I and I′–L′) <em>somatostatin</em> and (M and M′–P′) <em>insulin</em>. The genotypes are (A, A′, E, E′, I, I′, M and M′) <em>wt</em>, (B′, F′, J′ and N′) <em>mib<sup>tfi91</sup></em>, (C′, G′, K′ and O′) <em>mib<sup>m132</sup></em> and (D′, H′, L′ and P′) <em>mib<sup>ta52b</sup></em>. A′, E′, I′ and M′ are cropped from A, E, I and M, respectively. All panels are lateral views and anterior to the right.</p>
<span>doi:10.1371/journal.pone.0001479.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We further analyzed the expression of three endocrine pancreas-specific genes, <em>somatostatin</em> (δ-cell-specific), <em>insulin</em> (β-cell-specific), and <em>pdx1</em> (pancreas progenitor-specific). Levels of <em>somatostatin</em> (<a href="#pone-0001479-g001">Figure 1I and 1I</a>′–1L′) and <em>insulin</em> (<a href="#pone-0001479-g001">Figure 1M and 1M</a>′–1P′) were slightly increased in 4-dpf <em>mib</em> mutants. The <em>insulin</em>-GFP (<a href="#pone-0001479-g002">Figure 2E and 2E</a>′–2H′) and <em>pdx1</em>-GFP (<a href="#pone-0001479-g002">Figure 2I and 2I</a>′–2L′) expression analysis in <em>mib</em> mutants also showed that these genes are slightly up-regulated compared to their expression in wt embryos.</p>
<div class="figure" id="pone-0001479-g002"><div class="img"><a name="pone-0001479-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span><em>elastaseA</em>-GFP, <em>insulin</em>-GFP and <em>pdx1</em>-GFP expression in embryos of <em>mib</em> mutant alleles and their wt siblings.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>Expression of <em>elastaseA</em>-GFP in (A) <em>wt</em>, (B) <em>mib<sup>tfi91</sup></em>, (C) <em>mib<sup>m132</sup></em> and (D) <em>mib<sup>ta52b</sup></em> embryos was analyzed at 96 hpf. Expression of <em>insulin</em>-GFP in (E and E′) <em>wt</em>, (F′) <em>mib<sup>tfi91</sup></em>, (G′) <em>mib<sup>m132</sup></em> and (H′) <em>mib<sup>ta52b</sup></em> embryos was analyzed at 72 hpf. Expression of <em>pdx1</em>-GFP in (I and I′) <em>wt</em>, (J′) <em>mib<sup>tfi91</sup></em>, (K′) <em>mib<sup>m132</sup></em> and (L′) <em>mib<sup>ta52b</sup></em> embryos was analyzed at 72 hpf. Panels A–D are lateral views and the rest are dorsolateral views, oriented anterior to the right.</p>
<span>doi:10.1371/journal.pone.0001479.g002</span></div>

<h4>Genomewide expression profiling</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>We carried out the genomewide expression profiling of <em>mib</em> mutants at three different stages–24-, 48- and 72-hour post-fertilization (hpf). Through microarray analysis of 72-hpf embryos, we identified 1128 up-regulated and 936 down-regulated genes in <em>mib<sup>ta52b</sup></em> mutants (<a href="#pone.0001479.s001">Table S1</a>, q = 0.0); 1464 up-regulated and 2210 down-regulated genes in <em>mib<sup>m132</sup></em> mutants (<a href="#pone.0001479.s002">Table S2</a>, q = 0.0); and 2081 up-regulated and 2538 down-regulated genes in <em>mib<sup>tfi91</sup></em> mutants (<a href="#pone.0001479.s003">Table S3</a>, q = 0.0). Using a PERL script, we further identified the differentially expressed genes specific to each mutant allele and common to all three <em>mib</em> mutant alleles (<a href="#pone-0001479-g003">Figure 3</a>). In the list of up-regulated genes, the numbers of genes specific to <em>mib<sup>ta52b</sup></em>, <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> mutant alleles were 93, 287 and 768, respectively; in the list of down-regulated genes, the numbers of genes specific to <em>mib<sup>ta52b</sup></em>, <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> alleles were 33, 557 and 874, respectively (<a href="#pone-0001479-g003">Figure 3A and 3B</a>, <a href="#pone.0001479.s005">Table S5</a>, <a href="#pone.0001479.s006">S6</a> and <a href="#pone.0001479.s007">S7</a>, q = 0.0 and score(d)&gt;4.0). The majority of these differentially expressed genes were uncharacterized genes or ESTs (<a href="#pone-0001479-t001">Table 1</a>). There were 91 genes common to all three <em>mib</em> mutant alleles, of which 31 were up-regulated and 60 were down-regulated (<a href="#pone-0001479-g003">Figure 3A and 3B</a>, <a href="#pone-0001479-t001">Table 1</a>). Of these 91 genes, only 27 genes were previously characterized and 64 were uncharacterized genes or ESTs (<a href="#pone-0001479-t001">Table 1</a>, <a href="#pone.0001479.s004">Table S4</a>, q = 0.0 and score(d)&gt;4.0). The cluster tree view showed the expression profile of these 91 genes: the up-regulated ones were shown in red and the down-regulated ones in green (<a href="#pone-0001479-g004">Figure 4</a>). We further categorized these 91 common genes according to their known function or predicted function based on their homology to mouse and human orthologs (<a href="#pone.0001479.s004">Table S4</a>). Of the 27 characterized genes, there are 6 up-regulated genes, including <em>dab2</em>, <em>mcl1a</em>, <em>mcl1b</em>, <em>fn1l</em>, <em>ttn</em> and <em>nppa</em>, and 21 down-regulated genes, including <em>pbx3b</em>, <em>gpm6aa</em> (BI840762), <em>olig2</em>, <em>tfdp2</em>, <em>rpl13</em>, <em>gpm6aa</em> (BI839927), <em>atp1a1b</em>, <em>fabp7a</em>, <em>gpm6aa</em> (BG306150), <em>fkbp5</em>, <em>gfap</em>, <em>opn1sw1</em>, <em>opn1sw2</em>, <em>opn1mw1</em>, <em>vsx1</em>, <em>pou50</em>, <em>mdkb</em>, <em>tal1</em>, <em>dla</em>, <em>vamp2</em> and <em>her4</em>.</p>
<div class="figure" id="pone-0001479-g003"><div class="img"><a name="pone-0001479-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Venn diagrams show numbers of genes belonging to different groups.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>Differentially expressed genes specific to each <em>mib</em> mutant allele, common between two mutant alleles and common to all three mutant alleles are (A) up-regulated and (B) down-regulated at 72 hpf; and (C) up-regulated and (D) down-regulated at 48 hpf; and genes specific to each time point, common between two time points and common to all three time points are (E) up-regulated and (F) down-regulated in <em>mib<sup>ta52b</sup></em> mutants. Gene set for the analysis of 72-hpf data was selected based on the criteria, q = 0.0 and score(d)&gt;4.0; and for the analysis of 48-hpf data and the three time-point data for <em>mib<sup>ta52b</sup></em> mutants were both selected based on the criterion, q = 0.0.</p>
<span>doi:10.1371/journal.pone.0001479.g003</span></div><div class="figure" id="pone-0001479-g004"><div class="img"><a name="pone-0001479-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>Gene expression profile of 91 differentially expressed (q = 0.0, score(d)&gt;4.0) genes (Genbank ID) common to all three <em>mib</em> mutant alleles at 72 hpf.</span></strong></p><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1"></a><p>At least two biological repeats and two technical repeats were carried out for each mutant allele. Each horizontal strip represents expression of a single gene. Color in each cell reflects the expression level of corresponding gene in the respective sample. The up-regulation is shown in red and the down-regulation in green. Grey cells indicate the missing values. The median log2 ratio values for differential expression ranged between −4.07 and 3.32. Note: The gene symbols are mentioned instead of Genbank IDs, where the gene symbols corresponding to the Genbank IDs are available in the Zebrafish Chip Annotation Database. Three Genbank IDs, BI840762, BI839927 and BG306150, encode the same gene, <em>gpm6aa</em>.</p>
<span>doi:10.1371/journal.pone.0001479.g004</span></div><div class="figure" id="pone-0001479-t001"><div class="img"><a name="pone-0001479-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Differentially expressed genes in 72-hpf <em>mib</em> mutants categorized based on their characterization status.</span></strong></p><span>doi:10.1371/journal.pone.0001479.t001</span></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Using the same PERL script, we analyzed the 48-hpf time point data for three <em>mib</em> mutant alleles, and data of 24-hpf, 48-hpf and 72-hpf time points for the <em>mib<sup>ta52b</sup></em> mutant allele. Numbers of differentially expressed genes common to all three mutant alleles or three time points, and specific to each mutant allele or time point are shown in the Venn diagram (<a href="#pone-0001479-g003">Figure 3C–3F</a>). At 48 hpf, there are 44 (30 up-regulated and 14 down-regulated) differentially expressed genes common to all three mutant alleles, and 129 (54 up-regulated and 75 down-regulated), 124 (44 up-regulated and 80 down-regulated) and 693 (635 up-regulated and 58 down-regulated) genes specific to <em>mib<sup>ta52b</sup></em>, <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> mutant alleles, respectively (<a href="#pone-0001479-g003">Figure 3C and 3D</a>; <a href="#pone.0001479.s008">Table S8</a>, <a href="#pone.0001479.s009">S9</a>, <a href="#pone.0001479.s010">S10</a> and <a href="#pone.0001479.s011">S11</a>, q = 0.0). For <em>mib<sup>ta52b</sup></em> mutants, there are 61 (32 up-regulated and 29 down-regulated) genes common to all three time points, and 1821 (1128 up-regulated and 693 down-regulated), 49 (14 up-regulated and 35 down-regulated) and 1851 (930 up-regulated and 921 down-regulated) genes specific at 24 hpf, 48 hpf and 72 hpf, respectively (<a href="#pone-0001479-g003">Figure 3E and 3F</a>; <a href="#pone.0001479.s012">Table S12</a>, <a href="#pone.0001479.s013">S13</a>, <a href="#pone.0001479.s014">S14</a> and <a href="#pone.0001479.s015">S15</a>, q = 0.0).</p>


<h4>Functional categories</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>We used the microarray data of three <em>mib</em> mutant alleles (q = 0.0, score(d)&gt;4.0) and the 91 common genes at 72 hpf for functional analysis. We first searched for the gene ontology and the functional similarity of their human and mouse orthologs in the Zebrafish Chip Annotation Database. Then, we classified them into different functional categories based on their known functions in zebrafish or in mouse and human orthologs, such as ‘transcription factor/nuclear’, ‘signaling’, ‘cell adhesion/matrix’, ‘cell cycle/apoptosis’, ‘transcription factors’, ‘hormone activity’ and ‘structural proteins’. Finally, we plotted them in a pie chart format either including the uncharacterized genes and ESTs (<a href="#pone-0001479-g005">Figure 5A–5D</a>) or excluding them (<a href="#pone-0001479-g005">Figure 5E–5H</a>). The majority (&gt;70%) of the differentially expressed genes are uncharacterized genes or ESTs (<a href="#pone-0001479-g005">Figure 5A–5D</a>). Among the genes with known or related functions in the <em>mib<sup>ta52b</sup></em> data set, about 35% belong to the category of ‘transcription factor/nuclear’ related genes and the remaining genes to other categories, such as ‘transport proteins’ (25%), ‘signaling’ (19%), ‘structural proteins’ (8.6%), ‘cell adhesion/matrix’ (5.4%), ‘hormone activity’ (4.3%) and ‘cell cycle/apoptosis’ (3.2%) (<a href="#pone-0001479-g005">Figure 5E</a>).</p>
<div class="figure" id="pone-0001479-g005"><div class="img"><a name="pone-0001479-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>Pictorial representation of significantly up-regulated or down-regulated genes (q = 0.0, score(d)&gt;4.0) categorized based on known/related biological functions as assigned in the zebrafish gene ontology database (Unigene Build 85).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Genes in each functional group were searched by using specific or related key words appropriate for that function in the lists of differentially regulated genes for each allele, common 91 genes in all three <em>mib</em> mutant alleles, and the genes specific to each mutant allele. Absolute number of genes in each category is shown next to the corresponding pie sector. (A–D) Pie charts show numbers of genes belonging to different functional categories. (E–H) Pie charts show numbers of genes (excluding the genes without any known function) in different categories.</p>
<span>doi:10.1371/journal.pone.0001479.g005</span></div>

<h4>Functional groups and pathway analysis by IPA</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>To further analyze our data set, we used IPA (see <a href="#s4">Materials and Methods</a>) to identify functional groups and selected the top 15 ones as significantly enriched for their respective functions (<a href="#pone-0001479-g006">Figure 6A</a>). The significance of each function (calculated from negative log of p-value; -log 1.3 is equal to p-0.05) revealed that all the top 15 functions are highly and significantly enriched in all three <em>mib</em> alleles. Some of the significantly enriched functions are ‘cellular growth and proliferation’, ‘cellular development’, ‘gene expression’ and ‘embryonic development’. Genes related to ‘connective tissue disorders’, ‘skeletal muscular disorders’ and ‘cancer’ are significantly enriched in the <em>mib<sup>ta52b</sup></em> allele.</p>
<div class="figure" id="pone-0001479-g006"><div class="img"><a name="pone-0001479-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Significantly up-regulated and down-regulated genes analyzed by IPA (see <a href="#s4">Materials and Methods</a>) for (A) enrichment of functional groups and (B) enrichment of canonical pathways, shown in histograms.</span></strong></p><span>doi:10.1371/journal.pone.0001479.g006</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>One of the tools in IPA enables us to identify the enrichment of genes in selected canonical pathways from the input data set. The top 15 canonical pathways that are enriched in the <em>mib</em> mutants include Notch and Wnt signaling pathways (<a href="#pone-0001479-g006">Figure 6B</a>). Genes related to Wnt signaling were significantly enriched in <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> mutants. Notch signaling genes were enriched to a lesser extent and their enrichment was not significant. Canonical pathways, such as ‘actin cytoskeleton signaling’, ‘integrin signaling’, ‘calcium signaling’ and ‘TGF-beta signaling’, were enriched in <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em>.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>Notch and Wnt/β-catenin signaling were shown to be affected in <em>mib</em> mutants <a href="#pone.0001479-Itoh1">[27]</a>, <a href="#pone.0001479-Riley1">[54]</a>, <a href="#pone.0001479-Amoyel1">[55]</a>. We mapped the genes that are involved in these two signaling pathways and differentially expressed in different <em>mib</em> alleles. The genes involved in Notch signaling, such as <em>dll1, notch1, notch2, hey1</em> and <em>herp</em>, were down-regulated in one, two or three <em>mib</em> alleles (<a href="#pone.0001479.s021">Figure S1A</a>). In Wnt/β-catenin signaling, some genes, such as <em>nlk, cx43</em>, <em>sox2, sox3, sox4, sox9, sox11, tcf7</em> and <em>hdac1</em>, were down-regulated in one or three <em>mib</em> alleles; while <em>sox8</em> and <em>tab1</em> were up-regulated (<a href="#pone.0001479.s021">Figure S1B</a>). Few of the genes differed in the direction of expression change in different mutant alleles. For example, <em>β-catenin</em> was up-regulated in <em>mib<sup>m132</sup></em> while it was down-regulated in <em>mib<sup>ta52b</sup></em> and <em>mib<sup>tfi91</sup></em> (<a href="#pone.0001479.s021">Figure S1B</a>).</p>


<h4>Putative Notch signaling related genes</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>As positive controls of our microarray data, we searched for the known Notch signaling related genes in our list of differentially expressed genes. As expected, <em>heyL</em> was up-regulated and <em>dla, her4</em>, <em>her8a</em>, <em>hes5</em>, <em>hey1</em>, <em>neurod</em>, <em>neurod4</em>, <em>notch1a</em>, <em>notch2</em> and <em>notch3</em> were down-regulated (<a href="#pone-0001479-t002">Table 2</a>). To identify novel genes related to Notch signaling, we searched for the differentially expressed genes occurring at least two times in the gene sets common to all three <em>mib</em> mutant alleles at 72 hpf and 48 hpf, and the gene set common for the <em>mib<sup>ta52b</sup></em> mutant allele at 72 hpf, 48 hpf and 24 hpf. We identified 31 such genes, the majority of which are novel genes or ESTs (<a href="#pone.0001479.s016">Table S16</a>). Of these 31 genes, only eight genes are characterized and five of them are known to be directly related to Notch signaling: <em>olig2</em> <a href="#pone.0001479-Park1">[56]</a>, <em>her4</em> <a href="#pone.0001479-Takke1">[57]</a>, <em>hes5</em> (also known as <em>her15.1</em>, <a href="#pone.0001479-Bae1">[58]</a>, <a href="#pone.0001479-Shankaran1">[59]</a>), <em>dla</em> <a href="#pone.0001479-Appel1">[60]</a> and <em>nort</em> (<a href="#pone.0001479-Tsutsumi1">[61]</a>, BI886648 is 98% identical to AB259590). Moreover, <em>fn1l</em> (also known as <em>fn1b</em> or <em>fn3</em>), in conjunction with its homolog, <em>fn1</em>, has been shown to function cooperatively with Notch signaling via integrinα5 in somitogenesis <a href="#pone.0001479-Jlich2">[62]</a>, <a href="#pone.0001479-Chong1">[63]</a>. <em>gfap</em> and <em>gefiltin</em> are glia- and neuron-specific structural proteins respectively and, therefore, indirectly linked to Notch-modulated gliogenesis and neurogenesis <a href="#pone.0001479-Irvin1">[64]</a>–<a href="#pone.0001479-Leake1">[66]</a>. These observations suggest that a certain proportion of the 23 novel genes/ESTs are very likely to be involved in or related to Notch signaling.</p>
<div class="figure" id="pone-0001479-t002"><div class="img"><a name="pone-0001479-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Fold change of expression level in Notch signaling related genes at 72 hpf in three <em>mib</em> alleles.</span></strong></p><span>doi:10.1371/journal.pone.0001479.t002</span></div>

<h4>Putative pancreas development related genes</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We searched the gene expression data in the ZFIN database to identify the genes involved in pancreas development (<a href="#pone-0001479-t003">Table 3</a>, <a href="#pone-0001479-t004">Table 4</a> and <a href="#pone.0001479.s017">Table S17</a>). There are 175 zebrafish genes in this group; however, only 98 of these genes are represented in our microarray chip. 25 of these 98 genes show significant difference (q&lt;2.0) in gene expression in all three <em>mib</em> alleles: <em>cx43</em>, <em>neurod</em>, <em>ins</em>, <em>fabp2</em>, <em>cacna1c</em>, <em>zgc:92530</em>, <em>pax6b</em>, <em>try</em>, <em>olfm2</em>, <em>mt2</em>, <em>xbp1</em>, <em>hsd11b2</em>, <em>zgc:92292</em>, <em>gadd45a</em>, <em>wu:fb59c09</em>, <em>cbs</em>, <em>tdh</em>, <em>gstp1</em>, <em>ssr4</em>, <em>gatm</em>, <em>rims2</em>, <em>atp1a3a</em>, <em>isl1</em>, <em>hoxc8a</em> and <em>notch1a</em> (<a href="#pone-0001479-t003">Table 3</a>). The rest are differentially expressed either in one or two of these three alleles (<a href="#pone.0001479.s017">Table S17</a>). In addition, we identified 5 genes, namely <em>gad1</em> <a href="#pone.0001479-Mally1">[67]</a>, <em>igfbp2</em> <a href="#pone.0001479-Hill1">[68]</a>, <em>zgc:112198</em> <a href="#pone.0001479-Pleau1">[69]</a>, <em>wu:fc21h08</em> <a href="#pone.0001479-vanHaeften1">[70]</a> and <em>isl3</em> <a href="#pone.0001479-Lowe1">[71]</a> based on the available functional analysis reports for their mouse orthologs, and 4 other genes, namely <em>sfrs5</em>, <em>zgc:56374</em>, AW281193 and BI846588 based on their gene ontology information available in the zebrafish gene annotation database (<a href="#pone-0001479-t004">Table 4</a>). These 9 genes are likely to play a role in zebrafish pancreas development.</p>
<div class="figure" id="pone-0001479-t003"><div class="img"><a name="pone-0001479-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Expression level of differentially expressed genes (q&lt;2.0) involved in pancreas development at 72 hpf in three <em>mib</em> alleles shown in terms of fold change.</span></strong></p><span>doi:10.1371/journal.pone.0001479.t003</span></div><div class="figure" id="pone-0001479-t004"><div class="img"><a name="pone-0001479-t004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t004.TIF"></span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Expression level of differentially expressed genes (q&lt;2.0), predicted to be involved in pancreas development based on functional homology, at 72 hpf in three <em>mib</em> alleles shown in terms of fold change.</span></strong></p><span>doi:10.1371/journal.pone.0001479.t004</span></div>

<h4>Fluorescent double <em>in situ</em> hybridization validation of pancreas development related genes</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>To validate the microarray data for the genes predicted to be related to or involved in pancreas development, we carried out fluorescent double <em>in situ</em> hybridization at 3-dpf embryos for five down-regulated genes: <em>trypsin</em>, <em>isl1</em>, <em>cad</em>, <em>wu:fb59c09</em> and <em>notch1a</em>, and five up-regulated genes: <em>insulin</em>, <em>isl3</em>, <em>spon1b</em>, <em>glo1</em> and <em>txnip</em> (<a href="#pone-0001479-t003">Table 3</a>, <a href="#pone-0001479-t004">Table 4</a> and <a href="#pone.0001479.s017">Table S17</a>). Exocrine <em>trypsin</em> and endocrine <em>insulin</em> are predictably down- and up-regulated, respectively (<a href="#pone-0001479-g007">Figure 7</a>). However, <em>notch1a</em>, <em>glo1</em> and <em>txnip</em> were not detected in the pancreas (see <a href="#s3">Discussion</a>).</p>
<div class="figure" id="pone-0001479-g007"><div class="img"><a name="pone-0001479-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7.  <span>Validation of expression profile of differentially expressed genes with fluorescent double <em>in situ</em> hybridization.</span></strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Expression of down-regulated genes: (A and B) <em>isl1</em>, (C and D) <em>cad</em> and (E and F) <em>wu:fb59c09</em> and up-regulated genes: (G and H) <em>spon1b</em> and (I and J) <em>isl3</em> in (B, D, F, H and J) <em>mib<sup>ta52b</sup></em> mutants embryos and (A, C, E, G and I) wild-type embryos. Left panels show respective gene expression in green; middle panels show <em>insulin</em> (endocrine) or <em>trypsin</em> (exocrine) expression in red; right panels show merged pictures. Expression of <em>wu:fb59c09</em> is shown in 4-dpf embryos, and the rest are in 3-dpf embryos. All embryos are anterior to the left and ventral views.</p>
<span>doi:10.1371/journal.pone.0001479.g007</span></div><h5>Down-regulated genes.</h5><a id="article1.body1.sec2.sec7.sec1.p1" name="article1.body1.sec2.sec7.sec1.p1"></a><p><em>isl1</em> encodes a zebrafish insulin gene enhancer binding protein and it has 81% identity to mouse LIM/homeodomain transcription factor ISL1 (islet-1). <em>isl1</em> was expressed in nervous system, liver (data not shown) and pancreas of 3-dpf embryos. Our result is similar to that in earlier studies, which showed that <em>isl1</em> is expressed in organs such as nervous system, liver and pancreas at various developmental stages of zebrafish <a href="#pone.0001479-Bae1">[58]</a>, <a href="#pone.0001479-Inoue1">[72]</a>–<a href="#pone.0001479-Kim1">[77]</a>. Its expression was strongly diminished in the nervous system (data not shown) of <em>mib<sup>ta52b</sup></em> mutants; however, its expression was only slightly reduced in the endocrine pancreas compared to wt embryos (<a href="#pone-0001479-g007">Figure 7A and 7B</a>). Our microarray results also showed that <em>isl1</em> is down-regulated in <em>mib<sup>ta52b</sup></em> mutants (<a href="#pone-0001479-t003">Table 3</a>).</p>
<a id="article1.body1.sec2.sec7.sec1.p2" name="article1.body1.sec2.sec7.sec1.p2"></a><p><em>cad</em> encodes zebrafish carbamoyl-phosphate synthetase 2. Microarray analysis showed that <em>cad</em> is down-regulated (<a href="#pone.0001479.s017">Table S17</a>). <em>cad</em> was expressed in endodermal organs, such as pancreas, intestine and liver, which is similar to that observed in earlier reports <a href="#pone.0001479-Willer1">[78]</a>, <a href="#pone.0001479-Thisse2">[79]</a>. In <em>mib<sup>ta52b</sup></em> mutants, <em>cad</em> expression was down-regulated in all these tissues (<a href="#pone-0001479-g007">Figure 7C and 7D</a>; data not shown).</p>
<a id="article1.body1.sec2.sec7.sec1.p3" name="article1.body1.sec2.sec7.sec1.p3"></a><p><em>wu:fb59c09</em> is a zebrafish EST with significant similarity to a hypothetical gene, LOC570477. The human ortholog of this gene is <em>Peroxiredoxin 4</em> (<em>prx4</em>). <em>In situ</em> hybridization results showed that this gene is expressed in liver, pancreas, intestine and other endodermal tissues (data not shown). It was down-regulated in both endocrine and exocrine pancreas of <em>mib<sup>ta52b</sup></em> mutants compared to that in their wt siblings (<a href="#pone-0001479-g007">Figure 7E and 7F</a>).</p>

<h5>Up-regulated genes.</h5><a id="article1.body1.sec2.sec7.sec2.p1" name="article1.body1.sec2.sec7.sec2.p1"></a><p><em>spon1b</em> encodes an extracellular matrix protein, whose C-terminal contains five repeats identified previously in Thrombospondin and other proteins implicating in cell adhesion. Fluorescent double <em>in situ</em> hybridization results showed that this gene is expressed in endocrine pancreas, similar to earlier studies <a href="#pone.0001479-Kim1">[77]</a>, <a href="#pone.0001479-Thisse2">[79]</a>, and was up-regulated in <em>mib<sup>ta52b</sup></em> mutants compared to that in their wt siblings (<a href="#pone-0001479-g007">Figure 7G and 7H</a>).</p>
<a id="article1.body1.sec2.sec7.sec2.p2" name="article1.body1.sec2.sec7.sec2.p2"></a><p><em>isl3</em> encodes a zebrafish insulin gene enhancer binding protein homolog. It has high similarity to human ISL1 transcription factor, LIM/homeodomain (Islet-1) and ISL2 transcription factor, LIM/homeodomain (Islet-2). <em>isl3</em> was expressed in endocrine pancreas, liver and intestine and its expression was up-regulated in <em>mib<sup>ta52b</sup></em> mutants (<a href="#pone-0001479-g007">Figure 7I and 7J</a>; data not shown).</p>



<h4>Real-time PCR validation of global expression profile</h4>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1"></a><p>To further validate the microarray expression profile analysis, we carried out real-time PCR for 25 genes using the primers listed in <a href="#pone.0001479.s018">Table S18</a>. All the PCRs yielded the expected 300 base pair products. The real-time PCR ratios are basically similar to those of microarray data (<a href="#pone-0001479-t005">Table 5</a>), which confirms the validity of the array data.</p>
<div class="figure" id="pone-0001479-t005"><div class="img"><a name="pone-0001479-t005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001479" data-uri="info:doi/10.1371/journal.pone.0001479.t005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001479.t005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001479.t005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001479.t005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001479.t005.TIF"></span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Real-time PCR validation of microarray data.</span></strong></p><span>doi:10.1371/journal.pone.0001479.t005</span></div><a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2"></a><p>We noticed that the results from two methods for few genes are not consistent. Therefore, to further address this question statistically, we evaluated the linear relationship between the microarray ratio (fold change) and the real-time PCR ratio for <em>mib<sup>ta52b</sup></em> allele at 72 hpf using Pearson's correlation coefficient. It indicates a statistically significant positive correlation between these two sets of ratios (R = 0.914, R<sup>2</sup> = 0.835, p&lt;0.001). This shows that the two sets of ratios are significantly (positively) correlated and the data are acceptably reliable <a href="#pone.0001479-Wang1">[80]</a>.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p><em>mind bomb</em> mutants have compromised Notch activity due to mutations in the <em>mib</em> gene and serve as a unique resource to study the role of Notch signaling on various developmental processes <a href="#pone.0001479-Itoh1">[27]</a>, <a href="#pone.0001479-Jiang1">[29]</a>. Moreover, there are several <em>mib</em> alleles with different genetic severity available <a href="#pone.0001479-Zhang1">[34]</a>. In this study, we have examined the <em>mib</em> mutants for defects in pancreas development using <em>in situ</em> hybridization and pancreas-specific GFP expression analyses, and compared the global expression profile of three <em>mib</em> mutant alleles and their wt siblings using oligo microarray chips.</p>

<h4><em>mib</em> alleles are unique for studying Notch signaling</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p>The <em>mib</em> mutants have diminished exocrine pancreas development as evidenced by decrease in <em>elastaseA</em> and <em>trypsin</em> expression. The <em>mib<sup>ta52b</sup></em> mutant allele showed maximal decrease in exocrine pancreas followed by <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> alleles. This indicates that there is a dose-dependent response of Notch signaling on pancreas development, since these alleles have different degree of compromised Notch activity <a href="#pone.0001479-Zhang1">[34]</a>. Our GFP expression analysis in <em>elastaseA</em>-GFP lines also showed a similar effect of graded Notch signaling on pancreas development. Likewise, based on <em>in situ</em> hybridization results for exocrine pancreas-specific genes, such as <em>mnr2a</em>, <em>ptf1a</em> and <em>trypsin</em> in <em>mib<sup>ta52b</sup></em> mutants and DAPT-treated <em>mib<sup>ta52b</sup></em> mutants, Zecchin <em>et al</em>. concluded that a blockage of Notch signaling decreases the number of exocrine pancreatic cells <a href="#pone.0001479-Zecchin2">[40]</a>. Earlier study by Esni <em>et al</em>. also showed the role of Notch signaling in exocrine pancreas development of zebrafish <a href="#pone.0001479-Esni1">[37]</a>.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2"></a><p>The <em>in situ</em> hybridization results for the endocrine pancreas-specific genes, such as <em>insulin</em> and <em>somatostatin</em>, showed slight increase in their expression in <em>mib</em> mutant alleles; however, a dose-dependent response of Notch signaling was not obvious (<a href="#pone-0001479-g001">Figure 1</a>). The GFP expression analyses in <em>insulin</em>-GFP and <em>pdx1</em>-GFP lines also showed a slight increase in endocrine pancreas in <em>mib</em> mutant alleles (<a href="#pone-0001479-g002">Figure 2</a>). Similar to our results, Zecchin <em>et al</em>. also showed that in <em>mib</em> mutants (<em>mib<sup>ta52b</sup></em>) there is an increased expression of <em>insulin</em> and <em>somatostatin</em> <a href="#pone.0001479-Zecchin2">[40]</a>.</p>


<h4>Microarray analysis in <em>mib</em> alleles</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>In this study, we have used the microarray chips with 16,416 probes, representing 15,800 unique zebrafish genes. Earlier studies have successfully used this version of microarray chips for elucidating genes involved in zebrafish embryogenesis <a href="#pone.0001479-Mathavan1">[52]</a>, liver tumor progression <a href="#pone.0001479-Lam1">[81]</a> and Hedgehog signaling <a href="#pone.0001479-Xu1">[53]</a>. Comparison of our microarray results for three <em>mib</em> mutant alleles at 72 hpf showed that the number of differentially expressed genes (q = 0.0, score(d)&gt;4.0) in <em>mib<sup>tfi91</sup></em> allele (2352) is greater than that in <em>mib<sup>m132</sup></em> (1536) and <em>mib<sup>ta52b</sup></em> (321) alleles (<a href="#pone-0001479-g003">Figure 3A and 3B</a>). Interestingly, this is inversely related to the severity of their phenotypes, such as fused somite borders, diminished tail pigmentation, neuronal hyperplasia and diminished <em>her4</em> expression: most severe in <em>mib<sup>ta52b</sup></em> allele followed by those in <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> alleles <a href="#pone.0001479-Zhang1">[34]</a>. So far, there is no known molecular mechanism to explain this finding. Future studies need to be focused in this direction.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>We observed fewer differentially expressed genes at 48 hpf compared to that at 72 hpf for all three <em>mib</em> mutant alleles. This is in concurrence with the phenotypes of these three <em>mib</em> alleles, which are less obvious at 48 hpf compared to that at 72 hpf <a href="#pone.0001479-Zhang1">[34]</a>. Owing to the large number of differentially expressed genes at these two time points, we narrowed our focus on the 72-hpf data set alone. In all three <em>mib</em> mutant alleles at 72 hpf, the majority (&gt;70%) of the differentially expressed genes are uncharacterized genes or ESTs. This is because the zebrafish genome annotation has not been completed and the majority of genes remain as uncharacterized ESTs. Though a number of differentially expressed genes have been identified as full-length clones with ZGC IDs, they still remain unannotated. Gene Ontology (GO) analysis of differentially expressed genes of <em>mib<sup>ta52b</sup></em> allele showed that less than 29% of the genes are characterized, the majority (35%) of which belong to the category of ‘transcription factors/nuclear’ followed by the category of ‘signaling molecules’ (19%). This could be due to the fact that the <em>mib</em> mutants are defective in Notch signaling, which is one of the fundamental signaling pathways required for proper development of an organism.</p>


<h4>Genes potentially involved in Notch signaling</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>Our microarray data is reliable, because some expected Notch signaling related genes did appear in the transcriptome, such as up-regulated <em>heyL</em> and down-regulated <em>dla</em>. Comparison of three sets of genes (72 hpf and 48 hpf for three <em>mib</em> alleles and 24 hpf, 48 hpf and 72 hpf of the <em>mib<sup>ta52b</sup></em> allele) showed that there are 31 genes common to all three gene sets (<a href="#pone.0001479.s016">Table S16</a>). Five of these 31 genes, namely <em>olig2</em> <a href="#pone.0001479-Park1">[56]</a>, <em>her4</em> <a href="#pone.0001479-Takke1">[57]</a>, <em>hes5</em> (also known as <em>her15.1</em>, <a href="#pone.0001479-Bae1">[58]</a>, <a href="#pone.0001479-Shankaran1">[59]</a>), <em>dla</em> <a href="#pone.0001479-Appel1">[60]</a> and <em>nort</em> <a href="#pone.0001479-Tsutsumi1">[61]</a>, are previously shown to be involved in Notch signaling. Particularly, based on similar global expression analyses of <em>notch1a</em> and <em>notch3</em> morpholino morphants, Tsutsumi and Itoh showed that <em>nort</em> is a putative noncoding RNA regulated by Notch signaling in zebrafish <a href="#pone.0001479-Tsutsumi1">[61]</a>. Therefore, it is likely that the rest of the genes in this group are a part of the Notch connected network and hence serve as a useful resource to further identify novel genes working downstream of Notch.</p>


<h4>Genes potentially involved in pancreas development</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>As a major focus of our current study, we searched for the genes related to pancreas development. As of now, there is no bioinformatic tool available to classify the zebrafish genes according to their biological functions. Therefore, we carried out a detailed manual search for the genes related to pancreas development. Using the ZFIN <em>in situ</em> hybridization expression database, we found 98 zebrafish genes that have been previously shown to be expressed in pancreas (<a href="#pone-0001479-t003">Table 3</a>, <a href="#pone-0001479-t004">Table 4</a> and <a href="#pone.0001479.s017">Table S17</a>). The microarray expression profile for these genes showed that their expression profile is significantly different (q&lt;2.0) in at least one of the three <em>mib</em> alleles. Several genes, such as <em>neurod</em>, <em>isl1</em> and <em>pax6b</em> <a href="#pone.0001479-Jensen2">[82]</a>–<a href="#pone.0001479-Jensen3">[84]</a> were formerly shown to be responsive to Notch signaling in mice.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>However, the fold change of expression for each gene is different in these three alleles. Only 25 out of these 98 zebrafish genes involved in pancreas development show significant difference (q&lt;2.0) in expression profile in all three <em>mib</em> alleles. Out of these 25 genes, only 15 genes show consistent up-regulation (3 genes) or down-regulation (12 genes) in all three alleles. Apart from these 98 genes, we discovered 9 differentially expressed genes (<a href="#pone.0001479.s019">Table S19</a>), which are likely to play a role in zebrafish pancreas development based on their functional homology to mouse and human orthologs. It is definitely worth addressing in the future.</p>


<h4>Is Mib Notch-specific?</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>So far, the experiments have unequivocally proved that Mib is an essential component of Notch signaling: it ubiquitylates and then endocytoses Delta with the extracellular part of Notch and therefore allows the intracellular part of Notch to enter the nucleus in the receiving cell and activate downstream target genes <a href="#pone.0001479-Itoh1">[27]</a>. There is also evidence to suggest that Mib may be linked to Wnt signaling. Riley, et al. showed that heat shock-driven <em>wnt1</em> expression in <em>mib</em> mutants leads to a partial rescue of its hindbrain metameric patterning phenotype <a href="#pone.0001479-Riley1">[54]</a>. Furthermore, knockdown of <em>wnt3a</em> and <em>wnt8b</em> in <em>Dfw5</em> mutants, where <em>wnt1</em> and <em>wnt10b</em> are deleted, resulted in the lost of boundary cells in hindbrain, which is similar to that in <em>mib<sup>ta52b</sup></em> mutants <a href="#pone.0001479-Riley1">[54]</a>. However, this is in sharp contrast to <em>wnt1</em> and <em>tcf3b</em> morphants, where the boundary cells are increased <a href="#pone.0001479-Amoyel1">[55]</a>. Therefore, the mechanism remains to be determined.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2"></a><p>From our IPA pathway analysis in all three <em>mib</em> alleles, we found enrichment of several differentially expressed genes belonging to various canonical pathways that have not been shown to directly link to Mib, including IGF-1, PDGF, EGF and IL-2 signaling pathways (<a href="#pone-0001479-g006">Figure 6B</a>). This raises the possibility that Mib may be somehow involved in these signaling pathways. Furthermore, mouse DAPK and zebrafish Jagged2a have been shown to be substrates of Mib E3 ligase <a href="#pone.0001479-Jin1">[85]</a>, <a href="#pone.0001479-Ma1">[86]</a>. With a yeast two-hybrid screen, we also found that Mib binds to proteins involved in endocytosis and the ubiquitin-proteasome pathway (Chengjin Zhang, Jason Kin Wai Koo, Qing Li, Haoying Xu and Y.-J. J., unpublished data). Similarly, Snx5 has been identified as a Mib-binding protein from a yeast two-hybrid screen, which is colocalized with Mib in early endosomes and required for hematopoiesis and vasculogenesis <a href="#pone.0001479-Yoo1">[87]</a>. All these observations suggest that Mib may not be Notch-specific and can also work with other pathways, just as previously shown for another E3 ligase, Itch, which targets Notch receptor and links to TNF through JNK <a href="#pone.0001479-Qiu1">[88]</a>–<a href="#pone.0001479-Chang1">[90]</a>. Alternatively, the gene expression change could simply reflect the tissue/organ defects that are inflicted by a failure in Notch signaling. Our pathway analysis on microarray data supplies a good resource for examining whether Mib functions beyond Notch signaling and/or for testing what genes are involved in the tissue/organ mis-patterning caused by compromised Notch activity.</p>


<h4>Validation of microarray data</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>To validate the microarray expression profile, we carried out fluorescent double <em>in situ</em> hybridization for ten genes (5 down-regulated and 5 up-regulated) and real-time PCR for 25 genes. Out of these ten genes, <em>insulin</em>, <em>trypsin</em>, <em>spon1b</em>, <em>cad</em>, <em>isl1</em>, <em>wu:fb59c09</em>, <em>notch1a</em>, <em>txnip</em> and <em>glo1</em> are previously shown to be involved in the development of and/or expressed in zebrafish pancreas; and <em>isl3</em> is predicted to be related to pancreas development.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2"></a><p>The microarray expression profile of <em>trypsin</em> and <em>isl1</em> showed a down-regulation in <em>mib<sup>ta52b</sup></em> allele and this was validated by our fluorescent double <em>in situ</em> hybridization and real-time PCR (<a href="#pone-0001479-g007">Figure 7</a> and <a href="#pone-0001479-t005">Table 5</a>). <em>wu:fb59c09</em> was slightly up-regulated (1.24) in microarrays of <em>mib<sup>ta52b</sup></em> allele but it was down-regulated in <em>mib<sup>m132</sup></em> and <em>mib<sup>tfi91</sup></em> alleles (<a href="#pone-0001479-t003">Table 3</a>). However, our fluorescent double <em>in situ</em> hybridization and real-time PCR validations showed that it is down-regulated in the <em>mib<sup>ta52b</sup></em> allele (<a href="#pone-0001479-g007">Figure 7</a> and <a href="#pone-0001479-t005">Table 5</a>). <em>cad</em> expression was decreased in the <em>mib<sup>ta52b</sup></em> allele by double <em>in situ</em>, which is consistent with the microarray data (<a href="#pone-0001479-g007">Figure 7</a> and <a href="#pone.0001479.s017">Table S17</a>). Earlier studies have shown that all these four genes are expressed in zebrafish pancreas at various developmental stages <a href="#pone.0001479-Esni1">[37]</a>, <a href="#pone.0001479-Thisse1">[75]</a>, <a href="#pone.0001479-Willer1">[78]</a>, <a href="#pone.0001479-Thisse2">[79]</a>. However, the role of Notch signaling on their expression profile in pancreas is not known except <em>isl1</em>. <em>isl1</em> was shown to be up-regulated in primary neurons of <em>mib</em> mutants at 16s to 20s stage <a href="#pone.0001479-Schfer1">[91]</a>, but the effect of Notch signaling on its expression at later stages has not been studied. Furthermore, no observation has been made on its expression in pancreas. In contrast, our <em>in situ</em> hybridization results in 3-dpf embryos showed that the expression of <em>isl1</em> in <em>mib<sup>ta52b</sup></em> allele is down-regulated in nervous system but its expression in pancreas is only slightly reduced compared to that in wt embryos. In support of this observation, our microarray and real-time PCR results also showed that <em>isl1</em> is down-regulated in <em>mib<sup>ta52b</sup></em> mutants (<a href="#pone-0001479-t003">Table 3</a> and <a href="#pone-0001479-t005">5</a>).</p>
<a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3"></a><p>Two up-regulated genes, <em>insulin</em> and <em>spon1b</em>, were validated. The microarray data showed that these two genes are up-regulated in the <em>mib<sup>ta52b</sup></em> allele and this is supported by our fluorescent double <em>in situ</em> hybridization. <em>isl3</em> was slightly increased by fluorescent double <em>in situ</em> hybridization. However, it was consistently decreased in <em>mib<sup>ta52b</sup></em> allele by microarray analysis and real-time PCR (<a href="#pone-0001479-g007">Figure 7</a>, <a href="#pone-0001479-t004">Table 4</a> and <a href="#pone-0001479-t005">5</a>).</p>
<a id="article1.body1.sec3.sec6.p4" name="article1.body1.sec3.sec6.p4"></a><p>In contrast to the available information from ZFIN, which shows the gene expression pattern mainly up to 2 dpf, we did not detect <em>notch1a</em>, <em>glo1</em> and <em>txnip</em> expression in the pancreas using fluorescent double <em>in situ</em> hybridization at 3 dpf, though we did detect <em>notch1a</em> expressed in, for example, hindbrain. It could be due to the stage difference or technical reasons. However, our real-time PCR results validated microarray expression for <em>notch1a</em> and <em>glo1</em> (<a href="#pone-0001479-t005">Table 5</a>).</p>
<a id="article1.body1.sec3.sec6.p5" name="article1.body1.sec3.sec6.p5"></a><p>The ratio comparison between microarray and real-time PCR for genes, such as <em>ipf1</em>, <em>perp</em>, <em>igf2</em>, <em>isl3</em>, <em>spon1b</em> and BI846588, showed that there is slight up-regulation in microarray, but down-regulation in real-time PCR. However, it is evident that the overall expression alteration of all 25 genes is statistically comparable, although the actual fold change values in real-time PCR and microarray are not always commensurable. Such variation in values is likely due to the difference in sensitivity <a href="#pone.0001479-Wang1">[80]</a>. Nevertheless, our <em>in situ</em> hybridization results are highly consistent with our microarray profile.</p>
<a id="article1.body1.sec3.sec6.p6" name="article1.body1.sec3.sec6.p6"></a><p>In conclusion, the microarray analyses carried out in this study provide a useful resource of global gene expression profile of <em>mib</em> mutants defective in Notch signaling. Functional analysis of differentially expressed genes will shed light on their role in Notch signaling and various developmental processes.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Zebrafish wild type embryos and <em>mind bomb</em> mutants</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>We used AB strain wild type (wt) and three different <em>mib</em> alleles of different genetic severity, viz., <em>mib<sup>ta52b</sup></em>, and <em>mib<sup>m132</sup></em>, <em>mib<sup>tfi91</sup></em> <a href="#pone.0001479-Zhang1">[34]</a>. <em>mib<sup>ta52b</sup></em> carries a missense mutation (M1013R) in the C-terminal-most RING finger domain; <em>mib<sup>m132</sup></em> carries a nonsense mutation (C785stop) leading to a truncated protein and <em>mib<sup>tfi91</sup></em> contains a nonsense mutation (Y60stop) <a href="#pone.0001479-Itoh1">[27]</a>. <em>mib<sup>ta52b</sup></em> and <em>mib<sup>m132</sup></em> are strong and weak antimorphic alleles, respectively, whereas <em>mib<sup>tfi91</sup></em> is a null allele <a href="#pone.0001479-Zhang1">[34]</a>. All animal procedures were approved by the Biological Resource Centre, A*STAR.</p>


<h4>Fish maintenance and sample collection</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>Fish were maintained in the IMCB zebrafish facility according to standard procedures. Crosses were set up in the evening and the barrier was lifted in the next morning. After half an hour, the fertilized embryos were collected and maintained at 28.5°C in egg water supplemented with methylene blue. For microarray analyses, the wt embryos were collected at 24 hpf, 48 hpf and 72 hpf, snap-frozen in liquid nitrogen and stored at -80°C. Mutants were separated from their wt siblings and frozen stored in the same way. At least two independent biological replicates were taken for each sample. All the samples were collected from the same cohort of fish to maintain a uniform genetic background.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2"></a><p>For whole mount <em>in situ</em> hybridization and GFP analysis, the fertilized embryos were collected and grown at 28.5°C. Embryos were transferred to egg water with 0.033% phenylthiourea (PTU), which inhibits pigmentation, after 12 hpf and fixed/collected at appropriate stages.</p>


<h4><em>In situ</em> hybridization and fluorescent double <em>in situ</em> hybridization</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>DNA clones for making <em>in situ</em> hybridization probes were obtained from the Expressed Sequence Tag (EST) clone collection at the Genome Institute of Singapore (GIS) and the Institute of Molecular and Cell Biology (IMCB). Whole mount <em>in situ</em> hybridization using digoxigenin (DIG) (Roche) labeled RNA probes was carried out as previously reported <a href="#pone.0001479-Qiu2">[92]</a>. Goat anti-DIG antibody conjugated to alkaline phosphatase (AP) was used for probe detection and NBT-BCIP was used as the substrate for color development. <em>In situ</em> hybridized embryos were observed using light microscope and the photos were taken with Zeiss Imager M1 microscope.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>DIG- and fluorescein-labeled probes were generated via standard protocols. Embryos were proceeded for fluorescent double <em>in situ</em> hybridization with protocols previously reported <a href="#pone.0001479-Jlich1">[32]</a>, <a href="#pone.0001479-Ma1">[86]</a>. However, the incubation temperature for probes was changed to 60°C to reduce the background. Photos were taken with Olympus Fluoview FV1000 microscope.</p>


<h4>GFP expression analysis</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To screen for defects in pancreas development, we have used three pancreas-specific GFP transgenic lines, viz., <em>elastaseA</em>-GFP (<em>elaA</em>-GFP) <a href="#pone.0001479-Wan1">[93]</a>, <em>insulin</em>-GFP and <em>pdx1</em>-GFP lines <a href="#pone.0001479-Huang1">[94]</a>. These GFP lines were crossed with the heterozygous <em>mib</em> mutants and the offspring (F1) were grown until maturity. The F1 siblings carrying <em>mib</em> mutations were intercrossed to obtain embryos for GFP expression analysis. Photos were taken with Leica MZ FLIII microscope.</p>


<h4>RNA extraction</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Total RNA from the frozen embryos was extracted with Trizol (Gibco BRL) and cleaned with the Qiagen RNeasy mini kit. RNA quality was determined by gel electrophoresis, and the concentration was measured with a UV spectrophotometer. To reduce the bias, we used a common reference RNA for each time point, which was prepared at one time by extracting RNA from stage-matched wild type embryos. The RNA extracts were stored at −80°C.</p>


<h4>Microarray construction, target preparation and hybridization</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>The zebrafish microarrays were printed at GIS <a href="#pone.0001479-Mathavan1">[52]</a>. Oligonucleotide probes for this array were designed by Compugen (USA) and synthesized by Sigma Genesis (USA). For each gene, one 65-mer-oligonucleotide probe was designed from the 3′ region sequence. Each probe was selected from a sequence segment that is common to a maximum number of splice variants predicted for each gene. The arrays contained 16,416 probes, representing 15, 800 unique zebrafish genes (UniGene build 85). In addition, the arrays also contained 170 spots representing <em>β-actin</em> gene probes as controls. The probes were suspended at a concentration of 20 µM in 3X SSC and spotted onto poly-L-Lysine coated microscope slides using custom-built DNA microarrayer.</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2"></a><p>For fluorescent labeling of target cDNAs, 20 µg of total RNA (10 µg, when the RNA quantity was limited) from reference and experimental samples were reverse-transcribed in the presence of Cy3-dUTP and Cy5-dUTP (Amersham Biosciences), respectively. Labeled target cDNAs were combined, concentrated and resuspended in DIG EasyHyb (Roche). Hybridizations on microarray slides were performed at 42°C for 16 h using MAUI Mixer FL (BioMicro Systems) as explained earlier <a href="#pone.0001479-Mathavan1">[52]</a>, <a href="#pone.0001479-Eisen1">[95]</a>. At least two independent biological replicates were taken for each sample. At least two to three independent replicate hybridizations (technical repeats) were performed for each biological repeat sample (<a href="#pone.0001479.s020">Table S20</a>).</p>


<h4>Scanning, filtering and data normalization</h4>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1"></a><p>The arrays were scanned by the GenePix 4000B microarray scanner (Axon Instruments) to generate 16-bit TIFF image files. GenePix Pro 4.0 image analysis software (Axon Instruments) was used to measure the fluorescent signal intensity of the array features and local background on TIFF images. Only the gene features with signal background ratio more than 1.5 were used for analysis. The 16-bit TIFF image files and the gpr files with Cy3 and Cy5 signal intensities were uploaded into the GIS-developed Microarray Database (mAdb). Median normalization of the sample and reference channel intensity values was performed using the intensity-based log ratio median method <a href="#pone.0001479-Yang1">[96]</a>. The extracted intensity data from the mAdb database were normalized by Lowess normalization method and analyzed by modified <em>t</em>-statistic Significance Analysis of Microarrays (SAM) <a href="#pone.0001479-Tusher1">[97]</a>. The microarray data files have been submitted to the Gene Expression Omnibus (GEO) and the accession number is GSE8522.</p>


<h4>Gene annotation</h4>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1"></a><p>The gene annotations were carried out by using the Zebrafish Chip Annotation Database (MySqL) (UniGene build 85) (<a href="http://giscompute.gis.a-star.edu.sg/govind/zebrafish/version2">http://giscompute.gis.a-star.edu.sg/govi​nd/zebrafish/version2</a>) developed and maintained by GIS. This database contains putative annotations for the probes in the zebrafish oligonucleotide array. We queried this database with the Genbank ID to obtain the following information: (1) Compugen description, (2) Zebrafish UniGene ID (build 85), (3) Zebrafish UniGene description (build 85), (4) Entrez Gene description, (5) Entrez Gene ID and Gene symbol, (6) GO term, (7) Locus Link, (8) UniGene protein similarity and description (mouse and human), (9) Full-length or assembled sequences, (10) HomoloGene (human and mouse, build 38.1) and (11) chromosomal location of the gene. Full-length sequences or the longest available gene sequences were obtained from the NCBI UniGene database (<a href="http://www.ncbi.nlm.nih.gov/sites/entrezdbunigenecmd">http://www.ncbi.nlm.nih.gov/sites/entrez​dbunigenecmd</a>). The Genbank IDs are referred to as ‘genes’ throughout this article.</p>


<h4>Microarray data analysis</h4>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1"></a><p>We applied Significance Analysis of Microarrays (SAM) <a href="#pone.0001479-Tusher1">[97]</a> to identify statistically significant genes in each case. Since we have different number of replicates for different alleles, we used different thresholds (q = 0.0, score(d)&gt;4.0 [this represents that the absolute value of score(d) is greater than 4.0, namely, |score(d)|&gt;4.0] at 72 hpf; q = 0.0 at 48 hpf; q = 0.0 at 24 hpf) to select similar number of genes. However, the thresholds we used are all stringent (in every case the q-value is less than 2) and hence the false discovery rate (FDR, value expressed in %) in each case does not exceed 5. The FDR indicates the outcome with which the gene selected to be differentially expressed by the SAM analysis is likely to be occurring by chance. The score(d) indicates a statistic parameter, which is numerator(r) divided by denominator(s+s0) and hence serves as a cut-off point along with the q value. The numerator(r) value indicates the actual gene expression change shown as log2 value.</p>
<a id="article1.body1.sec4.sec9.p2" name="article1.body1.sec4.sec9.p2"></a><p>For the analysis of 72-hpf time point data, we selected those genes with q = 0.0 and scored(d)&gt;4.0 from the SAM generated data for all three <em>mib</em> alleles (<a href="#pone.0001479.s001">Table S1</a>, <a href="#pone.0001479.s002">S2</a> and <a href="#pone.0001479.s003">S3</a>). The description for all the gene sets was obtained from the Zebrafish Chip Annotation Database <a href="#pone.0001479-Mathavan1">[52]</a>. Based on the most recently available information of the zebrafish gene annotation, these gene sets were classified as characterized and uncharacterized genes (<a href="#pone-0001479-t001">Table 1</a>). Furthermore, we manually searched for two sets of genes: one with functions related to Notch signaling and the other with functions related to pancreas development (<a href="#pone-0001479-t002">Table 2</a>–<a href="#pone-0001479-t003"></a><a href="#pone-0001479-t004">4</a>). The expression profile values (log2) for the genes involved in the Notch pathway and pancreas development were obtained from the SAM analyzed data set. The <em>in situ</em> hybridization gene expression data for the genes related to pancreas development were obtained from the ZFIN database (<a href="http://zfin.org/cgi-bin/webdriverMIvalaa-xpatselect.apg">http://zfin.org/cgi-bin/webdriverMIvalaa​-xpatselect.apg</a>). If the gene was previously shown to be involved in pancreas development, we classified them as the ‘genes involved in pancreas development’. If there is no functional relationship to zebrafish pancreas development but only functional homology to the human or mouse genes related to pancreas development, we classified them as the ‘genes predicted to be involved in pancreas development’.</p>
<a id="article1.body1.sec4.sec9.p3" name="article1.body1.sec4.sec9.p3"></a><p>A PERL script was used to identify the differentially expressed (q = 0.0, score(d)&gt;4.0) genes (Genbank IDs) that are common among all three <em>mib</em> alleles (<a href="#pone.0001479.s004">Table S4</a>), between two different alleles and specific to each allele (<a href="#pone.0001479.s005">Table S5</a>, <a href="#pone.0001479.s006">S6</a> and <a href="#pone.0001479.s007">S7</a>) at 72 hpf, and to remove duplicate genes (Genbank IDs), if any. Using this method, a group of 91 genes common to all three <em>mib</em> alleles were classified based on their function and characterization status (<a href="#pone.0001479.s004">Table S4</a>). These 91 genes were hierarchically clustered with TreeView_vers_1_60 software <a href="#pone.0001479-Eisen2">[98]</a> and tree view image was generated using Adobe Illustrator (<a href="#pone-0001479-g004">Fig. 4</a>).</p>
<a id="article1.body1.sec4.sec9.p4" name="article1.body1.sec4.sec9.p4"></a><p>For the analysis of 48-hpf time point data, the gene sets with q = 0.0 were used. The same PERL script was also used here to find the common genes and the allele-specific genes (<a href="#pone.0001479.s008">Table S8</a>, <a href="#pone.0001479.s009">S9</a>, <a href="#pone.0001479.s010">S10</a> and <a href="#pone.0001479.s011">S11</a>). The SAM data (q = 0.0) for the <em>mib<sup>ta52b</sup></em> on all three time points (24 hpf, 48 hpf and 72 hpf) were analyzed and queried using the PERL script to find out the gene sets that are common to all time points and specific to each time point (<a href="#pone.0001479.s012">Table S12</a>, <a href="#pone.0001479.s013">S13</a>, <a href="#pone.0001479.s014">S14</a> and <a href="#pone.0001479.s015">S15</a>).</p>


<h4>Functional groups and pathway analysis</h4>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1"></a><p>Differentially expressed genes of three <em>mib</em> mutant alleles were subjected to Ingenuity Pathways Analysis (IPA) to identify the enrichment of genes in specific functional groups and pathways (IPA, Version 4, Ingenuity® Systems, <a href="http://www.ingenuity.com">http://www.ingenuity.com</a>). The IPA accepts human UniGene IDs as one of the identifiers for data upload and analysis. For this reason, the differentially expressed genes of <em>mib</em> mutants were mapped to their human homologs using the HomoloGene database and zebrafish UniGene mapping tool established at the GIS (<a href="http://giscompute.gis.a-star.edu.sg/govind/unigene_db/">http://giscompute.gis.a-star.edu.sg/govi​nd/unigene_db/</a>). Human homologs of up- and down-regulated genes of the mutants were analyzed by using IPA tools and the enrichment of functional categories and canonical pathways with reference to the Ingenuity Pathways Knowledge Base (IPKB) were documented. Initially, differentially expressed genes of the three mutant alleles were individually analyzed. Subsequently, the enrichment patterns were compared among the mutants to identify the conservation of functional groups among the mutants.</p>
<a id="article1.body1.sec4.sec10.p2" name="article1.body1.sec4.sec10.p2"></a><p>Using the input data set (human homologs of zebrafish genes differentially expressed in the <em>mib</em> mutants), IPA identified a set of genes that are enriched for a particular function or pathway and the enrichment is represented as ratio. The ratio refers to the number of input genes associated with each function/pathway versus the total number of genes (available in IPKB) involved in that particular function/pathway. The ratios may be affected by the variations in the total number of input identifiers. In order to find the significance of enrichment in a particular function, IPA calculates the significance value based on the measure of involvement of the gene in the input data set to their respective molecular functions/signaling pathways. Using the right-tailed Fisher's Exact Test, the p-value (significance) is calculated by comparing the number of user-specified genes of interest that participate in a given function or pathway, relative to the total number of occurrences of these genes in all functional/pathway annotations in the IPKB.</p>


<h4>Real-time PCR</h4>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1"></a><p>To validate the microarray results, we carried out the real-time PCR for 25 genes and <em>beta-actin</em> gene was used as a reference. The primers used for amplifying each gene were listed in <a href="#pone.0001479.s018">Table S18</a>. cDNA was generated using the same purified RNA preparations from 72-hpf embryos (one biological repeat of wt and two biological repeats of <em>mib<sup>ta52b</sup></em>) used in microarray, and two other biological repeats of reference RNA from 72 hpf and one biological repeat sample RNA from <em>mib<sup>ta52b</sup></em> mutants. RT-PCR was carried out using the LightCycler® FastStart DNA Master<sup>PLUS</sup> SYBR Green kit (Roche) and the Light Cycler machine as per the instructions of the manufacturer. The products of the RT-PCR were analyzed on the agarose gel electrophoresis for a single band of expected size. Relative cDNA amounts were calculated using the comparative C<sub>T</sub> method as explained in the real-time PCR manual of Applied Biosystems and normalized to the expression of <em>beta-actin</em>.</p>


<h4>Analysis of correlation between microarray data and real-time PCR results</h4>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1"></a><p>Subsequent to microarray and real-time PCR data analysis, an evaluation of linear correlation was performed for a set of 25 genes (<a href="#pone-0001479-t005">Table 5</a>), and the statistical significance of the correlation was determined using One-way ANOVA in SPS software. For the correlation analysis, the data input of the microarray was the fold change of expression and the data input of the real-time PCR was the ratio of relative expression for each gene. Both sets of ratios were obtained from the 72-hpf time point for the <em>mib<sup>ta52b</sup></em> allele.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001479"></div><a name="pone.0001479.s001" id="pone.0001479.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) in 72-hpf ta52b mutants generated with SAM.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.75 MB XLS)</p>
<a name="pone.0001479.s002" id="pone.0001479.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) in 72-hpf m132 mutants generated with SAM.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(3.12 MB XLS)</p>
<a name="pone.0001479.s003" id="pone.0001479.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) in 72-hpf tfi91 mutants generated with SAM.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(3.92 MB XLS)</p>
<a name="pone.0001479.s004" id="pone.0001479.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0, score(d)&gt;4.0) common to all three mib mutant alleles at 72 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0001479.s005" id="pone.0001479.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0, score(d)&gt;4.0) specific to ta52b mutants at 72 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0001479.s006" id="pone.0001479.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0, score(d)&gt;4.0) specific to m132 mutants at 72 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.52 MB XLS)</p>
<a name="pone.0001479.s007" id="pone.0001479.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s007">Table S7. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0, score(d)&gt;4.0) specific to tfi91 mutants at 72 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.93 MB XLS)</p>
<a name="pone.0001479.s008" id="pone.0001479.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s008">Table S8. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) common to all three mib mutant alleles at 48 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0001479.s009" id="pone.0001479.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s009">Table S9. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) specific to ta52b mutants at 48 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.08 MB XLS)</p>
<a name="pone.0001479.s010" id="pone.0001479.s010"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s010">Table S10. </a></strong></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) specific to m132 mutants at 48 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s010</p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2"></a><p class="postSiDOI">(0.09 MB XLS)</p>
<a name="pone.0001479.s011" id="pone.0001479.s011"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s011">Table S11. </a></strong></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) specific to tfi91 mutants at 48 hpf.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s011</p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2"></a><p class="postSiDOI">(0.38 MB XLS)</p>
<a name="pone.0001479.s012" id="pone.0001479.s012"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s012">Table S12. </a></strong></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) common at all three time points for ta52b mutants.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s012</p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2"></a><p class="postSiDOI">(0.05 MB XLS)</p>
<a name="pone.0001479.s013" id="pone.0001479.s013"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s013">Table S13. </a></strong></p><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) specific to 24 hpf time point in ta52b mutants.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s013</p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2"></a><p class="postSiDOI">(1.01 MB XLS)</p>
<a name="pone.0001479.s014" id="pone.0001479.s014"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s014">Table S14. </a></strong></p><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) specific to 48 hpf time point in ta52b mutants.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s014</p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2"></a><p class="postSiDOI">(0.03 MB XLS)</p>
<a name="pone.0001479.s015" id="pone.0001479.s015"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s015">Table S15. </a></strong></p><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) specific to 72 hpf time point in ta52b mutants.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s015</p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2"></a><p class="postSiDOI">(1.07 MB XLS)</p>
<a name="pone.0001479.s016" id="pone.0001479.s016"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s016">Table S16. </a></strong></p><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1"></a><p class="preSiDOI">List of significantly expressed genes (q = 0.0) common in at least two data sets.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s016</p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001479.s017" id="pone.0001479.s017"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s017">Table S17. </a></strong></p><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1"></a><p class="preSiDOI">Zebrafish pancreas development related genes.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s017</p><a id="article1.body1.sec5.supplementary-material17.caption1.p2" name="article1.body1.sec5.supplementary-material17.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001479.s018" id="pone.0001479.s018"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s018">Table S18. </a></strong></p><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1"></a><p class="preSiDOI">Sequence of primers used in real-time PCR validation on microarray data.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s018</p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2"></a><p class="postSiDOI">(0.07 MB DOC)</p>
<a name="pone.0001479.s019" id="pone.0001479.s019"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s019">Table S19. </a></strong></p><a id="article1.body1.sec5.supplementary-material19.caption1.p1" name="article1.body1.sec5.supplementary-material19.caption1.p1"></a><p class="preSiDOI">Genes predicted to be related to zebrafish pancreas development.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s019</p><a id="article1.body1.sec5.supplementary-material19.caption1.p2" name="article1.body1.sec5.supplementary-material19.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001479.s020" id="pone.0001479.s020"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s020">Table S20. </a></strong></p><a id="article1.body1.sec5.supplementary-material20.caption1.p1" name="article1.body1.sec5.supplementary-material20.caption1.p1"></a><p class="preSiDOI">Summary of biological and technical repeats.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s020</p><a id="article1.body1.sec5.supplementary-material20.caption1.p2" name="article1.body1.sec5.supplementary-material20.caption1.p2"></a><p class="postSiDOI">(0.01 MB XLS)</p>
<a name="pone.0001479.s021" id="pone.0001479.s021"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001479.s021">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material21.caption1.p1" name="article1.body1.sec5.supplementary-material21.caption1.p1"></a><p class="preSiDOI">Changes of expression levels in the canonical pathways. (A) The Notch signaling pathway and (B) the Wnt/β-catenin signaling pathway. The nodes in these pathways are highlighted with expression data from the <em>m132</em> data (red: up-regulated; green: down-regulated). Nodes with a histogram chart next to it represents gene expression in the i) <em>ta52b</em>, ii) <em>m132</em> and iii) <em>tfi91</em> data set from left to right. (TIF)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001479.s021</p><a id="article1.body1.sec5.supplementary-material21.caption1.p2" name="article1.body1.sec5.supplementary-material21.caption1.p2"></a><p class="postSiDOI">(1.71 MB TIF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Serene Gek Ping Lee for providing microarray reagents and protocols and Dr. Shuo Lin for supplying <em>insulin</em>-GFP and <em>pdx1</em>-GFP transgenic lines.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: YJ AH. Performed the experiments: AH KT NQ SH XQ. Analyzed the data: KG YJ JG AH FN SM. Contributed reagents/materials/analysis tools: ZG YJ SM. Wrote the paper: YJ AH SM.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001479-ArtavanisTsakonas1" id="pone.0001479-ArtavanisTsakonas1"></a>Artavanis-Tsakonas S, Rand MD, Lake RJ (1999) Notch signaling: cell fate control and signal integration in development. Science  284: 770–776.  <ul class="find" data-citedArticleID="1045588" data-doi="10.1126/science.284.5415.770"><li><a href="http://dx.doi.org/10.1126/science.284.5415.770" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+signaling%3A+cell+fate+control+and+signal+integration+in+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+signaling%3A+cell+fate+control+and+signal+integration+in+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001479-Pourqui1" id="pone.0001479-Pourqui1"></a>Pourquié O (2003) The segmentation clock: converting embryonic time into spatial pattern. Science  301: 328–330.  <ul class="find" data-citedArticleID="1045714" data-doi="10.1126/science.1085887"><li><a href="http://dx.doi.org/10.1126/science.1085887" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+segmentation+clock%3A+converting+embryonic+time+into+spatial+pattern." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+segmentation+clock%3A+converting+embryonic+time+into+spatial+pattern.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001479-Ellisen1" id="pone.0001479-Ellisen1"></a>Ellisen LW, Bird J, West DC, Soreng AL, Reynolds TC, et al.  (1991) TAN-1, the human homolog of the <em>Drosophila Notch</em> gene, is broken by chromosomal translocations in T lymphoblastic neoplasms. Cell  66: 649–661.  <ul class="find" data-citedArticleID="1045610" data-doi="10.1016/0092-8674(91)90111-b"><li><a href="http://dx.doi.org/10.1016/0092-8674(91)90111-b" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=TAN-1%2C+the+human+homolog+of+the+Drosophila+Notch+gene%2C+is+broken+by+chromosomal+translocations+in+T+lymphoblastic+neoplasms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22TAN-1%2C+the+human+homolog+of+the+Drosophila+Notch+gene%2C+is+broken+by+chromosomal+translocations+in+T+lymphoblastic+neoplasms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001479-Yan1" id="pone.0001479-Yan1"></a>Yan XQ, Sarmiento U, Sun Y, Huang G, Guo J, et al.  (2001) A novel Notch ligand, Dll4, induces T-cell leukemia/lymphoma when overexpressed in mice by retroviral-mediated gene transfer. Blood  98: 3793–3799.  <ul class="find" data-citedArticleID="1045758" data-doi="10.1182/blood.v98.13.3793"><li><a href="http://dx.doi.org/10.1182/blood.v98.13.3793" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+Notch+ligand%2C+Dll4%2C+induces+T-cell+leukemia%2Flymphoma+when+overexpressed+in+mice+by+retroviral-mediated+gene+transfer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+Notch+ligand%2C+Dll4%2C+induces+T-cell+leukemia%2Flymphoma+when+overexpressed+in+mice+by+retroviral-mediated+gene+transfer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001479-Thompson1" id="pone.0001479-Thompson1"></a>Thompson BJ, Buonamici S, Sulis ML, Palomero T, Vilimas T, et al.  (2007) The SCFFBW7 ubiquitin ligase complex as a tumor suppressor in T cell leukemia. J Exp Med  204: 1825–1835.  <ul class="find" data-citedArticleID="1045744" data-doi="10.1084/jem.20070872"><li><a href="http://dx.doi.org/10.1084/jem.20070872" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+SCFFBW7+ubiquitin+ligase+complex+as+a+tumor+suppressor+in+T+cell+leukemia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+SCFFBW7+ubiquitin+ligase+complex+as+a+tumor+suppressor+in+T+cell+leukemia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001479-Oda1" id="pone.0001479-Oda1"></a>Oda T, Elkahloun AG, Pike BL, Okajima K, Krantz ID, et al.  (1997) Mutations in the human <em>Jagged1</em> gene are responsible for Alagille syndrome. Nat Genet  16: 235–242.  <ul class="find" data-citedArticleID="1045702" data-doi="10.1038/ng0797-235"><li><a href="http://dx.doi.org/10.1038/ng0797-235" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+human+Jagged1+gene+are+responsible+for+Alagille+syndrome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+human+Jagged1+gene+are+responsible+for+Alagille+syndrome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001479-Li1" id="pone.0001479-Li1"></a>Li L, Krantz ID, Deng Y, Genin A, Banta AB, et al.  (1997) Alagille syndrome is caused by mutations in human <em>Jagged1</em>, which encodes a ligand for Notch1. Nat Genet  16: 243–251.  <ul class="find" data-citedArticleID="1045680" data-doi="10.1038/ng0797-243"><li><a href="http://dx.doi.org/10.1038/ng0797-243" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alagille+syndrome+is+caused+by+mutations+in+human+Jagged1%2C+which+encodes+a+ligand+for+Notch1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alagille+syndrome+is+caused+by+mutations+in+human+Jagged1%2C+which+encodes+a+ligand+for+Notch1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001479-McDaniell1" id="pone.0001479-McDaniell1"></a>McDaniell R, Warthen DM, Sanchez-Lara PA, Pai A, Krantz ID, et al.  (2006) <em>NOTCH2</em> mutations cause Alagille syndrome, a heterogeneous disorder of the notch signaling pathway. Am J Hum Genet  79: 169–173.  <ul class="find" data-citedArticleID="1045694" data-doi="10.1086/505332"><li><a href="http://dx.doi.org/10.1086/505332" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NOTCH2+mutations+cause+Alagille+syndrome%2C+a+heterogeneous+disorder+of+the+notch+signaling+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NOTCH2+mutations+cause+Alagille+syndrome%2C+a+heterogeneous+disorder+of+the+notch+signaling+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001479-Joutel1" id="pone.0001479-Joutel1"></a>Joutel A, Corpechot C, Ducros A, Vahedi K, Chabriat H, et al.  (1996) <em>Notch3</em> mutations in CADASIL, a hereditary adult-onset condition causing stroke and dementia. Nature  383: 707–710.  <ul class="find" data-citedArticleID="1045660" data-doi="10.1038/383707a0"><li><a href="http://dx.doi.org/10.1038/383707a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch3+mutations+in+CADASIL%2C+a+hereditary+adult-onset+condition+causing+stroke+and+dementia." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch3+mutations+in+CADASIL%2C+a+hereditary+adult-onset+condition+causing+stroke+and+dementia.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001479-Bulman1" id="pone.0001479-Bulman1"></a>Bulman MP, Kusumi K, Frayling TM, McKeown C, Garrett C, et al.  (2000) Mutations in the human <em>Delta</em> homologue, <em>DLL3</em>, cause axial skeletal defects in spondylocostal dysostosis. Nat Genet  24: 438–441.  <ul class="find" data-citedArticleID="1045598"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutations+in+the+human+Delta+homologue%2C+DLL3%2C+cause+axial+skeletal+defects+in+spondylocostal+dysostosis.&amp;auth=&amp;atitle=Mutations+in+the+human+Delta+homologue%2C+DLL3%2C+cause+axial+skeletal+defects+in+spondylocostal+dysostosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+in+the+human+Delta+homologue%2C+DLL3%2C+cause+axial+skeletal+defects+in+spondylocostal+dysostosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+in+the+human+Delta+homologue%2C+DLL3%2C+cause+axial+skeletal+defects+in+spondylocostal+dysostosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001479-Brou1" id="pone.0001479-Brou1"></a>Brou C, Logeat F, Gupta N, Bessia C, LeBail O, et al.  (2000) A novel proteolytic cleavage involved in Notch signaling: the role of the disintegrin-metalloprotease TACE. Mol Cell  5: 207–216.  <ul class="find" data-citedArticleID="1045596" data-doi="10.1016/s1097-2765(00)80417-7"><li><a href="http://dx.doi.org/10.1016/s1097-2765(00)80417-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+novel+proteolytic+cleavage+involved+in+Notch+signaling%3A+the+role+of+the+disintegrin-metalloprotease+TACE." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+novel+proteolytic+cleavage+involved+in+Notch+signaling%3A+the+role+of+the+disintegrin-metalloprotease+TACE.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001479-Fortini1" id="pone.0001479-Fortini1"></a>Fortini ME (2001) Notch and presenilin: a proteolytic mechanism emerges. Curr Opin Cell Biol  13: 627–634.  <ul class="find" data-citedArticleID="1045614" data-doi="10.1016/s0955-0674(00)00261-1"><li><a href="http://dx.doi.org/10.1016/s0955-0674(00)00261-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+and+presenilin%3A+a+proteolytic+mechanism+emerges." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+and+presenilin%3A+a+proteolytic+mechanism+emerges.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001479-Kopan1" id="pone.0001479-Kopan1"></a>Kopan R, Goate A (2000) A common enzyme connects notch signaling and Alzheimer's disease. Genes Dev  14: 2799–2806.  <ul class="find" data-citedArticleID="1045668" data-doi="10.1101/gad.836900"><li><a href="http://dx.doi.org/10.1101/gad.836900" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+common+enzyme+connects+notch+signaling+and+Alzheimer%27s+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+common+enzyme+connects+notch+signaling+and+Alzheimer%27s+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001479-Kidd1" id="pone.0001479-Kidd1"></a>Kidd S, Lieber T, Young MW (1998) Ligand-induced cleavage and regulation of nuclear entry of Notch in <em>Drosophila melanogaster</em> embryos. Genes Dev  12: 3728–3740.  <ul class="find" data-citedArticleID="1045662" data-doi="10.1101/gad.12.23.3728"><li><a href="http://dx.doi.org/10.1101/gad.12.23.3728" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ligand-induced+cleavage+and+regulation+of+nuclear+entry+of+Notch+in+Drosophila+melanogaster+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ligand-induced+cleavage+and+regulation+of+nuclear+entry+of+Notch+in+Drosophila+melanogaster+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001479-Kopan2" id="pone.0001479-Kopan2"></a>Kopan R, Schroeter EH, Weintraub H, Nye JS (1996) Signal transduction by activated mNotch: importance of proteolytic processing and its regulation by the extracellular domain. Proc Natl Acad Sci U S A  93: 1683–1688.  <ul class="find" data-citedArticleID="1045670" data-doi="10.1073/pnas.93.4.1683"><li><a href="http://dx.doi.org/10.1073/pnas.93.4.1683" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signal+transduction+by+activated+mNotch%3A+importance+of+proteolytic+processing+and+its+regulation+by+the+extracellular+domain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signal+transduction+by+activated+mNotch%3A+importance+of+proteolytic+processing+and+its+regulation+by+the+extracellular+domain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001479-Lecourtois1" id="pone.0001479-Lecourtois1"></a>Lecourtois M, Schweisguth F (1998) Indirect evidence for Delta-dependent intracellular processing of Notch in <em>Drosophila</em> embryos. Curr Biol  8: 771–774.  <ul class="find" data-citedArticleID="1045678" data-doi="10.1016/s0960-9822(98)70300-8"><li><a href="http://dx.doi.org/10.1016/s0960-9822(98)70300-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Indirect+evidence+for+Delta-dependent+intracellular+processing+of+Notch+in+Drosophila+embryos." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Indirect+evidence+for+Delta-dependent+intracellular+processing+of+Notch+in+Drosophila+embryos.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001479-Struhl1" id="pone.0001479-Struhl1"></a>Struhl G, Adachi A (1998) Nuclear access and action of Notch <em>in vivo</em>. Cell  93: 649–660.  <ul class="find" data-citedArticleID="1045734" data-doi="10.1016/s0092-8674(00)81193-9"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)81193-9" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Nuclear+access+and+action+of+Notch+in+vivo." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Nuclear+access+and+action+of+Notch+in+vivo.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001479-Jarriault1" id="pone.0001479-Jarriault1"></a>Jarriault S, Brou C, Logeat F, Schroeter EH, Kopan R, et al.  (1995) Signalling downstream of activated mammalian Notch. Nature  377: 355–358.  <ul class="find" data-citedArticleID="1045644" data-doi="10.1038/377355a0"><li><a href="http://dx.doi.org/10.1038/377355a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Signalling+downstream+of+activated+mammalian+Notch." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Signalling+downstream+of+activated+mammalian+Notch.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001479-Lu1" id="pone.0001479-Lu1"></a>Lu FM, Lux SE (1996) Constitutively active human Notch1 binds to the transcription factor CBF1 and stimulates transcription through a promoter containing a CBF1-responsive element. Proc Natl Acad Sci U S A  93: 5663–5667.  <ul class="find" data-citedArticleID="1045684" data-doi="10.1073/pnas.93.11.5663"><li><a href="http://dx.doi.org/10.1073/pnas.93.11.5663" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Constitutively+active+human+Notch1+binds+to+the+transcription+factor+CBF1+and+stimulates+transcription+through+a+promoter+containing+a+CBF1-responsive+element." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Constitutively+active+human+Notch1+binds+to+the+transcription+factor+CBF1+and+stimulates+transcription+through+a+promoter+containing+a+CBF1-responsive+element.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001479-Mumm1" id="pone.0001479-Mumm1"></a>Mumm JS, Kopan R (2000) Notch signaling: from the outside in. Dev Biol  228: 151–165.  <ul class="find" data-citedArticleID="1045698" data-doi="10.1006/dbio.2000.9960"><li><a href="http://dx.doi.org/10.1006/dbio.2000.9960" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+signaling%3A+from+the+outside+in." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+signaling%3A+from+the+outside+in.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001479-Bailey1" id="pone.0001479-Bailey1"></a>Bailey AM, Posakony JW (1995) Suppressor of hairless directly activates transcription of enhancer of split complex genes in response to Notch receptor activity. Genes Dev  9: 2609–2622.  <ul class="find" data-citedArticleID="1045592" data-doi="10.1101/gad.9.21.2609"><li><a href="http://dx.doi.org/10.1101/gad.9.21.2609" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Suppressor+of+hairless+directly+activates+transcription+of+enhancer+of+split+complex+genes+in+response+to+Notch+receptor+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Suppressor+of+hairless+directly+activates+transcription+of+enhancer+of+split+complex+genes+in+response+to+Notch+receptor+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001479-Bessho1" id="pone.0001479-Bessho1"></a>Bessho Y, Miyoshi G, Sakata R, Kageyama R (2001) <em>Hes7</em>: a bHLH-type repressor gene regulated by Notch and expressed in the presomitic mesoderm. Genes Cells  6: 175–185.  <ul class="find" data-citedArticleID="1045594" data-doi="10.1046/j.1365-2443.2001.00409.x"><li><a href="http://dx.doi.org/10.1046/j.1365-2443.2001.00409.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hes7%3A+a+bHLH-type+repressor+gene+regulated+by+Notch+and+expressed+in+the+presomitic+mesoderm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hes7%3A+a+bHLH-type+repressor+gene+regulated+by+Notch+and+expressed+in+the+presomitic+mesoderm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001479-Hirata1" id="pone.0001479-Hirata1"></a>Hirata H, Yoshiura S, Ohtsuka T, Bessho Y, Harada T, et al.  (2002) Oscillatory expression of the bHLH factor Hes1 regulated by a negative feedback loop. Science  298: 840–843.  <ul class="find" data-citedArticleID="1045628" data-doi="10.1126/science.1074560"><li><a href="http://dx.doi.org/10.1126/science.1074560" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Oscillatory+expression+of+the+bHLH+factor+Hes1+regulated+by+a+negative+feedback+loop." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Oscillatory+expression+of+the+bHLH+factor+Hes1+regulated+by+a+negative+feedback+loop.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001479-Holley1" id="pone.0001479-Holley1"></a>Holley SA, Jülich D, Rauch GJ, Geisler R, Nüsslein-Volhard C (2002) <em>her1</em> and the <em>notch</em> pathway function within the oscillator mechanism that regulates zebrafish somitogenesis. Development  129: 1175–1183.  <ul class="find" data-citedArticleID="1045630"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=her1+and+the+notch+pathway+function+within+the+oscillator+mechanism+that+regulates+zebrafish+somitogenesis.&amp;auth=&amp;atitle=her1+and+the+notch+pathway+function+within+the+oscillator+mechanism+that+regulates+zebrafish+somitogenesis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=her1+and+the+notch+pathway+function+within+the+oscillator+mechanism+that+regulates+zebrafish+somitogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22her1+and+the+notch+pathway+function+within+the+oscillator+mechanism+that+regulates+zebrafish+somitogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001479-Oates1" id="pone.0001479-Oates1"></a>Oates AC, Ho RK (2002) <em>Hairy</em>/<em>E(spl)-related</em> (<em>Her</em>) genes are central components of the segmentation oscillator and display redundancy with the Delta/Notch signaling pathway in the formation of anterior segmental boundaries in the zebrafish. Development  129: 2929–2946.  <ul class="find" data-citedArticleID="1045700"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Hairy%2FE%28spl%29-related+%28Her%29+genes+are+central+components+of+the+segmentation+oscillator+and+display+redundancy+with+the+Delta%2FNotch+signaling+pathway+in+the+formation+of+anterior+segmental+boundaries+in+the+zebrafish.&amp;auth=&amp;atitle=Hairy%2FE%28spl%29-related+%28Her%29+genes+are+central+components+of+the+segmentation+oscillator+and+display+redundancy+with+the+Delta%2FNotch+signaling+pathway+in+the+formation+of+anterior+segmental+boundaries+in+the+zebrafish." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hairy%2FE%28spl%29-related+%28Her%29+genes+are+central+components+of+the+segmentation+oscillator+and+display+redundancy+with+the+Delta%2FNotch+signaling+pathway+in+the+formation+of+anterior+segmental+boundaries+in+the+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hairy%2FE%28spl%29-related+%28Her%29+genes+are+central+components+of+the+segmentation+oscillator+and+display+redundancy+with+the+Delta%2FNotch+signaling+pathway+in+the+formation+of+anterior+segmental+boundaries+in+the+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001479-Ohtsuka1" id="pone.0001479-Ohtsuka1"></a>Ohtsuka T, Ishibashi M, Gradwohl G, Nakanishi S, Guillemot F, et al.  (1999) Hes1 and Hes5 as Notch effectors in mammalian neuronal differentiation. EMBO J  18: 2196–2207.  <ul class="find" data-citedArticleID="1045704" data-doi="10.1093/emboj/18.8.2196"><li><a href="http://dx.doi.org/10.1093/emboj/18.8.2196" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hes1+and+Hes5+as+Notch+effectors+in+mammalian+neuronal+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hes1+and+Hes5+as+Notch+effectors+in+mammalian+neuronal+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001479-Itoh1" id="pone.0001479-Itoh1"></a>Itoh M, Kim CH, Palardy G, Oda T, Jiang Y-J, et al.  (2003) Mind bomb is a ubiquitin ligase that is essential for efficient activation of Notch signaling by Delta. Dev Cell  4: 67–82.  <ul class="find" data-citedArticleID="1045642" data-doi="10.1016/s1534-5807(02)00409-4"><li><a href="http://dx.doi.org/10.1016/s1534-5807(02)00409-4" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mind+bomb+is+a+ubiquitin+ligase+that+is+essential+for+efficient+activation+of+Notch+signaling+by+Delta." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mind+bomb+is+a+ubiquitin+ligase+that+is+essential+for+efficient+activation+of+Notch+signaling+by+Delta.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001479-Lai1" id="pone.0001479-Lai1"></a>Lai EC (2002) Protein degradation: four E3s for the Notch pathway. Curr Biol  12: R74–R78.  <ul class="find" data-citedArticleID="1045672" data-doi="10.1016/s0960-9822(01)00679-0"><li><a href="http://dx.doi.org/10.1016/s0960-9822(01)00679-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Protein+degradation%3A+four+E3s+for+the+Notch+pathway." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Protein+degradation%3A+four+E3s+for+the+Notch+pathway.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001479-Jiang1" id="pone.0001479-Jiang1"></a>Jiang Y-J, Brand M, Heisenberg C-P, Beuchle D, Furutani-Seiki M, et al.  (1996) Mutations affecting neurogenesis and brain morphology in the zebrafish, <em>Danio rerio</em>. Development  123: 205–216.  <ul class="find" data-citedArticleID="1045652" data-doi="10.1016/0736-5748(96)80313-3"><li><a href="http://dx.doi.org/10.1016/0736-5748(96)80313-3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+affecting+neurogenesis+and+brain+morphology+in+the+zebrafish%2C+Danio+rerio." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+affecting+neurogenesis+and+brain+morphology+in+the+zebrafish%2C+Danio+rerio.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001479-vanEeden1" id="pone.0001479-vanEeden1"></a>van Eeden FJM, Granato M, Schach U, Brand M, Furutani-Seiki M, et al.  (1996) Mutations affecting somite formation and patterning in the zebrafish <em>Danio rerio</em>. Development  123: 153–164.  <ul class="find" data-citedArticleID="1045774"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mutations+affecting+somite+formation+and+patterning+in+the+zebrafish+Danio+rerio.&amp;auth=&amp;atitle=Mutations+affecting+somite+formation+and+patterning+in+the+zebrafish+Danio+rerio." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mutations+affecting+somite+formation+and+patterning+in+the+zebrafish+Danio+rerio." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mutations+affecting+somite+formation+and+patterning+in+the+zebrafish+Danio+rerio.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001479-Holley2" id="pone.0001479-Holley2"></a>Holley SA, Geisler R, Nüsslein-Volhard C (2000) Control of <em>her1</em> expression during zebrafish somitogenesis by a <em>Delta</em>-dependent oscillator and an independent wave-front activity. Genes Dev  14: 1678–1690.  <ul class="find" data-citedArticleID="1045632"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Control+of+her1+expression+during+zebrafish+somitogenesis+by+a+Delta-dependent+oscillator+and+an+independent+wave-front+activity.&amp;auth=&amp;atitle=Control+of+her1+expression+during+zebrafish+somitogenesis+by+a+Delta-dependent+oscillator+and+an+independent+wave-front+activity." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+her1+expression+during+zebrafish+somitogenesis+by+a+Delta-dependent+oscillator+and+an+independent+wave-front+activity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+her1+expression+during+zebrafish+somitogenesis+by+a+Delta-dependent+oscillator+and+an+independent+wave-front+activity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001479-Jlich1" id="pone.0001479-Jlich1"></a>Jülich D, Lim CH, Round J, Nicolaije C, Schroeder J, et al.  (2005) <em>beamter</em>/<em>deltaC</em> and the role of Notch ligands in the zebrafish somite segmentation, hindbrain neurogenesis and hypochord differentiation. Dev Biol  286: 391–404.  <ul class="find" data-citedArticleID="1045656" data-doi="10.1016/j.ydbio.2005.06.040"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.06.040" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=beamter%2FdeltaC+and+the+role+of+Notch+ligands+in+the+zebrafish+somite+segmentation%2C+hindbrain+neurogenesis+and+hypochord+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22beamter%2FdeltaC+and+the+role+of+Notch+ligands+in+the+zebrafish+somite+segmentation%2C+hindbrain+neurogenesis+and+hypochord+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001479-Rida1" id="pone.0001479-Rida1"></a>Rida PCG, Le Minh N, Jiang Y-J (2004) A Notch feeling of somite segmentation and beyond. Dev Biol  265: 2–22.  <ul class="find" data-citedArticleID="1045720" data-doi="10.1016/j.ydbio.2003.07.003"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2003.07.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+Notch+feeling+of+somite+segmentation+and+beyond." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+Notch+feeling+of+somite+segmentation+and+beyond.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001479-Zhang1" id="pone.0001479-Zhang1"></a>Zhang C, Li Q, Lim C-H, Qiu X, Jiang Y-J (2007) The characterization of zebrafish antimorphic <em>mib</em> alleles reveals that Mib and Mind bomb-2 (Mib2) function redundantly. Dev Biol  305: 14–27.  <ul class="find" data-citedArticleID="1045770" data-doi="10.1016/j.ydbio.2007.01.034"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2007.01.034" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+characterization+of+zebrafish+antimorphic+mib+alleles+reveals+that+Mib+and+Mind+bomb-2+%28Mib2%29+function+redundantly." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+characterization+of+zebrafish+antimorphic+mib+alleles+reveals+that+Mib+and+Mind+bomb-2+%28Mib2%29+function+redundantly.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001479-Zhang2" id="pone.0001479-Zhang2"></a>Zhang C, Li Q, Jiang Y-J (2007) Zebrafish Mib and Mib2 are mutual E3 ubiquitin ligases with common and specific Delta substrates. J Mol Biol  366: 1115–1128.  <ul class="find" data-citedArticleID="1045772" data-doi="10.1016/j.jmb.2006.11.096"><li><a href="http://dx.doi.org/10.1016/j.jmb.2006.11.096" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Zebrafish+Mib+and+Mib2+are+mutual+E3+ubiquitin+ligases+with+common+and+specific+Delta+substrates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Zebrafish+Mib+and+Mib2+are+mutual+E3+ubiquitin+ligases+with+common+and+specific+Delta+substrates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001479-Apelqvist1" id="pone.0001479-Apelqvist1"></a>Apelqvist Å, Li H, Sommer L, Beatus P, Anderson DJ, et al.  (1999) Notch signalling controls pancreatic cell differentiation. Nature  400: 877–881.  <ul class="find" data-citedArticleID="1045584" data-doi="10.1038/23716"><li><a href="http://dx.doi.org/10.1038/23716" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+signalling+controls+pancreatic+cell+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+signalling+controls+pancreatic+cell+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001479-Esni1" id="pone.0001479-Esni1"></a>Esni F, Ghosh B, Biankin AV, Lin JW, Albert MA, et al.  (2004) Notch inhibits Ptf1 function and acinar cell differentiation in developing mouse and zebrafish pancreas. Development  131: 4213–4224.  <ul class="find" data-citedArticleID="1045612" data-doi="10.1242/dev.01280"><li><a href="http://dx.doi.org/10.1242/dev.01280" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Notch+inhibits+Ptf1+function+and+acinar+cell+differentiation+in+developing+mouse+and+zebrafish+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Notch+inhibits+Ptf1+function+and+acinar+cell+differentiation+in+developing+mouse+and+zebrafish+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001479-Zecchin1" id="pone.0001479-Zecchin1"></a>Zecchin E, Mavropoulos A, Devos N, Filippi A, Tiso N, et al.  (2004) Evolutionary conserved role of ptf1a in the specification of exocrine pancreatic fates. Dev Biol  268: 174–184.  <ul class="find" data-citedArticleID="1045766" data-doi="10.1016/j.ydbio.2003.12.016"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2003.12.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Evolutionary+conserved+role+of+ptf1a+in+the+specification+of+exocrine+pancreatic+fates." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Evolutionary+conserved+role+of+ptf1a+in+the+specification+of+exocrine+pancreatic+fates.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001479-Yee1" id="pone.0001479-Yee1"></a>Yee NS, Lorent K, Pack M (2005) Exocrine pancreas development in zebrafish. Dev Biol  284: 84–101.  <ul class="find" data-citedArticleID="1045762" data-doi="10.1016/j.ydbio.2005.04.035"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.04.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exocrine+pancreas+development+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exocrine+pancreas+development+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001479-Zecchin2" id="pone.0001479-Zecchin2"></a>Zecchin E, Filippi A, Biemar F, Tiso N, Pauls S, et al.  (2007) Distinct <em>delta</em> and <em>jagged</em> genes control sequential segregation of pancreatic cell types from precursor pools in zebrafish. Dev Biol  301: 192–204.  <ul class="find" data-citedArticleID="1045768" data-doi="10.1016/j.ydbio.2006.09.041"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2006.09.041" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Distinct+delta+and+jagged+genes+control+sequential+segregation+of+pancreatic+cell+types+from+precursor+pools+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Distinct+delta+and+jagged+genes+control+sequential+segregation+of+pancreatic+cell+types+from+precursor+pools+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001479-Jensen1" id="pone.0001479-Jensen1"></a>Jensen J, Pedersen EE, Galante P, Hald J, Heller RS, et al.  (2000) Control of endodermal endocrine development by Hes-1. Nat Genet  24: 36–44.  <ul class="find" data-citedArticleID="1045646" data-doi="10.1038/71657"><li><a href="http://dx.doi.org/10.1038/71657" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Control+of+endodermal+endocrine+development+by+Hes-1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Control+of+endodermal+endocrine+development+by+Hes-1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001479-Gu1" id="pone.0001479-Gu1"></a>Gu G, Dubauskaite J, Melton DA (2002) Direct evidence for the pancreatic lineage: NGN3+ cells are islet progenitors and are distinct from duct progenitors. Development  129: 2447–2457.  <ul class="find" data-citedArticleID="1045618"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Direct+evidence+for+the+pancreatic+lineage%3A+NGN3%2B+cells+are+islet+progenitors+and+are+distinct+from+duct+progenitors.&amp;auth=&amp;atitle=Direct+evidence+for+the+pancreatic+lineage%3A+NGN3%2B+cells+are+islet+progenitors+and+are+distinct+from+duct+progenitors." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Direct+evidence+for+the+pancreatic+lineage%3A+NGN3%2B+cells+are+islet+progenitors+and+are+distinct+from+duct+progenitors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Direct+evidence+for+the+pancreatic+lineage%3A+NGN3%2B+cells+are+islet+progenitors+and+are+distinct+from+duct+progenitors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001479-Gradwohl1" id="pone.0001479-Gradwohl1"></a>Gradwohl G, Dierich A, LeMeur M, Guillemot F (2000) <em>neurogenin3</em> is required for the development of the four endocrine cell lineages of the pancreas. Proc Natl Acad Sci U S A  97: 1607–1611.  <ul class="find" data-citedArticleID="1045616" data-doi="10.1073/pnas.97.4.1607"><li><a href="http://dx.doi.org/10.1073/pnas.97.4.1607" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=neurogenin3+is+required+for+the+development+of+the+four+endocrine+cell+lineages+of+the+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22neurogenin3+is+required+for+the+development+of+the+four+endocrine+cell+lineages+of+the+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0001479-Pfaff1" id="pone.0001479-Pfaff1"></a>Pfaff SL, Mendelsohn M, Stewart CL, Edlund T, Jessell TM (1996) Requirement for LIM homeobox gene <em>Isl1</em> in motor neuron generation reveals a motor neuron-dependent step in interneuron differentiation. Cell  84: 309–320.  <ul class="find" data-citedArticleID="1045710" data-doi="10.1016/s0092-8674(00)80985-x"><li><a href="http://dx.doi.org/10.1016/s0092-8674(00)80985-x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Requirement+for+LIM+homeobox+gene+Isl1+in+motor+neuron+generation+reveals+a+motor+neuron-dependent+step+in+interneuron+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Requirement+for+LIM+homeobox+gene+Isl1+in+motor+neuron+generation+reveals+a+motor+neuron-dependent+step+in+interneuron+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0001479-Olbrot1" id="pone.0001479-Olbrot1"></a>Olbrot M, Rud J, Moss LG, Sharma A (2002) Identification of beta-cell-specific insulin gene transcription factor RIPE3b1 as mammalian MafA. Proc Natl Acad Sci U S A  99: 6737–6742.  <ul class="find" data-citedArticleID="1045706" data-doi="10.1073/pnas.102168499"><li><a href="http://dx.doi.org/10.1073/pnas.102168499" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+beta-cell-specific+insulin+gene+transcription+factor+RIPE3b1+as+mammalian+MafA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+beta-cell-specific+insulin+gene+transcription+factor+RIPE3b1+as+mammalian+MafA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0001479-SosaPineda1" id="pone.0001479-SosaPineda1"></a>Sosa-Pineda B, Chowdhury K, Torres M, Oliver G, Gruss P (1997) The <em>Pax4</em> gene is essential for differentiation of insulin-producing beta cells in the mammalian pancreas. Nature  386: 399–402.  <ul class="find" data-citedArticleID="1045730" data-doi="10.1038/386399a0"><li><a href="http://dx.doi.org/10.1038/386399a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+Pax4+gene+is+essential+for+differentiation+of+insulin-producing+beta+cells+in+the+mammalian+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+Pax4+gene+is+essential+for+differentiation+of+insulin-producing+beta+cells+in+the+mammalian+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0001479-StOnge1" id="pone.0001479-StOnge1"></a>St-Onge L, Sosa-Pineda B, Chowdhury K, Mansouri A, Gruss P (1997) <em>Pax6</em> is required for differentiation of glucagon-producing alpha-cells in mouse pancreas. Nature  387: 406–409.  <ul class="find" data-citedArticleID="1045732" data-doi="10.1038/387406a0"><li><a href="http://dx.doi.org/10.1038/387406a0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pax6+is+required+for+differentiation+of+glucagon-producing+alpha-cells+in+mouse+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pax6+is+required+for+differentiation+of+glucagon-producing+alpha-cells+in+mouse+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0001479-Schena1" id="pone.0001479-Schena1"></a>Schena M, Heller RA, Theriault TP, Konrad K, Lachenmeier E, et al.  (1998) Microarrays: biotechnology's discovery platform for functional genomics. Trends Biotechnol  16: 301–306.  <ul class="find" data-citedArticleID="1045724" data-doi="10.1016/s0167-7799(98)01219-0"><li><a href="http://dx.doi.org/10.1016/s0167-7799(98)01219-0" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Microarrays%3A+biotechnology%27s+discovery+platform+for+functional+genomics." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Microarrays%3A+biotechnology%27s+discovery+platform+for+functional+genomics.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">49.
              </span><a name="pone.0001479-Chiang1" id="pone.0001479-Chiang1"></a>Chiang MK, Melton DA (2003) Single-cell transcript analysis of pancreas development. Dev Cell  4: 383–393.  <ul class="find" data-citedArticleID="1045602" data-doi="10.1016/s1534-5807(03)00035-2"><li><a href="http://dx.doi.org/10.1016/s1534-5807(03)00035-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Single-cell+transcript+analysis+of+pancreas+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Single-cell+transcript+analysis+of+pancreas+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">50.
              </span><a name="pone.0001479-Gu2" id="pone.0001479-Gu2"></a>Gu G, Wells JM, Dombkowski D, Preffer F, Aronow B, et al.  (2004) Global expression analysis of gene regulatory pathways during endocrine pancreatic development. Development  131: 165–179.  <ul class="find" data-citedArticleID="1045620" data-doi="10.1242/dev.00921"><li><a href="http://dx.doi.org/10.1242/dev.00921" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+expression+analysis+of+gene+regulatory+pathways+during+endocrine+pancreatic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+expression+analysis+of+gene+regulatory+pathways+during+endocrine+pancreatic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">51.
              </span><a name="pone.0001479-Mellitzer1" id="pone.0001479-Mellitzer1"></a>Mellitzer G, Bonne S, Luco RF, Van De Casteele M, Lenne-Samuel N, et al.  (2006) IA1 is NGN3-dependent and essential for differentiation of the endocrine pancreas. EMBO J  25: 1344–1352.  <ul class="find" data-citedArticleID="1045696" data-doi="10.1038/sj.emboj.7601011"><li><a href="http://dx.doi.org/10.1038/sj.emboj.7601011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=IA1+is+NGN3-dependent+and+essential+for+differentiation+of+the+endocrine+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22IA1+is+NGN3-dependent+and+essential+for+differentiation+of+the+endocrine+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">52.
              </span><a name="pone.0001479-Mathavan1" id="pone.0001479-Mathavan1"></a>Mathavan S, Lee SG, Mak A, Miller LD, Murthy KR, et al.  (2005) Transcriptome analysis of zebrafish embryogenesis using microarrays. PLoS Genet  1: 260–276.  <ul class="find" data-citedArticleID="1045692" data-doi="10.1371/journal.pgen.0010029"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0010029" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+analysis+of+zebrafish+embryogenesis+using+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+analysis+of+zebrafish+embryogenesis+using+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">53.
              </span><a name="pone.0001479-Xu1" id="pone.0001479-Xu1"></a>Xu J, Srinivas BP, Tay SY, Mak A, Yu X, et al.  (2006) Genomewide expression profiling in the zebrafish embryo identifies target genes regulated by Hedgehog signaling during vertebrate development. Genetics  174: 735–752.  <ul class="find" data-citedArticleID="1045756" data-doi="10.1534/genetics.106.061523"><li><a href="http://dx.doi.org/10.1534/genetics.106.061523" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genomewide+expression+profiling+in+the+zebrafish+embryo+identifies+target+genes+regulated+by+Hedgehog+signaling+during+vertebrate+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genomewide+expression+profiling+in+the+zebrafish+embryo+identifies+target+genes+regulated+by+Hedgehog+signaling+during+vertebrate+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">54.
              </span><a name="pone.0001479-Riley1" id="pone.0001479-Riley1"></a>Riley BB, Chiang MY, Storch EM, Heck R, Buckles GR, et al.  (2004) Rhombomere boundaries are Wnt signaling centers that regulate metameric patterning in the zebrafish hindbrain. Dev Dyn  231: 278–291.  <ul class="find" data-citedArticleID="1045722" data-doi="10.1002/dvdy.20133"><li><a href="http://dx.doi.org/10.1002/dvdy.20133" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Rhombomere+boundaries+are+Wnt+signaling+centers+that+regulate+metameric+patterning+in+the+zebrafish+hindbrain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Rhombomere+boundaries+are+Wnt+signaling+centers+that+regulate+metameric+patterning+in+the+zebrafish+hindbrain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">55.
              </span><a name="pone.0001479-Amoyel1" id="pone.0001479-Amoyel1"></a>Amoyel M, Cheng YC, Jiang Y-J, Wilkinson DG (2005) Wnt1 regulates neurogenesis and mediates lateral inhibition of boundary cell specification in the zebrafish hindbrain. Development  132: 775–785.  <ul class="find" data-citedArticleID="1045582" data-doi="10.1242/dev.01616"><li><a href="http://dx.doi.org/10.1242/dev.01616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wnt1+regulates+neurogenesis+and+mediates+lateral+inhibition+of+boundary+cell+specification+in+the+zebrafish+hindbrain." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wnt1+regulates+neurogenesis+and+mediates+lateral+inhibition+of+boundary+cell+specification+in+the+zebrafish+hindbrain.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">56.
              </span><a name="pone.0001479-Park1" id="pone.0001479-Park1"></a>Park HC, Appel B (2003) Delta-Notch signaling regulates oligodendrocyte specification. Development  130: 3747–3755.  <ul class="find" data-citedArticleID="1045708" data-doi="10.1242/dev.00576"><li><a href="http://dx.doi.org/10.1242/dev.00576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Delta-Notch+signaling+regulates+oligodendrocyte+specification." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Delta-Notch+signaling+regulates+oligodendrocyte+specification.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">57.
              </span><a name="pone.0001479-Takke1" id="pone.0001479-Takke1"></a>Takke C, Dornseifer P, v Weizsäcker E, Campos-Ortega JA (1999) <em>her4</em>, a zebrafish homologue of the <em>Drosophila</em> neurogenic gene <em>E(spl)</em>, is a target of NOTCH signalling. Development  126: 1811–1821.  <ul class="find" data-citedArticleID="1045738"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=her4%2C+a+zebrafish+homologue+of+the+Drosophila+neurogenic+gene+E%28spl%29%2C+is+a+target+of+NOTCH+signalling.&amp;auth=&amp;atitle=her4%2C+a+zebrafish+homologue+of+the+Drosophila+neurogenic+gene+E%28spl%29%2C+is+a+target+of+NOTCH+signalling." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=her4%2C+a+zebrafish+homologue+of+the+Drosophila+neurogenic+gene+E%28spl%29%2C+is+a+target+of+NOTCH+signalling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22her4%2C+a+zebrafish+homologue+of+the+Drosophila+neurogenic+gene+E%28spl%29%2C+is+a+target+of+NOTCH+signalling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">58.
              </span><a name="pone.0001479-Bae1" id="pone.0001479-Bae1"></a>Bae YK, Shimizu T, Hibi M (2005) Patterning of proneuronal and inter-proneuronal domains by hairy- and enhancer of split-related genes in zebrafish neuroectoderm. Development  132: 1375–1385.  <ul class="find" data-citedArticleID="1045590" data-doi="10.1242/dev.01710"><li><a href="http://dx.doi.org/10.1242/dev.01710" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Patterning+of+proneuronal+and+inter-proneuronal+domains+by+hairy-+and+enhancer+of+split-related+genes+in+zebrafish+neuroectoderm." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Patterning+of+proneuronal+and+inter-proneuronal+domains+by+hairy-+and+enhancer+of+split-related+genes+in+zebrafish+neuroectoderm.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">59.
              </span><a name="pone.0001479-Shankaran1" id="pone.0001479-Shankaran1"></a>Shankaran SS, Sieger D, Schroter C, Czepe C, Pauly MC, et al.  (2007) Completing the set of h/E(spl) cyclic genes in zebrafish: <em>her12</em> and <em>her15</em> reveal novel modes of expression and contribute to the segmentation clock. Dev Biol  304: 615–632.  <ul class="find" data-citedArticleID="1045728" data-doi="10.1016/j.ydbio.2007.01.004"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2007.01.004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Completing+the+set+of+h%2FE%28spl%29+cyclic+genes+in+zebrafish%3A+her12+and+her15+reveal+novel+modes+of+expression+and+contribute+to+the+segmentation+clock." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Completing+the+set+of+h%2FE%28spl%29+cyclic+genes+in+zebrafish%3A+her12+and+her15+reveal+novel+modes+of+expression+and+contribute+to+the+segmentation+clock.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">60.
              </span><a name="pone.0001479-Appel1" id="pone.0001479-Appel1"></a>Appel B, Eisen JS (1998) Regulation of neuronal specification in the zebrafish spinal cord by Delta function. Development  125: 371–380.  <ul class="find" data-citedArticleID="1045586"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Regulation+of+neuronal+specification+in+the+zebrafish+spinal+cord+by+Delta+function.&amp;auth=&amp;atitle=Regulation+of+neuronal+specification+in+the+zebrafish+spinal+cord+by+Delta+function." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+neuronal+specification+in+the+zebrafish+spinal+cord+by+Delta+function." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+neuronal+specification+in+the+zebrafish+spinal+cord+by+Delta+function.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">61.
              </span><a name="pone.0001479-Tsutsumi1" id="pone.0001479-Tsutsumi1"></a>Tsutsumi M, Itoh M (2007) Novel transcript <em>nort</em> is a downstream target gene of the Notch signaling pathway in zebrafish. Gene Expr Patterns  7: 227–232.  <ul class="find" data-citedArticleID="1045746" data-doi="10.1016/j.modgep.2006.10.002"><li><a href="http://dx.doi.org/10.1016/j.modgep.2006.10.002" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Novel+transcript+nort+is+a+downstream+target+gene+of+the+Notch+signaling+pathway+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Novel+transcript+nort+is+a+downstream+target+gene+of+the+Notch+signaling+pathway+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">62.
              </span><a name="pone.0001479-Jlich2" id="pone.0001479-Jlich2"></a>Jülich D, Geisler R, Holley SA, Tübingen 2000 Screen Consortium (2005) Integrinα5 and Delta/Notch signaling have complementary spatiotemporal requirements during zebrafish somitogenesis. Dev Cell  8: 575–586.  <ul class="find" data-citedArticleID="1045658" data-doi="10.1016/j.devcel.2005.01.016"><li><a href="http://dx.doi.org/10.1016/j.devcel.2005.01.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Integrin%CE%B15+and+Delta%2FNotch+signaling+have+complementary+spatiotemporal+requirements+during+zebrafish+somitogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Integrin%CE%B15+and+Delta%2FNotch+signaling+have+complementary+spatiotemporal+requirements+during+zebrafish+somitogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">63.
              </span><a name="pone.0001479-Chong1" id="pone.0001479-Chong1"></a>Chong S-W, Jiang Y-J (2005) Off limits–integrins holding boundaries in somitogenesis. Trends Cell Biol  15: 453–457.  <ul class="find" data-citedArticleID="1045604" data-doi="10.1016/j.tcb.2005.07.008"><li><a href="http://dx.doi.org/10.1016/j.tcb.2005.07.008" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Off+limits%E2%80%93integrins+holding+boundaries+in+somitogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Off+limits%E2%80%93integrins+holding+boundaries+in+somitogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">64.
              </span><a name="pone.0001479-Irvin1" id="pone.0001479-Irvin1"></a>Irvin DK, Dhaka A, Hicks C, Weinmaster G, Kornblum HI (2003) Extrinsic and intrinsic factors governing cell fate in cortical progenitor cultures. Dev Neurosci  25: 162–172.  <ul class="find" data-citedArticleID="1045640" data-doi="10.1159/000072265"><li><a href="http://dx.doi.org/10.1159/000072265" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Extrinsic+and+intrinsic+factors+governing+cell+fate+in+cortical+progenitor+cultures." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Extrinsic+and+intrinsic+factors+governing+cell+fate+in+cortical+progenitor+cultures.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">65.
              </span><a name="pone.0001479-Kohyama1" id="pone.0001479-Kohyama1"></a>Kohyama J, Tokunaga A, Fujita Y, Miyoshi H, Nagai T, et al.  (2005) Visualization of spatiotemporal activation of Notch signaling: live monitoring and significance in neural development. Dev Biol  286: 311–325.  <ul class="find" data-citedArticleID="1045666" data-doi="10.1016/j.ydbio.2005.08.003"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2005.08.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Visualization+of+spatiotemporal+activation+of+Notch+signaling%3A+live+monitoring+and+significance+in+neural+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Visualization+of+spatiotemporal+activation+of+Notch+signaling%3A+live+monitoring+and+significance+in+neural+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">66.
              </span><a name="pone.0001479-Leake1" id="pone.0001479-Leake1"></a>Leake D, Asch WS, Canger AK, Schechter N (1999) Gefiltin in zebrafish embryos: sequential gene expression of two neurofilament proteins in retinal ganglion cells. Differentiation  65: 181–189.  <ul class="find" data-citedArticleID="1045676" data-doi="10.1046/j.1432-0436.1999.6540181.x"><li><a href="http://dx.doi.org/10.1046/j.1432-0436.1999.6540181.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gefiltin+in+zebrafish+embryos%3A+sequential+gene+expression+of+two+neurofilament+proteins+in+retinal+ganglion+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gefiltin+in+zebrafish+embryos%3A+sequential+gene+expression+of+two+neurofilament+proteins+in+retinal+ganglion+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">67.
              </span><a name="pone.0001479-Mally1" id="pone.0001479-Mally1"></a>Mally MI, Cirulli V, Hayek A, Otonkoski T (1996) ICA69 is expressed equally in the human endocrine and exocrine pancreas. Diabetologia  39: 474–480.  <ul class="find" data-citedArticleID="1045688" data-doi="10.1007/bf00400680"><li><a href="http://dx.doi.org/10.1007/bf00400680" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ICA69+is+expressed+equally+in+the+human+endocrine+and+exocrine+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ICA69+is+expressed+equally+in+the+human+endocrine+and+exocrine+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">68.
              </span><a name="pone.0001479-Hill1" id="pone.0001479-Hill1"></a>Hill DJ, Hogg J, Petrik J, Arany E, Han VK (1999) Cellular distribution and ontogeny of insulin-like growth factors (IGFs) and IGF binding protein messenger RNAs and peptides in developing rat pancreas. J Endocrinol  160: 305–317.  <ul class="find" data-citedArticleID="1045626" data-doi="10.1677/joe.0.1600305"><li><a href="http://dx.doi.org/10.1677/joe.0.1600305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cellular+distribution+and+ontogeny+of+insulin-like+growth+factors+%28IGFs%29+and+IGF+binding+protein+messenger+RNAs+and+peptides+in+developing+rat+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cellular+distribution+and+ontogeny+of+insulin-like+growth+factors+%28IGFs%29+and+IGF+binding+protein+messenger+RNAs+and+peptides+in+developing+rat+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">69.
              </span><a name="pone.0001479-Pleau1" id="pone.0001479-Pleau1"></a>Pleau JM, Throsby M, Esling A, Dardenne M (1997) Ontogeny of glutamic acid decarboxylase gene expression in the mouse pancreas. Biochem Biophys Res Commun  233: 227–230.  <ul class="find" data-citedArticleID="1045712" data-doi="10.1006/bbrc.1997.6432"><li><a href="http://dx.doi.org/10.1006/bbrc.1997.6432" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ontogeny+of+glutamic+acid+decarboxylase+gene+expression+in+the+mouse+pancreas." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ontogeny+of+glutamic+acid+decarboxylase+gene+expression+in+the+mouse+pancreas.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">70.
              </span><a name="pone.0001479-vanHaeften1" id="pone.0001479-vanHaeften1"></a>van Haeften TW, Twickler TB (2004) Insulin-like growth factors and pancreas beta cells. Eur J Clin Invest  34: 249–255.  <ul class="find" data-citedArticleID="1045776" data-doi="10.1111/j.1365-2362.2004.01337.x"><li><a href="http://dx.doi.org/10.1111/j.1365-2362.2004.01337.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Insulin-like+growth+factors+and+pancreas+beta+cells." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Insulin-like+growth+factors+and+pancreas+beta+cells.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">71.
              </span><a name="pone.0001479-Lowe1" id="pone.0001479-Lowe1"></a>Lowe AW, Olsen M, Hao Y, Lee SP, Taek Lee K, et al.  (2007) Gene expression patterns in pancreatic tumors, cells and tissues. PLoS ONE  2: e323.  <ul class="find" data-citedArticleID="1045682" data-doi="10.1371/journal.pone.0000323"><li><a href="http://dx.doi.org/10.1371/journal.pone.0000323" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+expression+patterns+in+pancreatic+tumors%2C+cells+and+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+expression+patterns+in+pancreatic+tumors%2C+cells+and+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">72.
              </span><a name="pone.0001479-Inoue1" id="pone.0001479-Inoue1"></a>Inoue A, Takahashi M, Hatta K, Hotta Y, Okamoto H (1994) Developmental regulation of <em>islet-1</em> mRNA expression during neuronal differentiation in embryonic zebrafish. Dev Dyn  199: 1–11.  <ul class="find" data-citedArticleID="1045638" data-doi="10.1002/aja.1001990102"><li><a href="http://dx.doi.org/10.1002/aja.1001990102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Developmental+regulation+of+islet-1+mRNA+expression+during+neuronal+differentiation+in+embryonic+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Developmental+regulation+of+islet-1+mRNA+expression+during+neuronal+differentiation+in+embryonic+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">73.
              </span><a name="pone.0001479-Hutchinson1" id="pone.0001479-Hutchinson1"></a>Hutchinson SA, Eisen JS (2006) Islet1 and Islet2 have equivalent abilities to promote motoneuron formation and to specify motoneuron subtype identity. Development  133: 2137–2147.  <ul class="find" data-citedArticleID="1045636" data-doi="10.1242/dev.02355"><li><a href="http://dx.doi.org/10.1242/dev.02355" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Islet1+and+Islet2+have+equivalent+abilities+to+promote+motoneuron+formation+and+to+specify+motoneuron+subtype+identity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Islet1+and+Islet2+have+equivalent+abilities+to+promote+motoneuron+formation+and+to+specify+motoneuron+subtype+identity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">74.
              </span><a name="pone.0001479-Takamiya1" id="pone.0001479-Takamiya1"></a>Takamiya M, Campos-Ortega JA (2006) Hedgehog signalling controls zebrafish neural keel morphogenesis via its level-dependent effects on neurogenesis. Dev Dyn  235: 978–997.  <ul class="find" data-citedArticleID="1045736" data-doi="10.1002/dvdy.20720"><li><a href="http://dx.doi.org/10.1002/dvdy.20720" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Hedgehog+signalling+controls+zebrafish+neural+keel+morphogenesis+via+its+level-dependent+effects+on+neurogenesis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Hedgehog+signalling+controls+zebrafish+neural+keel+morphogenesis+via+its+level-dependent+effects+on+neurogenesis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">75.
              </span><a name="pone.0001479-Thisse1" id="pone.0001479-Thisse1"></a>Thisse C, Thisse B (2005) High throughput expression analysis of ZF-models consortium clones. ZFIN Direct Data Submission.   <ul class="find" data-citedArticleID="1045740"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=High+throughput+expression+analysis+of+ZF-models+consortium+clones.&amp;auth=&amp;atitle=High+throughput+expression+analysis+of+ZF-models+consortium+clones." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=High+throughput+expression+analysis+of+ZF-models+consortium+clones." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22High+throughput+expression+analysis+of+ZF-models+consortium+clones.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">76.
              </span><a name="pone.0001479-Hans1" id="pone.0001479-Hans1"></a>Hans S, Scheer N, Riedl I, v Weizsäcker E, Blader P, et al.  (2004) <em>her3</em>, a zebrafish member of the <em>hairy-E(spl)</em> family, is repressed by Notch signalling. Development  131: 2957–2969.  <ul class="find" data-citedArticleID="1045624" data-doi="10.1242/dev.01167"><li><a href="http://dx.doi.org/10.1242/dev.01167" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=her3%2C+a+zebrafish+member+of+the+hairy-E%28spl%29+family%2C+is+repressed+by+Notch+signalling." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22her3%2C+a+zebrafish+member+of+the+hairy-E%28spl%29+family%2C+is+repressed+by+Notch+signalling.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">77.
              </span><a name="pone.0001479-Kim1" id="pone.0001479-Kim1"></a>Kim HJ, Schleiffarth JR, Jessurun J, Sumanas S, Petryk A, et al.  (2005) Wnt5 signaling in vertebrate pancreas development. BMC Biol  3: 23.  <ul class="find" data-citedArticleID="1045664" data-doi="10.1186/1741-7007-3-23"><li><a href="http://dx.doi.org/10.1186/1741-7007-3-23" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Wnt5+signaling+in+vertebrate+pancreas+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Wnt5+signaling+in+vertebrate+pancreas+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">78.
              </span><a name="pone.0001479-Willer1" id="pone.0001479-Willer1"></a>Willer GB, Lee VM, Gregg RG, Link BA (2005) Analysis of the zebrafish <em>perplexed</em> mutation reveals tissue-specific roles for <em>de novo</em> pyrimidine synthesis during development. Genetics  170: 1827–1837.  <ul class="find" data-citedArticleID="1045754" data-doi="10.1534/genetics.105.041608"><li><a href="http://dx.doi.org/10.1534/genetics.105.041608" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analysis+of+the+zebrafish+perplexed+mutation+reveals+tissue-specific+roles+for+de+novo+pyrimidine+synthesis+during+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analysis+of+the+zebrafish+perplexed+mutation+reveals+tissue-specific+roles+for+de+novo+pyrimidine+synthesis+during+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">79.
              </span><a name="pone.0001479-Thisse2" id="pone.0001479-Thisse2"></a>Thisse B, Pflumio S, Fürthauer M, Loppin B, Heyer V, et al.  (2001) Expression of the zebrafish genome during embryogenesis (NIH R01 RR15402). ZFIN Direct Data Submission.   <ul class="find" data-citedArticleID="1045742"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+of+the+zebrafish+genome+during+embryogenesis+%28NIH+R01+RR15402%29.&amp;auth=&amp;atitle=Expression+of+the+zebrafish+genome+during+embryogenesis+%28NIH+R01+RR15402%29." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+of+the+zebrafish+genome+during+embryogenesis+%28NIH+R01+RR15402%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+of+the+zebrafish+genome+during+embryogenesis+%28NIH+R01+RR15402%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">80.
              </span><a name="pone.0001479-Wang1" id="pone.0001479-Wang1"></a>Wang Y, Barbacioru C, Hyland F, Xiao W, Hunkapiller KL, et al.  (2006) Large scale real-time PCR validation on gene expression measurements from two commercial long-oligonucleotide microarrays. BMC Genomics  7: 59.  <ul class="find" data-citedArticleID="1045752"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Large+scale+real-time+PCR+validation+on+gene+expression+measurements+from+two+commercial+long-oligonucleotide+microarrays.&amp;auth=&amp;atitle=Large+scale+real-time+PCR+validation+on+gene+expression+measurements+from+two+commercial+long-oligonucleotide+microarrays." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large+scale+real-time+PCR+validation+on+gene+expression+measurements+from+two+commercial+long-oligonucleotide+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large+scale+real-time+PCR+validation+on+gene+expression+measurements+from+two+commercial+long-oligonucleotide+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">81.
              </span><a name="pone.0001479-Lam1" id="pone.0001479-Lam1"></a>Lam SH, Winata CL, Tong Y, Korzh S, Lim WS, et al.  (2006) Transcriptome kinetics of arsenic-induced adaptive response in zebrafish liver. Physiol Genomics  27: 351–361.  <ul class="find" data-citedArticleID="1045674" data-doi="10.1152/physiolgenomics.00201.2005"><li><a href="http://dx.doi.org/10.1152/physiolgenomics.00201.2005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptome+kinetics+of+arsenic-induced+adaptive+response+in+zebrafish+liver." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptome+kinetics+of+arsenic-induced+adaptive+response+in+zebrafish+liver.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">82.
              </span><a name="pone.0001479-Jensen2" id="pone.0001479-Jensen2"></a>Jensen J, Heller RS, Funder-Nielsen T, Pedersen EE, Lindsell C, et al.  (2000) Independent development of pancreatic alpha- and beta-cells from <em>neurogenin3</em>-expressing precursors: a role for the Notch pathway in repression of premature differentiation. Diabetes  49: 163–176.  <ul class="find" data-citedArticleID="1045648" data-doi="10.2337/diabetes.49.2.163"><li><a href="http://dx.doi.org/10.2337/diabetes.49.2.163" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Independent+development+of+pancreatic+alpha-+and+beta-cells+from+neurogenin3-expressing+precursors%3A+a+role+for+the+Notch+pathway+in+repression+of+premature+differentiation." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Independent+development+of+pancreatic+alpha-+and+beta-cells+from+neurogenin3-expressing+precursors%3A+a+role+for+the+Notch+pathway+in+repression+of+premature+differentiation.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">83.
              </span><a name="pone.0001479-Habener1" id="pone.0001479-Habener1"></a>Habener JF, Kemp DM, Thomas MK (2005) Minireview: transcriptional regulation in pancreatic development. Endocrinology  146: 1025–1034.  <ul class="find" data-citedArticleID="1045622" data-doi="10.1210/en.2004-1576"><li><a href="http://dx.doi.org/10.1210/en.2004-1576" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Minireview%3A+transcriptional+regulation+in+pancreatic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Minireview%3A+transcriptional+regulation+in+pancreatic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">84.
              </span><a name="pone.0001479-Jensen3" id="pone.0001479-Jensen3"></a>Jensen J (2004) Gene regulatory factors in pancreatic development. Dev Dyn  229: 176–200.  <ul class="find" data-citedArticleID="1045650" data-doi="10.1002/dvdy.10460"><li><a href="http://dx.doi.org/10.1002/dvdy.10460" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Gene+regulatory+factors+in+pancreatic+development." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Gene+regulatory+factors+in+pancreatic+development.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">85.
              </span><a name="pone.0001479-Jin1" id="pone.0001479-Jin1"></a>Jin Y, Blue EK, Dixon S, Shao Z, Gallagher PJ (2002) A death-associated protein kinase (DAPK)-interacting protein, DIP-1, is an E3 ubiquitin ligase that promotes tumor necrosis factor-induced apoptosis and regulates the cellular levels of DAPK. J Biol Chem  277: 46980–46986.  <ul class="find" data-citedArticleID="1045654" data-doi="10.1074/jbc.m208585200"><li><a href="http://dx.doi.org/10.1074/jbc.m208585200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+death-associated+protein+kinase+%28DAPK%29-interacting+protein%2C+DIP-1%2C+is+an+E3+ubiquitin+ligase+that+promotes+tumor+necrosis+factor-induced+apoptosis+and+regulates+the+cellular+levels+of+DAPK." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+death-associated+protein+kinase+%28DAPK%29-interacting+protein%2C+DIP-1%2C+is+an+E3+ubiquitin+ligase+that+promotes+tumor+necrosis+factor-induced+apoptosis+and+regulates+the+cellular+levels+of+DAPK.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">86.
              </span><a name="pone.0001479-Ma1" id="pone.0001479-Ma1"></a>Ma M, Jiang Y-J (2007) Jagged2a-Notch signaling mediates cell fate choice in zebrafish pronephric duct. PLoS Genet  3: e18.  <ul class="find" data-citedArticleID="1045686" data-doi="10.1371/journal.pgen.0030018"><li><a href="http://dx.doi.org/10.1371/journal.pgen.0030018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Jagged2a-Notch+signaling+mediates+cell+fate+choice+in+zebrafish+pronephric+duct." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Jagged2a-Notch+signaling+mediates+cell+fate+choice+in+zebrafish+pronephric+duct.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">87.
              </span><a name="pone.0001479-Yoo1" id="pone.0001479-Yoo1"></a>Yoo KW, Kim EH, Jung SH, Rhee M, Koo BK, et al.  (2006) Snx5, as a Mind bomb-binding protein, is expressed in hematopoietic and endothelial precursor cells in zebrafish. FEBS Lett  580: 4409–4416.  <ul class="find" data-citedArticleID="1045764" data-doi="10.1016/j.febslet.2006.07.009"><li><a href="http://dx.doi.org/10.1016/j.febslet.2006.07.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Snx5%2C+as+a+Mind+bomb-binding+protein%2C+is+expressed+in+hematopoietic+and+endothelial+precursor+cells+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Snx5%2C+as+a+Mind+bomb-binding+protein%2C+is+expressed+in+hematopoietic+and+endothelial+precursor+cells+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">88.
              </span><a name="pone.0001479-Qiu1" id="pone.0001479-Qiu1"></a>Qiu L, Joazeiro C, Fang N, Wang HY, Elly C, et al.  (2000) Recognition and ubiquitination of Notch by Itch, a hect-type E3 ubiquitin ligase. J Biol Chem  275: 35734–35737.  <ul class="find" data-citedArticleID="1045716" data-doi="10.1074/jbc.m007300200"><li><a href="http://dx.doi.org/10.1074/jbc.m007300200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Recognition+and+ubiquitination+of+Notch+by+Itch%2C+a+hect-type+E3+ubiquitin+ligase." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Recognition+and+ubiquitination+of+Notch+by+Itch%2C+a+hect-type+E3+ubiquitin+ligase.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">89.
              </span><a name="pone.0001479-Matesic1" id="pone.0001479-Matesic1"></a>Matesic LE, Haines DC, Copeland NG, Jenkins NA (2006) Itch genetically interacts with Notch1 in a mouse autoimmune disease model. Hum Mol Genet  15: 3485–3497.  <ul class="find" data-citedArticleID="1045690" data-doi="10.1093/hmg/ddl425"><li><a href="http://dx.doi.org/10.1093/hmg/ddl425" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Itch+genetically+interacts+with+Notch1+in+a+mouse+autoimmune+disease+model." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Itch+genetically+interacts+with+Notch1+in+a+mouse+autoimmune+disease+model.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">90.
              </span><a name="pone.0001479-Chang1" id="pone.0001479-Chang1"></a>Chang L, Kamata H, Solinas G, Luo JL, Maeda S, et al.  (2006) The E3 ubiquitin ligase Itch couples JNK activation to TNFα-induced cell death by inducing c-FLIP<sub>L</sub> turnover. Cell  124: 601–613.  <ul class="find" data-citedArticleID="1045600" data-doi="10.1016/j.cell.2006.01.021"><li><a href="http://dx.doi.org/10.1016/j.cell.2006.01.021" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+E3+ubiquitin+ligase+Itch+couples+JNK+activation+to+TNF%CE%B1-induced+cell+death+by+inducing+c-FLIPL+turnover." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+E3+ubiquitin+ligase+Itch+couples+JNK+activation+to+TNF%CE%B1-induced+cell+death+by+inducing+c-FLIPL+turnover.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">91.
              </span><a name="pone.0001479-Schfer1" id="pone.0001479-Schfer1"></a>Schäfer M, Kinzel D, Winkler C (2007) Discontinuous organization and specification of the lateral floor plate in zebrafish. Dev Biol  301: 117–129.  <ul class="find" data-citedArticleID="1045726" data-doi="10.1016/j.ydbio.2006.09.018"><li><a href="http://dx.doi.org/10.1016/j.ydbio.2006.09.018" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Discontinuous+organization+and+specification+of+the+lateral+floor+plate+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Discontinuous+organization+and+specification+of+the+lateral+floor+plate+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">92.
              </span><a name="pone.0001479-Qiu2" id="pone.0001479-Qiu2"></a>Qiu X, Xu H, Haddon C, Lewis J, Jiang Y-J (2004) Sequence and embryonic expression of three zebrafish <em>fringe</em> genes, <em>lunatic fringe</em>, <em>radical fringe</em>, and <em>manic fringe</em>. Dev Dyn  231: 621–630.  <ul class="find" data-citedArticleID="1045718" data-doi="10.1002/dvdy.20155"><li><a href="http://dx.doi.org/10.1002/dvdy.20155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+embryonic+expression+of+three+zebrafish+fringe+genes%2C+lunatic+fringe%2C+radical+fringe%2C+and+manic+fringe." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+embryonic+expression+of+three+zebrafish+fringe+genes%2C+lunatic+fringe%2C+radical+fringe%2C+and+manic+fringe.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">93.
              </span><a name="pone.0001479-Wan1" id="pone.0001479-Wan1"></a>Wan H, Korzh S, Li Z, Mudumana SP, Korzh V, et al.  (2006) Analyses of pancreas development by generation of <em>gfp</em> transgenic zebrafish using an exocrine pancreas-specific <em>elastaseA</em> gene promoter. Exp Cell Res  312: 1526–1539.  <ul class="find" data-citedArticleID="1045750" data-doi="10.1016/j.yexcr.2006.01.016"><li><a href="http://dx.doi.org/10.1016/j.yexcr.2006.01.016" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Analyses+of+pancreas+development+by+generation+of+gfp+transgenic+zebrafish+using+an+exocrine+pancreas-specific+elastaseA+gene+promoter." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Analyses+of+pancreas+development+by+generation+of+gfp+transgenic+zebrafish+using+an+exocrine+pancreas-specific+elastaseA+gene+promoter.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">94.
              </span><a name="pone.0001479-Huang1" id="pone.0001479-Huang1"></a>Huang H, Liu N, Lin S (2001) Pdx-1 knockdown reduces insulin promoter activity in zebrafish. Genesis  30: 134–136.  <ul class="find" data-citedArticleID="1045634" data-doi="10.1002/gene.1048"><li><a href="http://dx.doi.org/10.1002/gene.1048" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pdx-1+knockdown+reduces+insulin+promoter+activity+in+zebrafish." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pdx-1+knockdown+reduces+insulin+promoter+activity+in+zebrafish.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">95.
              </span><a name="pone.0001479-Eisen1" id="pone.0001479-Eisen1"></a>Eisen MB, Brown PO (1999) DNA arrays for analysis of gene expression. Methods Enzymol  303: 179–205.  <ul class="find" data-citedArticleID="1045606" data-doi="10.1016/s0076-6879(99)03014-1"><li><a href="http://dx.doi.org/10.1016/s0076-6879(99)03014-1" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=DNA+arrays+for+analysis+of+gene+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22DNA+arrays+for+analysis+of+gene+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">96.
              </span><a name="pone.0001479-Yang1" id="pone.0001479-Yang1"></a>Yang H, Haddad H, Tomas C, Alsaker K, Papoutsakis ET (2003) A segmental nearest neighbor normalization and gene identification method gives superior results for DNA-array analysis. Proc Natl Acad Sci U S A  100: 1122–1127.  <ul class="find" data-citedArticleID="1045760" data-doi="10.1073/pnas.0237337100"><li><a href="http://dx.doi.org/10.1073/pnas.0237337100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+segmental+nearest+neighbor+normalization+and+gene+identification+method+gives+superior+results+for+DNA-array+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+segmental+nearest+neighbor+normalization+and+gene+identification+method+gives+superior+results+for+DNA-array+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">97.
              </span><a name="pone.0001479-Tusher1" id="pone.0001479-Tusher1"></a>Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci U S A  98: 5116–5121.  <ul class="find" data-citedArticleID="1045748" data-doi="10.1073/pnas.091062498"><li><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">98.
              </span><a name="pone.0001479-Eisen2" id="pone.0001479-Eisen2"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863–14868.  <ul class="find" data-citedArticleID="1045608" data-doi="10.1073/pnas.95.25.14863"><li><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cluster+analysis+and+display+of+genome-wide+expression+patterns." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cluster+analysis+and+display+of+genome-wide+expression+patterns.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.XML" value="219710"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.PDF" value="512219"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g001.PNG_L" value="2919088"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g001.PNG_M" value="285589"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g001.PNG_S" value="22629"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g001.TIF" value="3621708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g001.PNG_I" value="196050"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g002.PNG_L" value="3360790"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g002.PNG_M" value="153514"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g002.PNG_S" value="10712"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g002.TIF" value="4019722"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g002.PNG_I" value="48792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g003.PNG_L" value="413077"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g003.PNG_M" value="149810"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g003.PNG_S" value="19470"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g003.TIF" value="669262"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g003.PNG_I" value="70934"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g004.PNG_L" value="365553"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g004.PNG_M" value="116795"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g004.PNG_S" value="21599"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g004.TIF" value="1782504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g004.PNG_I" value="156752"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t001.PNG_L" value="65063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t001.PNG_M" value="87129"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t001.PNG_S" value="13090"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t001.TIF" value="290504"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t001.PNG_I" value="29274"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g005.PNG_L" value="710905"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g005.PNG_M" value="109626"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g005.PNG_S" value="13034"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g005.TIF" value="1060210"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g005.PNG_I" value="36850"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g006.PNG_L" value="416950"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g006.PNG_M" value="73624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g006.PNG_S" value="11498"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g006.TIF" value="1533726"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g006.PNG_I" value="26091"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t002.PNG_L" value="191572"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t002.PNG_M" value="106558"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t002.PNG_S" value="13023"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t002.TIF" value="836304"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t002.PNG_I" value="32551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t003.PNG_L" value="316663"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t003.PNG_M" value="178568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t003.PNG_S" value="17404"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t003.TIF" value="1355106"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t003.PNG_I" value="53735"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t004.PNG_L" value="166025"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t004.PNG_M" value="93698"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t004.PNG_S" value="11898"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t004.TIF" value="726426"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t004.PNG_I" value="28892"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g007.PNG_L" value="2237035"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g007.PNG_M" value="167503"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g007.PNG_S" value="20834"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g007.TIF" value="3415866"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.g007.PNG_I" value="165920"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t005.PNG_L" value="301200"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t005.PNG_M" value="165282"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t005.PNG_S" value="16251"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t005.TIF" value="1371410"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.t005.PNG_I" value="50334"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s001.XLS" value="1754624"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s002.XLS" value="3116032"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s003.XLS" value="3916288"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s004.XLS" value="88576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s005.XLS" value="86016"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s006.XLS" value="520192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s007.XLS" value="931328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s008.XLS" value="35328"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s009.XLS" value="79360"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s010.XLS" value="91136"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s011.XLS" value="376832"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s012.XLS" value="45568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s013.XLS" value="1008128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s014.XLS" value="33792"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s015.XLS" value="1072640"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s016.XLS" value="24576"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s017.XLS" value="20992"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s018.DOC" value="93184"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s019.XLS" value="16896"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s020.XLS" value="8192"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001479.s021.TIF" value="1710330"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001479&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001479" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001479&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001479&volume=&issue=&title=Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes&author_name=Ashok%20Hegde%2C%20Nick%20Chuanxin%20Qiu%2C%20Xuehui%20Qiu%2C%20Steven%20Hao-Kee%20Ho%2C%20Kenny%20Qi-Ye%20Tay%2C%20Joshy%20George%2C%20Felicia%20Soo%20Lee%20Ng%2C%20Kunde%20Ramamoorthy%20Govindarajan%2C%20Zhiyuan%20Gong%2C%20Sinnakaruppan%20Mathavan%2C%20Yun-Jin%20Jiang&start_page=1&end_page=15" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001479" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479&amp;t=Genomewide%20Expression%20Analysis%20in%20Zebrafish%20mind%20bomb%20Alleles%20with%20Pancreas%20Defects%20of%20Different%20Severity%20Identifies%20Putative%20Notch%20Responsive%20Genes" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479&title=Genomewide%20Expression%20Analysis%20in%20Zebrafish%20mind%20bomb%20Alleles%20with%20Pancreas%20Defects%20of%20Different%20Severity%20Identifies%20Putative%20Notch%20Responsive%20Genes&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479&amp;title=Genomewide%20Expression%20Analysis%20in%20Zebrafish%20mind%20bomb%20Alleles%20with%20Pancreas%20Defects%20of%20Different%20Severity%20Identifies%20Putative%20Notch%20Responsive%20Genes" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001479&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Genomewide Expression Analysis in Zebrafish mind bomb Alleles with Pancreas Defects of Different Severity Identifies Putative Notch Responsive Genes';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001479';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Genomewide%20Expression%20Analysis%20in%20Zebrafish%20mind%20bomb%20Alleles%20with%20Pancreas%20Defects%20of%20Different%20Severity%20Identifies%20Putative%20Notch%20Responsive%20Genes http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001479" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001479" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">












          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alleles%22" title="Search for articles in the subject area:'Alleles'"><div class="flagText">Alleles</div></a>
              <div data-categoryid="46071" data-articleid="25786"
                   data-categoryname="Alleles"
                   class="flagImage" title="Flag 'Alleles' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Embryos%22" title="Search for articles in the subject area:'Embryos'"><div class="flagText">Embryos</div></a>
              <div data-categoryid="17321" data-articleid="25786"
                   data-categoryname="Embryos"
                   class="flagImage" title="Flag 'Embryos' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Gene+expression%22" title="Search for articles in the subject area:'Gene expression'"><div class="flagText">Gene expression</div></a>
              <div data-categoryid="17515" data-articleid="25786"
                   data-categoryname="Gene expression"
                   class="flagImage" title="Flag 'Gene expression' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Microarrays%22" title="Search for articles in the subject area:'Microarrays'"><div class="flagText">Microarrays</div></a>
              <div data-categoryid="17255" data-articleid="25786"
                   data-categoryname="Microarrays"
                   class="flagImage" title="Flag 'Microarrays' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Notch+signaling%22" title="Search for articles in the subject area:'Notch signaling'"><div class="flagText">Notch signaling</div></a>
              <div data-categoryid="17927" data-articleid="25786"
                   data-categoryname="Notch signaling"
                   class="flagImage" title="Flag 'Notch signaling' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pancreas%22" title="Search for articles in the subject area:'Pancreas'"><div class="flagText">Pancreas</div></a>
              <div data-categoryid="47467" data-articleid="25786"
                   data-categoryname="Pancreas"
                   class="flagImage" title="Flag 'Pancreas' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Pancreas+development%22" title="Search for articles in the subject area:'Pancreas development'"><div class="flagText">Pancreas development</div></a>
              <div data-categoryid="44829" data-articleid="25786"
                   data-categoryname="Pancreas development"
                   class="flagImage" title="Flag 'Pancreas development' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Zebrafish%22" title="Search for articles in the subject area:'Zebrafish'"><div class="flagText">Zebrafish</div></a>
              <div data-categoryid="32787" data-articleid="25786"
                   data-categoryname="Zebrafish"
                   class="flagImage" title="Flag 'Zebrafish' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=2721'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=1292'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5467&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
