

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0000294"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0000294" />

    <meta name="citation_title" content="Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome"/>
    <meta itemprop="name" content="Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome"/>

      <meta name="citation_author" content="Lei Li"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Xiangfeng Wang"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America"/>
            <meta name="citation_author_institution" content="National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing, China"/>
            <meta name="citation_author_institution" content="Peking-Yale Joint Research Center of Plant Molecular Genetics and Agrobiotechnology, College of Life Sciences, Peking University, Beijing, China"/>
      <meta name="citation_author" content="Rajkumar Sasidharan"/>
            <meta name="citation_author_institution" content="Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Viktor Stolc"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America"/>
            <meta name="citation_author_institution" content="Genome Research Facility, NASA Ames Research Center, Moffett Field, California, United States of America"/>
      <meta name="citation_author" content="Wei Deng"/>
            <meta name="citation_author_institution" content="National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing, China"/>
            <meta name="citation_author_institution" content="Bioinformatics Laboratory, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Hang He"/>
            <meta name="citation_author_institution" content="National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing, China"/>
            <meta name="citation_author_institution" content="Bioinformatics Laboratory, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Jan Korbel"/>
            <meta name="citation_author_institution" content="Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Xuewei Chen"/>
            <meta name="citation_author_institution" content="Department of Plant Pathology, University of California, Davis, California, United States of America"/>
      <meta name="citation_author" content="Waraporn Tongprasit"/>
            <meta name="citation_author_institution" content="Eloret Corporation, Sunnyvale, California, United States of America"/>
      <meta name="citation_author" content="Pamela Ronald"/>
            <meta name="citation_author_institution" content="Department of Plant Pathology, University of California, Davis, California, United States of America"/>
      <meta name="citation_author" content="Runsheng Chen"/>
            <meta name="citation_author_institution" content="Bioinformatics Laboratory, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China"/>
      <meta name="citation_author" content="Mark Gerstein"/>
            <meta name="citation_author_institution" content="Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut, United States of America"/>
      <meta name="citation_author" content="Xing Wang Deng"/>
            <meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America"/>

    <meta name="citation_date" content="2007/3/14"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0000294.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e294"/>
    <meta name="citation_issue" content="3"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp japonica).; citation_author=SA Goff; citation_author=D Ricke; citation_author=TH Lan; citation_author=G Presting; citation_author=R Wang; citation_journal_title=Science; citation_volume=296; citation_number=1; citation_pages=92-100; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp. indica).; citation_author=J Yu; citation_author=S Hu; citation_author=J Wang; citation_author=GK Wong; citation_author=S Li; citation_journal_title=Science; citation_volume=296; citation_number=2; citation_pages=79-92; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=The genomes of Oryza sativa: A history of duplications.; citation_author=J Yu; citation_author=J Wang; citation_author=W Lin; citation_author=S Li; citation_author=H Li; citation_journal_title=PLOS Biol; citation_volume=3; citation_number=3; citation_pages=e38; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The genome sequence and structure of rice chromosome 1.; citation_author=T Sasaki; citation_author=T Matsumoto; citation_author=K Yamamoto; citation_author=K Sakata; citation_author=T Baba; citation_journal_title=Nature; citation_volume=420; citation_number=4; citation_pages=312-316; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Sequence and analysis of rice chromosome 4.; citation_author=Q Feng; citation_author=Y Zhang; citation_author=P Hao; citation_author=S Wang; citation_author=G Fu; citation_journal_title=Nature; citation_volume=420; citation_number=5; citation_pages=316-320; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=In-depth view of structure, activity, and evolution of rice Chromosome 10.; citation_journal_title=Science; citation_volume=300; citation_number=6; citation_pages=1566-1569; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome.; citation_journal_title=Nature; citation_volume=436; citation_number=7; citation_pages=793-800; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=The institute for genomic research osa1 rice genome annotation database.; citation_author=Q Yuan; citation_author=S Ouyang; citation_author=A Wang; citation_author=W Zhu; citation_author=R Maiti; citation_journal_title=Plant Physiol; citation_volume=138; citation_number=8; citation_pages=18-26; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice.; citation_author=S Kikuchi; citation_author=K Satoh; citation_author=T Nagata; citation_author=N Kawagashira; citation_author=K Doi; citation_journal_title=Science; citation_volume=300; citation_number=9; citation_pages=1566-1569; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Transposable element annotation of the rice genome.; citation_author=N Juretic; citation_author=TE Bureau; citation_author=RM Bruskiewich; citation_journal_title=Bioinformatics; citation_volume=20; citation_number=10; citation_pages=155-160; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcript profiling in rice (Oryza sativa L.) seedlings using serial analysis of gene expression (SAGE).; citation_author=H Matsumura; citation_author=S Nirasawa; citation_author=R Terauchi; citation_journal_title=Plant J; citation_volume=20; citation_number=11; citation_pages=719-726; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Plant MPSS databases: signature-based transcriptional resources for analyses of mRNA and small RNA.; citation_author=M Nakano; citation_author=K Nobuta; citation_author=K Vemaraju; citation_author=SS Tej; citation_author=JW Skogen; citation_journal_title=Nucleic Acids Res; citation_volume=34; citation_number=12; citation_pages=D731-D735; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=SuperSAGE array: the direct use of 26-base-pair transcript tags in oligonucleotide arrays.; citation_author=H Matsumura; citation_author=KH Bin Nasir; citation_author=K Yoshida; citation_author=A Ito; citation_author=G Kahl; citation_journal_title=Nat Methods; citation_volume=3; citation_number=13; citation_pages=469-474; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Tiling microarray analysis of rice chromosome 10 to identify the transcriptome and relate its expression to chromosomal architecture.; citation_author=L Li; citation_author=X Wang; citation_author=M Xia; citation_author=V Stolc; citation_author=N Su; citation_journal_title=Genome Biol; citation_volume=6; citation_number=14; citation_pages=R52; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Genome-wide transcription analyses in rice using tiling microarrays.; citation_author=L Li; citation_author=X Wang; citation_author=V Stolc; citation_author=X Li; citation_author=D Zhang; citation_journal_title=Nat Genet; citation_volume=38; citation_number=15; citation_pages=124-129; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Applications of DNA tiling arrays for whole-genome analysis.; citation_author=TC Mockler; citation_author=JR Ecker; citation_journal_title=Genomics; citation_volume=85; citation_number=16; citation_pages=1-15; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Dark matter in the genome: evidence of widespread transcription detected by microarray tiling experiments.; citation_author=JM Johnson; citation_author=S Edwards; citation_author=D Shoemaker; citation_author=EE Schadt; citation_journal_title=Trends Genet; citation_volume=21; citation_number=17; citation_pages=93-102; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer.; citation_author=TR Hughes; citation_author=M Mao; citation_author=AR Jones; citation_author=J Burchard; citation_author=MJ Marton; citation_journal_title=Nat Biotechnol; citation_volume=19; citation_number=18; citation_pages=342-347; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Maskless fabrication of light-directed oligonucleotide microarrays using a digital micromirror array.; citation_author=S Singh-Gasson; citation_author=RD Green; citation_author=YJ Yue; citation_author=C Nelson; citation_author=F Blattner; citation_journal_title=Nat Biotechnol; citation_volume=17; citation_number=19; citation_pages=974-978; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=Empirical analysis of transcriptional activity in the Arabidopsis genome.; citation_author=K Yamada; citation_author=J Lim; citation_author=JM Dale; citation_author=H Chen; citation_author=P Shinn; citation_journal_title=Science; citation_volume=302; citation_number=20; citation_pages=842-846; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Experimental annotation of the human genome using microarray technology.; citation_author=DD Shoemaker; citation_author=EE Schadt; citation_author=CD Armour; citation_author=YD He; citation_author=P Garrett-Engele; citation_journal_title=Nature; citation_volume=409; citation_number=21; citation_pages=922-927; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=RNA expression analysis using a 30 base pair resolution Escherichia coli genome array.; citation_author=DW Selinger; citation_author=KJ Cheung; citation_author=R Mei; citation_author=EM Johansson; citation_author=CS Richmond; citation_journal_title=Nature Biotechnol; citation_volume=18; citation_number=22; citation_pages=1262-1268; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Large-scale transcriptional activity in chromosomes 21 and 22.; citation_author=P Kapranov; citation_author=SE Cawley; citation_author=J Drenkow; citation_author=S Bekiranov; citation_author=RL Strausberg; citation_journal_title=Science; citation_volume=296; citation_number=23; citation_pages=916-919; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Global identification of human transcribed sequences with genome tiling arrays.; citation_author=P Bertone; citation_author=V Stolc; citation_author=TE Royce; citation_author=JS Rozowsky; citation_author=AE Urban; citation_journal_title=Science; citation_volume=306; citation_number=24; citation_pages=2242-2246; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution.; citation_author=J Cheng; citation_author=P Kapranov; citation_author=J Drenkow; citation_author=S Dike; citation_author=S Brubaker; citation_journal_title=Science; citation_volume=308; citation_number=25; citation_pages=1149-1154; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=A gene expression map for the euchromatic genome of Drosophila melanogaster.; citation_author=V Stolc; citation_author=Z Gauhar; citation_author=C Mason; citation_author=G Halasz; citation_author=MF van Batenburg; citation_journal_title=Science; citation_volume=306; citation_number=26; citation_pages=655-660; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.; citation_author=BJ Haas; citation_author=AL Delcher; citation_author=SM Mount; citation_author=JR Wortman; citation_author=RK Smith; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=27; citation_pages=5654-5666; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational analysis suggests that alternative first exons are involved in tissue-specific transcription in rice (Oryza sativa).; citation_author=N Kitagawa; citation_author=T Washio; citation_author=S Kosugi; citation_author=T Yamashita; citation_author=K Higashi; citation_journal_title=Bioinformatics; citation_volume=21; citation_number=28; citation_pages=1758-1763; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Inhibition of c-erbA mRNA splicing by a naturally occurring antisense RNA.; citation_author=SH Munroe; citation_author=MA Lazar; citation_journal_title=J Biol Chem; citation_volume=266; citation_number=29; citation_pages=22083-22086; citation_date=1991; " />
      <meta name="citation_reference" content="citation_title=Naturally occurring antisense transcripts are present in chick embryo chondrocytes simultaneously with the down-regulation of the alpha 1 (I) collagen gene.; citation_author=CM Farrell; citation_author=LN Lukens; citation_journal_title=J Biol Chem; citation_volume=270; citation_number=30; citation_pages=3400-3408; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=Tsix, a gene antisense to Xist at the X-inactivation centre.; citation_author=JT Lee; citation_author=LS Davidow; citation_author=D Warshawsky; citation_journal_title=Nat Genet; citation_volume=21; citation_number=31; citation_pages=400-404; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=The role of RNA interference in heterochromatic silencing.; citation_author=Z Lippman; citation_author=RA Martienssen; citation_journal_title=Nature; citation_volume=431; citation_number=32; citation_pages=364-370; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Endogenous siRNAs derived from a pair of natural cis-antisense transcripts regulate salt tolerance in Arabidopsis.; citation_author=O Borsani; citation_author=J Zhu; citation_author=PE Verslues; citation_author=R Sunkar; citation_author=JK Zhu; citation_journal_title=Cell; citation_volume=123; citation_number=33; citation_pages=1279-1291; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=MicroRNA-targeted and small interfering RNA–mediated mRNA degradation is regulated by Argonaute, Dicer, and RNA-dependent RNA polymerase in Arabidopsis.; citation_author=M Ronemus; citation_author=MW Vaughn; citation_author=RA Martienssen; citation_journal_title=Plant Cell; citation_volume=18; citation_number=34; citation_pages=1559-1574; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Antisense transcripts in the human genome.; citation_author=B Lehner; citation_author=G Williams; citation_author=RD Campbell; citation_author=CM Sanderson; citation_journal_title=Trends Genet; citation_volume=18; citation_number=35; citation_pages=63-65; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Widespread occurrence of antisense transcription in the human genome.; citation_author=R Yelin; citation_author=D Dahary; citation_author=R Sorek; citation_author=EY Levanon; citation_author=O Goldstein; citation_journal_title=Nat Biotechnol; citation_volume=21; citation_number=36; citation_pages=379-386; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Genome-wide prediction and identification of cis-natural antisense transcripts in Arabidopsis thaliana.; citation_author=XJ Wang; citation_author=T Gaasterland; citation_author=NH Chua; citation_journal_title=Genome Biol; citation_volume=6; citation_number=37; citation_pages=R30; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Antisense transcripts with rice full-length cDNAs.; citation_author=N Osato; citation_author=H Yamada; citation_author=K Satoh; citation_author=H Ooka; citation_author=M Yamamoto; citation_journal_title=Genome Biol; citation_volume=5; citation_number=38; citation_pages=R5; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Pack-MULE transposable elements mediate gene evolution in plants.; citation_author=N Jiang; citation_author=Z Bao; citation_author=X Zhang; citation_author=SR Eddy; citation_author=SR Wessler; citation_journal_title=Nature; citation_volume=431; citation_number=39; citation_pages=569-573; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=The evolutionary fate of MULE-mediated duplications of host gene fragments in rice.; citation_author=N Juretic; citation_author=DR Hoen; citation_author=ML Huynh; citation_author=PM Harrison; citation_author=TE Bureau; citation_journal_title=Genome Res; citation_volume=15; citation_number=40; citation_pages=1292-1297; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Prediction of trans-antisense transcripts in Arabidopsis thaliana.; citation_author=H Wang; citation_author=NH Chua; citation_author=XJ Wang; citation_journal_title=Genome Biol; citation_volume=7; citation_number=41; citation_pages=R92; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=The new genes of rice: a closer look.; citation_author=K Jabbari; citation_author=S Cruveiller; citation_author=O Clay; citation_author=J Le Saux; citation_author=G Bernardi; citation_journal_title=Trends Plant Sci; citation_volume=9; citation_number=42; citation_pages=281-285; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Organization of the Caenorhabditis elegans small non-coding transcriptome: genomic features, biogenesis, and expression.; citation_author=W Deng; citation_author=X Zhu; citation_author=G Skogerbo; citation_author=Y Zhao; citation_author=Z Fu; citation_journal_title=Genome Res; citation_volume=16; citation_number=43; citation_pages=20-29; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Fast and reliable prediction of noncoding RNAs.; citation_author=S Washietl; citation_author=IL Hofacker; citation_author=PF Stadler; citation_journal_title=Proc Natl Acad Sci USA; citation_volume=102; citation_number=44; citation_pages=2454-2459; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=An improved technique for evaluating resistance of rice varieties to Xanthomonas oryzae.; citation_author=HE Kauffman; citation_author=APK Reddy; citation_author=SPY Hsieh; citation_author=SD Merca; citation_journal_title=Plant Dis Rep; citation_volume=57; citation_number=45; citation_pages=537-541; citation_date=1973; " />
      <meta name="citation_reference" content="citation_title=OligoArray 2.0: design of oligonucleotide probes for DNA microarrays using a thermodynamic approach.; citation_author=JM Rouillard; citation_author=M Zuker; citation_author=E Gulari; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=46; citation_pages=3057-3062; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Issues in the analysis of oligonucleotide tiling microarrays for transcript mapping.; citation_author=TE Royce; citation_author=JS Rozowsky; citation_author=P Bertone; citation_author=M Samanta; citation_author=V Stolc; citation_journal_title=Trends Genet; citation_volume=21; citation_number=47; citation_pages=466-475; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=NMPP: a user-customized NimbleGen microarray data processing pipeline.; citation_author=X Wang; citation_author=H He; citation_author=L Li; citation_author=R Chen; citation_author=XW Deng; citation_journal_title=Bioinformatics; citation_volume=22; citation_number=48; citation_pages=2955-2957; citation_date=2006; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome"/>
    <meta name="twitter:description" content="Genome tiling microarray studies have consistently documented rich transcriptional activity beyond the annotated genes. However, systematic characterization and transcriptional profiling of the putative novel transcripts on the genome scale are still lacking. We report here the identification of 25,352 and 27,744 transcriptionally active regions (TARs) not encoded by annotated exons in the rice (Oryza. sativa) subspecies japonica and indica, respectively. The non-exonic TARs account for approximately two thirds of the total TARs detected by tiling arrays and represent transcripts likely conserved between japonica and indica. Transcription of 21,018 (83%) japonica non-exonic TARs was verified through expression profiling in 10 tissue types using a re-array in which annotated genes and TARs were each represented by five independent probes. Subsequent analyses indicate that about 80% of the japonica TARs that were not assigned to annotated exons can be assigned to various putatively functional or structural elements of the rice genome, including splice variants, uncharacterized portions of incompletely annotated genes, antisense transcripts, duplicated gene fragments, and potential non-coding RNAs. These results provide a systematic characterization of non-exonic transcripts in rice and thus expand the current view of the complexity and dynamics of the rice transcriptome."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0000294.g007"/>

  <meta property="og:title" content="Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=6774'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=13'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=3562&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000294">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000294" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000294&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Lei Li
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Lei Li, Xiangfeng Wang, Rajkumar Sasidharan</p>

              
              

                <p>Affiliation:
                  Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xiangfeng Wang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Lei Li, Xiangfeng Wang, Rajkumar Sasidharan</p>

              
              

                <p>Affiliations:
                  Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America, 
                  National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing, China, 
                  Peking-Yale Joint Research Center of Plant Molecular Genetics and Agrobiotechnology, College of Life Sciences, Peking University, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Rajkumar Sasidharan
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Lei Li, Xiangfeng Wang, Rajkumar Sasidharan</p>

              
              

                <p>Affiliation:
                  Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Viktor Stolc, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America, 
                  Genome Research Facility, NASA Ames Research Center, Moffett Field, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Wei Deng, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing, China, 
                  Bioinformatics Laboratory, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Hang He, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliations:
                  National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing, China, 
                  Bioinformatics Laboratory, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jan Korbel, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xuewei Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Pathology, University of California, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Waraporn Tongprasit, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Eloret Corporation, Sunnyvale, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Pamela Ronald, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Plant Pathology, University of California, Davis, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Runsheng Chen, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Bioinformatics Laboratory, Institute of Biophysics, Chinese Academy of Sciences, Beijing, China
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Mark Gerstein, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Molecular Biophysics and Biochemistry, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Xing Wang Deng
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:xingwang.deng@yale.edu">xingwang.deng@yale.edu</a></p>

                <p>Affiliation:
                  Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, Connecticut, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: March 14, 2007</li>
    <li>DOI: 10.1371/journal.pone.0000294</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0000294-g001" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-g002" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-g003" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-g004" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-t001" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-g005" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-g006" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0000294-g007" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g007" title="Figure 7">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g007&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0000294">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0000294" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>Genome tiling microarray studies have consistently documented rich transcriptional activity beyond the annotated genes. However, systematic characterization and transcriptional profiling of the putative novel transcripts on the genome scale are still lacking. We report here the identification of 25,352 and 27,744 transcriptionally active regions (TARs) not encoded by annotated exons in the rice (<em>Oryza. sativa</em>) subspecies <em>japonica</em> and <em>indica</em>, respectively. The non-exonic TARs account for approximately two thirds of the total TARs detected by tiling arrays and represent transcripts likely conserved between <em>japonica</em> and <em>indica</em>. Transcription of 21,018 (83%) <em>japonica</em> non-exonic TARs was verified through expression profiling in 10 tissue types using a re-array in which annotated genes and TARs were each represented by five independent probes. Subsequent analyses indicate that about 80% of the <em>japonica</em> TARs that were not assigned to annotated exons can be assigned to various putatively functional or structural elements of the rice genome, including splice variants, uncharacterized portions of incompletely annotated genes, antisense transcripts, duplicated gene fragments, and potential non-coding RNAs. These results provide a systematic characterization of non-exonic transcripts in rice and thus expand the current view of the complexity and dynamics of the rice transcriptome.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Li L, Wang X, Sasidharan R, Stolc V, Deng W, et al.  (2007) Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome. PLoS ONE 2(3):
          e294.
            doi:10.1371/journal.pone.0000294</p><p><strong>Academic Editor: </strong>Brian Dilkes, University of California, Davis, United States of America</p><p><strong>Received:</strong> December 18, 2006; <strong>Accepted:</strong> February 21, 2007; <strong>Published:</strong> March 14, 2007</p><p><strong>Copyright:</strong> © 2007 Li et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant from the NSF Plant Genome Program (DBI-0421675).</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>Efforts to generate sequence from the rice (<em>Oryza sativa</em>) genome as a model for the Gramineae, the group that includes all major cereal crops, has produced draft sequences of the <em>japonica</em> and <em>indica</em> subspecies <a href="#pone.0000294-Goff1">[1]</a>–<a href="#pone.0000294-Yu2">[3]</a>. The current, map-based <em>japonica</em> genome sequence assembly covers over 95% of the genome <a href="#pone.0000294-Sasaki1">[4]</a>–<a href="#pone.0000294-International1">[7]</a>. These sequences have been subjected to extensive annotation using <em>ab initio</em> gene prediction, comparative genomics, and a variety of other computational methods <a href="#pone.0000294-Goff1">[1]</a>–<a href="#pone.0000294-Yuan1">[8]</a>. As such, our understanding of the rice genome is largely confined to the state-of-the-art in computational gene prediction and annotation. Improvement of rice genome annotation is progressing rapidly along several lines of analysis. Extensive expressed sequence tags (ESTs) and full-length cDNA (FL-cDNA) sequence data have been incorporated to improve the predicted gene structure <a href="#pone.0000294-International1">[7]</a>–<a href="#pone.0000294-Kikuchi1">[9]</a>. Databases of plant repeat sequences are being expanded and refined to mask open reading frames (ORFs) associated with transposable elements (TEs) <a href="#pone.0000294-Yuan1">[8]</a>, <a href="#pone.0000294-Juretic1">[10]</a>. On the other hand, application of various transcriptomic approaches in rice <a href="#pone.0000294-Kikuchi1">[9]</a>, <a href="#pone.0000294-Matsumura1">[11]</a>–<a href="#pone.0000294-Matsumura2">[13]</a>, including genome tiling microarray analysis <a href="#pone.0000294-Li1">[14]</a>, <a href="#pone.0000294-Li2">[15]</a>, has documented many novel transcripts that have yet to be incorporated into genome annotation.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Genome tiling arrays are a recent advances in microarray technology that involve the representation of a target genome by a virtual ‘tile path’ consisting of oligonucleotide probes <a href="#pone.0000294-Mockler1">[16]</a>, <a href="#pone.0000294-Johnson1">[17]</a>. The probes are synthesized or immobilized on glass slides at high feature densities so that even complex genome sequences can be accommodated within a manageable number of arrays <a href="#pone.0000294-Hughes1">[18]</a>–<a href="#pone.0000294-Yamada1">[20]</a>. Hybridization of the tiling arrays with fluorescence-labeled targets derived from various RNA sources generates signals that can be analyzed to identify transcribed regions of the genome at a resolution roughly equal to the average distance between neighboring probes <a href="#pone.0000294-Li2">[15]</a>, <a href="#pone.0000294-Yamada1">[20]</a>–<a href="#pone.0000294-Cheng1">[25]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Because genome tiling arrays provide end-to-end coverage of the target genome and measure transcriptional activity from multiple probes, they are capable of detecting the transcriptome in a comprehensive and relatively unbiased way. Thus, in the context of rice genome annotation, tiling array analysis can be utilized to verify predicted gene models and to identify novel transcription units <a href="#pone.0000294-Li1">[14]</a>, <a href="#pone.0000294-Li2">[15]</a>. For example, of the 43,914 annotated non-TE gene models from <em>indica</em> <a href="#pone.0000294-Yu2">[3]</a>, transcription of 35,970 (82%) was detected by tiling array analysis <a href="#pone.0000294-Li2">[15]</a>. Consistent with results from tiling analysis in other model organisms <a href="#pone.0000294-Yamada1">[20]</a>–<a href="#pone.0000294-Cheng1">[25]</a>, significant transcriptional activities were detected outside of the annotated exons of the rice genome <a href="#pone.0000294-Li1">[14]</a>, <a href="#pone.0000294-Li2">[15]</a>. A conservative scoring of <em>indica</em> tiling array data identified 5,464 unique transcriptionally active regions (TARs) in the intergenic regions <a href="#pone.0000294-Li2">[15]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In the present study, we performed systematic searches and characterization for TARs in the rice genome, taking advantage of the parallel <em>japonica</em> and <em>indica</em> genome tiling data. We identified 39,018 and 42,470 TARs from <em>japonica</em> and <em>indica</em>, respectively. About two thirds of the TARs, were detected at corresponding locations in the two rice subspecies and do not intersect with annotated exons (referred to as non-exonic TARs). We verified transcription of the <em>japonica</em> non-exonic TARs by using a re-array in which each TAR and gene model was represented by five probes to profile 10 different rice tissue types. The re-array results were consistent with other experimental evidence and validated &gt;83% non-exonic TARs detected in by the tiling array. Through genome-wide bioinformatic characterization, we assigned approximately 80% of the non-exonic TARs to various putative functional or structural elements of the rice genome.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results and Discussion"></a><h3>Results and Discussion</h3>
<h4>Identification of TARs from rice full-genome tiling arrays</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>Our characterization of the <em>japonica</em> transcriptome started with custom full-genome tiling microarrays containing 12,254,374 36-mer oligonucleotide probes, with an average space of 10 nucleotides (nt) between adjacent probes. The probes tile both DNA strands of the non-repetitive sequences of the genome and were synthesized in a set of 32 arrays <a href="#pone.0000294-Li2">[15]</a>, <a href="#pone.0000294-Bertone1">[24]</a>, <a href="#pone.0000294-Stolc1">[26]</a>. The tiling arrays were hybridized to a mixture of cDNA targets derived from four rice tissues or organs to optimize transcript representation (see <a href="#s3">Materials and Methods</a>). Transcription of the annotated gene models was scored using a previously described method <a href="#pone.0000294-Bertone1">[24]</a> and the results are shown in <a href="#pone.0000294.s001">Figure S1</a>. It is note worth that the PASA gene models (see below), which were improved by incorporation of EST and FL-cDNA information, were detected at higher rate than other gene models, attesting to the accuracy of tiling array detection.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>We next scored genome-wide transcription blind to the annotated gene models and described a transcriptome consisting of 39,018 unique TARs (see <a href="#s3">Materials and Methods</a>). Of these, 13,666 (35%) intersected with annotated exons (exonic-TAR; <a href="#pone-0000294-g001">Figure 1A</a>). Comparison of the corresponding sequences of non-exonic TARs to plant repeat databases yielded 748 hits (E≤e<sup>−10</sup>). Comparing the non-exonic TARs with rice organelle genome sequences, which have inserted into and contribute to ~0.4% of the nuclear genome <a href="#pone.0000294-International1">[7]</a>, revealed 136 and 238 matches (&gt;95% sequence identity) to the mitochondrial and chloroplast genomes, respectively. Thus, only 1122 (4%) non-exonic TARs are encoded by repeats (<a href="#pone-0000294-g001">Figure 1A</a>). Not surprisingly, distribution of the non-exonic TARs along the chromosomes indicates that the TE-abundant centromere regions are depleted of TARs (<a href="#pone-0000294-g001">Figure 1B</a> and <a href="#pone.0000294.s002">S2</a>).</p>
<div class="figure" id="pone-0000294-g001"><div class="img"><a name="pone-0000294-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1. </strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>Characterization of <em>japonica</em> TARs. (A) Classification of all identified TARs based on known transcriptional evidence and sequences of known origin. Exon, exons of annotated gene models; Unigene, rice Unigenes in GenBank; Repeat, plant repetitive sequences and organelle insertions. (B) Density of different sets of transcription units along Chromosome 8. The number of transcription units was calculated in 500 Kb-sliding-windows with a 100 Kb step. Position of the centromere is indicated by the triangle. (C) Venn diagram showing the comparison of <em>japonica</em> non-exonic TARs with <em>indica</em> transcription units. Two independent comparisons were made. First, the <em>japonica</em> non-exonic TARs were compared with annotated <em>indica</em> gene models. Second, the <em>japonica</em> non-exonic TARs were compared with <em>indica</em> non-exonic TARs, which were identified outside of gene models from tiling array analysis of the <em>indica</em> genome. Shaded areas indicate <em>japonica</em> TARs that overlap with <em>indica</em> gene models or non-exonic TARs.</p>
<span>doi:10.1371/journal.pone.0000294.g001</span></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3"></a><p>To facilitate comparison of TARs between <em>japonica</em> and <em>indica</em> rice, we re-analyzed the previously reported <em>indica</em> genome tiling data using the same method reported in the present study. Our re-analysis identified 42,470 <em>indica</em> TARs, of which 27,744 (65%) were non-exonic (<a href="#pone.0000294.s003">Figure S3</a>), significantly extending the previous report of 5464 TARs in the intergenic regions <a href="#pone.0000294-Li2">[15]</a>. The percentage of TARs intersecting with exons is the same (35%) in <em>japonica</em> and <em>indica</em>, implying that annotated exons account for approximately one third of the overall transcriptional activity of the rice genome as detected by tiling arrays.</p>


<h4>Non-exonic TARs are conserved between <em>japonica</em> and <em>indica</em> rice</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Comparison of the non-exonic TARs between the two subspecies showed that 8515 <em>japonica</em> TARs overlap with 7939 TARs from <em>indica</em>. In addition, 1713 <em>japonica</em> non-exonic TARs overlap with 1503 <em>indica</em> gene models (<a href="#pone-0000294-g001">Figure 1C</a>). Thus, 10,228 (40%) <em>japonica</em> non-exonic TARs have a direct match in the <em>indica</em> transcriptome and 1713 correspond to locations that may be misannotated as non-genic sequences in the initial genome annotation.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Because TARs were identified as transcribed regions of the genome, they likely represent fragments of larger discrete transcription units. Thus, <em>indica</em> and <em>japonica</em> TARs representing different portions of the same transcription units may not overlap directly. To facilitate examination of the conservation of <em>indica</em> and <em>japonica</em> TARs, we first identified a set of gene models shared between <em>japonica</em> (8921)&nbsp;and <em>indica</em> (8925) and supported by full-length cDNA sequences. These common gene models presumably represent the most conserved portion of the rice transcriptome. Within our data sets there were 3449 <em>japonica</em> and 4230 <em>indica</em> exonic TARs derived from 2808 and 3461 confirmed shared gene models, respectively. We found that 1595 (46%) <em>japonica</em> TARs overlapped with 1599 (38%) <em>indica</em> TARs (<a href="#pone.0000294.s003">Figure S3</a>). Thus, non-exonic TARs and exonic TARs from confirmed gene models contained similar portions that exhibit direct sequence overlap between the <em>japonica</em> and <em>indica</em> transcriptomes. These results indicate that the non-exonic TARs represent transcripts that were conserved between the two rice subspecies at a level comparable to genes.</p>


<h4>Transcriptional verification of non-exonic TARs</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>Rice TARs were scored using a method based on the binomial theorem in which at least five consecutive probes whose intensities lie in the 80<sup>th</sup> percentile were identified <a href="#pone.0000294-Bertone1">[24]</a>. In theory, this method should not produce false positives in excess of 10% among the 39,000 TARs from 12 million probes (12×10<sup>6</sup>×0.2<sup>5</sup>/39,000). To experimentally validate transcription of the non-exonic TARs and to describe their transcriptional regulation, we constructed a new array (designated the re-array) using five independent 36-mer probes to represent each of the 44,385 non-TE gene models and 25,313 TARs in the <em>japonica</em> genome (see <a href="#s3">Materials and Methods</a>). Using the re-array, we measured expression in triplicate for 10 different rice organs (<a href="#pone-0000294-g002">Figure 2A</a> and <a href="#pone.0000294.s004">S4</a>). Transcription of 7965 (18%) and 28,265 (64%) gene models were detected in every organ and in at least one of the assayed organ types, respectively. Among the 10 organs, the rate of gene expression detection ranged from 27% (seedling root) to 52% (flag leaf) (<a href="#pone-0000294-g002">Figure 2A</a>). Furthermore, the relative overall similarity of the transcription levels of gene models was consistent with the developmental and/or physiological state of the organs (<a href="#pone-0000294-g002">Figure 2A</a>).</p>
<div class="figure" id="pone-0000294-g002"><div class="img"><a name="pone-0000294-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2. </strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>Re-array analysis of <em>japonica</em> gene models and non-exonic TARs. (A) Transcription of gene models and TARs detected by the re-array. Shown on the left is a tree diagram of the overall transcriptional relation of gene models among the 10 assayed tissue types computed with the Manhattan distance function. Expression rates in percentage of all gene models, the PASA gene models, and non-exonic TARs are shown on the right. (B) Heat map of differentially expressed gene models and non-exonic TARs. The red and yellow colors represent up- and down-regulated transcription units, respectively. The gene tree is shown on the left. Numbers on top represent the tissue types: 1, developing seed, 2, carpel, 3, whole flower, 4, dark-grown seedling, 5, seedling root, 6, light-grown seedling, 7, flag leaf, 8, leaf, 9, <em>Xoo</em>-infected leaf.</p>
<span>doi:10.1371/journal.pone.0000294.g002</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>PASA is a genome annotation tool that exploits spliced alignments of expressed transcripts to model gene structures <a href="#pone.0000294-Haas1">[27]</a>. PASA gene models are therefore supported by and consistent with available experimental evidence. In the analysis of the re-array 14,740 rice PASA models were represented <a href="#pone.0000294-Yuan1">[8]</a>, <a href="#pone.0000294-Kikuchi1">[9]</a>. Higher detection rates for the PASA models, than for the 44,385 genes in the genome annotation, were obtained for all 10 organ types, which ranged from 33% (seedling grown in dark) to 63% (cultured cell), and 11,542 (78%) of the PASA gene models were detected in at least one organ type (<a href="#pone-0000294-g002">Figure 2A</a>). These results indicate that the re-array was an efficient method to detect expression from known rice transcripts.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>From the re-array, transcription of 9323 (37%) and 21,018 (83%) non-exonic TARs was detected in all and at least one organ types, respectively. Across organ types, detection rates of TARs ranged from 43% (seedling root) to 79% (flag leaf) (<a href="#pone-0000294-g002">Figure 2A</a>). We further observed that re-array detection rate of TARs was consistent with other experimental evidence. First, there were 3473 non-exonic TARs that had sequence matches to cDNA sequences among the rice Unigenes (see <a href="#s3">Materials and Methods</a>) (<a href="#pone-0000294-g001">Figure 1A</a>). Similar to the detection of exon-derived TAR, Unigene-supported non-exonic TARs exhibited a detection rate of 90% while 82% those without a Unigene match were confirmed in the re-array (<a href="#pone.0000294.s005">Figure S5</a>). Second, we performed reverse transcription coupled PCR (RT-PCR) analysis on 108 randomly selected non-exonic TARs. The Unigene-supported and re-array detected TARs exhibited higher PCR confirmation rates (92% and 78% respectively) than that (44%) of TARs not detected by the re-array (<a href="#pone.0000294.s005">Figure S5</a>).</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4"></a><p>We determined which genes and non-exonic TARs that were differentially expressed among the 10 assayed organ types. While 4232 (9.5%) genes were differentially expressed between organ types, fewer (545, or 2.2%) non-exonic TARs than genes were differentially expressed across the organ types assayed (<a href="#pone-0000294-g002">Figure 2B</a>). Together, our results indicate that non-exonic TARs are generally more ubiquitously transcribed than genes. Interestingly, detection rates of non-exonic TARs were highly correlated with the gene detection rates (Pearson correlation coefficient <em>r</em> = 0.92), suggesting that the transcriptional activity represented by non-exonic TARs reflects the activity of the transcriptome and is thus a worthy target for future transcriptome profiling efforts.</p>


<h4>Non-exonic TARs correspond to portions of existing gene models</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>Introns of the rice genome were relatively depleted of transcripts represented in the form of TARs (<a href="#pone-0000294-g003">Figure 3A and 3B</a>). This is because only intronic TARs present in a poly(A)-baring transcript would have been detected under our experimental conditions. We used an RT-PCR assay in which a pair of primers was designed to target the intronic TAR and a downstream exon to examine whether the intronic TARs could represent splice variants of the related genes (<a href="#pone-0000294-g003">Figure 3C</a>). However, only five out of 24 selected cases (21%) were unambiguously verified (<a href="#pone-0000294-g003">Figure 3D and 3E</a>). We observed that the frequency of intronic TARs correlated with the median sizes of the introns (<em>r</em> = 0.97). This prompted us to investigate whether TARs in the first introns, which are the largest and have the most TARs (358) among all introns, could represent alternative first exons that have been reported to contribute substantially to transcript diversity in mammals but only sparsely documented in plants <a href="#pone.0000294-Kitagawa1">[28]</a>. Through RT-PCR analysis, six of the 12 (50%) randomly selected cases were confirmed in which the intronic TAR was found as part of an alternative transcript derived from the annotated locus (<a href="#pone-0000294-g003">Figure 3E</a>). Detailed annotation of one such example is illustrated in <a href="#pone-0000294-g004">Figure 4A</a>. The novel TAR-initiated splice variant and the annotated gene model exhibited distinct transcription pattern across the examined conditions (<a href="#pone-0000294-g004">Figure 4B</a>), indicating that the alternative transcripts were differentially expressed.</p>
<div class="figure" id="pone-0000294-g003"><div class="img"><a name="pone-0000294-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3. </strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>Analysis of non-exonic TARs representing portions of annotated gene models. (A) Number of TARs in different components of the rice genome. (B) Relative frequency of TARs in different genomic compartments. Note that strandedness of the TARs was not considered in this analysis. TAR frequency was calculated as the number of TARs per Mb and normalized to the genomic TAR frequency. (C) The RT-PCR assay to examine whether a gene/TAR pair belongs to the same transcript in which primer pairs target the gene model and the TAR in a convergent fashion. (D) Examples of PCR products amplified in a reverse transcription (RT) dependent manner for the gene/TAR pairs Os10g21310 and Chr10fwd_10366435 (left), and Os02g50070 and Chr2fwd_30528038 (right). (E) Confirmation rates obtained for intronic TARs and proximal intergenic TARs by RT-PCR.</p>
<span>doi:10.1371/journal.pone.0000294.g003</span></div><div class="figure" id="pone-0000294-g004"><div class="img"><a name="pone-0000294-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4. </strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>Experimental verification of selected non-exonic TARs representing uncharacterized portions of gene models. (A) Annotation of an intronic TAR. The TAR Chr4rev_30188695 is in the first intron of the gene model Os04g51690. A TAR-containing transcript was proposed in which the TAR serves as the alternative first exon. (B) RT-PCR analysis of the transcript abundance in diverse conditions for the FL-cDNA AK102588 and the TAR-initiated alternative transcript. Positions of the primers are indicated by the asterisks shown in (A). (C) Annotation of a proximal intergenic TAR 3′ to a gene model. Arrows indicate the primer positions.</p>
<span>doi:10.1371/journal.pone.0000294.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>There were 15,472 TARs in the intergenic regions (<a href="#pone-0000294-g003">Figure 3A</a>) of which 1682 were located in close proximity (&lt;1 kb) and on the same strand as a gene model. In addition, 523 TARs exhibited strong co-transcription pattern (<em>r</em>&gt;0.7) with the corresponding gene model. As many of the rice gene models were only annotated for their coding regions <a href="#pone.0000294-Yuan1">[8]</a>, we speculated that the proximal intergenic TARs could correspond to untranslated regions or misannotated coding capacities of the nearby gene models. To test this, we performed RT-PCR analysis using primer pairs to test for physical linkage of 54 randomly selected TARs to proximal gene models. In this sample 42 (78%) generated a PCR band in an RT-dependent manner, indicating that the co-transcribed TARs represent previously uncharacterized portions of annotated gene models (<a href="#pone-0000294-g003">Figure 3E</a> and <a href="#pone-0000294-g004">4C</a>). Assuming the PCR confirmation rates can be applied to all non-exonic TARs within or close to annotated gene loci (857×21%+358×50%+523×78%), these results indicate that ~760 (3%) TARs represent various un-annotated portions of the existing gene models (<a href="#pone-0000294-t001">Table 1</a>).</p>
<div class="figure" id="pone-0000294-t001"><div class="img"><a name="pone-0000294-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of characterization of non-exonic TARs in <em>japonica</em> rice.</span></strong></p><span>doi:10.1371/journal.pone.0000294.t001</span></div>

<h4>Identification of non-exonic TARs antisense to annotated gene models</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Natural antisense transcripts (NATs) have been broadly implicated in regulating gene expression and chromosome structure <a href="#pone.0000294-Munroe1">[29]</a>–<a href="#pone.0000294-Ronemus1">[34]</a>. Large-scale efforts to systemically identify NATs have been made in several model organisms <a href="#pone.0000294-Lehner1">[35]</a>–<a href="#pone.0000294-Wang1">[37]</a>. In rice, 687 bi-directional transcript pairs were identified from a collection of 32,127 FL-cDNAs <a href="#pone.0000294-Osato1">[38]</a>. Analysis of our tiling array data detected 9023 non-exonic TARs located on the opposite DNA strand of annotated gene models that potentially represent <em>cis</em>-NATs. Antisense TARs exhibited slightly higher re-array detection rate than TARs on average. For example, 8256 of 8262 anti-exonic TARs were represented in the re-array of which 7061 (85%) were detected (<a href="#pone-0000294-g005">Figure 5A</a>). These observations suggest that rice NATs may be more prevalent than previously realized.</p>
<div class="figure" id="pone-0000294-g005"><div class="img"><a name="pone-0000294-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5. </strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>Characterization of antisense TARs. (A) Pie chart analysis of antisense TARs. Of the 8256 antisense TARs, 7061 were expressed in at least one tissue type according to the re-array (left), 4993 were antisense to a non-TE gene model (center), of which 884 had a transcriptional correlation coefficient with the corresponding gene models in the 10 assayed tissue types &lt;−0.4 (right). (B) Scatterplot of the relative transcription level of the gene model/antisense TAR pairs in the cultured cell versus their correlation coefficient across the 10 assayed tissue types. (C) Annotation of an antisense TAR. The TAR Chr4rev_5677533 overlaps convergently with the gene model Os04g10780. (D) Transcription level of the TAR Chr4rev_5677533 and the gene model Os04g10780 in the 10 assayed tissue types. 1, cultured cell, 2, seedling root, 3, dark-grown seedling, 4, light-grown seedling, 5, leaf, 6, <em>Xoo</em>-infected leaf, 7, flag leaf, 8, whole flower, 9, carpel, and 10, developing seed. (E) Northern blot analysis of small RNA related to antisense TARs. The probes were PCR products derived from antisense TARs. The migration positions of a 21 nt RNA are indicated by the black triangle.</p>
<span>doi:10.1371/journal.pone.0000294.g005</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>A total of 4993 TARs antisense to annotated exons in the non-TE genes were represented on the re-array, allowing us to investigate the transcriptional pattern of the TAR/gene pairs across the examined tissue types. The transcription of 884 antisense TAR/gene pairs showed strong negative correlations (<em>r</em>&lt;−0.4) (<a href="#pone-0000294-g005">Figure 5A</a>). This group of antisense TAR/gene pairs also had greater difference in their transcription level than other pairs (<a href="#pone-0000294-g005">Figure 5B</a>), manifesting a strong reciprocal transcription pattern in the assayed tissue types.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>Because antisense transcripts can function through small interfering RNA (siRNA) derived from paired sense–antisense transcripts <a href="#pone.0000294-Munroe1">[29]</a>–<a href="#pone.0000294-Lee1">[31]</a>, we investigated whether the negatively correlated antisense gene/TAR pairs could generate siRNA by Northern blot analysis using the cloned TARs as probes (<a href="#pone-0000294-g005">Figure 5C–5E</a>). This analysis detected low-molecular-weight RNA bands of similar size to known siRNA in six out eight cases (<a href="#pone-0000294-g005">Figure 5E</a>). Detailed analysis of one antisense TAR is illustrated in <a href="#pone-0000294-g005">Figure 5C–5E</a>.</p>


<h4>Analysis of TARs intersecting Pack-MuLEs</h4>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1"></a><p>We examined the intersection of TARs with elements of the genome other than the annotated protein-coding genes. Chimeric <em><span class="underline">Mu</span>tator</em>-<span class="underline">l</span>ike transposable <span class="underline">e</span>lements carrying fragments of genes, so called Pack-MuLEs, are a characteristic of plant genomes and may be involved in the creation of new genes during evolution <a href="#pone.0000294-Jiang1">[39]</a>, <a href="#pone.0000294-Juretic2">[40]</a>. In rice, 475 Pack-MuLEs in Chromosome 1 and 10 were previously annotated in detail <a href="#pone.0000294-Jiang1">[39]</a>. We report here the analysis of TARs associated with these annotated Pack-MuLEs in Chromosome 1 and 10. We observed that 248 Pack-MuLEs match with at least one of the TARs (446 in total, including 118 exonic and 328 non-exonic TARs), indicating that about half (248/475 = 52%) of the Pack-MuLEs were potentially transcribed. Extrapolating our observations on Chromosome 1 and 10 to the whole genome (328×390Mb/69Mb) suggests that as many as 1850 (7.3%) non-exonic TARs could be encoded by Pack-MuLEs (<a href="#pone-0000294-t001">Table 1</a>).</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2"></a><p>We found that 109 TARs intersected 96 ORFs encoded by Pack-MuLEs whilst 112 were located on the antisense strand of 97 ORFs. Among these, 48 ORFs were intersected by both sense and antisense TARs. Such a strong symmetric transcription pattern of Pack-MuLEs consisting of equal number of sense and antisense TARs is in keeping with the observation that most Pack-MuLE derived FL-cDNA were initiated within or near the terminal inverted repeats (TIRs) <a href="#pone.0000294-Jiang1">[39]</a>. In further support of this conclusion, we found 63 TARs with significant match to the TIR sequences (E≤10<sup>−10</sup>). Although it is possible that the detected transcriptional activity of Pack-MuLEs were derived from close homologs elsewhere in the genome, the strong symmetric transcription pattern argues against this possibility. The symmetrical pattern of transcription suggests that Pack-MuLE loci could be a rich source for <em>trans</em>-regulatory RNAs (i.e. double-stranded or <em>trans</em>-antisense) that can potentially affect expression of the original genes <a href="#pone.0000294-Munroe1">[29]</a>–<a href="#pone.0000294-Ronemus1">[34]</a>, <a href="#pone.0000294-Wang2">[41]</a>.</p>


<h4>Intergenic TARs and non-coding transcripts</h4>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1"></a><p>The intergenic TARs, distal to a gene model and not overlapping with other elements of the genome, numbered ~8400 in total and do not appear to encode proteins. Two lines of evidence supported conclusion. First, the linear relationship between the GC content of the second (GC2) and the third (GC3) codon positions of the longest deduced ORF, of the six possible translations, from these TARs deviated from that of known genes <a href="#pone.0000294-Jabbari1">[42]</a>. As shown in <a href="#pone-0000294-g006">Figure 6A</a>, the GC3/GC2 correlation of the intergenic TARs aligned along the diagonal, far from that of the PASA gene models. Consistent with this observation, the TAR-deduced peptide sequences were enriched for GC-rich codons, such as Pro and Arg (<a href="#pone.0000294.s006">Figure S6</a>). Although it is possible that some of the putative proteins have legitimate biological functions, the biased codon usage suggests that many of the intergenic TARs may not function through their deduced ORFs.</p>
<div class="figure" id="pone-0000294-g006"><div class="img"><a name="pone-0000294-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6. </strong></p><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1"></a><p>Analysis of non-coding intergenic TARs. (A) Scatterplot of GC2 versus GC3 in all gene models (n = 46,976), FL-cDNA-supported PASA gene models (n = 11,494), and intergenic TARs (n = 5256). The intergenic TARs were distal (&gt;1Kb) to a gene model excluding those with a hit in the ProSite database. (B) Overlapping TARs with putative non-coding transcripts. A 5-Kb region represented by the high-resolution tiling array is shown. The interrogating probes are aligned to the chromosomal coordinates, with the fluorescence intensity value depicted as a vertical line. Gene models, no-exonic TARs and putative non-coding transcripts are depicted as horizontal arrows, which point to the direction of transcription. A portion of the region containing four non-coding transcripts and a pair of TARs is enlarged and shown at the bottom. (C) Predicted secondary structure of the TAR Chr10fwd_10524178. The sequence corresponding to the putative nc-RNA ts_342 is highlighted.</p>
<span>doi:10.1371/journal.pone.0000294.g006</span></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2"></a><p>The non-exonic TARs were compared to putative non-coding RNA (nc-RNA) identified using a high resolution (5-bp) tiling array. This array, which represents a 1Mb region of Chromosome 10, was hybridized to mRNA- or small RNA-derived cDNA samples. Hybridization signals from labeled mRNA-derived cDNA matched known transcripts with high fidelity (<a href="#pone.0000294.s007">Figure S7</a>). When this array was hybridized to cDNA derived from specifically enriched small nc-RNA samples <a href="#pone.0000294-Deng1">[43]</a>, 27 of the 29 annotated tRNA genes in this region were detected, indicating that tRNA-sized nc-RNAs were well represented in the labeled sample. In addition, 47 of the hybridization signals corresponding to nc-RNAs overlapped with 35 non-exonic TARs. Eight of the TARs were antisense to exon sequences and intersected 12 nc-RNAs while 27 intergenic TAR intersected 35 nc-RNAs within the 1Mb region interrogated by the array. Extrapolating these results to the whole genome suggests that there could be ~10,500 (27×390Mb/1Mb) intergenic TARs and ~3120 (8×390Mb/1Mb) antisense TARs related to non-coding small RNA species (<a href="#pone-0000294-t001">Table 1</a>). Interestingly, six intergenic TARs, which intersected 13 nc-RNAs, also have their own antisense TARs (<a href="#pone-0000294-g006">Figure 6B</a>), suggesting that the small RNA species might be derived from the paired antisense transcripts.</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3"></a><p>Some TARs with corresponding small RNA derivatives were predicted to fold into stable secondary structures in which the nc-RNAs were located near stem-loop motifs (<a href="#pone-0000294-g006">Figure 6C</a>). It is a possibility that the small nc-RNAs are processed from these corresponding TARs. We were also able to predict stable secondary structures for the many intergenic TARs not represented by the 1Mb tiling array, particularly those located in close proximity to gene models in a divergent antisense orientation (<a href="#pone-0000294-g007">Figure 7</a>). We identified 234 intergenic TARs that have homology to sequences in different plant species. These clusters of sequence homologs were used to search for structural conservation using the RNAz algorithm <a href="#pone.0000294-Washietl1">[44]</a>, which uses comparative sequence analysis and RNA secondary structure prediction to detect evolutionarily conserved and exceptionally stable secondary structures.&nbsp;By this method, 32 of these 234 non-coding TAR clusters (14%) were predicted form stable secondary structures. This fraction is significantly higher (binomial <em>p</em>&lt;<em>e</em><sup>−15</sup>) than that for randomly shuffled TAR alignment blocks, for which only ~6% are predicted to form structural nc-RNA.</p>
<div class="figure" id="pone-0000294-g007"><div class="img"><a name="pone-0000294-g007" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0000294" data-uri="info:doi/10.1371/journal.pone.0000294.g007"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.g007&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g007/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g007/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g007/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g007/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g007.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0000294.g007/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0000294.g007/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0000294.g007.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 7. </strong></p><a id="article1.body1.sec2.sec7.fig2.caption1.p1" name="article1.body1.sec2.sec7.fig2.caption1.p1"></a><p>Analysis of 5′ proximal antisense TARs. (A) Annotation of TAR Chr4rev_22552532 that locates in a divergent antisense orientation to the gene model Os04g38590. (B) Predicted secondary structure of the RNA transcript derived from the TAR Chr4rev_22552532. RNA secondary structure was predicted using RNAfold, which is available as part of the Vienna RNA Package.</p>
<span>doi:10.1371/journal.pone.0000294.g007</span></div><a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4"></a><p>In summary, we identified 25,352 non-exonic TARs in the <em>japonica</em> genome that represent conserved transcripts between <em>japonica</em> and <em>indica</em> rice. We verified transcription for 83% of these TARs through expression profiling and assigned more than 80% TARs to various putative functional or structural elements of the rice genome (<a href="#pone-0000294-t001">Table 1</a>). The putative novel transcripts tagged by non-exonic TARs should therefore be useful to improve rice genome annotation and facilitate comparative genomics in other related but more complex cereal genomes. Among the non-exonic TARs, 3473 matched with the current rice Unigene set (<a href="#pone-0000294-g001">Figure 1A</a>), suggesting that those transcripts were missed from the genome annotation we used. In addition to the novel potentially protein-coding transcripts, many of the non-exonic TARs do not appear to encode proteins. Many of these non-exonic TARs were arranged into or transcribed from specific configurations with protein-coding genes, such as duplicated gene fragments or antisense transcripts (<a href="#pone-0000294-t001">Table 1</a>), and may function to modulate the genes' expression. The non-exonic TARs that could not be directly linked to annotated genes were often associated with small non-coding RNA species (<a href="#pone-0000294-g006">Figure 6</a>). These results thus expand the current view of the complexity and dynamics of the rice transcriptome.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Plant material and RNA preparation</h4>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1"></a><p><em>Oryza</em>. <em>sativa</em> ssp. <em>japonica</em> cv. Nipponbare was used for all experiments. Light- and dark-grown seedlings, suspension cultured cells, panicles, and stress-treated seedlings were prepared as previously described <a href="#pone.0000294-Li1">[14]</a>. For other materials, seedlings were transferred to soil and maintained under long-day conditions (16 hour light/8 hour dark) at 26–28°C in the greenhouse. After 7 weeks, mature leaves were inoculated with the pathogen <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> race PXO99 (<em>Xoo</em>) using the leaf clip method <a href="#pone.0000294-Kauffman1">[45]</a>. During the flowering stage, flag leaves, flowers (florets), stamens and carpel (before pollination) were harvested. Developing seeds were harvested three to four days after pollination. For microarrays, total RNA and mRNA were sequentially isolated using the RNeasy Plant Mini kit (Qiagen) and the Oligotex mRNA kit (Qiagen). For Northern blot analysis, total RNA was isolated from rice seedlings using the Trizol method (Invitrogen). Low-molecular-weight RNA was enriched and subsequent Northern blot analysis carried out as previously described <a href="#pone.0000294-Borsani1">[33]</a>. Non-coding RNA species were isolated from total RNA as previously described <a href="#pone.0000294-Deng1">[43]</a>. Fractions corresponding to tRNAs (sample I) and small RNAs of 80–500 nt (sample II) were collected separately.</p>


<h4>Microarray design</h4>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1"></a><p>Three different types of microarrays were used in this study: the full-genome tiling arrays, a 1Mb high-resolution tiling array, and a re-array. All arrays were fabricated at a density of 389,000 oligonucleotides per array as previously described <a href="#pone.0000294-Li1">[14]</a>, <a href="#pone.0000294-Li2">[15]</a>, <a href="#pone.0000294-Bertone1">[24]</a>. For the full-genome tiling arrays, we designed a minimal tiling path employing 36-mer oligonucleotides spaced by 10 nt on average to represent the <em>japonica</em> rice genome, based on the TIGR Rice Pseudomolecules release 1 (September 2003) <a href="#pone.0000294-Yuan1">[8]</a>. Details regarding design of the genome tiling array can be found in a previous study <a href="#pone.0000294-Li2">[15]</a>. The 1 Mb tiling array was used to represent the 10.0 Mb to 11.0 Mb region of <em>japonica</em> rice Chromosome 10. A tile path was designed with 36-mer probes at a step of 5 nt.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2"></a><p>We constructed the re-array in which five probes were used to represent each gene model and non-exonic TAR. To ensure that the probes are specific to its target and are free of secondary structures, we used the software OligoArray 2.1 to select probes <a href="#pone.0000294-Rouillard1">[46]</a>. We started with 44,497 gene models and 25,352 non-exonic TARs, which were used as the database sequence to search against. Two separate runs were initiated, one for the gene models and the other for the TARs. For transcription units for which the software could not generate five probes, the remaining probes were selected from the original tiling arrays that were in the 75<sup>th</sup> percentile and met most of the above criteria (hence the name ‘re-array’). 10,000 randomly selected probes from the genome tiling arrays and 9000 probes in the introns of PASA models with intensities below the median were also included as controls.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3"></a><p>The full-genome tiling arrays and the 1Mb tiling array were hybridized to cDNA mixtures derived from seedling root, seedling shoot, panicle and suspension-cultured cells. Note that the cDNA mixture was used as the template for RT-PCR analysis unless otherwise indicated. Additionally, the 1Mb tiling array was hybridized to targets derived from the non-coding RNA Sample I and II. The re-array was hybridized to three biological replicates of cDNA derived from 10 rice tissue types, namely, light-grown seedling, dark-grown seedling, seedling root, leaf, <em>Xoo</em>-infected leaf, flag leaf, whole flower (floret), carpel, developing seed, and suspension cultured cell.</p>


<h4>Full-genome tiling array scoring and TAR identification</h4>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1"></a><p>We were able to map &gt;95% of the 12,254,374 probes to the latest TIGR Rice Pseudomolecules (release 3, release date 02.18.05) at the time data analysis was initiated. We used the gene model annotation provided in the form of a .gff file distributed by TIGR <a href="#pone.0000294-Yuan1">[8]</a>, from which we identified 62,121 gene models (mRNA transcripts), including 13,467 that were TE-related. We excluded gene models that were represented by less than five probes, as one cannot do statistical tests on them. We also excluded 11,143 gene models that were not sufficiently probed, i.e., &lt;45% coverage of their coding region by tiling probes. There were 2549 loci that generated 5897 alternative gene models of which only one from each locus was included. Thus, a total of 5719 TE and 40,257 non-TE models were represented on the tiling arrays.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2"></a><p>We used the sign-test to score annotated gene models <a href="#pone.0000294-Bertone1">[24]</a>. We first identified probes that lie within the exons of a given gene model. For each probe, we checked if its intensity was greater than the median of the array to which it belongs. For each gene model, we determined whether or not the number of probes that are above the median is more than expected by chance alone. The probability, p, of obtaining h probes with intensities above median out of N probes is given by the equation:<a name="" id=""></a><span class="equation"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0000294.e001&amp;representation=PNG" class="inline-graphic"><span class="note">1</span></span><br>We chose a p-value cutoff of 0.05 (corresponding to a false-positive rate of 5%), meaning that gene models with a p-value less than 0.05 were scored as transcribed <a href="#pone.0000294-Bertone1">[24]</a>, <a href="#pone.0000294-Royce1">[47]</a>.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3"></a><p>To identify TARs, the tile path was scanned for regions in the genome that are covered by at least five consecutive probes whose intensities lie in the 80<sup>th</sup> percentile <a href="#pone.0000294-Bertone1">[24]</a>, <a href="#pone.0000294-Royce1">[47]</a>. To accommodate for shifted and overlapping probes because of re-mapping, we further applied a minimal length cutoff of 220 nt and a gap penalty of 30 nt between two probes. Using these criteria, we identified a total of 39,018 and 42, 470 TARs from the <em>japonica</em> and <em>indica</em> genome, respectively. To match non-exonic TARs to rice Unigenes (sets of transcript sequences derived from the same transcription unit), which were downloaded from NCBI (<a href="http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID%E2%80%8A=%E2%80%8A4530">http://www.ncbi.nlm.nih.gov/UniGene/UGOr​g.cgi?TAXID= 4530</a>), a cutoff of E≤e<sup>−10</sup> was used.</p>


<h4>Re-array scoring</h4>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1"></a><p>We developed the NMPP package as a bundle of user-customized tools based on established algorithms and methods to process the re-array data <a href="#pone.0000294-Wang3">[48]</a>. The package is available at: <a href="http://plantgenomics.biology.yale.edu/nmpp">http://plantgenomics.biology.yale.edu/nm​pp</a>. To correct potential skewed signal distribution within each array due to uneven hybridization or washing, we applied a global distance-weighted spatial smoothing to the raw probe intensities based on <em>MAS</em> 5.0 <a href="#pone.0000294-Wang3">[48]</a>. We follow a two-step process to normalize the smoothed intensity values. First, we performed a quantile normalization of the three replicates within each tissue type. We next carried out a global scaling to adjust the quantile-normalized intensity to the same baseline, which is the median of all the intensity values. For each transcription unit, we obtained a 5×3 matrix consisting of the normalized intensity values from five probes in triplicate. We assigned a value to a transcription unit based on this matrix to represent its transcription level using the Tukey's median polish procedure <a href="#pone.0000294-Wang3">[48]</a>.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2"></a><p>Of the 19,000 control probes included in the re-array, 7669 have a GC content higher than 15, which is the case for all probes interrogating transcription units. These 7669 probes were used as the control for subsequent analysis of the re-array data to minimize the impact of probe GC content on transcription calling. To determine from the control probes the noise distribution, we used a previously reported modeling process <a href="#pone.0000294-Li1">[14]</a>. Assuming that control probes with very low intensity (1-9 Log<sub>2</sub>, <a href="#pone.0000294.s004">Supplementary Fig. 4</a>) were primarily noise, we modeled the noise as a normal distribution by mirroring the low-intensity portion of the control probe distribution. For transcription detection of gene models or TARs, the calculated transcription level of each unit was compared with the noise distribution. We used a false-discovery rate (FDR) of 0.05, which represents the 95% confidence in the modeled noise distribution, as a cutoff for determining whether the transcriptional intensity of a given transcription unit is higher than the noise.</p>
<a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3"></a><p>Determination of differentially expressed transcription units was carried out using standard procedures. Briefly, normalized intensity values were fitted to a mixed effects ANOVA model (R/maanova, version 1.2.1, <a href="http://www.jax.org/staff/churchill/labsite/software/download.html">http://www.jax.org/staff/churchill/labsi​te/software/download.html</a>). An <em>F</em>-test was performed to detect differential expression of gene models or non-exonic TARs in the 10 tissue types. Next, FDR-adjusted <em>p</em> values were calculated from permuted samples. A significant level above 99% (<em>p</em>&lt;0.01) was selected for tissue-specific expression calling of gene models or TARs.</p>


<h4>1Mb tiling array scoring</h4>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1"></a><p>We converted probe intensity values from the 1 Mb tiling array into Z scores using a two-step process. First, the mean (μ<sub>G</sub>) and the standard deviation (σ<sub>G</sub>) of all intensity values were calculated and probes with intensity greater than μ<sub>G</sub>+2σ<sub>G</sub> were marked as group I probes. Next, a new mean (μ<sub>G′</sub>) and a new standard deviation (σ<sub>G′</sub>) were calculated excluding the group I probes. The Z scores for the <em>i</em><sup>th</sup> probe is given by the equation z<sub>i</sub> = (x<sub>i</sub>-μ<sub>G′</sub>)/σ<sub>G′</sub>. Probes with a Z score greater than two were considered signal probes. Sequences of neighboring signal probes were joined to form contigs. Putative non-coding transcripts were identified from the contigs with a minimal length of 55 nt (sample I) or 65 nt (sample II).</p>


<h4>RNA secondary structure analysis</h4>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1"></a><p>RNA secondary structures were predicted using RNAfold. To verify the secondary structures, we used 15,000 TARs sequences to BLAST search five plant Unigene databases (maize, wheat, barley, sorghum and Arabidopsis) downloaded from NCBI. This search identified 708 hits (E&lt;e<sup>−10</sup>, identity ≥ 70%, and minimum alignment length &gt;50 bp). Clusters of TARs and sequence homologs were aligned using the CLUSTALW algorithm to generate multiple sequence alignment blocks suited for RNAz analysis <a href="#pone.0000294-Washietl1">[44]</a>. Alignment blocks of a minimum length of 40 were subjected to RNAz, utilizing an offset of 40 (for alignment blocks &gt;80) and considering both DNA strands independently. Regions with an RNAz support vector machine (SVM) classification score P&gt;0.5 were collected. As a control, multiple sequence alignment blocks were shuffled using a randomization algorithm that takes care not to introduce randomization artifacts and produces random alignments of the same length, base composition, overall conservation, local conservation pattern, and gap pattern as the input alignment <a href="#pone.0000294-Washietl1">[44]</a>.</p>


<h4>Data availability</h4>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1"></a><p>Information on <em>japonica</em> rice TARs can be viewed and retrieved at this website: <a href="http://dart.gersteinlab.org/rice/">http://dart.gersteinlab.org/rice/</a>. Microarray data for the full-genome tiling array, the 1Mb tiling array, and the re-array are available in the NCBI Gene Expression Omnibus under series GSE6996, GSE6921, and GSE6922, respectively.</p>

</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0000294"></div><a name="pone.0000294.s001" id="pone.0000294.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s001">Figure S1. </a></strong></p><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1"></a><p class="preSiDOI">Tiling array detection of annotated gene models in rice. The tiling arrays were hybridized to a mixture of cDNA targets derived from four major rice tissues to optimize transcript representation (see <a href="#s3">Methods</a>). The 62,121 annotated rice gene models include 13,467 TE and 48,654 non-TE models. Of these, 40,257 non-TE models (a), 14,285 PASA models (a set of improved gene models from incorporation of EST and FL-cDNA information) (b), and 5719 TE models (c) were considered uniquely and sufficiently represented in the tiling arrays. While 21,391 (53%) non-TE models were detected, a higher detection rate (73%) was obtained for PASA models, attesting to the accuracy of tiling array detection. Of the 5719 TE models, 1716 (30%) were detected, indicating that a significant portion of the rice TE models is actively transcribed.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s001</p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(0.24 MB PDF)</p>
<a name="pone.0000294.s002" id="pone.0000294.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s002">Figure S2. </a></strong></p><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1"></a><p class="preSiDOI">Density of different sets of transcription units along the 12 rice chromosomes. The number of transcription units was calculated in 500 Kb-sliding-windows with a 100 Kb step. The black triangle sign in each panel indicates the position of the annotated centromere of the corresponding chromosome.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s002</p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(1.29 MB PDF)</p>
<a name="pone.0000294.s003" id="pone.0000294.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s003">Figure S3. </a></strong></p><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1"></a><p class="preSiDOI">Comparative analysis of TARs. (a), Comparison of 5464 indica TARs with transcription units in japonica. The indica TARs were identified in a previous genome tiling analysis of the indica genome (Ref.2). Exon, indica TARs matched with exons of japonica genes, Antisense Exon, indica TARs mapped to the antisense strand of japonica genes, TAR, indica TARs matched with japonica novel TARs, Antisense TAR, indica TARs mapped to the antisense strand of japonica novel TARs, Unmatched, indica TARs do not match with any japonica transcription unit, Unmapped, indica TARs not mapped to the japonica genome. (b), Classification of 42,470 indica TARs based on their physical relation with annotated indica gene models. The 42,470 TARs were identified by re-analyzing the previous indica genome tiling data (Ref.2) using the same criteria described in <a href="#s3">Methods</a>. (c), Intersection of exonic TARs with common FL-cDNA supported gene (CG) models. Gene models in both japonica and indica were aligned to rice FL-cDNA to identify the CG models, which were mapped against each other using BLAT. Models were considered common if they overlap &gt;100 bp in the annotated ORFs (Ref.1). Exonic TARs from japonica (green) and indica (red) were matched against the common CG models (blue) and represented by the cyan and magenta color, respectively. TARs matched to the CG models in japonica and indica were compared using BLAT and considered common if they overlap by at least 1 bp (white).</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s003</p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.25 MB PDF)</p>
<a name="pone.0000294.s004" id="pone.0000294.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s004">Figure S4. </a></strong></p><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1"></a><p class="preSiDOI">Re-array data processing. (a) Normalization of the re-array hybridization data. Spatial-effect smoothed probe intensities were normalized following a two-step process. First, quantile normalization was performed on the three replicates within each assayed tissue type. Next, a global scaling was carried out to adjust the quantile-normalized intensities to a common baseline. (b) Modeling the 7669 control probes. To determine from the control probes the noise distribution, we used a previously reported modeling process (Ref.1). Assuming the control probes that have very low intensities (1-9 Log2) were primarily noise, we modeled the noise as a normal distribution by mirroring the low-intensity portion of the overall control probe distribution. (c) Transcription threshold determination for individual tissue types. FDR = 0.05, which represents the 95% confidence in the modeled noise distribution, was used as a cutoff for transcription calling.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s004</p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(1.25 MB PDF)</p>
<a name="pone.0000294.s005" id="pone.0000294.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s005">Figure S5. </a></strong></p><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1"></a><p class="preSiDOI">Transcriptional verification of non-exonic TARs. (a) Re-array detection of different groups of TARs. Detection rate was calculated as the percentage of TARs detected in at least one of the 10 assayed tissue types by the re-array. All, all non-exonic TARs; +Hit, TARs match with rice Unigenes; -Hit, TARs do not match with Unigenes. (b) RT-PCR analysis of selected non-exonic TARs. Pie charts indicate the percentage of TARs generated a PCR product of the expected size in an RT-dependent manner. PCR for 36 TARs from each group (108 in total) was performed on reverse transcribed cDNA (+RT) and mRNA (-RT) and resolved on gel side by side. A typical gel from each group is shown. Top, TARs detected by the re-array and have hits against Unigenes; middle, TARs detected by the re-array but have no Unigene hits; bottom, TARs not re-array detected and have no Unigene hits. M, 100 bp molecular weight marker.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s005</p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.88 MB PDF)</p>
<a name="pone.0000294.s006" id="pone.0000294.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s006">Figure S6. </a></strong></p><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1"></a><p class="preSiDOI">Analysis of the coding potential of intergenic TARs. (a) Frequency of the amino acids contained in the deduced protein sequences of rice FL-cDNA, TE and non-TE gene models, and intergenic TARs. (b) Pie chart analysis of the hits against the ProSite database using deduced polypeptide sequences from the intergenic TARs.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s006</p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.22 MB PDF)</p>
<a name="pone.0000294.s007" id="pone.0000294.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0000294.s007">Figure S7. </a></strong></p><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1"></a><p class="preSiDOI">High-resolution tiling array analysis of mRNA target. (a) A region from rice Chromosome 10 represented by the tiling array. (b) Tiling array profile showing each probe aligned to the chromosomal coordinate with its fluorescence intensity depicted as a vertical line. (c) TIGR working gene model from the locus LOC_Os10g21910, which encodes a putative acetyl-coenzyme A carboxylase ACC1A. (d) Gene predictions and experimental evidence supporting the gene model. Red bars indicate significant matches with the working gene model. Data downloaded from the TIGR rice genome annotation database at <a href="http://rice.tigr.org/">http://rice.tigr.org/</a>.</p>
<p class="siDoi">doi:10.1371/journal.pone.0000294.s007</p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(1.28 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank E. Eysteinsdottir, C. Leplar, X. Li, N. Su, M. Chern, N. Carriero, X. Zhu and J. Wang for technical assistance, O. Emanuelsson, T. E. Royce and J. S. Rozowsky for discussions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: MG XD LL. Performed the experiments: LL. Analyzed the data: VS WT LL XW RS HH JK. Contributed reagents/materials/analysis tools: PR RC VS WD XC. Wrote the paper: XD LL.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0000294-Goff1" id="pone.0000294-Goff1"></a>Goff SA, Ricke D, Lan TH, Presting G, Wang R, et al.  (2002) A draft sequence of the rice genome (<em>Oryza sativa</em> L<em>. ssp japonica</em>). Science  296: 92–100.  <ul class="find" data-citedArticleID="937410" data-doi="10.1126/science.1068275"><li><a href="http://dx.doi.org/10.1126/science.1068275" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp+japonica%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp+japonica%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0000294-Yu1" id="pone.0000294-Yu1"></a>Yu J, Hu S, Wang J, Wong GK, Li S, et al.  (2002) A draft sequence of the rice genome (<em>Oryza sativa</em> L. ssp. <em>indica</em>). Science  296: 79–92.  <ul class="find" data-citedArticleID="937488" data-doi="10.1126/science.1068037"><li><a href="http://dx.doi.org/10.1126/science.1068037" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp.+indica%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+draft+sequence+of+the+rice+genome+%28Oryza+sativa+L.+ssp.+indica%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0000294-Yu2" id="pone.0000294-Yu2"></a>Yu J, Wang J, Lin W, Li S, Li H, et al.  (2005) The genomes of <em>Oryza sativa</em>: A history of duplications. PLOS Biol  3: e38.  <ul class="find" data-citedArticleID="937490" data-doi="10.1371/journal.pbio.0030038"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030038" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genomes+of+Oryza+sativa%3A+A+history+of+duplications." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genomes+of+Oryza+sativa%3A+A+history+of+duplications.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0000294-Sasaki1" id="pone.0000294-Sasaki1"></a>Sasaki T, Matsumoto T, Yamamoto K, Sakata K, Baba T, et al.  (2002) The genome sequence and structure of rice chromosome 1. Nature  420: 312–316.  <ul class="find" data-citedArticleID="937464"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=The+genome+sequence+and+structure+of+rice+chromosome+1.&amp;auth=&amp;atitle=The+genome+sequence+and+structure+of+rice+chromosome+1." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+genome+sequence+and+structure+of+rice+chromosome+1." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+genome+sequence+and+structure+of+rice+chromosome+1.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0000294-Feng1" id="pone.0000294-Feng1"></a>Feng Q, Zhang Y, Hao P, Wang S, Fu G, et al.  (2002) Sequence and analysis of rice chromosome 4. Nature  420: 316–320.  <ul class="find" data-citedArticleID="937408"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Sequence+and+analysis+of+rice+chromosome+4.&amp;auth=&amp;atitle=Sequence+and+analysis+of+rice+chromosome+4." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Sequence+and+analysis+of+rice+chromosome+4." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Sequence+and+analysis+of+rice+chromosome+4.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0000294-The1" id="pone.0000294-The1"></a>The Rice Chromosome 10 Sequencing Consortium (2003) In-depth view of structure, activity, and evolution of rice Chromosome 10. Science  300: 1566–1569.  <ul class="find" data-citedArticleID="937474" data-doi="10.1126/science.1083523"><li><a href="http://dx.doi.org/10.1126/science.1083523" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=In-depth+view+of+structure%2C+activity%2C+and+evolution+of+rice+Chromosome+10." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22In-depth+view+of+structure%2C+activity%2C+and+evolution+of+rice+Chromosome+10.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0000294-International1" id="pone.0000294-International1"></a>International Rice Genome Sequencing Project (2005) The map-based sequence of the rice genome. Nature  436: 793–800.  <ul class="find" data-citedArticleID="937416" data-doi="10.1038/nature03895"><li><a href="http://dx.doi.org/10.1038/nature03895" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+map-based+sequence+of+the+rice+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+map-based+sequence+of+the+rice+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0000294-Yuan1" id="pone.0000294-Yuan1"></a>Yuan Q, Ouyang S, Wang A, Zhu W, Maiti R, et al.  (2005) The institute for genomic research osa1 rice genome annotation database. Plant Physiol  138: 18–26.  <ul class="find" data-citedArticleID="937492" data-doi="10.1104/pp.104.059063"><li><a href="http://dx.doi.org/10.1104/pp.104.059063" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+institute+for+genomic+research+osa1+rice+genome+annotation+database." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+institute+for+genomic+research+osa1+rice+genome+annotation+database.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0000294-Kikuchi1" id="pone.0000294-Kikuchi1"></a>Kikuchi S, Satoh K, Nagata T, Kawagashira N, Doi K, et al.  (2003) Collection, mapping, and annotation of over 28,000 cDNA clones from <em>japonica</em> rice. Science  300: 1566–1569.  <ul class="find" data-citedArticleID="937432" data-doi="10.1126/science.1081288"><li><a href="http://dx.doi.org/10.1126/science.1081288" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Collection%2C+mapping%2C+and+annotation+of+over+28%2C000+cDNA+clones+from+japonica+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Collection%2C+mapping%2C+and+annotation+of+over+28%2C000+cDNA+clones+from+japonica+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0000294-Juretic1" id="pone.0000294-Juretic1"></a>Juretic N, Bureau TE, Bruskiewich RM (2004) Transposable element annotation of the rice genome. Bioinformatics  20: 155–160.  <ul class="find" data-citedArticleID="937424" data-doi="10.1093/bioinformatics/bth019"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth019" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transposable+element+annotation+of+the+rice+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transposable+element+annotation+of+the+rice+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0000294-Matsumura1" id="pone.0000294-Matsumura1"></a>Matsumura H, Nirasawa S, Terauchi R (1999) Transcript profiling in rice (<em>Oryza sativa</em> L.) seedlings using serial analysis of gene expression (SAGE). Plant J  20: 719–726.  <ul class="find" data-citedArticleID="937446" data-doi="10.1046/j.1365-313x.1999.00640.x"><li><a href="http://dx.doi.org/10.1046/j.1365-313x.1999.00640.x" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcript+profiling+in+rice+%28Oryza+sativa+L.%29+seedlings+using+serial+analysis+of+gene+expression+%28SAGE%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcript+profiling+in+rice+%28Oryza+sativa+L.%29+seedlings+using+serial+analysis+of+gene+expression+%28SAGE%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0000294-Nakano1" id="pone.0000294-Nakano1"></a>Nakano M, Nobuta K, Vemaraju K, Tej SS, Skogen JW, et al.  (2006) Plant MPSS databases: signature-based transcriptional resources for analyses of mRNA and small RNA. Nucleic Acids Res  34: D731–D735.  <ul class="find" data-citedArticleID="937454" data-doi="10.1093/nar/gkj077"><li><a href="http://dx.doi.org/10.1093/nar/gkj077" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Plant+MPSS+databases%3A+signature-based+transcriptional+resources+for+analyses+of+mRNA+and+small+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Plant+MPSS+databases%3A+signature-based+transcriptional+resources+for+analyses+of+mRNA+and+small+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0000294-Matsumura2" id="pone.0000294-Matsumura2"></a>Matsumura H, Bin Nasir KH, Yoshida K, Ito A, Kahl G, et al.  (2006) SuperSAGE array: the direct use of 26-base-pair transcript tags in oligonucleotide arrays. Nat Methods  3: 469–474.  <ul class="find" data-citedArticleID="937448" data-doi="10.1038/nmeth882"><li><a href="http://dx.doi.org/10.1038/nmeth882" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=SuperSAGE+array%3A+the+direct+use+of+26-base-pair+transcript+tags+in+oligonucleotide+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22SuperSAGE+array%3A+the+direct+use+of+26-base-pair+transcript+tags+in+oligonucleotide+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0000294-Li1" id="pone.0000294-Li1"></a>Li L, Wang X, Xia M, Stolc V, Su N, et al.  (2005) Tiling microarray analysis of rice chromosome 10 to identify the transcriptome and relate its expression to chromosomal architecture. Genome Biol  6: R52.  <ul class="find" data-citedArticleID="937440"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Tiling+microarray+analysis+of+rice+chromosome+10+to+identify+the+transcriptome+and+relate+its+expression+to+chromosomal+architecture.&amp;auth=&amp;atitle=Tiling+microarray+analysis+of+rice+chromosome+10+to+identify+the+transcriptome+and+relate+its+expression+to+chromosomal+architecture." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tiling+microarray+analysis+of+rice+chromosome+10+to+identify+the+transcriptome+and+relate+its+expression+to+chromosomal+architecture." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tiling+microarray+analysis+of+rice+chromosome+10+to+identify+the+transcriptome+and+relate+its+expression+to+chromosomal+architecture.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0000294-Li2" id="pone.0000294-Li2"></a>Li L, Wang X, Stolc V, Li X, Zhang D, et al.  (2006) Genome-wide transcription analyses in rice using tiling microarrays. Nat Genet  38: 124–129.  <ul class="find" data-citedArticleID="937442" data-doi="10.1038/ng1704"><li><a href="http://dx.doi.org/10.1038/ng1704" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+transcription+analyses+in+rice+using+tiling+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+transcription+analyses+in+rice+using+tiling+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0000294-Mockler1" id="pone.0000294-Mockler1"></a>Mockler TC, Ecker JR (2005) Applications of DNA tiling arrays for whole-genome analysis. Genomics  85: 1–15.  <ul class="find" data-citedArticleID="937450" data-doi="10.1016/j.ygeno.2004.10.005"><li><a href="http://dx.doi.org/10.1016/j.ygeno.2004.10.005" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Applications+of+DNA+tiling+arrays+for+whole-genome+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Applications+of+DNA+tiling+arrays+for+whole-genome+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0000294-Johnson1" id="pone.0000294-Johnson1"></a>Johnson JM, Edwards S, Shoemaker D, Schadt EE (2005) Dark matter in the genome: evidence of widespread transcription detected by microarray tiling experiments. Trends Genet  21: 93–102.  <ul class="find" data-citedArticleID="937422" data-doi="10.1016/j.tig.2004.12.009"><li><a href="http://dx.doi.org/10.1016/j.tig.2004.12.009" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dark+matter+in+the+genome%3A+evidence+of+widespread+transcription+detected+by+microarray+tiling+experiments." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dark+matter+in+the+genome%3A+evidence+of+widespread+transcription+detected+by+microarray+tiling+experiments.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0000294-Hughes1" id="pone.0000294-Hughes1"></a>Hughes TR, Mao M, Jones AR, Burchard J, Marton MJ, et al.  (2001) Expression profiling using microarrays fabricated by an ink-jet oligonucleotide synthesizer. Nat Biotechnol  19: 342–347.  <ul class="find" data-citedArticleID="937414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer.&amp;auth=&amp;atitle=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Expression+profiling+using+microarrays+fabricated+by+an+ink-jet+oligonucleotide+synthesizer.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0000294-SinghGasson1" id="pone.0000294-SinghGasson1"></a>Singh-Gasson S, Green RD, Yue YJ, Nelson C, Blattner F, et al.  (1999) Maskless fabrication of light-directed oligonucleotide microarrays using a digital micromirror array. Nat Biotechnol  17: 974–978.  <ul class="find" data-citedArticleID="937470" data-doi="10.1038/13664"><li><a href="http://dx.doi.org/10.1038/13664" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Maskless+fabrication+of+light-directed+oligonucleotide+microarrays+using+a+digital+micromirror+array." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Maskless+fabrication+of+light-directed+oligonucleotide+microarrays+using+a+digital+micromirror+array.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0000294-Yamada1" id="pone.0000294-Yamada1"></a>Yamada K, Lim J, Dale JM, Chen H, Shinn P, et al.  (2003) Empirical analysis of transcriptional activity in the <em>Arabidopsis</em> genome. Science  302: 842–846.  <ul class="find" data-citedArticleID="937484" data-doi="10.1126/science.1088305"><li><a href="http://dx.doi.org/10.1126/science.1088305" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Empirical+analysis+of+transcriptional+activity+in+the+Arabidopsis+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Empirical+analysis+of+transcriptional+activity+in+the+Arabidopsis+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0000294-Shoemaker1" id="pone.0000294-Shoemaker1"></a>Shoemaker DD, Schadt EE, Armour CD, He YD, Garrett-Engele P, et al.  (2001) Experimental annotation of the human genome using microarray technology. Nature  409: 922–927.  <ul class="find" data-citedArticleID="937468" data-doi="10.1038/35057141"><li><a href="http://dx.doi.org/10.1038/35057141" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Experimental+annotation+of+the+human+genome+using+microarray+technology." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Experimental+annotation+of+the+human+genome+using+microarray+technology.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0000294-Selinger1" id="pone.0000294-Selinger1"></a>Selinger DW, Cheung KJ, Mei R, Johansson EM, Richmond CS, et al.  (2000) RNA expression analysis using a 30 base pair resolution <em>Escherichia coli</em> genome array. Nature Biotechnol  18: 1262–1268.  <ul class="find" data-citedArticleID="937466"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=RNA+expression+analysis+using+a+30+base+pair+resolution+Escherichia+coli+genome+array.&amp;auth=&amp;atitle=RNA+expression+analysis+using+a+30+base+pair+resolution+Escherichia+coli+genome+array." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+expression+analysis+using+a+30+base+pair+resolution+Escherichia+coli+genome+array." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+expression+analysis+using+a+30+base+pair+resolution+Escherichia+coli+genome+array.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0000294-Kapranov1" id="pone.0000294-Kapranov1"></a>Kapranov P, Cawley SE, Drenkow J, Bekiranov S, Strausberg RL, et al.  (2002) Large-scale transcriptional activity in chromosomes 21 and 22. Science  296: 916–919.  <ul class="find" data-citedArticleID="937428" data-doi="10.1126/science.1068597"><li><a href="http://dx.doi.org/10.1126/science.1068597" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Large-scale+transcriptional+activity+in+chromosomes+21+and+22." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Large-scale+transcriptional+activity+in+chromosomes+21+and+22.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0000294-Bertone1" id="pone.0000294-Bertone1"></a>Bertone P, Stolc V, Royce TE, Rozowsky JS, Urban AE, et al.  (2004) Global identification of human transcribed sequences with genome tiling arrays. Science  306: 2242–2246.  <ul class="find" data-citedArticleID="937398" data-doi="10.1126/science.1103388"><li><a href="http://dx.doi.org/10.1126/science.1103388" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Global+identification+of+human+transcribed+sequences+with+genome+tiling+arrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Global+identification+of+human+transcribed+sequences+with+genome+tiling+arrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0000294-Cheng1" id="pone.0000294-Cheng1"></a>Cheng J, Kapranov P, Drenkow J, Dike S, Brubaker S, et al.  (2005) Transcriptional maps of 10 human chromosomes at 5-nucleotide resolution. Science  308: 1149–1154.  <ul class="find" data-citedArticleID="937402" data-doi="10.1126/science.1108625"><li><a href="http://dx.doi.org/10.1126/science.1108625" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Transcriptional+maps+of+10+human+chromosomes+at+5-nucleotide+resolution.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0000294-Stolc1" id="pone.0000294-Stolc1"></a>Stolc V, Gauhar Z, Mason C, Halasz G, van Batenburg MF, et al.  (2004) A gene expression map for the euchromatic genome of Drosophila melanogaster. Science  306: 655–660.  <ul class="find" data-citedArticleID="937472" data-doi="10.1126/science.1101312"><li><a href="http://dx.doi.org/10.1126/science.1101312" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+gene+expression+map+for+the+euchromatic+genome+of+Drosophila+melanogaster." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+gene+expression+map+for+the+euchromatic+genome+of+Drosophila+melanogaster.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0000294-Haas1" id="pone.0000294-Haas1"></a>Haas BJ, Delcher AL, Mount SM, Wortman JR, Smith RK, et al.  (2003) Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies. Nucleic Acids Res  31: 5654–5666.  <ul class="find" data-citedArticleID="937412" data-doi="10.1093/nar/gkg770"><li><a href="http://dx.doi.org/10.1093/nar/gkg770" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Improving+the+Arabidopsis+genome+annotation+using+maximal+transcript+alignment+assemblies." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Improving+the+Arabidopsis+genome+annotation+using+maximal+transcript+alignment+assemblies.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0000294-Kitagawa1" id="pone.0000294-Kitagawa1"></a>Kitagawa N, Washio T, Kosugi S, Yamashita T, Higashi K, et al.  (2005) Computational analysis suggests that alternative first exons are involved in tissue-specific transcription in rice (<em>Oryza sativa</em>). Bioinformatics  21: 1758–1763.  <ul class="find" data-citedArticleID="937434" data-doi="10.1093/bioinformatics/bti253"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bti253" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+analysis+suggests+that+alternative+first+exons+are+involved+in+tissue-specific+transcription+in+rice+%28Oryza+sativa%29." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+analysis+suggests+that+alternative+first+exons+are+involved+in+tissue-specific+transcription+in+rice+%28Oryza+sativa%29.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0000294-Munroe1" id="pone.0000294-Munroe1"></a>Munroe SH, Lazar MA (1991) Inhibition of c-erbA mRNA splicing by a naturally occurring antisense RNA. J Biol Chem  266: 22083–22086.  <ul class="find" data-citedArticleID="937452"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Inhibition+of+c-erbA+mRNA+splicing+by+a+naturally+occurring+antisense+RNA.&amp;auth=&amp;atitle=Inhibition+of+c-erbA+mRNA+splicing+by+a+naturally+occurring+antisense+RNA." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Inhibition+of+c-erbA+mRNA+splicing+by+a+naturally+occurring+antisense+RNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Inhibition+of+c-erbA+mRNA+splicing+by+a+naturally+occurring+antisense+RNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0000294-Farrell1" id="pone.0000294-Farrell1"></a>Farrell CM, Lukens LN (1995) Naturally occurring antisense transcripts are present in chick embryo chondrocytes simultaneously with the down-regulation of the alpha 1 (I) collagen gene. J Biol Chem  270: 3400–3408.  <ul class="find" data-citedArticleID="937406" data-doi="10.1074/jbc.270.7.3400"><li><a href="http://dx.doi.org/10.1074/jbc.270.7.3400" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Naturally+occurring+antisense+transcripts+are+present+in+chick+embryo+chondrocytes+simultaneously+with+the+down-regulation+of+the+alpha+1+%28I%29+collagen+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Naturally+occurring+antisense+transcripts+are+present+in+chick+embryo+chondrocytes+simultaneously+with+the+down-regulation+of+the+alpha+1+%28I%29+collagen+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0000294-Lee1" id="pone.0000294-Lee1"></a>Lee JT, Davidow LS, Warshawsky D (1999) Tsix, a gene antisense to Xist at the X-inactivation centre. Nat Genet  21: 400–404.  <ul class="find" data-citedArticleID="937436" data-doi="10.1038/7734"><li><a href="http://dx.doi.org/10.1038/7734" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Tsix%2C+a+gene+antisense+to+Xist+at+the+X-inactivation+centre." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Tsix%2C+a+gene+antisense+to+Xist+at+the+X-inactivation+centre.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0000294-Lippman1" id="pone.0000294-Lippman1"></a>Lippman Z, Martienssen RA (2004) The role of RNA interference in heterochromatic silencing. Nature  431: 364–370.  <ul class="find" data-citedArticleID="937444" data-doi="10.1038/nature02875"><li><a href="http://dx.doi.org/10.1038/nature02875" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+role+of+RNA+interference+in+heterochromatic+silencing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+role+of+RNA+interference+in+heterochromatic+silencing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0000294-Borsani1" id="pone.0000294-Borsani1"></a>Borsani O, Zhu J, Verslues PE, Sunkar R, Zhu JK (2005) Endogenous siRNAs derived from a pair of natural cis-antisense transcripts regulate salt tolerance in Arabidopsis. Cell  123: 1279–1291.  <ul class="find" data-citedArticleID="937400" data-doi="10.1016/j.cell.2005.11.035"><li><a href="http://dx.doi.org/10.1016/j.cell.2005.11.035" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Endogenous+siRNAs+derived+from+a+pair+of+natural+cis-antisense+transcripts+regulate+salt+tolerance+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Endogenous+siRNAs+derived+from+a+pair+of+natural+cis-antisense+transcripts+regulate+salt+tolerance+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0000294-Ronemus1" id="pone.0000294-Ronemus1"></a>Ronemus M, Vaughn MW, Martienssen RA (2006) MicroRNA-targeted and small interfering RNA–mediated mRNA degradation is regulated by Argonaute, Dicer, and RNA-dependent RNA polymerase in <em>Arabidopsis</em>. Plant Cell  18: 1559–1574.  <ul class="find" data-citedArticleID="937458" data-doi="10.1105/tpc.106.042127"><li><a href="http://dx.doi.org/10.1105/tpc.106.042127" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=MicroRNA-targeted+and+small+interfering+RNA%26%23x2013%3Bmediated+mRNA+degradation+is+regulated+by+Argonaute%2C+Dicer%2C+and+RNA-dependent+RNA+polymerase+in+Arabidopsis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22MicroRNA-targeted+and+small+interfering+RNA%26%23x2013%3Bmediated+mRNA+degradation+is+regulated+by+Argonaute%2C+Dicer%2C+and+RNA-dependent+RNA+polymerase+in+Arabidopsis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0000294-Lehner1" id="pone.0000294-Lehner1"></a>Lehner B, Williams G, Campbell RD, Sanderson CM (2002) Antisense transcripts in the human genome. Trends Genet  18: 63–65.  <ul class="find" data-citedArticleID="937438" data-doi="10.1016/s0168-9525(02)02598-2"><li><a href="http://dx.doi.org/10.1016/s0168-9525(02)02598-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+transcripts+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+transcripts+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0000294-Yelin1" id="pone.0000294-Yelin1"></a>Yelin R, Dahary D, Sorek R, Levanon EY, Goldstein O, et al.  (2003) Widespread occurrence of antisense transcription in the human genome. Nat Biotechnol  21: 379–386.  <ul class="find" data-citedArticleID="937486" data-doi="10.1038/nbt808"><li><a href="http://dx.doi.org/10.1038/nbt808" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Widespread+occurrence+of+antisense+transcription+in+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Widespread+occurrence+of+antisense+transcription+in+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0000294-Wang1" id="pone.0000294-Wang1"></a>Wang XJ, Gaasterland T, Chua NH (2005) Genome-wide prediction and identification of cis-natural antisense transcripts in <em>Arabidopsis thaliana</em>. Genome Biol  6: R30.  <ul class="find" data-citedArticleID="937476"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Genome-wide+prediction+and+identification+of+cis-natural+antisense+transcripts+in+Arabidopsis+thaliana.&amp;auth=&amp;atitle=Genome-wide+prediction+and+identification+of+cis-natural+antisense+transcripts+in+Arabidopsis+thaliana." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+prediction+and+identification+of+cis-natural+antisense+transcripts+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+prediction+and+identification+of+cis-natural+antisense+transcripts+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0000294-Osato1" id="pone.0000294-Osato1"></a>Osato N, Yamada H, Satoh K, Ooka H, Yamamoto M, et al.  (2003) Antisense transcripts with rice full-length cDNAs. Genome Biol  5: R5.  <ul class="find" data-citedArticleID="937456" data-doi="10.1186/gb-2003-5-1-r5"><li><a href="http://dx.doi.org/10.1186/gb-2003-5-1-r5" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Antisense+transcripts+with+rice+full-length+cDNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Antisense+transcripts+with+rice+full-length+cDNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0000294-Jiang1" id="pone.0000294-Jiang1"></a>Jiang N, Bao Z, Zhang X, Eddy SR, Wessler SR (2004) Pack-MULE transposable elements mediate gene evolution in plants. Nature  431: 569–573.  <ul class="find" data-citedArticleID="937420" data-doi="10.1038/nature02953"><li><a href="http://dx.doi.org/10.1038/nature02953" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pack-MULE+transposable+elements+mediate+gene+evolution+in+plants." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pack-MULE+transposable+elements+mediate+gene+evolution+in+plants.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0000294-Juretic2" id="pone.0000294-Juretic2"></a>Juretic N, Hoen DR, Huynh ML, Harrison PM, Bureau TE (2005) The evolutionary fate of MULE-mediated duplications of host gene fragments in rice. Genome Res  15: 1292–1297.  <ul class="find" data-citedArticleID="937426" data-doi="10.1101/gr.4064205"><li><a href="http://dx.doi.org/10.1101/gr.4064205" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+evolutionary+fate+of+MULE-mediated+duplications+of+host+gene+fragments+in+rice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+evolutionary+fate+of+MULE-mediated+duplications+of+host+gene+fragments+in+rice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0000294-Wang2" id="pone.0000294-Wang2"></a>Wang H, Chua NH, Wang XJ (2006) Prediction of <em>trans</em>-antisense transcripts in <em>Arabidopsis thaliana</em>. Genome Biol  7: R92.  <ul class="find" data-citedArticleID="937478" data-doi="10.1186/gb-2006-7-10-r92"><li><a href="http://dx.doi.org/10.1186/gb-2006-7-10-r92" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Prediction+of+trans-antisense+transcripts+in+Arabidopsis+thaliana." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Prediction+of+trans-antisense+transcripts+in+Arabidopsis+thaliana.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0000294-Jabbari1" id="pone.0000294-Jabbari1"></a>Jabbari K, Cruveiller S, Clay O, Le Saux J, Bernardi G (2004) The new genes of rice: a closer look. Trends Plant Sci  9: 281–285.  <ul class="find" data-citedArticleID="937418" data-doi="10.1016/j.tplants.2004.04.006"><li><a href="http://dx.doi.org/10.1016/j.tplants.2004.04.006" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+new+genes+of+rice%3A+a+closer+look." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+new+genes+of+rice%3A+a+closer+look.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0000294-Deng1" id="pone.0000294-Deng1"></a>Deng W, Zhu X, Skogerbo G, Zhao Y, Fu Z, et al.  (2006) Organization of the <em>Caenorhabditis elegans</em> small non-coding transcriptome: genomic features, biogenesis, and expression. Genome Res  16: 20–29.  <ul class="find" data-citedArticleID="937404" data-doi="10.1101/gr.4139206"><li><a href="http://dx.doi.org/10.1101/gr.4139206" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Organization+of+the+Caenorhabditis+elegans+small+non-coding+transcriptome%3A+genomic+features%2C+biogenesis%2C+and+expression." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Organization+of+the+Caenorhabditis+elegans+small+non-coding+transcriptome%3A+genomic+features%2C+biogenesis%2C+and+expression.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">44.
              </span><a name="pone.0000294-Washietl1" id="pone.0000294-Washietl1"></a>Washietl S, Hofacker IL, Stadler PF (2005) Fast and reliable prediction of noncoding RNAs. Proc Natl Acad Sci USA  102: 2454–2459.  <ul class="find" data-citedArticleID="937482" data-doi="10.1073/pnas.0409169102"><li><a href="http://dx.doi.org/10.1073/pnas.0409169102" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Fast+and+reliable+prediction+of+noncoding+RNAs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Fast+and+reliable+prediction+of+noncoding+RNAs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">45.
              </span><a name="pone.0000294-Kauffman1" id="pone.0000294-Kauffman1"></a>Kauffman HE, Reddy APK, Hsieh SPY, Merca SD (1973) An improved technique for evaluating resistance of rice varieties to <em>Xanthomonas oryzae</em>. Plant Dis Rep  57: 537–541.  <ul class="find" data-citedArticleID="937430"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+improved+technique+for+evaluating+resistance+of+rice+varieties+to+Xanthomonas+oryzae.&amp;auth=&amp;atitle=An+improved+technique+for+evaluating+resistance+of+rice+varieties+to+Xanthomonas+oryzae." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+improved+technique+for+evaluating+resistance+of+rice+varieties+to+Xanthomonas+oryzae." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+improved+technique+for+evaluating+resistance+of+rice+varieties+to+Xanthomonas+oryzae.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">46.
              </span><a name="pone.0000294-Rouillard1" id="pone.0000294-Rouillard1"></a>Rouillard JM, Zuker M, Gulari E (2003) OligoArray 2.0: design of oligonucleotide probes for DNA microarrays using a thermodynamic approach. Nucleic Acids Res  31: 3057–3062.  <ul class="find" data-citedArticleID="937460" data-doi="10.1093/nar/gkg426"><li><a href="http://dx.doi.org/10.1093/nar/gkg426" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=OligoArray+2.0%3A+design+of+oligonucleotide+probes+for+DNA+microarrays+using+a+thermodynamic+approach." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22OligoArray+2.0%3A+design+of+oligonucleotide+probes+for+DNA+microarrays+using+a+thermodynamic+approach.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">47.
              </span><a name="pone.0000294-Royce1" id="pone.0000294-Royce1"></a>Royce TE, Rozowsky JS, Bertone P, Samanta M, Stolc V, et al.  (2005) Issues in the analysis of oligonucleotide tiling microarrays for transcript mapping. Trends Genet  21: 466–475.  <ul class="find" data-citedArticleID="937462" data-doi="10.1016/j.tig.2005.06.007"><li><a href="http://dx.doi.org/10.1016/j.tig.2005.06.007" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Issues+in+the+analysis+of+oligonucleotide+tiling+microarrays+for+transcript+mapping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Issues+in+the+analysis+of+oligonucleotide+tiling+microarrays+for+transcript+mapping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">48.
              </span><a name="pone.0000294-Wang3" id="pone.0000294-Wang3"></a>Wang X, He H, Li L, Chen R, Deng XW, et al.  (2006) NMPP: a user-customized NimbleGen microarray data processing pipeline. Bioinformatics  22: 2955–2957.  <ul class="find" data-citedArticleID="937480" data-doi="10.1093/bioinformatics/btl525"><li><a href="http://dx.doi.org/10.1093/bioinformatics/btl525" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=NMPP%3A+a+user-customized+NimbleGen+microarray+data+processing+pipeline." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22NMPP%3A+a+user-customized+NimbleGen+microarray+data+processing+pipeline.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.XML" value="115890"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.PDF" value="748812"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g001.PNG_L" value="898673"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g001.PNG_M" value="140322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g001.PNG_S" value="4590"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g001.TIF" value="1211058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g001.PNG_I" value="52279"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g002.PNG_L" value="1309881"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g002.PNG_M" value="152667"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g002.PNG_S" value="6733"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g002.TIF" value="1819382"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g002.PNG_I" value="65996"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g003.PNG_L" value="817509"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g003.PNG_M" value="137037"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g003.PNG_S" value="5116"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g003.TIF" value="1123454"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g003.PNG_I" value="50703"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g004.PNG_L" value="1662110"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g004.PNG_M" value="175142"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g004.PNG_S" value="6072"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g004.TIF" value="2942574"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g004.PNG_I" value="83940"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.t001.PNG_L" value="173063"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.t001.PNG_M" value="62708"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.t001.PNG_S" value="2796"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.t001.TIF" value="764482"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.t001.PNG_I" value="26690"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g005.PNG_L" value="2687770"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g005.PNG_M" value="165221"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g005.PNG_S" value="9403"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g005.TIF" value="3494242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g005.PNG_I" value="138095"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g006.PNG_L" value="3096729"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g006.PNG_M" value="188043"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g006.PNG_S" value="7546"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g006.TIF" value="4120102"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g006.PNG_I" value="104493"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g007.PNG_L" value="599551"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g007.PNG_M" value="88242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g007.PNG_S" value="3291"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g007.TIF" value="1165260"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.g007.PNG_I" value="32058"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.e001.PNG" value="3292"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.e001.TIF" value="25216"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s001.PDF" value="243868"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s002.PDF" value="1290746"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s003.PDF" value="250999"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s004.PDF" value="1252244"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s005.PDF" value="882483"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s006.PDF" value="220156"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0000294.s007.PDF" value="1275859"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000294&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0000294" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0000294&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0000294&volume=&issue=&title=Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome&author_name=Lei%20Li%2C%20Xiangfeng%20Wang%2C%20Rajkumar%20Sasidharan%2C%20Viktor%20Stolc%2C%20Wei%20Deng%2C%20Hang%20He%2C%20Jan%20Korbel%2C%20Xuewei%20Chen%2C%20Waraporn%20Tongprasit%2C%20Pamela%20Ronald%2C%20Runsheng%20Chen%2C%20Mark%20Gerstein%2C%20Xing%20Wang%20Deng&start_page=1&end_page=13" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0000294" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294&amp;t=Global%20Identification%20and%20Characterization%20of%20Transcriptionally%20Active%20Regions%20in%20the%20Rice%20Genome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294&title=Global%20Identification%20and%20Characterization%20of%20Transcriptionally%20Active%20Regions%20in%20the%20Rice%20Genome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294&amp;title=Global%20Identification%20and%20Characterization%20of%20Transcriptionally%20Active%20Regions%20in%20the%20Rice%20Genome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0000294&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'Global Identification and Characterization of Transcriptionally Active Regions in the Rice Genome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0000294';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Global%20Identification%20and%20Characterization%20of%20Transcriptionally%20Active%20Regions%20in%20the%20Rice%20Genome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0000294" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0000294" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">



















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+transcription%22" title="Search for articles in the subject area:'DNA transcription'"><div class="flagText">DNA transcription</div></a>
              <div data-categoryid="20231" data-articleid="23426"
                   data-categoryname="DNA transcription"
                   class="flagImage" title="Flag 'DNA transcription' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genome+annotation%22" title="Search for articles in the subject area:'Genome annotation'"><div class="flagText">Genome annotation</div></a>
              <div data-categoryid="40359" data-articleid="23426"
                   data-categoryname="Genome annotation"
                   class="flagImage" title="Flag 'Genome annotation' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="23426"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Plant+genomics%22" title="Search for articles in the subject area:'Plant genomics'"><div class="flagText">Plant genomics</div></a>
              <div data-categoryid="33365" data-articleid="23426"
                   data-categoryname="Plant genomics"
                   class="flagImage" title="Flag 'Plant genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Rice%22" title="Search for articles in the subject area:'Rice'"><div class="flagText">Rice</div></a>
              <div data-categoryid="31915" data-articleid="23426"
                   data-categoryname="Rice"
                   class="flagImage" title="Flag 'Rice' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Seedlings%22" title="Search for articles in the subject area:'Seedlings'"><div class="flagText">Seedlings</div></a>
              <div data-categoryid="43017" data-articleid="23426"
                   data-categoryname="Seedlings"
                   class="flagImage" title="Flag 'Seedlings' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="23426"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Tar%22" title="Search for articles in the subject area:'Tar'"><div class="flagText">Tar</div></a>
              <div data-categoryid="46463" data-articleid="23426"
                   data-categoryname="Tar"
                   class="flagImage" title="Flag 'Tar' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=6042'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9145'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=1203&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
