

 



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:foaf="http://xmlns.com/foaf/0.1/"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">
<head prefix="og: http://ogp.me/ns#">
  <title>PLOS ONE: An Intronic Signal for Alternative Splicing in the Human Genome</title>


<link rel="stylesheet" type="text/css"  href="/css/global-min.css?v=izteQ6tu7kgsJZW_xmrYizvKiHM" />


    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->


<link rel="stylesheet" type="text/css"  href="/css/jquery-ui-min.css?v=eXDHTEJM0lIAmDe5k0I0Ad4nxNo" />


<link rel="stylesheet" type="text/css"  href="/css/journal.css?v=T7ZVxJfgk9jNxLAJ2qHz1vZpgYU" />


<link rel="stylesheet" type="text/css" media="print" href="/css/print-min.css?v=T5lb0B3q6EXBsuDluc5V5w+AkRc" />


  <link rel="stylesheet" href="http://f.fontdeck.com/s/css/js/www.plosone.org/24557.css" type="text/css"/>

  <!--chartbeat -->
  <script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9"/>
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world."/>
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering"/>
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles"/>
  <meta name="searchHost" content="http://api.plos.org/search" />
  <meta name="termsHost" content="http://api.plos.org/terms" />
  <meta name="solrApiKey" content="plos"/>
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft" />
  <meta name="currentJournal" content="PLoSONE" />
  <meta name="almRequestBatchSize" content="" />

  <meta name="citation_publisher" content="Public Library of Science"/>
  <meta name="citation_doi" content="10.1371/journal.pone.0001246"/>
  <meta name="dc.identifier" content="10.1371/journal.pone.0001246" />

    <meta name="citation_title" content="An Intronic Signal for Alternative Splicing in the Human Genome"/>
    <meta itemprop="name" content="An Intronic Signal for Alternative Splicing in the Human Genome"/>

      <meta name="citation_author" content="Necat Havlioglu"/>
            <meta name="citation_author_institution" content="Department of Pathology, Saint Louis University, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Jun Wang"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolution, The University of Chicago, Chicago, Illinois, United States of America"/>
      <meta name="citation_author" content="Kazuo Fushimi"/>
            <meta name="citation_author_institution" content="Department of Neurology, Lurie Comprehensive Cancer Center, Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, United States of America"/>
      <meta name="citation_author" content="Maria D. Vibranovski"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolution, The University of Chicago, Chicago, Illinois, United States of America"/>
      <meta name="citation_author" content="Zhengyan Kan"/>
            <meta name="citation_author_institution" content="Department of Genetics, Washington University in St. Louis, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Warren Gish"/>
            <meta name="citation_author_institution" content="Department of Genetics, Washington University in St. Louis, St. Louis, Missouri, United States of America"/>
      <meta name="citation_author" content="Alexei Fedorov"/>
            <meta name="citation_author_institution" content="Department of Medicine and Program in Bioinformatics and Proteomics/Genomics, Medical University of Ohio, Toledo, Ohio, United States of America"/>
      <meta name="citation_author" content="Manyuan Long"/>
            <meta name="citation_author_institution" content="Department of Ecology and Evolution, The University of Chicago, Chicago, Illinois, United States of America"/>
      <meta name="citation_author" content="Jane Y. Wu"/>
            <meta name="citation_author_institution" content="Department of Neurology, Lurie Comprehensive Cancer Center, Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, United States of America"/>

    <meta name="citation_date" content="2007/11/28"/>

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0001246.pdf" />

      <meta name="citation_journal_title" content="PLOS ONE" />
    <meta name="citation_firstpage" content="e1246"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_volume" content="2"/>
    <meta name="citation_issn" content="1932-6203"/>

    <meta name="citation_journal_abbrev" content="PLoS ONE" />

      <meta name="citation_reference" content="citation_title=Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays.; citation_author=JM Johnson; citation_author=J Castle; citation_author=P Garrett-Engele; citation_author=ZY Kan; citation_author=PM Loerch; citation_journal_title=Science; citation_volume=302; citation_number=1; citation_pages=2141-4; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Alternative pre-mRNA splicing and regulation of programmed cell death.; citation_author=JY Wu; citation_author=H Tang; citation_author=N Havlioglu; citation_journal_title=Prog Mol Subcell Biol; citation_volume=31; citation_number=2; citation_pages=153-85; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Alternatively spliced genes. In Encyclopedia of Molecular Cell Biology and Molecular Medicine Vol1. pp 125–177.; citation_author=JY Wu; citation_author=L Yuan; citation_author=N Havlioglu; citation_journal_title=Edited by Meyers R.A. (2nd edition) (Wiley-VCH); citation_number=3; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Caspases: an ancient cellular sword of Damocles.; citation_author=M Boyce; citation_author=A Degterev; citation_author=J Yuan; citation_journal_title=Cell Death Differ; citation_volume=11; citation_number=4; citation_pages=29-37; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Cytochrome C-mediated apoptosis.; citation_author=X Jiang; citation_author=X Wang; citation_journal_title=Annu Rev Biochem; citation_volume=73; citation_number=5; citation_pages=87-106; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Ich-1, an Ice/ced-3-related gene, encodes both positive and negative regulators of programmed cell death.; citation_author=L Wang; citation_author=M Miura; citation_author=L Bergeron; citation_author=H Zhu; citation_author=J Yuan; citation_journal_title=Cell; citation_volume=78; citation_number=6; citation_pages=739-50; citation_date=1994; " />
      <meta name="citation_reference" content="citation_title=Regulation of Ich-1 pre-mRNA alternative splicing and apoptosis by mammalian splicing factors.; citation_author=ZH Jiang; citation_author=WJ Zhang; citation_author=Y Rao; citation_author=JY Wu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=7; citation_pages=9155-60; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Defects in regulation of apoptosis in caspase-2-deficient mice.; citation_author=L Bergeron; citation_author=GI Perez; citation_author=G Macdonald; citation_author=LF Shi; citation_author=Y Sun; citation_journal_title=Genes Dev; citation_volume=12; citation_number=8; citation_pages=1304-14; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Caspase-2 pre-mRNA alternative splicing: Identification of an intronic element containing a decoy 3′ acceptor site.; citation_author=J Cote; citation_author=S Dupuis; citation_author=Z Jiang; citation_author=JY Wu; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=98; citation_number=9; citation_pages=938-43; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Polypyrimidine track-binding protein binding downstream of caspase-2 alternative exon 9 represses its inclusion.; citation_author=J Cote; citation_author=S Dupuis; citation_author=JY Wu; citation_journal_title=J Biol Chem; citation_volume=276; citation_number=10; citation_pages=8535-43; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Relationship between “proto-splice sites” and intron phases: evidence from dicodon analysis.; citation_author=M Long; citation_author=SJ de Souza; citation_author=C Rosenberg; citation_author=W Gilbert; citation_journal_title=Proc Natl Acad Sci U S A; citation_volume=95; citation_number=11; citation_pages=219-23; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=WebLogo: A sequence logo generator, Genome Research,; citation_author=GE Crooks; citation_author=G Hon; citation_author=JM Chandonia; citation_author=SE Brenner; citation_volume=14; citation_number=12; citation_pages=1188-1190; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Intron-exon structures of eukaryotic model organisms.; citation_author=M Deutsch; citation_author=M Long; citation_journal_title=Nucleic Acids Res; citation_volume=27; citation_number=13; citation_pages=3219-28; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=ASD: a bioinformatics resource on alternative splicing.; citation_author=S Stamm; citation_author=JJ Riethoven; citation_author=V Le Texier; citation_author=C Gopalakrishnan; citation_author=V Kumanduri; citation_journal_title=Nucleic Acids Research; citation_volume=34; citation_number=14; citation_pages=D46-D55; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Alternative pre-mRNA splicing: the logic of combinatorial control.; citation_author=CW Smith; citation_author=J Valcarcel; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=15; citation_pages=381-8; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Finding signals that regulate alternative splicing in the post-genomic era.; citation_author=AN Ladd; citation_author=TA Cooper; citation_journal_title=Genome Biol; citation_volume=3; citation_number=16; citation_pages=reviews0008; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=Mechanisms of alternative pre-messenger RNA splicing.; citation_author=DL Black; citation_journal_title=Annu Rev Biochem; citation_volume=72; citation_number=17; citation_pages=291-336; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Computational searches for splicing signals. Methods.; citation_author=XH Zhang; citation_author=CS Leslie; citation_author=LA Chasin; citation_volume=37(4); citation_number=18; citation_pages=292-305; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases.; citation_author=BJ Blencowe; citation_journal_title=Trends Biochem Sci; citation_volume=25; citation_number=19; citation_pages=106-10 (2000); citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Pre-mRNA splicing and human disease.; citation_author=NA Faustino; citation_author=TA Cooper; citation_journal_title=Genes Dev; citation_volume=17; citation_number=20; citation_pages=419-37; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Towards a splicing code.; citation_author=XD Fu; citation_journal_title=Cell; citation_volume=119; citation_number=21; citation_pages=736-8; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Predictive identification of exonic splicing enhancers in human genes.; citation_author=WG Fairbrother; citation_author=RF Yeh; citation_author=PA Sharp; citation_author=CB Burge; citation_journal_title=Science; citation_volume=297; citation_number=22; citation_pages=1007-13; citation_date=2002; " />
      <meta name="citation_reference" content="citation_title=ESEfinder: A web resource to identify exonic splicing enhancers.; citation_author=L Cartegni; citation_author=J Wang; citation_author=Z Zhu; citation_author=MQ Zhang; citation_author=AR Krainer; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=23; citation_pages=3568-71; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=Variation in alternative splicing across human tissues.; citation_author=G Yeo; citation_author=D Holste; citation_author=G Kreiman; citation_author=CB Burge; citation_journal_title=Genome Biol; citation_volume=5; citation_number=24; citation_pages=R74; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Systematic identification and analysis of exonic splicing silencers.; citation_author=Z Wang; citation_author=ME Rolish; citation_author=G Yeo; citation_author=V Tung; citation_author=M Mawson; citation_journal_title=Cell; citation_volume=119; citation_number=25; citation_pages=831-45; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=Computational definition of sequence motifs governing constitutive exon splicing.; citation_author=XH Zhang; citation_author=LA Chasin; citation_journal_title=Genes Dev; citation_volume=18; citation_number=26; citation_pages=1241-50; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A combinatorial code for splicing silencing: UAGG and GGGG motifs.; citation_author=K Han; citation_author=G Yeo; citation_author=P An; citation_author=CB Burge; citation_author=PJ Grabowski; citation_journal_title=PLoS Biol.; citation_volume=3(5); citation_number=27; citation_pages=e158; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=G triplets located throughout a class of small vertebrate introns enforce intron borders and regulate splice site selection.; citation_author=AJ McCullough; citation_author=SM Berget; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=28; citation_pages=4562-71; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=The splicing regulatory element, UGCAUG, is phylogenetically and spatially conserved in introns that flank tissue-specific alternative exons.; citation_author=S Minovitsky; citation_author=SL Gee; citation_author=S Schokrpur; citation_author=I Dubchak; citation_author=JG Conboy; citation_journal_title=Nucleic Acids Res; citation_volume=33; citation_number=29; citation_pages=714-24; citation_date=2005; " />
      <meta name="citation_reference" content="citation_title=Conserved sequence elements associated with exon skipping.; citation_author=E Miriami; citation_author=H Margalit; citation_author=R Sperling; citation_journal_title=Nucleic Acids Res; citation_volume=31; citation_number=30; citation_pages=1974-83; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=The mechanism of somatic inhibition of Drosophila P-element pre-mRNA splicing: multiprotein complexes at an exon pseudo-5′ splice site control U1 snRNP binding.; citation_author=CW Siebel; citation_author=LD Fresco; citation_author=DC Rio; citation_journal_title=Genes Dev; citation_volume=6; citation_number=31; citation_pages=1386-401; citation_date=1992; " />
      <meta name="citation_reference" content="citation_title=An intron enhancer containing a 5′ splice site sequence in the human calcitonin/calcitonin gene-related peptide gene.; citation_author=H Lou; citation_author=Y Yang; citation_author=GJ Cote; citation_author=SM Berget; citation_author=RF Gagel; citation_journal_title=Mol Cell Biol; citation_volume=15; citation_number=32; citation_pages=7135-42; citation_date=1995; " />
      <meta name="citation_reference" content="citation_title=The RNA-binding protein TIA-1 is a novel mammalian splicing regulator acting through intron sequences adjacent to a 5′ splice site.; citation_author=F Del Gatto-Konczak; citation_author=CF Bourgeois; citation_author=C Le Guiner; citation_author=L Kister; citation_author=MC Gesnel; citation_journal_title=Mol Cell Biol; citation_volume=20; citation_number=33; citation_pages=6287-99; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Cloning of Caenorhabditis U2AF65: an alternatively spliced RNA containing a novel exon.; citation_author=DA Zorio; citation_author=K Lea; citation_author=T Blumenthal; citation_journal_title=Mol Cell Biol; citation_volume=17; citation_number=34; citation_pages=946-53; citation_date=1997; " />
      <meta name="citation_reference" content="citation_title=Polypyrimidine tract binding protein antagonizes exon definition.; citation_author=EJ Wagner; citation_author=MA Garcia-Blanco; citation_journal_title=Mol Cell Biol; citation_volume=21; citation_number=35; citation_pages=3281-8; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Heterogeneous nuclear ribonucleoprotein particle A/B proteins and the control of alternative splicing of the mammalian heterogeneous nuclear ribonucleoprotein particle A1 pre-mRNA.; citation_author=B Chabot; citation_author=C LeBel; citation_author=S Hutchison; citation_author=FH Nasim; citation_author=MJ Simard; citation_journal_title=Prog Mol Subcell Biol; citation_volume=31; citation_number=36; citation_pages=59-88; citation_date=2003; " />
      <meta name="citation_reference" content="citation_title=RNA binding proteins and the regulation of neuronal synaptic plasticity.; citation_author=J Ule; citation_author=RB Darnell; citation_journal_title=Curr Opin Neurobiol.; citation_volume=16(1); citation_number=37; citation_pages=102-10; citation_date=2006; " />
      <meta name="citation_reference" content="citation_title=Molecular cloning and characterization of a novel caspase-3 variant that attenuates apoptosis induced by proteasome inhibition.; citation_author=Y Huang; citation_author=NH Shin; citation_author=Y Sun; citation_author=KK Wang; citation_journal_title=Biochem Biophys Res Commun; citation_volume=283; citation_number=38; citation_pages=762-9; citation_date=2001; " />
      <meta name="citation_reference" content="citation_title=Dominant expression of a novel splice variant of caspase-8 in human peripheral blood lymphocytes.; citation_author=T Horiuchi; citation_author=D Himeji; citation_author=H Tsukamoto; citation_author=S-i Harashima; citation_author=C Hashimura; citation_journal_title=Biochem Biophys Res Commun; citation_volume=272; citation_number=39; citation_pages=877-81; citation_date=2000; " />
      <meta name="citation_reference" content="citation_title=Identification of an endogenous dominant-negative short isoform of caspase-9 that can regulate apoptosis.; citation_author=SM Srinivasula; citation_author=M Ahmad; citation_author=Y Guo; citation_author=Y Zhan; citation_author=Y Lazebnik; citation_journal_title=Cancer Res; citation_volume=59; citation_number=40; citation_pages=999-1002; citation_date=1999; " />
      <meta name="citation_reference" content="citation_title=GOstat: Find statistically overrepresented Gene Ontologies within a group of genes.; citation_author=T Beissbarth; citation_author=TP Speed; citation_journal_title=Bioinformatics; citation_volume=20(9); citation_number=41; citation_pages=1464-1465; citation_date=2004; " />
      <meta name="citation_reference" content="citation_title=A computer program for aligning a cDNA sequence with a genomic DNA sequence.; citation_author=L Florea; citation_author=G Hartzell; citation_author=Z Zhang; citation_author=GM Rubin; citation_author=W Miller; citation_journal_title=Genome Res; citation_volume=8; citation_number=42; citation_pages=967-74; citation_date=1998; " />
      <meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome.; citation_journal_title=Nature; citation_volume=409; citation_number=43; citation_pages=860-927; citation_date=2001; " />

  <link rel="canonical" href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" />

    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="An Intronic Signal for Alternative Splicing in the Human Genome"/>
    <meta name="twitter:description" content="An important level at which the expression of programmed cell death (PCD) genes is regulated is alternative splicing. Our previous work identified an intronic splicing regulatory element in caspase-2 (casp-2) gene. This 100-nucleotide intronic element, In100, consists of an upstream region containing a decoy 3&prime; splice site and a downstream region containing binding sites for splicing repressor PTB. Based on the signal of In100 element in casp-2, we have detected the In100-like sequences as a family of sequence elements associated with alternative splicing in the human genome by using computational and experimental approaches. A survey of human genome reveals the presence of more than four thousand In100-like elements in 2757 genes. These In100-like elements tend to locate more frequent in intronic regions than exonic regions. EST analyses indicate that the presence of In100-like elements correlates with the skipping of their immediate upstream exons, with 526 genes showing exon skipping in such a manner. In addition, In100-like elements are found in several human caspase genes near exons encoding the caspase active domain. RT-PCR experiments show that these caspase genes indeed undergo alternative splicing in a pattern predicted to affect their functional activity. Together, these results suggest that the In100-like elements represent a family of intronic signals for alternative splicing in the human genome."/>
      <meta name="twitter:image" content="http://dx.plos.org/10.1371/journal.pone.0001246.g006"/>

  <meta property="og:title" content="An Intronic Signal for Alternative Splicing in the Human Genome" />
  <meta property="og:type" content="article" />
  <meta property="og:url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" />

 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback" />


  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon"/>
  <link rel="home" title="home" href="/"/>
  <link rel="alternate" type="application/rss+xml"
        title="PLOS ONE: New Articles"
        href="http://www.plosone.org/article/feed"/>
</head>
<body>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id='a3ac9da4' name='a3ac9da4'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=345&amp;cb=3464'
    frameborder='0' scrolling='no' width='730' height='90'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=6143'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=5795&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a
              href="/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001246">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="/"><img src="/images/logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="/static/publication">Publication Criteria</a></li>
            <li><a href="/static/editorial">Editorial Policies</a></li>
            <li><a href="/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width:248px; margin-left:-30px;">
        <div class="menu">
          <ul>
            <li><a href="/static/information">Journal Information</a></li>
            <li><a href="/static/edboard">Editorial Board</a></li>
            <li><a href="/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="/static/license">Open-Access License</a></li>
            <li><a href="/static/downloads">Media Downloads</a></li>
            <li><a href="/static/commentGuidelines">Guidelines for Comments</a></li>
            <li><a href="/static/corrections">Corrections</a></li>
            <li><a href="/static/help">Help Using this Site</a></li>
            <li><a href="/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001246" method="get" >
<input type="hidden" name="from" value="globalSimpleSearch" id="from"/><input type="hidden" name="filterJournals" value="PLoSONE" id="filterJournals"/>    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" type="text" name="query" placeholder="Search">
        <input type="image" alt="SEARCH" src="/images/icon.search.gif">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001246&filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="/javascript/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
      <span id="article-type-heading">
        Research Article
      </span>
</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">
    An Intronic Signal for Alternative Splicing in the Human Genome
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Necat Havlioglu
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Necat Havlioglu, Jun Wang</p>

              
              

                <p>Affiliation:
                  Department of Pathology, Saint Louis University, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jun Wang
              <span class="equal-contrib"
                    title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: Necat Havlioglu, Jun Wang</p>

              
              

                <p>Affiliation:
                  Department of Ecology and Evolution, The University of Chicago, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Kazuo Fushimi, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Neurology, Lurie Comprehensive Cancer Center, Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Maria D. Vibranovski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Ecology and Evolution, The University of Chicago, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Zhengyan Kan, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Washington University in St. Louis, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Warren Gish, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Genetics, Washington University in St. Louis, St. Louis, Missouri, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Alexei Fedorov, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Medicine and Program in Bioinformatics and Proteomics/Genomics, Medical University of Ohio, Toledo, Ohio, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Manyuan Long
              <span class="corresponding">mail</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mlong@midway.uchicago.edu">mlong@midway.uchicago.edu</a> (ML); <a href="mailto:jane-wu@northwestern.edu">jane-wu@northwestern.edu</a> (JW)</p>

                <p>Affiliation:
                  Department of Ecology and Evolution, The University of Chicago, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">
            Jane Y. Wu
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">*</span>To whom correspondence should be addressed. E-mail: <a href="mailto:mlong@midway.uchicago.edu">mlong@midway.uchicago.edu</a> (ML); <a href="mailto:jane-wu@northwestern.edu">jane-wu@northwestern.edu</a> (JW)</p>

                <p>Affiliation:
                  Department of Neurology, Lurie Comprehensive Cancer Center, Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>
  <ul class="date-doi-line">
    <li>Published: November 28, 2007</li>
    <li>DOI: 10.1371/journal.pone.0001246</li>
  </ul>


</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" name="comments">Comments</a>
  </li>
  <li>
      <a href="/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">
  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item">
                <a href="#pone-0001246-g001" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-g002" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-g003" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-t001" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t001&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-t002" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t002&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-g004" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g004" title="Figure 4">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g004&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-g005" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g005" title="Figure 5">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g005&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-g006" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g006" title="Figure 6">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g006&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0001246-t003" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t003&representation=PNG_I" alt="">
                  </span>
                </a>
              </div>
      </div>
    </div>
  </div>

  <div class="nav-col">
    <div class="nav" id="nav-article-page">
      <ul>
        <li class="nav-col-comments"><a href="/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0001246">Reader Comments (2)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0001246" >Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>An important level at which the expression of programmed cell death (PCD) genes is regulated is alternative splicing. Our previous work identified an intronic splicing regulatory element in caspase-2 (casp-2) gene. This 100-nucleotide intronic element, In100, consists of an upstream region containing a decoy 3′ splice site and a downstream region containing binding sites for splicing repressor PTB. Based on the signal of In100 element in casp-2, we have detected the In100-like sequences as a family of sequence elements associated with alternative splicing in the human genome by using computational and experimental approaches. A survey of human genome reveals the presence of more than four thousand In100-like elements in 2757 genes. These In100-like elements tend to locate more frequent in intronic regions than exonic regions. EST analyses indicate that the presence of In100-like elements correlates with the skipping of their immediate upstream exons, with 526 genes showing exon skipping in such a manner. In addition, In100-like elements are found in several human caspase genes near exons encoding the caspase active domain. RT-PCR experiments show that these caspase genes indeed undergo alternative splicing in a pattern predicted to affect their functional activity. Together, these results suggest that the In100-like elements represent a family of intronic signals for alternative splicing in the human genome.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Havlioglu N, Wang J, Fushimi K, Vibranovski MD, Kan Z, et al.  (2007) An Intronic Signal for Alternative Splicing in the Human Genome. PLoS ONE 2(11):
          e1246.
            doi:10.1371/journal.pone.0001246</p><p><strong>Academic Editor: </strong>Stefan Maas, Lehigh University, United States of America</p><p><strong>Received:</strong> May 30, 2007; <strong>Accepted:</strong> October 23, 2007; <strong>Published:</strong> November 28, 2007</p><p><strong>Copyright:</strong> © 2007 Havlioglu et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants to JYW from NIH (EY014576; AG17518, GM070967) and Muscular Dystrophy Association. JYW was also supported by a Scholar Award from the Leukemia Society of America. ML was supported by NIH (GM065429), NSF CAREER Award (MCB0238168) and Packard Fellowship for Science and Engineering. MDV was supported by Pew Latin America Fellowship.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>As a major mechanism of genetic diversity by generating transcript variability, alternative splicing affects every aspect of cell survival and function. It is estimated that more than 60% of human genes undergo alternative splicing <a href="#pone.0001246-Johnson1">[1]</a>. A variety of splicing regulatory elements have been identified by using molecular, biochemical, and computational approaches. However, our understanding of alternative splicing at the genomic level remains limited.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>A large number of genes critical for cell death utilize alternative splicing to generate functionally distinct or even antagonistic isoforms <a href="#pone.0001246-Wu1">[2]</a>, <a href="#pone.0001246-Wu2">[3]</a>. These genes range from death signals, death receptors, intracellular adaptor molecules to caspases <a href="#pone.0001246-Boyce1">[4]</a>, <a href="#pone.0001246-Jiang1">[5]</a>. To understand mechanisms regulating alternative splicing events important for cell death gene function, we previously established a model system using the murine caspase-2 (casp-2) gene. Exclusion or inclusion of exon 9 leads to the formation of two functionally antagonistic isoforms, casp-2S or casp-2L respectively <a href="#pone.0001246-Wang1">[6]</a>, <a href="#pone.0001246-Jiang2">[7]</a>. Casp-2L and casp-2S have antagonistic activities in cell death <a href="#pone.0001246-Wang1">[6]</a>, <a href="#pone.0001246-Bergeron1">[8]</a>. Dissection of cis-elements reveals an intronic element, In100, which is important for regulating casp-2 alternative splicing. In100 has two domains, an upstream one containing a decoy 3′ splice site and a downstream one with U/C-rich repeats that interact with PTB <a href="#pone.0001246-Cote1">[9]</a>, <a href="#pone.0001246-Cote2">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>In this study, we searched human genome for sequence elements with features of In100 and identified 2757 genes containing In100-like elements. These In100-containing genes are subdivided in 1865 gene families. Gene ontology analysis revealed that the products of these In100-containing genes are involved in various biology processes. These elements are significantly enriched in intronic regions. EST database analyses revealed a correlation of such elements with the skipping of their upstream exons. In addition, by RT-PCR, analyses of several human caspase genes indicate that these In100-containing caspase genes indeed undergo alternative splicing, with the skipping of exons immediate upstream of their In100-like elements. Taken together, these results support that In100-like elements represent a family of sequence signals for alternative splicing.</p>
</div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>A conserved intronic regulatory element, In100, in the caspase-2 gene</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>In our previous studies, we established a casp-2 splicing model system and identified In100 as an important cis-element for casp-2 alternative splicing <a href="#pone.0001246-Cote1">[9]</a>, <a href="#pone.0001246-Cote2">[10]</a>. This In100 element contains two domains, both of which are required for its activity in suppressing the use of the 5′ splice site of exon 9. The upstream domain contains a decoy 3′ splice site, highly similar to an authentic 3′ splice site, and a downstream domain with U/C-rich repeats interacting with polypyrimidine tract binding protein, PTB <a href="#pone.0001246-Cote1">[9]</a>, <a href="#pone.0001246-Cote2">[10]</a>. Our biochemical experiments indicate that In100 engages in non-productive interactions with the 5′SS of exon 9, hence blocking its usage by the spliceosome (<a href="#pone-0001246-g001">Figure 1</a>).</p>
<div class="figure" id="pone-0001246-g001"><div class="img"><a name="pone-0001246-g001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g001.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>A diagram depicting In100 and its role in regulating casp-2.</span></strong></p><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1"></a><p>As demonstrated by our previous studies <a href="#pone.0001246-Cote1">[9]</a>, <a href="#pone.0001246-Cote2">[10]</a>, sequential deletion and biochemical experiments reveal that In100 in casp-2 gene represses exon 9 inclusion by engaging non-productive interaction with the 5′ splice site of exon 9. Interactions between U1 snRNP associated with exon 9 at the 5′ splice site and complexes associated with In100, including U2 snRNP and PTB, block rather than promote the selection of this 5′ splice site. The In100 element is diagramed as the hatched box. Exons 8, 9 and 10 are depicted as E8, E9 and E10 in gray boxes. U1 snRNP and U2 snRNPs are illustrated as U1 and U2.</p>
<span>doi:10.1371/journal.pone.0001246.g001</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Comparative analyses of mouse and human casp-2 genomic sequences reveal that In100 element is conserved, with 74% nucleotide identity between mouse and human sequences (<a href="#pone-0001246-g002">Figure 2A</a>), although the overall nucleotide sequence of intron 9 is not conserved. This suggests a common mammalian ancestral In100-like element and the functional importance of In100-like elements in mammalian species. These observations raise a conjecture that the In100-like elements may represent a sequence motif regulating alternative splicing of many different genes. Testing this conjecture may help address several questions. First, are these In100-like elements wide spread in the human genome? Second, how are they distributed in the genome, e.g. among intronic and exonic regions? Third, is the presence of such elements associated with alternative splicing events in the human genome? To answer these questions, we designed the following bioinformatics and experimental approaches to search for and analyze In100-like elements in the human genome.</p>
<div class="figure" id="pone-0001246-g002"><div class="img"><a name="pone-0001246-g002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g002.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>The evolutionary conservation of casp-2 In100 element and nucleotide composition of In100-like elements.</span></strong></p><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1"></a><p>(A) Alignment between human and mouse casp-2 In100 elements shows 74% sequence identity. Putative branch site A is in bold and underlined (BS). The polypyrimidine tract in the decoy 3′splice site (Py)n and CAG are indicated. (B) The nucleotide composition is displayed based on the identified 4342 In100 elements.</p>
<span>doi:10.1371/journal.pone.0001246.g002</span></div>

<h4>In100-like sequence elements are widespread in the human genome</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>To test the hypothesis that a 3′ splice site-like sequence juxtaposed to U/C rich sequence may represent a general motif associated with alternative splicing, we screened the human genome for genes containing such sequences. First, an exon-intron sequence database was created containing all annotated human genes. An algorithm was developed to search for In100-like elements in this database. This algorithm used a 100-nucleotide sliding window to scan for the In100 signals from the 5′ to the 3′ end along each DNA sequence in the database. In100 signal is defined as: UAAC[branch site]-5~10nt-(Py)n[polypyrimidine tract]-N<sub>3</sub>-(U/C rich, 20–60 nt), where n is bounded around 30~40, “U/C rich” is defined as U/C content higher than 70%, and N<sub>3</sub> = CAG. We also tested N<sub>3</sub> = TAG, AAG and GAG (see below) and investigated the relative importance of these trinucleotides in defining 3′SS <a href="#pone.0001246-Long1">[11]</a>. This search revealed 4342 candidate sequences with the features of In100 in 2757 genes (<a href="#pone.0001246.s001">Table S1</a>), suggesting that In100-like elements may represent a general sequence motif. The nucleotide composition of these In100-like elements was analyzed using PERL scripts and Web-Logo <a href="#pone.0001246-Crooks1">[12]</a>, as displayed in <a href="#pone-0001246-g002">Figure 2B</a>.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2"></a><p>Gene family analysis showed that these In100-containing genes can be classified into 1865 gene families (<a href="#pone.0001246.s002">Table S2</a>). Gene ontology analysis showed that these genes are involved in various biological processes, molecular function and encoding different cell components (<a href="#pone-0001246-g003">Figure 3</a>). Among overrepresented In100-containing genes are those involved in biological processes of biopolymer modification, protein modification process, post-translation protein modification, with molecular functions of protein kinase activity, ATP binding and phosphotransferase activity and encoding cell components of basement membrane, cell membrane and synapses (<a href="#pone.0001246.s003">Table S3</a>).</p>
<div class="figure" id="pone-0001246-g003"><div class="img"><a name="pone-0001246-g003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g003.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>The distribution of gene ontology of In100-containing genes.</span></strong></p><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1"></a><p>The distribution of gene ontology terms of the In100-containing genes in (A) Biology Process, (B) Molecular Function and (C) Cell Component.</p>
<span>doi:10.1371/journal.pone.0001246.g003</span></div>

<h4>The significant enrichment of In100-like elements in intronic regions</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>To address whether these In100-like sequences are randomly distributed in intronic and exonic regions, we designed two tests. First, we compared the observed and expected frequencies of the candidate In100 elements in intronic and exonic sequences based on their total length distribution. The lengths of introns and exons for In100-containing genes were calculated, and frequencies of candidate In100 sequences were compared. If these In100-like elements were equally distributed, we would expect their frequencies in the intronic versus exonic regions to be 94.2% and 5.8% respectively, proportional to the relative lengths of introns and exons in the human exon-intron sequence database constructed (see <a href="#s4">Methods</a>). As shown in <a href="#pone-0001246-t001">Table 1</a>, there is a positive bias towards intronic distribution (98.5%), and a negative bias towards exonic distribution (1.5%) (χ<sup>2</sup> = 189.2, degree of freedom (df) = 1, P≈0). Thus, these In100-like elements are not randomly distributed and instead, significantly enriched in intronic regions.</p>
<div class="figure" id="pone-0001246-t001"><div class="img"><a name="pone-0001246-t001" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.t001"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t001&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t001/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t001/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t001/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t001/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.t001.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t001/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t001/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.t001.TIF"></span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Intronic versus exonic distribution of In100-like elements identified in human genome</span></strong></p><span>doi:10.1371/journal.pone.0001246.t001</span></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2"></a><p>Additionally, we examined the distribution pattern of In100-elements in randomized human intron-exon databases. By simulation, we randomized the nucleotide sequences of human intron-exon database but maintaining the same exon-intron structure and nucleotide compositions (see <a href="#s4">Methods</a>). We found that the frequency of In100-elements decreases by almost an order of magnitude in the randomized databases (<a href="#pone.0001246.s009">Figure S3</a>), as compared to the In100-element frequency observed in the real human intron-exon database. Thus, the significant decrease of In100-elements in the randomized human database indicates that there is a significant enrichment of In100-element in the human genome and that the detected genomic distribution of In100 in the human genome is not a statistical coincidence.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3"></a><p>Next, we analyzed the trinucleotide sequences inside the decoy 3′ splice site of the In100 elements (i.e., in “N<sub>3</sub>” position). Genomic analyses have shown that in intronic regions, CAG are present more frequently than expected at the last three nucleotide positions as 3′ splicing signal <a href="#pone.0001246-Long1">[11]</a>, <a href="#pone.0001246-Deutsch1">[13]</a>. Thus, if the detected In100-like sequences are intronic elements; in intronic regions, the frequency of the defined elements with CAG-(U/C rich, 20-60nt) should be higher than its expected frequency, whereas the elements with TAG, GAG, or AAG should be observed in depletion, or at least not in excess. In addition, the excess of In100-elements with CAG should be exclusively observed in the intronic regions, not in exonic regions. We analyzed and compared the number of In100-like elements with CAG, AAG, GAG, and TAG as N<sub>3</sub> in intronic and exonic regions respectively. Respectively, we identified 4277, 4643, 3814 and 5126 In100-like sequences in the intronic regions, and 65, 95, 57 and 80 in the exonic regions. As shown in <a href="#pone-0001246-t002">Table 2</a>, the frequency of CAG containing In100 elements are significantly higher in the intronic regions and lower than expected in the exonic regions, further supporting that the In100 elements are enriched in intronic regions.</p>
<div class="figure" id="pone-0001246-t002"><div class="img"><a name="pone-0001246-t002" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.t002"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t002&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t002/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t002/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t002/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t002/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.t002.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t002/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t002/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.t002.TIF"></span>)
                </a></li></ul></div><p><strong>Table 2.  <span>Frequency of defined trinucleotides of decoy 3′ splice sites in candidate In100 elements (numbers observed and expected for introns/exons)</span></strong></p><span>doi:10.1371/journal.pone.0001246.t002</span></div>

<h4>The presence of In100 element correlates with skipping of its upstream exons</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>If In100 elements play a role in suppressing the inclusion of the immediate upstream exons, similar to that in caspase-2 gene (<a href="#pone-0001246-g001">Figure 1</a>; see the previous studies <a href="#pone.0001246-Cote1">[9]</a>, <a href="#pone.0001246-Cote2">[10]</a>), it would be predicted that In100 immediate downstream exons would be more likely to appear in the human transcriptome than the immediate upstream exons. This prediction was tested by examining the relative abundance of the transcripts containing the upstream or downstream exons of In100-element in the human EST database. We isolated <em>in silico</em> the immediate upstream and downstream exons for 4277 In100-elements in the intronic regions and searched human EST databases for the transcripts that contain exons flanking In100-elements. We used equal or higher than 98% identity and 70% exon length coverage as alignment criteria to determine the match of an EST with a particular exon sequence. If an exon-EST alignment satisfied these criteria, we assumed the EST was transcribed from the exon. Among the 4277 sequences, we found 2728 In100-elements (in 2012 genes) that have both upstream and downstream exons annotated with ESTs. The frequency of ESTs representing the downstream exons of the In100-element is 12.5% higher than that of the upstream exons (<a href="#pone-0001246-g004">Figure 4</a>).</p>
<div class="figure" id="pone-0001246-g004"><div class="img"><a name="pone-0001246-g004" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g004"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g004&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g004/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g004/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g004/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g004/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g004.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g004/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g004/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g004.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 4.  <span>A biased distribution of transcripts favoring the downstream exons as revealed by EST analyses.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>ESTs frequency of upstream (5′) and downstream (3′) exons. The sample sizes for analyzed In100-element flanking exons and control exon pairs are: 2728 and 23624, respectively.</p>
<span>doi:10.1371/journal.pone.0001246.g004</span></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2"></a><p>In order to examine whether this increase in the frequency of ESTs representing the exons immediately downstream of In100-elements is associated with the presence of In100-elements, we constructed and analyzed the negative control exon pairs of In100-elements. We selected exon pairs that do not immediately flank those elements from the same 2757 In100-element containing genes. In this set of 23624 pairs of control exons, the frequency of ESTs representing the downstream exons is only 1.7% higher than that of the upstream exons. The difference in the comparison of In100-element flanking exons and the control exon pairs is highly significant (Fisher's Exact Test, P&lt;2.2×10<sup>−16</sup>, df = 1; <a href="#pone-0001246-g004">Figure 4</a>).</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3"></a><p>This control group of exon pairs also works as a control for over-representation of the downstream exons as a result of the biased cDNA cloning or EST distribution. Namely, the observed gradient of EST representation could merely be a reflection of an uneven EST distribution in the EST database, rather than the skipping of the In100- upstream exons. If this were true, we should expect to find a higher frequency of ESTs matched to the downstream than the upstream exon in the control group as well, since our control exon pairs show the same distribution of position along the mRNA sequences as the In100-element flanking exons (<a href="#pone.0001246.s007">Figure S1</a>). The highly significant difference (Fisher's Exact Test, P&lt;2.2×10<sup>−16</sup>) between the differential EST frequency (downstream versus upstream exons) of the In100-containing exon pairs (12.5%) and the control group (1.7%) indicates that the observed gradient is not a result of biased EST distribution along the mRNA transcripts. This excess of downstream exons of In100-containing sequences in the human transcriptome supports the association between upstream exon skipping and the presence of In100-like elements.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4"></a><p>In order to verify and quantify this association, we examined three-exon cassette sequences and compared middle exon skipping (<a href="#pone-0001246-g005">Figure 5A</a>) in In100-element containing genes versus In100-lacking control group cassettes. The rationale is that for In100-containing exon triplets (e.g. exon1-exon2-In100-exon3; <a href="#pone-0001246-g005">Figure 5A</a>), the presence of ESTs representing both complete exon1-exon2-exon3 transcript and exon2-lacking transcript (i.e., exon1-exon3) indicates the alternative splicing event with exon2 skipping. However, due to the possible incompleteness of ESTs database, such exon-skipping evidence may not be found for every exon triplet. Therefore, from In100-containing genes, we only included the ones with sufficient EST information, namely the ones showing at least one EST simultaneously matching exon1 and exon3 (with/without exon2). Among them, 1470 triplets (from 1252 genes) are In100-containing exon triplets, and 15943 triplets (from 2036 genes) are In100-lacking exon triplets. These two groups of exon triplets have similar distribution of position along mRNA sequence (<a href="#pone.0001246.s008">Figure S2</a>). We treated the former group as the experimental set and the latter group as the control set. In the experimental set of 1470 In100-containing exon triplets, 524 triplets (36% of 1470) have at least one EST presenting exon 2 skipping; whereas in the control set of 15943 In100-lacking exon triplets, only 4116 triplets (26%) have such ESTs (<a href="#pone-0001246-g005">Figure 5B</a> and <a href="#pone.0001246.s006">Table S6</a>). Fisher's Exact Test showed that the frequency of the exon-skipping events is significantly higher in In100-containing exon triplets than In100-lacking ones (P&lt;2.3×10<sup>−15</sup>). These 524 In100-containing exon triplets are from 482 genes (<a href="#pone.0001246.s004">Table S4</a>). Thus, 38% (482 out of 1252) of these In100-element containing genes show EST evidence for the skipping of exons immediate upstream of In100-residing introns.</p>
<div class="figure" id="pone-0001246-g005"><div class="img"><a name="pone-0001246-g005" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g005"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g005&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g005/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g005/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g005/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g005/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g005.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g005/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g005/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g005.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 5.  <span>The alternative splicing of In100-containing genes. (A) A diagram depicting the alternative splicing of In100-containing genes.</span></strong></p><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1"></a><p>In100-containing genes were identified with three flanking exons (E1 to E3) from EST database, illustrated as E1-E2-In100-E3. 482 genes from EST database and an additional 44 genes from AltSplice <a href="#pone.0001246-Stamm1">[14]</a> Database, with a total of 526 different genes show exon 2 skipping. (B) A higher frequency of skipping of exon2 in In100-containing exon triplets revealed by EST analyses. Frequency of skipping of exon2 in In100-containing and In100-lacking exon triplets by EST evidence is: 36% and 26%, respectively. The sample sizes for analyzed In100-containing and control exon triplets are: 1470 and 15943, respectively.</p>
<span>doi:10.1371/journal.pone.0001246.g005</span></div><a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5"></a><p>Finally, we searched AltSplice <a href="#pone.0001246-Stamm1">[14]</a> Database, a data source recording alternative-splicing events. We identified 69 In100-containing genes with clear evidence of skipping of exons upstream of the In100 element (<a href="#pone.0001246.s005">Table S5</a>). Twenty-five of these 69 genes were the same as the ones identified by our ESTs analysis. Thus, we have identified 526 In100-containing genes that have evidence of skipping of exons immediately upstream of In100-elements. These results provide strong evidence for the correlation of In100-elements with the skipping of their immediate upstream exons.</p>


<h4>The presence of In100-like elements in caspase genes and their alternative splicing</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>The analyses of human genomes described above have revealed a large number of In100-like elements and provide an opportunity to further investigate predicted alternative splicing patterns of the output candidate genes. Because In100 was first identified in casp-2 gene, we asked whether such elements are also found in other caspase genes. Indeed, the In100-like element was found in several human caspase gene sequences, including caspase 2, 3, 8 and 9 (<a href="#pone-0001246-t003">Table 3</a>). We systematically analyzed human caspase gene structures and examined the location of these In100-like elements. Interestingly, the In100-like elements in these caspase genes are located in introns adjacent to exons encoding caspase active sites with “QACXG” signature motif (<a href="#pone-0001246-g006">Figure 6A</a>). If these In100-like elements act to suppress the inclusion of the upstream exon, as predicted by our casp-2 study, such alternative splicing events will have significant functional impact because of the formation of isoforms that either contain or lack the caspase catalytically active domain.</p>
<div class="figure" id="pone-0001246-g006"><div class="img"><a name="pone-0001246-g006" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.g006"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.g006&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g006/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g006/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g006/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g006/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g006.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.g006/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.g006/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.g006.TIF"></span>)
                </a></li></ul></div><p><strong>Figure 6.  <span>Alternative splicing of caspase genes.</span></strong></p><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1"></a><p>(A) A schematic illustration of In100-like elements in caspase genes and their alternative splicing patterns. Exons are depicted in rectangles, and In100 elements are illustrated as small gray boxes. The positions of regions encoding “QACRG” pentapeptide sequence are marked with “*”. (B) Alternative splicing of Caspase 2, 3, 8, 9 genes as detected by RT-PCR. Specific primers corresponding to the upstream and downstream exons were used in RT-PCR as described in <a href="#s4">Materials and Methods</a>. The sizes of each exons are not drawn to scale. The positions of PCR primers are marked by arrows.</p>
<span>doi:10.1371/journal.pone.0001246.g006</span></div><div class="figure" id="pone-0001246-t003"><div class="img"><a name="pone-0001246-t003" title="Click for larger image " href="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0001246" data-uri="info:doi/10.1371/journal.pone.0001246.t003"><img src="/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0001246.t003&amp;representation=PNG_I" alt="thumbnail" class="thumbnail"></a></div><div class="figure-inline-download">
            Download:
            <ul><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t003/powerpoint">
                    PPT
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t003/powerpoint">
                  PowerPoint slide
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t003/largerimage">
                    PNG
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t003/largerimage">
                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0001246.t003.PNG_L"></span>)
                </a></li><li><div class="icon"><a href="/article/info:doi/10.1371/journal.pone.0001246.t003/originalimage">
                    TIFF
                  </a></div><a href="/article/info:doi/10.1371/journal.pone.0001246.t003/originalimage">
                  original image
                  (<span id="info:doi/10.1371/journal.pone.0001246.t003.TIF"></span>)
                </a></li></ul></div><p><strong>Table 3.  <span>The Ensembl gene ID and transcript ID used to assign exon numbers</span></strong></p><span>doi:10.1371/journal.pone.0001246.t003</span></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2"></a><p>We used two approaches to confirm these predicted alternative-splicing events in the corresponding caspase genes. First, we searched human EST and cDNA databases and aligned corresponding ESTs with the genomic sequences of these caspase genes. In each of the caspase genes examined, there are EST or cDNA sequences indicative of alternative splicing. The presence of these EST or cDNA sequences indicates that these caspase genes are alternatively spliced in this region. More importantly, caspase-2, 3, 8 and 9, all undergo alternative splicing in the exon encoding QACXG motif, the caspase active site.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3"></a><p>We then used a RT-PCR assay to examine the alternative splicing of selected caspase genes in human cells. Using specific primers corresponding to flanking exonic sequences to perform RT-PCR assay, we confirmed that casp- 2, 3, 8 and 9 all undergo alternative splicing as predicted (<a href="#pone-0001246-g006">Figure 6B</a>). These experimental data suggest that the presence of In100-like elements has a predictive value for alternative splicing and strongly support the hypothesis that In100 elements represent intronic signals for alternative splicing.</p>

</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>In this study, we employed bioinformatics and experimental approaches to search for sequence signals associated with alternative splicing. Our data reveal that In100, originally identified in the mouse casp-2 gene, is present in thousands of unrelated human genes and possibly involved in regulating their alternative splicing. Of 1252 In100-containing genes for which sufficient EST information is available, 482 genes display skipping of exons immediate upstream of the In100-residing introns. An additional 44 In100-containing genes with such alternative splicing pattern are found in AltSplice Database <a href="#pone.0001246-Stamm1">[14]</a>. The experimental analyses of elements found in human caspase genes, including casp-2, 3, 8 and 9, confirmed that these genes indeed undergo alternative splicing in the regions containing In100-elements. Our analyses support that In100 elements represent a family of intronic sequence motif involved in alternative splicing of many different genes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>Alternative splicing is a complex process involving interactions between multiple trans-acting factors with a large number of cis-acting regulatory elements. Identifying the In100-like elements in a large number of genes does not necessarily predict identical splicing patterns of these genes in different cell types, because In100-elements may interact with different cis- regulatory elements in different genes and trans-acting factors in different types of cells. How these unknown factors interact with In100-elements and define the alternative splicing patterns are interesting problems to be addressed by further experiments. The large number of In100 containing genes, each with its specifics in interaction with In100 elements, provides tremendous opportunities for the research community to further investigate detailed mechanisms of alternative splicing regulation in these genes (<a href="#pone.0001246.s001">Table S1</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Intronic and exonic regulatory elements can enhance or suppress splicing, including exonic splicing enhancers (ESEs), intronic splicing enhancers (ISEs) or exonic splicing silencers (ESSs) and intronic splicing silencers (ISSs). Most of these cis-acting regulatory elements have been identified by biochemical and molecular studies of alternative splicing of individual genes of interest, (for recent reviews, see <a href="#pone.0001246-Smith1">[15]</a>–<a href="#pone.0001246-Faustino1">[20]</a>). Programs predicting exonic splicing elements based on systematic evolution of ligands by exponential enrichment (SELEX) approach and computational analyses have been developed (for example, <a href="http://exon.cshl.org/ESE">http://exon.cshl.org/ESE</a>, also see <a href="#pone.0001246-Fu1">[21]</a>–<a href="#pone.0001246-Zhang2">[26]</a>). Recently, a ternary combination of exonic UAGG and 5′-splice-site-proximal GGGG motifs have been reported that function cooperatively to silence the brain-region-specific CI exon 19 of the glutamate NMDA R1 receptor transcript <a href="#pone.0001246-Han1">[27]</a>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>A variety of intronic splicing regulatory elements have been identified using biochemical approaches. Only a few studies have reported systematic approaches to search for intronic regulatory elements. Analyses of small vertebrate introns suggest that GGG-containing sequences may act as intronic splicing enhancers in small introns <a href="#pone.0001246-McCullough1">[28]</a>. A study of brain-enriched alternative exons shows that putative intronic splicing enhancers containing UGCAUG motif are often found in introns flanking alternatively spliced exons <a href="#pone.0001246-Minovitsky1">[29]</a>. Analyses of 54 alternatively spliced exons revealed C-rich and G-rich sequences in the flanking introns <a href="#pone.0001246-Miriami1">[30]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In100 element has a complex structure. Most of the previously reported splicing regulatory elements, especially those identified by computational approaches, are in the range of 6-15mers <a href="#pone.0001246-Fairbrother1">[22]</a>–<a href="#pone.0001246-Miriami1">[30]</a>, whereas In100-like elements described here are about 100 nucleotides in length, containing the decoy 3′ splice site domain and the pyrimidine-rich domain.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6"></a><p>A role for pseudo-splice sites in regulating splicing has been suggested by previous studies. In <em>Drosophila</em> P-element intron 3 splicing, a pseudo 5′ splice site competes with the normal 5′ SS for interaction with U1 snRNP <a href="#pone.0001246-Siebel1">[31]</a>. A sequence similar to a 5′ splice site <a href="#pone.0001246-Lou1">[32]</a> and a U-rich element adjacent to the regulated 5′ splice site <a href="#pone.0001246-DelGattoKonczak1">[33]</a> have been reported to act as intronic splicing repressors. In <em>C. elegans</em>, an alternative exon in U2AF65 pre-mRNA contains multiple copies of 3′ splice site consensus sequences <a href="#pone.0001246-Zorio1">[34]</a>. These single case studies appeal for further studies at the genomic level. Our study of In100-elements suggests that decoy 3′ splice sites juxtaposed to pyrimidine-rich sequences may act as general signals for alternative splicing.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7"></a><p>Mechanisms by which the decoy 3′ splice site of In100 is recognized as a signal for alternative splicing rather than an authentic 3′ splice site remain to be elucidated. PTB binding to the downstream domain of In100 plays an important role <a href="#pone.0001246-Cote2">[10]</a>. Our biochemical studies suggest the involvement of additional factors <a href="#pone.0001246-Cote1">[9]</a>, (KF and JYW unpublished results). We noticed that some of In100-like elements contain polyU or polyC stretches that do not resemble typical PTB-binding sites. It is conceivable that such sequences may interact with other splicing regulatory factors.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8"></a><p>In100 elements are distinct from other intronic splicing regulatory elements, including splicing suppressor elements containing hnRNP A1 or PTB binding sites, reviewed in <a href="#pone.0001246-Ladd1">[16]</a>, <a href="#pone.0001246-Black1">[17]</a>, <a href="#pone.0001246-Wagner1">[35]</a>, <a href="#pone.0001246-Chabot1">[36]</a>, UGCATG-containing elements <a href="#pone.0001246-Minovitsky1">[29]</a>, U/G-rich intronic elements that interact with CUG-repeat binding proteins, reviewed in <a href="#pone.0001246-Ladd1">[16]</a> or (UCAUY)3-containing sequences binding to Nova proteins <a href="#pone.0001246-Ule1">[37]</a> or the C-rich and G-rich motifs <a href="#pone.0001246-Han1">[27]</a>, <a href="#pone.0001246-Miriami1">[30]</a>. We examined these known splicing suppressor sequences and did not find sequence features of In100-elements.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9"></a><p>Although a great number of human EST sequences have been generated, we were only able to obtain 2259 In100-element containing genes with flanking exons annotated with EST sequences. These ESTs clearly show a gradient distribution, with significantly higher frequency of EST representation of the exons immediate downstream of In100-containing introns. Analyses of In100-containing sequences annotated with further upstream and downstream exons in 1252 genes revealed 526 genes that have EST evidence of alternative splicing, showing that the presence of In100 elements correlates with the skipping of their immediate upstream exons. As we focused on In100-containing genes with sufficient EST data, these 526 genes only represent a small subset of possible alternative spliced genes. The observation that In100 elements exist in different gene families and their correlation with alternative splicing provides an opportunity for further study of In100 elements as a genomic signal for alternative splicing.</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10"></a><p>The finding that the exons upstream of In100 are often skipped corroborates with the alternative splicing patterns observed for several caspase genes, including casp-2, 3, 8 and 9. Some of these splicing isoforms have been reported as distinct cDNAs with different biological activities <a href="#pone.0001246-Bergeron1">[8]</a>, <a href="#pone.0001246-Cote1">[9]</a>, <a href="#pone.0001246-Huang1">[38]</a>–<a href="#pone.0001246-Srinivasula1">[40]</a>. Our experiments also reveal previously unknown splicing isoforms, Casp-8x and Casp-9x, as a result of skipping of exons upstream of their In100-containing introns (<a href="#pone-0001246-g006">Figure 6</a>). These splicing isoforms encode products lacking the caspase active domain. It is obvious that the dosage of active caspases has to be tightly controlled in response to different cell death signals. By generating enzymatically active versus inactive isoforms, alternative splicing provides an excellent mechanism for fine-tuning the level of active caspases at the post-transcriptional level.</p>
</div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Construction of a human exon-intron sequence database</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>The sequence and intron/exon annotation of each gene in human genome were downloaded from BioMart, annotated by ENSEMBL 44 GENES (SANGER), (<a href="http://www.biomart.org">http://www.biomart.org</a>). We constructed database where each human gene spans from the 5′ most exon to the 3′ most exon of all transcripts, including all coding and non-coding exonic sequence as well as intronic sequences. In addition, exon and intron positions as well as description of gene sequences were recorded.</p>


<h4>Creation of an algorithm to scan In100 candidate sequences</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A program was written in C language to search In100 candidate sequences in the human exon-intron sequence database. In the program, we designed a sliding window along the DNA sequences to move from the 5′ to the 3′ end of each sequence. The sliding window is 100 nt in length based on the length of the In100 element defined by biochemical experiments on casp-2. The signal of In100 was defined as: UAAC[branch site]-5-10 nt-(Py)n[polypyrimidine tract = U/C rich]-CAG-(U/C rich, 20–60 nt). The program searched candidate sequences for In100 elements and recorded their locations in introns or exons, the locus names as well as the description of the gene. The nucleotide composition of In100-like elements in the regions with the features contributing to the alternative splicing was analyzed, using Web-Logo <a href="#pone.0001246-Crooks1">[12]</a> (<a href="http://www.weblogo.berkeley.edu">http://www.weblogo.berkeley.edu</a>) and PERL scripts.</p>


<h4>Analysis of Gene ontology and Gene super-family</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Both Gene family and Gene Ontology information were downloaded from BioMart, anotated by ENSEMBL 44 GENES (SANGER), (<a href="http://www.biomart.org">http://www.biomart.org</a>). The classification of In100-containing genes into gene families was executed with PERL scripts. The GO terms were binned into MGI GO_Slim categories, using MGI GO_Slim Chart Tool (<a href="http://proto.informatics.jax.org/prototypes/GOTools/web-docs/MGI_GO_Slim_Chart.html">http://proto.informatics.jax.org/prototy​pes/GOTools/web-docs/MGI_GO_Slim_Chart.h​tml</a>). With GOstat <a href="#pone.0001246-Beissbarth1">[41]</a> (<a href="http://gostat.wehi.edu.au/">http://gostat.wehi.edu.au/</a>), the GO distribution of In100-containing genes and all Ensemble annotated human genes were compared and the over-presented GO terms in In100-containing genes were revealed.</p>


<h4>Analysis of the factors associated with the enrichment of In100-elements</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>To construct the control set for analyzing the factors associated with the enrichment of In100-elements, we generated 1000 intron-exon databases by simulation. In each simulated intron-exon database, we randomly generated a) 1500 genes; b) for each gene, the number of exons and introns are 7 and 6, since the average number of exon and intron in the real human intron-exon database per gene is 7.3 and 6.3; c) for each exon and intron sequence, the frequencies of their nucleotide composition are the same as the ones in the real human intron-exon database; d) the distribution of the length of intronic and exonic regions are the same as those in the real database. Then we searched In100-element sequences in those 1000 simulated intron-exon databases. We found that the total number of In100-elements found in the simulated databases is 8 times lower than the observed one in the real human intron-exon database. We also normalized these numbers by the total length of the corresponding databases (i.e. the frequency of In100-elements) and compared the frequency of In100-elements in the simulated database with the one observed in the human genome. The frequency of In100-elements in the human genome is significantly different from that detected in the simulated databases (<a href="#pone.0001246.s009">Figure S3</a>).</p>


<h4>Sequence analyses and statistical tests</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>To determine whether the detected In100 candidates are authentic intronic elements or random sequences, we used two approaches: (1) Comparing the frequencies of the candidates in exons and introns based on the total length distribution of exons and introns; (2) examining the frequency of trinucleotide (CAG, TAG, GAG and AAG) at the last three positions of the decoy 3′ splice sites in intronic and exonic regions, respectively.</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2"></a><p>First, a chi-square test was used to compare the observed frequency distribution of the elements in introns and exons with the expected frequency distribution based on the ratio of total exon length over total intron length with a degree of freedom of 1. Second, if the frequency differences of the In100 candidate in the four types (CAG, TAG, GAG, and AAG) are a consequence of random distribution of nucleotide composition, they are expected to be proportional to the frequency of four nucleotides A, T, G, C, in the human intronic and exonic regions respectively. We thus used human nucleotide frequencies in the intron and exon sequences as a theoretical expectation of the four types of candidate elements. Chi-square test was used to test the difference between observed and expected distribution with a degree of freedom of 3.</p>
<a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3"></a><p>The Human EST database was downloaded from ncbi.nlm.nih.gov (dbEST release 041405). The pairs of immediate flanking exons of 4277 In100-elements (located within introns) were extracted according to the Ensembl 44 gene annotation. For In100-element lacking control group, we selected exon pairs located in the same In100-element containing genes by excluding exon pairs that immediately flank an In100-element. The first or the last exons of the genes were also excluded to avoid compounding effects due to potential influence of alternative promoter usage or alternative polyadenylation. In order to quantitatively analyze the splicing pattern of an exon pair, we separately aligned each exon against the human EST database using BLAT <a href="#pone.0001246-Florea1">[42]</a>. Alignments with sequence identity equal or higher than 98% that cover at least 70% exon length were used as the criteria to identify the match of an EST with a particular exon.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4"></a><p>From 2757 In100-containing genes, we chose the triple exon cassettes with sufficient EST information, namely the ones with at least one EST simultaneously representing Exon1 and Exon3 (with/without Exon2). Alignments with at least 98% sequence identity and 70% exon length coverage were used as the criteria to identify the representation of an EST with particular exons. Next, In100-containing exon triplets (i.e. exon1-exon2-In100-exon3) and In100-lacking exon triplets (i.e., exon triplets not overlapped with In100-containing ones) were separated as experimental and negative control sets. The presence of ESTs with exon1-exon3 (lacking exon2) sequences is considered as the evidence of skipping of Exon2. Thus, the frequencies of exon2 skipping were computed for In100-containing or In100-lacking exon triplets respectively (<a href="#pone.0001246.s006">Table S6</a>). In addition, AltSplice <a href="#pone.0001246-Stamm1">[14]</a> database (<a href="http://www.ebi.ac.uk/asd/altsplice">www.ebi.ac.uk/asd/altsplice</a>) was searched for In100-element containing genes. All the intermediate steps were carried out with PERL scripts.</p>


<h4>RT-PCR detection of alternative splicing of caspases</h4>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1"></a><p>RNA was extracted from HEK293 cells for detecting human caspase gene alternative splicing. Specific primers corresponding to the exonic regions of human caspase genes were used to detect splicing isoforms by RT-PCR with the previously described protocol <a href="#pone.0001246-Cote1">[9]</a>. The corresponding splicing isoforms were verified by sequencing.</p>

</div>

<div id="section5" class="section"><a id="s5" name="s5" toc="s5" title="Supporting Information"></a><h3>Supporting Information</h3><div class="figshare_widget" doi="10.1371/journal.pone.0001246"></div><a name="pone.0001246.s001" id="pone.0001246.s001"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s001">Table S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1"></a><p class="preSiDOI">List of 4342 sequences containing In100-like elements.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s001</p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2"></a><p class="postSiDOI">(1.85 MB PDF)</p>
<a name="pone.0001246.s002" id="pone.0001246.s002"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s002">Table S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1"></a><p class="preSiDOI">List of 1865 gene families involved in the 2491 In100-containing genes. These gene families are ordered by the number of In100-containing members in each family.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s002</p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2"></a><p class="postSiDOI">(0.35 MB XLS)</p>
<a name="pone.0001246.s003" id="pone.0001246.s003"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s003">Table S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1"></a><p class="preSiDOI">The top over-presented GO terms of In100-containing genes revealed in Biology Process, Molecular Function and Cell Component.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s003</p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2"></a><p class="postSiDOI">(0.04 MB XLS)</p>
<a name="pone.0001246.s004" id="pone.0001246.s004"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s004">Table S4. </a></strong></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1"></a><p class="preSiDOI">List of skipped In100-element upstream exon.Among total 524 cases, 395 show EST retaining and skipping the immediate upstream exons, and 129 present only EST skipping them.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s004</p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2"></a><p class="postSiDOI">(0.27 MB XLS)</p>
<a name="pone.0001246.s005" id="pone.0001246.s005"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s005">Table S5. </a></strong></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1"></a><p class="preSiDOI">List of 75 In100-like sequences (in 69 genes) with evidence of skipping of In100 upstream exon, identified from AltSplice <a href="#pone.0001246-Stamm1">[14]</a> database</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s005</p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2"></a><p class="postSiDOI">(0.02 MB XLS)</p>
<a name="pone.0001246.s006" id="pone.0001246.s006"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s006">Table S6. </a></strong></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1"></a><p class="preSiDOI">The frequencies of exon2 skipping of experimental and control exon triplets by EST evidence.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s006</p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2"></a><p class="postSiDOI">(0.07 MB PDF)</p>
<a name="pone.0001246.s007" id="pone.0001246.s007"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s007">Figure S1. </a></strong></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1"></a><p class="preSiDOI">Upstream exon positional distribution along the mRNA sequence.(A) In100-element flanking exon pairs (2728). (B) Negative Control exon pairs (23624).</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s007</p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
<a name="pone.0001246.s008" id="pone.0001246.s008"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s008">Figure S2. </a></strong></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1"></a><p class="preSiDOI">The 5′ end positional distribution along the mRNA sequence. (A) In100-containing exon triplets (524). (B) In100-lacking exon triplets (4116)</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s008</p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2"></a><p class="postSiDOI">(0.03 MB PDF)</p>
<a name="pone.0001246.s009" id="pone.0001246.s009"></a><p class="siTitle"><strong><a href="/article/fetchSingleRepresentation.action?uri=info:doi/10.1371/journal.pone.0001246.s009">Figure S3. </a></strong></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1"></a><p class="preSiDOI">The frequency of In100-elements found in the randomized exon-intron databases and in the human genome In100 frequency: the number of In100 elements per million nt in the sequence database. The histogram of the frequency of In100-elements found in simulated databases is in black and the range of those frequencies is: 0 - 0.89 per million nt, the mean is 0.54 per million nt and the standard deviation is 0.10 per million nt; the frequency of In100-elements in the real human intron-exon database is in red and the value is 3.8 per million nt. The probability to observe such frequency of In100-elements in a randomized database is less than 0.001.</p>
<p class="siDoi">doi:10.1371/journal.pone.0001246.s009</p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2"></a><p class="postSiDOI">(0.02 MB PDF)</p>
</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank Drs. Chris Burge, Jocelyn Cote and Tim Nilsen for helpful suggestions and critical reading of the manuscript and Wu Lab members for generous help and insightful suggestions.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: ML JYW. Performed the experiments: NH JW KF MV. Analyzed the data: ML AF JW JYW ZK WG MV. Wrote the paper: ML AF JW JYW ZK WG MV.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0001246-Johnson1" id="pone.0001246-Johnson1"></a>Johnson JM, Castle J, Garrett-Engele P, Kan ZY, Loerch PM, et al.  (2003) Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays. Science  302: 2141–4.  <ul class="find" data-citedArticleID="1024408" data-doi="10.1126/science.1090100"><li><a href="http://dx.doi.org/10.1126/science.1090100" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Genome-wide+survey+of+human+alternative+pre-mRNA+splicing+with+exon+junction+microarrays." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Genome-wide+survey+of+human+alternative+pre-mRNA+splicing+with+exon+junction+microarrays.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">2.
              </span><a name="pone.0001246-Wu1" id="pone.0001246-Wu1"></a>Wu JY, Tang H, Havlioglu N (2003) Alternative pre-mRNA splicing and regulation of programmed cell death. Prog Mol Subcell Biol  31: 153–85.  <ul class="find" data-citedArticleID="1024438" data-doi="10.1007/978-3-662-09728-1_6"><li><a href="http://dx.doi.org/10.1007/978-3-662-09728-1_6" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+pre-mRNA+splicing+and+regulation+of+programmed+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+pre-mRNA+splicing+and+regulation+of+programmed+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">3.
              </span><a name="pone.0001246-Wu2" id="pone.0001246-Wu2"></a>Wu JY, Yuan L, Havlioglu N (2004) Alternatively spliced genes. In Encyclopedia of Molecular Cell Biology and Molecular Medicine Vol1. pp 125–177. Edited by Meyers R.A. (2nd edition) (Wiley-VCH).   <ul class="find" data-citedArticleID="1024440" data-doi="10.1002/3527600906.mcb.200300189"><li><a href="http://dx.doi.org/10.1002/3527600906.mcb.200300189" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternatively+spliced+genes.+In+Encyclopedia+of+Molecular+Cell+Biology+and+Molecular+Medicine+Vol1.+pp+125%E2%80%93177." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternatively+spliced+genes.+In+Encyclopedia+of+Molecular+Cell+Biology+and+Molecular+Medicine+Vol1.+pp+125%E2%80%93177.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">4.
              </span><a name="pone.0001246-Boyce1" id="pone.0001246-Boyce1"></a>Boyce M, Degterev A, Yuan J (2004) Caspases: an ancient cellular sword of Damocles. Cell Death Differ  11: 29–37.  <ul class="find" data-citedArticleID="1024372" data-doi="10.1038/sj.cdd.4401339"><li><a href="http://dx.doi.org/10.1038/sj.cdd.4401339" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caspases%3A+an+ancient+cellular+sword+of+Damocles." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caspases%3A+an+ancient+cellular+sword+of+Damocles.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0001246-Jiang1" id="pone.0001246-Jiang1"></a>Jiang X, Wang X (2004) Cytochrome C-mediated apoptosis. Annu Rev Biochem  73: 87–106.  <ul class="find" data-citedArticleID="1024404" data-doi="10.1146/annurev.biochem.73.011303.073706"><li><a href="http://dx.doi.org/10.1146/annurev.biochem.73.011303.073706" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cytochrome+C-mediated+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cytochrome+C-mediated+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">6.
              </span><a name="pone.0001246-Wang1" id="pone.0001246-Wang1"></a>Wang L, Miura M, Bergeron L, Zhu H, Yuan J (1994) Ich-1, an Ice/ced-3-related gene, encodes both positive and negative regulators of programmed cell death. Cell  78: 739–50.  <ul class="find" data-citedArticleID="1024434" data-doi="10.1016/s0092-8674(94)90422-7"><li><a href="http://dx.doi.org/10.1016/s0092-8674(94)90422-7" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Ich-1%2C+an+Ice%2Fced-3-related+gene%2C+encodes+both+positive+and+negative+regulators+of+programmed+cell+death." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Ich-1%2C+an+Ice%2Fced-3-related+gene%2C+encodes+both+positive+and+negative+regulators+of+programmed+cell+death.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">7.
              </span><a name="pone.0001246-Jiang2" id="pone.0001246-Jiang2"></a>Jiang ZH, Zhang WJ, Rao Y, Wu JY (1998) Regulation of Ich-1 pre-mRNA alternative splicing and apoptosis by mammalian splicing factors. Proc Natl Acad Sci U S A  95: 9155–60.  <ul class="find" data-citedArticleID="1024406" data-doi="10.1073/pnas.95.16.9155"><li><a href="http://dx.doi.org/10.1073/pnas.95.16.9155" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Regulation+of+Ich-1+pre-mRNA+alternative+splicing+and+apoptosis+by+mammalian+splicing+factors." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Regulation+of+Ich-1+pre-mRNA+alternative+splicing+and+apoptosis+by+mammalian+splicing+factors.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">8.
              </span><a name="pone.0001246-Bergeron1" id="pone.0001246-Bergeron1"></a>Bergeron L, Perez GI, Macdonald G, Shi LF, Sun Y, et al.  (1998) Defects in regulation of apoptosis in caspase-2-deficient mice. Genes Dev  12: 1304–14.  <ul class="find" data-citedArticleID="1024366" data-doi="10.1101/gad.12.9.1304"><li><a href="http://dx.doi.org/10.1101/gad.12.9.1304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Defects+in+regulation+of+apoptosis+in+caspase-2-deficient+mice." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Defects+in+regulation+of+apoptosis+in+caspase-2-deficient+mice.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0001246-Cote1" id="pone.0001246-Cote1"></a>Cote J, Dupuis S, Jiang Z, Wu JY (2001) Caspase-2 pre-mRNA alternative splicing: Identification of an intronic element containing a decoy 3′ acceptor site. Proc Natl Acad Sci U S A  98: 938–43.  <ul class="find" data-citedArticleID="1024378" data-doi="10.1073/pnas.98.3.938"><li><a href="http://dx.doi.org/10.1073/pnas.98.3.938" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Caspase-2+pre-mRNA+alternative+splicing%3A+Identification+of+an+intronic+element+containing+a+decoy+3%E2%80%B2+acceptor+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Caspase-2+pre-mRNA+alternative+splicing%3A+Identification+of+an+intronic+element+containing+a+decoy+3%E2%80%B2+acceptor+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">10.
              </span><a name="pone.0001246-Cote2" id="pone.0001246-Cote2"></a>Cote J, Dupuis S, Wu JY (2001) Polypyrimidine track-binding protein binding downstream of caspase-2 alternative exon 9 represses its inclusion. J Biol Chem  276: 8535–43.  <ul class="find" data-citedArticleID="1024380" data-doi="10.1074/jbc.m008924200"><li><a href="http://dx.doi.org/10.1074/jbc.m008924200" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polypyrimidine+track-binding+protein+binding+downstream+of+caspase-2+alternative+exon+9+represses+its+inclusion." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polypyrimidine+track-binding+protein+binding+downstream+of+caspase-2+alternative+exon+9+represses+its+inclusion.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">11.
              </span><a name="pone.0001246-Long1" id="pone.0001246-Long1"></a>Long M, de Souza SJ, Rosenberg C, Gilbert W (1998) Relationship between “proto-splice sites” and intron phases: evidence from dicodon analysis. Proc Natl Acad Sci U S A  95: 219–23.  <ul class="find" data-citedArticleID="1024412" data-doi="10.1073/pnas.95.1.219"><li><a href="http://dx.doi.org/10.1073/pnas.95.1.219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Relationship+between+%E2%80%9Cproto-splice+sites%E2%80%9D+and+intron+phases%3A+evidence+from+dicodon+analysis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Relationship+between+%E2%80%9Cproto-splice+sites%E2%80%9D+and+intron+phases%3A+evidence+from+dicodon+analysis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">12.
              </span><a name="pone.0001246-Crooks1" id="pone.0001246-Crooks1"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE (2004) WebLogo: A sequence logo generator, Genome Research, 14: 1188–1190.  <ul class="find" data-citedArticleID="1024382" data-doi="10.1101/gr.849004"><li><a href="http://dx.doi.org/10.1101/gr.849004" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=WebLogo%3A+A+sequence+logo+generator%2C+Genome+Research%2C" target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22WebLogo%3A+A+sequence+logo+generator%2C+Genome+Research%2C%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">13.
              </span><a name="pone.0001246-Deutsch1" id="pone.0001246-Deutsch1"></a>Deutsch M, Long M (1999) Intron-exon structures of eukaryotic model organisms. Nucleic Acids Res  27: 3219–28.  <ul class="find" data-citedArticleID="1024386" data-doi="10.1093/nar/27.15.3219"><li><a href="http://dx.doi.org/10.1093/nar/27.15.3219" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Intron-exon+structures+of+eukaryotic+model+organisms." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Intron-exon+structures+of+eukaryotic+model+organisms.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">14.
              </span><a name="pone.0001246-Stamm1" id="pone.0001246-Stamm1"></a>Stamm S, Riethoven JJ, Le Texier V, Gopalakrishnan C, Kumanduri V, et al.  (2006) ASD: a bioinformatics resource on alternative splicing. Nucleic Acids Research  34: D46–D55.  <ul class="find" data-citedArticleID="1024428" data-doi="10.1093/nar/gkj031"><li><a href="http://dx.doi.org/10.1093/nar/gkj031" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ASD%3A+a+bioinformatics+resource+on+alternative+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ASD%3A+a+bioinformatics+resource+on+alternative+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">15.
              </span><a name="pone.0001246-Smith1" id="pone.0001246-Smith1"></a>Smith CW, Valcarcel J (2000) Alternative pre-mRNA splicing: the logic of combinatorial control. Trends Biochem Sci  25: 381–8.  <ul class="find" data-citedArticleID="1024424" data-doi="10.1016/s0968-0004(00)01604-2"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01604-2" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Alternative+pre-mRNA+splicing%3A+the+logic+of+combinatorial+control." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Alternative+pre-mRNA+splicing%3A+the+logic+of+combinatorial+control.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">16.
              </span><a name="pone.0001246-Ladd1" id="pone.0001246-Ladd1"></a>Ladd AN, Cooper TA (2002) Finding signals that regulate alternative splicing in the post-genomic era. Genome Biol  3: reviews0008.  <ul class="find" data-citedArticleID="1024410"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Finding+signals+that+regulate+alternative+splicing+in+the+post-genomic+era.&amp;auth=&amp;atitle=Finding+signals+that+regulate+alternative+splicing+in+the+post-genomic+era." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Finding+signals+that+regulate+alternative+splicing+in+the+post-genomic+era." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Finding+signals+that+regulate+alternative+splicing+in+the+post-genomic+era.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">17.
              </span><a name="pone.0001246-Black1" id="pone.0001246-Black1"></a>Black DL (2003) Mechanisms of alternative pre-messenger RNA splicing. Annu Rev Biochem  72: 291–336.  <ul class="find" data-citedArticleID="1024368"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Mechanisms+of+alternative+pre-messenger+RNA+splicing.&amp;auth=&amp;atitle=Mechanisms+of+alternative+pre-messenger+RNA+splicing." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Mechanisms+of+alternative+pre-messenger+RNA+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Mechanisms+of+alternative+pre-messenger+RNA+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">18.
              </span><a name="pone.0001246-Zhang1" id="pone.0001246-Zhang1"></a>Zhang XH, Leslie CS, Chasin LA (2005) Computational searches for splicing signals. Methods. 37(4): 292–305.  <ul class="find" data-citedArticleID="1024444" data-doi="10.1016/j.ymeth.2005.07.011"><li><a href="http://dx.doi.org/10.1016/j.ymeth.2005.07.011" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+searches+for+splicing+signals.+Methods." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+searches+for+splicing+signals.+Methods.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">19.
              </span><a name="pone.0001246-Blencowe1" id="pone.0001246-Blencowe1"></a>Blencowe BJ (2000) Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases. Trends Biochem Sci  25: 106–10 (2000).  <ul class="find" data-citedArticleID="1024370" data-doi="10.1016/s0968-0004(00)01549-8"><li><a href="http://dx.doi.org/10.1016/s0968-0004(00)01549-8" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Exonic+splicing+enhancers%3A+mechanism+of+action%2C+diversity+and+role+in+human+genetic+diseases.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">20.
              </span><a name="pone.0001246-Faustino1" id="pone.0001246-Faustino1"></a>Faustino NA, Cooper TA (2003) Pre-mRNA splicing and human disease. Genes Dev  17: 419–37.  <ul class="find" data-citedArticleID="1024390" data-doi="10.1101/gad.1048803"><li><a href="http://dx.doi.org/10.1101/gad.1048803" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Pre-mRNA+splicing+and+human+disease." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Pre-mRNA+splicing+and+human+disease.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">21.
              </span><a name="pone.0001246-Fu1" id="pone.0001246-Fu1"></a>Fu XD (2004) Towards a splicing code. Cell  119: 736–8.  <ul class="find" data-citedArticleID="1024394" data-doi="10.1016/j.cell.2004.11.039"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.039" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Towards+a+splicing+code." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Towards+a+splicing+code.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">22.
              </span><a name="pone.0001246-Fairbrother1" id="pone.0001246-Fairbrother1"></a>Fairbrother WG, Yeh RF, Sharp PA, Burge CB (2002) Predictive identification of exonic splicing enhancers in human genes. Science  297: 1007–13.  <ul class="find" data-citedArticleID="1024388" data-doi="10.1126/science.1073774"><li><a href="http://dx.doi.org/10.1126/science.1073774" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Predictive+identification+of+exonic+splicing+enhancers+in+human+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Predictive+identification+of+exonic+splicing+enhancers+in+human+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">23.
              </span><a name="pone.0001246-Cartegni1" id="pone.0001246-Cartegni1"></a>Cartegni L, Wang J, Zhu Z, Zhang MQ, Krainer AR (2003) ESEfinder: A web resource to identify exonic splicing enhancers. Nucleic Acids Res  31: 3568–71.  <ul class="find" data-citedArticleID="1024374" data-doi="10.1093/nar/gkg616"><li><a href="http://dx.doi.org/10.1093/nar/gkg616" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=ESEfinder%3A+A+web+resource+to+identify+exonic+splicing+enhancers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22ESEfinder%3A+A+web+resource+to+identify+exonic+splicing+enhancers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">24.
              </span><a name="pone.0001246-Yeo1" id="pone.0001246-Yeo1"></a>Yeo G, Holste D, Kreiman G, Burge CB (2004) Variation in alternative splicing across human tissues. Genome Biol  5: R74.  <ul class="find" data-citedArticleID="1024442" data-doi="10.1186/gb-2004-5-10-r74"><li><a href="http://dx.doi.org/10.1186/gb-2004-5-10-r74" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Variation+in+alternative+splicing+across+human+tissues." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Variation+in+alternative+splicing+across+human+tissues.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">25.
              </span><a name="pone.0001246-Wang2" id="pone.0001246-Wang2"></a>Wang Z, Rolish ME, Yeo G, Tung V, Mawson M, et al.  (2004) Systematic identification and analysis of exonic splicing silencers. Cell  119: 831–45.  <ul class="find" data-citedArticleID="1024436" data-doi="10.1016/j.cell.2004.11.010"><li><a href="http://dx.doi.org/10.1016/j.cell.2004.11.010" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Systematic+identification+and+analysis+of+exonic+splicing+silencers." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Systematic+identification+and+analysis+of+exonic+splicing+silencers.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">26.
              </span><a name="pone.0001246-Zhang2" id="pone.0001246-Zhang2"></a>Zhang XH, Chasin LA (2004) Computational definition of sequence motifs governing constitutive exon splicing. Genes Dev  18: 1241–50.  <ul class="find" data-citedArticleID="1024446" data-doi="10.1101/gad.1195304"><li><a href="http://dx.doi.org/10.1101/gad.1195304" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Computational+definition+of+sequence+motifs+governing+constitutive+exon+splicing." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Computational+definition+of+sequence+motifs+governing+constitutive+exon+splicing.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">27.
              </span><a name="pone.0001246-Han1" id="pone.0001246-Han1"></a>Han K, Yeo G, An P, Burge CB, Grabowski PJ (2005) A combinatorial code for splicing silencing: UAGG and GGGG motifs. PLoS Biol.  3(5): e158.  <ul class="find" data-citedArticleID="1024396" data-doi="10.1371/journal.pbio.0030158"><li><a href="http://dx.doi.org/10.1371/journal.pbio.0030158" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+combinatorial+code+for+splicing+silencing%3A+UAGG+and+GGGG+motifs." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+combinatorial+code+for+splicing+silencing%3A+UAGG+and+GGGG+motifs.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">28.
              </span><a name="pone.0001246-McCullough1" id="pone.0001246-McCullough1"></a>McCullough AJ, Berget SM (1997) G triplets located throughout a class of small vertebrate introns enforce intron borders and regulate splice site selection. Mol Cell Biol  17: 4562–71.  <ul class="find" data-citedArticleID="1024416"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=G+triplets+located+throughout+a+class+of+small+vertebrate+introns+enforce+intron+borders+and+regulate+splice+site+selection.&amp;auth=&amp;atitle=G+triplets+located+throughout+a+class+of+small+vertebrate+introns+enforce+intron+borders+and+regulate+splice+site+selection." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=G+triplets+located+throughout+a+class+of+small+vertebrate+introns+enforce+intron+borders+and+regulate+splice+site+selection." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22G+triplets+located+throughout+a+class+of+small+vertebrate+introns+enforce+intron+borders+and+regulate+splice+site+selection.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">29.
              </span><a name="pone.0001246-Minovitsky1" id="pone.0001246-Minovitsky1"></a>Minovitsky S, Gee SL, Schokrpur S, Dubchak I, Conboy JG (2005) The splicing regulatory element, UGCAUG, is phylogenetically and spatially conserved in introns that flank tissue-specific alternative exons. Nucleic Acids Res  33: 714–24.  <ul class="find" data-citedArticleID="1024418" data-doi="10.1093/nar/gki210"><li><a href="http://dx.doi.org/10.1093/nar/gki210" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+splicing+regulatory+element%2C+UGCAUG%2C+is+phylogenetically+and+spatially+conserved+in+introns+that+flank+tissue-specific+alternative+exons." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+splicing+regulatory+element%2C+UGCAUG%2C+is+phylogenetically+and+spatially+conserved+in+introns+that+flank+tissue-specific+alternative+exons.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">30.
              </span><a name="pone.0001246-Miriami1" id="pone.0001246-Miriami1"></a>Miriami E, Margalit H, Sperling R (2003) Conserved sequence elements associated with exon skipping. Nucleic Acids Res  31: 1974–83.  <ul class="find" data-citedArticleID="1024420" data-doi="10.1093/nar/gkg279"><li><a href="http://dx.doi.org/10.1093/nar/gkg279" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Conserved+sequence+elements+associated+with+exon+skipping." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Conserved+sequence+elements+associated+with+exon+skipping.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">31.
              </span><a name="pone.0001246-Siebel1" id="pone.0001246-Siebel1"></a>Siebel CW, Fresco LD, Rio DC (1992) The mechanism of somatic inhibition of Drosophila P-element pre-mRNA splicing: multiprotein complexes at an exon pseudo-5′ splice site control U1 snRNP binding. Genes Dev  6: 1386–401.  <ul class="find" data-citedArticleID="1024422" data-doi="10.1101/gad.6.8.1386"><li><a href="http://dx.doi.org/10.1101/gad.6.8.1386" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+mechanism+of+somatic+inhibition+of+Drosophila+P-element+pre-mRNA+splicing%3A+multiprotein+complexes+at+an+exon+pseudo-5%E2%80%B2+splice+site+control+U1+snRNP+binding." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+mechanism+of+somatic+inhibition+of+Drosophila+P-element+pre-mRNA+splicing%3A+multiprotein+complexes+at+an+exon+pseudo-5%E2%80%B2+splice+site+control+U1+snRNP+binding.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">32.
              </span><a name="pone.0001246-Lou1" id="pone.0001246-Lou1"></a>Lou H, Yang Y, Cote GJ, Berget SM, Gagel RF (1995) An intron enhancer containing a 5′ splice site sequence in the human calcitonin/calcitonin gene-related peptide gene. Mol Cell Biol  15: 7135–42.  <ul class="find" data-citedArticleID="1024414"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=An+intron+enhancer+containing+a+5%3F+splice+site+sequence+in+the+human+calcitonin%2Fcalcitonin+gene-related+peptide+gene.&amp;auth=&amp;atitle=An+intron+enhancer+containing+a+5%3F+splice+site+sequence+in+the+human+calcitonin%2Fcalcitonin+gene-related+peptide+gene." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=An+intron+enhancer+containing+a+5%E2%80%B2+splice+site+sequence+in+the+human+calcitonin%2Fcalcitonin+gene-related+peptide+gene." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22An+intron+enhancer+containing+a+5%E2%80%B2+splice+site+sequence+in+the+human+calcitonin%2Fcalcitonin+gene-related+peptide+gene.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">33.
              </span><a name="pone.0001246-DelGattoKonczak1" id="pone.0001246-DelGattoKonczak1"></a>Del Gatto-Konczak F, Bourgeois CF, Le Guiner C, Kister L, Gesnel MC, et al.  (2000) The RNA-binding protein TIA-1 is a novel mammalian splicing regulator acting through intron sequences adjacent to a 5′ splice site. Mol Cell Biol  20: 6287–99.  <ul class="find" data-citedArticleID="1024384" data-doi="10.1128/mcb.20.17.6287-6299.2000"><li><a href="http://dx.doi.org/10.1128/mcb.20.17.6287-6299.2000" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=The+RNA-binding+protein+TIA-1+is+a+novel+mammalian+splicing+regulator+acting+through+intron+sequences+adjacent+to+a+5%E2%80%B2+splice+site." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22The+RNA-binding+protein+TIA-1+is+a+novel+mammalian+splicing+regulator+acting+through+intron+sequences+adjacent+to+a+5%E2%80%B2+splice+site.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">34.
              </span><a name="pone.0001246-Zorio1" id="pone.0001246-Zorio1"></a>Zorio DA, Lea K, Blumenthal T (1997) Cloning of Caenorhabditis U2AF65: an alternatively spliced RNA containing a novel exon. Mol Cell Biol  17: 946–53.  <ul class="find" data-citedArticleID="1024448"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Cloning+of+Caenorhabditis+U2AF65%3A+an+alternatively+spliced+RNA+containing+a+novel+exon.&amp;auth=&amp;atitle=Cloning+of+Caenorhabditis+U2AF65%3A+an+alternatively+spliced+RNA+containing+a+novel+exon." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Cloning+of+Caenorhabditis+U2AF65%3A+an+alternatively+spliced+RNA+containing+a+novel+exon." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Cloning+of+Caenorhabditis+U2AF65%3A+an+alternatively+spliced+RNA+containing+a+novel+exon.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">35.
              </span><a name="pone.0001246-Wagner1" id="pone.0001246-Wagner1"></a>Wagner EJ, Garcia-Blanco MA (2001) Polypyrimidine tract binding protein antagonizes exon definition. Mol Cell Biol  21: 3281–8.  <ul class="find" data-citedArticleID="1024432" data-doi="10.1128/mcb.21.10.3281-3288.2001"><li><a href="http://dx.doi.org/10.1128/mcb.21.10.3281-3288.2001" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Polypyrimidine+tract+binding+protein+antagonizes+exon+definition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Polypyrimidine+tract+binding+protein+antagonizes+exon+definition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">36.
              </span><a name="pone.0001246-Chabot1" id="pone.0001246-Chabot1"></a>Chabot B, LeBel C, Hutchison S, Nasim FH, Simard MJ (2003) Heterogeneous nuclear ribonucleoprotein particle A/B proteins and the control of alternative splicing of the mammalian heterogeneous nuclear ribonucleoprotein particle A1 pre-mRNA. Prog Mol Subcell Biol  31: 59–88.  <ul class="find" data-citedArticleID="1024376" data-doi="10.1007/978-3-662-09728-1_3"><li><a href="http://dx.doi.org/10.1007/978-3-662-09728-1_3" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Heterogeneous+nuclear+ribonucleoprotein+particle+A%2FB+proteins+and+the+control+of+alternative+splicing+of+the+mammalian+heterogeneous+nuclear+ribonucleoprotein+particle+A1+pre-mRNA." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Heterogeneous+nuclear+ribonucleoprotein+particle+A%2FB+proteins+and+the+control+of+alternative+splicing+of+the+mammalian+heterogeneous+nuclear+ribonucleoprotein+particle+A1+pre-mRNA.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">37.
              </span><a name="pone.0001246-Ule1" id="pone.0001246-Ule1"></a>Ule J, Darnell RB (2006) RNA binding proteins and the regulation of neuronal synaptic plasticity. Curr Opin Neurobiol.  16(1): 102–10.  <ul class="find" data-citedArticleID="1024430" data-doi="10.1016/j.conb.2006.01.003"><li><a href="http://dx.doi.org/10.1016/j.conb.2006.01.003" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=RNA+binding+proteins+and+the+regulation+of+neuronal+synaptic+plasticity." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22RNA+binding+proteins+and+the+regulation+of+neuronal+synaptic+plasticity.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">38.
              </span><a name="pone.0001246-Huang1" id="pone.0001246-Huang1"></a>Huang Y, Shin NH, Sun Y, Wang KK (2001) Molecular cloning and characterization of a novel caspase-3 variant that attenuates apoptosis induced by proteasome inhibition. Biochem Biophys Res Commun  283: 762–9.  <ul class="find" data-citedArticleID="1024400" data-doi="10.1006/bbrc.2001.4871"><li><a href="http://dx.doi.org/10.1006/bbrc.2001.4871" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Molecular+cloning+and+characterization+of+a+novel+caspase-3+variant+that+attenuates+apoptosis+induced+by+proteasome+inhibition." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Molecular+cloning+and+characterization+of+a+novel+caspase-3+variant+that+attenuates+apoptosis+induced+by+proteasome+inhibition.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">39.
              </span><a name="pone.0001246-Horiuchi1" id="pone.0001246-Horiuchi1"></a>Horiuchi T, Himeji D, Tsukamoto H, Harashima S-i, Hashimura C, et al.  (2000) Dominant expression of a novel splice variant of caspase-8 in human peripheral blood lymphocytes. Biochem Biophys Res Commun  272: 877–81.  <ul class="find" data-citedArticleID="1024398" data-doi="10.1006/bbrc.2000.2841"><li><a href="http://dx.doi.org/10.1006/bbrc.2000.2841" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Dominant+expression+of+a+novel+splice+variant+of+caspase-8+in+human+peripheral+blood+lymphocytes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Dominant+expression+of+a+novel+splice+variant+of+caspase-8+in+human+peripheral+blood+lymphocytes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">40.
              </span><a name="pone.0001246-Srinivasula1" id="pone.0001246-Srinivasula1"></a>Srinivasula SM, Ahmad M, Guo Y, Zhan Y, Lazebnik Y, et al.  (1999) Identification of an endogenous dominant-negative short isoform of caspase-9 that can regulate apoptosis. Cancer Res  59: 999–1002.  <ul class="find" data-citedArticleID="1024426"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=Identification+of+an+endogenous+dominant-negative+short+isoform+of+caspase-9+that+can+regulate+apoptosis.&amp;auth=&amp;atitle=Identification+of+an+endogenous+dominant-negative+short+isoform+of+caspase-9+that+can+regulate+apoptosis." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Identification+of+an+endogenous+dominant-negative+short+isoform+of+caspase-9+that+can+regulate+apoptosis." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Identification+of+an+endogenous+dominant-negative+short+isoform+of+caspase-9+that+can+regulate+apoptosis.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">41.
              </span><a name="pone.0001246-Beissbarth1" id="pone.0001246-Beissbarth1"></a>Beissbarth T, Speed TP (2004) GOstat: Find statistically overrepresented Gene Ontologies within a group of genes. Bioinformatics  20(9): 1464–1465.  <ul class="find" data-citedArticleID="1024364" data-doi="10.1093/bioinformatics/bth088"><li><a href="http://dx.doi.org/10.1093/bioinformatics/bth088" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=GOstat%3A+Find+statistically+overrepresented+Gene+Ontologies+within+a+group+of+genes." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22GOstat%3A+Find+statistically+overrepresented+Gene+Ontologies+within+a+group+of+genes.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">42.
              </span><a name="pone.0001246-Florea1" id="pone.0001246-Florea1"></a>Florea L, Hartzell G, Zhang Z, Rubin GM, Miller W (1998) A computer program for aligning a cDNA sequence with a genomic DNA sequence. Genome Res  8: 967–74.  <ul class="find" data-citedArticleID="1024392"><li><a href="http://www.crossref.org/guestquery/?auth2=&amp;atitle2=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence.&amp;auth=&amp;atitle=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence." target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22A+computer+program+for+aligning+a+cDNA+sequence+with+a+genomic+DNA+sequence.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li><li><span class="label">43.
              </span><a name="pone.0001246-International1" id="pone.0001246-International1"></a>International Human Genome Sequencing Consortium (2001) Initial sequencing and analysis of the human genome. Nature  409: 860–927.  <ul class="find" data-citedArticleID="1024402" data-doi="10.1038/nature04072"><li><a href="http://dx.doi.org/10.1038/nature04072" target="_new" title="Go to article in CrossRef">
                          View Article
                        </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Initial+sequencing+and+analysis+of+the+human+genome." target="_new" title="Go to article in PubMed">
                          PubMed/NCBI
                        </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=%22Initial+sequencing+and+analysis+of+the+human+genome.%22" target="_new" title="Go to article in Google Scholar">
                          Google Scholar
                        </a></li></ul></li></ol></div>

  </div>

      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.XML" value="99094"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.PDF" value="474687"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g001.PNG_L" value="135870"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g001.PNG_M" value="48315"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g001.PNG_S" value="5435"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g001.TIF" value="144430"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g001.PNG_I" value="15439"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g002.PNG_L" value="181833"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g002.PNG_M" value="126214"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g002.PNG_S" value="8519"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g002.TIF" value="229162"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g002.PNG_I" value="42692"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g003.PNG_L" value="592840"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g003.PNG_M" value="115345"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g003.PNG_S" value="9568"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g003.TIF" value="967222"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g003.PNG_I" value="38879"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t001.PNG_L" value="60628"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t001.PNG_M" value="77031"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t001.PNG_S" value="12295"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t001.TIF" value="242588"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t001.PNG_I" value="26433"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t002.PNG_L" value="123371"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t002.PNG_M" value="77250"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t002.PNG_S" value="9877"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t002.TIF" value="503322"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t002.PNG_I" value="24237"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g004.PNG_L" value="64918"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g004.PNG_M" value="39655"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g004.PNG_S" value="6278"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g004.TIF" value="150092"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g004.PNG_I" value="14242"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g005.PNG_L" value="58971"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g005.PNG_M" value="52506"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g005.PNG_S" value="7128"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g005.TIF" value="100438"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g005.PNG_I" value="19676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g006.PNG_L" value="438108"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g006.PNG_M" value="105607"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g006.PNG_S" value="7945"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g006.TIF" value="763280"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.g006.PNG_I" value="37298"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t003.PNG_L" value="47813"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t003.PNG_M" value="67954"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t003.PNG_S" value="10786"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t003.TIF" value="200974"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.t003.PNG_I" value="22270"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s001.PDF" value="1973972"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s002.XLS" value="349696"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s003.XLS" value="43008"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s004.XLS" value="272384"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s005.XLS" value="23040"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s006.PDF" value="72676"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s007.PDF" value="16975"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s008.PDF" value="28265"/>
      <input type="hidden" class="assetSize" name="info:doi/10.1371/journal.pone.0001246.s009.PDF" value="18795"/>

</div>
<div class="sidebar">

  <div class="article-actions cf">
      <div class="download">
        <span class="btn"><a href="/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001246&amp;representation=PDF" title="Download" target="_blank">Download PDF</a></span>
      </div>
      <div class="btn-reveal dropdown">
        <div class="dropdown-icon">
          <span class="btn">&nbsp;</span>
        </div>

        <div class="content">
          <ul class="bullet">
            <li><a href="/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0001246" title="Download citations">Citation</a></li>
            <li><a href="/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0001246&amp;representation=XML" title="Download article XML">XML</a></li>
          </ul>
        </div>
      </div> <!-- end btn-reveal dropdown-->


    <div class="btn-reveal flt-l">
        <span class="btn">Print</span>
        <div class="content">
            <ul class="bullet">
                <li id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
                <li>
                  <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=7&doi=10.1371/journal.pone.0001246&volume=&issue=&title=An Intronic Signal for Alternative Splicing in the Human Genome&author_name=Necat%20Havlioglu%2C%20Jun%20Wang%2C%20Kazuo%20Fushimi%2C%20Maria%20D.%20Vibranovski%2C%20Zhengyan%20Kan%2C%20Warren%20Gish%2C%20Alexei%20Fedorov%2C%20Manyuan%20Long%2C%20Jane%20Y.%20Wu&start_page=1&end_page=10" title="Odyssey Press">EzReprint</a>
                </li>
            </ul>
        </div>
    </div>

    <div class="btn-reveal flt-r">
        <span class="btn">Share</span>
        <div class="content">
            <ul class="social">
                <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246" target="_blank" title="Submit to Reddit"><img src="/images/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

                <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246" target="_blank" title="Share on Google+"><img src="/images/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

                <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0001246" target="_blank" title="Add to StumbleUpon"><img src="/images/icon.stumble.16.png" width="16" height="16" alt="StumbleUpon">StumbleUpon</a></li>

                <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246&amp;t=An%20Intronic%20Signal%20for%20Alternative%20Splicing%20in%20the%20Human%20Genome" target="_blank" title="Share on Facebook"><img src="/images/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

                <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246&title=An%20Intronic%20Signal%20for%20Alternative%20Splicing%20in%20the%20Human%20Genome&summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_blank" title="Add to LinkedIn"><img src="/images/icon.linkedin.16.png" width="16" height="16" alt="Mendeley">LinkedIn</a></li>

                <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246&amp;title=An%20Intronic%20Signal%20for%20Alternative%20Splicing%20in%20the%20Human%20Genome" target="_blank" title="Add to CiteULike"><img src="/images/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

                <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246" target="_blank" title="Add to Mendeley"><img src="/images/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

                <li><a href="https://www.pubchase.com/library?add_aid=10.1371%2Fjournal.pone.0001246&amp;source=plos" target="_blank" title="Add to PubChase"><img src="/images/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>


                <script type="text/javascript">
                    // replace tweet with one that's pre-shortened to 140 chars
                    function truncateTweetText() {
                        var twtTitle = 'An Intronic Signal for Alternative Splicing in the Human Genome';
                        var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0001246';
                        // all URLs posted to twitter get auto-shortened to 20 chars.
                        var maxLength = 140 - (20 + 1);
                        // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
                        if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
                        // set the href to use the shortened tweet
                        $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
                    }
                </script>
                <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20An%20Intronic%20Signal%20for%20Alternative%20Splicing%20in%20the%20Human%20Genome http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0001246" onclick="truncateTweetText();" target="_blank" title="Share on Twitter" id="twitter-share-link"><img src="/images/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

                <li><a href="/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0001246" title="Email this article"><img src="/images/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>
            </ul>
        </div>
    </div><!--end btn-reveal flt-r-->
</div><!-- end article-actions-->

<!-- begin Crossmark -->

<a id="open-crossmark" href="#" style="margin-top: -28px; display:block"><img style="border: 0; display: none;
 padding: 10px 0 18px 0;"  id="crossmark-icon" src="/images/logo-crossmark-bw.png" /></a>
<div id="crossmark-dialog" style="display: none;" title="">
    <!-- the external CrossMark data is loaded inside this iframe -->
    <iframe id="crossmark-dialog-frame" frameborder="0"></iframe>
</div>

<!-- end crossmark -->


<div class="block" id="subject-area-sidebar-block">
    <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img align="right"
                                                                                                           alt="info" src="/images/button_info.png"/><div id="subject-area-sidebar-block-help"><img align="right"
                                                                                                                                                                                                    src="/images/button_info.png"/><p>
        <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
    </p></div></div>
    </div>


    <ul id="subject-area-sidebar-list">
















          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Alternative+splicing%22" title="Search for articles in the subject area:'Alternative splicing'"><div class="flagText">Alternative splicing</div></a>
              <div data-categoryid="17931" data-articleid="25324"
                   data-categoryname="Alternative splicing"
                   class="flagImage" title="Flag 'Alternative splicing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22DNA+sequences%22" title="Search for articles in the subject area:'DNA sequences'"><div class="flagText">DNA sequences</div></a>
              <div data-categoryid="32003" data-articleid="25324"
                   data-categoryname="DNA sequences"
                   class="flagImage" title="Flag 'DNA sequences' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+databases%22" title="Search for articles in the subject area:'Genomic databases'"><div class="flagText">Genomic databases</div></a>
              <div data-categoryid="42727" data-articleid="25324"
                   data-categoryname="Genomic databases"
                   class="flagImage" title="Flag 'Genomic databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Genomic+signal+processing%22" title="Search for articles in the subject area:'Genomic signal processing'"><div class="flagText">Genomic signal processing</div></a>
              <div data-categoryid="18635" data-articleid="25324"
                   data-categoryname="Genomic signal processing"
                   class="flagImage" title="Flag 'Genomic signal processing' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Human+genomics%22" title="Search for articles in the subject area:'Human genomics'"><div class="flagText">Human genomics</div></a>
              <div data-categoryid="17253" data-articleid="25324"
                   data-categoryname="Human genomics"
                   class="flagImage" title="Flag 'Human genomics' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Introns%22" title="Search for articles in the subject area:'Introns'"><div class="flagText">Introns</div></a>
              <div data-categoryid="20277" data-articleid="25324"
                   data-categoryname="Introns"
                   class="flagImage" title="Flag 'Introns' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+databases%22" title="Search for articles in the subject area:'Sequence databases'"><div class="flagText">Sequence databases</div></a>
              <div data-categoryid="42787" data-articleid="25324"
                   data-categoryname="Sequence databases"
                   class="flagImage" title="Flag 'Sequence databases' as inappropriate"></div>
          </li>
          <li>
              <a href="/search/advanced?unformattedQuery=subject%3A%22Sequence+motif+analysis%22" title="Search for articles in the subject area:'Sequence motif analysis'"><div class="flagText">Sequence motif analysis</div></a>
              <div data-categoryid="19657" data-articleid="25324"
                   data-categoryname="Sequence motif analysis"
                   class="flagImage" title="Flag 'Sequence motif analysis' as inappropriate"></div>
          </li>
    </ul>
</div>

<div class="ad">
    <div class="title">Advertisement</div>






  <iframe id='a0852f54' name='a0852f54'
    src='http://ads.plos.org/www/delivery/afr.php?zoneid=381&amp;cb=4106'
    frameborder='0' scrolling='no' width='160' height='600'>
    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=9554'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=5499&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



</div>

<div id="twitter-alm-timeline" class="twitter-alm-timeline"></div>

<div class="block sidebar-comments">
    <div class="header">
        <h3>Comments</h3>
    </div>
      <p><a href="/annotation/listThread.action?root=10773">Evolutionary implications</a><br>Posted by mkukrishna</p>
      <p><a href="/annotation/listThread.action?root=45801">Useful and interesting</a><br>Posted by mkukrishna</p>
</div>

</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="http://wl.figshare.com/static/p_widget.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="/images/logo-plos-footer.png" alt="PLOS Logo" class="logo" />
      <p><a href="/static/releaseNotes">Ambra 2.9.16</a> Managed Colocation provided <br />by <a href="http://www.isc.org/">Internet Systems Consortium</a>.<p>
      <div class="nav nav-aux">
        <a href="/static/privacy">Privacy Policy</a> |
        <a href="/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org">plos.org</a></p>
        <p><a href="http://blogs.plos.org">Blogs</a></p>
        <p><a href="http://www.ploscollections.org">Collections</a></p>
        <p><a href="/feedback/new">Send us feedback</a></p>

        <p>California (US) corporation #C2354500, based in San Francisco</p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="/javascript/jquery-1.8.1-min.js?v=Tm7VCOzZz3lE03ghpkS6SWkHbyI"></script>
<script type="text/javascript" src="/javascript/ga-min.js?v=lNQ4gt8QcPDatjsdOFl_FGpPhLY"></script>
<script type="text/javascript" src="/javascript/jquery.hoverIntent-min.js?v=mRiGNYY9cIXxVb8u0K_MdW7hHnc"></script>
<script type="text/javascript" src="/javascript/jquery.placeholder-min.js?v=21Pn56Ur9h1N4K4VZDa0nqI3Pxo"></script>
<script type="text/javascript" src="/javascript/jquery.jsonp-2.4.0-min.js?v=lqTpzoHfSq3I5Ygo01qq5WankEo"></script>
<script type="text/javascript" src="/javascript/jquery-ui-1.9.2.custom-min.js?v=raSSlfNO0YsV5uUpAKmTB9n5VTc"></script>
<script type="text/javascript" src="/javascript/jquery.tooltip-min.js?v=cw+6Smh+mdryIA25xvqIvHMrnZM"></script>
<script type="text/javascript" src="/javascript/jquery.uniform-min.js?v=kYUAnX6W2W_2fK3RIuQ2m_YFG9U"></script>
<script type="text/javascript" src="/javascript/jquery.pjax-min.js?v=939kLBjL5_YKbx71T1RHjYaD4l8"></script>
<script type="text/javascript" src="/javascript/imagesloaded-min.js?v=XeuAp8Gc3mvQUo+wZCSF8ttPwvw"></script>
<script type="text/javascript" src="/javascript/figviewer-min.js?v=yPUa0sUQ_iHkI+IRv2i9bjyZJFo"></script>
<script type="text/javascript" src="/javascript/global-min.js?v=0Q3PwjeaWtXYDnqIsQvnL_ou0qs"></script>
<script type="text/javascript" src="/javascript/jquery.touchswipe-min.js?v=huaek_e6HqTduvCNAN91dJolTyw"></script>
<script type="text/javascript" src="/javascript/jquery.base64-min.js?v=VwV1zeVqKZj5FCAdlK0q5NRxbBg"></script>
<script type="text/javascript" src="/javascript/alm-min.js?v=Y5gm6B0b4Kx2YHNObNrgEeBgXlY"></script>
<script type="text/javascript" src="/javascript/taxonomy-browser-min.js?v=vBVMuDMYkGJCXIUxLe35GoyiJNw"></script>
<script type="text/javascript" src="/javascript/jquery.filterize-min.js?v=j0ZKVnHyk2nhFy8eIuNJkp7xaM0"></script>
<script type="text/javascript" src="/javascript/plosone-min.js?v=TK4H4arL_XBSwwJq+K1N3kqYfAI"></script>
<script type="text/javascript" src="/javascript/twitter-min.js?v=xKgcxLsQFXy+at1ao1NVke8nFlM"></script>
<script type="text/javascript" src="/javascript/crossmark.1.4-min.js?v=3FO4k0SjwTaGNnKGNSqthar1080"></script>
<script type="text/javascript">
  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>
<!-- <script type="application/javascript" src="http://crossmark.crossref.org/javascripts/v1.3/crossmark.min.js"></script> -->

</body>
</html>
